Nat Commun

. 2025 Feb 24;16:1915. doi: [10.1038/s41467-025-57239-1](https://doi.org/10.1038/s41467-025-57239-1)

# Glutathione triggers leaf-to-leaf, calcium-based plant defense signaling

[Rui Li](https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20R%22%5BAuthor%5D)

### Rui Li

1MOE Key Laboratory of Bioinformatics, Tsinghua-Peking Joint Center for Life Sciences, and School of Life Sciences, Tsinghua University, Beijing, 100084 China

2Shenzhen Branch, Guangdong Laboratory of Lingnan Modern Agriculture, Key Laboratory of Synthetic Biology, Ministry of Agriculture and Rural Affairs, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, China

Find articles by [Rui Li](https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20R%22%5BAuthor%5D)

1,2,#, [Yongfang Yang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20Y%22%5BAuthor%5D)

### Yongfang Yang

1MOE Key Laboratory of Bioinformatics, Tsinghua-Peking Joint Center for Life Sciences, and School of Life Sciences, Tsinghua University, Beijing, 100084 China

3Key Laboratory of Seed Innovation, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, 100101 China

Find articles by [Yongfang Yang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20Y%22%5BAuthor%5D)

1,3,#, [Hao Lou](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lou%20H%22%5BAuthor%5D)

### Hao Lou

1MOE Key Laboratory of Bioinformatics, Tsinghua-Peking Joint Center for Life Sciences, and School of Life Sciences, Tsinghua University, Beijing, 100084 China

Find articles by [Hao Lou](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lou%20H%22%5BAuthor%5D)

1, [Weicheng Wang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20W%22%5BAuthor%5D)

### Weicheng Wang

1MOE Key Laboratory of Bioinformatics, Tsinghua-Peking Joint Center for Life Sciences, and School of Life Sciences, Tsinghua University, Beijing, 100084 China

Find articles by [Weicheng Wang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20W%22%5BAuthor%5D)

1, [Ran Du](https://pubmed.ncbi.nlm.nih.gov/?term=%22Du%20R%22%5BAuthor%5D)

### Ran Du

2Shenzhen Branch, Guangdong Laboratory of Lingnan Modern Agriculture, Key Laboratory of Synthetic Biology, Ministry of Agriculture and Rural Affairs, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, China

Find articles by [Ran Du](https://pubmed.ncbi.nlm.nih.gov/?term=%22Du%20R%22%5BAuthor%5D)

2, [Haidong Chen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20H%22%5BAuthor%5D)

### Haidong Chen

2Shenzhen Branch, Guangdong Laboratory of Lingnan Modern Agriculture, Key Laboratory of Synthetic Biology, Ministry of Agriculture and Rural Affairs, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, China

Find articles by [Haidong Chen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20H%22%5BAuthor%5D)

2, [Xiaoxi Du](https://pubmed.ncbi.nlm.nih.gov/?term=%22Du%20X%22%5BAuthor%5D)

### Xiaoxi Du

1MOE Key Laboratory of Bioinformatics, Tsinghua-Peking Joint Center for Life Sciences, and School of Life Sciences, Tsinghua University, Beijing, 100084 China

4Hainan Key Laboratory for Biosafety Monitoring and Molecular Breeding in Off-Season Reproduction Regions, Institute of Tropical Bioscience and Biotechnology & San Ya Research Institute, Chinese Academy of Tropical Agricultural Sciences, Haikou, China

Find articles by [Xiaoxi Du](https://pubmed.ncbi.nlm.nih.gov/?term=%22Du%20X%22%5BAuthor%5D)

1,4, [Shuai Hu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20S%22%5BAuthor%5D)

### Shuai Hu

1MOE Key Laboratory of Bioinformatics, Tsinghua-Peking Joint Center for Life Sciences, and School of Life Sciences, Tsinghua University, Beijing, 100084 China

4Hainan Key Laboratory for Biosafety Monitoring and Molecular Breeding in Off-Season Reproduction Regions, Institute of Tropical Bioscience and Biotechnology & San Ya Research Institute, Chinese Academy of Tropical Agricultural Sciences, Haikou, China

Find articles by [Shuai Hu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20S%22%5BAuthor%5D)

1,4, [Guo-Liang Wang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20GL%22%5BAuthor%5D)

### Guo-Liang Wang

5Department of Plant Pathology, Ohio State University, Columbus, OH 43210 USA

Find articles by [Guo-Liang Wang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20GL%22%5BAuthor%5D)

5,✉, [Jianbin Yan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Yan%20J%22%5BAuthor%5D)

### Jianbin Yan

2Shenzhen Branch, Guangdong Laboratory of Lingnan Modern Agriculture, Key Laboratory of Synthetic Biology, Ministry of Agriculture and Rural Affairs, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, China

Find articles by [Jianbin Yan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Yan%20J%22%5BAuthor%5D)

2,✉, [Xiaoyi Shan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Shan%20X%22%5BAuthor%5D)

### Xiaoyi Shan

1MOE Key Laboratory of Bioinformatics, Tsinghua-Peking Joint Center for Life Sciences, and School of Life Sciences, Tsinghua University, Beijing, 100084 China

Find articles by [Xiaoyi Shan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Shan%20X%22%5BAuthor%5D)

1,✉, [Daoxin Xie](https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20D%22%5BAuthor%5D)

### Daoxin Xie

1MOE Key Laboratory of Bioinformatics, Tsinghua-Peking Joint Center for Life Sciences, and School of Life Sciences, Tsinghua University, Beijing, 100084 China

6State Key Laboratory of Hybrid Rice, Hunan Hybrid Rice Research Center, Hunan Academy of Agricultural Sciences, Changsha, 410125 China

Find articles by [Daoxin Xie](https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20D%22%5BAuthor%5D)

1,6,✉

* Author information
* Article notes
* Copyright and License information

1MOE Key Laboratory of Bioinformatics, Tsinghua-Peking Joint Center for Life Sciences, and School of Life Sciences, Tsinghua University, Beijing, 100084 China

2Shenzhen Branch, Guangdong Laboratory of Lingnan Modern Agriculture, Key Laboratory of Synthetic Biology, Ministry of Agriculture and Rural Affairs, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, China

3Key Laboratory of Seed Innovation, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, 100101 China

4Hainan Key Laboratory for Biosafety Monitoring and Molecular Breeding in Off-Season Reproduction Regions, Institute of Tropical Bioscience and Biotechnology & San Ya Research Institute, Chinese Academy of Tropical Agricultural Sciences, Haikou, China

5Department of Plant Pathology, Ohio State University, Columbus, OH 43210 USA

6State Key Laboratory of Hybrid Rice, Hunan Hybrid Rice Research Center, Hunan Academy of Agricultural Sciences, Changsha, 410125 China

✉

Corresponding author.

#

Contributed equally.

Received 2024 Jun 19; Accepted 2025 Feb 14; Collection date 2025.

© The Author(s) 2025

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](https://creativecommons.org/licenses/by/4.0/).

[PMC Copyright notice](/about/copyright/)

PMCID: PMC11850895  PMID: [39994230](https://pubmed.ncbi.nlm.nih.gov/39994230/)

## Abstract

Animals rely on nervous systems to cope with environmental variability, whereas plants are characterized by lack of nervous system but still have evolved systemic communication systems through signaling molecules that trigger long-distance defense signaling events when encountered with environmental challenges. Here, our genetic screening of the previously constructed hairpin RNA-based *Arabidopsis* library identifies a glutathione (GSH)-deficient mutant that has high accumulation of glutamate (Glu), a previously defined wound signal essential for activating plant defense, but disharmoniously exhibits attenuation of defense signaling events. We further uncover GSH as a critical signaling molecule that relies on GLUTAMATE RECEPTOR-LIKE 3.3 (GLR3.3) to trigger long-distance calcium-based defense signaling events in plants. Our findings offer new insights into highly sophisticated systemic defense systems evolved by plants to defend against herbivory and pathogen invasion.

**Subject terms:** Herbivory, Biotic, Wounding

---

Li et al. identified GSH as a signaling molecule that triggers wound-induced systemic [Ca2 +]cyt transmission, activates JA biosynthesis, and regulates plant defense responses, highlighting the complex mechanisms of plant systemic defense signaling.

## Introduction

Animals rely on well-developed nervous systems to dexterously take prompt movements and rapidly facilitate long-range information exchange in front of the changing environment. In mammalian nervous systems, many different types of chemical substances, including amino acids (e.g., glutamate), monoamines (e.g., dopamine, norepinephrine), acetylcholine, peptides (e.g., substance P), serve as neurotransmitters to bind their corresponding receptors, then activate ion channels, and consequently produce ion flux[1](#CR1), thereby inducing membrane potential changes that lead to signal propagation for neurotransmission[1](#CR1)–[4](#CR4).

Plants are characterized by lack of a nervous system but still have the ability to systematically communicate the environmental signals from the site of perception to the distal tissues[5](#CR5). In response to herbivorous insect attack or wounding at local leaves, wound signaling molecules are released at the damaged plant cells, and may bind corresponding receptors to trigger propagation of slow wave potentials (SWPs)[6](#CR6)–[9](#CR9), initiate transmission of cytoplasmic calcium ([Ca2+]cyt)[10](#CR10)–[15](#CR15), and consequently activate biosynthesis of defensive phytohormone jasmonic acid (JA), thereby mediating expression of defense-responsive genes for systemic defense responses in plants[7](#CR7),[8](#CR8),[11](#CR11),[12](#CR12),[16](#CR16)–[18](#CR18).

Many different types of chemical substances were revealed as neurotransmitters that induce long-distance ionic responses to effectively mediate neurotransmission in mammals[4](#CR4). However, in plant kingdom, only an amino acid and a protein, namely the glutamate (Glu) and the recently discovered “Ricca’s factor” β-THIOGLUCOSIDE GLUCOHYDROLASE (TGG) are so far defined as wound signals that depend on glutamate receptor-like (GLR) protein, the homologous protein of mammalian ionotropic Glu receptors (iGluRs), to activate systemic plant defense response[6](#CR6),[7](#CR7),[11](#CR11). In this study, our genetic screening of the previously constructed hairpin RNA-based *Arabidopsis* library[19](#CR19) identified a glutathione (GSH)-deficient mutant that highly accumulated Glu, the previously defined molecule crucial for activation of plant defense, but disharmoniously exhibited attenuation of defense signaling events. We further uncovered GSH as a critical signaling molecule that relies on GLR3.3 to trigger long-distance propagation of calcium-based defense signaling events.

## Results

### Identification of GSH as a signaling molecule that activates long-distance plant defense signaling events

During the genetic screening of the rolling circle amplification-mediated hairpin RNA (RMHR)-based *Arabidopsis* library[19](#CR19),[20](#CR20) for mutants with altered defense response to a disastrous plant fungal pathogen *Botrytis cinerea* (*B. cinerea*), we identified a mutant, termed *hairpin RNA-52* (*hr-52*), which exhibited obvious susceptibility to *B. cinerea* infection (Supplementary Fig. [1](#MOESM1)). A single gene *AT4G23100* was identified with PCR analysis on the mutant genomic DNA[19](#CR19),[20](#CR20).

To further verify that the decreased resistance in the *hr-52* mutant is caused by the interference with *AT4G23100* gene, we obtained the previously reported mutant *pad2-1* with the Ser298Asn substitution (S298N) within AT4G23100[21](#CR21) to assess its responses to *B. cinerea* infection and insect attack. The results showed that *pad2-1* exhibited severe susceptibility to *B. cinerea* and *Spodoptera exigua*, a globally agricultural pest, while genetic transformation of *pad2-1* with *PAD2* (*AT4G23100*) completely restored the resistance (Supplementary Figs. [2](#MOESM1) and [3](#MOESM1)). Our results together with the previous findings[21](#CR21)–[23](#CR23) demonstrate that *AT4G23100* gene is essential for plant defense response against *B. cinerea* infection and insect attack.

It was known that the *AT4G23100* gene encodes the gamma-glutamylcysteine synthetase catalyzing the reaction between Glu and cysteine for generation of glutamylcysteine in the GSH biosynthesis pathway[24](#CR24). We found that a high level of Glu (875.42 μg/g FW) was accumulated in the *pad2-1* mutant compared to WT (494.96 μg/g FW) while the GSH content in *pad2-1* (4.91 μg/g FW) was severely decreased to 5.9% of WT (82.94 μg/g FW) (Fig. [1a, b](#Fig1)). It is surprising that such a high level of Glu failed to induce plant defense in *pad2-1* (Supplementary Figs. [2](#MOESM1) and [3](#MOESM1)), which seems to be inconsistent with the previously defined role of Glu as a critical wound signal that activates plant defense response[11](#CR11). The observed susceptibility in the GSH-deficient *pad2-1* mutant suggests an indispensable role for GSH in plant defense, which encourages us to further investigate whether GSH acts as wound signal to trigger long distance calcium-based defense signaling events.

#### Fig. 1. Defense signaling events are attenuated in GSH-deficient *pad2-1* mutant plant.

[![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c18c/11850895/da137f212eb8/41467_2025_57239_Fig1_HTML.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11850895_41467_2025_57239_Fig1_HTML.jpg)

[Open in a new tab](figure/Fig1/)

**a**, **b** Contents of GSH (**a**) and Glu (**b**) in Col-0, *pad2-1*, *PAD2*/*pad2-1*#1 and *PAD2*/*pad2-1*#3 plants. Data are mean ± SD (*n* = 5). Statistical significance was determined by Dunnett’s test. **c**–**e** JA (**c**), JA-Ile (**d**) contents and defense gene expression levels (**e**) in target leaf 6 before and 30 min after wounding leaf 1 of Col-0 and *pad2-1*. Data are mean ± SD (*n* = 3 for leaf 6 from unwounded plants), *n* = 8 for leaf 6 from wounded plants in (panel **c**, **d**); *n* = 3 for leaf 6 from unwounded plants, *n* = 21 for wounded *pad2-1* and *n* = 23 for wounded Col-0 plants in (panel **e**). Statistical significance was determined by two-sided Welch’s *t*-test. **f** [Ca2+]cyt fluorescence signal imaging in target leaf 6 of *GCaMP3* and *GCaMP3*/*pad2-1* plants upon cutting leaf 1 (0 s). Scale bars: 1 mm. **g**, **h** Quantitative measurement of [Ca2+]cyt levels (**h**) and maximum [Ca2+]cyt fluorescence changes (**i**) in target leaf 6 after wounding leaf 1 of *GCaMP3* and *GCaMP3*/*pad2-1* plants. Data are mean ± SD (*n* = 14 for GCaMP3, *n* = 16 for *GCaMP3*/*pad2-1*). Statistical significance was determined by two-sided Welch’s *t*-test. **i** Percentage of cases with [Ca2+]cyt transmission for each systemic leaf in *GCaMP3* and *GCaMP3/pad2-1* plants wounded at leaf 1. 16 *GCaMP3* and 17 *GCaMP3/pad2-1* plants were subjected to observation of [Ca2+]cyt fluorescence and only systemic leaf showing obvious [Ca2+]cyt fluorescence was calculated as a case with [Ca2+]cyt transmission. Statistical significances compared to *GCaMP3* was assessed using Fisher’s exact test, indicated as follows: \**P* < 0.05, \*\*\**P* < 0.001, \*\*\*\**P* < 0.0001, n.s. denotes not significant. Source data are provided as a Source Data file.

To verify the role of GSH in activating long-distance plant defense signaling events, we wounded leaf 1 of *pad2-1* and Col-0, and then measured the accumulation of wound-induced plant defensive hormonal molecules in leaf 6, considering that leaf n shares direct vascular connections with leaves n ± 5 and n ± 8 while leaves n ± 3 may represent contact parastichies formed by proximal but unconnected vasculature in adult *Arabidopsis* rosettes[7](#CR7),[25](#CR25). Consistent with the reduced plant resistance in *pad2-1*, the defensive phytohormone jasmonates were obviously decreased in the leaf 6 of *pad2-1* to ~63.04% (for JA) and ~45.81% (for JA-Ile) of those in Col-0 upon wounding leaf 1 (Fig. [1c, d](#Fig1)). Moreover, the expression of JA-inducible defense marker genes *OPR3*, *JAZ5*, and *JAZ7* was also significantly attenuated in leaf 6 of *pad2-1* compared to Col-0 upon wounding leaf 1 (Fig. [1e](#Fig1)). These results suggest that GSH might serve as a critical signaling molecule essential for activation of plant systemic defense signaling events.

To further explore the underlying mechanism of GSH in activating systemic plant defense responses, we introduced the [Ca2+]cyt reporter GCaMP3 into *pad2-1*, generating *GCaMP3*/*pad2-1* plant, to investigate whether the GSH-deficient mutant attenuated the wound-induced systemic [Ca2+]cyt transmission. Upon mechanical wounding of leaf 1, both amplitude and kinetics of [Ca2+]cyt propagation in leaf 6 were reduced in *GCaMP3*/*pad2-1* compared with *GCaMP3* plants (Fig. [1f, g](#Fig1); Supplementary Fig. [4](#MOESM1); Supplementary Movies [1](#MOESM8) and [2](#MOESM9)). The peak value of [Ca2+]cyt fluorescence changes in leaf 6 of *GCaMP3/pad2-1* was reduced to ~44.15% of that in *GCaMP3* (Fig. [1h](#Fig1)). Consistently, the percentage of each systemic leaf that showed apparent increased [Ca2+]cyt fluorescence was reduced in *GCaMP3/pad2-1* (Fig. [1i](#Fig1)). These data reinforce the role of GSH in triggering the wound-induced systemic [Ca2+]cyt propagation in plants.

Taken together, the above-mentioned data suggest that GSH might serve as an important wound signal that triggers wound-induced systemic [Ca2+]cyt transmission, activates JA biosynthesis, and regulates plant defense responses.

### GSH triggers long-distance [Ca2+]cyt transmission

To further validate that GSH acts as a wound signal triggering long-distance plant defense signaling events, we applied GSH (with Glu and sorbitol as positive and negative controls, respectively) to leaf 1 of the *GCaMP3* plants and visualized the systemic [Ca2+]cyt in the whole seedlings (Fig. [2a, b](#Fig2)). Intriguingly, the fast and transient increase of [Ca2+]cyt was notably triggered in systemic leaves (including leaf 3, 4, and 6) by GSH (Supplementary Movie [3](#MOESM10)) at the level comparable to that induced by Glu (Fig. [2a](#Fig2)). Specifically, [Ca2+]cyt rapidly increased to the peak value in leaf 6 at ~230 s upon treatment on leaf 1 with GSH rather than sorbitol (Fig. [2c](#Fig2); Supplementary Fig. [4](#MOESM1)), and more distal leaves showed [Ca2+]cyt propagation with the increased GSH concentrations (Supplementary Fig. [5](#MOESM1)). Moreover, the GSH-triggered systemic [Ca2+]cyt transmission was dramatically repressed by pretreatment on the petiole of the wounded leaf with the Ca2+ channel inhibitor LaCl3 (Fig. [2e-g](#Fig2)). To exclude the possibility that GSH is cleaved by enzymes to release glutamate for Ca²⁺ signal propagation, we employed GGsTop, a well-known chemical inhibitor that specifically targets gamma-glutamyl transpeptidase (GGT), the enzyme responsible for converting GSH to Glu[26](#CR26). Our results showed that there were no significant differences in the Ca²⁺ signaling response when GSH was used in combination with GGsTop compared to that of GSH treatment alone (Supplementary Fig. [6](#MOESM1)), suggesting that GSH was not converted to Glu for activation of systemic Ca2+ signaling. Additionally, we synthesized another tripeptide (Glu-Pro-Ala), which contains glutamate along with two different amino acids, and investigated its effects in triggering systemic Ca2+ signal propagation in plants. Unlike GSH treatment, application of Glu-Pro-Ala failed to induce long-distance transmission of Ca²⁺ signal in plants (Supplementary Fig. [7](#MOESM1)), reinforcing the role of GSH as a signaling molecule that effectively triggers systemic [Ca2+]cyt propagation in plants.

#### Fig. 2. GSH triggers [Ca2+]cyt propagation and defense gene expression in systemic leaves.

[![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c18c/11850895/0f3ecca2c1f1/41467_2025_57239_Fig2_HTML.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11850895_41467_2025_57239_Fig2_HTML.jpg)

[Open in a new tab](figure/Fig2/)

**a** [Ca2+]cyt fluorescence signal imaging of *GCaMP3* plants after 100 mM GSH, Glu, and sorbitol (sorb) application to leaf 1 (L1) (white arrow, 0 s) respectively. White arrowheads (200 s, 400 s) indicate leaf 6 (L6) and leaf 3 (L3). Scale bars: 2 mm. **b** Schematic diagram for application of small molecule to the cut edge of damaged leaf 1. Small molecule was added to the cut edge of the damaged leaf 30 min after wounding for assessment of small molecules-induced Ca2+ signal transmission. In contrast, a longer recovery period of 5 h post pre-cutting was required for the assessment of small molecules-activated gene expression levels. **c**, **d** Quantitative measurement of [Ca2+]cyt levels (**c**) and expression levels of defense genes (**d**) in target leaf 6 at indicated times after application of 100 mM GSH, Glu, and sorbitol to leaf 1 of *GCaMP3* plants. Data are mean ± SD (*n* = 3, 9, 9 for sorbitol, GSH and Glu treatment in **c**, n = 3 for d). Statistical significance was determined by Dunnett’s test. **e** [Ca2+]cyt fluorescence signal imaging of *GCaMP3* plants pretreated with LaCl3 or Silwet-L77 after 100 mM GSH application to leaf 1 (L1) (white arrow, 0 s). White arrowhead (400 s) indicates leaf 6 (L6). Scale bars: 2 mm. **f** Schematic diagram of LaCl3 pretreatment before GSH treatment. LaCl3 (50 mM) was added to a strip of Kimwipes laid over the petiole of leaf 1 (L1). 0.05% (v/v) Silwet-L77 was added as a wetting agent to improve LaCl3 penetration of the cuticle. **g**, **h** Quantitative measurement of [Ca2+]cyt levels (**g**) and defense gene induction (**h**) in target leaf 6 after application of 100 mM GSH to leaf 1 of *GCaMP3* plant pretreated with LaCl3 or Silwet-L77 at indicated times. Data are mean ± SD (*n* = 5 for g, *n* = 3 for h). Statistical significance was determined by two-sided Welch’s *t*-test. Source data are provided as a Source Data file.

Consistently, GSH application on leaf 1 significantly induced the expression of defense-responsive genes (*OPR3*, *JAZ5*, and *JAZ10*) in leaf 6 (by ~5.6, ~8.6 and ~5.7 folds), which is comparable to that induced by Glu (Fig. [2d](#Fig2)). Moreover, pretreatment on the petiole of the wounded leaf with LaCl3 severely reduced the GSH-inducible expression of defense genes in systemic leaves (Fig. [2h](#Fig2)), verifying that propagation of [Ca2+]cyt is required for GSH-induced defense gene expression. Collectively, these results demonstrate that GSH acts as an important wound signal in triggering leaf-to-leaf, calcium-based plant defense signaling.

### GSH activates the transcriptional expression of defense-related genes in plant systemic leaves

To get deeper insights into the GSH-triggered systemic defense signaling events in plants, we performed transcriptome profiling on leaf 6 collected after 1  h application of GSH to leaf 1 of WT plants. We identified 378 differentially expressed genes (DEGs) in the systemic leaf of GSH-treated plants compared to non-stimulated mock control plants (Supplementary Data [1](#MOESM3)). Among them, 354 genes including a large number of key defense-responsive genes were upregulated while 24 genes were downregulated (Fig. [3a](#Fig3)), which is well consistent with the tendency of transcriptome data from plants in response to wounding, namely, more transcripts are upregulated than downregulated[27](#CR27),[28](#CR28). Gene Ontology analysis (GO) of these DEGs further revealed significant enrichment of terms associated with plant defense signaling, including defense response, cellular responses to JA, response to wounding, and response to hormone (Fig. [3b](#Fig3); Supplementary Data [2](#MOESM4)). Consistent with the transcriptome analysis, qRT-PCR also showed that expression of defense marker genes (*CYP94C1*, *JAZ13*, *LOX4*, *OPR3*, *JAZ5*, *JAZ7*, and *JAZ10*) was highly induced in the systemic leaf tissues of GSH-treated plants (Fig. [3c, d](#Fig3); Supplementary Fig. [8](#MOESM1)). These data provide an overview of the GSH-induced gene expression in plant systemic leaf tissues and further indicate the important role of GSH in triggering plant long-distance defense signaling.

#### Fig. 3. Transcriptome analysis of GSH-induced defensive gene expression in plant systemic leaves.

[![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c18c/11850895/efe4a1c8964a/41467_2025_57239_Fig3_HTML.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11850895_41467_2025_57239_Fig3_HTML.jpg)

[Open in a new tab](figure/Fig3/)

**a** Volcano plot of differentially expressed genes (DEGs) in target leaf 6 of WT plants after application of 100 mM GSH to leaf 1 compared to non-stimulated mock control plants. **b** Gene ontology (GO) analysis of the responsive genes in target leaf 6 of GSH-treated plants. DEGs in (**a**) were used as input for GO analysis. **c** Heatmap showing the relative expression changes of defense-responsive genes in the target leaf 6 of GSH-treated plants and non-stimulated mock control plants. FP, Family Protein; rep, replicate. **d** The relative expression of selected defense-responsive genes in the target leaf 6 of GSH-treated plants and non-stimulated mock control plants determined by qRT-PCR. Data are mean ± SD (*n* = 3). Statistical significance was determined by two-sided Welch’s *t*-test. **e** Venn diagram showing the overlap of DEGs in wounding vs mock, GSH treatment vs mock, and Glu treatment vs mock. Source data are provided as a Source Data file.

To further investigate the relations between GSH and Glu in plant wound response, we generated transcriptome data sets from leaf 6 1 h after wounding leaf 1 (Supplementary Data [3](#MOESM5)) or application of Glu to leaf 1 (Supplementary Data [4](#MOESM6)), and then compared these data to that induced by GSH. This comparison showed that 207 DEGs were co-regulated by wounding and GSH treatment, while 228 DEGs were found in both GSH and Glu treatment groups. Moreover, a total of 267 DEGs from GSH- and Glu-treated groups existed in wounding treatment, among which 126 genes were co-regulated by GSH, Glu and wounding, 81 genes were only regulated by GSH and wounding, and 60 genes were only regulated by Glu and wounding (Fig. [3e](#Fig3)). These overlapped genes which are co-regulated by GSH, Glu, and wounding or independently regulated by GSH and wounding, Glu and wounding are all mainly involved in plant defense signaling, including plant response to wounding, stress, JA, and biotic stimulus, regulation of cell communication, and innate immune response, suggesting that GSH and Glu are involved in wound-induced plant systemic defense response.

### GLR3.3 is required for GSH-triggered systemic [Ca2+]cyt propagation

GLR proteins which generally contain an amino-terminal domain (ATD), a ligand-binding domain (LBD), a transmembrane domain (TMD), and a carboxyl-terminal domain (CTD), have broad agonist profiles including glycine, cysteine, methionine, Glu and GSH[29](#CR29)–[32](#CR32). Specially, Glu-triggered long-distance [Ca2+]cyt propagation is dramatically inhibited in *glr3.3glr3.6* mutant[11](#CR11). To figure out whether GSH-triggered long-distance calcium signaling depends on *GLR3.3* and *GLR3.6*, we compared the GSH-triggered Ca2+-related signaling events among the *glr3.3*, *glr3.6* and WT plants containing the GCaMP3 reporter (Fig. [4a, b](#Fig4); Supplementary Fig. [9](#MOESM1)). Interestingly, the GSH-triggered systemic [Ca2+]cyt transmission was severely reduced in the *GCaMP3/glr3.3* (Fig. [4a, b](#Fig4); Supplementary Movie [4](#MOESM11)). However, the GSH-triggered systemic [Ca2+]cyt transmission was not markedly attenuated in *GCaMP3/glr3.6*, which is similar to that of *GCaMP3* plant (Fig. [4a, b](#Fig4); Supplementary Movie [5](#MOESM12)). Consistently, the GSH-inducible expression of defense marker gene *OPR3*, *JAZ5*, and *JAZ10* was significantly repressed in the *GCaMP3/glr3.3*, but not obviously inhibited in the *GCaMP3/glr3.6* and the *GCaMP3* plants (Fig. [4c](#Fig4)). These results suggest that GLR3.3 plays a dominant role in regulating GSH-induced [Ca2+]cyt propagation. Similar to the systemic calcium signal transmission upon GSH treatment, Glu-induced [Ca2+]cyt propagation in systemic leaves were strongly inhibited in *GCaMP3/glr3.3* while remained unaffected in *GCaMP3/glr3.6* (Supplementary Fig. [10](#MOESM1)), confirming that Glu-induced long-distance calcium wave is also dependent on GLR3.3[8](#CR8),[11](#CR11).

#### Fig. 4. GSH induces systemic [Ca2+]cyt propagation and defense gene expression in a GLR3.3-dependent manner.

[![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c18c/11850895/1d9ec335e113/41467_2025_57239_Fig4_HTML.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11850895_41467_2025_57239_Fig4_HTML.jpg)

[Open in a new tab](figure/Fig4/)

**a** [Ca2+]cyt fluorescence signal imaging of *GCaMP3*, *GCaMP3*/*glr3.3* and *GCaMP3*/*glr3.6* plants after 100 mM GSH application (white arrow, 0 s) to leaf 1 (L1). White arrowhead (450 s) indicates leaf 6 (L6). Scale bars: 2 mm. **b**, **c** Quantitative measurement of [Ca2+]cyt levels (**b**) and expression levels of defense genes (**c**) in target leaf 6 at indicated times after 100 mM GSH application to leaf 1 of *GCaMP3*, *GCaMP3*/*glr3.3* and *GCaMP3*/*glr3.6* plants respectively. *GCaMP3* data from Fig. [2c](#Fig2) are reproduced (gray lines) to aid in comparison. Data are mean ± SD (*n* = 5, 9, 9 for *GCaMP3*/*glr3.3*, *GCaMP3*/*glr3.6*, *GCaMP3* in (**b**), *n* = 3 for (**c**)). Statistical significance was determined by two-sided Welch’s *t*-test. Source data are provided as a Source Data file.

To characterize the interaction between GLR3.3 and GSH, the predicted structure of GLR3.3 was used to perform molecular docking. The binding model showed that Glu binds to the LBD of GLR3.3 (referred to as GLR3.3LBD) (Supplementary Fig. [11](#MOESM1)), which is consistent with the recent report about the Glu binding to the GLR3.3LBD[29](#CR29). While GSH was predicted to bind the ATD of GLR3.3 (referred to as GLR3.3ATD), which is remarkably different from Glu in terms of binding domains and contacting amino acids (Supplementary Fig. [11](#MOESM1)). Consistent with the predicted interaction between GLR3.3ATD and GSH (Supplementary Fig. [11](#MOESM1)), recent X-ray crystallography and cryo-EM data revealed that GSH binds the ATD of GLR3.4[30](#CR30), another *GLR* protein with high sequence similarity to GLR3.3, reinforcing that GSH might bind the ATD domain of GLR3.3 for activation of ion channel. Together with the genetic evidence showing GLR3.3 is required for both GSH- and Glu-triggered systemic [Ca2+]cyt transmission (Fig. [4](#Fig4); Supplementary Fig. [10](#MOESM1)), these results collectively suggest that GSH and Glu may bind to different domains of GLR3.3 for activating ion channel activity and triggering leaf-to-leaf, calcium-based plant defense signaling.

Taken together, our results shown in Figs. [1](#Fig1)–[4](#Fig4) suggest that GSH serves as a critical wound signal to trigger long-distance propagation of [Ca2+]cyt waves in a GLR3.3-dependent manner, thus inducing rapid JA biosynthesis in systemic leaf organs to protect plants from herbivory ahead of time.

## Discussion

Initiation of a rapid defense against biotic attacks and mechanical damages is crucial for plant survival in changing environment. Due to lack of a nervous system, plants have evolved sophisticated mechanisms to systemically deliver the wound signal from the site of injured organs to the distal undamaged tissues, which helps them be well-prepared for upcoming dangers[33](#CR33). In this study, we find that the GSH-deficient mutant, in spite of the highly accumulated Glu previously defined as a critical wound signal essential for activating plant defense events, showed impaired plant defense signaling events (Fig. [1](#Fig1); Supplementary Figs. [1](#MOESM1)-[3](#MOESM1)). We further define GSH as a critical wound signal that depends on GLR3.3 to trigger long-distance calcium-based defense signaling in plants (Figs. [2](#Fig2)–[4](#Fig4); Supplementary Figs. [5](#MOESM1)-[11](#MOESM1)). Intriguingly, in the animal nervous system, GSH has been identified as a neuromodulator/neurotransmitter[34](#CR34)–[36](#CR36), suggesting a potential conceptual parallel in GSH-mediated signal transmission across different kingdoms of life. Moreover, we noticed that GSH and Glu induced distinct [Ca2+]cyt transmission patterns in systemic leaves. Specifically, the spread of systemic [Ca2+]cyt triggered by GSH is preferentially initiated from marginal veins and rapidly transmitted to the mid veins (Supplementary Fig. [12](#MOESM1) and Supplementary Movie [6](#MOESM13)), whereas Glu mediated [Ca2+]cyt transmission mainly propagated from mid veins to marginal veins (Supplementary Fig. [12](#MOESM1) and Supplementary Movie [7](#MOESM14)). Re-examination of the data shown in the previous study revealing Glu as a wound signal also supports our observation that Glu triggers [Ca2+]cyt transmission with the direction of mid-to-marginal veins[11](#CR11). Intriguingly, the transmission pattern of wounding-induced [Ca2+]cyt contained both directions of the mid-to-marginal veins and the marginal-to-mid veins (Supplementary Fig. [12](#MOESM1) and Supplementary Movie [8](#MOESM15)). Future work to elucidate the mechanisms and biological significance of different [Ca2+]cyt transmission patterns induced by GSH and Glu would help us to further understand the highly sophisticated systemic defense systems evolved by plants to defend against insect attack.

When plants are subjected to insect herbivory or mechanical wounding, they rapidly activate the JA signaling pathway in whole plant tissues, which in turn mediates the systemic defense responses against insects[10](#CR10)–[12](#CR12),[37](#CR37). In our study, we defined GSH as a critical wound signal to trigger long-distance [Ca2+]cyt transmission for activation of JA biosynthesis in distal organs, thereby inducing plant systemic defense response (Figs. [1](#Fig1)–[4](#Fig4)). Interestingly, as shown in our transcriptome data, in addition to the pronounced enrichment of genes associated with JA signaling in systemic leaf tissues after GSH treatment, genes responsive to other defense phytohormones such as ethylene and salicylic acid, which are responsible for plant resistance to pathogen infection, are also highly upregulated (Supplementary Data [1](#MOESM3)), implying that GSH might also act as a signaling molecule to activate the plant systemic immune response. Previously recognized as an antioxidant, GSH is intricately linked with reactive oxygen species (ROS) signaling[38](#CR38). Our transcriptome analysis also revealed that genes responsive to oxidative stress were significantly induced in the systemic leaf tissues following GSH treatment (Supplementary Data [1](#MOESM3)). Given that both calcium and ROS signals play a role in mediating plant systemic defense responses to wounding[11](#CR11),[12](#CR12),[39](#CR39)–[41](#CR41), exploring the potential involvement of ROS in GSH-triggered systemic defense response would be interesting.

Wound-induced systemic defense signaling requires the involvement of GLR ion channel proteins, such as GLR3.3 and GLR3.6. Our study found that the GSH-triggered long-distance calcium-mediated defense response was suppressed in *GCaMP3/glr3.3* but nearly unaffected in *GCaMP3/glr3.6* (Fig. [4](#Fig4)). Similar to GSH, it was also observed that Glu depends on GLR3.3 rather than GLR3.6 to initiate the systemic calcium-based defense signaling (Supplementary Fig. [10](#MOESM1)). Considering the recent X-ray crystallography and cryo-EM data, which revealed that GSH and Glu could bind to different domains of GLR3.4 for ion channel activation[30](#CR30), it would be intriguing to investigate whether GSH binds to GLR3.3 and further determine the potential relationship between GSH and Glu in activating GLR3.3. Furthermore, in contrast to GSH and Glu, the recently identified Ricca’s factor TGGs, was shown to rely on both GLR3.6 and GLR3.3 to trigger the systemic defense response[6](#CR6). It would be exciting to further uncover the signal interactions among Glu, TGGs, and GSH in wound-induced systemic defense signaling. Given that various chemical substances have been identified as neurotransmitters/neuromodulators in mammals, eliciting long-distance ionic responses that effectively mediate neurotransmission, identification of more systematic signaling molecules in the future would offer new insights into the complex systemic defense signaling in plants.

The successful transmission of neural signals from one cell to another in animals relies on the mutual conversion of chemical signals (neurotransmitters) and electrical signals, where ion fluxes are induced by neurotransmitters and the release of neurotransmitters is dependent on electrical signals[1](#CR1),[2](#CR2),[42](#CR42). In contrast, little is known about how signaling substances are transmitted in plants to activate long-distance signaling. Previous studies suggest that the dispersal of elicitors such as Glu may rely on bulk flow and diffusion[14](#CR14). Given that GSH has a molecular weight similar to that of amino acids, a plausible scenario is that when plants experience mechanical damage or insect attack, a large volume of GSH from the intracellular region may surge into the apoplastic region, inducing calcium influx by activation of the GLR3.3 ion channel. With the action of diffusion and bulk flow of GSH within the vascular bundles, calcium signals can be rapidly transmitted between leaves. In addition to the diffusion and bulk flow theory, other theories have also been proposed, such as the osmoelectric siphon model, which is based on membrane depolarization leading to cell water shedding into the apoplast followed by membrane repolarization and water uptake[43](#CR43). More recently, osmotic pressure has been suggested to induce the systemic apoplastic release of signaling molecules like Glu[15](#CR15). Future studies aimed at uncovering the potential mechanisms underlying GSH-triggered systemic Ca2+ signal transmission would be fascinating.

## Methods

### Plant materials and growth condition

*hairpin RNA-52 (hr-52)* was obtained from the rolling circle amplification-mediated hairpin RNA (RMHR)-based *Arabidopsis* library in our previous study[19](#CR19),[20](#CR20). *pad2-1* was ordered from the Nottingham Arabidopsis Stock Centre (NASC). Seeds of *GCaMP3* were from Simon Gilroy (Department of Botany, University of Wisconsin). The *GCaMP3/pad2-1* was generated by genetic crosses using standard procedures. *GCaMP3/glr3.3* and *GCaMP3/glr3.6* mutants were generated by CRISPR/Cas9 in WT plants overexpressing *GCaMP3*. To generate the transgenic plant expressing *35S::PAD2-6myc/pad2-1* (*PAD2*/*pad2-1*), the coding sequence of *PAD2* (*AT4G23100*) was cloned into the binary vector *pCAMBIA1300*, and then transformed into *pad2-1*.

*Arabidopsis* seeds were sterilized with 20% bleach, sown on agar plates containing Murashige and Skoog (MS) medium. After incubation in the dark at 4 °C for 2 days, plates were cultivated in a growth chamber at 22 °C with a 16 h/8 h light/dark (long day, LD) photoperiod. After 7 days, seedlings were then transplanted into soil and grown under the same condition. For insect feeding and pathogen inoculation assays, plants were grown in soil for 4–5 weeks at 22 °C with a 10 h/14 h light/dark (short day, SD) photoperiod. For real-time calcium imaging assay, plates were incubated at 4 °C for 2 days in the dark and then transferred to 22 °C with a SD photoperiod for 6–7 days. Seedlings were then transferred onto new plates and grown for additional 14–16 days prior to use.

### Real-time [Ca2+]cyt imaging in the entire plant

Transgenic *Arabidopsis* plants stably expressing *GCaMP3* were imaged with a motorized epifluorescence (SZX10, Olympus) stereomicroscope equipped with a 1 × objective lens and a [C13440](https://www.ncbi.nlm.nih.gov/nuccore/C13440) digital camera (ORCA-Flash4.0 V3, Hamamatsu Photonics). The GFP-based Ca2+ indicator *GCaMP3* was excited using mercury lamp (X-Cite 120 Q, Excelitas). The green fluorescent signal was acquired every 2 s with the [C13440](https://www.ncbi.nlm.nih.gov/nuccore/C13440) camera using Standard imaging software (Olympus).

Using ImageJ software, GCaMP3 signals were analyzed overtime at regions of interest (ROI). The equation ΔF/F0 = (F-F0)/F0 was used to calculate the fractional fluorescence changes. F0 denotes the average baseline fluorescence determined by the average of F over the frames of the recording before treatment. The entire leaf area was selected as ROI for Ca2+ signal analysis.

### Application of amino acids, tripeptides and chemical agents

GSH, Glu-Pro-Ala, L-Glu, and sorbitol were dissolved in growth medium [1/2 × MS salts, 1% (w/v) sucrose and 0.05% (w/v) MES], which was then adjusted to pH 5.1. 5 μL of the solution was applied to the cut edge of leaf 1 after 30 min recovery period after pre-cutting for assessment of small molecules-induced Ca2+ signal transmission. In contrast, a longer recovery period of 5 h post pre-cutting was required for the assessment of small molecules-activated gene expression levels. GSH at concentrations of 0.001 mM, 0.01 mM, 0.1 mM, 1 mM, 5 mM, 10 mM, 25 mM, 50 mM, and 100 mM, which are within or exceeding its physiological concentrations, were selected for analyzing GSH-triggered systemic calcium-based defense signaling in vitro. GGsTOP (HY-108467, MCE) was dissolved in DMSO and diluted with water. LaCl3 was dissolved into 0.05% (v/v) Silwet L-77 in water to make a 50 mM solution and then 10 μL was applied to a piece of Kiwmwipes placed on a petiole for at least 30 min prior to adding GSH.

### CRISPR/Cas9 vector construction and plant transformation

Egg cell-specific promoter-controlled CRISPR/Cas9 system was used to generate *glr3.3* and *glr3.6* single mutants. CRISPR/Cas9 vector was constructed according to previous studies[44](#CR44). Target sequence was selected by *CRISPR-PLANT* (<https://www.genome.arizona.edu/crispr/>). All the vectors were transformed into *Agrobacterium tumefaciens* strain GV3101 using electroporation. *Arabidopsis* plants were transformed using the floral dip methods. The primers used for cloning are shown in Supplementary Data [5](#MOESM7).

### DNA extraction and mutation detection

1–5 mg frozen leaf tissues were used for DNA extraction through SDS method. The extracted genomic DNA was then used as a template to amplify the desired fragments in each of the target genes using primers flanking the target sites. PCR products were directly sequenced using Sanger method to identify mutation. Superimposed sequence chromatograms produced by biallelic and heterozygous mutations were decoded by DSDecode (<http://dsdecode.scgene.com/>) and manual analyses.

### Total RNA isolation, cDNA synthesis, and qRT-PCR

TransZol RNA kit (ET101-01, TransGen Biotech) was used to extract total RNA from *Arabidopsis* plants according to manufacturer’s instructions. cDNA was synthesized from the total RNA in a 20 μL reaction by cDNA Synthesis SuperMix kit (AT311-03, TransGen Biotech). qRT-PCR was performed in QuantStudio 3 realtime PCR system (Applied Biosystems, USA) using ChamQ SYBR qPCR Master Mix (Low ROX Premixed) (Q331-02, Vazyme). The 2-ΔΔCt analysis method was used to calculate the relative expression levels of each sample.

### JA and JA-Ile quantification in leaf tissues

Metabolite extraction and quantification of JA and JA-Ile in leaf tissues were performed according to previously described methods using AB Sciex 4500 QTRAP triple quadrupole mass spectrometer (AB SCIEX, MA, USA) equipped with an ACQUITY UPLC™ BEH C18 column (Waters, Eschborn, Germany) (50 × 2.1 mm, 1.7 μm)[45](#CR45). d5-JA was added to the extraction solvent (isopropanol: formic acid at 99.5: 0.5, v/v) as internal standard. Multiple reaction monitoring (MRM) was applied in negative mode with mass transitions (precursor ions/product) as followed: 209.12 > 59.01 for JA, 322.20 > 130.09 for JA-Ile, and 214.15 > 62.03 for d5-JA. A 10 min gradient elution program was applied for target compounds separation with 0.1% formic acid in acetonitrile (A) and pure water (B) as mobile phase with a 200 μL/min flow rate. Then, mass spectra data were processed and calculated using AB SCIEX analyst software (version 1.6.3, AB SCIEX, MA, USA) with calculation curves made with standards.

### Measurement of endogenous Glu and GSH in *Arabidopsis*

Leaf tissues were collected and directly ground into powder in liquid nitrogen. Then, 100 mg of powder was weighed accurately and transferred into a 2 mL centrifuge tube. 10 µL of stable isotope internal standard (5 µg/mL 13C2-15N-GSH, Cambridge Isotope Laboratories) was added to each sample before extraction. Then, 1.5 mL of extraction buffer (80% methanol with 0.1% formic acid, v/v) was added, with vortexing for sample resuspension. After a 15 min centrifugation at 14000 × *g*, the supernatants were diluted 10 times, and 100 µL was transferred to sample vials for further detection.

Target compounds were detected with a 6500 plus QTrap mass spectrometer (AB SCIEX, USA) coupled with a ACQUITY UPLC H-Class system, equipped with a heated electrospray ionization (HESI) probe. Extracts were separated by a synergi Hydro-RP column (2.0 × 100 mm, 2.5 μm, phenomenex). A binary solvent system was used, in which mobile phase A consisted of 2 mM Triisobutylamine adjusted with 5 mM acetic acid in 100% water, and mobile phase B of 100% methanol. An 8 min gradient with flow rate of 500 μL/min was used as follows: 0–1.0 min at 2% B; 1.0-6 min, 5–40% B; 6–7 min, 98% B; 7.1-8 min, 2% B. Column chamber and sample tray were held at 35 °C and 10 °C, respectively. Data were acquired in multiple reaction monitor (MRM) mode with mass transitions (precursor ions/product) as followed: 308.1 > 179.1 for GSH, 147.9 > 83.9 for Glu and 311.0 > 181.9 for 13C2-15N-GSH. The nebulizer gas (Gas1), heater gas (Gas2), and curtain gas were set at 55, 55, and 35 psi, respectively. The ion spray voltage was 4500 V in positive mode. The optimal probe temperature was determined to be 500 °C. Calibration curves of Glu and GSH were made with standards, and the concentration of GSH was calculated by internal-standard method with 13C2-15N-GSH as internal standard, while the concentration of Glu was calculated by external-standard method. The SCIEX OS software (version 1.6, AB SCIEX, MA, USA) was applied for metabolite identification and concentration calculation.

### Insect feeding assay with *Spodoptera exigua*

*S. exigua* larvae (2nd instar) were purchased from Jiyuan Baiyun Industry Co., Ltd (China). For feeding assays, 2nd instar *S. exigua* larvae were first underwent 24 h of starvation treatment in growth chamber and were then gently placed on 5 week-old plants. Two *S. exigua* larvae were placed on a plant. Larval weights were measured 7 days after feeding. Representative *S. exigua* larvae recovered from corresponding plants were killed by 75% ethanol solution and photographed by stereomicroscope (SZX10, Olympus) equipped with a 0.75 × objective lens and a digital camera (SC180, Olympus).

### *Botrytis cinerea* infection assay

Conidiospores were collected from *B. cinerea* (B0510) plates and diluted to ~1 × 106 conidiospores/mL in PDB medium (Potato Dextrose Broth, BD). For drop-inoculation assay, 5 μL *B. cinerea* spores was dropped on the center of each detached leaf. Subsequently the detached leaves were placed in dark with high humidity for 2–3 days. Phenotypes were imaged with a digital camera and the lesion sizes were measured using Digimizer software (v3.1.2.0, Belgium, Germany). For spray-inoculation assay, 4 week-old soil-grown seedlings were sprayed evenly with *B. cinerea* conidiospores and kept in high humidity for 2–3 days. The disease severity was classified into four grades: healthy (H, green), light symptoms (L, pale green), severe symptoms (S, orange-yellow), or completely dead (D, red). PDB buffer served as mock in these infection assays. As described in previous studies[45](#CR45)–[47](#CR47), the assessment of *B. cinerea* biomass was determined by real-time PCR quantification of the fungal *CUTINASE* signal ([Z69264](https://www.ncbi.nlm.nih.gov/nuccore/Z69264)) in relation to *Arabidopsis ACTIN2*.

### RNA sequencing and data analysis

RNA integrity was checked using the RNA Nano 6000 Assay Kit of the Bioanalyzer 2100 system (Agilent Technologies, CA, USA). Sequencing libraries were generated using NEBNext® UltraTM RNA Library Prep Kit for Illumina® (NEB, USA) following manufacturer’s recommendations and index codes were added to attribute sequences to each sample. The clustering of the index-coded samples was performed on a cBot Cluster Generation System using TruSeq PE Cluster Kit v3-cBot-HS (Illumia) according to the manufacturer’s instructions. After cluster generation, the library preparations were sequenced on an Illumina Novaseq platform and 150 bp paired-end reads were generated. Low-quality reads were removed from the raw reads using Cutadapt[48](#CR48) and Trimmomatic[49](#CR49) software to get clean reads. Clean reads were mapped to the corresponding reference genome using HISAT2 software[50](#CR50). Gene expression levels and read count of each gene were calculated using StringTie software[51](#CR51). The R package DEseq2 (v1.16.1) was used to identify the differentially expressed genes based on the following criteria: *p*adj < 0.05 and |log2FoldChange | > 1[52](#CR52). Venn diagramming, GO enrichment analysis was performed using TBtools (v2.007)[53](#CR53).

### Structure prediction and molecular docking

The ab initio structure predictions of AtGLR3.3 protein were implemented by RoseTTAFold and AlphaFold 2 through ColabFold[54](#CR54)–[56](#CR56). The homology modelling of AtGLR3.3 was conducted using SWISS-MODEL with AtGLR3.4 crystallography model (7lzh) template[30](#CR30),[57](#CR57). Ligand mol2 files of GSH and Glu were generated by Avogadro 1.2.0[58](#CR58). Prepdock website (<https://swift.cmbi.umcn.nl/servers/html/prepdock.html>) and AutoDockTools 1.5.7 software were used for adding hydrogens and symmetry related waters, computing Gasteiger charges, and detecting torsion root[59](#CR59). Molecular dockings were performed on AutoDock Vina 1.2.0 software and CB-DOCK2 website[60](#CR60),[61](#CR61). For AutoDock Vina, the grid box was setting up with a range spacing of 0.897 Å, number of points in x, y and z dimensions as 52, 54, and 70 respectively and center grid box for x, y and z as 173.571, 198.031, and 203.573 respectively for the ATD of AtGLR3.3. Docking of LBD was conducted with 0.756 Å in range spacing. The number of points in x, y and z dimensions was 60, 56, and 78. The center grid box was 172.151, 160.102, and 157.178 in xyz-coordinates. The results were visualized and the polar contacts between ligands and their receptor were detected on PyMOL 2.5.8 software. Hydrogen bonds were determined on UCSF ChimeraX 1.7.1 software with distance tolerance of 0.400 Å and angle tolerance of 20.000°[62](#CR62).

### Statistical analysis

All data were statistically analyzed using GraphPad Prism 8.0, SPSS, and R software and presented by Adobe Illustrator. Means and deviations were reported in the Figures and the corresponding figure legends.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](#MOESM16) linked to this article.

## Supplementary information

[Supplementary Information](/articles/instance/11850895/bin/41467_2025_57239_MOESM1_ESM.pdf) (1.7MB, pdf)

[41467\_2025\_57239\_MOESM2\_ESM.pdf](/articles/instance/11850895/bin/41467_2025_57239_MOESM2_ESM.pdf) (122.8KB, pdf)

Description of Additional Supplementary Files

[Supplementary Data 1](/articles/instance/11850895/bin/41467_2025_57239_MOESM3_ESM.xlsx) (63.5KB, xlsx)

[Supplementary Data 2](/articles/instance/11850895/bin/41467_2025_57239_MOESM4_ESM.xlsx) (40.7KB, xlsx)

[Supplementary Data 3](/articles/instance/11850895/bin/41467_2025_57239_MOESM5_ESM.xlsx) (334.1KB, xlsx)

[Supplementary Data 4](/articles/instance/11850895/bin/41467_2025_57239_MOESM6_ESM.xlsx) (65KB, xlsx)

[Supplementary Data 5](/articles/instance/11850895/bin/41467_2025_57239_MOESM7_ESM.xlsx) (10.9KB, xlsx)

[Supplementary Movie 1](/articles/instance/11850895/bin/41467_2025_57239_MOESM8_ESM.mp4) (2.6MB, mp4)

[Supplementary Movie 2](/articles/instance/11850895/bin/41467_2025_57239_MOESM9_ESM.mp4) (2.6MB, mp4)

[Supplementary Movie 3](/articles/instance/11850895/bin/41467_2025_57239_MOESM10_ESM.mp4) (4.8MB, mp4)

[Supplementary Movie 4](/articles/instance/11850895/bin/41467_2025_57239_MOESM11_ESM.mp4) (4.9MB, mp4)

[Supplementary Movie 5](/articles/instance/11850895/bin/41467_2025_57239_MOESM12_ESM.mp4) (4.9MB, mp4)

[Supplementary Movie 6](/articles/instance/11850895/bin/41467_2025_57239_MOESM13_ESM.mp4) (16.9MB, mp4)

[Supplementary Movie 7](/articles/instance/11850895/bin/41467_2025_57239_MOESM14_ESM.mp4) (16.6MB, mp4)

[Supplementary Movie 8](/articles/instance/11850895/bin/41467_2025_57239_MOESM15_ESM.mp4) (8.9MB, mp4)

[Reporting Summary](/articles/instance/11850895/bin/41467_2025_57239_MOESM16_ESM.pdf) (1.1MB, pdf)

[Transparent Peer Review file](/articles/instance/11850895/bin/41467_2025_57239_MOESM17_ESM.pdf) (1.1MB, pdf)

## Source data

[Source Data](/articles/instance/11850895/bin/41467_2025_57239_MOESM18_ESM.xlsx) (784.5KB, xlsx)

## Acknowledgements

We thank Simon Gilroy (Department of Botany, University of Wisconsin) for providing us with the *GCaMP3* seeds. No conflict of interest is declared. This research was supported by grants from the National Natural Science Foundation of China (32388101 and 32250001 to D.X., 32425011 and 32488302 to J.Y.), National Key R&D Program of China (2022YFD1400800 and 2021YFA1300400 to D.X.), China Postdoctoral Science Foundation (2020M670286 to Y.Y.), Postdoctoral Fellowship of Tsinghua-Peking Joint Center for Life Sciences. The Innovation Program of Chinese Academy of Agricultural Sciences (CAAS) and the Elite Young Scientists Program of CAAS, the New Cornerstone Science Foundation through the XPLORER PRIZE.

## Author contributions

D.X., R.L., Y.Y., Y.J., X.S., G.-L.W. designed the research and experimental strategy. R.L. and Y.Y. performed most of the experiments with the help from H.L., W.W., and S.H. R.D. measured GSH, Glu, JA and JA-Ile contents in plants. H.C. conducted the bioinformatic analysis. X.D. performed molecular docking analysis. Y.Y. and R.L. analyzed the data. R.L. and Y.Y. wrote the manuscript. D.X., X.S., J.Y., R.L. and Y.Y. revised the manuscript.

## Peer review

### Peer review information

*Nature Communications* thanks Mitsuyasu Hasebe, Hiroaki Mano, Andrej Pavlovic and Michael Wudick for their contribution to the peer review of this work. A peer review file is available.

## Data availability

The RNA-seq data generated in this study has been deposited in the Gene Expression Omnibus (GEO) database at NCBI under accession code [GSE249592](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249592). All other data supporting the findings of this study are available in the main text or the Supplementary Data. [Source data](#Sec24) are provided with this paper.

## Competing interests

The authors declare no competing interests.

## Footnotes

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

These authors contributed equally: Rui Li, Yongfang Yang.

## Contributor Information

Guo-Liang Wang, Email: wang.620@osu.edu.

Jianbin Yan, Email: jianbinlab@caas.cn.

Xiaoyi Shan, Email: shanxy80@mail.tsinghua.edu.cn.

Daoxin Xie, Email: daoxinlab@mail.tsinghua.edu.cn.

## Supplementary information

The online version contains supplementary material available at 10.1038/s41467-025-57239-1.

## References

* 1.Muday, G. K. & Brown-Harding, H. Nervous system-like signaling in plant defense. *Science***361**, 1068–1069 (2018).
   [[DOI](https://doi.org/10.1126/science.aau9813)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30213898/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Muday,%20G.%20K.%20&%20Brown-Harding,%20H.%20Nervous%20system-like%20signaling%20in%20plant%20defense.%20Science361,%201068%E2%80%931069%20(2018).)]
* 2.Suudhof, T. C. Neurotransmitter release. *Handb. Exp. Pharmacol.***2008**, 1–21 (2008). [[DOI](https://doi.org/10.1007/978-3-540-74805-2_1)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18064409/)]
* 3.Wasser, C. R. & Kavalali, E. T. Leaky synapses: regulation of spontaneous neurotransmission in central synapses. *Neuroscience***158**, 177–188 (2009).
   [[DOI](https://doi.org/10.1016/j.neuroscience.2008.03.028)] [[PMC free article](/articles/PMC2662333/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18434032/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Wasser,%20C.%20R.%20&%20Kavalali,%20E.%20T.%20Leaky%20synapses:%20regulation%20of%20spontaneous%20neurotransmission%20in%20central%20synapses.%20Neuroscience158,%20177%E2%80%93188%20(2009).)]
* 4.Hyman, S. E. Neurotransmitters. *Curr. Biol.***15**, R154–R158 (2005).
   [[DOI](https://doi.org/10.1016/j.cub.2005.02.037)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15753022/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Hyman,%20S.%20E.%20Neurotransmitters.%20Curr.%20Biol.15,%20R154%E2%80%93R158%20(2005).)]
* 5.Christmann, A. & Grill, E. Plant biology: electric defence. *Nature***500**, 404–405 (2013).
   [[DOI](https://doi.org/10.1038/500404a)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23969452/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Christmann,%20A.%20&%20Grill,%20E.%20Plant%20biology:%20electric%20defence.%20Nature500,%20404%E2%80%93405%20(2013).)]
* 6.Gao, Y. Q. et al. Ricca’s factors as mobile proteinaceous effectors of electrical signaling. *Cell***186**, 1337–1351.e1320 (2023).
   [[DOI](https://doi.org/10.1016/j.cell.2023.02.006)] [[PMC free article](/articles/PMC10098372/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36870332/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Gao,%20Y.%20Q.%20et%20al.%20Ricca%E2%80%99s%20factors%20as%20mobile%20proteinaceous%20effectors%20of%20electrical%20signaling.%20Cell186,%201337%E2%80%931351.e1320%20(2023).)]
* 7.Mousavi, S. A., Chauvin, A., Pascaud, F., Kellenberger, S. & Farmer, E. E. GLUTAMATE RECEPTOR-LIKE genes mediate leaf-to-leaf wound signalling. *Nature***500**, 422–426 (2013).
   [[DOI](https://doi.org/10.1038/nature12478)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23969459/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Mousavi,%20S.%20A.,%20Chauvin,%20A.,%20Pascaud,%20F.,%20Kellenberger,%20S.%20&%20Farmer,%20E.%20E.%20GLUTAMATE%20RECEPTOR-LIKE%20genes%20mediate%20leaf-to-leaf%20wound%20signalling.%20Nature500,%20422%E2%80%93426%20(2013).)]
* 8.Nguyen, C. T., Kurenda, A., Stolz, S., Chetelat, A. & Farmer, E. E. Identification of cell populations necessary for leaf-to-leaf electrical signaling in a wounded plant. *Proc. Natl Acad. Sci. USA***115**, 10178–10183 (2018).
   [[DOI](https://doi.org/10.1073/pnas.1807049115)] [[PMC free article](/articles/PMC6176584/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30228123/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Nguyen,%20C.%20T.,%20Kurenda,%20A.,%20Stolz,%20S.,%20Chetelat,%20A.%20&%20Farmer,%20E.%20E.%20Identification%20of%20cell%20populations%20necessary%20for%20leaf-to-leaf%20electrical%20signaling%20in%20a%20wounded%20plant.%20Proc.%20Natl%20Acad.%20Sci.%20USA115,%2010178%E2%80%9310183%20(2018).)]
* 9.Gao, Y. Q. et al. Chloride, glutathiones, and insect-derived elicitors introduced into the xylem trigger electrical signaling. *Plant Physiol.***194**, 1091–1103 (2023). [[DOI](https://doi.org/10.1093/plphys/kiad584)] [[PMC free article](/articles/PMC10828190/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/37925642/)]
* 10.Shao, Q., Gao, Q., Lhamo, D., Zhang, H. & Luan, S. Two glutamate- and pH-regulated Ca(2+) channels are required for systemic wound signaling in *Arabidopsis*. *Sci. Signal***13**, eaba1453 (2020). [[DOI](https://doi.org/10.1126/scisignal.aba1453)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32665412/)]
* 11.Toyota, M. et al. Glutamate triggers long-distance, calcium-based plant defense signaling. *Science***361**, 1112–1115 (2018).
   [[DOI](https://doi.org/10.1126/science.aat7744)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30213912/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Toyota,%20M.%20et%20al.%20Glutamate%20triggers%20long-distance,%20calcium-based%20plant%20defense%20signaling.%20Science361,%201112%E2%80%931115%20(2018).)]
* 12.Yan, C. et al. Injury activates Ca(2+)/calmodulin-dependent phosphorylation of JAV1-JAZ8-WRKY51 complex for jasmonate biosynthesis. *Mol. Cell***70**, 136–149.e137 (2018).
   [[DOI](https://doi.org/10.1016/j.molcel.2018.03.013)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29625034/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Yan,%20C.%20et%20al.%20Injury%20activates%20Ca(2+)/calmodulin-dependent%20phosphorylation%20of%20JAV1-JAZ8-WRKY51%20complex%20for%20jasmonate%20biosynthesis.%20Mol.%20Cell70,%20136%E2%80%93149.e137%20(2018).)]
* 13.Moe-Lange, J. et al. Interdependence of a mechanosensitive anion channel and glutamate receptors in distal wound signaling. *Sci. Adv.***7**, eabg4298 (2021).
   [[DOI](https://doi.org/10.1126/sciadv.abg4298)] [[PMC free article](/articles/PMC8442888/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34516872/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Moe-Lange,%20J.%20et%20al.%20Interdependence%20of%20a%20mechanosensitive%20anion%20channel%20and%20glutamate%20receptors%20in%20distal%20wound%20signaling.%20Sci.%20Adv.7,%20eabg4298%20(2021).)]
* 14.Bellandi, A. et al. Diffusion and bulk flow of amino acids mediate calcium waves in plants. *Sci. Adv.***8**, eabo6693 (2022).
   [[DOI](https://doi.org/10.1126/sciadv.abo6693)] [[PMC free article](/articles/PMC9586480/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36269836/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Bellandi,%20A.%20et%20al.%20Diffusion%20and%20bulk%20flow%20of%20amino%20acids%20mediate%20calcium%20waves%20in%20plants.%20Sci.%20Adv.8,%20eabo6693%20(2022).)]
* 15.Grenzi, M. et al. Long-distance turgor pressure changes induce local activation of plant glutamate receptor-like channels. *Curr. Biol.***33**, 1019 (2023).
   [[DOI](https://doi.org/10.1016/j.cub.2023.01.042)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36796359/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Grenzi,%20M.%20et%20al.%20Long-distance%20turgor%20pressure%20changes%20induce%20local%20activation%20of%20plant%20glutamate%20receptor-like%20channels.%20Curr.%20Biol.33,%201019%20(2023).)]
* 16.Farmer, E. E., Gasperini, D. & Acosta, I. F. The squeeze cell hypothesis for the activation of jasmonate synthesis in response to wounding. *N. Phytol.***204**, 282–288 (2014). [[DOI](https://doi.org/10.1111/nph.12897)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25453132/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Farmer,%20E.%20E.,%20Gasperini,%20D.%20&%20Acosta,%20I.%20F.%20The%20squeeze%20cell%20hypothesis%20for%20the%20activation%20of%20jasmonate%20synthesis%20in%20response%20to%20wounding.%20N.%20Phytol.204,%20282%E2%80%93288%20(2014).)]
* 17.Koo, A. J., Gao, X., Jones, A. D. & Howe, G. A. A rapid wound signal activates the systemic synthesis of bioactive jasmonates in *Arabidopsis*. *Plant J. Cell Mol. Biol.***59**, 974–986 (2009). [[DOI](https://doi.org/10.1111/j.1365-313X.2009.03924.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19473329/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Koo,%20A.%20J.,%20Gao,%20X.,%20Jones,%20A.%20D.%20&%20Howe,%20G.%20A.%20A%20rapid%20wound%20signal%20activates%20the%20systemic%20synthesis%20of%20bioactive%20jasmonates%20in%20Arabidopsis.%20Plant%20J.%20Cell%20Mol.%20Biol.59,%20974%E2%80%93986%20(2009).)]
* 18.Kumari, A., Chetelat, A., Nguyen, C. T. & Farmer, E. E. *Arabidopsis* H(+)-ATPase AHA1 controls slow wave potential duration and wound-response jasmonate pathway activation. *Proc. Natl Acad. Sci. USA***116**, 20226–20231 (2019).
   [[DOI](https://doi.org/10.1073/pnas.1907379116)] [[PMC free article](/articles/PMC6778210/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31527254/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Kumari,%20A.,%20Chetelat,%20A.,%20Nguyen,%20C.%20T.%20&%20Farmer,%20E.%20E.%20Arabidopsis%20H(+)-ATPase%20AHA1%20controls%20slow%20wave%20potential%20duration%20and%20wound-response%20jasmonate%20pathway%20activation.%20Proc.%20Natl%20Acad.%20Sci.%20USA116,%2020226%E2%80%9320231%20(2019).)]
* 19.Wang, L. et al. Rolling circle amplification-mediated hairpin RNA (RMHR) library construction in plants. *Nucleic Acids Res.***36**, e149 (2008).
   [[DOI](https://doi.org/10.1093/nar/gkn715)] [[PMC free article](/articles/PMC2602787/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18978012/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Wang,%20L.%20et%20al.%20Rolling%20circle%20amplification-mediated%20hairpin%20RNA%20(RMHR)%20library%20construction%20in%20plants.%20Nucleic%20Acids%20Res.36,%20e149%20(2008).)]
* 20.Hu, P. et al. JAV1 controls jasmonate-regulated plant defense. *Mol. Cell***50**, 504–515 (2013).
   [[DOI](https://doi.org/10.1016/j.molcel.2013.04.027)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23706819/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Hu,%20P.%20et%20al.%20JAV1%20controls%20jasmonate-regulated%20plant%20defense.%20Mol.%20Cell50,%20504%E2%80%93515%20(2013).)]
* 21.Parisy, V. et al. Identification of PAD2 as a gamma-glutamylcysteine synthetase highlights the importance of glutathione in disease resistance of *Arabidopsis*. *Plant J. Cell Mol. Biol.***49**, 159–172 (2007). [[DOI](https://doi.org/10.1111/j.1365-313X.2006.02938.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17144898/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Parisy,%20V.%20et%20al.%20Identification%20of%20PAD2%20as%20a%20gamma-glutamylcysteine%20synthetase%20highlights%20the%20importance%20of%20glutathione%20in%20disease%20resistance%20of%20Arabidopsis.%20Plant%20J.%20Cell%20Mol.%20Biol.49,%20159%E2%80%93172%20(2007).)]
* 22.Schlaeppi, K., Bodenhausen, N., Buchala, A., Mauch, F. & Reymond, P. The glutathione-deficient mutant pad2-1 accumulates lower amounts of glucosinolates and is more susceptible to the insect herbivore Spodoptera littoralis. *Plant J. Cell Mol. Biol.***55**, 774–786 (2008). [[DOI](https://doi.org/10.1111/j.1365-313X.2008.03545.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18466300/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Schlaeppi,%20K.,%20Bodenhausen,%20N.,%20Buchala,%20A.,%20Mauch,%20F.%20&%20Reymond,%20P.%20The%20glutathione-deficient%20mutant%20pad2-1%20accumulates%20lower%20amounts%20of%20glucosinolates%20and%20is%20more%20susceptible%20to%20the%20insect%20herbivore%20Spodoptera%20littoralis.%20Plant%20J.%20Cell%20Mol.%20Biol.55,%20774%E2%80%93786%20(2008).)]
* 23.Ferrari, S., Plotnikova, J. M., De Lorenzo, G. & Ausubel, F. M. *Arabidopsis* local resistance to *Botrytis cinerea* involves salicylic acid and camalexin and requires EDS4 and PAD2, but not SID2, EDS5 or PAD4. *Plant J. Cell Mol. Biol.***35**, 193–205 (2003). [[DOI](https://doi.org/10.1046/j.1365-313x.2003.01794.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12848825/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Ferrari,%20S.,%20Plotnikova,%20J.%20M.,%20De%20Lorenzo,%20G.%20&%20Ausubel,%20F.%20M.%20Arabidopsis%20local%20resistance%20to%20Botrytis%20cinerea%20involves%20salicylic%20acid%20and%20camalexin%20and%20requires%20EDS4%20and%20PAD2,%20but%20not%20SID2,%20EDS5%20or%20PAD4.%20Plant%20J.%20Cell%20Mol.%20Biol.35,%20193%E2%80%93205%20(2003).)]
* 24.Noctor, G., Queval, G., Mhamdi, A., Chaouch, S. & Foyer, C. H. Glutathione. *Arabidopsis Book***9**, e0142 (2011).
   [[DOI](https://doi.org/10.1199/tab.0142)] [[PMC free article](/articles/PMC3267239/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22303267/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Noctor,%20G.,%20Queval,%20G.,%20Mhamdi,%20A.,%20Chaouch,%20S.%20&%20Foyer,%20C.%20H.%20Glutathione.%20Arabidopsis%20Book9,%20e0142%20(2011).)]
* 25.Dengler, N. G. The shoot apical meristem and development of vascular architecture. *Botany***84**, 1660–1671 (2006). [[Google Scholar](https://scholar.google.com/scholar_lookup?Dengler,%20N.%20G.%20The%20shoot%20apical%20meristem%20and%20development%20of%20vascular%20architecture.%20Botany84,%201660%E2%80%931671%20(2006).)]
* 26.Kamiyama, A. et al. Phosphonate-based irreversible inhibitors of human γ-glutamyl transpeptidase (GGT). GGsTop is a non-toxic and highly selective inhibitor with critical electrostatic interaction with an active-site residue Lys562 for enhanced inhibitory activity. *Bioorganic Med. Chem.***24**, 5340–5352 (2016). [[DOI](https://doi.org/10.1016/j.bmc.2016.08.050)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27622749/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Kamiyama,%20A.%20et%20al.%20Phosphonate-based%20irreversible%20inhibitors%20of%20human%20%CE%B3-glutamyl%20transpeptidase%20(GGT).%20GGsTop%20is%20a%20non-toxic%20and%20highly%20selective%20inhibitor%20with%20critical%20electrostatic%20interaction%20with%20an%20active-site%20residue%20Lys562%20for%20enhanced%20inhibitory%20activity.%20Bioorganic%20Med.%20Chem.24,%205340%E2%80%935352%20(2016).)]
* 27.Reymond, P. et al. A conserved transcript pattern in response to a specialist and a generalist herbivore. *Plant Cell***16**, 3132–3147 (2004).
   [[DOI](https://doi.org/10.1105/tpc.104.026120)] [[PMC free article](/articles/PMC527203/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15494554/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Reymond,%20P.%20et%20al.%20A%20conserved%20transcript%20pattern%20in%20response%20to%20a%20specialist%20and%20a%20generalist%20herbivore.%20Plant%20Cell16,%203132%E2%80%933147%20(2004).)]
* 28.Yan, Y. et al. A downstream mediator in the growth repression limb of the jasmonate pathway. *Plant Cell***19**, 2470–2483 (2007).
   [[DOI](https://doi.org/10.1105/tpc.107.050708)] [[PMC free article](/articles/PMC2002611/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17675405/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Yan,%20Y.%20et%20al.%20A%20downstream%20mediator%20in%20the%20growth%20repression%20limb%20of%20the%20jasmonate%20pathway.%20Plant%20Cell19,%202470%E2%80%932483%20(2007).)]
* 29.Alfieri, A. et al. The structural bases for agonist diversity in an *Arabidopsis thaliana* glutamate receptor-like channel. *Proc. Natl Acad. Sci. USA***117**, 752–760 (2020).
   [[DOI](https://doi.org/10.1073/pnas.1905142117)] [[PMC free article](/articles/PMC6955363/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31871183/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Alfieri,%20A.%20et%20al.%20The%20structural%20bases%20for%20agonist%20diversity%20in%20an%20Arabidopsis%20thaliana%20glutamate%20receptor-like%20channel.%20Proc.%20Natl%20Acad.%20Sci.%20USA117,%20752%E2%80%93760%20(2020).)]
* 30.Green, M. N. et al. Structure of the *Arabidopsis thaliana* glutamate receptor-like channel GLR3.4. *Mol. Cell***81**, 3216–3226.e3218 (2021).
   [[DOI](https://doi.org/10.1016/j.molcel.2021.05.025)] [[PMC free article](/articles/PMC8349882/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34161757/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Green,%20M.%20N.%20et%20al.%20Structure%20of%20the%20Arabidopsis%20thaliana%20glutamate%20receptor-like%20channel%20GLR3.4.%20Mol.%20Cell81,%203216%E2%80%933226.e3218%20(2021).)]
* 31.Qi, Z., Stephens, N. R. & Spalding, E. P. Calcium entry mediated by GLR3.3, an *Arabidopsis* glutamate receptor with a broad agonist profile. *Plant Physiol.***142**, 963–971 (2006).
   [[DOI](https://doi.org/10.1104/pp.106.088989)] [[PMC free article](/articles/PMC1630757/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17012403/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Qi,%20Z.,%20Stephens,%20N.%20R.%20&%20Spalding,%20E.%20P.%20Calcium%20entry%20mediated%20by%20GLR3.3,%20an%20Arabidopsis%20glutamate%20receptor%20with%20a%20broad%20agonist%20profile.%20Plant%20Physiol.142,%20963%E2%80%93971%20(2006).)]
* 32.Simon, A. A., Navarro-Retamal, C. & Feijó, J. A. Merging signaling with structure: functions and mechanisms of plant glutamate receptor ion channels. *Annu. Rev. Plant Biol.***74**, 415–452 (2023).
   [[DOI](https://doi.org/10.1146/annurev-arplant-070522-033255)] [[PMC free article](/articles/PMC11479355/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36854472/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Simon,%20A.%20A.,%20Navarro-Retamal,%20C.%20&%20Feij%C3%B3,%20J.%20A.%20Merging%20signaling%20with%20structure:%20functions%20and%20mechanisms%20of%20plant%20glutamate%20receptor%20ion%20channels.%20Annu.%20Rev.%20Plant%20Biol.74,%20415%E2%80%93452%20(2023).)]
* 33.Farmer, E. E., Gao, Y. Q., Lenzoni, G., Wolfender, J. L. & Wu, Q. Wound- and mechanostimulated electrical signals control hormone responses. *N. Pphytol.***227**, 1037–1050 (2020). [[DOI](https://doi.org/10.1111/nph.16646)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32392391/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Farmer,%20E.%20E.,%20Gao,%20Y.%20Q.,%20Lenzoni,%20G.,%20Wolfender,%20J.%20L.%20&%20Wu,%20Q.%20Wound-%20and%20mechanostimulated%20electrical%20signals%20control%20hormone%20responses.%20N.%20Pphytol.227,%201037%E2%80%931050%20(2020).)]
* 34.Janaky, R., Dohovics, R., Saransaari, P. & Oja, S. S. Modulation of [3H]dopamine release by glutathione in mouse striatal slices. *Neurochem. Res.***32**, 1357–1364 (2007).
   [[DOI](https://doi.org/10.1007/s11064-007-9315-z)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17401648/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Janaky,%20R.,%20Dohovics,%20R.,%20Saransaari,%20P.%20&%20Oja,%20S.%20S.%20Modulation%20of%20%5B3H%5Ddopamine%20release%20by%20glutathione%20in%20mouse%20striatal%20slices.%20Neurochem.%20Res.32,%201357%E2%80%931364%20(2007).)]
* 35.Janaky, R. et al. Glutathione and signal transduction in the mammalian CNS. *J. Neurochem.***73**, 889–902 (1999).
   [[DOI](https://doi.org/10.1046/j.1471-4159.1999.0730889.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10461878/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Janaky,%20R.%20et%20al.%20Glutathione%20and%20signal%20transduction%20in%20the%20mammalian%20CNS.%20J.%20Neurochem.73,%20889%E2%80%93902%20(1999).)]
* 36.Feng, W. et al. Gclc deficiency in mouse CNS causes mitochondrial damage and neurodegeneration. *Hum. Mol. Genet.***26**, 1376–1390 (2017).
   [[DOI](https://doi.org/10.1093/hmg/ddx040)] [[PMC free article](/articles/PMC6075078/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28158580/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Feng,%20W.%20et%20al.%20Gclc%20deficiency%20in%20mouse%20CNS%20causes%20mitochondrial%20damage%20and%20neurodegeneration.%20Hum.%20Mol.%20Genet.26,%201376%E2%80%931390%20(2017).)]
* 37.Yan, C. et al. Ca(2+)/calmodulin-mediated desensitization of glutamate receptors shapes plant systemic wound signalling and anti-herbivore defence. *Nat. Plants***10**, 145–160 (2024).
   [[DOI](https://doi.org/10.1038/s41477-023-01578-8)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/38168609/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Yan,%20C.%20et%20al.%20Ca(2+)/calmodulin-mediated%20desensitization%20of%20glutamate%20receptors%20shapes%20plant%20systemic%20wound%20signalling%20and%20anti-herbivore%20defence.%20Nat.%20Plants10,%20145%E2%80%93160%20(2024).)]
* 38.Diaz-Vivancos, P., de Simone, A., Kiddle, G. & Foyer, C. H. Glutathione-linking cell proliferation to oxidative stress. *Free Radic. Biol. Med.***89**, 1154–1164 (2015).
   [[DOI](https://doi.org/10.1016/j.freeradbiomed.2015.09.023)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26546102/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Diaz-Vivancos,%20P.,%20de%20Simone,%20A.,%20Kiddle,%20G.%20&%20Foyer,%20C.%20H.%20Glutathione-linking%20cell%20proliferation%20to%20oxidative%20stress.%20Free%20Radic.%20Biol.%20Med.89,%201154%E2%80%931164%20(2015).)]
* 39.Miller, G. et al. The plant NADPH oxidase RBOHD mediates rapid systemic signaling in response to diverse stimuli. *Sci. Signal***2**, ra45 (2009).
   [[DOI](https://doi.org/10.1126/scisignal.2000448)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19690331/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Miller,%20G.%20et%20al.%20The%20plant%20NADPH%20oxidase%20RBOHD%20mediates%20rapid%20systemic%20signaling%20in%20response%20to%20diverse%20stimuli.%20Sci.%20Signal2,%20ra45%20(2009).)]
* 40.Mittler, R., Zandalinas, S. I., Fichman, Y. & Van Breusegem, F. Reactive oxygen species signalling in plant stress responses. *Nat. Rev. Mol. Cell Biol.***23**, 663–679 (2022).
   [[DOI](https://doi.org/10.1038/s41580-022-00499-2)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35760900/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Mittler,%20R.,%20Zandalinas,%20S.%20I.,%20Fichman,%20Y.%20&%20Van%20Breusegem,%20F.%20Reactive%20oxygen%20species%20signalling%20in%20plant%20stress%20responses.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.23,%20663%E2%80%93679%20(2022).)]
* 41.Mittler, R. ROS Are Good. *TrendsPlant Sci.***22**, 11–19 (2017). [[DOI](https://doi.org/10.1016/j.tplants.2016.08.002)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27666517/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Mittler,%20R.%20ROS%20Are%20Good.%20TrendsPlant%20Sci.22,%2011%E2%80%9319%20(2017).)]
* 42.Lovinger, D. M. Communication networks in the brain: neurons, receptors, neurotransmitters, and alcohol. *Alcohol Res. Health***31**, 196–214 (2008).
   [[PMC free article](/articles/PMC3860493/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23584863/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Lovinger,%20D.%20M.%20Communication%20networks%20in%20the%20brain:%20neurons,%20receptors,%20neurotransmitters,%20and%20alcohol.%20Alcohol%20Res.%20Health31,%20196%E2%80%93214%20(2008).)]
* 43.Gao, Y. Q. & Farmer, E. E. Osmoelectric siphon models for signal and water dispersal in wounded plants. *J. Exp. Bot.***74**, 1207–1220 (2023).
   [[DOI](https://doi.org/10.1093/jxb/erac449)] [[PMC free article](/articles/PMC9923213/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36377754/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Gao,%20Y.%20Q.%20&%20Farmer,%20E.%20E.%20Osmoelectric%20siphon%20models%20for%20signal%20and%20water%20dispersal%20in%20wounded%20plants.%20J.%20Exp.%20Bot.74,%201207%E2%80%931220%20(2023).)]
* 44.Wang, Z. P. et al. Egg cell-specific promoter-controlled CRISPR/Cas9 efficiently generates homozygous mutants for multiple target genes in *Arabidopsis* in a single generation. *Genome Biol.***16**, 144 (2015).
   [[DOI](https://doi.org/10.1186/s13059-015-0715-0)] [[PMC free article](/articles/PMC4507317/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26193878/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Wang,%20Z.%20P.%20et%20al.%20Egg%20cell-specific%20promoter-controlled%20CRISPR/Cas9%20efficiently%20generates%20homozygous%20mutants%20for%20multiple%20target%20genes%20in%20Arabidopsis%20in%20a%20single%20generation.%20Genome%20Biol.16,%20144%20(2015).)]
* 45.Yi, R. et al. Dioxygenase JID1 mediates the modification of OPDA to regulate jasmonate homeostasis. *Cell Discov.***9**, 39 (2023).
   [[DOI](https://doi.org/10.1038/s41421-023-00530-6)] [[PMC free article](/articles/PMC10090039/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/37041145/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Yi,%20R.%20et%20al.%20Dioxygenase%20JID1%20mediates%20the%20modification%20of%20OPDA%20to%20regulate%20jasmonate%20homeostasis.%20Cell%20Discov.9,%2039%20(2023).)]
* 46.Berr, A. et al. *Arabidopsis* histone methyltransferase SET DOMAIN GROUP8 mediates induction of the jasmonate/ethylene pathway genes in plant defense response to necrotrophic fungi. *Plant Physiol.***154**, 1403–1414 (2010).
   [[DOI](https://doi.org/10.1104/pp.110.161497)] [[PMC free article](/articles/PMC2971616/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20810545/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Berr,%20A.%20et%20al.%20Arabidopsis%20histone%20methyltransferase%20SET%20DOMAIN%20GROUP8%20mediates%20induction%20of%20the%20jasmonate/ethylene%20pathway%20genes%20in%20plant%20defense%20response%20to%20necrotrophic%20fungi.%20Plant%20Physiol.154,%201403%E2%80%931414%20(2010).)]
* 47.Smirnova, E. et al. Jasmonic acid oxidase 2 hydroxylates jasmonic acid and represses basal defense and resistance responses against botrytis cinerea nifection. *Mol. Plant***10**, 1159–1173 (2017).
   [[DOI](https://doi.org/10.1016/j.molp.2017.07.010)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28760569/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Smirnova,%20E.%20et%20al.%20Jasmonic%20acid%20oxidase%202%20hydroxylates%20jasmonic%20acid%20and%20represses%20basal%20defense%20and%20resistance%20responses%20against%20botrytis%20cinerea%20nifection.%20Mol.%20Plant10,%201159%E2%80%931173%20(2017).)]
* 48.Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. *EMBnet. J.***17**, 10–12 (2011). [[Google Scholar](https://scholar.google.com/scholar_lookup?Martin,%20M.%20Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads.%20EMBnet.%20J.17,%2010%E2%80%9312%20(2011).)]
* 49.Bolger, A. M., Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. *Bioinformatics (Oxford, England)***30**, 2114–2120 (2014).
   [[DOI](https://doi.org/10.1093/bioinformatics/btu170)] [[PMC free article](/articles/PMC4103590/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24695404/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Bolger,%20A.%20M.,%20Lohse,%20M.%20&%20Usadel,%20B.%20Trimmomatic:%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data.%20Bioinformatics%20(Oxford,%20England)30,%202114%E2%80%932120%20(2014).)]
* 50.Pertea, M., Kim, D., Pertea, G. M., Leek, J. T. & Salzberg, S. L. Transcript-level expression analysis of RNA-seq experiments with HISAT, stringTie and ballgown. *Nat. Protoc.***11**, 1650–1667 (2016).
   [[DOI](https://doi.org/10.1038/nprot.2016.095)] [[PMC free article](/articles/PMC5032908/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27560171/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Pertea,%20M.,%20Kim,%20D.,%20Pertea,%20G.%20M.,%20Leek,%20J.%20T.%20&%20Salzberg,%20S.%20L.%20Transcript-level%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20HISAT,%20stringTie%20and%20ballgown.%20Nat.%20Protoc.11,%201650%E2%80%931667%20(2016).)]
* 51.Pertea, M. et al. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. *Nat. Biotechnol.***33**, 290–295 (2015).
   [[DOI](https://doi.org/10.1038/nbt.3122)] [[PMC free article](/articles/PMC4643835/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25690850/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Pertea,%20M.%20et%20al.%20StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads.%20Nat.%20Biotechnol.33,%20290%E2%80%93295%20(2015).)]
* 52.Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. *Genome Biol.***15**, 550 (2014).
   [[DOI](https://doi.org/10.1186/s13059-014-0550-8)] [[PMC free article](/articles/PMC4302049/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25516281/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Love,%20M.%20I.,%20Huber,%20W.%20&%20Anders,%20S.%20Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.15,%20550%20(2014).)]
* 53.Chen, C. et al. TBtools: An integrative toolkit developed for interactive analyses of big biological data. *Mol. Plant***13**, 1194–1202 (2020).
   [[DOI](https://doi.org/10.1016/j.molp.2020.06.009)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32585190/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Chen,%20C.%20et%20al.%20TBtools:%20An%20integrative%20toolkit%20developed%20for%20interactive%20analyses%20of%20big%20biological%20data.%20Mol.%20Plant13,%201194%E2%80%931202%20(2020).)]
* 54.Baek, M. et al. Accurate prediction of protein structures and interactions using a three-track neural network. *Science***373**, 871–876 (2021).
   [[DOI](https://doi.org/10.1126/science.abj8754)] [[PMC free article](/articles/PMC7612213/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34282049/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Baek,%20M.%20et%20al.%20Accurate%20prediction%20of%20protein%20structures%20and%20interactions%20using%20a%20three-track%20neural%20network.%20Science373,%20871%E2%80%93876%20(2021).)]
* 55.Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. *Nature***596**, 583–589 (2021).
   [[DOI](https://doi.org/10.1038/s41586-021-03819-2)] [[PMC free article](/articles/PMC8371605/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34265844/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Jumper,%20J.%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold.%20Nature596,%20583%E2%80%93589%20(2021).)]
* 56.Mirdita, M. et al. ColabFold: making protein folding accessible to all. *Nat. Methods***19**, 679–682 (2022).
   [[DOI](https://doi.org/10.1038/s41592-022-01488-1)] [[PMC free article](/articles/PMC9184281/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35637307/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Mirdita,%20M.%20et%20al.%20ColabFold:%20making%20protein%20folding%20accessible%20to%20all.%20Nat.%20Methods19,%20679%E2%80%93682%20(2022).)]
* 57.Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. *Nucleic Acids Res.***46**, W296–W303 (2018).
   [[DOI](https://doi.org/10.1093/nar/gky427)] [[PMC free article](/articles/PMC6030848/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29788355/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Waterhouse,%20A.%20et%20al.%20SWISS-MODEL:%20homology%20modelling%20of%20protein%20structures%20and%20complexes.%20Nucleic%20Acids%20Res.46,%20W296%E2%80%93W303%20(2018).)]
* 58.Hanwell, M. D. et al. Avogadro: an advanced semantic chemical editor, visualization, and analysis platform. *J. Cheminform.***4**, 17 (2012).
   [[DOI](https://doi.org/10.1186/1758-2946-4-17)] [[PMC free article](/articles/PMC3542060/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22889332/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Hanwell,%20M.%20D.%20et%20al.%20Avogadro:%20an%20advanced%20semantic%20chemical%20editor,%20visualization,%20and%20analysis%20platform.%20J.%20Cheminform.4,%2017%20(2012).)]
* 59.Morris, G. M. et al. AutoDock4 and AutoDockTools4: automated docking with selective receptor flexibility. *J. Comput. Chem.***30**, 2785–2791 (2009).
   [[DOI](https://doi.org/10.1002/jcc.21256)] [[PMC free article](/articles/PMC2760638/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19399780/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Morris,%20G.%20M.%20et%20al.%20AutoDock4%20and%20AutoDockTools4:%20automated%20docking%20with%20selective%20receptor%20flexibility.%20J.%20Comput.%20Chem.30,%202785%E2%80%932791%20(2009).)]
* 60.Liu, Y. et al. CB-Dock2: improved protein-ligand blind docking by integrating cavity detection, docking and homologous template fitting. *Nucleic Acids Res.***50**, W159–W164 (2022).
   [[DOI](https://doi.org/10.1093/nar/gkac394)] [[PMC free article](/articles/PMC9252749/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35609983/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Liu,%20Y.%20et%20al.%20CB-Dock2:%20improved%20protein-ligand%20blind%20docking%20by%20integrating%20cavity%20detection,%20docking%20and%20homologous%20template%20fitting.%20Nucleic%20Acids%20Res.50,%20W159%E2%80%93W164%20(2022).)]
* 61.Eberhardt, J., Santos-Martins, D., Tillack, A. F. & Forli, S. AutoDock Vina 1.2.0: new docking methods, expanded force field, and python bindings. *J. Chem. Inform. Model.***61**, 3891–3898 (2021). [[DOI](https://doi.org/10.1021/acs.jcim.1c00203)] [[PMC free article](/articles/PMC10683950/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34278794/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Eberhardt,%20J.,%20Santos-Martins,%20D.,%20Tillack,%20A.%20F.%20&%20Forli,%20S.%20AutoDock%20Vina%201.2.0:%20new%20docking%20methods,%20expanded%20force%20field,%20and%20python%20bindings.%20J.%20Chem.%20Inform.%20Model.61,%203891%E2%80%933898%20(2021).)]
* 62.Meng, E. C. et al. UCSF ChimeraX: tools for structure building and analysis. *Protein Sci.***32**, e4792 (2023).
   [[DOI](https://doi.org/10.1002/pro.4792)] [[PMC free article](/articles/PMC10588335/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/37774136/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?Meng,%20E.%20C.%20et%20al.%20UCSF%20ChimeraX:%20tools%20for%20structure%20building%20and%20analysis.%20Protein%20Sci.32,%20e4792%20(2023).)]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

[Supplementary Information](/articles/instance/11850895/bin/41467_2025_57239_MOESM1_ESM.pdf) (1.7MB, pdf)

[41467\_2025\_57239\_MOESM2\_ESM.pdf](/articles/instance/11850895/bin/41467_2025_57239_MOESM2_ESM.pdf) (122.8KB, pdf)

Description of Additional Supplementary Files

[Supplementary Data 1](/articles/instance/11850895/bin/41467_2025_57239_MOESM3_ESM.xlsx) (63.5KB, xlsx)

[Supplementary Data 2](/articles/instance/11850895/bin/41467_2025_57239_MOESM4_ESM.xlsx) (40.7KB, xlsx)

[Supplementary Data 3](/articles/instance/11850895/bin/41467_2025_57239_MOESM5_ESM.xlsx) (334.1KB, xlsx)

[Supplementary Data 4](/articles/instance/11850895/bin/41467_2025_57239_MOESM6_ESM.xlsx) (65KB, xlsx)

[Supplementary Data 5](/articles/instance/11850895/bin/41467_2025_57239_MOESM7_ESM.xlsx) (10.9KB, xlsx)

[Supplementary Movie 1](/articles/instance/11850895/bin/41467_2025_57239_MOESM8_ESM.mp4) (2.6MB, mp4)

[Supplementary Movie 2](/articles/instance/11850895/bin/41467_2025_57239_MOESM9_ESM.mp4) (2.6MB, mp4)

[Supplementary Movie 3](/articles/instance/11850895/bin/41467_2025_57239_MOESM10_ESM.mp4) (4.8MB, mp4)

[Supplementary Movie 4](/articles/instance/11850895/bin/41467_2025_57239_MOESM11_ESM.mp4) (4.9MB, mp4)

[Supplementary Movie 5](/articles/instance/11850895/bin/41467_2025_57239_MOESM12_ESM.mp4) (4.9MB, mp4)

[Supplementary Movie 6](/articles/instance/11850895/bin/41467_2025_57239_MOESM13_ESM.mp4) (16.9MB, mp4)

[Supplementary Movie 7](/articles/instance/11850895/bin/41467_2025_57239_MOESM14_ESM.mp4) (16.6MB, mp4)

[Supplementary Movie 8](/articles/instance/11850895/bin/41467_2025_57239_MOESM15_ESM.mp4) (8.9MB, mp4)

[Reporting Summary](/articles/instance/11850895/bin/41467_2025_57239_MOESM16_ESM.pdf) (1.1MB, pdf)

[Transparent Peer Review file](/articles/instance/11850895/bin/41467_2025_57239_MOESM17_ESM.pdf) (1.1MB, pdf)

[Source Data](/articles/instance/11850895/bin/41467_2025_57239_MOESM18_ESM.xlsx) (784.5KB, xlsx)

### Data Availability Statement

The RNA-seq data generated in this study has been deposited in the Gene Expression Omnibus (GEO) database at NCBI under accession code [GSE249592](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249592). All other data supporting the findings of this study are available in the main text or the Supplementary Data. [Source data](#Sec24) are provided with this paper.