<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11487192/"/><title>Protein kinase 2 of the giant sarcomeric protein UNC-89 regulates mitochondrial morphology and function - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Commun Biol</button></div>. 2024 Oct 17;7:1342. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s42003-024-07042-3" rel="noopener noreferrer" target="_blank">10.1038/s42003-024-07042-3</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Protein kinase 2 of the giant sarcomeric protein UNC-89 regulates mitochondrial morphology and function</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Matsunaga%20Y%22%5BAuthor%5D"><span class="name western">Yohei Matsunaga</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yohei Matsunaga</span></h3>
<div class="p">
<sup>1</sup>Department of Pathology, Emory University, Atlanta, GA USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Matsunaga%20Y%22%5BAuthor%5D"><span class="name western">Yohei Matsunaga</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qadota%20H%22%5BAuthor%5D"><span class="name western">Hiroshi Qadota</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Hiroshi Qadota</span></h3>
<div class="p">
<sup>1</sup>Department of Pathology, Emory University, Atlanta, GA USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qadota%20H%22%5BAuthor%5D"><span class="name western">Hiroshi Qadota</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ghazal%20N%22%5BAuthor%5D"><span class="name western">Nasab Ghazal</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Nasab Ghazal</span></h3>
<div class="p">
<sup>2</sup>Department of Pediatrics, Emory University, Atlanta, GA USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ghazal%20N%22%5BAuthor%5D"><span class="name western">Nasab Ghazal</span></a>
</div>
</div>
<sup>2</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lesanpezeshki%20L%22%5BAuthor%5D"><span class="name western">Leila Lesanpezeshki</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Leila Lesanpezeshki</span></h3>
<div class="p">
<sup>3</sup>Department of Chemical Engineering, Texas Tech University, Lubbock, TX USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lesanpezeshki%20L%22%5BAuthor%5D"><span class="name western">Leila Lesanpezeshki</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dorendorf%20T%22%5BAuthor%5D"><span class="name western">Till Dorendorf</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Till Dorendorf</span></h3>
<div class="p">
<sup>4</sup>Department of Biology, University of Konstanz, Konstanz, Germany </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dorendorf%20T%22%5BAuthor%5D"><span class="name western">Till Dorendorf</span></a>
</div>
</div>
<sup>4</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moody%20JC%22%5BAuthor%5D"><span class="name western">Jasmine C Moody</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jasmine C Moody</span></h3>
<div class="p">
<sup>1</sup>Department of Pathology, Emory University, Atlanta, GA USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moody%20JC%22%5BAuthor%5D"><span class="name western">Jasmine C Moody</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahier%20A%22%5BAuthor%5D"><span class="name western">Arnaud Ahier</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Arnaud Ahier</span></h3>
<div class="p">
<sup>5</sup>Clem Jones Centre for Ageing Dementia Research, Queensland Brain Institute, The University of Queensland, Brisbane, QLD Australia </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahier%20A%22%5BAuthor%5D"><span class="name western">Arnaud Ahier</span></a>
</div>
</div>
<sup>5</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Matheny%20CJ%22%5BAuthor%5D"><span class="name western">Courtney J Matheny</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Courtney J Matheny</span></h3>
<div class="p">
<sup>1</sup>Department of Pathology, Emory University, Atlanta, GA USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Matheny%20CJ%22%5BAuthor%5D"><span class="name western">Courtney J Matheny</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id9" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vanapalli%20SA%22%5BAuthor%5D"><span class="name western">Siva A Vanapalli</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Siva A Vanapalli</span></h3>
<div class="p">
<sup>3</sup>Department of Chemical Engineering, Texas Tech University, Lubbock, TX USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vanapalli%20SA%22%5BAuthor%5D"><span class="name western">Siva A Vanapalli</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id10" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zuryn%20S%22%5BAuthor%5D"><span class="name western">Steven Zuryn</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Steven Zuryn</span></h3>
<div class="p">
<sup>5</sup>Clem Jones Centre for Ageing Dementia Research, Queensland Brain Institute, The University of Queensland, Brisbane, QLD Australia </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zuryn%20S%22%5BAuthor%5D"><span class="name western">Steven Zuryn</span></a>
</div>
</div>
<sup>5</sup>, <a aria-describedby="id11" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mayans%20O%22%5BAuthor%5D"><span class="name western">Olga Mayans</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Olga Mayans</span></h3>
<div class="p">
<sup>4</sup>Department of Biology, University of Konstanz, Konstanz, Germany </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mayans%20O%22%5BAuthor%5D"><span class="name western">Olga Mayans</span></a>
</div>
</div>
<sup>4</sup>, <a aria-describedby="id12" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwong%20JQ%22%5BAuthor%5D"><span class="name western">Jennifer Q Kwong</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Jennifer Q Kwong</span></h3>
<div class="p">
<sup>2</sup>Department of Pediatrics, Emory University, Atlanta, GA USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwong%20JQ%22%5BAuthor%5D"><span class="name western">Jennifer Q Kwong</span></a>
</div>
</div>
<sup>2</sup>, <a aria-describedby="id13" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Benian%20GM%22%5BAuthor%5D"><span class="name western">Guy M Benian</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Guy M Benian</span></h3>
<div class="p">
<sup>1</sup>Department of Pathology, Emory University, Atlanta, GA USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Benian%20GM%22%5BAuthor%5D"><span class="name western">Guy M Benian</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Pathology, Emory University, Atlanta, GA USA </div>
<div id="Aff2">
<sup>2</sup>Department of Pediatrics, Emory University, Atlanta, GA USA </div>
<div id="Aff3">
<sup>3</sup>Department of Chemical Engineering, Texas Tech University, Lubbock, TX USA </div>
<div id="Aff4">
<sup>4</sup>Department of Biology, University of Konstanz, Konstanz, Germany </div>
<div id="Aff5">
<sup>5</sup>Clem Jones Centre for Ageing Dementia Research, Queensland Brain Institute, The University of Queensland, Brisbane, QLD Australia </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2023 May 23; Accepted 2024 Oct 10; Collection date 2024.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© The Author(s) 2024</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by-nc-nd/4.0/" rel="noopener noreferrer" target="_blank">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11487192  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/39420071/">39420071</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">UNC-89 is a giant sarcomeric M-line protein required for sarcomere organization and optimal muscle function. UNC-89 contains two protein kinase domains, PK1 and PK2, separated by an elastic region. Here we show that PK2 is a canonical kinase expected to be catalytically active. <em>C. elegans</em> expressing UNC-89 with a lysine to alanine (KtoA) mutation to inactivate PK2 have normally organized sarcomeres and SR, and normal muscle function. PK2 KtoA mutants have fragmented mitochondria, correlated with more mitochondrially-associated DRP-1. PK2 KtoA mutants have increased ATP levels, increased glycolysis and altered levels of electron transport chain complexes. Muscle mitochondria show increased complex I and decreased complex II basal respiration, each of which cannot be uncoupled. This suggests that mutant mitochondria are already uncoupled, possibly resulting from an increased level of the uncoupling protein, UCP-4. Our results suggest signaling from sarcomeres to mitochondria, to help match energy requirements with energy production.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Subject terms:</strong> Cytoskeleton, Cell signalling, Mitochondria</p></section></section><section class="abstract" id="Abs2"><hr class="headless"/>
<p id="Par2">UNC-89, a giant protein localized to muscle sarcomeres, contains 2 protein kinase domains, one of which, PK2 is a canonical kinase. A missense mutation expected to inactivate PK2 results in worms with an unexpected phenotype suggesting signaling from sarcomeres to mitochondria.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Muscle sarcomeres contain extraordinarily large proteins, &gt;700,000 Da<sup><a aria-describedby="CR1" class="usa-link" href="#CR1">1</a></sup>. Examples include twitchin, UNC-89, and TTN-1 in <em>C. elegans</em><sup><a aria-describedby="CR2" class="usa-link" href="#CR2">2</a></sup>; projectin, obscurin, and Sallimus in Drosophila<sup><a aria-describedby="CR3" class="usa-link" href="#CR3">3</a></sup>, and titin and obscurin in vertebrates<sup><a aria-describedby="CR4" class="usa-link" href="#CR4">4</a>,<a aria-describedby="CR5" class="usa-link" href="#CR5">5</a></sup>. These proteins contain multiple immunoglobulin (Ig) and fibronectin type III domains (Fn), one or two protein kinase domains, and often highly elastic intrinsically disordered regions.</p>
<p id="Par4">Loss-of-function mutations in the <em>C. elegans</em> gene <em>unc-89</em> result in slow-moving worms with disorganized muscle sarcomeres, including a lack of M-lines<sup><a aria-describedby="CR6" class="usa-link" href="#CR6">6</a>–<a aria-describedby="CR9" class="usa-link" href="#CR9">9</a></sup>. <em>unc-89</em> is a complex gene, which utilizes three promoters and alternative splicing to generate at least 16 polypeptides, ranging in size from 156,000 to 900,000 Da<sup><a aria-describedby="CR8" class="usa-link" href="#CR8">8</a>,<a aria-describedby="CR10" class="usa-link" href="#CR10">10</a>,<a aria-describedby="CR11" class="usa-link" href="#CR11">11</a></sup> (WormBase). Overlapping sets of isoforms are expressed in body wall muscle, pharyngeal muscle (for pumping in and grinding up bacterial food), egg-laying and intestinal muscles, and several gonadal epithelial cells. The largest of these isoforms, UNC-89-B, consists of 53 Ig domains, two Fn3 domains, a triplet of SH3, DH, and PH domains near their N-termini, and two protein kinase domains (PK1 and PK2) near their C termini. Antibodies localize UNC-89 to the M-line<sup><a aria-describedby="CR8" class="usa-link" href="#CR8">8</a>,<a aria-describedby="CR10" class="usa-link" href="#CR10">10</a></sup>. To learn how UNC-89 is localized and performs its functions, we have screened a yeast two-hybrid library with segments of the largest UNC-89 isoform and identified at least 7 interacting partners with 6 out of 7 having orthologs or homologs in humans<sup><a aria-describedby="CR2" class="usa-link" href="#CR2">2</a></sup>. Interactions were verified by a combination of in vitro binding assays using purified recombinant proteins, co-immunolocalization, and analysis of muscle mutant phenotypes. Two types of protein phosphatases interact with the protein kinase region of UNC-89. Segments containing either of the kinase domains, Fn1-Ig52-PK1 and Ig53-Fn2-PK2, interact with SCPL-1<sup><a aria-describedby="CR12" class="usa-link" href="#CR12">12</a></sup>, a CTD-type protein phosphatase. The segment, Ig53-Fn2, which lies just the N-terminal of PK2, interacts with PPTR-2, a regulatory subunit of protein phosphatase 2 A (PP2A)<sup><a aria-describedby="CR13" class="usa-link" href="#CR13">13</a></sup>. Although we do not know the substrates for UNC-89 kinases, or SCPL-1 and PP2A phosphatases, it could be envisioned that some of the substrates might be shared.</p>
<p id="Par5">The Drosophila genome contains a single <em>unc-89</em> gene called obsc. Mutant and RNAi knockdown flies are unable to fly and their indirect flight muscles have abnormal sarcomeric structures with missing M-lines and misplaced or absent H-zones resulting in abnormally long or short thin filaments<sup><a aria-describedby="CR14" class="usa-link" href="#CR14">14</a></sup>. Thus, the role of UNC-89 in sarcomere assembly is conserved in worms and flies.</p>
<p id="Par6">Mammalian striated muscle expresses three UNC-89-like proteins from separate genes: obscurin (OBSC)<sup><a aria-describedby="CR15" class="usa-link" href="#CR15">15</a>,<a aria-describedby="CR16" class="usa-link" href="#CR16">16</a></sup>, obscurin-like 1 (obsl1)<sup><a aria-describedby="CR17" class="usa-link" href="#CR17">17</a></sup>, and striated muscle preferentially expressed gene (SPEG)<sup><a aria-describedby="CR18" class="usa-link" href="#CR18">18</a></sup>. The most homologous protein to UNC-89 is obscurin, which also exists as many isoforms, including giant isoforms (obscurin A), and shorter isoforms with 2 protein kinase domains<sup><a aria-describedby="CR5" class="usa-link" href="#CR5">5</a></sup>. While obscurin A contains all the same domains as UNC-89, the SH3, DH, and PH domains are located near the C-terminus rather than near the N-terminus as they are in UNC-89. Although UNC-89 is located only at the M-line, various obscurin isoforms are located at either the M-line or the Z-disk<sup><a aria-describedby="CR16" class="usa-link" href="#CR16">16</a>,<a aria-describedby="CR19" class="usa-link" href="#CR19">19</a></sup>.</p>
<p id="Par7">Polymorphisms or mutations in the human OBSC gene are implicated, but not yet proven, to cause various types of cardiomyopathies, systolic heart failure, and distal skeletal myopathy<sup><a aria-describedby="CR20" class="usa-link" href="#CR20">20</a></sup>. In addition to serving as a platform for the assembly of M-line proteins, obscurin links myofibrils to the sarcoplasmic reticulum (SR). The C-terminus of obscurin interacts with an ankyrin protein localized to the SR<sup><a aria-describedby="CR21" class="usa-link" href="#CR21">21</a>,<a aria-describedby="CR22" class="usa-link" href="#CR22">22</a></sup>. The obscurin KO mouse has normal myofibril organization but disorganized SR<sup><a aria-describedby="CR23" class="usa-link" href="#CR23">23</a></sup>. This myofibril to SR linkage is conserved in nematode UNC-89<sup><a aria-describedby="CR24" class="usa-link" href="#CR24">24</a></sup>.</p>
<p id="Par8">Obsl1 is an ~200 kDa protein consisting solely of 20 tandem Ig domains, and in heart muscle is localized to M-lines, Z-disks, and intercalated disks<sup><a aria-describedby="CR17" class="usa-link" href="#CR17">17</a></sup>. Several portions interact with titin and myomesin at the M-line<sup><a aria-describedby="CR25" class="usa-link" href="#CR25">25</a></sup>. However, obsl1 is ubiquitously expressed, the global KO is embryonic lethal, and mutations in the human Obsl1 gene result in a growth defect (3M-growth syndrome)<sup><a aria-describedby="CR26" class="usa-link" href="#CR26">26</a></sup>.</p>
<p id="Par9">Substrates for UNC-89 PK2 have not yet been identified and an understanding of the in vivo substrates for the kinase domains of obscurin is in its infancy. However, sequence analysis suggests that human obscurin PK1 and PK2 are both likely to be active kinases<sup><a aria-describedby="CR27" class="usa-link" href="#CR27">27</a></sup>, and each has been shown to undergo autophosphorylation in vitro<sup><a aria-describedby="CR28" class="usa-link" href="#CR28">28</a></sup>. Obscurin PK2 can phosphorylate N-cadherin in vitro<sup><a aria-describedby="CR28" class="usa-link" href="#CR28">28</a></sup>. Obscurin PK1 has been shown to autophosphorylate several serines in the interkinase region immediately C-terminal of PK1<sup><a aria-describedby="CR29" class="usa-link" href="#CR29">29</a></sup>. Deletion of both OBSC and Obsl1 in the skeletal muscle of mice results in changes in proteins implicated in metabolic pathways and the mitochondrial electron transport chain (ETC)<sup><a aria-describedby="CR30" class="usa-link" href="#CR30">30</a></sup>. Specifically, nearly all mitochondrial electron transport chain proteins were downregulated. Analysis of the proteome data suggested downregulation of ETC proteins was more associated with loss of obscurin than with Obsl1.</p>
<p id="Par10">PK1 from Drosophila UNC-89 has been shown to be an inactive pseudokinase, with inactivity features being strictly conserved across arthropods<sup><a aria-describedby="CR31" class="usa-link" href="#CR31">31</a></sup>. Drosophila UNC-89 PK1 has also been shown to interact with PK2, suggesting that both kinases form a functional dual kinase hub, where the inactive PK1 likely serves as regulator of PK2<sup><a aria-describedby="CR31" class="usa-link" href="#CR31">31</a></sup>.</p>
<p id="Par11">So far, the experiments on obscurin have utilized a knockout of the entire protein, not specifically eliminating the activity of one or more kinase domains. Taking advantage of the genetic manipulability of <em>C. elegans</em>, we used CRISPR/Cas9 to create a worm that expresses UNC-89 having the established kinase inactivating mutation, lysine-to-alanine, in the ATP pocket of the PK2 catalytic domain. These mutants have normal sarcomere structure, normal force generation, even better than normal muscle function, and yet display fragmented mitochondria that have increased basal respiration that cannot be uncoupled further. We gained insight into molecules and processes that lead to mitochondrial fragmentation, uncoupling, and higher ATP. Our results suggest signaling from sarcomere to mitochondria which may function to match energy needs to energy production, and that PK2 of UNC-89 (obscurin) is key to this communication.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Bioinformatic analysis indicates that PK2 is an active kinase and that a mutation expected to abolish activity is not expected to result in instability</h3>
<p id="Par12">A sequence analysis of PK2 from <em>C. elegans</em> UNC-89 using <a class="usa-link" data-ga-action="click_feat_suppl" href="https://blast.ncbi.nlm.nih.gov" rel="noopener noreferrer" target="_blank">BLAST</a> confirmed that it belongs to the DMT class of CAMK kinases (Ca2+/Calmodulin-dependent kinases). The DMT class takes its name from their representative <span class="text-underline">D</span>eath-associated-protein kinases (DAPKs), <span class="text-underline">M</span>yosin light-chain-kinases (MLCKs), and <span class="text-underline">T</span>riple functional domain protein-related kinases (TRIO) and contains, in addition, various muscle kinases including human titin kinase, nematode twitchin kinase (TwcK) and mammalian obscurin and SPEG kinases<sup><a aria-describedby="CR32" class="usa-link" href="#CR32">32</a></sup>. The BLAST search revealed that PK2 shares the highest sequence identity levels with DAPKs, TwcK, and MLCK (Supplementary Fig. <a class="usa-link" href="#MOESM2">1</a>).</p>
<p id="Par13">DAPK, TwcK, and MLCK kinases are proven to be catalytically active and their phosphotransfer has been characterized<sup><a aria-describedby="CR33" class="usa-link" href="#CR33">33</a>–<a aria-describedby="CR35" class="usa-link" href="#CR35">35</a></sup>. Referencing the UNC-89-PK2 sequence to these other kinases reveals a high conservation of catalytically relevant motifs in PK2. Specifically, canonical kinases are characterized by a glycine-rich GxGxxG loop motif involved in ATP-binding, an AAK motif in β-strand β3 containing an essential lysine (that optimally positions the non-transferable α- and β-phosphate groups of ATP), a conserved glutamate residue in helix C that forms a salt bridge with the essential lysine during the catalytically committed state of the kinase, a DFG signature motif (where the aspartate residue chelates the magnesium ion required for catalysis), a catalytic aspartate and a conserved asparagine residue (also involved in cation coordination), both in the catalytic loop motif<sup><a aria-describedby="CR36" class="usa-link" href="#CR36">36</a>,<a aria-describedby="CR37" class="usa-link" href="#CR37">37</a></sup>. These groups are conserved in UNC-89-PK2 (Fig. <a class="usa-link" href="#Fig1">1c</a> and Supplementary Fig. <a class="usa-link" href="#MOESM2">1</a>). A minor variation is present only in the glycine-rich motif, which in UNC-89-PK2 adopts the sequence FxGxxS. In fact, this motif is commonly diversified across obscurin and UNC-89 PK2 kinases across phyla (Supplementary Fig. <a class="usa-link" href="#MOESM2">2</a>). Degeneration of the first and last positions of this motif is common among kinases and is known not to compromise ATP-binding or phosphotransfer. For example, the active kinases CDK11<sup><a aria-describedby="CR38" class="usa-link" href="#CR38">38</a></sup>, TP53RK<sup><a aria-describedby="CR39" class="usa-link" href="#CR39">39</a></sup>, and mouse obscurin PK2<sup><a aria-describedby="CR28" class="usa-link" href="#CR28">28</a></sup> contain ExGxxG, KxGxxA, and RxGxxS motif variants, respectively.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Three-dimensional model of UNC-89 PK2, and CRISPR/Cas9 generation of a nematode expressing PK2 with an inactivating KtoA mutation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 1" class="graphic" height="1141" id="d33e575" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6387/11487192/e05384b9dc63/42003_2024_7042_Fig1_HTML.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Cartoon representation (yellow) of the 3D model generated by AlphaFold<sup><a aria-describedby="CR68" class="usa-link" href="#CR68">68</a></sup>. Catalytically relevant segments are color-coded as in (<strong>c</strong>). As a transparent ribbon trace, the crystal structure of the active, canonical kinase DAPK1 in complex with the ATP-analog AMPPNP and Mg2+ and in the catalytically committed state is shown (PDB code 3F5U). <strong>b</strong> Close-up of the active site of UNC-89-PK2. Color code is as in (<strong>a</strong>). Catalytically relevant residues described in the main text are displayed and annotated. <strong>c</strong> Alignment of catalytically relevant sequence motifs in UNC-89-PK2 with those of representative DMT kinases: UNC-89-PK2 (PK2), human death-associated-protein kinase (DAPK1), twitchin kinase from <em>C. elegans</em> (TwcK), human myosin light-chain kinase (MLCK). <strong>d</strong> On top is a schematic representation of domains within the largest isoform of UNC-89. Ig domains, purple; Fn3 domains, green; KSPs, a region with tandem repeats and 44 KSPs; other domains as indicated. Note the two protein kinase domains, PK1 and PK2, at the C-terminus. Below is representation of part of the exon/intron organization of part of the <em>unc-89</em> gene, with the location of the PK2 coding region. Bottom right: confirmation of the CRISPR/Cas9 generated aag to gcc sequence change in <em>unc-89(sf22)</em> which specifies a K7816A mutation in PK2. Bottom, left: western blot comparing the levels of UNC-89 giant isoforms in wild type vs. <em>unc-89(sf22)</em>. The giant isoforms of UNC-89 are ~800 kDa. The asterisk indicates a protein of nonspecific reactivity detected by the anti-UNC-89 antibodies<sup><a aria-describedby="CR46" class="usa-link" href="#CR46">46</a></sup>.</p></figcaption></figure><p id="Par14">During the course of this study, the recombinant production of UNC-89-PK2 samples compatible with viable phosphotransfer experimentation could not be established. Thus, in order to further investigate whether a phosphor-transfer productive conformation is attainable by UNC-89-PK2, we calculated its 3D model using AlphaFold and tested in silico its capability of accommodating ATP/Mg2+ (Fig. <a class="usa-link" href="#Fig1">1a, b</a> and Supplementary Fig. <a class="usa-link" href="#MOESM2">3a</a>). In kinases, the mutual arrangement of N- and C-terminal fold lobes, the position of the ATP-binding glycine-rich loop, the orientation of the catalytic helix <em>α</em>C hosting the conserved glutamate residue that coordinates the magnesium counterion, the conformation of the activation loop that governs access to the catalytic aspartate at the peptide binding site, the conformation of the conserved DFG-motif, and the assembly state of two dynamic structural networks of hydrophobic residues termed <em>spines</em>, is dynamically altered to achieve catalytic readiness<sup><a aria-describedby="CR36" class="usa-link" href="#CR36">36</a></sup>. Nucleotide binding to the kinase normally induces the formation of the catalytically committed state. Distinctly, DMT kinases exist in a catalytically-primed state in the absence of ATP-binding, which does not induce a further conformational rearrangement<sup><a aria-describedby="CR31" class="usa-link" href="#CR31">31</a>,<a aria-describedby="CR32" class="usa-link" href="#CR32">32</a></sup>. In agreement with this existent knowledge, the 3D model of UNC-89-PK2 displayed a regular kinase fold with features typical of the catalytically-primed conformational state of a kinase (Fig. <a class="usa-link" href="#Fig1">1a, b</a>). At the residue level, the ATP-binding lysine K7816 is modeled forming a typical salt bridge to the conserved glutamic acid E7827 in the catalytic helix αC. The canonical DFG-motif (residues 7921-7923) is modeled in the “in”-conformation and the activation loop is in an open state. Both the regulatory (R)-spine (residues L7831, L7842, H7899 and F7922) and the catalytic (C-)spine (I7799, A7814, V7863, I7907, M7908, F7909, V7965 and L7969) are assembled. The modeled structure of UNC-89-PK2 closely resembles that of human DAPK1 and was also compatible with its ATP/Mg2+ binding mode (Fig. <a class="usa-link" href="#Fig1">1a, b</a> and Supplementary Fig. <a class="usa-link" href="#MOESM2">3b</a>). Thus, we conclude that UNC-89-PK2 shares the sequence and conformational features of established canonical kinases.</p>
<p id="Par15">Mutation of the K in the AAK motif to A or several other residues has been shown to inactivate most known kinases<sup><a aria-describedby="CR40" class="usa-link" href="#CR40">40</a></sup>. To assess whether the introduction of a KtoA mutation in PK2 could render the kinase unstable, we tested the effects of this mutation on the 3D model of PK2 using FoldX v4<sup><a aria-describedby="CR41" class="usa-link" href="#CR41">41</a></sup>. This yielded a calculated comparative protein unfolding energy, ΔΔG, of −0.15 kcal/mol and indicated that no significant stability change arises from the introduction of the mutation.</p>
<p id="Par16">To summarize, UNC-89-PK2 is modeled by Alphafold in a catalytically-primed conformational state compatible with kinase activity, the active site comprises the essential functional groups necessary for phosphor-transfer and the introduced mutation is not expected to cause structural instability. Taken together, bioinformatics analysis indicates that UNC-89-PK2 is a canonical, active kinase and that the observed nematode phenotype in this work results from hindered phosphor-transfer by PK2 resulting from the inactivating KtoA mutation.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Nematodes expressing UNC-89 with PK2 KtoA have normally organized sarcomeres, SR, and microtubules, and move faster than wild-type nematodes</h3>
<p id="Par17">We used CRISPR/Cas9 to create a worm (Fig. <a class="usa-link" href="#Fig1">1d</a>), <em>unc-89(sf22)</em>, in which PK2 has a K7816A mutation. The strain was outcrossed 4 times to wild type to remove most of the potential off-target mutations induced by CRISPR. By western blot, this KtoA mutation does not detectably affect the expression or stability of full-length UNC-89 isoforms (Fig. <a class="usa-link" href="#Fig1">1d</a>) using an established antibody<sup><a aria-describedby="CR10" class="usa-link" href="#CR10">10</a></sup>. Surprisingly, unlike all other <em>unc-89</em> mutants, <em>unc-89(sf22)</em> worms have normal sarcomere structure as assessed by immunostaining using antibodies to various sarcomeric proteins, including UNC-95 for the base of M-lines and dense bodies (Z-disks), MHC A myosin heavy chain for A-bands, UNC-89 for M-lines, and F-actin for I-bands (Fig. <a class="usa-link" href="#Fig2">2a–c</a>). Detection and localization of the PK2 KtoA mutant UNC-89 to M-lines, the normal localization of wild-type UNC-89 in sarcomeres<sup><a aria-describedby="CR8" class="usa-link" href="#CR8">8</a>,<a aria-describedby="CR10" class="usa-link" href="#CR10">10</a></sup>, is consistent with the western blot showing no major loss of the mutant UNC-89 protein. To examine the myofilament lattice at higher resolution we used EM. As shown in Fig. <a class="usa-link" href="#Fig2">2d</a>, the ultrastructure of the sarcomeres in <em>unc-89(sf22)</em> and in <em>unc-89(syb1360)</em>, a second, independently isolated PK2 KtoA mutant allele (see below), appear as in wild type; there are normally organized A- and I-bands, normal numbers of thick and thin filaments, and normally organized dense bodies, M-lines and membranous sacs of the SR.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Nematodes expressing UNC-89 with the KtoA mutation in PK2 have normal muscle structure.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11487192_42003_2024_7042_Fig2_HTML.jpg" target="_blank"><img alt="Fig. 2" class="graphic zoom-in" height="1248" id="d33e712" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6387/11487192/879791502265/42003_2024_7042_Fig2_HTML.jpg" width="772"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The drawing on the right shows a cross-section through an adult worm indicating that the body wall muscle is organized into four quadrants. Each quadrant is composed of interlocking pairs of mononuclear fusiform-shaped cells (23 or 24 per quadrant). The enlargement emphasizes that the myofilament lattice is restricted to one side of the cell. The drawing depicts several planes of section; one plane emphasizes that this is striated muscle with typical A-bands containing thick filaments organized around M-lines, and overlapping thin filaments attached to dense bodies (Z-disk analogs). The black oval indicates the plane viewed when an animal, lying on a glass slide, is examined by light microscopy as shown in part (<strong>c</strong>). This drawing is a slight modification of a published drawing<sup><a aria-describedby="CR2" class="usa-link" href="#CR2">2</a></sup>, under terms of a Creative Commons Attribution License. <strong>b</strong> Idealized sarcomere and SR with the location of proteins localized in part (<strong>c</strong>). MHC A: myosin heavy chain A. Red: muscle cell membrane; blue-green: ECM; green: hypodermis; and grey: cuticle. Orange ovals: location of SR determined by electron microscopy; solid orange ovals: location of UNC-68 (Ryanodine receptor). <strong>c</strong> Wild type (left column), and <em>unc-89(sf22)</em> (right column) animals immunostained with various antibodies to proteins indicated in part (<strong>b</strong>). Phalloidin was used to image F-actin in thin filaments. In each panel, several spindle-shaped body wall muscle cells are shown. Note that there is no difference in the pattern of localization of these sarcomeric markers between wild type and <em>unc-89(sf22)</em>. Scale bars, 10 μm. <strong>d</strong> Transmission EM of body wall muscle from wild type, <em>unc-89(sf22)</em> and a second independently-generated PK2 KtoA mutant, <em>unc-89(syb1360)</em>. Bottom row displays the enlargement of one portion of the image above it. Yellow arrows, M-lines; yellow arrowheads, dense bodies, green arrows, SR. There is no discernible difference between the wild type and the two mutants in terms of numbers and organization of thick and thin filaments, M-lines, dense bodies, and SR. Scale bars, 0.5 μm.</p></figcaption></figure><p id="Par18">Because deficiency of mouse obscurin<sup><a aria-describedby="CR23" class="usa-link" href="#CR23">23</a></sup> and worm UNC-89<sup><a aria-describedby="CR24" class="usa-link" href="#CR24">24</a></sup> result in disorganized SR, we examined SR organization using the marker UNC-68 (Ryanodine receptor). However, HA-tagged UNC-68 endogenously expressed from a CRISPR/Cas9 strain (Supplementary Fig. <a class="usa-link" href="#MOESM2">4a</a>), or myc-tagged UNC-68 overexpressed from a transgene (Supplementary Fig. <a class="usa-link" href="#MOESM2">4b</a>) show normal localization in <em>unc-89(sf22)</em>.</p>
<p id="Par19">Obscurin is also required for normal microtubule organization<sup><a aria-describedby="CR42" class="usa-link" href="#CR42">42</a></sup>. However, we found no difference in microtubule organization comparing wild type vs. <em>unc-89(sf22)</em>, using a transgenic line expressing TBA-1 (α-tubulin) tagged with HA (Supplementary Fig. <a class="usa-link" href="#MOESM2">5</a>).</p>
<p id="Par20"><em>unc-89</em> is also expressed in the single sarcomere muscle cells of the pharynx. UNC-89 is normally localized to the middle of the A-bands of pharyngeal sarcomeres<sup><a aria-describedby="CR8" class="usa-link" href="#CR8">8</a>,<a aria-describedby="CR10" class="usa-link" href="#CR10">10</a></sup>. All previously characterized <em>unc-89</em> mutants, except for <em>unc-89(tm752)</em>, have abnormally organized filaments as observed by polarized light or immunostaining<sup><a aria-describedby="CR6" class="usa-link" href="#CR6">6</a>,<a aria-describedby="CR9" class="usa-link" href="#CR9">9</a>,<a aria-describedby="CR11" class="usa-link" href="#CR11">11</a></sup>, with characteristic arc-like thick filaments at the periphery of the terminal bulb. As shown in Supplementary Fig. <a class="usa-link" href="#MOESM2">6</a>, both <em>unc-89(sf22)</em> and <em>unc-89(syb1360)</em> display normal pharyngeal muscle as assessed by staining with anti-UNC-89 and the pharyngeal muscle-specific MHC C.</p>
<p id="Par21">Nearly all previously characterized <em>unc-89</em> mutants have reduced whole animal locomotion and generate reduced maximal force<sup><a aria-describedby="CR9" class="usa-link" href="#CR9">9</a></sup>, most likely because sarcomere disorganization results in reduced muscle activity. The one known exception is <em>unc-89(ad539)</em>, but this mutant only affects the organization of the pharyngeal muscle. Two independently derived UNC-89 KtoA mutants, <em>unc-89(sf22)</em> and <em>unc-89(syb1360)</em>, crawl faster than wild-type worms on an agar surface (Fig. <a class="usa-link" href="#Fig3">3a</a>), and generate the same amount of maximal force<sup><a aria-describedby="CR43" class="usa-link" href="#CR43">43</a></sup> as wild type (Fig. <a class="usa-link" href="#Fig3">3b</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. UNC-89 PK2 KtoA mutants move faster than wild type but exert the same amount of force as wild type.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 3" class="graphic" height="1401" id="d33e861" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6387/11487192/39c57371fcb1/42003_2024_7042_Fig3_HTML.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Crawling assays on wild type (WT), several <em>unc-89</em> loss of function alleles (<em>e1460, su75</em>, <em>tm752</em>), and two independently derived PK2 KtoA mutants (<em>sf22</em> and <em>syb1360</em>). Body bends per second (BBPS) are quantified for individual animals of each strain. In the graphs, each dot represents the result from an independently selected animal. Welch’s <em>t</em>-test was used for significance. Error bars (red) indicate SEMs, *<em>p</em> ≤ 0.05, ****<em>p</em> ≤ 0.0001. <strong>b</strong> NemaFlex force measurements of individual worms of the same strains indicated in part (<strong>a</strong>). Statistical analysis was performed using the Wilcoxon rank-sum test. <em>N</em> = 25–36; error bars (red) indicate SEMs, *<em>p</em> ≤ 0.05, ***<em>p</em> ≤ 0.001.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Nematodes expressing UNC-89 PK2 KtoA, or an <em>unc-89</em> mutant lacking expression of all kinase-containing isoforms, have fragmented mitochondria</h3>
<p id="Par22">Blondelle et al.<sup><a aria-describedby="CR30" class="usa-link" href="#CR30">30</a></sup> created a mouse doubly knocked out for both obscurin and obsl1 and demonstrated that by proteome analysis, these mice have changes in metabolic pathways, including decreased levels of nearly all mitochondrial electron transport chain complexes. This report prompted us to examine the morphology and function of mitochondria in UNC-89 KtoA mutants. Using an antibody to the mitochondrial inner membrane protein ATP5A, and identifying the body wall muscle cells using antibodies to UNC-95, we found that mitochondria in wild-type muscle are organized into long wavy chains, parallel to the sarcomeric A- and I-bands (Fig. <a class="usa-link" href="#Fig4">4a</a>), as reported by others<sup><a aria-describedby="CR44" class="usa-link" href="#CR44">44</a></sup>. In comparison, mitochondria in <em>unc-89(sf22)</em> appear fragmented. This can be seen by confocal (Fig. <a class="usa-link" href="#Fig4">4a</a>) and structured illumination microscopy (SIM) (Fig. <a class="usa-link" href="#Fig4">4b</a>). In contrast, nematodes expressing the related giant protein twitchin with a catalytically dead protein kinase domain, <em>unc-22(sf21)</em>, or twitchin with a missense mutation in Ig7, <em>unc-22(e105)</em><sup><a aria-describedby="CR45" class="usa-link" href="#CR45">45</a></sup> have normally organized mitochondria (Supplementary Fig. <a class="usa-link" href="#MOESM2">7</a>). This suggests that effects on mitochondrial morphology are specific for UNC-89 PK2 and not a function shared by protein kinases of the other giant muscle proteins in the worm. We find similar abnormal fragmented morphology of the mitochondria in <em>unc-89(tm752)</em> animals, which lack expression of all kinase-containing isoforms, but do express other isoforms<sup><a aria-describedby="CR8" class="usa-link" href="#CR8">8</a>,<a aria-describedby="CR11" class="usa-link" href="#CR11">11</a></sup>(Fig. <a class="usa-link" href="#Fig4">4a</a>). In contrast, a more normal (wild type) morphology of mitochondria is seen in two other <em>unc-89</em> mutants: <em>unc-89(e1460)</em>—a stop codon in an alternatively spliced exon for Ig20-Ig26<sup><a aria-describedby="CR24" class="usa-link" href="#CR24">24</a>,<a aria-describedby="CR46" class="usa-link" href="#CR46">46</a></sup>, which expresses kinase-containing isoforms; and in <em>unc-89(su75)</em>—a frameshift mutation in the exon for Ig39<sup><a aria-describedby="CR46" class="usa-link" href="#CR46">46</a></sup>, that lacks expression of all giant isoforms but expresses the small kinase-containing isoforms (Fig. <a class="usa-link" href="#Fig4">4a, b</a>). Our impressions of mitochondrial morphology are supported by a quantitative analysis of the SIM images (Supplementary Fig. <a class="usa-link" href="#MOESM2">8</a>): binning the mitochondria for length indicates that <em>unc-89(sf22)</em> and <em>unc-89(tm752)</em> have no mitochondria &gt;10 μm, and fewer mitochondria between 5 and 10 μm as compared to either wild type or <em>unc-89(e1460)</em>. Therefore, either inactivation of PK2 kinase or lack of expression of PK2-containing isoforms results in fragmented mitochondria.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Body wall muscle cells of UNC-89 PK2 KtoA mutants have fragmented mitochondria.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11487192_42003_2024_7042_Fig4_HTML.jpg" target="_blank"><img alt="Fig. 4" class="graphic zoom-in" height="1249" id="d33e1017" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6387/11487192/e2e18ed8bd9c/42003_2024_7042_Fig4_HTML.jpg" width="653"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Confocal images of body wall muscle stained with antibodies to ATP5A to reveal the morphology of mitochondria. Note that wild type has mitochondria organized as long wavy lines running parallel to the A- and I-bands, described as “tubular”. In contrast, <em>unc-89(sf22)</em> which carries the PK2 KtoA mutation, and <em>unc-89(tm752)</em> which lacks expression of all kinase-containing isoforms have mitochondria that appear as dots or can be described as fragmented. By comparison, <em>unc-89(e1460)</em> which maintains expression of all kinase-containing isoforms, and <em>unc-89(su75)</em> which lacks expression of all giant isoforms but still expresses small kinase-containing isoforms have mitochondria that appear more tubular or more like wild type. <strong>b</strong> 3D rendering of SIM images of muscle mitochondria of the indicated strains. Note the presence of linear or tubular mitochondria in wild type and <em>unc-89(e1460)</em> and fragmented mitochondria in <em>unc-89(sf22)</em> and <em>unc-89(tm752)</em>. <strong>c</strong>, <strong>d</strong> Three additional independently isolated PK2 KtoA mutants (<em>sf24</em>, <em>syb1195</em>, and <em>syb1360)</em> show fragmented mitochondria. Scale bars, 10 μm in (<strong>a</strong> and <strong>c</strong>); 5 μm in (<strong>b</strong> and <strong>d</strong>).</p></figcaption></figure><p id="Par23">Because CRISPR/Cas9 can result in off-target effects, we generated 3 more CRISPR alleles (<em>sf24</em>, <em>syb1195</em>, and <em>syb1360</em>) resulting in the same KtoA amino acid change. <em>sf22</em> and <em>sf24</em> are independent alleles generated from the same sgRNA, and <em>syb1195</em> and <em>syb1360</em> are independent alleles generated using a different sgRNA. The mutant alleles were outcrossed 3X to wild type. The new mutants displayed the same mitochondrial fragmentation as <em>unc-89(sf22)</em> worms (Fig. <a class="usa-link" href="#Fig4">4c, d</a>), suggesting that the KtoA mutation in UNC-89 PK2 adversely affects mitochondrial morphology.</p></section><section id="Sec6"><h3 class="pmc_sec_title">In PK2 KtoA mutants, mitochondrial fragmentation results from greater association of DRP-1 with mitochondria</h3>
<p id="Par24">In <em>C. elegans</em>, mitochondrial fusion and fission are primarily controlled by 3 proteins<sup><a aria-describedby="CR44" class="usa-link" href="#CR44">44</a></sup> (Fig. <a class="usa-link" href="#Fig5">5a</a>): fusion is mediated by FZO-1 (ortholog of mitofusin 1 (Mfn1) and mitofusin 2 (Mfn2)) on the outer membranes of opposing mitochondria, and by the inner membrane protein EAT-3 (OPA1); fission is mediated by DRP-1 (Drp1), which is mostly cytosolic but can be recruited to mitochondria where it forms a large spiral that constricts mitochondrial membranes. We find that the <em>drp-1(tm1108)</em> mutant produces large clumped mitochondria as reported previously<sup><a aria-describedby="CR47" class="usa-link" href="#CR47">47</a></sup>(Fig. <a class="usa-link" href="#Fig5">5b</a>), and that an <em>unc-89(sf22); drp-1(tm1108)</em> double mutant shows the Drp-1 phenotype, i.e., <em>drp-1</em> is epistatic to <em>unc-89(sf22)</em>. The epistasis of <em>drp-1</em> over <em>sf22</em> indicates that the fragmentation of mitochondria in <em>unc-89(sf22)</em> requires DRP-1 protein. One possibility is that UNC-89 PK2 may be an inhibitor of DRP-1 activity, and DRP-1 may be more active when PK2 has no catalytic activity. This further suggests that DRP-1 protein could be a target of the protein kinase activity of UNC-89 PK2, directly or indirectly through a kinase cascade.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. <em>drp-1</em> is epistatic to <em>unc-89(sf22).</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 5" class="graphic" height="653" id="d33e1119" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6387/11487192/26076a9cb236/42003_2024_7042_Fig5_HTML.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Mitochondria are constantly undergoing fusion and fission, through the actions of several conserved GTPases of the dynamin family. The names of the nematode proteins are indicated with their mammalian orthologs shown in parentheses. <strong>b</strong> 3D rendering of SIM images of muscle mitochondria stained with anti-ATP5A. The <em>drp-1</em> null mutant shows abnormally clumped mitochondria, whereas <em>unc-89(sf22)</em> shows fragmented (smaller) mitochondria. The double mutant, <em>unc-89(sf22); drp-1(tm1108)</em> shows the Drp-1 single mutant phenotype. Scale bar, 5 μm.</p></figcaption></figure><p id="Par25">In mammals, Drp-1 shuttles between the cytosol and mitochondria, and upon association with mitochondria, fission is promoted<sup><a aria-describedby="CR48" class="usa-link" href="#CR48">48</a></sup>. We tested the hypothesis that part of the mechanism by which <em>unc-89</em> KtoA mutants have more mitochondrial fragmentation is that more DRP-1 is associated with their mitochondria. We developed an antibody to DRP-1 and validated the antibody by showing that it detects a protein of expected size from wild type (~80 kDa), but no protein is detectable from a <em>drp-1</em> deletion mutant (Fig. <a class="usa-link" href="#Fig6">6a</a>). We observed no differences in the total levels of DRP-1 between wild type and either of two <em>unc-89</em> PK2 KtoA mutants (<em>unc-89(sf22)</em> or <em>unc-89(syb1360)</em>) (Fig. <a class="usa-link" href="#Fig6">6b</a>). However, we found an approximately two-fold higher level of DRP-1 associated with purified mitochondria from either of the PK2 mutants compared to wild type (Fig. <a class="usa-link" href="#Fig6">6c</a>). Therefore, the greater association of DRP-1 with mitochondria at least partially explains how the mitochondria in PK2 KtoA mutants are more fragmented.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. More DRP-1 is associated with UNC-89 PK2 KtoA mutant mitochondria as compared to wild-type mitochondria.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11487192_42003_2024_7042_Fig6_HTML.jpg" target="_blank"><img alt="Fig. 6" class="graphic zoom-in" height="1201" id="d33e1192" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6387/11487192/a703c30eff38/42003_2024_7042_Fig6_HTML.jpg" width="725"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Left: Schematic representation of domains in DRP-1 as predicted by PFAM, and location of the immunogen used to raise antibodies to DRP-1. Right: Western blot showing the reaction of affinity-purified anti-DRP-1 to a protein of expected size for DRP-1 from wild type, and absence of detectable DRP-1 protein from the <em>drp-1</em> null mutant. <strong>b</strong> Left: A representative western blot showing the reaction of anti-DRP-1 against total protein extracts from wild type, <em>unc-89(sf22)</em>, and <em>unc-89(syb1360)</em>. Right: quantitation of results indicating no significant difference in total DRP-1 levels in wild type vs. the two <em>unc-89</em> mutants. au: arbitrary units; ns: not significant. Statistical significance was assessed using an unpaired <em>t</em>-test with Welch’s correction. <strong>c</strong> Left: A representative western blot showing the reaction of anti-DRP-1 against extracts of mitochondrial preparations from wild type, <em>unc-89(sf22)</em>, and <em>unc-89(syb1360)</em>. Right: quantitation of results indicating that there is more DRP-1 associated with mitochondria in the two mutants vs. wild type. au: arbitrary units. Statistical significance was assessed using an unpaired <em>t</em>-test with Welch’s correction.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">Increased levels of ATP in PK2 KtoA mutants</h3>
<p id="Par26">Because mitochondrial fragmentation is usually associated with decreased respiration and ATP production<sup><a aria-describedby="CR49" class="usa-link" href="#CR49">49</a>,<a aria-describedby="CR50" class="usa-link" href="#CR50">50</a></sup>, we compared levels of ATP in wild type vs. each of the 4 PK2 KtoA mutant alleles. Using a luciferase assay on whole worms, we were surprised to find that each mutant shows at least a 40% increase of ATP compared with wild type (Fig. <a class="usa-link" href="#Fig7">7</a>). However, <em>unc-89</em> is expressed only in the body wall, pharyngeal, vulval and intestinal muscle, and a few myoepithelial cells of the oviduct. Thus, we utilized a method to assess ATP levels primarily in the body wall muscle, the major site of UNC-89 expression. The ATP:ADP ratio, is a sensitive indicator of cellular energy status<sup><a aria-describedby="CR51" class="usa-link" href="#CR51">51</a></sup>. The fluorescent biosensor PercevalHR can measure this ratio in living cells<sup><a aria-describedby="CR52" class="usa-link" href="#CR52">52</a>,<a aria-describedby="CR53" class="usa-link" href="#CR53">53</a></sup>. The ATP:ADP ratio is proportional to fluorescence at 529 nm at two excitation wavelengths (405 and 488 nm)<sup><a aria-describedby="CR54" class="usa-link" href="#CR54">54</a></sup>. The <em>myo-3</em> gene, which encodes myosin heavy chain A (MHC A), is primarily expressed in body wall muscle but is also expressed in a few single sarcomere muscle cells and in a few myoepithelial cells of the gonadal sheath. We created transgenic worms that express PercevalHR primarily in body wall muscle using the <em>myo-3</em> promoter. We first validated the technique by showing: (i) the ATP:ADP ratio is decreased in animals treated with antimycin A (inhibits complex III) (Supplementary Fig. <a class="usa-link" href="#MOESM2">9a, c</a>), and (ii) the ATP/ADP ratio is decreased in <em>drp-1(tm1108</em>) (Supplementary Fig. <a class="usa-link" href="#MOESM2">9b, d</a>), consistent with a report that <em>drp-1</em> mutants have decreased ATP levels<sup><a aria-describedby="CR55" class="usa-link" href="#CR55">55</a></sup>. The ATP/ADP ratio is increased in <em>unc-89(sf22)</em> (Supplementary Fig. <a class="usa-link" href="#MOESM2">9b, d</a><em>)</em>, consistent with the increased levels of ATP in the whole animals (Fig. <a class="usa-link" href="#Fig7">7</a>). Although most of these results do not reach the level of statistical significance, they do show trends consistent with our tentative conclusions.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. PK2 KtoA mutants have elevated ATP levels.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 7" class="graphic" height="667" id="d33e1275" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6387/11487192/2389fb7c0bd3/42003_2024_7042_Fig7_HTML.jpg" width="750"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig7/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Results of luciferase assays on extracts of whole worms from the indicated strains. For each strain, <em>n</em> = 3, and *: <em>p</em> &lt; 0.05 by Dunnett’s test.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Complex I respiration is increased, complex II respiration is decreased, and each cannot be uncoupled in <em>unc-89</em> PK2 KtoA mutants</h3>
<p id="Par27">We next tested the hypothesis that the increase in ATP levels in PK2 KtoA mutants is due to enhanced mitochondrial function. We utilized a method to isolate mitochondria from any cell type in <em>C. elegans</em><sup><a aria-describedby="CR56" class="usa-link" href="#CR56">56</a></sup> to isolate mitochondria primarily from body wall muscle cells. We used a strain in which the fusion protein tomm-20-mKate2-HA was expressed from the <em>myo-3</em> promoter, total mitochondria were purified, and then mostly muscle-specific mitochondria were pulled out using anti-HA-magnetic beads. Mitochondrial respiration in muscle mitochondria was assayed using an oxygen electrode. We investigated both complex I and complex II-based respiration as electrons can take two paths through the electron transport chain: either I to III to IV or II to III to IV. When assessed for complex I induced respiration (addition of substrates malate and glutamate), muscle mitochondria from <em>unc-89(sf22)</em> and <em>unc-89(syb1360)</em> showed significantly higher rates of basal respiration, and the respiration could not be uncoupled by adding FCCP (Fig. <a class="usa-link" href="#Fig8">8a</a>). When assessed for complex II-induced respiration (addition of the substrate succinate), the UNC-89 KtoA mutants showed lower rates of basal respiration, which could also not be uncoupled (Fig. <a class="usa-link" href="#Fig8">8b</a>).</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8. PK2 KtoA mutants display altered mitochondrial respiration and altered levels of some electron transport chain complexes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11487192_42003_2024_7042_Fig8_HTML.jpg" target="_blank"><img alt="Fig. 8" class="graphic zoom-in" height="716" id="d33e1376" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6387/11487192/bc8f743a06cc/42003_2024_7042_Fig8_HTML.jpg" width="800"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig8/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, <strong>b</strong> Muscle mitochondria from PK2 KtoA mutants have increased complex I and decreased complex II basal respiration and each cannot be uncoupled. <strong>a</strong> Complex I (glutamate and malate) stimulated basal and uncoupled mitochondrial oxygen consumption rates measured in isolated muscle mitochondria from wild type (wt), <em>unc-89(sf22),</em> and <em>unc-89 (syb1360)</em> mutant strains (<em>n</em> = 3–8 per strain). <strong>b</strong> Complex II (succinate) stimulated basal and uncoupled mitochondrial oxygen consumption rates measured in isolated muscle mitochondria from wild type (wt), <em>unc-89(sf22),</em> and <em>unc-89 (syb1360)</em> mutant strains (<em>n</em> = 4–5 per strain). Means and standard errors of the mean are represented. Significance was tested by one-way ANOVA followed by Tukey’s post hoc test. *<em>p</em> ≤ 0.05, **<em>p</em> ≤ 0.005, ***<em>p</em> ≤ 0.0005, ****<em>p</em> ≤ 0.00005. <strong>c</strong>, <strong>d</strong> PK2 KtoA mutants have altered levels of electron transport chain complexes. <strong>c</strong> Representative western blots reacted against antibodies to the indicated components of complexes I, II, IV, V, and cytochrome C. <strong>d</strong> Quantitation of these results with <em>n</em> = 3. au: arbitrary units. Statistical significance was assessed using an unpaired <em>t</em>-test with Welch’s correction. Note that as compared to wild type, complexes I and V are normal, but complexes II and IV are reduced, and cytochrome C is elevated in the PK2 KtoA mutants.</p></figcaption></figure><p id="Par28">To gain insight into why complex I respiration is increased and complex II respiration is decreased, we assessed the protein levels of one component of each electron chain complex by western blots. Unfortunately, we could not identify a commercially available antibody that reacted against a complex III component, so complex III was not assessed. As shown in Fig. <a class="usa-link" href="#Fig8">8c, d</a>, the levels of complexes I and V are normal, complexes II and IV are decreased, and cytochrome C (which shunts electrons from III to IV) is increased. These results only partially explain why complex II-based respiration is decreased.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Increased glycolysis in PK2 KtoA mutants</h3>
<p id="Par29">Although the increase in complex I-based respiration might partially explain the increased level of ATP in the PK2 KtoA mutants, we explored the additional possibility that glycolysis was increased. There is precedence for a compensatory increase in glycolysis when mitochondria do not perform well<sup><a aria-describedby="CR57" class="usa-link" href="#CR57">57</a></sup>. We measured the activity of the rate-limiting enzyme in glycolysis, 6-phosphofructose kinase. The activity of this enzyme is significantly increased in all 4 PK2 KtoA mutants (Fig. <a class="usa-link" href="#Fig9">9</a>). Thus, increased glycolysis at least partially explains why there is an increase in total ATP levels in PK2 KtoA mutants, however, the mechanism of this increase remains elusive.</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9. PK2 KtoA mutants display increased activity of the rate-limiting enzyme of glycolysis, 6-phosphofructose kinase.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 9" class="graphic" height="1004" id="d33e1404" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6387/11487192/295e34c41b29/42003_2024_7042_Fig9_HTML.jpg" width="750"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig9/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Statistical significance was assessed using an unpaired <em>t</em>-test with Welch’s correction.</p></figcaption></figure></section><section id="Sec10"><h3 class="pmc_sec_title">Uncoupling might result from increased levels of the uncoupling protein UCP-4 in PK2 KtoA mutants</h3>
<p id="Par30">That both ETC I and ETC II-based respiration cannot be further increased by uncoupling reagents (Fig. <a class="usa-link" href="#Fig8">8a, b</a>), suggests that the mutant mitochondria are already uncoupled. The proton gradient across the intermembrane space is generated by electron transfer, but the gradient can be dissipated by uncoupling proteins (UCPs)<sup><a aria-describedby="CR58" class="usa-link" href="#CR58">58</a></sup>. UCPs are a family of transporters that permit proton movement back into the mitochondrial matrix, and some UCPs are involved in generating heat. Thus, uncoupling should result in less ATP production, and the observed increase in ATP is likely due to increased glycolysis (Fig. <a class="usa-link" href="#Fig9">9</a>). Mammals express 5 UCP proteins, but <em>C. elegans</em> express only a single UCP, called UCP-4. Nematode UCP-4 is primarily expressed in the body wall and pharyngeal muscles<sup><a aria-describedby="CR59" class="usa-link" href="#CR59">59</a></sup>. We wished to explore the possibility that the level of UCP-4 is altered in the PK2 KtoA mutants. We created a CRISPR/Cas9 strain in which UCP-4 has an HA tag at its C-terminus, and this strain was crossed into <em>unc-89(sf22)</em> and <em>unc-89(syb1360)</em>. As shown in Fig. <a class="usa-link" href="#Fig10">10</a>, the level of UCP-4 protein, but not mRNA, is increased in PK2 KtoA mutants. The higher level of UCP-4 likely explains the uncoupled state of PK2 KtoA mutants, but why UCP-4 levels are increased awaits further study.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10. PK2 KtoA mutants have elevated levels of UCP-4 protein but not mRNA.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 10" class="graphic" height="1213" id="d33e1471" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6387/11487192/bae3d3c2447e/42003_2024_7042_Fig10_HTML.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig10/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Representative western blot reacted with anti-HA to detect UCP-4-HA. An HA tag was inserted at the C-terminus of UCP-4 by CRISPR/Cas9 in a wild-type background and crossed into the <em>unc-89</em> mutants. <strong>b</strong> Quantitation of western blot results with <em>n</em> = 3–5. Statistical significance was assessed using an unpaired <em>t</em>-test with Welch’s correction. <strong>c</strong> Results of quantitative real-time PCR comparing levels of <em>ucp-4</em> mRNA in wild type vs. the <em>unc-89</em> mutants. Statistical significance was assessed using an unpaired <em>t</em>-test with Welch’s correction.</p></figcaption></figure></section><section id="Sec11"><h3 class="pmc_sec_title">Increasing the level of PK2 increases mitochondrial clumping and decreases DRP-1 associated with mitochondria</h3>
<p id="Par31">Since the presumed inactivation of PK2 leads to mitochondrial fragmentation we wondered what would happen if the activity of PK2 was increased. We tested this by overexpressing a portion of UNC-89 containing wild-type PK2. We reported that a segment consisting of the C-terminal third of the interkinase region followed by Ig53, Fn2, and PK2 (1/3 IK-Ig-Fn-Pk2), when transgenically expressed from a heat shock promoter localizes to the M-line, the normal location of UNC-89<sup><a aria-describedby="CR46" class="usa-link" href="#CR46">46</a></sup>. This strain and wild type, as day 1 adult animals, were heat-shocked at 30° for 2 h, and then immunostained using anti-ATP5A to image the mitochondria. As shown in Supplementary Fig. <a class="usa-link" href="#MOESM2">10a</a>, although mitochondria from the wild type showed the expected linear/tubular organization, mitochondria from the strain expressing the PK2 fragment showed more clumping. As shown in Supplementary Fig. <a class="usa-link" href="#MOESM2">10b</a>, heat shock of this extra PK2 did not change the total level of DRP-1, but as shown in Supplementary Fig. <a class="usa-link" href="#MOESM2">10c</a>, it decreased the level of DRP-1 associated with mitochondria.</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par32">We first performed bioinformatics analysis indicating that UNC-89 PK2 is a canonical active protein kinase and that a particular missense mutation (KtoA) is expected to abolish activity but not the stability of the protein. We used CRISPR/Cas9 to create worms that express UNC-89 isoforms with catalytically inactive PK2 protein kinase domains and we observed an unexpected and specific phenotype. In contrast to all previously described <em>unc-89</em> loss of function mutants, these worms have normal sarcomere and SR structures and move faster than the wild type. In addition, PK2 KtoA mutants have fragmented mitochondria. Although mitochondrial fragmentation is usually associated with decreased mitochondrial function, we found that ATP levels were increased and complex I basal respiration was increased and was already uncoupled, and there was no obvious increase in complex I proteins. Nevertheless, complex II basal respiration was decreased and the level of complex II was decreased. Thus, it seems that the effect of PK2 KtoA mutation both increases and decreases mitochondrial function. The observed increase in glycolysis might be a compensatory response to altered mitochondrial respiration.</p>
<p id="Par33">We gained molecular insight into how the mitochondria are fragmented. The conserved GTPase Drp1 is required for mitochondrial fragmentation, and in mammals, this occurs by recruitment of Drp1 to the outer membrane by adaptors, then oligomerization, recruitment of dynamin 2, and membrane scission<sup><a aria-describedby="CR60" class="usa-link" href="#CR60">60</a></sup>. We found that there is more DRP-1 associated with mitochondria of PK2 KtoA mutants, thus explaining the higher degree of fragmentation. Conversely, increasing the level of wild-type PK2 by transgenic overexpression increases mitochondrial clumping and decreases DRP-1 associated with mitochondria. The association of Drp1 is known to be regulated by phosphorylation in a complex manner, some phosphorylation sites promoting association, others inhibiting it<sup><a aria-describedby="CR48" class="usa-link" href="#CR48">48</a></sup>. Our finding that genetically <em>drp-1</em> is epistatic to <em>unc-89(sf22)</em> is consistent with the DRP-1 protein being a direct or indirect target of UNC-89 PK2 protein kinase activity.</p>
<p id="Par34">That both complex I and complex II respiration could not be further uncoupled led us to postulate that the mitochondria are already uncoupled. The proton gradient across the intermembrane space can be dissipated by uncoupling proteins which transport protons back into the mitochondrial matrix. Although mammals express 5 UCP proteins, <em>C. elegans</em> expresses only a single UCP, UCP-4. This allowed us to easily explore whether this mechanism of uncoupling is abnormal in PK2 KtoA mutants. We found that the level of UCP-4 protein is elevated in PK2 KtoA mutants as compared to wild type. The higher level of UCP-4 likely explains the uncoupled state of PK2 KtoA mutants; however, how this occurs awaits further experiments. One possibility is that the expression, stability, or activity of UCP-4 is regulated by phosphorylation, which also may occur by PK2 or a PK2 kinase cascade. Moreover, more uncoupling means that mitochondria produce less ATP, and thus elevated ATP levels in PK2 KtoA mutants are most likely due to increased glycolysis.</p>
<p id="Par35">Further insight into the mechanisms that generate the phenotype of PK2 KtoA mutants awaits determining the in vivo substrates for PK2. Unfortunately, during the course of this study, we were unsuccessful in producing recombinant PK2 samples suitable for in vitro phosphotransfer experimentation. Nonetheless, the phenotype of PK2 KtoA mutants suggests the existence of a signaling pathway between the sarcomere, where UNC-89 is located, and mitochondria. Such communication would help match the energy requirements of muscle contraction with energy production by mitochondria. In mammalian striated muscle, large quantities of ATP are required to fuel sarcomere contraction and the ionic pumps that drive muscle contraction, and most of this ATP is supplied by numerous mitochondria. There is very close packing of organelles in striated muscle and the spaces between adjacent myofibril/SR/T-tubule cylinders are filled with mitochondria. There is a complex 3D network of mitochondria in mammalian skeletal muscle, mostly in the form of fused or elongated mitochondria<sup><a aria-describedby="CR61" class="usa-link" href="#CR61">61</a></sup>. This organization likely ensures rapid and efficient transfer of ATP to the contractile apparatus<sup><a aria-describedby="CR62" class="usa-link" href="#CR62">62</a></sup>. It is believed that during muscle contraction, calcium ions, and ATP are released into a privileged microdomain and taken up into myofibrils before they have time to diffuse beyond the myofibrils. Because of the close proximity of mitochondria to sarcomeres, even in <em>C. elegans</em> muscle, and the need to match energetic output to contractile demands, we speculate that there is signaling from sarcomeres to mitochondria that coordinate the structural organization and metabolic activity of mitochondria, and this is essential for optimal muscle performance.</p>
<p id="Par36">Although our studies provide a possible molecular mechanism by which sarcomeres communicate with mitochondria, our study is not the first to provide evidence of communication between these organelles. In a remarkably sophisticated study using Drosophila, Avellaneda et al.<sup><a aria-describedby="CR63" class="usa-link" href="#CR63">63</a></sup> demonstrated that there is communication between mitochondria and myofibrils during muscle development. Drosophila have two types of muscle, fibrillar, exemplified by indirect flight muscle in which individual myofibrils are not laterally aligned with their neighbors, and body muscles such as in the legs in which myofibrils are aligned laterally to form a tube. The morphology and organization of mitochondria are different in these muscle types. In indirect flight muscle, the mitochondria are elongated and lie between the myofibrils, whereas in leg muscle the mitochondria lie mostly within the tube and peripherally. When Avellaneda et al. overexpressed a mitofusin homolog required for mitochondrial fusion or expressed a dominant negative Drp1 required for fission, the morphology of indirect flight muscle resembled leg muscle with myofibrils aligned laterally and often forming tubes and mitochondria located inside.</p>
<p id="Par37">Increased mitochondrial fragmentation has been linked to a reduction in mitochondrial function<sup><a aria-describedby="CR49" class="usa-link" href="#CR49">49</a>,<a aria-describedby="CR64" class="usa-link" href="#CR64">64</a></sup>. However, some studies suggest that mitochondrial fragmentation is not always associated with decreased function and may be beneficial. For example, in rat gastrocnemius muscle, resistance exercise resulted in an increase in activated Drp1 (Drp1 phosphorylated at serine 616), suggesting activation of mitochondrial fission<sup><a aria-describedby="CR65" class="usa-link" href="#CR65">65</a></sup>. Feng et al.<sup><a aria-describedby="CR66" class="usa-link" href="#CR66">66</a></sup> reported that in rats during early exercise, there was increased expression of the fission mediator Fis1. A more definitive study was reported on heart muscle<sup><a aria-describedby="CR67" class="usa-link" href="#CR67">67</a></sup>. After exercising mice for 1 h, the authors found that mitochondria were fragmented, total Drp1 and activated Drp1 were increased, oxidative stress was not increased, mitochondrial membrane potential was maintained, pro-mitophagy proteins decreased, and mitochondrial respiration was increased. Their conclusion is that mitochondrial fragmentation associated with increased mitochondrial function is part of the normal cardiac response to increased energetic demand encountered during normal levels of exercise. Our findings in <em>C. elegans</em> striated muscle—that a kinase-dead version of UNC-89 results in mitochondrial fragmentation but increased respiration and ATP levels—are consistent with this report on exercised mouse heart.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Bioinformatics analysis of PK2</h3>
<p id="Par38">Three-dimensional structural models of UNC-89-PK2 (UniProtKB <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/protein/O01761" rel="noopener noreferrer" target="_blank">O01761</a>) were calculated using AlphaFold v2.3.1<sup><a aria-describedby="CR68" class="usa-link" href="#CR68">68</a></sup> as implemented in the ColabFold 1.5.5 server (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/AlphaFold2.ipynb" rel="noopener noreferrer" target="_blank">https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/AlphaFold2.ipynb</a>).</p>
<p id="Par39">In order to estimate the effect of the KtoA mutation (K7816A in UniProtKB <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/protein/O01761" rel="noopener noreferrer" target="_blank">O01761</a>) on kinase stability, we calculated a comparative protein unfolding energy, ΔΔG, between wild type and in silico mutated PK2 models, where ΔΔG = ΔG<sub>KtoA</sub> – ΔG<sub>wt</sub>. To calculate the ΔΔG value, the best-ranking AlphaFold model was used as template in FoldX v4<sup><a aria-describedby="CR41" class="usa-link" href="#CR41">41</a></sup>, applying the RepairPDB function for gentle model equilibration, followed by the introduction of the KtoA mutation using the PositionScan function, as described in ref. <sup><a aria-describedby="CR69" class="usa-link" href="#CR69">69</a></sup>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">
<em>C. elegans</em> strains</h3>
<p id="Par40">N2 (wild type, Bristol)</p>
<p id="Par41">GB360 <em>unc-89(sf22)</em>, outcrossed 4× to wild type</p>
<p id="Par42">GB361 <em>unc-89(sf24)</em>, outcrossed 3× to wild type</p>
<p id="Par43">GB362 <em>unc-89(syb1195)</em>, outcrossed 3× to wild type</p>
<p id="Par44">GB363 <em>unc-89(syb1360)</em>, outcrossed 3× to wild type</p>
<p id="Par45">
<em>unc-89(tm752)</em>
</p>
<p id="Par46">CB1460 <em>unc-89(e1460)</em></p>
<p id="Par47">HE75 <em>unc-89(su75)</em></p>
<p id="Par48">GB359 <em>unc-22(sf21)</em>, outcrossed 4× to wild type</p>
<p id="Par49">
<em>unc-22(e105)</em>
</p>
<p id="Par50">CU6372 <em>drp-1(tm1108)</em></p>
<p id="Par51">GB364 <em>unc-89(sf22); drp-1(tm1108)</em></p>
<p id="Par52">ZW129 <em>zwIs108</em> [myo-3p::myc::unc-68; myo-3p::GFP]</p>
<p id="Par53">GB369 <em>unc-89(sf22); zwIs108</em></p>
<p id="Par54">SU1002 <em>unc-68</em>(jc78[<em>HA::unc-68</em>]) <em>cash-1</em>(jc60[TagRFP::myc::cash-1]V</p>
<p id="Par55">GB366 <em>unc-89(sf22)</em>; <em>unc-68</em>(jc78[<em>3Xha::unc-68</em>]) <em>cash-1</em>(jc60[TagRFP::myc::cash-1]</p>
<p id="Par56">GB357 <em>sfEx</em>[myo-3p::TBA-1::HA; sur-5::GFP]</p>
<p id="Par57">GB358 <em>unc-89(sf22); sfEx</em>[myo-3p::TBA-1::HA; sur-5::GFP]</p>
<p id="Par58">SJZ884 <em>foxSi127</em> [myo-3p::tomm20::mKate2::HA::tbb-2 3′UTR]</p>
<p id="Par59">GB353 <em>unc-89(sf22) I; foxSi127</em> [myo-3p ::tomm20::mKate2::HA::tbb-2 3′UTR]</p>
<p id="Par60">GB354 <em>unc-89(syb1360); foxSi127</em> [myo-3p ::tomm20::mKate2::HA::tbb-2 3′UTR]</p>
<p id="Par61">GB296 <em>sfIs21</em>[myo-3p::PercevalHR; lin-44p::GFP]</p>
<p id="Par62">GB355 <em>unc-89(sf22); sfIs21</em>[myo-3p ::PercevalHR; lin-44p::GFP]</p>
<p id="Par63">GB356 <em>drp-1(tm1108); sfIs21</em>[myo-3p ::PercevalHR; lin-44p::GFP]</p>
<p id="Par64">PHX6586 <em>ucp-4(syb6586)</em></p>
<p id="Par65">GB367 <em>unc-89(sf22); ucp-4(syb6586)</em></p>
<p id="Par66">GB368 <em>unc-89(syb1360); ucp-4(syb6586)</em></p>
<p id="Par67">GB267 <em>sfIs13</em>[hsp ::HA ::UNC-89 (1/3 IK-Ig-Fn-PK2 ; sur-5 ::GFP)]</p></section><section id="Sec16"><h3 class="pmc_sec_title">CRISPR/Cas9 generation of UNC-89 PK2 KtoA mutants, and HA tagging of UCP-4</h3>
<p id="Par68">The UNC-89 PK2 KtoA mutants, <em>unc-89(sf22)</em> and <em>unc-89(sf24)</em> were generated using the following strategy.</p>
<p id="Par69">Vector: pPD162 (Peft-3::Cas9 + Empty sgRNA)</p>
<p id="Par70">Target sequence in pPD162: ACGAATCTTCACAGCGCAA</p>
<p id="Par71">Template sequence (100 nt ssDNA):</p>
<p id="Par72">3′ATTCATACTCTGAGATCGCTTCAGACGATGGATGACGAA<span class="text-underline">TggcCA</span>CtGCGCAgTGtGCGTGACCTTCTGTTGTGGAATCGACGGCGTCGCGGATCACAGA5′</p>
<p id="Par73">ggc: KtoA point mutation</p>
<p id="Par74"><span class="text-underline">under lined</span>: MscI site used to detect heterozygotes</p>
<p id="Par75">Lower case g: silent mutation</p>
<p id="Par76">Primers for insertion of sgRNA target sequence:</p>
<p id="Par77">Fw: 5′ACGAATCTTCACAGCGCAAgttttagagctagaaatagcaagt3′</p>
<p id="Par78">Rv: 5′caagacatctcgcaataggaggtg3′</p>
<p id="Par79">PK2 coding region (length 968 bp) from genomic DNA:</p>
<p id="Par80">ATGATCCAACAGGACGGTTCCAGgtaagggacacttcaggaaataaaaagaaagtcatgtgaaattttccctttatactgggaataattaggcaaaatccctactaacaatatccatattttggtacactgaaagtaattctaaacctgtattttataacgagacatctataaattttctagcagaaatttgcccccaagaaaattatttttttgaataatttttaaaagagttttttttggtgttttggggtgagtttcataatacatatataattaatacaattttgcactcacaagtaaaaaatagttcttgtggcattttaaaatttaaatttatatcattttttcatttcactattccacataaaattgttcagATCGGTGGTCTCAAGTTCAAGGGACGTTTCTCTGTGATCCGCGACGCCGTCGATTCCACAACAGAAGGTCACGC<span class="text-underline">CCATTGCGCTGTGAAGATTCGTC</span>ATCCATCGTCTGAAGCGATCTCAGAGTATGAATCGCTTCGTGATGGTCAGCATGAAAATGTTCAACGCCTTATCGCCGCATTCAATAACTCCAATTTCTTGTATCTATTATCGGAAAGACTCTACGAAGATGTGTTTTCTCGTTTTGTGTTCAACGATTATTATACAGAAGAACAAGTTGCATTGACAATGAGACAAGTCACTTCGGCACTTCATTTCTTGCATTTCAAAGGgtgagctttaacagttattaacagttattaattcctaattctaatattttgattttagAATTGCCCATCTTGATGTGAATCCACACAACATAATGTTCCAATCAAAACGTAGTTGGGTCGTGAAACTAGTTGATTTTGGAAGAGCACAAAAAGTGTCGGGAGCTGTGAAACCAGTTGATTTTGATACTAAATGGGCTTCACCAGAATTCCATATTCCGGAAACTCCGGTTACCGTTCAAAGTGACATGTGGGGTATGGGA</p>
<p id="Par81"><span class="text-underline">Under lined</span>: PAM sequence (GN<sub>19</sub>NGG)</p>
<p id="Par82">Red Upper case: AAG (K) in PK2</p>
<p id="Par83">MscI digestion: (968 bp -&gt; 504 bp/464 bp)</p>
<p id="Par84">Primers for examination of the point mutations:</p>
<p id="Par85">Fw: 5′ATGATCCAACAGGACGGTTCCAG3′</p>
<p id="Par86">Rv: 5′TCCCATACCCCACATGTCACTTTG3′</p>
<p id="Par87">The UNC-89 PK2 KtoA mutants, <em>unc-89(syb1195)</em> and <em>unc-89(syb1360)</em>, and placement of an HA tag at C-terminus of UCP-4 was carried out by SunyBiotech (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.sunybiotech.com" rel="noopener noreferrer" target="_blank">http://www.sunybiotech.com</a>). Details about the sgRNAs and repair templates used are given in Supplementary Fig. <a class="usa-link" href="#MOESM2">11</a>.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Western blots</h3>
<p id="Par88">The method of Hannak et al.<sup><a aria-describedby="CR70" class="usa-link" href="#CR70">70</a></sup> was used to prepare total protein lysates from wild type, <em>unc-89(sf22)</em>, <em>unc-89(syb1360)</em>, <em>drp-1(tm1108)</em>, PHX6586 <em>ucp-4(syb6586)</em>, GB367 <em>unc-89(sf22); ucp-4(syb6586)</em>, GB368 <em>unc-89(syb1360); ucp-4(syb6586)</em>, and GB267 <em>sfIs13</em>[hsp::HA::UNC-89 (1/3 IK-Ig-Fn-PK2; sur-5::GFP)] worms. To detect UNC-89 proteins, we separated the protein lysate on a 3% stacking and 5% separating Laemmli SDS-PAGE gel, transferred to nitrocellulose membrane for 2 h, and reacted with anti-UNC-89 EU30 affinity-purified rabbit polyclonal antibodies at 1:400 dilution<sup><a aria-describedby="CR10" class="usa-link" href="#CR10">10</a></sup>, followed by reaction with anti-rabbit Ig-horseradish peroxidase and ECL. Western blots to detect DRP-1 were used in three types of experiments. First, to check the specificity of anti-DRP-1 antibodies, equal quantities of total protein lysates from wild type and <em>drp-1(tm1108)</em> were separated on a 10% Laemmli SDS-PAGE gel, transferred to nitrocellulose, and reacted against affinity-purified anti-DRP-1 at 1:200 dilution which had been pre-absorbed against an acetone powder of E. coli OP50 to reduce or eliminate reaction to bacterial proteins. Second, we used a similar approach to compare the levels of DRP-1 in total protein lysates from wild type, <em>unc-89(sf22)</em> and <em>unc-89(syb1360)</em>. Third, we compared the level of DRP-1 from total worm mitochondria from wild type, <em>unc-89(sf22)</em> and <em>unc-89(syb1360)</em>. Preparation of total worm mitochondria by differential centrifugation was conducted as described below in the section on purifying muscle mitochondria. We also performed western blots to compare the levels of representative components of electron chain complexes from total protein lysates. Antibodies and concentrations were: complex I, mouse monoclonal to NDUFS3 (Abcam cat. no. ab14711) at 1:2,500 dilution; complex II, rabbit anti-SDHA (Cell Signaling, cat. no. 11998 P) at 1:500 dilution; complex IV, mouse monoclonal to MTCO1 (Abcam cat. no. ab14705) at 1:2000 dilution; complex V, mouse monoclonal to ATP5A (Abcam cat. no. ab147481) at 1:5000 dilution; and mouse monoclonal to cytochrome C (Abcam cat. no. ab110325) at 1:1000 dilution. Finally, we performed western blots to measure the level of UCP-4-HA in wild type, <em>unc-89(sf22)</em> and <em>unc-89(syb1360)</em> backgrounds using anti-HA (rabbit monoclonal cat. no. C29F4 from Cell Signaling Technology) at 1:1000 dilution. Before use, the optimal dilutions of each antibody were determined to yield linear detection of the proteins. Protein bands were quantitated by comparing to scans of Ponceau S staining of a gel lane on the blot. The uncropped and unedited blot images used to prepare Figs. <a class="usa-link" href="#Fig1">1</a>d, <a class="usa-link" href="#Fig6">6</a>a–c, <a class="usa-link" href="#Fig8">8</a>c, <a class="usa-link" href="#Fig10">10a</a>, and Supplementary Fig. <a class="usa-link" href="#MOESM2">10b, c</a>, are shown respectively in Supplementary Figs. <a class="usa-link" href="#MOESM2">12</a>–<a class="usa-link" href="#MOESM2">16</a>.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Immunostaining and confocal microscopy of body wall muscle myofilament lattice and mitochondria, and pharyngeal myofilament lattice</h3>
<p id="Par89">Day 0 or day 1 adult worms were fixed and immunostained using the method described by Nonet et al.<sup><a aria-describedby="CR71" class="usa-link" href="#CR71">71</a></sup>. The following antibodies were used to stain parts of the myofilament lattice, all at 1:200 dilution: anti-UNC-95 (rabbit polyclonal Benian-13)<sup><a aria-describedby="CR72" class="usa-link" href="#CR72">72</a></sup>, anti-MHC A (mouse monoclonal 5–6)<sup><a aria-describedby="CR73" class="usa-link" href="#CR73">73</a></sup>, and anti-UNC-89(rabbit polyclonal EU30)<sup><a aria-describedby="CR10" class="usa-link" href="#CR10">10</a></sup>. Fixation and phalloidin-rhodamine staining were conducted as described in ref. <sup><a aria-describedby="CR74" class="usa-link" href="#CR74">74</a></sup>. To visualize mitochondria, we used mouse monoclonal antibody (15H4C4) also at 1:200 dilution (Abcam cat. no. ab14748). For pharyngeal muscle imaging, the fixed adult day 1 worms were incubated with anti-UNC-89 (EU30) at 1:100, and anti-myosin heavy chain C (MHC C; mouse monoclonal 9.2.1<sup><a aria-describedby="CR75" class="usa-link" href="#CR75">75</a></sup>) at 1:100. Secondary antibodies, used at 1:200 dilution, included anti-rabbit Alexa 488, and anti-mouse Alexa 594, all purchased from Invitrogen. Confocal images were captured at room temperature with a Zeiss confocal system (LSM510) equipped with an Axiovert 100 M microscope and an Apochromat x63/1.4 numerical aperture oil immersion objective, in 1× and 2.5× zoom mode. For the confocal images, the color balances were adjusted by using Adobe Photoshop (Adobe, San Jose, CA).</p>
<p id="Par90">Super-resolution microscopy was performed using a Nikon N-SIM system in 3D structured illumination mode on an Eclipse Ti-E microscope equipped with a 100×/1.49 NA oil immersion objective, 561-nm solid-state laser, and an EM-CCD camera (DU-897, Andor Technology). Super-resolution images were reconstructed using the N-SIM module in NIS-Elements software. The color balances of the images were adjusted by using Adobe Photoshop (Adobe, San Jose, CA). A Z-series every 0.2 μm was taken from the outer muscle cell membrane deeper into the muscle cell. This Z stack was then reconstructed into a 3D image using software on the N-SIM microscope. The reconstructed 3D z-stacks were analyzed using Imaris 8.6 (Bitplane), to visualize the mitochondria.</p>
<p id="Par91">Measurement of mitochondrial length in μm from the 3D reconstructions of SIM images was performed using Imaris software on a combination of 2 images from each wild type and mutant strain. The longest axis of each anti-ATP5A-staining object (mitochondrion) was determined using the Imaris parameter called “BoundingBoxOOLength”. After eliminating objects &lt;1 μm, the number of objects was placed and counted in 3 bins: 1 ≤ × &lt; 5 μm, 5 ≤ × &lt; 10 μm, and x ≥ 10 μm. The Y axis of the graph in Supplementary Fig. <a class="usa-link" href="#MOESM2">8</a> represents the number of objects in each bin divided by the total number objects.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Electron microscopy</h3>
<p id="Par92">Electron microscopy of body wall muscle was performed as described in Matsunaga et al.<sup><a aria-describedby="CR45" class="usa-link" href="#CR45">45</a></sup>. From each strain, approximately 200 days 2 adults were fixed and processed, and approximately 10 randomly selected ultrathin sections were examined. Approximately 30 low-powered images were captured that revealed all of the sarcomeres of single body wall muscle cells, and from these 10–20 images at higher magnification were captured that revealed 1–2 sarcomeres each. All these images were examined by two individuals both on a computer screen and by printouts, looking for any differences between the wild type and the two mutant alleles, <em>sf22</em> and <em>syb1360</em>. Representative images are shown in Fig. <a class="usa-link" href="#Fig2">2d</a>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Crawling assays and NemaFlex force measurement</h3>
<p id="Par93">For crawling assays, day two adults were harvested from one 6 cm NGM OP50 seeded plate with M9 buffer, and a final wash in M9 containing 0.2 g/L gelatin. Five microliters of worm suspension were added to the center of a 6 cm unseeded NGM plate and then the excess liquid was removed using a twisted KimWipe. After a five-minute adaptation time, worm crawling movement was recorded using a dissecting stereoscopic microscope fitted with a CMOS camera (Thorlabs). For all strains a total of fifteen, 10-s. videos were recorded from various sections of the plate with each video tracking an average of 8 individual animals. Video data was analyzed by Image J WrmTracker software plug-in to obtain body bends per second (BBPS) for individual animals. Worms that moved out of frame and outliers were removed during data analysis and an average of 20 animals were analyzed for each strain. The resulting BBPS values for each mutant strain were compared to the wild type and further tested for statistically significant differences using Welch’s <em>T</em>-test.</p>
<p id="Par94">The maximum exertable muscle force by <em>C. elegans</em> strains was measured using the NemaFlex method<sup><a aria-describedby="CR43" class="usa-link" href="#CR43">43</a></sup>. The technique is based on the deflection of soft micropillars as the animals crawl through an array of micropillars. The pillars are arranged in a square lattice with a pillar diameter of 44 μm and height of 87 μm. The gap between the pillars is 71 μm. Day 1 adult animals were loaded individually in each chamber<sup><a aria-describedby="CR76" class="usa-link" href="#CR76">76</a></sup>, and a 1-min video was captured for each animal in a food-free environment at a temperature of 20 ± 1 °C. Imaging was performed in brightfield using a Nikon Ti-E microscope and an Andor Zyla sCMOS 5.5 camera. Images were acquired at 5 frames per second with a 4× objective at a pixel resolution of 1.63 μm per pixel. Movies were processed and analyzed for force values using in-house-built image processing software (MATLAB, R2016a). Animal muscle strength measurements were calculated by identifying the maximal force exerted in each frame for an individual animal. We define the maximum exertable force <em>f</em><sub>95</sub> corresponding to the 95th percentile of these maximal forces. Statistical analysis was performed using the Wilcoxon rank-sum test.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Generation of antibodies to DRP-1</h3>
<p id="Par95">Rabbit polyclonal antibodies were generated to the C-terminal 143 residues of DRP-1b, amino acids 570-712. The coding sequence of this segment was amplified by PCR from a cDNA pool (RB2 library) and cloned into vectors pGEX-KK1 and pMAL-KK, to express glutathione S-transferase (GST) and MBP fusion proteins in E. coli<sup><a aria-describedby="CR77" class="usa-link" href="#CR77">77</a></sup>. In brief, to express each fusion protein, the plasmids were transformed into E. coli Rosetta 2 (DE3) (Millipore Corporation, cat. no. 71397-4), and grown in Luria-Bertani broth containing 10 µg/ml ampicillin at 37° to OD<sub>600</sub> = 0.6–0.8, followed by induction of expression with 0.5 mM isopropyl β-D-1-thiogalactopyranoside at 20° for 5 h. Cells were pelleted by centrifugation and resuspended in 50 mM Tris-HCl pH 8.0 containing 100 mM phenylmethylsulfonyl fluoride (PMSF) and cOmplete Mini protease inhibitor cocktail (Roche, Inc., cat. no. 11836170001), and broken in a French pressure cell at 1000 pounds inch<sup>−2</sup>, the addition of Triton X-100 to 1%, followed by spinning out debris by centrifugation at 12,000 × <em>g</em> for 20 min. The lysates were added to either a slurry of glutathione-agarose beads (Sigma, cat. no. G-4510) or amylose resin (New England BioLabs, cat. no. E8021L) and incubated with mixing for 30 min at 4°. The beads were washed 5X with 50 mM Tris-HCl pH 8.0 containing 1% Triton X-100 and 100 mM PMSF, and then 4× with 50 mM Tris-HCl pH 8.0 containing 100 mM PMSF. The beads were placed into minicolumns and eluted with either 10 mM free glutathione or 10 mM maltose in 50 mM Tris-HCl pH 8.0. The GST fusions were shipped to Noble Life Sciences (Sykesville, Maryland) for the production of rabbit antibodies. Antibodies were affinity-purified using Affi-gel (Bio-Rad)-conjugated MBP fusions, as described<sup><a aria-describedby="CR78" class="usa-link" href="#CR78">78</a></sup>. Briefly, the MBP fusion proteins were dialyzed against 100 mM 3-morpholinopropane-1-sulfonic acid (MOPS) pH 7.0, and then 3–10 mg were covalently coupled to 1 ml of a 50:50 mixture of Affi-Gel 10 and Affi-Gel 15 beads (Bio-Rad Laboratories, cat. nos. 1536099 and 153-6051). These columns were then used to affinity purify ~2 ml of antiserum and the antibodies eluted with low and high pH and concentrated with a centrifugal filter (Centriprep 10, Millipore, cat. no. 4304).</p></section><section id="Sec22"><h3 class="pmc_sec_title">Measurement of ATP levels in whole worms</h3>
<p id="Par96">100 day 1 adults were collected and washed free from bacteria, pelleted in a 1.5 ml Eppendorf tube, frozen in liquid nitrogen, and then placed in a boiling water bath for 15 min. After placing on ice for 5 min and vortexing for 5 s, debris was pelleted by centrifugation at maximum speed in a microfuge for 10 min at 4 °C. Supernatants were transferred to fresh tubes and diluted 5–10 fold with deionized water and kept on ice. Using this material, the total protein content was determined by the BCA protein assay kit (Pierce, cat. no. 23225), and the total ATP determined using a CellTiter-Glo Luminescent Cell Viability Assay kit from Promega Corporation. Results were expressed as nmoles of ATP per milligram of protein.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Use of PercevalHR to measure ATP/ADP ratio</h3>
<p id="Par97">The plasmid pRsetB-PercevalHR was purchased from Addgene (Cambridge, MA), and The XbaI/XhoI fragment of pRsetB-PercevalHR was inserted into the vector pPD95.86 via its XbaI/SalI sites to allow expression of PercevalHR from the muscle-specific promoter for the <em>myo-3</em> gene. The resulting plasmid, myo-3p::PercevalHR at 10 ng/ml, was co-injected into worms together with the transformation marker plasmid lin-44p::GFP at 90 ng/ml, into wild-type worms, and transgenic animals were selected on the basis of GFP expression in the worm tail. The extrachromosomal array was then integrated randomly into the genome by ultraviolet irradiation<sup><a aria-describedby="CR79" class="usa-link" href="#CR79">79</a></sup> with modifications (Peter Barrett, personal communication), to create strain GB296, <em>sfIs21</em>[myo-3p::PercevalHR; lin-44p::GFP]. <em>sfIs21</em> was crossed into <em>unc-89(sf22)</em> and into <em>drp-1(tm1108)</em>, to create strains GB355 and GB356, respectively. PercevalHR is based on GlnK1 which undergoes a major conformational change depending on whether ATP or ADP is bound. Coupling of GlnK1 to mVenus creates a fluorescent read-out of this conformation. An ATP to ADP ratio is proportional to emission at 533 nm upon excitation at 405 nm and 488 nm. Day 1 transgenic adults were immobilized for imaging without drugs using a combination of a PDMS microfluidic device with channels just large enough to accommodate adult worms and Pluronic F127<sup><a aria-describedby="CR80" class="usa-link" href="#CR80">80</a></sup>. The microfluidic devices were kindly manufactured and provided by Gongchen Sun from the Hang Lu laboratory at the Georgia Institute of Technology. Aqueous solutions of Pluronic F127 are fluid at 4°, and solid at room temperature. Body wall muscle was excited with a laser at 405 and at 488 nm using an Olympus FV1000 inverted confocal microscope. Ratiometric images of body wall muscle were created by dividing the emissions, Em 488/Em 405. Using Fiji software on these images, the total number of pixels with intensity greater than 100 were counted, and the means and standard deviations were calculated from multiple body wall muscle cells from 5 animals of each strain.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Purification of muscle mitochondria and respirometry</h3>
<p id="Par98">Body wall muscle-specific mitochondria were purified from strains expressing tomm20-mKate2-HA from the promoter of the muscle-specific gene <em>myo-3</em>, using a modified version of the procedure developed by Ahier et al.<sup><a aria-describedby="CR56" class="usa-link" href="#CR56">56</a></sup>. Briefly, for each strain, we harvested a mixed-stage population of nematodes from 2, 15 cm, high peptone plates seeded with E. coli strain NA22. Worms were washed free from bacteria by at least 3 washes of M9 buffer, followed by 2 washes with deionized water. The worm pellets were resuspended in 3.5 mls of ice-cold Mitochondrial Isolation Buffer (MIB; 50 mM KCl, 110 mM mannitol, 70 mM sucrose, 0.1 mM EDTA, 5 mM Tris pH 7.4 without protease inhibitors) and transferred to a 7 ml Dounce homogenizer. While on ice, we brought the pestle up and down 25 times. The homogenate was transferred to 3, 1.5 ml Eppendorf tubes, and spun in a microfuge at 4° for 5 min at 200 × <em>g</em>. The resulting supernatant was transferred to 3 fresh tubes, and centrifuged for 10 min at 800 × <em>g</em>. This supernatant was then centrifuged for 10 min at 12,000 × <em>g</em> to pellet the mitochondria. The pellets from all 3 Eppendorf tubes were resuspended in a final volume of 600 μl of ice-cold PEB (PBS, 2 mM EDTA, 1% bovine serum albumin, without protease inhibitors). To this suspension was added 30 μl of a suspension of anti-HA magnetic beads (Pierce, cat. no. 88836), and the tube placed on a rotating wheel at 4° for one hour. The beads were then pulled out of the solution using a tube stand containing neodymium magnets (Millipore Corporation), and then the beads were washed twice with 1 ml of PEB, and then resuspended in 50–100 μl of MIB and kept on ice.</p>
<p id="Par99">Mitochondrial respiration was measured using a Clark electrode (Oxytherm System, Hansatech) as described<sup><a aria-describedby="CR81" class="usa-link" href="#CR81">81</a></sup>. Briefly, isolated muscle mitochondria were suspended in respiration buffer containing 120 mM KCl, 3 mM HEPES, 1 mM EGTA, 5 mM KH2PO4, and 0.3% (w/v) fat-free BSA. Complex I substrates (10 mM glutamate, and 2 mM malate) or complex II substrates (7 mM succinate) were added in addition to 1.2 mM ADP to stimulate basal respiration. Uncoupled respiration was measured with the addition of 80 nM FCCP (Carbonyl cyanide-p-trifluoromethoxyphenylhydrazone), and non-mitochondrial oxygen consumption was assessed with the addition of potassium cyanide (500 μM). To assess mitochondrial respiration, the non-mitochondrial oxygen consumption rate was subtracted from the total oxygen consumption rate.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Glycolysis assay</h3>
<p id="Par100">We used Abcam’s “6-Phosphofructokinase Activity Assay Kit (colorimetric)” (ab155898) essentially according to the manufacturer’s instructions. Approximately 50 milligrams of packed worms were homogenized in 200 μl of PFK Assay Buffer, using a Pellet Pestle Motor (DWK Life Sciences Kimble Kontes). After determining the protein concentrations using a Bradford assay, approximately 1 μg of total protein extract was used per assay. PFK activity was recorded as mU/mg protein.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p id="Par101">GraphPad Prism9 was used to plot and analyze numerical data. Figure legends explain the n values, whether standard deviations or standard errors of the mean were plotted, which statistical tests were used, and the definition of * values. Statistical significance was assessed using an unpaired <em>t</em>-test with Welch’s correction unless otherwise specified. Each experiment was conducted several times independently and only if the same results or tendencies were obtained, were figures and conclusions made. Many experiments involved imaging and evaluation of the images was conducted by at least two individuals but in most cases, no quantitation was involved, so statistical analysis was not used.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par102">Further information on research design is available in the <a class="usa-link" href="#MOESM5">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec28"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11487192/bin/42003_2024_7042_MOESM1_ESM.pdf">Peer Review File</a><sup> (3.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11487192/bin/42003_2024_7042_MOESM2_ESM.pdf">Supplementary Information</a><sup> (32.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11487192/bin/42003_2024_7042_MOESM3_ESM.pdf">42003_2024_7042_MOESM3_ESM.pdf</a><sup> (87.3KB, pdf) </sup><p>Description of Additional Supplementary File</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11487192/bin/42003_2024_7042_MOESM4_ESM.xlsx">Supplementary Data</a><sup> (67.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11487192/bin/42003_2024_7042_MOESM5_ESM.pdf">Reporting Summary</a><sup> (3.3MB, pdf) </sup>
</div></div></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p> These studies were primarily supported by grant no. 2050009 from the National Science Foundation to G.M.B and J.Q.K. Additional partial support includes the following: NIH grant R01HL160693 to G.M.B.; NIH grant R01GM144729 to J.Q.K.; AAF grant 06/16 from the University of Konstanz to O.M.; a Stafford Fox Research Foundation Fellowship and a National Health and Medical Research Council (NHMRC) grant GNT2010813 to S.Z.; and NASA grant NNX15AL16G to S.A.V. The authors thank SunyBiotech Corporation for generating CRISPR/Cas9 lines PHX1195, PHX1360, and PHX6586; Jeannette Taylor of Emory University’s Integrated Electron Microscopy Core for EM images; Gongcheng Sun and Hang Lu (Georgia Institute of Technology) for the microfluidic device used to immobilize worms for the Perceval imaging; Laura Fox-Goharioon and April Reedy of Emory University’s Integrated Cellular Imaging (ICI) Core for help in using an N-SIM microscope, 3D rendering, and ratiometric imaging; Nathaniel Szewczyk (Ohio University) for advice on which antibodies to use to detect mitochondrial complex proteins; Kenneth Norman (Albany Medical School) for nematode strain ZW129; the Caenorhabditis Genetics Center, which is funded by the NIH Office of Research Infrastructure Programs (P40 OD010440) for multiple nematode strains; and Robert Barstead (Oklahoma Medical Research Foundation) for the cDNA library RB2.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>G.M.B. contributed by conceiving the project, designed and interpreted most experiments, carried out some experiments, and wrote most of the manuscript. J.Q.K. designed and interpreted some experiments carried out some experiments, and wrote some parts of the manuscript. O.M. and T.D. performed the bioinformatics analysis, and O.M. wrote some sections of the manuscript and edited it. Y.M., H.Q., N.G., L.L., J.C.M., A.A., and C.J.M. performed and interpreted experiments, with Y.M., H.Q., and N.G. doing most of them. These same 7 authors also wrote small parts of the manuscript and edited it. S.Z. and S.A.V. contributed by interpreting the results and edited the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par103"><em>Communications Biology</em> thanks Belinda Bullard and the other, anonymous, reviewers for their contribution to the peer review of this work. Primary Handling Editors: Martina Rauner and Joao Valente. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The authors declare that the data supporting the results of this study are displayed in the Figures and the Supplementary Figs. The numerical source data for the graphs presented in the figures are available in the Supplementary Data.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par104">The authors declare no competing interests.</p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec29"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s42003-024-07042-3.</p></section><section class="ref-list" id="Bib1"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Kontrogianni-Konstantopoulos, A., Ackermann, M. A., Bowman, A. L., Yap, S. V. &amp; Bloch, R. J. Muscle giants: molecular scaffolds in sarcomerogenesis. <em>Physiol. Rev.</em><strong>89</strong>, 1217–1267 (2009).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1152/physrev.00017.2009" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3076733/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19789381/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Kontrogianni-Konstantopoulos,%20A.,%20Ackermann,%20M.%20A.,%20Bowman,%20A.%20L.,%20Yap,%20S.%20V.%20&amp;%20Bloch,%20R.%20J.%20Muscle%20giants:%20molecular%20scaffolds%20in%20sarcomerogenesis.%20Physiol.%20Rev.89,%201217%E2%80%931267%20(2009)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Gieseler K, Qadota H, Benian GM. Development, structure, and maintenance of C. elegans body wall muscle. (2017). WormBook, ed. The C. elegans Research Community, WormBook, 10.1895/wormbook.1.173.1.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1895/wormbook.1.81.2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5410635/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27555356/">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Schueder, F. et al. Nanobodies combined with DNA-PAINT super-resolution reveal a staggered titin nanoarchitecture in flight muscles. <em>Elife</em><strong>12</strong>, e79344 (2023).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.7554/eLife.79344" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9886278/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36645127/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Schueder,%20F.%20et%20al.%20Nanobodies%20combined%20with%20DNA-PAINT%20super-resolution%20reveal%20a%20staggered%20titin%20nanoarchitecture%20in%20flight%20muscles.%20Elife12,%20e79344%20(2023)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Loescher, C. M., Hobbach, A. J. &amp; Linke, W. A. Titin (TTN): from molecule to modifications, mechanis, and medical significance. <em>Cardiovasc. Res.</em><strong>118</strong>, 2903–2918 (2022).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/cvr/cvab328" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9648829/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34662387/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Loescher,%20C.%20M.,%20Hobbach,%20A.%20J.%20&amp;%20Linke,%20W.%20A.%20Titin%20(TTN):%20from%20molecule%20to%20modifications,%20mechanis,%20and%20medical%20significance.%20Cardiovasc.%20Res.118,%202903%E2%80%932918%20(2022)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Perry, N. A., Ackermann, M. A., Shriver, M., Hu, L. Y. &amp; Kontrogianni-Konstantopoulos, A. Obscurins: unassuming giants enter the spotlight. <em>IUBMB Life</em><strong>65</strong>, 479–486 (2013).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/iub.1157" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3957234/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23512348/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Perry,%20N.%20A.,%20Ackermann,%20M.%20A.,%20Shriver,%20M.,%20Hu,%20L.%20Y.%20&amp;%20Kontrogianni-Konstantopoulos,%20A.%20Obscurins:%20unassuming%20giants%20enter%20the%20spotlight.%20IUBMB%20Life65,%20479%E2%80%93486%20(2013)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Waterston, R. H., Thomson, J. N. &amp; Brenner, S. Mutants with altered muscle structure in C. elegans. <em>Dev. Biol.</em><strong>77</strong>, 271–302 (1980).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0012-1606(80)90475-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/7190524/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Waterston,%20R.%20H.,%20Thomson,%20J.%20N.%20&amp;%20Brenner,%20S.%20Mutants%20with%20altered%20muscle%20structure%20in%20C.%20elegans.%20Dev.%20Biol.77,%20271%E2%80%93302%20(1980)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Benian, G. M., Ayme-Southgate, A. &amp; Tinley, T. L. The genetics and molecular biology of the titin/connectin-like proteins of invertebrates. <em>Rev. Physiol. Biochem. Pharmacol.</em><strong>138</strong>, 235–268 (1999).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/BFb0119629" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10396143/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Benian,%20G.%20M.,%20Ayme-Southgate,%20A.%20&amp;%20Tinley,%20T.%20L.%20The%20genetics%20and%20molecular%20biology%20of%20the%20titin/connectin-like%20proteins%20of%20invertebrates.%20Rev.%20Physiol.%20Biochem.%20Pharmacol.138,%20235%E2%80%93268%20(1999)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Small, T. M. et al. Three new isoforms of C. elegans UNC-89 containing MLCK-like protein kinase domains. <em>J. Mol. Biol.</em><strong>342</strong>, 91–108 (2004).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jmb.2004.07.006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15313609/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Small,%20T.%20M.%20et%20al.%20Three%20new%20isoforms%20of%20C.%20elegans%20UNC-89%20containing%20MLCK-like%20protein%20kinase%20domains.%20J.%20Mol.%20Biol.342,%2091%E2%80%93108%20(2004)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Qadota, H. et al. A region of UNC-89 (obscurin) lying between two protein kinase domains is a highly elastic spring required for proper sarcomere organization. <em>J. Mol. Biol.</em><strong>432</strong>, 4799–4814 (2020).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jmb.2020.06.024" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9549344/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32645312/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Qadota,%20H.%20et%20al.%20A%20region%20of%20UNC-89%20(obscurin)%20lying%20between%20two%20protein%20kinase%20domains%20is%20a%20highly%20elastic%20spring%20required%20for%20proper%20sarcomere%20organization.%20J.%20Mol.%20Biol.432,%204799%E2%80%934814%20(2020)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Benian, G. M., Tinley, T. L., Tang, X. &amp; Borodovsky, M. The Caenorhabditis elegans gene unc-89, required for muscle M-line assembly, encodes a giant modular protein composed of Ig and signal transduction domains. <em>J. Cell Biol.</em><strong>132</strong>, 835–848 (1996).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.132.5.835" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2120741/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8603916/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Benian,%20G.%20M.,%20Tinley,%20T.%20L.,%20Tang,%20X.%20&amp;%20Borodovsky,%20M.%20The%20Caenorhabditis%20elegans%20gene%20unc-89,%20required%20for%20muscle%20M-line%20assembly,%20encodes%20a%20giant%20modular%20protein%20composed%20of%20Ig%20and%20signal%20transduction%20domains.%20J.%20Cell%20Biol.132,%20835%E2%80%93848%20(1996)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Ferrara, T. M., Flaherty, D. B. &amp; Benian, G. M. Titin / connectin-related proteins in C. elegans: a review and new findings. <em>J. Musc. Res. Cell Motil.</em><strong>26</strong>, 435–447 (2005).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s10974-005-9027-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16453163/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Ferrara,%20T.%20M.,%20Flaherty,%20D.%20B.%20&amp;%20Benian,%20G.%20M.%20Titin%20/%20connectin-related%20proteins%20in%20C.%20elegans:%20a%20review%20and%20new%20findings.%20J.%20Musc.%20Res.%20Cell%20Motil.26,%20435%E2%80%93447%20(2005)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Qadota, H. et al. A novel protein phosphatase is a binding partner for the protein kinase domains of UNC-89 (obscurin) in C. elegans. <em>Mol. Biol. Cell</em><strong>19</strong>, 2424–2432 (2008).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E08-01-0053" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2397297/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18337465/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Qadota,%20H.%20et%20al.%20A%20novel%20protein%20phosphatase%20is%20a%20binding%20partner%20for%20the%20protein%20kinase%20domains%20of%20UNC-89%20(obscurin)%20in%20C.%20elegans.%20Mol.%20Biol.%20Cell19,%202424%E2%80%932432%20(2008)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Qadota, H. et al. Protein phosphatase 2A is crucial for sarcomere organization in Caenorhabditis elegans striated muscle. <em>Mol. Biol. Cell</em><strong>29</strong>, 2084–2097 (2018).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E18-03-0192" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6232960/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29949401/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Qadota,%20H.%20et%20al.%20Protein%20phosphatase%202A%20is%20crucial%20for%20sarcomere%20organization%20in%20Caenorhabditis%20elegans%20striated%20muscle.%20Mol.%20Biol.%20Cell29,%202084%E2%80%932097%20(2018)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Katzemich, A. et al. The function of the M-line protein obscurin in controlling the symmetry of the sarcomere in the flight muscle of Drosophila. <em>J. Cell Sci.</em><strong>125</strong>, 3367–3379 (2012).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/jcs.097345" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3516378/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22467859/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Katzemich,%20A.%20et%20al.%20The%20function%20of%20the%20M-line%20protein%20obscurin%20in%20controlling%20the%20symmetry%20of%20the%20sarcomere%20in%20the%20flight%20muscle%20of%20Drosophila.%20J.%20Cell%20Sci.125,%203367%E2%80%933379%20(2012)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Bang, M. L. et al. The complete gene sequence of titin, expression of an unusual approximately 700-kDa titin isoform, and its interaction with obscurin identify a novel Z-line to I-band linking system. <em>Circ. Res.</em><strong>89</strong>, 1065–1072 (2001).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1161/hh2301.100981" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11717165/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Bang,%20M.%20L.%20et%20al.%20The%20complete%20gene%20sequence%20of%20titin,%20expression%20of%20an%20unusual%20approximately%20700-kDa%20titin%20isoform,%20and%20its%20interaction%20with%20obscurin%20identify%20a%20novel%20Z-line%20to%20I-band%20linking%20system.%20Circ.%20Res.89,%201065%E2%80%931072%20(2001)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Young, P., Ehler, E. &amp; Gautel, M. Obscurin, a giant sarcomeric Rho guanine nucleotide exchange factor protein involved in sarcomere assembly. <em>J. Cell Biol.</em><strong>154</strong>, 123–136 (2001).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.200102110" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2196875/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11448995/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Young,%20P.,%20Ehler,%20E.%20&amp;%20Gautel,%20M.%20Obscurin,%20a%20giant%20sarcomeric%20Rho%20guanine%20nucleotide%20exchange%20factor%20protein%20involved%20in%20sarcomere%20assembly.%20J.%20Cell%20Biol.154,%20123%E2%80%93136%20(2001)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Geisler, S. B. et al. Obscurin-like 1, OBSL1, is a novel cytoskeletal protein related to obscurin. <em>Genomics</em><strong>89</strong>, 521–531 (2007).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ygeno.2006.12.004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1885211/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17289344/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Geisler,%20S.%20B.%20et%20al.%20Obscurin-like%201,%20OBSL1,%20is%20a%20novel%20cytoskeletal%20protein%20related%20to%20obscurin.%20Genomics89,%20521%E2%80%93531%20(2007)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Hsieh, C. M. et al. Striated muscle preferentially expressed genes alpha and beta are two serine/threonine protein kinases derived form the same gene as the aortic preferentially expressed gene-1. <em>J. Biol. Chem.</em><strong>275</strong>, 36966–36973 (2000).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M006028200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10973969/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Hsieh,%20C.%20M.%20et%20al.%20Striated%20muscle%20preferentially%20expressed%20genes%20alpha%20and%20beta%20are%20two%20serine/threonine%20protein%20kinases%20derived%20form%20the%20same%20gene%20as%20the%20aortic%20preferentially%20expressed%20gene-1.%20J.%20Biol.%20Chem.275,%2036966%E2%80%9336973%20(2000)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Bowman, A. L. et al. Different obscurin isoforms localize to distinct sites at sarcomeres. <em>FEBS Lett.</em><strong>581</strong>, 1549–1554 (2007).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.febslet.2007.03.011" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1899168/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17382936/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Bowman,%20A.%20L.%20et%20al.%20Different%20obscurin%20isoforms%20localize%20to%20distinct%20sites%20at%20sarcomeres.%20FEBS%20Lett.581,%201549%E2%80%931554%20(2007)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Grogan, A. &amp; Kontrogianni-Konstantopoulos, A. Unraveling obscurins in heart disease. <em>Pflug. Arch.</em><strong>471</strong>, 735–743 (2018).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00424-018-2191-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6397709/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30099631/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Grogan,%20A.%20&amp;%20Kontrogianni-Konstantopoulos,%20A.%20Unraveling%20obscurins%20in%20heart%20disease.%20Pflug.%20Arch.471,%20735%E2%80%93743%20(2018)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Kontrogianni-Konstantopoulos, A., Jones, E. M., Van Rossum, D. B. &amp; Bloch, R. J. Obscurin is a ligand for small ankyrin 1 in skeletal muscle. <em>Mol. Biol. Cell</em><strong>14</strong>, 1138–1148 (2003).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E02-07-0411" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC151585/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12631729/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Kontrogianni-Konstantopoulos,%20A.,%20Jones,%20E.%20M.,%20Van%20Rossum,%20D.%20B.%20&amp;%20Bloch,%20R.%20J.%20Obscurin%20is%20a%20ligand%20for%20small%20ankyrin%201%20in%20skeletal%20muscle.%20Mol.%20Biol.%20Cell14,%201138%E2%80%931148%20(2003)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Bagnato, P., Barone, V., Giacomello, E., Rossi, D. &amp; Sorrentino, V. Binding of an ankyrin-1 isoform to obscurin suggests a molecular link between the sarcoplasmic reticulum and myofibrils in striated muscles. <em>J. Cell Biol.</em><strong>160</strong>, 245–253 (2003).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.200208109" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2172649/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12527750/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Bagnato,%20P.,%20Barone,%20V.,%20Giacomello,%20E.,%20Rossi,%20D.%20&amp;%20Sorrentino,%20V.%20Binding%20of%20an%20ankyrin-1%20isoform%20to%20obscurin%20suggests%20a%20molecular%20link%20between%20the%20sarcoplasmic%20reticulum%20and%20myofibrils%20in%20striated%20muscles.%20J.%20Cell%20Biol.160,%20245%E2%80%93253%20(2003)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Lange, S. et al. Obscurin determines the architecture of the longitudinal sarcoplasmic reticulum. <em>J. Cell Sci.</em><strong>122</strong>, 2640–2650 (2009).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/jcs.046193" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2909316/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19584095/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Lange,%20S.%20et%20al.%20Obscurin%20determines%20the%20architecture%20of%20the%20longitudinal%20sarcoplasmic%20reticulum.%20J.%20Cell%20Sci.122,%202640%E2%80%932650%20(2009)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Spooner, P. M., Bonner, J., Maricq, A. V., Benian, G. M. &amp; Norman, K. R. Large isoforms of UNC-89 (obscurin) are required for muscle cell architecture and optimal calcium release in Caenorhabditis elegans. <em>PLoS ONE</em><strong>7</strong>, e40182 (2012).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0040182" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3388081/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22768340/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Spooner,%20P.%20M.,%20Bonner,%20J.,%20Maricq,%20A.%20V.,%20Benian,%20G.%20M.%20&amp;%20Norman,%20K.%20R.%20Large%20isoforms%20of%20UNC-89%20(obscurin)%20are%20required%20for%20muscle%20cell%20architecture%20and%20optimal%20calcium%20release%20in%20Caenorhabditis%20elegans.%20PLoS%20ONE7,%20e40182%20(2012)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Fukuzawa, A. et al. Interactions with titin and myomesin target obscurin and obscurin-like 1 to the M-band: implications for hereditary myopathies. <em>J. Cell Sci.</em><strong>121</strong>, 1841–1851 (2008).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/jcs.028019" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18477606/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Fukuzawa,%20A.%20et%20al.%20Interactions%20with%20titin%20and%20myomesin%20target%20obscurin%20and%20obscurin-like%201%20to%20the%20M-band:%20implications%20for%20hereditary%20myopathies.%20J.%20Cell%20Sci.121,%201841%E2%80%931851%20(2008)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Hanson, D. et al. The primordial growth disorder 3-M syndrome connects ubiquitination to the cytoskeletal adaptor OBSL1. <em>Am. J. Hum. Genet.</em><strong>84</strong>, 801–806 (2009).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ajhg.2009.04.021" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2694976/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19481195/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Hanson,%20D.%20et%20al.%20The%20primordial%20growth%20disorder%203-M%20syndrome%20connects%20ubiquitination%20to%20the%20cytoskeletal%20adaptor%20OBSL1.%20Am.%20J.%20Hum.%20Genet.84,%20801%E2%80%93806%20(2009)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Mayans, O., Benian, G. M., Simkovic, F. &amp; Rigden, D. J. Mechanistic and functional diversity in the mechanosensory kinases of the titin-like family. <em>Biochem. Soc. Trans.</em><strong>41</strong>, 1066–1071 (2013).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1042/BST20130085" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23863180/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Mayans,%20O.,%20Benian,%20G.%20M.,%20Simkovic,%20F.%20&amp;%20Rigden,%20D.%20J.%20Mechanistic%20and%20functional%20diversity%20in%20the%20mechanosensory%20kinases%20of%20the%20titin-like%20family.%20Biochem.%20Soc.%20Trans.41,%201066%E2%80%931071%20(2013)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Hu, L. Y. &amp; Kontrogianni-Konstantopoulos, A. The kinse domains of obscurin interact with intercellulrar adhesion proteins. <em>FASEB J.</em><strong>27</strong>, 2001–2012 (2013).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1096/fj.12-221317" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3633816/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23392350/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Hu,%20L.%20Y.%20&amp;%20Kontrogianni-Konstantopoulos,%20A.%20The%20kinse%20domains%20of%20obscurin%20interact%20with%20intercellulrar%20adhesion%20proteins.%20FASEB%20J.27,%202001%E2%80%932012%20(2013)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Fleming, J. R. et al. Exploring obscurin and SPEG kinase biology. <em>J. Clin. Med.</em><strong>10</strong>, 984 (2021).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/jcm10050984" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7957886/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33801198/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Fleming,%20J.%20R.%20et%20al.%20Exploring%20obscurin%20and%20SPEG%20kinase%20biology.%20J.%20Clin.%20Med.10,%20984%20(2021)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Blondelle, J. et al. Murine obscurin and Obsl1 have functionally redundant roles in sarcolemmal integrity, sarcoplasmic reticulum organization, and muscle metabolism. <em>Commun. Biol.</em><strong>2</strong>, 178 (2019).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s42003-019-0405-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6509138/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31098411/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Blondelle,%20J.%20et%20al.%20Murine%20obscurin%20and%20Obsl1%20have%20functionally%20redundant%20roles%20in%20sarcolemmal%20integrity,%20sarcoplasmic%20reticulum%20organization,%20and%20muscle%20metabolism.%20Commun.%20Biol.2,%20178%20(2019)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Zacharchenko, T. et al. PK1 from Drosophila obscurin is an inactive pseduokinase with scaffolding properties. <em>Open Biol.</em><strong>13</strong>, 220350 (2023).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1098/rsob.220350" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10129394/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37121260/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Zacharchenko,%20T.%20et%20al.%20PK1%20from%20Drosophila%20obscurin%20is%20an%20inactive%20pseduokinase%20with%20scaffolding%20properties.%20Open%20Biol.13,%20220350%20(2023)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Temmerman, K., Simon, B. &amp; Wilmanns, M. Structural and functional diversity in the activity and regulation of DAPK-related protein kinases. <em>FEBS J.</em><strong>280</strong>, 5533–5550 (2013).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/febs.12384" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23745726/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Temmerman,%20K.,%20Simon,%20B.%20&amp;%20Wilmanns,%20M.%20Structural%20and%20functional%20diversity%20in%20the%20activity%20and%20regulation%20of%20DAPK-related%20protein%20kinases.%20FEBS%20J.280,%205533%E2%80%935550%20(2013)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Birukov, K. G. et al. Differential regulation of alternatively spliced endothelial cell myosin light chain kinase isoforms by p60(Src). <em>J. Biol. Chem.</em><strong>276</strong>, 8567–8573 (2001).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M005270200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11113114/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Birukov,%20K.%20G.%20et%20al.%20Differential%20regulation%20of%20alternatively%20spliced%20endothelial%20cell%20myosin%20light%20chain%20kinase%20isoforms%20by%20p60(Src).%20J.%20Biol.%20Chem.276,%208567%E2%80%938573%20(2001)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Shohat, G. et al. The pro-apoptotic function of death-associated protein kinase is controlled by a unique inhibitory autophosphorylation-based mechanism. <em>J. Biol. Chem.</em><strong>276</strong>, 47460–47470 (2001).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M105133200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11579085/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Shohat,%20G.%20et%20al.%20The%20pro-apoptotic%20function%20of%20death-associated%20protein%20kinase%20is%20controlled%20by%20a%20unique%20inhibitory%20autophosphorylation-based%20mechanism.%20J.%20Biol.%20Chem.276,%2047460%E2%80%9347470%20(2001)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>von Castelmur, E. et al. Identification of an N-terminal inhibitory extension as the primary mechanosensory regulator of twitchin kinase. <em>Proc. Natl Acad. Sci. USA</em><strong>109</strong>, 13608–13613 (2012).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1200697109" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3427058/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22869697/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?von%20Castelmur,%20E.%20et%20al.%20Identification%20of%20an%20N-terminal%20inhibitory%20extension%20as%20the%20primary%20mechanosensory%20regulator%20of%20twitchin%20kinase.%20Proc.%20Natl%20Acad.%20Sci.%20USA109,%2013608%E2%80%9313613%20(2012)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Taylor, S. S. &amp; Kornev, A. P. Protein kinases: evolution of dynamic regulatory proteins. <em>Trends Biochem. Sci.</em><strong>36</strong>, 65–77 (2011).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tibs.2010.09.006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3084033/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20971646/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Taylor,%20S.%20S.%20&amp;%20Kornev,%20A.%20P.%20Protein%20kinases:%20evolution%20of%20dynamic%20regulatory%20proteins.%20Trends%20Biochem.%20Sci.36,%2065%E2%80%9377%20(2011)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Endicott, J. A., Noble, M. E. &amp; Johnson, L. N. The structural basis for control of eukaryotic protein kinases. <em>Annu. Rev. Biochem.</em><strong>81</strong>, 587–613 (2012).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev-biochem-052410-090317" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22482904/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Endicott,%20J.%20A.,%20Noble,%20M.%20E.%20&amp;%20Johnson,%20L.%20N.%20The%20structural%20basis%20for%20control%20of%20eukaryotic%20protein%20kinases.%20Annu.%20Rev.%20Biochem.81,%20587%E2%80%93613%20(2012)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Hu, D., Mayeda, A., Trembley, J. H., Lahti, J. M. &amp; Kidd, V. J. CDK11 complexes promote pre-mRNA splicing. <em>J. Biol. Chem.</em><strong>278</strong>, 8623–8629 (2003).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M210057200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12501247/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Hu,%20D.,%20Mayeda,%20A.,%20Trembley,%20J.%20H.,%20Lahti,%20J.%20M.%20&amp;%20Kidd,%20V.%20J.%20CDK11%20complexes%20promote%20pre-mRNA%20splicing.%20J.%20Biol.%20Chem.278,%208623%E2%80%938629%20(2003)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Abe, Y. et al. Cloning and characterization of a p53-related protein kinase expressed in interleukin-2-activated cytotoxic T-cells, epithelial tumor cell lines, and the testes. <em>J. Biol. Chem.</em><strong>276</strong>, 44003–44011 (2001).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M105669200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11546806/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Abe,%20Y.%20et%20al.%20Cloning%20and%20characterization%20of%20a%20p53-related%20protein%20kinase%20expressed%20in%20interleukin-2-activated%20cytotoxic%20T-cells,%20epithelial%20tumor%20cell%20lines,%20and%20the%20testes.%20J.%20Biol.%20Chem.276,%2044003%E2%80%9344011%20(2001)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Iyer, G. H., Moore, M. J. &amp; Taylor, S. S. Consequences of lysine 72 mutation on the phosphorylation and activation state of cAMP-dependent kinase. <em>J. Biol. Chem.</em><strong>280</strong>, 8800–8807 (2005).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M407586200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15618230/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Iyer,%20G.%20H.,%20Moore,%20M.%20J.%20&amp;%20Taylor,%20S.%20S.%20Consequences%20of%20lysine%2072%20mutation%20on%20the%20phosphorylation%20and%20activation%20state%20of%20cAMP-dependent%20kinase.%20J.%20Biol.%20Chem.280,%208800%E2%80%938807%20(2005)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Schymkowitz, J. et al. The FoldX web server: an online force field. <em>Nucleic Acids Res.</em><strong>33</strong>, W382–W388 (2005).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gki387" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1160148/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15980494/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Schymkowitz,%20J.%20et%20al.%20The%20FoldX%20web%20server:%20an%20online%20force%20field.%20Nucleic%20Acids%20Res.33,%20W382%E2%80%93W388%20(2005)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Randazzo, D. et al. Obscurin is required for ankyrinB-dependent dystrophin localization and sarcolemma integrity. <em>J. Cell Biol.</em><strong>200</strong>, 523–536 (2013).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.201205118" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3575540/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23420875/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Randazzo,%20D.%20et%20al.%20Obscurin%20is%20required%20for%20ankyrinB-dependent%20dystrophin%20localization%20and%20sarcolemma%20integrity.%20J.%20Cell%20Biol.200,%20523%E2%80%93536%20(2013)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Rahman, M. et al. NemaFlex: a microfluidics-based technology for standardized measurement of muscular strength of <em>C. elegans</em>. <em>Lab Chip</em><strong>18</strong>, 2187–2201 (2018).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1039/c8lc00103k" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6057834/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29892747/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Rahman,%20M.%20et%20al.%20NemaFlex:%20a%20microfluidics-based%20technology%20for%20standardized%20measurement%20of%20muscular%20strength%20of%20C.%20elegans.%20Lab%20Chip18,%202187%E2%80%932201%20(2018)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Van der Bliek, A. M., Sedensky, M. M. &amp; Morgan, P. G. Cell biology of the mitochondrion. <em>Genetics</em><strong>207</strong>, 843–871 (2017).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1534/genetics.117.300262" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5676242/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29097398/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Van%20der%20Bliek,%20A.%20M.,%20Sedensky,%20M.%20M.%20&amp;%20Morgan,%20P.%20G.%20Cell%20biology%20of%20the%20mitochondrion.%20Genetics207,%20843%E2%80%93871%20(2017)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Matsunaga, Y. et al. Twitchin kinase inhibits muscle activity. <em>Mol. Biol. Cell</em><strong>28</strong>, 1591–1600 (2017).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E16-10-0707" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5469603/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28428253/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Matsunaga,%20Y.%20et%20al.%20Twitchin%20kinase%20inhibits%20muscle%20activity.%20Mol.%20Biol.%20Cell28,%201591%E2%80%931600%20(2017)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Qadota, H. et al. The SH3 domain of UNC-89 (obscurin) interacts with paramyosin, a coiled-coil protein, in Caenorhabditis elegans muscle. <em>Mol. Biol. Cell</em><strong>27</strong>, 1606–1620 (2016).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E15-09-0675" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4865318/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27009202/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Qadota,%20H.%20et%20al.%20The%20SH3%20domain%20of%20UNC-89%20(obscurin)%20interacts%20with%20paramyosin,%20a%20coiled-coil%20protein,%20in%20Caenorhabditis%20elegans%20muscle.%20Mol.%20Biol.%20Cell27,%201606%E2%80%931620%20(2016)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Scholtes, C. et al. DRP-1-mediated apoptosis induces muscle degeneration in dystrophin mutants. <em>Sci. Rep.</em><strong>8</strong>, 7354 (2018).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-018-25727-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5943356/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29743663/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Scholtes,%20C.%20et%20al.%20DRP-1-mediated%20apoptosis%20induces%20muscle%20degeneration%20in%20dystrophin%20mutants.%20Sci.%20Rep.8,%207354%20(2018)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Hu, C., Huang, Y. &amp; Li, L. Drp1-dependent mitochondrial fission plays critical roles in physiological and pathological processes in mammals. <em>Int. J. Mol. Sci.</em><strong>18</strong>, 144 (2017).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/ijms18010144" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5297777/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28098754/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Hu,%20C.,%20Huang,%20Y.%20&amp;%20Li,%20L.%20Drp1-dependent%20mitochondrial%20fission%20plays%20critical%20roles%20in%20physiological%20and%20pathological%20processes%20in%20mammals.%20Int.%20J.%20Mol.%20Sci.18,%20144%20(2017)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Chen, H., Chomyn, A. &amp; Chan, D. C. Disruption of fusion results in mitochondrial heterogeneity and dysfunction. <em>J. Biol. Chem.</em><strong>280</strong>, 26185–26192 (2005).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M503062200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15899901/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Chen,%20H.,%20Chomyn,%20A.%20&amp;%20Chan,%20D.%20C.%20Disruption%20of%20fusion%20results%20in%20mitochondrial%20heterogeneity%20and%20dysfunction.%20J.%20Biol.%20Chem.280,%2026185%E2%80%9326192%20(2005)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Jheng, H.-F. et al. Mitochondrial fission contributes to mitochondrial dysfunction and insulin resistance in skeletal muscle. <em>Mol. Cell Biol.</em><strong>32</strong>, 309–319 (2012).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/MCB.05603-11" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3255771/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22083962/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Jheng,%20H.-F.%20et%20al.%20Mitochondrial%20fission%20contributes%20to%20mitochondrial%20dysfunction%20and%20insulin%20resistance%20in%20skeletal%20muscle.%20Mol.%20Cell%20Biol.32,%20309%E2%80%93319%20(2012)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Metallo, C. M. &amp; Vander Heiden, M. G. Understanding metabolic regulation and its influence on cellular physiology. <em>Mol. Cell</em><strong>49</strong>, 388–398 (2013).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.molcel.2013.01.018" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3569837/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23395269/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Metallo,%20C.%20M.%20&amp;%20Vander%20Heiden,%20M.%20G.%20Understanding%20metabolic%20regulation%20and%20its%20influence%20on%20cellular%20physiology.%20Mol.%20Cell49,%20388%E2%80%93398%20(2013)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Berg, J., Hung, Y. P. &amp; Yellen, G. A genetically encoded fluorescent reporter of ATP:ADP ratio. <em>Nat. Methods</em><strong>6</strong>, 161–166 (2009).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.1288" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2633436/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19122669/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Berg,%20J.,%20Hung,%20Y.%20P.%20&amp;%20Yellen,%20G.%20A%20genetically%20encoded%20fluorescent%20reporter%20of%20ATP:ADP%20ratio.%20Nat.%20Methods6,%20161%E2%80%93166%20(2009)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Tantama, M., Martinez-Francois, J. R., Mongeon, R. &amp; Yellen, G. Imaging energy status in live cells with a fluorescent biosensor of the intracellular ATP-to-ADP ratio. <em>Nat. Commun.</em><strong>4</strong>, 2550l (2013).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ncomms3550" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3852917/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24096541/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Tantama,%20M.,%20Martinez-Francois,%20J.%20R.,%20Mongeon,%20R.%20&amp;%20Yellen,%20G.%20Imaging%20energy%20status%20in%20live%20cells%20with%20a%20fluorescent%20biosensor%20of%20the%20intracellular%20ATP-to-ADP%20ratio.%20Nat.%20Commun.4,%202550l%20(2013)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Zanotelli, M. R. et al. Regulation of ATP utilization during metastatic cell migration by collagen architecture. <em>Mol. Biol. Cell</em><strong>29</strong>, 1–9 (2018).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E17-01-0041" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5746062/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29118073/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Zanotelli,%20M.%20R.%20et%20al.%20Regulation%20of%20ATP%20utilization%20during%20metastatic%20cell%20migration%20by%20collagen%20architecture.%20Mol.%20Biol.%20Cell29,%201%E2%80%939%20(2018)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Bess, A. S. et al. Effects of mutations in mitochondrial dynamics-related genes on the mitochondrial response to ultraviolet C radiation in developing Caenorhabditis elegans. <em>Worm</em><strong>2</strong>, e23763 (2013).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4161/worm.23763" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3670464/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24058863/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Bess,%20A.%20S.%20et%20al.%20Effects%20of%20mutations%20in%20mitochondrial%20dynamics-related%20genes%20on%20the%20mitochondrial%20response%20to%20ultraviolet%20C%20radiation%20in%20developing%20Caenorhabditis%20elegans.%20Worm2,%20e23763%20(2013)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Ahier, A. et al. Affinity purification of cell-specific mitochondria from whole animals resolves patterns of genetic mosaicism. <em>Nat. Cell Biol.</em><strong>20</strong>, 352–360 (2018).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41556-017-0023-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29358705/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Ahier,%20A.%20et%20al.%20Affinity%20purification%20of%20cell-specific%20mitochondria%20from%20whole%20animals%20resolves%20patterns%20of%20genetic%20mosaicism.%20Nat.%20Cell%20Biol.20,%20352%E2%80%93360%20(2018)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Liemburg-Apers, D. C., Schirris, T. J., Russel, F. G., Willems, P. H. &amp; Koopman, W. J. Mitoenergetic dysfunction triggers a rapid compensatory increase in steady-state glucose flux. <em>Biophys. J.</em><strong>109</strong>, 1372–1386 (2015).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.bpj.2015.08.002" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4601044/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26445438/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Liemburg-Apers,%20D.%20C.,%20Schirris,%20T.%20J.,%20Russel,%20F.%20G.,%20Willems,%20P.%20H.%20&amp;%20Koopman,%20W.%20J.%20Mitoenergetic%20dysfunction%20triggers%20a%20rapid%20compensatory%20increase%20in%20steady-state%20glucose%20flux.%20Biophys.%20J.109,%201372%E2%80%931386%20(2015)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Slocinska, M., Barylski, J. &amp; Jarmuszkiewicz, W. Uncoupling proteins of invertebrates: a review. <em>IUBMB Life</em><strong>68</strong>, 691–699 (2016).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/iub.1535" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27385510/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Slocinska,%20M.,%20Barylski,%20J.%20&amp;%20Jarmuszkiewicz,%20W.%20Uncoupling%20proteins%20of%20invertebrates:%20a%20review.%20IUBMB%20Life68,%20691%E2%80%93699%20(2016)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Iser, W. B., Kim, D., Bachman, E. &amp; Wolkow, C. Examination of the requirement for ucp-4, a putative homolog of mammalian uncoupling proteins, for stress tolerance and longevity in C. elegans. <em>Mech. Ageing Dev.</em><strong>126</strong>, 1090–1096 (2005).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.mad.2005.04.002" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2553218/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15893362/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Iser,%20W.%20B.,%20Kim,%20D.,%20Bachman,%20E.%20&amp;%20Wolkow,%20C.%20Examination%20of%20the%20requirement%20for%20ucp-4,%20a%20putative%20homolog%20of%20mammalian%20uncoupling%20proteins,%20for%20stress%20tolerance%20and%20longevity%20in%20C.%20elegans.%20Mech.%20Ageing%20Dev.126,%201090%E2%80%931096%20(2005)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Tilokani, L., Nagashima, S., Paupe, V. &amp; Prudent, J. Mitochondrial dynamics: overview of molecular mechanisms. <em>Essay Biochem.</em><strong>62</strong>, 341–360 (2018).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1042/EBC20170104" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6056715/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30030364/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Tilokani,%20L.,%20Nagashima,%20S.,%20Paupe,%20V.%20&amp;%20Prudent,%20J.%20Mitochondrial%20dynamics:%20overview%20of%20molecular%20mechanisms.%20Essay%20Biochem.62,%20341%E2%80%93360%20(2018)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Glancy, B. et al. Mitochondrial reticulum for cellular energy distribution in muscle. <em>Nature</em><strong>523</strong>, 617–620 (2015).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature14614" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6988728/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26223627/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Glancy,%20B.%20et%20al.%20Mitochondrial%20reticulum%20for%20cellular%20energy%20distribution%20in%20muscle.%20Nature523,%20617%E2%80%93620%20(2015)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Pasqualini, F. S., Nesmith, A. P., Horton, R. E., Sheehy, S. P. &amp; Parker, K. K. Mechanotransduction and metabolism in cardiomyocyte microdomains. <em>BioMed. Res. Int.</em><strong>2016</strong>, 4081638 (2016).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1155/2016/4081638" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5164897/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28044126/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Pasqualini,%20F.%20S.,%20Nesmith,%20A.%20P.,%20Horton,%20R.%20E.,%20Sheehy,%20S.%20P.%20&amp;%20Parker,%20K.%20K.%20Mechanotransduction%20and%20metabolism%20in%20cardiomyocyte%20microdomains.%20BioMed.%20Res.%20Int.2016,%204081638%20(2016)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Avellaneda, J. et al. Myofibril and mitochondria morphogenesis are coordinated by a mechanical feedback mechanism in muscle. <em>Nat. Commun.</em><strong>12</strong>, 2091 (2021).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41467-021-22058-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8027795/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33828099/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Avellaneda,%20J.%20et%20al.%20Myofibril%20and%20mitochondria%20morphogenesis%20are%20coordinated%20by%20a%20mechanical%20feedback%20mechanism%20in%20muscle.%20Nat.%20Commun.12,%202091%20(2021)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Bach, D. et al. Mitofusin-2 determines mitochondrial network architecture and mitochondrial metabolism. A novel regulatory mechanism altered in obesity. <em>J. Biol. Chem.</em><strong>278</strong>, 17190–17197 (2003).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M212754200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12598526/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Bach,%20D.%20et%20al.%20Mitofusin-2%20determines%20mitochondrial%20network%20architecture%20and%20mitochondrial%20metabolism.%20A%20novel%20regulatory%20mechanism%20altered%20in%20obesity.%20J.%20Biol.%20Chem.278,%2017190%E2%80%9317197%20(2003)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Kitaoka, Y., Ogasawara, R., Tamura, Y., Fujita, S. &amp; Hatta, H. Effect of electrical stimulation-induced resistance exercise on mitochondrial fission and fusion proteins in rat skeletal muscle. <em>Appl. Physiol. Nutr. Metab.</em><strong>40</strong>, 1137–1142 (2015).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1139/apnm-2015-0184" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26513006/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Kitaoka,%20Y.,%20Ogasawara,%20R.,%20Tamura,%20Y.,%20Fujita,%20S.%20&amp;%20Hatta,%20H.%20Effect%20of%20electrical%20stimulation-induced%20resistance%20exercise%20on%20mitochondrial%20fission%20and%20fusion%20proteins%20in%20rat%20skeletal%20muscle.%20Appl.%20Physiol.%20Nutr.%20Metab.40,%201137%E2%80%931142%20(2015)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Feng, Z. et al. Mitochondrial dynamic remodeling in strenuous exercise-induced muscle and mitochondrial dysfunction: regulatory effects of hydroxytyrosol. <em>Free Rad. Biol. Med.</em><strong>50</strong>, 1437–1446 (2011).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.freeradbiomed.2011.03.001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21421045/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Feng,%20Z.%20et%20al.%20Mitochondrial%20dynamic%20remodeling%20in%20strenuous%20exercise-induced%20muscle%20and%20mitochondrial%20dysfunction:%20regulatory%20effects%20of%20hydroxytyrosol.%20Free%20Rad.%20Biol.%20Med.50,%201437%E2%80%931446%20(2011)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Coronado, M. et al. Physiological mitochondrial fragmentation is a normal cardiac adaptation to increased energy demand. <em>Circ. Res.</em><strong>122</strong>, 282–295 (2017).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1161/CIRCRESAHA.117.310725" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5775047/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29233845/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Coronado,%20M.%20et%20al.%20Physiological%20mitochondrial%20fragmentation%20is%20a%20normal%20cardiac%20adaptation%20to%20increased%20energy%20demand.%20Circ.%20Res.122,%20282%E2%80%93295%20(2017)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <em>Nature</em><strong>596</strong>, 583–589 (2021).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41586-021-03819-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8371605/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34265844/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Jumper,%20J.%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold.%20Nature596,%20583%E2%80%93589%20(2021)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Bogomolovas, J., Gravenhorst, P. &amp; Mayans, O. Production and analysis of titin kinase: exploiting active/inactive kinase homologs in pseudokinase validation. <em>Methods Enzymol.</em><strong>667</strong>, 147–181 (2022).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/bs.mie.2022.03.028" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35525541/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Bogomolovas,%20J.,%20Gravenhorst,%20P.%20&amp;%20Mayans,%20O.%20Production%20and%20analysis%20of%20titin%20kinase:%20exploiting%20active/inactive%20kinase%20homologs%20in%20pseudokinase%20validation.%20Methods%20Enzymol.667,%20147%E2%80%93181%20(2022)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Hannak, E. et al. The kinetically dominant assembly pathway for centrosomal asters in <em>Caenorhabditis elegans</em> is gamma-tubulin dependent. <em>J. Cell Biol.</em><strong>157</strong>, 591–602 (2002).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.200202047" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2173857/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12011109/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Hannak,%20E.%20et%20al.%20The%20kinetically%20dominant%20assembly%20pathway%20for%20centrosomal%20asters%20in%20Caenorhabditis%20elegans%20is%20gamma-tubulin%20dependent.%20J.%20Cell%20Biol.157,%20591%E2%80%93602%20(2002)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Nonet, M. L., Grundahl, K., Meyer, B. J. &amp; Rand, J. B. Synaptic function is impaired but not eliminated in <em>C. elegans</em> mutants lacking synaptotagmin. <em>Cell</em><strong>73</strong>, 1291–1305 (1993).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0092-8674(93)90357-v" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8391930/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Nonet,%20M.%20L.,%20Grundahl,%20K.,%20Meyer,%20B.%20J.%20&amp;%20Rand,%20J.%20B.%20Synaptic%20function%20is%20impaired%20but%20not%20eliminated%20in%20C.%20elegans%20mutants%20lacking%20synaptotagmin.%20Cell73,%201291%E2%80%931305%20(1993)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Qadota, H., Mercer, K. B., Miller, R. K., Kaibuchi, K. &amp; Benian, G. M. Two LIM domain proteins and UNC-96 link UNC-97/pinch to myosin thick filaments in <em>Caenorhabditis elegans</em> muscle. <em>Mol. Biol. Cell</em><strong>18</strong>, 4317–4326 (2007).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E07-03-0278" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2043538/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17761533/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Qadota,%20H.,%20Mercer,%20K.%20B.,%20Miller,%20R.%20K.,%20Kaibuchi,%20K.%20&amp;%20Benian,%20G.%20M.%20Two%20LIM%20domain%20proteins%20and%20UNC-96%20link%20UNC-97/pinch%20to%20myosin%20thick%20filaments%20in%20Caenorhabditis%20elegans%20muscle.%20Mol.%20Biol.%20Cell18,%204317%E2%80%934326%20(2007)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Miller, D. M., Ortiz, I., Berliner, G. C. &amp; Epstein, H. F. Differential localization of two myosins within nematode thick filaments. <em>Cell</em><strong>34</strong>, 477–490 (1983).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0092-8674(83)90381-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/6352051/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Miller,%20D.%20M.,%20Ortiz,%20I.,%20Berliner,%20G.%20C.%20&amp;%20Epstein,%20H.%20F.%20Differential%20localization%20of%20two%20myosins%20within%20nematode%20thick%20filaments.%20Cell34,%20477%E2%80%93490%20(1983)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Waterston, R. H., Hirsh, D. &amp; Lane, T. R. Dominant mutations affecting muscle structure in <em>C. elegans</em> that map near the actin gene cluster. <em>J. Mol. Biol.</em><strong>180</strong>, 473–496 (1984).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0022-2836(84)90023-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/6527380/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Waterston,%20R.%20H.,%20Hirsh,%20D.%20&amp;%20Lane,%20T.%20R.%20Dominant%20mutations%20affecting%20muscle%20structure%20in%20C.%20elegans%20that%20map%20near%20the%20actin%20gene%20cluster.%20J.%20Mol.%20Biol.180,%20473%E2%80%93496%20(1984)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Miller, D. M., Stockdale, F. E. &amp; Karn, J. Immunological identification of the genes encoding the four myosin heavy chain isoforms of <em>C. elegans</em>. <em>Proc. Natl Acad. Sci. USA</em><strong>83</strong>, 2305–2309 (1986).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.83.8.2305" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC323285/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/2422655/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Miller,%20D.%20M.,%20Stockdale,%20F.%20E.%20&amp;%20Karn,%20J.%20Immunological%20identification%20of%20the%20genes%20encoding%20the%20four%20myosin%20heavy%20chain%20isoforms%20of%20C.%20elegans.%20Proc.%20Natl%20Acad.%20Sci.%20USA83,%202305%E2%80%932309%20(1986)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Hewitt, J. E. et al. Muscle strength deficiency and mitochondrial dysfunction in a muscular dystrophy model of Caenorhabditis elegans and its functional response to drugs. <em>Dis. Model Mech.</em><strong>11</strong>, 12 (2018).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/dmm.036137" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6307913/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30396907/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Hewitt,%20J.%20E.%20et%20al.%20Muscle%20strength%20deficiency%20and%20mitochondrial%20dysfunction%20in%20a%20muscular%20dystrophy%20model%20of%20Caenorhabditis%20elegans%20and%20its%20functional%20response%20to%20drugs.%20Dis.%20Model%20Mech.11,%2012%20(2018)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Mercer, K. B. et al. Caenorhabditis elegans UNC-96 is a new component of M-lines that interacts with UNC-98 and paramyosin and is required in adult muscle for assembly and/or maintenance of thick filaments. <em>Mol. Biol. Cell</em><strong>17</strong>, 3832–3847 (2006).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E06-02-0144" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1593161/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16790495/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Mercer,%20K.%20B.%20et%20al.%20Caenorhabditis%20elegans%20UNC-96%20is%20a%20new%20component%20of%20M-lines%20that%20interacts%20with%20UNC-98%20and%20paramyosin%20and%20is%20required%20in%20adult%20muscle%20for%20assembly%20and/or%20maintenance%20of%20thick%20filaments.%20Mol.%20Biol.%20Cell17,%203832%E2%80%933847%20(2006)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Mercer, K. B. et al. C. elegans UNC-98, a C2H2 Zn finger protein, is a novel partner of UNC-97/PINCH in muscle adhesion complexes. <em>Mol. Biol. Cell</em><strong>14</strong>, 2492–2507 (2003).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E02-10-0676" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC194897/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12808046/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Mercer,%20K.%20B.%20et%20al.%20C.%20elegans%20UNC-98,%20a%20C2H2%20Zn%20finger%20protein,%20is%20a%20novel%20partner%20of%20UNC-97/PINCH%20in%20muscle%20adhesion%20complexes.%20Mol.%20Biol.%20Cell14,%202492%E2%80%932507%20(2003)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Mitani, S. Genetic regulation of <em>mec-3</em> gene expression implicated in the specification of the mechanosensory neuron cell types in <em>Caenorhabditis elegans</em>. <em>Dev. Growth Diff.</em><strong>37</strong>, 551–557 (1995).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1440-169X.1995.t01-4-00010.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37281420/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Mitani,%20S.%20Genetic%20regulation%20of%20mec-3%20gene%20expression%20implicated%20in%20the%20specification%20of%20the%20mechanosensory%20neuron%20cell%20types%20in%20Caenorhabditis%20elegans.%20Dev.%20Growth%20Diff.37,%20551%E2%80%93557%20(1995)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Hwang, H., Krajniak, J., Matsunaga, Y., Benian, G. M. &amp; Lu, H. On-demand optical immobilization of C. elegans for high-resolution imaging and microinjection. <em>Lab Chip</em><strong>14</strong>, 3498–3501 (2014).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1039/c4lc00697f" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4148454/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25056343/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Hwang,%20H.,%20Krajniak,%20J.,%20Matsunaga,%20Y.,%20Benian,%20G.%20M.%20&amp;%20Lu,%20H.%20On-demand%20optical%20immobilization%20of%20C.%20elegans%20for%20high-resolution%20imaging%20and%20microinjection.%20Lab%20Chip14,%203498%E2%80%933501%20(2014)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Ghazal, N., Peoples, J. N., Mohiuddin, T. A. &amp; Kwong, J. Q. Mitochondrial functional resilience after TFAM ablation in the adult heart. <em>Am. J. Physiol. Cell Physiol.</em><strong>320</strong>, C929–C942 (2021).
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1152/ajpcell.00508.2020" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8285632/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33760663/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Ghazal,%20N.,%20Peoples,%20J.%20N.,%20Mohiuddin,%20T.%20A.%20&amp;%20Kwong,%20J.%20Q.%20Mitochondrial%20functional%20resilience%20after%20TFAM%20ablation%20in%20the%20adult%20heart.%20Am.%20J.%20Physiol.%20Cell%20Physiol.320,%20C929%E2%80%93C942%20(2021)." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11487192/bin/42003_2024_7042_MOESM1_ESM.pdf">Peer Review File</a><sup> (3.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11487192/bin/42003_2024_7042_MOESM2_ESM.pdf">Supplementary Information</a><sup> (32.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11487192/bin/42003_2024_7042_MOESM3_ESM.pdf">42003_2024_7042_MOESM3_ESM.pdf</a><sup> (87.3KB, pdf) </sup><p>Description of Additional Supplementary File</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11487192/bin/42003_2024_7042_MOESM4_ESM.xlsx">Supplementary Data</a><sup> (67.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11487192/bin/42003_2024_7042_MOESM5_ESM.pdf">Reporting Summary</a><sup> (3.3MB, pdf) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The authors declare that the data supporting the results of this study are displayed in the Figures and the Supplementary Figs. The numerical source data for the graphs presented in the figures are available in the Supplementary Data.</p></section></section></section></section></article>
</main></body></html>