<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10793275/"/><title>Draft genome sequence of Sphingomonas paucimobilis strain Sph5, isolated from tap water filtration membrane - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Microbiol Resour Announc</button></div>. 2023 Dec 1;13(1):e00345-23. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/MRA.00345-23" rel="noopener noreferrer" target="_blank">10.1128/MRA.00345-23</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Draft genome sequence of <em>Sphingomonas paucimobilis</em> strain Sph5, isolated from tap water filtration membrane</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koroli%20S%22%5BAuthor%5D"><span class="name western">Sara Koroli</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sara Koroli</span></h3>
<div class="p">
<sup>1</sup>
Biodesign Center for Fundamental and Applied Microbiomics, Biodesign Institute, Arizona State University, Tempe, Arizona, USA
</div>
<div class="p">
<sup>2</sup>
College of Medicine, University of Arizona, Phoenix, Arizona, USA
</div>
<div>Data curation, Formal analysis, Investigation, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koroli%20S%22%5BAuthor%5D"><span class="name western">Sara Koroli</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Buss%20K%22%5BAuthor%5D"><span class="name western">Kristina Buss</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Kristina Buss</span></h3>
<div class="p">
<sup>3</sup>
Bioinformatics Core Facility, Biosciences, Knowledge Enterprise, Arizona State University, Tempe, Arizona, USA
</div>
<div>Data curation, Formal analysis, Software, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Buss%20K%22%5BAuthor%5D"><span class="name western">Kristina Buss</span></a>
</div>
</div>
<sup>3,</sup><sup>#</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blain%20JM%22%5BAuthor%5D"><span class="name western">Joy M Blain</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Joy M Blain</span></h3>
<div class="p">
<sup>4</sup>
Genomics Core Facility, Biosciences, Knowledge Enterprise, Arizona State University, Tempe, Arizona, USA
</div>
<div>Methodology, Resources, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blain%20JM%22%5BAuthor%5D"><span class="name western">Joy M Blain</span></a>
</div>
</div>
<sup>4</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakka%20GS%22%5BAuthor%5D"><span class="name western">Gautam Sai Nakka</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Gautam Sai Nakka</span></h3>
<div class="p">
<sup>1</sup>
Biodesign Center for Fundamental and Applied Microbiomics, Biodesign Institute, Arizona State University, Tempe, Arizona, USA
</div>
<div>Data curation, Writing – original draft</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakka%20GS%22%5BAuthor%5D"><span class="name western">Gautam Sai Nakka</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hong%20M%22%5BAuthor%5D"><span class="name western">Mina Hong</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Mina Hong</span></h3>
<div class="p">
<sup>1</sup>
Biodesign Center for Fundamental and Applied Microbiomics, Biodesign Institute, Arizona State University, Tempe, Arizona, USA
</div>
<div>Data curation, Writing – review and editing</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hong%20M%22%5BAuthor%5D"><span class="name western">Mina Hong</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McLean%20RJC%22%5BAuthor%5D"><span class="name western">Robert JC McLean</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Robert JC McLean</span></h3>
<div class="p">
<sup>5</sup>
Department of Biology, Texas State University, San Marcos, Texas, USA
</div>
<div>Funding acquisition, Investigation, Resources, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McLean%20RJC%22%5BAuthor%5D"><span class="name western">Robert JC McLean</span></a>
</div>
</div>
<sup>5</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plugge%20CM%22%5BAuthor%5D"><span class="name western">Caroline M Plugge</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Caroline M Plugge</span></h3>
<div class="p">
<sup>6</sup>
Wetsus, European Centre of Excellence for Sustainable Water Technology, Leeuwarden, the Netherlands
</div>
<div>Resources, Writing – review and editing</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plugge%20CM%22%5BAuthor%5D"><span class="name western">Caroline M Plugge</span></a>
</div>
</div>
<sup>6</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20J%22%5BAuthor%5D"><span class="name western">Jiseon Yang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jiseon Yang</span></h3>
<div class="p">
<sup>1</sup>
Biodesign Center for Fundamental and Applied Microbiomics, Biodesign Institute, Arizona State University, Tempe, Arizona, USA
</div>
<div>Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Supervision, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20J%22%5BAuthor%5D"><span class="name western">Jiseon Yang</span></a>
</div>
</div>
<sup>1,</sup><sup>✉,</sup><sup>#</sup>
</div>
<div class="cg p">Editor: <span class="name western">Julia A Maresca</span><sup>7</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>
Biodesign Center for Fundamental and Applied Microbiomics, Biodesign Institute, Arizona State University, Tempe, Arizona, USA
</div>
<div id="aff2">
<sup>2</sup>
College of Medicine, University of Arizona, Phoenix, Arizona, USA
</div>
<div id="aff3">
<sup>3</sup>
Bioinformatics Core Facility, Biosciences, Knowledge Enterprise, Arizona State University, Tempe, Arizona, USA
</div>
<div id="aff4">
<sup>4</sup>
Genomics Core Facility, Biosciences, Knowledge Enterprise, Arizona State University, Tempe, Arizona, USA
</div>
<div id="aff5">
<sup>5</sup>
Department of Biology, Texas State University, San Marcos, Texas, USA
</div>
<div id="aff6">
<sup>6</sup>
Wetsus, European Centre of Excellence for Sustainable Water Technology, Leeuwarden, the Netherlands
</div>
<div id="aff7">
<sup>7</sup>
University of Delaware College of Engineering, Newark, Delaware, USA
</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>✉</sup><p class="display-inline">Address correspondence to Jiseon Yang, <span>jyang41@asu.edu</span>
</p>
</div>
<div class="fn" id="fn1">
<sup>#</sup><p class="display-inline">Sara Koroli, Kristina Buss, and Jiseon Yang contributed equally to this article. Sara Koroli led the sample preparation, experiments, and initial data collection. Kristina Buss was primarily responsible for the subsequent bioinformatics data collection and analysis. Both authors contributed equally to the analysis and interpretation of results and to the manuscript preparation.</p>
</div>
<div class="fn" id="fn2"><p>The authors declare no conflict of interest.</p></div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Sara Koroli</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Kristina Buss</span></strong>: <span class="role">Data curation, Formal analysis, Software, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Joy M Blain</span></strong>: <span class="role">Methodology, Resources, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Gautam Sai Nakka</span></strong>: <span class="role">Data curation, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mina Hong</span></strong>: <span class="role">Data curation, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Robert JC McLean</span></strong>: <span class="role">Funding acquisition, Investigation, Resources, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Caroline M Plugge</span></strong>: <span class="role">Resources, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jiseon Yang</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Supervision, Writing – original draft, Writing – review and editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Julia A Maresca</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2023 Apr 24; Accepted 2023 Oct 24; Collection date 2024 Jan.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2023 Koroli et al.</div>
<p>This is an open-access article distributed under the terms of the <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">Creative Commons Attribution 4.0 International license</a>.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC10793275  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/38038463/">38038463</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Sphingomonadaceae are common membrane colonizers and biofilm formers. As part of our studies on long-term genetic changes in drinking water biofilm species, we report the draft genome sequence of <em>Sphingomonas</em> strain Sph5, isolated from a tap water filtration membrane. The isolate was determined as <em>Sphingomonas paucimobilis</em> through whole genome sequencing and <em>de novo</em> assembly.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>KEYWORDS:</strong> draft genome, WGS, water, <em>Sphingomonas</em></p></section></section><section id="s1"><h2 class="pmc_sec_title">ANNOUNCEMENT</h2>
<p>Widely used for drinking water, membrane filtration systems face biofouling due to microbe accumulation and biofilm formation. Studies reveal Sphingomonadaceae as common initial colonizers, persisting dominantly during biofilm growth (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>
<a aria-describedby="B2" class="usa-link" href="#B2">–</a>
<a aria-describedby="B4" class="usa-link" href="#B4">4</a>). Here, we report the genome sequence of a prevalent isolate.</p>
<p>
<em>Sphingomonas</em> spp. Sph5 was isolated from a biofilm on a Nadir MP005 microfiltration membrane used for drinking water biofouling studies (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>, <a aria-describedby="B5" class="usa-link" href="#B5">5</a>). Sph5 was previously reported to be phylogenetically closest to <em>Sphingomonas sanguinis</em> strains BAB-7166 (99%) and NBRC 13937 (99%) based on 16S rRNA sequencing and cultivation methods (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>). However, the whole genome sequence (WGS) was not reported. WGS analysis is critical to uncover strain-specific traits associated with membrane biofouling. Here, we report the WGS of <em>Sphingomonas</em> spp. Sph5 using Illumina short-read sequencing and <em>de novo</em> assembly methods.</p>
<p>We received the <em>Sphingomonas</em> spp. Sph5 water isolate from Wetsus, Netherlands. Single isolated colonies were obtained on Reasoner’s 2A agar (R2A, Teknova R0005; Difco 214530) and cultured in R2A broth at room temperature. Genomic DNA was extracted using DNeasy UltraClean microbial kit (Qiagen 12224) following the company-provided protocol.</p>
<p>Sequencing libraries were generated using Kapa’s Hyperplus kit (KK8514) and IDT adapters (#00989130v2). Quality was verified using an Agilent Tapestation and qPCR (NEBNext Library Quant Kit, E7630L) on Thermo Fisher’s Quantstudio5.</p>
<p>Nineteen million 2 × 150 bp reads were obtained on a NovaSeq (Illumina) at Anschutz Medical Campus Genomics and Microarray Core, with an average Phred score of 33–36 calculated by FastQC [v0.11.9 (<a aria-describedby="B6" class="usa-link" href="#B6">6</a>)].</p>
<p>Assembly was performed with SPADES, default isolate settings [v3.15.2 (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>)]. Contigs were aligned to NCBI databases with DIAMOND [v2.0.13 (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>)]. Completeness was estimated using BUSCO [v5.2.2 (<a aria-describedby="B9" class="usa-link" href="#B9">9</a>)] proteobacteria marker genes. Of the 219 genes, there were 214 (97.7%) complete and single copy, 2 (0.9%) duplicated, 1 (0.5%) fragmented, and 2 (0.9%) missing.</p>
<p>For species identification, average nucleotide identity (ANI) was calculated between all complete <em>Sphingomonas</em> genomes and Sph5 with PYANI [v0.2.11 (<a aria-describedby="B10" class="usa-link" href="#B10">10</a>)] and MUMMER [v4.0.0 (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>)]. ANI scores between Sph5 and <em>S. paucimobilis</em> ranged from 99.77%–99.95%, while ANI between Sph5 and <em>S. sanguinis</em> was 87.74%.</p>
<p>Annotations were predicted with Prokka [v1.14.6 (<a aria-describedby="B12" class="usa-link" href="#B12">12</a>)], given <em>S. sanguinis</em> and <em>S. paucimobilis</em> references (<a class="usa-link" href="#T1">Table 1</a>), and used for orthogroup-based phylogeny construction with Orthofinder [v2.5.4 (<a aria-describedby="B13" class="usa-link" href="#B13">13</a>)], MAFFT [v7.490 (<a aria-describedby="B14" class="usa-link" href="#B14">14</a>)], and FastTree [v.2.1.10 (<a aria-describedby="B15" class="usa-link" href="#B15">15</a>)].</p>
<section class="tw xbox font-sm" id="T1"><h3 class="obj_head">TABLE 1.</h3>
<div class="caption p"><p>Summary of the draft whole genome sequences of <em>Sphingomonas paucimobilis</em> Sph5 from the tap water filtration membrane</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1"/>
<col span="1"/>
</colgroup>
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="top">Variable</th>
<th align="left" colspan="1" rowspan="1" valign="top">Data</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Genus and species</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>Sphingomonas paucimobilis</em>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Strain</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>Sph5</em>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">NCBI accession no.</td>
<td align="left" colspan="1" rowspan="1" valign="top"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JAMRJL000000000" rel="noopener noreferrer" target="_blank">JAMRJL000000000</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Country</td>
<td align="left" colspan="1" rowspan="1" valign="top">The Netherlands</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Source</td>
<td align="left" colspan="1" rowspan="1" valign="top">Biofilm on Nadir MP005 microfiltration membrane</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Feed water source</td>
<td align="left" colspan="1" rowspan="1" valign="top">Tap water from the city of Leeuwarden</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Genome size (bp)</td>
<td align="left" colspan="1" rowspan="1" valign="top">4680620</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">N<sub>50</sub> (bp)</td>
<td align="left" colspan="1" rowspan="1" valign="top">4016723</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Scaffold reference</td>
<td align="left" colspan="1" rowspan="1" valign="top">ASM1602843v1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Scaffolds ≥50,000 bp</td>
<td align="left" colspan="1" rowspan="1" valign="top">29, 4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Mean coverage</td>
<td align="left" colspan="1" rowspan="1" valign="top">1,089×</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">G + C content (%)</td>
<td align="left" colspan="1" rowspan="1" valign="top">65.30</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">No. of annotated CDS</td>
<td align="left" colspan="1" rowspan="1" valign="top">3,342</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">No. of raw reads</td>
<td align="left" colspan="1" rowspan="1" valign="top">19,032,602</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">No. of reads used for assembly</td>
<td align="left" colspan="1" rowspan="1" valign="top">19,032,602</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">No. of coding sequences</td>
<td align="left" colspan="1" rowspan="1" valign="top">4,321</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></section><p>The WGS phylogeny (<a class="usa-link" href="#F1">Fig. 1</a>) confirms that Sph5 is closest to <em>S. paucimobilis</em> while related to <em>S. sanguinis</em>. As a result, <em>S. paucimobilis</em> reference genomes were used for RagTag [v2.1.0 (<a aria-describedby="B16" class="usa-link" href="#B16">16</a>)] scaffolding of Sph5 contigs. ASM1602843v1 was the best scaffold (93.6% total length in three longest contigs of resulting assembly). Realignment of raw reads to ASM1602843v1-scaffolded Sph5 assembly with BWA [v0.7.17 (<a aria-describedby="B17" class="usa-link" href="#B17">17</a>)], SamTools [v1.12 (<a aria-describedby="B18" class="usa-link" href="#B18">18</a>)], and Picard [v2.25.0 (<a aria-describedby="B19" class="usa-link" href="#B19">19</a>)], evaluated by Mosdepth [v0.3.3 (<a aria-describedby="B20" class="usa-link" href="#B20">20</a>)], supports this (99.48% reads aligned, 1,089× coverage).</p>
<figure class="fig xbox font-sm" id="F1"><h3 class="obj_head">Fig 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10793275_mra.00345-23.f001.jpg" target="_blank"><img alt="Fig 1" class="graphic zoom-in" height="706" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6371/10793275/466a37a2a170/mra.00345-23.f001.jpg" width="731"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Orthofinder-generated rooted tree of 50 complete <em>Sphingomonas</em> genomes with the Sph5 strain. Branch lengths represent evolutionary distance. We used the multiple-sequence alignment option with MAFFT and used FastTree to infer the trees. The mathematical parameters used are built in to the OrthoFinder tool, and we did not alter them from the default.</p></figcaption></figure><p>Re-annotation with Prokka predicts 4,321 CDS, 3 rRNAs, 54 tRNAs, 1 tmRNA, and 1 repeat region. Of the CDS, 3,342 could be functionally annotated.</p></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This research was funded by NASA Space Biology, grant number 80NSSC19K1597, and NASA BAC, grant number HS5HWWK1AAU5.</p></section><section class="contrib-info" id="_ci93_" lang="en"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Jiseon Yang, Email: jyang41@asu.edu.</p>
<p>Julia A. Maresca, 
University of Delaware College of Engineering, Newark, Delaware, USA
</p></section><section id="s2-1"><h2 class="pmc_sec_title">DATA AVAILABILITY</h2>
<p>The assembly and raw data are deposited at NCBI through the SRA genome submission portal under the accession numbers <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JAMRJL000000000.1/" rel="noopener noreferrer" target="_blank">JAMRJL000000000</a> and <a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/?view=run_browser&amp;acc=SRR25337554&amp;display=metadata" rel="noopener noreferrer" target="_blank">SRR25337554</a>. The version deposited in this paper is the first version.</p></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>
de Vries HJ, Beyer F, Jarzembowska M, Lipińska J, van den Brink P, Zwijnenburg A, Timmers PHA, Stams AJM, Plugge CM. 2019. Isolation and characterization of Sphingomonadaceae from fouled membranes. NPJ Biofilms Microbiomes
5:6. doi: 10.1038/s41522-018-0074-1

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41522-018-0074-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6347639/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30701078/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Biofilms%20Microbiomes&amp;title=Isolation%20and%20characterization%20of%20Sphingomonadaceae%20from%20fouled%20membranes&amp;volume=5&amp;publication_year=2019&amp;pages=6&amp;pmid=30701078&amp;doi=10.1038/s41522-018-0074-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>
Koskinen R, Ali-Vehmas T, Kämpfer P, Laurikkala M, Tsitko I, Kostyal E, Atroshi F, Salkinoja-Salonen M. 2000. Characterization of Sphingomonas isolates from Finnish and Swedish drinking water distribution systems. J Appl Microbiol
89:687–696. doi: 10.1046/j.1365-2672.2000.01167.x

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1365-2672.2000.01167.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11054174/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Microbiol&amp;title=Characterization%20of%20Sphingomonas%20isolates%20from%20Finnish%20and%20Swedish%20drinking%20water%20distribution%20systems&amp;volume=89&amp;publication_year=2000&amp;pages=687-696&amp;pmid=11054174&amp;doi=10.1046/j.1365-2672.2000.01167.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>
Bereschenko LA, Stams AJM, Euverink GJW, van Loosdrecht MCM. 2010. Biofilm formation on reverse osmosis membranes is initiated and dominated by Sphingomonas spp. Appl Environ Microbiol
76:2623–2632. doi: 10.1128/AEM.01998-09

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.01998-09" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2849218/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20190090/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Biofilm%20formation%20on%20reverse%20osmosis%20membranes%20is%20initiated%20and%20dominated%20by%20Sphingomonas%20spp&amp;volume=76&amp;publication_year=2010&amp;pages=2623-2632&amp;pmid=20190090&amp;doi=10.1128/AEM.01998-09&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>
Gauthier V, Redercher S, Block JC. 1999. Chlorine inactivation of Sphingomonas cells attached to goethite particles in drinking water. Appl Environ Microbiol
65:355–357. doi: 10.1128/AEM.65.1.355-357.1999

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.65.1.355-357.1999" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC91032/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9872809/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Chlorine%20inactivation%20of%20Sphingomonas%20cells%20attached%20to%20goethite%20particles%20in%20drinking%20water&amp;volume=65&amp;publication_year=1999&amp;pages=355-357&amp;pmid=9872809&amp;doi=10.1128/AEM.65.1.355-357.1999&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>
Dreszer C, Wexler AD, Drusová S, Overdijk T, Zwijnenburg A, Flemming H-C, Kruithof JC, Vrouwenvelder JS. 2014. In-situ Biofilm characterization in membrane systems using optical coherence tomography: formation, structure, detachment and impact of flux change. Water Res
67:243–254. doi: 10.1016/j.watres.2014.09.006

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.watres.2014.09.006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25282092/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Water%20Res&amp;title=In-situ%20Biofilm%20characterization%20in%20membrane%20systems%20using%20optical%20coherence%20tomography:%20formation,%20structure,%20detachment%20and%20impact%20of%20flux%20change&amp;volume=67&amp;publication_year=2014&amp;pages=243-254&amp;pmid=25282092&amp;doi=10.1016/j.watres.2014.09.006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>
Andrews S. 2010. FastQC: a quality control tool for high throughput sequence data
</cite>
</li>
<li id="B7">
<span class="label">7.</span><cite>
Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 2012. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol
19:455–477. doi: 10.1089/cmb.2012.0021

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/cmb.2012.0021" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3342519/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22506599/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&amp;title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&amp;volume=19&amp;publication_year=2012&amp;pages=455-477&amp;pmid=22506599&amp;doi=10.1089/cmb.2012.0021&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>
Buchfink B, Xie C, Huson DH. 2015. Fast and sensitive protein alignment using DIAMOND. Nat Methods
12:59–60. doi: 10.1038/nmeth.3176

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.3176" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25402007/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Fast%20and%20sensitive%20protein%20alignment%20using%20DIAMOND&amp;volume=12&amp;publication_year=2015&amp;pages=59-60&amp;pmid=25402007&amp;doi=10.1038/nmeth.3176&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>
Simão FA, Waterhouse RM, Ioannidis P, Kriventseva EV, Zdobnov EM. 2015. BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs. Bioinformatics
31:3210–3212. doi: 10.1093/bioinformatics/btv351

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btv351" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26059717/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BUSCO:%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;volume=31&amp;publication_year=2015&amp;pages=3210-3212&amp;pmid=26059717&amp;doi=10.1093/bioinformatics/btv351&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>
Pritchard L, Glover RH, Humphris S, Elphinstone JG, Toth IK. 2016. Genomics and taxonomy in diagnostics for food security: soft-rotting enterobacterial plant pathogens. Anal Methods
8:12–24. doi: 10.1039/C5AY02550H
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1039/C5AY02550H" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Anal%20Methods&amp;title=Genomics%20and%20taxonomy%20in%20diagnostics%20for%20food%20security:%20soft-rotting%20enterobacterial%20plant%20pathogens&amp;volume=8&amp;publication_year=2016&amp;pages=12-24&amp;doi=10.1039/C5AY02550H&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>
Marçais G, Delcher AL, Phillippy AM, Coston R, Salzberg SL, Zimin A. 2018. MUMmer4: a fast and versatile genome alignment system. PLoS Comput Biol
14:e1005944. doi: 10.1371/journal.pcbi.1005944

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pcbi.1005944" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5802927/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29373581/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=MUMmer4:%20a%20fast%20and%20versatile%20genome%20alignment%20system&amp;volume=14&amp;publication_year=2018&amp;pages=e1005944&amp;pmid=29373581&amp;doi=10.1371/journal.pcbi.1005944&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>
Seemann T. 2014. Prokka: rapid prokaryotic genome annotation. Bioinformatics
30:2068–2069. doi: 10.1093/bioinformatics/btu153

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btu153" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24642063/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Prokka:%20rapid%20prokaryotic%20genome%20annotation&amp;volume=30&amp;publication_year=2014&amp;pages=2068-2069&amp;pmid=24642063&amp;doi=10.1093/bioinformatics/btu153&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>
Emms DM, Kelly S. 2019. Orthofinder: phylogenetic orthology inference for comparative genomics. Genome Biol
20:238. doi: 10.1186/s13059-019-1832-y

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s13059-019-1832-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6857279/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31727128/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Orthofinder:%20phylogenetic%20orthology%20inference%20for%20comparative%20genomics&amp;volume=20&amp;publication_year=2019&amp;pages=238&amp;pmid=31727128&amp;doi=10.1186/s13059-019-1832-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14">
<span class="label">14.</span><cite>
Katoh K, Standley DM. 2013. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol
30:772–780. doi: 10.1093/molbev/mst010

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/mst010" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3603318/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23329690/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=MAFFT%20multiple%20sequence%20alignment%20software%20version%207:%20improvements%20in%20performance%20and%20usability&amp;volume=30&amp;publication_year=2013&amp;pages=772-780&amp;pmid=23329690&amp;doi=10.1093/molbev/mst010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15">
<span class="label">15.</span><cite>
Price MN, Dehal PS, Arkin AP. 2009. FastTree: computing large minimum evolution trees with profiles instead of a distance matrix. Mol Biol Evol
26:1641–1650. doi: 10.1093/molbev/msp077

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/msp077" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2693737/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19377059/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=FastTree:%20computing%20large%20minimum%20evolution%20trees%20with%20profiles%20instead%20of%20a%20distance%20matrix&amp;volume=26&amp;publication_year=2009&amp;pages=1641-1650&amp;pmid=19377059&amp;doi=10.1093/molbev/msp077&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16">
<span class="label">16.</span><cite>
Alonge M, Soyk S, Ramakrishnan S, Wang X, Goodwin S, Sedlazeck FJ, Lippman ZB, Schatz MC. 2019. RaGOO: fast and accurate reference-guided scaffolding of draft genomes. Genome Biol
20:224. doi: 10.1186/s13059-019-1829-6

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s13059-019-1829-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6816165/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31661016/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=RaGOO:%20fast%20and%20accurate%20reference-guided%20scaffolding%20of%20draft%20genomes&amp;volume=20&amp;publication_year=2019&amp;pages=224&amp;pmid=31661016&amp;doi=10.1186/s13059-019-1829-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17">
<span class="label">17.</span><cite>
Li H, Durbin R. 2009. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics
25:1754–1760. doi: 10.1093/bioinformatics/btp324

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btp324" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2705234/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19451168/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;volume=25&amp;publication_year=2009&amp;pages=1754-1760&amp;pmid=19451168&amp;doi=10.1093/bioinformatics/btp324&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18">
<span class="label">18.</span><cite>
Danecek P, Bonfield JK, Liddle J, Marshall J, Ohan V, Pollard MO, Whitwham A, Keane T, McCarthy SA, Davies RM, Li H. 2021. Twelve years of SAMtools and BCFtools. Gigascience
10:giab008. doi: 10.1093/gigascience/giab008

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/gigascience/giab008" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7931819/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33590861/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Twelve%20years%20of%20SAMtools%20and%20BCFtools&amp;volume=10&amp;publication_year=2021&amp;pages=giab008&amp;pmid=33590861&amp;doi=10.1093/gigascience/giab008&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19">
<span class="label">19.</span><cite>
“Picard Toolkit” [Internet] . 2019. Broad Institute, Github repository. Available from: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://broadinstitute.github.io/picard" rel="noopener noreferrer" target="_blank">http://broadinstitute.github.io/picard</a>
</cite>
</li>
<li id="B20">
<span class="label">20.</span><cite>
Pedersen BS, Quinlan AR. 2018. Mosdepth: quick coverage calculation for genomes and exomes. Bioinformatics
34:867–868. doi: 10.1093/bioinformatics/btx699

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btx699" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6030888/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29096012/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Mosdepth:%20quick%20coverage%20calculation%20for%20genomes%20and%20exomes&amp;volume=34&amp;publication_year=2018&amp;pages=867-868&amp;pmid=29096012&amp;doi=10.1093/bioinformatics/btx699&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The assembly and raw data are deposited at NCBI through the SRA genome submission portal under the accession numbers <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JAMRJL000000000.1/" rel="noopener noreferrer" target="_blank">JAMRJL000000000</a> and <a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/?view=run_browser&amp;acc=SRR25337554&amp;display=metadata" rel="noopener noreferrer" target="_blank">SRR25337554</a>. The version deposited in this paper is the first version.</p></section></section></section></section></article>
</main></body></html>