Data Brief

. 2018 Mar 26;18:913–919. doi: [10.1016/j.dib.2018.03.107](https://doi.org/10.1016/j.dib.2018.03.107)

# Phenotypic characterization of an Arabidopsis T-DNA insertion line SALK\_063500

[Natasha J Sng](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sng%20NJ%22%5BAuthor%5D)

### Natasha J Sng

aPlant Molecular and Cellular Biology, University of Florida, Fifield Hall, 2550 Hull Road, Gainesville, FL 32611, USA

Find articles by [Natasha J Sng](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sng%20NJ%22%5BAuthor%5D)

a, [Anna-Lisa Paul](https://pubmed.ncbi.nlm.nih.gov/?term=%22Paul%20AL%22%5BAuthor%5D)

### Anna-Lisa Paul

aPlant Molecular and Cellular Biology, University of Florida, Fifield Hall, 2550 Hull Road, Gainesville, FL 32611, USA

bHorticultural Science Department, University of Florida, Fifield Hall, 2550 Hull Road, Gainesville, FL 32611, USA

Find articles by [Anna-Lisa Paul](https://pubmed.ncbi.nlm.nih.gov/?term=%22Paul%20AL%22%5BAuthor%5D)

a,b, [Robert J Ferl](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferl%20RJ%22%5BAuthor%5D)

### Robert J Ferl

aPlant Molecular and Cellular Biology, University of Florida, Fifield Hall, 2550 Hull Road, Gainesville, FL 32611, USA

bHorticultural Science Department, University of Florida, Fifield Hall, 2550 Hull Road, Gainesville, FL 32611, USA

cInterdisciplinary Center for Biotechnology Research (ICBR), University of Florida, 2033 Mowry Road, Gainesville, FL 32601, USA

Find articles by [Robert J Ferl](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferl%20RJ%22%5BAuthor%5D)

a,b,c,⁎

* Author information
* Article notes
* Copyright and License information

aPlant Molecular and Cellular Biology, University of Florida, Fifield Hall, 2550 Hull Road, Gainesville, FL 32611, USA

bHorticultural Science Department, University of Florida, Fifield Hall, 2550 Hull Road, Gainesville, FL 32611, USA

cInterdisciplinary Center for Biotechnology Research (ICBR), University of Florida, 2033 Mowry Road, Gainesville, FL 32601, USA

⁎

Corresponding author. robferl@ufl.edu

Received 2017 Oct 12; Accepted 2018 Mar 21; Collection date 2018 Jun.

© 2018 The Authors

This is an open access article under the CC BY license (http://creativecommons.org/licenses/by/4.0/).

[PMC Copyright notice](/about/copyright/)

PMCID: PMC5996828  PMID: [29900258](https://pubmed.ncbi.nlm.nih.gov/29900258/)

## Abstract

In this article we report the identification of a homozygous lethal T-DNA (transfer DNA) line within the coding region of the At1G05290 gene in the genome *of Arabidopsis thaliana* (Arabidopsis) line, SALK\_063500. The T-DNA insertion is found within exon one of the AT1G05290 gene, however a homozygous T-DNA allele is unattainable. In the heterozygous T-DNA allele the expression levels of AT1G05290 were compared to wild type Arabidopsis (Col-0, Columbia). Further analyses revealed an aberrant silique phenotype found in the heterozygous SALK\_063500 plants that is attributed to the reduced rate of pollen tube germination. These data are original and have not been published elsewhere.

**Keywords:** Arabidopsis, T-DNA, SALK\_063500, Silique, Pollen, Phenotype, AT1G05290

---

**Specifications Table**

|  |  |
| --- | --- |
| Subject area | *Biology* |
| More specific subject area | *Plant biology* |
| Type of data | *Tables, Graphs, Figures* |
| How data was acquired | *DNA-PCR, Quantitative Realtime PCR (qPCR), Plant phenotypes, Pollen tube germination assay, Image J analyses* |
| Data format | *Raw, Analyzed* |
| Experimental factors | *Col-0 (Columbia) and SALK\_063500 Arabidopsis plants* |
| Experimental features | *DNA-PCR was employed to identify the T-DNA insertion in SALK\_063500. AT1G05290 expression levels were examined with qPCR. Both silique and pollen tube germination phenotypes were recorded.* |
| Data source location | *Gainesville, Florida, USA* |
| Data accessibility | *Data is within this article.* |

[Open in a new tab](table/t0010/)

**Value of the data**

* •

  T-DNA insertion lines provide an important resource for genetic analyses in plant research, and SALK lines are the most commonly used T-DNA insertion lines. Therefore assessments of phenotypes observed in SALK lines are valuable assets for advancing our understanding of basic plant biology.
* •

  Documentation of the phenotype of the SALK\_063500 line will make the plant community aware of the role AT1G05290 plays in pollen development, thereby furthering research in this field.
* •

  The data presented could provide insights into understanding the molecular mechanisms of male sterility in plants.

## 1. Data

The data presented here provide information on the phenotype of a T-DNA insertion line, SALK\_063500. [Fig. 1](#f0005)A-B shows the T-DNA insertion within AT1G05290 and the wild type and heterozygous T-DNA alleles amplified. No homozygous T-DNA alleles were found after screening over a thousand SALK\_063500 progeny. [Fig. 2](#f0010) shows the level of AT1G05290 expression in seedlings under normal growth conditions. [Fig. 3](#f0015), [Fig. 4](#f0020) show the silique phenotype observed in both Col-0 and heterozygous SALK\_063500 plants. [Fig. 5](#f0025), [Fig. 6](#f0030) show the pollen tube germination rates *in vitro* and *in vivo* of Col-0 and heterozygous SALK\_063500 plants. [Table 1](#t0005) provides the sequences of the primers used in [Fig. 1](#f0005), [Fig. 2](#f0010).

### Fig. 1.

[![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c405/5996828/ef266da9d27b/gr1.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5996828_gr1.jpg)

[Open in a new tab](figure/f0005/)

T-DNA insertion in the exon1 of AT1G05290 in SALK\_063500. (A) Gene structure of AT1G05290. The T-DNA position is +127 bp after the transcription start site. Primers used to screen the SALK\_063500 line, indicated by black arrows RP, LBa1, and LP, and were derived from the SALK T\_DNA primer design web tool (<http://signal.salk.edu/tdnaprimers.2.html>). (B) PCR amplification of wild type allele band using forward (RP), reverse (LP) and T-DNA band (LBa1). 110 randomly selected SALK\_063500 seedlings and two Col-0 wild type (WT) seedlings were used in search for a homozygous T-DNA insertion line, none were identified. Homozygous wild type allele are seen as single bands at 991 bp whereas heterozygous individuals have double bands, with a single wild type allele band at 991 bp and a T-DNA allele band at around 440–740 bp. All primer sequences are listed in [Table 1](#t0005).

### Fig. 2.

[![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c405/5996828/da3982e28cb1/gr2.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5996828_gr2.jpg)

[Open in a new tab](figure/f0010/)

Relative expression levels of AT1G05290. The expression level of AT1G05290 in seedlings of Col-0 was set at “1” so that the relative expression level of SALK\_063500 can be compared to Col-0. UBQ11 (AT4G05050) used as an internal control. The Ct (cycle threshold) values are shown in [Supplementary table 1](#s0020). Data represent means of standard error (SE) (*n* = 6). The student's *t*-test was performed to show the significant difference of gene expression between Col-0 and SALK\_063500 (\**p*< 0.05).

### Fig. 3.

[![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c405/5996828/b0f02d02aa25/gr3.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5996828_gr3.jpg)

[Open in a new tab](figure/f0015/)

Phenotype of heterozygous SALK\_063500 siliques. (A) Phenotype of wild type Col-0 and heterozygous SALK\_063500 silique development. Scale bar = 5 mm. (B) Quantification of silique length and seed fill in Col-0 and heterozygous SALK\_063500. The numbers of siliques used are shown in [Supplementary table 2](#s0020). Data presented are means±standard deviation (SD) (*n* = 30). The student's *t*-test was performed to show the significant difference of silique length and seed fill between Col-0 and heterozygous SALK\_063500 (\**p*< 0.05, \*\**p*< 0.01).

### Fig. 4.

[![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c405/5996828/612ca590705a/gr4.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5996828_gr4.jpg)

[Open in a new tab](figure/f0020/)

Phenotype of siliques from reciprocal cross between Col-0 and heterozygous SALK\_063500 lines. (A) Phenotype of silique development in wild type Col-0, heterozygous SALK\_063500, and their respective reciprocal crosses. Scale bar=5 mm. (B) The number of siliques used are shown in [Supplementary table 3](#s0020). Data presented are means±SD (*n* = 10). Both an ANOVA and the student's *t*-test was performed to show the significant difference of silique length and seed fill between all four lines ([Supplementary table 3](#s0020)). Bar graphs with different letters show significant difference (*p* < 0.05).

### Fig. 5.

[![Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c405/5996828/86c7bb1639df/gr5.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5996828_gr5.jpg)

[Open in a new tab](figure/f0025/)

Pollen tube germination analysis *in vitro*. (A) *in vitro* pollen tube germination analysis of Col-0 and heterozygous SALK\_063500 pollen grown overnight at room temperature on pollen germinating media. (B) Quantification of *in vitro* pollen tube germination of Col-0 and heterozygous SALK\_063500 pollen. The numbers of pollen counted are shown in [Supplementary table 4](#s0020). Data presented are means±SD (*n* = 300). The student's *t*-test was performed to show the significant difference of *in vitro* pollen germination rates between Col-0 and heterozygous SALK\_063500 (\*\**p* < 0.01).

### Fig. 6.

[![Fig. 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c405/5996828/d0bdb847bcc6/gr6.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5996828_gr6.jpg)

[Open in a new tab](figure/f0030/)

Pollen tube germination analysis *in vivo.* (A) Aniline Blue stain of Col-0, heterozygous SALK\_063500 and, reciprocally crossed pollen tubes 6 h after pollination. Brightfield exposure was set at 3.59 ms and GFPA at 624 ms exposure. Scale bar = 500 um. (B) Quantification of the Corrected Total Cell Fluorescence (CTCF). Calculations for CTCF are shown in [Supplementary table 5](#s0020). Data presented are means±SD (*n* = 4). Both an ANOVA and the student's *t*-test was performed to show the significant difference of CTCF between all four lines. Bar graphs with different letters show significant difference (*p*< 0.05).

### Table 1.

Primers used in the data presented.

| *Gene* | *Name* | *Sequence (5’-3’)* | *Usage* |
| --- | --- | --- | --- |
| AT1G05290 | AT1G05290-LP | CGATCTTTCGGACTAAATCCC | T-DNA PCR |
|  | AT1G05290-RP | AGACCGCTCTTGTAGAATCCC |  |
|  | AT1G05290-F | CTCACGGAGCTAATCGATGAATT | qPCR |
|  | AT1G05290-R | CGATGACATACCAGATGCTGAAG |  |
| N/A | LBa1 | TGGTTCACGTAGTGGGCCATCG | T-DNA PCR |
| UBQ11 (AT4G05050) | UBQ11-F | AGCAACTTGAGGACGGCAGA | qPCR |
|  | UBQ11-R | GTGATGGTCTTTCCGGTCAAA |  |

[Open in a new tab](table/t0005/)

## 2. Experimental design, materials and methods

Arabidopsis seeds of wild type Columbia-0 (Col-0) and SALK\_063500 were obtained from the Arabidopsis Biological Resource Center [[1]](#bib1). Plants were grown in the growth chambers on soil at 22 °C under constant light at approaximately 80 μmol m−2 s−1. In the search for a homozygous T-DNA insertion line from SALK\_063500, over a thousand progeny of the heterozygous SALK\_063500 line were screened and 110 are presented in [Fig. 1](#f0005)B. AT1G05290-LP and AT1G05290-RP primers were used to amplify the wild type allele band (991 bp) while LBa1 and AT1G05290-RP primers were used to amplify the T-DNA band (440–740 bp). No homozygous T-DNA allele lines were found. For gene expression analysis, Col-0 and SALK\_063500 seeds were grown on 0.5% PhytagelTM plates (0.5× Murashige and Skoog (MS) salts [Invitrogen] containing 0.5% (w/v) sucrose, was adjusted to pH 5.72 with 1N KOH and solidified with 0.5% (w/v) PhytagelTM [Sigma]) at 22 °C under constant light condition [[2]](#bib2). The 12 day old plants were harvested from Phytagel plates and directly preserved in RNALater [Ambion] until ready for RNA extraction. Total RNA was extracted using the RNAeasy kits [Qiagen] according to the manufacturer's instructions. CDNA was transcribed from total RNA, using the High Capacity RNA to cDNA kit [Applied Biosystems] and RT-qPCR analysis was conducted with SYBR Green master mix [Applied Biosystems]. The 2−ΔΔCt method was used to calculate the relative expression of At1g05290 using *UBQ11* (AT4G0505) as an internal control. A total of 6 replicates were used for each sample. For silique phenotypes, 30 siliques from three individuals representing the genotype were collected, imaged under the Olympus stereo dissecting microscope (SZH10) and measured using image J [[3]](#bib3), [[4]](#bib4). For *in vitro* pollen germination studies, mature pollen was harvested from flowers left at room temperate for 2 h, and then grown overnight on slides coated with solid pollen germinating media (18% Sucrose, 0.01% Boric Acid, 1 mM CaCl2, 1 mM Ca(NO3)2, 1 mM MgSO4 and 0.5% Noble agar [Difco] was adjusted to pH 7.0 [[5]](#bib5). Pollen germination studies done *in vivo* were performed by collecting pollinated pistils 6 h after pollination and briefly fixed in ethanol:acetic acid (3:1) for 2 h at room temperature. Fixed pistils were then washed with distilled water three times and further treated in 8 M NaOH overnight. Pistil tissues were then washed in distilled water and stained with aniline blue solution (0.1% aniline blue in 0.1 M K2HPO4-KOH buffer, pH 11) for 5 h in the dark before being viewed under the Olympus BX51 compound light microscope [[6]](#bib6).

## Acknowledgements

Authors would like to thank the UF Space Plants lab. This work was supported by the NASA Space Life and Physical Sciences (SLPS) grants NNX09AL96G, NNX07AH270, NNX12AN69G and NNX14AT24G awarded to RJF and A-LP.

## Footnotes

Transparency document

Transparency data associated with this article can be found in the online version at [h10.1016/j.dib.2018.03.107](http://dx.doi.org/10.1016/j.dib.2018.03.107).

Appendix A

Supplementary data associated with this article can be found in the online version at [10.1016/j.dib.2018.03.107](http://dx.doi.org/10.1016/j.dib.2018.03.107).

## Transparency document. Supplementary material

Supplementary material

[mmc1.docx](/articles/instance/5996828/bin/mmc1.docx) (10KB, docx)

.

## Appendix A. Supplementary material

Supplementary material

[mmc2.xlsx](/articles/instance/5996828/bin/mmc2.xlsx) (14.9KB, xlsx)

.

Supplementary material

[mmc3.xlsx](/articles/instance/5996828/bin/mmc3.xlsx) (16.8KB, xlsx)

.

Supplementary material

[mmc4.xlsx](/articles/instance/5996828/bin/mmc4.xlsx) (19.3KB, xlsx)

.

Supplementary material

[mmc5.xlsx](/articles/instance/5996828/bin/mmc5.xlsx) (14.8KB, xlsx)

.

Supplementary material

[mmc6.xlsx](/articles/instance/5996828/bin/mmc6.xlsx) (20KB, xlsx)

.

## References

* 1.Lamesch P., Berardini T.Z., Li D., Swarbreck D., Wilks C., Sasidharan R., Muller R., Dreher K., Alexander D.L., Garcia-Hernandez M. The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools. Nucleic Acids Res. 2012;40:D1202. doi: 10.1093/nar/gkr1090. [[DOI](https://doi.org/10.1093/nar/gkr1090)] [[PMC free article](/articles/PMC3245047/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22140109/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=The%20Arabidopsis%20Information%20Resource%20(TAIR):%20improved%20gene%20annotation%20and%20new%20tools&author=P.%20Lamesch&author=T.Z.%20Berardini&author=D.%20Li&author=D.%20Swarbreck&author=C.%20Wilks&volume=40&publication_year=2012&pages=D1202&pmid=22140109&doi=10.1093/nar/gkr1090&)]
* 2.Sng N., Callaham J., Ferl R.J., Paul A.-L. Arabidopsis thaliana for spaceflight applications –preparing dormant biology for passive stowage and on orbit activation. Gravit. Space Res. 2014;2:2. [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gravit.%20Space%20Res.&title=Arabidopsis%20thaliana%20for%20spaceflight%20applications%20%E2%80%93preparing%20dormant%20biology%20for%20passive%20stowage%20and%20on%20orbit%20activation&author=N.%20Sng&author=J.%20Callaham&author=R.J.%20Ferl&author=A.-L.%20Paul&volume=2&publication_year=2014&pages=2&)]
* 3.Paul A.-L., Daugherty C.J., Bihn E.A., Chapman D.K., Norwood K.L., Ferl R.J. Transgene expression patterns indicate that spaceflight affects stress signal perception and transduction in arabidopsis. Plant Physiol. 2001;126:613. doi: 10.1104/pp.126.2.613. [[DOI](https://doi.org/10.1104/pp.126.2.613)] [[PMC free article](/articles/PMC111153/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11402191/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&title=Transgene%20expression%20patterns%20indicate%20that%20spaceflight%20affects%20stress%20signal%20perception%20and%20transduction%20in%20arabidopsis&author=A.-L.%20Paul&author=C.J.%20Daugherty&author=E.A.%20Bihn&author=D.K.%20Chapman&author=K.L.%20Norwood&volume=126&publication_year=2001&pages=613&pmid=11402191&doi=10.1104/pp.126.2.613&)]
* 4.Schneider C.A., Rasband W.S., Eliceiri K.W. NIH Image to ImageJ: 25 years of image analysis. Nat. methods. 2012;9:671. doi: 10.1038/nmeth.2089. [[DOI](https://doi.org/10.1038/nmeth.2089)] [[PMC free article](/articles/PMC5554542/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22930834/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20methods&title=NIH%20Image%20to%20ImageJ:%2025%20years%20of%20image%20analysis&author=C.A.%20Schneider&author=W.S.%20Rasband&author=K.W.%20Eliceiri&volume=9&publication_year=2012&pages=671&pmid=22930834&doi=10.1038/nmeth.2089&)]
* 5.Li H., Lin Y., Heath R.M., Zhu M.X., Yang Z. Control of pollen tube tip growth by a Rop GTPase-dependent pathway that leads to tip-localized calcium influx. Plant Cell. 1999;11:1731. doi: 10.1105/tpc.11.9.1731. [[DOI](https://doi.org/10.1105/tpc.11.9.1731)] [[PMC free article](/articles/PMC144310/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10488239/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=Control%20of%20pollen%20tube%20tip%20growth%20by%20a%20Rop%20GTPase-dependent%20pathway%20that%20leads%20to%20tip-localized%20calcium%20influx&author=H.%20Li&author=Y.%20Lin&author=R.M.%20Heath&author=M.X.%20Zhu&author=Z.%20Yang&volume=11&publication_year=1999&pages=1731&pmid=10488239&doi=10.1105/tpc.11.9.1731&)]
* 6.Sumie I., Akiko K., Junichi U., Ikuo N., Kiyotaka O. The DEFECTIVE IN ANTHER DEHISCENCE1 gene encodes a novel phospholipase A1 catalyzing the initial step of jasmonic acid biosynthesis, which synchronizes pollen maturation, anther dehiscence, and flower opening in Arabidopsis. Plant Cell. 2001;10:2191. doi: 10.1105/tpc.010192. [[DOI](https://doi.org/10.1105/tpc.010192)] [[PMC free article](/articles/PMC139153/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11595796/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=The%20DEFECTIVE%20IN%20ANTHER%20DEHISCENCE1%20gene%20encodes%20a%20novel%20phospholipase%20A1%20catalyzing%20the%20initial%20step%20of%20jasmonic%20acid%20biosynthesis,%20which%20synchronizes%20pollen%20maturation,%20anther%20dehiscence,%20and%20flower%20opening%20in%20Arabidopsis&author=I.%20Sumie&author=K.%20Akiko&author=U.%20Junichi&author=N.%20Ikuo&author=O.%20Kiyotaka&volume=10&publication_year=2001&pages=2191&pmid=11595796&doi=10.1105/tpc.010192&)]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

Supplementary material

[mmc1.docx](/articles/instance/5996828/bin/mmc1.docx) (10KB, docx)

Supplementary material

[mmc2.xlsx](/articles/instance/5996828/bin/mmc2.xlsx) (14.9KB, xlsx)

Supplementary material

[mmc3.xlsx](/articles/instance/5996828/bin/mmc3.xlsx) (16.8KB, xlsx)

Supplementary material

[mmc4.xlsx](/articles/instance/5996828/bin/mmc4.xlsx) (19.3KB, xlsx)

Supplementary material

[mmc5.xlsx](/articles/instance/5996828/bin/mmc5.xlsx) (14.8KB, xlsx)

Supplementary material

[mmc6.xlsx](/articles/instance/5996828/bin/mmc6.xlsx) (20KB, xlsx)