J Fungi (Basel)

. 2022 Feb 8;8(2):165. doi: [10.3390/jof8020165](https://doi.org/10.3390/jof8020165)

# Genomic Characterization of the Titan-like Cell Producing *Naganishia tulchinskyi*, the First Novel Eukaryote Isolated from the International Space Station

[Swati Bijlani](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bijlani%20S%22%5BAuthor%5D)

### Swati Bijlani

1Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, Los Angeles, CA 90089, USA; bijlani.swati@gmail.com (S.B.); clayw@usc.edu (C.C.C.W.)

Find articles by [Swati Bijlani](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bijlani%20S%22%5BAuthor%5D)

1,†,‡, [Ceth Parker](https://pubmed.ncbi.nlm.nih.gov/?term=%22Parker%20C%22%5BAuthor%5D)

### Ceth Parker

2Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA 91109, USA; Ceth.w.parker@jpl.nasa.gov (C.P.); Nitin.K.Singh@jpl.nasa.gov (N.K.S.)

Find articles by [Ceth Parker](https://pubmed.ncbi.nlm.nih.gov/?term=%22Parker%20C%22%5BAuthor%5D)

2,†, [Nitin K Singh](https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D)

### Nitin K Singh

2Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA 91109, USA; Ceth.w.parker@jpl.nasa.gov (C.P.); Nitin.K.Singh@jpl.nasa.gov (N.K.S.)

Find articles by [Nitin K Singh](https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D)

2, [Maria A Sierra](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sierra%20MA%22%5BAuthor%5D)

### Maria A Sierra

3Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY 10021, USA; mas4037@med.cornell.edu

4Tri-Institutional Computational Biology & Medicine Program, Weill Cornell Medicine, New York, NY 10021, USA; jof3004@med.cornell.edu

Find articles by [Maria A Sierra](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sierra%20MA%22%5BAuthor%5D)

3,4, [Jonathan Foox](https://pubmed.ncbi.nlm.nih.gov/?term=%22Foox%20J%22%5BAuthor%5D)

### Jonathan Foox

4Tri-Institutional Computational Biology & Medicine Program, Weill Cornell Medicine, New York, NY 10021, USA; jof3004@med.cornell.edu

5The WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, NY 10021, USA

Find articles by [Jonathan Foox](https://pubmed.ncbi.nlm.nih.gov/?term=%22Foox%20J%22%5BAuthor%5D)

4,5, [Clay C C Wang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CCC%22%5BAuthor%5D)

### Clay C C Wang

1Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, Los Angeles, CA 90089, USA; bijlani.swati@gmail.com (S.B.); clayw@usc.edu (C.C.C.W.)

Find articles by [Clay C C Wang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CCC%22%5BAuthor%5D)

1, [Christopher E Mason](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D)

### Christopher E Mason

4Tri-Institutional Computational Biology & Medicine Program, Weill Cornell Medicine, New York, NY 10021, USA; jof3004@med.cornell.edu

5The WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, NY 10021, USA

Find articles by [Christopher E Mason](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D)

4,5,\*, [Kasthuri Venkateswaran](https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D)

### Kasthuri Venkateswaran

2Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA 91109, USA; Ceth.w.parker@jpl.nasa.gov (C.P.); Nitin.K.Singh@jpl.nasa.gov (N.K.S.)

Find articles by [Kasthuri Venkateswaran](https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D)

2,\*

Editors: Ricardo Franco-Duarte, Celia Pais

* Author information
* Article notes
* Copyright and License information

1Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, Los Angeles, CA 90089, USA; bijlani.swati@gmail.com (S.B.); clayw@usc.edu (C.C.C.W.)

2Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA 91109, USA; Ceth.w.parker@jpl.nasa.gov (C.P.); Nitin.K.Singh@jpl.nasa.gov (N.K.S.)

3Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY 10021, USA; mas4037@med.cornell.edu

4Tri-Institutional Computational Biology & Medicine Program, Weill Cornell Medicine, New York, NY 10021, USA; jof3004@med.cornell.edu

5The WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, NY 10021, USA

\*

Correspondence: chm2042@med.cornell.edu (C.E.M.); kjvenkat@jpl.nasa.gov (K.V.); Tel.: +1-(203)-668-1448 (C.E.M.); +1-(818)-393-1481 (K.V.); Fax: +1-(646)-962-00383 (C.E.M.); +1-(818)-393-4176 (K.V.)

†

These authors contributed equally to this work.

‡

Present Address: Department of Surgery, City of Hope National Medical Center, Duarte, CA 91010, USA.

#### Roles

**Ricardo Franco-Duarte**: Academic Editor

**Celia Pais**: Academic Editor

Received 2021 Dec 24; Accepted 2022 Feb 4; Collection date 2022 Feb.

© 2022 by the authors.

Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<https://creativecommons.org/licenses/by/4.0/>).

[PMC Copyright notice](/about/copyright/)

PMCID: PMC8875396  PMID: [35205919](https://pubmed.ncbi.nlm.nih.gov/35205919/)

## Abstract

Multiple strains of a novel yeast belonging to genus *Naganishia* were isolated from environmental surfaces aboard the International Space Station (ISS). These strains exhibited a phenotype similar to Titan cell (~10 µm diameter) morphology when grown under a combination of simulated microgravity and 5% CO2 conditions. Confocal, scanning, and transmission electron microscopy revealed distinct morphological differences between the microgravity-grown cells and the standard Earth gravity-grown cells, including larger cells and thicker cell walls, altered intracellular morphology, modifications to extracellular fimbriae, budding, and the shedding of bud scars. Phylogenetic analyses via multi-locus sequence typing indicated that these ISS strains represented a single species in the genus *Naganishia* and were clustered with *Naganishia diffluens*. The name *Naganishia tulchinskyi* is proposed to accommodate these strains, with IF6SW-B1T as the holotype. The gene ontologies were assigned to the cell morphogenesis, microtubule-based response, and response to UV light, suggesting a variety of phenotypes that are well suited to respond to microgravity and radiation. Genomic analyses also indicated that the extracellular region, outer membrane, and cell wall were among the highest cellular component results, thus implying a set of genes associated with Titan-like cell plasticity. Finally, the highest molecular function matches included cytoskeletal motor activity, microtubule motor activity, and nuclear export signal receptor activity.

**Keywords:** *Naganisha tulchinskyi*, yeast, International Space Station, phylogenetic analyses, Titan-like cells, simulated microgravity

## 1. Introduction

The genus *Naganishia* [[1](#B1-jof-08-00165)] was resurrected from *Cryptococcus* and emended to accommodate the *albidus* clade of *Filobasidiales*, Tremellomycetes, and Basidiomycota [[2](#B2-jof-08-00165),[3](#B3-jof-08-00165),[4](#B4-jof-08-00165)]. The genus *Naganishia* contains the following species as legitimate in MycoBank: *N. adeliensis, N. albida, N. albidosimilis, N. antarctica, N. bhutanensis, N. brisbanensis, N. cerealis, N. diffluens, N. friedmannii, N. globosa, N. indica, N. liquefaciens, N. nivalis, N. onofrii, N. qatarensis, N. randhawae, N. uzbekistanensis, N. vaughanmartiniae*, and *N. vishniacii* [[2](#B2-jof-08-00165),[3](#B3-jof-08-00165),[4](#B4-jof-08-00165)]. However, the addition of *N. floricola* made the list a total of 20 *Naganishia* species, but several of these members were yet to be validly described. In an ongoing Microbial Tracking experiment on the International Space Station (ISS), six yeast strains belonging to the family *Tremellomycetes* were isolated [[5](#B5-jof-08-00165)], and whole-genome sequences (WGS) were generated [[6](#B6-jof-08-00165)]. Some of the *Naganishia* species that were phylogenetically related to these ISS strains, which were reported to be psychrophilic or psychrotolerant species, isolated from Antarctica [[4](#B4-jof-08-00165),[7](#B7-jof-08-00165),[8](#B8-jof-08-00165),[9](#B9-jof-08-00165),[10](#B10-jof-08-00165),[11](#B11-jof-08-00165)], also reported that the ISS *Naganishia* species might also display properties related to cold adaptation and an increase in radiation resistance. In addition, *Naganishia* species were also isolated from a variety of ecosystems, such as water [[12](#B12-jof-08-00165)], soil [[13](#B13-jof-08-00165)], tree trunk [[14](#B14-jof-08-00165)], air [[12](#B12-jof-08-00165)], Alpine glaciers [[15](#B15-jof-08-00165)], cheese, cereals [[16](#B16-jof-08-00165)], and human specimens [[12](#B12-jof-08-00165)].

Members of the *Cryptococcus* species, from which *Naganishia* species have recently been separated, were reported to possess unique virulence factors associated with the pathogenesis [[17](#B17-jof-08-00165),[18](#B18-jof-08-00165)]. The morphological transformation of normal *Cryptococcus* cells into giant (Titan) cells was one of the factors contributing to cryptococcosis (an inhalation acquired infection) [[19](#B19-jof-08-00165),[20](#B20-jof-08-00165),[21](#B21-jof-08-00165)]. It was also reported that titanization was unique to the *Cryptococcus neoformans/C. gattii* species complex, and not found in other budding basidiomycetous yeasts or in the ascomycetous yeast *Saccharomyces cerevisiae*, when grown under similar growth conditions [[21](#B21-jof-08-00165)]. However, occasional enlarged cells have been reported in *Cryptococcus terreus*. The ISS *Naganishia* strains exhibited Titan-like cells, a phenotype similar to the *C. neoformans/C. gattii* species complex, when the ISS isolates were grown under simulated microgravity conditions.

One of the objectives of this study was to characterize the Titan-like cells of the novel *Naganishia* species isolated from the ISS environment. The microscopy analyses revealed the presence of Titan-like giant cells with abnormal morphology when *Naganishia* species were subjected to several stressors, including a combination of hypoxic 5% CO2 and simulated ISS microgravity conditions. The second objective of this study was to utilize WGS for annotation in order to aid in the identification of genetic determinants related to cold adaptation and radiation resistance in *N. tulchinskyi*. The third objective was to determine the phylogenetic novelty of the ISS *Naganishia* strains (*n* = 4) using microscopy and taxonomic affiliation based on multi-locus sequence typing (MLST) analyses, which included a seven gene concatenated (ITS, LSU, SSU, *CYTB*, *RPB1*, *RPB2*, *TEF1*) phylogeny [[2](#B2-jof-08-00165)].

## 2. Materials and Methods

### 2.1. Sample Collection and Isolation of Microorganisms

During the flight experiments of Microbial Tracking-1, several surface samples (1 m2) were collected from various ISS environmental locations. Previous publications had detailed sample collection, processing, and isolation of cultivable microorganisms from the ISS [[5](#B5-jof-08-00165)]. Briefly, internal ISS surface samples were collected using polyester wipes and then transported to Earth for processing. Samples were collected just prior to departure (within 5 days) and stored at 4 °C in the ISS, transported in cold packs (~4 °C) from ISS to the Earth, hand delivered at Long Beach, California after splash down, which were then carried in ice packs (~4 °C) during ground transportation, before storing at JPL at 4 °C for further processing. Total number of days for sample storage time from sample collection to processing was ~7 days. Unless otherwise stated, all reagents were purchased from Sigma Aldrich (St. Louis, MO, USA). The materials associated with the sampling wipes were placed in 200 mL of sterile phosphate-buffered saline (PBS; pH 7.4) solution and plated onto potato dextrose agar (PDA), supplemented with chloramphenicol (100 µg/mL; *v*/*v*), to isolate fungi [[5](#B5-jof-08-00165),[22](#B22-jof-08-00165)]. Microbial colonies grown at 25 °C for 7 days were picked from the media plates, purified, and stored for further analyses. Based on internal transcribed spacer (ITS) sequencing, six yeast strains were tentatively assigned to the genus *Cryptococcus*, and subsequent WGS characterization revealed that two strains were identified as *Papiliotrema laurentii*, while four strains belonged to a yet to be described *Naganishia* species [[6](#B6-jof-08-00165)], making this species the first known novel eukaryote to be isolated from the ISS [[6](#B6-jof-08-00165)].

### 2.2. Whole Genome Sequencing Analyses

The WGS used for the analyses were previously generated and published [[6](#B6-jof-08-00165)]. The WGS raw data was filtered with NGS QC Toolkit v2.3 [[23](#B23-jof-08-00165)] for high-quality (HQ) vector and adaptor-free reads for genome assembly (cutoff read length for HQ, 80%; cutoff quality score, 20). The number of filtered reads obtained were used for assembly with SPAdes 3.14.0 [[24](#B24-jof-08-00165)] genome assembler (k-mer size—32 to 72 bases) using default parameters. Genomic DNA sequences and predicted proteomes (where available) of all other strains used in this study were downloaded from NCBI, followed by comparative analyses.

### 2.3. Phylogenetic Analyses

Among the 20 known *Naganishia* species, NCBI had ITS sequences available for all the species except *N. indica*. The gene sequences of all available strains were retrieved from NCBI, except for the four ISS strains, which were derived from their WGS during this study. Phylogenetic analyses based on individual genes (ITS, LSU, SSU, *CYTB*, *RPB1*, *RPB2*, *TEF1*) and MLST analyses (with the seven genes) were carried out with all available *Naganishia* species (*n* = 10). The individual gene sequences for all strains were first aligned separately using ClustalW, and for MLST, seven gene sequences for each strain were concatenated manually, aligned using ClusatalW, and used to generate a Maximum Likelihood Tree using MEGA 7.0.26 [[25](#B25-jof-08-00165)].

### 2.4. Morphological Characterization under Simulated Microgravity Conditions

#### 2.4.1. Growth Conditions for Phenotypic Analyses under Different Conditions

*Naganishia* sp. IF6SW-B1T was streaked on a Yeast extract- Peptone- Dextrose agar (1% yeast extract, 2% peptone, 2% glucose, and 1.5% agar) plate and incubated at 30 °C for 48 h. After 48 h of growth, colonies were resuspended in PBS, and the cells were enumerated using a hemocytometer. The cells were then inoculated at an initial density of 2 × 105 cells/mL in synthetic dextrose (SD) broth. Replicates (*n* = 3) of each yeast strain were grown under each condition tested. The cells were inoculated in three conditions; (a) the first set was inoculated in 5 mL of liquid media in a tube, incubated at 30 °C, and shaken at 150 rpm; (b) the second set was inoculated in 5 mL liquid media and incubated on a rotating platform in a CO2 incubator (5% CO2) at 30 °C; (c) the final set was grown in 10 mL of liquid media in a high aspect ratio vessels (HARV, Synthecon Inc., Houston, TX, USA) and mounted onto a vertical rotating device (at 30 rpm) to simulate microgravity conditions, these were placed in a CO2 incubator (5% CO2) at 30 °C. The CO2 level of ISS habitat is ~10× more (~4000 ppm) than at Earth conditions [[5](#B5-jof-08-00165)]. Additionally, Titan cells were observed under 5% CO2 level in *C. neoformans* [[26](#B26-jof-08-00165)]. Hence, morphological changes were characterized at this 5% CO2 level. The autoclavable HARVs provided oxygenation via a flat, silicone rubber gas transfer membrane (Synthecon Inc.). The cultures were all incubated as described for 4 days, followed by cell pelleting for further analyses.

The biochemical and physiological characterizations were characterized by employing BioLog Yeast plates, as described elsewhere [[27](#B27-jof-08-00165)]. Briefly, 4-days grown colonies in PDA plates were collected in sterile distilled water aseptically, and ~106 cells were transferred into the BioLog Yeast plates (BioLog Inc., Hayward, CA, USA) and incubated at 25 °C for 7 days. The change in coloration and oxidation/assimilation profiles of 94 different carbon sources were documented by visual examination. The clear positive and negatives were scored in 7 days incubation, and delayed reaction (weak) was documented by examining the tetrazolium violet color change after 14 days incubation. Further incubation and observation after 30 days period of time did not yield any difference in results compared to those that were observed at day 7.

#### 2.4.2. Confocal Microscopy

Yeast cell samples grown under the three conditions described above were fixed in 4% paraformaldehyde (PFA) in PBS for 15 min at 4 °C. These cells were gently pelleted by centrifugation for 5 min at 3500 rpm, and cells were rinsed 3× with sterile PBS. Subsequent processing steps were carried out in the dark, where samples were covered with aluminum foil and protected from the light. Cell pellets were resuspended in 15 µM Calcofluor White and incubated for 1 h at 37 °C. Samples were then rinsed and pelleted as previously mentioned but with distilled H2O. After the third rinse, samples were pelleted again, and the pellet was resuspended with 1 µM TO-PRO®-3 (Fisher Scientific, Los Angeles, CA, USA) and incubated for 30 min at room temperature. Samples were further rinsed and pelleted, as previously mentioned, using distilled H2O. The resuspended samples were filtered through a 0.2 µM 25 mm Isopore Membrane Filter (GTBP) using a filter manifold. This membrane was removed, placed on a microscope slide, and a drop of antifade was added to the top. A coverslip was placed, and then, the edges were sealed. The samples were viewed using a Zeiss LSM 880 upright laser scanning confocal microscope and an Alpha Plan-APOCHROMAT 63× lens (Zeiss, Oberkochen, Germany). Yeast cell size and cell wall thickness were analyzed using ImageJ [[28](#B28-jof-08-00165)].

#### 2.4.3. Scanning Electron Microscopy

Yeast cells grown under various conditions were preserved using 4% paraformaldehyde (PFA) in PBS for scanning electron microscopy (SEM) and transmission electron microscopy (TEM). Following the concentration, cells were immersed in a chilled 4% PFA and incubated at 4 °C for 10 min before the PFA was removed and washed in PBS buffer. Fixed cells were transferred into a 24-well plate, dehydrated in an isopropyl alcohol (IPA) series (50% to 100%), and stored at 4 °C. Samples were critically point dried in an Automegasamdri 915B critical point dryer (Tousimis, Rockville, MD, USA). Samples were attached to SEM stubs with carbon tape (Ted Pella Inc., Redding, CA, USA), followed by carbon coating with a Leica EM ACE600 Carbon Evaporator (Leica, Wetzlar, Germany) to a thickness of ~12 nm. SEM analysis was performed with an FEI Quanta 200F (Thermo Fisher, Waltham, MA, USA).

#### 2.4.4. Transmission Electron Microscopy

TEM was performed to examine cell wall thickness variations and analyze organelles’ differences between the different environmental stressors (CO2 and simulated microgravity) induced Titan-like cells and standard cells, grown under Earth gravity, with 5% CO2 and without CO2. For TEM, yeast cells were fixed in 2% glutaraldehyde for at least one hour at 4 °C. Samples were rinsed in 0.1 M sodium cacodylate buffer, pH 7.2, and immersed in 1% osmium tetroxide with 1.6% potassium ferricyanide in 0.1 M sodium cacodylate buffer for 1 h. After rinsing, samples were polymerized at 60 °C for 24–48 h in silicon molds. Thick sections (200 nm) were cut, stained with toluidine blue, and used. Thin sections (70 nm) were collected onto formvar-coated 50 mesh copper grids. The grids were post-stained with 2% uranyl acetate, followed by Reynold’s lead citrate. The sections were imaged using a Tecnai 12 120 kV TEM (FEI, Hillsboro, OR, USA) and data were recorded using an UltraScan 1000 with Digital Micrograph 3 software (Gatan Inc., Pleasanton, CA, USA).

#### 2.4.5. Statistical Analyses

All statistical analyses were performed using GraphPad Prism version 9.1.0 (GraphPad Software, San Diego, CA, USA). Specifically, data was tested for normal distribution with the D’Agostino and Pearson normality test, followed by non-parametric ANOVA using a post-hoc Kruskal–Wallis analysis.

### 2.5. Survey of Genes and Proteins Involved in Resistance

Genes and proteins involved in cold resistance from the closely related species *N. vishniacii* [[29](#B29-jof-08-00165)] were screened using tBLASTn and BLASTp ([Table 1](#jof-08-00165-t001)). Cold tolerance included genes involved in carotenoid synthesis: isopentenyl diphosphate isomerase, farnesyltransferase, farnesyl pyrophosphatase synthetase, phytoene-beta carotene synthase, NADP-cytochrome P450 reductase; genes involved in mycosporine synthesis: 2-epi-5-epi-valiolone synthase, catechol o-methyltransferase, and carbamoylphosphate synthase reported in *Phaffia rhodozyma* [[30](#B30-jof-08-00165)]; enzymes involved in the formation and degradation of trehalose: trehalose-6-phosphate synthase component TPS1 and TPS2, alpha, alpha-trehalase NTH2, NTH1, and ATH1. Additionally, resistance related proteins for fungi were screened using sequences obtained from NCBI by using ‘microgravity’ and ‘radioresistance’ keywords. Microgravity-related proteins included: Daughter-specific expression protein 1 and 2, GPI-anchored protein 24, Chain A Flocculation protein FLO1, Chain A and B Flocculin, and Chain A proteinase K. Radioresistance proteins including sequences from the DNA repair protein rad9.

#### Table 1.

Presence of various biomolecules responsible for pigment biosynthesis, radio-resistance, and microgravity tolerance in *Naganishia* species and *S. cerevisiae*.

| Functions | GenBank# | Enzyme, Protein, Other Biomolecules | *Naganishia tulchinskyi* | | | | *Naganishia vishniacii* | *Papiliotrema laurentii* | | *S. cerevisiae* | Reference |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| IF6SW-B1T | 1F7SW-B1 | 1F1SW-F1 | IIF5SW-F1 | CBS 10616 | IF7SW-B5 | IF7SW-F4 | S288C |
| Pigment Biosynthesis | [AAY33922](https://www.ncbi.nlm.nih.gov/protein/AAY33922) | Geranylgeranyl diphosphate synthase | \* | \*\* | \*\* | \*\* | \*\* | \* | \*\* | \*\* | [[39](#B39-jof-08-00165)] |
| [ACI43098](https://www.ncbi.nlm.nih.gov/protein/ACI43098) | Cytochrome P450 reductase | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | [[40](#B40-jof-08-00165)] |
| [AHW57996](https://www.ncbi.nlm.nih.gov/protein/AHW57996) | Farnesyl pyrophosphate synthase | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | [[41](#B41-jof-08-00165)] |
| [BAA33979](https://www.ncbi.nlm.nih.gov/protein/BAA33979) | Isopentenyl-diphosphate delta isomerase | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | [[42](#B42-jof-08-00165)] |
| [NP\_009555](https://www.ncbi.nlm.nih.gov/protein/NP_009555) | Alpha, alpha-trehalase NTH2 | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | [[43](#B43-jof-08-00165)] |
| [NP\_010284](https://www.ncbi.nlm.nih.gov/protein/NP_010284) | Alpha, alpha-trehalase NTH1 | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | [[44](#B44-jof-08-00165)] |
| [NP\_015351](https://www.ncbi.nlm.nih.gov/protein/NP_015351) | Alpha, alpha-trehalase ATH1 |  |  |  |  |  |  |  | \*\* | [[43](#B43-jof-08-00165)] |
| [NP\_594975](https://www.ncbi.nlm.nih.gov/protein/NP_594975) | Putative trehalose-phosphate synthase Tps2 | \*\* | \*\* | \* | \*\* | \*\* | \*\* | \*\* | \*\* | [[45](#B45-jof-08-00165)] |
| [AAO47570](https://www.ncbi.nlm.nih.gov/protein/AAO47570) | Phytoene-beta carotene synthase | \*\* | \*\* | \*\* | \*\* | \* | \* | \* |  | [[46](#B46-jof-08-00165)] |
| Radio- resistance | RAD51 | Recombinase RAD51 | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | [[47](#B47-jof-08-00165)] |
| RAD54 | DNA repair and recombination protein RAD54-like protein | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | [[47](#B47-jof-08-00165)] |
| RDH54 | DNA-dependent ATPase RDH54 | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | [[47](#B47-jof-08-00165)] |
| [CAA54491](https://www.ncbi.nlm.nih.gov/protein/CAA54491) | RAD9 |  |  |  |  |  | \* | \* |  | [[48](#B48-jof-08-00165)] |
| [CAA54492](https://www.ncbi.nlm.nih.gov/protein/CAA54492) | RAD9 |  |  |  |  |  |  | \* |  | [[48](#B48-jof-08-00165)] |
| Microgravity tolerance | 1HT3\_A | Chain A, Proteinase K | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | [[49](#B49-jof-08-00165)] |
| 1IC6\_A | Chain A, Proteinase K | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | \*\* | [[50](#B50-jof-08-00165)] |
| 4LHK\_A | Chain A, Flocculin |  |  |  |  |  |  |  | \*\* | [[51](#B51-jof-08-00165)] |
| 4LHK\_B | Chain B, Flocculin |  |  |  |  |  |  |  | \*\* | [[51](#B51-jof-08-00165)] |
| 4LHN\_A | Chain A, Flocculation protein FLO1 |  |  |  |  |  |  |  | \*\* | [[51](#B51-jof-08-00165)] |
| [P26306](https://www.ncbi.nlm.nih.gov/protein/P26306) | DNA repair protein rad9 |  |  |  |  |  | \* | \* |  | [[52](#B52-jof-08-00165)] |
| [P38844](https://www.ncbi.nlm.nih.gov/protein/P38844) | Daughter-specific expression protein 2 |  |  |  |  |  |  |  | \*\* | [[53](#B53-jof-08-00165)] |
| [P40077](https://www.ncbi.nlm.nih.gov/protein/P40077) | Daughter-specific expression protein 1 |  |  |  |  |  |  |  | \*\* | [[54](#B54-jof-08-00165)] |
| [P48013](https://www.ncbi.nlm.nih.gov/protein/P48013) | DNA repair protein rad9 |  |  |  |  |  |  | \* |  | [[48](#B48-jof-08-00165)] |
| [Q59Y31](https://www.ncbi.nlm.nih.gov/protein/Q59Y31) | Flocculation protein 1 |  |  |  |  |  |  |  | \*\* | [[55](#B55-jof-08-00165)] |

[Open in a new tab](table/jof-08-00165-t001/)

\*\* Match > 75% of the protein sequence and e-value < 1 × 10−25; \* Match < 75% of the protein sequence or e-value > 1 × 10−25.

### 2.6. De Novo Genome Annotation

Gene annotation on each of the six assemblies (four *Naganishia tuchinskyi* strains and two closely related species) was performed using MAKER [[31](#B31-jof-08-00165)]. MAKER identified repeats, aligned expressed sequence tags (ESTs) and proteins to a genome, and produced ab initio gene predictions. A total of 74,830 available ESTs from closely related *Naganishia* species were downloaded from NCBI ([Table 2](#jof-08-00165-t002)) and protein sequences from six strains were downloaded from UniProt [[32](#B32-jof-08-00165)].

#### Table 2.

Datasets used for De Novo gene annotation using MAKER.

| Name | Origin \* | # of Sequences | Date of Access | Data Set |
| --- | --- | --- | --- | --- |
| *Naganishia tulchinskyi* (*n* = 4) | This study |  |  | Assembly |
| *Naganishia cerealis* | UniProt | 3 | 4/5/21 | Protein sequences |
| *Naganishia diffluens* | UniProt | 7 | 4/5/21 | Protein sequences |
| *Naganishia liquefaciens* | UniProt | 4 | 4/5/21 | Protein sequences |
| *Naganishia* sp. | UniProt | 3 | 4/5/21 | Protein sequences |
| *Cryptococcus neoformans var grubii* | UniProt | 65 | 4/5/21 | Protein sequences |
| *Cryptococcus neoformans var neoformans* | UniProt | 370 | 4/5/21 | Protein sequences |
| *Cryptococcus neoformans var grubii* | ESTdb | 69 | 3/31/21 | Transcriptome |
| *Cryptococcus neoformans var neoformans* | ESTdb | 74,724 | 3/31/21 | Transcriptome |
| *Cryptococcus vishniacci (Naganishia vishniacii)* | ESTdb | 37 | 3/31/21 | Transcriptome |
| *Papiliotrema laurentii* (*n* = 2) | This study |  |  | Assembly |

[Open in a new tab](table/jof-08-00165-t002/)

\* Expressed Sequence Tags database (ESTdb; Nature Genetics 4:332–3;1993) is a division of GenBank that contains sequence data and other information on “single-pass” cDNA sequences from a number of organisms. UniProt (2021) [[32](#B32-jof-08-00165)]. UniProt: the universal protein knowledgebase in 2021. Nucleic Acids Res 49, D480–d489.

Firstly, de novo repeat identification and masking were performed using RepeatModeler [[33](#B33-jof-08-00165)]. Then, MAKER control files were generated (maker-CTL), and a first-round was run using the maker\_opts.ctl control file. Data for the transcriptome assembly (EST), protein sequences (protein), and repeat annotations (rm\_gff) was provided. Additionally, the model\_org was set to ‘simple’ to annotate simple repeats, and inference predictions of genes directly from ESTs and protein homology were set (est2genome = 1, protein2genome = 1). Lastly, GFF and FASTA outputs were assembled together with GFF3toolkit [[34](#B34-jof-08-00165)].

Next, we predicted genes using the gene models generated with MAKER by training SNAP [[35](#B35-jof-08-00165)] and Augustus [[36](#B36-jof-08-00165)] using BUSCO [[37](#B37-jof-08-00165)]. For SNAP, we used models with an AED of 0.25 or better and a length of 50 or more amino acids. To train Augustus, we excised regions that contain mRNA annotations based on our initial MAKER run, with 1000 bp on each side, as described [[38](#B38-jof-08-00165)].

To run BUSCO, the Eukaryota set of conserved genes was used. The -m genome option was specified to provide regions that included more than just transcripts. The initial HMM model selected was *Cryptococcus*, which corresponded to the closest genus of *Naganishia* (augustus\_species = cryptococcus). The long option was set to optimize self-training Augustus.

Following the BUSCO run, the second round of MAKER was performed using the SNAP and Augustus models. For this purpose, FASTA sequence files were replaced with GFF files (est\_gff, protein\_gff, and rm\_gff), and est2genome and protein2genome were switched to 0. The third run of MAKER was performed, and the gene models were evaluated by counting the number of gene models and the gene lengths after each round and running BUSCO using the transcript sequences (-m transcriptome).

### 2.7. Functional Annotation

Predicted protein sequences generated by MAKER were analyzed using InterProScan v5.51–85.0 [[56](#B56-jof-08-00165)] with default parameters. Functional annotations assigned by InterProScan per protein sequence were converted into collections of Gene Ontology (GO) terms using BLAST2GO [[57](#B57-jof-08-00165)] with default parameters. All assigned GO terms per protein were aggregated and visualized using the REVIGO web tool [[58](#B58-jof-08-00165)] with the “tiny” similarity cutoff (0.4) searched against the *Saccharomyces cerevisiae* S288C database and using the SimRel measurement of similarity. Visualizations were generated using the code provided by REVIGO and using R3.6.3.

### 2.8. Data Availability

The WGS and raw data have been deposited in GenBank under the BioProject accession number PRJNA623412 [[6](#B6-jof-08-00165)]. The genome sequences have also been deposited in the National Aeronautics and Space Administration (NASA) GeneLab system (GLDS-290; <https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-290>; accessed on 5 December 2021). In addition to the WGS, we performed amplicon sequencing the ITS region from each strain [[5](#B5-jof-08-00165)], and their accession numbers are given below. *Naganishia* IF6SW-B1 ([KY218715.1](https://www.ncbi.nlm.nih.gov/nuccore/KY218715.1)); *Naganishia* IF1SW-F1 ([KY218664.1](https://www.ncbi.nlm.nih.gov/nuccore/KY218664.1)); *Naganishia* IF7SW-B1 ([KY218717.1](https://www.ncbi.nlm.nih.gov/nuccore/KY218717.1)); *Naganishia* IIF5SW-F1 ([KY218695.1](https://www.ncbi.nlm.nih.gov/nuccore/KY218695.1)).

## 3. Results and Discussion

Several yeast strains belonging to the *Naganishia* (*Cryptococcus*) *diffluens* clade of *Filobasidiales* were isolated from two different flight collection experiments conducted aboard the ISS [[5](#B5-jof-08-00165)]. These strains, referred to as IF6SW-B1T, IF7SW-B1, IF1SW-F1, and IIF5SW-F1, were isolated from four different ISS locations and identified to be novel species belonging to the genus *Naganishia,* based on genomic characterization via polyphasic taxonomic approaches. The sampling of ISS surfaces, performed for this study, took place within the US on-orbit segments. Sampling campaign (denoted as I) was carried out on 4 March 2015, and the second one (denoted as II) was conducted on 15 May 2015. Samples collected during this study were: Node 3 (Locations #1, #2, and #3), Node 1 (Locations #4 and #5), Permanent Multipurpose Module (Location #6), U.S. Laboratory (Location #7), and Node 2 (Locations #8 and control). A detailed description of the various locations sampled was published elsewhere [[59](#B59-jof-08-00165)]. Strains IF6SW-B1, IF7SW-B1, and IF1SW-F1, were isolated during first sampling from Locations 6, 7, and 1, respectively. Strain IIF5SW-F1 was isolated from second sampling at Location 5.

### 3.1. Phylogenetic Analyses of Novel ISS Strains

To confirm that these four ISS strains form a novel species, their phylogenetic affiliations were analyzed with other species belonging to the genus *Naganishia*. The MycoBank database documented 19 *Naganishia* species, and the CBS database showed only 11 *Naganishia* species. The gene sequences available on NCBI for *Naganishia* species (*n* = 10) were used to generate phylogenetic maximum likelihood trees with *C. neoformans* CBS 132 strain as an outgroup.

Phylogenetic analyses were carried out using seven genes: internal transcribed spacer (ITS) region rRNA gene, D1/D2 domain of large subunit (LSU or 26S) rRNA gene, small subunit (SSU or 18S) rRNA gene, and housekeeping genes, including two subunits of RNA polymerase II (*RPB1* and *RPB2*), translation elongation factor 1-α (*TEF1*), and cytochrome b (*CYTB*), which were used for differentiating *Tremellomycetes* species [[2](#B2-jof-08-00165)]. The respective gene sequences that were available on NCBI for different *Naganishia* species were included in the phylogenetic analyses. In addition, genomes of closely related members from *Tremellomycetes* (*Goffeauzyma gastricus* CBS 2288 ([AF145323.1](https://www.ncbi.nlm.nih.gov/nuccore/AF145323.1)), *Heterocephalacria arrabidensis* CBS 8678 ([AF444362.2](https://www.ncbi.nlm.nih.gov/nuccore/AF444362.2)), *Piskurozyma cylindrica* CBS 8680 ([AF444360.1](https://www.ncbi.nlm.nih.gov/nuccore/AF444360.1)), *Solicoccozyma aerius* CBS155 ([AF145324.1](https://www.ncbi.nlm.nih.gov/nuccore/AF145324.1)), and *Tremella mesenterica* CBS 6973 ([AF444433.1](https://www.ncbi.nlm.nih.gov/nuccore/AF444433.1))) were included with *Cryptococcus neoformans* CBS 132 ([AF444326.1](https://www.ncbi.nlm.nih.gov/nuccore/AF444326.1)) as the outgroup.

Initially, seven individual phylogenetic trees using ITS, LSU, SSU, *CTYB*, *RPB1*, *RPB2*, and *TEF1* genes were generated and are presented in [Supplemental Figures S1–S7](#app1-jof-08-00165). When tested individually, the phylogenetic analyses generated based on individual gene, formed a single group for all four ISS strains. There were no significant variations in ITS, LSU, and SSU gene sequences ([Figure S3](#app1-jof-08-00165)), whereas the other four genes (*CYTB*, *RPB1*, *RPB2*, and *TEF1*) invariably exhibited higher resolution in differentiating all the *Naganishia* species. The ITS tree ([Supplemental Figure S1](#app1-jof-08-00165)) showed that the ISS strains were placed at the same position of the three recognized species, *N. diffluens, N. liquefaciens* and *N. albidosimilis*. ITS sequences of the four ISS isolates showed 100% similarity to the sequence of *N. albidosimilis* CBS 7711T ([AF145325](https://www.ncbi.nlm.nih.gov/nuccore/AF145325)), *N. liquefaciens* CBS 968T ([AF444345](https://www.ncbi.nlm.nih.gov/nuccore/AF444345)), and *N. diffluens* CBS 160T ([AF145330](https://www.ncbi.nlm.nih.gov/nuccore/AF145330)). However, ITS sequences of *N. vishniacii* CBS 7110T shared 97.84% similarity (13 bp difference to GenBank accession no. [AF145320](https://www.ncbi.nlm.nih.gov/nuccore/AF145320).), *N. uzbekistanensis* CBS 8683T had 97.55% similarity (14 bp difference to GenBank accession no. [AF444339](https://www.ncbi.nlm.nih.gov/nuccore/AF444339)), and *N. adeliensis* CBS 8351T exhibited 97.34% similarity (14 bp difference to GenBank accession no. [AF145328](https://www.ncbi.nlm.nih.gov/nuccore/AF145328)) with ISS isolates ([Supplemental Figure S1](#app1-jof-08-00165)). In the LSU tree ([Supplemental Figure S2](#app1-jof-08-00165)), the four ISS strains were placed at the same position that phylogenetically distinguished from *N. albida, N. adeliensis, N. vishniacii*, and *N. albidosimilis* with low bootstrap support. Comparing the LSU sequences of these ISS strains with closely related species showed a 97.71% similarity to *N. albida* (type strain, CBS 142; 14 different base pairs; GenBank accession no. [AF075474](https://www.ncbi.nlm.nih.gov/nuccore/AF075474)), 96.41% similarity to *N. albidosimilis* (type strain, CBS 1926; 22 different base pairs; GenBank accession no. [AF137601](https://www.ncbi.nlm.nih.gov/nuccore/AF137601)), 96.25% similarly to *N. liquefaciens* (type strain, CBS 968; 23 different base pairs; GenBank accession no. [AF181515](https://www.ncbi.nlm.nih.gov/nuccore/AF181515)), and 95.92% similarity to *N. diffluens* (type strain, CBS 160; 25 different base pairs; GenBank accession no. [AF075502](https://www.ncbi.nlm.nih.gov/nuccore/AF075502)) ([Supplemental Figure S2](#app1-jof-08-00165)). The taxonomic threshold predicted to discriminate basidiomycetous yeast species is 99.51% in the LSU region and 99.21% in the ITS region [[1](#B1-jof-08-00165)]. Hence, the sequence data and phylogenetic analysis of the LSU regions of the ISS isolates confirm they belong to a new species in the genus *Naganishia* (order Filobasidiales). The SSU tree ([Supplemental Figure S3](#app1-jof-08-00165)) demonstrated that these four strains were not separated from any of the recognized species.

The other phylogenetic trees based on *CYTB*, *RPB1*, *RPB2*, and *TEF1* grouped the ISS strains together, and the closest neighbor was identified as *N. diffluens*. Subsequently, MLST was carried out by manually concatenating the seven genes ([Figure 1](#jof-08-00165-f001)). The MLST phylogenetic tree constructed showed the four ISS strains clustered together in the same clade with *N. diffluens* CBS 160T ([Figure 1](#jof-08-00165-f001)). The individual gene-based and MLST tree-based analyses both further supported the finding that the four ISS strains belong to the same species which is closely related to, but separate from, *N. diffluens*. This suggested that the four ISS strains were novel species of the genus *Naganishia*. All four strains were isolated during different flights and sampling sessions at various locations, suggesting that these isolates are not clonal but persisting in the ISS environment. The ecological significance of this finding, in the closed system aboard the ISS warrants, further study. The genomes of four ISS strains were sequenced, draft genome was assembled, annotated, and the results were published elsewhere [[6](#B6-jof-08-00165)]. The genome size was ~19.4 Mbp with GC content between 53–56%, similar to other members of the genus *Naganishia*.

#### Figure 1.

[![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ab67/8875396/1ca66803d256/jof-08-00165-g001.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8875396_jof-08-00165-g001.jpg)

[Open in a new tab](figure/jof-08-00165-f001/)

Maximum Likelihood phylogenetic tree, based on seven gene sequences (ITS, LSU, SSU, *CYTB*, *RPB1*, *RPB2*, *TEF1*), extracted from WGS of the respective type strains (superscript T) or from the available genomes, concatenated manually, showing the phylogenetic relationship of *N. tulchinskyi* sp. nov. with closely related members of *Tremellomycetes*. The gene fragment lengths in base pairs (bp) are given in parenthesis: ITS (620 bp); LSU (612 bp); SSU (517 bp); *CYTB* (339 bp); *RPB1* (317 bp); *RPB2* (1282 bp); *TEF1* (1084 bp). Bootstrap values from 1000 replications are shown at branch points. Bar = 0.05 substitution per site.

### 3.2. Titan-like Cell Characterization of Novel ISS Strains

IF6SW-B1T strain was grown under culture conditions similar to those used for titanization of *C. neoformans*, and analyzed using a variety of microscopy techniques. Cell size and cell wall thickness were determined using Calcofluor White stain ([Figure 2](#jof-08-00165-f002)), Titan-like cell morphology using SEM ([Figure 3](#jof-08-00165-f003)), and cell wall structure using TEM ([Figure 4](#jof-08-00165-f004)). The Calcofluor White Stain ([Figure 2](#jof-08-00165-f002)A) binds with chitin contained within cell walls, and once triggered by its excitation wavelength (380 nm), the stain fluoresces and emits blue light at 475 nm. Bright-field images were captured alongside Calcofluor White images to show cell distribution and confirm successful Calcofluor White staining, while Calcofluor White micrographs were used to measure the size of the cells and their thickness ([Figure 2](#jof-08-00165-f002)A). [Figure 2](#jof-08-00165-f002)B shows the cell size in diameter (µm) under various growth conditions, whereas [Figure 2](#jof-08-00165-f002)C depicts the cell wall thickness. The size of the cells grown under standard Earth gravity conditions and in the presence of 5% CO2 with standard Earth gravity ranged from 2.5 µm to 6 µm; however, the cells grown under 5% CO2 were statistically significantly larger (*p* ≤ 0.05). When the strain was grown under 5% CO2 and simulated microgravity (µG) conditions, the cell size increased at least two to three times in size (4 µm to 13 µm; mean increased size of ~10 µm) and was statistically significantly larger than both the standard gravity control (*p* ≤ 0.0001) and the 5% CO2 treatment (*p* ≤ 0.0001) ([Figure 2](#jof-08-00165-f002)B). There was no statistical difference for cell wall thickness between the standard gravity control and the 5% CO2 grow cells (ns). However, the cell wall thickness was two to three-times larger when cells were grown under simulated microgravity conditions compared with the other two culture conditions (0.55 µm and 0.40–0.39 µm, respectively), these findings were statistically significant (*p* ≤ 0.0001) ([Figure 2](#jof-08-00165-f002)C).

#### Figure 2.

[![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ab67/8875396/c98568e54ba0/jof-08-00165-g002.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8875396_jof-08-00165-g002.jpg)

[Open in a new tab](figure/jof-08-00165-f002/)

Confocal microscope-based analyses of *N. tulchinskyi* cells when grown under standard Earth gravity (SD), normal gravity with 5% CO2 (CO), and simulated microgravity with 5% CO2 (µG). Representative images of Calcofluor White cell wall-stained cells adjacent to bright field images are shown in (**A**). The µG cells appear to be larger and misshapen when compared to the typical yeast morphology demonstrated by SD and CO. Measurements for cell diameter were taken using the of Calcofluor White confocal images (**B**). Though smaller than µG cells, SD cells were statically smaller in cell size than CO cells (*p* value = 0.0440), while µG cells were statistically significantly larger than both SD and CO cells (*p* value ≤ 0.0001). Cell wall thickness was also measured using confocal microscopy (**C**). Cell wall thickness of SD and CO were not statistically different in size (*p* value ≥ 0.9999); however, µG cells had statistically significantly larger cell walls than both SD and CO cells (*p* value ≤ 0.0001). The scale bars for both confocal and bright field microscopic images is 40 µm. Statistical significance is reported as *p* value > 0.05 (ns), ≤0.05 (\*), ≤0.01 (\*\*), ≤0.001 (\*\*\*), ≤0.0001 (\*\*\*\*). *p* values of ≤0.01 (\*\*) and ≤0.001 (\*\*\*) were not noticed.

#### Figure 3.

[![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ab67/8875396/c5a8d1e47847/jof-08-00165-g003.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8875396_jof-08-00165-g003.jpg)

[Open in a new tab](figure/jof-08-00165-f003/)

Analyses of scanning electron microscope (SEM) micrographs of SD, CO, and µG grown *N. tulchinskyi* cells demonstrated differences in conidia size, morphology, and daughter cell budding. SD (**A**) and CO (**B**) grown cells demonstrated similar standard yeast morphotypes, while µG (**C**,**D**) grown cells are larger, nonuniform in appearance, and exhibit random budding scars. In some instances, the µG budding scars have raised centers, while others appear to be concave. Additionally, some budding scars appear to be shedding or peeling from the surface of the cells. This phenomenon is highlighted with the white arrow (**D**). Representative SEM micrographs are given, and scale bar for all micrographs is 10 µm.

#### Figure 4.

[![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ab67/8875396/224bf765e62c/jof-08-00165-g004.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8875396_jof-08-00165-g004.jpg)

[Open in a new tab](figure/jof-08-00165-f004/)

Analyses of transmission electron microscope (TEM) micrographs of SD (**A**,**B**), CO (**C**,**D**), and µG (**E**,**F**) grown *N. tulchinskyi* cells demonstrated differences in cell size, cell wall thickness, and internal organelle structures. SD and CO demonstrate similar size cells and similar intracellular morphology, while µG grown cells appear to have a different internal organizational structure of organelles. At higher magnifications (lower panels), the cell wall thickness can be seen to be thicker for µG grown cells than for SD or CO grown cells. The cells also appear to be having lower abundance of fine scale fimbriae on the surface of µG cells, as compared to SD and CO grown cells. Along with their large size, the µG grown cells are distinguished by the shedding and peeling bud scars from their surfaces, denoted by white arrows (**C**). Additionally, µG grown cells have the presence of large unknown organelles/structures, as denoted by the black arrow. Representative TEM micrographs are given. The scale bar for (**A**,**C**,**E**) is 1 µm, and the scale bare for (**B**,**D**,**F**) is 0.2 µm.

The budding scars in IF6SW-B1T strain seemed to exhibit normal bipolar budding pattern when grown under standard gravity ([Figure 3](#jof-08-00165-f003)A) and when grown and exposed to 5% CO2 with standard Earth gravity ([Figure 3](#jof-08-00165-f003)B). However, when the strain is grown under simulated microgravity and 5% CO2 conditions, multilateral budding is seen ([Figure 3](#jof-08-00165-f003)C,D). In addition, the budding cells were not as predominant under microgravity as those grown under normal gravity conditions ([Figure 3](#jof-08-00165-f003)A). Finally, when the IF6SW-B1T strain was grown under 5% CO2, as well as simulated microgravity ([Figure 3](#jof-08-00165-f003)C,D) conditions, observations of budding cells were much less commonly observed, whereas the budding scars were evident and appeared to take up a larger proportional area on the yeast cell surface. The budding scars under simulated microgravity conditions range in morphology and size from 2–3 µm raised prominences extending 1–2 µm from the cell surface to large flat raised rings up to 5 µm in diameter ([Figure 3](#jof-08-00165-f003)C,D). Despite these morphological differences, microgravity grown yeast cells were viable when re-streaked onto a PDA plate. Comparatively, the number of viable cells was less when grown under 5% CO2 and simulated microgravity compared to the other standard atmosphere and gravity conditions (data not shown). These results suggested that the cells were defective in cytoskeleton formation, which might limit the formation of viable daughter cells. Cell wall structure analysis was then performed for the IF6SW-B1T strain using TEM data. A simple comparison of the cell wall thickness of the cells grown under standard Earth gravity ([Figure 4](#jof-08-00165-f004)A,B) with simulated microgravity ([Figure 4](#jof-08-00165-f004)E, F) conditions exhibited enormous size (~5×) differences and substantiated the confocal microscopy-based measurements.

Comparison of confocal, SEM, and TEM ([Figure 2](#jof-08-00165-f002), [Figure 3](#jof-08-00165-f003) and [Figure 4](#jof-08-00165-f004)) between the three different conditions indicated that the µG grown Titan-like cells were much larger and exhibited irregular morphologies, including the novel discovery of peeling and shedding bud scars ([Figure 3](#jof-08-00165-f003)D, [Figure 4](#jof-08-00165-f004)E, and [Figure S8A–G](#app1-jof-08-00165)). Indeed, a large majority of the disfigurement and abnormal cell morphology appeared to be associated with the surface location where daughter cell replication occurred ([Figure 3](#jof-08-00165-f003)C,D and [Figure S8A–C](#app1-jof-08-00165)). Further analyses of TEM images emphasized the differences between the growth conditions, with µG grown cells showing large structural alterations within the intracellular space, having thicker cell walls, and a decrease in the amount of surface-associated fimbria ([Figure 4](#jof-08-00165-f004)E,F). Fimbria have been shown to play roles in both biofilm formation [[60](#B60-jof-08-00165)] and pathogenesis [[61](#B61-jof-08-00165)]; thus, the modifications seen in µG grown cell fimbria could alter both biofilm formation and pathogenesis. The increased cell wall thickness of the µG grown cells could impart resistance to osmotic shock or add protection against radiation exposure. These structural alterations seen in TEM are a direct stress response to the microgravity and elevated CO2 conditions, and they could possibly serve to benefit (or hinder) the yeast cell’s survivability in the spaceflight environment.

Additionally, the µG grown cells showed irregular intracellular organization with what appears to be large unknown membrane bound compartments/structures ([Figure 4](#jof-08-00165-f004)E). The function and content of these structures is still unclear, and it is currently unknown if these are vacuole or lysome-like vacuoles ([Figure 4](#jof-08-00165-f004)E) [[62](#B62-jof-08-00165)]. Previous reports have shown that some yeast can produce large numbers of compounds that are subsequently stored in vacuoles, while other yeast can utilize these large storage vesicles to degrade a large variety of substances [[62](#B62-jof-08-00165),[63](#B63-jof-08-00165)]. Increased vacuole size could suggest a larger assortment and or quantity of nutrients being degraded by the Titan-like cells than by the standard gravity grown cells. Additionally, research has shown that portions of vacuoles can be passed to daughter cells [[64](#B64-jof-08-00165)]. Thus, it is possible that simulated microgravity induced compounds contained in the vacuoles may be passed to their progeny. Much further work is needed to better identify novel structures within the Titan-like cells, categorize the specific modifications that are occurring to organelles within the cells, determine how each of the cellular modifications alters the cells’ survivability in extreme environments (pH ranges, temperature ranges, osmotic shock, radiation exposure, etc.), and determine if these traits are passed on to their progeny.

*Cryptococcus* species have been shown to produce enlarged Titan-like cells to adapt to stress and contribute to *Cryptococcus* pathogenesis [[20](#B20-jof-08-00165),[65](#B65-jof-08-00165)]. Both functions may also be of relevance for these ISS strains. The polyploidy nature of the yeast cells should be investigated as this might shed insights into the adaptation of these cells to microgravity conditions. Even though confocal microscopy measurements and increased fluorescence content in the microgravity-grown cells were documented, analysis of the nucleic acid reads of heterozygous regions of the genomes and a comparison of the ratios for the various alleles is needed to confirm the polyploidy nature of these cells. It is documented in *C. neoformans* that the enlarged polyploid Titan-like cells were able to generate progeny with aneuploidy. It was additionally found that the progeny of these Titan-like cells have both growth and survival advantages over typical cell progeny under stressful conditions [[20](#B20-jof-08-00165)]. Further gene characterizations are required to understand the molecular mechanisms associated with adaptation of novel IF6SW-B1T yeast to space conditions [[20](#B20-jof-08-00165)].

While recent studies suggest that titanization is a rare phenomenon that occurs mostly in members of the *C. neoformans/C. gattii* species complex, the observation of Titan-like cells in ISS *Naganishia* species is novel. Understanding how microgravity influences this process is important to predict the emergence of virulence in *Naganishia* species and develop novel antifungal drugs. These antifungal drugs can be developed to inhibit the phenotypic transitions associated with fungal pathogenicity, especially since these strains were isolated across longitudinal sampling sessions of the ISS and appear persistent. Titanization and the increasing of cell size has been shown to aide *Cryptococcus* cells evade the immune system [[21](#B21-jof-08-00165),[26](#B26-jof-08-00165),[65](#B65-jof-08-00165),[66](#B66-jof-08-00165),[67](#B67-jof-08-00165)]. By growing from the normal range of ~1 to 3 μm in diameter to >10 μm, Titan cells cannot be readily ingested by phagocytic cells and, thus, persist in host tissue. Moreover, our data show that Titan-like cells are more likely to survive and produce offspring, when confronted by multiple environmental stress conditions, than typical cells. Additionally, their normally sized offspring maintained an advantage over regular cells in continued exposure to stress. Titan-like cell progeny’s increased survival and genomic diversity might promote rapid adaptation to new or high-stress space conditions; however, more research is needed to confirm this aspect.

### 3.3. Genomic Analyses of the Novel ISS Strain

Using BLAST, genes in ISS *Naganishia* species were compared to known genes that conferred resistance to cold, microgravity, and radiation ([Table 1](#jof-08-00165-t001)), as well as pigment biosynthesis. The reference genes were a compilation of NCBI queries and previously reported genes related to the spaceflight-related characteristics ([Table 1](#jof-08-00165-t001)). In addition to our four *N. tulchinskyi* strains, *N. vishniacii* (Antarctica strain), *Papiliotrema laurentii* (ISS isolates), and *Saccharomyces cerevisiae* (space flown strain) were included for comparison. A total of 24 genes aligned to one or more of the assemblies. Genes related to pigment synthesis and radioresistance were among the most prevalent, while genes associated with microgravity were mostly exclusive of *S. cerevisiae*.

Proteome prediction was performed with MAKER to determine biotechnologically important genetic elements in the four novel *Naganishia* strains [[31](#B31-jof-08-00165)]. Protein-coding genes were then assigned gene ontologies (GO) using InterProScan [[68](#B68-jof-08-00165)] ([Figure 5](#jof-08-00165-f005)) and passed through REVIGO [[58](#B58-jof-08-00165)] to summarize GO assignments and remove redundant terms ([Figure S1](#app1-jof-08-00165)). In total, 4894 protein-coding genes were predicted from the assembled draft genomes, which comprised 128 scaffolds ([Supplemental Table S1](#app1-jof-08-00165)). A total of 86, 47, and 89 top ontological hits were assigned, respectively, for Biological Process, Cellular Component, and Molecular Function ([Table S1](#app1-jof-08-00165)), ranked by ontological uniqueness of term with respect to all other terms cataloged in UniProt. Top Biological Processes include cell morphogenesis, microtubule-based response, and response to UV light, reflecting a repertoire that was suited to respond to microgravity and radiation ([Figure 5](#jof-08-00165-f005) and [Figure S2](#app1-jof-08-00165)). The extracellular region, outer membrane, and cell wall were top cellular component results, implying a set of genes associated with Titan-like cell plasticity. Finally, the top molecular function hits included cytoskeletal motor activity, microtubule motor activity, and nuclear export signal receptor activity. Changing these molecular mechanisms can alter the shape of cells, and in fungi, morphological changes in response to environmental stimuli have been reported and mostly attributed to pathogenesis [[20](#B20-jof-08-00165),[69](#B69-jof-08-00165)]. The highest ontological hits revealed the presence of protein coding genes that might provide insights into the underlying origins of Titan-like cells in *N. tulchinskyi*. However, more studies are necessary to depict the mechanisms and role of titanization in microgravity.

#### Figure 5.

[![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ab67/8875396/7a75ad80d5d8/jof-08-00165-g005.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8875396_jof-08-00165-g005.jpg)

[Open in a new tab](figure/jof-08-00165-f005/)

Enriched GOSlim terms for biological process, cellular components, and molecular function in *N. tulchinskyi* genome. The Gene Ontology terms were summarized by purging redundant GO terms that were deduced by orthologous searches coupled with semantic similarity-based scatterplots using the REVIGO platform [[58](#B58-jof-08-00165)].

The ability of *Naganishia* species to grow at low temperatures [[29](#B29-jof-08-00165)] and withstand high doses of UV radiation in extreme environments [[1](#B1-jof-08-00165)] has led to the ongoing microbial characterization of this genus; however, there are currently only a few genomes publicly available [[70](#B70-jof-08-00165),[71](#B71-jof-08-00165)]. Unlike the *S. cerevisiae* genome, which has been extensively studied [[72](#B72-jof-08-00165)], the genomic characterization of the *Naganishia* clade is still in its infancy. The presence of reported genes related to cold resistance [[39](#B39-jof-08-00165)], radiation tolerance [[47](#B47-jof-08-00165)], and microgravity ([Table 1](#jof-08-00165-t001)) in ISS strains might indicate the potential of the novel *N. tulchinskyi* to better withstand the ISS or spaceflight-related conditions. However, only a few genes associated with microgravity have been identified [[73](#B73-jof-08-00165),[74](#B74-jof-08-00165)]. It has been reported that the *RAD51*, *RAD54*, and *RDH54* genes seen in ISS strains also play critical roles in *C. neoformans* to repair gamma-radiation induced DNA damage [[47](#B47-jof-08-00165)].

Additionally, high carotenoid concentration has been correlated with an increased resistance of the cells against freeze damage, environmental stress, and radiation protection [[75](#B75-jof-08-00165),[76](#B76-jof-08-00165)], all of which are notable features of the extremophile *Deinococcus radiodurans* [[77](#B77-jof-08-00165),[78](#B78-jof-08-00165)]. Biomolecules responsible for the pigment biosynthesis, such as geranylgeranyl diphosphate synthase, cytochrome P450 reductase, farnesyl pyrophosphate synthase, isopentenyl-diphosphate delta isomerase, alpha, alpha-trehalase NTH2, and NTH1, putative trehalose-phosphate synthase Tps2 were present in all strains of *Naganishia* and *S. cerevisiae* ([Table 1](#jof-08-00165-t001)). It has been well documented that carotenoids can efficiently protect the skin from UV-light induced damages, scavenging singlet oxygen and peroxyl radicals [[79](#B79-jof-08-00165)]. The microbiological synthesis of carotenoids, significance, and biotechnological relevance were extensively reviewed [[80](#B80-jof-08-00165)]. The phytoene-beta carotene synthase (*pbs*) was present only in ISS strains but not found in *S. cerevisiae.* More research is needed here since biosynthesis of tetraterpenoids and *pbs* involvement in microorganisms is still poorly understood [[81](#B81-jof-08-00165)].

### 3.4. Taxonomy

Novel species: *Naganishia tulchinskyi,* N. K. Singh and K. Venkateswaran, sp. nov.

MycoBank number: MB838832.

**Etymology**: *Naganishia tulchinskyi* (tul.chin. s’ky.i. N.L. gen. n. *tulchinskyi* named after Igor Tulchinsky, an American Philanthropist supporting biomedical, computational, and space research).

**Diagnosis**: Based on the morphological characteristics, strains IF6SW-B1T, IF7SW-B1, IF1SW-F1, and IIF5SW-F1 are similar to *N. albida* group, but phylogenetically distant from all known *Naganishia* species. Hyphae and pseudohyphae were not present, sexual reproduction was not observed, budding cells were present and ballistoconidia were absent.

**Type strain**: IF6SW-B1T.

**Holotype**: USA: International Space Station surfaces, March 15, 2015. Nitin K. Singh and Kasthuri Venkateswaran, ex-type living culture (IF6SW-B1T) was preserved in a metabolically inactive state at the Northern Regional Research Laboratory (NRRL 64202) and at the Deutsche Sammlung von Mikroorganismen und Zellkulturen (DSM 113487).

**Description**: In PDA medium after 5 days at 25 °C, the cells are round, ovoid to ellipsoidal, 4.1 ± 0.97 μm and occur singly with polar budding. When grown under simulated microgravity at 30 °C, cells are ~3 times larger and formed Titan-like cells ([Figure 2](#jof-08-00165-f002)). After incubation at 25 °C, the streak culture on PDA agar is butyrous and cream colored, with a smooth surface and an entire margin. Growth occurs at 4, 25, and 30 °C and not at 35 °C on nutrient medium.

**Biochemical characteristics:** Differential biochemical characteristics of *N. tulchinskyi* with other closely related *Naganishia* species are shown in [Table 3](#jof-08-00165-t003). *N. tulchinskyi* strains are able to assimilate D-Arabinose, D-Ribose, L-Sorbose, D-Mannitol, α-D-Glucose, L-Arabinose, Salicin, D-Trehalose, Maltose, D-Xylose, Sucrose, D-Cellobiose, Succinic Acid Mono-Methyl Ester, D-Xylose, and Palatinose as sole carbon source. *N. tulchinskyi* cells did not utilize Cellobiose, Arbutin, D-Galactose, Glycerol, Hexadecane, L-Malic acid, L-Rhamnose, Lactose, Mannitol, Melezitose, Methyl a-glucoside, Myo-inositol, Raffinose, Xylitol, Citrate, Melibiose, Meso erythritol, Glucono D-lactone, Inulin, and N-acetyl D-glucosamine as sole carbon source. *N. tulchinskyi* cells assimilate D-arabinose and can be differentiated from *N. friedmannii, N. saitoi, N. globosa.* Similarly, several carbon substrates are differentially utilized by *N. tulchinskyi* and are detailed in [Table 3](#jof-08-00165-t003).

#### Table 3.

Physiological characteristics of *Naganishia tulchinskyi* and closely related species.

| Carbon Source | | *Naganishia tulchinskyi* (This Study) | | | | *N. vaughanmartiniae* a | *N. onofrii* a | *N. saitoi* b | *N. friedmannii* b | *N. qatarensis* c | *N. cerealis* d | *N. randhawae* e | *N. globosa* b |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| IF6SW-B1 | IF7SW-B1 | IF1SW-F1 | IIF5SW-F1 |
| **Growth at:** | |  |  |  |  |  |  |  |  |  |  |  |  |
|  | 4 °C | + | + | + | + | + | + | NT | + | NT | NT | NT | NT |
|  | 25 °C | + | + | + | + | + | + | + | - | + | + | + | + |
|  | 30 °C | + | + | + | + | - | - | + | - | + | + | + | + |
|  | 35 °C | - | - | - | - | - | - | - | - | + | - | - | - |
| **Differential characteristics among various *Naganishia* species** | | | | | | | | | | | | | |
|  | D-Arabinose | + | + | + | + | + | + | - | - | + | + | w | - |
|  | D-Ribose | + | + | + | + | + | v | v | - | + | v | - | - |
|  | L-Sorbose | + | + | + | + | v | v | - | - | + | + | - | - |
|  | D-Mannitol | + | + | + | - | NT | NT | NT | - | + | + | + | + |
|  | α-D-Glucose | + | + | + | + | + | + | + | + | - | NT | NT | NT |
|  | L-Arabinose | + | + | + | + | + | + | + | + | + | w | w | - |
|  | Salicin | - | + | + | + | + | + | + | + | - | NT | NT | NT |
|  | D-Trehalose | + | + | - | + | + | + | + | + | - | NT | NT | NT |
|  | Maltose | + | + | + | - | + | + | + | + | - | NT | NT | NT |
|  | D-Xylose | + | + | - | + | + | + | + | + | - | NT | NT | NT |
|  | Sucrose | + | + | - | + | + | + | + | v | - | NT | NT | NT |
|  | Cellobiose | - | - | - | - | + | + | + | + | - | NT | NT | NT |
|  | Arbutin | - | - | - | - | + | + | NT | NT | - | NT | NT | NT |
|  | D-Galactose | - | - | - | - | + | + | -/w | - | + | + | + | w |
|  | Glycerol | - | - | - | - | v | + | -/s | - | + | - | - | - |
|  | Hexadecane | - | - | - | - | - | - | NT | NT | NT | NT | NT | NT |
|  | L-Malic acid | - | - | - | - | - | - | + | v | NT | NT | NT | NT |
|  | L-Rhamnose | - | - | - | - | + | + | + | - | + | + | + | + |
|  | Lactose | - | - | - | - | v | + | v | - | + | + | + | v |
|  | Mannitol | - | - | - | - | + | + | + | - | + | - | + | - |
|  | Melezitose | - | - | - | - | + | + | + | + | - | NT | NT | NT |
|  | Methyl a-glucoside | - | - | - | - | + | + | + | v | - | NT | NT | NT |
|  | Myo-inositol | - | - | - | - | v | + | + | - | + | + | + | + |
|  | Raffinose | - | - | - | - | -/w | + | -/w | - | + | w | + | w |
|  | Xylitol | - | - | - | - | + | + | - | - | + | + | + | - |
|  | Citrate | - | - | - | - | - | - | + | v | - | - | - | + |
|  | Melibiose | - | - | - | - | - | - | - | - | + | - | - | - |
|  | Meso erythritol | - | - | - | - | - | - | - | - | w | - | - | - |
|  | Glucono D-lactone | - | - | - | - | - | v | NT | NT | + | + | - | NT |
|  | *N*-acetyl D-glucosamine | - | - | - | - | - | - | NT | NT | - | NT | NT | NT |
| **Common characteristics among *N. tulchinskyi* strains** | | | | | | | | | | | | | |
|  | D-Cellobiose | + | + | + | + |  |  |  |  |  |  |  |  |
|  | Succinic Acid Mono-Methyl Ester plus D-Xylose | + | + | + | + |  |  |  |  |  |  |  |  |
|  | Palatinose | + | + | + | + |  |  |  |  |  |  |  |  |
|  | Inulin | - | - | - | - |  |  |  |  |  |  |  |  |
| **Differential characteristics among *N. tulchinskyi* strains** | | | | | | | | | | | | | |
|  | Gentiobiose | + | + | + | - |  |  |  |  |  |  |  |  |
|  | D-Mannitol | + | + | + | - |  |  |  |  |  |  |  |  |
|  | Maltose | + | + | + | - |  |  |  |  |  |  |  |  |
|  | Acetic Acid | + | + | - | + |  |  |  |  |  |  |  |  |
|  | Maltotriose | + | + | - | + |  |  |  |  |  |  |  |  |
|  | Turanose | + | + | - | + |  |  |  |  |  |  |  |  |
|  | D- Glucuronic Acid plus D-Xylose | + | + | - | + |  |  |  |  |  |  |  |  |
|  | 1,2- Propanediol plus D-Xylose | + | + | - | + |  |  |  |  |  |  |  |  |
|  | D-Trehalose | + | + | - | + |  |  |  |  |  |  |  |  |
|  | D-Xylose | + | + | - | + |  |  |  |  |  |  |  |  |
|  | Sucrose | + | + | - | + |  |  |  |  |  |  |  |  |
|  | D-Melezitose | - | + | + | + |  |  |  |  |  |  |  |  |
|  | Dextrin | - | + | + | + |  |  |  |  |  |  |  |  |
|  | Propionic Acid | + | + | - | - |  |  |  |  |  |  |  |  |
|  | D-Raffinose | - | - | + | - |  |  |  |  |  |  |  |  |
|  | D-Sorbitol | - | - | + | - |  |  |  |  |  |  |  |  |

[Open in a new tab](table/jof-08-00165-t003/)

a Data from Turchetti et al. 2015 (11). b Data obtained from Fonseca et al. 2011 (13). c Data obtained from Fotedar et al. 2018 (1). d Data obtained from Passoth et al. 2009 (16). e Data obtained from Khan et al. 2010 (14). Abbreviation: “+” positive; “-” negative; “NT” not tested; “w” weak reaction; “v” variable reaction.

**Ecology/Substrate/Host**: Distinct *Naganishia* yeast colonies (*n* = 4) were isolated over two separate sample collection flights from three different locations. The type strain IF6SW-B1T and two other strains (IF7SW-B1 and IF1SW-F1) were isolated during Flight 1 (March 2015), while the fourth strain (IIF5SW-F1) was isolated during Flight 2 (May 2015). Strain IF6SW-B1T was isolated from a rack that contained crew clothes, crew preference items, office supplies, and ISS medical accessory kits (Location #6, Permanent Multipurpose Module Port 1). The second strain IF7SW-B1 was isolated from a rack being used for basic materials research in the microgravity environment of the ISS (Location #7, Overhead-3 panel surface of the Materials Science Research Rack 1). The third strain IF1SW-F1 was isolated from a viewing port in a small module devoted to the observation of operations outside the ISS, such as robotic activities, spacecraft approaches, and extravehicular activities (Location #1, Port panel of the Cupola). The fourth strain IIF5SW-F1 was the only strain in this study isolated during Flight 2, and it was cultivated from a storage rack made up of Nomex fabrics and used as a pantry to store batteries, printer cartridges, office supplies, etc. (Location #5, Overhead-4, Node 1, US module).

**Distribution**: This novel species has yet to be found in any samples from other Earth environments.

**Material examined**: The particulate matter associated with 1 m2 surface area of the ISS environmental surfaces was examined.

## 4. Conclusions

In summary, a novel yeast, *N. tulchinskyi*, was isolated from the ISS environmental surfaces that produced giant Titan-like cells under simulated microgravity conditions. Morphological characterization of *N. tulchinskyi* cells revealed that the cell wall is thicker under simulated microgravity and might help them withstand such unfavorable conditions, and these simulated microgravity-grown cells demonstrate a novel bud scar shedding trait. The phylogenetic analyses using the MLST approach showed that these are novel yeast. Further genomic characterization revealed genes responsible for cell morphogenesis and UVC resistance, reflecting a range of molecular mechanisms suited to respond to microgravity and radiation.

## Acknowledgments

The research described in this manuscript was performed at the Jet Propulsion Laboratory, California Institute of Technology under a contract with NASA and the University of Southern California. We would like to thank the Aleksandra Checinska-Sielaff for isolating the strain. We thank astronauts Captain Terry Virts for collecting samples aboard the ISS and the Implementation Team at NASA Ames Research Center (Fathi Karouia) for coordinating this effort. Confocal imaging was performed in the Biological Imagining Facility, with the support of the Caltech Beckman Institute and the Arnold and Mabel Beckman Foundation at the California Institute of Technology. The authors would like to thank Giada Spigolon for her expertise with confocal microscopy. Additionally, we thank Danielle Jorgens at the University of California Berkeley Electron Microscope Laboratory for advice and assistance in electron microscopy sample preparation and data collection. Scanning Electron Microscopy was performed at the California Institute of Technology’s Kavli Nanoscience Institute. We thank Ryan Kemp (Zymo Corp.) for extracting DNA and Dan Butler (Cornell Medicine) for performing shotgun sequencing using the NovaSeq platform. We also acknowledge the Jet Propulsion Laboratory supercomputing facility staff, notably Narendra J. (Jimmy) Patel and Edward Villanueva, for their continuous support in providing the best possible infrastructure for BIG-DATA analyses. We would like to thank the Genomics and Epigenomics Core Facilities at Weill Cornell Medicine, the Scientific Computing Unit (SCU), XSEDE Supercomputing Resources, and also Olivier Elemento for logistics and support. ©2021 California Institute of Technology. Government sponsorship acknowledged.

## Supplementary Materials

The following are available online at <https://www.mdpi.com/article/10.3390/jof8020165/s1>. Figure S1. Maximum Likelihood phylogenetic tree, based on ITS regions, showing the phylogenetic relationship of *N. tulchinskyi* with closely related members of Tremellomycetes. Bootstrap values from 1000 replications are shown at branch points. Bar, 0.02 substitution per site. Figure S2. Maximum Likelihood phylogenetic tree, based on LSU sequences, showing the phylogenetic relationship of *N. tulchinskyi* with closely related members of Tremellomycetes. Bootstrap values from 1000 replications are shown at branch points. Bar, 0.02 substitution per site. Figure S3. Maximum Likelihood phylogenetic tree, based on SSU sequences, showing the phylogenetic relationship of *N. tulchinskyi* with closely related members of Tremellomycetes. Bootstrap values from 1000 replications are shown at branch points. Bar, 0.01 substitution per site. Figure S4. Maximum Likelihood phylogenetic tree, based on cytB sequences, showing the phylogenetic relationship of *N. tulchinskyi* with closely related members of Tremellomycetes. Bootstrap values from 1000 replications are shown at branch points. Bar, 0.05 substitution per site. Figure S5. Maximum Likelihood phylogenetic tree, based on rpb1 sequences, showing the phylogenetic relationship of *N. tulchinskyi* with closely related members of Tremellomycetes. Bootstrap values from 1000 replications are shown at branch points. Bar, 0.10 substitution per site. Figure S6. Maximum Likelihood phylogenetic tree, based on rpb2 sequences, showing the phylogenetic relationship of *N. tulchinskyi* with closely related members of Tremellomycetes. Bootstrap values from 1000 replications are shown at branch points. Bar, 0.10 substitution per site. Figure S7. Maximum Likelihood phylogenetic tree, based on tef1 sequences, showing the phylogenetic relationship of *N. tulchinskyi* with closely related members of Tremellomycetes. Bootstrap values from 1000 replications are shown at branch points. Bar, 0.02 substitution per site. Figure S8. SEM (A-C) and TEM (D-G) micrographs highlighting the unusual μG grown Titan-like cell morphology. Titan-like cells show large concave and convex budding scars (A-C). A daughter cell can be seen budding from the mother yeast cell (black arrow), and appears to be altering the shape of the mother cell’s cell wall (B). Budding scars are shown to be peeling off of the yeast surfaces (white arrows) (A-C). This phenomenon is shown in TEM images, designated by white arrows (D-G). SEM scale bars are 10μm. TEM scale bars are 5μm (D), 1μm (E-F), and 0.5μm (G). The letter “N” distinguishes the nucleus (B, C, and E). Table S1. Reducing all protein-coding gene products from the predicted proteome of the novel type strain IF6SW-B1T to their most distinct Gene Ontology (GO) terms for Biological Process, Cellular Component, and Molecular Function. Frequency indicates the number of genes in UniProt associated with this term. Uniqueness is a measurement of average similarity of a term to all other GO terms, while dispensability measures the semantic distance between a term and all other GO terms.

[Click here for additional data file.](/articles/instance/8875396/bin/jof-08-00165-s001.zip) (5.2MB, zip)

## Author Contributions

K.V. and N.K.S. conceived and designed the experiments. K.V. and C.E.M. designed and supervised the analyses and genome profiling experiments. S.B. and N.K.S. carried out phylogenetic analyses and alignment for positional description of organism in the tree, S.B. conducted HARV culturing, S.B. independently verified the genome assembly, generated alignments for all gene trees in the manuscript, and manually curated the tree images, and C.P. generated microscopy and analyzed cell wall structure. M.A.S. and J.F. analyzed the genomic data inclusive of de novo assemblies and verification, scaffold quality assessment, and annotation and generated the draft assembly of the genomes. K.V. isolated the type strain and compiled the contribution of write-ups from all authors associated with microscopy (C.P.), genotype and table generation (S.B.), and phylogenetic trees and figure generation (S.B.), and genomic characteristics (M.A.S., J.F.). C.E.M. generated genomic libraries and sequenced the genomes of all strains. C.C.C.W. reviewed the manuscript. All authors have read and agreed to the published version of the manuscript.

## Funding

The research described in this manuscript was funded by a 2012 Space Biology NNH12ZTT001N grant no. 19-12829-26 under Task Order NNN13D111T award to KV, and NASA’s 2018 Space Biology (ROSBio) NNH18ZTT001N-FG App B: Flight and Ground Space Biology Research Grant 80NSSC19K1501 awarded to C.C.C.W. We also thank WorldQuant Foundation for their support, as well as the Starr Cancer Consortium: (I13-0052), the National Institutes of Health (R01AI151059, U01DA053941, R35GM138152), the Bill and Melinda Gates Foundation (OPP1151054), and the Alfred P. Sloan Foundation (G-2015-13964).

## Institutional Review Board Statement

This study did not involve humans or animals.

## Informed Consent Statement

Not applicable since this study did not involve humans.

## Data Availability Statement

The WGS and raw data have been deposited in GenBank under the BioProject accession number PRJNA623412 [[6](#B6-jof-08-00165)]. The genome sequences has also been deposited in the NASA GeneLab system (GLDS-290; <https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-290>; accessed on 5 December 2021).

## Conflicts of Interest

The author(s) declare that there are no conflict of interest. This manuscript was prepared as an account of work sponsored by NASA, an agency of the US Government. The US Government, NASA, California Institute of Technology, Jet Propulsion Laboratory, and their employees make no warranty, expressed or implied, or assume any liability or responsibility for the accuracy, completeness, or usefulness of information, apparatus, product, or process disclosed in this manuscript, or represents that its use would not infringe upon privately held rights. The use of, and references to any commercial product, process, or service does not necessarily constitute or imply endorsement, recommendation, or favoring by the US Government, NASA, California Institute of Technology, or Jet Propulsion Laboratory. Views and opinions presented herein by the authors of this manuscript do not necessarily reflect those of the US Government, NASA, California Institute of Technology, or Jet Propulsion Laboratory, and shall not be used for advertisements or product endorsements.

## Footnotes

**Publisher’s Note:** MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.

## References

* 1.Fotedar R., Kolecka A., Boekhout T., Fell J.W., Anand A., Al Malaki A., Zeyara A., Al Marri M. Naganishia qatarensis sp. nov., a novel basidiomycetous yeast species from a hypersaline marine environment in Qatar. Int. J. Syst. Evol. Microbiol. 2018;68:2924–2929. doi: 10.1099/ijsem.0.002920. [[DOI](https://doi.org/10.1099/ijsem.0.002920)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30070622/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Naganishia%20qatarensis%20sp.%20nov.,%20a%20novel%20basidiomycetous%20yeast%20species%20from%20a%20hypersaline%20marine%20environment%20in%20Qatar&author=R.%20Fotedar&author=A.%20Kolecka&author=T.%20Boekhout&author=J.W.%20Fell&author=A.%20Anand&volume=68&publication_year=2018&pages=2924-2929&pmid=30070622&doi=10.1099/ijsem.0.002920&)]
* 2.Liu X.Z., Wang Q.M., Theelen B., Groenewald M., Bai F.Y., Boekhout T. Phylogeny of tremellomycetous yeasts and related dimorphic and filamentous basidiomycetes reconstructed from multiple gene sequence analyses. Stud. Mycol. 2015;81:1–26. doi: 10.1016/j.simyco.2015.08.001. [[DOI](https://doi.org/10.1016/j.simyco.2015.08.001)] [[PMC free article](/articles/PMC4777771/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26955196/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stud.%20Mycol.&title=Phylogeny%20of%20tremellomycetous%20yeasts%20and%20related%20dimorphic%20and%20filamentous%20basidiomycetes%20reconstructed%20from%20multiple%20gene%20sequence%20analyses&author=X.Z.%20Liu&author=Q.M.%20Wang&author=B.%20Theelen&author=M.%20Groenewald&author=F.Y.%20Bai&volume=81&publication_year=2015&pages=1-26&pmid=26955196&doi=10.1016/j.simyco.2015.08.001&)]
* 3.Fell J.W., Boekhout T., Fonseca A., Scorzetti G., Statzell-Tallman A. Biodiversity and systematics of basidiomycetous yeasts as determined by large-subunit rDNA D1/D2 domain sequence analysis. Int. J. Syst. Evol. Microbiol. 2000;50:1351–1371. doi: 10.1099/00207713-50-3-1351. [[DOI](https://doi.org/10.1099/00207713-50-3-1351)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10843082/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Biodiversity%20and%20systematics%20of%20basidiomycetous%20yeasts%20as%20determined%20by%20large-subunit%20rDNA%20D1/D2%20domain%20sequence%20analysis&author=J.W.%20Fell&author=T.%20Boekhout&author=A.%20Fonseca&author=G.%20Scorzetti&author=A.%20Statzell-Tallman&volume=50&publication_year=2000&pages=1351-1371&pmid=10843082&doi=10.1099/00207713-50-3-1351&)]
* 4.Scorzetti G., Fell J.W., Fonseca A., Statzell-Tallman A. Systematics of basidiomycetous yeasts: A comparison of large subunit D1/D2 and internal transcribed spacer rDNA regions. FEMS Yeast Res. 2002;2:495–517. doi: 10.1016/S1567-1356(02)00128-9. [[DOI](https://doi.org/10.1016/S1567-1356%2802%2900128-9)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12702266/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Yeast%20Res.&title=Systematics%20of%20basidiomycetous%20yeasts:%20A%20comparison%20of%20large%20subunit%20D1/D2%20and%20internal%20transcribed%20spacer%20rDNA%20regions&author=G.%20Scorzetti&author=J.W.%20Fell&author=A.%20Fonseca&author=A.%20Statzell-Tallman&volume=2&publication_year=2002&pages=495-517&pmid=12702266&doi=10.1016/S1567-1356(02)00128-9&)]
* 5.Sielaff A.C., Urbaniak C., Mohan G.B.M., Stepanov V.G., Tran Q., Wood J.M., Minich J., McDonald D., Mayer T., Knight R., et al. Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome. 2019;7:50. doi: 10.1186/s40168-019-0666-x. [[DOI](https://doi.org/10.1186/s40168-019-0666-x)] [[PMC free article](/articles/PMC6452512/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&author=A.C.%20Sielaff&author=C.%20Urbaniak&author=G.B.M.%20Mohan&author=V.G.%20Stepanov&author=Q.%20Tran&volume=7&publication_year=2019&pages=50&pmid=30955503&doi=10.1186/s40168-019-0666-x&)]
* 6.Bijlani S., Singh N.K., Mason C.E., Wang C.C.C., Venkateswaran K. Draft Genome Sequences of Tremellomycetes Strains Isolated from the International Space Station. Microbiol. Resour. Announc. 2020;9:e00504-20. doi: 10.1128/MRA.00504-20. [[DOI](https://doi.org/10.1128/MRA.00504-20)] [[PMC free article](/articles/PMC7317102/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32586865/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Resour.%20Announc.&title=Draft%20Genome%20Sequences%20of%20Tremellomycetes%20Strains%20Isolated%20from%20the%20International%20Space%20Station&author=S.%20Bijlani&author=N.K.%20Singh&author=C.E.%20Mason&author=C.C.C.%20Wang&author=K.%20Venkateswaran&volume=9&publication_year=2020&pages=e00504-20&pmid=32586865&doi=10.1128/MRA.00504-20&)]
* 7.Goto S., Sugiyama J., Iizuka H. A Taxonomic Study of Antarctic Yeasts. Mycologia. 1969;61:748–774. doi: 10.2307/3757466. [[DOI](https://doi.org/10.2307/3757466)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/5382150/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=A%20Taxonomic%20Study%20of%20Antarctic%20Yeasts&author=S.%20Goto&author=J.%20Sugiyama&author=H.%20Iizuka&volume=61&publication_year=1969&pages=748-774&pmid=5382150&doi=10.2307/3757466&)]
* 8.Vishniac H.S., Kurtzman C.P. Cryptococcus antarcticus sp. nov. and Cryptococcus albidosimilis sp. nov., Basidioblastomycetes from Antarctic Soils. Int. J. Syst. Evol. Microbiol. 1992;42:547–553. doi: 10.1099/00207713-42-4-547. [[DOI](https://doi.org/10.1099/00207713-42-4-547)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Cryptococcus%20antarcticus%20sp.%20nov.%20and%20Cryptococcus%20albidosimilis%20sp.%20nov.,%20Basidioblastomycetes%20from%20Antarctic%20Soils&author=H.S.%20Vishniac&author=C.P.%20Kurtzman&volume=42&publication_year=1992&pages=547-553&doi=10.1099/00207713-42-4-547&)]
* 9.Vishniac H.S. Cryptococcus friedmannii, a New Species of Yeast from the Antarctic. Mycologia. 1985;77:149–153. doi: 10.2307/3793260. [[DOI](https://doi.org/10.2307/3793260)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11540828/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=Cryptococcus%20friedmannii,%20a%20New%20Species%20of%20Yeast%20from%20the%20Antarctic&author=H.S.%20Vishniac&volume=77&publication_year=1985&pages=149-153&pmid=11540828&doi=10.2307/3793260&)]
* 10.Vishniac H.S., Hempeling W.P. Cryptococcus vishniacii sp. nov., an Antarctic Yeast. Int. J. Syst. Evol. Microbiol. 1979;29:153–158. doi: 10.1099/00207713-29-2-153. [[DOI](https://doi.org/10.1099/00207713-29-2-153)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Cryptococcus%20vishniacii%20sp.%20nov.,%20an%20Antarctic%20Yeast&author=H.S.%20Vishniac&author=W.P.%20Hempeling&volume=29&publication_year=1979&pages=153-158&doi=10.1099/00207713-29-2-153&)]
* 11.Turchetti B., Selbmann L., Blanchette R.A., Di Mauro S., Marchegiani E., Zucconi L., Arenz B.E., Buzzini P. Cryptococcus vaughanmartiniae sp. nov. and Cryptococcus onofrii sp. nov.: Two new species isolated from worldwide cold environments. Extremophiles. 2015;19:149–159. doi: 10.1007/s00792-014-0692-3. [[DOI](https://doi.org/10.1007/s00792-014-0692-3)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25239569/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Extremophiles&title=Cryptococcus%20vaughanmartiniae%20sp.%20nov.%20and%20Cryptococcus%20onofrii%20sp.%20nov.:%20Two%20new%20species%20isolated%20from%20worldwide%20cold%20environments&author=B.%20Turchetti&author=L.%20Selbmann&author=R.A.%20Blanchette&author=S.%20Di%20Mauro&author=E.%20Marchegiani&volume=19&publication_year=2015&pages=149-159&pmid=25239569&doi=10.1007/s00792-014-0692-3&)]
* 12.Boekhout T., Bandoni R.J., Fell J.W., Kwon-Chung K.J. Chapter 74—Discussion of teleomorphic and anamorphic genera of heterobasidiomycetous yeasts. In: Kurtzman C.P., Fell J.W., editors. The Yeasts. 4th ed. Elsevier; Amsterdam, The Netherlands: 1998. pp. 609–625. [[Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Yeasts&author=T.%20Boekhout&author=R.J.%20Bandoni&author=J.W.%20Fell&author=K.J.%20Kwon-Chung&publication_year=1998&)]
* 13.Fonseca Á., Boekhout T., Fell J.W. Chapter 138—Cryptococcus Vuillemin (1901) In: Kurtzman C.P., Fell J.W., Boekhout T., editors. The Yeasts. 4th ed. Elsevier; London, UK: 2011. pp. 1661–1737. [[Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Yeasts&author=%C3%81.%20Fonseca&author=T.%20Boekhout&author=J.W.%20Fell&publication_year=2011&)]
* 14.Khan Z.U., Ahmad S., Hagen F., Fell J.W., Kowshik T., Chandy R., Boekhout T. Cryptococcus randhawai sp. nov., a novel anamorphic basidiomycetous yeast isolated from tree trunk hollow of Ficus religiosa (peepal tree) from New Delhi, India. Antonie Leeuwenhoek. 2010;97:253–259. doi: 10.1007/s10482-009-9406-8. [[DOI](https://doi.org/10.1007/s10482-009-9406-8)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20091225/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antonie%20Leeuwenhoek&title=Cryptococcus%20randhawai%20sp.%20nov.,%20a%20novel%20anamorphic%20basidiomycetous%20yeast%20isolated%20from%20tree%20trunk%20hollow%20of%20Ficus%20religiosa%20(peepal%20tree)%20from%20New%20Delhi,%20India&author=Z.U.%20Khan&author=S.%20Ahmad&author=F.%20Hagen&author=J.W.%20Fell&author=T.%20Kowshik&volume=97&publication_year=2010&pages=253-259&pmid=20091225&doi=10.1007/s10482-009-9406-8&)]
* 15.Turchetti B., Buzzini P., Goretti M., Branda E., Diolaiuti G., D’Agata C., Smiraglia C., Vaughan-Martini A. Psychrophilic yeasts in glacial environments of Alpine glaciers. FEMS Microbiol. Ecol. 2008;63:73–83. doi: 10.1111/j.1574-6941.2007.00409.x. [[DOI](https://doi.org/10.1111/j.1574-6941.2007.00409.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18067577/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Ecol.&title=Psychrophilic%20yeasts%20in%20glacial%20environments%20of%20Alpine%20glaciers&author=B.%20Turchetti&author=P.%20Buzzini&author=M.%20Goretti&author=E.%20Branda&author=G.%20Diolaiuti&volume=63&publication_year=2008&pages=73-83&pmid=18067577&doi=10.1111/j.1574-6941.2007.00409.x&)]
* 16.Passoth V., Andersson A.C., Olstorpe M., Theelen B., Boekhout T., Schnürer J. Cryptococcus cerealis sp. nov. a psychrophilic yeast species isolated from fermented cereals. Antonie Leeuwenhoek. 2009;96:635–643. doi: 10.1007/s10482-009-9379-7. [[DOI](https://doi.org/10.1007/s10482-009-9379-7)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19763872/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antonie%20Leeuwenhoek&title=Cryptococcus%20cerealis%20sp.%20nov.%20a%20psychrophilic%20yeast%20species%20isolated%20from%20fermented%20cereals&author=V.%20Passoth&author=A.C.%20Andersson&author=M.%20Olstorpe&author=B.%20Theelen&author=T.%20Boekhout&volume=96&publication_year=2009&pages=635-643&pmid=19763872&doi=10.1007/s10482-009-9379-7&)]
* 17.Furman-Kuklinska K., Naumnik B., Mysliwiec M. Fungaemia due to Cryptococcus laurentii as a complication of immunosuppressive therapy—A case report. Adv. Med. Sci. 2009;54:116–119. doi: 10.2478/v10039-009-0014-7. [[DOI](https://doi.org/10.2478/v10039-009-0014-7)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19482727/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Med.%20Sci.&title=Fungaemia%20due%20to%20Cryptococcus%20laurentii%20as%20a%20complication%20of%20immunosuppressive%20therapy%E2%80%94A%20case%20report&author=K.%20Furman-Kuklinska&author=B.%20Naumnik&author=M.%20Mysliwiec&volume=54&publication_year=2009&pages=116-119&pmid=19482727&doi=10.2478/v10039-009-0014-7&)]
* 18.Rimek D., Haase G., Lück A., Casper J., Podbielski A. First Report of a Case of Meningitis Caused by Cryptococcus adeliensis in a Patient with Acute Myeloid Leukemia. J. Clin. Microbiol. 2004;42:481–483. doi: 10.1128/JCM.42.1.481-483.2004. [[DOI](https://doi.org/10.1128/JCM.42.1.481-483.2004)] [[PMC free article](/articles/PMC321734/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/14715811/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Microbiol.&title=First%20Report%20of%20a%20Case%20of%20Meningitis%20Caused%20by%20Cryptococcus%20adeliensis%20in%20a%20Patient%20with%20Acute%20Myeloid%20Leukemia&author=D.%20Rimek&author=G.%20Haase&author=A.%20L%C3%BCck&author=J.%20Casper&author=A.%20Podbielski&volume=42&publication_year=2004&pages=481-483&pmid=14715811&doi=10.1128/JCM.42.1.481-483.2004&)]
* 19.Gerstein A.C., Fu M.S., Mukaremera L., Li Z., Ormerod K.L., Fraser J.A., Berman J., Nielsen K. Polyploid titan cells produce haploid and aneuploid progeny to promote stress adaptation. mBio. 2015;6:e01340-15. doi: 10.1128/mBio.01340-15. [[DOI](https://doi.org/10.1128/mBio.01340-15)] [[PMC free article](/articles/PMC4620463/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26463162/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=mBio&title=Polyploid%20titan%20cells%20produce%20haploid%20and%20aneuploid%20progeny%20to%20promote%20stress%20adaptation&author=A.C.%20Gerstein&author=M.S.%20Fu&author=L.%20Mukaremera&author=Z.%20Li&author=K.L.%20Ormerod&volume=6&publication_year=2015&pages=e01340-15&pmid=26463162&doi=10.1128/mBio.01340-15&)]
* 20.Dambuza I.M., Drake T., Chapuis A., Zhou X., Correia J., Taylor-Smith L., LeGrave N., Rasmussen T., Fisher M.C., Bicanic T., et al. The Cryptococcus neoformans Titan cell is an inducible and regulated morphotype underlying pathogenesis. PLoS Pathog. 2018;14:e1006978. doi: 10.1371/journal.ppat.1006978. [[DOI](https://doi.org/10.1371/journal.ppat.1006978)] [[PMC free article](/articles/PMC5959070/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29775474/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&title=The%20Cryptococcus%20neoformans%20Titan%20cell%20is%20an%20inducible%20and%20regulated%20morphotype%20underlying%20pathogenesis&author=I.M.%20Dambuza&author=T.%20Drake&author=A.%20Chapuis&author=X.%20Zhou&author=J.%20Correia&volume=14&publication_year=2018&pages=e1006978&pmid=29775474&doi=10.1371/journal.ppat.1006978&)]
* 21.Dyląg M., Colon-Reyes R.J., Kozubowski L. Titan cell formation is unique to Cryptococcus species complex. Virulence. 2020;11:719–729. doi: 10.1080/21505594.2020.1772657. [[DOI](https://doi.org/10.1080/21505594.2020.1772657)] [[PMC free article](/articles/PMC7549989/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32498590/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Virulence&title=Titan%20cell%20formation%20is%20unique%20to%20Cryptococcus%20species%20complex&author=M.%20Dyl%C4%85g&author=R.J.%20Colon-Reyes&author=L.%20Kozubowski&volume=11&publication_year=2020&pages=719-729&pmid=32498590&doi=10.1080/21505594.2020.1772657&)]
* 22.Checinska A., Probst A.J., Vaishampayan P., White J.R., Kumar D., Stepanov V.G., Fox G.E., Nilsson H.R., Pierson D.L., Perry J., et al. Microbiomes of the dust particles collected from the International Space Station and Spacecraft Assembly Facilities. Microbiome. 2015;3:50. doi: 10.1186/s40168-015-0116-3. [[DOI](https://doi.org/10.1186/s40168-015-0116-3)] [[PMC free article](/articles/PMC4624184/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26502721/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Microbiomes%20of%20the%20dust%20particles%20collected%20from%20the%20International%20Space%20Station%20and%20Spacecraft%20Assembly%20Facilities&author=A.%20Checinska&author=A.J.%20Probst&author=P.%20Vaishampayan&author=J.R.%20White&author=D.%20Kumar&volume=3&publication_year=2015&pages=50&pmid=26502721&doi=10.1186/s40168-015-0116-3&)]
* 23.Patel R.K., Jain M. NGS QC Toolkit: A toolkit for quality control of next generation sequencing data. PLoS ONE. 2012;7:e30619. doi: 10.1371/journal.pone.0030619. [[DOI](https://doi.org/10.1371/journal.pone.0030619)] [[PMC free article](/articles/PMC3270013/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22312429/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=NGS%20QC%20Toolkit:%20A%20toolkit%20for%20quality%20control%20of%20next%20generation%20sequencing%20data&author=R.K.%20Patel&author=M.%20Jain&volume=7&publication_year=2012&pages=e30619&pmid=22312429&doi=10.1371/journal.pone.0030619&)]
* 24.Bankevich A., Nurk S., Antipov D., Gurevich A.A., Dvorkin M., Kulikov A.S., Lesin V.M., Nikolenko S.I., Pham S., Prjibelski A.D., et al. SPAdes: A new genome assembly algorithm and its applications to single-cell sequencing. J. Comput. Biol. 2012;19:455–477. doi: 10.1089/cmb.2012.0021. [[DOI](https://doi.org/10.1089/cmb.2012.0021)] [[PMC free article](/articles/PMC3342519/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Biol.&title=SPAdes:%20A%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&author=A.%20Bankevich&author=S.%20Nurk&author=D.%20Antipov&author=A.A.%20Gurevich&author=M.%20Dvorkin&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)]
* 25.Kumar S., Stecher G., Tamura K. MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets. Mol. Biol. Evol. 2016;33:1870–1874. doi: 10.1093/molbev/msw054. [[DOI](https://doi.org/10.1093/molbev/msw054)] [[PMC free article](/articles/PMC8210823/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27004904/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&title=MEGA7:%20Molecular%20Evolutionary%20Genetics%20Analysis%20Version%207.0%20for%20Bigger%20Datasets&author=S.%20Kumar&author=G.%20Stecher&author=K.%20Tamura&volume=33&publication_year=2016&pages=1870-1874&pmid=27004904&doi=10.1093/molbev/msw054&)]
* 26.Hommel B., Mukaremera L., Cordero R.J.B., Coelho C., Desjardins C.A., Sturny-Leclère A., Janbon G., Perfect J.R., Fraser J.A., Casadevall A., et al. Titan cells formation in Cryptococcus neoformans is finely tuned by environmental conditions and modulated by positive and negative genetic regulators. PLoS Pathog. 2018;14:e1006982. doi: 10.1371/journal.ppat.1006982. [[DOI](https://doi.org/10.1371/journal.ppat.1006982)] [[PMC free article](/articles/PMC5959062/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29775480/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&title=Titan%20cells%20formation%20in%20Cryptococcus%20neoformans%20is%20finely%20tuned%20by%20environmental%20conditions%20and%20modulated%20by%20positive%20and%20negative%20genetic%20regulators&author=B.%20Hommel&author=L.%20Mukaremera&author=R.J.B.%20Cordero&author=C.%20Coelho&author=C.A.%20Desjardins&volume=14&publication_year=2018&pages=e1006982&pmid=29775480&doi=10.1371/journal.ppat.1006982&)]
* 27.Greetham D., Wimalasena T., Kerruish D.W., Brindley S., Ibbett R.N., Linforth R.L., Tucker G., Phister T.G., Smart K.A. Development of a phenotypic assay for characterisation of ethanologenic yeast strain sensitivity to inhibitors released from lignocellulosic feedstocks. J. Ind. Microbiol. Biotechnol. 2014;41:931–945. doi: 10.1007/s10295-014-1431-6. [[DOI](https://doi.org/10.1007/s10295-014-1431-6)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24664516/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Ind.%20Microbiol.%20Biotechnol.&title=Development%20of%20a%20phenotypic%20assay%20for%20characterisation%20of%20ethanologenic%20yeast%20strain%20sensitivity%20to%20inhibitors%20released%20from%20lignocellulosic%20feedstocks&author=D.%20Greetham&author=T.%20Wimalasena&author=D.W.%20Kerruish&author=S.%20Brindley&author=R.N.%20Ibbett&volume=41&publication_year=2014&pages=931-945&pmid=24664516&doi=10.1007/s10295-014-1431-6&)]
* 28.Schneider C.A., Rasband W.S., Eliceiri K.W. NIH Image to ImageJ: 25 years of image analysis. Nat. Methods. 2012;9:671–675. doi: 10.1038/nmeth.2089. [[DOI](https://doi.org/10.1038/nmeth.2089)] [[PMC free article](/articles/PMC5554542/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22930834/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=NIH%20Image%20to%20ImageJ:%2025%20years%20of%20image%20analysis&author=C.A.%20Schneider&author=W.S.%20Rasband&author=K.W.%20Eliceiri&volume=9&publication_year=2012&pages=671-675&pmid=22930834&doi=10.1038/nmeth.2089&)]
* 29.Nizovoy P., Bellora N., Haridas S., Sun H., Daum C., Barry K., Grigoriev I.V., Libkind D., Connell L.B., Moline M. Unique genomic traits for cold adaptation in Naganishia vishniacii, a polyextremophile yeast isolated from Antarctica. FEMS Yeast Res. 2021;21:foaa056. doi: 10.1093/femsyr/foaa056. [[DOI](https://doi.org/10.1093/femsyr/foaa056)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33232451/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Yeast%20Res.&title=Unique%20genomic%20traits%20for%20cold%20adaptation%20in%20Naganishia%20vishniacii,%20a%20polyextremophile%20yeast%20isolated%20from%20Antarctica&author=P.%20Nizovoy&author=N.%20Bellora&author=S.%20Haridas&author=H.%20Sun&author=C.%20Daum&volume=21&publication_year=2021&pages=foaa056&pmid=33232451&doi=10.1093/femsyr/foaa056&)]
* 30.Bellora N., Moliné M., David-Palma M., Coelho M.A., Hittinger C.T., Sampaio J.P., Gonçalves P., Libkind D. Comparative genomics provides new insights into the diversity, physiology, and sexuality of the only industrially exploited tremellomycete: Phaffia rhodozyma. BMC Genom. 2016;17:901. doi: 10.1186/s12864-016-3244-7. [[DOI](https://doi.org/10.1186/s12864-016-3244-7)] [[PMC free article](/articles/PMC5103461/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27829365/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&title=Comparative%20genomics%20provides%20new%20insights%20into%20the%20diversity,%20physiology,%20and%20sexuality%20of%20the%20only%20industrially%20exploited%20tremellomycete:%20Phaffia%20rhodozyma&author=N.%20Bellora&author=M.%20Molin%C3%A9&author=M.%20David-Palma&author=M.A.%20Coelho&author=C.T.%20Hittinger&volume=17&publication_year=2016&pages=901&pmid=27829365&doi=10.1186/s12864-016-3244-7&)]
* 31.Cantarel B.L., Korf I., Robb S.M., Parra G., Ross E., Moore B., Holt C., Alvarado A.S., Yandell M. MAKER: An easy-to-use annotation pipeline designed for emerging model organism genomes. Genome Res. 2008;18:188–196. doi: 10.1101/gr.6743907. [[DOI](https://doi.org/10.1101/gr.6743907)] [[PMC free article](/articles/PMC2134774/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18025269/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=MAKER:%20An%20easy-to-use%20annotation%20pipeline%20designed%20for%20emerging%20model%20organism%20genomes&author=B.L.%20Cantarel&author=I.%20Korf&author=S.M.%20Robb&author=G.%20Parra&author=E.%20Ross&volume=18&publication_year=2008&pages=188-196&pmid=18025269&doi=10.1101/gr.6743907&)]
* 32.UniProt UniProt: The universal protein knowledgebase in 2021. Nucleic Acids Res. 2021;49:D480–D489. doi: 10.1093/nar/gkaa1100. [[DOI](https://doi.org/10.1093/nar/gkaa1100)] [[PMC free article](/articles/PMC7778908/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33237286/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=UniProt:%20The%20universal%20protein%20knowledgebase%20in%202021&volume=49&publication_year=2021&pages=D480-D489&pmid=33237286&doi=10.1093/nar/gkaa1100&)]
* 33.Smit R.H.A. RepeatModeler. [(accessed on 5 December 2021)]. Available online: <http://www.repeatmasker.org/RepeatModeler/>
* 34.Chen M.M., Lin H., Chiang L.M., Childers C.P., Poelchau M.F. The GFF3toolkit: QC and Merge Pipeline for Genome Annotation. Methods Mol. Biol. 2019;1858:75–87. doi: 10.1007/978-1-4939-8775-7\_7. [[DOI](https://doi.org/10.1007/978-1-4939-8775-7_7)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30414112/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&title=The%20GFF3toolkit:%20QC%20and%20Merge%20Pipeline%20for%20Genome%20Annotation&author=M.M.%20Chen&author=H.%20Lin&author=L.M.%20Chiang&author=C.P.%20Childers&author=M.F.%20Poelchau&volume=1858&publication_year=2019&pages=75-87&pmid=30414112&doi=10.1007/978-1-4939-8775-7_7&)]
* 35.Korf I. Gene finding in novel genomes. BMC Bioinform. 2004;5:59. doi: 10.1186/1471-2105-5-59. [[DOI](https://doi.org/10.1186/1471-2105-5-59)] [[PMC free article](/articles/PMC421630/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15144565/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinform.&title=Gene%20finding%20in%20novel%20genomes&author=I.%20Korf&volume=5&publication_year=2004&pages=59&pmid=15144565&doi=10.1186/1471-2105-5-59&)]
* 36.Stanke M., Keller O., Gunduz I., Hayes A., Waack S., Morgenstern B. AUGUSTUS: Ab initio prediction of alternative transcripts. Nucleic Acids Res. 2006;34:W435–W439. doi: 10.1093/nar/gkl200. [[DOI](https://doi.org/10.1093/nar/gkl200)] [[PMC free article](/articles/PMC1538822/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16845043/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=AUGUSTUS:%20Ab%20initio%20prediction%20of%20alternative%20transcripts&author=M.%20Stanke&author=O.%20Keller&author=I.%20Gunduz&author=A.%20Hayes&author=S.%20Waack&volume=34&publication_year=2006&pages=W435-W439&pmid=16845043&doi=10.1093/nar/gkl200&)]
* 37.Seppey M., Manni M., Zdobnov E.M. BUSCO: Assessing Genome Assembly and Annotation Completeness. Methods Mol. Biol. 2019;1962:227–245. doi: 10.1007/978-1-4939-9173-0\_14. [[DOI](https://doi.org/10.1007/978-1-4939-9173-0_14)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31020564/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&title=BUSCO:%20Assessing%20Genome%20Assembly%20and%20Annotation%20Completeness&author=M.%20Seppey&author=M.%20Manni&author=E.M.%20Zdobnov&volume=1962&publication_year=2019&pages=227-245&pmid=31020564&doi=10.1007/978-1-4939-9173-0_14&)]
* 38.Card D. Maker\_Genome\_Annotation.md. [(accessed on 5 December 2021)]. Available online: <https://gist.github.com/darencard/bb1001ac1532dd4225b030cf0cd61ce2>.
* 39.Lodato P., Alcaíno J., Barahona S., Niklitschek M., Carmona M., Wozniak A., Baeza M., Jiménez A., Cifuentes V. Expression of the carotenoid biosynthesis genes in Xanthophyllomyces dendrorhous. Biol. Res. 2007;40:73–84. doi: 10.4067/S0716-97602007000100008. [[DOI](https://doi.org/10.4067/S0716-97602007000100008)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17657357/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol.%20Res.&title=Expression%20of%20the%20carotenoid%20biosynthesis%20genes%20in%20Xanthophyllomyces%20dendrorhous&author=P.%20Lodato&author=J.%20Alca%C3%ADno&author=S.%20Barahona&author=M.%20Niklitschek&author=M.%20Carmona&volume=40&publication_year=2007&pages=73-84&pmid=17657357&doi=10.4067/S0716-97602007000100008&)]
* 40.Alcaino J., Barahona S., Carmona M., Lozano C., Marcoleta A., Niklitschek M., Sepulveda D., Baeza M., Cifuentes V. Cloning of the cytochrome p450 reductase (crtR) gene and its involvement in the astaxanthin biosynthesis of Xanthophyllomyces dendrorhous. BMC Microbiol. 2008;8:169. doi: 10.1186/1471-2180-8-169. [[DOI](https://doi.org/10.1186/1471-2180-8-169)] [[PMC free article](/articles/PMC2575211/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18837978/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol.&title=Cloning%20of%20the%20cytochrome%20p450%20reductase%20(crtR)%20gene%20and%20its%20involvement%20in%20the%20astaxanthin%20biosynthesis%20of%20Xanthophyllomyces%20dendrorhous&author=J.%20Alcaino&author=S.%20Barahona&author=M.%20Carmona&author=C.%20Lozano&author=A.%20Marcoleta&volume=8&publication_year=2008&pages=169&pmid=18837978&doi=10.1186/1471-2180-8-169&)]
* 41.Alcaíno J., Romero I., Niklitschek M., Sepúlveda D., Rojas M.C., Baeza M., Cifuentes V. Functional characterization of the Xanthophyllomyces dendrorhous farnesyl pyrophosphate synthase and geranylgeranyl pyrophosphate synthase encoding genes that are involved in the synthesis of isoprenoid precursors. PLoS ONE. 2014;9:e96626. doi: 10.1371/journal.pone.0096626. [[DOI](https://doi.org/10.1371/journal.pone.0096626)] [[PMC free article](/articles/PMC4010515/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24796858/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Functional%20characterization%20of%20the%20Xanthophyllomyces%20dendrorhous%20farnesyl%20pyrophosphate%20synthase%20and%20geranylgeranyl%20pyrophosphate%20synthase%20encoding%20genes%20that%20are%20involved%20in%20the%20synthesis%20of%20isoprenoid%20precursors&author=J.%20Alca%C3%ADno&author=I.%20Romero&author=M.%20Niklitschek&author=D.%20Sep%C3%BAlveda&author=M.C.%20Rojas&volume=9&publication_year=2014&pages=e96626&pmid=24796858&doi=10.1371/journal.pone.0096626&)]
* 42.Kajiwara S., Fraser P.D., Kondo K., Misawa N. Expression of an exogenous isopentenyl diphosphate isomerase gene enhances isoprenoid biosynthesis in Escherichia coli. Biochem. J. 1997;324:421–426. doi: 10.1042/bj3240421. [[DOI](https://doi.org/10.1042/bj3240421)] [[PMC free article](/articles/PMC1218447/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/9182699/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&title=Expression%20of%20an%20exogenous%20isopentenyl%20diphosphate%20isomerase%20gene%20enhances%20isoprenoid%20biosynthesis%20in%20Escherichia%20coli&author=S.%20Kajiwara&author=P.D.%20Fraser&author=K.%20Kondo&author=N.%20Misawa&volume=324&publication_year=1997&pages=421-426&pmid=9182699&doi=10.1042/bj3240421&)]
* 43.Goffeau A., Barrell B.G., Bussey H., Davis R.W., Dujon B., Feldmann H., Galibert F., Hoheisel J.D., Jacq C., Johnston M., et al. Life with 6000 genes. Science. 1996;274:546–567. doi: 10.1126/science.274.5287.546. [[DOI](https://doi.org/10.1126/science.274.5287.546)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/8849441/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Life%20with%206000%20genes&author=A.%20Goffeau&author=B.G.%20Barrell&author=H.%20Bussey&author=R.W.%20Davis&author=B.%20Dujon&volume=274&publication_year=1996&pages=546-567&pmid=8849441&doi=10.1126/science.274.5287.546&)]
* 44.Jacq C., Alt-Mörbe J., Andre B., Arnold W., Bahr A., Ballesta J.P., Bargues M., Baron L., Becker A., Biteau N., et al. The nucleotide sequence of Saccharomyces cerevisiae chromosome IV. Nature. 1997;387:75–78. doi: 10.1038/387s075. [[DOI](https://doi.org/10.1038/387s075)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/9169867/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20nucleotide%20sequence%20of%20Saccharomyces%20cerevisiae%20chromosome%20IV&author=C.%20Jacq&author=J.%20Alt-M%C3%B6rbe&author=B.%20Andre&author=W.%20Arnold&author=A.%20Bahr&volume=387&publication_year=1997&pages=75-78&pmid=9169867&doi=10.1038/387s075&)]
* 45.Wood V., Gwilliam R., Rajandream M.A., Lyne M., Lyne R., Stewart A., Sgouros J., Peat N., Hayles J., Baker S., et al. The genome sequence of Schizosaccharomyces pombe. Nature. 2002;415:871–880. doi: 10.1038/nature724. [[DOI](https://doi.org/10.1038/nature724)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11859360/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20genome%20sequence%20of%20Schizosaccharomyces%20pombe&author=V.%20Wood&author=R.%20Gwilliam&author=M.A.%20Rajandream&author=M.%20Lyne&author=R.%20Lyne&volume=415&publication_year=2002&pages=871-880&pmid=11859360&doi=10.1038/nature724&)]
* 46.Lodato P., Alcaino J., Barahona S., Retamales P., Cifuentes V. Alternative splicing of transcripts from crtI and crtYB genes of Xanthophyllomyces dendrorhous. Appl. Environ. Microbiol. 2003;69:4676–4682. doi: 10.1128/AEM.69.8.4676-4682.2003. [[DOI](https://doi.org/10.1128/AEM.69.8.4676-4682.2003)] [[PMC free article](/articles/PMC169127/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12902257/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=Alternative%20splicing%20of%20transcripts%20from%20crtI%20and%20crtYB%20genes%20of%20Xanthophyllomyces%20dendrorhous&author=P.%20Lodato&author=J.%20Alcaino&author=S.%20Barahona&author=P.%20Retamales&author=V.%20Cifuentes&volume=69&publication_year=2003&pages=4676-4682&pmid=12902257&doi=10.1128/AEM.69.8.4676-4682.2003&)]
* 47.Jung K.-W., Yang D.-H., Kim M.-K., Seo H.S., Lim S., Bahn Y.-S. Unraveling Fungal Radiation Resistance Regulatory Networks through the Genome-Wide Transcriptome and Genetic Analyses of Cryptococcus neoformans. mBio. 2016;7:e01483-16. doi: 10.1128/mBio.01483-16. [[DOI](https://doi.org/10.1128/mBio.01483-16)] [[PMC free article](/articles/PMC5137497/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27899501/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=mBio&title=Unraveling%20Fungal%20Radiation%20Resistance%20Regulatory%20Networks%20through%20the%20Genome-Wide%20Transcriptome%20and%20Genetic%20Analyses%20of%20Cryptococcus%20neoformans&author=K.-W.%20Jung&author=D.-H.%20Yang&author=M.-K.%20Kim&author=H.S.%20Seo&author=S.%20Lim&volume=7&publication_year=2016&pages=e01483-16&pmid=27899501&doi=10.1128/mBio.01483-16&)]
* 48.Lieberman H.B., Hopkins K.M. Schizosaccharomyces malidevorans and Sz. octosporus homologues of Sz. pombe rad9, a gene that mediates radioresistance and cell-cycle progression. Gene. 1994;150:281–286. doi: 10.1016/0378-1119(94)90438-3. [[DOI](https://doi.org/10.1016/0378-1119%2894%2990438-3)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/7821792/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Schizosaccharomyces%20malidevorans%20and%20Sz.%20octosporus%20homologues%20of%20Sz.%20pombe%20rad9,%20a%20gene%20that%20mediates%20radioresistance%20and%20cell-cycle%20progression&author=H.B.%20Lieberman&author=K.M.%20Hopkins&volume=150&publication_year=1994&pages=281-286&pmid=7821792&doi=10.1016/0378-1119(94)90438-3&)]
* 49.Gourinath S., Degenhardt M., Eschenburg S., Moore K., Delucas L.J., Betzel C., Singh T.P. Mercury induced modifications in the stereochemistry of the active site through Cys-73 in a serine protease—Crystal structure of the complex of a partially modified proteinase K with mercury at 1.8 A resolution. Indian J. Biochem. Biophys. 2001;38:298–302. [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11886076/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20J.%20Biochem.%20Biophys.&title=Mercury%20induced%20modifications%20in%20the%20stereochemistry%20of%20the%20active%20site%20through%20Cys-73%20in%20a%20serine%20protease%E2%80%94Crystal%20structure%20of%20the%20complex%20of%20a%20partially%20modified%20proteinase%20K%20with%20mercury%20at%201.8%20A%20resolution&author=S.%20Gourinath&author=M.%20Degenhardt&author=S.%20Eschenburg&author=K.%20Moore&author=L.J.%20Delucas&volume=38&publication_year=2001&pages=298-302&pmid=11886076&)]
* 50.Betzel C., Gourinath S., Kumar P., Kaur P., Perbandt M., Eschenburg S., Singh T.P. Structure of a serine protease proteinase K from Tritirachium album limber at 0.98 A resolution. Biochemistry. 2001;40:3080–3088. doi: 10.1021/bi002538n. [[DOI](https://doi.org/10.1021/bi002538n)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11258922/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Structure%20of%20a%20serine%20protease%20proteinase%20K%20from%20Tritirachium%20album%20limber%20at%200.98%20A%20resolution&author=C.%20Betzel&author=S.%20Gourinath&author=P.%20Kumar&author=P.%20Kaur&author=M.%20Perbandt&volume=40&publication_year=2001&pages=3080-3088&pmid=11258922&doi=10.1021/bi002538n&)]
* 51.Ielasi F.S., Goyal P., Sleutel M., Wohlkonig A., Willaert R.G. The mannose-specific lectin domains of Flo1p from Saccharomyces cerevisiae and Lg-Flo1p from S. pastorianus: Crystallization and preliminary X-ray diffraction analysis of the adhesin-carbohydrate complexes. Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun. 2013;69:779–782. doi: 10.1107/S1744309113015030. [[DOI](https://doi.org/10.1107/S1744309113015030)] [[PMC free article](/articles/PMC3702324/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23832207/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20Sect.%20F%20Struct.%20Biol.%20Cryst.%20Commun.&title=The%20mannose-specific%20lectin%20domains%20of%20Flo1p%20from%20Saccharomyces%20cerevisiae%20and%20Lg-Flo1p%20from%20S.%20pastorianus:%20Crystallization%20and%20preliminary%20X-ray%20diffraction%20analysis%20of%20the%20adhesin-carbohydrate%20complexes&author=F.S.%20Ielasi&author=P.%20Goyal&author=M.%20Sleutel&author=A.%20Wohlkonig&author=R.G.%20Willaert&volume=69&publication_year=2013&pages=779-782&pmid=23832207&doi=10.1107/S1744309113015030&)]
* 52.Murray J.M., Carr A.M., Lehmann A.R., Watts F.Z. Cloning and characterisation of the rad9 DNA repair gene from Schizosaccharomyces pombe. Nucleic Acids Res. 1991;19:3525–3531. doi: 10.1093/nar/19.13.3525. [[DOI](https://doi.org/10.1093/nar/19.13.3525)] [[PMC free article](/articles/PMC328375/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/1852603/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Cloning%20and%20characterisation%20of%20the%20rad9%20DNA%20repair%20gene%20from%20Schizosaccharomyces%20pombe&author=J.M.%20Murray&author=A.M.%20Carr&author=A.R.%20Lehmann&author=F.Z.%20Watts&volume=19&publication_year=1991&pages=3525-3531&pmid=1852603&doi=10.1093/nar/19.13.3525&)]
* 53.Johnston M., Andrews S., Brinkman R., Cooper J., Ding H., Dover J., Du Z., Favello A., Fulton L., Gattung S. Complete nucleotide sequence of Saccharomyces cerevisiae chromosome VIII. Science. 1994;265:2077–2082. doi: 10.1126/science.8091229. [[DOI](https://doi.org/10.1126/science.8091229)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/8091229/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Complete%20nucleotide%20sequence%20of%20Saccharomyces%20cerevisiae%20chromosome%20VIII&author=M.%20Johnston&author=S.%20Andrews&author=R.%20Brinkman&author=J.%20Cooper&author=H.%20Ding&volume=265&publication_year=1994&pages=2077-2082&pmid=8091229&doi=10.1126/science.8091229&)]
* 54.Diederich B., Wilkinson J.F., Magnin T., Najafi S.M.A., Errington J., Yudkin M. Role of interactions between SpoIIAA and SpoIIAB in regulating cell-specific transcription factor sF of B. subtilis. Genes Dev. 1994;8:2653–2663. doi: 10.1101/gad.8.21.2653. [[DOI](https://doi.org/10.1101/gad.8.21.2653)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/7958923/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&title=Role%20of%20interactions%20between%20SpoIIAA%20and%20SpoIIAB%20in%20regulating%20cell-specific%20transcription%20factor%20sF%20of%20B.%20subtilis&author=B.%20Diederich&author=J.F.%20Wilkinson&author=T.%20Magnin&author=S.M.A.%20Najafi&author=J.%20Errington&volume=8&publication_year=1994&pages=2653-2663&pmid=7958923&doi=10.1101/gad.8.21.2653&)]
* 55.Jones T., Federspiel N.A., Chibana H., Dungan J., Kalman S., Magee B.B., Newport G., Thorstenson Y.R., Agabian N., Magee P.T., et al. The diploid genome sequence of Candida albicans. Proc. Natl. Acad. Sci. USA. 2004;101:7329–7334. doi: 10.1073/pnas.0401648101. [[DOI](https://doi.org/10.1073/pnas.0401648101)] [[PMC free article](/articles/PMC409918/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15123810/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=The%20diploid%20genome%20sequence%20of%20Candida%20albicans&author=T.%20Jones&author=N.A.%20Federspiel&author=H.%20Chibana&author=J.%20Dungan&author=S.%20Kalman&volume=101&publication_year=2004&pages=7329-7334&pmid=15123810&doi=10.1073/pnas.0401648101&)]
* 56.Jones P., Binns D., Chang H.Y., Fraser M., Li W., McAnulla C., McWilliam H., Maslen J., Mitchell A., Nuka G., et al. InterProScan 5: Genome-scale protein function classification. Bioinformatics. 2014;30:1236–1240. doi: 10.1093/bioinformatics/btu031. [[DOI](https://doi.org/10.1093/bioinformatics/btu031)] [[PMC free article](/articles/PMC3998142/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24451626/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=InterProScan%205:%20Genome-scale%20protein%20function%20classification&author=P.%20Jones&author=D.%20Binns&author=H.Y.%20Chang&author=M.%20Fraser&author=W.%20Li&volume=30&publication_year=2014&pages=1236-1240&pmid=24451626&doi=10.1093/bioinformatics/btu031&)]
* 57.Conesa A., Götz S., García-Gómez J.M., Terol J., Talón M., Robles M. Blast2GO: A universal tool for annotation, visualization and analysis in functional genomics research. Bioinformatics. 2005;21:3674–3676. doi: 10.1093/bioinformatics/bti610. [[DOI](https://doi.org/10.1093/bioinformatics/bti610)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16081474/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Blast2GO:%20A%20universal%20tool%20for%20annotation,%20visualization%20and%20analysis%20in%20functional%20genomics%20research&author=A.%20Conesa&author=S.%20G%C3%B6tz&author=J.M.%20Garc%C3%ADa-G%C3%B3mez&author=J.%20Terol&author=M.%20Tal%C3%B3n&volume=21&publication_year=2005&pages=3674-3676&pmid=16081474&doi=10.1093/bioinformatics/bti610&)]
* 58.Supek F., Bošnjak M., Škunca N., Šmuc T. REVIGO Summarizes and Visualizes Long Lists of Gene Ontology Terms. PLoS ONE. 2011;6:e21800. doi: 10.1371/journal.pone.0021800. [[DOI](https://doi.org/10.1371/journal.pone.0021800)] [[PMC free article](/articles/PMC3138752/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21789182/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=REVIGO%20Summarizes%20and%20Visualizes%20Long%20Lists%20of%20Gene%20Ontology%20Terms&author=F.%20Supek&author=M.%20Bo%C5%A1njak&author=N.%20%C5%A0kunca&author=T.%20%C5%A0muc&volume=6&publication_year=2011&pages=e21800&pmid=21789182&doi=10.1371/journal.pone.0021800&)]
* 59.Singh N.K., Wood J.M., Karouia F., Venkateswaran K. Succession and persistence of microbial communities and antimicrobial resistance genes associated with International Space Station environmental surfaces. Microbiome. 2018;6:214. doi: 10.1186/s40168-018-0609-y. [[DOI](https://doi.org/10.1186/s40168-018-0609-y)] [[PMC free article](/articles/PMC6234677/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30424821/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Succession%20and%20persistence%20of%20microbial%20communities%20and%20antimicrobial%20resistance%20genes%20associated%20with%20International%20Space%20Station%20environmental%20surfaces&author=N.K.%20Singh&author=J.M.%20Wood&author=F.%20Karouia&author=K.%20Venkateswaran&volume=6&publication_year=2018&pages=214&pmid=30424821&doi=10.1186/s40168-018-0609-y&)]
* 60.Shanks R.M., Stella N.A., Kalivoda E.J., Doe M.R., O’Dee D.M., Lathrop K.L., Guo F.L., Nau G.J. A Serratia marcescens OxyR homolog mediates surface attachment and biofilm formation. J. Bacteriol. 2007;189:7262–7272. doi: 10.1128/JB.00859-07. [[DOI](https://doi.org/10.1128/JB.00859-07)] [[PMC free article](/articles/PMC2168423/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17675374/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=A%20Serratia%20marcescens%20OxyR%20homolog%20mediates%20surface%20attachment%20and%20biofilm%20formation&author=R.M.%20Shanks&author=N.A.%20Stella&author=E.J.%20Kalivoda&author=M.R.%20Doe&author=D.M.%20O%E2%80%99Dee&volume=189&publication_year=2007&pages=7262-7272&pmid=17675374&doi=10.1128/JB.00859-07&)]
* 61.Schembri M.A., Blom J., Krogfelt K.A., Klemm P. Capsule and fimbria interaction in Klebsiella pneumoniae. Infect. Immun. 2005;73:4626–4633. doi: 10.1128/IAI.73.8.4626-4633.2005. [[DOI](https://doi.org/10.1128/IAI.73.8.4626-4633.2005)] [[PMC free article](/articles/PMC1201234/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16040975/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&title=Capsule%20and%20fimbria%20interaction%20in%20Klebsiella%20pneumoniae&author=M.A.%20Schembri&author=J.%20Blom&author=K.A.%20Krogfelt&author=P.%20Klemm&volume=73&publication_year=2005&pages=4626-4633&pmid=16040975&doi=10.1128/IAI.73.8.4626-4633.2005&)]
* 62.Li S.C., Kane P.M. The yeast lysosome-like vacuole: Endpoint and crossroads. Biochim. Biophys. Acta. 2009;1793:650–663. doi: 10.1016/j.bbamcr.2008.08.003. [[DOI](https://doi.org/10.1016/j.bbamcr.2008.08.003)] [[PMC free article](/articles/PMC2906225/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18786576/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&title=The%20yeast%20lysosome-like%20vacuole:%20Endpoint%20and%20crossroads&author=S.C.%20Li&author=P.M.%20Kane&volume=1793&publication_year=2009&pages=650-663&pmid=18786576&doi=10.1016/j.bbamcr.2008.08.003&)]
* 63.Thumm M. Structure and function of the yeast vacuole and its role in autophagy. Microsc. Res. Tech. 2000;51:563–572. doi: 10.1002/1097-0029(20001215)51:6&#x0003c;563::AID-JEMT6&#x0003e;3.0.CO;2-8. [[DOI](https://doi.org/10.1002/1097-0029%2820001215%2951%3A6%26#x0003c;563::AID-JEMT6&#x0003e;3.0.CO;2-8)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11169858/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microsc.%20Res.%20Tech.&title=Structure%20and%20function%20of%20the%20yeast%20vacuole%20and%20its%20role%20in%20autophagy&author=M.%20Thumm&volume=51&publication_year=2000&pages=563-572&pmid=11169858&doi=10.1002/1097-0029(20001215)51:6&#x0003c;563::AID-JEMT6&#x0003e;3.0.CO;2-8&)]
* 64.Weisman L.S. Yeast vacuole inheritance and dynamics. Annu. Rev. Genet. 2003;37:435–460. doi: 10.1146/annurev.genet.37.050203.103207. [[DOI](https://doi.org/10.1146/annurev.genet.37.050203.103207)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/14616069/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Genet.&title=Yeast%20vacuole%20inheritance%20and%20dynamics&author=L.S.%20Weisman&volume=37&publication_year=2003&pages=435-460&pmid=14616069&doi=10.1146/annurev.genet.37.050203.103207&)]
* 65.Okagaki L.H., Strain A.K., Nielsen J.N., Charlier C., Baltes N.J., Chrétien F., Heitman J., Dromer F., Nielsen K. Cryptococcal cell morphology affects host cell interactions and pathogenicity. PLoS Pathog. 2010;6:e1000953. doi: 10.1371/annotation/1b59fd9e-9ac9-4ea8-a083-14c413c80b03. [[DOI](https://doi.org/10.1371/annotation/1b59fd9e-9ac9-4ea8-a083-14c413c80b03)] [[PMC free article](/articles/PMC2887476/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20585559/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&title=Cryptococcal%20cell%20morphology%20affects%20host%20cell%20interactions%20and%20pathogenicity&author=L.H.%20Okagaki&author=A.K.%20Strain&author=J.N.%20Nielsen&author=C.%20Charlier&author=N.J.%20Baltes&volume=6&publication_year=2010&pages=e1000953&pmid=20585559&doi=10.1371/annotation/1b59fd9e-9ac9-4ea8-a083-14c413c80b03&)]
* 66.Zaragoza O., Nielsen K. Titan cells in Cryptococcus neoformans: Cells with a giant impact. Curr. Opin. Microbiol. 2013;16:409–413. doi: 10.1016/j.mib.2013.03.006. [[DOI](https://doi.org/10.1016/j.mib.2013.03.006)] [[PMC free article](/articles/PMC3723695/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23588027/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Microbiol.&title=Titan%20cells%20in%20Cryptococcus%20neoformans:%20Cells%20with%20a%20giant%20impact&author=O.%20Zaragoza&author=K.%20Nielsen&volume=16&publication_year=2013&pages=409-413&pmid=23588027&doi=10.1016/j.mib.2013.03.006&)]
* 67.Crabtree J.N., Okagaki L.H., Wiesner D.L., Strain A.K., Nielsen J.N., Nielsen K. Titan cell production enhances the virulence of Cryptococcus neoformans. Infect. Immun. 2012;80:3776–3785. doi: 10.1128/IAI.00507-12. [[DOI](https://doi.org/10.1128/IAI.00507-12)] [[PMC free article](/articles/PMC3486048/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22890995/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&title=Titan%20cell%20production%20enhances%20the%20virulence%20of%20Cryptococcus%20neoformans&author=J.N.%20Crabtree&author=L.H.%20Okagaki&author=D.L.%20Wiesner&author=A.K.%20Strain&author=J.N.%20Nielsen&volume=80&publication_year=2012&pages=3776-3785&pmid=22890995&doi=10.1128/IAI.00507-12&)]
* 68.Quevillon E., Silventoinen V., Pillai S., Harte N., Mulder N., Apweiler R., Lopez R. InterProScan: Protein domains identifier. Nucleic Acids Res. 2005;33:W116–W120. doi: 10.1093/nar/gki442. [[DOI](https://doi.org/10.1093/nar/gki442)] [[PMC free article](/articles/PMC1160203/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15980438/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=InterProScan:%20Protein%20domains%20identifier&author=E.%20Quevillon&author=V.%20Silventoinen&author=S.%20Pillai&author=N.%20Harte&author=N.%20Mulder&volume=33&publication_year=2005&pages=W116-W120&pmid=15980438&doi=10.1093/nar/gki442&)]
* 69.Zaragoza O., García-Rodas R., Nosanchuk J.D., Cuenca-Estrella M., Rodríguez-Tudela J.L., Casadevall A. Fungal Cell Gigantism during Mammalian Infection. PLOS Pathog. 2010;6:e1000945. doi: 10.1371/annotation/0675044c-d80f-456f-bb63-4f85fb1d0c33. [[DOI](https://doi.org/10.1371/annotation/0675044c-d80f-456f-bb63-4f85fb1d0c33)] [[PMC free article](/articles/PMC2887474/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20585557/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20Pathog.&title=Fungal%20Cell%20Gigantism%20during%20Mammalian%20Infection&author=O.%20Zaragoza&author=R.%20Garc%C3%ADa-Rodas&author=J.D.%20Nosanchuk&author=M.%20Cuenca-Estrella&author=J.L.%20Rodr%C3%ADguez-Tudela&volume=6&publication_year=2010&pages=e1000945&pmid=20585557&doi=10.1371/annotation/0675044c-d80f-456f-bb63-4f85fb1d0c33&)]
* 70.Han Y.-W., Kajitani R., Morimoto H., Palihati M., Kurokawa Y., Ryusui R., Argunhan B., Tsubouchi H., Abe F., Kajiwara S., et al. Draft Genome Sequence of Naganishia liquefaciens Strain N6, Isolated from the Japan Trench. Microbiol. Resour. Announc. 2020;9:e00827-20. doi: 10.1128/MRA.00827-20. [[DOI](https://doi.org/10.1128/MRA.00827-20)] [[PMC free article](/articles/PMC7679087/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33214294/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Resour.%20Announc.&title=Draft%20Genome%20Sequence%20of%20Naganishia%20liquefaciens%20Strain%20N6,%20Isolated%20from%20the%20Japan%20Trench&author=Y.-W.%20Han&author=R.%20Kajitani&author=H.%20Morimoto&author=M.%20Palihati&author=Y.%20Kurokawa&volume=9&publication_year=2020&pages=e00827-20&pmid=33214294&doi=10.1128/MRA.00827-20&)]
* 71.Vajpeyi S., Chandran K. Draft Genome Sequence of the Oleaginous Yeast Cryptococcus albidus var. albidus. Genome Announc. 2016;4:e00390-16. doi: 10.1128/genomeA.00390-16. [[DOI](https://doi.org/10.1128/genomeA.00390-16)] [[PMC free article](/articles/PMC4888980/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27198024/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=albidus.%20Genome%20Announc.&title=Draft%20Genome%20Sequence%20of%20the%20Oleaginous%20Yeast%20Cryptococcus%20albidus%20var&author=S.%20Vajpeyi&author=K.%20Chandran&volume=4&publication_year=2016&pages=e00390-16&pmid=27198024&doi=10.1128/genomeA.00390-16&)]
* 72.Giaever G., Chu A.M., Ni L., Connelly C., Riles L., Véronneau S., Dow S., Lucau-Danila A., Anderson K., André B., et al. Functional profiling of the Saccharomyces cerevisiae genome. Nature. 2002;418:387–391. doi: 10.1038/nature00935. [[DOI](https://doi.org/10.1038/nature00935)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12140549/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Functional%20profiling%20of%20the%20Saccharomyces%20cerevisiae%20genome&author=G.%20Giaever&author=A.M.%20Chu&author=L.%20Ni&author=C.%20Connelly&author=L.%20Riles&volume=418&publication_year=2002&pages=387-391&pmid=12140549&doi=10.1038/nature00935&)]
* 73.Nichols H.L., Zhang N., Wen X. Proteomics and genomics of microgravity. Physiol. Genom. 2006;26:163–171. doi: 10.1152/physiolgenomics.00323.2005. [[DOI](https://doi.org/10.1152/physiolgenomics.00323.2005)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16705019/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol.%20Genom.&title=Proteomics%20and%20genomics%20of%20microgravity&author=H.L.%20Nichols&author=N.%20Zhang&author=X.%20Wen&volume=26&publication_year=2006&pages=163-171&pmid=16705019&doi=10.1152/physiolgenomics.00323.2005&)]
* 74.Sheehan K.B., McInnerney K., Purevdorj-Gage B., Altenburg S.D., Hyman L.E. Yeast genomic expression patterns in response to low-shear modeled microgravity. BMC Genom. 2007;8:3. doi: 10.1186/1471-2164-8-3. [[DOI](https://doi.org/10.1186/1471-2164-8-3)] [[PMC free article](/articles/PMC1774566/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17201921/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&title=Yeast%20genomic%20expression%20patterns%20in%20response%20to%20low-shear%20modeled%20microgravity&author=K.B.%20Sheehan&author=K.%20McInnerney&author=B.%20Purevdorj-Gage&author=S.D.%20Altenburg&author=L.E.%20Hyman&volume=8&publication_year=2007&pages=3&pmid=17201921&doi=10.1186/1471-2164-8-3&)]
* 75.Seel W., Baust D., Sons D., Albers M., Etzbach L., Fuss J., Lipski A. Carotenoids are used as regulators for membrane fluidity by Staphylococcus xylosus. Sci. Rep. 2020;10:330. doi: 10.1038/s41598-019-57006-5. [[DOI](https://doi.org/10.1038/s41598-019-57006-5)] [[PMC free article](/articles/PMC6962212/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31941915/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Carotenoids%20are%20used%20as%20regulators%20for%20membrane%20fluidity%20by%20Staphylococcus%20xylosus&author=W.%20Seel&author=D.%20Baust&author=D.%20Sons&author=M.%20Albers&author=L.%20Etzbach&volume=10&publication_year=2020&pages=330&pmid=31941915&doi=10.1038/s41598-019-57006-5&)]
* 76.Dieser M., Greenwood M., Foreman C.M. Carotenoid Pigmentation in Antarctic Heterotrophic Bacteria as a Strategy to Withstand Environmental Stresses. Arct. Antarct. Alp. Res. 2010;42:396–405. doi: 10.1657/1938-4246-42.4.396. [[DOI](https://doi.org/10.1657/1938-4246-42.4.396)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arct.%20Antarct.%20Alp.%20Res.&title=Carotenoid%20Pigmentation%20in%20Antarctic%20Heterotrophic%20Bacteria%20as%20a%20Strategy%20to%20Withstand%20Environmental%20Stresses&author=M.%20Dieser&author=M.%20Greenwood&author=C.M.%20Foreman&volume=42&publication_year=2010&pages=396-405&doi=10.1657/1938-4246-42.4.396&)]
* 77.Slade D., Radman M. Oxidative Stress Resistance in Deinococcus radiodurans. Microbiol. Mol. Biol. Rev. 2011;75:133–191. doi: 10.1128/MMBR.00015-10. [[DOI](https://doi.org/10.1128/MMBR.00015-10)] [[PMC free article](/articles/PMC3063356/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21372322/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Mol.%20Biol.%20Rev.&title=Oxidative%20Stress%20Resistance%20in%20Deinococcus%20radiodurans&author=D.%20Slade&author=M.%20Radman&volume=75&publication_year=2011&pages=133-191&pmid=21372322&doi=10.1128/MMBR.00015-10&)]
* 78.Paulino-Lima I.G., Janot-Pacheco E., Galante D., Cockell C., Olsson-Francis K., Brucato J.R., Baratta G.A., Strazzulla G., Merrigan T., McCullough R., et al. Survival of Deinococcus radiodurans against laboratory-simulated solar wind charged particles. Astrobiology. 2011;11:875–882. doi: 10.1089/ast.2011.0649. [[DOI](https://doi.org/10.1089/ast.2011.0649)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22059692/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Survival%20of%20Deinococcus%20radiodurans%20against%20laboratory-simulated%20solar%20wind%20charged%20particles&author=I.G.%20Paulino-Lima&author=E.%20Janot-Pacheco&author=D.%20Galante&author=C.%20Cockell&author=K.%20Olsson-Francis&volume=11&publication_year=2011&pages=875-882&pmid=22059692&doi=10.1089/ast.2011.0649&)]
* 79.Sies H., Stahl W. Carotenoids and UV protection. Photochem. Photobiol. Sci. 2004;3:749–752. doi: 10.1039/b316082c. [[DOI](https://doi.org/10.1039/b316082c)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15295630/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Photochem.%20Photobiol.%20Sci.&title=Carotenoids%20and%20UV%20protection&author=H.%20Sies&author=W.%20Stahl&volume=3&publication_year=2004&pages=749-752&pmid=15295630&doi=10.1039/b316082c&)]
* 80.Rapoport A., Guzhova I., Bernetti L., Buzzini P., Kieliszek M., Kot A.M. Carotenoids and Some Other Pigments from Fungi and Yeasts. Metabolites. 2021;11:92. doi: 10.3390/metabo11020092. [[DOI](https://doi.org/10.3390/metabo11020092)] [[PMC free article](/articles/PMC7915786/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33561985/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Metabolites&title=Carotenoids%20and%20Some%20Other%20Pigments%20from%20Fungi%20and%20Yeasts&author=A.%20Rapoport&author=I.%20Guzhova&author=L.%20Bernetti&author=P.%20Buzzini&author=M.%20Kieliszek&volume=11&publication_year=2021&pages=92&pmid=33561985&doi=10.3390/metabo11020092&)]
* 81.Miao L., Chi S., Wu M., Liu Z., Li Y. Deregulation of phytoene-β-carotene synthase results in derepression of astaxanthin synthesis at high glucose concentration in Phaffia rhodozyma astaxanthin-overproducing strain MK19. BMC Microbiol. 2019;19:133. doi: 10.1186/s12866-019-1507-6. [[DOI](https://doi.org/10.1186/s12866-019-1507-6)] [[PMC free article](/articles/PMC6570914/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31202260/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol.&title=Deregulation%20of%20phytoene-%CE%B2-carotene%20synthase%20results%20in%20derepression%20of%20astaxanthin%20synthesis%20at%20high%20glucose%20concentration%20in%20Phaffia%20rhodozyma%20astaxanthin-overproducing%20strain%20MK19&author=L.%20Miao&author=S.%20Chi&author=M.%20Wu&author=Z.%20Liu&author=Y.%20Li&volume=19&publication_year=2019&pages=133&pmid=31202260&doi=10.1186/s12866-019-1507-6&)]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

[Click here for additional data file.](/articles/instance/8875396/bin/jof-08-00165-s001.zip) (5.2MB, zip)

### Data Availability Statement

The WGS and raw data have been deposited in GenBank under the BioProject accession number PRJNA623412 [[6](#B6-jof-08-00165)]. The genome sequences has also been deposited in the NASA GeneLab system (GLDS-290; <https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-290>; accessed on 5 December 2021).