<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5201052/"/><title>Draft Genome Sequences of Biosafety Level 2 Opportunistic Pathogens Isolated from the Environmental Surfaces of the International Space Station - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Genome Announc</button></div>. 2016 Dec 29;4(6):e01263-16. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/genomeA.01263-16" rel="noopener noreferrer" target="_blank">10.1128/genomeA.01263-16</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Draft Genome Sequences of Biosafety Level 2 Opportunistic Pathogens Isolated from the Environmental Surfaces of the International Space Station</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Checinska%20Sielaff%20A%22%5BAuthor%5D"><span class="name western">Aleksandra Checinska Sielaff</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Aleksandra Checinska Sielaff</span></h3>
<div class="p">
<sup>a</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Checinska%20Sielaff%20A%22%5BAuthor%5D"><span class="name western">Aleksandra Checinska Sielaff</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D"><span class="name western">Nitin K Singh</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Nitin K Singh</span></h3>
<div class="p">
<sup>a</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D"><span class="name western">Nitin K Singh</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Allen%20JE%22%5BAuthor%5D"><span class="name western">Jonathan E Allen</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jonathan E Allen</span></h3>
<div class="p">
<sup>b</sup>Lawrence Livermore National Laboratory, Livermore, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Allen%20JE%22%5BAuthor%5D"><span class="name western">Jonathan E Allen</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thissen%20J%22%5BAuthor%5D"><span class="name western">James Thissen</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">James Thissen</span></h3>
<div class="p">
<sup>b</sup>Lawrence Livermore National Laboratory, Livermore, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thissen%20J%22%5BAuthor%5D"><span class="name western">James Thissen</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaing%20C%22%5BAuthor%5D"><span class="name western">Crystal Jaing</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Crystal Jaing</span></h3>
<div class="p">
<sup>b</sup>Lawrence Livermore National Laboratory, Livermore, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaing%20C%22%5BAuthor%5D"><span class="name western">Crystal Jaing</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kasthuri Venkateswaran</span></h3>
<div class="p">
<sup>a</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a>
</div>
</div>
<sup>a,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>a</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div id="aff2">
<sup>b</sup>Lawrence Livermore National Laboratory, Livermore, California, USA</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>✉</sup><p class="display-inline">Address correspondence to Kasthuri Venkateswaran, <span>kjvenkat@jpl.nasa.gov</span>.</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2016 Sep 20; Accepted 2016 Oct 31; Collection date 2016 Nov-Dec.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2016 Checinska Sielaff et al.</div>
<p>This is an open-access article distributed under the terms of the <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">Creative Commons Attribution 4.0 International license</a>.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC5201052  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28034853/">28034853</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The draft genome sequences of 20 biosafety level 2 (BSL-2) opportunistic pathogens isolated from the environmental surfaces of the International Space Station (ISS) were presented. These genomic sequences will help in understanding the influence of microgravity on the pathogenicity and virulence of these strains when compared with Earth strains.</p></section><section id="h0.0"><h2 class="pmc_sec_title">GENOME ANNOUNCEMENT</h2>
<p>In an on-going Microbial Observatory experiment on the International Space Station (ISS), multiple biosafety level 2 (BSL-2) bacterial isolates were isolated, identified, and whole-genome sequences (WGS) were generated. The genomic data enables the determination of the microgravity influence on pathogenicity and virulence in these microorganisms by comparison to type strains of the corresponding species.</p>
<p><em>Acinetobacter pittii</em> is a nonmotile coccobacilli isolated for the first time from cerebrospinal fluid (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>). Multiple strains of <em>A. pitti</em> were isolated from the cupola area. The IIF1SW-P1 was resistant to cefazolin, cefoxitin, oxacillin, penicillin, and rifampin.</p>
<p>Two multidrug-resistant <em>Enterobacter</em> sp. isolates were found in the waste and hygiene compartment (WHC) location. Species of <em>Enterobacter cloacae</em> complex (Ecc) are commonly found in the environment, but are of high clinical significance (<a aria-describedby="B2" class="usa-link" href="#B2">2</a>).</p>
<p><em>Pantoea conspicua</em> was originally isolated from human blood (<a aria-describedby="B3" class="usa-link" href="#B3">3</a>). This was the second most prevalent species, and was only found in one location during two different flight samplings. <em>P. conspicua</em> isolates were resistant to erythromycin, oxacillin, penicillin, and rifampin.</p>
<p><em>Staphylococcus</em> isolates were the most prevalent from ISS surfaces. <em>Staphylococcus aureus</em> was the most abundant in all ISS locations. Although this species is a common human commensal (<a aria-describedby="B4" class="usa-link" href="#B4">4</a>), it causes various types of minor skin infections, bacteremia, or scalded skin syndrome, especially in immunocompromised individuals (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>). In this study, some of the isolates were found to be resistant to erythromycin (IF4SW-P1, IF7SW-P3) and most of the isolates were resistant to penicillin. A few isolates acquired rifampin resistance during the study (RA isolates).</p>
<p><em>Staphylococcus haemolyticus</em> and <em>Staphylococcus hominis</em> belong to coagulase-negative staphylococci (<a aria-describedby="B6" class="usa-link" href="#B6">6</a>, <a aria-describedby="B7" class="usa-link" href="#B7">7</a>). <em>S. hominis</em> IIF4SC-B9 was resistant to penicillin and erythromycin, but <em>S. haemolyticus</em> IIF2SW-P5 was susceptible to these antibiotics. All three species are reported to be methicillin resistant by acquiring the staphylococcal cassette chromosome <em>mec</em> (SCC<em>mec</em>) (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>), but the methicillin-resistant phenotype was not observed.</p>
<p>In this study, the draft genomes sequences of 20 strains from the ISS were obtained. WGS sequencing was performed on an Illumina NextSeq instrument with a paired-end module. The A5 assembly pipeline version 20150522 was used to generate draft assemblies applying the default parameter settings (<a aria-describedby="B9" class="usa-link" href="#B9">9</a>) and annotated with the help of the Rapid Annotations using Subsystems Technology (RAST) (<a aria-describedby="B10" class="usa-link" href="#B10">10</a>). <a class="usa-link" href="#tab1">Table 1</a> summarizes assembly statistics (number of contigs, total genome size, <em>N</em><sub>50</sub> size, median coverage, G+C percentage, error corrected reads used for assembly, and number of coding sequences). The raw reads were in the range of 24 to 82 Mbp per genome. The G+C content was in the range of 31.5 to 38.7% for <em>Staphylococcus</em> species and <em>A. pittii</em>; for other strains the G+C contents were 55.2 to 55.8%. The subsystem features created using RAST for all 20 strains are depicted in <a class="usa-link" href="#tab1">Table 1</a>.</p>
<section class="tw xbox font-sm" id="tab1"><h3 class="obj_head">TABLE 1 .</h3>
<div class="caption p"><p>Statistics summary for the 20 draft ISS BSL-2 bacterial genome sequences</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
</colgroup>
<thead><tr>
<th colspan="1" rowspan="1">Strain</th>
<th colspan="1" rowspan="1">NCBI accession no.</th>
<th colspan="1" rowspan="1">Isolation location</th>
<th colspan="1" rowspan="1">No. of contigs</th>
<th colspan="1" rowspan="1">Genome size (bp)</th>
<th colspan="1" rowspan="1">
<em>N</em><sub>50</sub> (bp)</th>
<th colspan="1" rowspan="1">Median coverage</th>
<th colspan="1" rowspan="1">G+C content (%)</th>
<th colspan="1" rowspan="1">Error corrected reads</th>
<th colspan="1" rowspan="1">Coding sequences</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">
<em>A. pittii</em> IIF1SW-P1</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZX00000000" rel="noopener noreferrer" target="_blank">MIZX00000000</a></td>
<td colspan="1" rowspan="1">Port panel next to cupola</td>
<td colspan="1" rowspan="1">150</td>
<td colspan="1" rowspan="1">4,041,255</td>
<td colspan="1" rowspan="1">144,373</td>
<td colspan="1" rowspan="1">799</td>
<td colspan="1" rowspan="1">38.7</td>
<td colspan="1" rowspan="1">25,486,884</td>
<td colspan="1" rowspan="1">3,821</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>Enterobacter</em> sp. IF2SW-B1</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MJAA00000000" rel="noopener noreferrer" target="_blank">MJAA00000000</a></td>
<td colspan="1" rowspan="1">WHC<a class="usa-link" href="#ngtab1.1"><sup>a</sup></a>
</td>
<td colspan="1" rowspan="1">437</td>
<td colspan="1" rowspan="1">5,097,299</td>
<td colspan="1" rowspan="1">306,837</td>
<td colspan="1" rowspan="1">686</td>
<td colspan="1" rowspan="1">55.2</td>
<td colspan="1" rowspan="1">24,992,043</td>
<td colspan="1" rowspan="1">4,671</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>Enterobacter</em> sp. IF2SW-P2</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MJAB00000000" rel="noopener noreferrer" target="_blank">MJAB00000000</a></td>
<td colspan="1" rowspan="1">WHC<a class="usa-link" href="#ngtab1.1"><sup>a</sup></a>
</td>
<td colspan="1" rowspan="1">230</td>
<td colspan="1" rowspan="1">4,974,814</td>
<td colspan="1" rowspan="1">298,912</td>
<td colspan="1" rowspan="1">850</td>
<td colspan="1" rowspan="1">55.8</td>
<td colspan="1" rowspan="1">30,618,796</td>
<td colspan="1" rowspan="1">4,629</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>P. conspicua</em> IF5SW-P1</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZY00000000" rel="noopener noreferrer" target="_blank">MIZY00000000</a></td>
<td colspan="1" rowspan="1">Node 1 overhead 4</td>
<td colspan="1" rowspan="1">280</td>
<td colspan="1" rowspan="1">5,126,609</td>
<td colspan="1" rowspan="1">216,776</td>
<td colspan="1" rowspan="1">797</td>
<td colspan="1" rowspan="1">55.6</td>
<td colspan="1" rowspan="1">34,104,170</td>
<td colspan="1" rowspan="1">4,852</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IF4SW-P1</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZH00000000" rel="noopener noreferrer" target="_blank">MIZH00000000</a></td>
<td colspan="1" rowspan="1">Dining table</td>
<td colspan="1" rowspan="1">498</td>
<td colspan="1" rowspan="1">2,980,137</td>
<td colspan="1" rowspan="1">64,789</td>
<td colspan="1" rowspan="1">3,695</td>
<td colspan="1" rowspan="1">32.7</td>
<td colspan="1" rowspan="1">76,859,228</td>
<td colspan="1" rowspan="1">2,733</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IF6SW-P2</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZI00000000" rel="noopener noreferrer" target="_blank">MIZI00000000</a></td>
<td colspan="1" rowspan="1">PMM port 1<a class="usa-link" href="#ngtab1.2"><sup>b</sup></a>
</td>
<td colspan="1" rowspan="1">204</td>
<td colspan="1" rowspan="1">2,836,553</td>
<td colspan="1" rowspan="1">355,893</td>
<td colspan="1" rowspan="1">2,578</td>
<td colspan="1" rowspan="1">32.8</td>
<td colspan="1" rowspan="1">51,467,673</td>
<td colspan="1" rowspan="1">2,657</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IF6SW-P2-RA</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZK00000000" rel="noopener noreferrer" target="_blank">MIZK00000000</a></td>
<td colspan="1" rowspan="1">PMM port 1<a class="usa-link" href="#ngtab1.2"><sup>b</sup></a>
</td>
<td colspan="1" rowspan="1">228</td>
<td colspan="1" rowspan="1">2,845,178</td>
<td colspan="1" rowspan="1">295,897</td>
<td colspan="1" rowspan="1">2,740</td>
<td colspan="1" rowspan="1">32.8</td>
<td colspan="1" rowspan="1">55,167,977</td>
<td colspan="1" rowspan="1">2,659</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IF6SW-P3A</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZJ00000000" rel="noopener noreferrer" target="_blank">MIZJ00000000</a></td>
<td colspan="1" rowspan="1">PMM port 1<a class="usa-link" href="#ngtab1.2"><sup>b</sup></a>
</td>
<td colspan="1" rowspan="1">276</td>
<td colspan="1" rowspan="1">2,868,506</td>
<td colspan="1" rowspan="1">232,680</td>
<td colspan="1" rowspan="1">2,254</td>
<td colspan="1" rowspan="1">32.8</td>
<td colspan="1" rowspan="1">46,555,897</td>
<td colspan="1" rowspan="1">2,694</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IF6SW-P3A-RA</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZL00000000" rel="noopener noreferrer" target="_blank">MIZL00000000</a></td>
<td colspan="1" rowspan="1">PMM port 1<a class="usa-link" href="#ngtab1.2"><sup>b</sup></a>
</td>
<td colspan="1" rowspan="1">257</td>
<td colspan="1" rowspan="1">2,861,821</td>
<td colspan="1" rowspan="1">264,865</td>
<td colspan="1" rowspan="1">2,733</td>
<td colspan="1" rowspan="1">32.8</td>
<td colspan="1" rowspan="1">47,711,605</td>
<td colspan="1" rowspan="1">2,690</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IF7SW-P3</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZM00000000" rel="noopener noreferrer" target="_blank">MIZM00000000</a></td>
<td colspan="1" rowspan="1">Lab overhead 3</td>
<td colspan="1" rowspan="1">452</td>
<td colspan="1" rowspan="1">2,951,917</td>
<td colspan="1" rowspan="1">52,140</td>
<td colspan="1" rowspan="1">3,487</td>
<td colspan="1" rowspan="1">32.8</td>
<td colspan="1" rowspan="1">71,062,021</td>
<td colspan="1" rowspan="1">2,738</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IIF6SW-P2</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZN00000000" rel="noopener noreferrer" target="_blank">MIZN00000000</a></td>
<td colspan="1" rowspan="1">PMM port 1<a class="usa-link" href="#ngtab1.2"><sup>b</sup></a>
</td>
<td colspan="1" rowspan="1">312</td>
<td colspan="1" rowspan="1">2,884,460</td>
<td colspan="1" rowspan="1">96,689</td>
<td colspan="1" rowspan="1">3,324</td>
<td colspan="1" rowspan="1">32.8</td>
<td colspan="1" rowspan="1">67,792,619</td>
<td colspan="1" rowspan="1">2,730</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IIF6SW-P2-RA</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZR00000000" rel="noopener noreferrer" target="_blank">MIZR00000000</a></td>
<td colspan="1" rowspan="1">PMM port 1<a class="usa-link" href="#ngtab1.2"><sup>b</sup></a>
</td>
<td colspan="1" rowspan="1">192</td>
<td colspan="1" rowspan="1">2,835,299</td>
<td colspan="1" rowspan="1">325,968</td>
<td colspan="1" rowspan="1">2,021</td>
<td colspan="1" rowspan="1">32.8</td>
<td colspan="1" rowspan="1">42,250,883</td>
<td colspan="1" rowspan="1">2,655</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IIF6SW-P3</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZO00000000" rel="noopener noreferrer" target="_blank">MIZO00000000</a></td>
<td colspan="1" rowspan="1">PMM port 1<a class="usa-link" href="#ngtab1.2"><sup>b</sup></a>
</td>
<td colspan="1" rowspan="1">194</td>
<td colspan="1" rowspan="1">2,837,901</td>
<td colspan="1" rowspan="1">467,825</td>
<td colspan="1" rowspan="1">2,638</td>
<td colspan="1" rowspan="1">32.8</td>
<td colspan="1" rowspan="1">54,334,144</td>
<td colspan="1" rowspan="1">2,657</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IIF6SW-P3-RA</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZS00000000" rel="noopener noreferrer" target="_blank">MIZS00000000</a></td>
<td colspan="1" rowspan="1">PMM port 1<a class="usa-link" href="#ngtab1.2"><sup>b</sup></a>
</td>
<td colspan="1" rowspan="1">217</td>
<td colspan="1" rowspan="1">2,841,156</td>
<td colspan="1" rowspan="1">411,108</td>
<td colspan="1" rowspan="1">2,272</td>
<td colspan="1" rowspan="1">32.8</td>
<td colspan="1" rowspan="1">47,711,605</td>
<td colspan="1" rowspan="1">2,656</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IIF8SW-P1</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZP00000000" rel="noopener noreferrer" target="_blank">MIZP00000000</a></td>
<td colspan="1" rowspan="1">Port crew quarters bump-out exterior aft wall</td>
<td colspan="1" rowspan="1">143</td>
<td colspan="1" rowspan="1">2,817,304</td>
<td colspan="1" rowspan="1">425,858</td>
<td colspan="1" rowspan="1">2,409</td>
<td colspan="1" rowspan="1">32.8</td>
<td colspan="1" rowspan="1">49,197,886</td>
<td colspan="1" rowspan="1">2,637</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IIF8SW-P1-RA</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZT00000000" rel="noopener noreferrer" target="_blank">MIZT00000000</a></td>
<td colspan="1" rowspan="1">Port crew quarters bump-out exterior aft wall</td>
<td colspan="1" rowspan="1">201</td>
<td colspan="1" rowspan="1">2,848,005</td>
<td colspan="1" rowspan="1">526,364</td>
<td colspan="1" rowspan="1">1,834</td>
<td colspan="1" rowspan="1">32.7</td>
<td colspan="1" rowspan="1">39,316,061</td>
<td colspan="1" rowspan="1">2,653</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IIF8SW-P2</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZQ00000000" rel="noopener noreferrer" target="_blank">MIZQ00000000</a></td>
<td colspan="1" rowspan="1">Port crew quarters bump-out exterior aft wall</td>
<td colspan="1" rowspan="1">194</td>
<td colspan="1" rowspan="1">2,830,972</td>
<td colspan="1" rowspan="1">329,726</td>
<td colspan="1" rowspan="1">2,557</td>
<td colspan="1" rowspan="1">32.8</td>
<td colspan="1" rowspan="1">51,625,221</td>
<td colspan="1" rowspan="1">2,650</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. aureus</em> IIF8SW-P2-RA</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZU00000000" rel="noopener noreferrer" target="_blank">MIZU00000000</a></td>
<td colspan="1" rowspan="1">Port crew quarters bump-out exterior aft wall</td>
<td colspan="1" rowspan="1">141</td>
<td colspan="1" rowspan="1">2,822,756</td>
<td colspan="1" rowspan="1">526,364</td>
<td colspan="1" rowspan="1">2,014</td>
<td colspan="1" rowspan="1">32.8</td>
<td colspan="1" rowspan="1">42,704,251</td>
<td colspan="1" rowspan="1">2,642</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. haemolyticus</em> IIF2SW-P5</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZW00000000" rel="noopener noreferrer" target="_blank">MIZW00000000</a></td>
<td colspan="1" rowspan="1">WHC<a class="usa-link" href="#ngtab1.1"><sup>a</sup></a>
</td>
<td colspan="1" rowspan="1">567</td>
<td colspan="1" rowspan="1">2,680,722</td>
<td colspan="1" rowspan="1">48,308</td>
<td colspan="1" rowspan="1">2,945</td>
<td colspan="1" rowspan="1">33.1</td>
<td colspan="1" rowspan="1">56,836,461</td>
<td colspan="1" rowspan="1">2,518</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>S. hominis</em> IIF4SC-B9</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZV00000000" rel="noopener noreferrer" target="_blank">MIZV00000000</a></td>
<td colspan="1" rowspan="1">Dining table</td>
<td colspan="1" rowspan="1">508</td>
<td colspan="1" rowspan="1">2,420,684</td>
<td colspan="1" rowspan="1">79,555</td>
<td colspan="1" rowspan="1">3,738</td>
<td colspan="1" rowspan="1">31.5</td>
<td colspan="1" rowspan="1">61,283,456</td>
<td colspan="1" rowspan="1">2,301</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/tab1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="ngtab1.1">
<sup><sup>a</sup></sup><p class="display-inline">WHC, waste and hygiene compartment.</p>
</div>
<div class="fn" id="ngtab1.2">
<sup><sup>b</sup></sup><p class="display-inline">PMM port 1, permanent multipurpose module.</p>
</div>
</div></section><section id="s1"><h3 class="pmc_sec_title">Accession number(s).</h3>
<p>The WGS data were deposited at DDBL/EMBL/GenBank under the accession no. listed in <a class="usa-link" href="#tab1">Table 1</a> and at the NASA GeneLab system (GLDS-67; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-67/##" rel="noopener noreferrer" target="_blank">https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-67/##</a>). The version described in this paper is the first version. The strains were deposited in the USDA Agricultural Research Station (NRRL) and German culture collections.</p></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank the implementation team (lead by Fathi Karouia) of the Microbial Observatory (Microbial Tracking) project at NASA Ames Research Center.</p>
<p>Part of the research described in this publication was carried out at the Jet Propulsion Laboratory, California Institute of Technology, and at the Lawrence Livermore National laboratory, under a contract with NASA.</p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn3"><p><strong>Citation</strong> Checinska Sielaff A, Singh NK, Allen JE, Thissen J, Jaing C, Venkateswaran K. 2016. Draft genome sequences of biosafety level 2 opportunistic pathogens isolated from the environmental surfaces of the International Space Station. Genome Announc 4(6):e01263-16. doi:10.1128/genomeA.01263-16.</p></div></div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>Nemec A, Krizova L, Maixnerova M, van der Reijden TJK, Deschaght P, Passet V, Vaneechoutte M, Brisse S, Dijkshoorn L. 
2011. 
Genotypic and phenotypic characterization of the <em>Acinetobacter</em>
<em>calcoaceticuse</em>-<em>Acinetobacter</em>
<em>baumannii</em> complex with the proposal of <em>Acinetobacter</em>
<em>pittii</em> sp. nov. (formerly <em>Acinetobacter</em> genomic species 3) and <em>Acinetobacter</em>
<em>nosocomialis</em> sp. nov. (formerly <em>Acinetobacter</em> genomic species 13TU). Res Microbiol
162:393–404. doi: 10.1016/j.resmic.2011.02.006.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.resmic.2011.02.006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21320596/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Res%20Microbiol&amp;title=Genotypic%20and%20phenotypic%20characterization%20of%20the%20Acinetobacter%20calcoaceticuse-Acinetobacter%20baumannii%20complex%20with%20the%20proposal%20of%20Acinetobacter%20pittii%20sp.%20nov.%20(formerly%20Acinetobacter%20genomic%20species%203)%20and%20Acinetobacter%20nosocomialis%20sp.%20nov.%20(formerly%20Acinetobacter%20genomic%20species%2013TU)&amp;author=A%20Nemec&amp;author=L%20Krizova&amp;author=M%20Maixnerova&amp;author=TJK%20van%20der%20Reijden&amp;author=P%20Deschaght&amp;volume=162&amp;publication_year=2011&amp;pages=393-404&amp;pmid=21320596&amp;doi=10.1016/j.resmic.2011.02.006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>Mezzatesta ML, Gona F, Stefani S. 
2012. 
<em>Enterobacter cloacae</em> complex: clinical impact and emerging antibiotic resistance. Future Microbiol
7:887–902. doi: 10.2217/fmb.12.61.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2217/fmb.12.61" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22827309/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Future%20Microbiol&amp;title=Enterobacter%20cloacae%20complex:%20clinical%20impact%20and%20emerging%20antibiotic%20resistance&amp;author=ML%20Mezzatesta&amp;author=F%20Gona&amp;author=S%20Stefani&amp;volume=7&amp;publication_year=2012&amp;pages=887-902&amp;pmid=22827309&amp;doi=10.2217/fmb.12.61&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>Brady CL, Cleenwerck I, Venter SN, Engelbeen K, De Vos P, Coutinho TA. 
2010. 
Emended description of the genus <em>Pantoea</em>, description of four species from human clinical samples, <em>Pantoea</em>
<em>septica</em> sp. nov., <em>Pantoea</em>
<em>eucrina</em> sp. nov., <em>Pantoea</em>
<em>brenneri</em> sp. nov. and <em>Pantoea</em>
<em>conspicua</em> sp. nov., and transfer of <em>Pectobacterium cypripedii</em> (Hori 1911) Brenner et al. 1973 emend. Hauben et al. 1998 to the genus as <em>Pantoea</em>
<em>cypripedii</em> comb. nov. Int J Syst Evol Microbiol
60:2430–2440. doi: 10.1099/ijs.0.017301-0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1099/ijs.0.017301-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19946052/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&amp;title=Emended%20description%20of%20the%20genus%20Pantoea,%20description%20of%20four%20species%20from%20human%20clinical%20samples,%20Pantoea%20septica%20sp.%20nov.,%20Pantoea%20eucrina%20sp.%20nov.,%20Pantoea%20brenneri%20sp.%20nov.%20and%20Pantoea%20conspicua%20sp.%20nov.,%20and%20transfer%20of%20Pectobacterium%20cypripedii%20(Hori%201911)%20Brenner%20et%20al.%201973%20emend.%20Hauben%20et%20al.%201998%20to%20the%20genus%20as%20Pantoea%20cypripedii%20comb.%20nov&amp;author=CL%20Brady&amp;author=I%20Cleenwerck&amp;author=SN%20Venter&amp;author=K%20Engelbeen&amp;author=P%20De%20Vos&amp;volume=60&amp;publication_year=2010&amp;pages=2430-2440&amp;pmid=19946052&amp;doi=10.1099/ijs.0.017301-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>Sakwinska O, Kuhn G, Balmelli C, Francioli P, Giddey M, Perreten V, Riesen A, Zysset F, Blanc DS, Moreillon P. 
2009. 
Genetic diversity and ecological success of <em>Staphylococcus aureus</em> strains colonizing humans. Appl Environ Microbiol
75:175–183. doi: 10.1128/AEM.01860-08.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.01860-08" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2612194/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18978084/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Genetic%20diversity%20and%20ecological%20success%20of%20Staphylococcus%20aureus%20strains%20colonizing%20humans&amp;author=O%20Sakwinska&amp;author=G%20Kuhn&amp;author=C%20Balmelli&amp;author=P%20Francioli&amp;author=M%20Giddey&amp;volume=75&amp;publication_year=2009&amp;pages=175-183&amp;pmid=18978084&amp;doi=10.1128/AEM.01860-08&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>Archer GL.
1998. 
<em>Staphylococcus aureus</em>: a well-armed pathogen. Clin Infect Dis
26:1179–1181. doi: 10.1086/520289.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1086/520289" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9597249/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&amp;title=Staphylococcus%20aureus:%20a%20well-armed%20pathogen&amp;author=GL%20Archer&amp;volume=26&amp;publication_year=1998&amp;pages=1179-1181&amp;pmid=9597249&amp;doi=10.1086/520289&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>Mendoza-Olazarán S, Morfin-Otero R, Rodríguez-Noriega E, Llaca-Díaz J, Flores-Treviño S, González-González GM, Villarreal-Treviño L, Garza-González E. 
2013. 
Microbiological and molecular characterization of <em>Staphylococcus hominis</em> isolates from blood. PLoS One
8:e61161. doi: 10.1371/journal.pone.0061161.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0061161" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3622014/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23585877/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Microbiological%20and%20molecular%20characterization%20of%20Staphylococcus%20hominis%20isolates%20from%20blood&amp;author=S%20Mendoza-Olazar%C3%A1n&amp;author=R%20Morfin-Otero&amp;author=E%20Rodr%C3%ADguez-Noriega&amp;author=J%20Llaca-D%C3%ADaz&amp;author=S%20Flores-Trevi%C3%B1o&amp;volume=8&amp;publication_year=2013&amp;pages=e61161&amp;pmid=23585877&amp;doi=10.1371/journal.pone.0061161&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>Barros EM, Ceotto H, Bastos MCF, dos Santos KRN, Giambiagi-deMarval M. 
2012. 
<em>Staphylococcus haemolyticus</em> as an important hospital pathogen and carrier of methicillin resistance genes. J Clin Microbiol
50:166–168. doi: 10.1128/JCM.05563-11.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JCM.05563-11" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3256717/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21976766/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Microbiol&amp;title=Staphylococcus%20haemolyticus%20as%20an%20important%20hospital%20pathogen%20and%20carrier%20of%20methicillin%20resistance%20genes&amp;author=EM%20Barros&amp;author=H%20Ceotto&amp;author=MCF%20Bastos&amp;author=KRN%20dos%20Santos&amp;author=M%20Giambiagi-deMarval&amp;volume=50&amp;publication_year=2012&amp;pages=166-168&amp;pmid=21976766&amp;doi=10.1128/JCM.05563-11&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>Katayama Y, Ito T, Hiramatsu K. 
2000. 
A new class of genetic element, <em>Staphylococcus</em> cassette chromosome <em>mec</em>, encodes methicillin resistance in <em>Staphylococcus aureus</em>. Antimicrob Agents Chemother
44:1549–1555. doi: 10.1128/AAC.44.6.1549-1555.2000.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AAC.44.6.1549-1555.2000" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC89911/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10817707/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&amp;title=A%20new%20class%20of%20genetic%20element,%20Staphylococcus%20cassette%20chromosome%20mec,%20encodes%20methicillin%20resistance%20in%20Staphylococcus%20aureus&amp;author=Y%20Katayama&amp;author=T%20Ito&amp;author=K%20Hiramatsu&amp;volume=44&amp;publication_year=2000&amp;pages=1549-1555&amp;pmid=10817707&amp;doi=10.1128/AAC.44.6.1549-1555.2000&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>Tritt A, Eisen JA, Facciotti MT, Darling AE. 
2012. 
An integrated pipeline for <em>de novo</em> assembly of microbial genomes. PLoS One
7:e42304. doi: 10.1371/journal.pone.0042304.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0042304" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3441570/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23028432/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=An%20integrated%20pipeline%20for%20de%20novo%20assembly%20of%20microbial%20genomes&amp;author=A%20Tritt&amp;author=JA%20Eisen&amp;author=MT%20Facciotti&amp;author=AE%20Darling&amp;volume=7&amp;publication_year=2012&amp;pages=e42304&amp;pmid=23028432&amp;doi=10.1371/journal.pone.0042304&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>Aziz RK, Bartels D, Best AA, DeJongh M, Disz T, Edwards RA, Formsma K, Gerdes S, Glass EM, Kubal M, Meyer F, Olsen GJ, Olson R, Osterman AL, Overbeek RA, McNeil LK, Paarmann D, Paczian T, Parrello B, Pusch GD, Reich C, Stevens R, Vassieva O, Vonstein V, Wilke A, Zagnitko O. 
2008. 
The RAST Server: Rapid Annotations using Subsystems Technology. BMC Genomics
9:75. doi: 10.1186/1471-2164-9-75.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2164-9-75" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2265698/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18261238/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=The%20RAST%20Server:%20Rapid%20Annotations%20using%20Subsystems%20Technology&amp;author=RK%20Aziz&amp;author=D%20Bartels&amp;author=AA%20Best&amp;author=M%20DeJongh&amp;author=T%20Disz&amp;volume=9&amp;publication_year=2008&amp;pages=75&amp;pmid=18261238&amp;doi=10.1186/1471-2164-9-75&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section></section></section></article>
</main></body></html>