Microbiol Spectr

. 2024 Jan 17;12(2):e01144-23. doi: [10.1128/spectrum.01144-23](https://doi.org/10.1128/spectrum.01144-23)

# Metabolic model predictions enable targeted microbiome manipulation through precision prebiotics

[Georgios Marinos](https://pubmed.ncbi.nlm.nih.gov/?term=%22Marinos%20G%22%5BAuthor%5D)

### Georgios Marinos

1Research Group Medical Systems Biology, University Hospital Schleswig-Holstein Campus Kiel, Kiel University, Kiel, Schleswig-Holstein, Germany

Conceptualization, Formal analysis, Investigation, Methodology, Software, Writing – original draft, Writing – review and editing, Data curation, Funding acquisition, Project administration, Visualization

Find articles by [Georgios Marinos](https://pubmed.ncbi.nlm.nih.gov/?term=%22Marinos%20G%22%5BAuthor%5D)

1,#, [Inga K Hamerich](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hamerich%20IK%22%5BAuthor%5D)

### Inga K Hamerich

2Research Group Evolutionary Ecology and Genetics, Zoological Institute, Kiel University, Kiel, Schleswig-Holstein, Germany

Formal analysis, Investigation, Methodology, Writing – original draft, Writing – review and editing, Data curation, Software, Visualization

Find articles by [Inga K Hamerich](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hamerich%20IK%22%5BAuthor%5D)

2,#, [Reena Debray](https://pubmed.ncbi.nlm.nih.gov/?term=%22Debray%20R%22%5BAuthor%5D)

### Reena Debray

3Department of Integrative Biology, University of California, Berkeley, California, USA

Formal analysis, Investigation, Writing – review and editing, Data curation, Funding acquisition, Methodology, Software, Visualization

Find articles by [Reena Debray](https://pubmed.ncbi.nlm.nih.gov/?term=%22Debray%20R%22%5BAuthor%5D)

3, [Nancy Obeng](https://pubmed.ncbi.nlm.nih.gov/?term=%22Obeng%20N%22%5BAuthor%5D)

### Nancy Obeng

2Research Group Evolutionary Ecology and Genetics, Zoological Institute, Kiel University, Kiel, Schleswig-Holstein, Germany

Formal analysis, Investigation, Writing – review and editing, Conceptualization, Methodology, Supervision

Find articles by [Nancy Obeng](https://pubmed.ncbi.nlm.nih.gov/?term=%22Obeng%20N%22%5BAuthor%5D)

2, [Carola Petersen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Petersen%20C%22%5BAuthor%5D)

### Carola Petersen

2Research Group Evolutionary Ecology and Genetics, Zoological Institute, Kiel University, Kiel, Schleswig-Holstein, Germany

Investigation, Writing – review and editing, Methodology, Software

Find articles by [Carola Petersen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Petersen%20C%22%5BAuthor%5D)

2, [Jan Taubenheim](https://pubmed.ncbi.nlm.nih.gov/?term=%22Taubenheim%20J%22%5BAuthor%5D)

### Jan Taubenheim

1Research Group Medical Systems Biology, University Hospital Schleswig-Holstein Campus Kiel, Kiel University, Kiel, Schleswig-Holstein, Germany

Writing – review and editing, Methodology, Software

Find articles by [Jan Taubenheim](https://pubmed.ncbi.nlm.nih.gov/?term=%22Taubenheim%20J%22%5BAuthor%5D)

1, [Johannes Zimmermann](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zimmermann%20J%22%5BAuthor%5D)

### Johannes Zimmermann

1Research Group Medical Systems Biology, University Hospital Schleswig-Holstein Campus Kiel, Kiel University, Kiel, Schleswig-Holstein, Germany

4Max-Planck Institute for Evolutionary Biology, Ploen, Schleswig-Holstein, Germany

Methodology, Writing – review and editing, Software

Find articles by [Johannes Zimmermann](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zimmermann%20J%22%5BAuthor%5D)

1,4, [Dana Blackburn](https://pubmed.ncbi.nlm.nih.gov/?term=%22Blackburn%20D%22%5BAuthor%5D)

### Dana Blackburn

5Alkek Center for Metagenomics and Microbiome Research, Baylor College of Medicine, Houston, Texas, USA

Resources, Writing – review and editing, Methodology

Find articles by [Dana Blackburn](https://pubmed.ncbi.nlm.nih.gov/?term=%22Blackburn%20D%22%5BAuthor%5D)

5, [Buck S Samuel](https://pubmed.ncbi.nlm.nih.gov/?term=%22Samuel%20BS%22%5BAuthor%5D)

### Buck S Samuel

5Alkek Center for Metagenomics and Microbiome Research, Baylor College of Medicine, Houston, Texas, USA

Resources, Writing – review and editing, Funding acquisition, Methodology

Find articles by [Buck S Samuel](https://pubmed.ncbi.nlm.nih.gov/?term=%22Samuel%20BS%22%5BAuthor%5D)

5, [Katja Dierking](https://pubmed.ncbi.nlm.nih.gov/?term=%22Dierking%20K%22%5BAuthor%5D)

### Katja Dierking

2Research Group Evolutionary Ecology and Genetics, Zoological Institute, Kiel University, Kiel, Schleswig-Holstein, Germany

Resources, Writing – review and editing, Funding acquisition, Methodology

Find articles by [Katja Dierking](https://pubmed.ncbi.nlm.nih.gov/?term=%22Dierking%20K%22%5BAuthor%5D)

2, [Andre Franke](https://pubmed.ncbi.nlm.nih.gov/?term=%22Franke%20A%22%5BAuthor%5D)

### Andre Franke

6Institute of Clinical Molecular Biology, Kiel University, Kiel, Schleswig-Holstein, Germany

Resources, Writing – review and editing

Find articles by [Andre Franke](https://pubmed.ncbi.nlm.nih.gov/?term=%22Franke%20A%22%5BAuthor%5D)

6, [Matthias Laudes](https://pubmed.ncbi.nlm.nih.gov/?term=%22Laudes%20M%22%5BAuthor%5D)

### Matthias Laudes

7Institute of Diabetes and Clinical Metabolic Research, University Hospital Schleswig-Holstein Campus Kiel, Kiel, Schleswig-Holstein, Germany

Resources, Writing – review and editing

Find articles by [Matthias Laudes](https://pubmed.ncbi.nlm.nih.gov/?term=%22Laudes%20M%22%5BAuthor%5D)

7, [Silvio Waschina](https://pubmed.ncbi.nlm.nih.gov/?term=%22Waschina%20S%22%5BAuthor%5D)

### Silvio Waschina

8Nutriinformatics, Institute for Human Nutrition and Food Science, Kiel University, Kiel, Schleswig-Holstein, Germany

Software, Writing – review and editing

Find articles by [Silvio Waschina](https://pubmed.ncbi.nlm.nih.gov/?term=%22Waschina%20S%22%5BAuthor%5D)

8, [Hinrich Schulenburg](https://pubmed.ncbi.nlm.nih.gov/?term=%22Schulenburg%20H%22%5BAuthor%5D)

### Hinrich Schulenburg

2Research Group Evolutionary Ecology and Genetics, Zoological Institute, Kiel University, Kiel, Schleswig-Holstein, Germany

4Max-Planck Institute for Evolutionary Biology, Ploen, Schleswig-Holstein, Germany

Conceptualization, Funding acquisition, Supervision, Writing – review and editing, Methodology, Resources

Find articles by [Hinrich Schulenburg](https://pubmed.ncbi.nlm.nih.gov/?term=%22Schulenburg%20H%22%5BAuthor%5D)

2,4,✉, [Christoph Kaleta](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaleta%20C%22%5BAuthor%5D)

### Christoph Kaleta

1Research Group Medical Systems Biology, University Hospital Schleswig-Holstein Campus Kiel, Kiel University, Kiel, Schleswig-Holstein, Germany

Conceptualization, Funding acquisition, Project administration, Supervision, Writing – original draft, Writing – review and editing, Formal analysis, Investigation, Methodology, Resources, Software

Find articles by [Christoph Kaleta](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaleta%20C%22%5BAuthor%5D)

1,✉

Editor: Paul A Jensen9

Reviewed by: Hyun-Seob Song10

* Author information
* Article notes
* Copyright and License information

1Research Group Medical Systems Biology, University Hospital Schleswig-Holstein Campus Kiel, Kiel University, Kiel, Schleswig-Holstein, Germany

2Research Group Evolutionary Ecology and Genetics, Zoological Institute, Kiel University, Kiel, Schleswig-Holstein, Germany

3Department of Integrative Biology, University of California, Berkeley, California, USA

4Max-Planck Institute for Evolutionary Biology, Ploen, Schleswig-Holstein, Germany

5Alkek Center for Metagenomics and Microbiome Research, Baylor College of Medicine, Houston, Texas, USA

6Institute of Clinical Molecular Biology, Kiel University, Kiel, Schleswig-Holstein, Germany

7Institute of Diabetes and Clinical Metabolic Research, University Hospital Schleswig-Holstein Campus Kiel, Kiel, Schleswig-Holstein, Germany

8Nutriinformatics, Institute for Human Nutrition and Food Science, Kiel University, Kiel, Schleswig-Holstein, Germany

9University of Michigan-Ann Arbor, Ann Arbor, Michigan, USA

10University of Nebraska-Lincoln, Lincoln, Nebraska, USA

✉

Address correspondence to Christoph Kaleta, c.kaleta@iem.uni-kiel.de

✉

Address correspondence to Hinrich Schulenburg, hschulenburg@zoologie.uni-kiel.de

#

Georgios Marinos and Inga K. Hamerich contributed equally to this article. Author order was determined in order of increasing seniority.

Hinrich Schulenburg and Christoph Kaleta are joint senior authors.

The authors declare no conflict of interest.

#

Contributed equally.

#### Roles

**Georgios Marinos**: Conceptualization, Formal analysis, Investigation, Methodology, Software, Writing – original draft, Writing – review and editing, Data curation, Funding acquisition, Project administration, Visualization

**Inga K Hamerich**: Formal analysis, Investigation, Methodology, Writing – original draft, Writing – review and editing, Data curation, Software, Visualization

**Reena Debray**: Formal analysis, Investigation, Writing – review and editing, Data curation, Funding acquisition, Methodology, Software, Visualization

**Nancy Obeng**: Formal analysis, Investigation, Writing – review and editing, Conceptualization, Methodology, Supervision

**Carola Petersen**: Investigation, Writing – review and editing, Methodology, Software

**Jan Taubenheim**: Writing – review and editing, Methodology, Software

**Johannes Zimmermann**: Methodology, Writing – review and editing, Software

**Dana Blackburn**: Resources, Writing – review and editing, Methodology

**Buck S Samuel**: Resources, Writing – review and editing, Funding acquisition, Methodology

**Katja Dierking**: Resources, Writing – review and editing, Funding acquisition, Methodology

**Andre Franke**: Resources, Writing – review and editing

**Matthias Laudes**: Resources, Writing – review and editing

**Silvio Waschina**: Software, Writing – review and editing

**Hinrich Schulenburg**: Conceptualization, Funding acquisition, Supervision, Writing – review and editing, Methodology, Resources

**Christoph Kaleta**: Conceptualization, Funding acquisition, Project administration, Supervision, Writing – original draft, Writing – review and editing, Formal analysis, Investigation, Methodology, Resources, Software

**Paul A Jensen**: Editor

**Hyun-Seob Song**: Ad Hoc Peer Reviewer

Received 2023 Mar 17; Accepted 2023 Dec 13; Collection date 2024 Feb.

Copyright © 2024 Marinos et al.

This is an open-access article distributed under the terms of the [Creative Commons Attribution 4.0 International license](https://creativecommons.org/licenses/by/4.0/).

[PMC Copyright notice](/about/copyright/)

PMCID: PMC10846184  PMID: [38230938](https://pubmed.ncbi.nlm.nih.gov/38230938/)

**Previous version available:** This article is based on a previously available preprint posted on bioRxiv on February 18, 2023: "[Metabolic model predictions enable targeted microbiome manipulation through precision prebiotics](/articles/PMC9949166/)".

## ABSTRACT

While numerous health-beneficial interactions between host and microbiota have been identified, there is still a lack of targeted approaches for modulating these interactions. Thus, we here identify precision prebiotics that specifically modulate the abundance of a microbiome member species of interest. In the first step, we show that defining precision prebiotics by compounds that are only taken up by the target species but no other species in a community is usually not possible due to overlapping metabolic niches. Subsequently, we use metabolic modeling to identify precision prebiotics for a two-member *Caenorhabditis elegans* microbiome community comprising the immune-protective target species *Pseudomonas lurida* MYb11 and the persistent colonizer *Ochrobactrum vermis* MYb71. We experimentally confirm four of the predicted precision prebiotics, L-serine, L-threonine, D-mannitol, and γ-aminobutyric acid, to specifically increase the abundance of MYb11. L-serine was further assessed *in vivo*, leading to an increase in MYb11 abundance also in the worm host. Overall, our findings demonstrate that metabolic modeling is an effective tool for the design of precision prebiotics as an important cornerstone for future microbiome-targeted therapies.

### IMPORTANCE

While various mechanisms through which the microbiome influences disease processes in the host have been identified, there are still only few approaches that allow for targeted manipulation of microbiome composition as a first step toward microbiome-based therapies. Here, we propose the concept of precision prebiotics that allow to boost the abundance of already resident health-beneficial microbial species in a microbiome. We present a constraint-based modeling pipeline to predict precision prebiotics for a minimal microbial community in the worm *Caenorhabditis elegans* comprising the host-beneficial *Pseudomonas lurida* MYb11 and the persistent colonizer *Ochrobactrum vermis* MYb71 with the aim to boost the growth of MYb11. Experimentally testing four of the predicted precision prebiotics, we confirm that they are specifically able to increase the abundance of MYb11 *in vitro* and *in vivo*. These results demonstrate that constraint-based modeling could be an important tool for the development of targeted microbiome-based therapies against human diseases.

**KEYWORDS:** nutritional supplements, flux balance analysis, genome-scale metabolic models, *Caenorhabditis elegans*, *Pseudomonas lurida* MYb11, *Ochrobactrum vermis* MYb71, serine

## INTRODUCTION

While traditionally considered commensals, it is becoming increasingly clear that the microbial communities living within and on higher host organisms (i.e., the microbiome) are integral parts of host physiology. These microbial communities fulfill essential roles in pathogen protection ([1](#B1)), immune system education ([2](#B2)), production of vitamins ([3](#B3)), metabolism of xenobiotics ([4](#B4), [5](#B5)), and increase the plasticity of their host to respond to evolutionary challenges ([6](#B6), [7](#B7)). Thus, the composition of the microbiota plays an important role in the well-being of hosts and its ability to respond to environmental challenges ([8](#B8), [9](#B9)), as indicated by the frequent shift of microbiota composition observed in the context of diseases ([10](#B10)) and exposure to environmental stressors ([11](#B11)). These observations and work in animal models have led to the development of the notion that the composition of the microbiome itself might be an important modulator of host health ([12](#B12)) as well as fitness and that a targeted modulation of microbiome composition might offer a completely new avenue for modulating the host.

It has been shown that dietary patterns (e.g., western diet) can affect the structure of the microbiome, however their effect is transient ([13](#B13)). Besides an influence of dietary uptake on microbiome composition, various approaches to modify the microbial communities of organisms have been used ([14](#B14)): the transfer of entire microbial communities from a donor to a recipient host (fecal transfer), the provision of compounds used by particular microbial species (prebiotics), the provision of microbial strains of natural or synthetic origin (probiotics), the provision of compounds of microbial origin (postbiotics), and combinations thereof (synbiotics). While fecal transfer is the most commonly used approach to modulate microbial community composition in animal models, it involves the transfer of entire microbial communities and therefore not only desirable traits. In consequence, there is only a single medical indication in which fecal transfers are routinely used thus far ([15](#B15)) and there are potential concerns about the safety and stability of the engraftment of transplants ([16](#B16)).

Although the concept of supplementation with pre-, pro-, or synbiotics has been commonly proposed as a method of microbial manipulation, these approaches are still severely limited. In humans, oligosaccharides are proposed as prebiotics to alter the microbial community on both the structural and functional level, since they can be fermented by microbial species, which in turn can proliferate and produce bioactive compounds such as short-chain fatty acids ([17](#B17)). However, there is only little knowledge about other classes of compounds that might alter the microbial composition. Another important aspect to consider is the specificity with which individual target microorganisms can be manipulated using prebiotics. Thus, while oligosaccharides are considered beneficial, there is often uncertainty about which species can degrade them, as multiple, unrelated species can degrade similar types of oligosaccharides (e.g., *Bacteroidetes*, *Lactobacilli*, and *Bifidobacterium* species can degrade fructans) ([17](#B17), [18](#B18)). In case of supplementation with probiotics and synbiotics, other concerns have been raised. For instance, the ability of specific Bifidobacteria to stably colonize the human gut was only reported for ⅓ of the participants in a clinical trial ([19](#B19)). Moreover, introducing exogenous bacteria can have unwanted side effects. For example, some of the *Bacillus* species that were used as probiotics in aquacultures were found to carry virulence and antibiotic-resistance genes ([20](#B20)). Hence, to overcome these limitations in current microbiome manipulation approaches, we need to focus on prebiotics with well-controlled modes of action (e.g., based on known microbial pathways that can degrade prebiotics) that preferentially modulate the abundance of resident microbial species. Since mechanisms of pathogenicity are often complex and context-specific ([21](#B21)), it is preferred if these species are native to the microbiome of the host, as unwanted side-effects due to the introduction of foreign bacterial species might be avoided. Within this work, we will refer to prebiotics that specifically modulate the abundance of selected bacterial strains within the microbiome of a specific host as “precision prebiotics.”

To design and evaluate such precision prebiotics, it is essential to start with a tractable model system such as the well-established model organism *Caenorhabditis elegans*. The native microbial community of the invertebrate *C. elegans* was first described in 2016 ([22](#B22)), yet the nematode has already been used for over 20 years as a model for studying host-microbial interactions ([23](#B23)). To similarly promote *C. elegans*-microbiome research, a synthetic 12-member *C. elegans*-specific microbiome resource, *CeMbio*, was recently established ([24](#B24)). Part of this consortium is the Gram-negative *Pseudomonas lurida* MYb11 that is known to possess antifungal ([22](#B22)) and antimicrobial properties, thereby providing protection against the pathogen *Bacillus thuringiensis* (Bt) ([21](#B21), [25](#B25)). Besides its protective effects, MYb11 is known to colonize and persist in the *C. elegans* intestine ([22](#B22)) and thus increase the nematode’s fitness during infection ([25](#B25)). *Ochrobactrum vermis* MYb71 is another member of the *CeMbio* community ([24](#B24)) and affects different host life-history traits ([24](#B24), [26](#B26), [27](#B27)), persistently colonizes the nematode ([22](#B22)), and has been described as competitive against other bacteria in terms of space and nutrients ([27](#B27)). Furthermore, the bacterial strain MYb11 was only able to grow on sucrose when co-cultured with MYb71 but not when cultured alone, suggesting that the two bacterial strains interact metabolically ([27](#B27)). Hence, a microbial community consisting of one beneficial bacterium and another one that is competitive offers an excellent opportunity for developing targeted approaches for microbiome manipulation.

A key prerequisite for the development of precision prebiotics is a detailed understanding of the metabolic functions present within a microbial community and how specific metabolic interventions affect microbiome metabolism. One important approach to investigate metabolic functions in microbial communities is constraint-based metabolic modeling ([28](#B28)[–](#B29)[30](#B30)). Based on the genetic information either on the level of each bacterium or from the overall community and biochemical databases, these methods allow one to simulate the network of chemical reactions that represent the metabolic reactions of the microorganism(s) or the community ([31](#B31)). To this end, genome-scale metabolic models of the bacterial species within a microbiome have to be reconstructed from genomic data. These models summarize the metabolic functions present within an organism and can be reconstructed using automated reconstruction approaches ([32](#B32), [33](#B33)) or obtained from existing model repositories ([34](#B34)). Furthermore, accounting for the growth environment of a species ([35](#B35)), optimization approaches such as flux balance analysis (FBA) for individual species ([28](#B28)) or constraint-based community modeling approaches ([36](#B36), [37](#B37)) can be used to predict metabolic activities within individual species as well as interactions between species. These methods have already been used to identify metabolic alterations in the microbiome of Parkinson’s disease patients ([38](#B38)) as well as patients with inflammatory bowel disease ([39](#B39)) and to study the interaction of *C. elegans* as well as its microbiota in the context of drug-microbiome-host interactions ([30](#B30)). For *C. elegans*, metabolic models of the *CeMbio* community including MYb11 and MYb71 ([24](#B24)) and other members of the microbiota are available ([27](#B27)).

In this work, we used metabolic networks of MYb11 and MYb71 also incorporating phenotypic information from *Biolog* growth assays in a computational screen for precision prebiotics that are able to specifically boost the abundance of the host-protective strain MYb11 over MYb71. We tested four of the predicted compounds, L-serine, L-threonine, D-mannitol, and γ-aminobutyric acid (GABA) *in vitro* and confirmed that they selectively boost the abundance of MYb11. Moving to the *C. elegans* host system, we moreover showed that L-serine is able to selectively boost the abundance of MYb11 when the *C. elegans* host was colonized with both bacterial species. Thus, we demonstrate that constraint-based microbial community modeling approaches could represent a key tool for the design of targeted microbiome intervention approaches.

## MATERIALS AND METHODS

### Computational methods

#### Identification of unique uptake compounds

We explored to what extent individual metabolites, which are only taken up by a single bacterial species and no other, could serve as potential supplements to target individual species. To this end, we considered gut microbiome data from a Kiel-based human cohort comprising 1,280 participants for which 16S rRNA gene sequencing data is available, which we had utilized previously ([30](#B30)). As reported previously, the reads were mapped to the 16S sequences of the genomes of bacterial species present in the AGORA bacterial collection ([34](#B34)), the read counts of each sample were normalized to a sum of 1 and removal of species below 0.1% abundance in each sample. We used *gapseq* version 1.1 ([32](#B32)) on the genomes of the identified species assuming the averaged dietary uptake of the human cohort ([30](#B30)) as input to the models to reconstruct genome-scale metabolic models. Subsequently, we identified for each bacterial species which metabolites can be taken up. To this end, for each exchange reaction of a metabolite, we set the respective reaction as an objective reaction. By convention, bacterial uptake of metabolites is modeled by exchange reactions where the forward direction corresponds to an excretion of a compound and the backward direction (i.e., negative flux) to its uptake. Therefore, we minimized the flux through each exchange reaction using FBA ([40](#B40)) implemented in the *R*-package *sybil* ([41](#B41)) which corresponds to maximizing the uptake of that metabolite. If the minimal flux through the exchange reaction was non-zero this metabolite could be taken up. Having repeated this procedure for each exchange reaction, we identified those compounds that can be taken up for each species. Then, we determined for each individual bacterial species in the gut microbiome of each participant whether it could take up a compound that could not be taken up by any other species in the bacterial community of this participant. Finally, we determined for each participant which fraction of the bacterial species contained in the gut microbiome had at least one unique uptake compound. Similarly, we reconstructed 78 bacterial models belonging to the *C. elegans* microbiota based on their published genomes ([27](#B27)) using *gapseq* version 1.2 ([32](#B32)). We then created random subsets of the models with increasing sizes (from 2 to 78, 50 iterations for each size) and determined the portion of strains with unique uptake compounds for each community. For the *CeMbio* community we used metabolic models reconstructed with *gapseq* 1.2 ([32](#B32)) from Zimmermann et al. ([42](#B42)). In general, the software *gapseq* ([32](#B32)) has an extensive published documentation regarding the reconstruction of genome-scale metabolic models, also available online (<https://gapseq.readthedocs.io/en/latest/>).

#### Computational supplementation experiments

In our simulations, we utilized metabolic networks of *P. lurida* MYb11 and *O. vermis* MYb71 that were created from their genome sequences ([27](#B27)) using *gapseq* ([32](#B32)) version 1.1. Information from phenotypic microarrays about the organisms' metabolic capacities was integrated into the models using the “adapt” module of *gapseq*. Specifically, we incorporated data from the *Biolog* EcoPlate (assuming growth if OD590–OD750 >0.1) ([24](#B24)). During growth simulations, exchange fluxes of the models were constrained by assuming a computational nematode growth medium (NGM) ([30](#B30)) and aerobic conditions. These constraints can be found in Table S1. Additionally, the inflow of copper and iron cations was increased, so that they did not restrict the growth of the models. Following published practices ([36](#B36)) and due to the lack of the required kinetic information, we used the concentration values of the NGM medium input for the lower bounds of the models.

We used three distinct approaches for modeling individual microbes and microbial communities. To model the growth of each bacterium independently, we used FBA ([40](#B40)). We also utilized individual-based modeling based on *BacArena* ([36](#B36)) *R*-package and a community-FBA as described previously ([39](#B39), [43](#B43), [44](#B44)). The community-FBA approach is implemented in the *R*-package *MicrobiomeGS2* ([www.github.com/Waschina/MicrobiomeGS2](http://www.github.com/Waschina/MicrobiomeGS2)) and an overview has been described in detail in the publication by Pryor et al. ([30](#B30)). To measure the effects of supplements on growth of the individual species, we modified the published approach, so that we did not fix species composition but let it be determined as part of the optimization of individual species biomasses. In all types of simulations, we focused on the identification of nutritional supplements that boosted the growth of the target species MYb11.

For the individual supplementation experiments, the metabolic models of both organisms were constrained with the NGM computational diet, assuming that the molecular values of the dietary compounds (molecular quantities) can be directly incorporated into the models (as fluxes in the lower bounds of the models) without any transformation. This is based on the assumption that the uptake rate of compounds is depicted by its maximum available amount ([36](#B36)). Supplementations for each metabolite were conducted by increasing the possible influx of the supplemented metabolite by 10 mmol/L. For testing the effects of supplements on single growth, the relative increase in growth rate compared to growth without supplements for each model was recorded. To prevent any numerical instabilities, all raw values were rounded to the sixth digit and relative changes bigger than 0.01 were taken into account.

For simulations using community FBA, the metabolic models of both organisms were merged into one microbial community, compartmentalized, model. Please note that we did not use a fixed community biomass composition as previously ([30](#B30)), since we employed community FBA to predict the resulting growth rates of individual species after maximizing the growth of the community. Thus, the community model was constrained with the NGM diet—extended with the individual supplements, as described in the previous paragraph. The growth rates of individual species were predicted by maximizing the sum of the growth rates of individual species with concomitant minimization of the sum of total fluxes as implemented in the *R*-package *sybil* ([41](#B41)) (with a coefficient of −10−6 for the sum of fluxes in the objective function). Since the supplementations usually also increased the growth of the entire community, the predicted growth rates of each species were normalized with the respective growth rate of the community. Predicted growth rates following supplementation were contrasted against growth rates without supplementation to identify compounds whose effect was larger for the growth of MYb11 in comparison to MYb71. To exclude any numerical artifacts, rounding to the sixth digit and a cutoff of 0.01 in compound selection were also applied in this section.

For individual-based modeling of microbial communities, the metabolic models of both species were simulated in a virtual environment in *BacArena*, where they could randomly move and reproduce ([36](#B36)) on a 30 × 30 grid. As a growth medium, we used the NGM diet (in mM) which we diluted 1,000-fold, because otherwise bacterial models overgrew the entire simulation environment before nutrients were exhausted. For the simulations, we inoculated the growth environment with 20 individuals of each species and simulated bacterial growth for 12 iterations with 15 replicates for each supplementation using FBA. Replicates are needed since, in contrast to FBA, *BacArena* includes a stochastic component such as a random initial placement of individuals, random movement of individuals and random cellular replication events. A secondary objective to minimize the total flux while maximizing the growth rates of individuals was applied. To simulate the supplementation experiments, 0.01 mM of each compound of interest was added once in the beginning of the simulation. To identify significant changes in the growth profiles of individual species following each supplementation, we extracted the cell biomass information from the simulations using the command *plotGrowthCurve* and we calculated the relative change of the total biomass values (time step 13; rounding to the sixth digit) due the supplementations for each model. Then, the relative changes between the bacterial models were compared using Wilcoxon rank-sum tests ([45](#B45)). The resulting *P* values were corrected for multiple testing using false discovery rate control implemented in *R*. We selected those compounds whose effect on MYb11 was larger than on MYb71 based on the median of the respective relative changes for each model.

Based on these simulation techniques, we identified those compounds that specifically boosted MYb11 across the simulations. Subsequently, we subsetted this list to compounds for which we had positive confirmation of their uptake by MYb11 from *Biolog* experiments ([24](#B24), [27](#B27)). For the enrichment analysis of growth-supporting compounds of MYb11, we used the human metabolome database (HMDB) annotation for the available metabolite classes ([46](#B46)). Briefly, we tested whether the sets of candidate compounds were enriched in the metabolite classes defined in HMDB using Fisher’s exact test. As the underlying set of metabolites, we used the metabolites contained in the metabolic model of MYb11 in the different metabolite classes.

#### System information and software

Simulations and analyses were performed in the *R* environment (versions 4.0.0, 4.0.3, and 4.2.1). Model reconstructions were performed with the *gapseq* software (versions 1.1 and 1.2) ([32](#B32)) Optimizations were generally conducted using the software *sybil* (versions 2.1.5 and 2.2.0) ([41](#B41)). Specifically, multi-model supplementation was conducted using the software *MicrobiomeGS2* (version 0.1.5, [www.github.com/Waschina/MicrobiomeGS2](http://www.github.com/Waschina/MicrobiomeGS2)) and the in-silico co-culturing was simulated on *BacArena* (version 1.8.2, commit fdb02bf7) ([36](#B36)). Both incorporate *sybil* in their pipelines. The linear programming solver was *CPLEX* (versions 12.10.0 and 22.1.0), based on the *R* package *cplexAPI* (version 1.4.0). For data management, the software *dplyr* (versions 1.0.2, 1.0.4, and 1.0.9), *tidyr* (version 1.1.3), *tidyverse* (version 1.3.2) ([47](#B47)), and *data.table* (version 1.14.2) were used. For parallel computing, the software *foreach* (version 1.5.0 and 1.5.1) and *doMC* (version 12.10.0) were used. For plotting, the software *ggplot2* (versions 3.3.3 and 3.3.6) ([48](#B48)), *egg* (version 0.4.5), and *ggVennDiagram* (version 1.2.2.) were utilized.

### Experimental methods

#### Bacterial strains

To determine bacterial growth in mono- and co-culture either in the presence or absence of supplementation, similarly to the computational section, the two naturally associated *C. elegans* microbiota isolates *P. lurida* MYb11 and *O. vermis* MYb71 were used. Fluorescently labeled MYb11::dTomato ([21](#B21)) and MYb71::GFP were used to distinguish the two bacterial species in a mixed community. For comparison, reverse labels were also tested, using MYb71::dTomato and MYb11::sfGFP, to ensure no impact of the fluorescent label (Fig. S2). Integrated fluorescent strains of MYb71 and the MYb11::sfGFP strain were constructed by transposon insertion at the *attTn7* site following published work ([49](#B49)). Briefly, two populations of *Escherichia coli* SM10 strains were prepared: (i) a donor strain containing either pTn7xKS-sfGFP (pTW415) or pTn7xKS-dTomato (pTW416) plasmids, and (ii) a helper strain containing the pTNS2 (pTW10) plasmid. Liquid cultures (5 mL LB) of donor, helper (both 37°C) and target strains (26°C) were grown with agitation overnight, subcultured, and collected at an OD 0.4–0.6. Tripartite mating mixtures consisted of equal parts (1:1:1), incubated overnight at 26°C, followed by selection on LB agar plates containing gentamicin (10 µg/mL) and IPTG (1 mM). Fluorescent colonies of MYb71 were verified by PCR and sequencing using two pairs of MYb71-specific primers to the glmS gene region: (i) oDB097 (5′-*CGCTCTGATCGATGAGACC*-3′) and oDB098 (5′-*TTGCGCGGCTG*RTCGAC-3′), and (ii) oDB097 and a transposon WP11 primer (5′-*CACGCCCCTCTTTAATACGA*-3′) ([49](#B49)). MYb11::sfGFP was similarly confirmed by PCR and sequencing, following our previous protocol ([21](#B21)).

Bacteria were thawed from frozen stocks for each experiment, streaked to tryptic soy agar (TSA) plates and grown for 16–42 h at 25°C. Bacterial cultures of MYb11 and MYb71 were grown in liquid NGM overnight at 28°C in a shaking incubator. Cultures were adjusted to OD600 = 0.5 with phosphate-buffered saline (PBS). We combined equal volumes of the adjusted cultures and used these as starting co-cultures for all experiments with the two strains, thus ensuring identical starting conditions across experimental treatments, as required for inference of an influence of the experimental manipulation on relative bacterial abundances.

#### Supplementation *in vitro*

L-serine (1714.1), D-mannitol (8883.1), and L-threonine (T206.1) were purchased from Carl Roth (Karlsruhe, Germany), and GABA (A2129) from Merck (Darmstadt, Germany) and dissolved to 1 M or 0.5 M stocks in water, depending on the maximum soluble concentration. Liquid NGM was supplemented with a final concentration of 10 mM of the respective supplement. For supplementation assays *in vitro*, 5 µL of either mono- or co-culture of the bacteria was added to 95 µL of liquid NGM in a 96-well plate, in the presence or absence of supplement. Treatments were randomized with respect to plate layout. Plates were incubated at 28°C for 24 h in an Epoch2 BioTek plate reader with double orbital shaking at 807 CPM. We assessed changes in bacterial abundances by quantifying the number of colony-forming units (CFUs) of a sample of the cultures plated out on agar, because this allows us to distinguish between the two strains, which is not possible using standard OD measurements. For this approach, 7 µL of different dilutions of each treatment bacterial suspension of respective mono- or co-cultures was plated onto TSA plates. The plates were stored at 25°C and CFUs scored after 24–48 h. Statistical calculations and plots were performed with *R* software (version 4.1.3). For all CFU assays *in vitro*, statistical comparisons were performed with Wilcoxon signed-rank tests ([45](#B45)).

#### Supplementation *in vivo*

The *C. elegans* strain N2 was used for all *in vivo* experiments and was originally obtained from the Caenorhabditis Genetics Center (CGC). Worms were maintained on NGM plates inoculated with *E. coli* OP50 at 20°C following standard procedures ([50](#B50)). Assay plates were prepared by adding L-serine at a concentration of 0, 10, or 100 mM to NGM. Please note that serine can affect *C. elegans* foraging behavior ([51](#B51)). However and importantly for our proof-of-concept study, none of the numerous *C. elegans* foraging studies [including our own long-term experience ([52](#B52)[–](#B53)[54](#B54))] indicated that this nematode is capable of a differential uptake of distinct bacterial strains from a mixed bacterial lawn. Therefore, any possible change in serine-induced nematode behavior is unlikely to cause any changes in the relative abundance of bacterial strains in the *C. elegans* gut—as long as identical starting cultures are used across experimental treatments, which we ensured with our experimental protocol. Overnight cultures of MYb11 and MYb71 were adjusted to OD600 = 2.0 in PBS. Thirty to forty synchronized first instar larvae (L1) were grown on supplemented NGM plates inoculated with 125 µL of respective mono- or co-culture of MYb11 and MYb71. After 72 h, bacterial lawns were sampled by pressing the circular end of a 200-µL pipette tip into the lawn, avoiding the edges and any worms. The resulting disc was homogenized in PBS with zirconia beads in a Bead Ruptor 96 (Omni International) for 3 min at 30 Hz.

To assess the number of CFU, worms were washed off plates using M9-buffer + 0.025% Triton X-100, washed five times and paralyzed with 5 mM tetramisole to stop pumping of the worms as described in ([24](#B24)). Worms were surface-sterilized following ([24](#B24)), then washed two times in PBS to remove residual bleach. Worms were transferred to a new tube to determine the exact worm number (~20) and PBS was added to a final volume of 400 µL. Worms were allowed to settle and 100 µL supernatant was collected (supernatant control). Worms were homogenized in the remaining PBS using 1 mm zirconia beads in a Bead Ruptor 96 for 3 min at 30 Hz. Homogenized worms, lawns, and supernatants were serially diluted in PBS and plated on TSA plates. After up to 48 h at 25°C, the colonies were scored at the appropriate dilutions and the CFUs per worm were calculated. Each colonization experiment was conducted in three independent runs and equal biological replicates. Data were pooled across runs, and a mixed model controlling for runs was used to assess the effect of L-serine supplementation on bacterial colonization in *C. elegans*. For all L-serine *in vitro* and *in vivo* experiments, statistical comparisons were performed with generalized linear models, if not otherwise mentioned. We calculated the expected proportion of MYb11 in worms as the product of MYb11 proportions available on the lawns and the observed shift in bacterial proportions due to host filtering:

|  |
| --- |
| E(Prop. MYb11 in supplemented worms)=(Prop. MYb11 onsupplemented lawns)×Prop. MYb11 in wormsProp. MYb11 on lawns |

Plots and statistical analysis were produced with *R* (version 4.1.3) and edited with Inkscape (version 1.1.2).

## RESULTS

### Overlapping metabolic niches between microbiome member species prevent the use of unique uptake compounds as precision prebiotics

The simplest approach to specifically boost the abundance of a particular bacterial species in a microbial community is to supply the community with a compound that is only taken up by the target species but no other species. Hence, these compounds correspond to unique metabolic niches for the respective bacterial species. We initially tested computationally whether such unique uptake compounds typically exist in bacterial communities, using bacterial communities from a human cohort as well as *C. elegans* ([30](#B30)). We evaluated for each bacterial species of a particular community whether it was able to take up a metabolite that no other species of the community could take up and determined the fraction of bacterial species in the community that had such unique uptake compounds (see Materials and Methods). For the human cohort, the bacterial communities comprised on average 26–34 bacterial species (25–75% quantiles). Within this community size, on average, 5–20% of bacterial species were inferred to possess unique uptake compounds that could potentially be used to target those species. For smaller communities, up to 50% of bacterial species and for larger communities, only 5–10% had unique uptake compounds ([Fig. 1A](#F1)). For *C. elegans*, we do not have matching data like for the human cohort with species abundance information for a large number of samples and genome sequences of the corresponding bacterial species for metabolic modeling. However, we can use strains that have been isolated from *C. elegans* and its environment as well as artificial minimal *C. elegans* communities to approximate the frequency of unique uptake compounds in the *C. elegans* microbiota. Thus, within a sample of 78 bacterial species isolated from *C. elegans* and its environment, we usually only identified 3–5% of species with unique uptake compounds in communities comprising 30 or more bacterial species, which is the typical size of bacterial communities observed in *C. elegans*-derived microbiome samples ([55](#B55)) ([Fig. 1B](#F1)). Similarly, in the artificial *CeMbio C. elegans* model microbiome community unique uptake compounds could usually only be detected for subsets of bacterial species up to a size of six member species ([Fig. 1C](#F1)).

#### Fig 1.

[![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4779/10846184/2a80f3636aa8/spectrum.01144-23.f001.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10846184_spectrum.01144-23.f001.jpg)

[Open in a new tab](figure/F1/)

Frequency of species with unique uptake compounds in microbiomes. Fraction of species with unique uptake compounds in (**A**) human gut bacterial communities and in (**B and C**) *C. elegans* bacterial communities that are subsets of previously reported *C. elegans* microbiome member species.

Thus, in a typical microbiome like those analyzed above, we would expect to be able to identify unique uptake compounds only for a minority of bacterial species and compounds to target specific bacterial species are typically expected to be consumed by several species. Thus, for our identification of precision prebiotics for MYb11, we first excluded 20 compounds from our search that could specifically be taken up only by MYb11 in the search for a MYb-targeted prebiotic (20 compounds, see Table S2). Of note, MYb11 and MYb71 share the vast majority of all uptake compounds (85%), while only 10% and 5% of all uptake compounds are unique to MYb11 and MYb71, respectively ([Fig. 2A](#F2)).

#### Fig 2.

[![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4779/10846184/74fb4ffebddc/spectrum.01144-23.f002.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10846184_spectrum.01144-23.f002.jpg)

[Open in a new tab](figure/F2/)

Metabolic-modeling guided identification of precision prebiotics. (**A**) Overlap of all exchange reactions across the models MYb11 and MYb71. (**B**) Community FBA supplementation of *Pseudomonas lurida MYb11* (pink) and *Ochrobactrum vermis MYb71* (green) in *MicrobiomeGS2*. Predicted relative abundance of MYb11 and MYb71 under the supplementation with 4-hydroxybenzoate, D-mannitol, GABA, glycerol-3-phosphate, L-serine, L-threonine, putrescine, and threhalose shown in dots, lines represent predicted relative abundance without any supplementation. (**C**) *In-silico* supplementation of *Pseudomonas lurida MYb11* (pink) and *Ochrobactrum vermis MYb71* (green) in *BacArena*. Predicted growth curves of MYb11 and MYb71 either with 10 mM of 4-hydroxybenzoate, D-mannitol, GABA, glycerol-3-phosphate, L-serine, L-threonine, putrescine, or trehalose(solid line) or without supplementation (dashed line). (**D**) Comparison of predicted precision prebiotics for MYb11 in community with MYb71 with different computational approaches (first three columns) and growth data on *Biolog* EcoPlates and *Biolog* GN2 plates ([24](#B24), [27](#B27)) (last two columns). Full squares represent cases in which a metabolite was predicted as precision prebiotic for MYb11, empty squares if modeling predicted no increased growth of MYb11 over MYb71. For the *Biolog* data, filled triangles represent cases of catabolization of the respective compound and empty ones if no catabolization was observed. Missing circles indicate compounds not tested by Biolog plates.

### Computational identification of precision prebiotics targeting *P. lurida* MYb11

In order to derive precision prebiotics for *P. lurida* MYb11, we used its metabolic network as input for flux balance analysis and two distinct community modeling approaches apart from the basic approach of FBA. The first one is based on *BacArena* that incorporates an individual-based modeling approach and the other one is so-called community FBA implemented in *MicrobiomeGS2*. These two modeling approaches differ in their underlying assumptions. In *BacArena* the two bacteria are modeled by their individual metabolic networks and fluxes are predicted by maximizing the biomass fluxes of individual bacteria. In community FBA, in turn, the metabolic networks of the individual bacterial species are merged into a community model for which the sum of biomass fluxes of both bacterial species is maximized. Thus, when using *BacArena*, metabolic interactions between species typically arise from one species secreting an end-product that another species can metabolize. In community FBA, in turn, the optimization of total growth assumes that bacterial species coordinate their metabolic fluxes so that overall growth is maximized. Thus, since we do not know to which extent MYb11 and MYb71 mutually interact with each other, we cover both ends of the spectrum of potential interactions between species from completely individually optimized behavior in *BacArena* to an absolute coordination of fluxes to maximize overall community growth.

Using FBA, we identified 29 compounds that could increase the growth of MYb11 in single growth. Employing the community modeling approaches, we identified 81 compounds that increased the growth of MYb11 over MYb71 in community FBA and 17 compounds using *BacArena* ([Fig. 2D](#F2)). In order to determine whether specific compound classes were enriched among the MYb11-supporting compounds, we performed an enrichment based on HMDB annotation ([46](#B46)) across the three different simulation types. We found that growth-supporting compounds for MYb11 were enriched for fatty acids (Fisher’s exact test *P* value = 3.16 × 10−5).

Interestingly, the growth dynamics of the models when they were part of a community model ([Fig. 2B](#F2)) differ from the growth dynamics of the models when they were treated as individual organisms ([Fig. 2C](#F2)). For instance, MYb71’s biomass flux was much higher in the community model, i.e., when coupled to the model MYb11. This is a typical observation when using community FBA when the biomass of the entire community is maximized. Especially for small communities, this often leads to one community member that most efficiently turns the growth medium into biomass to grow strongest with only little growth for other community members. On the other hand, in the *BacArena* simulations, we observed that most of those compounds boosted the growth of both species in single growth, however, the effect was larger for MYb11 than MYb71 ([Fig. 2C](#F2)). While *BacArena* also uses FBA to simulate bacterial growth, only a subset of FBA-predicted compounds that increase the growth of MYb11 over MYb71 was actually also identified by the *BacArena* approach (17 out of 29). The likely reason is that the FBA approach does not take into account potential interactions between species and that *BacArena* incorporates a stochastic component, for instance, for determining positioning as well as movement of bacteria and cellular replication events, such that small differences in growth rate between bacterial species might be filtered out by these stochastic events. Of the identified compounds, 17 compounds were shared across the three simulations. For eight of these, also information on uptake by MYb11 was available from *Biolog* plates which confirmed that MYb11 was able to consume them ([24](#B24), [27](#B27)) ([Fig. 2D](#F2)). Thus, combining modeling approaches and cross-referencing data from *Biolog* plates allowed us to considerably reduce our list of candidates to test experimentally to those giving the most consistent results. Based on the available literature information on the role of these compounds in longevity of the host ([56](#B56), [57](#B57)) or physiology ([58](#B58)), as nutritional compounds ([24](#B24), [59](#B59)), or compounds that are involved in host-microbial interactions ([58](#B58), [60](#B60)), we selected four candidates for further experimental testing as precision prebiotics to specifically increase the abundance of MYb11.

### *In vitro* validation of predicted precision prebiotics

As a proof-of-concept, we experimentally tested the ability of four of the predicted precision prebiotics compounds, L-serine, L-threonine, GABA, and D-mannitol, to specifically increase the growth of MYb11 in co-culture with MYb71. L-serine is a dietary amino acid that has been involved in nutrition-drug-host-microbial interactions ([60](#B60)) and can support the lifespan of the host ([57](#B57)), L-threonine, an essential amino acid ([59](#B59)), through its catabolite methylglyoxal (in low amounts) can also have a positive effect on host’s lifespan ([56](#B56)). GABA, apart from being a bacterial product, is protective for the host’s neurons ([58](#B58)). Finally, D-mannitol is a commonly used sugar alcohol ([24](#B24)), therefore its usage as a nutritional compound would make its usage as a supplement a safe option for the host.

We supplemented MYb11 and MYb71 in mono- as well as co-culture and measured bacterial growth *in vitro* in liquid NGM after 5 h (Fig. S1) and 24 h. Independently of the added supplement or the mono- or co-culturing, MYb11 and MYb71 showed increased growth in mono- and co-cultures when supplemented with 10 mM L-serine, L-threonine, GABA, as well as D-mannitol after 24 h ([Fig. 3A](#F3); Tables S6 to S9). Additionally, to quantify bacterial growth, we used fluorescently labeled strains. CFU of mono- and co-culture were plated and counted after 24 h growth in liquid culture in the presence (10 mM) or absence (0 mM) of the four compounds L-serine ([Fig. 3B](#F3); Table S10), L-threonine ([Fig. 3C](#F3); Table S12), GABA ([Fig. 3D](#F3); Table S13), and D-mannitol ([Fig. 3E](#F3); Table S14). MYb11 and MYb71 showed increased colony numbers in mono-culture when supplemented with L-serine and L-threonine compared to no supplementation ([Fig. 3B and C](#F3), Wilcoxon signed-rank test*, P* = 0.001, *P* = 0.006, *P* = 0.007, *P* = 0.032, respectively). GABA supplementation led to an increase in the growth of MYb11 in mono-culture, but not MYb71 ([Fig. 3D](#F3), Wilcoxon signed-rank test, *P* < 0.001, *P* = 0.922, respectively), while D-mannitol had no effect on the growth of MYb11 nor MYb71 in mono-culture ([Fig. 3E](#F3), Wilcoxon signed-rank test*, P* = 0.259, *P* = 0.661, respectively). Thus, while serine as well as threonine boosted growth of MYb11 in co-culture versus unsupplemented single growth, and mannitol as well as GABA did not change growth in co-culture versus unsupplemented single growth, all four compounds specifically promoted MYb11 growth in co-culture over MYb71 *in vitro* and thus confirmed the computational predictions ([Fig. 3B through E](#F3), Wilcoxon signed-rank test*, P* = 6.87E−06, *P* < 0.001, *P* = 7.15E−06, *P* = 7.07E−06, respectively) (see also: Table S11)

#### Fig 3.

[![Fig 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4779/10846184/033f36a0aa5c/spectrum.01144-23.f003.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10846184_spectrum.01144-23.f003.jpg)

[Open in a new tab](figure/F3/)

*In vitro* growth of *Pseudomonas lurida* MYb11 and *Ochrobactrum vermis* MYb71 in mono- and co-culture with four different supplements for 24 h. (**A**) Growth curves of co-cultures (gray) or mono-cultures of MYb11 (pink) and MYb71 (green) in liquid NGM for 24 h either with 10 mM of L-serine, L-threonine, GABA, or D-mannitol (solid line) or without supplementation (dashed line). (**B, C, D, E**) Colony-forming units (CFUs/µL) in mono-cultures of MYb11 (left) and MYb71 (right) or in co-culture (middle) after 24 h of 10 mM of the respective supplement. Shown are boxplots with the median as a thick horizontal line, the interquartile range as box, the outer quartiles as whiskers, and each replicate depicted by a dot. The CFU count data of every replicate was normalized by subtracting the median of the CFU count data from the corresponding non-supplemented control treatments (control and experimental data provided in the Tables S10, S12 to S14). Black asterisks indicate statistical comparisons between supplemented and non-supplemented median, gray asterisks indicate statistical comparisons between supplemented medians of MYb11 and MYb71. *n* = 5–14.

### Concentration-dependent effect of L-serine supplementation *in vitro*

The statistical analyses suggested that L-serine is the most promising precision prebiotic for further analysis among the compounds validated *in vitro*. We repeated the *in vitro* supplementation experiment across a concentration gradient to determine the optimal dose for enrichment of MYb11. Overall, the MYb11 growth as well as MYb71 growth in monoculture at 10 and 100 mM over time were increased following the increase in L-serine concentrations ([Fig. 4A](#F4); Table S15). When quantifying the growth with supplementation at the same concentration gradient ([Fig. 4B](#F4); Table S16), then starting with 1 mM did not significantly impact the abundance of MYb11 proportions (GLM, *P* = 0.778), nor MYb71 (GLM, *P* = 0.69). However, supplementation with 10 mM led to an enrichment of MYb11 (GLM, *P* < 0.001), thereby indicating a significant difference in proportions in co-culture between MYb11 and MYb71 (GLM*, P* = 5.77E−07). Essentially identical results were obtained at 100 mM, including an MYb11 enrichment (GLM, *P* = 0.004), and a significant difference between MYb11 and MYb71 proportions (GLM, *P* = 7.93E−05). (See also: Table S17)

#### Fig 4.

[![Fig 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4779/10846184/f03e926a7bdd/spectrum.01144-23.f004.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10846184_spectrum.01144-23.f004.jpg)

[Open in a new tab](figure/F4/)

Concentration-dependent effect of L-serine supplementation on bacterial growth *in vitro.* (**A**) Bacterial growth was measured based on optical density (OD600) of each species in mono-culture in liquid nematode growth medium (NGM) with the respective L-serine supplementation of 0, 1, 10, or 100 mM. Points represent individual replicates, while lines indicate the mean, *n* = 12. (**B**) Relative abundance of MYb11 (pink) and MYb71 (green) in co-cultures grown for 24 h with the respective supplement concentration in liquid NGM, based on colony counts of fluorescently labeled bacteria. Shown are boxplots with the median as a thick horizontal line, the interquartile range as box, the outer quartiles as whiskers, and each replicate depicted by a dot. Bacterial abundances of every replicate were normalized by subtracting the median of the corresponding non-supplemented control (0 mM) for each bacterial strain; the respective median values are indicated by the dashed lines. Statistical differences were determined by GLM and are indicated by asterisks (\*\*\* *P* < 0.001). Black asterisks indicate statistical comparisons between supplemented and non-supplemented median, gray asterisks indicate statistical comparisons between supplemented medians of MYb11 and MYb71. *n* = 9–12.

### Serine supplementation alters bacterial colonization of *C. elegans*

Finally, we tested whether L-serine supplementation to solid growth medium would alter bacterial proportions in the host *C. elegans*. Synchronized populations of L1 larvae were pipetted onto bacterial lawns of MYb11 and MYb71 on solid NGM plates supplemented with either 0, 10, or 100 mM L-serine. After 72 h, adult worms were collected to quantify bacterial proportions. Supplementation at 10 mM L-serine did not change the proportion of MYb11 in co-culture in worms compared to non-supplemented media (GLM, *P* = 0.459, [Fig. 5A](#F5) left panel). However, L-serine supplementation at 100 mM increased the proportion of MYb11 in the bacterial community in worms compared to non-supplemented media (GLM, *P* < 0.001, [Fig. 5A](#F5), right panel). We next asked whether the effect of supplementation on bacterial proportions in the host simply reflected changes in the proportion of bacteria that were available to them on the lawn (i.e., their diet). The relative abundance of MYb11 on lawns did not change by 10 mM supplementation (GLM, *P* = 0.61, [Fig. 5A](#F5), left panel) compared to non-supplementation. However, the relative abundance of MYb11 was approximately 60% in non-supplemented media and increased to approximately 75% with 100 mM L-serine supplementation (GLM, *P* < 0.002, [Fig. 5B](#F5), right panel). The observed proportions of MYb11 in worms at 100 mM serine were significantly higher than expected based on proportions in the lawn and thus colonization probability compared to MYb71 (chi-squared test, *Χ*2 = 10.244, *df* = 1, *P* = 0.0013; see Materials and Methods). This suggests that the provision of serine enriches MYb11 in the host independently from the enrichment observed on the lawn and that serine allows MYb11 to overcome its growth disadvantages in competition with MYb71 in the host (see also: Table S18 and S19).

#### Fig 5.

[![Fig 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4779/10846184/e6216f749969/spectrum.01144-23.f005.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10846184_spectrum.01144-23.f005.jpg)

[Open in a new tab](figure/F5/)

L-serine supplementation alters bacterial abundance in *C. elegans.* (**A**) Adult worms were collected from solid NGM plates, which had been inoculated with co-culture and supplemented with 10 mM L-serine (left panel) or 100 mM (right panel) L-serine. Worms were washed and surface-sterilized to remove residual bacteria, then homogenized in a buffer. Bacterial proportions were quantified based on colony counts. (**B**) Bacterial lawns were sampled by cutting an agar disc of standardized size from NGM plates supplemented with 10 mM (left panel) or 100 mM L-serine (right panel). Discs were homogenized in buffer, and CFUs were quantified. (**A and B**) Shown are boxplots with the median as a thick horizontal line, the interquartile range as box, the outer quartiles as whiskers, and each biological replicate depicted by a symbol. For each replicate and each bacterium, bacterial abundance from experimental treatments was normalized by subtracting the median of the corresponding non-supplemented controls (0 mM), as indicated by the dashed lines. Statistical differences were determined by GLM and are indicated by asterisks (\*\*\* *P* < 0.001, \*\**P* < 0.01) and show comparisons between supplemented and non-supplemented median.

## DISCUSSION

Although many factors are known to influence microbiome composition [e.g., dietary patterns, lifestyle, or medication ([61](#B61))], the mechanisms causing these changes are often not fully understood and often a large number of bacterial species are affected. We propose that the most effective way to modulate the composition of the microbiome is by modulating or enriching the abundance of an already existing bacterial species with health-beneficial effects using precision prebiotics. A direct approach to identify such precision prebiotics would be to provide compounds that are uniquely taken up by the bacterial target species. However, through an analysis of *C. elegans* and human microbiome data, we were able to show that such unique uptake compounds typically only exist for few species in a microbial community and especially for larger communities there are no such compounds at all. Thus, in a typical microbiome, precision prebiotics have to be identified from compounds that are potentially taken up by several species. To identify such compounds, we used genome-scale metabolic networks of two bacterial members from the *C. elegans* microbiome, *O. vermis* MYb71, and the host-protective *P. lurida* MYb11, in conjunction with constraint-based modeling to identify compounds that specifically increased the growth of MYb11. Using HMDB annotation ([46](#B46)), we found that fatty acids were enriched as predicted growth-supporting compounds for MYb11 which is in agreement with previous observations of the preference of bacteria of the genus *Pseudomonas* for these compounds ([62](#B62)). Considering compounds that were identified across all three simulation approaches and cross-referencing with *Biolog* data revealed that 4-hydroxybenzoate, D-mannitol, γ-aminobutyric acid (GABA), glycerol-3-phosphate, L-serine, trehalose, L-threonine, and putrescine are potential precision prebiotics for MYb11. We experimentally tested four of these predicted compounds, L-serine, L-threonine, D-mannitol, and GABA *in vitro* and confirmed their positive effect specifically on the growth of MYb11. Moving into the *C. elegans* host system, we were able to show that L-serine is also able to selectively boost the abundance of MYb11 *in vivo*, when the host was colonized with both bacterial species. Interestingly, while 10 mM of L-serine was sufficient for an enrichment of MYb11 in batch culture, 100 mM was necessary in the growth experiments with the host. This suggests differences in either bacterial uptake kinetics for L-serine or bacterial growth in general between batch growth and growth on agar plates. Moreover, while MYb11 was strongly enriched in worms to similar levels like MYb71 even though it had a very low abundance in the lawn, this enrichment was even stronger when supplying L-serine. Thus, our proof-of-principle work demonstrates that constraint-based microbial community modeling approaches are an effective tool for the future development of targeted microbiome therapies using precision prebiotics.

The effect of serine and threonine on the host and/or on its microbiome has been also documented in the literature. For instance, threonine is an essential amino acid for the host ([59](#B59)). Longevity effects on the host have been shown also after supplementation with threonine ([63](#B63)) and serine ([57](#B57)), while dietary serine reduced host fitness through acting on bacterial metabolism, when it was supplemented along with the chemotherapeutic 5-fluoro 2′deoxyuridine ([60](#B60)). Moreover, serine promotes the growth of *E. coli* LF82 in the mouse gut ([64](#B64)), which shows the potential of this compound to alter the abundance of multiple species. In accordance, we also observed the effects of serine on MYb71 growth, which was, however, stronger for MYb11 if they were grown in community.

One important challenge to specifically target the microbiome with nutritional supplements is to prevent host uptake or metabolization of the supplements ([65](#B65)). Different approaches to achieve restricted bioavailability, such as the supplementation of non-digestible carbohydrates ([66](#B66)) or coating of supplements with compounds that dissolve during passage through the digestive tract have been developed ([67](#B67)[–](#B68)[69](#B69)). Moreover, as indicated by our analysis of concentration-dependent effects of supplementation in the case of serine, it might be necessary to control the specific concentrations of a compound that are being released and their stability over time in the gut. From the bacterial side, *Biolog* assays have shown that MYb11 has the capacity to catabolize serine ([Fig. 2D](#F2)). However, the specific concentrations used on the *Biolog* plates and the mechanisms of transport that are possibly dependent on those concentrations are not known. From the host side, to maintain relatively constant concentrations in the host environment or to deviate from its normal concentrations, advanced formulations such as nanotechnology ([70](#B70)), delayed release, and other drug delivery technologies ([71](#B71), [72](#B72)) could be employed.

Although our combined computational-experimental approach can serve as a blueprint for future studies aimed at the identification of precision prebiotics, several factors should still be taken into consideration. As we showed, it is nearly impossible to identify a compound that is uniquely taken up by only one species within a community. This point becomes apparent, if we consider the numerous different species that reside in various hosts. For instance, the synthetic bacterial community of *C. elegans*, *CeMbio* ([24](#B24)) has 12 members, while the human-related bacterial community AGORA has more than 800 species ([34](#B34)). Thus, an important next step is to scale up our approach to more complex communities. Also along these lines, we did use automatically reconstructed bacterial metabolic networks from our gapseq approach rather than manually curated networks which resembles the typical situation in the human gut microbiota where a detailed manual reconstruction of the hundreds to thousands of already detected and sequenced bacterial species is practically impossible. Furthermore, since the physiology of worms cannot be directly compared to the physiology of humans, the efficacy and effectiveness of the approach [see more on the topic ([73](#B73))] need to be studied in more detail. At this point, the complexity of the regulatory affairs in various jurisdictions (e.g., the definition of nutritional supplementation vs medication, safety concerns) should also not be underestimated ([74](#B74)). Additionally, while the modeling approach we used has quite a number of underlying assumptions, it allowed us to considerably narrow down the list of potential candidate metabolites for experimental testing. Finally, we did not consider potential side effects of the supplemented metabolites on the worm host and did not optimize the mode of delivery, for instance, to release the supplement specifically in the worm gut, which are important considerations for future studies. Taking everything into consideration, we showed that the *in silico* design of targeted supplementation strategies to modulate the microbiome is not only feasible but also promising for the development of future microbiome-based treatment.

## Supplementary Material

Reviewer comments

[reviewer-comments.pdf](/articles/instance/10846184/bin/reviewer-comments.pdf) (123.6KB, pdf)

## ACKNOWLEDGMENTS

The authors acknowledge support by the German Research Foundation within the collaborative research center “Origin and Function of Metaorganisms,” CRC1182, sub-project A1 to C.K., K.D., and H.S.; the research unit miTarget (FOR5042) to C.K. and the excellence cluster “Precision medicine in chronic inflammation” (EXC2167) to C.K. and H.S. C.K. moreover acknowledges support by the German Ministry for Education and Research (E:Med iTREAT, support code 01ZX1902A). G.M. received support through a Young Investigator Award by the CRC1182. R.D. received support through a Short-Term Research Grant from the Deutscher Akademischer Austauschdienst (German Academic Exchange Service). The authors also acknowledge funding via NIH grant DP2DK116645, NASA grant 80NSSC22K0250, and JGI/DOE grant CSP-503338 (all to B.S.S.).

This research was supported in part through high-performance computing resources available at the Kiel University computing center. *C. elegans* N2 was originally provided by the Caenorhabditis Genetics Center (CGC; <https://cgc.umn.edu/>), which is funded by the NIH Office of Research Infrastructure Programs (P40 OD010440).

## Contributor Information

Hinrich Schulenburg, Email: hschulenburg@zoologie.uni-kiel.de.

Christoph Kaleta, Email: c.kaleta@iem.uni-kiel.de.

Paul A. Jensen, University of Michigan-Ann Arbor, Ann Arbor, Michigan, USA

Hyun-Seob Song, University of Nebraska-Lincoln, Lincoln, Nebraska, USA.

## DATA AVAILABILITY

The R scripts and the raw data to conduct the simulations are deposited in [Github](https://github.com/maringos/Nutritional-Supplements-Prediction) (version/commit: [e15eb1e](https://github.com/maringos/Nutritional-Supplements-Prediction/commit/e15eb1e5175fe042dff1e891c90286649ae676c0)). Metabolic models used in the prediction of unique uptake compounds have been deposited in [FigShare](https://doi.org/10.6084/m9.figshare.24968871.v1). The growth rate changes of the supplementation experiments with FBA sybil and MicrobiomeGS2 can be found in Supplementary Tables S3 and S4. The bacterial biomass data of the BacArena simulations can be found in Supplementary Table S5.

## ETHICS APPROVAL

The human cohort data used in this study originate from the publication of Pryor et al. ([30](#B30)), whose access was provided by the *PopGen* biobank (Schleswig-Holstein, Germany) ([75](#B75)). The raw data are available from the biobank through a structured application procedure ([https://www.uksh.de/p2n/Information+for+Researchers.html](https://www.uksh.de/p2n/Information%2Bfor%2BResearchers.html)).

## SUPPLEMENTAL MATERIAL

The following material is available online at <https://doi.org/10.1128/spectrum.01144-23>.

Supplementary figures. spectrum.01144-23-s0001.docx.

Figures S1 and S2.

[Click here for additional data file.](/articles/instance/10846184/bin/spectrum.01144-23-s0001.docx) (415.2KB, docx)

DOI: 10.1128/spectrum.01144-23.SuF1

Supplementary tables. spectrum.01144-23-s0002.xlsx.

Tables S1-S20.

[Click here for additional data file.](/articles/instance/10846184/bin/spectrum.01144-23-s0002.xlsx) (3.3MB, xlsx)

DOI: 10.1128/spectrum.01144-23.SuF2

OPEN PEER REVIEW. reviewer-comments.pdf.

An accounting of the reviewer comments and feedback.

[Click here for additional data file.](/articles/instance/10846184/bin/reviewer-comments.pdf) (123.6KB, pdf)

DOI: 10.1128/spectrum.01144-23.SuF3

ASM does not own the copyrights to Supplemental Material that may be linked to, or accessed through, an article. The authors have granted ASM a non-exclusive, world-wide license to publish the Supplemental Material files. Please contact the corresponding author directly for reuse.

## REFERENCES

* 1.
  Kamada N, Chen GY, Inohara N, Núñez G. 2013. Control of pathogens and pathobionts by the gut microbiota. Nat Immunol
  14:685–690. doi: 10.1038/ni.2608
   [[DOI](https://doi.org/10.1038/ni.2608)] [[PMC free article](/articles/PMC4083503/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23778796/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&title=Control%20of%20pathogens%20and%20pathobionts%20by%20the%20gut%20microbiota&volume=14&publication_year=2013&pages=685-690&pmid=23778796&doi=10.1038/ni.2608&)]
* 2.
  Zheng D, Liwinski T, Elinav E. 2020. Interaction between microbiota and immunity in health and disease. Cell Res
  30:492–506. doi: 10.1038/s41422-020-0332-7
   [[DOI](https://doi.org/10.1038/s41422-020-0332-7)] [[PMC free article](/articles/PMC7264227/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32433595/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=Interaction%20between%20microbiota%20and%20immunity%20in%20health%20and%20disease&volume=30&publication_year=2020&pages=492-506&pmid=32433595&doi=10.1038/s41422-020-0332-7&)]
* 3.
  Rowland I, Gibson G, Heinken A, Scott K, Swann J, Thiele I, Tuohy K. 2018. Gut microbiota functions: metabolism of nutrients and other food components. Eur J Nutr
  57:1–24. doi: 10.1007/s00394-017-1445-8 [[DOI](https://doi.org/10.1007/s00394-017-1445-8)] [[PMC free article](/articles/PMC5847071/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28393285/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Nutr&title=Gut%20microbiota%20functions:%20metabolism%20of%20nutrients%20and%20other%20food%20components&volume=57&publication_year=2018&pages=1-24&pmid=28393285&doi=10.1007/s00394-017-1445-8&)]
* 4.
  Clarke G, Sandhu KV, Griffin BT, Dinan TG, Cryan JF, Hyland NP. 2019. Gut reactions: breaking down Xenobiotic-microbiome interactions. Pharmacol Rev
  71:198–224. doi: 10.1124/pr.118.015768
   [[DOI](https://doi.org/10.1124/pr.118.015768)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30890566/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Gut%20reactions:%20breaking%20down%20Xenobiotic-microbiome%20interactions&volume=71&publication_year=2019&pages=198-224&pmid=30890566&doi=10.1124/pr.118.015768&)]
* 5.
  Weersma RK, Zhernakova A, Fu J. 2020. Interaction between drugs and the gut microbiome. Gut
  69:1510–1519. doi: 10.1136/gutjnl-2019-320204
   [[DOI](https://doi.org/10.1136/gutjnl-2019-320204)] [[PMC free article](/articles/PMC7398478/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32409589/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=Interaction%20between%20drugs%20and%20the%20gut%20microbiome&volume=69&publication_year=2020&pages=1510-1519&pmid=32409589&doi=10.1136/gutjnl-2019-320204&)]
* 6.
  Kolodny O, Schulenburg H. 2020. Microbiome-mediated plasticity directs host evolution along several distinct time scales. Philos Trans R Soc Lond B Biol Sci
  375:20190589. doi: 10.1098/rstb.2019.0589
   [[DOI](https://doi.org/10.1098/rstb.2019.0589)] [[PMC free article](/articles/PMC7435166/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32772662/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci&title=Microbiome-mediated%20plasticity%20directs%20host%20evolution%20along%20several%20distinct%20time%20scales&volume=375&publication_year=2020&pages=20190589&pmid=32772662&doi=10.1098/rstb.2019.0589&)]
* 7.
  Henry LP, Bruijning M, Forsberg SKG, Ayroles JF. 2021. The microbiome extends host evolutionary potential. Nat Commun
  12:5141. doi: 10.1038/s41467-021-25315-x
   [[DOI](https://doi.org/10.1038/s41467-021-25315-x)] [[PMC free article](/articles/PMC8390463/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34446709/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=The%20microbiome%20extends%20host%20evolutionary%20potential&volume=12&publication_year=2021&pages=5141&pmid=34446709&doi=10.1038/s41467-021-25315-x&)]
* 8.
  Liu H, Brettell LE, Qiu Z, Singh BK. 2020. Microbiome-mediated stress resistance in plants. Trends Plant Sci
  25:733–743. doi: 10.1016/j.tplants.2020.03.014
   [[DOI](https://doi.org/10.1016/j.tplants.2020.03.014)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32345569/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci&title=Microbiome-mediated%20stress%20resistance%20in%20plants&volume=25&publication_year=2020&pages=733-743&pmid=32345569&doi=10.1016/j.tplants.2020.03.014&)]
* 9.
  Hou K, Wu Z-X, Chen X-Y, Wang J-Q, Zhang D, Xiao C, Zhu D, Koya JB, Wei L, Li J, Chen Z-S. 2022. Microbiota in health and diseases. Signal Transduct Target Ther
  7:135. doi: 10.1038/s41392-022-00974-4
   [[DOI](https://doi.org/10.1038/s41392-022-00974-4)] [[PMC free article](/articles/PMC9034083/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35461318/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Signal%20Transduct%20Target%20Ther&title=Microbiota%20in%20health%20and%20diseases&volume=7&publication_year=2022&pages=135&pmid=35461318&doi=10.1038/s41392-022-00974-4&)]
* 10.
  Durack J, Lynch SV. 2019. The gut microbiome: relationships with disease and opportunities for therapy. J Exp Med
  216:20–40. doi: 10.1084/jem.20180448
   [[DOI](https://doi.org/10.1084/jem.20180448)] [[PMC free article](/articles/PMC6314516/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30322864/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&title=The%20gut%20microbiome:%20relationships%20with%20disease%20and%20opportunities%20for%20therapy&volume=216&publication_year=2019&pages=20-40&pmid=30322864&doi=10.1084/jem.20180448&)]
* 11.
  Karl JP, Hatch AM, Arcidiacono SM, Pearce SC, Pantoja-Feliciano IG, Doherty LA, Soares JW. 2018. Effects of psychological effects of psychological, environmental and physical stressors on the gut microbiota. Front Microbiol
  9:2013. doi: 10.3389/fmicb.2018.02013
   [[DOI](https://doi.org/10.3389/fmicb.2018.02013)] [[PMC free article](/articles/PMC6143810/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30258412/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&title=Effects%20of%20psychological%20effects%20of%20psychological,%20environmental%20and%20physical%20stressors%20on%20the%20gut%20microbiota&volume=9&publication_year=2018&pages=2013&pmid=30258412&doi=10.3389/fmicb.2018.02013&)]
* 12.
  Brugman S, Ikeda-Ohtsubo W, Braber S, Folkerts G, Pieterse CMJ, Bakker PAHM. 2018. A comparative review on microbiota manipulation: lessons from fish, plants, livestock, and human research. Front Nutr
  5:80. doi: 10.3389/fnut.2018.00080
   [[DOI](https://doi.org/10.3389/fnut.2018.00080)] [[PMC free article](/articles/PMC6134018/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30234124/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Nutr&title=A%20comparative%20review%20on%20microbiota%20manipulation:%20lessons%20from%20fish,%20plants,%20livestock,%20and%20human%20research&volume=5&publication_year=2018&pages=80&pmid=30234124&doi=10.3389/fnut.2018.00080&)]
* 13.
  Moles L, Otaegui D. 2020. The impact of diet on microbiota evolution and human health. Is diet an adequate tool for microbiota modulation?
  Nutrients
  12:1654. doi: 10.3390/nu12061654
   [[DOI](https://doi.org/10.3390/nu12061654)] [[PMC free article](/articles/PMC7352211/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32498430/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nutrients&title=The%20impact%20of%20diet%20on%20microbiota%20evolution%20and%20human%20health.%20Is%20diet%20an%20adequate%20tool%20for%20microbiota%20modulation?&volume=12&publication_year=2020&pages=1654&pmid=32498430&doi=10.3390/nu12061654&)]
* 14.
  Sorbara MT, Pamer EG. 2022. Microbiome-based therapeutics. Nat Rev Microbiol
  20:365–380. doi: 10.1038/s41579-021-00667-9
   [[DOI](https://doi.org/10.1038/s41579-021-00667-9)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34992261/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol&title=Microbiome-based%20therapeutics&volume=20&publication_year=2022&pages=365-380&pmid=34992261&doi=10.1038/s41579-021-00667-9&)]
* 15.
  Merrick B, Allen L, Masirah M Zain N, Forbes B, Shawcross DL, Goldenberg SD. 2020. Regulation, risk and safety of faecal microbiota transplant. Infect Prev Pract
  2:100069. doi: 10.1016/j.infpip.2020.100069
   [[DOI](https://doi.org/10.1016/j.infpip.2020.100069)] [[PMC free article](/articles/PMC7280140/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34316559/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect%20Prev%20Pract&title=Regulation,%20risk%20and%20safety%20of%20faecal%20microbiota%20transplant&volume=2&publication_year=2020&pages=100069&pmid=34316559&doi=10.1016/j.infpip.2020.100069&)]
* 16.
  Park S-Y, Seo GS. 2021. Fecal microbiota transplantation: is it safe?
  Clin Endosc
  54:157–160. doi: 10.5946/ce.2021.072
   [[DOI](https://doi.org/10.5946/ce.2021.072)] [[PMC free article](/articles/PMC8039753/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33827154/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Endosc&title=Fecal%20microbiota%20transplantation:%20is%20it%20safe?&volume=54&publication_year=2021&pages=157-160&pmid=33827154&doi=10.5946/ce.2021.072&)]
* 17.
  Guarino MPL, Altomare A, Emerenziani S, Di Rosa C, Ribolsi M, Balestrieri P, Iovino P, Rocchi G, Cicala M. 2020. Mechanisms of action of prebiotics and their effects on gastro-intestinal disorders in adults. Nutrients
  12:1037. doi: 10.3390/nu12041037
   [[DOI](https://doi.org/10.3390/nu12041037)] [[PMC free article](/articles/PMC7231265/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32283802/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nutrients&title=Mechanisms%20of%20action%20of%20prebiotics%20and%20their%20effects%20on%20gastro-intestinal%20disorders%20in%20adults&volume=12&publication_year=2020&pages=1037&pmid=32283802&doi=10.3390/nu12041037&)]
* 18.
  Davani-Davari D, Negahdaripour M, Karimzadeh I, Seifan M, Mohkam M, Masoumi SJ, Berenjian A, Ghasemi Y. 2019. Prebiotics: definition, types, sources, mechanisms, and clinical applications. Foods
  8:92. doi: 10.3390/foods8030092
   [[DOI](https://doi.org/10.3390/foods8030092)] [[PMC free article](/articles/PMC6463098/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30857316/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Foods&title=Prebiotics:%20definition,%20types,%20sources,%20mechanisms,%20and%20clinical%20applications&volume=8&publication_year=2019&pages=92&pmid=30857316&doi=10.3390/foods8030092&)]
* 19.
  Maldonado-Gómez MX, Martínez I, Bottacini F, O’Callaghan A, Ventura M, van Sinderen D, Hillmann B, Vangay P, Knights D, Hutkins RW, Walter J. 2016. Stable engraftment of Bifidobacterium longum AH1206 in the human gut depends on individualized features of the resident microbiome. Cell Host Microbe
  20:515–526. doi: 10.1016/j.chom.2016.09.001
   [[DOI](https://doi.org/10.1016/j.chom.2016.09.001)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27693307/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20Microbe&title=Stable%20engraftment%20of%20Bifidobacterium%20longum%20AH1206%20in%20the%20human%20gut%20depends%20on%20individualized%20features%20of%20the%20resident%20microbiome&volume=20&publication_year=2016&pages=515-526&pmid=27693307&doi=10.1016/j.chom.2016.09.001&)]
* 20.
  Anokyewaa MA, Amoah K, Li Y, Lu Y, Kuebutornye FKA, Asiedu B, Seidu I. 2021. Prevalence of virulence genes and antibiotic susceptibility of Bacillus used in commercial aquaculture probiotics in China. Aquac Rep
  21:100784. doi: 10.1016/j.aqrep.2021.100784 [[DOI](https://doi.org/10.1016/j.aqrep.2021.100784)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aquac%20Rep&title=Prevalence%20of%20virulence%20genes%20and%20antibiotic%20susceptibility%20of%20Bacillus%20used%20in%20commercial%20aquaculture%20probiotics%20in%20China&volume=21&publication_year=2021&pages=100784&doi=10.1016/j.aqrep.2021.100784&)]
* 21.
  Kissoyan KAB, Peters L, Giez C, Michels J, Pees B, Hamerich IK, Schulenburg H, Dierking K. 2022. Exploring effects of protective natural microbiota on host physiology. Front Cell Infect Microbiol
  12:775728. doi: 10.3389/fcimb.2022.775728
   [[DOI](https://doi.org/10.3389/fcimb.2022.775728)] [[PMC free article](/articles/PMC8884406/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35237530/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Infect%20Microbiol&title=Exploring%20effects%20of%20protective%20natural%20microbiota%20on%20host%20physiology&volume=12&publication_year=2022&pages=775728&pmid=35237530&doi=10.3389/fcimb.2022.775728&)]
* 22.
  Dirksen P, Marsh SA, Braker I, Heitland N, Wagner S, Nakad R, Mader S, Petersen C, Kowallik V, Rosenstiel P, Félix M-A, Schulenburg H. 2016. The native microbiome of the nematode Caenorhabditis elegans: gateway to a new host-microbiome model. BMC Biol
  14:38. doi: 10.1186/s12915-016-0258-1
   [[DOI](https://doi.org/10.1186/s12915-016-0258-1)] [[PMC free article](/articles/PMC4860760/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27160191/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Biol&title=The%20native%20microbiome%20of%20the%20nematode%20Caenorhabditis%20elegans:%20gateway%20to%20a%20new%20host-microbiome%20model&volume=14&publication_year=2016&pages=38&pmid=27160191&doi=10.1186/s12915-016-0258-1&)]
* 23.
  Zhang R, Hou A. 2013. Host-microbe interactions in Caenorhabditis elegans. ISRN Microbiol
  2013:356451. doi: 10.1155/2013/356451
   [[DOI](https://doi.org/10.1155/2013/356451)] [[PMC free article](/articles/PMC3747393/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23984180/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ISRN%20Microbiol&title=Host-microbe%20interactions%20in%20Caenorhabditis%20elegans&volume=2013&publication_year=2013&pages=356451&pmid=23984180&doi=10.1155/2013/356451&)]
* 24.
  Dirksen P, Assié A, Zimmermann J, Zhang F, Tietje A-M, Marsh SA, Félix M-A, Shapira M, Kaleta C, Schulenburg H, Samuel BS. 2020. CeMbio - the Caenorhabditis elegans microbiome resource. G3 (Bethesda):3025–3039. doi: 10.1534/g3.120.401309
   [[DOI](https://doi.org/10.1534/g3.120.401309)] [[PMC free article](/articles/PMC7466993/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32669368/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=G3%20(Bethesda)&title=CeMbio%20-%20the%20Caenorhabditis%20elegans%20microbiome%20resource&publication_year=2020&pages=3025-3039&pmid=32669368&doi=10.1534/g3.120.401309&)]
* 25.
  Kissoyan KAB, Drechsler M, Stange E-L, Zimmermann J, Kaleta C, Bode HB, Dierking K. 2019. Natural C. elegans microbiota protects against infection via production of a cyclic lipopeptide of the viscosin group. Curr Biol
  29:1030–1037. doi: 10.1016/j.cub.2019.01.050
   [[DOI](https://doi.org/10.1016/j.cub.2019.01.050)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30827913/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Natural%20C.%20elegans%20microbiota%20protects%20against%20infection%20via%20production%20of%20a%20cyclic%20lipopeptide%20of%20the%20viscosin%20group&volume=29&publication_year=2019&pages=1030-1037&pmid=30827913&doi=10.1016/j.cub.2019.01.050&)]
* 26.
  Zhang F, Weckhorst JL, Assié A, Hosea C, Ayoub CA, Khodakova AS, Cabrera ML, Vidal Vilchis D, Félix M-A, Samuel BS. 2021. Natural genetic variation drives microbiome selection in the Caenorhabditis elegans gut. Curr Biol
  31:2603–2618. doi: 10.1016/j.cub.2021.04.046
   [[DOI](https://doi.org/10.1016/j.cub.2021.04.046)] [[PMC free article](/articles/PMC8222194/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34048707/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Natural%20genetic%20variation%20drives%20microbiome%20selection%20in%20the%20Caenorhabditis%20elegans%20gut&volume=31&publication_year=2021&pages=2603-2618&pmid=34048707&doi=10.1016/j.cub.2021.04.046&)]
* 27.
  Zimmermann J, Obeng N, Yang W, Pees B, Petersen C, Waschina S, Kissoyan KA, Aidley J, Hoeppner MP, Bunk B, Spröer C, Leippe M, Dierking K, Kaleta C, Schulenburg H. 2020. The functional repertoire contained within the native microbiota of the model nematode Caenorhabditis elegans. ISME J
  14:26–38. doi: 10.1038/s41396-019-0504-y
   [[DOI](https://doi.org/10.1038/s41396-019-0504-y)] [[PMC free article](/articles/PMC6908608/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31484996/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ISME%20J&title=The%20functional%20repertoire%20contained%20within%20the%20native%20microbiota%20of%20the%20model%20nematode%20Caenorhabditis%20elegans&volume=14&publication_year=2020&pages=26-38&pmid=31484996&doi=10.1038/s41396-019-0504-y&)]
* 28.
  Bordbar A, Monk JM, King ZA, Palsson BO. 2014. Constraint-based models predict metabolic and associated cellular functions. Nat Rev Genet
  15:107–120. doi: 10.1038/nrg3643
   [[DOI](https://doi.org/10.1038/nrg3643)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24430943/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Constraint-based%20models%20predict%20metabolic%20and%20associated%20cellular%20functions&volume=15&publication_year=2014&pages=107-120&pmid=24430943&doi=10.1038/nrg3643&)]
* 29.
  Gottstein W, Olivier BG, Bruggeman FJ, Teusink B. 2016. Constraint-based stoichiometric modelling from single organisms to microbial communities. J R Soc Interface
  13:20160627. doi: 10.1098/rsif.2016.0627
   [[DOI](https://doi.org/10.1098/rsif.2016.0627)] [[PMC free article](/articles/PMC5134014/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28334697/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20R%20Soc%20Interface&title=Constraint-based%20stoichiometric%20modelling%20from%20single%20organisms%20to%20microbial%20communities&volume=13&publication_year=2016&pages=20160627&pmid=28334697&doi=10.1098/rsif.2016.0627&)]
* 30.
  Pryor R, Norvaisas P, Marinos G, Best L, Thingholm LB, Quintaneiro LM, De Haes W, Esser D, Waschina S, Lujan C, Smith RL, Scott TA, Martinez-Martinez D, Woodward O, Bryson K, Laudes M, Lieb W, Houtkooper RH, Franke A, Temmerman L, Bjedov I, Cochemé HM, Kaleta C, Cabreiro F. 2019. Host-microbe-drug-nutrient screen identifies bacterial effectors of metformin therapy. Cell
  178:1299–1312. doi: 10.1016/j.cell.2019.08.003
   [[DOI](https://doi.org/10.1016/j.cell.2019.08.003)] [[PMC free article](/articles/PMC6736778/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31474368/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Host-microbe-drug-nutrient%20screen%20identifies%20bacterial%20effectors%20of%20metformin%20therapy&volume=178&publication_year=2019&pages=1299-1312&pmid=31474368&doi=10.1016/j.cell.2019.08.003&)]
* 31.
  Ng RH, Lee JW, Baloni P, Diener C, Heath JR, Su Y. 2022. Constraint-based reconstruction and analyses of metabolic models: open-source python tools and applications to cancer. Front Oncol
  12:914594. doi: 10.3389/fonc.2022.914594
   [[DOI](https://doi.org/10.3389/fonc.2022.914594)] [[PMC free article](/articles/PMC9303011/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35875150/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&title=Constraint-based%20reconstruction%20and%20analyses%20of%20metabolic%20models:%20open-source%20python%20tools%20and%20applications%20to%20cancer&volume=12&publication_year=2022&pages=914594&pmid=35875150&doi=10.3389/fonc.2022.914594&)]
* 32.
  Zimmermann J, Kaleta C, Waschina S. 2021. gapseq: informed prediction of bacterial metabolic pathways and reconstruction of accurate metabolic models. Genome Biol
  22:81. doi: 10.1186/s13059-021-02295-1
   [[DOI](https://doi.org/10.1186/s13059-021-02295-1)] [[PMC free article](/articles/PMC7949252/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33691770/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=gapseq:%20informed%20prediction%20of%20bacterial%20metabolic%20pathways%20and%20reconstruction%20of%20accurate%20metabolic%20models&volume=22&publication_year=2021&pages=81&pmid=33691770&doi=10.1186/s13059-021-02295-1&)]
* 33.
  Machado D, Andrejev S, Tramontano M, Patil KR. 2018. Fast automated reconstruction of genome-scale metabolic models for microbial species and communities. Nucleic Acids Res
  46:7542–7553. doi: 10.1093/nar/gky537
   [[DOI](https://doi.org/10.1093/nar/gky537)] [[PMC free article](/articles/PMC6125623/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30192979/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Fast%20automated%20reconstruction%20of%20genome-scale%20metabolic%20models%20for%20microbial%20species%20and%20communities&volume=46&publication_year=2018&pages=7542-7553&pmid=30192979&doi=10.1093/nar/gky537&)]
* 34.
  Magnúsdóttir S, Heinken A, Kutt L, Ravcheev DA, Bauer E, Noronha A, Greenhalgh K, Jäger C, Baginska J, Wilmes P, Fleming RMT, Thiele I. 2017. Generation of genome-scale metabolic reconstructions for 773 members of the human gut microbiota. Nat Biotechnol
  35:81–89. doi: 10.1038/nbt.3703
   [[DOI](https://doi.org/10.1038/nbt.3703)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27893703/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Generation%20of%20genome-scale%20metabolic%20reconstructions%20for%20773%20members%20of%20the%20human%20gut%20microbiota&volume=35&publication_year=2017&pages=81-89&pmid=27893703&doi=10.1038/nbt.3703&)]
* 35.
  Marinos G, Kaleta C, Waschina S. 2020. Defining the nutritional input for genome-scale metabolic models: a roadmap. PLoS One
  15:e0236890. doi: 10.1371/journal.pone.0236890
   [[DOI](https://doi.org/10.1371/journal.pone.0236890)] [[PMC free article](/articles/PMC7428157/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32797084/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Defining%20the%20nutritional%20input%20for%20genome-scale%20metabolic%20models:%20a%20roadmap&volume=15&publication_year=2020&pages=e0236890&pmid=32797084&doi=10.1371/journal.pone.0236890&)]
* 36.
  Bauer E, Zimmermann J, Baldini F, Thiele I, Kaleta C. 2017. BacArena: individual-based metabolic modeling of heterogeneous microbes in complex communities. PLoS Comput Biol
  13:e1005544. doi: 10.1371/journal.pcbi.1005544
   [[DOI](https://doi.org/10.1371/journal.pcbi.1005544)] [[PMC free article](/articles/PMC5460873/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28531184/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&title=BacArena:%20individual-based%20metabolic%20modeling%20of%20heterogeneous%20microbes%20in%20complex%20communities&volume=13&publication_year=2017&pages=e1005544&pmid=28531184&doi=10.1371/journal.pcbi.1005544&)]
* 37.
  Heinken A, Thiele I. 2022. Microbiome modelling toolbox 2.0: efficient, tractable modelling of microbiome communities. Bioinformatics
  38:2367–2368. doi: 10.1093/bioinformatics/btac082
   [[DOI](https://doi.org/10.1093/bioinformatics/btac082)] [[PMC free article](/articles/PMC9004645/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35157025/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Microbiome%20modelling%20toolbox%202.0:%20efficient,%20tractable%20modelling%20of%20microbiome%20communities&volume=38&publication_year=2022&pages=2367-2368&pmid=35157025&doi=10.1093/bioinformatics/btac082&)]
* 38.
  Baldini F, Hertel J, Sandt E, Thinnes CC, Neuberger-Castillo L, Pavelka L, Betsou F, Krüger R, Thiele I, NCER-PD Consortium . 2020. Parkinson’s disease-associated alterations of the gut microbiome predict disease-relevant changes in metabolic functions. BMC Biol
  18:62. doi: 10.1186/s12915-020-00775-7
   [[DOI](https://doi.org/10.1186/s12915-020-00775-7)] [[PMC free article](/articles/PMC7285525/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32517799/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Biol&title=Parkinson%E2%80%99s%20disease-associated%20alterations%20of%20the%20gut%20microbiome%20predict%20disease-relevant%20changes%20in%20metabolic%20functions&volume=18&publication_year=2020&pages=62&pmid=32517799&doi=10.1186/s12915-020-00775-7&)]
* 39.
  Aden K, Rehman A, Waschina S, Pan W-H, Walker A, Lucio M, Nunez AM, Bharti R, Zimmerman J, Bethge J, Schulte B, Schulte D, Franke A, Nikolaus S, Schroeder JO, Vandeputte D, Raes J, Szymczak S, Waetzig GH, Zeuner R, Schmitt-Kopplin P, Kaleta C, Schreiber S, Rosenstiel P. 2019. Metabolic functions of gut microbes associate with efficacy of tumor necrosis factor antagonists in patients with inflammatory bowel diseases. Gastroenterology
  157:1279–1292. doi: 10.1053/j.gastro.2019.07.025
   [[DOI](https://doi.org/10.1053/j.gastro.2019.07.025)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31326413/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Metabolic%20functions%20of%20gut%20microbes%20associate%20with%20efficacy%20of%20tumor%20necrosis%20factor%20antagonists%20in%20patients%20with%20inflammatory%20bowel%20diseases&volume=157&publication_year=2019&pages=1279-1292&pmid=31326413&doi=10.1053/j.gastro.2019.07.025&)]
* 40.
  Orth JD, Thiele I, Palsson BØ. 2010. What is flux balance analysis?
  Nat Biotechnol
  28:245–248. doi: 10.1038/nbt.1614
   [[DOI](https://doi.org/10.1038/nbt.1614)] [[PMC free article](/articles/PMC3108565/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20212490/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=What%20is%20flux%20balance%20analysis?&volume=28&publication_year=2010&pages=245-248&pmid=20212490&doi=10.1038/nbt.1614&)]
* 41.
  Gelius-Dietrich G, Desouki AA, Fritzemeier CJ, Lercher MJ. 2013. sybil – Efficient constraint-based modelling in R. BMC Syst Biol
  7:125. doi: 10.1186/1752-0509-7-125
   [[DOI](https://doi.org/10.1186/1752-0509-7-125)] [[PMC free article](/articles/PMC3843580/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24224957/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Syst%20Biol&title=sybil%20%E2%80%93%20Efficient%20constraint-based%20modelling%20in%20R&volume=7&publication_year=2013&pages=125&pmid=24224957&doi=10.1186/1752-0509-7-125&)]
* 42.
  Zimmermann J, Piecyk A, Sieber M, Petersen C, Johnke J, Moitinho-Silva L, Künzel S, Bluhm L, Traulsen A, Kaleta C, Schulenburg H. 2023. Gut-associated functions are favored during microbiome assembly across C. elegans life. bioRxiv. doi: 10.1101/2023.03.25.534195 [[DOI](https://doi.org/10.1101/2023.03.25.534195)] [[PMC free article](/articles/PMC11077962/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/38634692/)]
* 43.
  Effenberger M, Reider S, Waschina S, Bronowski C, Enrich B, Adolph TE, Koch R, Moschen AR, Rosenstiel P, Aden K, Tilg H. 2021. Microbial butyrate synthesis indicates therapeutic efficacy of azathioprine in IBD patients. J Crohns Colitis
  15:88–98. doi: 10.1093/ecco-jcc/jjaa152
   [[DOI](https://doi.org/10.1093/ecco-jcc/jjaa152)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32687146/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Crohns%20Colitis&title=Microbial%20butyrate%20synthesis%20indicates%20therapeutic%20efficacy%20of%20azathioprine%20in%20IBD%20patients&volume=15&publication_year=2021&pages=88-98&pmid=32687146&doi=10.1093/ecco-jcc/jjaa152&)]
* 44.
  Heinken A, Sahoo S, Fleming RMT, Thiele I. 2013. Systems-level characterization of a host-microbe metabolic symbiosis in the mammalian gut. Gut Microbes
  4:28–40. doi: 10.4161/gmic.22370
   [[DOI](https://doi.org/10.4161/gmic.22370)] [[PMC free article](/articles/PMC3555882/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23022739/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut%20Microbes&title=Systems-level%20characterization%20of%20a%20host-microbe%20metabolic%20symbiosis%20in%20the%20mammalian%20gut&volume=4&publication_year=2013&pages=28-40&pmid=23022739&doi=10.4161/gmic.22370&)]
* 45.
  Wilcoxon F. 1945. Individual comparisons by ranking methods. Biometrics Bulletin
  1:80. doi: 10.2307/3001968 [[DOI](https://doi.org/10.2307/3001968)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrics%20Bulletin&title=Individual%20comparisons%20by%20ranking%20methods&volume=1&publication_year=1945&pages=80&doi=10.2307/3001968&)]
* 46.
  Wishart DS, Guo A, Oler E, Wang F, Anjum A, Peters H, Dizon R, Sayeeda Z, Tian S, Lee BL, et al. 2022. HMDB 5.0: the human metabolome database for 2022. Nucleic Acids Res
  50:D622–D631. doi: 10.1093/nar/gkab1062
   [[DOI](https://doi.org/10.1093/nar/gkab1062)] [[PMC free article](/articles/PMC8728138/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34986597/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HMDB%205.0:%20the%20human%20metabolome%20database%20for%202022&volume=50&publication_year=2022&pages=D622-D631&pmid=34986597&doi=10.1093/nar/gkab1062&)]
* 47.
  Wickham H, Averick M, Bryan J, Chang W, McGowan L, François R, Grolemund G, Hayes A, Henry L, Hester J, Kuhn M, Pedersen T, Miller E, Bache S, Müller K, Ooms J, Robinson D, Seidel D, Spinu V, Takahashi K, Vaughan D, Wilke C, Woo K, Yutani H. 2019. Welcome to the tidyverse. J Open Source Softw 4:1686. doi: 10.21105/joss.01686 [[DOI](https://doi.org/10.21105/joss.01686)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Open%20Source%20Softw&title=Welcome%20to%20the%20tidyverse&volume=4&publication_year=2019&pages=1686&doi=10.21105/joss.01686&)]
* 48.
  Wickham H. 2016. ggplot2: elegant graphics for data analysis. Springer, Cham. <http://link.springer.com/10.1007/978-3-319-24277-4>. [[Google Scholar](https://scholar.google.com/scholar_lookup?title=ggplot2:%20elegant%20graphics%20for%20data%20analysis&publication_year=2016&)]
* 49.
  Wiles TJ, Wall ES, Schlomann BH, Hay EA, Parthasarathy R, Guillemin K. 2018. Modernized tools for streamlined genetic manipulation and comparative study of wild and diverse proteobacterial lineages. mBio
  9:e01877-18. doi: 10.1128/mBio.01877-18
   [[DOI](https://doi.org/10.1128/mBio.01877-18)] [[PMC free article](/articles/PMC6178617/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30301859/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=mBio&title=Modernized%20tools%20for%20streamlined%20genetic%20manipulation%20and%20comparative%20study%20of%20wild%20and%20diverse%20proteobacterial%20lineages&volume=9&publication_year=2018&pages=e01877-18&pmid=30301859&doi=10.1128/mBio.01877-18&)]
* 50.
  Stiernagle T. 2006. Maintenance of C. elegans. WormBook:1–11. doi: 10.1895/wormbook.1.101.1 [[DOI](https://doi.org/10.1895/wormbook.1.101.1)] [[PMC free article](/articles/PMC4781397/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18050451/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=WormBook&title=Maintenance%20of%20C.%20elegans&publication_year=2006&pages=1-11&pmid=18050451&doi=10.1895/wormbook.1.101.1&)]
* 51.
  Saitoh Y, Katane M, Miyamoto T, Sekine M, Sakai-Kato K, Homma H. 2020. d-serine and d-alanine regulate adaptive foraging behavior in via the NMDA receptor. J Neurosci
  40:7531–7544.
   [[DOI](https://doi.org/10.1523/JNEUROSCI.2358-19.2020)] [[PMC free article](/articles/PMC7511192/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32855271/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=d-serine%20and%20d-alanine%20regulate%20adaptive%20foraging%20behavior%20in%20via%20the%20NMDA%20receptor&volume=40&publication_year=2020&pages=7531-7544&pmid=32855271&doi=10.1523/JNEUROSCI.2358-19.2020&)]
* 52.
  Hasshoff M, Böhnisch C, Tonn D, Hasert B, Schulenburg H. 2007. The role of Caenorhabditis elegans insulin-like signaling in the behavioral avoidance of pathogenic Bacillus thuringiensis. FASEB J
  21:1801–1812. doi: 10.1096/fj.06-6551com
   [[DOI](https://doi.org/10.1096/fj.06-6551com)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17314144/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J&title=The%20role%20of%20Caenorhabditis%20elegans%20insulin-like%20signaling%20in%20the%20behavioral%20avoidance%20of%20pathogenic%20Bacillus%20thuringiensis&volume=21&publication_year=2007&pages=1801-1812&pmid=17314144&doi=10.1096/fj.06-6551com&)]
* 53.
  Nakad R, Snoek LB, Yang W, Ellendt S, Schneider F, Mohr TG, Rösingh L, Masche AC, Rosenstiel PC, Dierking K, Kammenga JE, Schulenburg H. 2016. Contrasting invertebrate immune defense behaviors caused by a single gene, the Caenorhabditis elegans neuropeptide receptor gene npr-1. BMC Genomics
  17:280. doi: 10.1186/s12864-016-2603-8
   [[DOI](https://doi.org/10.1186/s12864-016-2603-8)] [[PMC free article](/articles/PMC4827197/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27066825/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=Contrasting%20invertebrate%20immune%20defense%20behaviors%20caused%20by%20a%20single%20gene,%20the%20Caenorhabditis%20elegans%20neuropeptide%20receptor%20gene%20npr-1&volume=17&publication_year=2016&pages=280&pmid=27066825&doi=10.1186/s12864-016-2603-8&)]
* 54.
  Schulenburg H, Müller S. 2004. Natural variation in the response of Caenorhabditis elegans towards Bacillus thuringiensis. Parasitology
  128:433–443. doi: 10.1017/s003118200300461x
   [[DOI](https://doi.org/10.1017/s003118200300461x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15151149/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Parasitology&title=Natural%20variation%20in%20the%20response%20of%20Caenorhabditis%20elegans%20towards%20Bacillus%20thuringiensis&volume=128&publication_year=2004&pages=433-443&pmid=15151149&doi=10.1017/s003118200300461x&)]
* 55.
  Johnke J, Dirksen P, Schulenburg H. 2020. Community assembly of the native C. elegans microbiome is influenced by time, substrate and individual bacterial taxa. Environ Microbiol
  22:1265–1279. doi: 10.1111/1462-2920.14932
   [[DOI](https://doi.org/10.1111/1462-2920.14932)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32003074/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&title=Community%20assembly%20of%20the%20native%20C.%20elegans%20microbiome%20is%20influenced%20by%20time,%20substrate%20and%20individual%20bacterial%20taxa&volume=22&publication_year=2020&pages=1265-1279&pmid=32003074&doi=10.1111/1462-2920.14932&)]
* 56.
  Ravichandran M, Priebe S, Grigolon G, Rozanov L, Groth M, Laube B, Guthke R, Platzer M, Zarse K, Ristow M. 2018. Impairing L-threonine catabolism promotes healthspan through methylglyoxal-mediated proteohormesis. Cell Metab
  27:914–925. doi: 10.1016/j.cmet.2018.02.004
   [[DOI](https://doi.org/10.1016/j.cmet.2018.02.004)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29551589/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Metab&title=Impairing%20L-threonine%20catabolism%20promotes%20healthspan%20through%20methylglyoxal-mediated%20proteohormesis&volume=27&publication_year=2018&pages=914-925&pmid=29551589&doi=10.1016/j.cmet.2018.02.004&)]
* 57.
  Liu YJ, Janssens GE, McIntyre RL, Molenaars M, Kamble R, Gao AW, Jongejan A, Weeghel M van, MacInnes AW, Houtkooper RH. 2019. Glycine promotes longevity in Caenorhabditis elegans in a methionine cycle-dependent fashion. PLoS Genet
  15:e1007633. doi: 10.1371/journal.pgen.1007633
   [[DOI](https://doi.org/10.1371/journal.pgen.1007633)] [[PMC free article](/articles/PMC6424468/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30845140/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Glycine%20promotes%20longevity%20in%20Caenorhabditis%20elegans%20in%20a%20methionine%20cycle-dependent%20fashion&volume=15&publication_year=2019&pages=e1007633&pmid=30845140&doi=10.1371/journal.pgen.1007633&)]
* 58.
  Urrutia A, García-Angulo VA, Fuentes A, Caneo M, Legüe M, Urquiza S, Delgado SE, Ugalde J, Burdisso P, Calixto A. 2020. Bacterially produced metabolites protect C. elegans neurons from degeneration. PLoS Biol
  18:e3000638. doi: 10.1371/journal.pbio.3000638
   [[DOI](https://doi.org/10.1371/journal.pbio.3000638)] [[PMC free article](/articles/PMC7092960/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32208418/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=Bacterially%20produced%20metabolites%20protect%20C.%20elegans%20neurons%20from%20degeneration&volume=18&publication_year=2020&pages=e3000638&pmid=32208418&doi=10.1371/journal.pbio.3000638&)]
* 59.
  Zečić A, Dhondt I, Braeckman BP. 2019. The nutritional requirements of Caenorhabditis elegans. Genes Nutr
  14:15. doi: 10.1186/s12263-019-0637-7
   [[DOI](https://doi.org/10.1186/s12263-019-0637-7)] [[PMC free article](/articles/PMC6501307/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31080524/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Nutr&title=The%20nutritional%20requirements%20of%20Caenorhabditis%20elegans&volume=14&publication_year=2019&pages=15&pmid=31080524&doi=10.1186/s12263-019-0637-7&)]
* 60.
  Ke W, Saba JA, Yao C-H, Hilzendeger MA, Drangowska-Way A, Joshi C, Mony VK, Benjamin SB, Zhang S, Locasale J, Patti GJ, Lewis N, O’Rourke EJ. 2020. Dietary serine-microbiota interaction enhances chemotherapeutic toxicity without altering drug conversion. Nat Commun
  11:2587. doi: 10.1038/s41467-020-16220-w
   [[DOI](https://doi.org/10.1038/s41467-020-16220-w)] [[PMC free article](/articles/PMC7244588/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32444616/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Dietary%20serine-microbiota%20interaction%20enhances%20chemotherapeutic%20toxicity%20without%20altering%20drug%20conversion&volume=11&publication_year=2020&pages=2587&pmid=32444616&doi=10.1038/s41467-020-16220-w&)]
* 61.
  Quigley EMM, Gajula P. 2020. Recent advances in modulating the microbiome. F1000Res
  9:F1000 Faculty Rev-46. doi: 10.12688/f1000research.20204.1 [[DOI](https://doi.org/10.12688/f1000research.20204.1)] [[PMC free article](/articles/PMC6993818/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32047611/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=F1000Res&title=Recent%20advances%20in%20modulating%20the%20microbiome&volume=9&publication_year=2020&pages=F1000%20Faculty%20Rev-46&pmid=32047611&doi=10.12688/f1000research.20204.1&)]
* 62.
  Stanier RY, Palleroni NJ, Doudoroff M. 1966. The aerobic pseudomonads: a taxonomic study. J Gen Microbiol
  43:159–271. doi: 10.1099/00221287-43-2-159
   [[DOI](https://doi.org/10.1099/00221287-43-2-159)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/5963505/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gen%20Microbiol&title=The%20aerobic%20pseudomonads:%20a%20taxonomic%20study&volume=43&publication_year=1966&pages=159-271&pmid=5963505&doi=10.1099/00221287-43-2-159&)]
* 63.
  Kim J, Jo Y, Cho D, Ryu D. 2022. L-threonine promotes healthspan by expediting ferritin-dependent ferroptosis inhibition in C. elegans. Nat Commun
  13:6554. doi: 10.1038/s41467-022-34265-x
   [[DOI](https://doi.org/10.1038/s41467-022-34265-x)] [[PMC free article](/articles/PMC9628521/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36323683/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=L-threonine%20promotes%20healthspan%20by%20expediting%20ferritin-dependent%20ferroptosis%20inhibition%20in%20C.%20elegans&volume=13&publication_year=2022&pages=6554&pmid=36323683&doi=10.1038/s41467-022-34265-x&)]
* 64.
  Kitamoto S, Alteri CJ, Rodrigues M, Nagao-Kitamoto H, Sugihara K, Himpsl SD, Bazzi M, Miyoshi M, Nishioka T, Hayashi A, et al. 2020. Dietary L-serine confers a competitive fitness advantage to Enterobacteriaceae in the inflamed gut. Nat Microbiol
  5:116–125. doi: 10.1038/s41564-019-0591-6
   [[DOI](https://doi.org/10.1038/s41564-019-0591-6)] [[PMC free article](/articles/PMC6925351/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31686025/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Microbiol&title=Dietary%20L-serine%20confers%20a%20competitive%20fitness%20advantage%20to%20Enterobacteriaceae%20in%20the%20inflamed%20gut&volume=5&publication_year=2020&pages=116-125&pmid=31686025&doi=10.1038/s41564-019-0591-6&)]
* 65.
  Sun Q, Vega NM, Cervantes B, Mancuso CP, Mao N, Taylor MN, Collins JJ, Khalil AS, Gore J, Lu TK. 2022. Enhancing nutritional niche and host defenses by modifying the gut microbiome. Mol Syst Biol
  18:e9933. doi: 10.15252/msb.20209933
   [[DOI](https://doi.org/10.15252/msb.20209933)] [[PMC free article](/articles/PMC9664710/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36377768/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Syst%20Biol&title=Enhancing%20nutritional%20niche%20and%20host%20defenses%20by%20modifying%20the%20gut%20microbiome&volume=18&publication_year=2022&pages=e9933&pmid=36377768&doi=10.15252/msb.20209933&)]
* 66.
  Lee HL, Shen H, Hwang IY, Ling H, Yew WS, Lee YS, Chang MW. 2018. Targeted approaches for in situ gut microbiome manipulation. Genes (Basel)
  9:351. doi: 10.3390/genes9070351
   [[DOI](https://doi.org/10.3390/genes9070351)] [[PMC free article](/articles/PMC6071227/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30002345/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20(Basel)&title=Targeted%20approaches%20for%20in%20situ%20gut%20microbiome%20manipulation&volume=9&publication_year=2018&pages=351&pmid=30002345&doi=10.3390/genes9070351&)]
* 67.
  Philip AK, Philip B. 2010. Colon targeted drug delivery systems: a review on primary and novel approaches. Oman Med J
  25:79–87. doi: 10.5001/omj.2010.24 [[DOI](https://doi.org/10.5001/omj.2010.24)] [[PMC free article](/articles/PMC3215502/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22125706/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oman%20Med%20J&title=Colon%20targeted%20drug%20delivery%20systems:%20a%20review%20on%20primary%20and%20novel%20approaches&volume=25&publication_year=2010&pages=79-87&pmid=22125706&doi=10.5001/omj.2010.24&)]
* 68.
  Naeem M, Awan UA, Subhan F, Cao J, Hlaing SP, Lee J, Im E, Jung Y, Yoo J-W. 2020. Advances in colon-targeted nano-drug delivery systems: challenges and solutions. Arch Pharm Res
  43:153–169. doi: 10.1007/s12272-020-01219-0
   [[DOI](https://doi.org/10.1007/s12272-020-01219-0)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31989477/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Pharm%20Res&title=Advances%20in%20colon-targeted%20nano-drug%20delivery%20systems:%20challenges%20and%20solutions&volume=43&publication_year=2020&pages=153-169&pmid=31989477&doi=10.1007/s12272-020-01219-0&)]
* 69.
  Fangmann D, Theismann E-M, Türk K, Schulte DM, Relling I, Hartmann K, Keppler JK, Knipp J-R, Rehman A, Heinsen F-A, Franke A, Lenk L, Freitag-Wolf S, Appel E, Gorb S, Brenner C, Seegert D, Waetzig GH, Rosenstiel P, Schreiber S, Schwarz K, Laudes M. 2018. Targeted microbiome intervention by microencapsulated delayed-release niacin beneficially affects insulin sensitivity in humans. Diabetes Care
  41:398–405. doi: 10.2337/dc17-1967
   [[DOI](https://doi.org/10.2337/dc17-1967)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29212824/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=Targeted%20microbiome%20intervention%20by%20microencapsulated%20delayed-release%20niacin%20beneficially%20affects%20insulin%20sensitivity%20in%20humans&volume=41&publication_year=2018&pages=398-405&pmid=29212824&doi=10.2337/dc17-1967&)]
* 70.
  Cao S-J, Xu S, Wang H-M, Ling Y, Dong J, Xia R-D, Sun X-H. 2019. Nanoparticles: oral delivery for protein and peptide drugs. AAPS PharmSciTech
  20:190. doi: 10.1208/s12249-019-1325-z
   [[DOI](https://doi.org/10.1208/s12249-019-1325-z)] [[PMC free article](/articles/PMC6527526/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31111296/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=AAPS%20PharmSciTech&title=Nanoparticles:%20oral%20delivery%20for%20protein%20and%20peptide%20drugs&volume=20&publication_year=2019&pages=190&pmid=31111296&doi=10.1208/s12249-019-1325-z&)]
* 71.
  Vargason AM, Anselmo AC, Mitragotri S. 2021. The evolution of commercial drug delivery technologies. Nat Biomed Eng
  5:951–967. doi: 10.1038/s41551-021-00698-w
   [[DOI](https://doi.org/10.1038/s41551-021-00698-w)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33795852/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biomed%20Eng&title=The%20evolution%20of%20commercial%20drug%20delivery%20technologies&volume=5&publication_year=2021&pages=951-967&pmid=33795852&doi=10.1038/s41551-021-00698-w&)]
* 72.
  Bae YH, Park K. 2020. Advanced drug delivery 2020 and beyond: perspectives on the future. Adv Drug Deliv Rev
  158:4–16. doi: 10.1016/j.addr.2020.06.018
   [[DOI](https://doi.org/10.1016/j.addr.2020.06.018)] [[PMC free article](/articles/PMC7736191/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32592727/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Drug%20Deliv%20Rev&title=Advanced%20drug%20delivery%202020%20and%20beyond:%20perspectives%20on%20the%20future&volume=158&publication_year=2020&pages=4-16&pmid=32592727&doi=10.1016/j.addr.2020.06.018&)]
* 73.
  Singal AG, Higgins PDR, Waljee AK. 2014. A primer on effectiveness and efficacy trials. Clin Transl Gastroenterol
  5:e45. doi: 10.1038/ctg.2013.13
   [[DOI](https://doi.org/10.1038/ctg.2013.13)] [[PMC free article](/articles/PMC3912314/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24384867/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Gastroenterol&title=A%20primer%20on%20effectiveness%20and%20efficacy%20trials&volume=5&publication_year=2014&pages=e45&pmid=24384867&doi=10.1038/ctg.2013.13&)]
* 74.
  Dwyer JT, Coates PM, Smith MJ. 2018. Dietary supplements: regulatory challenges and research resources. Nutrients
  10:41. doi: 10.3390/nu10010041
   [[DOI](https://doi.org/10.3390/nu10010041)] [[PMC free article](/articles/PMC5793269/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29300341/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nutrients&title=Dietary%20supplements:%20regulatory%20challenges%20and%20research%20resources&volume=10&publication_year=2018&pages=41&pmid=29300341&doi=10.3390/nu10010041&)]
* 75.
  Krawczak M, Nikolaus S, von Eberstein H, Croucher PJP, El Mokhtari NE, Schreiber S. 2006. PopGen: population-based recruitment of patients and controls for the analysis of complex genotype-phenotype relationships. Community Genet
  9:55–61. doi: 10.1159/000090694
   [[DOI](https://doi.org/10.1159/000090694)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16490960/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Community%20Genet&title=PopGen:%20population-based%20recruitment%20of%20patients%20and%20controls%20for%20the%20analysis%20of%20complex%20genotype-phenotype%20relationships&volume=9&publication_year=2006&pages=55-61&pmid=16490960&doi=10.1159/000090694&)]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

Reviewer comments

[reviewer-comments.pdf](/articles/instance/10846184/bin/reviewer-comments.pdf) (123.6KB, pdf)

Supplementary figures. spectrum.01144-23-s0001.docx.

Figures S1 and S2.

[Click here for additional data file.](/articles/instance/10846184/bin/spectrum.01144-23-s0001.docx) (415.2KB, docx)

DOI: 10.1128/spectrum.01144-23.SuF1

Supplementary tables. spectrum.01144-23-s0002.xlsx.

Tables S1-S20.

[Click here for additional data file.](/articles/instance/10846184/bin/spectrum.01144-23-s0002.xlsx) (3.3MB, xlsx)

DOI: 10.1128/spectrum.01144-23.SuF2

OPEN PEER REVIEW. reviewer-comments.pdf.

An accounting of the reviewer comments and feedback.

[Click here for additional data file.](/articles/instance/10846184/bin/reviewer-comments.pdf) (123.6KB, pdf)

DOI: 10.1128/spectrum.01144-23.SuF3

### Data Availability Statement

The R scripts and the raw data to conduct the simulations are deposited in [Github](https://github.com/maringos/Nutritional-Supplements-Prediction) (version/commit: [e15eb1e](https://github.com/maringos/Nutritional-Supplements-Prediction/commit/e15eb1e5175fe042dff1e891c90286649ae676c0)). Metabolic models used in the prediction of unique uptake compounds have been deposited in [FigShare](https://doi.org/10.6084/m9.figshare.24968871.v1). The growth rate changes of the supplementation experiments with FBA sybil and MicrobiomeGS2 can be found in Supplementary Tables S3 and S4. The bacterial biomass data of the BacArena simulations can be found in Supplementary Table S5.