<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9549344/"/><title>A Region of UNC-89 (obscurin) Lying Between Two Protein Kinase Domains is a Highly Elastic Spring Required for Proper Sarcomere Organization - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">J Mol Biol</button></div>. Author manuscript; available in PMC: 2022 Oct 10.</div>
<div>
<em>Published in final edited form as: </em>J Mol Biol. 2020 Jul 6;432(17):4799–4814. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jmb.2020.06.024" rel="noopener noreferrer" target="_blank">10.1016/j.jmb.2020.06.024</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A Region of UNC-89 (obscurin) Lying Between Two Protein Kinase Domains is a Highly Elastic Spring Required for Proper Sarcomere Organization</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qadota%20H%22%5BAuthor%5D"><span class="name western">Hiroshi Qadota</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Hiroshi Qadota</span></h3>
<div class="p">
<sup>a</sup>Department of Pathology, Emory University, Atlanta, Georgia 30322</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qadota%20H%22%5BAuthor%5D"><span class="name western">Hiroshi Qadota</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moody%20JC%22%5BAuthor%5D"><span class="name western">Jasmine C Moody</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jasmine C Moody</span></h3>
<div class="p">
<sup>a</sup>Department of Pathology, Emory University, Atlanta, Georgia 30322</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moody%20JC%22%5BAuthor%5D"><span class="name western">Jasmine C Moody</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lesanpezeshki%20L%22%5BAuthor%5D"><span class="name western">Leila Lesanpezeshki</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Leila Lesanpezeshki</span></h3>
<div class="p">
<sup>b</sup>Department of Chemical Engineering, Texas Tech University, Lubbock, Texas 79409</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lesanpezeshki%20L%22%5BAuthor%5D"><span class="name western">Leila Lesanpezeshki</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moncrief%20T%22%5BAuthor%5D"><span class="name western">Taylor Moncrief</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Taylor Moncrief</span></h3>
<div class="p">
<sup>c</sup>Department of Neuroscience, Cell Biology and Anatomy, University of Texas Medical Branch, Galveston, Texas 77555</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moncrief%20T%22%5BAuthor%5D"><span class="name western">Taylor Moncrief</span></a>
</div>
</div>
<sup>c</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kitzler%20D%22%5BAuthor%5D"><span class="name western">Deborah Kitzler</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Deborah Kitzler</span></h3>
<div class="p">
<sup>a</sup>Department of Pathology, Emory University, Atlanta, Georgia 30322</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kitzler%20D%22%5BAuthor%5D"><span class="name western">Deborah Kitzler</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhat%20PD%22%5BAuthor%5D"><span class="name western">Purnima Devaki Bhat</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Purnima Devaki Bhat</span></h3>
<div class="p">
<sup>a</sup>Department of Pathology, Emory University, Atlanta, Georgia 30322</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhat%20PD%22%5BAuthor%5D"><span class="name western">Purnima Devaki Bhat</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vanapalli%20SA%22%5BAuthor%5D"><span class="name western">Siva A Vanapalli</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Siva A Vanapalli</span></h3>
<div class="p">
<sup>b</sup>Department of Chemical Engineering, Texas Tech University, Lubbock, Texas 79409</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vanapalli%20SA%22%5BAuthor%5D"><span class="name western">Siva A Vanapalli</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Oberhauser%20AF%22%5BAuthor%5D"><span class="name western">Andres F Oberhauser</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Andres F Oberhauser</span></h3>
<div class="p">
<sup>c</sup>Department of Neuroscience, Cell Biology and Anatomy, University of Texas Medical Branch, Galveston, Texas 77555</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Oberhauser%20AF%22%5BAuthor%5D"><span class="name western">Andres F Oberhauser</span></a>
</div>
</div>
<sup>c</sup>, <a aria-describedby="id9" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Benian%20GM%22%5BAuthor%5D"><span class="name western">Guy M Benian</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Guy M Benian</span></h3>
<div class="p">
<sup>a</sup>Department of Pathology, Emory University, Atlanta, Georgia 30322</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Benian%20GM%22%5BAuthor%5D"><span class="name western">Guy M Benian</span></a>
</div>
</div>
<sup>a</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="A1">
<sup>a</sup>Department of Pathology, Emory University, Atlanta, Georgia 30322</div>
<div id="A2">
<sup>b</sup>Department of Chemical Engineering, Texas Tech University, Lubbock, Texas 79409</div>
<div id="A3">
<sup>c</sup>Department of Neuroscience, Cell Biology and Anatomy, University of Texas Medical Branch, Galveston, Texas 77555</div>
<div class="author-notes p"><div class="fn" id="CR1">
<sup>✉</sup><p class="display-inline">Corresponding author: Guy M. Benian, 404-727-5953, <span>pathgb@emory.edu</span>, Department of Pathology, Emory University, Whitehead Biomedical Research Building 105E, 615 Michael Street, Atlanta, Georgia 30322 USA</p>
</div></div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Issue date 2020 Aug 7.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none"><div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div></div>
</div>
<div>PMCID: PMC9549344  NIHMSID: NIHMS1838736  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32645312/">32645312</a>
</div>
<div aria-label="Related or updated information about this article" class="ra xbox p" role="complementary"><div>The publisher's version of this article is available at <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jmb.2020.06.024" rel="noopener noreferrer" target="_blank">J Mol Biol</a>
</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="ABS1"><h2>Abstract</h2>
<p id="P1">In <em>C. elegans</em>, <em>unc-89</em> encodes a set of giant multi-domain proteins (up 8,081 residues) localized to the M-lines of muscle sarcomeres and required for normal sarcomere organization and whole-animal locomotion. Multiple UNC-89 isoforms contain two protein kinase domains. There is conservation in arrangement of domains between UNC-89 and its two mammalian homologs, obscurin and SPEG: kinase, a non-domain region of 647–742 residues, Ig domain, Fn3 domain and a second kinase domain. In all three proteins, this non-domain “interkinase region” has low sequence complexity, high proline content and lacks predicted secondary structure. We report that a major portion of this interkinase (571 residues out of 647 residues) when examined by single molecule force spectroscopy <em>in vitro</em> displays the properties of a random coil and acts as an entropic spring. We used CRISPR/Cas9 to create nematodes carrying an in-frame deletion of the same 571-residue portion of the interkinase. These animals display severe disorganization of all portions of the sarcomere in body wall muscle. Super-resolution microscopy reveals extra, short-A-bands lying close to the outer muscle cell membrane and between normally spaced A-bands. Nematodes with this in-frame deletion show defective locomotion and muscle force generation. We designed our CRISPR-generated in-frame deletion to contain an HA tag at the N-terminus of the large UNC-89 isoforms. This HA tag results in normal organization of body wall muscle, but approximately half the normal levels of the giant UNC-89 isoforms, dis-organization of pharyngeal muscle, small body size, and reduced muscle force, likely due to poor nutritional uptake.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> sarcomere, giant polypeptides, UNC-89, obscurin, <em>C. elegans</em></p></section></section><section id="S1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="P2"><em>C. elegans</em> UNC-89 is the founding member of the UNC-89/obscurin family of giant muscle proteins. Loss-of-function mutations in the <em>C. elegans</em> gene <em>unc-89</em> result in slow moving worms with disorganized muscle sarcomeres, including a lack of M-lines [<a aria-describedby="R1" class="usa-link" href="#R1">1</a>–<a aria-describedby="R3" class="usa-link" href="#R3">3</a>]. <em>unc-89</em> is a complex gene: through the use of three promoters and alternative splicing at least 16 polypeptides are generated, ranging in size from 156,000 to 900,000 Da [<a aria-describedby="R3" class="usa-link" href="#R3">3</a>–<a aria-describedby="R5" class="usa-link" href="#R5">5</a>]. Overlapping sets of isoforms are expressed in body wall muscle, pharyngeal muscle (for pumping in and grinding up bacterial food), egg laying and intestinal muscles [<a aria-describedby="R3" class="usa-link" href="#R3">3</a>], and several gonadal epithelial cells [<a aria-describedby="R6" class="usa-link" href="#R6">6</a>]. The largest of these isoforms, UNC-89-B and UNC-89-F, consist of 53 Ig domains, two Fn3 domains, a triplet of SH3, DH and PH domains near their N-termini, and two protein kinase domains (PK1 and PK2) near their C termini. Antibodies localize UNC-89 to the M-line [<a aria-describedby="R3" class="usa-link" href="#R3">3</a>,<a aria-describedby="R4" class="usa-link" href="#R4">4</a>]. To learn how UNC-89 is localized and performs its functions, our lab has systematically identified its binding partners. We have screened a yeast two hybrid library with segments representing the largest UNC-89 isoform and identified at least 7 interacting partners, and 6/7 have orthologs or homologs in humans [<a aria-describedby="R7" class="usa-link" href="#R7">7</a>–<a aria-describedby="R13" class="usa-link" href="#R13">13</a>]. Interactions have been verified by a combination of demonstrating binding in vitro using purified recombinant proteins, co-immunolocalization, and identifying muscle mutant phenotypes.</p>
<p id="P3">Mammalian striated muscle express three UNC-89-like proteins from separate genes, obscurin (OBSC), obscurin-like 1 (obsl1) and striated muscle preferentially expressed gene (SPEG). UNC-89 is most similar to obscurin [<a aria-describedby="R14" class="usa-link" href="#R14">14</a>,<a aria-describedby="R15" class="usa-link" href="#R15">15</a>], which also expresses many isoforms, including giant isoforms (obscurin A), and shorter isoforms with 2 protein kinase domains [<a aria-describedby="R16" class="usa-link" href="#R16">16</a>]. Although UNC-89 is located only at the M-line, various obscurin isoforms are located at either the M-line, or the Z-disk [<a aria-describedby="R15" class="usa-link" href="#R15">15</a>,<a aria-describedby="R17" class="usa-link" href="#R17">17</a>]. Mutations in the human OBSC gene result in cardiomyopathies (HCM, DCM, LVNC)[<a aria-describedby="R18" class="usa-link" href="#R18">18</a>]. In addition to serving as a platform for assembly of M-line proteins, obscurin links myofibrils to the sarcoplasmic reticulum (SR). The C-terminus of obscurin interacts with an ankyrin localized to the SR [<a aria-describedby="R19" class="usa-link" href="#R19">19</a>,<a aria-describedby="R20" class="usa-link" href="#R20">20</a>]. The obscurin KO mouse has normal myofibril organization but disorganized SR [<a aria-describedby="R21" class="usa-link" href="#R21">21</a>]. This myofibril to SR linkage is conserved in nematode UNC-89 [<a aria-describedby="R22" class="usa-link" href="#R22">22</a>]. Obsl1 is an ~200 kDa protein consisting solely of 20 tandem Ig domains, and in heart muscle is localized to M-lines, Z-disks and intercalated disks [<a aria-describedby="R23" class="usa-link" href="#R23">23</a>]. Several portions interact with titin and myomesin at the M-line [<a aria-describedby="R24" class="usa-link" href="#R24">24</a>]. However, obsl1 is ubiquitously expressed, global KO is embryonic lethal and mutations in the human Obsl1 gene result in a growth defect (3M-growth syndrome)[<a aria-describedby="R25" class="usa-link" href="#R25">25</a>]. SPEG also exists as different sized isoforms, but the largest isoform is 3,267 aa and consists of 9 Ig and 2 Fn3 domains, and 2 protein kinase domains [<a aria-describedby="R26" class="usa-link" href="#R26">26</a>]. It can be regarded as similar to the C-terminal tandem kinase domain regions of obscurin and UNC-89. SPEG is located in a complex lying between the SR and plasma membrane (JMC) in cardiomyocytes. In the mouse, cardiac-specific KO results in DCM, whereas mutations in human SPEG result in centronuclear myopathy with DCM [<a aria-describedby="R26" class="usa-link" href="#R26">26</a>]. The <em>in vivo</em> substrates for the kinase domains of UNC-89, obscurin and SPEG are not known. However, sequence analysis suggests that: (i) UNC-89 PK1 is catalytically inactive, whereas PK2 is an active kinase [<a aria-describedby="R27" class="usa-link" href="#R27">27</a>]; (ii) human obscurin PK1 and PK2 are both likely to be active kinases [<a aria-describedby="R27" class="usa-link" href="#R27">27</a>], and each has been shown to undergo autophosphorylation <em>in vitro</em> [<a aria-describedby="R28" class="usa-link" href="#R28">28</a>]. Obscurin PK2 can phosphorylate N-cadherin in vitro [<a aria-describedby="R28" class="usa-link" href="#R28">28</a>]. There is a suggestion that SPEG phosphorylates the JMC protein junctophilin 2 [<a aria-describedby="R29" class="usa-link" href="#R29">29</a>].</p>
<p id="P4">As shown in <a class="usa-link" href="#F1">Fig. 1A</a>, UNC-89, obscurin and SPEG have an overall conservation in organization around the kinase domains: PK1, a non-domain region of 647–742 residues, Ig, Fn, and PK2. There is very little sequence homology amongst the non-domain or “interkinase” (IK) regions of these proteins (<a class="usa-link" href="#SD1">Supplemental Figure 1</a>). However, these IK regions have in common, a low sequence complexity, high proline content (11.8—13.3%), and an overall paucity of predicted α-helix and β-strands, and thus are likely to be predominantly random coils [<a aria-describedby="R9" class="usa-link" href="#R9">9</a>]. These properties of the IK regions are shared with the known highly elastic spring regions of human titin (PEVK and N2A/B)[<a aria-describedby="R30" class="usa-link" href="#R30">30</a>]. In this paper, we describe, for nematode UNC-89, the response of the interkinase to mechanical pulling and the phenotype of worms in which the interkinase has been deleted in-frame. This is the first time the function of this region has been explored for any member of the UNC-89/obscurin family.</p>
<figure class="fig xbox font-sm" id="F1"><h3 class="obj_head">Fig. 1. Schematic representation of domain organization around kinase domains, and secondary structure predictions for the interkinase regions of nematode UNC-89, mouse obscurin and SPEG.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 1." class="graphic" height="1780" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6219/9549344/7df97df76f05/nihms-1838736-f0001.jpg" width="2100"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P50">(A) Domain organization from PFAM predictions. Note that each protein, near its C-terminus consists of a protein kinase, a non-domain “interkinase” region (647–742 residues), Ig and Fn domains, and a second protein kinase domain. (B) Secondary structure predictions of the non-domain interkinase regions of UNC-89, obscurin and SPEG using JPred.</p></figcaption></figure></section><section id="S2"><h2 class="pmc_sec_title">Results</h2>
<p id="P5">To examine the nature of the IK regions of UNC-89, obscurin and SPEG in greater detail, we used JPred to predict secondary structure [<a aria-describedby="R30" class="usa-link" href="#R30">30</a>]. As shown in <a class="usa-link" href="#SD2">Supplemental Figure 2</a>, and in <a class="usa-link" href="#F1">Fig. 1B</a>, there are only short stretches of α-helix (15, 9, 9, 11, 7 and 7 amino acids, from N- to C-terminus of IK), and even fewer and shorter instances of β-strands (3, 2, 3, 4 and 6 amino acids). Except for the 15-residue long region near the IK N-terminus, all the other instances of α-helices reside in the C-terminal one third of the IK sequence. These α-helices reside within the C-terminal 232 residues which is the minimal binding site determined for LIM-9 (FHL2) [<a aria-describedby="R9" class="usa-link" href="#R9">9</a>], and thus may comprise an unrecognized or novel domain. Thus, the remaining sequence of UNC-89 IK, approximately, 475 residues, is random coil. As shown in <a class="usa-link" href="#SD2">Supplemental Figure 2</a> and <a class="usa-link" href="#F1">Fig. 1B</a>, obscurin and SPEG have even fewer predicted secondary structure elements, although like UNC-89, obscurin has a fairly long α-helix of 19 residues lying within the C-terminal 52 residues.</p>
<p id="P6">Given the nature of the amino acid sequence of the IK region of UNC-89, we wished to determine how it might respond to a mechanical pulling force using the single molecule force spectroscopy technique. To carry this out, we first expressed in E. coli 1 571 residue segment comprising the majority of the 645 residue interkinase region (IK) flanked on the N- and C-terminus by two copies of Ig27 from human titin. The response of Ig27 to pulling force is well documented and served as an internal control. In addition, the protein had a 6XHis tag at its N-terminus (<a class="usa-link" href="#F2">Fig. 2A</a>). After purification (<a class="usa-link" href="#F2">Fig. 2B</a>), the protein was allowed to attach to a glass surface coated with Ni<sup>2+</sup>-NTA to allow binding of the N-terminus of this polyprotein via its 6XHis tag. The stylus of the single-molecule force spectrometer was allowed to bind non-specifically and at random points along the polyprotein, and then pulling force was applied by moving the piezo-electric platform downward. A representative force curve of the mechanical unfolding of the polyprotein in which the stylus had attached to the C-terminus is shown in <a class="usa-link" href="#F2">Fig. 2C</a>. After a long and featureless trace without energy barriers, the typical unwinding of the 4 Ig domains is found. The statistical distribution of the contour length of the long featureless peak (<a class="usa-link" href="#F2">Fig. 2D</a>) shows that the mean contour is approximately 185 nm. This is a good match to the expected contour length of a fully unwound polypeptide of 571 amino acids, since the average length of an amino acid is 0.34 nm, i.e. 0.34 nm/aa × 571 aa = 194 nm. Based on this force curve and contour length, we can conclude that the IK has the mechanical properties of a random coil and behaves like an entropic spring.</p>
<figure class="fig xbox font-sm" id="F2"><h3 class="obj_head">Fig. 2. The interkinase region (IK) of UNC-89 is a highly elastic spring.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 2." class="graphic" height="1765" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6219/9549344/5f8b486f712d/nihms-1838736-f0002.jpg" width="2100"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P51">(A) Polyprotein construct encompassing the IK flanked on either side by two titin I27 domains used in single molecule force spectroscopy (SMFS) experiments. (<strong>B</strong>) SDS-PAGE after Coomassie Blue staining of ~2 μg of the polyprotein shown in (A). (<strong>C</strong>) Representative force curve of the mechanical unfolding of the polyprotein by SMFS. The dotted line is a worm-like-chain (WLC) fit to the force-extension curve with free contour length L<sub>C</sub> and a fixed persistence length L<sub>p</sub> = 0.36 nm. (<strong>D</strong>) Statistical distribution of the contour length of the first peak (L<sub>C</sub> = 186 ± 3 nm, n=18). The solid line represents a Gaussian fit of the histogram.</p></figcaption></figure><p id="P7">We wished to determine the <em>in vivo</em> consequences of removing the IK from UNC-89. To do this, we used CRISPR/Cas9 to create an in-frame deletion of the same 571 residues that we explored with the single molecule method in vitro. However, to be able to monitor the expression of UNC-89 large isoforms in which the IK had been deleted we added an HA tag to the N-terminus of the large isoforms. We thought such tagged UNC-89 could also be used in future immunoprecipitation experiments to identify proteins in a complex with UNC-89. Animals expressing HA-UNC-89 large isoforms were obtained by CRISPR/Cas9, and the strain is designated <em>unc-89(syb797)</em>. The proper fusion of a tag containing three consecutive HA sequences to the normal N-terminus of the large UNC-89 isoforms was verified by Sanger sequencing (<a class="usa-link" href="#SD3">Supplemental Figure 3</a>). The presence of the HA tag on the large UNC-89 isoforms was verified by western blot (<a class="usa-link" href="#F3">Fig. 3A</a>). Moreover, the HA-UNC-89 proteins could be detected by confocal microscopy to be normally localized to the sarcomeric M-lines of body wall muscle (<a class="usa-link" href="#F3">Fig. 3B</a>). The HA tag also does not affect the organization of body wall muscle sarcomeres as assessed by confocal microscopy using phalloidin for I-bands, or antibodies to myosin heavy chain A for A-bands, UNC-89 for M-lines, UNC-95 for the bases of M-lines and dense bodies, or ATN-1 (α-actinin) for the major portion of dense bodies (<a class="usa-link" href="#F4">Fig. 4</a>). As shown later, this HA tag also does not affect whole animal locomotion. However, HA tag does reduce the abundance of the UNC-89 giant isoforms (see <a class="usa-link" href="#F3">Fig. 3</a> and <a class="usa-link" href="#F5">Fig. 5</a>). With an n=2, the level of expression from <em>unc-89(syb797)</em> is 51.2% (standard deviation of 11%) of the level from wild type.</p>
<figure class="fig xbox font-sm" id="F3"><h3 class="obj_head">Fig. 3. Addition of HA tags at the N-termini of the largest UNC-89 isoforms does not affect the normal M-line localization of UNC-89 in body wall muscle sarcomeres, but does moderately reduce the level of UNC-89 giant isoforms in whole animal extracts.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 3." class="graphic" height="2100" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6219/9549344/d6065397227b/nihms-1838736-f0003.jpg" width="2015"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P52">(A) Total Laemmli-soluble proteins from wild type and <em>unc-89(syb797)</em> were separated on a 5% SDS-PAGE, transferred to membrane and reacted with antibody EU30, a well-established rabbit polyclonal antibody to UNC-89. On the right is shown Ponceau S staining of the blots before reaction with antibodies. Note the presence of HA tags in <em>unc-89(syb797)</em> but not in wild type, and a moderate reduction in the level of UNC-89 giant isoforms in <em>unc-89(sy797)</em> compared with wild type. The asterisk indicates the position of the UNC-89 giant isoforms. The thin upper-most band is likely to be some UNC-89 that could not enter the separating gel. The positions of size markers are shown on the left-hand side. (B) Co-immunostaining of parts of two body wall muscle cells in <em>unc-89(syb797)</em> with anti-HA and MH42, a well-established monoclonal that recognizes UNC-89. Note the co-localization of each antibody to the M-lines of these muscle sarcomeres (merged image). Scale bar, 10 μm.</p></figcaption></figure><figure class="fig xbox font-sm" id="F4"><h3 class="obj_head">Fig. 4. Addition of HA tags at the N-termini of the largest UNC-89 isoforms does not affect the organization of sarcomeres in body wall muscle.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 4." class="graphic" height="2100" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6219/9549344/0d6d3592ef18/nihms-1838736-f0004.jpg" width="1355"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P53">Each panel shows localization of the indicated sarcomere components using confocal microscopy. Scale bar, 10 μm.</p></figcaption></figure><figure class="fig xbox font-sm" id="F5"><h3 class="obj_head">Fig. 5. <em>unc-89(syb797 syb1257)</em> is an in-frame deletion of most of the IK region and results in mis-localization of UNC-89 in the sarcomeres of body wall muscle.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 5." class="graphic" height="2100" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6219/9549344/939685431ace/nihms-1838736-f0005.jpg" width="1548"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P54">(A) Representation of the exon (black boxes) and intron (lines) organization of the kinase-encoding region of the <em>unc-89</em> gene. Blue bar, portion deleted in <em>syb1257</em> by CRISPR/Cas9. This is a 4,614 bp genomic in-frame deletion which internally deletes 571 aa of the IK beginning 66 aa C-terminal of PK1 and continuing through 10 aa N-terminal of Ig53. (B) Western blots demonstrating that the large UNC-89 isoforms are expressed in <em>unc-89(syb797)</em> and <em>unc-89(syb797 syb1257)</em>. Western blots and reactions conducted as in <a class="usa-link" href="#F3">Fig. 3</a>. The asterisk indicates the position of the UNC-89 giant isoforms. The thin upper-most band is likely to be some UNC-89 that could not enter the separating gel. (C) Co-immunostaining of parts of several body wall muscle cells in <em>unc-89(syb797 syb1257)</em> with anti-HA and the anti-UNC-89 antibody MH42. Note the disorganization of M-lines in this mutant. Scale bar, 10 μm.</p></figcaption></figure><p id="P8">Next, we used <em>unc-89(syb797)</em> as starting point to make an in-frame deletion of 571 residues of the IK by CRISPR/Cas9, and this strain is designated <em>unc-89(syb797 syb1257)</em>. A schematic of the genomic region deleted is shown in <a class="usa-link" href="#F5">Fig. 5a</a>. Sanger sequencing was used to verify that the 4,614 bp genomic deletion resulted in an in-frame deletion of the <em>unc-89</em> coding sequence (<a class="usa-link" href="#SD4">Supplemental Figure 4</a>). Before characterizing either <em>unc-89(syb797)</em> or <em>unc-89(syb797syb1257)</em> strains further, each was outcrossed 5X to wild type, to remove most possible off-target effects of CRISPR/Cas9 gene editing.</p>
<p id="P9">As shown in <a class="usa-link" href="#F5">Fig. 5B</a>, the in-frame deletion of IK (in <em>syb797syb1257</em>), leads to expression of UNC-89 giant isoforms that can be detected with either antibodies to the N-terminal HA tag or our standard antibodies raised against epitopes in the middle of UNC-89. We had hoped to be able to use an antibody to PK2 to prove that the deletion is in-frame, however, two attempts to make such an antibody (in rabbits or rats) failed. Nevertheless, since the levels of large isoforms did not diminish below that shown with the HA tag (in <em>syb797</em>), it is likely that we achieved an in-frame deletion which does not destabilize the protein. We expected to observe slightly smaller giant isoforms from <em>syb797syb1257</em> compared to wild type, but there seems to be no discernible difference (<a class="usa-link" href="#F5">Fig. 5B</a>). Although there are at least 14 predicted giant isoforms ranging from 5,000 to 8,081 residues each, we do not know the relative abundance of each of these giant isoforms. Deletion of 571 residues out of a total of 8,081 residues for the largest isoform would be only a decrease of 7% and may not be a difference detectable by the gel system used. However, as shown in <a class="usa-link" href="#F5">Fig. 5C</a>, the internal deletion of the IK of UNC-89 results in disorganization of M-lines in body wall muscle. This can be seen when either an established monoclonal for UNC-89, MH42, or anti-HA is used in immunostaining. In fact, the signals from each antibody overlap.</p>
<p id="P10">To assess the <em>in vivo</em> consequences of removing the IK, we examined the organization of body wall muscle sarcomeres in greater detail, and measured whole animal locomotion and force generation. To assess sarcomere organization of the IK deletion, we took the same strategy as we did to characterize the sarcomere organization of <em>unc-89(syb797)</em>: immunolocalization of several proteins with known sarcomere locations using confocal microscopy. Deletion of IK (in mutant <em>syb797syb1257</em>) results in disorganization of sarcomeres in all substructures (<a class="usa-link" href="#F6">Fig. 6</a>). It is important to note however that in <em>syb797syb1257</em>, although the pattern of UNC-89 is disorganized (i.e. not parallel straight lines as in <em>syb797</em> or wild type), UNC-89 is still detectable at high levels consistent with the western blot result. Note also that in <em>syb797syb1257</em>, the disorganization of UNC-89 appears quite similar to the pattern of disorganization of myosin heavy chain A (MHC A). This is not unexpected as thick filaments in nematode body wall muscle have differential localization of two myosin heavy chain isoforms, with MHC A in the middle, and MHC B in the major outer portions [<a aria-describedby="R31" class="usa-link" href="#R31">31</a>]. Normally thick filaments are arranged parallel and in register, so that the M-line is the structure in the middle of the A-band, which contains MHC A, and where the thick filaments are crosslinked. Thus, when imaging <em>C. elegans</em> body wall muscle with light microscopy, the localization of MHC A and UNC-89 are nearly identical [<a aria-describedby="R4" class="usa-link" href="#R4">4</a>,<a aria-describedby="R32" class="usa-link" href="#R32">32</a>]. Note that upon anti-UNC-95 staining, the bases of M-lines are often branched and can overlap the bases of dense bodies (<a class="usa-link" href="#F6">Fig. 6</a>, right inset). Overall, these results indicate that the IK is required for the proper organization of sarcomeres.</p>
<figure class="fig xbox font-sm" id="F6"><h3 class="obj_head">Fig. 6. <em>unc-89(syb797 syb1257)</em> animals that express HA-UNC-89 isoforms with an in-frame deletion of the interkinase region have disorganized myofibrils.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 6." class="graphic" height="2100" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6219/9549344/0254f146a483/nihms-1838736-f0006.jpg" width="1370"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P55">Each panel shows representative confocal images of several body wall muscle cells stained with antibodies to the indicated sarcomeric components or with rhodamine-phalloidin. Scale bar, 10 μm.</p></figcaption></figure><p id="P11">We examined the organization of MHC A and UNC-89, proxies for the middle of A-bands, at higher resolution using structured illumination microscopy (SIM). In <a class="usa-link" href="#F7">Fig. 7 A</a> and <a class="usa-link" href="#F7">B</a>, it is clear that the N-terminal HA tag on the large UNC-89 isoforms (in <em>syb797</em>) has no effect on the normal pattern of straight parallel A-bands. However, in-frame deletion of IK (in <em>syb797syb1257</em>), for both MHC A and UNC-89 immunostaining, it is evident that the A-bands are broken, and much of the parallel arrangement is missing. In addition, there appear to be extra, short A-bands (indicated by arrows in <a class="usa-link" href="#F7">Fig. 7</a>) that lie in the spaces between otherwise nearly normal A-bands. By capturing optical sections with an interval of 0.2 μm, we noticed that these partial A-bands lie mostly close to the outer cell membrane of the muscle cells. From top to bottom of <a class="usa-link" href="#F7">Fig. 7A</a> and <a class="usa-link" href="#F7">B</a>, images are shown from close to the outer muscle cell membrane to deeper into the cell. Note that the partial A-bands lie near the outer muscle cell but do not extend as deeply as normal A-bands.</p>
<figure class="fig xbox font-sm" id="F7"><h3 class="obj_head">Fig. 7. <em>unc-89(syb797 syb1257)</em> animals that express HA-UNC-89 isoforms with an in-frame deletion of the interkinase region show extra A-bands that are not full-depth.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 7." class="graphic" height="2100" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6219/9549344/04108e937979/nihms-1838736-f0007.jpg" width="870"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F7/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P56">After immunostaining with antibodies to MHC A (A) or antibodies to UNC-89 (B) multiple optical slices with a 0.2 μm interval were obtained by structured illumination microscopy (SIM). Yellow and blue arrows point to examples of short extra A-bands that lie between normally located A-bands and reside close to the outer muscle cell membrane. In comparison, <em>unc-89(syb797)</em> that expresses HA-UNC-89 shows parallel, normally spaced and full-depth A-bands. Scale bar, 5 μm.</p></figcaption></figure><p id="P12">In <em>C. elegans</em>, the body wall muscle is required for the animal to generate the forces required for its normal swimming, crawling and burrowing locomotion [<a aria-describedby="R33" class="usa-link" href="#R33">33</a>]. Given the disorganization of sarcomeres in the body wall muscle of nematodes expressing large isoforms of UNC-89 with an in-frame deletion of the IK, we wondered if these animals would display defects in movement or in muscle force generation. First, because our strain with the IK deletion also contains an N-terminal HA tag, we measured locomotion in the strain with the HA tag alone, <em>syb797</em>. As shown in <a class="usa-link" href="#F8">Fig. 8A</a> and <a class="usa-link" href="#F8">B</a>, <em>syb797</em> is not slower in swimming or crawling as compared to wild type. In fact, <em>syb797</em> displays a statistically faster swimming than wild type. However, clearly, <em>syb797syb1257</em>, containing both an HA tag and IK deletion shows a 15% reduction in swimming and a 30% reduction in crawling (<a class="usa-link" href="#F8">Fig. 8A</a> and <a class="usa-link" href="#F8">B</a>). This decrease in swimming speed is comparable to that displayed by the canonical <em>unc-89</em> allele <em>e1460</em>, but not as severe as that of alleles <em>su75</em> or <em>tm752</em> (<a class="usa-link" href="#F8">Fig. 8A</a>). The decrease in crawling of the IK deletion strain is not as severe as 3 of 4 other alleles tested (<em>e1460, su75, tm752</em>). In terms of muscle force exerted, as measured by the NemaFlex assay [<a aria-describedby="R34" class="usa-link" href="#R34">34</a>], all mutant alleles (including <em>syb797 syb1257</em>) except for <em>tm752</em> are defective compared to wild type (<a class="usa-link" href="#F8">Fig. 8C</a>). Interestingly, <em>syb797</em>, which has an HA tag at the N-terminus of the large isoforms, shows no decrease in crawling, but an increase in swimming, and decreased ability to generate force. This phenotype of increased speed of swimming, no change or even increased speed of crawling, and yet decreased force, is shared with the allele <em>ad539</em>. <em>unc-89(ad539)</em>, like <em>syb797</em>, has normally organized sarcomeres in body wall muscle, which likely accounts for lack of any decrease in swimming or crawling. (We first showed that <em>unc-89(ad539)</em> has normally organized sarcomeres by EM [<a aria-describedby="R2" class="usa-link" href="#R2">2</a>], but to verify this result we have immunostained its muscle with a variety of antibodies to sarcomeric proteins, and all of these are normally localized (<a class="usa-link" href="#SD5">Supplemental Figure 5</a>)).</p>
<figure class="fig xbox font-sm" id="F8"><h3 class="obj_head">Fig. 8. Whole worm locomotion and muscle force measurements of wild type and various <em>unc-89</em> mutant alleles.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 8." class="graphic" height="2100" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6219/9549344/905aa5cb16cd/nihms-1838736-f0008.jpg" width="571"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F8/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P57">(A) Swimming assays show that <em>unc-89(syb797)</em> and <em>unc-89(ad539)</em> swim faster than wild type, whereas all other <em>unc-89</em> mutants, including <em>unc-89(syb797 syb1257)</em> swim slower than wild type. (B) Crawling assays show that <em>unc-89(ad539)</em> crawls faster than wild type, that <em>unc-89(syb797)</em> crawls at the same speed as wild type, and all other <em>unc-89</em> mutants crawl more slowly than wild type. C) NemaFlex force measurements show that <em>unc-89(tm752)</em> develops the same muscle force as wild type, whereas all other <em>unc-89</em> mutants develop less force than wild type, although, <em>unc-89(syb797 syb1257)</em>, <em>unc-89(e1460)</em>, and <em>unc-89(su75)</em> are more severely affected. The actual data, including sample sizes, means, and standard errors are shown in <a class="usa-link" href="#SD7">Supplementary Table 1</a> for swimming and crawling and <a class="usa-link" href="#SD8">Supplementary Table 2</a> for NemaFlex measurements.</p></figcaption></figure><p id="P13">In <em>C. elegans</em>, the pharynx is a neuromuscular pump required for bringing bacterial food into the worm and grinding it up before its entry into the intestine. <em>unc-89(ad539)</em> was first isolated in a screen for mutants that show defective feeding behavior, and found to be an allele of <em>unc-89</em> by complementation testing and 3 factor mapping; by polarized light microscopy the pharyngeal muscle was shown to have reduced birefringence and abnormal arcs in the terminal or posterior bulb [<a aria-describedby="R35" class="usa-link" href="#R35">35</a>]. Based on this, and the similarity of <em>ad539</em> with <em>syb797</em>, we examined the pharyngeal muscle structure of both these alleles, and additional alleles of <em>unc-89</em> using polarized light microscopy (<a class="usa-link" href="#F9">Fig. 9</a>). As first noted by Waterston et al. [<a aria-describedby="R1" class="usa-link" href="#R1">1</a>], in <em>unc-89(e1460)</em>, instead of the radially oriented filaments observed in the posterior bulb of wild type, this mutant has longitudinally oriented filaments at the periphery of the bulb (indicated by yellow arrow in <a class="usa-link" href="#F9">Fig. 9</a>). In fact, all the <em>unc-89</em> mutants that we imaged show this defect except for <em>tm752</em>, which shows normal pharyngeal muscle organization. Remarkably, <em>syb797</em>, with the HA tag, shows the same defect. However, <em>ad539</em> shows not only the defect in the terminal bulb; it also shows a defect in the anterior bulb (blue arrows in <a class="usa-link" href="#F9">Fig. 9</a>). The only other <em>unc-89</em> allele that shows both bulbs affected is <em>su75</em>.</p>
<figure class="fig xbox font-sm" id="F9"><h3 class="obj_head">Fig. 9. Polarized light images of the myofilament lattice in the pharynges of wild type and various <em>unc-89</em> mutant alleles.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 9." class="graphic" height="2100" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6219/9549344/fb38b518ad75/nihms-1838736-f0009.jpg" width="952"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F9/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P58">Yellow arrows indicate mis-localized arc-like thick filaments at the periphery of the terminal bulb, and blue arrows indicate mis-localized arc-like thick filaments at the periphery of the anterior bulb. Note that <em>unc-89(tm752)</em> shows pharyngeal muscle like wild type, whereas all the other <em>unc-89</em> mutants show defective organization. <em>unc-89(su75)</em> and <em>unc-89(ad539)</em> are the only alleles that show defects in both the posterior and anterior bulbs. Scale bar, 50 μm.</p></figcaption></figure><p id="P14">To understand the pharyngeal muscle defect further, we examined the expression of UNC-89 protein in the pharynges of wild type, <em>syb797</em> and <em>ad539</em>. This was done by co-immunostaining worms with antibodies to UNC-89 and to the pharyngeal myosin heavy chain MHC C [<a aria-describedby="R36" class="usa-link" href="#R36">36</a>]. We have previously demonstrated that in wild type, UNC-89 is localized to the middle of thick filaments in pharyngeal muscle [<a aria-describedby="R4" class="usa-link" href="#R4">4</a>]. As shown in <a class="usa-link" href="#F10">Fig. 10</a>, UNC-89 is localized to the middle of thick filaments throughout the pharynx. However, in <em>syb797</em> there is no detectable expression of UNC-89 in the posterior bulb. In <em>ad539</em> there is no observable expression of UNC-89 throughout the pharynx. Moreover, MHC C staining reveals that in both <em>syb797</em> and <em>ad539</em>, there are abnormal accumulations of thick filaments at the periphery of the posterior bulb (indicated by yellow arrows in <a class="usa-link" href="#F10">Fig. 10</a>), corresponding to the arcs observed by polarized light. The expression defects in the mutants seem to correspond with the defects in pharyngeal muscle organization and indicate that UNC-89 is required for normal organization of sarcomeres in pharyngeal muscle.</p>
<figure class="fig xbox font-sm" id="F10"><h3 class="obj_head">Fig. 10. <em>unc-89(syb797)</em> and <em>unc-89(ad539)</em> have reduced levels of expression of UNC-89 in pharyngeal muscle.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 10." class="graphic" height="1189" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6219/9549344/fe6b343058e7/nihms-1838736-f0010.jpg" width="2100"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F10/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P59">The indicated strains were immunostained with antibodies to UNC-89 and the pharyngeal-specific myosin heavy chain C (MHC C), and representative confocal images of the pharynx are shown. Note that <em>unc-89(syb797)</em> shows reduced UNC-89 expression in the terminal bulb, whereas <em>unc-89(ad539)</em> shows reduced UNC-89 expression in both the terminal and anterior bulbs. Green arrowhead, anterior bulb; red arrowhead, posterior bulb; yellow arrow, arc-like thick filaments mis-placed at the periphery of the posterior bulb. Scale bar, 10 μm.</p></figcaption></figure></section><section id="S3"><h2 class="pmc_sec_title">Discussion</h2>
<p id="P15">In this study, we explored the biophysical properties and <em>in vivo</em> function of an example of ~600 aa sequence that has similar features (low sequence complexity, lack of predicted domains and minimal levels of predicted secondary structure elements) and location between two protein kinase domains in the UNC-89/obscurin class of giant muscle proteins. In the case of UNC-89 IK, which was studied here, as shown in <a class="usa-link" href="#F1">Fig. 1B</a>, JPred predicts only short stretches of α-helices, primarily in the C-terminal third of the IK, and the majority of the sequence is predicted to be random coil. By use of single molecule force spectroscopy <em>in vitro</em> (<a class="usa-link" href="#F2">Fig. 2</a>) we showed that this IK sequence, is likely to be a random coil since it acts like an entropic spring upon application of pulling force. The nature of the IK sequence (random coil) and its physical properties (entropic spring) is shared with the highly elastic spring elements (PEVK, N2B) of another giant muscle protein, titin [<a aria-describedby="R37" class="usa-link" href="#R37">37</a>]. In the case of titin, there is good evidence to suggest that these elements unwind when muscle sarcomeres are stretched beyond their relaxed state, store potential energy and then provide a passive restoring force to allow the sarcomeres to return to their resting (relaxed) lengths [<a aria-describedby="R38" class="usa-link" href="#R38">38</a>].</p>
<p id="P16">To test the secondary structure prediction and have a better understanding of the response of IK to pulling force, we conducted Far-UV circular dichroism on I27-I27-IK(571)-I27-I27, and as a control protein, a polyprotein containing 9 identical repeats of the I27 domain (I27<sub>9</sub> polyprotein). As <a class="usa-link" href="#SD6">Supplemental Figure 6</a> shows the overall secondary structure of the I27 polyprotein (dotted line) is similar to published data on I27 polyproteins [<a aria-describedby="R39" class="usa-link" href="#R39">39</a>] showing maximum around 200 nm a minimum at ~212nm, a characteristic of a mainly β-strand structure. The overall CD spectra for the IK-I27 construct (solid line) shows a shift of the minimum to a lower wavelength (~208nm) and no clear maximum ~200 nm indicating a reduced β-strand structure and an additional random-coil component. Hence, these structural data are consistent with our AFM and JPred results and demonstrates that the interkinase region displays the properties of a random coil.</p>
<p id="P17">To explore the muscle function of the IK, we used CRISPR/Cas9 to create a worm strain, <em>syb797syb1257</em>, in which most of the IK of UNC-89 was deleted in-frame. This strain also has its giant UNC-89 isoforms N-terminally tagged with HA. <em>syb797syb1257</em> also seems to show reduced expression of the UNC-89 giant isoforms similar to that shown by the HA tagged strain, <em>syb797</em> (an example is shown in <a class="usa-link" href="#F5">Fig. 5B</a>). However, in contrast to <em>syb797</em> which shows normal body wall muscle sarcomere organization, <em>syb797syb1257</em>, shows highly disorganized body wall muscle sarcomeres. Therefore, the body wall muscle sarcomere defect in <em>syb797syb1257</em> is likely caused by the lack of the IK.</p>
<p id="P18">The normal location of UNC-89 is the M-line, the structure in which thick filaments are cross linked in the middle of the A-band. Deletion of the IK, in addition to disrupting the normal continuous and parallel M-lines or A-bands, results in formation of short A-bands which are not full-depth but confined to the portion nearest the outer muscle cell membrane, and these occur between normally spaced A-bands. Sarcomere assembly is known to occur from the outside inwards, first with the deposition of ECM proteins like perlecan, followed by localization of integrins, and then localization of integrin adhesion proteins (UNC-112 (kindlin), PAT-4 (ILK), UNC-97 (PINCH), PAT-6 (α-parvin), followed by assembly of M-line or dense body (Z-disk)-specific proteins [<a aria-describedby="R40" class="usa-link" href="#R40">40</a>,<a aria-describedby="R41" class="usa-link" href="#R41">41</a>]. We know that UNC-89 interacts with CPNA-1 [<a aria-describedby="R11" class="usa-link" href="#R11">11</a>], which in turn interacts with PAT-6, a member of the 4- protein complex that interacts with the cytoplasmic tail of PAT-3 (β-integrin)[<a aria-describedby="R42" class="usa-link" href="#R42">42</a>]. It should be noted that CPNA-1 interacts with Ig1-Ig3, which is near the N-terminus of UNC-89 and remote from IK. Our results seem to indicate that these protein interactions at the membrane are not sufficient for full assembly of a M-line/A-band, and the proper spacing between A-bands. Either the length or the elasticity of UNC-89 IK is required. In future experiments, we can determine the minimum length of IK that is sufficient for full A-band assembly, and whether elasticity is crucial. We could replace the elastic IK with a sequence of similar length that is rigid. An example of a rigid sequence that could be used is the series of 4 consecutive Ig domains in human titin that are not separated by linker sequences [<a aria-describedby="R43" class="usa-link" href="#R43">43</a>]. It is tempting to speculate that the length of the interkinase is important for somehow determining the spacing between adjacent A-bands. Another possibility is that deletion of the IK in some way interferes with the activity of the active protein kinase domain of UNC-89 PK2. This is not likely because inactivation of PK2 does not result in a defect in sarcomere organization (Matsunaga et al., manuscript in preparation). So far, we have only examined the organization of body wall muscle sarcomeres in adult muscle. Therefore, we cannot distinguish a role for UNC-89 IK for sarcomere assembly vs. sarcomere stability. Indeed, it is likely that UNC-89 has more of a role in maintaining or stabilizing sarcomere structure: In strong loss of function alleles like <em>unc-89(e1460)</em>, A-band organization is normal in early larvae [<a aria-describedby="R22" class="usa-link" href="#R22">22</a>], despite the fact that UNC-89 is expressed even in embryonic muscle [<a aria-describedby="R44" class="usa-link" href="#R44">44</a>].</p>
<p id="P19">In vertebrate muscle, the M-band is known to be elastic and helps titin maintain the registration of adjacent thick filaments during muscle contraction [<a aria-describedby="R45" class="usa-link" href="#R45">45</a>]. The cross-linking of thick filaments occurs by the binding of antiparallel dimers of myomesin to myosin of thick filaments. The M-band also includes complex interactions between Obsl1 and obscurin with titin, and myomesin with obscurin [<a aria-describedby="R46" class="usa-link" href="#R46">46</a>]. Although much of the elasticity and mechanical protection is provided by myomesin [<a aria-describedby="R47" class="usa-link" href="#R47">47</a>,<a aria-describedby="R48" class="usa-link" href="#R48">48</a>], our results suggest that the IK region of obscurin, which also lies in the M-band also provides some of this elasticity.</p>
<p id="P20">Because whole animal locomotion (swimming, crawling) and force generation are diminished upon removal of the IK, we also have learned that the level of sarcomere organization found in this mutant is not sufficient for proper contractile activity. In contrast, the better level of sarcomere organization in <em>unc-89</em>(<em>tm752)</em> is sufficient to generate normal force but is not sufficient to generate normal swimming or crawling speeds.</p>
<p id="P21">Our IK deletion strain also contains an HA tag appended to the N-terminus of the large UNC-89 isoforms. We designed this for future immunoprecipitation experiments to further test whether UNC-89 interacting partners that we have discovered primarily through two hybrid screens are actually associated with UNC-89: we would create in-frame deletions of segments of UNC-89 that are suspected to bind to a partner, then IP those internally deleted UNC-89 proteins, and look for absence of the association. Somewhat surprisingly, this HA tag reduces the level of expression of the giant UNC-89 isoforms to approximately half of their wild type levels in whole worm extracts. And although the HA tag does not affect the organization of sarcomeres in body wall muscle or the ability of whole animals to swim or crawl, it does reduce the ability to generate muscle force. In addition, the HA tag results in a defect in thick filaments in the terminal bulb of the pharynx. Moreover, the level of expression of UNC-89 is reduced in the terminal bulb, consistent with the western blot result. This suggests that the HA tag at the N-terminus either reduces the stability of pharyngeal-specific UNC-89 mRNA or protein isoforms, or interferes with interaction with binding partners found only in pharyngeal muscle. Although our extensive yeast two-hybrid screens did not reveal any binding partners for the N-terminus of UNC-89, these screens cannot reveal all relevant binding partners. Also, it is possible that the HA tag reduces the steady state levels of the giant UNC-89 isoforms in both pharyngeal and body wall muscle, below a critical level required for pharyngeal but not for body wall muscle sarcomere assembly and/or stability.</p>
<p id="P22">The reduced force generated by <em>syb797</em> containing the HA tag is also puzzling. However, it should be noted that <em>syb797</em> and <em>ad539</em> share these properties—each has normal body wall muscle, but defective pharyngeal muscle. Then, why do these two mutants generate less force? In fact, the only mutant studied that generates normal force is <em>tm752</em> that does so despite having mildly disorganized body wall muscle. The key to explaining why these two mutants generate less force appears to be their reduced body diameters. In fact, as shown in <a class="usa-link" href="#T1">Table 1</a>, all the mutants generating less force have smaller than wild type body diameters, and defective pharynxes. <em>tm752</em> has a normal body diameter and a normal pharynx. Our speculation is that all the mutants with defective pharynxes have reduced nutrition and consequently are smaller. Although our force measurements were normalized for body diameter, it is possible that reduced nutrition results in reduced metabolism and reduced energy production, and thus less ability to generate force. It is also curious that both <em>syb797</em> and <em>ad539</em> display greater swimming speeds and <em>ad539</em> shows greater crawling speed. Perhaps this is because these thinner animals can bend more easily.</p>
<section class="tw xbox font-sm" id="T1"><h3 class="obj_head">Table 1.</h3>
<div class="caption p"><p id="P60">The affects of <em>unc–89</em> mutations on expression of UNC–89 isoforms, body wall and pharyngeal muscle structure, locomotion, body wall muscle force and body diameter.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="box" rules="all">
<colgroup span="1">
<col align="left" span="1" valign="middle"/>
<col align="left" span="1" valign="middle"/>
<col align="left" span="1" valign="middle"/>
<col align="left" span="1" valign="middle"/>
<col align="left" span="1" valign="middle"/>
<col align="left" span="1" valign="middle"/>
<col align="left" span="1" valign="middle"/>
<col align="left" span="1" valign="middle"/>
<col align="left" span="1" valign="middle"/>
</colgroup>
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="top">Strain</th>
<th align="left" colspan="1" rowspan="1" valign="top">Mutation</th>
<th align="left" colspan="1" rowspan="1" valign="top">Protein Expression</th>
<th align="left" colspan="1" rowspan="1" valign="top">Body Wall Muscle Structure</th>
<th align="left" colspan="1" rowspan="1" valign="top">Swimming</th>
<th align="left" colspan="1" rowspan="1" valign="top">Crawling</th>
<th align="left" colspan="1" rowspan="1" valign="top">Force by NemaFlex</th>
<th align="left" colspan="1" rowspan="1" valign="top">Body Diameter (pm)</th>
<th align="left" colspan="1" rowspan="1" valign="top">Pharyngeal Muscle</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">wild type</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">73.5–78.3</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>syb797</em>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">HA tag at N–term. of giant isoforms</td>
<td align="left" colspan="1" rowspan="1" valign="top">giant isoforms reduced by ∼50% reduced in pharynx</td>
<td align="left" colspan="1" rowspan="1" valign="top">normal</td>
<td align="left" colspan="1" rowspan="1" valign="top">faster</td>
<td align="left" colspan="1" rowspan="1" valign="top">normal</td>
<td align="left" colspan="1" rowspan="1" valign="top">reduced</td>
<td align="center" colspan="1" rowspan="1" valign="top">62.1</td>
<td align="left" colspan="1" rowspan="1" valign="top">defective posterior bulb</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>e1460</em>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">nonsense mutation in alternatively spliced exon for Ig20–Ig26 [<a aria-describedby="R22" class="usa-link" href="#R22">22</a>]</td>
<td align="left" colspan="1" rowspan="1" valign="top">reduced levels of giant isoforms [<a aria-describedby="R12" class="usa-link" href="#R12">12</a>]</td>
<td align="left" colspan="1" rowspan="1" valign="top">moderate [<a aria-describedby="R12" class="usa-link" href="#R12">12</a>]</td>
<td align="left" colspan="1" rowspan="1" valign="top">slower</td>
<td align="left" colspan="1" rowspan="1" valign="top">slower (severe)</td>
<td align="left" colspan="1" rowspan="1" valign="top">reduced (severe)</td>
<td align="center" colspan="1" rowspan="1" valign="top">57.3</td>
<td align="left" colspan="1" rowspan="1" valign="top">defective posterior bulb</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>syb797syb1257</em>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">HA tag added to N–term. of giant isoforms &amp; in–frame deletion of IK</td>
<td align="left" colspan="1" rowspan="1" valign="top">giant isoforms reduced by ∼50% reduced in pharynx</td>
<td align="left" colspan="1" rowspan="1" valign="top">moderate–severe</td>
<td align="left" colspan="1" rowspan="1" valign="top">slower</td>
<td align="left" colspan="1" rowspan="1" valign="top">slower</td>
<td align="left" colspan="1" rowspan="1" valign="top">reduced</td>
<td align="center" colspan="1" rowspan="1" valign="top">55.7</td>
<td align="left" colspan="1" rowspan="1" valign="top">defective posterior bulb</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>su75</em>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">1 bp insertion in coding for Ig38 resulting in frameshift premature stop [<a aria-describedby="R12" class="usa-link" href="#R12">12</a>]</td>
<td align="left" colspan="1" rowspan="1" valign="top">all giant isoforms missing small kinase isoforms present [<a aria-describedby="R3" class="usa-link" href="#R3">3</a>,<a aria-describedby="R12" class="usa-link" href="#R12">12</a>]</td>
<td align="left" colspan="1" rowspan="1" valign="top">severe [<a aria-describedby="R12" class="usa-link" href="#R12">12</a>]</td>
<td align="left" colspan="1" rowspan="1" valign="top">slower (severe)</td>
<td align="left" colspan="1" rowspan="1" valign="top">slower (severe)</td>
<td align="left" colspan="1" rowspan="1" valign="top">reduced (severe)</td>
<td align="center" colspan="1" rowspan="1" valign="top">49.9</td>
<td align="left" colspan="1" rowspan="1" valign="top">defective posterior &amp; anterior bulbs</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>tm752</em>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">out of frame deletion in IK [<a aria-describedby="R5" class="usa-link" href="#R5">5</a>]</td>
<td align="left" colspan="1" rowspan="1" valign="top">all kinase isoforms (giant &amp; small) missing [<a aria-describedby="R5" class="usa-link" href="#R5">5</a>]</td>
<td align="left" colspan="1" rowspan="1" valign="top">mild [<a aria-describedby="R12" class="usa-link" href="#R12">12</a>]</td>
<td align="left" colspan="1" rowspan="1" valign="top">slower (severe)</td>
<td align="left" colspan="1" rowspan="1" valign="top">slower (severe)</td>
<td align="left" colspan="1" rowspan="1" valign="top">normal</td>
<td align="center" colspan="1" rowspan="1" valign="top">75.9</td>
<td align="left" colspan="1" rowspan="1" valign="top">normal</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>ad539</em>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ala to Val change in KSP region but this may not be causative</td>
<td align="left" colspan="1" rowspan="1" valign="top">giant isoforms normal by western none detectable in pharynx</td>
<td align="left" colspan="1" rowspan="1" valign="top">normal</td>
<td align="left" colspan="1" rowspan="1" valign="top">faster</td>
<td align="left" colspan="1" rowspan="1" valign="top">faster</td>
<td align="left" colspan="1" rowspan="1" valign="top">reduced</td>
<td align="center" colspan="1" rowspan="1" valign="top">56.2</td>
<td align="left" colspan="1" rowspan="1" valign="top">defective posterior &amp; anterior bulbs</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></section><p id="P23">Finally, it should be pointed out that the <em>C. elegans</em> pharynx is often considered analogous to the vertebrate heart, although the two organs are likely to result from convergent evolution between two muscular pumps tasked with similar roles [<a aria-describedby="R49" class="usa-link" href="#R49">49</a>]. Thus, in the future, cardiomyopathy-causing mutations in human obscurin might be studied in <em>C. elegans</em> by creating the analogous mutations in <em>unc-89</em>, and using these mutants for genetic suppressor and drug screens.</p></section><section id="S4"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="S5"><h3 class="pmc_sec_title">Recombinant protein production</h3>
<p id="P24">A cDNA encoding a 571 amino acid portion of the interkinase region of UNC-89, IYDYLRIQ…ADIEKVTP (residues 6945—7515 in UNC-89b) was synthesized using E. coli optimal codons and cloned into pUC57 by GenScript USA,. Inc. (Piscataway, New Jersey). This plasmid was digested with SpeI and SacI, and the insert was cloned into “I27 plasmid shuffled codons (003)”, in order to express a fusion protein consisting of 6xHis-I27-I27-UNC-89 IK 571 aa—I27-I27. (I27 is the human titin Ig27 domain.) The resulting plasmid was transformed into E. coli BL21 (DE3) and the fusion protein was expressed at 20° for 5 hours after induction with 750 mM Isopropyl β-D-1-thiogalactopyranoside (IPTG). The 6xHis tagged fusion protein was purified with Ni-NTA Agarose (Qiagen), as described in [<a aria-describedby="R13" class="usa-link" href="#R13">13</a>]. The buffer of the eluted protein was exchanged with PBS containing 10% glycerol.</p></section><section id="S6"><h3 class="pmc_sec_title">Single-molecule force spectroscopy (SMFS) experiments</h3>
<p id="P25">Pulling experiments using SMFS were carried out on a home-built single-molecule Atomic Force Spectrometer as previously described in the literature [<a aria-describedby="R50" class="usa-link" href="#R50">50</a>,<a aria-describedby="R51" class="usa-link" href="#R51">51</a>]. In each experiment, we deposited 10 μL of the UNC-89 IK domain polyprotein solution (0.5 mg/mL) in PBS (pH 7.4) onto a Ni<sup>2+</sup>-NTA functionalized glass coverslip [<a aria-describedby="R52" class="usa-link" href="#R52">52</a>] covered by 50 μL of PBS. We allowed the protein to adsorb onto the substrate for 10 minutes before pulling. The spring constant of each individual cantilever (MLCT, Bruker AFM Probes, Camarillo, CA) was determined experimentally by measuring the cantilever power spectrum and using the equipartition theorem [<a aria-describedby="R53" class="usa-link" href="#R53">53</a>,<a aria-describedby="R54" class="usa-link" href="#R54">54</a>] at the beginning of each experiment. All measurements were done at a constant velocity of 500 nm/s in PBS solution at room temperature. A worm-like chain (WLC) model [<a aria-describedby="R55" class="usa-link" href="#R55">55</a>] with a variable persistence length was fit to the first I27 peak to measure the contour-length (Lc) of the IK domain in the force extension data [<a aria-describedby="R56" class="usa-link" href="#R56">56</a>].</p></section><section id="S7"><h3 class="pmc_sec_title">Circular Dichroism</h3>
<p id="P26">The far UV CD spectra of the protein constructs were recorded on a Jasco J-815 Spectrometer. The protein concentration was 1 μM in 30 mM phosphate buffer pH 7.4, 100 mM KCl, 1 mM MgCl<sub>2</sub>, 1mM TCEP buffer. A 0.1 cm path length cuvette was used.</p></section><section id="S8"><h3 class="pmc_sec_title">Generation of CRISPR/Cas9 edited nematodes</h3>
<p id="P27">Strains PHX797 and PHX1257 were generated by SunyBiotech (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.sunybiotech.com/" rel="noopener noreferrer" target="_blank">http://www.sunybiotech.com</a>) using CRISPR/Cas9 technology in the following ways. PHX797, <em>unc-89(syb797)</em>, was created from wild type strain N2 to contain an HA tag at the N-terminus of the large UNC-89 isoforms, using sgRNAs with the following sequences: sg1: ccatcATGGCTAGTCGACGCCAAAAG, and sg2: cccaccttacccttaccatcATGG. PHX1257 <em>unc-89(syb797syb1257)</em> was created from PHX797 to contain a 4,614 bp in-frame deletion of the interkinase region, using sgRNAs with the following sequences: Sg1:CCTGTAGCACCG<strong>G</strong>AAGGTCGACG, Sg2:CCACCACCGACTGTGGAATACGT, Sg3:AACAGCAATTGAGAGAATTAAGG and Sg4:<span class="text-underline"><strong>CC</strong></span>ACAAAGAAGAATGATGATGGA. Sanger sequencing was used to verify the sequence alterations of each strain (<a class="usa-link" href="#SD3">Supplemental Figures 3</a> and <a class="usa-link" href="#SD4">4</a>). Both strains were outcrossed 5 times to wild type to remove most of the possible off target effects of CRISPR/Cas9; PHX797 to GB312, and PHX1257 to GB313.</p></section><section id="S9"><h3 class="pmc_sec_title">Antibodies and western blots</h3>
<p id="P28">We used the procedure of Hannak et al. [<a aria-describedby="R57" class="usa-link" href="#R57">57</a>] to prepare total protein lysates from wild type, <em>unc-89(syb797)</em> and <em>unc-89(syb797syb1257)</em> animals. Equal amounts of total protein from these strains were separated by a 5% separating and 3% stacking Laemmli SDS-PAGE gel, transferred to nitrocellulose membrane for 2 hours and reacted with anti-UNC-89 EU30 affinity-purified rabbit polyclonal antibodies [<a aria-describedby="R4" class="usa-link" href="#R4">4</a>] at 1:100 dilution, or with anti-HA antibodies (rabbit monoclonal antibody C29F4 from Cell Signaling Technology) at 1:100 dilution, followed by reaction with anti-mouse Ig-horseradish peroxidase, and ECL. For the EU30 antibody we used Pierce ECL Western Blotting Substrate (cat. no. 32106 from Thermo Scientific), but for the anti-HA antibody we needed the increased sensitivity of SuperSignal West Pico Chemiluminescent Substrate (cat. no. 34080, from Thermo Scientific). We found that using freshly prepared or freshly purchased acrylamide solutions (e.g. ProtoGel, cat. No. EC-890 from National Diagnostics) was absolutely essential to prevent horrible streaking of the protein bands.</p></section><section id="S10"><h3 class="pmc_sec_title">Immunofluorescence microscopy</h3>
<p id="P29">Adult worms were fixed as described previously [<a aria-describedby="R58" class="usa-link" href="#R58">58</a>,<a aria-describedby="R59" class="usa-link" href="#R59">59</a>]. For body wall muscle staining, these primary antibodies were used at the following dilutions: anti-myosin heavy chain A (MHC A; mouse monoclonal 5–6; [<a aria-describedby="R31" class="usa-link" href="#R31">31</a>]) at 1:200; anti-UNC-89 (mouse monoclonal MH42 [<a aria-describedby="R4" class="usa-link" href="#R4">4</a>, <a aria-describedby="R44" class="usa-link" href="#R44">44</a>]) at 1:100; anti-UNC-95 (rabbit polyclonal Benian-13; [<a aria-describedby="R60" class="usa-link" href="#R60">60</a>]) at 1:200; and anti-ATN-1 (α-actinin; MH35; [<a aria-describedby="R61" class="usa-link" href="#R61">61</a>]) at 1:200. For pharyngeal muscle staining, the fixed worms were incubated with anti-UNC-89 (rabbit polyclonal EU30; [<a aria-describedby="R4" class="usa-link" href="#R4">4</a>]) at 1:100, and anti-myosin heavy chain C (MHC C; mouse monoclonal 9.2.1; [<a aria-describedby="R36" class="usa-link" href="#R36">36</a>]), at 1:50. MH35 antibodies were kindly provided by Dr. Pamela Hoppe (Western Michigan University). 5–6 and 9.2.1 antibodies were obtained from the Developmental Studies Hybridoma Bank, created by the NICHD of the NIH and maintained at The University of Iowa, Department of Biology, Iowa City, IA 52242. The secondary antibodies included anti-mouse IgG-Alexa 594 (Invitrogen A11032), and anti-rabbit IgG Alexa 488 (Invitrogen A21206), each at 1:200 dilution. Images shown in <a class="usa-link" href="#F3">Figures 3</a>–<a class="usa-link" href="#F6">6</a>, and the <a class="usa-link" href="#SD5">Supplemental Figure 5</a> were captured at room temperature with a Zeiss confocal system (LSM510) equipped with an Axiovert 100M microscope and an Apochromat ×63/1.4 numerical aperture oil objective. The color balances of the images were adjusted by using Adobe Photoshop. We examined the staining of at least 3 worms in each strain.</p>
<p id="P30">Super-resolution microscopy was performed with a Nikon N-SIM system in 3D structured illumination mode on an Eclipse Ti-E microscope equipped with a 100×/1.49 NA oil immersion objective, 488- and 561-nm solid-state lasers, and an EM-CCD camera (DU-897, Andor Technology). The color balances of the images were adjusted by using Adobe Photoshop (Adobe, San Jose, CA). A Z-series at 0.2 μm intervals was taken from the outer muscle cell membrane deeper into the muscle cell, as shown in <a class="usa-link" href="#F7">Fig. 7</a>.</p></section><section id="S11"><h3 class="pmc_sec_title">Nematode locomotion assays</h3>
<section id="S12"><h4 class="pmc_sec_title">Swimming Assay</h4>
<p id="P31">Day 2 adults from two, 6 cm NGM OP50 seeded plates were washed off the plates, washed free from bacteria and collected into M9 buffer such that the ratio of worms to buffer was 1:1. Two ml of M9 buffer was added to an unseeded 6 cm NGM plate where upon 5 μl of worm suspension was added to the center of the plate. Worms were allowed 5 minutes to adapt before a video recording of their swimming motions was made using a dissecting stereoscopic microscope fitted with a CMOS camera (Thorlabs). Ten, 10 second videos were recorded for each nematode strain from different sections of the plate, each video tracking the motion of ~10 worms. The video data was analyzed by Image J FIJI WrmTracker software [<a aria-describedby="R62" class="usa-link" href="#R62">62</a>] to obtain body bends per second (BBPS) from a total of 48–83 individual worms for each strain. The resulting worm tracks for each animal were observed, with immobilized animals in addition to upper and lower range outliers being removed from BBPS values for each strain prior to the statistical analysis. The final values for each strain were compared to wild type and tested for statistical significance using a Student T-test.</p></section><section id="S13"><h4 class="pmc_sec_title">Crawling Assay</h4>
<p id="P32">Day 2 adults were harvested as above, except that all washing steps used M9 buffer containing 0.2 g/L gelatin. 5 μl of worm suspension was added to the center of a 6 cm unseeded NGM plate, and the excess liquid was removed. After 5 minutes for adaptation, worm crawling was recorded using the above-mentioned strategy for extraction of body bends per second (BBPS) from a total of 66–90 individual worms for each strain. The resulting values for each strain were compared to wild type and tested for statistical significance using a Student T-test.</p></section></section><section id="S14"><h3 class="pmc_sec_title">Measurement of <em>C. elegans</em> muscle force using NemaFlex</h3>
<p id="P33">The maximum exertable muscle force by <em>C. elegans</em> strains was measured using the NemaFlex technique as previously described [<a aria-describedby="R34" class="usa-link" href="#R34">34</a>]. The technique is based on deflection of soft micropillars as the animals are crawling through the micropillar arena. The micropillar devices had pillars arranged in a square lattice with a pillar diameter of 44 μm and height of 87 μm. The gap between the pillars is 71 μm. Age synchronized day 1 adult animals were loaded individually in each chamber [<a aria-describedby="R63" class="usa-link" href="#R63">63</a>], and a 1-minute video was captured for each animal in a food-free environment at a temperature of 20 ± 1 °C. Imaging was performed in brightfield using a Nikon Ti-E microscope and Andor Zyla sCMOS 5.5 camera. Images were acquired at 5 frames per second with a 4x objective at a pixel resolution of 1.63 μm per pixel. Movies were then processed and analyzed for force values using our in-house-built image processing software (MATLAB, R2016a). Animal force measurements were calculated by identifying the maximal force exerted in each frame for an individual animal. We define the maximum exertable force <em>f</em><sub>95</sub> corresponding to the 95<sup>th</sup> percentile of these maximal forces. To account for differences in animal body diameter, we normalize <em>f</em><sub>95</sub> by the cube of animal body diameter [<a aria-describedby="R34" class="usa-link" href="#R34">34</a>]. To compare muscle forces of mutants with wild-type animals, we take the ratio of this normalized maximum exertable force and denote it as fold change in muscle force. Statistical analysis was performed using Wilcoxon rank-sum test.</p></section><section id="S15"><h3 class="pmc_sec_title">Polarized light microscopy</h3>
<p id="P34">Polarized light images were captured with a Zeiss Axioskop microscope equipped with 20X/0.40 Pol strain-free objective and an AxioCam MRm digital camera and software (Carl Zeiss, Jena, Germany).</p></section></section><section id="SM1"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="SD1"><div class="caption p">
<span>Supp. Fig. 1</span><p id="P35">Supplemental Fig. 1. Sequence alignment of IK regions of UNC-89, obscurin and SPEG using ClustalW. (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.genome.jp/tools-bin/clustalw" rel="noopener noreferrer" target="_blank">https://www.genome.jp/tools-bin/clustalw</a>).</p>
</div>
<div class="media p" id="d64e987"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Fig__1.pdf">NIHMS1838736-supplement-Supp__Fig__1.pdf</a><sup> (55.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD2"><div class="caption p">
<span>Supp. Fig. 2</span><p id="P36">Supplemental Fig. 2. Secondary structure predictions of IK regions of UNC-89, obscurin and SPEG using JPred. (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.compbio.dundee.ac.uk/jpred/index.html" rel="noopener noreferrer" target="_blank">http://www.compbio.dundee.ac.uk/jpred/index.html</a>)</p>
</div>
<div class="media p" id="d64e996"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Fig__2.pdf">NIHMS1838736-supplement-Supp__Fig__2.pdf</a><sup> (72.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD3"><div class="caption p">
<span>Supp. Fig. 3</span><p id="P37">Supplemental Fig. 3. Results of Sanger sequencing and conceptual translation of PHX797, <em>unc-89(syb797)</em>.</p>
</div>
<div class="media p" id="d64e1005"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Fig__3.pdf">NIHMS1838736-supplement-Supp__Fig__3.pdf</a><sup> (38.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD4"><div class="caption p">
<span>Supp. Fig. 4</span><p id="P38">Supplemental Fig. 4. Results of Sanger sequencing and conceptual translation across genomic deletion in PHX1257, <em>unc-89(syb797syb1257)</em>.</p>
</div>
<div class="media p" id="d64e1014"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Fig__4.pdf">NIHMS1838736-supplement-Supp__Fig__4.pdf</a><sup> (116.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD5"><div class="caption p">
<span>Supp. Fig. 5</span><p id="P39">Supplementary Fig. 5. The body wall muscle of <em>unc-89(ad539)</em> has normally organized sarcomeres. Each panel shows representative confocal images of several body wall muscle cells stained with antibodies to the indicated sarcomeric components or with rhodamine-phalloidin for F-actin. Scale bar, 10 μm.</p>
</div>
<div class="media p" id="d64e1023"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Fig__5.tif">NIHMS1838736-supplement-Supp__Fig__5.tif</a><sup> (4.9MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD6"><div class="caption p">
<span>Supp. Fig. 6</span><p id="P40">Supplementary Figure 6. Secondary structure of the I27<sub>2</sub>-IK-I27<sub>2</sub> protein and a control I27<sub>9</sub> polyprotein determined by circular dichroism. (A) The overall secondary structure determined by Far-UV circular dichroism of the I27<sub>2</sub>-IK-I27<sub>2</sub> used in AFM experiments (solid line) and a control protein containing 9 identical repeats of the I27 domain (dotted line). The protein concentration was 1 μM in 30 mM phosphate buffer pH 7.4, 100 mM KCl, 1 mM MgCl<sub>2</sub>, 1mM TCEP buffer. A 0.1 cm path length cuvette was used. (B) The % of β-strand, α-helical and random-coil content was estimated by using the BeStSel tool [<a aria-describedby="R64" class="usa-link" href="#R64">64</a>] and these are: 16% β-strand, 29% α-helix, 55% random for the IK construct (box on the right) and 40% β-strand, 20% α-helix, 36% random-coil for the I27 polyprotein (box on the left).</p>
</div>
<div class="media p" id="d64e1045"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Fig__6.tif">NIHMS1838736-supplement-Supp__Fig__6.tif</a><sup> (2.3MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD7"><div class="caption p">
<span>Supp. Table 1</span><p id="P41">Supplementary Table 1. Detailed statistics for swimming and crawling assays on the different strains used in this study and summarized in <a class="usa-link" href="#F6">Fig. 6A</a> and <a class="usa-link" href="#F6">B</a>.</p>
</div>
<div class="media p" id="d64e1057"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Table_1.tiff">NIHMS1838736-supplement-Supp__Table_1.tiff</a><sup> (724.7KB, tiff) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD8"><div class="caption p">
<span>Supp. Table 2</span><p id="P42">Supplementary Table 2: Muscle force and body diameter data for the different strains used in this study and summarized in <a class="usa-link" href="#F6">Fig. 6C</a>.</p>
</div>
<div class="media p" id="d64e1066"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Table_2.tiff">NIHMS1838736-supplement-Supp__Table_2.tiff</a><sup> (747.8KB, tiff) </sup>
</div></div></section></section><section class="floats-group" id="floats-group1"><section class="bt xbox font-sm" id="BX1"><h2 class="obj_head">Highlights.</h2>
<ul class="list" id="L1" style="list-style-type:disc">
<li><p id="P61">By single molecule force spectroscopy, the ~645 residue non-domain region lying between the two protein kinase domains of UNC-89 behaves as an elastic random coil</p></li>
<li><p id="P62">Nematodes lacking this non-domain region have disorganized sarcomeres and reduced whole animal locomotion and force generation</p></li>
<li><p id="P63">UNC-89 non-domain region is required for proper assembly or stability of A-bands from thick filaments</p></li>
</ul></section></section><section class="ack" id="S16"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p id="P43">This study was supported in-part by a grant from the National Institutes of Health (R01 AR064307) to G.M.B. J.C.M. was supported in-part by the National Science Foundation-Graduate Research Fellowship Program (Grant #DGE 1444932). T.M. was partially supported by a grant from the National Institutes of Health (R01GM118534) to A.F.O. and G.M.B. L.L. and S.A.V. were partially supported by the National Aeronautics and Space Administration (Grant # NNX15AL16G).</p></section><section class="glossary" id="glossary1"><h2 class="pmc_sec_title">Abbreviations:</h2>
<dl class="def-list">
<dt>Ig domain</dt>
<dd><p id="P46">immunoglobulin domain</p></dd>
<dt>Fn3 domain</dt>
<dd><p id="P47">fibronectin type 3 domain</p></dd>
<dt>SPEG</dt>
<dd><p id="P48">striated muscle preferentially expressed gene</p></dd>
<dt>HA</dt>
<dd><p id="P49">hemagglutinin</p></dd>
</dl></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="FN1">
<p id="P44">Credit Author Statement</p>
<p id="P45">Hiroshi Qadota conceptualized the project, obtained most of the data, analyzed most of the data, supervised other authors, wrote the original draft and reviewed and edited the manuscript. Jasmine Moody obtained some data, analyzed some data, wrote and edited several sections of the manuscript, and provided resources through her NSF fellowship. Leila Lesanpezeshki obtained some data, analyzed some data, and wrote and edited several sections of the manuscript. Taylor Moncrief obtained some data, analyzed some data, and wrote and edited several sections of the manuscript. Deborah Kitzler obtained some data, analyzed some data, and wrote and edited several sections of the manuscript. Purnima Devaki Bhat obtained some data, analyzed some data, and wrote and edited several sections of the manuscript. Siva A. Vanapalli helped with the conceptualization and data analysis, reviewed and edited the manuscript and provided some resources through existing grants. Andres F. Oberhauser obtained some data, analyzed some data, wrote and edited several sections of the manuscript and also performed overall editing of the manuscript and provided some resources through existing grants. Guy M. Benian was the corresponding author and was involved in the overall conceptualization of the project, obtained some data, was involved in data curation, data analysis, project administration, wrote multiple versions (editing) of the manuscript, and provided most of the resources through existing grants.</p>
</div></div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">[1].</span><cite>Waterston RH, Thomson JN, Brenner S, Mutants with altered muscle structure in C. elegans, Dev. Biol. 77 (1980) 271–302.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0012-1606(80)90475-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/7190524/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Dev.%20Biol&amp;title=Mutants%20with%20altered%20muscle%20structure%20in%20C.%20elegans&amp;author=RH%20Waterston&amp;author=JN%20Thomson&amp;author=S%20Brenner&amp;volume=77&amp;publication_year=1980&amp;pages=271-302&amp;pmid=7190524&amp;doi=10.1016/0012-1606(80)90475-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R2">
<span class="label">[2].</span><cite>Benian GM, Ayme-Southgate A, Tinley TL, The genetics and molecular biology of the titin/connectin-like proteins of invertebrates, Rev. Physio. Biochem. Pharm
138 (1999) 235–268.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/BFb0119629" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10396143/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Rev.%20Physio.%20Biochem.%20Pharm&amp;title=The%20genetics%20and%20molecular%20biology%20of%20the%20titin/connectin-like%20proteins%20of%20invertebrates&amp;author=GM%20Benian&amp;author=A%20Ayme-Southgate&amp;author=TL%20Tinley&amp;volume=138&amp;publication_year=1999&amp;pages=235-268&amp;pmid=10396143&amp;doi=10.1007/BFb0119629&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R3">
<span class="label">[3].</span><cite>Small TM, Gernert KM, Flaherty DB, Mercer KB, Borodovsky M, Benian GM, Three new isoforms of C. elegans UNC-89 containing MLCK-like protein kinase domains, J. Mol. Biol. 342 (2004) 91–108.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jmb.2004.07.006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15313609/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&amp;title=Three%20new%20isoforms%20of%20C.%20elegans%20UNC-89%20containing%20MLCK-like%20protein%20kinase%20domains&amp;author=TM%20Small&amp;author=KM%20Gernert&amp;author=DB%20Flaherty&amp;author=KB%20Mercer&amp;author=M%20Borodovsky&amp;volume=342&amp;publication_year=2004&amp;pages=91-108&amp;pmid=15313609&amp;doi=10.1016/j.jmb.2004.07.006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R4">
<span class="label">[4].</span><cite>Benian GM, Tinley TL, Tang X, Borodovsky M, The Caenorhabditis elegans gene unc-89, required for muscle M-line assembly, encodes a giant modular protein composed of Ig and signal transduction domains, J. Cell Biol. 132 (1996) 835–848.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.132.5.835" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2120741/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8603916/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol&amp;title=The%20Caenorhabditis%20elegans%20gene%20unc-89,%20required%20for%20muscle%20M-line%20assembly,%20encodes%20a%20giant%20modular%20protein%20composed%20of%20Ig%20and%20signal%20transduction%20domains&amp;author=GM%20Benian&amp;author=TL%20Tinley&amp;author=X%20Tang&amp;author=M%20Borodovsky&amp;volume=132&amp;publication_year=1996&amp;pages=835-848&amp;pmid=8603916&amp;doi=10.1083/jcb.132.5.835&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R5">
<span class="label">[5].</span><cite>Ferrara TM, Flaherty DB, Benian GM, Titin / connectin-related proteins in C. elegans: a review and new findings, J. Musc. Res. Cell Motil. 26 (2005) 435–447.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s10974-005-9027-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16453163/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Musc.%20Res.%20Cell%20Motil&amp;title=Titin%20/%20connectin-related%20proteins%20in%20C.%20elegans:%20a%20review%20and%20new%20findings&amp;author=TM%20Ferrara&amp;author=DB%20Flaherty&amp;author=GM%20Benian&amp;volume=26&amp;publication_year=2005&amp;pages=435-447&amp;pmid=16453163&amp;doi=10.1007/s10974-005-9027-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R6">
<span class="label">[6].</span><cite>Ono K, Yu R, Ono S, Structural components of the nonstriated contractile apparatuses in the C. elegans gonadal myoepithelial sheath and their essential roles for ovulation, Dev. Dyn. (2007) 236, 1093–1105.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/dvdy.21091" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1994093/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17326220/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Dev.%20Dyn&amp;title=Structural%20components%20of%20the%20nonstriated%20contractile%20apparatuses%20in%20the%20C.%20elegans%20gonadal%20myoepithelial%20sheath%20and%20their%20essential%20roles%20for%20ovulation&amp;author=K%20Ono&amp;author=R%20Yu&amp;author=S%20Ono&amp;volume=236&amp;publication_year=2007&amp;pages=1093-1105&amp;pmid=17326220&amp;doi=10.1002/dvdy.21091&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R7">
<span class="label">[7].</span><cite>Qadota H, McGaha LA, Mercer KB, Stark TJ, Ferrara TM, Benian GM, A novel protein phosphatase is a binding partner for the protein kinase domains of UNC-89 (obscurin) in C. elegans, Mol. Biol. Cell
19 (2008a) 2424–2432.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E08-01-0053" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2397297/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18337465/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=A%20novel%20protein%20phosphatase%20is%20a%20binding%20partner%20for%20the%20protein%20kinase%20domains%20of%20UNC-89%20(obscurin)%20in%20C.%20elegans&amp;author=H%20Qadota&amp;author=LA%20McGaha&amp;author=KB%20Mercer&amp;author=TJ%20Stark&amp;author=TM%20Ferrara&amp;volume=19&amp;publication_year=2008a&amp;pages=2424-2432&amp;pmid=18337465&amp;doi=10.1091/mbc.E08-01-0053&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R8">
<span class="label">[8].</span><cite>Qadota H, Blangy A, Xiong G, Benian GM, The DH-PH region of the giant protein UNC-89 activates RHO-1 GTPase in C. elegans body wall muscle, J. Mol. Biol. 383 (2008b) 747–752.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jmb.2008.08.083" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2578821/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18801371/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&amp;title=The%20DH-PH%20region%20of%20the%20giant%20protein%20UNC-89%20activates%20RHO-1%20GTPase%20in%20C.%20elegans%20body%20wall%20muscle&amp;author=H%20Qadota&amp;author=A%20Blangy&amp;author=G%20Xiong&amp;author=GM%20Benian&amp;volume=383&amp;publication_year=2008b&amp;pages=747-752&amp;pmid=18801371&amp;doi=10.1016/j.jmb.2008.08.083&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R9">
<span class="label">[9].</span><cite>Xiong G, Qadota H, Mercer KB, McGaha LA, Oberhauser AF, Benian GM, A LIM-9 (FHL) / SCPL-1 (SCP) complex interacts with the C-terminal protein kinase regions of UNC-89 (obscurin) in C. elegans muscle, J. Mol. Biol. 386 (2009) 976–988.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jmb.2009.01.016" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4321865/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19244614/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&amp;title=A%20LIM-9%20(FHL)%20/%20SCPL-1%20(SCP)%20complex%20interacts%20with%20the%20C-terminal%20protein%20kinase%20regions%20of%20UNC-89%20(obscurin)%20in%20C.%20elegans%20muscle&amp;author=G%20Xiong&amp;author=H%20Qadota&amp;author=KB%20Mercer&amp;author=LA%20McGaha&amp;author=AF%20Oberhauser&amp;volume=386&amp;publication_year=2009&amp;pages=976-988&amp;pmid=19244614&amp;doi=10.1016/j.jmb.2009.01.016&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R10">
<span class="label">[10].</span><cite>Wilson KJ, Qadota H, Mains PE, Benian GM, UNC-89 (obscurin) binds to MEL-26, a BTB-domain protein, and affects the function of MEI-1 (katanin) in striated muscle of C. elegans, Mol. Biol. Cell
23 (2012) 2623–2634.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E12-01-0055" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3395652/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22621901/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=UNC-89%20(obscurin)%20binds%20to%20MEL-26,%20a%20BTB-domain%20protein,%20and%20affects%20the%20function%20of%20MEI-1%20(katanin)%20in%20striated%20muscle%20of%20C.%20elegans&amp;author=KJ%20Wilson&amp;author=H%20Qadota&amp;author=PE%20Mains&amp;author=GM%20Benian&amp;volume=23&amp;publication_year=2012&amp;pages=2623-2634&amp;pmid=22621901&amp;doi=10.1091/mbc.E12-01-0055&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R11">
<span class="label">[11].</span><cite>Warner A, Xiong G, Qadota H, Rogalski T, Vogl AW, Moerman DG, Benian GM, CPNA-1, a copine domain protein, is located at integrin adhesion sites and is required for myofilament stability in Caenorhabditis elegans, Mol. Biol. Cell
24 (2013) 601–616.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E12-06-0478" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3583664/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23283987/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=CPNA-1,%20a%20copine%20domain%20protein,%20is%20located%20at%20integrin%20adhesion%20sites%20and%20is%20required%20for%20myofilament%20stability%20in%20Caenorhabditis%20elegans&amp;author=A%20Warner&amp;author=G%20Xiong&amp;author=H%20Qadota&amp;author=T%20Rogalski&amp;author=AW%20Vogl&amp;volume=24&amp;publication_year=2013&amp;pages=601-616&amp;pmid=23283987&amp;doi=10.1091/mbc.E12-06-0478&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R12">
<span class="label">[12].</span><cite>Qadota H, Mayans O, Matsunaga Y, McMurry JL, Wilson KJ, Kwon GE, Stanford R, Deehan K, Tinley TL, Ngwa VM, Benian GM, The SH3 domain of UNC-89 (obscurin) interacts with paramyosin, a coiled-coil protein, in Caenorhabditis elegans muscle, Mol. Biol. Cell
27 (2016) 1606–1620.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E15-09-0675" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4865318/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27009202/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=The%20SH3%20domain%20of%20UNC-89%20(obscurin)%20interacts%20with%20paramyosin,%20a%20coiled-coil%20protein,%20in%20Caenorhabditis%20elegans%20muscle&amp;author=H%20Qadota&amp;author=O%20Mayans&amp;author=Y%20Matsunaga&amp;author=JL%20McMurry&amp;author=KJ%20Wilson&amp;volume=27&amp;publication_year=2016&amp;pages=1606-1620&amp;pmid=27009202&amp;doi=10.1091/mbc.E15-09-0675&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R13">
<span class="label">[13].</span><cite>Qadota H, Matsunaga Y, Bagchi P, Lange KI, Carrier KJ, Pols WV, Swartzbaugh E, Wilson KJ, Srayko M, Pallas DC, Benian GM, Protein phosphatase 2A is crucial for sarcomere organization in Caenorhabditis elegans striated muscle, Mol. Biol. Cell
29 (2018) 2084–2097.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E18-03-0192" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6232960/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29949401/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=Protein%20phosphatase%202A%20is%20crucial%20for%20sarcomere%20organization%20in%20Caenorhabditis%20elegans%20striated%20muscle&amp;author=H%20Qadota&amp;author=Y%20Matsunaga&amp;author=P%20Bagchi&amp;author=KI%20Lange&amp;author=KJ%20Carrier&amp;volume=29&amp;publication_year=2018&amp;pages=2084-2097&amp;pmid=29949401&amp;doi=10.1091/mbc.E18-03-0192&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R14">
<span class="label">[14].</span><cite>Bang ML, Centner T, Fornoff F, Geach AJ, Gotthardt M, McNabb M, Witt CC, Labeit D, Gregorio CC, Granzier H, Labeit S, The complete gene sequence of titin, expression of an unusual approximately 700-kDa titin isoform, and its interaction with obscurin identify a novel Z-line to I-band linking system, Circ. Res. 89 (2001) 1065–1072.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1161/hh2301.100981" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11717165/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Circ.%20Res&amp;title=The%20complete%20gene%20sequence%20of%20titin,%20expression%20of%20an%20unusual%20approximately%20700-kDa%20titin%20isoform,%20and%20its%20interaction%20with%20obscurin%20identify%20a%20novel%20Z-line%20to%20I-band%20linking%20system&amp;author=ML%20Bang&amp;author=T%20Centner&amp;author=F%20Fornoff&amp;author=AJ%20Geach&amp;author=M%20Gotthardt&amp;volume=89&amp;publication_year=2001&amp;pages=1065-1072&amp;pmid=11717165&amp;doi=10.1161/hh2301.100981&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R15">
<span class="label">[15].</span><cite>Young P, Ehler E, Gautel M, Obscurin, a giant sarcomeric Rho guanine nucleotide exchange factor protein involved in sarcomere assembly, J. Cell Biol. 154 (2001) 123–136.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.200102110" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2196875/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11448995/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol&amp;title=Obscurin,%20a%20giant%20sarcomeric%20Rho%20guanine%20nucleotide%20exchange%20factor%20protein%20involved%20in%20sarcomere%20assembly&amp;author=P%20Young&amp;author=E%20Ehler&amp;author=M%20Gautel&amp;volume=154&amp;publication_year=2001&amp;pages=123-136&amp;pmid=11448995&amp;doi=10.1083/jcb.200102110&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R16">
<span class="label">[16].</span><cite>Perry NA, Ackermann MA, Shriver M, Hu LY, Kontrogianni-Konstantopoulos A, Obscurins: unassuming giants enter the spotlight, IUBMB Life
65 (2013) 479–486.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/iub.1157" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3957234/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23512348/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=IUBMB%20Life&amp;title=Obscurins:%20unassuming%20giants%20enter%20the%20spotlight&amp;author=NA%20Perry&amp;author=MA%20Ackermann&amp;author=M%20Shriver&amp;author=LY%20Hu&amp;author=A%20Kontrogianni-Konstantopoulos&amp;volume=65&amp;publication_year=2013&amp;pages=479-486&amp;pmid=23512348&amp;doi=10.1002/iub.1157&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R17">
<span class="label">[17].</span><cite>Bowman AL, Kontrogianni-Konstantopoulos A, Hirsch SS, Geisler SB, Gonzalez-Serratos H, Russell MW, Bloch RJ, Different obscurin isoforms localize to distinct sites at sarcomeres, FEBS Lett. 581 (2007) 1549–54.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.febslet.2007.03.011" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1899168/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17382936/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&amp;title=Different%20obscurin%20isoforms%20localize%20to%20distinct%20sites%20at%20sarcomeres&amp;author=AL%20Bowman&amp;author=A%20Kontrogianni-Konstantopoulos&amp;author=SS%20Hirsch&amp;author=SB%20Geisler&amp;author=H%20Gonzalez-Serratos&amp;volume=581&amp;publication_year=2007&amp;pages=1549-54&amp;pmid=17382936&amp;doi=10.1016/j.febslet.2007.03.011&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R18">
<span class="label">[18].</span><cite>Grogan A, Kontrogianni-Konstantopoulos A, Unraveling obscurins in heart disease, Pflugers Arch. 471 (2018) 735–743.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00424-018-2191-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6397709/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30099631/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Pflugers%20Arch&amp;title=Unraveling%20obscurins%20in%20heart%20disease&amp;author=A%20Grogan&amp;author=A%20Kontrogianni-Konstantopoulos&amp;volume=471&amp;publication_year=2018&amp;pages=735-743&amp;pmid=30099631&amp;doi=10.1007/s00424-018-2191-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R19">
<span class="label">[19].</span><cite>Kontrogianni-Konstantopoulos A, Jones EM, Van Rossum DB, Bloch RJ RJ., Obscurin is a ligand for small ankyrin 1 in skeletal muscle, Mol Biol Cell. 14 (2003) 1138–1148.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E02-07-0411" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC151585/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12631729/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&amp;title=Obscurin%20is%20a%20ligand%20for%20small%20ankyrin%201%20in%20skeletal%20muscle&amp;author=A%20Kontrogianni-Konstantopoulos&amp;author=EM%20Jones&amp;author=DB%20Van%20Rossum&amp;author=RJ%20Bloch%20RJ&amp;volume=14&amp;publication_year=2003&amp;pages=1138-1148&amp;pmid=12631729&amp;doi=10.1091/mbc.E02-07-0411&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R20">
<span class="label">[20].</span><cite>Bagnato P, Barone V, Giacomello E, Rossi D, Sorrentino V, Binding of an ankyrin-1 isoform to obscurin suggests a molecular link between the sarcoplasmic reticulum and myofibrils in striated muscles, J Cell Biol. 160 (2003) 245–253.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.200208109" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2172649/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12527750/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&amp;title=Binding%20of%20an%20ankyrin-1%20isoform%20to%20obscurin%20suggests%20a%20molecular%20link%20between%20the%20sarcoplasmic%20reticulum%20and%20myofibrils%20in%20striated%20muscles&amp;author=P%20Bagnato&amp;author=V%20Barone&amp;author=E%20Giacomello&amp;author=D%20Rossi&amp;author=V%20Sorrentino&amp;volume=160&amp;publication_year=2003&amp;pages=245-253&amp;pmid=12527750&amp;doi=10.1083/jcb.200208109&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R21">
<span class="label">[21].</span><cite>Lange S, Ouyang K, Meyer G, Cui L, Cheng H, Lieber RL, Chen J, Obscurin determines the architecture of the longitudinal sarcoplasmic reticulum, J Cell Sci. 122 (2009) 2640–2650.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/jcs.046193" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2909316/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19584095/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&amp;title=Obscurin%20determines%20the%20architecture%20of%20the%20longitudinal%20sarcoplasmic%20reticulum&amp;author=S%20Lange&amp;author=K%20Ouyang&amp;author=G%20Meyer&amp;author=L%20Cui&amp;author=H%20Cheng&amp;volume=122&amp;publication_year=2009&amp;pages=2640-2650&amp;pmid=19584095&amp;doi=10.1242/jcs.046193&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R22">
<span class="label">[22].</span><cite>Spooner PM, Bonner J, Maricq AV, Benian GM, Norman KR, Large isoforms of UNC-89 (obscurin) are required for muscle cell architecture and optimal calcium release in Caenorhabditis elegans, PLoS ONE
7(7)(2012) e40182.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0040182" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3388081/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22768340/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Large%20isoforms%20of%20UNC-89%20(obscurin)%20are%20required%20for%20muscle%20cell%20architecture%20and%20optimal%20calcium%20release%20in%20Caenorhabditis%20elegans&amp;author=PM%20Spooner&amp;author=J%20Bonner&amp;author=AV%20Maricq&amp;author=GM%20Benian&amp;author=KR%20Norman&amp;volume=7&amp;issue=7&amp;publication_year=2012&amp;pages=e40182&amp;pmid=22768340&amp;doi=10.1371/journal.pone.0040182&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R23">
<span class="label">[23].</span><cite>Geisler SB, Robinson D, Hauringa M, Raeker MO, Borisov AB, Westfall MV, Russell MW, Obscurin-like 1, OBSL1, is a novel cytoskeletal protein related to obscurin, Genomics
89 (2007) 521–531.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ygeno.2006.12.004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1885211/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17289344/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Obscurin-like%201,%20OBSL1,%20is%20a%20novel%20cytoskeletal%20protein%20related%20to%20obscurin&amp;author=SB%20Geisler&amp;author=D%20Robinson&amp;author=M%20Hauringa&amp;author=MO%20Raeker&amp;author=AB%20Borisov&amp;volume=89&amp;publication_year=2007&amp;pages=521-531&amp;pmid=17289344&amp;doi=10.1016/j.ygeno.2006.12.004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R24">
<span class="label">[24].</span><cite>Fukuzawa A, Lange S, Holt M, Vihola A, Carmignac V, Ferreiro A, Udd B, Gautel M M, Interactions with titin and myomesin target obscurin and obscurin-like 1 to the M-band: implications for hereditary myopathies, J. Cell Sci. 121 (2008) 1841–1851.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/jcs.028019" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18477606/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci&amp;title=Interactions%20with%20titin%20and%20myomesin%20target%20obscurin%20and%20obscurin-like%201%20to%20the%20M-band:%20implications%20for%20hereditary%20myopathies&amp;author=A%20Fukuzawa&amp;author=S%20Lange&amp;author=M%20Holt&amp;author=A%20Vihola&amp;author=V%20Carmignac&amp;volume=121&amp;publication_year=2008&amp;pages=1841-1851&amp;pmid=18477606&amp;doi=10.1242/jcs.028019&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R25">
<span class="label">[25].</span><cite>Hanson D, Murray PG, Sud A, Temtamy SA, Aglan M, Superti-Furga A, Holder SE, Urquhart J, Hilton E, Manson FD, Scambler P, Black GC, Clayton PE, The primordial growth disorder 3-M syndrome connects ubiquitination to the cytoskeletal adaptor OBSL1, Am. J. Hum. Genet. 84 (2009) 801–806.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ajhg.2009.04.021" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2694976/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19481195/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&amp;title=The%20primordial%20growth%20disorder%203-M%20syndrome%20connects%20ubiquitination%20to%20the%20cytoskeletal%20adaptor%20OBSL1&amp;author=D%20Hanson&amp;author=PG%20Murray&amp;author=A%20Sud&amp;author=SA%20Temtamy&amp;author=M%20Aglan&amp;volume=84&amp;publication_year=2009&amp;pages=801-806&amp;pmid=19481195&amp;doi=10.1016/j.ajhg.2009.04.021&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R26">
<span class="label">[26].</span><cite>Agrawal PB, Pierson CR, Joshi M, Liu X, Ravenscroft G, Moghadaszadeh B, Talabere T, Viola M, Swanson LC, Haliloğlu G, Talim B, Yau KS, Allcock RJ, Laing NG, Perrella MA, Beggs AH, SPEG interacts with myotubularin, and its deficiency causes centronuclear myopathy with dilated cardiomyopathy, Am. J. Hum. Genet. 95 (2014) 218–226.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ajhg.2014.07.004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4129406/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25087613/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&amp;title=SPEG%20interacts%20with%20myotubularin,%20and%20its%20deficiency%20causes%20centronuclear%20myopathy%20with%20dilated%20cardiomyopathy&amp;author=PB%20Agrawal&amp;author=CR%20Pierson&amp;author=M%20Joshi&amp;author=X%20Liu&amp;author=G%20Ravenscroft&amp;volume=95&amp;publication_year=2014&amp;pages=218-226&amp;pmid=25087613&amp;doi=10.1016/j.ajhg.2014.07.004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R27">
<span class="label">[27].</span><cite>Mayans O, Benian GM, Simkovic F, Rigden DJ, Mechanistic and functional diversity in the mechanosensory kinases of the titin-like family, Biochem. Soc. Trans. 41 (2013) 1066–1071.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1042/BST20130085" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23863180/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Soc.%20Trans&amp;title=Mechanistic%20and%20functional%20diversity%20in%20the%20mechanosensory%20kinases%20of%20the%20titin-like%20family&amp;author=O%20Mayans&amp;author=GM%20Benian&amp;author=F%20Simkovic&amp;author=DJ%20Rigden&amp;volume=41&amp;publication_year=2013&amp;pages=1066-1071&amp;pmid=23863180&amp;doi=10.1042/BST20130085&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R28">
<span class="label">[28].</span><cite>Hu LY, Kontrogianni-Konstantopoulos A, The kinase domains of obscurin interact with intercellular adhesion proteins, FASEB J. 27 (2013) 2001–2012.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1096/fj.12-221317" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3633816/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23392350/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FASEB%20J&amp;title=The%20kinase%20domains%20of%20obscurin%20interact%20with%20intercellular%20adhesion%20proteins&amp;author=LY%20Hu&amp;author=A%20Kontrogianni-Konstantopoulos&amp;volume=27&amp;publication_year=2013&amp;pages=2001-2012&amp;pmid=23392350&amp;doi=10.1096/fj.12-221317&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R29">
<span class="label">[29].</span><cite>Quick AP, Wang Q, Philippen LE, Barreto-Torres G, Chiang DY, Beavers D, Wang G, Khalid M, Reynolds JO, Campbell HM, Showell J, McCauley MD, Scholten A, Wehrens XH, SPEG (Striated Muscle Preferentially Expressed Protein Kinase) Is Essential for Cardiac Function by Regulating Junctional Membrane Complex Activity, Circ. Res. 120 (2016) 110–119.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1161/CIRCRESAHA.116.309977" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5218854/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27729468/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Circ.%20Res&amp;title=SPEG%20(Striated%20Muscle%20Preferentially%20Expressed%20Protein%20Kinase)%20Is%20Essential%20for%20Cardiac%20Function%20by%20Regulating%20Junctional%20Membrane%20Complex%20Activity&amp;author=AP%20Quick&amp;author=Q%20Wang&amp;author=LE%20Philippen&amp;author=G%20Barreto-Torres&amp;author=DY%20Chiang&amp;volume=120&amp;publication_year=2016&amp;pages=110-119&amp;pmid=27729468&amp;doi=10.1161/CIRCRESAHA.116.309977&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R30">
<span class="label">[30].</span><cite>Drozdetskiy A, Cole C, Procter J, Barton GJ, JPred4: a protein secondary structure prediction server, Nucl. Acids Res. 43 (2015) W389–W394.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkv332" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4489285/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25883141/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucl.%20Acids%20Res&amp;title=JPred4:%20a%20protein%20secondary%20structure%20prediction%20server&amp;author=A%20Drozdetskiy&amp;author=C%20Cole&amp;author=J%20Procter&amp;author=GJ%20Barton&amp;volume=43&amp;publication_year=2015&amp;pages=W389-W394&amp;pmid=25883141&amp;doi=10.1093/nar/gkv332&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R31">
<span class="label">[31].</span><cite>Miller DM, Ortiz I, Berliner GC, Epstein HF, Differential localization of two myosins within nematode thick filaments, Cell
34 (1983) 477–490.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0092-8674(83)90381-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/6352051/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Differential%20localization%20of%20two%20myosins%20within%20nematode%20thick%20filaments&amp;author=DM%20Miller&amp;author=I%20Ortiz&amp;author=GC%20Berliner&amp;author=HF%20Epstein&amp;volume=34&amp;publication_year=1983&amp;pages=477-490&amp;pmid=6352051&amp;doi=10.1016/0092-8674(83)90381-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R32">
<span class="label">[32].</span><cite>Qadota H, Matsunaga Y, Nguyen KCQ, Mattheyses A, Hall DH, Benian GM, High-resolution imaging of muscle attachment structures in C. elegans, Cytoskeleton (2017) 1–17.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/cm.21410" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5811768/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28921913/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=elegans,%20Cytoskeleton&amp;title=High-resolution%20imaging%20of%20muscle%20attachment%20structures%20in%20C&amp;author=H%20Qadota&amp;author=Y%20Matsunaga&amp;author=KCQ%20Nguyen&amp;author=A%20Mattheyses&amp;author=DH%20Hall&amp;publication_year=2017&amp;pages=1-17&amp;pmid=28921913&amp;doi=10.1002/cm.21410&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R33">
<span class="label">[33].</span><cite>Waterston RH, Muscle, In The Nematode Caenorhabditis elegans (1988) (Wood WB, ed), pp. 281–335, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=The%20Nematode%20Caenorhabditis%20elegans&amp;author=RH%20Waterston&amp;author=WB%20Wood&amp;publication_year=1988&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R34">
<span class="label">[34].</span><cite>Rahman M, Hewitt JE, Van-Bussel F, Edwards H, Blawzdziewicz J, Szewczyk NJ, Driscoll M, Vanapalli SA, NemaFlex: a microfluidics-based technology for standardized measurement of muscular strength of C. elegans, Lab on a Chip
18.15 (2018) 2187–2201.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1039/c8lc00103k" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6057834/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29892747/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Lab%20on%20a%20Chip&amp;title=NemaFlex:%20a%20microfluidics-based%20technology%20for%20standardized%20measurement%20of%20muscular%20strength%20of%20C.%20elegans&amp;author=M%20Rahman&amp;author=JE%20Hewitt&amp;author=F%20Van-Bussel&amp;author=H%20Edwards&amp;author=J%20Blawzdziewicz&amp;volume=15&amp;publication_year=2018&amp;pages=2187-2201&amp;pmid=29892747&amp;doi=10.1039/c8lc00103k&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R35">
<span class="label">[35].</span><cite>Avery L, The genetics of feeding in C. elegans, Genetics
133 (1993) 897–917.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/genetics/133.4.897" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1205408/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8462849/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The%20genetics%20of%20feeding%20in%20C.%20elegans&amp;author=L%20Avery&amp;volume=133&amp;publication_year=1993&amp;pages=897-917&amp;pmid=8462849&amp;doi=10.1093/genetics/133.4.897&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R36">
<span class="label">[36].</span><cite>Miller DM, Stockdale FE, Karn J, Immunological identification of the genes encoding the four myosin heavy chain isoforms of <em>C. elegans</em>, Proc. Natl. Acad. Sci. 83 (1986) 2305–2309.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.83.8.2305" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC323285/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/2422655/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci&amp;title=Immunological%20identification%20of%20the%20genes%20encoding%20the%20four%20myosin%20heavy%20chain%20isoforms%20of%20C.%20elegans&amp;author=DM%20Miller&amp;author=FE%20Stockdale&amp;author=J%20Karn&amp;volume=83&amp;publication_year=1986&amp;pages=2305-2309&amp;pmid=2422655&amp;doi=10.1073/pnas.83.8.2305&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R37">
<span class="label">[37].</span><cite>Li H, Linke WA, Oberhauser AF, Carrion-Vazquez M, Kerkvliet JG, Lu H, Marszalek PE, Fernandez JM, Reverse engineering of the giant muscle protein titin, Nature
418 (2002) 998–1002.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature00938" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12198551/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Reverse%20engineering%20of%20the%20giant%20muscle%20protein%20titin&amp;author=H%20Li&amp;author=WA%20Linke&amp;author=AF%20Oberhauser&amp;author=M%20Carrion-Vazquez&amp;author=JG%20Kerkvliet&amp;volume=418&amp;publication_year=2002&amp;pages=998-1002&amp;pmid=12198551&amp;doi=10.1038/nature00938&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R38">
<span class="label">[38].</span><cite>Granzier H, Labeit S, Cardiac titin: an adjustable multi-functional spring, J. Physiol. 541(2002) 335–342.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1113/jphysiol.2001.014381" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2290327/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12042342/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Physiol&amp;title=Cardiac%20titin:%20an%20adjustable%20multi-functional%20spring&amp;author=H%20Granzier&amp;author=S%20Labeit&amp;volume=541&amp;publication_year=2002&amp;pages=335-342&amp;pmid=12042342&amp;doi=10.1113/jphysiol.2001.014381&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R39">
<span class="label">[39].</span><cite>Oroz J, Bruix M, Laurents DV, Galera-Prat A, Schonfelder J, Canada FJ, Carrion-Vasquez M, The Y9P variant of the titin I27 module: structural determinants of its revisited nanomechanics, Structure
24 (2016) 606–616.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.str.2016.02.016" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27021163/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=The%20Y9P%20variant%20of%20the%20titin%20I27%20module:%20structural%20determinants%20of%20its%20revisited%20nanomechanics&amp;author=J%20Oroz&amp;author=M%20Bruix&amp;author=DV%20Laurents&amp;author=A%20Galera-Prat&amp;author=J%20Schonfelder&amp;volume=24&amp;publication_year=2016&amp;pages=606-616&amp;pmid=27021163&amp;doi=10.1016/j.str.2016.02.016&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R40">
<span class="label">[40].</span><cite>Moerman DG, Williams BD, Sarcomere assembly in C. elegans muscle, WormBook;  (2006) 1–16.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1895/wormbook.1.81.1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4781162/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18050483/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Sarcomere%20assembly%20in%20C.%20elegans%20muscle&amp;author=DG%20Moerman&amp;author=BD%20Williams&amp;publication_year=2006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R41">
<span class="label">[41].</span><cite>Gieseler K, Qadota H, Benian GM, Development, structure, and maintenance of C. elegans body wall muscle, WormBook;  (2017) 1–59.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1895/wormbook.1.81.2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5410635/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27555356/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Development,%20structure,%20and%20maintenance%20of%20C.%20elegans%20body%20wall%20muscle&amp;author=K%20Gieseler&amp;author=H%20Qadota&amp;author=GM%20Benian&amp;publication_year=2017&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R42">
<span class="label">[42].</span><cite>Lin X, Qadota H, Moerman DG, Williams BD, elegans PAT-C 6/actopaxin plays a critical role in the assembly of integrin adhesion complexes in vivo, Curr. Biol. 13(2003) 922–932.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s0960-9822(03)00372-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12781130/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol&amp;title=6/actopaxin%20plays%20a%20critical%20role%20in%20the%20assembly%20of%20integrin%20adhesion%20complexes%20in%20vivo&amp;author=X%20Lin&amp;author=H%20Qadota&amp;author=DG%20Moerman&amp;author=BD%20Williams&amp;author=PAT-C%20elegans&amp;volume=13&amp;publication_year=2003&amp;pages=922-932&amp;pmid=12781130&amp;doi=10.1016/s0960-9822(03)00372-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R43">
<span class="label">[43].</span><cite>von Castelmur E, Marino M, Svergun DI, Kreplak L, Ucurum-Fotiadis Z, Konarev PV, Urzhumtsev A, Labeit D, Labeit S, Mayans O, A regular pattern of Ig super-motifs defines segmental flexibility as the elastic mechanism of the titin chain, Proc. Natl. Acad. Sci. U S A. 105 (2008) 1186–1191.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0707163105" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2234113/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18212128/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A&amp;title=A%20regular%20pattern%20of%20Ig%20super-motifs%20defines%20segmental%20flexibility%20as%20the%20elastic%20mechanism%20of%20the%20titin%20chain&amp;author=E%20von%20Castelmur&amp;author=M%20Marino&amp;author=DI%20Svergun&amp;author=L%20Kreplak&amp;author=Z%20Ucurum-Fotiadis&amp;volume=105&amp;publication_year=2008&amp;pages=1186-1191&amp;pmid=18212128&amp;doi=10.1073/pnas.0707163105&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R44">
<span class="label">[44].</span><cite>Hresko MC, Williams BD, Waterston RH, Assembly of body wall muscle and muscle cell attachment structures in Caenorhabditis elegans, J. Cell Biol. 124 (1994) 491–506.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.124.4.491" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2119906/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8106548/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol&amp;title=Assembly%20of%20body%20wall%20muscle%20and%20muscle%20cell%20attachment%20structures%20in%20Caenorhabditis%20elegans&amp;author=MC%20Hresko&amp;author=BD%20Williams&amp;author=RH%20Waterston&amp;volume=124&amp;publication_year=1994&amp;pages=491-506&amp;pmid=8106548&amp;doi=10.1083/jcb.124.4.491&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R45">
<span class="label">[45].</span><cite>Agarkova I, Perriard J-C, The M-band: an elastic web that crosslinks thick filaments in the center of the sarcomere, Trends Cell Biol. 15 (2005) 477–485.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tcb.2005.07.001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16061384/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Cell%20Biol&amp;title=The%20M-band:%20an%20elastic%20web%20that%20crosslinks%20thick%20filaments%20in%20the%20center%20of%20the%20sarcomere&amp;author=I%20Agarkova&amp;author=J-C%20Perriard&amp;volume=15&amp;publication_year=2005&amp;pages=477-485&amp;pmid=16061384&amp;doi=10.1016/j.tcb.2005.07.001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R46">
<span class="label">[46].</span><cite>Lange S, Pinotsis N, Agarkova I, Ehler E, The M-band: The underestimated part of the sarcomere, BBA-Mol. Cell Res. 1867 (2020) 118440.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.bbamcr.2019.02.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7023976/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30738787/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BBA-Mol.%20Cell%20Res&amp;title=The%20M-band:%20The%20underestimated%20part%20of%20the%20sarcomere&amp;author=S%20Lange&amp;author=N%20Pinotsis&amp;author=I%20Agarkova&amp;author=E%20Ehler&amp;volume=1867&amp;publication_year=2020&amp;pages=118440&amp;pmid=30738787&amp;doi=10.1016/j.bbamcr.2019.02.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R47">
<span class="label">[47].</span><cite>Berkemeier F, Bertz M, Xiao S, Pinotsis N, Wilmanns M, Grater F, Rief M, Fast-folding alpha-helices as reversible strain absorbers in the muscle protein myomesin, Proc. Natl. Acad. Sci. USA
108 (2011) 14139–14144.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1105734108" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3161521/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21825161/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Fast-folding%20alpha-helices%20as%20reversible%20strain%20absorbers%20in%20the%20muscle%20protein%20myomesin&amp;author=F%20Berkemeier&amp;author=M%20Bertz&amp;author=S%20Xiao&amp;author=N%20Pinotsis&amp;author=M%20Wilmanns&amp;volume=108&amp;publication_year=2011&amp;pages=14139-14144&amp;pmid=21825161&amp;doi=10.1073/pnas.1105734108&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R48">
<span class="label">[48].</span><cite>Pernigo S, Fukuzawa A, Beedle AEM, Holt M, Round A, Pandini A, Garcia-Manyes S, Gautel M, Steiner RA, Binding of myomesin to obscurin-like-1 at the muscle M-band provides a strategy for isoform-specific mechanical protection, Structure
25 (2017) 107–120.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.str.2016.11.015" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5222588/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27989621/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Binding%20of%20myomesin%20to%20obscurin-like-1%20at%20the%20muscle%20M-band%20provides%20a%20strategy%20for%20isoform-specific%20mechanical%20protection&amp;author=S%20Pernigo&amp;author=A%20Fukuzawa&amp;author=AEM%20Beedle&amp;author=M%20Holt&amp;author=A%20Round&amp;volume=25&amp;publication_year=2017&amp;pages=107-120&amp;pmid=27989621&amp;doi=10.1016/j.str.2016.11.015&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R49">
<span class="label">[49].</span><cite>Mango S, The C. elegans pharynx: a model for organogenesis. WormBook, ed. The C. elegans Research Community, WormBook, 10.1895/wormbook.1.129.1 (2007).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1895/wormbook.1.129.1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4781022/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18050503/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=The%20C.%20elegans%20Research%20Community&amp;author=S%20Mango&amp;publication_year=2007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R50">
<span class="label">[50].</span><cite>Bujalowski PJ, Oberhauser AF, Tracking unfolding and refolding reactions of single proteins using atomic force microscopy methods, Methods
60 (2013) 151–160.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ymeth.2013.03.010" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3669651/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23523554/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Tracking%20unfolding%20and%20refolding%20reactions%20of%20single%20proteins%20using%20atomic%20force%20microscopy%20methods&amp;author=PJ%20Bujalowski&amp;author=AF%20Oberhauser&amp;volume=60&amp;publication_year=2013&amp;pages=151-160&amp;pmid=23523554&amp;doi=10.1016/j.ymeth.2013.03.010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R51">
<span class="label">[51].</span><cite>Rabbi M, Marszalek PE, Measuring Protein Mechanics by Atomic Force Microscopy </cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/pdb.prot4901" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21357003/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Measuring%20Protein%20Mechanics%20by%20Atomic%20Force%20Microscopy&amp;author=M%20Rabbi&amp;author=PE%20Marszalek&amp;pmid=21357003&amp;doi=10.1101/pdb.prot4901&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R52">
<span class="label">[52].</span><cite>Hossain MD, Furuike S, Kinosita K Jr, The rotor tip inside a bearing of a thermophilic F1-ATPase is dispensable for torque generation, Biophys. J. 90 (2006) 4195–4203.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1529/biophysj.105.079087" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1459503/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16698789/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20J&amp;title=The%20rotor%20tip%20inside%20a%20bearing%20of%20a%20thermophilic%20F1-ATPase%20is%20dispensable%20for%20torque%20generation&amp;author=MD%20Hossain&amp;author=S%20Furuike&amp;author=K%20Kinosita&amp;volume=90&amp;publication_year=2006&amp;pages=4195-4203&amp;pmid=16698789&amp;doi=10.1529/biophysj.105.079087&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R53">
<span class="label">[53].</span><cite>Florin E-L, Rief M, Lehmann H, Ludwig M, Dornmair C, Moy VT, Gaub HE, Sensing specific molecular interactions with the atomic force microscope. Biosens. and Bioelectr. 10 (1995) 895–901.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biosens.%20and%20Bioelectr&amp;title=Sensing%20specific%20molecular%20interactions%20with%20the%20atomic%20force%20microscope&amp;author=E-L%20Florin&amp;author=M%20Rief&amp;author=H%20Lehmann&amp;author=M%20Ludwig&amp;author=C%20Dornmair&amp;volume=10&amp;publication_year=1995&amp;pages=895-901&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R54">
<span class="label">[54].</span><cite>Kim M, Abdi K, Marszalek PE, Fast and forceful refolding of stretched alpha-helical solenoid proteins, Biophys. J. 98 (2010) 3086–3092.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.bpj.2010.02.054" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2884255/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20550922/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20J&amp;title=Fast%20and%20forceful%20refolding%20of%20stretched%20alpha-helical%20solenoid%20proteins&amp;author=M%20Kim&amp;author=K%20Abdi&amp;author=PE%20Marszalek&amp;volume=98&amp;publication_year=2010&amp;pages=3086-3092&amp;pmid=20550922&amp;doi=10.1016/j.bpj.2010.02.054&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R55">
<span class="label">[55].</span><cite>Bustamante C, Marko JF, Smith S S, Entropic elasticity of lambda-phage DNA. Science
265 (1994) 1599–1600.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.8079175" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8079175/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Entropic%20elasticity%20of%20lambda-phage%20DNA&amp;author=C%20Bustamante&amp;author=JF%20Marko&amp;author=S%20Smith%20S&amp;volume=265&amp;publication_year=1994&amp;pages=1599-1600&amp;pmid=8079175&amp;doi=10.1126/science.8079175&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R56">
<span class="label">[56].</span><cite>Marko JF, Siggia ED, Stretching DNA, Macromolecules
28 (1995) 8759–8770.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Macromolecules&amp;title=Stretching%20DNA&amp;author=JF%20Marko&amp;author=ED%20Siggia&amp;volume=28&amp;publication_year=1995&amp;pages=8759-8770&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R57">
<span class="label">[57].</span><cite>Hannak E, Oegema K, Kirkham M, Gonczy P, Habermann B, Hyman AA, The kinetically dominant assembly pathway for centrosomal asters in Caenorhabditis elegans is gamma-tubulin dependent, J. Cell Biol. 157 (2002) 591–602.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.200202047" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2173857/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12011109/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol&amp;title=The%20kinetically%20dominant%20assembly%20pathway%20for%20centrosomal%20asters%20in%20Caenorhabditis%20elegans%20is%20gamma-tubulin%20dependent&amp;author=E%20Hannak&amp;author=K%20Oegema&amp;author=M%20Kirkham&amp;author=P%20Gonczy&amp;author=B%20Habermann&amp;volume=157&amp;publication_year=2002&amp;pages=591-602&amp;pmid=12011109&amp;doi=10.1083/jcb.200202047&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R58">
<span class="label">[58].</span><cite>Nonet ML, Grundahl K, Meyer BJ, Rand JB, Synaptic function is impaired but not eliminated in C. elegans mutants lacking synaptotagmin, Cell
73 (1993) 1291–1305.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0092-8674(93)90357-v" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8391930/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Synaptic%20function%20is%20impaired%20but%20not%20eliminated%20in%20C.%20elegans%20mutants%20lacking%20synaptotagmin&amp;author=ML%20Nonet&amp;author=K%20Grundahl&amp;author=BJ%20Meyer&amp;author=JB%20Rand&amp;volume=73&amp;publication_year=1993&amp;pages=1291-1305&amp;pmid=8391930&amp;doi=10.1016/0092-8674(93)90357-v&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R59">
<span class="label">[59].</span><cite>Wilson KJ, Qadota H, Benian GM, Immunofluorescent localization of proteins in Caenorhabditis elegans muscle, Methods
Mol. Biol. 798 (2012) 171–181.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-1-61779-343-1_10" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3731740/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22130837/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol&amp;title=Immunofluorescent%20localization%20of%20proteins%20in%20Caenorhabditis%20elegans%20muscle,%20Methods&amp;author=KJ%20Wilson&amp;author=H%20Qadota&amp;author=GM%20Benian&amp;volume=798&amp;publication_year=2012&amp;pages=171-181&amp;pmid=22130837&amp;doi=10.1007/978-1-61779-343-1_10&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R60">
<span class="label">[60].</span><cite>Qadota H, Mercer KB, Miller RK, Kaibuchi K, Benian GM, Two LIM domain proteins and UNC-96 link UNC-97/pinch to myosin thick filaments in Caenorhabditis elegans muscle, Mol. Biol. Cell
18 (2007) 4317–4326.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E07-03-0278" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2043538/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17761533/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=Two%20LIM%20domain%20proteins%20and%20UNC-96%20link%20UNC-97/pinch%20to%20myosin%20thick%20filaments%20in%20Caenorhabditis%20elegans%20muscle&amp;author=H%20Qadota&amp;author=KB%20Mercer&amp;author=RK%20Miller&amp;author=K%20Kaibuchi&amp;author=GM%20Benian&amp;volume=18&amp;publication_year=2007&amp;pages=4317-4326&amp;pmid=17761533&amp;doi=10.1091/mbc.E07-03-0278&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R61">
<span class="label">[61].</span><cite>Francis R, Waterston RH, Muscle cell attachment in Caenorhabditis elegans, J. Cell Biol. 114 (1991) 465–479.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.114.3.465" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2289102/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/1860880/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol&amp;title=Muscle%20cell%20attachment%20in%20Caenorhabditis%20elegans&amp;author=R%20Francis&amp;author=RH%20Waterston&amp;volume=114&amp;publication_year=1991&amp;pages=465-479&amp;pmid=1860880&amp;doi=10.1083/jcb.114.3.465&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R62">
<span class="label">[62].</span><cite>Nussbaum-Krammer C, Neto MF, Brielmann RM, Pedersen JS, Morimoto RI, Investigating the spreading of toxicity of prion-like proteins using the metazoan model organism C. elegans, J. Vis. Exp. 95 (2015) 52321.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3791/52321" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4354510/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25591151/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Vis.%20Exp&amp;title=Investigating%20the%20spreading%20of%20toxicity%20of%20prion-like%20proteins%20using%20the%20metazoan%20model%20organism%20C.%20elegans&amp;author=C%20Nussbaum-Krammer&amp;author=MF%20Neto&amp;author=RM%20Brielmann&amp;author=JS%20Pedersen&amp;author=RI%20Morimoto&amp;volume=95&amp;publication_year=2015&amp;pages=52321&amp;pmid=25591151&amp;doi=10.3791/52321&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R63">
<span class="label">[63].</span><cite>Hewitt JE, Pollard AK, Lesanpezeshki L, Deane CS, Gaffney CJ, Etheridge T, Szewczyk NJ, Vanapalli SA, Muscle strength deficiency and mitochondrial dysfunction in a muscular dystrophy model of Caenorhabditis elegans and its functional response to drugs, Disease Mod. &amp; Mechan. 11.12 (2018).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/dmm.036137" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6307913/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30396907/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Disease%20Mod.%20&amp;%20Mechan&amp;title=Muscle%20strength%20deficiency%20and%20mitochondrial%20dysfunction%20in%20a%20muscular%20dystrophy%20model%20of%20Caenorhabditis%20elegans%20and%20its%20functional%20response%20to%20drugs&amp;author=JE%20Hewitt&amp;author=AK%20Pollard&amp;author=L%20Lesanpezeshki&amp;author=CS%20Deane&amp;author=CJ%20Gaffney&amp;volume=11&amp;publication_year=2018&amp;pages=12&amp;pmid=30396907&amp;doi=10.1242/dmm.036137&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R64">
<span class="label">[64].</span><cite>Micsonai A, Wien F, Bulyaki E, Kun J, Moussong E, Lee Y-H, Goto Y, Refregiers M, Kardos J, BeStSel: a web server for accurate protein secondary structure prediction and fold recognition from the circular dichroism spectra, Nucl. Acids Res. 46 (2018) W315–W322.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gky497" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6031044/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29893907/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucl.%20Acids%20Res&amp;title=BeStSel:%20a%20web%20server%20for%20accurate%20protein%20secondary%20structure%20prediction%20and%20fold%20recognition%20from%20the%20circular%20dichroism%20spectra&amp;author=A%20Micsonai&amp;author=F%20Wien&amp;author=E%20Bulyaki&amp;author=J%20Kun&amp;author=E%20Moussong&amp;volume=46&amp;publication_year=2018&amp;pages=W315-W322&amp;pmid=29893907&amp;doi=10.1093/nar/gky497&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>Supp. Fig. 1</span><p>Supplemental Fig. 1. Sequence alignment of IK regions of UNC-89, obscurin and SPEG using ClustalW. (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.genome.jp/tools-bin/clustalw" rel="noopener noreferrer" target="_blank">https://www.genome.jp/tools-bin/clustalw</a>).</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Fig__1.pdf">NIHMS1838736-supplement-Supp__Fig__1.pdf</a><sup> (55.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>Supp. Fig. 2</span><p>Supplemental Fig. 2. Secondary structure predictions of IK regions of UNC-89, obscurin and SPEG using JPred. (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.compbio.dundee.ac.uk/jpred/index.html" rel="noopener noreferrer" target="_blank">http://www.compbio.dundee.ac.uk/jpred/index.html</a>)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Fig__2.pdf">NIHMS1838736-supplement-Supp__Fig__2.pdf</a><sup> (72.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>Supp. Fig. 3</span><p>Supplemental Fig. 3. Results of Sanger sequencing and conceptual translation of PHX797, <em>unc-89(syb797)</em>.</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Fig__3.pdf">NIHMS1838736-supplement-Supp__Fig__3.pdf</a><sup> (38.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>Supp. Fig. 4</span><p>Supplemental Fig. 4. Results of Sanger sequencing and conceptual translation across genomic deletion in PHX1257, <em>unc-89(syb797syb1257)</em>.</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Fig__4.pdf">NIHMS1838736-supplement-Supp__Fig__4.pdf</a><sup> (116.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>Supp. Fig. 5</span><p>Supplementary Fig. 5. The body wall muscle of <em>unc-89(ad539)</em> has normally organized sarcomeres. Each panel shows representative confocal images of several body wall muscle cells stained with antibodies to the indicated sarcomeric components or with rhodamine-phalloidin for F-actin. Scale bar, 10 μm.</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Fig__5.tif">NIHMS1838736-supplement-Supp__Fig__5.tif</a><sup> (4.9MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>Supp. Fig. 6</span><p>Supplementary Figure 6. Secondary structure of the I27<sub>2</sub>-IK-I27<sub>2</sub> protein and a control I27<sub>9</sub> polyprotein determined by circular dichroism. (A) The overall secondary structure determined by Far-UV circular dichroism of the I27<sub>2</sub>-IK-I27<sub>2</sub> used in AFM experiments (solid line) and a control protein containing 9 identical repeats of the I27 domain (dotted line). The protein concentration was 1 μM in 30 mM phosphate buffer pH 7.4, 100 mM KCl, 1 mM MgCl<sub>2</sub>, 1mM TCEP buffer. A 0.1 cm path length cuvette was used. (B) The % of β-strand, α-helical and random-coil content was estimated by using the BeStSel tool [<a aria-describedby="R64" class="usa-link" href="#R64">64</a>] and these are: 16% β-strand, 29% α-helix, 55% random for the IK construct (box on the right) and 40% β-strand, 20% α-helix, 36% random-coil for the I27 polyprotein (box on the left).</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Fig__6.tif">NIHMS1838736-supplement-Supp__Fig__6.tif</a><sup> (2.3MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>Supp. Table 1</span><p>Supplementary Table 1. Detailed statistics for swimming and crawling assays on the different strains used in this study and summarized in <a class="usa-link" href="#F6">Fig. 6A</a> and <a class="usa-link" href="#F6">B</a>.</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Table_1.tiff">NIHMS1838736-supplement-Supp__Table_1.tiff</a><sup> (724.7KB, tiff) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p">
<span>Supp. Table 2</span><p>Supplementary Table 2: Muscle force and body diameter data for the different strains used in this study and summarized in <a class="usa-link" href="#F6">Fig. 6C</a>.</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9549344/bin/NIHMS1838736-supplement-Supp__Table_2.tiff">NIHMS1838736-supplement-Supp__Table_2.tiff</a><sup> (747.8KB, tiff) </sup>
</div></div></section></section></section></section></section></article>
</main></body></html>