<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8441986/"/><title>Circulating miRNA Spaceflight Signature Reveals Targets for Countermeasure Development - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Cell Rep</button></div>. Author manuscript; available in PMC: 2021 Sep 15.</div>
<div>
<em>Published in final edited form as: </em>Cell Rep. 2020 Nov 25;33(10):108448. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.celrep.2020.108448" rel="noopener noreferrer" target="_blank">10.1016/j.celrep.2020.108448</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Circulating miRNA Spaceflight Signature Reveals Targets for Countermeasure Development</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Malkani%20S%22%5BAuthor%5D"><span class="name western">Sherina Malkani</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sherina Malkani</span></h3>
<div class="p">
<sup>1</sup>Blue Marble Space Institute of Science, Space Biosciences Division, NASA Ames Research Center, Moffett Field, CA 94035, USA</div>
<div class="p">
<sup>22</sup>These authors contributed equally</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Malkani%20S%22%5BAuthor%5D"><span class="name western">Sherina Malkani</span></a>
</div>
</div>
<sup>1,</sup><sup>22</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chin%20CR%22%5BAuthor%5D"><span class="name western">Christopher R Chin</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Christopher R Chin</span></h3>
<div class="p">
<sup>2</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>3</sup>The HRH Prince Alwaleed Bin Talal Bin Abdulaziz Alsaud Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>22</sup>These authors contributed equally</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chin%20CR%22%5BAuthor%5D"><span class="name western">Christopher R Chin</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>22</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cekanaviciute%20E%22%5BAuthor%5D"><span class="name western">Egle Cekanaviciute</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Egle Cekanaviciute</span></h3>
<div class="p">
<sup>4</sup>Space Biosciences Division, NASA Ames Research Center, Moffett Field, CA 94035, USA</div>
<div class="p">
<sup>22</sup>These authors contributed equally</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cekanaviciute%20E%22%5BAuthor%5D"><span class="name western">Egle Cekanaviciute</span></a>
</div>
</div>
<sup>4,</sup><sup>22</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mortreux%20M%22%5BAuthor%5D"><span class="name western">Marie Mortreux</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Marie Mortreux</span></h3>
<div class="p">
<sup>5</sup>Department of Neurology, Beth Israel Deaconess Medical Center, Harvard Medical School, Boston, MA 02215, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mortreux%20M%22%5BAuthor%5D"><span class="name western">Marie Mortreux</span></a>
</div>
</div>
<sup>5</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okinula%20H%22%5BAuthor%5D"><span class="name western">Hazeem Okinula</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Hazeem Okinula</span></h3>
<div class="p">
<sup>6</sup>Center for Radiological Research, Columbia University, New York, NY 10032, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okinula%20H%22%5BAuthor%5D"><span class="name western">Hazeem Okinula</span></a>
</div>
</div>
<sup>6</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tarbier%20M%22%5BAuthor%5D"><span class="name western">Marcel Tarbier</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Marcel Tarbier</span></h3>
<div class="p">
<sup>7</sup>Science for Life Laboratory, Department of Molecular Biosciences, the Wenner-Gren Institute, Stockholm University, Stockholm, Sweden</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tarbier%20M%22%5BAuthor%5D"><span class="name western">Marcel Tarbier</span></a>
</div>
</div>
<sup>7</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schreurs%20AS%22%5BAuthor%5D"><span class="name western">Ann-Sofie Schreurs</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Ann-Sofie Schreurs</span></h3>
<div class="p">
<sup>8</sup>KBR, Space Biosciences Division, NASA Ames Research Center, Moffett Field, CA 94035, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schreurs%20AS%22%5BAuthor%5D"><span class="name western">Ann-Sofie Schreurs</span></a>
</div>
</div>
<sup>8</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shirazi-Fard%20Y%22%5BAuthor%5D"><span class="name western">Yasaman Shirazi-Fard</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Yasaman Shirazi-Fard</span></h3>
<div class="p">
<sup>4</sup>Space Biosciences Division, NASA Ames Research Center, Moffett Field, CA 94035, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shirazi-Fard%20Y%22%5BAuthor%5D"><span class="name western">Yasaman Shirazi-Fard</span></a>
</div>
</div>
<sup>4</sup>, <a aria-describedby="id9" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tahimic%20CGT%22%5BAuthor%5D"><span class="name western">Candice GT Tahimic</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Candice GT Tahimic</span></h3>
<div class="p">
<sup>8</sup>KBR, Space Biosciences Division, NASA Ames Research Center, Moffett Field, CA 94035, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tahimic%20CGT%22%5BAuthor%5D"><span class="name western">Candice GT Tahimic</span></a>
</div>
</div>
<sup>8</sup>, <a aria-describedby="id10" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20DN%22%5BAuthor%5D"><span class="name western">Deyra N Rodriguez</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Deyra N Rodriguez</span></h3>
<div class="p">
<sup>9</sup>New England Biolabs, Ipswich, MA, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20DN%22%5BAuthor%5D"><span class="name western">Deyra N Rodriguez</span></a>
</div>
</div>
<sup>9</sup>, <a aria-describedby="id11" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sexton%20BS%22%5BAuthor%5D"><span class="name western">Brittany S Sexton</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Brittany S Sexton</span></h3>
<div class="p">
<sup>9</sup>New England Biolabs, Ipswich, MA, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sexton%20BS%22%5BAuthor%5D"><span class="name western">Brittany S Sexton</span></a>
</div>
</div>
<sup>9</sup>, <a aria-describedby="id12" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Butler%20D%22%5BAuthor%5D"><span class="name western">Daniel Butler</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Daniel Butler</span></h3>
<div class="p">
<sup>2</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>3</sup>The HRH Prince Alwaleed Bin Talal Bin Abdulaziz Alsaud Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Butler%20D%22%5BAuthor%5D"><span class="name western">Daniel Butler</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a aria-describedby="id13" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Verma%20A%22%5BAuthor%5D"><span class="name western">Akanksha Verma</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Akanksha Verma</span></h3>
<div class="p">
<sup>2</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>3</sup>The HRH Prince Alwaleed Bin Talal Bin Abdulaziz Alsaud Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Verma%20A%22%5BAuthor%5D"><span class="name western">Akanksha Verma</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a aria-describedby="id14" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bezdan%20D%22%5BAuthor%5D"><span class="name western">Daniela Bezdan</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Daniela Bezdan</span></h3>
<div class="p">
<sup>2</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>3</sup>The HRH Prince Alwaleed Bin Talal Bin Abdulaziz Alsaud Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>10</sup>Institute of Medical Virology and Epidemiology of Viral Diseases, University Hospital, Tubingen, Germany</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bezdan%20D%22%5BAuthor%5D"><span class="name western">Daniela Bezdan</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>10</sup>, <a aria-describedby="id15" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Durmaz%20C%22%5BAuthor%5D"><span class="name western">Ceyda Durmaz</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Ceyda Durmaz</span></h3>
<div class="p">
<sup>2</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>3</sup>The HRH Prince Alwaleed Bin Talal Bin Abdulaziz Alsaud Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Durmaz%20C%22%5BAuthor%5D"><span class="name western">Ceyda Durmaz</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a aria-describedby="id16" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22MacKay%20M%22%5BAuthor%5D"><span class="name western">Matthew MacKay</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Matthew MacKay</span></h3>
<div class="p">
<sup>2</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>3</sup>The HRH Prince Alwaleed Bin Talal Bin Abdulaziz Alsaud Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22MacKay%20M%22%5BAuthor%5D"><span class="name western">Matthew MacKay</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a aria-describedby="id17" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Melnick%20A%22%5BAuthor%5D"><span class="name western">Ari Melnick</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Ari Melnick</span></h3>
<div class="p">
<sup>11</sup>Department of Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Melnick%20A%22%5BAuthor%5D"><span class="name western">Ari Melnick</span></a>
</div>
</div>
<sup>11</sup>, <a aria-describedby="id18" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meydan%20C%22%5BAuthor%5D"><span class="name western">Cem Meydan</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Cem Meydan</span></h3>
<div class="p">
<sup>2</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>3</sup>The HRH Prince Alwaleed Bin Talal Bin Abdulaziz Alsaud Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meydan%20C%22%5BAuthor%5D"><span class="name western">Cem Meydan</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a aria-describedby="id19" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20S%22%5BAuthor%5D"><span class="name western">Sheng Li</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Sheng Li</span></h3>
<div class="p">
<sup>12</sup>The Jackson Laboratories, Farmington, CT, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20S%22%5BAuthor%5D"><span class="name western">Sheng Li</span></a>
</div>
</div>
<sup>12</sup>, <a aria-describedby="id20" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garrett-Bakelman%20F%22%5BAuthor%5D"><span class="name western">Francine Garrett-Bakelman</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Francine Garrett-Bakelman</span></h3>
<div class="p">
<sup>11</sup>Department of Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>13</sup>Department of Medicine, University of Virginia School of Medicine, Charlottesville, VA, USA</div>
<div class="p">
<sup>14</sup>Department of Biochemistry and Molecular Genetics, University of Virginia School of Medicine, Charlottesville, VA, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garrett-Bakelman%20F%22%5BAuthor%5D"><span class="name western">Francine Garrett-Bakelman</span></a>
</div>
</div>
<sup>11,</sup><sup>13,</sup><sup>14</sup>, <a aria-describedby="id21" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fromm%20B%22%5BAuthor%5D"><span class="name western">Bastian Fromm</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Bastian Fromm</span></h3>
<div class="p">
<sup>7</sup>Science for Life Laboratory, Department of Molecular Biosciences, the Wenner-Gren Institute, Stockholm University, Stockholm, Sweden</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fromm%20B%22%5BAuthor%5D"><span class="name western">Bastian Fromm</span></a>
</div>
</div>
<sup>7</sup>, <a aria-describedby="id22" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Afshinnekoo%20E%22%5BAuthor%5D"><span class="name western">Ebrahim Afshinnekoo</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Ebrahim Afshinnekoo</span></h3>
<div class="p">
<sup>2</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Afshinnekoo%20E%22%5BAuthor%5D"><span class="name western">Ebrahim Afshinnekoo</span></a>
</div>
</div>
<sup>2</sup>, <a aria-describedby="id23" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Langhorst%20BW%22%5BAuthor%5D"><span class="name western">Brad W Langhorst</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Brad W Langhorst</span></h3>
<div class="p">
<sup>9</sup>New England Biolabs, Ipswich, MA, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Langhorst%20BW%22%5BAuthor%5D"><span class="name western">Brad W Langhorst</span></a>
</div>
</div>
<sup>9</sup>, <a aria-describedby="id24" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dimalanta%20ET%22%5BAuthor%5D"><span class="name western">Eileen T Dimalanta</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Eileen T Dimalanta</span></h3>
<div class="p">
<sup>9</sup>New England Biolabs, Ipswich, MA, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dimalanta%20ET%22%5BAuthor%5D"><span class="name western">Eileen T Dimalanta</span></a>
</div>
</div>
<sup>9</sup>, <a aria-describedby="id25" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng-Campbell%20M%22%5BAuthor%5D"><span class="name western">Margareth Cheng-Campbell</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Margareth Cheng-Campbell</span></h3>
<div class="p">
<sup>15</sup>Center for Biotechnology &amp; Interdisciplinary Studies, Rensselaer Polytechnic Institute, Troy, NY 12180, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng-Campbell%20M%22%5BAuthor%5D"><span class="name western">Margareth Cheng-Campbell</span></a>
</div>
</div>
<sup>15</sup>, <a aria-describedby="id26" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blaber%20E%22%5BAuthor%5D"><span class="name western">Elizabeth Blaber</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Elizabeth Blaber</span></h3>
<div class="p">
<sup>15</sup>Center for Biotechnology &amp; Interdisciplinary Studies, Rensselaer Polytechnic Institute, Troy, NY 12180, USA</div>
<div class="p">
<sup>16</sup>Universities Space Research Association, Space Biosciences Division, NASA Ames Research Center, Mountain View, CA 94035, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blaber%20E%22%5BAuthor%5D"><span class="name western">Elizabeth Blaber</span></a>
</div>
</div>
<sup>15,</sup><sup>16</sup>, <a aria-describedby="id27" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schisler%20JC%22%5BAuthor%5D"><span class="name western">Jonathan C Schisler</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Jonathan C Schisler</span></h3>
<div class="p">
<sup>17</sup>McAllister Heart Institute, Department of Pharmacology, and Department of Pathology and Lab Medicine, The University of North Carolina at Chapel Hill, Chapel Hill, NC 27599, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schisler%20JC%22%5BAuthor%5D"><span class="name western">Jonathan C Schisler</span></a>
</div>
</div>
<sup>17</sup>, <a aria-describedby="id28" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vanderburg%20C%22%5BAuthor%5D"><span class="name western">Charles Vanderburg</span></a><div hidden="hidden" id="id28">
<h3><span class="name western">Charles Vanderburg</span></h3>
<div class="p">
<sup>18</sup>Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vanderburg%20C%22%5BAuthor%5D"><span class="name western">Charles Vanderburg</span></a>
</div>
</div>
<sup>18</sup>, <a aria-describedby="id29" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Friedl%C3%A4nder%20MR%22%5BAuthor%5D"><span class="name western">Marc R Friedländer</span></a><div hidden="hidden" id="id29">
<h3><span class="name western">Marc R Friedländer</span></h3>
<div class="p">
<sup>7</sup>Science for Life Laboratory, Department of Molecular Biosciences, the Wenner-Gren Institute, Stockholm University, Stockholm, Sweden</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Friedl%C3%A4nder%20MR%22%5BAuthor%5D"><span class="name western">Marc R Friedländer</span></a>
</div>
</div>
<sup>7</sup>, <a aria-describedby="id30" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McDonald%20JT%22%5BAuthor%5D"><span class="name western">J Tyson McDonald</span></a><div hidden="hidden" id="id30">
<h3><span class="name western">J Tyson McDonald</span></h3>
<div class="p">
<sup>19</sup>Department of Radiation Medicine, Georgetown University School of Medicine, Washington DC 20007, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McDonald%20JT%22%5BAuthor%5D"><span class="name western">J Tyson McDonald</span></a>
</div>
</div>
<sup>19</sup>, <a aria-describedby="id31" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Costes%20SV%22%5BAuthor%5D"><span class="name western">Sylvain V Costes</span></a><div hidden="hidden" id="id31">
<h3><span class="name western">Sylvain V Costes</span></h3>
<div class="p">
<sup>4</sup>Space Biosciences Division, NASA Ames Research Center, Moffett Field, CA 94035, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Costes%20SV%22%5BAuthor%5D"><span class="name western">Sylvain V Costes</span></a>
</div>
</div>
<sup>4</sup>, <a aria-describedby="id32" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rutkove%20S%22%5BAuthor%5D"><span class="name western">Seward Rutkove</span></a><div hidden="hidden" id="id32">
<h3><span class="name western">Seward Rutkove</span></h3>
<div class="p">
<sup>5</sup>Department of Neurology, Beth Israel Deaconess Medical Center, Harvard Medical School, Boston, MA 02215, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rutkove%20S%22%5BAuthor%5D"><span class="name western">Seward Rutkove</span></a>
</div>
</div>
<sup>5</sup>, <a aria-describedby="id33" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grabham%20P%22%5BAuthor%5D"><span class="name western">Peter Grabham</span></a><div hidden="hidden" id="id33">
<h3><span class="name western">Peter Grabham</span></h3>
<div class="p">
<sup>6</sup>Center for Radiological Research, Columbia University, New York, NY 10032, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grabham%20P%22%5BAuthor%5D"><span class="name western">Peter Grabham</span></a>
</div>
</div>
<sup>6</sup>, <a aria-describedby="id34" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D"><span class="name western">Christopher E Mason</span></a><div hidden="hidden" id="id34">
<h3><span class="name western">Christopher E Mason</span></h3>
<div class="p">
<sup>2</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>3</sup>The HRH Prince Alwaleed Bin Talal Bin Abdulaziz Alsaud Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>20</sup>The WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>21</sup>The Feil Family Brain and Mind Research Institute, Weill Cornell Medicine, New York, NY, USA</div>
<div class="p">
<sup>23</sup>Senior author</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D"><span class="name western">Christopher E Mason</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>20,</sup><sup>21,</sup><sup>23,</sup><sup>*</sup>, <a aria-describedby="id35" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Beheshti%20A%22%5BAuthor%5D"><span class="name western">Afshin Beheshti</span></a><div hidden="hidden" id="id35">
<h3><span class="name western">Afshin Beheshti</span></h3>
<div class="p">
<sup>8</sup>KBR, Space Biosciences Division, NASA Ames Research Center, Moffett Field, CA 94035, USA</div>
<div class="p">
<sup>18</sup>Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</div>
<div class="p">
<sup>23</sup>Senior author</div>
<div class="p">
<sup>24</sup>Lead Contact</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Beheshti%20A%22%5BAuthor%5D"><span class="name western">Afshin Beheshti</span></a>
</div>
</div>
<sup>8,</sup><sup>18,</sup><sup>23,</sup><sup>24,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="A1">
<sup>1</sup>Blue Marble Space Institute of Science, Space Biosciences Division, NASA Ames Research Center, Moffett Field, CA 94035, USA</div>
<div id="A2">
<sup>2</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</div>
<div id="A3">
<sup>3</sup>The HRH Prince Alwaleed Bin Talal Bin Abdulaziz Alsaud Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY, USA</div>
<div id="A4">
<sup>4</sup>Space Biosciences Division, NASA Ames Research Center, Moffett Field, CA 94035, USA</div>
<div id="A5">
<sup>5</sup>Department of Neurology, Beth Israel Deaconess Medical Center, Harvard Medical School, Boston, MA 02215, USA</div>
<div id="A6">
<sup>6</sup>Center for Radiological Research, Columbia University, New York, NY 10032, USA</div>
<div id="A7">
<sup>7</sup>Science for Life Laboratory, Department of Molecular Biosciences, the Wenner-Gren Institute, Stockholm University, Stockholm, Sweden</div>
<div id="A8">
<sup>8</sup>KBR, Space Biosciences Division, NASA Ames Research Center, Moffett Field, CA 94035, USA</div>
<div id="A9">
<sup>9</sup>New England Biolabs, Ipswich, MA, USA</div>
<div id="A10">
<sup>10</sup>Institute of Medical Virology and Epidemiology of Viral Diseases, University Hospital, Tubingen, Germany</div>
<div id="A11">
<sup>11</sup>Department of Medicine, Weill Cornell Medicine, New York, NY, USA</div>
<div id="A12">
<sup>12</sup>The Jackson Laboratories, Farmington, CT, USA</div>
<div id="A13">
<sup>13</sup>Department of Medicine, University of Virginia School of Medicine, Charlottesville, VA, USA</div>
<div id="A14">
<sup>14</sup>Department of Biochemistry and Molecular Genetics, University of Virginia School of Medicine, Charlottesville, VA, USA</div>
<div id="A15">
<sup>15</sup>Center for Biotechnology &amp; Interdisciplinary Studies, Rensselaer Polytechnic Institute, Troy, NY 12180, USA</div>
<div id="A16">
<sup>16</sup>Universities Space Research Association, Space Biosciences Division, NASA Ames Research Center, Mountain View, CA 94035, USA</div>
<div id="A17">
<sup>17</sup>McAllister Heart Institute, Department of Pharmacology, and Department of Pathology and Lab Medicine, The University of North Carolina at Chapel Hill, Chapel Hill, NC 27599, USA</div>
<div id="A18">
<sup>18</sup>Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA 02142, USA</div>
<div id="A19">
<sup>19</sup>Department of Radiation Medicine, Georgetown University School of Medicine, Washington DC 20007, USA</div>
<div id="A20">
<sup>20</sup>The WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, NY, USA</div>
<div id="A21">
<sup>21</sup>The Feil Family Brain and Mind Research Institute, Weill Cornell Medicine, New York, NY, USA</div>
<div id="A22">
<sup>22</sup>These authors contributed equally</div>
<div id="A23">
<sup>23</sup>Senior author</div>
<div id="A24">
<sup>24</sup>Lead Contact</div>
<div class="author-notes p">
<div class="fn" id="FN1">
<p id="P1">AUTHOR CONTRIBUTIONS</p>
<p id="P2">Conceptualization, A.B.; Methodology, A.B., M. Mortreux, and P.G.; Formal Analysis, A.B., M.T., and P.G.; Investigation, A.B., S.M., E.C., M.C.-C., C. E.M., and P.G.; miRNA Library Prep and Sequencing, D.N.R., B.S.S., D. Butler., D. Bezdan, B.W.L., E.T.D., and J.T.M.; Analytics and Design for NASA Twin Study miRNA-seq and scRNA-seq, C.R.C., E.A. C.E.M., C.M., A.V., C.D., M. MacKay, S.L., B.F., M.R.F., and M.T.; Resources, M. Mortreux, H.O., M.T., A.-S.S., Y.S.-F., E.B., C.V., C.G.T.T., J.T.M., S.V.C., C.E.M., and S.R.; Writing – Original Draft, S.M., E.C., C.R.C., and A.B.; Writing – Review &amp; Editing, A.B., C.E.M., C.V., J.T.M., M. Mortreux, P.G., S.V.C., C.R.C., S.R., E.C., and S.M.; Figures and Visualization, A.B.; For Graphical Abstract, J.C.S.; For Twins Study Data, F.G.-B., C.R.C., and C.E.M.; Funding Acquisition, A.B. and C.E.M. (for NASA Twin Study); Supervision, A.B.</p>
</div>
<div class="fn" id="CR1">
<sup>*</sup><p class="display-inline">Correspondence: <span>chm2042@med.cornell.edu</span> (C.E.M.), <span>afshin.beheshti@nasa.gov</span> (A.B.)</p>
</div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Issue date 2020 Dec 8.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<p>This is an open access article under the CC BY-NC-ND license (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by-nc-nd/4.0/" rel="noopener noreferrer" target="_blank">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>).</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC8441986  NIHMSID: NIHMS1653539  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33242410/">33242410</a>
</div>
<div aria-label="Related or updated information about this article" class="ra xbox p" role="complementary"><div>The publisher's version of this article is available at <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.celrep.2020.108448" rel="noopener noreferrer" target="_blank">Cell Rep</a>
</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="ABS1"><h2>SUMMARY</h2>
<p id="P3">We have identified and validated a spaceflight-associated microRNA (miRNA) signature that is shared by rodents and humans in response to simulated, short-duration and long-duration spaceflight. Previous studies have identified miRNAs that regulate rodent responses to spaceflight in low-Earth orbit, and we have confirmed the expression of these proposed spaceflight-associated miRNAs in rodents reacting to simulated spaceflight conditions. Moreover, astronaut samples from the NASA Twins Study confirmed these expression signatures in miRNA sequencing, single-cell RNA sequencing (scRNA-seq), and single-cell assay for transposase accessible chromatin (scATAC-seq) data. Additionally, a subset of these miRNAs (miR-125, miR-16, and let-7a) was found to regulate vascular damage caused by simulated deep space radiation. To demonstrate the physiological relevance of key spaceflight-associated miRNAs, we utilized antagomirs to inhibit their expression and successfully rescue simulated deep-space-radiation-mediated damage in human 3D vascular constructs.</p></section><section class="abstract" id="ABS2"><h2>Graphical Abstract</h2>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="graphic file with name nihms-1653539-f0008.jpg" class="graphic" height="639" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0f9/8441986/981237d38097/nihms-1653539-f0008.jpg" width="640"/></p></section><section class="abstract" id="ABS3"><h2>In Brief</h2>
<p id="P5">Malkani et al. uncover the role of circulating microRNAs as both a potential biomarker for health risks associated with spaceflight and a countermeasure to mitigate the damage caused to the body by the space environment.</p></section><section id="S1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p id="P6">Humans and vertebrate organisms exhibit a unique set of biological responses to the space environment, including bone and muscle atrophy (<a aria-describedby="R9" class="usa-link" href="#R9">Burkhart et al., 2019</a>; <a aria-describedby="R25" class="usa-link" href="#R25">Gerbaix et al., 2017</a>), immune dysfunction (<a aria-describedby="R1" class="usa-link" href="#R1">Baqai et al., 2009</a>), neurobehavioral changes (<a aria-describedby="R56" class="usa-link" href="#R56">Ronca et al., 2019</a>), vision impairment (<a aria-describedby="R41" class="usa-link" href="#R41">Mader et al., 2011</a>), and cardiovascular damage (<a aria-describedby="R18" class="usa-link" href="#R18">Elgart et al., 2018</a>). Ionizing radiation (IR) and microgravity are the main biological stressors in low-Earth orbit (LEO), which is the region encompassed by the Earth’s magnetosphere where all crewed missions currently occur. Although exposure to deep space radiation, which is primarily composed of galactic cosmic rays (GCRs), is reduced within LEO due to the magnetosphere, IR still poses a risk to astronauts in LEO, and GCRs are a critical health risk in the upcoming lunar and Mars missions (<a aria-describedby="R70" class="usa-link" href="#R70">Townsend, 2005</a>). Thus, combining LEO experiments with ground-based analogs of spaceflight stressors, including simulated GCRs, allows investigations into the mechanisms behind biological responses to spaceflight.</p>
<p id="P7">MicroRNAs (miRNAs) are short (~22 nucleotides long) sequences of non-coding RNA that can regulate gene expression at the transcriptional and translational level and interact directly with proteins (<a aria-describedby="R74" class="usa-link" href="#R74">Vanderburg and Beheshti, 2020</a>). miRNAs have been implicated as predictors and regulators for several diseases, including cancer (<a aria-describedby="R40" class="usa-link" href="#R40">Macfarlane and Murphy, 2010</a>), neurodegenerative diseases (<a aria-describedby="R37" class="usa-link" href="#R37">Konovalova et al., 2019</a>), and autoimmune disorders (<a aria-describedby="R15" class="usa-link" href="#R15">Chen et al., 2016</a>). miRNAs have also been found to be involved in biological responses to both microgravity (<a aria-describedby="R27" class="usa-link" href="#R27">Girardi et al., 2014</a>) and IR (<a aria-describedby="R61" class="usa-link" href="#R61">Simone et al., 2009</a>). Therefore, identifying miRNAs (and their networks) that are shared between humans and model organisms in response to spaceflight stressors could reveal targets for biomarker and countermeasure development to reduce the associated health risks.</p>
<p id="P8">Previously, we used spaceflown rodent RNA sequencing (RNA-seq) datasets from the NASA GeneLab database to make unbiased predictions about miRNAs potentially involved in regulating the transcriptional response to spaceflight (<a aria-describedby="R3" class="usa-link" href="#R3">Beheshti et al., 2018</a>). The datasets analyzed in that study are described in the <a class="usa-link" href="#S10">STAR Methods</a>. Our analysis identified a set of 13 spaceflight-associated miRNAs that were predicted to drive a systemic response and suggested transforming growth factor (TGF)-β1 as an overarching regulator of their expression (<a aria-describedby="R3" class="usa-link" href="#R3">Beheshti et al., 2018</a>). TGF-β1 is a major regulatory cytokine mediating the progression of cancer (<a aria-describedby="R57" class="usa-link" href="#R57">Rooke and Crosier, 2001</a>), autoimmunity (<a aria-describedby="R43" class="usa-link" href="#R43">Marie et al., 2006</a>), and neuroinflammation (<a aria-describedby="R13" class="usa-link" href="#R13">Cekanaviciute et al., 2014</a>). However, it is important to note that this spaceflight-associated miRNA signature was defined computationally, based on transcriptomic analysis, and not measured experimentally.</p>
<p id="P9">Here, we experimentally validated the predicted spaceflight-associated miRNA signature by quantifying circulating miRNAs in serum from rodents exposed to simulated spaceflight conditions: microgravity, partial gravity, LEO-relevant IR, simulated deep space radiation, and combined exposures to reduced gravity and simulated space radiation, as well as true spaceflight in LEO. We supported spaceflight-stressor-mediated changes in the expression of the predicted miRNAs by performing agnostic miRNA-seq in selected datasets, and our experimental validation also incorporated simulated deep space radiation conditions to help evaluate the extent to which the miRNA signature may be shared by different space environments. Finally, we addressed the human relevance of our findings by analyzing miRNAs released by irradiated human immune cells <em>ex vivo</em> and miRNA expression changes in astronauts from the NASA Twins Study <em>in vivo</em> (<a aria-describedby="R24" class="usa-link" href="#R24">Garrett-Bakelman et al., 2019</a>).</p>
<p id="P10">We expanded our initial panel of 13 candidate miRNAs (<a aria-describedby="R3" class="usa-link" href="#R3">Beheshti et al., 2018</a>) to include 15 target miRNAs with the following justifications: (1) miRNA let-7 was subdivided into two more specific family members (let-7a and let-7c) identified in recent space biology studies (<a aria-describedby="R55" class="usa-link" href="#R55">Režen et al., 2014</a>; <a aria-describedby="R76" class="usa-link" href="#R76">Wei et al., 2018</a>), and (2) an additional miRNA (miR-92a) was added after analyzing astronaut data from the NASA Twins Study. Furthermore, we investigated the potential downstream effects of spaceflight-associated human miRNAs using single-cell RNA (scRNA) sequencing of a targeted gene panel of leukocyte-depleted fractions in the NASA Twins Study (<a aria-describedby="R24" class="usa-link" href="#R24">Garrett-Bakelman et al., 2019</a>). Specifically, we compared gene expression in cells collected within 3 days upon spaceflown twin’s (TW) return to his preflight and postflight samples, as well as control samples taken from the ground control twin (HR). Single-cell transcriptomics also let us determine the individual cell types that display the strongest perturbations in gene expression consistent with miRNA signatures identified in bulk transcriptomics, and single-cell assay for transposase accessible chromatin (scATAC) sequencing data were used to delineate chromatin changes that may have altered the accessibility of the spaceflight-associated miRNAs.</p></section><section id="S2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="S3"><h3 class="pmc_sec_title">Predicted Spaceflight miRNA Signature in Rodents Was Validated Using Simulated Microgravity and Deep Space Radiation</h3>
<p id="P11">To validate the predicted spaceflight-associated miRNA panel, we quantified their changes in response to ground analogs of individual spaceflight stressors in rodent models: simulated microgravity using hindlimb unloading (HU) (<a aria-describedby="R45" class="usa-link" href="#R45">Morey-Holton and Globus, 1998</a>) as well as IR using either gamma rays or major components of galactic cosmic rays—protons and/or high mass and charge <sup>56</sup>Fe particles (deep space relevant exposure) (<a aria-describedby="R44" class="usa-link" href="#R44">Mewaldt, 1994</a>). We exposed male 16-week-old C57BL/6J mice to HU, IR, or their combination and measured the set of selected 15 miRNAs in their serum at 1 or 11 days post-exposure using droplet digital PCR (ddPCR) (<a aria-describedby="R31" class="usa-link" href="#R31">Hindson et al., 2013</a>).</p>
<p id="P12">Euclidean clustering of serum miRNA concentrations separated the miRNAs into two main clusters: (1) deep-space-radiation-associated clusters, which are significantly regulated by <sup>56</sup>Fe IR and <sup>56</sup>Fe IR in combination with HU; and (2) microgravity-associated clusters, primarily altered in HU-only conditions (<a class="usa-link" href="#F1">Figures 1A</a>–<a class="usa-link" href="#F1">1C</a>). An additional subset of miRNAs was significantly increased in mouse serum only, early (1 day) after exposure to the combination of mixed proton/<sup>56</sup>Fe irradiation and HU, but was not affected by individual stressors (<a class="usa-link" href="#F1">Figure 1A</a>). Of note, we have provided a discussion regarding potential hemolysis that might occur with the serum/plasma samples utilized throughout this manuscript in the <a class="usa-link" href="#S10">STAR Methods</a>.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Figure 1. Quantified miRNAs in Serum from Rodents Exposed to Ground Analogs of Spaceflight.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Figure 1." class="graphic" height="695" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0f9/8441986/f25c7f1466ef/nihms-1653539-f0001.jpg" width="1050"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P90">Microgravity and radiation conditions of spaceflight were simulated alone and in combination.</p>
<p id="P91">(A) Quantification of miRNAs in serum from rodents exposed to hindlimb unloading (HU) or normal loading (NL) for 3 days before 2 Gy gamma, 1 Gy proton, 1 Gy 600MeV/n <sup>56</sup>Fe, 2 Gy 600MeV/n<sup>56</sup>Fe, or sham irradiation. HU or NL was continued for another 1 or 11 days after irradiation.</p>
<p id="P92">(B) Fold changes (log<sub>2</sub>) of miRNAs compared to NL sham mice. *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001, two-tailed Student’s t test. The error bars represent SEM.</p>
<p id="P93">(C) t-Distributed Stochastic Neighbor Embedding (t-SNE) plot showing how the significantly regulated miRNAs are associated with each experimental group. Statistical significance is based on comparing each group with the sham irradiated NL mice.</p>
<p id="P94">(D) Top 50 predicted gene targets for all statistically significant miRNAs determined by Cytoscape plugin ClueGo.</p>
<p id="P95">(E) The predicted diseases regulated by the miRNA signature determined through miRNet. The blue nodes represent cancer; green, cardiovascular disease; yellow, neurological diseases; orange, muscle degeneration; purple, digestive issues.</p>
<p id="P96">(F) Gene Ontology (GO) pathways predicted to be regulated by all statistically significant spaceflight-associated miRNAs determined by the DIANA-microT-CDS (v5.0) algorithm.</p>
<p id="P97">(G) Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways predicted to be regulated by all statistically significant spaceflight-associated miRNAs.</p></figcaption></figure><p id="P13">Most miRNAs (with the exception of miR-217) were observed to increase with radiation dose and linear energy transfer (LET; gamma &lt; 150 MeV proton &lt; 600 MeV/n <sup>56</sup>Fe) in mouse serum both at 1 day and 11 days post-IR. At 11 days post-IR, the only significant miRNA increases were observed in IR-only (no HU) mice irradiated with either 1 Gy or 2 Gy 600 MeV/n <sup>56</sup>Fe or 1 Gy protons (<a class="usa-link" href="#F1">Figures 1A</a> and <a class="usa-link" href="#F1">1B</a>). In combination, these results indicate that the miRNA signature is mainly driven by the particle radiation component, which is proportional to dose and LET.</p>
<p id="P14">Based on all miRNAs that were significantly different in spaceflight-associated conditions, we predicted the top 50 targeted genes (<a class="usa-link" href="#F1">Figure 1D</a>), key diseases regulated by the miRNAs (<a class="usa-link" href="#F1">Figure 1E</a>), and significant pathways (<a class="usa-link" href="#F1">Figures 1F</a>, <a class="usa-link" href="#F1">1G</a>, and <a class="usa-link" href="#SD1">S1</a>). The top 50 genes targeted by these miRNAs (<a class="usa-link" href="#F1">Figure 1D</a>) primarily regulate cell-cycle pathways and protein kinase activity, in addition to hematopoiesis and blood cell development (<a class="usa-link" href="#F1">Figures 1F</a> and <a class="usa-link" href="#SD1">S1</a>). The majority of diseases regulated by these miRNAs are related to cancer (<a class="usa-link" href="#F1">Figure 1E</a>), though we also uncovered other diseases associated with spaceflight, including cardiovascular (green circles), neurological (yellow circles) and digestive disorders (purple circles), and muscular degeneration (orange circles) (<a class="usa-link" href="#F1">Figure 1E</a>). Similarly, key significantly regulated pathways include carcinogenesis as well as TGF-β signaling, which supports our previous prediction of TGF-β as a key regulator of spaceflight-associated miRNA functions and implies downstream carcinogenic and immune effects. Finally, the pathways also include adherens junctions, which, combined with cardiovascular disease associations, suggests vascular impacts (<a class="usa-link" href="#F1">Figures 1F</a> and <a class="usa-link" href="#F1">1G</a>).</p>
<p id="P15">To examine the contribution of reduced gravity to the miRNA signature at higher resolution, a partial weight bearing (PWB) Wistar male rat model (<a aria-describedby="R46" class="usa-link" href="#R46">Mortreux et al., 2018</a>) was used to simulate varying degrees of gravitational loading. We observed that decreasing amounts of gravitational loading increased the quantities of miRNAs of interest, suggesting a gravity-dependent dose response (<a class="usa-link" href="#F2">Figures 2A</a>–<a class="usa-link" href="#F2">2C</a>). These results also validated a reduced-gravity-specific miRNA signature in both mice and rats and separated the miRNAs associated with reduced gravity from those associated with exposure to simulated deep space radiation in mice. The only observed exception was miR-217, which did not change. A similar miRNA dose response was observed based on correlation with muscle mass (<a class="usa-link" href="#F2">Figure 2D</a>): the miRNAs of interest increased with reduced muscle mass (which is associated with lower simulated gravity both in this rat model and in astronauts).</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Figure 2. Serum miRNA Quantification from PWB Rats Modeled under Four Different Gravity Conditions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Figure 2." class="graphic" height="506" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0f9/8441986/29db5d2f711c/nihms-1653539-f0002.jpg" width="1050"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P98">(A) Global comparisons of miRNAs from 20%, 40%, 70%, and 100% gravitational loading conditions. The 100% loaded group is further split into FH or NH.</p>
<p id="P99">(B) t-SNE plot reveals distinct separation of the miRNAs between different experimental groups for each gravity condition, as well as a separation between the NH and FH 100% gravity groups.</p>
<p id="P100">(C) Fold changes (log<sub>2</sub>) for miRNA increases compared to 100% gravity with FH group.</p>
<p id="P101">(D) Correlation between serum miRNA increases and physiological muscle mass over different gravity conditions (PWB). Significance indicated in circles. *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001, two-tailed Student’s t test. Circle size is proportional to the correlation coefficient, also indicated by color (legend on right-hand side).</p>
<p id="P102">(E) Global comparison of miRNAs in serum from mice flown on the International Space Station (ISS) for 30 days, sacrificed 4 days after returning to Earth.</p>
<p id="P103">(F) Serum miRNA comparisons for only the 16-week-old mice.</p>
<p id="P104">(G) Serum miRNA comparisons for only the 37-week-old mice.</p>
<p id="P105">(H) t-SNE plot of all mice flown on the ISS shows subtle separation between young and old groups.</p>
<p id="P106">(I and J) t-SNE plot of old mice shows separation between spaceflight and ground mice, while t-SNE plot of young mice fails to show separation.</p>
<p id="P107">(K) Correlation of miRNA quantities and organ weights for all mice flown on the ISS. The significance of each correlation is indicated in the circles. *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001, two-tailed Student’s t test. Circle size is proportional to the correlation coefficient.</p></figcaption></figure><p id="P16">Notably, this model included two control groups of animals exposed to full gravitational loading: no harness (NH) and full harness (FH) (<a class="usa-link" href="#F2">Figures 2A</a> and <a class="usa-link" href="#F2">2B</a>). Both conditions had been included in prior studies, and no functional or phenotypical differences were recorded, although there was a difference in miRNA expression between NH and FH 100% gravity groups (<a class="usa-link" href="#F2">Figures 2A</a> and <a class="usa-link" href="#F2">2B</a>), possibly due to a trend in reduction in muscle mass in FH controls (not significant). Therefore, further comparison between groups included only the FH control (<a class="usa-link" href="#F2">Figure 2C</a>).</p></section><section id="S4"><h3 class="pmc_sec_title">Predicted Spaceflight miRNA Signature Was Partially Observed in Serum from Mice Exposed to LEO</h3>
<p id="P17">Next, we investigated whether the spaceflight-environment-linked miRNA signature is distinguishable in true spaceflight samples, using serum from young and old female BALB/cAnNTac mice flown to the International Space Station (ISS) on Rodent Research Mission 8. In contrast with the simulated spaceflight cohorts, these mice experienced short-lasting hypergravity during takeoff and landing in addition to microgravity while in orbit, spent 4 days reacclimating on ground after the flight before sample collection, and received a significantly lower dose of IR than simulation experiments: 0.01128 Gy over approximately 30 days on the ISS.</p>
<p id="P18">Although the combined mouse cohort did not exhibit any significantly increased miRNAs compared to basal (no spaceflight, normal housing conditions) controls (<a class="usa-link" href="#F2">Figure 2E</a>), when the samples were separated by age (<a class="usa-link" href="#F2">Figures 2F</a> and <a class="usa-link" href="#F2">2G</a>), we observed clustering between spaceflight and basal controls in the older group (31–37 weeks at sacrifice) corresponding to middle-age in humans, which is more representative of typical astronaut cohorts, and no detectable clustering in the younger group (10–16 weeks at sacrifice) (<a class="usa-link" href="#F2">Figures 2H</a>–<a class="usa-link" href="#F2">2J</a>). Considering that these mice had 4 days to reacclimate to ground conditions before the samples were collected, these results may suggest that it takes longer for serum miRNA levels to return to basal levels in older mice, while younger mice recover faster. This finding is consistent with our prior results from simulated spaceflight studies, in which miRNA levels in irradiated mice start to return to control by 11 days following an over-100-fold-higher dose of IR than received by the spaceflown mice (<a class="usa-link" href="#F1">Figure 1A</a>).</p>
<p id="P19">Another notable result from this spaceflight experiment is the correlation between organ weights and miRNA concentration in spaceflight mice and basal controls. Normalized to total body weight, adrenal gland weight increases with concentration of the majority of quantified miRNAs, and, conversely, brain weight decreases with increasing miRNA concentrations. While these correlations are not statistically significant, they suggest the possibility of lingering impacts on specific organs (<a class="usa-link" href="#F2">Figure 2K</a>). Increases in adrenal gland weight might be related to chronic stress, since the adrenal gland regulates stress responses (<a aria-describedby="R71" class="usa-link" href="#R71">Ulrich-Lai et al., 2006</a>) that have been previously associated with increased systemic corticosterone levels during spaceflight (<a aria-describedby="R50" class="usa-link" href="#R50">Pecaut et al., 2017</a>). Meanwhile, a volumetric reduction of brain features has been observed in cosmonauts (<a aria-describedby="R51" class="usa-link" href="#R51">Pechenkova et al., 2019</a>), though the physiological relevance of these findings remains to be elucidated. Since systemic increases in corticosterone levels also lead to reduced brain weight (<a aria-describedby="R16" class="usa-link" href="#R16">Coburn-Litvak et al., 2004</a>), chronic stress might be an underlying factor behind both adrenal gland and brain results, and future studies are needed to examine miRNA as its potential biomarker.</p></section><section id="S5"><h3 class="pmc_sec_title">Spaceflight-Relevant miRNA Signature Was Also Identified in Agnostic Sequencing Study of Simulated-Spaceflight-Exposed Mice</h3>
<p id="P20">To understand the specificity and variation of spaceflight-relevant miRNA levels within the global miRNA population, miRNA-seq was performed on tissue samples from 16-week-old female C57BL/6J mice after exposure to simulated spaceflight conditions: normal loading (NL) or HU for 2 weeks prior to irradiation with 5 Gy gamma, 1 Gy simulated solar particle event (simSPE), 0.5 Gy simulated simplified galactic cosmic rays (SimGCRSim), or sham control. SimGCRSim consists of five ions: protons at 1000 MeV, <sup>28</sup>Si at 600 MeV/n, <sup>4</sup>He at 250 MeV/n, <sup>16</sup>O at 350 MeV/n, <sup>56</sup>Fe at 600 MeV/n, and protons at 250 MeV (<a aria-describedby="R62" class="usa-link" href="#R62">Simonsen et al., 2020</a>). This dose of radiation is equivalent to what an astronaut is predicted to receive in deep space during a Mars mission, though it is modeled as a single exposure over 25 min instead of the actual chronic exposures over 1.5 years. Simulated solar particle event (SimSPE) irradiation consists of protons from 50 MeV to 150 MeV. Twenty-four hours after irradiation, the mice were sacrificed, and miRNA-seq was performed on tissue samples from the plasma, heart, liver, and soleus muscle.</p>
<p id="P21">We observed highly variable miRNA expression profiles between organs, with the exception of heart and soleus muscle (<a class="usa-link" href="#F3">Figures 3A</a> and <a class="usa-link" href="#F3">3B</a>). This variation underscores the importance of establishing a circulating systemic miRNA signature that can be measured in plasma, which is the most accessible source for biomarker evaluation in spaceflight conditions. Additionally, the high correlation between miRNA expression changes in heart and soleus muscle unsurprisingly suggests that the damage in these muscle tissues may be driven by similar changes in miRNAs (<a class="usa-link" href="#F3">Figures 3A</a> and <a class="usa-link" href="#F3">3B</a>). Notably, miRNA-seq profiles from plasma also are more heterogeneous than those from individual organs, suggesting that plasma miRNA quantification might allow a more finely tuned spaceflight signature that could differentiate between different types of exposures or provide personalized risk assessment (<a class="usa-link" href="#F3">Figures 3A</a> and <a class="usa-link" href="#F3">3B</a>).</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Figure 3. miRNA-seq of Heart, Liver, Plasma, and Soleus Muscle from Mice Exposed to Simulated Spaceflight Stressors.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Figure 3." class="graphic" height="912" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0f9/8441986/2e55965758db/nihms-1653539-f0003.jpg" width="1050"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P108">(A) Comparison of miRNA expression profiles between organs.</p>
<p id="P109">(B) Combined and individual t-SNE plots for significantly expressed miRNAs in each organ.</p>
<p id="P110">(C) Overlap of significant differentially expressed miRNAs between organs for GCR HU and sham NL and for SPE HU versus sham NL. The center, dark gray region indicates significant miRNAs shared by all organs.</p>
<p id="P111">(D) Top 20 genes regulated by the four common miRNAs differentially expressed in GCR HU versus sham NL.</p>
<p id="P112">(E) Pathways that are significantly regulated by the top 20 genes shown in (D).</p>
<p id="P113">(F) miRNA fold changes between HU and NL groups for each irradiation condition. miRNAs from our proposed signature are included, as well as the four additional miRNAs implicated by miRNA-seq in (C).</p>
<p id="P114">(G) C2 pathways analysis from miRNA-seq data shows pathways that are being suppressed. The larger the circle, the more significant the effect, and darker purple indicates a larger magnitude of suppression. miRNAs highlighted in yellow or blue were included in our predicted spaceflight signature. miRNAs highlighted in yellow are known to be involved in spaceflight-induced cardiac damage.</p></figcaption></figure><p id="P22">Comparison of significantly regulated miRNAs identified four miRNAs that are increased in all organs in the GCR+HU (“combined spaceflight stressors”) group compared to sham NL (“no spaceflight stressors” control) (<a class="usa-link" href="#F3">Figure 3C</a>). Therefore, we decided to focus on the GCR+HU irradiated group for further analysis. We computed the top 20 genes regulated by the four common differentially expressed miRNAs (<a class="usa-link" href="#F3">Figure 3D</a>) as well as the pathways that were significantly regulated by these genes (<a class="usa-link" href="#F3">Figure 3E</a>) and found potential cardiovascular as well as systemic circulatory targets (<a class="usa-link" href="#F3">Figure 3E</a>).</p>
<p id="P23">We next quantified miRNA fold changes between GCR/SPE/Gamma+HU and sham NL conditions, focusing on the panel of spaceflight-associated miRNAs (top 15) as well as the four additional miRNAs that were found to be shared among all tissues for GCR+HU versus NL by miRNA-seq (bottom 4) (<a class="usa-link" href="#F3">Figure 3F</a>). Additional data is available in a companion paper (<a aria-describedby="R80" class="usa-link" href="#R80">Wuu et al., 2020</a>). We observed trends or significance for multiple spaceflight miRNAs, and the same miRNA family members differentially expressed across multiple tissues. For example, for miRNA-503-5p, expression was decreased in the liver, soleus, and heart but strongly increased in plasma (<a class="usa-link" href="#F3">Figure 3F</a>), indicating potential organ-dependent miRNA changes with the key systemic stressor response stemming from the circulating miRNAs. MiR-503 belongs to the miR-16 extended family (<a aria-describedby="R11" class="usa-link" href="#R11">Caporali and Emanueli, 2011</a>; <a aria-describedby="R75" class="usa-link" href="#R75">Wang et al., 2019</a>), which is one of the miRNAs predicted to be involved in the spaceflight response (<a class="usa-link" href="#F1">Figure 1A</a>) and impacts cardiovascular diseases through the inhibition of angiogenesis (<a aria-describedby="R5" class="usa-link" href="#R5">Besnier et al., 2019</a>; <a aria-describedby="R11" class="usa-link" href="#R11">Caporali and Emanueli, 2011</a>).</p>
<p id="P24">Finally, we performed pathway analysis utilizing the miRNA-seq data with C2 curated gene sets from MSigDB on the miRNA-seq data and displayed all significantly regulated pathways (<a class="usa-link" href="#F3">Figure 3G</a>) for the liver and soleus muscle. Data for the plasma and heart tissues are in companion papers and show similar results (<a aria-describedby="R49" class="usa-link" href="#R49">Paul et al., 2020</a>; <a aria-describedby="R80" class="usa-link" href="#R80">Wuu et al., 2020</a>), and the pathways that showed suppression (false discovery rate [FDR]-adjusted p value &lt; 0.25; <a class="usa-link" href="#F3">Figure 3G</a>) have been previously implicated in cardiovascular adaptations to spaceflight (<a aria-describedby="R3" class="usa-link" href="#R3">Beheshti et al., 2018</a>, <a aria-describedby="R4" class="usa-link" href="#R4">2019</a>). These specific pathways were selected for the countermeasure experiments (below). This experiment demonstrates that although the individual response of the miRNAs might not be significantly regulated, the global response of miRNAs on downstream gene targets reveals regulation by miRNAs belonging to the spaceflight signature.</p></section><section id="S6"><h3 class="pmc_sec_title">Spaceflight-Associated Rodent miRNA Signature Was Partially Shared by Human Responses <em>Ex Vivo</em> and <em>In Vivo</em>
</h3>
<p id="P25">We next show that spaceflight-associated miRNA signature is partially conserved in human samples that had been exposed to real or simulated space environment conditions. First, we analyzed human peripheral blood mononuclear cell (PBMC) responses to 0.3 Gy 600 MeV/n <sup>56</sup>Fe particles that form a biologically significant component of GCRs, using PBMCs isolated from 12 healthy human (non-astronaut) donors. Thus, this experiment simulated deep-space-relevant radiation, without simulating microgravity. We observed an increase in spaceflight-associated miRNAs in the supernatant of irradiated PBMCs 4 hours, post-irradiation (<a class="usa-link" href="#F4">Figures 4A</a>–<a class="usa-link" href="#F4">4D</a>), as well as a positive correlation with increasing donor BMI and cell number (<a class="usa-link" href="#F4">Figures 4B</a> and <a class="usa-link" href="#F4">4D</a>; <a class="usa-link" href="#SD1">Table S3</a>) .</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Figure 4. Quantification of miRNAs in Human Samples Exposed to Spaceflight or IR.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Figure 4." class="graphic" height="528" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0f9/8441986/4f54c41b56e1/nihms-1653539-f0004.jpg" width="1050"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P115">(A) Comparison of human peripheral blood mononuclear cell (PBMC) samples from 12 subjects 4 h post-irradiation with 0.3 Gy or sham (0 Gy) <sup>56</sup>Fe irradiation.</p>
<p id="P116">(B) Correlation between miRNA quantities and cell phenotypes 4 h post-irradiation. The significance of each correlation is indicated in the circles. *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001, two-tailed Student’s t test. Circle size is proportional to the correlation coefficient.</p>
<p id="P117">(C) General additive model fit of the miRNA quantities as a function of dose.</p>
<p id="P118">(D) t-SNE plot of the samples clustering as a function of BMI based on miRNA concentration.</p>
<p id="P119">(E and F) NASA Twins Study data for (E) unsorted PBMCs and (F) LD cells in comparison to the spaceflight miRNA signature predictions.</p></figcaption></figure><p id="P26">Finally, in order to assess the impact of the real spaceflight environment on human cells, we analyzed miRNA-sequence data on total PBMC and lymphocyte-depleted (LD) samples from the NASA Twins Study. This study involved actual LEO (low-LET radiation and microgravity) exposure for 340 days for a total of 146.34 mSv (or 0.1463 Gy). Unsorted PBMCs isolated during and after LEO spaceflight showed persistent changes in miRNA expression for the astronaut in space, which partially follows the predicted spaceflight signature (<a class="usa-link" href="#F4">Figure 4E</a>). Similarly, LD cells sampled before and after flight also indicated persistent changes in miRNA expression, though not all miRNAs behaved as predicted by the spaceflight signature (<a class="usa-link" href="#F4">Figure 4F</a>). Notably, the spaceflown twin (TW) had consistently higher expression of the miRNA target genes compared to the HR even at baseline/preflight, despite having higher levels of miRNAs as well (<a class="usa-link" href="#F4">Figures 4E</a> and <a class="usa-link" href="#F4">4F</a>); thus, TW pre-flight levels were considered a better control than ground control twin (HR) pre-flight levels. This difference in miRNA expression is not surprising, due to known discordant miRNA expression profiles in twins (<a aria-describedby="R81" class="usa-link" href="#R81">Xiao et al., 2019</a>).</p>
<p id="P27">Of the miRNAs found in the irradiated PBMC panel, only miR-92a-3p was significantly downregulated and let-7c-5p was upregulated in the TW miRNA-sequence data (<a class="usa-link" href="#F4">Figure 4F</a>). However, the downstream targets of the irradiation-associated miRNAs were significantly decreased between preflight and postflight samples, except the targets of hsa-miR-92a-3p and hsa-miR-26a-5p. This suggests that there was an increase in some irradiation-associated miRNA functions in TW, and the effects of miR-92a-3p and miR-26a-5p were more acute than the other miRNAs. Similar to the lack of response of miR-217 in animal models based on ddPCR results, miR-217 only had three targets within the targeted panel, none of which were significantly changed by spaceflight, indicating that miR-217, although implicated by original unbiased <em>in silico</em> predictions, does not respond to the stress of the space environment.</p>
<p id="P28">We next analyzed the changes in miRNA expression across different blood cell populations that were associated with spaceflight-mediated miRNA differences in the same NASA Twins Study. Specifically, we conducted targeted scRNA-seq and scAb-seq (epitope mapping) of LD Twins Study samples across multiple time points. In total, 23,408 cells were sequenced across 12 samples: 6 TW and 6 HR (<a class="usa-link" href="#SD1">Figure S2</a> and <a class="usa-link" href="#SD2">Table S4</a>). Cell types were identified on a per cluster basis by both RNA and epitope levels (<a class="usa-link" href="#F5">Figure 5A</a>). Module scores for the targets of the subset of spaceflight-associated miRNAs that were found to be upregulated in irradiated human PBMCs were calculated using Seurat and compared among TW preflight, return, and postflight samples (<a class="usa-link" href="#F5">Figures 5B</a>–<a class="usa-link" href="#F5">5N</a>).</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Figure 5. Spaceflight-Associated miRNA Target Genes Are Downregulated upon TW Return and Postflight Compared to Preflight, with Individual Cell Types Being Differentially Affected.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Figure 5." class="graphic" height="1050" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0f9/8441986/eaba9ec9473e/nihms-1653539-f0005.jpg" width="638"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P120">(A) Uniform manifold approximation and projection (UMAP) plot of 23,408 cells grouped by cell-type-identified clusters.</p>
<p id="P121">(B–N) Violin plots with mean and first and third quartiles of the indicated miRNA target gene module scores. Cells were separated into four groups: cells from HR GD114, GD138, GD182, GD3, GD-66, and GD66 as Ground; TW L-162 and L-148 days as Preflight; TW R0 as Return; and TW R36 and R191 as Postflight. Differences in expression of miRNA target gene modules scores was tested by Wilcoxon rank sum test. *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001.</p>
<p id="P122">(O–W) Dot plots colored by average expression and size based on percent expression of the spaceflight-associated mRNA target gene module scores of cell types grouped by Ground, Preflight, Return, and Postflight.</p></figcaption></figure><p id="P29">When the cell population was split by cell type, we observed that the changes in miRNA-mediated gene expression were highly variable across cell types (<a class="usa-link" href="#F5">Figure 5O</a>). Dividing each cell type into ground, preflight, return, and postflight groups demonstrated that individual cell types were differentially affected by spaceflight-associated miRNAs. For example, CD14<sup>+</sup> and CD16<sup>+</sup> monocytes responded differently to hsa-miR-92a-3p: CD14<sup>+</sup> monocytes showed <em>decreased</em> expression of hsa-miR-92a-3p targets, while CD16<sup>+</sup> monocytes expressed <em>higher</em> levels of hsa-miR-92a-3p target genes upon return compared to either preflight or postflight. Progenitor (circulating hematopoietic stem) cells stood out in this analysis, as expression of the spaceflight-associated miRNA target genes increased specifically at the return time point. These differences among cell types indicate that immune cell composition may play a role in the downstream effects of spaceflight-associated miRNA increases and suggest return to Earth as a major stressor affecting miRNA expression.</p></section><section id="S7"><h3 class="pmc_sec_title">The Increase in Expression of Spaceflight-Associated miRNAs Is Not Driven by Epigenetic Changes</h3>
<p id="P30">To determine whether the changes in miRNA expression were due to spaceflight-induced chromatin changes, we used scATAC-seq on 11,344 cells (<a class="usa-link" href="#SD1">Figure S3</a> and <a class="usa-link" href="#SD3">Table S5</a>). SnapTools and SnapATAC were used to create 5-kb bins across the genome and determine genome accessibility based on ATAC activity per cell, and then genes and miRNAs were assigned the ATAC activity based on the overlapping bin. ATAC activity is associated with chromatin accessibility, which can be used to determine whether the miRNAs upregulated in irradiated PBMCs were differently accessible in TW during spaceflight (<a class="usa-link" href="#F6">Figure 6B</a>). Of note, open chromatin peaks were observed at let-7c-5p, miR-34a-5p, miR-125b-5p, let-7a-5p, and miR-146a-5p. Furthermore, both control PBMCs and HR ground PMBCs displayed consistent ATAC peaks across the majority of the miRNA loci. Together, this suggests that the chromatin around these miRNA loci is open but not strongly affected by spaceflight. In addition, we assessed ATAC activity in different cell types (<a class="usa-link" href="#F6">Figures 6C</a> and <a class="usa-link" href="#F6">6D</a>). The module score and dot plots of the miRNAs upregulated in irradiated PBMCs demonstrate that CD14<sup>+</sup> monocytes and pre-dendritic cells (pDCs) show more miRNA accessibility that other cell populations (<a class="usa-link" href="#F6">Figures 6C</a> and <a class="usa-link" href="#F6">6D</a>), indicating that chromatin states depend more on immune cell type than spaceflight.</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">Figure 6. Spaceflight Does Not Induce Epigenetic Changes around Spaceflight-Associated miRNA.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Figure 6." class="graphic" height="1050" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0f9/8441986/a66981f91045/nihms-1653539-f0006.jpg" width="942"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P123">(A) UMAP plot of 11,344 cells grouped by cell type predicted by Seurat’s transfer anchor function from the 10× PBMC dataset (see <a class="usa-link" href="#S26">Method Details</a>).</p>
<p id="P124">(B) Dot plot colored by ATAC activity and sized based on percentage ATAC activity at the 5-kb bins associated with spaceflight-associated miRNA. Cells were separated into five groups: cells from the control (non-LD) PBMCs as Control, HR GD-66 and GD125 as Ground, TW L-71 as Preflight, TW R3 as Return, and TW R36 as Postflight.</p>
<p id="P125">(C) Dot plot of spaceflight-associated miRNA module scores with cells grouped by cell type across all samples.</p>
<p id="P126">(D) Dot plot of the individual spaceflight-associated miRNA ATAC activity with cells grouped by cell type.</p>
<p id="P127">(E) Composition analysis per group based after removal of B and T cells across both scATAC and scRNA analysis.</p>
<p id="P128">(F and G) Dot plots of ATAC activity at miRNA upregulated (F) or downregulated (G) in the LD miRNA-seq.</p></figcaption></figure><p id="P31">We next investigated immune cell composition at different time points during flight, specifically the proportion of CD14<sup>+</sup> monocytes. Due to a low number of cells in the TW preflight sample, data from ATAC-seq and scRNA-seq experiments were combined to determine the immune cell distribution in LD fractions. (<a class="usa-link" href="#F6">Figure 6E</a>). Notably, the TW return samples had the highest proportion of CD14<sup>+</sup> monocytes. This result, combined with the scRNA-seq of downstream targets, suggests that miRNA expression during spaceflight is regulated not by chromatin accessibility, but by immune cell distribution, and is particularly high in CD14<sup>+</sup> monocytes. Finally, we evaluated whether specific ATAC peak changes correlated with expression levels from the Twins Study miRNA-seq (<a class="usa-link" href="#F6">Figures 6F</a> and <a class="usa-link" href="#F6">6G</a>). However, ATAC activity was observed in the HR samples for upregulated miRNAs, indicating that the chromatin was already open prior to spaceflight.</p></section><section id="S8"><h3 class="pmc_sec_title">Selective Inhibition of Spaceflight-Associated miRNAs Reduces Cardiovascular Damage in a Human Tissue Model after Exposure to Simulated Deep Space Radiation</h3>
<p id="P32">Finally, we developed and utilized a mature human 3D microvascular tissue model (<a aria-describedby="R28" class="usa-link" href="#R28">Grabham et al., 2011</a>; <a aria-describedby="R68" class="usa-link" href="#R68">Templin et al., 2016</a>; <a aria-describedby="R80" class="usa-link" href="#R80">Wuu et al., 2020</a>) seeded with human umbilical vein endothelial cells (HUVECs) to examine the physiological effects of the candidate miRNAs on structural cardiovascular damage caused by heavy ion irradiation (<a aria-describedby="R28" class="usa-link" href="#R28">Grabham et al., 2011</a>; unpublished data). Forty-eight hours after irradiation with 0.5 Gy SimGCRSim, several of the candidate miRNAs showed marked increases compared to controls (<a class="usa-link" href="#F7">Figure 7A</a>), though the increases were not significant, possibly due to low sample numbers.</p>
<figure class="fig xbox font-sm" id="F7"><h4 class="obj_head">Figure 7. miRNA Changes and Impacts in a 3D Microvascular Tissue Model.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Figure 7." class="graphic" height="549" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0f9/8441986/d42c0010d809/nihms-1653539-f0007.jpg" width="1050"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F7/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p id="P129">(A) Quantification of miRNAs from a 3D culture of mature human microvessels with human umbilical vein endothelial cells (HUVECs) irradiated with 0.5 Gy of simplified simulated galactic cosmic rays (SimGCRSim) compared to sham irradiated samples 48 h after irradiation. All candidate miRNAs were examined, but only the miRNAs that seemed to respond differently for each group are shown. The error bars represent SEM.</p>
<p id="P130">(B and C) Mature microvessels fixed and fluorescently stained with 5-(4,6-dichlorotriazinyl) aminofluorescein (DTAF) (<a class="usa-link" href="#S26">Method Details</a>) 48 h after SimGCRSim irradiation with or without antagomir-induced inhibition of miR-125b, miR-16, and let-7a starting at 24 h prior to irradiation. Scrambled version of the antagomir was used a vehicle control.</p>
<p id="P131">(D) Summary of all the fold-change values for all experiments utilized in this manuscript (including both ddPCR and miRNA-seq data).</p>
<p id="P132">(E) The summary of the overall impact of the miRNAs on the gene targets determined in <a class="usa-link" href="#F3">Figure 3D</a> through pathway analysis on the miRNA-seq data. The arrows indicate the degree of up- or downregulation, and the colors of the arrows indicate the significance (i.e., p values). The gray-shaded regions represent the 11-day time points after irradiation. The red-shaded miRNAs represent the miRNAs used for the antagomir experiments in (B) and (C).</p></figcaption></figure><p id="P33">Three of the miRNAs (miR-125, miR-16, and let-7a) that were increased by SimGCRSim irradiation were also previously predicted to be associated with space-radiation-induced cardiovascular damage (<a aria-describedby="R4" class="usa-link" href="#R4">Beheshti et al., 2019</a>) and were chosen to be inhibited using antagomirs (<a aria-describedby="R39" class="usa-link" href="#R39">Krützfeldt et al., 2005</a>). Further data showing that these miRNAs are associated with disrupted angiogenesis following IR have been reported elsewhere (<a aria-describedby="R80" class="usa-link" href="#R80">Wuu, et al., 2020</a>). In addition, these miRNAs show consistent presence and impact on downstream functions in the space environment (i.e., Twins Study data) compared to all other miRNAs. Antagomirs were applied 24 h before irradiation, and morphology of the tissue constructs was assessed 48 h after irradiation with 0.5 Gy SimGCRSim. We observed irradiation-mediated loss of vascularization because the microvessels had shrunk or collapsed, indicating that 0.5 Gy SimGCRSim is sufficient to disrupt microvessels. This impact of SimGCRSim exposure was reversed by application of the three antagomirs 24 h before irradiation, while a scrambled antagomir control did not induce the rescue phenotype (<a class="usa-link" href="#F7">Figures 7B</a> and <a class="usa-link" href="#F7">7C</a>). Based on the summary of the of spaceflight-stressor-induced changes across all miRNAs (<a class="usa-link" href="#F7">Figure 7D</a>) and the miRNAs that have global significant impact on the downstream gene targets determined in <a class="usa-link" href="#F3">Figure 3D</a> (<a class="usa-link" href="#F7">Figure 7E</a>), it is apparent that these three miRNAs have a uniform trend across all different conditions (red shaded miRNAs in <a class="usa-link" href="#F7">Figures 7D</a> and <a class="usa-link" href="#F7">7E</a>). Thus, microvessel structure was protected by selectively inhibiting three miRNAs that are the most universally associated with spaceflight responses, proving that the vascular impairments caused by simulated deep space radiation are at least partially mediated through miRNAs and in the future might be rescued using antagomirs in human tissue constructs. Future steps to further explore the impact of this potential countermeasure will be performed in murine models and are currently ongoing. Although the results with the antagomirs are promising in the <em>in vitro</em> model, we still cannot account for the complexity in an <em>in vivo</em> model and the efficacy of this treatment.</p></section></section><section id="S9"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p id="P34">In summary, we have validated the previously predicted panel of candidate spaceflight-relevant miRNAs using a combination of spaceflight, simulated microgravity, partial gravity, and simulated deep space radiation conditions in male and female mouse and rat <em>in vivo</em>, astronaut, and human <em>ex vivo</em> models. The majority of rodent serum miRNAs followed the pattern of dose-dependent increase in response to IR, with the notable exception of miR-217 (<a class="usa-link" href="#F7">Figures 7D</a> and <a class="usa-link" href="#F7">7E</a>). We evaluated the applicability of this miRNA signature to humans by examining miRNA content in the supernatant of primary human immune cell samples exposed to simulated deep space radiation, as well as immune cells from the NASA Twins Study, and observed partial conservation across species. The functional significance of key miRNAs was predicted by analyzing miRNA-dependent changes in gene expression at the single-cell level in the NASA Twins Study and experimentally evaluated utilizing antagomirs to mitigate vascular damage caused by simulated deep space radiation in a 3D human microvascular tissue model. When viewing these results independently, it may appear that some level of variability occurs for the miRNA between the different conditions (<a class="usa-link" href="#F1">Figures 1</a>, <a class="usa-link" href="#F2">2</a>, <a class="usa-link" href="#F3">3</a>, and <a class="usa-link" href="#F4">4</a>), but when observing all conditions together, this variability is in fact accounted for, based on which component of the space environment is being observed. A summary of how all the miRNA patterns and downstream targets demonstrate the consistency over all the models utilized for this study is shown in <a class="usa-link" href="#F7">Figures 7D</a> and <a class="usa-link" href="#F7">7E</a>. For example, certain miRNAs (e.g., let-7a-5p, let-7c-5p, miR-223-3p) are more significantly regulated with microgravity experiments as opposed space radiation experiments, while other miRNAs (e.g., miR-16-5p) are consistently being expressed in all conditions (<a class="usa-link" href="#F7">Figure 7D</a>).</p>
<p id="P35">Collectively, our samples identify two spaceflight miRNA signatures, the first associated with reduced gravity and LEO and the second representing the responses to simulated deep space radiation. Specifically, gamma-irradiated mice, sham HU mice, PWB rats, ISS-flown mice, and the Twins Study data collectively display a LEO/microgravity miRNA signature, while 600 MeV/n <sup>56</sup>Fe and proton-irradiated mice, human PBMCs, and human 3D microvascular tissue models exposed to 600 MeV/n <sup>56</sup>Fe ions and simplified simulated GCRs show a deep-space-radiation-dependent signature (<a class="usa-link" href="#F7">Figure 7D</a>). Notably, significant overlaps exist between both signatures, indicating a miRNA panel that might be particularly relevant to future lunar and Mars missions that will combine both stressors. The absence of significant differences in miRNA-217 expression in these datasets serves as a negative control and demonstrates that the responses that had been observed in the remaining candidate miRNAs are unlikely to be caused by universal miRNA increases induced by an arbitrary stressor.</p>
<p id="P36">A miRNA signature unique to the spaceflight response has potential uses as a biomarker of systemic biological damage as well as a source for targets for mitigating the health risks caused by spaceflight and deep space radiation. While specific miRNAs have previously been implicated in spaceflight-associated biological damage (<a aria-describedby="R27" class="usa-link" href="#R27">Girardi et al., 2014</a>; <a aria-describedby="R61" class="usa-link" href="#R61">Simone et al., 2009</a>; <a aria-describedby="R69" class="usa-link" href="#R69">Teodori et al., 2019</a>), recent systems biology studies indicate that using an entire miRNA network as a biomarker can provide a more robust association with biological changes (<a aria-describedby="R2" class="usa-link" href="#R2">Beheshti et al., 2017</a>). In addition, we have observed that individual components of the space environment have unique miRNA signatures: some miRNAs are associated with the microgravity component of real or simulated spaceflight, others are associated with exposure to deep space radiation, and still others are only significantly increased after exposure to both, suggesting that different miRNA-based biomarkers would be applicable to different conditions, with the latter being particularly important in deep space exploration. The sensitivity of miRNA responses is underscored by the fact that the magnitude of miRNA increases appears to be dose dependent for both radiation and microgravity, as observed in the mouse simulated space environment model and the PWB rat model, respectively. Similarly, for the IR component, the signature is also LET dependent, as we report more significant miRNA changes after 600 MeV/n <sup>56</sup>Fe compared to proton or gamma irradiation, which suggests its potential relevance for future deep space missions. Finally, conserved increases of key miRNAs across species, sexes, and experiments emphasizes the reliability of the miRNA signature.</p>
<p id="P37">Since the majority of the data in this paper are from males, there can be sex-specific differences that are not captured here. It has been shown that certain miRNAs and family members of miRNAs are significantly expressed in one sex over the other, in normal and cancerous (e.g., The Cancer Genome Atlas) tissues (<a aria-describedby="R29" class="usa-link" href="#R29">Guo et al., 2017</a>). For example, miR-503 is linked to female-related cardiovascular disease (<a aria-describedby="R21" class="usa-link" href="#R21">Florijn et al., 2018</a>), indicating that miRNAs in the same family have similar behaviors in different sexes while still linked to the original family of miRNAs. Future research will need to explore the sex differences and the spaceflight signatures for specific miRNAs as well as related miRNA family members.</p>
<p id="P38">Of note, miRNA signatures may be applied for improving astronaut health by providing the first step toward identifying a network of circulating regulatory molecules that are activated by spaceflight. Identifying, validating and targeting such systemic regulatory networks, as opposed to addressing symptomatic damage in isolated organs and tissues, may accelerate our ability to mitigate the damage to several organ systems at once. Additionally, this approach may be used for preemptive risk reduction.</p>
<p id="P39">Indeed, we experimentally demonstrated the countermeasure potential of miRNAs by using antagomirs to inhibit three miRNAs associated with spaceflight cardiovascular health risks in human 3D tissue models. In general, antagomirs are a powerful tool for preventing the downstream effects of miRNA, which can be extensive due to the ability of a single miRNA to interact with hundreds of genes at the transcriptional, translational, and protein levels (<a aria-describedby="R39" class="usa-link" href="#R39">Krützfeldt et al., 2005</a>). Antagomirs have been shown to effectively inhibit their miRNA targets <em>in vivo</em> in mouse models of disease; however, more research is required for understanding their off-target effects and tissue specificity (<a aria-describedby="R34" class="usa-link" href="#R34">Ishida and Selaru, 2013</a>). In our model, antagomirs were applied as a countermeasure prior to deep-space-relevant irradiation and successfully prevented microvascular damage. Thus, our suggested network of miRNAs and their implied targets not only provides mechanistic insight into spaceflight-induced organ damage, but also may be a valuable platform for preventing the adverse health effects of spaceflight. However, in our experiments, the antagomir therapy was only applied against the cardiovascular component of the miRNA signature due to the model only recapitulating human microvasculature. It would be important to measure the effects of miRNA inhibition of the rest of the miRNA signature in a model that more fully captures spaceflight-mediated immune dysfunction and cancer progression as well as <em>in vivo.</em></p>
<p id="P40">The panel of spaceflight-associated miRNAs has been predicted to disrupt cell signaling and metabolic pathways, at least in part via their interactions with TGF-β1 (<a aria-describedby="R3" class="usa-link" href="#R3">Beheshti et al., 2018</a>). TGF-β1 is a pleiotropic cytokine that has been shown previously to be significantly modulated during spaceflight conditions (<a aria-describedby="R35" class="usa-link" href="#R35">Jessup et al., 2000</a>; <a aria-describedby="R77" class="usa-link" href="#R77">Westerlind and Turner, 1995</a>), and its expression has been successfully altered as a therapeutic approach in cancer and inflammation. However, TGF-β1 is involved in a wide variety of endothelial and immune cell processes, which makes it a risky target (<a aria-describedby="R47" class="usa-link" href="#R47">Otten et al., 2010</a>). Furthermore, given that TGF-β1 has been found to exert both tumor-suppressive and tumor-promoting effects at different stages of tumorigenesis (<a aria-describedby="R33" class="usa-link" href="#R33">Ikushima and Miyazono, 2010</a>), the ability of miRNAs to target the TGF-β1 pathway at multiple specific sites may suggest a more effective and dynamic modulation of TGF-β1 effects than simple overexpression or inhibition (<a aria-describedby="R10" class="usa-link" href="#R10">Butz et al., 2012</a>). Thus, antagomir-mediated miRNA inhibition may allow more context-specific TGF-β1 pathway modulation and may be a promising countermeasure for spaceflight applications as well.</p>
<p id="P41">The interactions between the proposed miRNA candidates and TGF-β1 have important consequences for inflammation during different stages of spaceflight, particularly during landing. miR-145 and miR-146a, which are both associated with a pro-inflammatory response (<a aria-describedby="R65" class="usa-link" href="#R65">Tahamtan et al., 2018</a>), have been found to be increased during landing (<a aria-describedby="R12" class="usa-link" href="#R12">Capri et al., 2019</a>). miR-21, which is involved in the resolution of inflammation (<a aria-describedby="R65" class="usa-link" href="#R65">Tahamtan et al., 2018</a>), has been found to be decreased during spaceflight (<a aria-describedby="R32" class="usa-link" href="#R32">Hughes-Fulford et al., 2015</a>; <a aria-describedby="R42" class="usa-link" href="#R42">Makedonas et al., 2018</a>) and increased during landing (<a aria-describedby="R12" class="usa-link" href="#R12">Capri et al., 2019</a>). Changes in these miRNAs during different stages of spaceflight likely contribute to the overall dysregulation of inflammation, which may persist long after astronauts return to Earth (<a aria-describedby="R24" class="usa-link" href="#R24">Garrett-Bakelman et al., 2019</a>)</p>
<p id="P42">In general, miRNAs have yet to be studied extensively in the context of space biology. Previous rodent and human spaceflight (LEO) experiments that specifically examined miRNAs have been limited to <em>in vitro</em> cellular models (human fibroblasts and leukocytes), though some miRNAs (miR-34, miR-21, let-7a, and let-7c) identified in those studies do overlap with our proposed spaceflight signature (<a aria-describedby="R32" class="usa-link" href="#R32">Hughes-Fulford et al., 2015</a>; <a aria-describedby="R83" class="usa-link" href="#R83">Zhang et al., 2016</a>). Simulated microgravity studies in humans undergoing 21-day bedrest have also reported let-7a, let-7c, miR-25, miR-125b, and miR-92a from our panel (<a aria-describedby="R59" class="usa-link" href="#R59">Rullman et al., 2016</a>). Finally, simulated deep space radiation studies in mice have implicated miR-34 and miR-125 (<a aria-describedby="R67" class="usa-link" href="#R67">Templin et al., 2012</a>), which we have also observed in irradiated mouse serum.</p>
<p id="P43">Further studies will be necessary to compare spaceflight-associated miRNA effects at the individual organ, tissue, and cellular levels and to explore potential sex differences. Although we have observed strong similarities in miRNA expression in cardiac and soleus muscles (<a class="usa-link" href="#F7">Figures 7D</a> and <a class="usa-link" href="#F7">7E</a>), the miRNA content in these organs was not sufficient to explain the systemic changes in plasma miRNAs, indicating that other organs and tissues not quantified in this study may play a major role in miRNA expression. Therefore, there is a need for a more comprehensive and unbiased assessment of all miRNA through miRNA-seq in a wide range of tissues from each of the experimental models investigated here. In addition, scRNA-seq has indicated major changes in miRNA-associated gene expression among different blood immune cell types in the human blood from the NASA Twins Study (<a aria-describedby="R26" class="usa-link" href="#R26">Gertz et al., 2020</a>). CD14<sup>+</sup> monocytes are particularly likely to express and respond to miRNAs, suggesting a potential regulatory role of miRNAs on the innate immune response that is known to be dysregulated in spaceflight (<a aria-describedby="R17" class="usa-link" href="#R17">Crucian et al., 2011</a>). Despite the demonstrated contribution of CD14<sup>+</sup> monocytes to increases in the circulating miRNAs, it is also possible that the miRNAs could be coming from another organ where they are more abundantly expressed. Thus, further studies will be necessary to determine the exact origin of the increased miRNAs and tissue response, such as with cell-free DNA profiling (<a aria-describedby="R6" class="usa-link" href="#R6">Bezdan et al., 2020</a>), which will be beneficial for mapping and developing effective countermeasures. Finally, the successful prevention of vascular damage through antagomir therapy in our proof-of-concept human tissue model demonstrates the need for further experiments to fully characterize antagomir therapies <em>in vivo</em> to understand their clinical safety and efficacy and to examine which components of the spaceflight miRNA signature make effective targets for antagomir therapies.</p>
<p id="P44">Collectively, our results provide experimental confirmation of a previously predicted spaceflight-associated miRNA signature (<a aria-describedby="R3" class="usa-link" href="#R3">Beheshti et al., 2018</a>) in multiple rodent and human models in true spaceflight, simulated gravitational changes, and simulated deep space radiation. We also demonstrate the potential for preventing biological damage by strategic inhibition of specific miRNAs with antagomirs. The immediate impact of this study is the first step toward developing a systemic miRNA signature as a novel space biology research metric for assessing spaceflight-induced damage in astronauts and animal models of true spaceflight as well as ground analogs, while its expected long-term outcome is revealing new avenues for spaceflight countermeasure development that modulates systemic effects rather than isolated organ damage.</p></section><section id="S10"><h2 class="pmc_sec_title">STAR★METHODS</h2>
<section id="S11"><h3 class="pmc_sec_title">RESOURCE AVAILABILITY</h3>
<section id="S12"><h4 class="pmc_sec_title">Lead Contact</h4>
<p id="P45">Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Afshin Beheshti (<span>afshin.beheshti@nasa.gov</span>).</p></section><section id="S13"><h4 class="pmc_sec_title">Materials Availability</h4>
<p id="P46">This study did not generate new unique reagents.</p></section><section id="S14"><h4 class="pmc_sec_title">Data and Code Availability</h4>
<p id="P47">The published article includes all datasets generated and analyzed during this study. All miRNA-sequence raw data were deposited on the Database: NASA’s GeneLab data repository/platform. The following identifiers are used for the data in this paper: for all heart tissue related data: GLDS-334, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.26030/cg2g-as49" rel="noopener noreferrer" target="_blank">https://doi.org/10.26030/cg2g-as49</a>; for all liver tissue related data: GLDS-335, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.26030/72ke-1k67" rel="noopener noreferrer" target="_blank">https://doi.org/10.26030/72ke-1k67</a> for all soleus muscle related data: GLDS-337, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.26030/m73g-2477" rel="noopener noreferrer" target="_blank">https://doi.org/10.26030/m73g-2477</a> and for all plasma related data: GLDS-336, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.26030/qasa-rr29" rel="noopener noreferrer" target="_blank">https://doi.org/10.26030/qasa-rr29</a>.</p>
<p id="P48">Deposited data from the sequencing data from the NASA Twin Study can be found on the NASA Life Sciences Data Archive (LSDA) and the accession code is not available due to privacy concerns. LSDA is the repository for all human and animal research data, including that associated with this study. LSDA has a public facing portal where data requests can be initiated (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://lsda.jsc.nasa.gov/Request/dataRequestFAQ" rel="noopener noreferrer" target="_blank">https://lsda.jsc.nasa.gov/Request/dataRequestFAQ</a>). The LSDA team provides the appropriate processes, tools, and secure infrastructure for archival of experimental data and dissemination while complying with applicable rules, regulations, policies, and procedures governing the management and archival of sensitive data and information. The LSDA team enables data and information dissemination to the public or to authorized personnel either by providing public access to information or via an approved request process for information and data from the LSDA in accordance with NASA Human Research Program and JSC Institutional Review Board direction.</p></section></section><section id="S15"><h3 class="pmc_sec_title">EXPERIMENTAL MODEL AND SUBJECT DETAILS</h3>
<section id="S16"><h4 class="pmc_sec_title">Rodent models</h4>
<section id="S17"><h5 class="pmc_sec_title">Simulated space environment studies</h5>
<p id="P49">Mouse serum was obtained from Dr. Ann-Sofie Schreurs from a simulated microgravity and radiation study (<a class="usa-link" href="#SD1">Table S1</a>). Forty 16-week old male C57BL/6J (Jackson Labs, Sacramento, CA) mice were divided into four equal groups: sham, irradiated (IR), hindlimb unloaded (HU), or HU + IR. Mice were given sham or 2Gy of gamma (<sup>137</sup>Cs) irradiation, on day 3 of normal loading or HU, and were sacrificed on day 14.</p>
<p id="P50">Animals assigned to the HU group were suspended with traction tape based on previously established procedures (<a aria-describedby="R45" class="usa-link" href="#R45">Morey-Holton and Globus, 1998</a>). The room was maintained at an average 73.3°F, 31% humidity, 12-hour light/dark cycle, with food and water provided <em>ad libitum.</em> Animal weights were monitored throughout the duration of the experiment to assess health of the mice. The NASA Ames Research Center Institutional Animal Care and Use Committee (IACUC) approved all procedures. All experiments were performed in accordance with relevant guidelines and regulations. All animals were individually housed.</p>
<p id="P51">At 16 weeks of age, mice were exposed to 2 Gy gamma total body irradiation (<sup>137</sup>Cs at 0.83 Gy/min, JL Shepherd Mark I, San Gabriel, CA). For this treatment, mice were housed individually in a custom animal holding cage, maintaining HU mice in a head-down position. Mice were placed in the irradiator chamber and exposed to total body irradiation. The unloading of mice was maintained during irradiation by custom small HU cages. Sham-irradiated controls were handled similarly and placed in the irradiation chamber with the radiation source turned off.</p>
<p id="P52">Additional mouse serum was obtained from Dr. Yasaman Shirazi from a simulated microgravity and radiation study (<a class="usa-link" href="#SD1">Table S1</a>). Eighty 16-week old C57BL/6J male mice (Jackson Laboratory Maine, BNL protocol Globus 2015 #481) were divided into seven groups: sham (n = 14), 1Gy proton irradiation (<sup>1</sup>H, 150 MeV/n, LET ~0.52 KeV/um) (n = 9), 1Gy iron (<sup>56</sup>Fe, 600 MeV/n, LET ~175 KeV/um) irradiation (n = 9), 2Gy iron irradiation (n = 6), 1Gy mixed iron and proton irradiation (<sup>1</sup>H/<sup>56</sup>Fe/<sup>1</sup>H at 25/50/25cGy) (n = 14), hindlimb unloaded (HU) (n = 14), and HU+1Gy mixed iron and proton irradiation (n = 14). Total body irradiations were carried out at NASA Space Radiation Laboratory (NSRL) at Brookhaven National Laboratory (BNL) on conscious mice with 600 MeV/n <sup>56</sup>Fe (1 Gy), 150 MeV/n <sup>1</sup>H (1Gy), or mixed beam (<sup>1</sup>H/<sup>56</sup>Fe/<sup>1</sup>H at 25/50/25cGy), maintaining respective hindlimb unloading (HU) or normal loading (NL) conditions. Mice were irradiated after 3 days of HU or NL, and maintained HU or NL for 1 or 11 days before being sacrificed.</p>
<p id="P53">For both the above simulated microgravity and radiation studies, serum was isolated from blood draws from the vena cava of the mice. In order to prevent hemolysis from interfering in the results, the following protocol was uniformly conducted for all serum isolations: the blood was placed in a 4mL EDTA coated blood collection tube (BD Vacutainer Lavender K2-EDTA Cat# 367861) and kept at room temperature for 30 mins to allow for clotting. Then, the tubes were centrifuged at 2000 g for 15 minutes at 4°C followed by immediate freezing of the aliquots at −80°C. Samples were always mixed gently to prevent hemolysis.</p></section><section id="S18"><h5 class="pmc_sec_title">Partial weight bearing rat model</h5>
<p id="P54">Partial weight bearing rat serum was donated by Dr. Marie Mortreux and Dr. Seward Rutkove from a previous study. All experimental protocols were approved by the Beth Israel Deaconess Medical Center Institutional Animal Care and Use Committee. 40 adult male Wistar rats (14 weeks of age at baseline, Charles River Laboratories, Wilmington, MA) were assigned to 1 of 5 groups to ensure equal distribution of the body weights at baseline (428.3 ± 5.2g). Animals were assigned to either normal loading without apparatus (PWB100 N), normal loading with full harness (PWB100 FH), 70% of normal loading (PWB70), 40% of normal loading (PWB40), or 20% of normal loading (PWB20) for 28 consecutive days. Partial weight-bearing (PWB) was performed in custom cages using a quadrupedal unloading composed of a forelimb jacket and a pelvic harness and extensively described elsewhere (<a aria-describedby="R46" class="usa-link" href="#R46">Mortreux et al., 2018</a>).</p>
<p id="P55">Across our 28-day study, the only significant difference between the NH and FH control groups was observed in weight gain where the 100% gravity NH group gained significantly more weight than the 100% gravity FH group. The weight gain was also associated with a slightly (but not significant) increased daily food intake that did not result in a different cumulative food intake throughout our study. The device did not exert any pressure nor prevented the animals from moving freely in their cage.</p>
<p id="P56">Rats were singly housed in a room maintained at 22 ± 2°C with a 12h:12h light-dark cycle starting at 7:00 am, and allowed 48 hours to acclimate before baseline. On day 28, animals were euthanized using CO<sub>2</sub> inhalation per IACUC guidelines and blood was collected via cardiac puncture. Blood was allowed to clot at room temperature and was then centrifuged for 10 minutes at 10,000<em>g</em>. Serum was collected and frozen upon further analysis. 4 animals per group were selected randomly to perform miRNA analysis, and are outlined in <a class="usa-link" href="#SD1">Table S1</a>.</p></section><section id="S19"><h5 class="pmc_sec_title">ISS-flown mice</h5>
<p id="P57">Serum was isolated from 36 space-flown female BALB/cAnNTac mice (Taconic) (<a class="usa-link" href="#SD1">Table S1</a>). These mice were launched to the ISS on Rodent Research Reference Mission 1 (RRRM-1)/Rodent Research Mission 8 (RR-8) on 12/05/2018, docked to the ISS on 12/08/2018 and were returned to Earth as live animals on 01/13/2019, a total of 39 days after launch. The total absorbed radiation dose the mice received was 11.28mGy. The mice were sacrificed 4 days after returning to Earth. There were two different age groups, young (n = 18, 10-16 weeks) and old (n = 18, 31-37 weeks) as outlined in <a class="usa-link" href="#SD1">Table S1</a>.</p></section><section id="S20"><h5 class="pmc_sec_title">Space irradiation and hindlimb unloading of mice for miRNA-sequencing</h5>
<p id="P58">15-week ± 3-day old, <em>C57BL/6J</em> wild-type (<em>Wt</em>) female mice were purchased from Jackson Laboratories and housed at Brookhaven National Laboratory (BNL, Upton, NY). Upon arrival to BNL, mice were quarantined and acclimated to a standard 12:12h light:dark cycle, with controlled temperature/humidity for 1-week prior to cage acclimation. Food and water were given <em>ad libitum</em>, and standard bedding was changed once per week. Mice were cage acclimated (n = 10 mice per group; 2 mice per cage to maintain social interaction) 3-days prior to HU, followed by 14-days either normally loaded (NL) or hindlimb unloaded (HU, see details below). Irradiation was administered on day 13 and blood tissues were collected at 24-hours post-irradiation and post-euthanasia by CO<sub>2</sub> overdose, followed by cervical dislocation. Blood was collected via the abdominal aorta in EDTA-coated tubes (0.5 M) and plasma was separated by centrifugation at 2,000xg for 15 minutes. Plasma was collected and flash frozen in liquid nitrogen for −80°C storage. A 100 μL aliquot of cellular fraction was flash frozen and stored at −80°C for RNA analyses, while the remaining cellular fraction was lysed with 1x RBC lysis buffer (Thermo Fisher Scientific) followed by flow cytometric preparation and analyses, as describe below. The heart, liver, and soleus muscle were flash frozen and collected at the time of dissection. All tissues were stored at −80°C. Body weight tracking was performed on days −3, 0, 7 and 14.</p>
<p id="P59">Hindlimb unloading (HU) was performed using the adjusted Morey-Holton method for social housing (n = 2 per cage) (<a aria-describedby="R66" class="usa-link" href="#R66">Tahimic et al., 2019</a>). Briefly, mice were suspended from the tail using non-invasive traction tape attached to an adjustable pulley mounted on the top of a standard rodent cage. The adjustable nature of the device allows the user to position the animal in a head-down position (approximately 30° to the horizon) once attached. The crossbar height, lateral crossbar position, and chain length are adjustable. The pulley is free to move along the crossbar that spans the length of the top of the cage enabling the mouse to navigate freely on one half of the cage using its forepaws and to interact with the second mouse in the cage, without enabling ambulation of the limbs. Individual food and water sources are provided to each mouse and replaced daily. Control mice were housed in identical cages with normal ambulation.</p>
<p id="P60">On day 13, mice were transported on BNL base to the NASA Space Radiation Laboratory (NSRL) facility by animal care staff and were transferred to individual HU boxes. The following doses of irradiation were administered; simplified GCR sim (0.5 Gy), SPE (1 Gy), Gamma (5 Gy) and Sham control (0 Gy). Mice were positioned in the plateau region of the Bragg curve and irradiated at room temperature in individual HU boxes to enable whole body irradiation while maintaining hindlimb suspension. The NSRL physics staff performed the dosimetry. To simulate GCR, we used the simplified GCR simulation of ions, energy, and doses determined by a NASA consensus formula that consists of 5 ions: protons at 1000 MeV, <sup>28</sup>Si at 600 MeV/n, <sup>4</sup>He at 250 MeV/n, <sup>16</sup>O at 350 MeV/n, <sup>56</sup>Fe at 600 MeV/n, and protons at 250 MeV. This dose of radiation is equivalent to what an astronaut is predicted to receive in deep space during a Mars mission, though it is modeled as a single exposure over 25 minutes instead of the actual chronic exposures over 1.5 years. Further, GCR sim is a mixture of high and low LET ions in a ratio of 15% to 85%, respectfully. To simulate SPE, we used a total dose of 1Gy protons with energy ranges from 50MeV to 150MeV. For all irradiations a 60x60 beam was utilized at NSRL. For radiation dose equivalence and biological reference, we used 5 Gy gamma irradiation with the cesium resource available at BNL, in the absence of HU due to resource limitations. Sham controls were treated similar to GCR/SPE irradiated mice, including HU cage boxes and beam line (without irradiation) for the same duration as GCR simualtion, i.e., 25 minutes.</p></section></section><section id="S21"><h4 class="pmc_sec_title">
<em>In vitro</em> models</h4>
<section id="S22"><h5 class="pmc_sec_title">Irradiated human PBMCs</h5>
<p id="P61">Peripheral blood mononuclear cells (PBMCs) were obtained from Dr. Sylvain Costes from a human irradiation sensitivity study (NASA HRP #NNJ16HP24I, NASA Ames Research Center Human Research IRB #HRI-357) (<a class="usa-link" href="#SD1">Table S1</a>). Briefly, PBMCs were isolated from blood donated by 12 healthy adults: 8 females and 4 males. PBMC isolation was performed using Ficoll-Paque (VWR Cat# 95038-168) density gradient centrifugation followed by red blood cell lysis (VWR Cat# 420301-BL), and cells were frozen at 1x10<sup>7</sup> cells/mL until the day before irradiation. All subjects were of European descent and 20 - 64 years old.</p>
<p id="P62">Cells were plated in 12-well plates (VWR Cat# 29442-040) 24 hours before irradiation in RPMI 1640 (Sigma Cat# R8758) supplemented with 10% fetal bovine serum (VWR Cat# 1300-500H) and 1% Penicillin/Streptomycin/Glutamine (Fisher Cat# 10-378-016). Each well received approximately 1.6x10<sup>6</sup> cells and 2mL media. 24 hours later, the cells were irradiated with 0Gy or 0.3Gy 600MeV/n <sup>56</sup>Fe. Cell supernatants were collected 4 hours after irradiation and stored at −80°C until miRNA extractions.</p></section></section><section id="S23"><h4 class="pmc_sec_title">3D human vascular model</h4>
<p id="P63">Mature human 3D microvascular tissue models are grown by seeding-human umbilical vein endothelial cells (HUVECs) (Lonza Cat# CC-2519) into a collagen/matrigel mixture as described (<a aria-describedby="R28" class="usa-link" href="#R28">Grabham et al., 2011</a>). Briefly, HUVECs were first grown in EGM Endothelial Cell Growth Medium (Cat# CC-3124), then cells (1 × 10<sup>6</sup>/ml) were embedded into a collagen/matrigel mixture (Cat# 354236 and Cat# 356230). Microvessels are grown in EGM-2 Endothelial Cell Growth Medium (Cat# CC-3162) plus 50 nM Phorbol-12-myristate-13-acetate: (PMA) (Cat# 524400) for 7 days to form tubular microvessels before irradiations.</p>
<p id="P64">These tissue models were used to study vascular changes associated with specific miRNAs (<a class="usa-link" href="#SD1">Table S1</a>). Irradiations were carried out at Brookhaven National Lab using 50cGy of SimGCRSim (5-ion simplified galactic cosmic ray simulation: protons at 1000MeV, <sup>28</sup>Si at 600 MeV/n, <sup>4</sup>He at 250 MeV/n, <sup>16</sup>O at 350 MeV/n, <sup>56</sup>Fe at 600 MeV/n, and protons at 250 MeV). A dose of 0.5 μm of each antagomir was administered to the appropriate groups 24 hours prior to irradiation. Prior work has shown that lentivirus particles efficiently incorporate into these constructs, therefore it was assumed that antagomirs would have no trouble either. The tissue constructs used for fluorescence analysis were fixed with 4% paraformaldehyde (Sigma Cat# 158127100G) for 5 minutes at 37°C followed by 0.5% Triton X-100 (Sigma Cat# X100) to solubilize the fats. Cell structures were stained with 5-(4,6-Dichlorotriazinyl) Aminofluorescein (DTAF) (Cat# D 16) and Images captured on a Nikon TE 200 confocal C1 microscope with EZ-C1 software. The tissue constructs used for miRNA analysis were dissolved by submerging in TRIzol (ThermoFisher Cat# 15596026) 48 hours post irradiation, without any fixation or homogenization.</p></section><section id="S24"><h4 class="pmc_sec_title">Simulated GCR irradiation</h4>
<p id="P65">To simulate GCR, we used the simplified GCR simulation of ions, energy, and doses determined by a NASA consensus formula that consists of 5 ions: protons at 1000 MeV, <sup>28</sup>Si at 600 MeV/n, <sup>4</sup>He at 250 MeV/n, <sup>16</sup>O at 350 MeV/n, <sup>56</sup>Fe at 600 MeV/n, and protons at 250 MeV, in the following proportions; Protons 100MeV at 34.8%, Si at 1.1%, He at 18%, O at 5.8%, <sup>56</sup>Fe at 1%, and Protons 250 MeV at 39.3%. This mixture is the simplified version of the full GCR simulation and represents the proportions of ions found in space and thus translates to exploratory class missions (<a aria-describedby="R62" class="usa-link" href="#R62">Simonsen et al., 2020</a>). The low LET particles (Protons and Helium) make up the majority of radiation although high LET ions generally have a greater RBE.</p></section><section id="S25"><h4 class="pmc_sec_title">NASA Twins study samples</h4>
<p id="P66">The NASA Twins Study involved two male twin subjects, aged 50 years old at the time of launch. The flight twin spent 340 days aboard the ISS while his identical twin stayed on Earth as the ground control.</p>
<p id="P67">Blood samples were collected into 4mL CPT tubes (BD Biosciences Cat # 362760) per manufacturer’s recommendations. Cell separation was performed by centrifugation in at 1800 X g for 20 minutes at room temperature, both on the ISS and for the ground-based samples. Ambient blood collected samples slated for immediate return on Soyuz capsule were stored at 4°C until processing (average of 35-37 hours after collection, including repatriation time). Samples collected on Earth and the ISS and planned for long-term storage were mixed by inversion and immediately frozen at −80°C.</p>
<p id="P68">Fresh processing of CPT tubes was performed using the following steps: <strong>1.</strong> plasma was retrieved from the CPT tubes and flash frozen prior to long term storage at −80°C. <strong>2.</strong> The peripheral blood mononuclear cells (PBMCs) were recovered and washed in PBS. 0.5 million PBMCs were retrieved from one pre-flight and one post-flight samples, pelleted, flash frozen and stored at −80°C until use for RNA extractions. The flow through from cell sorting steps was recovered as the lymphocyte depleted (LD) fraction; LD and PBMC cell specimens were lysed into RLT+ buffer (QIAGEN Cat# 1053393), flash frozen and stored at −80°C until use.</p></section></section><section id="S26"><h3 class="pmc_sec_title">METHOD DETAILS</h3>
<section id="S27"><h4 class="pmc_sec_title">Accounting for Hemolysis in Samples</h4>
<p id="P69">Although hemolysis is unavoidable for blood/serum samples obtained from <em>in vivo</em> studies similar to what is described throughout this manuscript, we believe that we have accounted for this to the best to our ability for our analysis. There are many papers discussing this topic and impact on circulating miRNAs with the conclusions from these papers stating that for <em>in vivo</em> samples hemolysis can’t be avoided but procedures can be done to reduce the effects dramatically (<a aria-describedby="R8" class="usa-link" href="#R8">Blondal et al., 2013</a>; <a aria-describedby="R52" class="usa-link" href="#R52">Pizzamiglio et al., 2017</a>; <a aria-describedby="R53" class="usa-link" href="#R53">Poel et al., 2018</a>). Due the constraints for space biology experiments, the main step to reduce hemolysis variations for miRNAs is by handling all the samples identically. Both the controls and all other samples were handled under the exact same conditions throughout the manuscript. So if miRNAs levels would have been altered due to hemolyzed blood, then all miRNA levels would have been altered the same way between our comparisons. In fact in the negative controls all the miRNAs from our spaceflight signature are either nonexistent or at extremely low levels compared to samples under experimental conditions. This indicated that although it has been reported that hemolysis will alter miRNAs levels for all samples (<a aria-describedby="R52" class="usa-link" href="#R52">Pizzamiglio et al., 2017</a>), the levels of the miRNAs should have been altered identically in our controls and experimental conditions, allowing for reduced variability between results.</p></section><section id="S28"><h4 class="pmc_sec_title">miRNA extraction</h4>
<p id="P70">MiRNA extractions from serum, plasma, all tissues, and cell supernatants were carried out using the QIAGEN miRNeasy serum/plasma kit (#217184). MiRNA extractions from dissolved microvessel constructs were carried out using the QIAGEN miRNeasy Mini Kit (#217004). Quantitation of miRNA samples was done using a NanoDrop 2000 Spectrophotometer (ThermoFisher Scientific).</p>
<p id="P71">Total RNA extraction from the NASA Twin Study LD and PBMC lysates was carried out using the QIAGEN Allprep kit (Cat# 80204). Quantitation of RNA was assessed using Qubit 2.0 (Invitrogen) per manufacturer’s recommendations. RNA quality control was performed using RNA Nano kit (Agilent) on a Bioanalyzer 2100 (Agilent) per manufacturer’s recommendations.</p></section><section id="S29"><h4 class="pmc_sec_title">cDNA generation and Droplet Digital PCR</h4>
<p id="P72">First, cDNA was synthesized from miRNA samples using the QIAGEN miRCURY LNA RT Kit (Cat# 339340) using a concentration of 5ng/μl for the miRNA per sample. Next, samples were mixed with a 1:20 dilution of the generated cDNA with the BioRad QX200 ddPCR Evagreen Supermix (Cat# 1864034) and the appropriate miRNA primers from miRCURY LNA miRNA PCR Assays (QIAGEN). BioRad QX200 Automated Droplet Generator (Cat# 1864101) was used to create emulsion droplets. With the C1000 Touch Thermal Cycler with 96–Deep Well Reaction Module (Bio-Rad) the following PCR reaction was used for all the primers: 1 cycle 95°C for 5 min, 40 cycles of 95°C for 30 s and 58°C for 1 min (the annealing temperature can change depending on the primer), 1 cycle of 4°C for 5 min, and 1 cycle of 90°C for 5 min. Not all miRNA primers sets for ddPCR will have the same annealing temperature, so optimizing the annealing temperature is required for each primer set. Their respective annealing temperatures are found in <a class="usa-link" href="#SD1">Table S2</a>. Finally, the QX200 Droplet Digital PCR System (Bio-Rad) quantified the amount of miRNA for each primer set per sample. QuantaSoft software (Bio-Rad) generated the data for each primer set and sample. The same threshold setting was used for all samples per primer set. The concentration (miRNA copies/μl) value generated by QuantaSoft was converted to miRNA copies/ng of serum. These values were used for all miRNA analysis. For all analysis the miRNA concentrations were log<sub>2</sub>(x+1) transformed to allow for easy comparison between miRNAs and samples.</p></section><section id="S30"><h4 class="pmc_sec_title">miRNA sequencing on murine samples</h4>
<p id="P73">Library construction and sequencing was performed from miRNAs isolated from plasma, liver, heart, and soleus muscle from the mouse experiments described above. The miRNA extraction was carried out using the QIAgen miRNeasy kit (#217004). The total RNA quality and quantity were analyzed using a Bioanalyzer 2100 (Agilent, CA, USA) with RIN number &gt; 7. Approximately 1 μg of total RNA was used to prepare small RNA library according to protocol of TruSeq Small RNA Sample Prep Kits (Illumina, San Diego, USA). Single-end sequencing was performed using 50 bp on an Illumina Hiseq 2500 at the LC Sciences (Hangzhou, China) following the vendor’s recommended protocol. All miRNA-sequence raw data was deposited on NASA’s GeneLab data repository/platform with the following identifiers: for all heart tissue related data: GLDS-334, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.26030/cg2g-as49" rel="noopener noreferrer" target="_blank">https://doi.org/10.26030/cg2g-as49</a> for all liver tissue related data: GLDS-335, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.26030/72ke-1k67" rel="noopener noreferrer" target="_blank">https://doi.org/10.26030/72ke-1k67</a>: for all soleus muscle related data: GLDS-337, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.26030/m73g-2477" rel="noopener noreferrer" target="_blank">https://doi.org/10.26030/m73g-2477</a> and for all plasma related data: GLDS-336, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.26030/qasa-rr29" rel="noopener noreferrer" target="_blank">https://doi.org/10.26030/qasa-rr29</a>.</p></section><section id="S31"><h4 class="pmc_sec_title">miRNA sequencing on NASA Twins Study samples</h4>
<p id="P74">Small RNA libraries were prepared from 50 ng total RNA using the NEBNext® Multiplex Small RNA Library Prep Set for Illumina® (NEB #E7560) per manufacturer’s recommendations with the following modifications: adaptors and RT primer were diluted four fold, 17 cycles of PCR were used for library amplification, and no size selection was performed. The i7 primers in the NEBNext® Multiplex Oligos for Illumina® Dual Index Primers (NEB# E7600, NEB#E7780) were used to supplement the index primers in NEB #E7560. The libraries were sequenced in an Illumina NextSeq instrument (1x50bp).</p></section><section id="S32"><h4 class="pmc_sec_title">Single cell RNA and epitope sequencing on NASA Twin Study samples</h4>
<p id="P75">Cells were isolated prepared using the BD Rhapsody platform according to the BD Rhapsody Express Single-Cell Analysis System Instrument User Guide, with a custom-designed RNA and epitope panel. Briefly, cells from each sample were labeled with sample tags, then pooled after being washed twice with FACS buffer. Combined samples were then washed an additional time before being stained with the BD AbSeq Ab-Oligo reagents. After staining, the cells were washed twice before resuspension at approximately 20,000 cells in 620 μL. These cells were then isolated using the Single-Cell Analysis System and cDNA Synthesis with the BD Rhapsody Express Single-Cell Analysis System using the manufacturers’ protocol (BD Biosciences). Cells were loaded onto 3 BD Rhapsody nanowell cartridges. Cartridges were loaded with Cell Capture Beads (BD Biosciences) before shaking for 15 s at 1,000 rpm. Cells were lysed and cell capture beads were retrieved and washed prior to Exonuclease I treatment and reverse transcription. Targeted amplification of cDNA with the Human Immune Response Panel primers and custom supplemental panel (all targets listed in <a class="usa-link" href="#SD1">Table S3</a>) was done through 10 PCR cycles. PCR products were purified, and mRNA and AbSeq products were separated by SPRIselect beads with double-sided selection. mRNA products were amplified further with 15 PCR cycles. Final libraries were indexed with 8 PCR cycles. Library quality was assessed by Bioanalyzer (D1000 HS, Agilent). Library DNA concentration was quantified using a Qubit dsDNA HS Kit (ThermoFisher, #<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/protein/Q32854" rel="noopener noreferrer" target="_blank">Q32854</a>) on a Qubit Fluorometer. Libraries were diluted to 2nM and multiplexed before being sequenced on three lanes of a Novaseq-6000. Mean read depth per cell was 13,244.31 mRNA reads per cell and 11,506.98 AbSeq reads per cell for a combined 243751.30 reads per cell.</p></section><section id="S33"><h4 class="pmc_sec_title">Single cell ATAC sequencing on NASA Twins Study samples</h4>
<p id="P76">Cells were isolated and prepared according to the SureCell ATAC-Seq Library Prep Kit User Guide (17004620, Bio-Rad) by the Weill Cornell Epigenomics Core. Briefly, lysis and tagmentation was performed simultaneously on the leukocyte depleted fraction of PBMC samples. Cells were washed before resuspension in Whole Cell Tagmentation Mix. Cells were tagmented using a thermal protocol before being mixed and agitated for 30 minutes at 37°C. Prior to encapsulation, tagmented cells were kept on ice. Tagmented cells were loaded onto the ddSEQ Single-Cell Isolator (12004336, Bio-Rad). The SureCell ATAC-Seq Library Prep Kit (17004620, Bio-Rad) and SureCell ddSEQ Index Kit (12009360, Bio-Rad) were used to prepare scATAC libraries. Ampure XP beads (A63880, Beckman Coulter) were used to clean up the libraries. Barcoded amplicons were further amplified through 7 PCR cycles. These PCR products were cleaned up using Ampure XP beads and quality was assessed on both a TapeStation (D5000, Agilent) and Bioanalyzer. Library concentrations were quantified by on a Qubit Fluorometer by Qubit dsDNA HS Kit. Experiments were then pooled and sequenced by Bio-Rad on four lanes of a Novaseq-6000.</p></section></section><section id="S34"><h3 class="pmc_sec_title">QUANTIFICATION AND STATISTICAL ANALYSIS</h3>
<section id="S35"><h4 class="pmc_sec_title">miRNA quantification analysis from ddPCR data</h4>
<p id="P77">We previously utilized NASA GeneLab database to make unbiased predictions about miRNAs potentially involved in regulating the spaceflight response (<a aria-describedby="R3" class="usa-link" href="#R3">Beheshti et al., 2018</a>). The transcriptomic datasets analyzed in this study include GLDS-25 (livers from female C57BL/6J mice, n = 15 per group), GLDS-21 (gastrocnemius from female C57BL/6J mice, n = 4 per group), GLDS-63 (mammary gland, liver, and adipose tissue from pregnant female Sprague-Dawley rats, n = 4 per group), GLDS-111 (soleus and extensor digitorum longus from male C57/BL6 mice, n = 3 per group), GLDS-4 (thymus from C57BL/6NTac mice, n = 4 per group), GLDS-61 (skin from male C57Bl/J10 mice, n = 3 per group), and GLDS-48 (liver from female C57BL/6J mice, n = 6 per group). Details for this work and analysis is available in Beheshti et al. (<a aria-describedby="R3" class="usa-link" href="#R3">Beheshti et al., 2018</a>) and is the basis of the analysis we are describing in these methods.</p>
<p id="P78">Experimental groups and the number of replicates (n) within each group are summarized in <a class="usa-link" href="#SD1">Table S1</a>. For analysis on the archived serum from the simulated murine experiments, we grouped all miRNA copies/ng values for NL Sham controls together and all other experimental groups were compared to the NL Sham conditions. For all analysis the <em>p values</em> reported are two-sided and adjusted with the significance indicated by * p value &lt; 0.05, ** p value &lt; 0.01, and *** p value &lt; 0.001. Hierarchical clustering of the miRNA values were displayed as heatmaps with pheamap and t-Distributed Stochastic Neighbor Embedding (t-SNE) analysis (<a aria-describedby="R72" class="usa-link" href="#R72">van der Maaten and Hinton, 2008</a>) was done to determine how the samples cluster together based on the miRNA values. The miRNA values were also plotted using R-program ggplot2 (v3.2.1) (<a aria-describedby="R78" class="usa-link" href="#R78">Wickham, 2016</a>.) as scatterplots and data points were then fit using a generalized additive model (GAM) (<a aria-describedby="R30" class="usa-link" href="#R30">Hastie and Tibshirani, 1986</a>; <a aria-describedby="R79" class="usa-link" href="#R79">Wood et al., 2015</a>). Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway predictions from the miRNAs were determined by the DIANA-TarBase v7.0 algorithm. Correlation plots between miRNAs and different conditions were plotted using R program corrplot v0.84. Lastly, OriginPro 2015 was used to plot bar plots and of the fold-change values between conditions and also boxplots.</p></section><section id="S36"><h4 class="pmc_sec_title">
<em>In silico</em> predictions of genes from miRNAs</h4>
<p id="P79">Through the use of a Cytoscape (<a aria-describedby="R60" class="usa-link" href="#R60">Shannon et al., 2003</a>) plug-in called ClueGo/CluePedia (<a aria-describedby="R7" class="usa-link" href="#R7">Bindea et al., 2013</a>), we were able to predict genes targeted by the miRNAs determined. This involved entering all miRNAs in ClueGo and allowing the software to determine the top 50 genes that were significantly regulated and connected to the miRNAs. The predictions only reflect the functions that will be regulated by the miRNAs and do not show whether the function will be activated or inhibited. We also predicted the impact of the miRNAs on overall pathways by utilizing DIANA microT-CDS tool which allows for pathway discovery from targeted miRNAs (<a aria-describedby="R48" class="usa-link" href="#R48">Paraskevopoulou et al., 2013</a>; <a aria-describedby="R54" class="usa-link" href="#R54">Reczko et al., 2012</a>). Lastly, miRNet 2.0 was utilized to predict the diseases that are associated with the miRNAs (<a aria-describedby="R14" class="usa-link" href="#R14">Chang et al., 2020</a>).</p></section><section id="S37"><h4 class="pmc_sec_title">Analysis of miRNA sequencing from murine samples</h4>
<p id="P80">Raw reads were subjected to an in-house software program, ACGT101-miR (LC Sciences, Houston, Texas, USA) to remove adaptor dimers, junk, low complexity, common RNA families (rRNA, tRNA, snRNA, snoRNA) and repeats. Subsequently, unique sequences with length in 18~26 nucleotide were mapped to specific species precursors in miRBase 22.0 (<a aria-describedby="R38" class="usa-link" href="#R38">Kozomara et al., 2019</a>) by BLAST search to identify known miRNAs and novel 3p- and 5p- derived miRNAs. Length variation at both 3′ and 5′ ends and one mismatch inside of the sequence were allowed in the alignment. The unique sequences mapping to specific species mature miRNAs in hairpin arms were identified as known miRNAs. The unique sequences mapping to the other arm of known specific species precursor hairpin opposite to the annotated mature miRNA-containing arm were considered to be novel 5p- or 3p- derived miRNA candidates. The remaining sequences were mapped to other selected species precursors (with the exclusion of specific species) in miRBase 22.0 by BLAST search, and the mapped pre-miRNAs were further BLASTed against the specific species genomes to determine their genomic locations. The above two we defined as known miRNAs. The unmapped sequences were BLASTed against the specific genomes, and the hairpin RNA structures containing sequences were predicated from the flank 80 nt sequences using RNAfold software (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://rna.tbi.univie.ac.at/cgi-bin/RNAWebSuite/RNAfold.cgi" rel="noopener noreferrer" target="_blank">http://rna.tbi.univie.ac.at/cgi-bin/RNAWebSuite/RNAfold.cgi</a>). The criteria for secondary structure prediction were: (1) number of nucleotides in one bulge in stem (≤12); (2) number of base pairs in the stem region of the predicted hairpin (≥16); (3) cutoff of free energy (kCal/mol ≤ −15); (4) length of hairpin (up and down stems + terminal loop ≥ 50); (5) length of hairpin loop (≤20); (6) number of nucleotides in one bulge in mature region (≤8); (7) number of biased errors in one bulge in mature region (≤4); (8) number of biased bulges in mature region (≤2); (9) number of errors in mature region (≤7); (10) number of base pairs in the mature region of the predicted hairpin (≥12); and (11) percent of mature in stem (≥80). Differential expression of miRNAs based on normalized deep-sequencing counts was analyzed by selectively using Fisher exact test, Chi-square 2X2 test, Chi-square nXn test, Student t test, or ANOVA based on the experimental design. The significance threshold was set to be 0.01 and 0.05 in each test.</p>
<p id="P81">To determine gene ontology (GO) (<a aria-describedby="R63" class="usa-link" href="#R63">Smith and Eppig, 2009</a>) and C2 pathways being regulated by the miRNAs we performed miRNA gene set analysis utilizing the RBiomirGS v0.2.12 R package (<a aria-describedby="R82" class="usa-link" href="#R82">Zhang and Storey, 2018</a>) from the processed miRNA analysis for all conditions in the plasma and heart tissues. From the pathways we chose an FDR &lt; 0.25 cutoff for significantly regulated pathways. We plotted the specific pathways with R packages ggplot2.</p></section><section id="S38"><h4 class="pmc_sec_title">Analysis of miRNA sequencing from NASA Twin Study samples</h4>
<p id="P82">Standard libraries were preprocessed and quality controlled using miRTrace (<a aria-describedby="R36" class="usa-link" href="#R36">Kang et al., 2018</a>). Subsequently, reads were mapped against MirGeneDB sequences (<a aria-describedby="R23" class="usa-link" href="#R23">Fromm et al., 2020</a>) using the miRDeep2 (<a aria-describedby="R22" class="usa-link" href="#R22">Friedländer et al., 2012</a>) quantifier module. Expression values were normalized to reads per million (RPM) considering only miRNA counts. The normalized RPM value were utilized for all analysis. If the value for the miRNA was zero for all samples that miRNA was excluded from the analysis. To determine statistically significant miRNAs, ANOVA analysis with p value &lt; 0.2 was independently performed for each cell type. For flight only comparisons, the same statistical significance was applied for all time points excluding the ground samples and all ambient return samples. All statistics were run independently for each cell condition/type. To determine the overlap of the miRNA signature in this paper with the Twin Study miRNA-seq data, the proposed miRNAs were used as well as all mature components of the miRNAs included in the miRNA family.</p></section><section id="S39"><h4 class="pmc_sec_title">Analysis of scRNA and abseq from NASA Twins Study samples</h4>
<p id="P83">Fastq files were uploaded to Seven Bridges Genomics. Data was demultiplexed and sequences analyzed with BD’s Rhapsody pipeline (BD Rhapsody Analysis Pipeline 1.4 Beta) on Seven Bridges (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.sevenbridges.com" rel="noopener noreferrer" target="_blank">https://www.sevenbridges.com</a>). Data was then loaded into Seurat (version 3.2.0) for analysis (<a aria-describedby="R64" class="usa-link" href="#R64">Stuart et al., 2019</a>). This generated a sparse matrix file of features by barcodes. This sparse matrix data was then read into R using the R package Seurat 3.2.0, and standard quality control was run to remove cells with few genes. Data was then scaled and normalized. MAST (<a aria-describedby="R20" class="usa-link" href="#R20">Finak et al., 2015</a>) was used to determine DEGs between all combinations of preflight, return and postflight for both TW and HR samples. Linear dimensional reduction was performed by calculation of PCA from the most variable genes. Cells were then clustered using a resolution value of 0.5 and visualized by UMAP. Marker genes were checked for RNA and epitope expression for each cluster and assigned to the indicated cell types, with cell subtypes being grouped to match the scATAC data (<a class="usa-link" href="#SD1">Figures S2E</a> and <a class="usa-link" href="#SD1">S2F</a>). multiMiR was used to extract target genes for spaceflight-associated miRNA (<a aria-describedby="R58" class="usa-link" href="#R58">Ru et al., 2014</a>). Module scores for each spaceflight-associated miRNA target signature was calculated by AddModuleScore with a control value of 5. Violin plots were created with ggplot2 (version 3.3.2, (<a aria-describedby="R78" class="usa-link" href="#R78">Wickham, 2016</a>.) and significance tested by Wilcoxon rank sum using the geom_signif function. Cells were subsetted by cell type and dot plots were generated using the Seurat DotPlot function. Full list of RNA and AbSeq target genes are found in <a class="usa-link" href="#SD1">Table S3</a>.</p></section><section id="S40"><h4 class="pmc_sec_title">Analysis of scATAC from NASA Twins Study samples</h4>
<p id="P84">Fastq files from each sample were processed individually through BioRad’s scATAC processing pipeline according to the Bio-Rad ATAC-seq Analysis Toolkit Tutorial using the Bio-Rad docker images to run FastQC, debarcode, align reads, filter and deconvolute beads, and cell filtration. The aligned reads for each experiment were then processed using SnapTools (version 1.2.3) to create 5kb bins and a bin by cell matrix (<a aria-describedby="R19" class="usa-link" href="#R19">Fang et al., 2020</a>). The matrices for each sample were combined and then processed using SnapATAC (version 1.0.0) using the suggested parameters to filter for high-quality barcodes and remove bins that overlapped with ENCODE hg38 blacklisted regions or ChrM (<a aria-describedby="R19" class="usa-link" href="#R19">Fang et al., 2020</a>). Diffusion maps were computed for dimensionality reduction, and significant components were determined. Genes were assigned to each bin based off of GRCh38.p13. The resulting Snap object was then converted to a Seurat object and exported to Seurat (version 3.2.0). Samples were scaled and normalized, and anchors were transferred from the 10X PBMC dataset to the experiment from date 1/30/2019, as it contained the control, non-leukocyte depleted PBMCs. This data was then integrated using the IntegrateData function with the experiment from date 4/18/2019, and anchors were transferred from the 1/30/2019 experiment to the integrated object. Cells were assigned cell types based off of the predicted score, with cell subtypes being grouped to match the scRNA and epitope data (<a class="usa-link" href="#SD1">Figure S3I</a>). Due to the sparsity of the data, MAGIC (Rmagic 2.0.3) was then used to impute activity using the default parameters (<a aria-describedby="R73" class="usa-link" href="#R73">van Dijk et al., 2018</a>). The imputed activity was then used to generate dot plots and violin plots, with significance being tested by Wilcoxon rank sum using the geom_signif function. Cell composition percentages of cells after T and B cell removal was calculated by pooling the scATAC and scRNA/epitope cell identities per condition. <a class="usa-link" href="#SD2">Table S4</a> has all the determined DEGs between all combinations of preflight, return and postflight for both TW and HR samples.</p></section></section></section><section id="SM1"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="SD1"><div class="caption p"><span>1</span></div>
<div class="media p" id="d40e1576"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/8441986/bin/NIHMS1653539-supplement-1.pdf">NIHMS1653539-supplement-1.pdf</a><sup> (1.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD2"><div class="caption p"><span>2</span></div>
<div class="media p" id="d40e1579"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/8441986/bin/NIHMS1653539-supplement-2.csv">NIHMS1653539-supplement-2.csv</a><sup> (8.9KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD3"><div class="caption p"><span>3</span></div>
<div class="media p" id="d40e1582"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/8441986/bin/NIHMS1653539-supplement-3.csv">NIHMS1653539-supplement-3.csv</a><sup> (193KB, csv) </sup>
</div></div></section></section><section class="floats-group" id="floats-group1"><section class="tw xbox font-sm" id="T1"><div class="caption p"><p id="P133">KEY RESOURCES TABLE</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="rows">
<colgroup span="1">
<col align="left" span="1" valign="middle"/>
<col align="left" span="1" valign="middle"/>
<col align="left" span="1" valign="middle"/>
</colgroup>
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="middle">REAGENT or RESOURCE</th>
<th align="left" colspan="1" rowspan="1" valign="middle">SOURCE</th>
<th align="left" colspan="1" rowspan="1" valign="middle">IDENTIFIER</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Biological Samples</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Human PBMCs</td>
<td align="left" colspan="1" rowspan="1" valign="top">Consented Donors</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">NASA Twin Study Unsorted PBMCs</td>
<td align="left" colspan="1" rowspan="1" valign="top">Consented Donors</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">NASA Twin Study Lymphocyte Depleted Cells</td>
<td align="left" colspan="1" rowspan="1" valign="top">Consented Donors</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">NASA Twin Study CD8 CD4, CD14 and CD19 Cells</td>
<td align="left" colspan="1" rowspan="1" valign="top">Consented Donors</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Mouse Blood</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>C57BL/6J Wt</em> female</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Mouse Plasma</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>C57BL/6J Wt</em> female</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Mouse Liver</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>C57BL/6J Wt</em> female</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Mouse Heart</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>C57BL/6J Wt</em> female</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Chemicals, Peptides, and Recombinant Proteins</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">RLT+ Buffer</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 1053393</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Triton X-100</td>
<td align="left" colspan="1" rowspan="1" valign="top">Sigma</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# X100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">TRIzol</td>
<td align="left" colspan="1" rowspan="1" valign="top">ThermoFisher</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 15596026</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Ficoll-Paque</td>
<td align="left" colspan="1" rowspan="1" valign="top">VWR</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 95038-168</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">1x Phosphate Buffered Saline (pH 7.4)</td>
<td align="left" colspan="1" rowspan="1" valign="top">Thermofisher Scientific</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 10010023</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">EGM-2 Endothelial Cell Growth Medium-2 Bullet-Kit</td>
<td align="left" colspan="1" rowspan="1" valign="top">Lonza</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# CC-3162</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">EGM Endothelial Cell Growth Medium Bullet-Kit</td>
<td align="left" colspan="1" rowspan="1" valign="top">Lonza</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# CC-3124</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Rat tail Collagen Type 1</td>
<td align="left" colspan="1" rowspan="1" valign="top">BD Biosciences</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 354236</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Matrigel Matrix</td>
<td align="left" colspan="1" rowspan="1" valign="top">BD Biosciences</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 356230</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Phorbol-12-myristate-13-acetate: PMA</td>
<td align="left" colspan="1" rowspan="1" valign="top">Millipore</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 524400</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">5-(4,6-Dichlorotriazinyl) Aminofluorescein (DTAF)</td>
<td align="left" colspan="1" rowspan="1" valign="top">Thermofisher Scientific</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# D-16</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">MP Biomedicals Trypsin-EDTA Solution</td>
<td align="left" colspan="1" rowspan="1" valign="top">Thermofisher Scientific</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# ICN1689149</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Critical Commercial Assays</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">miRNeasy serum/plasma kit</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 217184</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">miRNeasy mini kit</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 217004</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Allprep kit</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 80204</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">miRCURY LNA RT kit</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 339340</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">QX200 ddPCR Evagreen Supermix</td>
<td align="left" colspan="1" rowspan="1" valign="top">BioRad</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 1864034</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">miRNA primers</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Listed under “Oligonucleotides”</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">NEBNext® Multiplex Small RNA Library Prep Set for Illumina®</td>
<td align="left" colspan="1" rowspan="1" valign="top">NEB</td>
<td align="left" colspan="1" rowspan="1" valign="top">NEB# E7560</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">NEBNext® Multiplex Oligos for Illumina® Dual Index Primers</td>
<td align="left" colspan="1" rowspan="1" valign="top">NEB</td>
<td align="left" colspan="1" rowspan="1" valign="top">NEB# E7600, NEB# E7780</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Index primers</td>
<td align="left" colspan="1" rowspan="1" valign="top">NEB</td>
<td align="left" colspan="1" rowspan="1" valign="top">NEB# E7560</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">iSCRIPT™ cDNA synthesis kit</td>
<td align="left" colspan="1" rowspan="1" valign="top">Bio-Rad</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 1708890</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">iQ™ SYBR® Green Supermix</td>
<td align="left" colspan="1" rowspan="1" valign="top">Bio-Rad</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 170-8880</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">TruSeq Small RNA Sample Prep Kits</td>
<td align="left" colspan="1" rowspan="1" valign="top">Illumina</td>
<td align="left" colspan="1" rowspan="1" valign="top">RS-200-0012</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Rhapsody AbSeq reagent pack</td>
<td align="left" colspan="1" rowspan="1" valign="top">BD Biosciences</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 633771</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Rhapsody Human Immune Response panel</td>
<td align="left" colspan="1" rowspan="1" valign="top">BD Biosciences</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 633750</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Rhapsody custom gene panel (See <a class="usa-link" href="#SD1">Table S3</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">BD Biosciences</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 633743</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Human Single cell multiplexing kit</td>
<td align="left" colspan="1" rowspan="1" valign="top">BD Biosciences</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 633781</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">SureCell ATAC-seq Library Prep Kit</td>
<td align="left" colspan="1" rowspan="1" valign="top">Bio-Rad</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 17004620</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">SureCell ddSEQ Index Kit</td>
<td align="left" colspan="1" rowspan="1" valign="top">Bio-Rad</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 12009360</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Deposited Data</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">MirGeneDB sequences</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R23" class="usa-link" href="#R23">Fromm et al., 2020</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://mirgenedb.org/" rel="noopener noreferrer" target="_blank">https://mirgenedb.org/</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">miRNA-seq for mice data</td>
<td align="left" colspan="1" rowspan="1" valign="top">NASA GeneLab (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://genelab.nasa.gov/" rel="noopener noreferrer" target="_blank">https://genelab.nasa.gov/</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">GLDS-334, GLDS-335, GLDS-336, GLDS-337</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">NASA Twin Study miRNA-seq results</td>
<td align="left" colspan="1" rowspan="1" valign="top">This paper</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">miRNA ddPCR raw data</td>
<td align="left" colspan="1" rowspan="1" valign="top">This paper</td>
<td align="left" colspan="1" rowspan="1" valign="top">LSDA (NASA)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">NASA Twin Study BD Rhapsody Data</td>
<td align="left" colspan="1" rowspan="1" valign="top">This paper and Getz et al. (2020)</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">NASA Twin Study scATAC Data</td>
<td align="left" colspan="1" rowspan="1" valign="top">This paper</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Experimental Models: Cell Lines</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Human: HUVECs</td>
<td align="left" colspan="1" rowspan="1" valign="top">Lonza</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# CC-2519</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Experimental Models: Organisms/Strains</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Mouse: C57BL/6J</td>
<td align="left" colspan="1" rowspan="1" valign="top">Jackson Labs, Sacramento, CA</td>
<td align="left" colspan="1" rowspan="1" valign="top">(IMSR Cat# JAX:000664, RRID:IMSR_JAX:000664)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Mouse: C57BL/6J</td>
<td align="left" colspan="1" rowspan="1" valign="top">Jackson Laboratory Maine,</td>
<td align="left" colspan="1" rowspan="1" valign="top">(IMSR Cat# JAX:000664, RRID:IMSR_JAX:000664)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Rat: Wistar</td>
<td align="left" colspan="1" rowspan="1" valign="top">Charles River Laboratories, Wilmington, MA</td>
<td align="left" colspan="1" rowspan="1" valign="top">(RGD Cat# 2312511, RRID:RGD_2312511)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Mouse: BALB/cAnNTac</td>
<td align="left" colspan="1" rowspan="1" valign="top">Taconic</td>
<td align="left" colspan="1" rowspan="1" valign="top">(IMSR Cat# TAC:balb, RRID:IMSR_TAC:balb)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Oligonucleotides</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-miR-17-5p miRCURY LNA miRNA PCR Assay: MIMAT0000070: 5′CAAAGUGCUUACAGUGCAGGUAG</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP02119304</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-miR-146a-5p miRCURY LNA miRNA PCR Assay: MIMAT0000449: 5′UGAGAACUGAAUUCCAUGGGUU</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00204688</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-miR-223-3p miRCURY LNA miRNA PCR Assay: MIMAT0000280: 5′UGUCAGUUUGUCAAAUACCCCA</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00205986</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-let-7a-5p miRCURY LNA miRNA PCR Assay: MIMAT0000062: 5′UGAGGUAGUAGGUUGUAUAGUU</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00205727</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-mir-92a-3p miRCURY LNA miRNA PCR Assay: MIMAT0000092: 5′UAUUGCACUUGUCCCGGCCUGU</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00204258</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-miR-24-3p miRCURY LNA miRNA PCR Assay: MIMAT0000080: 5′UGGCUCAGUUCAGCAGGAACAG</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00204260</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-miR-125b-5p miRCURY LNA miRNA PCR Assay: MIMAT0000423: 5′UCCCUGAGACCCUAACUUGUGA</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00205713</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-miR-16-5p miRCURY LNA miRNA PCR Assay: MIMAT0000069: 5′UAGCAGCACGUAAAUAUUGGCG</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00205702</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-miR-26a-5p miRCURY LNA miRNA PCR Assay: MIMAT0000082: 5′UUCAAGUAAUCCAGGAUAGGCU</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00206023</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-miR-145-5p miRCURY LNA miRNA PCR Assay: MIMAT0000437: 5′GUCCAGUUUUCCCAGGAAUCCCU</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00204483</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">mmu-miR-217-5p miRCURY LNA miRNA PCR Assay: MIMAT0000679: 5′UACUGCAUCAGGAACUGACUGGA</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00205070</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-miR-34a-5p miRCURY LNA miRNA PCR Assay: MIMAT0000255: 5′UGGCAGUGUCUUAGCUGGUUGU</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00204486</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-miR-21-5p miRCURY LNA miRNA PCR Assay: MIMAT0000076: 5′UAGCUUAUCAGACUGAUGUUGA</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00204230</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">mmu-miR-25-5p miRCURY LNA miRNA PCR Assay: MIMAT0017049: 5′AGGCGGAGACUUGGGCAAUUGC</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP02118413</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">hsa-let-7c-5p miRCURY LNA miRNA PCR Assay: MIMAT0000064: 5′UGAGGUAGUAGGUUGUAUGGUU</td>
<td align="left" colspan="1" rowspan="1" valign="top">QIAGEN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# YP00204767</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Software and Algorithms</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">miRTrace</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R36" class="usa-link" href="#R36">Kang et al., 2018</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://github.com/friedlanderlab/mirtrace" rel="noopener noreferrer" target="_blank">https://github.com/friedlanderlab/mirtrace</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">miRDeep2</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R22" class="usa-link" href="#R22">Friedländer et al., 2012</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.mdc-berlin.de/content/mirdeep2-documentation" rel="noopener noreferrer" target="_blank">https://www.mdc-berlin.de/content/mirdeep2-documentation</a>; RRID:SCR_010829</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Cytoscape</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R60" class="usa-link" href="#R60">Shannon et al., 2003</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://cytoscape.org" rel="noopener noreferrer" target="_blank">https://cytoscape.org</a>; RRID:SCR_003032</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">ClueGo/CluePedia</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R7" class="usa-link" href="#R7">Bindea et al., 2013</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.ici.upmc.fr/cluego/" rel="noopener noreferrer" target="_blank">http://www.ici.upmc.fr/cluego/</a>; RRID:SCR_005748</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">DIANA microT-CDS tool</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R48" class="usa-link" href="#R48">Paraskevopoulou et al., 2013</a>; <a aria-describedby="R54" class="usa-link" href="#R54">Reczko et al., 2012</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">http://carolina.imis.athena-innovation.gr/dianatools/web/index.php; RRID:SCR_016510</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">miRNet 2.0</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R14" class="usa-link" href="#R14">Chang et al., 2020</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.mirnet.ca/" rel="noopener noreferrer" target="_blank">https://www.mirnet.ca/</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Rtsne v0.15</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R72" class="usa-link" href="#R72">van der Maaten and Hinton, 2008</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://github.com/jkrijthe/Rtsne" rel="noopener noreferrer" target="_blank">https://github.com/jkrijthe/Rtsne</a>; RRID:SCR_016342</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">corrplot v0.84</td>
<td align="left" colspan="1" rowspan="1" valign="top">n/a</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://github.com/taiyun/corrplot" rel="noopener noreferrer" target="_blank">https://github.com/taiyun/corrplot</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">ggplot2 v3.3.1</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R78" class="usa-link" href="#R78">Wickham, 2016</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://ggplot2.tidyverse.org" rel="noopener noreferrer" target="_blank">https://ggplot2.tidyverse.org</a>; RRID:SCR_014601</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">pheatmap V1.0.12</td>
<td align="left" colspan="1" rowspan="1" valign="top">Kolde Rpheatmap: Pretty heatmaps software; 2015</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://cran.r-project.org/web/packages/pheatmap/index.html" rel="noopener noreferrer" target="_blank">https://cran.r-project.org/web/packages/pheatmap/index.html</a>; RRID:SCR_016418</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">ACGT101-miR</td>
<td align="left" colspan="1" rowspan="1" valign="top">LC Sciences</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">miRBase 22.0</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R38" class="usa-link" href="#R38">Kozomara et al., 2019</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.mirbase.org/" rel="noopener noreferrer" target="_blank">http://www.mirbase.org/</a>; RRID:SCR_003152</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">RBiomirGS V0.2.12 R package</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R82" class="usa-link" href="#R82">Zhang and Storey, 2018</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://github.com/jzhangc/git_RBiomirGS" rel="noopener noreferrer" target="_blank">https://github.com/jzhangc/git_RBiomirGS</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">GO Mouse Genome Informatics (MGI)</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R63" class="usa-link" href="#R63">Smith and Eppig, 2009</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.informatics.jax.org/" rel="noopener noreferrer" target="_blank">http://www.informatics.jax.org/</a>; RRID:SCR_006489</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">R package version 3.6.1</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.r-project.org/" rel="noopener noreferrer" target="_blank">https://www.r-project.org/</a>; RRID:SCR_001905</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">QuantaSoft V1.7.4.0917</td>
<td align="left" colspan="1" rowspan="1" valign="top">Bio-Rad Laboratories</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.bio-rad.com/webroot/web/pdf/lsr/literature/10047467.pdf" rel="noopener noreferrer" target="_blank">https://www.bio-rad.com/webroot/web/pdf/lsr/literature/10047467.pdf</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">OriginPro 2015</td>
<td align="left" colspan="1" rowspan="1" valign="top">OriginLab Corporation</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.originlab.com" rel="noopener noreferrer" target="_blank">https://www.originlab.com</a>; RRID:SCR_014212</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">R package version 4.0.2 (for single cell analysis)</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.r-project.org/" rel="noopener noreferrer" target="_blank">https://www.r-project.org/</a>; RRID:SCR_001905</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Seurat 3.2.0</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R64" class="usa-link" href="#R64">Stuart et al., 2019</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://github.com/satijalab/seurat" rel="noopener noreferrer" target="_blank">https://github.com/satijalab/seurat</a>; RRID:SCR_007322</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">MAST</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R20" class="usa-link" href="#R20">Finak et al., 2015</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.bioconductor.org/packages/release/bioc/html/MAST.html" rel="noopener noreferrer" target="_blank">https://www.bioconductor.org/packages/release/bioc/html/MAST.html</a>; RRID:SCR_016340</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">multiMiR</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R58" class="usa-link" href="#R58">Ru et al., 2014</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.bioconductor.org/packages/release/bioc/html/multiMiR.html" rel="noopener noreferrer" target="_blank">https://www.bioconductor.org/packages/release/bioc/html/multiMiR.html</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">SnapTools 1.4.1</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R19" class="usa-link" href="#R19">Fang et al., 2020</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://github.com/r3fang/SnapTools" rel="noopener noreferrer" target="_blank">https://github.com/r3fang/SnapTools</a>; RRID:SCR_018097</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">SnapATAC</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R19" class="usa-link" href="#R19">Fang et al., 2020</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://github.com/r3fang/SnapATAC" rel="noopener noreferrer" target="_blank">https://github.com/r3fang/SnapATAC</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Rmagic 2.0.3</td>
<td align="left" colspan="1" rowspan="1" valign="top">(<a aria-describedby="R73" class="usa-link" href="#R73">van Dijk et al., 2018</a>)</td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://github.com/KrishnaswamyLab/MAGIC" rel="noopener noreferrer" target="_blank">https://github.com/KrishnaswamyLab/MAGIC</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Patchwork 1.0.1</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://github.com/thomasp85/patchwork" rel="noopener noreferrer" target="_blank">https://github.com/thomasp85/patchwork</a>; RRID:SCR_000072</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Other</td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">QX200 Automated Droplet Generator</td>
<td align="left" colspan="1" rowspan="1" valign="top">BioRad</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 1864101</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">QX200 Droplet Reader</td>
<td align="left" colspan="1" rowspan="1" valign="top">BioRad</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 1864003</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Illumina Hiseq 2500</td>
<td align="left" colspan="1" rowspan="1" valign="top">LC Sciences</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">ABI 7500 Real-Time PCR</td>
<td align="left" colspan="1" rowspan="1" valign="top">Applied Biosystems</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">NovaSeq 6000</td>
<td align="left" colspan="1" rowspan="1" valign="top">Illumina</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BD Rhapsody</td>
<td align="left" colspan="1" rowspan="1" valign="top">BD Biosciences</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 633701</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">ddSEQ Single-Cell Isolator</td>
<td align="left" colspan="1" rowspan="1" valign="top">Bio-Rad</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# 12004366</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BioAnalyzer</td>
<td align="left" colspan="1" rowspan="1" valign="top">Agilent</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cat# D1000 HS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Simplified 5-ion GCR Simulation: Protons at 1000 MeV, <sup>28</sup>Si at 600 MeV/n, <sup>4</sup>He at 250 MeV/n, <sup>16</sup>O at 350 MeV/n, <sup>56</sup>Fe at 600 MeV/n, and protons at 250 MeV</td>
<td align="left" colspan="1" rowspan="1" valign="top">Brookhaven National Lab - NSRL</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Simulated SPE: Protons at 50MeV to 150MeV</td>
<td align="left" colspan="1" rowspan="1" valign="top">Brookhaven National Lab - NSRL</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">Gamma: <sup>137</sup>Cs source (5 Gy)</td>
<td align="left" colspan="1" rowspan="1" valign="top">Brookhaven National Lab – NSRL</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">
<sup>56</sup>Fe Irradiation: 600 MeV/n</td>
<td align="left" colspan="1" rowspan="1" valign="top">Brookhaven National Lab – NSRL</td>
<td align="left" colspan="1" rowspan="1" valign="top">N/A</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></section><section class="bt xbox font-sm" id="BX1"><h2 class="obj_head">Highlights.</h2>
<ul class="list" id="L1" style="list-style-type:disc">
<li><p id="P134">Spaceflight miRNA signature validated in multiple organism models</p></li>
<li><p id="P135">Components of miRNA signature related to space radiation and microgravity</p></li>
<li><p id="P136">Downstream targets and circulating dependence of miRNAs in NASA Twins Study</p></li>
<li><p id="P137">Inhibition of key microvasculature miRNAs mitigates space radiation impact</p></li>
</ul></section></section><section class="ack" id="S41"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p id="P85">Special thanks to Dr. Ruth Globus for her input on the manuscript and to Carolyn Reifsnyder, Zach Burkett, and Kelly Kaihara from Bio-Rad for early access to scATAC. We would also like to thank Drs. Peter Guida, Adam Rusek, Michael Sivertz, and the NSRL team at BNL for their assistance and expertise with irradiation studies. C.E.M. would like to thank the Epigenomics Core Facility at Weill Cornell Medicine, the Scientific Computing Unit (SCU), XSEDE Supercomputing Resources, STARR I13-0052, the Vallee Foundation, the WorldQuant Foundation, The Pershing Square Sohn Cancer Research Alliance, NASA (NNX14AH50G, NNX17AB26G, 19-19OMNI_2-0109), and the National Institutes for Health (R01MH117406, R01CA249054, TRISH [NNX16AO69A:0107, NNX16AO69A:0061], LLS 9238-16, Mak, LLS-MCL7001-18). This work was also supported by NASA grant 16-ROSBFP_GL-0005: NNH16ZTT001N-FG Appendix G: Solicitation of Proposals for Flight and Ground Space Biology Research (award 80NSSC19K0883) and The Translational Research Institute for Space Health through NASA Cooperative Agreement NNX16AO69A (T-0404) awarded to A.B. ISS-related mouse tissue research reported in this publication was supported by the International Space Station U.S. National Laboratory under award UA-2019-018 given to A.B.</p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="FN2">
<p id="P86">SUPPLEMENTAL INFORMATION</p>
<p id="P87"><a class="usa-link" href="#SD1">Supplemental Information</a> can be found online at <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.celrep.2020.108448" rel="noopener noreferrer" target="_blank">https://doi.org/10.1016/j.celrep.2020.108448</a>.</p>
</div>
<div class="fn p" id="FN3">
<p id="P88">DECLARATION OF INTERESTS</p>
<p id="P89">The authors declare no competing interests.</p>
</div>
</div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="R1">
<cite>Baqai FP, Gridley DS, Slater JM, Luo-Owen X, Stodieck LS, Ferguson V, Chapes SK, and Pecaut MJ (2009). Effects of spaceflight on innate immune function and antioxidant gene expression. J. Appl. Physiol
106, 1935–1942.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1152/japplphysiol.91361.2008" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2692779/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19342437/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Physiol&amp;title=Effects%20of%20spaceflight%20on%20innate%20immune%20function%20and%20antioxidant%20gene%20expression&amp;author=FP%20Baqai&amp;author=DS%20Gridley&amp;author=JM%20Slater&amp;author=X%20Luo-Owen&amp;author=LS%20Stodieck&amp;volume=106&amp;publication_year=2009&amp;pages=1935-1942&amp;pmid=19342437&amp;doi=10.1152/japplphysiol.91361.2008&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R2">
<cite>Beheshti A, Vanderburg C, McDonald JT, Ramkumar C, Kadungure T, Zhang H, Gartenhaus RB, and Evens AM (2017). A Circulating microRNA Signature Predicts Age-Based Development of Lymphoma. PLoS ONE
12, e0170521.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0170521" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5249061/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28107482/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=A%20Circulating%20microRNA%20Signature%20Predicts%20Age-Based%20Development%20of%20Lymphoma&amp;author=A%20Beheshti&amp;author=C%20Vanderburg&amp;author=JT%20McDonald&amp;author=C%20Ramkumar&amp;author=T%20Kadungure&amp;volume=12&amp;publication_year=2017&amp;pages=e0170521&amp;pmid=28107482&amp;doi=10.1371/journal.pone.0170521&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R3">
<cite>Beheshti A, Ray S, Fogle H, Berrios D, and Costes SV (2018). A microRNA signature and TGF-β1 response were identified as the key master regulators for spaceflight response. PLoS ONE
13, e0199621.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0199621" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6059388/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30044882/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=A%20microRNA%20signature%20and%20TGF-%CE%B21%20response%20were%20identified%20as%20the%20key%20master%20regulators%20for%20spaceflight%20response&amp;author=A%20Beheshti&amp;author=S%20Ray&amp;author=H%20Fogle&amp;author=D%20Berrios&amp;author=SV%20Costes&amp;volume=13&amp;publication_year=2018&amp;pages=e0199621&amp;pmid=30044882&amp;doi=10.1371/journal.pone.0199621&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R4">
<cite>Beheshti A, McDonald JT, Miller J, Grabham P, and Costes SV (2019). GeneLab Database Analyses Suggest Long-Term Impact of Space Radiation on the Cardiovascular System by the Activation of <em>FYN</em> Through Reactive Oxygen Species. Int. J. Mol. Sci
20, 661.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/ijms20030661" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6387434/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30717456/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci&amp;title=GeneLab%20Database%20Analyses%20Suggest%20Long-Term%20Impact%20of%20Space%20Radiation%20on%20the%20Cardiovascular%20System%20by%20the%20Activation%20of%20FYN%20Through%20Reactive%20Oxygen%20Species&amp;author=A%20Beheshti&amp;author=JT%20McDonald&amp;author=J%20Miller&amp;author=P%20Grabham&amp;author=SV%20Costes&amp;volume=20&amp;publication_year=2019&amp;pages=661&amp;pmid=30717456&amp;doi=10.3390/ijms20030661&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R5">
<cite>Besnier M, Shantikumar S, Anwar M, Dixit P, Chamorro-Jorganes A, Sweaad W, Sala-Newby G, Madeddu P, Thomas AC, Howard L, et al.  (2019). miR-15a/-16 Inhibit Angiogenesis by Targeting the Tie2 Coding Sequence: Therapeutic Potential of a miR-15a/16 Decoy System in Limb Ischemia. Mol. Ther. Nucleic Acids
17, 49–62.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.omtn.2019.05.002" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6586592/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31220779/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Ther.%20Nucleic%20Acids&amp;title=miR-15a/-16%20Inhibit%20Angiogenesis%20by%20Targeting%20the%20Tie2%20Coding%20Sequence:%20Therapeutic%20Potential%20of%20a%20miR-15a/16%20Decoy%20System%20in%20Limb%20Ischemia&amp;author=M%20Besnier&amp;author=S%20Shantikumar&amp;author=M%20Anwar&amp;author=P%20Dixit&amp;author=A%20Chamorro-Jorganes&amp;volume=17&amp;publication_year=2019&amp;pages=49-62&amp;pmid=31220779&amp;doi=10.1016/j.omtn.2019.05.002&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R6">
<cite>Bezdan D, et al.  (2020). Cell-free DNA (cfDNA) and exosome profiling from a year-long human spaceflight reveals circulating biomarkers. iScience. 10.1016/j.isci.2020.101844.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.isci.2020.101844" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7756145/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33376973/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=Cell-free%20DNA%20(cfDNA)%20and%20exosome%20profiling%20from%20a%20year-long%20human%20spaceflight%20reveals%20circulating%20biomarkers&amp;author=D%20Bezdan&amp;publication_year=2020&amp;pmid=33376973&amp;doi=10.1016/j.isci.2020.101844&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R7">
<cite>Bindea G, Galon J, and Mlecnik B (2013). CluePedia Cytoscape plugin: pathway insights using integrated experimental and in silico data. Bioinformatics
29, 661–663.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btt019" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3582273/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23325622/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=CluePedia%20Cytoscape%20plugin:%20pathway%20insights%20using%20integrated%20experimental%20and%20in%20silico%20data&amp;author=G%20Bindea&amp;author=J%20Galon&amp;author=B%20Mlecnik&amp;volume=29&amp;publication_year=2013&amp;pages=661-663&amp;pmid=23325622&amp;doi=10.1093/bioinformatics/btt019&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R8">
<cite>Blondal T, Jensby Nielsen S, Baker A, Andreasen D, Mouritzen P, Wrang Teilum M, and Dahlsveen IK (2013). Assessing sample and miRNA profile quality in serum and plasma or other biofluids. Methods
59, S1–S6.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ymeth.2012.09.015" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23036329/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Assessing%20sample%20and%20miRNA%20profile%20quality%20in%20serum%20and%20plasma%20or%20other%20biofluids&amp;author=T%20Blondal&amp;author=S%20Jensby%20Nielsen&amp;author=A%20Baker&amp;author=D%20Andreasen&amp;author=P%20Mouritzen&amp;volume=59&amp;publication_year=2013&amp;pages=S1-S6&amp;pmid=23036329&amp;doi=10.1016/j.ymeth.2012.09.015&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R9">
<cite>Burkhart K, Allaire B, and Bouxsein ML (2019). Negative Effects of Long-duration Spaceflight on Paraspinal Muscle Morphology. Spine
44, 879–886.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1097/BRS.0000000000002959" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30624302/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Spine&amp;title=Negative%20Effects%20of%20Long-duration%20Spaceflight%20on%20Paraspinal%20Muscle%20Morphology&amp;author=K%20Burkhart&amp;author=B%20Allaire&amp;author=ML%20Bouxsein&amp;volume=44&amp;publication_year=2019&amp;pages=879-886&amp;pmid=30624302&amp;doi=10.1097/BRS.0000000000002959&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R10">
<cite>Butz H, Rácz K, Hunyady L, and Patócs A (2012). Crosstalk between TGF-β signaling and the microRNA machinery. Trends Pharmacol. Sci
33, 382–393.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tips.2012.04.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22613783/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol.%20Sci&amp;title=Crosstalk%20between%20TGF-%CE%B2%20signaling%20and%20the%20microRNA%20machinery&amp;author=H%20Butz&amp;author=K%20R%C3%A1cz&amp;author=L%20Hunyady&amp;author=A%20Pat%C3%B3cs&amp;volume=33&amp;publication_year=2012&amp;pages=382-393&amp;pmid=22613783&amp;doi=10.1016/j.tips.2012.04.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R11">
<cite>Caporali A, and Emanueli C (2011). MicroRNA-503 and the extended microRNA-16 family in angiogenesis. Trends Cardiovasc. Med
21, 162–166.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tcm.2012.05.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3407871/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22814423/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Cardiovasc.%20Med&amp;title=MicroRNA-503%20and%20the%20extended%20microRNA-16%20family%20in%20angiogenesis&amp;author=A%20Caporali&amp;author=C%20Emanueli&amp;volume=21&amp;publication_year=2011&amp;pages=162-166&amp;pmid=22814423&amp;doi=10.1016/j.tcm.2012.05.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R12">
<cite>Capri M, Morsiani C, Santoro A, Moriggi M, Conte M, Martucci M, Bellavista E, Fabbri C, Giampieri E, Albracht K, et al.  (2019). Recovery from 6-month spaceflight at the International Space Station: muscle-related stress into a proinflammatory setting. FASEB J. 33, 5168–5180.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1096/fj.201801625R" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6436655/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30620616/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FASEB%20J&amp;title=Recovery%20from%206-month%20spaceflight%20at%20the%20International%20Space%20Station:%20muscle-related%20stress%20into%20a%20proinflammatory%20setting&amp;author=M%20Capri&amp;author=C%20Morsiani&amp;author=A%20Santoro&amp;author=M%20Moriggi&amp;author=M%20Conte&amp;volume=33&amp;publication_year=2019&amp;pages=5168-5180&amp;pmid=30620616&amp;doi=10.1096/fj.201801625R&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R13">
<cite>Cekanaviciute E, Fathali N, Doyle KP, Williams AM, Han J, and Buckwalter MS (2014). Astrocytic transforming growth factor-beta signaling reduces subacute neuroinflammation after stroke in mice. Glia
62, 1227–1240.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/glia.22675" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4061255/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24733756/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Glia&amp;title=Astrocytic%20transforming%20growth%20factor-beta%20signaling%20reduces%20subacute%20neuroinflammation%20after%20stroke%20in%20mice&amp;author=E%20Cekanaviciute&amp;author=N%20Fathali&amp;author=KP%20Doyle&amp;author=AM%20Williams&amp;author=J%20Han&amp;volume=62&amp;publication_year=2014&amp;pages=1227-1240&amp;pmid=24733756&amp;doi=10.1002/glia.22675&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R14">
<cite>Chang L, Zhou G, Soufan O, and Xia J (2020). miRNet 2.0: network-based visual analytics for miRNA functional analysis and systems biology. Nucleic Acids Res. 48, W244–W251.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkaa467" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7319552/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32484539/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=miRNet%202.0:%20network-based%20visual%20analytics%20for%20miRNA%20functional%20analysis%20and%20systems%20biology&amp;author=L%20Chang&amp;author=G%20Zhou&amp;author=O%20Soufan&amp;author=J%20Xia&amp;volume=48&amp;publication_year=2020&amp;pages=W244-W251&amp;pmid=32484539&amp;doi=10.1093/nar/gkaa467&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R15">
<cite>Chen JQ, Papp G, Szodoray P, and Zeher M (2016). The role of microRNAs in the pathogenesis of autoimmune diseases. Autoimmun. Rev
15, 1171–1180.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.autrev.2016.09.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27639156/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Autoimmun.%20Rev&amp;title=The%20role%20of%20microRNAs%20in%20the%20pathogenesis%20of%20autoimmune%20diseases&amp;author=JQ%20Chen&amp;author=G%20Papp&amp;author=P%20Szodoray&amp;author=M%20Zeher&amp;volume=15&amp;publication_year=2016&amp;pages=1171-1180&amp;pmid=27639156&amp;doi=10.1016/j.autrev.2016.09.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R16">
<cite>Coburn-Litvak PS, Tata DA, Gorby HE, McCloskey DP, Richardson G, and Anderson BJ (2004). Chronic corticosterone affects brain weight, and mitochondrial, but not glial volume fraction in hippocampal area CA3. Neuroscience
124, 429–438.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.neuroscience.2003.11.031" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14980392/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Neuroscience&amp;title=Chronic%20corticosterone%20affects%20brain%20weight,%20and%20mitochondrial,%20but%20not%20glial%20volume%20fraction%20in%20hippocampal%20area%20CA3&amp;author=PS%20Coburn-Litvak&amp;author=DA%20Tata&amp;author=HE%20Gorby&amp;author=DP%20McCloskey&amp;author=G%20Richardson&amp;volume=124&amp;publication_year=2004&amp;pages=429-438&amp;pmid=14980392&amp;doi=10.1016/j.neuroscience.2003.11.031&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R17">
<cite>Crucian B, Stowe R, Quiriarte H, Pierson D, and Sams C (2011). Monocyte phenotype and cytokine production profiles are dysregulated by short-duration spaceflight. Aviat. Space Environ. Med
82, 857–862.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3357/asem.3047.2011" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21888268/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Aviat.%20Space%20Environ.%20Med&amp;title=Monocyte%20phenotype%20and%20cytokine%20production%20profiles%20are%20dysregulated%20by%20short-duration%20spaceflight&amp;author=B%20Crucian&amp;author=R%20Stowe&amp;author=H%20Quiriarte&amp;author=D%20Pierson&amp;author=C%20Sams&amp;volume=82&amp;publication_year=2011&amp;pages=857-862&amp;pmid=21888268&amp;doi=10.3357/asem.3047.2011&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R18">
<cite>Elgart SR, Little MP, Chappell LJ, Milder CM, Shavers MR, Huff JL, and Patel ZS (2018). Radiation Exposure and Mortality from Cardiovascular Disease and Cancer in Early NASA Astronauts. Sci. Rep
8, 8480.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-018-25467-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5981602/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29855508/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep&amp;title=Radiation%20Exposure%20and%20Mortality%20from%20Cardiovascular%20Disease%20and%20Cancer%20in%20Early%20NASA%20Astronauts&amp;author=SR%20Elgart&amp;author=MP%20Little&amp;author=LJ%20Chappell&amp;author=CM%20Milder&amp;author=MR%20Shavers&amp;volume=8&amp;publication_year=2018&amp;pages=8480&amp;pmid=29855508&amp;doi=10.1038/s41598-018-25467-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R19">
<cite>Fang R, Preissl S, Li Y, Hou X, Lucero J, Wang X, Motamedi A, Shiau AK, Zhou X, Xie F, et al.  (2020). SnapATAC: A Comprehensive Analysis Package for Single Cell ATAC-seq. bioRxiv. 10.1101/615179.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/615179" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7910485/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33637727/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=SnapATAC:%20A%20Comprehensive%20Analysis%20Package%20for%20Single%20Cell%20ATAC-seq&amp;author=R%20Fang&amp;author=S%20Preissl&amp;author=Y%20Li&amp;author=X%20Hou&amp;author=J%20Lucero&amp;publication_year=2020&amp;pmid=33637727&amp;doi=10.1101/615179&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R20">
<cite>Finak G, McDavid A, Yajima M, Deng J, Gersuk V, Shalek AK, Slichter CK, Miller HW, McElrath MJ, Prlic M, et al.  (2015). MAST: a flexible statistical framework for assessing transcriptional changes and characterizing heterogeneity in single-cell RNA sequencing data. Genome Biol. 16, 278.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s13059-015-0844-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4676162/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26653891/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=MAST:%20a%20flexible%20statistical%20framework%20for%20assessing%20transcriptional%20changes%20and%20characterizing%20heterogeneity%20in%20single-cell%20RNA%20sequencing%20data&amp;author=G%20Finak&amp;author=A%20McDavid&amp;author=M%20Yajima&amp;author=J%20Deng&amp;author=V%20Gersuk&amp;volume=16&amp;publication_year=2015&amp;pages=278&amp;pmid=26653891&amp;doi=10.1186/s13059-015-0844-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R21">
<cite>Florijn BW, Bijkerk R, van der Veer EP, and van Zonneveld AJ (2018). Gender and cardiovascular disease: are sex-biased microRNA networks a driving force behind heart failure with preserved ejection fraction in women?
Cardiovasc. Res
114, 210–225.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/cvr/cvx223" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29186452/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cardiovasc.%20Res&amp;title=Gender%20and%20cardiovascular%20disease:%20are%20sex-biased%20microRNA%20networks%20a%20driving%20force%20behind%20heart%20failure%20with%20preserved%20ejection%20fraction%20in%20women?&amp;author=BW%20Florijn&amp;author=R%20Bijkerk&amp;author=EP%20van%20der%20Veer&amp;author=AJ%20van%20Zonneveld&amp;volume=114&amp;publication_year=2018&amp;pages=210-225&amp;pmid=29186452&amp;doi=10.1093/cvr/cvx223&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R22">
<cite>Friedländer MR, Mackowiak SD, Li N, Chen W, and Rajewsky N (2012). miRDeep2 accurately identifies known and hundreds of novel microRNA genes in seven animal clades. Nucleic Acids Res. 40, 37–52.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkr688" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3245920/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21911355/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=miRDeep2%20accurately%20identifies%20known%20and%20hundreds%20of%20novel%20microRNA%20genes%20in%20seven%20animal%20clades&amp;author=MR%20Friedl%C3%A4nder&amp;author=SD%20Mackowiak&amp;author=N%20Li&amp;author=W%20Chen&amp;author=N%20Rajewsky&amp;volume=40&amp;publication_year=2012&amp;pages=37-52&amp;pmid=21911355&amp;doi=10.1093/nar/gkr688&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R23">
<cite>Fromm B, Domanska D, Høye E, Ovchinnikov V, Kang W, Aparicio-Puerta E, Johansen M, Flatmark K, Mathelier A, Hovig E, et al.  (2020). MirGeneDB 2.0: the metazoan microRNA complement. Nucleic Acids Res. 48, D132–D141.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkz885" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6943042/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31598695/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=MirGeneDB%202.0:%20the%20metazoan%20microRNA%20complement&amp;author=B%20Fromm&amp;author=D%20Domanska&amp;author=E%20H%C3%B8ye&amp;author=V%20Ovchinnikov&amp;author=W%20Kang&amp;volume=48&amp;publication_year=2020&amp;pages=D132-D141&amp;pmid=31598695&amp;doi=10.1093/nar/gkz885&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R24">
<cite>Garrett-Bakelman FE, Darshi M, Green SJ, Gur RC, Lin L, Macias BR, McKenna MJ, Meydan C, Mishra T, Nasrini J, et al.  (2019). The NASA Twins Study: A multidimensional analysis of a year-long human spaceflight. Science
364, eaau8650.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.aau8650" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7580864/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30975860/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20NASA%20Twins%20Study:%20A%20multidimensional%20analysis%20of%20a%20year-long%20human%20spaceflight&amp;author=FE%20Garrett-Bakelman&amp;author=M%20Darshi&amp;author=SJ%20Green&amp;author=RC%20Gur&amp;author=L%20Lin&amp;volume=364&amp;publication_year=2019&amp;pages=eaau8650&amp;pmid=30975860&amp;doi=10.1126/science.aau8650&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R25">
<cite>Gerbaix M, Gnyubkin V, Farlay D, Olivier C, Ammann P, Courbon G, Laroche N, Genthial R, Follet H, Peyrin F, et al.  (2017). One-month spaceflight compromises the bone microstructure, tissue-level mechanical properties, osteocyte survival and lacunae volume in mature mice skeletons. Sci. Rep
7, 2659.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-017-03014-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5453937/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28572612/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep&amp;title=One-month%20spaceflight%20compromises%20the%20bone%20microstructure,%20tissue-level%20mechanical%20properties,%20osteocyte%20survival%20and%20lacunae%20volume%20in%20mature%20mice%20skeletons&amp;author=M%20Gerbaix&amp;author=V%20Gnyubkin&amp;author=D%20Farlay&amp;author=C%20Olivier&amp;author=P%20Ammann&amp;volume=7&amp;publication_year=2017&amp;pages=2659&amp;pmid=28572612&amp;doi=10.1038/s41598-017-03014-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R26">
<cite>Gertz ML, Chin CR, Tomoiaga D, MacKay M, Butler D, Afshinnekoo E, Bezdan D, Schmidt MA, Mozsary C, Melnick AM, et al.  (2020). Multiomic, Single-cell, and Biochemical Profiles of Astronauts Guide Pharmacological Strategies for Returning to Gravity. Cell Rep. Published online November 25, 2020.
10.1016/j.celrep.2020.108429.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.celrep.2020.108429" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9444344/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33242408/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Multiomic,%20Single-cell,%20and%20Biochemical%20Profiles%20of%20Astronauts%20Guide%20Pharmacological%20Strategies%20for%20Returning%20to%20Gravity&amp;author=ML%20Gertz&amp;author=CR%20Chin&amp;author=D%20Tomoiaga&amp;author=M%20MacKay&amp;author=D%20Butler&amp;publication_year=2020&amp;pmid=33242408&amp;doi=10.1016/j.celrep.2020.108429&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R27">
<cite>Girardi C, De Pittà C, Casara S, Calura E, Romualdi C, Celotti L, and Mognato M (2014). Integration analysis of microRNA and mRNA expression profiles in human peripheral blood lymphocytes cultured in modeled microgravity. BioMed Res. Int
2014, 296747.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1155/2014/296747" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4090438/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25045661/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BioMed%20Res.%20Int&amp;title=Integration%20analysis%20of%20microRNA%20and%20mRNA%20expression%20profiles%20in%20human%20peripheral%20blood%20lymphocytes%20cultured%20in%20modeled%20microgravity&amp;author=C%20Girardi&amp;author=C%20De%20Pitt%C3%A0&amp;author=S%20Casara&amp;author=E%20Calura&amp;author=C%20Romualdi&amp;volume=2014&amp;publication_year=2014&amp;pages=296747&amp;pmid=25045661&amp;doi=10.1155/2014/296747&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R28">
<cite>Grabham P, Hu B, Sharma P, and Geard C (2011). Effects of ionizing radiation on three-dimensional human vessel models: differential effects according to radiation quality and cellular development. Radiat. Res
175, 21–28.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1667/RR2289.1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21175343/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Radiat.%20Res&amp;title=Effects%20of%20ionizing%20radiation%20on%20three-dimensional%20human%20vessel%20models:%20differential%20effects%20according%20to%20radiation%20quality%20and%20cellular%20development&amp;author=P%20Grabham&amp;author=B%20Hu&amp;author=P%20Sharma&amp;author=C%20Geard&amp;volume=175&amp;publication_year=2011&amp;pages=21-28&amp;pmid=21175343&amp;doi=10.1667/RR2289.1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R29">
<cite>Guo L, Zhang Q, Ma X, Wang J, and Liang T (2017). miRNA and mRNA expression analysis reveals potential sex-biased miRNA expression. Sci. Rep
7, 39812.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/srep39812" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5206641/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28045090/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep&amp;title=miRNA%20and%20mRNA%20expression%20analysis%20reveals%20potential%20sex-biased%20miRNA%20expression&amp;author=L%20Guo&amp;author=Q%20Zhang&amp;author=X%20Ma&amp;author=J%20Wang&amp;author=T%20Liang&amp;volume=7&amp;publication_year=2017&amp;pages=39812&amp;pmid=28045090&amp;doi=10.1038/srep39812&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R30">
<cite>Hastie T, and Tibshirani R (1986). Generalized Additive Models. Stat. Sci
1, 297–310.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1177/096228029500400302" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8548102/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Stat.%20Sci&amp;title=Generalized%20Additive%20Models&amp;author=T%20Hastie&amp;author=R%20Tibshirani&amp;volume=1&amp;publication_year=1986&amp;pages=297-310&amp;pmid=8548102&amp;doi=10.1177/096228029500400302&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R31">
<cite>Hindson CM, Chevillet JR, Briggs HA, Gallichotte EN, Ruf IK, Hindson BJ, Vessella RL, and Tewari M (2013). Absolute quantification by droplet digital PCR versus analog real-time PCR. Nat. Methods
10, 1003–1005.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.2633" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4118677/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23995387/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Absolute%20quantification%20by%20droplet%20digital%20PCR%20versus%20analog%20real-time%20PCR&amp;author=CM%20Hindson&amp;author=JR%20Chevillet&amp;author=HA%20Briggs&amp;author=EN%20Gallichotte&amp;author=IK%20Ruf&amp;volume=10&amp;publication_year=2013&amp;pages=1003-1005&amp;pmid=23995387&amp;doi=10.1038/nmeth.2633&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R32">
<cite>Hughes-Fulford M, Chang TT, Martinez EM, and Li CF (2015). Spaceflight alters expression of microRNA during T-cell activation. FASEB J. 29, 4893–4900.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1096/fj.15-277392" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4653058/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26276131/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FASEB%20J&amp;title=Spaceflight%20alters%20expression%20of%20microRNA%20during%20T-cell%20activation&amp;author=M%20Hughes-Fulford&amp;author=TT%20Chang&amp;author=EM%20Martinez&amp;author=CF%20Li&amp;volume=29&amp;publication_year=2015&amp;pages=4893-4900&amp;pmid=26276131&amp;doi=10.1096/fj.15-277392&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R33">
<cite>Ikushima H, and Miyazono K (2010). Cellular context-dependent “colors” of transforming growth factor-beta signaling. Cancer Sci. 101, 306–312.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1349-7006.2009.01441.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC11158101/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20067465/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&amp;title=Cellular%20context-dependent%20%E2%80%9Ccolors%E2%80%9D%20of%20transforming%20growth%20factor-beta%20signaling&amp;author=H%20Ikushima&amp;author=K%20Miyazono&amp;volume=101&amp;publication_year=2010&amp;pages=306-312&amp;pmid=20067465&amp;doi=10.1111/j.1349-7006.2009.01441.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R34">
<cite>Ishida M, and Selaru FM (2013). miRNA-Based Therapeutic Strategies. Curr. Anesthesiol. Rep
1, 63–70.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s40139-012-0004-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3601800/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23524956/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Anesthesiol.%20Rep&amp;title=miRNA-Based%20Therapeutic%20Strategies&amp;author=M%20Ishida&amp;author=FM%20Selaru&amp;volume=1&amp;publication_year=2013&amp;pages=63-70&amp;pmid=23524956&amp;doi=10.1007/s40139-012-0004-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R35">
<cite>Jessup JM, Frantz M, Sonmez-Alpan E, Locker J, Skena K, Waller H, Battle P, Nachman A, Bhatti, Weber ME, et al.  (2000). Microgravity culture reduces apoptosis and increases the differentiation of a human colorectal carcinoma cell line. In Vitro Cell Dev. Biol. Anim
36, 367–373.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1290/1071-2690(2000)036&lt;0367:mcraai&gt;2.0.co;2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10949995/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=In%20Vitro%20Cell%20Dev.%20Biol.%20Anim&amp;title=Microgravity%20culture%20reduces%20apoptosis%20and%20increases%20the%20differentiation%20of%20a%20human%20colorectal%20carcinoma%20cell%20line&amp;author=JM%20Jessup&amp;author=M%20Frantz&amp;author=E%20Sonmez-Alpan&amp;author=J%20Locker&amp;author=K%20Skena&amp;volume=36&amp;publication_year=2000&amp;pages=367-373&amp;pmid=10949995&amp;doi=10.1290/1071-2690(2000)036&lt;0367:mcraai&gt;2.0.co;2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R36">
<cite>Kang W, Eldfjell Y, Fromm B, Estivill X, Biryukova I, and Friedländer MR (2018). miRTrace reveals the organismal origins of microRNA sequencing data. Genome Biol. 19, 213.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s13059-018-1588-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6280396/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30514392/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=miRTrace%20reveals%20the%20organismal%20origins%20of%20microRNA%20sequencing%20data&amp;author=W%20Kang&amp;author=Y%20Eldfjell&amp;author=B%20Fromm&amp;author=X%20Estivill&amp;author=I%20Biryukova&amp;volume=19&amp;publication_year=2018&amp;pages=213&amp;pmid=30514392&amp;doi=10.1186/s13059-018-1588-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R37">
<cite>Konovalova J, Gerasymchuk D, Parkkinen I, Chmielarz P, and Domanskyi A (2019). Interplay between MicroRNAs and Oxidative Stress in Neurodegenerative Diseases. Int. J. Mol. Sci
20, 6055.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/ijms20236055" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6929013/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31801298/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci&amp;title=Interplay%20between%20MicroRNAs%20and%20Oxidative%20Stress%20in%20Neurodegenerative%20Diseases&amp;author=J%20Konovalova&amp;author=D%20Gerasymchuk&amp;author=I%20Parkkinen&amp;author=P%20Chmielarz&amp;author=A%20Domanskyi&amp;volume=20&amp;publication_year=2019&amp;pages=6055&amp;pmid=31801298&amp;doi=10.3390/ijms20236055&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R38">
<cite>Kozomara A, Birgaoanu M, and Griffiths-Jones S (2019). miRBase: from microRNA sequences to function. Nucleic Acids Res. 47, D155–D162.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gky1141" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6323917/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30423142/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=miRBase:%20from%20microRNA%20sequences%20to%20function&amp;author=A%20Kozomara&amp;author=M%20Birgaoanu&amp;author=S%20Griffiths-Jones&amp;volume=47&amp;publication_year=2019&amp;pages=D155-D162&amp;pmid=30423142&amp;doi=10.1093/nar/gky1141&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R39">
<cite>Krützfeldt J, Rajewsky N, Braich R, Rajeev KG, Tuschl T, Manoharan M, and Stoffel M (2005). Silencing of microRNAs in vivo with ‘antagomirs’. Nature
438, 685–689.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature04303" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16258535/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Silencing%20of%20microRNAs%20in%20vivo%20with%20%E2%80%98antagomirs%E2%80%99&amp;author=J%20Kr%C3%BCtzfeldt&amp;author=N%20Rajewsky&amp;author=R%20Braich&amp;author=KG%20Rajeev&amp;author=T%20Tuschl&amp;volume=438&amp;publication_year=2005&amp;pages=685-689&amp;pmid=16258535&amp;doi=10.1038/nature04303&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R40">
<cite>Macfarlane LA, and Murphy PR (2010). MicroRNA: Biogenesis, Function and Role in Cancer. Curr. Genomics
11, 537–561.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2174/138920210793175895" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3048316/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21532838/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Genomics&amp;title=MicroRNA:%20Biogenesis,%20Function%20and%20Role%20in%20Cancer&amp;author=LA%20Macfarlane&amp;author=PR%20Murphy&amp;volume=11&amp;publication_year=2010&amp;pages=537-561&amp;pmid=21532838&amp;doi=10.2174/138920210793175895&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R41">
<cite>Mader TH, Gibson CR, Pass AF, Kramer LA, Lee AG, Fogarty J, Tarver WJ, Dervay JP, Hamilton DR, Sargsyan A, et al.  (2011). Optic disc edema, globe flattening, choroidal folds, and hyperopic shifts observed in astronauts after long-duration spaceflight. Ophthalmology
118, 2058–2069.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ophtha.2011.06.021" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21849212/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=Optic%20disc%20edema,%20globe%20flattening,%20choroidal%20folds,%20and%20hyperopic%20shifts%20observed%20in%20astronauts%20after%20long-duration%20spaceflight&amp;author=TH%20Mader&amp;author=CR%20Gibson&amp;author=AF%20Pass&amp;author=LA%20Kramer&amp;author=AG%20Lee&amp;volume=118&amp;publication_year=2011&amp;pages=2058-2069&amp;pmid=21849212&amp;doi=10.1016/j.ophtha.2011.06.021&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R42">
<cite>Makedonas G, Chouker A, Mehta S, Simpson R, Stowe R, Sams C, Pierson D, and Crucian B (2018). Mechanistic Clues to Overcome Spaceflight-Induced Immune Dysregulation. Curr. Pathobiol. Rep
6, 185–192.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Pathobiol.%20Rep&amp;title=Mechanistic%20Clues%20to%20Overcome%20Spaceflight-Induced%20Immune%20Dysregulation&amp;author=G%20Makedonas&amp;author=A%20Chouker&amp;author=S%20Mehta&amp;author=R%20Simpson&amp;author=R%20Stowe&amp;volume=6&amp;publication_year=2018&amp;pages=185-192&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R43">
<cite>Marie JC, Liggitt D, and Rudensky AY (2006). Cellular mechanisms of fatal early-onset autoimmunity in mice with the T cell-specific targeting of transforming growth factor-beta receptor. Immunity
25, 441–454.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.immuni.2006.07.012" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16973387/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Cellular%20mechanisms%20of%20fatal%20early-onset%20autoimmunity%20in%20mice%20with%20the%20T%20cell-specific%20targeting%20of%20transforming%20growth%20factor-beta%20receptor&amp;author=JC%20Marie&amp;author=D%20Liggitt&amp;author=AY%20Rudensky&amp;volume=25&amp;publication_year=2006&amp;pages=441-454&amp;pmid=16973387&amp;doi=10.1016/j.immuni.2006.07.012&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R44">
<cite>Mewaldt RA (1994). Galactic cosmic ray composition and energy spectra. Adv. Space Res
14, 737–747.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0273-1177(94)90536-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11540019/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res&amp;title=Galactic%20cosmic%20ray%20composition%20and%20energy%20spectra&amp;author=RA%20Mewaldt&amp;volume=14&amp;publication_year=1994&amp;pages=737-747&amp;pmid=11540019&amp;doi=10.1016/0273-1177(94)90536-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R45">
<cite>Morey-Holton ER, and Globus RK (1998). Hindlimb unloading of growing rats: a model for predicting skeletal changes during space flight. Bone
22, 83S–88S.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s8756-3282(98)00019-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9600759/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bone&amp;title=Hindlimb%20unloading%20of%20growing%20rats:%20a%20model%20for%20predicting%20skeletal%20changes%20during%20space%20flight&amp;author=ER%20Morey-Holton&amp;author=RK%20Globus&amp;volume=22&amp;publication_year=1998&amp;pages=83S-88S&amp;pmid=9600759&amp;doi=10.1016/s8756-3282(98)00019-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R46">
<cite>Mortreux M, Nagy JA, Ko FC, Bouxsein ML, and Rutkove SB (2018). A novel partial gravity ground-based analog for rats via quadrupedal unloading. J. Appl. Physiol
125, 175–182.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1152/japplphysiol.01083.2017" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29565773/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Physiol&amp;title=A%20novel%20partial%20gravity%20ground-based%20analog%20for%20rats%20via%20quadrupedal%20unloading&amp;author=M%20Mortreux&amp;author=JA%20Nagy&amp;author=FC%20Ko&amp;author=ML%20Bouxsein&amp;author=SB%20Rutkove&amp;volume=125&amp;publication_year=2018&amp;pages=175-182&amp;pmid=29565773&amp;doi=10.1152/japplphysiol.01083.2017&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R47">
<cite>Otten J, Bokemeyer C, and Fiedler W (2010). Tgf-Beta superfamily receptors-targets for antiangiogenic therapy?
J. Oncol
2010, 317068.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1155/2010/317068" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2871186/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20490264/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Oncol&amp;title=Tgf-Beta%20superfamily%20receptors-targets%20for%20antiangiogenic%20therapy?&amp;author=J%20Otten&amp;author=C%20Bokemeyer&amp;author=W%20Fiedler&amp;volume=2010&amp;publication_year=2010&amp;pages=317068&amp;pmid=20490264&amp;doi=10.1155/2010/317068&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R48">
<cite>Paraskevopoulou MD, Georgakilas G, Kostoulas N, Vlachos IS, Vergoulis T, Reczko M, Filippidis C, Dalamagas T, and Hatzigeorgiou AG (2013). DIANA-microT web server v5.0: service integration into miRNA functional analysis workflows. Nucleic Acids Res. 41, W169–73.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkt393" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3692048/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23680784/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=DIANA-microT%20web%20server%20v5.0:%20service%20integration%20into%20miRNA%20functional%20analysis%20workflows&amp;author=MD%20Paraskevopoulou&amp;author=G%20Georgakilas&amp;author=N%20Kostoulas&amp;author=IS%20Vlachos&amp;author=T%20Vergoulis&amp;volume=41&amp;publication_year=2013&amp;pages=W169-73&amp;pmid=23680784&amp;doi=10.1093/nar/gkt393&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R49">
<cite>Paul AM, Cheng-Campbell M, Blaber EA, Anand S, Bhattacharya S, Zwart S, Crucian B, Smith SM, Meller R, Grabham P, et al.  (2020). Beyond Low-Earth Orbit: Characterizing the Immune Profile Following Simulated Spaceflight Conditions for Deep Space Missions. iScience. Published online November 25, 2020.
10.1016/j.isci.2020.101747.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.isci.2020.101747" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7756144/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33376970/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=Beyond%20Low-Earth%20Orbit:%20Characterizing%20the%20Immune%20Profile%20Following%20Simulated%20Spaceflight%20Conditions%20for%20Deep%20Space%20Missions&amp;author=AM%20Paul&amp;author=M%20Cheng-Campbell&amp;author=EA%20Blaber&amp;author=S%20Anand&amp;author=S%20Bhattacharya&amp;publication_year=2020&amp;pmid=33376970&amp;doi=10.1016/j.isci.2020.101747&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R50">
<cite>Pecaut MJ, Mao XW, Bellinger DL, Jonscher KR, Stodieck LS, Ferguson VL, Bateman TA, Mohney RP, and Gridley DS (2017). Is spaceflight-induced immune dysfunction linked to systemic changes in metabolism?
PLoS ONE
12, e0174174.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0174174" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5443495/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28542224/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Is%20spaceflight-induced%20immune%20dysfunction%20linked%20to%20systemic%20changes%20in%20metabolism?&amp;author=MJ%20Pecaut&amp;author=XW%20Mao&amp;author=DL%20Bellinger&amp;author=KR%20Jonscher&amp;author=LS%20Stodieck&amp;volume=12&amp;publication_year=2017&amp;pages=e0174174&amp;pmid=28542224&amp;doi=10.1371/journal.pone.0174174&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R51">
<cite>Pechenkova E, Nosikova I, Rumshiskaya A, Litvinova L, Rukavishnikov I, Mershina E, Sinitsyn V, Van Ombergen A, Jeurissen B, Jillings S, et al.  (2019). Alterations of Functional Brain Connectivity After Long-Duration Spaceflight as Revealed by fMRI. Front. Physiol
10, 761.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fphys.2019.00761" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6621543/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31333476/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Physiol&amp;title=Alterations%20of%20Functional%20Brain%20Connectivity%20After%20Long-Duration%20Spaceflight%20as%20Revealed%20by%20fMRI&amp;author=E%20Pechenkova&amp;author=I%20Nosikova&amp;author=A%20Rumshiskaya&amp;author=L%20Litvinova&amp;author=I%20Rukavishnikov&amp;volume=10&amp;publication_year=2019&amp;pages=761&amp;pmid=31333476&amp;doi=10.3389/fphys.2019.00761&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R52">
<cite>Pizzamiglio S, Zanutto S, Ciniselli CM, Belfiore A, Bottelli S, Gariboldi M, and Verderio P (2017). A methodological procedure for evaluating the impact of hemolysis on circulating microRNAs. Oncol. Lett
13, 315–320.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3892/ol.2016.5452" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5244842/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28123561/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Oncol.%20Lett&amp;title=A%20methodological%20procedure%20for%20evaluating%20the%20impact%20of%20hemolysis%20on%20circulating%20microRNAs&amp;author=S%20Pizzamiglio&amp;author=S%20Zanutto&amp;author=CM%20Ciniselli&amp;author=A%20Belfiore&amp;author=S%20Bottelli&amp;volume=13&amp;publication_year=2017&amp;pages=315-320&amp;pmid=28123561&amp;doi=10.3892/ol.2016.5452&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R53">
<cite>Poel D, Buffart TE, Oosterling-Jansen J, Verheul HMW, and Voortman J (2018). Evaluation of several methodological challenges in circulating miRNA qPCR studies in patients with head and neck cancer. Exp. Mol. Med
50, e454.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/emm.2017.288" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5898892/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29520111/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Exp.%20Mol.%20Med&amp;title=Evaluation%20of%20several%20methodological%20challenges%20in%20circulating%20miRNA%20qPCR%20studies%20in%20patients%20with%20head%20and%20neck%20cancer&amp;author=D%20Poel&amp;author=TE%20Buffart&amp;author=J%20Oosterling-Jansen&amp;author=HMW%20Verheul&amp;author=J%20Voortman&amp;volume=50&amp;publication_year=2018&amp;pages=e454&amp;pmid=29520111&amp;doi=10.1038/emm.2017.288&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R54">
<cite>Reczko M, Maragkakis M, Alexiou P, Grosse I, and Hatzigeorgiou AG (2012). Functional microRNA targets in protein coding sequences. Bioinformatics
28, 771–776.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/bts043" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22285563/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Functional%20microRNA%20targets%20in%20protein%20coding%20sequences&amp;author=M%20Reczko&amp;author=M%20Maragkakis&amp;author=P%20Alexiou&amp;author=I%20Grosse&amp;author=AG%20Hatzigeorgiou&amp;volume=28&amp;publication_year=2012&amp;pages=771-776&amp;pmid=22285563&amp;doi=10.1093/bioinformatics/bts043&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R55">
<cite>Režen T, Kovanda A, Eiken O, Mekjavic IB, and Rogelj B (2014). Expression changes in human skeletal muscle miRNAs following 10 days of bed rest in young healthy males. Acta Physiol. (Oxf.)
210, 655–666.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/apha.12228" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24410893/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Acta%20Physiol.%20(Oxf.)&amp;title=Expression%20changes%20in%20human%20skeletal%20muscle%20miRNAs%20following%2010%20days%20of%20bed%20rest%20in%20young%20healthy%20males&amp;author=T%20Re%C5%BEen&amp;author=A%20Kovanda&amp;author=O%20Eiken&amp;author=IB%20Mekjavic&amp;author=B%20Rogelj&amp;volume=210&amp;publication_year=2014&amp;pages=655-666&amp;pmid=24410893&amp;doi=10.1111/apha.12228&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R56">
<cite>Ronca AE, Moyer EL, Talyansky Y, Lowe M, Padmanabhan S, Choi S, Gong C, Cadena SM, Stodieck L, and Globus RK (2019). Behavior of mice aboard the International Space Station. Sci. Rep
9, 4717.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-019-40789-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6459880/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30976012/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep&amp;title=Behavior%20of%20mice%20aboard%20the%20International%20Space%20Station&amp;author=AE%20Ronca&amp;author=EL%20Moyer&amp;author=Y%20Talyansky&amp;author=M%20Lowe&amp;author=S%20Padmanabhan&amp;volume=9&amp;publication_year=2019&amp;pages=4717&amp;pmid=30976012&amp;doi=10.1038/s41598-019-40789-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R57">
<cite>Rooke HM, and Crosier KE (2001). The smad proteins and TGFbeta signalling: uncovering a pathway critical in cancer. Pathology
33, 73–84.</cite> [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11280614/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Pathology&amp;title=The%20smad%20proteins%20and%20TGFbeta%20signalling:%20uncovering%20a%20pathway%20critical%20in%20cancer&amp;author=HM%20Rooke&amp;author=KE%20Crosier&amp;volume=33&amp;publication_year=2001&amp;pages=73-84&amp;pmid=11280614&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R58">
<cite>Ru Y, Kechris KJ, Tabakoff B, Hoffman P, Radcliffe RA, Bowler R, Mahaffey S, Rossi S, Calin GA, Bemis L, and Theodorescu D (2014). The multiMiR R package and database: integration of microRNA-target interactions along with their disease and drug associations. Nucleic Acids Res. 42, e133.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gku631" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4176155/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25063298/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20multiMiR%20R%20package%20and%20database:%20integration%20of%20microRNA-target%20interactions%20along%20with%20their%20disease%20and%20drug%20associations&amp;author=Y%20Ru&amp;author=KJ%20Kechris&amp;author=B%20Tabakoff&amp;author=P%20Hoffman&amp;author=RA%20Radcliffe&amp;volume=42&amp;publication_year=2014&amp;pages=e133&amp;pmid=25063298&amp;doi=10.1093/nar/gku631&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R59">
<cite>Rullman E, Mekjavic IB, Fischer H, and Eiken O (2016). PlanHab (Planetary Habitat Simulation): the combined and separate effects of 21 days bed rest and hypoxic confinement on human skeletal muscle miRNA expression. Physiol. Rep
4, e12753.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.14814/phy2.12753" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4848719/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27117806/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Physiol.%20Rep&amp;title=PlanHab%20(Planetary%20Habitat%20Simulation):%20the%20combined%20and%20separate%20effects%20of%2021%20days%20bed%20rest%20and%20hypoxic%20confinement%20on%20human%20skeletal%20muscle%20miRNA%20expression&amp;author=E%20Rullman&amp;author=IB%20Mekjavic&amp;author=H%20Fischer&amp;author=O%20Eiken&amp;volume=4&amp;publication_year=2016&amp;pages=e12753&amp;pmid=27117806&amp;doi=10.14814/phy2.12753&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R60">
<cite>Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, Amin N, Schwikowski B, and Ideker T (2003). Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res. 13, 2498–2504.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.1239303" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC403769/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14597658/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=Cytoscape:%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&amp;author=P%20Shannon&amp;author=A%20Markiel&amp;author=O%20Ozier&amp;author=NS%20Baliga&amp;author=JT%20Wang&amp;volume=13&amp;publication_year=2003&amp;pages=2498-2504&amp;pmid=14597658&amp;doi=10.1101/gr.1239303&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R61">
<cite>Simone NL, Soule BP, Ly D, Saleh AD, Savage JE, Degraff W, Cook J, Harris CC, Gius D, and Mitchell JB (2009). Ionizing radiation-induced oxidative stress alters miRNA expression. PLoS ONE
4, e6377.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0006377" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2712071/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19633716/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Ionizing%20radiation-induced%20oxidative%20stress%20alters%20miRNA%20expression&amp;author=NL%20Simone&amp;author=BP%20Soule&amp;author=D%20Ly&amp;author=AD%20Saleh&amp;author=JE%20Savage&amp;volume=4&amp;publication_year=2009&amp;pages=e6377&amp;pmid=19633716&amp;doi=10.1371/journal.pone.0006377&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R62">
<cite>Simonsen LC, Slaba TC, Guida P, and Rusek A (2020). NASA’s first ground-based Galactic Cosmic Ray Simulator: Enabling a new era in space radiobiology research. PLoS Biol. 18, e3000669.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pbio.3000669" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7236977/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32428004/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&amp;title=NASA%E2%80%99s%20first%20ground-based%20Galactic%20Cosmic%20Ray%20Simulator:%20Enabling%20a%20new%20era%20in%20space%20radiobiology%20research&amp;author=LC%20Simonsen&amp;author=TC%20Slaba&amp;author=P%20Guida&amp;author=A%20Rusek&amp;volume=18&amp;publication_year=2020&amp;pages=e3000669&amp;pmid=32428004&amp;doi=10.1371/journal.pbio.3000669&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R63">
<cite>Smith CL, and Eppig JT (2009). The mammalian phenotype ontology: enabling robust annotation and comparative analysis. Wiley Interdiscip. Rev. Syst. Biol. Med
1, 390–399.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/wsbm.44" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2801442/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20052305/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Wiley%20Interdiscip.%20Rev.%20Syst.%20Biol.%20Med&amp;title=The%20mammalian%20phenotype%20ontology:%20enabling%20robust%20annotation%20and%20comparative%20analysis&amp;author=CL%20Smith&amp;author=JT%20Eppig&amp;volume=1&amp;publication_year=2009&amp;pages=390-399&amp;pmid=20052305&amp;doi=10.1002/wsbm.44&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R64">
<cite>Stuart T, Butler A, Hoffman P, Hafemeister C, Papalexi E, Mauck WM 3rd, Hao Y, Stoeckius M, Smibert P, and Satija R (2019). Comprehensive Integration of Single-Cell Data. Cell
177, 1888–1902.e1821.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cell.2019.05.031" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6687398/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31178118/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Comprehensive%20Integration%20of%20Single-Cell%20Data&amp;author=T%20Stuart&amp;author=A%20Butler&amp;author=P%20Hoffman&amp;author=C%20Hafemeister&amp;author=E%20Papalexi&amp;volume=177&amp;publication_year=2019&amp;pages=1888-1902&amp;pmid=31178118&amp;doi=10.1016/j.cell.2019.05.031&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R65">
<cite>Tahamtan A, Teymoori-Rad M, Nakstad B, and Salimi V (2018). Anti-Inflammatory MicroRNAs and Their Potential for Inflammatory Diseases Treatment. Front. Immunol
9, 1377.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fimmu.2018.01377" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6026627/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29988529/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol&amp;title=Anti-Inflammatory%20MicroRNAs%20and%20Their%20Potential%20for%20Inflammatory%20Diseases%20Treatment&amp;author=A%20Tahamtan&amp;author=M%20Teymoori-Rad&amp;author=B%20Nakstad&amp;author=V%20Salimi&amp;volume=9&amp;publication_year=2018&amp;pages=1377&amp;pmid=29988529&amp;doi=10.3389/fimmu.2018.01377&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R66">
<cite>Tahimic CGT, Paul AM, Schreurs AS, Torres SM, Rubinstein L, Steczina S, Lowe M, Bhattacharya S, Alwood JS, Ronca AE, and Globus RK (2019). Influence of Social Isolation During Prolonged Simulated Weightlessness by Hindlimb Unloading. Front. Physiol
10, 1147.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fphys.2019.01147" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6753329/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31572207/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Physiol&amp;title=Influence%20of%20Social%20Isolation%20During%20Prolonged%20Simulated%20Weightlessness%20by%20Hindlimb%20Unloading&amp;author=CGT%20Tahimic&amp;author=AM%20Paul&amp;author=AS%20Schreurs&amp;author=SM%20Torres&amp;author=L%20Rubinstein&amp;volume=10&amp;publication_year=2019&amp;pages=1147&amp;pmid=31572207&amp;doi=10.3389/fphys.2019.01147&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R67">
<cite>Templin T, Young EF, and Smilenov LB (2012). Proton radiation-induced miRNA signatures in mouse blood: characterization and comparison with 56Fe-ion and gamma radiation. Int. J. Radiat. Biol
88, 531–539.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3109/09553002.2012.690549" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3590022/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22551419/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Radiat.%20Biol&amp;title=Proton%20radiation-induced%20miRNA%20signatures%20in%20mouse%20blood:%20characterization%20and%20comparison%20with%2056Fe-ion%20and%20gamma%20radiation&amp;author=T%20Templin&amp;author=EF%20Young&amp;author=LB%20Smilenov&amp;volume=88&amp;publication_year=2012&amp;pages=531-539&amp;pmid=22551419&amp;doi=10.3109/09553002.2012.690549&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R68">
<cite>Templin T, Sharma P, Guida P, and Grabham P (2016). Short-Term Effects of Low-LET Radiation on the Endothelial Barrier: Uncoupling of PECAM-1 and the Production of Endothelial Microparticles. Radiat. Res
186, 602–613.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1667/RR14510.1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27905868/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Radiat.%20Res&amp;title=Short-Term%20Effects%20of%20Low-LET%20Radiation%20on%20the%20Endothelial%20Barrier:%20Uncoupling%20of%20PECAM-1%20and%20the%20Production%20of%20Endothelial%20Microparticles&amp;author=T%20Templin&amp;author=P%20Sharma&amp;author=P%20Guida&amp;author=P%20Grabham&amp;volume=186&amp;publication_year=2016&amp;pages=602-613&amp;pmid=27905868&amp;doi=10.1667/RR14510.1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R69">
<cite>Teodori L, Costa A, Campanella L, and Albertini MC (2019). Skeletal Muscle Atrophy in Simulated Microgravity Might Be Triggered by Immune-Related microRNAs. Front. Physiol
9, 1926.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fphys.2018.01926" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6335973/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30687129/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Physiol&amp;title=Skeletal%20Muscle%20Atrophy%20in%20Simulated%20Microgravity%20Might%20Be%20Triggered%20by%20Immune-Related%20microRNAs&amp;author=L%20Teodori&amp;author=A%20Costa&amp;author=L%20Campanella&amp;author=MC%20Albertini&amp;volume=9&amp;publication_year=2019&amp;pages=1926&amp;pmid=30687129&amp;doi=10.3389/fphys.2018.01926&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R70">
<cite>Townsend LW (2005). Implications of the space radiation environment for human exploration in deep space. Radiat. Prot. Dosimetry
115, 44–50.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/rpd/nci141" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16381680/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Radiat.%20Prot.%20Dosimetry&amp;title=Implications%20of%20the%20space%20radiation%20environment%20for%20human%20exploration%20in%20deep%20space&amp;author=LW%20Townsend&amp;volume=115&amp;publication_year=2005&amp;pages=44-50&amp;pmid=16381680&amp;doi=10.1093/rpd/nci141&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R71">
<cite>Ulrich-Lai YM, Figueiredo HF, Ostrander MM, Choi DC, Engeland WC, and Herman JP (2006). Chronic stress induces adrenal hyperplasia and hypertrophy in a subregion-specific manner. Am. J. Physiol. Endocrinol. Metab
291, E965–E973.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1152/ajpendo.00070.2006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16772325/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Physiol.%20Endocrinol.%20Metab&amp;title=Chronic%20stress%20induces%20adrenal%20hyperplasia%20and%20hypertrophy%20in%20a%20subregion-specific%20manner&amp;author=YM%20Ulrich-Lai&amp;author=HF%20Figueiredo&amp;author=MM%20Ostrander&amp;author=DC%20Choi&amp;author=WC%20Engeland&amp;volume=291&amp;publication_year=2006&amp;pages=E965-E973&amp;pmid=16772325&amp;doi=10.1152/ajpendo.00070.2006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R72">
<cite>van der Maaten LJP, and Hinton GE (2008). Visualizing High-Dimensional Data Using t-SNE. J. Mach. Learn. Res
9, 2579–2605.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Mach.%20Learn.%20Res&amp;title=Visualizing%20High-Dimensional%20Data%20Using%20t-SNE&amp;author=LJP%20van%20der%20Maaten&amp;author=GE%20Hinton&amp;volume=9&amp;publication_year=2008&amp;pages=2579-2605&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R73">
<cite>van Dijk D, Sharma R, Nainys J, Yim K, Kathail P, Carr AJ, Burdziak C, Moon KR, Chaffer CL, Pattabiraman D, et al.  (2018). Recovering Gene Interactions from Single-Cell Data Using Data Diffusion. Cell
174, 716–729.e727.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cell.2018.05.061" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6771278/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29961576/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Recovering%20Gene%20Interactions%20from%20Single-Cell%20Data%20Using%20Data%20Diffusion&amp;author=D%20van%20Dijk&amp;author=R%20Sharma&amp;author=J%20Nainys&amp;author=K%20Yim&amp;author=P%20Kathail&amp;volume=174&amp;publication_year=2018&amp;pages=716-729&amp;pmid=29961576&amp;doi=10.1016/j.cell.2018.05.061&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R74">
<cite>Vanderburg C, and Beheshti A (2020). MicroRNAs (miRNAs), the Final Frontier: The Hidden Master Regulators Impacting Biological Response in All Organisms Due to Spaceflight (THREE).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=MicroRNAs%20(miRNAs),%20the%20Final%20Frontier:%20The%20Hidden%20Master%20Regulators%20Impacting%20Biological%20Response%20in%20All%20Organisms%20Due%20to%20Spaceflight%20(THREE)&amp;author=C%20Vanderburg&amp;author=A%20Beheshti&amp;publication_year=2020&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R75">
<cite>Wang S, Zhu W, Xu J, Guo Y, Yan J, Meng L, Jiang C, and Lu S (2019). Interpreting the MicroRNA-15/107 family: interaction identification by combining network based and experiment supported approach. BMC Med. Genet
20, 96.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12881-019-0824-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6544937/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31151434/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Med.%20Genet&amp;title=Interpreting%20the%20MicroRNA-15/107%20family:%20interaction%20identification%20by%20combining%20network%20based%20and%20experiment%20supported%20approach&amp;author=S%20Wang&amp;author=W%20Zhu&amp;author=J%20Xu&amp;author=Y%20Guo&amp;author=J%20Yan&amp;volume=20&amp;publication_year=2019&amp;pages=96&amp;pmid=31151434&amp;doi=10.1186/s12881-019-0824-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R76">
<cite>Wei W, Wang J, He J, and Xie X (2018). Serum microRNA as noninvasive indicator for space radiation. Acta Astronaut. 152, 101–104.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Acta%20Astronaut&amp;title=Serum%20microRNA%20as%20noninvasive%20indicator%20for%20space%20radiation&amp;author=W%20Wei&amp;author=J%20Wang&amp;author=J%20He&amp;author=X%20Xie&amp;volume=152&amp;publication_year=2018&amp;pages=101-104&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R77">
<cite>Westerlind KC, and Turner RT (1995). The skeletal effects of spaceflight in growing rats: tissue-specific alterations in mRNA levels for TGF-beta. J. Bone Miner. Res
10, 843–848.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/jbmr.5650100603" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/7572306/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Bone%20Miner.%20Res&amp;title=The%20skeletal%20effects%20of%20spaceflight%20in%20growing%20rats:%20tissue-specific%20alterations%20in%20mRNA%20levels%20for%20TGF-beta&amp;author=KC%20Westerlind&amp;author=RT%20Turner&amp;volume=10&amp;publication_year=1995&amp;pages=843-848&amp;pmid=7572306&amp;doi=10.1002/jbmr.5650100603&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R78">
<cite>Wickham H (2016). ggplot2: Elegant Graphics for Data Analysis (Springer-Verlag).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ggplot2:%20Elegant%20Graphics%20for%20Data%20Analysis%20(Springer-Verlag)&amp;author=H%20Wickham&amp;publication_year=2016&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R79">
<cite>Wood SN, Pya N, and Säfken B (2015). Smoothing Parameter and Model Selection for General Smooth Models. J. Am. Stat. Assoc
111, 1548–1563.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Stat.%20Assoc&amp;title=Smoothing%20Parameter%20and%20Model%20Selection%20for%20General%20Smooth%20Models&amp;author=SN%20Wood&amp;author=N%20Pya&amp;author=B%20S%C3%A4fken&amp;volume=111&amp;publication_year=2015&amp;pages=1548-1563&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R80">
<cite>Wuu YR, Hu B, Okunola H, Paul AM, Blaber EA, Cheng-Campbell M, Beheshti A, and Grabham P (2020). LET Dependent Low Dose and Synergistic Inhibition of Human Angiogenesis by Charged Particles: Validation of Mi RNAs that Drive Inhibition. iScience. Published online November 25, 2020.
10.1016/j.isci.2020.101771.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.isci.2020.101771" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7756138/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33376971/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=LET%20Dependent%20Low%20Dose%20and%20Synergistic%20Inhibition%20of%20Human%20Angiogenesis%20by%20Charged%20Particles:%20Validation%20of%20Mi%20RNAs%20that%20Drive%20Inhibition&amp;author=YR%20Wuu&amp;author=B%20Hu&amp;author=H%20Okunola&amp;author=AM%20Paul&amp;author=EA%20Blaber&amp;publication_year=2020&amp;pmid=33376971&amp;doi=10.1016/j.isci.2020.101771&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R81">
<cite>Xiao C, Pan C, Liu E, He H, Liu C, Huang Y, Yi S, and Huang D (2019). Differences of microRNA expression profiles between monozygotic twins’ blood samples. Forensic Sci. Int. Genet
41, 152–158.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.fsigen.2019.05.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31132533/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Forensic%20Sci.%20Int.%20Genet&amp;title=Differences%20of%20microRNA%20expression%20profiles%20between%20monozygotic%20twins%E2%80%99%20blood%20samples&amp;author=C%20Xiao&amp;author=C%20Pan&amp;author=E%20Liu&amp;author=H%20He&amp;author=C%20Liu&amp;volume=41&amp;publication_year=2019&amp;pages=152-158&amp;pmid=31132533&amp;doi=10.1016/j.fsigen.2019.05.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R82">
<cite>Zhang J, and Storey KB (2018). RBiomirGS: an all-in-one miRNA gene set analysis solution featuring target mRNA mapping and expression profile integration. PeerJ
6, e4262.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.7717/peerj.4262" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5768164/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29340253/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=RBiomirGS:%20an%20all-in-one%20miRNA%20gene%20set%20analysis%20solution%20featuring%20target%20mRNA%20mapping%20and%20expression%20profile%20integration&amp;author=J%20Zhang&amp;author=KB%20Storey&amp;volume=6&amp;publication_year=2018&amp;pages=e4262&amp;pmid=29340253&amp;doi=10.7717/peerj.4262&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R83">
<cite>Zhang Y, Lu T, Wong M, Wang X, Stodieck L, Karouia F, Story M, and Wu H (2016). Transient gene and microRNA expression profile changes of confluent human fibroblast cells in spaceflight. FASEB J. 30, 2211–2224.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1096/fj.201500121" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26917741/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FASEB%20J&amp;title=Transient%20gene%20and%20microRNA%20expression%20profile%20changes%20of%20confluent%20human%20fibroblast%20cells%20in%20spaceflight&amp;author=Y%20Zhang&amp;author=T%20Lu&amp;author=M%20Wong&amp;author=X%20Wang&amp;author=L%20Stodieck&amp;volume=30&amp;publication_year=2016&amp;pages=2211-2224&amp;pmid=26917741&amp;doi=10.1096/fj.201500121&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ol></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>1</span></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/8441986/bin/NIHMS1653539-supplement-1.pdf">NIHMS1653539-supplement-1.pdf</a><sup> (1.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>2</span></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/8441986/bin/NIHMS1653539-supplement-2.csv">NIHMS1653539-supplement-2.csv</a><sup> (8.9KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><span>3</span></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/8441986/bin/NIHMS1653539-supplement-3.csv">NIHMS1653539-supplement-3.csv</a><sup> (193KB, csv) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The published article includes all datasets generated and analyzed during this study. All miRNA-sequence raw data were deposited on the Database: NASA’s GeneLab data repository/platform. The following identifiers are used for the data in this paper: for all heart tissue related data: GLDS-334, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.26030/cg2g-as49" rel="noopener noreferrer" target="_blank">https://doi.org/10.26030/cg2g-as49</a>; for all liver tissue related data: GLDS-335, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.26030/72ke-1k67" rel="noopener noreferrer" target="_blank">https://doi.org/10.26030/72ke-1k67</a> for all soleus muscle related data: GLDS-337, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.26030/m73g-2477" rel="noopener noreferrer" target="_blank">https://doi.org/10.26030/m73g-2477</a> and for all plasma related data: GLDS-336, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.26030/qasa-rr29" rel="noopener noreferrer" target="_blank">https://doi.org/10.26030/qasa-rr29</a>.</p>
<p>Deposited data from the sequencing data from the NASA Twin Study can be found on the NASA Life Sciences Data Archive (LSDA) and the accession code is not available due to privacy concerns. LSDA is the repository for all human and animal research data, including that associated with this study. LSDA has a public facing portal where data requests can be initiated (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://lsda.jsc.nasa.gov/Request/dataRequestFAQ" rel="noopener noreferrer" target="_blank">https://lsda.jsc.nasa.gov/Request/dataRequestFAQ</a>). The LSDA team provides the appropriate processes, tools, and secure infrastructure for archival of experimental data and dissemination while complying with applicable rules, regulations, policies, and procedures governing the management and archival of sensitive data and information. The LSDA team enables data and information dissemination to the public or to authorized personnel either by providing public access to information or via an approved request process for information and data from the LSDA in accordance with NASA Human Research Program and JSC Institutional Review Board direction.</p></section></section></section></section></article>
</main></body></html>