<!DOCTYPE HTML>
<html lang="en">
<head><script type="text/javascript">/* <![CDATA[ */_cf_loadingtexthtml="<img alt=' ' src='/cf_scripts/scripts/ajax/resources/cf/images/loading.gif'/>";
_cf_contextpath="";
_cf_ajaxscriptsrc="/cf_scripts/scripts/ajax";
_cf_jsonprefix='//';
_cf_websocket_port=8585;
_cf_flash_policy_port=1243;
_cf_clientid='22ECA4488B9A5589C03BCE3C1A2348C4';/* ]]> */</script><script type="text/javascript" src="/cf_scripts/scripts/ajax/yui/yahoo-dom-event/yahoo-dom-event.js"></script>
<script type="text/javascript" src="/cf_scripts/scripts/ajax/yui/container/container-min.js"></script>
<script type="text/javascript" src="/cf_scripts/scripts/ajax/yui/animation/animation-min.js"></script>
<script type="text/javascript" src="/cf_scripts/scripts/ajax/messages/cfmessage.js"></script>
<script type="text/javascript" src="/cf_scripts/scripts/ajax/package/cfajax.js"></script>
<script type="text/javascript" src="/cf_scripts/scripts/ajax/package/cftooltip.js"></script>
<link rel="stylesheet" type="text/css" href="/cf_scripts/scripts/ajax/resources/yui/yui.css" />
<script type="text/javascript" src="/cf_scripts/scripts/ajax/ext/ext-all.js"></script>
<script type="text/javascript" src="/cf_scripts/scripts/ajax/package/cflayout.js"></script>
<link rel="stylesheet" type="text/css" href="/cf_scripts/scripts/ajax/resources/ext/css/ext-all.css" />
<script type="text/javascript" src="/cf_scripts/scripts/ajax/package/cfwindow.js"></script>
<link rel="stylesheet" type="text/css" href="/cf_scripts/scripts/ajax/resources/cf/cf.css" />

	<title>The NASA Task Book</title>
	<meta http-equiv="content-type" content="text/html; charset=UTF-8">
	<meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
<meta http-equiv="Pragma" content="no-cache" />
<meta http-equiv="Expires" content="0" />
<link href="_css/tbe.css" type="text/css" rel="stylesheet">
    


<script async src="https://www.googletagmanager.com/gtag/js?id=G-WHCJMJP4CD"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-WHCJMJP4CD');
</script>



<script async type="text/javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=NASA&subagency=HQ" id="_fed_an_ua_tag"></script>



<script type="text/javascript">/* <![CDATA[ */
	ColdFusion.Ajax.importTag('CFTOOLTIP');
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	
	
	YAHOO.util.Event.addListener("cf_tooltip_1245006494640301","mouseover",ColdFusion.Tooltip.getToolTip,{"_cf_url":"menudesc.cfm?itemid=bpsdata","_cf_query":"","context":"cf_tooltip_1245006494640301","hidedelay":250,"showdelay":200,"autodismissdelay":20000,"preventoverlap":"true"});
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	
	
	YAHOO.util.Event.addListener("cf_tooltip_1245006494640302","mouseover",ColdFusion.Tooltip.getToolTip,{"_cf_url":"menudesc.cfm?itemid=sbplan","_cf_query":"","context":"cf_tooltip_1245006494640302","hidedelay":250,"showdelay":200,"autodismissdelay":20000,"preventoverlap":"true"});
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	
	
	YAHOO.util.Event.addListener("cf_tooltip_1245006494640303","mouseover",ColdFusion.Tooltip.getToolTip,{"_cf_url":"menudesc.cfm?itemid=hrpd","_cf_query":"","context":"cf_tooltip_1245006494640303","hidedelay":250,"showdelay":200,"autodismissdelay":20000,"preventoverlap":"true"});
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	
	
	YAHOO.util.Event.addListener("cf_tooltip_1245006494640304","mouseover",ColdFusion.Tooltip.getToolTip,{"_cf_url":"menudesc.cfm?itemid=hrr","_cf_query":"","context":"cf_tooltip_1245006494640304","hidedelay":250,"showdelay":200,"autodismissdelay":20000,"preventoverlap":"true"});
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	
	
	YAHOO.util.Event.addListener("cf_tooltip_1245006494640305","mouseover",ColdFusion.Tooltip.getToolTip,{"_cf_url":"menudesc.cfm?itemid=techport","_cf_query":"","context":"cf_tooltip_1245006494640305","hidedelay":250,"showdelay":200,"autodismissdelay":20000,"preventoverlap":"true"});
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	
	
	YAHOO.util.Event.addListener("cf_tooltip_1245006494640306","mouseover",ColdFusion.Tooltip.getToolTip,{"_cf_url":"menudesc.cfm?itemid=issre","_cf_query":"","context":"cf_tooltip_1245006494640306","hidedelay":250,"showdelay":200,"autodismissdelay":20000,"preventoverlap":"true"});
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	
	
	YAHOO.util.Event.addListener("cf_tooltip_1245006494640307","mouseover",ColdFusion.Tooltip.getToolTip,{"_cf_url":"menudesc.cfm?itemid=lsda","_cf_query":"","context":"cf_tooltip_1245006494640307","hidedelay":250,"showdelay":200,"autodismissdelay":20000,"preventoverlap":"true"});
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	
	
	YAHOO.util.Event.addListener("cf_tooltip_1245006494640308","mouseover",ColdFusion.Tooltip.getToolTip,{"_cf_url":"menudesc.cfm?itemid=nspires","_cf_query":"","context":"cf_tooltip_1245006494640308","hidedelay":250,"showdelay":200,"autodismissdelay":20000,"preventoverlap":"true"});
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	
	
	YAHOO.util.Event.addListener("cf_tooltip_1245006494640309","mouseover",ColdFusion.Tooltip.getToolTip,{"_cf_url":"menudesc.cfm?itemid=tri","_cf_query":"","context":"cf_tooltip_1245006494640309","hidedelay":250,"showdelay":200,"autodismissdelay":20000,"preventoverlap":"true"});
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	
	
	YAHOO.util.Event.addListener("cf_tooltip_1245006494640310","mouseover",ColdFusion.Tooltip.getToolTip,{"_cf_url":"menudesc.cfm?itemid=nsrl","_cf_query":"","context":"cf_tooltip_1245006494640310","hidedelay":250,"showdelay":200,"autodismissdelay":20000,"preventoverlap":"true"});
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	ColdFusion.Ajax.importTag('CFLAYOUT-TAB');
/* ]]> */</script>

	<script type="text/javascript">/* <![CDATA[ */
		Ext.onReady(function(){var _cf_layout_init_1245006494640311=function()
		{
			var _cf_layout=ColdFusion.Layout.initializeTabLayout("contentLayout", false, null, null, false);
			ColdFusion.Layout.addTab(_cf_layout,"cf_layoutareaReport2011","Report2011","&nbsp;&nbsp;Final Report&nbsp;&nbsp;",null,false,false,false,true);
			ColdFusion.Layout.selectTab('contentLayout','Report2011');
			ColdFusion.Layout.addTab(_cf_layout,"cf_layoutareaReport2010","Report2010","&nbsp;&nbsp;FY 2010 Report&nbsp;&nbsp;",null,false,false,false,true);
			ColdFusion.Layout.addTab(_cf_layout,"cf_layoutareaReport2009","Report2009","&nbsp;&nbsp;FY 2009 Report&nbsp;&nbsp;",null,false,false,false,true);
			
		};ColdFusion.Event.registerOnLoad(_cf_layout_init_1245006494640311);});
	/* ]]> */</script>
	
<script type="text/javascript">/* <![CDATA[ */
	ColdFusion.Ajax.importTag('CFWINDOW');
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	Ext.onReady(function(){var _cf_window_init_1245006494640313=function()
	{
		_cf_bind_init_1245006494640314=function()
		{
			ColdFusion.Bind.register([],{'bindTo':'popbibinfo-body','bindExpr':['stackbib.cfm']},ColdFusion.Bind.urlBindHandler,false);
		};ColdFusion.Event.registerOnLoad(_cf_bind_init_1245006494640314);var _cf_window=ColdFusion.Window.create('popbibinfo','Bibliographic Listing of All Years','stackbib.cfm',{ modal:false, closable:true, divid:'cf_window1245006494640312', draggable:true, resizable:true, fixedcenter:true, width:750, height:500, shadow:true, bodystyle:'border-width:2px; background-color:fff', callfromtag:true, minwidth:0, minheight:0, initshow:false, destroyonclose:false, _cf_refreshOnShow:true});
	};ColdFusion.Event.registerOnLoad(_cf_window_init_1245006494640313);});
/* ]]> */</script>

<script type="text/javascript">/* <![CDATA[ */
	Ext.onReady(function(){var _cf_window_init_1245006494640316=function()
	{
		_cf_bind_init_1245006494640317=function()
		{
			ColdFusion.Bind.register([['tid','','change']],{'bindTo':'poptaskinfo-body','bindExpr':['taskinfo.cfm?t=',['tid','','value']]},ColdFusion.Bind.urlBindHandler,false);
		};ColdFusion.Event.registerOnLoad(_cf_bind_init_1245006494640317);var _cf_window=ColdFusion.Window.create('poptaskinfo','Task Brief Information','taskinfo.cfm?t={tid}',{ modal:false, closable:true, divid:'cf_window1245006494640315', draggable:true, resizable:true, fixedcenter:false, width:900, height:800, shadow:true, bodystyle:'border-width:12px; background-color:#f0f0f0; margin-top: -10px; margin-left: -10px;', callfromtag:true, minwidth:0, minheight:700, initshow:false, destroyonclose:false, x:200, y:250, _cf_refreshOnShow:true});
	};ColdFusion.Event.registerOnLoad(_cf_window_init_1245006494640316);});
/* ]]> */</script>
</head>



<body class="homebg">

	<div id="header">
	<a href="welcome.cfm" title="NASA Task Book"><img src="_images/nasaLogo.svg" alt="nasa" width="80" border="0" align="left" />
    <br><span class="sitehead">Task Book</span></a>
    <span id="headerlink">
    <a href="welcome.cfm" title="Home"><img src="_images/home-solid_white.svg" width="20" alt="" />Home</a>&nbsp;&nbsp; 
    
	<a href="contactus.cfm" title="Contact us">Contact Us</a> &nbsp;&nbsp; &nbsp;&nbsp;
	</span>
</div>
<div class="downtime">Due to the lapse in federal government funding, NASA is not updating this website. We sincerely regret this inconvenience.</div>

<div id="content">
	<!-- begin left menu -->
	<div id="left">
	<p>&nbsp;</p>
	<div class="left_header_bg"> 
	  <h2>Menu</h2>
	</div>

	<div id="menus">
	<ul>
		<li><a href="index.cfm" title="Search">Search Task Book</a><br /><span class="description">(FY2004 - present)</span></li>
			<ul>
				<li><a href="maps_metrics.cfm" title="Map Metrics">Maps/Metrics</a></li>
			</ul>
		<li><a href="archives.cfm" title="Archives">Search Task Book Archives </a><span class="description">(FY1995 - 2003)</span></li>
		<li><a href="index.cfm?action=bib_search" title="Bib Search">Search Bibliography</a><br /><span class="description">(FY2004 - present)</span></li>
        <li><a href="spaceline.cfm" title="Spaceline">SPACELINE Current Awareness Lists</a></li>
	    <li><a href="highlights.cfm" title="Highlights">Space Life & Physical Sciences Research Highlights</a></li>
		<li><a href="../exploration/taskbook/" title="PI Submission site" target="internal">PI Submission Site</a></li>
	    <li>Related Links
			<ul>
            	<li class="description"><span  id="cf_tooltip_1245006494640301">
	
	<a href="https://science.nasa.gov/biological-physical/data/" target="new">Biological and Physical Sciences Data (BPS Data)</a>
 </span></li>
            	<li class="description"><span  id="cf_tooltip_1245006494640302">
	
	<a href="https://science.nasa.gov/biological-physical/resources/decadal-surveys/" target="new">NASA Biological and Physical Sciences (BPS) Decadal Survey</a>
 </span></li>
			    <li class="description"><span  id="cf_tooltip_1245006494640303">
	
	<a href="https://humanresearchroadmap.nasa.gov/Documents/IRP_Rev-Current.pdf" target="new">NASA Human Research Program Integrated Research Plan (IRP)</a>
 </span></li>
			    <li class="description"><span  id="cf_tooltip_1245006494640304">
	
	<a href="http://humanresearchroadmap.nasa.gov/" target="new">Human Research Roadmap</a>
 </span></li>
				<li class="description"><span  id="cf_tooltip_1245006494640305">
	
	<a href="http://techport.nasa.gov/home" target="new">TechPort</a>
 </span></li>
				<li class="description"><span  id="cf_tooltip_1245006494640306">
	
	<a href="https://www.nasa.gov/mission/station/research-explorer/" target="new">Space Station Research Explorer</a>
 </span></li>
				<li class="description"><span  id="cf_tooltip_1245006494640307">
	
	<a href="https://nlsp.nasa.gov/explore/lsdahome" target="new">NASA Life Sciences Portal</a>
 </span></li>
				<li class="description"><span  id="cf_tooltip_1245006494640308">
	
	<a href="https://nspires.nasaprs.com/external/" target="new">NSPIRES</a>
 </span></li>
				<li class="description"><span  id="cf_tooltip_1245006494640309">
	
	<a href="https://www.bcm.edu/centers/space-medicine/translational-research-institute" target="new">Translational Research Institute for Space Health</a>
 </span></li>
			    <li class="description"><span  id="cf_tooltip_1245006494640310">
	
	<a href="https://www.bnl.gov/nsrl/" target="new">NASA Space Radiation Laboratory(NSRL)</a>
 </span></li>
			    
			</ul>
			
		</li>
	</ul>
	</div>
	<p>&nbsp;</p>
						
</div>
	<!-- end left menu -->
    
	<div id="mainpage">
    		
			<div><h0>The NASA Task Book</h0></div>
			<div id="guide" align="right">
			
                    <a href="index.cfm?st=advanced"><span class="white">Advanced Search</span></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
                
            <br />
            </div>
			<div class="boxpage">
           
				
<form name="data">
<input type="hidden" name="tid" value="0" />

<div  id="contentLayout">
	
	
    <div class="smallr"><a href="index.cfm?action=sort&sort=asc&col=pi" title="List Search Results">Back to Search Results</a>&nbsp;&nbsp;&nbsp;&nbsp;</div>
        <br />
    <div  id="cf_layoutareaReport2011" class="x-hidden" style="overflow:auto;;width:100%;" align="left">
		
		
<div id="bgdata">
	<div class="datarow">
    	<div class="titlerow">
        	<span class="title">Project Title:&nbsp; Molecular basis of LSMMG biofilm resistance <a href="" title="Expand/Reduce" onclick="toggleAll('2011');return false;"><img id="imgall2011" width="20" src="_images/_minus.gif" alt="" /><span id="objAll2011">Reduce</span></a></span>
    		<div id="pidetail">
            	<table width="85%"><tr>
                	
                    <td><span class="bold">Fiscal Year: </span>FY 2011&nbsp;<br/>
                    	<span class="bold">Division: </span> Space Biology&nbsp;<br/>
                        <span class="bold">Research Discipline/Element: </span><br/>
                        <b>Space Biology</b>: Cell & Molecular Biology &nbsp; | Microbiology &nbsp; | Animal Biology: Vertebrate &nbsp;<br/> 
                        </td>
                    <td><span class="bold">Start Date: </span>05/01/2009&nbsp;&nbsp;<br/>
                    	<span class="bold">End Date: </span>04/30/2011&nbsp;&nbsp;<br/>
                        <span class="bold">Task Last Updated: </span>10/20/2011&nbsp;</td>
                    <td><a title="download in pdf" href="tbpdf.cfm?id=8281" download>Download Task Book report in PDF <img src="_images/pdf.gif" alt="pdf" border="0" width="25" /></a><br/>
						</td>
                </tr></table>
            </div>
    	</div>
    </div>
    
    <div class="datarow">
    	<div class="titlerow">
        	
            <a title="toggle" href="" onclick="toggleAll('2011'); return false;" onkeypress="return false;"><span class="title">Principal Investigator/Affiliation:&nbsp;&nbsp; Matin,&nbsp;A.C.&nbsp;&nbsp;Ph.D.&nbsp;/ Stanford University&nbsp;<img id="imgpidetail2011" width="15" src="_images/_minus.gif" alt="" /></a></span>
			<div id="pidetail2011">
            	<table width="85%"><tr>
                	<td align="right" class="title" width="5%">Address:&nbsp;</td>
                    <td> 299 Campus Drive, West&nbsp;<br /> Mail Code 5124&nbsp;<br/>Stanford&nbsp;, CA&nbsp;94305-5101&nbsp;</td>
                    <td>
						<span class="bold">Email: </span><a href="mailto:A.Matin@stanford.edu">A.Matin@stanford.edu</a>&nbsp;<br/>
                        <span class="bold">Phone: </span>(650) 725-4745 &nbsp;&nbsp;<br/>
                        <span class="bold">Congressional District: </span>14&nbsp;<br/>
                        <span class="bold">Web: </span> &nbsp;<br/>
                        
					</td>
                    <td>
                    	<span class="bold">Organization Type: </span> UNIVERSITY&nbsp;<br/>
                        <span class="bold">Organization Name: </span> Stanford University&nbsp;<br/>
                        <span class="bold">Joint Agency: </span> &nbsp;<br/>
                        <span class="bold">Comments: </span> &nbsp;
                    </td>
                </tr>
                </table>
            </div>
    	<table width="100%">
            <tr height="1" bgcolor="#baa"><td colspan="5"></td></tr>
            <tr><td width="5%"><span class="bold">Key Personnel Changes / Previous PI:</span></td>
            <td colspan="3"> Dr. Michael Hsieh, Department of Urology, Stanford University: collaborator.<br/></td></tr>
            <tr height="1" bgcolor="#baa"><td colspan="5"></td></tr>
            </table>
            
    	</div>
    </div>
    
    <div class="datarow">
    	<div class="titlerow">
        	
            <a title="toggle" href="" onclick="toggleAll('2011'); return false;" onkeypress="return false;"><span class="title">Project Information:&nbsp;<span>Grant/Contract No. </span> NNX09AH41G&nbsp;<img id="imggrantdetail2011" width="15" src="_images/_minus.gif" alt="" /></a></span>
    		<div id="grantdetail2011">
            	<table width="100%">
                	<tr>
                    <td width="18%">
                    	<span class="bold">Responsible Center: </span>NASA ARC&nbsp;<br/>
                        <span class="bold">Grant Monitor: </span>Griko, Yuri &nbsp;<br/>
                        <span class="bold">Center Contact: </span>650-604-0519&nbsp;<br/><a href="mailto:Yuri.V.Griko@nasa.gov">Yuri.V.Griko@nasa.gov</a>&nbsp;<br/>
						<span class="bold">Unique ID: </span>7384&nbsp;
                    </td>
                    <td>
						<span class="bold">Solicitation / Funding Source: </span> 2008 Space Biology--Microbial, Plant, Cell Biology: NNH08ZTT003N&nbsp;<br/>
                        <span class="bold">Grant/Contract No.: </span> NNX09AH41G&nbsp;<br/>
                        <span class="bold">Project Type: </span> Ground&nbsp;<br/>
                        <span class="bold">Flight Program: </span> &nbsp;
                        </td>
                    <td>
                    	<span class="bold">No. of Post Docs: </span>3&nbsp;<br/>
                        <span class="bold">No. of PhD Candidates: </span>1&nbsp;<br/>
                        <span class="bold">No. of Master's Candidates: </span>0&nbsp;<br/>
                        <span class="bold">No. of Bachelor's Candidates: </span>1&nbsp;
                    </td>
                    <td>
                    	<span class="bold">No. of PhD Degrees: </span>0&nbsp;<br/>
                        <span class="bold">No. of Master's Degrees: </span>0&nbsp;<br/>
                        <span class="bold">No. of Bachelor's Degrees: </span>1&nbsp;
                    </td>
                	</tr>
                	<tr height="5"><td></td></tr>
                
                	<tr>
                	<td class="title">Space Biology Element: </td>
                    <td colspan="3">
						(1) Cell & Molecular Biology <br />(2) Microbiology <br />(3) Animal Biology: Vertebrate 
                    </td>
                    </tr>
                    <tr height="1" bgcolor="#baa"><td colspan="4"></td></tr>
                    <tr>
                	<td class="title">Space Biology Cross-Element Discipline: </td>
                    <td colspan="3">
                    	(1) Reproductive Biology 
                    </td>
                    </tr>
                    <tr height="1" bgcolor="#baa"><td colspan="4"></td></tr>
                    <tr>
                	<td class="title">Space Biology Special Category: </td>
                    <td colspan="3">
                    	(1) Translational (Countermeasure) Potential                     </td>
                    </tr> 
                	<tr height="2" bgcolor="#baa"><td colspan="4"></td></tr>
                    <tr>
                	<td class="title">Task Description: </td>
                    <td colspan="3"> Biofilms are resistant surface-attached communities of bacteria found on Earth and in spacecraft. Biofilms can cause serious diseases, but are also superior at beneficial processes, e.g., nutrient recycling and life support systems required for space exploration. We have developed Earth-based bioreactors for cultivating bacterial biofilms under low-shear modeled microgravity (LSMMG), which resembles microgravity of space. LSMMG-grown biofilms exhibit greater resistance than their Earth gravity (NG)-grown counterparts, making them even a greater threat to astronaut health during space travel/residence. This increased resistance encompasses both ‘disinfectants’ (e.g., ethanol) as well as antibiotics, and resembles the general stress response (GSR) of bacteria on Earth. The resemblance between LSMMG-conferred resistance and the GSR includes the facts that both make bacteria resistant to killing agents that destroy them by different mechanisms, and that both appear to depend on common regulatory elements, e.g., sigmaS and Hfq proteins. But while GSR and the stress proteins which underpin GSR are now understood in considerable detail, little is known of the mechanistic basis of LSMMG biofilms. The focus of this investigation is to explore the identity of the regulators of GSR (GSRR) and of stress proteins that determine LSMMG biofilm resistance. Wild type and selected GSRR mutant [sigmaS, Hfq, sigmaS-Hfq, or another GSRR double mutant (e.g., sigmaS-sigma54)] biofilms will be grown under LSMMG and NG conditions, using our Earth-based bioreactors. The development and resistance of the biofilms grown under the two conditions will be compared. For determining resistance, mechanistically different stressors (i.e., those with different modes of killing bacteria) including disinfectants (ethanol, NaCl) and antibiotics (ciprofloxacin, ampicillin) will be used. Survival will be determined using live dead stains and confocal microscopy in combination with appropriate software; viable cell count will also be used in selected cases. The relationship between the expression of GSR regulators and LSMMG-conferred resistance will be determined in bulk biofilms by Western analysis. It will also be determined within individual biofilm regions, using multiple approaches which will rely on GFP-expressing regulator fusions, live-dead stains, and confocal microscopy. The LSMMG-induced proteome of the wild type and the mutant showing greatest impairment in LSMMG resistance will be compared to detect the proteins involved in this resistance; their identification will be made more precise by excluding from among the implicated proteins those also induced in the mutant (impaired in LSMMG resistance). IPTG-inducible scFv antibodies will be used to functionally inactivate selected proteins in the wild type at appropriate times during biofilm development to confirm their role in LSMMG resistance. Fundamental information of the mechanistic basis of LSMMG and Earth biofilms will result, which can suggest ways for their better control. This research thus fully addresses the solicitation: it deals with the role of stress proteins in LSMMG biofilm resistance; it utilizes Earth-based approaches that can be tested in space; and it deals with the question of how life responds to gravity in space and on Earth.  <P></td>
                    </tr> 
                	<tr height="2" bgcolor="#baa"><td colspan="4"></td></tr>    
                    <tr>
                	<td class="title">Research Impact/Earth Benefits: </td>
                    <td colspan="3">      Urinary tract infections are primarily caused by bacteria known as uropathogenic Escherichia coli, or “UPEC” for short. These infections cause the disease cystitis, which leads to approximately 8 million doctor visits by women each year in the United States alone. UPEC attach to the epithelium of the bladder and upper urinary tract and form what are known as biofilms. Biofilms are communities of bacteria that attach to surfaces and can evade host immune responses, as well as resist antibiotics. This form of bacterial growth often leads to chronic infections that are difficult to cure. Biofilms also attach to surfaces onboard spacecraft, presenting a potential pathogen reservoir that threatens astronaut health during spaceflight. Our previous work indicated that biofilms exhibit greater resistance to disinfectants and antibiotics in conditions that mimic the microgravity environment on Earth. These conditions are generated in rotating vessel bioreactors, which prevent cell sedimentation and provide a quiescent or “low-shear” fluid environment similar to that of microgravity during spaceflight. We developed a system using glass beads for biofilm surface attachment permitting study of the “low-shear modeled microgravity environment” (LSMMG) on Earth. We found that this environment further increased UPEC resistance to antimicrobial treatment, making the bacteria superresistant. The objective of our current study is to better understand the stress-response mechanisms by which UPEC cope with the stress generated by antimicrobials. This includes UPEC in slow growing states such as stationary-phase and biofilms. LSMMG bioreactors provide a unique environment for studying resistance mechanisms of an important bacterial pathogen. We are also studying these stress response mechanisms under normal gravity conditions to better understand the underlying biology/biochemistry and to provide an appropriate framework for comparing the LSMMG results. Insights from our work will provide valuable knowledge for developing new forms of treatment aimed against UPEC, as well as other bacterial pathogens, which will benefit humans on Earth as well as onboard spacecraft.  <P></td>
                    </tr>
                </table>
            </div>
    	</div>
    </div>
    
    <div class="datarow">
    	<div class="titlerow">
        	
            <a title="toggle" href="" onclick="toggleAll('2011'); return false;" onkeypress="return false;"><span class="title">Task Progress & Bibliography Information FY2011&nbsp;<img id="imgtaskdetail2011" width="15" src="_images/_minus.gif" alt="" /></a></span>
    		<div id="taskdetail2011">
            	<table width="100%">
                	<tr>
                    <td width="18%" class="title">
                    	Task Progress:
                    </td>
                    <td>
						The proposed work concerned with testing of several stress-response regulator genes of Escherichia coli, especially sigmaS (sS)  as regards their resistance to antibiotics in our system for growing bacterial biofilms under low-shear modeled microgravity environment (LSMMG) (1). (Bacterial sigma factors play a central role in determining what genes a bacterium will express under a given set of conditions.) This work included both uropathogenic E. coli (UPEC), which causes cystitis, as well as the more standard and better studied E. coli K12 strain; the former for its pathogenic potential and importance and the latter because it provides a richer background framework for interpreting results and has available a complete library of mutants in all individual genes. Our central point of interest are stationary phase bacteria because they as planktonic as well as biofilm forms are responsible for most infections and are highly resistant to killing both by disinfectants as well as antibiotics. The UPEC strain used in these studies is a clinical isolate obtained from a patient at Stanford Hospital and studied in the Matin lab over the past 5 or so years (2,3); it is referred to as strain AMG1.    <P>
It is logical to first establish the mechanism of resistance under normal gravity (NG) conditions in order to properly interpret and put in context the experiments conducted under low-shear modeled microgravity (LSMMG) environment, and this is where we began. We first repeated the starting point experiments of this investigation to statistically reliable results as well as extending them to two both strains of E. coli.    <P>
Stationary phase cultures were exposed to antibiotics in a standard medium (M9) without glucose. No growth can occur in such media, so the bacteria experience antibiotic exposure in stationary phase. The cultures were not shaken to generate low oxygen conditions such as the ones that may be found in the gut (4). At daily intervals, a sample from each culture was serially diluted and plated on a solid nutrient medium (LB agar) to determine viability. The rpoS mutant (missing sS) consistently showed an approximately four-log reduction in the number of viable cells compared to the wild-type regardless of the primary mode of action of the three antibiotics used. Thus, ampicillin is a ß-lactam (inhibits cell wall synthesis), ciprofloxacin is a quinolone (inhibits DNA synthesis) and gentamicin is an aminoglycoside (inhibits protein synthesis). Very similar results were obtained with the K12 strain for all three antibiotics. From this point on, we focused on gentamicin to limit the scope of the investigation to permit an in-depth study and because this antibiotic is a common treatment for cystitis.    <P>
In accordance with our proposal, we also tested the effect of two other stress-response regulators, namely the RNA chaperone Hfq and another sigma factor sN.  Hfq is a positive regulator of RpoS (5), and RpoN has overlapping function with sS. As expected, the hfq mutant behaved like the rpoS mutant, but unexpectedly, not the rpoN mutant (missing sN); the latter finding points to diversity within the regulatory pathways determining resistance to individual antimicrobials and deserves further investigation.    <P>
To gain an insight in the molecular/biochemical mechanism underlying enhanced general antibiotic sensitivity of the stationary phase cells due to the loss of sS, we carried out proteomic analysis using two-dimensional gel electrophoresis (2D-DiGE). We used the K12 wild-type and isogenic sS-deficient strains in these studies because of the advantage offered by a sequenced strain to permit definitive identification of the implicated proteins (genes), and the availability in this strain of an existing gene-deletion library that would facilitate examination of individual proteins (genes) that might be implicated in RpoS-mediatd antibiotic resistance. Briefly, the two stationary phase E. coli strains (wild type and delta rpoS) were incubated in M9 medium (minus glucose) with or without gentamicin (16µg/ml) for 2 days. The cells were lysed, protein concentrations assayed and equal amounts from paired samples (individual strain grown with and without the antibiotic) were labeled by CyDye Difference in Gel Electrophoresis (DiGE) fluorophores (size and charge matched). 2D-DiGE was used (Applied Biomics, Hayward, California) to separate samples using isoelectric focusing in the first dimension and SDS polyacrylamide gel electrophoresis in the second dimension. The gels were scanned using a Typhoon image scanner to reveal CyDye signals. ImageQuant software was used to generate single and overlay images, which were then subjected to DeCyder software analysis to determine differentially expressed protein spots. An Ettan Spot Picker was used to pick spots of interest, which were subsequently identified by MS/MS. The approach increases sensitivity and permits comparison of protein expression levels within and between gels. Cluster analysis was employed to elucidate altered functional pathways.    <P>
Two striking findings resulted. a) Cluster analysis to indicate the key functional pathways altered in response to gentamicin gave the highest enrichment score for peroxidase activity, followed by iron storage, flavin adenine dinucleotide (FAD) and flavoprotein activity, metal ion binding, and altered monosaccharide metabolic/catabolic processes. And b) that a majority of the proteins (13 out of 21) that the absence of sS down-regulated were further decreased by gentamicin treatment, and consistent with the cluster analysis many of these included oxidative stress defense and catabolic proteins. In particular, these studies implicated a metabolic pathway that has never been proposed to be involved in antibiotic sensitivity.    <P>
Among other things, the proteomic analysis suggested that a major reason for greater sensitivity of the rpoS mutant to gentamicin and other antibiotics might be the inability of the latter strain to cope with the oxidative stress, which antibiotic treatment causes (5a). We therefore determined reactive oxygen species (ROS) levels in the wild type and the mutant following antibiotic treatment. These were measured by a commercially available kit (Invitrogen). Controls without antibiotics were used to determine the degree of increase in fluorescence due to antibiotic exposure. The mutant did indeed generate 5-6 times more ROS when challenged with the antibiotics compared to the wild type.    <P>
We rested mutants missing individual genes of the anti-oxidant defense mechanism and other stresses and found with this and other approaches that the oxidative stress did indeed play a role in antibiotic toxicity and this was exacerbated in the rpoS mutant.    <P>
We then asked the question if these findings were applicable to in vivo situations. To pursue this I established collaboration with the group of Dr. Michael Hsieh of our Urology Department who has developed an in vivo mouse model for studying cystitis. Seven week old CBA/J mice (Jackson Laboratories, Bar Harbor, Maine) were acclimated to our animal facility for one week prior to experimental manipulation. Transurethral inoculation of 107 bacteria/50µl/mouse was performed using the wild type or the delta rpoS (rpoS mutant) strain. Serial stationary phase cultures of the two strains were grown in LB broth in ambient temperature and air with shaking at 200 rpm in order to enhance type I pili expression – these are appendages that enable the bacteria to colonize the urinary tract. The bacteria were then suspended in phosphate-buffered saline (PBS). Mice were anesthetized with inhaled isoflurane and their perineums were cleaned with 70% ethanol. To ensure mouse bladders did not overflow during transurethral inoculation, animals were deprived of water for 30 minutes prior to anesthesia induction (to reduce urine output), and encouraged to void by scruffing and gently pressing the low abdomen. Using PE10 polyethylene tubing-covered 30 gauge needles, mice were transurethrally catheterized and the inoculum injected over 5 seconds. Immediately after infection, the mice were administered PBS or 50, 5, or 0.5  µg of gentamicin by subcutaneous injection, followed by four more doses every 2 hours (total of five doses). Two days after infection, the mice were sacrificed by a combination of isoflurane overdose and cervical dislocation. Mouse bladders were removed aseptically and homogenized using sterile 1.6 mm stainless steel beads and a Bullet Blender homogenizer. 100 ml of each bladder homogenate, as well as tenfold dilutions, were cultured on MacConkey agar overnight under ambient conditions. The following day, the number of bacteria was determined in each bladder by plate count. Both wild type and rpoS-deficient UPEC colonized the mouse bladder after transurethral inoculation. At 50 µg gentamicin eliminated both strains from the bladders; at 5 µg gentamicin bladder colonization of both strains continued to be suppressed. However, at 0.5 µg of gentamicin, wild type UPEC could successfully colonize the bladder, whereas rpoS-deficient UPEC was severely hampered (2 log reduction in median bacterial count. We were also able to establish, for the first time, that oxidative stress played a role in gentamicin toxicity in vio.    <P>
That oxidative stress has a central role in the antibacterial activity of antibiotics is consistent with previous findings, but the mechanistic basis indicated by our work differs drastically with the only one previously proposed. Kohanski et al (6) compared the effect on the proteome of E. coli treated with bactericidal or bacteriostatic antibiotics and concluded that the bactericidal antibiotic treatment resulted in an oxidative burst. During respiration, the NADH dehydrogenase of the electron transport chain leaks superoxide (O.) (7); according to Kohanski et al, this leakage is enhanced upon antibiotic treatment due to the oxidative burst. This in turn decomposes iron-sulfur clusters, releases iron, resulting in the Fenton reaction that greatly intensifies the oxidative stress.  They reasoned therefore that elimination of steps that generate NADH will, by minimizing the oxidative burst, protect the bacteria from the antibiotics, as will mutations that decrease the iron-sulfur clusters inside the cell (8).  Indeed, mutant strains lacking the Krebs cycle enzymes aconitase (acnB), isocitrate dehydrogenase (icd) succinate dehydrogenase (suc), and the mutant iscS, (the last impaired in iron sulfur cluster synthesis (8)) showed lowered sensitivity to the antibiotics. This Kohanski et al ascribed to decreased oxidative burst resulting from reduced NADH formation. However, no direct evidence was presented to support the notion that the primary event is the oxidative burst due to transient stimulation of metabolism caused by antibiotic addition. The only justification for this conclusion was the fact that Kohanski et al observed a decrease in NADH, concluding that this occurred because of its rapid oxidation by the respiratory chain NADH dehydrogenase.    <P>
Our proteomic data and subsequent experiments point to a very different primary event and we have also conducted experiments that indicate that the resistance of the mutants used by Kohanski et al can be explained on a basis other than the one they proposed. These results are included in the Final Technical Report to NASA and will be the subject of papers currently in preparation.    <P>
II. Microfuidic high-throughput device for screening flow biofilms    <P>
The above findings point to the desirability of screening small molecule libraries that would selectively inhibit ss (most desirably its unique sequences), and specific elements of the bacterial pathway implicated in antibiotic resistance by our studies in order to potentiate the killing effect of antibiotics. Flow biofilms (also called irrigated biofilms) are grown with continuous flow of fresh medium through the growth chamber and more closely approximate the natural biofilms. No system had been available for conducting high throughput screening of such biofilms. We therefore teamed up with Fluxion Biosciences, Inc (San Francisco) to achieve this. This work has been published (9), and will be briefly summarized here. A device was designed, which consists of microfluidic channels integrated into microtiter well plates. The use of a standard well plate format allowed compatibility with plate readers and multichannel pipettors, and the plate also could be examined microscopically. The microfluidic channels are generated using photolithography etched micropatterned silicon wafers as a mold for polydimethylsiloxane, which forms the sides and roofs of the channels. The bottom of the channels consists of standard 180-µm coverslip glass, which allows microscopic examination. The channels connect pairs of microtiter wells; one well of each pair is the inlet well from which fresh medium is pushed by pneumatic pressure through the channel into the outlet well, which holds the spent medium. The microfluidic channels are 1/10th the size of conventional flow cells, which permits laminar flow at a wider range of flow rates and results in uniform biofilm growth in the entire channel. We found that the Pseudomonas fluorescens PAO1 biofilm formation in this device resembled that found in the conventional flow cells (which are not amenable to high throughput screening).  The system includes an air compressor and electropneumatic regulators to deliver precisely controlled pressure to each channel via a manifold that interfaces with individual wells of the microtiter plate. Thus, the flow (and shear) rate in each channel can be controlled individually. A thin transparent heating stage provides temperature control, and a software interface permits control of the flow rate by the user and saving and reuse of experimental profiles. Unlike conventional flow cell systems, the BioFlux plates are pre-sterilized and do not require assembly or tubing connections, which minimizes the possibility of contamination. The plates each contain 24 channels connecting 48 wells. The BioFlux system controls four plates simultaneously, providing a total of 96 biofilms. Each inlet well and outlet well holds 1.25 ml of liquid, so there can be 11 to 20 h of continuous flow before replenishment is necessary at commonly used flow rates. The small volume of the microchannels also results in conservation of reagents.    <P>
We established that this device can be used to screen the effects of antimicrobial agents on flow biofilms using GFP-expressing PAO1 biofilms. These were treated with bleach or antibiotics (gentamicin, tobramycin, ciprofloxacin, or enrofloxacin) and then stained with propidium iodide (PI), followed by measurement of fluorescence in a plate reader. Both the bleach and the antibiotics caused dose dependent biofilm killing, as determined by fluorescence measured with the plate reader. The quinolones had the most potent effect; ciprofloxacin and enrofloxacin caused complete loss of viability at concentrations of 1 and 10 µg/ml, respectively. As a further check of the reliability of rapid plate reader scanning for determination of viability, we analyzed biofilm images obtained by fluorescence microscopy. The biofilm viability calculated from fluorescence ratios obtained using microscope images agreed with that measured with the plate reader; the correlation coefficient (r=0.9565; P <0.0001) and linear regression analysis verified that there was a strong correlation between the results of the two methods. The fluorescent microscope images confirmed that the quinolones completely eradicated the biofilms (ciprofloxacin at a concentration of 1 µg/ml and enrofloxacin at a concentration of 10 µg/ml), as indicated by the complete absence of fluorescence.    <P>
III. Device for high throughput screening of biofilms under LSMMG conditions    <P>
We have designed a prototype multi-well screening plate, which fits into the 50 ml HARV vessel and will allow screening of multiple mutants under LSMMG. The screening apparatus is composed of multiple growth chambers of dimensions 5 mm deep and 7 mm diameter (volume 462mm3) with a semi-permeable membrane heat-sealed to the rear of the plate and a syringe-permeable rubber mat to the front. This will allow 96 mutants to be screened per overnight HARV run. A randomly tagged mutant library can thus be tested for specific gene expression under LSMMG using standard methods.    <P>
IV. LSMMG studies    <P>
Throughout this project period we initiated studies to test the findings described in Section 1 under LSMMG conditions using our HARV-based system of growing biofilms under such conditions. Two issues prevented much progress on this front. a) The complexity of the system required persistence in the face of conflicting results by workers new to and inexperienced in the system; my previous team was dismantled due to hiatus in microgravity-related funding. And b), perhaps more importantly, the exciting progress that normal gravity studies provided, as discussed above, generated a momentum to obtain a clear picture. My co-workers felt that it was more important to sort out the mechanism in the much more amenable normal gravity system so as to crystallize more clearly the critical experiments that then can be done in the more complex LSMMG conditions – I agreed with them.    <P>
As a result of these studies, we are currently working with NASA Ames Research Center at Moffett Field to test these results for UPEC under conditions of real microgravity on a small satellite taking advantage of the existing PharmaSat hardware, which is being adjusted to the needs for work with bacteria; the test antibiotic is gentamycin. Viability determinations in space and their telemetric transmission to Earth will rely on the use of AlamarBlue (AB). We have obtained results that show that the AB use clearly indicates the much enhanced sensitivity of rpoS UPEC to gentamycin. AB turns magenta due to the activity of bacteria; it is clear that the mutant is much more impaired in generating the magenta color after  gentamicin treatment than the wild type.    <P>
V. Publications resulting from the grant    <P>
One publication and several abstracts have resulted; two more papers will be submitted in the near future:    <P>
Benoit, M.R., Conant, C.G., Ionescu-Zanetti, C., Schwartz, M., and Matin, A. (2010). New device for high-throughput viability screening of flow biofilms. Appl Environ Microbiol 76, 4136-4142.    <P>
Benoit, MR, Matin, A. New Methods for High Throughput Screening of Biofilms. American Society for Microbiology, 109th General Meeting. Philadelphia, PA. May 18, 2009.      <P>
Hsieh MH, Benoit MR, Sylvester MD, Keyhan M, Rheems MS, Kim J, Lim Y, Thathireddy A, Xie S, Matin A. 2009. The Role of rpoS and Related General Stress Response Genes in the Antibiotic. M.H. Hsieh, C.V./page 13/10-8-11     Resistance of Uropathogenic Escherichia coli In Vitro and In Vivo. Symposia of Undergraduate Research and Public Service, 2010, Stanford CA.    <P>
Hsieh MH, Benoit MB, Sylvester MD, Keyhan M, Rhee MS, Kim J, Lim Y-H, Thathireddy A, Matin AC. The Bacterial General Stress Response Confers Antibiotic Resistance to Uropathogens in Vivo. American Academy of Pediatrics, Urology Section, 2010, San Francisco, CA.    <P>
VI. References    <P>
1.	Lynch, S. V., Mukundakrishnan, K., Benoit, M. R., Ayyaswamy, P. S., and Matin, A. (2006) Appl Environ Microbiol 72, 7701-7710    <P>
2.	Lynch, S. V., Dixon, L., Benoit, M. R., Brodie, E. L., Keyhan, M., Hu, P., Ackerley, D. F., Andersen, G. L., and Matin, A. (2007) Antimicrob Agents Chemother 51, 3650-3658    <P>
3.	Stone, G., Wood, P., Dixon, L., Keyhan, M., and Matin, A. (2002) Antimicrob Agents Chemother 46, 2458-2461    <P>
4.	Snyder, J. A., Haugen, B. J., Buckles, E. L., Lockatell, C. V., Johnson, D. E., Donnenberg, M. S., Welch, R. A., and Mobley, H. L. (2004) Infect Immun 72, 6373-6381    <P>
5.	Ruiz, N., and Silhavy, T. J. (2003) J Bacteriol 185, 5984-5992    <P>
5a.         A Matin. Stress, Bacterial: General and Specific. Encyclopedia of Microbiology.                 (Moselio      Schaechter, Editor), pp. 485-500 Oxford: Elsevier.     <P>
6.	Kohanski, M. A., Dwyer, D. J., Hayete, B., Lawrence, C. A., and Collins, J. J. (2007) Cell 130, 797-810    <P>
7.	Imlay, J. A. (2006) Mol Microbiol 59, 1073-1082    <P>
8.	Djaman, O., Outten, F. W., and Imlay, J. A. (2004) J Biol Chem 279, 44590-44599    <P>
9.	Benoit, M. R., Conant, C. G., Ionescu-Zanetti, C., Schwartz, M., and Matin, A. (2010) Appl Environ Microbiol 76, 4136-4142   <P>
                    </td>
                	</tr>
                
                	<tr height="1" bgcolor="#baa"><td colspan="2"></td></tr>
               
                    
					<tr><td width="18%" class="title">
                    	Bibliography:
                    </td>
                    <td>
						Description: (Last Updated: 03/27/2020)&nbsp; 
						<br /><br /><a style="padding-top:5px;font-weight:bold; background-color:#fee" href="#" title="Show Complete List" onclick="document.forms['data'].tid.value='8281'; submitFormBib('data','stackbib_all.cfm','bibcontent2011'); return false;" >Show Cumulative Bibliography</a><br />&nbsp; </td>
                	</tr>
                  
					<tr><td colspan="2">
						<div id="bibcontent2011">
						<table width="100%">
						
                  	
					   
                    <tr height="1" bgcolor="#baa"><td colspan="2"></td></tr>
                    
                    <tr bgcolor="#F0F0FF">
                    <td width="18%" class="title">
                    	Abstracts for Journals and Proceedings
                    </td>
                    <td>
						Benoit MR, Matin A.  "New Methods for High Throughput Screening of Biofilms."  American Society for Microbiology, 109th General Meeting, Philadelphia, PA, May 17-21, 2009. <P>  American Society for Microbiology, 109th General Meeting, Philadelphia, PA, May 17-21, 2009. , May-2009 
                    </td>
                	</tr> 	
					   
                    <tr height="1" bgcolor="#baa"><td colspan="2"></td></tr>
                    
                    <tr bgcolor="#EEEEEE">
                    <td width="18%" class="title">
                    	Abstracts for Journals and Proceedings
                    </td>
                    <td>
						Hsieh MH, Benoit MR, Sylvester MD, Keyhan M, Rheems MS, Kim J, Lim Y, Thathireddy A, Xie S, Matin A.  "The Role of rpoS and Related General Stress Response Genes in the Antibiotic Resistance."  Symposia of Undergraduate Research and Public Service, Stanford, CA, January 2010. <P>  Symposia of Undergraduate Research and Public Service, Stanford, CA, January 2010. , Jan-2010 
                    </td>
                	</tr> 	
					   
                    <tr height="1" bgcolor="#baa"><td colspan="2"></td></tr>
                    
                    <tr bgcolor="#F0F0FF">
                    <td width="18%" class="title">
                    	Abstracts for Journals and Proceedings
                    </td>
                    <td>
						Hsieh MH, Benoit MB, Sylvester MD, Keyhan M, Rhee MS, Kim J, Lim Y-H, Thathireddy A, Matin AC.  "The Bacterial General Stress Response Confers Antibiotic Resistance to Uropathogens in Vivo."  American Academy of Pediatrics 2010 Conference, Urology Section, San Francisco, CA, October 2010. <P>  American Academy of Pediatrics 2010 Conference, Urology Section, San Francisco, CA, October 2010. , Oct-2010 
                    </td>
                	</tr> 	
					   
                    <tr height="1" bgcolor="#baa"><td colspan="2"></td></tr>
                    
                    <tr bgcolor="#EEEEEE">
                    <td width="18%" class="title">
                    	Articles in Peer-reviewed Journals
                    </td>
                    <td>
						Wang JH, Singh R, Benoit M, Keyhan M, Sylvester M, Hsieh M, Thathireddy A, Hsieh YJ, Matin AC.  "Sigma S-dependent antioxidant defense protects stationary-phase Escherichia coli against the bactericidal antibiotic gentamicin."  Antimicrob Agents Chemother. 2014 Oct;58(10):5964-75.  <a target="_blank" href="http://dx.doi.org/10.1128/AAC.03683-14">http://dx.doi.org/10.1128/AAC.03683-14</a>  ; Epub 2014 Jul 28. PubMed <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=25070093">PMID: 25070093</a>; PubMed Central <a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4187989">PMCID: PMC4187989</a>  , Oct-2014 
                    </td>
                	</tr> 	
					   
                    <tr height="1" bgcolor="#baa"><td colspan="2"></td></tr>
                    
                    <tr bgcolor="#F0F0FF">
                    <td width="18%" class="title">
                    	Articles in Peer-reviewed Journals
                    </td>
                    <td>
						Benoit MR, Conant CG, Ionescu-Zanetti C, Schwartz M, Matin A.  "New device for high-throughput viability screening of flow biofilms."  Appl Environ Microbiol. 2010 Jul;76(13):4136-42. Epub 2010 Apr 30. <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=20435763">PMID: 20435763</a>
 , Jul-2010 
                    </td>
                	</tr> 
						
				</td></tr></table>
				</div>
						
				</td></tr>
                </table>
            </div>
    	</div>
    </div>
</div>
    	<div class="smallr"><a href="index.cfm?action=sort&sort=asc&col=pi" title="List Search Results">Back to Search Results</a>&nbsp;&nbsp;&nbsp;&nbsp;</div>

	 </div><div  id="cf_layoutareaReport2010" class="x-hidden" style="overflow:auto;;width:100%;" align="left">
		
		
<div id="bgdata">
	<div class="datarow">
    	<div class="titlerow">
        	<span class="title">Project Title:&nbsp; Molecular basis of LSMMG biofilm resistance <a href="" title="Expand/Reduce" onclick="toggleAll('2010');return false;"><img id="imgall2010" width="20" src="_images/_minus.gif" alt="" /><span id="objAll2010">Reduce</span></a></span>
    		<div id="pidetail">
            	<table width="85%"><tr>
                	
                    <td><span class="bold">Fiscal Year: </span>FY 2010&nbsp;<br/>
                    	<span class="bold">Division: </span> Space Biology&nbsp;<br/>
                        <span class="bold">Research Discipline/Element: </span><br/>
                        <b>Space Biology</b>: Cell & Molecular Biology &nbsp; | Microbiology &nbsp; | Animal Biology: Vertebrate &nbsp;<br/> 
                        </td>
                    <td><span class="bold">Start Date: </span>05/01/2009&nbsp;&nbsp;<br/>
                    	<span class="bold">End Date: </span>04/30/2011&nbsp;&nbsp;<br/>
                        <span class="bold">Task Last Updated: </span>03/02/2010&nbsp;</td>
                    <td><a title="download in pdf" href="tbpdf.cfm?id=7812" download>Download Task Book report in PDF <img src="_images/pdf.gif" alt="pdf" border="0" width="25" /></a><br/>
						</td>
                </tr></table>
            </div>
    	</div>
    </div>
    
    <div class="datarow">
    	<div class="titlerow">
        	
            <a title="toggle" href="" onclick="toggleAll('2010'); return false;" onkeypress="return false;"><span class="title">Principal Investigator/Affiliation:&nbsp;&nbsp; Matin,&nbsp;A.C.&nbsp;&nbsp;Ph.D.&nbsp;/ Stanford University&nbsp;<img id="imgpidetail2010" width="15" src="_images/_minus.gif" alt="" /></a></span>
			<div id="pidetail2010">
            	<table width="85%"><tr>
                	<td align="right" class="title" width="5%">Address:&nbsp;</td>
                    <td> 299 Campus Drive, West&nbsp;<br /> Mail Code 5124&nbsp;<br/>Stanford&nbsp;, CA&nbsp;94305-5101&nbsp;</td>
                    <td>
						<span class="bold">Email: </span><a href="mailto:A.Matin@stanford.edu">A.Matin@stanford.edu</a>&nbsp;<br/>
                        <span class="bold">Phone: </span>(650) 725-4745 &nbsp;&nbsp;<br/>
                        <span class="bold">Congressional District: </span>14&nbsp;<br/>
                        <span class="bold">Web: </span> &nbsp;<br/>
                        
					</td>
                    <td>
                    	<span class="bold">Organization Type: </span> UNIVERSITY&nbsp;<br/>
                        <span class="bold">Organization Name: </span> Stanford University&nbsp;<br/>
                        <span class="bold">Joint Agency: </span> &nbsp;<br/>
                        <span class="bold">Comments: </span> &nbsp;
                    </td>
                </tr>
                </table>
            </div>
    	<table width="100%">
            <tr height="1" bgcolor="#baa"><td colspan="5"></td></tr>
            <tr><td width="5%"><span class="bold">Key Personnel Changes / Previous PI:</span></td>
            <td colspan="3"> No personnel changes.<br/></td></tr>
            <tr height="1" bgcolor="#baa"><td colspan="5"></td></tr>
            </table>
            
    	</div>
    </div>
    
    <div class="datarow">
    	<div class="titlerow">
        	
            <a title="toggle" href="" onclick="toggleAll('2010'); return false;" onkeypress="return false;"><span class="title">Project Information:&nbsp;<span>Grant/Contract No. </span> NNX09AH41G&nbsp;<img id="imggrantdetail2010" width="15" src="_images/_minus.gif" alt="" /></a></span>
    		<div id="grantdetail2010">
            	<table width="100%">
                	<tr>
                    <td width="18%">
                    	<span class="bold">Responsible Center: </span>NASA ARC&nbsp;<br/>
                        <span class="bold">Grant Monitor: </span>Sun, Sidney C&nbsp;<br/>
                        <span class="bold">Center Contact: </span>650-604-4835&nbsp;<br/><a href="mailto:Sidney.C.Sun@nasa.gov">Sidney.C.Sun@nasa.gov</a>&nbsp;<br/>
						<span class="bold">Unique ID: </span>7384&nbsp;
                    </td>
                    <td>
						<span class="bold">Solicitation / Funding Source: </span> 2008 Space Biology--Microbial, Plant, Cell Biology: NNH08ZTT003N&nbsp;<br/>
                        <span class="bold">Grant/Contract No.: </span> NNX09AH41G&nbsp;<br/>
                        <span class="bold">Project Type: </span> Ground&nbsp;<br/>
                        <span class="bold">Flight Program: </span> &nbsp;
                        </td>
                    <td>
                    	<span class="bold">No. of Post Docs: </span>1&nbsp;<br/>
                        <span class="bold">No. of PhD Candidates: </span>1&nbsp;<br/>
                        <span class="bold">No. of Master's Candidates: </span>&nbsp;<br/>
                        <span class="bold">No. of Bachelor's Candidates: </span>1&nbsp;
                    </td>
                    <td>
                    	<span class="bold">No. of PhD Degrees: </span>&nbsp;<br/>
                        <span class="bold">No. of Master's Degrees: </span>&nbsp;<br/>
                        <span class="bold">No. of Bachelor's Degrees: </span>&nbsp;
                    </td>
                	</tr>
                	<tr height="5"><td></td></tr>
                
                	<tr>
                	<td class="title">Space Biology Element: </td>
                    <td colspan="3">
						(1) Cell & Molecular Biology <br />(2) Microbiology <br />(3) Animal Biology: Vertebrate 
                    </td>
                    </tr>
                    <tr height="1" bgcolor="#baa"><td colspan="4"></td></tr>
                    <tr>
                	<td class="title">Space Biology Cross-Element Discipline: </td>
                    <td colspan="3">
                    	(1) Reproductive Biology 
                    </td>
                    </tr>
                    <tr height="1" bgcolor="#baa"><td colspan="4"></td></tr>
                    <tr>
                	<td class="title">Space Biology Special Category: </td>
                    <td colspan="3">
                    	(1) Translational (Countermeasure) Potential                     </td>
                    </tr> 
                	<tr height="2" bgcolor="#baa"><td colspan="4"></td></tr>
                    <tr>
                	<td class="title">Task Description: </td>
                    <td colspan="3"> Biofilms are resistant surface-attached communities of bacteria found on Earth and in spacecraft. Biofilms can cause serious diseases, but are also superior at beneficial processes, e.g., nutrient recycling and life support systems required for space exploration. We have developed Earth-based bioreactors for cultivating bacterial biofilms under low-shear modeled microgravity (LSMMG), which resembles microgravity of space. LSMMG-grown biofilms exhibit greater resistance than their Earth gravity (NG)-grown counterparts, making them even a greater threat to astronaut health during space travel/residence. This increased resistance encompasses both ‘disinfectants’ (e.g., ethanol) as well as antibiotics, and resembles the general stress response (GSR) of bacteria on Earth. The resemblance between LSMMG-conferred resistance and the GSR includes the facts that both make bacteria resistant to killing agents that destroy them by different mechanisms, and that both appear to depend on common regulatory elements, e.g., sigmaS and Hfq proteins. But while GSR and the stress proteins which underpin GSR are now understood in considerable detail, little is known of the mechanistic basis of LSMMG biofilms. The focus of this investigation is to explore the identity of the regulators of GSR (GSRR) and of stress proteins that determine LSMMG biofilm resistance. Wild type and selected GSRR mutant [sigmaS, Hfq, sigmaS-Hfq, or another GSRR double mutant (e.g., sigmaS-sigma54)] biofilms will be grown under LSMMG and NG conditions, using our Earth-based bioreactors. The development and resistance of the biofilms grown under the two conditions will be compared. For determining resistance, mechanistically different stressors (i.e., those with different modes of killing bacteria) including disinfectants (ethanol, NaCl) and antibiotics (ciprofloxacin, ampicillin) will be used. Survival will be determined using live dead stains and confocal microscopy in combination with appropriate software; viable cell count will also be used in selected cases. The relationship between the expression of GSR regulators and LSMMG-conferred resistance will be determined in bulk biofilms by Western analysis. It will also be determined within individual biofilm regions, using multiple approaches which will rely on GFP-expressing regulator fusions, live-dead stains, and confocal microscopy. The LSMMG-induced proteome of the wild type and the mutant showing greatest impairment in LSMMG resistance will be compared to detect the proteins involved in this resistance; their identification will be made more precise by excluding from among the implicated proteins those also induced in the mutant (impaired in LSMMG resistance). IPTG-inducible scFv antibodies will be used to functionally inactivate selected proteins in the wild type at appropriate times during biofilm development to confirm their role in LSMMG resistance. Fundamental information of the mechanistic basis of LSMMG and Earth biofilms will result, which can suggest ways for their better control. This research thus fully addresses the solicitation: it deals with the role of stress proteins in LSMMG biofilm resistance; it utilizes Earth-based approaches that can be tested in space; and it deals with the question of how life responds to gravity in space and on Earth.  <P></td>
                    </tr> 
                	<tr height="2" bgcolor="#baa"><td colspan="4"></td></tr>    
                    <tr>
                	<td class="title">Research Impact/Earth Benefits: </td>
                    <td colspan="3">      Urinary tract infections are primarily caused by bacteria known as uropathogenic Escherichia coli, or “UPEC” for short. These infections cause the disease cystitis, which leads to approximately 8 million doctor visits by women each year in the United States alone. UPEC attach to the epithelium of the bladder and upper urinary tract and form what are known as biofilms. Biofilms are communities of bacteria that attach to surfaces and can evade host immune responses, as well as resist antibiotics. This form of bacterial growth often leads to chronic infections that are difficult to cure. Biofilms also attach to surfaces onboard spacecraft, presenting a potential pathogen reservoir that threatens astronaut health during spaceflight. Our previous work indicated that biofilms exhibit greater resistance to disinfectants and antibiotics in conditions that mimic the microgravity environment on Earth. These conditions are generated in rotating vessel bioreactors, which prevent cell sedimentation and provide a quiescent or “low-shear” fluid environment similar to that of microgravity during spaceflight. We developed a system using glass beads for biofilm surface attachment permitting study of the “low-shear modeled microgravity environment” (LSMMG) on Earth. We found that this environment further increased UPEC resistance to antimicrobial treatment, making the bacteria superresistant. The objective of our current study is to better understand the stress-response mechanisms by which UPEC cope with the stress generated by antimicrobials. This includes UPEC in slow growing states such as stationary-phase and biofilms. LSMMG bioreactors provide a unique environment for studying resistance mechanisms of an important bacterial pathogen. We are also studying these stress response mechanisms under normal gravity conditions to better understand the underlying biology/biochemistry and to provide an appropriate framework for comparing the LSMMG results. Insights from our work will provide valuable knowledge for developing new forms of treatment aimed against UPEC, as well as other bacterial pathogens, which will benefit humans on Earth as well as onboard spacecraft.  <P></td>
                    </tr>
                </table>
            </div>
    	</div>
    </div>
    
    <div class="datarow">
    	<div class="titlerow">
        	
            <a title="toggle" href="" onclick="toggleAll('2010'); return false;" onkeypress="return false;"><span class="title">Task Progress & Bibliography Information FY2010&nbsp;<img id="imgtaskdetail2010" width="15" src="_images/_minus.gif" alt="" /></a></span>
    		<div id="taskdetail2010">
            	<table width="100%">
                	<tr>
                    <td width="18%" class="title">
                    	Task Progress:
                    </td>
                    <td>
						     Progress during the first year of support has focused on studying regulators involved in the response of Escherichia coli to antibiotic stress. Traditionally, antibiotics have been developed and characterized with respect to their activity against exponentially-growing planktonic cells. Yet, non-multiplying or slowly-multiplying bacteria contribute to inherent or phenotypic resistance. Examples include biofilms and stationary phase cells which are both likely to be important in chronic infections. These cells are killed with a much lower efficiency presumably due to their slower growth and metabolic rates.     <P>
     In our initial studies supported by this grant, we have characterized the effect of several antibiotics, representing different antibiotic classes and mechanisms of action, against stationary phase E. coli. We compared the wild-type strains with their rpoS-deficient mutants, which lack the stationary phase sigma factor sigmaS. It is important to first establish the role of sigmaS in antibiotic tolerance for stationary-phase cultures under NG conditions in order to properly interpret, and put into context, the findings for biofilm growth and LSMMG conditions. In our more recent work, we have begun repeating our key findings under LSMMG conditions.     <P>
     Stationary phase cultures of a clinically isolated strain of uropathogenic E. coli (UPEC) were exposed to antibiotics in conditions of low nutrients (M9 without glucose) and low oxygen (standing culture). Superinhibitory antibiotic concentrations were used due to the inherent tolerance of stationary phase bacteria. At daily intervals, a sample from each culture was taken to determine the remaining number of viable bacteria. The rpoS mutant consistently showed an approximately 10,000-fold reduction in the number of viable cells compared to the wild-type for bactericidal antibiotics. The difference was observed regardless of the primary mode of action: for example, ampicillin is a beta-lactam (interferes with cell wall synthesis), ciprofloxacin is a quinolone (inhibits DNA synthesis) and gentamicin is an aminoglycoside (interferes with protein synthesis).     <P>
     As stated in NRA NNH08ZTT003N, the ground based studies should provide preliminary evidence for future spaceflight experiments. Due to possible safety issues regarding the use of pathogenic bacteria onboard human spaceflight missions, we have also performed parallel studies with a nonpathogenic E. coli K12 for comparison. While the pathogenic strain is potentially more valuable for deciphering the causes of antimicrobial resistance during disease, the K12 strain is closely related and a model organism applicable to any spaceflight experiment.     <P>
     When we examined sigmaS-dependent tolerance of stationary-phase E. coli K12 to bactericidal antibiotics (as above), we observed the same phenotype as with UPEC. By using E. coli K12, we were able to take advantage of an existing gene-deletion library to rapidly assess the effect of antibiotics on mutation of sigmaS-related genes. Although sigmaS is known to regulate expression of over 100 genes, two likely candidates involved in antibiotic tolerance are sodA (superoxide dismutase) and katE (catalase). SodA converts the highly damaging superoxide anion (O2-) to oxygen (O2) and hydrogen peroxide (H2O2). KatE catalyzes H2O2 to H2O and O2. Reactive oxygen species (ROS) have previously been implicated in secondary killing mechanisms of bactericidal antibiotics, and the enzymes SodA and KatE have been used to suppress killing by different antibiotics that inhibit transcription or translation. Despite these implications, we found killing of sodA- and katE-mutants to appear more like wild-type than the rpoS mutant when exposed to gentamicin. On the other hand, hfq-deficient E. coli showed rpoS-like killing kinetics upon exposure to gentamicin. This we anticipated because very little sigmaS is synthesized by an hfq mutant.    <P>
     Because the genes (sodA and katE) most likely to be involved in ROS tolerance showed no further susceptibility to gentamicin compared to wild-type E. coli, we decided to examine proteome-wide changes in protein expression. We chose to study proteomics over genomics because we and others have shown that sigmaS levels are subject to extensive post-translational regulation (e.g., by the protease ClpXP). We used two dimensional difference in gel electrophoresis (2D-DiGE) to characterize the wild-type and rpoS-deficient proteomes following gentamicin exposure. 2D-DiGE is similar to traditional 2D gel electrophoresis, but with the additional step of labeling protein samples with fluorescent dyes prior to separation by 2D gel electrophoresis. This technique increases sensitivity and permits comparison of protein expression levels within and between gels. In response to gentamicin treatment, 25 proteins were differentially expressed in wild-type (18 up- and 7 down-regulated) and 27 in the rpoS mutant (9 up- and 18 down-regulated). The 8 proteins with the greatest change in expression were identified by mass spectrometry. These proteins indicate a shift in metabolism to resist reactive oxygen species. For instance, GcvH is a glycine cleavage protein; UspG acts as a universal stress protein; PspA regulates phage stress response; YhbW is a putative oxidoreductase; FtnA sequesters iron, reducing its availability to generate hydroxyl radicals via Fenton reactions; and, TalA and TktB are key proteins in the pentose phosphate pathway. A primary role of the latter pathway is generation of reducing equivalents in the form of NADPH, which is required to quench oxidative stress. To further assess the role of these 8 proteins, we tested the effect of gentamicin exposure, as above, on each gene-deletion mutant. Four mutants (tktB, yhbW, ftnA and gcvH) showed wild-type killing kinetics. Three mutants showed greater gentamicin sensitivity compared to wild-type: talA appeared similar to rpoS, while pspA and uspG showed intermediate sensitivity. Note that the lpd mutant had a severe growth defect and was therefore excluded.    <P>
     We have begun to characterize the role of sigmaS on the resistance of E. coli to antibiotics also under conditions of low-shear modeled microgravity. This work was delayed for a few months due to issues with the condition of our HARVs, which are now over 10 years old and had remained idle since 2007, when my previous NASA funding was abruptly cut short. The major issue with the condition of our HARVs was a high rate of water loss due to evaporation. Evaporative water loss causes air bubble formation, and air bubbles disrupt the low-shear fluid environment which is essential for mimicking microgravity conditions. Due to our relatively long term antibiotic treatment experiments (up to 6 days), even small amounts of evaporation caused extensive and problematic bubble formation. We attempted to mitigate evaporation by resealing all valves and seams and providing a 100% humidity environment, but neither remedy succeeded. Therefore, we shipped the HARVs to the manufacturer (Synthecon) for extensive refurbishment including replacement of all valves and silicon membranes and polishing of vessel face plates. Following refurbishment, pilot studies were conducted to monitor evaporation, which had ceased.    <P>
     With newly refurbished HARVs, growth curves were performed with the E. coli K12 strains (wild-type and rpoS-mutant) as we had not previously experimented with these strains in HARVs. The HARVs were rotated at 25 rpm and incubated at 37C for 46 h. We found that cultures reached higher final cell densities under LSMMG compared to NG conditions in agreement with previous reports. This was true for both wild-type and rpoS-mutant strains. Interestingly, the rpoS mutant did not grow as well as wild-type in the HARVs. This was true under both NG and LSMMG conditions. This finding is in contrast to our findings from traditional shake flask cultures in which both strains show almost identical growth.    <P>
     Gentamicin treatment experiments in HARVs were recently initiated. HARVs were loaded with stationary phase wild-type E. coli with or without 16 microg/ml gentamicin. Two HARVS were rotated about a horizontal axis (LSMMG) and two about a vertical axis (NG). The HARVs were rotated at 25 rpm in a 37C incubator and samples were withdrawn for viability determination by plate counting on days 0, 1 and 2. The results showed moderate reduction in viability due to gentamicin treatment for both gravity conditions. Negligible differences between NG and LSMMG conditions were observed. The experiment needs to be repeated before any conclusions can be drawn. Also, the experiment needs to be performed with the rpoS-deficient mutant for comparison to wild-type.     <P>
     The planned experiments for the remaining 2 months of the first year include repeating the UPEC gentamicin exposure experiment in HARVs with the wild-type strain. The second experiment will be the same except the rpoS mutant will be used instead. Because we have only 4 HARVs, the two strains must be tested separately. Following this study, other mechanistically-different antibiotics will be tested against UPEC (wild-type and rpoS mutant) in HARVs (NG and LSMMG), namely ampicillin and ciprofloxacin. As mentioned above, the proteins identified by 2D-DiGE implicate ROS; as such, we are currently determining the best method to directly quantify ROS in stationary phase UPEC upon gentamicin exposure in order to determine their role in cell killing. Once the best methods have been applied to cultures under traditional laboratory conditions, we will use them also for studying cultures grown in HARVs. In the second year of the proposal, the key findings from stationary phase cultures will be investigated with bead-biofilms grown in HARVs.   <P>
                    </td>
                	</tr>
                
                	<tr height="1" bgcolor="#baa"><td colspan="2"></td></tr>
               
                    
					<tr><td width="18%" class="title">
                    	Bibliography:
                    </td>
                    <td>
						Description: (Last Updated: 03/27/2020)&nbsp; 
						<br /><br /><a style="padding-top:5px;font-weight:bold; background-color:#fee" href="#" title="Show Complete List" onclick="document.forms['data'].tid.value='7812'; submitFormBib('data','stackbib_all.cfm','bibcontent2010'); return false;" >Show Cumulative Bibliography</a><br />&nbsp; </td>
                	</tr>
                  
					<tr><td colspan="2">
						<div id="bibcontent2010">
						<table width="100%">
						
                  	
                  <tr><td>&nbsp;</td><td>None in FY 2010</td></tr>
                  
						
				</td></tr></table>
				</div>
						
				</td></tr>
                </table>
            </div>
    	</div>
    </div>
</div>
    	<div class="smallr"><a href="index.cfm?action=sort&sort=asc&col=pi" title="List Search Results">Back to Search Results</a>&nbsp;&nbsp;&nbsp;&nbsp;</div>

	 </div><div  id="cf_layoutareaReport2009" class="x-hidden" style="overflow:auto;;width:100%;" align="left">
		
		
<div id="bgdata">
	<div class="datarow">
    	<div class="titlerow">
        	<span class="title">Project Title:&nbsp; Molecular basis of LSMMG biofilm resistance <a href="" title="Expand/Reduce" onclick="toggleAll('2009');return false;"><img id="imgall2009" width="20" src="_images/_minus.gif" alt="" /><span id="objAll2009">Reduce</span></a></span>
    		<div id="pidetail">
            	<table width="85%"><tr>
                	
                    <td><span class="bold">Fiscal Year: </span>FY 2009&nbsp;<br/>
                    	<span class="bold">Division: </span> Space Biology&nbsp;<br/>
                        <span class="bold">Research Discipline/Element: </span><br/>
                        <b>Space Biology</b>: Cell & Molecular Biology &nbsp; | Microbiology &nbsp; | Animal Biology: Vertebrate &nbsp;<br/> 
                        </td>
                    <td><span class="bold">Start Date: </span>05/01/2009&nbsp;&nbsp;<br/>
                    	<span class="bold">End Date: </span>04/30/2011&nbsp;&nbsp;<br/>
                        <span class="bold">Task Last Updated: </span>04/22/2009&nbsp;</td>
                    <td><a title="download in pdf" href="tbpdf.cfm?id=7384" download>Download Task Book report in PDF <img src="_images/pdf.gif" alt="pdf" border="0" width="25" /></a><br/>
						</td>
                </tr></table>
            </div>
    	</div>
    </div>
    
    <div class="datarow">
    	<div class="titlerow">
        	
            <a title="toggle" href="" onclick="toggleAll('2009'); return false;" onkeypress="return false;"><span class="title">Principal Investigator/Affiliation:&nbsp;&nbsp; Matin,&nbsp;A.C.&nbsp;&nbsp;Ph.D.&nbsp;/ Stanford University&nbsp;<img id="imgpidetail2009" width="15" src="_images/_minus.gif" alt="" /></a></span>
			<div id="pidetail2009">
            	<table width="85%"><tr>
                	<td align="right" class="title" width="5%">Address:&nbsp;</td>
                    <td> 299 Campus Drive, West&nbsp;<br /> Mail Code 5124&nbsp;<br/>Stanford&nbsp;, CA&nbsp;94305-5101&nbsp;</td>
                    <td>
						<span class="bold">Email: </span><a href="mailto:A.Matin@stanford.edu">A.Matin@stanford.edu</a>&nbsp;<br/>
                        <span class="bold">Phone: </span>(650) 725-4745 &nbsp;&nbsp;<br/>
                        <span class="bold">Congressional District: </span>14&nbsp;<br/>
                        <span class="bold">Web: </span> &nbsp;<br/>
                        
					</td>
                    <td>
                    	<span class="bold">Organization Type: </span> UNIVERSITY&nbsp;<br/>
                        <span class="bold">Organization Name: </span> Stanford University&nbsp;<br/>
                        <span class="bold">Joint Agency: </span> &nbsp;<br/>
                        <span class="bold">Comments: </span> &nbsp;
                    </td>
                </tr>
                </table>
            </div>
    	
    	</div>
    </div>
    
    <div class="datarow">
    	<div class="titlerow">
        	
            <a title="toggle" href="" onclick="toggleAll('2009'); return false;" onkeypress="return false;"><span class="title">Project Information:&nbsp;<span>Grant/Contract No. </span> NNX09AH41G&nbsp;<img id="imggrantdetail2009" width="15" src="_images/_minus.gif" alt="" /></a></span>
    		<div id="grantdetail2009">
            	<table width="100%">
                	<tr>
                    <td width="18%">
                    	<span class="bold">Responsible Center: </span>NASA ARC&nbsp;<br/>
                        <span class="bold">Grant Monitor: </span>Sun, Sidney C&nbsp;<br/>
                        <span class="bold">Center Contact: </span>650-604-4835&nbsp;<br/><a href="mailto:Sidney.C.Sun@nasa.gov">Sidney.C.Sun@nasa.gov</a>&nbsp;<br/>
						<span class="bold">Unique ID: </span>7384&nbsp;
                    </td>
                    <td>
						<span class="bold">Solicitation / Funding Source: </span> 2008 Space Biology--Microbial, Plant, Cell Biology: NNH08ZTT003N&nbsp;<br/>
                        <span class="bold">Grant/Contract No.: </span> NNX09AH41G&nbsp;<br/>
                        <span class="bold">Project Type: </span> Ground&nbsp;<br/>
                        <span class="bold">Flight Program: </span> &nbsp;
                        </td>
                    <td>
                    	<span class="bold">No. of Post Docs: </span>&nbsp;<br/>
                        <span class="bold">No. of PhD Candidates: </span>&nbsp;<br/>
                        <span class="bold">No. of Master's Candidates: </span>&nbsp;<br/>
                        <span class="bold">No. of Bachelor's Candidates: </span>&nbsp;
                    </td>
                    <td>
                    	<span class="bold">No. of PhD Degrees: </span>&nbsp;<br/>
                        <span class="bold">No. of Master's Degrees: </span>&nbsp;<br/>
                        <span class="bold">No. of Bachelor's Degrees: </span>&nbsp;
                    </td>
                	</tr>
                	<tr height="5"><td></td></tr>
                
                	<tr>
                	<td class="title">Space Biology Element: </td>
                    <td colspan="3">
						(1) Cell & Molecular Biology <br />(2) Microbiology <br />(3) Animal Biology: Vertebrate 
                    </td>
                    </tr>
                    <tr height="1" bgcolor="#baa"><td colspan="4"></td></tr>
                    <tr>
                	<td class="title">Space Biology Cross-Element Discipline: </td>
                    <td colspan="3">
                    	(1) Reproductive Biology 
                    </td>
                    </tr>
                    <tr height="1" bgcolor="#baa"><td colspan="4"></td></tr>
                    <tr>
                	<td class="title">Space Biology Special Category: </td>
                    <td colspan="3">
                    	(1) Translational (Countermeasure) Potential                     </td>
                    </tr> 
                	<tr height="2" bgcolor="#baa"><td colspan="4"></td></tr>
                    <tr>
                	<td class="title">Task Description: </td>
                    <td colspan="3"> Biofilms are resistant surface-attached communities of bacteria found on Earth and in spacecraft. Biofilms can cause serious diseases, but are also superior at beneficial processes, e.g., nutrient recycling and life support systems required for space exploration. We have developed Earth-based bioreactors for cultivating bacterial biofilms under low-shear modeled microgravity (LSMMG), which resembles microgravity of space. LSMMG-grown biofilms exhibit greater resistance than their Earth gravity (NG)-grown counterparts, making them even a greater threat to astronaut health during space travel/residence. This increased resistance encompasses both ‘disinfectants’ (e.g., ethanol) as well as antibiotics, and resembles the general stress response (GSR) of bacteria on Earth. The resemblance between LSMMG-conferred resistance and the GSR includes the facts that both make bacteria resistant to killing agents that destroy them by different mechanisms, and that both appear to depend on common regulatory elements, e.g., sigmaS and Hfq proteins. But while GSR and the stress proteins which underpin GSR are now understood in considerable detail, little is known of the mechanistic basis of LSMMG biofilms. The focus of this investigation is to explore the identity of the regulators of GSR (GSRR) and of stress proteins that determine LSMMG biofilm resistance. Wild type and selected GSRR mutant [sigmaS, Hfq, sigmaS-Hfq, or another GSRR double mutant (e.g., sigmaS-sigma54)] biofilms will be grown under LSMMG and NG conditions, using our Earth-based bioreactors. The development and resistance of the biofilms grown under the two conditions will be compared. For determining resistance, mechanistically different stressors (i.e., those with different modes of killing bacteria) including disinfectants (ethanol, NaCl) and antibiotics (ciprofloxacin, ampicillin) will be used. Survival will be determined using live dead stains and confocal microscopy in combination with appropriate software; viable cell count will also be used in selected cases. The relationship between the expression of GSR regulators and LSMMG-conferred resistance will be determined in bulk biofilms by Western analysis. It will also be determined within individual biofilm regions, using multiple approaches which will rely on GFP-expressing regulator fusions, live-dead stains, and confocal microscopy. The LSMMG-induced proteome of the wild type and the mutant showing greatest impairment in LSMMG resistance will be compared to detect the proteins involved in this resistance; their identification will be made more precise by excluding from among the implicated proteins those also induced in the mutant (impaired in LSMMG resistance). IPTG-inducible scFv antibodies will be used to functionally inactivate selected proteins in the wild type at appropriate times during biofilm development to confirm their role in LSMMG resistance. Fundamental information of the mechanistic basis of LSMMG and Earth biofilms will result, which can suggest ways for their better control. This research thus fully addresses the solicitation: it deals with the role of stress proteins in LSMMG biofilm resistance; it utilizes Earth-based approaches that can be tested in space; and it deals with the question of how life responds to gravity in space and on Earth.  <P></td>
                    </tr> 
                	<tr height="2" bgcolor="#baa"><td colspan="4"></td></tr>    
                    <tr>
                	<td class="title">Research Impact/Earth Benefits: </td>
                    <td colspan="3">  <P></td>
                    </tr>
                </table>
            </div>
    	</div>
    </div>
    
    <div class="datarow">
    	<div class="titlerow">
        	
            <a title="toggle" href="" onclick="toggleAll('2009'); return false;" onkeypress="return false;"><span class="title">Task Progress & Bibliography Information FY2009&nbsp;<img id="imgtaskdetail2009" width="15" src="_images/_minus.gif" alt="" /></a></span>
    		<div id="taskdetail2009">
            	<table width="100%">
                	<tr>
                    <td width="18%" class="title">
                    	Task Progress:
                    </td>
                    <td>
						New project for FY2009. <P>
                    </td>
                	</tr>
                
                	<tr height="1" bgcolor="#baa"><td colspan="2"></td></tr>
               
                    
					<tr><td width="18%" class="title">
                    	Bibliography:
                    </td>
                    <td>
						Description: (Last Updated: 03/27/2020)&nbsp; 
						<br /><br /><a style="padding-top:5px;font-weight:bold; background-color:#fee" href="#" title="Show Complete List" onclick="document.forms['data'].tid.value='7384'; submitFormBib('data','stackbib_all.cfm','bibcontent2009'); return false;" >Show Cumulative Bibliography</a><br />&nbsp; </td>
                	</tr>
                  
					<tr><td colspan="2">
						<div id="bibcontent2009">
						<table width="100%">
						
                  	
                  <tr><td>&nbsp;</td><td>None in FY 2009</td></tr>
                  
						
				</td></tr></table>
				</div>
						
				</td></tr>
                </table>
            </div>
    	</div>
    </div>
</div>
    	<div class="smallr"><a href="index.cfm?action=sort&sort=asc&col=pi" title="List Search Results">Back to Search Results</a>&nbsp;&nbsp;&nbsp;&nbsp;</div>

	 </div>
 </div>
</form>

<div  id="cf_window1245006494640312" class="x-hidden">
	
	
	<div  id="popbibinfo-body" class="" style="border-width:2px; background-color:fff;height:100%;">
		
		
	 </div>
 </div><div  id="cf_window1245006494640315" class="x-hidden">
	
	
	<div  id="poptaskinfo-body" class="" style="border-width:12px; background-color:#f0f0f0; margin-top: -10px; margin-left: -10px;;height:100%;">
		
		
	 </div>
 </div>

<script language="javascript">

	document.getElementById("pidetail2011").style.display = "block";
	document.getElementById("grantdetail2011").style.display = "block";
	document.getElementById("taskdetail2011").style.display = "block";

	document.getElementById("pidetail2010").style.display = "block";
	document.getElementById("grantdetail2010").style.display = "block";
	document.getElementById("taskdetail2010").style.display = "block";

	document.getElementById("pidetail2009").style.display = "block";
	document.getElementById("grantdetail2009").style.display = "block";
	document.getElementById("taskdetail2009").style.display = "block";

	var toggleobj = function(obj) {
		theObj = document.getElementById(obj);
		theImg = document.getElementById('img'+obj);
		if (theObj.style.display == "block")
			{ theObj.style.display = "none"; theImg.src = "_images/_plus.gif"; }
		else
			{ theObj.style.display = "block"; theImg.src = "_images/_minus.gif"; }		
	}
	
	var toggleAll = function(obj) {
		theText = document.getElementById('objAll'+obj);
		theImg = document.getElementById('imgall'+obj);
		theObj1 = document.getElementById('pidetail'+obj);
		theImg1 = document.getElementById('imgpidetail'+obj);
		theObj2 = document.getElementById('grantdetail'+obj);
		theImg2 = document.getElementById('imggrantdetail'+obj);
		theObj3 = document.getElementById('taskdetail'+obj);
		theImg3 = document.getElementById('imgtaskdetail'+obj);
		if (theText.innerHTML == "Reduce") {
		  theObj1.style.display = "none"; theImg1.src = "_images/_plus.gif";
		  theObj2.style.display = "none"; theImg2.src = "_images/_plus.gif";
		  theObj3.style.display = "none"; theImg3.src = "_images/_plus.gif";
		  theText.innerHTML = "Expand"; theImg.src = "_images/_plus.gif";
		} else {
		  theObj1.style.display = "block"; theImg1.src = "_images/_minus.gif";
		  theObj2.style.display = "block"; theImg2.src = "_images/_minus.gif";
		  theObj3.style.display = "block"; theImg3.src = "_images/_minus.gif";
		  theText.innerHTML = "Reduce"; theImg.src = "_images/_minus.gif";
		}
	}

	var bibData = "";
	var bibTarget = null;
	
	function submitFormBib(formname,pagename,target) {
        ColdFusion.Ajax.submitForm(formname, pagename, callbackBib, errorHandlerBib);
		bibTarget = document.getElementById(target);
		    }
	
    function callbackBib(text)
    {
		var value_obj = document.getElementById('hrp_risks');
        ColdFusion.Window.hide('popbibinfo');
		bibData = text;
		if (bibTarget) {
			if (text.length > 0)
				bibTarget.innerHTML=text;
			}
    }
    
    function errorHandlerBib(code, msg)
    {
        var value_obj = document.getElementById('hrp_risks');
        ColdFusion.Window.hide('popbibinfo');
		if (value_obj) {
			value_obj.innerHTML='Error Happened. Please try again.' + msg;
			}
    }

</script>
	<!-- end main area -->
			</div>&nbsp;&nbsp;
	</div>	
    		
	
	<div id="footer"> 
	<img src="_images/f_nress.gif" alt="NRESS Logo" width="55" height="65" border="0" align="left" />
	Developed and operated by: NASA Research and Education Support Services<br />
	NASA Application Owner: Diana Ly Biological & Physical Sciences Division; Carol Mullenax Human Research Program/Translational Research Institute for Space Health.<br />
	<a href="privacy_statement.cfm" title="Disclaimer">Task Book Policies and Notices</a> || 
	<a href="termsconditions.cfm" title="Terms">Terms and Conditions</a> ||
    <a target="_blank" href="https://www.nasa.gov/about/highlights/HP_Privacy.html" title="NASA Notice">NASA Web Privacy and Important Notices</a> || 
    <a target="_blank" href="https://www.nasa.gov/accessibility/" title="Accessibility">NASA Accessibility</a> || 
    <a href="contactus.cfm" title="Contact Us">Contact Us</a><br><br>
</div>
<br>


	


</div>
 
</body>
</html>

