<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7064724/"/><title>Root Skewing-Associated Genes Impact the Spaceflight Response of Arabidopsis thaliana - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Front Plant Sci</button></div>. 2020 Mar 4;11:239. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2020.00239" rel="noopener noreferrer" target="_blank">10.3389/fpls.2020.00239</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Root Skewing-Associated Genes Impact the Spaceflight Response of <em>Arabidopsis thaliana</em>
</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Califar%20B%22%5BAuthor%5D"><span class="name western">Brandon Califar</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Brandon Califar</span></h3>
<div class="p">
<sup>1</sup>Horticultural Sciences, University of Florida, Gainesville, FL, United States</div>
<div class="p">
<sup>2</sup>The Genetics Institute, University of Florida, Gainesville, FL, United States</div>
<div class="p">
<sup>3</sup>Program in Genetics and Genomics, University of Florida, Gainesville, FL, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Califar%20B%22%5BAuthor%5D"><span class="name western">Brandon Califar</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sng%20NJ%22%5BAuthor%5D"><span class="name western">Natasha J Sng</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Natasha J Sng</span></h3>
<div class="p">
<sup>1</sup>Horticultural Sciences, University of Florida, Gainesville, FL, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sng%20NJ%22%5BAuthor%5D"><span class="name western">Natasha J Sng</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zupanska%20A%22%5BAuthor%5D"><span class="name western">Agata Zupanska</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Agata Zupanska</span></h3>
<div class="p">
<sup>1</sup>Horticultural Sciences, University of Florida, Gainesville, FL, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zupanska%20A%22%5BAuthor%5D"><span class="name western">Agata Zupanska</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paul%20AL%22%5BAuthor%5D"><span class="name western">Anna-Lisa Paul</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Anna-Lisa Paul</span></h3>
<div class="p">
<sup>1</sup>Horticultural Sciences, University of Florida, Gainesville, FL, United States</div>
<div class="p">
<sup>2</sup>The Genetics Institute, University of Florida, Gainesville, FL, United States</div>
<div class="p">
<sup>3</sup>Program in Genetics and Genomics, University of Florida, Gainesville, FL, United States</div>
<div class="p">
<sup>4</sup>Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, FL, United States</div>
<div class="p">
<sup>5</sup>Interdisciplinary Center for Biotechnology and Research, University of Florida, Gainesville, FL, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paul%20AL%22%5BAuthor%5D"><span class="name western">Anna-Lisa Paul</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>*</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferl%20RJ%22%5BAuthor%5D"><span class="name western">Robert J Ferl</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Robert J Ferl</span></h3>
<div class="p">
<sup>1</sup>Horticultural Sciences, University of Florida, Gainesville, FL, United States</div>
<div class="p">
<sup>2</sup>The Genetics Institute, University of Florida, Gainesville, FL, United States</div>
<div class="p">
<sup>3</sup>Program in Genetics and Genomics, University of Florida, Gainesville, FL, United States</div>
<div class="p">
<sup>4</sup>Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, FL, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferl%20RJ%22%5BAuthor%5D"><span class="name western">Robert J Ferl</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>Horticultural Sciences, University of Florida, Gainesville, FL, United States</div>
<div id="aff2">
<sup>2</sup>The Genetics Institute, University of Florida, Gainesville, FL, United States</div>
<div id="aff3">
<sup>3</sup>Program in Genetics and Genomics, University of Florida, Gainesville, FL, United States</div>
<div id="aff4">
<sup>4</sup>Program in Plant Molecular and Cellular Biology, University of Florida, Gainesville, FL, United States</div>
<div id="aff5">
<sup>5</sup>Interdisciplinary Center for Biotechnology and Research, University of Florida, Gainesville, FL, United States</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Edited by: Valérie Legué, Université Clermont Auvergne, France</p></div>
<div class="fn" id="fn2"><p>Reviewed by: Stephanie Swarbreck, University of Cambridge, United Kingdom; Francisco Javier Medina, Center for Biological Research, Superior Council of Scientific Investigations, Spain</p></div>
<div class="fn" id="c001">
<sup>✉</sup><p class="display-inline">*Correspondence: Anna-Lisa Paul, <span>alp@ufl.edu</span></p>
</div>
<div class="fn" id="c002">
<sup>*</sup><p class="display-inline">Robert J. Ferl, <span>robferl@ufl.edu</span></p>
</div>
<div class="fn" id="fn004"><p>This article was submitted to Plant Abiotic Stress, a section of the journal Frontiers in Plant Science</p></div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2019 Sep 7; Accepted 2020 Feb 17; Collection date 2020.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2020 Califar, Sng, Zupanska, Paul and Ferl.</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC7064724  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32194611/">32194611</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The observation that plant roots skew in microgravity recently refuted the long-held conviction that skewing was a gravity-dependent phenomenon. Further, spaceflight root skewing suggests that specific root morphologies and cell wall remodeling systems may be important aspects of spaceflight physiological adaptation. However, connections between skewing, cell wall modification and spaceflight physiology are currently based on inferences rather than direct tests. Therefore, the Advanced Plant Experiments-03-2 (APEX-03-2) spaceflight study was designed to elucidate the contribution of two skewing- and cell wall-associated genes in Arabidopsis to root behavior and gene expression patterns in spaceflight, to assess whether interruptions of different skewing pathways affect the overall spaceflight-associated process. SPIRAL1 is a skewing-related protein implicated in directional cell expansion, and functions by regulating cortical microtubule dynamics. SKU5 is skewing-related glycosylphosphatidylinositol-anchored protein of the plasma membrane and cell wall implicated in stress response signaling. These two genes function in different cellular pathways that affect skewing on the Earth, and enable a test of the relevance of skewing pathways to spaceflight physiological adaptation. In this study, both <em>sku5</em> and <em>spr1</em> mutants showed different skewing behavior and markedly different patterns of gene expression in the spaceflight environment. The <em>spr1</em> mutant showed fewer differentially expressed genes than its Col-0 wild-type, whereas <em>sku5</em> showed considerably more than its WS wild-type. Developmental age played a substantial role in spaceflight acclimation in all genotypes, but particularly in <em>sku5</em> plants, where spaceflight 4d seedlings had almost 10-times as many highly differentially expressed genes as the 8d seedlings. These differences demonstrated that the two skewing pathways represented by <em>SKU5</em> and <em>SPR1</em> have unique and opposite contributions to physiological adaptation to spaceflight. The <em>spr1</em> response is less intense than wild type, suggesting that the loss of SPR1 positively impacts spaceflight adaptation. Conversely, the intensity of the <em>sku5</em> responses suggests that the loss of SKU5 initiates a much more complex, deeper and more stress related response to spaceflight. This suggests that proper SKU5 function is important to spaceflight adaptation.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> spaceflight, root, skewing, transcriptomics, abiotic stress, acclimation, physiological adaptation</p></section></section><section id="S1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Spaceflight studies offer unique insights into plant biological processes, and enable the exploration of the relationships between root morphology, gene expression and the physiological adaptation to spaceflight. The developmental patterns of plant organs are continuously altered through perception of the environment, signal integration, and response to environmental stimuli. A diversity of tropic gradients influence the path of growth in roots by initiating localized, asymmetrical changes in cell elongation. These changes are primarily brought about through hormonal interactions and subsequent remodeling of cell physiology (<a aria-describedby="B77" class="usa-link" href="#B77">Roy and Bassham, 2014</a>; <a aria-describedby="B116" class="usa-link" href="#B116">Vandenbrink et al., 2014</a>; <a aria-describedby="B87" class="usa-link" href="#B87">Schultz et al., 2017</a>). Root skewing and waving are phenomena in which root growth deviates from a gravity vector throughout its development, and which vary between the <em>Arabidopsis thaliana</em> (Arabidopsis) ecotypes Columbia (Col-0) and Wassilewskija (WS) (<a aria-describedby="B79" class="usa-link" href="#B79">Rutherford and Masson, 1996</a>; <a aria-describedby="B77" class="usa-link" href="#B77">Roy and Bassham, 2014</a>; <a aria-describedby="B87" class="usa-link" href="#B87">Schultz et al., 2017</a>). Although once thought to be a gravity-dependent growth behavior (<a aria-describedby="B66" class="usa-link" href="#B66">Oliva and Dunand, 2007</a>), skewing occurs in the microgravity of spaceflight (<a aria-describedby="B67" class="usa-link" href="#B67">Paul et al., 2012a</a>). This suggests that skewing is independent of both the tropic force of gravity and the gravity-induced contact forces between roots and growth media (<a aria-describedby="B59" class="usa-link" href="#B59">Millar et al., 2011</a>; <a aria-describedby="B67" class="usa-link" href="#B67">Paul et al., 2012a</a>, <a aria-describedby="B71" class="usa-link" href="#B71">2013</a>; <a aria-describedby="B63" class="usa-link" href="#B63">Nakashima et al., 2014</a>). Therefore, the spaceflight environment provides a unique and relevant context in which to study genes associated with skewing phenotypes.</p>
<p>Plants grown in the spaceflight environment exhibit complex and unique gene expression patterns (e.g., <a aria-describedby="B69" class="usa-link" href="#B69">Paul et al., 2005</a>; <a aria-describedby="B68" class="usa-link" href="#B68">Paul et al., 2012b</a>, <a aria-describedby="B71" class="usa-link" href="#B71">2013</a>, <a aria-describedby="B70" class="usa-link" href="#B70">2017</a>; <a aria-describedby="B20" class="usa-link" href="#B20">Correll et al., 2013</a>; <a aria-describedby="B49" class="usa-link" href="#B49">Kwon et al., 2015</a>; <a aria-describedby="B39" class="usa-link" href="#B39">Johnson et al., 2017</a>; <a aria-describedby="B19" class="usa-link" href="#B19">Choi et al., 2019</a>). The predominant feature of the spaceflight environment is microgravity. The lack of gravity has a direct effect on plant cells and signal transduction; this direct effect represents a novel environment for plants, and any terrestrial organism, and appears to be generally perceived as stressful. In addition, microgravity imposes secondary environmental stresses due to the disruption of fluid movement, and any processes influenced by convection, such as gas exchange and temperature redistribution. Combinatorial stresses induce changes in gene expression and tolerance that are not fully recapitulated by exposure to the individual stressors (e.g., <a aria-describedby="B75" class="usa-link" href="#B75">Ramegowda and Senthil-Kumar, 2015</a>; <a aria-describedby="B104" class="usa-link" href="#B104">Suzuki et al., 2016</a>). Thus it is important to think of the spaceflight environment as more complex than microgravity alone. Several classes of stress response genes have been identified as consistently involved in the response to spaceflight across ecotypes of Arabidopsis via analyses of gene expression. Heat shock genes are often induced by spaceflight (<a aria-describedby="B69" class="usa-link" href="#B69">Paul et al., 2005</a>, <a aria-describedby="B68" class="usa-link" href="#B68">2012b</a>; <a aria-describedby="B82" class="usa-link" href="#B82">Salmi and Roux, 2008</a>; <a aria-describedby="B91" class="usa-link" href="#B91">Shagimardanova et al., 2010</a>; <a aria-describedby="B128" class="usa-link" href="#B128">Zupanska et al., 2013</a>, <a aria-describedby="B130" class="usa-link" href="#B130">2017</a>, <a aria-describedby="B129" class="usa-link" href="#B129">2019</a>; <a aria-describedby="B39" class="usa-link" href="#B39">Johnson et al., 2017</a>; <a aria-describedby="B19" class="usa-link" href="#B19">Choi et al., 2019</a>). Reactive oxygen species (ROS) signaling and scavenging processes are also common in the spaceflight response, though ROS-associated genes have been observed as both up- and downregulated in spaceflight (<a aria-describedby="B91" class="usa-link" href="#B91">Shagimardanova et al., 2010</a>; <a aria-describedby="B20" class="usa-link" href="#B20">Correll et al., 2013</a>; <a aria-describedby="B71" class="usa-link" href="#B71">Paul et al., 2013</a>, <a aria-describedby="B70" class="usa-link" href="#B70">2017</a>; <a aria-describedby="B49" class="usa-link" href="#B49">Kwon et al., 2015</a>; <a aria-describedby="B19" class="usa-link" href="#B19">Choi et al., 2019</a>; <a aria-describedby="B127" class="usa-link" href="#B127">Zhou et al., 2019</a>). Some stress gene expression changes are also associated with spaceflight-induced changes in the Arabidopsis methylome (<a aria-describedby="B127" class="usa-link" href="#B127">Zhou et al., 2019</a>). Cell wall remodeling processes are enriched in spaceflight gene expression datasets (<a aria-describedby="B68" class="usa-link" href="#B68">Paul et al., 2012b</a>, <a aria-describedby="B71" class="usa-link" href="#B71">2013</a>, <a aria-describedby="B70" class="usa-link" href="#B70">2017</a>; <a aria-describedby="B20" class="usa-link" href="#B20">Correll et al., 2013</a>; <a aria-describedby="B49" class="usa-link" href="#B49">Kwon et al., 2015</a>; <a aria-describedby="B39" class="usa-link" href="#B39">Johnson et al., 2017</a>). Cell wall remodeling genes, typically associated with biotic stress and pathogen defense pathways, also contribute to spaceflight acclimation (<a aria-describedby="B68" class="usa-link" href="#B68">Paul et al., 2012b</a>, <a aria-describedby="B71" class="usa-link" href="#B71">2013</a>, <a aria-describedby="B70" class="usa-link" href="#B70">2017</a>; <a aria-describedby="B20" class="usa-link" href="#B20">Correll et al., 2013</a>; <a aria-describedby="B19" class="usa-link" href="#B19">Choi et al., 2019</a>). Spaceflight also affects abundances of proteins of defense pathways and cell wall remodeling (<a aria-describedby="B55" class="usa-link" href="#B55">Mazars et al., 2014</a>; <a aria-describedby="B27" class="usa-link" href="#B27">Ferl et al., 2015</a>). However, the particular genetic pathways activated and repressed in spaceflight vary among ecotypes of Arabidopsis, demonstrating that there are significant genotypic contributions to spaceflight physiological adaptation (<a aria-describedby="B70" class="usa-link" href="#B70">Paul et al., 2017</a>; <a aria-describedby="B4" class="usa-link" href="#B4">Beisel et al., 2019</a>; <a aria-describedby="B19" class="usa-link" href="#B19">Choi et al., 2019</a>).</p>
<p>Mutations in genes associated with stress response and signaling pathways can significantly alter the differential gene expression profiles of spaceflight physiological adaptation. Single gene mutations in heat shock transcription factors, gravity perception genes, and light signaling genes in Arabidopsis seedlings and cultures exhibit altered spaceflight responses (<a aria-describedby="B70" class="usa-link" href="#B70">Paul et al., 2017</a>; <a aria-describedby="B130" class="usa-link" href="#B130">Zupanska et al., 2017</a>, <a aria-describedby="B129" class="usa-link" href="#B129">2019</a>). Therefore, we sought to understand better the relationships among spaceflight responses, root morphology, and spaceflight adaptation by exploring the spaceflight responses of two single gene skewing-related mutant lines: <em>spiral1</em> and <em>sku5</em>.</p>
<p><em>Spiral1</em> (<em>Spr1</em>) is a skewing-related gene that contributes to the process of anisotropic cell expansion by regulating cortical microtubule dynamics. The <em>spr1</em> mutation results in axial rotation of cell files throughout the plant, which manifests in the roots as skewing to the left when viewed, according to convention, from beneath their growth medium (<a aria-describedby="B79" class="usa-link" href="#B79">Rutherford and Masson, 1996</a>; <a aria-describedby="B31" class="usa-link" href="#B31">Furutani et al., 2000</a>; <a aria-describedby="B62" class="usa-link" href="#B62">Nakajima et al., 2004</a>; <a aria-describedby="B89" class="usa-link" href="#B89">Sedbrook et al., 2004</a>; <a aria-describedby="B32" class="usa-link" href="#B32">Galva et al., 2014</a>). The SPR1 protein is a microtubule plus-end tracking protein that localizes to the plus ends of actively polymerizing cortical microtubules, dissociating upon a shift to microtubule depolymerization (<a aria-describedby="B89" class="usa-link" href="#B89">Sedbrook et al., 2004</a>; <a aria-describedby="B32" class="usa-link" href="#B32">Galva et al., 2014</a>). This association is mediated by two similar motifs on the N- and C-termini of the protein, which allow SPR1 to act as an intermolecular linker (<a aria-describedby="B62" class="usa-link" href="#B62">Nakajima et al., 2004</a>; <a aria-describedby="B89" class="usa-link" href="#B89">Sedbrook et al., 2004</a>; <a aria-describedby="B32" class="usa-link" href="#B32">Galva et al., 2014</a>). The protein END-BINDING 1B (EB1b) co-localizes with SPR1 at the microtubule plus-end, where each protein interacts with the other as well as tubulin subunits (<a aria-describedby="B32" class="usa-link" href="#B32">Galva et al., 2014</a>). This creates the dual effects of SPR1 and EB1b increasing microtubule stability and polymerization rate, while also enhancing the rescue rate of depolymerizing microtubules (<a aria-describedby="B32" class="usa-link" href="#B32">Galva et al., 2014</a>). However, the <em>spr1</em> mutation also leads to a decreased rate of shrinkage among depolymerizing microtubules (<a aria-describedby="B32" class="usa-link" href="#B32">Galva et al., 2014</a>). The regulation of microtubule dynamics is critical to many environmental responses, such as in salt stress where 26S proteasome-mediated degradation of SPR1 is known to occur in conjunction with microtubule reorganization to enable stress acclimation (<a aria-describedby="B94" class="usa-link" href="#B94">Shoji et al., 2006</a>; <a aria-describedby="B117" class="usa-link" href="#B117">Wang et al., 2011</a>; <a aria-describedby="B18" class="usa-link" href="#B18">Chen et al., 2016</a>). In addition to these micro-scale changes, the skewing phenotype of <em>spr1</em> is significantly enhanced by cold treatment, and suppressed by both salt and heat stresses (<a aria-describedby="B31" class="usa-link" href="#B31">Furutani et al., 2000</a>; <a aria-describedby="B89" class="usa-link" href="#B89">Sedbrook et al., 2004</a>). This connects <em>SPR1</em> to processes regulating morphology in both optimal and stressful environments.</p>
<p><em>Sku5</em> is a skewing-associated gene that encodes a protein in the SKU5-SIMILAR (SKS) family, which is related to known Arabidopsis copper oxidases, but which contains only one Type-II copper-coordinating domain of unknown specificity (<a aria-describedby="B88" class="usa-link" href="#B88">Sedbrook et al., 2002</a>). The <em>sku5</em> mutant was originally noted for the rightward skewing it exhibits on vertically oriented growth media, which manifests in both WS and Col-0 backgrounds (<a aria-describedby="B88" class="usa-link" href="#B88">Sedbrook et al., 2002</a>; <a aria-describedby="B105" class="usa-link" href="#B105">Swarbreck et al., 2019</a>). <em>Sku5</em> seedlings exhibit reductions of root length but not of cell size, and SKU5 is hypothesized to act in the process of cell division as a result (<a aria-describedby="B88" class="usa-link" href="#B88">Sedbrook et al., 2002</a>). SKU5 localizes to the plasma membrane, but is also present in soluble and cell wall-binding forms in the extracellular milieu (<a aria-describedby="B88" class="usa-link" href="#B88">Sedbrook et al., 2002</a>; <a aria-describedby="B8" class="usa-link" href="#B8">Borderies et al., 2003</a>; <a aria-describedby="B2" class="usa-link" href="#B2">Baral et al., 2015</a>; <a aria-describedby="B17" class="usa-link" href="#B17">Chen et al., 2018</a>). SKU5 is a glycosylphosphatidylinositol-anchored protein (GPI-AP), and as such is a component of discrete plasma membrane nanodomains known as “lipid rafts” (<a aria-describedby="B9" class="usa-link" href="#B9">Borner et al., 2005</a>; <a aria-describedby="B25" class="usa-link" href="#B25">Elortza et al., 2006</a>; <a aria-describedby="B17" class="usa-link" href="#B17">Chen et al., 2018</a>; <a aria-describedby="B53" class="usa-link" href="#B53">Mamode Cassim et al., 2019</a>). These nanodomains of the plasma membrane and their associated GPI-APs play roles in stress response signaling at the interface between the cell wall and membrane, where they are envisioned to help shape the “signatures” that activate downstream plant adaptive responses to the stresses they experience (<a aria-describedby="B60" class="usa-link" href="#B60">Minami et al., 2008</a>; <a aria-describedby="B108" class="usa-link" href="#B108">Takahashi et al., 2013</a>, <a aria-describedby="B109" class="usa-link" href="#B109">2016</a>, <a aria-describedby="B107" class="usa-link" href="#B107">2019</a>; <a aria-describedby="B90" class="usa-link" href="#B90">Shabala et al., 2015</a>; <a aria-describedby="B123" class="usa-link" href="#B123">Yeats et al., 2018</a>; <a aria-describedby="B58" class="usa-link" href="#B58">Miki et al., 2019</a>). The endocytosis of SKU5 and other GPI-APs is modulated across the root tip by salt and auxin treatments, linking the movement of these proteins to stress responses (<a aria-describedby="B2" class="usa-link" href="#B2">Baral et al., 2015</a>). The expression of SKU5 is itself regulated by boron deprivation, cold, salt, and immune responses, as well as abscisic acid (ABA) treatment (<a aria-describedby="B42" class="usa-link" href="#B42">Keinath et al., 2010</a>; <a aria-describedby="B24" class="usa-link" href="#B24">Elmore et al., 2012</a>; <a aria-describedby="B52" class="usa-link" href="#B52">Li et al., 2012</a>; <a aria-describedby="B110" class="usa-link" href="#B110">Tanaka et al., 2016</a>; <a aria-describedby="B92" class="usa-link" href="#B92">Shi et al., 2018</a>; <a aria-describedby="B58" class="usa-link" href="#B58">Miki et al., 2019</a>). While SKU5 interactors are unknown in Arabidopsis, a maize homolog of SKU5 interacts with a C-terminal peptide of AUXIN-BINDING PROTEIN 1 (ABP1) <em>in vitro</em> (<a aria-describedby="B93" class="usa-link" href="#B93">Shimomura, 2006</a>). ABP1 is associated with auxin signaling pathways that activate the plant TARGET OF RAPAMYCIN (TOR) regulatory complex, remodel the cortical microtubule network, and induce expansive growth by triggering cation influx (<a aria-describedby="B120" class="usa-link" href="#B120">Xu et al., 2014</a>; <a aria-describedby="B18" class="usa-link" href="#B18">Chen et al., 2016</a>; <a aria-describedby="B21" class="usa-link" href="#B21">Dahlke et al., 2017</a>; <a aria-describedby="B84" class="usa-link" href="#B84">Schepetilnikov et al., 2017</a>). This further connects SKU5 to characterized pathways which affect stress responses and regulate the balance between growth and autophagic processes under stressful conditions, such as that of spaceflight.</p>
<p>SKU5 and SPR1 are therefore very different proteins in seemingly unrelated physiological processes, but both proteins have roles in root skewing. Both of these proteins are also associated with cell wall remodeling, a process that is regularly involved in spaceflight acclimation. As such, these genes offer an appropriate initial genetic dissection of root skewing and the effect of skewing pathways in the microgravity of spaceflight environments. Extensive discussion of the relevance of skewing and waving in microgravity has been presented elsewhere (e.g., <a aria-describedby="B67" class="usa-link" href="#B67">Paul et al., 2012a</a>), but to paraphrase from that paper, prior to spaceflight research, the consensus was that gravitropism was the directional driver in skewing (e.g., <a aria-describedby="B96" class="usa-link" href="#B96">Simmons et al., 1995</a>; <a aria-describedby="B66" class="usa-link" href="#B66">Oliva and Dunand, 2007</a>). Yet we now know that gravity is not required, and root skewing in spaceflight appears to be an inherent feature of many Arabidopsis ecotypes, even in conditions lacking both light and gravity (<a aria-describedby="B59" class="usa-link" href="#B59">Millar et al., 2011</a>; <a aria-describedby="B67" class="usa-link" href="#B67">Paul et al., 2012a</a>, <a aria-describedby="B70" class="usa-link" href="#B70">2017</a>; <a aria-describedby="B63" class="usa-link" href="#B63">Nakashima et al., 2014</a>). These observations suggest a testable relationship among root morphologies governed by skewing regulation pathways. The hypothesis is that skewing pathways play a large role in spaceflight adaptation via the inherent cell wall remodeling that accompanies those morphologies and the physiological adaptation to spaceflight. This relationship was examined by growing <em>spr1</em>, <em>sku5</em> and their wild type controls for 4 and 8 days on the International Space Station and at the Kennedy Space Center, then observing both their growth morphologies and gene expression profiles in response to spaceflight as measures of response quality and complexity. A change in gene expression or morphology during spaceflight would suggest a significant role for these skewing genes in the physiological adaptation process.</p>
<p>Differential gene expression profiles are often deployed as measures of the underlying gene expression changes needed for physiological adaptation and developmental changes within an organism. Differential gene expression responses, in terms of the number of genes involved and their fold-change levels, may be considered a measure of the metabolic cost of adapting to that environment (e.g.: <a aria-describedby="B16" class="usa-link" href="#B16">Chan et al., 2016</a>). Therefore gene expression profiles morphologies were used to examine the relationships between skewing genes and the resulting complexity of spaceflight acclimation. SKU5 and SPR1 function in different cellular pathways that affect skewing on the Earth, which enabled a test of the relevance of contrasting skewing pathways to spaceflight physiological adaptation. Growth morphologies were used to examine the productivity and developmental success of these genotypes in spaceflight.</p></section><section id="S2"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="S2.SS1"><h3 class="pmc_sec_title">Plant Material and Plate Setup</h3>
<p><em>Arabidopsis thaliana</em> wild ecotype Columbia (Col-0, CS70000) seed stock, and T-DNA insertion lines for the <em>spr1</em> (CS6547 – Col-0 background) and <em>sku5</em> (CS16268 – WS background), were acquired from the Arabidopsis Biological Resource Center (ABRC) (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://arabidopsis.org" rel="noopener noreferrer" target="_blank">arabidopsis.org</a>; <a aria-describedby="B50" class="usa-link" href="#B50">Lamesch et al., 2011</a>). The wild type Wassilewskija (WS) line used in this study was propagated in our laboratory for more than 25 years. This WS line has been used in multiple spaceflight studies (<a aria-describedby="B67" class="usa-link" href="#B67">Paul et al., 2012a</a>, <a aria-describedby="B71" class="usa-link" href="#B71">2013</a>, <a aria-describedby="B70" class="usa-link" href="#B70">2017</a>; <a aria-describedby="B127" class="usa-link" href="#B127">Zhou et al., 2019</a>), and seed samples are available upon request. The ABRC denotes WS as stock CS915. Petri dishes (100 mm × 15 mm; Fisher Scientific, Pittsburgh, PA, United States), containing 50 mL of a 0.5% Phytagel-based growth medium supplemented with: 0.5× Murashige-Skoog salts, 0.5% (w/v) sucrose, and 1× Gamborg’s Vitamin Mixture, were prepared aseptically for planting. Seeds were sterilized and planted according to methods allowing for the maintenance of seed dormancy (<a aria-describedby="B99" class="usa-link" href="#B99">Sng et al., 2014</a>). Briefly, seeds were stored with Desiccant-Anhydrous Indicating Drierite (W.A. Hammond Drierite Company, stock #24001) for 1 week before sterilization. The seeds were sterilized with 70% ethanol for 10 min and dried in a laminar flow hood. Seeds were stored at 4°C in sterile screw-cap microcentrifuge tubes until planting. Seeds were suspended in sterile water and dispensed onto the media surface. Approximately 12–15 seeds were planted in a row on each plate, with one row on the 8-day (8d) plates and two rows on the 4-day (4d) plates. At time of harvest, each genotype and age was represented by 9–25 viable, replicate seedlings. All replicates contributed to morphological observations, whereas the transcriptome analyses were conducted with four biological replicates, comprising 5–8 4d plant roots, or 2–3 8d plant roots. The plates were sealed with Micropore<sup>®</sup> (3M, Maplewood, MN, United States) tape, and wrapped in Duvetyne Black-Out Fabric (Seattle Fabrics). The time from suspending dry seeds in water to the completed wrapping of the plate in Duvetyne was less than 10 min; this timing was essential to maintain seed dormancy. Wrapped plates were then transported to the Kennedy Space Center (KSC, FL, United States) under cold stowage, maintaining temperatures of 4–10°C until launch.</p></section><section id="S2.SS2"><h3 class="pmc_sec_title">Spaceflight Experimental Workflow</h3>
<p>The Advanced Plant Experiments 03-2 (APEX-03-2) study, also known under NASA Operational Nomenclature as the Transgenic Arabidopsis Gene Expression System – Intracellular Signaling Architecture (TAGES-ISA) study, has been previously described (<a aria-describedby="B28" class="usa-link" href="#B28">Ferl and Paul, 2016</a>; <a aria-describedby="B4" class="usa-link" href="#B4">Beisel et al., 2019</a>; <a aria-describedby="B127" class="usa-link" href="#B127">Zhou et al., 2019</a>). The SpaceX CRS-5 mission, carrying the dormant, seeded growth plates as part of its cargo, was launched to the International Space Center (ISS) from KSC on January 10, 2015. Capsule docking and transfer of materials to the ISS occurred on January 12 and 13, respectively. The growth plates were removed from cold stowage onboard the ISS on January 26, unwrapped, and inserted into the Vegetable Production System (VPS, colloquially “Veggie”) perpendicular to the light bank (<a class="usa-link" href="#F1">Figure 1A</a>). During the growth period, constant lighting at a level of 100–135 μmol m<sup>–2</sup> s<sup>–1</sup> was used. Each age and genotype was represented by a 10 cm Petri plate. There were 12–15 seedlings on the 8-day-old plates (8d) and 35–40 seedlings on the 4-day-old plates (4d). The plates were removed from the Veggie plant growth hardware after 4 and 8 days of growth, at which time they were photographed by the crew. The seedlings of each plate were then harvested into individual KSC Fixation Tubes (KFTs) pre-loaded with RNAlater<sup>TM</sup> (Ambion, Grand Island, NY, United States) preservative (one plate per KFT) (<a class="usa-link" href="#F1">Figure 1A</a>). Actuation of the KFT submerged the seedlings in RNAlater and sealed the tube. The harvest tubes remained at ambient temperature for 12 h to allow full perfusion of the tissues, and were then transferred to the MELFI −80°C freezer. The samples were transferred from the MELFI to cold stowage onboard the Dragon capsule on February 9<sup>th</sup> and remained frozen in transit to KSC for de-integration on February 15, at which time the samples were transferred to 50 mL Falcon tubes with minimal thawing. The samples were transferred to the Principal Investigators and were transported to the University of Florida, where they were kept under −80°C storage.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 1" class="graphic" height="828" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/dce93b24c773/fpls-11-00239-g001.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Advanced Plant Experiments-03-2 overview. <strong>(A)</strong> Images detailing the standard procedures used onboard the ISS during the experiment. In Panel 1, the Vegetable Production System (VPS, “Veggie”) and its local environment can be seen, while panel 2 provides a close-up of the plate setup inside Veggie after insertion. Panel 3 shows astronaut Butch Wilmore harvesting seedlings into a KFT (Kennedy Space Center Fixation Tube), and the workstation used for the orbital harvests. The images of the Veggie hardware and Butch Wilmore were taken on the ISS by NASA; all NASA photos are in the public domain. <strong>(B)</strong> Full images of plates taken at the time of harvest and from which tissue was used in the APEX-03-2 RNA-Seq experiment. Images from the ground control are in the top two rows, while those from the spaceflight experiment are in the bottom two rows. The genotype and developmental age of the seedlings are listed on each image. The provided images were taken from above the surface of the growth medium.</p></figcaption></figure><p>The ground control (GC) for the experiment, composed of an identical set of plants and plates, was performed on a 48-h delay at KSC using Veggie hardware within the ISS Environmental Simulator (ISSES) chamber. The same growth timeline was used, with seedlings being imaged and harvested into RNAlater-containing KFTs at 4d and 8d time points. The GC operations were as described above for the spaceflight experiment, with KSC personnel following the precise timing of the astronaut activities. Telemetry data also enabled the ISSES chamber to replicate the CO<sub>2</sub> levels, temperature, and ambient lighting in the vicinity of the Veggie hardware onboard the ISS across the course of the experiment.</p>
<p>The consistency of operations between spaceflight and GCs was also checked against video data captured on the ISS. Over-the-shoulder videos of astronaut activities for the 4d and 8d harvests were examined for timing of operations and showed that spaceflight seedling harvests were not disproportionately treated during the process at either time point or compared to GC harvests. The plates were harvested in comparable windows of time, ensuring that no additional stress, such as drought stress, was introduced into the harvest process (<a class="usa-link" href="#SM1">Supplementary Table S1</a>).</p></section><section id="S2.SS3"><h3 class="pmc_sec_title">RNA Isolation</h3>
<p>The spaceflight and GC seedlings, stored in RNAlater, were transferred from −80°C storage to 4°C overnight to thaw. Seedlings in RNAlater were examined with an Olympus SZX12 stereoscope (Olympus Corporation, Tokyo, Japan) and whole roots were dissected away from the shoot and hypocotyl. The remaining shoot and hypocotyl tissues were restored to −80°C in RNAlater. RNA was prepared from 5–8 4d plant roots, and 2–3 8d plant roots. Total RNA was extracted using the Qiashredder and RNAeasy kits from QIAGEN (QIAGEN Sciences, MD, United States) according to the manufacturer’s instructions. An on-column digestion with RNase-free DNase (QIAGEN GmbH, Hilden, Germany) was used to remove residual DNA.</p></section><section id="S2.SS4"><h3 class="pmc_sec_title">Library Preparation</h3>
<p>Library preparation was performed at the University of Florida’s Interdisciplinary Center for Biotechnology Research (ICBR) Gene Expression Core. RNA integrity was verified using the Agilent 2100 BioAnalyzer (Agilent Technologies, Santa Clara, CA, United States). Of the total RNA, 10 ng were used to construct cDNA libraries with the ClonTech SMART-Seq v4 ultra-low input RNA kit for sequencing (Clontech Laboratories, Inc., Cat#: 634890), according to the manufacturer’s instructions. Briefly, 1st strand cDNA was primed by the SMART-Seq v4 oligonucleotide, which then base-pairs with these additional nucleotides, creating an extended template. The reverse transcriptase then switches templates and continues transcribing to the end of the oligonucleotide, resulting full-length cDNA that contains an anchor sequence that serves as a universal priming site for second strand synthesis. Then, cDNA was amplified with primer II A for 8 PCR cycles. Illumina sequencing libraries were then generated with 150 pg of cDNA using the Illumina Nextera DNA Sample Preparation Kit (Cat#: FC-131-1024) according to the manufacturer’s instructions. Briefly, 150 pg of cDNA were fragmented by tagmentation reaction and adapter sequences were added onto template cDNA by PCR amplification. Libraries were quantitated using both the 2100 BioAnalyzer and qPCR (Kapa Biosystems, catalog number: KK4824).</p></section><section id="S2.SS5"><h3 class="pmc_sec_title">RNA-Seq</h3>
<p>Sequencing experiments were performed at the UF ICBR Next-Generation DNA Sequencing Core. In preparation for sequencing, barcoded libraries were sized on the 2100 BioAnalyzer, and quantitated by QUBIT (Thermo Fisher Scientific, Waltham, MA, United States) and qPCR (Kapa Biosystems, catalog number: KK4824). Individual libraries were pooled equimolarly at 4 nM. This “working pool” was used as input in the NextSeq500 instrument sample preparation protocol (Illumina, Part#: 15048776, Rev A). Typically, a 1.3 pM library concentration resulted in an optimal clustering density in our instrument (i.e., ∼200,000 clusters per mm<sup>2</sup>). Samples were sequenced on 5 flow cells (5 NextSeq500 runs), using a 2 × 75 cycles (paired-end) configuration. A typical sequencing run in the NextSeq500 produced 750–800 million paired-end reads with a Q30 ≥ 85%. For RNA-Seq, around 40 million paired-end reads per sample provided sufficient depth for transcriptome analysis (<a aria-describedby="B112" class="usa-link" href="#B112">Tarazona et al., 2011</a>).</p></section><section id="S2.SS6"><h3 class="pmc_sec_title">Bioinformatic Analysis</h3>
<p>Bioinformatic processing of the sequencing data was performed at the UF ICBR Bioinformatics Core. Fastq files were trimmed to remove sequencing adapters and low-quality base calls using Trimmomatic (version 0.36) with the parameters: LEADING:3, TRAILING:3, SLIDINGWINDOW:4:15, MINLEN:50 (<a aria-describedby="B7" class="usa-link" href="#B7">Bolger et al., 2014</a>). Quality control of the trimmed reads was performed using FastQC (version 0.11.2) (<a aria-describedby="B1" class="usa-link" href="#B1">Andrews, 2010</a>). Reads for each set of Arabidopsis lines were aligned to their respective reference genomes using the STAR aligner (version 2.5.1b) (<a aria-describedby="B23" class="usa-link" href="#B23">Dobin et al., 2012</a>). The Col-0 TAIR10 genome release was used for reads from Col-0 and <em>spr1</em> (<a aria-describedby="B50" class="usa-link" href="#B50">Lamesch et al., 2011</a>; <a aria-describedby="B5" class="usa-link" href="#B5">Berardini et al., 2015</a>), while reads from WS and <em>sku5</em> were aligned to the WS reference genome (<a aria-describedby="B33" class="usa-link" href="#B33">Gan et al., 2011</a>). Duplicate reads resulting from PCR artifacts were then removed using the Picard MarkDuplicates tool (<a aria-describedby="B12" class="usa-link" href="#B12">Broad Institute, 2019b</a>). In total, 502 million transcriptomic reads were aligned. Following alignment, expression quantification and differential gene expression analysis were performed using Cufflinks and Cuffdiff (version 2.2.1), respectively, with default parameters (<a aria-describedby="B114" class="usa-link" href="#B114">Trapnell et al., 2010</a>, <a aria-describedby="B113" class="usa-link" href="#B113">2012</a>). Output from Cuffdiff was then parsed with custom scripts to generate the final annotated tables of differentially expressed genes (DEGs). A cut-off false-discovery rate (FDR) of 0.05 was used for calling statistical significance in differential gene expression</p></section><section id="S2.SS7"><h3 class="pmc_sec_title">RNA-Seq Data Analysis</h3>
<p>Tables of DEGs for each comparison were combined and analyzed through Microsoft Excel, in order to generate lists of Arabidopsis Genome Initiative (AGI) identifiers with their corresponding [Log<sub>2</sub>(Fold Change)] values. Machine annotation of AGI ID lists was carried out using G:Profiler (<a aria-describedby="B76" class="usa-link" href="#B76">Raudvere et al., 2019</a>), while the TAIR (<a aria-describedby="B50" class="usa-link" href="#B50">Lamesch et al., 2011</a>; <a aria-describedby="B5" class="usa-link" href="#B5">Berardini et al., 2015</a>) and Araport ThaleMine (<a aria-describedby="B48" class="usa-link" href="#B48">Krishnakumar et al., 2016</a>) databases were used to investigate genes of interest. DEGs of a comparison (e.g., FLT vs. GC) with a greater than twofold change in at least one context of the comparison were retained for analysis (<a class="usa-link" href="#SM1">Supplementary File S1</a>). This was done in order to minimize the effect instituting a cut-off would have on analyzing the expression patterns of a gene. Significant DEGs with a fold-change of “inf” or “-inf” were retained and assigned a [Log<sub>2</sub>(Fold Change)] value of 10 for heat-mapping, as they have a consistent FPKM of zero in one context but are consistently detected at a significantly higher level in the opposing context. Gene ontology (GO) analyses of only those DEGs with a greater than twofold change were carried out using the lists of their AGI IDs, separated by their up- or downregulation, with the PANTHER statistical overrepresentation tool (<a aria-describedby="B57" class="usa-link" href="#B57">Mi et al., 2018</a>). GO terms for each comparison were separated into those unique to one context and those overlapping between contexts, and the output is available in <a class="usa-link" href="#SM1">Supplementary File S2</a>. Of those resulting, all GO terms overlapping between contexts were retained for heatmapping. Either one-half or five, whichever was greater, of each context’s unique GO terms were retained after sorting by <em>q</em>-value from most to least significant. GO term lists were then manually pruned in order to reduce highly redundant information (e.g., “cellular response to chemical” and “response to chemical”) only when the redundant terms demonstrated the same pattern of over- or underrepresentation across contexts. Complete lists of genes annotated to specific GO Terms of interest were gathered from AmiGO2 (<a aria-describedby="B14" class="usa-link" href="#B14">Carbon et al., 2008</a>). Heatmaps were then generated using DEG [Log<sub>2</sub>(Fold Change)] values or GO term [-Log(<em>q</em>-value)] values as input for the Morpheus webtool (<a aria-describedby="B11" class="usa-link" href="#B11">Broad Institute, 2019a</a>).</p></section><section id="S2.SS8"><h3 class="pmc_sec_title">Measurements of Seedling Roots</h3>
<p>Germination rates and the growth of roots in each environment were derived from images taken of the growth plates at the times of harvest (<a class="usa-link" href="#F1">Figure 1B</a>). Germination rates were assessed by counting the number of seedlings and ungerminated seeds from the 4-day images taken just prior to harvest. Statistical significance of germination rate changes were assessed using χ<sup>2</sup> tests with the conservative Yates correction (<a aria-describedby="B122" class="usa-link" href="#B122">Yates, 1934</a>). The comparisons made were between mutant and wild-type lines, and between spaceflight and GC growth conditions. To measure roots, images were analyzed using the FIJI distribution of ImageJ (<a aria-describedby="B85" class="usa-link" href="#B85">Schindelin et al., 2012</a>; <a aria-describedby="B78" class="usa-link" href="#B78">Rueden et al., 2017</a>). The JFilament plugin was used to track primary roots, creating a set of points describing each root (<a aria-describedby="B97" class="usa-link" href="#B97">Smith et al., 2010</a>). These data were processed using an R script that provided the length of each primary root as output, alongside other measures (<a aria-describedby="B86" class="usa-link" href="#B86">Schultz et al., 2016</a>; <a aria-describedby="B74" class="usa-link" href="#B74">R Core Team, 2019</a>). Lengths were corrected for scale based on the pixel length of the Petri dish grid-squares to allow for comparisons between images. Differences in root length were assessed through two-tailed Student’s <em>t</em>-tests, with the Bonferroni correction for multiple testing applied (<a aria-describedby="B98" class="usa-link" href="#B98">Snedecor and Cochran, 1989</a>). The comparisons made were between spaceflight and GC seedlings for each genotype, and between each skewing mutant and its respective wild-type. As conventional measurements of angles of root growth are dependent upon a reference gravity vector, the roots of 4-day-old seedlings were measured manually using FIJI (<a aria-describedby="B85" class="usa-link" href="#B85">Schindelin et al., 2012</a>; <a aria-describedby="B78" class="usa-link" href="#B78">Rueden et al., 2017</a>). The angle between the root tip and the growth direction of the beginning of the root was used. Positive and negative angles were used to represent rightward and leftward changes in directionality, respectively, relative to the initial direction of growth when viewed from below the growth media. Angles were plotted using the ggplot2 package in R (<a aria-describedby="B118" class="usa-link" href="#B118">Wickham, 2016</a>). In these polar plots, the magnitude of a bar corresponds to the number of roots within a particular bin, with the placement of the bar corresponding to the measured angle. Differing upper limits were used for spaceflight (0–20) and GC (0–10) plots in order to allow the many lower-magnitude bins in the spaceflight plots to be visualized by effectively zooming-in on the data. However, the scaling between inner rings is consistent between plots. As such, a bar meeting the second ring at the −25° line indicates that ten roots had an angle shift between 15° and 30° to the left during their growth.</p></section></section><section id="S3"><h2 class="pmc_sec_title">Results</h2>
<p>APEX-03 seedling growth in occurred in the Veggie hardware on the ISS (<a class="usa-link" href="#F1">Figure 1A</a>). Veggie is housed in the Columbus module (<a class="usa-link" href="#F1">Figure 1A</a>, left), and for the APEX-03 experiment, configured to accommodate racks of 10 cm Petri plates (<a class="usa-link" href="#F1">Figure 1A</a>, middle). Images of plant growth and morphology were recorded by astronaut Butch Wilmore just prior to harvest (<a class="usa-link" href="#F1">Figures 1A</a>, right, <a class="usa-link" href="#F1">B</a>). Col-0 and WS can display distinct patterns of root growth in response to environmental stimuli. In terrestrial environments, changing the angle of the growth surface can create ecotype-specific patterns of skewing (<a aria-describedby="B79" class="usa-link" href="#B79">Rutherford and Masson, 1996</a>; <a aria-describedby="B87" class="usa-link" href="#B87">Schultz et al., 2017</a>). In a microgravity environment with a gradient light source to impart a tropic cue, skewing patterns of WS and Col-0 recapitulate the patterns seen on terrestrial angled growth surfaces (<a aria-describedby="B67" class="usa-link" href="#B67">Paul et al., 2012a</a>). However, in a microgravity environment with uniform, non-directional lighting such as provided by the Veggie hardware in the current experiment, root growth patterns are more disorganized (<a class="usa-link" href="#F1">Figure 1B</a>). Nonetheless, skewing trends can still be discerned by following the angles of growth as the roots develop, and these are quantified in <a class="usa-link" href="#F2">Figure 2A</a> for Col-0 and <em>spr1</em>, and in <a class="usa-link" href="#F2">Figure 2B</a> for WS and <em>sku5</em>. In all representations and discussions of skewing, the direction indicated is from the perspective of being viewed from behind the growth medium.</p>
<figure class="fig xbox font-sm" id="F2"><h3 class="obj_head">FIGURE 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7064724_fpls-11-00239-g002.jpg" target="_blank"><img alt="FIGURE 2" class="graphic zoom-in" height="714" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/ade3f8631809/fpls-11-00239-g002.jpg" width="742"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Quantification of APEX-03-2 Root Growth. Images acquired at the time of harvest of spaceflight (FLT) and ground control (GC) plants were used for measurements. Primary root length measurements for <strong>(A)</strong> Col-0 and <em>spr1</em> and <strong>(B)</strong> WS and <em>sku5</em> were taken using Jfilament and analyzed in R. Significant differences between genotypes (<em>spr1</em> vs. Col-0, <em>sku5</em> vs. WS) and growth conditions (FLT vs. GC) were assessed via two-tailed <em>t</em>-tests and are indicated by asterisks above bars connecting the relevant sets of measurements (Bonferroni correction: <sup>∗</sup><em>p</em> &lt; 0.0125; <sup>∗∗</sup><em>p</em> &lt; 0.001). The 8d Col-0 FLT vs. GC comparison did not meet this cutoff (<em>p</em> = 0.0151). Error bars show the SEM of each set of measurements. <strong>(C)</strong> The angle measured between the root tip and the initial direction of root growth in the 4-day-old seedlings. Plots are separated into rows by growth condition, and into columns by genotype. Positive and negative angles represent rightward and leftward growth of roots, respectively, when viewed from behind the media. Angles are binned in 10° intervals, with each bar’s length representing the number of roots within that bin. As such, a bar reaching the second ring of each plot represents ten roots within that bin. However, note that the upper limit of bin size is lowered in the spaceflight plots to accommodate the higher variation in these data. The total number of roots represented in each plot is indicated in its bottom-right corner, and the mean angle of each plot is noted in its bottom-left corner. FLT, spaceflight; GC, ground control.</p></figcaption></figure><section id="S3.SS1"><h3 class="pmc_sec_title">Col-0 and <em>Spr1</em>
</h3>
<section id="S3.SS1.SSS1"><h4 class="pmc_sec_title">Germination and Morphology of <em>spr1</em> in Spaceflight</h4>
<p>Phenotypic differences among the Col-0 background plants in GC and spaceflight (FLT) environments were primarily limited to skewing angles in the roots (<a class="usa-link" href="#F2">Figures 2</a>, <a class="usa-link" href="#F3">3</a>). There were no statistically significant differences in the primary root length between GC and FLT in either Col-0 or <em>spr1</em> 4d or 8d plants. Although Col-0 root length appeared to be reduced in spaceflight in 8d plants, the <em>p-</em>value did not quite meet the requirements for significance (two-tailed <em>t</em>-test, <em>p</em> = 0.0152, whereas cut-off was <em>p</em> &lt; 0.0150) (<a class="usa-link" href="#F2">Figure 2A</a>). There were differences in root length between Col-0 and <em>spr1</em> genotypes in the GC (<em>p</em> = 1.48 E-9) but FLT Col-0 and FLT <em>spr1</em> roots were not statistically different.</p>
<figure class="fig xbox font-sm" id="F3"><h5 class="obj_head">FIGURE 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 3" class="graphic" height="845" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/9da18eae97fd/fpls-11-00239-g003.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>High resolution images of the Col-0 and <em>spr1</em> growth patterns. Images are arranged such that the ground control seedlings <strong>(A–D)</strong> are in the left column while spaceflight seedlings are on the right <strong>(E–H)</strong>. Vertically, images alternate between Col-0 <strong>(A,C,E,G)</strong> and <em>spr1</em>
<strong>(B,D,F,H)</strong>. The images taken at the 4 day time point <strong>(A,B,E,F)</strong> are grouped above those taken at the 8 day time point <strong>(C,D,G,H)</strong>. Images taken from above the growth medium at the time of harvest. Grid squares are 13 mm wide.</p></figcaption></figure><p>Both 4d and 8d GC Col-0 seedlings grew with very little left or right deviation down the vertical face of the plate medium, while the GC <em>spr1</em> mutant plants of both ages showed distinct skewing to the left (<a class="usa-link" href="#F2">Figures 2C</a>, <a class="usa-link" href="#F3">3A–D</a>). The 4d FLT Col-0 plants generally skewed slightly to the left with some variation (<a class="usa-link" href="#F2">Figures 2C</a>, <a class="usa-link" href="#F3">3E</a>). In contrast, the roots of the 4d FLT <em>spr1</em> plants demonstrated an increased severity of skewing, which was manifested as a strong left-hand curved growth pattern (<a class="usa-link" href="#F2">Figures 2C</a>, <a class="usa-link" href="#F3">3E,F</a>) that was significantly different between GC and FLT (two-tailed <em>t</em>-test, <em>p</em> = 0.00385). For both genotypes the variance in the growth angle (as noted in the SD values in <a class="usa-link" href="#F2">Figure 2</a>) was greater in the spaceflight samples. After 8d of growth, the increased skewing in FLT <em>spr1</em> led to associations with other roots, in contrast to FLT Col-0 which tended to spread more evenly across the plate (<a class="usa-link" href="#F3">Figures 3G,H</a>).</p>
<p>There were no differences between GC and FLT the germination rates of either Col-0 or <em>spr1</em>. The 4d harvest images (<a class="usa-link" href="#F3">Figures 3A,B,E,F</a>) were used to compare seedlings with any ungerminated seeds; germination in both genotypes was 100% under both GC and FLT conditions (<a class="usa-link" href="#SM1">Supplementary Table S2</a>).</p></section><section id="S3.SS1.SSS2"><h4 class="pmc_sec_title">The <em>spr1</em> Spaceflight Response Involved Fewer Transcriptomic Changes Than Col-0</h4>
<p>Spaceflight-conditioned differential gene expression patterns were highly dependent upon the genotype, ecotype, and developmental age of seedlings. Quantitative gene expression data revealed patterns from three perspectives: spaceflight versus GCs for each genotype at each developmental age (<a class="usa-link" href="#F4">Figure 4A</a>: FLT vs. GC), wild type plants versus the mutant genotype in each environment and age (<a class="usa-link" href="#F4">Figure 4B</a>: <em>spr1</em> vs. Col-0) and between the two developmental ages (<a class="usa-link" href="#F4">Figure 4C</a>: 4d vs. 8d). The numbers of differentially expressed genes (DEGs) were categorized according to the magnitude of their fold-change values in the Col-0 and <em>spr1</em> roots. In the FLT vs. GC comparison (<a class="usa-link" href="#F4">Figure 4A</a>), both lines differentially expressed fewer genes at the 8d timepoint than at 4d in all fold-change categories. The spaceflight acclimation of <em>spr1</em> also required fewer genes than Col-0 to be differentially expressed. In the direct comparison of <em>spr1</em> with Col-0 (<a class="usa-link" href="#F4">Figure 4B</a>), more DEGs were observed within each growth condition at 4d than at 8d. The difference in total DEG count between conditions was also reduced at 8d. This occurred in almost all fold-change categories, and differences in total DEGs between the lines diminished between timepoints irrespective of the growth condition. In the developmental comparison of expression between the 4d and 8d timepoints (<a class="usa-link" href="#F4">Figure 4C</a>), the roots from the FLT context showed lower counts of total DEGs than their respective GC roots. However, while <em>spr1</em> required slightly fewer genes to be differentially expressed during development in the GC context, the opposite was true in FLT, where <em>spr1</em> required more DEGs for spaceflight development than Col-0. Twofold-change categories that did not follow these trends were those of the DEGs upregulated and downregulated more than twofold across <em>spr1</em> and Col-0 development, respectively, which were larger in FLT than in GC.</p>
<figure class="fig xbox font-sm" id="F4"><h5 class="obj_head">FIGURE 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7064724_fpls-11-00239-g004.jpg" target="_blank"><img alt="FIGURE 4" class="graphic zoom-in" height="528" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/caba8da09eef/fpls-11-00239-g004.jpg" width="793"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Col-0 and <em>spr1</em> overall differential gene expression levels. A graphic representation of the genes differentially expressed in the <strong>(A)</strong> FLT vs. GC, <strong>(B)</strong> 4d vs. 8d, and <strong>(C)</strong>
<em>spr1</em> vs. Col-0 comparisons. Upregulation of a gene denotes a higher expression level in the first context of the comparison. Genes are separated into categories based on direction of differential expression and a Log<sub>2</sub>(Fold-Change) magnitude cut-off of one. d, day; DEG, differentially expressed gene; Downreg, downregulated; FLT, spaceflight; GC, ground control; Mag, magnitude; Upreg, upregulated. Data are based on four biological replicates.</p></figcaption></figure><p>The relative contributions of DEGs with high degrees of differential expression (greater than twofold change) in response to spaceflight were variable among genotypes and developmental age. In the FLT vs. GC comparison (<a class="usa-link" href="#F4">Figure 4A</a>), the proportion of these DEGs was elevated in <em>spr1</em> (4d: 41.8%, 8d: 22.0%) relative to Col-0 (4d: 17.8%, 8d: 12.5%), and the difference in this proportion between the lines declined over time. However, when the genotypic comparison of s<em>pr1</em> vs. Col-0 (<a class="usa-link" href="#F4">Figure 4B</a>) was made, the proportion of these DEGs showed the opposite trend. In this case, the level of differential expression was slightly higher in FLT at 4d (GC: 16.5%, FLT: 23.9%) and at 8d the proportion of these DEGs was elevated in GC (GC: 30.2%, FLT: 11.9%). In the developmental comparison (<a class="usa-link" href="#F4">Figure 4C</a>), the primary point of interest was the low number of genes whose expression increased to a high degree over time in Col-0 GC roots (33 DEGs) when compared to the other contexts. In the same time frame, <em>spr1</em> increased expression of 424 genes over twofold to accomplish the same development.</p></section><section id="S3.SS1.SSS3"><h4 class="pmc_sec_title">
<em>spr1</em> Differentially Expressed Defense Pathways in Its Spaceflight Response</h4>
<p>Most of the genes that are highly differentially expressed between the Col-0 and <em>spr1</em> are unique within the context of developmental age or growth environment (<a class="usa-link" href="#F5">Figure 5A</a>). However, a small number were differentially expressed between genotype irrespective of environment or age (<a class="usa-link" href="#F5">Figure 5A</a>). The coordinately expressed upregulated genes were related to the regulation of defense responses (NIMIN-1, ALD1, DLO1). The only genes that registered as coordinately downregulated in this comparison were SPIRAL1 and an antisense transcript (AT3G29644) of a transposable element which did not show differential expression itself.</p>
<figure class="fig xbox font-sm" id="F5"><h5 class="obj_head">FIGURE 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7064724_fpls-11-00239-g005.jpg" target="_blank"><img alt="FIGURE 5" class="graphic zoom-in" height="301" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/864b4a464742/fpls-11-00239-g005.jpg" width="784"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><em>Spr1</em> vs. Col-0; the effects of <em>SPR1</em> loss on gene expression. <strong>(A)</strong> Differentially expressed genes (DEGs) between the <em>spr1</em> mutant and its Col-0 wild-type background with greater than twofold magnitude in at least one context. DEG heatmaps are scaled such that yellow and blue represent more extreme levels of up- and downregulation, respectively, while red and green represent lower fold-changes in the same manner. In this case, upregulation of a gene indicates a higher expression level in <em>spr1</em> than in Col-0. <strong>(B,C)</strong> Gene ontology (GO) terms for <strong>(B)</strong> biological processes and <strong>(C)</strong> cellular compartments whose annotated genes were over- and underrepresented in the lists of DEGs for each context which met the twofold cutoff. The significance of the GO terms’ enrichment is represented by the –Log<sub>10</sub> transform of its <em>q</em>-value. The scaling scheme used for the GO term heatmaps was similar to that described for DEGs, where yellow and red denote overrepresented terms, and blue and green denote underrepresented terms. Data are based on four biological replicates.</p></figcaption></figure><p>The theme of stress response and defense was also evident in the GO enrichment analyses of the genes differentially expressed between Col-0 and <em>spr1</em> in the spaceflight environment. The significantly over- and underrepresented GO terms among biological processes are indicated in <a class="usa-link" href="#F5">Figure 5B</a>. Genes associated with general stress responses and environmental stimuli were prominent, as well as those annotated to terms related to defense responses. DEGs associated with these defense and stress pathways have been noted in the FLT vs. GC comparisons of previous studies (e.g., <a aria-describedby="B20" class="usa-link" href="#B20">Correll et al., 2013</a>; <a aria-describedby="B71" class="usa-link" href="#B71">Paul et al., 2013</a>, <a aria-describedby="B70" class="usa-link" href="#B70">2017</a>; <a aria-describedby="B19" class="usa-link" href="#B19">Choi et al., 2019</a>). Genes involved in responses to salicylic acid, a phytohormone linked to both plant defense signaling and regulation of plant development (<a aria-describedby="B44" class="usa-link" href="#B44">Klessig et al., 2018</a>), were also enriched. Carbon fixation pathways, and specifically multiple ribulose bisphosphate carboxylase (RuBisCo) genes, were also upregulated in 4d FLT <em>spr1</em> (<a class="usa-link" href="#F5">Figures 5B,C</a>). Genes encoding proteins localized to the apoplast were also enriched among those upregulated in FLT <em>spr1</em>, indicating that significant remodeling of processes within the cell was also needed by <em>spr1</em> within the spaceflight environment (<a class="usa-link" href="#F5">Figure 5C</a>).</p>
<p>Many of the DEGs associated with the acclimation to spaceflight were unique to genotype and to the developmental age of the plants (<a class="usa-link" href="#F6">Figure 6A</a>). There were 554 genes differentially expressed by at least twofold between spaceflight and ground among the two age groups (4d and 8d) of the two genotypes (Col-0 and <em>spr1</em>), with 63 genes coordinately expressed across all plants. Developmental age had a substantial impact on the spaceflight response of both genotypes; in both Col-0 and spr1, the 4d plants had a higher number of DEGs than the 8d plants. There was a large amount of overlap between the GO processes identified from the DEGs used to acclimate to the FLT condition for <em>spr1</em> and Col-0 (<a class="usa-link" href="#F6">Figure 6B</a>). Among the more widely shared GO categories in the 4d plants of both genotypes were those corresponding to the regulation of gene expression and transcriptional processes. These were more highly enriched in <em>spr1</em> roots than in Col-0, however, it can also be seen that there is a <em>spr1</em>-specific enrichment of the same terms among those genes which are downregulated in the response to FLT. The term for a response to karrikin compounds (phytohormone-like molecules derived from burning plant material) was also indicated. Many light signaling terms also appeared as enriched among DEGs downregulated in FLT, and these were involved with the response to various wavelengths of light as well as the response to elevated light intensity. However, specific photosynthetic terms appeared amongst those which were downregulated only at 8d in Col-0. Similar localization terms appeared specifically in 8d Col-0 as well (<a class="usa-link" href="#F6">Figure 6C</a>), indicating that these were changes in the expression of genes directly involved in the photosynthetic machinery. While both <em>spr1</em> and Col-0 4d were enriched with DEGs localized to the apoplast, these were among the upregulated genes in <em>spr1</em> and the downregulated genes in Col-0, alongside other terms related to extracellular structures. Furthermore, terms related to organellar localization were significantly underrepresented in those genes upregulated in Col-0 at 4d.</p>
<figure class="fig xbox font-sm" id="F6"><h5 class="obj_head">FIGURE 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 6" class="graphic" height="793" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/15172d0382e5/fpls-11-00239-g006.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>FLT vs. GC for Col-0 and <em>spr1</em>; the DEGs involved in spaceflight acclimation. <strong>(A)</strong> Differentially expressed genes (DEGs) between the GC and FLT conditions with greater than twofold magnitude in at least one context. DEG heatmaps are scaled such that yellow and blue represent more extreme levels of up- and downregulation, respectively, while red and green represent lower fold-changes in the same manner. In this case, upregulation of a gene indicates a higher expression level in FLT than the GC. <strong>(B,C)</strong> Gene ontology (GO) terms for <strong>(B)</strong> biological processes and <strong>(C)</strong> cellular compartments whose annotated genes were over- and underrepresented in the lists of DEGs for each context which met the twofold cutoff. The significance of the GO terms’ enrichment is represented by the –Log<sub>10</sub> transform of its <em>q</em>-value. The scaling scheme used for the GO term heatmaps was similar to that described for DEGs, where yellow and red denote overrepresented terms, and blue and green denote underrepresented terms. Data are based on four biological replicates.</p></figcaption></figure><p>The expression of genes associated with the developmental age was impacted by the spaceflight environment (<a class="usa-link" href="#F7">Figure 7A</a>). Approximately 26% of the total DEGs meeting the criteria for inclusion were required to be differentially expressed at some level regardless of the growth condition or genotypic background to facilitate development. However, among only those DEGs which met the twofold change cutoff, there were not GO terms shared so ubiquitously between contexts (<a class="usa-link" href="#F7">Figure 7B</a>). While broad GO terms for gene expression and metabolic processes were underrepresented in many of the contexts, more specific terms related to developmental and metabolic regulation were enriched during development in Col-0. Genes related to oxidative stress were also enriched among those genes more highly expressed at 4d in FLT in both genotypes. Localization GO term enrichments were highly divided between intracellular and extracellular compartments, which were consistently underrepresented and overrepresented, respectively (<a class="usa-link" href="#F7">Figure 7C</a>). These underrepresented GO terms occur mostly within the sets of DEGs which were upregulated between 4d and 8d, with many more significantly enriched in <em>spr1</em>. The primary exception to this was a set of terms related to the chloroplast and the RuBisCo complex, which were mostly enriched in those DEGs more highly expressed in early development of <em>spr1</em>.</p>
<figure class="fig xbox font-sm" id="F7"><h5 class="obj_head">FIGURE 7.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 7" class="graphic" height="852" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/b4a67cc458c1/fpls-11-00239-g007.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F7/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>4d vs. 8d seedlings of Col-0 and <em>spr1</em>; developmentally-associated DEGs. <strong>(A)</strong> Differentially expressed genes (DEGs) between the 4d and 8d timepoints with greater than twofold magnitude in at least one context. DEG heatmaps are scaled such that yellow and blue represent more extreme levels of up- and downregulation, respectively, while red and green represent lower fold-changes in the same manner. In this case, upregulation of a gene indicates a higher expression level at 4d than at 8d. <strong>(B,C)</strong> Gene ontology (GO) terms for <strong>(B)</strong> biological processes and <strong>(C)</strong> cellular compartments whose annotated genes were over- and underrepresented in the lists of DEGs for each context which met the twofold cutoff. The significance of the GO terms’ enrichment is represented by the –Log<sub>10</sub> transform of its <em>q</em>-value. The scaling scheme used for the GO term heatmaps was similar to that described for DEGs, where yellow and red denote overrepresented terms, and blue and green denote underrepresented terms. Data are based on four biological replicates.</p></figcaption></figure></section></section><section id="S3.SS2"><h3 class="pmc_sec_title">WS and <em>Sku5</em>
</h3>
<section id="S3.SS2.SSS1"><h4 class="pmc_sec_title">Germination and Morphology of <em>sku5</em> in Spaceflight</h4>
<p>Phenotypic differences among the WS background plants in GC and FLT environments included root length and skewing angles (<a class="usa-link" href="#F2">Figures 2</a>, <a class="usa-link" href="#F8">8</a>). The primary root lengths of FLT WS were decreased significantly compared to GC at 8d (two-tailed <em>t-</em>test, <em>p</em> = 0.000209), but not at 4d. There were no differences in root lengths between GC and FLT <em>sku5</em> plants of either 4d or 8d plants (<a class="usa-link" href="#F2">Figures 2B</a>, <a class="usa-link" href="#F8">8C,D,G,H</a>). There were differences in root length between WS and <em>sku5</em> genotypes in the GC (<em>p</em> = 0.0013) but FLT WS and FLT <em>sku5</em> roots were not statistically different (<a class="usa-link" href="#F2">Figures 2B</a>, <a class="usa-link" href="#F8">8C,D,G,H</a>).</p>
<figure class="fig xbox font-sm" id="F8"><h5 class="obj_head">FIGURE 8.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 8" class="graphic" height="828" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/c5d9c04e858d/fpls-11-00239-g008.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F8/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>High resolution images of the WS and <em>sku5</em> growth patterns. Images are arranged such that the ground control seedlings <strong>(A–D)</strong> are in the left column while spaceflight seedlings are on the right <strong>(E–H)</strong>. Vertically, images alternate between WS <strong>(A,C,E,G)</strong> and <em>sku5</em>
<strong>(B,D,F,H)</strong>. The images taken at the 4d time point <strong>(A,B,E,F)</strong> are above those taken at the 8d time point <strong>(C,D,G,H)</strong>. Images taken from above the growth medium at the time of harvest. Grid squares are 13 mm wide.</p></figcaption></figure><p>There were no statistically significant differences in the overall change from vertical between WS and <em>sku5</em> in either GC or FLT environments (<a class="usa-link" href="#F2">Figures 2C</a>, <a class="usa-link" href="#F8">8A–D</a>). The differences in root angles between GC and FLT for each genotype, and between the WS and <em>sku5</em> genotypes in both FLT and GC environments, also failed to meet statistical criteria. However, visual inspection of the images, and also the plots of <a class="usa-link" href="#F2">Figure 2C</a>, illustrate the difference in the directional trends for root growth in each genotype and environment. For both genotypes the variance in the growth angle (as noted in the SD values in <a class="usa-link" href="#F2">Figure 2</a>) was greater in the spaceflight samples.</p>
<p>The FLT <em>sku5</em> seeds germinated at a significantly reduced rate (48%) compared to WS FLT (85%), and to <em>sku5</em> GC (77%) and WS GC (93%) [χ<sup>2</sup> (3, <em>N</em> = 185) = 15.01, <em>p</em> = 0.00181] (<a class="usa-link" href="#SM1">Supplementary Table S2</a> and <a class="usa-link" href="#F8">Figures 8A,B,E,F</a>). However, the FLT <em>sku5</em> seeds that germinated produced plants that were comparable to their GC counterparts, as evidenced both by plate images and by the lack of difference in root length between FLT and GC <em>sku5</em> at 4 and 8d (<a class="usa-link" href="#F2">Figures 2B</a>, <a class="usa-link" href="#F8">8B,D,F,H</a>).</p></section><section id="S3.SS2.SSS2"><h4 class="pmc_sec_title">The Expression of More Genes Is Altered in <em>sku5</em> Than in WS During Spaceflight</h4>
<p>The trends of differential expression in WS and the <em>sku5</em> line contrasted with those seen in Col-0 and <em>spr1</em> in that WS and <em>sku5</em> show an elevated transcriptomic response to spaceflight over time, with <em>sku5</em> further differentially expressing more genes than WS (<a class="usa-link" href="#F9">Figure 9</a>). The clearest example of this was the FLT vs. GC comparison of gene expression for these lines, where the number of DEGs required for spaceflight acclimation of <em>sku5</em> were fivefold and twofold more than WS at 4d and 8d, respectively (<a class="usa-link" href="#F9">Figure 9A</a>: FLT vs. GC). However, both genotypes increase in their DEG counts between 4d and 8d. The proportion of DEGs with a greater than twofold change also decreased to a very similar degree for both WS (4d: 63.2%, 8d: 14.5%) and <em>sku5</em> (4d: 62.8%, 8d: 14.2%) despite the large difference in counts. When the genotypes were compared directly, the transcriptomic differences between them were exacerbated as they acclimated to the flight condition over time (<a class="usa-link" href="#F9">Figure 9B</a>: <em>sku5</em> vs. WS). The proportion of DEGs meeting the twofold change criteria followed a similar trend to that seen in flight acclimation, where it decreased in both GC (4d: 40.9%, 8d: 13.0%) and FLT (4d: 52.8%, 8d: 9.3%). Despite these trends, most genes among the FLT 4d DEGs that had greater than twofold change were those that were expressed more highly in <em>sku5</em>. This trend persisted into the 4d vs. 8d comparison (<a class="usa-link" href="#F9">Figure 9C</a>: 4d vs. 8d), where <em>sku5</em> in the FLT condition required more genes to be differentially expressed in order to develop. S<em>ku5</em> in the GC condition behaved much like WS in both FLT and GC conditions in terms of DEG counts in each fold-change category. Together, these patterns of gene expression indicated that the conditions of spaceflight had a disproportionate effect on the <em>sku5</em> roots’ transcriptomic responses, but that each genotype’s response to spaceflight over time involved a greater number of DEGs of lower fold-changes.</p>
<figure class="fig xbox font-sm" id="F9"><h5 class="obj_head">FIGURE 9.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7064724_fpls-11-00239-g009.jpg" target="_blank"><img alt="FIGURE 9" class="graphic zoom-in" height="491" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/f637e2337408/fpls-11-00239-g009.jpg" width="793"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F9/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>WS and <em>sku5</em> overall differential gene expression levels. A graphic representation of the genes differentially expressed in the <strong>(A)</strong> FLT vs. GC, <strong>(B)</strong>
<em>sku</em>5 vs. WS, and <strong>(C)</strong> 4d vs. 8d comparisons. Upregulation of a gene denotes a higher expression level in the first context of the comparison. Genes are separated into categories based on direction of differential expression and a Log2(Fold-Change) magnitude cut-off of one. d, day; DEG, differentially expressed gene; Downreg, downregulated; FLT, spaceflight; GC, ground control; Mag, magnitude; Upreg, upregulated. Data are based on four biological replicates.</p></figcaption></figure></section><section id="S3.SS2.SSS3"><h4 class="pmc_sec_title">
<em>Sku5</em> Highly Induced ABA- and Stress-Associated Genes in Spaceflight</h4>
<p>The genes differentially expressed between <em>sku5</em> and WS were similar between GC and FLT growth conditions (<a class="usa-link" href="#F10">Figure 10A</a>), in that the same genes that were highly upregulated in WS GC were highly upregulated in FLT for <em>sku5</em>. Among these were many late embryogenesis abundant (LEA) family proteins, whose expression has been connected to enhanced resistance to abiotic stresses and alterations in ABA sensitivity (<a aria-describedby="B126" class="usa-link" href="#B126">Zhao et al., 2011</a>; <a aria-describedby="B13" class="usa-link" href="#B13">Candat et al., 2014</a>). Additional LEA family genes and seed storage proteins, such as CRUCIFERINA (CRA1) and CRUCIFERIN 3 (CRA3) were seen as upregulated uniquely in <em>sku5</em> FLT. Alongside these were further genes associated with ABA- or ABA-independent dormancy signaling, notably REDUCED DORMANCY 5 (RDO5), DELAY OF GERMINATION 1 (DOG1), and HIGHLY ABA-INDUCED PROTEIN 2 and 3 (HAI2, HAI3). In 4d <em>sku5</em> spaceflight plants, genes encoding LEAs and other seed-associated proteins are among the most highly induced (Log<sub>2</sub>[5 to 10], or 30 to 1000-fold). Many of these patterns are further supported by GO biological process term enrichments (<a class="usa-link" href="#F10">Figure 10B</a>), where almost all of the overrepresented terms are enriched within the group of genes more highly expressed by <em>sku5</em> roots at 4d in FLT. Outside of the mentioned ABA signaling and seed development genes, terms for regulation of seed dormancy and negative regulation of post-embryonic development are overrepresented, alongside more general terms such as responses to temperature and stress. All of the significant GO localization enrichments were for genes more highly expressed in WS in the 4d FLT context, involving either DEGs related to the RuBisCo complex or extracellular structures (<a class="usa-link" href="#F10">Figure 10C</a>). DEGs encoding proteins anchored to the PM are enriched for this group, indicating an important role for them in WS.</p>
<figure class="fig xbox font-sm" id="F10"><h5 class="obj_head">FIGURE 10.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7064724_fpls-11-00239-g010.jpg" target="_blank"><img alt="FIGURE 10" class="graphic zoom-in" height="475" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/32ecea8bd75b/fpls-11-00239-g010.jpg" width="784"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F10/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><em>sku5</em> vs. WS; the effects of <em>SKU5</em> loss on gene expression. <strong>(A)</strong> Differentially expressed genes (DEGs) between the <em>sku5</em> mutant and its WS wild-type background with greater than twofold magnitude in at least one context. DEG heatmaps are scaled such that yellow and blue represent more extreme levels of up- and downregulation, respectively, while red and green represent lower fold-changes in the same manner. In this case, upregulation of a gene indicates a higher expression level in <em>sku5</em> than in WS. <strong>(B,C)</strong> Gene ontology (GO) terms for <strong>(B)</strong> biological processes and <strong>(C)</strong> cellular compartments whose annotated genes were over- and underrepresented in the lists of DEGs for each context which met the twofold cutoff. The significance of the GO terms’ enrichment is represented by the –Log<sub>10</sub> transform of its <em>q</em>-value. The scaling scheme used for the GO term heatmaps was similar to that described for DEGs, where yellow and red denote overrepresented terms, and blue and green denote underrepresented terms. Data are based on four biological replicates.</p></figcaption></figure><p>Many of the LEAs and ABA-responsive DEGs in the <em>sku5</em> vs. WS comparison were also highly upregulated by <em>sku5</em> in the FLT vs. GC comparison, and some were unique to the spaceflight acclimation of <em>sku5</em> (<a class="usa-link" href="#F10">Figures 10A</a>, <a class="usa-link" href="#F11">11A</a>). The majority of FLT vs. GC DEGs (70.3%) were only detected in one of the <em>sku5</em> FLT vs. GC comparison contexts. As such, the majority of the GO biological process terms found to be enriched were unique to the DEGs upregulated in FLT <em>sku5</em> at the 4d time point (<a class="usa-link" href="#F11">Figure 11B</a>). Hormone response and signal transduction terms were enriched, with ethylene signaling, ROS response, and light signaling among the more specific pathways to be represented. In tandem, these 4d <em>sku5</em> roots had decreased enrichment of DEGs associated with the synthesis of suberin. Uniquely to this comparison, 8d FLT WS roots upregulated genes related to chromatin remodeling. As seen in other comparisons (<a class="usa-link" href="#F6">Figures 6C</a>, <a class="usa-link" href="#F7">7C</a>, <a class="usa-link" href="#F10">10C</a>), WS upregulated genes annotated to the RuBisCo complex in FLT (<a class="usa-link" href="#F11">Figure 11C</a>). Additionally, the 8d downregulated DEGs for WS were overrepresented for extracellular localization, and DEGs whose products localized to intracellular and organellar compartments were underrepresented (<a class="usa-link" href="#F11">Figure 11C</a>). However, <em>sku5</em> showed overrepresentation of downregulated DEGs annotated to the extracellular region at the earlier time point of 4d.</p>
<figure class="fig xbox font-sm" id="F11"><h5 class="obj_head">FIGURE 11.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=7064724_fpls-11-00239-g011.jpg" target="_blank"><img alt="FIGURE 11" class="graphic zoom-in" height="722" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/2d13ebffc304/fpls-11-00239-g011.jpg" width="784"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F11/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>FLT vs. GC for WS and <em>sku5</em>; DEGs involved in spaceflight acclimation. <strong>(A)</strong> Differentially expressed genes (DEGs) between the GC and FLT conditions with greater than twofold magnitude in at least one context. DEG heatmaps are scaled such that yellow and blue represent more extreme levels of up- and downregulation, respectively, while red and green represent lower fold-changes in the same manner. In this case, upregulation of a gene indicates a higher expression level in FLT than the GC. <strong>(B,C)</strong> Gene ontology (GO) terms for b biological processes and <strong>(C)</strong> cellular compartments whose annotated genes were over- and underrepresented in the lists of DEGs for each context which met the twofold cutoff. The significance of the GO terms’ enrichment is represented by the –Log<sub>10</sub> transform of its <em>q</em>-value. The scaling scheme used for the GO term heatmaps was similar to that described for DEGs, where yellow and red denote overrepresented terms, and blue and green denote underrepresented terms. Data are based on four biological replicates.</p></figcaption></figure><p>Late embryogenesis abundant family genes were also prominently represented in the 4d vs. 8d comparison within GC and FLT environments (<a class="usa-link" href="#F12">Figure 12A</a>). Examination of the GO biological process terms yielded by the individual lists of DEGs showed many similar stress response terms, but these were primarily in the context of genes more highly expressed at the 4d time point (<a class="usa-link" href="#F12">Figure 12B</a>). However, based on the level of significance of the terms, the enrichment of DEGs related to many of these stress responses was enhanced in FLT. In the cases of oxidative stress and cold response <em>sku5</em> showed this enhancement. However, in some cases, such as the response to desiccation, both genotypes showed a lack of significant term enrichment in FLT. The <em>sku5</em> roots also showed slightly higher significance of enrichment of DEGs associated with responses to water deprivation and light stimuli in early FLT development, compared to WS. GO terms related to cell wall remodeling and immune responses, commonly seen in FLT acclimation, were also enhanced in their enrichment during FLT development of <em>sku5</em> compared to GC development. The remainder of the significant GO terms were unique to one context, and among these is the response to karrikin, which was previously seen in the context of <em>spr1</em> and Col-0 FLT vs. GC comparisons (<a class="usa-link" href="#F6">Figure 6B</a>). Of the terms enriched specifically in <em>sku5</em> FLT development, processes involved with salt and ROS stress, as well as light responses, were significantly higher at 4d. At 8d, the terms associated with <em>sku5</em> in FLT were primarily focused on cell wall remodeling and biosynthetic processes.</p>
<figure class="fig xbox font-sm" id="F12"><h5 class="obj_head">FIGURE 12.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 12" class="graphic" height="854" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/9f1f4ae1bcc1/fpls-11-00239-g012.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F12/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>4d vs. 8d seedlings of WS and <em>sku5</em>; developmentally-associated DEGs. <strong>(A)</strong> Differentially expressed genes (DEGs) between the 4d and 8d timepoints with greater than twofold magnitude in at least one context. DEG heatmaps are scaled such that yellow and blue represent more extreme levels of up- and downregulation, respectively, while red and green represent lower fold-changes in the same manner. In this case, upregulation of a gene indicates a higher expression level at 4d than at 8d. <strong>(B,C)</strong> Gene ontology (GO) terms for <strong>(B)</strong> biological processes and <strong>(C)</strong> cellular compartments whose annotated genes were over- and underrepresented in the lists of DEGs for each context which met the twofold cutoff. The significance of the GO terms’ enrichment is represented by the –Log<sub>10</sub> transform of its <em>q</em>-value. The scaling scheme used for the GO term heatmaps was similar to that described for DEGs, where yellow and red denote overrepresented terms, and blue and green denote underrepresented terms. Data are based on four biological replicates.</p></figcaption></figure></section></section></section><section id="S4"><h2 class="pmc_sec_title">Discussion</h2>
<p>The discovery that root skewing was not gravity-dependent, and prevalent in both light-grown and dark-grown plants in spaceflight habitats (<a aria-describedby="B59" class="usa-link" href="#B59">Millar et al., 2011</a>; <a aria-describedby="B67" class="usa-link" href="#B67">Paul et al., 2012a</a>; <a aria-describedby="B63" class="usa-link" href="#B63">Nakashima et al., 2014</a>), invited questions about the underlying mechanisms of this behavior in the microgravity environment and the relationship between those mechanisms and physiological adaptation to spaceflight. The experiments described here examined the contribution of two well-characterized skewing genes that are in two different ecotypic backgrounds and which affect different skewing control pathways. SPIRAL1 plays a role in directional cell expansion by regulating cortical microtubule dynamics (<a aria-describedby="B31" class="usa-link" href="#B31">Furutani et al., 2000</a>; <a aria-describedby="B62" class="usa-link" href="#B62">Nakajima et al., 2004</a>; <a aria-describedby="B89" class="usa-link" href="#B89">Sedbrook et al., 2004</a>; <a aria-describedby="B32" class="usa-link" href="#B32">Galva et al., 2014</a>). SKU5 is a skewing-related glycosylphosphatidylinositol-anchored cell wall and plasma membrane protein, which has also been implicated in stress response signaling (<a aria-describedby="B88" class="usa-link" href="#B88">Sedbrook et al., 2002</a>). The <em>sku5</em> and <em>spr1</em> mutants are well characterized in terrestrial environments, skewing rightward and leftward, respectively, when grown vertically (<a aria-describedby="B31" class="usa-link" href="#B31">Furutani et al., 2000</a>; <a aria-describedby="B88" class="usa-link" href="#B88">Sedbrook et al., 2002</a>, <a aria-describedby="B89" class="usa-link" href="#B89">2004</a>; <a aria-describedby="B62" class="usa-link" href="#B62">Nakajima et al., 2004</a>). The experimental readouts used to assess the physiological adjustment of seedlings to spaceflight were root and germination measurements, and the differences in transcriptomic responses among the genotypes. The opposing directions of the skewing phenotypes of the mutant lines are believed to stem from alterations of distinct cellular processes, and this is most clearly evidenced by the differential effects of microtubule-destabilizing agents on these mutants (<a aria-describedby="B31" class="usa-link" href="#B31">Furutani et al., 2000</a>; <a aria-describedby="B88" class="usa-link" href="#B88">Sedbrook et al., 2002</a>, <a aria-describedby="B89" class="usa-link" href="#B89">2004</a>; <a aria-describedby="B62" class="usa-link" href="#B62">Nakajima et al., 2004</a>; <a aria-describedby="B32" class="usa-link" href="#B32">Galva et al., 2014</a>). The distinct roles of these two genes in determining root directionality is supported by their different individual responses to spaceflight in the present experiments. The data from <em>sku5</em> and <em>spiral1</em> are analyzed in context with their parent genotype, WS and Col-0 respectively, as WS shows a much smaller number of spaceflight differentially expressed genes than Col-0 (<a aria-describedby="B70" class="usa-link" href="#B70">Paul et al., 2017</a>, and <a class="usa-link" href="#F4">Figures 4</a>, <a class="usa-link" href="#F12">12</a>).</p>
<section id="S4.SS1"><h3 class="pmc_sec_title">
<em>Spr1</em> Skews Leftward on Earth and Moreso in Space</h3>
<p>Terrestrial studies concluded that SPR1 reinforces rightward root growth by affecting the balance of thigmotropism and gravitropism (<a aria-describedby="B32" class="usa-link" href="#B32">Galva et al., 2014</a>). Morphological data from APEX-03-2 support this notion. At the 4d time point, <em>spr1</em> seedlings showed a consistent leftward skewing phenotype on Earth, and the leftward skewing was substantially enhanced by spaceflight (<a class="usa-link" href="#F2">Figures 2C</a>, <a class="usa-link" href="#F3">3B,F</a>). However, <em>spr1</em> and Col-0 roots were the same length in spaceflight, indicating that the role of SPR1 lies in directionality rather than overall root length in spaceflight (<a class="usa-link" href="#F2">Figure 2A</a>). These data suggest that in the absence of gravitropism during spaceflight, thigmotropism plays a dominant role in growth directionality and that SPR1 acts to positively enhance skewing while not affecting root length in spaceflight. The spaceflight effect of <em>spr1</em> lies in morphology management rather than growth management.</p>
<p>SPR1 binds with another microtubule plus-end tracking protein, EB1b (<a aria-describedby="B32" class="usa-link" href="#B32">Galva et al., 2014</a>). It has been postulated that EB1b and SPR1 work in concert to regulate mechanical force-based directional growth with a balance of gravitropism and thigmotropism, where SPR1 participates in the reinforcement of rightward growth through thigmotropic processes, while EB1b negatively regulates thigmotropism and positively regulates gravitropism (<a aria-describedby="B32" class="usa-link" href="#B32">Galva et al., 2014</a>). In an environment lacking a gravity cue such as spaceflight, this gravitropic reinforcement has no effect, resulting in the enhanced skewing of <em>spr1.</em> Similarly elevated leftward skewing of <em>spr1</em> occurs under cold conditions alongside alterations of the microtubule network, and skewing is conversely suppressed by heat or salt treatments (<a aria-describedby="B31" class="usa-link" href="#B31">Furutani et al., 2000</a>; <a aria-describedby="B89" class="usa-link" href="#B89">Sedbrook et al., 2004</a>). The elevated skewing seen in <em>spr1</em> may therefore be the result of spaceflight-induced signals in the root which are similar to those involved in cold response signaling. The microtubule-associated protein MAP65-1 is more abundant in spaceflight, and is also hypothesized to be linked to cold-responsive microtubule stabilization (<a aria-describedby="B18" class="usa-link" href="#B18">Chen et al., 2016</a>; <a aria-describedby="B101" class="usa-link" href="#B101">Soga et al., 2018</a>). Actin cytoskeleton mutants also exhibit a greater degree of skewing in microgravity when compared to wild-type and GC plants, though this skewing occurs to the right when regarded from behind the media (<a aria-describedby="B63" class="usa-link" href="#B63">Nakashima et al., 2014</a>). These structural systems both appear to have roles in suppressing endogenous root skewing patterns in spaceflight. The spaceflight-enhanced skewing of <em>spr1</em> may therefore be attributed to its altered microtubule dynamics in the spaceflight response, similar to altered dynamics induced by terrestrial cold stress.</p></section><section id="S4.SS2"><h3 class="pmc_sec_title">
<em>Sku5</em> Skews Inappropriately in Spaceflight</h3>
<p>The <em>sku5</em> mutant did not show an enhancement of skewing in spaceflight as compared to the GC, nor did <em>sku5</em> respond to the spaceflight environment in the same manner as it does to terrestrial disruption of the gravity vector (<a class="usa-link" href="#F2">Figure 2C</a>) (<a aria-describedby="B88" class="usa-link" href="#B88">Sedbrook et al., 2002</a>). However, <em>sku5</em> and WS roots were the same length in spaceflight, indicating that the role of SKU5 also lies in directionality rather than overall root length in spaceflight (<a class="usa-link" href="#F2">Figure 2B</a>). These data suggest that SKU5, like SPR1, regulates morphology and directionality rather than growth. However, SKU5 regulation is very different from SPR1 regulation of skewing morphology in that lack of SKU5 function prevents appropriate skewing in spaceflight. This prevention of skewing, the <em>sku5</em> interruption of the pathways involved in skewing, leads to dramatic changes in differential expression of genes for physiological adaptation to spaceflight.</p></section><section id="S4.SS3"><h3 class="pmc_sec_title">Differential Gene Expression as a Measure of the Quantity and Character of the Physiological Adaptation to Spaceflight</h3>
<p>Differential gene expression profiles reveal the underlying gene expression changes needed for physiological adaptation to environmental changes. There are numerous, well-documented environmentally-induced gene expression profiles in plants, all of which detail specific physiological changes related to the specific stresses. The intensity of the differential gene expression response, in terms of the number of genes involved and their fold-change levels, may be considered a measure of the metabolic cost of adapting to that environment. A terrestrial example of this metabolic cost is in the transcriptomic response to cold stress in Arabidopsis lines that are either sensitive or resistant to cold (<a aria-describedby="B16" class="usa-link" href="#B16">Chan et al., 2016</a>). Plants that overexpressed RNA-DIRECTED DNA METHYLATION 4 (RDM4) had a cold-tolerant phenotype, and exhibited a greatly reduced transcriptomic response to cold stress compared to the more cold-sensitive wild-type and <em>rdm4</em> knock-out plants (<a aria-describedby="B16" class="usa-link" href="#B16">Chan et al., 2016</a>). The plants that were least able to tolerate cold environments displayed a far higher number (up to 100×) of differentially expressed genes in response to cold than the cold-tolerant plants exposed to the same environment (<a aria-describedby="B16" class="usa-link" href="#B16">Chan et al., 2016</a>).</p>
<p><em>Spr1</em> adapts to spaceflight with far fewer DEGs than Col-0. This suggests that mutation of the SPR1 skewing pathway enhances spaceflight physiological adaptation. In contrast, the <em>sku5</em> mutation results in a doubling of DEGs in response to spaceflight, and many of those DEGs are members of strong environmental stress responses. This strong response suggests that the SKU5 pathway plays a key role in spaceflight physiological adaptation.</p>
<section id="S4.SS3.SSS1"><h4 class="pmc_sec_title">
<em>Sku5</em> Mutation Produces a Dramatic Increase in Differential Gene Expression in Spaceflight</h4>
<p><em>Sku5</em> showed a dramatic program of differential gene expression compared to the GCs, a program that was more pronounced and diverse than that of <em>spr1</em>. Many of the genes that are significantly differentially expressed between spaceflight and GCs are associated with skewing and cell wall remodeling, irrespective of intensity of skewing phenotype in the spaceflight-grown plants (<a class="usa-link" href="#F13">Figure 13</a>).</p>
<figure class="fig xbox font-sm" id="F13"><h5 class="obj_head">FIGURE 13.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 13" class="graphic" height="533" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a6/7064724/d3270a6d97ad/fpls-11-00239-g013.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F13/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Differentially expressed genes between FLT and GC of particular interest. Statistically significant DEGs with greater than twofold magnitude in at least one context. Each gene in this list corresponded to the following categories: potential and highly probable skew gene candidates (Skewing, HPSGC; <a aria-describedby="B87" class="usa-link" href="#B87">Schultz et al., 2017</a>), late embryogenesis abundant (LEA; <a aria-describedby="B38" class="usa-link" href="#B38">Hundertmark and Hincha, 2008</a>), associated with gravitropism (GO Terms; <a aria-describedby="B14" class="usa-link" href="#B14">Carbon et al., 2008</a>), and abscisic acid (ABA, GO Terms; <a aria-describedby="B14" class="usa-link" href="#B14">Carbon et al., 2008</a>). The heat map scale Log<sub>2</sub>(Fold-Change) is indicated in the bottom right corner. Categorical information is provided beside each gene group. Data are based on four biological replicates.</p></figcaption></figure><p>One of the unique aspects of the <em>sku5</em> spaceflight differential gene expression program involved genes usually associated with dehydration and ABA signaling. The FLT <em>sku5</em> expression patterns of LEAs and other ABA-associated genes suggest a spaceflight adaptation phenomenon involving, but not limited to, an altered ABA sensitivity in these plants. Many of the highest magnitude DEGs in the spaceflight acclimation of <em>sku5</em> were in the LEA family, primarily dehydrins and group-4 LEAs; many of these genes were induced by greater than 30-fold [Log<sub>2</sub>(5)] (<a class="usa-link" href="#F13">Figure 13</a>). LEAs are responsive to ABA signaling resulting from osmotic, cold, and drought stresses and act to enhance stress resistance through membrane stabilization, sequestration of ROS, and prevention of protein aggregation (<a aria-describedby="B38" class="usa-link" href="#B38">Hundertmark and Hincha, 2008</a>; <a aria-describedby="B126" class="usa-link" href="#B126">Zhao et al., 2011</a>; <a aria-describedby="B13" class="usa-link" href="#B13">Candat et al., 2014</a>; <a aria-describedby="B26" class="usa-link" href="#B26">Fang and Xiong, 2015</a>; <a aria-describedby="B104" class="usa-link" href="#B104">Suzuki et al., 2016</a>).</p>
<p>Many of the DEGs and LEAs upregulated in the spaceflight acclimation of <em>sku5</em> are associated with embryonic programs that are adjusted at the chromatin level during germination in response to environmental factors (<a aria-describedby="B106" class="usa-link" href="#B106">Tai et al., 2005</a>; <a aria-describedby="B111" class="usa-link" href="#B111">Tang et al., 2012</a>; <a aria-describedby="B61" class="usa-link" href="#B61">Molitor et al., 2014</a>). DOG1, for example, increases in expression in cold temperatures, acting to repress germination during those unfavorable conditions (<a aria-describedby="B30" class="usa-link" href="#B30">Footitt et al., 2013</a>, <a aria-describedby="B29" class="usa-link" href="#B29">2014</a>). The <em>sku5</em> DEGs HAI2 and HAI3 are involved in regulating ABA signaling and responding to soil water potential (<a aria-describedby="B6" class="usa-link" href="#B6">Bhaskara et al., 2012</a>; <a aria-describedby="B43" class="usa-link" href="#B43">Kim et al., 2013</a>). REDUCED DORMANCY 5 also acts to inhibit germination, but functions independently of DOG1 and ABA signaling pathways (<a aria-describedby="B119" class="usa-link" href="#B119">Xiang et al., 2014</a>). The GO term enrichments for <em>sku5</em> also support this, as DEGs associated with dormancy regulation are overrepresented in the dataset (<a class="usa-link" href="#F10">Figures 10B</a>, <a class="usa-link" href="#F12">12B</a>). It is important to note that SKU5 is expressed in floral tissue (<a aria-describedby="B88" class="usa-link" href="#B88">Sedbrook et al., 2002</a>). This altered ABA signaling in <em>sku5</em> could therefore be attributed to downstream effects of altered seed development. In support of these interpretations, germination rates were repressed in FLT <em>sku5</em> (<a class="usa-link" href="#F8">Figures 8F,H</a>) relative to GC <em>sku5</em> (<a class="usa-link" href="#F8">Figures 8B,D</a>), and to FLT WS (<a class="usa-link" href="#F8">Figures 8E,G</a>), as evidenced by counts of ungerminated seeds in the harvest images (<a class="usa-link" href="#SM1">Supplementary Table S2</a>). While <em>spr1</em> is also expressed in reproductive tissues, the differential expression of dormancy signaling genes was not prevalent in the Col-0 and <em>spr1</em> comparisons (<a aria-describedby="B62" class="usa-link" href="#B62">Nakajima et al., 2004</a>; <a aria-describedby="B89" class="usa-link" href="#B89">Sedbrook et al., 2004</a>). In fact, those few DEGs related to ABA signaling observed in Col-0 and <em>spr1</em> displayed a trend in expression that was opposite of what was seen in FLT <em>sku5</em>. As a prime example, the LEA ABA-RESPONSE PROTEIN (ABR) was only slightly downregulated in the spaceflight acclimation of 8d <em>spr1</em>. ABA-related dormancy signaling may be affected by the spaceflight environment.</p>
<p>Differential expression patterns also indicate that <em>sku5</em> mutants experience changes in the plasma membrane during spaceflight. Alterations in plasma membrane dynamics and composition occur in plant responses to terrestrial stressors (e.g., cold stress) and in the response of <em>Chlorella vulgaris</em> to spaceflight (<a aria-describedby="B72" class="usa-link" href="#B72">Popova et al., 1989</a>; <a aria-describedby="B102" class="usa-link" href="#B102">Solanke and Sharma, 2008</a>; <a aria-describedby="B41" class="usa-link" href="#B41">Kazemi-Shahandashti and Maali-Amiri, 2018</a>). Roots of <em>Pisum sativum</em> grown under clinorotation also show shifts in membrane composition in sterol-enriched membrane domains known as “lipid rafts” (<a aria-describedby="B47" class="usa-link" href="#B47">Kordyum et al., 2018</a>). Membrane-bound GPI-APs, such as SKU5, localize to these lipid raft nanodomains and are important to stress acclimation, including spaceflight-associated stress (<a aria-describedby="B60" class="usa-link" href="#B60">Minami et al., 2008</a>; <a aria-describedby="B55" class="usa-link" href="#B55">Mazars et al., 2014</a>; <a aria-describedby="B27" class="usa-link" href="#B27">Ferl et al., 2015</a>; <a aria-describedby="B109" class="usa-link" href="#B109">Takahashi et al., 2016</a>, <a aria-describedby="B107" class="usa-link" href="#B107">2019</a>; <a aria-describedby="B123" class="usa-link" href="#B123">Yeats et al., 2018</a>). ABA signaling proteins also localize to these membrane domains in a sterol-dependent manner (<a aria-describedby="B22" class="usa-link" href="#B22">Demir et al., 2013</a>). Given that <em>sku5</em> shows alterations of ABA signaling pathways in spaceflight (<a class="usa-link" href="#F13">Figure 13</a>), it can be hypothesized that the lack of SKU5 may affect the function of these nanodomain-localized ABA signaling processes under stress conditions. The expression of a chloroplastic aldehyde reductase, ChlADR (AT1G54870), involved in the detoxification of reactive carbonyls formed as a result of lipid peroxidation, and TSPO, which participates in mobilization of lipids, are induced by stress and highly induced in the spaceflight acclimation of <em>sku5</em> (<a aria-describedby="B35" class="usa-link" href="#B35">Guillaumot et al., 2009</a>; <a aria-describedby="B121" class="usa-link" href="#B121">Yamauchi et al., 2011</a>; <a aria-describedby="B40" class="usa-link" href="#B40">Jurkiewicz et al., 2018</a>). These patterns match the gene expression trends of the spaceflight-induced LEA family genes that are thought to directly participate in membrane stabilization under stress (<a aria-describedby="B3" class="usa-link" href="#B3">Battaglia et al., 2008</a>; <a aria-describedby="B38" class="usa-link" href="#B38">Hundertmark and Hincha, 2008</a>; <a aria-describedby="B13" class="usa-link" href="#B13">Candat et al., 2014</a>). The gene expression patterns of these ABA-responsive genes in <em>sku5</em> suggest that SKU5 affects ABA signaling in response to stresses. Furthermore, this shows that pathways affecting membrane stabilization and remodeling are required for the spaceflight acclimation of <em>sku5</em> that are not essential for its growth in terrestrial environments.</p>
<p>SKU5 may also directly participate in stress response signaling or cellular remodeling due to its localization to the outer aspects of the plasma membrane. A SKU5 homolog is highly upregulated [Log<sub>2</sub>(14)] in tobacco overexpressing a FATTY ACID DESATURASE 3 (CbFAD3-OE) homolog derived from the cryophyte <em>Chorispora bungeana</em> when compared to a wild-type line in the response to salt stress (<a aria-describedby="B92" class="usa-link" href="#B92">Shi et al., 2018</a>). CbFAD3-OE lines exhibited enhanced membrane fluidity and survival under cold, salt, and drought stresses, and demonstrated reduced lipid peroxidation and membrane leakage via activation of calcium signaling and suppression of ROS (<a aria-describedby="B92" class="usa-link" href="#B92">Shi et al., 2018</a>). As such, the <em>sku5</em> line may require alternative pathways to be activated to maintain membrane fluidity and integrity during spaceflight acclimation due to the removal of SKU5 from this response pathway. The transcriptomic response of <em>sku5</em> to the spaceflight condition was elevated compared to WS, which would support the activation of additional pathways (<a class="usa-link" href="#F10">Figures 10A</a>, <a class="usa-link" href="#F11">11A</a>). ROS-related genes are abundant in spaceflight transcriptomes, and reflect a response to spaceflight that is conserved among Arabidopsis cultivars and related species (<a aria-describedby="B71" class="usa-link" href="#B71">Paul et al., 2013</a>, <a aria-describedby="B70" class="usa-link" href="#B70">2017</a>; <a aria-describedby="B103" class="usa-link" href="#B103">Sugimoto et al., 2014</a>; <a aria-describedby="B27" class="usa-link" href="#B27">Ferl et al., 2015</a>; <a aria-describedby="B100" class="usa-link" href="#B100">Sng et al., 2018</a>; <a aria-describedby="B19" class="usa-link" href="#B19">Choi et al., 2019</a>; <a aria-describedby="B127" class="usa-link" href="#B127">Zhou et al., 2019</a>). Our findings were consistent with this trend across genotypes and the skewing mutants based on GO term enrichments associated with the spaceflight response (<a class="usa-link" href="#F6">Figures 6B</a>, <a class="usa-link" href="#F11">11B</a>). Notably, <em>sku5</em> FLT showed elevated expression of genes required to be expressed in response to ROS at the 4d when compared to FLT WS (<a class="usa-link" href="#F9">Figures 9B</a>, <a class="usa-link" href="#F10">10B</a>), supporting this hypothesized function of SKU5 in this membrane-associated ROS suppression. While FAD3 was not among the <em>sku5</em> FLT DEGs, FLORAL TRANSITION AT THE MERISTEM1 (FTM1), which catalyzes the desaturation of stearic acid to oleic acid upstream of FAD3, was upregulated in WS and <em>spr1</em> spaceflight acclimation, and downregulated in that of Col-0 and <em>sku5.</em> FTM1 participates alongside FAD3 in the mediation of stresses associated with Arabidopsis crown gall development through their enrichment with C18:3 fatty acids (<a aria-describedby="B45" class="usa-link" href="#B45">Klinkenberg et al., 2014</a>). The increase of C18:3 class of fatty acids also correlated with the enhanced membrane stability measures in the CbFAD3-OE lines (<a aria-describedby="B92" class="usa-link" href="#B92">Shi et al., 2018</a>). These results therefore suggest a role for SKU5 in stress metabolism with respect to the plasma membrane, a role that was not revealed in previous ground experiments.</p>
<p>An additional explanation for the altered spaceflight response of <em>sku5</em> comes from its likely association with auxin metabolism and signaling upstream of the plant TOR complex. A SKU5 homolog in maize encodes an interactor of AUXIN-BINDNG PROTEIN 1 (ABP1) (<a aria-describedby="B93" class="usa-link" href="#B93">Shimomura, 2006</a>). The <em>abp1</em> mutant demonstrates a leftward-skewing phenotype opposite to that seen in <em>sku5</em> (<a aria-describedby="B88" class="usa-link" href="#B88">Sedbrook et al., 2002</a>; <a aria-describedby="B34" class="usa-link" href="#B34">Gao et al., 2015</a>). Auxin gradients are still present in spaceflight root tips despite the lack of a gravity cue in the spaceflight environment (<a aria-describedby="B28" class="usa-link" href="#B28">Ferl and Paul, 2016</a>). The presence of SKU5 acting to mediate anisotropic cell growth in response to endogenous auxin gradients could facilitate the spaceflight acclimation of wild-type seedlings. GO terms associated with the DEGs of 4d spaceflight response of <em>sku5</em> suggested this connection as well, with many alterations occurring in signaling pathways related to ethylene, ROS, and light signaling (<a class="usa-link" href="#F11">Figure 11B</a>). Furthermore, ABP1 is associated with the activation of TOR and the promotion of growth through the action of an auxin-activated signaling pathway mediated by the RHO OF PLANTS 2 (ROP2) GTPase (<a aria-describedby="B120" class="usa-link" href="#B120">Xu et al., 2014</a>; <a aria-describedby="B84" class="usa-link" href="#B84">Schepetilnikov et al., 2017</a>; <a aria-describedby="B80" class="usa-link" href="#B80">Ryabova et al., 2019</a>). The TOR complex integrates diverse environmental signals to regulate growth, and ABA signaling inhibits TOR, promoting autophagy to allow for stress tolerance acquisition (<a aria-describedby="B80" class="usa-link" href="#B80">Ryabova et al., 2019</a>; <a aria-describedby="B95" class="usa-link" href="#B95">Signorelli et al., 2019</a>). SKU5 may impact this mechanism indirectly through affecting the action of ABP1, and a reduction of the auxin signaling into TOR would shift the balance in favor of stronger ABA responses, such as that seen in the spaceflight acclimation of <em>sku5</em>. A ROP6 pathway also mediated by ABP1 affects the activity of KATANIN, which severs cortical microtubules to facilitate reorganization of the microtubule network in response to environmental stimuli (<a aria-describedby="B18" class="usa-link" href="#B18">Chen et al., 2016</a>). As a result, the <em>spr1</em> mutant may be faster to form a spaceflight-tolerant microtubule network due to its altered microtubule kinetics, while the <em>sku5</em> mutant may not be able to perform this function as rapidly due to altered ABP1 signaling.</p></section><section id="S4.SS3.SSS2"><h4 class="pmc_sec_title">
<em>Spr1</em> Mutation Produces a Decrease in Differential Gene Expression in Spaceflight</h4>
<p>The spaceflight DEGs of <em>spr1</em> and Col-0 were primarily in the defense and salicylic acid-mediated signaling related to cell wall remodeling (<a class="usa-link" href="#F5">Figures 5B</a>, <a class="usa-link" href="#F6">6B</a>). Genes related to these cell wall pathways are common in spaceflight transcriptomes (<a aria-describedby="B68" class="usa-link" href="#B68">Paul et al., 2012b</a>, <a aria-describedby="B71" class="usa-link" href="#B71">2013</a>, <a aria-describedby="B70" class="usa-link" href="#B70">2017</a>; <a aria-describedby="B20" class="usa-link" href="#B20">Correll et al., 2013</a>; <a aria-describedby="B49" class="usa-link" href="#B49">Kwon et al., 2015</a>; <a aria-describedby="B39" class="usa-link" href="#B39">Johnson et al., 2017</a>). While plants grown in microgravity may be more susceptible to pathogenic infection (<a aria-describedby="B51" class="usa-link" href="#B51">Leach et al., 2001</a>; <a aria-describedby="B81" class="usa-link" href="#B81">Ryba-White et al., 2001</a>), the fact that all examples to date of upregulated pathogen response genes are from plants grown in sterile conditions suggests that these genes are serving other purposes of cell wall remodeling in spaceflight (<a aria-describedby="B68" class="usa-link" href="#B68">Paul et al., 2012b</a>, <a aria-describedby="B71" class="usa-link" href="#B71">2013</a>, <a aria-describedby="B70" class="usa-link" href="#B70">2017</a>; <a aria-describedby="B20" class="usa-link" href="#B20">Correll et al., 2013</a>; <a aria-describedby="B19" class="usa-link" href="#B19">Choi et al., 2019</a>). ALD1 is a positive regulator of salicylic acid accumulation pathways through the generation of metabolites (<a aria-describedby="B65" class="usa-link" href="#B65">Nie et al., 2011</a>; <a aria-describedby="B15" class="usa-link" href="#B15">Cecchini et al., 2015</a>). NIMIN1 and DLO1 contribute to opposing processes, acting to repress immune responses at the level of expression regulation (<a aria-describedby="B46" class="usa-link" href="#B46">Kohler et al., 2002</a>; <a aria-describedby="B125" class="usa-link" href="#B125">Zhang et al., 2013</a>; <a aria-describedby="B124" class="usa-link" href="#B124">Zeilmaker et al., 2015</a>). Salicylic acid induces the lipid raft-localized pathways induced by ABA, in order to promote cellular uptake of water (<a aria-describedby="B22" class="usa-link" href="#B22">Demir et al., 2013</a>; <a aria-describedby="B73" class="usa-link" href="#B73">Prodhan et al., 2018</a>). The upregulation of salicylic acid signaling in <em>spr1</em> may act through these pathways, promoting abiotic stress response through pathways normally associated with defense responses (<a class="usa-link" href="#F5">Figures 5B</a>, <a class="usa-link" href="#F6">6B</a>). This hypothesis might also explain why the induction of “defense” and wound response pathways occurs in spaceflight in the absence of pathogenic infections (<a aria-describedby="B64" class="usa-link" href="#B64">Nejat and Mantri, 2017</a>). The recently characterized ORBITALLY MANIFESTED GENE 1, a regulator of ROS signaling that is both flg22- and wound-inducible, provides a spaceflight-relevant example (<a aria-describedby="B100" class="usa-link" href="#B100">Sng et al., 2018</a>). Considering that wounding and pathogen invasion also mechanically disrupt membranes and cell walls (<a aria-describedby="B18" class="usa-link" href="#B18">Chen et al., 2016</a>), these pathways may be signaling remodeling of cellular structures and the microtubule network in spaceflight. Flg22 application to Arabidopsis cell cultures and induction of MAMP-triggered immunity also has the effect of redistributing flavonoid compounds from biosynthetic processes important to oxidative stress response processes to the production of compounds important to defense (<a aria-describedby="B83" class="usa-link" href="#B83">Schenke et al., 2019</a>). Genes associated with the production of phenylpropanoids are less-represented among the <em>spr1</em> DEGs downregulated in spaceflight compared to Col-0, although they are also more highly represented among <em>spr1</em> DEGs downregulated across development in spaceflight (<a class="usa-link" href="#F6">Figures 6B</a>, <a class="usa-link" href="#F7">7B</a>). As such, these DEGs negatively regulating defense signaling in <em>spr1</em> may contribute to an enhanced ability to mediate ROS generated in spaceflight through the redistribution of limited resources early in its development. This conclusion is also supported by the gene categories enriched in the spaceflight acclimations of Col-0 and <em>spr1</em>, where DEGs associated with responses to oxidative stress are more enriched among genes upregulated in FLT in <em>spr1</em> than in Col-0 (<a class="usa-link" href="#F6">Figure 6B</a>).</p>
<p>Many abiotic stresses trigger remodeling of cortical microtubules, with salt stress tolerance requiring the 26S proteasome-mediated degradation of SPR1 (<a aria-describedby="B94" class="usa-link" href="#B94">Shoji et al., 2006</a>; <a aria-describedby="B117" class="usa-link" href="#B117">Wang et al., 2011</a>; <a aria-describedby="B18" class="usa-link" href="#B18">Chen et al., 2016</a>). Microtubule reorientation also contributes to cell elongation in the spaceflight response of hypocotyls (<a aria-describedby="B101" class="usa-link" href="#B101">Soga et al., 2018</a>). The <em>spr1</em> mutant, which has altered microtubule dynamics in favor of decreased stability, may have enhanced capabilities for remodeling of microtubule networks in the response to many forms of environmental stress (<a aria-describedby="B32" class="usa-link" href="#B32">Galva et al., 2014</a>).</p></section><section id="S4.SS3.SSS3"><h4 class="pmc_sec_title">
<em>Spr1</em> and <em>Sku5</em> Mutants Both Differentially Regulate Skewing-Associated Genes in Response to Spaceflight</h4>
<p>Despite the many differing spaceflight responses of <em>spr1</em> and <em>sku5</em>, DEGs identified as likely to be associated with the integrative signaling of skewing (<a aria-describedby="B87" class="usa-link" href="#B87">Schultz et al., 2017</a>) were among those differentially expressed in both mutants in spaceflight (<a class="usa-link" href="#F13">Figure 13</a>). Two of the highly likely skewing genes, SENESCENCE 1 (SEN1) and ASPARAGINE SYNTHETASE 1 (ASN1) are induced in spaceflight (<a class="usa-link" href="#F13">Figure 13</a>). The SKU5 protein is endocytosed through different pathways within the root tip dependent on local auxin signals and perceived stress conditions (<a aria-describedby="B2" class="usa-link" href="#B2">Baral et al., 2015</a>). SKU5 could act to modulate growth regulation through its aforementioned association with the ABP1 apoplastic signaling pathway upstream of TOR, and both SKU5 and SPR1 can affect microtubule reorganizing pathways. This would enable SKU5 to impact the spaceflight response via the regulation of meristematic competence and cell cycle progression, processes that are affected in spaceflight (<a aria-describedby="B54" class="usa-link" href="#B54">Matía et al., 2010</a>), clinorotation, RPM exposure, and magnetic levitation (<a aria-describedby="B36" class="usa-link" href="#B36">Herranz and Medina, 2014</a>; <a aria-describedby="B37" class="usa-link" href="#B37">Herranz et al., 2014</a>; <a aria-describedby="B10" class="usa-link" href="#B10">Boucheron-Dubuisson et al., 2016</a>). This is supported by the M-phase peak of SKU5 expression and its previously hypothesized role in regulating the cell cycle (<a aria-describedby="B56" class="usa-link" href="#B56">Menges et al., 2002</a>; <a aria-describedby="B88" class="usa-link" href="#B88">Sedbrook et al., 2002</a>). However, the importance of senescence in the context of root skewing has not been well-studied. The recent association of strigolactone-independent karrikin-sensing pathways with skewing provides a potential direction for study (<a aria-describedby="B105" class="usa-link" href="#B105">Swarbreck et al., 2019</a>). The loss of the protein MORE AXILLARY BRANCHES 2 (MAX2), which also participates in strigolactone-associated senescence signaling, exacerbates the skewing phenotype of <em>sku5</em> in a <em>sku5 max2</em> line (<a aria-describedby="B115" class="usa-link" href="#B115">Ueda and Kusaba, 2015</a>; <a aria-describedby="B105" class="usa-link" href="#B105">Swarbreck et al., 2019</a>). Therefore, these results reinforce connections between environmental and senescence signaling and root skewing. The association of skewing-associated genes with these regulatory networks provide a means by which the mutation of these genes can alter the response to the spaceflight condition.</p></section></section></section><section id="S5"><h2 class="pmc_sec_title">Conclusion</h2>
<p>The physiological adaptation of <em>sku5</em> to spaceflight is characterized by powerful stress responses, as well as the strong induction of LEAs and other genes associated with seed development. In contrast to <em>sku5</em>, <em>spr1</em> differentially expressed fewer genes than Col-0 to physiologically adapt to spaceflight. This observation suggests that the <em>spr1</em> mutation imbued an enhanced ability for rapid spaceflight acclimation while <em>sku5</em> required an extended spaceflight response to accomplish acclimation, even to the point of initiating classic deep stress responses. These observations further suggest that the functional distinctions between <em>sku5</em> and <em>spr1</em> inform distinct aspects of the spaceflight response in plants, in particular the responses of young seedlings. The dramatic responses of <em>sku5</em> suggest that lipid raft nanodomains of the plasma membrane and their associated GPI-APs play important roles in spaceflight physiological adaptation.</p>
<p>The fact that <em>sku5</em> and <em>spr1</em> show markedly different gene expression patterns in spaceflight suggests that the two skewing pathways highlighted by SKU5 and SPR1 differentially affect the mechanisms used by plants to physiologically acclimate to spaceflight. Since <em>spr1</em> and <em>sku5</em> root lengths are not altered by spaceflight, the effects of these mutations lie not in the process of growth, but in the processes that direct morphology and structure, likely tying together the large number of cell wall remodeling genes observed as differentially expressed in spaceflight. The data from APEX-03-2 therefore provide an initial functional dissection of the spaceflight response in roots, one that allows an integration of pathways associated with root directionality, morphology and physiological responses to spaceflight.</p></section><section id="S6"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The datasets generated and analyzed, composed of the RNA-Seq data and full output of the differential gene expression pipeline, for this study can be found in the Gene Expression Omnibus (GEO – <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/geo/" rel="noopener noreferrer" target="_blank">https://www.ncbi.nlm.nih.gov/geo/</a>) under the accessions <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95620" rel="noopener noreferrer" target="_blank">GSE95620</a> (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?&amp;acc=GSE95620" rel="noopener noreferrer" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?&amp;acc=GSE95620</a>) and <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95582" rel="noopener noreferrer" target="_blank">GSE95582</a> (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95582" rel="noopener noreferrer" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95582</a>), and also in NASA’s GeneLab Database (GLDS-218). A presentation outlining the experimental process and which contains video data of the harvests can be found at the NASA LSDA (Life Sciences Data Archive (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://lsda.jsc.nasa.gov/Dataset/dataset_info/13840" rel="noopener noreferrer" target="_blank">https://lsda.jsc.nasa.gov/Dataset/dataset_info/13840</a>).</p></section><section id="S7"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>A-LP and RF conceived the experiments and acquired funding for the experiments. A-LP, RF, NS, and AZ prepared and carried out the experiments. All authors contributed to the data analyses. BC, A-LP, and RF prepared the manuscript.</p></section><section id="conf1"><h2 class="pmc_sec_title">Conflict of Interest</h2>
<p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank the Interdisciplinary Center for Biotechnology Research at UF for analytical support, particularly the Gene Expression and Bioinformatics Cores. The operational activities of Astronaut Butch Wilmore, who conducted the on orbit operations, and the Kennedy Space Center personnel who supported the ground operations and launch, are both acknowledged and much appreciated. We especially thank the members of the UF Space Plants lab who have contributed their time and energy to our spaceflight research.</p></section><section class="glossary" id="glossary1"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list" id="DL1">
<dt>d</dt>
<dd><p>day</p></dd>
<dt>DEG</dt>
<dd><p>differentially expressed gene</p></dd>
<dt>FLT</dt>
<dd><p>spaceflight</p></dd>
<dt>GC</dt>
<dd><p>ground control</p></dd>
<dt>GO</dt>
<dd><p>gene ontology</p></dd>
<dt>GPI-AP</dt>
<dd><p>glycosylphosphatidylinositol-anchored protein.</p></dd>
</dl></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn4"><p><strong>Funding.</strong> This work was supported by the NASA SLPSRA grants NNX12AN69G and NNX14AT24G awarded to RF and A-LP.</p></div></div></section><section id="S10"><h2 class="pmc_sec_title">Supplementary Material</h2>
<p>The Supplementary Material for this article can be found online at: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.frontiersin.org/articles/10.3389/fpls.2020.00239/full#supplementary-material" rel="noopener noreferrer" target="_blank">https://www.frontiersin.org/articles/10.3389/fpls.2020.00239/full#supplementary-material</a></p>
<section class="sm xbox font-sm" id="SM1"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/7064724/bin/Data_Sheet_1.zip">Click here for additional data file.</a><sup> (387.6KB, zip) </sup>
</div></div></section></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="B1">
<cite>Andrews S. (2010). <em>FastQC</em><em>: A Quality Control Tool for High-Throughput Sequence Data [Online].</em> Available online at: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" rel="noopener noreferrer" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>
(accessed January 20, 2016).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FastQC:%20A%20Quality%20Control%20Tool%20for%20High-Throughput%20Sequence%20Data%20%5BOnline%5D.&amp;author=S.%20Andrews&amp;publication_year=2010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<cite>Baral A., Irani N. G., Fujimoto M., Nakano A., Mayor S., Mathew M. K. (2015). Salt-induced remodeling of spatially restricted clathrin-independent endocytic pathways in Arabidopsis root.
<em>Plant Cell</em>
27
1297–1315. 10.1105/tpc.15.00154
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.15.00154" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4558706/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25901088/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Salt-induced%20remodeling%20of%20spatially%20restricted%20clathrin-independent%20endocytic%20pathways%20in%20Arabidopsis%20root.&amp;author=A.%20Baral&amp;author=N.%20G.%20Irani&amp;author=M.%20Fujimoto&amp;author=A.%20Nakano&amp;author=S.%20Mayor&amp;volume=27&amp;publication_year=2015&amp;pages=1297-1315&amp;pmid=25901088&amp;doi=10.1105/tpc.15.00154&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<cite>Battaglia M., Olvera-Carrillo Y., Garciarrubio A., Campos F., Covarrubias A. A. (2008). The enigmatic LEA proteins and other hydrophilins.
<em>Plant Physiol.</em>
148
6–24. 10.1104/pp.108.120725
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.108.120725" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2528095/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18772351/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=The%20enigmatic%20LEA%20proteins%20and%20other%20hydrophilins.&amp;author=M.%20Battaglia&amp;author=Y.%20Olvera-Carrillo&amp;author=A.%20Garciarrubio&amp;author=F.%20Campos&amp;author=A.%20A.%20Covarrubias&amp;volume=148&amp;publication_year=2008&amp;pages=6-24&amp;pmid=18772351&amp;doi=10.1104/pp.108.120725&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<cite>Beisel N. S., Noble J., Barbazuk W. B., Paul A.-L., Ferl R. J. (2019). Spaceflight-induced alternative splicing during seedling development in <em>Arabidopsis thaliana</em>.
<em>NPJ Microgravity</em>
5:9. 10.1038/s41526-019-0070-7
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41526-019-0070-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6447593/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30963109/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Microgravity&amp;title=Spaceflight-induced%20alternative%20splicing%20during%20seedling%20development%20in%20Arabidopsis%20thaliana.&amp;author=N.%20S.%20Beisel&amp;author=J.%20Noble&amp;author=W.%20B.%20Barbazuk&amp;author=A.-L.%20Paul&amp;author=R.%20J.%20Ferl&amp;volume=5&amp;issue=9&amp;publication_year=2019&amp;pmid=30963109&amp;doi=10.1038/s41526-019-0070-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<cite>Berardini T. Z., Reiser L., Li D., Mezheritsky Y., Muller R., Strait E., et al.  (2015). The Arabidopsis information resource: making and mining the “gold standard” annotated reference plant genome.
<em>Genesis</em>
53
474–485. 10.1002/dvg.22877
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/dvg.22877" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4545719/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26201819/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genesis&amp;title=The%20Arabidopsis%20information%20resource:%20making%20and%20mining%20the%20%E2%80%9Cgold%20standard%E2%80%9D%20annotated%20reference%20plant%20genome.&amp;author=T.%20Z.%20Berardini&amp;author=L.%20Reiser&amp;author=D.%20Li&amp;author=Y.%20Mezheritsky&amp;author=R.%20Muller&amp;volume=53&amp;publication_year=2015&amp;pages=474-485&amp;pmid=26201819&amp;doi=10.1002/dvg.22877&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<cite>Bhaskara G. B., Nguyen T. T., Verslues P. E. (2012). Unique drought resistance functions of the highly ABA-induced clade A protein phosphatase 2Cs.
<em>Plant Physiol.</em>
160
379–395. 10.1104/pp.112.202408
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.112.202408" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3440212/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22829320/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Unique%20drought%20resistance%20functions%20of%20the%20highly%20ABA-induced%20clade%20A%20protein%20phosphatase%202Cs.&amp;author=G.%20B.%20Bhaskara&amp;author=T.%20T.%20Nguyen&amp;author=P.%20E.%20Verslues&amp;volume=160&amp;publication_year=2012&amp;pages=379-395&amp;pmid=22829320&amp;doi=10.1104/pp.112.202408&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<cite>Bolger A. M., Lohse M., Usadel B. (2014). Trimmomatic: a flexible trimmer for Illumina sequence data.
<em>Bioinformatics</em>
30
2114–2120. 10.1093/bioinformatics/btu170
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btu170" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4103590/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24695404/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data.&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;volume=30&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;doi=10.1093/bioinformatics/btu170&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<cite>Borderies G., Jamet E., Lafitte C., Rossignol M., Jauneau A., Boudart G., et al.  (2003). Proteomics of loosely bound cell wall proteins of <em>Arabidopsis thaliana</em> cell suspension cultures: a critical analysis.
<em>Electrophoresis</em>
24
3421–3432. 10.1002/elps.200305608
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/elps.200305608" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14595688/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Electrophoresis&amp;title=Proteomics%20of%20loosely%20bound%20cell%20wall%20proteins%20of%20Arabidopsis%20thaliana%20cell%20suspension%20cultures:%20a%20critical%20analysis.&amp;author=G.%20Borderies&amp;author=E.%20Jamet&amp;author=C.%20Lafitte&amp;author=M.%20Rossignol&amp;author=A.%20Jauneau&amp;volume=24&amp;publication_year=2003&amp;pages=3421-3432&amp;pmid=14595688&amp;doi=10.1002/elps.200305608&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<cite>Borner G. H., Sherrier D. J., Weimar T., Michaelson L. V., Hawkins N. D., Macaskill A., et al.  (2005). Analysis of detergent-resistant membranes in Arabidopsis. Evidence for plasma membrane lipid rafts.
<em>Plant Physiol.</em>
137
104–116. 10.1104/pp.104.053041
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.104.053041" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC548842/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15618420/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Analysis%20of%20detergent-resistant%20membranes%20in%20Arabidopsis.%20Evidence%20for%20plasma%20membrane%20lipid%20rafts.&amp;author=G.%20H.%20Borner&amp;author=D.%20J.%20Sherrier&amp;author=T.%20Weimar&amp;author=L.%20V.%20Michaelson&amp;author=N.%20D.%20Hawkins&amp;volume=137&amp;publication_year=2005&amp;pages=104-116&amp;pmid=15618420&amp;doi=10.1104/pp.104.053041&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<cite>Boucheron-Dubuisson E., Manzano A. I., Le Disquet I., Matía I., Sáez-Vasquez J., van Loon J. J., et al.  (2016). Functional alterations of root meristematic cells of <em>Arabidopsis thaliana</em> induced by a simulated microgravity environment.
<em>J. Plant Physiol.</em>
207
30–41. 10.1016/j.jplph.2016.09.011
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jplph.2016.09.011" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27792899/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Plant%20Physiol.&amp;title=Functional%20alterations%20of%20root%20meristematic%20cells%20of%20Arabidopsis%20thaliana%20induced%20by%20a%20simulated%20microgravity%20environment.&amp;author=E.%20Boucheron-Dubuisson&amp;author=A.%20I.%20Manzano&amp;author=I.%20Le%20Disquet&amp;author=I.%20Mat%C3%ADa&amp;author=J.%20S%C3%A1ez-Vasquez&amp;volume=207&amp;publication_year=2016&amp;pages=30-41&amp;pmid=27792899&amp;doi=10.1016/j.jplph.2016.09.011&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<cite>Broad Institute (2019a). <em>Morpheus [Online].</em> Available online at: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://software.broadinstitute.org/morpheus" rel="noopener noreferrer" target="_blank">https://software.broadinstitute.org/morpheus</a>
(accessed September 5, 2019).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Morpheus%20%5BOnline%5D.&amp;publication_year=2019a&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<cite>Broad Institute (2019b). <em>Picard Toolkit [Online].</em> Available online at: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://broadinstitute.github.io/picard/" rel="noopener noreferrer" target="_blank">http://broadinstitute.github.io/picard/</a>
(accessed January 20, 2016).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Picard%20Toolkit%20%5BOnline%5D.&amp;publication_year=2019b&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<cite>Candat A., Paszkiewicz G., Neveu M., Gautier R., Logan D. C., Avelange-Macherel M.-H., et al.  (2014). The ubiquitous distribution of late embryogenesis abundant proteins across cell compartments in <em>Arabidopsis</em> offers tailored protection against abiotic stress.
<em>Plant Cell</em>
26
3148–3166. 10.1105/tpc.114.127316
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.114.127316" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4145138/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25005920/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20ubiquitous%20distribution%20of%20late%20embryogenesis%20abundant%20proteins%20across%20cell%20compartments%20in%20Arabidopsis%20offers%20tailored%20protection%20against%20abiotic%20stress.&amp;author=A.%20Candat&amp;author=G.%20Paszkiewicz&amp;author=M.%20Neveu&amp;author=R.%20Gautier&amp;author=D.%20C.%20Logan&amp;volume=26&amp;publication_year=2014&amp;pages=3148-3166&amp;pmid=25005920&amp;doi=10.1105/tpc.114.127316&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14">
<cite>Carbon S., Ireland A., Mungall C. J., Shu S., Marshall B., Lewis S., et al.  (2008). AmiGO: online access to ontology and annotation data.
<em>Bioinformatics</em>
25
288–289. 10.1093/bioinformatics/btn615
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btn615" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2639003/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19033274/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=AmiGO:%20online%20access%20to%20ontology%20and%20annotation%20data.&amp;author=S.%20Carbon&amp;author=A.%20Ireland&amp;author=C.%20J.%20Mungall&amp;author=S.%20Shu&amp;author=B.%20Marshall&amp;volume=25&amp;publication_year=2008&amp;pages=288-289&amp;pmid=19033274&amp;doi=10.1093/bioinformatics/btn615&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15">
<cite>Cecchini N. M., Jung H. W., Engle N. L., Tschaplinski T. J., Greenberg J. T. (2015). ALD1 regulates basal immune components and early inducible defense responses in <em>Arabidopsis</em>.
<em>Mol. Plant Microbe Interact.</em>
28
455–466. 10.1094/mpmi-06-14-0187-r
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1094/mpmi-06-14-0187-r" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25372120/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Plant%20Microbe%20Interact.&amp;title=ALD1%20regulates%20basal%20immune%20components%20and%20early%20inducible%20defense%20responses%20in%20Arabidopsis.&amp;author=N.%20M.%20Cecchini&amp;author=H.%20W.%20Jung&amp;author=N.%20L.%20Engle&amp;author=T.%20J.%20Tschaplinski&amp;author=J.%20T.%20Greenberg&amp;volume=28&amp;publication_year=2015&amp;pages=455-466&amp;pmid=25372120&amp;doi=10.1094/mpmi-06-14-0187-r&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16">
<cite>Chan Z., Wang Y., Cao M., Gong Y., Mu Z., Wang H., et al.  (2016). <em>RDM4</em> modulates cold stress resistance in <em>Arabidopsis</em> partially through the <em>CBF</em>-mediated pathway.
<em>New Phytol.</em>
209
1527–1539. 10.1111/nph.13727
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/nph.13727" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5515388/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26522658/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=RDM4%20modulates%20cold%20stress%20resistance%20in%20Arabidopsis%20partially%20through%20the%20CBF-mediated%20pathway.&amp;author=Z.%20Chan&amp;author=Y.%20Wang&amp;author=M.%20Cao&amp;author=Y.%20Gong&amp;author=Z.%20Mu&amp;volume=209&amp;publication_year=2016&amp;pages=1527-1539&amp;pmid=26522658&amp;doi=10.1111/nph.13727&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17">
<cite>Chen T., Ji D., Tian S. (2018). Variable-angle epifluorescence microscopy characterizes protein dynamics in the vicinity of plasma membrane in plant cells.
<em>BMC Plant Biol.</em>
18:43. 10.1186/s12870-018-1246-0
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12870-018-1246-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5853057/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29540149/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biol.&amp;title=Variable-angle%20epifluorescence%20microscopy%20characterizes%20protein%20dynamics%20in%20the%20vicinity%20of%20plasma%20membrane%20in%20plant%20cells.&amp;author=T.%20Chen&amp;author=D.%20Ji&amp;author=S.%20Tian&amp;volume=18&amp;issue=43&amp;publication_year=2018&amp;pmid=29540149&amp;doi=10.1186/s12870-018-1246-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18">
<cite>Chen X., Wu S., Liu Z., Friml J. (2016). Environmental and endogenous control of cortical microtubule orientation.
<em>Trends Cell Biol.</em>
26
409–419. 10.1016/j.tcb.2016.02.003
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tcb.2016.02.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26951762/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Cell%20Biol.&amp;title=Environmental%20and%20endogenous%20control%20of%20cortical%20microtubule%20orientation.&amp;author=X.%20Chen&amp;author=S.%20Wu&amp;author=Z.%20Liu&amp;author=J.%20Friml&amp;volume=26&amp;publication_year=2016&amp;pages=409-419&amp;pmid=26951762&amp;doi=10.1016/j.tcb.2016.02.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19">
<cite>Choi W.-G., Barker R. J., Kim S.-H., Swanson S. J., Gilroy S. (2019). Variation in the transcriptome of different ecotypes of <em>Arabidopsis thaliana</em> reveals signatures of oxidative stress in plant responses to spaceflight.
<em>Am. J. Bot.</em>
106
123–136. 10.1002/ajb2.1223
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/ajb2.1223" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30644539/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Bot.&amp;title=Variation%20in%20the%20transcriptome%20of%20different%20ecotypes%20of%20Arabidopsis%20thaliana%20reveals%20signatures%20of%20oxidative%20stress%20in%20plant%20responses%20to%20spaceflight.&amp;author=W.-G.%20Choi&amp;author=R.%20J.%20Barker&amp;author=S.-H.%20Kim&amp;author=S.%20J.%20Swanson&amp;author=S.%20Gilroy&amp;volume=106&amp;publication_year=2019&amp;pages=123-136&amp;pmid=30644539&amp;doi=10.1002/ajb2.1223&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B20">
<cite>Correll M. J., Pyle T. P., Millar K. D. L., Sun Y., Yao J., Edelmann R. E., et al.  (2013). Transcriptome analyses of <em>Arabidopsis thaliana</em> seedlings grown in space: implications for gravity-responsive genes.
<em>Planta</em>
238
519–533. 10.1007/s00425-013-1909-x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00425-013-1909-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23771594/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Transcriptome%20analyses%20of%20Arabidopsis%20thaliana%20seedlings%20grown%20in%20space:%20implications%20for%20gravity-responsive%20genes.&amp;author=M.%20J.%20Correll&amp;author=T.%20P.%20Pyle&amp;author=K.%20D.%20L.%20Millar&amp;author=Y.%20Sun&amp;author=J.%20Yao&amp;volume=238&amp;publication_year=2013&amp;pages=519-533&amp;pmid=23771594&amp;doi=10.1007/s00425-013-1909-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21">
<cite>Dahlke R. I., Fraas S., Ullrich K. K., Heinemann K., Romeiks M., Rickmeyer T., et al.  (2017). Protoplast swelling and hypocotyl growth depend on different auxin signaling pathways.
<em>Plant Physiol.</em>
175
982–994. 10.1104/pp.17.00733
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.17.00733" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5619902/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28860155/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Protoplast%20swelling%20and%20hypocotyl%20growth%20depend%20on%20different%20auxin%20signaling%20pathways.&amp;author=R.%20I.%20Dahlke&amp;author=S.%20Fraas&amp;author=K.%20K.%20Ullrich&amp;author=K.%20Heinemann&amp;author=M.%20Romeiks&amp;volume=175&amp;publication_year=2017&amp;pages=982-994&amp;pmid=28860155&amp;doi=10.1104/pp.17.00733&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22">
<cite>Demir F., Horntrich C., Blachutzik J. O., Scherzer S., Reinders Y., Kierszniowska S., et al.  (2013). <em>Arabidopsis</em> nanodomain-delimited ABA signaling pathway regulates the anion channel SLAH3.
<em>Proc. Natl. Acad. Sci. U.S.A.</em>
110
8296–8301. 10.1073/pnas.1211667110
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1211667110" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3657796/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23630285/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Arabidopsis%20nanodomain-delimited%20ABA%20signaling%20pathway%20regulates%20the%20anion%20channel%20SLAH3.&amp;author=F.%20Demir&amp;author=C.%20Horntrich&amp;author=J.%20O.%20Blachutzik&amp;author=S.%20Scherzer&amp;author=Y.%20Reinders&amp;volume=110&amp;publication_year=2013&amp;pages=8296-8301&amp;pmid=23630285&amp;doi=10.1073/pnas.1211667110&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23">
<cite>Dobin A., Davis C. A., Schlesinger F., Drenkow J., Zaleski C., Jha S., et al.  (2012). STAR: ultrafast universal RNA-seq aligner.
<em>Bioinformatics</em>
29
15–21. 10.1093/bioinformatics/bts635
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/bts635" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3530905/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23104886/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=STAR:%20ultrafast%20universal%20RNA-seq%20aligner.&amp;author=A.%20Dobin&amp;author=C.%20A.%20Davis&amp;author=F.%20Schlesinger&amp;author=J.%20Drenkow&amp;author=C.%20Zaleski&amp;volume=29&amp;publication_year=2012&amp;pages=15-21&amp;pmid=23104886&amp;doi=10.1093/bioinformatics/bts635&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24">
<cite>Elmore J. M., Liu J., Smith B., Phinney B., Coaker G. (2012). Quantitative proteomics reveals dynamic changes in the plasma membrane during Arabidopsis immune signaling.
<em>Mol. Cell. Proteomics</em>
11:M111.014555. 10.1074/mcp.M111.014555
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/mcp.M111.014555" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3322570/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22215637/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Quantitative%20proteomics%20reveals%20dynamic%20changes%20in%20the%20plasma%20membrane%20during%20Arabidopsis%20immune%20signaling.&amp;author=J.%20M.%20Elmore&amp;author=J.%20Liu&amp;author=B.%20Smith&amp;author=B.%20Phinney&amp;author=G.%20Coaker&amp;volume=11&amp;issue=M111.014555&amp;publication_year=2012&amp;pmid=22215637&amp;doi=10.1074/mcp.M111.014555&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25">
<cite>Elortza F., Mohammed S., Bunkenborg J., Foster L. J., Nuhse T. S., Brodbeck U., et al.  (2006). Modification-specific proteomics of plasma membrane proteins: identification and characterization of glycosylphosphatidylinositol-anchored proteins released upon phospholipase D treatment.
<em>J. Proteome Res.</em>
5
935–943. 10.1021/pr050419u
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/pr050419u" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16602701/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Modification-specific%20proteomics%20of%20plasma%20membrane%20proteins:%20identification%20and%20characterization%20of%20glycosylphosphatidylinositol-anchored%20proteins%20released%20upon%20phospholipase%20D%20treatment.&amp;author=F.%20Elortza&amp;author=S.%20Mohammed&amp;author=J.%20Bunkenborg&amp;author=L.%20J.%20Foster&amp;author=T.%20S.%20Nuhse&amp;volume=5&amp;publication_year=2006&amp;pages=935-943&amp;pmid=16602701&amp;doi=10.1021/pr050419u&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26">
<cite>Fang Y., Xiong L. (2015). General mechanisms of drought response and their application in drought resistance improvement in plants.
<em>Cell. Mol. Life Sci.</em>
72
673–689. 10.1007/s00018-014-1767-0
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00018-014-1767-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC11113132/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25336153/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell.%20Mol.%20Life%20Sci.&amp;title=General%20mechanisms%20of%20drought%20response%20and%20their%20application%20in%20drought%20resistance%20improvement%20in%20plants.&amp;author=Y.%20Fang&amp;author=L.%20Xiong&amp;volume=72&amp;publication_year=2015&amp;pages=673-689&amp;pmid=25336153&amp;doi=10.1007/s00018-014-1767-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27">
<cite>Ferl R. J., Koh J., Denison F., Paul A.-L. (2015). Spaceflight induces specific alterations in the proteomes of Arabidopsis.
<em>Astrobiology</em>
15
32–56. 10.1089/ast.2014.1210
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2014.1210" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4290804/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25517942/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Spaceflight%20induces%20specific%20alterations%20in%20the%20proteomes%20of%20Arabidopsis.&amp;author=R.%20J.%20Ferl&amp;author=J.%20Koh&amp;author=F.%20Denison&amp;author=A.-L.%20Paul&amp;volume=15&amp;publication_year=2015&amp;pages=32-56&amp;pmid=25517942&amp;doi=10.1089/ast.2014.1210&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28">
<cite>Ferl R. J., Paul A.-L. (2016). The effect of spaceflight on the gravity-sensing auxin gradient of roots: GFP reporter gene microscopy on orbit.
<em>NPJ Microgravity</em>
2:15023. 10.1038/npjmgrav.2015.23
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/npjmgrav.2015.23" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5515520/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28725721/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Microgravity&amp;title=The%20effect%20of%20spaceflight%20on%20the%20gravity-sensing%20auxin%20gradient%20of%20roots:%20GFP%20reporter%20gene%20microscopy%20on%20orbit.&amp;author=R.%20J.%20Ferl&amp;author=A.-L.%20Paul&amp;volume=2&amp;issue=15023&amp;publication_year=2016&amp;pmid=28725721&amp;doi=10.1038/npjmgrav.2015.23&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29">
<cite>Footitt S., Clay H. A., Dent K., Finch-Savage W. E. (2014). Environment sensing in spring-dispersed seeds of a winter annual Arabidopsis influences the regulation of dormancy to align germination potential with seasonal changes.
<em>New Phytol.</em>
202
929–939. 10.1111/nph.12694
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/nph.12694" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4235297/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24444091/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Environment%20sensing%20in%20spring-dispersed%20seeds%20of%20a%20winter%20annual%20Arabidopsis%20influences%20the%20regulation%20of%20dormancy%20to%20align%20germination%20potential%20with%20seasonal%20changes.&amp;author=S.%20Footitt&amp;author=H.%20A.%20Clay&amp;author=K.%20Dent&amp;author=W.%20E.%20Finch-Savage&amp;volume=202&amp;publication_year=2014&amp;pages=929-939&amp;pmid=24444091&amp;doi=10.1111/nph.12694&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30">
<cite>Footitt S., Huang Z., Clay H. A., Mead A., Finch-Savage W. E. (2013). Temperature, light and nitrate sensing coordinate Arabidopsis seed dormancy cycling, resulting in winter and summer annual phenotypes.
<em>Plant J.</em>
74
1003–1015. 10.1111/tpj.12186
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.12186" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3764396/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23590427/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=Temperature,%20light%20and%20nitrate%20sensing%20coordinate%20Arabidopsis%20seed%20dormancy%20cycling,%20resulting%20in%20winter%20and%20summer%20annual%20phenotypes.&amp;author=S.%20Footitt&amp;author=Z.%20Huang&amp;author=H.%20A.%20Clay&amp;author=A.%20Mead&amp;author=W.%20E.%20Finch-Savage&amp;volume=74&amp;publication_year=2013&amp;pages=1003-1015&amp;pmid=23590427&amp;doi=10.1111/tpj.12186&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31">
<cite>Furutani I., Watanabe Y., Prieto R., Masukawa M., Suzuki K., Naoi K., et al.  (2000). The SPIRAL genes are required for directional control of cell elongation in <em>Arabidopsis thaliana</em>.
<em>Development</em>
127
4443–4453. </cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/dev.127.20.4443" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11003843/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=The%20SPIRAL%20genes%20are%20required%20for%20directional%20control%20of%20cell%20elongation%20in%20Arabidopsis%20thaliana.&amp;author=I.%20Furutani&amp;author=Y.%20Watanabe&amp;author=R.%20Prieto&amp;author=M.%20Masukawa&amp;author=K.%20Suzuki&amp;volume=127&amp;publication_year=2000&amp;pages=4443-4453&amp;pmid=11003843&amp;doi=10.1242/dev.127.20.4443&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32">
<cite>Galva C., Kirik V., Lindeboom J. J., Kaloriti D., Rancour D. M., Hussey P. J., et al.  (2014). The microtubule plus-end tracking proteins SPR1 and EB1b interact to maintain polar cell elongation and directional organ growth in <em>Arabidopsis</em>.
<em>Plant Cell</em>
26
4409–4425. 10.1105/tpc.114.131482
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.114.131482" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4277225/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25415978/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20microtubule%20plus-end%20tracking%20proteins%20SPR1%20and%20EB1b%20interact%20to%20maintain%20polar%20cell%20elongation%20and%20directional%20organ%20growth%20in%20Arabidopsis.&amp;author=C.%20Galva&amp;author=V.%20Kirik&amp;author=J.%20J.%20Lindeboom&amp;author=D.%20Kaloriti&amp;author=D.%20M.%20Rancour&amp;volume=26&amp;publication_year=2014&amp;pages=4409-4425&amp;pmid=25415978&amp;doi=10.1105/tpc.114.131482&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B33">
<cite>Gan X., Stegle O., Behr J., Steffen J. G., Drewe P., Hildebrand K. L., et al.  (2011). Multiple reference genomes and transcriptomes for <em>Arabidopsis thaliana</em>.
<em>Nature</em>
477
419–423. 10.1038/nature10414
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature10414" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4856438/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21874022/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Multiple%20reference%20genomes%20and%20transcriptomes%20for%20Arabidopsis%20thaliana.&amp;author=X.%20Gan&amp;author=O.%20Stegle&amp;author=J.%20Behr&amp;author=J.%20G.%20Steffen&amp;author=P.%20Drewe&amp;volume=477&amp;publication_year=2011&amp;pages=419-423&amp;pmid=21874022&amp;doi=10.1038/nature10414&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34">
<cite>Gao Y., Zhang Y., Zhang D., Dai X., Estelle M., Zhao Y. (2015). Auxin binding protein 1 (ABP1) is not required for either auxin signaling or Arabidopsis development.
<em>Proc. Natl. Acad. Sci. U.S.A.</em>
112
2275–2280. 10.1073/pnas.1500365112
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1500365112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4343106/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25646447/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Auxin%20binding%20protein%201%20(ABP1)%20is%20not%20required%20for%20either%20auxin%20signaling%20or%20Arabidopsis%20development.&amp;author=Y.%20Gao&amp;author=Y.%20Zhang&amp;author=D.%20Zhang&amp;author=X.%20Dai&amp;author=M.%20Estelle&amp;volume=112&amp;publication_year=2015&amp;pages=2275-2280&amp;pmid=25646447&amp;doi=10.1073/pnas.1500365112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35">
<cite>Guillaumot D., Guillon S., Déplanque T., Vanhee C., Gumy C., Masquelier D., et al.  (2009). The Arabidopsis TSPO-related protein is a stress and abscisic acid-regulated, endoplasmic reticulum–Golgi-localized membrane protein.
<em>Plant J.</em>
60
242–256. 10.1111/j.1365-313X.2009.03950.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2009.03950.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19548979/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=The%20Arabidopsis%20TSPO-related%20protein%20is%20a%20stress%20and%20abscisic%20acid-regulated,%20endoplasmic%20reticulum%E2%80%93Golgi-localized%20membrane%20protein.&amp;author=D.%20Guillaumot&amp;author=S.%20Guillon&amp;author=T.%20D%C3%A9planque&amp;author=C.%20Vanhee&amp;author=C.%20Gumy&amp;volume=60&amp;publication_year=2009&amp;pages=242-256&amp;pmid=19548979&amp;doi=10.1111/j.1365-313X.2009.03950.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36">
<cite>Herranz R., Medina F. J. (2014). Cell proliferation and plant development under novel altered gravity environments.
<em>Plant Biol.</em>
16
23–30. 10.1111/plb.12103
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/plb.12103" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24112664/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biol.&amp;title=Cell%20proliferation%20and%20plant%20development%20under%20novel%20altered%20gravity%20environments.&amp;author=R.%20Herranz&amp;author=F.%20J.%20Medina&amp;volume=16&amp;publication_year=2014&amp;pages=23-30&amp;pmid=24112664&amp;doi=10.1111/plb.12103&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B37">
<cite>Herranz R., Valbuena M. A., Youssef K., Medina F.-J. (2014). Mechanisms of disruption of meristematic competence by microgravity in <em>Arabidopsis</em> seedlings.
<em>Plant Signal. Behav.</em>
9:e28289. 10.4161/psb.28289
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4161/psb.28289" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4091523/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24614101/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signal.%20Behav.&amp;title=Mechanisms%20of%20disruption%20of%20meristematic%20competence%20by%20microgravity%20in%20Arabidopsis%20seedlings.&amp;author=R.%20Herranz&amp;author=M.%20A.%20Valbuena&amp;author=K.%20Youssef&amp;author=F.-J.%20Medina&amp;volume=9&amp;issue=e28289&amp;publication_year=2014&amp;pmid=24614101&amp;doi=10.4161/psb.28289&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B38">
<cite>Hundertmark M., Hincha D. K. (2008). LEA (Late Embryogenesis Abundant) proteins and their encoding genes in <em>Arabidopsis thaliana</em>.
<em>BMC Genomics</em>
9:118. 10.1186/1471-2164-9-118
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2164-9-118" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2292704/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18318901/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=LEA%20(Late%20Embryogenesis%20Abundant)%20proteins%20and%20their%20encoding%20genes%20in%20Arabidopsis%20thaliana.&amp;author=M.%20Hundertmark&amp;author=D.%20K.%20Hincha&amp;volume=9&amp;issue=118&amp;publication_year=2008&amp;pmid=18318901&amp;doi=10.1186/1471-2164-9-118&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B39">
<cite>Johnson C. M., Subramanian A., Pattathil S., Correll M. J., Kiss J. Z. (2017). Comparative transcriptomics indicate changes in cell wall organization and stress response in seedlings during spaceflight.
<em>Am. J. Bot.</em>
104
1219–1231. 10.3732/ajb.1700079
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3732/ajb.1700079" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5821596/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28827451/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Bot.&amp;title=Comparative%20transcriptomics%20indicate%20changes%20in%20cell%20wall%20organization%20and%20stress%20response%20in%20seedlings%20during%20spaceflight.&amp;author=C.%20M.%20Johnson&amp;author=A.%20Subramanian&amp;author=S.%20Pattathil&amp;author=M.%20J.%20Correll&amp;author=J.%20Z.%20Kiss&amp;volume=104&amp;publication_year=2017&amp;pages=1219-1231&amp;pmid=28827451&amp;doi=10.3732/ajb.1700079&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B40">
<cite>Jurkiewicz P., Melser S., Maucourt M., Ayeb H., Veljanovski V., Maneta-Peyret L., et al.  (2018). The multistress-induced Translocator protein (TSPO) differentially modulates storage lipids metabolism in seeds and seedlings.
<em>Plant J.</em>
96
274–286. 10.1111/tpj.14028
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.14028" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30003614/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=The%20multistress-induced%20Translocator%20protein%20(TSPO)%20differentially%20modulates%20storage%20lipids%20metabolism%20in%20seeds%20and%20seedlings.&amp;author=P.%20Jurkiewicz&amp;author=S.%20Melser&amp;author=M.%20Maucourt&amp;author=H.%20Ayeb&amp;author=V.%20Veljanovski&amp;volume=96&amp;publication_year=2018&amp;pages=274-286&amp;pmid=30003614&amp;doi=10.1111/tpj.14028&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B41">
<cite>Kazemi-Shahandashti S.-S., Maali-Amiri R. (2018). Global insights of protein responses to cold stress in plants: signaling, defence, and degradation.
<em>J. Plant Physiol.</em>
226
123–135. 10.1016/j.jplph.2018.03.022
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jplph.2018.03.022" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29758377/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Plant%20Physiol.&amp;title=Global%20insights%20of%20protein%20responses%20to%20cold%20stress%20in%20plants:%20signaling,%20defence,%20and%20degradation.&amp;author=S.-S.%20Kazemi-Shahandashti&amp;author=R.%20Maali-Amiri&amp;volume=226&amp;publication_year=2018&amp;pages=123-135&amp;pmid=29758377&amp;doi=10.1016/j.jplph.2018.03.022&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B42">
<cite>Keinath N. F., Kierszniowska S., Lorek J., Bourdais G., Kessler S. A., Shimosato-Asano H., et al.  (2010). PAMP (Pathogen-associated Molecular Pattern)-induced changes in plasma membrane compartmentalization reveal novel components of plant immunity.
<em>J. Biol. Chem.</em>
285
39140–39149. 10.1074/jbc.M110.160531
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M110.160531" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2998143/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20843791/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=PAMP%20(Pathogen-associated%20Molecular%20Pattern)-induced%20changes%20in%20plasma%20membrane%20compartmentalization%20reveal%20novel%20components%20of%20plant%20immunity.&amp;author=N.%20F.%20Keinath&amp;author=S.%20Kierszniowska&amp;author=J.%20Lorek&amp;author=G.%20Bourdais&amp;author=S.%20A.%20Kessler&amp;volume=285&amp;publication_year=2010&amp;pages=39140-39149&amp;pmid=20843791&amp;doi=10.1074/jbc.M110.160531&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B43">
<cite>Kim W., Lee Y., Park J., Lee N., Choi G. (2013). HONSU, a protein phosphatase 2C, regulates seed dormancy by inhibiting ABA signaling in Arabidopsis.
<em>Plant Cell Physiol.</em>
54
555–572. 10.1093/pcp/pct017
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pct017" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23378449/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol.&amp;title=HONSU,%20a%20protein%20phosphatase%202C,%20regulates%20seed%20dormancy%20by%20inhibiting%20ABA%20signaling%20in%20Arabidopsis.&amp;author=W.%20Kim&amp;author=Y.%20Lee&amp;author=J.%20Park&amp;author=N.%20Lee&amp;author=G.%20Choi&amp;volume=54&amp;publication_year=2013&amp;pages=555-572&amp;pmid=23378449&amp;doi=10.1093/pcp/pct017&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B44">
<cite>Klessig D. F., Choi H. W., Dempsey D. M. A. (2018). Systemic acquired resistance and salicylic acid: past, present, and future.
<em>Mol. Plant Microbe Interact.</em>
31
871–888. 10.1094/mpmi-03-18-0067-cr
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1094/mpmi-03-18-0067-cr" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29781762/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Plant%20Microbe%20Interact.&amp;title=Systemic%20acquired%20resistance%20and%20salicylic%20acid:%20past,%20present,%20and%20future.&amp;author=D.%20F.%20Klessig&amp;author=H.%20W.%20Choi&amp;author=D.%20M.%20A.%20Dempsey&amp;volume=31&amp;publication_year=2018&amp;pages=871-888&amp;pmid=29781762&amp;doi=10.1094/mpmi-03-18-0067-cr&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B45">
<cite>Klinkenberg J., Faist H., Saupe S., Lambertz S., Krischke M., Stingl N., et al.  (2014). Two fatty acid desaturases, STEAROYL-ACYL CARRIER PROTEIN Δ9-DESATURASE6 and FATTY ACID DESATURASE3, are involved in drought and hypoxia stress signaling in Arabidopsis crown galls.
<em>Plant Physiol.</em>
164
570–583. 10.1104/pp.113.230326
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.113.230326" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3912090/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24368335/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Two%20fatty%20acid%20desaturases,%20STEAROYL-ACYL%20CARRIER%20PROTEIN%20%CE%949-DESATURASE6%20and%20FATTY%20ACID%20DESATURASE3,%20are%20involved%20in%20drought%20and%20hypoxia%20stress%20signaling%20in%20Arabidopsis%20crown%20galls.&amp;author=J.%20Klinkenberg&amp;author=H.%20Faist&amp;author=S.%20Saupe&amp;author=S.%20Lambertz&amp;author=M.%20Krischke&amp;volume=164&amp;publication_year=2014&amp;pages=570-583&amp;pmid=24368335&amp;doi=10.1104/pp.113.230326&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B46">
<cite>Kohler A., Schwindling S., Conrath U. (2002). Benzothiadiazole-induced priming for potentiated responses to pathogen infection, wounding, and infiltration of water into leaves requires the <em>NPR1/NIM1</em> gene in Arabidopsis.
<em>Plant Physiol.</em>
128
1046–1056. 10.1104/pp.010744
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.010744" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC152216/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11891259/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Benzothiadiazole-induced%20priming%20for%20potentiated%20responses%20to%20pathogen%20infection,%20wounding,%20and%20infiltration%20of%20water%20into%20leaves%20requires%20the%20NPR1/NIM1%20gene%20in%20Arabidopsis.&amp;author=A.%20Kohler&amp;author=S.%20Schwindling&amp;author=U.%20Conrath&amp;volume=128&amp;publication_year=2002&amp;pages=1046-1056&amp;pmid=11891259&amp;doi=10.1104/pp.010744&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B47">
<cite>Kordyum E. L., Zhupanov I., Klymenko O., Bulavin I., Vorobiova T. (2018). “Simulated microgravity impacts the lipid rafts in plant cells,” in <em>Proceedings of the 42nd COSPAR Scientific Assembly</em>, Pasadena, CA.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Proceedings%20of%20the%2042nd%20COSPAR%20Scientific%20Assembly&amp;author=E.%20L.%20Kordyum&amp;author=I.%20Zhupanov&amp;author=O.%20Klymenko&amp;author=I.%20Bulavin&amp;author=T.%20Vorobiova&amp;publication_year=2018&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B48">
<cite>Krishnakumar V., Contrino S., Cheng C.-Y., Belyaeva I., Ferlanti E. S., Miller J. R., et al.  (2016). ThaleMine: a warehouse for Arabidopsis data integration and discovery.
<em>Plant Cell Physiol.</em>
58:e4. 10.1093/pcp/pcw200
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pcw200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28013278/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol.&amp;title=ThaleMine:%20a%20warehouse%20for%20Arabidopsis%20data%20integration%20and%20discovery.&amp;author=V.%20Krishnakumar&amp;author=S.%20Contrino&amp;author=C.-Y.%20Cheng&amp;author=I.%20Belyaeva&amp;author=E.%20S.%20Ferlanti&amp;volume=58&amp;issue=e4&amp;publication_year=2016&amp;pmid=28013278&amp;doi=10.1093/pcp/pcw200&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B49">
<cite>Kwon T., Sparks J. A., Nakashima J., Allen S. N., Tang Y., Blancaflor E. B. (2015). Transcriptional response of Arabidopsis seedlings during spaceflight reveals peroxidase and cell wall remodeling genes associated with root hair development.
<em>Am. J. Bot.</em>
102
21–35. 10.3732/ajb.1400458
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3732/ajb.1400458" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25587145/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Bot.&amp;title=Transcriptional%20response%20of%20Arabidopsis%20seedlings%20during%20spaceflight%20reveals%20peroxidase%20and%20cell%20wall%20remodeling%20genes%20associated%20with%20root%20hair%20development.&amp;author=T.%20Kwon&amp;author=J.%20A.%20Sparks&amp;author=J.%20Nakashima&amp;author=S.%20N.%20Allen&amp;author=Y.%20Tang&amp;volume=102&amp;publication_year=2015&amp;pages=21-35&amp;pmid=25587145&amp;doi=10.3732/ajb.1400458&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B50">
<cite>Lamesch P., Berardini T. Z., Li D., Swarbreck D., Wilks C., Sasidharan R., et al.  (2011). The Arabidopsis information resource (TAIR): improved gene annotation and new tools.
<em>Nucleic Acids Res.</em>
40
D1202–D1210. 10.1093/nar/gkr1090
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkr1090" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3245047/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22140109/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20Arabidopsis%20information%20resource%20(TAIR):%20improved%20gene%20annotation%20and%20new%20tools.&amp;author=P.%20Lamesch&amp;author=T.%20Z.%20Berardini&amp;author=D.%20Li&amp;author=D.%20Swarbreck&amp;author=C.%20Wilks&amp;volume=40&amp;publication_year=2011&amp;pages=D1202-D1210&amp;pmid=22140109&amp;doi=10.1093/nar/gkr1090&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B51">
<cite>Leach J. E., Ryba-White M., Sun Q., Wu C., Hilaire E., Gartner C., et al.  (2001). Plants, plant pathogens, and microgravity—A deadly trio.
<em>Gravit. Space Res.</em>
14
15–23. </cite> [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11865865/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Gravit.%20Space%20Res.&amp;title=Plants,%20plant%20pathogens,%20and%20microgravity%E2%80%94A%20deadly%20trio.&amp;author=J.%20E.%20Leach&amp;author=M.%20Ryba-White&amp;author=Q.%20Sun&amp;author=C.%20Wu&amp;author=E.%20Hilaire&amp;volume=14&amp;publication_year=2001&amp;pages=15-23&amp;pmid=11865865&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B52">
<cite>Li B., Takahashi D., Kawamura Y., Uemura M. (2012). Comparison of plasma membrane proteomic changes of Arabidopsis suspension-cultured cells (T87 Line) after Cold and ABA treatment in association with freezing tolerance development.
<em>Plant Cell Physiol.</em>
53
543–554. 10.1093/pcp/pcs010
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pcs010" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22318864/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol.&amp;title=Comparison%20of%20plasma%20membrane%20proteomic%20changes%20of%20Arabidopsis%20suspension-cultured%20cells%20(T87%20Line)%20after%20Cold%20and%20ABA%20treatment%20in%20association%20with%20freezing%20tolerance%20development.&amp;author=B.%20Li&amp;author=D.%20Takahashi&amp;author=Y.%20Kawamura&amp;author=M.%20Uemura&amp;volume=53&amp;publication_year=2012&amp;pages=543-554&amp;pmid=22318864&amp;doi=10.1093/pcp/pcs010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B53">
<cite>Mamode Cassim A., Gouguet P., Gronnier J., Laurent N., Germain V., Grison M., et al.  (2019). Plant lipids: key players of plasma membrane organization and function.
<em>Prog. Lipid Res.</em>
73
1–27. 10.1016/j.plipres.2018.11.002
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.plipres.2018.11.002" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30465788/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Prog.%20Lipid%20Res.&amp;title=Plant%20lipids:%20key%20players%20of%20plasma%20membrane%20organization%20and%20function.&amp;author=A.%20Mamode%20Cassim&amp;author=P.%20Gouguet&amp;author=J.%20Gronnier&amp;author=N.%20Laurent&amp;author=V.%20Germain&amp;volume=73&amp;publication_year=2019&amp;pages=1-27&amp;pmid=30465788&amp;doi=10.1016/j.plipres.2018.11.002&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B54">
<cite>Matía I., González-Camacho F., Herranz R., Kiss J. Z., Gasset G., van Loon J. J. W. A., et al.  (2010). Plant cell proliferation and growth are altered by microgravity conditions in spaceflight.
<em>J. Plant Physiol.</em>
167
184–193. 10.1016/j.jplph.2009.08.012
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jplph.2009.08.012" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19864040/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Plant%20Physiol.&amp;title=Plant%20cell%20proliferation%20and%20growth%20are%20altered%20by%20microgravity%20conditions%20in%20spaceflight.&amp;author=I.%20Mat%C3%ADa&amp;author=F.%20Gonz%C3%A1lez-Camacho&amp;author=R.%20Herranz&amp;author=J.%20Z.%20Kiss&amp;author=G.%20Gasset&amp;volume=167&amp;publication_year=2010&amp;pages=184-193&amp;pmid=19864040&amp;doi=10.1016/j.jplph.2009.08.012&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B55">
<cite>Mazars C., Brière C., Grat S., Pichereaux C., Rossignol M., Pereda-Loth V., et al.  (2014). Microgravity induces changes in microsome-associated proteins of Arabidopsis seedlings grown on board the international space station.
<em>PLoS One</em>
9:e91814. 10.1371/journal.pone.0091814
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0091814" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3950288/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24618597/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Microgravity%20induces%20changes%20in%20microsome-associated%20proteins%20of%20Arabidopsis%20seedlings%20grown%20on%20board%20the%20international%20space%20station.&amp;author=C.%20Mazars&amp;author=C.%20Bri%C3%A8re&amp;author=S.%20Grat&amp;author=C.%20Pichereaux&amp;author=M.%20Rossignol&amp;volume=9&amp;issue=e91814&amp;publication_year=2014&amp;pmid=24618597&amp;doi=10.1371/journal.pone.0091814&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B56">
<cite>Menges M., Hennig L., Gruissem W., Murray J. A. H. (2002). Cell cycle-regulated gene expression in Arabidopsis.
<em>J. Biol. Chem.</em>
277
41987–42002. 10.1074/jbc.M207570200
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M207570200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12169696/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Cell%20cycle-regulated%20gene%20expression%20in%20Arabidopsis.&amp;author=M.%20Menges&amp;author=L.%20Hennig&amp;author=W.%20Gruissem&amp;author=J.%20A.%20H.%20Murray&amp;volume=277&amp;publication_year=2002&amp;pages=41987-42002&amp;pmid=12169696&amp;doi=10.1074/jbc.M207570200&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B57">
<cite>Mi H., Muruganujan A., Ebert D., Huang X., Thomas P. D. (2018). PANTHER version 14: more genomes, a new PANTHER GO-slim and improvements in enrichment analysis tools.
<em>Nucleic Acids Res.</em>
47
D419–D426. 10.1093/nar/gky1038
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gky1038" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6323939/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30407594/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=PANTHER%20version%2014:%20more%20genomes,%20a%20new%20PANTHER%20GO-slim%20and%20improvements%20in%20enrichment%20analysis%20tools.&amp;author=H.%20Mi&amp;author=A.%20Muruganujan&amp;author=D.%20Ebert&amp;author=X.%20Huang&amp;author=P.%20D.%20Thomas&amp;volume=47&amp;publication_year=2018&amp;pages=D419-D426&amp;pmid=30407594&amp;doi=10.1093/nar/gky1038&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B58">
<cite>Miki Y., Takahashi D., Kawamura Y., Uemura M. (2019). Temporal proteomics of Arabidopsis plasma membrane during cold- and de-acclimation.
<em>J. Proteomics</em>
197
71–81. 10.1016/j.jprot.2018.11.008
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jprot.2018.11.008" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30447334/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteomics&amp;title=Temporal%20proteomics%20of%20Arabidopsis%20plasma%20membrane%20during%20cold-%20and%20de-acclimation.&amp;author=Y.%20Miki&amp;author=D.%20Takahashi&amp;author=Y.%20Kawamura&amp;author=M.%20Uemura&amp;volume=197&amp;publication_year=2019&amp;pages=71-81&amp;pmid=30447334&amp;doi=10.1016/j.jprot.2018.11.008&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B59">
<cite>Millar K. D. L., Johnson C. M., Edelmann R. E., Kiss J. Z. (2011). An endogenous growth pattern of roots is revealed in seedlings grown in microgravity.
<em>Astrobiology</em>
11
787–797. 10.1089/ast.2011.0699
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2011.0699" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3233217/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21970704/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=An%20endogenous%20growth%20pattern%20of%20roots%20is%20revealed%20in%20seedlings%20grown%20in%20microgravity.&amp;author=K.%20D.%20L.%20Millar&amp;author=C.%20M.%20Johnson&amp;author=R.%20E.%20Edelmann&amp;author=J.%20Z.%20Kiss&amp;volume=11&amp;publication_year=2011&amp;pages=787-797&amp;pmid=21970704&amp;doi=10.1089/ast.2011.0699&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B60">
<cite>Minami A., Fujiwara M., Furuto A., Fukao Y., Yamashita T., Kamo M., et al.  (2008). Alterations in detergent-resistant plasma membrane microdomains in <em>Arabidopsis thaliana</em> during cold acclimation.
<em>Plant Cell Physiol.</em>
50
341–359. 10.1093/pcp/pcn202
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pcn202" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19106119/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol.&amp;title=Alterations%20in%20detergent-resistant%20plasma%20membrane%20microdomains%20in%20Arabidopsis%20thaliana%20during%20cold%20acclimation.&amp;author=A.%20Minami&amp;author=M.%20Fujiwara&amp;author=A.%20Furuto&amp;author=Y.%20Fukao&amp;author=T.%20Yamashita&amp;volume=50&amp;publication_year=2008&amp;pages=341-359&amp;pmid=19106119&amp;doi=10.1093/pcp/pcn202&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B61">
<cite>Molitor A. M., Bu Z., Yu Y., Shen W.-H. (2014). Arabidopsis AL PHD-PRC1 complexes promote seed germination through H3K4me3-to-H3K27me3 chromatin state switch in repression of seed developmental genes.
<em>PLoS Genet.</em>
10:e1004091. 10.1371/journal.pgen.1004091
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pgen.1004091" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3900384/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24465219/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Arabidopsis%20AL%20PHD-PRC1%20complexes%20promote%20seed%20germination%20through%20H3K4me3-to-H3K27me3%20chromatin%20state%20switch%20in%20repression%20of%20seed%20developmental%20genes.&amp;author=A.%20M.%20Molitor&amp;author=Z.%20Bu&amp;author=Y.%20Yu&amp;author=W.-H.%20Shen&amp;volume=10&amp;issue=e1004091&amp;publication_year=2014&amp;pmid=24465219&amp;doi=10.1371/journal.pgen.1004091&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B62">
<cite>Nakajima K., Furutani I., Tachimoto H., Matsubara H., Hashimoto T. (2004). <em>SPIRAL1</em> encodes a plant-specific microtubule-localized protein required for directional control of rapidly expanding Arabidopsis cells.
<em>Plant Cell</em>
16
1178–1190. 10.1105/tpc.017830
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.017830" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC423208/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15084720/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=SPIRAL1%20encodes%20a%20plant-specific%20microtubule-localized%20protein%20required%20for%20directional%20control%20of%20rapidly%20expanding%20Arabidopsis%20cells.&amp;author=K.%20Nakajima&amp;author=I.%20Furutani&amp;author=H.%20Tachimoto&amp;author=H.%20Matsubara&amp;author=T.%20Hashimoto&amp;volume=16&amp;publication_year=2004&amp;pages=1178-1190&amp;pmid=15084720&amp;doi=10.1105/tpc.017830&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B63">
<cite>Nakashima J., Liao F., Sparks J. A., Tang Y., Blancaflor E. B. (2014). The actin cytoskeleton is a suppressor of the endogenous skewing behaviour of Arabidopsis primary roots in microgravity.
<em>Plant Biol.</em>
16(Suppl. 1), 142–150. 10.1111/plb.12062
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/plb.12062" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23952736/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biol.&amp;title=The%20actin%20cytoskeleton%20is%20a%20suppressor%20of%20the%20endogenous%20skewing%20behaviour%20of%20Arabidopsis%20primary%20roots%20in%20microgravity.&amp;author=J.%20Nakashima&amp;author=F.%20Liao&amp;author=J.%20A.%20Sparks&amp;author=Y.%20Tang&amp;author=E.%20B.%20Blancaflor&amp;volume=16&amp;issue=Suppl.%201&amp;publication_year=2014&amp;pages=142-150&amp;pmid=23952736&amp;doi=10.1111/plb.12062&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B64">
<cite>Nejat N., Mantri N. (2017). Plant immune system: crosstalk between responses to biotic and abiotic stresses the missing link in understanding plant defence.
<em>Curr. Issues Mol. Biol.</em>
23
1–16. 10.21775/cimb.023.001
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.21775/cimb.023.001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28154243/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Issues%20Mol.%20Biol.&amp;title=Plant%20immune%20system:%20crosstalk%20between%20responses%20to%20biotic%20and%20abiotic%20stresses%20the%20missing%20link%20in%20understanding%20plant%20defence.&amp;author=N.%20Nejat&amp;author=N.%20Mantri&amp;volume=23&amp;publication_year=2017&amp;pages=1-16&amp;pmid=28154243&amp;doi=10.21775/cimb.023.001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B65">
<cite>Nie H., Wu Y., Yao C., Tang D. (2011). Suppression of edr2-mediated powdery mildew resistance, cell death and ethylene-induced senescence by mutations in ALD1 in Arabidopsis.
<em>J. Genet. Genomics</em>
38
137–148. 10.1016/j.jgg.2011.03.001
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jgg.2011.03.001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21530897/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Genet.%20Genomics&amp;title=Suppression%20of%20edr2-mediated%20powdery%20mildew%20resistance,%20cell%20death%20and%20ethylene-induced%20senescence%20by%20mutations%20in%20ALD1%20in%20Arabidopsis.&amp;author=H.%20Nie&amp;author=Y.%20Wu&amp;author=C.%20Yao&amp;author=D.%20Tang&amp;volume=38&amp;publication_year=2011&amp;pages=137-148&amp;pmid=21530897&amp;doi=10.1016/j.jgg.2011.03.001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B66">
<cite>Oliva M., Dunand C. (2007). Waving and skewing: how gravity and the surface of growth media affect root development in Arabidopsis.
<em>New Phytol.</em>
176
37–43. 10.1111/j.1469-8137.2007.02184.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1469-8137.2007.02184.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17692076/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Waving%20and%20skewing:%20how%20gravity%20and%20the%20surface%20of%20growth%20media%20affect%20root%20development%20in%20Arabidopsis.&amp;author=M.%20Oliva&amp;author=C.%20Dunand&amp;volume=176&amp;publication_year=2007&amp;pages=37-43&amp;pmid=17692076&amp;doi=10.1111/j.1469-8137.2007.02184.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B67">
<cite>Paul A.-L., Amalfitano C., Ferl R. (2012a). Plant growth strategies are remodeled by spaceflight.
<em>BMC Plant Biol.</em>
12:232. 10.1186/1471-2229-12-232
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2229-12-232" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3556330/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23217113/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biol.&amp;title=Plant%20growth%20strategies%20are%20remodeled%20by%20spaceflight.&amp;author=A.-L.%20Paul&amp;author=C.%20Amalfitano&amp;author=R.%20Ferl&amp;volume=12&amp;issue=232&amp;publication_year=2012a&amp;pmid=23217113&amp;doi=10.1186/1471-2229-12-232&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B69">
<cite>Paul A.-L., Popp M. P., Gurley W. B., Guy C., Norwood K. L., Ferl R. J. (2005). Arabidopsis gene expression patterns are altered during spaceflight.
<em>Adv. Space Res.</em>
36
1175–1181. 10.1016/j.asr.2005.03.066
23919896</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.asr.2005.03.066" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res.&amp;title=Arabidopsis%20gene%20expression%20patterns%20are%20altered%20during%20spaceflight.&amp;author=A.-L.%20Paul&amp;author=M.%20P.%20Popp&amp;author=W.%20B.%20Gurley&amp;author=C.%20Guy&amp;author=K.%20L.%20Norwood&amp;volume=36&amp;publication_year=2005&amp;pages=1175-1181&amp;doi=10.1016/j.asr.2005.03.066&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B70">
<cite>Paul A.-L., Sng N., Zupanska A., Krishnamurthy A., Schultz E., Ferl R. (2017). Genetic dissection of the Arabidopsis spaceflight transcriptome: are some responses dispensable for the physiological adaptation of plants to spaceflight?
<em>PLoS One</em>
12:e0180186. 10.1371/journal.pone.0180186
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0180186" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5491145/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28662188/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Genetic%20dissection%20of%20the%20Arabidopsis%20spaceflight%20transcriptome:%20are%20some%20responses%20dispensable%20for%20the%20physiological%20adaptation%20of%20plants%20to%20spaceflight?&amp;author=A.-L.%20Paul&amp;author=N.%20Sng&amp;author=A.%20Zupanska&amp;author=A.%20Krishnamurthy&amp;author=E.%20Schultz&amp;volume=12&amp;issue=e0180186&amp;publication_year=2017&amp;pmid=28662188&amp;doi=10.1371/journal.pone.0180186&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B68">
<cite>Paul A.-L., Zupanska A. K., Ostrow D. T., Zhang Y., Sun Y., Li J. L., et al.  (2012b). Spaceflight transcriptomes: unique responses to a novel environment.
<em>Astrobiology</em>
12
40–56. 10.1089/ast.2011.0696
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2011.0696" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3264962/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22221117/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Spaceflight%20transcriptomes:%20unique%20responses%20to%20a%20novel%20environment.&amp;author=A.-L.%20Paul&amp;author=A.%20K.%20Zupanska&amp;author=D.%20T.%20Ostrow&amp;author=Y.%20Zhang&amp;author=Y.%20Sun&amp;volume=12&amp;publication_year=2012b&amp;pages=40-56&amp;pmid=22221117&amp;doi=10.1089/ast.2011.0696&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B71">
<cite>Paul A.-L., Zupanska A., Schultz E., Ferl R. (2013). Organ-specific remodeling of the Arabidopsis transcriptome in response to spaceflight.
<em>BMC Plant Biol.</em>
13:112. 10.1186/1471-2229-13-112
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2229-13-112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3750915/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23919896/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biol.&amp;title=Organ-specific%20remodeling%20of%20the%20Arabidopsis%20transcriptome%20in%20response%20to%20spaceflight.&amp;author=A.-L.%20Paul&amp;author=A.%20Zupanska&amp;author=E.%20Schultz&amp;author=R.%20Ferl&amp;volume=13&amp;issue=112&amp;publication_year=2013&amp;pmid=23919896&amp;doi=10.1186/1471-2229-13-112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B72">
<cite>Popova A. F., Sytnik K. M., Kordyum E. L., Meleshko G. I., Sychev V. N., Levinskykh M. A. (1989). Ultrastructural and growth indices of chlorella culture in multicomponent aquatic systems under space flight conditions.
<em>Adv. Space Res.</em>
9
79–82. 10.1016/0273-1177(89)90059-8
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0273-1177(89)90059-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11537353/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res.&amp;title=Ultrastructural%20and%20growth%20indices%20of%20chlorella%20culture%20in%20multicomponent%20aquatic%20systems%20under%20space%20flight%20conditions.&amp;author=A.%20F.%20Popova&amp;author=K.%20M.%20Sytnik&amp;author=E.%20L.%20Kordyum&amp;author=G.%20I.%20Meleshko&amp;author=V.%20N.%20Sychev&amp;volume=9&amp;publication_year=1989&amp;pages=79-82&amp;pmid=11537353&amp;doi=10.1016/0273-1177(89)90059-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B73">
<cite>Prodhan M. Y., Munemasa S., Nahar M. N.-E.-N., Nakamura Y., Murata Y. (2018). Guard cell salicylic acid signaling is integrated into abscisic acid signaling via the Ca2+/CPK-dependent pathway.
<em>Plant Physiol.</em>
178
441–450. 10.1104/pp.18.00321
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.18.00321" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6130018/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30037808/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Guard%20cell%20salicylic%20acid%20signaling%20is%20integrated%20into%20abscisic%20acid%20signaling%20via%20the%20Ca2+/CPK-dependent%20pathway.&amp;author=M.%20Y.%20Prodhan&amp;author=S.%20Munemasa&amp;author=M.%20N.-E.-N.%20Nahar&amp;author=Y.%20Nakamura&amp;author=Y.%20Murata&amp;volume=178&amp;publication_year=2018&amp;pages=441-450&amp;pmid=30037808&amp;doi=10.1104/pp.18.00321&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B74">
<cite>R Core Team (2019). <em>R: A Language and Environment for Statistical Computing.</em>
Vienna: R Foundation for Statistical Computing.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=R:%20A%20Language%20and%20Environment%20for%20Statistical%20Computing.&amp;publication_year=2019&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B75">
<cite>Ramegowda V., Senthil-Kumar M. (2015). The interactive effects of simultaneous biotic and abiotic stresses on plants: mechanistic understanding from drought and pathogen combination.
<em>J. Plant Physiol.</em>
176
47–54. 10.1016/j.jplph.2014.11.008
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jplph.2014.11.008" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25546584/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Plant%20Physiol.&amp;title=The%20interactive%20effects%20of%20simultaneous%20biotic%20and%20abiotic%20stresses%20on%20plants:%20mechanistic%20understanding%20from%20drought%20and%20pathogen%20combination.&amp;author=V.%20Ramegowda&amp;author=M.%20Senthil-Kumar&amp;volume=176&amp;publication_year=2015&amp;pages=47-54&amp;pmid=25546584&amp;doi=10.1016/j.jplph.2014.11.008&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B76">
<cite>Raudvere U., Kolberg L., Kuzmin I., Arak T., Adler P., Peterson H., et al.  (2019). g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update).
<em>Nucleic Acids Res.</em>
47
W191–W198. 10.1093/nar/gkz369
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkz369" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6602461/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31066453/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=g:Profiler:%20a%20web%20server%20for%20functional%20enrichment%20analysis%20and%20conversions%20of%20gene%20lists%20(2019%20update).&amp;author=U.%20Raudvere&amp;author=L.%20Kolberg&amp;author=I.%20Kuzmin&amp;author=T.%20Arak&amp;author=P.%20Adler&amp;volume=47&amp;publication_year=2019&amp;pages=W191-W198&amp;pmid=31066453&amp;doi=10.1093/nar/gkz369&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B77">
<cite>Roy R., Bassham D. C. (2014). Root growth movements: waving and skewing.
<em>Plant Sci.</em>
221–222
42–47. 10.1016/j.plantsci.2014.01.007
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.plantsci.2014.01.007" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24656334/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Sci.&amp;title=Root%20growth%20movements:%20waving%20and%20skewing.&amp;author=R.%20Roy&amp;author=D.%20C.%20Bassham&amp;volume=221%E2%80%93222&amp;publication_year=2014&amp;pages=42-47&amp;pmid=24656334&amp;doi=10.1016/j.plantsci.2014.01.007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B78">
<cite>Rueden C. T., Schindelin J., Hiner M. C., DeZonia B. E., Walter A. E., Arena E. T., et al.  (2017). ImageJ2: imageJ for the next generation of scientific image data.
<em>BMC Bioinformatics</em>
18:529. 10.1186/s12859-017-1934-z
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12859-017-1934-z" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5708080/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29187165/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=ImageJ2:%20imageJ%20for%20the%20next%20generation%20of%20scientific%20image%20data.&amp;author=C.%20T.%20Rueden&amp;author=J.%20Schindelin&amp;author=M.%20C.%20Hiner&amp;author=B.%20E.%20DeZonia&amp;author=A.%20E.%20Walter&amp;volume=18&amp;issue=529&amp;publication_year=2017&amp;pmid=29187165&amp;doi=10.1186/s12859-017-1934-z&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B79">
<cite>Rutherford R., Masson P. H. (1996). <em>Arabidopsis thaliana</em> sku mutant seedlings show exaggerated surface-dependent alteration in root growth vector.
<em>Plant Physiol.</em>
111
987–998. 10.1104/pp.111.4.987
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.111.4.987" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC160968/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8756492/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Arabidopsis%20thaliana%20sku%20mutant%20seedlings%20show%20exaggerated%20surface-dependent%20alteration%20in%20root%20growth%20vector.&amp;author=R.%20Rutherford&amp;author=P.%20H.%20Masson&amp;volume=111&amp;publication_year=1996&amp;pages=987-998&amp;pmid=8756492&amp;doi=10.1104/pp.111.4.987&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B80">
<cite>Ryabova L. A., Robaglia C., Meyer C. (2019). Target of rapamycin kinase: central regulatory hub for plant growth and metabolism.
<em>J. Exp. Bot.</em>
70
2211–2216. 10.1093/jxb/erz108
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/erz108" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6463030/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30984977/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=Target%20of%20rapamycin%20kinase:%20central%20regulatory%20hub%20for%20plant%20growth%20and%20metabolism.&amp;author=L.%20A.%20Ryabova&amp;author=C.%20Robaglia&amp;author=C.%20Meyer&amp;volume=70&amp;publication_year=2019&amp;pages=2211-2216&amp;pmid=30984977&amp;doi=10.1093/jxb/erz108&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B81">
<cite>Ryba-White M., Nedukha O., Hilaire E., Guikema J. A., Kordyum E., Leach J. E. (2001). Growth in microgravity increases susceptibility of soybean to a fungal pathogen.
<em>Plant Cell Physiol.</em>
42
657–664. 10.1093/pcp/pce082
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pce082" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11427686/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol.&amp;title=Growth%20in%20microgravity%20increases%20susceptibility%20of%20soybean%20to%20a%20fungal%20pathogen.&amp;author=M.%20Ryba-White&amp;author=O.%20Nedukha&amp;author=E.%20Hilaire&amp;author=J.%20A.%20Guikema&amp;author=E.%20Kordyum&amp;volume=42&amp;publication_year=2001&amp;pages=657-664&amp;pmid=11427686&amp;doi=10.1093/pcp/pce082&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B82">
<cite>Salmi M. L., Roux S. J. (2008). Gene expression changes induced by space flight in single-cells of the fern <em>Ceratopteris richardii</em>.
<em>Planta</em>
229
151–159. 10.1007/s00425-008-0817-y
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00425-008-0817-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18807069/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Gene%20expression%20changes%20induced%20by%20space%20flight%20in%20single-cells%20of%20the%20fern%20Ceratopteris%20richardii.&amp;author=M.%20L.%20Salmi&amp;author=S.%20J.%20Roux&amp;volume=229&amp;publication_year=2008&amp;pages=151-159&amp;pmid=18807069&amp;doi=10.1007/s00425-008-0817-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B83">
<cite>Schenke D., Utami H. P., Zhou Z., Gallegos M.-T., Cai D. (2019). Suppression of UV-B stress induced flavonoids by biotic stress: is there reciprocal crosstalk?
<em>Plant Physiol. Biochem.</em>
134
53–63. 10.1016/j.plaphy.2018.06.026
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.plaphy.2018.06.026" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30558728/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.%20Biochem.&amp;title=Suppression%20of%20UV-B%20stress%20induced%20flavonoids%20by%20biotic%20stress:%20is%20there%20reciprocal%20crosstalk?&amp;author=D.%20Schenke&amp;author=H.%20P.%20Utami&amp;author=Z.%20Zhou&amp;author=M.-T.%20Gallegos&amp;author=D.%20Cai&amp;volume=134&amp;publication_year=2019&amp;pages=53-63&amp;pmid=30558728&amp;doi=10.1016/j.plaphy.2018.06.026&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B84">
<cite>Schepetilnikov M., Makarian J., Srour O., Geldreich A., Yang Z., Chicher J., et al.  (2017). GTPase ROP2 binds and promotes activation of target of rapamycin, TOR, in response to auxin.
<em>EMBO J.</em>
36
886–903. 10.15252/embj.201694816
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.15252/embj.201694816" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5376970/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28246118/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&amp;title=GTPase%20ROP2%20binds%20and%20promotes%20activation%20of%20target%20of%20rapamycin,%20TOR,%20in%20response%20to%20auxin.&amp;author=M.%20Schepetilnikov&amp;author=J.%20Makarian&amp;author=O.%20Srour&amp;author=A.%20Geldreich&amp;author=Z.%20Yang&amp;volume=36&amp;publication_year=2017&amp;pages=886-903&amp;pmid=28246118&amp;doi=10.15252/embj.201694816&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B85">
<cite>Schindelin J., Arganda-Carreras I., Frise E., Kaynig V., Longair M., Pietzsch T., et al.  (2012). Fiji: an open-source platform for biological-image analysis.
<em>Nat. Methods</em>
9
676–682. 10.1038/nmeth.2019
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.2019" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3855844/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22743772/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Fiji:%20an%20open-source%20platform%20for%20biological-image%20analysis.&amp;author=J.%20Schindelin&amp;author=I.%20Arganda-Carreras&amp;author=E.%20Frise&amp;author=V.%20Kaynig&amp;author=M.%20Longair&amp;volume=9&amp;publication_year=2012&amp;pages=676-682&amp;pmid=22743772&amp;doi=10.1038/nmeth.2019&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B86">
<cite>Schultz E., Paul A.-L., Ferl R. (2016). Root growth patterns and morphometric change based on the growth media.
<em>Microgravity Sci. Technol.</em>
28
621–631. 10.1007/s12217-016-9514-9</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s12217-016-9514-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microgravity%20Sci.%20Technol.&amp;title=Root%20growth%20patterns%20and%20morphometric%20change%20based%20on%20the%20growth%20media.&amp;author=E.%20Schultz&amp;author=A.-L.%20Paul&amp;author=R.%20Ferl&amp;volume=28&amp;publication_year=2016&amp;pages=621-631&amp;doi=10.1007/s12217-016-9514-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B87">
<cite>Schultz E. R., Zupanska A. K., Sng N. J., Paul A.-L., Ferl R. J. (2017). Skewing in Arabidopsis roots involves disparate environmental signaling pathways.
<em>BMC Plant Biol.</em>
17:31. 10.1186/s12870-017-0975-9
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12870-017-0975-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5286820/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28143395/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biol.&amp;title=Skewing%20in%20Arabidopsis%20roots%20involves%20disparate%20environmental%20signaling%20pathways.&amp;author=E.%20R.%20Schultz&amp;author=A.%20K.%20Zupanska&amp;author=N.%20J.%20Sng&amp;author=A.-L.%20Paul&amp;author=R.%20J.%20Ferl&amp;volume=17&amp;issue=31&amp;publication_year=2017&amp;pmid=28143395&amp;doi=10.1186/s12870-017-0975-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B88">
<cite>Sedbrook J. C., Carroll K. L., Hung K. F., Masson P. H., Somerville C. R. (2002). The Arabidopsis <em>SKU5</em> gene encodes an extracellular glycosyl phosphatidylinositol-anchored glycoprotein involved in directional root growth.
<em>Plant Cell</em>
14
1635–1648. 10.1105/tpc.002360
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.002360" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC150712/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12119380/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20Arabidopsis%20SKU5%20gene%20encodes%20an%20extracellular%20glycosyl%20phosphatidylinositol-anchored%20glycoprotein%20involved%20in%20directional%20root%20growth.&amp;author=J.%20C.%20Sedbrook&amp;author=K.%20L.%20Carroll&amp;author=K.%20F.%20Hung&amp;author=P.%20H.%20Masson&amp;author=C.%20R.%20Somerville&amp;volume=14&amp;publication_year=2002&amp;pages=1635-1648&amp;pmid=12119380&amp;doi=10.1105/tpc.002360&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B89">
<cite>Sedbrook J. C., Ehrhardt D. W., Fisher S. E., Scheible W.-R., Somerville C. R. (2004). The Arabidopsis <em>SKU6/SPIRAL1</em> gene encodes a plus end–localized microtubule-interacting protein involved in directional cell expansion.
<em>Plant Cell</em>
16
1506–1520. 10.1105/tpc.020644
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.020644" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC490042/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15155883/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20Arabidopsis%20SKU6/SPIRAL1%20gene%20encodes%20a%20plus%20end%E2%80%93localized%20microtubule-interacting%20protein%20involved%20in%20directional%20cell%20expansion.&amp;author=J.%20C.%20Sedbrook&amp;author=D.%20W.%20Ehrhardt&amp;author=S.%20E.%20Fisher&amp;author=W.-R.%20Scheible&amp;author=C.%20R.%20Somerville&amp;volume=16&amp;publication_year=2004&amp;pages=1506-1520&amp;pmid=15155883&amp;doi=10.1105/tpc.020644&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B90">
<cite>Shabala S., Bose J., Fuglsang A. T., Pottosin I. (2015). On a quest for stress tolerance genes: membrane transporters in sensing and adapting to hostile soils.
<em>J. Exp. Bot.</em>
67
1015–1031. 10.1093/jxb/erv465
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/erv465" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26507891/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=On%20a%20quest%20for%20stress%20tolerance%20genes:%20membrane%20transporters%20in%20sensing%20and%20adapting%20to%20hostile%20soils.&amp;author=S.%20Shabala&amp;author=J.%20Bose&amp;author=A.%20T.%20Fuglsang&amp;author=I.%20Pottosin&amp;volume=67&amp;publication_year=2015&amp;pages=1015-1031&amp;pmid=26507891&amp;doi=10.1093/jxb/erv465&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B91">
<cite>Shagimardanova E. I., Gusev O. A., Sychev V. N., Levinskikh M. A., Sharipova M. R., Il’inskaya O. N., et al.  (2010). Expression of stress response genes in barley <em>Hordeum vulgare</em> in a spaceflight environment.
<em>Mol. Biol.</em>
44
734–740. 10.1134/s0026893310050080</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1134/s0026893310050080" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21090239/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.&amp;title=Expression%20of%20stress%20response%20genes%20in%20barley%20Hordeum%20vulgare%20in%20a%20spaceflight%20environment.&amp;author=E.%20I.%20Shagimardanova&amp;author=O.%20A.%20Gusev&amp;author=V.%20N.%20Sychev&amp;author=M.%20A.%20Levinskikh&amp;author=M.%20R.%20Sharipova&amp;volume=44&amp;publication_year=2010&amp;pages=734-740&amp;pmid=21090239&amp;doi=10.1134/s0026893310050080&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B92">
<cite>Shi Y., Yue X., An L. (2018). Integrated regulation triggered by a cryophyte ω-3 desaturase gene confers multiple-stress tolerance in tobacco.
<em>J. Exp. Bot.</em>
69
2131–2148. 10.1093/jxb/ery050
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/ery050" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6019038/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29432580/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=Integrated%20regulation%20triggered%20by%20a%20cryophyte%20%CF%89-3%20desaturase%20gene%20confers%20multiple-stress%20tolerance%20in%20tobacco.&amp;author=Y.%20Shi&amp;author=X.%20Yue&amp;author=L.%20An&amp;volume=69&amp;publication_year=2018&amp;pages=2131-2148&amp;pmid=29432580&amp;doi=10.1093/jxb/ery050&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B93">
<cite>Shimomura S. (2006). Identification of a glycosylphosphatidylinositol-anchored plasma membrane protein interacting with the C-terminus of auxin-binding protein 1: a photoaffinity crosslinking study.
<em>Plant Mol. Biol.</em>
60
663–677. 10.1007/s11103-005-5471-1
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11103-005-5471-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16649105/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol.%20Biol.&amp;title=Identification%20of%20a%20glycosylphosphatidylinositol-anchored%20plasma%20membrane%20protein%20interacting%20with%20the%20C-terminus%20of%20auxin-binding%20protein%201:%20a%20photoaffinity%20crosslinking%20study.&amp;author=S.%20Shimomura&amp;volume=60&amp;publication_year=2006&amp;pages=663-677&amp;pmid=16649105&amp;doi=10.1007/s11103-005-5471-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B94">
<cite>Shoji T., Suzuki K., Abe T., Kaneko Y., Shi H., Zhu J. K., et al.  (2006). Salt stress affects cortical microtubule organization and helical growth in Arabidopsis.
<em>Plant Cell Physiol.</em>
47
1158–1168. 10.1093/pcp/pcj090
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pcj090" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16861712/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol.&amp;title=Salt%20stress%20affects%20cortical%20microtubule%20organization%20and%20helical%20growth%20in%20Arabidopsis.&amp;author=T.%20Shoji&amp;author=K.%20Suzuki&amp;author=T.%20Abe&amp;author=Y.%20Kaneko&amp;author=H.%20Shi&amp;volume=47&amp;publication_year=2006&amp;pages=1158-1168&amp;pmid=16861712&amp;doi=10.1093/pcp/pcj090&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B95">
<cite>Signorelli S., Tarkowski Ł. P., Van den Ende W., Bassham D. C. (2019). Linking autophagy to abiotic and biotic stress responses.
<em>Trends Plant Sci.</em>
24
413–430. 10.1016/j.tplants.2019.02.001
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tplants.2019.02.001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6475611/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30824355/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci.&amp;title=Linking%20autophagy%20to%20abiotic%20and%20biotic%20stress%20responses.&amp;author=S.%20Signorelli&amp;author=%C5%81.%20P.%20Tarkowski&amp;author=W.%20Van%20den%20Ende&amp;author=D.%20C.%20Bassham&amp;volume=24&amp;publication_year=2019&amp;pages=413-430&amp;pmid=30824355&amp;doi=10.1016/j.tplants.2019.02.001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B96">
<cite>Simmons C., Söll D., Migliaccio F. (1995). Circumnutation and gravitropism cause root waving in <em>Arabidopsis thaliana</em>.
<em>J. Exp. Bot.</em>
46
143–150. 10.1093/jxb/46.1.143</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/46.1.143" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=Circumnutation%20and%20gravitropism%20cause%20root%20waving%20in%20Arabidopsis%20thaliana.&amp;author=C.%20Simmons&amp;author=D.%20S%C3%B6ll&amp;author=F.%20Migliaccio&amp;volume=46&amp;publication_year=1995&amp;pages=143-150&amp;doi=10.1093/jxb/46.1.143&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B97">
<cite>Smith M. B., Li H., Shen T., Huang X., Yusuf E., Vavylonis D. (2010). Segmentation and tracking of cytoskeletal filaments using open active contours.
<em>Cytoskeleton</em>
67
693–705. 10.1002/cm.20481
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/cm.20481" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3020657/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20814909/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cytoskeleton&amp;title=Segmentation%20and%20tracking%20of%20cytoskeletal%20filaments%20using%20open%20active%20contours.&amp;author=M.%20B.%20Smith&amp;author=H.%20Li&amp;author=T.%20Shen&amp;author=X.%20Huang&amp;author=E.%20Yusuf&amp;volume=67&amp;publication_year=2010&amp;pages=693-705&amp;pmid=20814909&amp;doi=10.1002/cm.20481&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B98">
<cite>Snedecor G. W., Cochran W. G. (1989). <em>Statistical Methods</em>, 8th Edn
Ames: Iowa State University Press.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Statistical%20Methods&amp;author=G.%20W.%20Snedecor&amp;author=W.%20G.%20Cochran&amp;publication_year=1989&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B99">
<cite>Sng N. J., Callaham J., Ferl R. J., Paul A.-L. (2014). <em>Arabidopsis thaliana</em> for spaceflight applications–preparing dormant biology for passive stowage and on orbit activation.
<em>Gravit. Space Res.</em>
2
81–89.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Gravit.%20Space%20Res.&amp;title=Arabidopsis%20thaliana%20for%20spaceflight%20applications%E2%80%93preparing%20dormant%20biology%20for%20passive%20stowage%20and%20on%20orbit%20activation.&amp;author=N.%20J.%20Sng&amp;author=J.%20Callaham&amp;author=R.%20J.%20Ferl&amp;author=A.-L.%20Paul&amp;volume=2&amp;publication_year=2014&amp;pages=81-89&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B100">
<cite>Sng N. J., Kolaczkowski B., Ferl R. J., Paul A.-L. (2018). A member of the CONSTANS-like protein family is a putative regulator of reactive oxygen species homeostasis and spaceflight physiological adaptation.
<em>Ann. Bot. Plants</em>
11:ply075. 10.1093/aobpla/ply075
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/aobpla/ply075" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6348315/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30705745/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Ann.%20Bot.%20Plants&amp;title=A%20member%20of%20the%20CONSTANS-like%20protein%20family%20is%20a%20putative%20regulator%20of%20reactive%20oxygen%20species%20homeostasis%20and%20spaceflight%20physiological%20adaptation.&amp;author=N.%20J.%20Sng&amp;author=B.%20Kolaczkowski&amp;author=R.%20J.%20Ferl&amp;author=A.-L.%20Paul&amp;volume=11&amp;issue=ply075&amp;publication_year=2018&amp;pmid=30705745&amp;doi=10.1093/aobpla/ply075&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B101">
<cite>Soga K., Yamazaki C., Kamada M., Tanigawa N., Kasahara H., Yano S., et al.  (2018). Modification of growth anisotropy and cortical microtubule dynamics in Arabidopsis hypocotyls grown under microgravity conditions in space.
<em>Physiol. Plant.</em>
162
135–144. 10.1111/ppl.12640
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/ppl.12640" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28862767/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Physiol.%20Plant.&amp;title=Modification%20of%20growth%20anisotropy%20and%20cortical%20microtubule%20dynamics%20in%20Arabidopsis%20hypocotyls%20grown%20under%20microgravity%20conditions%20in%20space.&amp;author=K.%20Soga&amp;author=C.%20Yamazaki&amp;author=M.%20Kamada&amp;author=N.%20Tanigawa&amp;author=H.%20Kasahara&amp;volume=162&amp;publication_year=2018&amp;pages=135-144&amp;pmid=28862767&amp;doi=10.1111/ppl.12640&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B102">
<cite>Solanke A. U., Sharma A. K. (2008). Signal transduction during cold stress in plants.
<em>Physiol. Mol. Biol. Plants</em>
14
69–79. 10.1007/s12298-008-0006-2
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s12298-008-0006-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3550661/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23572874/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Physiol.%20Mol.%20Biol.%20Plants&amp;title=Signal%20transduction%20during%20cold%20stress%20in%20plants.&amp;author=A.%20U.%20Solanke&amp;author=A.%20K.%20Sharma&amp;volume=14&amp;publication_year=2008&amp;pages=69-79&amp;pmid=23572874&amp;doi=10.1007/s12298-008-0006-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B103">
<cite>Sugimoto M., Oono Y., Gusev O., Matsumoto T., Yazawa T., Levinskikh M. A., et al.  (2014). Genome-wide expression analysis of reactive oxygen species gene network in Mizuna plants grown in long-term spaceflight.
<em>BMC Plant Biol.</em>
14:4. 10.1186/1471-2229-14-4
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2229-14-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3927260/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24393219/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biol.&amp;title=Genome-wide%20expression%20analysis%20of%20reactive%20oxygen%20species%20gene%20network%20in%20Mizuna%20plants%20grown%20in%20long-term%20spaceflight.&amp;author=M.%20Sugimoto&amp;author=Y.%20Oono&amp;author=O.%20Gusev&amp;author=T.%20Matsumoto&amp;author=T.%20Yazawa&amp;volume=14&amp;issue=4&amp;publication_year=2014&amp;pmid=24393219&amp;doi=10.1186/1471-2229-14-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B104">
<cite>Suzuki N., Bassil E., Hamilton J. S., Inupakutika M. A., Zandalinas S. I., Tripathy D., et al.  (2016). ABA is required for plant acclimation to a combination of salt and heat stress.
<em>PLoS One</em>
11:e0147625. 10.1371/journal.pone.0147625
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0147625" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4733103/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26824246/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=ABA%20is%20required%20for%20plant%20acclimation%20to%20a%20combination%20of%20salt%20and%20heat%20stress.&amp;author=N.%20Suzuki&amp;author=E.%20Bassil&amp;author=J.%20S.%20Hamilton&amp;author=M.%20A.%20Inupakutika&amp;author=S.%20I.%20Zandalinas&amp;volume=11&amp;issue=e0147625&amp;publication_year=2016&amp;pmid=26824246&amp;doi=10.1371/journal.pone.0147625&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B105">
<cite>Swarbreck S. M., Guerringue Y., Matthus E., Jamieson F. J. C., Davies J. M. (2019). Impairment in karrikin but not strigolactone sensing enhances root skewing in <em>Arabidopsis thaliana</em>.
<em>Plant J.</em>
98
607–621. 10.1111/tpj.14233
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.14233" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6563046/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30659713/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=Impairment%20in%20karrikin%20but%20not%20strigolactone%20sensing%20enhances%20root%20skewing%20in%20Arabidopsis%20thaliana.&amp;author=S.%20M.%20Swarbreck&amp;author=Y.%20Guerringue&amp;author=E.%20Matthus&amp;author=F.%20J.%20C.%20Jamieson&amp;author=J.%20M.%20Davies&amp;volume=98&amp;publication_year=2019&amp;pages=607-621&amp;pmid=30659713&amp;doi=10.1111/tpj.14233&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B106">
<cite>Tai H. H., Tai G. C., Beardmore T. (2005). Dynamic histone acetylation of late embryonic genes during seed germination.
<em>Plant Mol. Biol.</em>
59
909–925. 10.1007/s11103-005-2081-x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11103-005-2081-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16307366/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol.%20Biol.&amp;title=Dynamic%20histone%20acetylation%20of%20late%20embryonic%20genes%20during%20seed%20germination.&amp;author=H.%20H.%20Tai&amp;author=G.%20C.%20Tai&amp;author=T.%20Beardmore&amp;volume=59&amp;publication_year=2005&amp;pages=909-925&amp;pmid=16307366&amp;doi=10.1007/s11103-005-2081-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B107">
<cite>Takahashi D., Gorka M., Erban A., Graf A., Kopka J., Zuther E., et al.  (2019). Both cold and sub-zero acclimation induce cell wall modification and changes in the extracellular proteome in <em>Arabidopsis thaliana</em>.
<em>Sci. Rep.</em>
9:2289. 10.1038/s41598-019-38688-3
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-019-38688-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6381082/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30783145/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Both%20cold%20and%20sub-zero%20acclimation%20induce%20cell%20wall%20modification%20and%20changes%20in%20the%20extracellular%20proteome%20in%20Arabidopsis%20thaliana.&amp;author=D.%20Takahashi&amp;author=M.%20Gorka&amp;author=A.%20Erban&amp;author=A.%20Graf&amp;author=J.%20Kopka&amp;volume=9&amp;issue=2289&amp;publication_year=2019&amp;pmid=30783145&amp;doi=10.1038/s41598-019-38688-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B108">
<cite>Takahashi D., Kawamura Y., Uemura M. (2013). Detergent-resistant plasma membrane proteome to elucidate microdomain functions in plant cells.
<em>Front. Plant Sci.</em>
4:27. 10.3389/fpls.2013.00027
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2013.00027" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3579295/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23440896/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=Detergent-resistant%20plasma%20membrane%20proteome%20to%20elucidate%20microdomain%20functions%20in%20plant%20cells.&amp;author=D.%20Takahashi&amp;author=Y.%20Kawamura&amp;author=M.%20Uemura&amp;volume=4&amp;issue=27&amp;publication_year=2013&amp;pmid=23440896&amp;doi=10.3389/fpls.2013.00027&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B109">
<cite>Takahashi D., Kawamura Y., Uemura M. (2016). Cold acclimation is accompanied by complex responses of glycosylphosphatidylinositol (GPI)-anchored proteins in Arabidopsis.
<em>J. Exp. Bot.</em>
67
5203–5215. 10.1093/jxb/erw279
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/erw279" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5014161/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27471282/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=Cold%20acclimation%20is%20accompanied%20by%20complex%20responses%20of%20glycosylphosphatidylinositol%20(GPI)-anchored%20proteins%20in%20Arabidopsis.&amp;author=D.%20Takahashi&amp;author=Y.%20Kawamura&amp;author=M.%20Uemura&amp;volume=67&amp;publication_year=2016&amp;pages=5203-5215&amp;pmid=27471282&amp;doi=10.1093/jxb/erw279&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B110">
<cite>Tanaka M., Sotta N., Yamazumi Y., Yamashita Y., Miwa K., Murota K., et al.  (2016). The minimum open reading frame, AUG-stop, induces boron-dependent ribosome stalling and mRNA degradation.
<em>Plant Cell</em>
28
2830–2849. 10.1105/tpc.16.00481
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.16.00481" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5155345/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27760805/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20minimum%20open%20reading%20frame,%20AUG-stop,%20induces%20boron-dependent%20ribosome%20stalling%20and%20mRNA%20degradation.&amp;author=M.%20Tanaka&amp;author=N.%20Sotta&amp;author=Y.%20Yamazumi&amp;author=Y.%20Yamashita&amp;author=K.%20Miwa&amp;volume=28&amp;publication_year=2016&amp;pages=2830-2849&amp;pmid=27760805&amp;doi=10.1105/tpc.16.00481&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B111">
<cite>Tang X., Lim M. H., Pelletier J., Tang M., Nguyen V., Keller W. A., et al.  (2012). Synergistic repression of the embryonic programme by SET DOMAIN GROUP 8 and EMBRYONIC FLOWER 2 in Arabidopsis seedlings.
<em>J. Exp. Bot.</em>
63
1391–1404. 10.1093/jxb/err383
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/err383" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3276103/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22162868/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=Synergistic%20repression%20of%20the%20embryonic%20programme%20by%20SET%20DOMAIN%20GROUP%208%20and%20EMBRYONIC%20FLOWER%202%20in%20Arabidopsis%20seedlings.&amp;author=X.%20Tang&amp;author=M.%20H.%20Lim&amp;author=J.%20Pelletier&amp;author=M.%20Tang&amp;author=V.%20Nguyen&amp;volume=63&amp;publication_year=2012&amp;pages=1391-1404&amp;pmid=22162868&amp;doi=10.1093/jxb/err383&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B112">
<cite>Tarazona S., García-Alcalde F., Dopazo J., Ferrer A., Conesa A. (2011). Differential expression in RNA-seq: a matter of depth.
<em>Genome Res.</em>
21
2213–2223. 10.1101/gr.124321.111
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.124321.111" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3227109/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21903743/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Differential%20expression%20in%20RNA-seq:%20a%20matter%20of%20depth.&amp;author=S.%20Tarazona&amp;author=F.%20Garc%C3%ADa-Alcalde&amp;author=J.%20Dopazo&amp;author=A.%20Ferrer&amp;author=A.%20Conesa&amp;volume=21&amp;publication_year=2011&amp;pages=2213-2223&amp;pmid=21903743&amp;doi=10.1101/gr.124321.111&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B113">
<cite>Trapnell C., Hendrickson D. G., Sauvageau M., Goff L., Rinn J. L., Pachter L. (2012). Differential analysis of gene regulation at transcript resolution with RNA-seq.
<em>Nat. Biotechnol.</em>
31
46–53. 10.1038/nbt.2450
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nbt.2450" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3869392/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23222703/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=Differential%20analysis%20of%20gene%20regulation%20at%20transcript%20resolution%20with%20RNA-seq.&amp;author=C.%20Trapnell&amp;author=D.%20G.%20Hendrickson&amp;author=M.%20Sauvageau&amp;author=L.%20Goff&amp;author=J.%20L.%20Rinn&amp;volume=31&amp;publication_year=2012&amp;pages=46-53&amp;pmid=23222703&amp;doi=10.1038/nbt.2450&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B114">
<cite>Trapnell C., Williams B. A., Pertea G., Mortazavi A., Kwan G., van Baren M. J., et al.  (2010). Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.
<em>Nat. Biotechnol.</em>
28
511–515. 10.1038/nbt.1621
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nbt.1621" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3146043/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20436464/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation.&amp;author=C.%20Trapnell&amp;author=B.%20A.%20Williams&amp;author=G.%20Pertea&amp;author=A.%20Mortazavi&amp;author=G.%20Kwan&amp;volume=28&amp;publication_year=2010&amp;pages=511-515&amp;pmid=20436464&amp;doi=10.1038/nbt.1621&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B115">
<cite>Ueda H., Kusaba M. (2015). Strigolactone regulates leaf senescence in concert with ethylene in Arabidopsis.
<em>Plant Physiol.</em>
169
138–147. 10.1104/pp.15.00325
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.15.00325" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4577378/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25979917/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Strigolactone%20regulates%20leaf%20senescence%20in%20concert%20with%20ethylene%20in%20Arabidopsis.&amp;author=H.%20Ueda&amp;author=M.%20Kusaba&amp;volume=169&amp;publication_year=2015&amp;pages=138-147&amp;pmid=25979917&amp;doi=10.1104/pp.15.00325&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B116">
<cite>Vandenbrink J. P., Kiss J. Z., Herranz R., Medina F. J. (2014). Light and gravity signals synergize in modulating plant development.
<em>Front. Plant Sci.</em>
5:563. 10.3389/fpls.2014.00563
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2014.00563" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4211383/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25389428/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=Light%20and%20gravity%20signals%20synergize%20in%20modulating%20plant%20development.&amp;author=J.%20P.%20Vandenbrink&amp;author=J.%20Z.%20Kiss&amp;author=R.%20Herranz&amp;author=F.%20J.%20Medina&amp;volume=5&amp;issue=563&amp;publication_year=2014&amp;pmid=25389428&amp;doi=10.3389/fpls.2014.00563&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B117">
<cite>Wang S., Kurepa J., Hashimoto T., Smalle J. A. (2011). Salt stress–induced disassembly of Arabidopsis cortical microtubule arrays involves 26S proteasome–dependent degradation of SPIRAL1.
<em>Plant Cell</em>
23
3412–3427. 10.1105/tpc.111.089920
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.111.089920" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3203425/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21954463/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Salt%20stress%E2%80%93induced%20disassembly%20of%20Arabidopsis%20cortical%20microtubule%20arrays%20involves%2026S%20proteasome%E2%80%93dependent%20degradation%20of%20SPIRAL1.&amp;author=S.%20Wang&amp;author=J.%20Kurepa&amp;author=T.%20Hashimoto&amp;author=J.%20A.%20Smalle&amp;volume=23&amp;publication_year=2011&amp;pages=3412-3427&amp;pmid=21954463&amp;doi=10.1105/tpc.111.089920&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B118">
<cite>Wickham H. (2016). <em>ggplot2: Elegant Graphics for Data Analysis.</em>
New York, NY: Springer-Verlag.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=ggplot2:%20Elegant%20Graphics%20for%20Data%20Analysis.&amp;author=H.%20Wickham&amp;publication_year=2016&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B119">
<cite>Xiang Y., Nakabayashi K., Ding J., He F., Bentsink L., Soppe W. J. J. (2014). REDUCED DORMANCY5 encodes a protein phosphatase 2C that is required for seed dormancy in Arabidopsis.
<em>Plant Cell</em>
26
4362–4375. 10.1105/tpc.114.132811
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.114.132811" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4277229/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25415980/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=REDUCED%20DORMANCY5%20encodes%20a%20protein%20phosphatase%202C%20that%20is%20required%20for%20seed%20dormancy%20in%20Arabidopsis.&amp;author=Y.%20Xiang&amp;author=K.%20Nakabayashi&amp;author=J.%20Ding&amp;author=F.%20He&amp;author=L.%20Bentsink&amp;volume=26&amp;publication_year=2014&amp;pages=4362-4375&amp;pmid=25415980&amp;doi=10.1105/tpc.114.132811&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B120">
<cite>Xu T., Dai N., Chen J., Nagawa S., Cao M., Li H., et al.  (2014). Cell surface ABP1-TMK auxin-sensing complex activates ROP GTPase signaling.
<em>Science</em>
343
1025–1028. 10.1126/science.1245125
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1245125" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4166562/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24578577/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Cell%20surface%20ABP1-TMK%20auxin-sensing%20complex%20activates%20ROP%20GTPase%20signaling.&amp;author=T.%20Xu&amp;author=N.%20Dai&amp;author=J.%20Chen&amp;author=S.%20Nagawa&amp;author=M.%20Cao&amp;volume=343&amp;publication_year=2014&amp;pages=1025-1028&amp;pmid=24578577&amp;doi=10.1126/science.1245125&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B121">
<cite>Yamauchi Y., Hasegawa A., Taninaka A., Mizutani M., Sugimoto Y. (2011). NADPH-dependent reductases involved in the detoxification of reactive carbonyls in plants.
<em>J. Biol. Chem.</em>
286
6999–7009. 10.1074/jbc.m110.202226
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.m110.202226" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3044956/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21169366/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=NADPH-dependent%20reductases%20involved%20in%20the%20detoxification%20of%20reactive%20carbonyls%20in%20plants.&amp;author=Y.%20Yamauchi&amp;author=A.%20Hasegawa&amp;author=A.%20Taninaka&amp;author=M.%20Mizutani&amp;author=Y.%20Sugimoto&amp;volume=286&amp;publication_year=2011&amp;pages=6999-7009&amp;pmid=21169366&amp;doi=10.1074/jbc.m110.202226&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B122">
<cite>Yates F. (1934). Contingency tables involving small numbers and the χ2 test.
<em>Suppl. J. R. Stat. Soc.</em>
1
217–235. 10.2307/2983604</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2307/2983604" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Suppl.%20J.%20R.%20Stat.%20Soc.&amp;title=Contingency%20tables%20involving%20small%20numbers%20and%20the%20%CF%872%20test.&amp;author=F.%20Yates&amp;volume=1&amp;publication_year=1934&amp;pages=217-235&amp;doi=10.2307/2983604&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B123">
<cite>Yeats T. H., Bacic A., Johnson K. L. (2018). Plant glycosylphosphatidylinositol anchored proteins at the plasma membrane-cell wall nexus.
<em>J. Integr. Plant Biol.</em>
60
649–669. 10.1111/jipb.12659
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/jipb.12659" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29667761/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Integr.%20Plant%20Biol.&amp;title=Plant%20glycosylphosphatidylinositol%20anchored%20proteins%20at%20the%20plasma%20membrane-cell%20wall%20nexus.&amp;author=T.%20H.%20Yeats&amp;author=A.%20Bacic&amp;author=K.%20L.%20Johnson&amp;volume=60&amp;publication_year=2018&amp;pages=649-669&amp;pmid=29667761&amp;doi=10.1111/jipb.12659&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B124">
<cite>Zeilmaker T., Ludwig N. R., Elberse J., Seidl M. F., Berke L., Van Doorn A., et al.  (2015). DOWNY MILDEW RESISTANT 6 and DMR6-LIKE OXYGENASE 1 are partially redundant but distinct suppressors of immunity in Arabidopsis.
<em>Plant J.</em>
81
210–222. 10.1111/tpj.12719
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.12719" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25376907/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=DOWNY%20MILDEW%20RESISTANT%206%20and%20DMR6-LIKE%20OXYGENASE%201%20are%20partially%20redundant%20but%20distinct%20suppressors%20of%20immunity%20in%20Arabidopsis.&amp;author=T.%20Zeilmaker&amp;author=N.%20R.%20Ludwig&amp;author=J.%20Elberse&amp;author=M.%20F.%20Seidl&amp;author=L.%20Berke&amp;volume=81&amp;publication_year=2015&amp;pages=210-222&amp;pmid=25376907&amp;doi=10.1111/tpj.12719&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B125">
<cite>Zhang K., Halitschke R., Yin C., Liu C.-J., Gan S.-S. (2013). Salicylic acid 3-hydroxylase regulates <em>Arabidopsis</em> leaf longevity by mediating salicylic acid catabolism.
<em>Proc. Natl. Acad. Sci. U.S.A.</em>
110
14807–14812. 10.1073/pnas.1302702110
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1302702110" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3767541/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23959884/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Salicylic%20acid%203-hydroxylase%20regulates%20Arabidopsis%20leaf%20longevity%20by%20mediating%20salicylic%20acid%20catabolism.&amp;author=K.%20Zhang&amp;author=R.%20Halitschke&amp;author=C.%20Yin&amp;author=C.-J.%20Liu&amp;author=S.-S.%20Gan&amp;volume=110&amp;publication_year=2013&amp;pages=14807-14812&amp;pmid=23959884&amp;doi=10.1073/pnas.1302702110&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B126">
<cite>Zhao P., Liu F., Ma M., Gong J., Wang Q., Jia P., et al.  (2011). Overexpression of AtLEA3-3 confers resistance to cold stress in <em>Escherichia coli</em> and provides enhanced osmotic stress tolerance and ABA sensitivity in <em>Arabidopsis thaliana</em>.
<em>Mol. Biol.</em>
45
851–862. 10.1134/S0026893311050165
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1134/S0026893311050165" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22393782/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.&amp;title=Overexpression%20of%20AtLEA3-3%20confers%20resistance%20to%20cold%20stress%20in%20Escherichia%20coli%20and%20provides%20enhanced%20osmotic%20stress%20tolerance%20and%20ABA%20sensitivity%20in%20Arabidopsis%20thaliana.&amp;author=P.%20Zhao&amp;author=F.%20Liu&amp;author=M.%20Ma&amp;author=J.%20Gong&amp;author=Q.%20Wang&amp;volume=45&amp;publication_year=2011&amp;pages=851-862&amp;pmid=22393782&amp;doi=10.1134/S0026893311050165&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B127">
<cite>Zhou M., Sng N. J., LeFrois C. E., Paul A.-L., Ferl R. J. (2019). Epigenomics in an extraterrestrial environment: organ-specific alteration of DNA methylation and gene expression elicited by spaceflight in <em>Arabidopsis thaliana</em>.
<em>BMC Genomics</em>
20:205. 10.1186/s12864-019-5554-z
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12864-019-5554-z" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6416986/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30866818/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Epigenomics%20in%20an%20extraterrestrial%20environment:%20organ-specific%20alteration%20of%20DNA%20methylation%20and%20gene%20expression%20elicited%20by%20spaceflight%20in%20Arabidopsis%20thaliana.&amp;author=M.%20Zhou&amp;author=N.%20J.%20Sng&amp;author=C.%20E.%20LeFrois&amp;author=A.-L.%20Paul&amp;author=R.%20J.%20Ferl&amp;volume=20&amp;issue=205&amp;publication_year=2019&amp;pmid=30866818&amp;doi=10.1186/s12864-019-5554-z&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B128">
<cite>Zupanska A. K., Denison F. C., Ferl R. J., Paul A.-L. (2013). Spaceflight engages heat shock protein and other molecular chaperone genes in tissue culture cells of <em>Arabidopsis thaliana</em>.
<em>Am. J. Bot.</em>
100
235–248. 10.3732/ajb.1200343
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3732/ajb.1200343" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23258370/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Bot.&amp;title=Spaceflight%20engages%20heat%20shock%20protein%20and%20other%20molecular%20chaperone%20genes%20in%20tissue%20culture%20cells%20of%20Arabidopsis%20thaliana.&amp;author=A.%20K.%20Zupanska&amp;author=F.%20C.%20Denison&amp;author=R.%20J.%20Ferl&amp;author=A.-L.%20Paul&amp;volume=100&amp;publication_year=2013&amp;pages=235-248&amp;pmid=23258370&amp;doi=10.3732/ajb.1200343&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B129">
<cite>Zupanska A. K., LeFrois C., Ferl R. J., Paul A.-L. (2019). HSFA2 functions in the physiological adaptation of undifferentiated plant cells to spaceflight.
<em>Int. J. Mol. Sci.</em>
20:E390. 10.3390/ijms20020390
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/ijms20020390" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6359015/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30658467/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=HSFA2%20functions%20in%20the%20physiological%20adaptation%20of%20undifferentiated%20plant%20cells%20to%20spaceflight.&amp;author=A.%20K.%20Zupanska&amp;author=C.%20LeFrois&amp;author=R.%20J.%20Ferl&amp;author=A.-L.%20Paul&amp;volume=20&amp;issue=E390&amp;publication_year=2019&amp;pmid=30658467&amp;doi=10.3390/ijms20020390&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B130">
<cite>Zupanska A. K., Schultz E. R., Yao J., Sng N. J., Zhou M., Callaham J. B., et al.  (2017). ARG1 functions in the physiological adaptation of undifferentiated plant cells to spaceflight.
<em>Astrobiology</em>
17
1077–1111. 10.1089/ast.2016.1538
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2016.1538" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8024390/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29088549/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=ARG1%20functions%20in%20the%20physiological%20adaptation%20of%20undifferentiated%20plant%20cells%20to%20spaceflight.&amp;author=A.%20K.%20Zupanska&amp;author=E.%20R.%20Schultz&amp;author=J.%20Yao&amp;author=N.%20J.%20Sng&amp;author=M.%20Zhou&amp;volume=17&amp;publication_year=2017&amp;pages=1077-1111&amp;pmid=29088549&amp;doi=10.1089/ast.2016.1538&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ol></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/7064724/bin/Data_Sheet_1.zip">Click here for additional data file.</a><sup> (387.6KB, zip) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated and analyzed, composed of the RNA-Seq data and full output of the differential gene expression pipeline, for this study can be found in the Gene Expression Omnibus (GEO – <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/geo/" rel="noopener noreferrer" target="_blank">https://www.ncbi.nlm.nih.gov/geo/</a>) under the accessions <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95620" rel="noopener noreferrer" target="_blank">GSE95620</a> (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?&amp;acc=GSE95620" rel="noopener noreferrer" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?&amp;acc=GSE95620</a>) and <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95582" rel="noopener noreferrer" target="_blank">GSE95582</a> (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95582" rel="noopener noreferrer" target="_blank">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95582</a>), and also in NASA’s GeneLab Database (GLDS-218). A presentation outlining the experimental process and which contains video data of the harvests can be found at the NASA LSDA (Life Sciences Data Archive (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://lsda.jsc.nasa.gov/Dataset/dataset_info/13840" rel="noopener noreferrer" target="_blank">https://lsda.jsc.nasa.gov/Dataset/dataset_info/13840</a>).</p></section></section></section></section></article>
</main></body></html>