<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11942576/"/><title>Genome-Wide Analysis of the PERK Gene Family in Brassica napus L. and Their Potential Roles in Clubroot Disease - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Int J Mol Sci</button></div>. 2025 Mar 17;26(6):2685. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/ijms26062685" rel="noopener noreferrer" target="_blank">10.3390/ijms26062685</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genome-Wide Analysis of the <em>PERK</em> Gene Family in <em>Brassica napus</em> L. and Their Potential Roles in Clubroot Disease</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Z%22%5BAuthor%5D"><span class="name western">Zeyu Zhang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zeyu Zhang</span></h3>
<div class="p">
<sup>1</sup>The Key Laboratory of Biology and Genetic Improvement of Oil Crops, The Ministry of Agriculture and Rural Affairs of the PRC, Oil Crops Research Institute, Chinese Academy of Agricultural Sciences, Wuhan 430062, China; 20241015050@stu.scau.edu.cn (Z.Z.); futongyv@163.com (T.F.); </div>
<div>Investigation, Data curation, Writing – original draft, Visualization</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Z%22%5BAuthor%5D"><span class="name western">Zeyu Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20T%22%5BAuthor%5D"><span class="name western">Tongyu Fu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Tongyu Fu</span></h3>
<div class="p">
<sup>1</sup>The Key Laboratory of Biology and Genetic Improvement of Oil Crops, The Ministry of Agriculture and Rural Affairs of the PRC, Oil Crops Research Institute, Chinese Academy of Agricultural Sciences, Wuhan 430062, China; 20241015050@stu.scau.edu.cn (Z.Z.); futongyv@163.com (T.F.); </div>
<div>Investigation, Visualization</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20T%22%5BAuthor%5D"><span class="name western">Tongyu Fu</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20C%22%5BAuthor%5D"><span class="name western">Cong Zhou</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Cong Zhou</span></h3>
<div class="p">
<sup>1</sup>The Key Laboratory of Biology and Genetic Improvement of Oil Crops, The Ministry of Agriculture and Rural Affairs of the PRC, Oil Crops Research Institute, Chinese Academy of Agricultural Sciences, Wuhan 430062, China; 20241015050@stu.scau.edu.cn (Z.Z.); futongyv@163.com (T.F.); </div>
<div>Investigation</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20C%22%5BAuthor%5D"><span class="name western">Cong Zhou</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20F%22%5BAuthor%5D"><span class="name western">Fan Liu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Fan Liu</span></h3>
<div class="p">
<sup>1</sup>The Key Laboratory of Biology and Genetic Improvement of Oil Crops, The Ministry of Agriculture and Rural Affairs of the PRC, Oil Crops Research Institute, Chinese Academy of Agricultural Sciences, Wuhan 430062, China; 20241015050@stu.scau.edu.cn (Z.Z.); futongyv@163.com (T.F.); </div>
<div>Project administration</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20F%22%5BAuthor%5D"><span class="name western">Fan Liu</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeng%20L%22%5BAuthor%5D"><span class="name western">Lingyi Zeng</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Lingyi Zeng</span></h3>
<div class="p">
<sup>1</sup>The Key Laboratory of Biology and Genetic Improvement of Oil Crops, The Ministry of Agriculture and Rural Affairs of the PRC, Oil Crops Research Institute, Chinese Academy of Agricultural Sciences, Wuhan 430062, China; 20241015050@stu.scau.edu.cn (Z.Z.); futongyv@163.com (T.F.); </div>
<div>Resources</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeng%20L%22%5BAuthor%5D"><span class="name western">Lingyi Zeng</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ren%20L%22%5BAuthor%5D"><span class="name western">Li Ren</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Li Ren</span></h3>
<div class="p">
<sup>1</sup>The Key Laboratory of Biology and Genetic Improvement of Oil Crops, The Ministry of Agriculture and Rural Affairs of the PRC, Oil Crops Research Institute, Chinese Academy of Agricultural Sciences, Wuhan 430062, China; 20241015050@stu.scau.edu.cn (Z.Z.); futongyv@163.com (T.F.); </div>
<div>Validation</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ren%20L%22%5BAuthor%5D"><span class="name western">Li Ren</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tong%20C%22%5BAuthor%5D"><span class="name western">Chaobo Tong</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Chaobo Tong</span></h3>
<div class="p">
<sup>1</sup>The Key Laboratory of Biology and Genetic Improvement of Oil Crops, The Ministry of Agriculture and Rural Affairs of the PRC, Oil Crops Research Institute, Chinese Academy of Agricultural Sciences, Wuhan 430062, China; 20241015050@stu.scau.edu.cn (Z.Z.); futongyv@163.com (T.F.); </div>
<div>Supervision, Funding acquisition</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tong%20C%22%5BAuthor%5D"><span class="name western">Chaobo Tong</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20L%22%5BAuthor%5D"><span class="name western">Lijiang Liu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Lijiang Liu</span></h3>
<div class="p">
<sup>1</sup>The Key Laboratory of Biology and Genetic Improvement of Oil Crops, The Ministry of Agriculture and Rural Affairs of the PRC, Oil Crops Research Institute, Chinese Academy of Agricultural Sciences, Wuhan 430062, China; 20241015050@stu.scau.edu.cn (Z.Z.); futongyv@163.com (T.F.); </div>
<div>Supervision</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20L%22%5BAuthor%5D"><span class="name western">Lijiang Liu</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id9" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20L%22%5BAuthor%5D"><span class="name western">Li Xu</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Li Xu</span></h3>
<div class="p">
<sup>1</sup>The Key Laboratory of Biology and Genetic Improvement of Oil Crops, The Ministry of Agriculture and Rural Affairs of the PRC, Oil Crops Research Institute, Chinese Academy of Agricultural Sciences, Wuhan 430062, China; 20241015050@stu.scau.edu.cn (Z.Z.); futongyv@163.com (T.F.); </div>
<div>Writing – original draft, Writing – review &amp; editing, Funding acquisition</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20L%22%5BAuthor%5D"><span class="name western">Li Xu</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Richard R-C Wang</span><sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="af1-ijms-26-02685">
<sup>1</sup>The Key Laboratory of Biology and Genetic Improvement of Oil Crops, The Ministry of Agriculture and Rural Affairs of the PRC, Oil Crops Research Institute, Chinese Academy of Agricultural Sciences, Wuhan 430062, China; 20241015050@stu.scau.edu.cn (Z.Z.); futongyv@163.com (T.F.); </div>
<div class="author-notes p">
<div class="fn" id="c1-ijms-26-02685">
<sup>*</sup><p class="display-inline">Correspondence: <span>xuli01@caas.cn</span></p>
</div>
<div class="fn" id="fn1-ijms-26-02685">
<sup>†</sup><p class="display-inline">These authors contributed equally to this work.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Zeyu Zhang</span></strong>: <span class="role">Investigation, Data curation, Writing – original draft, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Tongyu Fu</span></strong>: <span class="role">Investigation, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Cong Zhou</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Fan Liu</span></strong>: <span class="role">Project administration</span>
</div>
<div>
<strong class="contrib"><span class="name western">Lingyi Zeng</span></strong>: <span class="role">Resources</span>
</div>
<div>
<strong class="contrib"><span class="name western">Li Ren</span></strong>: <span class="role">Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Chaobo Tong</span></strong>: <span class="role">Supervision, Funding acquisition</span>
</div>
<div>
<strong class="contrib"><span class="name western">Lijiang Liu</span></strong>: <span class="role">Supervision</span>
</div>
<div>
<strong class="contrib"><span class="name western">Li Xu</span></strong>: <span class="role">Writing – original draft, Writing – review &amp; editing, Funding acquisition</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Richard R-C Wang</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2024 Dec 28; Revised 2025 Mar 2; Accepted 2025 Mar 11; Collection date 2025 Mar.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© 2025 by the authors.</div>
<p>Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>).</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11942576  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/40141326/">40141326</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The proline-rich extensin-like receptor kinase (<em>PERK</em>) gene family is crucial to various molecular and cellular processes in plants. We identified 50 <em>PERK</em> genes in <em>Brassica napus</em> to explore their evolutionary dynamics, structural diversity, and functional roles. These genes were grouped into four classes and unevenly distributed across 18 chromosomes. Phylogenetic studies and Ka/Ks ratios revealed purifying selection during the evolution process. They exhibited significant diversification in gene length, molecular weight, and isoelectric points, suggesting specialized function. Gene structure and motif analyses revealed variations among the <em>BnPERK</em> family members, with conserved tyrosine kinase domains suggesting functional importance. <em>Cis</em>-element analysis predicted the involvement in hormone signaling and stress responses. Expression profiling showed diverse patterns across tissues and hormone treatments, highlighting potential roles in growth regulation and hormone signaling. Protein–protein interaction networks suggested BnPERK proteins interact with a wide array of proteins, implicating them in multiple biological processes. The transcriptional downregulation of four <em>BnPERK</em> genes upon <em>Plasmodiophora brassicae</em> infection implied a role in clubroot disease response. Furthermore, the Arabidopsis <em>perk9</em> mutant displayed relieved disease severity and enhanced basal immune response, suggesting the negative role of <em>PERK9</em> in plant immunity. The study highlighted the potential role of <em>BnPERKs</em> in crop improvement strategies against clubroot disease.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> <em>Brassica napus</em>, <em>PERK</em> gene family, receptor-like kinase, hormones, clubroot disease</p></section></section><section id="sec1-ijms-26-02685"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Receptor-like kinases (RLKs) constitute the largest plant receptor family and are located on the cell surface to perceive and transduce a wide range of signals [<a aria-describedby="B1-ijms-26-02685" class="usa-link" href="#B1-ijms-26-02685">1</a>]. By initiating intracellular signaling cascades, RLKs play pivotal roles in various molecular and cellular processes, including plant development, stress response, and signal transduction [<a aria-describedby="B2-ijms-26-02685" class="usa-link" href="#B2-ijms-26-02685">2</a>]. These versatile receptors are capable of sensing plant hormones, secreted peptides, and molecules derived from pathogens involved in plant defense response [<a aria-describedby="B3-ijms-26-02685" class="usa-link" href="#B3-ijms-26-02685">3</a>]. For example, RLKs recognize pathogen-associated molecular patterns (PAMPs), which trigger the Pattern-Triggered Immunity (PTI) response, activating signaling pathways that defend against pathogen invasion.</p>
<p>RLKs typically consist of three key domains: an extracellular domain, a transmembrane domain, and an intracellular kinase domain [<a aria-describedby="B4-ijms-26-02685" class="usa-link" href="#B4-ijms-26-02685">4</a>]. The extracellular domain, characterized by significant structural diversity, forms the basis for classifying RLKs into different subclasses based on their motif structures. Among these, the proline-rich extensin-like receptor kinases (PERKs) represent a distinct group characterized by a proline-rich extracellular domain [<a aria-describedby="B5-ijms-26-02685" class="usa-link" href="#B5-ijms-26-02685">5</a>]. Interestingly, while some <em>PERKs</em> like <em>BnPERK1</em> and <em>AtPERK4</em> locate on the plasma membrane, <em>AtPERK13</em> has been demonstrated to anchor to the cell wall [<a aria-describedby="B6-ijms-26-02685" class="usa-link" href="#B6-ijms-26-02685">6</a>,<a aria-describedby="B7-ijms-26-02685" class="usa-link" href="#B7-ijms-26-02685">7</a>,<a aria-describedby="B8-ijms-26-02685" class="usa-link" href="#B8-ijms-26-02685">8</a>,<a aria-describedby="B9-ijms-26-02685" class="usa-link" href="#B9-ijms-26-02685">9</a>]. Functionally, the intracellular kinase domain of RLKs contains conserved motifs that exhibit kinase activity, which has been observed in <em>BnPERK1</em> and <em>AtPERK4</em> [<a aria-describedby="B7-ijms-26-02685" class="usa-link" href="#B7-ijms-26-02685">7</a>,<a aria-describedby="B10-ijms-26-02685" class="usa-link" href="#B10-ijms-26-02685">10</a>].</p>
<p>In <em>Arabidopsis thaliana</em>, the <em>PERK</em> gene family consists of 15 members, some of which show specific expression patterns in roots and floral organs [<a aria-describedby="B11-ijms-26-02685" class="usa-link" href="#B11-ijms-26-02685">11</a>]. However, some <em>AtPERK</em> genes exhibit broad expression patterns across various tissues, such as <em>AtPERK1,</em> which is ubiquitously expressed, suggesting its involvement in diverse molecular and cellular processes. Recent studies have elucidated the regulation of <em>PERKs</em> in response to different stress conditions [<a aria-describedby="B5-ijms-26-02685" class="usa-link" href="#B5-ijms-26-02685">5</a>]. For example, the expression of <em>TaPERKs</em> in wheat and <em>GhPERKs</em> in cotton is significantly influenced by abiotic stresses such as heat, drought, and salinity [<a aria-describedby="B12-ijms-26-02685" class="usa-link" href="#B12-ijms-26-02685">12</a>,<a aria-describedby="B13-ijms-26-02685" class="usa-link" href="#B13-ijms-26-02685">13</a>]. In <em>A. thaliana</em>, <em>AtPERK13</em> has been shown to respond to nutrient deficiencies such as phosphate, nitrogen, and iron deprivation, highlighting the role of <em>PERKs</em> in maintaining nutrient homeostasis and adaptation [<a aria-describedby="B9-ijms-26-02685" class="usa-link" href="#B9-ijms-26-02685">9</a>]. In rice, the <em>PERK</em> gene Z15 is activated by moderately low temperatures, emphasizing the adaptability of <em>PERKs</em> to diverse environmental cues [<a aria-describedby="B14-ijms-26-02685" class="usa-link" href="#B14-ijms-26-02685">14</a>]. Additionally, <em>PERKs</em> have been implicated in plant defense against pathogens. For example, the expression of <em>TaPERKs</em> is influenced by fungal pathogens, and <em>BnPERK1</em> has been shown to respond to the pathogen <em>Sclerotinia sclerotiorum</em> [<a aria-describedby="B7-ijms-26-02685" class="usa-link" href="#B7-ijms-26-02685">7</a>,<a aria-describedby="B12-ijms-26-02685" class="usa-link" href="#B12-ijms-26-02685">12</a>,<a aria-describedby="B15-ijms-26-02685" class="usa-link" href="#B15-ijms-26-02685">15</a>]. In summary, <em>PERKs</em> are versatile players in response to various stresses.</p>
<p><em>PERKs</em> also play essential roles in plant hormone signaling. Several studies have highlighted the involvement of <em>PERKs</em> in the regulation of growth and development in response to hormonal signals [<a aria-describedby="B10-ijms-26-02685" class="usa-link" href="#B10-ijms-26-02685">10</a>,<a aria-describedby="B13-ijms-26-02685" class="usa-link" href="#B13-ijms-26-02685">13</a>,<a aria-describedby="B16-ijms-26-02685" class="usa-link" href="#B16-ijms-26-02685">16</a>]. For instance, <em>AtPERK4</em> has been implicated in abscisic acid (ABA) signaling during root cell elongation [<a aria-describedby="B10-ijms-26-02685" class="usa-link" href="#B10-ijms-26-02685">10</a>]. Furthermore, the expression profiles of <em>OsPERKs</em> and <em>GhPERKs</em> have shown differential expression patterns under different plant hormone treatments, including ABA, methyl jasmona (MeJA), gibberellin (GA), salicylic acid (SA), and auxin [<a aria-describedby="B13-ijms-26-02685" class="usa-link" href="#B13-ijms-26-02685">13</a>,<a aria-describedby="B16-ijms-26-02685" class="usa-link" href="#B16-ijms-26-02685">16</a>]. The intricate regulation of <em>PERKs</em> by different plant hormones suggests their pivotal role in integrating hormonal signaling pathways to coordinate plant growth and stress responses.</p>
<p>In summary, <em>PERK</em> genes are known to be critical to plant growth, development, and stress responses. Despite the essential roles of <em>PERKs</em> in various processes across various plant species, comprehensive studies on the <em>PERK</em> gene family in <em>Brassica napus</em> (oilseed rape), an economically important crop serving as both an edible oil source and animal feed, are still limited [<a aria-describedby="B17-ijms-26-02685" class="usa-link" href="#B17-ijms-26-02685">17</a>]. This allopolyploid species (AACC, 2n = 38), formed by the hybridization between two diploid ancestors, <em>B. rapa</em> (AA = 20) and <em>B. oleracea</em> (CC = 18), emerged approximately 6800 to 12,500 years ago [<a aria-describedby="B18-ijms-26-02685" class="usa-link" href="#B18-ijms-26-02685">18</a>]. While a comprehensive study of <em>BrPERK</em> genes has been completed in male-sterile <em>B. rapa</em> plants, identifying 25 <em>BrPERK</em> genes and exploring their potential role in male sterility, studies in <em>B. napus</em> remain scarce [<a aria-describedby="B19-ijms-26-02685" class="usa-link" href="#B19-ijms-26-02685">19</a>].</p>
<p>This study aimed to provide a comprehensive study of <em>BnPERK</em> genes to include the exploration of its evolutionary history, chromosome location, and gene structure through various computational approaches. Tissue-specific and hormone treatment gene expression patterns were examined using available transcriptome datasets. Additionally, the study also investigated the role of <em>BnPERKs</em> in response to the obligate pathogen <em>Plasmodiophora brassicae,</em> which causes clubroot disease in <em>Brassica</em> species. By providing insights into the molecular and functional roles of <em>BnPERKs</em>, this study will contribute to a deeper understanding of this often-overlooked plant receptor subfamily and its potential applications in crop improvement.</p></section><section id="sec2-ijms-26-02685"><h2 class="pmc_sec_title">2. Results</h2>
<section id="sec2dot1-ijms-26-02685"><h3 class="pmc_sec_title">2.1. Identification and Phylogenetic Analysis of BnPERKs in Brassica napus</h3>
<p>Fifty <em>PERK</em> genes were identified within the <em>B. napus</em> ‘ZS11’ genome (<a class="usa-link" href="#app1-ijms-26-02685">Supplementary Tables S2 and S3</a>). Identification was accomplished through an extensive BLAST search using 15 <em>AtPERK</em> sequences, revealing a higher number of <em>BnPERK</em> genes compared with those reported in <em>Arabidopsis</em>, Chinese cabbage, soybean, rice, sorghum, maize, cotton, and wheat (<a class="usa-link" href="#app1-ijms-26-02685">Supplementary Table S4</a>). The BnPERK family of proteins exhibited varying lengths, ranging from 187 to 921 amino acids. Their molecular weights (MWs) spanned from 20.63 to 102.21 kDa, with BnPERK15C2 and BnPERK15A2 representing the extremes. Additionally, the isoelectric points (pIs) ranged from 5.26 to 8.59 (<a class="usa-link" href="#app1-ijms-26-02685">Supplementary Table S5</a>). The data plots illustrated that BnPERK proteins exhibit a broad spectrum of MW and pI. Moreover, predictions of subcellular localization suggested that most BnPERK proteins were located on the plasma membrane and endomembrane system, while others were associated with the nucleus and chloroplast (<a class="usa-link" href="#app1-ijms-26-02685">Supplementary Table S5</a>).</p>
<p>To investigate the phylogenetic distribution and evolutionary relationships of the <em>PERK</em> gene family, a phylogenetic tree was constructed using 50 BnPERK protein sequences and 15 AtPERK protein sequences by MEGA11 using the maximum likelihood (ML) method (<a class="usa-link" href="#app1-ijms-26-02685">Supplementary Table S6</a>). The <em>BnPERK</em> genes were categorized into four groups (I–IV) based on sequence similarity and functional domain analysis. Group III, being the largest group, included 5 paralogs with 24 <em>BnPERK</em> genes. Group I followed, containing 6 paralogs with 21 <em>BnPERK</em> genes. Group II included 3 paralogs, with 3 <em>BnPERK</em> genes, while Group IV contained only 1 paralog, with 3 <em>BnPERK</em> genes (<a class="usa-link" href="#ijms-26-02685-f001">Figure 1</a>). All <em>AtPERK</em> genes, with the exception of <em>AtPERK2</em>, exhibited a close syntenic association with the 50 <em>BnPERK</em> genes. Most <em>AtPERK</em> genes corresponded to 2–6 orthologous genes in <em>B. napus</em>, each comprising orthologous genes from both the A and C subgenomes of <em>B. napus</em>. Notably, <em>AtPERK3</em> was associated with a single ortholog, <em>BnPERK03A1</em>, as its counterpart in the C subgenome appeared to have been lost. These results suggest that genome rearrangement and gene loss occurred during the polyploidization of <em>B. napus</em>.</p>
<figure class="fig xbox font-sm" id="ijms-26-02685-f001"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11942576_ijms-26-02685-g001.jpg" target="_blank"><img alt="Figure 1" class="graphic zoom-in" height="681" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f01/11942576/da483c0d2d22/ijms-26-02685-g001.jpg" width="730"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/ijms-26-02685-f001/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Phylogenetic tree of the <em>PERK</em> gene family in <em>B. napus</em> and <em>A. thaliana</em>. The phylogenetic tree was constructed with 50 BnPERK protein sequences and 15 AtPERK protein sequences by MEGA11 using the maximum likelihood (ML) method. A total of 15 paralogs were identified, grouped into 15 distinct clusters, each represented by a different color (PERK1 to PERK15). Based on sequence similarity and functional domain analysis, the 15 paralogs were further categorized into four major groups, which are indicated by distinct colors (outer lane). Group I, light steel blue; Group II, cornflower blue; Group III, royal blue; Group IV, black.</p></figcaption></figure></section><section id="sec2dot2-ijms-26-02685"><h3 class="pmc_sec_title">2.2. Chromosomal Distribution and Collinearity Analysis of BnPERK Genes</h3>
<p>The 50 <em>BnPERK</em> genes were evenly distributed across the A and C subgenomes of <em>B. napus</em>, with 25 genes in each (<a class="usa-link" href="#ijms-26-02685-f002">Figure 2</a> and <a class="usa-link" href="#app1-ijms-26-02685">Supplementary Figure S1</a>). These genes were dispersed among all chromosomes, except for A10. The highest concentration of genes was on the chromosome A1, which harbored seven genes, followed by six genes on C6 and five genes on A7. Single <em>BnPERK</em> genes were identified on chromosomes A2, A3, A4, A5, C4, and C9. The results indicated an irregular arrangement of the <em>PERK</em> gene family across the rapeseed chromosomes (<a class="usa-link" href="#ijms-26-02685-f002">Figure 2</a>).</p>
<figure class="fig xbox font-sm" id="ijms-26-02685-f002"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11942576_ijms-26-02685-g002.jpg" target="_blank"><img alt="Figure 2" class="graphic zoom-in" height="797" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f01/11942576/2c7ceedf6d42/ijms-26-02685-g002.jpg" width="790"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/ijms-26-02685-f002/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Chromosomal distribution and collinearity relationships of <em>BnPERK</em> genes. The locations of all the chromosomal <em>BnPERK</em> genes are shown in different chromosomes. The light green boxes stand for A subgenomes and dark green boxes stand for C subgenomes in <em>Brassica napus</em>. The light brown lines represent collinearity in <em>BnPERK</em> gene pairs.</p></figcaption></figure><p>The collinear gene pairs in the <em>B. napus</em> genome were analyzed to explore syntenic relationships among <em>BnPERKs</em> (<a class="usa-link" href="#ijms-26-02685-f002">Figure 2</a>). We identified 81 collinear gene pairs, including 49 pairs resulting from segmental duplication between the A and C subgenomes and 32 pairs occurring within a subgenome. The result indicated a high conservation between the A and C subgenomes. Additionally, we assessed the selective pressure exerted on duplicated <em>BnPERK</em> genes by calculating the ratios of nonsynonymous to synonymous substitutions (Ka/Ks) for the paralogous gene pairs (<a class="usa-link" href="#app1-ijms-26-02685">Supplementary Table S7</a>). The Ka/Ks ratios ranged from 0.1409 to 0.2518 for 17 gene pairs in the A subgenome and from 0.1682 to 0.2519 for 15 gene pairs in the C subgenome. Furthermore, the Ka/Ks ratios for 49 gene pairs between the A and C subgenomes varied from 0.0482 to 0.5148. Notably, the Ka/Ks ratios for all paralogous gene pairs were found to be less than 1, indicating purifying selection acting on <em>BnPERK</em> genes.</p></section><section id="sec2dot3-ijms-26-02685"><h3 class="pmc_sec_title">2.3. Gene Structure, Conserved Motifs, and Cis-Elements Analysis of BnPERK Genes</h3>
<p>To elucidate the functional characteristics of <em>BnPERKs</em>, the conserved motif, protein domain, and exon–intron organization were analyzed together with the phylogenetic tree (<a class="usa-link" href="#ijms-26-02685-f003">Figure 3</a>). Ten conserved motifs were identified across the BnPERK proteins (<a class="usa-link" href="#ijms-26-02685-f003">Figure 3</a>A and <a class="usa-link" href="#app1-ijms-26-02685">Supplementary Figure S2</a>). These motifs exhibited consistent distribution patterns within the BnPERK proteins, with the exceptions of BnPERK15C2, BnPERK15A1, BnPERK15A3, and BnPERK10C1. Moreover, all <em>BnPERK</em> genes shared conserved tyrosine kinase domains, highlighting their functional significance (<a class="usa-link" href="#ijms-26-02685-f003">Figure 3</a>B). Exon–intron analysis revealed structural diversity within the <em>BnPERK</em> gene family, though orthologous genes generally shared similar gene structures (<a class="usa-link" href="#ijms-26-02685-f003">Figure 3</a>C). Most <em>BnPERK</em> genes harbored 5–9 introns, with the number of introns ranging from 3 (<em>BnPERK15C2</em>) to 16 (<em>BnPERK15A2</em>).</p>
<figure class="fig xbox font-sm" id="ijms-26-02685-f003"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11942576_ijms-26-02685-g003.jpg" target="_blank"><img alt="Figure 3" class="graphic zoom-in" height="604" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f01/11942576/cda870d1e91f/ijms-26-02685-g003.jpg" width="725"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/ijms-26-02685-f003/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Phylogenetic tree, conserved motifs, and gene structure analysis of <em>BnPERK</em> gene family. A total of 14 paralogs were identified based on the phylogenetic tree, each represented by a different color. (<strong>A</strong>) Motif composition of BnPERK proteins, with motifs 1–10 displayed in different colored boxes. (<strong>B</strong>) Identification of the protein kinase domain in BnPERK proteins. (<strong>C</strong>) Exon–intron analysis of <em>BnPERK</em> genes, with yellow boxes representing un-translated regions (UTRs) and orange boxes represented coding sequences (CDSs).</p></figcaption></figure><p><em>Cis</em>-elements analysis is crucial for understanding gene regulation and function. Accordingly, upstream 2 kb sequences from <em>BnPERKs</em> transcription start sites were analyzed (<a class="usa-link" href="#app1-ijms-26-02685">Supplementary Tables S2 and S8</a>). The results revealed that 75% of the identified <em>cis</em>-elements were core promoter and enhancer region elements. The remaining <em>cis</em>-elements were associated with hormonal responses, various stresses, and developmental processes (<a class="usa-link" href="#ijms-26-02685-f004">Figure 4</a>A and <a class="usa-link" href="#app1-ijms-26-02685">Supplementary Figure S3</a>). <em>Cis</em>-elements regulated responses to diverse hormones, including methyl jasmona (MeJA), abscisic acid (ABA), gibberellin (GA), auxin, ethylene (ET), and salicylic acid (SA), which were calculated (<a class="usa-link" href="#ijms-26-02685-f004">Figure 4</a>B). The ABA and MeJA responsive elements were abundantly detected, and the auxin responsive elements were secondarily enriched (<a class="usa-link" href="#ijms-26-02685-f004">Figure 4</a>B). These <em>cis</em>-elements relate to various biotic and abiotic stresses, such as light, anaerobic conditions, drought, low temperature, defense, and wounds (<a class="usa-link" href="#ijms-26-02685-f004">Figure 4</a>C). Notably, there were light-responsive elements in all of the <em>BnPERK</em> genes (<a class="usa-link" href="#app1-ijms-26-02685">Supplementary Figure S3</a>). In addition, there was a small part of the <em>cis</em>-elements that were categorized according to their roles in diverse developmental processes, including zein metabolism, endosperm expression, meristem expression, cell cycle regulation, and circadian control (<a class="usa-link" href="#ijms-26-02685-f004">Figure 4</a>D). The presence of such diverse <em>cis</em>-elements suggests potential regulatory roles of <em>BnPERKs</em> in diverse biological processes.</p>
<figure class="fig xbox font-sm" id="ijms-26-02685-f004"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11942576_ijms-26-02685-g004.jpg" target="_blank"><img alt="Figure 4" class="graphic zoom-in" height="629" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f01/11942576/692a44cb2230/ijms-26-02685-g004.jpg" width="754"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/ijms-26-02685-f004/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><em>Cis</em>-elements analysis of the <em>BnPERK</em> promoters. The <em>cis</em>-elements were predicted by analysis of 2 Kb upstream regions of <em>BnPERK</em> genes. They were categorized into four groups (<strong>A</strong>), including hormonal responses (<strong>B</strong>), biotic and abiotic stress (<strong>C</strong>), developmental processes (<strong>D</strong>), and promoter and enhancer regions, which are represented by different colors and further categorized.</p></figcaption></figure></section><section id="sec2dot4-ijms-26-02685"><h3 class="pmc_sec_title">2.4. Expression Pattern of BnPERK Gene Family Across Various Tissues</h3>
<p>The expression pattern of the <em>BnPERK</em> genes was analyzed across 14 tissues at various developmental stages through RNA-Seq analysis (<a class="usa-link" href="#ijms-26-02685-f005">Figure 5</a>). The results exhibited diverse expression patterns across these tissues. There were 13 genes showing constitutive expression in multiple tissues, among which <em>BnPERK01A1</em> and <em>BnPERK01C1</em> exhibited the highest expression levels. In contrast, certain genes displayed specific expression patterns. For example, <em>BnPERK03A1</em> showed highly specific but relatively low expression in the sepal, and <em>BnPERK15A3</em> specifically expressed in the leaf and silique. Moreover, 18 orthologous genes of <em>AtPERK5</em>, <em>AtPERK6</em>, <em>AtPERK7</em>, and <em>AtPERK12</em> in <em>B. napus</em> were highly expressed in pollen. <em>BnPERK15C2</em>, which encoded the shortest protein, was not expressed in any tissues, suggesting a potential loss of function. These differential expression patterns of <em>BnPERK</em> genes point to their diverse roles in plant growth and development.</p>
<figure class="fig xbox font-sm" id="ijms-26-02685-f005"><h4 class="obj_head">Figure 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11942576_ijms-26-02685-g005.jpg" target="_blank"><img alt="Figure 5" class="graphic zoom-in" height="227" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f01/11942576/520e87341e99/ijms-26-02685-g005.jpg" width="733"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/ijms-26-02685-f005/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Expression profiles of the <em>BnPERK</em> gene family in various tissues of <em>Brassica napus</em>. The heatmap shows the expression of 50 <em>BnPERK</em> genes across 14 different tissues, processed with log<sub>2</sub> normalization. The color gradient ranges from blue (indicating lower expression levels) to red (indicating higher expression levels), while the gray spots represent no detectable expression.</p></figcaption></figure><p>Furthermore, some paralogous gene pairs, such as <em>BnPERK01A1</em>/<em>BnPERK01C1</em> and <em>BnPERK09A1</em>/<em>BnPERK09C1</em>, showed similar expression patterns, being constitutively and highly expressed across all tested tissues. In contrast, other paralogous gene pairs, like <em>BnPERK06A1</em>/<em>BnPERK06A2</em> and <em>BnPERK06C1</em>/<em>BnPERK06C2</em>, showed specific high expression in floral organs. Additionally, <em>BnPERK15A4</em>/<em>BnPERK15C3</em> showed high expression levels in different tissues, except for pollen. However, some paralogous gene pairs exhibited distinct patterns, indicating functional divergence.</p></section><section id="sec2dot5-ijms-26-02685"><h3 class="pmc_sec_title">2.5. Expression Pattern of BnPERK Gene Family in Response to Hormone Treatments</h3>
<p>In order to further explore the function of <em>BnPERK</em>s during hormone responses, we investigated their expression patterns in both leaves and roots after different hormone treatments (<a class="usa-link" href="#ijms-26-02685-f006">Figure 6</a>). Among the 50 identified <em>BnPERK</em> genes, only a small subset was expressed in leaves and roots. Most of these expressed genes were strongly induced in leaves within 0.5 h after indole acetic acid (IAA), l-aminocyclopropane-l-carboxylic acid (ACC), gibberellin acid (GA), trans-zeatin (TZ), or brassinolide (BL) treatments (<a class="usa-link" href="#ijms-26-02685-f006">Figure 6</a>A). Additionally, four genes—<em>BnPERK15A4</em>, <em>BnPERK15C3</em>, <em>BnPERK08A1</em>, and <em>BnPERK08C1</em>—showed enhanced transcript levels at different time points after methyl jasmonate (MeJA) treatment in leaves. In contrast, their expression patterns in roots were distinct. Most genes showed enhanced expression levels after GA and ABA treatments but decreased expression levels after MeJA treatment (<a class="usa-link" href="#ijms-26-02685-f006">Figure 6</a>B). Notably, <em>BnPERK15A1</em> was strongly induced after MeJA treatment. Furthermore, three genes, <em>BnPERK09A1</em>, <em>BnPERK09C1</em>, and <em>BnPERK04C1</em>, which showed increased expression in leaves after IAA treatment, were also induced in roots.</p>
<figure class="fig xbox font-sm" id="ijms-26-02685-f006"><h4 class="obj_head">Figure 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11942576_ijms-26-02685-g006.jpg" target="_blank"><img alt="Figure 6" class="graphic zoom-in" height="369" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f01/11942576/d67729194c44/ijms-26-02685-g006.jpg" width="771"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/ijms-26-02685-f006/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Expression patterns of <em>BnPERKs</em> under hormone treatments in leaves (<strong>A</strong>) and roots (<strong>B</strong>). The heatmap displays the relative expression of <em>BnPERK</em>s at 0.5, 1, 3, and 6 h after treatment, derived from RNA-Seq analysis. IAA, indole acetic acid; GA, gibberellin acid; ABA, abscisic acid; ACC, l-aminocyclopropane-l-carboxylic acid; TZ, trans-zeatin; JA, methyl jasmonate; BL, brassinolide. Different dot colors represent different relative expression levels: red dot indicates up-regulated expression, blue dot indicates down-regulated expression, and the larger the dot, the greater the fold change.</p></figcaption></figure></section><section id="sec2dot6-ijms-26-02685"><h3 class="pmc_sec_title">2.6. Protein–Protein Interaction Network Analysis of BnPERK Genes</h3>
<p>Predictive interaction networks were employed to explore the biological mechanisms of BnPERK proteins. The analysis revealed that the BnPERK proteins putatively interact with 1133 proteins in <em>B. napus</em>. The results showed that most PERK proteins serve as core nodes in these networks, interacting with various proteins participating in different biological processes (<a class="usa-link" href="#ijms-26-02685-f007">Figure 7</a>A and <a class="usa-link" href="#app1-ijms-26-02685">Supplementary Table S9</a>).</p>
<figure class="fig xbox font-sm" id="ijms-26-02685-f007"><h4 class="obj_head">Figure 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11942576_ijms-26-02685-g007.jpg" target="_blank"><img alt="Figure 7" class="graphic zoom-in" height="578" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f01/11942576/fae748142def/ijms-26-02685-g007.jpg" width="738"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/ijms-26-02685-f007/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Proteins interacting with BnPERK proteins in <em>Brassica napus</em>. (<strong>A</strong>) Protein–protein interaction network of BnPERK proteins. Dark brown spots represent BnPERK proteins, blue spots represent proteins that interact with BnPERK proteins, and gray lines indicate the interaction between BnPERK proteins and other proteins. (<strong>B</strong>) KEGG enrichment analysis of proteins that interacted with BnPERK proteins. (<strong>C</strong>) GO enrichment analysis of proteins that interacted with BnPERK proteins.</p></figcaption></figure><p>KEGG enrichment analysis indicated that the interacting proteins were primarily involved in processes such as pentose and glucuronate interconversions, plant–pathogen interaction, starch and sucrose metabolism, cyanoamino acid metabolism, the biosynthesis of secondary metabolites, endoplasmic reticulum processing, and the MAPK signaling pathway (<a class="usa-link" href="#ijms-26-02685-f007">Figure 7</a>B and <a class="usa-link" href="#app1-ijms-26-02685">Supplementary Table S10</a>). Similarly, GO enrichment analysis revealed significant enrichment of terms like integral components of the membrane and plasma membrane in the cellular component category (<a class="usa-link" href="#ijms-26-02685-f007">Figure 7</a>C and <a class="usa-link" href="#app1-ijms-26-02685">Supplementary Table S11</a>). ATP binding and kinase activity were prominently enriched in the molecular function category. Biological process category terms such as protein phosphorylation, pectin catalytic activity, and cell wall modification were enriched. Protein interactions analysis provided a comprehensive view of the BnPERK proteins interaction network in <em>B. napus</em>, underscoring their central role and broad involvement in different biological processes, including sugar metabolism and plant–pathogen interaction.</p></section><section id="sec2dot7-ijms-26-02685"><h3 class="pmc_sec_title">2.7. Root Gall Formation and BnPERK Gene Expression in B. napus upon Plasmodiophora brassicae Inoculation</h3>
<p>Galls were observed on the roots of <em>B. napus</em> at 28 days after inoculation with <em>P. brassicae</em> (<a class="usa-link" href="#ijms-26-02685-f008">Figure 8</a>A). Cross-sections of the inoculated roots revealed the changes in the vascular system. Resting spores that stained blue formed in the diseased roots. Furthermore, diseased root cells exhibited hypertrophy and disorder, accompanied by thickened cell walls, while control roots grew normally (<a class="usa-link" href="#ijms-26-02685-f008">Figure 8</a>B). To better understand the response of <em>BnPERK</em> genes to clubroot disease, we analyzed the expression of <em>BnPERK</em> at 14 days after <em>P. brassicae</em> inoculation by RNA-Seq (<a class="usa-link" href="#ijms-26-02685-f008">Figure 8</a>C). Among the 50 identified <em>BnPERK</em> genes, 23 were expressed in either inoculated or control roots. Notably, four genes, namely <em>BnPERK09A1</em>, <em>BnPERK09C1</em>, <em>BnPERK14A1</em>, and <em>BnPERK06C2,</em> showed downregulated expression in response to <em>P. brassicae</em>, among which <em>BnPERK06C2</em> was completely inhibited after inoculation. Additionally, the paralogous gene pair <em>BnPERK09A1</em>/<em>BnPERK09C1</em> also exhibited consistent expression patterns across different tissues. These four downregulated <em>BnPERK</em> genes were implicated in the response to <em>P. brassicae</em>.</p>
<figure class="fig xbox font-sm" id="ijms-26-02685-f008"><h4 class="obj_head">Figure 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11942576_ijms-26-02685-g008.jpg" target="_blank"><img alt="Figure 8" class="graphic zoom-in" height="835" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f01/11942576/2574df7aa1ad/ijms-26-02685-g008.jpg" width="722"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/ijms-26-02685-f008/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Disease symptoms, histocytological analysis, and <em>PERK</em> gene expression analysis in control and inoculated <em>B. napus</em>. (<strong>A</strong>) Symptoms of clubroot disease in control and inoculated roots at 28 days after inoculation. (<strong>B</strong>,<strong>C</strong>) Histocytological analysis of cross-sections of control and inoculated roots at 28 days after inoculation. (<strong>D</strong>) Expression patterns of 23 <em>PERK</em> genes at 14 days after inoculation with <em>Plasmodiophora brassicae</em>. Error bars represent the standard error of the means of the three replicates. Stars indicate significant differences in expression between control and inoculated roots (|log<sub>2</sub> (fold-change)| &gt; 1). *, <em>p</em> &lt; 0.05; **, <em>p</em> &lt; 0.01.</p></figcaption></figure></section><section id="sec2dot8-ijms-26-02685"><h3 class="pmc_sec_title">2.8. Function of AtPERK9 in Clubroot Disease</h3>
<p>To explore the putative role of <em>BnPERK09A1</em> and <em>BnPERK09C1</em> in clubroot disease, the mutant of the orthologous gene <em>AtPERK9</em> (At1g68690) was used for clubroot disease investigation. Both the Col-0 and <em>perk9</em> showed disease symptoms on the main roots and the lateral roots (<a class="usa-link" href="#ijms-26-02685-f009">Figure 9</a>A). The percentage of severely diseased plants (disease grade 3) in Col-0 was much higher than that in <em>perk9</em>, which resulted in a significantly higher disease index in Col-0 (<a class="usa-link" href="#ijms-26-02685-f009">Figure 9</a>B,C). It indicated that <em>perk9</em> showed decreased severity of the clubroot disease.</p>
<figure class="fig xbox font-sm" id="ijms-26-02685-f009"><h4 class="obj_head">Figure 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11942576_ijms-26-02685-g009.jpg" target="_blank"><img alt="Figure 9" class="graphic zoom-in" height="492" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f01/11942576/496dc8c52356/ijms-26-02685-g009.jpg" width="773"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/ijms-26-02685-f009/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Enhanced disease severity of clubroot disease in the mutant of <em>AtPERK9</em>. (<strong>A</strong>) Root phenotypes of wild-type <em>A. thaliana</em> (Col-0) and <em>perk9</em> at 28 days after inoculation. (<strong>B</strong>) The occurrence of clubroot disease in Col-0 and <em>perk9</em> was investigated according to the 0–3 scoring system. (<strong>C</strong>) Disease index of Col-0 and <em>perk9</em>. (<strong>D</strong>) Expression patterns of <em>PERK9</em>, <em>PR2, PR5,</em> and <em>RBOHD</em> genes in Col-0 and <em>perk9</em> lines. Different letters represent significant differences (<em>p</em> &lt; 0.05).</p></figcaption></figure><p>Furthermore, the expression levels of <em>PERK9</em>, <em>PR2</em>, <em>PR5,</em> and <em>RBOHD</em> were evaluated in Col-0 and <em>perk9</em> lines at 14 days after <em>P. brassicae</em> inoculation. <em>PERK9</em> also showed downregulated expression in response to <em>P. brassicae</em> in Col-0, while it was totally inhibited in <em>perk9</em>. The expressions of <em>PR2</em>, <em>PR5</em>, and <em>RBOHD</em> genes were greatly enhanced in <em>perk9</em>. <em>PR2</em> and <em>RBOHD</em> not only could be induced in Col-0 after <em>P. brassicae</em> inoculation, but also in <em>perk9</em>. Enhanced disease resistance to clubroot disease and increased expression of these genes in <em>perk9</em> indicated that the basal immune response was triggered by the knockout of <em>PERK9.</em></p></section></section><section id="sec3-ijms-26-02685"><h2 class="pmc_sec_title">3. Discussion</h2>
<p>PERKs constitute a crucial subfamily of RLKs that play important roles in various molecular and cellular processes in plants, including development, signaling, and stress responses [<a aria-describedby="B5-ijms-26-02685" class="usa-link" href="#B5-ijms-26-02685">5</a>]. Despite their significance, there remains a lack of comprehensive study on the molecular functions of PERKs, particularly when compared with other RLKs. While genome-wide analyses of <em>PERK</em> genes have been conducted in some species [<a aria-describedby="B11-ijms-26-02685" class="usa-link" href="#B11-ijms-26-02685">11</a>,<a aria-describedby="B12-ijms-26-02685" class="usa-link" href="#B12-ijms-26-02685">12</a>,<a aria-describedby="B13-ijms-26-02685" class="usa-link" href="#B13-ijms-26-02685">13</a>,<a aria-describedby="B16-ijms-26-02685" class="usa-link" href="#B16-ijms-26-02685">16</a>,<a aria-describedby="B19-ijms-26-02685" class="usa-link" href="#B19-ijms-26-02685">19</a>], a thorough study of the <em>BnPERK</em> gene family has yet to be published. Therefore, this study systematically analyzed <em>BnPERK</em> genes, covering various aspects.</p>
<section id="sec3dot1-ijms-26-02685"><h3 class="pmc_sec_title">3.1. Deciphering the Evolutionary Dynamics of the BnPERK Genes</h3>
<p>There were 50 <em>PERK</em> genes identified in the rapeseed cultivar ‘ZS11’ genome, highlighting an increased number of <em>BnPERK</em> genes compared with previous studies (<a class="usa-link" href="#app1-ijms-26-02685">Supplementary Table S3</a>). This expansion of <em>BnPERK</em> genes was attributed to the evolutionary path of the <em>Brassica</em> genus, which experienced a genome triplication following its divergence from the <em>Arabidopsis</em> lineage [<a aria-describedby="B17-ijms-26-02685" class="usa-link" href="#B17-ijms-26-02685">17</a>,<a aria-describedby="B18-ijms-26-02685" class="usa-link" href="#B18-ijms-26-02685">18</a>]. Through interspecific hybridization between <em>B. rapa</em> (AA) and <em>B. oleracea</em> (CC), the allotetraploid <em>B. napus</em> (AACC) arose. This complex evolutionary process contributed to the expansion of the <em>BnPERK</em> gene family. Studies have consistently demonstrated that whole-genome duplication and segmental duplications are crucial mechanisms driving gene duplication and the expansion of gene families [<a aria-describedby="B20-ijms-26-02685" class="usa-link" href="#B20-ijms-26-02685">20</a>,<a aria-describedby="B21-ijms-26-02685" class="usa-link" href="#B21-ijms-26-02685">21</a>,<a aria-describedby="B22-ijms-26-02685" class="usa-link" href="#B22-ijms-26-02685">22</a>]. The prevalence of paralogous gene pairs resulting from segmental duplication highlights the role of genome duplication events in this expansion.</p>
<p>Initially, it was expected that each <em>Arabidopsis</em> gene would have six homologs in <em>B. napus</em> due to two recent duplication events. However, only 50 <em>BnPERK</em> genes were identified, suggesting a loss of duplicated <em>BnPERK</em> genes following <em>Brassica</em> genome triplication and hybridization. This indicated that gene loss was a crucial process during genomic arrangements and chromosome doubling [<a aria-describedby="B23-ijms-26-02685" class="usa-link" href="#B23-ijms-26-02685">23</a>]. This also implied a selective process, where functionally redundant <em>BnPERK</em> copies may have been lost, while the retained 50 <em>BnPERK</em> genes may be crucial for plant development, hormone signaling, and stress response.</p>
<p>Phylogenetic analysis revealed that most <em>AtPERK</em> genes corresponded to 2–6 orthologous genes in <em>B. napus</em>, each of which included orthologs from both the A and C subgenomes. However, some <em>AtPERK</em> genes showed variations in the number of corresponding orthologous genes from the A and C subgenomes, suggesting extensive chromosomal rearrangements and asymmetric gene loss within duplicated genomic blocks [<a aria-describedby="B17-ijms-26-02685" class="usa-link" href="#B17-ijms-26-02685">17</a>]. Analysis of Ka/Ks ratios of paralogous gene pairs indicated purifying selection on duplicated <em>BnPERK</em> genes, implying their functional conservation and evolutionary constraint at these loci [<a aria-describedby="B24-ijms-26-02685" class="usa-link" href="#B24-ijms-26-02685">24</a>].</p>
<p>Understanding the evolutionary dynamics of the <em>BnPERK</em> gene family not only enriches our knowledge of rapeseed genetics but also has broader implications for research in plant biology and stress-response mechanisms.</p></section><section id="sec3dot2-ijms-26-02685"><h3 class="pmc_sec_title">3.2. Exploring the Structural Diversity and Functional Roles of BnPERKs</h3>
<p>The <em>BnPERK</em> gene family exhibited significant structural diversity, including variations in gene length, molecular weight, and isoelectric points. These variations suggested potential functional specialization within the gene family [<a aria-describedby="B25-ijms-26-02685" class="usa-link" href="#B25-ijms-26-02685">25</a>].</p>
<p>Although functional studies of <em>BnPERKs</em> are limited, expression profiling can still provide valuable insights about the gene’s function and its role in different biological processes [<a aria-describedby="B26-ijms-26-02685" class="usa-link" href="#B26-ijms-26-02685">26</a>]. In accordance with the result in <em>A. thaliana</em>, <em>BnPERK</em> family members exhibited two distinct expression patterns: some were highly expressed in pollen, while others had broader expression patterns [<a aria-describedby="B11-ijms-26-02685" class="usa-link" href="#B11-ijms-26-02685">11</a>]. Constitutive expression of certain <em>BnPERK</em> genes indicated their essential roles in fundamental cellular functions, while some <em>BnPERKs</em> exhibited higher expression levels in floral organs, which is consistent with findings in other species like <em>A. thaliana</em>, <em>B. rapa</em>, <em>Triticum aestivum</em>, and <em>Oryza sativa</em> and indicates their importance in floral development [<a aria-describedby="B11-ijms-26-02685" class="usa-link" href="#B11-ijms-26-02685">11</a>,<a aria-describedby="B12-ijms-26-02685" class="usa-link" href="#B12-ijms-26-02685">12</a>,<a aria-describedby="B16-ijms-26-02685" class="usa-link" href="#B16-ijms-26-02685">16</a>,<a aria-describedby="B19-ijms-26-02685" class="usa-link" href="#B19-ijms-26-02685">19</a>]. Specifically, 18 orthologs of <em>AtPERK5</em>, <em>AtPERK6</em>, <em>AtPERK7,</em> and <em>AtPERK12</em> were notably expressed in pollen, among which <em>AtPERK5</em> and <em>AtPERK12</em> were shown to be essential for pollen tube growth [<a aria-describedby="B11-ijms-26-02685" class="usa-link" href="#B11-ijms-26-02685">11</a>,<a aria-describedby="B27-ijms-26-02685" class="usa-link" href="#B27-ijms-26-02685">27</a>]. Furthermore, certain <em>B. rapa</em> genes (orthologs of <em>AtPERK6</em> and <em>AtPERK12</em>) were associated with male sterility, as they were downregulated in male-sterile mutants [<a aria-describedby="B19-ijms-26-02685" class="usa-link" href="#B19-ijms-26-02685">19</a>]. <em>BnPERK01A1</em> and <em>BnPERK01C1</em> (orthologs of <em>AtPERK1</em>) showed constitutively high expression, suggesting essential roles in fundamental cellular functions. Functional analysis of <em>BnPERK1</em> indicated its role in regulating cell wall integrity, signaling in stress responses, and facilitating growth and development in <em>Arabidopsis</em> [<a aria-describedby="B15-ijms-26-02685" class="usa-link" href="#B15-ijms-26-02685">15</a>].</p>
<p>Phylogenetic analysis indicated that the <em>BnPERK</em> genes originated through segmental duplication [<a aria-describedby="B28-ijms-26-02685" class="usa-link" href="#B28-ijms-26-02685">28</a>]. Structural examination confirmed the phylogenetic arrangement, with most <em>BnPERK</em> genes sharing conserved motifs and tyrosine kinase domains. Differences in gene structure and <em>cis</em>-acting regulatory elements in promoters suggested evolutionary divergence, which may contribute to functional diversity and evolutionary mechanisms [<a aria-describedby="B29-ijms-26-02685" class="usa-link" href="#B29-ijms-26-02685">29</a>]. Analysis of the <em>cis</em>-element indicated that <em>BnPERK</em> genes were responsive to various stresses and hormone signaling pathways, with MeJA- and ABA-responsive elements being particularly abundant. This suggests that <em>BnPERK</em> genes could mediate diverse gene expression patterns and provide flexibility in stress responses [<a aria-describedby="B30-ijms-26-02685" class="usa-link" href="#B30-ijms-26-02685">30</a>].</p>
<p>Further study on the expression patterns of <em>BnPERK</em> genes in response to hormone treatments provided insights into their potential roles in hormone signaling and plant development. This rapid and robust response to IAA, ET, GA, CK, and BL suggested that <em>BnPERK</em> genes might play a crucial role in the early stages of hormonal signal transduction in leaves, possibly by mediating the downstream effects of these hormones. In contrast, the expression profiles in roots exhibited totally different expression patterns, which highlights the complexity and specificity of hormonal regulation in different tissues. Enhanced expression levels after GA and ABA treatments in roots suggest involvements in root growth and stress adaptation [<a aria-describedby="B31-ijms-26-02685" class="usa-link" href="#B31-ijms-26-02685">31</a>,<a aria-describedby="B32-ijms-26-02685" class="usa-link" href="#B32-ijms-26-02685">32</a>]. The contrasting expression patterns after MeJA treatment in roots and leaves imply distinct roles in different tissues. Additionally, the observation that genes such as <em>BnPERK09A1</em>, <em>BnPERK09C1</em>, and <em>BnPERK04C1</em> were induced in both leaves and roots after IAA treatment implies a role in auxin signaling, which is central to various developmental processes, including cell elongation and differentiation [<a aria-describedby="B33-ijms-26-02685" class="usa-link" href="#B33-ijms-26-02685">33</a>]. These genes may act as intermediaries in auxin signal transduction pathways, influencing growth and development in a tissue-specific manner.</p>
<p>In summary, the comprehensive examination illuminates the structural diversity and functional implications of the <em>BnPERK</em> gene family. The differential expression patterns of <em>BnPERK</em> genes in response to various hormone treatments highlight their potential involvement in diverse physiological processes, including growth regulation, stress responses, and defense mechanisms.</p></section><section id="sec3dot3-ijms-26-02685"><h3 class="pmc_sec_title">3.3. Revealing the Role of BnPERK in Clubroot Disease</h3>
<p>Recent studies have suggested that receptor-like kinases (RLKs) including PERKs are involved in plant–pathogen interactions and immune responses [<a aria-describedby="B2-ijms-26-02685" class="usa-link" href="#B2-ijms-26-02685">2</a>,<a aria-describedby="B3-ijms-26-02685" class="usa-link" href="#B3-ijms-26-02685">3</a>,<a aria-describedby="B4-ijms-26-02685" class="usa-link" href="#B4-ijms-26-02685">4</a>]. Analysis of <em>cis</em>-elements and expression profiles suggested that <em>BnPERKs</em> may act as novel sensors for various stresses, particularly plant pathogen infections [<a aria-describedby="B5-ijms-26-02685" class="usa-link" href="#B5-ijms-26-02685">5</a>,<a aria-describedby="B12-ijms-26-02685" class="usa-link" href="#B12-ijms-26-02685">12</a>]. Protein–protein interaction network analysis supported this result, revealing the multifunctionality of BnPERK proteins and emphasizing their broad participation in plant physiological processes specific to plant disease response [<a aria-describedby="B34-ijms-26-02685" class="usa-link" href="#B34-ijms-26-02685">34</a>]. The interactions involved cell wall organization, plant–pathogen interactions, plasma membrane dynamics, endomembrane activities, MAPK signaling pathways, and starch/sucrose metabolism. Given that pathogen-induced changes in cell walls affected disease resistance, it is possible that <em>BnPERKs</em> detect these alterations and initiate responses to maintain cell wall integrity [<a aria-describedby="B35-ijms-26-02685" class="usa-link" href="#B35-ijms-26-02685">35</a>]. The membrane localization of <em>BnPERKs</em> suggested they might perceive infection signals at the cell’s surface. Upon activation by cell wall components, <em>BnPERKs</em> likely trigger downstream signaling cascades, including MAPK pathways [<a aria-describedby="B36-ijms-26-02685" class="usa-link" href="#B36-ijms-26-02685">36</a>]. Otherwise, MAPK pathways also acted as downstream signaling components of hormones and played important roles in plant responses to different stresses [<a aria-describedby="B37-ijms-26-02685" class="usa-link" href="#B37-ijms-26-02685">37</a>]. Despite these insights, the exact role of <em>BnPERKs</em> in plant immunity remains largely unknown, and their interactions with RLKs or coreceptors remain largely unexplored.</p>
<p>Clubroot disease, caused by the obligate parasite <em>P. brassicae</em>, poses a significant threat to Brassicaceae plants [<a aria-describedby="B38-ijms-26-02685" class="usa-link" href="#B38-ijms-26-02685">38</a>]. The parasite is dependent on plant-derived sugars as a carbon source for colonization [<a aria-describedby="B39-ijms-26-02685" class="usa-link" href="#B39-ijms-26-02685">39</a>,<a aria-describedby="B40-ijms-26-02685" class="usa-link" href="#B40-ijms-26-02685">40</a>]. The enriched sugar metabolism reflects the altered homeostasis and increased demand for sugar in infected plants. However, the downstream signaling components and their regulation in response to <em>P. brassicae</em> infection are not yet clear. The <em>PERK</em> gene family, with its known functions in cell wall integrity and stress responses, may contribute to reinforcing the plant cell wall during pathogen infection or modulating defense-related signaling pathways [<a aria-describedby="B5-ijms-26-02685" class="usa-link" href="#B5-ijms-26-02685">5</a>]. Given the role of <em>PERKs</em> in disease response, the involvement of the <em>PERK</em> gene family in <em>B. napus</em> in clubroot disease was as expected. The transcriptional downregulation of <em>BnPERK09A1</em>, <em>BnPERK09C1</em>, <em>BnPERK14A1</em>, and <em>BnPERK06C2</em> upon <em>P. brassicae</em> infection suggests their putative roles in clubroot disease response. In accordance with the result in <em>B. napus</em>, the expression of <em>AtPERK9</em> was also downregulated in response to <em>P. brassicae</em>. The <em>perk9</em> mutant was used for clubroot disease investigation, indicating the roles of <em>BnPERK09A1</em> and <em>BnPERK09C1</em> in the disease resistance. Pathogenesis-related proteins including enzymes like β-1,3-glucanase (PR2) and thionins (PR5), which have antimicrobial activity and are involved in the basal immune response [<a aria-describedby="B41-ijms-26-02685" class="usa-link" href="#B41-ijms-26-02685">41</a>]. The <em>RBOHD</em> (Respiratory burst oxidase homolog D) gene is crucial in generating reactive oxygen species (ROS) as part of the plant immune system [<a aria-describedby="B42-ijms-26-02685" class="usa-link" href="#B42-ijms-26-02685">42</a>]. The increased expression of these genes suggested that the knockout of <em>AtPERK9</em> triggered the basal immune response, resulting in increased disease resistance in <em>perk9.</em> The induction of <em>PR2</em> and <em>RBOHD</em> in both Col-0 and <em>perk9</em> after inoculation suggested that basal immune response occurred in susceptible Col-0. Furthermore, the induction of <em>BnPERK09A1</em> and <em>BnPERK09C1</em> by IAA treatment was also particularly intriguing, as IAA is known to act as a signaling molecule potentially involved in the response to <em>P. brassicae</em> infection [<a aria-describedby="B43-ijms-26-02685" class="usa-link" href="#B43-ijms-26-02685">43</a>,<a aria-describedby="B44-ijms-26-02685" class="usa-link" href="#B44-ijms-26-02685">44</a>,<a aria-describedby="B45-ijms-26-02685" class="usa-link" href="#B45-ijms-26-02685">45</a>]. However, the exact roles of <em>BnPERK09A1</em> and <em>BnPERK09C1</em> need further function study. The predicted interaction proteins associated with these pathways implied their roles in mediating the interaction between <em>B. napus</em> and <em>P. brassicae</em>, providing targets for further functional characterization.</p>
<p>In summary, uncovering the role of <em>BnPERKs</em> in clubroot disease response provided valuable insights into plant–pathogen interactions and potential strategies for disease management.</p></section></section><section id="sec4-ijms-26-02685"><h2 class="pmc_sec_title">4. Materials and Methods</h2>
<section id="sec4dot1-ijms-26-02685"><h3 class="pmc_sec_title">4.1. Identification of BnPERK Genes in Brassica napus</h3>
<p>The <em>BnPERK</em> gene family was selected in this study due to its critical role in plant growth, development, and stress responses. It has not been extensively studied in <em>B. napus</em>, making it a valuable target for genome-wide analysis. The genomic data and annotation for the <em>B. napus</em> cultivar ‘ZS11’ were sourced from <em>B. napus</em> multi-omics information resource (BnIR) (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://yanglab.hzau.edu.cn/BnIR" rel="noopener noreferrer" target="_blank">https://yanglab.hzau.edu.cn/BnIR</a>, accessed on 1 March 2024) [<a aria-describedby="B46-ijms-26-02685" class="usa-link" href="#B46-ijms-26-02685">46</a>]. The <em>PERK</em> genes in <em>B. napus</em> were identified using BLASTp (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://blast.ncbi.nlm.nih.gov/" rel="noopener noreferrer" target="_blank">https://blast.ncbi.nlm.nih.gov/</a>) with an E-value threshold of 10<sup>−5</sup>, employing the protein sequence of 15 AtPERKs as queries [<a aria-describedby="B11-ijms-26-02685" class="usa-link" href="#B11-ijms-26-02685">11</a>]. The <em>BnPERK</em> gene family members were named based on their orthologous relationships and chromosomal location. By searching the conserved domain in InterPro (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.ebi.ac.uk/interpro" rel="noopener noreferrer" target="_blank">http://www.ebi.ac.uk/interpro</a>, accessed on 5 March 2024) and the NCBI Conserved Domain Database (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi" rel="noopener noreferrer" target="_blank">https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi</a>, accessed on 5 March 2024), the tyrosine kinase domain-containing protein sequences were verified for candidate <em>BnPERK</em> genes. Additionally, Expasy (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://web.expasy.org/protparam" rel="noopener noreferrer" target="_blank">https://web.expasy.org/protparam</a>, accessed on 8 March 2024) was utilized to assess the isoelectric points (pIs) and the molecular weights (MWs) of BnPERK proteins. Subcellular locations of these proteins were predicted using Euk-mPLoc [<a aria-describedby="B47-ijms-26-02685" class="usa-link" href="#B47-ijms-26-02685">47</a>] (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.csbio.sjtu.edu.cn/bioinf/euk-multi-2" rel="noopener noreferrer" target="_blank">http://www.csbio.sjtu.edu.cn/bioinf/euk-multi-2</a>, accessed on8 March 2024).</p></section><section id="sec4dot2-ijms-26-02685"><h3 class="pmc_sec_title">4.2. Chromosomal Location, Gene Duplication, and Phylogenetic Analysis</h3>
<p>The positions of the <em>BnPERK</em> genes on the chromosomes were obtained from the annotation of the <em>B. napus</em> genome. McScanX (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.rcac.purdue.edu/software/mcscanx" rel="noopener noreferrer" target="_blank">https://www.rcac.purdue.edu/software/mcscanx</a>) was employed to detect the duplication patterns, including segmental and tandem duplication [<a aria-describedby="B48-ijms-26-02685" class="usa-link" href="#B48-ijms-26-02685">48</a>]. Visualization of chromosomal distribution and duplication events was represented using Circos (Vers. 0.69-9) software [<a aria-describedby="B49-ijms-26-02685" class="usa-link" href="#B49-ijms-26-02685">49</a>]. The Ka/Ks (nonsynonymous substitution rate to synonymous substitution rate) ratio was calculated for the paralogous gene pairs using ParaAT2.0 [<a aria-describedby="B50-ijms-26-02685" class="usa-link" href="#B50-ijms-26-02685">50</a>]. Collinearity analysis of <em>BnPERK</em> genes was conducted and displayed using TBtools (Vers. 2.056) [<a aria-describedby="B51-ijms-26-02685" class="usa-link" href="#B51-ijms-26-02685">51</a>]. A phylogenetic tree was generated using MEGA 11 [<a aria-describedby="B52-ijms-26-02685" class="usa-link" href="#B52-ijms-26-02685">52</a>] through neighbor-joining method with 1000 bootstrap replications and visualized using iTol (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://itol.embl.de" rel="noopener noreferrer" target="_blank">https://itol.embl.de</a>, accessed on 11 March 2024).</p></section><section id="sec4dot3-ijms-26-02685"><h3 class="pmc_sec_title">4.3. Gene Structure, Conserved Motifs, and Cis-Elements Analysis</h3>
<p>The exon–intron structure of <em>PERK</em> genes was examined using Gene Structure Display Server 2.0 (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://gsds.gao-lab.org/index.php/" rel="noopener noreferrer" target="_blank">https://gsds.gao-lab.org/index.php/</a> accessed on 14 March 2024). Conserved motifs were predicted using Multiple Expectation Maximization for Motif Elicitation (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://meme-suite.org/meme/tools/meme" rel="noopener noreferrer" target="_blank">https://meme-suite.org/meme/tools/meme</a>, accessed on 14 March 2024). The sequences of 2 Kb upstream from the start codon of 50 <em>BnPERK</em> genes were extracted from the <em>B. napus</em> ‘ZS11’ genome and analyzed for <em>cis</em>-elements using PlantCARE [<a aria-describedby="B53-ijms-26-02685" class="usa-link" href="#B53-ijms-26-02685">53</a>] (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://bioinformatics.psb.ugent.be/webtools/plantcare/html" rel="noopener noreferrer" target="_blank">http://bioinformatics.psb.ugent.be/webtools/plantcare/html</a>, accessed on 14 March 2024) to predict <em>cis</em>-elements. The data were visualized using TBtools (Vers. 2.056) [<a aria-describedby="B51-ijms-26-02685" class="usa-link" href="#B51-ijms-26-02685">51</a>].</p></section><section id="sec4dot4-ijms-26-02685"><h3 class="pmc_sec_title">4.4. Expression Profiles of BnPERK Genes in Different Tissues and Under Hormone Treatments</h3>
<p>To display different expression profiles, RNA-Seq data from various tissues and the hormone response of <em>B. napus</em> cultivar ‘ZS11’ were obtained from BnIR. A total of 14 tissues at different development stages included seed, leaf, bud, root, petal, sepal, pollen, silique, filament, cotyledon, lower stem peel, middle stem peel, upper stem peel, and vegetative rosette stages. The roots and leaves were treated with hormones (IAA, auxin; ACC, l-aminocyclopropane-l-carboxylic acid; GA, gibberellin acid; ABA, abscisic acid; TZ, trans-zeatin; JA, methyl jasmonate; BL, brassinolide) and sampled at 0.5, 1, 3, and 6 h after treatment. Heatmaps were generated using these expression values after logarithmic transformed RPKM values and displayed using TBtools (Vers. 2.056) [<a aria-describedby="B51-ijms-26-02685" class="usa-link" href="#B51-ijms-26-02685">51</a>].</p></section><section id="sec4dot5-ijms-26-02685"><h3 class="pmc_sec_title">4.5. Network-Based Prediction of Protein Interactions</h3>
<p>Analysis of the protein–protein interaction network in <em>B. napus</em> was predicted using STRING (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://cn.string-db.org/" rel="noopener noreferrer" target="_blank">https://cn.string-db.org/</a>, accessed on 15 March 2024), and the interactions were displayed using Cytoscape (Vers. 3.9.1) [<a aria-describedby="B54-ijms-26-02685" class="usa-link" href="#B54-ijms-26-02685">54</a>]. In order to investigate the biological process involving <em>B. napus</em> genes, the putative interacted genes based on Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analyses were conducted by BnaOmics [<a aria-describedby="B55-ijms-26-02685" class="usa-link" href="#B55-ijms-26-02685">55</a>] (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://bnaomics.ocri-genomics.net/tools/enrich/" rel="noopener noreferrer" target="_blank">https://bnaomics.ocri-genomics.net/tools/enrich/</a>, accessed on 15 March 2024).</p></section><section id="sec4dot6-ijms-26-02685"><h3 class="pmc_sec_title">4.6. Plasmodiophora Brassicae Inoculation, Clubroot Disease Investigation, and Histocytological Analysis</h3>
<p>A collection of <em>Plasmodiophora brassicae</em> isolate from Zhijiang, Hubei, was used for inoculation in greenhouse. The seeds of <em>B. napus</em> cultivar ‘ZS11’, wild-type <em>A. thaliana</em> (Col-0), and <em>AtPERK9</em> mutant were germinated in a greenhouse maintained at 24 °C under a 16/8 h light/dark photoperiod. Seven-day-old rapeseed seedlings were inoculated with resting spores of <em>P. brassicae</em> at a concentration of 10<sup>−6</sup> spores/mL [<a aria-describedby="B56-ijms-26-02685" class="usa-link" href="#B56-ijms-26-02685">56</a>]. Two-week-old <em>Arabidopsis</em> seedlings were inoculated at a concentration of 10<sup>−5</sup> spores/mL. The clubroot disease was investigated and photographed at 28–35 days after inoculation. The disease index was assessed using the following 0–3 scoring system: 0 refers to healthy roots; 1 refers to small galls on lateral roots only; 2 refers to small and medium galls covering the main root and a few lateral roots; 3 indicates large galls on the main root. The disease index (DI) was calculated according to the following formula: DI = 100 × (1 × N<sub>1</sub> + 2 × N<sub>2</sub> + 3 × N<sub>3</sub>)/3 × N<sub>t</sub>. N<sub>1</sub> to N<sub>3</sub> refer to the number of plants in the indicated class, and N<sub>t</sub> is the total number of plants tested.</p>
<p>The progression of cortex infection by <em>P. brassicae</em> was studied. Approximately 0.5 cm long root segments were excised from the upper of the primary root. Following standard protocols for fixation and dehydration, the samples were embedded in paraffin and sectioned at a thickness of 5–8 μM. Sections were analyzed and photographed using an optical microscope.</p></section><section id="sec4dot7-ijms-26-02685"><h3 class="pmc_sec_title">4.7. Transcriptome Analysis of BnPERK Genes in B. napus During Clubroot Disease</h3>
<p>Root samples of <em>B. napus</em> infected with <em>P. brassicae</em> and uninfected controls were harvested at 14 days after treatment for RNA-Seq analysis. After sequencing and quality control, bioinformatics pipelines were using to clean raw sequencing data by removing adapters and low-quality reads. Then, the clean reads were aligned to the genome of ‘ZS11’. Gene expression levels were estimated using logarithmic transformed FPKM values. The differentially expressed genes (DEGs) between infected and control samples were determined with the DEGseq2 (adjusted <em>p</em> &lt; 0.05 and |log<sub>2</sub>(fold-change)| ≥ 1) and displayed using TBtools (Vers. 2.056).</p></section><section id="sec4dot8-ijms-26-02685"><h3 class="pmc_sec_title">4.8. qRT-PCR Analysis of Pathogenesis-Related (PR) Genes in A. thaliana</h3>
<p>Root samples from <em>Arabidopsis</em> Col-0 and <em>perk9</em> infected with <em>P. brassicae</em> and uninfected controls were harvested at 14 days after inoculation for qRT-PCR study. Total RNA was extracted from control and inoculated roots using the FastPure Universal Plant Total RNA Isolation Kit (Vazyme, Nanjing, China). First-strand cDNA was synthesized with HiScript IV RT SuperMix for qPCR (Vazyme). Gene-specific primers are listed in <a class="usa-link" href="#app1-ijms-26-02685">Supplementary Table S1</a>. <em>Actin2</em> was used as reference gene. After running the qRT-PCR reactions, Ct (cycle threshold) values were obtained for each sample. The relative expression level was determined using the 2<sup>−ΔΔCt</sup> method [<a aria-describedby="B57-ijms-26-02685" class="usa-link" href="#B57-ijms-26-02685">57</a>]. The data were presented as the mean ± standard error (SE) based on three technical replicates of three biological replicates. Significant difference was determined at a <em>p</em>-value &lt; 0.05.</p></section></section><section id="sec5-ijms-26-02685"><h2 class="pmc_sec_title">5. Conclusions</h2>
<p>The study emphasized the evolutionary dynamics, structural diversity, and functional roles of the <em>BnPERK</em> gene family. It suggested that <em>BnPERK</em> genes are involved in various physiological processes, including growth regulation, stress responses, and the defense against pathogens. The transcriptional downregulation of <em>BnPERK09A1</em> and <em>BnPERK09C1</em> upon <em>P. brassicae</em> infection and the potential involvement of these genes in clubroot disease response further emphasize their significance in plant immunity. The identification of <em>BnPERK09A1</em> and <em>BnPERK09C1</em> as potential targets for disease management strategies and their putative role in signaling networks could contribute to the resistance improvement of Brassicaceae crops.</p></section><section class="app" id="app1-ijms-26-02685"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<p>The following supporting information can be downloaded at: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.mdpi.com/article/10.3390/ijms26062685/s1" rel="noopener noreferrer" target="_blank">https://www.mdpi.com/article/10.3390/ijms26062685/s1</a>.</p>
<section class="sm xbox font-sm" id="ijms-26-02685-s001"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11942576/bin/ijms-26-02685-s001.zip">ijms-26-02685-s001.zip</a><sup> (1.8MB, zip) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>Validation, L.R.; Investigation, Z.Z., T.F. and C.Z.; Resources, L.Z.; Data curation, Z.Z.; Writing–original draft, Z.Z. and L.X.; Writing–review &amp; editing, L.X.; Visualization, Z.Z. and T.F.; Supervision, C.T. and L.L.; Project administration, F.L.; Funding acquisition, C.T. and L.X. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Institutional Review Board Statement</h2>
<p>Not applicable.</p></section><section id="notes3"><h2 class="pmc_sec_title">Informed Consent Statement</h2>
<p>Not applicable.</p></section><section id="notes4"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>Data are contained within the article and <a class="usa-link" href="#app1-ijms-26-02685">Supplementary Materials</a>.</p></section><section id="notes5"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflict of interest.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This research was supported by the National Key Research and Development Program of China (2023YFF1000700), the National Natural Science Foundation of China (32370693), and the Innovation Program of Chinese Academy of Agricultural Sciences (CAAS-CSIAF-202402).</p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn2"><p><strong>Disclaimer/Publisher’s Note:</strong> The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.</p></div></div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1-ijms-26-02685">
<span class="label">1.</span><cite>Morris E.R., Walker J.C. Receptor-like protein kinases: The keys to response. Curr. Opin. Plant Biol. 2003;6:339–342. doi: 10.1016/S1369-5266(03)00055-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S1369-5266(03)00055-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12873528/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Plant%20Biol.&amp;title=Receptor-like%20protein%20kinases:%20The%20keys%20to%20response&amp;author=E.R.%20Morris&amp;author=J.C.%20Walker&amp;volume=6&amp;publication_year=2003&amp;pages=339-342&amp;pmid=12873528&amp;doi=10.1016/S1369-5266(03)00055-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2-ijms-26-02685">
<span class="label">2.</span><cite>Cui Y.W., Lu X.T., Gou X.P. Receptor-like protein kinases in plant reproduction: Current understanding and future perspectives. Plant Commun. 2022;3:100273. doi: 10.1016/j.xplc.2021.100273.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.xplc.2021.100273" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8760141/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35059634/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Commun.&amp;title=Receptor-like%20protein%20kinases%20in%20plant%20reproduction:%20Current%20understanding%20and%20future%20perspectives&amp;author=Y.W.%20Cui&amp;author=X.T.%20Lu&amp;author=X.P.%20Gou&amp;volume=3&amp;publication_year=2022&amp;pages=100273&amp;pmid=35059634&amp;doi=10.1016/j.xplc.2021.100273&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3-ijms-26-02685">
<span class="label">3.</span><cite>Wu Y., Zhou J.M. Receptor-like kinases in plant innate immunity. J. Integr. Plant Biol. 2013;55:1271–1286. doi: 10.1111/jipb.12123.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/jipb.12123" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24308571/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Integr.%20Plant%20Biol.&amp;title=Receptor-like%20kinases%20in%20plant%20innate%20immunity&amp;author=Y.%20Wu&amp;author=J.M.%20Zhou&amp;volume=55&amp;publication_year=2013&amp;pages=1271-1286&amp;pmid=24308571&amp;doi=10.1111/jipb.12123&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4-ijms-26-02685">
<span class="label">4.</span><cite>Dievart A., Gottin C., Périn C., Ranwez V., Chantret N. Origin and Diversity of Plant Receptor-Like Kinases. Annu. Rev. Plant Biol. 2020;71:131–156. doi: 10.1146/annurev-arplant-073019-025927.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev-arplant-073019-025927" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32186895/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Plant%20Biol.&amp;title=Origin%20and%20Diversity%20of%20Plant%20Receptor-Like%20Kinases&amp;author=A.%20Dievart&amp;author=C.%20Gottin&amp;author=C.%20P%C3%A9rin&amp;author=V.%20Ranwez&amp;author=N.%20Chantret&amp;volume=71&amp;publication_year=2020&amp;pages=131-156&amp;pmid=32186895&amp;doi=10.1146/annurev-arplant-073019-025927&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5-ijms-26-02685">
<span class="label">5.</span><cite>Invernizzi M., Hanemian M., Keller J., Libourel C., Roby D. PERKing up our understanding of the proline-rich extensin-like receptor kinases, a forgotten plant receptor kinase family. New Phytol. 2022;235:875–884. doi: 10.1111/nph.18166.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/nph.18166" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35451507/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=PERKing%20up%20our%20understanding%20of%20the%20proline-rich%20extensin-like%20receptor%20kinases,%20a%20forgotten%20plant%20receptor%20kinase%20family&amp;author=M.%20Invernizzi&amp;author=M.%20Hanemian&amp;author=J.%20Keller&amp;author=C.%20Libourel&amp;author=D.%20Roby&amp;volume=235&amp;publication_year=2022&amp;pages=875-884&amp;pmid=35451507&amp;doi=10.1111/nph.18166&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6-ijms-26-02685">
<span class="label">6.</span><cite>Bai L., Zhou Y., Song C.P. Arabidopsis proline-rich extensin-like receptor kinase 4 modulates the early event toward abscisic acid response in root tip growth. Plant Signal. Behav. 2009;4:1075–1077. doi: 10.4161/psb.4.11.9739.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4161/psb.4.11.9739" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2819518/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20009554/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signal.%20Behav.&amp;title=Arabidopsis%20proline-rich%20extensin-like%20receptor%20kinase%204%20modulates%20the%20early%20event%20toward%20abscisic%20acid%20response%20in%20root%20tip%20growth&amp;author=L.%20Bai&amp;author=Y.%20Zhou&amp;author=C.P.%20Song&amp;volume=4&amp;publication_year=2009&amp;pages=1075-1077&amp;pmid=20009554&amp;doi=10.4161/psb.4.11.9739&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7-ijms-26-02685">
<span class="label">7.</span><cite>Silva N.F., Goring D.R. The proline-rich, extensin-like receptor kinase-1 (PERK1) gene is rapidly induced by wounding. Plant Mol. Biol. 2002;50:667–685. doi: 10.1023/A:1019951120788.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1023/A:1019951120788" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12374299/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol.%20Biol.&amp;title=The%20proline-rich,%20extensin-like%20receptor%20kinase-1%20(PERK1)%20gene%20is%20rapidly%20induced%20by%20wounding&amp;author=N.F.%20Silva&amp;author=D.R.%20Goring&amp;volume=50&amp;publication_year=2002&amp;pages=667-685&amp;pmid=12374299&amp;doi=10.1023/A:1019951120788&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8-ijms-26-02685">
<span class="label">8.</span><cite>Uemura T., Hachisu M., Desaki Y., Ito A., Hoshino R., Sano Y., Nozawa A., Mujiono K., Galis I., Yoshida A., et al.  Soy and Arabidopsis receptor-like kinases respond to polysaccharide signals from Spodoptera species and mediate herbivore resistance. Commun. Biol. 2020;3:224.  doi: 10.1038/s42003-020-0959-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s42003-020-0959-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7210110/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32385340/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Commun.%20Biol.&amp;title=Soy%20and%20Arabidopsis%20receptor-like%20kinases%20respond%20to%20polysaccharide%20signals%20from%20Spodoptera%20species%20and%20mediate%20herbivore%20resistance&amp;author=T.%20Uemura&amp;author=M.%20Hachisu&amp;author=Y.%20Desaki&amp;author=A.%20Ito&amp;author=R.%20Hoshino&amp;volume=3&amp;publication_year=2020&amp;pages=224&amp;pmid=32385340&amp;doi=10.1038/s42003-020-0959-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9-ijms-26-02685">
<span class="label">9.</span><cite>Xue C.W., Li W.F., Shen R.F., Lan P. PERK13 modulates phosphate deficiencyinduced root hair elongation in Arabidopsis. Plant Sci. 2021;312:111060. doi: 10.1016/j.plantsci.2021.111060.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.plantsci.2021.111060" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34620427/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Sci.&amp;title=PERK13%20modulates%20phosphate%20deficiencyinduced%20root%20hair%20elongation%20in%20Arabidopsis&amp;author=C.W.%20Xue&amp;author=W.F.%20Li&amp;author=R.F.%20Shen&amp;author=P.%20Lan&amp;volume=312&amp;publication_year=2021&amp;pages=111060&amp;pmid=34620427&amp;doi=10.1016/j.plantsci.2021.111060&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10-ijms-26-02685">
<span class="label">10.</span><cite>Bai L., Zhang G.Z., Zhou Y., Zhang Z.P., Wang W., Du Y.Y., Wu Z.Y., Song C.P. Plasma membrane-associated proline-rich extensin-like receptor kinase 4, a novel regulator of Ca signalling, is required for abscisic acid responses in. Plant J. 2009;60:314–327. doi: 10.1111/j.1365-313X.2009.03956.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2009.03956.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19566594/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=Plasma%20membrane-associated%20proline-rich%20extensin-like%20receptor%20kinase%204,%20a%20novel%20regulator%20of%20Ca%20signalling,%20is%20required%20for%20abscisic%20acid%20responses%20in&amp;author=L.%20Bai&amp;author=G.Z.%20Zhang&amp;author=Y.%20Zhou&amp;author=Z.P.%20Zhang&amp;author=W.%20Wang&amp;volume=60&amp;publication_year=2009&amp;pages=314-327&amp;pmid=19566594&amp;doi=10.1111/j.1365-313X.2009.03956.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11-ijms-26-02685">
<span class="label">11.</span><cite>Nakhamchik A., Zhao Z.Y., Provart N.J., Shiu S.H., Keatley S.K., Cameron R.K., Goring D.R. A comprehensive expression analysis of the Arabidopsis proline-rich extensin-like receptor kinase gene family using bioinformatic and experimental approaches. Plant Cell Physiol. 2004;45:1875–1881. doi: 10.1093/pcp/pch206.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pch206" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15653807/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol.&amp;title=A%20comprehensive%20expression%20analysis%20of%20the%20Arabidopsis%20proline-rich%20extensin-like%20receptor%20kinase%20gene%20family%20using%20bioinformatic%20and%20experimental%20approaches&amp;author=A.%20Nakhamchik&amp;author=Z.Y.%20Zhao&amp;author=N.J.%20Provart&amp;author=S.H.%20Shiu&amp;author=S.K.%20Keatley&amp;volume=45&amp;publication_year=2004&amp;pages=1875-1881&amp;pmid=15653807&amp;doi=10.1093/pcp/pch206&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12-ijms-26-02685">
<span class="label">12.</span><cite>Kesawat M.S., Kherawat B.S., Singh A., Dey P., Routray S., Mohapatra C., Saha D., Ram C., Siddique K.H.M., Kumar A., et al.  Genome-wide analysis and characterization of the proline-rich extensin-like receptor kinases (PERKs) gene family reveals their role in different developmental stages and stress conditions in wheat (Triticum aestivum L.) Plants. 2022;11:496.  doi: 10.3390/plants11040496.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/plants11040496" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8880425/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35214830/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Genome-wide%20analysis%20and%20characterization%20of%20the%20proline-rich%20extensin-like%20receptor%20kinases%20(PERKs)%20gene%20family%20reveals%20their%20role%20in%20different%20developmental%20stages%20and%20stress%20conditions%20in%20wheat%20(Triticum%20aestivum%20L.)&amp;author=M.S.%20Kesawat&amp;author=B.S.%20Kherawat&amp;author=A.%20Singh&amp;author=P.%20Dey&amp;author=S.%20Routray&amp;volume=11&amp;publication_year=2022&amp;pages=496&amp;pmid=35214830&amp;doi=10.3390/plants11040496&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13-ijms-26-02685">
<span class="label">13.</span><cite>Qanmber G., Liu J., Yu D.Q., Liu Z., Lu L.L., Mo H.J., Ma S.Y., Wang Z., Yang Z.R. Genome-wide identification and characterization of the PERK gene family in Gossypium hirsutum reveals gene duplication and functional divergence. Int. J. Mol. Sci. 2019;20:1750.  doi: 10.3390/ijms20071750.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/ijms20071750" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6479967/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30970629/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Genome-wide%20identification%20and%20characterization%20of%20the%20PERK%20gene%20family%20in%20Gossypium%20hirsutum%20reveals%20gene%20duplication%20and%20functional%20divergence&amp;author=G.%20Qanmber&amp;author=J.%20Liu&amp;author=D.Q.%20Yu&amp;author=Z.%20Liu&amp;author=L.L.%20Lu&amp;volume=20&amp;publication_year=2019&amp;pages=1750&amp;pmid=30970629&amp;doi=10.3390/ijms20071750&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14-ijms-26-02685">
<span class="label">14.</span><cite>Feng P., Shi J.Q., Zhang T., Zhong Y.Q., Zhang L.S., Yu G.L., Zhang T.Q., Zhu X.Y., Xing Y.D., Yin W.Z., et al.  Zebra leaf 15, a receptor-like protein kinase involved in moderate low temperature signaling pathway in rice. Rice. 2019;12:83. doi: 10.1186/s12284-019-0339-1.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12284-019-0339-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6858429/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31732821/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Rice&amp;title=Zebra%20leaf%2015,%20a%20receptor-like%20protein%20kinase%20involved%20in%20moderate%20low%20temperature%20signaling%20pathway%20in%20rice&amp;author=P.%20Feng&amp;author=J.Q.%20Shi&amp;author=T.%20Zhang&amp;author=Y.Q.%20Zhong&amp;author=L.S.%20Zhang&amp;volume=12&amp;publication_year=2019&amp;pages=83&amp;pmid=31732821&amp;doi=10.1186/s12284-019-0339-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15-ijms-26-02685">
<span class="label">15.</span><cite>Haffani Y.Z., Silva-Gagliardi N.F., Sewter S.K., Aldea M.G., Zhao Z.Y., Nakhamchik A., Cameron R.K., Goring D.R. Altered expression of PERK receptor kinases in leads to changes in Arabidopsis growth and floral organ formation. Plant Signal. Behav. 2006;1:251–260. doi: 10.4161/psb.1.5.3324.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4161/psb.1.5.3324" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2634126/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19516986/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signal.%20Behav.&amp;title=Altered%20expression%20of%20PERK%20receptor%20kinases%20in%20leads%20to%20changes%20in%20Arabidopsis%20growth%20and%20floral%20organ%20formation&amp;author=Y.Z.%20Haffani&amp;author=N.F.%20Silva-Gagliardi&amp;author=S.K.%20Sewter&amp;author=M.G.%20Aldea&amp;author=Z.Y.%20Zhao&amp;volume=1&amp;publication_year=2006&amp;pages=251-260&amp;pmid=19516986&amp;doi=10.4161/psb.1.5.3324&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16-ijms-26-02685">
<span class="label">16.</span><cite>Kesawat M.S., Kherawat B.S., Katara J.L., Parameswaran C., Misra N., Kumar M., Chung S.M., Alamri S., Siddiqui M.H. Genome-wide analysis of proline-rich extensin-like receptor kinases (PERKs) gene family reveals their roles in plant development and stress conditions in Oryza sativa L. Plant Sci. 2023;334:111749. doi: 10.1016/j.plantsci.2023.111749.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.plantsci.2023.111749" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37244501/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Sci.&amp;title=Genome-wide%20analysis%20of%20proline-rich%20extensin-like%20receptor%20kinases%20(PERKs)%20gene%20family%20reveals%20their%20roles%20in%20plant%20development%20and%20stress%20conditions%20in%20Oryza%20sativa%20L.&amp;author=M.S.%20Kesawat&amp;author=B.S.%20Kherawat&amp;author=J.L.%20Katara&amp;author=C.%20Parameswaran&amp;author=N.%20Misra&amp;volume=334&amp;publication_year=2023&amp;pages=111749&amp;pmid=37244501&amp;doi=10.1016/j.plantsci.2023.111749&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17-ijms-26-02685">
<span class="label">17.</span><cite>Chalhoub B., Denoeud F., Liu S.Y., Parkin I.A.P., Tang H.B., Wang X.Y., Chiquet J., Belcram H., Tong C.B., Samans B., et al.  Early allopolyploid evolution in the post-Neolithic Brassica napus oilseed genome. Science. 2014;345:950–953. doi: 10.1126/science.1253435.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1253435" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25146293/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Early%20allopolyploid%20evolution%20in%20the%20post-Neolithic%20Brassica%20napus%20oilseed%20genome&amp;author=B.%20Chalhoub&amp;author=F.%20Denoeud&amp;author=S.Y.%20Liu&amp;author=I.A.P.%20Parkin&amp;author=H.B.%20Tang&amp;volume=345&amp;publication_year=2014&amp;pages=950-953&amp;pmid=25146293&amp;doi=10.1126/science.1253435&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18-ijms-26-02685">
<span class="label">18.</span><cite>Song X.M., Wei Y.P., Xiao D., Gong K., Sun P.C., Ren Y.M., Yuan J.Q., Wu T., Yang Q.H., Li X.Y., et al.  Genome characterization clarifies U’s triangle model of evolution and polyploidy in Brassica. Plant Physiol. 2021;186:388–406. doi: 10.1093/plphys/kiab048.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/plphys/kiab048" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8154070/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33599732/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Genome%20characterization%20clarifies%20U%E2%80%99s%20triangle%20model%20of%20evolution%20and%20polyploidy%20in%20Brassica&amp;author=X.M.%20Song&amp;author=Y.P.%20Wei&amp;author=D.%20Xiao&amp;author=K.%20Gong&amp;author=P.C.%20Sun&amp;volume=186&amp;publication_year=2021&amp;pages=388-406&amp;pmid=33599732&amp;doi=10.1093/plphys/kiab048&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19-ijms-26-02685">
<span class="label">19.</span><cite>Chen G.H., Wang J., Wang H., Wang C.G., Tang X.Y., Li J., Zhang L., Song J.H., Hou J.F., Yuan L.Y. Genome-wide analysis of proline-rich extension-like receptor protein kinase (PERK) in Brassica rapa and its association with the pollen development. BMC Genom. 2020;21:401.  doi: 10.1186/s12864-020-06802-9.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12864-020-06802-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7296749/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32539701/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&amp;title=Genome-wide%20analysis%20of%20proline-rich%20extension-like%20receptor%20protein%20kinase%20(PERK)%20in%20Brassica%20rapa%20and%20its%20association%20with%20the%20pollen%20development&amp;author=G.H.%20Chen&amp;author=J.%20Wang&amp;author=H.%20Wang&amp;author=C.G.%20Wang&amp;author=X.Y.%20Tang&amp;volume=21&amp;publication_year=2020&amp;pages=401&amp;pmid=32539701&amp;doi=10.1186/s12864-020-06802-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B20-ijms-26-02685">
<span class="label">20.</span><cite>Gu B.J., Tong Y.K., Wang Y.Y., Zhang M.L., Ma G.J., Wu X.Q., Zhang J.F., Xu F., Li J., Ren F. Genome-wide evolution and expression analysis of the MYB-CC gene family in Brassica spp. PeerJ. 2022;10:e12882. doi: 10.7717/peerj.12882.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.7717/peerj.12882" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8884064/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35237467/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Genome-wide%20evolution%20and%20expression%20analysis%20of%20the%20MYB-CC%20gene%20family%20in%20Brassica%20spp.&amp;author=B.J.%20Gu&amp;author=Y.K.%20Tong&amp;author=Y.Y.%20Wang&amp;author=M.L.%20Zhang&amp;author=G.J.%20Ma&amp;volume=10&amp;publication_year=2022&amp;pages=e12882&amp;pmid=35237467&amp;doi=10.7717/peerj.12882&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21-ijms-26-02685">
<span class="label">21.</span><cite>Roy B.C., Shukla N., Gachhui R., Mukherjee A. Genome-wide analysis of glutamate receptor gene family in allopolyploid Brassica napus and its diploid progenitors. Genetica. 2023;151:293–310. doi: 10.1007/s10709-023-00192-y.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s10709-023-00192-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37624443/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genetica&amp;title=Genome-wide%20analysis%20of%20glutamate%20receptor%20gene%20family%20in%20allopolyploid%20Brassica%20napus%20and%20its%20diploid%20progenitors&amp;author=B.C.%20Roy&amp;author=N.%20Shukla&amp;author=R.%20Gachhui&amp;author=A.%20Mukherjee&amp;volume=151&amp;publication_year=2023&amp;pages=293-310&amp;pmid=37624443&amp;doi=10.1007/s10709-023-00192-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22-ijms-26-02685">
<span class="label">22.</span><cite>Xie M.L., Zuo R., Bai Z.T., Yang L.L., Zhao C.J., Gao F., Cheng X.H., Huang J.Y., Liu Y.Y., Li Y., et al.  Genome-wide characterization of serine/arginine-rich gene family and its genetic effects on agronomic traits of Brassica napus. Front. Plant Sci. 2022;13:829668.  doi: 10.3389/fpls.2022.829668.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2022.829668" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8889041/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35251101/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=Genome-wide%20characterization%20of%20serine/arginine-rich%20gene%20family%20and%20its%20genetic%20effects%20on%20agronomic%20traits%20of%20Brassica%20napus&amp;author=M.L.%20Xie&amp;author=R.%20Zuo&amp;author=Z.T.%20Bai&amp;author=L.L.%20Yang&amp;author=C.J.%20Zhao&amp;volume=13&amp;publication_year=2022&amp;pages=829668&amp;pmid=35251101&amp;doi=10.3389/fpls.2022.829668&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23-ijms-26-02685">
<span class="label">23.</span><cite>Albalat R., Cañestro C. Evolution by gene loss. Nat. Rev. Genet. 2016;17:379–391. doi: 10.1038/nrg.2016.39.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nrg.2016.39" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27087500/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=Evolution%20by%20gene%20loss&amp;author=R.%20Albalat&amp;author=C.%20Ca%C3%B1estro&amp;volume=17&amp;publication_year=2016&amp;pages=379-391&amp;pmid=27087500&amp;doi=10.1038/nrg.2016.39&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24-ijms-26-02685">
<span class="label">24.</span><cite>Liu J.F., Zhang Y., Lei X.Y., Zhang Z.M. Natural selection of protein structural and functional properties: A single nucleotide polymorphism perspective. Genome Biol. 2008;9:R69.  doi: 10.1186/gb-2008-9-4-r69.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/gb-2008-9-4-r69" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2643940/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18397526/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Natural%20selection%20of%20protein%20structural%20and%20functional%20properties:%20A%20single%20nucleotide%20polymorphism%20perspective&amp;author=J.F.%20Liu&amp;author=Y.%20Zhang&amp;author=X.Y.%20Lei&amp;author=Z.M.%20Zhang&amp;volume=9&amp;publication_year=2008&amp;pages=R69&amp;pmid=18397526&amp;doi=10.1186/gb-2008-9-4-r69&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25-ijms-26-02685">
<span class="label">25.</span><cite>Barberis E., Marengo E., Manfredi M. Protein subcellular localization prediction. Methods Mol. Biol. 2021;2361:197–212. doi: 10.1007/978-1-0716-1641-3_12.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-1-0716-1641-3_12" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34236663/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&amp;title=Protein%20subcellular%20localization%20prediction&amp;author=E.%20Barberis&amp;author=E.%20Marengo&amp;author=M.%20Manfredi&amp;volume=2361&amp;publication_year=2021&amp;pages=197-212&amp;pmid=34236663&amp;doi=10.1007/978-1-0716-1641-3_12&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26-ijms-26-02685">
<span class="label">26.</span><cite>Zhang C.X., Freddolino P.L., Zhang Y. COFACTOR: Improved protein function prediction by combining structure, sequence and protein-protein interaction information. Nucleic Acids Res. 2017;45:291–299. doi: 10.1093/nar/gkx366.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkx366" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5793808/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28472402/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=COFACTOR:%20Improved%20protein%20function%20prediction%20by%20combining%20structure,%20sequence%20and%20protein-protein%20interaction%20information&amp;author=C.X.%20Zhang&amp;author=P.L.%20Freddolino&amp;author=Y.%20Zhang&amp;volume=45&amp;publication_year=2017&amp;pages=291-299&amp;pmid=28472402&amp;doi=10.1093/nar/gkx366&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27-ijms-26-02685">
<span class="label">27.</span><cite>Borassi C., Sede A.R., Mecchia M.A., Mangano S., Marzol E., Denita-Juarez S.P., Salter J.D.S., Velasquez S.M., Muschietti J.P., Estevez J.M. Proline-rich extensin-like receptor kinases PERK5 and PERK12 are involved in pollen tube growth. FEBS Lett. 2021;595:2593–2607. doi: 10.1002/1873-3468.14185.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/1873-3468.14185" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34427925/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&amp;title=Proline-rich%20extensin-like%20receptor%20kinases%20PERK5%20and%20PERK12%20are%20involved%20in%20pollen%20tube%20growth&amp;author=C.%20Borassi&amp;author=A.R.%20Sede&amp;author=M.A.%20Mecchia&amp;author=S.%20Mangano&amp;author=E.%20Marzol&amp;volume=595&amp;publication_year=2021&amp;pages=2593-2607&amp;pmid=34427925&amp;doi=10.1002/1873-3468.14185&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28-ijms-26-02685">
<span class="label">28.</span><cite>Parkin I.A.P., Gulden S.M., Sharpe A.G., Lukens L., Trick M., Osborn T.C., Lydiate D.J. Segmental structure of the genome based on comparative analysis with Arabidopsis thaliana. Genetics. 2005;171:765–781. doi: 10.1534/genetics.105.042093.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1534/genetics.105.042093" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1456786/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16020789/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Segmental%20structure%20of%20the%20genome%20based%20on%20comparative%20analysis%20with%20Arabidopsis%20thaliana&amp;author=I.A.P.%20Parkin&amp;author=S.M.%20Gulden&amp;author=A.G.%20Sharpe&amp;author=L.%20Lukens&amp;author=M.%20Trick&amp;volume=171&amp;publication_year=2005&amp;pages=765-781&amp;pmid=16020789&amp;doi=10.1534/genetics.105.042093&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29-ijms-26-02685">
<span class="label">29.</span><cite>Lukens L.N., Quijada P.A., Udall J., Pires J.C., Schranz M.E., Osborn T.C. Genome redundancy and plasticity within ancient and recent Brassica crop species. Biol. J. Linn. Soc. 2004;82:665–674. doi: 10.1111/j.1095-8312.2004.00352.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1095-8312.2004.00352.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biol.%20J.%20Linn.%20Soc.&amp;title=Genome%20redundancy%20and%20plasticity%20within%20ancient%20and%20recent%20Brassica%20crop%20species&amp;author=L.N.%20Lukens&amp;author=P.A.%20Quijada&amp;author=J.%20Udall&amp;author=J.C.%20Pires&amp;author=M.E.%20Schranz&amp;volume=82&amp;publication_year=2004&amp;pages=665-674&amp;doi=10.1111/j.1095-8312.2004.00352.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30-ijms-26-02685">
<span class="label">30.</span><cite>Wittkopp P.J., Kalay G. Cis-regulatory elements: Molecular mechanisms and evolutionary processes underlying divergence. Nat. Rev. Genet. 2012;13:59–69. doi: 10.1038/nrg3095.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nrg3095" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22143240/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=Cis-regulatory%20elements:%20Molecular%20mechanisms%20and%20evolutionary%20processes%20underlying%20divergence&amp;author=P.J.%20Wittkopp&amp;author=G.%20Kalay&amp;volume=13&amp;publication_year=2012&amp;pages=59-69&amp;pmid=22143240&amp;doi=10.1038/nrg3095&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31-ijms-26-02685">
<span class="label">31.</span><cite>Bulgakov V.P., Wu H.C., Jinn T.L. Coordination of ABA and chaperone signaling in plant stress responses. Trends Plant Sci. 2019;24:636–651. doi: 10.1016/j.tplants.2019.04.004.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tplants.2019.04.004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31085125/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci.&amp;title=Coordination%20of%20ABA%20and%20chaperone%20signaling%20in%20plant%20stress%20responses&amp;author=V.P.%20Bulgakov&amp;author=H.C.%20Wu&amp;author=T.L.%20Jinn&amp;volume=24&amp;publication_year=2019&amp;pages=636-651&amp;pmid=31085125&amp;doi=10.1016/j.tplants.2019.04.004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32-ijms-26-02685">
<span class="label">32.</span><cite>Wu K., Xu H., Gao X.H., Fu X.D. New insights into gibberellin signaling in regulating plant growth-metabolic coordination. Curr. Opin. Plant Biol. 2021;63:102074.  doi: 10.1016/j.pbi.2021.102074.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.pbi.2021.102074" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34217918/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Plant%20Biol.&amp;title=New%20insights%20into%20gibberellin%20signaling%20in%20regulating%20plant%20growth-metabolic%20coordination&amp;author=K.%20Wu&amp;author=H.%20Xu&amp;author=X.H.%20Gao&amp;author=X.D.%20Fu&amp;volume=63&amp;publication_year=2021&amp;pages=102074&amp;pmid=34217918&amp;doi=10.1016/j.pbi.2021.102074&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B33-ijms-26-02685">
<span class="label">33.</span><cite>Gomes G.L.B., Scortecci K.C. Auxin and its role in plant development: Structure, signalling, regulation and response mechanisms. Plant Biol. 2021;23:894–904. doi: 10.1111/plb.13303.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/plb.13303" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34396657/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biol.&amp;title=Auxin%20and%20its%20role%20in%20plant%20development:%20Structure,%20signalling,%20regulation%20and%20response%20mechanisms&amp;author=G.L.B.%20Gomes&amp;author=K.C.%20Scortecci&amp;volume=23&amp;publication_year=2021&amp;pages=894-904&amp;pmid=34396657&amp;doi=10.1111/plb.13303&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34-ijms-26-02685">
<span class="label">34.</span><cite>Wimalagunasekara S.S., Weeraman J.W.J.K., Tirimanne S., Fernando P.C. Protein-protein interaction (PPI) network analysis reveals important hub proteins and sub-network modules for root development in rice (Oryza sativa) J. Genet. Eng. Biotechnol. 2023;21:69.  doi: 10.1186/s43141-023-00515-8.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s43141-023-00515-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10225403/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37246172/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Genet.%20Eng.%20Biotechnol.&amp;title=Protein-protein%20interaction%20(PPI)%20network%20analysis%20reveals%20important%20hub%20proteins%20and%20sub-network%20modules%20for%20root%20development%20in%20rice%20(Oryza%20sativa)&amp;author=S.S.%20Wimalagunasekara&amp;author=J.W.J.K.%20Weeraman&amp;author=S.%20Tirimanne&amp;author=P.C.%20Fernando&amp;volume=21&amp;publication_year=2023&amp;pages=69&amp;pmid=37246172&amp;doi=10.1186/s43141-023-00515-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35-ijms-26-02685">
<span class="label">35.</span><cite>Borassi C., Sede A.R., Mecchia M.A., Salter J.D.S., Marzol E., Muschietti J.P., Estevez J.M. An update on cell surface proteins containing extensin-motifs. J. Exp. Bot. 2016;67:477–487. doi: 10.1093/jxb/erv455.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/erv455" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26475923/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=An%20update%20on%20cell%20surface%20proteins%20containing%20extensin-motifs&amp;author=C.%20Borassi&amp;author=A.R.%20Sede&amp;author=M.A.%20Mecchia&amp;author=J.D.S.%20Salter&amp;author=E.%20Marzol&amp;volume=67&amp;publication_year=2016&amp;pages=477-487&amp;pmid=26475923&amp;doi=10.1093/jxb/erv455&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36-ijms-26-02685">
<span class="label">36.</span><cite>Bacete L., Mélida H., Miedes E., Molina A. Plant cell wall-mediated immunity: Cell wall changes trigger disease resistance responses. Plant J. 2018;93:614–636. doi: 10.1111/tpj.13807.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.13807" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29266460/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=Plant%20cell%20wall-mediated%20immunity:%20Cell%20wall%20changes%20trigger%20disease%20resistance%20responses&amp;author=L.%20Bacete&amp;author=H.%20M%C3%A9lida&amp;author=E.%20Miedes&amp;author=A.%20Molina&amp;volume=93&amp;publication_year=2018&amp;pages=614-636&amp;pmid=29266460&amp;doi=10.1111/tpj.13807&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B37-ijms-26-02685">
<span class="label">37.</span><cite>Raja V., Majeed U., Kang H., Andrabi K.I., John R. Abiotic stress: Interplay between ROS, hormones and MAPKs. Environ. Exp. Bot. 2017;137:142–157. doi: 10.1016/j.envexpbot.2017.02.010.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.envexpbot.2017.02.010" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Exp.%20Bot.&amp;title=Abiotic%20stress:%20Interplay%20between%20ROS,%20hormones%20and%20MAPKs&amp;author=V.%20Raja&amp;author=U.%20Majeed&amp;author=H.%20Kang&amp;author=K.I.%20Andrabi&amp;author=R.%20John&amp;volume=137&amp;publication_year=2017&amp;pages=142-157&amp;doi=10.1016/j.envexpbot.2017.02.010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B38-ijms-26-02685">
<span class="label">38.</span><cite>Chen W., Li Y., Yan R.B., Ren L., Liu F., Zeng L.Y., Sun S.N., Yang H.H., Chen K.R., Xu L., et al.  SnRK1.1-mediated resistance of Arabidopsis thaliana to clubroot disease is inhibited by the novel Plasmodiophora brassicae effector PBZF1. Mol. Plant Pathol. 2021;22:1057–1069. doi: 10.1111/mpp.13095.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/mpp.13095" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8358996/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34165877/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Plant%20Pathol.&amp;title=SnRK1.1-mediated%20resistance%20of%20Arabidopsis%20thaliana%20to%20clubroot%20disease%20is%20inhibited%20by%20the%20novel%20Plasmodiophora%20brassicae%20effector%20PBZF1&amp;author=W.%20Chen&amp;author=Y.%20Li&amp;author=R.B.%20Yan&amp;author=L.%20Ren&amp;author=F.%20Liu&amp;volume=22&amp;publication_year=2021&amp;pages=1057-1069&amp;pmid=34165877&amp;doi=10.1111/mpp.13095&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B39-ijms-26-02685">
<span class="label">39.</span><cite>Kemen E., Jones J.D.G. Obligate biotroph parasitism: Can we link genomes to lifestyles? Trends Plant Sci. 2012;17:448–457. doi: 10.1016/j.tplants.2012.04.005.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tplants.2012.04.005" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22613788/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci.&amp;title=Obligate%20biotroph%20parasitism:%20Can%20we%20link%20genomes%20to%20lifestyles?&amp;author=E.%20Kemen&amp;author=J.D.G.%20Jones&amp;volume=17&amp;publication_year=2012&amp;pages=448-457&amp;pmid=22613788&amp;doi=10.1016/j.tplants.2012.04.005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B40-ijms-26-02685">
<span class="label">40.</span><cite>Kong L., Li X., Zhan Z., Piao Z. Sugar transporters in Plasmodiophora brassicae: Genome-wide identification and functional verification. Int. J. Mol. Sci. 2022;23:5264.  doi: 10.3390/ijms23095264.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/ijms23095264" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9099952/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35563657/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Sugar%20transporters%20in%20Plasmodiophora%20brassicae:%20Genome-wide%20identification%20and%20functional%20verification&amp;author=L.%20Kong&amp;author=X.%20Li&amp;author=Z.%20Zhan&amp;author=Z.%20Piao&amp;volume=23&amp;publication_year=2022&amp;pages=5264&amp;pmid=35563657&amp;doi=10.3390/ijms23095264&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B41-ijms-26-02685">
<span class="label">41.</span><cite>Ali S., Ganai B.A., Kamili A.N., Bhat A.A., Mir Z.A., Bhat J.A., Tyagi A., Islam S.T., Mushtaq M., Yadav P., et al.  Pathogenesis-related proteins and peptides as promising tools for engineering plants with multiple stress tolerance. Microbiol. Res. 2018;212:29–37. doi: 10.1016/j.micres.2018.04.008.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.micres.2018.04.008" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29853166/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Res.&amp;title=Pathogenesis-related%20proteins%20and%20peptides%20as%20promising%20tools%20for%20engineering%20plants%20with%20multiple%20stress%20tolerance&amp;author=S.%20Ali&amp;author=B.A.%20Ganai&amp;author=A.N.%20Kamili&amp;author=A.A.%20Bhat&amp;author=Z.A.%20Mir&amp;volume=212&amp;publication_year=2018&amp;pages=29-37&amp;pmid=29853166&amp;doi=10.1016/j.micres.2018.04.008&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B42-ijms-26-02685">
<span class="label">42.</span><cite>Lee D., Lal N.K., Lin Z.J.D., Ma S.S., Liu J., Castro B., Toruño T., Dinesh-Kumar S.P., Coaker G. Regulation of reactive oxygen species during plant immunity through phosphorylation and ubiquitination of RBOHD. Nat. Commun. 2020;11:1838. doi: 10.1038/s41467-020-15601-5.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41467-020-15601-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7160206/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32296066/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Regulation%20of%20reactive%20oxygen%20species%20during%20plant%20immunity%20through%20phosphorylation%20and%20ubiquitination%20of%20RBOHD&amp;author=D.%20Lee&amp;author=N.K.%20Lal&amp;author=Z.J.D.%20Lin&amp;author=S.S.%20Ma&amp;author=J.%20Liu&amp;volume=11&amp;publication_year=2020&amp;pages=1838&amp;pmid=32296066&amp;doi=10.1038/s41467-020-15601-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B43-ijms-26-02685">
<span class="label">43.</span><cite>Ludwig-Muller J. Auxin homeostasis, signaling, and interaction with other growth hormones during the clubroot disease of Brassicaceae. Plant Signal. Behav. 2014;9:e28593. doi: 10.4161/psb.28593.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4161/psb.28593" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4091609/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24699875/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signal.%20Behav.&amp;title=Auxin%20homeostasis,%20signaling,%20and%20interaction%20with%20other%20growth%20hormones%20during%20the%20clubroot%20disease%20of%20Brassicaceae&amp;author=J.%20Ludwig-Muller&amp;volume=9&amp;publication_year=2014&amp;pages=e28593&amp;pmid=24699875&amp;doi=10.4161/psb.28593&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B44-ijms-26-02685">
<span class="label">44.</span><cite>Xu L., Ren L., Chen K.R., Liu F., Fang X.P. Putative role of IAA during the early response of Brassica napus L. to Plasmodiophora brassicae. Eur. J. Plant Pathol. 2016;145:601–613. doi: 10.1007/s10658-016-0877-y.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s10658-016-0877-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Plant%20Pathol.&amp;title=Putative%20role%20of%20IAA%20during%20the%20early%20response%20of%20Brassica%20napus%20L.%20to%20Plasmodiophora%20brassicae&amp;author=L.%20Xu&amp;author=L.%20Ren&amp;author=K.R.%20Chen&amp;author=F.%20Liu&amp;author=X.P.%20Fang&amp;volume=145&amp;publication_year=2016&amp;pages=601-613&amp;doi=10.1007/s10658-016-0877-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B45-ijms-26-02685">
<span class="label">45.</span><cite>Wei X.C., Liao R.J., Zhang X.W., Zhao Y.Y., Xie Z.Q., Yang S.J., Su H.A., Wang Z.Y., Zhang L.Y., Tian B.M., et al.  Integrative transcriptome, miRNAs, degradome, and phytohormone analysis of Brassica rapa L. in response to Plasmodiophora brassicae. Int. J. Mol. Sci. 2023;24:2652.  doi: 10.3390/ijms24032414.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/ijms24032414" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9916777/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36768734/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Integrative%20transcriptome,%20miRNAs,%20degradome,%20and%20phytohormone%20analysis%20of%20Brassica%20rapa%20L.%20in%20response%20to%20Plasmodiophora%20brassicae&amp;author=X.C.%20Wei&amp;author=R.J.%20Liao&amp;author=X.W.%20Zhang&amp;author=Y.Y.%20Zhao&amp;author=Z.Q.%20Xie&amp;volume=24&amp;publication_year=2023&amp;pages=2652&amp;pmid=36768734&amp;doi=10.3390/ijms24032414&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B46-ijms-26-02685">
<span class="label">46.</span><cite>Yang Z.Q., Wang S.B., Wei L.L., Huang Y.M., Liu D.X., Jia Y.P., Luo C.F., Lin Y.C., Liang C.Y., Hu Y., et al.  BnIR: A multi-omics database with various tools for research and breeding. Mol. Plant. 2023;16:775–789. doi: 10.1016/j.molp.2023.03.007.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.molp.2023.03.007" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36919242/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Plant&amp;title=BnIR:%20A%20multi-omics%20database%20with%20various%20tools%20for%20research%20and%20breeding&amp;author=Z.Q.%20Yang&amp;author=S.B.%20Wang&amp;author=L.L.%20Wei&amp;author=Y.M.%20Huang&amp;author=D.X.%20Liu&amp;volume=16&amp;publication_year=2023&amp;pages=775-789&amp;pmid=36919242&amp;doi=10.1016/j.molp.2023.03.007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B47-ijms-26-02685">
<span class="label">47.</span><cite>Chou K.C., Shen H.B. A new method for predicting the subcellular localization of eukaryotic proteins with both single and multiple sites: Euk-mPLoc 2.0. PLoS ONE. 2010;5:e9931.  doi: 10.1371/journal.pone.0009931.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0009931" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2848569/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20368981/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=A%20new%20method%20for%20predicting%20the%20subcellular%20localization%20of%20eukaryotic%20proteins%20with%20both%20single%20and%20multiple%20sites:%20Euk-mPLoc%202.0&amp;author=K.C.%20Chou&amp;author=H.B.%20Shen&amp;volume=5&amp;publication_year=2010&amp;pages=e9931&amp;pmid=20368981&amp;doi=10.1371/journal.pone.0009931&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B48-ijms-26-02685">
<span class="label">48.</span><cite>Wang Y.P., Tang H.B., Wang X.Y., Sun Y., Joseph P.V., Paterson A.H. Detection of colinear blocks and synteny and evolutionary analyses based on utilization of MCScanX. Nat. Protoc. 2024;19:2206–2229. doi: 10.1038/s41596-024-00968-2.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41596-024-00968-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/38491145/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Detection%20of%20colinear%20blocks%20and%20synteny%20and%20evolutionary%20analyses%20based%20on%20utilization%20of%20MCScanX&amp;author=Y.P.%20Wang&amp;author=H.B.%20Tang&amp;author=X.Y.%20Wang&amp;author=Y.%20Sun&amp;author=P.V.%20Joseph&amp;volume=19&amp;publication_year=2024&amp;pages=2206-2229&amp;pmid=38491145&amp;doi=10.1038/s41596-024-00968-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B49-ijms-26-02685">
<span class="label">49.</span><cite>Krzywinski M., Schein J., Birol I., Connors J., Gascoyne R., Horsman D., Jones S.J., Marra M.A. Circos: An information aesthetic for comparative genomics. Genome Res. 2009;19:1639–1645. doi: 10.1101/gr.092759.109.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.092759.109" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2752132/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19541911/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Circos:%20An%20information%20aesthetic%20for%20comparative%20genomics&amp;author=M.%20Krzywinski&amp;author=J.%20Schein&amp;author=I.%20Birol&amp;author=J.%20Connors&amp;author=R.%20Gascoyne&amp;volume=19&amp;publication_year=2009&amp;pages=1639-1645&amp;pmid=19541911&amp;doi=10.1101/gr.092759.109&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B50-ijms-26-02685">
<span class="label">50.</span><cite>Zhang Z., Xiao J.F., Wu J.Y., Zhang H.Y., Liu G.M., Wang X.M., Dai L. ParaAT: A parallel tool for constructing multiple protein-coding DNA alignments. Biochem. Biophys. Res. Commun. 2012;419:779–781. doi: 10.1016/j.bbrc.2012.02.101.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.bbrc.2012.02.101" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22390928/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;title=ParaAT:%20A%20parallel%20tool%20for%20constructing%20multiple%20protein-coding%20DNA%20alignments&amp;author=Z.%20Zhang&amp;author=J.F.%20Xiao&amp;author=J.Y.%20Wu&amp;author=H.Y.%20Zhang&amp;author=G.M.%20Liu&amp;volume=419&amp;publication_year=2012&amp;pages=779-781&amp;pmid=22390928&amp;doi=10.1016/j.bbrc.2012.02.101&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B51-ijms-26-02685">
<span class="label">51.</span><cite>Chen C.J., Wu Y., Li J.W., Wang X., Zeng Z.H., Xu J., Liu Y.L., Feng J.T., Chen H., He Y.H., et al.  TBtools-II: A “one for all, all for one” bioinformatics platform for biological big-data mining. Mol. Plant. 2023;16:1733–1742. doi: 10.1016/j.molp.2023.09.010.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.molp.2023.09.010" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37740491/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Plant&amp;title=TBtools-II:%20A%20%E2%80%9Cone%20for%20all,%20all%20for%20one%E2%80%9D%20bioinformatics%20platform%20for%20biological%20big-data%20mining&amp;author=C.J.%20Chen&amp;author=Y.%20Wu&amp;author=J.W.%20Li&amp;author=X.%20Wang&amp;author=Z.H.%20Zeng&amp;volume=16&amp;publication_year=2023&amp;pages=1733-1742&amp;pmid=37740491&amp;doi=10.1016/j.molp.2023.09.010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B52-ijms-26-02685">
<span class="label">52.</span><cite>Tamura K., Stecher G., Kumar S. MEGA11 Molecular Evolutionary Genetics Analysis Version 11. Mol. Biol. Evol. 2021;38:3022–3027. doi: 10.1093/molbev/msab120.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/msab120" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8233496/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33892491/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=MEGA11%20Molecular%20Evolutionary%20Genetics%20Analysis%20Version%2011&amp;author=K.%20Tamura&amp;author=G.%20Stecher&amp;author=S.%20Kumar&amp;volume=38&amp;publication_year=2021&amp;pages=3022-3027&amp;pmid=33892491&amp;doi=10.1093/molbev/msab120&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B53-ijms-26-02685">
<span class="label">53.</span><cite>Lescot M., Déhais P., Thijs G., Marchal K., Moreau Y., Van de Peer Y., Rouzé P., Rombauts S. PlantCARE, a database of plant cis-acting regulatory elements and a portal to tools for in silico analysis of promoter sequences. Nucleic Acids Res. 2002;30:325–327. doi: 10.1093/nar/30.1.325.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/30.1.325" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC99092/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11752327/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=PlantCARE,%20a%20database%20of%20plant%20cis-acting%20regulatory%20elements%20and%20a%20portal%20to%20tools%20for%20in%20silico%20analysis%20of%20promoter%20sequences&amp;author=M.%20Lescot&amp;author=P.%20D%C3%A9hais&amp;author=G.%20Thijs&amp;author=K.%20Marchal&amp;author=Y.%20Moreau&amp;volume=30&amp;publication_year=2002&amp;pages=325-327&amp;pmid=11752327&amp;doi=10.1093/nar/30.1.325&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B54-ijms-26-02685">
<span class="label">54.</span><cite>Otasek D., Morris J.H., Bouças J., Pico A.R., Demchak B. Cytoscape Automation: Empowering workflow-based network analysis. Genome Biol. 2019;20:185.  doi: 10.1186/s13059-019-1758-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s13059-019-1758-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6717989/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31477170/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Cytoscape%20Automation:%20Empowering%20workflow-based%20network%20analysis&amp;author=D.%20Otasek&amp;author=J.H.%20Morris&amp;author=J.%20Bou%C3%A7as&amp;author=A.R.%20Pico&amp;author=B.%20Demchak&amp;volume=20&amp;publication_year=2019&amp;pages=185&amp;pmid=31477170&amp;doi=10.1186/s13059-019-1758-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B55-ijms-26-02685">
<span class="label">55.</span><cite>Cui X.B., Hu M., Yao S.L., Zhang Y.Y., Tang M.Q., Liu L.J., Cheng X.H., Tong C.B., Liu S.Y. BnaOmics: A comprehensive platform combining pan-genome and multi-omics data from Brassica napus. Plant Commun. 2023;4:100609. doi: 10.1016/j.xplc.2023.100609.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.xplc.2023.100609" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10504585/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37098652/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Commun.&amp;title=BnaOmics:%20A%20comprehensive%20platform%20combining%20pan-genome%20and%20multi-omics%20data%20from%20Brassica%20napus&amp;author=X.B.%20Cui&amp;author=M.%20Hu&amp;author=S.L.%20Yao&amp;author=Y.Y.%20Zhang&amp;author=M.Q.%20Tang&amp;volume=4&amp;publication_year=2023&amp;pages=100609&amp;pmid=37098652&amp;doi=10.1016/j.xplc.2023.100609&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B56-ijms-26-02685">
<span class="label">56.</span><cite>Xu L., Yang H., Ren L., Chen W., Liu L.J., Liu F., Zeng L.Y., Yan R.B., Chen K.R., Fang X.P. Jasmonic acid-mediated aliphatic glucosinolate metabolism is involved in clubroot disease development in Brassica napus L. Front. Plant Sci. 2018;9:750.  doi: 10.3389/fpls.2018.00750.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2018.00750" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5996939/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29922320/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=Jasmonic%20acid-mediated%20aliphatic%20glucosinolate%20metabolism%20is%20involved%20in%20clubroot%20disease%20development%20in%20Brassica%20napus%20L.&amp;author=L.%20Xu&amp;author=H.%20Yang&amp;author=L.%20Ren&amp;author=W.%20Chen&amp;author=L.J.%20Liu&amp;volume=9&amp;publication_year=2018&amp;pages=750&amp;pmid=29922320&amp;doi=10.3389/fpls.2018.00750&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B57-ijms-26-02685">
<span class="label">57.</span><cite>Pfaffl M.W. A new mathematical model for relative quantification in real-time RT-PCR. Nucleic Acids Res. 2001;29:e45. doi: 10.1093/nar/29.9.e45.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/29.9.e45" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC55695/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11328886/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=A%20new%20mathematical%20model%20for%20relative%20quantification%20in%20real-time%20RT-PCR&amp;author=M.W.%20Pfaffl&amp;volume=29&amp;publication_year=2001&amp;pages=e45&amp;pmid=11328886&amp;doi=10.1093/nar/29.9.e45&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11942576/bin/ijms-26-02685-s001.zip">ijms-26-02685-s001.zip</a><sup> (1.8MB, zip) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data are contained within the article and <a class="usa-link" href="#app1-ijms-26-02685">Supplementary Materials</a>.</p></section></section></section></section></article>
</main></body></html>