<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8450295/"/><title>Telomere length assessments of muscle stem cells in rodent and human skeletal muscle sections - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">STAR Protoc</button></div>. 2021 Sep 14;2(4):100830. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.xpro.2021.100830" rel="noopener noreferrer" target="_blank">10.1016/j.xpro.2021.100830</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Telomere length assessments of muscle stem cells in rodent and human skeletal muscle sections</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tichy%20ED%22%5BAuthor%5D"><span class="name western">Elisia D Tichy</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Elisia D Tichy</span></h3>
<div class="p">
<sup>1</sup>Department of Orthopaedic Surgery, Perelman School of Medicine, The University of Pennsylvania, Philadelphia, PA 19104, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tichy%20ED%22%5BAuthor%5D"><span class="name western">Elisia D Tichy</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>∗</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mourkioti%20F%22%5BAuthor%5D"><span class="name western">Foteini Mourkioti</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Foteini Mourkioti</span></h3>
<div class="p">
<sup>1</sup>Department of Orthopaedic Surgery, Perelman School of Medicine, The University of Pennsylvania, Philadelphia, PA 19104, USA</div>
<div class="p">
<sup>2</sup>Department of Cell and Developmental Biology, Perelman School of Medicine, The University of Pennsylvania, Philadelphia, PA 19104, USA</div>
<div class="p">
<sup>3</sup>Institute of Regenerative Medicine, Musculoskeletal Regeneration Program, Perelman School of Medicine, The University of Pennsylvania, Philadelphia, PA 19104, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mourkioti%20F%22%5BAuthor%5D"><span class="name western">Foteini Mourkioti</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>5,</sup><sup>∗∗</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>Department of Orthopaedic Surgery, Perelman School of Medicine, The University of Pennsylvania, Philadelphia, PA 19104, USA</div>
<div id="aff2">
<sup>2</sup>Department of Cell and Developmental Biology, Perelman School of Medicine, The University of Pennsylvania, Philadelphia, PA 19104, USA</div>
<div id="aff3">
<sup>3</sup>Institute of Regenerative Medicine, Musculoskeletal Regeneration Program, Perelman School of Medicine, The University of Pennsylvania, Philadelphia, PA 19104, USA</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>∗</sup><p class="display-inline">Corresponding author <span>etichy@pennmedicine.upenn.edu</span></p>
</div>
<div class="fn" id="cor2">
<sup>∗∗</sup><p class="display-inline">Corresponding author <span>fmour@pennmedicine.upenn.edu</span></p>
</div>
<div class="fn" id="fn1">
<sup>4</sup><p class="display-inline" id="ntpara0010">Technical contact</p>
</div>
<div class="fn" id="fn2">
<sup>5</sup><p class="display-inline" id="ntpara0015">Lead contact</p>
</div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Collection date 2021 Dec 17.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© 2021 The Author(s)</div>
<p>This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC8450295  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34585163/">34585163</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abs0010"><h2>Summary</h2>
<p>Measurements of telomere length in skeletal muscle stem cells (MuSCs), a rare cell population within muscles, provide insights into cellular dysfunction in diseased conditions. Here, we describe a protocol (<span class="text-underline">cryo</span>section <span class="text-underline">mu</span>scle <span class="text-underline">q</span>uantitative <span class="text-underline">f</span>luorescent <em><span class="text-underline">i</span>n <span class="text-underline">s</span>itu</em><span class="text-underline">h</span>ybridization) using skeletal muscle cryosections for assessments of telomere length in MuSCs, in their native environment. Using a free software, telomere length measurements are assessed on a single-cell level. We also provide methodology to perform data analyses in several ways.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a aria-describedby="bib7" class="usa-link" href="#bib7">Tichy et al. (2021)</a>.</p>
<section class="kwd-group" id="kwrds0010"><p><strong>Subject areas:</strong> Cell Biology, Cell isolation, Cell-based Assays, In Situ Hybridization, Microscopy, Stem Cells</p></section></section><section class="abstract" id="abs0015"><h2>Graphical abstract</h2>
<figure class="fig xbox font-sm" id="undfig1"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="graphic file with name fx1.jpg" class="graphic" height="664" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/7bdb12062669/fx1.jpg" width="664"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/undfig1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></figure></section><section class="abstract" id="abs0020"><h2>Highlights</h2>
<ul class="list" id="ulist0010" style="list-style-type:none">
<li id="u0010">
<span class="label">•</span><p class="display-inline" id="p0010">CRYO MuQ-FISH can measure telomere length on resident skeletal muscle stem cells</p>
</li>
<li id="u0015">
<span class="label">•</span><p class="display-inline" id="p0015">This protocol is ideal for archival samples</p>
</li>
<li id="u0020">
<span class="label">•</span><p class="display-inline" id="p0020">Telomere length quantitation can be normalized to DAPI or centromere intensity</p>
</li>
<li id="u0025">
<span class="label">•</span><p class="display-inline" id="p0025">Telomere outputs include both total sum intensity and mean intensity per nucleus</p>
</li>
</ul></section><section class="abstract" id="abs0025"><hr class="headless"/>
<p>Measurements of telomere length in skeletal muscle stem cells (MuSCs), a rare cell population within muscles, provide insights into cellular dysfunction in diseased conditions. Here, we describe a protocol (<span class="text-underline">cryo</span>section <span class="text-underline">mu</span>scle <span class="text-underline">q</span>uantitative <span class="text-underline">f</span>luorescent <em><span class="text-underline">i</span>n <span class="text-underline">s</span>itu</em><span class="text-underline">h</span>ybridization) using skeletal muscle cryosections for assessments of telomere length in MuSCs, in their native environment. Using a free software, telomere length measurements are assessed on a single-cell level. We also provide methodology to perform data analyses in several ways.</p></section><section id="sec1"><h2 class="pmc_sec_title">Before you begin</h2>
<section id="sec1.1"><h3 class="pmc_sec_title">Experimental design and considerations</h3>
<p id="p0035">Reductions in telomere length in muscle stem cells (MuSCs) have been recently discovered in chronic muscle injury conditions, including Duchenne muscular dystrophy (<a aria-describedby="bib8" class="usa-link" href="#bib8">Tichy et al., 2017</a>, <a aria-describedby="bib7" class="usa-link" href="#bib7">2021</a>), which results in a more severe disease phenotype (<a aria-describedby="bib6" class="usa-link" href="#bib6">Sacco et al., 2010</a>). Many common methods for conducting telomere length assessments in other cell and tissue types requires the isolation of primary cells, which may disrupt the tissue environment, requires large numbers of cells to obtain enough genomic DNA for appraisal [Telomere Restriction Fragment/TRF analysis (<a aria-describedby="bib5" class="usa-link" href="#bib5">Mender and Shay, 2015</a>)], or only provides the average telomere length over the entirety of the cell population [Telomere:single copy gene QPCR methodology (<a aria-describedby="bib1" class="usa-link" href="#bib1">Cawthon, 2002</a>)] . The quantitative fluorescent in-situ hybridization (Q-FISH) technique, which allows for assessment of telomere length based on the fluorescence intensity of a hybridized probe, coupled with semi-automated telomere length assessments using a free software (Telometer), provides robust information that can eliminate such limitations. Here, we provide a protocol where our previously established and validated muscle stem cell-specific Q-FISH protocol for measuring telomeres in FACS-isolated MuSCs [MuQ-FISH (<a aria-describedby="bib8" class="usa-link" href="#bib8">Tichy et al., 2017</a>)], has been altered and adapted for use on cryosections of murine skeletal muscle (CRYO MuQ-FISH), leaving the MuSC niche intact. This method can be beneficial for telomere length quantitation of archival samples and may also be adapted to examine telomere lengths in additional muscle-resident cell types, as well as cell types from other tissues. This protocol is also compatible with skeletal muscle cryosections derived from species other than mice but testing and optimization should be conducted by the end user.</p>
<blockquote class="text-italic"><p><strong><em>Note:</em></strong> This protocol utilizes skeletal muscle tissue from research animals and human cohorts. Ensure that the use of research animals has been approved by the appropriate institutional animal care and use committee (IACUC) or equivalent oversight body. Additionally, procurement and use of tissues derived from human sources should follow the appropriate organizational rules regarding informed consent and be approved by the end user’s institutional review board. Sample collection for this protocol was done according to institutional guidelines and regulations of the University of Pennsylvania.</p></blockquote>
<blockquote class="text-italic"><p><strong><em>Note:</em></strong> For telomere data extraction from images, the following software must be installed on the analysis computer: java se (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.oracle.com/java/technologies/javase-downloads.html" rel="noopener noreferrer" target="_blank">https://www.oracle.com/java/technologies/javase-downloads.html</a>) and the freeware Telometer (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://demarzolab.pathology.jhmi.edu/telometer/download.html" rel="noopener noreferrer" target="_blank">https://demarzolab.pathology.jhmi.edu/telometer/download.html</a>). Install software according to instructions listed on the websites.</p></blockquote>
<p id="p0040">This protocol includes the following steps: 1) Skeletal muscle preparation, embedding, sectioning, 2) CRYO MuQ-FISH, 3) image collection, 4) analysis and statistics</p></section><section id="sec1.2"><h3 class="pmc_sec_title">Skeletal muscle preparation, embedding, and sectioning</h3>
<blockquote class="text-italic"><p><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/51ca064f7ff6/fx2.gif"/><strong>Timing: 2 days</strong></p></blockquote>
<ul class="list" id="olist0010" style="list-style-type:none"><li id="o0010">
<span class="label">1.</span><p class="display-inline" id="p0050">Humanely euthanize mice according to an approved IACUC protocol methodology and institutional guidelines.</p>
</li></ul>
<blockquote class="text-italic"><p><strong><em>Note:</em></strong> This protocol uses male mice between 2 and 5 months of age. However, this protocol can be applied to samples derived from other ages and both sexes of mice.</p></blockquote>
<ul class="list" id="olist0015" style="list-style-type:none">
<li id="o0015">
<span class="label">2.</span><p class="display-inline" id="p0055">Spray the lower half of mouse body with 70% ethanol.</p>
</li>
<li id="o0020">
<span class="label">3.</span><p class="display-inline" id="p0060">Lay mouse on its back and completely remove skin from the back legs.</p>
</li>
<li id="o0025">
<span class="label">4.</span><p class="display-inline" id="p0065">Dissect desired hindlimb skeletal muscles for assessment.</p>
</li>
</ul>
<blockquote class="text-italic"><p><strong><em>Note:</em></strong> For downstream assessments, it is ideal to compare the same types of muscles, rather than mixing different anatomical groups. We routinely utilize either the Tibialis anterior or the gastrocnemius muscles for telomere length measurements.</p></blockquote>
<ul class="list" id="olist0020" style="list-style-type:none"><li id="o0030">
<span class="label">5.</span><div class="display-inline p" id="p0070">Fix and process tissues for embedding.<ul class="list" id="olist0025" style="list-style-type:none">
<li id="o0035">
<span class="label">a.</span><div class="display-inline p" id="p0075">Place isolated muscles in 3–4 mL of cold (4°C) 4% paraformaldehyde in PBS per well in a 6-well plate.</div>
</li>
<li id="o0040">
<span class="label">b.</span><div class="display-inline p" id="p0080">Incubate tissue on a rocker at 4°C for at least 2 h.<blockquote class="text-italic"><div class="p">
<strong><em>Note:</em></strong> Tibialis anterior muscles up to 200 mg are fixed for 2 h, but larger muscles, such as the gastrocnemius, require a 3-h incubation for fixative to fully enter the tissue.</div></blockquote>
</div>
</li>
<li id="o0045">
<span class="label">c.</span><div class="display-inline p" id="p0085">Remove muscles and place in pre-chilled 30% sucrose in water (4°C) overnight (16 h) at 4°C.</div>
</li>
</ul>
</div>
</li></ul>
<ul class="list" id="olist0030" style="list-style-type:none"><li id="o0050">
<span class="label">6.</span><div class="display-inline p" id="p0090">Embed tissues in O.C.T. and freeze.<ul class="list" id="olist0035" style="list-style-type:none">
<li id="o0055">
<span class="label">a.</span><div class="display-inline p" id="p0095">Label 15 mm <strong>×</strong> 15 mm <strong>×</strong> 5 mm base molds with pertinent mouse information with an alcohol resistant marker and mark directionality of the muscle by labeling the terminal tendon location.</div>
</li>
<li id="o0060">
<span class="label">b.</span><div class="display-inline p" id="p0100">Fill base mold with O.C.T. to the point where it almost overflows from the inset. Do not allow bubbles to form. If they do, use a 100 μL micropipettor and tip to remove the bubbles.</div>
</li>
<li id="o0065">
<span class="label">c.</span><div class="display-inline p" id="p0105">Place fixed muscle tissue in the base mold and push it evenly to the bottom of the mold. The orientation should be as it was labeled previously.</div>
</li>
<li id="o0070">
<span class="label">d.</span><div class="display-inline p" id="p0110">Chill 2-methylbutane with dry ice or liquid nitrogen. Immerse muscles in base molds in chilled 2-methylbutane for &gt;30 s for TA muscles and &gt;1 min for gastrocnemius muscles.</div>
</li>
<li id="o0075">
<span class="label">e.</span><div class="display-inline p" id="p0115">Place chilled blocks on dry ice for 1 h to allow residual 2-methylbutane to evaporate.</div>
</li>
<li id="o0080">
<span class="label">f.</span><div class="display-inline p" id="p0120">Wrap blocks in base molds with parafilm and store at −80°C for at least overnight (16 h).</div>
</li>
</ul>
</div>
</li></ul>
<blockquote class="text-italic"><p><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/b9d7883d5879/fx3.gif"/><strong>Pause point:</strong> Cryoblocks can be stored at −80°C indefinitely.</p></blockquote>
<ul class="list" id="olist0040" style="list-style-type:none"><li id="o0085">
<span class="label">7.</span><div class="display-inline p" id="p0125">Sectioning of skeletal muscles.<ul class="list" id="olist0045" style="list-style-type:none">
<li id="o0090">
<span class="label">a.</span><div class="display-inline p" id="p0130">The evening before cryosectioning, remove cryoblocks from −80°C and place at −20°C.</div>
</li>
<li id="o0095">
<span class="label">b.</span><div class="display-inline p" id="p0135">Label superfrost plus slides with pertinent mouse information.</div>
</li>
<li id="o0100">
<span class="label">c.</span><div class="display-inline p" id="p0140">Remove block from base mold and section in a cryostat, cutting 10 μm slices. We try to keep the tissue orientation the same for all blocks. Both longitudinal, as well as cross sectional slices will work with this assay.<blockquote class="text-italic"><div class="p">
<strong><em>Note:</em></strong> For optimal adherence of sections, we firmly press cut sections onto room temperature (25°C) superfrost plus slides.</div></blockquote>
</div>
</li>
<li id="o0105">
<span class="label">d.</span><div class="display-inline p" id="p0145">Store slides at −20°C until all samples have been collected and sectioned.</div>
</li>
</ul>
</div>
</li></ul>
<blockquote class="text-italic"><p><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/b9d7883d5879/fx3.gif"/><strong>Pause point:</strong> Slides can be stored at −20°C indefinitely.</p></blockquote></section></section><section id="sec9"><h2 class="pmc_sec_title">Key resources table</h2>
<section class="tw xbox font-sm" id="undtbl9"><div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">REAGENT or RESOURCE</th>
<th colspan="1" rowspan="1">SOURCE</th>
<th colspan="1" rowspan="1">IDENTIFIER</th>
</tr></thead>
<tbody>
<tr><td colspan="3" rowspan="1"><strong>Antibodies</strong></td></tr>
<tr><td colspan="3" rowspan="1"><hr/></td></tr>
<tr>
<td colspan="1" rowspan="1">Alexa Fluor 488 Donkey anti-goat IgG; 1/300 dilution</td>
<td colspan="1" rowspan="1">Thermo Fisher Scientific</td>
<td colspan="1" rowspan="1">Cat# A11055; RRID:<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://nif-antibody:AB_2534102" rel="noopener noreferrer" target="_blank">AB_2534102</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Goat anti-mouse Vcam1; stock made at 1 mg/mL; 1/100 dilution</td>
<td colspan="1" rowspan="1">Fisher Scientific</td>
<td colspan="1" rowspan="1">Cat# PIPA547029; RRID:<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://nif-antibody:AB_2609027" rel="noopener noreferrer" target="_blank">AB_2609027</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Rat anti-CD11b-biotin, clone M1/70; 1/200 dilution</td>
<td colspan="1" rowspan="1">BD Biosciences</td>
<td colspan="1" rowspan="1">Cat# 553309; RRID:<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://nif-antibody:AB_394773" rel="noopener noreferrer" target="_blank">AB_394773</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Rat anti-CD31-biotin; 1/200 dilution</td>
<td colspan="1" rowspan="1">Fisher Scientific</td>
<td colspan="1" rowspan="1">Cat# 13-0311-85; RRID:<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://nif-antibody:AB_466421" rel="noopener noreferrer" target="_blank">AB_466421</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Rat anti-mouse CD34-BV421, Clone<br/>RAM34; 1/20 dilution</td>
<td colspan="1" rowspan="1">BD Biosciences</td>
<td colspan="1" rowspan="1">Cat# 562608; RRID:<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://nif-antibody:AB_11154576" rel="noopener noreferrer" target="_blank">AB_11154576</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Rat anti-CD45-biotin; 1/500 dilution</td>
<td colspan="1" rowspan="1">BD Biosciences</td>
<td colspan="1" rowspan="1">Cat# 553078; RRID:<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://nif-antibody:AB_394608" rel="noopener noreferrer" target="_blank">AB_394608</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Rat anti-Ly-6A/E-biotin (Sca1), clone E13–161.7; 1/200 dilution</td>
<td colspan="1" rowspan="1">BD Biosciences</td>
<td colspan="1" rowspan="1">Cat# 553334; RRID:<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://nif-antibody:AB_394790" rel="noopener noreferrer" target="_blank">AB_394790</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1">anti-Alpha 7 Integrin Alexa Fluor 647, clone<br/>R2F2; 1/25 dilution</td>
<td colspan="1" rowspan="1">Ablab.ca</td>
<td colspan="1" rowspan="1">Cat# 67-0010-05;<br/>RRID: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://nif-antibody:AB_2890939" rel="noopener noreferrer" target="_blank">AB_2890939</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Biotinylated mouse anti-human CD31; 1/50 dilution</td>
<td colspan="1" rowspan="1">Thermo Fisher Scientific</td>
<td colspan="1" rowspan="1">Cat# 13-0319-82;<br/>RRID:<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://nif-antibody:AB_466423" rel="noopener noreferrer" target="_blank">AB_466423</a>
</td>
</tr>
<tr><td colspan="3" rowspan="1"><hr/></td></tr>
<tr><td colspan="3" rowspan="1"><strong>Chemicals, peptides, and recombinant proteins</strong></td></tr>
<tr><td colspan="3" rowspan="1"><hr/></td></tr>
<tr>
<td colspan="1" rowspan="1">4% paraformaldehyde</td>
<td colspan="1" rowspan="1">Thermo Scientific</td>
<td colspan="1" rowspan="1">Cat# J19943-K2</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Antigen unmasking solution (Commercial citrate buffer)</td>
<td colspan="1" rowspan="1">Vector Labs</td>
<td colspan="1" rowspan="1">Cat# H-3300</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Bovine Serum Albumin (Fraction V)</td>
<td colspan="1" rowspan="1">Gemini Bio-Products</td>
<td colspan="1" rowspan="1">Cat# 700-100P</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Cenpb-Cy3</td>
<td colspan="1" rowspan="1">PNA Bio</td>
<td colspan="1" rowspan="1">Cat# F3002</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Dextran sulfate sodium salt</td>
<td colspan="1" rowspan="1">Sigma-Aldrich</td>
<td colspan="1" rowspan="1">Cat# D8906</td>
</tr>
<tr>
<td colspan="1" rowspan="1">EDTA</td>
<td colspan="1" rowspan="1">Invitrogen</td>
<td colspan="1" rowspan="1">Cat#15575-020</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Ethanol, 200 proof</td>
<td colspan="1" rowspan="1">Decon Labs</td>
<td colspan="1" rowspan="1">Cat# 2701</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Ethylene Carbonate</td>
<td colspan="1" rowspan="1">Sigma-Aldrich</td>
<td colspan="1" rowspan="1">Cat# E26258</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Formamide</td>
<td colspan="1" rowspan="1">Sigma-Aldrich</td>
<td colspan="1" rowspan="1">Cat# 47670</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Immersion oil, type A</td>
<td colspan="1" rowspan="1">Nikon</td>
<td colspan="1" rowspan="1">Cat# MXA20233</td>
</tr>
<tr>
<td colspan="1" rowspan="1">OCT</td>
<td colspan="1" rowspan="1">Thermo Scientific</td>
<td colspan="1" rowspan="1">Cat# 6502G</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Potassium chloride</td>
<td colspan="1" rowspan="1">Sigma Aldrich</td>
<td colspan="1" rowspan="1">Cat# P9541</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Potassium phosphate, monobasic</td>
<td colspan="1" rowspan="1">Sigma Aldrich</td>
<td colspan="1" rowspan="1">Cat# P9791</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Prolong gold plus DAPI</td>
<td colspan="1" rowspan="1">Thermo Scientific</td>
<td colspan="1" rowspan="1">Cat# <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/protein/P36935" rel="noopener noreferrer" target="_blank">P36935</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1">RNAse A</td>
<td colspan="1" rowspan="1">Fisher Scientific</td>
<td colspan="1" rowspan="1">Cat# AM9780</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Sodium Chloride</td>
<td colspan="1" rowspan="1">Sigma Aldrich</td>
<td colspan="1" rowspan="1">Cat# S6191</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Sodium Phosphate, dibasic, dihydrate</td>
<td colspan="1" rowspan="1">Sigma Aldrich</td>
<td colspan="1" rowspan="1">Cat# S3264</td>
</tr>
<tr>
<td colspan="1" rowspan="1">SSC buffer, 20<strong>×</strong>
</td>
<td colspan="1" rowspan="1">Corning</td>
<td colspan="1" rowspan="1">Cat# 46-020-CM</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Streptavidin PE-Cy7</td>
<td colspan="1" rowspan="1">BioLegend</td>
<td colspan="1" rowspan="1">Cat# 405206</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Streptavidin Cy3</td>
<td colspan="1" rowspan="1">Thermo Scientific</td>
<td colspan="1" rowspan="1">Cat# SA1010</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Sucrose</td>
<td colspan="1" rowspan="1">Sigma-Aldrich</td>
<td colspan="1" rowspan="1">Cat#S0389</td>
</tr>
<tr>
<td colspan="1" rowspan="1">TelC-Alexa Fluor 647</td>
<td colspan="1" rowspan="1">PNA Bio</td>
<td colspan="1" rowspan="1">Cat# F1013</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Tris-Cl</td>
<td colspan="1" rowspan="1">Quality Biological</td>
<td colspan="1" rowspan="1">Cat#351-006-131</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Triton X-100</td>
<td colspan="1" rowspan="1">Fisher Scientific</td>
<td colspan="1" rowspan="1">Cat# BP151</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Tween 20</td>
<td colspan="1" rowspan="1">Fisher Scientific</td>
<td colspan="1" rowspan="1">Cat# BP337</td>
</tr>
<tr><td colspan="3" rowspan="1"><hr/></td></tr>
<tr><td colspan="3" rowspan="1"><strong>Experimental models: organisms/strains</strong></td></tr>
<tr><td colspan="3" rowspan="1"><hr/></td></tr>
<tr>
<td colspan="1" rowspan="1">Mouse: C57Bl/6J, 2-5-month-old males</td>
<td colspan="1" rowspan="1">Jackson Laboratory</td>
<td colspan="1" rowspan="1">Stock# 000664; RRID:IMSR_JAX:000664</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Mouse: IKK2CA: B6.Cg-<em>Gt(ROSA)26Sor</em><sup><em>tm4(Ikbkb)/Rsky</em></sup>/J, 2–5-month-old males</td>
<td colspan="1" rowspan="1">Jackson Laboratory</td>
<td colspan="1" rowspan="1">Stock# 008242; RRID:IMSR_JAX:008242</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Mouse: Pax7<sup>ERT2Cre</sup>: B6.Cg<em>-Pax7</em><sup><em>tm1(cre/ERT2)Gaka</em></sup>/J, 2–5-month-old males</td>
<td colspan="1" rowspan="1">Jackson Laboratory</td>
<td colspan="1" rowspan="1">Stock# 017763; RRID:IMSR_Jax:017763</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Mouse: mdx/mTR: B6.Cg-<em>Terc</em><sup><em>tm1Rdp</em></sup> <em>Dmd</em><sup><em>mdx-4Cv</em></sup>/BlauJ, 7-month-old males</td>
<td colspan="1" rowspan="1">Jackson Laboratory</td>
<td colspan="1" rowspan="1">Stock# 023535;<br/>RRID: IMSR_Jax:023535</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Rat: Sprague Dawley, male, 14-month-old male</td>
<td colspan="1" rowspan="1">Charles River</td>
<td colspan="1" rowspan="1">Stock# 001;<br/>RRID: MGI:5651135</td>
</tr>
<tr><td colspan="3" rowspan="1"><hr/></td></tr>
<tr><td colspan="3" rowspan="1"><strong>Biological samples</strong></td></tr>
<tr><td colspan="3" rowspan="1"><hr/></td></tr>
<tr>
<td colspan="1" rowspan="1">Human skeletal muscle samples, 10–15-year-old male</td>
<td colspan="1" rowspan="1">Obtained in house</td>
<td colspan="1" rowspan="1">N/A</td>
</tr>
<tr><td colspan="3" rowspan="1"><hr/></td></tr>
<tr><td colspan="3" rowspan="1"><strong>Software and algorithms</strong></td></tr>
<tr><td colspan="3" rowspan="1"><hr/></td></tr>
<tr>
<td colspan="1" rowspan="1">Graphpad Prism v. 9.1.2</td>
<td colspan="1" rowspan="1">Licensed Software</td>
<td colspan="1" rowspan="1"><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.graphpad.com" rel="noopener noreferrer" target="_blank">https://www.graphpad.com</a></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Image J</td>
<td colspan="1" rowspan="1">Open Source</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://imagej.nih.gov/ij/" rel="noopener noreferrer" target="_blank">https://imagej.nih.gov/ij/</a></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Java SE 11</td>
<td colspan="1" rowspan="1">Open Source</td>
<td colspan="1" rowspan="1"><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.oracle.com/java/technologies/javase-downloads.html" rel="noopener noreferrer" target="_blank">https://www.oracle.com/java/technologies/javase-downloads.html</a></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Microsoft Excel</td>
<td colspan="1" rowspan="1">Licensed Software</td>
<td colspan="1" rowspan="1"><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.office.com" rel="noopener noreferrer" target="_blank">https://www.office.com</a></td>
</tr>
<tr>
<td colspan="1" rowspan="1">NIS-Elements</td>
<td colspan="1" rowspan="1">Licensed Software</td>
<td colspan="1" rowspan="1"><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.nikon.com/products/microscope-solutions/lineup/img_soft/nis-elements/" rel="noopener noreferrer" target="_blank">https://www.nikon.com/products/microscope-solutions/lineup/img_soft/nis-elements/</a></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Telometer ImageJ Plugin</td>
<td colspan="1" rowspan="1">Open Source</td>
<td colspan="1" rowspan="1"><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://demarzolab.pathology.jhmi.edu/telometer/" rel="noopener noreferrer" target="_blank">https://demarzolab.pathology.jhmi.edu/telometer/</a></td>
</tr>
<tr><td colspan="3" rowspan="1"><hr/></td></tr>
<tr><td colspan="3" rowspan="1"><strong>Other</strong></td></tr>
<tr><td colspan="3" rowspan="1"><hr/></td></tr>
<tr>
<td colspan="1" rowspan="1">Coplin Staining Jar</td>
<td colspan="1" rowspan="1">Thermo Fisher Scientific</td>
<td colspan="1" rowspan="1">Cat# S17495A</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Slide Tray for Immuno-Staining with Black Cover</td>
<td colspan="1" rowspan="1">RPI corp.</td>
<td colspan="1" rowspan="1">Cat# 195801</td>
</tr>
<tr>
<td colspan="1" rowspan="1">6 well plates</td>
<td colspan="1" rowspan="1">VWR</td>
<td colspan="1" rowspan="1">Cat#10062-892</td>
</tr>
<tr>
<td colspan="1" rowspan="1">1.5 mL microcentrifuge tubes</td>
<td colspan="1" rowspan="1">VWR</td>
<td colspan="1" rowspan="1">Cat# 10025-724</td>
</tr>
<tr>
<td colspan="1" rowspan="1">15 mL centrifuge tubes</td>
<td colspan="1" rowspan="1">VWR</td>
<td colspan="1" rowspan="1">Cat# 10025-692</td>
</tr>
<tr>
<td colspan="1" rowspan="1">50 mL centrifuge tubes</td>
<td colspan="1" rowspan="1">VWR</td>
<td colspan="1" rowspan="1">Cat# 10025-702</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Superfrost plus microscope slides</td>
<td colspan="1" rowspan="1">Thermo Fisher Scientific</td>
<td colspan="1" rowspan="1">Cat# 1255015</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Super HT PAP Pen, mini</td>
<td colspan="1" rowspan="1">RPI corp.</td>
<td colspan="1" rowspan="1">Cat# 195506</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Tissue Path disposable base molds, 15 <strong>×</strong> 15 <strong>×</strong> 5 mm</td>
<td colspan="1" rowspan="1">Thermo Fisher Scientific</td>
<td colspan="1" rowspan="1">Cat# 22-363-553</td>
</tr>
<tr>
<td colspan="1" rowspan="1">VWR VistaVision Cover Glass, No. 1</td>
<td colspan="1" rowspan="1">VWR</td>
<td colspan="1" rowspan="1">Cat# 16004-096</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Compact Digital Rocker</td>
<td colspan="1" rowspan="1">Thermo Scientific</td>
<td colspan="1" rowspan="1">Cat# 88880020</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Isotemp Water Bath</td>
<td colspan="1" rowspan="1">Fisher Scientific</td>
<td colspan="1" rowspan="1">Model# 2320</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Steamer</td>
<td colspan="1" rowspan="1">Hamilton Beach</td>
<td colspan="1" rowspan="1">Model# VS02</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Thermobrite Slide Warmer</td>
<td colspan="1" rowspan="1">Leica</td>
<td colspan="1" rowspan="1">Model# S500-12</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Thermomixer</td>
<td colspan="1" rowspan="1">Fisher Scientific</td>
<td colspan="1" rowspan="1">Cat# 05-412-500</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Widefield Microscope</td>
<td colspan="1" rowspan="1">Nikon</td>
<td colspan="1" rowspan="1">Model# NiU</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/undtbl9/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></section></section><section id="sec2"><h2 class="pmc_sec_title">Materials and equipment</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Buffers</h3>
<section class="tw xbox font-sm" id="undtbl1"><div class="caption p"><p>TET buffer</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Reagent</th>
<th colspan="1" rowspan="1">Final concentration</th>
<th colspan="1" rowspan="1">Amount</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">Tris, pH 7.5 (1M)</td>
<td colspan="1" rowspan="1">10mM</td>
<td colspan="1" rowspan="1">0.5 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">EDTA, pH 8.0 (500mM)</td>
<td colspan="1" rowspan="1">1mM</td>
<td colspan="1" rowspan="1">0.1 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Tween-20</td>
<td colspan="1" rowspan="1">0.05%</td>
<td colspan="1" rowspan="1">0.025 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ddH<sub>2</sub>O</td>
<td colspan="1" rowspan="1">n/a</td>
<td colspan="1" rowspan="1">49.375 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Total</strong></td>
<td colspan="1" rowspan="1"><strong>n/a</strong></td>
<td colspan="1" rowspan="1"><strong>50 mL</strong></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/undtbl1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn3"><p>pH 7.6. Buffer can be stored at room temperature (25°C) indefinitely if making a larger batch. Do not reuse buffer.</p></div></div></section><section class="tw xbox font-sm" id="undtbl2"><div class="caption p"><p>Hybridization buffer</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Reagent</th>
<th colspan="1" rowspan="1">Final concentration</th>
<th colspan="1" rowspan="1">Amount</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">Ethylene carbonate</td>
<td colspan="1" rowspan="1">15%</td>
<td colspan="1" rowspan="1">0.15g</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Dextran sulfate</td>
<td colspan="1" rowspan="1">20%</td>
<td colspan="1" rowspan="1">0.20g</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Sodium chloride, 5M</td>
<td colspan="1" rowspan="1">600mM</td>
<td colspan="1" rowspan="1">120 uL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">100<strong>×</strong> commercial citrate antigen retrieval buffer</td>
<td colspan="1" rowspan="1">1<strong>×</strong>
</td>
<td colspan="1" rowspan="1">10uL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ddH<sub>2</sub>O</td>
<td colspan="1" rowspan="1">n/a</td>
<td colspan="1" rowspan="1">Up to 1 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Total</strong></td>
<td colspan="1" rowspan="1"><strong>n/a</strong></td>
<td colspan="1" rowspan="1"><strong>1 mL</strong></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/undtbl2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn4"><p>pH 6.0. Use immediately after making buffer.</p></div></div></section><section class="tw xbox font-sm" id="undtbl3"><div class="caption p"><p>Wash Buffer I</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Reagent</th>
<th colspan="1" rowspan="1">Final concentration</th>
<th colspan="1" rowspan="1">Amount</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">SSC (20<strong>×</strong>)</td>
<td colspan="1" rowspan="1">2<strong>×</strong>
</td>
<td colspan="1" rowspan="1">100 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Tween-20</td>
<td colspan="1" rowspan="1">0.1%</td>
<td colspan="1" rowspan="1">1 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ddH<sub>2</sub>O</td>
<td colspan="1" rowspan="1">n/a</td>
<td colspan="1" rowspan="1">899 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Total</strong></td>
<td colspan="1" rowspan="1"><strong>n/a</strong></td>
<td colspan="1" rowspan="1"><strong>1000 mL</strong></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/undtbl3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn5"><p>pH 7.2. Wash buffer I can be prepared in large stocks and stored at room temperature (25°C) for several months.</p></div></div></section><section class="tw xbox font-sm" id="undtbl4"><div class="caption p"><p>Wash Buffer II</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Reagent</th>
<th colspan="1" rowspan="1">Final concentration</th>
<th colspan="1" rowspan="1">Amount</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">SSC (20<strong>×</strong>)</td>
<td colspan="1" rowspan="1">1<strong>×</strong>
</td>
<td colspan="1" rowspan="1">50 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Tween-20</td>
<td colspan="1" rowspan="1">0.1%</td>
<td colspan="1" rowspan="1">1 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ddH<sub>2</sub>O</td>
<td colspan="1" rowspan="1">n/a</td>
<td colspan="1" rowspan="1">949 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Total</strong></td>
<td colspan="1" rowspan="1"><strong>n/a</strong></td>
<td colspan="1" rowspan="1"><strong>1000 mL</strong></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/undtbl4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn6"><p>pH 7.2. Wash buffer II can be prepared in large stocks and stored at room temperature (25°C) for several months.</p></div></div></section><section class="tw xbox font-sm" id="undtbl5"><div class="caption p"><p>Wash Buffer III</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Reagent</th>
<th colspan="1" rowspan="1">Final concentration</th>
<th colspan="1" rowspan="1">Amount</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">SSC (20<strong>×</strong>)</td>
<td colspan="1" rowspan="1">0.5<strong>×</strong>
</td>
<td colspan="1" rowspan="1">25 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Tween-20</td>
<td colspan="1" rowspan="1">0.1%</td>
<td colspan="1" rowspan="1">1 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ddH<sub>2</sub>O</td>
<td colspan="1" rowspan="1">n/a</td>
<td colspan="1" rowspan="1">974 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Total</strong></td>
<td colspan="1" rowspan="1"><strong>n/a</strong></td>
<td colspan="1" rowspan="1"><strong>1000 mL</strong></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/undtbl5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn7"><p>pH 7.2. Wash buffer III can be prepared in large stocks and stored at room temperature (25°C) for several months.</p></div></div></section><section class="tw xbox font-sm" id="undtbl6"><div class="caption p"><p>Wash Buffer IV</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Reagent</th>
<th colspan="1" rowspan="1">Final concentration</th>
<th colspan="1" rowspan="1">Amount</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">SSC (20<strong>×</strong>)</td>
<td colspan="1" rowspan="1">0.25<strong>×</strong>
</td>
<td colspan="1" rowspan="1">12.5 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Tween-20</td>
<td colspan="1" rowspan="1">0.1%</td>
<td colspan="1" rowspan="1">1 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ddH<sub>2</sub>O</td>
<td colspan="1" rowspan="1">n/a</td>
<td colspan="1" rowspan="1">986.5 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Total</strong></td>
<td colspan="1" rowspan="1"><strong>n/a</strong></td>
<td colspan="1" rowspan="1"><strong>1000 mL</strong></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/undtbl6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn8"><p>pH 7.2. Wash buffer IV can be prepared in large stocks and stored at room temperature (25°C) for several months.</p></div></div></section><section class="tw xbox font-sm" id="undtbl7"><div class="caption p"><p>Antibody Blocking Buffer</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Reagent</th>
<th colspan="1" rowspan="1">Final concentration</th>
<th colspan="1" rowspan="1">Amount</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">Bovine serum albumin</td>
<td colspan="1" rowspan="1">3%</td>
<td colspan="1" rowspan="1">1.5g</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Triton X-100</td>
<td colspan="1" rowspan="1">0.1%</td>
<td colspan="1" rowspan="1">50 uL</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Phosphate Buffered Saline</td>
<td colspan="1" rowspan="1">1<strong>×</strong>
</td>
<td colspan="1" rowspan="1">Up to 50 mL</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Total</strong></td>
<td colspan="1" rowspan="1"><strong>n/a</strong></td>
<td colspan="1" rowspan="1"><strong>50 mL</strong></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/undtbl7/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn9"><p>pH 7.4. Blocking buffer should be stored at 4°C when not in use and is stable for several weeks.</p></div></div></section><section class="tw xbox font-sm" id="undtbl8"><div class="caption p"><p>Phosphate Buffered Saline (PBS), 1<strong>×</strong></p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Reagent</th>
<th colspan="1" rowspan="1">Final concentration</th>
<th colspan="1" rowspan="1">Amount</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">Sodium Chloride</td>
<td colspan="1" rowspan="1">137mM</td>
<td colspan="1" rowspan="1">8g</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Potassium Chloride</td>
<td colspan="1" rowspan="1">2.7mM</td>
<td colspan="1" rowspan="1">0.2g</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Sodium phosphate, dibasic, dihydrate</td>
<td colspan="1" rowspan="1">4.3mM</td>
<td colspan="1" rowspan="1">9g</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Potassium phosphate, monobasic</td>
<td colspan="1" rowspan="1">1.47mM</td>
<td colspan="1" rowspan="1">0.24g</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Total</strong></td>
<td colspan="1" rowspan="1"><strong>n/a</strong></td>
<td colspan="1" rowspan="1"><strong>1000 mL</strong></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/undtbl8/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn10"><p>pH 7.4. PBS is stored at room temperature (25°C) and is stable for several months.</p></div></div></section></section><section id="sec2.2"><h3 class="pmc_sec_title">Equipment</h3>
<p id="p0155">The following equipment was used for the described protocol.</p>
<ul class="list" id="ulist0015" style="list-style-type:none">
<li id="u0030">
<span class="label">•</span><p class="display-inline" id="p0160">−20°C freezer</p>
</li>
<li id="u0035">
<span class="label">•</span><p class="display-inline" id="p0165">−80°C freezer</p>
</li>
<li id="u0040">
<span class="label">•</span><p class="display-inline" id="p0170">4°C refrigerator</p>
</li>
<li id="u0045">
<span class="label">•</span><p class="display-inline" id="p0175">Compact Digital Rocker (Thermo Scientific, Cat#88880020)</p>
</li>
<li id="u0050">
<span class="label">•</span><p class="display-inline" id="p0180">Isotemp Water Bath (Fisher Scientific, Model #2320)</p>
</li>
<li id="u0055">
<span class="label">•</span><p class="display-inline" id="p0185">Steamer (Hamilton Beach, Model# VS02)</p>
</li>
<li id="u0060">
<span class="label">•</span><p class="display-inline" id="p0190">Thermobrite slide warmer (Leica, Model #S500-12)</p>
</li>
<li id="u0065">
<span class="label">•</span><p class="display-inline" id="p0195">Thermomixer (Eppendorf, Model# F1.5)</p>
</li>
<li id="u0070">
<span class="label">•</span><p class="display-inline" id="p0200">Widefield microscope (Nikon, Model# NiU)</p>
</li>
</ul>
<blockquote class="text-italic"><p><strong><em>Alternatives:</em></strong> A hybridization oven may also be utilized for wash buffer incubations and telomere probe preheating and hybridization, as well as additional heating steps. Common equipment such as freezers, refrigerators, rockers, water baths, steamers, and thermomixers are interchangeable with any other brand and model. Any widefield microscope equipped with the proper filter sets and a 100<strong>×</strong> objective can complete the imaging described in this protocol.</p></blockquote></section><section id="sec2.3"><h3 class="pmc_sec_title">Software</h3>
<p id="p0205">The following software are instrumental in analysis of datasets generated using this protocol.</p>
<ul class="list" id="ulist0020" style="list-style-type:none">
<li id="u0075">
<span class="label">•</span><p class="display-inline" id="p0210">Java SE 11 or later</p>
</li>
<li id="u0080">
<span class="label">•</span><p class="display-inline" id="p0215">Telometer v3.0.5</p>
</li>
<li id="u0085">
<span class="label">•</span><p class="display-inline" id="p0220">Microsoft Excel 365 or equivalent</p>
</li>
<li id="u0090">
<span class="label">•</span><p class="display-inline" id="p0225">Graphpad prism 9.1.2 or later</p>
</li>
</ul></section></section><section id="sec3"><h2 class="pmc_sec_title">Step-by-step method details</h2>
<section id="sec3.1"><h3 class="pmc_sec_title">Telomere, centromere, muscle stem cell, and nuclear staining</h3>
<blockquote class="text-italic"><p><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/51ca064f7ff6/fx2.gif"/><strong>Timing: 2 days</strong></p></blockquote>
<ul class="list" id="olist0050" style="list-style-type:none">
<li id="o0110">
<span class="label">1.</span><p class="display-inline" id="p0235">Remove one slide from each desired sample set from −20°C and warm to room temperature (25°C).</p>
</li>
<li id="o0115">
<span class="label">2.</span><p class="display-inline" id="p0240">Permeabilize sections by immersing slides in 0.5% Triton X-100 in PBS for 10 min in a coplin jar at room temperature (25°C).</p>
</li>
</ul>
<blockquote class="text-italic"><p><strong><em>Note:</em></strong> The coplin jar used in this protocol holds approximately 50 mL of solution and 8 slides. If more slides must be stained, we recommend using a tissue-tek staining dish and 24-place slide holder. This alternative setup requires 200–250 mL of solution volume for adequate coverage of the slides.</p></blockquote>
<ul class="list" id="olist0055" style="list-style-type:none"><li id="o0120">
<span class="label">3.</span><p class="display-inline" id="p0245">Wash slides twice with PBS for 3 min each at room temperature (25°C).</p>
</li></ul>
<ul class="list" id="olist0060" style="list-style-type:none"><li id="o0125">
<span class="label">4.</span><div class="display-inline p" id="p0250">Perform heat-mediated antigen retrieval with TET buffer.<ul class="list" id="olist0065" style="list-style-type:none">
<li id="o0130">
<span class="label">a.</span><div class="display-inline p" id="p0255">Heat buffer to boiling in a microwave first and then transfer buffer to a coplin jar placed in a preheated steamer (95°–100°C).</div>
</li>
<li id="o0135">
<span class="label">b.</span><div class="display-inline p" id="p0260">Add slides to the coplin jar containing hot TET buffer (95°–100°C) and incubate for 10 min. Remove coplin jar containing the slides from the steamer and let it cool at room temperature (25°C) for 15 min.<blockquote class="text-italic"><div class="p">
<strong><em>Note:</em></strong> Take care when handling hot solutions.</div></blockquote>
<blockquote class="text-italic"><div class="p">
<img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/83e6728c8b52/fx4.gif"/><strong>CRITICAL:</strong> Do not let the slides dry at any time following antigen retrieval, as doing so greatly increases autofluorescence of the tissue and will make distinguishing MuSCs difficult.</div></blockquote>
</div>
</li>
<li id="o0140">
<span class="label">c.</span><div class="display-inline p" id="p0265">Wash slides with PBS twice for 3 min each in a coplin jar at room temperature (25°C).</div>
</li>
</ul>
</div>
</li></ul>
<ul class="list" id="olist0070" style="list-style-type:none"><li id="o0145">
<span class="label">5.</span><div class="display-inline p" id="p0270">RNAse A treatment<ul class="list" id="olist0075" style="list-style-type:none">
<li id="o0150">
<span class="label">a.</span><div class="display-inline p" id="p0275">Remove slides from coplin jar and mark tissue boundaries with a hydrophobic/PAP pen.</div>
</li>
<li id="o0155">
<span class="label">b.</span><div class="display-inline p" id="p0280">Add sufficient RNase A (100 μg/mL) to cover tissue on the slide (100–200 μL) and cover the slide with a glass coverslip to prevent evaporation.<blockquote class="text-italic"><div class="p">
<img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/83e6728c8b52/fx4.gif"/><strong>CRITICAL:</strong> RNase should be DNase-free. We routinely prepare RNase stock concentrations of 10 mg/mL, heat at 100°C for 10 min to destroy DNAses and then dilute to 100ug/mL in water and store at −20°C.</div></blockquote>
</div>
</li>
<li id="o0160">
<span class="label">c.</span><div class="display-inline p" id="p0285">Incubate slides at 37°C in a slide warmer or alternative heating source for 20 min.</div>
</li>
<li id="o0165">
<span class="label">d.</span><div class="display-inline p" id="p0290">Remove coverslip and transfer slides into coplin jars containing PBS. Wash slides twice for 3 min each at room temperature (25°C).</div>
</li>
</ul>
</div>
</li></ul>
<ul class="list" id="olist0080" style="list-style-type:none"><li id="o0170">
<span class="label">6.</span><div class="display-inline p" id="p0295">Telomere Staining<ul class="list" id="olist0085" style="list-style-type:none">
<li id="o0175">
<span class="label">a.</span><div class="display-inline p" id="p0300">Prepare hybridization buffer [adapted from (<a aria-describedby="bib4" class="usa-link" href="#bib4">Matthiesen and Hansen, 2012</a>)] as described in the <a class="usa-link" href="#sec2">materials and equipment</a> section.<blockquote class="text-italic"><div class="p">
<strong><em>Note:</em></strong> The hybridization buffer requires extensive vortexing and pipetting to get all components completely into solution. Heating or alteration of pH is unnecessary. When all components are properly dissolved, the solution will be viscous.</div></blockquote>
</div>
</li>
<li id="o0180">
<span class="label">b.</span><div class="display-inline p" id="p0305">Prepare TelC-Alexa Fluor 647 telomere probe and optional Cy3-CENPB centromere probe.<ul class="list" id="olist0090" style="list-style-type:none">
<li id="o0185">
<span class="label">i.</span><div class="display-inline p" id="p0310">Dilute 5 nmol probe pellet provided by the supplier with 100uL formamide.</div>
</li>
<li id="o0190">
<span class="label">ii.</span><div class="display-inline p" id="p0315">Aliquot in 4 μL aliquots in 1.5 mL microcentrifuge tubes and store at −80°C.<blockquote class="text-italic"><div class="p">
<strong><em>Note:</em></strong> Probe is light sensitive. All steps proceeding the addition of probe should be done in the dark or low-light conditions.</div></blockquote>
<blockquote class="text-italic"><div class="p">
<strong><em>Note:</em></strong> Probes stored at −80°C are stable for at least several months.</div></blockquote>
<blockquote class="text-italic"><div class="p">
<strong><em>Note:</em></strong> We recommend using the Alexa Fluor 647-conjugated telomere probe, as it provides the least background in skeletal muscle cryosections.</div></blockquote>
</div>
</li>
</ul>
</div>
</li>
<li id="o0195">
<span class="label">c.</span><div class="display-inline p" id="p0320">Remove telomere probe from −80°C storage, thaw, and add probe at a 1/600 dilution to the hybridization buffer (1.67 μL/mL of buffer).<blockquote class="text-italic"><div class="p">
<strong><em>Optional:</em></strong> Centromere probe staining and analysis is not required, unless problems arise with abnormal nuclear architecture or DAPI staining intensities between samples. If centromere probe staining is desired, use Cy3-CENPB probe at 1/600 dilution and add it at the same time as the telomere probe above. The CENPB probe is prepared identically to the telomere probe.</div></blockquote>
</div>
</li>
<li id="o0200">
<span class="label">d.</span><div class="display-inline p" id="p0325">Heat probe(s) in hybridization buffer for 10 min at 67°C in a thermomixer or alternative heating source in the dark.</div>
</li>
<li id="o0205">
<span class="label">e.</span><div class="display-inline p" id="p0330">Add hybridization buffer containing probe to slides. We typically use 100 μL probe solution per slide. Cover slides with a glass coverslip to prevent evaporation during incubation.<blockquote class="text-italic"><div class="p">
<img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/83e6728c8b52/fx4.gif"/><strong>CRITICAL:</strong> It is crucial that bubbles are kept to a minimum, as areas that are not evenly covered will not stain properly.</div></blockquote>
</div>
</li>
<li id="o0210">
<span class="label">f.</span><div class="display-inline p" id="p0335">Incubate slides in a Thermobrite slide warmer or equivalent heating source, according to <a class="usa-link" href="#tbl1">Table 1</a>.</div>
</li>
<li id="o0215">
<span class="label">g.</span><div class="display-inline p" id="p0340">Preheat wash buffers I, II, III, IV to 55°C in a water bath or equivalent heating source.</div>
</li>
<li id="o0220">
<span class="label">h.</span><div class="display-inline p" id="p0345">Gently remove coverslips from slides after hybridization, place slides in coplin jars, and incubate with 50 mL prewarmed wash buffer I for 5 min. Keep coplin jar at 55°C during the wash process. Repeat wash once.</div>
</li>
<li id="o0225">
<span class="label">i.</span><div class="display-inline p" id="p0350">Continue washing stepwise with 2 washes each (50 mL per wash) of wash buffer II, wash buffer III, and wash buffer IV, in that order, for 5 min each wash and at 55°C.</div>
</li>
<li id="o0230">
<span class="label">j.</span><div class="display-inline p" id="p0355">Wash slides once with PBS for 5 min at room temperature (25°C).</div>
</li>
</ul>
</div>
</li></ul>
<ul class="list" id="olist0095" style="list-style-type:none"><li id="o0235">
<span class="label">7.</span><div class="display-inline p" id="p0360">Muscle Stem Cell Staining<ul class="list" id="olist0100" style="list-style-type:none">
<li id="o0240">
<span class="label">a.</span><div class="display-inline p" id="p0365">Block sections in 50 mL antibody blocking buffer for 1 h at room temperature (25°C).</div>
</li>
<li id="o0245">
<span class="label">b.</span><div class="display-inline p" id="p0370">Incubate slides in cold (4°C) blocking buffer with VCAM antibody (1/100 dilution); stock antibody concentration 1 mg/mL, in a dark, humidified staining chamber overnight (16 h) at 4°C. We use 100 μL of antibody solution per slide. Cover slides with a coverslip to ensure even coverage of antibody solution.</div>
</li>
<li id="o0250">
<span class="label">c.</span><div class="display-inline p" id="p0375">Wash slides the following day in coplin jars with 50 mL PBS, twice, 5 min each at room temperature (25°C).</div>
</li>
<li id="o0255">
<span class="label">d.</span><div class="display-inline p" id="p0380">Incubate sections for 1 h with secondary antibody (Alexa Fluor 488 Donkey anti-goat IgG; 1/300 dilution) in antibody blocking buffer at room temperature (25°C). We use 100 μL of antibody solution per slide. Cover slides with a coverslip to ensure even coverage of antibody solution.</div>
</li>
<li id="o0260">
<span class="label">e.</span><div class="display-inline p" id="p0385">Wash slides three times for 5 min each with PBS at room temperature (25°C). PBS washes are carried out in coplin jars and using 50 mL of solution.</div>
</li>
<li id="o0265">
<span class="label">f.</span><div class="display-inline p" id="p0390">Mount coverslips with Prolong gold with DAPI.</div>
</li>
<li id="o0270">
<span class="label">g.</span><div class="display-inline p" id="p0395">Allow mounting media to solidify overnight (16 h) at room temperature (25°C) in the dark.</div>
</li>
</ul>
</div>
</li></ul>
<section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Telomere/centromere probe hybridization conditions</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Steps</th>
<th colspan="1" rowspan="1">Temperature</th>
<th colspan="1" rowspan="1">Time</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">Initial Denaturation</td>
<td colspan="1" rowspan="1">67°C</td>
<td colspan="1" rowspan="1">10 min</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Hybridization</td>
<td colspan="1" rowspan="1">42°C</td>
<td colspan="1" rowspan="1">90 min</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/tbl1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></section></section><section id="sec3.2"><h3 class="pmc_sec_title">Image acquisition</h3>
<blockquote class="text-italic"><p><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/51ca064f7ff6/fx2.gif"/><strong>Timing: less than 3 days</strong></p></blockquote>
<blockquote class="text-italic"><p><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/83e6728c8b52/fx4.gif"/><strong>CRITICAL:</strong> The imaging must take place as soon as possible after staining, and imaging must be completed for all samples in as short a period of time as possible. Given the sensitivity of Q-FISH assays, minor changes in fluorescence intensities can result in skewed datasets. Even with antifade agents in the mounting medium, it is imperative to image cells of interest as quickly as possible, so we recommend completing imaging of all samples in 3 days or less.</p></blockquote>
<ul class="list" id="olist0105" style="list-style-type:none"><li id="o0275">
<span class="label">8.</span><div class="display-inline p" id="p0405">Identify VCAM+ cells present in tissue sections using a Nikon Eclipse NiU widefield epifluorescence microscope.<blockquote class="text-italic"><div class="p">
<strong><em>Note:</em></strong> Any widefield epifluorescence microscope with appropriate objectives, filters, and camera should be capable of completing this assay.</div></blockquote>
<blockquote class="text-italic"><div class="p">
<strong><em>Note:</em></strong> While VCAM1 is a good marker of MuSCs, it may also label other cells, such as endothelial cells. To prevent selecting the wrong cell types during image acquisition and analysis, we only image MuSCs in the satellite cell position (between the basal lamina and the sarcolemma). Additionally, staining with other non-muscle markers such as CD31 and CD45 (as described in <a aria-describedby="bib8" class="usa-link" href="#bib8">Tichy et al., 2017</a>) could be performed to exclude such populations from the analysis.</div></blockquote>
<ul class="list" id="olist0110" style="list-style-type:none">
<li id="o0280">
<span class="label">a.</span><div class="display-inline p" id="p0410">Identify VCAM+ cells (Alexa Fluor 488+) with a low power air objective (i.e., Nikon plan fluor 10<strong>×</strong>/0.30 objective).</div>
</li>
<li id="o0285">
<span class="label">b.</span><div class="display-inline p" id="p0415">Switch to a higher-powered 100<strong>×</strong> objective (i.e., Nikon plan apo100<strong>×</strong>/1.45 oil) for actual imaging and photographing cells of interest.</div>
</li>
</ul>
</div>
</li></ul>
<ul class="list" id="olist0115" style="list-style-type:none"><li id="o0290">
<span class="label">9.</span><p class="display-inline" id="p0420">Identify the optimal exposure settings for DAPI, telomere Alexa Fluor 647 signal, and optional centromere Cy3 imaging. <a class="usa-link" href="#sec7.1">Troubleshooting 1</a>, <a class="usa-link" href="#sec7.3">Troubleshooting 2</a></p>
</li></ul>
<blockquote class="text-italic"><p><strong><em>Note:</em></strong> Several slides should be examined before determining the optimal exposure settings. It is undesirable to have oversaturated or undersaturated pixels in any of the images. In either situation, fewer regions of interest are identified.</p></blockquote>
<blockquote class="text-italic"><p><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/83e6728c8b52/fx4.gif"/><strong>CRITICAL:</strong> Once intensity settings are determined, the exposure settings should not be changed for the duration of the particular experiment.</p></blockquote>
<ul class="list" id="olist0120" style="list-style-type:none"><li id="o0295">
<span class="label">10.</span><div class="display-inline p" id="p0425">Proceed with multichannel imaging of VCAM, DAPI, telomere, and optional centromere signals (<a class="usa-link" href="#fig1">Figure 1</a>).<ul class="list" id="olist0125" style="list-style-type:none">
<li id="o0300">
<span class="label">a.</span><div class="display-inline p" id="p0430">Identify focal planes containing the most telomere signals for every cell of interest.</div>
</li>
<li id="o0305">
<span class="label">b.</span><div class="display-inline p" id="p0435">A minimum of 35 VCAM+ cells should be imaged per biological sample.</div>
</li>
<li id="o0310">
<span class="label">c.</span><div class="display-inline p" id="p0440">Save multichannel images in raw file formats to maintain metadata. For Nikon Elements software, this is a .nd2 file.</div>
</li>
<li id="o0315">
<span class="label">d.</span><div class="display-inline p" id="p0445">Image as many cells as possible during scheduled equipment usage hours from the biological samples. <a class="usa-link" href="#sec7.5">Troubleshooting 3</a>
</div>
</li>
</ul>
</div>
</li></ul>
<blockquote class="text-italic"><p><strong><em>Optional:</em></strong> Obtaining z-stacks and flattening with an extended depth of focus algorithm will provide more telomere and optional centromere foci per cell, but a single focal plane with many foci in focus is sufficient to yield useable data for analysis.</p></blockquote>
<blockquote class="text-italic"><p><strong><em>Note:</em></strong> Imaging more VCAM+ cells will produce a better representation of the population of MuSCs throughout the muscle. Ensure that more than one muscle section is examined per slide.</p></blockquote>
<blockquote class="text-italic"><p><strong><em>Note:</em></strong> If imaging of all desired cells from all samples cannot be completed in one day, stagger the genotypes/treatments during imaging to ensure fading that occurs between imaging sessions will be similar between the groups.</p></blockquote>
<blockquote class="text-italic"><p><strong><em>Note:</em></strong> It is best to avoid imaging/assessing cells of interest that are positioned on top of or below other cells/nuclei, as telomere length quantitated from overlapping cells may confound the datasets.</p></blockquote>
<ul class="list" id="olist0130" style="list-style-type:none"><li id="o0320">
<span class="label">11.</span><div class="display-inline p" id="p0450">Export raw image files in two groups.<ul class="list" id="olist0135" style="list-style-type:none">
<li id="o0325">
<span class="label">a.</span><div class="display-inline p" id="p0455">Each channel should be first exported as grayscale/monochrome 16-bit .tiff images for analysis.</div>
</li>
<li id="o0330">
<span class="label">b.</span><div class="display-inline p" id="p0460">A merged 8-bit color .tiff image is also exported to identify cells of interest (VCAM+) for later analysis.</div>
</li>
</ul>
</div>
</li></ul>
<blockquote class="text-italic"><p><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/b9d7883d5879/fx3.gif"/><strong>Pause point:</strong> After all needed images are collected, downstream analysis can be conducted at the end user’s leisure.</p></blockquote>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8450295_gr1.jpg" target="_blank"><img alt="Figure 1" class="graphic zoom-in" height="886" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/9232652ce432/gr1.jpg" width="736"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Representative image of a skeletal muscle cryosection processed for telomere staining</p>
<p>Blue: DAPI staining. Red: telomere probe. Green: VCAM muscle stem cell marker. White: optional centromere probe staining for alternative normalization. Yellow text: DAPI, VCAM, and telomere merged image (typical processing). Cyan text: DAPI, VCAM, telomere and centromere merged image. Use of centromere probe requires telomere length normalization to DAPI. Then centromere normalization to DAPI. Finally, a ratio of the two values is generated. Scale bar: 10 μm.</p></figcaption></figure></section><section id="sec3.3"><h3 class="pmc_sec_title">Cryosection MuSC-fluorescent <em>in situ</em> hybridization (CRYO MuQ-FISH) telomere length data collection and analysis</h3>
<blockquote class="text-italic"><p><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/51ca064f7ff6/fx2.gif"/><strong>Timing: varies based on sample size. Several hours to several days.</strong></p></blockquote>
<p id="p0470">Telomere length is measured using a semi-automated ImageJ-based platform known as Telometer. Telometer provides step-by step instructions to analyze 16-bit monochrome images for telomere length. The premise involves creating a project that is saved on a local machine, identifying cells of interest, and measuring the intensity of telomeres, relative to the total DAPI intensity of that cell. The readout from Telometer produces both total sum intensity of telomeres, which can be normalized to DAPI intensity to account for differences in cell cycle position, as well as the mean intensity of telomeres, which can also be normalized to DAPI. Should nuclear architecture or staining be compromised, telomere measurements can instead be normalized to centromere intensities. In this fashion, Telometer steps are conducted twice. Once, where the telomere signal is assessed relative to DAPI, and then those same cells of interest are analyzed a second time, where the centromere intensities are measured instead of the telomere intensities. The ratio of these measurements would then be used for dataset readouts.</p>
<ul class="list" id="olist0140" style="list-style-type:none">
<li id="o0335">
<span class="label">12.</span><p class="display-inline" id="p0475">Open Telometer software and create a new project in the “Admin” tab (<a class="usa-link" href="#fig2">Figure 2</a>A).</p>
</li>
<li id="o0340">
<span class="label">13.</span><p class="display-inline" id="p0480">Click the “Score” tab and follow the on-screen instructions for each step.</p>
</li>
<li id="o0345">
<span class="label">14.</span><p class="display-inline" id="p0485">Select the Project name for your analysis grouping and the Preset objective (in this case 100<strong>×</strong> telomere) and subtract Cy3 background. The scoring tag adds metadata to the analysis. We do not usually alter this.</p>
</li>
<li id="o0350">
<span class="label">15.</span><p class="display-inline" id="p0490">Select the Telomere 16-bit monochrome exported tiff image when prompted for opening the first image.</p>
</li>
</ul>
<blockquote class="text-italic"><p><strong><em>Note:</em></strong> Telometer defaults as the telomere signal being a Cy3 image. In this protocol, the telomere image is the Alexa Fluor 647 channel image, and it is this image that is opened first when prompted.</p></blockquote>
<ul class="list" id="olist0145" style="list-style-type:none">
<li id="o0355">
<span class="label">16.</span><p class="display-inline" id="p0495">Next, open the DAPI 16-bit monochrome exported tiff image when prompted.</p>
</li>
<li id="o0360">
<span class="label">17.</span><p class="display-inline" id="p0500">Normalize the Telomere image by clicking and forming a box of any size in an area absent of features and click “normalize” (<a class="usa-link" href="#fig2">Figure 2</a>B).</p>
</li>
<li id="o0365">
<span class="label">18.</span><p class="display-inline" id="p0505">Repeat normalization for the DAPI image when prompted (<a class="usa-link" href="#fig2">Figure 2</a>C).</p>
</li>
<li id="o0370">
<span class="label">19.</span><p class="display-inline" id="p0510">Continue following software prompts for creating masked images to narrow down regions in the image that contain telomere foci.</p>
</li>
<li id="o0375">
<span class="label">20.</span><p class="display-inline" id="p0515">Threshold the masked image, if necessary, when prompted to obtain the greatest telomere foci in the image with the least background (<a class="usa-link" href="#fig2">Figure 2</a>D).</p>
</li>
</ul>
<blockquote class="text-italic"><p><strong><em>Note:</em></strong> The Telometer software typically does a decent job with auto-thresholding, so the investigator does not often need to further threshold.</p></blockquote>
<ul class="list" id="olist0150" style="list-style-type:none">
<li id="o0380">
<span class="label">21.</span><p class="display-inline" id="p0520">Manually draw a circle encompassing a nucleus of interest when prompted. Refer to the color image to determine which nuclei are clearly VCAM+ and designate the nucleus of interest. Selection of VCAM nuclei should only occur for cells in the satellite cell position (between the basal lamina and sarcolemma). Name the nucleus as desired (<a class="usa-link" href="#fig2">Figure 2</a>E).</p>
</li>
<li id="o0385">
<span class="label">22.</span><p class="display-inline" id="p0525">Select the “+ Add Nucleus” button. Repeat for additional VCAM+ cells in the image, if any. Click the “next” button (<a class="usa-link" href="#fig2">Figure 2</a>F).</p>
</li>
</ul>
<blockquote class="text-italic"><p><strong><em>Optional:</em></strong> Telometer next gives the option to separate conjoined telomere foci (if counting telomere number) and manually remove halo artifacts from out of focus light. These functions are completely optional for the analysis.</p></blockquote>
<ul class="list" id="olist0155" style="list-style-type:none">
<li id="o0390">
<span class="label">23.</span><p class="display-inline" id="p0530">Select “complete.” The data will be stored on your local machine. To view images at a later time that have been processed and analyzed, select the “View” option in the Telometer tab (<a class="usa-link" href="#fig2">Figure 2</a>G).</p>
</li>
<li id="o0395">
<span class="label">24.</span><p class="display-inline" id="p0535">Repeat step 15–23 for remaining images to analyze. Ensure that they are being saved in the proper project data file and appropriately named.</p>
</li>
<li id="o0400">
<span class="label">25.</span><div class="display-inline p" id="p0540">Export data to copy into excel (<a class="usa-link" href="#fig2">Figure 2</a>H).<ul class="list" id="olist0160" style="list-style-type:none">
<li id="o0405">
<span class="label">a.</span><div class="display-inline p" id="p0545">In the “Export” tab of Telometer, select the project name for data export.</div>
</li>
<li id="o0410">
<span class="label">b.</span><div class="display-inline p" id="p0550">Click the “Scoring” button that has an arrow.</div>
</li>
<li id="o0415">
<span class="label">c.</span><div class="display-inline p" id="p0555">Select all data under “Scorings” and click the “Cells” button that has an arrow.</div>
</li>
<li id="o0420">
<span class="label">d.</span><div class="display-inline p" id="p0560">Select all data under “Cells” and copy and paste into Excel or other spreadsheet software. Ensure that you also note the headings for each column. <a class="usa-link" href="#sec7.7">Troubleshooting 4</a>
</div>
</li>
</ul>
</div>
</li>
<li id="o0425">
<span class="label">26.</span><p class="display-inline" id="p0565">To analyze datasets using the Telomere sum option, refer to the last column of the dataset. It represents the telomere sum intensity divided by DAPI sum intensity formula∗100.</p>
</li>
</ul>
<blockquote class="text-italic"><p><strong><em>Optional:</em></strong> If telomere mean intensities are desired per cell instead of total (sum) intensities, calculate this value by dividing the datasets in the Telomere Mean column by its respective DAPI Sum.</p></blockquote>
<blockquote class="text-italic"><p><strong><em>Optional:</em></strong> If one desires to normalize to centromere intensity instead of DAPI intensity, the procedure for telomere analysis is repeated, but instead of using the Telomere images on prompt to open, the centromere images are opened instead. The nuclei should be named and selected similarly in the same order as the telomere analysis. Then the last column of data from the telomere analysis is divided by the last column of centromere data analysis. <a class="usa-link" href="#sec7.9">Troubleshooting 5</a></p></blockquote>
<ul class="list" id="olist0165" style="list-style-type:none"><li id="o0430">
<span class="label">27.</span><p class="display-inline" id="p0570">Graph datasets as desired in Graphpad prism. Data can be graphed as total intensity values, or data can be graphed as histograms to demonstrate minor shifts of telomere length within the populations (<a class="usa-link" href="#fig3">Figure 3</a>). <a class="usa-link" href="#sec7.11">Troubleshooting 6</a></p>
</li></ul>
<blockquote class="text-italic"><p><strong><em>Note:</em></strong> The telomere and centromere probes are probes that recognize mammalian DNA sequences and are functional in other species, including samples from rat and human. We have found that this protocol is compatible with muscle tissues of human and rat skeletal muscle cryosections (<a class="usa-link" href="#fig4">Figures 4</a> and <a class="usa-link" href="#fig5">5</a>). We also examined the adaptability of this protocol to several other tissue types (spleen, liver, kidney, testicle, and colon) and found this protocol does appropriately label telomeres in all tissues examined (<a class="usa-link" href="#fig5">Figure 5</a>), increasing the impact and usability of this method. However, in such tissues, it may be necessary to utilize thinner sections or more advanced microscopes (confocal microscopy), given the increased number of focal planes exhibiting telomere foci observed.</p></blockquote>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8450295_gr2.jpg" target="_blank"><img alt="Figure 2" class="graphic zoom-in" height="531" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/643a95f5e0b1/gr2.jpg" width="749"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Processing cells of interest for telomere analysis by Telometer</p>
<p>(A) Telomere radio button depicting different options for data acquisition and analysis.</p>
<p>(B) Representative monochrome image of telomere image from <a class="usa-link" href="#fig1">Figure 1</a>. Yellow box is the region chosen for background normalization.</p>
<p>(C). Representative DAPI monochrome image from <a class="usa-link" href="#fig1">Figure 1</a>. Yellow box is the region chosen for background normalization.</p>
<p>(D) Thresholding of telomere channel to determine optimal balance between telomere foci identification with the least background.</p>
<p>(E) Selection of VCAM+ nucleus from <a class="usa-link" href="#fig1">Figure 1</a> by manually outlining when prompted by the software.</p>
<p>(F) Naming of a selected VCAM+ nucleus.</p>
<p>(G) Masked image used for data analysis by Telometer.</p>
<p>(H) Selected data outputs provided by Telometer analysis. Cell ID: unique identifier for a particular cell that was analyzed. Experiment name: selected at beginning of analysis (project name). Scoring Name (Tag): prefix added to dataset. Usually the date of analysis. Sum (DAPI): measured DAPI total intensity. Sum (Telomere): Total measured telomere intensity. Mean (Telomere): Mean intensity of Telomere signal per nucleus. Telomere Sum/DAPI Sum (Percent): Value used to graph datasets.</p></figcaption></figure><figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8450295_gr3.jpg" target="_blank"><img alt="Figure 3" class="graphic zoom-in" height="567" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/df47426bda92/gr3.jpg" width="736"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Example graphical plots from Telometer analysis</p>
<p>(A) CRYO MuQ-FISH Telomere Sum Intensity/DAPI Sum Intensity (Percent) data graphed from 2 different mouse genotypes subjected to chronic injury. Note the significant reduction of telomere lengths in genotype 2. Genotypes depicted in this figure are those used in our companion article (<a aria-describedby="bib7" class="usa-link" href="#bib7">Tichy et al., 2021</a>) and are control and IKK2CA<sup>MuSC</sup> mice subjected to chronic injuries.</p>
<p>(B) CRYO MuQ-FISH data binned in Microsoft Excel from (A) and plotted as percent of cells per mouse and per genotype. Bins are arbitrarily assigned. Note that the shift of the population of genotype 2 to the left is indicative of telomere shortening.</p>
<p>(C) CRYO MuQ-FISH Telomere Mean Intensity/DAPI Sum Intensity (calculated percent) from analysis in (A).</p>
<p>(D) Histogram representation of data from (C). For all graphs, N=3 mice per genotype, with a minimum of 35 cells imaged per mouse. Depicted are mean ± SEM. ∗∗∗∗ p&lt;0.0001.</p></figcaption></figure><figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8450295_gr4.jpg" target="_blank"><img alt="Figure 4" class="graphic zoom-in" height="1251" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/d93872574e64/gr4.jpg" width="736"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Application of CRYO MUQ-FISH to human skeletal muscle samples reveals compatibility of the method with human tissues</p>
<p>Cryosections were stained with the endothelial marker CD31 and processed for CRYO MuQ-FISH. Scale bar: 10 μm.</p></figcaption></figure><figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">Figure 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8450295_gr5.jpg" target="_blank"><img alt="Figure 5" class="graphic zoom-in" height="927" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/36f0aeb38f1b/gr5.jpg" width="736"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Assessment of CRYO MuQ-FISH protocol translatability to additional tissue and organism types</p>
<p>Scale bar: 10 μm. Note- background was left intentionally higher to demonstrate surrounding tissue architecture. Adult 17-month-old male Sprague-Dawley rat tissues were collected and processed identically as described in the CRYO MuQ-FISH protocol and imaged for telomere signal and DAPI. We found this protocol capable of staining telomeres in tissues of the tibialis anterior muscle, colon, liver, kidney, spleen, and testicle. Scale bar: 10 μm.</p></figcaption></figure></section></section><section id="sec4"><h2 class="pmc_sec_title">Expected outcomes</h2>
<p id="p0580">This protocol demonstrates the methodology to process cryosections of skeletal muscle for telomere analysis. We utilized VCAM as a muscle stem cell marker (<a aria-describedby="bib2" class="usa-link" href="#bib2">Liu et al., 2015</a>; <a aria-describedby="bib3" class="usa-link" href="#bib3">Maesner et al., 2016</a>); however, this protocol can likely be applied to additional resident skeletal muscle populations, once appropriate markers are identified and tested for compatibility. Here, we utilized data from two different genotypes of mice that underwent chronic, repetitive injuries, to demonstrate different ways to represent the data (<a class="usa-link" href="#fig3">Figure 3</a>). Additionally, to confirm the validity of the CRYO MuQ-FISH method, we compared CRYO MuQ-FISH with our traditional MuQ-FISH method using isolated MuSCs. We found similar pattens of telomere shortening in a mouse model of Duchenne muscular dystrophy coupled with a telomere defect (<a class="usa-link" href="#fig6">Figure 6</a>).</p>
<figure class="fig xbox font-sm" id="fig6"><h3 class="obj_head">Figure 6.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8450295_gr6.jpg" target="_blank"><img alt="Figure 6" class="graphic zoom-in" height="646" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/10d6/8450295/9ca072223928/gr6.jpg" width="736"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Comparisons of CRYO MuQ-FISH with previously established telomere length assessments in isolated MuSCs demonstrates no differences in expected data outcomes</p>
<p>(A) MuSCs were FACS-sorted (CD11b-/CD31-/CD45-/Sca1-/CD34+/α7Integrin+) from wildtype mice or dystrophic mice lacking the telomere component Terc (mdx/mTR<sup>G2</sup>). MuSCs were subjected to our previously established and validated MuQ-FISH method to assess telomere length. Depicted are mean ± SEM. Percent difference between the two groups is displayed.</p>
<p>(B) Data from (A) plotted as a histogram to demonstrate the cellular distribution of telomere lengths.</p>
<p>(C) CRYO MuQ-FISH of VCAM+ cells in skeletal muscle cryosections from mice used in (A). Gastrocnemius muscles were processed for telomere length by the CRYO MuQ-FISH protocol. Note the similar percent difference in telomere length between the groups compared with what is shown in (A).</p>
<p>(D) Histogram representation of data depicted in (C). Note the similarity in structure of the histograms between (C) and (D). For all graphs, N=3 male mice per genotype (5–7 months of age), with a minimum of 35 cells imaged per mouse. Depicted are mean ± SEM.</p></figcaption></figure></section><section id="sec5"><h2 class="pmc_sec_title">Quantification and statistical analysis</h2>
<p id="p0585">Comparisons of data between genotypes are conducted using unpaired student T-tests with Welch’s correction. Cells from at 3 mice or samples per experimental condition should be examined, with at least 35 cells imaged per mouse.</p></section><section id="sec6"><h2 class="pmc_sec_title">Limitations</h2>
<p id="p0590">This protocol utilizes the quantitative fluorescence in situ hybridization method for measuring telomere length. Because measurements of telomere length are based on fluorescent intensities, it is critical to image the slides as soon as possible, since fading will occur over time. Samples should not be imaged between different experiments, but instead they should only be imaged in the same batch in as short a time window as possible, since fluorescent intensities will be variable between experiments. Normalization must be done to respective control samples to combine datasets. Since MuSCs comprise a rare population of cells and are very small in size, a 100<strong>×</strong> or equivalent high-powered objective is necessary to visualize the telomere foci to get meaningful datasets.</p></section><section id="sec7"><h2 class="pmc_sec_title">Troubleshooting</h2>
<section id="sec7.1"><h3 class="pmc_sec_title">Problem 1</h3>
<p id="p0595">No Q-FISH telomere probe signal present following imaging (step 9).</p></section><section id="sec7.2"><h3 class="pmc_sec_title">Potential solution</h3>
<p id="p0600">Antigen retrieval time was insufficient or not completed with boiling buffer. Ensure that temperatures and times are completed as outlined in this protocol.</p>
<p id="p0605">Probe was bleached or stored improperly after being prepared. Always use a new aliquot of telomere/centromere probe from −80°C and ensure that it is protected from light throughout this protocol.</p>
<p id="p0610">Inappropriate dilution of probe was used, hybridization buffer was not correctly made, hybridization conditions were not followed. If probes are too dilute, increase the amount of the Q-FISH probes used in the protocol. Confirm that hybridization buffer is completely dissolved and that no air bubbles were present when hybridization occurred. Confirm the heating source during denaturation and hybridization is at the proper temperatures.</p></section><section id="sec7.3"><h3 class="pmc_sec_title">Problem 2</h3>
<p id="p0615">High background staining (step 9).</p></section><section id="sec7.4"><h3 class="pmc_sec_title">Potential solution</h3>
<p id="p0620">Do not let the tissue dry out at any step following permeabilization. Avoid Cy3/Alexa fluor red/orange dyes except for centromeric region staining.</p></section><section id="sec7.5"><h3 class="pmc_sec_title">Problem 3</h3>
<p id="p0625">Large numbers of samples that cannot be imaged in the required timeframe (step 10d).</p></section><section id="sec7.6"><h3 class="pmc_sec_title">Potential solution</h3>
<p id="p0630">While imaging samples in the same time window is optimal, if too many samples are present, a 3-day window may not be sufficient time to complete the acquisition, resulting in fading of the sample and alteration of the data. To overcome this caveat, it may be possible to normalize the intensity of a particular reference sample imaged on different days using fluorescent beads, such as those found in the InSpeck Microscopy Image Intensity Calibration Kit. These kits contain beads of varying but defined intensities from which a standard curve can be generated on each day within the same experiment.</p></section><section id="sec7.7"><h3 class="pmc_sec_title">Problem 4</h3>
<p id="p0635">Telometer is not displaying recorded datasets (step 25d).</p></section><section id="sec7.8"><h3 class="pmc_sec_title">Potential solution</h3>
<p id="p0640">Ensure that you are trying to view data in the appropriate project file. It may be necessary to use the “refresh” button to repopulate the project file, particularly if one is trying to view data that was accumulated since Telometer was first opened.</p></section><section id="sec7.9"><h3 class="pmc_sec_title">Problem 5</h3>
<p id="p0645">DAPI staining intensity is not consistent between experimental groups or nuclear architecture is dissimilar (step 26).</p></section><section id="sec7.10"><h3 class="pmc_sec_title">Potential solution</h3>
<p id="p0650">Conduct telomere analysis by normalizing to a centromeric probe. First do telomere analysis normalized to DAPI. Then repeat the Telometer analysis on the same cells using the centromere image instead of the telomere image. Ensure to label cells of interest in the same order between the telomere identification and the centromere identification. Finally, divide the Telomere sum/DAPI sum percent by the Centromere sum/DAPI sum. Utilize those values for graphing.</p></section><section id="sec7.11"><h3 class="pmc_sec_title">Problem 6</h3>
<p id="p0655">Inconsistent data between experiments (step 27).</p></section><section id="sec7.12"><h3 class="pmc_sec_title">Potential solution</h3>
<p id="p0660">Given that Q-FISH is an intensity-based measurement, experimental samples are ideally measured at the same time with the same exposure settings. If too many samples are present, fading can occur between sample sets that would confound downstream analysis. Thus, it is optimal to image all experimental groups as soon as possible, and comparisons between different experiments are not possible. Additionally, telomere probe in hybridization buffer should completely cover the sample without any air bubbles. While the values themselves may be different, due to inherent differences between experimental preparations, calculations of percent difference between groups can be useful in confirming that datasets are similar.</p></section></section><section id="sec8"><h2 class="pmc_sec_title">Resource availability</h2>
<section id="sec8.1"><h3 class="pmc_sec_title">Lead contact</h3>
<p id="p0665">Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Foteini Mourkioti (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://mailto:fmour@pennmedicine.upenn.edu" rel="noopener noreferrer" target="_blank">fmour@pennmedicine.upenn.edu</a>).</p></section><section id="sec8.2"><h3 class="pmc_sec_title">Materials availability</h3>
<p id="p0670">This study did not generate new materials or unique reagents.</p></section></section><section class="ack" id="ack0010"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p id="p0680">We thank both Jim Morgan and Alan Meeker (Johns Hopkins) for the current version of Telometer. This work was supported by the NASA (18-FG_ind_2–0022) and the NIH (R01HL146662) to F.M.</p>
<section id="sec10"><h3 class="pmc_sec_title">Author contributions</h3>
<p id="p0685">E.D.T. designed and performed experiments, developed, and optimized the protocol, and wrote the manuscript. F.M. designed experiments, interpreted data, and wrote the manuscript.</p></section><section id="sec11"><h3 class="pmc_sec_title">Declaration of interests</h3>
<p id="p0690">The authors declare no competing interests.</p></section></section><section class="contrib-info" id="_ci93_" lang="en"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Elisia D. Tichy, Email: etichy@pennmedicine.upenn.edu.</p>
<p>Foteini Mourkioti, Email: fmour@pennmedicine.upenn.edu.</p></section><section id="sec32"><h2 class="pmc_sec_title">Data and code availability</h2>
<p id="p0030">This study did not generate any unique data sets or code.</p></section><section class="ref-list" id="cebib0010"><h2 class="pmc_sec_title">References</h2>
<section id="cebib0010_sec2"><ol class="ref-list font-sm">
<li id="bib1">
<cite>Cawthon R.M. Telomere measurement by quantitative PCR. Nucleic Acids Res. 2002;30:e47. doi: 10.1093/nar/30.10.e47.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/30.10.e47" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC115301/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12000852/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Telomere%20measurement%C2%A0by%20quantitative%20PCR&amp;author=R.M.%20Cawthon&amp;volume=30&amp;publication_year=2002&amp;pages=e47&amp;pmid=12000852&amp;doi=10.1093/nar/30.10.e47&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib2">
<cite>Liu L., Cheung T.H., Charville G.W., Rando T.A. Isolation of skeletal muscle stem cells by fluorescence-activated cell sorting. Nat. Protoc. 2015;10:1612–1624. doi: 10.1038/nprot.2015.110.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nprot.2015.110" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4793971/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26401916/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Isolation%20of%20skeletal%20muscle%20stem%20cells%20by%20fluorescence-activated%20cell%20sorting&amp;author=L.%20Liu&amp;author=T.H.%20Cheung&amp;author=G.W.%20Charville&amp;author=T.A.%20Rando&amp;volume=10&amp;publication_year=2015&amp;pages=1612-1624&amp;pmid=26401916&amp;doi=10.1038/nprot.2015.110&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib3">
<cite>Maesner C.C., Almada A.E., Wagers A.J. Established cell surface markers efficiently isolate highly overlapping populations of skeletal muscle satellite cells by fluorescence-activated cell sorting. Skelet. Muscle. 2016;6:35. doi: 10.1186/s13395-016-0106-6.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s13395-016-0106-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5100091/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27826411/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Skelet.%20Muscle&amp;title=Established%20cell%20surface%20markers%20efficiently%20isolate%20highly%20overlapping%20populations%20of%20skeletal%20muscle%20satellite%20cells%20by%20fluorescence-activated%20cell%20sorting&amp;author=C.C.%20Maesner&amp;author=A.E.%20Almada&amp;author=A.J.%20Wagers&amp;volume=6&amp;publication_year=2016&amp;pages=35&amp;pmid=27826411&amp;doi=10.1186/s13395-016-0106-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib4">
<cite>Matthiesen S.H., Hansen C.M. Fast and non-toxic in situ hybridization without blocking of repetitive sequences. PLoS One. 2012;7:e40675. doi: 10.1371/journal.pone.0040675.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0040675" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3404051/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22911704/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Fast%20and%20non-toxic%20in%C2%A0situ%20hybridization%20without%20blocking%20of%20repetitive%20sequences&amp;author=S.H.%20Matthiesen&amp;author=C.M.%20Hansen&amp;volume=7&amp;publication_year=2012&amp;pages=e40675&amp;pmid=22911704&amp;doi=10.1371/journal.pone.0040675&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib5">
<cite>Mender I., Shay J.W. Telomere restriction fragment (TRF) analysis. Bio Protoc. 2015;5:e1658. doi: 10.21769/bioprotoc.1658.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.21769/bioprotoc.1658" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4972328/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27500189/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bio%20Protoc.&amp;title=Telomere%20restriction%20fragment%20(TRF)%20analysis&amp;author=I.%20Mender&amp;author=J.W.%20Shay&amp;volume=5&amp;publication_year=2015&amp;pages=e1658&amp;pmid=27500189&amp;doi=10.21769/bioprotoc.1658&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib6">
<cite>Sacco A., Mourkioti F., Tran R., Choi J., Llewellyn M., Kraft P., Shkreli M., Delp S., Pomerantz J.H., Artandi S.E. Short telomeres and stem cell exhaustion model Duchenne muscular dystrophy in mdx/mTR mice. Cell. 2010;143:1059–1071. doi: 10.1016/j.cell.2010.11.039.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cell.2010.11.039" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3025608/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21145579/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Short%20telomeres%20and%20stem%20cell%20exhaustion%20model%20Duchenne%20muscular%20dystrophy%20in%20mdx/mTR%20mice&amp;author=A.%20Sacco&amp;author=F.%20Mourkioti&amp;author=R.%20Tran&amp;author=J.%20Choi&amp;author=M.%20Llewellyn&amp;volume=143&amp;publication_year=2010&amp;pages=1059-1071&amp;pmid=21145579&amp;doi=10.1016/j.cell.2010.11.039&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib7">
<cite>Tichy E.D., Ma N., Sidibe D., Loro E., Kocan J., Chen D.Z., Khurana T.S., Hasty P., Mourkioti F. Persistent NF-kappaB activation in muscle stem cells induces proliferation-independent telomere shortening. Cell Rep. 2021;35:109098. doi: 10.1016/j.celrep.2021.109098.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.celrep.2021.109098" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8183356/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33979621/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=Persistent%20NF-kappaB%20activation%20in%20muscle%20stem%20cells%20induces%20proliferation-independent%20telomere%20shortening&amp;author=E.D.%20Tichy&amp;author=N.%20Ma&amp;author=D.%20Sidibe&amp;author=E.%20Loro&amp;author=J.%20Kocan&amp;volume=35&amp;publication_year=2021&amp;pages=109098&amp;pmid=33979621&amp;doi=10.1016/j.celrep.2021.109098&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib8">
<cite>Tichy E.D., Sidibe D.K., Tierney M.T., Stec M.J., Sharifi-Sanjani M., Hosalkar H., Mubarak S., Johnson F.B., Sacco A., Mourkioti F. Single stem cell imaging and analysis reveals telomere length differences in diseased human and mouse skeletal muscles. Stem Cell Rep. 2017;9:1328–1341. doi: 10.1016/j.stemcr.2017.08.003.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.stemcr.2017.08.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5639167/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28890163/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Stem%20Cell%20Rep.&amp;title=Single%20stem%20cell%20imaging%20and%20analysis%20reveals%20telomere%20length%20differences%20in%20diseased%20human%20and%20mouse%20skeletal%20muscles&amp;author=E.D.%20Tichy&amp;author=D.K.%20Sidibe&amp;author=M.T.%20Tierney&amp;author=M.J.%20Stec&amp;author=M.%20Sharifi-Sanjani&amp;volume=9&amp;publication_year=2017&amp;pages=1328-1341&amp;pmid=28890163&amp;doi=10.1016/j.stemcr.2017.08.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ol></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>This study did not generate any unique data sets or code.</p></section></section></section></section></article>
</main></body></html>