<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8904044/"/><title>Environmental Dependence of Competitive Fitness in Rifampin-Resistant rpoB Mutants of Bacillus subtilis - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Appl Environ Microbiol</button></div>. 2022 Mar 8;88(5):e02422-21. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/aem.02422-21" rel="noopener noreferrer" target="_blank">10.1128/aem.02422-21</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Environmental Dependence of Competitive Fitness in Rifampin-Resistant <em>rpoB</em> Mutants of <em>Bacillus subtilis</em>
</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Leehan%20JD%22%5BAuthor%5D"><span class="name western">Joss D Leehan</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Joss D Leehan</span></h3>
<div class="p">
<sup>a</sup>Department of Microbiology and Cell Science, University of Florida, Merritt Island, Florida, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Leehan%20JD%22%5BAuthor%5D"><span class="name western">Joss D Leehan</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nicholson%20WL%22%5BAuthor%5D"><span class="name western">Wayne L Nicholson</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wayne L Nicholson</span></h3>
<div class="p">
<sup>a</sup>Department of Microbiology and Cell Science, University of Florida, Merritt Island, Florida, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nicholson%20WL%22%5BAuthor%5D"><span class="name western">Wayne L Nicholson</span></a>
</div>
</div>
<sup>a,</sup><sup>✉</sup>
</div>
<div class="cg p">Editor: <span class="name western">Maia Kivisaar</span><sup>b</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>a</sup>Department of Microbiology and Cell Science, University of Florida, Merritt Island, Florida, USA</div>
<div id="aff2">
<sup>b</sup>University of Tartu</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>The authors declare no conflict of interest.</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Maia Kivisaar</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2021 Dec 8; Accepted 2022 Jan 14; Prepublished 2022 Jan 19; Collection date 2022 Mar.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2022 American Society for Microbiology.</div>
<p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/ASMCopyrightv2" rel="noopener noreferrer" target="_blank">All Rights Reserved</a>.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC8904044  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35258334/">35258334</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>RNA polymerase (RNAP) is a highly conserved macromolecular machine that contributes to the flow of genetic information from genotype to phenotype. In <em>Bacillus subtilis</em>, mutations in the <em>rpoB</em> gene encoding the β-subunit of RNAP have been shown to alter a number of global phenotypes, including growth, utilization of unusual nutrient sources, sporulation, germination, and production of secondary metabolites. In addition, the spectrum of mutations in <em>rpoB</em> leading to rifampin resistance (Rif<sup>r</sup>) can change dramatically depending upon the environment to which <em>B. subtilis</em> cells or spores are exposed. Rif<sup>r</sup>
<em>rpoB</em> mutations have historically been associated with slower growth and reduced fitness; however, these assessments of fitness were conducted on limited collections of mutants in rich laboratory media that poorly reflect natural environments typically inhabited by <em>B. subtilis</em>. Using a novel deep-sequencing approach in addition to traditional measurements of growth rate, lag time, and pairwise competitions, we demonstrated that the competitive advantages of specific <em>rpoB</em> alleles differ depending on the growth environment in which they are determined.</p>
<p><strong>IMPORTANCE</strong> Microbial resistance to antibiotics is a growing threat to public health across the world. Historically, resistance to antibiotics has been associated with reduced fitness. A growing body of evidence indicates that resistance to rifampin, a frontline antibiotic used to treat mycobacterial and biofilm-associated infections, may increase fitness given an appropriate environment even in the absence of the selective antibiotic. Here, we experimentally confirm this phenomenon by directly comparing the fitness of multiple rifampin-resistant mutants of <em>Bacillus subtilis</em> in rich LB medium and an asparagine minimal medium. Our research demonstrates that the fitness cost of rifampin resistance can vary greatly depending upon the environment. This has important implications for understanding how microbes develop antimicrobial resistance in the absence of antibiotic selection.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>KEYWORDS:</strong> <em>Bacillus subtilis</em>, evolution, RNA polymerase, fitness, rifampin, rifampin resistance, transcription</p></section></section><section id="s1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p>Transcription in bacteria depends on a complex of proteins collectively known as RNA polymerase (RNAP), and bacterial responses to their environment are in large part due to alterations in their global pattern of transcription, i.e., the transcriptome. Because RNAP physically contacts every promoter in the bacterial genome, mutations altering RNAP can exert wide-ranging pleiotropic effects on the transcriptome, and hence, the resulting phenotype and fitness of a cell growing in a particular environment. The antibiotic rifampin (Rif) is a potent inhibitor of prokaryotic transcription initiation (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>) and has a long history of use in the treatment of infectious diseases, particularly tuberculosis and leprosy caused by <em>Mycobacterium</em> spp., as well as biofilm-associated infections (<a aria-describedby="B2" class="usa-link" href="#B2">2</a>). In addition to its clinical importance, Rif has long been used to probe the structure and function of RNAP. Resistance to Rif (Rif<sup>r</sup>) arises from mutations in the <em>rpoB</em> gene, encoding the β subunit of RNAP (<a aria-describedby="B3" class="usa-link" href="#B3">3</a>). Binding of Rif to RNAP occurs at a specific site on the β subunit, the Rif-binding pocket, which physically blocks the progression of nascent transcripts through the RNA exit channel (<a aria-describedby="B4" class="usa-link" href="#B4">4</a>, <a aria-describedby="B5" class="usa-link" href="#B5">5</a>). Mutations conferring Rif<sup>r</sup> are localized on the primary sequence of <em>rpoB</em> in four regions designated the N cluster and clusters I, II, and III (<a aria-describedby="B4" class="usa-link" href="#B4">4</a>, <a aria-describedby="B6" class="usa-link" href="#B6">6</a>). The majority of Rif<sup>r</sup> mutations occur in cluster I, and elucidation of the three-dimensional structure of RNAP bound to Rif has revealed that the Rif binding pocket consists of several key amino acids within cluster I that make direct contact with Rif (<a aria-describedby="B4" class="usa-link" href="#B4">4</a>). In the Gram-positive model organism <em>Bacillus subtilis</em>, Rif<sup>r</sup> mutations in <em>rpoB</em> have been shown to affect a number of global phenotypes, including deficiencies in sporulation, competence, growth rate, and yield (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>), activation of cryptic genes involved in the utilization of rare carbon sources (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>), altered resistance of spores to various sporicidal stresses (<a aria-describedby="B9" class="usa-link" href="#B9">9</a>), temperature-sensitive sporulation (<a aria-describedby="B10" class="usa-link" href="#B10">10</a>), and spore germination (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>).</p>
<p>Not all sites in cluster I of <em>rpoB</em> are prone to mutation at equal frequency; some bases mutate at higher frequency (“hot spots”) or lower frequency (“cold spots”). Such patterns of mutation in a gene are referred to as its mutational spectrum (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>). In prior communications, we have reported several instances in which the Rif<sup>r</sup> mutational spectrum in the <em>B. subtilis</em>
<em>rpoB</em> gene is changed when cells are cultivated in different physiological environments or when spores are exposed to different environmental extremes (<a aria-describedby="B12" class="usa-link" href="#B12">12</a><a aria-describedby="B13" class="usa-link" href="#B13">–</a><a aria-describedby="B17" class="usa-link" href="#B17">17</a>). Most recently, we reported on differences in the spectrum of Rif<sup>r</sup> mutations between populations grown in complete Miller LB medium (LB) and populations grown in Spizizen minimal medium containing <span class="font-variant-small-caps">l</span>-asparagine as the sole carbon source (SMM<sub>Asn</sub>) (<a aria-describedby="B18" class="usa-link" href="#B18">18</a>). We observed that the major Rif<sup>r</sup> amino acid change in LB cultures was H482Y (a change of H to Y at position 482), while the major mutation in SMM<sub>Asn</sub> cultures was S487L (<a aria-describedby="B18" class="usa-link" href="#B18">18</a>). How does exposure of a cell or spore to a particular environment lead to an altered mutational spectrum? At least two explanations for this phenomenon can be posited, which are not mutually exclusive.</p>
<p>(i) One of the explanations is that under different physiological conditions, changes in the intracellular environment can cause alterations in nucleoid architecture, DNA supercoiling, and DNA conformation; such structural alterations can alter the probability of mutation at a particular base within a gene. This notion is supported by the observed differences both in DNA structure and in the spectrum of <em>rpoB</em> mutations between vegetative cells and spores in <em>B. subtilis</em>, where the mutagenic spectrum of spores more closely resembled the spectrum observed in clinical <em>Mycobacterium tuberculosis</em> isolates (<a aria-describedby="B12" class="usa-link" href="#B12">12</a>). Differences in the spectrum of <em>rpoB</em> mutations have also been observed in <em>B. subtilis</em> spores exposed to a number of extreme environmental conditions, including simulated Mars conditions (<a aria-describedby="B13" class="usa-link" href="#B13">13</a>), ultrahigh vacuum (<a aria-describedby="B19" class="usa-link" href="#B19">19</a>), and exposure to space in low-Earth orbit (<a aria-describedby="B15" class="usa-link" href="#B15">15</a>). It should be noted that in the examples listed above, dormant spores themselves were exposed to different environments and then allowed to germinate and grow under the same laboratory conditions; hence, differences in mutational spectrum cannot be attributed to different growth environments.</p>
<p>(ii) The second explanation is that growth under selective pressure produces a spectrum of mutations within a population and those mutants with the lowest fitness cost or those with a fitness advantage become more prevalent. Supporting this line of thought, Wrande et al. (<a aria-describedby="B20" class="usa-link" href="#B20">20</a>) illustrated that the logarithmic accumulation of Rif<sup>r</sup> mutations in aging <em>Salmonella enterica</em> and <em>Escherichia coli</em> colonies is due to clonal growth of Rif<sup>r</sup> mutants in localized “jackpots” within the colony. These Rif<sup>r</sup> alleles were found to have a selective advantage over their respective ancestral strains within the context of these aging colonies. Similarly, study of Rif<sup>r</sup>
<em>Staphylococcus aureus</em> mutants isolated from biofilms revealed that Rif<sup>r</sup> mutants exhibited greater fitness when cocultured with the wild-type strain in a biofilm liquid culture (<a aria-describedby="B21" class="usa-link" href="#B21">21</a>).</p>
<p>It is possible that these two mechanisms work together in determining the spectra of mutations in both vegetative cells and spores, due to the integral roles that DNA bending and supercoiling play in regulating bacterial transcription and, in turn, the role that transcription plays in determining DNA architecture (reviewed in references <a aria-describedby="B22" class="usa-link" href="#B22">22</a>, <a aria-describedby="B23" class="usa-link" href="#B23">23</a>, and <a aria-describedby="B24" class="usa-link" href="#B24">24</a>). This inextricable linkage between gene expression and DNA supercoiling makes these notions explaining variations in spectra of mutations difficult to disentangle.</p>
<p>To determine whether growth under differing selective pressures is a contributing factor in differences in the spectra of mutations, we report here the development of a competition assay utilizing next-generation-sequencing technology to quantify the fitness of multiple <em>rpoB</em> alleles in a mixed culture simultaneously. We compared the population frequencies of 18 <em>rpoB</em> alleles in mixed cultures grown in two distinct environments: LB and SMM<sub>Asn</sub>. We found significant differences in the fitness of the various Rif<sup>r</sup> mutant <em>rpoB</em> alleles in each medium. In LB, the wild-type allele dominated, whereas in SMM<sub>Asn</sub>, the H482N mutant became the dominant allele present in the population. These results were subsequently confirmed by pairwise competition experiments, as well as assessments of exponential growth rates and lag times.</p></section><section id="s2"><h2 class="pmc_sec_title">RESULTS AND DISCUSSION</h2>
<section id="s2-1"><h3 class="pmc_sec_title">Levels of Rif resistance conferred by mutant <em>rpoB</em> alleles.</h3>
<p>The natural environment often contains inhibitory or subinhibitory levels of various antibiotics; thus, natural antibiotic resistance would confer a selective advantage in such environments (reviewed in reference <a aria-describedby="B25" class="usa-link" href="#B25">25</a>). Through our studies of Rif<sup>r</sup> over the years, we have amassed a collection of congenic <em>B. subtilis</em> strains, each carrying 1 of 17 different Rif<sup>r</sup> mutations in <em>rpoB</em> (<a class="usa-link" href="#T1">Table 1</a>). We measured the levels of Rif<sup>r</sup> in these strains by measuring the 50% inhibitory concentration (IC<sub>50</sub>) for Rif in LB medium by the broth dilution method (<a class="usa-link" href="#F1">Fig. 1</a>). As expected, the wild-type laboratory strain was very sensitive to Rif (IC<sub>50</sub> of 0.06 ± 0.01 μg/mL). All mutant <em>rpoB</em> alleles that had been selected on a high concentration of Rif (50 μg/mL) (<a aria-describedby="B12" class="usa-link" href="#B12">12</a>, <a aria-describedby="B13" class="usa-link" href="#B13">13</a>) exhibited high IC<sub>50</sub> values of &gt;100 μg/mL, whereas mutant <em>rpoB</em> alleles selected on a low concentration of Rif (5 μg/mL) (<a aria-describedby="B17" class="usa-link" href="#B17">17</a>) exhibited generally lower IC<sub>50</sub> values, ranging from 4 to 84 μg/mL, the lowest of which was still at least 67-fold higher than that of the wild-type strain (<a class="usa-link" href="#F1">Fig. 1</a>). Thus, in the context of growth in a selective environment containing Rif, all the Rif<sup>r</sup>
<em>rpoB</em> mutants would be expected to be more fit than the wild-type strain.</p>
<section class="tw xbox font-sm" id="T1"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p><em>B. subtilis</em> strains and plasmids used in this study</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1"/>
<col span="1"/>
<col span="1"/>
</colgroup>
<thead><tr>
<th colspan="1" rowspan="1">Strain or plasmid</th>
<th colspan="1" rowspan="1">Genotype, phenotype, or description</th>
<th colspan="1" rowspan="1">Reference and/or source<a class="usa-link" href="#T1F1"><sup><em>a</em></sup></a>
</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">Strain</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN547</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 cat 6His-rpoC</em>; Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B61" class="usa-link" href="#B61">61</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN624</td>
<td colspan="1" rowspan="1">
<em>trpC2 amyE</em>::<em>spc</em>; Spc<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B53" class="usa-link" href="#B53">53</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN758</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-Q469R <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B7" class="usa-link" href="#B7">7</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN759</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-H482R <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B7" class="usa-link" href="#B7">7</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN760</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-H482Y <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B7" class="usa-link" href="#B7">7</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN761</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-S487L <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B7" class="usa-link" href="#B7">7</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN999</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-S487F <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B13" class="usa-link" href="#B13">13</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1000</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-S487Y <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B13" class="usa-link" href="#B13">13</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1002</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-H482D <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B13" class="usa-link" href="#B13">13</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1004</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-V135F <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B13" class="usa-link" href="#B13">13</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1007</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-Q469K <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B13" class="usa-link" href="#B13">13</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1009</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-H482P <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B13" class="usa-link" href="#B13">13</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1011</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-Q469L <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B13" class="usa-link" href="#B13">13</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1191</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-A478D <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1261</td>
<td colspan="1" rowspan="1">
<em>trpC2 amyE</em>::<em>neo</em>; Neo<sup>r</sup>
</td>
<td colspan="1" rowspan="1">
<a aria-describedby="B62" class="usa-link" href="#B62">62</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1624</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-D472V <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1625</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-D472Y <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1626</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-Δ(E479-H482) <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1637</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-H482N <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1647</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-A478V <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>
</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1651</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 amyE</em>::<em>spc cat 6His-rpoC</em>; Cm<sup>r</sup> Spc<sup>r</sup>
</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1653</td>
<td colspan="1" rowspan="1">
<em>trpC2 pheA1 rpoB</em>-A478D,I528V <em>cat 6His-rpoC</em>; Rif<sup>r</sup> Cm<sup>r</sup>; reconstructed</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1658</td>
<td colspan="1" rowspan="1">
<em>trpC2 rpoB</em>-A478D <em>amyE</em>::<em>neo</em>; Neo<sup>r</sup> Rif<sup>r</sup>
</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1662</td>
<td colspan="1" rowspan="1">
<em>trpC2 rpoB</em>-A478D,I528V <em>amyE</em>::<em>neo</em>; Neo<sup>r</sup> Rif<sup>r</sup>
</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> WN1663</td>
<td colspan="1" rowspan="1">Isolate from LB culture no. 3 evolved from WN1191; <em>rpoB</em>-A478D,I528V; Rif<sup>r</sup>
</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Plasmids</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1"> pDG1730</td>
<td colspan="1" rowspan="1">
<em>amyE</em> spectinomycin integration vector</td>
<td colspan="1" rowspan="1">BGSC; <a aria-describedby="B63" class="usa-link" href="#B63">63</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> pJOE8999</td>
<td colspan="1" rowspan="1">CRISPR-Cas9 system</td>
<td colspan="1" rowspan="1">BGSC; <a aria-describedby="B51" class="usa-link" href="#B51">51</a>
</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> pWN1630</td>
<td colspan="1" rowspan="1">pJOE8999 + sgRpoB + 1409<a class="usa-link" href="#T1F2"><sup><em>b</em></sup></a>
</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> pWN1633</td>
<td colspan="1" rowspan="1">pWN1630 + <em>rpoB</em>-H482N homology template</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> pWN1641</td>
<td colspan="1" rowspan="1">pWN1630 + WT <em>rpoB</em> homology template</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
<tr>
<td colspan="1" rowspan="1"> pWN1645</td>
<td colspan="1" rowspan="1">pWN1630 + <em>rpoB</em>-A478V homology template</td>
<td colspan="1" rowspan="1">This study</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="T1F1">
<sup>a</sup><p class="display-inline">BGSC, <em>Bacillus</em> Genetic Stock Center.</p>
</div>
<div class="fn" id="T1F2">
<sup>b</sup><p class="display-inline">sgRpoB, single guide RNA for RpoB.</p>
</div>
</div></section><figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">FIG 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIG 1" class="graphic" height="641" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/512c/8904044/4fd60c99a083/aem.02422-21-f001.jpg" width="766"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Determination of IC<sub>50</sub> values of Rif in <em>B. subtilis</em> strains used in this study. Results are presented as averages ± standard deviations (<em>n </em>= 3). w.t., wild type.</p></figcaption></figure></section><section id="s2-2"><h3 class="pmc_sec_title">Growth rates of <em>rpoB</em> mutants in LB cultures.</h3>
<p>In the absence of Rif selection, it has been noted previously that mutations in <em>rpoB</em> leading to Rif<sup>r</sup> often exert a fitness cost, such as a lower growth rate as measured in standard laboratory media (<a aria-describedby="B26" class="usa-link" href="#B26">26</a>; reviewed in reference <a aria-describedby="B27" class="usa-link" href="#B27">27</a>). To test this notion, we measured the growth rates of wild-type <em>B. subtilis</em> and 17 congenic <em>rpoB</em> mutants in individual liquid-LB shake flask cultures (<a class="usa-link" href="#F2">Fig. 2</a>). It was observed that 10 of the 17 <em>rpoB</em> mutant strains grew significantly more slowly than the wild type (<a class="usa-link" href="#F2">Fig. 2</a>), consistent with the notion that <em>rpoB</em> mutations can exert a fitness cost (<a aria-describedby="B26" class="usa-link" href="#B26">26</a>). However, 8 of the 17 <em>rpoB</em> mutants grew at a rate statistically indistinguishable from that of the wild-type strain (<a class="usa-link" href="#F2">Fig. 2</a>). The results indicate that not all Rif<sup>r</sup> mutants exhibit lowered fitness in LB under standard laboratory conditions, as measured by growth rate.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">FIG 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIG 2" class="graphic" height="625" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/512c/8904044/b603c4dbf4d1/aem.02422-21-f002.jpg" width="766"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Growth rates of <em>B. subtilis</em> strains cultivated in liquid LB in shake flasks with aeration. Strains carrying the indicated <em>rpoB</em> mutations were grown as described in Materials and Methods. Results are presented as averages ± standard deviations (<em>n </em>= 3). *, <em>P &lt; </em>0.05; **, <em>P &lt; </em>0.01; ***, <em>P &lt; </em>0.001; ****, <em>P &lt; </em>0.0001 (ordinary one-way ANOVA with Dunnett’s test for multiple comparison to the mean value for the wild-type strain).</p></figcaption></figure></section><section id="s2-3"><h3 class="pmc_sec_title">Relative fitness of <em>rpoB</em> alleles in LB medium.</h3>
<p>The growth rate experiments described above were performed on individual strains in monoculture, which may not reflect their true fitness in a complex competitive environment such as would likely prevail in nature. Therefore, to directly compare the fitness of <em>rpoB</em> alleles, we performed, in triplicate cultures, a multigenerational and multistrain competition experiment in which the wild-type strain and all 17 Rif<sup>r</sup>
<em>rpoB</em> mutant strains were competed simultaneously in LB (<a class="usa-link" href="#F3">Fig. 3</a>). At daily (∼7-generation) intervals, the proportion of each <em>rpoB</em> allele in the mixed population was determined by next-generation sequencing of two ∼500-bp PCR amplicons spanning either the N cluster or clusters I, II, and III of the <em>rpoB</em> gene (<a class="usa-link" href="#F3">Fig. 3A</a>). At generation zero of the experiment, each <em>rpoB</em> allele was present at ∼5 to 6% of the total population. By generation 35, the wild-type strain came to dominate all three cultures, and the proportion of most Rif<sup>r</sup> mutant alleles had dropped below the level of detection (∼0.1%) (<a class="usa-link" href="#F3">Fig. 3A</a>). Two Rif<sup>r</sup>
<em>rpoB</em> mutants persisted until the end of the experiment, the H482N and A478D variants. Although the growth rate of the H482N mutant was not significantly higher than that of the wild-type strain when grown separately in monoculture (<a class="usa-link" href="#F2">Fig. 2</a>), the mutant appeared to be less fit in a competitive environment (<a class="usa-link" href="#F3">Fig. 3A</a>). Interestingly, the A478D mutant appeared to almost disappear from the population by 14 generations, but beginning at ∼35 generations, it increased again in frequency to an average of ∼25% of the population by generation 49 (<a class="usa-link" href="#F3">Fig. 3A</a>). This anomalous result prompted us to more closely examine the allelic frequencies of the wild-type, H482N, and A478D strains in each of the three replicate mixed cultures. We observed that in two of the three replicate cultures, the A478D allele behaved like the majority of the Rif<sup>r</sup>
<em>rpoB</em> mutations, becoming reduced to a very low, barely detectable frequency (<a class="usa-link" href="#F3">Fig. 3B</a> and <a class="usa-link" href="#F3">C</a>). In contrast, in the third culture, the A478D mutant initially dropped below the limit of detection (∼0.1%), but then starting at ∼generation 35, it increased dramatically in frequency to comprise over 80% of that population by generation 49 (<a class="usa-link" href="#F3">Fig. 3D</a>). Closer examination of the PCR amplicon spanning clusters I, II, and III from the A478D mutant arising after generation 35 in the third culture revealed the appearance of a second mutation in <em>rpoB</em>, I528V. Furthermore, Sanger sequencing of the <em>rpoB</em> gene of a Rif<sup>r</sup> isolate from this culture, strain WN1663, confirmed the presence of the I528V mutation in addition to the original A478D mutation. These observations led us to suspect that during the competition experiment in the third culture, the original A478D strain had suffered a second-site mutation in <em>rpoB</em>, dramatically increasing its competitive fitness. To determine if this compensatory mutation was responsible for the observed increase in relative fitness, both the ancestral (A478D) and evolved (A478D+I528V) alleles were transferred into a clean wild-type background to create strains WN1658 and WN1662, respectively. To test their competitive fitness, the four congenic strains WN1621 (wild type), WN1658 (A478D), WN1663 (evolved A478+I528V, isolated from the generation 49 culture of the experiment whose results are shown in <a class="usa-link" href="#F3">Fig. 3D</a>), and WN1662 (A478D+I528V, reconstructed in a wild-type background) were each competed separately against the same isogenic wild-type strain (strain WN624, carrying a spectinomycin resistance [Spc<sup>r</sup>] marker) in LB medium (<a class="usa-link" href="#F4">Fig. 4</a>). Pairwise competition experiments showed that strain WN1658, the A478D <em>rpoB</em> mutant, was dramatically less fit than the wild type (selection coefficient [<em>S</em>] = −1.26 ± 0.35), whereas the evolved isolate strain WN1663 (A478D+I528V) was dramatically more fit than the wild type (<em>S</em> = +0.84 ± 0.23) (<a class="usa-link" href="#F4">Fig. 4</a>). However, simple reconstruction of the A478D+I528V <em>rpoB</em> double mutation in an otherwise wild-type background (strain WN1662) resulted in only a modest increase in fitness (<em>S</em> = −0.47 ± 0.05) compared to that seen in the single A478D mutant and failed to replicate the dramatic increase in fitness seen in the evolved strain WN1663 (<a class="usa-link" href="#F4">Fig. 4</a>). We therefore conclude that while the second-site mutation I528V in <em>rpoB</em> increased competitive fitness to a certain extent, it was not sufficient to account for the dramatic increase in fitness seen in strain 1663 arising in the third multiallelic competition culture in the experiment whose results are shown in <a class="usa-link" href="#F3">Fig. 3D</a>. The results lead us to hypothesize that strain WN1663 had sustained genomic mutation(s) in addition to I528V that also contributed to its dramatically increased fitness. These putative additional mutations could be revealed by whole-genome sequencing, which is beyond the scope of the present study. Results from the experiment whose results are shown in <a class="usa-link" href="#F3">Fig. 3</a> indicated that, when challenged in a more complex competitive environment (18 <em>rpoB</em> variants growing together for 49 generation in LB medium in the absence of Rif selection), all of the Rif<sup>r</sup> mutations in <em>rpoB</em> led to a generalized reduction in fitness. This result is somewhat at odds with the conclusions drawn from measurements of growth rates of the same strains in monoculture (<a class="usa-link" href="#F2">Fig. 2</a>), indicating that fitness cannot be predicted by growth rate in monoculture alone.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">FIG 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8904044_aem.02422-21-f003.jpg" target="_blank"><img alt="FIG 3" class="graphic zoom-in" height="597" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/512c/8904044/dbc18ef929ae/aem.02422-21-f003.jpg" width="760"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>(</strong>A) Population dynamics of wild-type (WT) and 17 <em>rpoB</em> mutant strains of <em>B. subtilis</em> in LB liquid medium grown with aeration at 37°C. Data shown are average values (<em>n </em>= 3). (B to D) Population dynamics of wild-type (WT), A478D, and H482N strains during the three replicate competition experiments from which the average values in panel A were derived.</p></figcaption></figure><figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">FIG 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8904044_aem.02422-21-f004.jpg" target="_blank"><img alt="FIG 4" class="graphic zoom-in" height="532" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/512c/8904044/9453dea3db1b/aem.02422-21-f004.jpg" width="675"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Results from pairwise competition experiments in LB medium. The indicated Rif<sup>r</sup>
<em>rpoB</em> strains were competed against the appropriate congenic wild-type strain. The strains used were WN1261 (w.t.), WN1658 (A478D), WN1663 [A478D+I528V, evolved from the culture used in the experiment whose results are shown in <a class="usa-link" href="#F3">Fig. 3D</a>; displayed in the figure as A478D,I528V(E)], and WN1662 [A478D+I528V, reconstructed in a wild-type strain by transformation; displayed in the figure as A478D,I528V(R)]. The selection coefficients (<em>S</em>) (<a aria-describedby="B54" class="usa-link" href="#B54">54</a>) shown are the average values ± standard deviations from 3 independent experiments. **, <em>P &lt; </em>0.01; ***, <em>P &lt; </em>0.001 (ordinary one-way ANOVA with Tukey’s HSD test for multiple comparisons).</p></figcaption></figure></section><section id="s2-4"><h3 class="pmc_sec_title">Relative fitness of <em>rpoB</em> alleles in SMM<sub>Asn</sub> medium.</h3>
<p>The multiallele competition experiment whose results are shown in <a class="usa-link" href="#F3">Fig. 3</a> was performed in nutrient-rich LB medium. However, in nature, <em>B. subtilis</em> is found in soils or associated with plants and their roots. Such natural environments are either subjected to “feast or famine” conditions or are chronically oligotrophic; therefore, cells would need to utilize limited quantities of rarer, more ephemeral nutrient sources. In previous work, we performed OmniLog global metabolic profiling on wild-type <em>B. subtilis</em> and 11 Rif<sup>r</sup>
<em>rpoB</em> mutants to assess how mutations in <em>rpoB</em> could affect metabolic capabilities (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>). Indeed, we found that several environmental nutrients, normally utilized poorly by the wild-type laboratory strain of <em>B. subtilis</em>, were used significantly better by certain Rif<sup>r</sup>
<em>rpoB</em> mutants. In particular, we noted that a strain carrying the S487L <em>rpoB</em> substitution utilized the amino acids <span class="font-variant-small-caps">l</span>-alanine, <span class="font-variant-small-caps">l</span>-asparagine, <span class="font-variant-small-caps">l</span>-glutamate, and <span class="font-variant-small-caps">l</span>-serine significantly better than the wild type (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>). <span class="font-variant-small-caps">l</span>-Asparagine specifically has been found to be substantially present in root exudates and plays a significant role in determining the bacterial species composition of the rhizosphere (<a aria-describedby="B28" class="usa-link" href="#B28">28</a>). Based on these observations, we were interested in measuring the relative fitness of wild-type and 17 Rif<sup>r</sup>
<em>rpoB</em> mutant strains of <em>B. subtilis</em> in a more stringent selective environment. We therefore modified Spizizen minimal medium by replacing glucose with <span class="font-variant-small-caps">l</span>-asparagine as the sole carbon source (SMM<sub>Asn</sub>; see Materials and Methods for details) and again performed a multiallele, multigenerational competition experiment (<a class="usa-link" href="#F5">Fig. 5</a>). Based on its performance in OmniLog metabolic assays (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>), we reasoned that the S487L mutant would fare better in this more stringent environment than the wild type or other Rif<sup>r</sup> mutant strains.</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">FIG 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8904044_aem.02422-21-f005.jpg" target="_blank"><img alt="FIG 5" class="graphic zoom-in" height="638" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/512c/8904044/cb5b39deb86c/aem.02422-21-f005.jpg" width="760"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(A) Population dynamics of wild-type (WT) and 17 <em>rpoB</em> mutant strains of <em>B. subtilis</em> in SMM<sub>Asn</sub> liquid medium grown with aeration at 37°C. Data shown are average values (<em>n </em>= 3). (B to D) Population dynamics of wild-type (WT), D472Y, and H482N strains during the three replicate competition experiments from which the average values in panel A were derived.</p></figcaption></figure><p>In the multiallele competition experiment conducted in SMM<sub>Asn</sub>, each of the 18 strains was initially present at 5 to 6% of the total population (<a class="usa-link" href="#F5">Fig. 5A</a>). As seen earlier in the multiallele competition performed in LB medium (<a class="usa-link" href="#F3">Fig. 3</a>), most of the strains carrying various <em>rpoB</em> mutations dropped out of the population during the course of propagation in SMM<sub>Asn</sub> (<a class="usa-link" href="#F5">Fig. 5A</a>). Interestingly, in this experiment, we observed that the strain carrying the H482N <em>rpoB</em> mutation rose to dominate the population, gaining an average frequency of ∼70% by generation 49, followed by the strain carrying the D472Y mutation (∼20%) and then the wild-type strain (∼10%) (<a class="usa-link" href="#F5">Fig. 5A</a>). These results were in stark contrast to those obtained in LB medium, where the wild-type strain dominated the population, with the H482N mutant forming a minority (<a class="usa-link" href="#F3">Fig. 3</a>). Interestingly, the strain carrying the S487L mutation, which we predicted would become the most frequent allele, did not succeed in this medium as we had expected but, rather, dropped below the threshold of detection in the population by generation 14 of the competition (<a class="usa-link" href="#F5">Fig. 5A</a>). Comparison of the results presented in <a class="usa-link" href="#F3">Fig. 3</a> with the results in <a class="usa-link" href="#F5">Fig. 5</a> demonstrated a dramatic difference in the population dynamics and strain fitness of the same <em>rpoB</em> mutants depending upon whether they were competed in the rich LB or the minimal SMM<sub>Asn</sub> environment. In addition, the lack of increased competitive fitness displayed by the S487L <em>rpoB</em> mutant competed in SMM<sub>Asn</sub> (<a class="usa-link" href="#F5">Fig. 5A</a>) brought into question whether an increased utilization of a nutrient, such as <span class="font-variant-small-caps">l</span>-asparagine, as measured by Omnilog assay (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>), is a relevant parameter to predict fitness.</p>
<p>During multiallele competition in LB medium, we noted that the A478D <em>rpoB</em> mutant rose to prominence in one of the three replicate cultures (<a class="usa-link" href="#F3">Fig. 3D</a>), presumably due to a second-site mutation(s). Accordingly, we also examined the frequencies of the wild-type, D472Y, and H482N strains in each individual SMM<sub>Asn</sub> culture (<a class="usa-link" href="#F5">Fig. 5B</a> to <a class="usa-link" href="#F5">D</a>). In this case, all three cultures behaved similarly, with the H482N strain arising to dominate each of the three cultures. This observation suggested that no second-site mutation(s) improving fitness had arisen in the SMM<sub>Asn</sub> cultures during the course of the multiallele competition experiment.</p></section><section id="s2-5"><h3 class="pmc_sec_title">Growth rates and lag times of <em>rpoB</em> mutations in SMM<sub>Asn</sub> cultures.</h3>
<p>We sought to further explore the role played by growth rate in the fitness of <em>rpoB</em> mutants by measuring the exponential growth rates during cultivation in SMM<sub>Asn</sub> of the wild-type <em>B. subtilis</em> strain and congenic mutant strains harboring the D472Y, H482N, or S487L <em>rpoB</em> allele (<a class="usa-link" href="#F6">Fig. 6</a>). In individual SMM<sub>Asn</sub> monocultures, all three Rif<sup>r</sup> mutant strains exhibited a significantly increased exponential growth rate compared to the growth rate of the wild-type strain, but there were no statistical differences among the growth rates of the Rif<sup>r</sup> mutants themselves (<a class="usa-link" href="#F6">Fig. 6A</a>). These results were distinctly different from the relative growth rates of the same strains in LB medium, in which the <em>rpoB</em> mutants either grew more slowly (S487L) or at essentially the same rate (D472Y and H482N) as the wild type (<a class="usa-link" href="#F2">Fig. 2</a>). If the exponential growth rate were the sole determinant of fitness, it would be expected that the three Rif<sup>r</sup> mutants would have risen to dominate the multiallele competition experiment performed in SMM<sub>Asn</sub>, but this was clearly not the case (<a class="usa-link" href="#F5">Fig. 5</a>).</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">FIG 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8904044_aem.02422-21-f006.jpg" target="_blank"><img alt="FIG 6" class="graphic zoom-in" height="367" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/512c/8904044/ed5f05d4cebc/aem.02422-21-f006.jpg" width="760"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Exponential growth rates (A) and lag times (B) in SMM<sub>Asn</sub> of the wild-type (w.t.) strain and congenic <em>rpoB</em> mutant strains carrying the D472Y, H482N, and S487L variants. Data are averages ± standard deviations (<em>n </em>= 3). **, <em>P &lt; </em>0.01; ***, <em>P &lt; </em>0.001 (ordinary one-way ANOVA with Dunnett’s test for multiple comparison to the mean value for the wild type).</p></figcaption></figure><p>In addition to the exponential growth rate, the ability of cells to adjust to a new environment is reflected in the lag time preceding exponential growth. In one of the few studies examining this phenomenon in <em>rpoB</em> mutants, it was observed that a <em>Mycobacterium tuberculosis</em> Rif<sup>r</sup> mutant strain carrying an H526D substitution (the equivalent of the H482D substitution in <em>B. subtilis</em>
<em>rpoB</em>) suffered significantly delayed growth following resuscitation from nutrient starvation (<a aria-describedby="B29" class="usa-link" href="#B29">29</a>). We thus compared the lag time of the wild-type strain to the lag times of congenic strains carrying the D472Y, H482N, and S487L <em>rpoB</em> variants in SMM<sub>Asn</sub>. No significant difference was observed in the lag times between the wild type and the D472Y variant; however, both the H482N and S487L variants demonstrated significant increases in lag time relative to that of the wild type (<a class="usa-link" href="#F6">Fig. 6B</a>). In fact, the H482N and S487L variants were statistically indistinguishable in their growth rates (increased over that of the wild type) and lag times (longer than that of the wild type). Taken together, the results shown in <a class="usa-link" href="#F6">Fig. 6</a> indicated that in SMM<sub>Asn</sub> medium, the growth rate and/or lag time of individual monocultures corresponded poorly with increased or decreased relative fitness in competition experiments.</p></section><section id="s2-6"><h3 class="pmc_sec_title">Pairwise competitions in LB versus SMM<sub>Asn</sub>.</h3>
<p>To further examine the discrepancies between the observed growth metrics and the results of our multiallele competition, we performed a pairwise competition for each Rif<sup>r</sup> mutant strain against the wild-type strain in both LB and SMM<sub>Asn</sub> media (<a class="usa-link" href="#F7">Fig. 7</a>). The H482N Rif<sup>r</sup> mutant strain showed the most dramatic fitness difference compared to the fitness of the wild type; it was less fit in LB (<em>S</em> = −0.047 ± 0.01) but more fit (<em>S</em> = +0.403 ± 0.063) in SMM<sub>Asn</sub> medium (<a class="usa-link" href="#F7">Fig. 7</a>). The S487L mutant was less fit than the wild type in both media, although its fitness in SMM<sub>Asn</sub> (<em>S</em> = −0.144 ± 0.005) was significantly improved over that in LB (<em>S</em> = −0.337 ± 0.01) (<a class="usa-link" href="#F7">Fig. 7</a>). The D472Y mutant was also less fit than the wild type in both media, but the difference in its fitness between the two media was just above the cutoff for significance (<em>P = </em>0.061). Measurements of the relative fitness for each mutant <em>rpoB</em> allele in pairwise competition agreed with the results of the multiallelic competitions in SMM<sub>Asn</sub>.</p>
<figure class="fig xbox font-sm" id="F7"><h4 class="obj_head">FIG 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIG 7" class="graphic" height="484" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/512c/8904044/4dda32264a47/aem.02422-21-f007.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F7/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Pairwise competition experiments. Rif<sup>r</sup> strains carrying the variant D472Y (strain WN1625), H482N (strain WN1637), or S487L (strain WN761) were each competed against congenic wild-type strain WN1651 in LB or SMM<sub>Asn</sub> medium for 21 generations, and fitness coefficients (<em>S</em>) were calculated as described previously (<a aria-describedby="B54" class="usa-link" href="#B54">54</a>). Data are average values ± standard deviations (<em>n </em>= 3). ***, <em>P &lt; </em>0.001; ****, <em>P &lt; </em>0.0001; ns, not significantly different (two-way ANOVA with Tukey’s HSD test for multiple comparisons).</p></figcaption></figure><p>Previous work in our laboratory has shown differing spectra of mutations in the <em>rpoB</em> gene across several different environment (<a aria-describedby="B12" class="usa-link" href="#B12">12</a>, <a aria-describedby="B13" class="usa-link" href="#B13">13</a>, <a aria-describedby="B16" class="usa-link" href="#B16">16</a><a aria-describedby="B17" class="usa-link" href="#B17">–</a><a aria-describedby="B18" class="usa-link" href="#B18">18</a>). The reproducibility of these changes in the spectra of mutations indicates that the environment somehow dictates these changes. Multiple studies have found that mutations in the <em>rpoB</em> gene that confer rifampin resistance can provide a fitness advantage or increased growth rate in the environment from which they were isolated. Several examples illustrate the repeated occurrence of <em>rpoB</em> mutations in adaptive laboratory evolution of <em>Escherichia coli</em>, wherein these mutations significantly increased the fitness of the organism in different environments, including optimized growth on various minimal media and increased fitness at elevated temperatures (<a aria-describedby="B30" class="usa-link" href="#B30">30</a><a aria-describedby="B31" class="usa-link" href="#B31">–</a><a aria-describedby="B32" class="usa-link" href="#B32">32</a>).</p>
<p>Our multiallele, multigenerational competition experiments (<a class="usa-link" href="#F3">Fig. 3</a> and <a class="usa-link" href="#F5">5</a>) showed that the wild-type <em>rpoB</em> allele was most fit in nutrient-rich LB medium, while the H482N variant was most fit in populations cultivated in minimal SMM<sub>Asn</sub> medium. The different <em>rpoB</em> alleles and the corresponding changes in global gene expression associated with them likely explain the differences in relative fitness depending upon the growth environment. Numerous studies suggest changes in global gene expression to be the cause of changes in fitness observed with mutations in <em>rpoB</em> (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>, <a aria-describedby="B33" class="usa-link" href="#B33">33</a><a aria-describedby="B34" class="usa-link" href="#B34">–</a><a aria-describedby="B37" class="usa-link" href="#B37">37</a>), although one study examining the system-wide impacts of mutations in the <em>Pseudomonas aeruginosa</em>
<em>rpoB</em> gene found few differences in global gene expression and attributed mutant phenotypes to increased expression of RNAP (<a aria-describedby="B38" class="usa-link" href="#B38">38</a>). Ultimately, a better understanding of the specific mechanisms behind these differences in fitness will require an examination of global gene expression of these different <em>rpoB</em> alleles in SMM<sub>Asn</sub>.</p>
<p>Which changes in gene expression are specifically responsible for the success of the H482N mutant can only be speculated upon at present, as this particular allele is new to our collection and not yet very well characterized. However, the lack of fitness of the S487L mutant in SMM<sub>Asn</sub> may be a bit easier to explain. Previous work studying the <em>rpoB</em>-S487L allele in <em>B. subtilis</em> revealed that this particular allele autoinduced the expression of genes responsible for the production of the antibiotic 3,3′-neotrehalosadiamine (NTD), which is not produced by the wild-type strain under the same conditions (<a aria-describedby="B39" class="usa-link" href="#B39">39</a>). This autoinduction of NTD production might place a significant metabolic burden on the organism and may be responsible for the lack of relative fitness observed in the <em>rpoB</em>-S487L allele.</p>
<p>The discrepancies between fitness and exponential growth rate reveal that, while fairly simple to determine, growth rate is a poor proxy for relative fitness, as a similar study has found (<a aria-describedby="B40" class="usa-link" href="#B40">40</a>). Simply measuring the exponential growth rate fails to account for differences in fitness during stationary and lag phase, in addition to potential antagonistic or cooperative interactions between the strains when competing for the same resources, such as dramatic acidification of the medium (<a aria-describedby="B41" class="usa-link" href="#B41">41</a>). Our study highlights the danger of conflating exponential growth rates or results from simplified metabolic assays (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>) with relative fitness. Three Rif<sup>r</sup>
<em>rpoB</em> variants exhibited an increased growth rate relative to the growth rate of the wild-type strain in SMM<sub>Asn</sub> (<a class="usa-link" href="#F6">Fig. 6</a>); however, only one of those variants (H482N) proved to be more fit than the wild-type strain in the same medium (<a class="usa-link" href="#F7">Fig. 7</a>). There are many potential explanations for this discrepancy that could be controlled at the transcriptional level, such as increased probability of differentiation in the stationary phase (<a aria-describedby="B42" class="usa-link" href="#B42">42</a>) or shaping of the environment by an evolved variant in a manner that preferentially benefits itself (<a aria-describedby="B41" class="usa-link" href="#B41">41</a>, <a aria-describedby="B43" class="usa-link" href="#B43">43</a>).</p>
<p>Unlike other results from this study, the increased relative fitness of the evolved strain WN1663 cannot be explained solely by a compensatory mutation in <em>rpoB</em> (<a class="usa-link" href="#F4">Fig. 4</a>). Our data indicate that other changes within the <em>B. subtilis</em> genome must account for the observed sweep of the population by WN1663. The increase in fitness of the A478D+I528V double mutant relative to the fitness of the A478D single mutant is consistent with other studies that have reported the ability of compensatory mutations in <em>rpoB</em> and <em>rpoC</em> to restore fitness defects (<a aria-describedby="B44" class="usa-link" href="#B44">44</a><a aria-describedby="B45" class="usa-link" href="#B45">–</a><a aria-describedby="B46" class="usa-link" href="#B46">46</a>). However, these studies did not find increases in fitness relative to that of the wild-type allele. In summary, this study illustrates the fitness impacts of <em>rpoB</em> alleles in different environments, highlights the deficiencies of growth or metabolic metrics of monocultures as a sole measure of relative fitness, and demonstrates the role of compensatory mutations in <em>rpoB</em> to recover fitness cost.</p></section></section><section id="s3"><h2 class="pmc_sec_title">MATERIALS AND METHODS</h2>
<section id="s3-1"><h3 class="pmc_sec_title">Bacterial strains, media, plasmids, and growth conditions.</h3>
<p>All <em>B. subtilis</em> strains and plasmids used in this study are listed in <a class="usa-link" href="#T1">Table 1</a>. The strains were constructed by DNA-mediated transformation of competent cells with purified genomic DNA as previously described (<a aria-describedby="B47" class="usa-link" href="#B47">47</a>). The complex media used were Miller LB medium (<a aria-describedby="B48" class="usa-link" href="#B48">48</a>) and Trypticase-soy-yeast extract (TSY) medium (<a aria-describedby="B49" class="usa-link" href="#B49">49</a>). The minimal medium used was Spizizen minimal medium (<a aria-describedby="B50" class="usa-link" href="#B50">50</a>) containing the auxotrophic requirements tryptophan and phenylalanine (50 μg/mL each) and asparagine (95 mM) in place of glucose as the carbon source (SMM<sub>Asn</sub>). Media were solidified with agar (15 g/L) as needed. The antibiotics used (final concentration) were chloramphenicol (10 μg/mL), rifampin (5 μg/mL), kanamycin (5 μg/mL), neomycin (5 μg/mL), or spectinomycin (100 μg/mL). Plasmids for CRISPR-Cas9 introduction of point mutations were constructed and used as previously described (<a aria-describedby="B51" class="usa-link" href="#B51">51</a>). Site-directed mutagenesis was conducted via inverse PCR as previously described (<a aria-describedby="B52" class="usa-link" href="#B52">52</a>) to introduce the point mutation A478V into an <em>rpoB</em> template. All mutations were confirmed by Sanger sequencing (Genewiz LLC) of PCR amplicons.</p></section><section id="s3-2"><h3 class="pmc_sec_title">Rif IC<sub>50</sub> determination.</h3>
<p>For quantification of Rif<sup>r</sup>, an overnight culture of each strain was grown in liquid TSY without antibiotics. One microliter of overnight culture was inoculated into 100 μl of liquid TSY medium without Rif or containing Rif at a 2-fold range of concentrations from 4 to 256 μg/mL. Cells were grown in 96-well microtiter plates sealed with gas-permeable membranes (Breathe-Easy; Sigma-Aldrich). Growth at 37°C was measured by the optical density at 620 nm (OD<sub>620</sub>) at 15-minute intervals in a temperature-controlled microplate absorbance spectrophotometer (model ELx808; BioTek). The resulting growth data from triplicate experiments were then used to determine the 50% inhibitory concentration (IC<sub>50</sub>).</p></section><section id="s3-3"><h3 class="pmc_sec_title">Growth rate and lag time determinations.</h3>
<p>Liquid cultures were propagated in 125-mL sidearm (Klett) flasks in 10 mL of LB at 37°C in a rotary shaking bath with vigorous aeration. Growth was measured at 15-minute intervals by the OD<sub>660</sub> using a Klett-Summerson colorimeter fitted with the no. 66 (red, 660-nm) filter. Growth rates were calculated from the exponential portion of each growth curve. Lag times were determined from the point of intersection of the lag and exponential portions of each growth curve. Growth rates and lag times were determined from triplicate cultures. Data sets were tested for normality using the Shapiro-Wilk test online calculator (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.statskingdom.com/shapiro-wilk-test-calculator.html" rel="noopener noreferrer" target="_blank">https://www.statskingdom.com/shapiro-wilk-test-calculator.html</a>), and all data sets were found to be normally distributed. Statistical parameters and tests of significance (analysis of variance [ANOVA]) were computed using KaleidaGraph version 4.5.2 (Synergy Software, Reading, PA, USA).</p></section><section id="s3-4"><h3 class="pmc_sec_title">Competition assays.</h3>
<p>Individual overnight cultures of the wild-type strain and all congenic Rif<sup>r</sup> strains were inoculated into fresh individual 2-mL LB cultures. Upon entering stationary phase, the OD<sub>660</sub> of each culture was measured and a volume corresponding to ∼10<sup>8</sup> cells of each strain was combined into a single mixed culture, which was then diluted 1:100 into 10 mL of either LB or SMM<sub>Asn</sub> in 125-mL Erlenmeyer flasks. Cultures were grown at 37°C in a temperature-controlled rotary shaking bath at 200 rpm. Every 24 h, cultures were diluted 1:100 into fresh medium. Under these conditions, cultures progress through ∼7 generations per day (<a aria-describedby="B53" class="usa-link" href="#B53">53</a>). For multiallelic competition experiments, every 24 h, from each culture, a 1-mL aliquot was pelleted for chromosomal DNA extraction; 0.8 mL was stored in 25% (vol/vol) glycerol at −70°C, and 0.1 mL was diluted serially 10-fold in phosphate-buffered saline (PBS), plated on LB with chloramphenicol (Cm), and incubated overnight at 37°C for total viable counts. For two-allele competition experiments, every 24 h, 0.1 mL of each culture was serially diluted and plated on two different LB plates, each supplemented with a different antibiotic. These LB plates were then incubated overnight at 37°C to determine the viable counts of each strain. Relative fitness was quantified with a previously described selection coefficient model (<a aria-describedby="B54" class="usa-link" href="#B54">54</a>).</p></section><section id="s3-5"><h3 class="pmc_sec_title">Next-generation library preparation, sequencing, and data analysis.</h3>
<p>From purified chromosomal DNA, two ∼500-bp amplicons of the <em>rpoB</em> gene, one spanning the N cluster and one spanning clusters I, II, and III, were amplified by PCR using Q5 high-fidelity DNA polymerase (New England Biolabs) and the corresponding primer pairs (<a class="usa-link" href="#T2">Table 2</a>). PCR products were purified (Qiagen PCR purification kit) and quantified by Qubit fluorometry, and equal amounts were combined for library preparation. Next-generation sequencing libraries were prepared using an Illumina Nextera XT DNA library preparation kit with some modifications. Due to the high levels of NaOH in the Nextera XT bead-based normalization process, the native normalization process is not compatible with the iSeq100. In place of the bead-based normalization process, normalization was performed manually by capillary electrophoresis using the Agilent Bioanalyzer 2100 expert software to calculate the molar amount of DNA fragments in each library. Equivalent amounts of DNA fragments from 24 individual libraries were pooled at a final DNA fragment concentration of 4 nM. Pooled libraries were diluted to 50 pM prior to loading into an Illumina i1 reagent box and sequenced on an Illumina iSeq100 next-generation-sequencing platform.</p>
<section class="tw xbox font-sm" id="T2"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Oligonucleotide primers used in this study</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1"/>
<col span="1"/>
<col span="1"/>
</colgroup>
<thead><tr>
<th colspan="1" rowspan="1">Primer</th>
<th colspan="1" rowspan="1">Sequence (5′–3′)</th>
<th colspan="1" rowspan="1">Purpose<a class="usa-link" href="#T2F4"><sup><em>d</em></sup></a>
</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">I-cluster-F</td>
<td colspan="1" rowspan="1">
<em>CGGAGTAGGCGACACAGATG</em>
</td>
<td colspan="1" rowspan="1">Amplification of clusters I, II, and III for NGS</td>
</tr>
<tr>
<td colspan="1" rowspan="1">I-cluster-R</td>
<td colspan="1" rowspan="1">
<em>ATTCTGCCCGTTACCTTCCC</em>
</td>
<td colspan="1" rowspan="1">Amplification of clusters I, II, and III for NGS</td>
</tr>
<tr>
<td colspan="1" rowspan="1">N-cluster-F</td>
<td colspan="1" rowspan="1">
<em>TTCAGTATGGACGACACCGC</em>
</td>
<td colspan="1" rowspan="1">Amplification of N cluster for NGS</td>
</tr>
<tr>
<td colspan="1" rowspan="1">N-cluster-R</td>
<td colspan="1" rowspan="1">
<em>TTACGTGTGCGATCAATGCG</em>
</td>
<td colspan="1" rowspan="1">Amplification of N cluster for NGS</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Q5SDM_A478V_F</td>
<td colspan="1" rowspan="1">
<em>AACCCGCTTG<strong><span class="text-underline">T</span></strong>TGAATTAACG</em>
<a class="usa-link" href="#T2F1">
<sup>
<em>a</em>
</sup>
</a>
</td>
<td colspan="1" rowspan="1">Site-directed mutagenesis for <em>rpoB</em>-A478V</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Q5SDM_A478V_R</td>
<td colspan="1" rowspan="1">
<em>CGTCTGATCCATGAATTG</em>
</td>
<td colspan="1" rowspan="1">Site-directed mutagenesis for <em>rpoB</em>-A478V</td>
</tr>
<tr>
<td colspan="1" rowspan="1">rpoB+1409_sgRNA+</td>
<td colspan="1" rowspan="1">
<em>AAACcttgctgaattaacgcacaa</em>
<a class="usa-link" href="#T2F2">
<sup>
<em>b</em>
</sup>
</a>
</td>
<td colspan="1" rowspan="1">sgRNA oligonucleotide</td>
</tr>
<tr>
<td colspan="1" rowspan="1">rpoB+1409_sgRNA-</td>
<td colspan="1" rowspan="1">
<em>TACGttgtgcgttaattcagcaag</em>
<a class="usa-link" href="#T2F2">
<sup>
<em>b</em>
</sup>
</a>
</td>
<td colspan="1" rowspan="1">sgRNA oligonucleotide</td>
</tr>
<tr>
<td colspan="1" rowspan="1">rpoB-903F_HT</td>
<td colspan="1" rowspan="1">
<em>AAGGCCAACGAGGCCggcgagctttttctttgcca</em>
<a class="usa-link" href="#T2F3">
<sup>
<em>c</em>
</sup>
</a>
</td>
<td colspan="1" rowspan="1">
<em>rpoB</em> homology template</td>
</tr>
<tr>
<td colspan="1" rowspan="1">rpoC+181R_HT</td>
<td colspan="1" rowspan="1">
<em>AAGGCCTTATTGGCCtcgcttgtacttcccgcaat</em>
<em>
<sup>c</sup>
</em>
</td>
<td colspan="1" rowspan="1">
<em>rpoB</em> homology template</td>
</tr>
<tr>
<td colspan="1" rowspan="1">rpoB-24F</td>
<td colspan="1" rowspan="1">
<em>CGCATGATTTGAGGGG</em>
</td>
<td colspan="1" rowspan="1">N-cluster amplification and sequencing</td>
</tr>
<tr>
<td colspan="1" rowspan="1">rpoB+737R</td>
<td colspan="1" rowspan="1">
<em>GGCGGCTCTCCAGG</em>
</td>
<td colspan="1" rowspan="1">N-cluster amplification</td>
</tr>
<tr>
<td colspan="1" rowspan="1">rpoB+1319F</td>
<td colspan="1" rowspan="1">
<em>CGAATACAATTACGCCTCAGC</em>
</td>
<td colspan="1" rowspan="1">Cluster I, II, and III amplification and sequencing</td>
</tr>
<tr>
<td colspan="1" rowspan="1">rpoB+2000R</td>
<td colspan="1" rowspan="1">
<em>CCTGATACGTATTCCATACC</em>
</td>
<td colspan="1" rowspan="1">Cluster I, II, and III amplification</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="T2F1">
<sup>a</sup><p class="display-inline">Bold underlined letter represents mutation being introduced.</p>
</div>
<div class="fn" id="T2F2">
<sup>b</sup><p class="display-inline">Uppercase letters represent bases complementary to BsaI cut site on pJOE8999 (<a aria-describedby="B51" class="usa-link" href="#B51">51</a>); lowercase letters represent wild-type <em>rpoB</em> sequence.</p>
</div>
<div class="fn" id="T2F3">
<sup>c</sup><p class="display-inline">Uppercase letters represent SfiI recognition and cut site complementary to pJOE8999 (<a aria-describedby="B51" class="usa-link" href="#B51">51</a>); lowercase letters represent wild-type genomic sequence.</p>
</div>
<div class="fn" id="T2F4">
<sup>d</sup><p class="display-inline">NGS, next-generation sequencing; sgRNA, single guide RNA.</p>
</div>
</div></section><p>Raw data from the iSeq100 was demultiplexed and converted to fastq files using Illumina bcl2fastq software. Quality control of reads was performed with FastQC (<a aria-describedby="B55" class="usa-link" href="#B55">55</a>). Quality reads were mapped to the <em>Bacillus subtilis</em> strain 168 reference genome (GenBank accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/AL009126.3" rel="noopener noreferrer" target="_blank">AL009126.3</a>) using Bowtie 2 (<a aria-describedby="B56" class="usa-link" href="#B56">56</a>). From mapped reads, LoFreq (<a aria-describedby="B57" class="usa-link" href="#B57">57</a>) was used to call single-nucleotide polymorphisms and small deletions in the <em>rpoB</em> gene. Preliminary calibration experiments performed with known proportions of mutant <em>rpoB</em> alleles established that the lower limit of detection was ∼0.1% (data not shown).</p></section><section id="s3-6"><h3 class="pmc_sec_title">Statistical analysis.</h3>
<p>All data sets were tested for normality using the Shapiro-Wilk method (<a aria-describedby="B58" class="usa-link" href="#B58">58</a>). Data sets passing the normality test were analyzed for differences by ANOVA with Tukey’s honestly significant difference (HSD) test (<a aria-describedby="B59" class="usa-link" href="#B59">59</a>) or Dunnett’s test for multiple comparison analysis (<a aria-describedby="B60" class="usa-link" href="#B60">60</a>). All statistical analyses were performed with the statistical graphing software Prism (GraphPad Software). Differences with a <em>P</em> value of &lt;0.05 were considered statistically significant.</p></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank the Department of Microbiology and Cell Science at the University of Florida for use of the iSeq100 instrument.</p>
<p>This work was supported by grants from the NASA Space Biology (grant number NNX14AT38G) and USDA-Hatch (grants number FLA-MCS-005500 and FLA-MCS-006066) programs to W.L.N.</p></section><section class="contrib-info" id="_ci93_" lang="en"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Wayne L. Nicholson, Email: WLN@ufl.edu.</p>
<p>Maia Kivisaar, University of Tartu.</p></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>Wehrli W, Knüsel F, Schmid K, Staehelin M. 1968. Interaction of rifamycin with bacterial RNA polymerase. Proc Natl Acad Sci USA
61:667–673. 10.1073/pnas.61.2.667.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.61.2.667" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC225211/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/4879400/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Interaction%20of%20rifamycin%20with%20bacterial%20RNA%20polymerase&amp;volume=61&amp;publication_year=1968&amp;pages=667-673&amp;pmid=4879400&amp;doi=10.1073/pnas.61.2.667&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>Tupin A, Gualtieri M, Roquet-Banères F, Morichaud Z, Brodolin K, Leonetti J-P. 2010. Resistance to rifampicin: at the crossroads between ecological, genomic and medical concerns. Int J Antimicrob Agents
35:519–523. 10.1016/j.ijantimicag.2009.12.017.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ijantimicag.2009.12.017" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20185278/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Antimicrob%20Agents&amp;title=Resistance%20to%20rifampicin:%20at%20the%20crossroads%20between%20ecological,%20genomic%20and%20medical%20concerns&amp;volume=35&amp;publication_year=2010&amp;pages=519-523&amp;pmid=20185278&amp;doi=10.1016/j.ijantimicag.2009.12.017&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>Jin DJ, Gross CA. 1988. Mapping and sequencing of mutations in the <em>Escherichia coli rpoB</em> gene that lead to rifampicin resistance. J Mol Biol
202:45–58. 10.1016/0022-2836(88)90517-7.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0022-2836(88)90517-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/3050121/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Mapping%20and%20sequencing%20of%20mutations%20in%20the%20Escherichia%20coli%20rpoB%20gene%20that%20lead%20to%20rifampicin%20resistance&amp;volume=202&amp;publication_year=1988&amp;pages=45-58&amp;pmid=3050121&amp;doi=10.1016/0022-2836(88)90517-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>Campbell EA, Korzheva N, Mustaev A, Murakami K, Nair S, Goldfarb A, Darst SA. 2001. Structural mechanism for rifampicin inhibition of bacterial RNA polymerase. Cell
104:901–912. 10.1016/s0092-8674(01)00286-0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s0092-8674(01)00286-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11290327/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Structural%20mechanism%20for%20rifampicin%20inhibition%20of%20bacterial%20RNA%20polymerase&amp;volume=104&amp;publication_year=2001&amp;pages=901-912&amp;pmid=11290327&amp;doi=10.1016/s0092-8674(01)00286-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>Naryshkina T, Mustaev A, Darst SA, Severinov K. 2001. The beta′ subunit of <em>Escherichia coli</em> RNA polymerase is not required for interaction with initiating nucleotide but is necessary for interaction with rifampicin. J Biol Chem
276:13308–13313. 10.1074/jbc.M011041200.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M011041200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11278820/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=The%20beta%E2%80%B2%20subunit%20of%20Escherichia%20coli%20RNA%20polymerase%20is%20not%20required%20for%20interaction%20with%20initiating%20nucleotide%20but%20is%20necessary%20for%20interaction%20with%20rifampicin&amp;volume=276&amp;publication_year=2001&amp;pages=13308-13313&amp;pmid=11278820&amp;doi=10.1074/jbc.M011041200&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>Severinov K, Soushko M, Goldfarb A, Nikiforov V. 1993. Rifampicin region revisited. New rifampicin-resistant and streptolydigin-resistant mutants in the beta subunit of <em>Escherichia coli</em> RNA polymerase. J Biol Chem
268:14820–14825. 10.1016/S0021-9258(18)82407-3.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0021-9258(18)82407-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8325861/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Rifampicin%20region%20revisited.%20New%20rifampicin-resistant%20and%20streptolydigin-resistant%20mutants%20in%20the%20beta%20subunit%20of%20Escherichia%20coli%20RNA%20polymerase&amp;volume=268&amp;publication_year=1993&amp;pages=14820-14825&amp;pmid=8325861&amp;doi=10.1016/S0021-9258(18)82407-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>Maughan H, Galeano B, Nicholson WL. 2004. Novel <em>rpoB</em> mutations conferring rifampin resistance on <em>Bacillus subtilis</em>: global effects on growth, competence, sporulation, and germination. J Bacteriol
186:2481–2486. 10.1128/JB.186.8.2481-2486.2004.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JB.186.8.2481-2486.2004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC412138/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15060052/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Novel%20rpoB%20mutations%20conferring%20rifampin%20resistance%20on%20Bacillus%20subtilis:%20global%20effects%20on%20growth,%20competence,%20sporulation,%20and%20germination&amp;volume=186&amp;publication_year=2004&amp;pages=2481-2486&amp;pmid=15060052&amp;doi=10.1128/JB.186.8.2481-2486.2004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>Perkins AE, Nicholson WL. 2008. Uncovering new metabolic capabilities of <em>Bacillus subtilis</em> using phenotype profiling of rifampin-resistant <em>rpoB</em> mutants. J Bacteriol
190:807–814. 10.1128/JB.00901-07.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JB.00901-07" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2223569/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17644585/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Uncovering%20new%20metabolic%20capabilities%20of%20Bacillus%20subtilis%20using%20phenotype%20profiling%20of%20rifampin-resistant%20rpoB%20mutants&amp;volume=190&amp;publication_year=2008&amp;pages=807-814&amp;pmid=17644585&amp;doi=10.1128/JB.00901-07&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>Moeller R, Vlašić I, Reitz G, Nicholson WL. 2012. Role of altered <em>rpoB</em> alleles in <em>Bacillus subtilis</em> sporulation and spore resistance to heat, hydrogen peroxide, formaldehyde, and glutaraldehyde. Arch Microbiol
194:759–767. 10.1007/s00203-012-0811-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00203-012-0811-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22484477/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Arch%20Microbiol&amp;title=Role%20of%20altered%20rpoB%20alleles%20in%20Bacillus%20subtilis%20sporulation%20and%20spore%20resistance%20to%20heat,%20hydrogen%20peroxide,%20formaldehyde,%20and%20glutaraldehyde&amp;volume=194&amp;publication_year=2012&amp;pages=759-767&amp;pmid=22484477&amp;doi=10.1007/s00203-012-0811-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>Leighton TJ. 1973. An RNA polymerase mutation causing temperature-sensitive sporulation in <em>Bacillus subtilis</em>. Proc Natl Acad Sci USA
70:1179–1183. 10.1073/pnas.70.4.1179.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.70.4.1179" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC433452/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/4197929/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=An%20RNA%20polymerase%20mutation%20causing%20temperature-sensitive%20sporulation%20in%20Bacillus%20subtilis&amp;volume=70&amp;publication_year=1973&amp;pages=1179-1183&amp;pmid=4197929&amp;doi=10.1073/pnas.70.4.1179&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>Friedberg EC, Walker GC, Siede W. 2005. DNA repair and mutagenesis.
ASM Press, Washington, DC.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=DNA%20repair%20and%20mutagenesis.&amp;publication_year=2005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>Nicholson WL, Maughan H. 2002. The spectrum of spontaneous rifampin resistance mutations in the <em>rpoB</em> gene of <em>Bacillus subtilis</em> 168 spores differs from that of vegetative cells and resembles that of <em>Mycobacterium tuberculosis</em>. J Bacteriol
184:4936–4940. 10.1128/JB.184.17.4936-4940.2002.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JB.184.17.4936-4940.2002" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC135274/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12169622/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=The%20spectrum%20of%20spontaneous%20rifampin%20resistance%20mutations%20in%20the%20rpoB%20gene%20of%20Bacillus%20subtilis%20168%20spores%20differs%20from%20that%20of%20vegetative%20cells%20and%20resembles%20that%20of%20Mycobacterium%20tuberculosis&amp;volume=184&amp;publication_year=2002&amp;pages=4936-4940&amp;pmid=12169622&amp;doi=10.1128/JB.184.17.4936-4940.2002&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>Perkins AE, Schuerger AC, Nicholson WL. 2008. Isolation of <em>rpoB</em> mutations causing rifampicin resistance in <em>Bacillus subtilis</em> spores exposed to simulated Martian surface conditions. Astrobiology
8:1159–1167. 10.1089/ast.2007.0224.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2007.0224" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19191541/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Isolation%20of%20rpoB%20mutations%20causing%20rifampicin%20resistance%20in%20Bacillus%20subtilis%20spores%20exposed%20to%20simulated%20Martian%20surface%20conditions&amp;volume=8&amp;publication_year=2008&amp;pages=1159-1167&amp;pmid=19191541&amp;doi=10.1089/ast.2007.0224&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14">
<span class="label">14.</span><cite>Moeller R, Reitz G, Berger T, Okayasu R, Nicholson WL, Horneck G. 2010. Astrobiological aspects of the mutagenesis of cosmic radiation on bacterial spores. Astrobiology
10:509–521. 10.1089/ast.2009.0429.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2009.0429" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20624059/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Astrobiological%20aspects%20of%20the%20mutagenesis%20of%20cosmic%20radiation%20on%20bacterial%20spores&amp;volume=10&amp;publication_year=2010&amp;pages=509-521&amp;pmid=20624059&amp;doi=10.1089/ast.2009.0429&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15">
<span class="label">15.</span><cite>Moeller R, Reitz G, Nicholson WL, Horneck G, the PROTECT team. 2012. Mutagenesis in bacterial spores exposed to space and simulated Martian conditions: data from the EXPOSE-E spaceflight experiment PROTECT. Astrobiology
12:457–468. 10.1089/ast.2011.0739.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2011.0739" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22680692/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Mutagenesis%20in%20bacterial%20spores%20exposed%20to%20space%20and%20simulated%20Martian%20conditions:%20data%20from%20the%20EXPOSE-E%20spaceflight%20experiment%20PROTECT&amp;volume=12&amp;publication_year=2012&amp;pages=457-468&amp;pmid=22680692&amp;doi=10.1089/ast.2011.0739&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16">
<span class="label">16.</span><cite>Nicholson WL, Park R. 2015. Anaerobic growth of <em>Bacillus subtilis</em> alters the spectrum of spontaneous mutations in the <em>rpoB</em> gene leading to rifampicin resistance. FEMS Microbiol Lett
362:fnv213. 10.1093/femsle/fnv213.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/femsle/fnv213" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26538577/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol%20Lett&amp;title=Anaerobic%20growth%20of%20Bacillus%20subtilis%20alters%20the%20spectrum%20of%20spontaneous%20mutations%20in%20the%20rpoB%20gene%20leading%20to%20rifampicin%20resistance&amp;volume=362&amp;publication_year=2015&amp;pages=fnv213&amp;pmid=26538577&amp;doi=10.1093/femsle/fnv213&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17">
<span class="label">17.</span><cite>Fajardo-Cavazos P, Leehan JD, Nicholson WL. 2018. Alterations in the spectrum of spontaneous rifampicin-resistance mutations in the <em>Bacillus subtilis rpoB</em> gene after cultivation in the human spaceflight environment. Front Microbiol
9:192. 10.3389/fmicb.2018.00192.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2018.00192" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5817088/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29491852/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Alterations%20in%20the%20spectrum%20of%20spontaneous%20rifampicin-resistance%20mutations%20in%20the%20Bacillus%20subtilis%20rpoB%20gene%20after%20cultivation%20in%20the%20human%20spaceflight%20environment&amp;volume=9&amp;publication_year=2018&amp;pages=192&amp;pmid=29491852&amp;doi=10.3389/fmicb.2018.00192&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18">
<span class="label">18.</span><cite>Leehan JD, Nicholson WL. 2021. The spectrum of spontaneous rifampin resistance mutations in the <em>Bacillus subtilis rpoB</em> gene depends on the growth environment. Appl Environ Microbiol
87:e01237-21. 10.1128/AEM.01237-21.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.01237-21" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8552901/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34495706/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=The%20spectrum%20of%20spontaneous%20rifampin%20resistance%20mutations%20in%20the%20Bacillus%20subtilis%20rpoB%20gene%20depends%20on%20the%20growth%20environment&amp;volume=87&amp;publication_year=2021&amp;pages=e01237-21&amp;pmid=34495706&amp;doi=10.1128/AEM.01237-21&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19">
<span class="label">19.</span><cite>Munakata N, Takahashi K, Hieda K, Panitz C, Horneck G. 2004. Mutagenesis of <em>Bacillus subtilis</em> spores exposed to simulated space environment, abstr 04-A-00898. Abstr 35th COSPAR Sci Assembly, Paris, France.</cite>
</li>
<li id="B20">
<span class="label">20.</span><cite>Wrande M, Roth JR, Hughes D. 2008. Accumulation of mutants in “aging” bacterial colonies is due to growth under selection, not stress-induced mutagenesis. Proc Natl Acad Sci USA
105:11863–11868. 10.1073/pnas.0804739105.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0804739105" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2515621/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18701713/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Accumulation%20of%20mutants%20in%20%E2%80%9Caging%E2%80%9D%20bacterial%20colonies%20is%20due%20to%20growth%20under%20selection,%20not%20stress-induced%20mutagenesis&amp;volume=105&amp;publication_year=2008&amp;pages=11863-11868&amp;pmid=18701713&amp;doi=10.1073/pnas.0804739105&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21">
<span class="label">21.</span><cite>Maudsdotter L, Ushijima Y, Morikawa K. 2019. Fitness of spontaneous rifampicin-resistant <em>Staphylococcus aureus</em> isolates in a biofilm environment. Front Microbiol
10:988. 10.3389/fmicb.2019.00988.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2019.00988" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6514104/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31134027/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Fitness%20of%20spontaneous%20rifampicin-resistant%20Staphylococcus%20aureus%20isolates%20in%20a%20biofilm%20environment&amp;volume=10&amp;publication_year=2019&amp;pages=988&amp;pmid=31134027&amp;doi=10.3389/fmicb.2019.00988&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22">
<span class="label">22.</span><cite>Lee DJ, Minchin SD, Busby SJW. 2012. Activating transcription in bacteria. Annu Rev Microbiol
66:125–152. 10.1146/annurev-micro-092611-150012.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev-micro-092611-150012" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22726217/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Microbiol&amp;title=Activating%20transcription%20in%20bacteria&amp;volume=66&amp;publication_year=2012&amp;pages=125-152&amp;pmid=22726217&amp;doi=10.1146/annurev-micro-092611-150012&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23">
<span class="label">23.</span><cite>Pruss GJ, Drlica K. 1989. DNA supercoiling and prokaryotic transcription. Cell
56:521–523. 10.1016/0092-8674(89)90574-6.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0092-8674(89)90574-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/2645054/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=DNA%20supercoiling%20and%20prokaryotic%20transcription&amp;volume=56&amp;publication_year=1989&amp;pages=521-523&amp;pmid=2645054&amp;doi=10.1016/0092-8674(89)90574-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24">
<span class="label">24.</span><cite>Dorman CJ, Dorman MJ. 2016. DNA supercoiling is a fundamental regulatory principle in the control of bacterial gene expression. Biophys Rev
8:209–220. 10.1007/s12551-016-0205-y.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s12551-016-0205-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5425793/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28510224/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biophys%20Rev&amp;title=DNA%20supercoiling%20is%20a%20fundamental%20regulatory%20principle%20in%20the%20control%20of%20bacterial%20gene%20expression&amp;volume=8&amp;publication_year=2016&amp;pages=209-220&amp;pmid=28510224&amp;doi=10.1007/s12551-016-0205-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25">
<span class="label">25.</span><cite>Christaki E, Marcou M, Tofarides A. 2020. Antimicrobial resistance in bacteria: mechanisms, evolution, and persistence. J Mol Evol
88:26–40. 10.1007/s00239-019-09914-3.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00239-019-09914-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31659373/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Evol&amp;title=Antimicrobial%20resistance%20in%20bacteria:%20mechanisms,%20evolution,%20and%20persistence&amp;volume=88&amp;publication_year=2020&amp;pages=26-40&amp;pmid=31659373&amp;doi=10.1007/s00239-019-09914-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26">
<span class="label">26.</span><cite>Cohan FM, King EC, Zawadzki P. 1994. Amelioration of the deleterious pleiotropic effects of an adaptive mutation in <em>Bacillus subtilis</em>. Evolution
48:81–95. 10.1111/j.1558-5646.1994.tb01296.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1558-5646.1994.tb01296.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28567791/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Amelioration%20of%20the%20deleterious%20pleiotropic%20effects%20of%20an%20adaptive%20mutation%20in%20Bacillus%20subtilis&amp;volume=48&amp;publication_year=1994&amp;pages=81-95&amp;pmid=28567791&amp;doi=10.1111/j.1558-5646.1994.tb01296.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27">
<span class="label">27.</span><cite>Andersson DI, Hughes D. 2010. Antibiotic resistance and its cost: is it possible to reverse resistance?
Nat Rev Microbiol
8:260–271. 10.1038/nrmicro2319.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nrmicro2319" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20208551/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol&amp;title=Antibiotic%20resistance%20and%20its%20cost:%20is%20it%20possible%20to%20reverse%20resistance?&amp;volume=8&amp;publication_year=2010&amp;pages=260-271&amp;pmid=20208551&amp;doi=10.1038/nrmicro2319&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28">
<span class="label">28.</span><cite>Carvalhais LC, Dennis PG, Badri DV, Kidd BN, Vivanco JM, Schenk PM. 2015. Linking jasmonic acid signaling, root exudates, and rhizosphere microbiomes. Mol Plant Microbe Interact
28:1049–1058. 10.1094/MPMI-01-15-0016-R.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1094/MPMI-01-15-0016-R" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26035128/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol%20Plant%20Microbe%20Interact&amp;title=Linking%20jasmonic%20acid%20signaling,%20root%20exudates,%20and%20rhizosphere%20microbiomes&amp;volume=28&amp;publication_year=2015&amp;pages=1049-1058&amp;pmid=26035128&amp;doi=10.1094/MPMI-01-15-0016-R&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29">
<span class="label">29.</span><cite>Rifat D, Campodónico VL, Tao J, Miller JA, Alp A, Yao Y, Karakousis PC. 2017. In vitro and in vivo fitness costs associated with <em>Mycobacterium tuberculosis</em> RpoB mutation H526D. Future Microbiol
12:753–765. 10.2217/fmb-2017-0022.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2217/fmb-2017-0022" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5506874/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28343421/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Future%20Microbiol&amp;title=In%20vitro%20and%20in%20vivo%20fitness%20costs%20associated%20with%20Mycobacterium%20tuberculosis%20RpoB%20mutation%20H526D&amp;volume=12&amp;publication_year=2017&amp;pages=753-765&amp;pmid=28343421&amp;doi=10.2217/fmb-2017-0022&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30">
<span class="label">30.</span><cite>LaCroix RA, Sandberg TE, O’Brien EJ, Utrilla J, Ebrahim A, Guzman GI, Szubin R, Palsson BO, Feist AM. 2015. Use of adaptive laboratory evolution to discover key mutations enabling rapid growth of <em>Escherichia coli</em> K-12 MG1655 on glucose minimal medium. Appl Environ Microbiol
81:17–30. 10.1128/AEM.02246-14.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.02246-14" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4272732/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25304508/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Use%20of%20adaptive%20laboratory%20evolution%20to%20discover%20key%20mutations%20enabling%20rapid%20growth%20of%20Escherichia%20coli%20K-12%20MG1655%20on%20glucose%20minimal%20medium&amp;volume=81&amp;publication_year=2015&amp;pages=17-30&amp;pmid=25304508&amp;doi=10.1128/AEM.02246-14&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31">
<span class="label">31.</span><cite>Sandberg TE, Pedersen M, LaCroix RA, Ebrahim A, Bonde M, Herrgard MJ, Palsson BO, Sommer M, Feist AM. 2014. Evolution of <em>Escherichia coli</em> to 42°C and subsequent genetic engineering reveals adaptive mechanisms and novel mutations. Mol Biol Evol
31:2647–2662. 10.1093/molbev/msu209.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/msu209" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4166923/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25015645/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=Evolution%20of%20Escherichia%20coli%20to%2042%C2%B0C%20and%20subsequent%20genetic%20engineering%20reveals%20adaptive%20mechanisms%20and%20novel%20mutations&amp;volume=31&amp;publication_year=2014&amp;pages=2647-2662&amp;pmid=25015645&amp;doi=10.1093/molbev/msu209&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32">
<span class="label">32.</span><cite>Conrad TM, Frazier M, Joyce AR, Cho B-K, Knight EM, Lewis NE, Landick R, Palsson BØ. 2010. RNA polymerase mutants found through adaptive evolution reprogram <em>Escherichia coli</em> for optimal growth in minimal media. Proc Natl Acad Sci USA
107:20500–20505. 10.1073/pnas.0911253107.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0911253107" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2996682/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21057108/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=RNA%20polymerase%20mutants%20found%20through%20adaptive%20evolution%20reprogram%20Escherichia%20coli%20for%20optimal%20growth%20in%20minimal%20media&amp;volume=107&amp;publication_year=2010&amp;pages=20500-20505&amp;pmid=21057108&amp;doi=10.1073/pnas.0911253107&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B33">
<span class="label">33.</span><cite>Utrilla J, O’Brien EJ, Chen K, McCloskey D, Cheung J, Wang H, Armenta-Medina D, Feist AM, Palsson BO. 2016. Global rebalancing of cellular resources by pleiotropic point mutations illustrates a multi-scale mechanism of adaptive evolution. Cell Syst
2:260–271. 10.1016/j.cels.2016.04.003.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cels.2016.04.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4853925/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27135538/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=Global%20rebalancing%20of%20cellular%20resources%20by%20pleiotropic%20point%20mutations%20illustrates%20a%20multi-scale%20mechanism%20of%20adaptive%20evolution&amp;volume=2&amp;publication_year=2016&amp;pages=260-271&amp;pmid=27135538&amp;doi=10.1016/j.cels.2016.04.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34">
<span class="label">34.</span><cite>Rodríguez-Verdugo A, Tenaillon O, Gaut BS. 2016. First-step mutations during adaptation restore the expression of hundreds of genes. Mol Biol Evol
33:25–39. 10.1093/molbev/msv228.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/msv228" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4693981/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26500250/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=First-step%20mutations%20during%20adaptation%20restore%20the%20expression%20of%20hundreds%20of%20genes&amp;volume=33&amp;publication_year=2016&amp;pages=25-39&amp;pmid=26500250&amp;doi=10.1093/molbev/msv228&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35">
<span class="label">35.</span><cite>Gao W, Cameron DR, Davies JK, Kostoulias X, Stepnell J, Tuck KL, Yeaman MR, Peleg AY, Stinear TP, Howden BP. 2013. The RpoB H<sub>481</sub>Y rifampicin resistance mutation and an active stringent response reduce virulence and increase resistance to innate immune responses in <em>Staphylococcus aureus</em>. J Infect Dis
207:929–939. 10.1093/infdis/jis772.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/infdis/jis772" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3633451/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23255563/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&amp;title=The%20RpoB%20H481Y%20rifampicin%20resistance%20mutation%20and%20an%20active%20stringent%20response%20reduce%20virulence%20and%20increase%20resistance%20to%20innate%20immune%20responses%20in%20Staphylococcus%20aureus&amp;volume=207&amp;publication_year=2013&amp;pages=929-939&amp;pmid=23255563&amp;doi=10.1093/infdis/jis772&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36">
<span class="label">36.</span><cite>Colicchio R, Pagliuca C, Pastore G, Cicatiello AG, Pagliarulo C, Talà A, Scaglione E, Sammartino JC, Bucci C, Alifano P, Salvatore P. 2015. Fitness cost of rifampin resistance in <em>Neisseria meningitidis</em>: in vitro study of mechanisms associated with <em>rpoB</em> H553Y mutation. Antimicrob Agents Chemother
59:7637–7649. 10.1128/AAC.01746-15.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AAC.01746-15" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4649176/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26416867/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&amp;title=Fitness%20cost%20of%20rifampin%20resistance%20in%20Neisseria%20meningitidis:%20in%20vitro%20study%20of%20mechanisms%20associated%20with%20rpoB%20H553Y%20mutation&amp;volume=59&amp;publication_year=2015&amp;pages=7637-7649&amp;pmid=26416867&amp;doi=10.1128/AAC.01746-15&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B37">
<span class="label">37.</span><cite>Hua X, Wang H, Wang C, Tian B, Hua Y. 2011. Global effect of an RNA polymerase β-subunit mutation on gene expression in the radiation-resistant bacterium <em>Deinococcus radiodurans</em>. Sci China Life Sci
54:854–862. 10.1007/s11427-011-4209-3.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11427-011-4209-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21809039/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci%20China%20Life%20Sci&amp;title=Global%20effect%20of%20an%20RNA%20polymerase%20%CE%B2-subunit%20mutation%20on%20gene%20expression%20in%20the%20radiation-resistant%20bacterium%20Deinococcus%20radiodurans&amp;volume=54&amp;publication_year=2011&amp;pages=854-862&amp;pmid=21809039&amp;doi=10.1007/s11427-011-4209-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B38">
<span class="label">38.</span><cite>Qi Q, Preston GM, MacLean RC. 2014. Linking system-wide impacts of RNA polymerase mutations to the fitness cost of rifampin resistance in <em>Pseudomonas aeruginosa</em>. mBio
5:e01562-14. 10.1128/mBio.01562-14.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/mBio.01562-14" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4324240/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25491352/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Linking%20system-wide%20impacts%20of%20RNA%20polymerase%20mutations%20to%20the%20fitness%20cost%20of%20rifampin%20resistance%20in%20Pseudomonas%20aeruginosa&amp;volume=5&amp;publication_year=2014&amp;pages=e01562-14&amp;pmid=25491352&amp;doi=10.1128/mBio.01562-14&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B39">
<span class="label">39.</span><cite>Inaoka T, Takahashi K, Yada H, Yoshida M, Ochi K. 2004. RNA polymerase mutation activates the production of a dormant antibiotic 3,3′-neotrehalosadiamine via an autoinduction mechanism in <em>Bacillus subtilis</em>. J Biol Chem
279:3885–3892. 10.1074/jbc.M309925200.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M309925200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14612444/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=RNA%20polymerase%20mutation%20activates%20the%20production%20of%20a%20dormant%20antibiotic%203,3%E2%80%B2-neotrehalosadiamine%20via%20an%20autoinduction%20mechanism%20in%20Bacillus%20subtilis&amp;volume=279&amp;publication_year=2004&amp;pages=3885-3892&amp;pmid=14612444&amp;doi=10.1074/jbc.M309925200&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B40">
<span class="label">40.</span><cite>Concepción-Acevedo J, Weiss HN, Chaudhry WN, Levin BR. 2015. Malthusian parameters as estimators of the fitness of microbes: a cautionary tale about the low side of high throughput. PLoS One
10:e0126915. 10.1371/journal.pone.0126915.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0126915" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4482697/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26114477/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Malthusian%20parameters%20as%20estimators%20of%20the%20fitness%20of%20microbes:%20a%20cautionary%20tale%20about%20the%20low%20side%20of%20high%20throughput&amp;volume=10&amp;publication_year=2015&amp;pages=e0126915&amp;pmid=26114477&amp;doi=10.1371/journal.pone.0126915&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B41">
<span class="label">41.</span><cite>Maughan H, Nicholson WL. 2011. Increased fitness and alteration of metabolic pathways during <em>Bacillus subtilis</em> evolution in the laboratory. Appl Environ Microbiol
77:4105–4118. 10.1128/AEM.00374-11.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.00374-11" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3131648/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21531833/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Increased%20fitness%20and%20alteration%20of%20metabolic%20pathways%20during%20Bacillus%20subtilis%20evolution%20in%20the%20laboratory&amp;volume=77&amp;publication_year=2011&amp;pages=4105-4118&amp;pmid=21531833&amp;doi=10.1128/AEM.00374-11&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B42">
<span class="label">42.</span><cite>Yüksel M, Power JJ, Ribbe J, Volkmann T, Maier B. 2016. Fitness trade-offs in competence differentiation of <em>Bacillus subtilis</em>. Front Microbiol
7:888. 10.3389/fmicb.2016.00888.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2016.00888" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4896167/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27375604/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Fitness%20trade-offs%20in%20competence%20differentiation%20of%20Bacillus%20subtilis&amp;volume=7&amp;publication_year=2016&amp;pages=888&amp;pmid=27375604&amp;doi=10.3389/fmicb.2016.00888&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B43">
<span class="label">43.</span><cite>Nicholson WL. 2012. Increased competitive fitness of <em>Bacillus subtilis</em> under nonsporulating conditions via inactivation of pleiotropic regulators AlsR, SigD, and SigW. Appl Environ Microbiol
78:3500–3503. 10.1128/AEM.07742-11.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.07742-11" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3346490/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22344650/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Increased%20competitive%20fitness%20of%20Bacillus%20subtilis%20under%20nonsporulating%20conditions%20via%20inactivation%20of%20pleiotropic%20regulators%20AlsR,%20SigD,%20and%20SigW&amp;volume=78&amp;publication_year=2012&amp;pages=3500-3503&amp;pmid=22344650&amp;doi=10.1128/AEM.07742-11&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B44">
<span class="label">44.</span><cite>Stefan MA, Ugur FS, Garcia GA. 2018. Source of the fitness defect in rifamycin-resistant <em>Mycobacterium tuberculosis</em> RNA polymerase and the mechanism of compensation by mutations in the β′ subunit. Antimicrob Agents Chemother
62:e00164-18. 10.1128/AAC.00164-18.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AAC.00164-18" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5971562/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29661864/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&amp;title=Source%20of%20the%20fitness%20defect%20in%20rifamycin-resistant%20Mycobacterium%20tuberculosis%20RNA%20polymerase%20and%20the%20mechanism%20of%20compensation%20by%20mutations%20in%20the%20%CE%B2%E2%80%B2%20subunit&amp;volume=62&amp;publication_year=2018&amp;pages=e00164-18&amp;pmid=29661864&amp;doi=10.1128/AAC.00164-18&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B45">
<span class="label">45.</span><cite>Xu Z, Zhou A, Wu J, Zhou A, Li J, Zhang S, Wu W, Karakousis PC, Yao Y-F. 2018. Transcriptional approach for decoding the mechanism of <em>rpoC</em> compensatory mutations for the fitness cost in rifampicin-resistant <em>Mycobacterium tuberculosis</em>. Front Microbiol
9:2895. 10.3389/fmicb.2018.02895.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2018.02895" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6283890/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30555440/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Transcriptional%20approach%20for%20decoding%20the%20mechanism%20of%20rpoC%20compensatory%20mutations%20for%20the%20fitness%20cost%20in%20rifampicin-resistant%20Mycobacterium%20tuberculosis&amp;volume=9&amp;publication_year=2018&amp;pages=2895&amp;pmid=30555440&amp;doi=10.3389/fmicb.2018.02895&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B46">
<span class="label">46.</span><cite>Brandis G, Hughes D. 2018. Mechanisms of fitness cost reduction for rifampicin-resistant strains with deletion or duplication mutations in <em>rpoB</em>. Sci Rep
8:17488. 10.1038/s41598-018-36005-y.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-018-36005-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6269455/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30504835/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Mechanisms%20of%20fitness%20cost%20reduction%20for%20rifampicin-resistant%20strains%20with%20deletion%20or%20duplication%20mutations%20in%20rpoB&amp;volume=8&amp;publication_year=2018&amp;pages=17488&amp;pmid=30504835&amp;doi=10.1038/s41598-018-36005-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B47">
<span class="label">47.</span><cite>Cutting SM, Vander Horn PB. 1990. Genetic analysis, p 27–74, <em>In</em>
Harwood CR, Cutting SM (ed), Molecular biological methods for Bacillus.
John Wiley &amp; Sons Ltd, Chichester, England.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Molecular%20biological%20methods%20for%20Bacillus.&amp;publication_year=1990&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B48">
<span class="label">48.</span><cite>Miller JH. 1972. Experiments in molecular genetics.
Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Experiments%20in%20molecular%20genetics.&amp;publication_year=1972&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B49">
<span class="label">49.</span><cite>Morrison MD, Fajardo-Cavazos P, Nicholson WL. 2017. Cultivation in space flight produces minimal alterations in the susceptibility of <em>Bacillus subtilis</em> cells to 72 different antibiotics and growth-inhibiting compounds. Appl Environ Microbiol
83:e01584-17. 10.1128/AEM.01584-17.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.01584-17" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5648920/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28821547/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Cultivation%20in%20space%20flight%20produces%20minimal%20alterations%20in%20the%20susceptibility%20of%20Bacillus%20subtilis%20cells%20to%2072%20different%20antibiotics%20and%20growth-inhibiting%20compounds&amp;volume=83&amp;publication_year=2017&amp;pages=e01584-17&amp;pmid=28821547&amp;doi=10.1128/AEM.01584-17&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B50">
<span class="label">50.</span><cite>Spizizen J. 1958. Transformation of biochemically deficient strains of <em>Bacillus subtilis</em> by deoxyribonucleate. Proc Natl Acad Sci USA
44:1072–1078. 10.1073/pnas.44.10.1072.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.44.10.1072" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC528696/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16590310/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Transformation%20of%20biochemically%20deficient%20strains%20of%20Bacillus%20subtilis%20by%20deoxyribonucleate&amp;volume=44&amp;publication_year=1958&amp;pages=1072-1078&amp;pmid=16590310&amp;doi=10.1073/pnas.44.10.1072&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B51">
<span class="label">51.</span><cite>Altenbuchner J. 2016. Editing of the <em>Bacillus subtilis</em> genome by the CRISPR-Cas9 system. Appl Environ Microbiol
82:5421–5427. 10.1128/AEM.01453-16.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.01453-16" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4988203/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27342565/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Editing%20of%20the%20Bacillus%20subtilis%20genome%20by%20the%20CRISPR-Cas9%20system&amp;volume=82&amp;publication_year=2016&amp;pages=5421-5427&amp;pmid=27342565&amp;doi=10.1128/AEM.01453-16&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B52">
<span class="label">52.</span><cite>Silva D, Santos G, Barroca M, Collins T. 2017. Inverse PCR for point mutation introduction. Methods Mol Biol
1620:87–100. 10.1007/978-1-4939-7060-5_5.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-1-4939-7060-5_5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28540701/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&amp;title=Inverse%20PCR%20for%20point%20mutation%20introduction&amp;volume=1620&amp;publication_year=2017&amp;pages=87-100&amp;pmid=28540701&amp;doi=10.1007/978-1-4939-7060-5_5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B53">
<span class="label">53.</span><cite>Maughan H, Callicotte V, Hancock A, Birky CW, Nicholson WL, Masel J. 2006. The population genetics of phenotypic deterioration in experimental populations of <em>Bacillus subtilis</em>. Evolution
60:686–695. 10.1111/j.0014-3820.2006.tb01148.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.0014-3820.2006.tb01148.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16739451/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=The%20population%20genetics%20of%20phenotypic%20deterioration%20in%20experimental%20populations%20of%20Bacillus%20subtilis&amp;volume=60&amp;publication_year=2006&amp;pages=686-695&amp;pmid=16739451&amp;doi=10.1111/j.0014-3820.2006.tb01148.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B54">
<span class="label">54.</span><cite>Woods RJ, Barrick JE, Cooper TF, Shrestha U, Kauth MR, Lenski RE. 2011. Second-order selection for evolvability in a large <em>Escherichia coli</em> population. Science
331:1433–1436. 10.1126/science.1198914.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1198914" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3176658/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21415350/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Second-order%20selection%20for%20evolvability%20in%20a%20large%20Escherichia%20coli%20population&amp;volume=331&amp;publication_year=2011&amp;pages=1433-1436&amp;pmid=21415350&amp;doi=10.1126/science.1198914&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B55">
<span class="label">55.</span><cite>Andrews S. 2010. FastQC: a quality control tool for high throughput sequence data, v0.11.5. Babraham Bioinformatics. <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" rel="noopener noreferrer" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>.</cite>
</li>
<li id="B56">
<span class="label">56.</span><cite>Langmead B, Salzberg SL. 2012. Fast gapped-read alignment with Bowtie 2. Nat Methods
9:357–359. 10.1038/nmeth.1923.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.1923" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3322381/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22388286/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;volume=9&amp;publication_year=2012&amp;pages=357-359&amp;pmid=22388286&amp;doi=10.1038/nmeth.1923&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B57">
<span class="label">57.</span><cite>Wilm A, Aw PPK, Bertrand D, Yeo GHT, Ong SH, Wong CH, Khor CC, Petric R, Hibberd ML, Nagarajan N. 2012. LoFreq: a sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets. Nucleic Acids Res
40:11189–11201. 10.1093/nar/gks918.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gks918" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3526318/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23066108/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=LoFreq:%20a%20sequence-quality%20aware,%20ultra-sensitive%20variant%20caller%20for%20uncovering%20cell-population%20heterogeneity%20from%20high-throughput%20sequencing%20datasets&amp;volume=40&amp;publication_year=2012&amp;pages=11189-11201&amp;pmid=23066108&amp;doi=10.1093/nar/gks918&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B58">
<span class="label">58.</span><cite>Shapiro SS, Wilk MB. 1965. An analysis of variance test for normality (complete samples). Biometrika
52:591–611. 10.2307/2333709.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2307/2333709" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biometrika&amp;title=An%20analysis%20of%20variance%20test%20for%20normality%20(complete%20samples)&amp;volume=52&amp;publication_year=1965&amp;pages=591-611&amp;doi=10.2307/2333709&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B59">
<span class="label">59.</span><cite>Haynes W. 2013. Tukey’s test, p 2303–2304. <em>In</em>
Dubitzky W, Wolkenhauer O, Cho K-H, Yokota H (ed), Encyclopedia of systems biology.
Springer, New York, NY. 10.1007/978-1-4419-9863-7_1212.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-1-4419-9863-7_1212" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Encyclopedia%20of%20systems%20biology.&amp;publication_year=2013&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B60">
<span class="label">60.</span><cite>Dunnett CW. 1955. A multiple comparison procedure for comparing several treatments with a control. J Am Stat Assoc
50:1096–1121. 10.1080/01621459.1955.10501294.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/01621459.1955.10501294" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Stat%20Assoc&amp;title=A%20multiple%20comparison%20procedure%20for%20comparing%20several%20treatments%20with%20a%20control&amp;volume=50&amp;publication_year=1955&amp;pages=1096-1121&amp;doi=10.1080/01621459.1955.10501294&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B61">
<span class="label">61.</span><cite>Qi Y, Hulett FM. 1998. PhoP-P and RNA polymerase sigmaA holoenzyme are sufficient for transcription of Pho regulon promoters in <em>Bacillus subtilis</em>: PhoP-P activator sites within the coding region stimulate transcription in vitro. Mol Microbiol
28:1187–1197. 10.1046/j.1365-2958.1998.00882.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1365-2958.1998.00882.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9680208/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&amp;title=PhoP-P%20and%20RNA%20polymerase%20sigmaA%20holoenzyme%20are%20sufficient%20for%20transcription%20of%20Pho%20regulon%20promoters%20in%20Bacillus%20subtilis:%20PhoP-P%20activator%20sites%20within%20the%20coding%20region%20stimulate%20transcription%20in%20vitro&amp;volume=28&amp;publication_year=1998&amp;pages=1187-1197&amp;pmid=9680208&amp;doi=10.1046/j.1365-2958.1998.00882.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B62">
<span class="label">62.</span><cite>Waters SM, Zeigler DR, Nicholson WL. 2015. Experimental evolution of enhanced growth by <em>Bacillus subtilis</em> at low atmospheric pressure: genomic changes revealed by whole-genome sequencing. Appl Environ Microbiol
81:7525–7532. 10.1128/AEM.01690-15.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.01690-15" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4592849/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26296725/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Experimental%20evolution%20of%20enhanced%20growth%20by%20Bacillus%20subtilis%20at%20low%20atmospheric%20pressure:%20genomic%20changes%20revealed%20by%20whole-genome%20sequencing&amp;volume=81&amp;publication_year=2015&amp;pages=7525-7532&amp;pmid=26296725&amp;doi=10.1128/AEM.01690-15&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B63">
<span class="label">63.</span><cite>Guérout-Fleury AM, Frandsen N, Stragier P. 1996. Plasmids for ectopic integration in <em>Bacillus subtilis</em>. Gene
180:57–61. 10.1016/s0378-1119(96)00404-0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s0378-1119(96)00404-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8973347/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Plasmids%20for%20ectopic%20integration%20in%20Bacillus%20subtilis&amp;volume=180&amp;publication_year=1996&amp;pages=57-61&amp;pmid=8973347&amp;doi=10.1016/s0378-1119(96)00404-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section></section></section></article>
</main></body></html>