<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4340379/"/><title>Trimeric Structure of (+)-Pinoresinol-forming Dirigent Protein at 1.95 Å Resolution with Three Isolated Active Sites - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">J Biol Chem</button></div>. 2014 Nov 19;290(3):1308–1318. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M114.611780" rel="noopener noreferrer" target="_blank">10.1074/jbc.M114.611780</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Trimeric Structure of (+)-Pinoresinol-forming Dirigent Protein at 1.95 Å Resolution with Three Isolated Active Sites<sup><a class="usa-link" href="#FN1">*</a></sup>
</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20KW%22%5BAuthor%5D"><span class="name western">Kye-Won Kim</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Kye-Won Kim</span></h3>
<div class="p">From the <sup>‡</sup>Institute of Biological Chemistry, Washington State University, Pullman, Washington 99164-6340, </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20KW%22%5BAuthor%5D"><span class="name western">Kye-Won Kim</span></a>
</div>
</div>
<sup>‡</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20CA%22%5BAuthor%5D"><span class="name western">Clyde A Smith</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Clyde A Smith</span></h3>
<div class="p">the <sup>§</sup>Stanford Synchrotron Radiation Lightsource, Menlo Park, California 94025, and </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20CA%22%5BAuthor%5D"><span class="name western">Clyde A Smith</span></a>
</div>
</div>
<sup>§</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Daily%20MD%22%5BAuthor%5D"><span class="name western">Michael D Daily</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Michael D Daily</span></h3>
<div class="p">the <sup>¶</sup>Fundamental and Computational Sciences Directorate, Pacific Northwest National Laboratory, Richland, Washington 99354</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Daily%20MD%22%5BAuthor%5D"><span class="name western">Michael D Daily</span></a>
</div>
</div>
<sup>¶</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cort%20JR%22%5BAuthor%5D"><span class="name western">John R Cort</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">John R Cort</span></h3>
<div class="p">the <sup>¶</sup>Fundamental and Computational Sciences Directorate, Pacific Northwest National Laboratory, Richland, Washington 99354</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cort%20JR%22%5BAuthor%5D"><span class="name western">John R Cort</span></a>
</div>
</div>
<sup>¶</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Davin%20LB%22%5BAuthor%5D"><span class="name western">Laurence B Davin</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Laurence B Davin</span></h3>
<div class="p">From the <sup>‡</sup>Institute of Biological Chemistry, Washington State University, Pullman, Washington 99164-6340, </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Davin%20LB%22%5BAuthor%5D"><span class="name western">Laurence B Davin</span></a>
</div>
</div>
<sup>‡</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lewis%20NG%22%5BAuthor%5D"><span class="name western">Norman G Lewis</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Norman G Lewis</span></h3>
<div class="p">From the <sup>‡</sup>Institute of Biological Chemistry, Washington State University, Pullman, Washington 99164-6340, </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lewis%20NG%22%5BAuthor%5D"><span class="name western">Norman G Lewis</span></a>
</div>
</div>
<sup>‡,</sup><sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">From the <sup>‡</sup>Institute of Biological Chemistry, Washington State University, Pullman, Washington 99164-6340, </div>
<div id="aff2">the <sup>§</sup>Stanford Synchrotron Radiation Lightsource, Menlo Park, California 94025, and </div>
<div id="aff3">the <sup>¶</sup>Fundamental and Computational Sciences Directorate, Pacific Northwest National Laboratory, Richland, Washington 99354</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>1</sup><p class="display-inline">To whom correspondence should be addressed: Inst. of Biological Chemistry, Washington State University, Pullman, WA 99164-6340. Tel.: 509-335-2682; Fax: 509-335-8206; Email: <span>lewisn@wsu.edu</span>.</p>
</div></div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2014 Sep 16; Revised 2014 Nov 11; Issue date 2015 Jan 16.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© 2015 by The American Society for Biochemistry and Molecular Biology, Inc.</div>
<p><em>Author's Choice</em>—Final version full access.</p>
<p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://creativecommons.org/licenses/by/3.0/" rel="noopener noreferrer" target="_blank">Creative Commons Attribution Unported License</a> applies to Author Choice Articles</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC4340379  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25411250/">25411250</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><hr class="headless"/>
<p><strong>Background:</strong> Dirigent protein (DP) discovery gave new paradigm for monolignol-derived coupling <em>in planta.</em></p>
<p><strong>Results:</strong> (+)-Pinoresinol-forming DP (PsDRR206) three-dimensional structure was obtained at 1.95 Å resolution.</p>
<p><strong>Conclusion:</strong> The tightly packed trimeric DP has three putative substrate binding sites spatially far apart, suggesting that each site involves monomer coupling directly.</p>
<p><strong>Significance:</strong> New insights into monolignol radical-radical coupling <em>in planta</em>.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> Crystallography, Homology Modeling, Molecular Docking, Protein Crystallization, Secondary Metabolism, Dirigent Proteins, Phenoxy Radical Radical Coupling</p></section></section><section class="abstract" id="abstract2"><h2>Abstract</h2>
<p>Control over phenoxy radical-radical coupling reactions <em>in vivo</em> in vascular plants was enigmatic until our discovery of dirigent proteins (DPs, from the Latin <em>dirigere</em>, to guide or align). The first three-dimensional structure of a DP ((+)-pinoresinol-forming DP, 1.95 Å resolution, rhombohedral space group H32)) is reported herein. It has a tightly packed trimeric structure with an eight-stranded β-barrel topology for each DP monomer. Each putative substrate binding and orientation coupling site is located on the trimer surface but too far apart for intermolecular coupling between sites. It is proposed that each site enables stereoselective coupling (using either two coniferyl alcohol radicals or a radical and a monolignol). Interestingly, there are six differentially conserved residues in DPs affording either the (+)- or (−)-antipodes in the vicinity of the putative binding site and region known to control stereoselectivity. DPs are involved in lignan biosynthesis, whereas dirigent domains/sites have been implicated in lignin deposition.</p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>In terrestrial vascular plants, monolignol coupling affords the lignans (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>), as well as the polymeric structural lignins (<a aria-describedby="B2" class="usa-link" href="#B2">2</a>), with the latter being the most abundant plant biopolymers next to cellulose. The former are, however, a very diverse class of vascular plant natural products, typically dimers and/or higher oligomers, whose physiological roles <em>in planta</em> are considered mainly involved in plant defense, particularly in leaf, (heart)wood, knot, and seed coat tissues. Many lignans are optically active and can have diverse coupling modes, although the 8–8′ interunit (dimer) linkage is perhaps the most commonly encountered (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>). Some are also medicinally important, such as podophyllotoxin, or its semi-synthetic derivatives, etoposide, teniposide, and Etopophos® widely used in cancer therapies (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>).</p>
<p>The entry point into presumed phenoxy radical-radical coupling to the 8–8′ lignan class involves one electron oxidation of the monolignol, <em>e.g.</em> coniferyl alcohol (<a class="usa-link" href="#F1">Fig. 1</a><em>A</em>, <strong>1</strong>), which in the presence of a specific type of dirigent protein (DP; from the Latin <em>dirigere</em>, to guide or to align)<a class="usa-link" href="#FN2"><sup>2</sup></a> (<a aria-describedby="B3" class="usa-link" href="#B3">3</a>), can undergo distinct forms of stereoselective coupling to afford either the (+)- or the (−)-antipode of pinoresinol (<a class="usa-link" href="#F1">Fig. 1</a><em>A</em>, <strong>2a</strong> or <strong>2b</strong>) (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>, <a aria-describedby="B3" class="usa-link" href="#B3">3</a><a aria-describedby="B4" class="usa-link" href="#B4">–</a><a aria-describedby="B6" class="usa-link" href="#B6">6</a>). Mechanistically, one electron oxidation (<em>e.g.</em> catalyzed by laccases, peroxidases) is envisaged to generate the corresponding coniferyl alcohol (<strong>1</strong>)-derived free radical intermediate (CA<sup>•</sup>), which then binds to the DPs. Bound substrate orientation subsequently occurs in such a way as to enable 8–8′ coupling at the <em>si-si</em> face with concomitant intramolecular cyclization to afford (+)-pinoresinol (<a class="usa-link" href="#F1">Fig. 1</a><em>A</em>, <strong>2a</strong>), such as in <em>Forsythia intermedia</em> (<a aria-describedby="B3" class="usa-link" href="#B3">3</a>), <em>Thuja plicata</em> (<a aria-describedby="B4" class="usa-link" href="#B4">4</a>), <em>Schizandra chinensis</em> (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>), or pea (<em>Pisum sativum</em>) (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>), or alternatively <em>re-re</em> coupling to give the (−)-antipode (<a class="usa-link" href="#F1">Fig. 1</a><em>A</em>, <strong>2b</strong>) as in <em>Arabidopsis</em> (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>, <a aria-describedby="B5" class="usa-link" href="#B5">5</a>, <a aria-describedby="B6" class="usa-link" href="#B6">6</a>) and flax (<em>Linum usitatissimum</em>).<a class="usa-link" href="#FN3"><sup>3</sup></a> Of these, the (+)-pinoresinol-forming DP in pea is annotated as PsDRR206 (<a aria-describedby="B9" class="usa-link" href="#B9">9</a>), and the (−)-pinoresinol-forming DP in <em>Arabidopsis thaliana</em> is named AtDIR6 (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>).</p>
<figure class="fig xbox font-sm" id="F1"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4340379_zbc0051505840001.jpg" target="_blank"><img alt="FIGURE 1." class="graphic zoom-in" height="888" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1984/4340379/df25db16e8d0/zbc0051505840001.jpg" width="713"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>Pinoresinol-forming dirigent proteins.</strong>
<em>A</em>, proposed biosynthesis mechanism to afford (+)- or (−)-pinoresinols (<strong>2a</strong> or <strong>2b</strong>) in DP-guided stereoselective couplings with coniferyl alcohol (<strong>1</strong>) as substrate. <em>B</em>, amino acid sequence alignments of mature (+)- and (−)-pinoresinol-forming DPs with their structural homologue AtAOC2. Secondary structures of PsDRR206 (<em>orange</em>) and AtAOC2 (<em>blue</em>) are also drawn. Loop residues in the PsDRR206 structure absent in the (−)-pinoresinol-forming DPs are indicated with <em>green dashed lines</em> in the sequences.</p></figcaption></figure><p>In this investigation, determination of the x-ray crystal structure of the (+)-pinoresinol-forming DP PsDRR206 is reported. These results are discussed with regard to previously proposed mechanisms for DP-mediated CA<sup>•</sup> coupling (<a aria-describedby="B10" class="usa-link" href="#B10">10</a>) and a published homology model of AtDIR6 (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>), as well as existing x-ray structures of allene oxide cyclase (AOC) (<a aria-describedby="B12" class="usa-link" href="#B12">12</a>, <a aria-describedby="B13" class="usa-link" href="#B13">13</a>). AOC is a plant enzyme that forms 12-oxo-phytodienoic acid, an intermediate in jasmonic acid biosynthesis, from a reactive allene oxide precursor, 12,13-epoxyoctadeca-9,11,15-trienoate (<a aria-describedby="B14" class="usa-link" href="#B14">14</a>). AOC shares distant sequence similarity to DPs.</p></section><section id="sec2"><h2 class="pmc_sec_title">EXPERIMENTAL PROCEDURES</h2>
<section id="sec3"><h3 class="pmc_sec_title"></h3>
<section id="sec4"><h4 class="pmc_sec_title"></h4>
<section id="sec5"><h5 class="pmc_sec_title">Cloning, Expression, and Purification of PsDRR206</h5>
<p>The PsDRR206 gene (GenBank<sup>TM</sup> accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/U11716" rel="noopener noreferrer" target="_blank">U11716</a>) was isolated from pea (<em>P. sativum</em> cv. Alcan) genomic DNA using PCR amplification with gene specific primers (DRR206 forward, ATGGGTTCCAAACTTCTAGTACTA, and DRR206 reverse, TTACCAACACTCAAAGAACTTGAT). Vector construction and transformation were carried out using biolistic bombardment technology as described by Seneviratne <em>et al.</em> (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>), as was expression and purification of the (+)-pinoresinol-forming PsDRR206 recombinant DP in tomato cell culture.</p></section><section id="sec6"><h5 class="pmc_sec_title">Crystallization and X-ray Data Collection</h5>
<p>Initial crystallization conditions for the PsDRR206 protein were obtained using the microbatch under oil method employing 1536-well microassay plate high throughput screening (<a aria-describedby="B15" class="usa-link" href="#B15">15</a>) at the Hauptman Woodward Institute (Buffalo, NY). Four conditions from preliminary screening produced microcrystals: condition 265, 0.1 <span class="font-variant-small-caps">m</span> (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 0.1 <span class="font-variant-small-caps">m</span> CAPS pH 10, PEG8000 20% (v/v); condition 1006, 0.1 <span class="font-variant-small-caps">m</span> NH<sub>4</sub>Br, 0.1 <span class="font-variant-small-caps">m</span> Tris, pH 8, PEG 400 40% (v/v); condition 1077, 5% 1-butyl-2,3-dimethylimidazolium tetrafluoroborate (w/v), 0.09 <span class="font-variant-small-caps">m</span> Bis-Tris propane, pH 7.8, and 27% PEG 3350; and condition 1474, 0.1 <span class="font-variant-small-caps">m</span> NaCl, 0.1 <span class="font-variant-small-caps">m</span> NaOAc, pH 5.0, PEG 400 80% (w/v). Initial “hits” were scaled up in-house as hanging drop vapor diffusion methods on VDX 24-well plates (Hampton Research, CA) incubated at 22 °C with a drop size of 3 μl consisting of equal volumes of protein at a concentration of 4–5 mg/ml in 40 m<span class="font-variant-small-caps">m</span> MES, pH 5, 20 m<span class="font-variant-small-caps">m</span> Na<sub>2</sub>SO<sub>4</sub>, 20% glycerol, and reservoir solution. Each drop was equilibrated against a 500-μl reservoir and monitored periodically. Crystallization conditions were further optimized using grid screening. Diffraction quality crystals were obtained from the final optimized crystallization condition 1077, comprising 4% 1-butyl-2,3-dimethylimidazolium tetrafluoroborate (w/v), 0.1 <span class="font-variant-small-caps">m</span> Bis-Tris propane, pH 7.8, and 35% PEG 3350. Crystals appeared within 5–10 days and grew to the size of 170 × 170 × 150 μm within 10 weeks. The PsDRR206 crystals were subsequently flash cooled in a cryoprotectant crystallization buffer, stored in cryovials, and shipped to the Stanford Synchrotron Radiation Lightsource for data collection.</p>
<p>Native PsDRR206 crystals, diffracting to ∼1.95 Å resolution, belong to the rhombohedral space group H32 (see <a class="usa-link" href="#T1">Table 1</a>). A complete data set comprising 600 images with a rotation angle of 0.2° was collected from a single PsDRR206 crystal on Stanford Synchrotron Radiation Lightsource Beamline BL12-2 using x-rays at 12,658 eV (0.98093 Å) and a PILATUS 6 <span class="font-variant-small-caps">m</span> PAD detector running in the shutterless mode. The data were processed with XDS (<a aria-describedby="B16" class="usa-link" href="#B16">16</a>) and scaled with SCALA from the CCP4 suite of programs (<a aria-describedby="B17" class="usa-link" href="#B17">17</a>). The Matthews coefficient (<a aria-describedby="B18" class="usa-link" href="#B18">18</a>) assuming two molecules in the asymmetric unit was 2.0 Å<sup>3</sup>/Da (39% solvent content). The final data collection statistics are given in <a class="usa-link" href="#T1">Table 1</a> (<a aria-describedby="B19" class="usa-link" href="#B19">19</a>).</p>
<section class="tw xbox font-sm" id="T1"><h6 class="obj_head">TABLE 1.</h6>
<div class="caption p"><p><strong>Data collection and refinement statistics for PsDRR206</strong></p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups"><tbody valign="top">
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Data collection</strong></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Space group</td>
<td align="left" colspan="1" rowspan="1">H32</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Maximum resolution (<em>d</em><sub>min</sub>) (Å)</td>
<td align="left" colspan="1" rowspan="1">1.95</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Unit cell dimensions (Å)</td>
<td align="left" colspan="1" rowspan="1">
<em>a</em> = <em>b</em> = 88.4, <em>c</em> = 196.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Observed reflections</td>
<td align="left" colspan="1" rowspan="1">142,020</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Unique reflections to <em>d</em><sub>min</sub>
</td>
<td align="left" colspan="1" rowspan="1">21,935</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    <em>R</em><sub>merge</sub> (%)<a class="usa-link" href="#TF1-1"><em><sup>a</sup></em></a>
</td>
<td align="left" colspan="1" rowspan="1">3.7 (66.1)<a class="usa-link" href="#TF1-2"><em><sup>b</sup></em></a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    <em>I</em>/σ</td>
<td align="left" colspan="1" rowspan="1">21.9 (2.6)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Completeness (%)</td>
<td align="left" colspan="1" rowspan="1">99.6 (98.1)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    CC½<a class="usa-link" href="#TF1-3"><em><sup>c</sup></em></a>
</td>
<td align="left" colspan="1" rowspan="1">100.0 (85.7)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Multiplicity</td>
<td align="left" colspan="1" rowspan="1">6.5 (5.4)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Wilson B (Å<sup>2</sup>)</td>
<td align="left" colspan="1" rowspan="1">39.6</td>
</tr>
<tr><td colspan="2" rowspan="1"><hr/></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Refinement</strong></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Resolution range (Å)</td>
<td align="left" colspan="1" rowspan="1">38.5–1.95</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    <em>R</em> factor/<em>R</em><sub>free</sub> (%)<a class="usa-link" href="#TF1-4"><em><sup>d</sup></em></a>
</td>
<td align="left" colspan="1" rowspan="1">20.1/24.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    <em>R</em><sub>all</sub> (%)<a class="usa-link" href="#TF1-5"><em><sup>e</sup></em></a>
</td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Total atoms</td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">        Protein</td>
<td align="left" colspan="1" rowspan="1">1030/1012</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">        Solvent</td>
<td align="left" colspan="1" rowspan="1">50</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    B factors</td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">        Protein chain (A/B) (Å<sup>2</sup>)</td>
<td align="left" colspan="1" rowspan="1">59.1/59.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">        Solvent (Å<sup>2</sup>)</td>
<td align="left" colspan="1" rowspan="1">57.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    RMSD from ideality</td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">        Bonds (Å)</td>
<td align="left" colspan="1" rowspan="1">0.009</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">        1–3 distances (Å)</td>
<td align="left" colspan="1" rowspan="1">1.43</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    Ramachandran plot</td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">        Residues in preferred regions (%)<a class="usa-link" href="#TF1-6"><em><sup>f</sup></em></a>
</td>
<td align="left" colspan="1" rowspan="1">98.2</td>
</tr>
</tbody></table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="TF1-1"><p><em><sup>a</sup> R</em><sub>merge</sub> = Σ|<em>I</em> − &lt;<em>I</em>&gt;|/Σ<em>I</em> × 100, where <em>I</em> = the observed intensity, and &lt;<em>I</em>&gt; is the mean intensity.</p></div>
<div class="fn" id="TF1-2"><p><em><sup>b</sup></em> Numbers in parentheses relate to the highest resolution shell, 2.00–1.95 Å.</p></div>
<div class="fn" id="TF1-3"><p><em><sup>c</sup></em> Percentage of correlation between intensities from random half-sets calculated by XDS (16).</p></div>
<div class="fn" id="TF1-4"><p><em><sup>d</sup> r</em> = Σ||<em>F</em><sub>o</sub>| − <em>k</em>|<em>F</em><sub>c</sub> ||/Σ|<em>F</em><sub>o</sub> | × 100. <em>R</em><sub>free</sub> was calculated with 5% of the reflections.</p></div>
<div class="fn" id="TF1-5"><p><em><sup>e</sup></em> Final <em>R</em> factor calculated with all data using no sigma cutoff.</p></div>
<div class="fn" id="TF1-6"><p><em><sup>f</sup></em> As defined in MOLPROBITY (<a aria-describedby="B19" class="usa-link" href="#B19">19</a>).</p></div>
</div></section></section><section id="sec7"><h5 class="pmc_sec_title">Data Processing, Structure Determination, and Refinement</h5>
<p>PsDRR206 structure was solved by molecular replacement using the homology model generated for the (−)-pinoresinol-forming DP AtDIR6 from <em>A. thaliana</em> (PM0078038 (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>)), which was derived from an x-ray structure of <em>A. thaliana</em> AOC2 (AtAOC2; PDB code 2BRJ (<a aria-describedby="B12" class="usa-link" href="#B12">12</a>)). The sequences of PsDRR206 and AtDIR6 were aligned and the AtDIR6 model was converted into a pseudo-DRR206 model using the program CHAINSAW (<a aria-describedby="B20" class="usa-link" href="#B20">20</a>) from the CCP4 suite (<a aria-describedby="B17" class="usa-link" href="#B17">17</a>). Identical residues in the two sequences were retained, and those that differed were truncated at the Cβ atom. A weak molecular replacement solution, with a weighted <em>R</em> factor of 0.61, and a score of 0.42 was obtained with the program MOLREP (<a aria-describedby="B21" class="usa-link" href="#B21">21</a>). This solution was refined for 15 cycles using REFMAC (<a aria-describedby="B22" class="usa-link" href="#B22">22</a>) giving crystallographic <em>R</em> factor and <em>R</em><sub>free</sub> values of 0.48 and 0.54, respectively, and 2<em>F</em><sub>o</sub> − <em>F</em><sub>c</sub> and <em>F</em><sub>o</sub> − <em>F</em><sub>c</sub> electron density maps were calculated. Inspection of the electron density showed that the molecular replacement solution was correct, with side chains truncated by CHAINSAW clearly visible in the 2<em>F</em><sub>o</sub> − <em>F</em><sub>c</sub> and <em>F</em><sub>o</sub> − <em>F</em><sub>c</sub> maps. Refinement of PsDRR206 structure was completed with the PHENIX suite of programs (<a aria-describedby="B23" class="usa-link" href="#B23">23</a>) and manual building of the model using the molecular graphics program COOT (<a aria-describedby="B24" class="usa-link" href="#B24">24</a>). Water molecules were added at structurally and chemically relevant positions, and the atomic displacement parameters for all atoms in the structure were refined isotropically. The final refinement statistics are given in <a class="usa-link" href="#T1">Table 1</a>.</p>
<p>Atomic coordinates and structure factors for PsDRR206 were deposited to the Protein Data Bank (<a aria-describedby="B25" class="usa-link" href="#B25">25</a>) with PDB code 4REV. Superpositions were performed using the SSM procedure (<a aria-describedby="B26" class="usa-link" href="#B26">26</a>) as implemented in COOT (<a aria-describedby="B24" class="usa-link" href="#B24">24</a>) and the program LSQKAB in the CCP4 suite (<a aria-describedby="B17" class="usa-link" href="#B17">17</a>). The figures were generated using PyMOL (<a aria-describedby="B27" class="usa-link" href="#B27">27</a>). The surface topography was analyzed with CASTp (<a aria-describedby="B28" class="usa-link" href="#B28">28</a>) using a probe radius of 1.4 Å. Conserved residues were mapped onto the structure using ConSurf (<a aria-describedby="B29" class="usa-link" href="#B29">29</a>).</p></section><section id="sec8"><h5 class="pmc_sec_title">Computational Docking Experiments</h5>
<p>PsDRR206 and two ligands were prepared in AutoDockTools4 (<a aria-describedby="B30" class="usa-link" href="#B30">30</a>) with default bonded and nonbonded parameters. One ligand approximated the coniferyl alcohol quinone methide radical (CA<sup>•</sup>), with C7 and C8 set to be trivalent, trigonal planar carbon atoms coplanar with the ring carbon and quinone oxygen atoms. A second ligand was the 8–8′ linked <em>bis</em>-quinone methide. Both <em>syn</em> and <em>anti</em> isomers, referring to the relationship between C8 and the methoxyl group, were used in the docking experiments, and rotation about the C7-C8 bond was not allowed in CA<sup>•</sup>, reflecting its partial double bond character in the resonance hybrid. Flexibility was allowed for the side chains of residues Asp<sup>40</sup>, Phe<sup>79</sup>, Tyr<sup>101</sup>, Ser<sup>111</sup>, Leu<sup>113</sup>, Ile<sup>161</sup>, Thr<sup>163</sup>, and Leu<sup>174</sup>. The grid box was centered upon the ligand with sufficient size to cover the ligand, and all flexible residues and a grid spacing of 0.375 Å. Autodock Vina (<a aria-describedby="B31" class="usa-link" href="#B31">31</a>) was used to perform the docking calculations.</p></section><section id="sec9"><h5 class="pmc_sec_title">Homology Modeling</h5>
<p>Homology models of AtDIR6 were built in SWISS-MODEL (<a aria-describedby="B32" class="usa-link" href="#B32">32</a>).</p></section></section></section></section><section id="sec10"><h2 class="pmc_sec_title">RESULTS</h2>
<p>Fungally challenged pea (<em>P. sativum</em>) pods have an overall non-host disease response that includes induction of a gene called <em>PsDRR206</em> (<em>Disease Resistance Response 206</em>), whose protein biochemical function until recently (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>) was unknown (<a aria-describedby="B33" class="usa-link" href="#B33">33</a><a aria-describedby="B34" class="usa-link" href="#B34">–</a><a aria-describedby="B35" class="usa-link" href="#B35">35</a>). PsDRR206 shares 54% sequence identity with AtDIR6 over the extent of the mature polypeptide. For its study <em>in vitro</em>, recombinant PsDRR206 was constitutively expressed using the strong CaMV <sup>35</sup>S promoter of pART17 (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>). Large scale (3 liters) suspension culture of transformed tomato (<em>Solanum peruvianum</em>) cells was obtained by gradually increasing the culture volume (starter volume, 40 ml) with fresh liquid medium for 4 weeks at weekly subculturing intervals. Proteins were isolated using a series of strong cationic column chromatographic steps (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>) with ∼400 μg of PsDRR206 protein obtained from the 3-liter culture.</p>
<p>Detailed biochemical characterization next established it to be a (+)-pinoresinol-forming DP (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>), and a homologue to our much earlier discovery of this protein type and its physiological function from <em>Forsythia</em> species (<a aria-describedby="B3" class="usa-link" href="#B3">3</a>). Thus we have now discovered DPs producing both pinoresinol antipodes, with the recent discovery of a (−)-pinoresinol-forming DP in <em>Arabidopsis</em> (AtDIR6) (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>, <a aria-describedby="B5" class="usa-link" href="#B5">5</a>). Interestingly, the function of AtDIR6 was also reported independently by others (<a aria-describedby="B6" class="usa-link" href="#B6">6</a>). Of the various DPs under investigation thus far, however, crystallization conditions have only been identified for PsDRR206, the focus of this investigation.</p>
<section id="sec11"><h3 class="pmc_sec_title"></h3>
<section id="sec12"><h4 class="pmc_sec_title"></h4>
<section id="sec13"><h5 class="pmc_sec_title">Overall Structure</h5>
<p>PsDRR206 structure was solved by molecular replacement and refined at 1.95 Å resolution (<a class="usa-link" href="#T1">Table 1</a>). The asymmetric unit contains two independent molecules related by a noncrystallographic 2-fold rotation axis and connected by a single salt bridge between Glu<sup>169</sup> and Arg<sup>173</sup>. Molecule A comprises residues 28–43, 65–105, 110–135, 138–166, and 169–184, whereas molecule B contains residues 28–42, 66–104, 109–135, 138–166, and 168–184 (see PDB code 4REV for details). Superposition of the two independent molecules gives a root mean square deviation (RMSD) of 0.4 Å for 124 matching Cα atoms. The structure of the PsDRR206 monomer (<a class="usa-link" href="#F2">Fig. 2</a><em>A</em>) can be best described as an eight-stranded antiparallel β-meander forming a β-barrel. The displacement (shear number) at the suture between β1 and β8, caused by twisting of the β-sheet, is ten residues. The gaps in the structure correspond to the seven N-terminal residues and loops between strands β1–β2, β3–β4, β5–β6a, and β7–β8. In all of these gaps, the electron density was either very weak or missing completely, such that the residues in the loops could not be fitted. Interestingly, the four missing loops are all at one end of the β-barrel, designated here as the top of the molecule (<a class="usa-link" href="#F2">Fig. 2</a>, <em>A</em> and <em>B</em>). <em>N</em>-Linked glycans assumed to be present at Asn<sup>50</sup> and Asn<sup>120</sup> in PsDRR206 based on their occurrence at the equivalent residues in AtDIR6 (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>) are also missing in the electron density map.</p>
<figure class="fig xbox font-sm" id="F2"><h6 class="obj_head">FIGURE 2.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 2." class="graphic" height="410" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1984/4340379/f95ef667461c/zbc0051505840002.jpg" width="693"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>Structure of the PsDRR206 monomer.</strong>
<em>A</em>, ribbon representation of the eight-stranded β-barrel rainbow-colored from the N terminus (<em>blue</em>) to the C terminus (<em>red</em>), showing the secondary structure labeling. <em>B</em>, the PsDRR206 monomer rotated 90° about a vertical axis and colored to show the two component β-sheets in <em>blue</em> (β-sheet 1) and <em>red</em> (β-sheet 2). The location of the domain-swapped strand β2′ from a neighboring PsDRR206 molecule making up the trimer is shown as a <em>transparent gray strand</em>. The two long loops between strands β2/β3 and β6/β7 are colored <em>green</em>.</p></figcaption></figure><p>Alternatively, the β-barrel can be viewed as two highly curved anti-parallel sheets formed by strands β2, β3, β4, and β5 (β-sheet 1) and by strands β6, β7, β8, and β1 (β-sheet 2) (<a class="usa-link" href="#F2">Fig. 2</a><em>B</em>), respectively. Moreover, strand β6 is divided into two shorter strands such that the first part of the strand (β6a) forms the outer strand of β-sheet 2, whereas the C-terminal end (β6b) adds a fifth strand to β-sheet 1. The bifurcation of β6 occurs at a β-bulge at residues Val<sup>145</sup>-Thr<sup>146</sup>; a similar bulge is present at the equivalent position in AOC.</p>
<p>Analysis of the crystal symmetry shows that PsDRR206 forms a tightly packed trimer, with the trimer axis being orientated along the 3-fold axis of the H32 space group (<a class="usa-link" href="#F3">Fig. 3</a>). Nevertheless, formation of the trimer buries almost 2500 Å<sup>2</sup> of surface per monomer and over 30% of the total surface of each monomer. Calculation of the electrostatic surface of the PsDRR206 monomer shows that the outer surface is hydrophilic (<em>left side</em> of <a class="usa-link" href="#F4">Fig. 4</a><em>A</em>), whereas the inner surface involved in trimer interface formation is predominantly hydrophobic (<a class="usa-link" href="#F4">Fig. 4</a><em>A, right side</em>, and <em>B</em>). Along with a large number of hydrophobic contacts at the trimer interface, the monomers are also held together by six hydrogen bonding interactions (three for each monomer-monomer pair), and two electrostatic interactions (not shown). Inspection of the trimer shows that the loop between the first two strands is involved in a domain-swapping event (<a class="usa-link" href="#F3">Fig. 3</a><em>B</em>), wherein the loop does not simply reverse direction and become the second β-strand in the same monomer. Instead, it traverses the interface into the neighboring monomer, such that the N-terminal half of the first strand β2 is inserted into the second monomer next to its strand β2 (<a class="usa-link" href="#F3">Fig. 3</a>, <em>A</em> and <em>B</em>), adding a sixth strand to β-sheet 1. The electron density in this region clearly shows the N-terminal extension of this strand (<a class="usa-link" href="#F3">Fig. 3</a><em>C</em>). This extension of strand β2 provides an additional nine hydrogen bonds, which include seven main chain interactions with strand β2 from the second monomer and two interactions with side chain atoms from the β2-β3 loop (<a class="usa-link" href="#F3">Fig. 3</a><em>C</em>).</p>
<figure class="fig xbox font-sm" id="F3"><h6 class="obj_head">FIGURE 3.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4340379_zbc0051505840003.jpg" target="_blank"><img alt="FIGURE 3." class="graphic zoom-in" height="618" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1984/4340379/ac17581abc28/zbc0051505840003.jpg" width="745"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>Structure of the PsDRR206 trimer.</strong>
<em>A</em>, the trimer viewed down the 3-fold <em>c</em>-axis of the H32 space group. The <em>a</em>- and <em>b</em>-axes are indicated. <em>B</em>, the trimer rotated 90° about the <em>a</em>-axis. One of the extended β2 strands (<em>blue</em>) traverses the trimer interface into neighboring monomer (<em>green</em>), and this region is highlighted by the <em>black oval. C</em>, final 2<em>F</em><sub>o</sub> − <em>F</em><sub>c</sub> electron density, contoured at 1.0 σ, in the vicinity of the domain-swapped β2 strand from the <em>blue</em> monomer. This extended β2 strand makes a typical anti-parallel hydrogen bonding network with the β2 strand from the <em>green</em> monomer, along with interactions with two residues from the omega loop between strands β2 and β3 (Gln<sup>93</sup> and Arg<sup>96</sup>). Some residues in the <em>blue</em> and <em>green</em> β2 strands are indicated.</p></figcaption></figure><figure class="fig xbox font-sm" id="F4"><h6 class="obj_head">FIGURE 4.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4340379_zbc0051505840004.jpg" target="_blank"><img alt="FIGURE 4." class="graphic zoom-in" height="682" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1984/4340379/5aea5744c0e9/zbc0051505840004.jpg" width="745"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>Surface representation of PsDRR206.</strong>
<em>A</em>, the outer (solvent exposed, <em>left panel</em>) and the inner (buried in trimer formation, <em>right panel</em>) electrostatic surfaces of the monomer. The surface potentials range from −4 kT/e (<em>red</em>) to +4 kT/e (<em>blue</em>). <em>B</em>, the electrostatic surface of two of the monomers, X and Y, with the third monomer (Z) indicated as a semi-transparent green ribbon. The inner surface of the Z monomer is represented by the <em>right panel</em> in <em>A</em>, rotated 180° about a vertical axis relative to the orientation of the <em>green ribbon. C</em>, the electrostatic surface of one of the monomers orientated to view down into the putative active site. The other two monomers making up the trimer are colored <em>red</em> and <em>blue</em>, and the orientation of the trimer is a 60° rotation about the <em>a</em>-axis relative to <em>A</em>.</p></figcaption></figure><p>At the top of each barrel and slightly toward the outside of the trimer is a substantial pocket lined mostly with hydrophobic residues (<a class="usa-link" href="#F4">Figs. 4</a><em>C</em> and <a class="usa-link" href="#F5">5</a>, <em>A</em> and <em>B</em>). Several conserved residues cluster around this pocket including strictly conserved charged polar residues Asp<sup>40</sup> and Arg<sup>141</sup> (<a class="usa-link" href="#F5">Fig. 5</a><em>A</em>). This narrow pocket is approximately 12 Å deep, 12 Å wide, and 5 Å across. The solvent excluded molecular surface area (Connolly surface) of the pocket is 339 Å<sup>2</sup>, and its volume is 463 Å<sup>3</sup>.</p>
<figure class="fig xbox font-sm" id="F5"><h6 class="obj_head">FIGURE 5.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 5." class="graphic" height="883" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1984/4340379/cbe829009db6/zbc0051505840005.jpg" width="600"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>Residues lining the interior of the active site pocket.</strong>
<em>A</em>, several of these residues are conserved, although in only one or the other but not both of the groups of known (+)- or (−)-pinoresinol-forming DPs as indicated in <a class="usa-link" href="#F1">Fig. 1</a><em>B</em>, suggesting a role in determining substrate orientation. The omega loop is also labeled to indicate proximity to the back side of the active site. <em>B</em>, surface rendering of the active site pocket and surrounding area from the same perspective as <em>A</em>, with residues shown as <em>sticks</em> in <em>A</em> now shown as <em>colored spheres</em> visible under the transparent surface.</p></figcaption></figure><p>Adjacent to Asp<sup>40</sup> and on the opposite side of the β-sheet is found His<sup>39</sup>, one of three conserved residues in an unusual cluster of highly conserved residues that also includes Thr<sup>84</sup> and Ser<sup>91</sup>. The latter two residues occur in an omega loop between strands β2 and β3 that folds back upon the exterior of the barrel (<a class="usa-link" href="#F2">Figs. 2</a><em>B</em> and <a class="usa-link" href="#F5">5</a><em>A</em>). An omega loop is a nonregular secondary structure element characterized by a loop of varying structure with its N and C termini located on adjacent secondary structure elements, typically β-strands, close together in space (<a aria-describedby="B36" class="usa-link" href="#B36">36</a>).</p></section><section id="sec14"><h5 class="pmc_sec_title">Comparison of PsDRR206 to AOC</h5>
<p>A Dali search of the PDB (<a aria-describedby="B37" class="usa-link" href="#B37">37</a>) identified several β-barrel proteins as being closely related to PsDRR206, including AOC (the first nine hits with Z scores ranging from 12.1 to 11.5), followed by five proteins of unknown function (PDB codes 2Q03, 3G7G, 2OOJ, 3C5O, and 4PUX) with Z scores between 10.5 and 7.8. Unlike AOC, these latter proteins do not form trimeric structures. The Dali search also identified numerous structures from the avidin/streptavidin fold family, but all with Z scores less than 7.0. Like AOC, these are eight-stranded β-barrel proteins with shear number 10; however, they meander in the opposite direction around the barrel.</p>
<p>A structure-based sequence alignment of PsDRR206 and the highest scoring Dali hit, <em>Physcomitrella patens</em> AOC (PpAOC2; PDB code 4H69) (<a aria-describedby="B13" class="usa-link" href="#B13">13</a>) with bound inhibitor, identified 109 residues within spans of 151 and 164 residues in the PsDRR206 and AOC crystal structures, respectively, that could be aligned with 2.2 Å RMSD for the backbone atoms (<a class="usa-link" href="#F6">Fig. 6</a>, <em>A</em> and <em>B</em>). Similar alignments can also be generated with AtAOC2 structures (<a aria-describedby="B12" class="usa-link" href="#B12">12</a>). Most of the aligned residues constitute the common core barrel structure of both proteins, whereas the unaligned portions are in loops linking β-strands. Many of these loops also have substantial insertions and deletions, relative to their counterparts in the other structure. However, the prominent pocket or cleft on each PsDRR206 monomer aligns well with its counterpart in the AOC structure. The shapes of the PsDRR206 and AOC pockets are somewhat similar (<a class="usa-link" href="#F6">Fig. 6</a><em>A</em>), and their volumes are also quite similar. The size and length of AOC ligands such as vernolic acid (PDB code 2DIO) are comparable with pinoresinol (<strong>2</strong>).</p>
<figure class="fig xbox font-sm" id="F6"><h6 class="obj_head">FIGURE 6.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 6." class="graphic" height="1033" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1984/4340379/f1badcb451dc/zbc0051505840006.jpg" width="600"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>PsDRR206 comparison with AOC.</strong>
<em>A</em>, PsDRR206 (<em>magenta</em>) superposition with <em>P. patens</em> AOC (4H69, <em>chartreuse</em>; with ligand, <em>spheres</em>), with RMSD ∼2.1–2.2 Å over 102 residues. <em>B</em>, aligned (pairwise Dali) residues only, Cα trace, for one monomer. All 102 residues are in the core β-barrel.</p></figcaption></figure></section><section id="sec15"><h5 class="pmc_sec_title">Computational Docking Studies</h5>
<p>Because the putative substrate for PsDRR206 is a reactive radical species, experimental investigation of the protein-substrate complex is not straightforward. Accordingly, computational docking studies were conducted using the PsDRR206 x-ray structure and models of both the coniferyl alcohol quinone methide radical (CA<sup>•</sup>) and the 8–8′ <em>bis</em>-quinone methide, with the latter being the putative intermediate in pinoresinol (<strong>2</strong>) formation following (CA<sup>•</sup>) radical coupling prior to cyclization of the furan rings (<a class="usa-link" href="#F1">Fig. 1</a><em>A</em>). These docking studies suggest that the binding site can accommodate at least one CA<sup>•</sup> (<a class="usa-link" href="#F7">Fig. 7</a><em>A</em>) and that its dimensions are appropriate for the long axis to span the distance from Arg<sup>141</sup> to Asp<sup>40</sup>, with the quinone oxygen and the 9-OH potentially interacting favorably with these charged residues.</p>
<figure class="fig xbox font-sm" id="F7"><h6 class="obj_head">FIGURE 7.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 7." class="graphic" height="324" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1984/4340379/0dd9b4b10542/zbc0051505840007.jpg" width="700"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F7/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>Docked models of substrates en route to (+)-pinoresinol (2a), in the putative active site of PsDRR206.</strong>
<em>A,</em> coniferyl alcohol radical (CA<sup>•</sup>) in the <em>syn</em> configuration bound such that the <em>re</em> face is exposed, the orientation that would lead to coupling with the <em>re</em> face of another CA<sup>•</sup> to give the <em>R</em>,<em>R</em>-stereoisomer of 8–8′ <em>bis</em>-quinone methide, the putative initial product from coupling of two coniferyl alcohol radicals as shown in <em>B</em>. In this particular model of the <em>bis</em>-quinone methide, the 3 and 3′-methoxy groups are <em>anti</em> with respect to C8 and C8′; models with <em>syn</em> regiochemistry were also docked. The torsion angle about the 8–8′ bond was restricted to values in which cyclization to form the furan rings of pinoresinol (<strong>2</strong>) would produce the correct configuration at C7 and C7′. PsDRR206 side chains allowed to be flexible in the docking simulation are labeled and shown as <em>sticks</em>.</p></figcaption></figure></section></section></section></section><section id="sec16"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p>The study herein obtained the (+)-pinoresinol-forming DP (PsDRR206) in crystalline form, and at 1.95 Å resolution. In this regard, determination of the PsDRR206 three-dimensional structure, and its analysis suggests that the corresponding trimer has three identical substrate binding sites that are spatially distributed so that intermolecular coupling apparently cannot occur between two distinct substrate binding sites in the trimer model. Furthermore, this trimer observation is in agreement with our earlier studies (<a aria-describedby="B3" class="usa-link" href="#B3">3</a>), where the <em>Forsythia</em> DP was considered as a ∼78-kDa trimer based on gel filtration analyses, in contrast to subsequent mass spectrometric data obtained, which had suggested a dimeric protein (<a aria-describedby="B38" class="usa-link" href="#B38">38</a>).</p>
<p>As indicated above, we also established that PsDRR206 is an eight-stranded β-barrel with the same β-meander topology and remarkably similar trimeric quaternary structure as AOC. Together with the previously mentioned low (∼17% identity), but likely meaningful, sequence similarity between DP and AOC protein families, this suggests a possible common ancestor for these functionally distinct proteins.</p>
<p>Although there are striking structural similarities between PsDRR206 and AOC, it remains to be seen whether there are mechanistic similarities as well. Studies of AOC with and without a bound inhibitor suggested that conformational changes in the binding pocket upon formation of a tight enzyme-substrate complex precede the catalytic cycle and that steric restrictions in the active site determine the stereoselectivity of the cyclization reaction (<a aria-describedby="B12" class="usa-link" href="#B12">12</a>, <a aria-describedby="B13" class="usa-link" href="#B13">13</a>). Both of these characteristics could be seen to be important in a protein guiding stereoselective phenoxy radical coupling. Conceivably, the trimeric β-barrel DP/AOC scaffold is well suited for these roles.</p>
<section id="sec17"><h3 class="pmc_sec_title"></h3>
<section id="sec18"><h4 class="pmc_sec_title"></h4>
<section id="sec19"><h5 class="pmc_sec_title">DP Surface Features and Patterns of Conserved Residues</h5>
<p>We hypothesize that the prominent outward facing pocket near the top of each barrel is the substrate binding (active) site located within the internal cavity of the barrel, based upon comparison with the structures and similar dimensions of <em>P. patens</em> AOC1 and AOC2 with bound substrate analog (PDB codes 4H69 and 4H6B (<a aria-describedby="B13" class="usa-link" href="#B13">13</a>)). We further propose that this is the binding site for one or possibly two coniferyl alcohol phenoxy radicals (CA<sup>•</sup>) in PsDRR206. Conserved residues Thr<sup>163</sup> and Phe<sup>172</sup> line the edge of the putative binding pocket (<a class="usa-link" href="#F8">Fig. 8</a>, <em>A–D</em>) (<a aria-describedby="B29" class="usa-link" href="#B29">29</a>), and disordered loop residues for which electron density is missing occur around the edge of the pocket. Two polar residues, Asp<sup>40</sup> and Arg<sup>141</sup>, conserved in all pinoresinol-forming DPs but not AOCs, are also located at either end of the pocket (<a class="usa-link" href="#F5">Figs. 5</a><em>A</em> and <a class="usa-link" href="#F7">7</a><em>A</em>). The span between them is ∼12–14 Å, suitable for a single <em>trans</em> coniferyl alcohol molecule or quinone methide radical, whose lengths are ∼8–10 Å. Residues in the interior of the pocket are less conserved, however, possibly because they help determine substrate specificity.</p>
<figure class="fig xbox font-sm" id="F8"><h6 class="obj_head">FIGURE 8.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4340379_zbc0051505840008.jpg" target="_blank"><img alt="FIGURE 8." class="graphic zoom-in" height="660" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1984/4340379/00152b1ab70c/zbc0051505840008.jpg" width="745"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F8/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>Surface features and conserved residues.</strong>
<em>A</em>, top view of accessible surface showing that putative active site pockets (entrance indicated by <em>arrows</em>, residues colored by atom type) are isolated from each other in the trimer. The omega loop is also indicated. <em>B</em>, top view (same as in <em>A</em>) generated with ConSurf (<a aria-describedby="B29" class="usa-link" href="#B29">29</a>) showing conserved residue Phe<sup>73</sup> in relation to the active site and omega loop. Conserved residues are shown in <em>magenta</em>. The sequence alignment input to ConSurf contained PsDRR206 and 16 dirigent proteins and their homologues from <em>Arabidopsis. C</em>, view directly into the active site, at ∼45° away from the top view shown in <em>A</em> and <em>B. D</em>, same view as in <em>C</em>, but with residues colored according to ConSurf. The network of conserved residues formed by the omega loop and by residues on the back side of the active site pocket is visible.</p></figcaption></figure><p>In PsDRR206, but not AOC, an omega loop consisting of approximately eight residues (Leu<sup>85</sup>–Lys<sup>92</sup>) occurs at the bottom of the trimer between β2 and β3 and folds back upon the outside of the barrel (<a class="usa-link" href="#F5">Fig. 5</a><em>A</em>). A cluster of three highly conserved polar residues (His<sup>39</sup>, Thr<sup>84</sup>, and Ser<sup>91</sup>) also occur (<a class="usa-link" href="#F8">Fig. 8</a><em>D</em>) where this loop contacts the exterior surface of the barrel. These three residues are conserved in all characterized (+)- and (−)-pinoresinol-forming DPs (<a class="usa-link" href="#F1">Fig. 1</a><em>B</em>), whereas they are not conserved in AOCs. The cluster adjoins the active site pocket, and indeed His<sup>39</sup> neighbors conserved residue Asp<sup>40</sup> in the putative active site region, although on the opposite side of the strand in the β-barrel (<a class="usa-link" href="#F5">Fig. 5</a><em>A</em>). This proximity of conserved residues thus provisionally suggests the omega loop may be the site of an interaction that modulates either substrate orientation or binding in the active site. As indicated above, because the proposed pockets point outwards relative to the trimer axis, a substrate in one pocket is unlikely to be able to couple with those in other pockets elsewhere in the trimer.</p>
<p>Another notable residue conserved in DPs, although not in AOCs, is Phe<sup>73</sup> (Tyr in FiDIR and some other DPs), which lies near its symmetry mates on the exterior surface of the barrel at its topmost point and near the rim of the putative active site. This positioning suggests it may serve as a hydrophobic anchor point (<a class="usa-link" href="#F5">Figs. 5</a> and <a class="usa-link" href="#F8">8</a> (<em>A</em> and <em>B</em>)). Its conservation and location on the surface suggest it is maintained there by selective pressure. Because they do not appear likely to be necessary for stabilization of the structure, these conserved surface-exposed hydrophobic side chains might instead indicate interaction site(s) for an oxidase, other dirigent proteins, or perhaps even a surface such as that of an actively growing lignin polymer. Several other conserved residues in DPs are also found buried in the trimer interface, supporting the idea that the trimer is the functional form of this and probably other DPs.</p></section><section id="sec20"><h5 class="pmc_sec_title">Comparison of Amino Acid Sequences for DPs with Opposite Stereoselectivities and Homology Models of AtDIR6</h5>
<p>In our earlier work (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>) and that of others (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>), homology models of ScDIR and AtDIR6, (+)- and (−)-pinoresinol-forming DPs, respectively, were made using the very distantly related AOC structures as potential templates. The x-ray structure of PsDRR206, however, now allows construction of accurate homology models of AtDIR6 (∼50% amino acid identity to PsDRR206).</p>
<p>Modeling of missing loop residues permits consideration of loop regions surrounding the putative active site where weak or missing electron density prevented fitting of the structure. All of the known (+)-pinoresinol-forming DPs have six more residues in the longest of these loops, between strands β1 and β2 (Leu42 to Gly74 of PsDRR206), than the (−)-pinoresinol DPs. This difference is conserved in various plant species (<a class="usa-link" href="#F1">Fig. 1</a><em>B</em>). This correlation with stereoselectivity may indicate a role for this loop in binding or orientation of the substrate, for example by acting as a “lid” that closes on the active site as substrates bind.</p>
<p>Furthermore, the precise residues essential for distinct stereoselectivities in PsDRR206 and AtDIR6 were of considerable interest, because previously this was only narrowed down to the Region B in the DPs (<a class="usa-link" href="#F1">Fig. 1</a><em>B</em>) (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>). That is, using a region swapping approach, we established regions in the DPs that affected different stereoselectivities resulting in formation of opposite antipodes of pinoresinol (<strong>2</strong>) (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>).</p>
<p>Our recent characterization of pea (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>) and flax (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>) DPs, however, now expands the number of DPs of known function to eleven in seven different plant species, namely <em>F. intermedia</em>, <em>T. plicata</em>, <em>A. thaliana</em>, <em>S. chinensis</em>, <em>Piper regnellii</em>, <em>P. sativum</em>, and <em>L. usitatissimum</em>. Among them, seven DPs (FiDIR, TpDIR5, TpDIR8, ScDIR, PrDIR, PsDRR206, and LuDIR1) afford (+)-pinoresinol (<strong>2a</strong>), whereas four others (AtDIR5, AtDIR6, LuDIR5, and LuDIR6) generate (−)-pinoresinol (<strong>2b</strong>). Interestingly, flax has both (+)- and (−)-pinoresinol-forming DPs (LuDIR1 <em>versus</em> LuDIR5 and LuDIR6<sup>3</sup>, respectively).</p>
<p>Although previously we had noted 14 residues differentially conserved in (+)- and (−)-pinoresinol-forming DPs (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>), the recent identification of pea and flax DPs enabled us to narrow down candidate residues further to six. These now include Gly<sup>95</sup>, Leu<sup>113</sup>, Phe<sup>115</sup>, Phe<sup>117</sup>, Phe<sup>130</sup>, and Ile/Leu<sup>161</sup> in the (+)-pinoresinol-forming DPs (numbering is based on PsDRR206), as well as Ala<sup>98</sup>, Phe<sup>116</sup>, Tyr<sup>118</sup>, Leu<sup>120</sup>, Ile<sup>133</sup>, and Phe<sup>164</sup> in the (−)-pinoresinol-forming DPs; the numbering is based on AtDIR6 (<a class="usa-link" href="#F1">Fig. 1</a><em>B</em>, with residues marked with green dots). Interestingly, except for Ile<sup>161</sup> of PsDRR206 (and Phe<sup>164</sup> in the (−)-pinoresinol-forming DP), all of these residues are located in Region-B (Lys<sup>90</sup>–Leu<sup>138</sup> of AtDIR6) that was previously found responsible for altered stereoselectivity by region swapping (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>). Furthermore, residues Leu<sup>113</sup> and Phe<sup>115</sup> of PsDRR206 (Phe<sup>116</sup> and Tyr<sup>118</sup> of AtDIR6) are located within the putative active site region (<a class="usa-link" href="#F5">Fig. 5</a>, <em>A</em> and <em>B</em>), with both residues being on the β4 strand and having their side chains positioned at distances less than 10 Å to the conserved polar residues Asp<sup>40</sup> and Arg<sup>141</sup>; the latter provisionally appears to be positioned in a way that could determine whether the <em>si</em> or <em>re</em> face of bound CA<sup>•</sup> is exposed and available for coupling. Accordingly, in the future, site-directed mutagenesis will examine whether one or more of the six residues determine distinct stereoselectivities.</p></section><section id="sec21"><h5 class="pmc_sec_title">Homology Modeling of the Putative Active Site in (−)-Pinoresinol-forming AtDIR6</h5>
<p>The backbone RMSD between PsDRR206 and the published AtDIR6 homology model of Pickel <em>et al.</em> (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>) is ∼5 Å, varying depending on which residues are superimposed. Three β-strands are out of register by two residues each with respect to their counterparts in the experimentally determined x-ray structure. Although the same side chains occupy the core of the β-barrel, they are misplaced because of incorrect alignment of neighboring strands. Our own models (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>) were similarly poor. Such results are not unexpected given the low sequence similarity (∼17% identity in the core β-barrel) between AtDIR6 and AtAOC2, the template used to generate the homology model.</p>
<p>Nonetheless, the model of Pickel <em>et al.</em> (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>) was sufficient to obtain a molecular replacement solution for the PsDRR206 crystal structure. This is because the core β-barrel of the template, AtAOC2 (PDB code 2BRJ) aligns very well with that of PsDRR206 (RMSD 3.2 Å over 113 residues). The structure has a RMSD 2.2 Å with <em>P. patens</em> AOC (PpAOC2, PDB code 4H69; <a class="usa-link" href="#F6">Fig. 6</a><em>B</em>, which itself has a 1 Å RMSD alignment with AtAOC2). More specifically, because side chain atoms are not typically used in finding structure solutions by molecular replacement, the misalignment of several β-strands did not matter in this instance so long as positions of backbone atoms in the barrel are not perturbed by the misalignment.</p>
<p>Using the PsDRR206 x-ray structure as a template, a new homology model of AtDIR6 was thus constructed to investigate the basis for differing stereoselectivities in pinoresinol-forming DPs. (<a class="usa-link" href="#F9">Fig. 9</a>). Five of the six residues that are differentially conserved among (+)- and (−)-pinoresinol-forming DPs from a range of plant species cluster together in the deep interior of the putative active site pocket. A seventh active site residue (Val<sup>176</sup> in PsDRR206), not included in the original set because it is substituted with Thr in one (+)-pinoresinol-forming DP, becomes Met in (−)-pinoresinol-forming DPs. These residues are located opposite each another on either side of the pocket, in the center of one or the other of the two highly curved antiparallel β-sheets that form the β-barrel. Other active site residues nearer the mouth of the pocket are strictly conserved, such as Asp<sup>40</sup>, Arg<sup>141</sup>, and Thr<sup>163</sup>. We propose that the two different sets of differentially conserved residues force CA<sup>•</sup> to bind such that only the <em>si</em> (for (+)-pinoresinol (<strong>2a</strong>)) or the <em>re</em> (for (−)-pinoresinol (<strong>2b</strong>)) face is exposed and available to couple with a second CA<sup>•</sup> (or possibly CA), which itself must be oriented properly, perhaps by residues in the β1-β2 loop. Uncovering specific details of the mechanism, such as the actual roles of Asp<sup>40</sup> and Arg<sup>141</sup> and the orientation of each substrate, will likely require structural characterization of DPs with substrate or product analogues. Missing residues in loops surrounding the putative active site at the top of the β-barrel were also modeled to establish the range through which their motion can occur assuming the core barrel structure remains intact (<a class="usa-link" href="#F10">Fig. 10</a>). In particular, the 23 missing residues in the β1-β2 loop span a distance of ∼32 Å, conceivably providing ample potential to adopt a range of conformations.</p>
<figure class="fig xbox font-sm" id="F9"><h6 class="obj_head">FIGURE 9.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 9." class="graphic" height="494" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1984/4340379/a674d362a16b/zbc0051505840009.jpg" width="693"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F9/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>AtDIR6 homology model (<em>lavender</em>) generated from the PsDRR206 x-ray structure (<em>green</em>) on which it is superimposed.</strong> Differentially conserved active site side chains from (+)- and (−)-pinoresinol-forming DPs are shown and labeled in terms of the PsDRR206 residue type and number then the equivalent AtDIR6 residue type. (Valine 176 is substituted with threonine in the (+)-pinoresinol-forming DP ScDIR (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>).)</p></figcaption></figure><figure class="fig xbox font-sm" id="F10"><h6 class="obj_head">FIGURE 10.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="FIGURE 10." class="graphic" height="742" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1984/4340379/ba16574d648b/zbc0051505840010.jpg" width="800"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F10/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>PsDRR206 structure with residues in missing loops built in as disordered polypeptide chains (<em>black</em>) and subjected to backbone torsion minimization.</strong> Putative <em>N</em>-glycosylation sites (Asn<sup>50</sup> and Asn<sup>120</sup>) are indicated with <em>red asterisks</em>.</p></figcaption></figure></section><section id="sec22"><h5 class="pmc_sec_title">Mechanistic Considerations</h5>
<p>The trimeric DP structure shows that each active site is isolated from the others and will not allow two substrates bound at two different active sites to interact. In light of this finding, the most likely mechanism is that a single active site binds two substrates that combine to form product. The putative PsDRR206 binding pocket would thus bind two CA<sup>•</sup> in an orientation allowing <em>si-si</em> coupling to form the (+)-pinoresinol antipode (<strong>2a</strong>), whereas the homologue AtDIR6 would afford the corresponding (−)-enantiomer (<strong>2b</strong>) through <em>re-re</em> coupling. However, a variation in which one CA<sup>•</sup> and one coniferyl alcohol (<strong>1</strong>) bind and react cannot be ruled out.</p>
<p>Docking studies (<a class="usa-link" href="#F7">Fig. 7</a>) suggested that two CA<sup>•</sup> substrates could bind in the active site, as could an 8–8′ <em>bis</em>-quinone methide (<a class="usa-link" href="#F1">Fig. 1</a><em>A</em>). Although two positions of the methoxy group (<em>syn</em> or <em>anti</em>) with respect to C8 are possible in the quinone methide, discrimination of <em>si</em> and <em>re</em> faces that determine the stereochemistry at C8/8′ in pinoresinol (<strong>2</strong>) does not require the methoxy group to be in one position or the other. After the <em>bis</em>-quinone methide is formed, each of the 9/9′ -OH groups could nucleophilically attack the C7/C7′ quinone methide moieties to form the pinoresinol furanofuran product. Both cyclizations apparently occur stereospecifically, with no rotation about C7-C8, because this would instead lead to 7-epi-pinoresinol (or <em>bis</em> 7-epi-pinoresinol if a 180° rotation about both C7-C8 bonds occurred). This suggests that the <em>bis</em>-quinone methide intermediate is not released from the active site and that both substrates are bound such that both 8–8′ coupling and C7-O9′ and C7′-O9 ring closure occur in the bound state.</p>
<p>Pinoresinol (<strong>2</strong>) formation probably concludes with protonation of the quinone oxygen after rearomatization to form a phenol. Residue Asp<sup>40</sup> in the putative binding pocket could be the source of this proton. Alternatively, if CA<sup>•</sup> is bound in the opposite lengthwise orientation, the positively charged side chain of Arg<sup>141</sup> could stabilize the quinone form of the phenolic oxygen in CA<sup>•</sup>, helping to localize the radical at C8 and favor coupling there. Indeed, binding in either orientation potentially provides ample opportunity for π-stacking interactions with aromatic residues in the active site (<a class="usa-link" href="#F7">Fig. 7</a><em>B</em>). Asp<sup>40</sup> and Arg<sup>141</sup> are conserved in all DPs, whereas AOC does not have the corresponding residues, suggesting that they are required for substrate binding and activity specific to DPs, rather than for structural stability.</p>
<p>The 2-fold rotational symmetry of the pinoresinol molecule may have helped shape previous hypotheses (<a aria-describedby="B10" class="usa-link" href="#B10">10</a>, <a aria-describedby="B11" class="usa-link" href="#B11">11</a>) for the mechanism of its formation involving a DP dimer with a single CA<sup>•</sup> substrate bound to each monomer. The structure presented here is not consistent with such a mechanism, without invocation of a substrate-induced head to head joining of two trimers, for which there is currently no evidence.</p>
<p>Interestingly, in addition to stereoselective control over monolignol coupling in lignan biosynthesis, another study has also reported DP domain involvement in lignification in the root Casparian strip (<a aria-describedby="B39" class="usa-link" href="#B39">39</a>). Although this is in conceptual agreement with our earlier considerations of DPs and proteins harboring dirigent (substrate binding) sites/domains for lignan biosynthesis and lignification (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>, <a aria-describedby="B40" class="usa-link" href="#B40">40</a>), respectively, it must be emphasized that most DPs still await discovery of their true physiological functions, as well as in terms of chemical structural and <em>in vitro</em> function analyses. In due course, we will obtain definitive mechanistic models for substrate binding, orientation, coupling, intramolecular cyclization, and product release.</p></section></section></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Marlisa Burton, Terri Lincoln, Mia Ryckman, and Paul Ziegler for assistance in DP purification. Use of the Stanford Synchrotron Radiation Lightsource, SLAC National Accelerator Laboratory, is supported by the U.S. Dept. of Energy, Office of Science, Basic Energy Sciences (BES), under Contract No. DE-AC02–76SF00515. The SSRL Structural Molecular Biology Program is supported by the DOE Office of Biological and Environmental Research (BER), and by the National Institutes of Health, National Institute of General Medical Sciences (including P41GM103393). A portion of the work was performed in the Environmental Molecular Sciences Laboratory (EMSL), a national scientific user facility sponsored by the Dept. of Energy's BER and located at Pacific Northwest National Laboratory (PNNL), Richland, WA.</p></section><section class="fn-group" id="fn-group1"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="FN1">
<sup>*</sup><p class="display-inline">This work was supported by National Science Foundation Grant MCB-1052557, Chemical Sciences, Geosciences and Biosciences Division, Department of Energy Office of Basic Energy Sciences Grant DE-FG-0397ER20259, and the G. Thomas and Anita Hargrove Center for Plant Genomic Research.</p>
</div>
<div class="fn p" id="FN3">
<sup>3</sup><p class="display-inline">D. S. Dalisay, K. W. Kim, C. Lee, H. Yang, O. Rübel, B. P. Bowen, L. B. Davin, and N. G. Lewis, submitted for publication.</p>
</div>
</div></section><section class="fn-group" id="fn-group2"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="FN2">
<sup>2</sup><div class="display-inline p">The abbreviations used are:
<dl class="def-list">
<dt id="G1">DP</dt>
<dd><div class="p">dirigent protein</div></dd>
<dt id="G2">AOC</dt>
<dd><div class="p">allene oxide cyclase</div></dd>
<dt id="G3">CA<sup>•</sup>
</dt>
<dd><div class="p">coniferyl alcohol derived free radical</div></dd>
<dt id="G4">PDB</dt>
<dd><div class="p">Protein Data Bank</div></dd>
<dt id="G5">RMSD</dt>
<dd><div class="p">root mean square deviation.</div></dd>
</dl>
</div>
</div></div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>
Vassão D. G., Kim K.-W., Davin L. B., Lewis N. G. (2010) Lignans (neolignans) and allyl/propenyl phenols: biogenesis, structural biology, and biological/human health considerations. In Comprehensive Natural Products Chemistry II (Mander L., Lui H.-W., eds) pp. 815–928, Elsevier, Oxford, UK</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20Natural%20Products%20Chemistry%20II&amp;author=D.%20G.%20Vass%C3%A3o&amp;author=K.-W.%20Kim&amp;author=L.%20B.%20Davin&amp;author=N.%20G.%20Lewis&amp;publication_year=2010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>
Davin L. B., Jourdes M., Patten A. M., Kim K.-W., Vassão D. G., Lewis N. G. (2008) Dissection of lignin macromolecular configuration and assembly: comparison to related biochemical processes in allyl/propenyl phenol and lignan biosynthesis. Nat. Prod. Rep. 25, 1015–1090</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1039/b510386j" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19030603/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Prod.%20Rep&amp;title=Dissection%20of%20lignin%20macromolecular%20configuration%20and%20assembly:%20comparison%20to%20related%20biochemical%20processes%20in%20allyl/propenyl%20phenol%20and%20lignan%20biosynthesis&amp;author=L.%20B.%20Davin&amp;author=M.%20Jourdes&amp;author=A.%20M.%20Patten&amp;author=K.-W.%20Kim&amp;author=D.%20G.%20Vass%C3%A3o&amp;volume=25&amp;publication_year=2008&amp;pages=1015-1090&amp;pmid=19030603&amp;doi=10.1039/b510386j&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>
Davin L. B., Wang H.-B., Crowell A. L., Bedgar D. L., Martin D. M., Sarkanen S., Lewis N. G. (1997) Stereoselective bimolecular phenoxy radical coupling by an auxiliary (dirigent) protein without an active center. Science
275, 362–366</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.275.5298.362" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8994027/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Stereoselective%20bimolecular%20phenoxy%20radical%20coupling%20by%20an%20auxiliary%20(dirigent)%20protein%20without%20an%20active%20center&amp;author=L.%20B.%20Davin&amp;author=H.-B.%20Wang&amp;author=A.%20L.%20Crowell&amp;author=D.%20L.%20Bedgar&amp;author=D.%20M.%20Martin&amp;volume=275&amp;publication_year=1997&amp;pages=362-366&amp;pmid=8994027&amp;doi=10.1126/science.275.5298.362&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>
Kim M. K., Jeon J.-H., Fujita M., Davin L. B., Lewis N. G. (2002) The western red cedar (<em>Thuja plicata</em>) 8–8′ DIRIGENT family displays diverse expression patterns and conserved monolignol coupling specificity. Plant Mol. Biol. 49, 199–214</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1023/a:1014940930703" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11999375/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol.%20Biol&amp;title=The%20western%20red%20cedar%20(Thuja%20plicata)%208%E2%80%938%E2%80%B2%20DIRIGENT%20family%20displays%20diverse%20expression%20patterns%20and%20conserved%20monolignol%20coupling%20specificity&amp;author=M.%20K.%20Kim&amp;author=J.-H.%20Jeon&amp;author=M.%20Fujita&amp;author=L.%20B.%20Davin&amp;author=N.%20G.%20Lewis&amp;volume=49&amp;publication_year=2002&amp;pages=199-214&amp;pmid=11999375&amp;doi=10.1023/a:1014940930703&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>
Kim K.-W., Moinuddin S. G., Atwell K. M., Costa M. A., Davin L. B., Lewis N. G. (2012) Opposite stereoselectivities of dirigent proteins in <em>Arabidopsis</em> and <em>Schizandra</em> species. J. Biol. Chem. 287, 33957–33972</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M112.387423" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3464507/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22854967/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&amp;title=Opposite%20stereoselectivities%20of%20dirigent%20proteins%20in%20Arabidopsis%20and%20Schizandra%20species&amp;author=K.-W.%20Kim&amp;author=S.%20G.%20Moinuddin&amp;author=K.%20M.%20Atwell&amp;author=M.%20A.%20Costa&amp;author=L.%20B.%20Davin&amp;volume=287&amp;publication_year=2012&amp;pages=33957-33972&amp;pmid=22854967&amp;doi=10.1074/jbc.M112.387423&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>
Pickel B., Constantin M.-A., Pfannstiel J., Conrad J., Beifuss U., Schaller A. (2010) An enantiocomplementary dirigent protein for the enantioselective laccase-catalyzed oxidative coupling of phenols. Angew. Chem. Int. Ed. Engl. 49, 202–204</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/anie.200904622" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19946920/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.%20Int.%20Ed.%20Engl&amp;title=An%20enantiocomplementary%20dirigent%20protein%20for%20the%20enantioselective%20laccase-catalyzed%20oxidative%20coupling%20of%20phenols&amp;author=B.%20Pickel&amp;author=M.-A.%20Constantin&amp;author=J.%20Pfannstiel&amp;author=J.%20Conrad&amp;author=U.%20Beifuss&amp;volume=49&amp;publication_year=2010&amp;pages=202-204&amp;pmid=19946920&amp;doi=10.1002/anie.200904622&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>
Seneviratne H. K., Dalisay D. S., Kim K.-W., Moinuddin S. G. A., Yang H., Hartshorn C. M., Davin L. B., Lewis N. G. (2014) Non-host disease resistance response in pea (<em>Pisum sativum</em>) pods: biochemical function of DRR206 and phytoalexin pathway localization. Phytochemistry
10.1016/j.phytochem.2014.10.013</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.phytochem.2014.10.013" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25457488/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Phytochemistry&amp;title=Non-host%20disease%20resistance%20response%20in%20pea%20(Pisum%20sativum)%20pods:%20biochemical%20function%20of%20DRR206%20and%20phytoalexin%20pathway%20localization&amp;author=H.%20K.%20Seneviratne&amp;author=D.%20S.%20Dalisay&amp;author=K.-W.%20Kim&amp;author=S.%20G.%20A.%20Moinuddin&amp;author=H.%20Yang&amp;publication_year=2014&amp;pmid=25457488&amp;doi=10.1016/j.phytochem.2014.10.013&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>
Davin L. B., Lewis N. G. (2005) Dirigent phenoxy radical coupling: advances and challenges. Curr. Opin. Biotechnol. 16, 398–406</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.copbio.2005.06.010" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16023845/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Biotechnol&amp;title=Dirigent%20phenoxy%20radical%20coupling:%20advances%20and%20challenges&amp;author=L.%20B.%20Davin&amp;author=N.%20G.%20Lewis&amp;volume=16&amp;publication_year=2005&amp;pages=398-406&amp;pmid=16023845&amp;doi=10.1016/j.copbio.2005.06.010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>
Culley D. E., Horovitz D., Hadwiger L. A. (1995) Molecular characterization of disease-resistance response gene <em>DRR206-d</em> from <em>Pisum sativum</em> (L.). Plant Physiol. 107, 301–302</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.107.1.301" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC161220/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/7870833/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Molecular%20characterization%20of%20disease-resistance%20response%20gene%20DRR206-d%20from%20Pisum%20sativum%20(L.)&amp;author=D.%20E.%20Culley&amp;author=D.%20Horovitz&amp;author=L.%20A.%20Hadwiger&amp;volume=107&amp;publication_year=1995&amp;pages=301-302&amp;pmid=7870833&amp;doi=10.1104/pp.107.1.301&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>
Halls S. C., Davin L. B., Kramer D. M., Lewis N. G. (2004) Kinetic study of coniferyl alcohol radical binding to the (+)-pinoresinol forming dirigent protein. Biochemistry
43, 2587–2595</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/bi035959o" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14992596/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Kinetic%20study%20of%20coniferyl%20alcohol%20radical%20binding%20to%20the%20(+)-pinoresinol%20forming%20dirigent%20protein&amp;author=S.%20C.%20Halls&amp;author=L.%20B.%20Davin&amp;author=D.%20M.%20Kramer&amp;author=N.%20G.%20Lewis&amp;volume=43&amp;publication_year=2004&amp;pages=2587-2595&amp;pmid=14992596&amp;doi=10.1021/bi035959o&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>
Pickel B., Pfannstiel J., Steudle A., Lehmann A., Gerken U., Pleiss J., Schaller A. (2012) A model of dirigent proteins derived from structural and functional similarities with allene oxide cyclase and lipocalins. FEBS J. 279, 1980–1993</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1742-4658.2012.08580.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22443713/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20J&amp;title=A%20model%20of%20dirigent%20proteins%20derived%20from%20structural%20and%20functional%20similarities%20with%20allene%20oxide%20cyclase%20and%20lipocalins&amp;author=B.%20Pickel&amp;author=J.%20Pfannstiel&amp;author=A.%20Steudle&amp;author=A.%20Lehmann&amp;author=U.%20Gerken&amp;volume=279&amp;publication_year=2012&amp;pages=1980-1993&amp;pmid=22443713&amp;doi=10.1111/j.1742-4658.2012.08580.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>
Hofmann E., Zerbe P., Schaller F. (2006) The crystal structure of <em>Arabidopsis thaliana</em> allene oxide cyclase: insights into the oxylipin cyclization reaction. Plant Cell
18, 3201–3217</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.106.043984" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1693953/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17085685/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20crystal%20structure%20of%20Arabidopsis%20thaliana%20allene%20oxide%20cyclase:%20insights%20into%20the%20oxylipin%20cyclization%20reaction&amp;author=E.%20Hofmann&amp;author=P.%20Zerbe&amp;author=F.%20Schaller&amp;volume=18&amp;publication_year=2006&amp;pages=3201-3217&amp;pmid=17085685&amp;doi=10.1105/tpc.106.043984&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>
Neumann P., Brodhun F., Sauer K., Herrfurth C., Hamberg M., Brinkmann J., Scholz J., Dickmanns A., Feussner I., Ficner R. (2012) Crystal structures of <em>Physcomitrella patens</em> AOC1 and AOC2: insights into the enzyme mechanism and differences in substrate specificity. Plant Physiol. 160, 1251–1266</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.112.205138" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3490582/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22987885/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Crystal%20structures%20of%20Physcomitrella%20patens%20AOC1%20and%20AOC2:%20insights%20into%20the%20enzyme%20mechanism%20and%20differences%20in%20substrate%20specificity&amp;author=P.%20Neumann&amp;author=F.%20Brodhun&amp;author=K.%20Sauer&amp;author=C.%20Herrfurth&amp;author=M.%20Hamberg&amp;volume=160&amp;publication_year=2012&amp;pages=1251-1266&amp;pmid=22987885&amp;doi=10.1104/pp.112.205138&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14">
<span class="label">14.</span><cite>
Schaller F., Schaller A., Stintzi A. (2005) Biosynthesis and metabolism of jasmonates. J. Plant Growth Regul. 23, 179–199</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Plant%20Growth%20Regul&amp;title=Biosynthesis%20and%20metabolism%20of%20jasmonates&amp;author=F.%20Schaller&amp;author=A.%20Schaller&amp;author=A.%20Stintzi&amp;volume=23&amp;publication_year=2005&amp;pages=179-199&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15">
<span class="label">15.</span><cite>
Luft J. R., Collins R. J., Fehrman N. A., Lauricella A. M., Veatch C. K., DeTitta G. T. (2003) A deliberate approach to screening for initial crystallization conditions of biological macromolecules. J. Struct. Biol. 142, 170–179</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s1047-8477(03)00048-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12718929/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Struct.%20Biol&amp;title=A%20deliberate%20approach%20to%20screening%20for%20initial%20crystallization%20conditions%20of%20biological%20macromolecules&amp;author=J.%20R.%20Luft&amp;author=R.%20J.%20Collins&amp;author=N.%20A.%20Fehrman&amp;author=A.%20M.%20Lauricella&amp;author=C.%20K.%20Veatch&amp;volume=142&amp;publication_year=2003&amp;pages=170-179&amp;pmid=12718929&amp;doi=10.1016/s1047-8477(03)00048-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16">
<span class="label">16.</span><cite>
Kabsch W. (2010) XDS. Acta Crystallogr. D Biol. Crystallogr. 66, 125–132</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1107/S0907444909047337" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2815665/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20124692/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr&amp;title=XDS&amp;author=W.%20Kabsch&amp;volume=66&amp;publication_year=2010&amp;pages=125-132&amp;pmid=20124692&amp;doi=10.1107/S0907444909047337&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17">
<span class="label">17.</span><cite>
Winn M. D., Ballard C. C., Cowtan K. D., Dodson E. J., Emsley P., Evans P. R., Keegan R. M., Krissinel E. B., Leslie A. G. W., McCoy A., McNicholas S. J., Murshudov G. N., Pannu N. S., Potterton E. A., Powell H. R., Read R. J., Vagin A., Wilson K. S. (2011) Overview of the <em>CCP4</em> suite and current developments. Acta Crystallogr. D Biol. Crystallogr. 67, 235–242</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1107/S0907444910045749" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3069738/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21460441/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr&amp;title=Overview%20of%20the%20CCP4%20suite%20and%20current%20developments&amp;author=M.%20D.%20Winn&amp;author=C.%20C.%20Ballard&amp;author=K.%20D.%20Cowtan&amp;author=E.%20J.%20Dodson&amp;author=P.%20Emsley&amp;volume=67&amp;publication_year=2011&amp;pages=235-242&amp;pmid=21460441&amp;doi=10.1107/S0907444910045749&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18">
<span class="label">18.</span><cite>
Matthews B. W. (1968) Solvent content of protein crystals. J. Mol. Biol. 33, 491–497</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0022-2836(68)90205-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/5700707/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol&amp;title=Solvent%20content%20of%20protein%20crystals&amp;author=B.%20W.%20Matthews&amp;volume=33&amp;publication_year=1968&amp;pages=491-497&amp;pmid=5700707&amp;doi=10.1016/0022-2836(68)90205-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19">
<span class="label">19.</span><cite>
Chen V. B., Arendall W. B., 3rd, Headd J. J., Keedy D. A., Immormino R. M., Kapral G. J., Murray L. W., Richardson J. S., Richardson D. C. (2010) <em>MolProbity</em>: all-atom structure validation for macromolecular crystallography. Acta Crystallogr. D Biol. Crystallogr. 66, 12–21</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1107/S0907444909042073" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2803126/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20057044/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr&amp;title=MolProbity:%20all-atom%20structure%20validation%20for%20macromolecular%20crystallography&amp;author=V.%20B.%20Chen&amp;author=W.%20B.%20Arendall&amp;author=J.%20J.%20Headd&amp;author=D.%20A.%20Keedy&amp;author=R.%20M.%20Immormino&amp;volume=66&amp;publication_year=2010&amp;pages=12-21&amp;pmid=20057044&amp;doi=10.1107/S0907444909042073&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B20">
<span class="label">20.</span><cite>
Stein N. (2008) <em>CHAINSAW</em>: a program for mutating pdb files used as templates in molecular replacement. J. Appl. Crystallogr. 41, 641–643</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Crystallogr&amp;title=CHAINSAW:%20a%20program%20for%20mutating%20pdb%20files%20used%20as%20templates%20in%20molecular%20replacement&amp;author=N.%20Stein&amp;volume=41&amp;publication_year=2008&amp;pages=641-643&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21">
<span class="label">21.</span><cite>
Vagin A., Teplyakov A. (1997) <em>MOLREP</em>: an automated program for molecular replacement. J. Appl. Crystallogr. 30, 1022–1025</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Crystallogr&amp;title=MOLREP:%20an%20automated%20program%20for%20molecular%20replacement&amp;author=A.%20Vagin&amp;author=A.%20Teplyakov&amp;volume=30&amp;publication_year=1997&amp;pages=1022-1025&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22">
<span class="label">22.</span><cite>
Murshudov G. N., Vagin A. A., Lebedev A., Wilson K. S., Dodson E. J. (1999) Efficient anisotropic refinement of macromolecular structures using FFT. Acta Crystallogr. D Biol. Crystallogr. 55, 247–255</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1107/S090744499801405X" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10089417/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr&amp;title=Efficient%20anisotropic%20refinement%20of%20macromolecular%20structures%20using%20FFT&amp;author=G.%20N.%20Murshudov&amp;author=A.%20A.%20Vagin&amp;author=A.%20Lebedev&amp;author=K.%20S.%20Wilson&amp;author=E.%20J.%20Dodson&amp;volume=55&amp;publication_year=1999&amp;pages=247-255&amp;pmid=10089417&amp;doi=10.1107/S090744499801405X&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23">
<span class="label">23.</span><cite>
Adams P. D., Afonine P. V., Bunkóczi G., Chen V. B., Davis I. W., Echols N., Headd J. J., Hung L.-W., Kapral G. J., Grosse-Kunstleve R. W., McCoy A. J., Moriarty N. W., Oeffner R., Read R. J., Richardson D. C., Richardson J. S., Terwilliger T. C., Zwart P. H. (2010) <em>PHENIX</em>: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr. D Biol. Crystallogr. 66, 213–221</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1107/S0907444909052925" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2815670/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20124702/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr&amp;title=PHENIX:%20a%20comprehensive%20Python-based%20system%20for%20macromolecular%20structure%20solution&amp;author=P.%20D.%20Adams&amp;author=P.%20V.%20Afonine&amp;author=G.%20Bunk%C3%B3czi&amp;author=V.%20B.%20Chen&amp;author=I.%20W.%20Davis&amp;volume=66&amp;publication_year=2010&amp;pages=213-221&amp;pmid=20124702&amp;doi=10.1107/S0907444909052925&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24">
<span class="label">24.</span><cite>
Emsley P., Cowtan K. (2004) <em>Coot</em>: model-building tools for molecular graphics. Acta Crystallogr. D Biol. Crystallogr. 60, 2126–2132</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1107/S0907444904019158" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15572765/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr&amp;title=Coot:%20model-building%20tools%20for%20molecular%20graphics&amp;author=P.%20Emsley&amp;author=K.%20Cowtan&amp;volume=60&amp;publication_year=2004&amp;pages=2126-2132&amp;pmid=15572765&amp;doi=10.1107/S0907444904019158&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25">
<span class="label">25.</span><cite>
Berman H. M., Westbrook J., Feng Z., Gilliland G., Bhat T. N., Weissig H., Shindyalov I. N., Bourne P. E. (2000) The Protein Data Bank. Nucleic Acids Res. 28, 235–242</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/28.1.235" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC102472/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10592235/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20Protein%20Data%20Bank&amp;author=H.%20M.%20Berman&amp;author=J.%20Westbrook&amp;author=Z.%20Feng&amp;author=G.%20Gilliland&amp;author=T.%20N.%20Bhat&amp;volume=28&amp;publication_year=2000&amp;pages=235-242&amp;pmid=10592235&amp;doi=10.1093/nar/28.1.235&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26">
<span class="label">26.</span><cite>
Krissinel E., Henrick K. (2004) Secondary-structure matching (SSM), a new tool for fast protein structure alignment in three dimensions. Acta Crystallogr. D Biol. Crystallogr. 60, 2256–2268</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1107/S0907444904026460" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15572779/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr&amp;title=Secondary-structure%20matching%20(SSM),%20a%20new%20tool%20for%20fast%20protein%20structure%20alignment%20in%20three%20dimensions&amp;author=E.%20Krissinel&amp;author=K.%20Henrick&amp;volume=60&amp;publication_year=2004&amp;pages=2256-2268&amp;pmid=15572779&amp;doi=10.1107/S0907444904026460&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27">
<span class="label">27.</span><cite>
DeLano W. L. (2002) The PyMOL Molecular Graphics System, Version 1.5.0.4, Schrödinger, LLC</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=The%20PyMOL%20Molecular%20Graphics%20System&amp;author=W.%20L.%20DeLano&amp;publication_year=2002&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28">
<span class="label">28.</span><cite>
Binkowski T. A., Naghibzadeh S., Liang J. (2003) CASTp: computed atlas of surface topography of proteins. Nucleic Acids Res. 31, 3352–3355</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkg512" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC168919/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12824325/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=CASTp:%20computed%20atlas%20of%20surface%20topography%20of%20proteins&amp;author=T.%20A.%20Binkowski&amp;author=S.%20Naghibzadeh&amp;author=J.%20Liang&amp;volume=31&amp;publication_year=2003&amp;pages=3352-3355&amp;pmid=12824325&amp;doi=10.1093/nar/gkg512&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29">
<span class="label">29.</span><cite>
Glaser F., Pupko T., Paz I., Bell R. E., Bechor-Shental D., Martz E., Ben-Tal N. (2003) ConSurf: identification of functional regions in proteins by surface-mapping of phylogenetic information. Bioinformatics
19, 163–164</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/19.1.163" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12499312/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=ConSurf:%20identification%20of%20functional%20regions%20in%20proteins%20by%20surface-mapping%20of%20phylogenetic%20information&amp;author=F.%20Glaser&amp;author=T.%20Pupko&amp;author=I.%20Paz&amp;author=R.%20E.%20Bell&amp;author=D.%20Bechor-Shental&amp;volume=19&amp;publication_year=2003&amp;pages=163-164&amp;pmid=12499312&amp;doi=10.1093/bioinformatics/19.1.163&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30">
<span class="label">30.</span><cite>
Morris G. M., Huey R., Lindstrom W., Sanner M. F., Belew R. K., Goodsell D. S., Olson A. J. (2009) AutoDock4 and AutoDockTools4: automated docking with selective receptor flexibility. J. Comput. Chem. 30, 2785–2791</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/jcc.21256" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2760638/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19399780/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem&amp;title=AutoDock4%20and%20AutoDockTools4:%20automated%20docking%20with%20selective%20receptor%20flexibility&amp;author=G.%20M.%20Morris&amp;author=R.%20Huey&amp;author=W.%20Lindstrom&amp;author=M.%20F.%20Sanner&amp;author=R.%20K.%20Belew&amp;volume=30&amp;publication_year=2009&amp;pages=2785-2791&amp;pmid=19399780&amp;doi=10.1002/jcc.21256&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31">
<span class="label">31.</span><cite>
Trott O., Olson A. J. (2010) AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading. J. Comput. Chem. 31, 455–461</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/jcc.21334" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3041641/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19499576/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem&amp;title=AutoDock%20Vina:%20improving%20the%20speed%20and%20accuracy%20of%20docking%20with%20a%20new%20scoring%20function,%20efficient%20optimization,%20and%20multithreading&amp;author=O.%20Trott&amp;author=A.%20J.%20Olson&amp;volume=31&amp;publication_year=2010&amp;pages=455-461&amp;pmid=19499576&amp;doi=10.1002/jcc.21334&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32">
<span class="label">32.</span><cite>
Bordoli L., Kiefer F., Arnold K., Benkert P., Battey J., Schwede T. (2009) Protein structure homology modeling using SWISS-MODEL workspace. Nat. Protoc. 4, 1–13</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nprot.2008.197" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19131951/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc&amp;title=Protein%20structure%20homology%20modeling%20using%20SWISS-MODEL%20workspace&amp;author=L.%20Bordoli&amp;author=F.%20Kiefer&amp;author=K.%20Arnold&amp;author=P.%20Benkert&amp;author=J.%20Battey&amp;volume=4&amp;publication_year=2009&amp;pages=1-13&amp;pmid=19131951&amp;doi=10.1038/nprot.2008.197&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B33">
<span class="label">33.</span><cite>
Fristensky B., Horovitz D., Hadwiger L. (1988) cDNA sequences for pea disease resistance response genes. Plant Mol. Biol. 11, 713–715</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/BF00017470" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24272504/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol.%20Biol&amp;title=cDNA%20sequences%20for%20pea%20disease%20resistance%20response%20genes&amp;author=B.%20Fristensky&amp;author=D.%20Horovitz&amp;author=L.%20Hadwiger&amp;volume=11&amp;publication_year=1988&amp;pages=713-715&amp;pmid=24272504&amp;doi=10.1007/BF00017470&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34">
<span class="label">34.</span><cite>
Fristensky B., Riggleman R. C., Wagoner W., Hadwiger L. A. (1985) Gene expression in susceptible and disease resistant interactions of peas induced with <em>Fusarium solani</em> pathogens and chitosan. Physiol. Plant Pathol. 27, 15–28</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Physiol.%20Plant%20Pathol&amp;title=Gene%20expression%20in%20susceptible%20and%20disease%20resistant%20interactions%20of%20peas%20induced%20with%20Fusarium%20solani%20pathogens%20and%20chitosan&amp;author=B.%20Fristensky&amp;author=R.%20C.%20Riggleman&amp;author=W.%20Wagoner&amp;author=L.%20A.%20Hadwiger&amp;volume=27&amp;publication_year=1985&amp;pages=15-28&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35">
<span class="label">35.</span><cite>
Riggleman R. C., Fristensky B., Hadwiger L. A. (1985) The disease resistance response in pea is associated with increased levels of specific mRNAs. Plant Mol. Biol. 4, 81–86</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/BF02418753" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24310743/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol.%20Biol&amp;title=The%20disease%20resistance%20response%20in%20pea%20is%20associated%20with%20increased%20levels%20of%20specific%20mRNAs&amp;author=R.%20C.%20Riggleman&amp;author=B.%20Fristensky&amp;author=L.%20A.%20Hadwiger&amp;volume=4&amp;publication_year=1985&amp;pages=81-86&amp;pmid=24310743&amp;doi=10.1007/BF02418753&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36">
<span class="label">36.</span><cite>
Fetrow J. S. (1995) Omega loops: nonregular secondary structures significant in protein function and stability. FASEB J. 9, 708–717</cite> [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/7601335/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FASEB%20J&amp;title=Omega%20loops:%20nonregular%20secondary%20structures%20significant%20in%20protein%20function%20and%20stability&amp;author=J.%20S.%20Fetrow&amp;volume=9&amp;publication_year=1995&amp;pages=708-717&amp;pmid=7601335&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B37">
<span class="label">37.</span><cite>
Holm L., Rosenström P. (2010) Dali server: conservation mapping in 3D. Nucleic Acids Res. 38, W545–W549</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkq366" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2896194/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20457744/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Dali%20server:%20conservation%20mapping%20in%203D&amp;author=L.%20Holm&amp;author=P.%20Rosenstr%C3%B6m&amp;volume=38&amp;publication_year=2010&amp;pages=W545-W549&amp;pmid=20457744&amp;doi=10.1093/nar/gkq366&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B38">
<span class="label">38.</span><cite>
Halls S. C., Lewis N. G. (2002) Secondary and quaternary structures of the (+)-pinoresinol forming dirigent protein. Biochemistry
41, 9455–9461</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/bi0259709" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12135368/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Secondary%20and%20quaternary%20structures%20of%20the%20(+)-pinoresinol%20forming%20dirigent%20protein&amp;author=S.%20C.%20Halls&amp;author=N.%20G.%20Lewis&amp;volume=41&amp;publication_year=2002&amp;pages=9455-9461&amp;pmid=12135368&amp;doi=10.1021/bi0259709&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B39">
<span class="label">39.</span><cite>
Hosmani P. S., Kamiya T., Danku J., Naseer S., Geldner N., Guerinot M. L., Salt D. E. (2013) Dirigent domain-containing protein is part of the machinery required for formation of the lignin-based Casparian strip in the root. Proc. Natl. Acad. Sci. U.S.A. 110, 14498–14503</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1308412110" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3761638/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23940370/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;title=Dirigent%20domain-containing%20protein%20is%20part%20of%20the%20machinery%20required%20for%20formation%20of%20the%20lignin-based%20Casparian%20strip%20in%20the%20root&amp;author=P.%20S.%20Hosmani&amp;author=T.%20Kamiya&amp;author=J.%20Danku&amp;author=S.%20Naseer&amp;author=N.%20Geldner&amp;volume=110&amp;publication_year=2013&amp;pages=14498-14503&amp;pmid=23940370&amp;doi=10.1073/pnas.1308412110&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B40">
<span class="label">40.</span><cite>
Davin L. B., Lewis N. G. (2005) Lignin primary structures and dirigent sites. Curr. Opin. Biotechnol. 16, 407–415</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.copbio.2005.06.011" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16023847/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Biotechnol&amp;title=Lignin%20primary%20structures%20and%20dirigent%20sites&amp;author=L.%20B.%20Davin&amp;author=N.%20G.%20Lewis&amp;volume=16&amp;publication_year=2005&amp;pages=407-415&amp;pmid=16023847&amp;doi=10.1016/j.copbio.2005.06.011&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section></section></section></article>
</main></body></html>