<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4321547/"/><title>Natural variation in the expression of ORGANIC CATION TRANSPORTER 1 affects root length responses to cadaverine in Arabidopsis - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">J Exp Bot</button></div>. 2014 Nov 16;66(3):853–862. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/eru444" rel="noopener noreferrer" target="_blank">10.1093/jxb/eru444</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Natural variation in the expression of <em>ORGANIC CATION TRANSPORTER 1</em> affects root length responses to cadaverine in <em>Arabidopsis</em>
</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Strohm%20AK%22%5BAuthor%5D"><span class="name western">Allison K Strohm</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Allison K Strohm</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Genetics, University of Wisconsin Madison, 425G Henry Mall, Madison, WI 53706, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Strohm%20AK%22%5BAuthor%5D"><span class="name western">Allison K Strohm</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vaughn%20LM%22%5BAuthor%5D"><span class="name western">Laura M Vaughn</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Laura M Vaughn</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Genetics, University of Wisconsin Madison, 425G Henry Mall, Madison, WI 53706, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vaughn%20LM%22%5BAuthor%5D"><span class="name western">Laura M Vaughn</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Masson%20PH%22%5BAuthor%5D"><span class="name western">Patrick H Masson</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Patrick H Masson</span></h3>
<div class="p">
<sup>1</sup>Laboratory of Genetics, University of Wisconsin Madison, 425G Henry Mall, Madison, WI 53706, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Masson%20PH%22%5BAuthor%5D"><span class="name western">Patrick H Masson</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="AF0001">
<sup>1</sup>Laboratory of Genetics, University of Wisconsin Madison, 425G Henry Mall, Madison, WI 53706, USA</div>
<div class="author-notes p">
<div class="fn" id="c2">
<sup>✉</sup><p class="display-inline">* These authors contributed equally to this work.</p>
</div>
<div class="fn" id="c1">
<sup>†</sup><p class="display-inline">To whom correspondence should be addressed. E-mail: <span>phmasson@wisc.edu</span></p>
</div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Issue date 2015 Feb.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© The Author 2014. Published by Oxford University Press on behalf of the Society for Experimental Biology. </div>
<p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://creativecommons.org/licenses/by/3.0/" rel="noopener noreferrer" target="_blank">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC4321547  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25403917/">25403917</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Highlight</h2>
<p>How the polyamine cadaverine alters plant development is poorly understood. Here it is shown that natural variation in <em>ORGANIC CATION TRANSPORTER 1</em> affects root length sensitivity to exogenous cadaverine.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Key words:</strong> <em>Arabidopsis</em>, cadaverine, natural variation, polyamines, roots, transporter.</p></section></section><section class="abstract" id="abstract2"><h2>Abstract</h2>
<p>Polyamines, including cadaverine, are organic cations that affect numerous biological processes including transcription, translation, cell signalling, and ion channel activity. They often function in biotic and abiotic stress responses in plants. Because little is known about how plants respond to cadaverine, a quantitative natural variation approach was used to identify genetic factors that contribute to this response. Here it is shown that <em>Arabidopsis thaliana</em> accessions have varying root length responses to exogenous cadaverine: Cape Verde Islands (Cvi) was one of the most resistant accessions tested, whereas Landsberg <em>erecta</em> (L<em>er</em>) was one of the most sensitive. Recombinant inbred lines, near isogenic lines, and a microarray were used to show that variation in <em>ORGANIC CATION TRANSPORTER 1</em> (<em>OCT1</em>) is at least partially responsible for this difference. <em>OCT1</em> expression was higher in Cvi than in L<em>er</em>, and <em>oct1</em> mutants were more sensitive to cadaverine than wild-type plants. In <em>oct1</em> mutants transformed with an ectopic copy of <em>OCT1</em> originating from either Cvi or L<em>er</em>, the expression level of the transgene, not its accession, correlated with the cadaverine response. These results suggest that decreased <em>OCT1</em> expression confers cadaverine sensitivity in some accessions.</p></section><section id="s1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Polyamines are organic cations that are prevalent in both prokaryotes and eukaryotes. Putrescine, spermidine, and spermine are the most common polyamines in plants, although thermospermine and cadaverine have also been found in many species including <em>Arabidopsis thaliana</em> (<a aria-describedby="CIT0040" class="usa-link" href="#CIT0040">Shevyakova <em>et al.</em>, 2001</a>; <a aria-describedby="CIT0021" class="usa-link" href="#CIT0021">Knott <em>et al.</em>, 2007</a>). Polyamines exist both as free molecules and as conjugates to other small molecules, mainly hydroxycinnamic acids, or proteins (<a aria-describedby="CIT0006" class="usa-link" href="#CIT0006">Bagni and Tassoni, 2001</a>). Both free and conjugated polyamines are important for cellular processes including cell growth, cell division, transcription, translation, and cell death. Their positive charges allow for interactions with nucleic acids, lipids, and proteins, which affect the activities and stabilities of these molecules (for a review, see <a aria-describedby="CIT0018" class="usa-link" href="#CIT0018">Hussain <em>et al.</em>, 2011</a>).</p>
<p>Polyamines accumulate in certain plant tissues under many abiotic stresses including drought, salt, osmotic, and temperature stress, which all result in metabolic reprogramming and modifications to growth (<a aria-describedby="CIT0011" class="usa-link" href="#CIT0011">Cuevas <em>et al.</em>, 2008</a>; <a aria-describedby="CIT0001" class="usa-link" href="#CIT0001">Alet <em>et al.</em>, 2011</a>; <a aria-describedby="CIT0046" class="usa-link" href="#CIT0046">Wang <em>et al.</em>, 2011</a>). Polyamines may protect against these stresses in several ways, including by altering gene expression, increasing the activities of antioxidant enzymes, and altering the cellular accumulation of ions (<a aria-describedby="CIT0026" class="usa-link" href="#CIT0026">Liu <em>et al.</em>, 2000</a>; <a aria-describedby="CIT0042" class="usa-link" href="#CIT0042">Takahashi <em>et al.</em>, 2003</a>; <a aria-describedby="CIT0043" class="usa-link" href="#CIT0043">Tang and Newton, 2005</a>). They may also function as signals that regulate cross-talk between hormonal pathways during stress responses (<a aria-describedby="CIT0033" class="usa-link" href="#CIT0033">Obata and Fernie, 2012</a>). Though essential for proper development, and beneficial at higher levels during stress, catabolism of polyamines produces H<sub>2</sub>O<sub>2</sub> (<a aria-describedby="CIT0044" class="usa-link" href="#CIT0044">Tisi <em>et al.</em>, 2011</a>). Although it is toxic, H<sub>2</sub>O<sub>2</sub> also diffuses to nearby tissues where it activates various responses (<a aria-describedby="CIT0031" class="usa-link" href="#CIT0031">Neill, 2002</a>). To balance these effects, polyamine synthesis, degradation, localization, and conjugation must be tightly controlled.</p>
<p>Five potential <em>Arabidopsis</em> POLYAMINE UPTAKE TRANSPORTER (PUT) proteins were identified based on sequence similarity to known polyamine transporters in <em>Leishmania major</em> and <em>Trympanosoma cruzi</em> (<a aria-describedby="CIT0029" class="usa-link" href="#CIT0029">Mulangi <em>et al</em>., 2012<em>a</em>, <em>b</em>
</a>). When expressed in the polyamine uptake-deficient <em>Δagp2 Saccharomyces cerevisiae</em> mutant (<a aria-describedby="CIT0004" class="usa-link" href="#CIT0004">Aouida <em>et al.</em>, 2005</a>), these proteins partially restored sensitivity to the polyamine analogue paraquat as well as to putrescine and spermidine. They also partially restored putrescine and spermidine uptake into the cells (<a aria-describedby="CIT0029" class="usa-link" href="#CIT0029">Mulangi <em>et al.</em>, 2012<em>a</em>
</a>). PAR1/PUT2 has subsequently been shown to localize to the Golgi where it is required for the subcellular localization of paraquat in the chloroplast (<a aria-describedby="CIT0025" class="usa-link" href="#CIT0025">Li <em>et al.</em>, 2013</a>). <em>RMV1</em>/<em>PUT3</em> has been shown to be plasma membrane associated and to function in paraquat, putrescine, spermidine, and spermine uptake in <em>Arabidopsis</em> (<a aria-describedby="CIT0015" class="usa-link" href="#CIT0015">Fujita <em>et al.</em>, 2012</a>). However, cadaverine transport by these proteins was not tested.</p>
<p>Less is known about cadaverine compared with the other polyamines. It is synthesized from lysine and therefore not directly derived from the polyamine putrescine, unlike spermidine, spermine, and thermospermine. Cadaverine is produced by some soil bacteria and yeast, possibly contributing to rhizosphere–root signalling (<a aria-describedby="CIT0034" class="usa-link" href="#CIT0034">Perrig <em>et al.</em>, 2007</a>; <a aria-describedby="CIT0009" class="usa-link" href="#CIT0009">Cloete <em>et al.</em>, 2009</a>). Exogenous cadaverine affects root development in several ways: it increases adventitious root growth in pine trees, lateral root formation in soybeans, and root weight in rice (<a aria-describedby="CIT0016" class="usa-link" href="#CIT0016">Gamarnik and Frydman, 1991</a>; <a aria-describedby="CIT0032" class="usa-link" href="#CIT0032">Niemi <em>et al.</em>, 2002</a>; <a aria-describedby="CIT0008" class="usa-link" href="#CIT0008">Cassán <em>et al.</em>, 2009</a>), whereas it shortens primary roots in soybeans (<a aria-describedby="CIT0016" class="usa-link" href="#CIT0016">Gamarnik and Frydman, 1991</a>).</p>
<p>Cadaverine levels and localization change upon exposure to stress. This molecule accumulates dramatically upon oxidative stress in rape leaves and upon drought stress in peppers (<a aria-describedby="CIT0005" class="usa-link" href="#CIT0005">Aziz <em>et al.</em>, 1997</a>; <a aria-describedby="CIT0041" class="usa-link" href="#CIT0041">Sziderics <em>et al.</em>, 2010</a>). In the ice plant, salt stress causes cadaverine to accumulate in true leaves (<a aria-describedby="CIT0023" class="usa-link" href="#CIT0023">Kuznetsov <em>et al.</em>, 2007</a>), and heat shock stimulates cadaverine transport away from the heated tissue (<a aria-describedby="CIT0040" class="usa-link" href="#CIT0040">Shevyakova <em>et al.</em>, 2001</a>). Cadaverine also confers protection against stress. It induces stomatal closure, a common stress response, in fava beans (<a aria-describedby="CIT0026" class="usa-link" href="#CIT0026">Liu <em>et al.</em>, 2000</a>), and it prevents decreases in root and shoot weight upon oxidative stress in rice (<a aria-describedby="CIT0008" class="usa-link" href="#CIT0008">Cassán <em>et al.</em>, 2009</a>).</p>
<p>Despite its profound impacts on overall cellular homeostasis and plant growth under stressful conditions, the underlying mechanisms that explain how cadaverine alters plant growth and development are still unknown. To address this gap, natural variation in root length sensitivity to exogenous cadaverine was examined, and <em>ORGANIC CATION TRANSPORTER 1</em> (<em>OCT1</em>, <em>AT1G73220</em>) was identified as an important quantitative trait locus (QTL) involved in this response.</p></section><section id="s2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="s3"><h3 class="pmc_sec_title">Plant materials</h3>
<p>QTL analysis used the following stocks from the Arabidopsis Biological Resource Center (Columbus, OH, USA): Cape Verde Islands (Cvi; CS8580), Landsberg <em>erecta</em>-2 L<em>er</em>-2; CS8581), and a population of Cvi/L<em>er</em> recombinant inbred lines (RILs; CS22000). This population consists of 162 individual RILs that were genotyped at 293 marker loci (<a aria-describedby="CIT0003" class="usa-link" href="#CIT0003">Alonso-Blanco <em>et al.</em>, 1998</a>). The near isogenic lines (NILs) LCN1-12, -15, -14, -21, -22, and -28 (N717058, N17060, N17061, N17067, N17068, and N17074) (<a aria-describedby="CIT0020" class="usa-link" href="#CIT0020">Keurentjes <em>et al.</em>, 2007</a>) were obtained from the Nottingham Arabidopsis Stock Center (Nottingham, UK). An-1 (CS22626), Bs-1 (CS6627), Knox-18 (CS22567), Sq-8 (CS22601), Ull2-3 (CS22587), and Wei-0 (CS22622) seed came from the Arabidopsis Biological Resource Center. The <em>oct1-1</em> mutant and <em>oct1-1</em> [35S<sub>pro</sub>:OCT1<sup>WS</sup>] transgenic lines were provided by Christine Lelandais-Briere (<a aria-describedby="CIT0024" class="usa-link" href="#CIT0024">Lelandais-Brière <em>et al.</em>, 2007</a>).</p></section><section id="s4"><h3 class="pmc_sec_title">Growth conditions</h3>
<p>The seeds were sterilized by washing them with 95% ethanol. Except where noted in the text, they were plated on half-strength buffered Linsmaier and Skoog medium containing micro- and macronutrients, vitamins, and 1.5% sucrose (Caisson Laboratories, North Logan, UT, USA) that was supplemented with 1.5% agar type E (Sigma-Aldrich, St. Louis, MO, USA) and polyamines (Sigma-Aldrich) as required. The seedlings were stratified for 2–8 d and grown in either a Conviron TC16 growth chamber or an Enconair (now BioChambers, Winnipeg, Manitoba, Canada) Controller 6000 growth chamber for each experiment. Both chambers were set at 22 °C and a 16h light/8h dark cycle, and the light intensity was 50–70 μmol m<sup>–2</sup> s<sup>–1</sup> and provided by cool-white fluorescent bulbs (Grainger, Lake Forest, IL, USA).</p></section><section id="s5"><h3 class="pmc_sec_title">Root length measurements</h3>
<p>Seedling images were taken from the backs of the plates using a scanner (Epson, Suwa, Nagano Prefecture, Japan). Root length was measured using the segmented line tool in NIH Image (publically available at <a class="usa-link" data-ga-action="click_feat_suppl" href="http://rsb.info.nih.gov/nih-image/" rel="noopener noreferrer" target="_blank">http://rsb.info.nih.gov/nih-image/</a>) or the Neuron J plugin (<a aria-describedby="CIT0028" class="usa-link" href="#CIT0028">Meijering <em>et al.</em>, 2004</a>) for Image J (<a aria-describedby="CIT0037" class="usa-link" href="#CIT0037">Schneider <em>et al.</em>, 2012</a>).</p></section><section id="s6"><h3 class="pmc_sec_title">Microscopy</h3>
<p>To calculate cell length, the cell walls were stained by transferring the seedlings to 10 μg ml<sup>–1</sup> propidium iodide (Sigma-Aldrich) for ~20min. Imaging was conducted using a Zeiss LSM 510 Meta Confocal microscope (Zeiss, Thornwood, NY, USA), and cell length measurements were performed in Adobe Photoshop.</p></section><section id="s7"><h3 class="pmc_sec_title">QTL mapping</h3>
<p>Plants were grown at a 30 ° tilt for 6 d on media with or without cadaverine. A concentration of 50 μM cadaverine was used in trial 1, and 100 μM cadaverine was used in trial 2. The mean root length value on medium containing cadaverine was subtracted from the mean value on control medium and then divided by the mean on control medium. This number was subtracted from 1, and this ratio, along with genotypic data for each RIL (<a aria-describedby="CIT0003" class="usa-link" href="#CIT0003">Alonso-Blanco <em>et al.</em>, 1998</a>), was entered into WinQTL Cartographer version 2.5 (<a aria-describedby="CIT0047" class="usa-link" href="#CIT0047">Wang <em>et al.</em>, 2010</a>) and the r/qtl package in R (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.r-project.org" rel="noopener noreferrer" target="_blank">http://www.r-project.org</a>). For WinQTL Cartographer, maps were created using the Kosambi map function. During composite interval mapping, the model incorporated a 0.5 step rate with a 1.0 cM window size and 10 control markers. LOD thresholds were determined by 1000 permutations. In r/qtl, the function ‘scanone’ was used for interval mapping, and ‘scantwo’ was used to generate the two-dimensional plot. Significance levels were determined by 1000 permutations, implementing the EM algorithm. Fine mapping of the NILs was performed using standard methods and the primers listed in <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Table S1</a> available at <em>JXB</em> online.</p></section><section id="s8"><h3 class="pmc_sec_title">RT–PCR and qRT-PCR</h3>
<p>Approximately 24 seedlings were grown at a 30 ° backward tilt at staggered times for each biological replicate on medium containing no cadaverine unless otherwise indicated. Whole roots from 7-day-old seedlings were dissected and immediately frozen in liquid nitrogen. RNA was prepared using an RNeasy Plant Mini Kit (QIAGEN, Venlo, The Netherlands) and stored at –80 ºC until use, typically within a few days. RNA samples were treated with RQ1 DNase (Promega, Madison, WI, USA) according to the instructions. The ratios at absorbencies of 260 to 280 and at 260 to 230 as well as gel electrophoresis were used to verify RNA quality. Poor quality samples were not used for subsequent experiments. RNA concentrations were determined using a NanoDrop 2000 (Thermo Scientific, Waltham, MA, USA).</p>
<p>For reverse transcription–PCR (RT–PCR), cDNA was made using the iScript™ cDNA Synthesis Kit (Bio-Rad, Hercules, CA, USA). For quantitative real-time PCR (qRT-PCR), cDNA synthesis and qRT–PCR were done simultaneously using a qScript™ One-Step qRT-PCR Kit (Quanta Biosciences, Gaithersburg, MD, USA). The samples were run on a LightCycler<sup>®</sup> 480 Real Time-PCR System (Roche Applied Science, Basel, Switzerland). Each 6 μl reaction contained RNA at a final concentration of 2ng μl<sup>–1</sup> and each primer at a concentration of 0.65 μM. Other components were included at the concentrations defined by the kit. Control reactions without reverse transcriptase were included and did not yield any detectable expression. Cycling conditions were 49 ºC for 10min, 95 ºC for 5min, and 45 cycles of 95 ºC for 5 s, 55 ºC for 15 s, and 72 ºC for 10 s. Melt curves were used to verify specificity. Four technical replicates were performed for each sample for each biological replicate, and three biological replicates were performed. LinRegPCR was used to analyse the data (<a aria-describedby="CIT0035" class="usa-link" href="#CIT0035">Ramakers <em>et al.</em>, 2003</a>).</p>
<p>The primers used to amplify <em>OCT1</em> were: 1F (5′ GTGGCTGTT CCTTCCACACT 3′), 1R (5′ AAAGGCCGTGACGAAAGTTA 3′), 2F (5′ CATCCTCGACAGCGTATGAC 3′), 2R (5′ CCTTC CTAACGCAACTAGCA 3′), 3F (5′ TTTGGTGTTGCATCAGTG CT 3′), and 3R (5′ CGGAGCGTTTCGAGTTTCT 3′). 4R (5′ CGTGAAATGCGTGTTGAAAG 3′) was used to detect possible transcript that extended into the T-DNA region. Primers 3F and 3R as well as 5′ CCTCTTGGATACGCGGTTT 3′ and 5′ CAAGA AGCCATCGAGGAGAC 3′ were used for <em>OCT1</em> qRT–PCR. Primers used to amplify the reference gene At1g58050 (<a aria-describedby="CIT0013" class="usa-link" href="#CIT0013">Czechowski <em>et al.</em>, 2005</a>) were: 5′ CCATTCTACTTTTTGGCGGCT 3′ and 5′ TCAATGGTAACTGATCCACTCTGATG 3′. This reference gene was chosen due to its consistent expression and its similar expression level to <em>OCT1</em> (<a aria-describedby="CIT0013" class="usa-link" href="#CIT0013">Czechowski <em>et al.</em>, 2005</a>)</p></section><section id="s9"><h3 class="pmc_sec_title">Transgenic constructs</h3>
<p>Genomic DNA from Cvi and L<em>er</em> was used to amplify the region extending from 2079 bases before the start codon through to 417 bases after the stop codon of <em>OCT1</em>. This region was cloned in between the attL1 and attL2 sites in the Gateway entry vector pENTR/D-TOPO (Life Technologies, Madison, WI, USA). An LR reaction was performed to transfer this region into the binary vector pMDC99 (<a aria-describedby="CIT0012" class="usa-link" href="#CIT0012">Curtis and Grossniklaus, 2003</a>; <a aria-describedby="CIT0048" class="usa-link" href="#CIT0048">Xu and Li, 2008</a>). The constructs were sequenced and transformed into plants using the <em>Agrobacterium</em>-mediated floral dip method (<a aria-describedby="CIT0010" class="usa-link" href="#CIT0010">Clough and Bent, 1998</a>). T<sub>3</sub> seedlings likely to all carry the transgene as determined by antibiotic resistance were used in all experiments (<a aria-describedby="CIT0017" class="usa-link" href="#CIT0017">Harrison <em>et al.</em>, 2006</a>).</p></section></section><section id="s10"><h2 class="pmc_sec_title">Results</h2>
<section id="s11"><h3 class="pmc_sec_title">
<em>Arabidopsis</em> accessions vary in their root length responses to cadaverine</h3>
<p>As a first step toward identifying genes involved in root growth responses to exogenous cadaverine, seedlings of eight <em>Arabidopsis</em> accessions were grown on media containing 0, 100, and 500 μM cadaverine. A decrease in root length between control and increasing cadaverine treatments was observed for all accessions tested (<a class="usa-link" href="#F1">Fig. 1A</a>). Alterations in root waving and skewing and an increase in adventitious and lateral root emergence were also noticed in most accessions (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Fig. S1</a> at <em>JXB</em> online). In terms of root length, Cvi-1 and Knox-18 were some of the most resistant accessions to cadaverine, whereas L<em>er</em>-2 was one of the most sensitive (<a class="usa-link" href="#F1">Fig. 1A</a>, <a class="usa-link" href="#F1">B</a>).</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4321547_exbotj_eru444_f0001.jpg" target="_blank"><img alt="Fig. 1." class="graphic zoom-in" height="992" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f376/4321547/30c89cd47a75/exbotj_eru444_f0001.jpg" width="799"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>
<em>Arabidopsis</em> accessions show varying root length inhibition responses to exogenous cadaverine. (A) Seedlings were grown vertically for 3 d and then at a 30 ° backward tilt for another 3 d. Average root length on control medium was set to 1. Absolute average root lengths on control medium were 1.4, 1.6, 2.5, 2.1, 2.1, 2.3, 2.0, and 2.3cm in the accessions as shown from left to right. The asterisk indicates the first cadaverine concentration corresponding to a significantly shorter root compared with control medium (<em>P</em>&lt;0.05, Student’s <em>t</em>-test). Error bars indicate ±SE, and <em>n</em>=12–27. (B) Images of Cvi-1 (top) and L<em>er</em>-2 (bottom) seedlings growing on media without (left) and with (right) 100 μM cadaverine. The white scale bar is 1cm. (C) Average length of epidermal cells in the mature root zone from seedlings grown vertically for 7 d on either control medium or medium containing 100 μM cadaverine. Error bars indicate ±SE, and <em>n</em>=11–18.</p></figcaption></figure><p>To determine whether the effect of cadaverine on root length was due to reduced cell elongation or division, the length of the epidermal cells in the mature root zone of L<em>er</em> seedlings grown on media containing 0 μM or 100 μM cadaverine was measured. It was found that although L<em>er</em> roots were only ~35% as long on this concentration of cadaverine as on control medium (<a class="usa-link" href="#F1">Fig. 1A</a>), the average cell length of the seedlings on the medium containing cadaverine was ~64% of that of seedlings on control medium (<a class="usa-link" href="#F1">Fig. 1C</a>). This suggests that the reduction in root length is probably due to a combination of reduced cell elongation and division.</p></section><section id="s12"><h3 class="pmc_sec_title">A QTL on chromosome 1 affects root length responses to cadaverine</h3>
<p>Due to the large difference between Cvi and L<em>er</em> in root growth responses to exogenous cadaverine, a QTL study was performed with a group of RILs created from the two accessions. Other pairs of accessions would have been equally interesting to use for QTL analysis but, at the time this experiment was initiated, the Cvi×L<em>er</em> population was the available population created from accessions with the most differing cadaverine responses. These RILs were previously characterized for genotypic contribution from the two parental lines along the five chromosomes (<a aria-describedby="CIT0003" class="usa-link" href="#CIT0003">Alonso-Blanco <em>et al.</em>, 1998</a>). For both the parental lines and the RILs, root growth on medium containing cadaverine compared with growth on control plates lacking this compound was quantified. RIL mean root length ratios showed transgression with respect to the parental means (<a class="usa-link" href="#F2">Fig. 2A</a>). Furthermore, growth on 0 μM and 50 μM cadaverine displayed broad sense heritability values of 0.46 and 0.56, respectively. Taken together, these results indicate that this population is well suited for QTL mapping of root growth responses to exogenous cadaverine.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4321547_exbotj_eru444_f0002.jpg" target="_blank"><img alt="Fig. 2." class="graphic zoom-in" height="1438" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f376/4321547/3afea3e24766/exbotj_eru444_f0002.jpg" width="683"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>QTL analysis of root length responses to cadaverine in the Cvi/L<em>er</em> RIL population. (A) Histogram showing root length ratios for the RILs on 100 μM cadaverine compared with control medium. (B) Composite interval mapping analysis for the root length ratio. The top portion of the graph gives the LOD score across each chromosome. The LOD significance threshold is shown as a horizontal line across the graph. The bottom graph shows the additive value toward the phenotype of each genomic region with respect to the L<em>er</em> allele. (C) Two-dimensional QTL scan for the root length ratio from trial 1. The <em>x</em>- and <em>y</em>-axes represent positions along each chromosome. The region of the plot below the diagonal gives the additive QTL model, while the region above the diagonal shows epistatic interactions. A scale relating colour to LOD score is provided on the right of the heat map. Numbers on the left of this scale represent LOD scores for the epistatic portion of the plot, whereas those on the right represent scores for the additive model. The arrows indicate the significance thresholds for the epistatic (left) and additive (right) portions of the plot.</p></figcaption></figure><p>Using both composite interval mapping and a 2D scan approach, a strong QTL for relative root growth response to cadaverine was identified at the end of chromosome 1 in two independent trials (<a class="usa-link" href="#F2">Fig. 2B</a>). The additive portion of the QTL graph (<a class="usa-link" href="#F2">Fig. 2B</a>, bottom) predicts that having a L<em>er</em> allele at this location results in a shorter root upon cadaverine treatment compared with a Cvi allele. No epistasis was observed between this QTL and other parts of the genome (<a class="usa-link" href="#F2">Fig. 2C</a>).</p></section><section id="s13"><h3 class="pmc_sec_title">Near isogenic lines confirm the chromosome 1 QTL</h3>
<p>Six NILs with Cvi introgression of varying segments of chromosome 1 within an otherwise L<em>er</em> background (<a aria-describedby="CIT0020" class="usa-link" href="#CIT0020">Keurentjes <em>et al.</em>, 2007</a>) were used to confirm the root length QTL. The four LCN1 lines that contain a Cvi introgression at the end of chromosome 1 showed greater, more Cvi-like resistance to cadaverine than LCN1-12 and LCN1-22, the NILs that do not contain a Cvi introgression in this region (<a class="usa-link" href="#F3">Fig. 3</a>). The L<em>er</em> to Cvi breakpoints in these lines were fine-mapped, and the candidate region was narrowed to a 5.3Mb interval at the end of chromosome 1 (primers shown in <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Table S1</a> at <em>JXB</em> online).</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4321547_exbotj_eru444_f0003.jpg" target="_blank"><img alt="Fig. 3." class="graphic zoom-in" height="789" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f376/4321547/060a6f43f6a8/exbotj_eru444_f0003.jpg" width="751"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Cadaverine root length responses of chromosome 1 NILs and fine mapping. Relative root lengths on media containing 0, 100, and 500 μM cadaverine are shown. Plants were grown at a 30 ° backward tilt for 6 d. The average root length on control medium was set to 1. Absolute average root lengths on control medium were 1.0, 1.2, 1.2, 1.2, 1.1, 1.2, 1.1, and 1.1cm in the lines as shown from left to right. The asterisk indicates the first concentration on which the root length was significantly different from the length on control medium (<em>P</em>&lt;0.05, Student’s <em>t</em>-test). Error bars indicate ±SE, and <em>n</em>=18–34. In the diagram of chromosome 1 NILs (bottom), white bars are L<em>er</em> chromosomal regions, black bars are Cvi regions, and grey bars indicate regions in which the breakpoint is undetermined. The parallel dashed lines flank the probable region of a locus controlling root length responses to cadaverine. Accession breakpoints were determined using primers based on polymorphisms described by Monsanto/CEREON between Columbia and L<em>er</em> (<a aria-describedby="CIT0019" class="usa-link" href="#CIT0019">Jander <em>et al.</em>, 2002</a>).</p></figcaption></figure></section><section id="s14"><h3 class="pmc_sec_title">Identification of candidate genes for the chromosome 1 QTL</h3>
<p>Because many of the causative loci behind the QTL are due to differences in gene expression (<a aria-describedby="CIT0002" class="usa-link" href="#CIT0002">Alonso-Blanco <em>et al.</em>, 2009</a>), gene expression differences between Cvi and L<em>er</em> were examined in the candidate region using previously published microarray data (<a aria-describedby="CIT0045" class="usa-link" href="#CIT0045">Vaughn and Masson, 2011</a>). The primary data are publicly available at Gene Expression Omnibus (NCBI; accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE28275" rel="noopener noreferrer" target="_blank">GSE28275</a>). From these arrays, 19 probe sets representing genes located in the candidate chromosomal segment give hybridization signals that differ significantly (95% confidence interval) by ≥2-fold between Cvi and L<em>er</em> (<a class="usa-link" href="#T1">Table 1</a>). The largest predicted differentially expressed gene in this region was <em>OCT1</em>, which was estimated to show a 73-fold increase in Cvi compared with L<em>er</em>.</p>
<section class="tw xbox font-sm" id="T1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Probe sets on chromosome 1 between 25.3Mb and 30.6Mb that show ≥2-fold differential expression between Cvi and L<em>er</em> at the 95% confidence level</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="vsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="1">Probe set ID</th>
<th align="left" colspan="1" rowspan="1">Locus ID</th>
<th align="left" colspan="1" rowspan="1">L<em>er</em> over Cvi fold change</th>
<th align="left" colspan="2" rowspan="1">Log<sub>2</sub> expression level</th>
<th align="left" colspan="1" rowspan="1">Description</th>
</tr>
<tr>
<th colspan="1" rowspan="1">
</th>
<th align="left" colspan="1" rowspan="1">
</th>
<th align="left" colspan="1" rowspan="1">
</th>
<th align="left" colspan="1" rowspan="1">L<em>er</em>
</th>
<th align="left" colspan="1" rowspan="1">Cvi</th>
<th colspan="1" rowspan="1">
</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">260097_at</td>
<td colspan="1" rowspan="1">AT1G73220</td>
<td colspan="1" rowspan="1">72.7 down</td>
<td colspan="1" rowspan="1">3.7</td>
<td colspan="1" rowspan="1">9.9</td>
<td colspan="1" rowspan="1">Organic cation transporter (OCT1)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">264100_at</td>
<td colspan="1" rowspan="1">AT1G78970</td>
<td colspan="1" rowspan="1">48.5 down</td>
<td colspan="1" rowspan="1">1.9</td>
<td colspan="1" rowspan="1">7.5</td>
<td colspan="1" rowspan="1">Lupeol synthase (LUP1)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">245215_at</td>
<td colspan="1" rowspan="1">AT1G67830</td>
<td colspan="1" rowspan="1">6.0 down</td>
<td colspan="1" rowspan="1">5.9</td>
<td colspan="1" rowspan="1">8.5</td>
<td colspan="1" rowspan="1">α-Fucosidase (FXG1)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">245736_at</td>
<td colspan="1" rowspan="1">AT1G73330</td>
<td colspan="1" rowspan="1">5.1 down</td>
<td colspan="1" rowspan="1">10.9</td>
<td colspan="1" rowspan="1">13.3</td>
<td colspan="1" rowspan="1">Protease inhibitor (DR4)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">259941_s_at</td>
<td colspan="1" rowspan="1">AT1G71280</td>
<td colspan="1" rowspan="1">4.4 down</td>
<td colspan="1" rowspan="1">5.2</td>
<td colspan="1" rowspan="1">7.4</td>
<td colspan="1" rowspan="1">DEAD/DEAH box helicase</td>
</tr>
<tr>
<td colspan="1" rowspan="1">260806_at</td>
<td colspan="1" rowspan="1">AT1G78260</td>
<td colspan="1" rowspan="1">2.9 down</td>
<td colspan="1" rowspan="1">9.4</td>
<td colspan="1" rowspan="1">11.0</td>
<td colspan="1" rowspan="1">RNA-binding protein</td>
</tr>
<tr>
<td colspan="1" rowspan="1">260401_at</td>
<td colspan="1" rowspan="1">AT1G69840</td>
<td colspan="1" rowspan="1">2.5 down</td>
<td colspan="1" rowspan="1">8.9</td>
<td colspan="1" rowspan="1">10.2</td>
<td colspan="1" rowspan="1">Unknown protein</td>
</tr>
<tr>
<td colspan="1" rowspan="1">264144_at</td>
<td colspan="1" rowspan="1">AT1G79320</td>
<td colspan="1" rowspan="1">2.2 down</td>
<td colspan="1" rowspan="1">7.4</td>
<td colspan="1" rowspan="1">8.5</td>
<td colspan="1" rowspan="1">Metacaspase (MC6)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">259705_at</td>
<td colspan="1" rowspan="1">AT1G77450</td>
<td colspan="1" rowspan="1">2.0 up</td>
<td colspan="1" rowspan="1">12.1</td>
<td colspan="1" rowspan="1">11.1</td>
<td colspan="1" rowspan="1">NAC transcription factor (NAC032)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">259977_at</td>
<td colspan="1" rowspan="1">AT1G76590</td>
<td colspan="1" rowspan="1">3.1 up</td>
<td colspan="1" rowspan="1">8.3</td>
<td colspan="1" rowspan="1">6.7</td>
<td colspan="1" rowspan="1">Zinc-binding protein</td>
</tr>
<tr>
<td colspan="1" rowspan="1">261901_at</td>
<td colspan="1" rowspan="1">AT1G80920</td>
<td colspan="1" rowspan="1">3.4 up</td>
<td colspan="1" rowspan="1">12.4</td>
<td colspan="1" rowspan="1">10.6</td>
<td colspan="1" rowspan="1">DNA-J protein (J8)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">260411_at</td>
<td colspan="1" rowspan="1">AT1G69890</td>
<td colspan="1" rowspan="1">3.4 up</td>
<td colspan="1" rowspan="1">7.6</td>
<td colspan="1" rowspan="1">5.9</td>
<td colspan="1" rowspan="1">Unknown protein</td>
</tr>
<tr>
<td colspan="1" rowspan="1">264957_at</td>
<td colspan="1" rowspan="1">AT1G77000</td>
<td colspan="1" rowspan="1">3.7 up</td>
<td colspan="1" rowspan="1">10.1</td>
<td colspan="1" rowspan="1">8.2</td>
<td colspan="1" rowspan="1">F-box protein (SKP2B)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">262362_at</td>
<td colspan="1" rowspan="1">AT1G72840</td>
<td colspan="1" rowspan="1">4.0 up</td>
<td colspan="1" rowspan="1">7.4</td>
<td colspan="1" rowspan="1">5.4</td>
<td colspan="1" rowspan="1">ATP-binding transmembrane receptor</td>
</tr>
<tr>
<td colspan="1" rowspan="1">260207_at</td>
<td colspan="1" rowspan="1">AT1G70730</td>
<td colspan="1" rowspan="1">4.7 up</td>
<td colspan="1" rowspan="1">10.8</td>
<td colspan="1" rowspan="1">8.6</td>
<td colspan="1" rowspan="1">Phosphoglucomutase</td>
</tr>
<tr>
<td colspan="1" rowspan="1">245731_at</td>
<td colspan="1" rowspan="1">AT1G73500</td>
<td colspan="1" rowspan="1">8.9 up</td>
<td colspan="1" rowspan="1">10.1</td>
<td colspan="1" rowspan="1">6.9</td>
<td colspan="1" rowspan="1">MAP kinase kinase (MKK9)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">260101_at</td>
<td colspan="1" rowspan="1">AT1G73260</td>
<td colspan="1" rowspan="1">12.2 up</td>
<td colspan="1" rowspan="1">12.6</td>
<td colspan="1" rowspan="1">9.0</td>
<td colspan="1" rowspan="1">Trypsin protease inhibitor</td>
</tr>
<tr>
<td colspan="1" rowspan="1">259866_at</td>
<td colspan="1" rowspan="1">AT1G76640</td>
<td colspan="1" rowspan="1">14.3 up</td>
<td colspan="1" rowspan="1">7.4</td>
<td colspan="1" rowspan="1">3.6</td>
<td colspan="1" rowspan="1">Calmodulin-related protein</td>
</tr>
<tr>
<td colspan="1" rowspan="1">264720_at</td>
<td colspan="1" rowspan="1">AT1G70080</td>
<td colspan="1" rowspan="1">20.8 up</td>
<td colspan="1" rowspan="1">10.3</td>
<td colspan="1" rowspan="1">5.9</td>
<td colspan="1" rowspan="1">Terpene synthase</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn0001"><p>Abbreviations: NIL, near isogenic line; OCT, organic cation transporter; RIL, recombinant inbred line.</p></div></div></section><p>No other genes in <a class="usa-link" href="#T1">Table 1</a> encode proteins that are directly relevant to polyamine metabolism or transport. Genes encoding a transglutaminase (<em>At1g69820</em>) and a cinnamyl alcohol dehydrogenase (<em>At1g72680</em>), both possibly linked to the formation of polyamine conjugates (<a aria-describedby="CIT0038" class="usa-link" href="#CIT0038">Serafini-Fracassini and Del Duca, 2008</a>; <a aria-describedby="CIT0014" class="usa-link" href="#CIT0014">Fellenberg <em>et al.</em>, 2009</a>), are found in this QTL interval; however, they do not show ≥2-fold expression differences between Cvi and L<em>er</em>.</p>
<p>
<em>OCT1</em> from Cvi and L<em>er</em> genomic DNA was sequenced. No large insertions or deletions were found, and only one amino acid change was found: residue 431 in the second exon is serine in Cvi and alanine in L<em>er</em> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Table S2</a> at <em>JXB</em> online). Analysing the 2kb of sequence upstream of the start codon revealed numerous single nucleotide polymorphisms and small insertions but no large insertions or deletions (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Table S2</a>). Based on these data, it was hypothesized that <em>OCT1</em> expression differences contribute to root length responses to exogenous cadaverine. qRT-PCR data confirmed the microarray result and showed that <em>OCT1</em> expression was ~14 times as high in Cvi roots as in L<em>er</em> roots (<a class="usa-link" href="#F4">Fig. 4</a>). Cadaverine treatment did not significantly change <em>OCT1</em> expression in either accession (<a class="usa-link" href="#F4">Fig. 4</a>).</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4321547_exbotj_eru444_f0004.jpg" target="_blank"><img alt="Fig. 4." class="graphic zoom-in" height="476" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f376/4321547/93c6bb89fd8d/exbotj_eru444_f0004.jpg" width="723"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>
<em>OCT1</em> expression relative to the reference gene <em>At1G58050</em> is higher in Cvi roots than in L<em>er</em> roots. Seedlings were grown on 0 μM or 50 μM cadaverine. Error bars indicate ±SE among biological replicates, and <em>OCT1</em> expression for L<em>er</em> on control medium was set to 1. The asterisk indicates a significant difference in expression compared with L<em>er</em> for the given condition (<em>P</em>&lt;0.05, Student’s <em>t</em>-test). Differences within each accession between cadaverine treatments are not significant.</p></figcaption></figure></section><section id="s15"><h3 class="pmc_sec_title">
<em>oct1-1</em> mutants express a truncated <em>OCT1</em> transcript</h3>
<p>To evaluate a role for <em>OCT1</em> in root length responses to cadaverine, the <em>oct1-1</em> mutant, which was previously found to carry a single T-DNA insertion near the 3’ end of the gene, was obtained; both ends of the insertion were characterized (<a aria-describedby="CIT0024" class="usa-link" href="#CIT0024">Lelandais-Brière <em>et al.</em>, 2007</a>). This mutant was also reported to not express any <em>OCT1</em> RNA, as determined by RT–PCR using primers aligning on each side of the intron (<a aria-describedby="CIT0024" class="usa-link" href="#CIT0024">Lelandais-Brière <em>et al.</em>, 2007</a>). To confirm this result, RT–PCR was performed on <em>oct1-1</em> and (Wassilewskija) WS wild-type root RNA samples. As expected, PCR products were obtained from WS wild-type cDNA samples of the expected sizes using multiple primer sets (1F/1R, 2F/2R, and 3F/3R) (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Fig. S2</a> at <em>JXB</em> online). Also, no transcript was observed for <em>oct1-1</em> mutants using primer sets near the 3′ end of the gene (2F/2R, 3F/3R, and 2F/4R) (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Fig. S2</a>).</p>
<p>However, it was found that <em>oct1-1</em> plants do express a partial <em>OCT1</em> transcript using primers on each side of the intron (1F/1R) (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Fig. S2</a> at <em>JXB</em> online). This product size corresponds to approximately the predicted length if the intron is not spliced out of the RNA (238 bases instead of 135 bases), which would result in a premature stop codon in the intron. No product for any primer set was detected in control reactions without reverse transcriptase, which confirms the lack of genomic DNA contamination (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Fig. S2</a>). Therefore, <em>oct1-1</em> produces a truncated <em>OCT1</em> transcript that is highly unlikely to encode a functional protein.</p></section><section id="s16"><h3 class="pmc_sec_title">
<em>oct1-1</em> shows increased cadaverine sensitivity</h3>
<p>If L<em>er</em> roots have higher sensitivity to cadaverine compared with Cvi roots because they have lower <em>OCT1</em> expression, then <em>oct1</em> mutant roots should also respond more strongly to cadaverine than wild-type plants. To test this, WS wild type, <em>oct1-1</em>, and two independent transgenic rescue lines (<em>oct1-1</em> [35S<sub>pro</sub>:OCT1<sup>WS</sup>]) were grown on control medium and medium supplemented with cadaverine. Rescue line 1 had significantly higher <em>OCT1</em> expression compared with the WS wild type, whereas rescue line 2 had a similar level of <em>OCT1</em> expression (<a class="usa-link" href="#F5">Fig. 5A</a>). It was found that all of these plants had the same root length on control medium lacking cadaverine, but <em>oct1-1</em> mutant roots were significantly shorter than wild-type roots on media containing 50 μM and 500 μM cadaverine (<a class="usa-link" href="#F5">Fig. 5B</a>). Robust root growth on medium containing cadaverine was restored in both independent transgenic rescue lines. Rescue line 2 showed a wild-type-like response to cadaverine, and rescue line 1 showed increased resistance to cadaverine (<a class="usa-link" href="#F5">Fig. 5B</a>). These results suggest that reduced <em>OCT1</em> expression confers increased cadaverine sensitivity.</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4321547_exbotj_eru444_f0005.jpg" target="_blank"><img alt="Fig. 5." class="graphic zoom-in" height="1517" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f376/4321547/66511d261467/exbotj_eru444_f0005.jpg" width="680"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>
<em>oct1-1</em> roots are more sensitive to exogenous cadaverine compared with wild-type plants but respond similarly to putrescine-derived polyamines. (A) <em>OCT1</em> expression relative to the reference gene <em>At1G58050</em> is shown. Error bars indicate ±SE among biological replicates, and <em>OCT1</em> expression for the WS wild type on control medium was set to 1. The asterisk indicates a significant difference in expression compared with the WS wild type (<em>P</em>&lt;0.05, Student’s <em>t</em>-test). The abbreviation n.d. indicates that expression was not detected. (B) Plants were grown for 6 d at a 30 ° backward tilt on media containing 0, 50, and 500 μM cadaverine. Root tips were marked after 3 d of growth, and the root growth over the following 3 d was measured for each treatment. Average root growth over the final 3 d on control medium was 1.2cm for the wild type, 1.4cm for <em>oct1-1</em>, 1.3cm for rescue line 1, and 1.4cm for rescue line 2; these lengths were set to 1. For each cadaverine concentration, the letters indicate significant differences in relative root length (<em>P</em>&lt;0.01, pairwise Student’s <em>t</em>-tests). <em>n</em>=37–68, and error bars indicate ±SE. (C) Plants were grown for 6 d at a 30 ° backward tilt on media containing various concentrations of polyamines as indicated. Average root length on control medium was 1.3cm for the wild type and 1.4cm for oct1-1; these lengths were set to 1. <em>n</em>=33–44, and error bars indicate ±SE. No significant root length differences were observed for any treatment other than cadaverine (<em>P</em>&lt;0.05, Student’s <em>t</em>-test).</p></figcaption></figure><p>The WS wild type and <em>oct1-1</em> had similar root lengths on media containing the polyamines putrescine, spermidine, and spermine (<a class="usa-link" href="#F5">Fig. 5C</a>), which shows that this sensitivity is specific to cadaverine. Polyamines have been shown to shorten root growth through the generation of H<sub>2</sub>O<sub>2</sub> by diamine and polyamine oxidases, which promotes premature cell differentiation and programmed cell death (<a aria-describedby="CIT0044" class="usa-link" href="#CIT0044">Tisi <em>et al.</em>, 2011</a>). However, because the cadaverine-hypersensitive response of <em>oct1-1</em> was specific to cadaverine (<a class="usa-link" href="#F5">Fig. 5C</a>), OCT1 probably functions in a different aspect of polyamine-mediated root growth regulation.</p></section><section id="s17"><h3 class="pmc_sec_title">The expression level of <em>OCT1</em>, not its accession origin, correlates with cadaverine sensitivity</h3>
<p>To determine if differences in the <em>OCT1</em> sequence between Cvi and L<em>er</em> or only <em>OCT1</em> expression levels correlated with the cadaverine response, <em>OCT1</em> was cloned from Cvi and L<em>er</em>, and these constructs were introduced into <em>oct1-1</em> mutants. These transgenic lines were then tested on media containing 0 μM and 50 μM cadaverine, and their relative root lengths were measured. A range of cadaverine resistance levels that did not correlate with the accession source of the transgene was found (<a class="usa-link" href="#F6">Fig. 6A</a>).</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4321547_exbotj_eru444_f0006.jpg" target="_blank"><img alt="Fig. 6." class="graphic zoom-in" height="1113" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f376/4321547/d05372365ee9/exbotj_eru444_f0006.jpg" width="680"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Higher <em>OCT1</em> expression correlates with cadaverine resistance. OCT1<sup>Cvi</sup> and OCT1<sup>L<em>er</em></sup> indicate <em>oct1-1</em> [OCT1<sub>pro</sub>:OCT1<sup>Cvi</sup>] and <em>oct1-1</em> [OCT1<sub>pro</sub>:OCT1<sup>L<em>er</em></sup>], respectively. (A) Relative root lengths of plants on media containing 0 μM and 50 μM cadaverine. Plants were grown for 6 d at a 30 ° backward tilt. Average root length on control medium was set to 1. Absolute average root length on control medium was 2.2cm for Cvi, 1.9cm for L<em>er</em>, 2.0cm for WS, 2.0cm for <em>oct1-1</em>, and 1.9, 2.0, 1.7, and 2.1cm for the transgenic lines as shown from left to right. Error bars indicate ±SE, and <em>n</em>=17–24. (B) Root <em>OCT1</em> expression determined by qRT-PCR is shown relative to the reference gene <em>At1G58050</em>. <em>OCT1</em> expression in L<em>er</em> was set to 1. Error bars indicate ±SE among the biological replicates. The abbreviation n.d. indicates that expression was not detected.</p></figcaption></figure><p>Because the insertion location can affect transgene expression, it was predicted that the <em>OCT1</em> expression levels would correlate with cadaverine sensitivity. To test this, <em>OCT1</em> expression was measured in four of these lines by qRT-PCR. As expected, a range of <em>OCT1</em> expression levels was found in the transgenic lines regardless of whether the <em>OCT1</em> untranslated and coding regions came from Cvi or L<em>er</em>. In general, lines with increased resistance to cadaverine had higher <em>OCT1</em> expression (<a class="usa-link" href="#F6">Fig. 6</a>).</p></section><section id="s18"><h3 class="pmc_sec_title">Genes other than <em>OCT1</em> probably also contribute to the cadaverine response</h3>
<p>The data suggest that differences in <em>OCT1</em> expression contribute significantly to the different cadaverine root length responses in Cvi and L<em>er</em>. To determine if differences in <em>OCT1</em> expression among other natural accessions also correlate with variability in cadaverine sensitivity, <em>OCT1</em> expression was measured in two other accessions, one resistant to cadaverine (Knox-18) and one sensitive to cadaverine (Sq-8). It was found that both lines had higher <em>OCT1</em> expression than L<em>er</em> even though only Knox-18 was more resistant to cadaverine than L<em>er</em>. Furthermore, Sq-8 displayed higher <em>OCT1</em> expression than Knox-18 despite being more sensitive to cadaverine (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Fig. S3</a> at <em>JXB</em> online). The <em>OCT1</em> sequences of the eight natural accessions initially selected for the analysis of cadaverine response were also compared. Polymorphisms shared among the three most resistant accessions (Cvi-1, Knox-18, and Ull2-3) or the three most sensitive accessions (An-1, L<em>er</em>-2, and Sq-8) were not seen (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Table S2</a>). Together, these data suggest that factors other than <em>OCT1</em> expression may also contribute significantly to differential cadaverine sensitivity among these accessions.</p>
<p>Ectopic copies of <em>OCT1</em> derived from Cvi or L<em>er</em> were also introduced into Columbia and L<em>er</em> wild-type plants. L<em>er</em> root length is highly sensitive to cadaverine, whereas Columbia root length is less sensitive to it. In general, the Columbia plants showed a range of responses to cadaverine regardless of the accession from which the transgene was derived (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Fig. S4</a> at <em>JXB</em> online). This result is similar to that obtained when these constructs were transformed into the <em>oct1-1</em> mutant (<a class="usa-link" href="#F6">Fig. 6</a>). In contrast, when these constructs were transformed into L<em>er</em> plants, the tested lines generally did not show increased resistance to cadaverine (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Supplementary Fig. S4</a>). Therefore, the ability of <em>OCT1</em> to affect the cadaverine response may depend on the genetic background.</p></section></section><section id="s19"><h2 class="pmc_sec_title">Discussion</h2>
<p>It was found that <em>Arabidopsis</em> roots exhibited varying responses to exogenously applied cadaverine depending on the accession. In general, when cadaverine concentrations increased, a reduction in root length as well as alterations in skewing, lateral root branching, and root waving were seen. The reduction in root length is probably due to a combination of reduced cell division and expansion. A major QTL was identified on chromosome 1 that at least partially explains the root length difference in response to cadaverine between Cvi and L<em>er</em>. By comparing phenotypes and gene expression in natural accessions, a mutant, and transgenic lines, <em>OCT1</em> was identified as a key gene involved in this response. Restoring <em>OCT1</em> expression in the mutant background resulted in a positive correlation between <em>OCT1</em> expression levels and cadaverine resistance. Because the <em>OCT1</em> expression levels in other natural accessions do not correlate well with cadaverine resistance, other genes are also likely to contribute to the variation of this trait between <em>Arabidopsis</em> accessions. These genes could contribute to endogenous cadaverine homeostasis, transport, and/or response. Unfortunately, the levels of endogenous cadaverine in <em>Arabidopsis</em> seedling roots and shoots were below the level of detection using HPLC methods, preventing a distinction from being made between these possibilities.</p>
<p>OCT1 encodes an organic cation transporter. In animals, OCT proteins take up a wide variety of endogenous compounds and xenobiotics, and, therefore, they affect substrate absorption, distribution, and excretion in many tissues. Most are polyspecific, and transport can occur in either direction (for a review, see <a aria-describedby="CIT0036" class="usa-link" href="#CIT0036">Roth <em>et al.</em>, 2012</a>). <em>Arabidopsis</em> OCT1 is one of six OCT-like proteins in <em>Arabidopsis</em> (<a aria-describedby="CIT0024" class="usa-link" href="#CIT0024">Lelandais-Brière <em>et al.</em>, 2007</a>). It was previously reported to localize to the plasma membrane, and promoter–β-glucuronidase (GUS) fusions showed activity in the vasculature of many organs including roots (<a aria-describedby="CIT0024" class="usa-link" href="#CIT0024">Lelandais-Brière <em>et al.</em>, 2007</a>; <a aria-describedby="CIT0022" class="usa-link" href="#CIT0022">Küfner and Koch, 2008</a>). Because the other six OCT-like proteins localize to the tonoplast rather than the plasma membrane, it is likely that OCT1 has a different function.</p>
<p>The present work expands on these studies and suggests a role for OCT1 in affecting root growth responses to cadaverine. Future experiments will be aimed at clarifying the mechanism by which this occurs. One possibility is that OCT1 transports cadaverine away from elongating root cells and through the vasculature, causing it to accumulate in aerial tissues. Alternatively, cadaverine may alter the transport of another molecule through the membranes. Polyamines block many kinds of channels in multiple organisms. For example, spermine and spermidine directly block fast vacuolar cation channels in barley (<a aria-describedby="CIT0007" class="usa-link" href="#CIT0007">Bruggemann <em>et al.</em>, 1998</a>), and all natural polyamines including cadaverine can regulate cation channel activity through cytoplasmic pathways (<a aria-describedby="CIT0026" class="usa-link" href="#CIT0026">Liu <em>et al.</em>, 2000</a>; <a aria-describedby="CIT0039" class="usa-link" href="#CIT0039">Shabala <em>et al.</em>, 2007</a>).</p>
<p>It has been demonstrated that exogenous cadaverine affects <em>Arabidopsis</em> root morphology and that there is natural variation in this response due in part to varying expression levels of <em>OCT1</em>. This work has potential implications for better understanding how polyamines contribute to development and mediate stress responses.</p></section><section id="s20"><h2 class="pmc_sec_title">Supplementary data</h2>
<p>Supplementary data are available at <em>JXB</em> online.</p>
<p>
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Figure S1.</a>
<em>Arabidopsis</em> natural accessions show varying root growth behaviours in response to cadaverine.</p>
<p>
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Figure S2.</a>
<em>oct1-1</em> produces a truncated transcript.</p>
<p>
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Figure S3.</a>
<em>OCT1</em> expression does not correlate with cadaverine sensitivity in all natural accessions.</p>
<p>
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Figure S4.</a>
<em>OCT1</em> expression results in varying degrees of cadaverine sensitivity in Col and L<em>er</em>.</p>
<p>
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Table S1.</a> Primers for chromosome 1 fine mapping.</p>
<p>
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://jxb.oxfordjournals.org/lookup/suppl/doi:10.1093/jxb/eru444/-/DC1" rel="noopener noreferrer" target="_blank">Table S2.</a> Comparison of <em>OCT1</em> sequences from eight <em>Arabidopsis</em> natural accessions.</p>
<section class="sm xbox font-sm" id="PMC_1"><div class="caption p"><span>Supplementary Data</span></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4321547/bin/supp_66_3_853__index.html">supp_66_3_853__index.html</a><sup> (1KB, html) </sup>
</div></div></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by grants from the University of Wisconsin–Madison College of Agricultural and Life Sciences Hatch program and the National Science Foundation (IOS-0821884 and IOS-1121694) to PHM, by a National Science Foundation Graduate Research Fellowship to AKS, and by a National Institutes of Health Training Grant in Genetics at the University of Wisconsin–Madison to LMV. We thank Hiroshi Maeda for critically reviewing the manuscript and Christine Lelandais-Brière for giving us the <em>oct1-1</em> mutant and the <em>oct1-1</em> [35S<sub>pro</sub>:OCT1<sup>WS</sup>] rescue lines. We also thank Katherine Baldwin and Shih-Heng Su for help with figures and technical assistance.</p></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="CIT0001">
<cite>
Alet AI, Sánchez DH, Ferrando A, et al. 
2011. 
Homeostatic control of polyamine levels under long-term salt stress in Arabidopsis: changes in putrescine content do not alleviate ionic toxicity. Plant Signaling and Behavior
6, 237–242.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4161/psb.6.2.14214" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3121984/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21330788/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signaling%20and%20Behavior&amp;title=Homeostatic%20control%20of%20polyamine%20levels%20under%20long-term%20salt%20stress%20in%20Arabidopsis:%20changes%20in%20putrescine%20content%20do%20not%20alleviate%20ionic%20toxicity&amp;author=AI%20Alet&amp;author=DH%20S%C3%A1nchez&amp;author=A%20Ferrando&amp;volume=6&amp;publication_year=2011&amp;pages=237-242&amp;pmid=21330788&amp;doi=10.4161/psb.6.2.14214&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0002">
<cite>
Alonso-Blanco C, Aarts MGM, Bentsink L, Keurentjes JJB, Reymond M, Vreugdenhil D, Koornneef M.
2009. 
What has natural variation taught us about plant development, physiology, and adaptation?
The Plant Cell
21, 1877–1896.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.109.068114" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2729614/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19574434/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=What%20has%20natural%20variation%20taught%20us%20about%20plant%20development,%20physiology,%20and%20adaptation?&amp;author=C%20Alonso-Blanco&amp;author=MGM%20Aarts&amp;author=L%20Bentsink&amp;author=JJB%20Keurentjes&amp;author=M%20Reymond&amp;volume=21&amp;publication_year=2009&amp;pages=1877-1896&amp;pmid=19574434&amp;doi=10.1105/tpc.109.068114&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0003">
<cite>
Alonso-Blanco C, Peeters AJ, Koornneef M, Lister C, Dean C, van den Bosch N, Pot J, Kuiper MT.
1998. 
Development of an AFLP based linkage map of L<em>er</em>, Col and Cvi <em>Arabidopsis thaliana</em> ecotypes and construction of a L<em>er</em>/Cvi recombinant inbred line population. The Plant Journal
14, 259–271.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1365-313x.1998.00115.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9628021/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=Development%20of%20an%20AFLP%20based%20linkage%20map%20of%20Ler,%20Col%20and%20Cvi%20Arabidopsis%20thaliana%20ecotypes%20and%20construction%20of%20a%20Ler/Cvi%20recombinant%20inbred%20line%20population&amp;author=C%20Alonso-Blanco&amp;author=AJ%20Peeters&amp;author=M%20Koornneef&amp;author=C%20Lister&amp;author=C%20Dean&amp;volume=14&amp;publication_year=1998&amp;pages=259-271&amp;pmid=9628021&amp;doi=10.1046/j.1365-313x.1998.00115.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0004">
<cite>
Aouida M, Leduc A, Poulin R, Ramotar D.
2005. 

<em>AGP2</em> encodes the major permease for high affinity polyamine import in <em>Saccharomyces cerevisiae</em>
. Journal of Biological Chemistry
280, 24267–24276.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M503071200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15855155/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=AGP2%20encodes%20the%20major%20permease%20for%20high%20affinity%20polyamine%20import%20in%20Saccharomyces%20cerevisiae&amp;author=M%20Aouida&amp;author=A%20Leduc&amp;author=R%20Poulin&amp;author=D%20Ramotar&amp;volume=280&amp;publication_year=2005&amp;pages=24267-24276&amp;pmid=15855155&amp;doi=10.1074/jbc.M503071200&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0005">
<cite>
Aziz A, Martin-Tanguy J, Larher F.
1997. 
Plasticity of polyamine metabolism associated with high osmotic stress in rape leaf discs and with ethylene treatment. Plant Growth Regulation
21, 153–163.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Growth%20Regulation&amp;title=Plasticity%20of%20polyamine%20metabolism%20associated%20with%20high%20osmotic%20stress%20in%20rape%20leaf%20discs%20and%20with%20ethylene%20treatment&amp;author=A%20Aziz&amp;author=J%20Martin-Tanguy&amp;author=F%20Larher&amp;volume=21&amp;publication_year=1997&amp;pages=153-163&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0006">
<cite>
Bagni N, Tassoni A.
2001. 
Biosynthesis, oxidation and conjugation of aliphatic polyamines in higher plants. Amino Acids
20, 301–317.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s007260170046" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11354606/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Amino%20Acids&amp;title=Biosynthesis,%20oxidation%20and%20conjugation%20of%20aliphatic%20polyamines%20in%20higher%20plants&amp;author=N%20Bagni&amp;author=A%20Tassoni&amp;volume=20&amp;publication_year=2001&amp;pages=301-317&amp;pmid=11354606&amp;doi=10.1007/s007260170046&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0007">
<cite>
Bruggemann LI, Pottosin II, Schonknecht G.
1998. 
Cytoplasmic polyamines block the fast-activating vacuolar cation channel. The Plant Journal
16, 101–105.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=Cytoplasmic%20polyamines%20block%20the%20fast-activating%20vacuolar%20cation%20channel&amp;author=LI%20Bruggemann&amp;author=II%20Pottosin&amp;author=G%20Schonknecht&amp;volume=16&amp;publication_year=1998&amp;pages=101-105&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0008">
<cite>
Cassán F, Maiale S, Masciarelli O, Vidal A, Luna V, Ruiz O.
2009. 
Cadaverine production by <em>Azospirillum brasilense</em> and its possible role in plant growth promotion and osmotic stress mitigation. European Journal of Soil Biology
45, 12–19.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Soil%20Biology&amp;title=Cadaverine%20production%20by%20Azospirillum%20brasilense%20and%20its%20possible%20role%20in%20plant%20growth%20promotion%20and%20osmotic%20stress%20mitigation&amp;author=F%20Cass%C3%A1n&amp;author=S%20Maiale&amp;author=O%20Masciarelli&amp;author=A%20Vidal&amp;author=V%20Luna&amp;volume=45&amp;publication_year=2009&amp;pages=12-19&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0009">
<cite>
Cloete KJ, Valentine AJ, Stander MA, Blomerus LM, Botha A.
2009. 
Evidence of symbiosis between the soil yeast <em>Cryptococcus laurentii</em> and a sclerophyllous medicinal shrub, <em>Agathosma betulina</em> (Berg.) Pillans. Microbial Ecology
57, 624–632.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00248-008-9457-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18958514/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbial%20Ecology&amp;title=Evidence%20of%20symbiosis%20between%20the%20soil%20yeast%20Cryptococcus%20laurentii%20and%20a%20sclerophyllous%20medicinal%20shrub,%20Agathosma%20betulina%20(Berg.)%20Pillans&amp;author=KJ%20Cloete&amp;author=AJ%20Valentine&amp;author=MA%20Stander&amp;author=LM%20Blomerus&amp;author=A%20Botha&amp;volume=57&amp;publication_year=2009&amp;pages=624-632&amp;pmid=18958514&amp;doi=10.1007/s00248-008-9457-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0010">
<cite>
Clough SJ, Bent AF.
1998. 
Floral dip: a simplified method for Agrobacterium-mediated transformation of <em>Arabidopsis thaliana</em>
. The Plant Journal
16, 735–743.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1365-313x.1998.00343.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10069079/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=Floral%20dip:%20a%20simplified%20method%20for%20Agrobacterium-mediated%20transformation%20of%20Arabidopsis%20thaliana&amp;author=SJ%20Clough&amp;author=AF%20Bent&amp;volume=16&amp;publication_year=1998&amp;pages=735-743&amp;pmid=10069079&amp;doi=10.1046/j.1365-313x.1998.00343.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0011">
<cite>
Cuevas JC, López-Cobollo R, Alcázar R, Zarza X, Koncz C, Altabella T, Salinas J, Tiburcio AF, Ferrando A.
2008. 
Putrescine is involved in Arabidopsis freezing tolerance and cold acclimation by regulating abscisic acid levels in response to low temperature. Plant Physiology
148, 1094–1105.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.108.122945" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2556839/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18701673/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Putrescine%20is%20involved%20in%20Arabidopsis%20freezing%20tolerance%20and%20cold%20acclimation%20by%20regulating%20abscisic%20acid%20levels%20in%20response%20to%20low%20temperature&amp;author=JC%20Cuevas&amp;author=R%20L%C3%B3pez-Cobollo&amp;author=R%20Alc%C3%A1zar&amp;author=X%20Zarza&amp;author=C%20Koncz&amp;volume=148&amp;publication_year=2008&amp;pages=1094-1105&amp;pmid=18701673&amp;doi=10.1104/pp.108.122945&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0012">
<cite>
Curtis MD, Grossniklaus U.
2003. 
A gateway cloning vector set for high-throughput functional analysis of genes in planta. Plant Physiology
133, 462–469.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.103.027979" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC523872/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14555774/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=A%20gateway%20cloning%20vector%20set%20for%20high-throughput%20functional%20analysis%20of%20genes%20in%20planta&amp;author=MD%20Curtis&amp;author=U%20Grossniklaus&amp;volume=133&amp;publication_year=2003&amp;pages=462-469&amp;pmid=14555774&amp;doi=10.1104/pp.103.027979&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0013">
<cite>
Czechowski T, Stitt M, Altmann T, Udvardi MK, Scheible W-R.
2005. 
Genome-wide identification and testing of superior reference genes for transcript normalization in Arabidopsis. Plant Physiology
139, 5–17.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.105.063743" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1203353/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16166256/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Genome-wide%20identification%20and%20testing%20of%20superior%20reference%20genes%20for%20transcript%20normalization%20in%20Arabidopsis&amp;author=T%20Czechowski&amp;author=M%20Stitt&amp;author=T%20Altmann&amp;author=MK%20Udvardi&amp;author=W-R%20Scheible&amp;volume=139&amp;publication_year=2005&amp;pages=5-17&amp;pmid=16166256&amp;doi=10.1104/pp.105.063743&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0014">
<cite>
Fellenberg C, Böttcher C, Vogt T.
2009. 
Phenylpropanoid polyamine conjugate biosynthesis in <em>Arabidopsis thaliana</em> flower buds. Phytochemistry
70, 1392–1400.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.phytochem.2009.08.010" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19762055/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Phytochemistry&amp;title=Phenylpropanoid%20polyamine%20conjugate%20biosynthesis%20in%20Arabidopsis%20thaliana%20flower%20buds&amp;author=C%20Fellenberg&amp;author=C%20B%C3%B6ttcher&amp;author=T%20Vogt&amp;volume=70&amp;publication_year=2009&amp;pages=1392-1400&amp;pmid=19762055&amp;doi=10.1016/j.phytochem.2009.08.010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0015">
<cite>
Fujita M, Fujita Y, Iuchi S, Yamada K, Kobayashi Y, Urano K, Kobayashi M, Yamaguchi-Shinozaki K, Shinozaki K.
2012. 
Natural variation in a polyamine transporter determines paraquat tolerance in Arabidopsis. Proceedings of the National Academy of Sciences, USA
109, 6343–6347.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1121406109" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3341036/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22492932/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences,%20USA&amp;title=Natural%20variation%20in%20a%20polyamine%20transporter%20determines%20paraquat%20tolerance%20in%20Arabidopsis&amp;author=M%20Fujita&amp;author=Y%20Fujita&amp;author=S%20Iuchi&amp;author=K%20Yamada&amp;author=Y%20Kobayashi&amp;volume=109&amp;publication_year=2012&amp;pages=6343-6347&amp;pmid=22492932&amp;doi=10.1073/pnas.1121406109&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0016">
<cite>
Gamarnik A, Frydman RB.
1991. 
Cadaverine, an essential diamine for the normal root development of germinating soybean (<em>Glycine max</em>) seeds. Plant Physiology
97, 778–785.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.97.2.778" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1081074/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16668466/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Cadaverine,%20an%20essential%20diamine%20for%20the%20normal%20root%20development%20of%20germinating%20soybean%20(Glycine%20max)%20seeds&amp;author=A%20Gamarnik&amp;author=RB%20Frydman&amp;volume=97&amp;publication_year=1991&amp;pages=778-785&amp;pmid=16668466&amp;doi=10.1104/pp.97.2.778&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0017">
<cite>
Harrison SJ, Mott EK, Parsley K, Aspinall S, Gray JC, Cottage A.
2006. 
A rapid and robust method of identifying transformed <em>Arabidopsis thaliana</em> seedlings following floral dip transformation. Plant Methods
2, 19.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1746-4811-2-19" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1636043/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17087829/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Methods&amp;title=A%20rapid%20and%20robust%20method%20of%20identifying%20transformed%20Arabidopsis%20thaliana%20seedlings%20following%20floral%20dip%20transformation&amp;author=SJ%20Harrison&amp;author=EK%20Mott&amp;author=K%20Parsley&amp;author=S%20Aspinall&amp;author=JC%20Gray&amp;volume=2&amp;publication_year=2006&amp;pages=19&amp;pmid=17087829&amp;doi=10.1186/1746-4811-2-19&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0018">
<cite>
Hussain SS, Ali M, Ahmad M, Siddique KHM.
2011. 
Polyamines: natural and engineered abiotic and biotic stress tolerance in plants. Biotechnology Advances
29, 300–311.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.biotechadv.2011.01.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21241790/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biotechnology%20Advances&amp;title=Polyamines:%20natural%20and%20engineered%20abiotic%20and%20biotic%20stress%20tolerance%20in%20plants&amp;author=SS%20Hussain&amp;author=M%20Ali&amp;author=M%20Ahmad&amp;author=KHM%20Siddique&amp;volume=29&amp;publication_year=2011&amp;pages=300-311&amp;pmid=21241790&amp;doi=10.1016/j.biotechadv.2011.01.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0019">
<cite>
Jander G, Norris SR, Rounsley SD, Bush DF, Levin IM, Last RL.
2002. 
Arabidopsis map-based cloning in the post-genome era. Plant Physiology
129, 440–450.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.003533" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1540230/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12068090/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Arabidopsis%20map-based%20cloning%20in%20the%20post-genome%20era&amp;author=G%20Jander&amp;author=SR%20Norris&amp;author=SD%20Rounsley&amp;author=DF%20Bush&amp;author=IM%20Levin&amp;volume=129&amp;publication_year=2002&amp;pages=440-450&amp;pmid=12068090&amp;doi=10.1104/pp.003533&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0020">
<cite>
Keurentjes JJB, Bentsink L, Alonso-Blanco C, Hanhart CJ, Blankestijn-de Vries H, Effgen S, Vreugdenhil D, Koornneef M.
2007. 
Development of a near-isogenic line population of <em>Arabidopsis thaliana</em> and comparison of mapping power with a recombinant inbred line population. Genetics
175, 891–905.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1534/genetics.106.066423" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1800614/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17179089/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Development%20of%20a%20near-isogenic%20line%20population%20of%20Arabidopsis%20thaliana%20and%20comparison%20of%20mapping%20power%20with%20a%20recombinant%20inbred%20line%20population&amp;author=JJB%20Keurentjes&amp;author=L%20Bentsink&amp;author=C%20Alonso-Blanco&amp;author=CJ%20Hanhart&amp;author=H%20Blankestijn-de%20Vries&amp;volume=175&amp;publication_year=2007&amp;pages=891-905&amp;pmid=17179089&amp;doi=10.1534/genetics.106.066423&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0021">
<cite>
Knott JM, Römer P, Sumper M.
2007. 
Putative spermine synthases from <em>Thalassiosira pseudonana</em> and <em>Arabidopsis thaliana</em> synthesize thermospermine rather than spermine. FEBS Letters
581, 3081–3086.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.febslet.2007.05.074" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17560575/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Letters&amp;title=Putative%20spermine%20synthases%20from%20Thalassiosira%20pseudonana%20and%20Arabidopsis%20thaliana%20synthesize%20thermospermine%20rather%20than%20spermine&amp;author=JM%20Knott&amp;author=P%20R%C3%B6mer&amp;author=M%20Sumper&amp;volume=581&amp;publication_year=2007&amp;pages=3081-3086&amp;pmid=17560575&amp;doi=10.1016/j.febslet.2007.05.074&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0022">
<cite>
Küfner I, Koch W.
2008. 
Stress regulated members of the plant organic cation transporter family are localized to the vacuolar membrane. BMC Research Notes
1, 43.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1756-0500-1-43" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2519074/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18710496/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Research%20Notes&amp;title=Stress%20regulated%20members%20of%20the%20plant%20organic%20cation%20transporter%20family%20are%20localized%20to%20the%20vacuolar%20membrane&amp;author=I%20K%C3%BCfner&amp;author=W%20Koch&amp;volume=1&amp;publication_year=2008&amp;pages=43&amp;pmid=18710496&amp;doi=10.1186/1756-0500-1-43&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0023">
<cite>
Kuznetsov V, Shorina M, Aronova E, Stetsenko L, Rakitin V, Shevyakova N.
2007. 
NaCl- and ethylene-dependent cadaverine accumulation and its possible protective role in the adaptation of the common ice plant to salt stress. Plant Science
172, 363–370.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Science&amp;title=NaCl-%20and%20ethylene-dependent%20cadaverine%20accumulation%20and%20its%20possible%20protective%20role%20in%20the%20adaptation%20of%20the%20common%20ice%20plant%20to%20salt%20stress&amp;author=V%20Kuznetsov&amp;author=M%20Shorina&amp;author=E%20Aronova&amp;author=L%20Stetsenko&amp;author=V%20Rakitin&amp;volume=172&amp;publication_year=2007&amp;pages=363-370&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0024">
<cite>
Lelandais-Brière C, Jovanovic M, Torres GAM, Perrin Y, Lemoine R, Corre-Menguy F, Hartmann C.
2007. 
Disruption of <em>AtOCT1</em>, an organic cation transporter gene, affects root development and carnitine-related responses in Arabidopsis. The Plant Journal
51, 154–164.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2007.03131.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17521409/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=Disruption%20of%20AtOCT1,%20an%20organic%20cation%20transporter%20gene,%20affects%20root%20development%20and%20carnitine-related%20responses%20in%20Arabidopsis&amp;author=C%20Lelandais-Bri%C3%A8re&amp;author=M%20Jovanovic&amp;author=GAM%20Torres&amp;author=Y%20Perrin&amp;author=R%20Lemoine&amp;volume=51&amp;publication_year=2007&amp;pages=154-164&amp;pmid=17521409&amp;doi=10.1111/j.1365-313X.2007.03131.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0025">
<cite>
Li J, Mu J, Rai J, Fu F, et al. 
2013. 
PARAQUAT RESISTANT1, a Golgi-localized putative transporter protein, is involved in intracellular transport of paraquat. Plant Physiology
162, 470–483.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.113.213892" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3641224/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23471133/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=PARAQUAT%20RESISTANT1,%20a%20Golgi-localized%20putative%20transporter%20protein,%20is%20involved%20in%20intracellular%20transport%20of%20paraquat&amp;author=J%20Li&amp;author=J%20Mu&amp;author=J%20Rai&amp;author=F%20Fu&amp;volume=162&amp;publication_year=2013&amp;pages=470-483&amp;pmid=23471133&amp;doi=10.1104/pp.113.213892&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0026">
<cite>
Liu K, Fu H, Bei Q, Luan S.
2000. 
Inward potassium channel in guard cells as a target for polyamine regulation of stomatal movements. Plant Physiology
124, 1315–1326.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.124.3.1315" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC59229/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11080307/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Inward%20potassium%20channel%20in%20guard%20cells%20as%20a%20target%20for%20polyamine%20regulation%20of%20stomatal%20movements&amp;author=K%20Liu&amp;author=H%20Fu&amp;author=Q%20Bei&amp;author=S%20Luan&amp;volume=124&amp;publication_year=2000&amp;pages=1315-1326&amp;pmid=11080307&amp;doi=10.1104/pp.124.3.1315&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0027">
<cite>
Long Q, Rabanl FA, Meng D, et al. 
2013. 
Massive genomic variation and strong selection in Arabidopsis thaliana lines from Sweden. Nature Genetics
45, 884–890.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ng.2678" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3755268/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23793030/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Massive%20genomic%20variation%20and%20strong%20selection%20in%20Arabidopsis%20thaliana%20lines%20from%20Sweden&amp;author=Q%20Long&amp;author=FA%20Rabanl&amp;author=D%20Meng&amp;volume=45&amp;publication_year=2013&amp;pages=884-890&amp;pmid=23793030&amp;doi=10.1038/ng.2678&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0028">
<cite>
Meijering E, Jacob M, Sarria J-CF, Steiner P, Hirling H, Unser M.
2004. 
Design and validation of a tool for neurite tracing and analysis in fluorescence microscopy images. Cytometry
58A, 167–176.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/cyto.a.20022" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15057970/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cytometry&amp;title=Design%20and%20validation%20of%20a%20tool%20for%20neurite%20tracing%20and%20analysis%20in%20fluorescence%20microscopy%20images&amp;author=E%20Meijering&amp;author=M%20Jacob&amp;author=J-CF%20Sarria&amp;author=P%20Steiner&amp;author=H%20Hirling&amp;volume=58A&amp;publication_year=2004&amp;pages=167-176&amp;pmid=15057970&amp;doi=10.1002/cyto.a.20022&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0029">
<cite>
Mulangi V, Chibucos MC, Phuntumart V, Morris PF.
2012a. 
Kinetic and phylogenetic analysis of plant polyamine uptake transporters. Planta
236, 1261–1273.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00425-012-1668-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22711282/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Kinetic%20and%20phylogenetic%20analysis%20of%20plant%20polyamine%20uptake%20transporters&amp;author=V%20Mulangi&amp;author=MC%20Chibucos&amp;author=V%20Phuntumart&amp;author=PF%20Morris&amp;volume=236&amp;publication_year=2012a&amp;pages=1261-1273&amp;pmid=22711282&amp;doi=10.1007/s00425-012-1668-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0030">
<cite>
Mulangi V, Phuntumart V, Aouida M, Ramotar D, Morris P.
2012b. 
Functional analysis of OsPUT1, a rice polyamine uptake transporter. Planta
235, 1–11.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00425-011-1486-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21796369/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Functional%20analysis%20of%20OsPUT1,%20a%20rice%20polyamine%20uptake%20transporter&amp;author=V%20Mulangi&amp;author=V%20Phuntumart&amp;author=M%20Aouida&amp;author=D%20Ramotar&amp;author=P%20Morris&amp;volume=235&amp;publication_year=2012b&amp;pages=1-11&amp;pmid=21796369&amp;doi=10.1007/s00425-011-1486-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0031">
<cite>
Neill S.
2002. 
Hydrogen peroxide signalling. Current Opinion in Plant Biology
5, 388–395.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s1369-5266(02)00282-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12183176/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Plant%20Biology&amp;title=Hydrogen%20peroxide%20signalling&amp;author=S%20Neill&amp;volume=5&amp;publication_year=2002&amp;pages=388-395&amp;pmid=12183176&amp;doi=10.1016/s1369-5266(02)00282-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0032">
<cite>
Niemi K, Häggman H, Sarjala T.
2002. 
Effects of exogenous diamines on the interaction between ectomycorrhizal fungi and adventitious root formation in Scots pine <em>in vitro</em>
. Tree Physiology
22, 373–381.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/treephys/22.6.373" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11960762/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Tree%20Physiology&amp;title=Effects%20of%20exogenous%20diamines%20on%20the%20interaction%20between%20ectomycorrhizal%20fungi%20and%20adventitious%20root%20formation%20in%20Scots%20pine%20in%20vitro&amp;author=K%20Niemi&amp;author=H%20H%C3%A4ggman&amp;author=T%20Sarjala&amp;volume=22&amp;publication_year=2002&amp;pages=373-381&amp;pmid=11960762&amp;doi=10.1093/treephys/22.6.373&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0033">
<cite>
Obata T, Fernie AR.
2012. 
The use of metabolomics to dissect plant responses to abiotic stresses. Cellular and Molecular Life Sciences
69, 3225–3243.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00018-012-1091-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3437017/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22885821/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cellular%20and%20Molecular%20Life%20Sciences&amp;title=The%20use%20of%20metabolomics%20to%20dissect%20plant%20responses%20to%20abiotic%20stresses&amp;author=T%20Obata&amp;author=AR%20Fernie&amp;volume=69&amp;publication_year=2012&amp;pages=3225-3243&amp;pmid=22885821&amp;doi=10.1007/s00018-012-1091-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0034">
<cite>
Perrig D, Boiero ML, Masciarelli OA, Penna C, Ruiz OA, Cassán FD, Luna MV.
2007. 
Plant-growth-promoting compounds produced by two agronomically important strains of <em>Azospirillum brasilense</em>, and implications for inoculant formulation. Applied Microbiology and Biotechnology
75, 1143–1150.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00253-007-0909-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17345081/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Applied%20Microbiology%20and%20Biotechnology&amp;title=Plant-growth-promoting%20compounds%20produced%20by%20two%20agronomically%20important%20strains%20of%20Azospirillum%20brasilense,%20and%20implications%20for%20inoculant%20formulation&amp;author=D%20Perrig&amp;author=ML%20Boiero&amp;author=OA%20Masciarelli&amp;author=C%20Penna&amp;author=OA%20Ruiz&amp;volume=75&amp;publication_year=2007&amp;pages=1143-1150&amp;pmid=17345081&amp;doi=10.1007/s00253-007-0909-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0035">
<cite>
Ramakers C, Ruijter JM, Deprez RHL, Moorman AFM.
2003. 
Assumption-free analysis of quantitative real-time polymerase chain reaction (PCR) data. Neuroscience Letters
339, 62–66.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s0304-3940(02)01423-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12618301/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Neuroscience%20Letters&amp;title=Assumption-free%20analysis%20of%20quantitative%20real-time%20polymerase%20chain%20reaction%20(PCR)%20data&amp;author=C%20Ramakers&amp;author=JM%20Ruijter&amp;author=RHL%20Deprez&amp;author=AFM%20Moorman&amp;volume=339&amp;publication_year=2003&amp;pages=62-66&amp;pmid=12618301&amp;doi=10.1016/s0304-3940(02)01423-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0036">
<cite>
Roth M, Obaidat A, Hagenbuch B.
2012. 
OATPs, OATs and OCTs: the organic anion and cation transporters of the SLCO and SLC22A gene superfamilies. British Journal of Pharmacology
165, 1260–1287.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1476-5381.2011.01724.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3372714/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22013971/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=British%20Journal%20of%20Pharmacology&amp;title=OATPs,%20OATs%20and%20OCTs:%20the%20organic%20anion%20and%20cation%20transporters%20of%20the%20SLCO%20and%20SLC22A%20gene%20superfamilies&amp;author=M%20Roth&amp;author=A%20Obaidat&amp;author=B%20Hagenbuch&amp;volume=165&amp;publication_year=2012&amp;pages=1260-1287&amp;pmid=22013971&amp;doi=10.1111/j.1476-5381.2011.01724.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0037">
<cite>
Schneider CA, Rasband WS, Eliceiri KW.
2012. 
NIH Image to ImageJ: 25 years of image analysis. Nature Methods
9, 671–675.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.2089" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5554542/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22930834/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=NIH%20Image%20to%20ImageJ:%2025%20years%20of%20image%20analysis&amp;author=CA%20Schneider&amp;author=WS%20Rasband&amp;author=KW%20Eliceiri&amp;volume=9&amp;publication_year=2012&amp;pages=671-675&amp;pmid=22930834&amp;doi=10.1038/nmeth.2089&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0038">
<cite>
Serafini-Fracassini D, Del Duca S.
2008. 
Transglutaminases: widespread cross-linking enzymes in plants. Annals of Botany
102, 145–152.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/aob/mcn075" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2712369/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18492735/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Botany&amp;title=Transglutaminases:%20widespread%20cross-linking%20enzymes%20in%20plants&amp;author=D%20Serafini-Fracassini&amp;author=S%20Del%20Duca&amp;volume=102&amp;publication_year=2008&amp;pages=145-152&amp;pmid=18492735&amp;doi=10.1093/aob/mcn075&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0039">
<cite>
Shabala S, Cuin TA, Pottosin I.
2007. 
Polyamines prevent NaCl-induced K<sup>+</sup> efflux from pea mesophyll by blocking non-selective cation channels. FEBS Letters
581, 1993–1999.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.febslet.2007.04.032" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17467698/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Letters&amp;title=Polyamines%20prevent%20NaCl-induced%20K+%20efflux%20from%20pea%20mesophyll%20by%20blocking%20non-selective%20cation%20channels&amp;author=S%20Shabala&amp;author=TA%20Cuin&amp;author=I%20Pottosin&amp;volume=581&amp;publication_year=2007&amp;pages=1993-1999&amp;pmid=17467698&amp;doi=10.1016/j.febslet.2007.04.032&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0040">
<cite>
Shevyakova NI, Rakitin VY, Duong DB, Sadomov NG, Kuznetsov VV.
2001. 
Heat shock-induced cadaverine accumulation and translocation throughout the plant. Plant Science
161, 1125–1133.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Science&amp;title=Heat%20shock-induced%20cadaverine%20accumulation%20and%20translocation%20throughout%20the%20plant&amp;author=NI%20Shevyakova&amp;author=VY%20Rakitin&amp;author=DB%20Duong&amp;author=NG%20Sadomov&amp;author=VV%20Kuznetsov&amp;volume=161&amp;publication_year=2001&amp;pages=1125-1133&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0041">
<cite>
Sziderics AH, Oufir M, Trognitz F, Kopecky D, Matusíková I, Hausman J-F, Wilhelm E.
2010. 
Organ-specific defence strategies of pepper (<em>Capsicum annuum</em> L.) during early phase of water deficit. Plant Cell Reports
29, 295–305.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00299-010-0822-z" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20087595/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Reports&amp;title=Organ-specific%20defence%20strategies%20of%20pepper%20(Capsicum%20annuum%20L.)%20during%20early%20phase%20of%20water%20deficit&amp;author=AH%20Sziderics&amp;author=M%20Oufir&amp;author=F%20Trognitz&amp;author=D%20Kopecky&amp;author=I%20Matus%C3%ADkov%C3%A1&amp;volume=29&amp;publication_year=2010&amp;pages=295-305&amp;pmid=20087595&amp;doi=10.1007/s00299-010-0822-z&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0042">
<cite>
Takahashi Y, Berberich T, Miyazaki A, Seo S, Ohashi Y, Kusano T.
2003. 
Spermine signalling in tobacco: activation of mitogen-activated protein kinases by spermine is mediated through mitochondrial dysfunction. The Plant Journal
36, 820–829.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1365-313x.2003.01923.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14675447/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=Spermine%20signalling%20in%20tobacco:%20activation%20of%20mitogen-activated%20protein%20kinases%20by%20spermine%20is%20mediated%20through%20mitochondrial%20dysfunction&amp;author=Y%20Takahashi&amp;author=T%20Berberich&amp;author=A%20Miyazaki&amp;author=S%20Seo&amp;author=Y%20Ohashi&amp;volume=36&amp;publication_year=2003&amp;pages=820-829&amp;pmid=14675447&amp;doi=10.1046/j.1365-313x.2003.01923.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0043">
<cite>
Tang W, Newton RJ.
2005. 
Polyamines reduce salt-induced oxidative damage by increasing the activities of antioxidant enzymes and decreasing lipid peroxidation in Virginia pine. Plant Growth Regulation
46, 31–43.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Growth%20Regulation&amp;title=Polyamines%20reduce%20salt-induced%20oxidative%20damage%20by%20increasing%20the%20activities%20of%20antioxidant%20enzymes%20and%20decreasing%20lipid%20peroxidation%20in%20Virginia%20pine&amp;author=W%20Tang&amp;author=RJ%20Newton&amp;volume=46&amp;publication_year=2005&amp;pages=31-43&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0044">
<cite>
Tisi A, Federico R, Moreno S, Lucretti S, Moschou PN, Roubelakis-Angelakis KA, Angelini R, Cona A.
2011. 
Perturbation of polyamine catabolism can strongly affect root development and xylem differentiation. Plant Physiology
157, 200–215.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.111.173153" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3165870/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21746808/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Perturbation%20of%20polyamine%20catabolism%20can%20strongly%20affect%20root%20development%20and%20xylem%20differentiation&amp;author=A%20Tisi&amp;author=R%20Federico&amp;author=S%20Moreno&amp;author=S%20Lucretti&amp;author=PN%20Moschou&amp;volume=157&amp;publication_year=2011&amp;pages=200-215&amp;pmid=21746808&amp;doi=10.1104/pp.111.173153&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0045">
<cite>
Vaughn LM, Masson PH.
2011. 
A QTL study for regions contributing to <em>Arabidopsis thaliana</em> root skewing on tilted surfaces. G3
1, 105–115.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1534/g3.111.000331" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3276130/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22384323/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=G3&amp;title=A%20QTL%20study%20for%20regions%20contributing%20to%20Arabidopsis%20thaliana%20root%20skewing%20on%20tilted%20surfaces&amp;author=LM%20Vaughn&amp;author=PH%20Masson&amp;volume=1&amp;publication_year=2011&amp;pages=105-115&amp;pmid=22384323&amp;doi=10.1534/g3.111.000331&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0046">
<cite>
Wang J, Sun P-P, Chen C-L, Wang Y, Fu X-Z, Liu J-H.
2011. 
An arginine decarboxylase gene <em>PtADC</em> from <em>Poncirus trifoliata</em> confers abiotic stress tolerance and promotes primary root growth in Arabidopsis. Journal of Experimental Botany
62, 2899–2914.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/erq463" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21282323/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=An%20arginine%20decarboxylase%20gene%20PtADC%20from%20Poncirus%20trifoliata%20confers%20abiotic%20stress%20tolerance%20and%20promotes%20primary%20root%20growth%20in%20Arabidopsis&amp;author=J%20Wang&amp;author=P-P%20Sun&amp;author=C-L%20Chen&amp;author=Y%20Wang&amp;author=X-Z%20Fu&amp;volume=62&amp;publication_year=2011&amp;pages=2899-2914&amp;pmid=21282323&amp;doi=10.1093/jxb/erq463&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CIT0047"><cite>
Wang S, Basten CJ, Zeng ZB.
2010.  Windows QTL Cartographer 2.5. Department of Statistics, North Carolina State University, Raleigh, NC (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://statgen.ncsu.edu/qtlcart/" rel="noopener noreferrer" target="_blank">http://statgen.ncsu.edu/qtlcart/</a>).</cite></li>
<li id="CIT0048">
<cite>
Xu R, Li QQ.
2008. 
Protocol: streamline cloning of genes into binary vectors in Agrobacterium via the Gateway(R) TOPO vector system. Plant Methods
4, 4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1746-4811-4-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2257932/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18211693/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Methods&amp;title=Protocol:%20streamline%20cloning%20of%20genes%20into%20binary%20vectors%20in%20Agrobacterium%20via%20the%20Gateway(R)%20TOPO%20vector%20system&amp;author=R%20Xu&amp;author=QQ%20Li&amp;volume=4&amp;publication_year=2008&amp;pages=4&amp;pmid=18211693&amp;doi=10.1186/1746-4811-4-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ol></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Supplementary Data</span></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4321547/bin/supp_66_3_853__index.html">supp_66_3_853__index.html</a><sup> (1KB, html) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4321547/bin/supp_eru444_jexbot126433_file001.pdf">supp_eru444_jexbot126433_file001.pdf</a><sup> (294.4KB, pdf) </sup>
</div></div></section></section></section></section></section></article>
</main></body></html>