<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7303416/"/><title>Draft Genome Sequences of Sphingomonas Species Associated with the International Space Station - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Microbiol Resour Announc</button></div>. 2020 Jun 18;9(25):e00578-20. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/MRA.00578-20" rel="noopener noreferrer" target="_blank">10.1128/MRA.00578-20</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Draft Genome Sequences of <em>Sphingomonas</em> Species Associated with the International Space Station</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bijlani%20S%22%5BAuthor%5D"><span class="name western">Swati Bijlani</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Swati Bijlani</span></h3>
<div class="p">
<sup>a</sup>University of Southern California, Los Angeles, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bijlani%20S%22%5BAuthor%5D"><span class="name western">Swati Bijlani</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D"><span class="name western">Nitin K Singh</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Nitin K Singh</span></h3>
<div class="p">
<sup>b</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D"><span class="name western">Nitin K Singh</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D"><span class="name western">Christopher E Mason</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Christopher E Mason</span></h3>
<div class="p">
<sup>c</sup>The WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, New York, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D"><span class="name western">Christopher E Mason</span></a>
</div>
</div>
<sup>c</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CCC%22%5BAuthor%5D"><span class="name western">Clay C C Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Clay C C Wang</span></h3>
<div class="p">
<sup>a</sup>University of Southern California, Los Angeles, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CCC%22%5BAuthor%5D"><span class="name western">Clay C C Wang</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Kasthuri Venkateswaran</span></h3>
<div class="p">
<sup>b</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a>
</div>
</div>
<sup>b,</sup><sup>✉</sup>
</div>
<div class="cg p">Editor: <span class="name western">Vincent Bruno</span><sup>d</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>a</sup>University of Southern California, Los Angeles, California, USA</div>
<div id="aff2">
<sup>b</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div id="aff3">
<sup>c</sup>The WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, New York, USA</div>
<div id="aff4">
<sup>d</sup>University of Maryland School of Medicine</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>✉</sup><p class="display-inline">Address correspondence to Kasthuri Venkateswaran, <span>kjvenkat@jpl.nasa.gov</span>.</p>
</div>
<div class="fn" id="fn1"><p><strong>Citation</strong> Bijlani S, Singh NK, Mason CE, Wang CCC, Venkateswaran K. 2020. Draft genome sequences of <em>Sphingomonas</em> species associated with the International Space Station. Microbiol Resour Announc 9:e00578-20. <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/MRA.00578-20" rel="noopener noreferrer" target="_blank">https://doi.org/10.1128/MRA.00578-20</a>.</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Vincent Bruno</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2020 May 18; Accepted 2020 May 26; Collection date 2020 Jun.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2020 Bijlani et al.</div>
<p>This is an open-access article distributed under the terms of the <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">Creative Commons Attribution 4.0 International license</a>.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC7303416  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32554796/">32554796</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><hr class="headless"/>
<p>The draft genome sequences of three <em>Sphingomonas</em> strains isolated from the International Space Station (ISS) were assembled. These genomic sequences will help in understanding the influence of microgravity conditions on their potential bioactive compound production and other important characteristics compared to their Earth counterparts.</p></section><section class="abstract" id="abstract2"><h2>ABSTRACT</h2>
<p>The draft genome sequences of three <em>Sphingomonas</em> strains isolated from the International Space Station (ISS) were assembled. These genomic sequences will help in understanding the influence of microgravity conditions on their potential bioactive compound production and other important characteristics compared to their Earth counterparts.</p></section><section id="s1"><h2 class="pmc_sec_title">ANNOUNCEMENT</h2>
<p><em>Sphingomonas</em> species have been isolated from a variety of habitats, and some species possess the unique capability to degrade pollutants (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>). Some of the species are known for biofilm formation and eventually corrode the metal surfaces (<a aria-describedby="B2" class="usa-link" href="#B2">2</a>), production of bioactive compounds (<a aria-describedby="B3" class="usa-link" href="#B3">3</a>), and plant-pathogenic characteristics (<a aria-describedby="B4" class="usa-link" href="#B4">4</a>). <em>Sphingomonas paucimobilis</em> has been reported to be associated with infections in immunocompromised patients (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>, <a aria-describedby="B6" class="usa-link" href="#B6">6</a>).</p>
<p>In an ongoing microbial observatory experiment, several microbial strains were isolated from the International Space Station (ISS) (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>). The generation of whole-genome sequences (WGS) to enable the comparative genomic characterization of ISS <em>Sphingomonas</em> species with their Earth counterparts would lead to the identification of the genetic determinants potentially responsible for their important characteristics due to microgravity and elevated radiation conditions.</p>
<p>The WGS belonging to two <em>Sphingomonas sanguinis</em> strains and one <em>S. paucimobilis</em> isolate were assembled into scaffolds. Sample collection, processing, and presumptive identification of these isolates based on 16S rRNA gene sequences were published elsewhere (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>). Briefly, samples collected from the ISS were processed, and 100 μl of each dilution was plated on Reasoner’s 2A (R2A) agar. The plates were incubated at 25°C for 7 days. The single colony obtained was restreaked onto R2A plates and incubated at 25°C for 3 days. A biomass of approximately 1 μg wet weight was collected for each strain and pooled for DNA extraction. Total nucleic acid extraction was carried out using a ZymoBIOMICS 96 MagBead DNA kit (lysis tubes; Zymo Research, USA) after bead beating using a Bertin Precellys instrument. This was followed by library preparation using the Nextera Flex protocol as per Illumina document number 1000000025416 v07. The initial amount of DNA for library preparation was quantified, and depending on the input DNA concentration, 5 to 12 cycles of PCR were carried out to normalize the output. The amplified genomic DNA fragments were indexed and pooled in 384-plex configuration. Whole-genome shotgun sequencing was performed on a NovaSeq 6000 S4 flow cell in paired-end (PE) 2 × 150-bp format. The data were filtered with NGS QC Toolkit v2.3 (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>) for high-quality (HQ) vector- and adaptor-free reads for genome assembly (cutoff read length for HQ, 80%; cutoff quality score, 20). The number of filtered reads obtained (<a class="usa-link" href="#tab1">Table 1</a>) was used for assembly with the SPAdes v3.14.0 (<a aria-describedby="B9" class="usa-link" href="#B9">9</a>) genome assembler (k-mer size, 32 to 72 bases). The genome was annotated using the NCBI Prokaryotic Genome Annotation Pipeline (PGAP) v4.11 (<a aria-describedby="B10" class="usa-link" href="#B10">10</a>, <a aria-describedby="B11" class="usa-link" href="#B11">11</a>). Default parameters were used for all software except where otherwise noted.</p>
<section class="tw xbox font-sm" id="tab1"><h3 class="obj_head">TABLE 1.</h3>
<div class="caption p"><p>Summary of the draft whole-genome sequences of three <em>Sphingomonas</em> strains isolated from the ISS</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
</colgroup>
<thead><tr>
<th colspan="1" rowspan="1">Species and strain</th>
<th colspan="1" rowspan="1">NCBI accession no.</th>
<th colspan="1" rowspan="1">Isolation location</th>
<th colspan="1" rowspan="1">No. of scaffolds</th>
<th colspan="1" rowspan="1">Genome size (bp)</th>
<th colspan="1" rowspan="1">
<em>N</em><sub>50</sub> (bp)</th>
<th colspan="1" rowspan="1">Median coverage (×)</th>
<th colspan="1" rowspan="1">G+C content (%)</th>
<th colspan="1" rowspan="1">No. of raw reads (millions)</th>
<th colspan="1" rowspan="1">No. of filtered reads used for assembly (millions)</th>
<th colspan="1" rowspan="1">No. of coding sequences</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">
<em>Sphingomonas sanguinis</em> IIF7SW-B3A</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JABEOW000000000.1" rel="noopener noreferrer" target="_blank">JABEOW000000000.1</a></td>
<td colspan="1" rowspan="1">Lab 3 overhead</td>
<td colspan="1" rowspan="1">53</td>
<td colspan="1" rowspan="1">3,989,786</td>
<td colspan="1" rowspan="1">149,509</td>
<td colspan="1" rowspan="1">1,102</td>
<td colspan="1" rowspan="1">66.23</td>
<td colspan="1" rowspan="1">30.91</td>
<td colspan="1" rowspan="1">30.67</td>
<td colspan="1" rowspan="1">3,659</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>Sphingomonas sanguinis</em> IIF7SW-B5</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JABEOV000000000.1" rel="noopener noreferrer" target="_blank">JABEOV000000000.1</a></td>
<td colspan="1" rowspan="1">Lab 3 overhead</td>
<td colspan="1" rowspan="1">51</td>
<td colspan="1" rowspan="1">4,398,996</td>
<td colspan="1" rowspan="1">375,188</td>
<td colspan="1" rowspan="1">916</td>
<td colspan="1" rowspan="1">66.08</td>
<td colspan="1" rowspan="1">28.3</td>
<td colspan="1" rowspan="1">28.1</td>
<td colspan="1" rowspan="1">4,047</td>
</tr>
<tr>
<td colspan="1" rowspan="1">
<em>Sphingomonas paucimobilis</em> FKI-L5-BR-P1</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JABEOU000000000.1" rel="noopener noreferrer" target="_blank">JABEOU000000000.1</a></td>
<td colspan="1" rowspan="1">KSC-PHSF<a class="usa-link" href="#ngtab1.1"><sup><em>a</em></sup>
</a> cleanroom floor</td>
<td colspan="1" rowspan="1">73</td>
<td colspan="1" rowspan="1">4,572,738</td>
<td colspan="1" rowspan="1">138,969</td>
<td colspan="1" rowspan="1">448</td>
<td colspan="1" rowspan="1">65.53</td>
<td colspan="1" rowspan="1">14.27</td>
<td colspan="1" rowspan="1">14.18</td>
<td colspan="1" rowspan="1">4,182</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/tab1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="ngtab1.1">
<sup>a</sup><p class="display-inline">KSC-PHSF, Kennedy Space Center Payload Hazardous Servicing Facility.</p>
</div></div></section><p>The details of the final assembly are shown in <a class="usa-link" href="#tab1">Table 1</a>. The phylogenetic affiliations of the strains isolated in this study were confirmed based on the similarity of the 16S rRNA gene sequences extracted from the genomes (<a aria-describedby="B12" class="usa-link" href="#B12">12</a>) and the average nucleotide identity (<a aria-describedby="B13" class="usa-link" href="#B13">13</a>). The average nucleotide identities (ANIs) of the queried genomes were calculated using EzBioCloud (<a aria-describedby="B14" class="usa-link" href="#B14">14</a>) with their corresponding type strains.</p>
<section id="s1.1"><h3 class="pmc_sec_title">Data availability.</h3>
<p>The WGS and raw data are deposited under BioProject accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA629834" rel="noopener noreferrer" target="_blank">PRJNA629834</a>. The WGS accession numbers are listed in <a class="usa-link" href="#tab1">Table 1</a>. The WGS was also deposited at GeneLab (GeneLab data set GLDS-298, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-298/" rel="noopener noreferrer" target="_blank">https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-298/</a>). The version described in this paper is the first version.</p></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>Part of the research described in this publication was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with NASA. We thank astronauts Terry Virts and Jeffrey Williams for collecting samples aboard the ISS, the Implementation Team at NASA Ames Research Center (Fathi Karouia) for coordinating this effort, and Aleksandra Chechinska-Sielaff for the isolation of the strains. We thank Ryan Kemp (Zymo Corporation) for extracting the DNA and Dan Butler (Cornell Medicine) for the shotgun sequencing. We acknowledge the JPL supercomputing facility staff, notably Narendra J. Patel (Jimmy) and Edward Villanueva, for their continuous support in providing the best possible infrastructure for BIG-DATA analysis.</p>
<p>We acknowledge government sponsorship. This research was funded by 2012 Space Biology NNH12ZTT001N grant number 19-12829-26 under task order NNN13D111T awarded to K.V., which was also subcontracted to C.C.C.W. and funded a postdoctoral fellowship for S.B.</p></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>Leys NMEJ, Ryngaert A, Bastiaens L, Verstraete W, Top EM, Springael D. 
2004. 
Occurrence and phylogenetic diversity of <em>Sphingomonas</em> strains in soils contaminated with polycyclic aromatic hydrocarbons. Appl Environ Microbiol
70:1944–1955. doi: 10.1128/aem.70.4.1944-1955.2004.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/aem.70.4.1944-1955.2004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC383131/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15066784/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Occurrence%20and%20phylogenetic%20diversity%20of%20Sphingomonas%20strains%20in%20soils%20contaminated%20with%20polycyclic%20aromatic%20hydrocarbons&amp;author=NMEJ%20Leys&amp;author=A%20Ryngaert&amp;author=L%20Bastiaens&amp;author=W%20Verstraete&amp;author=EM%20Top&amp;volume=70&amp;publication_year=2004&amp;pages=1944-1955&amp;pmid=15066784&amp;doi=10.1128/aem.70.4.1944-1955.2004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>Pavissich JP, Vargas IT, Gonzalez B, Pasten PA, Pizarro GE. 
2010. 
Culture dependent and independent analyses of bacterial communities involved in copper plumbing corrosion. J Appl Microbiol
109:771–782. doi: 10.1111/j.1365-2672.2010.04704.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-2672.2010.04704.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20337760/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Microbiol&amp;title=Culture%20dependent%20and%20independent%20analyses%20of%20bacterial%20communities%20involved%20in%20copper%20plumbing%20corrosion&amp;author=JP%20Pavissich&amp;author=IT%20Vargas&amp;author=B%20Gonzalez&amp;author=PA%20Pasten&amp;author=GE%20Pizarro&amp;volume=109&amp;publication_year=2010&amp;pages=771-782&amp;pmid=20337760&amp;doi=10.1111/j.1365-2672.2010.04704.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>Asaf S, Khan AL, Khan MA, Al-Harrasi A, Lee I-J. 
2018. 
Complete genome sequencing and analysis of endophytic <em>Sphingomonas</em> sp. LK11 and its potential in plant growth. 3 Biotech
8:389. doi: 10.1007/s13205-018-1403-z.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s13205-018-1403-z" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6111035/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30175026/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=3%20Biotech&amp;title=Complete%20genome%20sequencing%20and%20analysis%20of%20endophytic%20Sphingomonas%20sp.%20LK11%20and%20its%20potential%20in%20plant%20growth&amp;author=S%20Asaf&amp;author=AL%20Khan&amp;author=MA%20Khan&amp;author=A%20Al-Harrasi&amp;author=I-J%20Lee&amp;volume=8&amp;publication_year=2018&amp;pages=389&amp;pmid=30175026&amp;doi=10.1007/s13205-018-1403-z&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>Liu F, Zhan R-L, He Z-Q. 
2018. 
First report of bacterial dry rot of mango caused by <em>Sphingomonas sanguinis</em> in China. Plant Dis
102:2632. doi: 10.1094/PDIS-04-18-0589-PDN.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1094/PDIS-04-18-0589-PDN" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Dis&amp;title=First%20report%20of%20bacterial%20dry%20rot%20of%20mango%20caused%20by%20Sphingomonas%20sanguinis%20in%20China&amp;author=F%20Liu&amp;author=R-L%20Zhan&amp;author=Z-Q%20He&amp;volume=102&amp;publication_year=2018&amp;pages=2632&amp;doi=10.1094/PDIS-04-18-0589-PDN&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>Göker T, Aşık RZ, Yılmaz MB, Çelik İ, Tekiner A. 
2017. 
<em>Sphingomonas paucimobilis</em>: a rare infectious agent found in cerebrospinal fluid. J Korean Neurosurg Soc
60:481–483. doi: 10.3340/jkns.2014.0102.004.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3340/jkns.2014.0102.004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5544368/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28689399/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Korean%20Neurosurg%20Soc&amp;title=Sphingomonas%20paucimobilis:%20a%20rare%20infectious%20agent%20found%20in%20cerebrospinal%20fluid&amp;author=T%20G%C3%B6ker&amp;author=RZ%20A%C5%9F%C4%B1k&amp;author=MB%20Y%C4%B1lmaz&amp;author=%C4%B0%20%C3%87elik&amp;author=A%20Tekiner&amp;volume=60&amp;publication_year=2017&amp;pages=481-483&amp;pmid=28689399&amp;doi=10.3340/jkns.2014.0102.004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>Hardjo Lugito NP, Cucunawangsih, Kurniawan A. 
2016. 
A lethal case of <em>Sphingomonas paucimobilis</em> bacteremia in an immunocompromised patient. Case Rep Infect Dis
2016:3294639. doi: 10.1155/2016/3294639.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1155/2016/3294639" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4818805/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27088020/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Case%20Rep%20Infect%20Dis&amp;title=A%20lethal%20case%20of%20Sphingomonas%20paucimobilis%20bacteremia%20in%20an%20immunocompromised%20patient&amp;author=NP%20Hardjo%20Lugito&amp;author=%20Cucunawangsih&amp;author=A%20Kurniawan&amp;volume=2016&amp;publication_year=2016&amp;pages=3294639&amp;pmid=27088020&amp;doi=10.1155/2016/3294639&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, Minich J, McDonald D, Mayer T, Knight R, Karouia F, Fox GE, Venkateswaran K. 
2019. 
Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome
7:50. doi: 10.1186/s40168-019-0666-x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40168-019-0666-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6452512/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30955503/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&amp;author=A%20Checinska%20Sielaff&amp;author=C%20Urbaniak&amp;author=GBM%20Mohan&amp;author=VG%20Stepanov&amp;author=Q%20Tran&amp;volume=7&amp;publication_year=2019&amp;pages=50&amp;pmid=30955503&amp;doi=10.1186/s40168-019-0666-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>Patel RK, Jain M. 
2012. 
NGS QC Toolkit: a toolkit for quality control of next generation sequencing data. PLoS One
7:e30619. doi: 10.1371/journal.pone.0030619.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0030619" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3270013/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22312429/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=NGS%20QC%20Toolkit:%20a%20toolkit%20for%20quality%20control%20of%20next%20generation%20sequencing%20data&amp;author=RK%20Patel&amp;author=M%20Jain&amp;volume=7&amp;publication_year=2012&amp;pages=e30619&amp;pmid=22312429&amp;doi=10.1371/journal.pone.0030619&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 
2012. 
SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol
19:455–477. doi: 10.1089/cmb.2012.0021.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/cmb.2012.0021" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3342519/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22506599/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&amp;title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&amp;author=A%20Bankevich&amp;author=S%20Nurk&amp;author=D%20Antipov&amp;author=AA%20Gurevich&amp;author=M%20Dvorkin&amp;volume=19&amp;publication_year=2012&amp;pages=455-477&amp;pmid=22506599&amp;doi=10.1089/cmb.2012.0021&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>Haft DH, DiCuccio M, Badretdin A, Brover V, Chetvernin V, O'Neill K, Li W, Chitsaz F, Derbyshire MK, Gonzales NR, Gwadz M, Lu F, Marchler GH, Song JS, Thanki N, Yamashita RA, Zheng C, Thibaud-Nissen F, Geer LY, Marchler-Bauer A, Pruitt KD. 
2018. 
RefSeq: an update on prokaryotic genome annotation and curation. Nucleic Acids Res
46:D851–D860. doi: 10.1093/nar/gkx1068.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkx1068" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5753331/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29112715/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=RefSeq:%20an%20update%20on%20prokaryotic%20genome%20annotation%20and%20curation&amp;author=DH%20Haft&amp;author=M%20DiCuccio&amp;author=A%20Badretdin&amp;author=V%20Brover&amp;author=V%20Chetvernin&amp;volume=46&amp;publication_year=2018&amp;pages=D851-D860&amp;pmid=29112715&amp;doi=10.1093/nar/gkx1068&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>Tatusova T, DiCuccio M, Badretdin A, Chetvernin V, Nawrocki EP, Zaslavsky L, Lomsadze A, Pruitt KD, Borodovsky M, Ostell J. 
2016. 
NCBI Prokaryotic Genome Annotation Pipeline. Nucleic Acids Res
44:6614–6624. doi: 10.1093/nar/gkw569.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkw569" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5001611/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27342282/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=NCBI%20Prokaryotic%20Genome%20Annotation%20Pipeline&amp;author=T%20Tatusova&amp;author=M%20DiCuccio&amp;author=A%20Badretdin&amp;author=V%20Chetvernin&amp;author=EP%20Nawrocki&amp;volume=44&amp;publication_year=2016&amp;pages=6614-6624&amp;pmid=27342282&amp;doi=10.1093/nar/gkw569&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>Kim O-S, Cho Y-J, Lee K, Yoon S-H, Kim M, Na H, Park S-C, Jeon YS, Lee J-H, Yi H, Won S, Chun J. 
2012. 
Introducing EzTaxon-e: a prokaryotic 16S rRNA gene sequence database with phylotypes that represent uncultured species. Int J Syst Evol Microbiol
62:716–721. doi: 10.1099/ijs.0.038075-0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1099/ijs.0.038075-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22140171/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&amp;title=Introducing%20EzTaxon-e:%20a%20prokaryotic%2016S%20rRNA%20gene%20sequence%20database%20with%20phylotypes%20that%20represent%20uncultured%20species&amp;author=O-S%20Kim&amp;author=Y-J%20Cho&amp;author=K%20Lee&amp;author=S-H%20Yoon&amp;author=M%20Kim&amp;volume=62&amp;publication_year=2012&amp;pages=716-721&amp;pmid=22140171&amp;doi=10.1099/ijs.0.038075-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>Goris J, Konstantinidis KT, Klappenbach JA, Coenye T, Vandamme P, Tiedje JM. 
2007. 
DNA–DNA hybridization values and their relationship to whole-genome sequence similarities. Int J Syst Evol Microbiol
57:81–91. doi: 10.1099/ijs.0.64483-0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1099/ijs.0.64483-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17220447/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&amp;title=DNA%E2%80%93DNA%20hybridization%20values%20and%20their%20relationship%20to%20whole-genome%20sequence%20similarities&amp;author=J%20Goris&amp;author=KT%20Konstantinidis&amp;author=JA%20Klappenbach&amp;author=T%20Coenye&amp;author=P%20Vandamme&amp;volume=57&amp;publication_year=2007&amp;pages=81-91&amp;pmid=17220447&amp;doi=10.1099/ijs.0.64483-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14">
<span class="label">14.</span><cite>Yoon S-H, Ha S, Lim J, Kwon S, Chun J. 
2017. 
A large-scale evaluation of algorithms to calculate average nucleotide identity. Antonie Van Leeuwenhoek
110:1281–1286. doi: 10.1007/s10482-017-0844-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s10482-017-0844-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28204908/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Antonie%20Van%20Leeuwenhoek&amp;title=A%20large-scale%20evaluation%20of%20algorithms%20to%20calculate%20average%20nucleotide%20identity&amp;author=S-H%20Yoon&amp;author=S%20Ha&amp;author=J%20Lim&amp;author=S%20Kwon&amp;author=J%20Chun&amp;volume=110&amp;publication_year=2017&amp;pages=1281-1286&amp;pmid=28204908&amp;doi=10.1007/s10482-017-0844-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The WGS and raw data are deposited under BioProject accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA629834" rel="noopener noreferrer" target="_blank">PRJNA629834</a>. The WGS accession numbers are listed in <a class="usa-link" href="#tab1">Table 1</a>. The WGS was also deposited at GeneLab (GeneLab data set GLDS-298, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-298/" rel="noopener noreferrer" target="_blank">https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-298/</a>). The version described in this paper is the first version.</p></section></section></section></section></article>
</main></body></html>