<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9023564/"/><title>Emergence of novel cephalopod gene regulation and expression through large-scale genome reorganization - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Nat Commun</button></div>. 2022 Apr 21;13:2172. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41467-022-29694-7" rel="noopener noreferrer" target="_blank">10.1038/s41467-022-29694-7</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Emergence of novel cephalopod gene regulation and expression through large-scale genome reorganization</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schmidbaur%20H%22%5BAuthor%5D"><span class="name western">Hannah Schmidbaur</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Hannah Schmidbaur</span></h3>
<div class="p">
<sup>1</sup>Department of Neurosciences and Developmental Biology, University of Vienna, Vienna, Austria </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schmidbaur%20H%22%5BAuthor%5D"><span class="name western">Hannah Schmidbaur</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kawaguchi%20A%22%5BAuthor%5D"><span class="name western">Akane Kawaguchi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Akane Kawaguchi</span></h3>
<div class="p">
<sup>2</sup>Institute for Molecular Pathology, Vienna, Austria </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kawaguchi%20A%22%5BAuthor%5D"><span class="name western">Akane Kawaguchi</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Clarence%20T%22%5BAuthor%5D"><span class="name western">Tereza Clarence</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Tereza Clarence</span></h3>
<div class="p">
<sup>3</sup>Biomolecular Modelling Laboratory, The Francis Crick Institute, London, UK </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Clarence%20T%22%5BAuthor%5D"><span class="name western">Tereza Clarence</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20X%22%5BAuthor%5D"><span class="name western">Xiao Fu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Xiao Fu</span></h3>
<div class="p">
<sup>3</sup>Biomolecular Modelling Laboratory, The Francis Crick Institute, London, UK </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20X%22%5BAuthor%5D"><span class="name western">Xiao Fu</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoang%20OP%22%5BAuthor%5D"><span class="name western">Oi Pui Hoang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Oi Pui Hoang</span></h3>
<div class="p">
<sup>1</sup>Department of Neurosciences and Developmental Biology, University of Vienna, Vienna, Austria </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoang%20OP%22%5BAuthor%5D"><span class="name western">Oi Pui Hoang</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zimmermann%20B%22%5BAuthor%5D"><span class="name western">Bob Zimmermann</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Bob Zimmermann</span></h3>
<div class="p">
<sup>1</sup>Department of Neurosciences and Developmental Biology, University of Vienna, Vienna, Austria </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zimmermann%20B%22%5BAuthor%5D"><span class="name western">Bob Zimmermann</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ritschard%20EA%22%5BAuthor%5D"><span class="name western">Elena A Ritschard</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Elena A Ritschard</span></h3>
<div class="p">
<sup>1</sup>Department of Neurosciences and Developmental Biology, University of Vienna, Vienna, Austria </div>
<div class="p">
<sup>4</sup>Department of Biology and Evolution of Marine Organisms, Stazione Zoologica Anton Dohrn, Naples, Italy </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ritschard%20EA%22%5BAuthor%5D"><span class="name western">Elena A Ritschard</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weissenbacher%20A%22%5BAuthor%5D"><span class="name western">Anton Weissenbacher</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Anton Weissenbacher</span></h3>
<div class="p">
<sup>5</sup>Vienna Zoo, Maxingstraße 13b, 1130 Vienna, Austria </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weissenbacher%20A%22%5BAuthor%5D"><span class="name western">Anton Weissenbacher</span></a>
</div>
</div>
<sup>5</sup>, <a aria-describedby="id9" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Foster%20JS%22%5BAuthor%5D"><span class="name western">Jamie S Foster</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Jamie S Foster</span></h3>
<div class="p">
<sup>6</sup>Department of Microbiology and Cell Science, University of Florida, Space Life Science Lab, Merritt Island, FL USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Foster%20JS%22%5BAuthor%5D"><span class="name western">Jamie S Foster</span></a>
</div>
</div>
<sup>6</sup>, <a aria-describedby="id10" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nyholm%20SV%22%5BAuthor%5D"><span class="name western">Spencer V Nyholm</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Spencer V Nyholm</span></h3>
<div class="p">
<sup>7</sup>Department of Molecular and Cell Biology, University of Connecticut, Storrs, CT USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nyholm%20SV%22%5BAuthor%5D"><span class="name western">Spencer V Nyholm</span></a>
</div>
</div>
<sup>7</sup>, <a aria-describedby="id11" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bates%20PA%22%5BAuthor%5D"><span class="name western">Paul A Bates</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Paul A Bates</span></h3>
<div class="p">
<sup>3</sup>Biomolecular Modelling Laboratory, The Francis Crick Institute, London, UK </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bates%20PA%22%5BAuthor%5D"><span class="name western">Paul A Bates</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id12" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Albertin%20CB%22%5BAuthor%5D"><span class="name western">Caroline B Albertin</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Caroline B Albertin</span></h3>
<div class="p">
<sup>8</sup>Bell Center for Regenerative Biology and Tissue Engineering, Marine Biological Laboratory, Woods Hole, MA USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Albertin%20CB%22%5BAuthor%5D"><span class="name western">Caroline B Albertin</span></a>
</div>
</div>
<sup>8,</sup><sup>✉</sup>, <a aria-describedby="id13" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tanaka%20E%22%5BAuthor%5D"><span class="name western">Elly Tanaka</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Elly Tanaka</span></h3>
<div class="p">
<sup>2</sup>Institute for Molecular Pathology, Vienna, Austria </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tanaka%20E%22%5BAuthor%5D"><span class="name western">Elly Tanaka</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a aria-describedby="id14" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Simakov%20O%22%5BAuthor%5D"><span class="name western">Oleg Simakov</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Oleg Simakov</span></h3>
<div class="p">
<sup>1</sup>Department of Neurosciences and Developmental Biology, University of Vienna, Vienna, Austria </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Simakov%20O%22%5BAuthor%5D"><span class="name western">Oleg Simakov</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Neurosciences and Developmental Biology, University of Vienna, Vienna, Austria </div>
<div id="Aff2">
<sup>2</sup>Institute for Molecular Pathology, Vienna, Austria </div>
<div id="Aff3">
<sup>3</sup>Biomolecular Modelling Laboratory, The Francis Crick Institute, London, UK </div>
<div id="Aff4">
<sup>4</sup>Department of Biology and Evolution of Marine Organisms, Stazione Zoologica Anton Dohrn, Naples, Italy </div>
<div id="Aff5">
<sup>5</sup>Vienna Zoo, Maxingstraße 13b, 1130 Vienna, Austria </div>
<div id="Aff6">
<sup>6</sup>Department of Microbiology and Cell Science, University of Florida, Space Life Science Lab, Merritt Island, FL USA </div>
<div id="Aff7">
<sup>7</sup>Department of Molecular and Cell Biology, University of Connecticut, Storrs, CT USA </div>
<div id="Aff8">
<sup>8</sup>Bell Center for Regenerative Biology and Tissue Engineering, Marine Biological Laboratory, Woods Hole, MA USA </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2020 Aug 19; Accepted 2022 Mar 28; Collection date 2022.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© The Author(s) 2022</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC9023564  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35449136/">35449136</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Coleoid cephalopods (squid, cuttlefish, octopus) have the largest nervous system among invertebrates that together with many lineage-specific morphological traits enables complex behaviors. The genomic basis underlying these innovations remains unknown. Using comparative and functional genomics in the model squid <em>Euprymna scolopes</em>, we reveal the unique genomic, topological, and regulatory organization of cephalopod genomes. We show that coleoid cephalopod genomes have been extensively restructured compared to other animals, leading to the emergence of hundreds of tightly linked and evolutionary unique gene clusters (microsyntenies). Such novel microsyntenies correspond to topological compartments with a distinct regulatory structure and contribute to complex expression patterns. In particular, we identify a set of microsyntenies associated with cephalopod innovations (MACIs) broadly enriched in cephalopod nervous system expression. We posit that the emergence of MACIs was instrumental to cephalopod nervous system evolution and propose that microsyntenic profiling will be central to understanding cephalopod innovations.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Subject terms:</strong> Evolutionary developmental biology, Evolutionary genetics, Molecular evolution</p></section></section><section class="abstract" id="Abs2"><hr class="headless"/>
<p id="Par2">Cephalopods are an enigmatic animal group with complex and adaptive behaviors such as camouflage; however the genetic basis for these traits is not well understood. Here the authors reveal a set of cephalopod-restricted rearranged genomic loci, involving known neuronal regulators but also unexpected gene families, that confer topological organization and gene regulation.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Cephalopods have the largest invertebrate nervous systems and possess many lineage-specific adaptations such as rapid adaptive camouflage, arms with suckers and camera-type eyes. Many cephalopod characteristics evolved convergently to those of vertebrates, which makes them an attractive system to study the genetic basis of wide-scale organismal innovations and the pathways behind their evolution.</p>
<p id="Par4">On a genomic level, the emergence of novel genes, extensive gene duplications, and wide-ranging RNA editing have been described in cephalopod genomes<sup><a aria-describedby="CR1" class="usa-link" href="#CR1">1</a></sup>. Expansions of gene families such as C2H2s, Protocadherins, and GPCRs, and extensive RNA editing allowed the diversification of protein-coding transcripts in the nervous system and is proposed to have played an important role in its evolution. While similar innovations are known from vertebrate genomes, the mechanisms driving the evolution of these features are different: vertebrates went through several rounds of whole-genome duplications that produced large sets of multi-copy genes and the diversification of their functions, there is no indication for similar events in cephalopods<sup><a aria-describedby="CR1" class="usa-link" href="#CR1">1</a>–<a aria-describedby="CR3" class="usa-link" href="#CR3">3</a></sup>. In contrast, it has been suggested that the coleoid cephalopod (squids, cuttlefish, octopus) lineage went through large-scale genome reorganization<sup><a aria-describedby="CR2" class="usa-link" href="#CR2">2</a>,<a aria-describedby="CR3" class="usa-link" href="#CR3">3</a></sup>.</p>
<p id="Par5">A property of metazoan genomes is that local gene order or microsynteny is conserved between even distantly related species<sup><a aria-describedby="CR4" class="usa-link" href="#CR4">4</a>–<a aria-describedby="CR6" class="usa-link" href="#CR6">6</a></sup>. This conservation is supported by functional studies of regulatory constraints, shown in genomic regulatory blocks (GRBs)<sup><a aria-describedby="CR4" class="usa-link" href="#CR4">4</a>,<a aria-describedby="CR5" class="usa-link" href="#CR5">5</a>,<a aria-describedby="CR7" class="usa-link" href="#CR7">7</a></sup>, as well as co-expression of neighboring genes in tissues or cell types<sup><a aria-describedby="CR8" class="usa-link" href="#CR8">8</a></sup>. Early genome assemblies in several coleoids indicate that local gene order has been greatly disrupted, breaking ancient microsyntenies and bringing previously unlinked genes together<sup><a aria-describedby="CR2" class="usa-link" href="#CR2">2</a>,<a aria-describedby="CR3" class="usa-link" href="#CR3">3</a></sup>. This event, potentially at a whole-genome scale, could have affected hundreds of gene families, disrupting the order of genes in comparison to the last common ancestor of coleoid cephalopods and other molluscs. The extent of this event is difficult to estimate due to the lack of chromosomal-scale assemblies in cephalopods. To begin to understand the extent of the genome reorganization and its impact on cephalopod genome biology and evolution, we study the emerging model species <em>Euprymna scolopes</em> (Hawaiian bobtail squid). This species has been at the center of symbiosis research for over 30 years<sup><a aria-describedby="CR9" class="usa-link" href="#CR9">9</a>,<a aria-describedby="CR10" class="usa-link" href="#CR10">10</a></sup>, but is also an attractive model system for evolution and development research due to its small adult size, large egg clutches, and relative ease of culture.</p>
<p id="Par6">To reconstruct the regulatory landscape in the <em>E. scolopes</em> genome, we applied chromosomal conformation capture (Hi-C) and open chromatin profiling techniques (ATAC-seq) as well as collected additional expression data. Hi-C allowed us to both improve the previously published <em>E. scolopes</em> genome assembly as well as to capture the three-dimensional organization of the genome. Using comparative genomic approaches, we describe the global nature of the genome reshuffling in coleoid cephalopods and demonstrate the emergence of many microsyntenic regions that were previously unlinked in other species. Our data also reveals interactions between distant genomic loci (the topological organization of the genome) shedding light on the three-dimensional organization of the <em>E. scolopes</em> genome, as well as identifying genes located in regulatory loops and topologically associating domains (TADs). Our open chromatin data reveals regions accessible to transcription factors and thus potentially constituting regulatory elements. Together, these data allow us to gain insights into the impact of evolutionary changes in gene linkages and the emergence of novel gene regulation. This study provides the basis for the understanding of the evolution of cephalopod genomes and possible implications on morphological novelties in this clade.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="Sec3"><h3 class="pmc_sec_title">The impact of a large-scale genome reorganization on the coleoid cephalopod genome</h3>
<p id="Par7">Linkage information from chromosome conformation capture allowed us to reconstruct 46 chromosomal scaffolds in <em>E. scolopes</em> (“Methods”, <a class="usa-link" href="#MOESM1">Supplementary Notes</a> <a class="usa-link" href="#MOESM1">1</a> and <a class="usa-link" href="#MOESM1">2</a>, Supplementary Fig. <a class="usa-link" href="#MOESM1">1a, b</a>) based on the published assembly<sup><a aria-describedby="CR3" class="usa-link" href="#CR3">3</a></sup>. We then compared the order of genes with orthologs found in another 24 animal species ranging from sponges to vertebrates, which allowed us to reconstruct microsyntenic blocks shared between different clades (“Methods”, Supplementary Note <a class="usa-link" href="#MOESM1">3</a>, Supplementary Fig. <a class="usa-link" href="#MOESM1">2a</a>, Supplementary Data <a class="usa-link" href="#MOESM1">1</a>). Briefly, we define microsyntenic blocks as at least three or more co-occurring orthologous genes with up to five intervening genes with no constraints on their collinearity. This definition of microsynteny yields the fewest false-positive blocks (compared to just pairs of genes) while providing enough flexibility to detect syntenic regions that underwent local rearrangement and expansion. We recover 505 microsyntenies unique to cephalopods, representing blocks of genes only found in close proximity to each other in <em>E. scolopes</em> and at least one octopus species. For the same species sampling and same microsynteny detection parameters only 2 blocks would have been expected by chance (median from 3 rounds of randomization, as described in<sup><a aria-describedby="CR6" class="usa-link" href="#CR6">6</a></sup>). Five out of these 505 blocks were paralogous. In total, only 48 out of 2290 genes in these 505 blocks were identified as orphan genes with no homology outside of cephalopods, while all others have orthologs in other animals, suggesting that the origin of microsynteny was due to changes in gene locations rather than novel gene emergence. These microsyntenies have been conserved in coleoid cephalopods despite their long divergence time (Fig. <a class="usa-link" href="#Fig1">1b, c</a>), suggesting an evolutionary constraint that kept those blocks of genes together. Similarly, a comparison in other molluscs, such as the scallop <em>Mizuhopecten yessoensis</em><sup><a aria-describedby="CR11" class="usa-link" href="#CR11">11</a></sup> and the bivalve <em>Crassostrea gigas</em> showed that a much smaller number of bivalve specific microsyntenies (152) is shared between these species. To infer the set of highly conserved microsyntenic blocks, we reconstructed microsyntenies shared between <em>E. scolopes</em> and at least six more distantly related species out of a set of 23 species (Supplementary Fig. <a class="usa-link" href="#MOESM1">2</a>). We recovered 275 such metazoan microsyntenies, which are retained in the <em>E. scolopes</em> lineage and are inferred to date back to at least the last common bilaterian ancestor (Fig. <a class="usa-link" href="#Fig1">1c</a>, Supplementary Fig. <a class="usa-link" href="#MOESM1">2a</a>, “Methods”). In comparison, the bivalve <em>M. yessoensis</em> retains a similar number of metazoan microsyntenies (216). These results provide evidence for a large-scale microsyntenic gain in coleoid cephalopods.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Large-scale syntenic reorganization of cephalopod genomes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9023564_41467_2022_29694_Fig1_HTML.jpg" target="_blank"><img alt="Fig. 1" class="graphic zoom-in" height="312" id="d32e652" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c43/9023564/b07026eb6fbd/41467_2022_29694_Fig1_HTML.jpg" width="799"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Photograph of an <em>Euprymna scolopes</em> hatchling. <strong>b</strong> Schematic tree with divergence times of major cephalopod lineages from deuterostomes and other protostomes<sup><a aria-describedby="CR53" class="usa-link" href="#CR53">53</a>,<a aria-describedby="CR54" class="usa-link" href="#CR54">54</a></sup>. <strong>c</strong> Circos plot showing loss of syntenies conserved in other metazoans and the emergence of a large number of novel, cephalopod-specific microsyntenies within cephalopods. Each line represents a syntenic cluster shared between different species. Orange lines indicate syntenic clusters shared between at least seven out of these 24 species (ancestral, metazoan clusters); green lines represent novel molluscan syntenies, shared between five or more molluscs but not present in any non-molluscan species. Blue lines represent cephalopod-specific syntenies shared between either all three cephalopod species (dark blue) or two of the three cephalopod species (light blue) but not present in any non-cephalopod species; gray lines represent other syntenies that do not fall in either of the previous categories. Abbreviations: Aca - <em>Acanthaster planci</em>, Aqu – <em>Amphimedon queenslandica</em>, Bfl - <em>Branchiostoma floridae</em>,Cel - <em>Caenorhabditis elegans</em>, Cgi -<em> Crassostrea gigas</em>, Cmi - <em>Callistoctopus minor</em>, Cte - <em>Capitella teleta</em>, Dme - <em>Drosophila melanogaster</em>, Esc – <em>Euprymna scolopes</em>, Lgi - <em>Lottia gigantea</em>, Mle - <em>Mnemiopsis leidyi</em>, Mye - <em>Mizuhopecten yessoensis</em>, Nve - <em>Nematostella vectensis</em>, Obi – <em>Octopus bimaculoides</em>, Sko - <em>Saccoglossus kowalevskii</em>
<strong>d</strong> Example of one whole chromosomal-scale scaffold (right) of <em>E. scolopes</em> showing the distribution of gene density, cephalopod-specific (blue) and conserved, metazoan (orange) syntenies. Inset (left) with locations of genes within two specific syntenic blocks.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Chromosomal distribution and properties of metazoan and novel cephalopod microsyntenies</h3>
<p id="Par8">Both cephalopod-specific and conserved metazoan microsyntenic blocks are present on 44 out of 46 chromosomal scaffolds (with two chromosomes being too small to contain any microsyntenies). While some chromosomes have higher proportion of novel cephalopod microsyntenies (Supplementary Fig. <a class="usa-link" href="#MOESM1">1b, c</a>), both microsynteny types are intermixed in the genome (Fig. <a class="usa-link" href="#Fig1">1d</a>). This result suggests a genome-wide mechanism for the emergence of novel microsyntenies. The vast majority of single-copy genes (71%) that comprise 232 novel cephalopod microsyntenies are located on different chromosomes in the scallop <em>M. yessoensis</em>. As the organization of the recently published Nautilus genome<sup><a aria-describedby="CR12" class="usa-link" href="#CR12">12</a></sup> is similar to other molluscs, these results suggest that either many translocations or chromosomal-level fusions occurred in the coleoid ancestor.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Genomic properties of novel and conserved microsyntenies</h3>
<p id="Par9">Novel cephalopod and conserved metazoan microsynteny show different genomic properties. Novel cephalopod microsyntenies are on average smaller in size than the metazoan microsyntenies still present in cephalopod genomes (Supplementary Fig. <a class="usa-link" href="#MOESM1">2b</a>), despite having similar numbers of genes (Supplementary Fig. <a class="usa-link" href="#MOESM1">2c</a>). While introns of genes in cephalopod-specific microsyntenies are smaller than those of metazoan microsyntenies, the majority of size differences stem from intergenic regions (~0.2 kb compared to ~7 kb difference, respectively).</p>
<p id="Par10">We also find evidence for differential enrichment of functional categories between the two microsynteny types. Metazoan microsyntenies<sup><a aria-describedby="CR6" class="usa-link" href="#CR6">6</a></sup> are enriched in signaling pathway components of the Wnt-signaling pathway, neurotransmitter transport and synaptic vesicle exocytosis, G-protein coupled receptor signaling, negative regulation of transcription, and BMP signaling pathways, among others. Genes in novel, cephalopod-specific microsynteny, on the other hand, play a role in translation, redox processes, regulation of store-operated calcium entry, mRNA cleavage, transport, and chromatin organization (<em>p</em>-values &lt;0.05) (Supplementary Fig. <a class="usa-link" href="#MOESM1">3</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Spatial organization of the <em>E. scolopes</em> genome</h3>
<p id="Par11">Three-dimensional chromatin structure including topologically associated domains (TADs) facilitates distant regulatory interactions involved in gene regulation<sup><a aria-describedby="CR13" class="usa-link" href="#CR13">13</a>,<a aria-describedby="CR14" class="usa-link" href="#CR14">14</a></sup>. While very little data exists to-date on invertebrate genome topological organization, we found that in comparison to the data known for vertebrate TAD sizes the interaction distances were generally larger in the squid (Fig. <a class="usa-link" href="#Fig2">2a, b</a>, Supplementary Note <a class="usa-link" href="#MOESM1">4</a>). TAD prediction tools (see “Methods”) reveal a median <em>E. scolopes</em> TAD size of 2.5 Mb, in comparison to an average of 1.2 Mb in human<sup><a aria-describedby="CR15" class="usa-link" href="#CR15">15</a></sup>. In addition, the distribution of TAD sizes in <em>E. scolopes</em> was considerably wider than in human, suggesting a higher variability.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Topological and spatial organization of the cephalopod genome.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9023564_41467_2022_29694_Fig2_HTML.jpg" target="_blank"><img alt="Fig. 2" class="graphic zoom-in" height="736" id="d32e753" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c43/9023564/d7d2827a87cc/41467_2022_29694_Fig2_HTML.jpg" width="782"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Top: Hi-C normalized interaction matrix at 100 kb resolution for chromosomal scaffold 10. Bottom: Relative Hi-C count and TAD boundary scores as predicted by tadbit (1 = lowest, 10 = highest). <strong>b</strong> Violin plot of TAD size distributions for human and <em>Euprymna scolopes</em> computed at 100 kb resolution, plotted in 100 kb bins. <strong>c</strong> CTCF binding motif as identified by homer motif search in 100 kbp of predicted TAD boundaries. <strong>d</strong> Solvent area surface exposure (SASA) per bp for individual chromosomal scaffolds (observed and random). Conserved metazoan synteny on x-axis, cephalopod-specific microsynteny on y-axis. <strong>e</strong> Three-dimensional model of chromosomal scaffold 10 with novel (blue) and conserved (yellow) microsynteny locations labeled. Left and right models are the same, shifted by 90°.</p></figcaption></figure><p id="Par12">In vertebrates, TAD formation is mediated by proteins, such as CTCF and cohesin<sup><a aria-describedby="CR16" class="usa-link" href="#CR16">16</a>,<a aria-describedby="CR17" class="usa-link" href="#CR17">17</a></sup>. We infer that the same mechanisms, including CTCF and the proteins Smc1 and Smc3 of the cohesin complex, are present in the <em>E. scolopes</em> genome and conserved with other animals, suggesting that similar mechanisms may be deployed in cephalopods (Supplementary Fig. <a class="usa-link" href="#MOESM1">4</a>). We also find TAD boundaries to be enriched for a CCCTC-like motif<sup><a aria-describedby="CR18" class="usa-link" href="#CR18">18</a>–<a aria-describedby="CR20" class="usa-link" href="#CR20">20</a></sup> reminiscent of a CTCF binding site (Fig. <a class="usa-link" href="#Fig2">2c</a>, <em>p</em> = 1e−12, “Methods”, Supplementary Note <a class="usa-link" href="#MOESM1">4</a>).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Topological organization around microsyntenies</h3>
<p id="Par13">Several studies suggested the possibility of correspondence between microsynteny and regulatory domains in metazoan genomes<sup><a aria-describedby="CR4" class="usa-link" href="#CR4">4</a>,<a aria-describedby="CR5" class="usa-link" href="#CR5">5</a>,<a aria-describedby="CR7" class="usa-link" href="#CR7">7</a>,<a aria-describedby="CR8" class="usa-link" href="#CR8">8</a>,<a aria-describedby="CR21" class="usa-link" href="#CR21">21</a>–<a aria-describedby="CR23" class="usa-link" href="#CR23">23</a></sup>. To understand the relationship of microsyntenies and TADs we compared the localization of randomly computed microsyntenic blocks, that follow the same properties as our observed blocks but are randomly distributed throughout the genome to the observed microsyntenies (“Methods”). We find a tendency of conserved metazoan microsyntenies to be localized towards the center of the predicted TADs, whereas new cephalopod microsyntenies appear to be more evenly distributed (Fig. <a class="usa-link" href="#Fig3">3a</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Novel cephalopod microsyntenies and their regulatory properties in <em>Euprymna scolopes</em>.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9023564_41467_2022_29694_Fig3_HTML.jpg" target="_blank"><img alt="Fig. 3" class="graphic zoom-in" height="824" id="d32e910" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c43/9023564/cecc5cf89b42/41467_2022_29694_Fig3_HTML.jpg" width="800"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Ratio between densities of the center of observed and randomized microsynteny locations within normalized TADs. Ratio increases towards the center of TADs for metazoan syntenies and decreases for cephalopod synteny. <strong>b</strong> Compactness of novel and metazoan microsynteny, compared to random simulations. Microsyntenic clusters must contain at least 3 genes, if fewer genes were annotated to the tree, these clusters were excluded. Plotted are the ratios between the number of bins (at 40 kb Hi-C resolution, cephalopod micrysynteny bins <em>n</em> = 143,969, metazoan microsynteny bins <em>n</em> = 82,417, random cephalopod microsynteny bins <em>n</em> = 3,499,849, random metazoan microsynteny bins <em>n</em> = 1,889,687) in microsyntenic clusters (within 7 and 25 bins, valid microsyntenies: cephalopod <em>n</em> = 265, metazoan <em>n</em> = 125, random cephalopod <em>n</em> = 4265, random metazoan <em>n</em> = 2180) and the number of “descendant” bins from the last common ancestor of those microsyntenic bins (“Methods”, cephalopod <em>n</em> = 143,969, metazoan <em>n</em> = 82,417, random cephalopod <em>n</em> = 3,499,849, random metazoan <em>n</em> = 1,889,687). The ratio (<em>n</em> = 6835) of the number of bins in a cluster and the number of bins in the sub-tree were compared by two sided Wilcoxon rank-sum test comparing the linked groups (****<em>p</em> &lt; 0.0001, * &lt; 0.05, ns: not significant). The closer the ratio to 1, the lower is the difference between the size of the syntenic block and the number of bins in an extracted tree. Violin plots—distribution, boxes—interquartile range (cephalopod = lower 0.06, upper 0.56, metazoan = lower 0.01, upper 0.58, random cephalopod = lower 0.01, upper 0.49, random metazoan = lower 0.01, upper 0.46), bars—median (cephalopod = 0.31, metazoan = 0.28, random cephalopod = 0.18, random metazoan = 0.17), whiskers—furthest sample within 1.5x interquartile range (cephalopod = min 0.002, max 0.94, metazoan = min 0.002, max 0.95, random cephalopod = min 0.002, max 0.96, random metazoan = min 0.002, max 0.96), maximum and minimum ratios: cephalopod = min 0.00189, max = 0.941, metazoan = min 0.00217, max = 0.952, random cephalopod = min 0.00151, max 0.962, random metazoan = min 0.00150, max 0.96. <strong>c</strong> Co-expression correlation of genes in microsyntenic clusters (cephalopod <em>n</em> = 476, metazoan <em>n</em> = 236, random cephalopod <em>n</em> = 6925, random metazoan <em>n</em> = 4038). The co-expression correlation of metazoan syntenies is higher than that of cephalopod-specific syntenies or random clusters (****<em>p</em> &lt; 0.0001, ***<em>p</em> &lt; 0.001, * &lt; 0.05). Violin plots—distribution, boxes—furthest sample within 1.5x interquartile range (cephalopod = min −0.69, max 0.87, metazoan = min −0.63, max 1.0, random cephalopod = min −0.72, max 0.95, random metazoan min −0.66, max 0.9), bars—median (cephalopod = 0.05, metazoan = 0.15, random cephalopod = 0.07670835, random metazoan = 0.07) outliers were excluded from these numbers. Maximum and minimum values: −1 and 1 in all cases. <strong>d</strong> Clustering of mean expression per synteny cluster, color-coded by synteny type, expression among <em>Euprymna scolopes</em> adult tissues. Syntenic clusters form eight expression modules with specific expression patterns. Expression matrix is z-score normalized. Light organ—<em>E. scolopes</em>-specific organ harboring symbionts, accessory nidamental gland—female-specific reproductive organ of some squid species. <strong>e</strong> Annotation of ATAC peak location in late organogenesis. Peaks annotated as associated with cephalopod-specific microsyntenies are more often found in intronic regions. Promotor defined as +10 kb and −10 kb predicted transcription start site.</p></figcaption></figure><p id="Par14">To further study the relationships of genomic regions and their interactions independent of TAD predictions, we computed a tree structure reflecting the organization of each chromosomal scaffold (“Methods”, Supplementary Note <a class="usa-link" href="#MOESM1">4</a>). Each bifurcating branch reflects the relationships of genomic regions in Hi-C signal strength, allowing us to track interaction intensities in microsyntenies (Fig. <a class="usa-link" href="#Fig3">3b</a>). Surprisingly, we found that novel microsyntenies are more likely to form tight interaction regions, reflected by subtrees with few branches, when compared to randomly sampled syntenies (Fig. <a class="usa-link" href="#Fig3">3b</a>). This result indicates significantly higher levels of compartmentalization in both cephalopod and ancestral microsyntenies.</p>
<p id="Par15">Motivated by the importance of three-dimensional genome architecture and microsyntenic co-localization in <em>Euprymna scolopes</em>, three-dimensional modeling based on Hi-C interaction matrices was performed (“Methods”, Supplementary Fig. <a class="usa-link" href="#MOESM1">5</a>, Supplementary Note <a class="usa-link" href="#MOESM1">5</a>) to provide a deeper understanding of spatial properties and co-localization of both novel and ancient microsyntenic regions within modeled chromosomal scaffolds. Three-dimensional models revealed that novel cephalopod microsyntenies have distinct spatial properties from ancient microsyntenies. In particular, both synteny types showed differential solvent accessibility on some chromosomes when compared to random distributions (Fig. <a class="usa-link" href="#Fig2">2d</a>). Moreover, novel cephalopod microsyntenies were on average less buried, thus covering a larger proportion of chromosomal surface (Supplementary Fig. <a class="usa-link" href="#MOESM1">6</a>). This result was in contrast to the conserved metazoan microsyntenies, which tend to participate in the formation of the chromosomal core (Fig. <a class="usa-link" href="#Fig2">2e</a>, Supplementary Fig. <a class="usa-link" href="#MOESM1">6</a>). Since the novel microsyntenies are transcriptionally active (Fig. <a class="usa-link" href="#Fig3">3</a>, see below), their location on the chromosomal surface may be reflective of highly dynamic inter-chromosomal regulation, as well as being more accessible to transcription factors.</p>
<p id="Par16">The GC content of metazoan and cephalopod-specific microsynteny was evaluated along with predictions of A/B compartments based on the Hi-C interaction matrix ("Methods"). The analysis did not provide sufficient evidence for one of the mycrosyntenic types being more prevalent in either of the compartments. Until further experimental data are available (such as methylation and acetylation profiling) for <em>Euprymna scolopes</em>, we cannot accurately infer the distribution of cephalopod-specific and metazoan microsynteny within A/B compartments.</p>
<p id="Par17">Taken together, the strong genomic conservation among sequenced cephalopods, the comparably tight packaging (short inter-gene distances) of microsyntenic clusters and their prevalent association with defined subcompartments within detected TADs suggests strong selective pressure to maintain regulatory properties of novel microsyntenic units in the cephalopod genomes.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Co-expression properties of microsyntenies</h3>
<p id="Par18">Co-expression of syntenic genes is an important property that can reflect their regulation. Genes in cephalopod-specific microsyntenies do not tend to be co-expressed, despite their tight co-localization (“Methods”, Supplementary Note <a class="usa-link" href="#MOESM1">7</a>). When compared to randomly sampled groups of genes that follow a similar distribution to the observed, novel microsyntenies, the mean co-expression coefficient is even slightly lower in the observed data (Wilcoxon test, <em>p</em> ≤ 0.05, Fig. <a class="usa-link" href="#Fig3">3c</a>). In contrast, conserved metazoan microsyntenies show significant (Wilcoxon test, <em>p</em> ≤ 0.001) co-expression when compared to simulated microsyntenies (Fig. <a class="usa-link" href="#Fig3">3c</a>). This result indicates that genes in metazoan microsynteny tend to co-express in a defined set of tissues, similar to previous findings for ancient metazoan microsynteny<sup><a aria-describedby="CR8" class="usa-link" href="#CR8">8</a></sup>. A similar pattern was also observed for co-expression of novel and conserved metazoan microsyntenies in <em>O. bimaculoides</em> (Supplementary Fig. <a class="usa-link" href="#MOESM1">7a</a>). No type of microsynteny showed enrichment of expression-specificity in a particular tissue<sup><a aria-describedby="CR24" class="usa-link" href="#CR24">24</a></sup>.</p>
<p id="Par19">To further categorize expression profiles, we investigated averaged expression of syntenic regions. This analysis showed complex patterns that fell into eight distinct expression modules across adult <em>E. scolopes</em> tissues (Fig. <a class="usa-link" href="#Fig3">3d</a>, “Methods”, Supplementary Note <a class="usa-link" href="#MOESM1">7</a>). Although most modules had similar proportions of both microsynteny types, some clusters formed outliers. For example, module 8 showed eye-specific expression and was mostly dominated by metazoan microsynteny. Interestingly, clusters encompassing multiple nervous tissues, in particular modules 2 and 4, were enriched in novel cephalopod microsyntenies (Fisher’s exact test <em>p</em>-values ≤0.02 and ≤1e−07, respectively). Their orthologs were similarly expressed in <em>O. bimaculoides</em> nervous tissues, with novel microsyntenies dominating in modules associated with the strongest brain expression (Supplementary Fig. <a class="usa-link" href="#MOESM1">7b</a>). However, the overall module correspondence was impacted by the difference in tissue sampling (Supplementary Fig. <a class="usa-link" href="#MOESM1">7c</a>).</p>
<p id="Par20">We next wanted to investigate whether these expression patterns are biased due to a single highly expressed gene per syntenic block. The vast majority of microsyntenic blocks (76%, Supplementary Fig. <a class="usa-link" href="#MOESM1">8b</a>) have one gene that contributes to more than 50% of the cumulative expression. We then calculated relative expression levels across tissues per gene and averaged it for each block, showing that overall expression module identities are retained (Supplementary Fig. <a class="usa-link" href="#MOESM1">8a</a>). In general, expression variance correlates with absolute expression of genes (Supplementary Fig. <a class="usa-link" href="#MOESM1">8c</a>). In some metazoan syntenies, however, especially in the tissues defining a module, the variance was low, indicating higher co-expression constraints (Supplementary Fig. <a class="usa-link" href="#MOESM1">8c</a>).</p>
<p id="Par21">Together, these results highlight the complex expression domain contribution of microsyntenic regions and identifies a discrepancy in the co-expression dynamics between novel cephalopod and metazoan microsyntenies. This paucity of co-expression in cephalopod microsyntenies indicates a potentially different mode of their gene regulation.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Regulatory signatures of microsyntenies</h3>
<p id="Par22">Expression modules showed specific signatures of regulatory motifs associated with them. We predicted regulatory regions using assay for transposase-accessible chromatin (ATAC-seq<sup><a aria-describedby="CR25" class="usa-link" href="#CR25">25</a></sup>) data from a developmental time course (“Methods”, Supplementary Note <a class="usa-link" href="#MOESM1">9</a>). Predicted peaks associated with each expression cluster were then further analyzed for known transcription factor motif enrichment, separately for cephalopod and ancient microsynteny (“Methods”, Supplementary Data <a class="usa-link" href="#MOESM1">2</a>). We find that the cephalopod microsynteny module 2, which is associated with multiple nervous tissues, was enriched for the transcription factors binding motifs Chop<sup><a aria-describedby="CR26" class="usa-link" href="#CR26">26</a></sup>, E2F1<sup><a aria-describedby="CR27" class="usa-link" href="#CR27">27</a></sup>, NeuroG2<sup><a aria-describedby="CR28" class="usa-link" href="#CR28">28</a></sup>, COUP-TFII<sup><a aria-describedby="CR29" class="usa-link" href="#CR29">29</a></sup>, Atf4<sup><a aria-describedby="CR30" class="usa-link" href="#CR30">30</a></sup> involved in nervous system differentiation and developmental transcription factors ZBTB18, Esrrb, Tcf21, Pitx1, GATA in all three developmental stages (<em>p</em> &lt; 1e−3). Module 4 was similarly enriched in transcription factor binding motifs involved in nervous system and general development such as Tcf3, TCFL2, GATA, Tcf21, and Pitx1 in all three developmental stages (<em>p</em> &lt; 1e−3). This suggests a common regulatory scheme responsible for gene expression in each of the expression modules and an association of those motifs with novel cephalopod microsynteny.</p>
<p id="Par23">To complement our ATAC-seq data, we conducted whole-genome alignments between available cephalopod genomes using different alignment similarity and length thresholds (“Methods”, Supplementary Note <a class="usa-link" href="#MOESM1">8</a>, Supplementary Table <a class="usa-link" href="#MOESM1">5</a>, and Supplementary Fig. <a class="usa-link" href="#MOESM1">9</a>). This helped identify potential conserved non-coding elements (CNEs) and their association with gene bodies and genome topology (“Methods”, Supplementary Fig. <a class="usa-link" href="#MOESM1">10</a>, Supplementary Note <a class="usa-link" href="#MOESM1">8</a>). Due to evolutionary distance between squid and octopus lineages, our approach yielded only 1187 coleoid cephalopod CNE candidates with a similarity threshold of 0,95 and a minimum size of 100 bp (Supplementary Table <a class="usa-link" href="#MOESM1">5</a>), of which 613 could be localized to gene features (Supplementary Fig. <a class="usa-link" href="#MOESM1">10a–c</a>). 139 were associated with novel cephalopod microsyntenies (inside or within 1 kb of microsynteny), 73 with metazoan syntenies inside or within 1 kb of microsynteny), and 401 were located outside any synteny (Supplementary Fig. <a class="usa-link" href="#MOESM1">10b</a>). Only 12 of the 1187 candidates had overlap with ATAC-seq peaks (Supplementary Table <a class="usa-link" href="#MOESM1">5</a>). For CNEs shared among squid genomes we found 42920 putative CNEs with a similarity threshold of 0.95 and a minimum size of 100 bp (Supplementary Table <a class="usa-link" href="#MOESM1">5</a>), of which 13889 could be localized to gene features (Supplementary Fig. <a class="usa-link" href="#MOESM1">10a–c</a>). 2444 were associated with novel cephalopod microsynteny (inside or within 1 kb of microsynteny), 3255 with metazoan synteny, and 8190 were located around genes outside any synteny (Supplementary Fig. 10c). Similarly, very few overlapped with ATAC peaks (14). Therefore, the regulatory role of these regions remains unclear.</p>
<p id="Par24">A potential contribution to this observation could be the high evolutionary turnover of regulatory regions within the squid lineage, diminishing the insight gained from genome alignment/conservation-based inferences. Cephalopod genomes are large with over 50% of genome length attributed to repetitive elements<sup><a aria-describedby="CR2" class="usa-link" href="#CR2">2</a>,<a aria-describedby="CR3" class="usa-link" href="#CR3">3</a></sup>. We thus assessed the transposable element composition of ATAC-seq peaks associated with microsyntenic clusters. Both microsynteny types showed an average repeat content of 40%. ATAC-seq peak sequences in cephalopod microsyntenies in <em>E. scolopes</em> showed an elevated repetitive element content of 82% and were most frequently associated with LINE/CR1-Zenon elements (44%, compared to 35% in metazoan microsyntenic peaks). The LINE/CR1 expansion was identified as the most common and specifically expanded repeat element class in the squid (<em>E. scolopes</em>) lineage<sup><a aria-describedby="CR3" class="usa-link" href="#CR3">3</a></sup>.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Evolutionary scenarios for functional microsynteny emergence in cephalopods</h3>
<p id="Par25">Conserved gene co-localization may be explained by regulatory regions within a neighboring gene, even though the function of the gene may be unrelated (the bystander scenario), together forming a genomic regulatory block (GRB)<sup><a aria-describedby="CR4" class="usa-link" href="#CR4">4</a>,<a aria-describedby="CR5" class="usa-link" href="#CR5">5</a></sup>, or via shared regulatory elements controlling the expression of all syntenic genes, resulting in higher co-expression<sup><a aria-describedby="CR8" class="usa-link" href="#CR8">8</a></sup>. Our microsynteny approach is agnostic to this distinction, focusing on any conserved co-localization of three or more genes. We thus may use our data to independently profile the propensity of metazoan or cephalopod microsyntenies to correspond to known functional microsyntenic models. The location of ATAC-seq peaks within the microsyntenic clusters reveals that open chromatin regions associated with cephalopod-specific microsynteny were more often found in introns than peaks found in conserved, metazoan microsynteny or other non-syntenic genes (Fig. <a class="usa-link" href="#Fig3">3e</a>, Fisher’s exact test <em>p</em> &lt; 1e−5). Interestingly, while sharing little overlap, CNE distribution, in particular of CNEs shared among squids, showed significant localization towards distal regions in metazoan syntenies (Fisher’s exact test <em>p</em> &lt; 2.2e−16) (Supplementary Fig. <a class="usa-link" href="#MOESM1">10</a>). This result indicates that, unlike more conserved microsyntenies, novel cephalopod microsyntenies are more similar to the GRB scenario, in particular the bystander model, in which putative enhancers are found within closely located (bystander) genes inside the same microsyntenic cluster and may potentially lack distal regulatory domains. This observation may also complement the finding of weaker co-expression of microsyntenic genes in cepholopod-specific microsynteny, compared to more ancient, conserved microsyntenies.</p>
<p id="Par26">Given this insight, we further sought to investigate the potential function of novel cephalopod microsyntenies in expression modules 2 and 4 that showed the highest contribution to neuronal tissue expression domains. Such microsyntenies can be considered a useful test set of functional microsyntenies that were involved in the evolution of the cephalopod nervous system. We thus examined the genomic rearrangement, the regulatory landscape, and expression of genes within one of the representative cephalopod-specific microsyntenies from expression module 2. It was one of the clusters with the highest number of genes, encoding for ceramide-1 phosphate transfer protein, phenylalanine-tRNA ligase, splicing factor 3B subunit, integrator complex subunit, and amyloid protein-binding protein. Orthologs of this microsynteny were widely spread across two chromosomes in scallop (Fig. <a class="usa-link" href="#Fig4">4a</a>), yet were densely packed in the <em>E. scolopes</em> genome with almost no intergenic space and a few dominant ATAC peaks towards one end of the cluster in the intron of phenylalanine-tRNA ligase (Fig. <a class="usa-link" href="#Fig4">4b</a>). Similar to the general trend, this clustering in cephalopods implies either several local translocations or a large-scale chromosomal fusion, followed by rearrangements. The cluster was localized towards the center of the predicted TAD (Fig. <a class="usa-link" href="#Fig4">4b</a>), close to another novel microsyntenic unit (from the same expression module). Together, these two units form a compartment of high Hi-C interaction density, separate from the closest metazoan microsyntenic compartment and its associated ATAC-peaks (Fig. <a class="usa-link" href="#Fig4">4b</a>). In our three-dimensional chromosomal model, this microsynteny was also found on the surface of chromosomal scaffold 2 (Fig. <a class="usa-link" href="#Fig4">4c</a>). Interestingly, the genes in this cluster show nervous system expression both in scallop and <em>E. scolopes</em> (Fig. <a class="usa-link" href="#Fig5">5a</a>, Supplementary Note <a class="usa-link" href="#MOESM1">10</a>). Despite some of these genes being considered as purely metabolic or housekeeping genes, in vertebrates they are known to play an important role in nervous system development and activity<sup><a aria-describedby="CR31" class="usa-link" href="#CR31">31</a>–<a aria-describedby="CR35" class="usa-link" href="#CR35">35</a></sup>. We conducted in situ hybridization to visualize gene expression during development in <em>E. scolopes</em> embryos and confirmed the expression in all major central brain regions (Fig. <a class="usa-link" href="#Fig5">5b, c</a>, Supplementary Note <a class="usa-link" href="#MOESM1">11</a>). However, we also revealed expression in other novel cephalopod tissues and organs, such as the axial nerve cords as well as heart and gills (Fig. <a class="usa-link" href="#Fig5">5b, c</a>, Supplementary Fig. <a class="usa-link" href="#MOESM1">11</a>). This result provides evidence that this microsyntenic cluster and its siblings from expression module 2 could comprise functional microsynteny of the bystander model type that were crucial contributors to the emergence of novel cephalopod expression domains. More generally, the observation of microsyntenies associated with cephalopod innovations (MACIs) and their further investigation could help dissect the evolution of complex cephalopod tissue expression patterns.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Emergence of compact clusters and their unique expression patterns in nervous tissues.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9023564_41467_2022_29694_Fig4_HTML.jpg" target="_blank"><img alt="Fig. 4" class="graphic zoom-in" height="1247" id="d32e1236" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c43/9023564/b9feb81a8aa3/41467_2022_29694_Fig4_HTML.jpg" width="673"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Location of orthologous genes of one of the MACIs in <em>Mizuhopecten yessoensis</em><em>.</em> The genes are located on two separate chromosomes (top—whole chromosome, bottom—zoom in, black—gene density, blue—location of orthologous genes) with many intervening genes in-between. Scallop orthologs of genes in the microsyntenic cluster are plotted at the bottom highlighted in blue. <strong>b</strong> Genes of the same cluster in <em>Euprymna scolopes</em>. The genes (highlighted in blue) are tightly packed on one chromosomal scaffold with only one intervening gene (top—whole chromosomal scaffold, bottom—zoom in, orange—conserved, metazoan microsyntenic clusters, blue—cephalopod-specific microsyntenic clusters). The cluster is located within a TAD. Two major ATAC-seq peaks are present in intronic regions of the last gene of the cluster in three developmental stages (y-axis—signal value). RNA-seq read count of three developmental stages shows several small peaks in the region of the cluster (y-axis—read count). Early-stage—early organogenesis, stage 20, middle stage—late organogenesis, stage 24/25, late stage—close to hatching, stage 28/29. <strong>c</strong> Three-dimensional reconstruction of chromosomal scaffold 2. The cephalopod-specific syntenic cluster is located on the surface. Left and right views are shifted by 90°.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Expression of genes in a representative MACI in late developmental stages of <em>E. scolopes</em>.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=9023564_41467_2022_29694_Fig5_HTML.jpg" target="_blank"><img alt="Fig. 5" class="graphic zoom-in" height="633" id="d32e1267" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c43/9023564/bc1064b9562b/41467_2022_29694_Fig5_HTML.jpg" width="799"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Heatmap showing expression of orthologous genes of the cephalopod-specific cluster in adult scallop and <em>E. scolopes</em> tissues showing neuronal expression in both species. Scale bar shows z-score normalized expression levels per row. <strong>b</strong> Scheme of late stage (Stage 27–29) <em>E. scolopes</em> anatomy and section of hatchling. <strong>c</strong> Expression of MACI genes in nervous tissues and inner organs of late developmental stages of <em>E. scolopes</em>. All genes show expression in different lobes of the brain, most dominantly the ASM and PSM. Light expression patterns are present in the optic lobes, the digestive gland and intestines in most genes. Expression patterns differ to Beta-tubulin, which was chosen as a control for its pan-neuronal expression domain, in its intensity and distribution. Scale bars = 100 µm. ASM anterior subesophageal mass, DG digestive gland, ES esophagus, FT funnel organ, GI gills, INT intestine, IS ink sack, IYO internal yolk, OL optic lobe, PSM posterior subesophageal mass, ST statocyst. Dotted circles - color trapping.</p></figcaption></figure><p id="Par27">In summary, we present a comprehensive study of topological and regulatory genome organization in a coleoid cephalopod. Characterized by megabase-range interactions, cephalopod genomes have been impacted by a genome-wide syntenic reorganization, with an extent that is rare among animal genomes. This reorganization led to the gain of hundreds of cephalopod-specific microsyntenies that are associated with compact topological regions and a distinct mode of gene regulation. Their putative regulatory sequences were often located within the introns of genes within the same microsyntenic cluster, as has been proposed for functional gene linkage in the bystander model. Our analysis of the microsyntenic expression data revealed complex expression patterns of novel microsyntenies associated with a specific set of cephalopod neural tissues and other novel organs. We identify two such expression modules most prominently affected by the emergence of novel cephalopod microsyntenies, each associated with a specific regulatory signature. We propose that this syntenic ‘locking-in’, i.e., high compactness and regulatory streamlining, was responsible for the emergence and extension of ancestral molluscan neural tissue expression domains. As much of cephalopod molecular biology remains elusive, our study proposes the use of these microsyntenies associated with cephalopod innovations (MACIs) to begin to unravel molecular changes associated with cephalopod developmental and organismal innovations. This study sets the stage for further investigation of MACIs and their roles in the emergence of novel expression domains and organismal innovations in cephalopods.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Collection of animals</h3>
<p id="Par28"><em>E. scolopes</em> eggs were obtained from cultures and maintained at the Vienna Zoo or at the Marine Biological Laboratory. All work was performed in compliance with the EU Directive 2010/63/EU on cephalopod use and AAALAC guidelines on the care and welfare of cephalopods<sup><a aria-describedby="CR36" class="usa-link" href="#CR36">36</a></sup>. Adult <em>E. scolopes</em> spawned naturally in their tanks, and embryos were collected shortly after spawning and maintained in a closed aquarium system filled with artificial seawater. Embryos developed to the appropriate stage and were anesthetized with 2% Ethanol before use<sup><a aria-describedby="CR37" class="usa-link" href="#CR37">37</a>–<a aria-describedby="CR39" class="usa-link" href="#CR39">39</a></sup>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Hi-C, genome scaffolding, and 3D analysis</h3>
<p id="Par29">Hi-C sample preparation was performed as described in Supplementary Note <a class="usa-link" href="#MOESM1">2</a>. Briefly, Hi-C samples were generated on developmental stage 27<sup><a aria-describedby="CR40" class="usa-link" href="#CR40">40</a></sup> with 30 pooled embryos using the six base restriction enzyme Hind3. Paired-end sequencing of 50 bp was done on an Illumina HiSeq2500. Hi-C reads were aligned to the reference genome (excluding scaffolds &lt;50k) resulting in over 106 million valid interaction pairs (alignment rate ~71%). Aligned reads were used to scaffold the genome to chromosomal scaffolds. Assembly statistics are summarized in Supplementary Note <a class="usa-link" href="#MOESM1">2</a>. Raw Hi-C reads were then again mapped to the new chromosomal scaffolds, recovering over 106 million valid interaction pairs (alignment rate ~71%). Three-dimensional modeling of chromosomal scaffolds is described in Supplementary Note <a class="usa-link" href="#MOESM1">3</a>. For human samples, Hi-C samples of B-lymphoblastoids were downloaded from NCBI (ref. <sup><a aria-describedby="CR41" class="usa-link" href="#CR41">41</a></sup>, SRR1658570, HIC001) and aligned to the human reference genome (GRCh38.p12) obtaining over 144 million valid interaction pairs (~73% alignment rate).</p></section><section id="Sec14"><h3 class="pmc_sec_title">Synteny analysis</h3>
<p id="Par30">Gene orthology was reconstructed using 27 species spanning all major metazoan clades. Microsynteny was computed using in-house tools as described in detail in Supplementary Note <a class="usa-link" href="#MOESM1">3</a>. Metazoan synteny was defined as all syntenic blocks shared between at least seven other species. Novel, cephalopod-specific synteny was defined as synteny shared between <em>E. scolopes</em> and at least one octopus species. Random microsyntenies were modeled after the distribution of observed syntenies as described in ref. <sup><a aria-describedby="CR8" class="usa-link" href="#CR8">8</a></sup> for 20 iterations. Additional details of the scripts and steps are found in Supplementary Note <a class="usa-link" href="#MOESM1">3</a>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Chromatin conformation analysis</h3>
<p id="Par31">TADs for <em>E. scolopes</em> and Human were called with Tadbit<sup><a aria-describedby="CR42" class="usa-link" href="#CR42">42</a></sup> with the Tadbit algorithm and with HiCExplorer<sup><a aria-describedby="CR43" class="usa-link" href="#CR43">43</a></sup>. <em>E. scolopes</em> TADs were averaged and the location of the middle of each syntenic cluster was mapped to analyse the distribution of syntenies within TADs. If syntenies spanned several TADs, only the TAD mapping the middle of that synteny was considered. To analyse the topology of microsyntenic clusters further, the normalized Hi-C interaction matrix was used to cluster each bin to its closest neighbor by the bin interaction strength. An interaction cladogram for each chromosome was reconstructed that way. To understand how well a syntenic region is defined by its interactions, we extracted the last common ancestor of that region (i.e., the bins in that region) from the whole tree making up the chromosome. Then the ratio between those sub-trees and the number of bins in a syntenic cluster was calculated and differences between groups were tested using the Wilcoxon test.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Chromatin conformation analysis of synteny</h3>
<p id="Par32">The center of each syntenic cluster was localized within predicted TAD boundaries. The locations were then normalized and plotted for observed and random microsyntenies. To visualize the differences between observed and random, the ratio between densities of each were calculated and plotted in normalized TAD locations (Fig. <a class="usa-link" href="#Fig3">3a</a>). A density over one signifies an enrichment of observed synteny compared to random clusters. A density lower than one signifies a depleted signal of observed synteny compared to random clusters. Random synteny represents clusters sampled from the distribution of observed blocks from random locations in the genome.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Co-expression analysis</h3>
<p id="Par33">Co-expression coefficients for adult tissues of <em>E. scolopes</em> and <em>O. bimaculoides</em> were computed as described in ref. <sup><a aria-describedby="CR8" class="usa-link" href="#CR8">8</a></sup>. A detailed description of the steps is provided in Supplementary Note <a class="usa-link" href="#MOESM1">7</a>.</p></section><section id="Sec18"><h3 class="pmc_sec_title">CNE identification</h3>
<p id="Par34">The genomes of <em>E. scolopes</em> and <em>O. bimaculoides</em> and of <em>E. scolope</em>s and <em>A. dux</em> were aligned with megablast, using <em>E. scolopes</em> as the query sequence. Five different settings for BLAST<sup><a aria-describedby="CR44" class="usa-link" href="#CR44">44</a></sup> similarity scores (-perc_identity) were used: 0%, 70%, 80%, 95%, and 98% (Supplementary Figs. <a class="usa-link" href="#MOESM1">9</a>, <a class="usa-link" href="#MOESM1">10</a>, Supplementary Table <a class="usa-link" href="#MOESM1">5</a>, see refs. <sup><a aria-describedby="CR4" class="usa-link" href="#CR4">4</a>,<a aria-describedby="CR7" class="usa-link" href="#CR7">7</a>,<a aria-describedby="CR45" class="usa-link" href="#CR45">45</a>,<a aria-describedby="CR46" class="usa-link" href="#CR46">46</a></sup>). For further settings see Supplementary Note <a class="usa-link" href="#MOESM1">8</a>. Multimapping regions were excluded if they overlapped by more than 50% and occurred more than 3 times using BEDOPS<sup><a aria-describedby="CR47" class="usa-link" href="#CR47">47</a></sup> bedmap. bedmap --count --echo --fraction-both 0.5 --delim '\t' prefiltered_megablast.bed | awk '$1&lt;4' | cut -f2- |sort-bed - | uniq. Remaining overlapping regions were merged with bedops –merge. Any region overlapping with an exon by 1 bp or more was excluded using bedtools<sup><a aria-describedby="CR48" class="usa-link" href="#CR48">48</a></sup> subtract. To exclude repetitive regions, fasta sequences were extracted from the filtered putative CNE locations and meme’s<sup><a aria-describedby="CR49" class="usa-link" href="#CR49">49</a></sup> dust (cut-off 10) function was used to mask repeats. In addition, two datasets were created for each similarity score with a minimum size of 100 bp or 50 bp respectively. For similarity scores of 0%, only 100 bp regions were kept. Any region with more than 25% Ns was excluded. To remove any remaining coding sequences, the remaining putative CNE sequences were blasted against the NCBI<sup><a aria-describedby="CR50" class="usa-link" href="#CR50">50</a></sup> NR database and any regions overlapping with a BLAST match were removed.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Chromatin accessibility assay with ATAC-seq</h3>
<p id="Par35">ATAC-seq samples preparation and analysis are described in Supplementary Note <a class="usa-link" href="#MOESM1">9</a>. ATAC-seq was generated for stages 20, 25, and 28/29<sup><a aria-describedby="CR40" class="usa-link" href="#CR40">40</a></sup> with two biological replicates each as described in refs. <sup><a aria-describedby="CR25" class="usa-link" href="#CR25">25</a>,<a aria-describedby="CR51" class="usa-link" href="#CR51">51</a>,<a aria-describedby="CR52" class="usa-link" href="#CR52">52</a></sup> with slight modifications. Each ATAC-seq library was generated with two biological replicate samples. Samples were sequenced on Illumina HiSeqV4 using 125 bp paired-end reads. The reads were trimmed with BBDuk (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbduk-guide/" rel="noopener noreferrer" target="_blank">https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbduk-guide/</a>) and mapped to the chromosomal scaffolds. Peaks were called with Genrich (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://github.com/jsh58/Genrich" rel="noopener noreferrer" target="_blank">https://github.com/jsh58/Genrich</a>). After trimming between 72 and 143 million reads remained which were mapped at between 79 and 83% and between 22,443 and 36,933 peaks were called for each sample.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Fluorescence in situ hybridization (FISH) and in situ hybridization (ISH)</h3>
<p id="Par36"><em>E. scolopes</em> embryos removed from eggs and jelly layers and hatchlings were anesthetized in 4% EtOH in seawater or 4% EtOH and MgCl<sub>2</sub> (2 M solution added slowly to seawater) and subsequently fixed in 4% paraformaldehyde<sup><a aria-describedby="CR38" class="usa-link" href="#CR38">38</a></sup>. Sequences of interest were identified from the adult <em>E. scolopes</em> transcriptomes. cDNA of pooled developmental stages was used for PCR with Q5 polymerase. Products were cloned in pjet vectors and isolated with an innuPREP Plasmid Mini Kit (Analytik jena (Jena, Germany)) and sequenced. Riboprobes were generated from amplified minipreps and reverse transcribed with DIG-labeled nucleotides. Details on the FISH and ISH protocol can be found in Supplementary Note <a class="usa-link" href="#MOESM1">11</a>. Embryos and hatchlings were imaged on an inverted Zeiss (Oberkochen, Germany) LSM 780 multiphoton Confocal Laser Scanning Microscope.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par37">Further information on research design is available in the <a class="usa-link" href="#MOESM6">Nature Research Reporting Summary</a> linked to this article.</p></section></section><section id="sec22"><h2 class="pmc_sec_title">Supplementary information</h2>
<section id="Sec22"><section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM1_ESM.pdf">Supplementary Information</a><sup> (2.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM2_ESM.pdf">Peer Review File</a><sup> (4.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM3_ESM.pdf">41467_2022_29694_MOESM3_ESM.pdf</a><sup> (80.8KB, pdf) </sup><p>Description of additional Supplementary File</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM4_ESM.xlsx">Supplementary Dataset 1</a><sup> (814.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM5_ESM.xlsx">Supplementary Dataset 2</a><sup> (41.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM6_ESM.pdf">Reporting Summary</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>H.S., O.P.H., E.R., and O.S. were supported by the Austrian Science Fund (FWF) grant P30686-B29. O.S. was supported by Whitman Center Early Career Fellowship (Frank R. Lillie Quasi-Endowment Fund, L. &amp; A. Colwin Summer Research Fellowship, Bell Research Award in Tissue Engineering). H.S. was supported by the short-term grant abroad (KWA) of the University of Vienna. H.S. and O.S. were supported by the University of Chicago/Vienna Strategic Partnership Programme Mobility Grant. A.K. was supported by the JSPS Postdoctoral Fellowship for Overseas Researchers program from Japan. C.B.A. was supported by the Hibbitt Early Career Fellowship. Eggs and paralarvae of <em>E. scolopes</em> were generated in part by support by the NASA Space Biology 80NSSC18K1465 awarded to J.S.F. S.V.N. was supported by the National Science Foundation IOS-1557914. This work was supported by the Francis Crick Institute, which receives its core funding from Cancer Research UK (FC0001003), the UK Medical Research Council (FC001003), and the Wellcome Trust (FC001003). Authors wish to thank Vienna Zoo (Tiergarten Schönbrunn), in particular, Roland Halbauer and the aquaristics team for animal husbandry, as well as the MBL Cephalopod Program, their team, Emily Garcia, and the MBL Central Microscopy Facility (MBL, Woods Hole). Authors thank the Department of Neuroscience and Developmental Biology at the University of Vienna, especially Andreas Denner. Computation was done using the Life Sciences Cluster at the University of Vienna. Sectioning was done at the Core Facility CIUS (University of Vienna). Authors wish to thank Daniel Rokhsar and Clifton Ragsdale for guidance and advice.</p></section><section class="app" id="App1"><section id="Sec23"><h3 class="pmc_sec_title">Source data</h3>
<div class="media p" id="MOESM7"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM7_ESM.xlsx">Source Data</a><sup> (27.6KB, xlsx) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>H.S. performed comparative genomic analysis with the support of O.S., A.K. performed Hi-C experiments, H.S. and A.K. performed ATAC-seq data collection. H.S. and C.A. performed in situ hybridization. T.G., X.F., and P.B. performed 3D reconstruction and analysis. H.S. and O.P.H. performed ATAC-seq quantification. H.S. and B.Z. performed co-expression and syntenic randomization analysis, E.A.R., H.S., and C.A. performed RNA-seq extraction. J.F. and S.N. supplied animals. A.W. and Vienna Zoo housed animals. H.S., C.A., E.T., and O.S. designed the research. H.S. and O.S. wrote the manuscript with input from all other authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par38"><em>Nature Communications</em> thanks Jose Martín-Durán and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. <a class="usa-link" href="#MOESM2">Peer reviewer reports</a> are available</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data that support this study are available from the corresponding authors upon reasonable request. The Hi-C and ATAC-seq data have been deposited in the NCBI database under Bioproject <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA661684" rel="noopener noreferrer" target="_blank">PRJNA661684</a>. All expression, ATAC-seq, and CNE data that is mapped to the reference genome is available on a genome browser (currently, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.csb.univie.ac.at:8000/euprymna/jbrowse" rel="noopener noreferrer" target="_blank">http://metazoa.csb.univie.ac.at:8000/euprymna/jbrowse</a> or upon request). All other genomic and trascriptomic data used was downloaded from NCBI (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_002113885.1/" rel="noopener noreferrer" target="_blank">GCA_002113885.2</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000002075.1/" rel="noopener noreferrer" target="_blank">GCA_000002075.2</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.38/" rel="noopener noreferrer" target="_blank">GRCh38.p12</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_001949145.1/" rel="noopener noreferrer" target="_blank">GCA_001949145.1 OLI-Apl_1.0</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000003605.2" rel="noopener noreferrer" target="_blank">GCA_000003605.1</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000224145.3" rel="noopener noreferrer" target="_blank">GCA_000224145.2</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000003815.1/#/st" rel="noopener noreferrer" target="_blank">GCA_000003815.1</a> Version 2, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCA_004765925.1/" rel="noopener noreferrer" target="_blank">GCA_004765925.1</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000002235.3/" rel="noopener noreferrer" target="_blank">Spur_3.1</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000001635.26/" rel="noopener noreferrer" target="_blank">GRCm38.p6</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/biosample/?term=SAMN00691532" rel="noopener noreferrer" target="_blank">SAMN00691532</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/biosample/SAMN00152410/" rel="noopener noreferrer" target="_blank">SAMN00152410</a>), ENSEMBL (BDGP6.28 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.ensembl.org/Drosophila_melanogaster/Info/Index" rel="noopener noreferrer" target="_blank">http://www.ensembl.org/Drosophila_melanogaster/Info/Index</a>, WBcel235 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://m.ensembl.org/Caenorhabditis_elegans/Info/Annotation" rel="noopener noreferrer" target="_blank">http://m.ensembl.org/Caenorhabditis_elegans/Info/Annotation</a>, Capitella_teleta_v1.0 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Capitella_teleta/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Capitella_teleta/Info/Index</a>, ASM23792v2 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Schistosoma_mansoni/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Schistosoma_mansoni/Info/Index</a>, oyster_v9 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Crassostrea_gigas/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Crassostrea_gigas/Info/Index</a>, Helro1 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Helobdella_robusta/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Helobdella_robusta/Info/Index</a>, Lotgi1 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Lottia_gigantea/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Lottia_gigantea/Info/Index</a>, PRJNA270931 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://metazoa.ensembl.org/Octopus_bimaculoides/Info/Index" rel="noopener noreferrer" target="_blank">https://metazoa.ensembl.org/Octopus_bimaculoides/Info/Index</a>, Stegodyphus_mimosarum_v1 [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://metazoa.ensembl.org/Stegodyphus_mimosarum/Info/Index" rel="noopener noreferrer" target="_blank">https://metazoa.ensembl.org/Stegodyphus_mimosarum/Info/Index</a>], Tcas5.2 [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Tribolium_castaneum/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Tribolium_castaneum/Info/Index</a>, AMS_PRJEB1171_v1 [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://metazoa.ensembl.org/Adineta_vaga/Info/Index" rel="noopener noreferrer" target="_blank">https://metazoa.ensembl.org/Adineta_vaga/Info/Index</a>, GRCh37.p13 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://grch37.ensembl.org/Homo_sapiens/Info/Index" rel="noopener noreferrer" target="_blank">https://grch37.ensembl.org/Homo_sapiens/Info/Index</a>, ASM20922v1 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://metazoa.ensembl.org/Nematostella_vectensis/Info/Index" rel="noopener noreferrer" target="_blank">https://metazoa.ensembl.org/Nematostella_vectensis/Info/Index</a>, Aqu1 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://metazoa.ensembl.org/Amphimedon_queenslandica/Info/Index" rel="noopener noreferrer" target="_blank">https://metazoa.ensembl.org/Amphimedon_queenslandica/Info/Index</a>, MneLei_Aug2011 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Mnemiopsis_leidyi/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Mnemiopsis_leidyi/Info/Index</a>) or GIGA (PRJNA421033 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.ebi.ac.uk/ena/browser/view/PRJNA421033" rel="noopener noreferrer" target="_blank">https://www.ebi.ac.uk/ena/browser/view/PRJNA421033</a>). Human Hi-C data was downloaded from NCBI (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra/SRR1658570" rel="noopener noreferrer" target="_blank">SRR1658570</a>, HIC001). Processed files and tables needed to re-create the figures are accessible via a bitbucket repository: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://bitbucket.org/hannahschm/ceph_regulation_microsynteny/" rel="noopener noreferrer" target="_blank">https://bitbucket.org/hannahschm/ceph_regulation_microsynteny/</a>. <a class="usa-link" href="#Sec23">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>All bioinformatic protocols will be made available under <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://bitbucket.org/hannahschm/ceph_regulation_microsynteny/" rel="noopener noreferrer" target="_blank">https://bitbucket.org/hannahschm/ceph_regulation_microsynteny/</a> with detailed settings for each program and example scripts. The C++ script for 3D structures of individual chromosomes can be accessed upon request to T. Clarence (tereza.clarence@crick.ac.uk).</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par39">The authors declare no competing interests.</p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Hannah Schmidbaur, Akane Kawaguchi.</p></div>
</div></section><section class="contrib-info" id="_ci93_" lang="en"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Caroline B. Albertin, Email: calbertin@mbl.edu</p>
<p>Elly Tanaka, Email: elly.tanaka@imp.ac.at.</p>
<p>Oleg Simakov, Email: oleg.simakov@univie.ac.at.</p></section><section id="sec25"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-022-29694-7.</p></section><section class="ref-list" id="Bib1"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Ritschard EA, et al.  Coupled genomic evolutionary histories as signatures of organismal innovations in cephalopods: co-evolutionary signatures across levels of genome organization may shed light on functional linkage and origin of cephalopod novelties. BioEssays N. Rev. Mol. Cell. Dev. Biol. 2019;41:e1900073. doi: 10.1002/bies.201900073.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/bies.201900073" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31664724/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BioEssays%20N.%20Rev.%20Mol.%20Cell.%20Dev.%20Biol.&amp;title=Coupled%20genomic%20evolutionary%20histories%20as%20signatures%20of%20organismal%20innovations%20in%20cephalopods:%20co-evolutionary%20signatures%20across%20levels%20of%20genome%20organization%20may%20shed%20light%20on%20functional%20linkage%20and%20origin%20of%20cephalopod%20novelties&amp;author=EA%20Ritschard&amp;volume=41&amp;publication_year=2019&amp;pages=e1900073&amp;pmid=31664724&amp;doi=10.1002/bies.201900073&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Albertin CB, et al.  The octopus genome and the evolution of cephalopod neural and morphological novelties. Nature. 2015;524:220–224. doi: 10.1038/nature14668.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature14668" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4795812/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26268193/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20octopus%20genome%20and%20the%20evolution%20of%20cephalopod%20neural%20and%20morphological%20novelties&amp;author=CB%20Albertin&amp;volume=524&amp;publication_year=2015&amp;pages=220-224&amp;pmid=26268193&amp;doi=10.1038/nature14668&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Belcaid M, et al.  Symbiotic organs shaped by distinct modes of genome evolution in cephalopods. Proc. Natl Acad. Sci. USA. 2019;116:3030–3035. doi: 10.1073/pnas.1817322116.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1817322116" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6386654/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30635418/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=Symbiotic%20organs%20shaped%20by%20distinct%20modes%20of%20genome%20evolution%20in%20cephalopods&amp;author=M%20Belcaid&amp;volume=116&amp;publication_year=2019&amp;pages=3030-3035&amp;pmid=30635418&amp;doi=10.1073/pnas.1817322116&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Engström PG, Ho Sui SJ, Drivenes O, Becker TS, Lenhard B. Genomic regulatory blocks underlie extensive microsynteny conservation in insects. Genome Res. 2007;17:1898–1908. doi: 10.1101/gr.6669607.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.6669607" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2099597/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17989259/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Genomic%20regulatory%20blocks%20underlie%20extensive%20microsynteny%20conservation%20in%20insects&amp;author=PG%20Engstr%C3%B6m&amp;author=SJ%20Ho%20Sui&amp;author=O%20Drivenes&amp;author=TS%20Becker&amp;author=B%20Lenhard&amp;volume=17&amp;publication_year=2007&amp;pages=1898-1908&amp;pmid=17989259&amp;doi=10.1101/gr.6669607&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Irimia M, et al.  Extensive conservation of ancient microsynteny across metazoans due to cis-regulatory constraints. Genome Res. 2012;22:2356–2367. doi: 10.1101/gr.139725.112.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.139725.112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3514665/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22722344/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Extensive%20conservation%20of%20ancient%20microsynteny%20across%20metazoans%20due%20to%20cis-regulatory%20constraints&amp;author=M%20Irimia&amp;volume=22&amp;publication_year=2012&amp;pages=2356-2367&amp;pmid=22722344&amp;doi=10.1101/gr.139725.112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Simakov O, et al.  Insights into bilaterian evolution from three spiralian genomes. Nature. 2013;493:526–531. doi: 10.1038/nature11696.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature11696" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4085046/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23254933/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Insights%20into%20bilaterian%20evolution%20from%20three%20spiralian%20genomes&amp;author=O%20Simakov&amp;volume=493&amp;publication_year=2013&amp;pages=526-531&amp;pmid=23254933&amp;doi=10.1038/nature11696&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Kikuta H, et al.  Genomic regulatory blocks encompass multiple neighboring genes and maintain conserved synteny in vertebrates. Genome Res. 2007;17:545–555. doi: 10.1101/gr.6086307.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.6086307" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1855176/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17387144/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Genomic%20regulatory%20blocks%20encompass%20multiple%20neighboring%20genes%20and%20maintain%20conserved%20synteny%20in%20vertebrates&amp;author=H%20Kikuta&amp;volume=17&amp;publication_year=2007&amp;pages=545-555&amp;pmid=17387144&amp;doi=10.1101/gr.6086307&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zimmermann B, Robert NSM, Technau U, Simakov O. Ancient animal genome architecture reflects cell type identities. Nat. Ecol. Evol. 2019;3:1289–1293. doi: 10.1038/s41559-019-0946-7.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41559-019-0946-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31383947/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Ecol.%20Evol.&amp;title=Ancient%20animal%20genome%20architecture%20reflects%20cell%20type%20identities&amp;author=B%20Zimmermann&amp;author=NSM%20Robert&amp;author=U%20Technau&amp;author=O%20Simakov&amp;volume=3&amp;publication_year=2019&amp;pages=1289-1293&amp;pmid=31383947&amp;doi=10.1038/s41559-019-0946-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>McFall-Ngai MJ, Ruby EG. Symbiont recognition and subsequent morphogenesis as early events in an animal-bacterial mutualism. Science. 1991;254:1491–1494. doi: 10.1126/science.1962208.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1962208" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/1962208/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Symbiont%20recognition%20and%20subsequent%20morphogenesis%20as%20early%20events%20in%20an%20animal-bacterial%20mutualism&amp;author=MJ%20McFall-Ngai&amp;author=EG%20Ruby&amp;volume=254&amp;publication_year=1991&amp;pages=1491-1494&amp;pmid=1962208&amp;doi=10.1126/science.1962208&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Nyholm, S. V. &amp; McFall-Ngai, M. J. A lasting symbiosis: how the Hawaiian bobtail squid finds and keeps its bioluminescent bacterial partner. <em>Nat. Rev. Microbiol</em>. 10.1038/s41579-021-00567-y (2021).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41579-021-00567-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8440403/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34089010/">PubMed</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Wang S, et al.  Scallop genome provides insights into evolution of bilaterian karyotype and development. Nat. Ecol. Evol. 2017;1:s41559-017–0120–017. doi: 10.1038/s41559-017-0120.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41559-017-0120" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10970998/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28812685/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Ecol.%20Evol.&amp;title=Scallop%20genome%20provides%20insights%20into%20evolution%20of%20bilaterian%20karyotype%20and%20development&amp;author=S%20Wang&amp;volume=1&amp;publication_year=2017&amp;pages=s41559-017-0120%E2%80%93017&amp;pmid=28812685&amp;doi=10.1038/s41559-017-0120&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Zhang Y, et al.  The genome of Nautilus pompilius illuminates eye evolution and biomineralization. Nat. Ecol. Evol. 2021;5:927–938. doi: 10.1038/s41559-021-01448-6.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41559-021-01448-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8257504/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33972735/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Ecol.%20Evol.&amp;title=The%20genome%20of%20Nautilus%20pompilius%20illuminates%20eye%20evolution%20and%20biomineralization&amp;author=Y%20Zhang&amp;volume=5&amp;publication_year=2021&amp;pages=927-938&amp;pmid=33972735&amp;doi=10.1038/s41559-021-01448-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>van Berkum, N. L. et al. Hi-C: a method to study the three-dimensional architecture of genomes. <em>J. Vis. Exp. JoVE</em>10.3791/1869 (2010).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3791/1869" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3149993/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20461051/">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Friedman N, Rando OJ. Epigenomics and the structure of the living genome. Genome Res. 2015;25:1482–1490. doi: 10.1101/gr.190165.115.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.190165.115" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4579333/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26430158/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Epigenomics%20and%20the%20structure%20of%20the%20living%20genome&amp;author=N%20Friedman&amp;author=OJ%20Rando&amp;volume=25&amp;publication_year=2015&amp;pages=1482-1490&amp;pmid=26430158&amp;doi=10.1101/gr.190165.115&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Dali R, Blanchette M. A critical assessment of topologically associating domain prediction tools. Nucleic Acids Res. 2017;45:2994–3005. doi: 10.1093/nar/gkx145.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkx145" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5389712/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28334773/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=A%20critical%20assessment%20of%20topologically%20associating%20domain%20prediction%20tools&amp;author=R%20Dali&amp;author=M%20Blanchette&amp;volume=45&amp;publication_year=2017&amp;pages=2994-3005&amp;pmid=28334773&amp;doi=10.1093/nar/gkx145&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Cubeñas-Potts C, Corces VG. Architectural proteins, transcription, and the three-dimensional organization of the genome. FEBS Lett. 2015;589:2923–2930. doi: 10.1016/j.febslet.2015.05.025.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.febslet.2015.05.025" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4598269/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26008126/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&amp;title=Architectural%20proteins,%20transcription,%20and%20the%20three-dimensional%20organization%20of%20the%20genome&amp;author=C%20Cube%C3%B1as-Potts&amp;author=VG%20Corces&amp;volume=589&amp;publication_year=2015&amp;pages=2923-2930&amp;pmid=26008126&amp;doi=10.1016/j.febslet.2015.05.025&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Symmons O, et al.  Functional and topological characteristics of mammalian regulatory domains. Genome Res. 2014;24:390–400. doi: 10.1101/gr.163519.113.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.163519.113" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3941104/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24398455/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Functional%20and%20topological%20characteristics%20of%20mammalian%20regulatory%20domains&amp;author=O%20Symmons&amp;volume=24&amp;publication_year=2014&amp;pages=390-400&amp;pmid=24398455&amp;doi=10.1101/gr.163519.113&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Heger P, Marin B, Bartkuhn M, Schierenberg E, Wiehe T. The chromatin insulator CTCF and the emergence of metazoan diversity. Proc. Natl Acad. Sci. USA. 2012;109:17507–17512. doi: 10.1073/pnas.1111941109.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1111941109" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3491479/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23045651/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=The%20chromatin%20insulator%20CTCF%20and%20the%20emergence%20of%20metazoan%20diversity&amp;author=P%20Heger&amp;author=B%20Marin&amp;author=M%20Bartkuhn&amp;author=E%20Schierenberg&amp;author=T%20Wiehe&amp;volume=109&amp;publication_year=2012&amp;pages=17507-17512&amp;pmid=23045651&amp;doi=10.1073/pnas.1111941109&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Rubio ED, et al.  CTCF physically links cohesin to chromatin. Proc. Natl Acad. Sci. USA. 2008;105:8309–8314. doi: 10.1073/pnas.0801273105.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0801273105" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2448833/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18550811/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=CTCF%20physically%20links%20cohesin%20to%20chromatin&amp;author=ED%20Rubio&amp;volume=105&amp;publication_year=2008&amp;pages=8309-8314&amp;pmid=18550811&amp;doi=10.1073/pnas.0801273105&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Filippova GN, et al.  An exceptionally conserved transcriptional repressor, CTCF, employs different combinations of zinc fingers to bind diverged promoter sequences of avian and mammalian c-myc oncogenes. Mol. Cell. Biol. 1996;16:2802–2813. doi: 10.1128/MCB.16.6.2802.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/MCB.16.6.2802" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC231272/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8649389/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Biol.&amp;title=An%20exceptionally%20conserved%20transcriptional%20repressor,%20CTCF,%20employs%20different%20combinations%20of%20zinc%20fingers%20to%20bind%20diverged%20promoter%20sequences%20of%20avian%20and%20mammalian%20c-myc%20oncogenes&amp;author=GN%20Filippova&amp;volume=16&amp;publication_year=1996&amp;pages=2802-2813&amp;pmid=8649389&amp;doi=10.1128/MCB.16.6.2802&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Acemel, R. D., Maeso, I. &amp; Gómez-Skarmeta, J. L. Topologically associated domains: a successful scaffold for the evolution of gene regulation in animals. <em>Wiley Interdiscip. Rev. Dev. Biol</em>. <strong>6</strong>, e265 (2017).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/wdev.265" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28251841/">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Lazar NH, et al.  Epigenetic maintenance of topological domains in the highly rearranged gibbon genome. Genome Res. 2018;28:983–997. doi: 10.1101/gr.233874.117.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.233874.117" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6028127/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29914971/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Epigenetic%20maintenance%20of%20topological%20domains%20in%20the%20highly%20rearranged%20gibbon%20genome&amp;author=NH%20Lazar&amp;volume=28&amp;publication_year=2018&amp;pages=983-997&amp;pmid=29914971&amp;doi=10.1101/gr.233874.117&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Touceda-Suárez M, et al.  Ancient genomic regulatory blocks are a source for regulatory gene deserts in vertebrates after whole-genome duplications. Mol. Biol. Evol. 2020;37:2857–2864. doi: 10.1093/molbev/msaa123.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/msaa123" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7530604/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32421818/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=Ancient%20genomic%20regulatory%20blocks%20are%20a%20source%20for%20regulatory%20gene%20deserts%20in%20vertebrates%20after%20whole-genome%20duplications&amp;author=M%20Touceda-Su%C3%A1rez&amp;volume=37&amp;publication_year=2020&amp;pages=2857-2864&amp;pmid=32421818&amp;doi=10.1093/molbev/msaa123&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Kryuchkova-Mostacci N, Robinson-Rechavi M. Tissue-specificity of gene expression diverges slowly between orthologs, and rapidly between paralogs. PLOS Comput. Biol. 2016;12:e1005274. doi: 10.1371/journal.pcbi.1005274.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pcbi.1005274" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5193323/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28030541/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Comput.%20Biol.&amp;title=Tissue-specificity%20of%20gene%20expression%20diverges%20slowly%20between%20orthologs,%20and%20rapidly%20between%20paralogs&amp;author=N%20Kryuchkova-Mostacci&amp;author=M%20Robinson-Rechavi&amp;volume=12&amp;publication_year=2016&amp;pages=e1005274&amp;pmid=28030541&amp;doi=10.1371/journal.pcbi.1005274&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Buenrostro JD, Giresi PG, Zaba LC, Chang HY, Greenleaf WJ. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Nat. Methods. 2013;10:1213. doi: 10.1038/nmeth.2688.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.2688" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3959825/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24097267/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Transposition%20of%20native%20chromatin%20for%20fast%20and%20sensitive%20epigenomic%20profiling%20of%20open%20chromatin,%20DNA-binding%20proteins%20and%20nucleosome%20position&amp;author=JD%20Buenrostro&amp;author=PG%20Giresi&amp;author=LC%20Zaba&amp;author=HY%20Chang&amp;author=WJ%20Greenleaf&amp;volume=10&amp;publication_year=2013&amp;pages=1213&amp;pmid=24097267&amp;doi=10.1038/nmeth.2688&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Hood KN, et al.  Endoplasmic reticulum stress contributes to the loss of newborn hippocampal neurons after traumatic brain injury. J. Neurosci. 2018;38:2372–2384. doi: 10.1523/JNEUROSCI.1756-17.2018.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1523/JNEUROSCI.1756-17.2018" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5830522/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29386258/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Neurosci.&amp;title=Endoplasmic%20reticulum%20stress%20contributes%20to%20the%20loss%20of%20newborn%20hippocampal%20neurons%20after%20traumatic%20brain%20injury&amp;author=KN%20Hood&amp;volume=38&amp;publication_year=2018&amp;pages=2372-2384&amp;pmid=29386258&amp;doi=10.1523/JNEUROSCI.1756-17.2018&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Wang Y, et al.  The role of E2F1-topoIIβ signaling in regulation of cell cycle exit and neuronal differentiation of human SH-SY5Y cells. Differ. Res. Biol. Divers. 2018;104:1–12. doi: 10.1016/j.diff.2018.07.002.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.diff.2018.07.002" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30216786/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Differ.%20Res.%20Biol.%20Divers&amp;title=The%20role%20of%20E2F1-topoII%CE%B2%20signaling%20in%20regulation%20of%20cell%20cycle%20exit%20and%20neuronal%20differentiation%20of%20human%20SH-SY5Y%20cells&amp;author=Y%20Wang&amp;volume=104&amp;publication_year=2018&amp;pages=1-12&amp;pmid=30216786&amp;doi=10.1016/j.diff.2018.07.002&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Lu C, et al.  Overexpression of NEUROG2 and NEUROG1 in human embryonic stem cells produces a network of excitatory and inhibitory neurons. FASEB J. 2019;33:5287–5299. doi: 10.1096/fj.201801110RR.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1096/fj.201801110RR" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6436650/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30698461/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FASEB%20J.&amp;title=Overexpression%20of%20NEUROG2%20and%20NEUROG1%20in%20human%20embryonic%20stem%20cells%20produces%20a%20network%20of%20excitatory%20and%20inhibitory%20neurons&amp;author=C%20Lu&amp;volume=33&amp;publication_year=2019&amp;pages=5287-5299&amp;pmid=30698461&amp;doi=10.1096/fj.201801110RR&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kanatani S, et al.  The COUP-TFII/Neuropilin-2 is a molecular switch steering diencephalon-derived GABAergic neurons in the developing mouse brain. Proc. Natl Acad. Sci. USA. 2015;112:E4985–E4994. doi: 10.1073/pnas.1420701112.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1420701112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4568674/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26305926/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=The%20COUP-TFII/Neuropilin-2%20is%20a%20molecular%20switch%20steering%20diencephalon-derived%20GABAergic%20neurons%20in%20the%20developing%20mouse%20brain&amp;author=S%20Kanatani&amp;volume=112&amp;publication_year=2015&amp;pages=E4985-E4994&amp;pmid=26305926&amp;doi=10.1073/pnas.1420701112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Corona C, et al.  Activating transcription factor 4 (ATF4) regulates neuronal activity by controlling GABABR trafficking. J. Neurosci. 2018;38:6102–6113. doi: 10.1523/JNEUROSCI.3350-17.2018.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1523/JNEUROSCI.3350-17.2018" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6596153/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29875265/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Neurosci.&amp;title=Activating%20transcription%20factor%204%20(ATF4)%20regulates%20neuronal%20activity%20by%20controlling%20GABABR%20trafficking&amp;author=C%20Corona&amp;volume=38&amp;publication_year=2018&amp;pages=6102-6113&amp;pmid=29875265&amp;doi=10.1523/JNEUROSCI.3350-17.2018&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Shinghal R, Scheller RH, Bajjalieh SM. Ceramide 1-phosphate phosphatase activity in brain. J. Neurochem. 1993;61:2279–2285. doi: 10.1111/j.1471-4159.1993.tb07470.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1471-4159.1993.tb07470.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8245978/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Neurochem.&amp;title=Ceramide%201-phosphate%20phosphatase%20activity%20in%20brain&amp;author=R%20Shinghal&amp;author=RH%20Scheller&amp;author=SM%20Bajjalieh&amp;volume=61&amp;publication_year=1993&amp;pages=2279-2285&amp;pmid=8245978&amp;doi=10.1111/j.1471-4159.1993.tb07470.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Oegema R, et al.  Human mutations in integrator complex subunits link transcriptome integrity to brain development. PLoS Genet. 2017;13:e1006809. doi: 10.1371/journal.pgen.1006809.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pgen.1006809" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5466333/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28542170/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Human%20mutations%20in%20integrator%20complex%20subunits%20link%20transcriptome%20integrity%20to%20brain%20development&amp;author=R%20Oegema&amp;volume=13&amp;publication_year=2017&amp;pages=e1006809&amp;pmid=28542170&amp;doi=10.1371/journal.pgen.1006809&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Hamley IW. The amyloid beta peptide: a chemist’s perspective. Role in Alzheimer’s and fibrillization. Chem. Rev. 2012;112:5147–5192. doi: 10.1021/cr3000994.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/cr3000994" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22813427/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Rev.&amp;title=The%20amyloid%20beta%20peptide:%20a%20chemist%E2%80%99s%20perspective.%20Role%20in%20Alzheimer%E2%80%99s%20and%20fibrillization&amp;author=IW%20Hamley&amp;volume=112&amp;publication_year=2012&amp;pages=5147-5192&amp;pmid=22813427&amp;doi=10.1021/cr3000994&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Wang Q, Moore MJ, Adelmant G, Marto JA, Silver PA. PQBP1, a factor linked to intellectual disability, affects alternative splicing associated with neurite outgrowth. Genes Dev. 2013;27:615–626. doi: 10.1101/gad.212308.112.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gad.212308.112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3613609/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23512658/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=PQBP1,%20a%20factor%20linked%20to%20intellectual%20disability,%20affects%20alternative%20splicing%20associated%20with%20neurite%20outgrowth&amp;author=Q%20Wang&amp;author=MJ%20Moore&amp;author=G%20Adelmant&amp;author=JA%20Marto&amp;author=PA%20Silver&amp;volume=27&amp;publication_year=2013&amp;pages=615-626&amp;pmid=23512658&amp;doi=10.1101/gad.212308.112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Zadjali F, et al.  Homozygosity for FARSB mutation leads to Phe-tRNA synthetase-related disease of growth restriction, brain calcification, and interstitial lung disease. Hum. Mutat. 2018;39:1355–1359. doi: 10.1002/humu.23595.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/humu.23595" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30014610/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Hum.%20Mutat.&amp;title=Homozygosity%20for%20FARSB%20mutation%20leads%20to%20Phe-tRNA%20synthetase-related%20disease%20of%20growth%20restriction,%20brain%20calcification,%20and%20interstitial%20lung%20disease&amp;author=F%20Zadjali&amp;volume=39&amp;publication_year=2018&amp;pages=1355-1359&amp;pmid=30014610&amp;doi=10.1002/humu.23595&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Fiorito G, et al.  Guidelines for the care and welfare of cephalopods in research—a consensus based on an initiative by CephRes, FELASA and the Boyd Group. Lab. Anim. 2015;49:1–90. doi: 10.1177/0023677215580006.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1177/0023677215580006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26354955/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Lab.%20Anim.&amp;title=Guidelines%20for%20the%20care%20and%20welfare%20of%20cephalopods%20in%20research%E2%80%94a%20consensus%20based%20on%20an%20initiative%20by%20CephRes,%20FELASA%20and%20the%20Boyd%20Group&amp;author=G%20Fiorito&amp;volume=49&amp;publication_year=2015&amp;pages=1-90&amp;pmid=26354955&amp;doi=10.1177/0023677215580006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Butler-Struben, H. M., Brophy, S. M., Johnson, N. A. &amp; Crook, R. J. In vivo recording of neural and behavioral correlates of anesthesia induction, reversal, and euthanasia in cephalopod molluscs. <em>Front. Physiol</em>. <strong>9</strong>, 109 (2018).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fphys.2018.00109" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5826266/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29515454/">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Shigeno, S., Parnaik, R., Albertin, C. B. &amp; Ragsdale, C. W. Evidence for a cordal, not ganglionic, pattern of cephalopod brain neurogenesis. <em>Zoological Lett</em>. 10.1186/s40851-015-0026-z (2015).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40851-015-0026-z" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4657373/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26605071/">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Collins, A. J. &amp; Nyholm, S. V. Obtaining hemocytes from the Hawaiian bobtail squid Euprymna scolopes and observing their adherence to symbiotic and non-symbiotic bacteria. <em>J. Vis. Exp. JoVE</em> 1714, 10.3791/1714 (2010).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3791/1714" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2830252/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20150890/">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Lee PN, Callaerts P, de Couet HG. The embryonic development of the Hawaiian bobtail squid (Euprymna scolopes) Cold Spring Harb. Protoc. 2009;2009:pdb.ip77. doi: 10.1101/pdb.ip77.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/pdb.ip77" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20150049/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb.%20Protoc.&amp;title=The%20embryonic%20development%20of%20the%20Hawaiian%20bobtail%20squid%20(Euprymna%20scolopes)&amp;author=PN%20Lee&amp;author=P%20Callaerts&amp;author=HG%20de%20Couet&amp;volume=2009&amp;publication_year=2009&amp;pages=pdb.ip77&amp;pmid=20150049&amp;doi=10.1101/pdb.ip77&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Rao SSP, et al.  A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell. 2014;159:1665–1680. doi: 10.1016/j.cell.2014.11.021.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cell.2014.11.021" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5635824/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25497547/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%203D%20map%20of%20the%20human%20genome%20at%20kilobase%20resolution%20reveals%20principles%20of%20chromatin%20looping&amp;author=SSP%20Rao&amp;volume=159&amp;publication_year=2014&amp;pages=1665-1680&amp;pmid=25497547&amp;doi=10.1016/j.cell.2014.11.021&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Serra F, et al.  Automatic analysis and 3D-modelling of Hi-C data using TADbit reveals structural features of the fly chromatin colors. PLOS Comput. Biol. 2017;13:e1005665. doi: 10.1371/journal.pcbi.1005665.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pcbi.1005665" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5540598/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28723903/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Comput.%20Biol.&amp;title=Automatic%20analysis%20and%203D-modelling%20of%20Hi-C%20data%20using%20TADbit%20reveals%20structural%20features%20of%20the%20fly%20chromatin%20colors&amp;author=F%20Serra&amp;volume=13&amp;publication_year=2017&amp;pages=e1005665&amp;pmid=28723903&amp;doi=10.1371/journal.pcbi.1005665&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Wolff, J. et al. Galaxy HiCExplorer 3: a web server for reproducible Hi-C, capture Hi-C and single-cell Hi-C data analysis, quality control and visualization. <em>Nucleic Acids Res</em>. 10.1093/nar/gkaa220 (2020).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkaa220" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7319437/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32301980/">PubMed</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Camacho C, et al.  BLAST+: architecture and applications. BMC Bioinforma. 2009;10:421. doi: 10.1186/1471-2105-10-421.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2105-10-421" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2803857/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20003500/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinforma.&amp;title=BLAST+:%20architecture%20and%20applications&amp;author=C%20Camacho&amp;volume=10&amp;publication_year=2009&amp;pages=421&amp;pmid=20003500&amp;doi=10.1186/1471-2105-10-421&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Engström PG, Fredman D, Lenhard B. Ancora: a web resource for exploring highly conserved noncoding elements and their association with developmental regulatory genes. Genome Biol. 2008;9:R34. doi: 10.1186/gb-2008-9-2-r34.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/gb-2008-9-2-r34" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2374709/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18279518/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Ancora:%20a%20web%20resource%20for%20exploring%20highly%20conserved%20noncoding%20elements%20and%20their%20association%20with%20developmental%20regulatory%20genes&amp;author=PG%20Engstr%C3%B6m&amp;author=D%20Fredman&amp;author=B%20Lenhard&amp;volume=9&amp;publication_year=2008&amp;pages=R34&amp;pmid=18279518&amp;doi=10.1186/gb-2008-9-2-r34&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Bejerano G, et al.  Ultraconserved elements in the human genome. Science. 2004;304:1321–1325. doi: 10.1126/science.1098119.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1098119" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15131266/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Ultraconserved%20elements%20in%20the%20human%20genome&amp;author=G%20Bejerano&amp;volume=304&amp;publication_year=2004&amp;pages=1321-1325&amp;pmid=15131266&amp;doi=10.1126/science.1098119&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Neph S, et al.  BEDOPS: high-performance genomic feature operations. Bioinformatics. 2012;28:1919–1920. doi: 10.1093/bioinformatics/bts277.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/bts277" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3389768/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22576172/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BEDOPS:%20high-performance%20genomic%20feature%20operations&amp;author=S%20Neph&amp;volume=28&amp;publication_year=2012&amp;pages=1919-1920&amp;pmid=22576172&amp;doi=10.1093/bioinformatics/bts277&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Quinlan AR, Hall IM. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics. 2010;26:841–842. doi: 10.1093/bioinformatics/btq033.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btq033" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2832824/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20110278/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BEDTools:%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;author=AR%20Quinlan&amp;author=IM%20Hall&amp;volume=26&amp;publication_year=2010&amp;pages=841-842&amp;pmid=20110278&amp;doi=10.1093/bioinformatics/btq033&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Bailey TL, Johnson J, Grant CE, Noble WS. The MEME suite. Nucleic Acids Res. 2015;43:W39–W49. doi: 10.1093/nar/gkv416.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkv416" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4489269/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25953851/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20MEME%20suite&amp;author=TL%20Bailey&amp;author=J%20Johnson&amp;author=CE%20Grant&amp;author=WS%20Noble&amp;volume=43&amp;publication_year=2015&amp;pages=W39-W49&amp;pmid=25953851&amp;doi=10.1093/nar/gkv416&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Sayers EW, et al.  Database resources of the national center for biotechnology information. Nucleic Acids Res. 2022;50:D20–D26. doi: 10.1093/nar/gkab1112.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkab1112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8728269/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34850941/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Database%20resources%20of%20the%20national%20center%20for%20biotechnology%20information&amp;author=EW%20Sayers&amp;volume=50&amp;publication_year=2022&amp;pages=D20-D26&amp;pmid=34850941&amp;doi=10.1093/nar/gkab1112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Corces MR, et al.  An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues. Nat. Methods. 2017;14:959–962. doi: 10.1038/nmeth.4396.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.4396" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5623106/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28846090/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=An%20improved%20ATAC-seq%20protocol%20reduces%20background%20and%20enables%20interrogation%20of%20frozen%20tissues&amp;author=MR%20Corces&amp;volume=14&amp;publication_year=2017&amp;pages=959-962&amp;pmid=28846090&amp;doi=10.1038/nmeth.4396&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Buenrostro, et al.  ATAC-seq: a method for assaying chromatin accessibility genome-wide. Curr. Protoc. Mol. Biol. Ed. Frederick M Ausubel Al. 2015;109:21.29.1–21.29.9. doi: 10.1002/0471142727.mb2129s109.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/0471142727.mb2129s109" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4374986/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25559105/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Mol.%20Biol.%20Ed.%20Frederick%20M%20Ausubel%20Al&amp;title=ATAC-seq:%20a%20method%20for%20assaying%20chromatin%20accessibility%20genome-wide&amp;author=%20Buenrostro&amp;volume=109&amp;publication_year=2015&amp;pages=21.29.1-21.29.9&amp;pmid=25559105&amp;doi=10.1002/0471142727.mb2129s109&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Tanner AR, et al.  Molecular clocks indicate turnover and diversification of modern coleoid cephalopods during the Mesozoic Marine Revolution. Proc. R. Soc. B Biol. Sci. 2017;284:20162818. doi: 10.1098/rspb.2016.2818.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1098/rspb.2016.2818" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5360930/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28250188/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20R.%20Soc.%20B%20Biol.%20Sci.&amp;title=Molecular%20clocks%20indicate%20turnover%20and%20diversification%20of%20modern%20coleoid%20cephalopods%20during%20the%20Mesozoic%20Marine%20Revolution&amp;author=AR%20Tanner&amp;volume=284&amp;publication_year=2017&amp;pages=20162818&amp;pmid=28250188&amp;doi=10.1098/rspb.2016.2818&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Kröger B, Vinther J, Fuchs D. Cephalopod origin and evolution: a congruent picture emerging from fossils, development and molecules. BioEssays. 2011;33:602–613. doi: 10.1002/bies.201100001.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/bies.201100001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21681989/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BioEssays&amp;title=Cephalopod%20origin%20and%20evolution:%20a%20congruent%20picture%20emerging%20from%20fossils,%20development%20and%20molecules&amp;author=B%20Kr%C3%B6ger&amp;author=J%20Vinther&amp;author=D%20Fuchs&amp;volume=33&amp;publication_year=2011&amp;pages=602-613&amp;pmid=21681989&amp;doi=10.1002/bies.201100001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM1_ESM.pdf">Supplementary Information</a><sup> (2.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM2_ESM.pdf">Peer Review File</a><sup> (4.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM3_ESM.pdf">41467_2022_29694_MOESM3_ESM.pdf</a><sup> (80.8KB, pdf) </sup><p>Description of additional Supplementary File</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM4_ESM.xlsx">Supplementary Dataset 1</a><sup> (814.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM5_ESM.xlsx">Supplementary Dataset 2</a><sup> (41.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/9023564/bin/41467_2022_29694_MOESM6_ESM.pdf">Reporting Summary</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support this study are available from the corresponding authors upon reasonable request. The Hi-C and ATAC-seq data have been deposited in the NCBI database under Bioproject <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA661684" rel="noopener noreferrer" target="_blank">PRJNA661684</a>. All expression, ATAC-seq, and CNE data that is mapped to the reference genome is available on a genome browser (currently, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.csb.univie.ac.at:8000/euprymna/jbrowse" rel="noopener noreferrer" target="_blank">http://metazoa.csb.univie.ac.at:8000/euprymna/jbrowse</a> or upon request). All other genomic and trascriptomic data used was downloaded from NCBI (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_002113885.1/" rel="noopener noreferrer" target="_blank">GCA_002113885.2</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000002075.1/" rel="noopener noreferrer" target="_blank">GCA_000002075.2</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.38/" rel="noopener noreferrer" target="_blank">GRCh38.p12</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_001949145.1/" rel="noopener noreferrer" target="_blank">GCA_001949145.1 OLI-Apl_1.0</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000003605.2" rel="noopener noreferrer" target="_blank">GCA_000003605.1</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000224145.3" rel="noopener noreferrer" target="_blank">GCA_000224145.2</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000003815.1/#/st" rel="noopener noreferrer" target="_blank">GCA_000003815.1</a> Version 2, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCA_004765925.1/" rel="noopener noreferrer" target="_blank">GCA_004765925.1</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000002235.3/" rel="noopener noreferrer" target="_blank">Spur_3.1</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000001635.26/" rel="noopener noreferrer" target="_blank">GRCm38.p6</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/biosample/?term=SAMN00691532" rel="noopener noreferrer" target="_blank">SAMN00691532</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/biosample/SAMN00152410/" rel="noopener noreferrer" target="_blank">SAMN00152410</a>), ENSEMBL (BDGP6.28 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.ensembl.org/Drosophila_melanogaster/Info/Index" rel="noopener noreferrer" target="_blank">http://www.ensembl.org/Drosophila_melanogaster/Info/Index</a>, WBcel235 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://m.ensembl.org/Caenorhabditis_elegans/Info/Annotation" rel="noopener noreferrer" target="_blank">http://m.ensembl.org/Caenorhabditis_elegans/Info/Annotation</a>, Capitella_teleta_v1.0 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Capitella_teleta/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Capitella_teleta/Info/Index</a>, ASM23792v2 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Schistosoma_mansoni/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Schistosoma_mansoni/Info/Index</a>, oyster_v9 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Crassostrea_gigas/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Crassostrea_gigas/Info/Index</a>, Helro1 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Helobdella_robusta/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Helobdella_robusta/Info/Index</a>, Lotgi1 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Lottia_gigantea/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Lottia_gigantea/Info/Index</a>, PRJNA270931 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://metazoa.ensembl.org/Octopus_bimaculoides/Info/Index" rel="noopener noreferrer" target="_blank">https://metazoa.ensembl.org/Octopus_bimaculoides/Info/Index</a>, Stegodyphus_mimosarum_v1 [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://metazoa.ensembl.org/Stegodyphus_mimosarum/Info/Index" rel="noopener noreferrer" target="_blank">https://metazoa.ensembl.org/Stegodyphus_mimosarum/Info/Index</a>], Tcas5.2 [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Tribolium_castaneum/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Tribolium_castaneum/Info/Index</a>, AMS_PRJEB1171_v1 [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://metazoa.ensembl.org/Adineta_vaga/Info/Index" rel="noopener noreferrer" target="_blank">https://metazoa.ensembl.org/Adineta_vaga/Info/Index</a>, GRCh37.p13 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://grch37.ensembl.org/Homo_sapiens/Info/Index" rel="noopener noreferrer" target="_blank">https://grch37.ensembl.org/Homo_sapiens/Info/Index</a>, ASM20922v1 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://metazoa.ensembl.org/Nematostella_vectensis/Info/Index" rel="noopener noreferrer" target="_blank">https://metazoa.ensembl.org/Nematostella_vectensis/Info/Index</a>, Aqu1 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://metazoa.ensembl.org/Amphimedon_queenslandica/Info/Index" rel="noopener noreferrer" target="_blank">https://metazoa.ensembl.org/Amphimedon_queenslandica/Info/Index</a>, MneLei_Aug2011 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://metazoa.ensembl.org/Mnemiopsis_leidyi/Info/Index" rel="noopener noreferrer" target="_blank">http://metazoa.ensembl.org/Mnemiopsis_leidyi/Info/Index</a>) or GIGA (PRJNA421033 <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.ebi.ac.uk/ena/browser/view/PRJNA421033" rel="noopener noreferrer" target="_blank">https://www.ebi.ac.uk/ena/browser/view/PRJNA421033</a>). Human Hi-C data was downloaded from NCBI (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra/SRR1658570" rel="noopener noreferrer" target="_blank">SRR1658570</a>, HIC001). Processed files and tables needed to re-create the figures are accessible via a bitbucket repository: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://bitbucket.org/hannahschm/ceph_regulation_microsynteny/" rel="noopener noreferrer" target="_blank">https://bitbucket.org/hannahschm/ceph_regulation_microsynteny/</a>. <a class="usa-link" href="#Sec23">Source data</a> are provided with this paper.</p>
<p>All bioinformatic protocols will be made available under <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://bitbucket.org/hannahschm/ceph_regulation_microsynteny/" rel="noopener noreferrer" target="_blank">https://bitbucket.org/hannahschm/ceph_regulation_microsynteny/</a> with detailed settings for each program and example scripts. The C++ script for 3D structures of individual chromosomes can be accessed upon request to T. Clarence (tereza.clarence@crick.ac.uk).</p></section></section></section></section></article>
</main></body></html>