<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7561690/"/><title>Draft Genome Sequences of Klebsiella Species Isolated from the International Space Station - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Microbiol Resour Announc</button></div>. 2020 Oct 15;9(42):e00923-20. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/MRA.00923-20" rel="noopener noreferrer" target="_blank">10.1128/MRA.00923-20</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Draft Genome Sequences of <em>Klebsiella</em> Species Isolated from the International Space Station</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Solomon%20SA%22%5BAuthor%5D"><span class="name western">Samuel A Solomon</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Samuel A Solomon</span></h3>
<div class="p">
<sup>a</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Solomon%20SA%22%5BAuthor%5D"><span class="name western">Samuel A Solomon</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bharadwaj%20AR%22%5BAuthor%5D"><span class="name western">Achintya R Bharadwaj</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Achintya R Bharadwaj</span></h3>
<div class="p">
<sup>a</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bharadwaj%20AR%22%5BAuthor%5D"><span class="name western">Achintya R Bharadwaj</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D"><span class="name western">Nitin K Singh</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Nitin K Singh</span></h3>
<div class="p">
<sup>a</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D"><span class="name western">Nitin K Singh</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wood%20JM%22%5BAuthor%5D"><span class="name western">Jason M Wood</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jason M Wood</span></h3>
<div class="p">
<sup>a</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wood%20JM%22%5BAuthor%5D"><span class="name western">Jason M Wood</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Debieu%20M%22%5BAuthor%5D"><span class="name western">Marilyne Debieu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Marilyne Debieu</span></h3>
<div class="p">
<sup>b</sup>Biotia, New York, New York, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Debieu%20M%22%5BAuthor%5D"><span class="name western">Marilyne Debieu</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Hara%20NB%22%5BAuthor%5D"><span class="name western">Niamh B O’Hara</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Niamh B O’Hara</span></h3>
<div class="p">
<sup>b</sup>Biotia, New York, New York, USA</div>
<div class="p">
<sup>c</sup>Department of Cell Biology, College of Medicine, SUNY Downstate Health Sciences University, Brooklyn, New York, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Hara%20NB%22%5BAuthor%5D"><span class="name western">Niamh B O’Hara</span></a>
</div>
</div>
<sup>b,</sup><sup>c</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D"><span class="name western">Christopher E Mason</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Christopher E Mason</span></h3>
<div class="p">
<sup>d</sup>WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, New York, USA</div>
<div class="p">
<sup>e</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, New York, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D"><span class="name western">Christopher E Mason</span></a>
</div>
</div>
<sup>d,</sup><sup>e</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Kasthuri Venkateswaran</span></h3>
<div class="p">
<sup>a</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a>
</div>
</div>
<sup>a,</sup><sup>✉</sup>
</div>
<div class="cg p">Editor: <span class="name western">David A Baltrus</span><sup>f</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>a</sup>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div id="aff2">
<sup>b</sup>Biotia, New York, New York, USA</div>
<div id="aff3">
<sup>c</sup>Department of Cell Biology, College of Medicine, SUNY Downstate Health Sciences University, Brooklyn, New York, USA</div>
<div id="aff4">
<sup>d</sup>WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, New York, USA</div>
<div id="aff5">
<sup>e</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, New York, USA</div>
<div id="aff6">
<sup>f</sup>University of Arizona</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>✉</sup><p class="display-inline">Address correspondence to Kasthuri Venkateswaran, <span>kjvenkat@jpl.nasa.gov</span>.</p>
</div>
<div class="fn" id="fn1"><p><strong>Citation</strong> Solomon SA, Bharadwaj AR, Singh NK, Wood JM, Debieu M, O’Hara NB, Mason CE, Venkateswaran K. 2020. Draft genome sequences of <em>Klebsiella</em> species isolated from the International Space Station. Microbiol Resour Announc 9:e00923-20. <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/MRA.00923-20" rel="noopener noreferrer" target="_blank">https://doi.org/10.1128/MRA.00923-20</a>.</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">David A Baltrus</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2020 Aug 11; Accepted 2020 Sep 24; Collection date 2020 Oct.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2020 Solomon et al.</div>
<p>This is an open-access article distributed under the terms of the <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">Creative Commons Attribution 4.0 International license</a>.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC7561690  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33060271/">33060271</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><hr class="headless"/>
<p>Isolated across four locations aboard the International Space Station (ISS), 10 bacterial strains were compared using whole-genome sequencing analysis and were phylogenetically identified as <em>Klebsiella</em>. The whole-genome sequences will aid in comparative genomic studies of ISS <em>Klebsiella</em> strains with Earth counterparts, to gain insight into their adaptation to space conditions.</p></section><section class="abstract" id="abstract2"><h2>ABSTRACT</h2>
<p>Isolated across four locations aboard the International Space Station (ISS), 10 bacterial strains were compared using whole-genome sequencing analysis and were phylogenetically identified as <em>Klebsiella</em>. The whole-genome sequences will aid in comparative genomic studies of ISS <em>Klebsiella</em> strains with Earth counterparts to gain insight into their adaptation to space conditions.</p></section><section id="s1"><h2 class="pmc_sec_title">ANNOUNCEMENT</h2>
<p>The genus <em>Klebsiella</em> was discovered by Carl Friedlander in 1882 from the lungs of patients who had died of pneumonia (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>). In infected individuals, <em>Klebsiella</em> species can populate the gastrointestinal tract and nasopharynx, surviving on mucosal surfaces, and are known for being highly virulent and resistant to antibiotics (<a aria-describedby="B2" class="usa-link" href="#B2">2</a>, <a aria-describedby="B3" class="usa-link" href="#B3">3</a>). When found on Earth, this genus of bacterial pathogens has various degrees of pathogenicity, which can lead to severe breathing problems necessitating a ventilator and rapid on-site treatment (<a aria-describedby="B4" class="usa-link" href="#B4">4</a>). When exposed to space conditions, however, <em>Klebsiella</em> species might become immunogenic and thus pose a risk to already immunocompromised astronauts aboard the International Space Station (ISS) (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>, <a aria-describedby="B6" class="usa-link" href="#B6">6</a>). Since microgravity and radiation in space are reported to induce multiple genetic adaptations in microbial species, such as structural modifications of the cell membrane, that can subsequently alter their virulence (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>), the strains identified here might potentially pose a problem for the health of astronauts. Therefore, a genetic comparison is necessary to provide more details on the survival of <em>Klebsiella</em> species, which have gained more attention because classical <em>Klebsiella pneumoniae</em> and its hypervirulent pathotype are becoming increasingly resistant to various antibiotics, such as carbapenems (<a aria-describedby="B3" class="usa-link" href="#B3">3</a>, <a aria-describedby="B4" class="usa-link" href="#B4">4</a>, <a aria-describedby="B7" class="usa-link" href="#B7">7</a>). The pathogenicity and resistance of the members of the genus <em>Klebsiella</em> might potentially create a problem for space travel, specifically for the safety of astronauts.</p>
<p>Several strains of <em>Klebsiella</em> species, including <em>K. aerogenes</em> (<em>n</em> = 1), <em>K. pneumoniae</em> (<em>n</em> = 1), and <em>K. quasipneumoniae</em> (<em>n</em> = 8), were isolated from various locations on ISS environmental surfaces (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>). The flight number, location, and other sampling characteristics of the ISS <em>Klebsiella</em> isolates are detailed in <a class="usa-link" href="#tab1">Table 1</a>. Briefly, the environmental samples collected from the ISS and subsequently brought down to Earth at room temperature were aseptically handled according to established procedures (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>), and 100-μl aliquots of concentrated samples were spread onto either Reasoner’s 2A (R2A) agar (25°C for 7 days) or blood agar (37°C for 2 days) for isolation of microorganisms. After morphological observation, pure colonies were archived at −80°C until further analyses. Cultures of the 10 <em>Klebsiella</em> strains were grown overnight on tryptic soy agar at 25°C until harvesting and DNA extraction using the ZymoBIOMICS DNA Magbead kit.</p>
<section class="tw xbox font-sm" id="tab1"><h3 class="obj_head">TABLE 1.</h3>
<div class="caption p"><p>Metadata and genome statistics of <em>Klebsiella</em> strains isolated from various ISS environmental surfaces during the Microbial Tracking-1 flight project</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
</colgroup>
<thead><tr>
<th colspan="1" rowspan="1">Sample name</th>
<th colspan="1" rowspan="1">Nearest species<a class="usa-link" href="#ngtab1.1"><sup><em>a</em></sup></a>
</th>
<th colspan="1" rowspan="1">ANI (%)<a class="usa-link" href="#ngtab1.2"><sup><em>b</em></sup></a>
</th>
<th colspan="1" rowspan="1">GenBank accession no.</th>
<th colspan="1" rowspan="1">Raw read accession no.</th>
<th colspan="1" rowspan="1">Flight-location<a class="usa-link" href="#ngtab1.3"><sup><em>c</em></sup></a>
</th>
<th colspan="1" rowspan="1">Location description<a class="usa-link" href="#ngtab1.4"><sup><em>d</em></sup></a>
</th>
<th colspan="1" rowspan="1">No. of contigs</th>
<th colspan="1" rowspan="1">Genome size (bp)</th>
<th colspan="1" rowspan="1">
<em>N</em><sub>50</sub> (bp)</th>
<th colspan="1" rowspan="1">Median coverage (×)</th>
<th colspan="1" rowspan="1">No. of raw reads (×10<sup>6</sup>)</th>
<th colspan="1" rowspan="1">No. of quality-controlled reads (×10<sup>6</sup>)</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">IIIF7SW-P1</td>
<td colspan="1" rowspan="1">
<em>K. aerogenes</em> ATCC 13048<sup>T</sup>
</td>
<td colspan="1" rowspan="1">98.66</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACBPC000000000" rel="noopener noreferrer" target="_blank">JACBPC000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra/SRR12071884" rel="noopener noreferrer" target="_blank">SRR12071884</a></td>
<td colspan="1" rowspan="1">F3-7</td>
<td colspan="1" rowspan="1">Lab 3</td>
<td colspan="1" rowspan="1">25</td>
<td colspan="1" rowspan="1">5,155,046</td>
<td colspan="1" rowspan="1">502,580</td>
<td colspan="1" rowspan="1">346.88</td>
<td colspan="1" rowspan="1">21.6</td>
<td colspan="1" rowspan="1">21.5</td>
</tr>
<tr>
<td colspan="1" rowspan="1">F3-2P(2*)</td>
<td colspan="1" rowspan="1">
<em>K. pneumoniae</em> ATCC 13883<sup>T</sup>
</td>
<td colspan="1" rowspan="1">99.01</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACAUF000000000" rel="noopener noreferrer" target="_blank">JACAUF000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra/SRR12068826" rel="noopener noreferrer" target="_blank">SRR12068826</a></td>
<td colspan="1" rowspan="1">F3-2</td>
<td colspan="1" rowspan="1">WHC</td>
<td colspan="1" rowspan="1">75</td>
<td colspan="1" rowspan="1">5,489,009</td>
<td colspan="1" rowspan="1">332,420</td>
<td colspan="1" rowspan="1">293.62</td>
<td colspan="1" rowspan="1">19.0</td>
<td colspan="1" rowspan="1">18.9</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IF1SW-B2</td>
<td colspan="1" rowspan="1">
<em>K. quasipneumoniae</em> 01A030<sup>T</sup>
</td>
<td colspan="1" rowspan="1">96.54</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JABWPD000000000" rel="noopener noreferrer" target="_blank">JABWPD000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra/SRR11884995" rel="noopener noreferrer" target="_blank">SRR11884995</a></td>
<td colspan="1" rowspan="1">F1-1</td>
<td colspan="1" rowspan="1">Cupola</td>
<td colspan="1" rowspan="1">24</td>
<td colspan="1" rowspan="1">5,192,853</td>
<td colspan="1" rowspan="1">601,624</td>
<td colspan="1" rowspan="1">533.04</td>
<td colspan="1" rowspan="1">31.4</td>
<td colspan="1" rowspan="1">31.2</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IF1SW-P3</td>
<td colspan="1" rowspan="1">
<em>K. quasipneumoniae</em> 01A030<sup>T</sup>
</td>
<td colspan="1" rowspan="1">96.60</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JABWOZ000000000" rel="noopener noreferrer" target="_blank">JABWOZ000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra/SRR11885008" rel="noopener noreferrer" target="_blank">SRR11885008</a></td>
<td colspan="1" rowspan="1">F1-1</td>
<td colspan="1" rowspan="1">Cupola</td>
<td colspan="1" rowspan="1">28</td>
<td colspan="1" rowspan="1">5,192,422</td>
<td colspan="1" rowspan="1">600,611</td>
<td colspan="1" rowspan="1">467.41</td>
<td colspan="1" rowspan="1">28.2</td>
<td colspan="1" rowspan="1">28.0</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IF1SW-P4</td>
<td colspan="1" rowspan="1">
<em>K. quasipneumoniae</em> 01A030<sup>T</sup>
</td>
<td colspan="1" rowspan="1">96.55</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JABWPA000000000" rel="noopener noreferrer" target="_blank">JABWPA000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra/SRR11885009" rel="noopener noreferrer" target="_blank">SRR11885009</a></td>
<td colspan="1" rowspan="1">F1-1</td>
<td colspan="1" rowspan="1">Cupola</td>
<td colspan="1" rowspan="1">29</td>
<td colspan="1" rowspan="1">5,192,468</td>
<td colspan="1" rowspan="1">343,111</td>
<td colspan="1" rowspan="1">297.32</td>
<td colspan="1" rowspan="1">18.6</td>
<td colspan="1" rowspan="1">18.6</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IF2SW-B3</td>
<td colspan="1" rowspan="1">
<em>K. quasipneumoniae</em> 01A030<sup>T</sup>
</td>
<td colspan="1" rowspan="1">96.54</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JABWPC000000000" rel="noopener noreferrer" target="_blank">JABWPC000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra/SRR11885011" rel="noopener noreferrer" target="_blank">SRR11885011</a></td>
<td colspan="1" rowspan="1">F1-2</td>
<td colspan="1" rowspan="1">WHC</td>
<td colspan="1" rowspan="1">24</td>
<td colspan="1" rowspan="1">5,192,354</td>
<td colspan="1" rowspan="1">600,611</td>
<td colspan="1" rowspan="1">712.50</td>
<td colspan="1" rowspan="1">39.8</td>
<td colspan="1" rowspan="1">39.6</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IF2SW-P1</td>
<td colspan="1" rowspan="1">
<em>K. quasipneumoniae</em> 01A030<sup>T</sup>
</td>
<td colspan="1" rowspan="1">96.61</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JABWPB000000000" rel="noopener noreferrer" target="_blank">JABWPB000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra/SRR11885010" rel="noopener noreferrer" target="_blank">SRR11885010</a></td>
<td colspan="1" rowspan="1">F1-2</td>
<td colspan="1" rowspan="1">WHC</td>
<td colspan="1" rowspan="1">27</td>
<td colspan="1" rowspan="1">5,192,385</td>
<td colspan="1" rowspan="1">601,139</td>
<td colspan="1" rowspan="1">480.80</td>
<td colspan="1" rowspan="1">30.2</td>
<td colspan="1" rowspan="1">30.1</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIIF3SW-P1</td>
<td colspan="1" rowspan="1">
<em>K. quasipneumoniae</em> 01A030<sup>T</sup>
</td>
<td colspan="1" rowspan="1">96.63</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JABXWM000000000" rel="noopener noreferrer" target="_blank">JABXWM000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra/SRR12070037" rel="noopener noreferrer" target="_blank">SRR12070037</a></td>
<td colspan="1" rowspan="1">F3-3</td>
<td colspan="1" rowspan="1">ARED</td>
<td colspan="1" rowspan="1">31</td>
<td colspan="1" rowspan="1">5,154,415</td>
<td colspan="1" rowspan="1">766,557</td>
<td colspan="1" rowspan="1">286.96</td>
<td colspan="1" rowspan="1">18.0</td>
<td colspan="1" rowspan="1">17.8</td>
</tr>
<tr>
<td colspan="1" rowspan="1">F3-6P(1)</td>
<td colspan="1" rowspan="1">
<em>K. quasipneumoniae</em> 01A030<sup>T</sup>
</td>
<td colspan="1" rowspan="1">96.57</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JABXWL000000000" rel="noopener noreferrer" target="_blank">JABXWL000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra/SRR12070038" rel="noopener noreferrer" target="_blank">SRR12070038</a></td>
<td colspan="1" rowspan="1">F3-6</td>
<td colspan="1" rowspan="1">PMM</td>
<td colspan="1" rowspan="1">28</td>
<td colspan="1" rowspan="1">5,196,291</td>
<td colspan="1" rowspan="1">1,009,008</td>
<td colspan="1" rowspan="1">371.62</td>
<td colspan="1" rowspan="1">23.2</td>
<td colspan="1" rowspan="1">22.9</td>
</tr>
<tr>
<td colspan="1" rowspan="1">F3-6P(2)</td>
<td colspan="1" rowspan="1">
<em>K. quasipneumoniae</em> 01A030<sup>T</sup>
</td>
<td colspan="1" rowspan="1">96.60</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JABXWK000000000" rel="noopener noreferrer" target="_blank">JABXWK000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra/SRR12070039" rel="noopener noreferrer" target="_blank">SRR12070039</a></td>
<td colspan="1" rowspan="1">F3-6</td>
<td colspan="1" rowspan="1">PMM</td>
<td colspan="1" rowspan="1">31</td>
<td colspan="1" rowspan="1">5,196,372</td>
<td colspan="1" rowspan="1">1,009,008</td>
<td colspan="1" rowspan="1">367.78</td>
<td colspan="1" rowspan="1">23.4</td>
<td colspan="1" rowspan="1">23.1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/tab1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="ngtab1.1">
<sup>a</sup><p class="display-inline">The 16S rRNA gene sequences were retrieved from the whole-genome sequences of the queried genomes and subjected to BLAST analysis against type strains for all 16S rRNA sequences in the NCBI database. Bacterial species identity was determined when the queried sequence showed &gt;97.5% similarity to the 16S rRNA gene sequence of the type strain. The whole-genome sequence of the nearest neighbor listed was selected for ANI evaluation.</p>
</div>
<div class="fn" id="ngtab1.2">
<sup>b</sup><p class="display-inline">ANI calculations were carried out using the EZBioCloud ANI calculator (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.ezbiocloud.net/tools/ani" rel="noopener noreferrer" target="_blank">https://www.ezbiocloud.net/tools/ani</a>) by comparing with the listed type strain.</p>
</div>
<div class="fn" id="ngtab1.3">
<sup>c</sup><p class="display-inline">Hyphenated designations indicate the flight number followed by the location; for example, F3-7 indicates flight 3 and location 7.</p>
</div>
<div class="fn" id="ngtab1.4">
<sup>d</sup><p class="display-inline">WHC, waste and hygiene compartment; ARED, advanced resistive exercise device; PMM, permanent multipurpose module port 1.</p>
</div>
</div></section><p>Genomes were sequenced using the Illumina (San Diego, CA) Nextera Flex protocol for library preparation, and a NovaSeq 6000 S4 flow cell (paired end, 2 × 150 bp) was used for paired-end sequencing. The quality was assessed with FastQC (v0.11.7) (<a aria-describedby="B9" class="usa-link" href="#B9">9</a>). Adapter trimming and quality filtering were then carried out with fastp (v0.20.0) (<a aria-describedby="B10" class="usa-link" href="#B10">10</a>). After quality control, the sequences were assembled using SPAdes (v3.11.1) (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>). To assess the quality of the final sequences, a QUAST (v5.0.2) analysis (<a aria-describedby="B12" class="usa-link" href="#B12">12</a>) was performed to check the <em>N</em><sub>50</sub> values, the number of contigs, and the total genome length (<a class="usa-link" href="#tab1">Table 1</a>). The GC contents are 54.96% for <em>K. aerogenes</em>, 57.25% for <em>K. pneumoniae</em>, and 58.11 to 58.13% for <em>K. quasipneumoniae</em>. The 16S rRNA gene sequences of the <em>Klebsiella</em> strains were compared to find the nearest neighbor, and phylogenic characterization was determined by calculating the average nucleotide identity (ANI) using the EZBioCloud calculator (<a aria-describedby="B13" class="usa-link" href="#B13">13</a>), in comparison with the respective type strains (<em>K. aerogenes</em> ATCC 13048<sup>T</sup>, <em>K. pneumoniae</em> ATCC 13883<sup>T</sup>, and <em>K. quasipneumoniae</em> 01A030<sup>T</sup>). Default parameters were used for all software.</p>
<section id="s1.1"><h3 class="pmc_sec_title">Data availability.</h3>
<p>This whole-genome sequencing project has been deposited in GenBank, and the GenBank and raw read accession numbers are given in <a class="usa-link" href="#tab1">Table 1</a>. The BioProject accession numbers are <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA635942" rel="noopener noreferrer" target="_blank">PRJNA635942</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA640688" rel="noopener noreferrer" target="_blank">PRJNA640688</a>, and <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA640693" rel="noopener noreferrer" target="_blank">PRJNA640693</a>. Whole-genome sequencing data have also been deposited in NASA GeneLab (accession numbers <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-302" rel="noopener noreferrer" target="_blank">GLDS-302</a>, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-309" rel="noopener noreferrer" target="_blank">GLDS-309</a>, and <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-311" rel="noopener noreferrer" target="_blank">GLDS-311</a>). The versions described in this paper are the first versions.</p></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank astronauts Captain Terry Virts and Commander Jeffrey Williams for collecting samples aboard the ISS and the Implementation Team at NASA Ames Research Center (Fathi Karouia) for coordinating this effort. We thank Ryan Kemp (Zymo Corp.) for extracting DNA and Dan Butler (Cornell Medicine) for performing shotgun sequencing using the NovaSeq platform. We also acknowledge the Jet Propulsion Laboratory supercomputing facility staff, notably Narendra J. Patel (Jimmy) and Edward Villanueva, for their continuous support in providing the best possible infrastructure for BIG-DATA analysis.</p>
<p>The research described in this article was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with NASA. This research was funded by 2012 Space Biology project NNH12ZTT001N grant 19-12829-26 under task order NNN13D111T awarded to K.V., which also funded a postdoctoral fellowship for J.M.W. and a subcontract to Biotia, Inc. Government sponsorship is acknowledged.</p></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>Friedlaender C.
1882. 
Ueber die Schizomyceten bei der acuten fibrösen Pneumonie. Arch Pathol Anat
87:319–324. doi: 10.1007/BF01880516.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/BF01880516" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Arch%20Pathol%20Anat&amp;title=Ueber%20die%20Schizomyceten%20bei%20der%20acuten%20fibr%C3%B6sen%20Pneumonie&amp;author=C%20Friedlaender&amp;volume=87&amp;publication_year=1882&amp;pages=319-324&amp;doi=10.1007/BF01880516&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>Martin RM, Bachman MA. 
2018. 
Colonization, infection, and the accessory genome of <em>Klebsiella pneumoniae</em>. Front Cell Infect Microbiol
8:4. doi: 10.3389/fcimb.2018.00004.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fcimb.2018.00004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5786545/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29404282/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Infect%20Microbiol&amp;title=Colonization,%20infection,%20and%20the%20accessory%20genome%20of%20Klebsiella%20pneumoniae&amp;author=RM%20Martin&amp;author=MA%20Bachman&amp;volume=8&amp;publication_year=2018&amp;pages=4&amp;pmid=29404282&amp;doi=10.3389/fcimb.2018.00004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>Garbati MA, Al Godhair AI. 
2013. 
The growing resistance of <em>Klebsiella pneumoniae</em>; the need to expand our antibiogram: case report and review of the literature. Afr J Infect Dis
7:8–10. doi: 10.4314/ajid.v7i1.2.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4314/ajid.v7i1.2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3647522/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24381721/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Afr%20J%20Infect%20Dis&amp;title=The%20growing%20resistance%20of%20Klebsiella%20pneumoniae;%20the%20need%20to%20expand%20our%20antibiogram:%20case%20report%20and%20review%20of%20the%20literature&amp;author=MA%20Garbati&amp;author=AI%20Al%20Godhair&amp;volume=7&amp;publication_year=2013&amp;pages=8-10&amp;pmid=24381721&amp;doi=10.4314/ajid.v7i1.2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>Domínguez-Andrés J, Eleveld M, Renieris G, Boltje TJ, Mesman RJ, van Niftrik L, Moons SJ, Rettberg P, van der Meer JWM, Giamarellos-Bourboulis EJ, Op den Camp HJM, de Jonge MI, Netea MG. 
2020. 
Growth on carbohydrates from carbonaceous meteorites alters the immunogenicity of environment-derived bacterial pathogens. Astrobiology. doi: 10.1089/ast.2019.2173.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2019.2173" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32391711/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Growth%20on%20carbohydrates%20from%20carbonaceous%20meteorites%20alters%20the%20immunogenicity%20of%20environment-derived%20bacterial%20pathogens&amp;author=J%20Dom%C3%ADnguez-Andr%C3%A9s&amp;author=M%20Eleveld&amp;author=G%20Renieris&amp;author=TJ%20Boltje&amp;author=RJ%20Mesman&amp;publication_year=2020&amp;pmid=32391711&amp;doi=10.1089/ast.2019.2173&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>Taylor PW.
2015. 
Impact of space flight on bacterial virulence and antibiotic susceptibility. Infect Drug Resist
8:249–262. doi: 10.2147/IDR.S67275.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2147/IDR.S67275" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4524529/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26251622/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Infect%20Drug%20Resist&amp;title=Impact%20of%20space%20flight%20on%20bacterial%20virulence%20and%20antibiotic%20susceptibility&amp;author=PW%20Taylor&amp;volume=8&amp;publication_year=2015&amp;pages=249-262&amp;pmid=26251622&amp;doi=10.2147/IDR.S67275&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>Russo TA, Marr CM. 
2019. 
Hypervirulent <em>Klebsiella pneumoniae</em>. Clin Microbiol Rev
32:e00001-19. doi: 10.1128/CMR.00001-19.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/CMR.00001-19" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6589860/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31092506/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Rev&amp;title=Hypervirulent%20Klebsiella%20pneumoniae&amp;author=TA%20Russo&amp;author=CM%20Marr&amp;volume=32&amp;publication_year=2019&amp;pages=e00001-19&amp;pmid=31092506&amp;doi=10.1128/CMR.00001-19&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>Singh NK, Bezdan D, Checinska Sielaff A, Wheeler K, Mason CE, Venkateswaran K. 
2018. 
Multi-drug resistant <em>Enterobacter bugandensis</em> species isolated from the International Space Station and comparative genomic analyses with human pathogenic strains. BMC Microbiol
18:175. doi: 10.1186/s12866-018-1325-2.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12866-018-1325-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6251167/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30466389/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&amp;title=Multi-drug%20resistant%20Enterobacter%20bugandensis%20species%20isolated%20from%20the%20International%20Space%20Station%20and%20comparative%20genomic%20analyses%20with%20human%20pathogenic%20strains&amp;author=NK%20Singh&amp;author=D%20Bezdan&amp;author=A%20Checinska%20Sielaff&amp;author=K%20Wheeler&amp;author=CE%20Mason&amp;volume=18&amp;publication_year=2018&amp;pages=175&amp;pmid=30466389&amp;doi=10.1186/s12866-018-1325-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, Minich J, McDonald D, Mayer T, Knight R, Karouia F, Fox GE, Venkateswaran K. 
2019. 
Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome
7:50. doi: 10.1186/s40168-019-0666-x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40168-019-0666-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6452512/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30955503/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&amp;author=A%20Checinska%20Sielaff&amp;author=C%20Urbaniak&amp;author=GBM%20Mohan&amp;author=VG%20Stepanov&amp;author=Q%20Tran&amp;volume=7&amp;publication_year=2019&amp;pages=50&amp;pmid=30955503&amp;doi=10.1186/s40168-019-0666-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>Andrews S.
2010. 
FastQC: a quality control tool for high throughput sequence data. <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc" rel="noopener noreferrer" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>.</cite>
</li>
<li id="B10">
<span class="label">10.</span><cite>Chen S, Zhou Y, Chen Y, Gu J. 
2018. 
fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics
34:i884–i890. doi: 10.1093/bioinformatics/bty560.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/bty560" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6129281/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30423086/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&amp;author=S%20Chen&amp;author=Y%20Zhou&amp;author=Y%20Chen&amp;author=J%20Gu&amp;volume=34&amp;publication_year=2018&amp;pages=i884-i890&amp;pmid=30423086&amp;doi=10.1093/bioinformatics/bty560&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 
2012. 
SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol
19:455–477. doi: 10.1089/cmb.2012.0021.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/cmb.2012.0021" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3342519/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22506599/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&amp;title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&amp;author=A%20Bankevich&amp;author=S%20Nurk&amp;author=D%20Antipov&amp;author=AA%20Gurevich&amp;author=M%20Dvorkin&amp;volume=19&amp;publication_year=2012&amp;pages=455-477&amp;pmid=22506599&amp;doi=10.1089/cmb.2012.0021&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>Gurevich A, Saveliev V, Vyahhi N, Tesler G. 
2013. 
QUAST: quality assessment tool for genome assemblies. Bioinformatics
29:1072–1075. doi: 10.1093/bioinformatics/btt086.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btt086" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3624806/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23422339/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=QUAST:%20quality%20assessment%20tool%20for%20genome%20assemblies&amp;author=A%20Gurevich&amp;author=V%20Saveliev&amp;author=N%20Vyahhi&amp;author=G%20Tesler&amp;volume=29&amp;publication_year=2013&amp;pages=1072-1075&amp;pmid=23422339&amp;doi=10.1093/bioinformatics/btt086&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>Yoon S-H, Ha S, Lim J, Kwon S, Chun J. 
2017. 
A large-scale evaluation of algorithms to calculate average nucleotide identity. Antonie Van Leeuwenhoek
110:1281–1286. doi: 10.1007/s10482-017-0844-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s10482-017-0844-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28204908/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Antonie%20Van%20Leeuwenhoek&amp;title=A%20large-scale%20evaluation%20of%20algorithms%20to%20calculate%20average%20nucleotide%20identity&amp;author=S-H%20Yoon&amp;author=S%20Ha&amp;author=J%20Lim&amp;author=S%20Kwon&amp;author=J%20Chun&amp;volume=110&amp;publication_year=2017&amp;pages=1281-1286&amp;pmid=28204908&amp;doi=10.1007/s10482-017-0844-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>This whole-genome sequencing project has been deposited in GenBank, and the GenBank and raw read accession numbers are given in <a class="usa-link" href="#tab1">Table 1</a>. The BioProject accession numbers are <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA635942" rel="noopener noreferrer" target="_blank">PRJNA635942</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA640688" rel="noopener noreferrer" target="_blank">PRJNA640688</a>, and <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA640693" rel="noopener noreferrer" target="_blank">PRJNA640693</a>. Whole-genome sequencing data have also been deposited in NASA GeneLab (accession numbers <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-302" rel="noopener noreferrer" target="_blank">GLDS-302</a>, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-309" rel="noopener noreferrer" target="_blank">GLDS-309</a>, and <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-311" rel="noopener noreferrer" target="_blank">GLDS-311</a>). The versions described in this paper are the first versions.</p></section></section></section></section></article>
</main></body></html>