NPJ Microgravity

. 2016 Dec 1;2:16038. doi: [10.1038/npjmgrav.2016.38](https://doi.org/10.1038/npjmgrav.2016.38)

# Exposure of *Mycobacterium marinum* to low-shear modeled microgravity: effect on growth, the transcriptome and survival under stress

[Camille F Abshire](https://pubmed.ncbi.nlm.nih.gov/?term=%22Abshire%20CF%22%5BAuthor%5D)

### Camille F Abshire

1Department of Molecular and Cellular Physiology, Louisiana State University Health Sciences Center, Shreveport, LA, USA

Find articles by [Camille F Abshire](https://pubmed.ncbi.nlm.nih.gov/?term=%22Abshire%20CF%22%5BAuthor%5D)

1, [Kanchanjunga Prasai](https://pubmed.ncbi.nlm.nih.gov/?term=%22Prasai%20K%22%5BAuthor%5D)

### Kanchanjunga Prasai

1Department of Molecular and Cellular Physiology, Louisiana State University Health Sciences Center, Shreveport, LA, USA

Find articles by [Kanchanjunga Prasai](https://pubmed.ncbi.nlm.nih.gov/?term=%22Prasai%20K%22%5BAuthor%5D)

1, [Israel Soto](https://pubmed.ncbi.nlm.nih.gov/?term=%22Soto%20I%22%5BAuthor%5D)

### Israel Soto

1Department of Molecular and Cellular Physiology, Louisiana State University Health Sciences Center, Shreveport, LA, USA

Find articles by [Israel Soto](https://pubmed.ncbi.nlm.nih.gov/?term=%22Soto%20I%22%5BAuthor%5D)

1, [Runhua Shi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20R%22%5BAuthor%5D)

### Runhua Shi

2Department of Medicine and Feist-Weiller Cancer Center, Louisiana State University Health Sciences Center, Shreveport, LA, USA

Find articles by [Runhua Shi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20R%22%5BAuthor%5D)

2, [Monica Concha](https://pubmed.ncbi.nlm.nih.gov/?term=%22Concha%20M%22%5BAuthor%5D)

### Monica Concha

3Department of Pathology and Tulane Cancer Center, Tulane University Health Sciences Center, New Orleans, LA, USA

Find articles by [Monica Concha](https://pubmed.ncbi.nlm.nih.gov/?term=%22Concha%20M%22%5BAuthor%5D)

3, [Melody Baddoo](https://pubmed.ncbi.nlm.nih.gov/?term=%22Baddoo%20M%22%5BAuthor%5D)

### Melody Baddoo

3Department of Pathology and Tulane Cancer Center, Tulane University Health Sciences Center, New Orleans, LA, USA

Find articles by [Melody Baddoo](https://pubmed.ncbi.nlm.nih.gov/?term=%22Baddoo%20M%22%5BAuthor%5D)

3, [Erik K Flemington](https://pubmed.ncbi.nlm.nih.gov/?term=%22Flemington%20EK%22%5BAuthor%5D)

### Erik K Flemington

3Department of Pathology and Tulane Cancer Center, Tulane University Health Sciences Center, New Orleans, LA, USA

Find articles by [Erik K Flemington](https://pubmed.ncbi.nlm.nih.gov/?term=%22Flemington%20EK%22%5BAuthor%5D)

3, [Don G Ennis](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ennis%20DG%22%5BAuthor%5D)

### Don G Ennis

4Department of Biology, University of Louisiana, Lafayette, LA, USA

Find articles by [Don G Ennis](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ennis%20DG%22%5BAuthor%5D)

4, [Rona S Scott](https://pubmed.ncbi.nlm.nih.gov/?term=%22Scott%20RS%22%5BAuthor%5D)

### Rona S Scott

5Department of Microbiology and Immunology, Feist-Weiller Cancer Center, Louisiana State University Health Sciences Center, Shreveport, LA, USA

Find articles by [Rona S Scott](https://pubmed.ncbi.nlm.nih.gov/?term=%22Scott%20RS%22%5BAuthor%5D)

5, [Lynn Harrison](https://pubmed.ncbi.nlm.nih.gov/?term=%22Harrison%20L%22%5BAuthor%5D)

### Lynn Harrison

1Department of Molecular and Cellular Physiology, Louisiana State University Health Sciences Center, Shreveport, LA, USA

Find articles by [Lynn Harrison](https://pubmed.ncbi.nlm.nih.gov/?term=%22Harrison%20L%22%5BAuthor%5D)

1,\*

* Author information
* Article notes
* Copyright and License information

1Department of Molecular and Cellular Physiology, Louisiana State University Health Sciences Center, Shreveport, LA, USA

2Department of Medicine and Feist-Weiller Cancer Center, Louisiana State University Health Sciences Center, Shreveport, LA, USA

3Department of Pathology and Tulane Cancer Center, Tulane University Health Sciences Center, New Orleans, LA, USA

4Department of Biology, University of Louisiana, Lafayette, LA, USA

5Department of Microbiology and Immunology, Feist-Weiller Cancer Center, Louisiana State University Health Sciences Center, Shreveport, LA, USA

\*

(lclary@lsuhsc.edu)

CA, KP, IS and LH (guarantor) performed experiments. MC, MB and EKF performed the bioinformatic analyses on the RNA-Seq data to obtain the FPKMs for each gene and to determine the statistically significant transcript changes between LSMMG and NG samples. LH assigned the genes to functions and/or pathways and performed the GeneSifter analyses. RSS advised on the gene expression studies and DGE advised on *M. marinum* culture. RS graphed the violin plots (Figure 2) and performed the statistical analyses for the growth and survival studies. LH directed and supervised the experiments and prepared the manuscript.

Received 2016 Feb 17; Revised 2016 Sep 15; Accepted 2016 Sep 17; Collection date 2016.

Copyright © 2016 The Author(s)

This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit <http://creativecommons.org/licenses/by/4.0/>

[PMC Copyright notice](/about/copyright/)

PMCID: PMC5515531  PMID: [28725743](https://pubmed.ncbi.nlm.nih.gov/28725743/)

## Abstract

Waterborne pathogenic mycobacteria can form biofilms, and certain species can cause hard-to-treat human lung infections. Astronaut health could therefore be compromised if the spacecraft environment or water becomes contaminated with pathogenic mycobacteria. This work uses *Mycobacterium marinum* to determine the physiological changes in a pathogenic mycobacteria grown under low-shear modeled microgravity (LSMMG). *M. marinum* were grown in high aspect ratio vessels (HARVs) using a rotary cell culture system subjected to LSMMG or the control orientation (normal gravity, NG) and the cultures used to determine bacterial growth, bacterium size, transcriptome changes, and resistance to stress. Two exposure times to LSMMG and NG were examined: bacteria were grown for ~40 h (short), or 4 days followed by re-dilution and growth for ~35 h (long). *M. marinum* exposed to LSMMG transitioned from exponential phase earlier than the NG culture. They were more sensitive to hydrogen peroxide but showed no change in resistance to gamma radiation or pH 3.5. RNA-Seq detected significantly altered transcript levels for 562 and 328 genes under LSMMG after short and long exposure times, respectively. Results suggest that LSMMG induced a reduction in translation, a downregulation of metabolism, an increase in lipid degradation, and increased chaperone and mycobactin expression. Sigma factor H (*sigH*) was the only sigma factor transcript induced by LSMMG after both short and long exposure times. In summary, transcriptome studies suggest that LSMMG may simulate a nutrient-deprived environment similar to that found within macrophage during infection. SigH is also implicated in the *M. marinum* LSMMG transcriptome response.

## Introduction

Recent studies have demonstrated that the spacecraft environment can be contaminated with numerous microbial species.[1](#bib1) Samples to check for microbes are frequently taken from surfaces, air, and water on the International Space Station (ISS) and tests to identify bacterial species include sequencing of 16s rRNA and culturing viable bacteria.[1](#bib1),[2](#bib2) A variety of bacterial phyla have been identified including Actinobacteria, Bacteroidetes, Cyanobacteria, Acidobacteria, Firmicutes, Proteobacteria, SR1, Tenericutes, and TM7 and a number of genera have been identified within each phylum.[1](#bib1) The bacteria most frequently found on the ISS include species of *Staphylococcus*, *Bacillus*, and *Micrococcus.*[2](#bib2) Experiments to determine the effect of spaceflight or low-shear modeled microgravity (LSMMG) on bacteria have determined that different phenotypes are induced in different types of bacteria. Compared with normal gravity controls alterations to growth, culture density, biofilm formation, aggregation, virulence, and changes in resistance to stress from acid, oxidative stress, heat shock, or antibiotics have been identified.[2–10](#bib2) It is not possible to generalize or predict how a bacterial species will react to microgravity or LSMMG. An example is resistance to oxidative stress: exposure to LSMMG increased oxidative stress resistance of *Pseudomonas aeruginosa*[11](#bib11) but decreased oxidative stress resistance of *Staphylococcus aureus.*[12](#bib12) Changes in the bacterial transcriptome[11–14](#bib11) are also induced by microgravity and LSMMG, and changes to the bacterial phenotype are likely due to alterations in the bacterial transcriptome. The diverse phenotypes induced by spaceflight or LSMMG[2–7](#bib2) highlights the need to understand more about how different species of bacteria are altered by microgravity and the importance of identifying key molecules that promote survival under the stress of microgravity.

*Mycobacterium marinum* is a member of a family of closely related mycobacteria, most of which are pathogenic. They include *Mycobacterium tuberculosis*, *Mycobacterium avium,* and *Mycobacterium ulcerans.*[15](#bib15) Although *M. tuberculosis*, the cause of human tuberculosis, live and replicate only in a host in nature,[15](#bib15),[16](#bib16)
*M. marinum* and *M. avium* grow in macrophage and aquatic environments.[16](#bib16)
*M. marinum* grows at 28–35 °C and infects poikilothermic hosts such as fish and frogs, but can infect humans resulting in granulomatous skin lesions, which pathologically are identical to lesions produced by dermal *M. tuberculosis*. The pathogenic *M. avium* complex, which includes the subspecies *avium*, *paratuberculosis,* and *intracellulare*, has been linked with hard-to-treat non-tuberculosis human pulmonary infections in non-immunocompromised[17](#bib17) as well as immunocompromised individuals.[18](#bib18)

*M. marinum*[19](#bib19) and *M. avium* form biofilms and can grow in biofilms in water distribution systems,[20](#bib20) taps, and showerheads. Thirty percent of biofilms tested from showerheads across the USA contained *M. avium* even though they were linked to municipal water supplies treated with chlorine.[21](#bib21)
*M. avium* grown in a biofilm is more resistant to chlorine, even when the biofilm is disrupted.[22](#bib22) Pathogenic waterborne mycobacteria could pose a potential threat during space travel not only to astronaut health, but also to the spacecraft environment and potable water. In fact, *M. avium* was identified in water samples taken from the Russian Mir space station.[23](#bib23)

Pathogenic bacteria that have the potential to have increased antimicrobial resistance and virulence are a concern inside spacecraft because astronauts already have a compromised immune system, especially when in space for long periods of time.[24](#bib24),[25](#bib25) Uncovering the signaling pathways that result in the altered physiology of bacteria has the potential to reveal new targets for drug development or disinfection that will be essential for long space flights.

There are no published studies on the effect of microgravity or LSMMG on waterborne mycobacteria. Hfq has been identified as a factor regulating microgravity gene expression in *P. aeruginosa,*[13](#bib13)
*S. aureus*,[12](#bib12) and *Salmonella enteric* serovar Typhimurium[14](#bib14) (*S. typhimurium*), but mycobacteria do not encode Hfq.[26](#bib26)
*M. marinum* is closely related genetically to *M. tuberculosis* and *M. avium* and has similar mechanisms of infection and survival inside host macrophage. Although all of these mycobacterium species can grow in specialized medium in the laboratory,[27](#bib27) growth of *M. marinum* even at the lower temperature of 30–32 °C is faster than *M. tuberculosis*[27](#bib27) and *M. avium.*[28](#bib28)
*M. marinum* is also a lower risk (bisoafety level 2) to researchers. Hence *M. marinum* is a good model for pathogenic mycobacteria.

The aim of this study was to identify factors involved in the adaptation of *M. marinum* to the LSMMG stressful environment and determine how LSMMG alters mycobacterium physiology. Two strains of *M. marinum* (1218R and LHM4) were used for this study. Strain 1218R was originally isolated from fish and LHM4 was generated from 1218R by integrating a plasmid into the genome that encodes for kanamycin resistance and the red fluorescent protein. Previous work studying the effect of LSMMG on bacteria with doubling times of <1 h exposed the bacteria to LSMMG for 10,[29](#bib29) 20,[30](#bib30) 24 h[11](#bib11) or 10 generations.[31](#bib31)
*M. marinum* has a doubling time of 6–7 h at 30 °C and hence two exposure times to LSMMG or NG (short ~40 h and long 4 days and ~35 h) were selected to allow the bacteria to undergo multiple replication cycles during the experiments. The exposure protocol was designed so that short and long LSMMG- or NG-exposed cultures were in exponential phase and hence alterations to the transcriptome or resistance to treatment with hydrogen peroxide (H2O2), acid or gamma radiation were due to different exposure times to LSMMG and not due to different phases of growth.

In summary, this work characterizes the growth, stress survival, and transcriptome of *M. marinum* grown under LSMMG and identifies nutrient deprivation as a signal involved in changes to gene expression and the sigma factor SigH as a factor modulating the LSMMG transcriptome in a waterborne mycobacterium.

## Results

The effect of LSMMG exposure time had to be assessed with bacteria in the same phase of growth, as we found stress resistance altered as the *M. marinum* progressed from early to late exponential phase (data not shown) and it would be expected that the transcriptome of replicating bacteria is very different from bacteria in stationary phase. For the short exposure, *M. marinum* were diluted to an optical density at 600 nm (OD600) of 0.002 at the initiation of the high aspect ratio vessel (HARV) culture to make sure that the bacteria would be in exponential phase (OD600~0.1–0.2) after ~40 h. This exposure time allowed ~6 generations. For the longer exposure time, *M. marinum* were subjected to 4 days of growth under LSMMG or NG, the bacteria removed from the HARV, re-diluted in fresh medium and the LSMMG or NG exposure continued in new HARVs for a further ~35 h (see Materials and Methods for full details).

### LSMMG did not alter aggregation of *M. marinum*

Previous published studies observed increased aggregation of bacteria exposed to microgravity or LSMMG.[4](#bib4),[12](#bib12),[14](#bib14) Mycobacterium under normal laboratory conditions grow as aggregates and standard culture procedure[27](#bib27),[32](#bib32) uses Tween 80 in the medium to minimize clumping. Even in the presence of 0.2% Tween 80, *M. marinum* still aggregate. Cultures exposed to NG or LSMMG for various times were examined by microscopy to determine whether exposure to LSMMG altered aggregation. Similar size aggregates were detected in both LSMMG and NG cultures ([Supplementary Figure 1A](#xob2)), so growth under LSMMG and standard culture medium did not increase clumping of *M. marinum*. To determine whether the Tween 80 in the culture medium prevented the detection of structure alterations under LSMMG, 1218R and LHM4 were grown initially in a tube with 0.2% Tween 80 until OD600 ~1 in the rotating incubator as described in the Materials and Methods section. The cultures were then diluted to OD600 0.002 in growth medium without Tween 80. One dilution for each strain was used to prepare two 50 ml HARVs and a tube culture. The cultures in the tube and HARVs were grown for 67 h (LHM4) or 69 h (1218R). The tube was incubated in the rotating incubator at 120 r.p.m. and the HARVs were grown under NG or LSMMG. The culture in the tube grew as small clumps (data not shown) and clumps were also visible in the HARV cultures. When the culture was removed from the HARVs it was not possible to determine the optical density of the culture as bacteria had adhered to the base of the HARV grown under NG, preventing an accurate measurement. Samples of each culture were examined by microscopy ([Supplementary Figure 1B and C](#xob2)) and cording of the mycobacterium was seen in all cultures. This cording structure is known to form in broth as well as biofilms.[19](#bib19),[33](#bib33) Under the conditions examined, there was no obvious difference between the NG and LSMMG cultures. Further studies are required to determine whether there are subtle structure differences between the cords or whether cord formation is accelerated under LSMMG. As growth in the absence of Tween 80 prevents accurate measurements of culture density and total recovery of the culture from the HARVs, all further studies were performed in the presence of 0.2% Tween 80.

### *M. marinum* grown under LSMMG transition from exponential phase earlier than NG-exposed *M. marinum*

Thirteen independent cultures of LHM4 and three independent cultures of 1218R were grown continuously for 4 days in 10 ml HARVs under LSMMG or NG and significantly different OD600 were obtained for LSMMG compared with NG cultures ([Figure 1a](#fig1)). The OD600 of the LSMMG culture was reduced by at least 50% compared with the NG culture. To determine in more detail how growth was altered by LSMMG, growth studies of LHM4 ([Figure 1b](#fig1)) and 1218R ([Figure 1c](#fig1)) were performed over a period of 150 h where the OD600 was monitored by removal of a sample of the culture at the different time points. Experiments were performed in triplicate and all data points from these experiments were pooled and plotted to examine the effect of LSMMG on growth ([Figure 1b and c](#fig1)). The LSMMG cultures of LHM4 and 1218R had slightly lower OD600 compared with the NG culture at ~37 h after culture initiation and this difference increased with time ([Figure 1e and f](#fig1)). This was very consistent and detected in all experiments with both *M. marinum* strains. A similar result was found when LHM4 colony forming units per ml (CFU/ ml) was plotted against time ([Supplementary Figure 2](#xob2)). As the OD600 measurements for the NG cultures began to plateau at around 90 h, indicating an exit from logarithmic growth, the data between 37 and 70 h were considered ([Figure 1e and f](#fig1)). Regression analyses determined that during this time period that the data were significantly different (*P*<0.0001) for the LSMMG and NG cultures of LHM4 and 1218R. To examine the extent of change on growth during this transition from exponential phase, the doubling times during this time period were calculated from [Figure 1e and f](#fig1): doubling times were 10 h (LHM4) and 11 h (1218R) for NG cultures, and 15.5 h (LHM4) and 16 h (1218R) for LSMMG cultures. The LSMMG culture did eventually reach a similar culture density to the NG culture at later time points ([Figure 1b](#fig1)).

#### Figure 1.

[![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e09/5515531/9cc26d6cf4b4/npjmgrav201638-f1.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5515531_npjmgrav201638-f1.jpg)

[Open in a new tab](figure/fig1/)

Growth under LSMMG alters culture density of *M. marinum*. LHM4 and 1218R was grown in 10 ml HARVs under NG or LSMMG for 4 days continuously and the OD600 determined. The data from 13 HARVs for LHM4 and 3 HARVs for 1218R for each condition was used to obtain the average and s.d. (**a**). OD600 was monitored for LHM4 (**b**) and 1218R (**c**) in 50 ml HARVs. At each time point, a small culture volume (1–2 ml) was removed to measure OD600 and the volume replaced with fresh medium (2–4% total volume). Strain 1218R was also grown in medium with catalase (**d**). Three independent experiments were performed for each strain and growth condition. As data were not gathered at the exact time point in each experiment, all the data points were pooled for a strain and condition and are shown graphically (**b**–**d**). A transition from exponential phase was detected at ~37 h in LSMMG cultures compared with NG cultures. The data between 37 and 70 h for the three experiments were plotted (**e**, **f**) and regression analysis performed, demonstrating that the increase in OD600 during this time period was statistically different when the bacteria were grown under NG and LSMMG (*P*<0.0001).

As OD600 can be influenced by size of the bacteria, the diameter and length of 50 bacteria in an LHM4 culture grown for 4 days under LSMMG (OD600=1.3) or NG (OD600=2.3) were determined. No statistical significant difference was found for the diameter (NG=0.72±0.05 μm, LSMMG=0.73±0.07 μm) or the length (NG=2.7±0.45 μm, LSMMG=2.7±0.5 μm) between the two cultures.

Published studies have determined that LSMMG induces oxidative stress genes in bacteria;[30](#bib30) suggesting LSMMG disturbs the normal redox environment. Catalase breaks down H2O2 and is present in standard *M. tuberculosis* medium (3 μg/ml). Growth experiments for 1218R were therefore performed with medium containing catalase (3 μg/ml) to remove extracellular H2O2. No change in growth was detected when the medium contained catalase, which suggests extracellular H2O2 did not hinder growth under LSMMG ([Figure 1d](#fig1)).

### *M. marinum* transcriptome response to LSMMG

RNA samples isolated from triplicate independent cultures of 1218R grown under LSMMG or NG cultures for the short or long exposure times were used for RNA-Seq analyses, and 6–10 million reads were sequenced for each biological sample ([Supplementary Table 1](#xob1)). RNA-Seq using 5–10 million non-rRNA reads has previously been found to provide sufficient depth of sequencing to detect the majority of alterations in bacterial gene expression.[34](#bib34) Following transcript quantification a comparison of LSMMG with NG after short or long exposure times ([Supplementary Excel Worksheets 1](#xob3) and [2](#xob4)) identified 562 and 328 genes, respectively, with significantly altered transcript levels. From the 10 categories of mycobacterial genes (<http://mycobrowser.epfl.ch/marinolist.html>), the intermediary metabolism and respiration category had the greatest number of genes with significant transcript changes ([Supplementary Table 2](#xob1)).

To determine the genes with the greatest physiological relevance to growth under LSMMG the real fold change (RFC) was calculated ([Supplementary Excel Worksheets](#xob3)). Reverse-transcriptase quantitative PCR (RT-qPCR) was also performed for 9 different genes and the majority of the LSMMG/NG fold changes in expression determined by RNA-Seq and RT-qPCR were similar ([Supplementary Table 3](#xob1)). The genes with the greatest RNA-Seq RFC (top 10%) after short and long exposure times are shown in [Supplementary Tables 4 and 5](#xob1).

Genes that changed expression ⩾1.5-fold (RFC⩽0.67 or ⩾1.5) by RNA-Seq were identified and the log10 RFC plotted for the genes according to category ([Figure 2](#fig2)). The category with the greatest number of genes for LSMMG versus NG at each exposure time ([Figure 2a and b](#fig2)) was intermediary metabolism, and there was an even distribution of genes with increased and decreased expression. The categories of genes relating to the cell wall and the conserved hypotheticals had the next highest number of genes that significantly changed. Even though these categories also had an even distribution of genes with increased and decreased expression ([Figure 2a and b](#fig2)), some genes relating to these two categories had the highest RFC, especially after a short exposure to LSMMG ([Supplementary Tables 4 and 5](#xob1)). Unfortunately the function of the conserved hypothetical genes is unknown. The majority of genes involved in the information pathway and lipid metabolism categories decreased expression, whereas the genes in the PE/PPE and virulence categories increased expression ([Figure 2a and b](#fig2)). There were also genes in the regulatory protein category that were significantly changed by LSMMG, but genes were again evenly distributed for increased and decreased expression. When the data from the LSMMG samples for short versus long exposure times were statistically compared, it was evident that gene expression continued to change with greater exposure to LSMMG ([Figure 2c](#fig2)), and the categories for cell wall, intermediary metabolism, PE/PPE, and virulence increased expression.

#### Figure 2.

[![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e09/5515531/fbfd4df6ff63/npjmgrav201638-f2.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5515531_npjmgrav201638-f2.jpg)

[Open in a new tab](figure/fig2/)

Gene expression changes identified by RNA-Seq. Transcriptional profiles were compared between 1218R grown at LSMMG and NG for 40 h (**a**, LSMMG/NG) or 4 days ~35 h (**b**, LSMMG/NG) and between 1218R grown under LSMMG for the two time periods (**c**, 4 days ~35 h/40 h). The significant (*P*<0.05) changes in transcript levels were identified. The real fold change (RFC) is the ratio of the normalized mean count values for LSMMG divided by the normalized mean count values for NG. The genes that changed ⩾1.5-fold were assigned to categories (1–9) and the distribution of the RFC is shown graphically. No genes were significantly altered ⩾1.5-fold in the insertion sequences and phage category and so this category is not included. The mean value for the RFC for a category is marked by the black spot and the number of genes in the category is shown in parentheses.

To identify genes involved in adaptation to LSMMG, genes were considered that significantly changed ⩾1.5-fold at both the short and long exposure time points ([Table 1](#tbl1)). The changes for the majority of these 97 genes were very similar at the two time points. Transcript changes include a ~2-fold decrease for ribosomal proteins and a decrease for most of the genes in categories related to lipid metabolism and intermediary metabolism ([Table 1](#tbl1)). The greatest changes were in genes encoding chaperone proteins.

#### Table 1. *M. marinum* genes that significantly altered expression when grown under LSMMG for both 40 h, and 4 days ~35 h (LSMMG/NG).

| *Gene name* | *Function* | *RFC at 40 h* | *RFC at 4 days ~35 h* |
| --- | --- | --- | --- |
| *ltp1\_1* | Cell wall and cell processes | 2.2 | 1.9 |
| *mmpL3* |  | 0.63 | 0.67 |
| *MMAR\_0493-1* |  | 1.5 | 1.6 |
| *MMAR\_1554-1* |  | 1.7 | 1.5 |
| *MMAR\_2268-1* |  | 0.46 | 0.50 |
| *cydD* |  | 2.2 | 1.9 |
| *MMAR\_3568-1* |  | 3.4 | 2.4 |
| *MMAR\_3658-1* |  | 8.1 | 9.1 |
| *lsr2* |  | 0.48 | 0.57 |
| *fbpA* |  | 0.54 | 0.63 |
|  |  |  |  |
| *MMAR\_0119-1* | Conserved hypotheticals | 1.9 | 1.7 |
| *MMAR\_0519-1* |  | 0.39 | 0.35 |
| *MMAR\_0853-1* |  | 9.1 | 9.0 |
| *MMAR\_0982-1* |  | 0.46 | 0.51 |
| *MMAR\_2771-1* |  | 1.7 | 2.2 |
| *MMAR\_3007-1* |  | 1.7 | 1.6 |
| *MMAR\_3010-1* |  | 1.8 | 1.9 |
| *MMAR\_3070-1* |  | 2.2 | 1.8 |
| *MMAR\_3088-1* |  | 1.8 | 1.5 |
| *MMAR\_3549-1* |  | 0.24 | 0.29 |
| *MMAR\_4248-1* |  | 3.8 | 2.2 |
| *MMAR\_4306-1* |  | 8.9 | 4.9 |
| *MMAR\_4498-1* |  | 3.5 | 5.1 |
| *MMAR\_4609-1* |  | 2.1 | 2.4 |
| *MMAR\_4647-1* |  | 3.0 | 1.5 |
| *MMAR\_5103-1* |  | 0.39 | 0.42 |
| *MMAR\_5106-1* |  | 0.43 | 0.58 |
| *MMAR\_5437-1* |  | 1.5 | 1.8 |
| *MMAR\_5455-1* |  | 1.8 | 1.5 |
|  |  |  |  |
| *rpsR1* | Information pathway | 0.55 | 0.57 |
| *rplA* |  | 0.58 | 0.65 |
| *rplJ* |  | 0.48 | 0.57 |
| *rplC* |  | 0.49 | 0.55 |
| *rplB* |  | 0.52 | 0.63 |
| *rplV* |  | 0.46 | 0.63 |
| *rpmC* |  | 0.61 | 0.64 |
| *rpsN* |  | 0.67 | 0.62 |
| *rpsE* |  | 0.51 | 0.63 |
| *infA* |  | 0.64 | 0.66 |
| *rpsD* |  | 0.65 | 0.64 |
| *rpoA* |  | 0.50 | 0.59 |
| *rplQ* |  | 0.52 | 0.57 |
| *rplM* |  | 0.61 | 0.60 |
| *rpsI* |  | 0.63 | 0.60 |
| *sigH* |  | 1.5 | 1.5 |
| *nrdE* |  | 1.8 | 2.2 |
| *hupB* |  | 0.42 | 0.66 |
| *rpsP* |  | 0.66 | 0.66 |
| *tsf* |  | 0.61 | 0.66 |
| *rplY* |  | 0.55 | 0.64 |
| *rnpA* |  | 0.57 | 0.58 |
| *celA* | Intermediary metabolism | 0.53 | 0.47 |
| *nirB* | and respiration | 0.39 | 0.48 |
| *hemC* |  | 0.48 | 0.48 |
| *mrsA* |  | 0.60 | 0.62 |
| *sdhB* |  | 0.59 | 0.81 |
| *sdhD* |  | 0.55 | 0.67 |
| *MMAR\_1644-1* |  | 1.8 | 1.5 |
| *amiC* |  | 0.63 | 0.64 |
| *csd* |  | 0.49 | 0.50 |
| *MMAR\_2557-1* |  | 1.7 | 1.9 |
| *MMAR\_2844-1* |  | 0.46 | 0.58 |
| *folE\_1* |  | 1.6 | 1.9 |
| *glnA2* |  | 0.57 | 0.62 |
| *MMAR\_3558-1* | intermediary metabolism and respiration | 0.23 | 0.07 |
| *mobA* |  | 0.33 | 0.52 |
| *porB* |  | 0.33 | 0.53 |
| *porA* |  | 0.31 | 0.46 |
| *putA* |  | 4.0 | 2.6 |
| *pth* |  | 0.49 | 0.61 |
| *MMAR\_4497-1* |  | 3.3 | 5.0 |
| *panD* |  | 0.47 | 0.54 |
| *panC* |  | 0.56 | 0.58 |
|  |  |  |  |
| *desA3\_2* | Lipid metabolism | 3.9 | 3.6 |
| *desA3* |  | 2.2 | 1.8 |
| *fadE23* |  | 1.6 | 1.9 |
| *pks15/1* |  | 0.53 | 0.66 |
| *fabG3\_1* |  | 2.1 | 1.7 |
| *fabD* |  | 0.32 | 0.48 |
| *kasA* |  | 0.28 | 0.37 |
| *kasB* |  | 0.28 | 0.37 |
| *accD6* |  | 0.29 | 0.36 |
| *accA2* |  | 0.63 | 0.58 |
| *fadB* |  | 0.57 | 0.59 |
| *pks13* |  | 0.38 | 0.57 |
| *fadD32* |  | 0.33 | 0.55 |
|  |  |  |  |
| *MMAR\_0641-1* | PE/PPE | 2.5 | 2.8 |
| *MMAR\_4562-1* |  | 1.9 | 2.3 |
| *MMAR\_4899-1* |  | 2.6 | 2.0 |
|  |  |  |  |
| *hspR* | Regulatory protein | 2.6 | 4.4 |
| *whiB3* |  | 0.16 | 0.38 |
| *whiB7* |  | 0.65 | 2.6 |
| *MMAR\_2281-1* |  | 0.49 | 0.60 |
| *MMAR\_3703-1* |  | 3.1 | 2.7 |
| *MMAR\_5069-1* |  | 0.64 | 0.46 |
|  |  |  |  |
| *hsp* | Virulence, adaptation | 5.7 | 40.8 |
| *dnaK* | detoxification | 3.5 | 5.38 |

[Open in a new tab](table/tbl1/)

Only genes that altered significantly by ⩾1.5-fold at both time points are shown. RFC is the ratio of the normalized mean count values for LSMMG over the normalized mean count values for NG.

### Expression of cell metabolism genes

Groups of genes linked with specific metabolic pathways were examined using hierarchical clustering in GeneSifter to determine whether gene expression changes were linked to growth under LSMMG. Hierarchical clustering of all the LSMMG samples suggests changes in gene expression were linked to LSMMG. Genes involved in glycolysis (12 genes), electron transport chain (ETC, 87 genes), pentose phosphate pathway (7 genes), gluconeogenesis (15 genes), methylcitrate cycle (10 genes), glyoxylate cycle (14 genes), and citric acid cycle (TCA, 19 genes) were examined. The majority of these pathways had few significant changes or changes not related to LSMMG (data not shown). An example is ETC, where 22 genes had significant alterations but the changes were not linked to growth under LSMMG ([Supplementary Figure 3A](#xob2)). However, 17/19 TCA genes had altered expression linked to LSMMG ([Supplementary Figure 3B](#xob2)). Transcript levels for aconitase decreased after short exposure to LSMMG. Succinate dehydrogenase is also a TCA enzyme, but also forms complex II of the ETC.[35](#bib35),[36](#bib36) There are two forms of succinate dehydrogenase (Sdh1 and Sdh2) and transcript levels of 3 (*sdhB-D*) of the four genes encoding Sdh2 decreased at the short exposure time point, while transcripts from *sdhB* and *sdhD* decreased after long exposure. There was no change in expression of the genes encoding Sdh1. Proline oxidation can also donate electrons to ETC.[36](#bib36) Transcripts from *MMAR\_4252* (*putA,* proline dehydrogenase) and *MMAR\_4253* (pyrroline-5-carboxylate dehydrogenase) were increased 2–4-fold by LSMMG after the short exposure time ([Supplementary Worksheets 1 and 2](#xob1)). After the long exposure time, only *MMAR\_4252* was increased 2.6-fold ([Table 1](#tbl1)). Isocitrate lyase (Icl) is an enzyme that connects the TCA and glyoxylate cycle and transcript levels from *icl* (*MMAR\_0792*) decreased by ~10-fold in LSMMG bacteria after the long exposure time ([Supplementary Table 5](#xob1)).

Transcription of genes involved in lipid degradation and catabolism of fatty acids were altered significantly in LSMMG bacteria. We identified 104 genes in *M. marinum* involved in lipid degradation and 53 had ⩾1.5-fold significant change in transcript levels after growth under LSMMG ([Figure 3a](#fig3)). Of the 53 genes, 16 had decreased expression and 37 had increased expression, the majority of which encode enzymes involved in fatty acid β oxidation. Fatty acids are also used for polyketide lipid synthesis of phenolic glycolipids and phthiocerol dimycocerosates, mycolic acid (MA) synthesis, production of cell membrane phospholipids, as well as production of carbon stores in the form of triacylglycerol.[37](#bib37) Although there were no alterations in expression of genes encoding triacylglycerol synthase (*MMAR\_2634, MMAR\_2598, MMAR\_1519 and MMAR\_1157*), genes involved in polyketide lipid synthesis (*fadD22* and *pks15/1*) were decreased 1.5- to 2-fold ([Supplementary Excel Worksheets 1](#xob3) and [2](#xob4)) and multiple genes[38](#bib38) encoding proteins involved in MA synthesis were also decreased ([Table 1](#tbl1) and [Figure 3b](#fig3)). Two genes (*desA3* and *desA3\_2*) encoding membrane-bound desaturases showed increased expression at both time points in LSMMG bacteria ([Table 1](#tbl1) and [Figure 3b](#fig3)). These desaturases are involved in oleic acid production and potentially the production of cis double bonds in meromycolic acids.[38](#bib38)

#### Figure 3.

[![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e09/5515531/63591f0ffe51/npjmgrav201638-f3.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5515531_npjmgrav201638-f3.jpg)

[Open in a new tab](figure/fig3/)

Growth under LSMMG increases expression of genes involved in lipid degradation and mycobactin synthesis but decreases expression of genes involved in mycolic acid synthesis. Genes involved in the designated categories lipid degradation (**a**), mycolic acid synthesis (**b**) and mycobactin synthesis (**c**) were identified using the mycobrowser website (<http://mycobrowser.epfl.ch/>) and the literature.[38](#bib38) The FPKMs (fragments per kilobase of transcript per million reads) for each gene in the 12 RNA samples were analyzed using Genesifter. Only genes with ⩾1.5-fold change in expression (*P*<0.05) were considered. The data were log transformed. The heat maps were generated showing expression in short (growth for ~40 h) and long (growth for 4 days ~35 h) exposure samples. Genes colored brown are discussed in the manuscript text.

Mycobactins chelate iron from the environment and are also known to be important for mycobacterial growth during infection of macrophage.[39](#bib39) Sixteen genes involved in mycobactin synthesis were identified and six increased expression after LSMMG exposure for 4 days and ~35 h ([Figure 3c](#fig3)).

### SigH: a possible regulator of the mycobacteria LSMMG response

*sigH* transcript levels increased 1.5-fold at both time points ([Table 1](#tbl1)). SigH is known to regulate chaperone encoding genes[40](#bib40),[41](#bib41) (*hsp* and the *dnaK*, *dnaJ*, *grpE* and *hspR* operon) and *hsp* transcript levels increased at both time points ([Table 1](#tbl1)). Transcripts from all the *dnaK* operon genes increased after ~40 h of LSMMG exposure ([Supplementary Table 4 and Excel Worksheet 1](#xob1)), while only transcripts from *dnaK* and *hspR* increased after LSMMG exposure for 4 days and ~35 h ([Supplementary Table 5 and Excel Worksheet 2](#xob1)). *M. tuberculosis* SigH directly/indirectly regulated genes have been identified in a study[41](#bib41) where wild-type and *sigH* mutant *M. tuberculosis* were allowed to recover after diamide treatment: RNA analysis identified 44 genes at 0 min and 91 genes at 30 min of recovery as SigH-regulated genes. We identified the *M. marinum* orthologs of these genes (40 at 0 min and 78 at 30 min) and using GeneSifter determined that >50% of these orthologs altered expression significantly after growth under LSMMG ([Figure 4](#fig4)).

#### Figure 4.

[![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e09/5515531/355ff2e57f24/npjmgrav201638-f4.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5515531_npjmgrav201638-f4.jpg)

[Open in a new tab](figure/fig4/)

Changes in expression of SigH-regulated genes show a linkage to growth under LSMMG. Mehr *et al.*[41](#bib41) treated wild-type and *sigH* mutant *M. tuberculosis* with diamide and determined SigH-related gene expression changes after 0 and 30 min recovery time. Peak *sigH* transcript was at 30 min. *M. marinum* orthlogs of these genes were identified (<http://mycobrowser.epfl.ch/>). The FPKMs (fragments per kilobase of transcript per million reads) for each gene in the 12 RNA samples were analyzed for hierarchical clustering using GeneSifter with a threshold of 1.5-fold change and a statistical difference (*P*<0.05) determined using an ANOVA. The data were log transformed. The heat maps show expression changes during LSMMG and NG for the orthologs of the *M. tuberculosis* genes that changed after 0 (**a**) and 30 min (**b**). Genes colored brown are discussed in the text.

### Changes in expression of genes related to the enduring hypoxic response and the nutrient starvation response

The enduring hypoxic response (EHR)[42](#bib42) and the *relA* nutrient starvation response[43](#bib43) are well characterized in *M. tuberculosis*. We identified 147 *M. marinum* orthologs from the 230 *M. tuberculosis* genes involved in EHR, and 105 *M. marinum* orthologs from the 147 *M. tuberculosis* genes involved in the nutrient starvation response. These orthologs (58 for EHR and 46 for nutrient starvation response) were also found to significantly alter expression⩾1.5-fold due to LSMMG ([Figure 5](#fig5)). The EHR genes that overlap with the LSMMG changes in *M. marinum* ([Figure 5a](#fig5)) include *sigH* and SigH-regulated genes. For *M. tuberculosis*, all the genes in [Figure 5a](#fig5) increase expression during EHR, whereas 25 increased transcript levels under LSMMG. A major group of overlapping genes between the nutrient starvation response in *M. tuberculosis* and the LSMMG response in *M. marinum* are related to translation. The nutrient starvation response includes 23 translation-related genes and 22 decrease expression during the response. There are 21 corresponding *M. marinum* orthologs, 17 changed expression in *M. marinum* under LSMMG and 16 decreased expression ([Figure 5b](#fig5) and [Table 1](#tbl1)).

#### Figure 5.

[![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e09/5515531/e29e2991c300/npjmgrav201638-f5.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5515531_npjmgrav201638-f5.jpg)

[Open in a new tab](figure/fig5/)

The effect of LSMMG on genes involved in the enduring hypoxic response (EHR) or nutrient starvation response. Genes involved in these two stress responses have been identified in *M. tuberculosis.*[42](#bib42),[43](#bib43) The *M. marinum* orthologs were identified (http://mycobrowser.epfl.ch/) for the EHR (**a**) and *relA* nutrient starvation (**b**) response[43](#bib43) (Supplemental Data Set 1 and section 3 in ref. [43](#bib43)). The FPKMs (fragments per kilobase of transcript per million reads) for each gene in the 12 RNA samples were analyzed for hierarchical clustering using GeneSifter with a threshold of 1.5-fold change and a statistical difference (*P*<0.05) determined using an ANOVA analysis. The data were log transformed. Genes colored brown are discussed in the text.

### Comparison of *M. marinum* exposed to LSMMG and *P. aeruginosa* exposed to spaceflight or LSMMG

During spaceflight, 167 genes were differentially regulated in *P. aeruginosa* and Hfq was identified as a transcription regulator of the bacterial response to microgravity[13](#bib13) and LSMMG.[11](#bib11) Mycobacteria do not have a gene encoding Hfq, but there is overlap in the lists of genes that are differentially regulated in *P. aeruginosa* exposed to spaceflight or LSMMG and *M. marinum* subjected to LSMMG ([Supplementary Tables 6 and 7](#xob1)). A hypergeometric distribution test was performed to determine whether the number of genes found to overlap were significant or could have occurred by chance. The number of protein coding sequences in *M. marinum* is 5,452 (refs [15](#bib15),[44](#bib44)) and from the short exposure to LSMMG there were 562 significant changes, of which 38 overlapped with *P. aeruginosa* exposed to spaceflight (where 167 changes were detected) and 47 overlapped with *P. aeruginosa* exposed to LSMMG (where 330 changes were detected). From the long exposure of *M. marinum* to LSMMG, there were 328 significant changes, of which 38 overlapped with *P. aeruginosa* exposed to spaceflight and 34 overlapped with *P. aeruginosa* exposed to LSMMG. The number of overlapping genes in each case was found to be significant (short exposure *M. marinum* and spaceflight *P. aeruginosa* representation factor 2.2, *P*<1.513e−6, short exposure *M. marinum* and LSMMG *P. aeruginosa* representation factor 1.4, *P*<0.012, long exposure *M. marinum* and spaceflight *P. aeruginosa* representation factor 3.8, *P*<2.17e−13, long exposure *M. marinum* and LSMMG *P. aeruginosa* representation factor 1.7, *P*<0.001).

When comparing the LSMMG *M. marinum* data with the spaceflight data from *P. aeruginosa*, the largest overlap in downregulated genes encode ribosomal proteins and other common downregulated genes produce proteins involved in transcription and translation. Genes encoding the metabolic enzymes Icl and Sdh2 are also downregulated in both organisms, suggesting similar downregulation of metabolic pathways. Interestingly there were no genes with decreased transcript levels detected for *P. aeruginosa* grown under LSMMG, but 330 transcript levels increased by ⩾1.5-fold.[11](#bib11) From this list a number of transcripts encoding proteins of similar function were increased in *P. aeruginosa* subjected to LSMMG but decreased in LSMMG-grown *M. marinum* ([Supplementary Table 7](#xob1)). These include transcripts for 13 ribosomal proteins, Sdh2 (*sdhB* and *sdhD*), Icl, RNA polymerase alpha subunit (*rpoA*) and elongation factor Ts (*tsf*). A common group of genes with increased transcript levels in LSMMG *M. marinum* and *P. aeruginosa* were the stress response genes for heat shock and chaperone proteins.

Transcriptome data therefore suggest that *M. marinum* grown under LSMMG exhibit similar stress responses as LSMMG-grown *P. aeruginosa*, but have transcriptional, translational, and metabolic alterations similar to spaceflight-exposed *P. aeruginosa*.

### Survival of LHM4 NG- and LSMMG-exposed cultures to stressful conditions

Ionizing radiation and H2O2 produce an oxidizing environment in cells. Compared with NG, LSMMG-exposed bacteria showed no change in resistance to ionizing radiation ([Figure 6a](#fig6)), but were more sensitive to H2O2 ([Figure 6b](#fig6)) with the sensitivity increasing with longer exposure to LSMMG. LHM4 exposed to NG and LSMMG for 4 days and ~35 h were also tested for sensitivity to pH 3.5 for up to 4 h and no cell death occurred ([Figure 6c](#fig6)).

#### Figure 6.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e09/5515531/c3416ce7ea34/npjmgrav201638-f6.jpg)

[Open in a new tab](figure/fig6/)

LHM4 grown under LSMMG are more sensitive to H2O2 but not gamma radiation or acid in comparison with NG cultures. Bacteria were grown in HARVs under LSMMG or NG for ~40 h (short exposure time) or 4 days ~35 h (long exposure time). Cultures were removed and treated with gamma radiation using a 137Cesium source (**a**), H2O2 (**b**), or growth medium at pH 3.5 (**c**). Bacteria were grown on solid medium and colonies counted. The percent survival was calculated by dividing the colony number/ml by that obtained for a culture treated with no dose at the same time. At least three experiments were performed for each culture and treatment type, except for the short exposure time LSMMG and NG cultures treated with H2O2 where two experiments were performed. The average and s.d. are shown. Survival was compared between the LSMMG and NG samples using the area under the curve and the only statistically significant difference was between the long exposure LSMMG culture and the long exposure NG culture treated with H2O2 (\**P*=0.004).

## Discussion

Previous studies examining liquid bacterial growth during space travel[3](#bib3) or LSMMG[2](#bib2),[3](#bib3) have found species-specific alterations: LSMMG-grown *S. typhimurium* had decreased doubling time in minimal medium,[29](#bib29)
*Escherichia coli* and *Bacillus subtilis* reached higher cell densities and had increased growth rate during spaceflight compared with ground controls,[45](#bib45) while *S. aureus* achieved higher cell densities in the normal gravity control compared with the LSMMG culture.[12](#bib12) In our study, in a 4-day continuous growth culture, *M. marinum* subjected to NG achieved a higher cell density than the LSMMG culture. As the size of the bacteria did not change and aggregates were disrupted before optical density measurements were determined, the lower LSMMG cell density was likely due to an alteration in growth. A more detailed examination of the growth indicated an early transition of the LSMMG bacteria from exponential phase resulting in a slowing of growth. Even though RNA-Seq analysis was performed at the point where this transition was just detectable, the slowing of growth is apparent from changes in the transcriptome. MA synthesis is linked to new membrane synthesis and cell wall expansion[38](#bib38) and is needed for bacterial cell division, and MA synthesis gene transcript levels decreased ~3-fold under LSMMG. During exponential growth there is also a linear relationship between bacterial growth rate and the number of ribosomes in a cell.[46](#bib46) The slower growing LSMMG *M. marinum* culture had ~2-fold reduction in transcripts of multiple ribosomal proteins, indicating a downregulation of translation. *P. aeruginosa*[13](#bib13) and *S. typhimurium*[14](#bib14) cultured during spaceflight also had decreased ribosomal protein transcripts, but growth studies were not performed for these bacteria.

### Transcriptome changes imply a nutrient-deprived environment under LSMMG

Forty-six *M. marinum* orthologs of the *M. tuberculosis relA* nutrient starvation response genes[43](#bib43) had expression changes linked to LSMMG. Nutrient starvation results in decreased growth rate, translation, lipid biosynthesis (e.g., MA synthesis), and downregulation of metabolism.[43](#bib43),[47](#bib47) These phenotypes were also implied from the transcriptome data for LSMMG-exposed *M. marinum*.

Key bacterial metabolic pathways include β-oxidation of fatty acids, ETC, TCA, and the methylcitrate and glyoxylate cycles.[35](#bib35) In LSMMG *M. marinum*, a number of genes encoding enzymes involved in lipid degradation increased expression, but *icl* transcript levels decreased ~10-fold after long exposure to LSMMG. Icl converts isocitrate into glyoxylate, and glyoxylate combined with acetyl CoA from β oxidation of fatty acids produces malate for the glyoxylate cycle.[35](#bib35) Also transcript levels decreased 3–6-fold in LSMMG *M. marinum* for the WhiB3 transcription regulator, which is implicated in the adaptation of mycobacteria to growth on long-chain fatty acids.[48](#bib48) The transcriptome data therefore suggest energy generation from fatty acids was depressed in LSMMG *M. marinum*.

Decreased transcript levels of genes encoding Sdh2 and aconitase suggest that ETC, TCA, and the methylcitrate and glyoxylate cycles are depressed in LSMMG *M. marinum*. Sdh2 is an enzyme that is essential for growth and Sdh1 cannot substitute for all activities of Sdh2 (ref. [49](#bib49)), which links oxidative phosphorylation energy production to carbon metabolism.[36](#bib36) Succinate oxidation generates electrons for ETC, but in mycobacteria proline oxidation can substitute for succinate oxidation under carbon starvation and slow growth conditions.[36](#bib36) The transcriptome data indicate that *M. marinum* increased proline oxidation under LSMMG; transcript levels increased for proline oxidation enzymes when transcript levels from the Sdh2 genes decreased. Sdh1 and 2 levels in *Mycobacterium smegmatis* have been linked to energy availability and oxygen levels: under energy-limiting conditions Sdh2 is downregulated and Sdh1 is upregulated, and this is reversed under oxygen-limiting conditions.[36](#bib36) Even though in our study, the genes encoding Sdh1 did not alter expression significantly, the transcript changes for Sdh2 and the proline oxidation enzymes suggest that LSMMG *M. marinum* could still produce energy by ETC, and were sensing a carbon-starved/energy-limiting environment.

Taken together, the slower growth and the transcriptome data relating to translation and metabolism suggest that LSMMG simulates nutrient deprivation conditions in *M. marinum*. Fluid dynamics in cultures under microgravity or LSMMG are different from NG cultures.[8](#bib8),[11](#bib11) Crabbé *et al.*[11](#bib11) examined fluid mixing in HARVs subjected to NG and LSMMG by adding a drop of crystal violet to the fluid-filled vessels. This experiment demonstrated different mixing rates under the two conditions: even after 12 h the crystal violet had dispersed only around the circumference of the HARV subjected to LSMMG, while complete dispersal had occurred under NG. This decreased mixing and low fluid shear under LSMMG and the fact that *M. marinum* are non-motile in liquid culture results in the environment immediately around the bacterium being influenced more by diffusion. Under LSMMG, diffusion is likely the main method moving nutrients to and waste products away from the bacterium. The fluid dynamics in the LSMMG culture could therefore generate a local nutrient-deprived condition. This idea is also supported by increased transcript levels of the WhiB7 transcription regulator and the mycobactin synthesis genes after long exposure to LSMMG. *whiB7* is induced by nutrient starvation and growth under low-iron conditions,[50](#bib50),[51](#bib51) while mycobactin synthesis genes are induced under iron-restricted conditions.[52](#bib52) This is very relevant to cultures in potable water during spaceflight, which is where waterborne pathogenic mycobacteria could be found. In fact, *M. avium* 16s rRNA was identified by PCR amplification in three different water condensation samples from the Mir space station.[23](#bib23)

### Stress response genes

Even though there were LSMMG-linked transcript alterations related to *M. tuberculosis* EHR, the metabolic transcriptome data does not suggest the LSMMG *M. marinum* were under hypoxia. Many of the EHR genes that changed were stress response genes and included *sigH*, chaperone and heat-shock-regulated genes. SigH is one of 18 sigma factors in *M. marinum*[53](#bib53) and sigma factors function in promoter sequence recognition; directing initiation of transcription by RNA polymerase.[54](#bib54)
*sigH* was the only sigma factor gene to increase expression under LSMMG at both the short and long exposure times. Transcript levels of genes regulated directly or indirectly by SigH also changed significantly due to LSMMG exposure. SigH autoregulates its expression by binding to its own promoter and increasing transcription, and SigH is post-translationally regulated by RshA and PknB. RshA binds to SigH and prevents SigH from binding to RNA polymerase and SigH-RshA binding is disrupted by heat or redox stress,[55](#bib55) as well as phosphorylation by PknB.[56](#bib56) Small/no changes were detected in *rshA* and *pknB* transcript levels in LSMMG *M. marinum*, but conditions inside the LSMMG mycobacteria could have induced a substantial activity change in SigH by disrupting binding to RshA. A 1.5-fold increase in the *sigH* transcript is implicated in the 40-fold increase in *hsp* transcript after long exposure to LSMMG.

SigH is also required for the induction of *sigB* and *sigE*,[54](#bib54) and transcript levels from these genes increased after exposure to LSMMG for 4 days and ~35 h albeit by a small amount ([Figure 4](#fig4)). SigE has been implicated in the repression of *icl,*[57](#bib57) so increased SigE expression after the long exposure time to LSMMG may explain the decrease in the *icl* transcript.

SigH is known to be involved in the response of mycobacteria to oxidative stress as well as heat shock. Therefore *M. marinum* exposed to LSMMG would be expected to be resistant to H2O2. AlgU is another alternate sigma factor involved in heat shock and oxidative stress that has been implicated in the LSMMG response of *P. aeruginosa*[11](#bib11) and LSMMG-exposed *P. aeruginosa* were more resistant to H2O2 treatment.[11](#bib11) However, LSMMG-exposed LHM4 were more sensitive to H2O2 in our study. LHM4 was derived from 1218R by integration of an RFP-expressing plasmid (pDEAM2) at the *attB* site in the genome. The *attB* site is used to integrate plasmids into mycobacterium, as integration does not disrupt any genes.[58](#bib58) Previous studies have demonstrated that a *M. marinum* strain similar to LHM4 that was derived by pDEAM2 integration into 1218R had similar growth characteristics and virulence in Medaka fish to 1218R.[59](#bib59) We also determined that LHM4 and 1218R had similar growth characteristics under LSMMG. Therefore, LSMMG-exposed 1218R would be expected to be sensitive to H2O2. Interestingly, *S. aureus*, *S. typhimurium*, *E. coli*, *Enterobacter cloacae*, and *Citrobacter freundii* grown under LSMMG have also been found to be more sensitive to H2O2 than NG controls.[12](#bib12),[60](#bib60) A second group using *S. typhimurium* did find that LSMMG induced resistance to H2O2 and that resistance involved induction of catalase.[30](#bib30) Sensitivity to H2O2 could be altered by the availability of detoxification enzymes (catalase and alkyl hydroperoxidases), proteins involved in ‘repairing’ oxidized proteins (thioredoxins and methionine sulfoxide reductases) or redox buffers such as mycothiol and ergothioneine.[51](#bib51) Transcript levels of the genes encoding these proteins or the enzymes involved in the synthesis of mycothiol and ergothioneine did not significantly decrease in LSMMG *M. marinum*, which suggests these protection mechanisms were not compromised compared with the NG control. In fact, transcriptome data indicate that the *trxB1* transcript, which encodes one of the thioredoxins, increased by 4-fold after the long exposure time. Cysteine, which is key to many of the protecting enzymes/compounds, was also being conserved and production increased: the transcript of cysteine desulfurase decreased by 2-fold after LSMMG exposure and the *cysK2* transcript increased ~4-fold after the long exposure to LSMMG. Even though LSMMG *M. marinum* were more sensitive to H2O2, they had no change in sensitivity to killing by ionizing radiation compared with the NG control. Ionizing radiation and H2O2 are similar in that they both induce oxidative stress and generate similar types of DNA damage[61](#bib61) and DNA is a critical target in cells for cell killing. Lack of enhanced killing by radiation suggests the increased sensitivity to H2O2 was due to a different mechanism of radical formation/ cell killing by H2O2 in LSMMG bacteria that could not be defended against by the SigH response. In mycobacteria, the outer cell wall is the first line of defense against H2O2. This is not the case for ionizing radiation. Ionizing radiation deposits energy in the molecules of the cell and generates DNA damage by depositing energy directly on the DNA or by depositing energy in water, which produces hydroxyl radicals that then induce DNA damage.[62](#bib62) The energy deposited by the radiation can also generate lipid peroxides and oxidation of protein molecules, but the delivery of the energy to the macromolecules is not impeded by the structures surrounding or within the bacterium. Therefore, ionizing radiation energy deposition and DNA damage production will occur even if the cell wall composition changes. H2O2 can pass through lipid bilayers but is broken down to hydroxyl radicals by the Fenton reaction, which requires Fe2+ (ref. [63](#bib63)). The hydroxyl radicals can generate lipid peroxides from reaction with lipids in the cell wall that can diffuse and generate DNA damage.[64](#bib64) The composition of the cell wall and the iron content of the cell could therefore modify radical formation and alter the sensitivity of the cell to H2O2. There were a substantial number of transcript changes in LSMMG *M. marinum* related to the cell wall. Transcripts of genes related to MA and phenolic glycolipid synthesis were decreased, while transcripts encoding proteins involved in modifying the types of MA were increased. LSMMG-exposed *M. marinum* also had an increase in transcripts related to mycobactin synthesis and mycobactins bind iron from the environment and transport the charged iron across the hydrophobic cell wall.[39](#bib39) An increase in mycobactin synthesis could therefore increase iron content in the mycobacterium. Alterations to mycobactin synthesis and to the composition of the mycobacterium cell wall could therefore explain why the LSMMG-exposed mycobacterium, even with the *sigH* operon response, were more sensitive to H2O2 but not ionizing radiation.

The changes at the cell wall did not sensitize the LSMMG *M. marinum* to pH 3.5 and so cell wall lipid composition may not influence acid resistance. Other bacteria exposed to LSMMG have also been found not to have increased sensitivity to killing by acid[60](#bib60) even though they were more sensitive to H2O2. Recently, *M. tuberculosis* acid resistance was linked to asparagine metabolism.[65](#bib65) An asparagine transporter AnsP2 imports asparagine into the bacterium where asparaginase AnsA breaks down asparagine to aspartate with the production of ammonia. The ammonia buffers hydrogen ions by forming NH4+. AnsA is also secreted to perform this reaction in the environment around the mycobacterium. In *M. marinum*, AnsP2 is encoded by *MMAR\_0627* and AnsA is encoded by *MMAR\_2360*. Transcript levels of these genes did not alter during exposure to LSMMG and hence the ability to produce ammonia may not have changed in LSMMG-exposed *M. marinum*. This may explain why LSMMG-exposed *M. marinum* were not more sensitive to killing by acid.

### Could LSMMG alter the virulence of *M. marinum*?

*sigB*, *E* and *H* as well as *hsp* and *dnaK* are induced when *M. marinum* infect fish or are ingested by mosquito larvae,[53](#bib53) and *M. marinum* ingested by mosquito larvae are more virulent in fish compared with *in vitro* cultured *M. marinum.*[59](#bib59) Exposure to the mosquito larvae gut therefore primes the *M. marinum* for survival inside fish macrophage. The *hsp* gene is also the most induced *M. tuberculosis* gene during macrophage infection,[52](#bib52) and *sigH* mutants of *M. tuberculosis* and *M. avium* subsp. *paratuberculosis* have reduced survival in animal infection models.[66](#bib66),[67](#bib67) Induction of the SigH stress response by LSMMG before infection should aid survival of mycobacteria in macrophage and potentially increase virulence. The transcriptome of LSMMG *M. marinum* is also similar to mycobacteria inside macrophage in that they have altered growth and metabolism to survive under a nutrient- and iron-starved environment.[39](#bib39),[43](#bib43),[52](#bib52) LSMMG increased transcripts involved in lipid degradation and mycobacteria inside macrophage use host lipids for a source of energy.[37](#bib37) However, Icl expression would need to increase; studies have shown that Icl is essential to connect lipid degradation with the glyoxylate shunt, and is required for survival in animals.[35](#bib35) Even though LSMMG *M. marinum* do have a number of adaptations that would prime them for increased virulence, the enhanced killing by H2O2 is likely to be detrimental for bacterial survival inside macrophage. Future studies examining LSMMG- or spaceflight-exposed *M. marinum* survival inside macrophage and fish are needed to determine whether *M. marinum* in space could pose a risk to the health of astronauts. As *M. marinum* is an accepted model of waterborne mycobacteria and *M. tuberculosis*, these results do have implications for other pathogenic mycobacteria exposed to LSMMG as they may respond with similar transcriptome changes.

## Materials and methods

### Bacterial strains and culture

*M. marinum* strain 1218R,[53](#bib53) a strain isolated from infected fish, and a derivative of this strain, LHM4, were used for this study. LHM4 carries an integrated plasmid pDEAM2, which encodes red fluorescent protein and confers kanamycin resistance. Bacteria were grown in the dark at 30 °C in tubes in a rotating incubator (120 r.p.m.)[27](#bib27) or in HARVs attached to a rotating cell culture system (RCCS, Synthecon, Houston, TX, USA) in Middlebrook 7H9 broth (Difco, Detroit, MI, USA) supplemented with ADS (5% BSA Fraction V, 2% Dextrose, 0.81% NaCl) to 10%, 0.5% glycerol, 100 μg cycloheximide/ml, and 0.2% Tween 80. Cultures were also grown on solid Middlebrook 7H10 agar (Difco) supplemented with 10% ADS, 0.5% glycerol, and 100 μg cycloheximide/ml. LHM4 cultures also contained 20 μg kanamycin/ml. To initiate cultures, solid medium was inoculated from a frozen stock and incubated for ~7 days to obtain colonies. Colonies were used to inoculate tube cultures (~3 ml). The tube cultures were grown for 5–7 days to an OD600 ~1. This tube culture was then diluted to OD600 0.0005 or 0.002 before growth in 10 or 50 ml HARVs, respectively. The lower OD600 of the culture at the start of growth in the 10 ml HARVs prevented the cultures from being at saturation density for the majority of the 4-day exposure time. For the short exposure time, the cultures were grown in 50 ml HARVs. For the long exposure, the cultures were grown in 10 ml HARVs for 4 days and then re-diluted to OD600 0.002 in 50 ml HARVs for another ~35 h. For NG, HARVs were rotated (25 r.p.m.) on an axis parallel to the direction of gravitational force, while they were rotated (25 r.p.m.) on an axis perpendicular to the direction of the gravitational force for LSMMG.[3](#bib3)

### Growth studies

For the continuous growth studies, one large volume of OD600 0.0005 culture was prepared from an initiating tube culture and split between two 10 ml HARVs: one was subjected to NG and the other to LSMMG. After 4 days the cultures were removed and the OD600 was determined. The statistical difference between the LSMMG and NG cultures was tested by a *T*-test performed using SAS 9.4 (SAS Gary, NC, USA). To examine culture density during exponential growth, one large volume of OD600 0.002 culture was prepared from an initiating tube culture and split between two 50 ml HARVs: one was subjected to NG and the other to LSMMG. Where specified, 3 μg catalase/ml (0.06 U/ml; ThermoFisher Scientific, Waltham, MA, USA) was included. Bacteria were grown for ~150 h but the HARVs were stopped to remove samples (1–2 ml). Liquid medium was added to the HARV to replace the removed sample volume (2–4% of the volume) to maintain a low-shear environment. In order to obtain accurate OD600 and CFU measurements, a disaggregation procedure was used: SDS (final concentration 0.01%) was added; the culture was passed through a 27.5 gauge needle and either vortexed before measurement of the OD600 or preparation of dilutions, or the bacteria were allowed to settle for 5 min before growth on solid medium. Use of the syringe needle, vortexing, and then allowing the culture to settle before plating on solid medium is standard procedure for mycobacterium.[32](#bib32) Colonies from each sample were grown on at least two plates of solid medium for 7–10 days. Colonies were counted for each time point and the CFU/ml calculated. Preliminary experiments confirmed by microscopy that disaggregation of cultures was achieved using the syringe needle, and initial experiments determined that the addition of the SDS to the NG and LSMMG cultures only during the disaggregation and dilution stage did not alter colony forming ability (data not shown).

To determine statistical differences between LSMMG and NG cultures during the 37–70 h time period of growth, regression analysis was performed on the OD600 data using SAS 9.4 (SAS).

### Size of bacterium

A portion of the LHM4 culture grown for 4 days in a 10 ml HARV was passed through a 27.5 gauge needle and fixed by the addition of 30 μl 4% paraformaldehyde to 10 μl of culture. Samples were placed at 4 °C for at least 24 h before the measurement of the diameter and length of 50 bacteria using microscopy. The Instat3 program and the Unpaired *T*-test or the Mann–Whitney test was used to compare the lengths and diameters of the bacterium.

### RNA preparation

RNA extraction from *M. marinum* strain 1218R was performed according to Rustad *et al.*[68](#bib68) RNA integrity ([Supplementary Table 1](#xob1)) was assessed on an Agilent 2100 Bioanalyzer (Agilent, Santa Clara, CA, USA) and quantitated using the Qubit RNA BR assay (Invitrogen, ThermoFisher, Waltham, MA, USA).

### RNA-Seq analyses

RNA samples were subjected to DNase treatment, purification with the RNeasy MinElute Cleanup kit (Qiagen, Valencia, CA, USA) and removal of the rRNA and preparation of the RNA-Seq libraries using the ScriptSeq Complete kit for bacteria (Epicentre Biotechnologies, Madison, WI, USA). The DNA libraries (4 nmol/l) were pooled, denatured with 0.2 mol/l sodium hydroxide, and diluted to 12 pmol/l. A 76-cycle paired-end read sequencing run was performed on an Illumina Miseq desktop sequencer (Illumina, Inc., San Diego, USA). The number of sequence reads is shown in [Supplementary Table 1](#xob1).

Seventy-five base-paired-end RNA-Seq reads were aligned to the *M. marinum* strain M genome (Genbank accession number [CP000854](https://www.ncbi.nlm.nih.gov/nuccore/CP000854)) using the Spliced Transcripts Alignment to a Reference (STAR) aligner. Reads were visualized using the Integrative Genomics Viewer (IGV, Broad Institute, <http://www.broadinstitute.org/igv/home>). Reads were aligned to the *M. marinum* strain M transcriptome using RSEM version 1.2.9 for transcript quantification. The fragments per kilobase of transcript per million mapped reads (FPKM) was calculated for each gene and sample gene level differential expression analysis was performed using EBseq version 1.1.5 within RSEM. For the short or long exposure times, the LSMMG samples were compared with NG samples. The LSMMG samples from the two exposure times (short and long) were also compared. Changes in expression were significant if *P*<0.05. The functions of genes were assigned according to the Marinolist (<http://mycobrowser.epfl.ch/marinolist.html>). Pathway assignment used the Marinolist advanced search and the literature, and *M. marinum* orthologs of *M. tuberculosis* genes were identified using <http://tuberculist.epfl.ch>. Transcript levels for the genes assigned to pathways were analyzed and heat maps generated using GeneSifter (<http://www.geospiza.com/Products/AnalysisEdition.shtml>). RNA-Seq data has been deposited in the GeneLab database.

### Reverse-transcriptase quantitative PCR

RNA samples were treated with DNase and converted to cDNA using Moloney Murine Leukemia Virus reverse transcriptase. qPCR reactions in duplicate were performed with primers ([Supplementary Table 8](#xob1)) and Power SYBR Green PCR Master Mix (Applied Biosystems, Foster City, CA, USA) using a 7500 Fast Real-Time PCR system (Applied Biosystems) according to the manufacturer’s recommendations. The cDNA level for transcripts was normalized to the cDNA level for 16s rRNA for each sample. This ratio was averaged for the three independent samples and the fold change in expression was calculated by dividing the LSMMG average by the NG average.

### Survival studies

LHM4 were removed from the HARV and the OD600 measured in a sample after disaggregation. Bacteria were in exponential phase (OD600 0.09–0.2). For radiation and hydrogen peroxide (H2O2) studies, bacteria were centrifuged and washed in phosphate-buffered saline containing 0.2% Tween 80 (Tw-PBS), before resuspension at OD600 ~0.2 in Tw-PBS. Two samples (200 μl in a sealed tube) were prepared for each time point or dose. One sample was treated (1–4 h) with 5 mmol/l H2O2 at 30 °C or with gamma radiation (100–400 Gy) at room temperature. The second sample was not treated and was used as the control to determine the CFU/ml at zero treatment at the time point. A similar method was used to treat the bacteria with acid. The bacteria were centrifuged and resuspended at OD600 ~0.2 in growth medium (control) or growth medium at pH 3.5. A small amount of one mole per liter citric acid was added to the medium to reach pH 3.5. Once the bacteria were resuspended, the pH was checked using pH paper. The control and acid-treated samples were incubated at 30 °C for 1–4 h. After all treatment regimens, bacteria were disaggregated and diluted as described above and grown on three plates containing solid medium. Bacteria were incubated at 30 °C for 7–10 days and the CFU/ml calculated for each control and treated sample. Survival was calculated by:

|  |
| --- |
| %Survival=AvgCFU/mloftreatedcultureAvgCFU/mlofcontrolculture×100 |

The survival data were compared by calculating the area under the curve for each data set using the Trapezoidal Rule, and the Student’s *T-*test was used to compare the values. These statistical analyses were performed using SAS 9.4 (SAS).

## Acknowledgments

We thank Ms Paula Polk in the Genomic Core at LSUHSC for performing the preparation and sequencing of RNA-Seq samples. This work was supported by grant number NNX13AM02G (LH) from NASA and funds for the RNA-Seq sample processing were provided by the Feist-Weiller Cancer Center at LSU Health Sciences Center—Shreveport (LH). Bioinformatic analysis was supported by grants R01 AI101046 (EKF) and R01 AI106676 (EKF) from National Institute of Allergy and Infectious Diseases.

## Footnotes

[Supplementary Information](#xob1) accompanies the paper on the *npj Microgravity* (http://www.nature.com/npmgrav)

The authors declare no conflict of interest.

Supplementary Tables

[Click here for additional data file.](/articles/instance/5515531/bin/npjmgrav201638-s1.doc) (208.5KB, doc)

Supplementary Figures

[Click here for additional data file.](/articles/instance/5515531/bin/npjmgrav201638-s2.ppt) (1.7MB, ppt)

Supplementary Excel Worksheet 1

[Click here for additional data file.](/articles/instance/5515531/bin/npjmgrav201638-s3.pdf) (749.3KB, pdf)

Supplementary Excel Worksheet 2

[Click here for additional data file.](/articles/instance/5515531/bin/npjmgrav201638-s4.pdf) (500.7KB, pdf)

## References

1. Venkateswaran, K. et al. International space station environmental microbiome—microbial inventories of ISS filter debris. Appl. Microbiol. Biotechnol.
   98, 6453–6466 (2014). [[DOI](https://doi.org/10.1007/s00253-014-5650-6)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24695826/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&title=International%20space%20station%20environmental%20microbiome%E2%80%94microbial%20inventories%20of%20ISS%20filter%20debris&volume=98&publication_year=2014&pages=6453&pmid=24695826&doi=10.1007/s00253-014-5650-6&)]
2. Yamaguchi, N. et al. Microbial monitoring of crewed habitats in space—current status and future perspectives. Microbes Environ.
   29, 250–260 (2014). [[DOI](https://doi.org/10.1264/jsme2.ME14031)] [[PMC free article](/articles/PMC4159036/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25130885/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbes%20Environ.&title=Microbial%20monitoring%20of%20crewed%20habitats%20in%20space%E2%80%94current%20status%20and%20future%20perspectives&volume=29&publication_year=2014&pages=250&pmid=25130885&doi=10.1264/jsme2.ME14031&)]
3. Nickerson, C. A., Ott, C. M., Wilson, J. W., Ramamurthy, R. & Pierson, D. L. Microbial responses to microgravity and other low-shear environments. Microbiol. Mol. Biol. Rev.
   68, 345–361 (2004). [[DOI](https://doi.org/10.1128/MMBR.68.2.345-361.2004)] [[PMC free article](/articles/PMC419922/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15187188/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Mol.%20Biol.%20Rev.&title=Microbial%20responses%20to%20microgravity%20and%20other%20low-shear%20environments&volume=68&publication_year=2004&pages=345&pmid=15187188&doi=10.1128/MMBR.68.2.345-361.2004&)]
4. Rosenzweig, J. A., Ahmed, S., Eunson, J. Jr. & Chopra, A. K. Low-shear force associated with modeled microgravity and spaceflight does not similarly impact the virulence of notable bacterial pathogens. Appl. Microbiol. Biotechnol.
   98, 8797–8807 (2014). [[DOI](https://doi.org/10.1007/s00253-014-6025-8)] [[PMC free article](/articles/PMC4199916/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25149449/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&title=Low-shear%20force%20associated%20with%20modeled%20microgravity%20and%20spaceflight%20does%20not%20similarly%20impact%20the%20virulence%20of%20notable%20bacterial%20pathogens&volume=98&publication_year=2014&pages=8797&pmid=25149449&doi=10.1007/s00253-014-6025-8&)]
5. Coil, D. A. et al. Growth of 48 built environment bacterial isolates on board the International Space Station (ISS). PeerJ
   4, e1842 (2016). [[DOI](https://doi.org/10.7717/peerj.1842)] [[PMC free article](/articles/PMC4806633/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27019789/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PeerJ&title=Growth%20of%2048%20built%20environment%20bacterial%20isolates%20on%20board%20the%20International%20Space%20Station%20(ISS)&volume=4&publication_year=2016&pages=e1842&pmid=27019789&doi=10.7717/peerj.1842&)]
6. Kalpana, D., Im, C. & Lee, Y. S. Comparative growth, cross stress resistance, transcriptomics of *Streptococcus pyogenes* cultured under low shear modeled microgravity and normal gravity. Saudi J. Biol. Sci.
   23, 24–33 (2016). [[DOI](https://doi.org/10.1016/j.sjbs.2015.02.004)] [[PMC free article](/articles/PMC4705266/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26858535/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Saudi%20J.%20Biol.%20Sci.&title=Comparative%20growth,%20cross%20stress%20resistance,%20transcriptomics%20of%20Streptococcus%20pyogenes%20cultured%20under%20low%20shear%20modeled%20microgravity%20and%20normal%20gravity&volume=23&publication_year=2016&pages=24&pmid=26858535&doi=10.1016/j.sjbs.2015.02.004&)]
7. Kim, H. W. & Rhee, M. S. Influence of low-shear modeled microgravity on heat resistance, membrane fatty acid composition, and heat stress-related gene expression in *Escherichia coli* O157:H7 ATCC 35150, 43889 and 43895. Appl. Environ. Microbiol.
   82, 2893–2901 (2016). [[DOI](https://doi.org/10.1128/AEM.00050-16)] [[PMC free article](/articles/PMC4959082/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26944847/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=Influence%20of%20low-shear%20modeled%20microgravity%20on%20heat%20resistance,%20membrane%20fatty%20acid%20composition,%20and%20heat%20stress-related%20gene%20expression%20in%20Escherichia%20coli%20O157:H7%20ATCC%2035150,%2043889%20and%2043895&volume=82&publication_year=2016&pages=2893&pmid=26944847&doi=10.1128/AEM.00050-16&)]
8. Horneck, G., Klaus, D. M. & Mancinelli, R. L. Space microbiology. Microbiol. Mol. Biol. Rev.
   74, 121–156 (2010). [[DOI](https://doi.org/10.1128/MMBR.00016-09)] [[PMC free article](/articles/PMC2832349/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20197502/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Mol.%20Biol.%20Rev.&title=Space%20microbiology&volume=74&publication_year=2010&pages=121&pmid=20197502&doi=10.1128/MMBR.00016-09&)]
9. Xu, B. et al. Simulated microgravity affects ciprofloxacin susceptibility and expression of acrAB-tolC genes in *E.coli* ATCC25922. Int. J. Clin. Exp. Pathol.
   8, 7945–7952 (2015). [[PMC free article](/articles/PMC4555688/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26339360/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Clin.%20Exp.%20Pathol.&title=Simulated%20microgravity%20affects%20ciprofloxacin%20susceptibility%20and%20expression%20of%20acrAB-tolC%20genes%20in%20E.coli%20ATCC25922&volume=8&publication_year=2015&pages=7945&pmid=26339360&)]
10. Taylor, P. W. Impact of space flight on bacterial virulence and antibiotic susceptibility. Infect. Drug Resist.
    8, 249–262 (2015). [[DOI](https://doi.org/10.2147/IDR.S67275)] [[PMC free article](/articles/PMC4524529/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26251622/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect.%20Drug%20Resist.&title=Impact%20of%20space%20flight%20on%20bacterial%20virulence%20and%20antibiotic%20susceptibility&volume=8&publication_year=2015&pages=249&pmid=26251622&doi=10.2147/IDR.S67275&)]
11. Crabbé, A. et al. Response of *Pseudomonas aeruginosa* PAO1 to low shear modeled microgravity involves AlgU regulation. Environ. Microbiol.
    12, 1545–1564 (2010). [[DOI](https://doi.org/10.1111/j.1462-2920.2010.02184.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20236169/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&title=Response%20of%20Pseudomonas%20aeruginosa%20PAO1%20to%20low%20shear%20modeled%20microgravity%20involves%20AlgU%20regulation&volume=12&publication_year=2010&pages=1545&pmid=20236169&doi=10.1111/j.1462-2920.2010.02184.x&)]
12. Castro, S. L., Nelman-Gonzalez, M., Nickerson, C. A. & Ott, C. M. Induction of attachment-independent biofilm formation and repression of hfq expression by low-fluid shear culture of *Staphylococcus aureus*. Appl. Environ. Microbiol.
    77, 6368–6378 (2011). [[DOI](https://doi.org/10.1128/AEM.00175-11)] [[PMC free article](/articles/PMC3187170/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21803898/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=Induction%20of%20attachment-independent%20biofilm%20formation%20and%20repression%20of%20hfq%20expression%20by%20low-fluid%20shear%20culture%20of%20Staphylococcus%20aureus&volume=77&publication_year=2011&pages=6368&pmid=21803898&doi=10.1128/AEM.00175-11&)]
13. Crabbé, A. et al. Transcriptional and proteomic responses of *Pseudomonas aeruginosa* PAO1 to spaceflight conditions involve Hfq regulation and reveal a role for oxygen. Appl. Environ. Microbiol.
    77, 1221–1230 (2011). [[DOI](https://doi.org/10.1128/AEM.01582-10)] [[PMC free article](/articles/PMC3067220/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21169425/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=Transcriptional%20and%20proteomic%20responses%20of%20Pseudomonas%20aeruginosa%20PAO1%20to%20spaceflight%20conditions%20involve%20Hfq%20regulation%20and%20reveal%20a%20role%20for%20oxygen&volume=77&publication_year=2011&pages=1221&pmid=21169425&doi=10.1128/AEM.01582-10&)]
14. Wilson, J. W. et al. Space flight alters bacterial gene expression and virulence and reveals a role for global regulator Hfq. Proc. Natl. Acad. Sci. USA
    104, 16299–16304 (2007). [[DOI](https://doi.org/10.1073/pnas.0707155104)] [[PMC free article](/articles/PMC2042201/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17901201/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Space%20flight%20alters%20bacterial%20gene%20expression%20and%20virulence%20and%20reveals%20a%20role%20for%20global%20regulator%20Hfq&volume=104&publication_year=2007&pages=16299&pmid=17901201&doi=10.1073/pnas.0707155104&)]
15. Stinear, T. P. et al. Insights from the complete genome sequence of *Mycobacterium marinum* on the evolution of *Mycobacterium tuberculosis*. Genome Res.
    18, 729–741 (2008). [[DOI](https://doi.org/10.1101/gr.075069.107)] [[PMC free article](/articles/PMC2336800/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18403782/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Insights%20from%20the%20complete%20genome%20sequence%20of%20Mycobacterium%20marinum%20on%20the%20evolution%20of%20Mycobacterium%20tuberculosis&volume=18&publication_year=2008&pages=729&pmid=18403782&doi=10.1101/gr.075069.107&)]
16. Cosma, C. L., Sherman, D. R. & Ramakrishnan, L. The secret lives of the pathogenic mycobacteria. Ann. Rev. Microbiol.
    57, 641–676 (2003). [[DOI](https://doi.org/10.1146/annurev.micro.57.030502.091033)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/14527294/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Rev.%20Microbiol.&title=The%20secret%20lives%20of%20the%20pathogenic%20mycobacteria&volume=57&publication_year=2003&pages=641&pmid=14527294&doi=10.1146/annurev.micro.57.030502.091033&)]
17. Dailloux, M. et al. Respiratory infections associated with nontuberculosis mycobacteria in non-HIV patients. Eur. Respir. J.
    28, 1211–1215 (2006). [[DOI](https://doi.org/10.1183/09031936.00063806)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17138678/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Respir.%20J.&title=Respiratory%20infections%20associated%20with%20nontuberculosis%20mycobacteria%20in%20non-HIV%20patients&volume=28&publication_year=2006&pages=1211&pmid=17138678&doi=10.1183/09031936.00063806&)]
18. Fordham von Reyn, C. et al. The international epidemiology of disseminated *Mycobacterium avium* complex infection in AIDS. International MAC study group. AIDS
    10, 1025–1032 (1996). [[DOI](https://doi.org/10.1097/00002030-199610090-00014)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/8853737/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=The%20international%20epidemiology%20of%20disseminated%20Mycobacterium%20avium%20complex%20infection%20in%20AIDS.%20International%20MAC%20study%20group&volume=10&publication_year=1996&pages=1025&pmid=8853737&doi=10.1097/00002030-199610090-00014&)]
19. Hall-Stoodley, L., Brun, O. S., Polshyna, G. & Barker, L. P. *Mycobacterium marinum* biofilm formation reveals cording morphology. FEMS Microbiol. Lett.
    257, 43–49 (2006). [[DOI](https://doi.org/10.1111/j.1574-6968.2006.00143.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16553830/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Lett.&title=Mycobacterium%20marinum%20biofilm%20formation%20reveals%20cording%20morphology&volume=257&publication_year=2006&pages=43&pmid=16553830&doi=10.1111/j.1574-6968.2006.00143.x&)]
20. Falkinham, J. O. III, Norton, C. D. & LeChevallier, M. W. Factors influencing numbers of *Mycobacterium avium*, *Mycobacterium intracellulare*, and other mycobacteria in drinking water distribution systems. Appl. Environ. Microbiol.
    67, 1225–1231 (2001). [[DOI](https://doi.org/10.1128/AEM.67.3.1225-1231.2001)] [[PMC free article](/articles/PMC92717/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11229914/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=Factors%20influencing%20numbers%20of%20Mycobacterium%20avium,%20Mycobacterium%20intracellulare,%20and%20other%20mycobacteria%20in%20drinking%20water%20distribution%20systems&volume=67&publication_year=2001&pages=1225&pmid=11229914&doi=10.1128/AEM.67.3.1225-1231.2001&)]
21. Feazel, L. M. et al. Opportunistic pathogens enriched in showerhead biofilms. Proc. Natl. Acad. Sci. USA
    106, 16393–16399 (2009). [[DOI](https://doi.org/10.1073/pnas.0908446106)] [[PMC free article](/articles/PMC2752528/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19805310/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Opportunistic%20pathogens%20enriched%20in%20showerhead%20biofilms&volume=106&publication_year=2009&pages=16393&pmid=19805310&doi=10.1073/pnas.0908446106&)]
22. Steed, K. A. & Falkinham, J. O. III Effect of growth in biofilms on chlorine susceptibility of *Mycobacterium avium* and *Mycobacterium intracellulare*. Appl. Environ. Microbiol.
    72, 4007–4011 (2006). [[DOI](https://doi.org/10.1128/AEM.02573-05)] [[PMC free article](/articles/PMC1489660/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16751509/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=Effect%20of%20growth%20in%20biofilms%20on%20chlorine%20susceptibility%20of%20Mycobacterium%20avium%20and%20Mycobacterium%20intracellulare&volume=72&publication_year=2006&pages=4007&pmid=16751509&doi=10.1128/AEM.02573-05&)]
23. Kawamura, Y. et al. Bacterial population in Russian space station ‘Mir’. Microbiol. Immunol.
    45, 819–828 (2001). [[DOI](https://doi.org/10.1111/j.1348-0421.2001.tb01321.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11838899/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Immunol.&title=Bacterial%20population%20in%20Russian%20space%20station%20%E2%80%98Mir%E2%80%99&volume=45&publication_year=2001&pages=819&pmid=11838899&doi=10.1111/j.1348-0421.2001.tb01321.x&)]
24. Mermel, L. A. Infection prevention and control during prolonged human space travel. Clin. Infect. Dis.
    56, 123–130 (2013). [[DOI](https://doi.org/10.1093/cid/cis861)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23051761/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Infect.%20Dis.&title=Infection%20prevention%20and%20control%20during%20prolonged%20human%20space%20travel&volume=56&publication_year=2013&pages=123&pmid=23051761&doi=10.1093/cid/cis861&)]
25. Cervantes, J. L. & Hong, B.-Y. Dysbiosis and immune dysregulation in outer space. Int. Rev. Immunol.
    35, 67–82 (2016). [[DOI](https://doi.org/10.3109/08830185.2015.1027821)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25970037/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20Rev.%20Immunol.&title=Dysbiosis%20and%20immune%20dysregulation%20in%20outer%20space&volume=35&publication_year=2016&pages=67&pmid=25970037&doi=10.3109/08830185.2015.1027821&)]
26. Chao, Y. & Vogel, J. The role of Hfq in bacterial pathogens. Curr. Opin. Microbiol.
    13, 24–33 (2010). [[DOI](https://doi.org/10.1016/j.mib.2010.01.001)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20080057/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Microbiol.&title=The%20role%20of%20Hfq%20in%20bacterial%20pathogens&volume=13&publication_year=2010&pages=24&pmid=20080057&doi=10.1016/j.mib.2010.01.001&)]
27. Gao, L.-Y. & Manoranjan, J. Laboratory maintenance of *Mycobacterium marinum*. Curr. Protoc. Microbiol.
    10B.1.1–10B.1.8; doi:10.1002/978047172959.mc10b01s00 (2005). [[DOI](https://doi.org/10.1002/9780471729259.mc10b01s00)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18770549/)]
28. Bannantine, J. P., Zhang, Q., Li, L.-L. & Kapur, V. Genomic homogeneity between *Mycobacterium avium* subsp. *avium* and *Mycobacterium avium* subsp. paratuberculosis belies their divergent growth rates. BMC Microbiol.
    3, 10 (2003). [[DOI](https://doi.org/10.1186/1471-2180-3-10)] [[PMC free article](/articles/PMC156888/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12740027/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol.&title=Genomic%20homogeneity%20between%20Mycobacterium%20avium%20subsp.%20avium%20and%20Mycobacterium%20avium%20subsp.%20paratuberculosis%20belies%20their%20divergent%20growth%20rates&volume=3&publication_year=2003&pages=10&pmid=12740027&doi=10.1186/1471-2180-3-10&)]
29. Wilson, J. W. et al. Low-shear modeled microgravity alters the *Salmonella enteric* serovar Typhimurium stress response in an RpoS-independent manner. Appl. Environ. Microbiol.
    68, 5408–5416 (2002). [[DOI](https://doi.org/10.1128/AEM.68.11.5408-5416.2002)] [[PMC free article](/articles/PMC129924/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12406731/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=Low-shear%20modeled%20microgravity%20alters%20the%20Salmonella%20enteric%20serovar%20Typhimurium%20stress%20response%20in%20an%20RpoS-independent%20manner&volume=68&publication_year=2002&pages=5408&pmid=12406731&doi=10.1128/AEM.68.11.5408-5416.2002&)]
30. Pacello, F., Rotilio, G. & Battistoni, A. Low-shear modeled microgravity enhances *Salmonella enteric* resistance to hydrogen peroxide through a mechanism involving KatG and KatN. Open Microbiol. J.
    6, 53–64 (2012). [[DOI](https://doi.org/10.2174/1874285801206010053)] [[PMC free article](/articles/PMC3414715/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22888375/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Open%20Microbiol.%20J.&title=Low-shear%20modeled%20microgravity%20enhances%20Salmonella%20enteric%20resistance%20to%20hydrogen%20peroxide%20through%20a%20mechanism%20involving%20KatG%20and%20KatN&volume=6&publication_year=2012&pages=53&pmid=22888375&doi=10.2174/1874285801206010053&)]
31. Tucker, D. L. et al. Characterization of *Escherichia coli* MG1655 grown in a low-shear modeled microgravity environment. BMC Microbiol.
    7, 15 (2007). [[DOI](https://doi.org/10.1186/1471-2180-7-15)] [[PMC free article](/articles/PMC1852313/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17343762/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol.&title=Characterization%20of%20Escherichia%20coli%20MG1655%20grown%20in%20a%20low-shear%20modeled%20microgravity%20environment&volume=7&publication_year=2007&pages=15&pmid=17343762&doi=10.1186/1471-2180-7-15&)]
32. Park, B., Subbian, S., El-Etr, S. H., Cirillo, S. L. G. & Cirillo, J. D. Use of gene dosage effects for a whole-genome screen to identify *Mycobacterium marinum* macrophage infection loci. Infect. Immun.
    76, 3100–3115 (2008). [[DOI](https://doi.org/10.1128/IAI.00015-08)] [[PMC free article](/articles/PMC2446711/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18443095/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&title=Use%20of%20gene%20dosage%20effects%20for%20a%20whole-genome%20screen%20to%20identify%20Mycobacterium%20marinum%20macrophage%20infection%20loci&volume=76&publication_year=2008&pages=3100&pmid=18443095&doi=10.1128/IAI.00015-08&)]
33. Staropoli, J. F. & Branda, J. A. Cord formation in a clinical isolate of *Mycobacterium marinum*. J. Clin. Microbiol.
    46, 2814–2816 (2008). [[DOI](https://doi.org/10.1128/JCM.00197-08)] [[PMC free article](/articles/PMC2519507/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18579723/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Microbiol.&title=Cord%20formation%20in%20a%20clinical%20isolate%20of%20Mycobacterium%20marinum&volume=46&publication_year=2008&pages=2814&pmid=18579723&doi=10.1128/JCM.00197-08&)]
34. Haas, B. J., Chin, M., Nusbaum, C., Birren, B. W. & Livny, J. How deep is deep enough for RNA-Seq profiling of bacterial transcriptomes?
    BMC Genomics
    13, 734 (2012). [[DOI](https://doi.org/10.1186/1471-2164-13-734)] [[PMC free article](/articles/PMC3543199/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23270466/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=How%20deep%20is%20deep%20enough%20for%20RNA-Seq%20profiling%20of%20bacterial%20transcriptomes?&volume=13&publication_year=2012&pages=734&pmid=23270466&doi=10.1186/1471-2164-13-734&)]
35. Munoz-Elias, E. J., Upton, A. M., Cherian, J. & McKinney, J. D. Role of the methylcitrate cycle in *Mycobacterium tuberculosis* metabolism, intracellular growth, and virulence. Mol. Microbiol.
    60, 1109–1122 (2006). [[DOI](https://doi.org/10.1111/j.1365-2958.2006.05155.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16689789/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&title=Role%20of%20the%20methylcitrate%20cycle%20in%20Mycobacterium%20tuberculosis%20metabolism,%20intracellular%20growth,%20and%20virulence&volume=60&publication_year=2006&pages=1109&pmid=16689789&doi=10.1111/j.1365-2958.2006.05155.x&)]
36. Cook, G. M., Hards, K., Vilcheze, C., Hartman, T. & Berney, M. Energetics of respiration and oxidative phosphorylation in mycobacteria. Microbiol. Spectr.
    2(3): MGM2-0015-2013. doi:10.1128/microbiolspec.MGM2-0015-2013 (2014). [[DOI](https://doi.org/10.1128/microbiolspec.MGM2-0015-2013)] [[PMC free article](/articles/PMC4205543/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25346874/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Spectr.&title=Energetics%20of%20respiration%20and%20oxidative%20phosphorylation%20in%20mycobacteria&volume=2&publication_year=2014&pmid=25346874&doi=10.1128/microbiolspec.MGM2-0015-2013&)]
37. Lovewell, R. R., Sassetti, C. M. & VanderVen, B. C. Chewing the fat: lipid metabolism and homeostasis during *M. tuberculosis* infection. Curr. Opin. Microbiol.
    29, 30–36 (2016). [[DOI](https://doi.org/10.1016/j.mib.2015.10.002)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26544033/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Microbiol.&title=Chewing%20the%20fat:%20lipid%20metabolism%20and%20homeostasis%20during%20M.%20tuberculosis%20infection&volume=29&publication_year=2016&pages=30&pmid=26544033&doi=10.1016/j.mib.2015.10.002&)]
38. Marrakchi, H., Laneelle, M.-A. & Daffe, M. Mycolic acids: structures, biosynthesis and beyond. Chem. Biol.
    21, 67–85 (2014). [[DOI](https://doi.org/10.1016/j.chembiol.2013.11.011)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24374164/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&title=Mycolic%20acids:%20structures,%20biosynthesis%20and%20beyond&volume=21&publication_year=2014&pages=67&pmid=24374164&doi=10.1016/j.chembiol.2013.11.011&)]
39. DeVoss, J. J. et al. The salicylate-derived mycobactin siderphores of *Mycobacterium tuberculosis* are essential for growth in macrophages. Proc. Natl. Acad. Sci. USA
    97, 1252–1257 (2000). [[DOI](https://doi.org/10.1073/pnas.97.3.1252)] [[PMC free article](/articles/PMC15586/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10655517/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=The%20salicylate-derived%20mycobactin%20siderphores%20of%20Mycobacterium%20tuberculosis%20are%20essential%20for%20growth%20in%20macrophages&volume=97&publication_year=2000&pages=1252&pmid=10655517&doi=10.1073/pnas.97.3.1252&)]
40. Raman, S. et al. The alternative sigma factor SigH regulates major components of oxidative and heat stress responses in *Mycobacterium tuberculosis*. J. Bacteriol.
    183, 6119–6125 (2001). [[DOI](https://doi.org/10.1128/JB.183.20.6119-6125.2001)] [[PMC free article](/articles/PMC99691/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11567012/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=The%20alternative%20sigma%20factor%20SigH%20regulates%20major%20components%20of%20oxidative%20and%20heat%20stress%20responses%20in%20Mycobacterium%20tuberculosis&volume=183&publication_year=2001&pages=6119&pmid=11567012&doi=10.1128/JB.183.20.6119-6125.2001&)]
41. Mehra, S. & Kaushal, D. Functional genomics reveals extended roles of the Mycobacterium stress response factor σH. J. Bacteriol.
    191, 3965–3980 (2009). [[DOI](https://doi.org/10.1128/JB.00064-09)] [[PMC free article](/articles/PMC2698404/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19376862/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=Functional%20genomics%20reveals%20extended%20roles%20of%20the%20Mycobacterium%20stress%20response%20factor%20%CF%83H&volume=191&publication_year=2009&pages=3965&pmid=19376862&doi=10.1128/JB.00064-09&)]
42. Rustad, T. R., Harrell, M. I., Liao, R. & Sherman, D. R. The enduring hypoxic response of *Mycobacterium tuberculosis*. PLoS ONE
    3, e1502 (2008). [[DOI](https://doi.org/10.1371/journal.pone.0001502)] [[PMC free article](/articles/PMC2198943/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18231589/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=The%20enduring%20hypoxic%20response%20of%20Mycobacterium%20tuberculosis&volume=3&publication_year=2008&pages=e1502&pmid=18231589&doi=10.1371/journal.pone.0001502&)]
43. Dahl, J. L. et al. The role of RelMtb-mediated adaptation to stationary phase in long-term persistence of *Mycobacterium tuberculosis* in mice. Proc. Natl. Acad. Sci. USA
    100, 10026–10031 (2003). [[DOI](https://doi.org/10.1073/pnas.1631248100)] [[PMC free article](/articles/PMC187750/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12897239/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=The%20role%20of%20RelMtb-mediated%20adaptation%20to%20stationary%20phase%20in%20long-term%20persistence%20of%20Mycobacterium%20tuberculosis%20in%20mice&volume=100&publication_year=2003&pages=10026&pmid=12897239&doi=10.1073/pnas.1631248100&)]
44. KEGG Genome information available at http://www.genome.jp/kegg-bin/show\_organism?org=mmi. Accessed on 20 April, 2016.
45. Kacena, M. A. et al. Bacterial growth in space flight: logistic growth curve parameters for *Escherichia coli* and *Bacillus subtilis*. Appl. Microbiol. Biotechnol.
    51, 229–234 (1999). [[DOI](https://doi.org/10.1007/s002530051386)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10091330/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&title=Bacterial%20growth%20in%20space%20flight:%20logistic%20growth%20curve%20parameters%20for%20Escherichia%20coli%20and%20Bacillus%20subtilis&volume=51&publication_year=1999&pages=229&pmid=10091330&doi=10.1007/s002530051386&)]
46. Nomura, M. Regulation of ribosome biosynthesis in *Escherichia coli* and *Saccharomyces cerevisiae*: diversity and common principles. J. Bacteriol.
    181, 6857–6864 (1999). [[DOI](https://doi.org/10.1128/jb.181.22.6857-6864.1999)] [[PMC free article](/articles/PMC94158/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10559149/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=Regulation%20of%20ribosome%20biosynthesis%20in%20Escherichia%20coli%20and%20Saccharomyces%20cerevisiae:%20diversity%20and%20common%20principles&volume=181&publication_year=1999&pages=6857&pmid=10559149&doi=10.1128/jb.181.22.6857-6864.1999&)]
47. Betts, J. C., Lukey, P. T., Robb, L. C., McAdam, R. A. & Duncan, K. Evaluation of a nutrient starvation model of *Mycobacterium tuberculosis* persistence by gene and protein expression profiling. Mol. Microbiol.
    43, 717–731 (2002). [[DOI](https://doi.org/10.1046/j.1365-2958.2002.02779.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11929527/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&title=Evaluation%20of%20a%20nutrient%20starvation%20model%20of%20Mycobacterium%20tuberculosis%20persistence%20by%20gene%20and%20protein%20expression%20profiling&volume=43&publication_year=2002&pages=717&pmid=11929527&doi=10.1046/j.1365-2958.2002.02779.x&)]
48. Rodriguez, J. G. et al. Global adaptation to a lipid environment triggers the dormancy-related phenotype of *Mycobacterium tuberculosis*. mBio
    5, e01125–14 (2014). [[DOI](https://doi.org/10.1128/mBio.01125-14)] [[PMC free article](/articles/PMC4030484/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24846381/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=mBio&title=Global%20adaptation%20to%20a%20lipid%20environment%20triggers%20the%20dormancy-related%20phenotype%20of%20Mycobacterium%20tuberculosis&volume=5&publication_year=2014&pages=e01125&pmid=24846381&doi=10.1128/mBio.01125-14&)]
49. Pecsi, I. et al. Essentiality of succinate dehydrogenase in *Mycobacterium smegmatis* and its role in the generation of membrane potential under hypoxia. mBio
    5, e01093–14 (2014). [[DOI](https://doi.org/10.1128/mBio.01093-14)] [[PMC free article](/articles/PMC4145680/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25118234/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=mBio&title=Essentiality%20of%20succinate%20dehydrogenase%20in%20Mycobacterium%20smegmatis%20and%20its%20role%20in%20the%20generation%20of%20membrane%20potential%20under%20hypoxia&volume=5&publication_year=2014&pages=e01093&pmid=25118234&doi=10.1128/mBio.01093-14&)]
50. Geiman, D. E., Raghunand, T. R., Agarwal, N. & Bishai, W. R. Differential gene expression in response to exposure to antimycobacterial agents and other stress conditions among seven *Mycobacterium tuberculosis* whiB-like genes. Antimicrob. Agents Chemother.
    50, 2836–2841 (2006). [[DOI](https://doi.org/10.1128/AAC.00295-06)] [[PMC free article](/articles/PMC1538666/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16870781/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Differential%20gene%20expression%20in%20response%20to%20exposure%20to%20antimycobacterial%20agents%20and%20other%20stress%20conditions%20among%20seven%20Mycobacterium%20tuberculosis%20whiB-like%20genes&volume=50&publication_year=2006&pages=2836&pmid=16870781&doi=10.1128/AAC.00295-06&)]
51. Cumming, B. M. et al. The physiology and genetics of oxidative stress in mycobacteria. Microbiol. Spectr.
    2: MGM2-0019-2013; doi: 10.1128/microbiolspec.MGM2-0019-2013 (2014). [[DOI](https://doi.org/10.1128/microbiolspec.MGM2-0019-2013)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26103972/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Spectr.&title=The%20physiology%20and%20genetics%20of%20oxidative%20stress%20in%20mycobacteria&volume=2&publication_year=2014&pmid=26103972&doi=10.1128/microbiolspec.MGM2-0019-2013&)]
52. Schnappinger, D. et al. Transcriptional adaptation of *Mycobacterium tuberculosis* within macrophages: insights into the phagosomal environment. J. Exp. Med.
    198, 693–704 (2003). [[DOI](https://doi.org/10.1084/jem.20030846)] [[PMC free article](/articles/PMC2194186/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12953091/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Med.&title=Transcriptional%20adaptation%20of%20Mycobacterium%20tuberculosis%20within%20macrophages:%20insights%20into%20the%20phagosomal%20environment&volume=198&publication_year=2003&pages=693&pmid=12953091&doi=10.1084/jem.20030846&)]
53. Pettersson, B. M. F. et al. Comparative sigma factor-mRNA levels in *Mycobacterium marinum* under stress conditions and during host infection. PLoS ONE
    10, e0139823 (2015). [[DOI](https://doi.org/10.1371/journal.pone.0139823)] [[PMC free article](/articles/PMC4596819/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26445268/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Comparative%20sigma%20factor-mRNA%20levels%20in%20Mycobacterium%20marinum%20under%20stress%20conditions%20and%20during%20host%20infection&volume=10&publication_year=2015&pages=e0139823&pmid=26445268&doi=10.1371/journal.pone.0139823&)]
54. Sachdeva, P., Misra, R., Tyagi, A. K. & Singh, Y. The sigma factors of *Mycobacterium tuberculosis*: regulation of the regulators. FEBS J.
    277, 605–626 (2010). [[DOI](https://doi.org/10.1111/j.1742-4658.2009.07479.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19951358/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20J.&title=The%20sigma%20factors%20of%20Mycobacterium%20tuberculosis:%20regulation%20of%20the%20regulators&volume=277&publication_year=2010&pages=605&pmid=19951358&doi=10.1111/j.1742-4658.2009.07479.x&)]
55. Song, T., Dove, S. L., Lee, K. H. & Husson, R. N. RshA, an anti-sigma factor that regulates the activity of the mycobacterial stress response sigma factor SigH. Mol. Microbiol.
    50, 949–959 (2003). [[DOI](https://doi.org/10.1046/j.1365-2958.2003.03739.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/14617153/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&title=RshA,%20an%20anti-sigma%20factor%20that%20regulates%20the%20activity%20of%20the%20mycobacterial%20stress%20response%20sigma%20factor%20SigH&volume=50&publication_year=2003&pages=949&pmid=14617153&doi=10.1046/j.1365-2958.2003.03739.x&)]
56. Park, S. T., Kang, C.-M. & Husson, R. N. Regulation of the SigH stress response regulon by an essential protein kinase in *Mycobacterium tuberculosis*. Proc. Natl. Acad. Sci. USA
    105, 13105–13110 (2008). [[DOI](https://doi.org/10.1073/pnas.0801143105)] [[PMC free article](/articles/PMC2529121/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18728196/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Regulation%20of%20the%20SigH%20stress%20response%20regulon%20by%20an%20essential%20protein%20kinase%20in%20Mycobacterium%20tuberculosis&volume=105&publication_year=2008&pages=13105&pmid=18728196&doi=10.1073/pnas.0801143105&)]
57. Manganelli, R., Voskuil, M. I., Schoolnik, G. K. & Smith, I. The *Mycobacterium tuberculosis* ECF sigma factor σE: role in global gene expression and survival in macrophages. Mol. Microbiol.
    41, 423–437 (2001). [[DOI](https://doi.org/10.1046/j.1365-2958.2001.02525.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11489128/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&title=The%20Mycobacterium%20tuberculosis%20ECF%20sigma%20factor%20%CF%83E:%20role%20in%20global%20gene%20expression%20and%20survival%20in%20macrophages&volume=41&publication_year=2001&pages=423&pmid=11489128&doi=10.1046/j.1365-2958.2001.02525.x&)]
58. Lee, M. H., Pascopella, L., Jacobs, W. R. & Hatfull, G. F. Site-specific integration of mycobacteriophage L5: Integration-proficient vectors for *Mycobacterium smegmatis*, *Mycobacterium tuberculosis*, and bacilli Calmette-Guérin. Proc. Natl. Acad. Sci. USA
    88, 3111–3115 (1991). [[DOI](https://doi.org/10.1073/pnas.88.8.3111)] [[PMC free article](/articles/PMC51395/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/1901654/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Site-specific%20integration%20of%20mycobacteriophage%20L5:%20Integration-proficient%20vectors%20for%20Mycobacterium%20smegmatis,%20Mycobacterium%20tuberculosis,%20and%20bacilli%20Calmette-Gu%C3%A9rin&volume=88&publication_year=1991&pages=3111&pmid=1901654&doi=10.1073/pnas.88.8.3111&)]
59. Mutoji, K. N. & Ennis, D. G. Expression of common fluorescent reporters may modulate virulence for *Mycobacterium marinum*: dramatic attenuation results from Gfp over-expression. Comp. Biochem. Physiol. C Toxicol. Pharmacol.
    155, 39–48 (2012). [[DOI](https://doi.org/10.1016/j.cbpc.2011.05.011)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21658470/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Comp.%20Biochem.%20Physiol.%20C%20Toxicol.%20Pharmacol.&title=Expression%20of%20common%20fluorescent%20reporters%20may%20modulate%20virulence%20for%20Mycobacterium%20marinum:%20dramatic%20attenuation%20results%20from%20Gfp%20over-expression&volume=155&publication_year=2012&pages=39&pmid=21658470&doi=10.1016/j.cbpc.2011.05.011&)]
60. Soni, A. et al. Conservation of the low-shear modeled microgravity response in enterobacteriaceae and analysis of the *trp* genes in this response. Open Microbiol. J.
    8, 51–58 (2014). [[DOI](https://doi.org/10.2174/1874285801408010051)] [[PMC free article](/articles/PMC4085587/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25006354/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Open%20Microbiol.%20J.&title=Conservation%20of%20the%20low-shear%20modeled%20microgravity%20response%20in%20enterobacteriaceae%20and%20analysis%20of%20the%20trp%20genes%20in%20this%20response&volume=8&publication_year=2014&pages=51&pmid=25006354&doi=10.2174/1874285801408010051&)]
61. Ward, J. F., Evans, J. W., Limoli, C. L. & Calabro-Jones, P. M. Radiation and hydrogen peroxide induced free radical damage to DNA. Br. J. Cancer
    55(suppl. VIII): 105–112 (1987). [[PMC free article](/articles/PMC2149485/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/2820457/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Cancer&title=Radiation%20and%20hydrogen%20peroxide%20induced%20free%20radical%20damage%20to%20DNA&volume=55&publication_year=1987&pages=105&pmid=2820457&)]
62. Harrison, L. in DNA Repair and Cancer (eds Madhusudan S. & Wilson D. M. III) Ch. 4 (Science Publishers/CRC Press, 2013) 92–136.
63. Kalyanaraman, B. Teaching the basics of redox biology to medical and graduate students: oxidants, antioxidants and disease mechanisms. Redox Biol.
    1, 244–257 (2013). [[DOI](https://doi.org/10.1016/j.redox.2013.01.014)] [[PMC free article](/articles/PMC3757692/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24024158/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Redox%20Biol.&title=Teaching%20the%20basics%20of%20redox%20biology%20to%20medical%20and%20graduate%20students:%20oxidants,%20antioxidants%20and%20disease%20mechanisms&volume=1&publication_year=2013&pages=244&pmid=24024158&doi=10.1016/j.redox.2013.01.014&)]
64. Termini, J. Hydroperoxide-induced DNA damage and mutations. Mutat. Res.
    450, 107–124 (2000). [[DOI](https://doi.org/10.1016/s0027-5107%2800%2900019-1)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10838137/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutat.%20Res.&title=Hydroperoxide-induced%20DNA%20damage%20and%20mutations&volume=450&publication_year=2000&pages=107&pmid=10838137&doi=10.1016/s0027-5107(00)00019-1&)]
65. Gouzy, A. et al. *Mycobacterium tuberculosis* exploits asparagine to assimilate nitrogen and resist acid stress during infection. PLoS Pathog.
    10, e1003928 (2014). [[DOI](https://doi.org/10.1371/journal.ppat.1003928)] [[PMC free article](/articles/PMC3930563/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24586151/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&title=Mycobacterium%20tuberculosis%20exploits%20asparagine%20to%20assimilate%20nitrogen%20and%20resist%20acid%20stress%20during%20infection&volume=10&publication_year=2014&pages=e1003928&pmid=24586151&doi=10.1371/journal.ppat.1003928&)]
66. Mehra, S. et al. The *Mycobacterium tuberculosis* stress response factor SigH is required for bacterial burden as well as immunopathology in primate lungs. J. Infect. Dis.
    205, 1203–1213 (2012). [[DOI](https://doi.org/10.1093/infdis/jis102)] [[PMC free article](/articles/PMC3308902/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22402035/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Infect.%20Dis.&title=The%20Mycobacterium%20tuberculosis%20stress%20response%20factor%20SigH%20is%20required%20for%20bacterial%20burden%20as%20well%20as%20immunopathology%20in%20primate%20lungs&volume=205&publication_year=2012&pages=1203&pmid=22402035&doi=10.1093/infdis/jis102&)]
67. Ghosh, P., Wu, C.-W. & Talaat, A. M. Key role for the alternative sigma factor, SigH, in the intracellular life of *Mycobacterium avium* subsp. *paratuberculosis* during macrophage infection. Infect. Immun.
    81, 2242–2257 (2013). [[DOI](https://doi.org/10.1128/IAI.01273-12)] [[PMC free article](/articles/PMC3676012/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23569115/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&title=Key%20role%20for%20the%20alternative%20sigma%20factor,%20SigH,%20in%20the%20intracellular%20life%20of%20Mycobacterium%20avium%20subsp.%20paratuberculosis%20during%20macrophage%20infection&volume=81&publication_year=2013&pages=2242&pmid=23569115&doi=10.1128/IAI.01273-12&)]
68. Rustad, T. R., Roberts, D. M., Liao, R. P. & Sherman, D. R. Isolation of mycobacterial RNA. Methods Mol. Biol.
    465, 13–22 (2008). [[DOI](https://doi.org/10.1007/978-1-59745-207-6_2)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20560069/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&title=Isolation%20of%20mycobacterial%20RNA&volume=465&publication_year=2008&pages=13&pmid=20560069&doi=10.1007/978-1-59745-207-6_2&)]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

Supplementary Tables

[Click here for additional data file.](/articles/instance/5515531/bin/npjmgrav201638-s1.doc) (208.5KB, doc)

Supplementary Figures

[Click here for additional data file.](/articles/instance/5515531/bin/npjmgrav201638-s2.ppt) (1.7MB, ppt)

Supplementary Excel Worksheet 1

[Click here for additional data file.](/articles/instance/5515531/bin/npjmgrav201638-s3.pdf) (749.3KB, pdf)

Supplementary Excel Worksheet 2

[Click here for additional data file.](/articles/instance/5515531/bin/npjmgrav201638-s4.pdf) (500.7KB, pdf)