<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5523082/"/><title>Spatial regulation of a common precursor from two distinct genes generates metabolite diversity - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Chem Sci</button></div>. 2015 Jul 13;6(10):5913–5921. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1039/c5sc01058f" rel="noopener noreferrer" target="_blank">10.1039/c5sc01058f</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Spatial regulation of a common precursor from two distinct genes generates metabolite diversity<sup><a class="usa-link" href="#fn1">†</a></sup><sup>
</sup>
</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20CJ%22%5BAuthor%5D"><span class="name western">Chun-Jun Guo</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Chun-Jun Guo</span></h3>
<div class="p">
<sup>a</sup> Department of Pharmacology and Pharmaceutical Sciences , School of Pharmacy , University of Southern California , Los Angeles , CA 90089 , USA . Email: clayw@usc.edu
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20CJ%22%5BAuthor%5D"><span class="name western">Chun-Jun Guo</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20WW%22%5BAuthor%5D"><span class="name western">Wei-Wen Sun</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wei-Wen Sun</span></h3>
<div class="p">
<sup>a</sup> Department of Pharmacology and Pharmaceutical Sciences , School of Pharmacy , University of Southern California , Los Angeles , CA 90089 , USA . Email: clayw@usc.edu
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20WW%22%5BAuthor%5D"><span class="name western">Wei-Wen Sun</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bruno%20KS%22%5BAuthor%5D"><span class="name western">Kenneth S Bruno</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Kenneth S Bruno</span></h3>
<div class="p">
<sup>b</sup> Chemical and Biological Process Development Group , Energy and Environment Directorate , Pacific Northwest National Laboratory , Richland , WA 99352 , USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bruno%20KS%22%5BAuthor%5D"><span class="name western">Kenneth S Bruno</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Oakley%20BR%22%5BAuthor%5D"><span class="name western">Berl R Oakley</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Berl R Oakley</span></h3>
<div class="p">
<sup>c</sup> Department of Molecular Biosciences , University of Kansas , Lawrence , KS 66045 , USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Oakley%20BR%22%5BAuthor%5D"><span class="name western">Berl R Oakley</span></a>
</div>
</div>
<sup>c</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Keller%20NP%22%5BAuthor%5D"><span class="name western">Nancy P Keller</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Nancy P Keller</span></h3>
<div class="p">
<sup>d</sup> Department of Medical Microbiology and Immunology , University of Wisconsin-Madison , Madison , WI 53706 , USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Keller%20NP%22%5BAuthor%5D"><span class="name western">Nancy P Keller</span></a>
</div>
</div>
<sup>d</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CCC%22%5BAuthor%5D"><span class="name western">Clay C C Wang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Clay C C Wang</span></h3>
<div class="p">
<sup>a</sup> Department of Pharmacology and Pharmaceutical Sciences , School of Pharmacy , University of Southern California , Los Angeles , CA 90089 , USA . Email: clayw@usc.edu
</div>
<div class="p">
<sup>e</sup> Department of Chemistry , College of Letters, Arts, and Sciences , University of Southern California , Los Angeles , CA 90089 , USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CCC%22%5BAuthor%5D"><span class="name western">Clay C C Wang</span></a>
</div>
</div>
<sup>a,</sup><sup>e,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="affa">
<sup>a</sup> Department of Pharmacology and Pharmaceutical Sciences , School of Pharmacy , University of Southern California , Los Angeles , CA 90089 , USA . Email: clayw@usc.edu
</div>
<div id="affb">
<sup>b</sup> Chemical and Biological Process Development Group , Energy and Environment Directorate , Pacific Northwest National Laboratory , Richland , WA 99352 , USA </div>
<div id="affc">
<sup>c</sup> Department of Molecular Biosciences , University of Kansas , Lawrence , KS 66045 , USA </div>
<div id="affd">
<sup>d</sup> Department of Medical Microbiology and Immunology , University of Wisconsin-Madison , Madison , WI 53706 , USA </div>
<div id="affe">
<sup>e</sup> Department of Chemistry , College of Letters, Arts, and Sciences , University of Southern California , Los Angeles , CA 90089 , USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2015 Mar 24; Accepted 2015 Jul 12; Issue date 2015 Oct 1.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>This journal is © The Royal Society of Chemistry 2015</div>
<p>This is an Open Access article distributed under the terms of the Creative Commons Attribution 3.0 Unported License (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://creativecommons.org/licenses/by/3.0/" rel="noopener noreferrer" target="_blank">http://creativecommons.org/licenses/by/3.0/</a>) which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC5523082  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28791090/">28791090</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><hr class="headless"/>
<p>
<a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5523082_c5sc01058f-ga.jpg" target="_blank"><img alt="graphic file with name c5sc01058f-ga.jpg" class="graphic zoom-in" height="167" id="ga" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f768/5523082/90941d11d5a5/c5sc01058f-ga.jpg" width="800"/></a>We have demonstrated that spatial regulation of the same product from two distinct genes generates metabolite diversity.</p></section><section class="abstract" id="abstract2"><h2>Abstract</h2>
<p>In secondary metabolite biosynthesis, core synthetic genes such as polyketide synthase genes usually encode proteins that generate various backbone precursors. These precursors are modified by other tailoring enzymes to yield a large variety of different secondary metabolites. The number of core synthesis genes in a given species correlates, therefore, with the number of types of secondary metabolites the organism can produce. In our study, heterologous expression of all the <em>A. terreus</em> NRPS-like genes showed that two NRPS-like proteins, encoded by <em>atmelA</em> and <em>apvA</em>, release the same natural product, aspulvinone E. In hyphae this compound is converted to aspulvinones whereas in conidia it is converted to melanin. The genes are expressed in different tissues and this spatial control is probably regulated by their own specific promoters. Comparative genomics indicates that <em>atmelA</em> and <em>apvA</em> might share a same ancestral gene and the gene <em>apvA</em> is located in a highly conserved region in <em>Aspergillus</em> species that contains genes coding for life-essential proteins. Our data reveal the first case in secondary metabolite biosynthesis in which the tissue specific production of a single compound directs it into two separate pathways, producing distinct compounds with different functions. Our data also reveal that a single <em>trans</em>-prenyltransferase, AbpB, prenylates two substrates, aspulvinones and butyrolactones, revealing that genes outside of contiguous secondary metabolism gene clusters can modify more than one compound thereby expanding metabolite diversity. Our study raises the possibility of incorporation of spatial, cell-type specificity in expression of secondary metabolites of biological interest and provides new insight into designing and reconstituting their biosynthetic pathways.</p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Filamentous fungi, such as species found within the genus <em>Aspergillus</em>, are well known producers of secondary metabolites (SMs) with interesting biological activities. The genome sequencing of <em>Aspergillus</em> species revealed that the number of putative SM genes greatly exceeds the number of identified SMs, suggesting many novel types of SMs still remain to be discovered.<sup><a aria-describedby="cit1" class="usa-link" href="#cit1">1</a></sup> In general, these SM genes are clustered in the genome.<sup><a aria-describedby="cit2" class="usa-link" href="#cit2">2</a></sup> A typical cluster contains one key gene required for the synthesis of the precursor skeleton. These genes usually encode large multidomain enzymes belonging to the polyketide synthase (PKS) or non-ribosomal peptide synthetase (NRPS) class.<sup><a aria-describedby="cit3" class="usa-link" href="#cit3">3</a></sup> Other adjacent genes encode enzymes that are involved in tailoring modifications, transport of the product, or co-regulation of the cluster genes.<sup><a aria-describedby="cit4" class="usa-link" href="#cit4">4</a></sup> In a given fungal species, the SM arsenal is usually governed by the number of key synthetic genes in the genome, and this diversity is multiplied by various tailoring enzymes that give rise to a considerable number of different natural products.<sup><a aria-describedby="cit5" class="usa-link" href="#cit5">5</a></sup> For instance, the fungus <em>A. nidulans</em> has 27 PKS genes, 12 NRPS genes, 14 NRPS-like genes and one PKS/NRPS hybrid gene.<sup><a aria-describedby="cit6" class="usa-link" href="#cit6">6</a></sup> In a previous study, Ahuja <em>et al.</em> systematically characterized the polyketide products of eight non reducing PKSs (NR-PKSs) in <em>A. nidulans</em>.<sup><a aria-describedby="cit5" class="usa-link" href="#cit5">5</a></sup> In combination with the six previously characterized NR-PKS genes, the study demonstrated that the 14 NR-PKS genes in <em>A. nidulans</em> could be divided into seven groups based on phylogenetic analysis. More importantly, each of these NR-PKSs produces a unique product which can be modified by other tailoring enzymes and incorporate into various SM biosynthetic pathways, indicating the potential of <em>A. nidulans</em> for biosynthesizing a large variety of different PKS-derived SMs.<sup><a aria-describedby="cit5" class="usa-link" href="#cit5">5</a></sup>
</p>
<p>Recently, application of an efficient gene targeting system enabled us to link two NRPS-like genes <em>apvA</em> and <em>btyA</em> to their corresponding SMs, aspulvinones and butyrolactones, respectively, in <em>A. terreus</em> strain NIH 2624.<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup> In this study, we demonstrated that one NRPS-like gene, <em>atmelA</em>, is involved in the synthesis of a brown conidial melanin in this fungus.<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup> Compared to typical NRPSs, NRPS-like genes encode single module (A-T-TE) proteins missing the condensation (C) domain.<sup><a aria-describedby="cit3" class="usa-link" href="#cit3">3</a>,<a aria-describedby="cit8" class="usa-link" href="#cit8">8</a></sup> The adenylation (A) domain is responsible for aryl acid substrate recognition and activation. The activated substrate is loaded onto the thiolation (T) domain. The thioesterase (TE) domain is suggested to be involved in the condensation and releasing of the final product.<sup><a aria-describedby="cit8" class="usa-link" href="#cit8">8</a></sup> Therefore, the diversity of NRPS-like products is expanded by a combination of different A domains (substrate selection and activation) and TE domains (various cyclization and release mechanisms). The <em>A. terreus</em> genome contains 14 NRPS-like genes with predicted A-T-TE or similar domain architecture and the SM products for the majority of these are unknown.</p>
<p>Here we report our efforts to systematically characterize the products of these 14 NRPS-like genes. We heterologously expressed each gene in <em>A. nidulans</em> under the control of the inducible <em>alcA</em> promoter. Surprisingly, our study reveals that two NRPS-like genes, <em>apvA</em> and <em>atmelA</em>, are responsible for the formation of the same intermediate, aspulvinone E (<strong>1</strong>, <a class="usa-link" href="#fig1">Fig. 1A</a>). The aspulvinone E produced by AtmelA is further modified by a tyrosinase, AtmelB, and is incorporated into the brown melanin biosynthesis pathway in <em>A. terreus</em> (<a class="usa-link" href="#fig1">Fig. 1A</a>). In parallel, the aspulvinone E synthesized by ApvA is further prenylated by a <em>trans</em>-prenyltransferase (AbpB) to produce aspulvinones (<a class="usa-link" href="#fig1">Fig. 1B</a>). AbpB also prenylates butyrolactones, and this reveals that modifying genes outside of specific clusters can be responsible for modifying compounds from more than one SM gene cluster, thus expanding the diversity of SMs produced by an organism (<a class="usa-link" href="#fig1">Fig. 1B</a>).</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">Fig. 1. (A) AtmelA and ApvA may synthesize the identical natural product <strong>1</strong> accumulated in different fungal tissues. (B) A <em>trans</em>-prenyltransferase, AbpB, prenylates two substrates, aspulvinones and butyrolactones.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5523082_c5sc01058f-f1.jpg" target="_blank"><img alt="Fig. 1" class="graphic zoom-in" height="588" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f768/5523082/6a4256175c8a/c5sc01058f-f1.jpg" width="741"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></figure><p>Our results suggest that the promoter of <em>apvA</em> drives expression in hyphae resulting in the production of aspulvinone E (<strong>1</strong>), which is modified to produce aspulvinone variants. The <em>atmelA</em> promoter drives expression in conidia, also resulting in the production of aspulvinone E (<strong>1</strong>) but in this cell type it is converted to melanin. Further genetic analysis of <em>apvA</em> and <em>atmelA</em> indicates that these two genes may share a common ancestral gene. The gene <em>apvA</em>, which may result from duplication of the ancestral gene, is inserted in a genomic region consisting of genes that codes for life-essential proteins. Our study suggests an unprecedented pathway for conidial pigment biosynthesis in <em>A. terreus</em> that incorporates an NRPS-like product aspulvinone E (<strong>1</strong>) as its substrate (<a class="usa-link" href="#fig1">Fig. 1</a>). Our data also demonstrated how the SM diversity can be expanded by (1) allocating the same natural product in different fungal compartments to produce molecules with different functions; (2) encoding tailoring enzyme that is capable of chemically modifying more than one type of SMs.</p></section><section id="sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="sec3"><h3 class="pmc_sec_title">Heterologous expression of the NRPS-like genes <em>apvA</em> and <em>atmelA</em> in <em>A. nidulans</em> both result in aspulvinone E production</h3>
<p>Using a recently reported efficient heterologous expression (HE) system in <em>A. nidulans</em>,<sup><a aria-describedby="cit9" class="usa-link" href="#cit9">9</a></sup> the individual NRPS-like genes identified in the <em>A. terreus</em> genome were expressed at either the <em>wA</em> or <em>yA</em> locus of <em>A. nidulans</em> under regulation of the inducible <em>alcA</em> promoter (Fig. S1A<a class="usa-link" href="#fn1">†</a>). In a previous study, targeted deletion of <em>apvA</em> depleted production of aspulvinones in <em>A. terreus</em> indicating that this NRPS-like gene is responsible for the biosynthesis of the aspulvinone core.<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup> As expected, heterologous expression of <em>apvA</em> results in accumulation of aspulvinone E (<strong>1</strong>) which is speculated to be the first intermediate in the aspulvinone pathway (<a class="usa-link" href="#fig1">Fig. 1</a> and <a class="usa-link" href="#fig2">2A and B</a>).<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup> Unexpectedly, our HE experiments revealed that AtmelA produces the same compound (<a class="usa-link" href="#fig2">Fig. 2A and B</a>). The gene <em>apvA</em> is responsible for aspulvinone biosynthesis while <em>atmelA</em> is involved in the synthesis of the brown conidial pigment.<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup> The brown conidial melanin is still produced in the <em>apvA</em> deletant strain as shown in <a class="usa-link" href="#fig3">Fig. 3A</a>. In contrast, deletion of <em>atmelA</em> generates an albino mutant that is still capable of synthesizing aspulvinones.<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup> Together, these data reveal that despite having the same activity (<em>i.e.</em> synthesis of aspulvinone E), ApvA and AtmelA function in different roles in the fungus.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">Fig. 2. (A) Compounds related to this study. (B) HPLC profiles of extracts of HE strains as detected by total scan UV.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5523082_c5sc01058f-f2.jpg" target="_blank"><img alt="Fig. 2" class="graphic zoom-in" height="256" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f768/5523082/ec1a7f0ee243/c5sc01058f-f2.jpg" width="743"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">Fig. 3. (A) Phenotype of <em>A. terreus</em> wild type and other <em>A. terreus</em> mutant strains growing on LCMM for 5 days. Morphological change of the fungal conidia can be observed if the deleted genes are involved in the biosynthesis of the brown conidial pigment. Total extracts (B), conidial extracts (C) and hyphal extracts (D) of <em>A. terreus</em> wild type and mutant strains as detected by UV at 370 nm. Aspulvinone related natural products are labeled in red. In B–D, the black box represents the top layer agar as described in tissue-specific extraction. The green box represents the bottom layer agar in tissue-specific extraction. The numbering of the peaks corresponds to the natural products shown in <a class="usa-link" href="#fig2">Fig. 2A</a>. The aspulvinone E (<strong>1</strong>) and its related natural products can be detected in LC/MS traces Bi, Bii, Biii, Ciii, Di, Dii, and Diii. Trace amount of <strong>1</strong> identified in D iii is due to the diffusion of this compound synthesized in the conidia of the mutant strain. *This metabolite is related with aspulvinones according to its UV absorption and MS spectrum.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5523082_c5sc01058f-f3.jpg" target="_blank"><img alt="Fig. 3" class="graphic zoom-in" height="1445" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f768/5523082/62ecebaad9dc/c5sc01058f-f3.jpg" width="783"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></figure></section><section id="sec4"><h3 class="pmc_sec_title">Aspulvinone E occurs as a precursor in both aspulvinone and melanin pathways</h3>
<p>To test our hypothesis that the aspulvinone E is an intermediate in two different pathways, the aspulvinone pathway and melanin pathway, we wished to delete the first tailoring gene in each pathway to accumulate the precursor produced by the gene responsible for the backbone metabolite. Since we have established the genetic linkage between <em>apvA</em> and aspulvinones,<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup> we set out to identify the first tailoring enzyme, which we presumed was responsible for prenylating <strong>1</strong> to give <strong>2</strong> (<a class="usa-link" href="#fig2">Fig. 2A</a>). However, we could not locate a prenyl transferase (PT) gene proximal to <em>apvA</em>.<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup> We then targeted each of the putative PT genes in the <em>A. terreus</em> genome for deletion (<a class="usa-link" href="#fig4">Fig. 4</a>). The individual genes were knocked out using fragments created by fusion PCR.<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup> The SM profiles of the correct mutants were examined by LC-MS, and of the 11 different mutants, only the ATEG_01730.1Δ strain accumulated aspulvinone E (<strong>1</strong>) (<a class="usa-link" href="#fig4">Fig. 4</a>). Unexpectedly, removal of ATEG_01730.1 leads to the accumulation of butyrolactone II (<strong>5</strong>) as well (<a class="usa-link" href="#fig4">Fig. 4</a>). A previous study showed that the NRPS-like gene <em>btyA</em> is responsible for the biosynthesis of butyrolactone core.<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup> The expression of three genes, <em>apvA</em>, <em>btyA</em>, and <em>abpB</em> were also analyzed using real-time quantitative reverse transcription PCR (qRT-PCR). (Fig. S2C<a class="usa-link" href="#fn1">†</a>) Our data showed that these three genes are co-expressed under aspulvinone and butyrolactone producing conditions. These pieces of evidence suggest that this single PT is responsible for the prenylation of two different metabolites, aspulvinone E (<strong>1</strong>) and butyrolactone II (<strong>5</strong>). Thus, we name the gene <em>a</em> (aspulvinone) <em>b</em> (butyrolactone) <em>p</em> (PT) <em>B</em>.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Fig. 4. HPLC profiles of extracts of <em>A. terreus</em> prenyltransferase (PT) genes deletants as detected by UV at 330 nm and 370 nm. The numbering of the peaks corresponds to the natural products shown in <a class="usa-link" href="#fig2">Fig. 2A</a>. Compounds <strong>3</strong> and <strong>4</strong> are prenylated derivatives of compound <strong>5</strong>. The “*” compound is an aspulvinone derivative according to its UV absorption spectrum. ATXXXXX is abbreviated for “ATEG_XXXXX.1”. Deletion of the gene ATEG_00702.1, a homolog of <em>tdiB</em> involved in the asterriquinone biosynthesis, was not achieved due to the unsuccessful PCR amplification of its flanking region.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5523082_c5sc01058f-f4.jpg" target="_blank"><img alt="Fig. 4" class="graphic zoom-in" height="337" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f768/5523082/edbb909a4dea/c5sc01058f-f4.jpg" width="708"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></figure><p>Definitively establishing the role of AtmelA in the synthesis of aspulvinone E (<strong>1</strong>) required additional gene deletion experiments. We first deleted the gene <em>apvA</em> using the direct repeat (DR) strategy<sup><a aria-describedby="cit10" class="usa-link" href="#cit10">10</a></sup> followed by the recycling of the <em>AfpyrG</em> marker (Fig. S1B<a class="usa-link" href="#fn1">†</a>). This ensures that aspulvinone E (<strong>1</strong>) identified in the later mutants is not from ApvA. However, limited information is available about the biosynthesis of the brown conidial melanin in <em>A. terreus</em>. Our study suggests that the melanin pathway originates from the NRPS-like product aspulvinone E (<strong>1</strong>), which is unlike the two main precursors [di-hydroxyphenylalanin (DOPA) and dihydroxynaphthalene (DHN)] of currently known melanins.<sup><a aria-describedby="cit11" class="usa-link" href="#cit11">11</a></sup> The biosynthesis of DOPA melanin starts with a tyrosine that is oxidized to either DOPA or dopaquinone (DAQ) by a tyrosinase.<sup><a aria-describedby="cit11" class="usa-link" href="#cit11">11</a></sup> Considering that aspulvinone E (<strong>1</strong>) shares the same phenol moiety as tyrosine, we speculate that a similar tyrosinase might catalyze the hydroxylation of <strong>1</strong> at the <em>ortho</em> position to give a dihydroxylated intermediate that becomes incorporated into the brown melanin (<a class="usa-link" href="#fig1">Fig. 1</a>). Following the cluster paradigm, we examined the genes surrounding <em>atmelA</em> and identified one gene ATEG_03564.1 (<em>atmelB</em>) that encodes a putative tyrosinase. Removal of this gene in the <em>apvA</em>Δ background changed the strain's phenotype: the brown melanin was no longer synthesized and the conidia became bright yellow (<a class="usa-link" href="#fig3">Fig. 3A</a>). As expected, the yellow material is aspulvinone E (<strong>1</strong>), which accumulated in the metabolite profiles of the <em>apvA</em> and <em>atmelB</em> double deletion strain (<a class="usa-link" href="#fig3">Fig. 3B</a>). We next deleted the gene <em>atmelA</em> in the double <em>apvA</em>Δ, <em>atmelB</em>Δ strain. The LC-MS profile of this triple deletion mutant shows the abrogation of the aspulvinone E (<strong>1</strong>) which had reappeared in the double mutant (<a class="usa-link" href="#fig3">Fig. 3B</a>), in accord with our HE results that AtmelA was also capable of producing Aspulvinone E (<strong>1</strong>).</p></section><section id="sec5"><h3 class="pmc_sec_title">Aspulvinone E (<strong>1</strong>) produced by ApvA and AtmelA accumulates in different fungal tissues</h3>
<p>Our studies indicate that ApvA and AtmelA synthesize the same product aspulvinone E (<strong>1</strong>). We next asked how the fungus is able to allocate the same chemical synthesized by different proteins into their own specific pathway without cross-interference. Previous literature reports have shown that the production of SMs and/or their precursors can be specific to both cellular organelle and fungal tissue.<sup><a aria-describedby="cit12" class="usa-link" href="#cit12">12</a>,<a aria-describedby="cit13" class="usa-link" href="#cit13">13</a></sup> We speculated that the aspulvinone E (<strong>1</strong>) from the two genes might be generated in different fungal tissues. Since removal of <em>atmelA</em> or <em>atmelB</em> changes the phenotype of <em>A. terreus</em> conidia (<a class="usa-link" href="#fig3">Fig. 3A</a>), it is likely that aspulvinone E (<strong>1</strong>) from AtmelA might be produced in conidia. Likewise aspulvinones, derived from <strong>1</strong> that is produced by ApvA, might be produced inside the hyphae and secreted into the medium.</p>
<p>To test this hypothesis, we performed tissue-specific extraction<sup><a aria-describedby="cit12" class="usa-link" href="#cit12">12</a>,<a aria-describedby="cit13" class="usa-link" href="#cit13">13</a></sup> to reinvestigate the SM profiles of the four strains ((1) wild type; (2) <em>abpB</em>Δ; (3) <em>apvA</em>Δ, <em>atmelB</em>Δ; (4) <em>apvA</em>Δ, <em>atmelB</em>Δ, <em>atmelA</em>Δ). Fungal cultures were divided into three fractions: (1) conidial (mostly conidia and minor conidiophore), (2) top layer agar (mixture of conidiophore, vegetative hyphae, minor invasive hyphae), (3) lower layer agar (mostly invasive hyphae) (<a class="usa-link" href="#fig3">Fig. 3C and D</a>). Compared to the SM profiles of total extracts (<a class="usa-link" href="#fig3">Fig. 3B</a>), extraction of the conidia showed the accumulation of <strong>1</strong> only in the <em>apvA</em>Δ, <em>atmelB</em>Δ strain (<a class="usa-link" href="#fig3">Fig. 3C</a>), not in strains carrying <em>abpB</em>Δ or <em>atmelA</em>Δ. This result indicates that aspulvinone E (<strong>1</strong>) from AtmelA is produced in conidia. Extraction of the hyphal (mostly invasive) fraction showed the production of <strong>1</strong> in the <em>abpB</em>Δ strain and compound <strong>2</strong> in wild type, indicating that aspulvinone E (<strong>1</strong>) from ApvA and its derivative <strong>2</strong> are specifically produced in hyphae.</p></section><section id="sec6"><h3 class="pmc_sec_title">Exchanging <em>atmelA</em> with <em>apvA</em>, under control of the <em>atmelA</em> promoter, restores melanin production in <em>A. terreus</em>
</h3>
<p>Next, we asked about the molecular mechanism for regulating the tissue-specific production of <strong>1</strong>. We first examined the expression profiling of <em>atmelA</em> and <em>apvA</em> in different tissues using Real-Time qRT-PCR (Fig. S2, see ESI<a class="usa-link" href="#fn1">†</a> for experimental details). As expected, the gene <em>atmelA</em> is specifically expressed in conidia compartment (Fig. S2B<a class="usa-link" href="#fn1">†</a>) while the gene <em>apvA</em> is locally expressed in hyphae (Fig. S2C<a class="usa-link" href="#fn1">†</a>). We then probed this question by determining whether ApvA could replace AtmelA in brown melanin biosynthesis. We assumed that tailoring enzymes like AtmelB could still recognize aspulvinone E (<strong>1</strong>) generated from either ApvA or AtmelA. Another study suggests that the products of two tubulin genes, <em>benA</em> and <em>tubC</em>, are functionally interchangeable. The method they used was to disrupt <em>benA</em> and then put <em>tubC</em> under control of the <em>benA</em> promoter.<sup><a aria-describedby="cit14" class="usa-link" href="#cit14">14</a>,<a aria-describedby="cit15" class="usa-link" href="#cit15">15</a></sup> Herein we implemented a similar strategy by replacing the coding region of <em>atmelA</em> with that of <em>apvA</em>, placing <em>apvA</em> under control of the <em>atmelA</em> promoter (<em>atmelAp</em>) in the <em>apvA</em>Δ background (Fig. S1Ci<a class="usa-link" href="#fn1">†</a>). As shown in <a class="usa-link" href="#fig5">Fig. 5A</a>, the mutant stain (<em>apvA</em>Δ, <em>atmelA</em>::<em>apvA</em>) produces brownish conidia indicating that the brown conidial pigment is produced in the mutant strain (<em>apvA</em>Δ, <em>atmelA</em>::<em>apvA</em>). As anticipated, the production of aspulvinones (<strong>1</strong> or <strong>2</strong>) were not detected (Fig. S3<a class="usa-link" href="#fn1">†</a>). The pigment is not produced as much as in the wild type (<a class="usa-link" href="#fig5">Fig. 5A</a>) probably because <em>atmelAp</em>, after all, is not the native promoter of ApvA. Quantitative analysis of their expression using Real-Time qRT-PCR reveals that the expression level of <em>apvA</em>, which is under the control of atmelA promoter, is lower than that of <em>atmelA</em>. (Fig. S2B<a class="usa-link" href="#fn1">†</a>) This could be one of the reasons for reduced production of the conidial melanin in the mutant strain (<em>apvA</em>Δ, <em>atmelA</em>::<em>apvA</em>). But more importantly, our experiment shows that compound <strong>1</strong> from <em>apvA</em> can be incorporated into the melanin pathway when <em>apvA</em> is regulated by <em>atmelAp</em>. As mentioned earlier, the production of <strong>1</strong> from <em>atmelA</em> is conidia-specific. Thus, this experiment shows that the product of ApvA can also be produced inside conidia and incorporated into melanin, suggesting that the tissue specific allocation of their products may be due to cell-type specific expression of the genes <em>atmelA</em> and <em>apvA</em> under regulation of their specific promoters.</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">Fig. 5. (A) Phenotype of the <em>A. terreus</em> mutant strain <em>apvA</em>Δ, <em>atmelA</em>::<em>apvA</em> (the <em>A. terreus</em> wild type is used as positive control that produces melanin, the <em>atmelA</em>Δ strain is used as negative control). (B) Replacing <em>atmelA</em> and <em>apvA</em> with <em>gfp</em>. Top column: conidia. Bottom column: hyphae. Top row: GFP. Middle row: differential interference contrast (DIC). Bottom row: merged GFP.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5523082_c5sc01058f-f5.jpg" target="_blank"><img alt="Fig. 5" class="graphic zoom-in" height="1858" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f768/5523082/868904641894/c5sc01058f-f5.jpg" width="783"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></figure></section><section id="sec7"><h3 class="pmc_sec_title">Tissue specific expression of <em>gfp</em> occurs when the coding sequence of <em>gfp</em> replaces <em>atmelA</em> and <em>apvA</em>
</h3>
<p>To test our hypothesis, we generated two mutants (<em>atmelAp-gfp</em>, <em>apvAp-gfp</em>) in which the coding regions of <em>apvA</em> and <em>atmelA</em> were replaced by the green fluorescent protein coding sequence (<em>gfp</em>), placing <em>gfp</em> under control of their specific promoters (Fig. S1Cii<a class="usa-link" href="#fn1">†</a>). Both the melanin pathway and aspulvinones biosynthetic pathways are active when <em>A. terreus</em> is cultivated on LCMM agar. Under this culture condition, we hypothesized that <em>atmelAp</em> would turn on the expression of <em>gfp</em> specifically inside the conidia while the <em>gfp</em> regulated by <em>apvAp</em> would display a green fluorescent signal in the hyphae. As expected, we were able to visualize localization of GFP within the conidia only in the strain carrying <em>atmelAp-gfp</em> but not in the parental strain (used as control) or the strain <em>apvAp-gfp</em>. (<a class="usa-link" href="#fig5">Fig. 5B</a>) In comparison, hyphal localization of GFP fluorescence was observed only in strain <em>apvAp-gfp</em>. (<a class="usa-link" href="#fig5">Fig. 5B</a>) Thus, the tissue specific accumulation of aspulvinone E (<strong>1</strong>) is probably due to cell-type-specific expression of the two genes <em>apvA</em> and <em>atmelA</em>.</p></section></section><section id="sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p>Our study suggests that metabolite diversity can be expanded by spatial regulation of the same precursor biosynthesized by two distinct genes. Melanin plays an important role in fungal pathogenesis as well as in the protection of the producing organisms from ultraviolet radiation.<sup><a aria-describedby="cit11" class="usa-link" href="#cit11">11</a>,<a aria-describedby="cit16" class="usa-link" href="#cit16">16</a>,<a aria-describedby="cit17" class="usa-link" href="#cit17">17</a></sup> No definite structures for melanins have been elucidated due to their large molecular weight, insolubility in aqueous or organic solvents, and heterogeneity.<sup><a aria-describedby="cit11" class="usa-link" href="#cit11">11</a></sup> The characterized conidial melanin of most <em>Aspergillus</em> species belongs to the dihydroxynaphthalene (DHN) melanin class.<sup><a aria-describedby="cit11" class="usa-link" href="#cit11">11</a></sup> Our study reports a unique and unprecedented pathway of conidial pigment biosynthesis in <em>A. terreus</em> that originates from the NRPS-like product aspulvinone E (<strong>1</strong>) (<a class="usa-link" href="#fig1">Fig. 1</a>). We also identified one putative tyrosinase encoded by the gene <em>atmelB</em> that is involved in a tailoring modification of aspulvinone E (<strong>1</strong>) to yield the brown conidial pigment (<a class="usa-link" href="#fig1">Fig. 1</a>). We speculate that the function of the tyrosinase, AtmelB, might resemble that of its homolog which participates in dihydroxyphenylalanine (DOPA)-melanin biosynthesis<sup><a aria-describedby="cit11" class="usa-link" href="#cit11">11</a></sup> and catalyzes the <em>ortho</em> hydroxylation of the phenol moiety in the aspulvinone E (<strong>1</strong>) core (<a class="usa-link" href="#fig1">Fig. 1</a>). It is entirely possible that other genes are involved in the polymerization of the dual hydroxylated aspulvinone E and more efforts will be needed to decipher the complete biosynthetic pathway of the brown conidial melanin in <em>A. terreus</em>.</p>
<p>Compounds in the aspulvinone family have been found to have anti-influenza A viral (H1N1) activity and are potent inhibitors of firefly luciferase.<sup><a aria-describedby="cit18" class="usa-link" href="#cit18">18</a>,<a aria-describedby="cit19" class="usa-link" href="#cit19">19</a></sup> The production of aspulvinones, as revealed in a previous study, is a general stress response of <em>A. terreus</em>.<sup><a aria-describedby="cit20" class="usa-link" href="#cit20">20</a></sup> The biological functions of aspulvinones, and what role they play in the biology of <em>A. terreus</em>, remain elusive. AtmelA shares 73% sequence similarity with ApvA. We noted that there is a 63 bp sequence in <em>atmelA</em>, corresponding to the TE domain of AtmelA, that has strong nucleotide identity with a portion of <em>apvA</em>, indicating that these two genes may share a common origin (Fig. S5C<a class="usa-link" href="#fn1">†</a>). Interestingly, homology analysis of <em>apvA</em> and its surrounding genes showed that <em>apvA</em> is inserted into a conserved genome region consisting of genes that are essential for fungal survival (Fig. S4<a class="usa-link" href="#fn1">†</a>). It is possible that the insertion of <em>apvA</em> is due to the duplication and translocation of its ancestral homolog. The production of aspulvinone-type SMs by <em>A. terreus</em>, in response to general stress, could be related to the chromosomal location of its producing gene <em>apvA</em>. However, more experiments are definitely necessary to elucidate the exact role the aspulvinone-type metabolites play in the growth, survival or metabolic processes of the fungus <em>A. terreus</em>.</p>
<p>Previous literature suggested that <em>p</em>-hydroxylphenylpyruvate (HPP), which originates from the shikimate pathway, is the biosynthetic precursor of aspulvinones.<sup><a aria-describedby="cit21" class="usa-link" href="#cit21">21</a>,<a aria-describedby="cit22" class="usa-link" href="#cit22">22</a></sup> The shikimate pathway, highly conserved in bacteria, fungi, and plant species, generates carbon skeletons for the aromatic amino acids including tryptophan, tyrosine, and phenylalanine.<sup><a aria-describedby="cit23" class="usa-link" href="#cit23">23</a></sup> HPP is proposed to be an intermediate in the conversion of prephenate to <span class="font-variant-small-caps">l</span>-tyrosine in the shikimate pathway. Limited information has been reported regarding the tissue localization of HPP in fungi. Our study suggests that the distribution of HPP is not limited to certain types of fungal compartments since both the biosynthesis of brown conidial melanin and hyphae specific aspulvinones requires the presence of HPP. Our data indicate that the tissue-specific expression of <em>apvA</em> and <em>atmelA</em> are due to the promoters of the two genes. The promoter of <em>apvA</em> drives expression specifically in hyphae while the promoter of <em>atmelA</em> drives expression in conidia. There are many examples of hyphal <em>versus</em> spore specific gene regulation. For example, the expression of the melanin synthesis gene <em>atmelA</em> might regulated by spore specific transcriptional regulators as demonstrated in a recent study showing that BrlA, the transcription factor required to initiate conidiophore development in <em>Aspergillus</em> spp., is necessary for fumiquinazoline gene expression and product production.<sup><a aria-describedby="cit13" class="usa-link" href="#cit13">13</a></sup> The fact that <em>atmelAp</em> is capable of turning on the expression of both <em>apvA</em> (SM gene) and the <em>gfp</em> (reporter gene), suggests that it might be a useful tool for directing the expression of genes specifically inside of conidia.</p>
<p>Besides spatial regulation of the intermediate, the SM diversity can be further expanded, as shown in our study, through tailoring enzymes that are capable of modifying SMs with different chemical scaffolds. Early literature reported the enzymatic characterization of aspulvinone dimethylallyltransferase in <em>A. terreus</em>. This enzyme is capable of catalyzing the mono or dual prenylation of aspulvinone E (<strong>1</strong>).<sup><a aria-describedby="cit24" class="usa-link" href="#cit24">24</a></sup> The substrate promiscuity of some PTs has also been tested <em>in vitro</em> by feeding experiments.<sup><a aria-describedby="cit25" class="usa-link" href="#cit25">25</a></sup> Our study reveals the <em>in vivo</em> versatility of AbpB, as it accepts substrates with different chemical scaffolds. Interestingly, the three genes are dispersed in the <em>A. terreus</em> genome (<em>abpB</em> on chromosome II; <em>apvA</em> on chromosome III; <em>btyA</em> on chromosome IV), representing another deviation from the SM gene cluster paradigm. We have discovered in <em>A. nidulans</em> that in some cases, the corresponding PT genes are not located in the same cluster as the core PKS genes.<sup><a aria-describedby="cit26" class="usa-link" href="#cit26">26</a>,<a aria-describedby="cit27" class="usa-link" href="#cit27">27</a></sup> Further comparison of the butyrolactones with aspulvinones shows that the chemical modifications after prenylation, including epoxidation and dehydrogenation, are very similar (Fig. S6<a class="usa-link" href="#fn1">†</a>). Thus, these two natural product families may share the same set of tailoring enzymes that specifically modifies the prenyl groups attached by AbpB.</p>
<p>Fungi are capable of producing a large variety of SMs with unique chemical scaffolds. It is usually expected that core synthetic genes, in a given species, are behind the biosynthesis of different core intermediates, as demonstrated in a study showing that all individual NRPKS in <em>A. nidulans</em> generate a unique PKS product.<sup><a aria-describedby="cit5" class="usa-link" href="#cit5">5</a></sup> Phylogenetic analysis, using the protein sequences of those NRPS-like homologs obtained from the Broad Institute <em>Aspergillus</em> comparative database, revealed several other characterized NRPS-like homologs including TdiA (terrequinone A biosynthesis in <em>A. nidulans</em>),<sup><a aria-describedby="cit8" class="usa-link" href="#cit8">8</a></sup> RalA (ralfuranone biosynthesis in <em>Ralstonia solanacearum</em>),<sup><a aria-describedby="cit28" class="usa-link" href="#cit28">28</a></sup> MicA (microperfuranone biosynthesis in <em>A. nidulans</em>),<sup><a aria-describedby="cit29" class="usa-link" href="#cit29">29</a></sup> AtqA (asterriquinone biosynthesis in <em>A. terreus</em>),<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup> BtyA (butyrolactone biosynthesis in <em>A. terreus</em>),<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup> and EchA (echosides biosynthesis in <em>Streptomyces</em> sp. LZ35)<sup><a aria-describedby="cit30" class="usa-link" href="#cit30">30</a></sup> (Fig. S5<a class="usa-link" href="#fn1">†</a>). These NRPS-like enzymes, with A-T-TE domain architecture, fall within Clade I of the phylogenetic tree (Fig. S5<a class="usa-link" href="#fn1">†</a>). They are capable of synthesizing natural products with various chemical scaffolds using similar substrates like phenylpyruvic acid. Previous literature suggested that these pyruvic acid substrates could be produced <em>via</em> the shikimate pathway.<sup><a aria-describedby="cit21" class="usa-link" href="#cit21">21</a>,<a aria-describedby="cit31" class="usa-link" href="#cit31">31</a>,<a aria-describedby="cit32" class="usa-link" href="#cit32">32</a></sup> The TE domains are proposed to catalyze various condensation, cyclization and releasing reactions to yield different chemical backbones.<sup><a aria-describedby="cit8" class="usa-link" href="#cit8">8</a></sup> In comparison, another characterized NRPS-like protein (encoded by ATEG_03630.1), with A-T-R domain arrangement, belongs to Clade V (Fig. S5<a class="usa-link" href="#fn1">†</a>).<sup><a aria-describedby="cit33" class="usa-link" href="#cit33">33</a></sup> In this case, the aryl acid substrate, generated by an adjacent PKS (encoded by ATEG_03629.1), is loaded onto the A domain and is reduced to its aryl-aldehyde precursor by the R domain. Thus, SM diversity is enriched from these disparate starting points. It is also common to identify two highly homologous core synthetic genes, in different species, that produces the same precursor. For example, in the biosynthesis of the meroterpenoids austinol (<em>A. nidulans</em>) and terretonin (<em>A. terreus</em>), the two PKS genes <em>ausA</em> and <em>trt4</em> synthesize the same intermediate 3, 5-dimethylorsellinic acid.<sup><a aria-describedby="cit34" class="usa-link" href="#cit34">34</a></sup>
</p></section><section id="sec9"><h2 class="pmc_sec_title">Conclusion</h2>
<p>Our work indicates that chemical diversity in SM biosynthesis can be expanded <em>via</em> multiple dimensions. Prior to our work, the richness of the fungal SM pool was presumed to be determined by the number of SM gene clusters while the clustered genes are associated with the biosynthesis of a distinct type of SM, with some exceptions.<sup><a aria-describedby="cit12" class="usa-link" href="#cit12">12</a>,<a aria-describedby="cit13" class="usa-link" href="#cit13">13</a>,<a aria-describedby="cit35" class="usa-link" href="#cit35">35</a></sup> In our study, we have shown that <em>A. terreus</em> deploys an additional strategy to enrich its natural product pool. Although the same precursor aspulvinone E (<strong>1</strong>) is shared in two pathways (the aspulvinone pathway, a typical SM pathway and the melanin pathway, producing a self-protection pigment that might also be involved in the pathogenicity of this fungus), the tissue specific expression of their biosynthetic genes results in the production of the same compound in different fungal tissues and allows it to be modified into two different products that, we assume, confer selective advantages in the specific tissues in which they are produced. The localization of production is possibly regulated by their specific promoters, but it is entirely possible that a more complex regulatory mechanism underlies this phenomenon. This expands our insight into spatial regulation of SMs in fungi.<sup><a aria-describedby="cit36" class="usa-link" href="#cit36">36</a></sup> More investigation into these promoters might provide a means to the cell-type-directed biosynthesis of SMs or manipulating the location of the expression of some genes responsible for products with interesting biological activities. Finally, our data demonstrate that AbpB prenylates compounds in two pathways revealing that two pathways may share the same tailoring genes. It will be of interest to determine if this characteristic is common to, and specific to, prenyl transferases.</p></section><section id="sec10"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="sec11"><h3 class="pmc_sec_title">Strains and molecular manipulations</h3>
<p>Primers used in this study are listed in Table S1.<a class="usa-link" href="#fn1">†</a> The fungal strains used in this study are listed in Table S2.<a class="usa-link" href="#fn1">†</a> The scheme for heterologously expressing the NRPS-like genes of <em>A. terreus</em> in <em>A. nidulans</em> is shown in Fig. S1.<a class="usa-link" href="#fn1">†</a> The direct repeat (DR) deletion and <em>AfpyG</em> marker recycling experimental design is shown in Fig. S1.<a class="usa-link" href="#fn1">†</a> The construction of fusion PCR products, protoplast generation, and transformation were carried out as previously described.<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a>,<a aria-describedby="cit9" class="usa-link" href="#cit9">9</a></sup> The scheme of diagnostic PCR is shown in Fig. S8.<a class="usa-link" href="#fn1">†</a>
</p>
<p>For real time qRT-PCR, the <em>A. terreus</em> wild type strain and the mutant strain (<em>apvA</em>Δ, <em>atmelA</em>::<em>apvA</em>) were cultivated on LCMM agar for mRNA extraction from conidia. The <em>A. terreus</em> wild type strain was cultivated in LCMM liquid broth for mRNA extraction from hyphae. Total mRNA was extracted by using the Qiagen RNeasy Plant Mini Kit. The cDNA was made from the equal amount of mRNA. The expression of every gene was analyzed with the ABI 7900HT Fast Real-Time PCR system (see ESI for experimental details<a class="usa-link" href="#fn1">†</a>).</p></section><section id="sec12"><h3 class="pmc_sec_title">Fermentation and LC-MS analysis</h3>
<p>(1) <em>A. nidulans</em> strain LO4389 and other HE mutant strains were cultivated at 37 °C in 50 ml LMM liquid medium (6 g l<sup>–1</sup> NaNO<sub>3</sub>, 0.52 g l<sup>–1</sup> KCl, 0.52 g l<sup>–1</sup> MgSO<sub>4</sub>·7H<sub>2</sub>O, 1.52 g l<sup>–1</sup> KH<sub>2</sub>PO<sub>4</sub>, 20 g l<sup>–1</sup> lactose supplemented with 1 ml l<sup>–1</sup> of trace element solution) at 1 × 10<sup>6</sup> spore per ml per 125 ml flask with shaking at 180 rpm. The nutrients uridine, uracil, pyridoxine and riboflavin were supplemented if necessary. As previously reported,<sup><a aria-describedby="cit9" class="usa-link" href="#cit9">9</a></sup> 44 μl (10 mM) cyclopentanone was added into the medium after 18 h of incubation. The incubator temperature was then changed to 30 °C and the culture medium was collected 72 h after cyclopentanone induction. The medium was filtrated and extracted twice with ethyl acetate (EtOAc, 50 ml). Preparation of the HPLC-MS samples and the condition for MS analysis were as previously described.<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup>
</p>
<p>(2) For the prenyltransferase screening experiment, <em>A. terreus</em> NIH 2624 and the mutant strains were point inoculated at 30 °C on LCMM agar plates (6 g l<sup>–1</sup> NaNO<sub>3</sub>, 0.52 g l<sup>–1</sup> KCl, 0.52 g l<sup>–1</sup> MgSO<sub>4</sub>·7H<sub>2</sub>O, 1.52 g l<sup>–1</sup> KH<sub>2</sub>PO<sub>4</sub>, 10 g l<sup>–1</sup>
<span class="font-variant-small-caps">d</span>-glucose, 20 g l<sup>–1</sup> lactose, 15 g l<sup>–1</sup> agar supplemented with 1 ml l<sup>–1</sup> of trace element solution) per plate (<em>D</em> = 10 cm). After 5 days, agar was chopped into small pieces and extracted with 50 ml MeOH followed by 60 ml 1 : 1 CH<sub>2</sub>Cl<sub>2</sub>/MeOH. The extract was evaporated <em>in vacuo</em> to yield a water residue, which was suspended in 25 ml H<sub>2</sub>O and partitioned with 25 ml EtOAc, twice. Preparation of the HPLC-MS samples and the condition for MS analysis were the same as previously described.<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup>
</p>
<p>(3) For the extraction of different fungal tissues for aspulvinone production, the procedure was similar to a previously reported procedure.<sup><a aria-describedby="cit12" class="usa-link" href="#cit12">12</a>,<a aria-describedby="cit13" class="usa-link" href="#cit13">13</a></sup> Plates containing 20 ml LCMM agar (1.5% agar) were overlaid with another 10 ml of LCMM agar (0.75% agar). Plates were point inoculated with wild type or the DR mutant strains and grown at 30 °C for 4 days. Conidia were harvested by adding 7 ml salt solution (8.5 g l<sup>–1</sup> NaCl) followed by gently scraping with a sterile spreader. The conidial solution was inspected under the microscope to be largely free of hyphae and conidiophores. The conidial fraction was sonicated for 1 h and extracted twice with an equal volume of EtOAc. The top layer was then washed with 10 ml sterile water twice and removed with a sterile spatula. The bottom agar layer was then chopped into small pieces and extracted as previously described. Preparation of the HPLC-MS samples from different tissues and the conditions for MS analysis were as previously described.<sup><a aria-describedby="cit7" class="usa-link" href="#cit7">7</a></sup>
</p></section><section id="sec13"><h3 class="pmc_sec_title">Isolation of secondary metabolites</h3>
<p>For scale up, <em>A. nidulans</em> HE strains were cultivated at 37 °C in 1 liter LMM liquid medium (∼100 ml per flask) at 1 × 10<sup>6</sup> spores per ml with shaking at 180 rpm. The nutrients uridine, uracil, pyridoxine and riboflavin were supplemented if necessary. To induce expression, 88 μl cyclopentanone was added into the medium after 18 h of incubation. The incubator temperature was then changed to 30 °C and the culture medium was collected 72 h after cyclopentanone induction. The medium was filtrated and extracted three times with equal volume of EtOAc. The combined EtOAc layers were evaporated to a crude extract. Further purification of fractions with targeted compounds was carried out by gradient HPLC on a C18 reverse phase column [Phenomenex Luna 5 μm C18, 250 × 10 mm] with a flow rate of 5.0 ml min<sup>–1</sup> and measured by a UV detector at 254 nm.</p></section><section id="sec14"><h3 class="pmc_sec_title">Fluorescence microscopy</h3>
<p>The wild type and two mutant strains (<em>atmelAp-gfp</em>, <em>apvAp-gfp</em>) were cultivated on the two-layer agar plates as mentioned. Plates were point inoculated with wild type or the mutant strains and grown at 30 °C for 5 days. The conidia were collected as previously described and the conidial solution was diluted 10 times with sterile water. A 10 μl portion of the diluted solution was placed on a pre-cleaned microscope slide and covered with a coverslip. For imaging of the hyphal GFP fluorescence signal, the top layer was removed as previously described. A small portion of hyphae containing agar (&lt;10 μl) was placed on a pre-cleaned microscope slide and covered with a coverslip. All the samples were examined under an optical microscope to confirm that the conidia and hyphae could be clearly visualized. Images were taken with a Zeiss LSM 510 Meta NLO (Thornwood, NY) confocal imaging system equipped with Argon, red HeNe, and green HeNe lasers and a Coherent Chameleon Ti–Sapphire laser mounted on a vibration-free table.</p></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Dr James F. Sanchez, Dr Yi-Ming Chiang, and Dr Shu-Lin Chang for their insightful discussion. Chun-Jun Guo was supported by the National Science Foundation – Emerging Frontiers in Research and Innovation-MIKS (Grant 1136903). Research conducted in the lab of Clay C. C. Wang is supported by the National Institute of General Medical Sciences GM084077, the National Science Foundation – Emerging Frontiers in Research and Innovation-MIKS (Grant 1136903), the Department of Energy, Bioenergy Technologies office, the Department of Defense Strategic Environmental Research and Development program (SERDP), and the National Aeronautics and Space Administration Space Biology program.</p></section><section class="fn-group" id="article1" lang="en"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><div class="p">†Electronic supplementary information (ESI) available. See DOI: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://dx.doi.org/10.1039/c5sc01058f" rel="noopener noreferrer" target="_blank">10.1039/c5sc01058f</a>
<section class="sm xbox font-sm" id="S1"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/5523082/bin/SC-006-C5SC01058F-s001.pdf">Click here for additional data file.</a><sup> (1.3MB, pdf) </sup>
</div></div></section>
</div></div></div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="cit1">
<cite>Khaldi N., Seifuddin F. T., Turner G., Haft D., Nierman W. C., Wolfe K. H., Fedorova N. D. Fungal Genet. Biol. 2010;47:736. doi: 10.1016/j.fgb.2010.06.003.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.fgb.2010.06.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2916752/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20554054/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&amp;volume=47&amp;publication_year=2010&amp;pages=736&amp;pmid=20554054&amp;doi=10.1016/j.fgb.2010.06.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit2">
<cite>Keller N. P., Turner G., Bennett J. W. Nat. Rev. Microbiol. 2005;3:937. doi: 10.1038/nrmicro1286.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nrmicro1286" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16322742/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&amp;volume=3&amp;publication_year=2005&amp;pages=937&amp;pmid=16322742&amp;doi=10.1038/nrmicro1286&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit3">
<cite>Fischbach M. A., Walsh C. T. Chem. Rev. 2006;106:3468. doi: 10.1021/cr0503097.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/cr0503097" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16895337/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Rev.&amp;volume=106&amp;publication_year=2006&amp;pages=3468&amp;pmid=16895337&amp;doi=10.1021/cr0503097&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit4">
<cite>Brown D. W., Yu J. H., Kelkar H. S., Fernandes M., Nesbitt T. C., Keller N. P., Adams T. H., Leonard T. J. Proc. Natl. Acad. Sci. U. S. A. 1996;93:1418. doi: 10.1073/pnas.93.4.1418.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.93.4.1418" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC39953/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8643646/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;volume=93&amp;publication_year=1996&amp;pages=1418&amp;pmid=8643646&amp;doi=10.1073/pnas.93.4.1418&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit5">
<cite>Ahuja M., Chiang Y.-M., Chang S.-L., Praseuth M. B., Entwistle R., Sanchez J. F., Lo H.-C., Yeh H.-H., Oakley B. R., Wang C. C. C. J. Am. Chem. Soc. 2012;134:8212. doi: 10.1021/ja3016395.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/ja3016395" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3357392/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22510154/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;volume=134&amp;publication_year=2012&amp;pages=8212&amp;pmid=22510154&amp;doi=10.1021/ja3016395&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit6">
<cite>von Döhren H. Fungal Genet. Biol. 2009;46:S45. doi: 10.1016/j.fgb.2008.08.008.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.fgb.2008.08.008" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18804170/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&amp;volume=46&amp;publication_year=2009&amp;pages=S45&amp;pmid=18804170&amp;doi=10.1016/j.fgb.2008.08.008&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit7">
<cite>Guo C. J., Knox B. P., Sanchez J. F., Chiang Y. M., Bruno K. S., Wang C. C. Org. Lett. 2013;15:3562. doi: 10.1021/ol401384v.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/ol401384v" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3833251/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23841722/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&amp;volume=15&amp;publication_year=2013&amp;pages=3562&amp;pmid=23841722&amp;doi=10.1021/ol401384v&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit8">
<cite>Balibar C. J., Howard-Jones A. R., Walsh C. T. Nat. Chem. Biol. 2007;3:584. doi: 10.1038/nchembio.2007.20.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nchembio.2007.20" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17704773/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.%20Biol.&amp;volume=3&amp;publication_year=2007&amp;pages=584&amp;pmid=17704773&amp;doi=10.1038/nchembio.2007.20&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit9">
<cite>Chiang Y. M., Oakley C. E., Ahuja M., Entwistle R., Schultz A., Chang S. L., Sung C. T., Wang C. C., Oakley B. R. J. Am. Chem. Soc. 2013;135:7720. doi: 10.1021/ja401945a.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/ja401945a" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3697937/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23621425/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;volume=135&amp;publication_year=2013&amp;pages=7720&amp;pmid=23621425&amp;doi=10.1021/ja401945a&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit10">
<cite>Nielsen J. B., Nielsen M. L., Mortensen U. H. Fungal Genet. Biol. 2008;45:165. doi: 10.1016/j.fgb.2007.07.003.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.fgb.2007.07.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17703973/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&amp;volume=45&amp;publication_year=2008&amp;pages=165&amp;pmid=17703973&amp;doi=10.1016/j.fgb.2007.07.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit11">
<cite>Langfelder K., Streibel M., Jahn B., Haase G., Brakhage A. A. Fungal Genet. Biol. 2003;38:143. doi: 10.1016/s1087-1845(02)00526-1.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s1087-1845(02)00526-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12620252/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&amp;volume=38&amp;publication_year=2003&amp;pages=143&amp;pmid=12620252&amp;doi=10.1016/s1087-1845(02)00526-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit12">
<cite>Berthier E., Lim F. Y., Deng Q., Guo C. J., Kontoyiannis D. P., Wang C. C., Rindy J., Beebe D. J., Huttenlocher A., Keller N. P. PLoS Pathog. 2013;9:e1003289. doi: 10.1371/journal.ppat.1003289.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.ppat.1003289" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3623715/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23592999/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;volume=9&amp;publication_year=2013&amp;pages=e1003289&amp;pmid=23592999&amp;doi=10.1371/journal.ppat.1003289&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit13">
<cite>Lim F. Y., Ames B., Walsh C. T., Keller N. P. Cell. Microbiol. 2014;16:1267. doi: 10.1111/cmi.12284.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/cmi.12284" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4114987/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24612080/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell.%20Microbiol.&amp;volume=16&amp;publication_year=2014&amp;pages=1267&amp;pmid=24612080&amp;doi=10.1111/cmi.12284&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit14">
<cite>May G. S. J. Cell Biol. 1989;109:2267. doi: 10.1083/jcb.109.5.2267.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.109.5.2267" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2115864/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/2681229/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol.&amp;volume=109&amp;publication_year=1989&amp;pages=2267&amp;pmid=2681229&amp;doi=10.1083/jcb.109.5.2267&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit15">
<cite>Oakley B. R. Fungal Genet. Biol. 2004;41:420. doi: 10.1016/j.fgb.2003.11.013.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.fgb.2003.11.013" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14998525/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&amp;volume=41&amp;publication_year=2004&amp;pages=420&amp;pmid=14998525&amp;doi=10.1016/j.fgb.2003.11.013&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit16">
<cite>Eisenman H. C., Casadevall A. Appl. Microbiol. Biotechnol. 2012;93:931. doi: 10.1007/s00253-011-3777-2.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00253-011-3777-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4318813/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22173481/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=93&amp;publication_year=2012&amp;pages=931&amp;pmid=22173481&amp;doi=10.1007/s00253-011-3777-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit17">
<cite>Gao Q., Garcia-Pichel F. Nat. Rev. Microbiol. 2011;9:791. doi: 10.1038/nrmicro2649.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nrmicro2649" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21963801/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&amp;volume=9&amp;publication_year=2011&amp;pages=791&amp;pmid=21963801&amp;doi=10.1038/nrmicro2649&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit18">
<cite>Gao H., Guo W., Wang Q., Zhang L., Zhu M., Zhu T., Gu Q., Wang W., Li D. Bioorg. Med. Chem. Lett. 2013;23:1776. doi: 10.1016/j.bmcl.2013.01.051.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.bmcl.2013.01.051" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23411074/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Med.%20Chem.%20Lett.&amp;volume=23&amp;publication_year=2013&amp;pages=1776&amp;pmid=23411074&amp;doi=10.1016/j.bmcl.2013.01.051&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit19">
<cite>Cruz P. G., Auld D. S., Schultz P. J., Lovell S., Battaile K. P., MacArthur R., Shen M., Tamayo-Castillo G., Inglese J., Sherman D. H. Chem. Biol. 2011;18:1442. doi: 10.1016/j.chembiol.2011.08.011.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.chembiol.2011.08.011" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3225805/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22118678/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&amp;volume=18&amp;publication_year=2011&amp;pages=1442&amp;pmid=22118678&amp;doi=10.1016/j.chembiol.2011.08.011&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit20"><cite>
Hanlon A. and O'Connor S., 2006, http://hdl.handle.net/1721.1/37692.</cite></li>
<li id="cit21">
<cite>Nitta K., Fujita N., Yoshimura T., Arai K., Yamamoto Y. Chem. Pharm. Bull. 1983;31:1528.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Pharm.%20Bull.&amp;volume=31&amp;publication_year=1983&amp;pages=1528&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit22">
<cite>Dewick P. M. Nat. Prod. Rep. 1984;1:451. doi: 10.1039/np9840100451.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1039/np9840100451" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/6399353/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Prod.%20Rep.&amp;volume=1&amp;publication_year=1984&amp;pages=451&amp;pmid=6399353&amp;doi=10.1039/np9840100451&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit23">
<cite>Tohge T., Watanabe M., Hoefgen R., Fernie A. R. Front. Plant Sci. 2013;4:62. doi: 10.3389/fpls.2013.00062.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2013.00062" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3608921/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23543266/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;volume=4&amp;publication_year=2013&amp;pages=62&amp;pmid=23543266&amp;doi=10.3389/fpls.2013.00062&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit24">
<cite>Takahashi I., Ojima N., Ogura K., Seto S. Biochemistry. 1978;17:2696. doi: 10.1021/bi00606a037.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/bi00606a037" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/678538/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;volume=17&amp;publication_year=1978&amp;pages=2696&amp;pmid=678538&amp;doi=10.1021/bi00606a037&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit25">
<cite>Yu X., Xie X., Li S. M. Appl. Microbiol. Biotechnol. 2011;92:737. doi: 10.1007/s00253-011-3351-y.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00253-011-3351-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21643703/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=92&amp;publication_year=2011&amp;pages=737&amp;pmid=21643703&amp;doi=10.1007/s00253-011-3351-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit26">
<cite>Sanchez J. F., Entwistle R., Hung J.-H., Yaegashi J., Jain S., Chiang Y.-M., Wang C. C. C., Oakley B. R. J. Am. Chem. Soc. 2011;133:4010. doi: 10.1021/ja1096682.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/ja1096682" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3119361/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21351751/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;volume=133&amp;publication_year=2011&amp;pages=4010&amp;pmid=21351751&amp;doi=10.1021/ja1096682&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit27">
<cite>Lo H.-C., Entwistle R., Guo C.-J., Ahuja M., Szewczyk E., Hung J.-H., Chiang Y.-M., Oakley B. R., Wang C. C. C. J. Am. Chem. Soc. 2012;134:4709. doi: 10.1021/ja209809t.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/ja209809t" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3350773/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22329759/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;volume=134&amp;publication_year=2012&amp;pages=4709&amp;pmid=22329759&amp;doi=10.1021/ja209809t&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit28">
<cite>Wackler B., Schneider P., Jacobs J. M., Pauly J., Allen C., Nett M., Hoffmeister D. Chem. Biol. 2011;18:354. doi: 10.1016/j.chembiol.2011.01.010.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.chembiol.2011.01.010" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21439480/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&amp;volume=18&amp;publication_year=2011&amp;pages=354&amp;pmid=21439480&amp;doi=10.1016/j.chembiol.2011.01.010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit29">
<cite>Yeh H.-H., Chiang Y.-M., Entwistle R., Ahuja M., Lee K.-H., Bruno K., Wu T.-K., Oakley B., Wang C. C. Appl. Microbiol. Biotechnol. 2012;96:739. doi: 10.1007/s00253-012-4098-9.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00253-012-4098-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3713075/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22627757/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=96&amp;publication_year=2012&amp;pages=739&amp;pmid=22627757&amp;doi=10.1007/s00253-012-4098-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit30">
<cite>Zhu J., Chen W., Li Y.-Y., Deng J.-J., Zhu D.-Y., Duan J., Liu Y., Shi G.-Y., Xie C., Wang H.-X., Shen Y.-M. Gene. 2014;546:352. doi: 10.1016/j.gene.2014.05.053.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.gene.2014.05.053" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24865933/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;volume=546&amp;publication_year=2014&amp;pages=352&amp;pmid=24865933&amp;doi=10.1016/j.gene.2014.05.053&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit31">
<cite>Knaggs A. R. Nat. Prod. Rep. 2003;20:119. doi: 10.1039/b100399m.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1039/b100399m" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12636087/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Prod.%20Rep.&amp;volume=20&amp;publication_year=2003&amp;pages=119&amp;pmid=12636087&amp;doi=10.1039/b100399m&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit32">
<cite>Arai K., Yamamoto Y. Chem. Pharm. Bull. 1990;38:2929.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Pharm.%20Bull.&amp;volume=38&amp;publication_year=1990&amp;pages=2929&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit33">
<cite>Wang M., Beissner M., Zhao H. Chem. Biol. 2014;21:257. doi: 10.1016/j.chembiol.2013.12.005.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.chembiol.2013.12.005" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3943900/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24412543/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&amp;volume=21&amp;publication_year=2014&amp;pages=257&amp;pmid=24412543&amp;doi=10.1016/j.chembiol.2013.12.005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit34">
<cite>Guo C.-J., Knox B. P., Chiang Y.-M., Lo H.-C., Sanchez J. F., Lee K.-H., Oakley B. R., Bruno K. S., Wang C. C. C. Org. Lett. 2012;14:5684. doi: 10.1021/ol302682z.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/ol302682z" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3538129/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23116177/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&amp;volume=14&amp;publication_year=2012&amp;pages=5684&amp;pmid=23116177&amp;doi=10.1021/ol302682z&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit35">
<cite>Wiemann P., Guo C. J., Palmer J. M., Sekonyela R., Wang C. C., Keller N. P. Proc. Natl. Acad. Sci. U. S. A. 2013;110:17065. doi: 10.1073/pnas.1313258110.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1313258110" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3801025/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24082142/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;volume=110&amp;publication_year=2013&amp;pages=17065&amp;pmid=24082142&amp;doi=10.1073/pnas.1313258110&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="cit36">
<cite>Lim F. Y., Keller N. P. Nat. Prod. Rep. 2014;31:1277. doi: 10.1039/c4np00083h.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1039/c4np00083h" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4162804/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25142354/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Prod.%20Rep.&amp;volume=31&amp;publication_year=2014&amp;pages=1277&amp;pmid=25142354&amp;doi=10.1039/c4np00083h&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ol></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/5523082/bin/SC-006-C5SC01058F-s001.pdf">Click here for additional data file.</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section></section></section></section></article>
</main></body></html>