<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11673879/"/><title>Thermal Plasticity and Evolutionary Constraints in Bacillus: Implications for Climate Change Adaptation - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Biology (Basel)</button></div>. 2024 Dec 23;13(12):1088. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/biology13121088" rel="noopener noreferrer" target="_blank">10.3390/biology13121088</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Thermal Plasticity and Evolutionary Constraints in <em>Bacillus</em>: Implications for Climate Change Adaptation</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hurtado-Bautista%20E%22%5BAuthor%5D"><span class="name western">Enrique Hurtado-Bautista</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Enrique Hurtado-Bautista</span></h3>
<div class="p">
<sup>1</sup>Departamento de Ingeniería Genética, Unidad Irapuato, Cinvestav 36824, Mexico; enrique.hurtado@cinvestav.mx (E.H.-B.); africa.islas@cinvestav.mx (A.I.-R.)</div>
<div>Conceptualization, Methodology, Validation, Formal analysis, Investigation, Writing – original draft, Visualization</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hurtado-Bautista%20E%22%5BAuthor%5D"><span class="name western">Enrique Hurtado-Bautista</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Islas-Robles%20A%22%5BAuthor%5D"><span class="name western">Africa Islas-Robles</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Africa Islas-Robles</span></h3>
<div class="p">
<sup>1</sup>Departamento de Ingeniería Genética, Unidad Irapuato, Cinvestav 36824, Mexico; enrique.hurtado@cinvestav.mx (E.H.-B.); africa.islas@cinvestav.mx (A.I.-R.)</div>
<div>Methodology, Validation, Resources, Project administration</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Islas-Robles%20A%22%5BAuthor%5D"><span class="name western">Africa Islas-Robles</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moreno-Hagelsieb%20G%22%5BAuthor%5D"><span class="name western">Gabriel Moreno-Hagelsieb</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Gabriel Moreno-Hagelsieb</span></h3>
<div class="p">
<sup>2</sup>Department of Biology, Wilfrid Laurier University, Waterloo, ON N2L 3C5, Canada; gmoreno@wlu.ca</div>
<div>Methodology, Software, Validation, Formal analysis, Writing – review &amp; editing</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moreno-Hagelsieb%20G%22%5BAuthor%5D"><span class="name western">Gabriel Moreno-Hagelsieb</span></a>
</div>
</div>
<sup>2</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Olmedo-Alvarez%20G%22%5BAuthor%5D"><span class="name western">Gabriela Olmedo-Alvarez</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Gabriela Olmedo-Alvarez</span></h3>
<div class="p">
<sup>1</sup>Departamento de Ingeniería Genética, Unidad Irapuato, Cinvestav 36824, Mexico; enrique.hurtado@cinvestav.mx (E.H.-B.); africa.islas@cinvestav.mx (A.I.-R.)</div>
<div>Conceptualization, Methodology, Formal analysis, Investigation, Resources, Writing – original draft, Writing – review &amp; editing, Supervision, Funding acquisition</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Olmedo-Alvarez%20G%22%5BAuthor%5D"><span class="name western">Gabriela Olmedo-Alvarez</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Andreas Walzer</span>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="af1-biology-13-01088">
<sup>1</sup>Departamento de Ingeniería Genética, Unidad Irapuato, Cinvestav 36824, Mexico; enrique.hurtado@cinvestav.mx (E.H.-B.); africa.islas@cinvestav.mx (A.I.-R.)</div>
<div id="af2-biology-13-01088">
<sup>2</sup>Department of Biology, Wilfrid Laurier University, Waterloo, ON N2L 3C5, Canada; gmoreno@wlu.ca</div>
<div class="author-notes p"><div class="fn" id="c1-biology-13-01088">
<sup>*</sup><p class="display-inline">Correspondence: <span>golmedo@cinvestav.com</span></p>
</div></div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Enrique Hurtado-Bautista</span></strong>: <span class="role">Conceptualization, Methodology, Validation, Formal analysis, Investigation, Writing – original draft, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Africa Islas-Robles</span></strong>: <span class="role">Methodology, Validation, Resources, Project administration</span>
</div>
<div>
<strong class="contrib"><span class="name western">Gabriel Moreno-Hagelsieb</span></strong>: <span class="role">Methodology, Software, Validation, Formal analysis, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Gabriela Olmedo-Alvarez</span></strong>: <span class="role">Conceptualization, Methodology, Formal analysis, Investigation, Resources, Writing – original draft, Writing – review &amp; editing, Supervision, Funding acquisition</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Andreas Walzer</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2024 Nov 22; Revised 2024 Dec 18; Accepted 2024 Dec 20; Collection date 2024 Dec.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© 2024 by the authors.</div>
<p>Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>).</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11673879  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/39765755/">39765755</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Simple Summary</h2>
<p>As global temperatures rise due to climate change, understanding how bacteria respond to heat is essential. Bacteria are critical to ecosystems, helping recycle nutrients, break down organic matter, and support plant growth. If they struggle to adapt to warmer conditions, the effects could ripple through ecosystems. Our study tested whether <em>Bacillus</em> bacteria, common in soil and water, could adapt to gradual temperature increases. We exposed seven strains from two groups to experimental evolution under increasing warming conditions, expecting them to evolve heat tolerance. Although some bacteria showed small improvements in heat tolerance, neither group could evolve to grow well at temperatures just three degrees Celsius above their natural range. This is concerning, as climate change could raise global temperatures by 2–4 °C, which could push these bacteria beyond their limits. We also found that c-di-AMP, a molecule that helps regulate potassium levels in cells, was involved in temperature tolerance. These findings highlight the vulnerability of essential bacteria to climate change and underscore the need for further research to predict and mitigate its impacts on ecosystems.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> experimental evolution, critical high temperature, phenotypic plasticity, norms of reaction to temperature, convergent evolution, c-di-AMP, upper thermal limit, evolutionary rescue, thermal plasticity, thermal niche</p></section></section><section class="abstract" id="abstract2"><h2>Abstract</h2>
<p>The ongoing rise in global temperatures poses significant challenges to ecosystems, particularly impacting bacterial communities that are central to biogeochemical cycles. The resilience of wild mesophilic bacteria to temperature increases of 2–4 °C remains poorly understood. In this study, we conducted experimental evolution on six wild <em>Bacillus</em> strains from two lineages (<em>Bacillus cereus</em> and <em>Bacillus subtilis</em>) to examine their thermal adaptation strategies. We exposed the bacteria to gradually increasing temperatures to assess their thermal plasticity, focusing on the genetic mechanisms underlying adaptation. While <em>B. subtilis</em> lineages improved growth at highly critical temperatures, only one increased its thermal niche to 4 °C above their natural range. This finding is concerning given climate change projections. <em>B. cereus</em> strains exhibited higher mutation rates but were not able to grow at increasing temperatures, while <em>B. subtilis</em> required fewer genetic changes to increase heat tolerance, indicating distinct adaptive strategies. We observed convergent evolution in five evolved lines, with mutations in genes involved in c-di-AMP synthesis, which is crucial for potassium transport, implicating this chemical messenger for the first time in heat tolerance. These insights highlight the vulnerability of bacteria to climate change and underscore the importance of genetic background in shaping thermal adaptation.</p></section><section id="sec1-biology-13-01088"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Bacteria are often regarded as highly adaptable to extreme environmental changes, including rising temperatures driven by global warming. However, this perception oversimplifies the complexity of bacterial adaptation and does not account for the millions of years it has taken for the diversity of bacteria to adapt to particular niches. This highlights that their adaptive capacity is deeply rooted in long evolutionary processes.</p>
<p>Understanding bacterial responses to heat stress is critical, given their central role in nutrient cycling and ecosystem stability, especially as global temperatures are projected to increase by 2 to 4 °C [<a aria-describedby="B1-biology-13-01088" class="usa-link" href="#B1-biology-13-01088">1</a>,<a aria-describedby="B2-biology-13-01088" class="usa-link" href="#B2-biology-13-01088">2</a>]. The impact of rising temperatures extends beyond bacterial physiology, with significant consequences for ecosystems and biogeochemical cycles; for example, elevated ocean temperatures alter nitrogen-fixing microbial populations, threatening the nitrogen cycle in coral reefs [<a aria-describedby="B3-biology-13-01088" class="usa-link" href="#B3-biology-13-01088">3</a>]. Similarly, warming freshwater ecosystems decreases bacterial diversity, disrupting ecological balance [<a aria-describedby="B2-biology-13-01088" class="usa-link" href="#B2-biology-13-01088">2</a>]. Rising temperatures are also linked to the spread of antibiotic resistance [<a aria-describedby="B4-biology-13-01088" class="usa-link" href="#B4-biology-13-01088">4</a>,<a aria-describedby="B5-biology-13-01088" class="usa-link" href="#B5-biology-13-01088">5</a>] and increased abundance of pathogens like <em>Vibrio cholerae</em> in coastal waters [<a aria-describedby="B6-biology-13-01088" class="usa-link" href="#B6-biology-13-01088">6</a>]. These examples underscore the urgency of understanding bacterial thermal adaptation in the context of global climate change.</p>
<p>Phenotypic plasticity—the ability of organisms to adjust their traits in response to changing environmental conditions [<a aria-describedby="B7-biology-13-01088" class="usa-link" href="#B7-biology-13-01088">7</a>,<a aria-describedby="B8-biology-13-01088" class="usa-link" href="#B8-biology-13-01088">8</a>]—allows species to buffer against environmental fluctuations. This adaptability can extend an organism’s ecological range, potentially leading to genetic assimilation, where initially flexible traits become genetically stabilized [<a aria-describedby="B9-biology-13-01088" class="usa-link" href="#B9-biology-13-01088">9</a>]. Nevertheless, plasticity has inherent limitations and costs, especially under sustained stress [<a aria-describedby="B10-biology-13-01088" class="usa-link" href="#B10-biology-13-01088">10</a>]. Reaction norms, which describe how a genotype’s phenotype changes across environmental gradients, are valuable tools for assessing thermal limits and understanding adaptive strategies. We can learn about a given organism’s thermal niche through reaction norms (Wolterek, 1909 [<a aria-describedby="B11-biology-13-01088" class="usa-link" href="#B11-biology-13-01088">11</a>], cited by Stearns [<a aria-describedby="B12-biology-13-01088" class="usa-link" href="#B12-biology-13-01088">12</a>]). Mesophilic bacteria, such as <em>Escherichia coli</em> [<a aria-describedby="B13-biology-13-01088" class="usa-link" href="#B13-biology-13-01088">13</a>] and some <em>Bacillus</em> species [<a aria-describedby="B14-biology-13-01088" class="usa-link" href="#B14-biology-13-01088">14</a>], typically thrive between 27 °C and 40 °C.</p>
<p>When exposed to higher temperatures, mesophilic bacteria activate a heat-shock response (HSR), which involves the rapid production of heat-shock proteins that stabilize cellular components and temporarily protect against heat stress [<a aria-describedby="B15-biology-13-01088" class="usa-link" href="#B15-biology-13-01088">15</a>,<a aria-describedby="B16-biology-13-01088" class="usa-link" href="#B16-biology-13-01088">16</a>]. However, these adaptations are generally short-lived and insufficient for prolonged survival under extreme temperatures. <em>Escherichia coli</em>, as a well-established model organism, has been extensively studied for its physiological responses to thermal stress. Unlike thermal tolerance, which temporarily protects cells from heat damage by activating heat-shock proteins [<a aria-describedby="B17-biology-13-01088" class="usa-link" href="#B17-biology-13-01088">17</a>], thermal adaptation requires genetic changes that can withstand and sustain growth at higher temperatures beyond the initial threshold [<a aria-describedby="B18-biology-13-01088" class="usa-link" href="#B18-biology-13-01088">18</a>].</p>
<p>Experimental evolution is a powerful tool for gaining insights into the capacity of bacteria to adapt to specific biotic and abiotic challenges and understanding the types of mutations that shape or enable such adaptations to changing environments [<a aria-describedby="B18-biology-13-01088" class="usa-link" href="#B18-biology-13-01088">18</a>,<a aria-describedby="B19-biology-13-01088" class="usa-link" href="#B19-biology-13-01088">19</a>]. It allows researchers to address a key question in evolutionary biology: whether bacteria can evolve similar phenotypic outcomes through the same or different genetic mechanisms—a phenomenon known as convergent evolution [<a aria-describedby="B20-biology-13-01088" class="usa-link" href="#B20-biology-13-01088">20</a>,<a aria-describedby="B21-biology-13-01088" class="usa-link" href="#B21-biology-13-01088">21</a>]. Convergent evolution can manifest in multiple ways, including traits that arise via a conserved genetic mechanism within a clade or through distinct mechanistic bases even among closely related species.</p>
<p>Previous experiments have documented a rapid adaptive response of <em>E. coli</em> to high temperatures [<a aria-describedby="B22-biology-13-01088" class="usa-link" href="#B22-biology-13-01088">22</a>], with evidence that the genetic response can be diverse [<a aria-describedby="B23-biology-13-01088" class="usa-link" href="#B23-biology-13-01088">23</a>]. Similarly, Tenaillon et al. [<a aria-describedby="B24-biology-13-01088" class="usa-link" href="#B24-biology-13-01088">24</a>] took a whole-genome approach to estimate the extent of evolutionary convergence that occurs during adaptation in 115 populations of <em>E. coli</em> at a high temperature of 42.2 °C, analyzing the overall genetic basis of adaptation. Their findings highlighted convergence among genes, with the most mutated gene being <em>rpoB</em>, along with proteins that regulate the <em>rpoS</em> stress response (RSS).</p>
<p>Although numerous studies have explored the ability of <em>E. coli</em> to evolve at different temperatures [<a aria-describedby="B25-biology-13-01088" class="usa-link" href="#B25-biology-13-01088">25</a>], few studies have aimed to evolve bacteria to expand their thermal niche. In this context, Murata et al. [<a aria-describedby="B26-biology-13-01088" class="usa-link" href="#B26-biology-13-01088">26</a>] subjected an <em>E. coli</em> strain to 43 °C and identified 51 genes crucial for thermotolerance, highlighting the diverse genetic pathways involved in developing heat tolerance. Murata introduced the term “critically high temperatures” (CHT) to describe temperatures beyond the thermal limit.</p>
<p>Bennett and Lenski [<a aria-describedby="B13-biology-13-01088" class="usa-link" href="#B13-biology-13-01088">13</a>] sought to isolate evolved lines capable of surpassing the upper thermal limit and found that this limit did not significantly increase in any of the derived lines, even those that evolved at 41–42 °C. Exposure to 44 °C proved lethal, except for one thermotolerant mutant, which managed to grow at 44 °C [<a aria-describedby="B27-biology-13-01088" class="usa-link" href="#B27-biology-13-01088">27</a>]. Mongold et al. [<a aria-describedby="B28-biology-13-01088" class="usa-link" href="#B28-biology-13-01088">28</a>] further investigated evolutionary adaptation to what they termed a “lethal thermal environment” by directly shifting evolution to 44 °C. Their findings revealed that three thermotolerant mutants were derived from two progenitor lines that had previously adapted to 41–42 °C, suggesting that these lines extended their upper thermal limit in a stepwise manner relative to their progenitors. Additionally, they noted that not all evolved lines exhibited trade-offs, such as a loss of fitness at lower temperatures.</p>
<p><em>Bacillus</em> strains are particularly important to study in the context of prolonged survival at temperatures exceeding their niche range because they are ubiquitous and ecologically significant, playing key roles in decomposing organic matter and nutrient cycling. Although <em>Bacillus</em> can form heat-tolerant spores that remain dormant for decades, their heterotrophic functions occur only in a vegetative state. Therefore, understanding whether <em>Bacillus</em> can evolve more heat-tolerant vegetative cells is crucial for predicting how these bacteria will respond to global warming. Recent research has shown that wild <em>Bacillus</em> strains exhibit considerable variation in thermal tolerance, influenced by genetic background [<a aria-describedby="B14-biology-13-01088" class="usa-link" href="#B14-biology-13-01088">14</a>], emphasizing that temperature is a critical factor affecting bacterial growth and survival. Despite extensive research on acute bacterial responses, the long-term evolution of bacterial thermotolerance and the upper thermal limits they can achieve remain poorly understood and research has not been conducted on <em>Bacillus</em> lineages.</p>
<p>This study addresses this knowledge gap by investigating whether wild <em>Bacillus</em> strains from two distinct lineages can evolve heat tolerance in their vegetative cells through experimental evolution. We aim to determine whether gradual temperature increases enable these bacteria to develop the plasticity needed to survive projected global temperature rises of 2 to 4 °C and to identify the genetic mechanisms underlying their adaptations. By focusing on different lineages and genetic backgrounds, our research explores the constraints and potential pathways for bacterial adaptation to extreme heat, providing insights into their evolutionary resilience and broader ecosystem impacts.</p>
<p>The main conclusions of this work are as follows: 1. The observed lineage-specific outcomes and convergence within strains underscore the critical role of genetic background in shaping adaptive responses. <em>B. cereus</em> and <em>B. subtilis</em> have different plasticity and <em>B. cereus</em> lineages, contrary to those of <em>B. subtilis</em>, and were not able to increase thermal tolerance relative to ancestral lines. Only one evolved <em>B. subtilis</em> line successfully extended its thermal niche. 2. Tolerance to higher temperatures was observed in <em>B. subtilis</em> lines that evolved constantly at 37 °C, indicating a metabolic adjustment driven by continuous growth conditions and the growth medium rather than selective pressure from increased temperatures. 3. Convergent mutations in genes regulating c-di-AMP synthesis uncover, for the first time, its critical role in thermal tolerance. This discovery links molecular mechanisms to thermal adaptation across lineages. 4. Finally, our findings warn about bacteria’s inability to evolve significant thermal tolerance, even under gradual selection. This raises concerns about bacterial communities’ resilience to climate change and the potential cascading effects on biogeochemical cycles and ecosystem stability.</p></section><section id="sec2-biology-13-01088"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="sec2dot1-biology-13-01088"><h3 class="pmc_sec_title">2.1. Strains and Microbial Methods</h3>
<p><em>Bacillus</em> strains were originally isolated from Cuatro Cienegas, Coahuila, México, as described [<a aria-describedby="B29-biology-13-01088" class="usa-link" href="#B29-biology-13-01088">29</a>]. The strains were collected from sediment in the Churince water system. Three strains grouped with the <em>B. cereus lineage</em> were 99% similar based on sequence variation of the 16S rRNA gene (accession numbers Bc_370a <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MK850162.1" rel="noopener noreferrer" target="_blank">MK850162.1</a>, Bc_102 <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MK880370" rel="noopener noreferrer" target="_blank">MK880370</a>, and Bc_111b <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MK850167.1" rel="noopener noreferrer" target="_blank">MK850167.1</a>), as well as four strains from the <em>B. subtilis</em> lineage (accession numbers Bs_21 <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MK850173.1" rel="noopener noreferrer" target="_blank">MK850173.1</a>, Bs_90 <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MK850153.1" rel="noopener noreferrer" target="_blank">MK850153.1</a>, and Bs_427 <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MK850159.1" rel="noopener noreferrer" target="_blank">MK850159.1</a> and laboratory strain PY79 <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MK850170.1" rel="noopener noreferrer" target="_blank">MK850170.1</a>). Accession numbers for the genomes of the strains are:</p>
<ul class="list" style="list-style-type:none">
<li><p>SAMN09761860: <em>B.subtilis</em>_21_anc_ehb001 (TaxID: 1423);</p></li>
<li><p>SAMN09761861: <em>B.subtilis</em>_90_anc_ehb002 (TaxID: 1423);</p></li>
<li><p>SAMN09761862: <em>B.subtilis</em>_427_anc_ehb003 (TaxID: 1423);</p></li>
<li><p>SAMN09761863: <em>B.subtilis</em>_PY79_anc_ehb004 (TaxID: 1423);</p></li>
<li><p>SAMN09761864: <em>B.cereus</em>_370a_anc_ehb005 (TaxID: 1396);</p></li>
<li><p>SAMN09761865: <em>B.cereus</em>_102_anc_ehb006 (TaxID: 1396);</p></li>
<li><p>SAMN09761866: <em>B.cereus</em>_111b_anc_ehb007 (TaxID: 1396).</p></li>
</ul>
<p>Strains were preserved at −70 °C in a marine medium [<a aria-describedby="B30-biology-13-01088" class="usa-link" href="#B30-biology-13-01088">30</a>] supplemented with 80% glycerol. For experimental use, strains were revived by streaking onto marine medium agar plates (1.6% agar) and incubated overnight at 28 °C. Single colonies were transferred into a liquid marine medium and incubated at 37 °C overnight to establish starting cultures.</p></section><section id="sec2dot2-biology-13-01088"><h3 class="pmc_sec_title">2.2. Plate Incubation Experiments</h3>
<p>Temperature tolerance was assessed by streaking the strains onto marine medium agar plates and incubating them at 37 °C, 44 °C, and 55 °C for 24 h. After the initial incubation, plates were transferred to 28 °C and incubated for an additional 48 h. Photographs were taken at both incubation stages to document growth and colony morphology.</p></section><section id="sec2dot3-biology-13-01088"><h3 class="pmc_sec_title">2.3. Experimental Evolution</h3>
<p>Seven <em>Bacillus</em> strains with previously characterized thermal tolerance [<a aria-describedby="B14-biology-13-01088" class="usa-link" href="#B14-biology-13-01088">14</a>] were selected, comprising four strains from the <em>B. subtilis</em> lineage (three wild isolates and the lab strain <em>B. subtilis</em> PY79) and three strains from the <em>B. cereus</em> lineage. Ancestral strains were frozen as reference samples. Experimental evolution was conducted in 50 mL Falcon tubes with cultures agitated at 150 rpm. For each strain, three populations evolved at a constant temperature of 37 °C, while another three populations were exposed to an incremental temperature regime starting at 43 °C and increasing to 47 °C over a period of 90 days.</p>
<p>Cultures were diluted 100-fold [<a aria-describedby="B31-biology-13-01088" class="usa-link" href="#B31-biology-13-01088">31</a>] every 12 h by transferring 50 µL of the culture into 5 mL of fresh marine medium, achieving approximately 1000 generations over 105 days. Generation counts were estimated based on viable cell numbers at saturation and calculated division rates. Turbidity measurements were taken every 12 h. In the incremental temperature treatment, the temperature was raised by 0.1 °C per day, contingent on observable growth (turbidity). If growth rates decreased significantly, indicated by low turbidity, temperature increases were paused or reduced, and inoculum volume was adjusted from 50 µL to 100 µL as needed. Temperature adjustments over time were recorded and represented as a parabolic curve (<a class="usa-link" href="#biology-13-01088-f001">Figure 1</a>A). Weekly samples of each evolving population were stored at −70 °C in 80% glycerol. At the experiment’s conclusion, colonies from each of the 42 evolved populations were isolated, and a representative evolved strain from each population was selected for further analysis.</p>
<figure class="fig xbox font-sm" id="biology-13-01088-f001"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11673879_biology-13-01088-g001.jpg" target="_blank"><img alt="Figure 1" class="graphic zoom-in" height="511" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3820/11673879/c6e12cb66ab4/biology-13-01088-g001.jpg" width="793"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/biology-13-01088-f001/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Experimental setup and lineage descent. (<strong>A</strong>) Inoculation and transfers were performed every 12 h until reaching approximately 1000 generations (eight generations per transfer). The plot illustrates the measured temperature profiles of the incubator over 1000 generations both for the population evolved under highly critical temperature (HCT) and the constant (lower) temperature treatment (CTT) at 37 °C. The mathematical model for HCT is represented as 4.04 + 0.59(t<sup>0.5</sup>), with an R<sup>2</sup> value of 0.8685. (<strong>B</strong>) The figure depicts the four parental strains from the <em>B. subtilis</em> lineage (left) and their respective descending lines labeled from A to L (HCT) and a to l (CTT_37 °C). Similarly, three parental strains from the <em>B. cereus</em> lineage (right) are depicted along with their corresponding descending lines named from A to L (HCT) and a to l (CTT_37 °C).</p></figcaption></figure></section><section id="sec2dot4-biology-13-01088"><h3 class="pmc_sec_title">2.4. Norms of Reaction to Temperature</h3>
<p>Reaction norms for parental and evolved strains were determined using growth kinetics at various temperatures. Doubling times were calculated from the exponential growth phase to plot the growth rate against temperature. Strains were reactivated from −70 °C stocks on marine medium agar and incubated at 37 °C for 20 h. A single colony was used to inoculate 5 mL of marine medium broth and incubated overnight at 37 °C with shaking. Cultures were transferred to achieve exponential phase by subculturing 50 µL into fresh broth for 2 h.</p>
<p>Growth measurements were performed using a Bioscreen C plate reader (Labsystems, Helsinki, Finland) with a 420–580 nm filter. Each well of a 200-well microtiter plate was inoculated with 5 µL of culture and 175 µL of fresh marine medium broth, with three technical replicates per condition. Optical density was recorded every 30 min for 20 h. Growth kinetics were assessed at 17 °C, 27 °C, 37 °C, 41 °C, 43 °C, 46 °C, 49 °C, and 55 °C. Doubling times were calculated using exponential growth models. Reaction norms were compared using <em>t</em>-tests and multiple ANOVA tests (significance level 0.05) in Statgraphics 15.2.06 and R version 3.6.2.</p></section><section id="sec2dot5-biology-13-01088"><h3 class="pmc_sec_title">2.5. Genome Sequencing and Data Analysis</h3>
<p>DNA was extracted using the QuickGene DNA tissue kit S (Kurabo Industries LTD., FujiFilm, Osaka, Japan). Sequencing was conducted at Cinvestav-Langebio on an Illumina MiSeq platform at 50× coverage in a paired-end format, with ~180 bp fragment sizes. Paired-end reads were quality-checked using FastQC and trimmed with Trimmomatic. Genome assemblies were generated with SPAdes v3.13.0 and evaluated for quality using N50 and %GC content.</p>
<p>Variant calling was performed using GATK Haplotype Caller, and mutations were annotated with Prokka and Patric. Custom scripts were used to integrate Prokka GFF files with VCF data from GATK. Additionally, we employed a “fragmentome” analysis to ensure correct linkage of evolved genomes to their respective parental strains. In this approach, we first generated sets of diagnostic DNA fragments (1000 bp each) from the original ancestral genome sequences. By comparing these fragments against all assembled genomes, we identified those that were uniquely present (i.e., matched only their strain of origin) and thus served as reliable strain-specific genetic markers. Using these diagnostic fragments, we could confidently map evolved reads back to the appropriate parental genome. Unlike standard alignments to a single reference genome, this method provides a lineage-specific signature that helps distinguish closely related ancestral strains and their evolved descendants. This ensures that any potential cross-contamination or mislabeling during experimental evolution or sequencing could be easily detected, as evolved strains must match their diagnostic fragments and not produce hits against other parental genomes. Assembled reference genomes of each representative ancestral strain were obtained, making it possible to generate these strain-specific diagnostic fragments prior to the mapping of evolved sequences.</p>
<p>Variants were filtered with a quality threshold of 100, and low-quality mutations (&lt;40) were excluded. Mutation categories included intergenic, genic, indels, and single substitutions. Mutations of interest were validated by Sanger sequencing. Annotated mutations were analyzed using UniProt, GO, and PFAM databases. Gene lists, oligonucleotide sequences, and accession numbers for sequencing data for evolved strains are provided in <a class="usa-link" href="#app1-biology-13-01088">Supplementary Tables</a>.</p></section><section id="sec2dot6-biology-13-01088"><h3 class="pmc_sec_title">2.6. Sporulation Capacity</h3>
<p>Sporulation was assessed on marine medium plates after seven days at room temperature. Samples were stained with malachite green and examined under a Leica 020-518.500 DM/LS optical microscope (Wetzlar, Germany) to confirm the presence or absence of spores.</p></section></section><section id="sec3-biology-13-01088"><h2 class="pmc_sec_title">3. Results</h2>
<section id="sec3dot1-biology-13-01088"><h3 class="pmc_sec_title">3.1. Experimental Evolution at Highly Critical Temperatures and Evolutionary Rescue</h3>
<p>We conducted experimental evolution to test if the thermal tolerance of vegetative <em>Bacillus</em> strains could be enhanced through gradual temperature increases (<a class="usa-link" href="#biology-13-01088-f001">Figure 1</a>A). Our study included seven strains from two lineages: three <em>Bacillus cereus</em> strains and four <em>Bacillus subtilis</em> strains, both of which are part of strongly supported monophyletic clades within the <em>Bacillaceae</em> family [<a aria-describedby="B32-biology-13-01088" class="usa-link" href="#B32-biology-13-01088">32</a>,<a aria-describedby="B33-biology-13-01088" class="usa-link" href="#B33-biology-13-01088">33</a>]. The wild strains were originally isolated from a temperate hydrological system [<a aria-describedby="B29-biology-13-01088" class="usa-link" href="#B29-biology-13-01088">29</a>,<a aria-describedby="B34-biology-13-01088" class="usa-link" href="#B34-biology-13-01088">34</a>], and previous research indicated that these strains are mesophilic, with thermal tolerance largely influenced by lineage [<a aria-describedby="B14-biology-13-01088" class="usa-link" href="#B14-biology-13-01088">14</a>].</p>
<p>We hypothesized that a gradual increase in temperature would serve as a selective force, potentially expanding the thermal tolerance limits of these strains. With genetic variation among the seven strains, we anticipated diverse adaptive evolutionary strategies. The experimental design involved two temperature regimes: one with a constant temperature of 37 °C and another with incremental increases from 43 °C, aiming for approximately 1000 generations (<a class="usa-link" href="#biology-13-01088-f001">Figure 1</a>B).</p>
<p>Growth at 43 °C initially varied significantly among strains. Specifically, <em>B. cereus</em> strains were highly sensitive, unable to sustain growth beyond 48 h at this temperature, while <em>B. subtilis</em> strains exhibited greater resilience. For <em>B. cereus</em>, growth restoration occurred after lowering the temperature to 42 °C, suggesting an evolutionary rescue mechanism. <em>B. subtilis</em>, in contrast, maintained growth more consistently, allowing for temperature increments of 0.1 °C per day.</p>
<p>Temperature increments continued until the strains reached a plateau at 45 °C after approximately 300 generations and then slowed down (<a class="usa-link" href="#biology-13-01088-f001">Figure 1</a>A). The increment frequency decreased further until no additional adaptation was observed over the last 300 generations, indicating a stabilization of phenotypes. <em>B. subtilis</em> lineages showed a greater overall capacity to survive and adapt than <em>B. cereus</em>, consistent with their initial phenotypic plasticity.</p></section><section id="sec3dot2-biology-13-01088"><h3 class="pmc_sec_title">3.2. Differences in Plasticity and Critical Temperature Limits</h3>
<p>To assess changes in thermal tolerance, we compared the reaction norms of evolved and parental strains. Growth kinetics were measured across a temperature range from 17 °C to 55 °C and doubling times were used to construct growth rate curves.</p>
<section id="sec3dot2dot1-biology-13-01088"><h4 class="pmc_sec_title">3.2.1. Plasticity and Critical Temperature Limits for <em>Bacillus subtilis</em> Lineages</h4>
<p><a class="usa-link" href="#biology-13-01088-f002">Figure 2</a> and <a class="usa-link" href="#biology-13-01088-f003">Figure 3</a> illustrate the reaction norms for <em>B. subtilis</em> strains. The Bs_21 and Bs_427 strains demonstrated the most substantial increases in tolerance at higher temperatures. Notably, several Bs_21 lines grew successfully at up to 49 °C, although the results varied among lines (a-b-c-A-C). All Bs_427-evolved lines showed improved growth at 46 °C and 49 °C, regardless of whether they were evolved under constant temperature treatment (CTT) or the high critical temperature (HCT) regime. This suggests that adaptation may be influenced by factors related to the growth medium. A statistically significant reduction in growth was observed for some Bs_427 lines (B and b) at 37 °C.</p>
<figure class="fig xbox font-sm" id="biology-13-01088-f002"><h5 class="obj_head">Figure 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11673879_biology-13-01088-g002.jpg" target="_blank"><img alt="Figure 2" class="graphic zoom-in" height="524" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3820/11673879/1807a3dfffc4/biology-13-01088-g002.jpg" width="798"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/biology-13-01088-f002/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Norms of reaction <em>Bacillus subtilis</em> parental strains (black) and their respective evolved lines at HCT treatment (red) and CTT treatment (37 °C) (blue). Vertical bars are confidence intervals; qt 95% (ANOVA analysis, <em>p</em> = 0.05). Horizontal lines at the top of each graph represent the temperature niche of the ancestral strains (black) and strains evolved under CTT (blue) or HCT treatments (red).</p></figcaption></figure><figure class="fig xbox font-sm" id="biology-13-01088-f003"><h5 class="obj_head">Figure 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11673879_biology-13-01088-g003.jpg" target="_blank"><img alt="Figure 3" class="graphic zoom-in" height="521" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3820/11673879/a6502068c5b1/biology-13-01088-g003.jpg" width="789"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/biology-13-01088-f003/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Norms of reaction <em>Bacillus cereus</em> parental strains (black) and their respective evolved lines upon HCT treatment (red) and CTT treatment (37 °C) (blue). Vertical bars are confidence intervals; qt 95% (ANOVA analysis, <em>p</em> = 0.05). Horizontal lines at the top of each graph represent the temperature niche of the ancestral strains (black) and strains evolved under CTT (blue) or HCT treatments (red).</p></figcaption></figure><p>Bs_90 exhibited limited improvement in thermal tolerance, with only minor increases in growth at 43 °C for the D-E-d-e lines and barely significant growth at 46 °C for the D-E-F lines. The Bs_90 strain was the least capable of growing at 49 °C, though all evolved lines still outperformed the parental strain. A trade-off was evident in Bs_90, with all evolved lines showing reduced growth at lower temperatures (17–37 °C). In contrast, the laboratory strain Bs_PY79 showed consistent growth across temperatures up to 37 °C. While some high-temperature-evolved lines (J-K-L) demonstrated improved tolerance at 49 °C, they experienced a marked reduction in growth at 17 °C and 27 °C. Interestingly, all Bs_PY79 lines performed similarly to the parental strain at 43 °C, without surpassing its tolerance at 46 °C.</p></section><section id="sec3dot2dot2-biology-13-01088"><h4 class="pmc_sec_title">3.2.2. Plasticity and Critical Temperature Limits for <em>Bacillus cereus</em> Lineages</h4>
<p>For <em>B. cereus</em>, the reaction norms revealed minimal improvements post-evolution. Strain Bc_370a exhibited poorer growth than the parental strain, particularly for lines evolved under the HCT regime, which showed decreased growth at 27 °C and 37 °C. Bc_102 demonstrated the best improvements in thermal tolerance among HCT-evolved lines, though CTT-evolved lines generally showed reduced growth compared to the parental strain. Bc_111b presented a slight increase in growth at 43–46 °C for line U, indicating potential thermal adaptation. However, the S-T-U lines (HCT-evolved) exhibited a reduction in fitness across a lower temperature range (17–39 °C), suggesting maladaptive consequences of high-temperature evolution.</p></section></section><section id="sec3dot3-biology-13-01088"><h3 class="pmc_sec_title">3.3. Changes in Carrying Capacity (K) and Growth Rate (G)</h3>
<p>To further understand the adaptive strategies of evolved <em>Bacillus</em> strains, we assessed changes in two critical growth parameters: carrying capacity (K) and specific growth rate (G) at 37 °C and 43 °C. These measurements provided insights into whether the adaptations were multifactorial, impacting not just the growth rate but also the overall population density the cultures could sustain. The Principal Component Analysis (PCA) of experimental evolution treatments of the traits evaluated for each strain at both 37 °C and 43 °C highlighted the phenotypic responses and adaptations under different temperature regimes (<a class="usa-link" href="#app1-biology-13-01088">Supplementary Figure S4</a>).</p>
<section id="sec3dot3dot1-biology-13-01088"><h4 class="pmc_sec_title">3.3.1. Impact of Temperature on Growth and Carrying Capacity on <em>Bacillus subtilis</em> Lineages</h4>
<p><a class="usa-link" href="#biology-13-01088-f004">Figure 4</a> shows the normalized results for <em>B. subtilis</em> strains. In Bs_21, significant changes were observed in both K37 and K43, suggesting substantial adaptation, particularly for the D-E-F lines under the high-temperature treatment. Bs_90 displayed pronounced effects in K43, specifically for the D-E-F lines, indicating a temperature-driven adaptation. However, the adaptation appeared limited, as growth improvements did not extend uniformly across all lines.</p>
<figure class="fig xbox font-sm" id="biology-13-01088-f004"><h5 class="obj_head">Figure 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11673879_biology-13-01088-g004.jpg" target="_blank"><img alt="Figure 4" class="graphic zoom-in" height="370" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3820/11673879/044d6bd3eb53/biology-13-01088-g004.jpg" width="740"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/biology-13-01088-f004/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Growth and carrying capacity in parental and experimentally evolved lines.</p></figcaption></figure><p>In Bs_427, we observed increases in both K37 and K43 values compared to the parental strain, reflecting an adaptation that may be driven by the growth medium. The enhanced carrying capacity at 43 °C suggests some degree of temperature adaptation, especially for the D-E-F lines. In Bs_PY79, all evolved lines showed increases in K37 and K43, consistent with medium-driven adaptation rather than temperature-specific changes.</p>
<p>Parental strains of <em>Bacillus subtilis</em> are represented by black bars, while their descendant lines under highly critical temperature (HCT) treatment are depicted with red bars and those under constant temperature treatment at 37 °C are shown in blue bars. Each panel comprises charts illustrating the maximum change in growth cultures of lines evolved at a constant low temperature (37 °C) and at a highly critical temperature (43 °C) on the left and carrying capacity (K) at 37 °C and 43 °C on the right.</p></section><section id="sec3dot3dot2-biology-13-01088"><h4 class="pmc_sec_title">3.3.2. Impact of Temperature on Growth and Carrying Capacity on <em>Bacillus cereus</em> Lineages</h4>
<p><a class="usa-link" href="#biology-13-01088-f005">Figure 5</a> illustrates the phenotypic changes in <em>B. cereus</em> strains. Bc_370a showed a strong effect on all lines for the parameter G37, implying adaptation to the growth medium rather than temperature. However, a significant impact on K43 was observed only in the M-N-O lines, which underwent high-temperature evolution, indicating some level of thermal adaptation.</p>
<figure class="fig xbox font-sm" id="biology-13-01088-f005"><h5 class="obj_head">Figure 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11673879_biology-13-01088-g005.jpg" target="_blank"><img alt="Figure 5" class="graphic zoom-in" height="427" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3820/11673879/b2f055c1d255/biology-13-01088-g005.jpg" width="739"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/biology-13-01088-f005/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Growth and carrying capacity in parental and experimentally evolved lines.</p></figcaption></figure><p>In Bc_102, the analysis of K43 revealed stronger adaptation to high temperature in the P-Q-R lines compared to p-q-r. This suggests that thermal adaptation occurred despite the reaction norms not clearly indicating such changes. Specific growth rates (G37) also demonstrated some improvements, although these were less pronounced. For Bc_111b, G43 showed a marked increase across all evolved lines, indicating medium adaptation. However, K43 was significantly higher in the S-T-U lines compared to s-t-u and the parental strain, pointing to an evident effect of temperature adaptation.</p>
<p>Parental strains of <em>Bacillus subtilis</em> are represented by black bars, while their descendant lines under highly critical temperature (HCT) treatment are depicted with red bars, and those under constant temperature treatment at 37 °C are shown in blue bars. Each panel comprises charts illustrating the maximum change in growth cultures of lines evolved at a constant low temperature (37 °C) and at a highly critical temperature (43 °C) to the left and the carrying capacity (K) at 37 °C and 43 °C to the right.</p>
<p><a class="usa-link" href="#biology-13-01088-t001">Table 1</a> is a summary of the overall impact that evolution at either CTT or HCT had on the <em>B. cereus</em> and <em>B. subtilis</em> lineages. It is evident that a single line, Bs _427, was able to extend its thermal tolerance. None of the lines from the laboratory strain PY79 exhibited changes at any of the temperatures evaluated. All the evolved <em>B. cereus</em> lines reduced their growth capacity compared to the ancestral line.</p>
<section class="tw xbox font-sm" id="biology-13-01088-t001"><h5 class="obj_head">Table 1.</h5>
<div class="caption p"><p>Summary of temperature plasticity gained on growth and carrying capacity by the evolved lines.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="center" colspan="1" rowspan="1" style="border-top:solid thin" valign="middle">
</th>
<th align="center" colspan="10" rowspan="1" style="border-top:solid thin;border-bottom:solid thin" valign="middle">Growth</th>
<th align="center" colspan="4" rowspan="1" style="border-top:solid thin;border-bottom:solid thin" valign="middle">Carrying Capacity (K) </th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">Evaluated at:</th>
<th align="center" colspan="2" rowspan="1" style="border-bottom:solid thin" valign="middle">27 °C</th>
<th align="center" colspan="2" rowspan="1" style="border-bottom:solid thin" valign="middle">37 °C</th>
<th align="center" colspan="2" rowspan="1" style="border-bottom:solid thin" valign="middle">43 °C</th>
<th align="center" colspan="2" rowspan="1" style="border-bottom:solid thin" valign="middle">46 °C</th>
<th align="center" colspan="2" rowspan="1" style="border-bottom:solid thin" valign="middle">49 °C</th>
<th align="center" colspan="2" rowspan="1" style="border-bottom:solid thin" valign="middle">37 °C</th>
<th align="center" colspan="2" rowspan="1" style="border-bottom:solid thin" valign="middle">43 °C</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">Strain evolved at:</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">37 °C</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">HCT</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">37 °C</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">HCT</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">37 °C</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">HCT</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">37 °C</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">HCT</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">37 °C</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">HCT</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">37 °C</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">HCT</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">37 °C</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">HCT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">Bc 370a *</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">Bc 11b *</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">Bc 102 *</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▲</em>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">Bs 21</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">Bs 90 *</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">Bs 427</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" valign="middle">
<em>▲</em>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">BS PY79</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▼</em>
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▲</em>
</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">
<em>▲</em>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/biology-13-01088-t001/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn2"><p>▼ Lower capacity than ancestral line. ▲ Higher capacity than the ancestral line. Empty squares mean the same or similar capacity than the ancestral line. Red, evolved at temperatures above highly critical temperature. Blue, evolved at 37 °C. * Points to strains where some lines had mutations in c-di-AMP synthesis genes.</p></div></div></section><section id="sec3dot4-biology-13-01088"><h5 class="pmc_sec_title">3.4. Changes in Colony Growth, Pigmentation, and Loss of Sporulation in Evolved Lines</h5>
<p>We evaluated changes in colonial phenotypes by growing parental and evolved <em>Bacillus</em> strains on marine medium agar plates (<a class="usa-link" href="#app1-biology-13-01088">Supplementary Figures S1 and S2</a>). The top rows in the figures show the parental strains, while the lower rows display the six evolved strains (three evolved under the low-temperature treatment, LTT, and three under the highly critical temperature, HCT). Our semi-quantitative assessments revealed significant changes in colony growth among evolved strains. Notably, the <em>B. cereus</em>-evolved lines exhibited a substantial reduction in growth compared to their parental strains, a finding consistent with the reaction norm data. This reduction was particularly evident in lines M, N, Q, S, and T, which underwent HCT treatment. In contrast, the <em>B. subtilis</em>-evolved strains demonstrated more resilience, retaining a higher growth capacity.</p>
<p>The observed differences suggest a trade-off in <em>B. cereus</em>, where the selective pressure to endure high temperatures may have compromised their ability to grow rapidly. This trade-off likely represents an adaptive strategy prioritizing survival over growth efficiency. Specifically, the <em>B. cereus</em> strains appear to have evolved a mechanism to endure heat stress by reducing metabolic demands, which could explain their diminished growth rates.</p>
<p>Additionally, phenotypic changes were evident in colony pigmentation, especially among the <em>B. subtilis</em>-evolved lines. We observed distinct pigmentation changes in lines A, B, G, and I, indicative of altered metabolic or regulatory pathways as a response to prolonged heat exposure (<a class="usa-link" href="#app1-biology-13-01088">Supplementary Figure S2</a>).</p>
<p>We also noted a loss of sporulation ability in several evolved strains. This was particularly pronounced in the <em>B. cereus</em> HCT-evolved lines, suggesting that the evolutionary pressure to maintain vegetative cell function at high temperatures led to a trade-off, where sporulation capacity was compromised. This finding aligns with the hypothesis that <em>Bacillus</em> strains may sacrifice their ability to form heat-resistant spores to maintain active metabolic processes, which may be crucial for survival in fluctuating but non-lethal high temperatures.</p></section></section></section><section id="sec3dot5-biology-13-01088"><h3 class="pmc_sec_title">3.5. Convergence in Mutations of Strains Evolved at Highly Critical Temperature Reveals the Role of di-C-AMP and Osmolytes in Temperature Tolerance</h3>
<p>We sequenced the genomes of a representative clone from both the parental and evolved lines (seven parental genomes, 21 evolved genomes at CTT, and 21 evolved genomes at HCT). The genomes were assembled and annotated, and mutations were identified by comparing each evolved genome to its respective parental genome using GATK tools (see Materials and Methods). A workflow of Bioinformatic Analysis of Evolved Genomes and Mutations is shown in <a class="usa-link" href="#app1-biology-13-01088">Supplementary Figure S3</a>.</p>
<section id="sec3dot5dot1-biology-13-01088"><h4 class="pmc_sec_title">3.5.1. Identification and Exclusion of Hypermutator Lines</h4>
<p>Seven lines exhibited hypermutator phenotypes, characterized by the accumulation of 39 to 219 mutations. These hypermutator lines were excluded from further analysis to avoid skewed results. The hypermutant criterion was established based on a significant deviation from the expected linear distribution of mutation accumulation (<a class="usa-link" href="#app1-biology-13-01088">Supplementary Figure S5</a>). Hypermutators were identified in both <em>B. subtilis</em> (lines A and G) and <em>B. cereus</em> (lines N, m, q, r, and S), with no apparent correlation to temperature treatment or bacterial lineage, suggesting that the hypermutant phenotype was independent of these factors. These hypermutant lines are depicted as gray labels in <a class="usa-link" href="#biology-13-01088-f006">Figure 6</a>.</p>
<figure class="fig xbox font-sm" id="biology-13-01088-f006"><h5 class="obj_head">Figure 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11673879_biology-13-01088-g006.jpg" target="_blank"><img alt="Figure 6" class="graphic zoom-in" height="412" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3820/11673879/b62c91b1e7f1/biology-13-01088-g006.jpg" width="754"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/biology-13-01088-f006/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Mutation analysis of strains evolved from the <em>Bacillus subtilis</em> and <em>Bacillus cereus</em> lineages. Each lineage underwent highly critical temperature (HCT, red) treatment and constant temperature treatment (CTT, blue) with three replicates. The diagram illustrates the treatment conditions for each lineage and the total number of mutations observed after the genome sequencing of a colony from the final evolution transfer. Loss of sporulation in evolved lines occurred in strains marked with red or blue triangles. Lines in gray represent hypermutant phenotypes, which were excluded from our analyses to reduce noise in data correlation. Orange asterisks denote convergent mutations at the gene level in the cyclic d-AMP synthase gene (<em>cdaR</em> in <em>B. subtilis</em> or <em>disA</em> in <em>B. cereus</em>), found in independent evolving lines under highly critical temperatures, supporting evidence of positive selection (statistical analysis in the results section).</p></figcaption></figure></section><section id="sec3dot5dot2-biology-13-01088"><h4 class="pmc_sec_title">3.5.2. Classification of Mutations in Lines Evolved at 37 °C (CTT) and Above the HCT</h4>
<p>After excluding hypermutators, mutations were categorized into single-nucleotide substitutions, one to three base indels, and large genomic rearrangements, further classified as intergenic or intragenic. Most of the identified mutations were single-base indels, predominantly affecting open reading frames (ORFs). <a class="usa-link" href="#biology-13-01088-f007">Figure 7</a> and <a class="usa-link" href="#biology-13-01088-f008">Figure 8</a> illustrate the specific intragenic mutations. <a class="usa-link" href="#app1-biology-13-01088">Supplementary Table S1</a> shows a list of all types of mutations in each evolved line. Sporulation loss and the quantification for each mutation type are shown in <a class="usa-link" href="#app1-biology-13-01088">Supplementary Figure S6</a>.</p>
<figure class="fig xbox font-sm" id="biology-13-01088-f007"><h5 class="obj_head">Figure 7.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11673879_biology-13-01088-g007.jpg" target="_blank"><img alt="Figure 7" class="graphic zoom-in" height="632" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3820/11673879/fc805d0c7ece/biology-13-01088-g007.jpg" width="714"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/biology-13-01088-f007/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Distribution of mutations in lines evolved at constant low temperature (37 °C) across gene functional categories. The figure displays the distribution of mutations observed in lines evolved at a constant low temperature (37 °C), categorized by gene functional categories. Black squares indicate genes with at least one mutation in the depicted category, with the number within the black squares representing the number of mutated genes within that category. White squares denote the absence of mutations in the respective gene functional categories. Gray squares are used for all hypermutant strains that were not analyzed. Additionally, red squares highlight genes that have been described as being associated with the observed phenotype.</p></figcaption></figure><figure class="fig xbox font-sm" id="biology-13-01088-f008"><h5 class="obj_head">Figure 8.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11673879_biology-13-01088-g008.jpg" target="_blank"><img alt="Figure 8" class="graphic zoom-in" height="666" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3820/11673879/1ea637d0e99e/biology-13-01088-g008.jpg" width="755"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/biology-13-01088-f008/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Distribution of mutations in lines evolved under highly critical temperature treatment (HCT) across gene functional categories. The figure illustrates the distribution of mutations observed in lines evolved under highly critical temperature treatment (HCT), categorized by gene functional categories. Black squares represent genes with at least one mutation in the depicted category, with the number within the black squares indicating the number of mutated genes within that category. White squares indicate the absence of mutations in the respective gene functional categories. Gray squares denote hypermutant strains that were not analyzed. Additionally, red squares highlight genes associated with the observed phenotype. An orange square specifically highlights di-adenylate synthase-encoding genes.</p></figcaption></figure></section><section id="sec3dot5dot3-biology-13-01088"><h4 class="pmc_sec_title">3.5.3. Mutational Analysis in CTT-Evolved Lines</h4>
<p>In CTT-evolved lines, we identified eight mutations in genes associated with transcription regulation (a-o-p-u lines), six mutations in transport-related genes (a-c-j-p lines), and six in sporulation-related genes (a-d-j-p lines). A significant mutation was detected in strain Bs_a within the SigB regulon, known for its role in thermal stress response [<a aria-describedby="B35-biology-13-01088" class="usa-link" href="#B35-biology-13-01088">35</a>]. Additional mutations were found in genes involved in DNA replication, methylation, flagellum assembly, metabolite biosynthesis, and cell wall structure (<a class="usa-link" href="#biology-13-01088-f008">Figure 8</a>).</p></section><section id="sec3dot5dot4-biology-13-01088"><h4 class="pmc_sec_title">3.5.4. Mutational Analysis in HCT-Evolved Lines</h4>
<p>In HCT-evolved lines, we found six mutations in transcription-regulation genes (B-F-M-O-Q-T lines) and six in transport genes (B-D-L-M-P-U lines), reflecting the impact on general metabolic processes. Only one mutation affected a sporulation-related gene, <em>lipC</em>, in line B of the Bs_21 strain without causing sporulation defects. Mutations of particular interest included those in glycine–betaine transport genes, crucial for both osmotic and thermal stress resistance [<a aria-describedby="B36-biology-13-01088" class="usa-link" href="#B36-biology-13-01088">36</a>]. Mutations interrupting the DnaD and RNase H encoding genes were found in line B of the Bs_21 strain. DnaD has been reported to be part of the replisome Turner 2004 and for replication fork progression under replication stress [<a aria-describedby="B37-biology-13-01088" class="usa-link" href="#B37-biology-13-01088">37</a>]. <em>B. subtilis</em> has two RNase H genes [<a aria-describedby="B38-biology-13-01088" class="usa-link" href="#B38-biology-13-01088">38</a>]. In bacteria, RNAse H is involved in the removal of RNA in RNA:DNA hybrids, and, in <em>B. subtilis</em>, the absence of one of the RNase H genes results in an accumulation of hybrids upstream of active coding genes with an impact in transcription and increased mutations [<a aria-describedby="B39-biology-13-01088" class="usa-link" href="#B39-biology-13-01088">39</a>]. Interestingly, strain Bs_B has 37 mutations, one of the most numerous in evolved lines (<a class="usa-link" href="#app1-biology-13-01088">Supplementary Table S1</a>). We also identified mutations in arginine metabolism genes linked to thermal stress in <em>B. subtilis</em> [<a aria-describedby="B35-biology-13-01088" class="usa-link" href="#B35-biology-13-01088">35</a>] and inositol metabolism genes known for their thermoprotective properties [<a aria-describedby="B40-biology-13-01088" class="usa-link" href="#B40-biology-13-01088">40</a>]. A chi-squared test comparing five thermal stress-associated mutations in HCT-treated lines to one in CTT-treated lines suggested a significant temperature-dependent effect.</p></section><section id="sec3dot5dot5-biology-13-01088"><h4 class="pmc_sec_title">3.5.5. Uncovering of di-Adenylate Cyclases (DACs) in Temperature Adaptation</h4>
<p>A noteworthy finding was the emergence of mutations in genes encoding di-adenylate cyclases (DACs), observed exclusively in HCT-evolved lines (<a class="usa-link" href="#biology-13-01088-f009">Figure 9</a>). DACs synthesize c-di-AMP from ATP, a secondary metabolite involved in growth regulation, membrane stability, glycine-betaine transport, and potassium uptake. We identified three loss-of-function mutations in the <em>disA</em> gene of <em>B. cereus</em> (lines O-P-T) and two in the <em>cdaR</em> gene of <em>B. subtilis</em> (lines D and F), with frameshifts occurring in the first half of the ORFs (<a class="usa-link" href="#app1-biology-13-01088">Supplementary Figure S7</a>). Binomial tests indicated a statistically significant occurrence of these mutations: a probability of 1.68 × 10<sup>−4</sup> for <em>B. cereus</em> and 1.37 × 10<sup>−3</sup> for <em>B. subtilis</em>. Given the total number of genes (5200 in <em>B. cereus</em> and 4100 in <em>B. subtilis</em>), these findings suggest a selective advantage for DAC-related mutations under high-temperature conditions.</p>
<figure class="fig xbox font-sm" id="biology-13-01088-f009"><h5 class="obj_head">Figure 9.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11673879_biology-13-01088-g009.jpg" target="_blank"><img alt="Figure 9" class="graphic zoom-in" height="426" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3820/11673879/1b58c014b989/biology-13-01088-g009.jpg" width="774"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/biology-13-01088-f009/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Convergent evolution in lines evolved at highly critical temperatures revealed cyclic di-AMP as a key player. The figure illustrates convergent evolution observed in lines evolved at highly critical temperature (HCT), highlighting cyclic di-AMP as a significant factor. A red X denotes the proteins predicted to be absent due to gene mutation. Both <em>Bacillus subtilis</em> and <em>Bacillus cereus</em> possess three cyclic di-AMP synthase genes, with two expressed during the vegetative phase and the third restricted to sporulation. In <em>B. subtilis</em>, the mutated gene was <em>cdaR</em>, a regulator of the synthetase gene, while in <em>B. cereus</em>, mutations impacted the <em>disA</em> gene, the synthetase itself. Cyclic di-AMP plays a crucial role in regulating K+ uptake, a critical factor in maintaining osmotic balance, suggesting its role in adapting to heat stress.</p></figcaption></figure></section></section></section><section id="sec4-biology-13-01088"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>Our study highlights lineage-specific strategies by using experimental evolution with gradual temperature increases. It reveals significant differences in how <em>Bacillus subtilis</em> and <em>Bacillus cereus</em> lineages evolved thermal tolerance when subjected to gradual temperature increases beyond their thermal niche. <em>B. subtilis</em> strains displayed minimal changes in reaction norms, reflecting their greater inherent phenotypic plasticity, while <em>B. cereus</em> strains showed clear trade-offs, such as reduced growth rates at lower and even higher temperatures, indicating that high-temperature selection imposed a substantial evolutionary cost. These results suggest that larger initial plasticity might buffer against rapid adaptation under short-term experimental conditions.</p>
<p>In our study, the <em>B. cereus</em> strains initially struggled under high-temperature stress but managed to stabilize and recover growth within weeks, although, several times, we had to stop the regime of increasing temperature until the final 300 generations of constant high-temperature selection. These strains accumulated more mutations than those of the <em>B. subtilis</em> lineage. This is similar to what Mongold et al. [<a aria-describedby="B28-biology-13-01088" class="usa-link" href="#B28-biology-13-01088">28</a>] observed for <em>E. coli</em> growth at the limit of their niche temperature and suggests that evolutionary rescue mechanisms were at play, allowing the strains to persist despite the harsh conditions. The concept of evolutionary rescue is particularly relevant to our observations. Evolutionary rescue occurs when adaptive genetic changes enable populations to recover from near extinction under severe environmental stress [<a aria-describedby="B41-biology-13-01088" class="usa-link" href="#B41-biology-13-01088">41</a>,<a aria-describedby="B42-biology-13-01088" class="usa-link" href="#B42-biology-13-01088">42</a>].</p>
<p>While <em>B. subtilis</em> benefited from its broad thermal tolerance, <em>B. cereus</em> needed substantial genetic adjustments to cope with high temperatures. This observation raises critical questions about the role of plasticity in evolutionary dynamics. Does higher plasticity hinder adaptation by reducing selective pressure, or does it enable immediate survival, allowing for gradual genetic changes? Our data support the notion that short-term evolution favors species with inherent plasticity, while long-term selection can drive adaptive genetic changes in species with less initial plasticity. <em>B. subtilis</em> can be considered to possess more thermal plasticity compared to <em>B. cereus</em>. This conclusion aligns with previous findings by Hurtado et al. [<a aria-describedby="B14-biology-13-01088" class="usa-link" href="#B14-biology-13-01088">14</a>], which highlighted differences between the <em>B. cereus</em> and <em>B. subtilis</em> lineages in their upper tolerance limits, comparing strains from a thermal spring to those from a temperate lagoon. In this study, only <em>B. subtilis</em>-evolved lines exhibited better growth than the parental strain at the HCT. At the same time, three of four <em>B. subtilis</em> lineages surpassed their ancestral strain’s tolerance limit even at 49 °C. However, it is essential to note that the <em>B. subtilis</em> lines evolved at 37 °C were also able to increase their thermal limit. The pressure of maintaining continuous growth in this particular medium impacted the metabolism that favored thermal tolerance. This is concordant with the change in carrying capacity that resulted from the experimental evolution, particularly for the <em>B. subtilis</em> lines, for which higher carrying capacity changes occur in all lines, whether evolved at 37 °C or at high temperature. This contrasts with <em>B. cereus</em> lines, for which only the lines evolved at higher temperatures exhibit increased carrying capacity under heat stress. These results suggest a shift in metabolism. Accordingly, the expression of proteins that provide stress tolerance or the capacity to adapt to new environmental conditions rapidly is expected to be at the expense of growth-related proteins, reducing the instantaneous growth rate [<a aria-describedby="B43-biology-13-01088" class="usa-link" href="#B43-biology-13-01088">43</a>].</p>
<p>Finally, adaptation at high temperatures came with trade-offs, as evidenced by poorer growth at lower temperatures. This was observed for <em>B. cereus</em> lines but not for <em>B. subtilis</em>. Our results showed decreased fitness of some of the evolved <em>B. cereus</em> lines (Bc111 and Bc370) at what was the ancestral strain’s optimal growth temperature, as a trade-off for adapting to high temperatures. This supports evolutionary theory, which suggests pleiotropic effects, where genes affecting multiple traits may constrain adaptation and result in maladaptation [<a aria-describedby="B44-biology-13-01088" class="usa-link" href="#B44-biology-13-01088">44</a>].</p>
<p>Genetic analysis revealed that mutations in genes involved in osmolyte metabolism and heat-shock responses played critical roles in thermal tolerance. For instance, arginine and trehalose metabolism mutations were associated with improved thermal resilience. Arginine biosynthesis is known to be induced under heat stress [<a aria-describedby="B35-biology-13-01088" class="usa-link" href="#B35-biology-13-01088">35</a>], and trehalose has been suggested to act as a thermoprotectant, stabilizing proteins and membranes [<a aria-describedby="B45-biology-13-01088" class="usa-link" href="#B45-biology-13-01088">45</a>]. These mutations differ from those described in Ref. [<a aria-describedby="B46-biology-13-01088" class="usa-link" href="#B46-biology-13-01088">46</a>] from an experimental evolution experiment with <em>B. subtilis</em>, with a temperature regime of 50 °C. Sequencing revealed mutations in genes encoding proteins associated with pathways known to be involved in heat stress response (<em>hrcA</em>, <em>ftsH</em>) and other stress proteins (<em>relA</em> and <em>sigW</em>). We do not expect to find mutations in genes involved in the thermal response, such as those described by Hecker et al. (1996) [<a aria-describedby="B36-biology-13-01088" class="usa-link" href="#B36-biology-13-01088">36</a>] and Helmann et al. (2001) [<a aria-describedby="B35-biology-13-01088" class="usa-link" href="#B35-biology-13-01088">35</a>], which, if mutated, would be expected to reduce tolerance to temperature.</p>
<p>A novel and significant finding of our study was the convergence of mutations in DAC genes in several strains in both lineages. DAC genes are involved in the synthesis of c-di-AMP. Both <em>B. cereus</em> and <em>B. subtilis</em> possess three genes for the synthesis of c-di-AMP, DisA, and CdaA function in the vegetative phase, while CdaS is sporulation-specific [<a aria-describedby="B47-biology-13-01088" class="usa-link" href="#B47-biology-13-01088">47</a>]. Most studies have been carried out in <em>B. subtilis</em>, and it is known that the activity of CdaA is modulated by CdaR. c-di-AMP is essential for the growth of <em>B. subtilis</em> and DisA and CdaA contribute to modulating different DNA damage responses during exponential growth [<a aria-describedby="B48-biology-13-01088" class="usa-link" href="#B48-biology-13-01088">48</a>]. The parallel occurrence of mutations in DAC genes (<em>cdaR</em> in <em>B. subtilis</em> and <em>disA</em> in <em>B. cereus</em>, <a class="usa-link" href="#biology-13-01088-f009">Figure 9</a>) is intriguing. Several lines of evidence suggest that the di-adenylate cyclase CdaA is part of the conserved essential <em>cda-glm</em> module involved in cell wall metabolism. Notably, this molecule regulates potassium transport and osmotic balance [<a aria-describedby="B49-biology-13-01088" class="usa-link" href="#B49-biology-13-01088">49</a>]; when bacteria are challenged with high osmolarity, they acquire increased resistance to high temperature and oxidative stresses. The high-osmolarity-dependent increase in thermotolerance has two manifestations: the elevation of the upper limit of the growth temperature and enhanced survival at otherwise lethal high temperatures [<a aria-describedby="B50-biology-13-01088" class="usa-link" href="#B50-biology-13-01088">50</a>]. Although c-di-AMP is involved in many essential pathways, to participate in virulence and osmotic stress, it has not been implicated in thermal stress. The parallel occurrence of mutations in DAC genes (<em>cdaR</em> in <em>B. subtilis</em> and <em>disA</em> in <em>B. cereus</em>, <a class="usa-link" href="#biology-13-01088-f009">Figure 9</a>) strongly suggests its role in heat stress tolerance. Our results suggest a lineage-specific evolution in the modulation of the c-di-AMP regulation. In either case, potassium uptake is likely involved in mitigating membrane destabilization caused by high temperatures. In <em>B. subtilis</em> and many other bacteria, a sudden increase in the intracellular potassium concentration is the first response to osmotic stress [<a aria-describedby="B51-biology-13-01088" class="usa-link" href="#B51-biology-13-01088">51</a>]. Thus, the control of potassium homeostasis by c-di-AMP might also be part of a larger picture of the regulation of osmoadaptation. It has been shown that a variety of compatible solutes (i.e., glycine–betaine, proline) serve as heat protectants for <em>B. subtilis</em>. <em>B subtilis</em> has several redundant transporters, such that a mutation is one of the transporters that has no impact on heat stress [<a aria-describedby="B52-biology-13-01088" class="usa-link" href="#B52-biology-13-01088">52</a>] on molecular adaptation regardless of whether it restores physiological and molecular processes from a stressed state back toward the unstressed, wild-type state, or whether it instead tends to drive the evolution of novelty [<a aria-describedby="B53-biology-13-01088" class="usa-link" href="#B53-biology-13-01088">53</a>]. Previous studies have suggested the former because studies have shown that <em>E. coli</em> adapts to high-temperature stress (42.2 °C) by restoring both gene expression [<a aria-describedby="B54-biology-13-01088" class="usa-link" href="#B54-biology-13-01088">54</a>,<a aria-describedby="B55-biology-13-01088" class="usa-link" href="#B55-biology-13-01088">55</a>] and phenotypic characteristics [<a aria-describedby="B27-biology-13-01088" class="usa-link" href="#B27-biology-13-01088">27</a>] toward that of the unstressed ancestor. We think that expanding the thermal niche requires both adapting gene expression to a new threshold, as the previous level is inadequate for the permanent extra-limit temperature stress and the novelty to withstand permanent stress. In this sense, we suggest that cells regulate the levels of c-di-AMP to fit the novel stressful conditions. These findings open up new avenues for understanding the role of c-di-AMP in bacterial heat stress response.</p>
<p>Our experimental design, starting with two species and multiple strains, allowed us to explore the potential for parallel evolution [<a aria-describedby="B56-biology-13-01088" class="usa-link" href="#B56-biology-13-01088">56</a>] and identify multiple pathways leading to thermal tolerance. Parallel evolution, defined as the independent evolution of the same phenotype or genotype in response to identical selection pressures, provides insights into the predictability and diversity of evolutionary outcomes. In this study, we observed that while mutations affecting c-di-AMP synthesis emerged in all three <em>B. cereus</em> lineages, only one <em>B. subtilis</em> lineage (Bs_90) evolved a similar mutation. This finding suggests that while the modulation of c-di-AMP synthesis is a common and potentially “go-to” solution for thermal tolerance, it is not the only pathway. The presence of c-di-AMP mutations in multiple lineages across species highlights its broad utility, yet the absence of these mutations in some evolved lines underscores the diversity of evolutionary strategies. This observation supports the notion that standing genetic variation provides multiple routes to equivalent phenotypes, with natural selection shaping the most accessible solutions within specific genetic and physiological contexts.</p>
<p>Multiple pathways can lead to thermal tolerance, even under identical selective pressures, emphasizing the power of parallel evolution to reveal evolutionary constraints and opportunities. These findings align with the idea that evolution often proceeds along a single pathway within a given lineage but that alternative routes remain viable under different genetic or environmental conditions. By leveraging two species and multiple strains, our study underscores the importance of comparative approaches in experimental evolution for uncovering shared and lineage-specific adaptation mechanisms.</p>
<p>Our findings suggest that while different genetic pathways can lead to thermal tolerance, these pathways may converge on similar metabolic routes, such as osmotic protection. Mutations in genes regulating c-di-AMP synthesis highlight one such path, given its role in potassium transport and osmotic balance under stress. However, it is important to acknowledge a significant caveat in our analysis: genome sequencing identifies intragenic mutations that could be associated with the phenotype but cannot infer the impact of mutations in intergenic regions. Additionally, when multiple mutations occur within a genome, it is challenging to discern which specific mutations are responsible for the observed phenotypes. When working with wild strains, it is more challenging to reintroduce single mutations to validate the phenotype. Transcriptomic analyses would be necessary to uncover the overarching metabolic strategies and determine how these mutations integrate into the broader cellular response to thermal stress. Such analyses could reveal differential gene expression patterns and regulatory networks contributing to adaptation, providing a more comprehensive understanding of the metabolic adjustments underlying thermal tolerance.</p>
<p>The limited thermal tolerance observed in our study, even after experimental evolution, raises concerns about the capacity of microbial communities to adapt to climate change. Given the vital roles of <em>Bacillus</em> species in biogeochemical cycles, constraints on their thermal adaptability could have cascading effects on ecosystem functions, nutrient cycling, and soil health. Our findings suggest that while some bacterial populations may experience evolutionary rescue, the associated trade-offs could limit their ecological success and impact global processes.</p>
<p>It is notable that the three strains within each <em>Bacillus</em> lineage (<em>B. subtilis</em> and <em>B. cereus</em>) exhibited consistent, lineage-specific outcomes. Furthermore, the three independently evolved lines for each strain displayed similar trends in temperature tolerance, despite differences in the underlying mutations driving their responses. This result suggests a strong influence of both genetic background and lineage-specific traits in shaping the adaptive responses to temperature stress in <em>Bacillus</em> species. The consistency in outcomes within each lineage (<em>B. subtilis</em> and <em>B. cereus</em>) highlights that certain inherent genetic or physiological characteristics predispose these lineages to follow specific evolutionary trajectories.</p>
<p>Our observations indicate that <em>B. cereus</em> lineages tended to accumulate more mutations yet did not achieve markedly better fitness at higher temperatures, whereas <em>B. subtilis</em> lineages adapted to elevated temperatures with comparatively fewer mutations, suggesting greater inherent thermal plasticity or more efficient physiological and regulatory mechanisms for coping with thermal stress. Several factors may underlie these differences. First, <em>B. subtilis</em> may possess a more robust stress response system, including efficient chaperones, heat-shock proteins, and membrane adaptations that minimize the need for extensive genetic change. Second, differences in metabolic versatility and regulatory network flexibility may enable <em>B. subtilis</em> to rapidly optimize gene expression and protein function at higher temperatures, while <em>B. cereus</em> may rely more on genetic alterations that do not directly translate into fitness gains. Additionally, species-specific genomic architecture and mutation biases could shape distinct evolutionary trajectories, with <em>B. cereus</em> potentially having less efficient DNA repair or a mutational spectrum that fails to confer adaptive benefits under thermal stress. Furthermore, the evolutionary histories and ecological niches of these species influence their baseline adaptability; for instance, the laboratory-adapted <em>B. subtilis</em> strain-like PY79 may have evolved pre-adaptations suited to fluctuating conditions, whereas <em>B. cereus</em>, adapted to more stable environments, struggles to exploit novel thermal niches despite generating substantial genetic variation. For example, such lineage-specific adaptations have been documented by Guinebretière et al. (2008) [<a aria-describedby="B57-biology-13-01088" class="usa-link" href="#B57-biology-13-01088">57</a>], who showed that different taxonomic groups of <em>B. cereus</em> display lineage-associated thermal profiles ranging from 7 to 50 °C, explaining why <em>B. cereus</em> populations in warm parts of a dairy process differ from those in colder parts [<a aria-describedby="B58-biology-13-01088" class="usa-link" href="#B58-biology-13-01088">58</a>].</p>
<p>The observed lineage-specific outcomes and convergence within strains underscore the critical role of genetic background in shaping adaptive responses. However, the diversity of underlying mutations leading to similar phenotypic outcomes highlights the unpredictability of evolutionary pathways. This variability suggests that microbial responses to environmental stressors, such as climate change, may be highly context-dependent and difficult to generalize. Each genetic background provides a distinct set of constraints and opportunities, leading to diverse adaptive trajectories even under similar selective pressures.</p>
<p>Consequently, this result makes it more challenging to predict how microbial communities, composed of diverse genetic lineages, will respond to global warming. While some lineages may adapt via predictable mechanisms, others may follow entirely different and unforeseen evolutionary paths. This underscores the need for further studies that explore the interplay between genetic background, evolutionary dynamics, and environmental change to better understand and anticipate microbial responses at both the species and community levels.</p>
<p>In addition to well-characterized experimental evolution studies in <em>E. coli</em> (e.g., Tenaillon et al. [<a aria-describedby="B24-biology-13-01088" class="usa-link" href="#B24-biology-13-01088">24</a>]), recent research has increasingly explored evolutionary dynamics in <em>Bacillus</em> and other wild bacterial lineages under various selective pressures. For instance, <em>B. subtilis</em> has served as a model in experimental evolution to investigate adaptation in plant-associated environments and in the presence of competing species. Pomerleau et al. [<a aria-describedby="B59-biology-13-01088" class="usa-link" href="#B59-biology-13-01088">59</a>] demonstrated that <em>B. subtilis</em> rapidly evolved enhanced root colonization capacity under biotic stress imposed by <em>Pseudomonas</em> species, pinpointing mutations in regulators linked to biofilm development. Likewise, Blake et al. [<a aria-describedby="B60-biology-13-01088" class="usa-link" href="#B60-biology-13-01088">60</a>] showed that <em>B. subtilis</em> populations adapted to the rhizosphere of <em>Arabidopsis thaliana</em> diversified into distinct lineages, with some evolved strains improving their host-specific colonization abilities. These findings highlight how environmental context shapes adaptive trajectories and can promote genetic diversification.</p>
<p>Other studies emphasize the role of genetic mechanisms, such as horizontal gene transfer (HGT), in shaping adaptation. Slomka et al. [<a aria-describedby="B61-biology-13-01088" class="usa-link" href="#B61-biology-13-01088">61</a>] found that <em>B. subtilis</em> populations evolving under high-salt conditions acquired extensive foreign DNA, improving salt tolerance via bursts of HGT. Although our study focused on thermal adaptation rather than osmotic stress, these results underscore the importance of both point mutations and gene acquisition in driving evolutionary responses and raise the question of whether thermal adaptation might also draw on similar genetic sources if available.</p>
<p>Environmental fluctuations and life-history traits can further influence evolutionary outcomes. Shoemaker et al. [<a aria-describedby="B62-biology-13-01088" class="usa-link" href="#B62-biology-13-01088">62</a>] reported that the capacity for dormancy (“seed banking”) in <em>B. subtilis</em> alters the trajectory of molecular evolution over extended timescales, increasing genetic diversity and affecting the direction of selection on certain genes. Although our experimental design did not directly examine dormancy states, such findings suggest that the intrinsic traits of species, including the ability to form spores, may shape the pace and path of thermal adaptation.</p>
<p>Finally, recent molecular and genomic studies highlight the complexity of responses at the RNA and gene-regulatory levels. For example, Jolley et al. [<a aria-describedby="B63-biology-13-01088" class="usa-link" href="#B63-biology-13-01088">63</a>] demonstrated that <em>B. subtilis</em> dynamically adjusts its RNA structure in response to temperature changes, potentially providing an additional layer of rapid, non-mutational adaptation. Similarly, Hawkins et al. [<a aria-describedby="B64-biology-13-01088" class="usa-link" href="#B64-biology-13-01088">64</a>] and Shi et al. [<a aria-describedby="B65-biology-13-01088" class="usa-link" href="#B65-biology-13-01088">65</a>] explored how essential genes and sporulation-related loci evolve over time, offering insights into how baseline genomic architecture and gene regulatory networks can influence adaptive potential.</p>
<p>Taken together, these studies show that <em>Bacillus</em> lineages can evolve through multiple genetic routes—ranging from single-nucleotide changes and HGT events to alterations in regulatory networks—to cope with diverse selective pressures, including thermal stress. They also reveal common themes, such as the role of regulatory mutations and pre-existing physiological traits, which can set certain lineages on more efficient adaptive trajectories. Integrating this broader perspective into our findings suggests that while the mutation rate and the spectrum of mutations we observed may not necessarily correlate with thermal niche expansion, they reflect a complex interplay of genetic background, regulatory flexibility, ecological context, and evolutionary history. By situating our results alongside these other <em>Bacillus</em> and bacterial evolution studies, we gain a more nuanced understanding of how constraints and contingencies influence the adaptive landscape of bacterial populations facing environmental challenges.</p>
<p>The experimental constraints of this study, which included the passes for only 1000 generations and laboratory-based conditions, may not fully reflect the complexities of long-term evolutionary dynamics or the ecological interactions that occur in natural settings. Additionally, while we identified several mutations, especially those related to di-adenylate cyclase (DAC) genes, more studies would be necessary to understand their roles. This could involve transcriptomic analysis to better understand gene expression patterns in response to thermal stress. Furthermore, exploring the broader ecological context of these findings, such as how they translate to natural microbial populations and how bacteria respond to other environmental stressors like desiccation or nutrient scarcity, will be essential to evaluate bacterial adaptability and resilience comprehensively.</p></section><section id="sec5-biology-13-01088"><h2 class="pmc_sec_title">5. Conclusions</h2>
<p>Our results demonstrate that <em>Bacillus</em> strains exhibit limited thermal adaptation and are lineage-specific, with immediate trade-offs even under prolonged selective pressure. This constrained evolvability poses significant implications for microbial responses to global warming, potentially affecting biogeochemical cycles and ecosystem stability. Understanding these evolutionary limits is crucial as we prepare for the ecological impacts of climate change.</p>
<p>This study emphasizes the need for more research into the evolutionary and ecological limits of microbial adaptability to predict and mitigate the broader consequences of climate change on vital biogeochemical processes.</p></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Leonardo Meléndez Cerda for his participation in the amplification and sequencing to validate many of the gene mutations and Antonio Cisneros for photographic work.</p></section><section class="app" id="app1-biology-13-01088"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<p>The following supporting information can be downloaded at: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.mdpi.com/article/10.3390/biology13121088/s1" rel="noopener noreferrer" target="_blank">https://www.mdpi.com/article/10.3390/biology13121088/s1</a>. Supplementary Figure S1A,B. Recovery Ability of <em>B. subtilis</em> and <em>B. cereus</em> Strains After Incubation at High Temperatures. Supplementary Figure S2. Growth phenotypes on semisolid medium of Evolved <em>B. subtilis</em> and <em>B. cereus</em> Strains at High Temperature. Supplementary Figure S3. Workflow of Bioinformatic Analysis of Evolved Genomes and Mutations. Supplementary Figure S4. Principal Component Analysis (PCA) of Experimental Evolution Treatments. Supplementary Figure S5. Criterion for Classification of Hypermutant Phenotypes. Supplementary Figure S6. Sporulation Loss and Mutation Type Quantification. Supplementary Figure S7. Open reading frame analysis of intragenic mutations in selected genes. Supplementary Table S1. Raw Data for Mutation Counts and Sporulation Loss (Excel table). Supplementary Table S2. Evolved lines and chosen regions re-sequenced to validate mutations and Oligonucleotides used for the verification of Mutations. Supplementary Table S3. Mutations validated by PCR amplification and sequencing.</p>
<section class="sm xbox font-sm" id="biology-13-01088-s001"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11673879/bin/biology-13-01088-s001.zip">biology-13-01088-s001.zip</a><sup> (1.2MB, zip) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>Conceptualization, G.O.-A. and E.H.-B.; methodology, E.H.-B., G.M.-H. and A.I.-R.; validation, A.I.-R.; formal analysis, E.H.-B. and G.M.-H.; investigation, E.H.-B., A.I.-R., G.M.-H. and G.O.-A.; resources, A.I.-R.; Resources, A.I.-R. and G.M.-H.; data curation, E.H.-B. and G.M.-H.; writing—original draft preparation, E.H.-B. and G.O.-A.; writing—review and editing, E.H.-B., G.M.-H. and G.O.-A.; supervision, G.O.-A.; project administration, A.I.-R.; funding acquisition, G.O.-A. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Institutional Review Board Statement</h2>
<p>Not applicable.</p></section><section id="notes3"><h2 class="pmc_sec_title">Informed Consent Statement</h2>
<p>Not applicable.</p></section><section id="notes4"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The strains are available upon request to interested researchers.</p></section><section id="notes5"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This research was funded by Conahcyt Ciencia de Frontera grant number 39589. E.H.-B. acknowledges a scholarship from Conahcyt.</p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Disclaimer/Publisher’s Note:</strong> The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.</p></div></div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1-biology-13-01088">
<span class="label">1.</span><cite>Armstrong Mckay A., Staal D.I., Abrams A., Winkelmann J.F., Sakschewski R., Loriani B., Fetzer S., Cornell I., Rockström S.E. Exceeding 1.5 °C Global Warming Could Trigger Multiple Climate Tipping Points. Science. 2022;377:eabn7950. doi: 10.1126/science.abn7950.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.abn7950" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36074831/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Exceeding%201.5%20%C2%B0C%20Global%20Warming%20Could%20Trigger%20Multiple%20Climate%20Tipping%20Points&amp;author=A.%20Armstrong%20Mckay&amp;author=D.I.%20Staal&amp;author=A.%20Abrams&amp;author=J.F.%20Winkelmann&amp;author=R.%20Sakschewski&amp;volume=377&amp;publication_year=2022&amp;pages=eabn7950&amp;pmid=36074831&amp;doi=10.1126/science.abn7950&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2-biology-13-01088">
<span class="label">2.</span><cite>Grabowska-Grucza K., Bukowska A., Siuda W., Chróst R.J., Kiersztyn B. Impact of Increasing Temperature on the Taxonomic and Metabolic Structure of Bacterial Communities in a Global Warming Context. Aquat. Microb. Ecol. 2022;88:135–148. doi: 10.3354/ame01988.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3354/ame01988" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Aquat.%20Microb.%20Ecol.&amp;title=Impact%20of%20Increasing%20Temperature%20on%20the%20Taxonomic%20and%20Metabolic%20Structure%20of%20Bacterial%20Communities%20in%20a%20Global%20Warming%20Context&amp;author=K.%20Grabowska-Grucza&amp;author=A.%20Bukowska&amp;author=W.%20Siuda&amp;author=R.J.%20Chr%C3%B3st&amp;author=B.%20Kiersztyn&amp;volume=88&amp;publication_year=2022&amp;pages=135-148&amp;doi=10.3354/ame01988&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3-biology-13-01088">
<span class="label">3.</span><cite>Santos H.F., Carmo F.L., Duarte G., Dini-Andreote F., Castro C.B., Rosado A.S., van Elsas J.D., Peixoto R.S. Climate Change Affects Key Nitrogen-Fixing Bacterial Populations on Coral Reefs. ISME J. 2014;8:2272–2279. doi: 10.1038/ismej.2014.70.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ismej.2014.70" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4992079/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24830827/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=Climate%20Change%20Affects%20Key%20Nitrogen-Fixing%20Bacterial%20Populations%20on%20Coral%20Reefs&amp;author=H.F.%20Santos&amp;author=F.L.%20Carmo&amp;author=G.%20Duarte&amp;author=F.%20Dini-Andreote&amp;author=C.B.%20Castro&amp;volume=8&amp;publication_year=2014&amp;pages=2272-2279&amp;pmid=24830827&amp;doi=10.1038/ismej.2014.70&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4-biology-13-01088">
<span class="label">4.</span><cite>Bennett J.M., Sunday J., Calosi P., Villalobos F., Martínez B., Molina-Venegas R., Araújo M.B., Algar A.C., Clusella-Trullas S., Hawkins B.A., et al.  The Evolution of Critical Thermal Limits of Life on Earth. Nat. Commun. 2021;12:1198. doi: 10.1038/s41467-021-21263-8.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41467-021-21263-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7895938/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33608528/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=The%20Evolution%20of%20Critical%20Thermal%20Limits%20of%20Life%20on%20Earth&amp;author=J.M.%20Bennett&amp;author=J.%20Sunday&amp;author=P.%20Calosi&amp;author=F.%20Villalobos&amp;author=B.%20Mart%C3%ADnez&amp;volume=12&amp;publication_year=2021&amp;pages=1198&amp;pmid=33608528&amp;doi=10.1038/s41467-021-21263-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5-biology-13-01088">
<span class="label">5.</span><cite>Rodríguez-Verdugo A., Lozano-Huntelman N., Cruz-Loya M., Savage V., Yeh P. Compounding Effects of Climate Warming and Antibiotic Resistance. iScience. 2020;23:101024. doi: 10.1016/j.isci.2020.101024.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.isci.2020.101024" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7160571/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32299057/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=Compounding%20Effects%20of%20Climate%20Warming%20and%20Antibiotic%20Resistance&amp;author=A.%20Rodr%C3%ADguez-Verdugo&amp;author=N.%20Lozano-Huntelman&amp;author=M.%20Cruz-Loya&amp;author=V.%20Savage&amp;author=P.%20Yeh&amp;volume=23&amp;publication_year=2020&amp;pages=101024&amp;pmid=32299057&amp;doi=10.1016/j.isci.2020.101024&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6-biology-13-01088">
<span class="label">6.</span><cite>Vezzulli L., Pezzati E., Brettar I., Höfle M., Pruzzo C. Effects of Global Warming on Vibrio Ecology. Microbiol. Spectr. 2015;3 doi: 10.1128/microbiolspec.VE-0004-2014.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/microbiolspec.VE-0004-2014" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26185070/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Spectr.&amp;title=Effects%20of%20Global%20Warming%20on%20Vibrio%20Ecology&amp;author=L.%20Vezzulli&amp;author=E.%20Pezzati&amp;author=I.%20Brettar&amp;author=M.%20H%C3%B6fle&amp;author=C.%20Pruzzo&amp;volume=3&amp;publication_year=2015&amp;pmid=26185070&amp;doi=10.1128/microbiolspec.VE-0004-2014&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7-biology-13-01088">
<span class="label">7.</span><cite>Murren C.J., Auld J.R., Callahan H., Ghalambor C.K., Handelsman C.A., Heskel M.A., Kingsolver J.G., Maclean H.J., Masel J., Maughan H., et al.  Constraints on the Evolution of Phenotypic Plasticity: Limits and Costs of Phenotype and Plasticity. Heredity. 2015;115:293–301. doi: 10.1038/hdy.2015.8.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/hdy.2015.8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4815460/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25690179/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Constraints%20on%20the%20Evolution%20of%20Phenotypic%20Plasticity:%20Limits%20and%20Costs%20of%20Phenotype%20and%20Plasticity&amp;author=C.J.%20Murren&amp;author=J.R.%20Auld&amp;author=H.%20Callahan&amp;author=C.K.%20Ghalambor&amp;author=C.A.%20Handelsman&amp;volume=115&amp;publication_year=2015&amp;pages=293-301&amp;pmid=25690179&amp;doi=10.1038/hdy.2015.8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8-biology-13-01088">
<span class="label">8.</span><cite>van Gestel J., Weissing F.J. Regulatory Mechanisms Link Phenotypic Plasticity to Evolvability. Sci. Rep. 2016;6:24524.  doi: 10.1038/srep24524.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/srep24524" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4834480/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27087393/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Regulatory%20Mechanisms%20Link%20Phenotypic%20Plasticity%20to%20Evolvability&amp;author=J.%20van%20Gestel&amp;author=F.J.%20Weissing&amp;volume=6&amp;publication_year=2016&amp;pages=24524&amp;pmid=27087393&amp;doi=10.1038/srep24524&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9-biology-13-01088">
<span class="label">9.</span><cite>Pigliucci M., Murren C.J. Perspective: Genetic Assimilation and a Possible Evolutionary Paradox: Can Macroevolution Sometimes Be so Fast as to Pass Us By? Evolution. 2003;57:1455–1464. doi: 10.1111/j.0014-3820.2003.tb00354.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.0014-3820.2003.tb00354.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12940351/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Perspective:%20Genetic%20Assimilation%20and%20a%20Possible%20Evolutionary%20Paradox:%20Can%20Macroevolution%20Sometimes%20Be%20so%20Fast%20as%20to%20Pass%20Us%20By?&amp;author=M.%20Pigliucci&amp;author=C.J.%20Murren&amp;volume=57&amp;publication_year=2003&amp;pages=1455-1464&amp;pmid=12940351&amp;doi=10.1111/j.0014-3820.2003.tb00354.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10-biology-13-01088">
<span class="label">10.</span><cite>Snell-Rood E.C., Van Dyken J.D., Cruickshank T., Wade M.J., Moczek A.P. Toward a Population Genetic Framework of Developmental Evolution: The Costs, Limits, and Consequences of Phenotypic Plasticity. Bioessays. 2010;32:71–81. doi: 10.1002/bies.200900132.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/bies.200900132" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3151734/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20020499/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioessays&amp;title=Toward%20a%20Population%20Genetic%20Framework%20of%20Developmental%20Evolution:%20The%20Costs,%20Limits,%20and%20Consequences%20of%20Phenotypic%20Plasticity&amp;author=E.C.%20Snell-Rood&amp;author=J.D.%20Van%20Dyken&amp;author=T.%20Cruickshank&amp;author=M.J.%20Wade&amp;author=A.P.%20Moczek&amp;volume=32&amp;publication_year=2010&amp;pages=71-81&amp;pmid=20020499&amp;doi=10.1002/bies.200900132&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11-biology-13-01088">
<span class="label">11.</span><cite>Woltereck R. Weitere Experimentelle Untersuchungen Uber Artveranderung, Speziell Uberdas Wesen Quantitativer Artunterschyiede Bei Daphniden. Verhandlungen Dtsch. Zool. Ges. 1909;1909:110–172.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Verhandlungen%20Dtsch.%20Zool.%20Ges.&amp;title=Weitere%20Experimentelle%20Untersuchungen%20Uber%20Artveranderung,%20Speziell%20Uberdas%20Wesen%20Quantitativer%20Artunterschyiede%20Bei%20Daphniden&amp;author=R.%20Woltereck&amp;volume=1909&amp;publication_year=1909&amp;pages=110-172&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12-biology-13-01088">
<span class="label">12.</span><cite>Stearns S.C. The Evolutionary Significance of Phenotypic Plasticity. Bioscience. 1989;39:436–445. doi: 10.2307/1311135.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2307/1311135" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioscience&amp;title=The%20Evolutionary%20Significance%20of%20Phenotypic%20Plasticity&amp;author=S.C.%20Stearns&amp;volume=39&amp;publication_year=1989&amp;pages=436-445&amp;doi=10.2307/1311135&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13-biology-13-01088">
<span class="label">13.</span><cite>Bennett A.F., Lenski R.E. Evolutionary Adaptation to Temperature II. Thermal Niches of Experimental Lines of Escherichia coli. Evolution. 1993;47:1–12. doi: 10.2307/2410113.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2307/2410113" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28568084/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Evolutionary%20Adaptation%20to%20Temperature%20II.%20Thermal%20Niches%20of%20Experimental%20Lines%20of%20Escherichia%20coli&amp;author=A.F.%20Bennett&amp;author=R.E.%20Lenski&amp;volume=47&amp;publication_year=1993&amp;pages=1-12&amp;pmid=28568084&amp;doi=10.2307/2410113&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14-biology-13-01088">
<span class="label">14.</span><cite>Hurtado-Bautista E., Pérez Sánchez L.F., Islas-Robles A., Santoyo G., Olmedo-Alvarez G. Phenotypic Plasticity and Evolution of Thermal Tolerance in Bacteria from Temperate and Hot Spring Environments. PeerJ. 2021;9:e11734. doi: 10.7717/peerj.11734.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.7717/peerj.11734" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8312496/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34386300/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Phenotypic%20Plasticity%20and%20Evolution%20of%20Thermal%20Tolerance%20in%20Bacteria%20from%20Temperate%20and%20Hot%20Spring%20Environments&amp;author=E.%20Hurtado-Bautista&amp;author=L.F.%20P%C3%A9rez%20S%C3%A1nchez&amp;author=A.%20Islas-Robles&amp;author=G.%20Santoyo&amp;author=G.%20Olmedo-Alvarez&amp;volume=9&amp;publication_year=2021&amp;pages=e11734&amp;pmid=34386300&amp;doi=10.7717/peerj.11734&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15-biology-13-01088">
<span class="label">15.</span><cite>Schumann W. Regulation of Bacterial Heat Shock Stimulons. Cell Stress Chaperones. 2016;21:959–968. doi: 10.1007/s12192-016-0727-z.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s12192-016-0727-z" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5083672/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27518094/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell%20Stress%20Chaperones&amp;title=Regulation%20of%20Bacterial%20Heat%20Shock%20Stimulons&amp;author=W.%20Schumann&amp;volume=21&amp;publication_year=2016&amp;pages=959-968&amp;pmid=27518094&amp;doi=10.1007/s12192-016-0727-z&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16-biology-13-01088">
<span class="label">16.</span><cite>Craig E.A. The Heat Shock Response. CRC Crit. Rev. Biochem. 1985;18:239–280. doi: 10.3109/10409238509085135.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3109/10409238509085135" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/2412760/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=CRC%20Crit.%20Rev.%20Biochem.&amp;title=The%20Heat%20Shock%20Response&amp;author=E.A.%20Craig&amp;volume=18&amp;publication_year=1985&amp;pages=239-280&amp;pmid=2412760&amp;doi=10.3109/10409238509085135&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17-biology-13-01088">
<span class="label">17.</span><cite>Arsène F., Tomoyasu T., Bukau B. The Heat Shock Response of Escherichia coli. Int. J. Food Microbiol. 2000;55:3–9. doi: 10.1016/S0168-1605(00)00206-3.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0168-1605(00)00206-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10791710/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Food%20Microbiol.&amp;title=The%20Heat%20Shock%20Response%20of%20Escherichia%20coli&amp;author=F.%20Ars%C3%A8ne&amp;author=T.%20Tomoyasu&amp;author=B.%20Bukau&amp;volume=55&amp;publication_year=2000&amp;pages=3-9&amp;pmid=10791710&amp;doi=10.1016/S0168-1605(00)00206-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18-biology-13-01088">
<span class="label">18.</span><cite>Elena S.F., Lenski R.E. Evolution Experiments with Microorganisms: The Dynamics and Genetic Bases of Adaptation: Microbial Genetics. Nat. Rev. Genet. 2003;4:457–469. doi: 10.1038/nrg1088.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nrg1088" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12776215/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=Evolution%20Experiments%20with%20Microorganisms:%20The%20Dynamics%20and%20Genetic%20Bases%20of%20Adaptation:%20Microbial%20Genetics&amp;author=S.F.%20Elena&amp;author=R.E.%20Lenski&amp;volume=4&amp;publication_year=2003&amp;pages=457-469&amp;pmid=12776215&amp;doi=10.1038/nrg1088&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19-biology-13-01088">
<span class="label">19.</span><cite>Connolly J.P.R., Roe A.J., O’Boyle N. Prokaryotic Life Finds a Way: Insights from Evolutionary Experimentation in Bacteria. Crit. Rev. Microbiol. 2021;47:126–140. doi: 10.1080/1040841X.2020.1854172.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/1040841X.2020.1854172" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33332206/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Microbiol.&amp;title=Prokaryotic%20Life%20Finds%20a%20Way:%20Insights%20from%20Evolutionary%20Experimentation%20in%20Bacteria&amp;author=J.P.R.%20Connolly&amp;author=A.J.%20Roe&amp;author=N.%20O%E2%80%99Boyle&amp;volume=47&amp;publication_year=2021&amp;pages=126-140&amp;pmid=33332206&amp;doi=10.1080/1040841X.2020.1854172&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B20-biology-13-01088">
<span class="label">20.</span><cite>Agrawal A.A. Toward a Predictive Framework for Convergent Evolution: Integrating Natural History, Genetic Mechanisms, and Consequences for the Diversity of Life. Am. Nat. 2017;190:S1–S12. doi: 10.1086/692111.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1086/692111" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28731831/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am.%20Nat.&amp;title=Toward%20a%20Predictive%20Framework%20for%20Convergent%20Evolution:%20Integrating%20Natural%20History,%20Genetic%20Mechanisms,%20and%20Consequences%20for%20the%20Diversity%20of%20Life&amp;author=A.A.%20Agrawal&amp;volume=190&amp;publication_year=2017&amp;pages=S1-S12&amp;pmid=28731831&amp;doi=10.1086/692111&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21-biology-13-01088">
<span class="label">21.</span><cite>Hao Y., Qu Y., Song G., Lei F. Genomic Insights into the Adaptive Convergent Evolution. Curr. Genom. 2019;20:81–89. doi: 10.2174/1389202920666190313162702.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2174/1389202920666190313162702" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6728901/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31555059/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Genom.&amp;title=Genomic%20Insights%20into%20the%20Adaptive%20Convergent%20Evolution&amp;author=Y.%20Hao&amp;author=Y.%20Qu&amp;author=G.%20Song&amp;author=F.%20Lei&amp;volume=20&amp;publication_year=2019&amp;pages=81-89&amp;pmid=31555059&amp;doi=10.2174/1389202920666190313162702&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22-biology-13-01088">
<span class="label">22.</span><cite>Bennett A.F., Dao K.M., Lenski R.E. Rapid Evolution in Response to High-Temperature Selection. Nature. 1990;346:79–81. doi: 10.1038/346079a0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/346079a0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/2195353/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Rapid%20Evolution%20in%20Response%20to%20High-Temperature%20Selection&amp;author=A.F.%20Bennett&amp;author=K.M.%20Dao&amp;author=R.E.%20Lenski&amp;volume=346&amp;publication_year=1990&amp;pages=79-81&amp;pmid=2195353&amp;doi=10.1038/346079a0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23-biology-13-01088">
<span class="label">23.</span><cite>Riehle M.M., Bennett A.F., Long A.D. Genetic Architecture of Thermal Adaptation in Escherichia coli. Proc. Natl. Acad. Sci. USA. 2001;98:525–530. doi: 10.1073/pnas.98.2.525.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.98.2.525" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC14620/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11149947/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Genetic%20Architecture%20of%20Thermal%20Adaptation%20in%20Escherichia%20coli&amp;author=M.M.%20Riehle&amp;author=A.F.%20Bennett&amp;author=A.D.%20Long&amp;volume=98&amp;publication_year=2001&amp;pages=525-530&amp;pmid=11149947&amp;doi=10.1073/pnas.98.2.525&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24-biology-13-01088">
<span class="label">24.</span><cite>Tenaillon O., Rodríguez-Verdugo A., Gaut R.L., McDonald P., Bennett A.F., Long A.D., Gaut B.S. The Molecular Diversity of Adaptive Convergence. Science. 2012;335:457–461. doi: 10.1126/science.1212986.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1212986" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22282810/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20Molecular%20Diversity%20of%20Adaptive%20Convergence&amp;author=O.%20Tenaillon&amp;author=A.%20Rodr%C3%ADguez-Verdugo&amp;author=R.L.%20Gaut&amp;author=P.%20McDonald&amp;author=A.F.%20Bennett&amp;volume=335&amp;publication_year=2012&amp;pages=457-461&amp;pmid=22282810&amp;doi=10.1126/science.1212986&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25-biology-13-01088">
<span class="label">25.</span><cite>Bennett A.F., Lenski R.E. evolutionary adaptation to temperature. Vi. Phenotypic acclimation and its evolution in Escherichia coli. Evolution. 1997;51:36–44. doi: 10.2307/2410958.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2307/2410958" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28568801/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=evolutionary%20adaptation%20to%20temperature.%20Vi.%20Phenotypic%20acclimation%20and%20its%20evolution%20in%20Escherichia%20coli&amp;author=A.F.%20Bennett&amp;author=R.E.%20Lenski&amp;volume=51&amp;publication_year=1997&amp;pages=36-44&amp;pmid=28568801&amp;doi=10.2307/2410958&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26-biology-13-01088">
<span class="label">26.</span><cite>Murata M., Ishii A., Fujimoto H., Nishimura K., Kosaka T., Mori H., Yamada M. Update of Thermotolerant Genes Essential for Survival at a Critical High Temperature in Escherichia coli. PLoS ONE. 2018;13:e0189487.  doi: 10.1371/journal.pone.0189487.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0189487" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5828445/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29485997/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Update%20of%20Thermotolerant%20Genes%20Essential%20for%20Survival%20at%20a%20Critical%20High%20Temperature%20in%20Escherichia%20coli&amp;author=M.%20Murata&amp;author=A.%20Ishii&amp;author=H.%20Fujimoto&amp;author=K.%20Nishimura&amp;author=T.%20Kosaka&amp;volume=13&amp;publication_year=2018&amp;pages=e0189487&amp;pmid=29485997&amp;doi=10.1371/journal.pone.0189487&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27-biology-13-01088">
<span class="label">27.</span><cite>Batarseh T.N., Hug S.M., Batarseh S.N., Gaut B.S. Genetic Mutations That Drive Evolutionary Rescue to Lethal Temperature in Escherichia coli. Genome Biol. Evol. 2020;12:2029–2044. doi: 10.1093/gbe/evaa174.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/gbe/evaa174" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7750951/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32785667/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&amp;title=Genetic%20Mutations%20That%20Drive%20Evolutionary%20Rescue%20to%20Lethal%20Temperature%20in%20Escherichia%20coli&amp;author=T.N.%20Batarseh&amp;author=S.M.%20Hug&amp;author=S.N.%20Batarseh&amp;author=B.S.%20Gaut&amp;volume=12&amp;publication_year=2020&amp;pages=2029-2044&amp;pmid=32785667&amp;doi=10.1093/gbe/evaa174&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28-biology-13-01088">
<span class="label">28.</span><cite>Mongold J.A., Bennett A.F., Lenski R.E. Evolutionary adaptation to temperature. Vii. Extension of the upper thermal limit of Escherichia coli. Evolution. 1999;53:386–394. doi: 10.2307/2640775.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2307/2640775" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28565406/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Evolutionary%20adaptation%20to%20temperature.%20Vii.%20Extension%20of%20the%20upper%20thermal%20limit%20of%20Escherichia%20coli&amp;author=J.A.%20Mongold&amp;author=A.F.%20Bennett&amp;author=R.E.%20Lenski&amp;volume=53&amp;publication_year=1999&amp;pages=386-394&amp;pmid=28565406&amp;doi=10.2307/2640775&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29-biology-13-01088">
<span class="label">29.</span><cite>Rodríguez-Torres M.D., Islas-Robles Á., Gómez-Lunar Z., Delaye L., Hernández-González I., Souza V., Travisano M., Olmedo-Álvarez G. Phenotypic Microdiversity and Phylogenetic Signal Analysis of Traits Related to Social Interaction in Bacillus spp. from Sediment Communities. Front. Microbiol. 2017;8:29.  doi: 10.3389/fmicb.2017.00029.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2017.00029" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5276817/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28194138/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Phenotypic%20Microdiversity%20and%20Phylogenetic%20Signal%20Analysis%20of%20Traits%20Related%20to%20Social%20Interaction%20in%20Bacillus%20spp.%20from%20Sediment%20Communities&amp;author=M.D.%20Rodr%C3%ADguez-Torres&amp;author=%C3%81.%20Islas-Robles&amp;author=Z.%20G%C3%B3mez-Lunar&amp;author=L.%20Delaye&amp;author=I.%20Hern%C3%A1ndez-Gonz%C3%A1lez&amp;volume=8&amp;publication_year=2017&amp;pages=29&amp;pmid=28194138&amp;doi=10.3389/fmicb.2017.00029&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30-biology-13-01088">
<span class="label">30.</span><cite>Cerritos R., Eguiarte L.E., Avitia M., Siefert J., Travisano M., Rodríguez-Verdugo A., Souza V. Diversity of Culturable Thermo-Resistant Aquatic Bacteria along an Environmental Gradient in Cuatro Ciénegas, Coahuila, México. Antonie Leeuwenhoek. 2011;99:303–318. doi: 10.1007/s10482-010-9490-9.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s10482-010-9490-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20711674/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Antonie%20Leeuwenhoek&amp;title=Diversity%20of%20Culturable%20Thermo-Resistant%20Aquatic%20Bacteria%20along%20an%20Environmental%20Gradient%20in%20Cuatro%20Ci%C3%A9negas,%20Coahuila,%20M%C3%A9xico&amp;author=R.%20Cerritos&amp;author=L.E.%20Eguiarte&amp;author=M.%20Avitia&amp;author=J.%20Siefert&amp;author=M.%20Travisano&amp;volume=99&amp;publication_year=2011&amp;pages=303-318&amp;pmid=20711674&amp;doi=10.1007/s10482-010-9490-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31-biology-13-01088">
<span class="label">31.</span><cite>Deatherage D.E., Kepner J.L., Bennett A.F., Lenski R.E., Barrick J.E. Specificity of Genome Evolution in Experimental Populations of Escherichia coli Evolved at Different Temperatures. Proc. Natl. Acad. Sci. USA. 2017;114:E1904–E1912. doi: 10.1073/pnas.1616132114.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1616132114" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5347587/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28202733/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Specificity%20of%20Genome%20Evolution%20in%20Experimental%20Populations%20of%20Escherichia%20coli%20Evolved%20at%20Different%20Temperatures&amp;author=D.E.%20Deatherage&amp;author=J.L.%20Kepner&amp;author=A.F.%20Bennett&amp;author=R.E.%20Lenski&amp;author=J.E.%20Barrick&amp;volume=114&amp;publication_year=2017&amp;pages=E1904-E1912&amp;pmid=28202733&amp;doi=10.1073/pnas.1616132114&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32-biology-13-01088">
<span class="label">32.</span><cite>Bhandari V., Ahmod N.Z., Shah H.N., Gupta R.S. Molecular Signatures for Bacillus Species: Demarcation of the Bacillus Subtilis and Bacillus cereus Clades in Molecular Terms and Proposal to Limit the Placement of New Species into the Genus Bacillus. Int. J. Syst. Evol. Microbiol. 2013;63:2712–2726. doi: 10.1099/ijs.0.048488-0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1099/ijs.0.048488-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23475340/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;title=Molecular%20Signatures%20for%20Bacillus%20Species:%20Demarcation%20of%20the%20Bacillus%20Subtilis%20and%20Bacillus%20cereus%20Clades%20in%20Molecular%20Terms%20and%20Proposal%20to%20Limit%20the%20Placement%20of%20New%20Species%20into%20the%20Genus%20Bacillus&amp;author=V.%20Bhandari&amp;author=N.Z.%20Ahmod&amp;author=H.N.%20Shah&amp;author=R.S.%20Gupta&amp;volume=63&amp;publication_year=2013&amp;pages=2712-2726&amp;pmid=23475340&amp;doi=10.1099/ijs.0.048488-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B33-biology-13-01088">
<span class="label">33.</span><cite>Hernández-González I.L., Moreno-Hagelsieb G., Olmedo-Álvarez G. Environmentally-Driven Gene Content Convergence and the Bacillus Phylogeny. BMC Evol. Biol. 2018;18:148.  doi: 10.1186/s12862-018-1261-7.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12862-018-1261-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6171248/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30285626/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evol.%20Biol.&amp;title=Environmentally-Driven%20Gene%20Content%20Convergence%20and%20the%20Bacillus%20Phylogeny&amp;author=I.L.%20Hern%C3%A1ndez-Gonz%C3%A1lez&amp;author=G.%20Moreno-Hagelsieb&amp;author=G.%20Olmedo-%C3%81lvarez&amp;volume=18&amp;publication_year=2018&amp;pages=148&amp;pmid=30285626&amp;doi=10.1186/s12862-018-1261-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34-biology-13-01088">
<span class="label">34.</span><cite>Pérez-Gutiérrez R.-A., López-Ramírez V., Islas Á., Alcaraz L.D., Hernández-González I., Olivera B.C.L., Santillán M., Eguiarte L.E., Souza V., Travisano M., et al.  Antagonism Influences Assembly of a Bacillus Guild in a Local Community and Is Depicted as a Food-Chain Network. ISME J. 2013;7:487–497. doi: 10.1038/ismej.2012.119.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ismej.2012.119" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3578566/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23096405/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=Antagonism%20Influences%20Assembly%20of%20a%20Bacillus%20Guild%20in%20a%20Local%20Community%20and%20Is%20Depicted%20as%20a%20Food-Chain%20Network&amp;author=R.-A.%20P%C3%A9rez-Guti%C3%A9rrez&amp;author=V.%20L%C3%B3pez-Ram%C3%ADrez&amp;author=%C3%81.%20Islas&amp;author=L.D.%20Alcaraz&amp;author=I.%20Hern%C3%A1ndez-Gonz%C3%A1lez&amp;volume=7&amp;publication_year=2013&amp;pages=487-497&amp;pmid=23096405&amp;doi=10.1038/ismej.2012.119&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35-biology-13-01088">
<span class="label">35.</span><cite>Helmann J.D., Wu M.F., Kobel P.A., Gamo F.J., Wilson M., Morshedi M.M., Navre M., Paddon C. Global Transcriptional Response of Bacillus subtilis to Heat Shock. J. Bacteriol. 2001;183:7318–7328. doi: 10.1128/JB.183.24.7318-7328.2001.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JB.183.24.7318-7328.2001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC95581/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11717291/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&amp;title=Global%20Transcriptional%20Response%20of%20Bacillus%20subtilis%20to%20Heat%20Shock&amp;author=J.D.%20Helmann&amp;author=M.F.%20Wu&amp;author=P.A.%20Kobel&amp;author=F.J.%20Gamo&amp;author=M.%20Wilson&amp;volume=183&amp;publication_year=2001&amp;pages=7318-7328&amp;pmid=11717291&amp;doi=10.1128/JB.183.24.7318-7328.2001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36-biology-13-01088">
<span class="label">36.</span><cite>Hecker M., Schumann W., Völker U. Heat-Shock and General Stress Response in Bacillus subtilis. Mol. Microbiol. 1996;19:417–428. doi: 10.1046/j.1365-2958.1996.396932.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1365-2958.1996.396932.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8830234/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&amp;title=Heat-Shock%20and%20General%20Stress%20Response%20in%20Bacillus%20subtilis&amp;author=M.%20Hecker&amp;author=W.%20Schumann&amp;author=U.%20V%C3%B6lker&amp;volume=19&amp;publication_year=1996&amp;pages=417-428&amp;pmid=8830234&amp;doi=10.1046/j.1365-2958.1996.396932.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B37-biology-13-01088">
<span class="label">37.</span><cite>Carrasco B., Torres R., Moreno-Del Álamo M., Ramos C., Ayora S., Alonso J.C. Processing of Stalled Replication Forks in Bacillus subtilis. FEMS Microbiol. Rev. 2024;48:fuad065.  doi: 10.1093/femsre/fuad065.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/femsre/fuad065" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10804225/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/38052445/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Rev.&amp;title=Processing%20of%20Stalled%20Replication%20Forks%20in%20Bacillus%20subtilis&amp;author=B.%20Carrasco&amp;author=R.%20Torres&amp;author=M.%20Moreno-Del%20%C3%81lamo&amp;author=C.%20Ramos&amp;author=S.%20Ayora&amp;volume=48&amp;publication_year=2024&amp;pages=fuad065&amp;pmid=38052445&amp;doi=10.1093/femsre/fuad065&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B38-biology-13-01088">
<span class="label">38.</span><cite>Itaya M., Omori A., Kanaya S., Crouch R.J., Tanaka T., Kondo K. Isolation of RNase H Genes That Are Essential for Growth of Bacillus subtilis 168. J. Bacteriol. 1999;181:2118–2123. doi: 10.1128/JB.181.7.2118-2123.1999.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JB.181.7.2118-2123.1999" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC93624/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10094689/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&amp;title=Isolation%20of%20RNase%20H%20Genes%20That%20Are%20Essential%20for%20Growth%20of%20Bacillus%20subtilis%20168&amp;author=M.%20Itaya&amp;author=A.%20Omori&amp;author=S.%20Kanaya&amp;author=R.J.%20Crouch&amp;author=T.%20Tanaka&amp;volume=181&amp;publication_year=1999&amp;pages=2118-2123&amp;pmid=10094689&amp;doi=10.1128/JB.181.7.2118-2123.1999&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B39-biology-13-01088">
<span class="label">39.</span><cite>Schroeder J.W., Hurto R.L., Randall J.R., Wozniak K.J., Timko T.A., Nye T.M., Wang J.D., Freddolino P.L., Simmons L.A. RNase H Genes Cause Distinct Impacts on RNA:DNA Hybrid Formation and Mutagenesis Genome Wide. Sci. Adv. 2023;9:eadi5945. doi: 10.1126/sciadv.adi5945.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/sciadv.adi5945" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10371020/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37494439/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=RNase%20H%20Genes%20Cause%20Distinct%20Impacts%20on%20RNA:DNA%20Hybrid%20Formation%20and%20Mutagenesis%20Genome%20Wide&amp;author=J.W.%20Schroeder&amp;author=R.L.%20Hurto&amp;author=J.R.%20Randall&amp;author=K.J.%20Wozniak&amp;author=T.A.%20Timko&amp;volume=9&amp;publication_year=2023&amp;pages=eadi5945&amp;pmid=37494439&amp;doi=10.1126/sciadv.adi5945&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B40-biology-13-01088">
<span class="label">40.</span><cite>Roberts M.F.  Methods in Microbiology. Volume 35. Elsevier; Amsterdam, The Netherlands: 2006. 26 Characterization of Organic Compatible Solutes of Halotolerant and Halophilic Microorganisms; pp. 615–647.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Microbiology&amp;author=M.F.%20Roberts&amp;publication_year=2006&amp;isbn=9780125215374&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B41-biology-13-01088">
<span class="label">41.</span><cite>Carlson S.M., Cunningham C.J., Westley P.A.H. Evolutionary Rescue in a Changing World. Trends Ecol. Evol. 2014;29:521–530. doi: 10.1016/j.tree.2014.06.005.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tree.2014.06.005" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25038023/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Ecol.%20Evol.&amp;title=Evolutionary%20Rescue%20in%20a%20Changing%20World&amp;author=S.M.%20Carlson&amp;author=C.J.%20Cunningham&amp;author=P.A.H.%20Westley&amp;volume=29&amp;publication_year=2014&amp;pages=521-530&amp;pmid=25038023&amp;doi=10.1016/j.tree.2014.06.005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B42-biology-13-01088">
<span class="label">42.</span><cite>Bell G. Evolutionary Rescue. Annu. Rev. Ecol. Evol. Syst. 2017;48:605–627. doi: 10.1146/annurev-ecolsys-110316-023011.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev-ecolsys-110316-023011" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Ecol.%20Evol.%20Syst.&amp;title=Evolutionary%20Rescue&amp;author=G.%20Bell&amp;volume=48&amp;publication_year=2017&amp;pages=605-627&amp;doi=10.1146/annurev-ecolsys-110316-023011&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B43-biology-13-01088">
<span class="label">43.</span><cite>Bruggeman F.J., Teusink B., Steuer R. Trade-Offs between the Instantaneous Growth Rate and Long-Term Fitness: Consequences for Microbial Physiology and Predictive Computational Models. Bioessays. 2023;45:e2300015.  doi: 10.1002/bies.202300015.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/bies.202300015" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37559168/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioessays&amp;title=Trade-Offs%20between%20the%20Instantaneous%20Growth%20Rate%20and%20Long-Term%20Fitness:%20Consequences%20for%20Microbial%20Physiology%20and%20Predictive%20Computational%20Models&amp;author=F.J.%20Bruggeman&amp;author=B.%20Teusink&amp;author=R.%20Steuer&amp;volume=45&amp;publication_year=2023&amp;pages=e2300015&amp;pmid=37559168&amp;doi=10.1002/bies.202300015&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B44-biology-13-01088">
<span class="label">44.</span><cite>Crespi B.J. The Evolution of Maladaptation. Heredity. 2000;84:623–629. doi: 10.1046/j.1365-2540.2000.00746.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1365-2540.2000.00746.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10886377/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=The%20Evolution%20of%20Maladaptation&amp;author=B.J.%20Crespi&amp;volume=84&amp;publication_year=2000&amp;pages=623-629&amp;pmid=10886377&amp;doi=10.1046/j.1365-2540.2000.00746.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B45-biology-13-01088">
<span class="label">45.</span><cite>Hounsa C.-G., Brandt E.V., Thevelein J., Hohmann S., Prior B.A. Role of Trehalose in Survival of Saccharomyces Cerevisiae under Osmotic Stress. Microbiology. 1998;144:671–680. doi: 10.1099/00221287-144-3-671.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1099/00221287-144-3-671" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9534237/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Role%20of%20Trehalose%20in%20Survival%20of%20Saccharomyces%20Cerevisiae%20under%20Osmotic%20Stress&amp;author=C.-G.%20Hounsa&amp;author=E.V.%20Brandt&amp;author=J.%20Thevelein&amp;author=S.%20Hohmann&amp;author=B.A.%20Prior&amp;volume=144&amp;publication_year=1998&amp;pages=671-680&amp;pmid=9534237&amp;doi=10.1099/00221287-144-3-671&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B46-biology-13-01088">
<span class="label">46.</span><cite>Zeigler D.R., Nicholson W.L. Experimental Evolution of Bacillus subtilis. Environ. Microbiol. 2017;19:3415–3422. doi: 10.1111/1462-2920.13831.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/1462-2920.13831" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28631363/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&amp;title=Experimental%20Evolution%20of%20Bacillus%20subtilis&amp;author=D.R.%20Zeigler&amp;author=W.L.%20Nicholson&amp;volume=19&amp;publication_year=2017&amp;pages=3415-3422&amp;pmid=28631363&amp;doi=10.1111/1462-2920.13831&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B47-biology-13-01088">
<span class="label">47.</span><cite>Gundlach J., Mehne F.M.P., Herzberg C., Kampf J., Valerius O., Kaever V., Stülke J. An Essential Poison: Synthesis and Degradation of Cyclic Di-AMP in Bacillus subtilis. J. Bacteriol. 2015;197:3265–3274. doi: 10.1128/JB.00564-15.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JB.00564-15" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4573722/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26240071/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&amp;title=An%20Essential%20Poison:%20Synthesis%20and%20Degradation%20of%20Cyclic%20Di-AMP%20in%20Bacillus%20subtilis&amp;author=J.%20Gundlach&amp;author=F.M.P.%20Mehne&amp;author=C.%20Herzberg&amp;author=J.%20Kampf&amp;author=O.%20Valerius&amp;volume=197&amp;publication_year=2015&amp;pages=3265-3274&amp;pmid=26240071&amp;doi=10.1128/JB.00564-15&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B48-biology-13-01088">
<span class="label">48.</span><cite>Gándara C., Alonso J.C. DisA and C-Di-AMP Act at the Intersection between DNA-Damage Response and Stress Homeostasis in Exponentially Growing Bacillus subtilis Cells. DNA Repair. 2015;27:1–8. doi: 10.1016/j.dnarep.2014.12.007.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.dnarep.2014.12.007" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25616256/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=DNA%20Repair&amp;title=DisA%20and%20C-Di-AMP%20Act%20at%20the%20Intersection%20between%20DNA-Damage%20Response%20and%20Stress%20Homeostasis%20in%20Exponentially%20Growing%20Bacillus%20subtilis%20Cells&amp;author=C.%20G%C3%A1ndara&amp;author=J.C.%20Alonso&amp;volume=27&amp;publication_year=2015&amp;pages=1-8&amp;pmid=25616256&amp;doi=10.1016/j.dnarep.2014.12.007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B49-biology-13-01088">
<span class="label">49.</span><cite>Gundlach J., Herzberg C., Kaever V., Gunka K., Hoffmann T., Weiß M., Gibhardt J., Thürmer A., Hertel D., Daniel R., et al.  Control of Potassium Homeostasis Is an Essential Function of the Second Messenger Cyclic Di-AMP in Bacillus subtilis. Sci. Signal. 2017;10:eaal3011. doi: 10.1126/scisignal.aal3011.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/scisignal.aal3011" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28420751/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Signal.&amp;title=Control%20of%20Potassium%20Homeostasis%20Is%20an%20Essential%20Function%20of%20the%20Second%20Messenger%20Cyclic%20Di-AMP%20in%20Bacillus%20subtilis&amp;author=J.%20Gundlach&amp;author=C.%20Herzberg&amp;author=V.%20Kaever&amp;author=K.%20Gunka&amp;author=T.%20Hoffmann&amp;volume=10&amp;publication_year=2017&amp;pages=eaal3011&amp;pmid=28420751&amp;doi=10.1126/scisignal.aal3011&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B50-biology-13-01088">
<span class="label">50.</span><cite>Gunasekera T.S., Csonka L.N., Paliy O. Genome-Wide Transcriptional Responses of Escherichia Coli K-12 to Continuous Osmotic and Heat Stresses. J. Bacteriol. 2008;190:3712–3720. doi: 10.1128/JB.01990-07.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JB.01990-07" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2395010/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18359805/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&amp;title=Genome-Wide%20Transcriptional%20Responses%20of%20Escherichia%20Coli%20K-12%20to%20Continuous%20Osmotic%20and%20Heat%20Stresses&amp;author=T.S.%20Gunasekera&amp;author=L.N.%20Csonka&amp;author=O.%20Paliy&amp;volume=190&amp;publication_year=2008&amp;pages=3712-3720&amp;pmid=18359805&amp;doi=10.1128/JB.01990-07&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B51-biology-13-01088">
<span class="label">51.</span><cite>Bremer E., Krämer R. Responses of Microorganisms to Osmotic Stress. Annu. Rev. Microbiol. 2019;73:313–334. doi: 10.1146/annurev-micro-020518-115504.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev-micro-020518-115504" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31180805/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Microbiol.&amp;title=Responses%20of%20Microorganisms%20to%20Osmotic%20Stress&amp;author=E.%20Bremer&amp;author=R.%20Kr%C3%A4mer&amp;volume=73&amp;publication_year=2019&amp;pages=313-334&amp;pmid=31180805&amp;doi=10.1146/annurev-micro-020518-115504&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B52-biology-13-01088">
<span class="label">52.</span><cite>Holtmann G., Bremer E. Thermoprotection of Bacillus subtilis by Exogenously Provided Glycine Betaine and Structurally Related Compatible Solutes: Involvement of Opu Transporters. J. Bacteriol. 2004;186:1683–1693. doi: 10.1128/JB.186.6.1683-1693.2004.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JB.186.6.1683-1693.2004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC355977/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14996799/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&amp;title=Thermoprotection%20of%20Bacillus%20subtilis%20by%20Exogenously%20Provided%20Glycine%20Betaine%20and%20Structurally%20Related%20Compatible%20Solutes:%20Involvement%20of%20Opu%20Transporters&amp;author=G.%20Holtmann&amp;author=E.%20Bremer&amp;volume=186&amp;publication_year=2004&amp;pages=1683-1693&amp;pmid=14996799&amp;doi=10.1128/JB.186.6.1683-1693.2004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B53-biology-13-01088">
<span class="label">53.</span><cite>Carroll S.M., Marx C.J. Evolution after Introduction of a Novel Metabolic Pathway Consistently Leads to Restoration of Wild-Type Physiology. PLoS Genet. 2013;9:e1003427.  doi: 10.1371/journal.pgen.1003427.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pgen.1003427" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3616920/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23593025/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Evolution%20after%20Introduction%20of%20a%20Novel%20Metabolic%20Pathway%20Consistently%20Leads%20to%20Restoration%20of%20Wild-Type%20Physiology&amp;author=S.M.%20Carroll&amp;author=C.J.%20Marx&amp;volume=9&amp;publication_year=2013&amp;pages=e1003427&amp;pmid=23593025&amp;doi=10.1371/journal.pgen.1003427&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B54-biology-13-01088">
<span class="label">54.</span><cite>Rodríguez-Verdugo A., Tenaillon O., Gaut B.S. First-Step Mutations during Adaptation Restore the Expression of Hundreds of Genes. Mol. Biol. Evol. 2016;33:25–39. doi: 10.1093/molbev/msv228.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/msv228" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4693981/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26500250/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=First-Step%20Mutations%20during%20Adaptation%20Restore%20the%20Expression%20of%20Hundreds%20of%20Genes&amp;author=A.%20Rodr%C3%ADguez-Verdugo&amp;author=O.%20Tenaillon&amp;author=B.S.%20Gaut&amp;volume=33&amp;publication_year=2016&amp;pages=25-39&amp;pmid=26500250&amp;doi=10.1093/molbev/msv228&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B55-biology-13-01088">
<span class="label">55.</span><cite>González-González A., Hug S.M., Rodríguez-Verdugo A., Patel J.S., Gaut B.S. Adaptive Mutations in RNA Polymerase and the Transcriptional Terminator Rho Have Similar Effects on Escherichia coli Gene Expression. Mol. Biol. Evol. 2017;34:2839–2855. doi: 10.1093/molbev/msx216.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/msx216" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5815632/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28961910/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=Adaptive%20Mutations%20in%20RNA%20Polymerase%20and%20the%20Transcriptional%20Terminator%20Rho%20Have%20Similar%20Effects%20on%20Escherichia%20coli%20Gene%20Expression&amp;author=A.%20Gonz%C3%A1lez-Gonz%C3%A1lez&amp;author=S.M.%20Hug&amp;author=A.%20Rodr%C3%ADguez-Verdugo&amp;author=J.S.%20Patel&amp;author=B.S.%20Gaut&amp;volume=34&amp;publication_year=2017&amp;pages=2839-2855&amp;pmid=28961910&amp;doi=10.1093/molbev/msx216&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B56-biology-13-01088">
<span class="label">56.</span><cite>Bailey S.F., Blanquart F., Bataillon T., Kassen R. What Drives Parallel Evolution?: How Population Size and Mutational Variation Contribute to Repeated Evolution. Bioessays. 2017;39:1–9. doi: 10.1002/bies.201600176.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/bies.201600176" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27859467/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioessays&amp;title=What%20Drives%20Parallel%20Evolution?:%20How%20Population%20Size%20and%20Mutational%20Variation%20Contribute%20to%20Repeated%20Evolution&amp;author=S.F.%20Bailey&amp;author=F.%20Blanquart&amp;author=T.%20Bataillon&amp;author=R.%20Kassen&amp;volume=39&amp;publication_year=2017&amp;pages=1-9&amp;pmid=27859467&amp;doi=10.1002/bies.201600176&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B57-biology-13-01088">
<span class="label">57.</span><cite>Guinebretière M.-H., Thompson F.L., Sorokin A., Normand P., Dawyndt P., Ehling-Schulz M., Svensson B., Sanchis V., Nguyen-The C., Heyndrickx M., et al.  Ecological Diversification in the Bacillus cereus Group. Environ. Microbiol. 2008;10:851–865. doi: 10.1111/j.1462-2920.2007.01495.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1462-2920.2007.01495.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18036180/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&amp;title=Ecological%20Diversification%20in%20the%20Bacillus%20cereus%20Group&amp;author=M.-H.%20Guinebreti%C3%A8re&amp;author=F.L.%20Thompson&amp;author=A.%20Sorokin&amp;author=P.%20Normand&amp;author=P.%20Dawyndt&amp;volume=10&amp;publication_year=2008&amp;pages=851-865&amp;pmid=18036180&amp;doi=10.1111/j.1462-2920.2007.01495.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B58-biology-13-01088">
<span class="label">58.</span><cite>Pirttijärvi T.S., Ahonen L.M., Maunuksela L.M., Salkinoja-Salonen M.S. Bacillus cereus in a Whey Process. Int. J. Food Microbiol. 1998;44:31–41. doi: 10.1016/S0168-1605(98)00117-2.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0168-1605(98)00117-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9849782/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Food%20Microbiol.&amp;title=Bacillus%20cereus%20in%20a%20Whey%20Process&amp;author=T.S.%20Pirttij%C3%A4rvi&amp;author=L.M.%20Ahonen&amp;author=L.M.%20Maunuksela&amp;author=M.S.%20Salkinoja-Salonen&amp;volume=44&amp;publication_year=1998&amp;pages=31-41&amp;pmid=9849782&amp;doi=10.1016/S0168-1605(98)00117-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B59-biology-13-01088">
<span class="label">59.</span><cite>Pomerleau M., Charron-Lamoureux V., Léonard L., Grenier F., Rodrigue S., Beauregard P.B. Adaptive Laboratory Evolution Reveals Regulators Involved in Repressing Biofilm Development as Key Players in Bacillus subtilis Root Colonization. mSystems. 2024;9:e0084323. doi: 10.1128/msystems.00843-23.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/msystems.00843-23" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10878085/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/38206029/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Adaptive%20Laboratory%20Evolution%20Reveals%20Regulators%20Involved%20in%20Repressing%20Biofilm%20Development%20as%20Key%20Players%20in%20Bacillus%20subtilis%20Root%20Colonization&amp;author=M.%20Pomerleau&amp;author=V.%20Charron-Lamoureux&amp;author=L.%20L%C3%A9onard&amp;author=F.%20Grenier&amp;author=S.%20Rodrigue&amp;volume=9&amp;publication_year=2024&amp;pages=e0084323&amp;pmid=38206029&amp;doi=10.1128/msystems.00843-23&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B60-biology-13-01088">
<span class="label">60.</span><cite>Blake C., Christensen M.N., Kovács Á.T. Molecular Aspects of Plant Growth Promotion and Protection by Bacillus subtilis. Mol. Plant-Microbe. Interact. 2021;34:15–25. doi: 10.1094/MPMI-08-20-0225-CR.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1094/MPMI-08-20-0225-CR" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32986513/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Plant-Microbe.%20Interact.&amp;title=Molecular%20Aspects%20of%20Plant%20Growth%20Promotion%20and%20Protection%20by%20Bacillus%20subtilis&amp;author=C.%20Blake&amp;author=M.N.%20Christensen&amp;author=%C3%81.T.%20Kov%C3%A1cs&amp;volume=34&amp;publication_year=2021&amp;pages=15-25&amp;pmid=32986513&amp;doi=10.1094/MPMI-08-20-0225-CR&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B61-biology-13-01088">
<span class="label">61.</span><cite>Slomka S., Françoise I., Hornung G., Asraf O., Biniashvili T., Pilpel Y., Dahan O. Experimental Evolution of Bacillus subtilis Reveals the Evolutionary Dynamics of Horizontal Gene Transfer and Suggests Adaptive and Neutral Effects. Genetics. 2020;216:543–558. doi: 10.1534/genetics.120.303401.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1534/genetics.120.303401" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7536860/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32847815/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Experimental%20Evolution%20of%20Bacillus%20subtilis%20Reveals%20the%20Evolutionary%20Dynamics%20of%20Horizontal%20Gene%20Transfer%20and%20Suggests%20Adaptive%20and%20Neutral%20Effects&amp;author=S.%20Slomka&amp;author=I.%20Fran%C3%A7oise&amp;author=G.%20Hornung&amp;author=O.%20Asraf&amp;author=T.%20Biniashvili&amp;volume=216&amp;publication_year=2020&amp;pages=543-558&amp;pmid=32847815&amp;doi=10.1534/genetics.120.303401&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B62-biology-13-01088">
<span class="label">62.</span><cite>Shoemaker W.R., Polezhaeva E., Givens K.B., Lennon J.T. Seed Banks Alter the Molecular Evolutionary Dynamics of Bacillus subtilis. Genetics. 2022;221:iyac071. doi: 10.1093/genetics/iyac071.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/genetics/iyac071" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9157070/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35511143/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Seed%20Banks%20Alter%20the%20Molecular%20Evolutionary%20Dynamics%20of%20Bacillus%20subtilis&amp;author=W.R.%20Shoemaker&amp;author=E.%20Polezhaeva&amp;author=K.B.%20Givens&amp;author=J.T.%20Lennon&amp;volume=221&amp;publication_year=2022&amp;pages=iyac071&amp;pmid=35511143&amp;doi=10.1093/genetics/iyac071&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B63-biology-13-01088">
<span class="label">63.</span><cite>Jolley E.A., Yakhnin H., Tack D., Babitzke P., Bevilacqua P. Transcriptome-Wide Probing Reveals RNA Thermometers That Regulate Translation of Glycerol Permease Genes in Bacillus subtilis. RNA. 2023;29:1365–1378. doi: 10.1261/rna.079652.123.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1261/rna.079652.123" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10573289/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37217261/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=RNA&amp;title=Transcriptome-Wide%20Probing%20Reveals%20RNA%20Thermometers%20That%20Regulate%20Translation%20of%20Glycerol%20Permease%20Genes%20in%20Bacillus%20subtilis&amp;author=E.A.%20Jolley&amp;author=H.%20Yakhnin&amp;author=D.%20Tack&amp;author=P.%20Babitzke&amp;author=P.%20Bevilacqua&amp;volume=29&amp;publication_year=2023&amp;pages=1365-1378&amp;pmid=37217261&amp;doi=10.1261/rna.079652.123&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B64-biology-13-01088">
<span class="label">64.</span><cite>Hawkins J.S., Silvis M.R., Koo B.-M., Peters J.M., Osadnik H., Jost M., Hearne C.C., Weissman J.S., Todor H., Gross C.A. Mismatch-CRISPRi Reveals the Co-Varying Expression-Fitness Relationships of Essential Genes in Escherichia coli and Bacillus Subtilis. Cell Syst. 2020;11:523–535.e9. doi: 10.1016/j.cels.2020.09.009.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cels.2020.09.009" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7704046/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33080209/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst.&amp;title=Mismatch-CRISPRi%20Reveals%20the%20Co-Varying%20Expression-Fitness%20Relationships%20of%20Essential%20Genes%20in%20Escherichia%20coli%20and%20Bacillus%20Subtilis&amp;author=J.S.%20Hawkins&amp;author=M.R.%20Silvis&amp;author=B.-M.%20Koo&amp;author=J.M.%20Peters&amp;author=H.%20Osadnik&amp;volume=11&amp;publication_year=2020&amp;pages=523-535.e9&amp;pmid=33080209&amp;doi=10.1016/j.cels.2020.09.009&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B65-biology-13-01088">
<span class="label">65.</span><cite>Shi L., Derouiche A., Pandit S., Rahimi S., Kalantari A., Futo M., Ravikumar V., Jers C., Mokkapati V.R.S.S., Vlahoviček K., et al.  Evolutionary Analysis of the Bacillus subtilis Genome Reveals New Genes Involved in Sporulation. Mol. Biol. Evol. 2020;37:1667–1678. doi: 10.1093/molbev/msaa035.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/msaa035" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7426031/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32061128/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=Evolutionary%20Analysis%20of%20the%20Bacillus%20subtilis%20Genome%20Reveals%20New%20Genes%20Involved%20in%20Sporulation&amp;author=L.%20Shi&amp;author=A.%20Derouiche&amp;author=S.%20Pandit&amp;author=S.%20Rahimi&amp;author=A.%20Kalantari&amp;volume=37&amp;publication_year=2020&amp;pages=1667-1678&amp;pmid=32061128&amp;doi=10.1093/molbev/msaa035&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11673879/bin/biology-13-01088-s001.zip">biology-13-01088-s001.zip</a><sup> (1.2MB, zip) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The strains are available upon request to interested researchers.</p></section></section></section></section></article>
</main></body></html>