<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2998437/"/><title>S. aureus MscL Is a Pentamer In Vivo but of Variable Stoichiometries In Vitro: Implications for Detergent-Solubilized Membrane Proteins - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">PLoS Biol</button></div>. 2010 Dec 7;8(12):e1000555. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pbio.1000555" rel="noopener noreferrer" target="_blank">10.1371/journal.pbio.1000555</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>
<em>S. aureus</em> MscL Is a Pentamer In Vivo but of Variable Stoichiometries In Vitro: Implications for Detergent-Solubilized Membrane Proteins</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dorwart%20MR%22%5BAuthor%5D"><span class="name western">Michael R Dorwart</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Michael R Dorwart</span></h3>
<div class="p">
<sup>1</sup>Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">
<sup>2</sup>Howard Hughes Medical Institute, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dorwart%20MR%22%5BAuthor%5D"><span class="name western">Michael R Dorwart</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wray%20R%22%5BAuthor%5D"><span class="name western">Robin Wray</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Robin Wray</span></h3>
<div class="p">
<sup>1</sup>Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wray%20R%22%5BAuthor%5D"><span class="name western">Robin Wray</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brautigam%20CA%22%5BAuthor%5D"><span class="name western">Chad A Brautigam</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Chad A Brautigam</span></h3>
<div class="p">
<sup>3</sup>Department of Biochemistry, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brautigam%20CA%22%5BAuthor%5D"><span class="name western">Chad A Brautigam</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D"><span class="name western">Youxing Jiang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Youxing Jiang</span></h3>
<div class="p">
<sup>1</sup>Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">
<sup>2</sup>Howard Hughes Medical Institute, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D"><span class="name western">Youxing Jiang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blount%20P%22%5BAuthor%5D"><span class="name western">Paul Blount</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Paul Blount</span></h3>
<div class="p">
<sup>1</sup>Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blount%20P%22%5BAuthor%5D"><span class="name western">Paul Blount</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Gregory A Petsko</span><sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>Department of Physiology, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div id="aff2">
<sup>2</sup>Howard Hughes Medical Institute, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div id="aff3">
<sup>3</sup>Department of Biochemistry, University of Texas Southwestern Medical Center at Dallas, Dallas, Texas, United States of America</div>
<div id="edit1">
<sup>4</sup>Brandeis University, United States of America</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>✉</sup><p class="display-inline">* E-mail: <span>Paul.Blount@UTSouthwestern.edu</span></p>
</div>
<div class="fn" id="fn1"><p>The author(s) have made the following declarations about their contributions: Conceived and designed the experiments: MRD RW CAB YJ PB. Performed the experiments: MRD RW CAB. Analyzed the data: MRD RW CAB PB. Wrote the paper: MRD PB.</p></div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Gregory A Petsko</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2010 Jul 9; Accepted 2010 Oct 27; Collection date 2010 Dec.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Dorwart et al.</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC2998437  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21151884/">21151884</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><hr class="headless"/>
<p>Detergent-induced rearrangements of membrane-protein subunits explain why two MscL channel stoichiometries have been resolved by X-ray crystallography - but <em>S. aureus</em> MscL is truly a pentamer in vivo.</p></section><section class="abstract" id="abstract2"><h2>Abstract</h2>
<p>While the bacterial mechanosensitive channel of large conductance (MscL) is the best studied biological mechanosensor and serves as a paradigm for how a protein can sense and respond to membrane tension, the simple matter of its oligomeric state has led to debate, with models ranging from tetramers to hexamers. Indeed, two different oligomeric states of the bacterial mechanosensitive channel MscL have been resolved by X-ray crystallography: The <em>M. tuberculosis</em> channel (MtMscL) is a pentamer, while the <em>S. aureus</em> protein (SaMscL) forms a tetramer. Because several studies suggest that, like MtMscL, the <em>E. coli</em> MscL (EcoMscL) is a pentamer, we re-investigated the oligomeric state of SaMscL. To determine the structural organization of MscL in the cell membrane we developed a disulfide-trapping approach. Surprisingly, we found that virtually all SaMscL channels in vivo are pentameric, indicating this as the physiologically relevant and functional oligomeric state. Complementing our in vivo results, we purified SaMscL and assessed its oligomeric state using three independent approaches (sedimentation equilibrium centrifugation, crosslinking, and light scattering) and established that SaMscL is a pentamer when solubilized in Triton X-100 and C<sub>8</sub>E<sub>5</sub> detergents. However, performing similar experiments on SaMscL solubilized in LDAO, the detergent used in the crystallographic study, confirmed the tetrameric oligomerization resolved by X-ray crystallography. We further demonstrate that this stoichiometric shift is reversible by conventional detergent exchange experiments. Our results firmly establish the pentameric organization of SaMscL in vivo. Furthermore they demonstrate that detergents can alter the subunit stoichiometry of membrane protein complexes in vitro; thus, in vivo assays are necessary to firmly establish a membrane protein's true functionally relevant oligomeric state.</p></section><section class="abstract" id="abstract3"><h2>Author Summary</h2>
<p>The ability to detect mechanical forces is at the basis of not only the senses of touch hearing and balance but also cardiovascular and osmotic regulation. One of the primary ways that organisms detect forces is through mechanosensitive channels, and mechanosensation is so vital that essentially all organisms have at least one such sensor. Indeed, the best-studied mechanosensitive channel is from bacteria, and because relatively little is known of mechanosensors from higher organisms, these channels are a model for how a protein can sense and respond to mechanical forces. Although the bacterial mechanosensitive channel MscL has been well studied, the simple issue of how many subunits it has is hotly debated. There are even two published crystal structures showing either tetrameric or pentameric complexes. Here we show that the channel is actually pentameric in vivo and that the detergent used to solubilize the protein can rearrange the complexes from pentamers to tetramers. The finding that detergents can have such a profound effect on structure may have broad implications for the study of other membrane proteins.</p></section><section id="s1"><h2 class="pmc_sec_title">Introduction</h2>
<p>The bacterial mechanosensitive channel MscL serves as a biological “emergency release valve,” allowing rapid loss of solutes in response to a sudden decrease in the osmolarity of a bacterium's environment <a aria-describedby="pbio.1000555-Levina1" class="usa-link" href="#pbio.1000555-Levina1">[1]</a>. It is perhaps the best characterized mechanosensor <a aria-describedby="pbio.1000555-Blount1" class="usa-link" href="#pbio.1000555-Blount1">[2]</a>, thus serving as a paradigm of how a membrane protein can detect and respond to mechanical forces <a aria-describedby="pbio.1000555-Blount2" class="usa-link" href="#pbio.1000555-Blount2">[3]</a>. Ironically, something as simple as the stoichiometry of the MscL complex has plagued the field with debate since its inception.</p>
<p>The original model for the <em>E. coli</em> MscL (EcoMscL) stoichiometry was a homo-hexameric organization, which was suggested by crosslinking and the study of tandem subunits <a aria-describedby="pbio.1000555-Blount3" class="usa-link" href="#pbio.1000555-Blount3">[4]</a>. This model then appeared to be supported by low-resolution two-dimensional crystallization of EcoMscL <a aria-describedby="pbio.1000555-Saint1" class="usa-link" href="#pbio.1000555-Saint1">[5]</a>. But the subsequent elucidation of the <em>M. tuberculosis</em> channel (MtMscL) by X-ray crystallography <a aria-describedby="pbio.1000555-Chang1" class="usa-link" href="#pbio.1000555-Chang1">[6]</a> then suggested a pentameric organization, at least for this orthologue. This result led to a re-evaluation of EcoMscL stoichiometry <a aria-describedby="pbio.1000555-Chang1" class="usa-link" href="#pbio.1000555-Chang1">[6]</a>,<a aria-describedby="pbio.1000555-Sukharev1" class="usa-link" href="#pbio.1000555-Sukharev1">[7]</a>, which supported a pentameric organization and brought into question whether the two-dimensional crystallization data could be fit by 5-fold as well as 6-fold symmetry. Thus, the field transiently seemed to have settled that MscL was most likely a pentamer. However, the recent crystallographic structure of the <em>S. aureus</em> homolog (SaMscL) reveals a tetramer variant <a aria-describedby="pbio.1000555-Liu1" class="usa-link" href="#pbio.1000555-Liu1">[8]</a>. This latter finding has again raised questions regarding the true oligomeric state of MscL and evokes the possibilities either that MscL from different species assemble into complexes with different stoichiometries or that the channel exists as multiple functional oligomeric complexes in the cell membrane. Therefore, we set out to identify the MscL oligomeric state in the cell membrane and to understand how the SaMscL channel, which shares approximately 40% sequence identity with EcoMscL and MtMscL <a aria-describedby="pbio.1000555-Liu1" class="usa-link" href="#pbio.1000555-Liu1">[8]</a>, could exist in the non-pentameric subunit organization resolved by X-ray crystallography. We found not only that the true in vivo oligomeric state of SaMscL is a pentamer but also that at least one detergent, LDAO, artificially but reversibly reorganizes this structure into a tetrameric stoichiometry.</p></section><section id="s2"><h2 class="pmc_sec_title">Results</h2>
<p>With multiple oligomeric states identified in vitro, we devised a disulfide-trapping strategy to identify the oligomeric state of MscL in bacterial membranes. Using this approach, we were able to directly measure the subunit stoichiometry of SaMscL in vivo by generating a series of double-cysteine mutants in regions predicted to be in close proximity from both existing crystal structures (<a class="usa-link" href="#pbio-1000555-g001">Figure 1A</a>). These modifications allow crosslinking in the cell membrane via disulfide bonds as previously described <a aria-describedby="pbio.1000555-Iscla1" class="usa-link" href="#pbio.1000555-Iscla1">[9]</a>,<a aria-describedby="pbio.1000555-Li1" class="usa-link" href="#pbio.1000555-Li1">[10]</a>. Briefly, the cells were osmotically shocked in the presence of the oxidizing agent copper-phenanthroline, centrifuged, and then immediately resuspended in SDS loading buffer to avoid unnecessary manipulation of the proteins. As we previously observed <a aria-describedby="pbio.1000555-Iscla1" class="usa-link" href="#pbio.1000555-Iscla1">[9]</a>, osmotic shock increased disulfide bridging, presumably because reductants including glutathione and thioredoxin are released from the cell <a aria-describedby="pbio.1000555-Ajouz1" class="usa-link" href="#pbio.1000555-Ajouz1">[11]</a>, thus decreasing the reducing potential of the cytosol. The protein content of the lysed cells was evaluated using SDS-PAGE and molecular weight standards. For the L10C/L89C or L10C/M91C SaMscL channels, we were able to observe efficient in vivo disulfide trapping (<a class="usa-link" href="#pbio-1000555-g001">Figure 1B</a>). For the L10C/L89C protein, a range of oligomeric states was observed, a pentamer being the maximal state. However, for the L10C/M91C protein, one primary band was observed that reflects a pentameric complex. These data demonstrate that the pentamer is by far the major species within the cell membrane and thus appears to be the true structural and functional unit in vivo.</p>
<figure class="fig xbox font-sm" id="pbio-1000555-g001"><h3 class="obj_head">Figure 1. 
<em>S. aureus</em> MscL is a pentamer in the cell membrane.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2998437_pbio.1000555.g001.jpg" target="_blank"><img alt="Figure 1" class="graphic zoom-in" height="432" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/fe2ef5a72403/pbio.1000555.g001.jpg" width="656"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/pbio-1000555-g001/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(A) Two orientations of the SaMscL and MtMscL crystal structures are shown with the molecules viewed from the periplasmic side (top) and from the lipid bilayer (bottom). The locations of the residues used to perform the in vivo disulfide crosslinking experiments are highlighted on the structures in surface representation: blue represents L10 (SaMscL) and A10 (MtMscL), orange represents T89 (SaMscL) and T96 (MtMscL), and red represents M91 (SaMscL) and R98 (MtMscL). (B) Western blot analysis of L10C/L89C and L10C/M91C SaMscL channels after in vivo disulfide bond trapping using copper-phenanthroline in both osmotically shocked (S+) and non-shocked (NS) samples.</p></figcaption></figure><p>With SaMscL pentamers being the predominant oligomeric state in the cell membrane, we wanted to understand why SaMscL crystallized as a tetramer. We began by confirming the previously observed pentameric and tetrameric states of MscL by performing crosslinking experiments of purified EcoMscL and SaMscL, visualizing their oligomeric state by Western blot analysis (<a class="usa-link" href="#pbio-1000555-g002">Figure 2</a>). Five crosslinked EcoMscL products were observed, consistent with previous reports showing that EcoMscL is a pentamer <a aria-describedby="pbio.1000555-Chang1" class="usa-link" href="#pbio.1000555-Chang1">[6]</a>,<a aria-describedby="pbio.1000555-Sukharev1" class="usa-link" href="#pbio.1000555-Sukharev1">[7]</a>. In contrast with previous findings <a aria-describedby="pbio.1000555-Liu1" class="usa-link" href="#pbio.1000555-Liu1">[8]</a>, SaMscL also presented five crosslinked products, suggesting that SaMscL also exists in a pentameric oligomeric state. No difference was observed between proteins expressed by a high-expressing plasmid (pET) and those encoded a moderate-expressing plasmid (pB10), demonstrating that expressions levels have no effect on MscL oligomerization. To substantiate these results, sedimentation equilibrium analytical ultracentrifugation (SE) was performed to directly measure the mass of the SaMscL channel. The results for SaMscL were fit to a single-species model and yielded a molar mass of 71.2 kDa, consistent with a pentameric subunit organization (SaMscL theoretical pentameric molar mass is 72.2 kDa) (<a class="usa-link" href="#pbio-1000555-g003">Figure 3A, B</a>). As a third independent method to directly measure the mass of the SaMscL channel, we performed size exclusion chromatography multi-angle light scattering (SEC-MALS) experiments <a aria-describedby="pbio.1000555-Slotboom1" class="usa-link" href="#pbio.1000555-Slotboom1">[12]</a>. Analysis of nine independent SEC-MALS experiments resulted in an average calculated SaMscL mass of 72.8±0.8 kDa (Ave ± SD, <em>n</em> = 9), again consistent with a pentameric subunit organization of SaMscL (representative experiment shown in <a class="usa-link" href="#pbio-1000555-g003">Figure 3C</a>).</p>
<figure class="fig xbox font-sm" id="pbio-1000555-g002"><h3 class="obj_head">Figure 2. 
<em>E. coli</em> and <em>S. aureus</em> MscL are pentameric multimers by crosslinking.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2998437_pbio.1000555.g002.jpg" target="_blank"><img alt="Figure 2" class="graphic zoom-in" height="361" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/7c1037cd54d4/pbio.1000555.g002.jpg" width="408"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/pbio-1000555-g002/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Western blot analysis of DSS-treated <em>E. coli</em>-MscL (Eco) and <em>S. aureus</em>-MscL (Sa) expressed with either the pET21a (pET) or pB10 vector show five distinct bands at both expression levels. Molecular weight markers are shown to the left of the blot and the approximate location of monomers (1×) to pentamers (5×) is shown between the blots. The right-hand blot shows a separate experiment where the majority of the EcoMscL and the SaMscL protein are in the pentameric and monomeric forms.</p></figcaption></figure><figure class="fig xbox font-sm" id="pbio-1000555-g003"><h3 class="obj_head">Figure 3. Purified <em>S. aureus</em> MscL is pentameric by equilibrium sedimentation centrifugation and SEC-MALS.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2998437_pbio.1000555.g003.jpg" target="_blank"><img alt="Figure 3" class="graphic zoom-in" height="657" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/8de34c1e57ce/pbio.1000555.g003.jpg" width="476"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/pbio-1000555-g003/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Sedimentation equilibrium centrifugation was performed on 40 µM SaMscL in the neutrally buoyant detergent C<sub>8</sub>E<sub>5</sub>. The upper two panels in (A) and (B) show scans from three different rotor speeds monitored at either 250 nm (A) or 280 nm (B). The lines represent fits to the data of a single species model that yield a calculated protein mass of 71.2 kDa with a local rmsd of 0.005799 (A) and 0.005059 (B) (SaMscL monomer mass is 14.4 kDa; pentamer mass is 72.2 kDa). The lower two panels in (A) and (B) show the residuals from the data fitting. A representative SEC-MALS experiment of SaMscL in C<sub>8</sub>E<sub>5</sub> is shown in (C) with the UV chromatogram colored blue and the 90° light scattering chromatogram colored red. The total computed mass is shown as the black line across the elution peak, with the SaMscL and C<sub>8</sub>E<sub>5</sub> mass shown in green and orange, respectively.</p></figcaption></figure><p>The data presented in <a class="usa-link" href="#pbio-1000555-g002">Figures 2</a> and <a class="usa-link" href="#pbio-1000555-g003">3</a> show three independent approaches that all confirm the pentameric oligomeric state of SaMscL, in agreement with our in vivo crosslinking results. These data are also consistent with the MtMscL crystal structure <a aria-describedby="pbio.1000555-Chang1" class="usa-link" href="#pbio.1000555-Chang1">[6]</a>,<a aria-describedby="pbio.1000555-Steinbacher1" class="usa-link" href="#pbio.1000555-Steinbacher1">[13]</a> and biochemical studies of EcoMscL <a aria-describedby="pbio.1000555-Sukharev1" class="usa-link" href="#pbio.1000555-Sukharev1">[7]</a> but inconsistent with the subunit organization in the SaMscL crystal structure <a aria-describedby="pbio.1000555-Liu1" class="usa-link" href="#pbio.1000555-Liu1">[8]</a>. The major difference in the protein handling between our experiments and those previously published for the SaMscL channel <a aria-describedby="pbio.1000555-Liu1" class="usa-link" href="#pbio.1000555-Liu1">[8]</a> was the choice of solubilizing detergent. Our measurements were made in the presence of either Triton X-100 (crosslinking) or pentaethylene glycol monooctyl ether (C<sub>8</sub>E<sub>5</sub>) detergent (SE and SEC-MALS), not Lauryldimethylamine-oxide (LDAO), the detergent used previously to crystallize SaMscL and evaluate its oligomeric state <a aria-describedby="pbio.1000555-Liu1" class="usa-link" href="#pbio.1000555-Liu1">[8]</a>. To determine if the solubilizing detergent could account for the discrepancies in the data, we tested whether LDAO may affect the oligomeric state of SaMscL.</p>
<p>We performed SaMscL crosslinking experiments in the presence of LDAO and found essentially no crosslinked pentamer and a substantial increase in tetramers when compared to the experiments performed in Triton X-100 (<a class="usa-link" href="#pbio-1000555-g004">Figure 4A</a>). To confirm that LDAO was altering the oligomeric state of the SaMscL channel rather than simply affecting our crosslinking efficiency, we directly measured the mass of the SaMscL protein in the presence of LDAO by SEC-MALS (representative experiment shown in <a class="usa-link" href="#pbio-1000555-g004">Figure 4B</a>). The results show that the average protein mass in the presence of LDAO is 60.0±2.8 kDa (Ave ± SD, <em>n</em> = 8), a mass that is dramatically reduced compared to the C<sub>8</sub>E<sub>5</sub> experiments (72.8±0.8 kDa) and that is consistent with a tetrameric oligomeric state (theoretical SaMscL tetramer is 57.8 kDa). As a third independent approach to assess the oligomeric state of the protein, we performed sedimentation velocity experiments on the SaMscL protein in LDAO. Two different methods were used to calculate the molar mass of the protein from these data (see <a class="usa-link" href="#s4">Methods</a> for details). Both resulted in similar values: 62.3±1.3 kDa (“<em>s</em> and <em>D</em>” method, <a class="usa-link" href="#pbio-1000555-g004">Figure 4C</a>) and 64±3 kDa (“Stokes radius” method) (Weighted Ave ± 1σ). These results again show a protein molar mass well below that of a pentamer, consistent with LDAO affecting the oligomeric state of the SaMscL channel.</p>
<figure class="fig xbox font-sm" id="pbio-1000555-g004"><h3 class="obj_head">Figure 4. LDAO stabilizes a MscL tetrameric oligomeric state as measured by crosslinking, SEC-MALS, and Sedimentation Velocity Centrifugation.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2998437_pbio.1000555.g004.jpg" target="_blank"><img alt="Figure 4" class="graphic zoom-in" height="740" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/6f7302a3692b/pbio.1000555.g004.jpg" width="656"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/pbio-1000555-g004/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(A) Western blot analysis of DSS-treated SaMscL solubilized with either Triton-X100 or LDAO yield different quantities of tetramers and pentamers. (B) A representative SEC-MALS experiment of SaMscL in LDAO is shown with the UV chromatogram colored blue and the 90° light scattering chromatogram colored red. The total, SaMscL, and LDAO masses are shown as black, green, and orange lines across the elution peak, respectively. The mass of the SaMscL monomer is 14.4 kDa and the tetramer mass is 57.8 kDa. (C) SaMscL in 4 mM LDAO was subjected to sedimentation velocity centrifugation at 50,000 rpm, and the data were analyzed using the noninteracting “species model” of SEDPHAT. Four species, including the detergent micelles, were analyzed. The molar mass of the dominant sedimenting species was 62.5 kDa. In the <em>upper part</em>, the individual data points are depicted as circles, and the best-fit model to those data is shown as lines. The data and fit lines are color coded by color: Violet for the earliest scans, then progressing through indigo, blue, green, yellow, orange, and red as the scans go further forward in time. For clarity only every other scan used in the data analysis is shown. In the <em>lower part</em>, the residuals between the data points and the fitted line are shown and color coded as above.</p></figcaption></figure><p>We also tested the reversibility of this oligomeric state reorganization by using SaMscL purified in LDAO and then exchanged the LDAO detergent for C<sub>8</sub>E<sub>5</sub>. This protocol resulted in the SaMscL mass increasing from 61.4±0.7 kDa (Ave ± SD, <em>n</em> = 3) to 72.0±2.6 kDa (Ave ± SD, <em>n</em> = 4) as measured by SEC-MALS (<a class="usa-link" href="#pbio-1000555-g005">Figure 5</a>). The choice of initial detergent made no difference in the reversibility, as purifying SaMscL in C<sub>8</sub>E<sub>5</sub> and then exchanging the detergent for LDAO showed a mass decrease from 72.2±0.5 kDa in C<sub>8</sub>E<sub>5</sub> (Ave ± SD, <em>n</em> = 5) to 58.7±0.7 kDa in LDAO (Ave ± SD, <em>n</em> = 3). Hence, the oligomeric rearrangement is reversible and is completely dependent upon the detergent solubilizing the MscL protein.</p>
<figure class="fig xbox font-sm" id="pbio-1000555-g005"><h3 class="obj_head">Figure 5. The <em>S. aureus</em> MscL stoichiometry change is reversible.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=2998437_pbio.1000555.g005.jpg" target="_blank"><img alt="Figure 5" class="graphic zoom-in" height="672" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/e493ae495bef/pbio.1000555.g005.jpg" width="467"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/pbio-1000555-g005/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(A) The oligomeric state of SaMscL in 4 mM LDAO was determined using SEC-MALS, which measured a protein mass of 61.3 kDa, consistent with a tetrameric channel (top chromatogram). (B) The SaMscL-LDAO sample was exchanged into C<sub>8</sub>E<sub>5</sub>, and the following day the oligomeric state of SaMscL-C<sub>8</sub>E<sub>5</sub> sample was measured by SEC-MALS. Analysis of the SEC-MALS results for the SaMscL-C<sub>8</sub>E<sub>5</sub> sample showed a protein mass of 74.4 kDa, consistent with the channel in a pentameric state.</p></figcaption></figure></section><section id="s3"><h2 class="pmc_sec_title">Discussion</h2>
<p>Essentially all previous structural studies on MscL, as with most membrane proteins, have been performed on channels that have been solubilized, under the assumption that detergents do not alter the subunit stoichiometry. Our data examining the effect of different detergents on the oligomeric state of SaMscL have far-reaching implications for membrane protein research, as we now clearly demonstrate that changes in oligomeric state can and do occur upon detergent solubilization, and these alterations are not easily detected by SEC (see <a class="usa-link" href="#pbio-1000555-g003">Figures 3C</a> and <a class="usa-link" href="#pbio-1000555-g004">4B</a>). To circumvent these artifacts associated with solubilization, we have developed an in vivo cross-linking assay. Although the pentamer is by far the major product observed in <a class="usa-link" href="#pbio-1000555-g001">Figure 1B</a>, very small amounts of smaller oligomers can be resolved upon overloading of protein gels and long Western blot exposures. The small amount of tetramers and smaller complexes observed upon long exposure could be incompletely assembled protein or may simply be due to the lack of disulfide bridging, which is rarely observed to be this efficient (e.g. see <a aria-describedby="pbio.1000555-Iscla1" class="usa-link" href="#pbio.1000555-Iscla1">[9]</a>,<a aria-describedby="pbio.1000555-Li1" class="usa-link" href="#pbio.1000555-Li1">[10]</a>). In addition, we cannot completely rule out the possibility that oligomerization is a dynamic and even reversible process in vivo. Regardless, it seems unlikely that oligomeric species smaller than pentamers have a physiological role given that there are tens of channels per cell <a aria-describedby="pbio.1000555-Blount4" class="usa-link" href="#pbio.1000555-Blount4">[14]</a>, and the smaller species compose only a few percent of the total channels. Our data definitively show that the vast majority of the SaMscL channel adopts a pentameric structure in the cell membrane, consistent with the MtMscL crystal structure <a aria-describedby="pbio.1000555-Chang1" class="usa-link" href="#pbio.1000555-Chang1">[6]</a>, but incompatible with the SaMscL crystal structure <a aria-describedby="pbio.1000555-Liu1" class="usa-link" href="#pbio.1000555-Liu1">[8]</a>.</p>
<p>A close inspection of the recent tetrameric SaMscL structure, which was speculated to be in an expanded gating intermediate state, reveals structural interactions between residues in TM1 and TM2 residues of a neighboring subunit similar to those in the closed MtMscL structure. This result was surprising because several studies using various techniques have suggested TM1 undergoes a significant clockwise rotation (as viewed from the periplasmic side) upon channel gating. The approaches supporting this interpretation include: spin labeling combined with electron paramagnetic resonance (EPR) studies <a aria-describedby="pbio.1000555-Perozo1" class="usa-link" href="#pbio.1000555-Perozo1">[15]</a>, accessibility of sulfhydryl reagents to cysteine mutants in TM1 upon gating <a aria-describedby="pbio.1000555-Bartlett1" class="usa-link" href="#pbio.1000555-Bartlett1">[16]</a>,<a aria-describedby="pbio.1000555-Bartlett2" class="usa-link" href="#pbio.1000555-Bartlett2">[17]</a>, accessibility of heavy metals to engineered binding sites in the pore <a aria-describedby="pbio.1000555-Iscla2" class="usa-link" href="#pbio.1000555-Iscla2">[18]</a>, suppression mutagenesis demonstrating interactions between TM1 and TM2 upon gating <a aria-describedby="pbio.1000555-Li2" class="usa-link" href="#pbio.1000555-Li2">[19]</a>, and confirmation of multiple interaction sites between TM1 and TM2 upon gating by using an electrostatic repulsion approach as well as disulfide trapping <a aria-describedby="pbio.1000555-Li1" class="usa-link" href="#pbio.1000555-Li1">[10]</a>. Both the EPR studies as well as the heavy-metal binding in the pore strongly suggest that this rotation occurs quite early in the gating process, before ion permeation. By this model, V21 and G24 (V23 and G26 in EcoMscL) should rotate away from the pore constriction, allowing V22 and the positively charged K29 (I24 and K31 in EcoMscL) to line the open pore. The observation that this rearrangement has not occurred strongly suggests that the crystal SaMscL structure actually reflects a strained closed state due to improper oligomeriazation, rather than an expanded intermediate state.</p>
<p>The observed stoichiometry changes in detergents also evoke questions regarding how membrane protein subunits assemble into physiological multimeric complexes. A previous report on EcoMscL demonstrated that when the protein is translated in vitro in the absence of any lipid or detergent and is subsequently purified in the presence of Triton X-100, the channel spontaneously assembles into pentamers <a aria-describedby="pbio.1000555-Berrier1" class="usa-link" href="#pbio.1000555-Berrier1">[20]</a>. A separate study demonstrated that even synthetically synthesized EcoMscL will oligomerize into functional channels when incorporated into lipid membranes <a aria-describedby="pbio.1000555-Clayton1" class="usa-link" href="#pbio.1000555-Clayton1">[21]</a>. These studies demonstrate that the EcoMscL channel has the ability to self-assemble into the functional oligomer in vitro. Similarly, it appears that C<sub>8</sub>E<sub>5</sub> and Triton X-100 allow SaMscL to properly fold and assemble into a pentameric structure, which correlates with the observed oligomeric state in vivo. In contrast, LDAO reorganizes SaMscL into tetramers that do not appear to exist in significant quantities in vivo, suggesting that they are not physiologically relevant. The observed self-assembly properties of EcoMscL under many conditions, combined with the reversible nature of the detergent-induced oligomeric state changes for SaMscL, may explain why Liu et al. observed channel function for the LDAO solubilized SaMscL tetramers <a aria-describedby="pbio.1000555-Liu1" class="usa-link" href="#pbio.1000555-Liu1">[8]</a>; presumably, once the tetrameric LDAO-solubilized SaMscL is reconstituted into lipids, it rearranges back into functional pentameric structures.</p>
<p>The crystal structure of the SaMscL was derived from a truncation mutant in which several amino acids at the C-terminus had been removed <a aria-describedby="pbio.1000555-Liu1" class="usa-link" href="#pbio.1000555-Liu1">[8]</a>. One previous study suggested that at least some C-terminus deletions of EcoMscL could lead to the formation of larger aggregates when translated in a cell-free system <a aria-describedby="pbio.1000555-Yoshimura1" class="usa-link" href="#pbio.1000555-Yoshimura1">[22]</a>. While we cannot rule out the possibility that the SaMscL deletion contributed to the stabilization of the tetramer in the crystal, our data do demonstrate that detergents alone can stabilize alternative oligomeric states of the full-length SaMscL in solution.</p>
<p>Our observations make it tempting to speculate that larger, more hydrated head groups (i.e. the head groups of C<sub>8</sub>E<sub>5</sub> and Triton X-100) promote pentamer formation, while smaller head groups (i.e. LDAO) promote tetramer formation. However, with such a small number of detergents tested thus far, additional experiments will be required to identify what chemical properties of the detergents cause the oligomeric state change and what effect, if any, lipids may have on the stabilization of specific oligomeric states.</p>
<p>In sum, our data demonstrate that the physiologically active SaMscL oligomer in vivo is a pentamer and that purification in detergents can cause a reorganization of the SaMscL channel's oligomeric state. Our findings also show that SEC alone is insufficient to evaluate the usefulness of a particular detergent for the study of membrane proteins; ideally, SEC-MALS and/or analytical ultracentrifugation should be correlated with in vivo experiments that can more confidently determine the functional and physiologically relevant oligomeric state of membrane proteins.</p></section><section id="s4"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="s4a"><h3 class="pmc_sec_title">Purification</h3>
<p>SaMscL (accession number: <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/protein/CAG43065" rel="noopener noreferrer" target="_blank">CAG43065</a>) was expressed using the pET21a vector in PB116 <em>E. coli</em> cells grown to O.D.<sub>600</sub> 0.6–0.8 and induced with 1 mM IPTG for 2–3 h at 37°C. Cells were harvested and stored at −80°C until needed. Cells containing SaMscL were lysed in 50 mM NaPi pH 8.0, 300 mM NaCl, 0.5 µg/ml DNAse, 1 mg/ml lysozyme, and 150 µl protease inhibitors (Sigma) using a French press at 16K PSI at 4°C. The cell lysate was incubated with 40 mM n-Decyl-β-D-Maltopyranoside (DM) for 1 h at 4°C and then centrifuged at 24,000× g for 30 min at 4°C to clarify the lysate. 4 mM imidazol was added to the resulting supernatant and then bound to 4 ml of Ni-NTA slury pre-equilibrated with 50 mM NaPi pH 8.0, 300 mM NaCl, 5 mM imidazol, and 40 mM DM for 1 h at 4°C. Standard metal affinity chromatography was performed with 10 column volumes of wash buffer containing 50 mM NaPi pH 8.0, 300 mM NaCl, 30 mM imidazol, 4 mM DM, and SaMscL was eluted with 50 mM NaPi pH 8.0, 300 mM NaCl, 300 mM imidazol, and 4 mM DM. The purest protein fractions were combined and concentrated using a 10,000 molecular weight cutoff filter (Amicon). Detergent exchange was accomplished by adding 40 mM Lauryldimethylamine-oxide (LDAO) or 5% (w/v) pentaethylene glycol monooctyl ether (C<sub>8</sub>E<sub>5</sub>) to the concentrated SaMscL and allowing the protein-detergent complex to equilibrate overnight at room temperature. The following day the protein was subjected to size exclusion chromatography using a superdex 200 column (GE Healthcare) in 20 mM NaPi pH 7.5, 170 mM NaCl, and 4 mM LDAO or 0.5% (w/v) C<sub>8</sub>E<sub>5</sub>. The eluting peak was collected and concentrated using a 50,000 molecular weight cutoff filter (Amicon) and used for either sedimentation equilibrium experiments or size exclusion chromatography-multiangle light scattering experiments (SEC-MALS).</p></section><section id="s4b"><h3 class="pmc_sec_title">Crosslinking</h3>
<p>SaMscL and EcoMscL were cloned into pB10 and pET21a vectors and transformed into PB104 or PB116 <em>E. coli</em> and purified as previously described <a aria-describedby="pbio.1000555-Li1" class="usa-link" href="#pbio.1000555-Li1">[10]</a>. 2% Triton X-100 or 44 mM LDAO was used for MscL extraction with a final concentration of 0.2% and 2.1 mM, respectively, after purification. For crosslinking reactions 5 µg of protein was incubated with 2 mM DSS (Thermo Scientific) on ice for 5 min. The reaction was quenched with a final concentration of 100 mM Tris pH 7.5 for 15 min at room temperature as recommended by the manufacturer. Samples were brought up in non-reducing sample buffer, were loaded on a Criterion 4–20% gel, and were subjected to Western as previously described <a aria-describedby="pbio.1000555-Li1" class="usa-link" href="#pbio.1000555-Li1">[10]</a>. The primary antibody anti-Penta His (Qiagen) was used at 1∶4,000 and the secondary Goat anti-Mouse HRP (Bio-Rad) at 1∶100,000. Blots were developed using HRP substrate (Millipore) and exposed to film.</p></section><section id="s4c"><h3 class="pmc_sec_title">In Vivo Disulfide Trapping</h3>
<p>Overnight cultures were diluted 1∶100 and grown 1 h at 37°C in LB. LB with 1 M NaCl was then added for a final concentration of 0.5 M. Cultures were then induced with 1 mM IPTG for 1 h when an OD 600 of 0.2 was reached. Cultures were either Mock shocked (0.5 M NaCl LB) or shocked (water with 1.5 µM copper phenanthroline) at a 1∶20 dilution for 15 min at 37°C. Samples were spun down at 4,000 g for 20 min and immediately re-suspended in non-reducing sample buffer, adjusted for final OD, and run on a 4%–20% gel (Bio-Rad) for Western blot analysis <a aria-describedby="pbio.1000555-Li1" class="usa-link" href="#pbio.1000555-Li1">[10]</a>. The primary antibody anti-Penta His (Qiagen) was used at 1∶4,000 and the secondary Goat anti-Mouse HRP (Bio-Rad) at 1∶40,000. Blots were developed using HRP substrate (Millipore) and exposed to film.</p></section><section id="s4d"><h3 class="pmc_sec_title">SEC-MALS</h3>
<p>A Shimadzu Prominence HPLC system equipped with a Shodex KW-803 SEC column was connected in line with a miniDAWN-TREOS light scattering instrument (Wyatt Technologies) and Optilab rEX refractometer (Wyatt Technologies). The light scattering and refractive index instruments were calibrated following the manufacturer's guidelines, and 40 ml of running buffer (20 mM NaPi pH 7.5, 170 mM NaCl and 4 mM LDAO or 0.5% (w/v) C<sub>8</sub>E<sub>5</sub>) was used to equilibrate the SEC column and establish stable baselines for the light scattering and refractive index instruments. 25–75 µg of purified protein was used for each SEC-MALS experiment, and data were collected using ASTRA V software (Wyatt Technologies) and processed following the manufacturer's guidelines. LDAO and C<sub>8</sub>E<sub>5</sub> dn/dc values (0.1490 and 0.1240, respectively) were measured using the Optilab rEX following the manufacturer's guidelines and the SaMscL extinction coefficient was calculated from the amino acid sequence.</p></section><section id="s4e"><h3 class="pmc_sec_title">Analytical Ultracentrifugation</h3>
<p>Buffer density-matching experiments were performed in a Beckman Coulter XL-I Analytical Ultracentrifuge to identify the appropriate solution density to match the density of the C<sub>8</sub>E<sub>5</sub> detergent. Volumes of 180 µL of solutions containing 20 mM NaPi pH 7.5, 0.5% (w/v) C<sub>8</sub>E<sub>5</sub>, and various concentrations of NaCl were placed in the sample sector of a dual-sectored, charcoal-filled Epon centerpiece (Beckman-Coulter; all AUC experiments were carried out in such centerpieces) that was sandwiched between sapphire windows. Identical buffers without detergent were placed in the reference sector. The centrifugation cells were centrifuged at a temperature of 25°C and a rotor speed of 50,000 rpm; an An-50Ti rotor was used for all centrifugation experiments. The formation of any detergent concentration gradient was monitored using interference optics. C<sub>8</sub>E<sub>5</sub> was found to be neutrally buoyant in 20 mM NaPi pH 7.5, 170 mM NaCl, which was then used for all sedimentation equilibrium centrifugation experiments with 0.5% C<sub>8</sub>E<sub>5</sub> at 25°C. 4, 8, 40, and 80 µM SaMscL samples were prepared and loaded into the assembled ultracentrifuge cells; buffer with detergent was deposited in the reference sector. Long solution columns (180 µL) were also used in this experiment. Centrifugation was performed at 25°C at 9,500, 14,000, and 17,500 rpm. The data were analyzed using SEDFIT to determine if the samples had come to equilibrium, and SEDPHAT <a aria-describedby="pbio.1000555-Vistica1" class="usa-link" href="#pbio.1000555-Vistica1">[23]</a> was used to fit the sedimentation equilibrium data to a single species model. Multispeed analysis with mass-conservation constraints was performed, allowing the time-invariant and radially invariant noise elements to be calculated and subtracted from the data <a aria-describedby="pbio.1000555-Vistica1" class="usa-link" href="#pbio.1000555-Vistica1">[23]</a>. A 68.3% confidence interval was calculated as previously described <a aria-describedby="pbio.1000555-Bevington1" class="usa-link" href="#pbio.1000555-Bevington1">[24]</a>,<a aria-describedby="pbio.1000555-Houtman1" class="usa-link" href="#pbio.1000555-Houtman1">[25]</a>. The error number reported is the average of the positive and negative deviations from best mass value. For the determination of the amount of detergent bound per gram of protein (δ<em><sub>D</sub></em>), buffer without detergent was used in the reference sector, and two different concentrations (41 and 66 µM) of MscL were placed in the sample sector. The volume of each sample and reference was 390 µl. The rotor speed used was 50,000 rpm, and the temperature setting was 20°C. The data were analyzed in SEDFIT using the non-interacting discrete species model <a aria-describedby="pbio.1000555-Brown1" class="usa-link" href="#pbio.1000555-Brown1">[26]</a>,<a aria-describedby="pbio.1000555-Schuck1" class="usa-link" href="#pbio.1000555-Schuck1">[27]</a>. The flotation of the detergent micelles (the density of LDAO is less than that of water) was explicitly accounted for by giving that species a negative sedimentation coefficient. The amounts of absorbance units and interference fringes present for the main sedimenting species were derived from this analysis. Using the calculated extinction coefficient of the protein and the known refractive index increments of the detergent and the protein, <em>δ<sub>D</sub></em> was calculated as in le Maire et al. <a aria-describedby="pbio.1000555-leMaire1" class="usa-link" href="#pbio.1000555-leMaire1">[28]</a>. Subsequent sedimentation velocity experiments were performed with 20, 32, and 64 µM SaMscL samples in 4 mM LDAO at 50,000 rpm at 20°C. The volume centrifuged was 390 µL. In these cases, the reference buffer contained the same [LDAO] as the sample. The absorbance data (interference data were not collected) were initially analyzed using the <em>c</em>(<em>s</em>) model in SEDFIT <a aria-describedby="pbio.1000555-Brown1" class="usa-link" href="#pbio.1000555-Brown1">[26]</a>. Although LDAO has a very low extinction coefficient at 280 nm, its floatation was detectable; the range of data near to the meniscus was therefore eliminated for this analysis. In these analyses, small quantities (2%–3% of the total signal) of large species were detected. Using this information, all species, including the floating detergent, were modeled in the “species analysis” model in SEDPHAT <a aria-describedby="pbio.1000555-Brown1" class="usa-link" href="#pbio.1000555-Brown1">[26]</a>. This approach is referred to as the “<em>s</em> and <em>D</em>” method in the text, because both the sedimentation coefficient and the diffusion coefficient are refined to arrive at the buoyant molar mass (<em>M<sub>b</sub></em>). <em>M<sub>b</sub></em> was transformed to <em>M</em> (the molar mass of the protein) using the formula</p>
<table class="disp-formula p" id="disp-formula1"><tr>
<td class="formula"><img alt="graphic file with name pbio.1000555.e001.jpg" class="graphic" height="41" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/e54a207998f9/pbio.1000555.e001.jpg" width="106"/></td>
<td class="label">(1)</td>
</tr></table>
<p>where</p>
<table class="disp-formula p" id="disp-formula2"><tr>
<td class="formula"><img alt="graphic file with name pbio.1000555.e002.jpg" class="graphic" height="18" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/8485b0298955/pbio.1000555.e002.jpg" width="243"/></td>
<td class="label">(2)</td>
</tr></table>
<p>
<a aria-describedby="pbio.1000555-Tanford1" class="usa-link" href="#pbio.1000555-Tanford1">[29]</a>
</p>
<p>The symbol <em>ρ</em> is the solution density, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/c2f09937b05d/pbio.1000555.e003.jpg"/></span> is the partial specific volume of the protein, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/cf3ff1e4b67e/pbio.1000555.e004.jpg"/></span> is the partial specific volume of the detergent. Any lipids bound to SaMscL were neglected in these calculations. <em>M<sub>b</sub></em> was also calculated using the relationship</p>
<table class="disp-formula p" id="disp-formula3"><tr>
<td class="formula"><img alt="graphic file with name pbio.1000555.e005.jpg" class="graphic" height="17" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/27f2bd8fd046/pbio.1000555.e005.jpg" width="113"/></td>
<td class="label">(3)</td>
</tr></table>
<p>where <em>N</em>
<sub>0</sub> is Avogadro's number, <em>η</em> is the solution viscosity, <em>R<sub>s</sub></em> is the Stokes radius of the protein/detergent complex, and <em>s</em> is the sedimentation coefficient. The Stokes radius of the protein was obtained by calibrating the SEC column described above with proteins of known Stokes radius. <em>R<sub>s</sub></em> for SaMscL in LDAO was 4.2 nm. Again, <em>M<sub>b</sub></em> was transformed to <em>M</em> using Eq. 1. Values for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/c2f09937b05d/pbio.1000555.e006.jpg"/></span>, <em>η</em>, and <em>ρ</em> were estimated using the program SEDNTERP <a aria-describedby="pbio.1000555-Laue1" class="usa-link" href="#pbio.1000555-Laue1">[30]</a>. The value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/cf3ff1e4b67e/pbio.1000555.e007.jpg"/></span> was taken as 1.13 mL/g <a aria-describedby="pbio.1000555-leMaire2" class="usa-link" href="#pbio.1000555-leMaire2">[31]</a>. Confidence intervals of <em>s</em> and <em>M<sub>b</sub></em> were calculated as detailed above. A confidence interval for the value of Stokes radius, which was derived from a linear regression, was:</p>
<table class="disp-formula p" id="disp-formula4"><tr><td class="formula"><img alt="graphic file with name pbio.1000555.e008.jpg" class="graphic" height="51" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/6fc22f801b74/pbio.1000555.e008.jpg" width="223"/></td></tr></table>
<p>where <em>t</em> is the <em>t</em> statistic, <em>s<sub>s</sub></em> is the estimate of <em>σ</em> for the regression line, <em>V<sub>e</sub></em> is the elution volume of MscL from the SEC column, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/348af5018125/pbio.1000555.e009.jpg"/></span> is the average elution volume of the standards, <em>n</em> is the number of standards, and <em>S<sub>VV</sub></em> is the sum of squared deviations of the individual standard compared to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img alt="Inline graphic" class="inline" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/348af5018125/pbio.1000555.e010.jpg"/></span>. In cases of asymmetric error intervals, the <em>σ</em> was taken as the average of the difference between the boundary value and the best-fitted value. Because multiple experiments, each having different errors, were performed, the value of <em>M</em> reported in the main text is the weighted mean <em>μ<sub>w</sub></em> ± the weighted <em>σ</em> (<em>σ</em>
<sub>w</sub>), calculated using the formulas:</p>
<table class="disp-formula p" id="disp-formula5"><tr><td class="formula"><img alt="graphic file with name pbio.1000555.e011.jpg" class="graphic" height="66" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/55f98b81bbb7/pbio.1000555.e011.jpg" width="138"/></td></tr></table>
<table class="disp-formula p" id="disp-formula6"><tr><td class="formula"><img alt="graphic file with name pbio.1000555.e012.jpg" class="graphic" height="51" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e33/2998437/a0dbe1e9d508/pbio.1000555.e012.jpg" width="71"/></td></tr></table>
<p>for the <em>I</em> experiments preformed (<em>i</em> = 3 in this case).</p></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors thank Christina Eaton and Drs. Irene Iscla, Yuezhou Li, and Limin Yang for helpful discussions.</p></section><section class="glossary" id="glossary1"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>C<sub>8</sub>E<sub>5</sub>
</dt>
<dd><p>pentaethylene glycol monooctyl ether</p></dd>
<dt>EcoMscL</dt>
<dd><p>
<em>E. coli</em> MscL</p></dd>
<dt>LDAO</dt>
<dd><p>n-dodecyl-N,N-dimethylamine-N-oxide</p></dd>
<dt>MtMscL</dt>
<dd><p>
<em>M. tuberculosis</em> MscL</p></dd>
<dt>SaMscL</dt>
<dd><p>
<em>S. auresus</em> MscL</p></dd>
<dt>SDS</dt>
<dd><p>sodium dodecyl sulphate</p></dd>
<dt>SE</dt>
<dd><p>sedimentation equilibrium analytical ultracentrifugation</p></dd>
<dt>SEC-MALS</dt>
<dd><p>size exclusion chromatography multi-angle light scattering</p></dd>
</dl></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn2"><p>The authors have declared that no competing interests exist.</p></div>
<div class="fn p" id="fn3"><p>MRD and YJ were supported by the Howard Hughes Medical Institute, the National Institutes of Health (grant GM079179 to YJ) and the Welch Foundation (grant I-1578 to YJ). RW and PB were supported by grant I-1420 of the Welch Foundation, grant NNH08ZTT003N NRA from NASA, grant RP100146 from the Cancer Prevention &amp; Research Institute of Texas (CPRIT), and grant GM61028 and its supplement from the National Institutes of Health. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></div>
</div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pbio.1000555-Levina1">
<span class="label">1.</span><cite>Levina N, Totemeyer S, Stokes N. R, Louis P, Jones M. A, et al.  Protection of Escherichia coli cells against extreme turgor by activation of MscS and MscL mechanosensitive channels: identification of genes required for MscS activity. EMBO Journal. 1999;18:1730–1737. doi: 10.1093/emboj/18.7.1730.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/emboj/18.7.1730" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1171259/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10202137/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Journal&amp;title=Protection%20of%20Escherichia%20coli%20cells%20against%20extreme%20turgor%20by%20activation%20of%20MscS%20and%20MscL%20mechanosensitive%20channels:%20identification%20of%20genes%20required%20for%20MscS%20activity.&amp;author=N%20Levina&amp;author=S%20Totemeyer&amp;author=N.%20R%20Stokes&amp;author=P%20Louis&amp;author=M.%20A%20Jones&amp;volume=18&amp;publication_year=1999&amp;pages=1730-1737&amp;pmid=10202137&amp;doi=10.1093/emboj/18.7.1730&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Blount1">
<span class="label">2.</span><cite>Blount P, Iscla I, Moe P. C, Li Y.  MscL: the bacterial mechanosensitive channel of large conductance. In: Hamill O. P, editor. Mechanosensitive ion channels (a volume in the current topics in membranes series) St. Louis, MO: Elsievier Press; 2007. pp. 202–233.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Mechanosensitive%20ion%20channels%20(a%20volume%20in%20the%20current%20topics%20in%20membranes%20series)&amp;author=P%20Blount&amp;author=I%20Iscla&amp;author=P.%20C%20Moe&amp;author=Y%20Li&amp;publication_year=2007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Blount2">
<span class="label">3.</span><cite>Blount P, Li Y, Moe P. C, Iscla I.  Mechanosensitive channels gated by membrane tension: bacteria and beyond. In: Kamkin A, Kiseleva I, editors. Mechanosensitive ion channels (a volume in the mechanosensitivity in cells and tissues, Moscow Academia series) New York: Springer Press; 2008. pp. 71–101.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Mechanosensitive%20ion%20channels%20(a%20volume%20in%20the%20mechanosensitivity%20in%20cells%20and%20tissues,%20Moscow%20Academia%20series)&amp;author=P%20Blount&amp;author=Y%20Li&amp;author=P.%20C%20Moe&amp;author=I%20Iscla&amp;publication_year=2008&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Blount3">
<span class="label">4.</span><cite>Blount P, Sukharev S. I, Moe P. C, Schroeder M. J, Guy H. R, et al.  Membrane topology and multimeric structure of a mechanosensitive channel protein of Escherichia coli. EMBO Journal. 1996;15:4798–4805.</cite> [<a class="usa-link" href="/articles/PMC452216/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8890153/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Journal&amp;title=Membrane%20topology%20and%20multimeric%20structure%20of%20a%20mechanosensitive%20channel%20protein%20of%20Escherichia%20coli.&amp;author=P%20Blount&amp;author=S.%20I%20Sukharev&amp;author=P.%20C%20Moe&amp;author=M.%20J%20Schroeder&amp;author=H.%20R%20Guy&amp;volume=15&amp;publication_year=1996&amp;pages=4798-4805&amp;pmid=8890153&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Saint1">
<span class="label">5.</span><cite>Saint N, Lacapere J. J, Gu L. Q, Ghazi A, Martinac B, et al.  A hexameric transmembrane pore revealed by two-dimensional crystallization of the large mechanosensitive ion channel (MscL) of Escherichia coli. Journal of Biological Chemistry. 1998;273:14667–14670. doi: 10.1074/jbc.273.24.14667.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.273.24.14667" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9614061/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=A%20hexameric%20transmembrane%20pore%20revealed%20by%20two-dimensional%20crystallization%20of%20the%20large%20mechanosensitive%20ion%20channel%20(MscL)%20of%20Escherichia%20coli.&amp;author=N%20Saint&amp;author=J.%20J%20Lacapere&amp;author=L.%20Q%20Gu&amp;author=A%20Ghazi&amp;author=B%20Martinac&amp;volume=273&amp;publication_year=1998&amp;pages=14667-14670&amp;pmid=9614061&amp;doi=10.1074/jbc.273.24.14667&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Chang1">
<span class="label">6.</span><cite>Chang G, Spencer R. H, Lee A. T, Barclay M. T, Rees D. C. Structure of the MscL homolog from Mycobacterium tuberculosis: a gated mechanosensitive ion channel. Science. 1998;282:2220–2226. doi: 10.1126/science.282.5397.2220.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.282.5397.2220" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9856938/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Structure%20of%20the%20MscL%20homolog%20from%20Mycobacterium%20tuberculosis:%20a%20gated%20mechanosensitive%20ion%20channel.&amp;author=G%20Chang&amp;author=R.%20H%20Spencer&amp;author=A.%20T%20Lee&amp;author=M.%20T%20Barclay&amp;author=D.%20C%20Rees&amp;volume=282&amp;publication_year=1998&amp;pages=2220-2226&amp;pmid=9856938&amp;doi=10.1126/science.282.5397.2220&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Sukharev1">
<span class="label">7.</span><cite>Sukharev S. I, Schroeder M. J, McCaslin D. R. Stoichiometry of the large conductance bacterial mechanosensitive channel of E. coli. A biochemical study. Journal of Membrane Biology. 1999;171:183–193. doi: 10.1007/s002329900570.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s002329900570" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10501827/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Membrane%20Biology&amp;title=Stoichiometry%20of%20the%20large%20conductance%20bacterial%20mechanosensitive%20channel%20of%20E.%20coli.%20A%20biochemical%20study.&amp;author=S.%20I%20Sukharev&amp;author=M.%20J%20Schroeder&amp;author=D.%20R%20McCaslin&amp;volume=171&amp;publication_year=1999&amp;pages=183-193&amp;pmid=10501827&amp;doi=10.1007/s002329900570&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Liu1">
<span class="label">8.</span><cite>Liu Z, Gandhi C. S, Rees D. C. Structure of a tetrameric MscL in an expanded intermediate state. Nature. 2009;461:120–124. doi: 10.1038/nature08277.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature08277" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2737600/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19701184/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure%20of%20a%20tetrameric%20MscL%20in%20an%20expanded%20intermediate%20state.&amp;author=Z%20Liu&amp;author=C.%20S%20Gandhi&amp;author=D.%20C%20Rees&amp;volume=461&amp;publication_year=2009&amp;pages=120-124&amp;pmid=19701184&amp;doi=10.1038/nature08277&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Iscla1">
<span class="label">9.</span><cite>Iscla I, Wray R, Blount P. On the structure of the N-terminal domain of the MscL channel: helical bundle or membrane interface. Biophys J. 2008;95:2283–2291. doi: 10.1529/biophysj.107.127423.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1529/biophysj.107.127423" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2517020/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18515388/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=On%20the%20structure%20of%20the%20N-terminal%20domain%20of%20the%20MscL%20channel:%20helical%20bundle%20or%20membrane%20interface.&amp;author=I%20Iscla&amp;author=R%20Wray&amp;author=P%20Blount&amp;volume=95&amp;publication_year=2008&amp;pages=2283-2291&amp;pmid=18515388&amp;doi=10.1529/biophysj.107.127423&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Li1">
<span class="label">10.</span><cite>Li Y, Wray R, Eaton C, Blount P. An open-pore structure of the mechanosensitive channel MscL derived by determining transmembrane domain interactions upon gating. FASEB J. 2009;23:2197–2204. doi: 10.1096/fj.09-129296.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1096/fj.09-129296" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2704598/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19261722/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FASEB%20J&amp;title=An%20open-pore%20structure%20of%20the%20mechanosensitive%20channel%20MscL%20derived%20by%20determining%20transmembrane%20domain%20interactions%20upon%20gating.&amp;author=Y%20Li&amp;author=R%20Wray&amp;author=C%20Eaton&amp;author=P%20Blount&amp;volume=23&amp;publication_year=2009&amp;pages=2197-2204&amp;pmid=19261722&amp;doi=10.1096/fj.09-129296&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Ajouz1">
<span class="label">11.</span><cite>Ajouz B, Berrier C, Garrigues A, Besnard M, Ghazi A. Release of thioredoxin via the mechanosensitive channel MscL during osmotic downshock of Escherichia coli cells. Journal of Biological Chemistry. 1998;273:26670–26674. doi: 10.1074/jbc.273.41.26670.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.273.41.26670" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9756908/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Release%20of%20thioredoxin%20via%20the%20mechanosensitive%20channel%20MscL%20during%20osmotic%20downshock%20of%20Escherichia%20coli%20cells.&amp;author=B%20Ajouz&amp;author=C%20Berrier&amp;author=A%20Garrigues&amp;author=M%20Besnard&amp;author=A%20Ghazi&amp;volume=273&amp;publication_year=1998&amp;pages=26670-26674&amp;pmid=9756908&amp;doi=10.1074/jbc.273.41.26670&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Slotboom1">
<span class="label">12.</span><cite>Slotboom D. J, Duurkens R. H, Olieman K, Erkens G. B. Static light scattering to characterize membrane proteins in detergent solution. Methods. 2008;46:73–82. doi: 10.1016/j.ymeth.2008.06.012.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ymeth.2008.06.012" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18625320/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Static%20light%20scattering%20to%20characterize%20membrane%20proteins%20in%20detergent%20solution.&amp;author=D.%20J%20Slotboom&amp;author=R.%20H%20Duurkens&amp;author=K%20Olieman&amp;author=G.%20B%20Erkens&amp;volume=46&amp;publication_year=2008&amp;pages=73-82&amp;pmid=18625320&amp;doi=10.1016/j.ymeth.2008.06.012&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Steinbacher1">
<span class="label">13.</span><cite>Steinbacher S, Bass R, Strop P, Rees D. C.  Structures of the prokaryotic mechanosensitive channels MscL and MscS. In: Hamill O. P, editor. Mechanosensitive ion channels (a volume in the Current Topics in Membranes series) St. Louis, MO: Elsievier Press; 2007. pp. 1–20.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Mechanosensitive%20ion%20channels%20(a%20volume%20in%20the%20Current%20Topics%20in%20Membranes%20series)&amp;author=S%20Steinbacher&amp;author=R%20Bass&amp;author=P%20Strop&amp;author=D.%20C%20Rees&amp;publication_year=2007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Blount4">
<span class="label">14.</span><cite>Blount P, Sukharev S. I, Moe P. C, Martinac B, Kung C.  Mechanosensitive channels of bacteria. In: Conn P. M, editor. Methods in enzymology. San Diego, CA: Academic Press; 1999. pp. 458–482.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s0076-6879(99)94027-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9916243/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20enzymology&amp;author=P%20Blount&amp;author=S.%20I%20Sukharev&amp;author=P.%20C%20Moe&amp;author=B%20Martinac&amp;author=C%20Kung&amp;publication_year=1999&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Perozo1">
<span class="label">15.</span><cite>Perozo E, Cortes D. M, Sompornpisut P, Kloda A, Martinac B. Open channel structure of MscL and the gating mechanism of mechanosensitive channels. Nature. 2002;418:942–948. doi: 10.1038/nature00992.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature00992" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12198539/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Open%20channel%20structure%20of%20MscL%20and%20the%20gating%20mechanism%20of%20mechanosensitive%20channels.&amp;author=E%20Perozo&amp;author=D.%20M%20Cortes&amp;author=P%20Sompornpisut&amp;author=A%20Kloda&amp;author=B%20Martinac&amp;volume=418&amp;publication_year=2002&amp;pages=942-948&amp;pmid=12198539&amp;doi=10.1038/nature00992&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Bartlett1">
<span class="label">16.</span><cite>Bartlett J. L, Levin G, Blount P. An in vivo assay identifies changes in residue accessibility on mechanosensitive channel gating. Proc Natl Acad Sci U S A. 2004;101:10161–10165. doi: 10.1073/pnas.0402040101.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0402040101" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC454182/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15226501/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=An%20in%20vivo%20assay%20identifies%20changes%20in%20residue%20accessibility%20on%20mechanosensitive%20channel%20gating.&amp;author=J.%20L%20Bartlett&amp;author=G%20Levin&amp;author=P%20Blount&amp;volume=101&amp;publication_year=2004&amp;pages=10161-10165&amp;pmid=15226501&amp;doi=10.1073/pnas.0402040101&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Bartlett2">
<span class="label">17.</span><cite>Bartlett J. L, Li Y, Blount P. Mechanosensitive channel gating transitions resolved by functional changes upon pore modification. Biophys J. 2006;91:3684–3691. doi: 10.1529/biophysj.106.088062.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1529/biophysj.106.088062" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1630475/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16935962/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Mechanosensitive%20channel%20gating%20transitions%20resolved%20by%20functional%20changes%20upon%20pore%20modification.&amp;author=J.%20L%20Bartlett&amp;author=Y%20Li&amp;author=P%20Blount&amp;volume=91&amp;publication_year=2006&amp;pages=3684-3691&amp;pmid=16935962&amp;doi=10.1529/biophysj.106.088062&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Iscla2">
<span class="label">18.</span><cite>Iscla I, Levin G, Wray R, Reynolds R, Blount P. Defining the physical gate of a mechanosensitive channel, MscL, by engineering metal-binding sites. Biophys J. 2004;87(5):3172–3180. doi: 10.1529/biophysj.104.049833.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1529/biophysj.104.049833" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1304787/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15339809/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Defining%20the%20physical%20gate%20of%20a%20mechanosensitive%20channel,%20MscL,%20by%20engineering%20metal-binding%20sites.&amp;author=I%20Iscla&amp;author=G%20Levin&amp;author=R%20Wray&amp;author=R%20Reynolds&amp;author=P%20Blount&amp;volume=87&amp;issue=5&amp;publication_year=2004&amp;pages=3172-3180&amp;pmid=15339809&amp;doi=10.1529/biophysj.104.049833&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Li2">
<span class="label">19.</span><cite>Li Y, Wray R, Blount P. Intragenic suppression of gain-of-function mutations in the Escherichia coli mechanosensitive channel, MscL. Mol Microbiol. 2004;53:485–495. doi: 10.1111/j.1365-2958.2004.04150.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-2958.2004.04150.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15228529/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&amp;title=Intragenic%20suppression%20of%20gain-of-function%20mutations%20in%20the%20Escherichia%20coli%20mechanosensitive%20channel,%20MscL.&amp;author=Y%20Li&amp;author=R%20Wray&amp;author=P%20Blount&amp;volume=53&amp;publication_year=2004&amp;pages=485-495&amp;pmid=15228529&amp;doi=10.1111/j.1365-2958.2004.04150.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Berrier1">
<span class="label">20.</span><cite>Berrier C, Park K. H, Abes S, Bibonne A, Betton J. M, et al.  Cell-free synthesis of a functional ion channel in the absence of a membrane and in the presence of detergent. Biochemistry. 2004;43:12585–12591. doi: 10.1021/bi049049y.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/bi049049y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15449948/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Cell-free%20synthesis%20of%20a%20functional%20ion%20channel%20in%20the%20absence%20of%20a%20membrane%20and%20in%20the%20presence%20of%20detergent.&amp;author=C%20Berrier&amp;author=K.%20H%20Park&amp;author=S%20Abes&amp;author=A%20Bibonne&amp;author=J.%20M%20Betton&amp;volume=43&amp;publication_year=2004&amp;pages=12585-12591&amp;pmid=15449948&amp;doi=10.1021/bi049049y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Clayton1">
<span class="label">21.</span><cite>Clayton D, Shapovalov G, Maurer J. A, Dougherty D. A, Lester H. A, et al.  Total chemical synthesis and electrophysiological characterization of mechanosensitive channels from Escherichia coli and Mycobacterium tuberculosis. Proc Natl Acad Sci U S A. 2004;101:4764–4769. doi: 10.1073/pnas.0305693101.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0305693101" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC387322/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15041744/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Total%20chemical%20synthesis%20and%20electrophysiological%20characterization%20of%20mechanosensitive%20channels%20from%20Escherichia%20coli%20and%20Mycobacterium%20tuberculosis.&amp;author=D%20Clayton&amp;author=G%20Shapovalov&amp;author=J.%20A%20Maurer&amp;author=D.%20A%20Dougherty&amp;author=H.%20A%20Lester&amp;volume=101&amp;publication_year=2004&amp;pages=4764-4769&amp;pmid=15041744&amp;doi=10.1073/pnas.0305693101&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Yoshimura1">
<span class="label">22.</span><cite>Yoshimura K, Usukura J, Sokabe M. Gating-associated conformational changes in the mechanosensitive channel MscL. Proc Natl Acad Sci U S A. 2008;105:4033–4038. doi: 10.1073/pnas.0709436105.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0709436105" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2268802/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18310324/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Gating-associated%20conformational%20changes%20in%20the%20mechanosensitive%20channel%20MscL.&amp;author=K%20Yoshimura&amp;author=J%20Usukura&amp;author=M%20Sokabe&amp;volume=105&amp;publication_year=2008&amp;pages=4033-4038&amp;pmid=18310324&amp;doi=10.1073/pnas.0709436105&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Vistica1">
<span class="label">23.</span><cite>Vistica J, Dam J, Balbo A, Yikilmaz E, Mariuzza R. A, et al.  Sedimentation equilibrium analysis of protein interactions with global implicit mass conservation constraints and systematic noise decomposition. Anal Biochem. 2004;326:234–256. doi: 10.1016/j.ab.2003.12.014.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ab.2003.12.014" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15003564/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&amp;title=Sedimentation%20equilibrium%20analysis%20of%20protein%20interactions%20with%20global%20implicit%20mass%20conservation%20constraints%20and%20systematic%20noise%20decomposition.&amp;author=J%20Vistica&amp;author=J%20Dam&amp;author=A%20Balbo&amp;author=E%20Yikilmaz&amp;author=R.%20A%20Mariuzza&amp;volume=326&amp;publication_year=2004&amp;pages=234-256&amp;pmid=15003564&amp;doi=10.1016/j.ab.2003.12.014&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Bevington1">
<span class="label">24.</span><cite>Bevington P, Robinson D. K. Data reduction and error analysis for the physical sciences: McGraw Hill. 2002:336.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?Bevington%20P,%20Robinson%20D.%20K.%20Data%20reduction%20and%20error%20analysis%20for%20the%20physical%20sciences:%20McGraw%20Hill.%202002:336." rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Houtman1">
<span class="label">25.</span><cite>Houtman J. C, Brown P. H, Bowden B, Yamaguchi H, Appella E, et al.  Studying multisite binary and ternary protein interactions by global analysis of isothermal titration calorimetry data in SEDPHAT: application to adaptor protein complexes in cell signaling. Protein Sci. 2007;16:30–42. doi: 10.1110/ps.062558507.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1110/ps.062558507" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1794685/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17192587/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Studying%20multisite%20binary%20and%20ternary%20protein%20interactions%20by%20global%20analysis%20of%20isothermal%20titration%20calorimetry%20data%20in%20SEDPHAT:%20application%20to%20adaptor%20protein%20complexes%20in%20cell%20signaling.&amp;author=J.%20C%20Houtman&amp;author=P.%20H%20Brown&amp;author=B%20Bowden&amp;author=H%20Yamaguchi&amp;author=E%20Appella&amp;volume=16&amp;publication_year=2007&amp;pages=30-42&amp;pmid=17192587&amp;doi=10.1110/ps.062558507&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Brown1">
<span class="label">26.</span><cite>Brown P. H, Schuck P. A new adaptive grid-size algorithm for the simulation of sedimentation velocity profiles in analytical ultracentrifugation. Comput Phys Commun. 2008;178:105–120. doi: 10.1016/j.cpc.2007.08.012.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cpc.2007.08.012" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2267755/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18196178/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Comput%20Phys%20Commun&amp;title=A%20new%20adaptive%20grid-size%20algorithm%20for%20the%20simulation%20of%20sedimentation%20velocity%20profiles%20in%20analytical%20ultracentrifugation.&amp;author=P.%20H%20Brown&amp;author=P%20Schuck&amp;volume=178&amp;publication_year=2008&amp;pages=105-120&amp;pmid=18196178&amp;doi=10.1016/j.cpc.2007.08.012&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Schuck1">
<span class="label">27.</span><cite>Schuck P. Size-distribution analysis of macromolecules by sedimentation velocity ultracentrifugation and lamm equation modeling. Biophys J. 2000;78:1606–1619. doi: 10.1016/S0006-3495(00)76713-0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0006-3495(00)76713-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1300758/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10692345/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Size-distribution%20analysis%20of%20macromolecules%20by%20sedimentation%20velocity%20ultracentrifugation%20and%20lamm%20equation%20modeling.&amp;author=P%20Schuck&amp;volume=78&amp;publication_year=2000&amp;pages=1606-1619&amp;pmid=10692345&amp;doi=10.1016/S0006-3495(00)76713-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-leMaire1">
<span class="label">28.</span><cite>le Maire M, Arnou B, Olesen C, Georgin D, Ebel C, et al.  Gel chromatography and analytical ultracentrifugation to determine the extent of detergent binding and aggregation, and Stokes radius of membrane proteins using sarcoplasmic reticulum Ca2+-ATPase as an example. Nat Protoc. 2008;3:1782–1795. doi: 10.1038/nprot.2008.177.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nprot.2008.177" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18974737/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&amp;title=Gel%20chromatography%20and%20analytical%20ultracentrifugation%20to%20determine%20the%20extent%20of%20detergent%20binding%20and%20aggregation,%20and%20Stokes%20radius%20of%20membrane%20proteins%20using%20sarcoplasmic%20reticulum%20Ca2+-ATPase%20as%20an%20example.&amp;author=M%20le%20Maire&amp;author=B%20Arnou&amp;author=C%20Olesen&amp;author=D%20Georgin&amp;author=C%20Ebel&amp;volume=3&amp;publication_year=2008&amp;pages=1782-1795&amp;pmid=18974737&amp;doi=10.1038/nprot.2008.177&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Tanford1">
<span class="label">29.</span><cite>Tanford C, Reynolds J. A. Characterization of membrane proteins in detergent solutions. Biochim Biophys Acta. 1976;457:133–170. doi: 10.1016/0304-4157(76)90009-5.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0304-4157(76)90009-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/135582/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&amp;title=Characterization%20of%20membrane%20proteins%20in%20detergent%20solutions.&amp;author=C%20Tanford&amp;author=J.%20A%20Reynolds&amp;volume=457&amp;publication_year=1976&amp;pages=133-170&amp;pmid=135582&amp;doi=10.1016/0304-4157(76)90009-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-Laue1">
<span class="label">30.</span><cite>Laue T. M, Shah B. D, Ridgeway R. M, Pelletier S. L.  Computer-aided interpretation of analytical sedimentation data for proteins. In: Harding S. E, Rowe A. J, Horton J. C, editors. Analytical ultracentrifugation in biochemistry and polymer science. Cambridge, UK: The Royal Society of Chemistry; 1992. pp. 90–125.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Analytical%20ultracentrifugation%20in%20biochemistry%20and%20polymer%20science&amp;author=T.%20M%20Laue&amp;author=B.%20D%20Shah&amp;author=R.%20M%20Ridgeway&amp;author=S.%20L%20Pelletier&amp;publication_year=1992&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pbio.1000555-leMaire2">
<span class="label">31.</span><cite>le Maire M, Champeil P, Moller J. V. Interaction of membrane proteins and lipids with solubilizing detergents. Biochim Biophys Acta. 2000;1508:86–111. doi: 10.1016/s0304-4157(00)00010-1.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s0304-4157(00)00010-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11090820/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&amp;title=Interaction%20of%20membrane%20proteins%20and%20lipids%20with%20solubilizing%20detergents.&amp;author=M%20le%20Maire&amp;author=P%20Champeil&amp;author=J.%20V%20Moller&amp;volume=1508&amp;publication_year=2000&amp;pages=86-111&amp;pmid=11090820&amp;doi=10.1016/s0304-4157(00)00010-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section></section></section></article>
</main></body></html>