Open Microbiol J

. 2014 Jun 13;8:51–58. doi: [10.2174/1874285801408010051](https://doi.org/10.2174/1874285801408010051)

# Conservation of the Low-shear Modeled Microgravity Response in Enterobacteriaceae and Analysis of the *trp* Genes in this Response

[Anjali Soni](https://pubmed.ncbi.nlm.nih.gov/?term=%22Soni%20A%22%5BAuthor%5D)

### Anjali Soni

1Villanova University, Biology Department, 800 Lancaster Avenue, Villanova, PA 19085

2Virginia Commonwealth University, School of Dentistry, Richmond, VA23298

Find articles by [Anjali Soni](https://pubmed.ncbi.nlm.nih.gov/?term=%22Soni%20A%22%5BAuthor%5D)

1,2, [Laura O’Sullivan](https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Sullivan%20L%22%5BAuthor%5D)

### Laura O’Sullivan

1Villanova University, Biology Department, 800 Lancaster Avenue, Villanova, PA 19085

3University of Pennsylvania,School of Veterinary Medicine, Philadelphia, PA 19104

Find articles by [Laura O’Sullivan](https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Sullivan%20L%22%5BAuthor%5D)

1,3, [Laura N Quick](https://pubmed.ncbi.nlm.nih.gov/?term=%22Quick%20LN%22%5BAuthor%5D)

### Laura N Quick

1Villanova University, Biology Department, 800 Lancaster Avenue, Villanova, PA 19085

4Children’s Hospital of Philadelphia, Philadelphia, PA 19104

Find articles by [Laura N Quick](https://pubmed.ncbi.nlm.nih.gov/?term=%22Quick%20LN%22%5BAuthor%5D)

1,4, [C Mark Ott](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ott%20CM%22%5BAuthor%5D)

### C Mark Ott

5NASA/Johnson Space Center, Habitability and Environmental Factors Division, Houston, TX77058

Find articles by [C Mark Ott](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ott%20CM%22%5BAuthor%5D)

5, [Cheryl A Nickerson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Nickerson%20CA%22%5BAuthor%5D)

### Cheryl A Nickerson

6Arizona State University, Biodesign Institute, Center for Infectious Diseases and Vaccinology, Tempe, AZ85281

Find articles by [Cheryl A Nickerson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Nickerson%20CA%22%5BAuthor%5D)

6, [James W Wilson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilson%20JW%22%5BAuthor%5D)

### James W Wilson

1Villanova University, Biology Department, 800 Lancaster Avenue, Villanova, PA 19085

Find articles by [James W Wilson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilson%20JW%22%5BAuthor%5D)

1,\*

* Author information
* Article notes
* Copyright and License information

1Villanova University, Biology Department, 800 Lancaster Avenue, Villanova, PA 19085

2Virginia Commonwealth University, School of Dentistry, Richmond, VA23298

3University of Pennsylvania,School of Veterinary Medicine, Philadelphia, PA 19104

4Children’s Hospital of Philadelphia, Philadelphia, PA 19104

5NASA/Johnson Space Center, Habitability and Environmental Factors Division, Houston, TX77058

6Arizona State University, Biodesign Institute, Center for Infectious Diseases and Vaccinology, Tempe, AZ85281

\*

Address correspondence to this author at the Villanova University, Biology Department, 800 Lancaster Avenue, Villanova, PA 19085;
Tel: 610-519-3037; Fax: 610-519-7863; E-mail: james.w.wilson@villanova.edu

Received 2014 Jan 30; Revised 2014 Mar 19; Accepted 2014 Apr 3; Collection date 2014.

© Soni *et al.*; Licensee *Bentham Open.*

This is an open access article licensed under the terms of the Creative Commons Attribution Non-Commercial License (<http://creativecommons.org/licenses/by-nc/3.0/>) which permits unrestricted, non-commercial use, distribution and reproduction in any medium, provided the work is properly cited.

[PMC Copyright notice](/about/copyright/)

PMCID: PMC4085587  PMID: [25006354](https://pubmed.ncbi.nlm.nih.gov/25006354/)

## Abstract

Low fluid shear force, including that encountered in microgravity models, induces bacterial responses, but the range of bacteria capable of responding to this signal remains poorly characterized. We systematically analyzed a range of Gram negative Enterobacteriaceae for conservation of the low-shear modeled microgravity (LSMMG) response using phenotypic assays, qPCR, and targeted mutations. Our results indicate LSMMG response conservation across Enterobacteriacae with potential variance in up- or down-regulation of a given response depending on genus. Based on the data, we analyzed the role of the *trp* operon genes and the TrpR regulator in the LSMMG response using targeted mutations in these genes in *S.* Typhimurium and *E. coli*. We found no alteration of the LSMMG response compared to WT in these mutant strains under the conditions tested here. To our knowledge, this study is first-of-kind for *Citrobacter, Enterobacter*, and *Serratia,* presents novel data for *Escherichia*, and provides the first analysis of *trp* genes in LSMMG responses. This impacts our understanding of how LSMMG affects bacteria and our ability to modify bacteria with this condition in the future.

**Keywords:** Enterobacteriaceae, environmental response, low shear modeled microgravity, rotating wall vessel, Salmonella Typhimurium.

## INTRODUCTION

A large body of studies aimed at characterizing the effects of low fluid shear force environments on bacterial cells,including the microgravity environment of spaceflight and ground-based rotating wall vessel (RWV) culture, have focused on the Gram negative enteric pathogen *Salmonella enterica* serovar Typhimurium [[1](#R1)-[7](#R7)]. Other studies have also focused on additional bacteria including *Escherichia coli* [[8](#R8)-[15](#R15)], *Pseudomonas aeruginosa* [[16](#R16)-[18](#R18)], *Yersinia pestis* [[19](#R19), [20](#R20)], and *Staphylococcus aureus* [[15](#R15), [21](#R21), [22](#R22)] with a range of different phenotypic results. However, the range of bacteria capable of responding to this condition and the potential similarities/differences in this response remain poorly characterized. Additionally, to our knowledge, a systematic, “side-by-side” study to examine the conservation of the low fluid shear response in a range of different bacterial genera using common assay conditions has not been reported in the literature. In this study,we analyzed different members of the Gram negative Enterobacteriaceae family for conservation of the low fluid shear response in the RWV using phenotypic and molecular assays to delineate the commonalities and differences of these organisms to culture in this environmental condition. Many members of the Enterobactericeae family are enteric organisms that inhabit the intestine as part of their lifecycles, and consequently, these bacteria likely encounter low fluid shear regions in their natural habitat such as the spaces between microvilli [[23](#R23)]. In addition, as bacterial engineering expands to include a larger range of bacteria, the use of novel growth conditions will be applied to a greater variety of genera.

The RWV culture apparatus (Fig. **[1](#F1)**) allows a physiolo-gically-relevant low fluid shear force growth environment that induces a number of phenotypic responses in bacteria including altered stress resistance [[1](#R1)-[5](#R5), [7](#R7), [12](#R12), [16](#R16), [21](#R21)], increased survival in cellular and animal hosts [[1](#R1), [3](#R3), [4](#R4), [7](#R7)], and altered gene expression [[1](#R1), [3](#R3), [5](#R5), [7](#R7), [13](#R13), [14](#R14), [17](#R17), [18](#R18)]. The RWV is used to induce prokaryotic and eukaryotic cellular phenotypes that are difficult or not possible to obtain using conventional culture conditions [[1](#R1), [18](#R18), [21](#R21), [24](#R24)]. This includes use as a ground-based modeled microgravity environment to study effects/phenotypes that correlate to those observed in spaceflight experiments [[3](#R3), [7](#R7), [17](#R17), [24](#R24)]. The alteration of bacterial virulence observed during growth in spaceflight and in the RWV is currently being used for applications related to bacterial engineering and modification *via* growth conditions [[1](#R1), [3](#R3), [7](#R7), [18](#R18), [25](#R25)-[27](#R27)]. Given these commonalities and based upon previous modeling, the environmental growth condition obtained in the RWV apparatus has been termed low shear modeled microgravity (LSMMG) [[2](#R2), [4](#R4), [5](#R5)].

### Fig. (1).

[![Fig. (1)](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5b7/4085587/be7c800edc6b/TOMICROJ-8-51_F1.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4085587_TOMICROJ-8-51_F1.jpg)

[Open in a new tab](figure/F1/)

**The rotating wall vessel (RWV).**

Panel **A**:
The rotating wall vessel serves as a low fluid shear cell culture apparatus. The
vessel is completely filled with media containing cells (no air bubbles) and
rotated at 25 rpm. In the pictured orientation of the vessel (LSMMG condition),
this creates a low shear, low turbulence suspension culture environment for cell
growth. Aeration is provided *via* an air-permeable membrane located in the
back of the vessel (not visible). Panel **B**: The two orientations of
the RWV for experimental set up. For RWV experiments, two separate apparatuses
are used that are set up in two different orientations using the same culture
inoculation. In the control orientation, the low fluid shear environment is
abolished and increased fluid shear levels are present. The only difference in
the two cultures is the orientation of each vessel. Panel **C**: A
diagram of the orbital path of a representative cell in the LSMMG orientation of
the RWV to demonstrate the nature of the low fluid shear cell suspension. This
is achieved in large part by the solid body rotation of the media in the
apparatus in such a way that sedimentation is offset to allow suspension with a
minimum of fluid shear forces.

Previous experimental results in the literature point to a possible relationship between *trp* genes and the LSMMG/ spaceflight response in bacteria. In *S.* Typhimurium , the *trpD* gene was one of the few genes in a group found to be up-regulated by all low-shear/microgravity conditions tested including LSMMG in LB media [[5](#R5)] and by spaceflight in both LB and M9 media [[3](#R3), [7](#R7)], and this result was observed using both microarray and qPCR analysis [[3](#R3), [5](#R5), [7](#R7)]. In addition, a study with adherent-invasive *E. coli* showed that a mutation in a tryptophan permease gene decreased LSMMG-induced adherence to tissue culture cells, and this phenotype was complemented by restoring the WT gene or by exogenous indole supplementation [[28](#R28)]. These results, combined with new observations in this report, provided significant justification for analyzing the role of *trp* genes in the conservation of the LSMMG response. Overall, our results indicate that the LSMMG response is conserved across Enterobacteriaceae but with potential variation in the “direction” of regulation of a given phenotype, and that associated *trp* genes are not involved in this conservation under the conditions tested here.

## MATERIALS AND METHODS

### Rotating Wall Vessel (RWV) Cultures

Cultures were grown in the RWV in LB (Lennox) broth as previously described [[3](#R3), [4](#R4), [7](#R7)]. Experimental set-up with RWV cultures is depicted in Fig. (**[1](#F1)**). The RWV apparatus used was model RCCS-1 from Synthecon, Inc., Houston, TX as used in previous studies [[3](#R3), [4](#R4), [7](#R7)]. Strains used in this study were as follows: *S.* Typhimurium χ3339 [[29](#R29)], *E. coli* TOP10 (Invitrogen, Carlsbad, CA), *E. coli* MG1655 [[30](#R30)], *E. coli* DH5a (Invitrogen, Carlsbad, CA), *E. coli* AS11 [[31](#R31)], *E. cloacae* ATCC23355 [[32](#R32)], *C. freundii* ATCC8090 [[32](#R32)], and *S. marcescens* ATCC14041 [[33](#R33)].

**Stress Assays and qPCR**

Acid and oxidative stress assays were performed as previously described [[3](#R3), [4](#R4), [7](#R7)]. The qPCR reactions were performed as previously described using 16S rRNA and *lpxC* genes for normalization [[3](#R3), [34](#R34)]. DNA oligonucleotides used as primers in qPCR are provided in Table **[1](#T1)**.

#### Table 1.

DNA oligonucleotides used in this study.

| Name | Sequence |
| --- | --- |
| *S.* Typhimurium | |
| *hfq* | acaagatccgttcctgaacgcattgcgtcg |
| tgttgctgtgatgggaaaccgggcgagacg |
| *trpD* | agcgcctttgtcgcggcggcctgtgga |
| gttgatcagcgggccgagtacgttgaacag |
| *E. coli* | |
| *hfq* | acaagatccgttcctgaacgcactgcgtcg |
| tgttactgtgatgagaaaccgggcgagacg |
| *trpD* | agtgcgtttgtcgccgcagcctgtggg |
| gttaatcaatggccccagcacattgaacag |
| *C. freundii* | |
| *hfq* | acaagatccgttcctgaacgcactgcgtcg |
| tgttactgtgatgagaaaccgggcgagacg |
| *trpD* | agcgcctttgttgcggcggcttgcgga |
| gttaatcaacgggccgagcacgttaaacag |
| *E. cloacae* | |
| *hfq* | acaagatccgttcctgaacgcattgcgtcg |
| tattgctgtgatgagataccggacgggacg |
| *ydcI* | ctgaacgaactggaacaactcacc |
| catcgtattgttcatggtcgcgacctg |
| Normalization | |
| *16S rRNA* | gtaacggctcaccaaggcgacgatccctag |
| cttcgccaccggtattcctccagatctctac |
| *lpxC* | ccgttgagcacctgaatgctgctctggcgg |
| tctggcgcatgaacgcatccgcagagaagt |
| *S.* Typhimurium  recombineering | |
| Δ*trpR* | ctcgtgtaacagtacaaacggcggtataacatgacccagccatatgaatatcctccttagttcc |
| tggccgcttcgccttatccggcctacgagcaaatcaggcgtgtgtaggctggagctgcttc |
| Δ*trpDE* | acccacgctcgaactgttgacctgcgatgccgcctatcgggatgtgtaggctggagctgcttc |
| ccaaatcgccttgtctgcgacgattttcgctaaaacggtttgcatatgaatatcctccttagttcc |
| *E. coli* recombineering | |
| Δ*trpR* | cccgctaacaatggcgacatattatggcccaacaatcacccatatgaatatcctccttagttcc |
| gatgcgccacgtcttatcaggcctacaaaatcaatcgcttgtgtaggctggagctgcttc |
| Δ*trpDE* | acgtaaaagagtcgatattatcgagcagcagaatgtcagccatatgaatatcctccttagttcc |
| aaccgactctcgaactgctaacctgcgaaggcgcttatcgtgtgtaggctggagctgcttc |

[Open in a new tab](table/T1/)

### Mutant Strain Construction

Deletion mutations in the *trpR* and *trpDE* genes were constructed using standard recombineering techniques in *S.* Typhimurium and *E. coli* and confirmed using both PCR and phenotypic indications [[35](#R35)]. DNA oligonucleotides used in the recombineering are provided in Table **[1](#T1)**. Briefly, PCR products were amplified from the template plasmid pKD3 using primers containing 40-bp regions of homology to the ends of the *trpR* and *trpDE* genes such that deletion of the genes would be obtained. Strains containing plasmid pKD46 were electroporated with the PCR products, and colonies were selected on LB-Cm medium.

## RESULTS

### RWV Growth Kinetics

To determine if the individual members of Enterobacteriaceae exhibited similar growth kinetics in both LSMMG and control conditions (an important parameter for comparing the two culture orientations with a given species), we grew the following members of this family in the RWV in the LSMMG and control orientations (Fig. **[2](#F2)**): *S.* Typhimurium , *E. coli*, *Enterobacter cloacae*, *Citrobacter freundii*, and *Serratia marcescens.* Data obtained with *S.* Typhimurium have previously shown that growth in LSMMG and control conditions results in very similar kinetic profiles for this organism in LB medium [[1](#R1), [4](#R4), [7](#R7)]. Analysis of bacterial growth in the LSMMG and control conditions for *E. coli, E. cloacae, C. freundii*, and *S. marcescens* revealed essentially identical kinetics in both environments for each organism (Fig. **[2](#F2)**). This assures that phenotypic assays performed at a given point in the growth of LSMMG and control cultures are using cells at equivalent growth phases.

#### Fig. (2).

[![Fig. (2)](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5b7/4085587/bc2862c95ea2/TOMICROJ-8-51_F2.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4085587_TOMICROJ-8-51_F2.jpg)

[Open in a new tab](figure/F2/)

**Growth of Enterobacteriaceae in the RWV.**
 Panel **A**:
A representative growth curve for Enterobacteriaceae (*Enterobacter cloacae*
ATCC23355) in the RWV for the LSMMG and control conditions in LB media, 37 °C.
The RWV apparatuses were set-up and operated as shown in Fig. (**1**) and as
previously described [1, 3-5, 7]. At the indicated time points, aliquots were removed from the indicated
apparatuses, serial-diluted, and plated for CFU/ml counts in triplicate. This
experiment was also performed for *Escherichia coli* TOP10, *Citrobacter
freudii* ATCC8090, and *Serratia marcsescens* ATCC14041 with equivalent
results (data shown in panel B below).
Panel **B**: Cultures of the indicated bacteria were grown in the RWV
in the LSMMG and control conditions in LB media and samples processed as in panel A. For each time point, the ratio of CFU/ml
LSMMG to control was calculated. The results were obtained from at least two
independent cultures for each strain, and the average and standard deviation are
plotted. For *E. coli*, the data shown here is for strain TOP10. Strains
*E. coli* MG1655, *E. coli* DH5 α,
and *E. coli* AS11 displayed results equivalent to those presented here
(data not shown).

### Acid and Oxidative Stress Phenotypes

Previous studies have demonstrated that resistance to acid and oxidative stress is altered by LSMMG in *S.* Typhimurium [[1](#R1)-[5](#R5), [7](#R7), [33](#R33)]. Therefore, we tested each genus (including *S.* Typhimurium ) for these phenotypes by comparing LSMMG and control cultures for resistance to acid stress (pH=3.5, citric acid) and oxidative stress (hydrogen peroxide, 35 - 70 mM) (Figs. **[3](#F3)** and **[4](#F4)**). It is worth noting that acid stress resistance was tested using the conditions reported in the Wilson, *et al.* (2007) and Wilson, *et al*. (2008) references in which resistance is decreased by LSMMG in *S.* Typhimurium for cultures grown 24 hours which more appropriately matched spaceflight conditions involved in those studies [[3](#R3), [7](#R7)].

#### Fig. (3).

[![Fig. (3)](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5b7/4085587/adfaad3d45b0/TOMICROJ-8-51_F3.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4085587_TOMICROJ-8-51_F3.jpg)

[Open in a new tab](figure/F3/)

**LSMMG-mediated alteration of acid stress resistance in
Enterobacteriaceae.**  The indicated strains were grown in the RWV in the LSMMG and control
conditions for testing of acid stress resistance (pH=3.5, citric
acid, 2 hours stress exposure) as previously described
[3, 7]. To test survival of the stress in the LSMMG and control samples,
aliquots from each sample were removed from the stress,
serially-diluted in PBS, and plated for CFU counts on LB media. The
percent survival of the bacteria compared to the number present at
t=0 before addition of the stress was determined for each sample,
and then a ratio of percent survival values was calculated and
plotted as indicated in the graph (either control/control or
LSMMG/control). The data was obtained from at least three
independent experiments each plated in triplicate, and the average
and standard deviation are plotted. The differences between LSMMG
and control were significant at p-value
< 0.05.

#### Fig. (4).

[![Fig. (4)](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5b7/4085587/6125bb7e93d7/TOMICROJ-8-51_F4.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4085587_TOMICROJ-8-51_F4.jpg)

[Open in a new tab](figure/F4/)

**Oxidative stress resistance altered by LSMMG in Enterobacteriaceae.**
RWV cultures of the indicated strains were tested for oxidative stress resistance
(hydrogen peroxide, 35 - 70 mM, 2 hours stress exposure)
as described in Fig. (**3**) and as previously described
[4, 33] The data was obtained from at least three independent experiments
each plated in triplicate, and the average and standard deviation
are plotted. The differences between LSMMG and control were
significant at p-value < 0.05.

For *S.* Typhimurium , acid stress resistance was decreased in LSMMG as reported previously (Fig. **[3](#F3)**) [[3](#R3), [7](#R7)]. However, under the same conditions, resistance to acid stress was increased by LSMMG for *E. coli*, and this result was repeated in four different *E. coli* strains to ensure that strain differences did not explain the deviation from the *S.* Typhimurium phenotype (Fig. **[3](#F3)**). In *E. cloacae* and *S. marcescens*, LSMMG decreased and increased acid stress resistance, respectively (Fig. **[3](#F3)**). For *C. freundii*, acid stress resistance was not significantly different between LSMMG and control cultures (Fig. **[3](#F3)**). For oxidative stress resistance, LSMMG decreased resistance for all tested genera (Fig. **[4](#F4)**). These results are equivalent to previously published data for *S.* Typhimurium [[4](#R4), [33](#R33)]. For *S. marcescens*, oxidative stress resistance could not be tested due to the fact that this organism could not be killed by the levels of hydrogen peroxide we tested for this study (up to 175 mM for this organism).

### qPCR Assays

To test for changes in gene expression induced by LSMMG in different genera, we isolated total RNA from *S.* Typhimurium , *E. coli*, *E. cloacae*, and *C. freundii* cultures grown at LSMMG and control conditions and analyzed gene expression using qRT-PCR [[3](#R3), [18](#R18)]. In *S.* Typhimurium , the expression of the *hfq* and *trpD* genes was decreased and increased, respectively, by LSMMG (Fig. **[5](#F5)**). This result is consistent with previously published data which identified altered expression of these genes by LSMMG using microarray analysis and qPCR [[3](#R3), [5](#R5), [7](#R7)]. In *E. coli*, LSMMG decreased the expression of both genes indicating that LSMMG altered gene expression in this genus, but that the direction of *trpD* regulation was reversed compared to *S.* Typhimurium (Fig. **[5](#F5)**). This is interesting given the fact that the direction of LSMMG-induced acid resistance alteration was also reversed accordingly in these two genera (Fig. **[3](#F3)**). In *C. freundii*, *hfq* expression was also decreased by LSMMG, but *trpD* expression was not significantly altered (Fig. **[5](#F5)**). As in *E. coli*, the *trpD* result also correlates to the acid resistance phenotype in *C. freundii* where LSMMG did not cause altered resistance to this stress. In *E. cloacae*, we did not observe alteration of *hfq* expression by LSMMG but did observe an increase in expression of the *ydcI* gene by LSMMG (Fig. **[5](#F5)**). The *ydcI* result is consistent with previously reported data which identified this gene as LSMMG-regulated in *S.* Typhimurium [[4](#R4), [5](#R5)]. Overall, qRT-PCR analysis demonstrated that a molecular response to LSMMG resulting in gene expression changes is conserved across Enterobactericeae, but the direction of gene expression change (i.e. up or down) can potentially vary depending on genus.

#### Fig. (5).

[![Fig. (5)](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5b7/4085587/f9ed15f86ead/TOMICROJ-8-51_F5.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4085587_TOMICROJ-8-51_F5.jpg)

[Open in a new tab](figure/F5/)

qRT-PCR analysis to determine LSMMG-induced alterations of gene expression in Enterobacteriaceae. Total RNA harvested from LSMMG and control cultures was converted to single-stranded cDNA and used as template in qPCR analysis with primers hybridizing to the indicated genes as described previously [3, 18]. PCR product levels were normalized to both the 16S rRNA and *lpxC* genes [3, 34], and a ratio of each gene level for LSMMG to control was calculated to give a fold-difference in expression between the two samples as previously described [3]. Differences in expression between LSMMG and control samples were significant at p-value < 0.05. The data was obtained from at least four qPCR reactions using RNA from at least two independent cultures for each condition, and the data is plotted as the average and standard deviation.

### Analysis of Role of *trp* Genes

In the experiments above, we observed an inverse correlation between the direction of change in acid resistance by LSMMG and the direction of regulation of *trpD* expression by LSMMG in *S.* Typhimurium and *E. coli* (Figs. **[3](#F3)**, **[5](#F5)**). In addition,we observed that in *C. freundii*, these phenotypes correlated to eachother in that LSMMG did not induce alteration of acid stress or *trpD* expression in this species (Figs. **[3](#F3)**, **[5](#F5)**). This observation is curious since the *trp* genes are not involved in acid resistance, and the activity of the anthrylate synthase enzyme encoded by the *trp* operon is not involved in environmental stress resistance. However,in *S.* Typhimurium , the *trpD* gene was one of the few genes in a group found to be up-regulated by all low-shear/micro-gravity conditions tested including LSMMG in LB media [[5](#R5)] and by spaceflight in both LB and M9 media [[3](#R3), [7](#R7)], and this result was observed using both microarray and qPCR analysis [[3](#R3), [5](#R5), [7](#R7)]. In addition, a study with adherent-invasive *E. coli* showed that a mutation in a tryptophan permease gene altered an LSMMG-induced phenotype related to cell adherence, and this alteration was complemented by restoring the WT gene or by exogenous indole supplementation [[28](#R28)]. Taken together, the above observations provided justification to analyze the role of *trp* genes in the LSMMG response.We reasoned that the changes in *trpD* expression under LSMMG could be related to a regulatory mechanism acting up-stream of the *trp* operon that is responsive to and/or involved in the LSMMG response pathway. The TrpR protein is a transcriptional regulator that controls expression of the *trp* operon as well as several other genes distributed across the *S.* Typhimurium and *E. coli* genomes [[36](#R36)]. Since this protein regulates a global gene regulon, we viewed TrpR as a possible candidate regulatory mechanism through which the LSMMG response acts in bacteria. We therefore constructed Δ*trpR* mutations in *S.* Typhimurium and *E. coli*, and we tested these mutants for display of LSMMG phenotypes (Fig. **[6](#F6)**). We found that the deletion of TrpR had no effect on the acid and oxidative stress LSMMG responses in these genera indicating that TrpR is likely not involved in the LSMMG response under the conditions tested here (Fig. **[6](#F6)**). In addition, we also considered the possibility that the anthrylate synthase enzyme encoded by the *trp* operon could be having a role in the LSMMG response. Therefore, we constructed a Δ*trpDE* mutation in *S.* Typhimurium and *E. coli* that knocked out expression of this enzyme, and we tested these mutants for LSMMG phenotypes compared to WT (Fig. **[6](#F6)**). As with the Δ*trpR* strains, we observed no effect of Δ*trpDE* on LSMMG responses in these genera indicating that the *trp* operon genes are not involved the LSMMG response as tested here.

#### Fig. (6).

[![Fig. (6)](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e5b7/4085587/4260750be5c5/TOMICROJ-8-51_F6.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4085587_TOMICROJ-8-51_F6.jpg)

[Open in a new tab](figure/F6/)

**Analysis of role of *trp* genes in LSMMG phenotypes**. *S.* Typhimurium  and *E.coli*
Δ *trpR* and Δ *trpDE* strains were compared to respective WT strains for LSMMG-mediated alteration of
acid and oxidative stress as described in Figs. (**3**, **4**).
The data was obtained from at least three independent experiments each plated in
triplicate, and the average and standard deviation are plotted. The differences
between LSMMG and control were significant at p-value < 0.05.

## DISCUSSION

The results presented here provide significant evidence using several strains and multiple assays that the response to the LSMMG growth environment is conserved across Enterobacteriaceae.Thus, the pathways and mechanisms used for sensing this environmental signal appear to be present in these different cells. This report also demonstrates that in a “side-by-side” study, the direction of LSMMG-induced regulation of phenotypes and gene expression can vary depending on genus. These results indicate that applications using the LSMMG environment (and related spaceflight conditions) have the potential to be targeted to a range of Entrobacteriaceae genera. Future work will focus on understanding the underlying causes of the variance in the direction of LSMMG regulation observed here and on the use of LSMMG/spaceflight to engineer bacteria in different ways.

## ACKNOWLEDGMENTS

We thank the Villanova University Biology Department, Sigma Xi Grants-in-Aid G20111015158058, and NASA grant NNX09AH40G for support of this work. We thank members of the Villanova Biology Department for discussions and assistance during this work.

## CONFLICT OF INTEREST

The authors confirm that this article content has no conflict of interest.

## REFERENCES

* 1.Nickerson CA, Ott CM, Mister SJ, Morrow BJ, Burns-Keliher L, Pierson DL. Microgravity as a novel environmental signal affecting Salmonella enterica serovar Typhimurium virulence. Infect Immun. 2000;68(6):3147–52. doi: 10.1128/iai.68.6.3147-3152.2000. [[DOI](https://doi.org/10.1128/iai.68.6.3147-3152.2000)] [[PMC free article](/articles/PMC97548/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10816456/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&title=Microgravity%20as%20a%20novel%20environmental%20signal%20affecting%20Salmonella%20enterica%20serovar%20Typhimurium%20virulence.&author=CA%20Nickerson&author=CM%20Ott&author=SJ%20Mister&author=BJ%20Morrow&author=L%20Burns-Keliher&volume=68&issue=6&publication_year=2000&pages=3147-52&pmid=10816456&doi=10.1128/iai.68.6.3147-3152.2000&)]
* 2.Nauman EA, Ott CM, Sander E , et al. Novel quantitative biosystem for modeling physiological fluid shear stress on cells. Appl Environ Microbiol. 2007;73(3):699–705. doi: 10.1128/AEM.02428-06. [[DOI](https://doi.org/10.1128/AEM.02428-06)] [[PMC free article](/articles/PMC1800738/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17142365/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Novel%20quantitative%20biosystem%20for%20modeling%20physiological%20fluid%20shear%20stress%20on%20cells.&author=EA%20Nauman&author=CM%20Ott&author=E%20Sander&volume=73&issue=3&publication_year=2007&pages=699-705&pmid=17142365&doi=10.1128/AEM.02428-06&)]
* 3.Wilson JW, Ott CM, Quick L , et al. Media ion composition controls regulatory and virulence response of Salmonella in spaceflight. PloS One. 2008;3(12):e3923. doi: 10.1371/journal.pone.0003923. [[DOI](https://doi.org/10.1371/journal.pone.0003923)] [[PMC free article](/articles/PMC2592540/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19079590/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Media%20ion%20composition%20controls%20regulatory%20and%20virulence%20response%20of%20Salmonella%20in%20spaceflight.&author=JW%20Wilson&author=CM%20Ott&author=L%20Quick&volume=3&issue=12&publication_year=2008&pages=e3923&pmid=19079590&doi=10.1371/journal.pone.0003923&)]
* 4.Wilson JW, Ott CM, Ramamurthy R , et al. Low-Shear modeled microgravity alters the Salmonella enterica serovar Typhimurium stress response in an RpoS-independent manner. Appl Environ Microbiol. 2002;68(11):5408–16. doi: 10.1128/AEM.68.11.5408-5416.2002. [[DOI](https://doi.org/10.1128/AEM.68.11.5408-5416.2002)] [[PMC free article](/articles/PMC129924/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12406731/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Low-Shear%20modeled%20microgravity%20alters%20the%20Salmonella%20enterica%20serovar%20Typhimurium%20stress%20response%20in%20an%20RpoS-independent%20manner.&author=JW%20Wilson&author=CM%20Ott&author=R%20Ramamurthy&volume=68&issue=11&publication_year=2002&pages=5408-16&pmid=12406731&doi=10.1128/AEM.68.11.5408-5416.2002&)]
* 5.Wilson JW, Ramamurthy R, Porwollik S , et al. Microarray analysis identifies Salmonella genes belonging to the low-shear modeled microgravity regulon. Proc Natl Acad Sci USA. 2002;99(21):13807–12. doi: 10.1073/pnas.212387899. [[DOI](https://doi.org/10.1073/pnas.212387899)] [[PMC free article](/articles/PMC129779/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12370447/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Microarray%20analysis%20identifies%20Salmonella%20genes%20belonging%20to%20the%20low-shear%20modeled%20microgravity%20regulon.&author=JW%20Wilson&author=R%20Ramamurthy&author=S%20Porwollik&volume=99&issue=21&publication_year=2002&pages=13807-12&pmid=12370447&doi=10.1073/pnas.212387899&)]
* 6.Pacello F, Rotilio G, Battistoni A. Low-Shear modeled microgravity enhances Salmonella Enterica Resistance to hydrogen peroxide through a mechanism involving KatG and KatN. Open Microbiol J. 2012;6:53–64. doi: 10.2174/1874285801206010053. [[DOI](https://doi.org/10.2174/1874285801206010053)] [[PMC free article](/articles/PMC3414715/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22888375/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Open%20Microbiol%20J&title=Low-Shear%20modeled%20microgravity%20enhances%20Salmonella%20Enterica%20Resistance%20to%20hydrogen%20peroxide%20through%20a%20mechanism%20involving%20KatG%20and%20KatN.&author=F%20Pacello&author=G%20Rotilio&author=A%20Battistoni&volume=6&publication_year=2012&pages=53-64&pmid=22888375&doi=10.2174/1874285801206010053&)]
* 7.Wilson JW, Ott CM, Honer zu Bentrup K , et al. Space flight alters bacterial gene expression and virulence and reveals a role for global regulator Hfq. Proc Natl Acad Sci USA. 2007;104(41):16299–304. doi: 10.1073/pnas.0707155104. [[DOI](https://doi.org/10.1073/pnas.0707155104)] [[PMC free article](/articles/PMC2042201/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17901201/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Space%20flight%20alters%20bacterial%20gene%20expression%20and%20virulence%20and%20reveals%20a%20role%20for%20global%20regulator%20Hfq.&author=JW%20Wilson&author=CM%20Ott&author=K%20Honer%20zu%20Bentrup&volume=104&issue=41&publication_year=2007&pages=16299-304&pmid=17901201&doi=10.1073/pnas.0707155104&)]
* 8.Fang A, Pierson DL, Koenig DW, Mishra SK, Demain AL. Effect of simulated microgravity and shear stress on microcin B17 production by Escherichia coli and on its excretion into the medium. Appl Environ Microbiol. 1997;63(10):4090–2. doi: 10.1128/aem.63.10.4090-4092.1997. [[DOI](https://doi.org/10.1128/aem.63.10.4090-4092.1997)] [[PMC free article](/articles/PMC168721/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/9327574/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Effect%20of%20simulated%20microgravity%20and%20shear%20stress%20on%20microcin%20B17%20production%20by%20Escherichia%20coli%20and%20on%20its%20excretion%20into%20the%20medium.&author=A%20Fang&author=DL%20Pierson&author=DW%20Koenig&author=SK%20Mishra&author=AL%20Demain&volume=63&issue=10&publication_year=1997&pages=4090-2&pmid=9327574&doi=10.1128/aem.63.10.4090-4092.1997&)]
* 9.Fang A, Pierson DL, Mishra SK, Demain AL. Relief from glucose interference in microcin B17 biosynthesis by growth in a rotating-wall bioreactor. Lett Appl Microbiol. 2000;31(1):39–41. [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10886612/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lett%20Appl%20Microbiol&title=Relief%20from%20glucose%20interference%20in%20microcin%20B17%20biosynthesis%20by%20growth%20in%20a%20rotating-wall%20bioreactor.&author=A%20Fang&author=DL%20Pierson&author=SK%20Mishra&author=AL%20Demain&volume=31&issue=1&publication_year=2000&pages=39-41&pmid=10886612&)]
* 10.Gao Q, Fang A, Pierson DL, Mishra SK, Demain AL. Shear stress enhances microcin B17 production in a rotating wall bioreactor. but ethanol stress does not. . Appl Microbiol Biotechnol. 2001;56(3-4):384–7. doi: 10.1007/s002530100610. [[DOI](https://doi.org/10.1007/s002530100610)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11549006/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&title=Shear%20stress%20enhances%20microcin%20B17%20production%20in%20a%20rotating%20wall%20bioreactor.%20but%20ethanol%20stress%20does%20not.&author=Q%20Gao&author=A%20Fang&author=DL%20Pierson&author=SK%20Mishra&author=AL%20Demain&volume=56&issue=3-4&publication_year=2001&pages=384-7&pmid=11549006&doi=10.1007/s002530100610&)]
* 11.Lynch SV, Brodie EL, Matin A. Role and regulation of sigma S in general resistance conferred by low-shear simulated microgravity in Escherichia coli. J Bacteriol. 2004;186(24):8207–12. doi: 10.1128/JB.186.24.8207-8212.2004. [[DOI](https://doi.org/10.1128/JB.186.24.8207-8212.2004)] [[PMC free article](/articles/PMC532419/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15576768/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&title=Role%20and%20regulation%20of%20sigma%20S%20in%20general%20resistance%20conferred%20by%20low-shear%20simulated%20microgravity%20in%20Escherichia%20coli.&author=SV%20Lynch&author=EL%20Brodie&author=A%20Matin&volume=186&issue=24&publication_year=2004&pages=8207-12&pmid=15576768&doi=10.1128/JB.186.24.8207-8212.2004&)]
* 12.Lynch SV, Mukundakrishnan K, Benoit MR, Ayyaswamy PS, Matin A. Escherichia coli biofilms formed under low-shear modeled microgravity in a ground-based system. Appl Environ Microbiol. 2006;72(12):7701–10. doi: 10.1128/AEM.01294-06. [[DOI](https://doi.org/10.1128/AEM.01294-06)] [[PMC free article](/articles/PMC1694224/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17028231/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Escherichia%20coli%20biofilms%20formed%20under%20low-shear%20modeled%20microgravity%20in%20a%20ground-based%20system.&author=SV%20Lynch&author=K%20Mukundakrishnan&author=MR%20Benoit&author=PS%20Ayyaswamy&author=A%20Matin&volume=72&issue=12&publication_year=2006&pages=7701-10&pmid=17028231&doi=10.1128/AEM.01294-06&)]
* 13.Tucker DL, Ott CM, Huff S , et al. Characterization of Escherichia coli MG1655 grown in a low-shear modeled microgravity environment. BMC Microbiol. 2007;7:15. doi: 10.1186/1471-2180-7-15. [[DOI](https://doi.org/10.1186/1471-2180-7-15)] [[PMC free article](/articles/PMC1852313/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17343762/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&title=Characterization%20of%20Escherichia%20coli%20MG1655%20grown%20in%20a%20low-shear%20modeled%20microgravity%20environment.&author=DL%20Tucker&author=CM%20Ott&author=S%20Huff&volume=7&publication_year=2007&pages=15&pmid=17343762&doi=10.1186/1471-2180-7-15&)]
* 14.Arunasri K, Adil M, Venu Charan K, Suvro C, Himabindu Reddy S, Shivaji S. Effect of simulated microgravity on E.coli K12 MG1655 growth and gene expression. PloS One. 2013;8(3):e57860. doi: 10.1371/journal.pone.0057860. [[DOI](https://doi.org/10.1371/journal.pone.0057860)] [[PMC free article](/articles/PMC3589462/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23472115/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Effect%20of%20simulated%20microgravity%20on%20E.coli%20K12%20MG1655%20growth%20and%20gene%20expression.&author=K%20Arunasri&author=M%20Adil&author=K%20Venu%20Charan&author=C%20Suvro&author=S%20Himabindu%20Reddy&volume=8&issue=3&publication_year=2013&pages=e57860&pmid=23472115&doi=10.1371/journal.pone.0057860&)]
* 15.Vukanti R, Model MA, Leff LG. Effect of modeled reduced gravity conditions on bacterial morphology and physiology. BMC Microbiol. 2012;12:4. doi: 10.1186/1471-2180-12-4. [[DOI](https://doi.org/10.1186/1471-2180-12-4)] [[PMC free article](/articles/PMC3274431/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22239851/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&title=Effect%20of%20modeled%20reduced%20gravity%20conditions%20on%20bacterial%20morphology%20and%20physiology.&author=R%20Vukanti&author=MA%20Model&author=LG%20Leff&volume=12&publication_year=2012&pages=4&pmid=22239851&doi=10.1186/1471-2180-12-4&)]
* 16.Crabbe A, Pycke B, Van Houdt R , et al. Response of Pseudomonas aeruginosa PAO1 to low shear modelled microgravity involves AlgU regulation. Environ Microbiol. 2010;12(6):1545–64. doi: 10.1111/j.1462-2920.2010.02184.x. [[DOI](https://doi.org/10.1111/j.1462-2920.2010.02184.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20236169/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&title=Response%20of%20Pseudomonas%20aeruginosa%20PAO1%20to%20low%20shear%20modelled%20microgravity%20involves%20AlgU%20regulation.&author=A%20Crabbe&author=B%20Pycke&author=R%20Van%20Houdt&volume=12&issue=6&publication_year=2010&pages=1545-64&pmid=20236169&doi=10.1111/j.1462-2920.2010.02184.x&)]
* 17.Crabbe A, Schurr MJ, Monsieurs P , et al. Transcriptional and proteomic responses of Pseudomonas aeruginosa PAO1 to spaceflight conditions involve Hfq regulation and reveal a role for oxygen. Appl Environ Microbiol. 2011;77(4):1221–30. doi: 10.1128/AEM.01582-10. [[DOI](https://doi.org/10.1128/AEM.01582-10)] [[PMC free article](/articles/PMC3067220/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21169425/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Transcriptional%20and%20proteomic%20responses%20of%20Pseudomonas%20aeruginosa%20PAO1%20to%20spaceflight%20conditions%20involve%20Hfq%20regulation%20and%20reveal%20a%20role%20for%20oxygen.&author=A%20Crabbe&author=MJ%20Schurr&author=P%20Monsieurs&volume=77&issue=4&publication_year=2011&pages=1221-30&pmid=21169425&doi=10.1128/AEM.01582-10&)]
* 18.Crabbe A, De Boever P, Van Houdt R, Moors H, Mergeay M, Cornelis P. Use of the rotating wall vessel technology to study the effect of shear stress on growth behaviour of Pseudomonas aeruginosa PA01. Environ Microbiol. 2008;10(8):2098–110. doi: 10.1111/j.1462-2920.2008.01631.x. [[DOI](https://doi.org/10.1111/j.1462-2920.2008.01631.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18430020/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&title=Use%20of%20the%20rotating%20wall%20vessel%20technology%20to%20study%20the%20effect%20of%20shear%20stress%20on%20growth%20behaviour%20of%20Pseudomonas%20aeruginosa%20PA01.&author=A%20Crabbe&author=P%20De%20Boever&author=R%20Van%20Houdt&author=H%20Moors&author=M%20Mergeay&volume=10&issue=8&publication_year=2008&pages=2098-110&pmid=18430020&doi=10.1111/j.1462-2920.2008.01631.x&)]
* 19.Lawal A, Jejelowo OA, Rosenzweig JA. The effects of low-shear mechanical stress on Yersinia pestis virulence. Astrobiology. 2010;10(9):881–8. doi: 10.1089/ast.2010.0493. [[DOI](https://doi.org/10.1089/ast.2010.0493)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21118021/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=The%20effects%20of%20low-shear%20mechanical%20stress%20on%20Yersinia%20pestis%20virulence.&author=A%20Lawal&author=OA%20Jejelowo&author=JA%20Rosenzweig&volume=10&issue=9&publication_year=2010&pages=881-8&pmid=21118021&doi=10.1089/ast.2010.0493&)]
* 20.Lawal A, Kirtley ML, van Lier CJ , et al. The effects of modeled microgravity on growth kinetics. antibiotic susceptibiity.cold growth., and the virulence potential of a Yersinia pestisymo-
  a deficient mutant and its isogenic parental strain. . Astrobiology. 2013;13(9):821–32. doi: 10.1089/ast.2013.0968. [[DOI](https://doi.org/10.1089/ast.2013.0968)] [[PMC free article](/articles/PMC3779001/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23988036/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=The%20effects%20of%20modeled%20microgravity%20on%20growth%20kinetics.%20antibiotic%20susceptibiity.cold%20growth.,%20and%20the%20virulence%20potential%20of%20a%20Yersinia%20pestisymo-%20a%20deficient%20mutant%20and%20its%20isogenic%20parental%20strain.&author=A%20Lawal&author=ML%20Kirtley&author=CJ%20van%20Lier&volume=13&issue=9&publication_year=2013&pages=821-32&pmid=23988036&doi=10.1089/ast.2013.0968&)]
* 21.Castro SL, Nelman-Gonzalez M, Nickerson CA, Ott CM. Induction of attachment-independent biofilm formation and repression of Hfq expression by low-fluid-shear culture of Staphylococcus aureus. Appl Environ Microbiol. 2011;77(18):6368–78. doi: 10.1128/AEM.00175-11. [[DOI](https://doi.org/10.1128/AEM.00175-11)] [[PMC free article](/articles/PMC3187170/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21803898/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Induction%20of%20attachment-independent%20biofilm%20formation%20and%20repression%20of%20Hfq%20expression%20by%20low-fluid-shear%20culture%20of%20Staphylococcus%20aureus.&author=SL%20Castro&author=M%20Nelman-Gonzalez&author=CA%20Nickerson&author=CM%20Ott&volume=77&issue=18&publication_year=2011&pages=6368-78&pmid=21803898&doi=10.1128/AEM.00175-11&)]
* 22.Rosado H, Doyle M, Hinds J, Taylor P. Low-shear modelled microgravity alters expression of virulence determinants of Staphylococcus aureus. Acta Astronautica. 2010;66(3-4):408–13. [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Astronautica&title=Low-shear%20modelled%20microgravity%20alters%20expression%20of%20virulence%20determinants%20of%20Staphylococcus%20aureus.&author=H%20Rosado&author=M%20Doyle&author=J%20Hinds&author=P%20Taylor&volume=66&issue=3-4&publication_year=2010&pages=408-13&)]
* 23.Guo P, Weinstein AM, Weinbaum S. A hydrodynamic mechanosensory hypothesis for brush border microvilli. Am J Physiol Renal Physiol. 2000;279(4):F698–712. doi: 10.1152/ajprenal.2000.279.4.F698. [[DOI](https://doi.org/10.1152/ajprenal.2000.279.4.F698)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10997920/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Renal%20Physiol&title=A%20hydrodynamic%20mechanosensory%20hypothesis%20for%20brush%20border%20microvilli.&author=P%20Guo&author=AM%20Weinstein&author=S%20Weinbaum&volume=279&issue=4&publication_year=2000&pages=F698-712&pmid=10997920&doi=10.1152/ajprenal.2000.279.4.F698&)]
* 24.Barrila J, Radtke AL, Crabbe A , et al. Organotypic 3D cell culture models: using the rotating wall vessel to study host-pathogen interactions. Nature Rev Microbiol. 2010;8(11):791–801. doi: 10.1038/nrmicro2423. [[DOI](https://doi.org/10.1038/nrmicro2423)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20948552/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Rev%20Microbiol&title=Organotypic%203D%20cell%20culture%20models:%20using%20the%20rotating%20wall%20vessel%20to%20study%20host-pathogen%20interactions.&author=J%20Barrila&author=AL%20Radtke&author=A%20Crabbe&volume=8&issue=11&publication_year=2010&pages=791-801&pmid=20948552&doi=10.1038/nrmicro2423&)]
* 25.Nickerson CA, Ott CM, Wilson JW , et al. Low-shear modeled microgravity: a global environmental regulatory signal affecting bacterial gene expression. physioogy.and pathogenesis. . J Microbiol Methods. 2003;54(1):1–11. doi: 10.1016/s0167-7012(03)00018-6. [[DOI](https://doi.org/10.1016/s0167-7012%2803%2900018-6)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12732416/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Methods&title=Low-shear%20modeled%20microgravity:%20a%20global%20environmental%20regulatory%20signal%20affecting%20bacterial%20gene%20expression.%20physioogy.and%20pathogenesis.&author=CA%20Nickerson&author=CM%20Ott&author=JW%20Wilson&volume=54&issue=1&publication_year=2003&pages=1-11&pmid=12732416&doi=10.1016/s0167-7012(03)00018-6&)]
* 26.Nickerson CA, Ott CM, Wilson JW, Ramamurthy R, Pierson DL. Microbial responses to microgravity and other low-shear environments. Microbiol Mol Biol Rev. 2004;68(2):345–61. doi: 10.1128/MMBR.68.2.345-361.2004. [[DOI](https://doi.org/10.1128/MMBR.68.2.345-361.2004)] [[PMC free article](/articles/PMC419922/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15187188/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol%20Mol%20Biol%20Rev&title=Microbial%20responses%20to%20microgravity%20and%20other%20low-shear%20environments.&author=CA%20Nickerson&author=CM%20Ott&author=JW%20Wilson&author=R%20Ramamurthy&author=DL%20Pierson&volume=68&issue=2&publication_year=2004&pages=345-61&pmid=15187188&doi=10.1128/MMBR.68.2.345-361.2004&)]
* 27.Rosenzweig JA, Chopra AK. The effect of low shear force on the virulence potential of Yersinia pestis: new aspects that space-like growth conditions and the final frontier can teach us about a formidable pathogen. Front Cell Infect Microbiol. 2012;2:107. doi: 10.3389/fcimb.2012.00107. [[DOI](https://doi.org/10.3389/fcimb.2012.00107)] [[PMC free article](/articles/PMC3417468/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22919696/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Infect%20Microbiol&title=The%20effect%20of%20low%20shear%20force%20on%20the%20virulence%20potential%20of%20Yersinia%20pestis:%20new%20aspects%20that%20space-like%20growth%20conditions%20and%20the%20final%20frontier%20can%20teach%20us%20about%20a%20formidable%20pathogen.&author=JA%20Rosenzweig&author=AK%20Chopra&volume=2&publication_year=2012&pages=107&pmid=22919696&doi=10.3389/fcimb.2012.00107&)]
* 28.Allen CA, Niesel DW, Torres AG. The effects of low-shear stress on adherent-invasive Escherichia coli. Environ Microbiol. 2008;10(6):1512–25. doi: 10.1111/j.1462-2920.2008.01567.x. [[DOI](https://doi.org/10.1111/j.1462-2920.2008.01567.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18312396/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&title=The%20effects%20of%20low-shear%20stress%20on%20adherent-invasive%20Escherichia%20coli.&author=CA%20Allen&author=DW%20Niesel&author=AG%20Torres&volume=10&issue=6&publication_year=2008&pages=1512-25&pmid=18312396&doi=10.1111/j.1462-2920.2008.01567.x&)]
* 29.Gulig PA, Curtiss R 3rd. Plasmid-associated virulence of Salmonella Typhimurium. Infect Immun. 1987;55(12):2891–901. doi: 10.1128/iai.55.12.2891-2901.1987. [[DOI](https://doi.org/10.1128/iai.55.12.2891-2901.1987)] [[PMC free article](/articles/PMC260003/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/3316027/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect%20Immun&title=Plasmid-associated%20virulence%20of%20Salmonella%20Typhimurium.&author=PA%20Gulig&author=R%203rd%20Curtiss&volume=55&issue=12&publication_year=1987&pages=2891-901&pmid=3316027&doi=10.1128/iai.55.12.2891-2901.1987&)]
* 30.Blattner FR, Plunkett G3rd, Bloch CA , et al. The complete genome sequence of Escherichia coli K-12. Science. 1997;277(5331):1453–62. doi: 10.1126/science.277.5331.1453. [[DOI](https://doi.org/10.1126/science.277.5331.1453)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/9278503/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20complete%20genome%20sequence%20of%20Escherichia%20coli%20K-12.&author=FR%20Blattner&author=G3rd%20Plunkett&author=CA%20Bloch&volume=277&issue=5331&publication_year=1997&pages=1453-62&pmid=9278503&doi=10.1126/science.277.5331.1453&)]
* 31.Ayres EK, Thomson VJ, Merino G, Balderes D, Figurski DH. Precise deletions in large bacterial genomes by vector-mediated excision (VEX).The trfA gene of promiscuous plasmid RK2 is essential for replication in several gram-negative hosts. J Mol Biol. 1993;230(1):174–85. doi: 10.1006/jmbi.1993.1134. [[DOI](https://doi.org/10.1006/jmbi.1993.1134)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/8450534/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&title=Precise%20deletions%20in%20large%20bacterial%20genomes%20by%20vector-mediated%20excision%20(VEX).The%20trfA%20gene%20of%20promiscuous%20plasmid%20RK2%20is%20essential%20for%20replication%20in%20several%20gram-negative%20hosts.&author=EK%20Ayres&author=VJ%20Thomson&author=G%20Merino&author=D%20Balderes&author=DH%20Figurski&volume=230&issue=1&publication_year=1993&pages=174-85&pmid=8450534&doi=10.1006/jmbi.1993.1134&)]
* 32.O'Sullivan LE, Nickerson CA, Wilson JW. A series of IncQ-based reporter plasmids for use in a range of Gram negative genera.
  J Microbiol Biotechnol. 2010;20(5):871–4. [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20519909/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Biotechnol&title=A%20series%20of%20IncQ-based%20reporter%20plasmids%20for%20use%20in%20a%20range%20of%20Gram%20negative%20genera.&author=LE%20O'Sullivan&author=CA%20Nickerson&author=JW%20Wilson&volume=20&issue=5&publication_year=2010&pages=871-4&pmid=20519909&)]
* 33.Jennings ME, Quick LN, Soni A , et al. Characterization of the Salmonella enterica serovar Typhimurium ydcI gene. which encodes a conserved DNA binding protein required for full acid stress resistance. J Bacteriol. 2011;193(9):2208–17. doi: 10.1128/JB.01335-10. [[DOI](https://doi.org/10.1128/JB.01335-10)] [[PMC free article](/articles/PMC3133089/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21398541/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&title=Characterization%20of%20the%20Salmonella%20enterica%20serovar%20Typhimurium%20ydcI%20gene.%20which%20encodes%20a%20conserved%20DNA%20binding%20protein%20required%20for%20full%20acid%20stress%20resistance.&author=ME%20Jennings&author=LN%20Quick&author=A%20Soni&volume=193&issue=9&publication_year=2011&pages=2208-17&pmid=21398541&doi=10.1128/JB.01335-10&)]
* 34.Hommais F, Zghidi-Abouzid O, Oger-Desfeux C , et al. lpxC and yafS are the most suitable internal controls to normalize real time RT-qPCR expression in the phytopathogenic bacteria Dickeya dadantii. PloS One. 2011;6(5):e20269. doi: 10.1371/journal.pone.0020269. [[DOI](https://doi.org/10.1371/journal.pone.0020269)] [[PMC free article](/articles/PMC3102694/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21637857/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=lpxC%20and%20yafS%20are%20the%20most%20suitable%20internal%20controls%20to%20normalize%20real%20time%20RT-qPCR%20expression%20in%20the%20phytopathogenic%20bacteria%20Dickeya%20dadantii.&author=F%20Hommais&author=O%20Zghidi-Abouzid&author=C%20Oger-Desfeux&volume=6&issue=5&publication_year=2011&pages=e20269&pmid=21637857&doi=10.1371/journal.pone.0020269&)]
* 35.Datsenko KA, Wanner BL. One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proc Natl Acad Sci USA. 2000;97(12):6640–5. doi: 10.1073/pnas.120163297. [[DOI](https://doi.org/10.1073/pnas.120163297)] [[PMC free article](/articles/PMC18686/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10829079/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=One-step%20inactivation%20of%20chromosomal%20genes%20in%20Escherichia%20coli%20K-12%20using%20PCR%20products.&author=KA%20Datsenko&author=BL%20Wanner&volume=97&issue=12&publication_year=2000&pages=6640-5&pmid=10829079&doi=10.1073/pnas.120163297&)]
* 36.Cho BK, Federowicz S, Park YS, Zengler K, Palsson BO. Deciphering the transcriptional regulatory logic of amino acid metabolism. Nat Chem Biol. 2012;8(1):65–71. doi: 10.1038/nchembio.710. [[DOI](https://doi.org/10.1038/nchembio.710)] [[PMC free article](/articles/PMC3777760/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22082910/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Chem%20Biol&title=Deciphering%20the%20transcriptional%20regulatory%20logic%20of%20amino%20acid%20metabolism.&author=BK%20Cho&author=S%20Federowicz&author=YS%20Park&author=K%20Zengler&author=BO%20Palsson&volume=8&issue=1&publication_year=2012&pages=65-71&pmid=22082910&doi=10.1038/nchembio.710&)]