Appl Microbiol Biotechnol

. Author manuscript; available in PMC: 2022 May 12.

*Published in final edited form as:* Appl Microbiol Biotechnol. 2018 Dec 12;103(3):1363–1377. doi: [10.1007/s00253-018-9525-0](https://doi.org/10.1007/s00253-018-9525-0)

# International Space Station conditions alter genomics, proteomics, and metabolomics in *Aspergillus nidulans*

[Jillian Romsdahl](https://pubmed.ncbi.nlm.nih.gov/?term=%22Romsdahl%20J%22%5BAuthor%5D)

### Jillian Romsdahl

1Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, 1985 Zonal Avenue, Los Angeles, CA 90089, USA.

Find articles by [Jillian Romsdahl](https://pubmed.ncbi.nlm.nih.gov/?term=%22Romsdahl%20J%22%5BAuthor%5D)

1, [Adriana Blachowicz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Blachowicz%20A%22%5BAuthor%5D)

### Adriana Blachowicz

1Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, 1985 Zonal Avenue, Los Angeles, CA 90089, USA.

2Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, 4800 Oak Grove Drive, Pasadena, CA 91101, USA.

Find articles by [Adriana Blachowicz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Blachowicz%20A%22%5BAuthor%5D)

1,2, [Abby J Chiang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Chiang%20AJ%22%5BAuthor%5D)

### Abby J Chiang

3Department of Molecular Imaging and Therapy, Beckman Research Institute of City of Hope, 1500 East Duarte Road, Duarte, CA 91010, USA.

Find articles by [Abby J Chiang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Chiang%20AJ%22%5BAuthor%5D)

3, [Yi-Ming Chiang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Chiang%20YM%22%5BAuthor%5D)

### Yi-Ming Chiang

1Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, 1985 Zonal Avenue, Los Angeles, CA 90089, USA.

Find articles by [Yi-Ming Chiang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Chiang%20YM%22%5BAuthor%5D)

1, [Sawyer Masonjones](https://pubmed.ncbi.nlm.nih.gov/?term=%22Masonjones%20S%22%5BAuthor%5D)

### Sawyer Masonjones

4Department of Microbiology & Plant Pathology and Institute of Integrative Genome Biology, University of California-Riverside, 900 University Avenue, Riverside, CA 92521, USA.

Find articles by [Sawyer Masonjones](https://pubmed.ncbi.nlm.nih.gov/?term=%22Masonjones%20S%22%5BAuthor%5D)

4, [Junko Yaegashi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Yaegashi%20J%22%5BAuthor%5D)

### Junko Yaegashi

1Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, 1985 Zonal Avenue, Los Angeles, CA 90089, USA.

Find articles by [Junko Yaegashi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Yaegashi%20J%22%5BAuthor%5D)

1, [Stefanie Countryman](https://pubmed.ncbi.nlm.nih.gov/?term=%22Countryman%20S%22%5BAuthor%5D)

### Stefanie Countryman

5BioServe Space Technologies, University of Colorado, 429 UCB, Boulder, CO 80309, USA.

Find articles by [Stefanie Countryman](https://pubmed.ncbi.nlm.nih.gov/?term=%22Countryman%20S%22%5BAuthor%5D)

5, [Fathi Karouia](https://pubmed.ncbi.nlm.nih.gov/?term=%22Karouia%20F%22%5BAuthor%5D)

### Fathi Karouia

6Space Biosciences Division, NASA Ames Research Center, MS-239-4, Moffett Field, Mountain View, CA 94035, USA.

7Department of Pharmaceutical Chemistry, School of Pharmacy, University of California San Francisco, 600 16th Street, San Francisco, CA 94143, USA.

Find articles by [Fathi Karouia](https://pubmed.ncbi.nlm.nih.gov/?term=%22Karouia%20F%22%5BAuthor%5D)

6,7, [Markus Kalkum](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kalkum%20M%22%5BAuthor%5D)

### Markus Kalkum

3Department of Molecular Imaging and Therapy, Beckman Research Institute of City of Hope, 1500 East Duarte Road, Duarte, CA 91010, USA.

Find articles by [Markus Kalkum](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kalkum%20M%22%5BAuthor%5D)

3, [Jason E Stajich](https://pubmed.ncbi.nlm.nih.gov/?term=%22Stajich%20JE%22%5BAuthor%5D)

### Jason E Stajich

4Department of Microbiology & Plant Pathology and Institute of Integrative Genome Biology, University of California-Riverside, 900 University Avenue, Riverside, CA 92521, USA.

Find articles by [Jason E Stajich](https://pubmed.ncbi.nlm.nih.gov/?term=%22Stajich%20JE%22%5BAuthor%5D)

4, [Kasthuri Venkateswaran](https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D)

### Kasthuri Venkateswaran

2Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, 4800 Oak Grove Drive, Pasadena, CA 91101, USA.

Find articles by [Kasthuri Venkateswaran](https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D)

2, [Clay CC Wang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CCC%22%5BAuthor%5D)

### Clay CC Wang

1Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, 1985 Zonal Avenue, Los Angeles, CA 90089, USA.

8Department of Chemistry, College of Letters, Arts, and Sciences, University of Southern California, 3551 Trousdale Pkwy, Los Angeles, CA 90089, USA.

Find articles by [Clay CC Wang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CCC%22%5BAuthor%5D)

1,8

* Author information
* Article notes
* Copyright and License information

1Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, 1985 Zonal Avenue, Los Angeles, CA 90089, USA.

2Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, 4800 Oak Grove Drive, Pasadena, CA 91101, USA.

3Department of Molecular Imaging and Therapy, Beckman Research Institute of City of Hope, 1500 East Duarte Road, Duarte, CA 91010, USA.

4Department of Microbiology & Plant Pathology and Institute of Integrative Genome Biology, University of California-Riverside, 900 University Avenue, Riverside, CA 92521, USA.

5BioServe Space Technologies, University of Colorado, 429 UCB, Boulder, CO 80309, USA.

6Space Biosciences Division, NASA Ames Research Center, MS-239-4, Moffett Field, Mountain View, CA 94035, USA.

7Department of Pharmaceutical Chemistry, School of Pharmacy, University of California San Francisco, 600 16th Street, San Francisco, CA 94143, USA.

8Department of Chemistry, College of Letters, Arts, and Sciences, University of Southern California, 3551 Trousdale Pkwy, Los Angeles, CA 90089, USA.

✉

**Corresponding author**, Clay C.C. Wang, School of Pharmacy, University of Southern California, 1985 Zonal Ave Rm 406A, Los Angeles, CA, 90089-9121, Phone: (323) 442-1670, clayw@usc.edu

Issue date 2019 Feb.

[PMC Copyright notice](/about/copyright/)

PMCID: PMC9098231  NIHMSID: NIHMS1065202  PMID: [30539259](https://pubmed.ncbi.nlm.nih.gov/30539259/)

The publisher's version of this article is available at [Appl Microbiol Biotechnol](https://doi.org/10.1007/s00253-018-9525-0)

## Abstract

The first global genomic, proteomic, and secondary metabolomic characterization of the filamentous fungus *Aspergillus nidulans* following growth onboard the International Space Station (ISS) is reported. The investigation included the *A. nidulans* wild-type and 3 mutant strains, two of which were genetically engineered to enhance secondary metabolite production. Whole genome sequencing revealed that ISS conditions altered the *A. nidulans* genome in specific regions. In strain CW12001, which features overexpression of the secondary metabolite global regulator *laeA*, ISS conditions induced the loss of the *laeA* stop codon. Differential expression of proteins involved in stress response, carbohydrate metabolic processes, and secondary metabolite biosynthesis was also observed. ISS conditions significantly decreased prenyl xanthone production in the wild-type strain and increased asperthecin production in LO1362 and CW12001, which are deficient in a major DNA repair mechanism. These data provide valuable insights into the adaptation mechanism of *A. nidulans* to spacecraft environments.

**Keywords:** *Aspergillus nidulans*, International Space Station, genomics, proteomics, metabolomics

## INTRODUCTION

It has been well documented that fungal populations persist in extreme conditions, such as various temperatures and pH ([López-Archilla et al. 2001](#R46); [Amaral Zettler et al. 2002](#R1); [Onofri et al. 2007](#R60); [de Crecy et al. 2009](#R23); [Gunde-Cimerman et al. 2009](#R28)), desiccation ([Barnard et al. 2013](#R6)), ionizing radiation ([Dadachova and Casadevall 2008](#R22); [Tkavc et al. 2018](#R76)), Mars-like conditions ([de la Torre Noetzel et al. 2018](#R24); [Onofri et al. 2018a](#R61); [Onofri et al. 2018b](#R62)), and the spacecraft environment ([Onofri et al. 2008](#R59); [Checinska et al. 2015](#R18)). Through adaptation, fungi have developed the capacity to sense and respond to external stimuli, enabling their survival in a wide variety of ecological niches ([Onofri et al. 2007](#R60); [Onofri et al. 2008](#R59); [Dadachova and Casadevall 2008](#R22); [de Crecy et al. 2009](#R23); [Checinska et al. 2015](#R18)). These omnipresent microorganisms can be both beneficial and detrimental to human health, as fungi produce a myriad of secondary metabolites (SMs) in response to environmental stressors with activities ranging from therapeutic to toxic ([Segal 2009](#R69); [Newman and Cragg 2012](#R55); [Sharpe et al. 2015](#R70)). SMs have had a tremendous impact on human health, as the majority of small-molecule drugs introduced between 1981 and 2010 were either SMs, SM derivatives, SM mimics, or possessed a SM pharmacophore. In fact, approximately 49% of all anticancer drugs are SMs or were inspired by SMs ([Newman and Cragg 2012](#R55)). Fungi are also potent producers of enzymes, and therefore have various industrial applications ([Vries and Visser 2001](#R81); [MacCabe et al. 2002](#R48)). We are on the cusp of significant advances in human interplanetary space exploration, as the National Aeronautics and Space Administration (NASA) aims to send humans back to the Moon and subsequently to Mars in the 2030s. In entering this new era of human spaceflight, a thorough understanding of how fungi respond and adapt to the various stimuli encountered while in space will play an important role in ensuring a health living environment for crew-members. Further, such studies enable the evaluation of fungi as drug production hosts during these exploration class missions, as fungi currently play an indispensable role in pharmaceutical biotechnology on Earth.

Fungi residing onboard the International Space Station (ISS) are exposed to microgravity and increased levels of radiation due to being outside of Earth’s protective atmosphere ([Horneck et al. 2010](#R33)). Microgravity is thought to decrease the transfer of extracellular nutrients and metabolic by-products, which may alter the chemical environment that the cell is exposed to ([Horneck et al. 2010](#R33); [Huang et al. 2018](#R34)). Radiation alters biological processes by acting as a promoter of mutagenesis, which may result in an increased rate of biological evolution leading to the development of adaptive responses ([Horneck et al. 2006](#R32)). Although it has been established that fungi are ubiquitous in spacecraft environments ([Pierson 2001](#R63); [Novikova et al. 2006](#R56); [Van Houdt et al. 2012](#R78); [Venkateswaran et al. 2014](#R79); [Checinska et al. 2015](#R18)), our understanding of how fungi respond and adapt to the various conditions encountered during spaceflight remains in its infancy ([Council 2011](#R21)). The objective of this study was to investigate the changes encountered in various aspects of fungal “omics” under ISS conditions using the well-characterized organism *Aspergillus nidulans*. Of 30 distinct fungal species retrieved from ISS habitat surfaces during one microbial monitoring study, *Aspergillus* was the dominant genus, featuring a diverse population of 13 species, with *A. nidulans* being one of four fungal species isolated from both surfaces and air onboard the ISS ([Novikova et al. 2006](#R56)). Of particular interest is how the space environment alters secondary metabolism, as fungal SM production is highly variable and dependent on external stimuli ([Haas 2015](#R30)). *A. nidulans* is an extensively studied model organism and in the last decade many of the genes and regulatory networks involved in SM formation have been characterized ([Yaegashi et al. 2014](#R84)). This new information enabled a comprehensive investigation into genomic, proteomic, and metabolomic alterations in response to the space environment.

Herein, we report the multi-omic characterization of wild-type (WT) *A. nidulans* FGSC A4 and three mutant strains, displayed in [Supplemental Table S1](#SD1), following 4 and 7 days of growth onboard the ISS and compared to ground counterparts ([Fig. 1](#F1)). The first mutant strain, LO1362, is the *A. nidulans nkuA* deletion strain, which is a homolog of the human KU70 gene. These genes are crucial for non-homologous end joining of DNA double strand break repair, and therefore deletion of *nkuA* disrupts a major DNA repair mechanism in *A. nidulans* ([Krappmann 2007](#R40)). The second mutant strain, LO8158, is deficient in *mcrA*, which is a negative regulator of at least 10 SM gene clusters ([Oakley et al. 2017](#R58)). Deletion of this gene stimulates SM production while impairing fungal growth. The final mutant strain used in this study was an *A. nidulans laeA* overexpression mutant, CW12001. LaeA is a global positive regulator of secondary metabolism, and therefore overexpression of *laeA* increases the production levels of a number of SMs ([Bok and Keller 2004](#R10)). ISS- and Earth-grown strains were subjected to genomic, proteomic, and metabolomic analyses with the aim to identify genetic and molecular alterations that occur in fungi in response to exposure to the space environment that could pave the way to new discoveries that could benefit human spaceflight and people on Earth.

### Fig. 1.

[![Fig. 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc83/9098231/9e5d5fb65b53/nihms-1065202-f0001.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9098231_nihms-1065202-f0001.jpg)

[Open in a new tab](figure/F1/)

Schematic overview of the *A. nidulans* ISS experiment. The *A. nidulans* wild-type (FGSC A4) and three mutant strains (LO1362, LO8158, and CW12001) were seeded onto GMM agar Omnitray plates and integrated into PHAB systems. Samples were transported to the ISS at 4°C, where they remained for approximately 26 or 23 days until being subjected to growth in SABL systems at 37°C for either 4 or 7 days, respectively. Earth-grown PHABs were simultaneously transferred to on-ground containers mimicking ISS SABL systems. Following growth, all samples were subjected to 4°C and ISS-grown samples were transported to Earth. All samples were subjected to genomic, proteomic, and metabolomics analyses.

## MATERIALS AND METHODS

### Strains, media, and growth conditions

The WT *A. nidulans* FGSC A4 strain was obtained from the Fungal Genetics Stock Center ([McCluskey et al. 2010](#R50)). LO1362 and LO8158 were obtained from previous studies ([Oakley et al. 2017](#R58)). The *laeA* overexpression strain (CW12001) was generated using the constitutive *gpdA* promoter according to standard protocol ([Hunter et al. 1992](#R35)). Protoplast production, construction of fusion PCR products, and transformation were carried out as described previously ([Szewczyk et al. 2006](#R74); [Nayak et al. 2006](#R54)). Primers used are listed in [Table S2](#SD1) and correct transformants were verified using diagnostic PCR.

All strains were seeded onto Nunc OmniTray plates (Thermo Fisher Scientific, Waltham, Massachusetts, USA) ([Supplementary Fig. S1a](#SD1)) containing 46 ml of solid GMM media (6g/L NaNO3, 0.52 g/L KCl, 0.52 g/L MgSO4\*7H2O, 1.52 g/L KH2PO4, 10g/L D-glucose, 15 g/L agar supplemented with 1ml/L of Hutner’s trace elements). Seeded Nunc OmniTrays were loaded into Plate Habitat (PHAB) systems (BioServe Space Technologies, Boulder, Colorado, USA) ([Supplementary Fig. S1b](#SD1)), with 6 Omnitray plates in each PHAB, and immediately transferred to 4°C. The PHAB system is a growth platform for biological materials that allows for gas exchange. Each PHAB was equipped with a temperature logger (HOBO) ([Supplementary Fig. S1c](#SD1)) that accurately measured the temperature throughout the duration of the experiment. PHABs containing cultures bound for the ISS were transferred to 4°C cold bags approximately 28 hours prior to launch, and transported to the ISS on the SpaceX CRS-8 (Space Exploration Technologies) mission that launched on April 8, 2016. On the ISS, PHABs were loaded into Space Automated Bioproduct Lab (SABL) systems ([Supplementary Fig. S1d](#SD1)), where the 7- and 4-day samples remained at 4°C for approximately 23 and 26 days, respectively. Ground culture PHABs were near-synchronously transferred to on-ground containers mimicking ISS SABL systems. To initiate growth, ISS and ground cultures were subjected to 37°C, where they remained for either 4 or 7 days, which are optimal growth conditions for SM production in *A. nidulans*. Following growth, all samples were subjected to 4°C, where they remained until ISS-grown agar plates were transported back to Earth on May 11, 2016. A more detailed explanation of space flight hardware, science preparation and loading, the flight operation timeline, and ISS environmental parameters can be found in the [Supplemental Methods](#SD1). All subsequent analyses of ISS- and ground-grown fungal samples, including genomic, proteomic, and metabolomic characterization, were conducted on Earth.

### Genomic DNA extraction, library preparation, and genome sequencing

Mycelia was collected from Earth-grown (7-day) and space-grown (4- and 7-day) GMM agar Omnitrays for all strains (FGSC A4, LO1362, LO8158, and CW12001), frozen with liquid nitrogen, and ground using a mortar and pestle. DNA was extracted using the Mo Bio PowerMax Soil DNA Isolation Kit (Mo Bio Laboratories, Carlsbad, California, USA) according to the manufacturers protocol. Library preparation and whole genome sequencing were performed at the Duke Center for Genomic and Computational Biology (Duke University Medical Center, Durham, North Carolina, USA). DNA quality and quantity were checked using the Agilent 2100 Bioanalyzer DNA assay (Agilent Technologies, Santa Clara, California, USA) and Qubit 2.0 Fluorometer (Thermo Fisher Scientific, Waltham, Massachusetts, USA). The library was prepared for paired-end sequencing using the TruSeq Nano DNA Library Preparation Kit (Illumina, San Diego, California, USA). Samples were sequenced using a HiSeq 4000 Illumina Sequencer and 101 base read lengths were generated.

### Genetic mutation identification

Illumina sequence reads were trimmed using Trimmomatic v 0.36 ([Bolger et al. 2014](#R12)) and quality was checked using FastQC v 0.11.7 ([Andrews 2010](#R3)). The genome and annotation files for *A. nidulans* FGSC A4 ([Galagan et al. 2005](#R27)) were downloaded from the FungiDB web portal ([Stajich et al. 2012](#R72)). Reads were mapped to the FGSC A4 reference genome using the Burrows-Wheeler Aligner (BWA) software package v 0.7.12 ([Li and Durbin 2009](#R43)), and further processed with SAMtools v 1.6 to generate sorted BAM files (L[i et al. 2009](#R44)). SNPs and indels were identified using GATK v 3.7 ([DePristo et al. 2011](#R25)). Duplicates were marked using Picard-tools MarkDuplicates (<https://broadinstitute.github.io/picard/>) to remove PCR artifacts. Sequence reads containing putative indels were realigned using GATK’s IndelRealigner to generate an updated BAM file. Variants within each sample were called using GATK’s Haplotype Caller, and the resulting Variant Call Format (VCF) files were combined using GATK’s Genotype GVCFs so that there was one VCF file for each strain (4 in total). GATK’s VariantFiltration was used to filter each VCF file based on stringent cutoffs for quality and coverage {SNPs: QD <2.0, MQ <40.0, QUAL <100, FS >60.0, MQRankSum <−12.5, SOR >4.0, ReadPosRankSum <−8.0; Indels: QD <2.0, FS >200.0, MQRankSum <−12.5, SOR >4, InbreedingCoeff <−0.8, ReadPosRankSum <−20.0}, so that only high-quality variants remained.

### Protein extraction

Mycelia was collected from Earth-grown (7-day) and space-grown (4- and 7-day) GMM agar Omnitrays for all strains (FGSC A4, LO1362, LO8158, and CW12001), frozen with liquid nitrogen, and ground using a mortar and pestle. For protein extraction, the lysis buffer consisted of 100 mM triethylammonium bicarbonate (TEAB) with 1X Halt Protease Inhibitor Cocktail (100X) (Thermo Fisher Scientific, Waltham, Massachusetts, USA) and 1mM phenylmethylsulfonyl fluoride (Sigma-Aldrich, St. Louis, Missouri, USA). The frozen ground mycelia were transferred and subjected to a Precellys 24 homogenizer (Bertin Instruments, Rockville, Maryland, USA) in which each sample was processed inside a 2-mL cryotube with 1.0 mm glass beads for three times (at 4°C, 6500 rpm, 1 min., repeated 3 times with 15 s pauses in between). The lysed cells were centrifuged at 17,000 × g for 15 min. Protein concentrations in the supernatants were measured by a Bradford assay (Bio-Rad Laboratories, Inc. Hercules, California, USA).

### Tandem mass tag (TMT) labeling

About 200 μg proteins from each sample were precipitated in 20% trichloroacetic acid (TCA) at 4°C. Protein pellets were obtained by centrifugation (17,000 g), washed with ice-cold acetone, and resuspended in 25μL TEAB (100 mM) and 25μL 2,2,2-trifluoroethanol (TFE). Proteins were reduced by adding 1μL of tris(2-carboxyethyl)phosphine (TCEP, 500 mM) and incubated for 1 hour at 37 °C (10 mM final TCEP concentration). Proteins were alkylated in presence of iodoacetamide (IAA, 30 mM) in the dark for 1 hour at room temperature. 2.5 μg per sample of mass spec grade trypsin/lysC (Promega, Madison, Wisconsin, USA) was used to digest the peptides overnight at 37 °C.

The digested peptides were quantified using the Pierce Quantitative Colorimetric Peptide Assay (Thermo Fisher Scientific, Waltham, Massachusetts, USA). 40 μg of peptides from each sample was labeled with the Thermo Scientific TMT 6-plex (TMT6) Isobaric Mass Tagging Kit according to the manufacturer’s protocol. The TMT6-130 label was used as either 4 day or 7 day strains’ reference that contained 5 μg of peptides from each of the 8 strain. The TMT6-131 label was used as a total mixture reference that contained 2.5 μg of peptides from each of the 16 strains. All six labeled-peptide mixtures were combined into a single tube, mixed, and fractionated using the Pierce High pH Reversed-Phase Peptide Fractionation Kit (Thermo Fisher Scientific, Waltham, Massachusetts, USA). While this kit usually uses only eight fractions with step elution of up to 50% acetonitrile, we added a ninths fraction eluting at 100% acetonitrile. Nine fractionated samples were dried using a SpeedVac concentrator and re-suspended in 1% formic acid prior to LC-MS/MS analysis.

### LC-MS/MS analysis

The samples were analyzed on an Orbitrap Fusion Tribrid mass spectrometer with an EASYnLC 1000 Liquid Chromatograph, a 75 μm x 2 cm Acclaim PepMap100 C18 trapping column, a 75 μm x 25 cm PepMap RSLC C18 analytical column, and an Easy-Spray ion source (Thermo Fisher Scientific, Waltham, Massachusetts, USA). The column temperature was maintained at 45 °C and the peptides were eluted at a flow rate of 300 nL/min over a 110 min gradient, from 3–30% solvent B (100 min), 30–50% solvent B (3 min), 50–90% solvent B (2 min), and 90% solvent B (2 min). The solvent A was 0.1 % formic acid in water and the solvent B was 0.1% formic acid in acetonitrile.

The full MS survey scan (m/z 400–1500) was acquired in the Orbitrap at a resolution of 120,000 and with an automatic gain control (AGC) target of 2×105. The maximum injection time for MS scans was 50 ms. Monoisotopic precursor ions were selected with charge states 2–7 with a ±10 ppm mass window using a 70 s dynamic exclusion. The MS2 scan (m/z 400–2000) was performed using the linear ion trap with the CID collision energy set to 35 %. The ion trap scan rate was set to “rapid”, with an AGC target of 4×103, and a maximum injection time of 150 ms. Ten fragment ions from each MS2 experiment were subsequently simultaneously selected for an MS3 experiment. The MS3 scan (m/z 100–500) was performed to generate the TMT reporter ions in the linear ion trap using HCD at a collision energy setting of 55 %, a rapid scan rate and an AGC target of 5×103, and a maximum injection time of 250 ms.

### Quantitative proteomics analysis

All MS/MS spectra were analyzed using the Proteome Discoverer v 2.2.0.388 (Thermo Fisher Scientific, Waltham, Massachusetts, USA) with the Sequest-HT searching engines against an *Aspergillus nidulans* FGSC A4 database containing 10,525 protein group sequences based on the annotated genome (NCBI, BioProject PRJEA40559, Assembly GCA\_000011425.1, Release date 2009/09/24). The search was performed with the following parameters: 2 maximum missed cleavage sites, 6 minimum peptide length, 5 ppm tolerance for precursor ion masses, and 0.6 Da tolerance for fragment ion masses. The static modification settings included carbamidomethyl of cysteine residues and dynamic modifications included oxidation of methionine, TMT6plex modification of lysine ε-amino groups and peptide N-termini, and acetyl modification of protein N-terminus. A false discovery rate (FDR) of 1% for peptides and proteins was obtained using a target-decoy database search. The reporter ions integration tolerance was 0.5 Da while the coisolation threshold was 75%. The average signal-to-noise threshold of all reported peaks was greater than 10. The quantitative abundance of each protein is determined from the total intensity of the detected reporter ions. The ratios between reporter and the reference reporter ion (TMT6- 131) were used to estimate the abundance ratio of each protein.

For the statistical analysis, technical triplicate measurements for each protein were averaged. Only proteins that were identified and quantified with at least one peptide detected in all three technical replicates were considered for the analysis. The normalization across two biological sample sets in eight TMT experiments was carried out according to Plubell, D.L. et al. with modifications ([Plubell et al. 2017](#R64)). Briefly, the data from the eight TMT experiments were first corrected for small systematic differences resulting from sample loading variations and labeling efficiency, by normalizing the reporter ion totals for each channel. The trimmed mean of M values (TMM) normalization corrected the compositional bias by aligning the median of the distribution of abundance intensities between samples ([Robinson and Oshlack 2010](#R66)). Internal reference scaling was used to adjust eight TMT data sets onto the same intensity scale. The normalized data was then averaged and log2 transformed. One-way ANOVA was preformed to identify proteins that were differentially expressed among strains in either 4 days or 7 days, respectively (p-value ≤ 0.05). The identified proteins were also evaluated for up- and downregulation by setting a ±2-fold change cut-off.

### Secondary metabolite extraction and analysis

Organic compounds were extracted by taking 3 plugs of fungal-grown agar and extracting with 3 ml methanol (MeOH), followed by 3 ml 1:1 MeOH-dichloromethane, each with 1 hour of sonication and filtration. The extract was evaporated *in vacuo* using a rotary evaporator, re-dissolved in 250 μl of 20% dimethyl sulfoxide in MeOH, and a portion (10 μl) was examined by high performance liquid chromatography-photodiode array detection-mass spectroscopy (HPLCDAD-MS) analysis. HPLC-MS was carried out using a ThermoFinnigan LCQ Advantage ion trap mass spectrometer with a reverse-phase C18 column (3 μm; 2.1 by 100 μm; Alltech Prevail) at a flow rate of 125 μl/min. The solvent gradient for HPLC-DAD-MS was 95% MeCN/H2O (solvent B) in 5% MeCN/H2O (solvent A) both containing 0.05% formic acid, as follows: 0% solvent B from 0 to 5 min, 0 to 100% solvent B from 5 min to 35 min, 100 to 0% solvent B from 40 to 45 min, and re-equilibration with 0% solve B from 45 to 50 min. For quantification, positive-ion electrospray ionization (ESI) was used for the detection of austinol, dehydroaustinol, sterigmatocystin, nidulanin A and analogues, emericellamides A and C-F, emericellin, shamixanthone, and epishamixanthone. Negative-ion ESI was used for the detection of asperthecin, terrequinone, and sterigmatocystin intermediate. Relative production levels were quantified by integrating the area under each SM’s ESI trace.

### Data availability

Raw WGS data for ISS-grown *A. nidulans* strains and ground-grown counterparts are available in the NCBI SRA under accession numbers SRR7724113- SRR7724124 and BioProject accession number PRJNA486827. Proteomics data is accessible through the ProteomeXchange Consortium via PRIDE with the dataset identifier PXD010778.

## RESULTS

### Genome variation among ISS-grown samples

To identify genomic alterations occurring in *A. nidulans* strains in response to ISS conditions, whole genome paired-end sequencing (WGS) was performed on ISS-grown samples and ground-grown control strains, which were subjected to growth for 7 days. Reads were aligned to the FGSC A4 reference genome and any single nucleotide polymorphisms (SNPs) present in ground controls were removed from each strain’s sample set, which resulted in the removal of 208, 194, 231, and 224 SNPs from the FGSC A4, LO1362, LO8158, and CW12001, respectively. This revealed 129, 136, 108, and 106 SNPs, and 36, 39, 41, and 31 INDELs in ISS-grown FGSC A4, LO1362, LO8158, and CW12001, respectively, that occurred in ISS-grown samples when compared to ground controls, the features of which are displayed in [Table 1](#T1). All identified SNPs and INDELs are summarized in [Supplementary Tables S3](#SD1)–[6](#SD1), presenting FGSC A4, LO1362, LO8158, and CW12001, respectively. The total number of missense mutations observed in ISS-grown strains ranged from 9 to 15 for each strain’s sample set. These mutations were observed within only 5 protein-coding genes, with the same mutation often present in multiple samples ([Table 2](#T2)). A total of 13 unique missense base mutations were observed within AN5254, which encodes a protein containing domains predicted to be involved in RNA binding and RNA-directed DNA polymerase activity. Two unique mutations that resulted in premature stop codon were also observed within AN5254, one of which occurred in both the 4- and 7-day LO8158 and CW12001 ISS-grown samples. The remaining missense mutations occurred within AN0532, AN0535, AN0537, and AN0538, which are within a few genes away from one another in the genome. AN0532 encodes a predicted DDE1 transposable element gene, while the products of AN05235, AN0537, and AN0538 are uncharacterized. In both CW12001 ISS-grown samples, identical mutations that resulted in the loss of the original stop codon were observed within the *laeA* (AN0807) gene. The same frameshift stop-gain mutation independently occurred in all space-grown LO1362 and LO8158 samples within the *mnpA* gene (AN10311) ([Supplementary Tables S4](#SD1)–[5](#SD1)), which encodes a hyphal cell wall mannoprotein that may influence the surface structure ([Jeong et al. 2004](#R36)).

#### Table 1.

Features of SNPs and INDELs

| Strain ID | FGSC A4 | LO1362 | LO8158 | CW12001 |
| --- | --- | --- | --- | --- |
| --- | | | | |
| No. of SNPs | 129 | 137 | 108 | 106 |
| Intergenic | 111 | 116 | 84 | 88 |
| Missense | 10 | 11 | 15 | 9 |
| Synonymous | 4 | 3 | 6 | 3 |
| Intron | 0 | 4 | 2 | 2 |
| UTR | 1 | 2 | 0 | 2 |
| Stop gained | 2 | 1 | 1 | 1 |
| Stop lost | 0 | 0 | 0 | 1 |
| --- | | | | |
| No. of INDELs | 36 | 38 | 41 | 31 |
| Intergenic | 36 | 34 | 37 | 29 |
| Frameshift | 0 | 2 | 1 | 1 |
| UTR | 0 | 2 | 3 | 1 |
| Stop gained | 0 | 1 | 1 | 0 |

[Open in a new tab](table/T1/)

#### Table 2.

Comparative analysis of non-synonymous SNPs occurring during spaceflight

|  |  |  | FGSC A4 | | LO1362 | | LO8158 | | CW12001 | |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Gene | Base mutation | Type of mutation | 4d | 7d | 4d | 7d | 4d | 7d | 4d | 7d |
| --- | | | | | | | | | | |
|  | ChrV\_A3367369G | Missense | + | − | − | − | + | + | − | − |
|  | ChrV\_C3367409T | Missense | + | − | − | + | − | − | − | − |
|  | ChrV\_A3367453G | Missense | − | + | − | − | − | − | − | − |
|  | ChrV\_C3367733T | Missense | − | − | − | + | − | − | − | − |
|  | ChrV\_G3367916A | Missense | + | + | − | − | − | − | − | − |
|  | ChrV\_T3367958C | Missense | − | − | − | − | + | + | − | − |
|  | ChrV\_C3367973T | Missense | − | − | − | − | + | + | − | − |
| AN5254 | ChrV\_C3368005T | Missense | − | − | − | − | + | + | + | + |
|  | ChrV\_C3368023T | Stop gained | − | − | − | − | + | + | + | + |
|  | ChrV\_T3368023C | Stop gained | − | + | − | − | − | − | − | − |
|  | ChrV\_A3368024G | Missense | − | − | − | − | + | + | + | + |
|  | ChrV\_G3368024A | Missense | − | + | − | − | − | − | − | − |
|  | ChrV\_T3368096C | Missense | − | + | − | − | − | − | + | − |
|  | ChrV\_T3368312C | Missense | − | − | + | + | − | − | − | − |
|  | ChrV\_C3368312T | Missense | − | − | − | − | − | − | + | − |
| --- | | | | | | | | | | |
| AN0807 | ChrVIII\_G2423110A | Stop lost | − | − | − | − | − | − | + | + |
| --- | | | | | | | | | | |
|  | ChrVIII\_A3254138C | Missense | − | − | − | + | − | − | − | − |
| AN0538 | ChrVIII\_T3254236C | Missense | − | − | + | − | − | − | − | − |
|  | ChrVIII\_C3254236T | Missense | − | − | − | − | + | + | − | − |
| --- | | | | | | | | | | |
|  | ChrVIII\_A3255566G | Missense | − | − | − | − | − | + | − | − |
|  | ChrVIII\_C3255576T | Missense | − | − | − | − | − | − | + | − |
|  | ChrVIII\_T3255576C | Missense | − | − | − | − | − | + | − | − |
|  | ChrVIII\_G3255581A | Missense | − | − | − | − | − | − | + | − |
| AN0537 | ChrVIII\_A3255581G | Missense | − | − | − | − | − | + | − | − |
|  | ChrVIII\_T3255781C | Missense | + | + | − | + | − | + | − | − |
|  | ChrVIII\_C3255975A | Splice region | − | − | − | − | − | − | + | + |
|  | ChrVIII\_G3256068A | Missense | − | − | + | − | − | − | − | − |
|  | ChrVIII\_A3256068G | Missense | − | − | − | − | − | + | − | − |
| --- | | | | | | | | | | |
| AN0535 | ChrVIII\_A3259230G | Stop gained | − | + | + | + | − | − | − | − |
| ChrVIII\_A3259256G | Missense | − | + | + | + | − | − | − | − |
| ChrVIII\_C3259257T | Missense | − | + | + | + | − | − | − | − |
| ChrVIII\_T3259346C | Missense | − | − | − | − | + | − | − | − |
| ChrVIII\_C3259346T | Missense | + | + | − | − | − | − | − | − |
| ChrVIII\_G3259467C | Missense | − | − | + | − | + | + | − | − |
| ChrVIII\_G3259508C | Missense | − | − | − | − | + | + | − | − |
| ChrVIII\_C3259563T | Missense | − | − | − | − | − | − | − | + |
| --- | | | | | | | | | | |
| AN0532 | ChrV\_C3267200T | Missense | − | − | − | − | − | − | − | + |
| ChrV\_T3267230C | Missense | − | − | + | + | + | + | − | − |

[Open in a new tab](table/T2/)

+ indicates presence of point mutation;

- indicates absence of point mutation

For all strains, most SNPs (>77%) and INDELs (>89%) occurred in intergenic regions. Most intergenic SNPs were clustered nearby several specific genes and did not appear to be strain specific ([Supplementary Tables S3](#SD1)–[6](#SD1)). Among all strains, many intergenic SNPs occurred near genes involved in transcription and translation, including the putative C6 transcription factor AN4972, the transcription elongation factor AN11131, the U3 small nucleolar ribonucleoprotein AN4298, the S-adenosylmethionine-dependent methyltransferase AN10829 with a predicted role in translational read-through, and AN6968 which is predicted to have RNA-directed DNA polymerase activity. Intergenic SNPs were also clustered nearby the putative alanine-tRNA ligase AN9419, the putative C4 sterol methyl oxidase AN6973 which has a predicted role in sterol metabolism, and AN9410 which has a predicted role in lipid metabolic processes. Intergenic SNPs were also observed near AN0538 and AN0539, which are within the uncharacterized cluster of genes that were reported above to possess high numbers of missense mutations. Most of the remaining intergenic SNPs occurred near AN6972, AN7848, AN10328, and AN11577, the products of which remain uncharacterized.

### Proteomic profiling of ISS-grown *A. nidulans*

To investigate alterations in the proteome of *A. nidulans* strains following growth on the ISS, total protein was extracted from two biological replicates of each ISS-grown sample and Earth-grown counterpart. All samples were subjected to tandem mass tag (TMT) labeling and LC-MS analysis. The resulting MS data were analyzed using Proteome Discoverer with the Sequest-HT search engine against the *A. nidulans* FGSC A4 protein database (NCBI). The abundance ratios for all ISS-grown samples were normalized to their Earth-grown counterparts ([Supplementary Table S7](#SD1), 4-day samples; [Supplemental Table S8](#SD1), 7-day samples), which led to the identification of up- and down-regulated proteins (fold-change (FC) >∣2∣, P < 0.05) in response to the ISS environment ([Fig. 2a](#F2)). Interestingly, only two proteins, QutC (AN1140), which is involved in quinic acid utilization, and AN2704, a putative aryl-alcohol oxidase-related protein, were up-regulated in *A. nidulans* WT strain. The number of proteins up-regulated and down-regulated in the three ISS-grown mutant strains ranged from 4 to 28, and 2 to 77, respectively, of 10,525 predicted proteins in *A. nidulans* FGSC A4 in total. Distribution of AspGD Gene Ontology (GO) Slim terms ([Arnaud et al. 2010](#R5)) among the differentially expressed proteins for ISS-grown mutant strains is displayed in [Fig. 2b](#F2)–[d](#F2). The GO Slim categories that possessed the highest number of differentially expressed proteins in LO1362 were stress response and carbohydrate metabolic processes. Similarly, in both LO8158 and CW12001, most differentially expressed proteins were involved in carbohydrate metabolism.

#### Fig. 2.

[![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc83/9098231/a7eebd824069/nihms-1065202-f0002.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9098231_nihms-1065202-f0002.jpg)

[Open in a new tab](figure/F2/)

Overview of proteomic analysis. **a** Number of up- and down-regulated proteins in ISSgrown strains (FC >∣2∣, P < 0.05) compared to ground-grown counterparts. **b-d** Biological process GO Slim categories of differentially expressed proteins. Differentially expressed proteins in **b** LO1362, **c** LO8158, and **d** CW12001 were mapped to terms representing various biological processes using AspGD Gene Ontology (GO) Slim Mapper.

Our study revealed differential abundance of proteins involved in the *A. nidulans* stress response following growth on the ISS ([Table 3](#T3)). The heat shock protein Hsp20 (AN10507) was highly affected by ISS conditions, displaying a twofold and fourfold increase in protein abundance in LO1362 and CW12001, respectively, and a twofold decrease in protein abundance in LO8158. Induction of Hsp20 has been reported following exposure to osmotic stress in *A. nidulans* ([Wu et al. 2016](#R83)). Conversely, the osmotic stress response protein CipB (AN7895) was down-regulated in LO1362 and the chaperone/heat shock protein Awh11 (AN3725) was down-regulated in LO1362 and LO8158. Differential abundance of proteins involved in oxidative stress response was observed among all ISS-grown strains relative to Earth-grown counterparts. The glutathione S-transferase GstB (AN6024), which has been reported to be significantly induced in response to menadione-induced oxidative stress ([Pusztahelyi et al. 2011](#R65)), was up-regulated over twofold in LO8158, but down-regulated 1.7-fold and 1.8-fold in LO1362 and CW12001, respectively, after 7 days of growth on the ISS. Two catalase, CatA (AN8637) and AN8553, exhibited decreased protein abundance in LO8158 and increased protein abundance in CW12001. Similarly, both the nitrosative stress response protein AN2470 and the menadione stress-induced protein AN5564 were down-regulated approximately threefold in LO8158 and upregulated approximately 1.7-fold in CW12001. Up-regulation was observed for phosphatidylinositol phospholipase C AN3636 whose ortholog plays a major role in responding to nutrient deprivation in *Candida albicans* ([Uhl et al. 2003](#R77)). Additionally, the GPI-anchored protein EcmA (AN4390) whose ortholog plays a major role in cell wall integrity, morphogenesis, and virulence, was up-regulated twofold in LO8158 after 7 days of growth on the ISS ([Martinez-Lopez et al. 2004](#R49)).

#### Table 3.

Differentially expressed proteins by strain and biological process

| Biological process |  |  | FGSC A4 | | LO1362 | | LO8158 | | CW12001 | |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ORF | Protein | 4d | 7d | 4d | 7d | 4d | 7d | 4d | 7d |
| --- | | | | | | | | | | |
| Response to stress | AN2470 |  | −0.04 | 0.01 | 0.72 | 0.51 | 0.42 | −1.56 | −0.24 | 0.82 |
| AN3636 |  | 0.10 |  | 1.02 |  | 0.09 |  | −0.42 |  |
| AN4891 | Asfl |  | 0.42 |  | 1.05 |  | −0.33 |  | 0.22 |
| AN5564 |  | −0.15 | 0.08 | −0.04 | 0.14 | 0.07 | −1.49 | −0.12 | 0.75 |
| AN3725 | Awhll | −0.04 | −0.16 | −1.27 | −0.68 | −0.43 | −1.07 | 0.22 | −0.02 |
| AN8637 | CatA | −0.21 | −0.24 | −0.64 | −0.41 | −0.35 | −1.65 | 0.31 | 0.90 |
| AN8553 |  | −0.24 | −0.36 | −0.30 | 0.03 | 0.04 | −1.53 | 0.00 | 0.68 |
| AN7895 | CipB | −0.02 | 0.09 | −0.65 | −1.21 | 0.25 | 0.72 | −0.39 | −0.53 |
| AN4390 | EcmA | 0.13 | 0.24 | −0.38 | −0.17 | 0.30 | 1.23 | −0.02 | −0.72 |
| AN1216 | GppA | 0.02 | 0.11 | 1.07 | 0.33 | 0.15 | −0.69 | −0.33 | 0.38 |
| AN6024 | GstB | −0.04 | 0.10 | −0.81 | −0.75 | 0.11 | 1.26 | −0.19 | −0.85 |
| AN10507 | Hsp20 | 0.60 | 0.69 | 1.03 | 0.99 | −0.48 | −1.38 | 0.99 | 2.20 |
| AN5217 | PilA | 0.30 | 0.01 | 0.31 | 0.41 | 0.00 | −1.04 | 0.05 | 0.75 |
| --- | | | | | | | | | | |
| Secondary metabolism | AN6000 | AptA | −0.02 | 0.90 | 0.24 | −0.42 | −0.25 | −0.61 | −0.15 | 1.06 |
| AN0147 | MdpD |  | −0.11 |  | 1.24 |  | −0.04 |  | 0.37 |
| AN10038 | MdpJ | 0.23 |  | 0.89 |  | −0.19 |  | −1.06 |  |
| AN7911 | OrsB | −0.24 | 0.01 | −0.66 | −0.72 | −1.01 | 0.39 | 0.03 | −0.61 |
| AN7812 | StcN | −0.05 | 0.07 | −0.69 | −0.74 | 0.19 | 0.96 | −0.18 | −1.15 |
| AN7817 |  | 0.43 | −0.37 | 0.32 | 0.32 | 1.24 | 0.07 | 0.22 | 0.33 |
| --- | | | | | | | | | | |
| Carbohydrate metabolism | AN0567 |  | 0.08 | 0.23 | −0.51 | −0.62 | −0.24 | 1.37 | 0.41 | −0.65 |
| AN10124 |  | 0.37 | 0.44 | −0.71 | −0.72 | 1.14 | 0.32 | 0.15 | 0.13 |
| AN1715 |  | −0.18 | −0.16 | −0.52 | −0.39 | −0.20 | −1.49 | 0.26 | 0.92 |
| AN2334 |  | −0.05 | 0.08 | −0.01 | −0.49 | −0.05 | 1.13 | 0.11 | −0.69 |
| AN6035 |  | 0.00 | −0.20 | −0.72 | −0.68 | 0.55 | 1.58 | 0.17 | −0.85 |
| AN8068 |  | 0.32 | −0.25 | −0.91 | −0.79 | −0.11 | 1.53 | −0.05 | −1.03 |
| AN9443 |  |  | 0.55 |  | −0.32 |  | 1.14 |  | −0.58 |
| AN1277 | AbfC | −0.20 | −0.04 | −0.76 | −0.72 | −0.22 | 1.09 | 0.05 | −0.71 |
| AN5634 | AcuD | −0.17 | −0.03 | −0.19 | 0.09 | 0.02 | −1.57 | 0.30 | 0.91 |
| AN1918 | AcuF | −0.22 | −0.26 | −0.03 | −0.16 | −0.01 | −1.38 | 0.17 | 0.90 |
| AN7345 | AgdC/AgdD | −0.09 | 0.13 | −0.38 | −0.52 | −1.63 | −0.14 | 0.49 | 0.10 |
| AN7396 | BglM | 0.19 | 0.04 | −0.72 | −0.56 | 0.29 | 1.33 | 0.21 | −0.46 |
| AN0494 | CbhB | 0.55 | −0.07 | −1.02 | −1.13 | −0.52 | 0.43 | 0.22 | −1.25 |
| AN6792 | GfdB | 0.03 | −0.02 | 0.46 | 0.21 | 0.26 | −1.35 | −0.07 | 1.03 |
| AN0756 | LacA | 0.09 | 0.31 | −0.16 | −0.30 | 0.56 | 1.09 | −0.15 | −1.05 |
| AN3368 | MndB | 0.13 | 0.55 | −0.74 | −0.82 | 1.00 | 0.38 | −0.44 | −0.07 |
| AN7349 | MutA | −0.28 | 0.16 | −0.50 | −0.13 | −1.26 | −0.51 | 0.43 | 0.22 |
| AN7135 | RglA | 0.33 | 0.35 | −0.68 | −0.60 | 0.32 | 1.48 | −0.06 | −1.08 |
| AN5061 | XgeB | 0.22 |  | −1.12 |  | −0.07 |  | −0.03 |  |
| AN1818 | XlnC | −0.11 | 0.11 | −0.85 | −0.82 | −0.08 | 1.01 | −0.32 | −0.88 |
| AN7401 | XlnE | 0.12 |  | −1.10 |  | −0.36 |  | −0.11 |  |

[Open in a new tab](table/T3/)

Values depict the log2 fold change of ISS-grown samples relative to ground-grown counterparts (P < 0.05).

Among all strains, proteins involved in secondary metabolism were differentially expressed in response to ISS conditions ([Table 3](#T3)). The polyketide synthase AptA (AN6000), which is required for biosynthesis of the spore pigment asperthecin ([Szewczyk et al. 2008](#R73)), was up-reguated over twofold in CW12001 and nearly twofold in FGSC A4 strain after 7 days of growth on the ISS. LO1362 also exhibited up-regulation of the monooxygenase MdpD (AN0147) after 7 days of growth. The MdpD protein is the product of a gene in the prenyl xanthone gene cluster, which is responsible for the production of monodictyphenone, emericellin, shamixanthone, and epishamixanthone ([Chiang et al. 2010](#R19); [Sanchez et al. 2011](#R67)). Interestingly, the opposite was observed in CW12001, as MdpJ (AN10038), which is the product of another gene in the prenyl xanthone gene cluster, exhibited decreased protein abundance in ISS-grown samples after 4 days. A similar trend was observed for proteins involved in the biosynthesis of the potent carcinogenic mycotoxin sterigmatocystin (ST) ([Fujii et al. 1976](#R26); [Brown et al. 1996b](#R15)). ST gene cluster products AN7817 and StcN (AN7812) exhibited increased protein abundance in LO8158, while StcN was down-regulated in CW12001, following growth in ISS conditions.

The proteome of ISS-grown *A. nidulans* samples also revealed differential levels of proteins involved in carbohydrate metabolism when compared to Earth-grown controls ([Table 3](#T3)). Many glycoside hydrolases involved in carbohydrate degradation processes were up-regulated more than twofold in ISS-grown LO8158 samples, including beta-1,4-endoglucanase AN8068 and beta-glucosidase BglM (AN7396), both of which are involved in cellulose degradation, alpha-arabinofuranosidase AbfC (AN1277), involved in pectin degradation, endo-1,4-beta-xylanase XlnC (AN1818), involved in xylan degradation, and beta-galactosidase LacA (AN0756), involved in xyloglucan, xylan, pectin, and galactomannan degradation. These glycoside hydrolases also exhibited decreased protein abundance in CW12001 when compared to ground counterparts. A similar trend was observed with alcohol oxidase AN0567, beta-glycosidase AN10124, ketose-1,6-bisphosphate aldolase AN2334, dehydratase AN6035, and rhamnogalacturonan lyase RglA (AN7135), which were up-regulated at least twofold in ISS-grown LO8158 samples and down-regulated at least 1.5-fold in CW12001. Conversely, mannose-6-phosphate isomerase AN1715, isocitrate lyase AcuD (AN5634), phosphoenolpyruvate carboxykinase AcuF (AN1918), and NAD+ dependent glycerol 3-phosphate dehydrogenase GfdB (AN6792) were down-regulated at least 2.5-fold in ISS-grown LO8158 samples and up-regulated at least 1.8-fold in CW12001 samples.

### Secondary metabolome alterations in ISS-grown *A. nidulans*

Alterations in SM production of *A. nidulans* in response to ISS conditions were assessed by extracting SMs from three biological replicates of each ISS- and Earth-grown counterpart, and were analyzed using high-performance liquid chromatography coupled with diode-array detection and electrospray ionization tandem mass spectrometry (HPLC-DAD-MS). All SMs were identified based on mass, UV absorption, and retention time, which led to the identification of austinol and dehydroaustinol ([Lo et al. 2012](#R45)), terrequinone ([Bok et al. 2006](#R9)), sterigmatocystin and its intermediate ([Brown et al. 1996a](#R14)), nidulanin A and its analogues ([Andersen et al. 2013](#R2)), the emericellamides ([Chiang et al. 2008](#R20)), the prenyl xanthones ([Sanchez et al. 2011](#R67)), and asperthecin ([Sanchez et al. 2012](#R68)) ([Supplementary Fig. S2](#SD1)). Relative differences in SM production levels of ISS-grown samples and Earth-grown counterparts were quantified by integrating the area under each SM’s EIC trace ([Supplementary Fig. S3](#SD1)). ISS conditions induced asperthecin production in LO1362 and CW12001 after 7 days of growth, with production levels increased by over 300% and 150%, respectively ([Fig. 3](#F3)). Production levels of prenyl xanthones decreased approximately fivefold in FGSC A4 ISS-grown samples ([Supplementary Fig. S3f](#SD1)). In LO8158 samples, emericellamide and terrequinone production decreased ([Supplementary Fig. S3b](#SD1)&[e](#SD1)), while nidulanin and sterigmatocystin production increased in ISS grown samples ([Supplementary Fig. S3c](#SD1)&[d](#SD1)).

#### Fig. 3.

[![Fig. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc83/9098231/b5a37e1b5d70/nihms-1065202-f0003.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9098231_nihms-1065202-f0003.jpg)

[Open in a new tab](figure/F3/)

Asperthecin production in ISS-grown LO1362 and CW12001. **a** LC-MS profiles depicting asperthecin production after 7 days of growth on the ISS, as detected by UV total scan. **b** Quantification of asperthecin production showing percent change for ISS-grown samples relative to Earth-grown counterparts. Significance was determined using Welch’s t-test. **c** Chemical structure of asperthecin.

## DISCUSSION

Although the persistence of fungi within space vessels is well-documented and unavoidable ([Novikova 2004](#R57); [Novikova et al. 2006](#R56); [Checinska et al. 2015](#R18)), little is understood about how fungi respond and adapt to spacecraft conditions, such as microgravity and enhanced radiation. To date, most microbiological studies conducted in such environments have focused on changes occurring within bacteria or the microbiome as a whole ([Tixador et al. 1985](#R75); [Wilson et al. 2008](#R82); [Checinska et al. 2015](#R18); [Mora et al. 2016](#R53); [Huang et al. 2018](#R34)). Additionally, despite the various therapeutic and industrial applications of SMs, few studies have analyzed the global influence of space conditions on fungal secondary metabolism, as previous investigations have often focused on the production of a single SM ([Lam et al. 1998](#R42); [Lam et al. 2002](#R41); [Benoit et al. 2006](#R8)). Therefore, with the duration of space missions expected to increase, a major goal of this study was to investigate space-induced alternations in fungal ‘omics’ to identify specific adaptation biomarkers and acquire insight into potential benefits of fungi that may not be discernable using traditional methodology.

This study revealed that the spacecraft environment alters the *A. nidulans* genome in specific regions. Five protein-coding genes displayed signatures of positive selection in the form of a high ratio of non-synonymous to synonymous SNPs across all ISS-grown samples. These data stand in agreement with a study that investigated genomic alterations occurring in the bacterium *Staphylococcus aureus* during spaceflight, in which missense mutations were clustered and occurred within only 9 protein-coding genes ([Guo et al. 2015](#R29)). High numbers of intergenic mutations were clustered near genes encoding transcriptional and translational machinery. One gene with several non-synonymous and two unique stop gain mutation encodes a putative retrotransposon, suggesting its suppression confers selective advantage during growth in spacecraft environments. Other missense and intergenic mutations were clustered within and around a specific region of the genome (AN0532-AN0538), suggesting that it underwent positive selection and therefore plays a role in adapting to the space environment. One of these genes also encoded an uncharacterized transposable element gene, underscoring the significance of alterations in transposable element activity in response to growth in ISS conditions. These findings are consistent with results from the aforementioned *S. aureus* study, as variations were also observed within a putative transposase ([Guo et al. 2015](#R29)). Interestingly, transposable element activity has been associated with stress response due to the novel variation it introduces into the genome ([Capy et al. 2000](#R16)). Future work should focus on characterizing the activities of the rest of these genes, as many of their functions remain unknown. Such knowledge may provide information key to elucidating the genetic adaptation mechanisms of fungi residing in spacecraft environments.

Correlations linking genomic and proteomic data were observed with the two strains genetically engineered to increase SM production. Expsoure to the space environment appeared to curtail enhanced SM production in the *laeA* overexpression strain through the introduction of a point mutation that resulted in loss of the *laeA* stop codon, thereby activating nonstop decay degradation of *laeA* mRNA ([Hoof et al. 2002](#R31); [Klauer and van Hoof 2012](#R37)). The ISS-induced down-regulation of *laeA* was also observed in the proteome of those samples, which exhibited an expression profile opposite that of the *mcrA* deletion strain for many proteins. Proteins involved in SM biosynthesis pathways regulated by *laeA* exhibited decreased and increased abundance in ISS-grown CW12001 and LO8158, respectively. The extensively studied protein LaeA forms a nuclear complex with VeA and VelB that coordinates secondary metabolism regulation with fungal development ([Bayram et al. 2010](#R7)). LaeA is often referred to as a global regulator of secondary metabolism ([Bok and Keller 2016](#R11)), and has been reported to also influence proteins involved in carbohydrate metabolism and oxidative stress response ([Lv et al. 2018](#R47)). Accordingly, several proteins involved in carbohydrate and antioxidant metabolic processes exhibited opposite protein expression profiles in LO8158 and CW12001. Regulation by *laeA* has been reported for some of these proteins in *A. flavus*, including endo-1,4-beta-xylanase, betaglycoside, endo-beta-1,4-glucanase, and oxidative stress response proteins CatA and GstB ([Lv et al. 2018](#R47)). These findings verify the extensive regulatory role of *laeA* and highlight the complexity involved in identifying the *laeA*-controlled processes responsible for conferring selective advantage of the observed *laeA* stop lost mutation. This is compounded by reports that *laeA* also alters chromatin remodeling, cell growth and metabolism, conidiogenesis, conidialchain elongation, sporulation, pigmentation and colony hydrophobicity in various fungal species ([Kosalková et al. 2009](#R39); [Chang et al. 2012](#R17); [Brakhage 2013](#R13); [Lv et al. 2018](#R47)). It is also possible that curtailing *laeA* overexpression is favored in ISS conditions to reduce energy expenditures in a stressful environment.

ISS conditions significantly increased production of asperthecin, an anthraquinone pigment, in LO1362 and CW12001. Both mutants are deficient in NkuA production, which facilitates non-homologous end-joining (NHEJ) DNA repair, the favored DNA double strand break repair pathway in filamentous fungi ([Krappmann 2007](#R40)). We had anticipated that NkuA deficient strains would be particularly interesting in flight studies due to impairment of a preferred DNA repair pathway in a mutagenic, high-radiation environment. Additionally, deletion of the *nkuA* homolog has been reported to increase sensitivity toward gamma irradiation in other fungal species ([Meyer et al. 2007](#R51)). Although we cannot discriminate the specific ISS condition that induced asperthecin production in LO1362, we hypothesize that it served as an alternative protective mechanism from the high levels of radiation present on the ISS. Our observation stays in agreement with other reports suggesting that pigment production is a key adaptive response of fungi exposed to similar environments ([Volz and Dublin 1973](#R80); [Singaravelan et al. 2008](#R71); [Dadachova and Casadevall 2008](#R22)). It is therefore possible that asperthecin can be used to protect other forms of life present on the spacecraft.

Cultivating plants for food in space will be crucial for the success of future space missions. However, space radiation can generate mutations in plant DNA, including base substitutions, deletions, and chromosomal alterations, which can result in genetic changes in seeds or tissue damage ([Micco et al. 2011](#R52); [Arena et al. 2014](#R4)). The transformation of asperthecin biosynthesis genes into plants may potentially minimize plant DNA damage, and optimize plant and astronaut health. Future studies should be conducted both on ground and on the ISS to verify this hypothesis. Interestingly, space conditions did not increase asperthecin production in LO8158, which also possesses the *nkuA-* genetic background. Global regulation of SM production was altered to increase SM production in both LO8158 and CW12001, but the genetic alteration was reversed through a stop lost point mutation only in CW12001. It is therefore possible that alternative metabolomic protective mechanisms were sufficient in LO8158, and therefore asperthecin production was not enhanced in ISS conditions.

Only a small proportion of proteins were differentially expressed in ISS-grown *A. nidulans* samples, which emphasizes the potential and safety of *A. nidulans* as a therapeutic production host during outer space missions. This finding may not hold true across the *Aspergillus* genus, as increased virulence has been reported in *Aspergillus fumigatus* strains isolated from the ISS ([Knox et al. 2016](#R38)). Currently, if pharmaceutical stocks in space are depleted, a small, unmanned spacecraft is launched to restock crew supplies. In the era of long-term space travel, the duration of future space missions is expected to drastically increase. The inability to deliver required drugs to astronauts in a timely manner could result in serious complications. Through heterologous expression of specific genes, *A. nidulans* introduces the ability to biosynthesize a wide range of pharmaceutical drugs within a week, which could significantly improve astronauts’ safety during long-term manned space missions.

In summary, this work has revealed the multi-omic response of the well-characterized model filamentous fungus*, A. nidulans*, to spacecraft conditions. These findings illustrate the potential of asperthecin to confer radiation resistance and of *A. nidulans* to be utilized as a small molecule production host in space. Further, specific genetic mutations involved in the adaptive mechanism of fungi in space environments were identified. Such knowledge may be valuable to NASA’s Space Biology Program (<https://www.nasa.gov/spacebio>) in planning for future outer space explorations.

## Supplementary Material

Sup\_1

[NIHMS1065202-supplement-Sup\_1.pdf](/articles/instance/9098231/bin/NIHMS1065202-supplement-Sup_1.pdf) (35.2MB, pdf)

## ACKNOWLEDGEMENTS

Part of the research described in this publication was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with NASA. We would like to thank astronauts Tim Peake, Tim Kopra, and Jeff Williams for handling the samples aboard the ISS, the Implementation Team at NASA Ames Research Center, and BioServe Space Technologies for coordinating this effort. © 2018 California Institute of Technology. Government sponsorship acknowledged.

JR drafted the manuscript, contributed to sample processing, and was responsible for data analysis and interpretation. AB contributed to sample processing and data interpretation. AC and MK conducted protein sample processing, LC/MS analyses, and proteome data processing. YC contributed to secondary metabolic analysis and interpretation. JY generated the CW12001 strain. SC was responsible for sample integration into flight hardware. FK was responsible for project implementation and generating metadata from the ISS. JS contributed to genome data processing and variant analysis. KV and CW designed the study, interpreted the data, and drafted the manuscript. All authors read and approved the final manuscript.

Reference herein to any specific commercial product, process, or service by trade name, trademark, manufacturer, or otherwise does not constitute or imply its endorsement by the U.S. Government or the Jet Propulsion Laboratory, California Institute of Technology.

### Funding:

This research was funded by a 2012 Space Biology NNH12ZTT001N grant nos. 1912829–26 under Task Order NNN13D111T award to CW and KV, which also funded JR and AB.

## Footnotes

**Conflict of interest:** The authors declare no conflicts of interest.

COMPLIANCE WITH ETHICAL STANDARDS

**Ethical approval:** This article does not contain any studies with human participants or animals performed by any of the authors.

## REFERENCES

1. Amaral Zettler LA, Gómez F, Zettler E, Keenan BG, Amils R, Sogin ML (2002) Microbiology: eukaryotic diversity in Spain’s River of Fire. Nature
   417:137. doi: 10.1038/417137a [[DOI](https://doi.org/10.1038/417137a)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12000949/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Microbiology:%20eukaryotic%20diversity%20in%20Spain%E2%80%99s%20River%20of%20Fire&author=LA%20Amaral%20Zettler&author=F%20G%C3%B3mez&author=E%20Zettler&author=BG%20Keenan&author=R%20Amils&volume=417&publication_year=2002&pages=137&pmid=12000949&doi=10.1038/417137a&)]
2. Andersen MR, Nielsen JB, Klitgaard A, Petersen LM, Zachariasen M, Hansen TJ, Blicher LH, Gotfredsen CH, Larsen TO, Nielsen KF, Mortensen UH (2013) Accurate prediction of secondary metabolite gene clusters in filamentous fungi. Proc Natl Acad Sci
   110:E99–E107. doi: 10.1073/pnas.1205532110 [[DOI](https://doi.org/10.1073/pnas.1205532110)] [[PMC free article](/articles/PMC3538241/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23248299/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&title=Accurate%20prediction%20of%20secondary%20metabolite%20gene%20clusters%20in%20filamentous%20fungi&author=MR%20Andersen&author=JB%20Nielsen&author=A%20Klitgaard&author=LM%20Petersen&author=M%20Zachariasen&volume=110&publication_year=2013&pages=E99-E107&pmid=23248299&doi=10.1073/pnas.1205532110&)]
3. Andrews S (2010) FastQC: a quality control tool for high throughput sequence data. [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FastQC:%20a%20quality%20control%20tool%20for%20high%20throughput%20sequence%20data&author=S%20Andrews&publication_year=2010&)]
4. Arena C, De Micco V, Macaeva E, Quintens R (2014) Space radiation effects on plant and mammalian cells. Acta Astronaut
   104:419–431. doi: 10.1016/j.actaastro.2014.05.005 [[DOI](https://doi.org/10.1016/j.actaastro.2014.05.005)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Astronaut&title=Space%20radiation%20effects%20on%20plant%20and%20mammalian%20cells&author=C%20Arena&author=V%20De%20Micco&author=E%20Macaeva&author=R%20Quintens&volume=104&publication_year=2014&pages=419-431&doi=10.1016/j.actaastro.2014.05.005&)]
5. Arnaud MB, Chibucos MC, Costanzo MC, Crabtree J, Inglis DO, Lotia A, Orvis J, Shah P, Skrzypek MS, Binkley G, Miyasato SR, Wortman JR, Sherlock G (2010) The Aspergillus Genome Database, a curated comparative genomics resource for gene, protein and sequence information for the Aspergillus research community. Nucleic Acids Res
   38:D420–D427. doi: 10.1093/nar/gkp751 [[DOI](https://doi.org/10.1093/nar/gkp751)] [[PMC free article](/articles/PMC2808984/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19773420/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20Aspergillus%20Genome%20Database,%20a%20curated%20comparative%20genomics%20resource%20for%20gene,%20protein%20and%20sequence%20information%20for%20the%20Aspergillus%20research%20community&author=MB%20Arnaud&author=MC%20Chibucos&author=MC%20Costanzo&author=J%20Crabtree&author=DO%20Inglis&volume=38&publication_year=2010&pages=D420-D427&pmid=19773420&doi=10.1093/nar/gkp751&)]
6. Barnard RL, Osborne CA, Firestone MK (2013) Responses of soil bacterial and fungal communities to extreme desiccation and rewetting. ISME J
   7:2229–2241. doi: 10.1038/ismej.2013.104 [[DOI](https://doi.org/10.1038/ismej.2013.104)] [[PMC free article](/articles/PMC3806258/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23823489/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ISME%20J&title=Responses%20of%20soil%20bacterial%20and%20fungal%20communities%20to%20extreme%20desiccation%20and%20rewetting&author=RL%20Barnard&author=CA%20Osborne&author=MK%20Firestone&volume=7&publication_year=2013&pages=2229-2241&pmid=23823489&doi=10.1038/ismej.2013.104&)]
7. Bayram ÖS, Bayram Ö, Valerius O, Park HS, Irniger S, Gerke J, Ni M, Han K-H, Yu J-H, Braus GH (2010) LaeA control of velvet family regulatory proteins for light-dependent development and fungal cell-type specificity. PLOS Genet
   6:e1001226. doi: 10.1371/journal.pgen.1001226 [[DOI](https://doi.org/10.1371/journal.pgen.1001226)] [[PMC free article](/articles/PMC2996326/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21152013/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20Genet&title=LaeA%20control%20of%20velvet%20family%20regulatory%20proteins%20for%20light-dependent%20development%20and%20fungal%20cell-type%20specificity&author=%C3%96S%20Bayram&author=%C3%96%20Bayram&author=O%20Valerius&author=HS%20Park&author=S%20Irniger&volume=6&publication_year=2010&pmid=21152013&doi=10.1371/journal.pgen.1001226&)]
8. Benoit MR, Li W, Stodieck LS, Lam KS, Winther CL, Roane TM, Klaus DM (2006) Microbial antibiotic production aboard the International Space Station. Appl Microbiol Biotechnol
   70:403–411. doi: 10.1007/s00253-005-0098-3 [[DOI](https://doi.org/10.1007/s00253-005-0098-3)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16091928/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&title=Microbial%20antibiotic%20production%20aboard%20the%20International%20Space%20Station&author=MR%20Benoit&author=W%20Li&author=LS%20Stodieck&author=KS%20Lam&author=CL%20Winther&volume=70&publication_year=2006&pages=403-411&pmid=16091928&doi=10.1007/s00253-005-0098-3&)]
9. Bok JW, Hoffmeister D, Maggio-Hall LA, Murillo R, Glasner JD, Keller NP (2006) Genomic mining for Aspergillus natural products. Chem Biol
   13:31–37. doi: 10.1016/j.chembiol.2005.10.008 [[DOI](https://doi.org/10.1016/j.chembiol.2005.10.008)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16426969/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol&title=Genomic%20mining%20for%20Aspergillus%20natural%20products&author=JW%20Bok&author=D%20Hoffmeister&author=LA%20Maggio-Hall&author=R%20Murillo&author=JD%20Glasner&volume=13&publication_year=2006&pages=31-37&pmid=16426969&doi=10.1016/j.chembiol.2005.10.008&)]
10. Bok JW, Keller NP (2004) LaeA, a regulator of secondary metabolism in Aspergillus spp. Eukaryot Cell
    3:527–535 [[DOI](https://doi.org/10.1128/EC.3.2.527-535.2004)] [[PMC free article](/articles/PMC387652/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15075281/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eukaryot%20Cell&title=LaeA,%20a%20regulator%20of%20secondary%20metabolism%20in%20Aspergillus%20spp&author=JW%20Bok&author=NP%20Keller&volume=3&publication_year=2004&pages=527-535&pmid=15075281&doi=10.1128/EC.3.2.527-535.2004&)]
11. Bok JW, Keller NP (2016) Insight into fungal secondary metabolism from ten years of LaeA research. In: Biochemistry and Molecular Biology. Springer, Cham, pp 21–29 [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Biochemistry%20and%20Molecular%20Biology&author=JW%20Bok&author=NP%20Keller&publication_year=2016&)]
12. Bolger AM, Lohse M, Usadel B (2014) Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics
    30:2114–2120. doi: 10.1093/bioinformatics/btu170 [[DOI](https://doi.org/10.1093/bioinformatics/btu170)] [[PMC free article](/articles/PMC4103590/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24695404/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Trimmomatic:%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&author=AM%20Bolger&author=M%20Lohse&author=B%20Usadel&volume=30&publication_year=2014&pages=2114-2120&pmid=24695404&doi=10.1093/bioinformatics/btu170&)]
13. Brakhage AA (2013) Regulation of fungal secondary metabolism. Nat Rev Microbiol
    11:21–32. doi: 10.1038/nrmicro2916 [[DOI](https://doi.org/10.1038/nrmicro2916)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23178386/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol&title=Regulation%20of%20fungal%20secondary%20metabolism&author=AA%20Brakhage&volume=11&publication_year=2013&pages=21-32&pmid=23178386&doi=10.1038/nrmicro2916&)]
14. Brown DW, Adams TH, Keller NP (1996a) Aspergillus has distinct fatty acid synthases for primary and secondary metabolism. Proc Natl Acad Sci
    93:14873–14877. doi: 10.1073/pnas.93.25.14873 [[DOI](https://doi.org/10.1073/pnas.93.25.14873)] [[PMC free article](/articles/PMC26229/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/8962148/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&title=Aspergillus%20has%20distinct%20fatty%20acid%20synthases%20for%20primary%20and%20secondary%20metabolism&author=DW%20Brown&author=TH%20Adams&author=NP%20Keller&volume=93&publication_year=1996a&pmid=8962148&doi=10.1073/pnas.93.25.14873&)]
15. Brown DW, Yu JH, Kelkar HS, Fernandes M, Nesbitt TC, Keller NP, Adams TH, Leonard TJ (1996b) Twenty-five coregulated transcripts define a sterigmatocystin gene cluster in Aspergillus nidulans. Proc Natl Acad Sci
    93:1418–1422. doi: 10.1073/pnas.93.4.1418 [[DOI](https://doi.org/10.1073/pnas.93.4.1418)] [[PMC free article](/articles/PMC39953/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/8643646/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&title=Twenty-five%20coregulated%20transcripts%20define%20a%20sterigmatocystin%20gene%20cluster%20in%20Aspergillus%20nidulans&author=DW%20Brown&author=JH%20Yu&author=HS%20Kelkar&author=M%20Fernandes&author=TC%20Nesbitt&volume=93&publication_year=1996b&pages=1418-1422&pmid=8643646&doi=10.1073/pnas.93.4.1418&)]
16. Capy P, Gasperi G, Biémont C, Bazin C (2000) Stress and transposable elements: co-evolution or useful parasites?
    Heredity
    85:101–106. doi: 10.1046/j.1365-2540.2000.00751.x [[DOI](https://doi.org/10.1046/j.1365-2540.2000.00751.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11012710/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heredity&title=Stress%20and%20transposable%20elements:%20co-evolution%20or%20useful%20parasites?&author=P%20Capy&author=G%20Gasperi&author=C%20Bi%C3%A9mont&author=C%20Bazin&volume=85&publication_year=2000&pages=101-106&pmid=11012710&doi=10.1046/j.1365-2540.2000.00751.x&)]
17. Chang P-K, Scharfenstein LL, Ehrlich KC, Wei Q, Bhatnagar D, Ingber BF (2012) Effects of laeA deletion on Aspergillus flavus conidial development and hydrophobicity may contribute to loss of aflatoxin production. Fungal Biol
    116:298–307. doi: 10.1016/j.funbio.2011.12.003 [[DOI](https://doi.org/10.1016/j.funbio.2011.12.003)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22289775/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Biol&title=Effects%20of%20laeA%20deletion%20on%20Aspergillus%20flavus%20conidial%20development%20and%20hydrophobicity%20may%20contribute%20to%20loss%20of%20aflatoxin%20production&author=P-K%20Chang&author=LL%20Scharfenstein&author=KC%20Ehrlich&author=Q%20Wei&author=D%20Bhatnagar&volume=116&publication_year=2012&pages=298-307&pmid=22289775&doi=10.1016/j.funbio.2011.12.003&)]
18. Checinska A, Probst AJ, Vaishampayan P, White JR, Kumar D, Stepanov VG, Fox GE, Nilsson HR, Pierson DL, Perry J, Venkateswaran K (2015) Microbiomes of the dust particles collected from the International Space Station and Spacecraft Assembly Facilities. Microbiome
    3:50. doi: 10.1186/s40168-015-0116-3 [[DOI](https://doi.org/10.1186/s40168-015-0116-3)] [[PMC free article](/articles/PMC4624184/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26502721/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Microbiomes%20of%20the%20dust%20particles%20collected%20from%20the%20International%20Space%20Station%20and%20Spacecraft%20Assembly%20Facilities&author=A%20Checinska&author=AJ%20Probst&author=P%20Vaishampayan&author=JR%20White&author=D%20Kumar&volume=3&publication_year=2015&pages=50&pmid=26502721&doi=10.1186/s40168-015-0116-3&)]
19. Chiang Y-M, Szewczyk E, Davidson AD, Entwistle R, Keller NP, Wang CCC, Oakley BR (2010) Characterization of the Aspergillus nidulans monodictyphenone gene cluster. Appl Environ Microbiol
    76:2067–2074. doi: 10.1128/AEM.02187-09 [[DOI](https://doi.org/10.1128/AEM.02187-09)] [[PMC free article](/articles/PMC2849234/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20139316/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Characterization%20of%20the%20Aspergillus%20nidulans%20monodictyphenone%20gene%20cluster&author=Y-M%20Chiang&author=E%20Szewczyk&author=AD%20Davidson&author=R%20Entwistle&author=NP%20Keller&volume=76&publication_year=2010&pages=2067-2074&pmid=20139316&doi=10.1128/AEM.02187-09&)]
20. Chiang Y-M, Szewczyk E, Nayak T, Davidson AD, Sanchez JF, Lo H-C, Ho W-Y, Simityan H, Kuo E, Praseuth A, Watanabe K, Oakley BR, Wang CCC (2008) Molecular genetic mining of the Aspergillus secondary metabolome: discovery of the emericellamide biosynthetic pathway. Chem Biol
    15:527–532. doi: 10.1016/j.chembiol.2008.05.010 [[DOI](https://doi.org/10.1016/j.chembiol.2008.05.010)] [[PMC free article](/articles/PMC2494592/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18559263/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol&title=Molecular%20genetic%20mining%20of%20the%20Aspergillus%20secondary%20metabolome:%20discovery%20of%20the%20emericellamide%20biosynthetic%20pathway&author=Y-M%20Chiang&author=E%20Szewczyk&author=T%20Nayak&author=AD%20Davidson&author=JF%20Sanchez&volume=15&publication_year=2008&pages=527-532&pmid=18559263&doi=10.1016/j.chembiol.2008.05.010&)]
21. Council NR (2011) Recapturing a future for space exploration: life and physical sciences research for a new era  [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Recapturing%20a%20future%20for%20space%20exploration:%20life%20and%20physical%20sciences%20research%20for%20a%20new%20era&author=NR%20Council&publication_year=2011&)]
22. Dadachova E, Casadevall A (2008) Ionizing radiation: how fungi cope, adapt, and exploit with the help of melanin. Curr Opin Microbiol
    11:525–531. doi: 10.1016/j.mib.2008.09.013 [[DOI](https://doi.org/10.1016/j.mib.2008.09.013)] [[PMC free article](/articles/PMC2677413/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18848901/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Microbiol&title=Ionizing%20radiation:%20how%20fungi%20cope,%20adapt,%20and%20exploit%20with%20the%20help%20of%20melanin&author=E%20Dadachova&author=A%20Casadevall&volume=11&publication_year=2008&pages=525-531&pmid=18848901&doi=10.1016/j.mib.2008.09.013&)]
23. de Crecy E, Jaronski S, Lyons B, Lyons TJ, Keyhani NO (2009) Directed evolution of a filamentous fungus for thermotolerance. BMC Biotechnol
    9:74. doi: 10.1186/1472-67509-74 [[DOI](https://doi.org/10.1186/1472-67509-74)] [[PMC free article](/articles/PMC2741439/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19709419/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Biotechnol&title=Directed%20evolution%20of%20a%20filamentous%20fungus%20for%20thermotolerance&author=E%20de%20Crecy&author=S%20Jaronski&author=B%20Lyons&author=TJ%20Lyons&author=NO%20Keyhani&volume=9&publication_year=2009&pages=74&pmid=19709419&doi=10.1186/1472-67509-74&)]
24. de la Torre Noetzel R, Miller AZ, Rosa DL, M J, Pacelli C, Onofri S, García Sancho L, Cubero B, Lorek A, Wolter D, Vera D, P J (2018) Cellular responses of the lichen Circinaria gyrosa in Mars-like conditions. Front Microbiol
    9:. doi: 10.3389/fmicb.2018.00308 [[DOI](https://doi.org/10.3389/fmicb.2018.00308)] [[PMC free article](/articles/PMC5845166/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29556220/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&title=Cellular%20responses%20of%20the%20lichen%20Circinaria%20gyrosa%20in%20Mars-like%20conditions&author=R%20de%20la%20Torre%20Noetzel&author=AZ%20Miller&author=DL%20Rosa&author=J%20M&author=C%20Pacelli&volume=9&publication_year=2018&pmid=29556220&doi=10.3389/fmicb.2018.00308&)]
25. DePristo MA, Banks E, Poplin RE, Garimella KV, Maguire JR, Hartl C, Philippakis AA, del Angel G, Rivas M., Hanna M, McKenna A, Fennell TJ, Kernytsky AM, Sivachenko AY, Cibulskis K, Gabriel SB, Altshuler D, Daly MJ (2011) A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet
    43:491–498. doi: 10.1038/ng.806 [[DOI](https://doi.org/10.1038/ng.806)] [[PMC free article](/articles/PMC3083463/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21478889/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&author=MA%20DePristo&author=E%20Banks&author=RE%20Poplin&author=KV%20Garimella&author=JR%20Maguire&volume=43&publication_year=2011&pages=491-498&pmid=21478889&doi=10.1038/ng.806&)]
26. Fujii K, Kurata H, Odashima S, Hatsuda Y (1976) Tumor induction by a single subcutaneous injection of sterigmatocystin in newborn mice. Cancer Res
    36:1615–1618 [[PubMed](https://pubmed.ncbi.nlm.nih.gov/1268822/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Tumor%20induction%20by%20a%20single%20subcutaneous%20injection%20of%20sterigmatocystin%20in%20newborn%20mice&author=K%20Fujii&author=H%20Kurata&author=S%20Odashima&author=Y%20Hatsuda&volume=36&publication_year=1976&pages=1615-1618&pmid=1268822&)]
27. Galagan JE, Calvo SE, Cuomo C, Ma L-J, Wortman JR, Batzoglou S, Lee S-I, Baştürkmen M, Spevak CC, Clutterbuck J, Kapitonov V, Jurka J, Scazzocchio C, Farman M, Butler J, Purcell S, Harris S, Braus GH, Draht O, Busch S, D’Enfert C, Bouchier C, Goldman GH, Bell-Pedersen D, Griffiths-Jones S, Doonan JH, Yu J, Vienken K, Pain A, Freitag M, Selker EU, Archer DB, Peñalva MA, Oakley BR, Momany M, Tanaka T, Kumagai T, Asai K, Machida M, Nierman WC, Denning DW, Caddick M, Hynes M, Paoletti M, Fischer R, Miller B, Dyer P, Sachs MS, Osmani SA, Birren BW (2005) Sequencing of Aspergillus nidulans and comparative analysis with A. fumigatus and A. oryzae. Nature
    438:1105–1115. doi: 10.1038/nature04341 [[DOI](https://doi.org/10.1038/nature04341)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16372000/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Sequencing%20of%20Aspergillus%20nidulans%20and%20comparative%20analysis%20with%20A.%20fumigatus%20and%20A.%20oryzae&author=JE%20Galagan&author=SE%20Calvo&author=C%20Cuomo&author=L-J%20Ma&author=JR%20Wortman&volume=438&publication_year=2005&pages=1105-1115&pmid=16372000&doi=10.1038/nature04341&)]
28. Gunde-Cimerman N, Ramos J, Plemenitas A (2009) Halotolerant and halophilic fungi. Mycol Res
    113:1231–1241. doi: 10.1016/j.mycres.2009.09.002 [[DOI](https://doi.org/10.1016/j.mycres.2009.09.002)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19747974/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycol%20Res&title=Halotolerant%20and%20halophilic%20fungi&author=N%20Gunde-Cimerman&author=J%20Ramos&author=A%20Plemenitas&volume=113&publication_year=2009&pages=1231-1241&pmid=19747974&doi=10.1016/j.mycres.2009.09.002&)]
29. Guo J, Han N, Zhang Y, Wang H, Zhang X, Su L, Liu C, Li J, Chen C, Liu C (2015) Use of genome sequencing to assess nucleotide structure variation of Staphylococcus aureus strains cultured in spaceflight on Shenzhou-X, under simulated microgravity and on the ground. Microbiol Res
    170:61–68. doi: 10.1016/j.micres.2014.09.001 [[DOI](https://doi.org/10.1016/j.micres.2014.09.001)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25304992/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol%20Res&title=Use%20of%20genome%20sequencing%20to%20assess%20nucleotide%20structure%20variation%20of%20Staphylococcus%20aureus%20strains%20cultured%20in%20spaceflight%20on%20Shenzhou-X,%20under%20simulated%20microgravity%20and%20on%20the%20ground&author=J%20Guo&author=N%20Han&author=Y%20Zhang&author=H%20Wang&author=X%20Zhang&volume=170&publication_year=2015&pages=61-68&pmid=25304992&doi=10.1016/j.micres.2014.09.001&)]
30. Haas H (2015) Microbial Ecology: How to trigger a fungal weapon. eLife
    4:e10504. doi: 10.7554/eLife.10504 [[DOI](https://doi.org/10.7554/eLife.10504)] [[PMC free article](/articles/PMC4555636/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26327696/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=eLife&title=Microbial%20Ecology:%20How%20to%20trigger%20a%20fungal%20weapon&author=H%20Haas&volume=4&publication_year=2015&pmid=26327696&doi=10.7554/eLife.10504&)]
31. Hoof van A, Frischmeyer PA, Dietz HC, Parker R(2002) Exosome-mediated recognition and degradation of mRNAs lacking a termination codon. Science
    295:2262–2264. doi: 10.1126/science.1067272 [[DOI](https://doi.org/10.1126/science.1067272)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11910110/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Exosome-mediated%20recognition%20and%20degradation%20of%20mRNAs%20lacking%20a%20termination%20codon&author=A%20Hoof%20van&author=PA%20Frischmeyer&author=HC%20Dietz&author=R%20Parker&volume=295&publication_year=2002&pages=2262-2264&pmid=11910110&doi=10.1126/science.1067272&)]
32. Horneck G, Baumstark-Khan C, Facius R (2006) Radiation biology. In: Clément G, Slenzka K (eds) Fundamentals of Space Biology. Springer New York, New York, NY, pp 291–336 [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Fundamentals%20of%20Space%20Biology&author=G%20Horneck&author=C%20Baumstark-Khan&author=R%20Facius&author=G%20Cl%C3%A9ment&author=K%20Slenzka&publication_year=2006&)]
33. Horneck G, Klaus DM, Mancinelli RL (2010) Space microbiology. Microbiol Mol Biol Rev
    74:121–156. doi: 10.1128/MMBR.00016-09 [[DOI](https://doi.org/10.1128/MMBR.00016-09)] [[PMC free article](/articles/PMC2832349/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20197502/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol%20Mol%20Biol%20Rev&title=Space%20microbiology&author=G%20Horneck&author=DM%20Klaus&author=RL%20Mancinelli&volume=74&publication_year=2010&pages=121-156&pmid=20197502&doi=10.1128/MMBR.00016-09&)]
34. Huang B, Li D-G, Huang Y, Liu C-T (2018) Effects of spaceflight and simulated microgravity on microbial growth and secondary metabolism. Mil Med Res
    5:. doi: 10.1186/s40779018-0162-9 [[DOI](https://doi.org/10.1186/s40779018-0162-9)] [[PMC free article](/articles/PMC5971428/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29807538/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mil%20Med%20Res&title=Effects%20of%20spaceflight%20and%20simulated%20microgravity%20on%20microbial%20growth%20and%20secondary%20metabolism&author=B%20Huang&author=D-G%20Li&author=Y%20Huang&author=C-T%20Liu&volume=5&publication_year=2018&pmid=29807538&doi=10.1186/s40779018-0162-9&)]
35. Hunter GD, Bailey CR, Arst HN (1992) Expression of a bacterial aspartase gene in Aspergillus nidulans: an efficient system for selecting multicopy transformants. Curr Genet
    22:377–383. doi: 10.1007/BF00352439 [[DOI](https://doi.org/10.1007/BF00352439)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/1423725/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Genet&title=Expression%20of%20a%20bacterial%20aspartase%20gene%20in%20Aspergillus%20nidulans:%20an%20efficient%20system%20for%20selecting%20multicopy%20transformants&author=GD%20Hunter&author=CR%20Bailey&author=HN%20Arst&volume=22&publication_year=1992&pages=377-383&pmid=1423725&doi=10.1007/BF00352439&)]
36. Jeong H-Y, Chae K-S, Whang SS (2004) Presence of a mannoprotein, MnpAp, in the hyphal cell wall of Aspergillus nidulans. Mycologia
    96:52–56. doi: 10.1080/15572536.2005.11832996 [[DOI](https://doi.org/10.1080/15572536.2005.11832996)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21148828/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=Presence%20of%20a%20mannoprotein,%20MnpAp,%20in%20the%20hyphal%20cell%20wall%20of%20Aspergillus%20nidulans&author=H-Y%20Jeong&author=K-S%20Chae&author=SS%20Whang&volume=96&publication_year=2004&pages=52-56&pmid=21148828&doi=10.1080/15572536.2005.11832996&)]
37. Klauer AA, van Hoof A (2012) Degradation of mRNAs that lack a stop codon: A decade of nonstop progress. Wiley Interdiscip Rev RNA
    3:649–660. doi: 10.1002/wrna.1124 [[DOI](https://doi.org/10.1002/wrna.1124)] [[PMC free article](/articles/PMC3638749/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22740367/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Wiley%20Interdiscip%20Rev%20RNA&title=Degradation%20of%20mRNAs%20that%20lack%20a%20stop%20codon:%20A%20decade%20of%20nonstop%20progress&author=AA%20Klauer&author=A%20van%20Hoof&volume=3&publication_year=2012&pages=649-660&pmid=22740367&doi=10.1002/wrna.1124&)]
38. Knox BP, Blachowicz A, Palmer JM, Romsdahl J, Huttenlocher A, Wang CCC, Keller NP, Venkateswaran K (2016) Characterization of Aspergillus fumigatus isolates from air and surfaces of the International Space Station. mSphere
    1:e00227–16. doi: 10.1128/mSphere.00227-16 [[DOI](https://doi.org/10.1128/mSphere.00227-16)] [[PMC free article](/articles/PMC5082629/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27830189/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=mSphere&title=Characterization%20of%20Aspergillus%20fumigatus%20isolates%20from%20air%20and%20surfaces%20of%20the%20International%20Space%20Station&author=BP%20Knox&author=A%20Blachowicz&author=JM%20Palmer&author=J%20Romsdahl&author=A%20Huttenlocher&volume=1&publication_year=2016&pmid=27830189&doi=10.1128/mSphere.00227-16&)]
39. Kosalková K, García-Estrada C, Ullán RV, Godio RP, Feltrer R, Teijeira F, Mauriz E, Martín JF (2009) The global regulator LaeA controls penicillin biosynthesis, pigmentation and sporulation, but not roquefortine C synthesis in Penicillium chrysogenum. Biochimie
    91:214–225. doi: 10.1016/j.biochi.2008.09.004 [[DOI](https://doi.org/10.1016/j.biochi.2008.09.004)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18952140/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochimie&title=The%20global%20regulator%20LaeA%20controls%20penicillin%20biosynthesis,%20pigmentation%20and%20sporulation,%20but%20not%20roquefortine%20C%20synthesis%20in%20Penicillium%20chrysogenum&author=K%20Kosalkov%C3%A1&author=C%20Garc%C3%ADa-Estrada&author=RV%20Ull%C3%A1n&author=RP%20Godio&author=R%20Feltrer&volume=91&publication_year=2009&pages=214-225&pmid=18952140&doi=10.1016/j.biochi.2008.09.004&)]
40. Krappmann S (2007) Gene targeting in filamentous fungi: the benefits of impaired repair. Fungal Biol Rev
    21:25–29. doi: 10.1016/j.fbr.2007.02.004 [[DOI](https://doi.org/10.1016/j.fbr.2007.02.004)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Biol%20Rev&title=Gene%20targeting%20in%20filamentous%20fungi:%20the%20benefits%20of%20impaired%20repair&author=S%20Krappmann&volume=21&publication_year=2007&pages=25-29&doi=10.1016/j.fbr.2007.02.004&)]
41. Lam KS, Gustavson DR, Pirnik DL, Pack E, Bulanhagui C, Mamber SW, Forenza S, Stodieck LS, Klaus DM (2002) The effect of space flight on the production of actinomycin D by Streptomyces plicatus. J Ind Microbiol Biotechnol
    29:299–302. doi: 10.1038/sj.jim.7000312 [[DOI](https://doi.org/10.1038/sj.jim.7000312)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12483468/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Ind%20Microbiol%20Biotechnol&title=The%20effect%20of%20space%20flight%20on%20the%20production%20of%20actinomycin%20D%20by%20Streptomyces%20plicatus&author=KS%20Lam&author=DR%20Gustavson&author=DL%20Pirnik&author=E%20Pack&author=C%20Bulanhagui&volume=29&publication_year=2002&pages=299-302&pmid=12483468&doi=10.1038/sj.jim.7000312&)]
42. Lam KS, Mamber SW, Pack EJ, Forenza S, Fernandes PB, Klaus DM (1998) The effects of space flight on the production of monorden by Humicola fuscoatra WC5157 in solidstate fermentation. Appl Microbiol Biotechnol
    49:579–583 [[DOI](https://doi.org/10.1007/s002530051216)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/9650257/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&title=The%20effects%20of%20space%20flight%20on%20the%20production%20of%20monorden%20by%20Humicola%20fuscoatra%20WC5157%20in%20solidstate%20fermentation&author=KS%20Lam&author=SW%20Mamber&author=EJ%20Pack&author=S%20Forenza&author=PB%20Fernandes&volume=49&publication_year=1998&pages=579-583&pmid=9650257&doi=10.1007/s002530051216&)]
43. Li H, Durbin R (2009) Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics
    25:1754–1760. doi: 10.1093/bioinformatics/btp324 [[DOI](https://doi.org/10.1093/bioinformatics/btp324)] [[PMC free article](/articles/PMC2705234/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19451168/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&author=H%20Li&author=R%20Durbin&volume=25&publication_year=2009&pages=1754-1760&pmid=19451168&doi=10.1093/bioinformatics/btp324&)]
44. Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R, 1000 Genome Project Data Processing Subgroup (2009) The Sequence Alignment/Map format and SAMtools. Bioinforma Oxf Engl
    25:2078–2079. doi: 10.1093/bioinformatics/btp352 [[DOI](https://doi.org/10.1093/bioinformatics/btp352)] [[PMC free article](/articles/PMC2723002/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19505943/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinforma%20Oxf%20Engl&title=The%20Sequence%20Alignment/Map%20format%20and%20SAMtools&author=H%20Li&author=B%20Handsaker&author=A%20Wysoker&author=T%20Fennell&author=J%20Ruan&volume=25&publication_year=2009&pages=2078-2079&pmid=19505943&doi=10.1093/bioinformatics/btp352&)]
45. Lo H-C, Entwistle R, Guo C-J, Ahuja M, Szewczyk E, Hung J-H, Chiang Y-M, Oakley BR, Wang CCC (2012) Two separate gene clusters encode the biosynthetic pathway for the meroterpenoids austinol and dehydroaustinol in Aspergillus nidulans. J Am Chem Soc
    134:4709–4720. doi: 10.1021/ja209809t [[DOI](https://doi.org/10.1021/ja209809t)] [[PMC free article](/articles/PMC3350773/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22329759/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&title=Two%20separate%20gene%20clusters%20encode%20the%20biosynthetic%20pathway%20for%20the%20meroterpenoids%20austinol%20and%20dehydroaustinol%20in%20Aspergillus%20nidulans&author=H-C%20Lo&author=R%20Entwistle&author=C-J%20Guo&author=M%20Ahuja&author=E%20Szewczyk&volume=134&publication_year=2012&pages=4709-4720&pmid=22329759&doi=10.1021/ja209809t&)]
46. López-Archilla AI, Marin I, Amils R (2001) Microbial community composition and ecology of an acidic aquatic environment: The Tinto River, Spain. Microb Ecol
    41:20–35. doi: 10.1007/s002480000044 [[DOI](https://doi.org/10.1007/s002480000044)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11252161/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microb%20Ecol&title=Microbial%20community%20composition%20and%20ecology%20of%20an%20acidic%20aquatic%20environment:%20The%20Tinto%20River,%20Spain&author=AI%20L%C3%B3pez-Archilla&author=I%20Marin&author=R%20Amils&volume=41&publication_year=2001&pages=20-35&pmid=11252161&doi=10.1007/s002480000044&)]
47. Lv Y, Lv A, Zhai H, Zhang S, Li L, Cai J, Hu Y (2018) Insight into the global regulation of laeA in Aspergillus flavus based on proteomic profiling. Int J Food Microbiol
    284:11–21. doi: 10.1016/j.ijfoodmicro.2018.06.024 [[DOI](https://doi.org/10.1016/j.ijfoodmicro.2018.06.024)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29990635/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Food%20Microbiol&title=Insight%20into%20the%20global%20regulation%20of%20laeA%20in%20Aspergillus%20flavus%20based%20on%20proteomic%20profiling&author=Y%20Lv&author=A%20Lv&author=H%20Zhai&author=S%20Zhang&author=L%20Li&volume=284&publication_year=2018&pages=11-21&pmid=29990635&doi=10.1016/j.ijfoodmicro.2018.06.024&)]
48. MacCabe AP, Orejas M, Tamayo EN, Villanueva A, Ramón D (2002) Improving extracellular production of food-use enzymes from Aspergillus nidulans. J Biotechnol
    96:43–54 [[DOI](https://doi.org/10.1016/s0168-1656%2802%2900036-6)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12142142/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biotechnol&title=Improving%20extracellular%20production%20of%20food-use%20enzymes%20from%20Aspergillus%20nidulans&author=AP%20MacCabe&author=M%20Orejas&author=EN%20Tamayo&author=A%20Villanueva&author=D%20Ram%C3%B3n&volume=96&publication_year=2002&pages=43-54&pmid=12142142&doi=10.1016/s0168-1656(02)00036-6&)]
49. Martinez-Lopez R, Monteoliva L, Diez-Orejas R, Nombela C, Gil C (2004) The GPI-anchored protein CaEcm33p is required for cell wall integrity, morphogenesis and virulence in Candida albicans. Microbiology
    150:3341–3354. doi: 10.1099/mic.0.27320-0 [[DOI](https://doi.org/10.1099/mic.0.27320-0)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15470113/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiology&title=The%20GPI-anchored%20protein%20CaEcm33p%20is%20required%20for%20cell%20wall%20integrity,%20morphogenesis%20and%20virulence%20in%20Candida%20albicans&author=R%20Martinez-Lopez&author=L%20Monteoliva&author=R%20Diez-Orejas&author=C%20Nombela&author=C%20Gil&volume=150&publication_year=2004&pages=3341-3354&pmid=15470113&doi=10.1099/mic.0.27320-0&)]
50. McCluskey K, Wiest A, Plamann M (2010) The Fungal Genetics Stock Center: a repository for 50 years of fungal genetics research. J Biosci
    35:119–126. doi: 10.1007/s12038-0100014-6 [[DOI](https://doi.org/10.1007/s12038-0100014-6)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20413916/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biosci&title=The%20Fungal%20Genetics%20Stock%20Center:%20a%20repository%20for%2050%20years%20of%20fungal%20genetics%20research&author=K%20McCluskey&author=A%20Wiest&author=M%20Plamann&volume=35&publication_year=2010&pages=119-126&pmid=20413916&doi=10.1007/s12038-0100014-6&)]
51. Meyer V, Arentshorst M, El-Ghezal A, Drews A-C, Kooistra R, van den Hondel CAMJJ, Ram AFJ(2007) Highly efficient gene targeting in the Aspergillus niger kusA mutant. J Biotechnol
    128:770–775. doi: 10.1016/j.jbiotec.2006.12.021 [[DOI](https://doi.org/10.1016/j.jbiotec.2006.12.021)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17275117/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biotechnol&title=Highly%20efficient%20gene%20targeting%20in%20the%20Aspergillus%20niger%20kusA%20mutant&author=V%20Meyer&author=M%20Arentshorst&author=A%20El-Ghezal&author=A-C%20Drews&author=R%20Kooistra&volume=128&publication_year=2007&pages=770-775&pmid=17275117&doi=10.1016/j.jbiotec.2006.12.021&)]
52. Micco VD, Arena C, Pignalosa D, Durante M (2011) Effects of sparsely and densely ionizing radiation on plants. Radiat Environ Biophys
    50:1–19. doi: 10.1007/s00411-010-0343-8 [[DOI](https://doi.org/10.1007/s00411-010-0343-8)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21113610/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Radiat%20Environ%20Biophys&title=Effects%20of%20sparsely%20and%20densely%20ionizing%20radiation%20on%20plants&author=VD%20Micco&author=C%20Arena&author=D%20Pignalosa&author=M%20Durante&volume=50&publication_year=2011&pages=1-19&pmid=21113610&doi=10.1007/s00411-010-0343-8&)]
53. Mora M, Perras A, Alekhova TA, Wink L, Krause R, Aleksandrova A, Novozhilova T, MoisslEichinger C (2016) Resilient microorganisms in dust samples of the International Space Station—survival of the adaptation specialists. Microbiome
    4:. doi: 10.1186/s40168-0160217-7 [[DOI](https://doi.org/10.1186/s40168-0160217-7)] [[PMC free article](/articles/PMC5175303/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27998314/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Resilient%20microorganisms%20in%20dust%20samples%20of%20the%20International%20Space%20Station%E2%80%94survival%20of%20the%20adaptation%20specialists&author=M%20Mora&author=A%20Perras&author=TA%20Alekhova&author=L%20Wink&author=R%20Krause&volume=4&publication_year=2016&pmid=27998314&doi=10.1186/s40168-0160217-7&)]
54. Nayak T, Szewczyk E, Oakley CE, Osmani A, Ukil L, Murray SL, Hynes MJ, Osmani SA, Oakley BR (2006) A versatile and efficient gene-targeting system for Aspergillus nidulans. Genetics
    172:1557–1566. doi: 10.1534/genetics.105.052563 [[DOI](https://doi.org/10.1534/genetics.105.052563)] [[PMC free article](/articles/PMC1456264/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16387870/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=A%20versatile%20and%20efficient%20gene-targeting%20system%20for%20Aspergillus%20nidulans&author=T%20Nayak&author=E%20Szewczyk&author=CE%20Oakley&author=A%20Osmani&author=L%20Ukil&volume=172&publication_year=2006&pages=1557-1566&pmid=16387870&doi=10.1534/genetics.105.052563&)]
55. Newman DJ, Cragg GM (2012) Natural products as sources of new drugs over the 30 years from 1981 to 2010. J Nat Prod
    75:311–335. doi: 10.1021/np200906s [[DOI](https://doi.org/10.1021/np200906s)] [[PMC free article](/articles/PMC3721181/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22316239/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Nat%20Prod&title=Natural%20products%20as%20sources%20of%20new%20drugs%20over%20the%2030%20years%20from%201981%20to%202010&author=DJ%20Newman&author=GM%20Cragg&volume=75&publication_year=2012&pages=311-335&pmid=22316239&doi=10.1021/np200906s&)]
56. Novikova N, De Boever P, Poddubko S, Deshevaya E, Polikarpov N, Rakova N, Coninx I, Mergeay M (2006) Survey of environmental biocontamination on board the International Space Station. Res Microbiol
    157:5–12. doi: 10.1016/j.resmic.2005.07.010 [[DOI](https://doi.org/10.1016/j.resmic.2005.07.010)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16364606/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Res%20Microbiol&title=Survey%20of%20environmental%20biocontamination%20on%20board%20the%20International%20Space%20Station&author=N%20Novikova&author=P%20De%20Boever&author=S%20Poddubko&author=E%20Deshevaya&author=N%20Polikarpov&volume=157&publication_year=2006&pages=5-12&pmid=16364606&doi=10.1016/j.resmic.2005.07.010&)]
57. Novikova ND (2004) Review of the knowledge of microbial contamination of the Russian manned spacecraft. Microb Ecol
    47:127–132. doi: 10.1007/s00248-003-1055-2 [[DOI](https://doi.org/10.1007/s00248-003-1055-2)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/14994178/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microb%20Ecol&title=Review%20of%20the%20knowledge%20of%20microbial%20contamination%20of%20the%20Russian%20manned%20spacecraft&author=ND%20Novikova&volume=47&publication_year=2004&pages=127-132&pmid=14994178&doi=10.1007/s00248-003-1055-2&)]
58. Oakley CE, Ahuja M, Sun W-W, Entwistle R, Akashi T, Yaegashi J, Guo C-J, Cerqueira GC, Russo Wortman J, Wang CCC, Chiang Y-M, Oakley BR (2017) Discovery of McrA, a master regulator of Aspergillus secondary metabolism. Mol Microbiol
    103:347–365. doi: 10.1111/mmi.13562 [[DOI](https://doi.org/10.1111/mmi.13562)] [[PMC free article](/articles/PMC5218965/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27775185/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&title=Discovery%20of%20McrA,%20a%20master%20regulator%20of%20Aspergillus%20secondary%20metabolism&author=CE%20Oakley&author=M%20Ahuja&author=W-W%20Sun&author=R%20Entwistle&author=T%20Akashi&volume=103&publication_year=2017&pages=347-365&pmid=27775185&doi=10.1111/mmi.13562&)]
59. Onofri S, Barreca D, Selbmann L, Isola D, Rabbow E, Horneck G, de Vera JPP, Hatton J, Zucconi L (2008) Resistance of Antarctic black fungi and cryptoendolithic communities to simulated space and Martian conditions. Stud Mycol
    61:99–109. doi: 10.3114/sim.2008.61.10 [[DOI](https://doi.org/10.3114/sim.2008.61.10)] [[PMC free article](/articles/PMC2610303/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19287532/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stud%20Mycol&title=Resistance%20of%20Antarctic%20black%20fungi%20and%20cryptoendolithic%20communities%20to%20simulated%20space%20and%20Martian%20conditions&author=S%20Onofri&author=D%20Barreca&author=L%20Selbmann&author=D%20Isola&author=E%20Rabbow&volume=61&publication_year=2008&pages=99-109&pmid=19287532&doi=10.3114/sim.2008.61.10&)]
60. Onofri S, Selbmann L, de Hoog GS, Grube M, Barreca D, Ruisi S, Zucconi L (2007) Evolution and adaptation of fungi at boundaries of life. Adv Space Res
    40:1657–1664. doi: 10.1016/j.asr.2007.06.004 [[DOI](https://doi.org/10.1016/j.asr.2007.06.004)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Space%20Res&title=Evolution%20and%20adaptation%20of%20fungi%20at%20boundaries%20of%20life&author=S%20Onofri&author=L%20Selbmann&author=GS%20de%20Hoog&author=M%20Grube&author=D%20Barreca&volume=40&publication_year=2007&pages=1657-1664&doi=10.1016/j.asr.2007.06.004&)]
61. Onofri S, Selbmann L, Pacelli C, de Vera JP, Horneck G, Hallsworth JE, Zucconi L (2018a) Integrity of the DNA and cellular ultrastructure of cryptoendolithic fungi in space or Mars conditions: a 1.5-year study at the International Space Station. Life Basel Switz
    8:. doi: 10.3390/life8020023 [[DOI](https://doi.org/10.3390/life8020023)] [[PMC free article](/articles/PMC6027225/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29921763/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Basel%20Switz&title=Integrity%20of%20the%20DNA%20and%20cellular%20ultrastructure%20of%20cryptoendolithic%20fungi%20in%20space%20or%20Mars%20conditions:%20a%201.5-year%20study%20at%20the%20International%20Space%20Station&author=S%20Onofri&author=L%20Selbmann&author=C%20Pacelli&author=JP%20de%20Vera&author=G%20Horneck&volume=8&publication_year=2018a&pmid=29921763&doi=10.3390/life8020023&)]
62. Onofri S, Selbmann L, Pacelli C, Zucconi L, Rabbow E, de Vera J-P (2018b) Survival, DNA, and ultrastructural integrity of a cryptoendolithic Antarctic fungus in Mars and lunar rock analogues exposed outside the International Space Station. Astrobiology. doi: 10.1089/ast.2017.1728 [[DOI](https://doi.org/10.1089/ast.2017.1728)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30376361/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Survival,%20DNA,%20and%20ultrastructural%20integrity%20of%20a%20cryptoendolithic%20Antarctic%20fungus%20in%20Mars%20and%20lunar%20rock%20analogues%20exposed%20outside%20the%20International%20Space%20Station&author=S%20Onofri&author=L%20Selbmann&author=C%20Pacelli&author=L%20Zucconi&author=E%20Rabbow&publication_year=2018b&pmid=30376361&doi=10.1089/ast.2017.1728&)]
63. Pierson DL (2001) Microbial contamination of spacecraft. Gravitational Space Biol Bull Publ Am Soc Gravitational Space Biol
    14:1–6 [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11865864/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gravitational%20Space%20Biol%20Bull%20Publ%20Am%20Soc%20Gravitational%20Space%20Biol&title=Microbial%20contamination%20of%20spacecraft&author=DL%20Pierson&volume=14&publication_year=2001&pages=1-6&pmid=11865864&)]
64. Plubell DL, Wilmarth PA, Zhao Y, Fenton AM, Minnier J, Reddy AP, Klimek J, Yang X, David LL, Pamir N (2017) Extended multiplexing of tandem mass tags (TMT) labeling reveals age and high fat diet specific proteome changes in mouse epididymal adipose tissue. Mol Cell Proteomics
    16:873–890. doi: 10.1074/mcp.M116.065524 [[DOI](https://doi.org/10.1074/mcp.M116.065524)] [[PMC free article](/articles/PMC5417827/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28325852/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Proteomics&title=Extended%20multiplexing%20of%20tandem%20mass%20tags%20(TMT)%20labeling%20reveals%20age%20and%20high%20fat%20diet%20specific%20proteome%20changes%20in%20mouse%20epididymal%20adipose%20tissue&author=DL%20Plubell&author=PA%20Wilmarth&author=Y%20Zhao&author=AM%20Fenton&author=J%20Minnier&volume=16&publication_year=2017&pages=873-890&pmid=28325852&doi=10.1074/mcp.M116.065524&)]
65. Pusztahelyi T, Klement É, Szajli E, Klem J, Miskei M, Karányi Z, Emri T, Kovács S, Orosz G, Kovács KL, Medzihradszky KF, Prade RA, Pócsi I (2011) Comparison of transcriptional and translational changes caused by long-term menadione exposure in Aspergillus nidulans. Fungal Genet Biol
    48:92–103. doi: 10.1016/j.fgb.2010.08.006 [[DOI](https://doi.org/10.1016/j.fgb.2010.08.006)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20797444/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet%20Biol&title=Comparison%20of%20transcriptional%20and%20translational%20changes%20caused%20by%20long-term%20menadione%20exposure%20in%20Aspergillus%20nidulans&author=T%20Pusztahelyi&author=%C3%89%20Klement&author=E%20Szajli&author=J%20Klem&author=M%20Miskei&volume=48&publication_year=2011&pages=92-103&pmid=20797444&doi=10.1016/j.fgb.2010.08.006&)]
66. Robinson MD, Oshlack A (2010) A scaling normalization method for differential expression analysis of RNA-seq data. Genome Biol
    11:R25. doi: 10.1186/gb-2010-11-3-r25 [[DOI](https://doi.org/10.1186/gb-2010-11-3-r25)] [[PMC free article](/articles/PMC2864565/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20196867/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=A%20scaling%20normalization%20method%20for%20differential%20expression%20analysis%20of%20RNA-seq%20data&author=MD%20Robinson&author=A%20Oshlack&volume=11&publication_year=2010&pages=R25&pmid=20196867&doi=10.1186/gb-2010-11-3-r25&)]
67. Sanchez JF, Entwistle R, Hung J-H, Yaegashi J, Jain S, Chiang Y-M, Wang CCC, Oakley BR (2011) Genome-based deletion analysis reveals the prenyl xanthone biosynthesis pathway in Aspergillus nidulans. J Am Chem Soc
    133:4010–4017. doi: 10.1021/ja1096682 [[DOI](https://doi.org/10.1021/ja1096682)] [[PMC free article](/articles/PMC3119361/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21351751/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&title=Genome-based%20deletion%20analysis%20reveals%20the%20prenyl%20xanthone%20biosynthesis%20pathway%20in%20Aspergillus%20nidulans&author=JF%20Sanchez&author=R%20Entwistle&author=J-H%20Hung&author=J%20Yaegashi&author=S%20Jain&volume=133&publication_year=2011&pages=4010-4017&pmid=21351751&doi=10.1021/ja1096682&)]
68. Sanchez JF, Somoza AD, Keller NP, Wang CCC (2012) Advances in Aspergillus secondary metabolite research in the post-genomic era. Nat Prod Rep
    29:351–371. doi: 10.1039/c2np00084a [[DOI](https://doi.org/10.1039/c2np00084a)] [[PMC free article](/articles/PMC4568942/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22228366/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Prod%20Rep&title=Advances%20in%20Aspergillus%20secondary%20metabolite%20research%20in%20the%20post-genomic%20era&author=JF%20Sanchez&author=AD%20Somoza&author=NP%20Keller&author=CCC%20Wang&volume=29&publication_year=2012&pages=351-371&pmid=22228366&doi=10.1039/c2np00084a&)]
69. Segal BH (2009) Aspergillosis. N Engl J Med
    360:1870–1884. doi: 10.1056/NEJMra0808853 [[DOI](https://doi.org/10.1056/NEJMra0808853)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19403905/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Aspergillosis&author=BH%20Segal&volume=360&publication_year=2009&pages=1870-1884&pmid=19403905&doi=10.1056/NEJMra0808853&)]
70. Sharpe RA, Bearman N, Thornton CR, Husk K, Osborne NJ (2015) Indoor fungal diversity and asthma: a meta-analysis and systematic review of risk factors. J Allergy Clin Immunol
    135:110–122. doi: 10.1016/j.jaci.2014.07.002 [[DOI](https://doi.org/10.1016/j.jaci.2014.07.002)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25159468/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Indoor%20fungal%20diversity%20and%20asthma:%20a%20meta-analysis%20and%20systematic%20review%20of%20risk%20factors&author=RA%20Sharpe&author=N%20Bearman&author=CR%20Thornton&author=K%20Husk&author=NJ%20Osborne&volume=135&publication_year=2015&pages=110-122&pmid=25159468&doi=10.1016/j.jaci.2014.07.002&)]
71. Singaravelan N, Grishkan I, Beharav A, Wakamatsu K, Ito S, Nevo E (2008) Adaptive Melanin Response of the Soil Fungus Aspergillus niger to UV Radiation Stress at “Evolution Canyon”, Mount Carmel, Israel. PLoS ONE
    3:e2993. doi: 10.1371/journal.pone.0002993 [[DOI](https://doi.org/10.1371/journal.pone.0002993)] [[PMC free article](/articles/PMC2500164/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18714346/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Adaptive%20Melanin%20Response%20of%20the%20Soil%20Fungus%20Aspergillus%20niger%20to%20UV%20Radiation%20Stress%20at%20%E2%80%9CEvolution%20Canyon%E2%80%9D,%20Mount%20Carmel,%20Israel&author=N%20Singaravelan&author=I%20Grishkan&author=A%20Beharav&author=K%20Wakamatsu&author=S%20Ito&volume=3&publication_year=2008&pmid=18714346&doi=10.1371/journal.pone.0002993&)]
72. Stajich JE, Harris T, Brunk BP, Brestelli J, Fischer S, Harb OS, Kissinger JC, Li W, Nayak V, Pinney DF, Stoeckert CJ, Roos DS (2012) FungiDB: an integrated functional genomics database for fungi. Nucleic Acids Res
    40:D675–D681. doi: 10.1093/nar/gkr918 [[DOI](https://doi.org/10.1093/nar/gkr918)] [[PMC free article](/articles/PMC3245123/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22064857/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=FungiDB:%20an%20integrated%20functional%20genomics%20database%20for%20fungi&author=JE%20Stajich&author=T%20Harris&author=BP%20Brunk&author=J%20Brestelli&author=S%20Fischer&volume=40&publication_year=2012&pages=D675-D681&pmid=22064857&doi=10.1093/nar/gkr918&)]
73. Szewczyk E, Chiang Y-M, Oakley CE, Davidson AD, Wang CCC, Oakley BR (2008) Identification and characterization of the asperthecin gene cluster of Aspergillus nidulans. Appl Environ Microbiol
    74:7607–7612. doi: 10.1128/AEM.01743-08 [[DOI](https://doi.org/10.1128/AEM.01743-08)] [[PMC free article](/articles/PMC2607171/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18978088/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Identification%20and%20characterization%20of%20the%20asperthecin%20gene%20cluster%20of%20Aspergillus%20nidulans&author=E%20Szewczyk&author=Y-M%20Chiang&author=CE%20Oakley&author=AD%20Davidson&author=CCC%20Wang&volume=74&publication_year=2008&pages=7607-7612&pmid=18978088&doi=10.1128/AEM.01743-08&)]
74. Szewczyk E, Nayak T, Oakley CE, Edgerton H, Xiong Y, Taheri-Talesh N, Osmani SA, Oakley BR, Oakley B (2006) Fusion PCR and gene targeting in Aspergillus nidulans. Nat Protoc
    1:3111–3120. doi: 10.1038/nprot.2006.405 [[DOI](https://doi.org/10.1038/nprot.2006.405)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17406574/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Fusion%20PCR%20and%20gene%20targeting%20in%20Aspergillus%20nidulans&author=E%20Szewczyk&author=T%20Nayak&author=CE%20Oakley&author=H%20Edgerton&author=Y%20Xiong&volume=1&publication_year=2006&pages=3111-3120&pmid=17406574&doi=10.1038/nprot.2006.405&)]
75. Tixador R, Richoilley G, Gasset G, Templier J, Bes J, Moatti N, Lapchine L (1985) Study of minimal inhibitory concentration of antibiotics on bacteria cultivated in vitro in space (Cytos 2 experiment). Aviat Space Environ Med
    56:748–751 [[PubMed](https://pubmed.ncbi.nlm.nih.gov/3899095/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aviat%20Space%20Environ%20Med&title=Study%20of%20minimal%20inhibitory%20concentration%20of%20antibiotics%20on%20bacteria%20cultivated%20in%20vitro%20in%20space%20(Cytos%202%20experiment)&author=R%20Tixador&author=G%20Richoilley&author=G%20Gasset&author=J%20Templier&author=J%20Bes&volume=56&publication_year=1985&pages=748-751&pmid=3899095&)]
76. Tkavc R, Matrosova VY, Grichenko OE, Gostinčar C, Volpe RP, Klimenkova P, Gaidamakova EK, Zhou CE, Stewart BJ, Lyman MG, Malfatti SA, Rubinfeld B, Courtot M, Singh J, Dalgard CL, Hamilton T, Frey KG, Gunde-Cimerman N, Dugan L, Daly MJ (2018) Prospects for fungal bioremediation of acidic radioactive waste sites: characterization and genome sequence of Rhodotorula taiwanensis MD1149. Front Microbiol
    8:. doi: 10.3389/fmicb.2017.02528 [[DOI](https://doi.org/10.3389/fmicb.2017.02528)] [[PMC free article](/articles/PMC5766836/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29375494/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&title=Prospects%20for%20fungal%20bioremediation%20of%20acidic%20radioactive%20waste%20sites:%20characterization%20and%20genome%20sequence%20of%20Rhodotorula%20taiwanensis%20MD1149&author=R%20Tkavc&author=VY%20Matrosova&author=OE%20Grichenko&author=C%20Gostin%C4%8Dar&author=RP%20Volpe&volume=8&publication_year=2018&pmid=29375494&doi=10.3389/fmicb.2017.02528&)]
77. Uhl MA, Biery M, Craig N, Johnson AD (2003) Haploinsufficiency-based large-scale forward genetic analysis of filamentous growth in the diploid human fungal pathogen C.albicans. EMBO J
    22:2668–2678. doi: 10.1093/emboj/cdg256 [[DOI](https://doi.org/10.1093/emboj/cdg256)] [[PMC free article](/articles/PMC156753/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12773383/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Haploinsufficiency-based%20large-scale%20forward%20genetic%20analysis%20of%20filamentous%20growth%20in%20the%20diploid%20human%20fungal%20pathogen%20C.albicans&author=MA%20Uhl&author=M%20Biery&author=N%20Craig&author=AD%20Johnson&volume=22&publication_year=2003&pages=2668-2678&pmid=12773383&doi=10.1093/emboj/cdg256&)]
78. Van Houdt R, Mijnendonckx K, Leys N (2012) Microbial contamination monitoring and control during human space missions. Planet Space Sci
    60:115–120. doi: 10.1016/j.pss.2011.09.001 [[DOI](https://doi.org/10.1016/j.pss.2011.09.001)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Planet%20Space%20Sci&title=Microbial%20contamination%20monitoring%20and%20control%20during%20human%20space%20missions&author=R%20Van%20Houdt&author=K%20Mijnendonckx&author=N%20Leys&volume=60&publication_year=2012&pages=115-120&doi=10.1016/j.pss.2011.09.001&)]
79. Venkateswaran K, Vaishampayan P, Cisneros J, Pierson DL, Rogers SO, Perry J (2014) International Space Station environmental microbiome - microbial inventories of ISS filter debris. Appl Microbiol Biotechnol
    98:6453–6466. doi: 10.1007/s00253-014-5650-6 [[DOI](https://doi.org/10.1007/s00253-014-5650-6)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24695826/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&title=International%20Space%20Station%20environmental%20microbiome%20-%20microbial%20inventories%20of%20ISS%20filter%20debris&author=K%20Venkateswaran&author=P%20Vaishampayan&author=J%20Cisneros&author=DL%20Pierson&author=SO%20Rogers&volume=98&publication_year=2014&pages=6453-6466&pmid=24695826&doi=10.1007/s00253-014-5650-6&)]
80. Volz PA, Dublin M (1973) Filamentous fungi exposed to spaceflight stresses including known levels of ultraviolet irradiations. Space Life Sci
    4:402–414. doi: 10.1007/BF00930352 [[DOI](https://doi.org/10.1007/BF00930352)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/4771132/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Space%20Life%20Sci&title=Filamentous%20fungi%20exposed%20to%20spaceflight%20stresses%20including%20known%20levels%20of%20ultraviolet%20irradiations&author=PA%20Volz&author=M%20Dublin&volume=4&publication_year=1973&pages=402-414&pmid=4771132&doi=10.1007/BF00930352&)]
81. Vries de RP, Visser J (2001) Aspergillus enzymes involved in degradation of plant cell wall polysaccharides. Microbiol Mol Biol Rev
    65:497–522. doi: 10.1128/MMBR.65.4.497522.2001 [[DOI](https://doi.org/10.1128/MMBR.65.4.497522.2001)] [[PMC free article](/articles/PMC99039/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11729262/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol%20Mol%20Biol%20Rev&title=Aspergillus%20enzymes%20involved%20in%20degradation%20of%20plant%20cell%20wall%20polysaccharides&author=RP%20Vries%20de&author=J%20Visser&volume=65&publication_year=2001&pages=497-522&pmid=11729262&doi=10.1128/MMBR.65.4.497522.2001&)]
82. Wilson JW, Ott CM, Quick L, Davis R, zu Bentrup KH, Crabbé A, Richter E, Sarker S, Barrila J, Porwollik S, Cheng P, McClelland M, Tsaprailis G, Radabaugh T, Hunt A, Shah M, Nelman-Gonzalez M, Hing S, Parra M, Dumars P, Norwood K, Bober R, Devich J, Ruggles A, CdeBaca A, Narayan S, Benjamin J, Goulart C, Rupert M, Catella L, Schurr MJ, Buchanan K, Morici L, McCracken J, Porter MD, Pierson DL, Smith SM, Mergeay M, Leys N, Stefanyshyn-Piper HM, Gorie D, Nickerson CA (2008) Media ion composition controls regulatory and virulence response of Salmonella in spaceflight. PLoS ONE
    3:e3923. doi: 10.1371/journal.pone.0003923 [[DOI](https://doi.org/10.1371/journal.pone.0003923)] [[PMC free article](/articles/PMC2592540/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19079590/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Media%20ion%20composition%20controls%20regulatory%20and%20virulence%20response%20of%20Salmonella%20in%20spaceflight&author=JW%20Wilson&author=CM%20Ott&author=L%20Quick&author=R%20Davis&author=KH%20zu%20Bentrup&volume=3&publication_year=2008&pmid=19079590&doi=10.1371/journal.pone.0003923&)]
83. Wu J, Wang M, Zhou L, Yu D (2016) Small heat shock proteins, phylogeny in filamentous fungi and expression analyses in Aspergillus nidulans. Gene
    575:675–679. doi: 10.1016/j.gene.2015.09.044 [[DOI](https://doi.org/10.1016/j.gene.2015.09.044)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26403724/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Small%20heat%20shock%20proteins,%20phylogeny%20in%20filamentous%20fungi%20and%20expression%20analyses%20in%20Aspergillus%20nidulans&author=J%20Wu&author=M%20Wang&author=L%20Zhou&author=D%20Yu&volume=575&publication_year=2016&pages=675-679&pmid=26403724&doi=10.1016/j.gene.2015.09.044&)]
84. Yaegashi J, Oakley BR, Wang CCC (2014) Recent advances in genome mining of secondary metabolite biosynthetic gene clusters and the development of heterologous expression systems in Aspergillus nidulans. J Ind Microbiol Biotechnol
    41:433–442. doi: 10.1007/s10295-013-1386-z [[DOI](https://doi.org/10.1007/s10295-013-1386-z)] [[PMC free article](/articles/PMC4324552/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24342965/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Ind%20Microbiol%20Biotechnol&title=Recent%20advances%20in%20genome%20mining%20of%20secondary%20metabolite%20biosynthetic%20gene%20clusters%20and%20the%20development%20of%20heterologous%20expression%20systems%20in%20Aspergillus%20nidulans&author=J%20Yaegashi&author=BR%20Oakley&author=CCC%20Wang&volume=41&publication_year=2014&pages=433-442&pmid=24342965&doi=10.1007/s10295-013-1386-z&)]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

Sup\_1

[NIHMS1065202-supplement-Sup\_1.pdf](/articles/instance/9098231/bin/NIHMS1065202-supplement-Sup_1.pdf) (35.2MB, pdf)

### Data Availability Statement

Raw WGS data for ISS-grown *A. nidulans* strains and ground-grown counterparts are available in the NCBI SRA under accession numbers SRR7724113- SRR7724124 and BioProject accession number PRJNA486827. Proteomics data is accessible through the ProteomeXchange Consortium via PRIDE with the dataset identifier PXD010778.