<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10386755/"/><title>Sequencing the Genomes of the First Terrestrial Fungal Lineages: What Have We Learned? - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Microorganisms</button></div>. 2023 Jul 18;11(7):1830. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/microorganisms11071830" rel="noopener noreferrer" target="_blank">10.3390/microorganisms11071830</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Sequencing the Genomes of the First Terrestrial Fungal Lineages: What Have We Learned?</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gryganskyi%20AP%22%5BAuthor%5D"><span class="name western">Andrii P Gryganskyi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Andrii P Gryganskyi</span></h3>
<div class="p">
<sup>1</sup>Division of Biological &amp; Nanoscale Technologies, UES, Inc., Dayton, OH 45432, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gryganskyi%20AP%22%5BAuthor%5D"><span class="name western">Andrii P Gryganskyi</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Golan%20J%22%5BAuthor%5D"><span class="name western">Jacob Golan</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jacob Golan</span></h3>
<div class="p">
<sup>2</sup>Department of Botany, University of Wisconsin-Madison, Madison, WI 53706, USA; jjg384@nyu.edu</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Golan%20J%22%5BAuthor%5D"><span class="name western">Jacob Golan</span></a>
</div>
</div>
<sup>2</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Muszewska%20A%22%5BAuthor%5D"><span class="name western">Anna Muszewska</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Anna Muszewska</span></h3>
<div class="p">
<sup>3</sup>Institute of Biochemistry &amp; Biophysics, Polish Academy of Sciences, 01-224 Warsaw, Poland; ania.muszewska@gmail.com</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Muszewska%20A%22%5BAuthor%5D"><span class="name western">Anna Muszewska</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Idnurm%20A%22%5BAuthor%5D"><span class="name western">Alexander Idnurm</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Alexander Idnurm</span></h3>
<div class="p">
<sup>4</sup>School of BioSciences, University of Melbourne, Parkville, VIC 3010, Australia; alexander.idnurm@unimelb.edu.au</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Idnurm%20A%22%5BAuthor%5D"><span class="name western">Alexander Idnurm</span></a>
</div>
</div>
<sup>4</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dolatabadi%20S%22%5BAuthor%5D"><span class="name western">Somayeh Dolatabadi</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Somayeh Dolatabadi</span></h3>
<div class="p">
<sup>5</sup>Biology Department, Hakim Sabzevari University, Sabzevar 96179-76487, Iran; somayeh99@gmail.com</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dolatabadi%20S%22%5BAuthor%5D"><span class="name western">Somayeh Dolatabadi</span></a>
</div>
</div>
<sup>5</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mondo%20SJ%22%5BAuthor%5D"><span class="name western">Stephen J Mondo</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Stephen J Mondo</span></h3>
<div class="p">
<sup>6</sup>U.S. Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA 94720, USA; sjmondo@lbl.gov (S.J.M.); ivgrigoriev@lbl.gov (I.V.G.)</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mondo%20SJ%22%5BAuthor%5D"><span class="name western">Stephen J Mondo</span></a>
</div>
</div>
<sup>6</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kutovenko%20VB%22%5BAuthor%5D"><span class="name western">Vira B Kutovenko</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Vira B Kutovenko</span></h3>
<div class="p">
<sup>7</sup>Department of Agrobiology, National University of Life &amp; Environmental Sciences, 03041 Kyiv, Ukraine; virakutovenko@gmail.com (V.B.K.); </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kutovenko%20VB%22%5BAuthor%5D"><span class="name western">Vira B Kutovenko</span></a>
</div>
</div>
<sup>7</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kutovenko%20VO%22%5BAuthor%5D"><span class="name western">Volodymyr O Kutovenko</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Volodymyr O Kutovenko</span></h3>
<div class="p">
<sup>7</sup>Department of Agrobiology, National University of Life &amp; Environmental Sciences, 03041 Kyiv, Ukraine; virakutovenko@gmail.com (V.B.K.); </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kutovenko%20VO%22%5BAuthor%5D"><span class="name western">Volodymyr O Kutovenko</span></a>
</div>
</div>
<sup>7</sup>, <a aria-describedby="id9" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gajdeczka%20MT%22%5BAuthor%5D"><span class="name western">Michael T Gajdeczka</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Michael T Gajdeczka</span></h3>
<div class="p">
<sup>8</sup>LF Lambert Spawn Co., Coatesville, PA 19320, USA; mgajdeczka@gmail.com</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gajdeczka%20MT%22%5BAuthor%5D"><span class="name western">Michael T Gajdeczka</span></a>
</div>
</div>
<sup>8</sup>, <a aria-describedby="id10" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anishchenko%20IM%22%5BAuthor%5D"><span class="name western">Iryna M Anishchenko</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Iryna M Anishchenko</span></h3>
<div class="p">
<sup>9</sup>MG Kholodny Institute of Botany, National Academy of Sciences, 01030 Kyiv, Ukraine; ira_anishchenko@hotmail.com</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anishchenko%20IM%22%5BAuthor%5D"><span class="name western">Iryna M Anishchenko</span></a>
</div>
</div>
<sup>9</sup>, <a aria-describedby="id11" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pawlowska%20J%22%5BAuthor%5D"><span class="name western">Julia Pawlowska</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Julia Pawlowska</span></h3>
<div class="p">
<sup>10</sup>Institute of Evolutionary Biology, Faculty of Biology, Biological &amp; Chemical Research Centre, University of Warsaw, 02-089 Warsaw, Poland; julia.z.pawlowska@uw.edu.pl</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pawlowska%20J%22%5BAuthor%5D"><span class="name western">Julia Pawlowska</span></a>
</div>
</div>
<sup>10</sup>, <a aria-describedby="id12" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20NV%22%5BAuthor%5D"><span class="name western">Ngoc Vinh Tran</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Ngoc Vinh Tran</span></h3>
<div class="p">
<sup>11</sup>Plant Pathology Department, University of Florida, Gainesville, FL 32611, USA; tran@bio.uni-frankfurt.de (N.V.T.); gbenny@ufl.edu (G.L.B.); trufflesmith@gmail.com (M.E.S.)</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20NV%22%5BAuthor%5D"><span class="name western">Ngoc Vinh Tran</span></a>
</div>
</div>
<sup>11</sup>, <a aria-describedby="id13" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ebersberger%20I%22%5BAuthor%5D"><span class="name western">Ingo Ebersberger</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Ingo Ebersberger</span></h3>
<div class="p">
<sup>12</sup>Leibniz Institute for Natural Product Research &amp; Infection Biology, 07745 Jena, Germany; ebersberger@bio.uni-frankfurt.de (I.E.); kerstin.voigt@leibniz-hki.de (K.V.)</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ebersberger%20I%22%5BAuthor%5D"><span class="name western">Ingo Ebersberger</span></a>
</div>
</div>
<sup>12</sup>, <a aria-describedby="id14" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Voigt%20K%22%5BAuthor%5D"><span class="name western">Kerstin Voigt</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Kerstin Voigt</span></h3>
<div class="p">
<sup>12</sup>Leibniz Institute for Natural Product Research &amp; Infection Biology, 07745 Jena, Germany; ebersberger@bio.uni-frankfurt.de (I.E.); kerstin.voigt@leibniz-hki.de (K.V.)</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Voigt%20K%22%5BAuthor%5D"><span class="name western">Kerstin Voigt</span></a>
</div>
</div>
<sup>12</sup>, <a aria-describedby="id15" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D"><span class="name western">Yan Wang</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Yan Wang</span></h3>
<div class="p">
<sup>13</sup>Department of Ecology &amp; Evolutionary Biology, University of Toronto, Toronto, ON M5S 1A1, Canada; yanxw.wang@utoronto.ca</div>
<div class="p">
<sup>14</sup>Department of Biological Sciences, University of Toronto Scarborough, Toronto, ON M1C 1A4, Canada</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D"><span class="name western">Yan Wang</span></a>
</div>
</div>
<sup>13,</sup><sup>14</sup>, <a aria-describedby="id16" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chang%20Y%22%5BAuthor%5D"><span class="name western">Ying Chang</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Ying Chang</span></h3>
<div class="p">
<sup>15</sup>Department of Biological Sciences, National University of Singapore, Singapore 119077, Singapore; ying.chang@yale-nus.edu.sg</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chang%20Y%22%5BAuthor%5D"><span class="name western">Ying Chang</span></a>
</div>
</div>
<sup>15</sup>, <a aria-describedby="id17" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pawlowska%20TE%22%5BAuthor%5D"><span class="name western">Teresa E Pawlowska</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Teresa E Pawlowska</span></h3>
<div class="p">
<sup>16</sup>School of Integrative Plant Science, Cornell University, Ithaca, NY 14850, USA; tep8@cornell.edu (T.E.P.); nr299@cornell.edu (N.R.)</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pawlowska%20TE%22%5BAuthor%5D"><span class="name western">Teresa E Pawlowska</span></a>
</div>
</div>
<sup>16</sup>, <a aria-describedby="id18" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heitman%20J%22%5BAuthor%5D"><span class="name western">Joseph Heitman</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Joseph Heitman</span></h3>
<div class="p">
<sup>17</sup>Department of Molecular Genetics &amp; Microbiology, Duke University School of Medicine, Durham, NC 27710, USA; heitm001@duke.edu</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heitman%20J%22%5BAuthor%5D"><span class="name western">Joseph Heitman</span></a>
</div>
</div>
<sup>17</sup>, <a aria-describedby="id19" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vilgalys%20R%22%5BAuthor%5D"><span class="name western">Rytas Vilgalys</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Rytas Vilgalys</span></h3>
<div class="p">
<sup>18</sup>Biology Department, Duke University, Durham, NC 27708, USA; fungi@duke.edu</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vilgalys%20R%22%5BAuthor%5D"><span class="name western">Rytas Vilgalys</span></a>
</div>
</div>
<sup>18</sup>, <a aria-describedby="id20" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bonito%20G%22%5BAuthor%5D"><span class="name western">Gregory Bonito</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Gregory Bonito</span></h3>
<div class="p">
<sup>19</sup>Department of Plant, Soil &amp; Microbial Sciences, Michigan State University, East Lansing, MI 48824, USA; bonito@msu.edu</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bonito%20G%22%5BAuthor%5D"><span class="name western">Gregory Bonito</span></a>
</div>
</div>
<sup>19</sup>, <a aria-describedby="id21" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Benny%20GL%22%5BAuthor%5D"><span class="name western">Gerald L Benny</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Gerald L Benny</span></h3>
<div class="p">
<sup>11</sup>Plant Pathology Department, University of Florida, Gainesville, FL 32611, USA; tran@bio.uni-frankfurt.de (N.V.T.); gbenny@ufl.edu (G.L.B.); trufflesmith@gmail.com (M.E.S.)</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Benny%20GL%22%5BAuthor%5D"><span class="name western">Gerald L Benny</span></a>
</div>
</div>
<sup>11</sup>, <a aria-describedby="id22" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20ME%22%5BAuthor%5D"><span class="name western">Matthew E Smith</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Matthew E Smith</span></h3>
<div class="p">
<sup>11</sup>Plant Pathology Department, University of Florida, Gainesville, FL 32611, USA; tran@bio.uni-frankfurt.de (N.V.T.); gbenny@ufl.edu (G.L.B.); trufflesmith@gmail.com (M.E.S.)</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20ME%22%5BAuthor%5D"><span class="name western">Matthew E Smith</span></a>
</div>
</div>
<sup>11</sup>, <a aria-describedby="id23" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reynolds%20N%22%5BAuthor%5D"><span class="name western">Nicole Reynolds</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Nicole Reynolds</span></h3>
<div class="p">
<sup>16</sup>School of Integrative Plant Science, Cornell University, Ithaca, NY 14850, USA; tep8@cornell.edu (T.E.P.); nr299@cornell.edu (N.R.)</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reynolds%20N%22%5BAuthor%5D"><span class="name western">Nicole Reynolds</span></a>
</div>
</div>
<sup>16</sup>, <a aria-describedby="id24" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22James%20TY%22%5BAuthor%5D"><span class="name western">Timothy Y James</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Timothy Y James</span></h3>
<div class="p">
<sup>20</sup>Department of Ecology &amp; Evolutionary Biology, University of Michigan, Ann Arbor, MI 48109, USA; tyjames@umich.edu</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22James%20TY%22%5BAuthor%5D"><span class="name western">Timothy Y James</span></a>
</div>
</div>
<sup>20</sup>, <a aria-describedby="id25" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grigoriev%20IV%22%5BAuthor%5D"><span class="name western">Igor V Grigoriev</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Igor V Grigoriev</span></h3>
<div class="p">
<sup>6</sup>U.S. Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA 94720, USA; sjmondo@lbl.gov (S.J.M.); ivgrigoriev@lbl.gov (I.V.G.)</div>
<div class="p">
<sup>21</sup>Department of Plant &amp; Microbial Biology, University of California, Berkeley, CA 94720, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grigoriev%20IV%22%5BAuthor%5D"><span class="name western">Igor V Grigoriev</span></a>
</div>
</div>
<sup>6,</sup><sup>21</sup>, <a aria-describedby="id26" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Spatafora%20JW%22%5BAuthor%5D"><span class="name western">Joseph W Spatafora</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Joseph W Spatafora</span></h3>
<div class="p">
<sup>22</sup>Department of Botany &amp; Plant Pathology, Oregon State University, Corvallis, OR 97331, USA; spatafoj@science.oregonstate.edu</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Spatafora%20JW%22%5BAuthor%5D"><span class="name western">Joseph W Spatafora</span></a>
</div>
</div>
<sup>22</sup>, <a aria-describedby="id27" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stajich%20JE%22%5BAuthor%5D"><span class="name western">Jason E Stajich</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Jason E Stajich</span></h3>
<div class="p">
<sup>23</sup>Department of Microbiology &amp; Plant Pathology, University of California, Riverside, CA 93106, USA; jason.stajich@ucr.edu</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stajich%20JE%22%5BAuthor%5D"><span class="name western">Jason E Stajich</span></a>
</div>
</div>
<sup>23</sup>
</div>
<div class="cg p">Editor: <span class="name western">Gary A Strobel</span>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="af1-microorganisms-11-01830">
<sup>1</sup>Division of Biological &amp; Nanoscale Technologies, UES, Inc., Dayton, OH 45432, USA</div>
<div id="af2-microorganisms-11-01830">
<sup>2</sup>Department of Botany, University of Wisconsin-Madison, Madison, WI 53706, USA; jjg384@nyu.edu</div>
<div id="af3-microorganisms-11-01830">
<sup>3</sup>Institute of Biochemistry &amp; Biophysics, Polish Academy of Sciences, 01-224 Warsaw, Poland; ania.muszewska@gmail.com</div>
<div id="af4-microorganisms-11-01830">
<sup>4</sup>School of BioSciences, University of Melbourne, Parkville, VIC 3010, Australia; alexander.idnurm@unimelb.edu.au</div>
<div id="af5-microorganisms-11-01830">
<sup>5</sup>Biology Department, Hakim Sabzevari University, Sabzevar 96179-76487, Iran; somayeh99@gmail.com</div>
<div id="af6-microorganisms-11-01830">
<sup>6</sup>U.S. Department of Energy Joint Genome Institute, Lawrence Berkeley National Laboratory, Berkeley, CA 94720, USA; sjmondo@lbl.gov (S.J.M.); ivgrigoriev@lbl.gov (I.V.G.)</div>
<div id="af7-microorganisms-11-01830">
<sup>7</sup>Department of Agrobiology, National University of Life &amp; Environmental Sciences, 03041 Kyiv, Ukraine; virakutovenko@gmail.com (V.B.K.); </div>
<div id="af8-microorganisms-11-01830">
<sup>8</sup>LF Lambert Spawn Co., Coatesville, PA 19320, USA; mgajdeczka@gmail.com</div>
<div id="af9-microorganisms-11-01830">
<sup>9</sup>MG Kholodny Institute of Botany, National Academy of Sciences, 01030 Kyiv, Ukraine; ira_anishchenko@hotmail.com</div>
<div id="af10-microorganisms-11-01830">
<sup>10</sup>Institute of Evolutionary Biology, Faculty of Biology, Biological &amp; Chemical Research Centre, University of Warsaw, 02-089 Warsaw, Poland; julia.z.pawlowska@uw.edu.pl</div>
<div id="af11-microorganisms-11-01830">
<sup>11</sup>Plant Pathology Department, University of Florida, Gainesville, FL 32611, USA; tran@bio.uni-frankfurt.de (N.V.T.); gbenny@ufl.edu (G.L.B.); trufflesmith@gmail.com (M.E.S.)</div>
<div id="af12-microorganisms-11-01830">
<sup>12</sup>Leibniz Institute for Natural Product Research &amp; Infection Biology, 07745 Jena, Germany; ebersberger@bio.uni-frankfurt.de (I.E.); kerstin.voigt@leibniz-hki.de (K.V.)</div>
<div id="af13-microorganisms-11-01830">
<sup>13</sup>Department of Ecology &amp; Evolutionary Biology, University of Toronto, Toronto, ON M5S 1A1, Canada; yanxw.wang@utoronto.ca</div>
<div id="af14-microorganisms-11-01830">
<sup>14</sup>Department of Biological Sciences, University of Toronto Scarborough, Toronto, ON M1C 1A4, Canada</div>
<div id="af15-microorganisms-11-01830">
<sup>15</sup>Department of Biological Sciences, National University of Singapore, Singapore 119077, Singapore; ying.chang@yale-nus.edu.sg</div>
<div id="af16-microorganisms-11-01830">
<sup>16</sup>School of Integrative Plant Science, Cornell University, Ithaca, NY 14850, USA; tep8@cornell.edu (T.E.P.); nr299@cornell.edu (N.R.)</div>
<div id="af17-microorganisms-11-01830">
<sup>17</sup>Department of Molecular Genetics &amp; Microbiology, Duke University School of Medicine, Durham, NC 27710, USA; heitm001@duke.edu</div>
<div id="af18-microorganisms-11-01830">
<sup>18</sup>Biology Department, Duke University, Durham, NC 27708, USA; fungi@duke.edu</div>
<div id="af19-microorganisms-11-01830">
<sup>19</sup>Department of Plant, Soil &amp; Microbial Sciences, Michigan State University, East Lansing, MI 48824, USA; bonito@msu.edu</div>
<div id="af20-microorganisms-11-01830">
<sup>20</sup>Department of Ecology &amp; Evolutionary Biology, University of Michigan, Ann Arbor, MI 48109, USA; tyjames@umich.edu</div>
<div id="af21-microorganisms-11-01830">
<sup>21</sup>Department of Plant &amp; Microbial Biology, University of California, Berkeley, CA 94720, USA</div>
<div id="af22-microorganisms-11-01830">
<sup>22</sup>Department of Botany &amp; Plant Pathology, Oregon State University, Corvallis, OR 97331, USA; spatafoj@science.oregonstate.edu</div>
<div id="af23-microorganisms-11-01830">
<sup>23</sup>Department of Microbiology &amp; Plant Pathology, University of California, Riverside, CA 93106, USA; jason.stajich@ucr.edu</div>
<div class="author-notes p"><div class="fn" id="c1-microorganisms-11-01830">
<sup>*</sup><p class="display-inline">Correspondence: <span>agryganskyi@ues.com</span></p>
</div></div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Gary A Strobel</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2023 Jun 15; Revised 2023 Jul 13; Accepted 2023 Jul 16; Collection date 2023 Jul.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© 2023 by the authors.</div>
<p>Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>).</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC10386755  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37513002/">37513002</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The first genome sequenced of a eukaryotic organism was for <em>Saccharomyces cerevisiae</em>, as reported in 1996, but it was more than 10 years before any of the zygomycete fungi, which are the early-diverging terrestrial fungi currently placed in the phyla <em>Mucoromycota</em> and <em>Zoopagomycota</em>, were sequenced. The genome for <em>Rhizopus delemar</em> was completed in 2008; currently, more than 1000 zygomycete genomes have been sequenced. Genomic data from these early-diverging terrestrial fungi revealed deep phylogenetic separation of the two major clades—primarily plant—associated saprotrophic and mycorrhizal <em>Mucoromycota</em> versus the primarily mycoparasitic or animal-associated parasites and commensals in the <em>Zoopagomycota</em>. Genomic studies provide many valuable insights into how these fungi evolved in response to the challenges of living on land, including adaptations to sensing light and gravity, development of hyphal growth, and co-existence with the first terrestrial plants. Genome sequence data have facilitated studies of genome architecture, including a history of genome duplications and horizontal gene transfer events, distribution and organization of mating type loci, rDNA genes and transposable elements, methylation processes, and genes useful for various industrial applications. Pathogenicity genes and specialized secondary metabolites have also been detected in soil saprobes and pathogenic fungi. Novel endosymbiotic bacteria and viruses have been discovered during several zygomycete genome projects. Overall, genomic information has helped to resolve a plethora of research questions, from the placement of zygomycetes on the evolutionary tree of life and in natural ecosystems, to the applied biotechnological and medical questions.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> fungal evolution, ecological relevance, saprotrophs, pathogens, model organisms</p></section></section><section id="sec1-microorganisms-11-01830"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The first “land” fungi retained their flagella for motility and remained adapted to life in water, and several diverging lineages switched to multicellularity while exploiting new environments [<a aria-describedby="B1-microorganisms-11-01830" class="usa-link" href="#B1-microorganisms-11-01830">1</a>]. A hallmark of the process of adapting to terrestrial life was the loss of the flagellum, which occurred independently several times during fungal evolution, giving rise to fungi with a hyphal growth form [<a aria-describedby="B2-microorganisms-11-01830" class="usa-link" href="#B2-microorganisms-11-01830">2</a>]. Prominent among early-divergent fungi is a group of lineages historically known as the zygomycetes, which are unified by similarities in reproductive biology (e.g., the formation of sexual zygospores via undifferentiated gametangia) and mostly do not form macroscopic fruiting bodies, making them more difficult to study than <em>Dikarya</em> [<a aria-describedby="B3-microorganisms-11-01830" class="usa-link" href="#B3-microorganisms-11-01830">3</a>]. Zygomycetes have also been challenging to study because many species are involved in obligate symbiotic interactions and have not been successfully grown in pure culture [<a aria-describedby="B4-microorganisms-11-01830" class="usa-link" href="#B4-microorganisms-11-01830">4</a>]. However, subsequent molecular studies have shown that the zygomycete fungi are paraphyletic [<a aria-describedby="B5-microorganisms-11-01830" class="usa-link" href="#B5-microorganisms-11-01830">5</a>]. Today, zygomycetes are recognized as two independent monophyletic groups divided into six subphyla [<a aria-describedby="B6-microorganisms-11-01830" class="usa-link" href="#B6-microorganisms-11-01830">6</a>]. Terrestrial environments posed novel challenges to early fungi, and the adaptive solutions that these lineages evolved can today be gleaned from the vast library of genes contained within their genomes [<a aria-describedby="B7-microorganisms-11-01830" class="usa-link" href="#B7-microorganisms-11-01830">7</a>,<a aria-describedby="B8-microorganisms-11-01830" class="usa-link" href="#B8-microorganisms-11-01830">8</a>]. While these first land colonizers are now extinct, many of their features are preserved in the genomes of extant zygomycetes. One instance, in which whole-genome data have proven useful, is in understanding the origin and evolution of fungi. About 450 million years ago, combined activities of fungi and plants placed the terrestrial world on a trajectory toward its current state, setting the stage for subsequent evolutionary transitions. Primitive mycorrhiza-like relationships and other forms of fungal–plant associations helped facilitate the successful colonization of land by the first terrestrial plants, and the genes responsible for such interactions are still preserved in fungal genomes [<a aria-describedby="B9-microorganisms-11-01830" class="usa-link" href="#B9-microorganisms-11-01830">9</a>]. Hence, zygomycete genomes have and will continue to play an integral role in our current understanding of the initial transition of organisms to land [<a aria-describedby="B10-microorganisms-11-01830" class="usa-link" href="#B10-microorganisms-11-01830">10</a>].</p>
<p>The number of sequenced fungal genomes continues to grow exponentially, and genomic data have become widely and freely accessible. While most sequenced genomes are from <em>Ascomycota</em> and <em>Basidiomycota</em> (<em>Dikarya</em>) species, there have been concerted efforts to sequence genomes of early-diverging and underrepresented lineages [<a aria-describedby="B11-microorganisms-11-01830" class="usa-link" href="#B11-microorganisms-11-01830">11</a>,<a aria-describedby="B12-microorganisms-11-01830" class="usa-link" href="#B12-microorganisms-11-01830">12</a>,<a aria-describedby="B13-microorganisms-11-01830" class="usa-link" href="#B13-microorganisms-11-01830">13</a>,<a aria-describedby="B14-microorganisms-11-01830" class="usa-link" href="#B14-microorganisms-11-01830">14</a>,<a aria-describedby="B15-microorganisms-11-01830" class="usa-link" href="#B15-microorganisms-11-01830">15</a>]. Since 2014, the number of zygomycete genomes has grown from a dozen [<a aria-describedby="B16-microorganisms-11-01830" class="usa-link" href="#B16-microorganisms-11-01830">16</a>] to nearly a thousand. However, 70% of these genomes are still in the draft stage, and not all of them are available. While the number of species with sequenced genomes for all organisms is on the rise, the available pool of zygomycete genomes is exceptional due to its diversity, i.e., one-third of ca. 1000 known zygomycete species have already been sequenced (<a class="usa-link" href="#app1-microorganisms-11-01830">Supplementary Table S1</a> [<a aria-describedby="B17-microorganisms-11-01830" class="usa-link" href="#B17-microorganisms-11-01830">17</a>]).</p>
<p>The sequencing efforts have not been uniform among different groups of <em>Mucoromycota</em> and <em>Zoopagomycota</em>; for most species, only a single representative genome is available. While several major clades lack a single published genome (e.g., <em>Eryniopsis</em> and <em>Neozygites</em> in <em>Entomophthoromycotina</em>, and several genera in <em>Zoopagomycotina</em>), numerous isolates of several other species have been sequenced (<a class="usa-link" href="#microorganisms-11-01830-f001">Figure 1</a>). This discrepancy can be largely explained by practical interests since focal species are sometimes chosen for their relevance to human health or biotechnology [<a aria-describedby="B18-microorganisms-11-01830" class="usa-link" href="#B18-microorganisms-11-01830">18</a>]. In addition, some taxa are common in culture collections whereas other taxa are challenging to grow in vitro or presumably uncultivable [<a aria-describedby="B4-microorganisms-11-01830" class="usa-link" href="#B4-microorganisms-11-01830">4</a>,<a aria-describedby="B11-microorganisms-11-01830" class="usa-link" href="#B11-microorganisms-11-01830">11</a>]. The goal of this review is to (1) provide background into the methods that have been used to sequence zygomycete genomes and which genomes have been sequenced, and (2) consider insights into the genomic architecture, gene content, and evolutionary history gained from these data.</p>
<figure class="fig xbox font-sm" id="microorganisms-11-01830-f001"><h3 class="obj_head">Figure 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10386755_microorganisms-11-01830-g001.jpg" target="_blank"><img alt="Figure 1" class="graphic zoom-in" height="958" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f82a/10386755/800cce7a2c76/microorganisms-11-01830-g001.jpg" width="759"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/microorganisms-11-01830-f001/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Distribution of sequenced genomes among genera. Different shades of green and brown—<em>Mucoromycota</em>; purple and blue—<em>Zoopagomycota</em>.</p></figcaption></figure></section><section id="sec2-microorganisms-11-01830"><h2 class="pmc_sec_title">2. Sequencing the Zygomycetes</h2>
<p>Axenic cultures remain the preferred and most straightforward method of obtaining high-molecular-weight genomic DNA for sequencing projects [<a aria-describedby="B19-microorganisms-11-01830" class="usa-link" href="#B19-microorganisms-11-01830">19</a>]. Typically, the easier a species is to grow in culture, the easier it is to obtain pure, unfragmented DNA of high quality. In some cases, however, DNA from a single cell or even nuclei can be extracted and sequenced. This technique has been especially successful for the mycoparasites or fungal parasites of animals (e.g., <em>Piptocephalis</em>, <em>Stylopage</em>, and <em>Zoopage</em> [<a aria-describedby="B11-microorganisms-11-01830" class="usa-link" href="#B11-microorganisms-11-01830">11</a>,<a aria-describedby="B20-microorganisms-11-01830" class="usa-link" href="#B20-microorganisms-11-01830">20</a>,<a aria-describedby="B21-microorganisms-11-01830" class="usa-link" href="#B21-microorganisms-11-01830">21</a>]) or for resolving questions on genetic diversity of the nuclei in <em>Glomeromycotina</em>, such as <em>Rhizophagus</em> [<a aria-describedby="B22-microorganisms-11-01830" class="usa-link" href="#B22-microorganisms-11-01830">22</a>]. When neither culturing nor single-cell genomics is possible, fungal DNA can be extracted along with that of the host organism. The two genomes can be distinguished bioinformatically (e.g., in the case of <em>Entomopthorales</em> pathogens of insects [<a aria-describedby="B23-microorganisms-11-01830" class="usa-link" href="#B23-microorganisms-11-01830">23</a>]). In rare cases, high-quality tissue from field-collected fruiting bodies can be used to generate genomes (e.g., <em>Endogone</em>, <em>Jimgerdemannia</em>, and <em>Modicella</em> [<a aria-describedby="B12-microorganisms-11-01830" class="usa-link" href="#B12-microorganisms-11-01830">12</a>,<a aria-describedby="B14-microorganisms-11-01830" class="usa-link" href="#B14-microorganisms-11-01830">14</a>,<a aria-describedby="B24-microorganisms-11-01830" class="usa-link" href="#B24-microorganisms-11-01830">24</a>]).</p>
<p>The diversity of DNA sequencing technologies allows researchers to select the tools that are most appropriate for addressing specific questions but also requires setting minimal reporting standards for each type of sequencing technology. During sequence processing, genomic DNA is cut and sequenced in length spans from 100–300 base pairs (short reads, Illumina) to 1000–60,000 base pairs (long reads, PacBio, or MinION). Illumina short-read sequencing remains the dominant technology used to generate fungal genomes, while long-read technologies such as PacBio and Oxford nanopore are being used with increased frequency, especially for larger genomes (<a class="usa-link" href="#app1-microorganisms-11-01830">Supplementary Table S1</a>). Illumina sequencing generally provides the greatest sequencing quality and accuracy, although millions of short raw reads (150–300 bp) are produced; as a result, genome assembly can be challenging especially with genomes featuring large amounts of repetitive DNA. Assembly quality might be adjusted by combining the use of short reads from Illumina with long reads from another sequencing platform. The use of longer reads (up to 60,000 bp) facilitates read alignment and/or mapping, often achieving the length of the chromosome [<a aria-describedby="B25-microorganisms-11-01830" class="usa-link" href="#B25-microorganisms-11-01830">25</a>] and allows precise resolution of repetitive sequences of rDNA [<a aria-describedby="B26-microorganisms-11-01830" class="usa-link" href="#B26-microorganisms-11-01830">26</a>]. However, PacBio and other long-read technologies typically demand higher quality and quantity of DNA than Illumina. On the other hand, these technologies are changing rapidly as different companies compete for the market share and make efforts to increase the number and length of reads and to maintain and improve sequence accuracy [<a aria-describedby="B27-microorganisms-11-01830" class="usa-link" href="#B27-microorganisms-11-01830">27</a>].</p>
<p>The majority of zygomycete genomes are assembled de novo, even when assemblies of closely related species are already available. Choice of assembly software usually depends on the data type. For example, AllPaths-LG works best for assembling Illumina short-read sequences, while Falcon is preferred for longer PacBio reads (<a class="usa-link" href="#app1-microorganisms-11-01830">Supplementary Table S1</a>) [<a aria-describedby="B28-microorganisms-11-01830" class="usa-link" href="#B28-microorganisms-11-01830">28</a>]. The most widely used genome assemblers for zygomycetes include gsAssembler (Roche Diagnostics, Indianapolis, IN, USA), Newbler [<a aria-describedby="B18-microorganisms-11-01830" class="usa-link" href="#B18-microorganisms-11-01830">18</a>], Velvet [<a aria-describedby="B29-microorganisms-11-01830" class="usa-link" href="#B29-microorganisms-11-01830">29</a>], Arachne, Celera [<a aria-describedby="B30-microorganisms-11-01830" class="usa-link" href="#B30-microorganisms-11-01830">30</a>], HGAP [<a aria-describedby="B31-microorganisms-11-01830" class="usa-link" href="#B31-microorganisms-11-01830">31</a>], JGI Jazz (especially among earlier assemblies from 2001–2014) [<a aria-describedby="B32-microorganisms-11-01830" class="usa-link" href="#B32-microorganisms-11-01830">32</a>], Ray [<a aria-describedby="B33-microorganisms-11-01830" class="usa-link" href="#B33-microorganisms-11-01830">33</a>], CLC Genomics Workbench 20.0 (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://digitalinsights.qiagen.com" rel="noopener noreferrer" target="_blank">https://digitalinsights.qiagen.com</a>, accessed on 10 July 2023), MaSuRCA [<a aria-describedby="B34-microorganisms-11-01830" class="usa-link" href="#B34-microorganisms-11-01830">34</a>], Falcon, AllPaths-LG [<a aria-describedby="B35-microorganisms-11-01830" class="usa-link" href="#B35-microorganisms-11-01830">35</a>], SOAPdenovo [<a aria-describedby="B36-microorganisms-11-01830" class="usa-link" href="#B36-microorganisms-11-01830">36</a>], SPAdes [<a aria-describedby="B37-microorganisms-11-01830" class="usa-link" href="#B37-microorganisms-11-01830">37</a>], SOLiD [<a aria-describedby="B38-microorganisms-11-01830" class="usa-link" href="#B38-microorganisms-11-01830">38</a>], Canu [<a aria-describedby="B39-microorganisms-11-01830" class="usa-link" href="#B39-microorganisms-11-01830">39</a>], FinisherSC [<a aria-describedby="B40-microorganisms-11-01830" class="usa-link" href="#B40-microorganisms-11-01830">40</a>], Flye [<a aria-describedby="B41-microorganisms-11-01830" class="usa-link" href="#B41-microorganisms-11-01830">41</a>], and MEGAHIT [<a aria-describedby="B42-microorganisms-11-01830" class="usa-link" href="#B42-microorganisms-11-01830">42</a>] (<a class="usa-link" href="#microorganisms-11-01830-f002">Figure 2</a>).</p>
<figure class="fig xbox font-sm" id="microorganisms-11-01830-f002"><h3 class="obj_head">Figure 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10386755_microorganisms-11-01830-g002.jpg" target="_blank"><img alt="Figure 2" class="graphic zoom-in" height="1253" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f82a/10386755/6661337749ed/microorganisms-11-01830-g002.jpg" width="753"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/microorganisms-11-01830-f002/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Approaches used in the generation of zygomycete genomes: sequencing (<strong>upper panel</strong>) and assembly (<strong>lower panel</strong>) technologies. Different shades of green and brown—<em>Mucoromycota</em>; purple and blue—<em>Zoopagomycota</em>.</p></figcaption></figure><p>Most zygomycete genome assemblies are similar in size to the average fungal genome: 30–50 Mb (<a class="usa-link" href="#app1-microorganisms-11-01830">Supplementary Table S1</a>). Notable exceptions include species in <em>Entomophthoromycotina</em>, <em>Glomeromycotina</em>, and <em>Mucoromycotina</em>, whose large genomes compare in size to <em>Arabidopsis thaliana</em> (135 Mb), <em>Caenorhabditis elegans</em> (100 Mb), or <em>Drosophila melanogaster</em> (140 Mb) [<a aria-describedby="B43-microorganisms-11-01830" class="usa-link" href="#B43-microorganisms-11-01830">43</a>,<a aria-describedby="B44-microorganisms-11-01830" class="usa-link" href="#B44-microorganisms-11-01830">44</a>]. However, there are still far smaller than the genomes of some of the Basidiomycete fungi that cause rust diseases on plants, which includes the largest fungal genome sequenced to date (<em>Austropuccinia psidii</em>, 1 Gb) [<a aria-describedby="B45-microorganisms-11-01830" class="usa-link" href="#B45-microorganisms-11-01830">45</a>]. Most zygomycete genomes contain over 10,000 predicted genes. <em>Rhizophagus</em> spp. (<em>Glomeromycotina</em>) are a notable exception; they contain more than double the typical number with 26–30 K predicted genes (<a class="usa-link" href="#microorganisms-11-01830-f003">Figure 3</a>). The smallest numbers of genes are found in <em>Zoopagomycotina</em> (4–8 K), followed by 6–11 K in <em>Kickxellomycotina</em>, 9–16 K in <em>Entomophthoromycotina</em>, <em>Mucoromycotina</em> and <em>Mortierellomycotina</em>, and 20–30 K in <em>Glomeromycotina</em> (<a class="usa-link" href="#app1-microorganisms-11-01830">Supplementary Table S1</a>). However, these numbers may change for <em>Entomophthoromycotina</em> and <em>Zoopagomycotina</em> since there are only a few representative genomes available for those lineages.</p>
<figure class="fig xbox font-sm" id="microorganisms-11-01830-f003"><h3 class="obj_head">Figure 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10386755_microorganisms-11-01830-g003.jpg" target="_blank"><img alt="Figure 3" class="graphic zoom-in" height="639" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f82a/10386755/b1e7eb24efa3/microorganisms-11-01830-g003.jpg" width="775"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/microorganisms-11-01830-f003/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Genome statistics for zygomycete sequences. Sizes of the genomes and number of the genomes sequenced since 2005 (<strong>upper left</strong>), number of contigs in assemblies (<strong>upper right</strong>), estimated sequencing depth coverage relative to the single genome (<strong>lower left</strong>), and the number of the predicted genes (<strong>lower right</strong>). Different shades of green and brown—<em>Mucoromycota</em>; purple and blue—<em>Zoopagomycota</em>.</p></figcaption></figure><p>During the assembly process DNA sequences are combined into contigs, which are sometimes merged to form scaffolds (if sufficient evidence exists to support these merges). These scaffolds of assembled DNA sequences should ideally equal the number of chromosomes [<a aria-describedby="B46-microorganisms-11-01830" class="usa-link" href="#B46-microorganisms-11-01830">46</a>]. In practice, the number of scaffolds rarely matches exactly the number of chromosomes and, for most species, the exact number of chromosomes is unknown [<a aria-describedby="B47-microorganisms-11-01830" class="usa-link" href="#B47-microorganisms-11-01830">47</a>]. Many genomes of easily culturable <em>Mucoromycota</em> with small genomes have the lowest number of scaffolds (e.g., 12 in <em>Actinomortierella wolfii</em>), while the genomes of species with larger genomes (e.g., taxa in <em>Glomeromycotina</em> and <em>Entomophthoromycotina</em>) are more fragmented such as 210 contigs in <em>Rhizophagus irregularis</em> [<a aria-describedby="B26-microorganisms-11-01830" class="usa-link" href="#B26-microorganisms-11-01830">26</a>], or 373,021 in <em>Massospora cicadina</em> [<a aria-describedby="B48-microorganisms-11-01830" class="usa-link" href="#B48-microorganisms-11-01830">48</a>]. These larger assemblies are due to larger overall genomes (e.g., larger number of chromosomes) and because of the difficulties with assembly caused by polyploidy, numerous repeats, and transposons [<a aria-describedby="B49-microorganisms-11-01830" class="usa-link" href="#B49-microorganisms-11-01830">49</a>]; these aspects are prevalent in many <em>Mucoromycota</em> and <em>Zoopagomycota</em> genomes (<a class="usa-link" href="#app1-microorganisms-11-01830">Supplementary Table S1</a>). However, the sequenced genomes with hundreds of thousands of scaffolds often reflect low sequence coverage relative to the size of the genome, resulting in highly fragmented assemblies. All these factors contribute to the number of the scaffolds, not reflecting the actual numbers of chromosomes.</p>
<p>Genome quality is also improved by increasing the sequence coverage. Some zygomycete genomes have been sequenced at a depth of 500–700× and many genome projects aim for a mean depth of ca. 100× coverage (<a class="usa-link" href="#app1-microorganisms-11-01830">Supplement Table S1</a>). High-quality genome assemblies rely on increased sequencing depth, particularly in cases of high intraspecific polymorphism or variable genome architecture, e.g., species of <em>Glomeromycotina</em> and many noncoding regions and transposons, as in the genomes of <em>Entomophthoromycotina</em> species. However, some researchers have used low-coverage genome sequencing (ca. 10× coverage) as a cheap and efficient method to obtain a smaller number of orthologous gene regions for phylogenomic analyses [<a aria-describedby="B12-microorganisms-11-01830" class="usa-link" href="#B12-microorganisms-11-01830">12</a>,<a aria-describedby="B15-microorganisms-11-01830" class="usa-link" href="#B15-microorganisms-11-01830">15</a>,<a aria-describedby="B24-microorganisms-11-01830" class="usa-link" href="#B24-microorganisms-11-01830">24</a>].</p></section><section id="sec3-microorganisms-11-01830"><h2 class="pmc_sec_title">3. Genomes and the Evolution of the Zygomycetes</h2>
<p>The first genome sequence of a zygomycete fungus was from a clinical strain of <em>Rhizopus delemar</em> that was isolated from a patient with mucormycosis—a highly destructive and lethal infection that is typically seen in immunocompromised hosts. Sequencing of the <em>R. delemar</em> genome was driven from the clinical perspective, but it also provided the first genome of a fungus outside of the <em>Dikarya</em> [<a aria-describedby="B18-microorganisms-11-01830" class="usa-link" href="#B18-microorganisms-11-01830">18</a>]. Moving forward a decade, a landmark project generated genomic sequences of all species known to cause mucoromycosis in humans. Thirty zygomycete genomes were sequenced and assembled for this project [<a aria-describedby="B50-microorganisms-11-01830" class="usa-link" href="#B50-microorganisms-11-01830">50</a>]. Another substantial advance in zygomycete genomics was made by the ZyGoLife team, comprising the efforts of 12 mycological labs across the USA and Canada in collaboration with the US Department of Energy Joint Genome Institute (JGI) [<a aria-describedby="B14-microorganisms-11-01830" class="usa-link" href="#B14-microorganisms-11-01830">14</a>]. The phylogenomic study by Spatafora et al. [<a aria-describedby="B6-microorganisms-11-01830" class="usa-link" href="#B6-microorganisms-11-01830">6</a>] found that the former <em>Zygomycota</em> phylum was not monophyletic but instead comprised two distinct lineages—the primarily plant-associated saprotrophic and mycorrhizal <em>Mucoromycota</em> versus the primarily mycoparasitic or animal-associated parasites and commensals in the <em>Zoopagomycota</em>. However, further analyses using phylogenomics data to resolve the evolutionary relationships of zygomycetes have yielded slightly conflicting results. Although <em>Zoopagomycota</em> and <em>Mucoromycota</em> are maintained as monophyletic lineages, a recent study by Li et al. [<a aria-describedby="B51-microorganisms-11-01830" class="usa-link" href="#B51-microorganisms-11-01830">51</a>] found that <em>Zoopagomycota</em> and <em>Mucoromycota</em> formed one larger monophyletic clade in some (but not all) of their analyses. In contrast, other recent phylogenomic studies by Strassert and Monaghan [<a aria-describedby="B52-microorganisms-11-01830" class="usa-link" href="#B52-microorganisms-11-01830">52</a>] and Galindo et al. [<a aria-describedby="B53-microorganisms-11-01830" class="usa-link" href="#B53-microorganisms-11-01830">53</a>] recovered a topology that was similar to Spatafora et al. [<a aria-describedby="B6-microorganisms-11-01830" class="usa-link" href="#B6-microorganisms-11-01830">6</a>], where the <em>Zoopagomycota</em> were resolved as the sister group to all terrestrial fungi. In many of the studies, the position of <em>Glomeromycotina</em> has been problematic, suggesting that more work will be needed to resolve the placement of this clade [<a aria-describedby="B51-microorganisms-11-01830" class="usa-link" href="#B51-microorganisms-11-01830">51</a>,<a aria-describedby="B54-microorganisms-11-01830" class="usa-link" href="#B54-microorganisms-11-01830">54</a>].</p>
<p>One of the greatest debates in early-diverging fungal evolution was the ploidy of the dominant life form given the predominantly haploid nature seen in the <em>Dikarya</em>. One outcome of the analysis of genome sequences of zygomycetes species is that diploids are more common than expected [<a aria-describedby="B9-microorganisms-11-01830" class="usa-link" href="#B9-microorganisms-11-01830">9</a>,<a aria-describedby="B55-microorganisms-11-01830" class="usa-link" href="#B55-microorganisms-11-01830">55</a>,<a aria-describedby="B56-microorganisms-11-01830" class="usa-link" href="#B56-microorganisms-11-01830">56</a>,<a aria-describedby="B57-microorganisms-11-01830" class="usa-link" href="#B57-microorganisms-11-01830">57</a>].</p>
<section id="sec3dot1-microorganisms-11-01830"><h3 class="pmc_sec_title">3.1. Mucoromycota</h3>
<p><em>Mucoromycota</em> comprises many critically important plant symbionts, including the arbuscular mycorrhizal fungi (<em>Glomeromycotina</em>) [<a aria-describedby="B58-microorganisms-11-01830" class="usa-link" href="#B58-microorganisms-11-01830">58</a>], <em>Endogonales</em> (<em>Mucoromycotina</em>) [<a aria-describedby="B24-microorganisms-11-01830" class="usa-link" href="#B24-microorganisms-11-01830">24</a>], “fine root endophytes” (<em>Mucoromycotina</em>) [<a aria-describedby="B59-microorganisms-11-01830" class="usa-link" href="#B59-microorganisms-11-01830">59</a>], and putative root endophytic species including <em>Mortierella</em> (<em>Mortierellomycotina</em>) [<a aria-describedby="B60-microorganisms-11-01830" class="usa-link" href="#B60-microorganisms-11-01830">60</a>], <em>Pygmaeomycetaceae</em> (<em>Mucoromycotina</em>) [<a aria-describedby="B61-microorganisms-11-01830" class="usa-link" href="#B61-microorganisms-11-01830">61</a>], and <em>Umbelopsis</em> (<em>Mucoromycotina</em>) [<a aria-describedby="B62-microorganisms-11-01830" class="usa-link" href="#B62-microorganisms-11-01830">62</a>]. On the other hand, many fungi within this group are also noted plant pathogens (e.g., <em>Rhizopus microsporus</em>, which causes rice seedling blight, and various <em>Choanephora</em>, <em>Gilbertella</em>, and <em>Mucor</em> species that cause post-harvest rots) as well as numerous species (<em>Actinomucor</em>, <em>Apophysomyces</em>, <em>Cokeromyces</em>, <em>Cunninghamella</em>, <em>Lichtheimia</em>, <em>Mucor</em>, <em>Rhizomucor</em>, <em>Rhizopus</em>, <em>Saksenea</em>, <em>Syncephalastrum</em>) that can cause the disease mucormycosis in humans and other animals [<a aria-describedby="B63-microorganisms-11-01830" class="usa-link" href="#B63-microorganisms-11-01830">63</a>].</p>
<p>Most zygomycete genomes are of species within <em>Mucoromycota</em> (<a class="usa-link" href="#microorganisms-11-01830-f001">Figure 1</a>); as a result, much of what we know about genome evolution among early-diverging fungi comes from this phylum. Genome duplication appears to have played a significant role in their evolution, as first described for <em>Rhizopus delemar</em> and <em>Mucor circinelloides</em> [<a aria-describedby="B18-microorganisms-11-01830" class="usa-link" href="#B18-microorganisms-11-01830">18</a>,<a aria-describedby="B64-microorganisms-11-01830" class="usa-link" href="#B64-microorganisms-11-01830">64</a>]. Genome sequencing has also revealed a diversity of endobacteria that may be found as symbionts in the <em>Mucoromycota</em>, e.g., in <em>Rhizopus microsporus</em> and <em>Endogone pisiformis</em> (<em>Mucoromycotina</em>), <em>Linnemannia elongata</em> and <em>Mortierella alpina</em> (<em>Mortierellomycotina</em>), and <em>Rhizophagus irregularis</em> (<em>Glomeromycotina</em>) [<a aria-describedby="B65-microorganisms-11-01830" class="usa-link" href="#B65-microorganisms-11-01830">65</a>,<a aria-describedby="B66-microorganisms-11-01830" class="usa-link" href="#B66-microorganisms-11-01830">66</a>,<a aria-describedby="B67-microorganisms-11-01830" class="usa-link" href="#B67-microorganisms-11-01830">67</a>,<a aria-describedby="B68-microorganisms-11-01830" class="usa-link" href="#B68-microorganisms-11-01830">68</a>,<a aria-describedby="B69-microorganisms-11-01830" class="usa-link" href="#B69-microorganisms-11-01830">69</a>,<a aria-describedby="B70-microorganisms-11-01830" class="usa-link" href="#B70-microorganisms-11-01830">70</a>]. Many plant-associated fungal lineages contain endobacteria, but the potential role of these symbionts in the land colonization process remain to be characterized. “Foreign” DNA in the form of diverse mycoviruses has also been frequently detected during genome sequencing of zygomycetes [<a aria-describedby="B30-microorganisms-11-01830" class="usa-link" href="#B30-microorganisms-11-01830">30</a>,<a aria-describedby="B71-microorganisms-11-01830" class="usa-link" href="#B71-microorganisms-11-01830">71</a>]. Genomics enabled the systematic study of genome architecture [<a aria-describedby="B72-microorganisms-11-01830" class="usa-link" href="#B72-microorganisms-11-01830">72</a>,<a aria-describedby="B73-microorganisms-11-01830" class="usa-link" href="#B73-microorganisms-11-01830">73</a>,<a aria-describedby="B74-microorganisms-11-01830" class="usa-link" href="#B74-microorganisms-11-01830">74</a>], genetic manipulations [<a aria-describedby="B75-microorganisms-11-01830" class="usa-link" href="#B75-microorganisms-11-01830">75</a>], and intra-isolate genomic variations [<a aria-describedby="B76-microorganisms-11-01830" class="usa-link" href="#B76-microorganisms-11-01830">76</a>]. Genome-wide searches and comparisons have also detected evidence for transposon spread through genomes [<a aria-describedby="B77-microorganisms-11-01830" class="usa-link" href="#B77-microorganisms-11-01830">77</a>,<a aria-describedby="B78-microorganisms-11-01830" class="usa-link" href="#B78-microorganisms-11-01830">78</a>,<a aria-describedby="B79-microorganisms-11-01830" class="usa-link" href="#B79-microorganisms-11-01830">79</a>] and uniquely high 6-methyladenine DNA modifications in zygomycete fungi as compared to <em>Dikarya</em> [<a aria-describedby="B80-microorganisms-11-01830" class="usa-link" href="#B80-microorganisms-11-01830">80</a>]. Studies on metabolic patterns across the fungal kingdom, the search for the mating gene loci and protein families [<a aria-describedby="B81-microorganisms-11-01830" class="usa-link" href="#B81-microorganisms-11-01830">81</a>,<a aria-describedby="B82-microorganisms-11-01830" class="usa-link" href="#B82-microorganisms-11-01830">82</a>,<a aria-describedby="B83-microorganisms-11-01830" class="usa-link" href="#B83-microorganisms-11-01830">83</a>], and the phylogenies of certain taxonomic groups [<a aria-describedby="B84-microorganisms-11-01830" class="usa-link" href="#B84-microorganisms-11-01830">84</a>,<a aria-describedby="B85-microorganisms-11-01830" class="usa-link" href="#B85-microorganisms-11-01830">85</a>] have also been conducted using-whole genome sequencing data. Genome-based research has also provided insights into the trophic modes of zygomycete fungi, including the evolution of mycorrhizae [<a aria-describedby="B24-microorganisms-11-01830" class="usa-link" href="#B24-microorganisms-11-01830">24</a>,<a aria-describedby="B86-microorganisms-11-01830" class="usa-link" href="#B86-microorganisms-11-01830">86</a>] and commonalities in the pathogenicity of Mucoralean fungi [<a aria-describedby="B50-microorganisms-11-01830" class="usa-link" href="#B50-microorganisms-11-01830">50</a>,<a aria-describedby="B87-microorganisms-11-01830" class="usa-link" href="#B87-microorganisms-11-01830">87</a>].</p></section><section id="sec3dot2-microorganisms-11-01830"><h3 class="pmc_sec_title">3.2. Mucoromycotina</h3>
<p>Genome sequencing of this subphylum has been the most comprehensive among all the basal fungal lineages although genomes are still unavailable for several genera, including species of <em>Siepmannia</em>, <em>Chlamydoabsidia</em>, and <em>Utharomyces</em>, as well as for two new root-associated species of <em>Pygmaeomyces</em> of the new genus and family <em>Pygmaeomycetaceae</em> [<a aria-describedby="B61-microorganisms-11-01830" class="usa-link" href="#B61-microorganisms-11-01830">61</a>]. Furthermore, the plant-associated order <em>Endogonales</em> remains highly undersampled despite recent advances [<a aria-describedby="B24-microorganisms-11-01830" class="usa-link" href="#B24-microorganisms-11-01830">24</a>,<a aria-describedby="B88-microorganisms-11-01830" class="usa-link" href="#B88-microorganisms-11-01830">88</a>]. Some of its “fine root endophytes” colonize early-diverging plants, and sometimes co-occur with AMF [<a aria-describedby="B89-microorganisms-11-01830" class="usa-link" href="#B89-microorganisms-11-01830">89</a>,<a aria-describedby="B90-microorganisms-11-01830" class="usa-link" href="#B90-microorganisms-11-01830">90</a>]. The average genome size in <em>Mucoromycotina</em> species ranges from 35 to 50 Mbp, with species of <em>Umbelopsis</em> at the low end of the spectrum (22–30 Mbp) and <em>Endogonales</em> such as <em>Jimgerdemannia</em> species at the higher end (240 Mbp) (<a class="usa-link" href="#app1-microorganisms-11-01830">Supplementary Table S1</a>). For some genera, several hundred genomes have already been sequenced. For example, there are &gt;220 genomes available for species of <em>Rhizopus</em>, mostly of <em>R. arrhizus</em> (syn. <em>R. oryzae</em>) and its varieties and synonyms, and of <em>R. microsporus</em>, <em>R. delemar</em>, and <em>R. stolonifer</em> (<a class="usa-link" href="#app1-microorganisms-11-01830">Supplement Table S1</a>). <em>Rhizopus</em> is increasingly studied as a model organism thanks to its availability in culture collections and to its important role in biotechnology and human health. Genomic information might be helpful for future studies of pathogenicity in <em>Rhizopus</em> species since infections by members of this genus are the leading cause of human mucormycosis [<a aria-describedby="B50-microorganisms-11-01830" class="usa-link" href="#B50-microorganisms-11-01830">50</a>,<a aria-describedby="B84-microorganisms-11-01830" class="usa-link" href="#B84-microorganisms-11-01830">84</a>]. Phylogenomic data suggest that species of <em>Rhizopus</em> are part of a well resolved monophyletic group. However, this is in stark contrast to the situation among members of the genus <em>Mucor</em> (&gt;150 genomes), which is incredibly diverse and is likely polyphyletic (<a class="usa-link" href="#microorganisms-11-01830-f004">Figure 4</a>). The additional <em>Mucoromycotina</em> genomes consist mostly of saprotrophic species that are relevant to human health (species of <em>Lichtheimia</em> and <em>Cunninghamella</em>) and biotechnology (<em>Phycomyces blakesleeanus</em>) [<a aria-describedby="B91-microorganisms-11-01830" class="usa-link" href="#B91-microorganisms-11-01830">91</a>,<a aria-describedby="B92-microorganisms-11-01830" class="usa-link" href="#B92-microorganisms-11-01830">92</a>,<a aria-describedby="B93-microorganisms-11-01830" class="usa-link" href="#B93-microorganisms-11-01830">93</a>]. Genomes of <em>Endogonales</em> (as well as <em>Glomeromycotina</em>) contain effectors to facilitate plant–fungus interactions, which apparently aided in the terrestrialization of Earth by early land plants. Genome information helped to describe and clarify the phylogenetic placement of new lineages and species, such as <em>Bifiguratus</em> [<a aria-describedby="B94-microorganisms-11-01830" class="usa-link" href="#B94-microorganisms-11-01830">94</a>] and <em>Calcarisporiella</em> [<a aria-describedby="B95-microorganisms-11-01830" class="usa-link" href="#B95-microorganisms-11-01830">95</a>].</p>
<figure class="fig xbox font-sm" id="microorganisms-11-01830-f004"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10386755_microorganisms-11-01830-g004.jpg" target="_blank"><img alt="Figure 4" class="graphic zoom-in" height="1014" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f82a/10386755/920492c0fbbc/microorganisms-11-01830-g004.jpg" width="761"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/microorganisms-11-01830-f004/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Distributions of the number of genomes sequenced imposed onto phylogenetic tree. Different shades of green and brown—<em>Mucoromycota</em>, purple and blue—<em>Zoopagomycota</em>.</p></figcaption></figure></section><section id="sec3dot3-microorganisms-11-01830"><h3 class="pmc_sec_title">3.3. Mortierellomycotina</h3>
<p>This subphylum contains an estimated 170 species [<a aria-describedby="B96-microorganisms-11-01830" class="usa-link" href="#B96-microorganisms-11-01830">96</a>] and was recently circumscribed into 13 monophyletic genera using multi-locus and low-coverage genome sequencing [<a aria-describedby="B12-microorganisms-11-01830" class="usa-link" href="#B12-microorganisms-11-01830">12</a>]. Reference genome sequences of both low quality and high quality are now available for representatives for most genera, including <em>Actinomortierella</em>, <em>Benniella</em>, <em>Dissophora</em>, <em>Entomortierella</em>, <em>Gamsiella</em>, <em>Gryganskiella</em>, <em>Linnemannia</em>, <em>Lobosporangium</em>, <em>Lunasporangiospora</em>, <em>Mortierella</em>, <em>Necromortierella</em>, <em>Podila</em>, and the sporocarpic genus <em>Modicella</em> [<a aria-describedby="B12-microorganisms-11-01830" class="usa-link" href="#B12-microorganisms-11-01830">12</a>,<a aria-describedby="B97-microorganisms-11-01830" class="usa-link" href="#B97-microorganisms-11-01830">97</a>]. A high diversity of <em>Mortierellomycotina</em> species can be isolated from soil, but it appears that genera may differ in their ecological roles. For example, <em>Necromortierella</em> species appear to be necrotrophic, whereas <em>Actinomortierella</em> spp. are often associated with millipedes, and <em>Entomortierella</em> spp. are often associated with arthropods [<a aria-describedby="B98-microorganisms-11-01830" class="usa-link" href="#B98-microorganisms-11-01830">98</a>]. Many genera, such as <em>Linnemannia</em> and <em>Podila</em>, have been recovered as rhizosphere associates and endophytes of plant roots [<a aria-describedby="B99-microorganisms-11-01830" class="usa-link" href="#B99-microorganisms-11-01830">99</a>,<a aria-describedby="B100-microorganisms-11-01830" class="usa-link" href="#B100-microorganisms-11-01830">100</a>], and they appear to improve plant growth and impact flower and seed production [<a aria-describedby="B101-microorganisms-11-01830" class="usa-link" href="#B101-microorganisms-11-01830">101</a>]. Several species such as <em>Mortierella alpina</em> have been sequenced due to their use in industry and biotechnology for lipid production [<a aria-describedby="B102-microorganisms-11-01830" class="usa-link" href="#B102-microorganisms-11-01830">102</a>].</p></section><section id="sec3dot4-microorganisms-11-01830"><h3 class="pmc_sec_title">3.4. Glomeromycotina</h3>
<p>Arbuscular mycorrhizal fungi (AMFs) are arguably the most common and oldest symbionts of terrestrial plants [<a aria-describedby="B58-microorganisms-11-01830" class="usa-link" href="#B58-microorganisms-11-01830">58</a>], credited with the role in the facilitation of plant transition to the terrestrial habitat [<a aria-describedby="B103-microorganisms-11-01830" class="usa-link" href="#B103-microorganisms-11-01830">103</a>]. AMFs supply their plant hosts with mineral nutrients translocated from the soil in return for plant-assimilated carbon [<a aria-describedby="B104-microorganisms-11-01830" class="usa-link" href="#B104-microorganisms-11-01830">104</a>]. Yet, there are not many genomic sequences of these organisms available due to historical impediments hampering sequencing projects, including multinucleate spores with different genotypes, obligate biotrophy of AMFs, and the initial uncertainty concerning their genome sizes, with conflicting size estimates in <em>Rhizophagus irregularis</em> ranging from 14.1 [<a aria-describedby="B105-microorganisms-11-01830" class="usa-link" href="#B105-microorganisms-11-01830">105</a>] to 154.8 Mb [<a aria-describedby="B106-microorganisms-11-01830" class="usa-link" href="#B106-microorganisms-11-01830">106</a>]. Once these obstacles were surmounted, genomic data confirmed that AMFs have unusually large genomes, ranging from 153 Mb in <em>R. irregularis</em> [<a aria-describedby="B26-microorganisms-11-01830" class="usa-link" href="#B26-microorganisms-11-01830">26</a>] to 773.1 Mb in <em>Gigaspora margarita</em> [<a aria-describedby="B107-microorganisms-11-01830" class="usa-link" href="#B107-microorganisms-11-01830">107</a>]. This size enhancement relative to other <em>Mucoromycota</em> is driven by lineage-specific expansions of gene families and massive proliferation of transposable elements [<a aria-describedby="B26-microorganisms-11-01830" class="usa-link" href="#B26-microorganisms-11-01830">26</a>,<a aria-describedby="B86-microorganisms-11-01830" class="usa-link" href="#B86-microorganisms-11-01830">86</a>,<a aria-describedby="B107-microorganisms-11-01830" class="usa-link" href="#B107-microorganisms-11-01830">107</a>]. Genomic data also revealed that AMFs display several other features that differentiate them not only from <em>Mucoromycota</em> but also from other fungi. For example, despite the lack of morphological evidence of sexual reproduction, the <em>R. irregularis</em> genome harbors a candidate mating-type locus with two genes encoding homeodomain-like transcription factors [<a aria-describedby="B108-microorganisms-11-01830" class="usa-link" href="#B108-microorganisms-11-01830">108</a>]. If these homeodomain-like genes indeed control sex, then regulation of the reproductive processes in AMFs would resemble mating in <em>Dikarya</em> rather than in <em>Mucoromycota</em>, which rely on HMG domain proteins for sex determination [<a aria-describedby="B109-microorganisms-11-01830" class="usa-link" href="#B109-microorganisms-11-01830">109</a>]. <em>Rhizophagus irregularis</em> differs also from other fungi in the organization of its rRNA operons. Instead of being tandemly arrayed as they are in other fungi, the rRNA operons in <em>Glomeromycotina</em> are dispersed individually across the genome [<a aria-describedby="B26-microorganisms-11-01830" class="usa-link" href="#B26-microorganisms-11-01830">26</a>]. This important discovery put an end to a long-lasting debate on whether intraindividual rRNA gene variation typical for AMFs is distributed among dissimilar nuclei [<a aria-describedby="B110-microorganisms-11-01830" class="usa-link" href="#B110-microorganisms-11-01830">110</a>] or contained in each nucleus [<a aria-describedby="B111-microorganisms-11-01830" class="usa-link" href="#B111-microorganisms-11-01830">111</a>], supporting the latter scenario. Lastly, genome-based inferences of metabolic capacity revealed that AMFs lack the gene encoding fatty acid synthase [<a aria-describedby="B112-microorganisms-11-01830" class="usa-link" href="#B112-microorganisms-11-01830">112</a>,<a aria-describedby="B113-microorganisms-11-01830" class="usa-link" href="#B113-microorganisms-11-01830">113</a>,<a aria-describedby="B114-microorganisms-11-01830" class="usa-link" href="#B114-microorganisms-11-01830">114</a>], which makes them obligately dependent on plant hosts for lipids [<a aria-describedby="B115-microorganisms-11-01830" class="usa-link" href="#B115-microorganisms-11-01830">115</a>,<a aria-describedby="B116-microorganisms-11-01830" class="usa-link" href="#B116-microorganisms-11-01830">116</a>,<a aria-describedby="B117-microorganisms-11-01830" class="usa-link" href="#B117-microorganisms-11-01830">117</a>,<a aria-describedby="B118-microorganisms-11-01830" class="usa-link" href="#B118-microorganisms-11-01830">118</a>]. As in other fastidious organisms, this information is important for devising cultivation strategies that complement fatty acid auxotrophy of AMF [<a aria-describedby="B119-microorganisms-11-01830" class="usa-link" href="#B119-microorganisms-11-01830">119</a>,<a aria-describedby="B120-microorganisms-11-01830" class="usa-link" href="#B120-microorganisms-11-01830">120</a>]. Even though accumulation of genomic data for AMF has lagged behind other <em>Mucoromycota</em>, the insights gathered so far have been critical for resolving several puzzles in the biology of these unique organisms. PacBio HiFi and Hi-C sequencing of all available AMF heterokaryons helped to discover the two sets of coexisting homologous chromosomes in one heterokaryon. Genes required for plant colonization were found in gene-sparse, repeat-reach compartments [<a aria-describedby="B121-microorganisms-11-01830" class="usa-link" href="#B121-microorganisms-11-01830">121</a>]. Another important question that remains to be addressed is which genomic features of AMFs contribute to superior yield outcomes in agronomic crop species, as AMFs do not universally benefit all plant hosts.</p></section><section id="sec3dot5-microorganisms-11-01830"><h3 class="pmc_sec_title">3.5. Zoopagomycota</h3>
<p>In contrast to <em>Mucoromycota</em>, most species of <em>Zoopagomycota</em> are parasites or pathogenes of animals and fungi. This includes the insect pathogenic group <em>Entomophthoromycotina</em> (some members of which exhibit behavioral control over their hosts) [<a aria-describedby="B122-microorganisms-11-01830" class="usa-link" href="#B122-microorganisms-11-01830">122</a>], the <em>Kickxellomycotina</em> that includes a mixture of ecologies (including commensalistic arthropods gut fungi) [<a aria-describedby="B15-microorganisms-11-01830" class="usa-link" href="#B15-microorganisms-11-01830">15</a>], and <em>Zoopagomycotina</em>, which are parasites of fungi (mainly in <em>Mucoromycota</em>) and microinvertebrates (e.g., amoebae, nematodes, rotifers) [<a aria-describedby="B20-microorganisms-11-01830" class="usa-link" href="#B20-microorganisms-11-01830">20</a>]. There are also opportunistic human pathogens in this phylum, including species of <em>Basidiobolus</em> and <em>Conidiobolus</em> [<a aria-describedby="B123-microorganisms-11-01830" class="usa-link" href="#B123-microorganisms-11-01830">123</a>]. Axenic culturing of some fungi in <em>Zoopagomycota</em> is possible [<a aria-describedby="B124-microorganisms-11-01830" class="usa-link" href="#B124-microorganisms-11-01830">124</a>,<a aria-describedby="B125-microorganisms-11-01830" class="usa-link" href="#B125-microorganisms-11-01830">125</a>] but extracting high-quality genomic DNA from many species remains difficult. Sometimes there is a need to extract genomic material of the parasite together with the host (often another fungus or insect) and separate their genomes during assembly, which creates additional challenges. Small amounts of available DNA have necessitated the use of single-cell DNA extraction in some cases, especially in species with dramatically reduced vegetative mycelial structures. Much greater success has been achieved for putative saprotrophic fungi such as dung-inhabiting species of <em>Kickxellales</em> (<a class="usa-link" href="#app1-microorganisms-11-01830">Supplementary Table S1</a>). Genomes and transcriptomes have been used primarily to resolve various evolutionary questions, including the monophyletic origin of <em>Zoopagales</em> [<a aria-describedby="B20-microorganisms-11-01830" class="usa-link" href="#B20-microorganisms-11-01830">20</a>], evolution of ploidy from a diploid ancestral stage [<a aria-describedby="B11-microorganisms-11-01830" class="usa-link" href="#B11-microorganisms-11-01830">11</a>], and loss of a large number of pectinases and other plant cell-wall-degrading enzymes in <em>Zoopagomycota</em> [<a aria-describedby="B126-microorganisms-11-01830" class="usa-link" href="#B126-microorganisms-11-01830">126</a>]. Genome data were also successfully used to resolve evolutionary relationships within particular taxonomic groups, such as <em>Massospora</em> [<a aria-describedby="B98-microorganisms-11-01830" class="usa-link" href="#B98-microorganisms-11-01830">98</a>], to track other evolutionary events such as horizontal gene transfer events in gut fungi [<a aria-describedby="B127-microorganisms-11-01830" class="usa-link" href="#B127-microorganisms-11-01830">127</a>], and to reveal the genetic toolbox in insect symbionts [<a aria-describedby="B128-microorganisms-11-01830" class="usa-link" href="#B128-microorganisms-11-01830">128</a>]. These studies help us to understand the evolutionary trajectories of genome size, namely, a dramatic increase in genome size for the insect pathogens and insect endosymbionts as compared to the genome size in the saprobes and mycoparasites. Genome data have also helped to clarify fundamental differences among <em>Zoopagomycota</em> lineages in their production of secondary metabolites [<a aria-describedby="B129-microorganisms-11-01830" class="usa-link" href="#B129-microorganisms-11-01830">129</a>], as well as metabolic pathways in general [<a aria-describedby="B11-microorganisms-11-01830" class="usa-link" href="#B11-microorganisms-11-01830">11</a>]. Mating genetics in this phylum is still not well understood, and that is a question that remains to be addressed with genomic data.</p></section><section id="sec3dot6-microorganisms-11-01830"><h3 class="pmc_sec_title">3.6. Kickxellomycotina</h3>
<p>This subphylum contains an interesting mix of mycoparasites (<em>Dimargaritales</em>), putative saprotrophs (<em>Kickxellales</em>), and obligate insect endobionts (<em>Asselariales</em>, <em>Barbatosporales</em>, <em>Harpellales</em>, and <em>Orphellales</em>). The <em>Dimargaritales</em> are similar to mycoparasites in the <em>Zoopagales</em> in that they form penetration hyphae (haustoria) and mostly attack similar host species in the <em>Mucoromycota</em>. In contrast to the <em>Zoopagales</em> mycoparasites, species of <em>Dimargaritales</em> appear to be rare in the environment and have mostly been isolated from dung rather than soil. Recent genomic analyses have also highlighted that members of the <em>Dimargaritales</em> that have larger genomes and more predicted secondary metabolite genes compared to their close relatives [<a aria-describedby="B15-microorganisms-11-01830" class="usa-link" href="#B15-microorganisms-11-01830">15</a>,<a aria-describedby="B129-microorganisms-11-01830" class="usa-link" href="#B129-microorganisms-11-01830">129</a>]. Furthermore, single-cell genome data have suggested that <em>Dimargaris cristalligena</em> may be nonhaploid and missing certain enzymes from metabolic pathways (e.g., thiamine biosynthesis and biotin metabolism) [<a aria-describedby="B11-microorganisms-11-01830" class="usa-link" href="#B11-microorganisms-11-01830">11</a>]. Identifying such genomic features of symbiotic species can provide targets for further experimental tests to understand the molecular interactions between hosts and parasites.</p>
<p>On the other hand, the <em>Kickxellales</em> is the only group of <em>Zoopagomycota</em> containing mostly saprotrophic species, and many appear to be rare in the environment, having only been reported in the literature one or a few times. An exception is the genus <em>Coemansia</em>, the species of which are commonly isolated from soil and dung samples [<a aria-describedby="B4-microorganisms-11-01830" class="usa-link" href="#B4-microorganisms-11-01830">4</a>,<a aria-describedby="B130-microorganisms-11-01830" class="usa-link" href="#B130-microorganisms-11-01830">130</a>]. Genome sequencing efforts have demonstrated that <em>Kickxellales</em> species have small genomes (less than 40 MB) and variability in their predicted secondary metabolite genes, with <em>Coemansia</em> and <em>Linderina</em>, along with more polyketide synthases than other <em>Kickxellales</em> and <em>D. cristalligena</em> having more nonribosomal peptide synthetases than other <em>Kickxellales</em> [<a aria-describedby="B129-microorganisms-11-01830" class="usa-link" href="#B129-microorganisms-11-01830">129</a>]. A preliminary rDNA phylogeny showed polyphyly among <em>Kickxellales</em> [<a aria-describedby="B130-microorganisms-11-01830" class="usa-link" href="#B130-microorganisms-11-01830">130</a>], and a large-scale sequencing project that utilized low-coverage genome data from &gt;100 isolates to test these hypotheses, as well as explore the diversity of <em>Coemansia</em> species, resolved many of these relationships [<a aria-describedby="B15-microorganisms-11-01830" class="usa-link" href="#B15-microorganisms-11-01830">15</a>].</p>
<p>The gut fungi (<em>Trichomycetes</em>) include arthropod symbionts for which several draft genomes have been available since 2016 [<a aria-describedby="B131-microorganisms-11-01830" class="usa-link" href="#B131-microorganisms-11-01830">131</a>]. These fungi are distinct from the saprotrophic and coprophilic relatives (i.e., <em>Kickxellales</em>) and belong to four distinct phylogenetic lineages—<em>Asellariales</em>, Barbatosporales, <em>Harpellales</em>, and <em>Orphellales</em> [<a aria-describedby="B132-microorganisms-11-01830" class="usa-link" href="#B132-microorganisms-11-01830">132</a>]. <em>Asellariales</em> are associated with isopods and spring tails (<em>Collembola</em>) [<a aria-describedby="B133-microorganisms-11-01830" class="usa-link" href="#B133-microorganisms-11-01830">133</a>]. Species within the three genera (<em>Asellaria</em>, <em>Baltomyces</em>, and <em>Orchesellaria</em>) have been recovered from the both terrestrial and aquatic hosts [<a aria-describedby="B133-microorganisms-11-01830" class="usa-link" href="#B133-microorganisms-11-01830">133</a>], but their phylogenetic relationships remain unclear, as sequence data have only been obtained from <em>Asellaria</em> [<a aria-describedby="B132-microorganisms-11-01830" class="usa-link" href="#B132-microorganisms-11-01830">132</a>], and no cultures are available for this group.</p>
<p><em>Harpellales</em> species are usually found in lower <em>Diptera</em> insect larvae and have coevolved with their hosts for over 200 million years [<a aria-describedby="B134-microorganisms-11-01830" class="usa-link" href="#B134-microorganisms-11-01830">134</a>]. Species of <em>Orphellales</em> are stonefly nymph symbionts with unusual morphological characters (i.e., asexual and sexual spore shapes, and thalli protruding beyond the anus of insect hosts) [<a aria-describedby="B135-microorganisms-11-01830" class="usa-link" href="#B135-microorganisms-11-01830">135</a>], which cluster together with other <em>Trichomycetes</em> and the <em>Spiromycetales</em> for a sister clade to <em>Kickxellales</em> [<a aria-describedby="B15-microorganisms-11-01830" class="usa-link" href="#B15-microorganisms-11-01830">15</a>]. <em>Smittium</em> is a well-studied genus within <em>Harpellales</em> with approximately 100 described species [<a aria-describedby="B136-microorganisms-11-01830" class="usa-link" href="#B136-microorganisms-11-01830">136</a>]. However, there are only nine whole-genome sequences available for these insect gut-dwelling fungi, and all were made from culturable <em>Harpellales</em> species, with the best assembly—<em>Capniomyces stellatus</em>—having 72 scaffolds [<a aria-describedby="B131-microorganisms-11-01830" class="usa-link" href="#B131-microorganisms-11-01830">131</a>]. <em>Barbatosporales</em> are monotypic, and the single species (<em>Barbatospora ambicaudata</em>) has only been recorded from black fly larvae collected in the Great Smoky Mountains National Park in Tennessee, obtained in axenic culture [<a aria-describedby="B137-microorganisms-11-01830" class="usa-link" href="#B137-microorganisms-11-01830">137</a>].</p>
<p>Although we only have a handful of <em>Harpellales</em> genomes, they have already expanded our knowledge and understanding of these enigmatic microbial fungi in multiple ways. For example, a mosquito-like polyubiquitin gene has been identified in <em>Zancudomyces culisetae</em> (<em>Harpellales</em>), which was presumably acquired via a horizontal gene transfer event [<a aria-describedby="B127-microorganisms-11-01830" class="usa-link" href="#B127-microorganisms-11-01830">127</a>]. Comparative genomics revealed a fungus–insect symbiotic core gene toolbox (FISCoG) that is shared among the insect-associated fungi (e.g., <em>Harpellales</em>, <em>Basidiobolus</em>, and <em>Conidiobolus</em>) and higher-ranked <em>Hypocreales</em> (<em>Ascomycota</em>). However, the insect pathogens (<em>Hypocreales</em> and <em>Entomophthoromycotina</em> members) tend to have genomes enriched in genes that are useful for pathogenic processes such as the platelet-activating factor acetyl-hydrolase coding genes, whereas the gut commensals have genomes enriched in cell adhesion genes for a successful gut-dwelling lifestyle [<a aria-describedby="B128-microorganisms-11-01830" class="usa-link" href="#B128-microorganisms-11-01830">128</a>]. In addition, <em>Harpellales</em> genomes also facilitated a kingdom-wide study to confirm the production of selenoproteins in early-diverging fungal lineages [<a aria-describedby="B138-microorganisms-11-01830" class="usa-link" href="#B138-microorganisms-11-01830">138</a>]. As a major group of early-diverging fungi, representing seven of the nine fungal species that utilize selenoproteins, <em>Harpellales</em> may take the advantages of selenocysteine over cysteine for specific oxidoreductase functions.</p>
<p>Unfortunately, no efforts have been successful in obtaining genome information of the <em>Asellariales</em>, <em>Barbatosporales</em>, and <em>Orphellales</em>, which colonize different host types. However, multigene analyses incorporating low-coverage genome data (along with data from <em>Asellariales</em> and <em>Orphellales</em>) suggested an alternate hypothesis for the evolution of fungal–insect associations. Contrary to previous topologies that suggested multiple events [<a aria-describedby="B132-microorganisms-11-01830" class="usa-link" href="#B132-microorganisms-11-01830">132</a>], the larger dataset showed a possible single origin of gut fungi [<a aria-describedby="B15-microorganisms-11-01830" class="usa-link" href="#B15-microorganisms-11-01830">15</a>]. Further evolutionary evidence may be identified with the help of their genome sequences to understand how these gut-dwelling microbial fungi interact with various aquatic insect hosts and evaluate the origins of the symbiosis.</p></section><section id="sec3dot7-microorganisms-11-01830"><h3 class="pmc_sec_title">3.7. Zoopagomycotina</h3>
<p><em>Zoopagomycotina</em> species are fascinating, partially due to their cryptic ecological niches and ability to parasitize various hosts such as amoebae, nematodes, rotifers, and other fungi. It is still challenging to culture most of the <em>Zoopagomycotina</em> isolates. Thus, it is not surprising to see that the <em>Zoopagomycotina</em> has the lowest number of available genomes, the qualities of which are not comparable to other lineages, especially the well-sequenced <em>Mucoromycotina</em>. Recently developed single-cell genomic techniques have provided an alternative, culture-independent method to obtain their genomic information [<a aria-describedby="B11-microorganisms-11-01830" class="usa-link" href="#B11-microorganisms-11-01830">11</a>]. On the basis of single-cell or multiple-cell libraries, the completeness of genome assemblies can be as high as 89.6% (<em>Piptocephalis tieghemiana</em> RSA 1565) [<a aria-describedby="B20-microorganisms-11-01830" class="usa-link" href="#B20-microorganisms-11-01830">20</a>]. The assembled <em>Zoopagomycotina</em> genomes indicate that this clade of fungi is characterized by small genome size, but the numbers of protein-coding genes are not necessarily low. Some of the genomes can encode close to 10,000 genes (e.g., <em>Zoophagus insidians</em>, a predator of rotifers and nematodes (<a class="usa-link" href="#app1-microorganisms-11-01830">Supplement Table S1</a>). It seems likely that these completely microscopic fungi are capable of more complicated and yet-to-be identified interactions with their animal and/or fungal hosts, but more work will be needed to explore these interactions.</p>
<p>Two additional <em>Zoopagales</em> have remarkably small genomes: a species of <em>Acaulopage</em> (11 Mbp) and a species of <em>Zoopage</em> (14 Mbp). No genomes have yet been published for the following genera: <em>Amoebophilus</em>, <em>Aplectosoma</em>, <em>Bdellospora</em>, <em>Brachymyces</em>, <em>Cystopage</em>, <em>Endocochlus</em>, <em>Euryancale</em>, <em>Helicocephalum</em>, <em>Reticulocephalis</em>, and <em>Sigmoideomyces</em>.</p></section><section id="sec3dot8-microorganisms-11-01830"><h3 class="pmc_sec_title">3.8. Entomophthoromycotina</h3>
<p>This lineage remains the most insufficiently sequenced fungal group at the subphylum level. Despite many genomes from the genus <em>Conidiobolus</em>, there are only a few other species (all from the key family <em>Entomophthoraceae</em>) for which genomes are available. The genus <em>Conidiobolus</em> is a polyphyletic assemblage of multiple lineages. This group of <em>Conidiobolus</em>-like fungi was recently phylogenetically divided into three new families, using a combined multi-gene and phylogenomic approach, revealing that the ballistic conidia arose prior to their transition to a parasitic lifestyle [<a aria-describedby="B85-microorganisms-11-01830" class="usa-link" href="#B85-microorganisms-11-01830">85</a>,<a aria-describedby="B139-microorganisms-11-01830" class="usa-link" href="#B139-microorganisms-11-01830">139</a>]. Difficulties with culturing and, therefore, obtaining enough high-molecular-weight genomic DNA represent a reason why some species only have transcriptomes available. Alternatively, in some cases, the fungal pathogen is sequenced together with its host, and then the reads are separated bioinformatically. Aside from the polyphyletic genus <em>Conidiobolus</em> and entomophthoroid-like <em>Basidiobolus</em>, genome or transcriptome data are available only for a few species of <em>Entomophaga</em>, <em>Entomophthora</em>, <em>Massospora</em>, <em>Pandora</em>, and <em>Zoophthora</em>. No genomes yet exist for <em>Ancylistes</em>, <em>Apterivorax</em>, <em>Batkoa</em>, <em>Erynia</em>, <em>Eryniopsis</em>, <em>Furia</em>, <em>Macrobiotophthora</em>, <em>Meristacrum</em>, <em>Neozygites</em>, <em>Orthomyces</em>, <em>Tabanomyces</em>, <em>Thaxterosporium</em>, <em>Schizangiella</em>, and <em>Strongwellsea</em>. The largest genomes recovered are over 2000 Mbp in <em>Massospora</em> and 650 Mbp in <em>Zoophthora</em>, with possibly similar genome sizes in <em>Entomophthora</em> and <em>Entomophaga</em> (<a class="usa-link" href="#app1-microorganisms-11-01830">Supplementary Table S1</a>). Phylogenetic analysis recently demonstrated that <em>Basidiobolus</em>, a genus of amphibian gut-dwelling fungi, does not belong to <em>Entomophthorales</em>, where it was traditionally placed, mostly on the basis of its production of forcibly discharged conidia that are morphologically similar to those in other entomophthoralean fungi [<a aria-describedby="B140-microorganisms-11-01830" class="usa-link" href="#B140-microorganisms-11-01830">140</a>]. The genus was difficult to place with rDNA data [<a aria-describedby="B141-microorganisms-11-01830" class="usa-link" href="#B141-microorganisms-11-01830">141</a>], but was inferred either as a separate clade sister to the <em>Mucoromycota</em> [<a aria-describedby="B51-microorganisms-11-01830" class="usa-link" href="#B51-microorganisms-11-01830">51</a>] or as the earliest diverging lineage within <em>Zoopagomycota</em> [<a aria-describedby="B128-microorganisms-11-01830" class="usa-link" href="#B128-microorganisms-11-01830">128</a>], on the basis of phylogenomic data. The ballistic spore dispersal mechanism, which differs structurally from that found in the <em>Entomophthorales</em>, is likely an example of convergent evolution among fungi that appear to be similar but are distantly related. In addition to being evolutionarily enigmatic, the genomes of <em>B. meristosporus</em> and <em>B. heterosporus</em> are larger than most other basal fungi and contained a much higher proportion of secondary metabolite-encoding genes than other <em>Zoopagomycota</em> species that have been studied thus far. Furthermore, <em>Basidiobolus</em> genomes contain numerous genes that putatively originated from horizontal gene transfer events from bacteria [<a aria-describedby="B129-microorganisms-11-01830" class="usa-link" href="#B129-microorganisms-11-01830">129</a>].</p></section></section><section id="sec4-microorganisms-11-01830"><h2 class="pmc_sec_title">4. Insights from Genomics</h2>
<section id="sec4dot1-microorganisms-11-01830"><h3 class="pmc_sec_title">4.1. Adaptation to Terrestrial Life Inferred from Genomes</h3>
<p>The transition to a terrestrial environment placed novel constraints on species of early-diverging fungi, especially in terms of their resistance to desiccation and solar radiation. Genomes have been essential to studying which zygomycete genes made this transition possible.</p>
<p>Sensing of and response to light have been explored in the model <em>Mucorales</em> species <em>Phycomyces blakesleeanus</em>. Mutants impaired in photoresponses, especially phototropism, were isolated as reported in the 1970s [<a aria-describedby="B142-microorganisms-11-01830" class="usa-link" href="#B142-microorganisms-11-01830">142</a>]. However, the first genes related to photoresponses were not reported until 2006 [<a aria-describedby="B143-microorganisms-11-01830" class="usa-link" href="#B143-microorganisms-11-01830">143</a>]. These were identified using a candidate gene approach in which <em>P. blakesleeanus</em> was assumed to have similar blue light sensors as the <em>Dikarya</em> species [<a aria-describedby="B143-microorganisms-11-01830" class="usa-link" href="#B143-microorganisms-11-01830">143</a>,<a aria-describedby="B144-microorganisms-11-01830" class="usa-link" href="#B144-microorganisms-11-01830">144</a>], wherein a two-protein complex was first identified in <em>Neurospora crassa</em>, defining a sensing and signaling system that uses a light-sensing domain coupled to transcription factor roles for signal transmission. With the release of the <em>P. blakesleeanus</em> genome sequence in 2010, identifying a candidate gene for <em>madB</em> as encoding the interacting protein was achieved using BLAST against the genome sequence [<a aria-describedby="B145-microorganisms-11-01830" class="usa-link" href="#B145-microorganisms-11-01830">145</a>]. The genome sequence was performed along with that of <em>M. circinelloidies</em> and revealed an expansion of this pair of genes required for light responses (3–4 copies of each). Being able to make targeted gene mutations in <em>M. circinelloides</em> of these homologs, as based of genome sequence information, provided a clear example of subfunctionalization after gene duplication [<a aria-describedby="B146-microorganisms-11-01830" class="usa-link" href="#B146-microorganisms-11-01830">146</a>]. The identification of the mutations in <em>madI</em> and <em>madC</em> mutants in <em>P. blakesleeanus</em> used traditional genetic mapping with molecular markers derived from the comparison of the genomes between two strains. With the regions narrowed, candidate genes could be identified and confirmed by sequencing [<a aria-describedby="B64-microorganisms-11-01830" class="usa-link" href="#B64-microorganisms-11-01830">64</a>,<a aria-describedby="B147-microorganisms-11-01830" class="usa-link" href="#B147-microorganisms-11-01830">147</a>]. These genes play conserved roles in photobiology in the <em>Dikarya</em> species, suggesting that this innovation for terrestrial life had already evolved in early-diverging fungal lineages. For instance, phenotypic analysis of the <em>P. blakesleeanus madA</em> and <em>madB</em> mutants indicated that this pair of genes has a conserved function in protecting fungi from harmful ultraviolet light [<a aria-describedby="B148-microorganisms-11-01830" class="usa-link" href="#B148-microorganisms-11-01830">148</a>].</p>
<p>In addition to responding to light, many <em>Mucorales</em> species can sense gravity. This has been correlated with the formation of vacuolar crystals in <em>P. blakesleeanus</em> [<a aria-describedby="B149-microorganisms-11-01830" class="usa-link" href="#B149-microorganisms-11-01830">149</a>]. Through purification of such crystals, followed by mass spectrometry of the associated proteins using the mass database derived from the genome sequence, the OCTIN protein was identified. A gravity-insensitive mutant had a mutation within the corresponding gene. Remarkably, phylogenetic comparison revealed that the origin of the gravity-sensing gene OCTIN in the <em>Mucorales</em> was most likely via a horizontal gene transfer from a bacterial species [<a aria-describedby="B150-microorganisms-11-01830" class="usa-link" href="#B150-microorganisms-11-01830">150</a>]. Other challenges associated with terrestrial life, such as fluctuations in temperature and desiccation, have yet to explored from a genomic standpoint among zygomycetes. Some of the early diverging lineages such as <em>Glomeromycotina</em> and <em>Mortierellomycotina</em> lack ergosterol, what can increase their membrane stiffness [<a aria-describedby="B77-microorganisms-11-01830" class="usa-link" href="#B77-microorganisms-11-01830">77</a>,<a aria-describedby="B112-microorganisms-11-01830" class="usa-link" href="#B112-microorganisms-11-01830">112</a>]. Ergosterol has long been considered as one of the regulators of dry–wet cycles [<a aria-describedby="B151-microorganisms-11-01830" class="usa-link" href="#B151-microorganisms-11-01830">151</a>].</p></section><section id="sec4dot2-microorganisms-11-01830"><h3 class="pmc_sec_title">4.2. Endobacteria</h3>
<p>Symbiotic associations with highly coevolved endocellular bacteria stand out among the most prominent features differentiating <em>Mucoromycota</em> from other fungi, including the <em>Dikarya</em> [<a aria-describedby="B152-microorganisms-11-01830" class="usa-link" href="#B152-microorganisms-11-01830">152</a>,<a aria-describedby="B153-microorganisms-11-01830" class="usa-link" href="#B153-microorganisms-11-01830">153</a>,<a aria-describedby="B154-microorganisms-11-01830" class="usa-link" href="#B154-microorganisms-11-01830">154</a>]. Thus far, most of these endosymbionts have been identified as representatives of two bacterial families, <em>Burkholderiaceae</em> and <em>Mycoplasmataceae</em>. While some of the <em>Mucoromycota</em> associations with bacteria are mutually beneficial for both partners, in others, the fungi appear to be negatively impacted by their endosymbionts. The application of “omics” tools has been essential in elucidating the molecular underpinnings of the mutualism between <em>Glomeromycotina</em> and their ancient and uncultivable endosymbiont “<em>Candidatus</em> Glomeribacter gigasporarum” (CaGg, <em>Burkholderiaceae</em>) [<a aria-describedby="B155-microorganisms-11-01830" class="usa-link" href="#B155-microorganisms-11-01830">155</a>]. In this symbiosis, CaGg improves hyphal proliferation in the presymbiotic growth stage of the host fungus [<a aria-describedby="B156-microorganisms-11-01830" class="usa-link" href="#B156-microorganisms-11-01830">156</a>], which can be attributed to endobacteria-mediated reprograming of the fungal metabolism [<a aria-describedby="B157-microorganisms-11-01830" class="usa-link" href="#B157-microorganisms-11-01830">157</a>,<a aria-describedby="B158-microorganisms-11-01830" class="usa-link" href="#B158-microorganisms-11-01830">158</a>]. “Omics” approaches were also critical in understanding the biology and devising a cultivation strategy for <em>Mycoavidus cysteinexigens</em> (<em>Burkholderiaceae</em>) [<a aria-describedby="B159-microorganisms-11-01830" class="usa-link" href="#B159-microorganisms-11-01830">159</a>], a cysteine auxotroph and ancient antagonistic endosymbiont of <em>Mortierellomycotina</em> [<a aria-describedby="B69-microorganisms-11-01830" class="usa-link" href="#B69-microorganisms-11-01830">69</a>,<a aria-describedby="B160-microorganisms-11-01830" class="usa-link" href="#B160-microorganisms-11-01830">160</a>,<a aria-describedby="B161-microorganisms-11-01830" class="usa-link" href="#B161-microorganisms-11-01830">161</a>]. Comparative genomics also revealed that “<em>Candidatus</em> Mycoavidus necroximicus” produces several nematocidal metabolites that protect the fungal host <em>Mortierella verticillata</em> from feeding by nematodes [<a aria-describedby="B162-microorganisms-11-01830" class="usa-link" href="#B162-microorganisms-11-01830">162</a>]. Endobacteria have also been shown to regulate and control mating and asexual reproduction in <em>Rhizopus microsporus</em> [<a aria-describedby="B163-microorganisms-11-01830" class="usa-link" href="#B163-microorganisms-11-01830">163</a>], produce toxins (rhizoxin) that enable pathogenicity on plants, and provide protection from predators [<a aria-describedby="B164-microorganisms-11-01830" class="usa-link" href="#B164-microorganisms-11-01830">164</a>], as well as potentially protect fungal spores from the human immune system via anti-phagocyte activity [<a aria-describedby="B165-microorganisms-11-01830" class="usa-link" href="#B165-microorganisms-11-01830">165</a>]. The ease of partner manipulation in the mutualism between <em>R. microsporus</em> and <em>Mortierellaceae</em> (<em>Mucoromycotina</em>) and <em>Mycetohabitans</em> spp. (formerly <em>Burkholderia</em>) [<a aria-describedby="B166-microorganisms-11-01830" class="usa-link" href="#B166-microorganisms-11-01830">166</a>,<a aria-describedby="B167-microorganisms-11-01830" class="usa-link" href="#B167-microorganisms-11-01830">167</a>] elevated this association to a model system for studying fungal-bacterial symbioses [<a aria-describedby="B163-microorganisms-11-01830" class="usa-link" href="#B163-microorganisms-11-01830">163</a>,<a aria-describedby="B168-microorganisms-11-01830" class="usa-link" href="#B168-microorganisms-11-01830">168</a>].</p>
<p>While the symbioses of <em>Mucoromycota</em> with <em>Burkholderiaceae</em> endobacteria are well characterized for some species, <em>Mucoromycota</em> associations with <em>Mycoplasmataceae</em>, referred to as <em>Mollicutes</em>- or mycoplasma-related endobacteria (MREs) [<a aria-describedby="B169-microorganisms-11-01830" class="usa-link" href="#B169-microorganisms-11-01830">169</a>,<a aria-describedby="B170-microorganisms-11-01830" class="usa-link" href="#B170-microorganisms-11-01830">170</a>], are only beginning to be unraveled. MREs have been detected in all three subphyla of <em>Mucoromycota</em>: <em>Glomeromycotina</em> [<a aria-describedby="B169-microorganisms-11-01830" class="usa-link" href="#B169-microorganisms-11-01830">169</a>], <em>Mortierellomycotina</em> [<a aria-describedby="B171-microorganisms-11-01830" class="usa-link" href="#B171-microorganisms-11-01830">171</a>], and <em>Mucoromycotina</em> [<a aria-describedby="B24-microorganisms-11-01830" class="usa-link" href="#B24-microorganisms-11-01830">24</a>,<a aria-describedby="B172-microorganisms-11-01830" class="usa-link" href="#B172-microorganisms-11-01830">172</a>]. MREs of <em>Glomeromycotina</em> are classified as “<em>Candidatus</em> Moeniiplasma glomeromycotorum” (CaMg) [<a aria-describedby="B173-microorganisms-11-01830" class="usa-link" href="#B173-microorganisms-11-01830">173</a>]. Genomic features of CaMg have been examined in depth [<a aria-describedby="B66-microorganisms-11-01830" class="usa-link" href="#B66-microorganisms-11-01830">66</a>,<a aria-describedby="B174-microorganisms-11-01830" class="usa-link" href="#B174-microorganisms-11-01830">174</a>], including evidence of gene acquisition from fungal hosts. However, the role of CaMg in the biology of <em>Glomeromycotina</em> remains uncertain. Naito et al. [<a aria-describedby="B174-microorganisms-11-01830" class="usa-link" href="#B174-microorganisms-11-01830">174</a>] speculated that CaMg is an antagonist of <em>Glomeromycotina</em>. If this is confirmed, then CaMg would resemble the lifestyle of the MREs associated with <em>Mortierellomycotina</em> [<a aria-describedby="B171-microorganisms-11-01830" class="usa-link" href="#B171-microorganisms-11-01830">171</a>].</p>
<p>Moreover, bacterial–fungal interactions seem to be much more common and diverse than previously thought [<a aria-describedby="B175-microorganisms-11-01830" class="usa-link" href="#B175-microorganisms-11-01830">175</a>], and <em>Mucoromycota</em> representatives are no exception. Telagathoti et al. [<a aria-describedby="B176-microorganisms-11-01830" class="usa-link" href="#B176-microorganisms-11-01830">176</a>] showed recently that 30% of <em>Mortierellales</em> isolates were associated with different groups of bacteria. Another survey of <em>Mucoromycota</em> cultures from soil indicated diverse <em>Paraburkholderia</em>-related sequences detected with 20% frequency from <em>Mortierella</em> and <em>Umbelopsis</em> strains [<a aria-describedby="B154-microorganisms-11-01830" class="usa-link" href="#B154-microorganisms-11-01830">154</a>]. The ecological meaning of these interactions remains to be described.</p>
<p>Outside of the <em>Mucoromycota</em>, however, the importance or prevalence of endosymbiotic bacteria remains less clear. Despite bacterial sequences always present in the environmental samples of entomophthoralean fungi (A.P. Gryganskyi, personal observation), nothing is known about their connection to the entomopathogens and their role in pathogenic processes that impact insects. However, six recently constructed metagenome assemblies of four various bacterial species in connection with <em>Massospora cicadina</em> might indicate their symbiotic relationship to this entomopathogenic species [<a aria-describedby="B177-microorganisms-11-01830" class="usa-link" href="#B177-microorganisms-11-01830">177</a>]. Among <em>Zoopagomycotina</em>, Davis et al. [<a aria-describedby="B20-microorganisms-11-01830" class="usa-link" href="#B20-microorganisms-11-01830">20</a>] detected 16S sequences with close matches to <em>Mycoavidus cysteinexexigens</em> in the genome of <em>Stylopage hadra</em> (nematode parasitic fungus). A recent genomic assessment of <em>Kickxellomycotina</em> fungi did not reveal any obvious colonization by endosymbiotic bacteria [<a aria-describedby="B15-microorganisms-11-01830" class="usa-link" href="#B15-microorganisms-11-01830">15</a>], suggesting that these bacterial associations might be less important or prevalent in <em>Zoopagomycota</em> than in <em>Mucoromycota</em>.</p></section><section id="sec4dot3-microorganisms-11-01830"><h3 class="pmc_sec_title">4.3. Secretomes, Gene Clusters, and Secondary Metabolites</h3>
<p>All fungi explore and exploit their environment by secreting various substances. The secretome composition of basal fungal lineages was shaped by diversification early in their evolution. Characterization of secretome compositions of 132 zygomycete genomes revealed that current trophic mode plays a less significant role as compared to the phylogenetic relationships of the fungi [<a aria-describedby="B178-microorganisms-11-01830" class="usa-link" href="#B178-microorganisms-11-01830">178</a>]. This is in part because <em>Mucoromycota</em> and <em>Zoopagomycota</em> each had their own unique lineage-specific expansion of secreted digestive enzymes, according to their historical substrate differences and their different evolutionary trajectories [<a aria-describedby="B178-microorganisms-11-01830" class="usa-link" href="#B178-microorganisms-11-01830">178</a>].</p>
<p>Genomic screenings of available genomes for novel natural compounds also show promising potential [<a aria-describedby="B179-microorganisms-11-01830" class="usa-link" href="#B179-microorganisms-11-01830">179</a>,<a aria-describedby="B180-microorganisms-11-01830" class="usa-link" href="#B180-microorganisms-11-01830">180</a>,<a aria-describedby="B181-microorganisms-11-01830" class="usa-link" href="#B181-microorganisms-11-01830">181</a>]. These screenings discovered the presence of gene clusters harboring polyketide synthases (PKS), non-ribosomal peptides, terpenoids, and L-tryptophan dimethylallyl transferases with transcription factor coding genes in proximity to some of the gene clusters. Since then, NRPS clusters have been detected in some <em>Mortierellomycotina</em> and <em>Mucoromycotina</em>. For example, the mycelium of <em>Mortierella alpina</em> produces diverse novel compounds [<a aria-describedby="B182-microorganisms-11-01830" class="usa-link" href="#B182-microorganisms-11-01830">182</a>]. Similarly, the transcripts corresponding to PKS and NRPS were recently described in transcriptomes of five <em>Mucor</em> species [<a aria-describedby="B183-microorganisms-11-01830" class="usa-link" href="#B183-microorganisms-11-01830">183</a>]. <em>Mucor circinelloides</em> was shown to produce carotenoids and terpenoids [<a aria-describedby="B180-microorganisms-11-01830" class="usa-link" href="#B180-microorganisms-11-01830">180</a>], and the genes associated with terpene biosynthesis were identified recently in <em>Mucor</em> and <em>Rhizopus</em> genomes [<a aria-describedby="B74-microorganisms-11-01830" class="usa-link" href="#B74-microorganisms-11-01830">74</a>]. Siderophore production has been reported from <em>Glomus</em> [<a aria-describedby="B184-microorganisms-11-01830" class="usa-link" href="#B184-microorganisms-11-01830">184</a>], <em>Apophysomyces</em> [<a aria-describedby="B185-microorganisms-11-01830" class="usa-link" href="#B185-microorganisms-11-01830">185</a>], <em>Rhizopus</em> [<a aria-describedby="B186-microorganisms-11-01830" class="usa-link" href="#B186-microorganisms-11-01830">186</a>], and <em>Basidiobolus</em> [<a aria-describedby="B129-microorganisms-11-01830" class="usa-link" href="#B129-microorganisms-11-01830">129</a>]. Rhizoferrin from <em>Rhizopus delemar</em> has been further characterized and showed a glycosylation pattern [<a aria-describedby="B187-microorganisms-11-01830" class="usa-link" href="#B187-microorganisms-11-01830">187</a>] that was classified as a member of the sodium (Na)–iodide symporter gene family [<a aria-describedby="B186-microorganisms-11-01830" class="usa-link" href="#B186-microorganisms-11-01830">186</a>]. The best-studied compound isolated from early diverging fungi is an antimitotic agent called rhizoxin produced by an endosymbiotic bacterium (<em>Mycetohabitans rhizoxinica</em>) and encoded by a huge NRPS–PKS cluster [<a aria-describedby="B166-microorganisms-11-01830" class="usa-link" href="#B166-microorganisms-11-01830">166</a>]. The genome of amphibian gut symbiont <em>Basidiobolus</em> is enriched with the genes and gene clusters involved in the production of secondary metabolites through horizontal gene transfer from bacteria [<a aria-describedby="B129-microorganisms-11-01830" class="usa-link" href="#B129-microorganisms-11-01830">129</a>]. Genomes of mycoparasitic <em>Dimargaritales</em> (<em>Kickxellomycotina</em>) are also enriched for secondary metabolites [<a aria-describedby="B15-microorganisms-11-01830" class="usa-link" href="#B15-microorganisms-11-01830">15</a>], especially when compared to mycoparasitic <em>Zoopagomycotina</em> (<em>Piptocephalis</em> and <em>Syncephalis</em>) whose genomes appear depauperate in these genes [<a aria-describedby="B129-microorganisms-11-01830" class="usa-link" href="#B129-microorganisms-11-01830">129</a>]. These groups of mycoparasites are very similar in their morphology and ecology; thus, the reasons underpinning the observed difference in metabolite genes remain unclear.</p></section><section id="sec4dot4-microorganisms-11-01830"><h3 class="pmc_sec_title">4.4. Transposable Elements</h3>
<p>Transposable elements (TEs) are abundant in zygomycete genomes [<a aria-describedby="B79-microorganisms-11-01830" class="usa-link" href="#B79-microorganisms-11-01830">79</a>]. Genomes of <em>Entomophtorales</em>, <em>Basidiobolales</em> [<a aria-describedby="B129-microorganisms-11-01830" class="usa-link" href="#B129-microorganisms-11-01830">129</a>], and <em>Endogonales</em> [<a aria-describedby="B24-microorganisms-11-01830" class="usa-link" href="#B24-microorganisms-11-01830">24</a>] are particularly rich in TEs, while species of <em>Umbelopsidales</em> [<a aria-describedby="B188-microorganisms-11-01830" class="usa-link" href="#B188-microorganisms-11-01830">188</a>] and <em>Zoopagomycotina</em> [<a aria-describedby="B11-microorganisms-11-01830" class="usa-link" href="#B11-microorganisms-11-01830">11</a>] have very few. Genomes of <em>Rhizopus</em> species comprised approximately 40% repetitive sequences with only ~10% potentially active elements [<a aria-describedby="B84-microorganisms-11-01830" class="usa-link" href="#B84-microorganisms-11-01830">84</a>]. In taxa with more compact genomes such as <em>Umbelopsis isabellina</em>, transposons account for as little as 5% of their genome size. In contrast, <em>Jimgerdemannia flammicorona</em> can have more than 77% of its genome occupied by repetitive elements and transposons [<a aria-describedby="B24-microorganisms-11-01830" class="usa-link" href="#B24-microorganisms-11-01830">24</a>]. In most of the analyzed genomes, long terminal repeat retrotransposons from the Ty3/Gypsy family dominate the TE landscape [<a aria-describedby="B84-microorganisms-11-01830" class="usa-link" href="#B84-microorganisms-11-01830">84</a>]. <em>Mucoromycotina</em>, <em>Kickxellomycotina</em>, and <em>Blastocladiomycota</em> possess transposons with a tyrosine recombinase from a gene family which seem to be lost in <em>Dikarya</em> [<a aria-describedby="B78-microorganisms-11-01830" class="usa-link" href="#B78-microorganisms-11-01830">78</a>]. While the function of mobile elements in zygomycete genomes is not well understood, in Ascomycota, large mobile elements (termed “starships”) may carry a variety of genes (e.g., secondary metabolites, virulence factors, and heavy-metal tolerance) and contribute to the overall variation in genome structure [<a aria-describedby="B189-microorganisms-11-01830" class="usa-link" href="#B189-microorganisms-11-01830">189</a>,<a aria-describedby="B190-microorganisms-11-01830" class="usa-link" href="#B190-microorganisms-11-01830">190</a>].</p></section></section><section id="sec5-microorganisms-11-01830"><h2 class="pmc_sec_title">5. Conclusions</h2>
<p>There has been renewed research interest in the zygomycete fungi during the last decade, and this is reflected in the rapidly increasing number of genome sequences from this group of fungi. While the quality of the genomes varies according to the sequencing platform and starting materials available, the genomes are sufficient for phylogenomic reconstructions, as well as for the discovery of functional genes and gene clusters of interest. Much of the work has focused on several key topics, including the evolution and phylogeny of the group, adaptation, and transition to a terrestrial habitat (e.g., light and gravity sensors, and reinforcement of cell-wall structure to tolerate a nonaquatic environment), genome duplication, endosymbionts and horizontal gene transfer, genome architecture, pathogenicity, and coevolution with host species. The number of genomes has grown rapidly over the past decade, and we expect that the next decade will bring increased taxon sampling (e.g., new genomes for taxa that have not yet been sampled) and improvements in genome quality due to advances in long-read sequencing technologies and bioinformatics approaches. In the future, we expect that genomic analyses of these fungi will take new directions, including a detailed comparison of the genomes of aquatic vs. terrestrial lineages, structural studies, a comparison of the zygomycete vs. <em>Dikarya</em> genomes, experimental manipulations of host/parasite interactions to understand the molecular mechanisms underpinning pathogenesis, and building links among genomic, proteomic, and metabolomic data.</p></section><section class="app" id="app1-microorganisms-11-01830"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<p>The following supporting information can be downloaded at <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.mdpi.com/article/10.3390/microorganisms11071830/s1" rel="noopener noreferrer" target="_blank">https://www.mdpi.com/article/10.3390/microorganisms11071830/s1</a>: Supplementary Table S1. The genomes of <em>Mucoromycota</em> and <em>Zoopagomycota</em> available in open public sources: NCBI Bioproject, Genome and Nucleotide databases.</p>
<section class="sm xbox font-sm" id="microorganisms-11-01830-s001"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10386755/bin/microorganisms-11-01830-s001.zip">Click here for additional data file.</a><sup> (104KB, zip) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflict of interest.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This material is based upon work supported by the National Science Foundation (DEB-1441604 to JWS, DEB-1441715 to JES, DEB-1441677 to TYJ). Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the National Science Foundation. The work (proposals 10.46936/10.25585/60001019 and 10.46936/10.25585/60001062) conducted by the U.S. Department of Energy Joint Genome Institute (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://ror.org/04xm1d337" rel="noopener noreferrer" target="_blank">https://ror.org/04xm1d337</a>), a DOE Office of Science User Facility, is supported by the Office of Science of the U.S. Department of Energy operated under Contract No. DE-AC02-05CH11231. J.E.S. is a paid consultant for Michroma and Sincarne and CIFAR fellow in the program Fungal Kingdom: Threats and Opportunities. </p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Disclaimer/Publisher’s Note:</strong> The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.</p></div></div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1-microorganisms-11-01830">
<span class="label">1.</span><cite>Heaton L.L.M., Jones N.S., Fricker M.D. A Mechanistic Explanation of the Transition to Simple Multicellularity in Fungi. Nat. Commun. 2020;11:2594. doi: 10.1038/s41467-020-16072-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41467-020-16072-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7244713/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32444651/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=A%20Mechanistic%20Explanation%20of%20the%20Transition%20to%20Simple%20Multicellularity%20in%20Fungi&amp;author=L.L.M.%20Heaton&amp;author=N.S.%20Jones&amp;author=M.D.%20Fricker&amp;volume=11&amp;publication_year=2020&amp;pages=2594&amp;pmid=32444651&amp;doi=10.1038/s41467-020-16072-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2-microorganisms-11-01830">
<span class="label">2.</span><cite>Nagy L.G., Varga T., Csernetics Á., Virágh M. Fungi Took a Unique Evolutionary Route to Multicellularity: Seven Key Challenges for Fungal Multicellular Life. Fungal Biol. Rev. 2020;34:151–169. doi: 10.1016/j.fbr.2020.07.002.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.fbr.2020.07.002" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Biol.%20Rev.&amp;title=Fungi%20Took%20a%20Unique%20Evolutionary%20Route%20to%20Multicellularity:%20Seven%20Key%20Challenges%20for%20Fungal%20Multicellular%20Life&amp;author=L.G.%20Nagy&amp;author=T.%20Varga&amp;author=%C3%81.%20Csernetics&amp;author=M.%20Vir%C3%A1gh&amp;volume=34&amp;publication_year=2020&amp;pages=151-169&amp;doi=10.1016/j.fbr.2020.07.002&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3-microorganisms-11-01830">
<span class="label">3.</span><cite>de Hoog G.S., Guarro J., Gené J., Figueras M.J.  Atlas of Clinical Fungi. 2nd ed. Centraalbureau voor Schimmelcultures (CBS); Utrecht, The Netherlands: 2000. </cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Atlas%20of%20Clinical%20Fungi&amp;author=G.S.%20de%20Hoog&amp;author=J.%20Guarro&amp;author=J.%20Gen%C3%A9&amp;author=M.J.%20Figueras&amp;publication_year=2000&amp;isbn=90-70351-43-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4-microorganisms-11-01830">
<span class="label">4.</span><cite>Benny G.L., Smith M.E., Kirk P.M., Tretter E.D., White M.M.  Challenges and Future Perspectives in the Systematics of Kickxellomycotina, Mortierellomycotina, Mucoromycotina, and Zoopagomycotina. In: Li D.-W., editor. Biology of Microfungi. Springer International Publishing; Cham, Switzerland: 2016. pp. 65–126. Fungal Biology.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Biology%20of%20Microfungi&amp;author=G.L.%20Benny&amp;author=M.E.%20Smith&amp;author=P.M.%20Kirk&amp;author=E.D.%20Tretter&amp;author=M.M.%20White&amp;publication_year=2016&amp;isbn=978-3-319-29137-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5-microorganisms-11-01830">
<span class="label">5.</span><cite>James T.Y., Letcher P.M., Longcore J.E., Mozley-Standridge S.E., Porter D., Powell M.J., Griffith G.W., Vilgalys R. A Molecular Phylogeny of the Flagellated Fungi (Chytridiomycota) and Description of a New Phylum (Blastocladiomycota) Mycologia. 2006;98:860–871. doi: 10.1080/15572536.2006.11832616.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/15572536.2006.11832616" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17486963/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=A%20Molecular%20Phylogeny%20of%20the%20Flagellated%20Fungi%20(Chytridiomycota)%20and%20Description%20of%20a%20New%20Phylum%20(Blastocladiomycota)&amp;author=T.Y.%20James&amp;author=P.M.%20Letcher&amp;author=J.E.%20Longcore&amp;author=S.E.%20Mozley-Standridge&amp;author=D.%20Porter&amp;volume=98&amp;publication_year=2006&amp;pages=860-871&amp;pmid=17486963&amp;doi=10.1080/15572536.2006.11832616&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6-microorganisms-11-01830">
<span class="label">6.</span><cite>Spatafora J.W., Chang Y., Benny G.L., Lazarus K., Smith M.E., Berbee M.L., Bonito G., Corradi N., Grigoriev I., Gryganskyi A., et al.  A Phylum-Level Phylogenetic Classification of Zygomycete Fungi Based on Genome-Scale Data. Mycologia. 2016;108:1028–1046. doi: 10.3852/16-042.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3852/16-042" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6078412/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27738200/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=A%20Phylum-Level%20Phylogenetic%20Classification%20of%20Zygomycete%20Fungi%20Based%20on%20Genome-Scale%20Data&amp;author=J.W.%20Spatafora&amp;author=Y.%20Chang&amp;author=G.L.%20Benny&amp;author=K.%20Lazarus&amp;author=M.E.%20Smith&amp;volume=108&amp;publication_year=2016&amp;pages=1028-1046&amp;pmid=27738200&amp;doi=10.3852/16-042&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7-microorganisms-11-01830">
<span class="label">7.</span><cite>Nagy L.G., Kovács G.M., Krizsán K. Complex Multicellularity in Fungi: Evolutionary Convergence, Single Origin, or Both? Biol. Rev. Camb. Philos. Soc. 2018;93:1778–1794. doi: 10.1111/brv.12418.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/brv.12418" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29675836/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biol.%20Rev.%20Camb.%20Philos.%20Soc.&amp;title=Complex%20Multicellularity%20in%20Fungi:%20Evolutionary%20Convergence,%20Single%20Origin,%20or%20Both?&amp;author=L.G.%20Nagy&amp;author=G.M.%20Kov%C3%A1cs&amp;author=K.%20Krizs%C3%A1n&amp;volume=93&amp;publication_year=2018&amp;pages=1778-1794&amp;pmid=29675836&amp;doi=10.1111/brv.12418&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8-microorganisms-11-01830">
<span class="label">8.</span><cite>Kiss E., Hegedüs B., Virágh M., Varga T., Merényi Z., Kószó T., Bálint B., Prasanna A.N., Krizsán K., Kocsubé S., et al.  Comparative Genomics Reveals the Origin of Fungal Hyphae and Multicellularity. Nat. Commun. 2019;10:4080. doi: 10.1038/s41467-019-12085-w.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41467-019-12085-w" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6733946/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31501435/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Comparative%20Genomics%20Reveals%20the%20Origin%20of%20Fungal%20Hyphae%20and%20Multicellularity&amp;author=E.%20Kiss&amp;author=B.%20Heged%C3%BCs&amp;author=M.%20Vir%C3%A1gh&amp;author=T.%20Varga&amp;author=Z.%20Mer%C3%A9nyi&amp;volume=10&amp;publication_year=2019&amp;pages=4080&amp;pmid=31501435&amp;doi=10.1038/s41467-019-12085-w&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9-microorganisms-11-01830">
<span class="label">9.</span><cite>Stajich J.E. Fungal Genomes and Insights into the Evolution of the Kingdom. Microbiol. Spectr. 2017;5:1–25. doi: 10.1128/microbiolspec.FUNK-0055-2016.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/microbiolspec.FUNK-0055-2016" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6078396/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28820125/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Spectr.&amp;title=Fungal%20Genomes%20and%20Insights%20into%20the%20Evolution%20of%20the%20Kingdom&amp;author=J.E.%20Stajich&amp;volume=5&amp;publication_year=2017&amp;pages=1-25&amp;pmid=28820125&amp;doi=10.1128/microbiolspec.FUNK-0055-2016&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10-microorganisms-11-01830">
<span class="label">10.</span><cite>Berbee M.L., James T.Y., Strullu-Derrien C. Early Diverging Fungi: Diversity and Impact at the Dawn of Terrestrial Life. Annu. Rev. Microbiol. 2017;71:41–60. doi: 10.1146/annurev-micro-030117-020324.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev-micro-030117-020324" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28525299/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Microbiol.&amp;title=Early%20Diverging%20Fungi:%20Diversity%20and%20Impact%20at%20the%20Dawn%20of%20Terrestrial%20Life&amp;author=M.L.%20Berbee&amp;author=T.Y.%20James&amp;author=C.%20Strullu-Derrien&amp;volume=71&amp;publication_year=2017&amp;pages=41-60&amp;pmid=28525299&amp;doi=10.1146/annurev-micro-030117-020324&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11-microorganisms-11-01830">
<span class="label">11.</span><cite>Ahrendt S.R., Quandt C.A., Ciobanu D., Clum A., Salamov A., Andreopoulos B., Cheng J.-F., Woyke T., Pelin A., Henrissat B., et al.  Leveraging Single-Cell Genomics to Expand the Fungal Tree of Life. Nat. Microbiol. 2018;3:1417–1428. doi: 10.1038/s41564-018-0261-0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41564-018-0261-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6784888/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30297742/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Microbiol.&amp;title=Leveraging%20Single-Cell%20Genomics%20to%20Expand%20the%20Fungal%20Tree%20of%20Life&amp;author=S.R.%20Ahrendt&amp;author=C.A.%20Quandt&amp;author=D.%20Ciobanu&amp;author=A.%20Clum&amp;author=A.%20Salamov&amp;volume=3&amp;publication_year=2018&amp;pages=1417-1428&amp;pmid=30297742&amp;doi=10.1038/s41564-018-0261-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12-microorganisms-11-01830">
<span class="label">12.</span><cite>Vandepol N., Liber J., Desirò A., Na H., Kennedy M., Barry K., Grigoriev I.V., Miller A.N., O’Donnell K., Stajich J.E., et al.  Resolving the Mortierellaceae Phylogeny through Synthesis of Multi-Gene Phylogenetics and Phylogenomics. Fungal Divers. 2020;104:267–289. doi: 10.1007/s13225-020-00455-5.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s13225-020-00455-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7751987/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33364917/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Divers.&amp;title=Resolving%20the%20Mortierellaceae%20Phylogeny%20through%20Synthesis%20of%20Multi-Gene%20Phylogenetics%20and%20Phylogenomics&amp;author=N.%20Vandepol&amp;author=J.%20Liber&amp;author=A.%20Desir%C3%B2&amp;author=H.%20Na&amp;author=M.%20Kennedy&amp;volume=104&amp;publication_year=2020&amp;pages=267-289&amp;pmid=33364917&amp;doi=10.1007/s13225-020-00455-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13-microorganisms-11-01830">
<span class="label">13.</span><cite>Chang M.C., Hur J., Park D. Interpreting the COVID-19 Test Results: A Guide for Physiatrists. Am. J. Phys. Med. Rehabil. 2020;99:583–585. doi: 10.1097/PHM.0000000000001471.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1097/PHM.0000000000001471" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7268832/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32427600/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Phys.%20Med.%20Rehabil.&amp;title=Interpreting%20the%20COVID-19%20Test%20Results:%20A%20Guide%20for%20Physiatrists&amp;author=M.C.%20Chang&amp;author=J.%20Hur&amp;author=D.%20Park&amp;volume=99&amp;publication_year=2020&amp;pages=583-585&amp;pmid=32427600&amp;doi=10.1097/PHM.0000000000001471&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14-microorganisms-11-01830">
<span class="label">14.</span><cite>Spatafora J.W., Aime M.C., Grigoriev I.V., Stajich J.E., Blackwell M. The Fungal Tree of Life: From Molecular Systematics to Genome-Scale Phylogenies. Microbiol. Spectrum. 2017;5:1–32. doi: 10.1128/microbiolspec.FUNK-0053-2016.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/microbiolspec.FUNK-0053-2016" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC11687545/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28917057/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Spectrum.&amp;title=The%20Fungal%20Tree%20of%20Life:%20From%20Molecular%20Systematics%20to%20Genome-Scale%20Phylogenies&amp;author=J.W.%20Spatafora&amp;author=M.C.%20Aime&amp;author=I.V.%20Grigoriev&amp;author=J.E.%20Stajich&amp;author=M.%20Blackwell&amp;volume=5&amp;publication_year=2017&amp;pages=1-32&amp;pmid=28917057&amp;doi=10.1128/microbiolspec.FUNK-0053-2016&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15-microorganisms-11-01830">
<span class="label">15.</span><cite>Reynolds N.K., Stajich J.E., Benny G.L., Barry K., Mondo S., LaButti K., Lipzen A., Daum C., Grigoriev I.V., Ho H.-M., et al.  Mycoparasites, Gut Dwellers, and Saprotrophs: Phylogenomic Reconstructions and Comparative Analyses of Kickxellomycotina Fungi. Genome Biol. Evol. 2023;15:evac185. doi: 10.1093/gbe/evac185.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/gbe/evac185" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9866270/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36617272/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&amp;title=Mycoparasites,%20Gut%20Dwellers,%20and%20Saprotrophs:%20Phylogenomic%20Reconstructions%20and%20Comparative%20Analyses%20of%20Kickxellomycotina%20Fungi&amp;author=N.K.%20Reynolds&amp;author=J.E.%20Stajich&amp;author=G.L.%20Benny&amp;author=K.%20Barry&amp;author=S.%20Mondo&amp;volume=15&amp;publication_year=2023&amp;pages=evac185&amp;pmid=36617272&amp;doi=10.1093/gbe/evac185&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16-microorganisms-11-01830">
<span class="label">16.</span><cite>Gryganskyi A.P., Muszewska A. Whole Genome Sequencing and the Zygomycota. Fungal Genom. Biol. 2014;4:e116. doi: 10.4172/2165-8056.1000e116.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4172/2165-8056.1000e116" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genom.%20Biol.&amp;title=Whole%20Genome%20Sequencing%20and%20the%20Zygomycota&amp;author=A.P.%20Gryganskyi&amp;author=A.%20Muszewska&amp;volume=4&amp;publication_year=2014&amp;pages=e116&amp;doi=10.4172/2165-8056.1000e116&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17-microorganisms-11-01830">
<span class="label">17.</span><cite>Stephens Z.D., Lee S.Y., Faghri F., Campbell R.H., Zhai C., Efron M.J., Iyer R., Schatz M.C., Sinha S., Robinson G.E. Big Data: Astronomical or Genomical? PLoS Biol. 2015;13:e1002195.  doi: 10.1371/journal.pbio.1002195.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pbio.1002195" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4494865/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26151137/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&amp;title=Big%20Data:%20Astronomical%20or%20Genomical?&amp;author=Z.D.%20Stephens&amp;author=S.Y.%20Lee&amp;author=F.%20Faghri&amp;author=R.H.%20Campbell&amp;author=C.%20Zhai&amp;volume=13&amp;publication_year=2015&amp;pages=e1002195&amp;pmid=26151137&amp;doi=10.1371/journal.pbio.1002195&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18-microorganisms-11-01830">
<span class="label">18.</span><cite>Ma L.-J., Ibrahim A.S., Skory C., Grabherr M.G., Burger G., Butler M., Elias M., Idnurm A., Lang B.F., Sone T., et al.  Genomic Analysis of the Basal Lineage Fungus Rhizopus oryzae Reveals a Whole-Genome Duplication. PLoS Genet. 2009;5:e1000549.  doi: 10.1371/journal.pgen.1000549.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pgen.1000549" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2699053/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19578406/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Genomic%20Analysis%20of%20the%20Basal%20Lineage%20Fungus%20Rhizopus%20oryzae%20Reveals%20a%20Whole-Genome%20Duplication&amp;author=L.-J.%20Ma&amp;author=A.S.%20Ibrahim&amp;author=C.%20Skory&amp;author=M.G.%20Grabherr&amp;author=G.%20Burger&amp;volume=5&amp;publication_year=2009&amp;pages=e1000549&amp;pmid=19578406&amp;doi=10.1371/journal.pgen.1000549&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19-microorganisms-11-01830">
<span class="label">19.</span><cite>Zhang D., Yang Y., Castlebury L.A., Cerniglia C.E. A Method for the Large Scale Isolation of High Transformation Efficiency Fungal Genomic DNA. FEMS Microbiol. Lett. 1996;145:261–265. doi: 10.1111/j.1574-6968.1996.tb08587.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1574-6968.1996.tb08587.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8961565/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Lett.&amp;title=A%20Method%20for%20the%20Large%20Scale%20Isolation%20of%20High%20Transformation%20Efficiency%20Fungal%20Genomic%20DNA&amp;author=D.%20Zhang&amp;author=Y.%20Yang&amp;author=L.A.%20Castlebury&amp;author=C.E.%20Cerniglia&amp;volume=145&amp;publication_year=1996&amp;pages=261-265&amp;pmid=8961565&amp;doi=10.1111/j.1574-6968.1996.tb08587.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B20-microorganisms-11-01830">
<span class="label">20.</span><cite>Davis W.J., Amses K.R., Benny G.L., Carter-House D., Chang Y., Grigoriev I., Smith M.E., Spatafora J.W., Stajich J.E., James T.Y. Genome-Scale Phylogenetics Reveals a Monophyletic Zoopagales (Zoopagomycota, Fungi) Mol. Phylogenetics Evol. 2019;133:152–163. doi: 10.1016/j.ympev.2019.01.006.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ympev.2019.01.006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6886740/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30639767/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Phylogenetics%20Evol.&amp;title=Genome-Scale%20Phylogenetics%20Reveals%20a%20Monophyletic%20Zoopagales%20(Zoopagomycota,%20Fungi)&amp;author=W.J.%20Davis&amp;author=K.R.%20Amses&amp;author=G.L.%20Benny&amp;author=D.%20Carter-House&amp;author=Y.%20Chang&amp;volume=133&amp;publication_year=2019&amp;pages=152-163&amp;pmid=30639767&amp;doi=10.1016/j.ympev.2019.01.006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21-microorganisms-11-01830">
<span class="label">21.</span><cite>Montoliu-Nerin M., Sánchez-García M., Bergin C., Grabherr M., Ellis B., Kutschera V.E., Kierczak M., Johannesson H., Rosling A. Building de Novo Reference Genome Assemblies of Complex Eukaryotic Microorganisms from Single Nuclei. Sci. Rep. 2020;10:1303. doi: 10.1038/s41598-020-58025-3.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-020-58025-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6987183/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31992756/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Building%20de%20Novo%20Reference%20Genome%20Assemblies%20of%20Complex%20Eukaryotic%20Microorganisms%20from%20Single%20Nuclei&amp;author=M.%20Montoliu-Nerin&amp;author=M.%20S%C3%A1nchez-Garc%C3%ADa&amp;author=C.%20Bergin&amp;author=M.%20Grabherr&amp;author=B.%20Ellis&amp;volume=10&amp;publication_year=2020&amp;pages=1303&amp;pmid=31992756&amp;doi=10.1038/s41598-020-58025-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22-microorganisms-11-01830">
<span class="label">22.</span><cite>Lin K., Limpens E., Zhang Z., Ivanov S., Saunders D.G.O., Mu D., Pang E., Cao H., Cha H., Lin T., et al.  Single Nucleus Genome Sequencing Reveals High Similarity among Nuclei of an Endomycorrhizal Fungus. PLoS Genet. 2014;10:e1004078.  doi: 10.1371/journal.pgen.1004078.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pgen.1004078" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3886924/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24415955/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Single%20Nucleus%20Genome%20Sequencing%20Reveals%20High%20Similarity%20among%20Nuclei%20of%20an%20Endomycorrhizal%20Fungus&amp;author=K.%20Lin&amp;author=E.%20Limpens&amp;author=Z.%20Zhang&amp;author=S.%20Ivanov&amp;author=D.G.O.%20Saunders&amp;volume=10&amp;publication_year=2014&amp;pages=e1004078&amp;pmid=24415955&amp;doi=10.1371/journal.pgen.1004078&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23-microorganisms-11-01830">
<span class="label">23.</span><cite>Małagocka J., Grell M.N., Lange L., Eilenberg J., Jensen A.B. Transcriptome of an Entomophthoralean Fungus (Pandora formicae) Shows Molecular Machinery Adjusted for Successful Host Exploitation and Transmission. J. Invertebr. Pathol. 2015;128:47–56. doi: 10.1016/j.jip.2015.05.001.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jip.2015.05.001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25968105/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Invertebr.%20Pathol.&amp;title=Transcriptome%20of%20an%20Entomophthoralean%20Fungus%20(Pandora%20formicae)%20Shows%20Molecular%20Machinery%20Adjusted%20for%20Successful%20Host%20Exploitation%20and%20Transmission&amp;author=J.%20Ma%C5%82agocka&amp;author=M.N.%20Grell&amp;author=L.%20Lange&amp;author=J.%20Eilenberg&amp;author=A.B.%20Jensen&amp;volume=128&amp;publication_year=2015&amp;pages=47-56&amp;pmid=25968105&amp;doi=10.1016/j.jip.2015.05.001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24-microorganisms-11-01830">
<span class="label">24.</span><cite>Chang Y., Desirò A., Na H., Sandor L., Lipzen A., Clum A., Barry K., Grigoriev I.V., Martin F.M., Stajich J.E., et al.  Phylogenomics of Endogonaceae and Evolution of Mycorrhizas within Mucoromycota. New Phytol. 2019;222:511–525. doi: 10.1111/nph.15613.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/nph.15613" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30485448/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Phylogenomics%20of%20Endogonaceae%20and%20Evolution%20of%20Mycorrhizas%20within%20Mucoromycota&amp;author=Y.%20Chang&amp;author=A.%20Desir%C3%B2&amp;author=H.%20Na&amp;author=L.%20Sandor&amp;author=A.%20Lipzen&amp;volume=222&amp;publication_year=2019&amp;pages=511-525&amp;pmid=30485448&amp;doi=10.1111/nph.15613&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25-microorganisms-11-01830">
<span class="label">25.</span><cite>Carneiro M.O., Russ C., Ross M.G., Gabriel S.B., Nusbaum C., DePristo M.A. Pacific Biosciences Sequencing Technology for Genotyping and Variation Discovery in Human Data. BMC Genom. 2012;13:375.  doi: 10.1186/1471-2164-13-375.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2164-13-375" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3443046/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22863213/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&amp;title=Pacific%20Biosciences%20Sequencing%20Technology%20for%20Genotyping%20and%20Variation%20Discovery%20in%20Human%20Data&amp;author=M.O.%20Carneiro&amp;author=C.%20Russ&amp;author=M.G.%20Ross&amp;author=S.B.%20Gabriel&amp;author=C.%20Nusbaum&amp;volume=13&amp;publication_year=2012&amp;pages=375&amp;pmid=22863213&amp;doi=10.1186/1471-2164-13-375&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26-microorganisms-11-01830">
<span class="label">26.</span><cite>Maeda T., Kobayashi Y., Kameoka H., Okuma N., Takeda N., Yamaguchi K., Bino T., Shigenobu S., Kawaguchi M. Evidence of Non-Tandemly Repeated RDNAs and Their Intragenomic Heterogeneity in Rhizophagus irregularis. Commun. Biol. 2018;1:87. doi: 10.1038/s42003-018-0094-7.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s42003-018-0094-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6123716/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30271968/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Commun.%20Biol.&amp;title=Evidence%20of%20Non-Tandemly%20Repeated%20RDNAs%20and%20Their%20Intragenomic%20Heterogeneity%20in%20Rhizophagus%20irregularis&amp;author=T.%20Maeda&amp;author=Y.%20Kobayashi&amp;author=H.%20Kameoka&amp;author=N.%20Okuma&amp;author=N.%20Takeda&amp;volume=1&amp;publication_year=2018&amp;pages=87&amp;pmid=30271968&amp;doi=10.1038/s42003-018-0094-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27-microorganisms-11-01830">
<span class="label">27.</span><cite>Segerman B. The Most Frequently Used Sequencing Technologies and Assembly Methods in Different Time Segments of the Bacterial Surveillance and RefSeq Genome Databases. Front. Cell. Infect. Microbiol. 2020;10:527102. doi: 10.3389/fcimb.2020.527102.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fcimb.2020.527102" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7604302/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33194784/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell.%20Infect.%20Microbiol.&amp;title=The%20Most%20Frequently%20Used%20Sequencing%20Technologies%20and%20Assembly%20Methods%20in%20Different%20Time%20Segments%20of%20the%20Bacterial%20Surveillance%20and%20RefSeq%20Genome%20Databases&amp;author=B.%20Segerman&amp;volume=10&amp;publication_year=2020&amp;pages=527102&amp;pmid=33194784&amp;doi=10.3389/fcimb.2020.527102&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28-microorganisms-11-01830">
<span class="label">28.</span><cite>El-Metwally S., Hamza T., Zakaria M., Helmy M. Next-Generation Sequence Assembly: Four Stages of Data Processing and Computational Challenges. PLoS Comput. Biol. 2013;9:e1003345.  doi: 10.1371/journal.pcbi.1003345.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pcbi.1003345" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3861042/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24348224/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&amp;title=Next-Generation%20Sequence%20Assembly:%20Four%20Stages%20of%20Data%20Processing%20and%20Computational%20Challenges&amp;author=S.%20El-Metwally&amp;author=T.%20Hamza&amp;author=M.%20Zakaria&amp;author=M.%20Helmy&amp;volume=9&amp;publication_year=2013&amp;pages=e1003345&amp;pmid=24348224&amp;doi=10.1371/journal.pcbi.1003345&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29-microorganisms-11-01830">
<span class="label">29.</span><cite>Zerbino D.R., Birney E. Velvet: Algorithms for de Novo Short Read Assembly Using de Bruijn Graphs. Genome Res. 2008;18:821–829. doi: 10.1101/gr.074492.107.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.074492.107" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2336801/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18349386/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Velvet:%20Algorithms%20for%20de%20Novo%20Short%20Read%20Assembly%20Using%20de%20Bruijn%20Graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;volume=18&amp;publication_year=2008&amp;pages=821-829&amp;pmid=18349386&amp;doi=10.1101/gr.074492.107&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30-microorganisms-11-01830">
<span class="label">30.</span><cite>Myers J.M., Bonds A.E., Clemons R.A., Thapa N.A., Simmons D.R., Carter-House D., Ortanez J., Liu P., Miralles-Durán A., Desirò A., et al.  Survey of Early-Diverging Lineages of Fungi Reveals Abundant and Diverse Mycoviruses. mBio. 2020;11:e02027-20. doi: 10.1128/mBio.02027-20.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/mBio.02027-20" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7482067/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32900807/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Survey%20of%20Early-Diverging%20Lineages%20of%20Fungi%20Reveals%20Abundant%20and%20Diverse%20Mycoviruses&amp;author=J.M.%20Myers&amp;author=A.E.%20Bonds&amp;author=R.A.%20Clemons&amp;author=N.A.%20Thapa&amp;author=D.R.%20Simmons&amp;volume=11&amp;publication_year=2020&amp;pages=e02027-20&amp;pmid=32900807&amp;doi=10.1128/mBio.02027-20&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31-microorganisms-11-01830">
<span class="label">31.</span><cite>Chin C.-S., Alexander D.H., Marks P., Klammer A.A., Drake J., Heiner C., Clum A., Copeland A., Huddleston J., Eichler E.E., et al.  Nonhybrid, Finished Microbial Genome Assemblies from Long-Read SMRT Sequencing Data. Nat. Methods. 2013;10:563–569. doi: 10.1038/nmeth.2474.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.2474" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23644548/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Nonhybrid,%20Finished%20Microbial%20Genome%20Assemblies%20from%20Long-Read%20SMRT%20Sequencing%20Data&amp;author=C.-S.%20Chin&amp;author=D.H.%20Alexander&amp;author=P.%20Marks&amp;author=A.A.%20Klammer&amp;author=J.%20Drake&amp;volume=10&amp;publication_year=2013&amp;pages=563-569&amp;pmid=23644548&amp;doi=10.1038/nmeth.2474&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32-microorganisms-11-01830">
<span class="label">32.</span><cite>Shapiro H. Outline of the Assembly Process: JAZZ, the JGI In-House Assembler.  [(accessed on 13 March 2023)].  Available online:  <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://escholarship.org/uc/item/0g8792vf" rel="noopener noreferrer" target="_blank">https://escholarship.org/uc/item/0g8792vf</a>.</cite>
</li>
<li id="B33-microorganisms-11-01830">
<span class="label">33.</span><cite>Boisvert S., Raymond F., Godzaridis É., Laviolette F., Corbeil J. Ray Meta: Scalable de Novo Metagenome Assembly and Profiling. Genome Biol. 2012;13:R122. doi: 10.1186/gb-2012-13-12-r122.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/gb-2012-13-12-r122" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4056372/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23259615/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Ray%20Meta:%20Scalable%20de%20Novo%20Metagenome%20Assembly%20and%20Profiling&amp;author=S.%20Boisvert&amp;author=F.%20Raymond&amp;author=%C3%89.%20Godzaridis&amp;author=F.%20Laviolette&amp;author=J.%20Corbeil&amp;volume=13&amp;publication_year=2012&amp;pages=R122&amp;pmid=23259615&amp;doi=10.1186/gb-2012-13-12-r122&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34-microorganisms-11-01830">
<span class="label">34.</span><cite>Zimin A.V., Marçais G., Puiu D., Roberts M., Salzberg S.L., Yorke J.A. The MaSuRCA Genome Assembler. Bioinformatics. 2013;29:2669–2677. doi: 10.1093/bioinformatics/btt476.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btt476" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3799473/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23990416/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The%20MaSuRCA%20Genome%20Assembler&amp;author=A.V.%20Zimin&amp;author=G.%20Mar%C3%A7ais&amp;author=D.%20Puiu&amp;author=M.%20Roberts&amp;author=S.L.%20Salzberg&amp;volume=29&amp;publication_year=2013&amp;pages=2669-2677&amp;pmid=23990416&amp;doi=10.1093/bioinformatics/btt476&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35-microorganisms-11-01830">
<span class="label">35.</span><cite>Butler J., MacCallum I., Kleber M., Shlyakhter I.A., Belmonte M.K., Lander E.S., Nusbaum C., Jaffe D.B. ALLPATHS: De Novo Assembly of Whole-Genome Shotgun Microreads. Genome Res. 2008;18:810–820. doi: 10.1101/gr.7337908.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.7337908" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2336810/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18340039/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=ALLPATHS:%20De%20Novo%20Assembly%20of%20Whole-Genome%20Shotgun%20Microreads&amp;author=J.%20Butler&amp;author=I.%20MacCallum&amp;author=M.%20Kleber&amp;author=I.A.%20Shlyakhter&amp;author=M.K.%20Belmonte&amp;volume=18&amp;publication_year=2008&amp;pages=810-820&amp;pmid=18340039&amp;doi=10.1101/gr.7337908&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36-microorganisms-11-01830">
<span class="label">36.</span><cite>Luo R., Liu B., Xie Y., Li Z., Huang W., Yuan J., He G., Chen Y., Pan Q., Liu Y., et al.  SOAPdenovo2: An Empirically Improved Memory-Efficient Short-Read de Novo Assembler. Gigascience. 2012;1:18. doi: 10.1186/2047-217X-1-18.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/2047-217X-1-18" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3626529/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23587118/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=SOAPdenovo2:%20An%20Empirically%20Improved%20Memory-Efficient%20Short-Read%20de%20Novo%20Assembler&amp;author=R.%20Luo&amp;author=B.%20Liu&amp;author=Y.%20Xie&amp;author=Z.%20Li&amp;author=W.%20Huang&amp;volume=1&amp;publication_year=2012&amp;pages=18&amp;pmid=23587118&amp;doi=10.1186/2047-217X-1-18&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B37-microorganisms-11-01830">
<span class="label">37.</span><cite>Bankevich A., Nurk S., Antipov D., Gurevich A.A., Dvorkin M., Kulikov A.S., Lesin V.M., Nikolenko S.I., Pham S., Prjibelski A.D., et al.  SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing. J. Comput. Biol. 2012;19:455–477. doi: 10.1089/cmb.2012.0021.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/cmb.2012.0021" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3342519/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22506599/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Biol.&amp;title=SPAdes:%20A%20New%20Genome%20Assembly%20Algorithm%20and%20Its%20Applications%20to%20Single-Cell%20Sequencing&amp;author=A.%20Bankevich&amp;author=S.%20Nurk&amp;author=D.%20Antipov&amp;author=A.A.%20Gurevich&amp;author=M.%20Dvorkin&amp;volume=19&amp;publication_year=2012&amp;pages=455-477&amp;pmid=22506599&amp;doi=10.1089/cmb.2012.0021&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B38-microorganisms-11-01830">
<span class="label">38.</span><cite>Miller J.R., Koren S., Sutton G. Assembly Algorithms for Next-Generation Sequencing Data. Genomics. 2010;95:315–327. doi: 10.1016/j.ygeno.2010.03.001.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ygeno.2010.03.001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2874646/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20211242/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Assembly%20Algorithms%20for%20Next-Generation%20Sequencing%20Data&amp;author=J.R.%20Miller&amp;author=S.%20Koren&amp;author=G.%20Sutton&amp;volume=95&amp;publication_year=2010&amp;pages=315-327&amp;pmid=20211242&amp;doi=10.1016/j.ygeno.2010.03.001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B39-microorganisms-11-01830">
<span class="label">39.</span><cite>Koren S., Walenz B.P., Berlin K., Miller J.R., Bergman N.H., Phillippy A.M. Canu: Scalable and Accurate Long-Read Assembly via Adaptive k-Mer Weighting and Repeat Separation. Genome Res. 2017;27:722–736. doi: 10.1101/gr.215087.116.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.215087.116" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5411767/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28298431/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Canu:%20Scalable%20and%20Accurate%20Long-Read%20Assembly%20via%20Adaptive%20k-Mer%20Weighting%20and%20Repeat%20Separation&amp;author=S.%20Koren&amp;author=B.P.%20Walenz&amp;author=K.%20Berlin&amp;author=J.R.%20Miller&amp;author=N.H.%20Bergman&amp;volume=27&amp;publication_year=2017&amp;pages=722-736&amp;pmid=28298431&amp;doi=10.1101/gr.215087.116&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B40-microorganisms-11-01830">
<span class="label">40.</span><cite>Lam K.-K., LaButti K., Khalak A., Tse D. FinisherSC: A Repeat-Aware Tool for Upgrading de Novo Assembly Using Long Reads. Bioinformatics. 2015;31:3207–3209. doi: 10.1093/bioinformatics/btv280.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btv280" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26040454/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=FinisherSC:%20A%20Repeat-Aware%20Tool%20for%20Upgrading%20de%20Novo%20Assembly%20Using%20Long%20Reads&amp;author=K.-K.%20Lam&amp;author=K.%20LaButti&amp;author=A.%20Khalak&amp;author=D.%20Tse&amp;volume=31&amp;publication_year=2015&amp;pages=3207-3209&amp;pmid=26040454&amp;doi=10.1093/bioinformatics/btv280&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B41-microorganisms-11-01830">
<span class="label">41.</span><cite>Kolmogorov M., Yuan J., Lin Y., Pevzner P.A. Assembly of Long, Error-Prone Reads Using Repeat Graphs. Nat. Biotechnol. 2019;37:540–546. doi: 10.1038/s41587-019-0072-8.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41587-019-0072-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30936562/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=Assembly%20of%20Long,%20Error-Prone%20Reads%20Using%20Repeat%20Graphs&amp;author=M.%20Kolmogorov&amp;author=J.%20Yuan&amp;author=Y.%20Lin&amp;author=P.A.%20Pevzner&amp;volume=37&amp;publication_year=2019&amp;pages=540-546&amp;pmid=30936562&amp;doi=10.1038/s41587-019-0072-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B42-microorganisms-11-01830">
<span class="label">42.</span><cite>Li D., Liu C.-M., Luo R., Sadakane K., Lam T.-W. MEGAHIT: An Ultra-Fast Single-Node Solution for Large and Complex Metagenomics Assembly via Succinct de Bruijn Graph. Bioinformatics. 2015;31:1674–1676. doi: 10.1093/bioinformatics/btv033.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btv033" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25609793/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MEGAHIT:%20An%20Ultra-Fast%20Single-Node%20Solution%20for%20Large%20and%20Complex%20Metagenomics%20Assembly%20via%20Succinct%20de%20Bruijn%20Graph&amp;author=D.%20Li&amp;author=C.-M.%20Liu&amp;author=R.%20Luo&amp;author=K.%20Sadakane&amp;author=T.-W.%20Lam&amp;volume=31&amp;publication_year=2015&amp;pages=1674-1676&amp;pmid=25609793&amp;doi=10.1093/bioinformatics/btv033&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B43-microorganisms-11-01830">
<span class="label">43.</span><cite>Gregory T.R.  CHAPTER 1—Genome Size Evolution in Animals. In: Gregory T.R., editor. The Evolution of the Genome. Academic Press; Burlington, NJ, USA: 2005. pp. 3–87.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=The%20Evolution%20of%20the%20Genome&amp;author=T.R.%20Gregory&amp;publication_year=2005&amp;isbn=978-0-12-301463-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B44-microorganisms-11-01830">
<span class="label">44.</span><cite>Pellicer J., Hidalgo O., Dodsworth S., Leitch I.J. Genome Size Diversity and Its Impact on the Evolution of Land Plants. Genes. 2018;9:88.  doi: 10.3390/genes9020088.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/genes9020088" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5852584/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29443885/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=Genome%20Size%20Diversity%20and%20Its%20Impact%20on%20the%20Evolution%20of%20Land%20Plants&amp;author=J.%20Pellicer&amp;author=O.%20Hidalgo&amp;author=S.%20Dodsworth&amp;author=I.J.%20Leitch&amp;volume=9&amp;publication_year=2018&amp;pages=88&amp;pmid=29443885&amp;doi=10.3390/genes9020088&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B45-microorganisms-11-01830">
<span class="label">45.</span><cite>Tavares S., Ramos A.P., Pires A.S., Azinheira H.G., Caldeirinha P., Link T., Abranches R., do Céu Silva M., Voegele R.T., Loureiro J., et al.  Genome Size Analyses of Pucciniales Reveal the Largest Fungal Genomes. Front. Plant Sci. 2014;5:422. doi: 10.3389/fpls.2014.00422.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2014.00422" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4143883/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25206357/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=Genome%20Size%20Analyses%20of%20Pucciniales%20Reveal%20the%20Largest%20Fungal%20Genomes&amp;author=S.%20Tavares&amp;author=A.P.%20Ramos&amp;author=A.S.%20Pires&amp;author=H.G.%20Azinheira&amp;author=P.%20Caldeirinha&amp;volume=5&amp;publication_year=2014&amp;pages=422&amp;pmid=25206357&amp;doi=10.3389/fpls.2014.00422&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B46-microorganisms-11-01830">
<span class="label">46.</span><cite>Salazar A.N., Gorter de Vries A.R., van den Broek M., Brouwers N., de la Torre Cortès P., Kuijpers N.G.A., Daran J.-M.G., Abeel T. Chromosome Level Assembly and Comparative Genome Analysis Confirm Lager-Brewing Yeasts Originated from a Single Hybridization. BMC Genom. 2019;20:916.  doi: 10.1186/s12864-019-6263-3.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12864-019-6263-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6889557/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31791228/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&amp;title=Chromosome%20Level%20Assembly%20and%20Comparative%20Genome%20Analysis%20Confirm%20Lager-Brewing%20Yeasts%20Originated%20from%20a%20Single%20Hybridization&amp;author=A.N.%20Salazar&amp;author=A.R.%20Gorter%20de%20Vries&amp;author=M.%20van%20den%20Broek&amp;author=N.%20Brouwers&amp;author=P.%20de%20la%20Torre%20Cort%C3%A8s&amp;volume=20&amp;publication_year=2019&amp;pages=916&amp;pmid=31791228&amp;doi=10.1186/s12864-019-6263-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B47-microorganisms-11-01830">
<span class="label">47.</span><cite>Wieloch W. Chromosome Visualisation in Filamentous Fungi. J. Microbiol. Methods. 2006;67:1–8. doi: 10.1016/j.mimet.2006.05.022.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.mimet.2006.05.022" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16870283/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Microbiol.%20Methods&amp;title=Chromosome%20Visualisation%20in%20Filamentous%20Fungi&amp;author=W.%20Wieloch&amp;volume=67&amp;publication_year=2006&amp;pages=1-8&amp;pmid=16870283&amp;doi=10.1016/j.mimet.2006.05.022&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B48-microorganisms-11-01830">
<span class="label">48.</span><cite>Boyce G.R., Gluck-Thaler E., Slot J.C., Stajich J.E., Davis W.J., James T.Y., Cooley J.R., Panaccione D.G., Eilenberg J., De Fine Licht H.H., et al.  Psychoactive Plant- and Mushroom-Associated Alkaloids from Two Behavior Modifying Cicada Pathogens. Fungal Ecol. 2019;41:147–164. doi: 10.1016/j.funeco.2019.06.002.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.funeco.2019.06.002" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6876628/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31768192/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Ecol.&amp;title=Psychoactive%20Plant-%20and%20Mushroom-Associated%20Alkaloids%20from%20Two%20Behavior%20Modifying%20Cicada%20Pathogens&amp;author=G.R.%20Boyce&amp;author=E.%20Gluck-Thaler&amp;author=J.C.%20Slot&amp;author=J.E.%20Stajich&amp;author=W.J.%20Davis&amp;volume=41&amp;publication_year=2019&amp;pages=147-164&amp;pmid=31768192&amp;doi=10.1016/j.funeco.2019.06.002&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B49-microorganisms-11-01830">
<span class="label">49.</span><cite>Vinson J.P., Jaffe D.B., O’Neill K., Karlsson E.K., Stange-Thomann N., Anderson S., Mesirov J.P., Satoh N., Satou Y., Nusbaum C., et al.  Assembly of Polymorphic Genomes: Algorithms and Application to Ciona savignyi. Genome Res. 2005;15:1127–1135. doi: 10.1101/gr.3722605.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.3722605" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1182225/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16077012/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Assembly%20of%20Polymorphic%20Genomes:%20Algorithms%20and%20Application%20to%20Ciona%20savignyi&amp;author=J.P.%20Vinson&amp;author=D.B.%20Jaffe&amp;author=K.%20O%E2%80%99Neill&amp;author=E.K.%20Karlsson&amp;author=N.%20Stange-Thomann&amp;volume=15&amp;publication_year=2005&amp;pages=1127-1135&amp;pmid=16077012&amp;doi=10.1101/gr.3722605&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B50-microorganisms-11-01830">
<span class="label">50.</span><cite>Chibucos M.C., Soliman S., Gebremariam T., Lee H., Daugherty S., Orvis J., Shetty A.C., Crabtree J., Hazen T.H., Etienne K.A., et al.  An Integrated Genomic and Transcriptomic Survey of Mucormycosis-Causing Fungi. Nat. Commun. 2016;7:12218. doi: 10.1038/ncomms12218.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ncomms12218" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4961843/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27447865/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=An%20Integrated%20Genomic%20and%20Transcriptomic%20Survey%20of%20Mucormycosis-Causing%20Fungi&amp;author=M.C.%20Chibucos&amp;author=S.%20Soliman&amp;author=T.%20Gebremariam&amp;author=H.%20Lee&amp;author=S.%20Daugherty&amp;volume=7&amp;publication_year=2016&amp;pages=12218&amp;pmid=27447865&amp;doi=10.1038/ncomms12218&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B51-microorganisms-11-01830">
<span class="label">51.</span><cite>Li Y., Steenwyk J.L., Chang Y., Wang Y., James T.Y., Stajich J.E., Spatafora J.W., Groenewald M., Dunn C.W., Hittinger C.T., et al.  A Genome-Scale Phylogeny of the Kingdom Fungi. Curr. Biol. 2021;31:1653–1665.e5. doi: 10.1016/j.cub.2021.01.074.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cub.2021.01.074" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8347878/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33607033/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=A%20Genome-Scale%20Phylogeny%20of%20the%20Kingdom%20Fungi&amp;author=Y.%20Li&amp;author=J.L.%20Steenwyk&amp;author=Y.%20Chang&amp;author=Y.%20Wang&amp;author=T.Y.%20James&amp;volume=31&amp;publication_year=2021&amp;pages=1653-1665.e5&amp;pmid=33607033&amp;doi=10.1016/j.cub.2021.01.074&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B52-microorganisms-11-01830">
<span class="label">52.</span><cite>Strassert J.F.H., Monaghan M.T. Phylogenomic Insights into the Early Diversification of Fungi. Curr. Biol. 2022;32:3628–3635.e3. doi: 10.1016/j.cub.2022.06.057.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cub.2022.06.057" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35830854/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=Phylogenomic%20Insights%20into%20the%20Early%20Diversification%20of%20Fungi&amp;author=J.F.H.%20Strassert&amp;author=M.T.%20Monaghan&amp;volume=32&amp;publication_year=2022&amp;pages=3628-3635.e3&amp;pmid=35830854&amp;doi=10.1016/j.cub.2022.06.057&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B53-microorganisms-11-01830">
<span class="label">53.</span><cite>Galindo L.J., López-García P., Torruella G., Karpov S., Moreira D. Phylogenomics of a New Fungal Phylum Reveals Multiple Waves of Reductive Evolution across Holomycota. Nat. Commun. 2021;12:4973. doi: 10.1038/s41467-021-25308-w.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41467-021-25308-w" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8371127/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34404788/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Phylogenomics%20of%20a%20New%20Fungal%20Phylum%20Reveals%20Multiple%20Waves%20of%20Reductive%20Evolution%20across%20Holomycota&amp;author=L.J.%20Galindo&amp;author=P.%20L%C3%B3pez-Garc%C3%ADa&amp;author=G.%20Torruella&amp;author=S.%20Karpov&amp;author=D.%20Moreira&amp;volume=12&amp;publication_year=2021&amp;pages=4973&amp;pmid=34404788&amp;doi=10.1038/s41467-021-25308-w&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B54-microorganisms-11-01830">
<span class="label">54.</span><cite>Naranjo-Ortiz M.A., Gabaldón T. Fungal Evolution: Diversity, Taxonomy and Phylogeny of the Fungi. Biol. Rev. Camb. Philos. Soc. 2019;94:2101–2137. doi: 10.1111/brv.12550.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/brv.12550" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6899921/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31659870/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biol.%20Rev.%20Camb.%20Philos.%20Soc.&amp;title=Fungal%20Evolution:%20Diversity,%20Taxonomy%20and%20Phylogeny%20of%20the%20Fungi&amp;author=M.A.%20Naranjo-Ortiz&amp;author=T.%20Gabald%C3%B3n&amp;volume=94&amp;publication_year=2019&amp;pages=2101-2137&amp;pmid=31659870&amp;doi=10.1111/brv.12550&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B55-microorganisms-11-01830">
<span class="label">55.</span><cite>Campbell M.A., Ganley A.R.D., Gabaldón T., Cox M.P. The Case of the Missing Ancient Fungal Polyploids. Am. Nat. 2016;188:602–614. doi: 10.1086/688763.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1086/688763" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27860510/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am.%20Nat.&amp;title=The%20Case%20of%20the%20Missing%20Ancient%20Fungal%20Polyploids&amp;author=M.A.%20Campbell&amp;author=A.R.D.%20Ganley&amp;author=T.%20Gabald%C3%B3n&amp;author=M.P.%20Cox&amp;volume=188&amp;publication_year=2016&amp;pages=602-614&amp;pmid=27860510&amp;doi=10.1086/688763&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B56-microorganisms-11-01830">
<span class="label">56.</span><cite>Nielsen K.N., Salgado J.F.M., Natsopoulou M.E., Kristensen T., Stajich J.E., De Fine Licht H.H. Diploidy within a Haploid Genus of Entomopathogenic Fungi. Genome Biol. Evol. 2021;13:evab158. doi: 10.1093/gbe/evab158.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/gbe/evab158" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8325562/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34247231/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&amp;title=Diploidy%20within%20a%20Haploid%20Genus%20of%20Entomopathogenic%20Fungi&amp;author=K.N.%20Nielsen&amp;author=J.F.M.%20Salgado&amp;author=M.E.%20Natsopoulou&amp;author=T.%20Kristensen&amp;author=J.E.%20Stajich&amp;volume=13&amp;publication_year=2021&amp;pages=evab158&amp;pmid=34247231&amp;doi=10.1093/gbe/evab158&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B57-microorganisms-11-01830">
<span class="label">57.</span><cite>Amses K.R., Simmons D.R., Longcore J.E., Mondo S.J., Seto K., Jerônimo G.H., Bonds A.E., Quandt C.A., Davis W.J., Chang Y., et al.  Diploid-Dominant Life Cycles Characterize the Early Evolution of Fungi. Proc. Natl. Acad. Sci. USA. 2022;119:e2116841119. doi: 10.1073/pnas.2116841119.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.2116841119" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9457484/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36037379/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Diploid-Dominant%20Life%20Cycles%20Characterize%20the%20Early%20Evolution%20of%20Fungi&amp;author=K.R.%20Amses&amp;author=D.R.%20Simmons&amp;author=J.E.%20Longcore&amp;author=S.J.%20Mondo&amp;author=K.%20Seto&amp;volume=119&amp;publication_year=2022&amp;pages=e2116841119&amp;pmid=36037379&amp;doi=10.1073/pnas.2116841119&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B58-microorganisms-11-01830">
<span class="label">58.</span><cite>Genre A., Lanfranco L., Perotto S., Bonfante P. Unique and Common Traits in Mycorrhizal Symbioses. Nat. Rev. Microbiol. 2020;18:649–660. doi: 10.1038/s41579-020-0402-3.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41579-020-0402-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32694620/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&amp;title=Unique%20and%20Common%20Traits%20in%20Mycorrhizal%20Symbioses&amp;author=A.%20Genre&amp;author=L.%20Lanfranco&amp;author=S.%20Perotto&amp;author=P.%20Bonfante&amp;volume=18&amp;publication_year=2020&amp;pages=649-660&amp;pmid=32694620&amp;doi=10.1038/s41579-020-0402-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B59-microorganisms-11-01830">
<span class="label">59.</span><cite>Hoysted G.A., Jacob A.S., Kowal J., Giesemann P., Bidartondo M.I., Duckett J.G., Gebauer G., Rimington W.R., Schornack S., Pressel S., et al.  Mucoromycotina Fine Root Endophyte Fungi Form Nutritional Mutualisms with Vascular Plants. Plant Physiol. 2019;181:565–577. doi: 10.1104/pp.19.00729.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.19.00729" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6776871/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31358684/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Mucoromycotina%20Fine%20Root%20Endophyte%20Fungi%20Form%20Nutritional%20Mutualisms%20with%20Vascular%20Plants&amp;author=G.A.%20Hoysted&amp;author=A.S.%20Jacob&amp;author=J.%20Kowal&amp;author=P.%20Giesemann&amp;author=M.I.%20Bidartondo&amp;volume=181&amp;publication_year=2019&amp;pages=565-577&amp;pmid=31358684&amp;doi=10.1104/pp.19.00729&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B60-microorganisms-11-01830">
<span class="label">60.</span><cite>Wani Z.A., Kumar A., Sultan P., Bindu K., Riyaz-Ul-Hassan S., Ashraf N. Mortierella alpina CS10E4, an Oleaginous Fungal Endophyte of Crocus sativus L. Enhances Apocarotenoid Biosynthesis and Stress Tolerance in the Host Plant. Sci. Rep. 2017;7:8598. doi: 10.1038/s41598-017-08974-z.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-017-08974-z" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5561177/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28819197/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Mortierella%20alpina%20CS10E4,%20an%20Oleaginous%20Fungal%20Endophyte%20of%20Crocus%20sativus%20L.%20Enhances%20Apocarotenoid%20Biosynthesis%20and%20Stress%20Tolerance%20in%20the%20Host%20Plant&amp;author=Z.A.%20Wani&amp;author=A.%20Kumar&amp;author=P.%20Sultan&amp;author=K.%20Bindu&amp;author=S.%20Riyaz-Ul-Hassan&amp;volume=7&amp;publication_year=2017&amp;pages=8598&amp;pmid=28819197&amp;doi=10.1038/s41598-017-08974-z&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B61-microorganisms-11-01830">
<span class="label">61.</span><cite>Walsh E., Luo J., Khiste S., Scalera A., Sajjad S., Zhang N. Pygmaeomycetaceae, a New Root-Associated Family in Mucoromycotina from the Pygmy Pine Plains. Mycologia. 2021;113:134–145. doi: 10.1080/00275514.2020.1803649.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/00275514.2020.1803649" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33085937/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=Pygmaeomycetaceae,%20a%20New%20Root-Associated%20Family%20in%20Mucoromycotina%20from%20the%20Pygmy%20Pine%20Plains&amp;author=E.%20Walsh&amp;author=J.%20Luo&amp;author=S.%20Khiste&amp;author=A.%20Scalera&amp;author=S.%20Sajjad&amp;volume=113&amp;publication_year=2021&amp;pages=134-145&amp;pmid=33085937&amp;doi=10.1080/00275514.2020.1803649&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B62-microorganisms-11-01830">
<span class="label">62.</span><cite>Liu S., Liu M., Liao Q.-G., Lü F.-B., Zhao X.-L. Effects of Inoculated Mycorrhizal Fungi and Non-Mycorrhizal Beneficial Micro-Organisms on Plant Traits, Nutrient Uptake and Root-Associated Fungal Community Composition of the Cymbidium hybridum in Greenhouse. J. Appl. Microbiol. 2021;131:413–424. doi: 10.1111/jam.14967.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/jam.14967" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33320986/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Microbiol.&amp;title=Effects%20of%20Inoculated%20Mycorrhizal%20Fungi%20and%20Non-Mycorrhizal%20Beneficial%20Micro-Organisms%20on%20Plant%20Traits,%20Nutrient%20Uptake%20and%20Root-Associated%20Fungal%20Community%20Composition%20of%20the%20Cymbidium%20hybridum%20in%20Greenhouse&amp;author=S.%20Liu&amp;author=M.%20Liu&amp;author=Q.-G.%20Liao&amp;author=F.-B.%20L%C3%BC&amp;author=X.-L.%20Zhao&amp;volume=131&amp;publication_year=2021&amp;pages=413-424&amp;pmid=33320986&amp;doi=10.1111/jam.14967&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B63-microorganisms-11-01830">
<span class="label">63.</span><cite>Steinbrink J.M., Miceli M.H. Clinical Review of Mucormycosis. Infect. Dis. Clin. N. Am. 2021;35:435–452. doi: 10.1016/j.idc.2021.03.009.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.idc.2021.03.009" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10110349/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34016285/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Dis.%20Clin.%20N.%20Am.&amp;title=Clinical%20Review%20of%20Mucormycosis&amp;author=J.M.%20Steinbrink&amp;author=M.H.%20Miceli&amp;volume=35&amp;publication_year=2021&amp;pages=435-452&amp;pmid=34016285&amp;doi=10.1016/j.idc.2021.03.009&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B64-microorganisms-11-01830">
<span class="label">64.</span><cite>Corrochano L.M., Kuo A., Marcet-Houben M., Polaino S., Salamov A., Villalobos-Escobedo J.M., Grimwood J., Álvarez M.I., Avalos J., Bauer D., et al.  Expansion of Signal Transduction Pathways in Fungi by Extensive Genome Duplication. Curr. Biol. 2016;26:1577–1584. doi: 10.1016/j.cub.2016.04.038.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cub.2016.04.038" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5089372/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27238284/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=Expansion%20of%20Signal%20Transduction%20Pathways%20in%20Fungi%20by%20Extensive%20Genome%20Duplication&amp;author=L.M.%20Corrochano&amp;author=A.%20Kuo&amp;author=M.%20Marcet-Houben&amp;author=S.%20Polaino&amp;author=A.%20Salamov&amp;volume=26&amp;publication_year=2016&amp;pages=1577-1584&amp;pmid=27238284&amp;doi=10.1016/j.cub.2016.04.038&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B65-microorganisms-11-01830">
<span class="label">65.</span><cite>Partida-Martinez L.P., Hertweck C. Pathogenic Fungus Harbours Endosymbiotic Bacteria for Toxin Production. Nature. 2005;437:884–888. doi: 10.1038/nature03997.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature03997" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16208371/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Pathogenic%20Fungus%20Harbours%20Endosymbiotic%20Bacteria%20for%20Toxin%20Production&amp;author=L.P.%20Partida-Martinez&amp;author=C.%20Hertweck&amp;volume=437&amp;publication_year=2005&amp;pages=884-888&amp;pmid=16208371&amp;doi=10.1038/nature03997&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B66-microorganisms-11-01830">
<span class="label">66.</span><cite>Torres-Cortés G., Ghignone S., Bonfante P., Schüßler A. Mosaic Genome of Endobacteria in Arbuscular Mycorrhizal Fungi: Transkingdom Gene Transfer in an Ancient Mycoplasma-Fungus Association. Proc. Natl. Acad. Sci. USA. 2015;112:7785–7790. doi: 10.1073/pnas.1501540112.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1501540112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4485150/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25964335/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Mosaic%20Genome%20of%20Endobacteria%20in%20Arbuscular%20Mycorrhizal%20Fungi:%20Transkingdom%20Gene%20Transfer%20in%20an%20Ancient%20Mycoplasma-Fungus%20Association&amp;author=G.%20Torres-Cort%C3%A9s&amp;author=S.%20Ghignone&amp;author=P.%20Bonfante&amp;author=A.%20Sch%C3%BC%C3%9Fler&amp;volume=112&amp;publication_year=2015&amp;pages=7785-7790&amp;pmid=25964335&amp;doi=10.1073/pnas.1501540112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B67-microorganisms-11-01830">
<span class="label">67.</span><cite>Mondo S.J., Salvioli A., Bonfante P., Morton J.B., Pawlowska T.E. Nondegenerative Evolution in Ancient Heritable Bacterial Endosymbionts of Fungi. Mol. Biol. Evol. 2016;33:2216–2231. doi: 10.1093/molbev/msw086.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/msw086" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27189571/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=Nondegenerative%20Evolution%20in%20Ancient%20Heritable%20Bacterial%20Endosymbionts%20of%20Fungi&amp;author=S.J.%20Mondo&amp;author=A.%20Salvioli&amp;author=P.%20Bonfante&amp;author=J.B.%20Morton&amp;author=T.E.%20Pawlowska&amp;volume=33&amp;publication_year=2016&amp;pages=2216-2231&amp;pmid=27189571&amp;doi=10.1093/molbev/msw086&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B68-microorganisms-11-01830">
<span class="label">68.</span><cite>Bonfante P., Desirò A. Who Lives in a Fungus? The Diversity, Origins and Functions of Fungal Endobacteria Living in Mucoromycota. ISME J. 2017;11:1727–1735. doi: 10.1038/ismej.2017.21.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ismej.2017.21" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5520026/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28387771/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=Who%20Lives%20in%20a%20Fungus?%20The%20Diversity,%20Origins%20and%20Functions%20of%20Fungal%20Endobacteria%20Living%20in%20Mucoromycota&amp;author=P.%20Bonfante&amp;author=A.%20Desir%C3%B2&amp;volume=11&amp;publication_year=2017&amp;pages=1727-1735&amp;pmid=28387771&amp;doi=10.1038/ismej.2017.21&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B69-microorganisms-11-01830">
<span class="label">69.</span><cite>Uehling J., Gryganskyi A., Hameed K., Tschaplinski T., Misztal P.K., Wu S., Desirò A., Vande Pol N., Du Z., Zienkiewicz A., et al.  Comparative Genomics of Mortierella elongata and Its Bacterial Endosymbiont Mycoavidus cysteinexigens. Environ. Microbiol. 2017;19:2964–2983. doi: 10.1111/1462-2920.13669.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/1462-2920.13669" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28076891/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&amp;title=Comparative%20Genomics%20of%20Mortierella%20elongata%20and%20Its%20Bacterial%20Endosymbiont%20Mycoavidus%20cysteinexigens&amp;author=J.%20Uehling&amp;author=A.%20Gryganskyi&amp;author=K.%20Hameed&amp;author=T.%20Tschaplinski&amp;author=P.K.%20Misztal&amp;volume=19&amp;publication_year=2017&amp;pages=2964-2983&amp;pmid=28076891&amp;doi=10.1111/1462-2920.13669&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B70-microorganisms-11-01830">
<span class="label">70.</span><cite>Lastovetsky O.A., Ahn E., Mondo S.J., Toomer K.H., Zhang A., Johnson L.M., Pawlowska T.E. Distribution and Population Structure of Endobacteria in Arbuscular Mycorrhizal Fungi at North Atlantic Dunes. ISME J. 2018;12:3001–3013. doi: 10.1038/s41396-018-0246-2.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41396-018-0246-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6246606/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30097664/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=Distribution%20and%20Population%20Structure%20of%20Endobacteria%20in%20Arbuscular%20Mycorrhizal%20Fungi%20at%20North%20Atlantic%20Dunes&amp;author=O.A.%20Lastovetsky&amp;author=E.%20Ahn&amp;author=S.J.%20Mondo&amp;author=K.H.%20Toomer&amp;author=A.%20Zhang&amp;volume=12&amp;publication_year=2018&amp;pages=3001-3013&amp;pmid=30097664&amp;doi=10.1038/s41396-018-0246-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B71-microorganisms-11-01830">
<span class="label">71.</span><cite>Espino-Vázquez A.N., Bermúdez-Barrientos J.R., Cabrera-Rangel J.F., Córdova-López G., Cardoso-Martínez F., Martínez-Vázquez A., Camarena-Pozos D.A., Mondo S.J., Pawlowska T.E., Abreu-Goodger C., et al.  Narnaviruses: Novel Players in Fungal-Bacterial Symbioses. ISME J. 2020;14:1743–1754. doi: 10.1038/s41396-020-0638-y.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41396-020-0638-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7305303/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32269378/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=Narnaviruses:%20Novel%20Players%20in%20Fungal-Bacterial%20Symbioses&amp;author=A.N.%20Espino-V%C3%A1zquez&amp;author=J.R.%20Berm%C3%BAdez-Barrientos&amp;author=J.F.%20Cabrera-Rangel&amp;author=G.%20C%C3%B3rdova-L%C3%B3pez&amp;author=F.%20Cardoso-Mart%C3%ADnez&amp;volume=14&amp;publication_year=2020&amp;pages=1743-1754&amp;pmid=32269378&amp;doi=10.1038/s41396-020-0638-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B72-microorganisms-11-01830">
<span class="label">72.</span><cite>Bever J.D., Kang H.-J., Kaonongbua W., Wang M.  Genomic Organization and Mechanisms of Inheritance in Arbuscular Mycorrhizal Fungi: Contrasting the Evidence and Implications of Current Theories. In: Varma A., editor. Mycorrhiza. Springer; Berlin/Heidelberg, Germany: 2008. pp. 135–148.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Mycorrhiza&amp;author=J.D.%20Bever&amp;author=H.-J.%20Kang&amp;author=W.%20Kaonongbua&amp;author=M.%20Wang&amp;publication_year=2008&amp;isbn=978-3-540-78824-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B73-microorganisms-11-01830">
<span class="label">73.</span><cite>Martin F., Gianinazzi-Pearson V., Hijri M., Lammers P., Requena N., Sanders I.R., Shachar-Hill Y., Shapiro H., Tuskan G.A., Young J.P.W. The Long Hard Road to a Completed Glomus intraradices Genome. New Phytol. 2008;180:747–750. doi: 10.1111/j.1469-8137.2008.02671.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1469-8137.2008.02671.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19138232/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=The%20Long%20Hard%20Road%20to%20a%20Completed%20Glomus%20intraradices%20Genome&amp;author=F.%20Martin&amp;author=V.%20Gianinazzi-Pearson&amp;author=M.%20Hijri&amp;author=P.%20Lammers&amp;author=N.%20Requena&amp;volume=180&amp;publication_year=2008&amp;pages=747-750&amp;pmid=19138232&amp;doi=10.1111/j.1469-8137.2008.02671.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B74-microorganisms-11-01830">
<span class="label">74.</span><cite>Lebreton A., Corre E., Jany J.-L., Brillet-Guéguen L., Pèrez-Arques C., Garre V., Monsoor M., Debuchy R., Le Meur C., Coton E., et al.  Comparative Genomics Applied to Mucor Species with Different Lifestyles. BMC Genom. 2020;21:135.  doi: 10.1186/s12864-019-6256-2.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12864-019-6256-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7011435/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32039703/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&amp;title=Comparative%20Genomics%20Applied%20to%20Mucor%20Species%20with%20Different%20Lifestyles&amp;author=A.%20Lebreton&amp;author=E.%20Corre&amp;author=J.-L.%20Jany&amp;author=L.%20Brillet-Gu%C3%A9guen&amp;author=C.%20P%C3%A8rez-Arques&amp;volume=21&amp;publication_year=2020&amp;pages=135&amp;pmid=32039703&amp;doi=10.1186/s12864-019-6256-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B75-microorganisms-11-01830">
<span class="label">75.</span><cite>Vellanki S., Navarro-Mendoza M.I., Garcia A.E., Murcia L., Perez-Arques C., Garre V., Nicolas F.E., Lee S.C. Mucor circinelloides: Growth, Maintenance and Genetic Manipulation. Curr. Protoc. Microbiol. 2018;49:e53. doi: 10.1002/cpmc.53.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/cpmc.53" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6060640/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30040216/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Microbiol.&amp;title=Mucor%20circinelloides:%20Growth,%20Maintenance%20and%20Genetic%20Manipulation&amp;author=S.%20Vellanki&amp;author=M.I.%20Navarro-Mendoza&amp;author=A.E.%20Garcia&amp;author=L.%20Murcia&amp;author=C.%20Perez-Arques&amp;volume=49&amp;publication_year=2018&amp;pages=e53&amp;pmid=30040216&amp;doi=10.1002/cpmc.53&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B76-microorganisms-11-01830">
<span class="label">76.</span><cite>Boon E., Zimmerman E., Lang B.F., Hijri M. Intra-Isolate Genome Variation in Arbuscular Mycorrhizal Fungi Persists in the Transcriptome. J. Evol. Biol. 2010;23:1519–1527. doi: 10.1111/j.1420-9101.2010.02019.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1420-9101.2010.02019.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20492090/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Evol.%20Biol.&amp;title=Intra-Isolate%20Genome%20Variation%20in%20Arbuscular%20Mycorrhizal%20Fungi%20Persists%20in%20the%20Transcriptome&amp;author=E.%20Boon&amp;author=E.%20Zimmerman&amp;author=B.F.%20Lang&amp;author=M.%20Hijri&amp;volume=23&amp;publication_year=2010&amp;pages=1519-1527&amp;pmid=20492090&amp;doi=10.1111/j.1420-9101.2010.02019.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B77-microorganisms-11-01830">
<span class="label">77.</span><cite>Muszewska A., Okrasińska A., Steczkiewicz K., Drgas O., Orłowska M., Perlińska-Lenart U., Aleksandrzak-Piekarczyk T., Szatraj K., Zielenkiewicz U., Piłsyk S., et al.  Metabolic Potential, Ecology and Presence of Associated Bacteria Is Reflected in Genomic Diversity of Mucoromycotina. Front. Microbiol. 2021;12:636986. doi: 10.3389/fmicb.2021.636986.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2021.636986" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7928374/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33679672/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Metabolic%20Potential,%20Ecology%20and%20Presence%20of%20Associated%20Bacteria%20Is%20Reflected%20in%20Genomic%20Diversity%20of%20Mucoromycotina&amp;author=A.%20Muszewska&amp;author=A.%20Okrasi%C5%84ska&amp;author=K.%20Steczkiewicz&amp;author=O.%20Drgas&amp;author=M.%20Or%C5%82owska&amp;volume=12&amp;publication_year=2021&amp;pages=636986&amp;pmid=33679672&amp;doi=10.3389/fmicb.2021.636986&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B78-microorganisms-11-01830">
<span class="label">78.</span><cite>Muszewska A., Steczkiewicz K., Ginalski K. DIRS and Ngaro Retrotransposons in Fungi. PLoS ONE. 2013;8:e76319.  doi: 10.1371/journal.pone.0076319.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0076319" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3783388/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24086727/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=DIRS%20and%20Ngaro%20Retrotransposons%20in%20Fungi&amp;author=A.%20Muszewska&amp;author=K.%20Steczkiewicz&amp;author=K.%20Ginalski&amp;volume=8&amp;publication_year=2013&amp;pages=e76319&amp;pmid=24086727&amp;doi=10.1371/journal.pone.0076319&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B79-microorganisms-11-01830">
<span class="label">79.</span><cite>Muszewska A., Steczkiewicz K., Stepniewska-Dziubinska M., Ginalski K. Cut-and-Paste Transposons in Fungi with Diverse Lifestyles. Genome Biol. Evol. 2017;9:3463–3477. doi: 10.1093/gbe/evx261.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/gbe/evx261" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5751038/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29228286/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&amp;title=Cut-and-Paste%20Transposons%20in%20Fungi%20with%20Diverse%20Lifestyles&amp;author=A.%20Muszewska&amp;author=K.%20Steczkiewicz&amp;author=M.%20Stepniewska-Dziubinska&amp;author=K.%20Ginalski&amp;volume=9&amp;publication_year=2017&amp;pages=3463-3477&amp;pmid=29228286&amp;doi=10.1093/gbe/evx261&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B80-microorganisms-11-01830">
<span class="label">80.</span><cite>Mondo S.J., Dannebaum R.O., Kuo R.C., Louie K.B., Bewick A.J., LaButti K., Haridas S., Kuo A., Salamov A., Ahrendt S.R., et al.  Widespread Adenine N6-Methylation of Active Genes in Fungi. Nat. Genet. 2017;49:964–968. doi: 10.1038/ng.3859.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ng.3859" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28481340/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&amp;title=Widespread%20Adenine%20N6-Methylation%20of%20Active%20Genes%20in%20Fungi&amp;author=S.J.%20Mondo&amp;author=R.O.%20Dannebaum&amp;author=R.C.%20Kuo&amp;author=K.B.%20Louie&amp;author=A.J.%20Bewick&amp;volume=49&amp;publication_year=2017&amp;pages=964-968&amp;pmid=28481340&amp;doi=10.1038/ng.3859&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B81-microorganisms-11-01830">
<span class="label">81.</span><cite>Corradi N., Sanders I.R. Evolution of the P-Type II ATPase Gene Family in the Fungi and Presence of Structural Genomic Changes among Isolates of Glomus intraradices. BMC Evol. Biol. 2006;6:21.  doi: 10.1186/1471-2148-6-21.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2148-6-21" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1479386/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16529655/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evol.%20Biol.&amp;title=Evolution%20of%20the%20P-Type%20II%20ATPase%20Gene%20Family%20in%20the%20Fungi%20and%20Presence%20of%20Structural%20Genomic%20Changes%20among%20Isolates%20of%20Glomus%20intraradices&amp;author=N.%20Corradi&amp;author=I.R.%20Sanders&amp;volume=6&amp;publication_year=2006&amp;pages=21&amp;pmid=16529655&amp;doi=10.1186/1471-2148-6-21&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B82-microorganisms-11-01830">
<span class="label">82.</span><cite>Halary S., Malik S.-B., Lildhar L., Slamovits C.H., Hijri M., Corradi N. Conserved Meiotic Machinery in Glomus spp., a Putatively Ancient Asexual Fungal Lineage. Genome Biol. Evol. 2011;3:950–958. doi: 10.1093/gbe/evr089.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/gbe/evr089" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3184777/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21876220/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&amp;title=Conserved%20Meiotic%20Machinery%20in%20Glomus%20spp.,%20a%20Putatively%20Ancient%20Asexual%20Fungal%20Lineage&amp;author=S.%20Halary&amp;author=S.-B.%20Malik&amp;author=L.%20Lildhar&amp;author=C.H.%20Slamovits&amp;author=M.%20Hijri&amp;volume=3&amp;publication_year=2011&amp;pages=950-958&amp;pmid=21876220&amp;doi=10.1093/gbe/evr089&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B83-microorganisms-11-01830">
<span class="label">83.</span><cite>Halary S., Daubois L., Terrat Y., Ellenberger S., Wöstemeyer J., Hijri M. Mating Type Gene Homologues and Putative Sex Pheromone-Sensing Pathway in Arbuscular Mycorrhizal Fungi, a Presumably Asexual Plant Root Symbiont. PLoS ONE. 2013;8:e80729.  doi: 10.1371/journal.pone.0080729.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0080729" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3834313/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24260466/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Mating%20Type%20Gene%20Homologues%20and%20Putative%20Sex%20Pheromone-Sensing%20Pathway%20in%20Arbuscular%20Mycorrhizal%20Fungi,%20a%20Presumably%20Asexual%20Plant%20Root%20Symbiont&amp;author=S.%20Halary&amp;author=L.%20Daubois&amp;author=Y.%20Terrat&amp;author=S.%20Ellenberger&amp;author=J.%20W%C3%B6stemeyer&amp;volume=8&amp;publication_year=2013&amp;pages=e80729&amp;pmid=24260466&amp;doi=10.1371/journal.pone.0080729&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B84-microorganisms-11-01830">
<span class="label">84.</span><cite>Gryganskyi A.P., Golan J., Dolatabadi S., Mondo S., Robb S., Idnurm A., Muszewska A., Steczkiewicz K., Masonjones S., Liao H.-L., et al.  Phylogenetic and Phylogenomic Definition of Rhizopus Species. G3 (Bethesda) 2018;8:2007–2018. doi: 10.1534/g3.118.200235.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1534/g3.118.200235" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5982828/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29674435/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=G3%20(Bethesda)&amp;title=Phylogenetic%20and%20Phylogenomic%20Definition%20of%20Rhizopus%20Species&amp;author=A.P.%20Gryganskyi&amp;author=J.%20Golan&amp;author=S.%20Dolatabadi&amp;author=S.%20Mondo&amp;author=S.%20Robb&amp;volume=8&amp;publication_year=2018&amp;pages=2007-2018&amp;pmid=29674435&amp;doi=10.1534/g3.118.200235&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B85-microorganisms-11-01830">
<span class="label">85.</span><cite>Gryganskyi A.P., Nie Y., Hajek A.E., Hodge K.T., Liu X.-Y., Aadland K., Voigt K., Anishchenko I.M., Kutovenko V.B., Kava L., et al.  The Early Terrestrial Fungal Lineage of Conidiobolus-Transition from Saprotroph to Parasitic Lifestyle. J. Fungi. 2022;8:789.  doi: 10.3390/jof8080789.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/jof8080789" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9409958/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36012777/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Fungi&amp;title=The%20Early%20Terrestrial%20Fungal%20Lineage%20of%20Conidiobolus-Transition%20from%20Saprotroph%20to%20Parasitic%20Lifestyle&amp;author=A.P.%20Gryganskyi&amp;author=Y.%20Nie&amp;author=A.E.%20Hajek&amp;author=K.T.%20Hodge&amp;author=X.-Y.%20Liu&amp;volume=8&amp;publication_year=2022&amp;pages=789&amp;pmid=36012777&amp;doi=10.3390/jof8080789&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B86-microorganisms-11-01830">
<span class="label">86.</span><cite>Tisserant E., Malbreil M., Kuo A., Kohler A., Symeonidi A., Balestrini R., Charron P., Duensing N., Frei dit Frey N., Gianinazzi-Pearson V., et al.  Genome of an Arbuscular Mycorrhizal Fungus Provides Insight into the Oldest Plant Symbiosis. Proc. Natl. Acad. Sci. USA. 2013;110:20117–20122. doi: 10.1073/pnas.1313452110.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1313452110" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3864322/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24277808/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Genome%20of%20an%20Arbuscular%20Mycorrhizal%20Fungus%20Provides%20Insight%20into%20the%20Oldest%20Plant%20Symbiosis&amp;author=E.%20Tisserant&amp;author=M.%20Malbreil&amp;author=A.%20Kuo&amp;author=A.%20Kohler&amp;author=A.%20Symeonidi&amp;volume=110&amp;publication_year=2013&amp;pages=20117-20122&amp;pmid=24277808&amp;doi=10.1073/pnas.1313452110&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B87-microorganisms-11-01830">
<span class="label">87.</span><cite>Wang Y., Chang Y., Ortañez J., Peña J.F., Carter-House D., Reynolds N.K., Smith M.E., Benny G., Mondo S.J., Salamov A., et al.  Divergent Evolution of Early Terrestrial Fungi Reveals the Evolution of Mucormycosis Pathogenicity Factors. Genome Biol. Evol. 2023;15:evad046. doi: 10.1093/gbe/evad046.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/gbe/evad046" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10079182/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36930540/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&amp;title=Divergent%20Evolution%20of%20Early%20Terrestrial%20Fungi%20Reveals%20the%20Evolution%20of%20Mucormycosis%20Pathogenicity%20Factors&amp;author=Y.%20Wang&amp;author=Y.%20Chang&amp;author=J.%20Orta%C3%B1ez&amp;author=J.F.%20Pe%C3%B1a&amp;author=D.%20Carter-House&amp;volume=15&amp;publication_year=2023&amp;pages=evad046&amp;pmid=36930540&amp;doi=10.1093/gbe/evad046&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B88-microorganisms-11-01830">
<span class="label">88.</span><cite>Desirò A., Rimington W.R., Jacob A., Pol N.V., Smith M.E., Trappe J.M., Bidartondo M.I., Bonito G. Multigene Phylogeny of Endogonales, an Early Diverging Lineage of Fungi Associated with Plants. IMA Fungus. 2017;8:245–257. doi: 10.5598/imafungus.2017.08.02.03.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.5598/imafungus.2017.08.02.03" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5729711/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29242774/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=IMA%20Fungus&amp;title=Multigene%20Phylogeny%20of%20Endogonales,%20an%20Early%20Diverging%20Lineage%20of%20Fungi%20Associated%20with%20Plants&amp;author=A.%20Desir%C3%B2&amp;author=W.R.%20Rimington&amp;author=A.%20Jacob&amp;author=N.V.%20Pol&amp;author=M.E.%20Smith&amp;volume=8&amp;publication_year=2017&amp;pages=245-257&amp;pmid=29242774&amp;doi=10.5598/imafungus.2017.08.02.03&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B89-microorganisms-11-01830">
<span class="label">89.</span><cite>Ogura-Tsujita Y., Yamamoto K., Hirayama Y., Ebihara A., Morita N., Imaichi R. Fern Gametophytes of Angiopteris lygodiifolia and Osmunda japonica Harbor Diverse Mucoromycotina Fungi. J. Plant Res. 2019;132:581–588. doi: 10.1007/s10265-019-01121-x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s10265-019-01121-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31292767/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Plant%20Res.&amp;title=Fern%20Gametophytes%20of%20Angiopteris%20lygodiifolia%20and%20Osmunda%20japonica%20Harbor%20Diverse%20Mucoromycotina%20Fungi&amp;author=Y.%20Ogura-Tsujita&amp;author=K.%20Yamamoto&amp;author=Y.%20Hirayama&amp;author=A.%20Ebihara&amp;author=N.%20Morita&amp;volume=132&amp;publication_year=2019&amp;pages=581-588&amp;pmid=31292767&amp;doi=10.1007/s10265-019-01121-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B90-microorganisms-11-01830">
<span class="label">90.</span><cite>Rimington W.R., Pressel S., Duckett J.G., Field K.J., Bidartondo M.I. Evolution and Networks in Ancient and Widespread Symbioses between Mucoromycotina and Liverworts. Mycorrhiza. 2019;29:551–565. doi: 10.1007/s00572-019-00918-x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00572-019-00918-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6890582/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31720838/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycorrhiza&amp;title=Evolution%20and%20Networks%20in%20Ancient%20and%20Widespread%20Symbioses%20between%20Mucoromycotina%20and%20Liverworts&amp;author=W.R.%20Rimington&amp;author=S.%20Pressel&amp;author=J.G.%20Duckett&amp;author=K.J.%20Field&amp;author=M.I.%20Bidartondo&amp;volume=29&amp;publication_year=2019&amp;pages=551-565&amp;pmid=31720838&amp;doi=10.1007/s00572-019-00918-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B91-microorganisms-11-01830">
<span class="label">91.</span><cite>Chaudhary S., Polaino S., Shakya V.P.S., Idnurm A. A New Genetic Linkage Map of the Zygomycete Fungus Phycomyces blakesleeanus. PLoS ONE. 2013;8:e58931.  doi: 10.1371/journal.pone.0058931.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0058931" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3597544/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23516579/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=A%20New%20Genetic%20Linkage%20Map%20of%20the%20Zygomycete%20Fungus%20Phycomyces%20blakesleeanus&amp;author=S.%20Chaudhary&amp;author=S.%20Polaino&amp;author=V.P.S.%20Shakya&amp;author=A.%20Idnurm&amp;volume=8&amp;publication_year=2013&amp;pages=e58931&amp;pmid=23516579&amp;doi=10.1371/journal.pone.0058931&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B92-microorganisms-11-01830">
<span class="label">92.</span><cite>Linde J., Schwartze V., Binder U., Lass-Flörl C., Voigt K., Horn F. De Novo Whole-Genome Sequence and Genome Annotation of Lichtheimia ramosa. Genome Announc. 2014;2:e00888-14. doi: 10.1128/genomeA.00888-14.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/genomeA.00888-14" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4161746/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25212617/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Announc.&amp;title=De%20Novo%20Whole-Genome%20Sequence%20and%20Genome%20Annotation%20of%20Lichtheimia%20ramosa&amp;author=J.%20Linde&amp;author=V.%20Schwartze&amp;author=U.%20Binder&amp;author=C.%20Lass-Fl%C3%B6rl&amp;author=K.%20Voigt&amp;volume=2&amp;publication_year=2014&amp;pages=e00888-14&amp;pmid=25212617&amp;doi=10.1128/genomeA.00888-14&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B93-microorganisms-11-01830">
<span class="label">93.</span><cite>Schwartze V.U., Winter S., Shelest E., Marcet-Houben M., Horn F., Wehner S., Linde J., Valiante V., Sammeth M., Riege K., et al.  Gene Expansion Shapes Genome Architecture in the Human Pathogen Lichtheimia corymbifera: An Evolutionary Genomics Analysis in the Ancient Terrestrial Mucorales (Mucoromycotina) PLoS Genet. 2014;10:e1004496.  doi: 10.1371/journal.pgen.1004496.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pgen.1004496" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4133162/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25121733/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Gene%20Expansion%20Shapes%20Genome%20Architecture%20in%20the%20Human%20Pathogen%20Lichtheimia%20corymbifera:%20An%20Evolutionary%20Genomics%20Analysis%20in%20the%20Ancient%20Terrestrial%20Mucorales%20(Mucoromycotina)&amp;author=V.U.%20Schwartze&amp;author=S.%20Winter&amp;author=E.%20Shelest&amp;author=M.%20Marcet-Houben&amp;author=F.%20Horn&amp;volume=10&amp;publication_year=2014&amp;pages=e1004496&amp;pmid=25121733&amp;doi=10.1371/journal.pgen.1004496&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B94-microorganisms-11-01830">
<span class="label">94.</span><cite>Torres-Cruz T.J., Billingsley Tobias T.L., Almatruk M., Hesse C.N., Kuske C.R., Desirò A., Benucci G.M.N., Bonito G., Stajich J.E., Dunlap C., et al.  Bifiguratus adelaidae, Gen. et Sp. Nov., a New Member of Mucoromycotina in Endophytic and Soil-Dwelling Habitats. Mycologia. 2017;109:363–378. doi: 10.1080/00275514.2017.1364958.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/00275514.2017.1364958" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28876195/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=Bifiguratus%20adelaidae,%20Gen.%20et%20Sp.%20Nov.,%20a%20New%20Member%20of%20Mucoromycotina%20in%20Endophytic%20and%20Soil-Dwelling%20Habitats&amp;author=T.J.%20Torres-Cruz&amp;author=T.L.%20Billingsley%20Tobias&amp;author=M.%20Almatruk&amp;author=C.N.%20Hesse&amp;author=C.R.%20Kuske&amp;volume=109&amp;publication_year=2017&amp;pages=363-378&amp;pmid=28876195&amp;doi=10.1080/00275514.2017.1364958&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B95-microorganisms-11-01830">
<span class="label">95.</span><cite>Hirose D., Degawa Y., Inaba S., Tokumasu S. The Anamorphic Genus Calcarisporiella Is a New Member of the Mucoromycotina. Mycoscience. 2012;53:256–260. doi: 10.1007/S10267-011-0160-1.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/S10267-011-0160-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycoscience&amp;title=The%20Anamorphic%20Genus%20Calcarisporiella%20Is%20a%20New%20Member%20of%20the%20Mucoromycotina&amp;author=D.%20Hirose&amp;author=Y.%20Degawa&amp;author=S.%20Inaba&amp;author=S.%20Tokumasu&amp;volume=53&amp;publication_year=2012&amp;pages=256-260&amp;doi=10.1007/S10267-011-0160-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B96-microorganisms-11-01830">
<span class="label">96.</span><cite>Nagy L.G., Petkovits T., Kovács G.M., Voigt K., Vágvölgyi C., Papp T. Where Is the Unseen Fungal Diversity Hidden? A Study of Mortierella Reveals a Large Contribution of Reference Collections to the Identification of Fungal Environmental Sequences. New Phytol. 2011;191:789–794. doi: 10.1111/j.1469-8137.2011.03707.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1469-8137.2011.03707.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21453289/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Where%20Is%20the%20Unseen%20Fungal%20Diversity%20Hidden?%20A%20Study%20of%20Mortierella%20Reveals%20a%20Large%20Contribution%20of%20Reference%20Collections%20to%20the%20Identification%20of%20Fungal%20Environmental%20Sequences&amp;author=L.G.%20Nagy&amp;author=T.%20Petkovits&amp;author=G.M.%20Kov%C3%A1cs&amp;author=K.%20Voigt&amp;author=C.%20V%C3%A1gv%C3%B6lgyi&amp;volume=191&amp;publication_year=2011&amp;pages=789-794&amp;pmid=21453289&amp;doi=10.1111/j.1469-8137.2011.03707.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B97-microorganisms-11-01830">
<span class="label">97.</span><cite>Smith M.E., Gryganskyi A., Bonito G., Nouhra E., Moreno-Arroyo B., Benny G. Phylogenetic Analysis of the Genus Modicella Reveals an Independent Evolutionary Origin of Sporocarp-Forming Fungi in the Mortierellales. Fungal Genet. Biol. 2013;61:61–68. doi: 10.1016/j.fgb.2013.10.001.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.fgb.2013.10.001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24120560/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&amp;title=Phylogenetic%20Analysis%20of%20the%20Genus%20Modicella%20Reveals%20an%20Independent%20Evolutionary%20Origin%20of%20Sporocarp-Forming%20Fungi%20in%20the%20Mortierellales&amp;author=M.E.%20Smith&amp;author=A.%20Gryganskyi&amp;author=G.%20Bonito&amp;author=E.%20Nouhra&amp;author=B.%20Moreno-Arroyo&amp;volume=61&amp;publication_year=2013&amp;pages=61-68&amp;pmid=24120560&amp;doi=10.1016/j.fgb.2013.10.001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B98-microorganisms-11-01830">
<span class="label">98.</span><cite>Macias A.M., Geiser D.M., Stajich J.E., Łukasik P., Veloso C., Bublitz D.C., Berger M.C., Boyce G.R., Hodge K., Kasson M.T. Evolutionary Relationships among Massospora Spp. (Entomophthorales), Obligate Pathogens of Cicadas. Mycologia. 2020;112:1060–1074. doi: 10.1080/00275514.2020.1742033.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/00275514.2020.1742033" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32412847/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=Evolutionary%20Relationships%20among%20Massospora%20Spp.%20(Entomophthorales),%20Obligate%20Pathogens%20of%20Cicadas&amp;author=A.M.%20Macias&amp;author=D.M.%20Geiser&amp;author=J.E.%20Stajich&amp;author=P.%20%C5%81ukasik&amp;author=C.%20Veloso&amp;volume=112&amp;publication_year=2020&amp;pages=1060-1074&amp;pmid=32412847&amp;doi=10.1080/00275514.2020.1742033&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B99-microorganisms-11-01830">
<span class="label">99.</span><cite>Bonito G., Hameed K., Ventura R., Krishnan J., Schadt C.W., Vilgalys R. Isolating a Functionally Relevant Guild of Fungi from the Root Microbiome of Populus. Fungal Ecol. 2016;22:35–42. doi: 10.1016/j.funeco.2016.04.007.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.funeco.2016.04.007" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Ecol.&amp;title=Isolating%20a%20Functionally%20Relevant%20Guild%20of%20Fungi%20from%20the%20Root%20Microbiome%20of%20Populus&amp;author=G.%20Bonito&amp;author=K.%20Hameed&amp;author=R.%20Ventura&amp;author=J.%20Krishnan&amp;author=C.W.%20Schadt&amp;volume=22&amp;publication_year=2016&amp;pages=35-42&amp;doi=10.1016/j.funeco.2016.04.007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B100-microorganisms-11-01830">
<span class="label">100.</span><cite>Vandepol N., Liber J., Yocca A., Matlock J., Edger P., Bonito G. Linnemannia elongata (Mortierellaceae) Stimulates Arabidopsis thaliana Aerial Growth and Responses to Auxin, Ethylene, and Reactive Oxygen Species. PLoS ONE. 2022;17:e0261908.  doi: 10.1371/journal.pone.0261908.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0261908" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9004744/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35413060/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Linnemannia%20elongata%20(Mortierellaceae)%20Stimulates%20Arabidopsis%20thaliana%20Aerial%20Growth%20and%20Responses%20to%20Auxin,%20Ethylene,%20and%20Reactive%20Oxygen%20Species&amp;author=N.%20Vandepol&amp;author=J.%20Liber&amp;author=A.%20Yocca&amp;author=J.%20Matlock&amp;author=P.%20Edger&amp;volume=17&amp;publication_year=2022&amp;pages=e0261908&amp;pmid=35413060&amp;doi=10.1371/journal.pone.0261908&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B101-microorganisms-11-01830">
<span class="label">101.</span><cite>Zhang K., Bonito G., Hsu C.-M., Hameed K., Vilgalys R., Liao H.-L. Mortierella elongata Increases Plant Biomass among Non-Leguminous Crop Species. Agronomy. 2020;10:754.  doi: 10.3390/agronomy10050754.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/agronomy10050754" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Mortierella%20elongata%20Increases%20Plant%20Biomass%20among%20Non-Leguminous%20Crop%20Species&amp;author=K.%20Zhang&amp;author=G.%20Bonito&amp;author=C.-M.%20Hsu&amp;author=K.%20Hameed&amp;author=R.%20Vilgalys&amp;volume=10&amp;publication_year=2020&amp;pages=754&amp;doi=10.3390/agronomy10050754&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B102-microorganisms-11-01830">
<span class="label">102.</span><cite>Wurlitzer J.M., Stanišić A., Wasmuth I., Jungmann S., Fischer D., Kries H., Gressler M. Bacterial-like Nonribosomal Peptide Synthetases Produce Cyclopeptides in the Zygomycetous Fungus Mortierella alpina. Appl. Environ. Microbiol. 2021;87:e02051-20. doi: 10.1128/AEM.02051-20.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.02051-20" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7848919/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33158886/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&amp;title=Bacterial-like%20Nonribosomal%20Peptide%20Synthetases%20Produce%20Cyclopeptides%20in%20the%20Zygomycetous%20Fungus%20Mortierella%20alpina&amp;author=J.M.%20Wurlitzer&amp;author=A.%20Stani%C5%A1i%C4%87&amp;author=I.%20Wasmuth&amp;author=S.%20Jungmann&amp;author=D.%20Fischer&amp;volume=87&amp;publication_year=2021&amp;pages=e02051-20&amp;pmid=33158886&amp;doi=10.1128/AEM.02051-20&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B103-microorganisms-11-01830">
<span class="label">103.</span><cite>Remy W., Taylor T.N., Hass H., Kerp H. Four Hundred-Million-Year-Old Vesicular Arbuscular Mycorrhizae. Proc. Natl. Acad. Sci. USA. 1994;91:11841–11843. doi: 10.1073/pnas.91.25.11841.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.91.25.11841" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC45331/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11607500/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Four%20Hundred-Million-Year-Old%20Vesicular%20Arbuscular%20Mycorrhizae&amp;author=W.%20Remy&amp;author=T.N.%20Taylor&amp;author=H.%20Hass&amp;author=H.%20Kerp&amp;volume=91&amp;publication_year=1994&amp;pages=11841-11843&amp;pmid=11607500&amp;doi=10.1073/pnas.91.25.11841&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B104-microorganisms-11-01830">
<span class="label">104.</span><cite>Smith S., Read D.  Mycorrhizal Symbiosis. 3rd ed. Academic Press; Cambridge, MA, USA: 2008. </cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Mycorrhizal%20Symbiosis&amp;author=S.%20Smith&amp;author=D.%20Read&amp;publication_year=2008&amp;isbn=978-0-12-370526-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B105-microorganisms-11-01830">
<span class="label">105.</span><cite>Hijri M., Sanders I.R. The Arbuscular Mycorrhizal Fungus Glomus intraradices Is Haploid and Has a Small Genome Size in the Lower Limit of Eukaryotes. Fungal Genet. Biol. 2004;41:253–261. doi: 10.1016/j.fgb.2003.10.011.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.fgb.2003.10.011" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14732270/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&amp;title=The%20Arbuscular%20Mycorrhizal%20Fungus%20Glomus%20intraradices%20Is%20Haploid%20and%20Has%20a%20Small%20Genome%20Size%20in%20the%20Lower%20Limit%20of%20Eukaryotes&amp;author=M.%20Hijri&amp;author=I.R.%20Sanders&amp;volume=41&amp;publication_year=2004&amp;pages=253-261&amp;pmid=14732270&amp;doi=10.1016/j.fgb.2003.10.011&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B106-microorganisms-11-01830">
<span class="label">106.</span><cite>Sędzielewska K.A., Fuchs J., Temsch E.M., Baronian K., Watzke R., Kunze G. Estimation of the Glomus intraradices Nuclear DNA Content. New Phytol. 2011;192:794–797. doi: 10.1111/j.1469-8137.2011.03937.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1469-8137.2011.03937.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21988748/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Estimation%20of%20the%20Glomus%20intraradices%20Nuclear%20DNA%20Content&amp;author=K.A.%20S%C4%99dzielewska&amp;author=J.%20Fuchs&amp;author=E.M.%20Temsch&amp;author=K.%20Baronian&amp;author=R.%20Watzke&amp;volume=192&amp;publication_year=2011&amp;pages=794-797&amp;pmid=21988748&amp;doi=10.1111/j.1469-8137.2011.03937.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B107-microorganisms-11-01830">
<span class="label">107.</span><cite>Venice F., Ghignone S., Salvioli di Fossalunga A., Amselem J., Novero M., Xianan X., Sędzielewska Toro K., Morin E., Lipzen A., Grigoriev I.V., et al.  At the Nexus of Three Kingdoms: The Genome of the Mycorrhizal Fungus Gigaspora margarita Provides Insights into Plant, Endobacterial and Fungal Interactions. Environ. Microbiol. 2020;22:122–141. doi: 10.1111/1462-2920.14827.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/1462-2920.14827" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31621176/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&amp;title=At%20the%20Nexus%20of%20Three%20Kingdoms:%20The%20Genome%20of%20the%20Mycorrhizal%20Fungus%20Gigaspora%20margarita%20Provides%20Insights%20into%20Plant,%20Endobacterial%20and%20Fungal%20Interactions&amp;author=F.%20Venice&amp;author=S.%20Ghignone&amp;author=A.%20Salvioli%20di%20Fossalunga&amp;author=J.%20Amselem&amp;author=M.%20Novero&amp;volume=22&amp;publication_year=2020&amp;pages=122-141&amp;pmid=31621176&amp;doi=10.1111/1462-2920.14827&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B108-microorganisms-11-01830">
<span class="label">108.</span><cite>Ropars J., Toro K.S., Noel J., Pelin A., Charron P., Farinelli L., Marton T., Krüger M., Fuchs J., Brachmann A., et al.  Evidence for the Sexual Origin of Heterokaryosis in Arbuscular Mycorrhizal Fungi. Nat. Microbiol. 2016;1:16033. doi: 10.1038/nmicrobiol.2016.33.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmicrobiol.2016.33" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27572831/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Microbiol.&amp;title=Evidence%20for%20the%20Sexual%20Origin%20of%20Heterokaryosis%20in%20Arbuscular%20Mycorrhizal%20Fungi&amp;author=J.%20Ropars&amp;author=K.S.%20Toro&amp;author=J.%20Noel&amp;author=A.%20Pelin&amp;author=P.%20Charron&amp;volume=1&amp;publication_year=2016&amp;pages=16033&amp;pmid=27572831&amp;doi=10.1038/nmicrobiol.2016.33&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B109-microorganisms-11-01830">
<span class="label">109.</span><cite>Idnurm A., Walton F.J., Floyd A., Heitman J. Identification of the Sex Genes in an Early Diverged Fungus. Nature. 2008;451:193–196. doi: 10.1038/nature06453.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature06453" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18185588/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Identification%20of%20the%20Sex%20Genes%20in%20an%20Early%20Diverged%20Fungus&amp;author=A.%20Idnurm&amp;author=F.J.%20Walton&amp;author=A.%20Floyd&amp;author=J.%20Heitman&amp;volume=451&amp;publication_year=2008&amp;pages=193-196&amp;pmid=18185588&amp;doi=10.1038/nature06453&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B110-microorganisms-11-01830">
<span class="label">110.</span><cite>Kuhn G., Hijri M., Sanders I.R. Evidence for the Evolution of Multiple Genomes in Arbuscular Mycorrhizal Fungi. Nature. 2001;414:745–748. doi: 10.1038/414745a.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/414745a" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11742398/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Evidence%20for%20the%20Evolution%20of%20Multiple%20Genomes%20in%20Arbuscular%20Mycorrhizal%20Fungi&amp;author=G.%20Kuhn&amp;author=M.%20Hijri&amp;author=I.R.%20Sanders&amp;volume=414&amp;publication_year=2001&amp;pages=745-748&amp;pmid=11742398&amp;doi=10.1038/414745a&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B111-microorganisms-11-01830">
<span class="label">111.</span><cite>Pawlowska T.E., Taylor J.W. Organization of Genetic Variation in Individuals of Arbuscular Mycorrhizal Fungi. Nature. 2004;427:733–737. doi: 10.1038/nature02290.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature02290" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14973485/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Organization%20of%20Genetic%20Variation%20in%20Individuals%20of%20Arbuscular%20Mycorrhizal%20Fungi&amp;author=T.E.%20Pawlowska&amp;author=J.W.%20Taylor&amp;volume=427&amp;publication_year=2004&amp;pages=733-737&amp;pmid=14973485&amp;doi=10.1038/nature02290&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B112-microorganisms-11-01830">
<span class="label">112.</span><cite>Wewer V., Brands M., Dörmann P. Fatty Acid Synthesis and Lipid Metabolism in the Obligate Biotrophic Fungus Rhizophagus irregularis during Mycorrhization of Lotus japonicus. Plant J. 2014;79:398–412. doi: 10.1111/tpj.12566.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.12566" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24888347/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=Fatty%20Acid%20Synthesis%20and%20Lipid%20Metabolism%20in%20the%20Obligate%20Biotrophic%20Fungus%20Rhizophagus%20irregularis%20during%20Mycorrhization%20of%20Lotus%20japonicus&amp;author=V.%20Wewer&amp;author=M.%20Brands&amp;author=P.%20D%C3%B6rmann&amp;volume=79&amp;publication_year=2014&amp;pages=398-412&amp;pmid=24888347&amp;doi=10.1111/tpj.12566&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B113-microorganisms-11-01830">
<span class="label">113.</span><cite>Tang N., San Clemente H., Roy S., Bécard G., Zhao B., Roux C. A Survey of the Gene Repertoire of Gigaspora rosea Unravels Conserved Features among Glomeromycota for Obligate Biotrophy. Front. Microbiol. 2016;7:233. doi: 10.3389/fmicb.2016.00233.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2016.00233" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4771724/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26973612/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=A%20Survey%20of%20the%20Gene%20Repertoire%20of%20Gigaspora%20rosea%20Unravels%20Conserved%20Features%20among%20Glomeromycota%20for%20Obligate%20Biotrophy&amp;author=N.%20Tang&amp;author=H.%20San%20Clemente&amp;author=S.%20Roy&amp;author=G.%20B%C3%A9card&amp;author=B.%20Zhao&amp;volume=7&amp;publication_year=2016&amp;pages=233&amp;pmid=26973612&amp;doi=10.3389/fmicb.2016.00233&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B114-microorganisms-11-01830">
<span class="label">114.</span><cite>Kobayashi Y., Maeda T., Yamaguchi K., Kameoka H., Tanaka S., Ezawa T., Shigenobu S., Kawaguchi M. The Genome of Rhizophagus clarus HR1 Reveals a Common Genetic Basis for Auxotrophy among Arbuscular Mycorrhizal Fungi. BMC Genom. 2018;19:465.  doi: 10.1186/s12864-018-4853-0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12864-018-4853-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6007072/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29914365/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&amp;title=The%20Genome%20of%20Rhizophagus%20clarus%20HR1%20Reveals%20a%20Common%20Genetic%20Basis%20for%20Auxotrophy%20among%20Arbuscular%20Mycorrhizal%20Fungi&amp;author=Y.%20Kobayashi&amp;author=T.%20Maeda&amp;author=K.%20Yamaguchi&amp;author=H.%20Kameoka&amp;author=S.%20Tanaka&amp;volume=19&amp;publication_year=2018&amp;pages=465&amp;pmid=29914365&amp;doi=10.1186/s12864-018-4853-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B115-microorganisms-11-01830">
<span class="label">115.</span><cite>Bravo A., Brands M., Wewer V., Dörmann P., Harrison M.J. Arbuscular Mycorrhiza-Specific Enzymes FatM and RAM2 Fine-Tune Lipid Biosynthesis to Promote Development of Arbuscular Mycorrhiza. New Phytol. 2017;214:1631–1645. doi: 10.1111/nph.14533.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/nph.14533" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28380681/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Arbuscular%20Mycorrhiza-Specific%20Enzymes%20FatM%20and%20RAM2%20Fine-Tune%20Lipid%20Biosynthesis%20to%20Promote%20Development%20of%20Arbuscular%20Mycorrhiza&amp;author=A.%20Bravo&amp;author=M.%20Brands&amp;author=V.%20Wewer&amp;author=P.%20D%C3%B6rmann&amp;author=M.J.%20Harrison&amp;volume=214&amp;publication_year=2017&amp;pages=1631-1645&amp;pmid=28380681&amp;doi=10.1111/nph.14533&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B116-microorganisms-11-01830">
<span class="label">116.</span><cite>Jiang Y., Wang W., Xie Q., Liu N., Liu L., Wang D., Zhang X., Yang C., Chen X., Tang D., et al.  Plants Transfer Lipids to Sustain Colonization by Mutualistic Mycorrhizal and Parasitic Fungi. Science. 2017;356:1172–1175. doi: 10.1126/science.aam9970.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.aam9970" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28596307/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Plants%20Transfer%20Lipids%20to%20Sustain%20Colonization%20by%20Mutualistic%20Mycorrhizal%20and%20Parasitic%20Fungi&amp;author=Y.%20Jiang&amp;author=W.%20Wang&amp;author=Q.%20Xie&amp;author=N.%20Liu&amp;author=L.%20Liu&amp;volume=356&amp;publication_year=2017&amp;pages=1172-1175&amp;pmid=28596307&amp;doi=10.1126/science.aam9970&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B117-microorganisms-11-01830">
<span class="label">117.</span><cite>Keymer A., Pimprikar P., Wewer V., Huber C., Brands M., Bucerius S.L., Delaux P.-M., Klingl V., von Röpenack-Lahaye E., Wang T.L., et al.  Lipid Transfer from Plants to Arbuscular Mycorrhiza Fungi. eLife. 2017;6:e29107. doi: 10.7554/eLife.29107.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.7554/eLife.29107" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5559270/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28726631/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Lipid%20Transfer%20from%20Plants%20to%20Arbuscular%20Mycorrhiza%20Fungi&amp;author=A.%20Keymer&amp;author=P.%20Pimprikar&amp;author=V.%20Wewer&amp;author=C.%20Huber&amp;author=M.%20Brands&amp;volume=6&amp;publication_year=2017&amp;pages=e29107&amp;pmid=28726631&amp;doi=10.7554/eLife.29107&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B118-microorganisms-11-01830">
<span class="label">118.</span><cite>Luginbuehl L.H., Menard G.N., Kurup S., Van Erp H., Radhakrishnan G.V., Breakspear A., Oldroyd G.E.D., Eastmond P.J. Fatty Acids in Arbuscular Mycorrhizal Fungi Are Synthesized by the Host Plant. Science. 2017;356:1175–1178. doi: 10.1126/science.aan0081.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.aan0081" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28596311/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Fatty%20Acids%20in%20Arbuscular%20Mycorrhizal%20Fungi%20Are%20Synthesized%20by%20the%20Host%20Plant&amp;author=L.H.%20Luginbuehl&amp;author=G.N.%20Menard&amp;author=S.%20Kurup&amp;author=H.%20Van%20Erp&amp;author=G.V.%20Radhakrishnan&amp;volume=356&amp;publication_year=2017&amp;pages=1175-1178&amp;pmid=28596311&amp;doi=10.1126/science.aan0081&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B119-microorganisms-11-01830">
<span class="label">119.</span><cite>Kameoka H., Tsutsui I., Saito K., Kikuchi Y., Handa Y., Ezawa T., Hayashi H., Kawaguchi M., Akiyama K. Stimulation of Asymbiotic Sporulation in Arbuscular Mycorrhizal Fungi by Fatty Acids. Nat. Microbiol. 2019;4:1654–1660. doi: 10.1038/s41564-019-0485-7.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41564-019-0485-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31235957/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Microbiol.&amp;title=Stimulation%20of%20Asymbiotic%20Sporulation%20in%20Arbuscular%20Mycorrhizal%20Fungi%20by%20Fatty%20Acids&amp;author=H.%20Kameoka&amp;author=I.%20Tsutsui&amp;author=K.%20Saito&amp;author=Y.%20Kikuchi&amp;author=Y.%20Handa&amp;volume=4&amp;publication_year=2019&amp;pages=1654-1660&amp;pmid=31235957&amp;doi=10.1038/s41564-019-0485-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B120-microorganisms-11-01830">
<span class="label">120.</span><cite>Sugiura Y., Akiyama R., Tanaka S., Yano K., Kameoka H., Marui S., Saito M., Kawaguchi M., Akiyama K., Saito K. Myristate Can Be Used as a Carbon and Energy Source for the Asymbiotic Growth of Arbuscular Mycorrhizal Fungi. Proc. Natl. Acad. Sci. USA. 2020;117:25779–25788. doi: 10.1073/pnas.2006948117.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.2006948117" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7568319/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32999061/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Myristate%20Can%20Be%20Used%20as%20a%20Carbon%20and%20Energy%20Source%20for%20the%20Asymbiotic%20Growth%20of%20Arbuscular%20Mycorrhizal%20Fungi&amp;author=Y.%20Sugiura&amp;author=R.%20Akiyama&amp;author=S.%20Tanaka&amp;author=K.%20Yano&amp;author=H.%20Kameoka&amp;volume=117&amp;publication_year=2020&amp;pages=25779-25788&amp;pmid=32999061&amp;doi=10.1073/pnas.2006948117&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B121-microorganisms-11-01830">
<span class="label">121.</span><cite>Sperschneider J., Yildirir G., Rizzi Y., Malar C.M., Sorwar E., Chen E.C., Iwasaki W., Brauer E.K., Bosnich W., Gutjahr C., et al.  Resolving the Haplotypes of Arbuscular Mycorrhizal Fungi Highlights the Role of Two Nuclear Populations in Host Interactions. bioRxiv. 2023 doi: 10.1101/2023.01.15.524138.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/2023.01.15.524138" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Resolving%20the%20Haplotypes%20of%20Arbuscular%20Mycorrhizal%20Fungi%20Highlights%20the%20Role%20of%20Two%20Nuclear%20Populations%20in%20Host%20Interactions&amp;author=J.%20Sperschneider&amp;author=G.%20Yildirir&amp;author=Y.%20Rizzi&amp;author=C.M.%20Malar&amp;author=E.%20Sorwar&amp;publication_year=2023&amp;doi=10.1101/2023.01.15.524138&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B122-microorganisms-11-01830">
<span class="label">122.</span><cite>Hughes D.P., Araújo J.P.M., Loreto R.G., Quevillon L., de Bekker C., Evans H.C. From So Simple a Beginning: The Evolution of Behavioral Manipulation by Fungi. Adv. Genet. 2016;94:437–469. doi: 10.1016/bs.adgen.2016.01.004.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/bs.adgen.2016.01.004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27131331/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Genet.&amp;title=From%20So%20Simple%20a%20Beginning:%20The%20Evolution%20of%20Behavioral%20Manipulation%20by%20Fungi&amp;author=D.P.%20Hughes&amp;author=J.P.M.%20Ara%C3%BAjo&amp;author=R.G.%20Loreto&amp;author=L.%20Quevillon&amp;author=C.%20de%20Bekker&amp;volume=94&amp;publication_year=2016&amp;pages=437-469&amp;pmid=27131331&amp;doi=10.1016/bs.adgen.2016.01.004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B123-microorganisms-11-01830">
<span class="label">123.</span><cite>Vilela R., Mendoza L. Human Pathogenic Entomophthorales. Clin. Microbiol. Rev. 2018;31:e00014-18. doi: 10.1128/CMR.00014-18.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/CMR.00014-18" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6148186/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30158298/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Microbiol.%20Rev.&amp;title=Human%20Pathogenic%20Entomophthorales&amp;author=R.%20Vilela&amp;author=L.%20Mendoza&amp;volume=31&amp;publication_year=2018&amp;pages=e00014-18&amp;pmid=30158298&amp;doi=10.1128/CMR.00014-18&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B124-microorganisms-11-01830">
<span class="label">124.</span><cite>Lazarus K.L., Benny G.L., Ho H.-M., Smith M.E. Phylogenetic Systematics of Syncephalis (Zoopagales, Zoopagomycotina), a Genus of Ubiquitous Mycoparasites. Mycologia. 2017;109:333–349. doi: 10.1080/00275514.2017.1307005.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/00275514.2017.1307005" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28489971/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=Phylogenetic%20Systematics%20of%20Syncephalis%20(Zoopagales,%20Zoopagomycotina),%20a%20Genus%20of%20Ubiquitous%20Mycoparasites&amp;author=K.L.%20Lazarus&amp;author=G.L.%20Benny&amp;author=H.-M.%20Ho&amp;author=M.E.%20Smith&amp;volume=109&amp;publication_year=2017&amp;pages=333-349&amp;pmid=28489971&amp;doi=10.1080/00275514.2017.1307005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B125-microorganisms-11-01830">
<span class="label">125.</span><cite>Lovett B., Macias A., Stajich J.E., Cooley J., Eilenberg J., de Fine Licht H.H., Kasson M.T. Behavioral Betrayal: How Select Fungal Parasites Enlist Living Insects to Do Their Bidding. PLoS Pathog. 2020;16:e1008598.  doi: 10.1371/journal.ppat.1008598.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.ppat.1008598" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7302443/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32555716/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=Behavioral%20Betrayal:%20How%20Select%20Fungal%20Parasites%20Enlist%20Living%20Insects%20to%20Do%20Their%20Bidding&amp;author=B.%20Lovett&amp;author=A.%20Macias&amp;author=J.E.%20Stajich&amp;author=J.%20Cooley&amp;author=J.%20Eilenberg&amp;volume=16&amp;publication_year=2020&amp;pages=e1008598&amp;pmid=32555716&amp;doi=10.1371/journal.ppat.1008598&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B126-microorganisms-11-01830">
<span class="label">126.</span><cite>Chang Y., Wang S., Sekimoto S., Aerts A.L., Choi C., Clum A., LaButti K.M., Lindquist E.A., Yee Ngan C., Ohm R.A., et al.  Phylogenomic Analyses Indicate That Early Fungi Evolved Digesting Cell Walls of Algal Ancestors of Land Plants. Genome Biol. Evol. 2015;7:1590–1601. doi: 10.1093/gbe/evv090.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/gbe/evv090" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4494064/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25977457/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&amp;title=Phylogenomic%20Analyses%20Indicate%20That%20Early%20Fungi%20Evolved%20Digesting%20Cell%20Walls%20of%20Algal%20Ancestors%20of%20Land%20Plants&amp;author=Y.%20Chang&amp;author=S.%20Wang&amp;author=S.%20Sekimoto&amp;author=A.L.%20Aerts&amp;author=C.%20Choi&amp;volume=7&amp;publication_year=2015&amp;pages=1590-1601&amp;pmid=25977457&amp;doi=10.1093/gbe/evv090&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B127-microorganisms-11-01830">
<span class="label">127.</span><cite>Wang Y., White M.M., Kvist S., Moncalvo J.-M. Genome-Wide Survey of Gut Fungi (Harpellales) Reveals the First Horizontally Transferred Ubiquitin Gene from a Mosquito Host. Mol. Biol. Evol. 2016;33:2544–2554. doi: 10.1093/molbev/msw126.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/msw126" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5026252/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27343289/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=Genome-Wide%20Survey%20of%20Gut%20Fungi%20(Harpellales)%20Reveals%20the%20First%20Horizontally%20Transferred%20Ubiquitin%20Gene%20from%20a%20Mosquito%20Host&amp;author=Y.%20Wang&amp;author=M.M.%20White&amp;author=S.%20Kvist&amp;author=J.-M.%20Moncalvo&amp;volume=33&amp;publication_year=2016&amp;pages=2544-2554&amp;pmid=27343289&amp;doi=10.1093/molbev/msw126&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B128-microorganisms-11-01830">
<span class="label">128.</span><cite>Wang Y., Stata M., Wang W., Stajich J.E., White M.M., Moncalvo J.-M. Comparative Genomics Reveals the Core Gene Toolbox for the Fungus-Insect Symbiosis. mBio. 2018;9:e00636-18. doi: 10.1128/mBio.00636-18.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/mBio.00636-18" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5954228/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29764946/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Comparative%20Genomics%20Reveals%20the%20Core%20Gene%20Toolbox%20for%20the%20Fungus-Insect%20Symbiosis&amp;author=Y.%20Wang&amp;author=M.%20Stata&amp;author=W.%20Wang&amp;author=J.E.%20Stajich&amp;author=M.M.%20White&amp;volume=9&amp;publication_year=2018&amp;pages=e00636-18&amp;pmid=29764946&amp;doi=10.1128/mBio.00636-18&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B129-microorganisms-11-01830">
<span class="label">129.</span><cite>Tabima J.F., Trautman I.A., Chang Y., Wang Y., Mondo S., Kuo A., Salamov A., Grigoriev I.V., Stajich J.E., Spatafora J.W. Phylogenomic Analyses of Non-Dikarya Fungi Supports Horizontal Gene Transfer Driving Diversification of Secondary Metabolism in the Amphibian Gastrointestinal Symbiont, Basidiobolus. G3 (Bethesda) 2020;10:3417–3433. doi: 10.1534/g3.120.401516.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1534/g3.120.401516" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7466969/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32727924/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=G3%20(Bethesda)&amp;title=Phylogenomic%20Analyses%20of%20Non-Dikarya%20Fungi%20Supports%20Horizontal%20Gene%20Transfer%20Driving%20Diversification%20of%20Secondary%20Metabolism%20in%20the%20Amphibian%20Gastrointestinal%20Symbiont,%20Basidiobolus&amp;author=J.F.%20Tabima&amp;author=I.A.%20Trautman&amp;author=Y.%20Chang&amp;author=Y.%20Wang&amp;author=S.%20Mondo&amp;volume=10&amp;publication_year=2020&amp;pages=3417-3433&amp;pmid=32727924&amp;doi=10.1534/g3.120.401516&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B130-microorganisms-11-01830">
<span class="label">130.</span><cite>Chuang S.-C., Ho H.-M., Reynolds N., Smith M.E., Benny G.L., Chien C.-Y., Tsai J.-L. Preliminary Phylogeny of Coemansia (Kickxellales), with Descriptions of Four New Species from Taiwan. Mycologia. 2017;109:815–831. doi: 10.1080/00275514.2017.1401892.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/00275514.2017.1401892" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29336682/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=Preliminary%20Phylogeny%20of%20Coemansia%20(Kickxellales),%20with%20Descriptions%20of%20Four%20New%20Species%20from%20Taiwan&amp;author=S.-C.%20Chuang&amp;author=H.-M.%20Ho&amp;author=N.%20Reynolds&amp;author=M.E.%20Smith&amp;author=G.L.%20Benny&amp;volume=109&amp;publication_year=2017&amp;pages=815-831&amp;pmid=29336682&amp;doi=10.1080/00275514.2017.1401892&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B131-microorganisms-11-01830">
<span class="label">131.</span><cite>Wang Y., White M.M., Moncalvo J.-M. Draft Genome Sequence of Capniomyces stellatus, the Obligate Gut Fungal Symbiont of Stonefly. Genome Announc. 2016;4:e00761-16. doi: 10.1128/genomeA.00761-16.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/genomeA.00761-16" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4974312/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27491991/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Announc.&amp;title=Draft%20Genome%20Sequence%20of%20Capniomyces%20stellatus,%20the%20Obligate%20Gut%20Fungal%20Symbiont%20of%20Stonefly&amp;author=Y.%20Wang&amp;author=M.M.%20White&amp;author=J.-M.%20Moncalvo&amp;volume=4&amp;publication_year=2016&amp;pages=e00761-16&amp;pmid=27491991&amp;doi=10.1128/genomeA.00761-16&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B132-microorganisms-11-01830">
<span class="label">132.</span><cite>Tretter E.D., Johnson E.M., Benny G.L., Lichtwardt R.W., Wang Y., Kandel P., Novak S.J., Smith J.F., White M.M. An Eight-Gene Molecular Phylogeny of the Kickxellomycotina, Including the First Phylogenetic Placement of Asellariales. Mycologia. 2014;106:912–935. doi: 10.3852/13-253.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3852/13-253" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24891422/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=An%20Eight-Gene%20Molecular%20Phylogeny%20of%20the%20Kickxellomycotina,%20Including%20the%20First%20Phylogenetic%20Placement%20of%20Asellariales&amp;author=E.D.%20Tretter&amp;author=E.M.%20Johnson&amp;author=G.L.%20Benny&amp;author=R.W.%20Lichtwardt&amp;author=Y.%20Wang&amp;volume=106&amp;publication_year=2014&amp;pages=912-935&amp;pmid=24891422&amp;doi=10.3852/13-253&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B133-microorganisms-11-01830">
<span class="label">133.</span><cite>Lichtward R.W.  The Trichomycetes, Fungal Associates of Arthropods. 2nd ed. Springer Science &amp; Business Media; New York, NY, USA: Berlin/Heidelberg, Germany: Tokio, Japan: 2012. </cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=The%20Trichomycetes,%20Fungal%20Associates%20of%20Arthropods&amp;author=R.W.%20Lichtward&amp;publication_year=2012&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B134-microorganisms-11-01830">
<span class="label">134.</span><cite>Wang Y., White M.M., Moncalvo J.-M. Diversification of the Gut Fungi Smittium and Allies (Harpellales) Co-Occurred with the Origin of Complete Metamorphosis of Their Symbiotic Insect Hosts (Lower Diptera) Mol. Phylogenet. Evol. 2019;139:106550. doi: 10.1016/j.ympev.2019.106550.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ympev.2019.106550" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31279967/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Phylogenet.%20Evol.&amp;title=Diversification%20of%20the%20Gut%20Fungi%20Smittium%20and%20Allies%20(Harpellales)%20Co-Occurred%20with%20the%20Origin%20of%20Complete%20Metamorphosis%20of%20Their%20Symbiotic%20Insect%20Hosts%20(Lower%20Diptera)&amp;author=Y.%20Wang&amp;author=M.M.%20White&amp;author=J.-M.%20Moncalvo&amp;volume=139&amp;publication_year=2019&amp;pages=106550&amp;pmid=31279967&amp;doi=10.1016/j.ympev.2019.106550&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B135-microorganisms-11-01830">
<span class="label">135.</span><cite>White M.M., Guàrdia Valle L., Lichtwardt R.W., Siri A., Strongman D.B., William R.T., Gause W.J., Tretter E.D. New Species and Emendations of Orphella: Taxonomic and Phylogenetic Reassessment of the Genus to Establish the Orphellales, for Stonefly Gut Fungi with a Twist. Mycologia. 2018;110:147–178. doi: 10.1080/00275514.2018.1448198.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/00275514.2018.1448198" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29863986/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=New%20Species%20and%20Emendations%20of%20Orphella:%20Taxonomic%20and%20Phylogenetic%20Reassessment%20of%20the%20Genus%20to%20Establish%20the%20Orphellales,%20for%20Stonefly%20Gut%20Fungi%20with%20a%20Twist&amp;author=M.M.%20White&amp;author=L.%20Gu%C3%A0rdia%20Valle&amp;author=R.W.%20Lichtwardt&amp;author=A.%20Siri&amp;author=D.B.%20Strongman&amp;volume=110&amp;publication_year=2018&amp;pages=147-178&amp;pmid=29863986&amp;doi=10.1080/00275514.2018.1448198&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B136-microorganisms-11-01830">
<span class="label">136.</span><cite>Wang Y., Tretter E.D., Lichtwardt R.W., White M.M. Overview of 75 Years of Smittium Research, Establishing a New Genus for Smittium culisetae, and Prospects for Future Revisions of the “Smittium” Clade. Mycologia. 2013;105:90–111. doi: 10.3852/11-311.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3852/11-311" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22802392/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=Overview%20of%2075%20Years%20of%20Smittium%20Research,%20Establishing%20a%20New%20Genus%20for%20Smittium%20culisetae,%20and%20Prospects%20for%20Future%20Revisions%20of%20the%20%E2%80%9CSmittium%E2%80%9D%20Clade&amp;author=Y.%20Wang&amp;author=E.D.%20Tretter&amp;author=R.W.%20Lichtwardt&amp;author=M.M.%20White&amp;volume=105&amp;publication_year=2013&amp;pages=90-111&amp;pmid=22802392&amp;doi=10.3852/11-311&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B137-microorganisms-11-01830">
<span class="label">137.</span><cite>White M.M., Siri A., Lichtwardt R.W. Trichomycete Insect Symbionts in Great Smoky Mountains National Park and Vicinity. Mycologia. 2006;98:333–352. doi: 10.1080/15572536.2006.11832705.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/15572536.2006.11832705" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16894978/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=Trichomycete%20Insect%20Symbionts%20in%20Great%20Smoky%20Mountains%20National%20Park%20and%20Vicinity&amp;author=M.M.%20White&amp;author=A.%20Siri&amp;author=R.W.%20Lichtwardt&amp;volume=98&amp;publication_year=2006&amp;pages=333-352&amp;pmid=16894978&amp;doi=10.1080/15572536.2006.11832705&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B138-microorganisms-11-01830">
<span class="label">138.</span><cite>Mariotti M., Salinas G., Gabaldón T., Gladyshev V.N. Utilization of Selenocysteine in Early-Branching Fungal Phyla. Nat. Microbiol. 2019;4:759–765. doi: 10.1038/s41564-018-0354-9.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41564-018-0354-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6551623/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30742068/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Microbiol.&amp;title=Utilization%20of%20Selenocysteine%20in%20Early-Branching%20Fungal%20Phyla&amp;author=M.%20Mariotti&amp;author=G.%20Salinas&amp;author=T.%20Gabald%C3%B3n&amp;author=V.N.%20Gladyshev&amp;volume=4&amp;publication_year=2019&amp;pages=759-765&amp;pmid=30742068&amp;doi=10.1038/s41564-018-0354-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B139-microorganisms-11-01830">
<span class="label">139.</span><cite>Nie Y., Yu D.-S., Wang C.-F., Liu X.-Y., Huang B. A Taxonomic Revision of the Genus Conidiobolus (Ancylistaceae, Entomophthorales): Four Clades Including Three New Genera. MycoKeys. 2020;66:55–81. doi: 10.3897/mycokeys.66.46575.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3897/mycokeys.66.46575" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7136305/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32273794/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=MycoKeys&amp;title=A%20Taxonomic%20Revision%20of%20the%20Genus%20Conidiobolus%20(Ancylistaceae,%20Entomophthorales):%20Four%20Clades%20Including%20Three%20New%20Genera&amp;author=Y.%20Nie&amp;author=D.-S.%20Yu&amp;author=C.-F.%20Wang&amp;author=X.-Y.%20Liu&amp;author=B.%20Huang&amp;volume=66&amp;publication_year=2020&amp;pages=55-81&amp;pmid=32273794&amp;doi=10.3897/mycokeys.66.46575&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B140-microorganisms-11-01830">
<span class="label">140.</span><cite>Humber R.A. Synopsis of a Revised Classification for the Entomophthorales (Zygomycotina) Mycotaxon. 1989;34:441–460.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycotaxon&amp;title=Synopsis%20of%20a%20Revised%20Classification%20for%20the%20Entomophthorales%20(Zygomycotina)&amp;author=R.A.%20Humber&amp;volume=34&amp;publication_year=1989&amp;pages=441-460&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B141-microorganisms-11-01830">
<span class="label">141.</span><cite>Nagahama T., Sato H., Shimazu M., Sugiyama J. Phylogenetic Divergence of the Entomophthoralean Fungi: Evidence from Nuclear 18S Ribosomal RNA Gene Sequences. Mycologia. 1995;87:203–209. doi: 10.1080/00275514.1995.12026522.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/00275514.1995.12026522" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycologia&amp;title=Phylogenetic%20Divergence%20of%20the%20Entomophthoralean%20Fungi:%20Evidence%20from%20Nuclear%2018S%20Ribosomal%20RNA%20Gene%20Sequences&amp;author=T.%20Nagahama&amp;author=H.%20Sato&amp;author=M.%20Shimazu&amp;author=J.%20Sugiyama&amp;volume=87&amp;publication_year=1995&amp;pages=203-209&amp;doi=10.1080/00275514.1995.12026522&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B142-microorganisms-11-01830">
<span class="label">142.</span><cite>Bergman K., Eslava A.P., Cerdá-Olmedo E. Mutants of Phycomyces with Abnormal Phototropism. Mol. Gen. Genet. 1973;123:1–16. doi: 10.1007/BF00282984.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/BF00282984" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/4726375/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Gen.%20Genet.&amp;title=Mutants%20of%20Phycomyces%20with%20Abnormal%20Phototropism&amp;author=K.%20Bergman&amp;author=A.P.%20Eslava&amp;author=E.%20Cerd%C3%A1-Olmedo&amp;volume=123&amp;publication_year=1973&amp;pages=1-16&amp;pmid=4726375&amp;doi=10.1007/BF00282984&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B143-microorganisms-11-01830">
<span class="label">143.</span><cite>Idnurm A., Rodríguez-Romero J., Corrochano L.M., Sanz C., Iturriaga E.A., Eslava A.P., Heitman J. The Phycomyces MadA Gene Encodes a Blue-Light Photoreceptor for Phototropism and Other Light Responses. Proc. Natl. Acad. Sci. USA. 2006;103:4546–4551. doi: 10.1073/pnas.0600633103.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0600633103" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1450208/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16537433/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=The%20Phycomyces%20MadA%20Gene%20Encodes%20a%20Blue-Light%20Photoreceptor%20for%20Phototropism%20and%20Other%20Light%20Responses&amp;author=A.%20Idnurm&amp;author=J.%20Rodr%C3%ADguez-Romero&amp;author=L.M.%20Corrochano&amp;author=C.%20Sanz&amp;author=E.A.%20Iturriaga&amp;volume=103&amp;publication_year=2006&amp;pages=4546-4551&amp;pmid=16537433&amp;doi=10.1073/pnas.0600633103&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B144-microorganisms-11-01830">
<span class="label">144.</span><cite>Liu Y., He Q., Cheng P. Photoreception in Neurospora: A Tale of Two White Collar Proteins. Cell. Mol. Life Sci. 2003;60:2131–2138. doi: 10.1007/s00018-003-3109-5.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00018-003-3109-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC11146069/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14618260/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell.%20Mol.%20Life%20Sci.&amp;title=Photoreception%20in%20Neurospora:%20A%20Tale%20of%20Two%20White%20Collar%20Proteins&amp;author=Y.%20Liu&amp;author=Q.%20He&amp;author=P.%20Cheng&amp;volume=60&amp;publication_year=2003&amp;pages=2131-2138&amp;pmid=14618260&amp;doi=10.1007/s00018-003-3109-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B145-microorganisms-11-01830">
<span class="label">145.</span><cite>Sanz C., Rodríguez-Romero J., Idnurm A., Christie J.M., Heitman J., Corrochano L.M., Eslava A.P. Phycomyces MadB Interacts with MadA to Form the Primary Photoreceptor Complex for Fungal Phototropism. Proc. Natl. Acad. Sci. USA. 2009;106:7095–7100. doi: 10.1073/pnas.0900879106.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0900879106" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2678449/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19380729/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Phycomyces%20MadB%20Interacts%20with%20MadA%20to%20Form%20the%20Primary%20Photoreceptor%20Complex%20for%20Fungal%20Phototropism&amp;author=C.%20Sanz&amp;author=J.%20Rodr%C3%ADguez-Romero&amp;author=A.%20Idnurm&amp;author=J.M.%20Christie&amp;author=J.%20Heitman&amp;volume=106&amp;publication_year=2009&amp;pages=7095-7100&amp;pmid=19380729&amp;doi=10.1073/pnas.0900879106&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B146-microorganisms-11-01830">
<span class="label">146.</span><cite>Silva F., Torres-Martínez S., Garre V. Distinct White Collar-1 Genes Control Specific Light Responses in Mucor circinelloides. Mol. Microbiol. 2006;61:1023–1037. doi: 10.1111/j.1365-2958.2006.05291.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-2958.2006.05291.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16879651/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&amp;title=Distinct%20White%20Collar-1%20Genes%20Control%20Specific%20Light%20Responses%20in%20Mucor%20circinelloides&amp;author=F.%20Silva&amp;author=S.%20Torres-Mart%C3%ADnez&amp;author=V.%20Garre&amp;volume=61&amp;publication_year=2006&amp;pages=1023-1037&amp;pmid=16879651&amp;doi=10.1111/j.1365-2958.2006.05291.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B147-microorganisms-11-01830">
<span class="label">147.</span><cite>Polaino S., Villalobos-Escobedo J.M., Shakya V.P.S., Miralles-Durán A., Chaudhary S., Sanz C., Shahriari M., Luque E.M., Eslava A.P., Corrochano L.M., et al.  A Ras GTPase Associated Protein Is Involved in the Phototropic and Circadian Photobiology Responses in Fungi. Sci. Rep. 2017;7:44790. doi: 10.1038/srep44790.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/srep44790" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5359613/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28322269/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=A%20Ras%20GTPase%20Associated%20Protein%20Is%20Involved%20in%20the%20Phototropic%20and%20Circadian%20Photobiology%20Responses%20in%20Fungi&amp;author=S.%20Polaino&amp;author=J.M.%20Villalobos-Escobedo&amp;author=V.P.S.%20Shakya&amp;author=A.%20Miralles-Dur%C3%A1n&amp;author=S.%20Chaudhary&amp;volume=7&amp;publication_year=2017&amp;pages=44790&amp;pmid=28322269&amp;doi=10.1038/srep44790&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B148-microorganisms-11-01830">
<span class="label">148.</span><cite>Verma S., Idnurm A. The Uve1 Endonuclease Is Regulated by the White Collar Complex to Protect Cryptococcus neoformans from UV Damage. PLoS Genet. 2013;9:e1003769.  doi: 10.1371/journal.pgen.1003769.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pgen.1003769" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3764193/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24039606/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=The%20Uve1%20Endonuclease%20Is%20Regulated%20by%20the%20White%20Collar%20Complex%20to%20Protect%20Cryptococcus%20neoformans%20from%20UV%20Damage&amp;author=S.%20Verma&amp;author=A.%20Idnurm&amp;volume=9&amp;publication_year=2013&amp;pages=e1003769&amp;pmid=24039606&amp;doi=10.1371/journal.pgen.1003769&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B149-microorganisms-11-01830">
<span class="label">149.</span><cite>Schimek C., Eibel P., Horie T., Galland P., Ootaki T. Protein Crystals in Phycomyces Sporangiophores Are Involved in Graviperception. Adv. Space Res. 1999;24:687–696. doi: 10.1016/S0273-1177(99)00400-7.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0273-1177(99)00400-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11542610/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res.&amp;title=Protein%20Crystals%20in%20Phycomyces%20Sporangiophores%20Are%20Involved%20in%20Graviperception&amp;author=C.%20Schimek&amp;author=P.%20Eibel&amp;author=T.%20Horie&amp;author=P.%20Galland&amp;author=T.%20Ootaki&amp;volume=24&amp;publication_year=1999&amp;pages=687-696&amp;pmid=11542610&amp;doi=10.1016/S0273-1177(99)00400-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B150-microorganisms-11-01830">
<span class="label">150.</span><cite>Nguyen T.A., Greig J., Khan A., Goh C., Jedd G. Evolutionary Novelty in Gravity Sensing through Horizontal Gene Transfer and High-Order Protein Assembly. PLoS Biol. 2018;16:e2004920.  doi: 10.1371/journal.pbio.2004920.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pbio.2004920" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5915273/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29689046/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&amp;title=Evolutionary%20Novelty%20in%20Gravity%20Sensing%20through%20Horizontal%20Gene%20Transfer%20and%20High-Order%20Protein%20Assembly&amp;author=T.A.%20Nguyen&amp;author=J.%20Greig&amp;author=A.%20Khan&amp;author=C.%20Goh&amp;author=G.%20Jedd&amp;volume=16&amp;publication_year=2018&amp;pages=e2004920&amp;pmid=29689046&amp;doi=10.1371/journal.pbio.2004920&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B151-microorganisms-11-01830">
<span class="label">151.</span><cite>Dupont S., Lemetais G., Ferreira T., Cayot P., Gervais P., Beney L. Ergosterol Biosynthesis: A Fungal Pathway for Life on Land? Evolution. 2012;66:2961–2968. doi: 10.1111/j.1558-5646.2012.01667.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1558-5646.2012.01667.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22946816/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Ergosterol%20Biosynthesis:%20A%20Fungal%20Pathway%20for%20Life%20on%20Land?&amp;author=S.%20Dupont&amp;author=G.%20Lemetais&amp;author=T.%20Ferreira&amp;author=P.%20Cayot&amp;author=P.%20Gervais&amp;volume=66&amp;publication_year=2012&amp;pages=2961-2968&amp;pmid=22946816&amp;doi=10.1111/j.1558-5646.2012.01667.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B152-microorganisms-11-01830">
<span class="label">152.</span><cite>Pawlowska T.E., Gaspar M.L., Lastovetsky O.A., Mondo S.J., Real-Ramirez I., Shakya E., Bonfante P. Biology of Fungi and Their Bacterial Endosymbionts. Annu. Rev. Phytopathol. 2018;56:289–309. doi: 10.1146/annurev-phyto-080417-045914.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev-phyto-080417-045914" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30149793/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Phytopathol.&amp;title=Biology%20of%20Fungi%20and%20Their%20Bacterial%20Endosymbionts&amp;author=T.E.%20Pawlowska&amp;author=M.L.%20Gaspar&amp;author=O.A.%20Lastovetsky&amp;author=S.J.%20Mondo&amp;author=I.%20Real-Ramirez&amp;volume=56&amp;publication_year=2018&amp;pages=289-309&amp;pmid=30149793&amp;doi=10.1146/annurev-phyto-080417-045914&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B153-microorganisms-11-01830">
<span class="label">153.</span><cite>Bonfante P., Venice F., Lanfranco L. The Mycobiota: Fungi Take Their Place between Plants and Bacteria. Curr. Opin. Microbiol. 2019;49:18–25. doi: 10.1016/j.mib.2019.08.004.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.mib.2019.08.004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31654910/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Microbiol.&amp;title=The%20Mycobiota:%20Fungi%20Take%20Their%20Place%20between%20Plants%20and%20Bacteria&amp;author=P.%20Bonfante&amp;author=F.%20Venice&amp;author=L.%20Lanfranco&amp;volume=49&amp;publication_year=2019&amp;pages=18-25&amp;pmid=31654910&amp;doi=10.1016/j.mib.2019.08.004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B154-microorganisms-11-01830">
<span class="label">154.</span><cite>Okrasińska A., Bokus A., Duk K., Gęsiorska A., Sokołowska B., Miłobędzka A., Wrzosek M., Pawłowska J. New Endohyphal Relationships between Mucoromycota and Burkholderiaceae Representatives. Appl. Environ. Microbiol. 2021;87:e02707-20. doi: 10.1128/AEM.02707-20.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.02707-20" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8091615/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33483310/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&amp;title=New%20Endohyphal%20Relationships%20between%20Mucoromycota%20and%20Burkholderiaceae%20Representatives&amp;author=A.%20Okrasi%C5%84ska&amp;author=A.%20Bokus&amp;author=K.%20Duk&amp;author=A.%20G%C4%99siorska&amp;author=B.%20Soko%C5%82owska&amp;volume=87&amp;publication_year=2021&amp;pages=e02707-20&amp;pmid=33483310&amp;doi=10.1128/AEM.02707-20&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B155-microorganisms-11-01830">
<span class="label">155.</span><cite>Bianciotto V., Lumini E., Bonfante P., Vandamme P. “Candidatus Glomeribacter Gigasporarum” Gen. Nov., Sp. Nov., an Endosymbiont of Arbuscular Mycorrhizal Fungi. Int. J. Syst. Evol. Microbiol. 2003;53:121–124. doi: 10.1099/ijs.0.02382-0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1099/ijs.0.02382-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12656162/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;title=%E2%80%9CCandidatus%20Glomeribacter%20Gigasporarum%E2%80%9D%20Gen.%20Nov.,%20Sp.%20Nov.,%20an%20Endosymbiont%20of%20Arbuscular%20Mycorrhizal%20Fungi&amp;author=V.%20Bianciotto&amp;author=E.%20Lumini&amp;author=P.%20Bonfante&amp;author=P.%20Vandamme&amp;volume=53&amp;publication_year=2003&amp;pages=121-124&amp;pmid=12656162&amp;doi=10.1099/ijs.0.02382-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B156-microorganisms-11-01830">
<span class="label">156.</span><cite>Lumini E., Bianciotto V., Jargeat P., Novero M., Salvioli A., Faccio A., Bécard G., Bonfante P. Presymbiotic Growth and Sporal Morphology Are Affected in the Arbuscular Mycorrhizal Fungus Gigaspora margarita Cured of Its Endobacteria. Cell. Microbiol. 2007;9:1716–1729. doi: 10.1111/j.1462-5822.2007.00907.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1462-5822.2007.00907.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17331157/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell.%20Microbiol.&amp;title=Presymbiotic%20Growth%20and%20Sporal%20Morphology%20Are%20Affected%20in%20the%20Arbuscular%20Mycorrhizal%20Fungus%20Gigaspora%20margarita%20Cured%20of%20Its%20Endobacteria&amp;author=E.%20Lumini&amp;author=V.%20Bianciotto&amp;author=P.%20Jargeat&amp;author=M.%20Novero&amp;author=A.%20Salvioli&amp;volume=9&amp;publication_year=2007&amp;pages=1716-1729&amp;pmid=17331157&amp;doi=10.1111/j.1462-5822.2007.00907.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B157-microorganisms-11-01830">
<span class="label">157.</span><cite>Salvioli A., Ghignone S., Novero M., Navazio L., Venice F., Bagnaresi P., Bonfante P. Symbiosis with an Endobacterium Increases the Fitness of a Mycorrhizal Fungus, Raising Its Bioenergetic Potential. ISME J. 2016;10:130–144. doi: 10.1038/ismej.2015.91.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ismej.2015.91" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4681866/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26046255/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=Symbiosis%20with%20an%20Endobacterium%20Increases%20the%20Fitness%20of%20a%20Mycorrhizal%20Fungus,%20Raising%20Its%20Bioenergetic%20Potential&amp;author=A.%20Salvioli&amp;author=S.%20Ghignone&amp;author=M.%20Novero&amp;author=L.%20Navazio&amp;author=F.%20Venice&amp;volume=10&amp;publication_year=2016&amp;pages=130-144&amp;pmid=26046255&amp;doi=10.1038/ismej.2015.91&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B158-microorganisms-11-01830">
<span class="label">158.</span><cite>Dearth S.P., Castro H.F., Venice F., Tague E.D., Novero M., Bonfante P., Campagna S.R. Metabolome Changes Are Induced in the Arbuscular Mycorrhizal Fungus Gigaspora margarita by Germination and by Its Bacterial Endosymbiont. Mycorrhiza. 2018;28:421–433. doi: 10.1007/s00572-018-0838-8.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00572-018-0838-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29860608/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycorrhiza&amp;title=Metabolome%20Changes%20Are%20Induced%20in%20the%20Arbuscular%20Mycorrhizal%20Fungus%20Gigaspora%20margarita%20by%20Germination%20and%20by%20Its%20Bacterial%20Endosymbiont&amp;author=S.P.%20Dearth&amp;author=H.F.%20Castro&amp;author=F.%20Venice&amp;author=E.D.%20Tague&amp;author=M.%20Novero&amp;volume=28&amp;publication_year=2018&amp;pages=421-433&amp;pmid=29860608&amp;doi=10.1007/s00572-018-0838-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B159-microorganisms-11-01830">
<span class="label">159.</span><cite>Ohshima S., Sato Y., Fujimura R., Takashima Y., Hamada M., Nishizawa T., Narisawa K., Ohta H. Mycoavidus cysteinexigens Gen. Nov., Sp. Nov., an Endohyphal Bacterium Isolated from a Soil Isolate of the Fungus Mortierella elongata. Int. J. Syst. Evol. Microbiol. 2016;66:2052–2057. doi: 10.1099/ijsem.0.000990.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1099/ijsem.0.000990" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26920389/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;title=Mycoavidus%20cysteinexigens%20Gen.%20Nov.,%20Sp.%20Nov.,%20an%20Endohyphal%20Bacterium%20Isolated%20from%20a%20Soil%20Isolate%20of%20the%20Fungus%20Mortierella%20elongata&amp;author=S.%20Ohshima&amp;author=Y.%20Sato&amp;author=R.%20Fujimura&amp;author=Y.%20Takashima&amp;author=M.%20Hamada&amp;volume=66&amp;publication_year=2016&amp;pages=2052-2057&amp;pmid=26920389&amp;doi=10.1099/ijsem.0.000990&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B160-microorganisms-11-01830">
<span class="label">160.</span><cite>Li Z., Yao Q., Dearth S.P., Entler M.R., Castro Gonzalez H.F., Uehling J.K., Vilgalys R.J., Hurst G.B., Campagna S.R., Labbé J.L., et al.  Integrated Proteomics and Metabolomics Suggests Symbiotic Metabolism and Multimodal Regulation in a Fungal-Endobacterial System. Environ. Microbiol. 2017;19:1041–1053. doi: 10.1111/1462-2920.13605.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/1462-2920.13605" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27871150/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&amp;title=Integrated%20Proteomics%20and%20Metabolomics%20Suggests%20Symbiotic%20Metabolism%20and%20Multimodal%20Regulation%20in%20a%20Fungal-Endobacterial%20System&amp;author=Z.%20Li&amp;author=Q.%20Yao&amp;author=S.P.%20Dearth&amp;author=M.R.%20Entler&amp;author=H.F.%20Castro%20Gonzalez&amp;volume=19&amp;publication_year=2017&amp;pages=1041-1053&amp;pmid=27871150&amp;doi=10.1111/1462-2920.13605&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B161-microorganisms-11-01830">
<span class="label">161.</span><cite>Takashima Y., Seto K., Degawa Y., Guo Y., Nishizawa T., Ohta H., Narisawa K. Prevalence and Intra-Family Phylogenetic Divergence of Burkholderiaceae-Related Endobacteria Associated with Species of Mortierella. Microbes Environ. 2018;33:417–427. doi: 10.1264/jsme2.ME18081.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1264/jsme2.ME18081" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6307997/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30531154/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbes%20Environ.&amp;title=Prevalence%20and%20Intra-Family%20Phylogenetic%20Divergence%20of%20Burkholderiaceae-Related%20Endobacteria%20Associated%20with%20Species%20of%20Mortierella&amp;author=Y.%20Takashima&amp;author=K.%20Seto&amp;author=Y.%20Degawa&amp;author=Y.%20Guo&amp;author=T.%20Nishizawa&amp;volume=33&amp;publication_year=2018&amp;pages=417-427&amp;pmid=30531154&amp;doi=10.1264/jsme2.ME18081&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B162-microorganisms-11-01830">
<span class="label">162.</span><cite>Büttner H., Niehs S.P., Vandelannoote K., Cseresnyés Z., Dose B., Richter I., Gerst R., Figge M.T., Stinear T.P., Pidot S.J., et al.  Bacterial Endosymbionts Protect Beneficial Soil Fungus from Nematode Attack. Proc. Natl. Acad. Sci. USA. 2021;118:e2110669118. doi: 10.1073/pnas.2110669118.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.2110669118" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8449335/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34504005/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Bacterial%20Endosymbionts%20Protect%20Beneficial%20Soil%20Fungus%20from%20Nematode%20Attack&amp;author=H.%20B%C3%BCttner&amp;author=S.P.%20Niehs&amp;author=K.%20Vandelannoote&amp;author=Z.%20Cseresny%C3%A9s&amp;author=B.%20Dose&amp;volume=118&amp;publication_year=2021&amp;pages=e2110669118&amp;pmid=34504005&amp;doi=10.1073/pnas.2110669118&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B163-microorganisms-11-01830">
<span class="label">163.</span><cite>Mondo S.J., Lastovetsky O.A., Gaspar M.L., Schwardt N.H., Barber C.C., Riley R., Sun H., Grigoriev I.V., Pawlowska T.E. Bacterial Endosymbionts Influence Host Sexuality and Reveal Reproductive Genes of Early Divergent Fungi. Nat. Commun. 2017;8:1843. doi: 10.1038/s41467-017-02052-8.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41467-017-02052-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5705715/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29184190/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Bacterial%20Endosymbionts%20Influence%20Host%20Sexuality%20and%20Reveal%20Reproductive%20Genes%20of%20Early%20Divergent%20Fungi&amp;author=S.J.%20Mondo&amp;author=O.A.%20Lastovetsky&amp;author=M.L.%20Gaspar&amp;author=N.H.%20Schwardt&amp;author=C.C.%20Barber&amp;volume=8&amp;publication_year=2017&amp;pages=1843&amp;pmid=29184190&amp;doi=10.1038/s41467-017-02052-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B164-microorganisms-11-01830">
<span class="label">164.</span><cite>Richter I., Radosa S., Cseresnyés Z., Ferling I., Büttner H., Niehs S.P., Gerst R., Scherlach K., Figge M.T., Hillmann F., et al.  Toxin-Producing Endosymbionts Shield Pathogenic Fungus against Micropredators. mBio. 2022;13:e01440-22. doi: 10.1128/mbio.01440-22.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/mbio.01440-22" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9600703/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36005392/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Toxin-Producing%20Endosymbionts%20Shield%20Pathogenic%20Fungus%20against%20Micropredators&amp;author=I.%20Richter&amp;author=S.%20Radosa&amp;author=Z.%20Cseresny%C3%A9s&amp;author=I.%20Ferling&amp;author=H.%20B%C3%BCttner&amp;volume=13&amp;publication_year=2022&amp;pages=e01440-22&amp;pmid=36005392&amp;doi=10.1128/mbio.01440-22&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B165-microorganisms-11-01830">
<span class="label">165.</span><cite>Itabangi H., Sephton-Clark P.C.S., Tamayo D.P., Zhou X., Starling G.P., Mahamoud Z., Insua I., Probert M., Correia J., Moynihan P.J., et al.  A Bacterial Endosymbiont of the Fungus Rhizopus microsporus Drives Phagocyte Evasion and Opportunistic Virulence. Curr. Biol. 2022;32:1115–1130.e6. doi: 10.1016/j.cub.2022.01.028.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cub.2022.01.028" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8926845/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35134329/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=A%20Bacterial%20Endosymbiont%20of%20the%20Fungus%20Rhizopus%20microsporus%20Drives%20Phagocyte%20Evasion%20and%20Opportunistic%20Virulence&amp;author=H.%20Itabangi&amp;author=P.C.S.%20Sephton-Clark&amp;author=D.P.%20Tamayo&amp;author=X.%20Zhou&amp;author=G.P.%20Starling&amp;volume=32&amp;publication_year=2022&amp;pages=1115-1130.e6&amp;pmid=35134329&amp;doi=10.1016/j.cub.2022.01.028&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B166-microorganisms-11-01830">
<span class="label">166.</span><cite>Partida-Martinez L.P., Hertweck C. A Gene Cluster Encoding Rhizoxin Biosynthesis in “Burkholderia rhizoxina”, the Bacterial Endosymbiont of the Fungus Rhizopus microsporus. Chembiochem. 2007;8:41–45. doi: 10.1002/cbic.200600393.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/cbic.200600393" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17154220/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Chembiochem&amp;title=A%20Gene%20Cluster%20Encoding%20Rhizoxin%20Biosynthesis%20in%20%E2%80%9CBurkholderia%20rhizoxina%E2%80%9D,%20the%20Bacterial%20Endosymbiont%20of%20the%20Fungus%20Rhizopus%20microsporus&amp;author=L.P.%20Partida-Martinez&amp;author=C.%20Hertweck&amp;volume=8&amp;publication_year=2007&amp;pages=41-45&amp;pmid=17154220&amp;doi=10.1002/cbic.200600393&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B167-microorganisms-11-01830">
<span class="label">167.</span><cite>Estrada-de Los Santos P., Palmer M., Chávez-Ramírez B., Beukes C., Steenkamp E.T., Briscoe L., Khan N., Maluk M., Lafos M., Humm E., et al.  Whole Genome Analyses Suggests That Burkholderia sensu lato Contains Two Additional Novel Genera (Mycetohabitans gen. nov., and Trinickia gen. nov.): Implications for the Evolution of Diazotrophy and Nodulation in the Burkholderiaceae. Genes. 2018;9:389.  doi: 10.3390/genes9080389.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/genes9080389" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6116057/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30071618/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=Whole%20Genome%20Analyses%20Suggests%20That%20Burkholderia%20sensu%20lato%20Contains%20Two%20Additional%20Novel%20Genera%20(Mycetohabitans%20gen.%20nov.,%20and%20Trinickia%20gen.%20nov.):%20Implications%20for%20the%20Evolution%20of%20Diazotrophy%20and%20Nodulation%20in%20the%20Burkholderiaceae&amp;author=P.%20Estrada-de%20Los%20Santos&amp;author=M.%20Palmer&amp;author=B.%20Ch%C3%A1vez-Ram%C3%ADrez&amp;author=C.%20Beukes&amp;author=E.T.%20Steenkamp&amp;volume=9&amp;publication_year=2018&amp;pages=389&amp;pmid=30071618&amp;doi=10.3390/genes9080389&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B168-microorganisms-11-01830">
<span class="label">168.</span><cite>Lastovetsky O.A., Krasnovsky L.D., Qin X., Gaspar M.L., Gryganskyi A.P., Huntemann M., Clum A., Pillay M., Palaniappan K., Varghese N., et al.  Molecular Dialogues between Early Divergent Fungi and Bacteria in an Antagonism versus a Mutualism. mBio. 2020;11:e02088-20. doi: 10.1128/mBio.02088-20.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/mBio.02088-20" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7482071/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32900811/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Molecular%20Dialogues%20between%20Early%20Divergent%20Fungi%20and%20Bacteria%20in%20an%20Antagonism%20versus%20a%20Mutualism&amp;author=O.A.%20Lastovetsky&amp;author=L.D.%20Krasnovsky&amp;author=X.%20Qin&amp;author=M.L.%20Gaspar&amp;author=A.P.%20Gryganskyi&amp;volume=11&amp;publication_year=2020&amp;pages=e02088-20&amp;pmid=32900811&amp;doi=10.1128/mBio.02088-20&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B169-microorganisms-11-01830">
<span class="label">169.</span><cite>Naumann M., Schüßler A., Bonfante P. The Obligate Endobacteria of Arbuscular Mycorrhizal Fungi Are Ancient Heritable Components Related to the Mollicutes. ISME J. 2010;4:862–871. doi: 10.1038/ismej.2010.21.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ismej.2010.21" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20237515/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=The%20Obligate%20Endobacteria%20of%20Arbuscular%20Mycorrhizal%20Fungi%20Are%20Ancient%20Heritable%20Components%20Related%20to%20the%20Mollicutes&amp;author=M.%20Naumann&amp;author=A.%20Sch%C3%BC%C3%9Fler&amp;author=P.%20Bonfante&amp;volume=4&amp;publication_year=2010&amp;pages=862-871&amp;pmid=20237515&amp;doi=10.1038/ismej.2010.21&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B170-microorganisms-11-01830">
<span class="label">170.</span><cite>Toomer K.H., Chen X., Naito M., Mondo S.J., den Bakker H.C., VanKuren N.W., Lekberg Y., Morton J.B., Pawlowska T.E. Molecular Evolution Patterns Reveal Life History Features of Mycoplasma-Related Endobacteria Associated with Arbuscular Mycorrhizal Fungi. Mol. Ecol. 2015;24:3485–3500. doi: 10.1111/mec.13250.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/mec.13250" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26011293/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Ecol.&amp;title=Molecular%20Evolution%20Patterns%20Reveal%20Life%20History%20Features%20of%20Mycoplasma-Related%20Endobacteria%20Associated%20with%20Arbuscular%20Mycorrhizal%20Fungi&amp;author=K.H.%20Toomer&amp;author=X.%20Chen&amp;author=M.%20Naito&amp;author=S.J.%20Mondo&amp;author=H.C.%20den%20Bakker&amp;volume=24&amp;publication_year=2015&amp;pages=3485-3500&amp;pmid=26011293&amp;doi=10.1111/mec.13250&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B171-microorganisms-11-01830">
<span class="label">171.</span><cite>Desirò A., Hao Z., Liber J.A., Benucci G.M.N., Lowry D., Roberson R., Bonito G. Mycoplasma-Related Endobacteria within Mortierellomycotina Fungi: Diversity, Distribution and Functional Insights into Their Lifestyle. ISME J. 2018;12:1743–1757. doi: 10.1038/s41396-018-0053-9.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41396-018-0053-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6018737/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29476142/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=Mycoplasma-Related%20Endobacteria%20within%20Mortierellomycotina%20Fungi:%20Diversity,%20Distribution%20and%20Functional%20Insights%20into%20Their%20Lifestyle&amp;author=A.%20Desir%C3%B2&amp;author=Z.%20Hao&amp;author=J.A.%20Liber&amp;author=G.M.N.%20Benucci&amp;author=D.%20Lowry&amp;volume=12&amp;publication_year=2018&amp;pages=1743-1757&amp;pmid=29476142&amp;doi=10.1038/s41396-018-0053-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B172-microorganisms-11-01830">
<span class="label">172.</span><cite>Desirò A., Faccio A., Kaech A., Bidartondo M.I., Bonfante P. Endogone, One of the Oldest Plant-Associated Fungi, Host Unique Mollicutes-Related Endobacteria. New Phytol. 2015;205:1464–1472. doi: 10.1111/nph.13136.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/nph.13136" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25345989/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Endogone,%20One%20of%20the%20Oldest%20Plant-Associated%20Fungi,%20Host%20Unique%20Mollicutes-Related%20Endobacteria&amp;author=A.%20Desir%C3%B2&amp;author=A.%20Faccio&amp;author=A.%20Kaech&amp;author=M.I.%20Bidartondo&amp;author=P.%20Bonfante&amp;volume=205&amp;publication_year=2015&amp;pages=1464-1472&amp;pmid=25345989&amp;doi=10.1111/nph.13136&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B173-microorganisms-11-01830">
<span class="label">173.</span><cite>Naito M., Desirò A., González J.B., Tao G., Morton J.B., Bonfante P., Pawlowska T.E. “Candidatus Moeniiplasma Glomeromycotorum”, an Endobacterium of Arbuscular Mycorrhizal Fungi. Int. J. Syst. Evol. Microbiol. 2017;67:1177–1184. doi: 10.1099/ijsem.0.001785.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1099/ijsem.0.001785" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28073398/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;title=%E2%80%9CCandidatus%20Moeniiplasma%20Glomeromycotorum%E2%80%9D,%20an%20Endobacterium%20of%20Arbuscular%20Mycorrhizal%20Fungi&amp;author=M.%20Naito&amp;author=A.%20Desir%C3%B2&amp;author=J.B.%20Gonz%C3%A1lez&amp;author=G.%20Tao&amp;author=J.B.%20Morton&amp;volume=67&amp;publication_year=2017&amp;pages=1177-1184&amp;pmid=28073398&amp;doi=10.1099/ijsem.0.001785&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B174-microorganisms-11-01830">
<span class="label">174.</span><cite>Naito M., Morton J.B., Pawlowska T.E. Minimal Genomes of Mycoplasma-Related Endobacteria Are Plastic and Contain Host-Derived Genes for Sustained Life within Glomeromycota. Proc. Natl. Acad. Sci. USA. 2015;112:7791–7796. doi: 10.1073/pnas.1501676112.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1501676112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4485128/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25964324/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Minimal%20Genomes%20of%20Mycoplasma-Related%20Endobacteria%20Are%20Plastic%20and%20Contain%20Host-Derived%20Genes%20for%20Sustained%20Life%20within%20Glomeromycota&amp;author=M.%20Naito&amp;author=J.B.%20Morton&amp;author=T.E.%20Pawlowska&amp;volume=112&amp;publication_year=2015&amp;pages=7791-7796&amp;pmid=25964324&amp;doi=10.1073/pnas.1501676112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B175-microorganisms-11-01830">
<span class="label">175.</span><cite>Robinson A.J., House G.L., Morales D.P., Kelliher J.M., Gallegos-Graves L.V., LeBrun E.S., Davenport K.W., Palmieri F., Lohberger A., Bregnard D., et al.  Widespread Bacterial Diversity within the Bacteriome of Fungi. Commun. Biol. 2021;4:1168. doi: 10.1038/s42003-021-02693-y.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s42003-021-02693-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8497576/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34621007/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Commun.%20Biol.&amp;title=Widespread%20Bacterial%20Diversity%20within%20the%20Bacteriome%20of%20Fungi&amp;author=A.J.%20Robinson&amp;author=G.L.%20House&amp;author=D.P.%20Morales&amp;author=J.M.%20Kelliher&amp;author=L.V.%20Gallegos-Graves&amp;volume=4&amp;publication_year=2021&amp;pages=1168&amp;pmid=34621007&amp;doi=10.1038/s42003-021-02693-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B176-microorganisms-11-01830">
<span class="label">176.</span><cite>Telagathoti A., Probst M., Peintner U. Habitat, Snow-Cover and Soil PH, Affect the Distribution and Diversity of Mortierellaceae Species and Their Associations to Bacteria. Front. Microbiol. 2021;12:669784. doi: 10.3389/fmicb.2021.669784.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2021.669784" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8283828/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34276602/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Habitat,%20Snow-Cover%20and%20Soil%20PH,%20Affect%20the%20Distribution%20and%20Diversity%20of%20Mortierellaceae%20Species%20and%20Their%20Associations%20to%20Bacteria&amp;author=A.%20Telagathoti&amp;author=M.%20Probst&amp;author=U.%20Peintner&amp;volume=12&amp;publication_year=2021&amp;pages=669784&amp;pmid=34276602&amp;doi=10.3389/fmicb.2021.669784&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B177-microorganisms-11-01830">
<span class="label">177.</span><cite>Ettinger C.L., Lovett B., Kasson M.T., Stajich J.E. Metagenome-Assembled Genomes of Bacteria Associated with Massospora cicadina Fungal Plugs from Infected Brood VIII Periodical Cicadas. Microbiol. Resour. Announc. 2022;11:e00413-22. doi: 10.1128/mra.00413-22.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/mra.00413-22" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9584288/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36036588/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Resour.%20Announc.&amp;title=Metagenome-Assembled%20Genomes%20of%20Bacteria%20Associated%20with%20Massospora%20cicadina%20Fungal%20Plugs%20from%20Infected%20Brood%20VIII%20Periodical%20Cicadas&amp;author=C.L.%20Ettinger&amp;author=B.%20Lovett&amp;author=M.T.%20Kasson&amp;author=J.E.%20Stajich&amp;volume=11&amp;publication_year=2022&amp;pages=e00413-22&amp;pmid=36036588&amp;doi=10.1128/mra.00413-22&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B178-microorganisms-11-01830">
<span class="label">178.</span><cite>Chang Y., Wang Y., Mondo S., Ahrendt S., Andreopoulos W., Barry K., Beard J., Benny G.L., Blankenship S., Bonito G., et al.  Evolution of Zygomycete Secretomes and the Origins of Terrestrial Fungal Ecologies. iScience. 2022;25:104840. doi: 10.1016/j.isci.2022.104840.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.isci.2022.104840" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9391592/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35996588/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=Evolution%20of%20Zygomycete%20Secretomes%20and%20the%20Origins%20of%20Terrestrial%20Fungal%20Ecologies&amp;author=Y.%20Chang&amp;author=Y.%20Wang&amp;author=S.%20Mondo&amp;author=S.%20Ahrendt&amp;author=W.%20Andreopoulos&amp;volume=25&amp;publication_year=2022&amp;pages=104840&amp;pmid=35996588&amp;doi=10.1016/j.isci.2022.104840&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B179-microorganisms-11-01830">
<span class="label">179.</span><cite>Shelest E., Voigt K.  Fungal Genomics. Volume XIII. Springer; Berlin/Heidelberg, Germany: 2014. 2 Genomics to Study Basal Lineage Fungal Biology: Phylogenomics Suggests a Common Origin; pp. 31–60. The Mycota.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Fungal%20Genomics&amp;author=E.%20Shelest&amp;author=K.%20Voigt&amp;publication_year=2014&amp;isbn=978-3-642-45217-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B180-microorganisms-11-01830">
<span class="label">180.</span><cite>Voigt K., Wolf T., Ochsenreiter K., Nagy G., Kaerger K., Shelest E., Papp T.  Biochemistry and Molecular Biology. Volume III. Springer; Berlin/Heidelberg, Germany: 2016. 15 Genetic and Metabolic Aspects of Primary and Secondary Metabolism of the Zygomycetes; pp. 361–385. The Mycota.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Biochemistry%20and%20Molecular%20Biology&amp;author=K.%20Voigt&amp;author=T.%20Wolf&amp;author=K.%20Ochsenreiter&amp;author=G.%20Nagy&amp;author=K.%20Kaerger&amp;publication_year=2016&amp;isbn=978-3-319-27788-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B181-microorganisms-11-01830">
<span class="label">181.</span><cite>Koczyk G., Pawłowska J., Muszewska A. Terpenoid Biosynthesis Dominates among Secondary Metabolite Clusters in MUcoromycotina Genomes. J. Fungi. 2021;7:285.  doi: 10.3390/jof7040285.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/jof7040285" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8070225/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33918813/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Fungi&amp;title=Terpenoid%20Biosynthesis%20Dominates%20among%20Secondary%20Metabolite%20Clusters%20in%20MUcoromycotina%20Genomes&amp;author=G.%20Koczyk&amp;author=J.%20Paw%C5%82owska&amp;author=A.%20Muszewska&amp;volume=7&amp;publication_year=2021&amp;pages=285&amp;pmid=33918813&amp;doi=10.3390/jof7040285&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B182-microorganisms-11-01830">
<span class="label">182.</span><cite>Baldeweg F., Warncke P., Fischer D., Gressler M. Fungal Biosurfactants from Mortierella alpina. Org. Lett. 2019;21:1444–1448. doi: 10.1021/acs.orglett.9b00193.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/acs.orglett.9b00193" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30789272/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&amp;title=Fungal%20Biosurfactants%20from%20Mortierella%20alpina&amp;author=F.%20Baldeweg&amp;author=P.%20Warncke&amp;author=D.%20Fischer&amp;author=M.%20Gressler&amp;volume=21&amp;publication_year=2019&amp;pages=1444-1448&amp;pmid=30789272&amp;doi=10.1021/acs.orglett.9b00193&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B183-microorganisms-11-01830">
<span class="label">183.</span><cite>Lebreton A., Meslet-Cladière L., Morin-Sardin S., Coton E., Jany J.-L., Barbier G., Corre E. Comparative Analysis of Five Mucor Species Transcriptomes. Genomics. 2019;111:1306–1314. doi: 10.1016/j.ygeno.2018.09.003.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ygeno.2018.09.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30195068/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Comparative%20Analysis%20of%20Five%20Mucor%20Species%20Transcriptomes&amp;author=A.%20Lebreton&amp;author=L.%20Meslet-Cladi%C3%A8re&amp;author=S.%20Morin-Sardin&amp;author=E.%20Coton&amp;author=J.-L.%20Jany&amp;volume=111&amp;publication_year=2019&amp;pages=1306-1314&amp;pmid=30195068&amp;doi=10.1016/j.ygeno.2018.09.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B184-microorganisms-11-01830">
<span class="label">184.</span><cite>Winkelmann G. A Search for Glomuferrin: A Potential Siderophore of Arbuscular Mycorrhizal Fungi of the Genus Glomus. Biometals. 2017;30:559–564. doi: 10.1007/s10534-017-0026-x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s10534-017-0026-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28616783/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biometals&amp;title=A%20Search%20for%20Glomuferrin:%20A%20Potential%20Siderophore%20of%20Arbuscular%20Mycorrhizal%20Fungi%20of%20the%20Genus%20Glomus&amp;author=G.%20Winkelmann&amp;volume=30&amp;publication_year=2017&amp;pages=559-564&amp;pmid=28616783&amp;doi=10.1007/s10534-017-0026-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B185-microorganisms-11-01830">
<span class="label">185.</span><cite>Prakash H., Rudramurthy S.M., Gandham P.S., Ghosh A.K., Kumar M.M., Badapanda C., Chakrabarti A. Apophysomyces variabilis: Draft Genome Sequence and Comparison of Predictive Virulence Determinants with Other Medically Important Mucorales. BMC Genom. 2017;18:736.  doi: 10.1186/s12864-017-4136-1.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12864-017-4136-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5604411/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28923009/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&amp;title=Apophysomyces%20variabilis:%20Draft%20Genome%20Sequence%20and%20Comparison%20of%20Predictive%20Virulence%20Determinants%20with%20Other%20Medically%20Important%20Mucorales&amp;author=H.%20Prakash&amp;author=S.M.%20Rudramurthy&amp;author=P.S.%20Gandham&amp;author=A.K.%20Ghosh&amp;author=M.M.%20Kumar&amp;volume=18&amp;publication_year=2017&amp;pages=736&amp;pmid=28923009&amp;doi=10.1186/s12864-017-4136-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B186-microorganisms-11-01830">
<span class="label">186.</span><cite>Carroll C.S., Grieve C.L., Murugathasan I., Bennet A.J., Czekster C.M., Liu H., Naismith J., Moore M.M. The Rhizoferrin Biosynthetic Gene in the Fungal Pathogen Rhizopus delemar Is a Novel Member of the NIS Gene Family. Int. J. Biochem. Cell Biol. 2017;89:136–146. doi: 10.1016/j.biocel.2017.06.005.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.biocel.2017.06.005" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28610916/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Biochem.%20Cell%20Biol.&amp;title=The%20Rhizoferrin%20Biosynthetic%20Gene%20in%20the%20Fungal%20Pathogen%20Rhizopus%20delemar%20Is%20a%20Novel%20Member%20of%20the%20NIS%20Gene%20Family&amp;author=C.S.%20Carroll&amp;author=C.L.%20Grieve&amp;author=I.%20Murugathasan&amp;author=A.J.%20Bennet&amp;author=C.M.%20Czekster&amp;volume=89&amp;publication_year=2017&amp;pages=136-146&amp;pmid=28610916&amp;doi=10.1016/j.biocel.2017.06.005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B187-microorganisms-11-01830">
<span class="label">187.</span><cite>Škríba A., Patil R.H., Hubáček P., Dobiáš R., Palyzová A., Marešová H., Pluháček T., Havlíček V. Rhizoferrin Glycosylation in Rhizopus microsporus. J. Fungi. 2020;6:89.  doi: 10.3390/jof6020089.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/jof6020089" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7344610/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32570979/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Fungi&amp;title=Rhizoferrin%20Glycosylation%20in%20Rhizopus%20microsporus&amp;author=A.%20%C5%A0kr%C3%ADba&amp;author=R.H.%20Patil&amp;author=P.%20Hub%C3%A1%C4%8Dek&amp;author=R.%20Dobi%C3%A1%C5%A1&amp;author=A.%20Palyzov%C3%A1&amp;volume=6&amp;publication_year=2020&amp;pages=89&amp;pmid=32570979&amp;doi=10.3390/jof6020089&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B188-microorganisms-11-01830">
<span class="label">188.</span><cite>Takeda I., Tamano K., Yamane N., Ishii T., Miura A., Umemura M., Terai G., Baker S.E., Koike H., Machida M. Genome Sequence of the Mucoromycotina Fungus Umbelopsis isabellina, an Effective Producer of Lipids. Genome Announc. 2014;2:e00071-14. doi: 10.1128/genomeA.00071-14.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/genomeA.00071-14" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3937604/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24578268/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Announc.&amp;title=Genome%20Sequence%20of%20the%20Mucoromycotina%20Fungus%20Umbelopsis%20isabellina,%20an%20Effective%20Producer%20of%20Lipids&amp;author=I.%20Takeda&amp;author=K.%20Tamano&amp;author=N.%20Yamane&amp;author=T.%20Ishii&amp;author=A.%20Miura&amp;volume=2&amp;publication_year=2014&amp;pages=e00071-14&amp;pmid=24578268&amp;doi=10.1128/genomeA.00071-14&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B189-microorganisms-11-01830">
<span class="label">189.</span><cite>Gluck-Thaler E., Ralston T., Konkel Z., Ocampos C.G., Ganeshan V.D., Dorrance A.E., Niblack T.L., Wood C.W., Slot J.C., Lopez-Nicora H.D., et al.  Giant Starship Elements Mobilize Accessory Genes in Fungal Genomes. Mol. Biol. Evol. 2022;39:msac109. doi: 10.1093/molbev/msac109.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/msac109" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9156397/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35588244/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=Giant%20Starship%20Elements%20Mobilize%20Accessory%20Genes%20in%20Fungal%20Genomes&amp;author=E.%20Gluck-Thaler&amp;author=T.%20Ralston&amp;author=Z.%20Konkel&amp;author=C.G.%20Ocampos&amp;author=V.D.%20Ganeshan&amp;volume=39&amp;publication_year=2022&amp;pages=msac109&amp;pmid=35588244&amp;doi=10.1093/molbev/msac109&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B190-microorganisms-11-01830">
<span class="label">190.</span><cite>Urquhart A.S., Chong N.F., Yang Y., Idnurm A. A Large Transposable Element Mediates Metal Resistance in the Fungus Paecilomyces variotii. Curr. Biol. 2022;32:937–950.e5. doi: 10.1016/j.cub.2021.12.048.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cub.2021.12.048" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35063120/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=A%20Large%20Transposable%20Element%20Mediates%20Metal%20Resistance%20in%20the%20Fungus%20Paecilomyces%20variotii&amp;author=A.S.%20Urquhart&amp;author=N.F.%20Chong&amp;author=Y.%20Yang&amp;author=A.%20Idnurm&amp;volume=32&amp;publication_year=2022&amp;pages=937-950.e5&amp;pmid=35063120&amp;doi=10.1016/j.cub.2021.12.048&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10386755/bin/microorganisms-11-01830-s001.zip">Click here for additional data file.</a><sup> (104KB, zip) </sup>
</div></div></section></section></section></section></section></article>
</main></body></html>