RNA

. 2015 Dec;21(12):2103–2118. doi: [10.1261/rna.053934.115](https://doi.org/10.1261/rna.053934.115)

# An extensive allelic series of *Drosophila kae1* mutants reveals diverse and tissue-specific requirements for t6A biogenesis

[Ching-Jung Lin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20CJ%22%5BAuthor%5D)

### Ching-Jung Lin

1Department of Developmental Biology, Sloan-Kettering Institute, New York, New York 10065, USA

Find articles by [Ching-Jung Lin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20CJ%22%5BAuthor%5D)

1,7, [Peter Smibert](https://pubmed.ncbi.nlm.nih.gov/?term=%22Smibert%20P%22%5BAuthor%5D)

### Peter Smibert

1Department of Developmental Biology, Sloan-Kettering Institute, New York, New York 10065, USA

2Research School of Biological Sciences, The Australian National University, Acton ACT 2601, Australia

Find articles by [Peter Smibert](https://pubmed.ncbi.nlm.nih.gov/?term=%22Smibert%20P%22%5BAuthor%5D)

1,2,7,8, [Xiaoyu Zhao](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20X%22%5BAuthor%5D)

### Xiaoyu Zhao

3Department of Biochemistry and Cell Biology, Stony Brook University, Stony Brook, New York 11794, USA

Find articles by [Xiaoyu Zhao](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20X%22%5BAuthor%5D)

3, [Jennifer F Hu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20JF%22%5BAuthor%5D)

### Jennifer F Hu

4Department of Biological Engineering, Massachusetts Institute of Technology (MIT), Cambridge, Massachusetts 02139, USA

Find articles by [Jennifer F Hu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20JF%22%5BAuthor%5D)

4, [Johnny Ramroop](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramroop%20J%22%5BAuthor%5D)

### Johnny Ramroop

5Department of Biology, The City College of the City University of New York, New York 10031, USA

6The Graduate Center of the City University of New York, New York 10016, USA

Find articles by [Johnny Ramroop](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramroop%20J%22%5BAuthor%5D)

5,6, [Stefanie M Kellner](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kellner%20SM%22%5BAuthor%5D)

### Stefanie M Kellner

4Department of Biological Engineering, Massachusetts Institute of Technology (MIT), Cambridge, Massachusetts 02139, USA

Find articles by [Stefanie M Kellner](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kellner%20SM%22%5BAuthor%5D)

4, [Matthew A Benton](https://pubmed.ncbi.nlm.nih.gov/?term=%22Benton%20MA%22%5BAuthor%5D)

### Matthew A Benton

2Research School of Biological Sciences, The Australian National University, Acton ACT 2601, Australia

Find articles by [Matthew A Benton](https://pubmed.ncbi.nlm.nih.gov/?term=%22Benton%20MA%22%5BAuthor%5D)

2,9, [Shubha Govind](https://pubmed.ncbi.nlm.nih.gov/?term=%22Govind%20S%22%5BAuthor%5D)

### Shubha Govind

5Department of Biology, The City College of the City University of New York, New York 10031, USA

6The Graduate Center of the City University of New York, New York 10016, USA

Find articles by [Shubha Govind](https://pubmed.ncbi.nlm.nih.gov/?term=%22Govind%20S%22%5BAuthor%5D)

5,6, [Peter C Dedon](https://pubmed.ncbi.nlm.nih.gov/?term=%22Dedon%20PC%22%5BAuthor%5D)

### Peter C Dedon

4Department of Biological Engineering, Massachusetts Institute of Technology (MIT), Cambridge, Massachusetts 02139, USA

Find articles by [Peter C Dedon](https://pubmed.ncbi.nlm.nih.gov/?term=%22Dedon%20PC%22%5BAuthor%5D)

4, [Rolf Sternglanz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sternglanz%20R%22%5BAuthor%5D)

### Rolf Sternglanz

3Department of Biochemistry and Cell Biology, Stony Brook University, Stony Brook, New York 11794, USA

Find articles by [Rolf Sternglanz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sternglanz%20R%22%5BAuthor%5D)

3, [Eric C Lai](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lai%20EC%22%5BAuthor%5D)

### Eric C Lai

1Department of Developmental Biology, Sloan-Kettering Institute, New York, New York 10065, USA

Find articles by [Eric C Lai](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lai%20EC%22%5BAuthor%5D)

1

* Author information
* Article notes
* Copyright and License information

1Department of Developmental Biology, Sloan-Kettering Institute, New York, New York 10065, USA

2Research School of Biological Sciences, The Australian National University, Acton ACT 2601, Australia

3Department of Biochemistry and Cell Biology, Stony Brook University, Stony Brook, New York 11794, USA

4Department of Biological Engineering, Massachusetts Institute of Technology (MIT), Cambridge, Massachusetts 02139, USA

5Department of Biology, The City College of the City University of New York, New York 10031, USA

6The Graduate Center of the City University of New York, New York 10016, USA

7

These authors contributed equally to this work.

8

Present address: New York Genome Center, New York, NY 10013, USA

9

Present address: University of Cologne, D-50923 Cologne, Germany

✉

Corresponding author: laie@mskcc.org

Received 2015 Aug 11; Accepted 2015 Sep 14.

[© 2015 Lin et al.; Published by Cold Spring Harbor Laboratory Press for the RNA Society](http://www.rnajournal.org/site/misc/terms.xhtml)

This article is distributed exclusively by the RNA Society for the first 12 months after the full-issue publication date (see <http://rnajournal.cshlp.org/site/misc/terms.xhtml>). After 12 months, it is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), as described at <http://creativecommons.org/licenses/by-nc/4.0/>.

[PMC Copyright notice](/about/copyright/)

PMCID: PMC4647464  PMID: [26516084](https://pubmed.ncbi.nlm.nih.gov/26516084/)

## Abstract

*N**6*-threonylcarbamoyl-adenosine (t6A) is one of the few RNA modifications that is universally present in life. This modification occurs at high frequency at position 37 of most tRNAs that decode ANN codons, and stabilizes cognate anticodon–codon interactions. Nearly all genetic studies of the t6A pathway have focused on single-celled organisms. In this study, we report the isolation of an extensive allelic series in the *Drosophila* ortholog of the core t6A biosynthesis factor Kae1. *kae1* hemizygous larvae exhibit decreases in t6A that correlate with allele strength; however, we still detect substantial t6A-modified tRNAs even during the extended larval phase of null alleles. Nevertheless, complementation of *Drosophila* Kae1 and other t6A factors in corresponding yeast null mutants demonstrates that these metazoan genes execute t6A synthesis. Turning to the biological consequences of t6A loss, we characterize prominent *kae1* melanotic masses and show that they are associated with lymph gland overgrowth and ectopic generation of lamellocytes. On the other hand, *kae1* mutants exhibit other phenotypes that reflect insufficient tissue growth. Interestingly, whole-tissue and clonal analyses show that strongly mitotic tissues such as imaginal discs are exquisitely sensitive to loss of *kae1*, whereas nonproliferating tissues are less affected. Indeed, despite overt requirements of t6A for growth of many tissues, certain strong *kae1* alleles achieve and sustain enlarged body size during their extended larval phase. Our studies highlight tissue-specific requirements of the t6A pathway in a metazoan context and provide insights into the diverse biological roles of this fundamental RNA modification during animal development and disease.

**Keywords:** *Drosophila*, *Kae1*, *N*6-threonylcarbamoyladenosine, t6A

## INTRODUCTION

Beyond the four canonical ribonucleosides present in RNA, nearly all life forms bear an array of modified bases in various transcript populations. Over 100 chemical modifications have been documented, with dozens often present in individual species; the majority of distinct modified ribonucleosides occur on tRNA ([Machnicka et al. 2013](#LINRNA053934C34)). Altogether, RNA modifications collectively play diverse roles in RNA quality control, secondary structure, transcript stability, translational efficiency, translational fidelity, and regulation of RNA–protein interactions ([Gu et al. 2014](#LINRNA053934C15); [Hopper and Huang 2015](#LINRNA053934C21)) and can influence disease and cancer progression ([Grewal 2015](#LINRNA053934C13); [Kirchner and Ignatova 2015](#LINRNA053934C26)).

While many post-transcriptional modifications of RNA are broadly conserved, only a few are universal across known life forms. One prominent example is *N**6*-threonylcarbamoyladenosine (t6A), which is found at high frequency at position 37 of tRNAs that decode ANN codons ([Grosjean et al. 1995](#LINRNA053934C14)). This modification was recognized decades ago, and it stabilizes cognate codon–anticodon interactions through a stacking interaction that stabilizes the weak interaction with the first codon base, which comprises a weak A–U pair ([Weissenbach and Grosjean 1981](#LINRNA053934C52); [Yarian et al. 2002](#LINRNA053934C53); [Agris 2008](#LINRNA053934C1)). However, the enzymes that generate this modification remained elusive until relatively recently.

Based on the inference that all organisms harbor t6A-modified tRNAs ([Sprinzl and Vassilenko 2005](#LINRNA053934C45)), bioinformatic isolation of universally conserved genes of unknown function ([Galperin and Koonin 2004](#LINRNA053934C11)) yielded a short list of candidate genes that first led to the identification of YrdC/Sua5 as a t6A pathway component ([El Yacoubi et al. 2009](#LINRNA053934C9)). Subsequently, members of the universal gene family Kae1/Qri7/YgjD were recognized to be essential for t6A biosynthesis, along with its partner proteins (in yeast) Pcc1 and Bud32 ([El Yacoubi et al. 2011](#LINRNA053934C10); [Srinivasan et al. 2011](#LINRNA053934C46)). Of note, mutations in components of the Kae1, Pcc1, Bud32, and Cgi121 complex (collectively termed KEOPS) had been isolated some years earlier, via divergent approaches intended at identifying factors in telomere maintenance ([Downey et al. 2006](#LINRNA053934C8)) and splicing/transcription ([Kisseleva-Romanova et al. 2006](#LINRNA053934C27)). At the time, the molecular role of the KEOPS complex was obscure, but its deep conservation justified solving the atomic structures of the archaeal Kae1/Bud32 natural fusion protein ([Hecker et al. 2008](#LINRNA053934C19)) and the entire eukaryotic KEOPS complex ([Mao et al. 2008](#LINRNA053934C35)). These studies assigned Kae1 as an ASKHA-type ATPase, Bud32 as a kinase, and Pcc1 as a dimerization subunit, with Kae1 physically interacting with both Bud32 and Pcc1.

Only later, with the recognition that these genes are involved in t6A accumulation, did efforts become possible to reconstitute the biosynthesis of t6A-modified tRNA in vitro, using purified bacterial YrdC (Sua5) and YgjD (Kae1) and two bacterial-specific genes that cluster with YgjD, namely YjeE and YeaZ ([Deutsch et al. 2012](#LINRNA053934C6)). Similarly, yeast Sua5 and KEOPS complex sufficed to generate t6A-modified tRNA in vitro ([Perrochia et al. 2013](#LINRNA053934C42)). The proposed mechanism is that Sua5 converts bicarbonate and threonine via a series of intermediates into threonylcarbamoyladenylate, which is transferred to the active site of Kae1 within the KEOPS complex to modify tRNA position A37.

Notably, Kae1 is the only truly universally conserved component of KEOPS ([Galperin and Koonin 2004](#LINRNA053934C11)), and eukaryotes contain a paralog of Kae1 termed Qri7 that is mitochondrially localized ([Huh et al. 2003](#LINRNA053934C22)). The fact that no other KEOPS components localize to mitochondria led to a suspicion that Qri7 represents a primal core factor that does not require additional presumably accessory KEOPS components. Indeed, subcellular redirection of Qri7 to the cytoplasm rescued the slow growth phenotypes of all KEOPS members that are defective for t6A accumulation ([Wan et al. 2013](#LINRNA053934C49)). Moreover, in vitro synthesis of t6A-modified tRNA was reconstituted solely with purified Sua5, Qri7, and substrate tRNA, in the presence of threonine, bicarbonate, and ATP ([Wan et al. 2013](#LINRNA053934C49)). Thus, Kae1/Qri7 is the central downstream enzyme in the generation of t6A-modified tRNA.

The assignment of core genes in the t6A pathway permits organismal phenotypes to be correlated with the absence of these modified tRNAs. Budding yeast t6A pathway mutants ([El Yacoubi et al. 2009](#LINRNA053934C9), [2011](#LINRNA053934C10); [Srinivasan et al. 2011](#LINRNA053934C46)) are very slow-growing, while the bacterial pathway (e.g., in *Bacillus subtilis* and *Escherichia coli*) is essential for viability and cell morphology ([Arigoni et al. 1998](#LINRNA053934C3); [Handford et al. 2009](#LINRNA053934C17)). These seemingly general defects befit loss of a fundamental RNA modification pathway, but do not lend insight into particular processes. Otherwise, mutants of KEOPS complex members in single-celled organisms have impact on telomere maintenance ([Downey et al. 2006](#LINRNA053934C8)), transcription ([Kisseleva-Romanova et al. 2006](#LINRNA053934C27)), and genome stability ([Oberto et al. 2009](#LINRNA053934C40)), although it remains unclear how directly the loss of t6A relates to these phenotypes.

However, certain phenotypes now seem to be directly related to the biochemical function of t6A in improving translational fidelity. Of note, yeast *sua5* was originally isolated as a suppressor of *cyc1* alleles harboring an aberrant upstream start codon (uORF) ([Hampsey et al. 1991](#LINRNA053934C16); [Na et al. 1992](#LINRNA053934C39)), implicating Sua5 in start site selection; AUG is indeed recognized by a t6A-modified tRNA. Correspondingly, Sua5 affects various aspects of normal translation, as mutants exhibit increased leaky scanning through AUG, frameshifting, and nonsense suppression ([Lin et al. 2010](#LINRNA053934C32)). One prominent locus controlled via endogenous uORFs is the amino acid starvation-responsive transcription factor GCN4 ([Hinnebusch 2005](#LINRNA053934C20)). Indeed, not only does *sua5* derepress *GCN4*, KEOPS mutants (*kae1* and *pcc1*) induce a gene expression signature that is substantially enriched for Gcn4 targets ([Daugeron et al. 2011](#LINRNA053934C5)). These precedents suggest that it may be possible to make linkages between altered translation and organismal phenotypes, even for this “housekeeping” cellular activity.

To date, genetic analysis of t6A pathway components has mostly been limited to single cell organisms, such as microbes and yeast. One study on the *Drosophila melanogaster* Bud32/PRPK homolog utilized RNAi-mediated knockdowns to define growth related defects upon *bud32* depletion, although biochemical evidence was not presented for an impact on t6A biogenesis in this condition ([Ibar et al. 2013](#LINRNA053934C23)). Interestingly, the growth defects were related to PI3K/TOR signaling, and could be partially restored by expression of activated S6-kinase ([Ibar et al. 2013](#LINRNA053934C23)).

In this study, we describe the isolation of a large allelic series of *D. melanogaster kae1* mutants, ranging from weak hypomorphs, strong mutants, to null alleles. These present various growth-related defects, developmental delay, and melanotic tumors. We present extensive biochemical and genetic evidence that *Drosophila kae1* and its partners are core factors in t6A biogenesis. Notably, our phenotypic analysis reveals substantial differences in the genetic requirements for t6A in proliferating versus nonproliferating tissues. In particular, analysis of mitotic tissues in full knockout and in clonal settings indicates an essential requirement for *kae1*, whereas various nonproliferating tissues can be maintained in the null state for weeks. Moreover, *kae1* appears to act as a tumor suppressor in the larval hematopoietic tissue. These data highlight biological complexity in the requirements of a general regulatory factor, and the utility of the *Drosophila* model to further understand the roles of t6A modification during animal development and disease.

## RESULTS

### Isolation of the “*Black Spot*” complementation group

During efforts to isolate mutants in the 72D-F region, we conducted an F2 EMS mutagenesis screen against *Df(3L)ED220*, which deletes 324,210 bp of chromosome 3L:16,080,584–16,404,777 (see Materials and Methods). We screened ∼7000 crosses of animals bearing mutagenized chromosomes to this deficiency, and recovered 64 lethals. Within this collection, we recognized numerous alleles for which hemizygous larvae bore characteristic melanotic masses, which we colloquially termed “*Black Spot*” mutants ([Fig. 1](#LINRNA053934F1)A). These mutants failed to complement each other, and they typically exhibited an extended larval period prior to lethality ([Fig. 1](#LINRNA053934F1)B–D; Supplemental Fig. 1). Notably, while *Drosophila* larvae complete their larval stages around day 5 and commence pupation, some *Black Spot* mutants survive as larvae for up to 18 d. Systematic tests against all of the lethal alleles we obtained in this genomic interval revealed additional noncomplementing mutations. Many of these mutants lacked black spots or generated them with low frequency ([Fig. 1](#LINRNA053934F1)B–D). Several of these alleles were able to develop to pupal stages or even into pharate adults as hemizygotes, suggesting they might represent hypomorphs. Overall, these eight alleles appear to represent an extensive allelic series.

#### FIGURE 1.

![FIGURE 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a649/4647464/13bc8472152b/2103F01.jpg)

[Open in a new tab](figure/LINRNA053934F1/)

Isolation and characterization of *Black Spot* mutants. (*A*) Exemplar of the *Black Spot* phenotype. *[1A13]*/*Df(3L)ED220* larvae exhibit melanotic nodules of various sizes; a particularly large one is labeled with an arrowhead (“BS”). (*B*–*D*) Complementation tests identified a collection of *Black Spot* mutants, only some of which actually generate melanotic masses with high penetrance (examples labeled with arrowheads). Normal (wt) animals complete their third instar larval stage by day ∼5, and proceed to ∼5 d of pupal development. Pupae containing pharate animals are visible by day ∼10, and they eclose shortly thereafter as adult flies. Wt animals are shown as a reference to the *left* of *B* and *C*. In contrast, many hemizygous *Black Spot* alleles are substantially developmentally delayed and maintain their larval phase until days 12 (*C*) to 16 (*D*), with some living mutants detected at day 18 (not shown). All of the larvae collected in this summary were mobile and thus alive. Hypomorphic mutants could pupate and even advance to the pharate stage; however, it is likely that some of the pupae shown at the latest days are deceased. See Supplemental Figure 1 for additional notes regarding the larval extended phase, lethal phase, and black spot penetrance at different ages in the different hemizygous mutants. (*E*) Several of the *Black Spot* mutants attain larger larval size than normal; shown are *[1A13]* hemizygotes as an example. Other alleles such as *[f10978]* are growth deficient. All alleles tested could be rescued by *kae1* genomic transgenes; shown is rescue of *[f10978]/Df* larva.

### *Black Spot* mutants affect the *Drosophila* ortholog of Kae1

To identify the gene mutated in *Black Spot* mutants, we used *P*-element mapping ([Zhai et al. 2003](#LINRNA053934C54)) to narrow the region of interest within the deficiency (Supplemental Fig. 2). Coding exons from genes in the region were amplified by PCR from DNA extracted from hemizygous mutant larvae and sequenced. We identified one locus (*CG4933*), the fly ortholog of *Kae1*, that harbored nonconservative changes in all eight alleles ([Fig. 2](#LINRNA053934F2)A). Subsequently, we identified two uncharacterized mutants in 72F-73A (*l(3)72Fb[209]* and *l(3)72Fb[331]*, V Bedian, pers. comm. to FlyBase) that failed to complement *Black Spot* mutants. Sequencing showed that the *l(3)72Fb* alleles exhibit missense and nonsense mutations in *CG4933*, respectively. Notably, while *l(3)72Fb[209]* mutates a residue in the ATP-binding pocket, none of the other point mutations directly affect any functionally characterized regions (e.g., other residues that bind ATP or metal ion, or Kae1 faces that bind its partner KEOPS proteins Pcc1 or Bud32, [Fig. 2](#LINRNA053934F2)A). Finally, we found that *PBac{WH}CG4933[f01978]*, which carries a piggyBac insertion into the *CG4933* 5′ UTR ([dos Santos et al. 2015](#LINRNA053934C7)), also failed to complement *Black Spot* mutants. These observations strongly support that *CG4933* corresponds to *Black Spot*. Consistent with this, qRT-PCR analysis of several independent point mutants showed moderate decreases in *CG4933* transcripts, and these were undetectable in *f01978* hemizygotes ([Fig. 2](#LINRNA053934F2)B,C).

#### FIGURE 2.

[![FIGURE 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a649/4647464/0f63c224b70b/2103F02.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4647464_2103F02.jpg)

[Open in a new tab](figure/LINRNA053934F2/)

Summary of *Drosophila kae1 (CG4933)* mutant alleles. (*A*) An alignment of fruit fly, human, yeast, and archaeal orthologs of Kae1. Functionally/structurally defined regions are highlighted, including the binding regions for Pcc1 and Bud32, and the ATP- and ion-binding regions. All *Black Spot* alleles recovered from our screen revealed that point mutations affected well-conserved residues. Notably, none of the mutations directly affects a known functional site, although a few mutations are in the vicinity of such regions. The noncomplementing allele *l(3)72Fb[331]*, identified from a public stock collection, bears a nonsense mutation in *kae1*. Note that 1A2 and 1A19 induce the same coding change, even though these were isolated from different chromosomes in the screen. (*B*) qRT-PCR analysis of *kae1* from various homozygous or hemizygous mutants. All of our point alleles maintain substantial levels of kae1 transcripts, although several of them are reduced by 20%–30%. The pBac insertion into the 5′ UTR of *kae1*, *f[10978]*, appears to be a null allele as no transcript was detected. (*C*) Agarose gel analysis of qRT-PCR amplicons confirms the absence of *kae1* transcripts in *f[10978]/Df* larvae.

To confirm that *Black Spot* phenotypes were functionally due to loss of *CG4933*, we generated untagged and GFP-tagged genomic transgenes using the *P[acman]* backbone. Both CG4933 constructs rescued independent lethal hemizygotes (*1A13* and *f01978*) to adult viability, and no melanotic masses were observed in rescued animals (e.g., [Figs. 1](#LINRNA053934F1)E, [3](#LINRNA053934F3) and other data not shown). In summary, we have isolated metazoan mutants of *kae1*, encoding a universal tRNA modification factor, representing the first extensive allelic series of a t6A pathway factor in any organism.

#### FIGURE 3.

[![FIGURE 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a649/4647464/a0480146e9bb/2103F03.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4647464_2103F03.jpg)

[Open in a new tab](figure/LINRNA053934F3/)

Analysis of t6A-modified tRNAs in yeast KEOPS and *Drosophila kae1* mutants. (*A*) Analysis of yeast t6A pathway mutants is shown as a control. Primer extension assays yield a ladder of bands indicating the presence of stalled products. These include a band at position A37 of tRNA-Ile[AAU] of wild-type yeast that is strongly depleted in yeast t6A pathway mutants, thus indicating a t6A-modified nucleoside. tRNA-Val[UAC] is shown as a control for a non-t6A-bearing substrate. The ladder positions are inaccurate due to incomplete digestion; therefore, sizes are labeled according to the input primer. (*B*) Analysis of *Drosophila kae1* mutants. Modifications of the A37 positions of tRNA-Met[CAU] and tRNA-Ile[AAU] are decreased although not eliminated in two strong hemizygous conditions of *kae1*. The reduction observed in *kae1[1A13]/Df* was rescued by a *P[acman]-kae1* genomic transgene (“kae1+”). Note that mutant larvae were collected at an advanced age in their extended larval period to separate them from potential maternal contributions; the normal larval period ends at ∼5 d. (*C*) LC-MS/MS analysis of total tRNAs from *Drosophila kae1* hemizygous mutant larvae. We observe a minor decrease in total t6A-modified tRNAs in the hypomorphic allele *[7A9]* and stronger decreases in the strong/null alleles *[f01978]* and *[1A13]*. The levels of t6A-modified tRNAs did not decrease further in exceptionally long-lived *kae1* mutant larvae, and the loss of t6A was rescued by a *kae1* genomic transgene.

### Decrease, but not elimination, of t6A-modified tRNAs in *kae1* mutants

Although Sua5/KEOPS genes are well-conserved across different domains of life, their involvement in t6A synthesis has not formally been demonstrated outside of unicellular models. We initially utilized primer extension assays to assess potential defects in the accumulation of t6A-modified tRNAs. As positive controls, we assayed RNAs isolated from wild-type yeast and *sua5*, *kae1*, and *bud32* knockouts ([Srinivasan et al. 2011](#LINRNA053934C46)). Extensions using an end-labeled primer specific for tRNA-Ile[AAU] showed that t6A pathway mutants were specifically depleted of a stalled product at the normally modified position A37, whereas primer extension patterns for control tRNA-Val[UAC] were not substantially altered ([Fig. 3](#LINRNA053934F3)A). Similar assays of third instar larvae bearing various strong *Drosophila kae1* alleles in *trans* to a deficiency also revealed alteration at A37 ([Fig. 3](#LINRNA053934F3)B). However, the loss of modified tRNAs in the total population were notably modest compared to the yeast mutants, since yeast *kae1* mutants retain only a few percent of total t6A ([El Yacoubi et al. 2011](#LINRNA053934C10); [Srinivasan et al. 2011](#LINRNA053934C46)); see also [Figure 4](#LINRNA053934F4).

#### FIGURE 4.

[![FIGURE 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a649/4647464/3bf22f408124/2103F04.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4647464_2103F04.jpg)

[Open in a new tab](figure/LINRNA053934F4/)

Genetic and molecular complementation of yeast t6A mutants with *Drosophila* orthologs. (*A*–*C*) Single complementation tests. The schematic depicts the transformations of cognate yeast genes and their fly orthologs, assayed for complementation of *sua5* (*A*), *kae1* (*B*), and *bud32* (*C*) null mutants. Comparison of single colony sizes indicates substantial growth rescue by fly proteins, although not as robust as the yeast counterparts. (*D,E*) Double complementation tests. The schematics depict combinations of fly orthologs assayed for complementation of *kae1* (*D*) and *bud32* (*E*) null mutants. Improved growth rescue by supplying combinations of fly factors was noticeable, particularly for *Drosophila kae1 + bud32*. (*F*) Primer extension analysis demonstrating molecular rescue of t6A-modified tRNA-Ile by yeast and fly factors. (*G*) Mass spectrometry analysis of total yeast tRNAs. *kae1* mutant yeast has borderline detectable levels of t6A-modified tRNAs, which are restored by complementation with yeast *KAE1* and by *Drosophila* KEOPS genes. Consistent with the behavior in yeast growth (*D*) and in primer extension assays (*F*), the combination of *Drosophila kae1 + bud32* conferred the best rescue of t6A-modified tRNAs.

Inspection of modENCODE data ([Brown et al. 2014](#LINRNA053934C4)) shows that *Drosophila kae1* is ubiquitously expressed but present at highest levels in ovaries and the early blastoderm embryo, indicating its strong maternal deposition. Some maternal gene products can support organismal processes for days ([Lecourtois and Schweisguth 1995](#LINRNA053934C30); [Lu et al. 2000](#LINRNA053934C33)), and tRNAs themselves are typically expected to be relatively stable ([Hanoune and Agarwal 1970](#LINRNA053934C18)), although loss of some tRNA modifications is documented to render tRNAs unstable ([Alexandrov et al. 2006](#LINRNA053934C2)). Because of these considerations, we sought to extend the analysis of *kae1* zygotic mutants as late as possible. Bearing in mind that normal *Drosophila* complete their larval stages by ∼5 d, we collected RNA from 12-d and 18-d *kae1* mutant larvae. However, we did not observe further decreases in the level of t6A-modified tRNAs even at these extended timepoints ([Fig. 3](#LINRNA053934F3)).

To provide a global assessment on the amount of t6A-modified tRNAs in *kae1* mutants, we purified total tRNA, enzymatically hydrolyzed the tRNA to ribonucleosides, and analyzed these using liquid chromatography-coupled triple quadrupole mass spectrometry (LC-MS/MS) using a dynamic MRM method ([Su et al. 2014](#LINRNA053934C47)). This analysis showed a mild (∼10%) decrease of t6A-modified tRNAs in hemizygotes of the weak allele *kae1[7A9]*, and more substantial decreases (∼40%) in hemizygotes of the strong/null alleles *kae1[1A13]* and *kae1[f01978]* ([Fig. 3](#LINRNA053934F3)C). We also observed full restoration of t6A-modified tRNAs in LC-MS/MS analysis of *P[acman]-CG4933/+; kae1[1A13]/Df* larvae, consistent with the viability rescue conferred by the genomic transgene.

Together, these observations provide evidence that *kae1* is required for normal accumulation t6A-modified tRNAs in a metazoan setting. We cannot rule out that the remaining modified tRNAs are exceptionally stable. However, quantitative analysis did not reveal further decrease of t6A-modified tRNAs in *kae1[f01978]/Df* between 6- and 12-d larvae, suggesting that perdurance is not the major underlying cause of the remaining t6A. An alternate scenario is that they are contributed by the mitochondrial pool, which is presumably generated by the mitochondrial Kae1 paralog Qri7 (annotated as *Drosophila* CG14231). We note that Qri7 generates only a minute fraction of total t6A-modified tRNAs in yeast ([Srinivasan et al. 2011](#LINRNA053934C46)), suggesting a potential difference between these eukaryotic species.

### *Drosophila* Sua5/KEOPS genes complement corresponding yeast null mutants

Because the fly null mutants did not fully eliminate t6A, we sought supporting evidence on the role of *Drosophila* Kae1 in this pathway. Therefore, we conducted genetic replacement assays in a simpler heterologous host system, namely, in *S. cerevisiae* mutants in t6A pathway genes. As described, these are slow-growing but can be maintained in the null condition and are nearly completely deficient for t6A ([El Yacoubi et al. 2011](#LINRNA053934C10); [Srinivasan et al. 2011](#LINRNA053934C46)).

To judge the overall relationship of the fly orthologs to their yeast counterparts, we tested for direct complementation of several yeast t6A mutants ([Fig. 3](#LINRNA053934F3)A) by their *Drosophila* orthologs. We cloned the open reading frames for *Drosophila kae1*, *bud32*, and *sua5* into the yeast expression plasmid, pADH416. Because yeast mutants in these three genes grow very poorly ([Srinivasan et al. 2011](#LINRNA053934C46)), we transformed the *Drosophila* expression constructs into the corresponding yeast heterozygous diploid mutants. As controls, the same diploids were transformed with plasmids expressing the yeast protein, and with an empty vector. The transformed diploids were sporulated and then dissected to yield haploid segregants. Haploids containing plasmids expressing the *Drosophila* or yeast protein, or the empty vector, were identified and then streaked to individual colonies on selective medium.

The results of growth after ∼66 h at 30°C are shown in [Figure 4](#LINRNA053934F4)A–C. Colony size is a sensitive measure of growth rate for yeast. *Drosophila sua5*, whose product generates the substrate for the KEOPS complex, yielded growth rescue that was comparable to its yeast ortholog ([Fig. 4](#LINRNA053934F4)A). On the other hand, the two KEOPS components *Drosophila kae1* and *bud32* both conferred substantial rescue, as compared with vector control transformants, but growth was not as robust as with the cognate yeast factors ([Fig. 4](#LINRNA053934F4)B,C). The relative degree of complementation was also apparent on the dissection plates on which the haploid spores germinated. Representative tetrads are shown in Supplemental Figure 3 for the *kae1* and *bud32* heterozygotes. In both cases, colonies identified as containing the *Drosophila* gene were larger than colonies containing vector or no plasmid, but not as large as wild-type colonies.

As the rescues by individual fly KEOPS proteins were less robust than with the cognate yeast factors, we tested if their function could be improved by supplying multiple *Drosophila* factors. This proved to be the case. [Figure 4](#LINRNA053934F4)D shows a *kae1* mutant expressing *Drosophila* Kae1 + Pcc1 (sector 1), *Drosophila* Kae1 + Bud32 (sector 2), or *Drosophila* Kae1 alone (sector 3). Coexpression with either Kae1 partner improved growth rescue. Similarly, coexpression of *Drosophila* Bud32 and Kae1 yielded enhanced rescue of yeast *bud32* ([Fig. 4](#LINRNA053934F4)E, sector 2), relative to *Drosophila* Bud32 alone (sector 3). Notably, expressing *Drosophila* Bud32 and Pcc1 together did not improve complementation of yeast *bud32* relative to Bud32 alone ([Fig. 4](#LINRNA053934F4)E, cf. sectors 1 and 3). This is consistent with the fact that Bud32 and Pcc1 do not interact directly in the complex, whereas both of them physically contact different faces of Kae1 in the KEOPS complex ([Hecker et al. 2008](#LINRNA053934C19); [Mao et al. 2008](#LINRNA053934C35)). The growth improvements in these double complementation tests were also visible on tetrad dissection plates (Supplemental Fig. 3).

### Rescue of t6A-modified tRNAs in yeast *kae1* mutants by *Drosophila* KEOPS factors

We performed biochemical analysis of tRNA modification in mutant and rescued yeast. Primer extensions using a probe against tRNA-Ile[AAU] recapitulated strong loss in the A37 band in *kae1* mutant yeast transformed with an empty vector, which was fully restored by the *S. cerevisiae KAE1* expression construct ([Fig. 4](#LINRNA053934F4)F). Comparison with yeast transformed with various combinations of *Drosophila* KEOPS factors showed strong agreement between growth properties and accumulation of t6A. That is, we observed moderate rescue by fly *kae1* alone, but much stronger improvement by including other members of the *Drosophila* KEOPS complex ([Fig. 4](#LINRNA053934F4)F).

We extended the above analysis of individual tRNAs to the total tRNA population using LC-MS/MS. These assays were strongly correlated with the relative degree of rescue observed in primer extension analysis. In particular, this quantitative method allowed us to distinguish greater restoration of t6A by *Drosophila kae1 + bud32*, relative to *kae1 + pcc1* ([Fig. 4](#LINRNA053934F4)G). In particular, the rescue by *Drosophila kae1 + bud32* was equivalent to the cognate rescue by yeast *kae1*. Finally, we note that LC-MS/MS analysis of yeast samples also detected cyclic t6A (ct6A). It has been reported that ct6A is formed by cyclization of t6A and was observed in diverse domains of life including yeast, but not in any vertebrates tested ([Miyauchi et al. 2013](#LINRNA053934C38)). Consistent with this, we did not detect ct6A above background in the analysis of fly tRNAs (data not shown).

In the yeast profilings, trends in ct6A levels closely mirrored those of t6A ([Fig. 4](#LINRNA053934F4)G). ct6A levels were below the limit of detection in *kae1* mutants, and rescued to similar relative levels as with t6A by the KEOPS expression constructs. However, ct6A peak areas were approximately fivefold smaller than corresponding levels of t6A. Despite efforts to minimize hydrolysis by conducting the analysis at neutral pH, it is possible that a portion of ct6A was hydrolyzed to t6A during sample preparation. In any case, these results clearly demonstrate the biochemical activity of *Drosophila* KEOPS genes including Kae1 in the biogenesis of *N**6*-threonylcarbamoyladenosine-modified tRNAs. Since we did not detect ct6A in *Drosophila* samples, we will continue to generally refer to t6A in the remainder of this study.

### *kae1* mutants exhibit aberrant hematopoietic development and activity

Having established the role of *Drosophila* Kae1 and its partners in t6A biosynthesis, we sought to examine the nature of developmental defects caused by absence of this tRNA modification pathway. We began with the analysis of the namesake “Black Spots,” which resemble phenotypes associated with derangement in the development and/or deployment of the hematopoietic system ([Minakhina and Steward 2006](#LINRNA053934C37)). Therefore, we investigated the status of the lymph gland as well as circulating blood cells in the hemolymph.

We first examined *kae1[1A13]/Df(3L)ED220* larvae, which exhibit a strong molecular defect in t6A biogenesis ([Fig. 3](#LINRNA053934F3)C) and generate “black spots” with nearly complete penetrance ([Fig. 1](#LINRNA053934F1)A). The hemolymph of control TM6B sibling third instar larvae exhibit normal distributions of blood cells ([Fig. 5](#LINRNA053934F5)A), primarily plasmatocytes (identifiable as ∼10 µM diameter cells) and more rare lamellocytes (40–50 µM diameter cells). In contrast, hemolymph smears of ∼10-d-old *kae1[1A13]* hemizygous larvae showed increased density of blood cells, and in particular, massive overrepresentation of lamellocytes ([Fig. 5](#LINRNA053934F5)B). We also identified large circulating aggregates that resembled lymph gland fragments ([Fig. 5](#LINRNA053934F5)C), as well as apparent phalloidin-rich tumors that were packed with lamellocytes ([Fig. 5](#LINRNA053934F5)D). The latter could be melanized and were identifiable as “black spots,” but they were not necessarily pigmented, suggesting they had not yet initiated the melanogenesis cascade.

#### FIGURE 5.

[![FIGURE 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a649/4647464/4edce67f16c0/2103F05.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=4647464_2103F05.jpg)

[Open in a new tab](figure/LINRNA053934F5/)

Hematopoietic overgrowth and blood cell defects in *kae1* mutants. All samples were stained with Hoechst (blue) to label DNA and rhodamine phalloidin (red) to label F-actin; all scale bars, 50 µm. Control animals were TM6B siblings of crosses used to generate *kae1/Df(3L)ED220* larvae and are thus heterozygous for *kae1*. (*A*) Control hemolymph shows the typical profile in which most circulating hemocytes are plasmatocytes ([P] asterisks). (*B*) *kae1[1A13]* hemizygous hemolymph shows increased density of blood cells, as well as numerous lamellocytes, often present in clusters ([L] arrowheads). Lamellocytes (∼40–50 µm in diameter) are larger than plasmatocytes (10 µm in diameter). (*C*) Large aggregates bearing organized structure were observed in circulation as were overt tumors that are packed with plasmatocytes and lamellocytes (*D*). The latter correspond to the melanotic masses observed in live “black spot” larvae. (*E*–*H*) Anterior lobe of lymph gland from third instar in control (*E*) and from aged *kae1/Df(3L)ED220* larvae (*F*–*H*); the dorsal vessel (DV) is labeled. The *kae1* mutants exhibit strong overcommitment to lamellocyte fate in the lymph gland (*F*) and typically have overgrown lobes (*G*). Frequently, the anterior lobe was observed in the process of dispersal or was detached entirely from the dorsal vessel (*H*). (*I*–*L*) Analysis of anterior lymph gland lobes in second instar larvae. Compared with control (*I*), all *kae1* mutants exhibit ectopic lamellocytes (*J*–*L*). The hypomorphs *[1A2]* (*J*) and *[1A13]* (*K*) are relatively normally sized, but the null *[f01978]* (*L*) is strongly overgrown. (*M,N*) Analysis of mitotic cells (labeled with anti-phospho-Histone H3, green) in posterior lobes of lymph glands from control (*M*) and *[1A13]/Df* (*N*). The *kae1* mutant has overgrown posterior lobes that harbor more actively dividing cells.

Notably, lamellocyte overproduction is a frequent feature of other characterized mutants bearing melanotic masses ([Lemaitre et al. 1995](#LINRNA053934C31); [Minakhina and Steward 2006](#LINRNA053934C37)). Therefore, we analyzed the status of the *Drosophila* hematopoietic tissue, the lymph gland. The compact disposition of the anterior lobe in normal third instar lymph gland is shown in [Figure 5](#LINRNA053934F5)E and shows few lamellocytes. In contrast, ∼10-d-old *kae1[1A13]* hemizygous larval lymph glands show abundant maturation of lamellocytes and are typically enlarged ([Fig. 5](#LINRNA053934F5)F,G). Often, we find that the anterior lobe was in the process of dispersing, or completely detached from the dorsal vessel ([Fig. 5](#LINRNA053934F5)H). This correlates with the observation of apparent lymph tissue in the circulatory system ([Fig. 5](#LINRNA053934F5)C). We observed similar phenotypes with the hypomorphic combination *kae1[1A2]*/*Df(3L)ED220*, except that its hematopoietic alterations were weaker and/or slower to develop (Supplemental Fig. 4). That is, defects were less robust at day 8 relative to day 10 larvae.

As mentioned, most *kae1* mutants exhibit a vastly extended larval stage. Thus, we considered it plausible that the enlarged lymph tissues developed progressively over this aberrant larval period. To ask whether *kae1* mutants exhibit defects during the normal course of larval development, we examined *[1A2]* (weak), *[1A13]* (strong), and *[f10978]* (null) *kae1* hemizygotes at the second instar. We observed overrepresentation of lamellocytes in anterior lymph gland lobes in all genotypes, although *[1A2]* was not overgrown and *[1A13]* was only slightly enlarged relative to control ([Fig. 5](#LINRNA053934F5)I–K). However, *[f10978]* exhibited both considerable overgrowth and ectopic commitment to lamellocyte fates ([Fig. 5](#LINRNA053934F5)L). This is despite the fact that *[f10978]* develops modest frequencies of black spot compared with *[1A13]* ([Fig. 1](#LINRNA053934F1)), indicating that hematopoietic tumors can exist without melanization of nodules. We also note that *[f10978]* exhibit lymph gland overgrowth despite the comparably small size of these larvae ([Fig. 1](#LINRNA053934F1)), indicating partial uncoupling of organism size and individual organ size.

To verify their status as “tumors,” we stained second instar lymph glands for phospho-Histone H3 (pH3), a marker of mitotic cells. [Figure 5](#LINRNA053934F5)M,N shows that *kae1[1A13]/Df* mutants also show expansion of their posterior lobes, although not as robust as with the anterior lobe. Nevertheless, these tissues show clear increases in the frequency of anti-pH3-labeled cells, indicating that the expanded hematopoietic tissues are accounted for by increased cell division, as well as alteration in cell fate toward lamellocytes.

### Essential requirement for *kae1* in imaginal discs but not throughout the larval CNS

A conspicuous aspect of the strong *kae1* alleles were their nearly complete absence of imaginal discs. These begin as small cell populations that are set aside in the embryo, and proliferate during larval stages to increase their numbers ∼1000-fold ([Milán et al. 1996](#LINRNA053934C36)), generating the tissues that will form the adult structures during metamorphosis. We mounted larval brain-imaginal disc complexes from wild-type and two strong *kae1* mutants for comparison ([Fig. 6](#LINRNA053934F6)A–C). Discs were completely absent in most larvae, or at best yielded tiny tissue rudiments that might potentially be discs, but cannot confidently be assigned to any particular disc identity. In contrast, the CNS was slightly smaller but remained present. Closer inspection revealed that the optic lobes were strongly reduced in size in *kae1* mutants, whereas the ventral nerve cord (VNC) was less affected ([Fig. 6](#LINRNA053934F6)A–C). Notably, the optic lobes are the main proliferating centers in the larval CNS, whereas the VNC is primarily composed of post-mitotic neurons.

#### FIGURE 6.

![FIGURE 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a649/4647464/20ede217d58f/2103F06.jpg)

[Open in a new tab](figure/LINRNA053934F6/)

Essential requirement for *kae1* in imaginal disc cells. (*A*–*C*) Preparations of third instar brain-disc complexes, stained for DAPI (gray), phalloidin (red) and the post-mitotic neuronal marker Elav (blue). Scale bars, 200 μm. (*A*) Wt complexes show the CNS (in outline) and an array of discs that generate the adult body structures. Because not all the discs remain attached to each other during dissection, a composite image that captures all of the discs is shown. These include pairs of wing (W), haltere (H), and eye-antennal (EA) discs, and six leg (L) discs. In *kae1[1A13]* (*B*) and *kae1[f01978]* (*C*) hemizygotes, essentially no imaginal discs are found, even though the CNS can be readily identified. The CNS can be divided into the optic lobes (OL, dotted lines) and the ventral nerve cord (VNC). The CNS is mildly smaller in *kae1[1A13]/Df* larvae (*B*) and substantially reduced in *kae1[f01978]/Df* larvae (*C*); however, note that the OL is preferentially reduced. The VNC harbors a stronger proportion of post-mitotic neurons than the OLs, as evidenced by Elav staining. (*D,E*) Analysis of mitotic clones generated in wing imaginal discs of control *lacZ* FRT80 (*D*) and *kae1* FRT80 (*E*) chromosomes; enlargements are shown in *D*′ and *E*′. The clones are generated in a heterozygous (+/−) *ub-GFP* FRT80 background, which produces an intermediate level of GFP. Homozygous mutant clones (−/−) lack GFP whereas their sibling twin-spot clones (+/+) express elevated levels of GFP. Seventy-two hours after clone induction, all three cell populations are identified in the control experiment (*D*), whereas homozygous *kae1* mutant cells were not recovered in wing discs (*E*). (*F*) Quantification of relative sizes of clones and their sibling twin-spots. *n* = 26 *lacZ* clones and 20 *kae1* clones analyzed. Scale bars, 200 μm, except *D*′, *E*′, 50 μm.

We further examined the role of *kae1* in proliferating tissues by generating negatively marked somatic clones in wing imaginal discs. In this system, both daughters of a mitotic recombination event are marked and remain distinguished as clones. The homozygous mutant clone is detected by loss of a marker (e.g., *ubi-GFP*) whereas its homozygous wild-type “twinspot” clone expresses the marker more intensely than the surrounding heterozygous tissue. Control *FRT80* clones exhibit similar relative sizes to their twinspots ([Fig. 6](#LINRNA053934F6)D,D′). In contrast, *FRT80 kae1[1A13]* clones were never recovered in imaginal discs, even though their twinspots were sizable ([Fig. 6](#LINRNA053934F6)E,E′). We quantified the differences in clone/twinspot sizes in [Figure 6](#LINRNA053934F6)F. These data imply substantial growth and/or competitive disadvantage of cells lacking the t6A modification pathway, which must have been removed from development after only a few cell divisions at most. We note that somatic clones of *kae1* should still have inherited at least some Kae1 protein, mRNA, and/or t6A-modified tRNAs, yet failed to survive. From this, we infer that perdurance is not a major cause for the ability of *kae1* null zygotic mutants to survive for over two and a half weeks ([Fig. 1](#LINRNA053934F1)).

### Differential requirements of *kae1* in mitotic and nonproliferating larval tissues

The clonal analyses caused us to evaluate more carefully the extent to which different tissues were affected in *kae1* mutants. It is clear that *kae1*-deficient cells exhibit growth defects in some contexts. Nevertheless, our original isolation of *kae1* mutants as extremely long-lived larvae exhibiting hematopoietic expansion ([Fig. 1](#LINRNA053934F1)) indicates that many cells and tissues can not only survive, but seemingly thrive in the *kae1* null state for weeks. For example, differentiated post-mitotic blood cells such as lamellocytes ([Krzemien et al. 2010](#LINRNA053934C28)) are both overrepresented and actively functional in encapsulation during the extended larval phase of *kae1* mutants ([Fig. 5](#LINRNA053934F5)D; Supplemental Fig. 4).

We surveyed other tissues that were maintained in *kae1* mutant larvae, including the gut (Supplemental Fig. 5), salivary glands ([Fig. 7](#LINRNA053934F7)A–F), fat body, and somatic musculature ([Fig. 7](#LINRNA053934F7)G–I). For the most part, all of these tissues are dominantly or solely composed of nonproliferating cells during the larval stages. Therefore, these observations solidify the notion that mitotic cells are preferentially affected by loss of *kae1*. However, this does not mean that defects were lacking in these other larval tissues. Interestingly, quantitative analysis showed tissue-specific and allele-specific differences between their behavior. For example, even though *kae1[1A13]/Df* mutants maintain only marginally more t6A than *kae1[f10978]/Df* mutants ([Fig. 3](#LINRNA053934F3)C), we observed progressively smaller salivary glands in *kae1* mutants with less t6A ([Fig. 7](#LINRNA053934F7)J). More strikingly, the size of salivary gland cells was mostly unaffected in *[1A13]* hemizygotes, whereas they were tiny by comparison in *[f10978]* hemizygotes ([Fig. 7](#LINRNA053934F7)K). Thus, the slight amount of t6A in a strongly hypomorphic *kae1* condition can maintain salivary gland cell size, even though these glands evidently have fewer cells than normal, whereas the null condition compromises both cell and organ size in the salivary gland. In contrast, the absolute size of the body wall musculature is largely unaffected in *kae1[1A13]/Df*, and only slightly reduced in the *kae1[f10978]/Df* mutants ([Fig. 7](#LINRNA053934F7)L,M). Indeed, the small size of *kae1* null larvae ([Fig. 1](#LINRNA053934F1)E) may be sufficient to explain the small size of its muscles, which are expected to scale to body size.

#### FIGURE 7.

![FIGURE 7.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a649/4647464/1bb9258a5e5f/2103F07.jpg)

[Open in a new tab](figure/LINRNA053934F7/)

Differential requirements of *kae1* in mitotic vs. nonproliferating tissues. Larval tissues were analyzed from *kae1[1A13]* and *kae1[f01978]* hemizygotes, which fully lack imaginal discs ([Fig. 6](#LINRNA053934F6)). (*A*–*I*) Immunostaining of dissected tissues, using DAPI (blue) and phalloidin (red). (*A*–*F*) Salivary glands (and some attached fat bodies) imaged at 10× to visualize entire glands (*A*–*C*) and at 20× to emphasize individual cell sizes (*D*–*F*). (*G*–*I*) Body wall musculature. Selected muscles VL1-4 and LO1 are labeled. The length of a body segment is denoted with the double arrow lines. (*J*) Quantifications of salivary gland sizes (*n* = 10 all genotypes) and (*K*) salivary gland cell sizes (*n* = 15 cells from the distal tip measured in each of eight glands, all genotypes). Although *kae1[1A13]/Df* larvae exhibit severe defects in mitotic tissues, the overall salivary gland size and cell size are more modestly affected. The null allele *kae1[f01978]* is severely compromised in both aspects of the salivary gland. (*L*) Quantifications of VL1 muscle length and (*M*) VL2 muscle length. *kae1[1A13]/Df* larvae exhibit normal muscle sizes, while the null allele is only ∼40% reduced. Error bar shows standard error; all statistical analyses were unpaired Student's *t*-tests. (*N*–*P*) Adult eyes bearing Gal4 driver alone (*N*) or expressing *kae1-RNAi* under control of *ey-Gal4* (*O*) or *GMR-Gal4* (*P*). The former is active in the proliferating eye primordium, whereas the latter is active in post-mitotic photoreceptor neurons. (*Q*,*R*) Third instar eye-antennal imaginal discs stained for DAPI, Cut (green, to label cone cells) and Elav (red, to label photoreceptors). Compared to the control disc (*Q*), depletion of *kae1* using *ey-Gal4* induces severely defective eye development. All scale bars, 200 μm.

### Distinct effects of depleting *kae1* in mitotic and post-mitotic contexts of an individual tissue

Our extensive phenotypic analyses reveal strikingly distinct requirements for Kae1 in mitotic and nonproliferating larval tissues. To test for potentially distinct requirements for *kae1* in mitotic and post-mitotic contexts within the same tissue, we turned to an RNAi approach. The developing eye is convenient for this purpose, as characterized drivers can be used to express transgenes in the undifferentiated, mitotic portion of the eye primordium (*ey-Gal4*) or in the post-mitotic photoreceptors (*GMR-Gal4*). Analysis of *ey-Gal4 > UAS-kae1-RNAi* flies showed strongly malformed eyes, which were small and rough ([Fig. 7](#LINRNA053934F7)N,O). We obtained similar results with independent RNAi transgenes (data not shown). In contrast, *GMR-Gal4 > UAS-kae1-RNAi* produced eyes that developed normally, with all transgenes tested ([Fig. 7](#LINRNA053934F7)P).

Cellular analysis demonstrated that depletion of *kae1* using *ey-Gal4* caused developmental defects. Compared to the normal size and differentiation status of normal third instar eye discs, stained for the photoreceptor marker Elav and cone cell marker Cut, *ey-Gal4 > UAS-kae1-RNAi* discs exhibit rudimentary eye discs with disturbed patterning ([Fig. 7](#LINRNA053934F7)Q,R). On the other hand, knockdown with *GMR-Gal4* did not affect larval eye development (data not shown). Therefore, *kae1* is critically required in mitotic, but not in post-mitotic, eye cells.

## DISCUSSION

### Confirmation of the t6A modification pathway in a metazoan

To date, nearly all studies of the t6A biogenesis pathway have been conducted in unicellular organisms such as bacteria or yeast. The first functional study of a metazoan t6A component involved knockdown of the mitochondrial Kae1 paralog Qri7 (OSGEP-L) in *Caenorhabditis elegans*, which induced defective mitochondrial morphology and function ([Oberto et al. 2009](#LINRNA053934C40)). Subsequently, a knockdown study of the *D. melanogaster* Bud32/PRPK reported growth defects relating to defective TOR signaling ([Ibar et al. 2013](#LINRNA053934C23)). However, neither study assessed biochemical defects in t6A accumulation per se. Furthermore, knockdown approaches do not generally achieve genetic null states.

We demonstrate loss of t6A-modified tRNAs in *Drosophila kae1* mutants, using both primer extension of individual species and mass spectrometry to examine the bulk population. Moreover, we demonstrate substantial cross-species rescue by fly Kae1 (and its partners) in yeast *kae1* mutants, both in terms of growth capacity and in recovery of t6A-modified tRNAs. While this work was in preparation, similar observations using the *kae1* transposon insertion alleles were reported by the Glavic group ([Rojas-Benitez et al. 2015](#LINRNA053934C43)). Our findings corroborate and extend their results. Nevertheless, our detection of substantial t6A-modified tRNAs in *Drosophila kae1* mutants was unexpected.

Previous characterization of yeast *kae1* mutants ([El Yacoubi et al. 2011](#LINRNA053934C10); [Srinivasan et al. 2011](#LINRNA053934C46)), which we confirm here, showed nearly complete absence of t6A-modified tRNAs. This raises two general possibilities. First, it may be that t6A-modified tRNAs are exceptionally stable in *Drosophila*. In this scenario, even though we assayed timepoints going well past the established range of maternal effects (e.g., up to 18 d, or the equivalent of a week-old adult fly), those modified tRNAs that were maternally deposited or generated from maternal *kae1* mRNA and/or protein persisted all that time. Alternatively, many species, including yeast and *Drosophila*, express a mitochondrial Kae1 paralog named Qri7. Although mitochondrially derived t6A appears to comprise a minute portion of the total pool in single-celled organisms, the situation might be different in a metazoan. For example, the substantial populations of nonproliferating tissues that tolerate deletion of *kae1* include the larval musculature, which sustain larval locomotion in mutants for weeks. Therefore, the remaining t6A-modified tRNAs in *kae1* null mutant larvae may potentially be contributed by the abundant mitochondria in muscles. In the future, characterization of a *Drosophila Qri7* (*CG14231*) knockout would provide a useful comparison, especially to see if Qri7 loss preferentially affects tissues with exceptional needs for oxidative phosphorylation.

### An extensive allelic series of *kae1* mutants reveal diverse biology of t6A

The first allele in a t6A biogenesis component, yeast *sua5-1*, was isolated in a forward genetic screen as an extragenic suppressor of a mutant *cyc1* allele bearing an aberrant uORF ([Hampsey et al. 1991](#LINRNA053934C16)). UV and X-rays were used as mutagens in these screens, and cloning of *sua5-1* revealed a single-base substitution resulting in an S107F missense alteration ([Na et al. 1992](#LINRNA053934C39)). Subsequently, other genetic efforts hit various KEOPS components of the t6A pathway, but these studies utilized screening of a deletion collection ([Downey et al. 2006](#LINRNA053934C8)) or a multicopy suppressor scheme ([Kisseleva-Romanova et al. 2006](#LINRNA053934C27)). All of these studies pre-dated the connection of Sua5/KEOPS factors to t6A biogenesis, and for the most part, studies have focused on generation of straight or conditional knockout alleles ([Arigoni et al. 1998](#LINRNA053934C3); [El Yacoubi et al. 2009](#LINRNA053934C9), [2011](#LINRNA053934C10); [Handford et al. 2009](#LINRNA053934C17); [Srinivasan et al. 2011](#LINRNA053934C46)).

We describe the first extensive set of point mutations in a t6A pathway component. These mutations sort along a comprehensive allelic series that includes null alleles, strong mutants, and weak hypomorphs. We document allele-specific phenotypic severities in different tissues, and also note that biochemically strong hypomorphs exhibit modest phenotypes in some settings (e.g., exhibit normal somatic musculature) but phenocopy null alleles in other settings (e.g., completely lack imaginal discs). These discrepancies highlight the utility of the allelic series in dissecting multiple requirements of essential housekeeping genes in regulating fundamental cellular processes in vivo. Future studies can continue to exploit these alleles to elucidate the biology of t6A modification. For example, it remains unclear what underlying causes provoke the dramatic hematopoietic responses in *kae1* mutants, and our alleles provide unique means for querying this. Of note, substantial hematopoietic overgrowth is observed in both strong hypomorphs and the null condition, but only the former proceeds reliably toward generation of melanotic nodules.

The *kae1* alleles we report should also be valuable for understanding the mechanism of t6A modification. Extensive structural studies delineate the Kae1 active site and binding surfaces with its partners Pcc1 and Bud32 ([Hecker et al. 2008](#LINRNA053934C19); [Mao et al. 2008](#LINRNA053934C35); [Wan et al. 2013](#LINRNA053934C49); [Zhang et al. 2015](#LINRNA053934C55)). Interestingly, only one of our many point mutations directly affects any of these functionally characterized regions ([Fig. 2](#LINRNA053934F2)A). This highlights the usage of a forward genetic screen to identify relevant mutations. This mutant collection can be utilized as a basis of structure-function studies to assess how these uncharacterized residues might affect formation of the KEOPS complex, productive t6A transfer, or potentially the specificity of substrate tRNA recognition.

### Differential requirements of t6A in proliferating and nonproliferating cells

Studies of the biology of t6A modification pathway in metazoan systems are only in their infancy. The Glavic group reported foundational studies using transgenic RNAi against *Drosophila Prpk/Bud32* ([Ibar et al. 2013](#LINRNA053934C23)), and their recent work ([Rojas-Benitez et al. 2015](#LINRNA053934C43)) and our current study extend this to genuine *kae1* mutants. There are general similarities between the phenotypes of fruit fly and unicellular organism mutants, in that these genes are lethal (as in bacteria) and exhibit substantially defective growth (as in yeast). Of course, one of the major opportunities specifically available in higher eukaryotes is to examine potential cell- and tissue-specific requirements. Here, we find evidence for differential requirements for *kae1* in distinct cell types.

Quite noticeably, strong *kae1* mutants completely lack imaginal discs. The progenitor populations of discs are set aside during embryonic development, and they proliferate extensively during larval stages to generate the tissues that will form the adult body plan during metamorphosis. Although we cannot precisely assess the timing of effective depletion of maternal stores of t6A-modifed tRNAs (i.e., that are directly deposited into the oocyte, or that are generated via maternally deposited *kae1* mRNA and/or protein), our clonal studies demonstrate a timely impact on removal of the *kae1* genomic locus. That is, even though there exists substantial likelihood of Kae1 and t6A-modified tRNAs inherited from the mother cell prior to mitotic recombination, *kae1* mutant disc cells are quickly disadvantaged and rapidly removed during the course of normal development.

Overall, we note these *kae1* phenotypes are strikingly similar to certain other *Drosophila* mutants. For example, we and others previously characterized mutants of the central E1 and E2 enzymes in the SUMO pathway (the heterodimeric pair Aos1/Uba2 and Ubc9, respectively), and these develop a range of defects that are strikingly similar to what we documented for *kae1* mutants. Phenotypes common to loss of Kae1 and SUMO include extended larval phases before lethality, presence of melanotic tumors, and preferential defects in mitotic versus nonproliferating cells, as evidenced by their lack of imaginal discs while retaining diverse other larval tissues ([Kalamarz et al. 2012](#LINRNA053934C24); [Kanakousaki and Gibson 2012](#LINRNA053934C25)). Similarly, a number of zygotic cell cycle mutants specifically abrogate development of imaginal discs while generally sparing other endoreplicated and post-mitotic larval tissues ([Gatti and Baker 1989](#LINRNA053934C12)). Altogether, these analyses highlight how mutations in seemingly “general” cellular regulators can reveal tissue-specific requirements. Moreover, they suggest that several fundamental regulatory pathways are preferentially required in highly mitotic tissues. Whether this simply indicates that diverse processes mutate to a common terminal zygotic phenotype, reflecting the distinct biology of mitotic and nonproliferating cells, or whether there are actually mechanistic links between t6A, SUMO, and the cell cycle, remains to be dissected in the future.

## MATERIALS AND METHODS

### Genetic screen for *kae1* alleles

The region uncovered by *Df(3L)ED220* was subjected to a lethal F2 screen after EMS mutagenesis. In total, we conducted nine rounds of mutagenesis using isogenized *w[1118]* males (rounds 1 and 2) or *w ; FRT80B* males (subsequent rounds). The animals were desiccated and fed 25mM EMS in 20% sucrose, then mated with *w; TM3 Sb/TM6B* females. Individual F1 *TM6B* males were crossed to *w; Df(3L)ED220 w+/TM6B* females (Bloomington #8077). Vials were scored for lethality by the absence of empty non-Tb pupal cases. Owing to the *w*+ marker integral to *Df(3L)ED220*, stocks of new lethal alleles were recovered by collecting white-eyed flies carrying TM6B, and re-balancing. In total, we conducted nine rounds of mutagenesis, and scored >7000 viable crosses to the deficiency, yielding 64 lethals.

The kae1 alleles [1A2], [1A13], and [1A19] were recovered in the w[1118] background, while [3A7], [5C3], [7A6], [7A9], and [9D1], FRT80B/TM6B were recovered in the *w[1118]*; *FRT80B* background. We subsequently identified other alleles of *kae1* in stocks obtained from the Bloomington Stock Center, namely *pBac-CG4933[f01978]/TM6B* (#18497), *l(3)72Fb[209]/TM6B* (#4793), and *l(3)72Fb[331]/TM6B* (#4794).

### Construction of *CG4933* rescue transgenes

We amplified the genomic region encompassing *CG4933* and part of the upstream flanking gene using the primers below. The fragment was cloned into pGEM-T-easy and sequence verified. The fragment was then excised with NotI and cloned into NotI digested attB-P[acman] ([Venken et al. 2006](#LINRNA053934C48)), which had been treated with calf intestinal phosphatase.

* 4925AR TAAGCAATTATTTGATAGTCAATCTTTCG
* CG4933R AAGGAACAGGCCCTTTAAGTGGG

We generated GFP-tagged *CG4933* genomic transgene using recombineering, utilizing a two-step positive/negative selection ([Warming et al. 2005](#LINRNA053934C50)). We performed positive/negative galK selection mediated by growth on MacConkey agar (positive selection for integration of the galK cassette) and growth on media containing 2-deoxygalactose (negative selection for replacement of galK cassette with GFP[S65T] CDS). Sequences (galK cassette and GFP[S65T] CDS) were amplified with primers described below from pgalK and pBD1010 (Barry Dickson), respectively. Purified PCR products were electroporated into SW102 bacteria ([Warming et al. 2005](#LINRNA053934C50)) with the appropriate attB p[acman] CG4933 vector and selection was carried out as described previously ([Venken et al. 2006](#LINRNA053934C48)).

Primers for galK recombineering of GFP into C-terminus of CG4933:

* C term - galK insertion
* CT4933galK-F
* GTGACTCAGCGATTTCGAACGGACGAAGTGCTGGTCAGCTGGCGGGATGACCCTGTTGACAATTAATCATCGGCA
* CT4933galK-R
* TAGTTATTTAATAAACAAACGATGCACAGAAATAAGTTCTTGTGGTGATTATCAGCACTGTCCTGCTCCTT
* C term - GFP insertion
* CT4933GFP-F
* GTGACTCAGCGATTTCGAACGGACGAAGTGCTGGTCAGCTGGCGGGATGACATGAGTAAAGGAGAAGAACTTTTC
* CT4933GFP-R
* TAGTTATTTAATAAACAAACGATGCACAGAAATAAGTTCTTGTGGTGATTATTTGTATAGTTCATCCATGCC

### Immunostaining and clonal analysis

#### Analysis of hematopoietic system

Developmentally synchronized larvae from crosses *w; kae1[IA2]/TM6B* X *Df(3L)ED220/TM6B* and *w; kae1[IA13]/TM6B* X *Df(3L)ED220/TM6B* were collected, washed in phosphate buffered saline, pH 7.4, and either imaged without dissection using a Leica MZFLIII, or were dissected for hemolymph contents, lymph glands, and tumors, at 4, 8, and 10 d after egg laying, according to previously described methods ([Small et al. 2012](#LINRNA053934C44)). To obtain larval blood cells and tumors, animals were bled gently, and hemolymph contents, including tumors, were recovered directly on slides. Larval samples were fixed in 4% paraformaldehyde in PBS, pH 7.4, stained with Rhodamine Phalloidin (Invitrogen) and Hoechst 33258 (Molecular Probes), and imaged with Zeiss laser Scanning 510 confocal microscope. Images were recorded at 8 bit. AxioVision LE 4.5 or Zeiss LSM5 AxioVision software was used to process images.

#### Analysis of other larval tissues

Imaginal discs, brains, and muscles were dissected and stained following previously described protocol ([Lai and Rubin 2001](#LINRNA053934C29)). Antibodies used were mouse anti-Cut (DSHB, 1:50), rat anti-Elav (DSHB, 1:500), Alexa Fluor-568 phalloidin (Life Technologies, 1:1000). Appropriate secondary antibodies conjugated to Alexa 488, 568, and 647 were then used. Tissues were mounted in VECTASHIELD Antifade Mounting Medium with DAPI.

#### Clonal analysis

We analyzed *hsFlp; [w+, arm-lacZ] FRT80B/[w+, ubi-GFP] FRT80B* and *hsFlp; kae1[1A13] FRT80B/[w+, ubi-GFP] FRT80B* animals. Larvae were heat-shocked at 37°C for 40 min at 72 h after egg laying, and were dissected 3 d after heat shock.

### RT-PCR

Total RNA from homozygous or hemizygous larvae were extracted with TRIzol (Invitrogen), followed by DNase treatment using TURBO DNA-free Kit (Ambion). cDNA were synthesized using SuperScript III First-Strand Synthesis System with random primers (Invitrogen). QPCR reaction was performed with CFX96 Touch Real-Time PCR Detection System (Bio-Rad). Primers used were Kae1\_RT\_F, TTTGGGTATTGAAGGCAGTG, and Kae1\_RT\_R, CAGGAAACCTTCTCCTGGTG, and RPL32\_RT\_F, CCGCTTCAAGGGACAGTATC, and RPL32\_RT\_R, ACGTTGTGCACCAGGAACTT.

### tRNA primer extension assays

Two micrograms of tRNA extracted with mirVana miRNA isolation kit (Ambion) was mixed with 1 pmol of [γ-32P]-labeled primer in 5 μL 1× First-strand buffer (Invitrogen), heated to 95°C and then slowly cooled down to 37°C. Annealed RNA/primer was collected by brief centrifugation, added with 5 μL retro-transcription reaction mix, including 0.5 mM dNTPs, 0.01 M DTT, 200 units SuperScript III (Invitrogen) in 1× First-strand buffer and incubated at 50°C for 50 min, followed by heat inactivation at 70°C for 15 min. Reaction samples were then electrophoresed with 15% polyacrylamide–urea sequencing gels (40 cm long).

* *S.c.* Ile primer: gggatcgaaccgctgatc
* *S.c.* Met primer: aggttcgaactctcgacc
* *S.c.* Val primer: aggttcgaactcgggatc
* *D.m.* Ile primer: gggatcgaacccgcgac
* *D.m.* Met primer: aggtttcgatcctcggacc
* *D.m.* Val Primer: gggatcgaaccggagacc

### Yeast strains and plasmids

All yeast strains are derivatives of W303 **a**/α diploids homozygous for *trp1 leu2 ura3 his3 ade2*. The following derivatives of W303 were used: EJP51 *KAE1*/*kae1*::*kanMX6*, RS2586 *BUD32*/*bud32*::*kanMX6*, and YYF1 SUA5/*sua5*::*kanMX6*. PCR was used to amplify the *Drosophila* cDNAs for *KAE1*, *BUD32*, *PCC1*, and *SUA5* open reading frames*.* The cDNAs were cloned into a yeast expression plasmid vector, pADH416 or pADH415, with selectable markers *URA3* or *LEU2*, respectively. These plasmids express the *Drosophila* proteins from the *ADH* promoter on a single copy plasmid. As controls, the corresponding yeast genes were expressed under their own promoters in the single copy vector pRS316. The plasmid expressing yeast Kae1 was described previously ([Srinivasan et al. 2011](#LINRNA053934C46)) and yeast Pcc1, Bud32, and Sua5 expressing plasmids are pXZ1, pXZ2, and pXZ3, respectively.

### Mass spectrometric analysis of tRNA modifications

#### Chemicals

All chemicals and reagents used were obtained at the highest purity available and were used without further purification unless stated. Benzonase, bacterial alkaline phosphatase, butylated hydroxytoluene, acetonitrile, and buffer salts were purchased from Sigma-Aldrich. Coformycin was obtained from the National Cancer Institute. Phosphodiesterase I was purchased from Worthington. Tetrahydrouridine was purchased from Calbiochem. Water purified through a Milli-Q system (Millipore) was used throughout our studies. Sartorius Vivaspin 500-brand centrifugal filter units were used for dialysis and sample concentration.

#### Extraction and purification of *Drosophila* and yeast RNA

Total RNAs were isolated using TRIzol and then subjected to a two-step purification to obtain tRNA. First, a PureLink RNA Mini extraction kit (Life Technologies) was used to precipitate the ribosomal RNA in the sample. In the second step, size-exclusion chromatography (SEC-HPLC) was performed using an Agilent Bio SEC-3 column with 10 mM ammonium acetate at pH 7 as the mobile phase. Neutral pH was selected to minimize hydrolysis of ct6A. tRNA and rRNA were collected as separate fractions, vacuum concentrated and reconstituted in water. tRNA concentrations were determined by UV spectroscopy at 260 nm. In the yeast samples, the proportion of tRNA to ribosomal RNA was much higher so only SEC-HPLC fractionation was required to obtain pure tRNA for analysis.

#### Nucleoside preparation

Four micrograms of purified tRNA from each sample was digested using a mixture of Benzonase (21.5 U), bacterial alkaline phosphatase (0.5 U) and phosphodiesterase I (0.21 U) in a final reaction volume of 100 µL. Enzymes were dialyzed into 10 mM ammonium acetate at pH 7 immediately prior to use by washing twice with 10 kDa MWCO Vivaspin 500 spin filters. The reaction mixture was supplemented with MgCl2, CaCl2, and ZnCl2, each at a concentration of 0.1 mM. Nucleobase deaminase inhibitors coformycin and tetrahydrouridine were added at a concentration of 10 µg/mL and 50 µg/mL, respectively, and butylated hydroxytoluene (an antioxidant) was added at a concentration of 0.5 mM. The digestion was allowed to proceed for 2 h at 37°C and was stopped upon removal of the enzymes by microfiltration with 10 kDa spin filters.

#### LC-MS/MS analysis of RNA nucleosides

Ribonucleosides were separated using a Synergy Fusion RP, 2.5 µm particle size, 100 Å pore size, 100 mm length, 2 mm inner diameter from Phenomenex, on an Agilent 1290 series HPLC system equipped with a diode array detector. Mobile phase A was 5 mM ammonium acetate and mobile phase B was pure acetonitrile. Gradient elution started with 100% A for 1 min, increased to 10% B after 10 min, 40% after 14 min, 80% after 20 min and regeneration of starting conditions with 100% A for 5 additional minutes. The flow rate was 0.35 mL/min and the column temperature 35°C. The effluent from the column was directed through the DAD before entering the Agilent 6430 triple quadrupole mass spectrometer in dynamic multiple reaction monitoring (MRM) mode. The MS was operated in positive ion mode with the following parameters: electro-spray ionization (ESI-MS), fragmentor voltage (average) 80 V, cell accelerator voltage 2 V, N2-gas temperature 350°C, N2-gas flow 10 L/min, nebulizer 40 psi, capillary 3500 V.

Modified nucleosides were identified based on having both the correct retention time and mass transition. Under our chromatography conditions, t6A eluted at 8.6 min while ct6A eluted at 6.8 min. Mass transitions were *m/z* 413 → 281 for t6A and *m/z* 395 → 263 for ct6A, both corresponding to the neutral loss of ribose. 15N-2-Deoxyadenosine was added to each sample as an internal standard.

#### Data analysis

In each sample, the UV peak areas at 260 nm of the four canonical ribonucleosides—cytidine, uridine, guanosine, and adenosine—were integrated and summed together. Peaks corresponding to detection of MRM transitions for the modified nucleosides were first normalized to the peak area of the internal standard (15N-2-deoxyadenosine) to account for inter-sample detection fluctuations, followed by a second normalization to the summed peak areas of the canonical nucleosides to account for the amount of injected RNA. The adjusted peak areas of the five samples were calculated as a percentage of the corresponding wild-type control sample.

## SUPPLEMENTAL MATERIAL

Supplemental material is available for this article.

## Supplementary Material

Supplemental Material

[supp\_21\_12\_2103\_\_index.html](/articles/instance/4647464/bin/supp_21_12_2103__index.html) (1,001B, html)

## ACKNOWLEDGMENTS

We thank Fuqu Hu for Kae1 structural comparisons and the Bloomington Stock Center for stocks. Work in the Govind laboratory was supported by NSF-1121817, NASA award NNX15AB42G, and a Howard and Vicki Palefsky Fellowship to J.R. Work in the Dedon laboratory was supported by the Singapore National Research Foundation under its Singapore-MIT Alliance for Research and Technology, the National Institute of Environmental Health Science (ES017010, ES022858, ES002109), the National Science Foundation (MCB-1412379), and a fellowship from the Deutsche Forschungsgemeinschaft (S.M.K.). Work in the Sternglanz laboratory was supported by P01-GM088297. Work in the Lai laboratory was supported by the Burroughs Wellcome Foundation, the National Institutes of Health (R01-GM083300), the National Institute of Neurological Disorders and Stroke (R01-NS083833), and Memorial Sloan-Kettering Core grant P30-CA008748.

## Footnotes

Article published online ahead of print. Article and publication date are at <http://www.rnajournal.org/cgi/doi/10.1261/rna.053934.115>.

## REFERENCES

1. Agris PF. 2008.
   Bringing order to translation: the contributions of transfer RNA anticodon-domain modifications. EMBO Rep
   9: 629–635. [[DOI](https://doi.org/10.1038/embor.2008.104)] [[PMC free article](/articles/PMC2475317/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18552770/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20Rep&title=Bringing%20order%20to%20translation:%20the%20contributions%20of%20transfer%20RNA%20anticodon-domain%20modifications&author=PF%20Agris&volume=9&publication_year=2008&pages=629-635&pmid=18552770&doi=10.1038/embor.2008.104&)]
2. Alexandrov A, Chernyakov I, Gu W, Hiley SL, Hughes TR, Grayhack EJ, Phizicky EM. 2006.
   Rapid tRNA decay can result from lack of nonessential modifications. Mol Cell
   21: 87–96. [[DOI](https://doi.org/10.1016/j.molcel.2005.10.036)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16387656/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell&title=Rapid%20tRNA%20decay%20can%20result%20from%20lack%20of%20nonessential%20modifications&author=A%20Alexandrov&author=I%20Chernyakov&author=W%20Gu&author=SL%20Hiley&author=TR%20Hughes&volume=21&publication_year=2006&pages=87-96&pmid=16387656&doi=10.1016/j.molcel.2005.10.036&)]
3. Arigoni F, Talabot F, Peitsch M, Edgerton MD, Meldrum E, Allet E, Fish R, Jamotte T, Curchod ML, Loferer H. 1998.
   A genome-based approach for the identification of essential bacterial genes. Nat Biotechnol
   16: 851–856. [[DOI](https://doi.org/10.1038/nbt0998-851)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/9743119/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=A%20genome-based%20approach%20for%20the%20identification%20of%20essential%20bacterial%20genes&author=F%20Arigoni&author=F%20Talabot&author=M%20Peitsch&author=MD%20Edgerton&author=E%20Meldrum&volume=16&publication_year=1998&pages=851-856&pmid=9743119&doi=10.1038/nbt0998-851&)]
4. Brown JB, Boley N, Eisman R, May G, Stoiber M, Duff MO, Booth BW, Wen J, Park S, Suzuki AM, et al.
   2014.
   Diversity and dynamics of the *Drosophila* transcriptome. Nature
   512: 393–399. [[DOI](https://doi.org/10.1038/nature12962)] [[PMC free article](/articles/PMC4152413/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24670639/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Diversity%20and%20dynamics%20of%20the%20Drosophila%20transcriptome&author=JB%20Brown&author=N%20Boley&author=R%20Eisman&author=G%20May&author=M%20Stoiber&volume=512&publication_year=2014&pages=393-399&pmid=24670639&doi=10.1038/nature12962&)]
5. Daugeron MC, Lenstra TL, Frizzarin M, El Yacoubi B, Liu X, Baudin-Baillieu A, Lijnzaad P, Decourty L, Saveanu C, Jacquier A, et al.
   2011.
   Gcn4 misregulation reveals a direct role for the evolutionary conserved EKC/KEOPS in the t6A modification of tRNAs. Nucleic Acids Res
   39: 6148–6160. [[DOI](https://doi.org/10.1093/nar/gkr178)] [[PMC free article](/articles/PMC3152333/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21459853/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Gcn4%20misregulation%20reveals%20a%20direct%20role%20for%20the%20evolutionary%20conserved%20EKC/KEOPS%20in%20the%20t6A%20modification%20of%20tRNAs&author=MC%20Daugeron&author=TL%20Lenstra&author=M%20Frizzarin&author=B%20El%20Yacoubi&author=X%20Liu&volume=39&publication_year=2011&pages=6148-6160&pmid=21459853&doi=10.1093/nar/gkr178&)]
6. Deutsch C, El Yacoubi B, de Crécy-Lagard V, Iwata-Reuyl D. 2012.
   Biosynthesis of threonylcarbamoyl adenosine (t6A), a universal tRNA nucleoside. J Biol Chem
   287: 13666–13673. [[DOI](https://doi.org/10.1074/jbc.M112.344028)] [[PMC free article](/articles/PMC3340167/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22378793/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Biosynthesis%20of%20threonylcarbamoyl%20adenosine%20(t6A),%20a%20universal%20tRNA%20nucleoside&author=C%20Deutsch&author=B%20El%20Yacoubi&author=V%20de%20Cr%C3%A9cy-Lagard&author=D%20Iwata-Reuyl&volume=287&publication_year=2012&pages=13666-13673&pmid=22378793&doi=10.1074/jbc.M112.344028&)]
7. dos Santos G, Schroeder AJ, Goodman JL, Strelets VB, Crosby MA, Thurmond J, Emmert DB, Gelbart WM, FlyBase Consortium. 2015.
   FlyBase: introduction of the *Drosophila melanogaster* Release 6 reference genome assembly and large-scale migration of genome annotations. Nucleic Acids Res
   43: D690–D697. [[DOI](https://doi.org/10.1093/nar/gku1099)] [[PMC free article](/articles/PMC4383921/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25398896/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=FlyBase:%20introduction%20of%20the%20Drosophila%20melanogaster%20Release%206%20reference%20genome%20assembly%20and%20large-scale%20migration%20of%20genome%20annotations&author=G%20dos%20Santos&author=AJ%20Schroeder&author=JL%20Goodman&author=VB%20Strelets&author=MA%20Crosby&volume=43&publication_year=2015&pages=D690-D697&pmid=25398896&doi=10.1093/nar/gku1099&)]
8. Downey M, Houlsworth R, Maringele L, Rollie A, Brehme M, Galicia S, Guillard S, Partington M, Zubko MK, Krogan NJ, et al.
   2006.
   A genome-wide screen identifies the evolutionarily conserved KEOPS complex as a telomere regulator. Cell
   124: 1155–1168. [[DOI](https://doi.org/10.1016/j.cell.2005.12.044)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16564010/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=A%20genome-wide%20screen%20identifies%20the%20evolutionarily%20conserved%20KEOPS%20complex%20as%20a%20telomere%20regulator&author=M%20Downey&author=R%20Houlsworth&author=L%20Maringele&author=A%20Rollie&author=M%20Brehme&volume=124&publication_year=2006&pages=1155-1168&pmid=16564010&doi=10.1016/j.cell.2005.12.044&)]
9. El Yacoubi B, Lyons B, Cruz Y, Reddy R, Nordin B, Agnelli F, Williamson JR, Schimmel P, Swairjo MA, de Crécy-Lagard V. 2009.
   The universal YrdC/Sua5 family is required for the formation of threonylcarbamoyladenosine in tRNA. Nucleic Acids Res
   37: 2894–2909. [[DOI](https://doi.org/10.1093/nar/gkp152)] [[PMC free article](/articles/PMC2685093/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19287007/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20universal%20YrdC/Sua5%20family%20is%20required%20for%20the%20formation%20of%20threonylcarbamoyladenosine%20in%20tRNA&author=B%20El%20Yacoubi&author=B%20Lyons&author=Y%20Cruz&author=R%20Reddy&author=B%20Nordin&volume=37&publication_year=2009&pages=2894-2909&pmid=19287007&doi=10.1093/nar/gkp152&)]
10. El Yacoubi B, Hatin I, Deutsch C, Kahveci T, Rousset JP, Iwata-Reuyl D, Murzin AG, de Crécy-Lagard V. 2011.
    A role for the universal Kae1/Qri7/YgjD (COG0533) family in tRNA modification. EMBO J
    30: 882–893. [[DOI](https://doi.org/10.1038/emboj.2010.363)] [[PMC free article](/articles/PMC3049207/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21285948/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=A%20role%20for%20the%20universal%20Kae1/Qri7/YgjD%20(COG0533)%20family%20in%20tRNA%20modification&author=B%20El%20Yacoubi&author=I%20Hatin&author=C%20Deutsch&author=T%20Kahveci&author=JP%20Rousset&volume=30&publication_year=2011&pages=882-893&pmid=21285948&doi=10.1038/emboj.2010.363&)]
11. Galperin MY, Koonin EV. 2004.
    ‘Conserved hypothetical’ proteins: prioritization of targets for experimental study. Nucleic Acids Res
    32: 5452–5463. [[DOI](https://doi.org/10.1093/nar/gkh885)] [[PMC free article](/articles/PMC524295/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15479782/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=%E2%80%98Conserved%20hypothetical%E2%80%99%20proteins:%20prioritization%20of%20targets%20for%20experimental%20study&author=MY%20Galperin&author=EV%20Koonin&volume=32&publication_year=2004&pages=5452-5463&pmid=15479782&doi=10.1093/nar/gkh885&)]
12. Gatti M, Baker BS. 1989.
    Genes controlling essential cell-cycle functions in *Drosophila melanogaster*. Genes Dev
    3: 438–453. [[DOI](https://doi.org/10.1101/gad.3.4.438)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/2498166/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&title=Genes%20controlling%20essential%20cell-cycle%20functions%20in%20Drosophila%20melanogaster&author=M%20Gatti&author=BS%20Baker&volume=3&publication_year=1989&pages=438-453&pmid=2498166&doi=10.1101/gad.3.4.438&)]
13. Grewal SS. 2015.
    Why should cancer biologists care about tRNAs? tRNA synthesis, mRNA translation and the control of growth. Biochim Biophys Acta
    1849: 898–907. [[DOI](https://doi.org/10.1016/j.bbagrm.2014.12.005)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25497380/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Why%20should%20cancer%20biologists%20care%20about%20tRNAs?%20tRNA%20synthesis,%20mRNA%20translation%20and%20the%20control%20of%20growth&author=SS%20Grewal&volume=1849&publication_year=2015&pages=898-907&pmid=25497380&doi=10.1016/j.bbagrm.2014.12.005&)]
14. Grosjean H, Sprinzl M, Steinberg S. 1995.
    Posttranscriptionally modified nucleosides in transfer RNA: their locations and frequencies. Biochimie
    77: 139–141. [[DOI](https://doi.org/10.1016/0300-9084%2896%2988117-x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/7541252/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochimie&title=Posttranscriptionally%20modified%20nucleosides%20in%20transfer%20RNA:%20their%20locations%20and%20frequencies&author=H%20Grosjean&author=M%20Sprinzl&author=S%20Steinberg&volume=77&publication_year=1995&pages=139-141&pmid=7541252&doi=10.1016/0300-9084(96)88117-x&)]
15. Gu C, Begley TJ, Dedon PC. 2014.
    tRNA modifications regulate translation during cellular stress. FEBS Lett
    588: 4287–4296. [[DOI](https://doi.org/10.1016/j.febslet.2014.09.038)] [[PMC free article](/articles/PMC4403629/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25304425/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&title=tRNA%20modifications%20regulate%20translation%20during%20cellular%20stress&author=C%20Gu&author=TJ%20Begley&author=PC%20Dedon&volume=588&publication_year=2014&pages=4287-4296&pmid=25304425&doi=10.1016/j.febslet.2014.09.038&)]
16. Hampsey M, Na JG, Pinto I, Ware DE, Berroteran RW. 1991.
    Extragenic suppressors of a translation initiation defect in the cyc1 gene of *Saccharomyces cerevisiae*. Biochimie
    73: 1445–1455. [[DOI](https://doi.org/10.1016/0300-9084%2891%2990177-3)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/1666843/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochimie&title=Extragenic%20suppressors%20of%20a%20translation%20initiation%20defect%20in%20the%20cyc1%20gene%20of%20Saccharomyces%20cerevisiae&author=M%20Hampsey&author=JG%20Na&author=I%20Pinto&author=DE%20Ware&author=RW%20Berroteran&volume=73&publication_year=1991&pages=1445-1455&pmid=1666843&doi=10.1016/0300-9084(91)90177-3&)]
17. Handford JI, Ize B, Buchanan G, Butland GP, Greenblatt J, Emili A, Palmer T. 2009.
    Conserved network of proteins essential for bacterial viability. J Bacteriol
    191: 4732–4749. [[DOI](https://doi.org/10.1128/JB.00136-09)] [[PMC free article](/articles/PMC2715707/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19376873/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&title=Conserved%20network%20of%20proteins%20essential%20for%20bacterial%20viability&author=JI%20Handford&author=B%20Ize&author=G%20Buchanan&author=GP%20Butland&author=J%20Greenblatt&volume=191&publication_year=2009&pages=4732-4749&pmid=19376873&doi=10.1128/JB.00136-09&)]
18. Hanoune J, Agarwal MK. 1970.
    Studies on the half life time of rat liver transfer RNA species. FEBS Lett
    11: 78–80. [[DOI](https://doi.org/10.1016/0014-5793%2870%2980496-3)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11945454/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&title=Studies%20on%20the%20half%20life%20time%20of%20rat%20liver%20transfer%20RNA%20species&author=J%20Hanoune&author=MK%20Agarwal&volume=11&publication_year=1970&pages=78-80&pmid=11945454&doi=10.1016/0014-5793(70)80496-3&)]
19. Hecker A, Lopreiato R, Graille M, Collinet B, Forterre P, Libri D, van Tilbeurgh H. 2008.
    Structure of the archaeal Kae1/Bud32 fusion protein MJ1130: a model for the eukaryotic EKC/KEOPS subcomplex. EMBO J
    27: 2340–2351. [[DOI](https://doi.org/10.1038/emboj.2008.157)] [[PMC free article](/articles/PMC2529370/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19172740/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Structure%20of%20the%20archaeal%20Kae1/Bud32%20fusion%20protein%20MJ1130:%20a%20model%20for%20the%20eukaryotic%20EKC/KEOPS%20subcomplex&author=A%20Hecker&author=R%20Lopreiato&author=M%20Graille&author=B%20Collinet&author=P%20Forterre&volume=27&publication_year=2008&pages=2340-2351&pmid=19172740&doi=10.1038/emboj.2008.157&)]
20. Hinnebusch AG. 2005.
    Translational regulation of GCN4 and the general amino acid control of yeast. Annu Rev Microbiol
    59: 407–450. [[DOI](https://doi.org/10.1146/annurev.micro.59.031805.133833)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16153175/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Microbiol&title=Translational%20regulation%20of%20GCN4%20and%20the%20general%20amino%20acid%20control%20of%20yeast&author=AG%20Hinnebusch&volume=59&publication_year=2005&pages=407-450&pmid=16153175&doi=10.1146/annurev.micro.59.031805.133833&)]
21. Hopper AK, Huang HY. 2015.
    Quality control pathways for nucleus-encoded eukaryotic tRNA biosynthesis and subcellular trafficking. Mol Cell Biol
    35: 2052–2058. [[DOI](https://doi.org/10.1128/MCB.00131-15)] [[PMC free article](/articles/PMC4438251/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25848089/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Quality%20control%20pathways%20for%20nucleus-encoded%20eukaryotic%20tRNA%20biosynthesis%20and%20subcellular%20trafficking&author=AK%20Hopper&author=HY%20Huang&volume=35&publication_year=2015&pages=2052-2058&pmid=25848089&doi=10.1128/MCB.00131-15&)]
22. Huh WK, Falvo JV, Gerke LC, Carroll AS, Howson RW, Weissman JS, O'Shea EK. 2003.
    Global analysis of protein localization in budding yeast. Nature
    425: 686–691. [[DOI](https://doi.org/10.1038/nature02026)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/14562095/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Global%20analysis%20of%20protein%20localization%20in%20budding%20yeast&author=WK%20Huh&author=JV%20Falvo&author=LC%20Gerke&author=AS%20Carroll&author=RW%20Howson&volume=425&publication_year=2003&pages=686-691&pmid=14562095&doi=10.1038/nature02026&)]
23. Ibar C, Cataldo VF, Vásquez-Doorman C, Olguín P, Glavic A. 2013.
    *Drosophila* p53-related protein kinase is required for PI3K/TOR pathway-dependent growth. Development
    140: 1282–1291. [[DOI](https://doi.org/10.1242/dev.086918)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23444356/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Development&title=Drosophila%20p53-related%20protein%20kinase%20is%20required%20for%20PI3K/TOR%20pathway-dependent%20growth&author=C%20Ibar&author=VF%20Cataldo&author=C%20V%C3%A1squez-Doorman&author=P%20Olgu%C3%ADn&author=A%20Glavic&volume=140&publication_year=2013&pages=1282-1291&pmid=23444356&doi=10.1242/dev.086918&)]
24. Kalamarz ME, Paddibhatla I, Nadar C, Govind S. 2012.
    Sumoylation is tumor-suppressive and confers proliferative quiescence to hematopoietic progenitors in *Drosophila melanogaster* larvae. Biol Open
    1: 161–172. [[DOI](https://doi.org/10.1242/bio.2012043)] [[PMC free article](/articles/PMC3507282/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23213407/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Open&title=Sumoylation%20is%20tumor-suppressive%20and%20confers%20proliferative%20quiescence%20to%20hematopoietic%20progenitors%20in%20Drosophila%20melanogaster%20larvae&author=ME%20Kalamarz&author=I%20Paddibhatla&author=C%20Nadar&author=S%20Govind&volume=1&publication_year=2012&pages=161-172&pmid=23213407&doi=10.1242/bio.2012043&)]
25. Kanakousaki K, Gibson MC. 2012.
    A differential requirement for SUMOylation in proliferating and non-proliferating cells during *Drosophila* development. Development
    139: 2751–2762. [[DOI](https://doi.org/10.1242/dev.082974)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22745316/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Development&title=A%20differential%20requirement%20for%20SUMOylation%20in%20proliferating%20and%20non-proliferating%20cells%20during%20Drosophila%20development&author=K%20Kanakousaki&author=MC%20Gibson&volume=139&publication_year=2012&pages=2751-2762&pmid=22745316&doi=10.1242/dev.082974&)]
26. Kirchner S, Ignatova Z. 2015.
    Emerging roles of tRNA in adaptive translation, signalling dynamics and disease. Nat Rev Genet
    16: 98–112. [[DOI](https://doi.org/10.1038/nrg3861)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25534324/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Emerging%20roles%20of%20tRNA%20in%20adaptive%20translation,%20signalling%20dynamics%20and%20disease&author=S%20Kirchner&author=Z%20Ignatova&volume=16&publication_year=2015&pages=98-112&pmid=25534324&doi=10.1038/nrg3861&)]
27. Kisseleva-Romanova E, Lopreiato R, Baudin-Baillieu A, Rousselle JC, Ilan L, Hofmann K, Namane A, Mann C, Libri D. 2006.
    Yeast homolog of a cancer-testis antigen defines a new transcription complex. EMBO J
    25: 3576–3585. [[DOI](https://doi.org/10.1038/sj.emboj.7601235)] [[PMC free article](/articles/PMC1538566/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16874308/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Yeast%20homolog%20of%20a%20cancer-testis%20antigen%20defines%20a%20new%20transcription%20complex&author=E%20Kisseleva-Romanova&author=R%20Lopreiato&author=A%20Baudin-Baillieu&author=JC%20Rousselle&author=L%20Ilan&volume=25&publication_year=2006&pages=3576-3585&pmid=16874308&doi=10.1038/sj.emboj.7601235&)]
28. Krzemien J, Oyallon J, Crozatier M, Vincent A. 2010.
    Hematopoietic progenitors and hemocyte lineages in the *Drosophila* lymph gland. Dev Biol
    346: 310–319. [[DOI](https://doi.org/10.1016/j.ydbio.2010.08.003)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20707995/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dev%20Biol&title=Hematopoietic%20progenitors%20and%20hemocyte%20lineages%20in%20the%20Drosophila%20lymph%20gland&author=J%20Krzemien&author=J%20Oyallon&author=M%20Crozatier&author=A%20Vincent&volume=346&publication_year=2010&pages=310-319&pmid=20707995&doi=10.1016/j.ydbio.2010.08.003&)]
29. Lai EC, Rubin GM. 2001.
    *neuralized* functions cell-autonomously to regulate a subset of Notch-dependent processes during adult *Drosophila* development. Dev Biol
    231: 217–233. [[DOI](https://doi.org/10.1006/dbio.2000.0124)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11180964/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dev%20Biol&title=neuralized%20functions%20cell-autonomously%20to%20regulate%20a%20subset%20of%20Notch-dependent%20processes%20during%20adult%20Drosophila%20development&author=EC%20Lai&author=GM%20Rubin&volume=231&publication_year=2001&pages=217-233&pmid=11180964&doi=10.1006/dbio.2000.0124&)]
30. Lecourtois M, Schweisguth F. 1995.
    The neurogenic suppressor of sairless DNA-binding protein mediates the transcriptional activation of the *enhancer of split* complex genes triggered by Notch signaling. Genes Dev
    9: 2598–2608. [[DOI](https://doi.org/10.1101/gad.9.21.2598)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/7590238/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&title=The%20neurogenic%20suppressor%20of%20sairless%20DNA-binding%20protein%20mediates%20the%20transcriptional%20activation%20of%20the%20enhancer%20of%20split%20complex%20genes%20triggered%20by%20Notch%20signaling&author=M%20Lecourtois&author=F%20Schweisguth&volume=9&publication_year=1995&pages=2598-2608&pmid=7590238&doi=10.1101/gad.9.21.2598&)]
31. Lemaitre B, Meister M, Govind S, Georgel P, Steward R, Reichhart JM, Hoffmann JA. 1995.
    Functional analysis and regulation of nuclear import of dorsal during the immune response in *Drosophila*. EMBO J
    14: 536–545. [[DOI](https://doi.org/10.1002/j.1460-2075.1995.tb07029.x)] [[PMC free article](/articles/PMC398111/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/7859742/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Functional%20analysis%20and%20regulation%20of%20nuclear%20import%20of%20dorsal%20during%20the%20immune%20response%20in%20Drosophila&author=B%20Lemaitre&author=M%20Meister&author=S%20Govind&author=P%20Georgel&author=R%20Steward&volume=14&publication_year=1995&pages=536-545&pmid=7859742&doi=10.1002/j.1460-2075.1995.tb07029.x&)]
32. Lin CA, Ellis SR, True HL. 2010.
    The Sua5 protein is essential for normal translational regulation in yeast. Mol Cell Biol
    30: 354–363. [[DOI](https://doi.org/10.1128/MCB.00754-09)] [[PMC free article](/articles/PMC2798302/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19884342/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=The%20Sua5%20protein%20is%20essential%20for%20normal%20translational%20regulation%20in%20yeast&author=CA%20Lin&author=SR%20Ellis&author=HL%20True&volume=30&publication_year=2010&pages=354-363&pmid=19884342&doi=10.1128/MCB.00754-09&)]
33. Lu BY, Emtage PC, Duyf BJ, Hilliker AJ, Eissenberg JC. 2000.
    Heterochromatin protein 1 is required for the normal expression of two heterochromatin genes in *Drosophila*. Genetics
    155: 699–708. [[DOI](https://doi.org/10.1093/genetics/155.2.699)] [[PMC free article](/articles/PMC1461102/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/10835392/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Heterochromatin%20protein%201%20is%20required%20for%20the%20normal%20expression%20of%20two%20heterochromatin%20genes%20in%20Drosophila&author=BY%20Lu&author=PC%20Emtage&author=BJ%20Duyf&author=AJ%20Hilliker&author=JC%20Eissenberg&volume=155&publication_year=2000&pages=699-708&pmid=10835392&doi=10.1093/genetics/155.2.699&)]
34. Machnicka MA, Milanowska K, Osman Oglou O, Purta E, Kurkowska M, Olchowik A, Januszewski W, Kalinowski S, Dunin-Horkawicz S, Rother KM, et al.
    2013.
    MODOMICS: a database of RNA modification pathways—2013 update. Nucleic Acids Res
    41: D262–D267. [[DOI](https://doi.org/10.1093/nar/gks1007)] [[PMC free article](/articles/PMC3531130/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23118484/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=MODOMICS:%20a%20database%20of%20RNA%20modification%20pathways%E2%80%942013%20update&author=MA%20Machnicka&author=K%20Milanowska&author=O%20Osman%20Oglou&author=E%20Purta&author=M%20Kurkowska&volume=41&publication_year=2013&pages=D262-D267&pmid=23118484&doi=10.1093/nar/gks1007&)]
35. Mao DY, Neculai D, Downey M, Orlicky S, Haffani YZ, Ceccarelli DF, Ho JS, Szilard RK, Zhang W, Ho CS, et al.
    2008.
    Atomic structure of the KEOPS complex: an ancient protein kinase-containing molecular machine. Mol Cell
    32: 259–275. [[DOI](https://doi.org/10.1016/j.molcel.2008.10.002)] [[PMC free article](/articles/PMC3098719/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18951093/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell&title=Atomic%20structure%20of%20the%20KEOPS%20complex:%20an%20ancient%20protein%20kinase-containing%20molecular%20machine&author=DY%20Mao&author=D%20Neculai&author=M%20Downey&author=S%20Orlicky&author=YZ%20Haffani&volume=32&publication_year=2008&pages=259-275&pmid=18951093&doi=10.1016/j.molcel.2008.10.002&)]
36. Milán M, Campuzano S, García-Bellido A. 1996.
    Cell cycling and patterned cell proliferation in the wing primordium of *Drosophila*. Proc Natl Acad Sci
    93: 640–645. [[DOI](https://doi.org/10.1073/pnas.93.2.640)] [[PMC free article](/articles/PMC40104/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/8570608/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&title=Cell%20cycling%20and%20patterned%20cell%20proliferation%20in%20the%20wing%20primordium%20of%20Drosophila&author=M%20Mil%C3%A1n&author=S%20Campuzano&author=A%20Garc%C3%ADa-Bellido&volume=93&publication_year=1996&pages=640-645&pmid=8570608&doi=10.1073/pnas.93.2.640&)]
37. Minakhina S, Steward R. 2006.
    Melanotic mutants in *Drosophila*: pathways and phenotypes. Genetics
    174: 253–263. [[DOI](https://doi.org/10.1534/genetics.106.061978)] [[PMC free article](/articles/PMC1569781/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16816412/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Melanotic%20mutants%20in%20Drosophila:%20pathways%20and%20phenotypes&author=S%20Minakhina&author=R%20Steward&volume=174&publication_year=2006&pages=253-263&pmid=16816412&doi=10.1534/genetics.106.061978&)]
38. Miyauchi K, Kimura S, Suzuki T. 2013.
    A cyclic form of *N*6-threonylcarbamoyladenosine as a widely distributed tRNA hypermodification. Nat Chem Biol
    9: 105–111. [[DOI](https://doi.org/10.1038/nchembio.1137)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23242255/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Chem%20Biol&title=A%20cyclic%20form%20of%20N6-threonylcarbamoyladenosine%20as%20a%20widely%20distributed%20tRNA%20hypermodification&author=K%20Miyauchi&author=S%20Kimura&author=T%20Suzuki&volume=9&publication_year=2013&pages=105-111&pmid=23242255&doi=10.1038/nchembio.1137&)]
39. Na JG, Pinto I, Hampsey M. 1992.
    Isolation and characterization of SUA5, a novel gene required for normal growth in *Saccharomyces cerevisiae*. Genetics
    131: 791–801. [[DOI](https://doi.org/10.1093/genetics/131.4.791)] [[PMC free article](/articles/PMC1205092/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/1325384/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Isolation%20and%20characterization%20of%20SUA5,%20a%20novel%20gene%20required%20for%20normal%20growth%20in%20Saccharomyces%20cerevisiae&author=JG%20Na&author=I%20Pinto&author=M%20Hampsey&volume=131&publication_year=1992&pages=791-801&pmid=1325384&doi=10.1093/genetics/131.4.791&)]
40. Oberto J, Breuil N, Hecker A, Farina F, Brochier-Armanet C, Culetto E, Forterre P. 2009.
    Qri7/OSGEPL, the mitochondrial version of the universal Kae1/YgjD protein, is essential for mitochondrial genome maintenance. Nucleic Acids Res
    37: 5343–5352. [[DOI](https://doi.org/10.1093/nar/gkp557)] [[PMC free article](/articles/PMC2760799/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19578062/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Qri7/OSGEPL,%20the%20mitochondrial%20version%20of%20the%20universal%20Kae1/YgjD%20protein,%20is%20essential%20for%20mitochondrial%20genome%20maintenance&author=J%20Oberto&author=N%20Breuil&author=A%20Hecker&author=F%20Farina&author=C%20Brochier-Armanet&volume=37&publication_year=2009&pages=5343-5352&pmid=19578062&doi=10.1093/nar/gkp557&)]
41. Perrochia L, Crozat E, Hecker A, Zhang W, Bareille J, Collinet B, van Tilbeurgh H, Forterre P, Basta T. 2013.
    In vitro biosynthesis of a universal t6A tRNA modification in *Archaea* and *Eukarya*. Nucleic Acids Res
    41: 1953–1964. [[DOI](https://doi.org/10.1093/nar/gks1287)] [[PMC free article](/articles/PMC3561968/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23258706/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=In%20vitro%20biosynthesis%20of%20a%20universal%20t6A%20tRNA%20modification%20in%20Archaea%20and%20Eukarya&author=L%20Perrochia&author=E%20Crozat&author=A%20Hecker&author=W%20Zhang&author=J%20Bareille&volume=41&publication_year=2013&pages=1953-1964&pmid=23258706&doi=10.1093/nar/gks1287&)]
42. Rojas-Benitez D, Thiaville PC, de Crécy-Lagard V, Glavic A. 2015.
    The levels of a universally conserved tRNA modification regulate cell growth. J Biol Chem
    290: 18699–18707. [[DOI](https://doi.org/10.1074/jbc.M115.665406)] [[PMC free article](/articles/PMC4513126/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26063805/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20levels%20of%20a%20universally%20conserved%20tRNA%20modification%20regulate%20cell%20growth&author=D%20Rojas-Benitez&author=PC%20Thiaville&author=V%20de%20Cr%C3%A9cy-Lagard&author=A%20Glavic&volume=290&publication_year=2015&pages=18699-18707&pmid=26063805&doi=10.1074/jbc.M115.665406&)]
43. Small C, Paddibhatla I, Rajwani R, Govind S. 2012.
    An introduction to parasitic wasps of *Drosophila* and the antiparasite immune response. J Vis Exp
    7: e3347. [[DOI](https://doi.org/10.3791/3347)] [[PMC free article](/articles/PMC3466936/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22588641/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Vis%20Exp&title=An%20introduction%20to%20parasitic%20wasps%20of%20Drosophila%20and%20the%20antiparasite%20immune%20response&author=C%20Small&author=I%20Paddibhatla&author=R%20Rajwani&author=S%20Govind&volume=7&publication_year=2012&pages=e3347&pmid=22588641&doi=10.3791/3347&)]
44. Sprinzl M, Vassilenko KS. 2005.
    Compilation of tRNA sequences and sequences of tRNA genes. Nucleic Acids Res
    33: D139–D140. [[DOI](https://doi.org/10.1093/nar/gki012)] [[PMC free article](/articles/PMC539966/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15608164/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Compilation%20of%20tRNA%20sequences%20and%20sequences%20of%20tRNA%20genes&author=M%20Sprinzl&author=KS%20Vassilenko&volume=33&publication_year=2005&pages=D139-D140&pmid=15608164&doi=10.1093/nar/gki012&)]
45. Srinivasan M, Mehta P, Yu Y, Prugar E, Koonin EV, Karzai AW, Sternglanz R. 2011.
    The highly conserved KEOPS/EKC complex is essential for a universal tRNA modification, t6A. EMBO J
    30: 873–881. [[DOI](https://doi.org/10.1038/emboj.2010.343)] [[PMC free article](/articles/PMC3049205/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21183954/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=The%20highly%20conserved%20KEOPS/EKC%20complex%20is%20essential%20for%20a%20universal%20tRNA%20modification,%20t6A&author=M%20Srinivasan&author=P%20Mehta&author=Y%20Yu&author=E%20Prugar&author=EV%20Koonin&volume=30&publication_year=2011&pages=873-881&pmid=21183954&doi=10.1038/emboj.2010.343&)]
46. Su D, Chan CT, Gu C, Lim KS, Chionh YH, McBee ME, Russell BS, Babu IR, Begley TJ, Dedon PC. 2014.
    Quantitative analysis of ribonucleoside modifications in tRNA by HPLC-coupled mass spectrometry. Nat Protoc
    9: 828–841. [[DOI](https://doi.org/10.1038/nprot.2014.047)] [[PMC free article](/articles/PMC4313537/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24625781/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Quantitative%20analysis%20of%20ribonucleoside%20modifications%20in%20tRNA%20by%20HPLC-coupled%20mass%20spectrometry&author=D%20Su&author=CT%20Chan&author=C%20Gu&author=KS%20Lim&author=YH%20Chionh&volume=9&publication_year=2014&pages=828-841&pmid=24625781&doi=10.1038/nprot.2014.047&)]
47. Venken KJ, He Y, Hoskins RA, Bellen HJ. 2006.
    P[acman]: a BAC transgenic platform for targeted insertion of large DNA fragments in *D. melanogaster*. Science
    314: 1747–1751. [[DOI](https://doi.org/10.1126/science.1134426)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17138868/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=P%5Bacman%5D:%20a%20BAC%20transgenic%20platform%20for%20targeted%20insertion%20of%20large%20DNA%20fragments%20in%20D.%20melanogaster&author=KJ%20Venken&author=Y%20He&author=RA%20Hoskins&author=HJ%20Bellen&volume=314&publication_year=2006&pages=1747-1751&pmid=17138868&doi=10.1126/science.1134426&)]
48. Wan LC, Mao DY, Neculai D, Strecker J, Chiovitti D, Kurinov I, Poda G, Thevakumaran N, Yuan F, Szilard RK, et al.
    2013.
    Reconstitution and characterization of eukaryotic N6-threonylcarbamoylation of tRNA using a minimal enzyme system. Nucleic Acids Res
    41: 6332–6346. [[DOI](https://doi.org/10.1093/nar/gkt322)] [[PMC free article](/articles/PMC3695523/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23620299/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Reconstitution%20and%20characterization%20of%20eukaryotic%20N6-threonylcarbamoylation%20of%20tRNA%20using%20a%20minimal%20enzyme%20system&author=LC%20Wan&author=DY%20Mao&author=D%20Neculai&author=J%20Strecker&author=D%20Chiovitti&volume=41&publication_year=2013&pages=6332-6346&pmid=23620299&doi=10.1093/nar/gkt322&)]
49. Warming S, Costantino N, Court DL, Jenkins NA, Copeland NG. 2005.
    Simple and highly efficient BAC recombineering using galK selection. Nucleic Acids Res
    33: e36. [[DOI](https://doi.org/10.1093/nar/gni035)] [[PMC free article](/articles/PMC549575/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15731329/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Simple%20and%20highly%20efficient%20BAC%20recombineering%20using%20galK%20selection&author=S%20Warming&author=N%20Costantino&author=DL%20Court&author=NA%20Jenkins&author=NG%20Copeland&volume=33&publication_year=2005&pages=e36&pmid=15731329&doi=10.1093/nar/gni035&)]
50. Weissenbach J, Grosjean H. 1981.
    Effect of threonylcarbamoyl modification (t6A) in yeast tRNA Arg III on codon-anticodon and anticodon-anticodon interactions. A thermodynamic and kinetic evaluation. Eur J Biochem
    116: 207–213. [[DOI](https://doi.org/10.1111/j.1432-1033.1981.tb05320.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/6788546/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Biochem&title=Effect%20of%20threonylcarbamoyl%20modification%20(t6A)%20in%20yeast%20tRNA%20Arg%20III%20on%20codon-anticodon%20and%20anticodon-anticodon%20interactions.%20A%20thermodynamic%20and%20kinetic%20evaluation&author=J%20Weissenbach&author=H%20Grosjean&volume=116&publication_year=1981&pages=207-213&pmid=6788546&doi=10.1111/j.1432-1033.1981.tb05320.x&)]
51. Yarian C, Townsend H, Czestkowski W, Sochacka E, Malkiewicz AJ, Guenther R, Miskiewicz A, Agris PF. 2002.
    Accurate translation of the genetic code depends on tRNA modified nucleosides. J Biol Chem
    277: 16391–16395. [[DOI](https://doi.org/10.1074/jbc.M200253200)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11861649/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Accurate%20translation%20of%20the%20genetic%20code%20depends%20on%20tRNA%20modified%20nucleosides&author=C%20Yarian&author=H%20Townsend&author=W%20Czestkowski&author=E%20Sochacka&author=AJ%20Malkiewicz&volume=277&publication_year=2002&pages=16391-16395&pmid=11861649&doi=10.1074/jbc.M200253200&)]
52. Zhai RG, Hiesinger PR, Koh TW, Verstreken P, Schulze KL, Cao Y, Jafar-Nejad H, Norga KK, Pan H, Bayat V, et al.
    2003.
    Mapping *Drosophila* mutations with molecularly defined P element insertions. Proc Natl Acad Sci
    100: 10860–10865. [[DOI](https://doi.org/10.1073/pnas.1832753100)] [[PMC free article](/articles/PMC196893/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12960394/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&title=Mapping%20Drosophila%20mutations%20with%20molecularly%20defined%20P%20element%20insertions&author=RG%20Zhai&author=PR%20Hiesinger&author=TW%20Koh&author=P%20Verstreken&author=KL%20Schulze&volume=100&publication_year=2003&pages=10860-10865&pmid=12960394&doi=10.1073/pnas.1832753100&)]
53. Zhang W, Collinet B, Perrochia L, Durand D, van Tilbeurgh H. 2015.
    The ATP-mediated formation of the YgjD-YeaZ-YjeE complex is required for the biosynthesis of tRNA t6A in *Escherichia coli*. Nucleic Acids Res
    43: 1804–1817. [[DOI](https://doi.org/10.1093/nar/gku1397)] [[PMC free article](/articles/PMC4330362/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25578970/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20ATP-mediated%20formation%20of%20the%20YgjD-YeaZ-YjeE%20complex%20is%20required%20for%20the%20biosynthesis%20of%20tRNA%20t6A%20in%20Escherichia%20coli&author=W%20Zhang&author=B%20Collinet&author=L%20Perrochia&author=D%20Durand&author=H%20van%20Tilbeurgh&volume=43&publication_year=2015&pages=1804-1817&pmid=25578970&doi=10.1093/nar/gku1397&)]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

Supplemental Material

[supp\_21\_12\_2103\_\_index.html](/articles/instance/4647464/bin/supp_21_12_2103__index.html) (1,001B, html)

[supp\_053934.115\_SuppMaterial.pdf](/articles/instance/4647464/bin/supp_053934.115_SuppMaterial.pdf) (9.1MB, pdf)