{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "a0c1239b",
   "metadata": {},
   "outputs": [],
   "source": [
    "from dotenv import load_dotenv\n",
    "from qdrant_client import QdrantClient\n",
    "from qdrant_client.models import Distance, VectorParams\n",
    "from langchain_mistralai import MistralAIEmbeddings\n",
    "from langchain_text_splitters import RecursiveCharacterTextSplitter\n",
    "import os"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "8a23a333",
   "metadata": {},
   "outputs": [],
   "source": [
    "load_dotenv()\n",
    "QDRANT_URL = os.getenv(\"QDRANT_URL\")\n",
    "QDRANT_API_KEY = os.getenv(\"QDRANT_API_KEY\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "64837678",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "collections=[]\n"
     ]
    }
   ],
   "source": [
    "qdrant_client = QdrantClient(\n",
    "    url=QDRANT_URL, \n",
    "    api_key=QDRANT_API_KEY,\n",
    ")\n",
    "\n",
    "print(qdrant_client.get_collections())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "597cf4ae",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "c:\\Users\\dinho\\Desktop\\nasa\\nasa_env\\Lib\\site-packages\\tqdm\\auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n",
      "c:\\Users\\dinho\\Desktop\\nasa\\nasa_env\\Lib\\site-packages\\pydantic\\main.py:253: UserWarning: Could not download mistral tokenizer from Huggingface for calculating batch sizes. Set a Huggingface token via the HF_TOKEN environment variable to download the real tokenizer. Falling back to a dummy tokenizer that uses `len()`.\n",
      "  validated_self = self.__pydantic_validator__.validate_python(data, self_instance=self)\n"
     ]
    }
   ],
   "source": [
    "embeddings = MistralAIEmbeddings(model=\"mistral-embed\")\n",
    "vector_size = len(embeddings.embed_query(\"sample text\"))\n",
    "collection_articles = \"nasa_articles\"\n",
    "try:\n",
    "    qdrant_client.get_collection(collection_name=collection_articles)\n",
    "except Exception:\n",
    "    qdrant_client.create_collection(\n",
    "        collection_name=collection_articles,\n",
    "        vectors_config=VectorParams(size=vector_size, distance=Distance.COSINE),\n",
    "    )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "020f9ddd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "collections=[CollectionDescription(name='nasa_articles')]\n"
     ]
    }
   ],
   "source": [
    "print(qdrant_client.get_collections())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "bc392037",
   "metadata": {},
   "outputs": [],
   "source": [
    "text_splitter = RecursiveCharacterTextSplitter.from_tiktoken_encoder(\n",
    "    model_name=\"gpt-4\",\n",
    "    chunk_size=500,\n",
    "    chunk_overlap=200,\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "2bb12afe",
   "metadata": {},
   "outputs": [],
   "source": [
    "from pathlib import Path\n",
    "\n",
    "ruta = Path(\"md_out/www.ncbi.nlm.nih.gov/pmc/articles/PMC2824534/index.md\")\n",
    "\n",
    "with ruta.open(\"r\", encoding=\"utf-8\") as f:\n",
    "    contenido_md = f.read()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "e89f3a61",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Biomaterials\\n\\n. Author manuscript; available in PMC: 2011 Mar 1.\\n\\n*Published in final edited form as:* Biomaterials. 2010 Jan 19;31(8):2193–2200. doi: [10.1016/j.biomaterials.2009.11.092](https://doi.org/10.1016/j.biomaterials.2009.11.092)\\n\\n# *In vitro* generation of mechanically functional cartilage grafts based on adult human stem cells and 3D-woven poly(ε-caprolactone) scaffolds\\n\\n[PK Valonen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Valonen%20PK%22%5BAuthor%5D)\\n\\n### PK Valonen\\n\\n1Harvard-MIT D'"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "contenido_md[:500]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "0017d2f3",
   "metadata": {},
   "outputs": [],
   "source": [
    "texts = text_splitter.split_text(contenido_md)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "f831776d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "9"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(texts)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "6ba647bf",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Biomaterials\n",
      "\n",
      ". Author manuscript; available in PMC: 2011 Mar 1.\n",
      "\n",
      "*Published in final edited form as:* Biomaterials. 2010 Jan 19;31(8):2193–2200. doi: [10.1016/j.biomaterials.2009.11.092](https://doi.org/10.1016/j.biomaterials.2009.11.092)\n",
      "\n",
      "# *In vitro* generation of mechanically functional cartilage grafts based on adult human stem cells and 3D-woven poly(ε-caprolactone) scaffolds\n",
      "\n",
      "[PK Valonen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Valonen%20PK%22%5BAuthor%5D)\n",
      "\n",
      "### PK Valonen\n",
      "\n",
      "1Harvard-MIT Division of Health Sciences and Technology, Massachusetts Institute of Technology, Cambridge, MA, 02139 USA\n",
      "\n",
      "Find articles by [PK Valonen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Valonen%20PK%22%5BAuthor%5D)\n",
      "\n",
      "1, [FT Moutos](https://pubmed.ncbi.nlm.nih.gov/?term=%22Moutos%20FT%22%5BAuthor%5D)\n",
      "\n",
      "### FT Moutos\n",
      "\n",
      "2Departments of Surgery, Biomedical Engineering, and Mechanical Engineering & Materials Science, Duke University Medical Center, Durham, NC 27710 USA\n",
      "\n",
      "Find articles by [FT Moutos](https://pubmed.ncbi.nlm.nih.gov/?term=%22Moutos%20FT%22%5BAuthor%5D)\n",
      "\n",
      "2, [A Kusanagi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kusanagi%20A%22%5BAuthor%5D)\n",
      "\n",
      "### A Kusanagi\n",
      "\n",
      "3Harvard-MIT Division of Health Sciences and Technology, Massachusetts Institute of Technology, Cambridge, MA, 02139 USA\n",
      "\n",
      "Find articles by [A Kusanagi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kusanagi%20A%22%5BAuthor%5D)\n",
      "\n",
      "3, [M Moretti](https://pubmed.ncbi.nlm.nih.gov/?term=%22Moretti%20M%22%5BAuthor%5D)\n",
      "\n",
      "### M Moretti\n",
      "\n",
      "4I.R.C.C.S. Galeazzi Orthopedic Institute and MIT Research Affiliate, Via R. Galeazzi 4, Milan 20161, Italy\n",
      "\n",
      "Find articles by [M Moretti](https://pubmed.ncbi.nlm.nih.gov/?term=%22Moretti%20M%22%5BAuthor%5D)\n",
      "\n",
      "4, [BO Diekman](https://pubmed.ncbi.nlm.nih.gov/?term=%22Diekman%20BO%22%5BAuthor%5D)\n",
      "\n",
      "### BO Diekman\n",
      "\n",
      "5Departments of Surgery, Biomedical Engineering, and Mechanical Engineering & Materials Science, Duke University Medical Center, Durham, NC 27710 USA\n",
      "\n",
      "Find articles by [BO Diekman](https://pubmed.ncbi.nlm.nih.gov/?term=%22Diekman%20BO%22%5BAuthor%5D)\n",
      "\n",
      "5, [JF Welter](https://pubmed.ncbi.nlm.nih.gov/?term=%22Welter%20JF%22%5BAuthor%5D)\n",
      "\n",
      "### JF Welter\n",
      "\n",
      "6Skeletal Research Center, Department of Biology, Case Western Reserve University, Cleveland, OH, 44106 USA\n",
      "\n",
      "Find articles by [JF Welter](https://pubmed.ncbi.nlm.nih.gov/?term=%22Welter%20JF%22%5BAuthor%5D)\n",
      "\n",
      "6, [AI Caplan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Caplan%20AI%22%5BAuthor%5D)\n",
      "\n",
      "### AI Caplan\n",
      "\n",
      "7Skeletal Research Center, Department of Biology, Case Western Reserve University, Cleveland, OH, 44106 USA\n",
      "\n",
      "Find articles by [AI Caplan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Caplan%20AI%22%5BAuthor%5D)\n",
      "\n",
      "7, [F Guilak](https://pubmed.ncbi.nlm.nih.gov/?term=%22Guilak%20F%22%5BAuthor%5D)\n",
      "\n",
      "### F Guilak\n",
      "\n",
      "8Departments of Surgery, Biomedical Engineering, and Mechanical Engineering & Materials Science, Duke University Medical Center, Durham, NC 27710 USA\n",
      "\n",
      "Find articles by [F Guilak](https://pubmed.ncbi.nlm.nih.gov/?term=%22Guilak%20F%22%5BAuthor%5D)\n",
      "\n",
      "8, [LE Freed](https://pubmed.ncbi.nlm.nih.gov/?term=%22Freed%20LE%22%5BAuthor%5D)\n",
      "\n",
      "### LE Freed\n",
      "\n",
      "9Biomedical Engineering Group, C.S. Draper Laboratory and MIT-Affiliated Research Scientist, 555 Technology Square-Mail Stop 32, Cambridge, MA 02139-4307 USA. Tel: (617) 258-4234; Fax: 617-258-3858; Lfreed@draper.com\n",
      "\n",
      "Find articles by [LE Freed](https://pubmed.ncbi.nlm.nih.gov/?term=%22Freed%20LE%22%5BAuthor%5D)\n",
      "\n",
      "9,✉\n",
      "\n",
      "* Author information\n",
      "* Article notes\n",
      "* Copyright and License information\n",
      "\n",
      "1Harvard-MIT Division of Health Sciences and Technology, Massachusetts Institute of Technology, Cambridge, MA, 02139 USA\n",
      "\n",
      "2Departments of Surgery, Biomedical Engineering, and Mechanical Engineering & Materials Science, Duke University Medical Center, Durham, NC 27710 USA\n",
      "\n",
      "3Harvard-MIT Division of Health Sciences and Technology, Massachusetts Institute of Technology, Cambridge, MA, 02139 USA\n",
      "\n",
      "4I.R.C.C.S. Galeazzi Orthopedic Institute and MIT Research Affiliate, Via R. Galeazzi 4, Milan 20161, Italy\n",
      "\n",
      "5Departments of Surgery, Biomedical Engineering, and Mechanical Engineering & Materials Science, Duke University Medical Center, Durham, NC 27710 USA\n",
      "\n",
      "6Skeletal Research Center, Department of Biology, Case Western Reserve University, Cleveland, OH, 44106 USA\n",
      "\n",
      "7Skeletal Research Center, Department of Biology, Case Western Reserve University, Cleveland, OH, 44106 USA\n",
      "\n",
      "8Departments of Surgery, Biomedical Engineering, and Mechanical Engineering & Materials Science, Duke University Medical Center, Durham, NC 27710 USA\n",
      "\n",
      "9Biomedical Engineering Group, C.S. Draper Laboratory and MIT-Affiliated Research Scientist, 555 Technology Square-Mail Stop 32, Cambridge, MA 02139-4307 USA. Tel: (617) 258-4234; Fax: 617-258-3858; Lfreed@draper.com\n",
      "\n",
      "✉\n",
      "\n",
      "Corresponding author.\n",
      "\n",
      "Issue date 2010 Mar.\n",
      "\n",
      "© 2009 Elsevier Ltd. All rights reserved.\n",
      "\n",
      "[PMC Copyright notice](/about/copyright/)\n",
      "\n",
      "PMCID: PMC2824534  NIHMSID: NIHMS163978  PMID: [20034665](https://pubmed.ncbi.nlm.nih.gov/20034665/)\n",
      "\n",
      "The publisher's version of this article is available at [Biomaterials](https://doi.org/10.1016/j.biomaterials.2009.11.092)\n",
      "\n",
      "## Abstract\n",
      "\n",
      "Three-dimensionally woven poly(ε-caprolactone)(PCL) scaffolds were combined with adult human mesenchymal stem cells (hMSC) to engineer mechanically functional cartilage constructs *in vitro*. The specific objectives were to: (i) produce PCL scaffolds with cartilage-like mechanical properties, (ii) demonstrate that hMSCs formed cartilage after 21-days of culture on PCL scaffolds, and (iii) study effects of scaffold structure (loosely vs. tightly woven), culture vessel (static dish vs. oscillating bioreactor), and medium composition (chondrogenic additives with or without serum). Aggregate moduli of 21-day constructs approached normal articular cartilage for tightly woven PCL cultured in bioreactors, were lower for tightly woven PCL cultured statically, and lowest for loosely woven PCL cultured statically (*p*<0.05). Construct DNA, total collagen, and glyocosaminoglycans (GAG) increased in a manner dependent on time, culture vessel, and medium composition. Chondrogenesis was verified histologically by rounded cells within a hyaline-like matrix that immunostained for collagen type II but not type I. Bioreactors yielded constructs with higher collagen content (*p*<0.05) and more homogenous matrix than static controls. Chondrogenic additives yielded constructs with higher GAG (*p*<0.05) and earlier expression of collagen II mRNA if serum was not present in medium. These results show feasibility of functional cartilage tissue engineering from hMSC and 3D woven PCL scaffolds.\n",
      "\n",
      "**Keywords:** Cartilage repair, stem cell, bioreactor, biomechanics, growth factors\n",
      "\n",
      "## INTRODUCTION\n",
      "\n",
      "Degenerative joint disease affects twenty million adults with an economic burden of over $40 billion per year in the US[[1](#R1)]. Once damaged, adult human articular cartilage has a limited capacity for intrinsic repair[[2](#R2)] and hence injuries can lead to progressive damage, joint degeneration, pain, and disability. Cell-based repair of small cartilage defects in the knee joint was first demonstrated clinically fifteen years ago[[3](#R3)]. Many cartilage tissue engineering studies use chondrocytes as the cell source[[4](#R4),[5](#R5)], however this approach is challenged by limited supply of chondrocytes, their limited regenerative potential due to age, disease, de-differentiation during *in vitro* expansion, and the morbidity caused by chondrocyte harvest[[6](#R6)]. Therefore, other studies use mesenchymal stem cells (MSC) as the cell source[[7](#R7),[8](#R8)], as these stem cells can be safely harvested by marrow biopsy, readily expanded *in vitro*, and selectively differentiated into chondrocytes[[9](#R9)].\n",
      "\n",
      "Clinical translation of tissue engineered cartilage is currently limited by inadequate construct structure, mechanical function, and integration[[2](#R2),[10](#R10)] Currently, most tissue engineered constructs for articular cartilage repair possess cartilage-mimetic material properties only after long-term (*e.g.*, 1 to 6 month) *in vitro* culture[[5](#R5),[11](#R11),[12](#R12)]. This lack of early construct mechanical function implies a need for new tissue engineering technologies such as scaffolds and bioreactors[[13](#R13),[14](#R14)]. For example, the stiffness and strength of previously used scaffolds were several orders of magnitude below normal articular cartilage, particularly in tension[[12](#R12),[15](#R15),[16](#R16)]. Likewise, mechanical properties of engineered cartilage produced using these scaffolds and hMSC were at least one order of magnitude below values reported for normal cartilage despite prolonged *in vitro* culture[[17](#R17),[18](#R18)].\n",
      "\n",
      "The goal of the present study was to produce mechanically functional tissue engineered cartilage from adult hMSC and three-dimensionally (3D) woven PCL scaffolds in 21 days *in vitro*. Effects of (i) scaffold structure (loosely *vs.* tightly woven PCL), (ii) culture vessel (static dish *vs.* oscillating bioreactor) and (iii) medium composition (chondrogenic additives with or without serum) on construct mechanical, biochemical, and molecular properties were quantified. A 3D weaving method[[19](#R19)] was applied to multi-filament PCL yarn to create scaffolds with cartilage-mimetic mechanical properties. The PCL was selected because it is a FDA-approved, biocompatible material[[20](#R20),[21](#R21)] that supports chondrogenesis[[22](#R22)] and degrades slowly (*i.e.* less than 5% degradation at 2 years, as measured by mass loss) into by-products that are entirely cleared from the body[[23](#R23),[24](#R24)].\n",
      "\n",
      "The 3D woven PCL scaffolds were seeded with hMSC mixed with Matrigel® such that gel entrapment enhanced cell seeding efficiency[[25](#R25)] and also helped maintain spherical cell morphology for promotion of chondrogenesis[[26](#R26)]. The hMSC-PCL constructs were cultured either in static dishes or in an oscillatory bioreactor that provided bi-directional percolation of culture medium directly through the construct[[27](#R27)]. Bioreactors were studied because these devices are known to enable functional tissue engineering due to combined effects of enhanced mass transport and mechanotransduction[[14](#R14),[28](#R28)–[34](#R34)]. Bi-directional rather than uni-directional perfusion was selected because the latter yielded different conditions at opposing construct upper and lower surfaces resulting in spatial concentration gradients and inhomogeneous tissue development[[35](#R35),[36](#R36)].\n",
      "\n",
      "Three different culture media were tested as follows. Differentiation medium 1 (DM1) containing serum and chondrogenic additives (TGF-β, ITS+ Premix, dexamethasone, ascorbic acid, proline, and non-essential amino acids) was selected based on our previous work[[8](#R8),[17](#R17),[37](#R37)]. Differentiation medium 2 (DM2) containing chondrogenic additives but not serum was selected based on reports that serum inhibited chondrogenesis by synoviocytes[[38](#R38),[39](#R39)] and caused hypertrophy of chondrocytes[[40](#R40)]. Control medium (CM) without chondrogenic additives was tested to assess spontaneous chondrogenic differentiation in hMSC-PCL constructs.\n",
      "\n",
      "## MATERIALS AND METHODS\n",
      "\n",
      "All tissue culture reagents were from Invitrogen (Carlsbad, CA) unless otherwise specified.\n",
      "\n",
      "### Poly(ε-caprolactone) (PCL) scaffolds\n",
      "\n",
      "A custom-built loom[[19](#R19)] was used to weave PCL multi-filament yarns (24 µm diameter per filament; 44 filaments/yarn, Grilon KE-60, EMS/Griltech, Domat, Switzerland) in three orthogonal directions (x-warp, y-weft, and a vertical z-direction) ([Figure 1A](#F1)). A loosely woven scaffold was made with widely spaced warp yarns (8 yarns/cm), closely spaced weft yarns (20 yarns/cm) and two z-layers between each warp yarn ([Figure 1B](#F1)). A tightly woven scaffold was made with closely spaced warp and weft yarns (24 and 20 yarns/cm, respectively) and one z-layer between each warp yarn. ([Figure 1C](#F1)). These weaving parameters, in conjunction with fiber size and the density of PCL (1.145 g/cm3)[[41](#R41)], determined scaffold porosity and pore dimensions. The loosely woven scaffold had porosity of 68 ± 0.3%, approximate pore dimensions of 850 µm ×1100 µm × 100 µm, and approximate thickness of 0.9 mm. The tightly woven scaffold had porosity of 61 ± 0.2 %, approximate pore dimensions of 330 µm × 260 µm × 100 µm and approximate thickness of 1.3 mm. Prior to cell culture, scaffolds were immersed in 4N NaOH for 18 h, thoroughly rinsed in de-ionized water, dried, ethylene oxide sterilized, and punched into 7 mm diameter discs using dermal punches (Acuderm Inc., Ft. Lauderdale, FL).\n",
      "\n",
      "#### Figure 1.\n",
      "\n",
      "[![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/de82/2824534/91b9156c8d20/nihms163978f1.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2824534_nihms163978f1.jpg)\n",
      "\n",
      "[Open in a new tab](figure/F1/)\n",
      "\n",
      "The 3D-woven PCL scaffold. (A) schematic; (B–C) scanning electron micrographs of (B) loosely and (C) tightly woven scaffolds. Scale bars: 1 mm.\n",
      "\n",
      "### Human mesenchymal stem cells\n",
      "\n",
      "The hMSCs were derived from bone marrow aspirates obtained from a healthy middle-aged adult male at the Hematopoietic Stem Cell Core Facility at Case Western Reserve University. Informed consent was obtained, and an Institutional Review Board-approved aspiration procedure was used[[42](#R42)]. Briefly, the bone marrow sample was washed with Dulbecco’s modified Eagle’s medium (DMEM-LG, Gibco) supplemented with 10% fetal bovine serum (FBS) from a selected lot[[9](#R9)]. The sample was centrifuged at 460×*g* on a pre-formed Percoll density gradient (1.073 g/mL) to isolate the mononucleated cells. These cells were resuspended in serum-supplemented medium and seeded at a density of 1.8×105 cells/cm2 in 10 cm diameter plates. Non-adherent cells were removed after four days by changing the medium. For the remainder of the cell expansion phase, the medium was additionally supplemented with 10 ng/mL of recombinant human fibroblast growth factor-basic (rhFGF-2, Peprotech, Rocky Hill, NJ)[[43](#R43)], and was replaced twice per week. The primary culture was trypsinized after approximately two weeks, and then cryopreserved using Gibco Freezing medium.\n",
      "\n",
      "### Tissue engineered constructs\n"
     ]
    }
   ],
   "source": [
    "print(texts[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7ec962bd",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "1248104b",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2025-10-05 11:33:44.582 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.\n",
      "/local1/ronaldinho/enviroments/nasa_env/lib/python3.10/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n",
      "/local1/ronaldinho/enviroments/nasa_env/lib/python3.10/site-packages/pydantic/main.py:253: UserWarning: Could not download mistral tokenizer from Huggingface for calculating batch sizes. Set a Huggingface token via the HF_TOKEN environment variable to download the real tokenizer. Falling back to a dummy tokenizer that uses `len()`.\n",
      "  validated_self = self.__pydantic_validator__.validate_python(data, self_instance=self)\n"
     ]
    }
   ],
   "source": [
    "import streamlit as st\n",
    "# Assuming get_graph is in components/kw_graph.py\n",
    "# from components.kw_graph import get_graph \n",
    "from streamlit_agraph import agraph, Node, Edge, Config\n",
    "import json\n",
    "from qdrant_client import QdrantClient\n",
    "from langchain_mistralai import MistralAIEmbeddings\n",
    "from langchain_qdrant import QdrantVectorStore\n",
    "from dotenv import load_dotenv\n",
    "import os\n",
    "\n",
    "load_dotenv()\n",
    "QDRANT_URL = os.getenv(\"QDRANT_URL\")\n",
    "QDRANT_API_KEY = os.getenv(\"QDRANT_API_KEY\")\n",
    "CELLECTION_NAME = \"tasks_proyects\"\n",
    "client = QdrantClient(\n",
    "        url=QDRANT_URL, \n",
    "        api_key=QDRANT_API_KEY,\n",
    "    )\n",
    "embeddings = MistralAIEmbeddings(model=\"mistral-embed\")\n",
    "store = QdrantVectorStore(\n",
    "        client=client,\n",
    "        collection_name=CELLECTION_NAME,\n",
    "        embedding=embeddings,\n",
    "    )\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "cf1c2d45",
   "metadata": {},
   "outputs": [],
   "source": [
    "retrieved_docs = store.similarity_search('hola', k=10)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "d6593975",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Document(metadata={'project_title': 'Genomic and Functional Analysis of Desiccation Tolerance of International Space Station Isolated Enterococcus faecalis and Their Pathogenicity in Caenorhabditis elegans   Genomic and Functional Analysis of Biofilm Morphotypes of International Space Station Isolated Staphylococcus epidermidis and Their Pathogenicity in Caenorhabditis elegans)Reduce', 'fiscal_year': 'FY 2020', 'division': 'Space Biology', 'research_discipline/element': '', 'start_date': '09/13/2017', 'end_date': '11/30/2019', 'task_last_updated': '01/24/2020', 'research_discipline_element': 'Fiscal Year:FY 2020Division:Space BiologyResearch Discipline/Element:Space Biology: Microbiology', 'pdf_download_link': 'tbpdf.cfm?id=12404', 'principal_investigator': {}, 'project_information': {'responsible_center': 'NASA ARC', 'grant_monitor': 'Griko, Yuri (Yuri.V.Griko@nasa.gov)', 'center_contact': '650-604-0519', 'unique_id': '11568', 'solicitation__funding_source': '2016 Space Biology (ROSBio) NNH16ZTT001N-MOBE. Appendix B: Post-Doctoral Fellowships to Study the Microbiome of the ISS as a Built Environment', 'grantcontract_no': '80NSSC17K0688', 'project_type': 'Ground', 'no_of_post_docs': '1', 'no_of_phd_candidates': '0', \"no_of_master's_candidates\": '0', \"no_of_bachelor's_candidates\": '0', 'no_of_phd_degrees': '0', \"no_of_master's_degrees\": '0', \"no_of_bachelor's_degrees\": '0', 'space_biology_element': '(1) Microbiology', 'space_biology_cross-element_discipline': 'None', 'space_biology_special_category': 'None', 'flight_assignment/project_notes': 'NOTE: This was extended to 9/30/2020; however, PI moved to NASA Ames Research Center and the fellowship then ended as of 11/30/2019 (Ed., 2/4/2020)', 'task_description': 'POSTDOCTORAL FELLOWSHIP\\nNOTE: This project is funded by NASA as a result of a Space Act Agreement between NASA and the Sloan Foundation, for Collaborative Research Proposal for Microbiology of the Built Environment (MoBE). The purpose of this Space Act Agreement is to enhance the ability of NASA and the Sloan Foundation to support scientific research and the dissemination of information about the effect of spaceflight on microbial ecosystems in spacecraft such as the International Space Station (ISS) through sponsorship of postdoctoral fellowships. NASA is providing the isolates for all studies.\\nNEW TITLE--July 2018: Genomic and Functional Analysis of Desiccation Tolerance of International Space Station Isolated Enterococcus faecalis and Their Pathogenicity in Caenorhabditis elegans. Change in organism used in this project, approved by NASA Headquarters.\\nEnterococcus faecalis is an opportunistic human pathogen consistently isolated and detected by molecular analysis on board the International Space Station (ISS). The ability of this organism to survive outside of the human gut may have important implications for its spread to a new host environment.  The goal of this study is to characterize the desiccation survival capacity of ISS isolated E. faecalis strains and to investigate their pathogenicity in the model organism Caenorhabditis elegans. First, we shall provide the first comparative genomic information of E. faecalis strains to characterize strain-specific sequence variants and the identification of traits that may confer increased resistance to desiccation and antibiotics. We will also examine the presence of the clustered regularly interspaced short palindromic repeats-CRISPR-associated proteins (CRISPR-Cas) that confer bacterial resistance to foreign DNA and may provide an additional method of persistence to the transfer of mobile genetic elements. Phenotypic data will determine how desiccation tolerance varies, and how these variations relate to the clinical and commensal strain types. In vivo pathogenicity tests will culminate our investigation into the effect of E. faecalis biofilm morphotypes on the ability to establish infection in the model organism Caenorhabditis elegans. This work will help to determine genomic, phenotypic, and physiological basis for various desiccation and antibiotic resistance of ISS isolated E. faecalis, and provide insight to pathogenicity of spaceflight isolated microorganisms, where astronaut immunity is decreased and the availability of antibiotics is limited.', 'research_impact/earth_benefits': 'This project will investigate the potential for opportunistic human pathogens, E. faecalis, to survive outside of the human gut.  These isolates have not been genetically or phenotypically characterized and those are the first steps of this project.'}, 'task_progress_and_bibliography': {'task_progress': 'Enterococcus faecalis is a multidrug resistant, opportunistic human pathogen and a leading cause of hospital acquired infections. Recently, isolates have been recovered from the air and surfaces onboard the International Space Station (ISS). Pangenomic and functional analyses were carried out to assess their potential impact on astronaut health. Genomes of each ISS isolate, and both clinical and commensal reference strains, were evaluated for their core and unique gene content, acquired antibiotic resistance genes, phage, plasmid content, and virulence traits. In order to determine their potential survival when outside of the human host, isolates were also challenged with three weeks of desiccation at 30% relative humidity. Finally, pathogenicity of the ISS strains was evaluated in the model organism Caenorhabditis elegans. At the culmination of this study, there were no defining signatures that separated known pathogenic strains from the more commensal phenotypes. As a result, the current reliance on database information alone must be shifted to experimentally evaluated characteristics of clinically relevant microorganisms. By harnessing the technology already onboard the ISS, the crew will then be able to achieve near-real time detection of isolates that may impact crew health.', 'bibliography_description': 'Description: (Last Updated: 02/01/2021)', 'bibliography_articles': \"Ver 'Show Cumulative Bibliography'\"}, 'id': '11864', 'source_file': '11864.html', '_id': 'fef53133-1ce1-46f7-83ba-8ce0156803c1', '_collection_name': 'tasks_proyects'}, page_content='POSTDOCTORAL FELLOWSHIP\\nNOTE: This project is funded by NASA as a result of a Space Act Agreement between NASA and the Sloan Foundation, for Collaborative Research Proposal for Microbiology of the Built Environment (MoBE). The purpose of this Space Act Agreement is to enhance the ability of NASA and the Sloan Foundation to support scientific research and the dissemination of information about the effect of spaceflight on microbial ecosystems in spacecraft such as the International Space Station (ISS) through sponsorship of postdoctoral fellowships. NASA is providing the isolates for all studies.\\nNEW TITLE--July 2018: Genomic and Functional Analysis of Desiccation Tolerance of International Space Station Isolated Enterococcus faecalis and Their Pathogenicity in Caenorhabditis elegans. Change in organism used in this project, approved by NASA Headquarters.\\nEnterococcus faecalis is an opportunistic human pathogen consistently isolated and detected by molecular analysis on board the International Space Station (ISS). The ability of this organism to survive outside of the human gut may have important implications for its spread to a new host environment.  The goal of this study is to characterize the desiccation survival capacity of ISS isolated E. faecalis strains and to investigate their pathogenicity in the model organism Caenorhabditis elegans. First, we shall provide the first comparative genomic information of E. faecalis strains to characterize strain-specific sequence variants and the identification of traits that may confer increased resistance to desiccation and antibiotics. We will also examine the presence of the clustered regularly interspaced short palindromic repeats-CRISPR-associated proteins (CRISPR-Cas) that confer bacterial resistance to foreign DNA and may provide an additional method of persistence to the transfer of mobile genetic elements. Phenotypic data will determine how desiccation tolerance varies, and how these variations relate to the clinical and commensal strain types. In vivo pathogenicity tests will culminate our investigation into the effect of E. faecalis biofilm morphotypes on the ability to establish infection in the model organism Caenorhabditis elegans. This work will help to determine genomic, phenotypic, and physiological basis for various desiccation and antibiotic resistance of ISS isolated E. faecalis, and provide insight to pathogenicity of spaceflight isolated microorganisms, where astronaut immunity is decreased and the availability of antibiotics is limited.'),\n",
       " Document(metadata={'project_title': 'Genomic and Functional Analysis of Desiccation Tolerance of International Space Station Isolated Enterococcus faecalis and Their Pathogenicity in Caenorhabditis elegans   Genomic and Functional Analysis of Biofilm Morphotypes of International Space Station Isolated Staphylococcus epidermidis and Their Pathogenicity in Caenorhabditis elegans)Reduce', 'fiscal_year': 'FY 2020', 'division': 'Space Biology', 'research_discipline/element': '', 'start_date': '09/13/2017', 'end_date': '11/30/2019', 'task_last_updated': '01/24/2020', 'research_discipline_element': 'Fiscal Year:FY 2020Division:Space BiologyResearch Discipline/Element:Space Biology: Microbiology', 'pdf_download_link': 'tbpdf.cfm?id=12404', 'principal_investigator': {}, 'project_information': {'responsible_center': 'NASA ARC', 'grant_monitor': 'Griko, Yuri (Yuri.V.Griko@nasa.gov)', 'center_contact': '650-604-0519', 'unique_id': '11568', 'solicitation__funding_source': '2016 Space Biology (ROSBio) NNH16ZTT001N-MOBE. Appendix B: Post-Doctoral Fellowships to Study the Microbiome of the ISS as a Built Environment', 'grantcontract_no': '80NSSC17K0688', 'project_type': 'Ground', 'no_of_post_docs': '1', 'no_of_phd_candidates': '0', \"no_of_master's_candidates\": '0', \"no_of_bachelor's_candidates\": '0', 'no_of_phd_degrees': '0', \"no_of_master's_degrees\": '0', \"no_of_bachelor's_degrees\": '0', 'space_biology_element': '(1) Microbiology', 'space_biology_cross-element_discipline': 'None', 'space_biology_special_category': 'None', 'flight_assignment/project_notes': 'NOTE: This was extended to 9/30/2020; however, PI moved to NASA Ames Research Center and the fellowship then ended as of 11/30/2019 (Ed., 2/4/2020)', 'task_description': 'POSTDOCTORAL FELLOWSHIP\\nNOTE: This project is funded by NASA as a result of a Space Act Agreement between NASA and the Sloan Foundation, for Collaborative Research Proposal for Microbiology of the Built Environment (MoBE). The purpose of this Space Act Agreement is to enhance the ability of NASA and the Sloan Foundation to support scientific research and the dissemination of information about the effect of spaceflight on microbial ecosystems in spacecraft such as the International Space Station (ISS) through sponsorship of postdoctoral fellowships. NASA is providing the isolates for all studies.\\nNEW TITLE--July 2018: Genomic and Functional Analysis of Desiccation Tolerance of International Space Station Isolated Enterococcus faecalis and Their Pathogenicity in Caenorhabditis elegans. Change in organism used in this project, approved by NASA Headquarters.\\nEnterococcus faecalis is an opportunistic human pathogen consistently isolated and detected by molecular analysis on board the International Space Station (ISS). The ability of this organism to survive outside of the human gut may have important implications for its spread to a new host environment.  The goal of this study is to characterize the desiccation survival capacity of ISS isolated E. faecalis strains and to investigate their pathogenicity in the model organism Caenorhabditis elegans. First, we shall provide the first comparative genomic information of E. faecalis strains to characterize strain-specific sequence variants and the identification of traits that may confer increased resistance to desiccation and antibiotics. We will also examine the presence of the clustered regularly interspaced short palindromic repeats-CRISPR-associated proteins (CRISPR-Cas) that confer bacterial resistance to foreign DNA and may provide an additional method of persistence to the transfer of mobile genetic elements. Phenotypic data will determine how desiccation tolerance varies, and how these variations relate to the clinical and commensal strain types. In vivo pathogenicity tests will culminate our investigation into the effect of E. faecalis biofilm morphotypes on the ability to establish infection in the model organism Caenorhabditis elegans. This work will help to determine genomic, phenotypic, and physiological basis for various desiccation and antibiotic resistance of ISS isolated E. faecalis, and provide insight to pathogenicity of spaceflight isolated microorganisms, where astronaut immunity is decreased and the availability of antibiotics is limited.', 'research_impact/earth_benefits': 'This project will investigate the potential for opportunistic human pathogens, E. faecalis, to survive outside of the human gut.  These isolates have not been genetically or phenotypically characterized and those are the first steps of this project.'}, 'task_progress_and_bibliography': {'task_progress': 'Enterococcus faecalis is a multidrug resistant, opportunistic human pathogen and a leading cause of hospital acquired infections. Recently, isolates have been recovered from the air and surfaces onboard the International Space Station (ISS). Pangenomic and functional analyses were carried out to assess their potential impact on astronaut health. Genomes of each ISS isolate, and both clinical and commensal reference strains, were evaluated for their core and unique gene content, acquired antibiotic resistance genes, phage, plasmid content, and virulence traits. In order to determine their potential survival when outside of the human host, isolates were also challenged with three weeks of desiccation at 30% relative humidity. Finally, pathogenicity of the ISS strains was evaluated in the model organism Caenorhabditis elegans. At the culmination of this study, there were no defining signatures that separated known pathogenic strains from the more commensal phenotypes. As a result, the current reliance on database information alone must be shifted to experimentally evaluated characteristics of clinically relevant microorganisms. By harnessing the technology already onboard the ISS, the crew will then be able to achieve near-real time detection of isolates that may impact crew health.', 'bibliography_description': 'Description: (Last Updated: 02/01/2021)', 'bibliography_articles': \"Ver 'Show Cumulative Bibliography'\"}, 'id': '12404', 'source_file': '12404.html', '_id': '0fa4c9d1-642f-4a49-bda4-6612c738ef14', '_collection_name': 'tasks_proyects'}, page_content='POSTDOCTORAL FELLOWSHIP\\nNOTE: This project is funded by NASA as a result of a Space Act Agreement between NASA and the Sloan Foundation, for Collaborative Research Proposal for Microbiology of the Built Environment (MoBE). The purpose of this Space Act Agreement is to enhance the ability of NASA and the Sloan Foundation to support scientific research and the dissemination of information about the effect of spaceflight on microbial ecosystems in spacecraft such as the International Space Station (ISS) through sponsorship of postdoctoral fellowships. NASA is providing the isolates for all studies.\\nNEW TITLE--July 2018: Genomic and Functional Analysis of Desiccation Tolerance of International Space Station Isolated Enterococcus faecalis and Their Pathogenicity in Caenorhabditis elegans. Change in organism used in this project, approved by NASA Headquarters.\\nEnterococcus faecalis is an opportunistic human pathogen consistently isolated and detected by molecular analysis on board the International Space Station (ISS). The ability of this organism to survive outside of the human gut may have important implications for its spread to a new host environment.  The goal of this study is to characterize the desiccation survival capacity of ISS isolated E. faecalis strains and to investigate their pathogenicity in the model organism Caenorhabditis elegans. First, we shall provide the first comparative genomic information of E. faecalis strains to characterize strain-specific sequence variants and the identification of traits that may confer increased resistance to desiccation and antibiotics. We will also examine the presence of the clustered regularly interspaced short palindromic repeats-CRISPR-associated proteins (CRISPR-Cas) that confer bacterial resistance to foreign DNA and may provide an additional method of persistence to the transfer of mobile genetic elements. Phenotypic data will determine how desiccation tolerance varies, and how these variations relate to the clinical and commensal strain types. In vivo pathogenicity tests will culminate our investigation into the effect of E. faecalis biofilm morphotypes on the ability to establish infection in the model organism Caenorhabditis elegans. This work will help to determine genomic, phenotypic, and physiological basis for various desiccation and antibiotic resistance of ISS isolated E. faecalis, and provide insight to pathogenicity of spaceflight isolated microorganisms, where astronaut immunity is decreased and the availability of antibiotics is limited.'),\n",
       " Document(metadata={'project_title': 'Bacterial, Archaeal, & Fungal Diversity of the ISSReduce', 'images': '', 'fiscal_year': 'FY 2021', 'division': 'Space Biology', 'research_discipline/element': '', 'start_date': '07/01/2013', 'end_date': '03/31/2021', 'task_last_updated': '04/15/2021', 'research_discipline_element': 'Fiscal Year:FY 2021Division:Space BiologyResearch Discipline/Element:Space Biology: Cell & Molecular Biology | Microbiology', 'pdf_download_link': 'tbpdf.cfm?id=14332', 'principal_investigator': {}, 'project_information': {'responsible_center': 'NASA ARC', 'grant_monitor': 'Loftus, David (david.j.loftus@nasa.gov)', 'center_contact': '650-604-1011', 'unique_id': '9349', 'solicitation__funding_source': '2012 Space Biology NNH12ZTT001N', 'grantcontract_no': '47-700062 under Task Order NNN17D016T  ; 19-12829-27 under Task Order NNN13D111T', 'project_type': 'Flight', 'flight_program': 'ISS', 'no_of_post_docs': '3', 'no_of_phd_candidates': '2', \"no_of_bachelor's_candidates\": '4', 'no_of_phd_degrees': '1', \"no_of_master's_degrees\": '0', \"no_of_bachelor's_degrees\": '1', 'space_biology_element': '(1) Cell & Molecular Biology\\n(2) Microbiology', 'space_biology_cross-element_discipline': '(1) Reproductive Biology', 'space_biology_special_category': 'None', 'flight_assignment/project_notes': 'NOTE: Title change in May 2015 to \"Bacterial, Archaeal, & Fungal Diversity of the ISS--HEPA Filter System\" from \"Bacterial, Archaeal, & Fungal Diversity of Kibo-JEM Surfaces,\" per A. Chu/ NASA ARC via NASA JPL Rev. A to Task Order  (Ed., 9/24/15)\\nNOTE: End date changed to 3/31/2021 per F. Hernandez/ARC (Ed., 12/16/21)\\nNOTE: End date changed to 10/01/2021 per F. Hernandez/ARC (Ed., 4/19/21)\\nNOTE: End date changed to 3/31/2021 per F. Hernandez/ARC (Ed., 1/18/21)\\nNOTE: End date changed to 12/31/2020 per F. Hernandez/ARC (Ed., 12/6/19)\\nNOTE: Extended to 12/31/2019 and project number changed to 47-700062 under Task Order NNN17D016T per F. Hernandez/ARC (Ed., 12/17/18)\\nNOTE: Extended to 9/30/2018 per F. Hernandez/ARC (Ed., 3/26/18)\\nNOTE: Extended to 3/31/2018 per F. Hernandez/ARC (Ed., 8/31/17)\\nNOTE: Extended to 6/30/2017 per A. Chu/ARC (Ed., 9/17/15)', 'task_description': 'May 2015 Revision to Task Agreement (Ed. note: information received September 2015).\\nDue to problems obtaining samples from Kibo-JEM (Japanese Experiment Module), the project has been modified with a new title, “Bacterial, Archaeal, & Fungal Diversity of the ISS--HEPA Filter System.” The revised scope will allow the Jet Propulsion Laboratory (JPL) to undertake a 28-month flight study to examine the microorganisms associated with HEPA filters. The ISS-HEPA project will develop a comprehensive census of the microbial population on the HEPA filter collected from various locations of the International Space Station (ISS), using genetic biosignatures and computational modeling. This microbial census will provide NASA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL team will determine which microbial taxa pose particular threats to crew health.\\nOriginal Task Description: Space microbiology research working groups in Japan, authorized by the Japan Aerospace Exploration Agency (JAXA), submitted proposals in response to the JAXA Announcement of Opportunity, 2012. The JAXA proposal that is managed by Principal Investigator Prof. Masao Nasu, Osaka University, contained several subgroups, including subgroup 4, headed by Dr. Takashi Yamazaki, JAXA. This subgroup 4 will conduct an investigation on “Analysis of microbial dynamics in the space habitation system by using new molecular microbial ecological techniques and also monitoring microbial contamination from air and surfaces of the ‘Kibo’ Japanese Experiment Module (Kibo-JEM).” The JAXA proposal submitted by subgroup 4 identifies the Jet Propulsion Laboratory (JPL) team headed by Dr. Kasthuri Venkateswaran as a collaborator to jointly explore the investigation.\\nThe Bacterial, Archaeal, and Fungal Diversity of Kibo-JEM Surfaces project (JPL-JAXA project) will undertake a three-year flight study to examine the changes in the microorganism population on the Kibo-JEM surfaces. The JPL-JAXA project proposes to develop a comprehensive census of the microbial population on the Kibo-JEM surfaces, using genetic biosignatures and computational modeling. This census will provide NASA and JAXA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL-JAXA team will determine which microbial taxa pose particular threats to crew health. Specific JPL-JAXA project goals will enable us to resolve applicable Human Research Program (HRP) integrated research plan (IRP) risks: (1) determine if changes in Kibo-JEM microbiome are occurring during a spaceflight mission and if so, (2) determine whether any changes noted would create a greater health risk for the crew.\\nThere are three aims proposed in this JPL-JAXA project and study of the Kibo-JEM microbiome. Through optimizations and comparative evaluations of methods designed to collect, process, analyze, and archive biomolecules sampled, the JPL-JAXA team will increase downstream sensitivity of laboratory assays as well as enhance the current understanding of the population dynamics of the Kibo-JEM module (Aim 1). The JPL-JAXA team will adapt molecular technologies currently in use with the National Institute of Health (NIH) – Human Microbiome Program (HMP) Roadmap—such as rRNA gene-variable region tag sequencing, whole genome shotgun sequencing (WGS), and DNA microarrays, as well as cloning and sequencing of 16S ribosomal DNA (rDNA)—to comprehensively elucidate the microbial diversity profiles associated with the Kibo-JEM surfaces (Aim 2). The final project deliverable to NASA will be the Kibo-JEM database, a compilation of all of the genomic sequences and genetic information of microbes encountered on and within the Kibo-JEM habitat (Aim 3). Computational modeling of the Kibo-JEM microbial population dynamics inferred from the database will then be carried out to predict the complete spectrum of organisms present.\\nThe JPL-JAXA project will compare microbial diversity, microbial characteristics, and functional capabilities of the microorganisms associated with the Kibo-JEM. The resulting database will enable HRP to evaluate the effect of spaceflight radiation and microgravity on the Kibo-JEM microbiome. The JPL-JAXA project in combination with results from studies of crew susceptibility will be used to determine the risk of microbiologically induced adverse health effects during a spaceflight mission. This assessment will allow HRP to determine if current operational and engineering controls used to mitigate microbiological risks are adequate or whether additional countermeasures should be developed.', 'research_impact/earth_benefits': 'The JPL-JAXA project proposes to develop a comprehensive census of the microbial population on the Kibo-JEM surfaces, using genetic biosignatures and computational modeling. This microbial census will provide NASA and JAXA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL-JAXA team will determine which microbial taxa pose particular threats to crew health.'}, 'task_progress_and_bibliography': {'task_progress': 'During this task, an inventory of microorganisms isolated from the HEPA filters and environmental surfaces of the International Space Station (ISS) as well as crew resupply vehicles (CRV) during multiple NASA projects have been archived, and inventoried under Space Biology Microbial Culture Collection (SBMCC). The whole-genome sequences (WGS) of SBMCC were generated and made available to the public. In addition to multiple copies of same microbial species, the SBMCC consists of several newly identified microorganisms. The WGS of the characterized bacterial isolates include Lactobacillales, Enterobacteriales, Bacillaceae, Agrobacterium, Klebsiella, Methylobacterium, and Staphylococcus. Moreover, isolates of fungi have also been sequenced, belonging to the members of fungal phyla Ascomycota and Basidiomycota.\\nThis SBMCC archival of the ISS and CRV will aid in the ongoing characterization of short sequence, shot-gun metagenome surveys of the microbial diversity present in these built environments. The availability of these new WGS will also enable genetic comparisons between microorganisms sampled from Earth-based environments with those sampled from spacecraft surfaces of the ISS and CRV. Such comparisons will help to elucidate genetic adaptations that allow some of these microorganisms to become more virulent and allow all of them to survive the harsh environment of space (e.g., ionizing radiation, microgravity, desiccation, and oxidative stress). Additionally, when annotated, functional analyses of these WGS can help to identify targeted strategies for the control of problematic microorganisms that may degrade metals and plastics, become potentially pathogenic and harm astronauts, or obstruct water supply due to their biofilm forming capabilities in the engineering systems of the closed habitat.', 'bibliography_description': 'Description: (Last Updated: 09/17/2025)', 'bibliography_articles': \"Ver 'Show Cumulative Bibliography'\"}, 'id': '12275', 'source_file': '12275.html', '_id': '12dc8d15-af4e-400a-82a4-8eeaa9cb57cf', '_collection_name': 'tasks_proyects'}, page_content='May 2015 Revision to Task Agreement (Ed. note: information received September 2015).\\nDue to problems obtaining samples from Kibo-JEM (Japanese Experiment Module), the project has been modified with a new title, “Bacterial, Archaeal, & Fungal Diversity of the ISS--HEPA Filter System.” The revised scope will allow the Jet Propulsion Laboratory (JPL) to undertake a 28-month flight study to examine the microorganisms associated with HEPA filters. The ISS-HEPA project will develop a comprehensive census of the microbial population on the HEPA filter collected from various locations of the International Space Station (ISS), using genetic biosignatures and computational modeling. This microbial census will provide NASA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL team will determine which microbial taxa pose particular threats to crew health.\\nOriginal Task Description: Space microbiology research working groups in Japan, authorized by the Japan Aerospace Exploration Agency (JAXA), submitted proposals in response to the JAXA Announcement of Opportunity, 2012. The JAXA proposal that is managed by Principal Investigator Prof. Masao Nasu, Osaka University, contained several subgroups, including subgroup 4, headed by Dr. Takashi Yamazaki, JAXA. This subgroup 4 will conduct an investigation on “Analysis of microbial dynamics in the space habitation system by using new molecular microbial ecological techniques and also monitoring microbial contamination from air and surfaces of the ‘Kibo’ Japanese Experiment Module (Kibo-JEM).” The JAXA proposal submitted by subgroup 4 identifies the Jet Propulsion Laboratory (JPL) team headed by Dr. Kasthuri Venkateswaran as a collaborator to jointly explore the investigation.\\nThe Bacterial, Archaeal, and Fungal Diversity of Kibo-JEM Surfaces project (JPL-JAXA project) will undertake a three-year flight study to examine the changes in the microorganism population on the Kibo-JEM surfaces. The JPL-JAXA project proposes to develop a comprehensive census of the microbial population on the Kibo-JEM surfaces, using genetic biosignatures and computational modeling. This census will provide NASA and JAXA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL-JAXA team will determine which microbial taxa pose particular threats to crew health. Specific JPL-JAXA project goals will enable us to resolve applicable Human Research Program (HRP) integrated research plan (IRP) risks: (1) determine if changes in Kibo-JEM microbiome are occurring during a spaceflight mission and if so, (2) determine whether any changes noted would create a greater health risk for the crew.\\nThere are three aims proposed in this JPL-JAXA project and study of the Kibo-JEM microbiome. Through optimizations and comparative evaluations of methods designed to collect, process, analyze, and archive biomolecules sampled, the JPL-JAXA team will increase downstream sensitivity of laboratory assays as well as enhance the current understanding of the population dynamics of the Kibo-JEM module (Aim 1). The JPL-JAXA team will adapt molecular technologies currently in use with the National Institute of Health (NIH) – Human Microbiome Program (HMP) Roadmap—such as rRNA gene-variable region tag sequencing, whole genome shotgun sequencing (WGS), and DNA microarrays, as well as cloning and sequencing of 16S ribosomal DNA (rDNA)—to comprehensively elucidate the microbial diversity profiles associated with the Kibo-JEM surfaces (Aim 2). The final project deliverable to NASA will be the Kibo-JEM database, a compilation of all of the genomic sequences and genetic information of microbes encountered on and within the Kibo-JEM habitat (Aim 3). Computational modeling of the Kibo-JEM microbial population dynamics inferred from the database will then be carried out to predict the complete spectrum of organisms present.\\nThe JPL-JAXA project will compare microbial diversity, microbial characteristics, and functional capabilities of the microorganisms associated with the Kibo-JEM. The resulting database will enable HRP to evaluate the effect of spaceflight radiation and microgravity on the Kibo-JEM microbiome. The JPL-JAXA project in combination with results from studies of crew susceptibility will be used to determine the risk of microbiologically induced adverse health effects during a spaceflight mission. This assessment will allow HRP to determine if current operational and engineering controls used to mitigate microbiological risks are adequate or whether additional countermeasures should be developed.'),\n",
       " Document(metadata={'project_title': 'Bacterial, Archaeal, & Fungal Diversity of the ISSReduce', 'images': '', 'fiscal_year': 'FY 2021', 'division': 'Space Biology', 'research_discipline/element': '', 'start_date': '07/01/2013', 'end_date': '03/31/2021', 'task_last_updated': '04/15/2021', 'research_discipline_element': 'Fiscal Year:FY 2021Division:Space BiologyResearch Discipline/Element:Space Biology: Cell & Molecular Biology | Microbiology', 'pdf_download_link': 'tbpdf.cfm?id=14332', 'principal_investigator': {}, 'project_information': {'responsible_center': 'NASA ARC', 'grant_monitor': 'Loftus, David (david.j.loftus@nasa.gov)', 'center_contact': '650-604-1011', 'unique_id': '9349', 'solicitation__funding_source': '2012 Space Biology NNH12ZTT001N', 'grantcontract_no': '47-700062 under Task Order NNN17D016T  ; 19-12829-27 under Task Order NNN13D111T', 'project_type': 'Flight', 'flight_program': 'ISS', 'no_of_post_docs': '3', 'no_of_phd_candidates': '2', \"no_of_bachelor's_candidates\": '4', 'no_of_phd_degrees': '1', \"no_of_master's_degrees\": '0', \"no_of_bachelor's_degrees\": '1', 'space_biology_element': '(1) Cell & Molecular Biology\\n(2) Microbiology', 'space_biology_cross-element_discipline': '(1) Reproductive Biology', 'space_biology_special_category': 'None', 'flight_assignment/project_notes': 'NOTE: Title change in May 2015 to \"Bacterial, Archaeal, & Fungal Diversity of the ISS--HEPA Filter System\" from \"Bacterial, Archaeal, & Fungal Diversity of Kibo-JEM Surfaces,\" per A. Chu/ NASA ARC via NASA JPL Rev. A to Task Order  (Ed., 9/24/15)\\nNOTE: End date changed to 3/31/2021 per F. Hernandez/ARC (Ed., 12/16/21)\\nNOTE: End date changed to 10/01/2021 per F. Hernandez/ARC (Ed., 4/19/21)\\nNOTE: End date changed to 3/31/2021 per F. Hernandez/ARC (Ed., 1/18/21)\\nNOTE: End date changed to 12/31/2020 per F. Hernandez/ARC (Ed., 12/6/19)\\nNOTE: Extended to 12/31/2019 and project number changed to 47-700062 under Task Order NNN17D016T per F. Hernandez/ARC (Ed., 12/17/18)\\nNOTE: Extended to 9/30/2018 per F. Hernandez/ARC (Ed., 3/26/18)\\nNOTE: Extended to 3/31/2018 per F. Hernandez/ARC (Ed., 8/31/17)\\nNOTE: Extended to 6/30/2017 per A. Chu/ARC (Ed., 9/17/15)', 'task_description': 'May 2015 Revision to Task Agreement (Ed. note: information received September 2015).\\nDue to problems obtaining samples from Kibo-JEM (Japanese Experiment Module), the project has been modified with a new title, “Bacterial, Archaeal, & Fungal Diversity of the ISS--HEPA Filter System.” The revised scope will allow the Jet Propulsion Laboratory (JPL) to undertake a 28-month flight study to examine the microorganisms associated with HEPA filters. The ISS-HEPA project will develop a comprehensive census of the microbial population on the HEPA filter collected from various locations of the International Space Station (ISS), using genetic biosignatures and computational modeling. This microbial census will provide NASA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL team will determine which microbial taxa pose particular threats to crew health.\\nOriginal Task Description: Space microbiology research working groups in Japan, authorized by the Japan Aerospace Exploration Agency (JAXA), submitted proposals in response to the JAXA Announcement of Opportunity, 2012. The JAXA proposal that is managed by Principal Investigator Prof. Masao Nasu, Osaka University, contained several subgroups, including subgroup 4, headed by Dr. Takashi Yamazaki, JAXA. This subgroup 4 will conduct an investigation on “Analysis of microbial dynamics in the space habitation system by using new molecular microbial ecological techniques and also monitoring microbial contamination from air and surfaces of the ‘Kibo’ Japanese Experiment Module (Kibo-JEM).” The JAXA proposal submitted by subgroup 4 identifies the Jet Propulsion Laboratory (JPL) team headed by Dr. Kasthuri Venkateswaran as a collaborator to jointly explore the investigation.\\nThe Bacterial, Archaeal, and Fungal Diversity of Kibo-JEM Surfaces project (JPL-JAXA project) will undertake a three-year flight study to examine the changes in the microorganism population on the Kibo-JEM surfaces. The JPL-JAXA project proposes to develop a comprehensive census of the microbial population on the Kibo-JEM surfaces, using genetic biosignatures and computational modeling. This census will provide NASA and JAXA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL-JAXA team will determine which microbial taxa pose particular threats to crew health. Specific JPL-JAXA project goals will enable us to resolve applicable Human Research Program (HRP) integrated research plan (IRP) risks: (1) determine if changes in Kibo-JEM microbiome are occurring during a spaceflight mission and if so, (2) determine whether any changes noted would create a greater health risk for the crew.\\nThere are three aims proposed in this JPL-JAXA project and study of the Kibo-JEM microbiome. Through optimizations and comparative evaluations of methods designed to collect, process, analyze, and archive biomolecules sampled, the JPL-JAXA team will increase downstream sensitivity of laboratory assays as well as enhance the current understanding of the population dynamics of the Kibo-JEM module (Aim 1). The JPL-JAXA team will adapt molecular technologies currently in use with the National Institute of Health (NIH) – Human Microbiome Program (HMP) Roadmap—such as rRNA gene-variable region tag sequencing, whole genome shotgun sequencing (WGS), and DNA microarrays, as well as cloning and sequencing of 16S ribosomal DNA (rDNA)—to comprehensively elucidate the microbial diversity profiles associated with the Kibo-JEM surfaces (Aim 2). The final project deliverable to NASA will be the Kibo-JEM database, a compilation of all of the genomic sequences and genetic information of microbes encountered on and within the Kibo-JEM habitat (Aim 3). Computational modeling of the Kibo-JEM microbial population dynamics inferred from the database will then be carried out to predict the complete spectrum of organisms present.\\nThe JPL-JAXA project will compare microbial diversity, microbial characteristics, and functional capabilities of the microorganisms associated with the Kibo-JEM. The resulting database will enable HRP to evaluate the effect of spaceflight radiation and microgravity on the Kibo-JEM microbiome. The JPL-JAXA project in combination with results from studies of crew susceptibility will be used to determine the risk of microbiologically induced adverse health effects during a spaceflight mission. This assessment will allow HRP to determine if current operational and engineering controls used to mitigate microbiological risks are adequate or whether additional countermeasures should be developed.', 'research_impact/earth_benefits': 'The JPL-JAXA project proposes to develop a comprehensive census of the microbial population on the Kibo-JEM surfaces, using genetic biosignatures and computational modeling. This microbial census will provide NASA and JAXA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL-JAXA team will determine which microbial taxa pose particular threats to crew health.'}, 'task_progress_and_bibliography': {'task_progress': 'During this task, an inventory of microorganisms isolated from the HEPA filters and environmental surfaces of the International Space Station (ISS) as well as crew resupply vehicles (CRV) during multiple NASA projects have been archived, and inventoried under Space Biology Microbial Culture Collection (SBMCC). The whole-genome sequences (WGS) of SBMCC were generated and made available to the public. In addition to multiple copies of same microbial species, the SBMCC consists of several newly identified microorganisms. The WGS of the characterized bacterial isolates include Lactobacillales, Enterobacteriales, Bacillaceae, Agrobacterium, Klebsiella, Methylobacterium, and Staphylococcus. Moreover, isolates of fungi have also been sequenced, belonging to the members of fungal phyla Ascomycota and Basidiomycota.\\nThis SBMCC archival of the ISS and CRV will aid in the ongoing characterization of short sequence, shot-gun metagenome surveys of the microbial diversity present in these built environments. The availability of these new WGS will also enable genetic comparisons between microorganisms sampled from Earth-based environments with those sampled from spacecraft surfaces of the ISS and CRV. Such comparisons will help to elucidate genetic adaptations that allow some of these microorganisms to become more virulent and allow all of them to survive the harsh environment of space (e.g., ionizing radiation, microgravity, desiccation, and oxidative stress). Additionally, when annotated, functional analyses of these WGS can help to identify targeted strategies for the control of problematic microorganisms that may degrade metals and plastics, become potentially pathogenic and harm astronauts, or obstruct water supply due to their biofilm forming capabilities in the engineering systems of the closed habitat.', 'bibliography_description': 'Description: (Last Updated: 09/17/2025)', 'bibliography_articles': \"Ver 'Show Cumulative Bibliography'\"}, 'id': '10728', 'source_file': '10728.html', '_id': '1229e5fc-12c5-44ad-a44a-e3e7e7a2906d', '_collection_name': 'tasks_proyects'}, page_content='May 2015 Revision to Task Agreement (Ed. note: information received September 2015).\\nDue to problems obtaining samples from Kibo-JEM (Japanese Experiment Module), the project has been modified with a new title, “Bacterial, Archaeal, & Fungal Diversity of the ISS--HEPA Filter System.” The revised scope will allow the Jet Propulsion Laboratory (JPL) to undertake a 28-month flight study to examine the microorganisms associated with HEPA filters. The ISS-HEPA project will develop a comprehensive census of the microbial population on the HEPA filter collected from various locations of the International Space Station (ISS), using genetic biosignatures and computational modeling. This microbial census will provide NASA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL team will determine which microbial taxa pose particular threats to crew health.\\nOriginal Task Description: Space microbiology research working groups in Japan, authorized by the Japan Aerospace Exploration Agency (JAXA), submitted proposals in response to the JAXA Announcement of Opportunity, 2012. The JAXA proposal that is managed by Principal Investigator Prof. Masao Nasu, Osaka University, contained several subgroups, including subgroup 4, headed by Dr. Takashi Yamazaki, JAXA. This subgroup 4 will conduct an investigation on “Analysis of microbial dynamics in the space habitation system by using new molecular microbial ecological techniques and also monitoring microbial contamination from air and surfaces of the ‘Kibo’ Japanese Experiment Module (Kibo-JEM).” The JAXA proposal submitted by subgroup 4 identifies the Jet Propulsion Laboratory (JPL) team headed by Dr. Kasthuri Venkateswaran as a collaborator to jointly explore the investigation.\\nThe Bacterial, Archaeal, and Fungal Diversity of Kibo-JEM Surfaces project (JPL-JAXA project) will undertake a three-year flight study to examine the changes in the microorganism population on the Kibo-JEM surfaces. The JPL-JAXA project proposes to develop a comprehensive census of the microbial population on the Kibo-JEM surfaces, using genetic biosignatures and computational modeling. This census will provide NASA and JAXA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL-JAXA team will determine which microbial taxa pose particular threats to crew health. Specific JPL-JAXA project goals will enable us to resolve applicable Human Research Program (HRP) integrated research plan (IRP) risks: (1) determine if changes in Kibo-JEM microbiome are occurring during a spaceflight mission and if so, (2) determine whether any changes noted would create a greater health risk for the crew.\\nThere are three aims proposed in this JPL-JAXA project and study of the Kibo-JEM microbiome. Through optimizations and comparative evaluations of methods designed to collect, process, analyze, and archive biomolecules sampled, the JPL-JAXA team will increase downstream sensitivity of laboratory assays as well as enhance the current understanding of the population dynamics of the Kibo-JEM module (Aim 1). The JPL-JAXA team will adapt molecular technologies currently in use with the National Institute of Health (NIH) – Human Microbiome Program (HMP) Roadmap—such as rRNA gene-variable region tag sequencing, whole genome shotgun sequencing (WGS), and DNA microarrays, as well as cloning and sequencing of 16S ribosomal DNA (rDNA)—to comprehensively elucidate the microbial diversity profiles associated with the Kibo-JEM surfaces (Aim 2). The final project deliverable to NASA will be the Kibo-JEM database, a compilation of all of the genomic sequences and genetic information of microbes encountered on and within the Kibo-JEM habitat (Aim 3). Computational modeling of the Kibo-JEM microbial population dynamics inferred from the database will then be carried out to predict the complete spectrum of organisms present.\\nThe JPL-JAXA project will compare microbial diversity, microbial characteristics, and functional capabilities of the microorganisms associated with the Kibo-JEM. The resulting database will enable HRP to evaluate the effect of spaceflight radiation and microgravity on the Kibo-JEM microbiome. The JPL-JAXA project in combination with results from studies of crew susceptibility will be used to determine the risk of microbiologically induced adverse health effects during a spaceflight mission. This assessment will allow HRP to determine if current operational and engineering controls used to mitigate microbiological risks are adequate or whether additional countermeasures should be developed.'),\n",
       " Document(metadata={'project_title': 'Bacterial, Archaeal, & Fungal Diversity of the ISSReduce', 'images': '', 'fiscal_year': 'FY 2021', 'division': 'Space Biology', 'research_discipline/element': '', 'start_date': '07/01/2013', 'end_date': '03/31/2021', 'task_last_updated': '04/15/2021', 'research_discipline_element': 'Fiscal Year:FY 2021Division:Space BiologyResearch Discipline/Element:Space Biology: Cell & Molecular Biology | Microbiology', 'pdf_download_link': 'tbpdf.cfm?id=14332', 'principal_investigator': {}, 'project_information': {'responsible_center': 'NASA ARC', 'grant_monitor': 'Loftus, David (david.j.loftus@nasa.gov)', 'center_contact': '650-604-1011', 'unique_id': '9349', 'solicitation__funding_source': '2012 Space Biology NNH12ZTT001N', 'grantcontract_no': '47-700062 under Task Order NNN17D016T  ; 19-12829-27 under Task Order NNN13D111T', 'project_type': 'Flight', 'flight_program': 'ISS', 'no_of_post_docs': '3', 'no_of_phd_candidates': '2', \"no_of_bachelor's_candidates\": '4', 'no_of_phd_degrees': '1', \"no_of_master's_degrees\": '0', \"no_of_bachelor's_degrees\": '1', 'space_biology_element': '(1) Cell & Molecular Biology\\n(2) Microbiology', 'space_biology_cross-element_discipline': '(1) Reproductive Biology', 'space_biology_special_category': 'None', 'flight_assignment/project_notes': 'NOTE: Title change in May 2015 to \"Bacterial, Archaeal, & Fungal Diversity of the ISS--HEPA Filter System\" from \"Bacterial, Archaeal, & Fungal Diversity of Kibo-JEM Surfaces,\" per A. Chu/ NASA ARC via NASA JPL Rev. A to Task Order  (Ed., 9/24/15)\\nNOTE: End date changed to 3/31/2021 per F. Hernandez/ARC (Ed., 12/16/21)\\nNOTE: End date changed to 10/01/2021 per F. Hernandez/ARC (Ed., 4/19/21)\\nNOTE: End date changed to 3/31/2021 per F. Hernandez/ARC (Ed., 1/18/21)\\nNOTE: End date changed to 12/31/2020 per F. Hernandez/ARC (Ed., 12/6/19)\\nNOTE: Extended to 12/31/2019 and project number changed to 47-700062 under Task Order NNN17D016T per F. Hernandez/ARC (Ed., 12/17/18)\\nNOTE: Extended to 9/30/2018 per F. Hernandez/ARC (Ed., 3/26/18)\\nNOTE: Extended to 3/31/2018 per F. Hernandez/ARC (Ed., 8/31/17)\\nNOTE: Extended to 6/30/2017 per A. Chu/ARC (Ed., 9/17/15)', 'task_description': 'May 2015 Revision to Task Agreement (Ed. note: information received September 2015).\\nDue to problems obtaining samples from Kibo-JEM (Japanese Experiment Module), the project has been modified with a new title, “Bacterial, Archaeal, & Fungal Diversity of the ISS--HEPA Filter System.” The revised scope will allow the Jet Propulsion Laboratory (JPL) to undertake a 28-month flight study to examine the microorganisms associated with HEPA filters. The ISS-HEPA project will develop a comprehensive census of the microbial population on the HEPA filter collected from various locations of the International Space Station (ISS), using genetic biosignatures and computational modeling. This microbial census will provide NASA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL team will determine which microbial taxa pose particular threats to crew health.\\nOriginal Task Description: Space microbiology research working groups in Japan, authorized by the Japan Aerospace Exploration Agency (JAXA), submitted proposals in response to the JAXA Announcement of Opportunity, 2012. The JAXA proposal that is managed by Principal Investigator Prof. Masao Nasu, Osaka University, contained several subgroups, including subgroup 4, headed by Dr. Takashi Yamazaki, JAXA. This subgroup 4 will conduct an investigation on “Analysis of microbial dynamics in the space habitation system by using new molecular microbial ecological techniques and also monitoring microbial contamination from air and surfaces of the ‘Kibo’ Japanese Experiment Module (Kibo-JEM).” The JAXA proposal submitted by subgroup 4 identifies the Jet Propulsion Laboratory (JPL) team headed by Dr. Kasthuri Venkateswaran as a collaborator to jointly explore the investigation.\\nThe Bacterial, Archaeal, and Fungal Diversity of Kibo-JEM Surfaces project (JPL-JAXA project) will undertake a three-year flight study to examine the changes in the microorganism population on the Kibo-JEM surfaces. The JPL-JAXA project proposes to develop a comprehensive census of the microbial population on the Kibo-JEM surfaces, using genetic biosignatures and computational modeling. This census will provide NASA and JAXA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL-JAXA team will determine which microbial taxa pose particular threats to crew health. Specific JPL-JAXA project goals will enable us to resolve applicable Human Research Program (HRP) integrated research plan (IRP) risks: (1) determine if changes in Kibo-JEM microbiome are occurring during a spaceflight mission and if so, (2) determine whether any changes noted would create a greater health risk for the crew.\\nThere are three aims proposed in this JPL-JAXA project and study of the Kibo-JEM microbiome. Through optimizations and comparative evaluations of methods designed to collect, process, analyze, and archive biomolecules sampled, the JPL-JAXA team will increase downstream sensitivity of laboratory assays as well as enhance the current understanding of the population dynamics of the Kibo-JEM module (Aim 1). The JPL-JAXA team will adapt molecular technologies currently in use with the National Institute of Health (NIH) – Human Microbiome Program (HMP) Roadmap—such as rRNA gene-variable region tag sequencing, whole genome shotgun sequencing (WGS), and DNA microarrays, as well as cloning and sequencing of 16S ribosomal DNA (rDNA)—to comprehensively elucidate the microbial diversity profiles associated with the Kibo-JEM surfaces (Aim 2). The final project deliverable to NASA will be the Kibo-JEM database, a compilation of all of the genomic sequences and genetic information of microbes encountered on and within the Kibo-JEM habitat (Aim 3). Computational modeling of the Kibo-JEM microbial population dynamics inferred from the database will then be carried out to predict the complete spectrum of organisms present.\\nThe JPL-JAXA project will compare microbial diversity, microbial characteristics, and functional capabilities of the microorganisms associated with the Kibo-JEM. The resulting database will enable HRP to evaluate the effect of spaceflight radiation and microgravity on the Kibo-JEM microbiome. The JPL-JAXA project in combination with results from studies of crew susceptibility will be used to determine the risk of microbiologically induced adverse health effects during a spaceflight mission. This assessment will allow HRP to determine if current operational and engineering controls used to mitigate microbiological risks are adequate or whether additional countermeasures should be developed.', 'research_impact/earth_benefits': 'The JPL-JAXA project proposes to develop a comprehensive census of the microbial population on the Kibo-JEM surfaces, using genetic biosignatures and computational modeling. This microbial census will provide NASA and JAXA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL-JAXA team will determine which microbial taxa pose particular threats to crew health.'}, 'task_progress_and_bibliography': {'task_progress': 'During this task, an inventory of microorganisms isolated from the HEPA filters and environmental surfaces of the International Space Station (ISS) as well as crew resupply vehicles (CRV) during multiple NASA projects have been archived, and inventoried under Space Biology Microbial Culture Collection (SBMCC). The whole-genome sequences (WGS) of SBMCC were generated and made available to the public. In addition to multiple copies of same microbial species, the SBMCC consists of several newly identified microorganisms. The WGS of the characterized bacterial isolates include Lactobacillales, Enterobacteriales, Bacillaceae, Agrobacterium, Klebsiella, Methylobacterium, and Staphylococcus. Moreover, isolates of fungi have also been sequenced, belonging to the members of fungal phyla Ascomycota and Basidiomycota.\\nThis SBMCC archival of the ISS and CRV will aid in the ongoing characterization of short sequence, shot-gun metagenome surveys of the microbial diversity present in these built environments. The availability of these new WGS will also enable genetic comparisons between microorganisms sampled from Earth-based environments with those sampled from spacecraft surfaces of the ISS and CRV. Such comparisons will help to elucidate genetic adaptations that allow some of these microorganisms to become more virulent and allow all of them to survive the harsh environment of space (e.g., ionizing radiation, microgravity, desiccation, and oxidative stress). Additionally, when annotated, functional analyses of these WGS can help to identify targeted strategies for the control of problematic microorganisms that may degrade metals and plastics, become potentially pathogenic and harm astronauts, or obstruct water supply due to their biofilm forming capabilities in the engineering systems of the closed habitat.', 'bibliography_description': 'Description: (Last Updated: 09/17/2025)', 'bibliography_articles': \"Ver 'Show Cumulative Bibliography'\"}, 'id': '11236', 'source_file': '11236.html', '_id': '40c024df-a06d-412c-a19f-75d3b0ed67a2', '_collection_name': 'tasks_proyects'}, page_content='May 2015 Revision to Task Agreement (Ed. note: information received September 2015).\\nDue to problems obtaining samples from Kibo-JEM (Japanese Experiment Module), the project has been modified with a new title, “Bacterial, Archaeal, & Fungal Diversity of the ISS--HEPA Filter System.” The revised scope will allow the Jet Propulsion Laboratory (JPL) to undertake a 28-month flight study to examine the microorganisms associated with HEPA filters. The ISS-HEPA project will develop a comprehensive census of the microbial population on the HEPA filter collected from various locations of the International Space Station (ISS), using genetic biosignatures and computational modeling. This microbial census will provide NASA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL team will determine which microbial taxa pose particular threats to crew health.\\nOriginal Task Description: Space microbiology research working groups in Japan, authorized by the Japan Aerospace Exploration Agency (JAXA), submitted proposals in response to the JAXA Announcement of Opportunity, 2012. The JAXA proposal that is managed by Principal Investigator Prof. Masao Nasu, Osaka University, contained several subgroups, including subgroup 4, headed by Dr. Takashi Yamazaki, JAXA. This subgroup 4 will conduct an investigation on “Analysis of microbial dynamics in the space habitation system by using new molecular microbial ecological techniques and also monitoring microbial contamination from air and surfaces of the ‘Kibo’ Japanese Experiment Module (Kibo-JEM).” The JAXA proposal submitted by subgroup 4 identifies the Jet Propulsion Laboratory (JPL) team headed by Dr. Kasthuri Venkateswaran as a collaborator to jointly explore the investigation.\\nThe Bacterial, Archaeal, and Fungal Diversity of Kibo-JEM Surfaces project (JPL-JAXA project) will undertake a three-year flight study to examine the changes in the microorganism population on the Kibo-JEM surfaces. The JPL-JAXA project proposes to develop a comprehensive census of the microbial population on the Kibo-JEM surfaces, using genetic biosignatures and computational modeling. This census will provide NASA and JAXA with the ability to accurately and confidently assess the status of microbes associated with closed habitation and crew health maintenance. In addition to overall microbial profiles, the JPL-JAXA team will determine which microbial taxa pose particular threats to crew health. Specific JPL-JAXA project goals will enable us to resolve applicable Human Research Program (HRP) integrated research plan (IRP) risks: (1) determine if changes in Kibo-JEM microbiome are occurring during a spaceflight mission and if so, (2) determine whether any changes noted would create a greater health risk for the crew.\\nThere are three aims proposed in this JPL-JAXA project and study of the Kibo-JEM microbiome. Through optimizations and comparative evaluations of methods designed to collect, process, analyze, and archive biomolecules sampled, the JPL-JAXA team will increase downstream sensitivity of laboratory assays as well as enhance the current understanding of the population dynamics of the Kibo-JEM module (Aim 1). The JPL-JAXA team will adapt molecular technologies currently in use with the National Institute of Health (NIH) – Human Microbiome Program (HMP) Roadmap—such as rRNA gene-variable region tag sequencing, whole genome shotgun sequencing (WGS), and DNA microarrays, as well as cloning and sequencing of 16S ribosomal DNA (rDNA)—to comprehensively elucidate the microbial diversity profiles associated with the Kibo-JEM surfaces (Aim 2). The final project deliverable to NASA will be the Kibo-JEM database, a compilation of all of the genomic sequences and genetic information of microbes encountered on and within the Kibo-JEM habitat (Aim 3). Computational modeling of the Kibo-JEM microbial population dynamics inferred from the database will then be carried out to predict the complete spectrum of organisms present.\\nThe JPL-JAXA project will compare microbial diversity, microbial characteristics, and functional capabilities of the microorganisms associated with the Kibo-JEM. The resulting database will enable HRP to evaluate the effect of spaceflight radiation and microgravity on the Kibo-JEM microbiome. The JPL-JAXA project in combination with results from studies of crew susceptibility will be used to determine the risk of microbiologically induced adverse health effects during a spaceflight mission. This assessment will allow HRP to determine if current operational and engineering controls used to mitigate microbiological risks are adequate or whether additional countermeasures should be developed.'),\n",
       " Document(metadata={'project_title': 'Microbial Evolution and Transmission Aboard the ISS: Inferring Mutation Rates, Assessing Pangenomes, and Tracking Microbiome Transmission between Astronauts and the Space-based Built EnvironmentReduce', 'images': '', 'fiscal_year': 'FY 2021', 'division': 'Space Biology', 'research_discipline/element': '', 'start_date': '09/13/2018', 'end_date': '07/31/2021', 'task_last_updated': '01/18/2022', 'research_discipline_element': 'Fiscal Year:FY 2021Division:Space BiologyResearch Discipline/Element:Space Biology: Cell & Molecular Biology | Microbiology', 'pdf_download_link': 'tbpdf.cfm?id=14460', 'principal_investigator': {}, 'project_information': {'responsible_center': 'NASA ARC', 'grant_monitor': 'Everroad, Craig (craig.everroad@nasa.gov)', 'center_contact': '650-604-4997', 'unique_id': '12112', 'solicitation__funding_source': '2016 Space Biology (ROSBio) NNH16ZTT001N-MOBE. Appendix B: Post-Doctoral Fellowships to Study the Microbiome of the ISS as a Built Environment', 'grantcontract_no': '80NSSC18M0110 (note funded under BAERI grant)', 'project_type': 'Ground', 'no_of_post_docs': '1', 'space_biology_element': '(1) Cell & Molecular Biology\\n(2) Microbiology', 'space_biology_cross-element_discipline': 'None', 'space_biology_special_category': 'None', 'flight_assignment/project_notes': 'NOTE: End date changed to 7/31/2021 per NSSC information (Ed., 6/15/20)', 'task_description': 'POSTDOCTORAL FELLOWSHIP\\nNOTE: This project is funded by NASA as a result of a Space Act Agreement between NASA and the Sloan Foundation, for Collaborative Research Proposal for Microbiology of the Built Environment (MoBE). The purpose of this Space Act Agreement is to enhance the ability of NASA and the Sloan Foundation to support scientific research and the dissemination of information about the effect of spaceflight on microbial ecosystems in spacecraft such as the International Space Station (ISS) through sponsorship of postdoctoral fellowships. NASA is providing the isolates for all studies.\\nOVERVIEW\\nThe proposed work will offer genetics-based avenues to gain insight into all 3 of the guiding questions as listed in this solicitation, but will directly address: “Does long-duration spaceflight alter normal rates of evolutionary change?”\\nWe propose to perform whole-genome sequencing of 50 isolates from the International Space Station (ISS) focused within the genera Staphylococcus and Ralstonia. These ISS-isolate genomes will be analyzed within the context of both an exhaustive, Earth-based reference genomic library constructed for each genus, and ISS astronaut microbiome metagenomic data. Co-Investigator Alexander Voorhies and collaborator Hernan Lorenzi are currently working with ‘before,’ ‘during,’ and ‘after’ microbiome metagenome from 9 ISS astronauts that were onboard the space station for various lengths of time spanning 2013–2016. Our unique position of being able to integrate microbiome sequencing data from another NASA effort with newly sequenced genomes of ISS isolates offers an unprecedented opportunity to explore many Space Biology evolutionary questions at the molecular level.\\nOBJECTIVES/METHODS\\nGenera Staphylococcus and Ralstonia were selected based on several criteria including ubiquity within human microbiota, potential pathogenicity, and their abundant representation within Earth-based reference databases, the ISS archive of available isolates, and our currently in-hand astronaut microbiome metagenomic data. We will investigate many metrics of molecular evolution aboard the ISS by directly addressing the following questions:\\n1) Are any particular genetic elements (i.e., genes/gene cassettes/operons) disproportionately represented in ISS isolates as compared to their Earth-based, nearest phylogenetic relatives?\\n2) Are there different propensities for specific individual genome mutations (i.e., single-point mutations such as substitutions/insertions/deletions, non-synonymous to synonymous substitution ratios, transition to transversion ratios, and genetic rearrangements) for organisms in the space-based built environment?\\n3) Which specific genes (and codons within these genes) are under positive, negative, or neutral selection aboard the ISS, but not within terrestrial references?\\n4) Can we track microbial transmission from an individual astronaut to the built environment, and/or the ISS serving as a vector and contributing to the microbiome of resident astronauts?\\n5) As assessed via comparison to metagenome-assembled genomes reconstructed from ISS-astronaut microbiomes, how representative are the isolate genomes recovered from the ISS of the actual microbiota of ISS astronauts?', 'research_impact/earth_benefits': 'SIGNIFICANCE\\nMicroorganisms are not only integral to human health, but are also inexorable and necessary components of any self-contained elemental cycling, food production, or water system, and longer-duration human space exploration will require all of these systems. To design safe, built-environment microbial ecosystems to support these life-support systems requires not only understanding short-term, “plastic” microbial responses to the space environment, but also longer-term propensities for microbial evolution and adaptation. This work will leverage newly developed genetics-based computational tools, technologies, and approaches to generate the first comprehensive, whole-genomic database of microorganisms isolated from the space-based built environment while addressing fundamental questions about evolution aboard the ISS.'}, 'task_progress_and_bibliography': {'task_progress': 'Update: The primary work from this time period is described in our publication “Reference-guided metagenomics reveals genome-level evidence of potential microbial transmission from the ISS environment to an astronaut’s microbiome” (Lee et al. 2021). Using assembled genomes of our ISS-derived microbial isolates as references and recruiting metagenomic reads from an astronaut’s nasal microbiome revealed no recruitment to a Staphylococcus aureus isolate from samples before launch, yet systematic recruitment across the genome when sampled after 3 months aboard the ISS, with a median percent identity of 100%. This suggests that either a highly similar S. aureus population colonized the astronaut’s nasal microbiome while the astronaut was aboard the ISS or that it may have been below detection before spaceflight, instead supporting a shift in community composition. This work highlights the value in generating genomic libraries of microbes from built-environments such as the ISS and demonstrates one way such data can be integrated with metagenomics to facilitate the tracking and monitoring of astronaut microbiomes and health.\\nLee et al. (2021). Reference-guided metagenomics reveals genome-level evidence of potential microbial transmission from the ISS environment to an astronaut’s microbiome. iScience.\\nhttps://doi.org/10.1016/j.isci.2021.102114\\n(see also Bibliography section below)\\nWe are still working on a publication focusing on the genomics and physiology of Staphylococcus isolates, similar to the publication focused on Burkholderia, previously reported in FY2020 report (O\\'Rourke A et al. \"Genomic and phenotypic characterization of Burkholderia isolates from the potable water system of the International Space Station.\" PLoS One. 2020 Feb 20;15(2):e0227152.\\nhttps://doi.org/10.1371/journal.pone.0227152', 'bibliography_description': 'Description: (Last Updated: 01/19/2022)', 'bibliography_articles': \"Ver 'Show Cumulative Bibliography'\"}, 'id': '12405', 'source_file': '12405.html', '_id': 'd765949e-3d1d-4783-87c7-32df67494b29', '_collection_name': 'tasks_proyects'}, page_content='POSTDOCTORAL FELLOWSHIP\\nNOTE: This project is funded by NASA as a result of a Space Act Agreement between NASA and the Sloan Foundation, for Collaborative Research Proposal for Microbiology of the Built Environment (MoBE). The purpose of this Space Act Agreement is to enhance the ability of NASA and the Sloan Foundation to support scientific research and the dissemination of information about the effect of spaceflight on microbial ecosystems in spacecraft such as the International Space Station (ISS) through sponsorship of postdoctoral fellowships. NASA is providing the isolates for all studies.\\nOVERVIEW\\nThe proposed work will offer genetics-based avenues to gain insight into all 3 of the guiding questions as listed in this solicitation, but will directly address: “Does long-duration spaceflight alter normal rates of evolutionary change?”\\nWe propose to perform whole-genome sequencing of 50 isolates from the International Space Station (ISS) focused within the genera Staphylococcus and Ralstonia. These ISS-isolate genomes will be analyzed within the context of both an exhaustive, Earth-based reference genomic library constructed for each genus, and ISS astronaut microbiome metagenomic data. Co-Investigator Alexander Voorhies and collaborator Hernan Lorenzi are currently working with ‘before,’ ‘during,’ and ‘after’ microbiome metagenome from 9 ISS astronauts that were onboard the space station for various lengths of time spanning 2013–2016. Our unique position of being able to integrate microbiome sequencing data from another NASA effort with newly sequenced genomes of ISS isolates offers an unprecedented opportunity to explore many Space Biology evolutionary questions at the molecular level.\\nOBJECTIVES/METHODS\\nGenera Staphylococcus and Ralstonia were selected based on several criteria including ubiquity within human microbiota, potential pathogenicity, and their abundant representation within Earth-based reference databases, the ISS archive of available isolates, and our currently in-hand astronaut microbiome metagenomic data. We will investigate many metrics of molecular evolution aboard the ISS by directly addressing the following questions:\\n1) Are any particular genetic elements (i.e., genes/gene cassettes/operons) disproportionately represented in ISS isolates as compared to their Earth-based, nearest phylogenetic relatives?\\n2) Are there different propensities for specific individual genome mutations (i.e., single-point mutations such as substitutions/insertions/deletions, non-synonymous to synonymous substitution ratios, transition to transversion ratios, and genetic rearrangements) for organisms in the space-based built environment?\\n3) Which specific genes (and codons within these genes) are under positive, negative, or neutral selection aboard the ISS, but not within terrestrial references?\\n4) Can we track microbial transmission from an individual astronaut to the built environment, and/or the ISS serving as a vector and contributing to the microbiome of resident astronauts?\\n5) As assessed via comparison to metagenome-assembled genomes reconstructed from ISS-astronaut microbiomes, how representative are the isolate genomes recovered from the ISS of the actual microbiota of ISS astronauts?'),\n",
       " Document(metadata={'project_title': 'Virulence and Drug Resistance of Burkholderia Species Isolated from ISS Potable Water SystemsReduce', 'fiscal_year': 'FY 2019', 'division': 'Space Biology', 'research_discipline/element': '', 'start_date': '06/06/2017', 'end_date': '09/01/2019', 'task_last_updated': '01/24/2020', 'research_discipline_element': 'Fiscal Year:FY 2019Division:Space BiologyResearch Discipline/Element:Space Biology: Microbiology', 'pdf_download_link': 'tbpdf.cfm?id=12241', 'principal_investigator': {}, 'project_information': {'responsible_center': 'NASA KSC', 'grant_monitor': 'Levine, Howard (howard.g.levine@nasa.gov)', 'center_contact': '321-861-3502', 'unique_id': '11527', 'solicitation__funding_source': '2016 Space Biology (ROSBio) NNH16ZTT001N-MOBE. Appendix B: Post-Doctoral Fellowships to Study the Microbiome of the ISS as a Built Environment', 'grantcontract_no': '80NSSC17K0035', 'project_type': 'Ground', 'no_of_post_docs': '1', 'no_of_phd_degrees': '1', 'space_biology_element': '(1) Microbiology', 'space_biology_cross-element_discipline': 'None', 'space_biology_special_category': 'None', 'flight_assignment/project_notes': 'NOTE: End date changed to 9/1/2019 per NSSC information (Ed., 1/24/2020)', 'task_description': 'POSTDOCTORAL FELLOWSHIP\\nNOTE: This project is funded by NASA as a result of a Space Act Agreement between NASA and the Sloan Foundation, for Collaborative Research Proposal for Microbiology of the Built Environment (MoBE). The purpose of this Space Act Agreement is to enhance the ability of NASA and the Sloan Foundation to support scientific research and the dissemination of information about the effect of spaceflight on microbial ecosystems in spacecraft such as the International Space Station (ISS) through sponsorship of postdoctoral fellowships. NASA is providing the isolates for all studies.\\nThe disease-causing organisms Burkholderia multivorans, B. cepacia, genomovars of the B. cepacia complex (BCC), as well as an unclassified species of Burkholderia, and its relatives Ralstonia pickettii and R. insidiosa account for 60% of the bacterial isolates obtained from the filtrated water system on board the International Space Station (ISS). Members of these genera can survive long periods in a planktonic state in distilled water or as biofilms within an infected individual. They are also natively resistant to several antibiotics leaving treatment options complex. For BCC organisms, this complexity is increased by the phenomenon of bacterial persistence where a genetically identical subpopulation of the bacteria sustains under high antibiotic challenge by a reduction in metabolism and other unknown mechanisms. A healthy astronaut could easily clear a BCC or Ralstonia sp. infection; however, after prolonged spaceflight, a potentially immunocompromised astronaut with repeated exposure to these organisms could be at risk for disease.\\nTo understand the mechanisms that would contribute to altered virulence in ISS BCC and Ralstonia species, the genomes of 67 isolates from 22 missions will be sequenced and compared to their respective ATCC BCC or Ralstonia strain. The potential for increased virulence will be determined by the presence of genes clustered as pathogenicity islands in the respective genome. To establish if the strains are becoming more resistant to antibiotics, a minimum inhibitory concentration (MIC) will be determined for each isolate using six antibiotics currently used to treat the infections and then compared to the respective ATCC control strain. Finally, a transposon library will be generated for the most virulent, resistant, tolerant, or persistent strain, and this library will be cultured in distilled water to simulate conditions on the ISS. Here we will identify genes which aid in the survivorship of BCC and Ralstonia strains in the water system of the ISS. The knowledge gained from this genomic sequencing and Tnseq study will allow us to assess any increased virulence or infection potential, as well as help to make antibiotic recommendations, predict target genes where inhibition by adjuvants will enhance antibiotic therapy, or present novel water treatment options to eradicate the BCC/Ralstonia strains from the potable water of the ISS.', 'research_impact/earth_benefits': 'The overall goal of the project # 80NSSC17K0035 entitled, “Virulence and drug resistance of Burkholderia species isolated from ISS potable water systems” is to use “omic” studies to characterize the Burkholderia cepacia complex (BCC) and Ralstonia species that have been isolated from the potable water dispenser (PWD) of International Space Station (ISS) across 22 missions. The PWD is the source of all consumed water aboard the ISS; therefore, the surveillance of these bacterial species, which are known to be opportunistic pathogens, is relevant to ensure crew health. These genera are commonly found as soil organisms with the ability to survive in water. As a result, they are often found as contaminants in treated water supplies. These bacterial isolates cultured from the ISS PWD have originated from Earth. The effects upon them, as the result of exposure to microgravity, lowered shear forces, and increased radiation in the Earth’s upper atmosphere where the ISS orbits, are unknown. Our work benefits life on Earth by way of planetary protection, a facet of NASA since the start of the Apollo missions. The goal of planetary protection is to prevent any forward contamination to other celestial bodies as well as to prevent any back contamination to Earth. Our work will shed light on new ways to contain or treat these genera isolated form the ISS PWD, as well as to provide a better understanding of the structure and plasticity within the genomes of these space-adapted terrestrial bacteria.'}, 'task_progress_and_bibliography': {'task_progress': 'FINAL REPORTING JANUARY 2020\\nThe overall goal of the project # 80NSSC17K0035 is to use “omic” studies to characterize the Burkholderia cepacia complex (BCC) and Ralstonia species that have been isolated from the potable water dispenser (PWD) of International Space Station (ISS). The PWD is the source of all consumed water aboard the ISS; therefore, the surveillance of these species, which are known to be opportunistic pathogens, is relevant to ensure crew health. The isolates were evaluated for any increase in virulence or infection potential and assayed for antibiotic resistance to address. The genomes and phenotypic results for the ISS BCC isolates have now been written up and the manuscript has been accepted by PLOS One as of December 16, 2019.\\nIn Year 2, a transposon library was successfully constructed in an ISS Burkholderia contaminans isolate. However, we were unable to run the experiment due to the elemental iodine selected for treatment being a controlled pre-cursor substance in the state of California; unfortunately, the experiment has been on hold due to the inability to procure iodine.\\nOverall our research shows that within each species, the 19 B. cepacia and 5 B. contaminans, recovered from the ISS PWD over roughly a 5-year timespan, were highly similar on a whole genome scale, suggesting each population (here meaning one for each species) that was sampled may have stemmed from two distinct founding strains. This is supported by all isolates demonstrating greater than 99% ANI with 95-99.9% alignment in all cases, and sharing a core pangenome of greater than 90% of identified gene clusters (within species). The vast majority of differences in gene-content that were identified in the pangenome were localized on putative plasmids. Physiologically, including antibiotic sensitivities, the ISS-derived isolates were overall generally similar to each other, as well as to the reference type-strains analyzed. In cases where ISS-derived isolates did vary, no strong correlations were found between physiology and either single-nucleotide variant (SNV) distances or date of isolation (though it is also import to keep in mind date of isolation does not necessarily indicate any information about how long an isolate had been aboard the ISS). While it is interesting the 5 recovered B. contaminans isolates were all isolated in a span of time over 2013 and 2014 during which no B. cepacia were isolated, we cannot speak at this time to what may have caused this (if anything). That aside, our analysis here suggests these two species’ populations sampled by way of isolation seem to be highly similar genetically regardless of date of isolation, with the majority of variation that does exist being maintained among putative plasmids. Based on our phenotypic evaluation of the ability of these isolates to lyse or intracellularize within macrophage, in comparison to the reference type-strains, it seems likely that the two populations of Burkholderia present in the ISS PWS are not more virulent than those that might be encountered on planet.\\nCompleted Publications\\n1.\\tAubrie O’Rourke, Michael D. Lee, William C. Nierman, R. Craig Everroad, Chris L. Dupont. Genomic and phenotypic characterization of Burkholderia isolates from the potable water system of the International Space Station. PLOSONE. Accepted and in final stages before publication.\\n2.\\tAubrie O’Rourke, Angela Zoumplis, Paul Wilburn, Michael D. Lee, Zhi Lee, Marissa Vecina, Kysha Mercader. Following the Astrobiology Roadmap: Origins, Habitability and Future Exploration. Astrobiology for Caister Academic Press and as a Focus Issue of the journal Current Issues in Molecular Biology. Accepted and published.\\nANNUAL REPORTING APRIL 2019\\nAim 1: To determine the potential for enhanced virulence and pathogenicity of Burkholderia cepacia complex (BCC) and Ralstonia strains isolated from the filtration unit of the ISS by genomic sequencing.\\nUnder Aim 1, to determine the potential for enhanced virulence and pathogenicity of BCC and Ralstonia strains isolated from the filtration unit of the ISS, the hypothesis follows that the sequencing and analysis of the 67 genomes of these two genera from 22 missions and their comparison to the genomic sequences of the respective ATCC BCC and Ralstonia strain controls will reveal known and novel pathogenicity islands and integrative conjugative elements (ICEs). The genes contained within these mobile genetic elements (MGEs) will allow us to gauge to what degree the presence of the genera in the ISS filtrated water systems pose a human health threat via their infection potential.\\nAim 1 objectives are complete; we have successfully sequenced all genomes and analyzed the dataset. We find that the ISS isolates have genomes that are distinct from that of terrestrial isolates and are set apart by the genes harbored on conjugative plasmids. Furthermore, among the ISS isolates plasmid composition also varies. We hypothesize that while the isolates exhibit varying degrees of genomic erosion, such erosion is mitigated by the presence of conjugative plasmids which can rescue a reduced genome when facing environmental stress. In the scenario of the ISS PWD, the isolates may form biofilms, as we have displayed in year 1 that Burkholderia species have the propensity to do so. Plasmids may be shuffled among the biofilm or aid planktonic isolates in overcoming the iodine challenges used to decontaminate the system. Year 2 transposon sequencing will shed light on whether or not this mechanism of plasmid shuffling is used to counter disinfection.\\nAim 2: To determine antibiotic persistence of BCC and Ralstonia strains isolated from the filtration unit of the ISS using Tnseq.\\nUnder Aim 2, performed in order to determine how the BCC isolates persist in the potable water system of the ISS, we first hypothesized that ISS strains, which have been placed under the selective pressure of the ISS water systems oligotrophic environment, will experience varying degrees of antibiotic tolerance due to this previous selective pressure. We found that their minimum inhibitory concentrations (MICs) do not differ from that of the terrestrial control under our antibiotics tested. However, a Tnseq analysis will reveal the mechanisms, including virulence factors that contribute to the ability of the BCC and Ralstonia strains to persist in the oligotrophic filtrated ISS water system. We hypothesize that genes conferred by conjugative plasmids aid in isolate survival of these bacterial species within the water system.\\nThe Year 2 goal for Aim 2 will include generating and assessing the gene survivorship index of a representative transposon library subject to distilled water conditions to simulate the environment of the ISS potable water system. In a Tnseq study, I will determine the genes associated with aiding in the maintenance of BCC bacteria in this ISS filtrated water system. Here a representative strain of BCC, which exhibits the greatest reduction in antibiotic susceptibility or tolerance will be selected for transposon mutagenesis. This technique of introducing a transposon to generate a library of insertional mutants and subsequent culturing of the library under a selective condition is known as Tnseq. The ISS specimen Tnseq library will be cultured in distilled water simulating the environmental conditions of the ISS water system. Subsequent sequencing analysis will allow us to determine which genes, including virulence factors, play a key role in the maintenance of the genera in the water systems of the ISS. All data will be made public and results will be published.\\nAim 2 objectives are 50% complete--we have successfully profiled all isolates for their antibiotic susceptibility. We are currently optimizing the Tnseq experiment. We expect to observe the genes which when mutated increase and detract from survival under distilled water treatment over time. We expect that the addition of iodine at 8 ppm or 40 ppm will alter this gene set in relation to the distilled water control. This comparison will reveal the mechanism used by the isolates to persist in the PWD despite disinfection treatment which is used to reduce the microbial load after stagnation in the ISS potable water system. We reason that if the genes conferred by plasmids are beneficial to overcome stress, then the mutants of these genes in the Tnseq library will be less represented in the iodine treated samples and therefore have a lower survival index than in the distilled water sample.', 'bibliography_description': 'Description: (Last Updated: )', 'bibliography_articles': \"Ver 'Show Cumulative Bibliography'\"}, 'id': '12241', 'source_file': '12241.html', '_id': 'f9d0ea22-9d64-44dd-950b-bdd7d3acd66d', '_collection_name': 'tasks_proyects'}, page_content='POSTDOCTORAL FELLOWSHIP\\nNOTE: This project is funded by NASA as a result of a Space Act Agreement between NASA and the Sloan Foundation, for Collaborative Research Proposal for Microbiology of the Built Environment (MoBE). The purpose of this Space Act Agreement is to enhance the ability of NASA and the Sloan Foundation to support scientific research and the dissemination of information about the effect of spaceflight on microbial ecosystems in spacecraft such as the International Space Station (ISS) through sponsorship of postdoctoral fellowships. NASA is providing the isolates for all studies.\\nThe disease-causing organisms Burkholderia multivorans, B. cepacia, genomovars of the B. cepacia complex (BCC), as well as an unclassified species of Burkholderia, and its relatives Ralstonia pickettii and R. insidiosa account for 60% of the bacterial isolates obtained from the filtrated water system on board the International Space Station (ISS). Members of these genera can survive long periods in a planktonic state in distilled water or as biofilms within an infected individual. They are also natively resistant to several antibiotics leaving treatment options complex. For BCC organisms, this complexity is increased by the phenomenon of bacterial persistence where a genetically identical subpopulation of the bacteria sustains under high antibiotic challenge by a reduction in metabolism and other unknown mechanisms. A healthy astronaut could easily clear a BCC or Ralstonia sp. infection; however, after prolonged spaceflight, a potentially immunocompromised astronaut with repeated exposure to these organisms could be at risk for disease.\\nTo understand the mechanisms that would contribute to altered virulence in ISS BCC and Ralstonia species, the genomes of 67 isolates from 22 missions will be sequenced and compared to their respective ATCC BCC or Ralstonia strain. The potential for increased virulence will be determined by the presence of genes clustered as pathogenicity islands in the respective genome. To establish if the strains are becoming more resistant to antibiotics, a minimum inhibitory concentration (MIC) will be determined for each isolate using six antibiotics currently used to treat the infections and then compared to the respective ATCC control strain. Finally, a transposon library will be generated for the most virulent, resistant, tolerant, or persistent strain, and this library will be cultured in distilled water to simulate conditions on the ISS. Here we will identify genes which aid in the survivorship of BCC and Ralstonia strains in the water system of the ISS. The knowledge gained from this genomic sequencing and Tnseq study will allow us to assess any increased virulence or infection potential, as well as help to make antibiotic recommendations, predict target genes where inhibition by adjuvants will enhance antibiotic therapy, or present novel water treatment options to eradicate the BCC/Ralstonia strains from the potable water of the ISS.'),\n",
       " Document(metadata={'project_title': 'Developing Predictive Model Systems of Polymicrobial Biofilm Formation and Susceptibility to Chemical Disinfectant: A Longitudinal Study with Implications for Spaceflight Systems Integrity and Health RisksReduce', 'fiscal_year': 'FY 2020', 'division': 'Space Biology', 'research_discipline/element': '', 'start_date': '06/01/2017', 'end_date': '06/30/2020', 'task_last_updated': '09/27/2024', 'research_discipline_element': 'Fiscal Year:FY 2020Division:Space BiologyResearch Discipline/Element:Space Biology: Microbiology', 'pdf_download_link': 'tbpdf.cfm?id=12801', 'principal_investigator': {}, 'project_information': {'responsible_center': 'NASA HQ', 'grant_monitor': 'Wolf, Yolanda (wolf@sloan.org)', 'unique_id': '11456', 'solicitation__funding_source': '2016 SLOANMOBE2016. Sloan ISS MoBE Postdoctoral Fellowship - Post-Doctoral Fellowships in Space Biology to Study the Microbiome of the ISS as a Built Environment: Using ISS as a Microbiological Observatory', 'grantcontract_no': 'Sloan P-2017-9852', 'project_type': 'Ground', 'no_of_post_docs': '1', \"no_of_master's_candidates\": '1', 'space_biology_element': '(1) Microbiology', 'space_biology_cross-element_discipline': 'None', 'space_biology_special_category': 'None', 'flight_assignment/project_notes': 'NOTE: End date changed to 6/30/2020 per Y. Wolf/Sloan Foundation; original end date was 5/31/2019 (Ed., 4/6/2020)\\nNOTE: End date changed to 12/31/2019 per Y. Wolf/Sloan Foundation; original end date was 5/31/2019 (Ed., 11/18/19)', 'task_description': 'POSTDOCTORAL FELLOWSHIP\\nNOTE: This project is funded by the Alfred P. Sloan Foundation, as a result of a Space Act Agreement between NASA and the Sloan Foundation, for Collaborative Research Proposal for Microbiology of the Built Environment (MoBE). The purpose of this Space Act Agreement is to enhance the ability of NASA and the Sloan Foundation to support scientific research and the dissemination of information about the effect of spaceflight on microbial ecosystems in spacecraft such as the International Space Station (ISS) through sponsorship of postdoctoral fellowships. NASA is providing the isolates for all studies.\\nUnderstanding the impact of spaceflight on microbial ecosystem dynamics and biofilm formation is critical for spacecraft design, operations, and astronaut health. Biofilms are microbial communities that that adhere to surfaces, causing materials degradation, operational systems failure, and serious infections -- and are resistant to disinfectants/antimicrobials. Biofilms have been reported on spacecraft materials associated with regenerative life support systems, including the International Space Station (ISS) potable water system, which presents a serious threat to onboard systems operation and crew health. The ISS is a built environment that has housed over 200 international crew members and their accompanying microbes since 2000. As such, it is characterized by its own microbiome that has been consistently exposed and adapted to the unique low fluid shear environment of microgravity, which is known to cause unexpected microbial responses, and may present a selective pressure that could shape the adaptation and evolution of these microorganisms. Many ISS microbial isolates are known to be biofilm formers and naturally exist in multispecies ecosystems. While a variety of microbial isolates have been recovered from the ISS potable water over the years, an understanding of how adaptation to spaceflight impacts the formation and function of these organisms as mixed polymicrobial communities in their normal ISS habitat is not known. Likewise, there is extremely limited understanding of how these microbial communities impact spaceflight systems operation and integrity (including regenerative life support systems) and risk to human health.\\nA limited number of spaceflight and RWV (rotating wall vessel) studies have shown that single microbial species form biofilms, with evidence suggesting unique biofilm architecture in microgravity. However, biofilms naturally exist as consortia of multiple microbial species, and there is extremely limited knowledge of how multiple bacterial species form polymicrobial communities in ISS liquid environments, or how these microbial communities present in ISS water systems have changed over time under the selective pressures of the microgravity environment. To begin to address this gap, we propose to develop a series of independent ISS potable water microbial community model systems using predominant bacterial isolates obtained in the same year, and compare how adaptation to the ISS environment in each of these water-microbiome systems has impacted their interactions, ability to form biofilms, and resistance to a commonly used chemical disinfectant aboard the ISS. The archived microbial isolates proposed for use have been collected from the ISS potable water system during three specific years over a six-year period. Microbial isolates from the same year will be simultaneously co-cultured as mixed microbial communities in the NASA Rotating Wall Vessel (RWV) bioreactor as a spaceflight analogue culture system. The proposed methods to accomplish the research objectives include: real-time video (to monitor polymicrobial growth and biofilm formation in RWVs); use of selective media (to monitor viable population dynamics within the polymicrobial communities and susceptibility to chemical disinfectants); and various staining techniques in combination with microscopy (to investigate biofilm formation/architecture).\\nWe hypothesize that polymicrobial interactions within the ISS water microbiome cultured in ground-based spaceflight analogue conditions will present distinct differences in biofilm formation, architecture, population dynamics and resistance to disinfectant.', 'research_impact/earth_benefits': 'This research will enrich life on Earth through the use of space technology and the application of that knowledge to understand microbial risks to systems operation and integrity of potable water systems, as well as to human health.  Specifically, this study will utilize the spaceflight analogue platform to broaden our knowledge of 1) how microbial populations cause biofouling of potable water systems, and 2) for the development of new strategies to prevent microbial biofouling, maintain function, and ensure public health.'}, 'task_progress_and_bibliography': {'task_progress': 'September 2024 update: Please see the Cumulative Bibliography for peer-reviewed publications that describe this research, as well as prior year NASA Task Book reports (Ed., 9/27/24).', 'bibliography_description': 'Description: (Last Updated: 09/02/2025)', 'bibliography_articles': \"Ver 'Show Cumulative Bibliography'\"}, 'id': '12801', 'source_file': '12801.html', '_id': '53dcceaa-3df6-4d20-99e7-d76ea2df34c3', '_collection_name': 'tasks_proyects'}, page_content='POSTDOCTORAL FELLOWSHIP\\nNOTE: This project is funded by the Alfred P. Sloan Foundation, as a result of a Space Act Agreement between NASA and the Sloan Foundation, for Collaborative Research Proposal for Microbiology of the Built Environment (MoBE). The purpose of this Space Act Agreement is to enhance the ability of NASA and the Sloan Foundation to support scientific research and the dissemination of information about the effect of spaceflight on microbial ecosystems in spacecraft such as the International Space Station (ISS) through sponsorship of postdoctoral fellowships. NASA is providing the isolates for all studies.\\nUnderstanding the impact of spaceflight on microbial ecosystem dynamics and biofilm formation is critical for spacecraft design, operations, and astronaut health. Biofilms are microbial communities that that adhere to surfaces, causing materials degradation, operational systems failure, and serious infections -- and are resistant to disinfectants/antimicrobials. Biofilms have been reported on spacecraft materials associated with regenerative life support systems, including the International Space Station (ISS) potable water system, which presents a serious threat to onboard systems operation and crew health. The ISS is a built environment that has housed over 200 international crew members and their accompanying microbes since 2000. As such, it is characterized by its own microbiome that has been consistently exposed and adapted to the unique low fluid shear environment of microgravity, which is known to cause unexpected microbial responses, and may present a selective pressure that could shape the adaptation and evolution of these microorganisms. Many ISS microbial isolates are known to be biofilm formers and naturally exist in multispecies ecosystems. While a variety of microbial isolates have been recovered from the ISS potable water over the years, an understanding of how adaptation to spaceflight impacts the formation and function of these organisms as mixed polymicrobial communities in their normal ISS habitat is not known. Likewise, there is extremely limited understanding of how these microbial communities impact spaceflight systems operation and integrity (including regenerative life support systems) and risk to human health.\\nA limited number of spaceflight and RWV (rotating wall vessel) studies have shown that single microbial species form biofilms, with evidence suggesting unique biofilm architecture in microgravity. However, biofilms naturally exist as consortia of multiple microbial species, and there is extremely limited knowledge of how multiple bacterial species form polymicrobial communities in ISS liquid environments, or how these microbial communities present in ISS water systems have changed over time under the selective pressures of the microgravity environment. To begin to address this gap, we propose to develop a series of independent ISS potable water microbial community model systems using predominant bacterial isolates obtained in the same year, and compare how adaptation to the ISS environment in each of these water-microbiome systems has impacted their interactions, ability to form biofilms, and resistance to a commonly used chemical disinfectant aboard the ISS. The archived microbial isolates proposed for use have been collected from the ISS potable water system during three specific years over a six-year period. Microbial isolates from the same year will be simultaneously co-cultured as mixed microbial communities in the NASA Rotating Wall Vessel (RWV) bioreactor as a spaceflight analogue culture system. The proposed methods to accomplish the research objectives include: real-time video (to monitor polymicrobial growth and biofilm formation in RWVs); use of selective media (to monitor viable population dynamics within the polymicrobial communities and susceptibility to chemical disinfectants); and various staining techniques in combination with microscopy (to investigate biofilm formation/architecture).\\nWe hypothesize that polymicrobial interactions within the ISS water microbiome cultured in ground-based spaceflight analogue conditions will present distinct differences in biofilm formation, architecture, population dynamics and resistance to disinfectant.'),\n",
       " Document(metadata={'project_title': 'Spaceflight Effects on Bacterial Antibiotic Resistance and Its Genetic Basis “AntimicrobialSat”Reduce', 'fiscal_year': 'FY 2019', 'division': 'Space Biology', 'research_discipline/element': '', 'start_date': '07/01/2010', 'end_date': '12/31/2018', 'task_last_updated': '05/22/2019', 'research_discipline_element': 'Fiscal Year:FY 2019Division:Space BiologyResearch Discipline/Element:Space Biology: Cell & Molecular Biology | Microbiology', 'pdf_download_link': 'tbpdf.cfm?id=12284', 'principal_investigator': {}, 'project_information': {'responsible_center': 'NASA ARC', 'grant_monitor': 'Sato, Kevin (kevin.y.sato@nasa.gov)', 'center_contact': '650-604-1104', 'unique_id': '8002', 'solicitation__funding_source': '2008 SALMON--Astrobiology & FSB NNH08ZDA009O-SCMAFSB', 'grantcontract_no': '80NSSC17M0006 ; NNX10AM90A', 'project_type': 'Flight', 'flight_program': 'Small Satellites', 'no_of_post_docs': '4', 'no_of_phd_candidates': '1', \"no_of_master's_candidates\": '0', \"no_of_bachelor's_candidates\": '0', 'no_of_phd_degrees': '1', \"no_of_master's_degrees\": '0', \"no_of_bachelor's_degrees\": '0', 'space_biology_element': '(1) Cell & Molecular Biology\\n(2) Microbiology', 'space_biology_cross-element_discipline': '(1) Developmental Biology', 'space_biology_special_category': '(1) Translational (Countermeasure) Potential', 'flight_assignment/project_notes': 'EcAMSat\\nNOTE: End date change to 12/31/2018 per F. Hernandez/ARC (Ed., 6/26/18)\\nNOTE: End date changed to 6/30/2018 with new grant number 80NSSC17M0006 in August 2017 (Ed., 5/22/18)\\nNOTE: End date changed to 04/30/2017 per F. Hernandez/ARC and NSSC information (Ed., 4/12/17)\\nNOTE: End date changed to 3/31/2017 per PI and NSSC information (Ed., 6/13/16)\\nNOTE: End date changed to 3/31/2016 per NSSC information (Ed., 1/20/16)\\nNOTE: End date changed to 11/30/2015 per NSSC information and A. Chu/ARC (Ed., 12/16/14)\\nNOTE: New end date is 11/30/2014 per NSSC information (Ed., 4/4/14)\\nNOTE: New end date is 4/30/2014 per A. Chu/ARC and NSSC (Ed., 2/21/2013)', 'task_description': 'This project has accomplished one of its two milestones, and the updated summary provided now reflects not only the conceptual genesis of the project, but also as it was refined as the project progressed. While, of course, the original essence remains, what is presented here is the summit of the first milestone, and precedes previous reports. This milestone is concerned with Earth studies to develop and rigorously test the technology for determining the sensitivity of uropathogenic Escherichia coli (UPEC) to the antibiotic gentamicin (Gm) during spaceflight, and to elucidate the genetic basis of whatever change is seen. The second milestone is experimentation during the spaceflight itself, which was launched in late 2017.\\nA paper entitled: “Payload hardware and experimental protocol development to enable future testing of the effect of space microgravity on the resistance to gentamicin of uropathogenic Escherichia coli and its (sigma)s-deficient mutant,” was published online in May 2017: Life Sciences in Space Research 15 (2017) 1–10 2214-5524/ © 2017 The Committee on Space Research (COSPAR). Published by Elsevier Ltd.; [\\nhttp://dx.doi.org/10.1016/j.lssr.2017.05.001\\n] As has been pointed out before, this project represents highly collaborative and collegial collaboration between Stanford and NASA Ames Research Center at Moffett Field. The published paper reflects this not only by the joint authorship of it by both Stanford and Moffett Field scientists, but also the benefits that accrue from collaboration between biological expertise (Stanford) and the flight technological expertise (Moffett Field).\\nThe list of authors is as follows: AC Matin, J-H Wang, Mimi Keyhan, Rachna Singh, and Michael Benoit of the Department of Microbiology & Immunology at Stanford School of Medicine; and Macarena P. Parra, Michael R. Padgen, Antonio J. Ricco, Matthew Chin, Charlie R. Friedericks, Tori N. Chinn, Aaron Cohen, Michael B. Henschke, Timothy V. Snyder, Matthew P. Lera, Shannon S. Ross, Christina M. Mayberry, Sungshin Choi, Diana T. Wu, Ming X. Tan, Travis D. Boone, Christopher C. Beasley, and Stevan M. Spremo, of NASA Ames Research Center, Moffett Field, CA 94035, USA.   The abstract of this paper, reproduced here (in quotes), succinctly describes the evolved conceptual background, and demonstrates in detail the development of the EcAMSat platform, and its thorough testing using spaceflight-like commands.\\n“Human immune response is compromised and bacteria can become more antibiotic resistant in space microgravity (MG). We report that under low-shear modeled microgravity (LSMMG), stationary-phase uropathogenic Escherichia coli (UPEC) become more resistant to gentamicin (Gm), and that this increase is dependent on the presence of s (a transcription regulator encoded by the rpoS gene). UPEC causes urinary tract infections (UTIs), reported to afflict astronauts; Gm is a standard treatment, so these findings could impact astronaut health. Because LSMMG findings can differ from MG, we report preparations to examine UPEC’s Gm sensitivity during spaceflight using the E. coli Anti-Microbial Satellite (EcAMSat) on a free-flying “nanosatellite” in low Earth orbit. Within EcAMSat’s payload, a 48-microwell fluidic card contains and supports study of bacterial cultures at constant temperature; optical absorbance changes in cell suspensions were made at three wavelengths for each microwell and a fluid-delivery system provided growth medium and predefined Gm concentrations. Performance characterization is reported here for spaceflight prototypes of this payload system. Using conventional microtiter plates, we show that Alamar Blue (AB) absorbance changes can assess the Gm effect on E. coli viability, permitting telemetric transfer of the spaceflight data to Earth. Laboratory results using payload prototypes are consistent with wellplate and flask findings of differential sensitivity of UPEC and its delta rpoS strain to Gm. if (sigma)s plays the same role in space MG as in LSMMG and Earth gravity, countermeasures discovered in recent Earth studies (aimed at weakening the UPEC antioxidant defense) to control UPEC infections would prove useful also in space flights. Further, EcAMSat results should clarify inconsistencies from previous space experiments on bacterial antibiotic sensitivity and other issues.” The manuscript may be consulted for further details and all the relevant references; see Bibliography section in 2017 report.\\nIn addition, the work has brought to light issues, such as the stress exerted by the flight equipment on bacteria, which we are now in the process of elucidating and ameliorating for the development of better future platforms for spaceflight experimentation on bacterial biology, resistance, and virulence.', 'research_impact/earth_benefits': 'Antibiotic resistance in bacteria is becoming an extremely serious problem on Earth that impacts space travelers harder given the deleterious effect of microgravity on human immune response. The research has confirmed the hypothesis that underpinned it, and has shown the way to enhance effectiveness of antibiotics in general. Subsequent work, which is in progress, has identified compounds whose co-administration with antibiotics will enhance their curative power; these can also permit lowering the needed amount of antibiotics for treating infections, thereby minimizing their side effects. This promises to improve life in space as well as on Earth.'}, 'task_progress_and_bibliography': {'task_progress': 'The spaceflight took place onboard a free-flying “nanosatellite,” which, for the first time utilized a 6-U configuration (a “U” being a 10-cm cube). The experiments were conducted on the EcAMSat hardware platform that we developed and described previously in the task book reports. Cell viability in this platform was measured by changes in the absorption of the dye Alamar Blue (AB); these were telemetered to Earth and recorded by daily communications with the spacecraft at Santa Clara University. Parallel ground controls were run. The flight performed very smoothly and all the needed data were successfully obtained. The major conclusion from the results is that the gene rpoS continues to play a critical role in making urinary pathogenic Escherichia coli (UPEC) highly resistant to the antibiotic gentamicin (Gm). This antibiotic is used to treat urinary tract infections often reported in astronauts. Therefore, the potential small molecule inhibitors that we have identified (using bioinformatics approaches) of the protein encoded by rpoS and the antioxidant proteins it controls hold the promise to promote astronaut health in spaceflights. That these molecules are valuable also in meeting the rampant bacterial antibiotic resistance on Earth has been pointed out before.\\nA second important finding is that the high-aspect-ratio vessels (HARVs) used to simulate microgravity on Earth did indeed provide correct information of UPEC biology in this respect because it is with the use of this system that we had first discovered the rpoS role in Gm resistance. The cumulative work on this grant earned the entire EcAMSat team individual NASA awards.\\nThe publication  concerning this work will be submitted within the coming weeks.', 'bibliography_description': 'Description: (Last Updated: 03/27/2020)', 'bibliography_articles': \"Ver 'Show Cumulative Bibliography'\"}, 'id': '10174', 'source_file': '10174.html', '_id': '0db4fcf3-be6c-413a-a6a1-41956f64dd49', '_collection_name': 'tasks_proyects'}, page_content='This project has accomplished one of its two milestones, and the updated summary provided now reflects not only the conceptual genesis of the project, but also as it was refined as the project progressed. While, of course, the original essence remains, what is presented here is the summit of the first milestone, and precedes previous reports. This milestone is concerned with Earth studies to develop and rigorously test the technology for determining the sensitivity of uropathogenic Escherichia coli (UPEC) to the antibiotic gentamicin (Gm) during spaceflight, and to elucidate the genetic basis of whatever change is seen. The second milestone is experimentation during the spaceflight itself, which was launched in late 2017.\\nA paper entitled: “Payload hardware and experimental protocol development to enable future testing of the effect of space microgravity on the resistance to gentamicin of uropathogenic Escherichia coli and its (sigma)s-deficient mutant,” was published online in May 2017: Life Sciences in Space Research 15 (2017) 1–10 2214-5524/ © 2017 The Committee on Space Research (COSPAR). Published by Elsevier Ltd.; [\\nhttp://dx.doi.org/10.1016/j.lssr.2017.05.001\\n] As has been pointed out before, this project represents highly collaborative and collegial collaboration between Stanford and NASA Ames Research Center at Moffett Field. The published paper reflects this not only by the joint authorship of it by both Stanford and Moffett Field scientists, but also the benefits that accrue from collaboration between biological expertise (Stanford) and the flight technological expertise (Moffett Field).\\nThe list of authors is as follows: AC Matin, J-H Wang, Mimi Keyhan, Rachna Singh, and Michael Benoit of the Department of Microbiology & Immunology at Stanford School of Medicine; and Macarena P. Parra, Michael R. Padgen, Antonio J. Ricco, Matthew Chin, Charlie R. Friedericks, Tori N. Chinn, Aaron Cohen, Michael B. Henschke, Timothy V. Snyder, Matthew P. Lera, Shannon S. Ross, Christina M. Mayberry, Sungshin Choi, Diana T. Wu, Ming X. Tan, Travis D. Boone, Christopher C. Beasley, and Stevan M. Spremo, of NASA Ames Research Center, Moffett Field, CA 94035, USA.   The abstract of this paper, reproduced here (in quotes), succinctly describes the evolved conceptual background, and demonstrates in detail the development of the EcAMSat platform, and its thorough testing using spaceflight-like commands.\\n“Human immune response is compromised and bacteria can become more antibiotic resistant in space microgravity (MG). We report that under low-shear modeled microgravity (LSMMG), stationary-phase uropathogenic Escherichia coli (UPEC) become more resistant to gentamicin (Gm), and that this increase is dependent on the presence of s (a transcription regulator encoded by the rpoS gene). UPEC causes urinary tract infections (UTIs), reported to afflict astronauts; Gm is a standard treatment, so these findings could impact astronaut health. Because LSMMG findings can differ from MG, we report preparations to examine UPEC’s Gm sensitivity during spaceflight using the E. coli Anti-Microbial Satellite (EcAMSat) on a free-flying “nanosatellite” in low Earth orbit. Within EcAMSat’s payload, a 48-microwell fluidic card contains and supports study of bacterial cultures at constant temperature; optical absorbance changes in cell suspensions were made at three wavelengths for each microwell and a fluid-delivery system provided growth medium and predefined Gm concentrations. Performance characterization is reported here for spaceflight prototypes of this payload system. Using conventional microtiter plates, we show that Alamar Blue (AB) absorbance changes can assess the Gm effect on E. coli viability, permitting telemetric transfer of the spaceflight data to Earth. Laboratory results using payload prototypes are consistent with wellplate and flask findings of differential sensitivity of UPEC and its delta rpoS strain to Gm. if (sigma)s plays the same role in space MG as in LSMMG and Earth gravity, countermeasures discovered in recent Earth studies (aimed at weakening the UPEC antioxidant defense) to control UPEC infections would prove useful also in space flights. Further, EcAMSat results should clarify inconsistencies from previous space experiments on bacterial antibiotic sensitivity and other issues.” The manuscript may be consulted for further details and all the relevant references; see Bibliography section in 2017 report.\\nIn addition, the work has brought to light issues, such as the stress exerted by the flight equipment on bacteria, which we are now in the process of elucidating and ameliorating for the development of better future platforms for spaceflight experimentation on bacterial biology, resistance, and virulence.'),\n",
       " Document(metadata={'project_title': 'Spaceflight Effects on Bacterial Antibiotic Resistance and Its Genetic Basis “AntimicrobialSat”Reduce', 'fiscal_year': 'FY 2019', 'division': 'Space Biology', 'research_discipline/element': '', 'start_date': '07/01/2010', 'end_date': '12/31/2018', 'task_last_updated': '05/22/2019', 'research_discipline_element': 'Fiscal Year:FY 2019Division:Space BiologyResearch Discipline/Element:Space Biology: Cell & Molecular Biology | Microbiology', 'pdf_download_link': 'tbpdf.cfm?id=12284', 'principal_investigator': {}, 'project_information': {'responsible_center': 'NASA ARC', 'grant_monitor': 'Sato, Kevin (kevin.y.sato@nasa.gov)', 'center_contact': '650-604-1104', 'unique_id': '8002', 'solicitation__funding_source': '2008 SALMON--Astrobiology & FSB NNH08ZDA009O-SCMAFSB', 'grantcontract_no': '80NSSC17M0006 ; NNX10AM90A', 'project_type': 'Flight', 'flight_program': 'Small Satellites', 'no_of_post_docs': '4', 'no_of_phd_candidates': '1', \"no_of_master's_candidates\": '0', \"no_of_bachelor's_candidates\": '0', 'no_of_phd_degrees': '1', \"no_of_master's_degrees\": '0', \"no_of_bachelor's_degrees\": '0', 'space_biology_element': '(1) Cell & Molecular Biology\\n(2) Microbiology', 'space_biology_cross-element_discipline': '(1) Developmental Biology', 'space_biology_special_category': '(1) Translational (Countermeasure) Potential', 'flight_assignment/project_notes': 'EcAMSat\\nNOTE: End date change to 12/31/2018 per F. Hernandez/ARC (Ed., 6/26/18)\\nNOTE: End date changed to 6/30/2018 with new grant number 80NSSC17M0006 in August 2017 (Ed., 5/22/18)\\nNOTE: End date changed to 04/30/2017 per F. Hernandez/ARC and NSSC information (Ed., 4/12/17)\\nNOTE: End date changed to 3/31/2017 per PI and NSSC information (Ed., 6/13/16)\\nNOTE: End date changed to 3/31/2016 per NSSC information (Ed., 1/20/16)\\nNOTE: End date changed to 11/30/2015 per NSSC information and A. Chu/ARC (Ed., 12/16/14)\\nNOTE: New end date is 11/30/2014 per NSSC information (Ed., 4/4/14)\\nNOTE: New end date is 4/30/2014 per A. Chu/ARC and NSSC (Ed., 2/21/2013)', 'task_description': 'This project has accomplished one of its two milestones, and the updated summary provided now reflects not only the conceptual genesis of the project, but also as it was refined as the project progressed. While, of course, the original essence remains, what is presented here is the summit of the first milestone, and precedes previous reports. This milestone is concerned with Earth studies to develop and rigorously test the technology for determining the sensitivity of uropathogenic Escherichia coli (UPEC) to the antibiotic gentamicin (Gm) during spaceflight, and to elucidate the genetic basis of whatever change is seen. The second milestone is experimentation during the spaceflight itself, which was launched in late 2017.\\nA paper entitled: “Payload hardware and experimental protocol development to enable future testing of the effect of space microgravity on the resistance to gentamicin of uropathogenic Escherichia coli and its (sigma)s-deficient mutant,” was published online in May 2017: Life Sciences in Space Research 15 (2017) 1–10 2214-5524/ © 2017 The Committee on Space Research (COSPAR). Published by Elsevier Ltd.; [\\nhttp://dx.doi.org/10.1016/j.lssr.2017.05.001\\n] As has been pointed out before, this project represents highly collaborative and collegial collaboration between Stanford and NASA Ames Research Center at Moffett Field. The published paper reflects this not only by the joint authorship of it by both Stanford and Moffett Field scientists, but also the benefits that accrue from collaboration between biological expertise (Stanford) and the flight technological expertise (Moffett Field).\\nThe list of authors is as follows: AC Matin, J-H Wang, Mimi Keyhan, Rachna Singh, and Michael Benoit of the Department of Microbiology & Immunology at Stanford School of Medicine; and Macarena P. Parra, Michael R. Padgen, Antonio J. Ricco, Matthew Chin, Charlie R. Friedericks, Tori N. Chinn, Aaron Cohen, Michael B. Henschke, Timothy V. Snyder, Matthew P. Lera, Shannon S. Ross, Christina M. Mayberry, Sungshin Choi, Diana T. Wu, Ming X. Tan, Travis D. Boone, Christopher C. Beasley, and Stevan M. Spremo, of NASA Ames Research Center, Moffett Field, CA 94035, USA.   The abstract of this paper, reproduced here (in quotes), succinctly describes the evolved conceptual background, and demonstrates in detail the development of the EcAMSat platform, and its thorough testing using spaceflight-like commands.\\n“Human immune response is compromised and bacteria can become more antibiotic resistant in space microgravity (MG). We report that under low-shear modeled microgravity (LSMMG), stationary-phase uropathogenic Escherichia coli (UPEC) become more resistant to gentamicin (Gm), and that this increase is dependent on the presence of s (a transcription regulator encoded by the rpoS gene). UPEC causes urinary tract infections (UTIs), reported to afflict astronauts; Gm is a standard treatment, so these findings could impact astronaut health. Because LSMMG findings can differ from MG, we report preparations to examine UPEC’s Gm sensitivity during spaceflight using the E. coli Anti-Microbial Satellite (EcAMSat) on a free-flying “nanosatellite” in low Earth orbit. Within EcAMSat’s payload, a 48-microwell fluidic card contains and supports study of bacterial cultures at constant temperature; optical absorbance changes in cell suspensions were made at three wavelengths for each microwell and a fluid-delivery system provided growth medium and predefined Gm concentrations. Performance characterization is reported here for spaceflight prototypes of this payload system. Using conventional microtiter plates, we show that Alamar Blue (AB) absorbance changes can assess the Gm effect on E. coli viability, permitting telemetric transfer of the spaceflight data to Earth. Laboratory results using payload prototypes are consistent with wellplate and flask findings of differential sensitivity of UPEC and its delta rpoS strain to Gm. if (sigma)s plays the same role in space MG as in LSMMG and Earth gravity, countermeasures discovered in recent Earth studies (aimed at weakening the UPEC antioxidant defense) to control UPEC infections would prove useful also in space flights. Further, EcAMSat results should clarify inconsistencies from previous space experiments on bacterial antibiotic sensitivity and other issues.” The manuscript may be consulted for further details and all the relevant references; see Bibliography section in 2017 report.\\nIn addition, the work has brought to light issues, such as the stress exerted by the flight equipment on bacteria, which we are now in the process of elucidating and ameliorating for the development of better future platforms for spaceflight experimentation on bacterial biology, resistance, and virulence.', 'research_impact/earth_benefits': 'Antibiotic resistance in bacteria is becoming an extremely serious problem on Earth that impacts space travelers harder given the deleterious effect of microgravity on human immune response. The research has confirmed the hypothesis that underpinned it, and has shown the way to enhance effectiveness of antibiotics in general. Subsequent work, which is in progress, has identified compounds whose co-administration with antibiotics will enhance their curative power; these can also permit lowering the needed amount of antibiotics for treating infections, thereby minimizing their side effects. This promises to improve life in space as well as on Earth.'}, 'task_progress_and_bibliography': {'task_progress': 'The spaceflight took place onboard a free-flying “nanosatellite,” which, for the first time utilized a 6-U configuration (a “U” being a 10-cm cube). The experiments were conducted on the EcAMSat hardware platform that we developed and described previously in the task book reports. Cell viability in this platform was measured by changes in the absorption of the dye Alamar Blue (AB); these were telemetered to Earth and recorded by daily communications with the spacecraft at Santa Clara University. Parallel ground controls were run. The flight performed very smoothly and all the needed data were successfully obtained. The major conclusion from the results is that the gene rpoS continues to play a critical role in making urinary pathogenic Escherichia coli (UPEC) highly resistant to the antibiotic gentamicin (Gm). This antibiotic is used to treat urinary tract infections often reported in astronauts. Therefore, the potential small molecule inhibitors that we have identified (using bioinformatics approaches) of the protein encoded by rpoS and the antioxidant proteins it controls hold the promise to promote astronaut health in spaceflights. That these molecules are valuable also in meeting the rampant bacterial antibiotic resistance on Earth has been pointed out before.\\nA second important finding is that the high-aspect-ratio vessels (HARVs) used to simulate microgravity on Earth did indeed provide correct information of UPEC biology in this respect because it is with the use of this system that we had first discovered the rpoS role in Gm resistance. The cumulative work on this grant earned the entire EcAMSat team individual NASA awards.\\nThe publication  concerning this work will be submitted within the coming weeks.', 'bibliography_description': 'Description: (Last Updated: 03/27/2020)', 'bibliography_articles': \"Ver 'Show Cumulative Bibliography'\"}, 'id': '12284', 'source_file': '12284.html', '_id': 'e272449f-6a3d-42cf-8290-3e24e925485d', '_collection_name': 'tasks_proyects'}, page_content='This project has accomplished one of its two milestones, and the updated summary provided now reflects not only the conceptual genesis of the project, but also as it was refined as the project progressed. While, of course, the original essence remains, what is presented here is the summit of the first milestone, and precedes previous reports. This milestone is concerned with Earth studies to develop and rigorously test the technology for determining the sensitivity of uropathogenic Escherichia coli (UPEC) to the antibiotic gentamicin (Gm) during spaceflight, and to elucidate the genetic basis of whatever change is seen. The second milestone is experimentation during the spaceflight itself, which was launched in late 2017.\\nA paper entitled: “Payload hardware and experimental protocol development to enable future testing of the effect of space microgravity on the resistance to gentamicin of uropathogenic Escherichia coli and its (sigma)s-deficient mutant,” was published online in May 2017: Life Sciences in Space Research 15 (2017) 1–10 2214-5524/ © 2017 The Committee on Space Research (COSPAR). Published by Elsevier Ltd.; [\\nhttp://dx.doi.org/10.1016/j.lssr.2017.05.001\\n] As has been pointed out before, this project represents highly collaborative and collegial collaboration between Stanford and NASA Ames Research Center at Moffett Field. The published paper reflects this not only by the joint authorship of it by both Stanford and Moffett Field scientists, but also the benefits that accrue from collaboration between biological expertise (Stanford) and the flight technological expertise (Moffett Field).\\nThe list of authors is as follows: AC Matin, J-H Wang, Mimi Keyhan, Rachna Singh, and Michael Benoit of the Department of Microbiology & Immunology at Stanford School of Medicine; and Macarena P. Parra, Michael R. Padgen, Antonio J. Ricco, Matthew Chin, Charlie R. Friedericks, Tori N. Chinn, Aaron Cohen, Michael B. Henschke, Timothy V. Snyder, Matthew P. Lera, Shannon S. Ross, Christina M. Mayberry, Sungshin Choi, Diana T. Wu, Ming X. Tan, Travis D. Boone, Christopher C. Beasley, and Stevan M. Spremo, of NASA Ames Research Center, Moffett Field, CA 94035, USA.   The abstract of this paper, reproduced here (in quotes), succinctly describes the evolved conceptual background, and demonstrates in detail the development of the EcAMSat platform, and its thorough testing using spaceflight-like commands.\\n“Human immune response is compromised and bacteria can become more antibiotic resistant in space microgravity (MG). We report that under low-shear modeled microgravity (LSMMG), stationary-phase uropathogenic Escherichia coli (UPEC) become more resistant to gentamicin (Gm), and that this increase is dependent on the presence of s (a transcription regulator encoded by the rpoS gene). UPEC causes urinary tract infections (UTIs), reported to afflict astronauts; Gm is a standard treatment, so these findings could impact astronaut health. Because LSMMG findings can differ from MG, we report preparations to examine UPEC’s Gm sensitivity during spaceflight using the E. coli Anti-Microbial Satellite (EcAMSat) on a free-flying “nanosatellite” in low Earth orbit. Within EcAMSat’s payload, a 48-microwell fluidic card contains and supports study of bacterial cultures at constant temperature; optical absorbance changes in cell suspensions were made at three wavelengths for each microwell and a fluid-delivery system provided growth medium and predefined Gm concentrations. Performance characterization is reported here for spaceflight prototypes of this payload system. Using conventional microtiter plates, we show that Alamar Blue (AB) absorbance changes can assess the Gm effect on E. coli viability, permitting telemetric transfer of the spaceflight data to Earth. Laboratory results using payload prototypes are consistent with wellplate and flask findings of differential sensitivity of UPEC and its delta rpoS strain to Gm. if (sigma)s plays the same role in space MG as in LSMMG and Earth gravity, countermeasures discovered in recent Earth studies (aimed at weakening the UPEC antioxidant defense) to control UPEC infections would prove useful also in space flights. Further, EcAMSat results should clarify inconsistencies from previous space experiments on bacterial antibiotic sensitivity and other issues.” The manuscript may be consulted for further details and all the relevant references; see Bibliography section in 2017 report.\\nIn addition, the work has brought to light issues, such as the stress exerted by the flight equipment on bacteria, which we are now in the process of elucidating and ameliorating for the development of better future platforms for spaceflight experimentation on bacterial biology, resistance, and virulence.')]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "retrieved_docs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "eb805a8b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'11864'"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "retrieved_docs[0].metadata['id']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "fd3ef2c6",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "nasa_env (3.10.18)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.18"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
