Eur J Hum Genet

. 2022 May 16;30(8):889–898. doi: [10.1038/s41431-022-01108-8](https://doi.org/10.1038/s41431-022-01108-8)

# A comprehensive SARS-CoV-2 and COVID-19 review, Part 1: Intracellular overdrive for SARS-CoV-2 infection

[David A Jamison Jr](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jamison%20DA%22%5BAuthor%5D)

### David A Jamison Jr

1COVID-19 International Research Team, Medford, MA USA

Find articles by [David A Jamison Jr](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jamison%20DA%22%5BAuthor%5D)

1,#, [S Anand Narayanan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Anand%20Narayanan%20S%22%5BAuthor%5D)

### S Anand Narayanan

1COVID-19 International Research Team, Medford, MA USA

2Department of Nutrition & Integrative Physiology, Florida State University, Tallahassee, FL USA

Find articles by [S Anand Narayanan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Anand%20Narayanan%20S%22%5BAuthor%5D)

1,2,✉,#, [Nídia S Trovão](https://pubmed.ncbi.nlm.nih.gov/?term=%22Trov%C3%A3o%20NS%22%5BAuthor%5D)

### Nídia S Trovão

1COVID-19 International Research Team, Medford, MA USA

3Fogarty International Center, National Institutes of Health, Bethesda, MD USA

Find articles by [Nídia S Trovão](https://pubmed.ncbi.nlm.nih.gov/?term=%22Trov%C3%A3o%20NS%22%5BAuthor%5D)

1,3, [Joseph W Guarnieri](https://pubmed.ncbi.nlm.nih.gov/?term=%22Guarnieri%20JW%22%5BAuthor%5D)

### Joseph W Guarnieri

1COVID-19 International Research Team, Medford, MA USA

4Center for Mitochondrial and Epigenomic Medicine, Children’s Hospital of Philadelphia, Philadelphia, PA USA

Find articles by [Joseph W Guarnieri](https://pubmed.ncbi.nlm.nih.gov/?term=%22Guarnieri%20JW%22%5BAuthor%5D)

1,4, [Michael J Topper](https://pubmed.ncbi.nlm.nih.gov/?term=%22Topper%20MJ%22%5BAuthor%5D)

### Michael J Topper

1COVID-19 International Research Team, Medford, MA USA

5Department of Oncology, The Sidney Kimmel Comprehensive Cancer Center at Johns Hopkins, Baltimore, MD USA

Find articles by [Michael J Topper](https://pubmed.ncbi.nlm.nih.gov/?term=%22Topper%20MJ%22%5BAuthor%5D)

1,5, [Pedro M Moraes-Vieira](https://pubmed.ncbi.nlm.nih.gov/?term=%22Moraes-Vieira%20PM%22%5BAuthor%5D)

### Pedro M Moraes-Vieira

1COVID-19 International Research Team, Medford, MA USA

6Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP Brazil

7Obesity and Comorbidities research Center (OCRC), University of Campinas, Campinas, SP Brazil

8Experimental Medicine Research Cluster, University of Campinas, Campinas, Brazil

Find articles by [Pedro M Moraes-Vieira](https://pubmed.ncbi.nlm.nih.gov/?term=%22Moraes-Vieira%20PM%22%5BAuthor%5D)

1,6,7,8, [Viktorija Zaksas](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zaksas%20V%22%5BAuthor%5D)

### Viktorija Zaksas

1COVID-19 International Research Team, Medford, MA USA

9Center for Translational Data Science, University of Chicago, Chicago, IL USA

Find articles by [Viktorija Zaksas](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zaksas%20V%22%5BAuthor%5D)

1,9, [Keshav K Singh](https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20KK%22%5BAuthor%5D)

### Keshav K Singh

1COVID-19 International Research Team, Medford, MA USA

10Department of Genetics, Heersink School of Medicine, University of Alabama at Birmingham, Birmingham, AL USA

Find articles by [Keshav K Singh](https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20KK%22%5BAuthor%5D)

1,10, [Eve Syrkin Wurtele](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wurtele%20ES%22%5BAuthor%5D)

### Eve Syrkin Wurtele

1COVID-19 International Research Team, Medford, MA USA

11Center for Metabolic Biology, Bioinformatics and Computational Biology, and Genetics Development, and Cell Biology, Iowa State University, Ames, IA USA

Find articles by [Eve Syrkin Wurtele](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wurtele%20ES%22%5BAuthor%5D)

1,11, [Afshin Beheshti](https://pubmed.ncbi.nlm.nih.gov/?term=%22Beheshti%20A%22%5BAuthor%5D)

### Afshin Beheshti

1COVID-19 International Research Team, Medford, MA USA

12Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA USA

13KBR, Space Biosciences Division, NASA Ames Research Center, Moffett Field, CA USA

Find articles by [Afshin Beheshti](https://pubmed.ncbi.nlm.nih.gov/?term=%22Beheshti%20A%22%5BAuthor%5D)

1,12,13,✉

* Author information
* Article notes
* Copyright and License information

1COVID-19 International Research Team, Medford, MA USA

2Department of Nutrition & Integrative Physiology, Florida State University, Tallahassee, FL USA

3Fogarty International Center, National Institutes of Health, Bethesda, MD USA

4Center for Mitochondrial and Epigenomic Medicine, Children’s Hospital of Philadelphia, Philadelphia, PA USA

5Department of Oncology, The Sidney Kimmel Comprehensive Cancer Center at Johns Hopkins, Baltimore, MD USA

6Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP Brazil

7Obesity and Comorbidities research Center (OCRC), University of Campinas, Campinas, SP Brazil

8Experimental Medicine Research Cluster, University of Campinas, Campinas, Brazil

9Center for Translational Data Science, University of Chicago, Chicago, IL USA

10Department of Genetics, Heersink School of Medicine, University of Alabama at Birmingham, Birmingham, AL USA

11Center for Metabolic Biology, Bioinformatics and Computational Biology, and Genetics Development, and Cell Biology, Iowa State University, Ames, IA USA

12Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA USA

13KBR, Space Biosciences Division, NASA Ames Research Center, Moffett Field, CA USA

✉

Corresponding author.

#

Contributed equally.

Received 2021 Nov 24; Revised 2022 Mar 20; Accepted 2022 Apr 12; Issue date 2022 Aug.

© The Author(s) 2022

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](https://creativecommons.org/licenses/by/4.0/).

[PMC Copyright notice](/about/copyright/)

PMCID: PMC9108708  PMID: [35577935](https://pubmed.ncbi.nlm.nih.gov/35577935/)

## Abstract

COVID-19, the disease caused by SARS-CoV-2, has claimed approximately 5 million lives and 257 million cases reported globally. This virus and disease have significantly affected people worldwide, whether directly and/or indirectly, with a virulent pathogen that continues to evolve as we race to learn how to prevent, control, or cure COVID-19. The focus of this review is on the SARS-CoV-2 virus’ mechanism of infection and its proclivity at adapting and restructuring the intracellular environment to support viral replication. We highlight current knowledge and how scientific communities with expertize in viral, cellular, and clinical biology have contributed to increase our understanding of SARS-CoV-2, and how these findings may help explain the widely varied clinical observations of COVID-19 patients.

**Subject terms:** Viral infection, Infection

## Introduction

As of November 21st, 2021, over 257 million cases of coronavirus disease 2019 (COVID-19) have been reported, and more than 5 million lives claimed globally [[1](#CR1)]. The disease is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Development of vaccines against SARS-CoV-2 provide a major step forward in reducing COVID-19’s impact. However, the pandemic is ongoing, and the continued viral transmission allows for accumulation of mutations in the viral genome, which can provide advantages in replication, immune escape, increased transmissibility, or diagnostic detection failure [[2](#CR2)]. With the quickly evolving SARS-CoV-2 variants and the slow rate of vaccination globally, it is critical to fully understand this novel virus and disease.

Coronaviruses are named as such because the S proteins resemble a halo or corona on scanning electron microscope imagery [[3](#CR3)]. SARS-CoV-2 belongs to the genus Betacoronavirus. Of the human Betacoronavirus, including OC43, HKU1, SARS-CoV-1, and Middle East Respiratory Syndrome-Coronavirus (MERS-CoV) [[4](#CR4)]. SARS-CoV-2 bears the highest genetic sequence similarity to SARS-CoV-1 [[5](#CR5)]. Accordingly, COVID-19, caused by SARS-CoV-2, resembles SARS, caused by SARS-CoV-1, in many ways, but with some important differences [[6](#CR6)]. Key characteristics of SARS-CoV-1 and 2 include: 1) a positive-sense RNA virus with a large genome of ~30 kilobases; 2) a large, enveloped virus containing a helical nucleocapsid with the virus’s genetic code, with an exterior studded in several spike proteins that facilitate the infection of host cells), and 3) similar genomic structures. The first 2/3 of both genomes encodes for two macro polypeptides pp1a/pp1b (see Fig. [1](#Fig1)). Pp1a/pp1b are auto-proteolytically processed to generate 16 non-structural proteins (NSP).

### Fig. 1. SARS-CoV-2 structure.

[![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5550/9349314/b087dc279ef3/41431_2022_1108_Fig1_HTML.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9349314_41431_2022_1108_Fig1_HTML.jpg)

[Open in a new tab](figure/Fig1/)

Structural elements of the virus, including the spike protein, envelope, membrane, and internal components such as the viral single-stranded RNA and nucleocapsid proteins (above). SARS-CoV-2 genome components (below).

The main virus-specific functions of the SARS-CoV-2 NSPs are: NSP1 - cellular mRNA degradation, global translation inhibition; NSP2 - cell cycle progression disruption; NSP3 - formation of double-membrane vesicles (DMVs; SARS-CoV-2 protease); NSP4 - formation of DMVs; NSP5 - main SARS-CoV-2 protease; NSP6 - formation of DMVs, NSP7 - replication complex; NSP8 – primase; NSP9 - RNA binding protein; NSP10 - cofactor of NSP14 & NSP16; NSP11 - unknown, NSP12 - RNA-dependent RNA polymerase; NSP13 - RNA helicase, 5ʹ phosphatase, NSP14 - N7-MTase, 3ʹ-5ʹ exonuclease; NSP15 – endonuclease; and NSP16–2ʹ-O-MTase, mRNA capping.

The remaining 1/3 of the SARS-CoV-2 genome encodes for the structural proteins S (spike), E (envelope), M (membrane), and N (nucleocapsid), and several open reading frames (ORFs; (3a, 6, 7a, 7b, 8, 9b, and 10) [[7](#CR7)]. The S protein binds the host cell receptor, which for SARS-CoV-1/2 is the human angiotensin-converting enzyme 2 (hACE2) (see Fig. [2](#Fig2) and Supplementary Table [1](#MOESM1)). These proteins share homology and function with SARS-CoV-1.

### Fig. 2. SARS-CoV-2 binding, intracellular internalization, and intracellular processes.

[![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5550/9349314/51271396d4a9/41431_2022_1108_Fig2_HTML.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9349314_41431_2022_1108_Fig2_HTML.jpg)

[Open in a new tab](figure/Fig2/)

Structural interactions between the virus and target cell, including the viral spike protein, ACE2-receptor, TMPRSS2 reaction to cleave and begin the viral intracellular internalization (above, **A**), and consequent signal transduction pathways stimulated by the virus as it hijacks pathways to turn the infected cell into a SARS-CoV-2 producing factory (below, **B**).

There are two notable differences between SARS-CoV-1 and SARS-CoV-2. First is the presence of the ORF8 polypeptide found in SARS-CoV-2 but not in SARS-CoV-1. SARS-CoV-1 has a 29 nucleotide (nt) deletion (del) which splits it into ORF8a and ORF8b. Second, SARS-CoV-2 contains a gene encoding a novel orphan protein, ORF10, which is not present in SARS-CoV-1 [[7](#CR7)].

SARS-CoV-2’s evolutionary rate has been estimated to be around 9’×’10−4 substitutions per site per year [[8](#CR8)], while also having a high transmissibility, large portion of asymptomatic cases [[9](#CR9)], large pool of susceptible hosts to replicate in [[10](#CR10), [11](#CR11)], and on-going environmental pressures (e.g., low vaccination rates and changes in policies allowing human carriers to continue to transmit the virus), which have allowed SARS-CoV-2 to accumulate mutation in its genome.

Mutations have been detected in all parts of the viral genome, including in the leader 5ʹ untranslated region (UTR), orf1ab (NSP1, NSP2, NSP3, NSP6, NSP12, NSP13, and NSP14), spike, ORF3a, ORF8, nucleocapsid, and ORF10 [[8](#CR8)]. These genomic changes have been shown to influence viral immune evasion, inflammasome interaction, helicase, exonuclease proofreading mechanism, the activity of the RNA-dependent RNA polymerase (RdRp) and thereby viral replication, infectivity, and cell release [[12](#CR12)].

Mutations associated with the spike are of particular interest, as they influence human-to-human transmission, as well as human-to-animal passage. Within the spike, mutations tend to fall into four general classes, those that affect the receptor-binding domain (RBD), which are of importance because some may provide both immune escape or a fitness advantage, as well as facilitate reverse zoonotic events. There are some mutations that occur in the N-terminal domain (NTD), which is the portion most exposed on the virus surface. There is evidence for immune selection in this region, and preliminary evidence that at least one of these changes (delH69/delV70) could improve fitness [[13](#CR13)]. Mutations in or near the furin cleavage site, and several groupings close to the D614G mutation, possibly affect infection efficiency and can also be important for neutralizing antibodies.

This large SARS-CoV-2 genome diversity has been categorized by different nomenclature systems, describing variants of varied public health interest or concern. Pango lineages B.1.1.7 (Alpha), B.1.351 (Beta), P.1 (Gamma) and B.1.617.2 (Delta) have been classified as “variants of concern” (VOC) because they present mutations that have been shown to impact diagnostics, treatments, or vaccines, conferring increased transmissibility and increased disease severity. The impact of these mutations highlights the need for further research not only on the mechanisms of SARS-CoV-2 intracellular processes, but also how the extracellular environment may lead to further spread of the virus and subsequent public health burden.

The SARS-CoV-2 virus can exert physiological effects by directly infecting cells and via intercellular signaling by the infected cells. In this review, we provide insight into SARS-CoV-2 infection and intracellular host responses (targets, pathways, networks, biological processes, and functional adaptations) to viral invasion. We emphasize a canonical set of reactions induced by SARS-CoV-2, which we have organized for the reader’s consideration. However, there is tremendous variation in cellular responses to SARS-CoV-2, depending on factors including the cell type, organ type, metabolic and physiological context, patient genetics, individual clinical characteristics (e.g., age, sex, comorbidities), and stage/severity in the COVID-19 disease.

This is the first of a three-part comprehensive series of linked reviews on SARS-CoV-2 covering: intracellular effects (present study); extracellular consequences (review 2); and current and potential therapeutics (review 3). This review and the two that will follow aim to provide a foundational understanding of the current knowledge on SARS-CoV-2, from basic biology to clinical outcome and therapy avenues, that highlight future areas of research and could help inform public health interventions across the world.

### Infected tissue and cell types

SARS-CoV-2 targets the nasal cavity and lungs; however, the detailed cellular tropism remains unclear, and likely varies among individuals. Furthermore, there is increased variability of viral cellular tropism with the emergent SARS-CoV-2 variants, which include Alpha, Beta, Gamma, Epsilon, Eta, Iota, Kappa, 1.617.3, Mu, Zeta, and in particular, Delta and Omicron, as well as the various lineages of each variant [[14](#CR14)]. This is related with SARS-CoV-2’s mutation ability affecting its antigenic phenotype to circumvent immunity. The spike protein mediates attachment of the virus to host cell-surface receptors and fusion between virus and cell membranes; it is also the principal target for neutralizing antibodies generated following infection, and is the component for both mRNA and adenovirus-based vaccines [[15](#CR15)]. Several studies have contributed to the current understanding of how mutations in the SARS-CoV-2 spike protein affect neutralization and emergence of new strains, which include studies of traditional escape mutation, targeted characterization of particular mutations, and wider investigations of large numbers of circulating variants [[16](#CR16)]. These are active areas of research, in particular given the continued emergence of new lineages of new variants. A study of human, bat, non-human primate, and mouse cell lines showed various cell types were susceptible to the virus. These included pulmonary, intestinal, hepatic, renal, and neuronal, with cell lines expressing the hACE-2 receptor (hACE-2) having a generally greater viral load [[17](#CR17)]. Although cell lines do not reflect physiological conditions, this research indicates that SARS-CoV-2 can infect many cell types, and that hACE-2 provides a critical entry mechanism [[18](#CR18)]. Epithelial, vascular endothelial, pancreas, and mucosal cell types can all be infected by the virus [[19](#CR19)–[21](#CR21)].

Several investigations have employed 3D organoid cultures to simulate more physiological conditions than cell cultures [[22](#CR22)]. In one such study, lung and colonic organoid models showed SARS-CoV-2 infection was reduced when various SARS-CoV-2 entry inhibitors were applied [[22](#CR22)]. Another study illustrated the flexibility of different organoid models, such as pancreatic endocrine cells, liver organoids, cardiomyocytes, and dopaminergic neurons from human pluripotent stem cells, and adult primary cells (human islets, hepatocyte, and cholangiocytes) to test viral effects such as cytokine production, gene expression, and other physiological responses. The resultant data correlated well with some patient autopsy samples [[22](#CR22)] indicating organoids provide a valuable disease modeling tool [[18](#CR18)].

In one study of post-mortem patients, immunohistochemistry and immunofluorescence revealed viral antigen (spike protein) in pneumocytes and hyperplastic cells around the bronchioles, mucosal epithelia, submucosal glands, gland ducts of the trachea, glands of the small intestine, distal tubules and collecting ducts of the kidneys, islets of Langerhans, glands and intra-islet ducts of the pancreas, and vascular tissues of the brain and heart [[23](#CR23)]. Few viral antigens were present in the large intestine and renal proximal tubules, and none in the liver. A follow-up colocalization analysis showed ACE2 and viral antigen in the lung, trachea, small intestine, kidney, pancreas, and heart. In the brain, ACE2-expressing cells were detected, but they were negative for the viral antigen [[23](#CR23)]. Endothelial cells of multiple organs were infected, supporting the clinical observations of endotheliitis in some COVID-19 patients.

Single-cell RNA sequencing (scRNA-seq) demonstrated ACE2 receptor expression was primarily restricted to lung pneumocytes, gut absorptive enterocytes, and nasal mucosa goblet secretory cells [[24](#CR24)]. In general, the distribution of ACE2 receptors may in part explain the systemic diversity and range of SARS-CoV-2’s effects. Further research into infection of these cell types versus others in mucosal barrier organs will be important to determine cell-types that serve as initial entry ways for the virus into the body.

Human autopsy studies [[21](#CR21)] have shown that SARS-CoV-2 infects multiple organs including lungs, pharynx, liver, nasal mucosa, trachea, intestines, skin, pancreas, kidney, brain, and heart. A study of 27 patients showed multi-organ tropisms (lung, pharynx, heart, liver, brain, and kidneys), with the highest levels of SARS-CoV-2 copies per cell, as detected by in situ hybridization and indirect immunofluorescence, in the respiratory tract, and lower levels in the kidneys, liver, heart, and brain [[21](#CR21)]. Transcriptional profiling of nasopharyngeal swabs, patient autopsy, and body-wide tissues (e.g. heart, liver, lung, kidney, and lymph nodes), provided further evidence of the physiologically systemic effects of SARS-CoV-2 [[24](#CR24)].

These studies suggest that the virus has a varying range of expression within each organ, which may be influenced by levels of the ACE2-receptor and related entry factors (Transmembrane protease, serine-2 [TMPRSS2], transferrin receptor protein 1 [TRFC1], cluster of differentiation 4 [CD4], and neuropilin-1 [NRP1]) within each organ-type [[24](#CR24)]. This further highlights the varied organ and tissues that are capable of being infected by the virus, and the resultant wide-range of patient symptoms.

The physiological status of the individual significantly affects COVID-19 morbidity and mortality [[25](#CR25), [26](#CR26)]. Notably, patients with pre-existing conditions of obesity, hypertension, and diabetes have a less favorable disease outcome, likely in part due to the elevated levels of inflammation and metabolic disturbances associated with those conditions [[25](#CR25)]. Conversely, SARS-CoV-2 infection may exacerbate pre-existing conditions, leading to more severe COVID-19 outcome [[27](#CR27)].

### SARS-CoV-2 Receptors – Angiotensin Converting Enzyme-2 (ACE2)

The cellular surface receptor ACE2, a key regulator of the Renin-Angiotensin Aldosterone System (RAAS). It is speculated to be the primary SARS-CoV-2 viral target for entry. SARS-CoV-2 is thought to infect multiple organs in part due to the widespread distribution, expression, and polymorphisms of ACE2 [[28](#CR28), [29](#CR29)].

ACE2’s molecular function in the human RAAS pathway is to cleave Angiotensin I to produce Angiotensin 1–9, and break down Angiotensin II into Angiotensin 1–7. RAAS moderates blood pressure and osmolarity by means of hormonal feedback control. In response to binding of ACE2 to the ACE2 receptor (ACE2R), blood vessels vasoconstrict. This process is mediated by G-protein-signaling, activating phospholipase C and increasing cytosolic Ca2+ concentrations. ACE2 also plays an important role in inactivating Des-Arg9-Bradykinin (DABK), a bradykinin involved in inflammation. This inactivation promotes C-X-C motif chemokine 5 (CXCL5), macrophage inflammatory protein-2 (MIPS2), keratinocytes-derived chemokine (KC), and tumor necrosis factor-〈 (TNF-α) activity, drawing leukocytes into the affected tissues [[30](#CR30)].

Decreased ACE2 receptor expression can have detrimental effects. Computational models of COVID-19 suggest the role of a bradykinin storm in the pathophysiology of the disease. In this model, the Kallikrein-Kinin and Renin-Angiotensin-Aldosterone Systems are integrated, with cross-talk mediated by the degradation of bradykinin by ACE and prolylcarboxypeptidase [[31](#CR31)]. This behavior makes the SARS-CoV-2 spike protein behave akin to an ACE-inhibiting drug [[32](#CR32)]. Thus, disruption of ACE2 expression from SARS-CoV-2 binding can lead to altered tissue function and exacerbate disease.

The ~600-kDa trimeric S proteins can bind to ACE2 through the RBD required for membrane fusion (see Fig. [2](#Fig2)). The binding initiates viral internalization, with the cleavage of S1/S2 inducing a conformational change from prefusion into post-fusion. S1 consists of the NTD, the RBD, and subdomain 1 and 2 (SD1 and SD2). S2 contains the hydrophobic fusion peptide and is responsible for the viral and cell membrane fusion [[33](#CR33)]. SARS-CoV-2 S-protein shows varying states of conformational shifts of the RBD site progressing towards proteolytic processing, making the viral RBD more accessible to ACE2, with the cleavage at the S1/S2 leading towards RBD open confirmation and viral internalization [[33](#CR33), [34](#CR34)]. The S- and RBD-viral sites are notable for affecting transmission and disease severity, and variants have been shown to accumulate mutations at these sites leading to increased S- and RBD affinity with ACE2 [[35](#CR35)]. Understanding the biology of the SARS-CoV-2 surface interactions will help elucidate how the virus can invade multiple organ systems and cell types.

### Calcium Ion (Ca2+) Signaling

The calcium ion (Ca2+) is essential for many aspects of cellular physiology and viral replication. Experimental data on the relation between Ca2+ signaling and SARS-CoV-2 infection and replication is sparse. However, studies of other coronaviruses (e.g., SARS-CoV-1, MERS-CoV) have reported that these viruses utilize Ca2+ for host fusion [[36](#CR36)]. The fusion protein (FP) of MERS-CoV binds to one Ca2+ ion, while the SARS-CoV-2 spike (S) protein has two FP domains, FP1 and FP2, and binds to two Ca2+ ions for host cell entry [[37](#CR37)]. SARS-CoV-2 appears to affect cellular function by altering the host Ca2+ homeostasis in ways that promote viral infection and reproduction (see Fig. [3](#Fig3)). One mechanism is through disruption of calcium channels and pumps (e.g., voltage-gated calcium channels (VGCCs), receptor-operated calcium channels, store-operated calcium channels, transient receptor-potential ion channels, and Ca2+-ATPase) [[28](#CR28), [37](#CR37)]. This leads to increased intracellular Ca2+ concentrations, resulting in virus-induced cell lysis [[28](#CR28), [37](#CR37)]. The interaction between the virus and VGCCs may also promote virus-host cell fusion for entry [[28](#CR28)].

#### Fig. 3. SARS-CoV-2 effects on Ca2+ signaling.

[![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5550/9349314/316668e3f5c6/41431_2022_1108_Fig3_HTML.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9349314_41431_2022_1108_Fig3_HTML.jpg)

[Open in a new tab](figure/Fig3/)

Structural elements of the virus, including the spike protein, envelope, membrane, and internal components such as the viral single-stranded RNA and nucleocapsid proteins (above).

Viroporins, transmembrane pore-forming proteins that alter membrane permeability to ions including Ca2+ by forming membrane channels, are a characteristic of a diversity of virus. SARS-CoV-1/2 each encode viroporins. SARS-CoV-1 encodes for three viroporin proteins ORF3a, E and ORF8b, which alter ion homeostasis within the cell, and have important roles in pathogenesis and promoting viral fitness. SARS-CoV-2 encodes two of these viroporin proteins, E and ORF3a; however, the ORF8 protein of SARS-CoV-2 is highly divergent from SARS-CoV-1 ORF8b and lacks the viroporin sequence of SARS-CoV-1 ORF8b.

The E and ORF3a proteins of coronaviruses impact Ca2+ homeostasis in the host, by acting as calcium ion channels, enhancing the virion’s entry and replication potential [[38](#CR38)]. The SARS-CoV-2-E protein is a 76 amino acid (aa) integral membrane protein with one transmembrane domain (TMD) that allows the E protein to form protein-lipid channels in membranes that promote permeability to Ca2+ ions. The SARS-CoV-2-ORF3a protein is 274 aa in length, harbors three helical TMD, and is a Na+ or Ca2+ ion channel protein. The alteration of Ca2+ homeostasis by SARS-COV-1-E and SARS-COV-2-E proteins promotes SARS-CoV-1/2 fitness and elicits the production of chemokines and cytokines, contributing to pathogenesis. Ion channel activity modulation by the SARS-CoV-1-ORF3a protein also modulates viral release [[39](#CR39)]. Therefore, when SARS-CoV-2 infects the human body, the resultant dysregulation of Ca2+ homeostasis may contribute to morbidity and mortality. COVID-19 patients have been noted to have low serum calcium levels overall [[40](#CR40)].

We speculate that Ca2+ dysregulation could lead to increased cellular oxidative stress and shifts in metabolic activity. Low Ca2+ may also be coupled with viral infection and internalization through the ACE2R, which synergizes with Ca2+ signaling pathways. Understanding these reverberations will increase our insight into the basic biology of the effects of SARS-CoV-2 infection on the various organ systems.

### Intracellular signaling

Viral infection and hijacking of cell-surface receptors begin to trigger activation of multiple intracellular pathways in addition to Ca2+ signaling. As infection proceeds, SARS-CoV-2 manipulates, or totally reprograms, the normal metabolism and signaling of the host cell, optimizing the molecular environment to enable the viral replication cycle. This involves interfering with signaling pathways that regulate processes of DNA repair and replication, immune response, transcription, metabolism, cell cycle, and apoptosis [[39](#CR39)].

SARS-CoV-2 infection alters phosphoinositide 3-kinase (PI3K)/protein kinase B (AKT), Type I and III interferon, transforming growth factor-β (TGF-β), Toll-like receptors (TLR), and nuclear factor kappa-light chain enhancer (NF-κB) pathways. These pathways are dysregulated in the setting of SARS-CoV-2 to antagonize host antiviral responses and are vital for viral replication, entry, propagation, and/or apoptosis/viral release. For instance, severe COVID-19 is characterized by an inflammatory profile dominated by NF-κB activity [[41](#CR41)]. The SARS-CoV-2-encoded NSP13 and Open Reading Frame 9c (ORF9c) proteins can interact directly with elements of the transducin-like enhancer (TLE) family of proteins and thus regulate the NF-κB inflammatory response [[42](#CR42)]. While broad activation of NF-κB is induced by a variety SARS-CoV-2-encoded products, Open Reading Frame 7a (ORF7a) specifically is a potent stimulator of NF-κB associated proinflammatory chemo- and cytokines, which are elevated in the presence of severe COVID-19. NF-κB plays a similar role in other coronavirus infections.

Host antiviral immunity requires an optimal and coordinated response to control viral infections; this immunity is mediated by several host sensors, notably pattern recognition receptors (PRR). PRRs identify damage- and pathogen-associated molecular patterns (DAMPs and PAMPs, respectively). SARS-CoV-2 infects the cell via the endosomal compartment, and may activate TLRs, such as TLR4, resulting in increased NF-κB activity and expression [[42](#CR42), [43](#CR43)]. The MyD88-mediated TRIF activation of TLR downstream pathways triggers the nuclear translocation of NF-κB, IFN regulatory factor 3 (IRF-3), and IFN regulatory factor 7 (IRF-7), resulting in the expression of innate immunity proinflammatory cytokines (interleukin-1 [IL-1], interleukin-6 [IL- 6], TNF-α) and interferons (IFNs). Continuous activation of TLR can increase MyD88 and Interleukin-1β (IL-1β), which then can further activate NF-κB [[43](#CR43)]. RNA viruses are detected by several sensors, such as TLRs 3, 7 and 8. TLR3 recognizes double stranded RNA, while TLR7 and TLR8 single stranded RNA. In addition to ssRNA and dsRNA, viral proteins can act as PAMPS and potentiate inflammatory signaling through the stimulation of surface TLRs. Interestingly, in SARS-CoV-2, TLR2 is a critical mediator of envelope protein detection and driver of pathogenesis through inflammatory process augmentation [[44](#CR44)]. Some individuals with severe COVID-19 have mutations in genes associated with type I and III IFN pathways [[45](#CR45)]. Ten percent of individuals that progress to severe COVID-19 pneumonia display elevated amounts of neutralizing antibodies against type I IFN-α2 and IFN-ω [[46](#CR46)]; these antibodies are not present in healthy or asymptomatic individuals. Of note, albeit TLR3 activation is critical for viral clearance, TLR3 hyperactivation can lead to a cytokine storm and the subsequent severe COVID-19. Other receptors are also involved in SARS-CoV-2 recognition, such as the proteins of retinoic acid-inducible gene I (RIG-I) and melanoma differentiation-associated gene 5 (MDA-5). Once inside the cell, double strand viral RNA can be recognized by RIG-I/MDA5, thus initiating an antiviral response through mitochondrial antiviral signaling (MAVS). MAVS activated the downstream pathways, IκB kinase α/β (IKK) and TBK1/IKKε, leading to translocation of NF-κB and/or IRF3 into the nucleus and induction of genes involved with innate antiviral immunity and the subsequent induction of IFN-stimulated genes (ISGs).

Inhibition of IFNs and ISGs is a tactic used by several viruses to evade host antiviral responses [[47](#CR47)], and SARS-CoV-2-mediated IFNs and ISGs dysregulation appears to be an important strategy used by this virus to replicate and disrupt immune homeostasis. Furthermore, therapies with type I and III IFNs alone or combined with other drugs suppressed SARS-CoV-2, ameliorating COVID-19 disease [[48](#CR48)].

The cytokine TGF-β triggers the Janus kinases (JAKs) / signal transducer and activator of transcription (STAT) proteins (JAK/STAT) pathway in certain contexts, while suppressing it in others [[49](#CR49)]. It has been proposed that SARS-CoV-2 proteins, particularly NSP1 and ORF6, may dysregulate STAT1 and STAT3, leading to a positive feedback loop where coagulopathy triggers TLR4 via PAI-1 binding, circularly activating STAT; for this reason, therapeutic targeting of the Janus kinase pathway has been proposed [[50](#CR50)].

The innate immune response is a first step to protecting against pathogens, which stimulate the interferon signaling pathway and expression of IFN-I, leading to an antiviral cellular response [[51](#CR51)]. Coronaviruses have developed mechanisms to hinder IFN-expression and reduce the production of IFN. This suppression has been shown to correlate with disease severity and mortality [[52](#CR52)]. This holds true for SARS-CoV-2, with recent studies showing that viral proteins ORF6, ORF8, and nucleocapsid being potent inhibitors of the IFN-I signaling pathway [[53](#CR53)].

### Metabolic adaptations

Viruses rely on host cell machinery to propagate, promoting anabolism for generation of macromolecules needed for virion replication and assembly (see Fig. [2](#Fig2)). Consequently, viral proteins (see Supplementary Table [1](#MOESM1)) affect intracellular pathways, leading to subsequent adaptations by the cellular metabolism where the mitochondria plays a central role.

We reported recently through study of COVID-19 patient samples (i.e., nasopharyngeal swab samples, various organs from autopsy COVID-19 samples, murine lung tissues, and various organs from hamsters being infected with SARS-CoV-2) that heavy suppression occurs of mitochondrial functions in various organs [[54](#CR54)]. Specifically, in the course of SARS-CoV-2 infection, the virus blocks the transcription of discrete groups of mitochondrial genes from major bioenergetic organs, while upregulation occurs in others as a compensatory mechanism to rescue the damage occurring in the major bioenergetic organs. This demonstrated a dynamic evolution of mitochondrial gene expression and cellular energetics as the virus progresses from one organ to the next. Transcriptomic changes in the nasopharyngeal infected samples revealed that during initial SARS-CoV-2 infection, nDNA coded mitochondrial genes are blocked and the co-inhibited genes were found to group together as components of preassembly modules of the mitochondrial oxidative phosphorylation (OXPHOS) complexes I, II, III, IV and V. At the time of death for COVID-19 patients, we showed virtually all mitochondrial function were inhibited in the heart, suggesting cardiac mitochondrial dysfunction in longer term COVID-19 pathology. In addition, mTOR signaling and the integrated stress response were highly dysregulated throughout all organs. Lastly, mitochondrial inhibition was shown to activate HIF-1α and its target genes shifting cellular metabolism away from catabolism and towards viral synthesis. Our results indicate that manipulation of mitochondrial function may be an important approach for mitigating the severity of COIVID-19.

SARS-CoV-2 infection of human monocytes [[55](#CR55)] and human pulmonary alveolar epithelial (HPAEpiC) cells [[56](#CR56)] induced mitochondrial reactive oxygen species (mROS) production, increased HIF-1α protein levels and upregulated expression of HIF-1α target genes [[57](#CR57)]. The stability of hypoxia inducible factor-1α (HIF-1α) during a SARS-CoV-2 infection was shown to increase the production of pro-inflammatory cytokines and SARS-CoV-2 replication [[55](#CR55)–[57](#CR57)]. The expression of ORF3a in human embryonic kidney 293 T-antigen cells (HEK293T) cells increased the stability of HIF-1α and induced mROS production, which is an activator of HIF-1α. Together these results suggest that ORF3a induces mROS production to activate HIF-1α, which in turn triggers a shift in cellular metabolism to favor glycolysis, resulting in increased viral replication and transcription of pro-inflammatory cytokines.

Following host cell infection, the SARS-CoV-2 replication/transcription complex synthesizes ~30 kb viral genomes as well as the subgenomic RNAs required to encode for viral structural and mechanistic proteins. Between 1–5 h post-infection, the percentage of coronavirus-encoded protein per total cellular protein translation may increase by as much as 20,000 times, with the fraction of viral to cellular RNA ultimately reaching as high as 90% intracellularly [[58](#CR58)]. To accommodate this huge shift towards viral replication, there is certainly a requirement of a shift in cellular metabolism to accommodate for viral synthesis. An investigation of SARS-CoV-2 metabolism during the initial 48-hours post viral infection showed that amino acid availability and synthesis are altered, *de novo* purine synthesis intermediates are accumulated, intracellular glucose and folate are depleted, and lactate levels are elevated [[59](#CR59)]. This suggests a viral strategy of upregulating purine metabolism at the post-translational level to coincide with the shutting off of the majority of host proteins at translation levels.

Virus-infected cells also commonly exhibit the Warburg effect - increased glycolytic metabolism in the presence of inadequate oxygen for oxidative phosphorylation - to supply reducing equivalents and precursors for macromolecule biosynthesis, and to support generation of ATP needed for also increasing nucleotide and lipid biosynthesis. Metabolic shifts include dysregulated Ca2+ signaling and increased mitochondrial generation of ROS. How SARS-CoV-2 induces host cell nucleotide metabolism remains unanswered.

Mitochondrial metabolism and function are highly impacted in multiple ways. With the shift towards glycolysis, there is a reduction in oxidative phosphorylation affecting the mitochondria and its function. SARS-CoV-2 may interact with the mitochondria to destabilize its oxidative phosphorylation capacity. Coronavirus replication requires the formation of double-membrane vesicles (DMVs) derived from endoplasmic reticulum (ER). These DMVs serve as a site for viral replication and help conceal the virus from host cellular defenses. Interestingly, mitochondrial stress is known to induce mitochondria-derived vesicles (MDVs) that communicate with the ER. It is conceivable that SARS-CoV-2 disruption of mitochondrial function results in the induction of (double-membrane) MDVs. SARS-CoV-2 RNA present in the mitochondria induces mitochondrial dysfunction. Increased DMVs can provide opportunity for viruses to hide and replicate [[60](#CR60)].

SARS-CoV-2 and all subgenomic RNAs are enriched in the host mitochondria, and viral genome’s 5ʹ - and 3ʹ -UTRs contain distinct mitochondrial localization signals [[61](#CR61)], indicating that the viral RNA may hijack the mitochondria, an interesting hypothesis for experimental validation [[61](#CR61)]. Other recent studies have mapped physical interactions of SARS-CoV-2 encoded proteins with mitochondrial localized proteins. These interactions include: NSP8 interaction with mitochondrial ribosomal protein s2 (MRPS2), mitochondrial ribosomal protein s5 (MRPS5), mitochondrial ribosomal protein s25 (MRPS25), and mitochondrial ribosomal protein s27 (MRPS27) ribosomal proteins; ORF9c interaction with mitochondrial NADH:Ubiquinone Oxidoreductase Complex Assembly Factor 1 (NDUFAF1) and NADH:Ubiquinone Oxidoreductase Complex Assembly Factor 9 (NDUFB9); ORF10 interaction with TIMM8; and NSP7 interaction with mitochondrial NADH:Ubiquinone Oxidoreductase Complex Assembly Factor 2 (NDUFAF2). NDUFAF1, NDUFAF2, NDUFB9, and NADH:Ubiquinone Oxidoreductase Complex Assembly Factor 10 (NDUFA10) are all key players in the assembly of complex I, and NDUFA10 is suggested as being one of the master regulators of the SARS-CoV-2 pathology [[7](#CR7)]. Interactions were also observed between viral M protein and ATPase Na+/K+ Transporting Subunit Beta 1 (ATP1B1), ATPase H + Transporting V1 Subunit A (ATP6V1A), acyl-coenzyme A dehydrogenase (ACADM), Alpha-aminoadipic semialdehyde synthase (AASS), Peptidase, Mitochondrial Processing Subunit Beta (PMPCB), Pitrilysin Metallopeptidase 1 (PITRM1), Coenzyme Q8B (COQ8B), and Peptidase, Mitochondrial Processing Subunit Alpha (PMPCA); these proteins are each components of critical mitochondrial metabolic pathways. SARS-CoV-2-encoded ORF9b protein interacts and localizes with Translocase Of Outer Mitochondrial Membrane 70 (TOMM70) [[7](#CR7)], a mitochondrial import receptor important for transporting proteins into mitochondria and, more importantly, in modulating anti-viral cellular defense pathways [[62](#CR62)]. These mitochondrial interactions offer glimpses of the viral effect on glycolytic and oxidative phosphorylation pathways and the potential side effects.

Another example of COVID-19’s mitochondrial-related impacts is the over-production of cellular ROS [[63](#CR63)]. ROS and reactive nitrogen species have diverse functions in biological systems; oxidatively attacking pathogens, regulating cell proliferation, and key signaling functions [[64](#CR64)]. However, dysregulation of ROS is implicated in many diseases, including the hyper-inflammatory late phase of COVID-19 [[65](#CR65)]. As a part of normal redox metabolism, superoxide radicals are converted into hydrogen peroxide by the action of superoxide dismutase. The hydrogen peroxide is subsequently broken down into water by glutathione peroxidase. During COVID-19, isoforms of enzymes, including glutathione peroxidase and thioredoxin reductase, may be directly targeted and proteolyzed by the SARS-CoV-2 protease, Mpro.

SARS-CoV-2 is thought to suppress the ROS-associated Nuclear factor erythroid 2-related factor 2 (Nrf2) pathway. Nrf2 is a transcription factor that regulates the expression of antioxidant proteins that protect against oxidative damage. Dysregulation of the Nrf2 pathway will exacerbate the pro-oxidative stress caused by the virus [[66](#CR66)]. SARS-CoV-2 may suppress the accumulation of the selenoprotein transcripts, which are crucial for the correct functioning of Phospholipid hydroperoxide glutathione peroxidase (GPX4) and mitochondria function [[67](#CR67)]. This redox impairment would lead to a buildup of hydrogen peroxide, which could trigger inflammation by promoting the activity of the mitogen-activated protein kinase (MAPK), NF-κB, and the nuclear NOD-like receptor (NLR) family pyrin domain containing 3 (NLRP3) inflammasome [[68](#CR68)].

Due to the multiple effects of SARS-CoV-2 that alter cellular metabolic and oxidative states, there are multiple directions to deplete NAD+, loss of cellular ATP and reduced Poly-ADP-ribose polymerase (PARP) activity, each of which have cytotoxic effects of their own [[69](#CR69)]. In general, NADPH, synthesized from NAD+, is necessary for many key redox reactions; a reduced level of NADPH could play a mechanistic role in cellular metabolic changes from SARS-CoV-2 infection.

### SARS-CoV-2 mediated reduction in ATP and nitric oxide signaling induces cell stress

Cellular metabolism adapts to the alterations induced by SARS-CoV-2 infection of the cell (see Fig. [4](#Fig4)). These adaptations depend on the cell and tissue type. Here, we focus on ATP signaling, which is relevant to epithelial cells, and nitric oxide (NO) signaling, which tends to be perturbed in endothelial cells [[70](#CR70)]. Activation of each pathway at low levels provides protection to the host.

#### Fig. 4. SARS-CoV-2 viral internalization & cellular hijacking.

[![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5550/9349314/a49721ac06ca/41431_2022_1108_Fig4_HTML.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9349314_41431_2022_1108_Fig4_HTML.jpg)

[Open in a new tab](figure/Fig4/)

Metabolic pathways and shifts that lead to cellular dysregulation and viral activation to lead towards viral replication (above).

ATP production from oxidative phosphorylation, glycolysis, and other pathways is critical to support cellular physiology, but this molecule also has signaling properties, which can be particularly beneficial in epithelial cells [[70](#CR70)]. Perturbations in ATP generation induced by the virus in epithelial cells [[71](#CR71)] can lead to ATP release from the apical or basolateral spaces, and subsequent extracellular ATP signaling [[72](#CR72)]. It can stimulate P2 receptors on neighboring epithelial cells to activate signal transduction pathways and alter cellular function in adjacent cells even if they are not infected, thus priming naive host cells for confrontation with the virus [[71](#CR71), [72](#CR72)].

In the endothelium, nitric oxide directly affects mitochondrial metabolism through interaction with cytochrome C, providing cytoprotection against free radicals. However, reduction of NO bioavailability, due to the increased oxidative stress state caused by SARS-CoV-2-elevated superoxides, results in the formation of peroxynitrites (ONOO-). The reduced NO diffusion to neighboring vascular smooth muscle may impair vascular function [[73](#CR73)]. Peroxynitrite also causes injury to the mitochondria and reduces ATP synthesis, with all of the concomitant negative effects. Therefore, loss of NO bioavailability has major cellular consequences, inducing shifts in multiple enzymatic pathways, cell injury, and death.

Like ATP, NO acts as a biological signaling molecule. This dissolved gas rapidly diffuses across cell membranes and regulates various functions across the body [[73](#CR73)]. The vascular endothelium is the predominant cellular source of NO production, and it plays a critical role in maintaining cardiovascular function. Factors that reduce endothelial NO production (increased oxidative stress, changes in NO synthase synthesis) negatively affect endothelial function [[73](#CR73)]. The cascade of inflammation and oxidative stress triggered by COVID-19 leads to the formation of superoxide free radicals, impairing biological processes and increasing cytotoxicity in the host cells [[74](#CR74)]. The instantaneous reaction of superoxide and NO yields ONOO-, a powerful, cytotoxic nitrating agent. This reaction effectively destroys the NO, rendering it unavailable for its normal regulatory purposes. Thus, the downregulation of NO bioavailability is thought to be a central factor in the severity of COVID-19-associated endotheliitis and the onset of endothelial dysfunction [[75](#CR75)].

## Conclusion

The causative agent of the COVID-19 the pandemic, SARS-CoV-2, has caused loss of incomes, economic crises, morbidities, and loss of life worldwide. Here, we describe the virus and review state-of-the-art information about the processes it utilizes to enter and reprogram the human host machinery. We detail research on early infection using evidence from patient samples, organoids and cells, and non-human animal studies. Each of these has limitations but taken together provide unique observational and mechanistic insight on SARS-CoV-2 infection.

COVID-19 is a pleiotropic condition. Viral insults and subsequent cellular metabolic adaptations differ in the context of cell-type, genotype and environmental influences. Thus, much of what we have presented applies to specific cell types and contexts, and we have attempted to cover these contexts.

Key avenues of future research on SARS-CoV-2 infection and propagation include: 1) defining the mechanisms of how the virus enters cells, and the protein and receptor molecules that are critical to this process; 2) elucidating the dynamics of how protein machinery is captured and retrofitted for viral purposes in a cell-specific manner; 3) understanding how the host genetics and environment can affect the ability of the virus to infect; 4) understanding the impact of SARS-CoV-2 on glycolysis and oxidative phosphorylation; and 5) revealing how the mitochondria adapts to ultimately shift its physiology from steady-state.

In the best-case scenario for the SARS-CoV-2 virus, infection leads to a cascade of intracellular adaptations in which multiple networks are remodeled, from transcription to metabolism to signal transduction, shifting the invaded host cell from its original physiology into a SARS-CoV-2 replication system, and causing the emission of new viral particles and signaling molecules. The subsequent disease events will reverberate across the body’s cells and organs. This will be the subject of our Part 2 review (in preparation).

## Supplementary information

[Supplemental Table 1](/articles/instance/9349314/bin/41431_2022_1108_MOESM1_ESM.pdf) (100.4KB, pdf)

## Acknowledgements

The opinions expressed in this article are those of the authors and do not reflect the view of the National Institutes of Health, the Department of Health and Human Services, or the United States government.

## Author contributions

Conceptualization: DAJ; Writing – Original Draft: DAJ, SN; Writing – Review and Editing: AB, ESW, KKS, JWG, MJT, VZ, PMMV, DAJ, SN, NST. Visualization: DAJ, SN, JWG; Supervision: AB; Funding Acquisition: AB.

## Funding

This work was supported by supplemental funds for COVID-19 research from Translational Research Institute of Space Health through NASA Cooperative Agreement NNX16AO69A (T-0404) to AB, and by a NASA Space Biology Postdoctoral Fellowship (80NSSC19K0426) to SAN. MJT is a recipient of The Evelyn Grollman Glick Scholar Award and supported by research funding from The Dr. Miriam and Sheldon G. Adelson Medical Research Foundation, Van Andel Research Institute through the Van Andel Research Institute – Stand Up To Cancer Epigenetics Dream Team. Stand Up To Cancer is a program of the Entertainment Industry Foundation, administered by AACR, and Specialized Program of Research Excellence (SPORE) program, through the National Cancer Institute (NCI), grant P50CA254897.

## Competing interests

The authors declare no competing interests.

## Footnotes

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

These authors contributed equally: David A. Jamison and S. Anand Narayanan.

This author jointly supervised this work: Afshin Beheshti.

## Contributor Information

S. Anand Narayanan, Email: anarayan09@gmail.com

Afshin Beheshti, Email: afshin.beheshti@nasa.gov.

## Supplementary information

The online version contains supplementary material available at 10.1038/s41431-022-01108-8.

## References

* 1.Dong E, Du H, Gardner L. An interactive web-based dashboard to track COVID-19 in real time. Lancet Infect Dis. 2020;20:533–4. doi: 10.1016/S1473-3099(20)30120-1. [[DOI](https://doi.org/10.1016/S1473-3099%2820%2930120-1)] [[PMC free article](/articles/PMC7159018/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32087114/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Infect%20Dis&title=An%20interactive%20web-based%20dashboard%20to%20track%20COVID-19%20in%20real%20time&author=E%20Dong&author=H%20Du&author=L%20Gardner&volume=20&publication_year=2020&pages=533-4&pmid=32087114&doi=10.1016/S1473-3099(20)30120-1&)]
* 2.Van Egeren D, Novokhodko A, Stoddard M, Tran U, Zetter B, Rogers M, et al. Risk of rapid evolutionary escape from biomedical interventions targeting SARS-CoV-2 spike protein. PloS One. 2021;16:e0250780. doi: 10.1371/journal.pone.0250780. [[DOI](https://doi.org/10.1371/journal.pone.0250780)] [[PMC free article](/articles/PMC8081162/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33909660/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Risk%20of%20rapid%20evolutionary%20escape%20from%20biomedical%20interventions%20targeting%20SARS-CoV-2%20spike%20protein&author=D%20Van%20Egeren&author=A%20Novokhodko&author=M%20Stoddard&author=U%20Tran&author=B%20Zetter&volume=16&publication_year=2021&pages=e0250780&pmid=33909660&doi=10.1371/journal.pone.0250780&)]
* 3.Yao H, Song Y, Chen Y, Wu N, Xu J, Sun C, et al. Molecular architecture of the SARS-CoV-2 virus. Cell. 2020;183:730–8. doi: 10.1016/j.cell.2020.09.018. [[DOI](https://doi.org/10.1016/j.cell.2020.09.018)] [[PMC free article](/articles/PMC7474903/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32979942/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell.&title=Molecular%20architecture%20of%20the%20SARS-CoV-2%20virus&author=H%20Yao&author=Y%20Song&author=Y%20Chen&author=N%20Wu&author=J%20Xu&volume=183&publication_year=2020&pages=730-8&pmid=32979942&doi=10.1016/j.cell.2020.09.018&)]
* 4.Liu J, Xie W, Wang Y, Xiong Y, Chen S, Han J, et al. A comparative overview of COVID-19, MERS and SARS. Int J Surg. 2020;81:1–8. doi: 10.1016/j.ijsu.2020.07.032. [[DOI](https://doi.org/10.1016/j.ijsu.2020.07.032)] [[PMC free article](/articles/PMC7382925/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32730205/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Surg&title=A%20comparative%20overview%20of%20COVID-19,%20MERS%20and%20SARS&author=J%20Liu&author=W%20Xie&author=Y%20Wang&author=Y%20Xiong&author=S%20Chen&volume=81&publication_year=2020&pages=1-8&pmid=32730205&doi=10.1016/j.ijsu.2020.07.032&)]
* 5.Grifoni A, Sidney J, Zhang Y, Scheuermann RH, Peters B, Sette A. A sequence homology and bioinformatic approach can predict candidate targets for immune responses to SARS-CoV-2. Cell Host Microbe. 2020;27:671–80. doi: 10.1016/j.chom.2020.03.002. [[DOI](https://doi.org/10.1016/j.chom.2020.03.002)] [[PMC free article](/articles/PMC7142693/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32183941/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20Microbe&title=A%20sequence%20homology%20and%20bioinformatic%20approach%20can%20predict%20candidate%20targets%20for%20immune%20responses%20to%20SARS-CoV-2&author=A%20Grifoni&author=J%20Sidney&author=Y%20Zhang&author=RH%20Scheuermann&author=B%20Peters&volume=27&publication_year=2020&pages=671-80&pmid=32183941&doi=10.1016/j.chom.2020.03.002&)]
* 6.Caldaria A, Conforti C, Di Meo N, Dianzani C, Jafferany M, Lotti T, et al. COVID-19 and SARS: Differences and similarities. Dermatol Ther. 2020;33:e13395. [[DOI](https://doi.org/10.1111/dth.13395)] [[PMC free article](/articles/PMC7235519/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32277530/)]
* 7.Gordon DE, Jang GM, Bouhaddou M, Xu J, Obernier K, White KM, et al. A SARS-CoV-2 protein interaction map reveals targets for drug repurposing. Nature. 2020;583:459–68.. doi: 10.1038/s41586-020-2286-9. [[DOI](https://doi.org/10.1038/s41586-020-2286-9)] [[PMC free article](/articles/PMC7431030/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32353859/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature.&title=A%20SARS-CoV-2%20protein%20interaction%20map%20reveals%20targets%20for%20drug%20repurposing&author=DE%20Gordon&author=GM%20Jang&author=M%20Bouhaddou&author=J%20Xu&author=K%20Obernier&volume=583&publication_year=2020&pages=459-68.&pmid=32353859&doi=10.1038/s41586-020-2286-9&)]
* 8.Benvenuto D, Giovanetti M, Salemi M, Prosperi M, De Flora C, Junior Alcantara LC, Angeletti S, Ciccozzi M. The global spread of 2019-nCoV: a molecular evolutionary analysis. Pathog Glob Health. 2020;114:64–7. doi: 10.1080/20477724.2020.1725339. [[DOI](https://doi.org/10.1080/20477724.2020.1725339)] [[PMC free article](/articles/PMC7099638/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32048560/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pathog%20Glob%20Health&title=The%20global%20spread%20of%202019-nCoV:%20a%20molecular%20evolutionary%20analysis&author=D%20Benvenuto&author=M%20Giovanetti&author=M%20Salemi&author=M%20Prosperi&author=C%20De%20Flora&volume=114&publication_year=2020&pages=64-7&pmid=32048560&doi=10.1080/20477724.2020.1725339&)]
* 9.Alene M, Yismaw L, Assemie MA, Ketema DB, Mengist B, Kassie B, et al. Magnitude of asymptomatic COVID-19 cases throughout the course of infection: A systematic review and meta-analysis. PloS One. 2021;16:e0249090. doi: 10.1371/journal.pone.0249090. [[DOI](https://doi.org/10.1371/journal.pone.0249090)] [[PMC free article](/articles/PMC7987199/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33755688/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Magnitude%20of%20asymptomatic%20COVID-19%20cases%20throughout%20the%20course%20of%20infection:%20A%20systematic%20review%20and%20meta-analysis&author=M%20Alene&author=L%20Yismaw&author=MA%20Assemie&author=DB%20Ketema&author=B%20Mengist&volume=16&publication_year=2021&pages=e0249090&pmid=33755688&doi=10.1371/journal.pone.0249090&)]
* 10.Mahdy MA, Younis W, Ewaida Z. An overview of SARS-CoV-2 and animal infection. Front Vet Sci. 2020;7:1084. doi: 10.3389/fvets.2020.596391. [[DOI](https://doi.org/10.3389/fvets.2020.596391)] [[PMC free article](/articles/PMC7759518/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33363234/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Vet%20Sci&title=An%20overview%20of%20SARS-CoV-2%20and%20animal%20infection&author=MA%20Mahdy&author=W%20Younis&author=Z%20Ewaida&volume=7&publication_year=2020&pages=1084&pmid=33363234&doi=10.3389/fvets.2020.596391&)]
* 11.Fang S, Li K, Shen J, Liu S, Liu J, Yang L, et al. GESS: a database of global evaluation of SARS-CoV-2/hCoV-19 sequences. Nucleic Acids Res. 2021;49:D706–14. doi: 10.1093/nar/gkaa808. [[DOI](https://doi.org/10.1093/nar/gkaa808)] [[PMC free article](/articles/PMC7778918/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33045727/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=GESS:%20a%20database%20of%20global%20evaluation%20of%20SARS-CoV-2/hCoV-19%20sequences&author=S%20Fang&author=K%20Li&author=J%20Shen&author=S%20Liu&author=J%20Liu&volume=49&publication_year=2021&pages=D706-14&pmid=33045727&doi=10.1093/nar/gkaa808&)]
* 12.Banoun H. Evolution of SARS-CoV-2: Review of Mutations, Role of the Host Immune System. Nephron. 2021;145:392–403. [[DOI](https://doi.org/10.1159/000515417)] [[PMC free article](/articles/PMC8247830/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33910211/)]
* 13.Meng B, Kemp SA, Papa G, Datir R, Ferreira IATM, Marelli S, et al. Recurrent emergence of SARS-CoV-2 spike deletion H69/V70 and its role in the Alpha variant B.1.1.7. Cell Rep. 2021;29:109292. [[DOI](https://doi.org/10.1016/j.celrep.2021.109292)] [[PMC free article](/articles/PMC8185188/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34166617/)]
* 14.Centers for Disease Control and Prevention. “SARS-CoV-2 variant classifications and definitions.” (2021).
* 15.Letko M, Marzi A, Munster V. Functional assessment of cell entry and receptor usage for SARS-CoV-2 and other lineage B betacoronaviruses. Nat Microbiol. 2020;5:562–9. doi: 10.1038/s41564-020-0688-y. [[DOI](https://doi.org/10.1038/s41564-020-0688-y)] [[PMC free article](/articles/PMC7095430/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32094589/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Microbiol&title=Functional%20assessment%20of%20cell%20entry%20and%20receptor%20usage%20for%20SARS-CoV-2%20and%20other%20lineage%20B%20betacoronaviruses&author=M%20Letko&author=A%20Marzi&author=V%20Munster&volume=5&publication_year=2020&pages=562-9&pmid=32094589&doi=10.1038/s41564-020-0688-y&)]
* 16.Dai L, Gao GF. Viral targets for vaccines against COVID-19. Nat Rev Immunol. 2021;21:73–82. doi: 10.1038/s41577-020-00480-0. [[DOI](https://doi.org/10.1038/s41577-020-00480-0)] [[PMC free article](/articles/PMC7747004/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33340022/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol.&title=Viral%20targets%20for%20vaccines%20against%20COVID-19&author=L%20Dai&author=GF%20Gao&volume=21&publication_year=2021&pages=73-82&pmid=33340022&doi=10.1038/s41577-020-00480-0&)]
* 17.Ali A, Vijayan R. Dynamics of the ACE2–SARS-CoV-2/SARS-CoV spike protein interface reveal unique mechanisms. Sci Rep. 2020;10:1–2. doi: 10.1038/s41598-019-56847-4. [[DOI](https://doi.org/10.1038/s41598-019-56847-4)] [[PMC free article](/articles/PMC7449962/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32848162/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Dynamics%20of%20the%20ACE2%E2%80%93SARS-CoV-2/SARS-CoV%20spike%20protein%20interface%20reveal%20unique%20mechanisms&author=A%20Ali&author=R%20Vijayan&volume=10&publication_year=2020&pages=1-2&pmid=32848162&doi=10.1038/s41598-019-56847-4&)]
* 18.Song W, Gui M, Wang X, Xiang Y. Cryo-EM structure of the SARS coronavirus spike glycoprotein in complex with its host cell receptor ACE2. PLoS Pathog. 2018;14:e1007236. doi: 10.1371/journal.ppat.1007236. [[DOI](https://doi.org/10.1371/journal.ppat.1007236)] [[PMC free article](/articles/PMC6107290/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30102747/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&title=Cryo-EM%20structure%20of%20the%20SARS%20coronavirus%20spike%20glycoprotein%20in%20complex%20with%20its%20host%20cell%20receptor%20ACE2&author=W%20Song&author=M%20Gui&author=X%20Wang&author=Y%20Xiang&volume=14&publication_year=2018&pages=e1007236&pmid=30102747&doi=10.1371/journal.ppat.1007236&)]
* 19.Chu H, Chan JF, Yuen TT, Shuai H, Yuan S, Wang Y, et al. Comparative tropism, replication kinetics, and cell damage profiling of SARS-CoV-2 and SARS-CoV with implications for clinical manifestations, transmissibility, and laboratory studies of COVID-19: an observational study. Lancet Microbe. 2020;1:e14–23. doi: 10.1016/S2666-5247(20)30004-5. [[DOI](https://doi.org/10.1016/S2666-5247%2820%2930004-5)] [[PMC free article](/articles/PMC7173822/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32835326/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Microbe&title=Comparative%20tropism,%20replication%20kinetics,%20and%20cell%20damage%20profiling%20of%20SARS-CoV-2%20and%20SARS-CoV%20with%20implications%20for%20clinical%20manifestations,%20transmissibility,%20and%20laboratory%20studies%20of%20COVID-19:%20an%20observational%20study&author=H%20Chu&author=JF%20Chan&author=TT%20Yuen&author=H%20Shuai&author=S%20Yuan&volume=1&publication_year=2020&pages=e14-23&pmid=32835326&doi=10.1016/S2666-5247(20)30004-5&)]
* 20.Yan R, Zhang Y, Li Y, Ye F, Guo Y, Xia L, et al. Structural basis for the different states of the spike protein of SARS-CoV-2 in complex with ACE2. Cell Res. 2021;31:717–9. doi: 10.1038/s41422-021-00490-0. [[DOI](https://doi.org/10.1038/s41422-021-00490-0)] [[PMC free article](/articles/PMC7972335/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33737693/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=Structural%20basis%20for%20the%20different%20states%20of%20the%20spike%20protein%20of%20SARS-CoV-2%20in%20complex%20with%20ACE2&author=R%20Yan&author=Y%20Zhang&author=Y%20Li&author=F%20Ye&author=Y%20Guo&volume=31&publication_year=2021&pages=717-9&pmid=33737693&doi=10.1038/s41422-021-00490-0&)]
* 21.Puelles VG, Lütgehetmann M, Lindenmeyer MT, Sperhake JP, Wong MN, Allweiss L, et al. Multiorgan and renal tropism of SARS-CoV-2. N. Engl J Med. 2020;383:590–2. doi: 10.1056/NEJMc2011400. [[DOI](https://doi.org/10.1056/NEJMc2011400)] [[PMC free article](/articles/PMC7240771/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32402155/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl%20J%20Med&title=Multiorgan%20and%20renal%20tropism%20of%20SARS-CoV-2&author=VG%20Puelles&author=M%20L%C3%BCtgehetmann&author=MT%20Lindenmeyer&author=JP%20Sperhake&author=MN%20Wong&volume=383&publication_year=2020&pages=590-2&pmid=32402155&doi=10.1056/NEJMc2011400&)]
* 22.Menter T, Haslbauer JD, Nienhold R, Savic S, Hopfer H, Deigendesch N, et al. Postmortem examination of COVID‐19 patients reveals diffuse alveolar damage with severe capillary congestion and variegated findings in lungs and other organs suggesting vascular dysfunction. Histopathology. 2020;77:198–209. doi: 10.1111/his.14134. [[DOI](https://doi.org/10.1111/his.14134)] [[PMC free article](/articles/PMC7496150/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32364264/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Histopathology.&title=Postmortem%20examination%20of%20COVID%E2%80%9019%20patients%20reveals%20diffuse%20alveolar%20damage%20with%20severe%20capillary%20congestion%20and%20variegated%20findings%20in%20lungs%20and%20other%20organs%20suggesting%20vascular%20dysfunction&author=T%20Menter&author=JD%20Haslbauer&author=R%20Nienhold&author=S%20Savic&author=H%20Hopfer&volume=77&publication_year=2020&pages=198-209&pmid=32364264&doi=10.1111/his.14134&)]
* 23.Ziegler CG, Allon SJ, Nyquist SK, Mbano IM, Miao VN, Tzouanas CN, et al. SARS-CoV-2 receptor ACE2 is an interferon-stimulated gene in human airway epithelial cells and is detected in specific cell subsets across tissues. Cell. 2020;181:1016–35.. doi: 10.1016/j.cell.2020.04.035. [[DOI](https://doi.org/10.1016/j.cell.2020.04.035)] [[PMC free article](/articles/PMC7252096/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32413319/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell.&title=SARS-CoV-2%20receptor%20ACE2%20is%20an%20interferon-stimulated%20gene%20in%20human%20airway%20epithelial%20cells%20and%20is%20detected%20in%20specific%20cell%20subsets%20across%20tissues&author=CG%20Ziegler&author=SJ%20Allon&author=SK%20Nyquist&author=IM%20Mbano&author=VN%20Miao&volume=181&publication_year=2020&pages=1016-35.&pmid=32413319&doi=10.1016/j.cell.2020.04.035&)]
* 24.Park J, Foox J, Hether T, Danko DC, Warren S, Kim Y, et al. System-wide transcriptome damage and tissue identity loss in COVID-19 patients. Cell Rep Med. 2022;3:100522. [[DOI](https://doi.org/10.1016/j.xcrm.2022.100522)] [[PMC free article](/articles/PMC8784611/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35233546/)]
* 25.Han Y, Duan X, Yang L, Nilsson-Payant BE, Wang P, Duan F, et al. Identification of SARS-CoV-2 inhibitors using lung and colonic organoids. Nature. 2021;589:270–5. doi: 10.1038/s41586-020-2901-9. [[DOI](https://doi.org/10.1038/s41586-020-2901-9)] [[PMC free article](/articles/PMC8034380/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33116299/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature.&title=Identification%20of%20SARS-CoV-2%20inhibitors%20using%20lung%20and%20colonic%20organoids&author=Y%20Han&author=X%20Duan&author=L%20Yang&author=BE%20Nilsson-Payant&author=P%20Wang&volume=589&publication_year=2021&pages=270-5&pmid=33116299&doi=10.1038/s41586-020-2901-9&)]
* 26.Ashraf UM, Abokor AA, Edwards JM, Waigi EW, Royfman RS, Hasan SA, et al. SARS-CoV-2, ACE2 expression, and systemic organ invasion. Physiological Genomics. 2021;53:51–60. doi: 10.1152/physiolgenomics.00087.2020. [[DOI](https://doi.org/10.1152/physiolgenomics.00087.2020)] [[PMC free article](/articles/PMC7900915/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33275540/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiological%20Genomics&title=SARS-CoV-2,%20ACE2%20expression,%20and%20systemic%20organ%20invasion&author=UM%20Ashraf&author=AA%20Abokor&author=JM%20Edwards&author=EW%20Waigi&author=RS%20Royfman&volume=53&publication_year=2021&pages=51-60&pmid=33275540&doi=10.1152/physiolgenomics.00087.2020&)]
* 27.Suryamohan K, Diwanji D, Stawiski EW, Gupta R, Miersch S, Liu J, et al. Human ACE2 receptor polymorphisms and altered susceptibility to SARS-CoV-2. Commun Biol. 2021;4:1–1. doi: 10.1038/s42003-021-02030-3. [[DOI](https://doi.org/10.1038/s42003-021-02030-3)] [[PMC free article](/articles/PMC8041869/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33846513/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Commun%20Biol&title=Human%20ACE2%20receptor%20polymorphisms%20and%20altered%20susceptibility%20to%20SARS-CoV-2&author=K%20Suryamohan&author=D%20Diwanji&author=EW%20Stawiski&author=R%20Gupta&author=S%20Miersch&volume=4&publication_year=2021&pages=1-1&pmid=33846513&doi=10.1038/s42003-021-02030-3&)]
* 28.Ozono S, Zhang Y, Ode H, Sano K, Tan TS, Imai K, et al. SARS-CoV-2 D614G spike mutation increases entry efficiency with enhanced ACE2-binding affinity. Nat Commun. 2021;12:1–9. doi: 10.1038/s41467-021-21118-2. [[DOI](https://doi.org/10.1038/s41467-021-21118-2)] [[PMC free article](/articles/PMC7870668/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33558493/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=SARS-CoV-2%20D614G%20spike%20mutation%20increases%20entry%20efficiency%20with%20enhanced%20ACE2-binding%20affinity&author=S%20Ozono&author=Y%20Zhang&author=H%20Ode&author=K%20Sano&author=TS%20Tan&volume=12&publication_year=2021&pages=1-9&pmid=33558493&doi=10.1038/s41467-021-21118-2&)]
* 29.Sodhi CP, Wohlford-Lenane C, Yamaguchi Y, Prindle T, Fulton WB, Wang S, et al. Attenuation of pulmonary ACE2 activity impairs inactivation of des-Arg9 bradykinin/BKB1R axis and facilitates LPS-induced neutrophil infiltration. Am J Physiol-Lung Cell Mol Physiol. 2018;314:L17–31. doi: 10.1152/ajplung.00498.2016. [[DOI](https://doi.org/10.1152/ajplung.00498.2016)] [[PMC free article](/articles/PMC5866432/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28935640/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol-Lung%20Cell%20Mol%20Physiol&title=Attenuation%20of%20pulmonary%20ACE2%20activity%20impairs%20inactivation%20of%20des-Arg9%20bradykinin/BKB1R%20axis%20and%20facilitates%20LPS-induced%20neutrophil%20infiltration&author=CP%20Sodhi&author=C%20Wohlford-Lenane&author=Y%20Yamaguchi&author=T%20Prindle&author=WB%20Fulton&volume=314&publication_year=2018&pages=L17-31&pmid=28935640&doi=10.1152/ajplung.00498.2016&)]
* 30.Calixto JB, Medeiros R, Fernandes ES, Ferreira J, Cabrini DA, Campos MM. Kinin B1 receptors: key G‐protein‐coupled receptors and their role in inflammatory and painful processes. Br J Pharmacol. 2004;143:803–18. doi: 10.1038/sj.bjp.0706012. [[DOI](https://doi.org/10.1038/sj.bjp.0706012)] [[PMC free article](/articles/PMC1575942/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15520046/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=Kinin%20B1%20receptors:%20key%20G%E2%80%90protein%E2%80%90coupled%20receptors%20and%20their%20role%20in%20inflammatory%20and%20painful%20processes&author=JB%20Calixto&author=R%20Medeiros&author=ES%20Fernandes&author=J%20Ferreira&author=DA%20Cabrini&volume=143&publication_year=2004&pages=803-18&pmid=15520046&doi=10.1038/sj.bjp.0706012&)]
* 31.Huang Y, Yang C, Xu XF, Xu W, Liu SW. Structural and functional properties of SARS-CoV-2 spike protein: potential antivirus drug development for COVID-19. Acta Pharmacologica Sin. 2020;41:1141–9. doi: 10.1038/s41401-020-0485-4. [[DOI](https://doi.org/10.1038/s41401-020-0485-4)] [[PMC free article](/articles/PMC7396720/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32747721/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Pharmacologica%20Sin&title=Structural%20and%20functional%20properties%20of%20SARS-CoV-2%20spike%20protein:%20potential%20antivirus%20drug%20development%20for%20COVID-19&author=Y%20Huang&author=C%20Yang&author=XF%20Xu&author=W%20Xu&author=SW%20Liu&volume=41&publication_year=2020&pages=1141-9&pmid=32747721&doi=10.1038/s41401-020-0485-4&)]
* 32.Lan J, Ge J, Yu J, Shan S, Zhou H, Fan S, et al. Structure of the SARS-CoV-2 spike receptor-binding domain bound to the ACE2 receptor. Nature. 2020;581:215–20.. doi: 10.1038/s41586-020-2180-5. [[DOI](https://doi.org/10.1038/s41586-020-2180-5)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32225176/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature.&title=Structure%20of%20the%20SARS-CoV-2%20spike%20receptor-binding%20domain%20bound%20to%20the%20ACE2%20receptor&author=J%20Lan&author=J%20Ge&author=J%20Yu&author=S%20Shan&author=H%20Zhou&volume=581&publication_year=2020&pages=215-20.&pmid=32225176&doi=10.1038/s41586-020-2180-5&)]
* 33.Davidson AM, Wysocki J, Batlle D. Interaction of SARS-CoV-2 and other coronavirus with ACE (angiotensin-converting enzyme)-2 as their main receptor: therapeutic implications. Hypertension. 2020;76:1339–49.. doi: 10.1161/HYPERTENSIONAHA.120.15256. [[DOI](https://doi.org/10.1161/HYPERTENSIONAHA.120.15256)] [[PMC free article](/articles/PMC7480804/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32851855/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Interaction%20of%20SARS-CoV-2%20and%20other%20coronavirus%20with%20ACE%20(angiotensin-converting%20enzyme)-2%20as%20their%20main%20receptor:%20therapeutic%20implications&author=AM%20Davidson&author=J%20Wysocki&author=D%20Batlle&volume=76&publication_year=2020&pages=1339-49.&pmid=32851855&doi=10.1161/HYPERTENSIONAHA.120.15256&)]
* 34.Ko PJ, Woodrow C, Dubreuil MM, Martin BR, Skouta R, Bassik MC, et al. A ZDHHC5-GOLGA7 protein acyltransferase complex promotes nonapoptotic cell death. Cell Chem Biol. 2019;26:1716–24.. doi: 10.1016/j.chembiol.2019.09.014. [[DOI](https://doi.org/10.1016/j.chembiol.2019.09.014)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31631010/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Chem%20Biol&title=A%20ZDHHC5-GOLGA7%20protein%20acyltransferase%20complex%20promotes%20nonapoptotic%20cell%20death&author=PJ%20Ko&author=C%20Woodrow&author=MM%20Dubreuil&author=BR%20Martin&author=R%20Skouta&volume=26&publication_year=2019&pages=1716-24.&pmid=31631010&doi=10.1016/j.chembiol.2019.09.014&)]
* 35.Van De Veerdonk F, Netea MG, Van Deuren M, Van Der Meer JW, De Mast Q, Bruggemann RJ, et al. Kinins and cytokines in COVID-19: a comprehensive pathophysiological approach.
* 36.Mohammad S, Bouchama A, Mohammad Alharbi B, Rashid M, Saleem Khatlani T, Gaber NS, et al. SARS-CoV-2 ORF8 and SARS-CoV ORF8ab: genomic divergence and functional convergence. Pathogens. 2020;9:677. doi: 10.3390/pathogens9090677. [[DOI](https://doi.org/10.3390/pathogens9090677)] [[PMC free article](/articles/PMC7558349/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32825438/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pathogens.&title=SARS-CoV-2%20ORF8%20and%20SARS-CoV%20ORF8ab:%20genomic%20divergence%20and%20functional%20convergence&author=S%20Mohammad&author=A%20Bouchama&author=B%20Mohammad%20Alharbi&author=M%20Rashid&author=T%20Saleem%20Khatlani&volume=9&publication_year=2020&pages=677&pmid=32825438&doi=10.3390/pathogens9090677&)]
* 37.Castaño-Rodriguez C, Honrubia JM, Gutiérrez-Álvarez J, DeDiego ML, Nieto-Torres JL, Jimenez-Guardeño JM, et al. Role of severe acute respiratory syndrome coronavirus viroporins E, 3a, and 8a in replication and pathogenesis. MBio. 2018;9:e02325–17. doi: 10.1128/mBio.02325-17. [[DOI](https://doi.org/10.1128/mBio.02325-17)] [[PMC free article](/articles/PMC5964350/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29789363/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=MBio&title=Role%20of%20severe%20acute%20respiratory%20syndrome%20coronavirus%20viroporins%20E,%203a,%20and%208a%20in%20replication%20and%20pathogenesis&author=C%20Casta%C3%B1o-Rodriguez&author=JM%20Honrubia&author=J%20Guti%C3%A9rrez-%C3%81lvarez&author=ML%20DeDiego&author=JL%20Nieto-Torres&volume=9&publication_year=2018&pages=e02325-17&pmid=29789363&doi=10.1128/mBio.02325-17&)]
* 38.McClenaghan C, Hanson A, Lee SJ, Nichols CG. Coronavirus proteins as ion channels: Current and potential research. Front Immunol. 2020;11:2651. doi: 10.3389/fimmu.2020.573339. [[DOI](https://doi.org/10.3389/fimmu.2020.573339)] [[PMC free article](/articles/PMC7586316/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33154751/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=Coronavirus%20proteins%20as%20ion%20channels:%20Current%20and%20potential%20research&author=C%20McClenaghan&author=A%20Hanson&author=SJ%20Lee&author=CG%20Nichols&volume=11&publication_year=2020&pages=2651&pmid=33154751&doi=10.3389/fimmu.2020.573339&)]
* 39.Di Filippo L, Formenti AM, Rovere-Querini P, Carlucci M, Conte C, Ciceri F, et al. Hypocalcemia is highly prevalent and predicts hospitalization in patients with COVID-19. Endocrine. 2020;68:475–8. doi: 10.1007/s12020-020-02383-5. [[DOI](https://doi.org/10.1007/s12020-020-02383-5)] [[PMC free article](/articles/PMC7292572/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32533508/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrine.&title=Hypocalcemia%20is%20highly%20prevalent%20and%20predicts%20hospitalization%20in%20patients%20with%20COVID-19&author=L%20Di%20Filippo&author=AM%20Formenti&author=P%20Rovere-Querini&author=M%20Carlucci&author=C%20Conte&volume=68&publication_year=2020&pages=475-8&pmid=32533508&doi=10.1007/s12020-020-02383-5&)]
* 40.Schett G, Manger B, Simon D, Caporali R COVID-19 revisiting inflammatory pathways of arthritis. Nat Rev Rheumatol. 2020;16:465–70. [[DOI](https://doi.org/10.1038/s41584-020-0451-z)] [[PMC free article](/articles/PMC7304381/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32561873/)]
* 41.Huang J, Hume AJ, Abo KM, Werder RB, Villacorta-Martin C, Alysandratos KD, et al. SARS-CoV-2 infection of pluripotent stem cell-derived human lung alveolar type 2 cells elicits a rapid epithelial-intrinsic inflammatory response. Cell Stem Cell. 2020;27:962–73.. doi: 10.1016/j.stem.2020.09.013. [[DOI](https://doi.org/10.1016/j.stem.2020.09.013)] [[PMC free article](/articles/PMC7500949/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32979316/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Stem%20Cell&title=SARS-CoV-2%20infection%20of%20pluripotent%20stem%20cell-derived%20human%20lung%20alveolar%20type%202%20cells%20elicits%20a%20rapid%20epithelial-intrinsic%20inflammatory%20response&author=J%20Huang&author=AJ%20Hume&author=KM%20Abo&author=RB%20Werder&author=C%20Villacorta-Martin&volume=27&publication_year=2020&pages=962-73.&pmid=32979316&doi=10.1016/j.stem.2020.09.013&)]
* 42.Zheng M, Karki R, Williams EP, Yang D, Fitzpatrick E, Vogel P, et al. TLR2 senses the SARS-CoV-2 envelope protein to produce inflammatory cytokines. Nat Immunology. 2021;1-0. [[DOI](https://doi.org/10.1038/s41590-021-00937-x)] [[PMC free article](/articles/PMC8882317/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33963333/)]
* 43.Mukherjee R, Bhattacharya A, Bojkova D, Mehdipour AR, Shin D, Khan KS, et al. Famotidine inhibits toll-like receptor 3-mediated inflammatory signaling in SARS-CoV-2 infection. J Biolog Chem. 2021;297:100925. [[DOI](https://doi.org/10.1016/j.jbc.2021.100925)] [[PMC free article](/articles/PMC8241579/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34214498/)]
* 44.Zhang Q, Bastard P, Liu Z, Le Pen J, Moncada-Velez M, Chen J, et al. Inborn errors of type I IFN immunity in patients with life-threatening COVID-19. Science. 2020;370:eabd4570. [[DOI](https://doi.org/10.1126/science.abd4570)] [[PMC free article](/articles/PMC7857407/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32972995/)]
* 45.Bastard P, Rosen LB, Zhang Q, Michailidis E, Hoffmann HH, Zhang Y, et al. Autoantibodies against type I IFNs in patients with life-threatening COVID-19. Science. 2020;370:eabd4585. [[DOI](https://doi.org/10.1126/science.abd4585)] [[PMC free article](/articles/PMC7857397/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32972996/)]
* 46.Hung IF, Lung KC, Tso EY, Liu R, Chung TW, Chu MY, et al. Triple combination of interferon beta-1b, lopinavir–ritonavir, and ribavirin in the treatment of patients admitted to hospital with COVID-19: an open-label, randomised, phase 2 trial. Lancet. 2020;395:1695–704. doi: 10.1016/S0140-6736(20)31042-4. [[DOI](https://doi.org/10.1016/S0140-6736%2820%2931042-4)] [[PMC free article](/articles/PMC7211500/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32401715/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Triple%20combination%20of%20interferon%20beta-1b,%20lopinavir%E2%80%93ritonavir,%20and%20ribavirin%20in%20the%20treatment%20of%20patients%20admitted%20to%20hospital%20with%20COVID-19:%20an%20open-label,%20randomised,%20phase%202%20trial&author=IF%20Hung&author=KC%20Lung&author=EY%20Tso&author=R%20Liu&author=TW%20Chung&volume=395&publication_year=2020&pages=1695-704&pmid=32401715&doi=10.1016/S0140-6736(20)31042-4&)]
* 47.Irvani SS, Golmohammadi M, Pourhoseingholi MA, Shokouhi S, Darazam IA. Effectiveness of Interferon Beta 1a, compared to Interferon Beta 1b and the usual therapeutic regimen to treat adults with moderate to severe COVID-19: structured summary of a study protocol for a randomized controlled trial. Trials. 2020;21:1–3. doi: 10.1186/s13063-020-04382-3. [[DOI](https://doi.org/10.1186/s13063-020-04382-3)] [[PMC free article](/articles/PMC7268172/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32493468/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trials.&title=Effectiveness%20of%20Interferon%20Beta%201a,%20compared%20to%20Interferon%20Beta%201b%20and%20the%20usual%20therapeutic%20regimen%20to%20treat%20adults%20with%20moderate%20to%20severe%20COVID-19:%20structured%20summary%20of%20a%20study%20protocol%20for%20a%20randomized%20controlled%20trial&author=SS%20Irvani&author=M%20Golmohammadi&author=MA%20Pourhoseingholi&author=S%20Shokouhi&author=IA%20Darazam&volume=21&publication_year=2020&pages=1-3&pmid=32493468&doi=10.1186/s13063-020-04382-3&)]
* 48.Thevarajan I, Buising KL, Cowie BC. Clinical presentation and management of COVID-19. Med J Aust. 2020;213:134–9. doi: 10.5694/mja2.50698. [[DOI](https://doi.org/10.5694/mja2.50698)] [[PMC free article](/articles/PMC7404664/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32677734/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20J%20Aust&title=Clinical%20presentation%20and%20management%20of%20COVID-19&author=I%20Thevarajan&author=KL%20Buising&author=BC%20Cowie&volume=213&publication_year=2020&pages=134-9&pmid=32677734&doi=10.5694/mja2.50698&)]
* 49.Tan L, Wang Q, Zhang D, Ding J, Huang Q, Tang YQ, et al. Lymphopenia predicts disease severity of COVID-19: a descriptive and predictive study. Signal Transduct Target Ther. 2020;5:1–3. doi: 10.1038/s41392-019-0089-y. [[DOI](https://doi.org/10.1038/s41392-019-0089-y)] [[PMC free article](/articles/PMC7100419/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32296069/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Signal%20Transduct%20Target%20Ther&title=Lymphopenia%20predicts%20disease%20severity%20of%20COVID-19:%20a%20descriptive%20and%20predictive%20study&author=L%20Tan&author=Q%20Wang&author=D%20Zhang&author=J%20Ding&author=Q%20Huang&volume=5&publication_year=2020&pages=1-3&pmid=32296069&doi=10.1038/s41392-019-0089-y&)]
* 50.Yang J, Wise L, Fukuchi KI. TLR4 Cross-talk with NLRP3 inflammasome and complement signaling pathways in Alzheimer’s disease. Front Immunol. 2020;11:724. doi: 10.3389/fimmu.2020.00724. [[DOI](https://doi.org/10.3389/fimmu.2020.00724)] [[PMC free article](/articles/PMC7190872/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32391019/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=TLR4%20Cross-talk%20with%20NLRP3%20inflammasome%20and%20complement%20signaling%20pathways%20in%20Alzheimer%E2%80%99s%20disease&author=J%20Yang&author=L%20Wise&author=KI%20Fukuchi&volume=11&publication_year=2020&pages=724&pmid=32391019&doi=10.3389/fimmu.2020.00724&)]
* 51.Li JY, Liao CH, Wang Q, Tan YJ, Luo R, Qiu Y, et al. The ORF6, ORF8 and nucleocapsid proteins of SARS-CoV-2 inhibit type I interferon signaling pathway. Virus Res. 2020;286:198074. doi: 10.1016/j.virusres.2020.198074. [[DOI](https://doi.org/10.1016/j.virusres.2020.198074)] [[PMC free article](/articles/PMC7309931/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32589897/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Virus%20Res&title=The%20ORF6,%20ORF8%20and%20nucleocapsid%20proteins%20of%20SARS-CoV-2%20inhibit%20type%20I%20interferon%20signaling%20pathway&author=JY%20Li&author=CH%20Liao&author=Q%20Wang&author=YJ%20Tan&author=R%20Luo&volume=286&publication_year=2020&pages=198074&pmid=32589897&doi=10.1016/j.virusres.2020.198074&)]
* 52.Lei X, Dong X, Ma R, Wang W, Xiao X, Tian Z, et al. Activation and evasion of type I interferon responses by SARS-CoV-2. Nat Commun. 2020;11:1–2. doi: 10.1038/s41467-019-13993-7. [[DOI](https://doi.org/10.1038/s41467-019-13993-7)] [[PMC free article](/articles/PMC7392898/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32733001/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Activation%20and%20evasion%20of%20type%20I%20interferon%20responses%20by%20SARS-CoV-2&author=X%20Lei&author=X%20Dong&author=R%20Ma&author=W%20Wang&author=X%20Xiao&volume=11&publication_year=2020&pages=1-2&pmid=32733001&doi=10.1038/s41467-019-13993-7&)]
* 53.Codo AC, Davanzo GG, de Brito Monteiro L, de Souza GF, Muraro SP, Virgilio-da-Silva JV, et al. Elevated glucose levels favor SARS-CoV-2 infection and monocyte response through a HIF-1α/glycolysis-dependent axis. Cell Metab. 2020;32:437–46.. doi: 10.1016/j.cmet.2020.07.007. [[DOI](https://doi.org/10.1016/j.cmet.2020.07.007)] [[PMC free article](/articles/PMC7367032/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32697943/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Metab&title=Elevated%20glucose%20levels%20favor%20SARS-CoV-2%20infection%20and%20monocyte%20response%20through%20a%20HIF-1%CE%B1/glycolysis-dependent%20axis&author=AC%20Codo&author=GG%20Davanzo&author=L%20de%20Brito%20Monteiro&author=GF%20de%20Souza&author=SP%20Muraro&volume=32&publication_year=2020&pages=437-46.&pmid=32697943&doi=10.1016/j.cmet.2020.07.007&)]
* 54.Guarnieri JW, Dybas JM, Fazelinia H, Kim MS, Frere J, Zhang Y, et al. Targeted down regulation of core mitochondrial genes during SARS-CoV-2 infection. bioRxiv. 2022 Feb 22.
* 55.Wang P, Luo R, Zhang M, Wang Y, Song T, Tao T, et al. A cross-talk between epithelium and endothelium mediates human alveolar–capillary injury during SARS-CoV-2 infection. Cell death Dis. 2020;11:1–7. doi: 10.1038/s41419-019-2182-0. [[DOI](https://doi.org/10.1038/s41419-019-2182-0)] [[PMC free article](/articles/PMC7721862/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33293527/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20death%20Dis&title=A%20cross-talk%20between%20epithelium%20and%20endothelium%20mediates%20human%20alveolar%E2%80%93capillary%20injury%20during%20SARS-CoV-2%20infection&author=P%20Wang&author=R%20Luo&author=M%20Zhang&author=Y%20Wang&author=T%20Song&volume=11&publication_year=2020&pages=1-7&pmid=33293527&doi=10.1038/s41419-019-2182-0&)]
* 56.Tian M, Liu W, Li X, Zhao P, Shereen MA, Zhu C, et al. HIF-1α promotes SARS-CoV-2 infection and aggravates inflammatory responses to COVID-19. Signal Transduct Target Ther. 2021;6:1–3. doi: 10.1038/s41392-021-00726-w. [[DOI](https://doi.org/10.1038/s41392-021-00726-w)] [[PMC free article](/articles/PMC8371950/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34408131/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Signal%20Transduct%20Target%20Ther&title=HIF-1%CE%B1%20promotes%20SARS-CoV-2%20infection%20and%20aggravates%20inflammatory%20responses%20to%20COVID-19&author=M%20Tian&author=W%20Liu&author=X%20Li&author=P%20Zhao&author=MA%20Shereen&volume=6&publication_year=2021&pages=1-3&pmid=34408131&doi=10.1038/s41392-021-00726-w&)]
* 57.Irigoyen N, Firth AE, Jones JD, Chung BY, Siddell SG, Brierley I. High-resolution analysis of coronavirus gene expression by RNA sequencing and ribosome profiling. PLoS Pathog. 2016;12:e1005473. doi: 10.1371/journal.ppat.1005473. [[DOI](https://doi.org/10.1371/journal.ppat.1005473)] [[PMC free article](/articles/PMC4769073/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26919232/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&title=High-resolution%20analysis%20of%20coronavirus%20gene%20expression%20by%20RNA%20sequencing%20and%20ribosome%20profiling&author=N%20Irigoyen&author=AE%20Firth&author=JD%20Jones&author=BY%20Chung&author=SG%20Siddell&volume=12&publication_year=2016&pages=e1005473&pmid=26919232&doi=10.1371/journal.ppat.1005473&)]
* 58.Singh KK, Chaubey G, Chen JY, Suravajhala P. Decoding SARS-CoV-2 hijacking of host mitochondria in COVID-19 pathogenesis. Am J Physiol-Cell Physiol. 2020;319:C258–67. doi: 10.1152/ajpcell.00224.2020. [[DOI](https://doi.org/10.1152/ajpcell.00224.2020)] [[PMC free article](/articles/PMC7381712/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32510973/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol-Cell%20Physiol&title=Decoding%20SARS-CoV-2%20hijacking%20of%20host%20mitochondria%20in%20COVID-19%20pathogenesis&author=KK%20Singh&author=G%20Chaubey&author=JY%20Chen&author=P%20Suravajhala&volume=319&publication_year=2020&pages=C258-67&pmid=32510973&doi=10.1152/ajpcell.00224.2020&)]
* 59.Ajaz S, McPhail MJ, Singh KK, Mujib S, Trovato FM, Napoli S, et al. Mitochondrial metabolic manipulation by SARS-CoV-2 in peripheral blood mononuclear cells of patients with COVID-19. Am J Physiol-Cell Physiol. 2021;320:C57–65. doi: 10.1152/ajpcell.00426.2020. [[DOI](https://doi.org/10.1152/ajpcell.00426.2020)] [[PMC free article](/articles/PMC7816428/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33151090/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol-Cell%20Physiol&title=Mitochondrial%20metabolic%20manipulation%20by%20SARS-CoV-2%20in%20peripheral%20blood%20mononuclear%20cells%20of%20patients%20with%20COVID-19&author=S%20Ajaz&author=MJ%20McPhail&author=KK%20Singh&author=S%20Mujib&author=FM%20Trovato&volume=320&publication_year=2021&pages=C57-65&pmid=33151090&doi=10.1152/ajpcell.00426.2020&)]
* 60.Stukalov A, Girault V, Grass V, Karayel O, Bergant V, Urban C, et al. Multilevel proteomics reveals host perturbations by SARS-CoV-2 and SARS-CoV. Nature. 2021;594:246–52.. doi: 10.1038/s41586-021-03493-4. [[DOI](https://doi.org/10.1038/s41586-021-03493-4)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33845483/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Multilevel%20proteomics%20reveals%20host%20perturbations%20by%20SARS-CoV-2%20and%20SARS-CoV&author=A%20Stukalov&author=V%20Girault&author=V%20Grass&author=O%20Karayel&author=V%20Bergant&volume=594&publication_year=2021&pages=246-52.&pmid=33845483&doi=10.1038/s41586-021-03493-4&)]
* 61.Thaker SK, Ch’ng J, Christofk HR. Viral hijacking of cellular metabolism. BMC Biol. 2019;17:1–5. doi: 10.1186/s12915-019-0678-9. [[DOI](https://doi.org/10.1186/s12915-019-0678-9)] [[PMC free article](/articles/PMC6637495/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31319842/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Biol&title=Viral%20hijacking%20of%20cellular%20metabolism&author=SK%20Thaker&author=J%20Ch%E2%80%99ng&author=HR%20Christofk&volume=17&publication_year=2019&pages=1-5&pmid=31319842&doi=10.1186/s12915-019-0678-9&)]
* 62.Cecchini R, Cecchini AL. SARS-CoV-2 infection pathogenesis is related to oxidative stress as a response to aggression. Med Hypotheses. 2020;143:110102. doi: 10.1016/j.mehy.2020.110102. [[DOI](https://doi.org/10.1016/j.mehy.2020.110102)] [[PMC free article](/articles/PMC7357498/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32721799/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Hypotheses&title=SARS-CoV-2%20infection%20pathogenesis%20is%20related%20to%20oxidative%20stress%20as%20a%20response%20to%20aggression&author=R%20Cecchini&author=AL%20Cecchini&volume=143&publication_year=2020&pages=110102&pmid=32721799&doi=10.1016/j.mehy.2020.110102&)]
* 63.Suhail S, Zajac J, Fossum C, Lowater H, McCracken C, Severson N, et al. Role of oxidative stress on SARS-CoV (SARS) and SARS-CoV-2 (COVID-19) infection: a review. The protein journal. 2020;39:644–56. [[DOI](https://doi.org/10.1007/s10930-020-09935-8)] [[PMC free article](/articles/PMC7587547/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33106987/)]
* 64.Taylor EW, Radding W. Understanding selenium and glutathione as antiviral factors in COVID-19: Does the viral Mpro protease target host selenoproteins and glutathione synthesis? Front Nutr. 2020;7:143. doi: 10.3389/fnut.2020.00143. [[DOI](https://doi.org/10.3389/fnut.2020.00143)] [[PMC free article](/articles/PMC7492384/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32984400/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Nutr&title=Understanding%20selenium%20and%20glutathione%20as%20antiviral%20factors%20in%20COVID-19:%20Does%20the%20viral%20Mpro%20protease%20target%20host%20selenoproteins%20and%20glutathione%20synthesis?&author=EW%20Taylor&author=W%20Radding&volume=7&publication_year=2020&pages=143&pmid=32984400&doi=10.3389/fnut.2020.00143&)]
* 65.Wang Y, Huang J, Sun Y, Stubbs D, He J, Li W, et al. SARS-CoV-2 suppresses mRNA expression of selenoproteins associated with ferroptosis, endoplasmic reticulum stress and DNA synthesis. Food Chem Toxicol. 2021;153:112286. doi: 10.1016/j.fct.2021.112286. [[DOI](https://doi.org/10.1016/j.fct.2021.112286)] [[PMC free article](/articles/PMC8139185/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34023458/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Food%20Chem%20Toxicol&title=SARS-CoV-2%20suppresses%20mRNA%20expression%20of%20selenoproteins%20associated%20with%20ferroptosis,%20endoplasmic%20reticulum%20stress%20and%20DNA%20synthesis&author=Y%20Wang&author=J%20Huang&author=Y%20Sun&author=D%20Stubbs&author=J%20He&volume=153&publication_year=2021&pages=112286&pmid=34023458&doi=10.1016/j.fct.2021.112286&)]
* 66.Abais JM, Xia M, Zhang Y, Boini KM, Li PL. Redox regulation of NLRP3 inflammasomes: ROS as trigger or effector? Antioxid Redox Signal. 2015;22:1111–29. doi: 10.1089/ars.2014.5994. [[DOI](https://doi.org/10.1089/ars.2014.5994)] [[PMC free article](/articles/PMC4403231/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25330206/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antioxid%20Redox%20Signal&title=Redox%20regulation%20of%20NLRP3%20inflammasomes:%20ROS%20as%20trigger%20or%20effector?&author=JM%20Abais&author=M%20Xia&author=Y%20Zhang&author=KM%20Boini&author=PL%20Li&volume=22&publication_year=2015&pages=1111-29&pmid=25330206&doi=10.1089/ars.2014.5994&)]
* 67.Olagnier D, Farahani E, Thyrsted J, Blay-Cadanet J, Herengt A, Idorn M, et al. SARS-CoV2-mediated suppression of NRF2-signaling reveals potent antiviral and anti-inflammatory activity of 4-octyl-itaconate and dimethyl fumarate. Nat Commun. 2020;11:1–2. doi: 10.1038/s41467-019-13993-7. [[DOI](https://doi.org/10.1038/s41467-019-13993-7)] [[PMC free article](/articles/PMC7532469/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33009401/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=SARS-CoV2-mediated%20suppression%20of%20NRF2-signaling%20reveals%20potent%20antiviral%20and%20anti-inflammatory%20activity%20of%204-octyl-itaconate%20and%20dimethyl%20fumarate&author=D%20Olagnier&author=E%20Farahani&author=J%20Thyrsted&author=J%20Blay-Cadanet&author=A%20Herengt&volume=11&publication_year=2020&pages=1-2&pmid=33009401&doi=10.1038/s41467-019-13993-7&)]
* 68.Heer CD, Sanderson DJ, Voth LS, Alhammad YM, Schmidt MS, Trammell SA, et al. Coronavirus infection and PARP expression dysregulate the NAD metabolome: An actionable component of innate immunity. J Biol Chem. 2020;295:17986–96.. doi: 10.1074/jbc.RA120.015138. [[DOI](https://doi.org/10.1074/jbc.RA120.015138)] [[PMC free article](/articles/PMC7834058/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33051211/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Coronavirus%20infection%20and%20PARP%20expression%20dysregulate%20the%20NAD%20metabolome:%20An%20actionable%20component%20of%20innate%20immunity&author=CD%20Heer&author=DJ%20Sanderson&author=LS%20Voth&author=YM%20Alhammad&author=MS%20Schmidt&volume=295&publication_year=2020&pages=17986-96.&pmid=33051211&doi=10.1074/jbc.RA120.015138&)]
* 69.Agledal L, Niere M, Ziegler M. The phosphate makes a difference: cellular functions of NADP. Redox Rep. 2010;15:2–10. doi: 10.1179/174329210X12650506623122. [[DOI](https://doi.org/10.1179/174329210X12650506623122)] [[PMC free article](/articles/PMC7067316/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20196923/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Redox%20Rep&title=The%20phosphate%20makes%20a%20difference:%20cellular%20functions%20of%20NADP&author=L%20Agledal&author=M%20Niere&author=M%20Ziegler&volume=15&publication_year=2010&pages=2-10&pmid=20196923&doi=10.1179/174329210X12650506623122&)]
* 70.Schwiebert EM, Zsembery A. Extracellular ATP as a signaling molecule for epithelial cells. Biochimica et Biophysica Acta (BBA)-Biomembranes. 2003;1615:7–32. doi: 10.1016/S0005-2736(03)00210-4. [[DOI](https://doi.org/10.1016/S0005-2736%2803%2900210-4)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12948585/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta%20(BBA)-Biomembranes&title=Extracellular%20ATP%20as%20a%20signaling%20molecule%20for%20epithelial%20cells&author=EM%20Schwiebert&author=A%20Zsembery&volume=1615&publication_year=2003&pages=7-32&pmid=12948585&doi=10.1016/S0005-2736(03)00210-4&)]
* 71.Novak I. ATP as a signaling molecule: the exocrine focus. Physiology. 2003;18:12–7. doi: 10.1152/nips.01409.2002. [[DOI](https://doi.org/10.1152/nips.01409.2002)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/12531926/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiology.&title=ATP%20as%20a%20signaling%20molecule:%20the%20exocrine%20focus&author=I%20Novak&volume=18&publication_year=2003&pages=12-7&pmid=12531926&doi=10.1152/nips.01409.2002&)]
* 72.Green SJ. Covid-19 accelerates endothelial dysfunction and nitric oxide deficiency. Microbes Infect. 2020;22:149. doi: 10.1016/j.micinf.2020.05.006. [[DOI](https://doi.org/10.1016/j.micinf.2020.05.006)] [[PMC free article](/articles/PMC7229726/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32425647/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbes%20Infect&title=Covid-19%20accelerates%20endothelial%20dysfunction%20and%20nitric%20oxide%20deficiency&author=SJ%20Green&volume=22&publication_year=2020&pages=149&pmid=32425647&doi=10.1016/j.micinf.2020.05.006&)]
* 73.Yamasaki H. Blood nitrate and nitrite modulating nitric oxide bioavailability: potential therapeutic functions in COVID-19. Nitric Oxide. 2020;103:29–30. doi: 10.1016/j.niox.2020.07.005. [[DOI](https://doi.org/10.1016/j.niox.2020.07.005)] [[PMC free article](/articles/PMC7377740/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32712272/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nitric%20Oxide&title=Blood%20nitrate%20and%20nitrite%20modulating%20nitric%20oxide%20bioavailability:%20potential%20therapeutic%20functions%20in%20COVID-19&author=H%20Yamasaki&volume=103&publication_year=2020&pages=29-30&pmid=32712272&doi=10.1016/j.niox.2020.07.005&)]
* 74.Lingappan K. NF-κB in oxidative stress. Curr Opin Toxicol. 2018;7:81–6. doi: 10.1016/j.cotox.2017.11.002. [[DOI](https://doi.org/10.1016/j.cotox.2017.11.002)] [[PMC free article](/articles/PMC5978768/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29862377/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Toxicol&title=NF-%CE%BAB%20in%20oxidative%20stress&author=K%20Lingappan&volume=7&publication_year=2018&pages=81-6&pmid=29862377&doi=10.1016/j.cotox.2017.11.002&)]
* 75.Ozdemir B, Yazici A. Could the decrease in the endothelial nitric oxide (NO) production and NO bioavailability be the crucial cause of COVID-19 related deaths? Med Hypotheses. 2020;144:109970. doi: 10.1016/j.mehy.2020.109970. [[DOI](https://doi.org/10.1016/j.mehy.2020.109970)] [[PMC free article](/articles/PMC7276137/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32534341/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Hypotheses&title=Could%20the%20decrease%20in%20the%20endothelial%20nitric%20oxide%20(NO)%20production%20and%20NO%20bioavailability%20be%20the%20crucial%20cause%20of%20COVID-19%20related%20deaths?&author=B%20Ozdemir&author=A%20Yazici&volume=144&publication_year=2020&pages=109970&pmid=32534341&doi=10.1016/j.mehy.2020.109970&)]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

[Supplemental Table 1](/articles/instance/9349314/bin/41431_2022_1108_MOESM1_ESM.pdf) (100.4KB, pdf)