Eur J Hum Genet

. 2023 Nov 8;32(1):10–20. doi: [10.1038/s41431-023-01462-1](https://doi.org/10.1038/s41431-023-01462-1)

# A comprehensive SARS-CoV-2 and COVID-19 review, Part 2: host extracellular to systemic effects of SARS-CoV-2 infection

[S Anand Narayanan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Narayanan%20SA%22%5BAuthor%5D)

### S Anand Narayanan

1COVID-19 International Research Team, Medford, MA 02155 USA

2Department of Health, Nutrition and Food Sciences, Florida State University, Tallahassee, FL 32301 USA

Find articles by [S Anand Narayanan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Narayanan%20SA%22%5BAuthor%5D)

1,2,✉,#, [David A Jamison Jr](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jamison%20DA%22%5BAuthor%5D)

### David A Jamison Jr

1COVID-19 International Research Team, Medford, MA 02155 USA

Find articles by [David A Jamison Jr](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jamison%20DA%22%5BAuthor%5D)

1,#, [Joseph W Guarnieri](https://pubmed.ncbi.nlm.nih.gov/?term=%22Guarnieri%20JW%22%5BAuthor%5D)

### Joseph W Guarnieri

1COVID-19 International Research Team, Medford, MA 02155 USA

3Center for Mitochondrial and Epigenomic Medicine, Division of Human Genetics, The Children’s Hospital of Philadelphia, Philadelphia, PA 19104 USA

Find articles by [Joseph W Guarnieri](https://pubmed.ncbi.nlm.nih.gov/?term=%22Guarnieri%20JW%22%5BAuthor%5D)

1,3, [Victoria Zaksas](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zaksas%20V%22%5BAuthor%5D)

### Victoria Zaksas

1COVID-19 International Research Team, Medford, MA 02155 USA

4Center for Translational Data Science, University of Chicago, Chicago, IL 60637 USA

5Clever Research Lab, Springfield, IL 62704 USA

Find articles by [Victoria Zaksas](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zaksas%20V%22%5BAuthor%5D)

1,4,5, [Michael Topper](https://pubmed.ncbi.nlm.nih.gov/?term=%22Topper%20M%22%5BAuthor%5D)

### Michael Topper

1COVID-19 International Research Team, Medford, MA 02155 USA

6Departments of Oncology and Medicine and the Sidney Comprehensive Cancer Center, The Johns Hopkins Medical Institutions, Baltimore, MD USA

Find articles by [Michael Topper](https://pubmed.ncbi.nlm.nih.gov/?term=%22Topper%20M%22%5BAuthor%5D)

1,6, [Andrew P Koutnik](https://pubmed.ncbi.nlm.nih.gov/?term=%22Koutnik%20AP%22%5BAuthor%5D)

### Andrew P Koutnik

7Human Healthspan, Resilience, and Performance, Florida Institute for Human and Machine Cognition, Pensacola, FL 32502 USA

8Sansum Diabetes Research Institute, Santa Barbara, CA 93015 USA

Find articles by [Andrew P Koutnik](https://pubmed.ncbi.nlm.nih.gov/?term=%22Koutnik%20AP%22%5BAuthor%5D)

7,8, [Jiwoon Park](https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20J%22%5BAuthor%5D)

### Jiwoon Park

9Department of Physiology, Biophysics and Systems Biology, Weill Cornell Medicine, New York, NY USA

10Laboratory of Virology and Infectious Disease, The Rockefeller University, New York, NY 10065 USA

Find articles by [Jiwoon Park](https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20J%22%5BAuthor%5D)

9,10, [Kevin B Clark](https://pubmed.ncbi.nlm.nih.gov/?term=%22Clark%20KB%22%5BAuthor%5D)

### Kevin B Clark

1COVID-19 International Research Team, Medford, MA 02155 USA

11Cures Within Reach, Chicago, IL 60602 USA

12Campus and Domain Champions Program, Multi-Tier Assistance, Training, and Computational Help (MATCH) Track, National Science Foundation’s Advanced Cyberinfrastructure Coordination Ecosystem: Services and Support (ACCESS), Philadelphia, PA USA

13Expert Network, Penn Center for Innovation, University of Pennsylvania, Philadelphia, PA 19104 USA

14Biometrics and Nanotechnology Councils, Institute for Electrical and Electronics Engineers, New York, NY 10016 USA

15Peace Innovation Institute, The Hague 2511, Netherlands and Stanford University, Palo Alto, 94305 CA USA

Find articles by [Kevin B Clark](https://pubmed.ncbi.nlm.nih.gov/?term=%22Clark%20KB%22%5BAuthor%5D)

1,11,12,13,14,15, [Francisco J Enguita](https://pubmed.ncbi.nlm.nih.gov/?term=%22Enguita%20FJ%22%5BAuthor%5D)

### Francisco J Enguita

1COVID-19 International Research Team, Medford, MA 02155 USA

16Instituto de Medicina Molecular João Lobo Antunes, Faculdade de Medicina, Universidade de Lisboa, 1649-028 Lisboa, Portugal

Find articles by [Francisco J Enguita](https://pubmed.ncbi.nlm.nih.gov/?term=%22Enguita%20FJ%22%5BAuthor%5D)

1,16, [Ana Lúcia Leitão](https://pubmed.ncbi.nlm.nih.gov/?term=%22Leit%C3%A3o%20AL%22%5BAuthor%5D)

### Ana Lúcia Leitão

17MEtRICs, Department of Chemistry, NOVA School of Science and Technology, Universidade NOVA de Lisboa, 2829-516 Caparica, Portugal

Find articles by [Ana Lúcia Leitão](https://pubmed.ncbi.nlm.nih.gov/?term=%22Leit%C3%A3o%20AL%22%5BAuthor%5D)

17, [Saswati Das](https://pubmed.ncbi.nlm.nih.gov/?term=%22Das%20S%22%5BAuthor%5D)

### Saswati Das

1COVID-19 International Research Team, Medford, MA 02155 USA

18Atal Bihari Vajpayee Institute of Medical Sciences and Dr Ram Mannohar Lohia Hospital, New Delhi, 110001 India

Find articles by [Saswati Das](https://pubmed.ncbi.nlm.nih.gov/?term=%22Das%20S%22%5BAuthor%5D)

1,18, [Pedro M Moraes-Vieira](https://pubmed.ncbi.nlm.nih.gov/?term=%22Moraes-Vieira%20PM%22%5BAuthor%5D)

### Pedro M Moraes-Vieira

1COVID-19 International Research Team, Medford, MA 02155 USA

19Department of Genetics, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, Brazil

20Experimental Medicine Research Cluster (EMRC) and Obesity and Comorbidities Research Center (OCRC), University of Campinas, Campinas, Brazil

Find articles by [Pedro M Moraes-Vieira](https://pubmed.ncbi.nlm.nih.gov/?term=%22Moraes-Vieira%20PM%22%5BAuthor%5D)

1,19,20, [Diego Galeano](https://pubmed.ncbi.nlm.nih.gov/?term=%22Galeano%20D%22%5BAuthor%5D)

### Diego Galeano

1COVID-19 International Research Team, Medford, MA 02155 USA

21Facultad de Ingeniería, Universidad Nacional de Asunción, San Lorenzo, Paraguay

Find articles by [Diego Galeano](https://pubmed.ncbi.nlm.nih.gov/?term=%22Galeano%20D%22%5BAuthor%5D)

1,21, [Christopher E Mason](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D)

### Christopher E Mason

1COVID-19 International Research Team, Medford, MA 02155 USA

9Department of Physiology, Biophysics and Systems Biology, Weill Cornell Medicine, New York, NY USA

22New York Genome Center, New York, NY USA

23The Feil Family Brain and Mind Research Institute, Weill Cornell Medicine, New York, NY USA

Find articles by [Christopher E Mason](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D)

1,9,22,23, [Nídia S Trovão](https://pubmed.ncbi.nlm.nih.gov/?term=%22Trov%C3%A3o%20NS%22%5BAuthor%5D)

### Nídia S Trovão

1COVID-19 International Research Team, Medford, MA 02155 USA

24Fogarty International Center, National Institutes of Health, Bethesda, MD 20892 USA

Find articles by [Nídia S Trovão](https://pubmed.ncbi.nlm.nih.gov/?term=%22Trov%C3%A3o%20NS%22%5BAuthor%5D)

1,24, [Robert E Schwartz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Schwartz%20RE%22%5BAuthor%5D)

### Robert E Schwartz

1COVID-19 International Research Team, Medford, MA 02155 USA

25Division of Gastroenterology and Hepatology, Department of Medicine, Weill Cornell Medicine, New York, NY USA

26Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY USA

Find articles by [Robert E Schwartz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Schwartz%20RE%22%5BAuthor%5D)

1,25,26, [Jonathan C Schisler](https://pubmed.ncbi.nlm.nih.gov/?term=%22Schisler%20JC%22%5BAuthor%5D)

### Jonathan C Schisler

1COVID-19 International Research Team, Medford, MA 02155 USA

27McAllister Heart Institute and Department of Pharmacology, The University of North Carolina at Chapel Hill, Chapel Hill, NC USA

Find articles by [Jonathan C Schisler](https://pubmed.ncbi.nlm.nih.gov/?term=%22Schisler%20JC%22%5BAuthor%5D)

1,27, [Jordana G A Coelho-dos-Reis](https://pubmed.ncbi.nlm.nih.gov/?term=%22Coelho-dos-Reis%20JGA%22%5BAuthor%5D)

### Jordana G A Coelho-dos-Reis

1COVID-19 International Research Team, Medford, MA 02155 USA

28Basic and Applied Virology Lab, Department of Microbiology, Institute for Biological Sciences (ICB), Federal University of Minas Gerais, Belo Horizonte, MG Brazil

Find articles by [Jordana G A Coelho-dos-Reis](https://pubmed.ncbi.nlm.nih.gov/?term=%22Coelho-dos-Reis%20JGA%22%5BAuthor%5D)

1,28, [Eve Syrkin Wurtele](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wurtele%20ES%22%5BAuthor%5D)

### Eve Syrkin Wurtele

1COVID-19 International Research Team, Medford, MA 02155 USA

29Genetics Program, Department of Genetics, Development and Cell Biology, Iowa State University, Ames, IA 90011 USA

30Bioinformatics and Computational Biology Program, Center for Metabolomics, Department of Genetics, Development and Cell Biology, Iowa State University, Ames, IA 90011 USA

Find articles by [Eve Syrkin Wurtele](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wurtele%20ES%22%5BAuthor%5D)

1,29,30, [Afshin Beheshti](https://pubmed.ncbi.nlm.nih.gov/?term=%22Beheshti%20A%22%5BAuthor%5D)

### Afshin Beheshti

1COVID-19 International Research Team, Medford, MA 02155 USA

31Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA 02142 USA

32Blue Marble Space Institute of Science, Space Biosciences Division, NASA Ames Research Center, Moffett Field, Santa Clara, CA 94035 USA

Find articles by [Afshin Beheshti](https://pubmed.ncbi.nlm.nih.gov/?term=%22Beheshti%20A%22%5BAuthor%5D)

1,31,32,✉

* Author information
* Article notes
* Copyright and License information

1COVID-19 International Research Team, Medford, MA 02155 USA

2Department of Health, Nutrition and Food Sciences, Florida State University, Tallahassee, FL 32301 USA

3Center for Mitochondrial and Epigenomic Medicine, Division of Human Genetics, The Children’s Hospital of Philadelphia, Philadelphia, PA 19104 USA

4Center for Translational Data Science, University of Chicago, Chicago, IL 60637 USA

5Clever Research Lab, Springfield, IL 62704 USA

6Departments of Oncology and Medicine and the Sidney Comprehensive Cancer Center, The Johns Hopkins Medical Institutions, Baltimore, MD USA

7Human Healthspan, Resilience, and Performance, Florida Institute for Human and Machine Cognition, Pensacola, FL 32502 USA

8Sansum Diabetes Research Institute, Santa Barbara, CA 93015 USA

9Department of Physiology, Biophysics and Systems Biology, Weill Cornell Medicine, New York, NY USA

10Laboratory of Virology and Infectious Disease, The Rockefeller University, New York, NY 10065 USA

11Cures Within Reach, Chicago, IL 60602 USA

12Campus and Domain Champions Program, Multi-Tier Assistance, Training, and Computational Help (MATCH) Track, National Science Foundation’s Advanced Cyberinfrastructure Coordination Ecosystem: Services and Support (ACCESS), Philadelphia, PA USA

13Expert Network, Penn Center for Innovation, University of Pennsylvania, Philadelphia, PA 19104 USA

14Biometrics and Nanotechnology Councils, Institute for Electrical and Electronics Engineers, New York, NY 10016 USA

15Peace Innovation Institute, The Hague 2511, Netherlands and Stanford University, Palo Alto, 94305 CA USA

16Instituto de Medicina Molecular João Lobo Antunes, Faculdade de Medicina, Universidade de Lisboa, 1649-028 Lisboa, Portugal

17MEtRICs, Department of Chemistry, NOVA School of Science and Technology, Universidade NOVA de Lisboa, 2829-516 Caparica, Portugal

18Atal Bihari Vajpayee Institute of Medical Sciences and Dr Ram Mannohar Lohia Hospital, New Delhi, 110001 India

19Department of Genetics, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, Brazil

20Experimental Medicine Research Cluster (EMRC) and Obesity and Comorbidities Research Center (OCRC), University of Campinas, Campinas, Brazil

21Facultad de Ingeniería, Universidad Nacional de Asunción, San Lorenzo, Paraguay

22New York Genome Center, New York, NY USA

23The Feil Family Brain and Mind Research Institute, Weill Cornell Medicine, New York, NY USA

24Fogarty International Center, National Institutes of Health, Bethesda, MD 20892 USA

25Division of Gastroenterology and Hepatology, Department of Medicine, Weill Cornell Medicine, New York, NY USA

26Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY USA

27McAllister Heart Institute and Department of Pharmacology, The University of North Carolina at Chapel Hill, Chapel Hill, NC USA

28Basic and Applied Virology Lab, Department of Microbiology, Institute for Biological Sciences (ICB), Federal University of Minas Gerais, Belo Horizonte, MG Brazil

29Genetics Program, Department of Genetics, Development and Cell Biology, Iowa State University, Ames, IA 90011 USA

30Bioinformatics and Computational Biology Program, Center for Metabolomics, Department of Genetics, Development and Cell Biology, Iowa State University, Ames, IA 90011 USA

31Stanley Center for Psychiatric Research, Broad Institute of MIT and Harvard, Cambridge, MA 02142 USA

32Blue Marble Space Institute of Science, Space Biosciences Division, NASA Ames Research Center, Moffett Field, Santa Clara, CA 94035 USA

✉

Corresponding author.

#

Contributed equally.

Received 2023 Jan 16; Revised 2023 Sep 1; Accepted 2023 Sep 13; Issue date 2024 Jan.

© The Author(s) 2023

**Open Access** This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit [http://creativecommons.org/licenses/by/4.0/](https://creativecommons.org/licenses/by/4.0/).

[PMC Copyright notice](/about/copyright/)

PMCID: PMC10772081  PMID: [37938797](https://pubmed.ncbi.nlm.nih.gov/37938797/)

## Abstract

COVID-19, the disease caused by SARS-CoV-2, has caused significant morbidity and mortality worldwide. The betacoronavirus continues to evolve with global health implications as we race to learn more to curb its transmission, evolution, and sequelae. The focus of this review, the second of a three-part series, is on the biological effects of the SARS-CoV-2 virus on post-acute disease in the context of tissue and organ adaptations and damage. We highlight the current knowledge and describe how virological, animal, and clinical studies have shed light on the mechanisms driving the varied clinical diagnoses and observations of COVID-19 patients. Moreover, we describe how investigations into SARS-CoV-2 effects have informed the understanding of viral pathogenesis and provide innovative pathways for future research on the mechanisms of viral diseases.

**Subject terms:** Viral infection, Diagnostic markers, Pathology, Chronic inflammation

## Introduction

As of January 13, 2023, over 671 million cases of the coronavirus disease 2019 (COVID-19) worldwide have been reported, and more than 6.71 million lives have been claimed globally. The COVID-19 pandemic is caused by infection with the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) virus. The development and administration of vaccines and antiviral therapeutics against SARS-CoV-2 significantly reduced the local to global impact and burden of COVID-19. However, the global distribution of these resources has been unequal [[1](#CR1)], ultimately prolonging viral transmission. Moreover, SARS-CoV-2 quickly evolved into variants with increased transmission rates more capable of immune escape [[2](#CR2)]. Therefore, we must increase our understanding of this novel virus, its variants, and the resulting short- and long-term effects of COVID-19 on human health.

SARS-CoV-2 belongs to the genus Betacoronavirus, which includes viruses such as OC43, HKU1, SARS-CoV, and MERS-CoV [[3](#CR3)]. SARS-CoV-2 is an enveloped positive-sense RNA virus, whose genome is packaged in a helical ribonucleocapsid complex. The outer surface of the virus is studded with Spike (S) proteins that facilitate the infection of host cells by binding to its cognate cellular surface receptor angiotensin-converting enzyme 2 (ACE2) through the S protein receptor binding domain (RBD) to allow for viral membrane fusion. Fusion leads to viral genome transfer into the host cell cytoplasm where viral replication can commence, leading to its various intracellular impacts. To read in more detail these processes, as well as structural elements of the SARS-CoV-2 virus, see our first review, ref. [[3](#CR3)], focusing on intracellular SARS-CoV-2 viral effects [[3](#CR3)].

In this second of a three-part comprehensive series of our series of SARS-CoV-2 reviews, we cover the extracellular and circulating signals and physiological effects of the viral disease, including the consequences and remodeling of organs, metabolism, humoral factors, and the immune system. In turn, the aim of this review series is to provide an overview of our current knowledge of SARS-CoV-2 and its biological effects from basic biology to clinical patient outcomes, highlighting also future areas of research given the ever-evolving nature of SARS-CoV-2,COVID-19, and its consequences for healthcare across the world.

## Metabolic adaptations caused directly by viral infection of cells

We previously described [[3](#CR3)] SARS-CoV-2 viral infection intracellular effects and host adaptations. These include major shifts in metabolic and biochemical pathways, including the oxidative phosphorylation and the tricarboxylic acid (TCA) cycle [[3](#CR3)]. The immune system is also involved in these adaptations, with a study showing upregulation of Type I interferon (IFN) gene signaling pathways leading to changes with cellular responses to IFN and triggering changes with associated cytokine-mediated signaling pathways [[4](#CR4)]. These stimuli lead to cellular adaptations including endoplasmic reticulum stress, upregulated unfolded protein response and downregulation of pathways including spliceosome-mediated RNA processing, mitochondrial electron transport chain inhibition, and impairment in the biogenesis and assembly of the electron transport chain complexes and oxidative phosphorylation [[5](#CR5), [6](#CR6)]. Indeed, SARS-CoV-2 infection leads to decreased mitochondrial function, increased glycolysis, and shunting towards the pentose 5 phosphate pathway, resulting in changes in cellular metabolism and bioenergetics that lead to pathophysiological outcomes.

These various cellular biochemical adaptations also result in production of circulating metabolites in COVID-19 patients that lead to systemic effects, including tissue remodeling and function, shifts in whole-body metabolism and bioenergetics, changes with immune and endocrine state, etc., resulting in the symptoms and disease outcomes seen with COVID-19 patients. For example, the TCA cycle is the main cellular source of energy and critical for aerobic respiration; serum levels of the TCA metabolites (citrate, fumarate, malate, and aconitate) were significantly lower in the SARS-CoV-2-infected patients as compared with controls, with choline specifically observed to be decreased in patients with severe COVID-19 [[7](#CR7)]. Serum metabolites that were increased in these patients included deoxy-thymidine, deoxyuridine, adenine, cystine, and homocysteine. These data provide context as to how specific biochemical and physiological pathways, e.g., the TCA cycle, shift as a result of COVID-19 infection and lead to pathogenesis. For example, elevated homocysteine is associated with endothelial damage [[8](#CR8)]. These shifts in metabolites reflect a systemic shift during acute COVID-19 infection, and warrant study of subsequent specific organ adaptations (e.g., bone as a major source of citrate, which is released into the plasma during bone resorption, with soft tissues not generally providing a significant source of citrate into blood plasma).

Separate from metabolic pathways (e.g., oxidative phosphorylation, glycolysis, etc.) that produce energy from sugars are alternative pathways that utilize compounds such as lipids. Excess energy is stored in the form of lipids, with lipids also used as an immediate energy source or for later use. Lipids also serve as second messengers, activating protein and immune pathways. Indeed, it has been reported SARS-CoV-2 infection causes lipids levels to change and lipid metabolism to alter, with one study using untargeted metabolomic and lipidomic approaches observing specific circulating lipids associated with SARS-CoV-2 infection, including increases in triglycerides and free fatty acids (e.g., arachidonic acid and oleic acid). These lipids also correlated with disease severity [[9](#CR9)]. A specific class of lipids are lipid mediators (LMs), which are bioactive lipids produced locally in response to extracellular stimuli and are involved in immune regulation. Studies have investigated their role in relation to COVID-19 severity as well. One report involving 19 healthy patients, 18 COVID-19 patients who did not need ICU admission (mild), and 20 patients that needed ICU admission (severe) showed an increase of free poly-unsaturated fatty acids (PUFAs) and diminished amounts of PUFA-containing plasmalogens in patients with COVID-19 that also correlated with disease severity [[10](#CR10)]. Another study performing metabolomics and lipidomics measurements on patients (49 subjects, 33 were COVID-19–positive subjects and 16 COVID-19–negative subjects) showed alterations in fatty acid metabolism that were associated with SARS-CoV-2 infection. More specifically, short- and medium-chain acylcarnitines were significantly diminished in all patients with COVID-19, while all Non-Esterified Fatty Acids (except nonanoic acid) were increased in all COVID-19 patients [[11](#CR11), [12](#CR12)].

In summary, SARS-CoV-2 infection affects whole-body metabolism in different ways, with emerging evidence supporting a correlation between these adaptations and COVID-19 disease severity. Moreover, the balance of biochemical components, whether the machinery (e.g., TCA proteins), fuel sources (e.g., lipids), or messenger molecules (e.g., metabolites) appear to shift towards a physiological catabolic state that may be further exacerbated by a patient’s underlying conditions and demographics [[7](#CR7), [9](#CR9)–[12](#CR12)]. Metabolic biofactors are also intrinsically tied with the recovery and response to viral infection, with immune-regulating lipids, pro-inflammatory lipids, and lipid mediators plausibly modulating the immune response during COVID-19, as one example. The immune system’s physiology is also reliant on its metabolic function, whether from a development, proliferation, activation, or memory perspective. Immunity is shaped and guided by humoral factors, which we address in the next section, as to the SARS-CoV-2 viral influence on these factors and the effects on the body.

## Humoral adaptations and changes

Humoral factors are compounds that are transported by the circulatory system throughout the body, and include immune factors (e.g., cytokines, chemokines, antibodies, complement proteins, clotting factors, etc.), hormones, RNAs, lipids, metabolites, etc. These play a role in steady-state physiology as well as pathophysiological adaptations, with changes in humoral factors occurring from SARS-CoV-2 infection. One example is bradykinin, a peptide that is involved with physiological effects, including dilation of arterioles through release of prostacyclin, nitric oxide, and EDHF, constriction of veins by prostaglandin F2, as well as various biochemical effects including upregulation of intracellular Ca2+, regulation of cAMP/cGMP, arachidonic acid release, promotion of VEGF expression and angiogenesis, and activation of TGF-B, JAK/STAT pathways [[13](#CR13)]. Additionally, bradykinin is degraded by angiotensin-converting enzyme (ACE) and enhanced by angiotensin produced by ACE2. Bradykinin, angiotensin, ACE, and ACE2, are all part of the renin-angiotensin system (RAS), which serves a role within the endocrine system in regulation of renal, cardiac, and vascular physiology, and in turn, systemic physiology [[14](#CR14)]. Moreover, SARS-CoV-2 enters the cell through receptors such as ACE2 and NRP-1, in a process requiring the transmembrane protease 2 (TMPRSS2) protein [[3](#CR3), [15](#CR15)]; thus, SARS-CoV-2 directly interacts with the RAS system and influences its activity. Indeed, gene analysis of bronchoalveolar lavage fluid from COVID-19 patients showed a critical imbalance of RAS-related factors, including decreased expression of ACE, while increases in ACE2, renin, angiotensin, and bradykinin, which were suggestive of elevated bradykinin levels systemically that lead to the cardiovascular symptoms seen with patients [[16](#CR16)]. Another study of the bradykinin cascade of COVID-19 patients correlated changes in this pathway with disease severity, showing accumulated bradykinin levels were related with patient disease severity, as well as with levels of inflammation, coagulation, and lymphopenia [[17](#CR17)].

In addition to the RAS axis, the endocrine system is also vulnerable to SARS-CoV-2 infection [[14](#CR14), [15](#CR15)]. For example, the ACE2 receptor is expressed in endocrine glands, including the hypothalamus, pituitary gland, adrenals, pancreas, thyroid glands, ovaries, and testes [[15](#CR15)]. Studies have shown reduced thyroid (e.g., subacute thyroiditis, thyrotoxicosis), adrenal (e.g., hypocortisolism, hyponatremia), hypothalamic-pituitary-thyroid (e.g., HPA dysfunction, hypocortisolism, hypothyroidism), and pancreatic islet function (e.g., hyperglycemia, pancreatic injury based from elevated amylase and lipase), as well as damage to the testes and changes with menstrual cycle being described in COVID-19 patients [[14](#CR14), [15](#CR15)]. SARS-CoV-2 infection has also been suggested to affect glycemic control, with an international registry having been organized to investigate the interaction between diabetes and COVID-19 [[14](#CR14)]. Moreover, clinical studies show more severe outcomes in patients with diabetes, obesity, and hypertension; however, there is a lack of data in humans on ACE2 expression in pathological conditions with endocrine tissues, an area which needs future investigation [[15](#CR15)]. Various mouse studies have enabled us to investigate organ-specific adaptations, including examining the effects of ACE2 deletion in endocrine organs and observing the beneficial effects of ACE2 gene therapy [[15](#CR15)]. The details of these studies are outside the scope of this review, but are elaborated in ref. [[14](#CR14), [15](#CR15)]. As with bradykinin, the remodeling of endocrine factors results in changes with the factors that endocrine organs produce and secrete, though this is an area warrants more investigation. Indeed, these include the non-coding transcriptome, genes which are not involved with protein synthesis, but are transcribed and often secreted into the circulation, in turn providing information on the genetic state of their originating cells, but also controlling and regulating biological adaptations due to the ability of the non-coding RNAs to interact with other macromolecules, including DNA, RNA, and proteins. We elaborate on the changes with the non-coding RNA transcriptome seen in COVID-19 patients in the next section.

## Exosomes influence on SARS-CoV-2 infection

Exosomes are extracellular vesicles generated by all cells that carry cellular components, including nucleic acids, proteins, lipids, and metabolites. They are carried and transported in blood and lymph, and serve near- and long-distance intercellular communication and biological effects. They also serve as a biomarker based on their contents, which may be altered in different conditions and physiological states. It is known that during viral infection, extracellular vesicles become vectors of viral material. More recent reports have also highlighted this occurs with SARS-CoV-2 infection [[18](#CR18)].

One study investigated and compared exosomes from mild or severe COVID-19 patients. The exosomes from both groups contained SARS-CoV-2 spike-derived peptides and immunomodulatory molecules, though only those from mild patients could stimulate an antigen-specific CD4 T-cell response [[19](#CR19)]. This study also observed that the proteome of exosomes of mild patients correlated with a normally functioning immune system, while that of severe patients was associated with increased and chronic inflammation.

As mentioned above, exosomes may affect distant cells, and one study explored this connection in the context of COVID-19 by evaluating exosome effects on endothelial cells from exosomes derived from plasma [[19](#CR19)]. Exosomes affected the endothelial cells by inducing inflammation-related pathways (e.g., NRLP3, caspase-1, and interleukin-1-beta) through mRNA expression, compared with plasma from mild disease or healthy donors. The study showed that these adaptations were tied with exosomes containing tenascin-C and fibrinogen-beta, which triggered the inflammatory pathways by inducing NF-kB mediated pathways. While this study focused on endothelial cell responses, it was noted that these effects may promote inflammation in other cell types systemically.

Exosomes can also be involved with the inflammatory response. A subset of exosomes, called defensosomes, mediate host defense by binding and inhibiting pathogenic responses. A study showed that exosomes in SARS-CoV-2-infected patients can also be involved in this capacity by interfering with surface protein interactions [[20](#CR20)]. Exosomes containing high levels of the viral receptor, ACE2, were induced from SARS-CoV-2 infection and activation of viral sensors in cell culture utilizing the autophagy protein, ATG16L1, and were shown to be able to bind and block viral entry. These findings support that exosomes may also be involved with the SARS-CoV-2 antiviral response, with more research of this area increasing our understanding of exosomes and their role in host defense mechanisms.

Finally, exosomes may provide insight into organ adaptations based on exosomes from the source organ, in general as well as in specific circumstances such as from viral infection. The exosomes produced from specific cells will communicate their cellular phenotypic state, with each exosome sharing the identity of its source cell. In turn, organ-specific adaptations from SARS-CoV-2 infection may be able to be delineated through measurement of exosomes. One study explored this by investigating how SARS-CoV-2 is known to invade neural cell mitochondria, with SARS-CoV-2 proteins and mitochondrial proteins contained within neuron- and astrocyte-derived exosomes quantified in post COVID-19 infection. Indeed, these exosomes contained SARS-CoV-2 N and S1 proteins and varied depending on the context of the disease severity and timeline of COVID-19 [[21](#CR21)]. This study is one example of the utility exosomes may provide for complex, multi-organ conditions like COVID-19, in studying and determining physiological alterations resulting from the disease condition.

## Roles of the non-coding transcriptome in SARS-CoV-2 infection

RNA viruses such as SARS-CoV-2 are flexible in their ability to circumvent cellular defenses through a variety of mechanisms that often involve specific interactions with cellular components. Additionally, some viral genomes can also be processed by cellular machinery to generate non-coding RNAs (ncRNAs) that could have regulatory functions. Among them, specific elements of the host ncRNA transcriptome include microRNAs (miRNAs), long non-coding RNAs (lncRNAs) and circular RNAs (circRNAs), have been described as important players in the regulation of the viral replicative cycle and the cellular antiviral responses (Fig. [1](#Fig1)). To summarize the current knowledge about ncRNAs and SARS-CoV-2, we share the latest details of the different ncRNA families and their regulatory roles during infection and disease progression.

### Fig. 1. Different RNA and their role in COVID-19.

[![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d5e4/10772081/732b81e3ba26/41431_2023_1462_Fig1_HTML.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10772081_41431_2023_1462_Fig1_HTML.jpg)

[Open in a new tab](figure/Fig1/)

Structure and description of different categories of RNA involved in COVID-19.

## miRNAs

We begin with highlighting microRNAs (miRNAs), which are short regulatory non-coding RNAs generated by the enzymatic processing of specific RNA transcripts. Their regulatory functions at the cellular level are exerted by recruiting regulatory proteins (RNA-induced silencing complex, or RISC) which are further targeted to the 3′-UTR of mRNAs by base complementarity, causing a repressive effect on gene expression [[22](#CR22)]. Considering the possible involvement of dysregulated miRNAs upon infection on the systemic manifestations of COVID-19, multiple research groups profiled circulating miRNA levels in SARS-CoV-2 patients. One study of a stratified cohort of COVID-19 patients showed a molecular signature composed of three blood-circulating miRNAs (hsa-miR-423-5p, hsa-miR-23a-3p and hsa-miR-195-5p), independently classified COVID-19 cases, distinguishing them from other viral infections such as H1N1 influenza and healthy controls [[23](#CR23)].

Recently a circulating miRNA, hsa-miR-2392, was shown and suggested to be one of the main drivers of a suppressive effect of mitochondrial gene expression and inflammation effectors, promoting many of the described symptoms associated with COVID-19. This miRNA was also detected in SARS-CoV-2-infected patient serum and urine, opening the possibility of its use as a prognostic biomarker for the disease [[24](#CR24)]. Mitochondrial analysis showed miR-2392 targets were OXPHOS, mitochondrial translation, and mitochondrial metabolism related transcripts. In summary, SARS-CoV-2 elevates miRNAs, such as miR-2392, which can result in affecting biochemical pathways (e.g., downregulation of OXPHOS), and treatment with anti-miR-2393 therapeutics potentially blocks SARS-CoV-2 infection in hamsters and hACE2-A549 cells [[24](#CR24)].

Several additional miRNA’s have also been observed to be involved with COVID-19 outcomes (e.g., level of viral load, biomarkers, etc.), as well as having various biochemical roles (e.g., viral entry, cytokine storm development, activation of signaling pathways, etc.). These include miRNA interaction with cell death processes, regulating pathways such as p53 by miR-101, miR-100, miR-7, miR-107, etc., as well with the viral entry process (e.g., hsa-miR-98-5p binding to TMPRSS2 transcript). Additionally, cellular inflammation processes in COVID-19 infected individuals are affected by miRNA’s (e.g., miR-146a, miR-21, and miR-142), by promoting MAPK and NF-kB signaling causing cellular pro-inflammatory phenotypic adaptations, which has additional consequences such as mitochondrial stress from production of inflammatory factors and reactive oxygen species. These highlight the significant and diverse roles miRNA’s play in steady-state physiology and viral infection induced pathophysiology in COVID-19, which here we briefly describe, but are covered in more detail in the following review [[25](#CR25)].

## lncRNAs

Long non-coding RNAs (lncRNAs) are a wide family of non-coding transcripts characterized by their lack of coding potential, their larger sizes when compared to other ncRNAs (more than 200 nucleotides), and their origin from devoted transcriptional units. Functionally, lncRNAs can regulate the dynamics of the genomic output from chromatin to the cytoplasm, mainly by acting as molecular scaffolds of biological complexes that establish interactions with proteins, DNA, and other RNA molecules. Early transcriptomic studies using cell models show SARS-CoV-2 infection induces a transcriptional pattern that produces several differentially expressed lncRNAs [[26](#CR26)]. Through co-expression network analysis, another study identified four differentially expressed lncRNAs correlating with genes involved in various immune-related pathways, suggesting that the aberrant expression of these four lncRNAs can be associated with SARS-CoV-2 cytokine storms and antiviral responses [[27](#CR27)]. A different study using nasopharyngeal swabs from infected patients characterized an lncRNA, LASI, that is antisense to the ICAM-1 gene and had an expression profile strongly correlating with SARS-CoV-2 viral loads. LASI’s mechanism of action was shown to be through its interaction with the viral genome and subsequent knock-down of viral replication processes, which also resulted in the presence of inflammation markers [[28](#CR28)]. In another study, a pattern of lncRNAs (NRIR, BISPR, and MIR155HG) was observed from nasopharyngeal COVID-19 patient samples that correlated with viral loads [[29](#CR29)]. Interestingly, NRIR and BISPR lncRNAs were also upregulated in the blood of COVID-19 patients as reported in the COVIDOME project [[30](#CR30)], suggesting possible involvement in the systemic manifestations of the viral infection.

## circRNAs

Circular RNAs (circRNAs) are covalently closed RNA molecules generated by non-canonical back-splicing events from coding and non-coding transcripts. Several studies on the role of circRNAs in viral infections, such as SARS-CoV-2, have been published recently; however reports showing the involvement and function of circRNAs with SARS-CoV-2 infection are few and need to be considered with caution due to the limited number of samples and the intrinsic characteristics of the infection models. In one example, a lung epithelial cell model infected with SARS-CoV-2 showed differential circRNA expression upon infection. Analyzing these circRNA parental genes revealed they were involved with inflammation and immune responses [[31](#CR31)]. CircRNAs related to SARS-CoV-2 infection have also been shown to be detectable in the peripheral blood circulation, highlighting their potential as COVID-19 biomarkers [[32](#CR32)].

In this section, we describe humoral factors (and in the previous, metabolism related components) involved in steady-state physiology and responding to SARS-CoV-2 infection in COVID-19 patients, with these intrinsically affecting organ structure and function, and extrinsically affecting systemic physiology based on their production, secretion, and distribution through the circulatory system (Fig. [2](#Fig2)). The circulatory system, part of the cardiovascular system, in of itself adapts to SARS-CoV-2 infection, which we will describe in the next section.

### Fig. 2. SARS-CoV-2 Intracellular Replication and Extracellular Release of Circulating Factors.

[![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d5e4/10772081/c5a3e9eb9678/41431_2023_1462_Fig2_HTML.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10772081_41431_2023_1462_Fig2_HTML.jpg)

[Open in a new tab](figure/Fig2/)

An overview of the processes involved with SARS-CoV-2 infection and replication that lead to cellular adaptations (see ref. [[3](#CR3)] for more details) and extracellular release of various biological factors into the circulation causing systemic physiological effects.

## Cardiovascular adaptations and changes

The cardiovascular system includes the heart, arteries, veins, capillaries, and lymphatics, which are the organ systems responsible for the transport of blood, lymph, and their constituents to every organ in the body. Blood and lymph transport multiple components, including water, red blood cells, white blood cells, macromolecules (e.g., lipids, proteins, carbohydrates, macromolecule, etc.), micromolecules (e.g., nutrients, ions, etc.), of which are circulated systemically and transported to and from all organs of the body. Included in the transport of these components by blood and lymph are pathogens, such as the SARS-CoV-2 virus, which benefits from the cardiovascular system not only by binding to a cell-type expressing its cognate receptor (e.g., endothelial cells and ACE2 receptor), but also from the cardiovascular system’s function in transporting the virus and its replicants away from the host infected cell, tissue, and organ. In turn, the COVID-19 cardiovascular-related symptoms vary greatly, ranging from asymptomatic or mild respiratory symptoms to severe life-threatening respiratory and cardiac failure [[33](#CR33)].

Early reports have described common symptoms of COVID-19 to include fever (88%), fatigue (38%), chills (11%), headache (14%), nasal congestion (5%), sore throat (14%), dry cough (68%), dyspnea (19%), nausea/emesis (5%), diarrhea (4–14%), and myalgias (15%). A meta-analysis of eight studies showed a higher prevalence of hypertension (17 ± 7%) and diabetes mellitus (8 ± 6%) followed by cardiovascular disease (5 ± 4%) in COVID-19 patients [[33](#CR33), [34](#CR34)]. Moreover, concomitant cardiovascular (CV) conditions may be occurring in 8–25% of COVID-19 infected population and an even greater proportion of those who die from the disease [[33](#CR33)]. Indeed, evidence suggests that among the primary causes of death and overall mortality from COVID-19 infection are cardiac issues and respiratory failure [[34](#CR34)].

These symptoms result from changes occurring at the molecular level from SARS-CoV-2 infection. A large autopsy study investigating multiple organs (heart, lung, kidney, liver, and lymph nodes) showed that the cellular and functional gene signatures from each organ’s major cell types (e.g., cardiomyocytes in the heart) decrease as a function of the viral load. More specifically in the heart (*n* = 41 from patient autopsy samples), despite no histological change, many heart functional markers (e.g., TNNI3, TNNT2, MYBPC3, PLN, and HAND1) showed decreased levels resulting from COVID-19 infection [[35](#CR35)]. Another study showed the presence of SARS-CoV-2 in the myocardial heart tissue from autopsy cases, suggested to be tied to myocardial injury [[36](#CR36)]. Indeed, these cellular adaptations extend into the vascular system (e.g., arteries, veins, etc.), with histopathological evidence from autopsy studies showing direct viral infection of endothelial cells and micro- and macrovascular thrombosis occurring in both the arterial and venous circulations. An autopsy study investigating COVID-19 in three patients showed SARS-CoV-2 viral components present in the endothelial cells from different autopsied organs, including the kidney, lung, heart, and liver were each infected with SARS-CoV-2 [[37](#CR37)]. Inflammation of the endothelium was also observed, along with recruitment of neutrophils and mononuclear cells to the site of damage. In vitro studies also support SARS-CoV-2 being able to directly infect cardiovascular cells as seen from experiments showing viral infection of engineered human blood vessel organoids [[38](#CR38)]. Severe COVID-19 has also been associated with endothelial specific changes and effects such as elevated endothelial activation, hypercoagulability, and increased presence of von Willebrand Factor [[39](#CR39), [40](#CR40)]. Endotheliitis from COVID-19 can occur, leading to a loss of cell–cell junctions, loss of endothelial cells, exposure of the basement membrane, abnormal cytokine release, abnormal reactive oxygen species release, and intussusceptive angiogenesis. Endothelin-1 has been suggested as a possible biomarker for those at risk with COVID-19 [[34](#CR34)].

These cardiovascular adaptations from SARS-CoV-2 infection lead to a number of pathophysiological consequences including hypoxia, with prolonged ischemia leading to formation of hypoxanthine as an ATP breakdown product [[36](#CR36)]. These effects may be from tissue remodeling caused by viral infection, as well as changes in blood viscosity, clotting, or inflammation, though more research of these areas would benefit our understanding of the effects of SARS-CoV-2 on the cardiovascular system and organ remodeling [[33](#CR33), [34](#CR34), [36](#CR36)]. For example, a meta-analysis showed there are diverse cardiovascular complications including cardiac injury (e.g., myocardial injury, microinfarcts), heart failure, microembolic infarcts, acute coronary syndrome, arrhythmia, hypertension, and atrial and ventricular complications (e.g., fibrosis) [[33](#CR33), [34](#CR34)]. Autopsy and case reports from COVID-19 patients are also varied in their observations of cardiac tissue changes, highlighting the complexity of the disease and individual differences influencing pathophysiological progression and outcomes. Identifying cardiovascular specific biomarkers would help evaluate, as well as measure progression of disease, with troponin I as one example of a clinical marker evaluating myocardial injury is troponin I. Troponin I has been shown to be elevated in >10% of patients hospitalized with COVID-19, increasing with disease severity [[41](#CR41)]. Identifying more markers with cardiovascular specific outcomes (i.e. metabolites, lipids, humoral factors, etc., mentioned in the above sections) would help evaluate, correlate, and triage patients and improve their outcomes, as well as facilitate identification of specific biochemical pathways that may mechanistically explain disease symptoms.

Indeed, with the metabolic and physiological adaptations seen with COVID-19 patients, understanding mitochondrial adaptations in these contexts is relevant, given mitochondrial function in bioenergetics. As discussed in our first review, the mitochondria is heavily dysregulated and suppressed in the host during SARS-CoV-2 infection [[3](#CR3)]. One study specifically analyzed transcriptomic data on heart samples collected from deceased COVID-19 patients [[6](#CR6)]. This analysis showed heavy suppression at the transcript level of the majority of mitochondrial pathways and genes [[6](#CR6)]. The non-coding transcriptome response to SARS-CoV-2 may also play a significant response and regulatory role with the cardiovascular adaptations developing from COVID-19 disease. Whether miRNA’s (e.g., hsa-miR-2392), lncRNA’s (e.g., NRIR and BISPR), or circRNA’s that are involved in suppressing mitochondrial gene expression, changing oxidative phosphorylation, mitochondrial structure (e.g., translation, metabolism, etc.), and causing inflammatory effects, may be involved with local cardiac adaptations or systemic vascular adaptations is unknown, but warrants future investigations to explore the specific roles of the transcriptome with cardiovascular adaptations. Furthermore, given the changes in cardiovascular structure, function, biochemical adaptations, that occur from SARS-CoV-2 infection, further study of cardiovascular mitochondrial adaptations is of relevance, and would help elucidate the underlying mechanisms impacting the cardiovascular system resultant from SARS-CoV-2 infection.

## Immune adaptations and changes

The immune system includes the innate and adaptive immune systems. The innate system involves nonspecific responses to injury, and includes cell populations such as monocytes, macrophages, neutrophils, basophils, eosinophils, etc. The adaptive immune system involves specific response to pathogen infection, and includes cell populations such as T and B cells. The two arms of the immune system communicate, interact, and support each other in responding to injury, aided by humoral, circulating factors such as cytokines and chemokines, that provide information and context to immune cells as to how, where, and what to respond to, as well as the magnitude of the response. In turn, there are various feedback loops that provide guidance and direction to the immune system, both innate and adaptive, with SARS-CoV-2 activating both the innate and adaptive immune systems.

For the innate immune system, activation appears to be guided by neutrophils, which display increased levels of ROS and activated NETosis [[42](#CR42)]. Additionally, the loss of ACE2 and DABK in the lungs of patients encourages neutrophil infiltration and inflammation. SARS-CoV-2 has also been suggested to dysregulate type 1 interferon signaling, which contributes to immune evasion early in infection. Consistent with this finding, bulk analysis of whole blood cell transcriptomes of severe COVID-19 patients compared with non-critically ill COVID-19 patients showed upregulation of Interferon alpha as the primary contributor to severe COVID-19. Activated monocytes and macrophages also migrate to the lungs (chemotaxis), further exacerbating lung inflammation and pulmonary fibrosis [[43](#CR43)]. Several studies have reported disordered structure of the infected lung, along with extensive immune infiltration [[35](#CR35), [44](#CR44)]. For example, the lung alveoli in COVID-19 patients develop additional hyaluronan, forming a coating that can reduce oxygen exchange, as well as leading to pulmonary edema. Another study of COVID-19 patients showed chronic lung impairment accompanied by persistent respiratory immune alterations, showing specific memory T and B cells enriched at the site of infection [[45](#CR45)]. With the significant neutrophil, macrophage, T and B cell infiltration and activation observed, and localized hyperinflammatory states of these immune cells around alveolar epithelial cells, this develops into more lung damage and loss of compartmental identity. When a timeline and pattern of the cytokine storm in tracheal aspirate samples was organized and reported, divergences in the systemic versus airway cytokine production during COVID-19 were observed [[46](#CR46)]. Indeed, a study assessing lung function of critical COVID-19 patients showed 55% of them experienced persistent impairment three months after ICU discharge, including declines in patient lung function, exercise capacity, and quality of life [[47](#CR47)]. This suggests respiratory failure is another significant factor involved with patient mortality with severe SARS-CoV-2 infection, with an investigation of single-nuclei RNA sequencing of COVID-19 patient lung samples identified substantial alterations in cellular composition, transcriptional cell states, and cell-to-cell interactions, with the data showing COVID-19 patient lungs were highly inflamed, with dense infiltration of aberrantly activated macrophages, though with impaired T-cell responses and suggestions of impaired lung regeneration, based on alveolar type 1 cell phenotypes [[48](#CR48)].

With regard to adaptive immune cell responses to SARS-CoV-2, these include lymphopenia (lowered CD4 and CD8 T-cell number) in COVID-19 patients, also seen in autopsy reports [[49](#CR49), [50](#CR50)]. There are case by case variations; for example, an influx of T cells and myeloid cells occurs into the myocardium. In response to this event, COVID-19’s pathogen-associated molecular patterns and damage-associated molecular patterns (PAMPs and DAMPs) are activated, as well as cytokines released by alveolar macrophages [[51](#CR51), [52](#CR52)]. These pathways are thought to contribute to the dysregulation of monocytes and macrophages in COVID-19 [[53](#CR53)] and the subsequent dysfunction of T cells. Changes also occur within the heart, as seen with gene expression analysis of cardiac tissue for TNF-ɑ, interferon-γ, chemokine ligand 5, interleukin-6, interleukin-8 (CXCL8), and interleukin-18. Severe COVID-19 is also associated with elevated endothelial activation, hypercoagulability, and the presence of von Willebrand Factor (vWF) [[53](#CR53), [54](#CR54)]. Indeed, COVID-19 induces a complex pattern of cytokine release as part of an overall dysregulated immune response that, in severe cases, injures the parenchyma of vital organs and leads to viremia and sepsis [[55](#CR55), [56](#CR56)].

Severe COVID-19 is characterized by an inflammatory profile that involves initial immune evasion by downregulation of Type I Interferon pathways along with significant upregulation of IL-6, IL-8 and TNF-α in the lungs that coincides with increased NF-κB activity [[57](#CR57), [58](#CR58)]. In one study of COVID-19 patients, a notable increase in multiple serum cytokines (e.g., TNF-α, IFN-γ, IL-2, IL-4, IL-6 and IL-10) was seen [[59](#CR59)], while another study showed high plasma levels of IL-1**β**, IL-1RA, IL-7, IL-8, IL-9, IL-10, bFGF, GCSF, GM-CSF, IFN-γ, IP-10, MCP1, MIP-1α, MIP-1**β**, PDGF, TNF-α, and VEGF. A different study showed increased levels of inflammatory markers such as C-reactive protein, D-dimer, ferritin, IL-6 and lactate dehydrogenase (LDH) associated with higher mortality in COVID-19 patients [[60](#CR60)]. These results suggest an overproduction of cytokines in response to lung damage, or as a compensatory response to attempt to re-direct, accelerate, or shutdown the immune response. These cytokine fluctuations also coincide with systemic inflammation and myocardial injury; in 137 patients admitted for COVID-19 disease in Hubei province, heart palpitations were noted as one of the presenting symptoms in 7.3% of patients [[61](#CR61)]. Moreover, there have been reports of complete heart blockage and atrial fibrillation in patients with COVID-19 infection [[62](#CR62)]. In patients diagnosed with severe COVID-19, increased levels of pro-inflammatory cytokines, (e.g., soluble IL-2R, IL-6, IL-8, and TNF-α) have been observed, with these cytokines disrupting endothelial cells antithrombotic and anti-inflammatory functions, leading to coagulation dysregulation, complement and platelet activation, and leukocyte recruitment in the microvasculature [[63](#CR63), [64](#CR64)]. While increased levels of circulating chemokines, pro-inflammatory cytokines, regulatory cytokines, and growth factors were observed in severe COVID patients who died as compared to surviving patients, a more restricted and selective increase of cytokines such as IL-10 was observed in the airway samples with actual decreased levels of pro-inflammatory cytokines such as IFN-γ, IL-17, IL-5, and IL-2 as compared to uninfected controls [[46](#CR46)]. These results demonstrate that inflammation is also highly modulated in the airways as an attempt to decrease exacerbated inflammation. Figure [1](#Fig1) illustrates a snapshot of extracellular factors involved with immune adaptations during COVID-19 and the uniqueness of systemic versus airway inflammatory milieu.

## Organ alterations from SARS-CoV-2 infection

In addition to the cardiovascular system, SARS-CoV-2 infects and has an effect on many organ systems (e.g., cardiovascular, immune, and endocrine systems, as described above), including the lungs, lymph nodes, brain, liver, kidneys, gastrointestinal tract, testis (Fig. [3](#Fig3)) [[65](#CR65)–[67](#CR67)]. COVID-19 is known to cause multi-organ dysfunction, though the burden of infection outside the respiratory tract and time to viral clearance is not well understood. A recent postmortem study of 44 patients quantified and mapped the distribution, replication, and cell-type specificity of SARS-CoV-2 across the body, specifically the brain but other organs as well, from acute infection to more than seven months following symptom onset. It was seen that SARS-CoV-2 infection of the body was widely spread across respiratory and non-respiratory tissues (e.g., the brain) [[68](#CR68)]. This spread of infection provides context for the different symptoms patients experience.

### Fig. 3. COVID-19 organ adaptations and symptoms.

[![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d5e4/10772081/534e062342a7/41431_2023_1462_Fig3_HTML.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10772081_41431_2023_1462_Fig3_HTML.jpg)

[Open in a new tab](figure/Fig3/)

An overview of the systemic organ adaptations resulting from SARS-CoV-2 infection as well as symptoms that develop.

Neurological and psychiatric symptoms have been reported for acute, post-acute, and longer-term SARS-CoV-2 infection in human patients across the human lifespan [[69](#CR69)]. Although neurological manifestations remain incompletely characterized in terms of etiology, severity, and incidence, they present signs of nonspecific (e.g., headache, dizziness, myalgia, fatigue, somnolence, insomnia, etc.) and specific (e.g., encephalopathy, encephalitis, stroke, status epilepticus, ataxia, anosmia, ageusia, demyelinating syndromes, etc.) deficits of the central nervous system, the peripheral nervous system, or both, with new-onset or relapsed associated prodromal, subthreshold, or fully expressed (i.e., suprathreshold) psychiatric conditions. The spectrum of presumed virogenic psychiatric disorders across age cohorts and other demographic classifications is broad and includes anxiety, depression, confusion or disordered thought (e.g., brain fogor psychosis). Structural and functional changes in brain anatomy, such as brief to protracted alterations in gray and white matter volume, blood perfusion and oxygenation, and metabolic activity [[70](#CR70)], often parallel and are consistent with onset and duration of mental health diagnoses in infected populations. Areas caudal to the neocortex tend to be most affected, with olfactory cortex, basal ganglia, hippocampus, amygdala, cerebellum, and brain stem abnormalities occurring in greater frequency and magnitude in COVID-19 patients. Uncertainty regarding the cause of COVID-19-related neuropathology centers on SARS-CoV-2 neuroinvasion, with the olfactory epithelium as a plausible entry point with subsequent retrograde transsynaptic propagation to other brain areas.

Astrocytes and Schwann cells infected by SARS-CoV-2 may also be negatively impacted with their neurotransmission, synapse modulation, reinnervation, metabolite and electrolyte homeostasis, and cell signaling functions. This may cause, contribute, or lead to observed central and peripheral nervous system trauma and dysfunction [[70](#CR70)]. There is data that also supports hematogenous mechanisms enabling viral access to nervous tissue through infection of blood-brain-barrier (BBB) endothelial cells or epithelial cells of the choroid plexus blood–CSF barrier. Indeed, a study examining SARS-CoV-2 interaction with brain organoids suggests a compromised blood-brain-barrier (BBB) as manifested by injured endothelial cells and increased BBB permeability [[71](#CR71)]. Multisystem hyperinflammatory immunoreactions most appropriately describe for the diverse set of pathologies observed in post mortem autopsy analyses, as well as in live in vivo or in vitro tissue, the result of hemorrhagic, ischemic, and hypoxic injury, mitochondrial dysfunction, neuronal damage and apoptosis, and microglial and astrocytic activation.

With regard to gastrointestinal (GI) effects of SARS-CoV-2, an autopsy study of 29 cases showed the GI to be one of the primary organs infected by the virus, as well as being supported by the literature of postmortem organ analysis [[65](#CR65)]. GI involvement and symptoms (e.g., diarrhea, anorexia, nausea/vomiting, abdominal pain, etc.) is associated with long hospital stays and severity of disease, with the rates of GI symptoms of the percent of individuals ranging from 11% to 53%. ACE2 is frequently found in human intestinal epithelial cells, with SARS-CoV-2 being detected in patient GI samples [[72](#CR72)]. Indeed, while there is a paucity of data of SARS-CoV-2 effects on GI physiology, GI symptoms are common in patients with COVID-19, though and should be considered when evaluating patient conditions as well as treatment options.

Here we summarize a few examples of organ adaptations resulting from SARS-CoV-2 infection. These highlight the virus’s diverse biological targets and outcomes, though more studies are warranted given our comprehensive lack of understanding of these, and other, organ alterations, as well in particular, the biochemical pathways that are affected by viral infection. In addressing these knowledge gaps, patient care will improve as well as new treatment modalities will be identified. Moreover, these adaptations are related with long-term implications of SARS-CoV-2 infection, of which we describe in the next section.

## Impact of circulating factors on post-acute sequelae of SARS-CoV-2 (PASC, long COVID-19)

In addition to COVID-19 from SARS-CoV-2 infection, patients may experience long-term symptoms as a result of organ or tissue damage due to SARS-CoV-2. This condition is referred to as post-acute sequelae of SARS-CoV-2 (PASC), or long COVID-19 [[73](#CR73), [74](#CR74)]. PASC patients are burdened with multiple symptoms, including more than a third of PASC patients continuing to experience lung fibrotic abnormalities following hospital discharge [[75](#CR75)]. Many are also at risk of developing microemboli due to maladaptations with clotting physiology (e.g., massively increased levels of vWF which leads to increased platelet activation, etc.) [[76](#CR76)]. Indeed, virus-driven cellular alterations have been suggested to contribute to the pathophysiology of PASC patients [[3](#CR3), [77](#CR77)]. Persistent viral presence and inflammation within the olfactory epithelium has been evident from histological analysis, as well as long-term impairments in taste and smell in the cells of the tongue’s taste buds [[78](#CR78)].

PASC is also characterized by chronic fatigue, which has numerous underlying causes. As a comparison, the post-flu or fever fatigue that occurs (e.g., Myalgic Encephalomyelitis/Chronic Fatigue Syndrome (ME/CFS)) may have some similarities to the persistent fatigue in PASC. PASC symptoms may also be influenced by abnormalities of the central nervous system, such as brain hypometabolism [[79](#CR79)]. A recent postmortem study also quantified and mapped the SARS-CoV-2 organ tropism, finding long-term signs of infection in the brain, as well as across the body within the tissue (e.g., respiratory, cardiovascular, lymphoid, gastrointestinal, genitourinary, endocrine, skin, and peripheral nervous systems) seven months following infection onset, confirmed through in situ hybridization, immunohistochemistry, and immunofluorescence analyses of the tissue, isolation of SARS-CoV-2 from tissue by cell culture, and detection of the spike sequence variants in non-respiratory sites [[71](#CR71)]. While psychological and environmental factors are likely to be involved, long-term biochemical adaptations, including increased oxidative stress and muscle mitochondrial failure, may contribute to the chronic fatigue syndrome found in PASC patients [[80](#CR80)]. In addition, due to the activation of the fibrosis pathway from persistent virus-induced inflammation, the heart may subsequently undergo structural remodeling that may also explain the PASC symptoms [[81](#CR81)]. There are also long-term immune changes, with PASC patients having elevated expression of type I IFN (IFN-) and type III IFN (IFN-1), highly activated innate immune cells, while lacking naive T and B cells [[82](#CR82)], which may also be related with the symptoms PASC patients experience, but also increase their risk of infection, etc.

Indeed, while many individuals recover from SARS-CoV-2, a significant number of patients continue to suffer with long-term disease symptoms (e.g., PASC) long term after recovery from the primary infection [[73](#CR73), [74](#CR74)]. The severity of the primary infection appears to be independent from the possibility and severity of long-term symptoms, though this requires further research. Indeed, it is critical to understand the systemic effects of PASC, its biological mechanism and symptoms, and to develop treatments that promote functional recovery in symptomatic individuals, given the negative health-effects of PASC, as well as the socioeconomic and healthcare burden this condition will impose. Further research on PASC will improve our knowledge and understanding of the condition, as well as enable identification of relevant treatments and interventions to improve patient outcomes.

## Conclusion

SARS-CoV-2 and its resultant acute and chronic diseases, COVID-19 and PASC, have caused significant impacts worldwide. Here we describe and review the latest knowledge about the viral effect on our physiology, as well as the latest research on relevant whole-body metabolism pathways, circulating biological components (e.g., RNA, cytokine, hormones, etc.), and affected organ systems and their physiological adaptations consequent to SARS-CoV-2 infection.

COVID-19 is a pleiotropic, complex, and dynamic condition (Fig. [4](#Fig4)). The viral insults and cellular adaptations differ depending on the context, from cellular to organ systems, and intricate biochemistry ties and integrates these systems together. Herein, we have attempted to capture the prevalent host responses, as well as highlight gaps of knowledge that could be addressed with future research. We have also touched on relevant biochemical targets with potential for mitigating COVID-19 in the human host, and we will describe current and in-development treatment modalities that affect these in Part 3 of our review.

### Fig. 4. COVID-19 biological adaptations and effects.

[![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d5e4/10772081/da1bf892d703/41431_2023_1462_Fig4_HTML.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10772081_41431_2023_1462_Fig4_HTML.jpg)

[Open in a new tab](figure/Fig4/)

The multi-faceted and multiplex physiological adaptations resulting from SARS-CoV-2 infection, including the different cells, tissues, and organs that respond to viral infection. The airway milieu refers to the viral factors contained from inhalation and response to these factors in the lungs, while the systemic milieu refers to the viral factors transported through the circulation system and across body, and in turn, the physiological responses to these factors.

## Acknowledgements

The opinions expressed in this article are those of the authors and do not reflect the view of the National Institutes of Health, the Department of Health and Human Services, the National Science Foundation, or the United States government. APK’s research is supported by the Department of Defense and Swezey & Jewell, Moore and MacKenzie Research Fund.

## Author contributions

Conceptualization: AB, and SAN; Original draft: SAN, DAJ, FJE, and AB; Review & editing: all authors; Figures and visualization: JCAC-d-R, SAN; Funding acquisition: SAN; Supervision: AB.

## Funding

This work was supported by supplemental funds for COVID-19 research from Translational Research Institute of Space Health through NASA Cooperative Agreement NNX16AO69A (T-0404) to AB, and by a NASA Space Biology Postdoctoral Fellowship (80NSSC19K0426) and Human Research Program Augmentation Award (80NSSC19K1322) to SAN.

## Competing interests

RES is on the scientific advisory board of Miromatrix Inc and Lime Therapeutics and is a consultant and speaker for Alnylam Inc. The remaining authors declare no competing financial interests.

## Footnotes

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

These authors contributed equally: S. Anand Narayanan, David A. Jamison Jr.

## Contributor Information

S. Anand Narayanan, Email: anarayan09@gmail.com.

Afshin Beheshti, Email: afshin.beheshti@nasa.gov.

## References

* 1.Rydland HT, Friedman J, Stringhini S, Link BG, Eikemo TA. The radically unequal distribution of Covid-19 vaccinations: a predictable yet avoidable symptom of the fundamental causes of inequality. Humanit Soc Sci Commun. 2022;9:1–16.
* 2.Jalali N, Brustad HK, Frigessi A, MacDonald EA, Meijerink H, Feruglio SL, et al. Increased household transmission and immune escape of the SARS-CoV-2 Omicron compared to Delta variants. Nat Commun. 2022;13:5706. doi: 10.1038/s41467-022-33233-9. [[DOI](https://doi.org/10.1038/s41467-022-33233-9)] [[PMC free article](/articles/PMC9520116/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36175424/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Increased%20household%20transmission%20and%20immune%20escape%20of%20the%20SARS-CoV-2%20Omicron%20compared%20to%20Delta%20variants&author=N%20Jalali&author=HK%20Brustad&author=A%20Frigessi&author=EA%20MacDonald&author=H%20Meijerink&volume=13&publication_year=2022&pages=5706&pmid=36175424&doi=10.1038/s41467-022-33233-9&)]
* 3.Jamison DA, Jr, Anand Narayanan S, Trovão NS, Guarnieri JW, Topper MJ, Moraes-Vieira PM, et al. A comprehensive SARS-CoV-2 and COVID-19 review, Part 1: Intracellular overdrive for SARS-CoV-2 infection. Eur J Hum Genet. 2022;30:889–98. doi: 10.1038/s41431-022-01108-8. [[DOI](https://doi.org/10.1038/s41431-022-01108-8)] [[PMC free article](/articles/PMC9108708/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35577935/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=A%20comprehensive%20SARS-CoV-2%20and%20COVID-19%20review,%20Part%201:%20Intracellular%20overdrive%20for%20SARS-CoV-2%20infection&author=DA%20Jamison&author=S%20Anand%20Narayanan&author=NS%20Trov%C3%A3o&author=JW%20Guarnieri&author=MJ%20Topper&volume=30&publication_year=2022&pages=889-98&pmid=35577935&doi=10.1038/s41431-022-01108-8&)]
* 4.Prasad K, Khatoon F, Rashid S, Ali N, AlAsmari AF, Ahmed MZ, et al. Targeting hub genes and pathways of innate immune response in COVID-19: a network biology perspective. Int J Biol Macromol. 2020;163:1–8. doi: 10.1016/j.ijbiomac.2020.06.228. [[DOI](https://doi.org/10.1016/j.ijbiomac.2020.06.228)] [[PMC free article](/articles/PMC7319641/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32599245/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Biol%20Macromol&title=Targeting%20hub%20genes%20and%20pathways%20of%20innate%20immune%20response%20in%20COVID-19:%20a%20network%20biology%20perspective&author=K%20Prasad&author=F%20Khatoon&author=S%20Rashid&author=N%20Ali&author=AF%20AlAsmari&volume=163&publication_year=2020&pages=1-8&pmid=32599245&doi=10.1016/j.ijbiomac.2020.06.228&)]
* 5.Minakshi R, Padhan K, Rani M, Khan N, Ahmad F, Jameel S. The SARS Coronavirus 3a protein causes endoplasmic reticulum stress and induces ligand-independent downregulation of the type 1 interferon receptor. PLoS ONE. 2009;4:e8342. doi: 10.1371/journal.pone.0008342. [[DOI](https://doi.org/10.1371/journal.pone.0008342)] [[PMC free article](/articles/PMC2791231/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20020050/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=The%20SARS%20Coronavirus%203a%20protein%20causes%20endoplasmic%20reticulum%20stress%20and%20induces%20ligand-independent%20downregulation%20of%20the%20type%201%20interferon%20receptor&author=R%20Minakshi&author=K%20Padhan&author=M%20Rani&author=N%20Khan&author=F%20Ahmad&volume=4&publication_year=2009&pages=e8342&pmid=20020050&doi=10.1371/journal.pone.0008342&)]
* 6.Guarnieri JW, Dybas JM, Fazelinia H, Kim MS, Frere J, Zhang Y, et al. Core mitochondrial genes are down-regulated during SARS-CoV-2 infection of rodent and human hosts. Sci Transl Med. 2023;15:eabq1533. doi: 10.1126/scitranslmed.abq1533. [[DOI](https://doi.org/10.1126/scitranslmed.abq1533)] [[PMC free article](/articles/PMC11624572/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/37556555/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&title=Core%20mitochondrial%20genes%20are%20down-regulated%20during%20SARS-CoV-2%20infection%20of%20rodent%20and%20human%20hosts&author=JW%20Guarnieri&author=JM%20Dybas&author=H%20Fazelinia&author=MS%20Kim&author=J%20Frere&volume=15&publication_year=2023&pages=eabq1533&pmid=37556555&doi=10.1126/scitranslmed.abq1533&)]
* 7.Shen B, Yi X, Sun Y, Bi X, Du J, Zhang C, et al. Proteomic and metabolomic characterization of COVID-19 patient sera. Cell. 2020;182:59–72. doi: 10.1016/j.cell.2020.05.032. [[DOI](https://doi.org/10.1016/j.cell.2020.05.032)] [[PMC free article](/articles/PMC7254001/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32492406/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Proteomic%20and%20metabolomic%20characterization%20of%20COVID-19%20patient%20sera&author=B%20Shen&author=X%20Yi&author=Y%20Sun&author=X%20Bi&author=J%20Du&volume=182&publication_year=2020&pages=59-72&pmid=32492406&doi=10.1016/j.cell.2020.05.032&)]
* 8.Cheng ZJ, Yang X, Wang H. Hyperhomocysteinemia and endothelial dysfunction. Curr Hypertens Rev. 2009;5:158–65. doi: 10.2174/157340209788166940. [[DOI](https://doi.org/10.2174/157340209788166940)] [[PMC free article](/articles/PMC2873778/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20495681/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Hypertens%20Rev&title=Hyperhomocysteinemia%20and%20endothelial%20dysfunction&author=ZJ%20Cheng&author=X%20Yang&author=H%20Wang&volume=5&publication_year=2009&pages=158-65&pmid=20495681&doi=10.2174/157340209788166940&)]
* 9.Barberis E, Timo S, Amede E, Vanella VV, Puricelli C, Cappellano G, et al. Large-scale plasma analysis revealed new mechanisms and molecules associated with the host response to SARS-CoV-2. Int J Mol Sci. 2020;21:8623. doi: 10.3390/ijms21228623. [[DOI](https://doi.org/10.3390/ijms21228623)] [[PMC free article](/articles/PMC7696386/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33207699/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&title=Large-scale%20plasma%20analysis%20revealed%20new%20mechanisms%20and%20molecules%20associated%20with%20the%20host%20response%20to%20SARS-CoV-2&author=E%20Barberis&author=S%20Timo&author=E%20Amede&author=VV%20Vanella&author=C%20Puricelli&volume=21&publication_year=2020&pages=8623&pmid=33207699&doi=10.3390/ijms21228623&)]
* 10.Schwarz B, Sharma L, Roberts L, Peng X, Bermejo S, Leighton I, et al. Severe SARS-CoV-2 infection in humans is defined by a shift in the serum lipidome resulting in dysregulation of eicosanoid immune mediators. J Immunol. 2021;206:329. doi: 10.4049/jimmunol.2001025. [[DOI](https://doi.org/10.4049/jimmunol.2001025)] [[PMC free article](/articles/PMC7962598/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33277388/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol.&title=Severe%20SARS-CoV-2%20infection%20in%20humans%20is%20defined%20by%20a%20shift%20in%20the%20serum%20lipidome%20resulting%20in%20dysregulation%20of%20eicosanoid%20immune%20mediators&author=B%20Schwarz&author=L%20Sharma&author=L%20Roberts&author=X%20Peng&author=S%20Bermejo&volume=206&publication_year=2021&pages=329&pmid=33277388&doi=10.4049/jimmunol.2001025&)]
* 11.Thomas T, Stefanoni D, Reisz JA, Nemkov T, Bertolone L, Francis RO, et al. COVID-19 infection alters kynurenine and fatty acid metabolism, correlating with IL-6 levels and renal status. JCI insight. 2020;5:e140327. [[DOI](https://doi.org/10.1172/jci.insight.140327)] [[PMC free article](/articles/PMC7453907/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32559180/)]
* 12.Tanner JE, Alfieri C. The fatty acid lipid metabolism nexus in COVID-19. Viruses. 2021;13:90. doi: 10.3390/v13010090. [[DOI](https://doi.org/10.3390/v13010090)] [[PMC free article](/articles/PMC7826519/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33440724/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Viruses&title=The%20fatty%20acid%20lipid%20metabolism%20nexus%20in%20COVID-19&author=JE%20Tanner&author=C%20Alfieri&volume=13&publication_year=2021&pages=90&pmid=33440724&doi=10.3390/v13010090&)]
* 13.Garvin MR, Alvarez C, Miller JI, Prates ET, Walker AM, Amos BK, et al. A mechanistic model and therapeutic interventions for COVID-19 involving a RAS-mediated bradykinin storm. elife. 2020;9:e59177. doi: 10.7554/eLife.59177. [[DOI](https://doi.org/10.7554/eLife.59177)] [[PMC free article](/articles/PMC7410499/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32633718/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=elife&title=A%20mechanistic%20model%20and%20therapeutic%20interventions%20for%20COVID-19%20involving%20a%20RAS-mediated%20bradykinin%20storm&author=MR%20Garvin&author=C%20Alvarez&author=JI%20Miller&author=ET%20Prates&author=AM%20Walker&volume=9&publication_year=2020&pages=e59177&pmid=32633718&doi=10.7554/eLife.59177&)]
* 14.Clarke SA, Abbara A, Dhillo WS. Impact of COVID-19 on the endocrine system: a mini-review. Endocrinology. 2022;163:bqab203. doi: 10.1210/endocr/bqab203. [[DOI](https://doi.org/10.1210/endocr/bqab203)] [[PMC free article](/articles/PMC8500009/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34543404/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrinology&title=Impact%20of%20COVID-19%20on%20the%20endocrine%20system:%20a%20mini-review&author=SA%20Clarke&author=A%20Abbara&author=WS%20Dhillo&volume=163&publication_year=2022&pages=bqab203&pmid=34543404&doi=10.1210/endocr/bqab203&)]
* 15.Lazartigues E, Qadir MM, Mauvais-Jarvis F. Endocrine significance of SARS-CoV-2’s reliance on ACE2. Endocrinology. 2020;161:bqaa108. doi: 10.1210/endocr/bqaa108. [[DOI](https://doi.org/10.1210/endocr/bqaa108)] [[PMC free article](/articles/PMC7454499/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32652001/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrinology&title=Endocrine%20significance%20of%20SARS-CoV-2%E2%80%99s%20reliance%20on%20ACE2&author=E%20Lazartigues&author=MM%20Qadir&author=F%20Mauvais-Jarvis&volume=161&publication_year=2020&pages=bqaa108&pmid=32652001&doi=10.1210/endocr/bqaa108&)]
* 16.Kitsou K, Kotanidou A, Paraskevis D, Karamitros T, Katzourakis A, Tedder R, et al. Upregulation of human endogenous retroviruses in bronchoalveolar lavage fluid of COVID-19 patients. Microbiol Spectr. 2021;9:e01260–21. doi: 10.1128/Spectrum.01260-21. [[DOI](https://doi.org/10.1128/Spectrum.01260-21)] [[PMC free article](/articles/PMC8510252/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34612698/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol%20Spectr&title=Upregulation%20of%20human%20endogenous%20retroviruses%20in%20bronchoalveolar%20lavage%20fluid%20of%20COVID-19%20patients&author=K%20Kitsou&author=A%20Kotanidou&author=D%20Paraskevis&author=T%20Karamitros&author=A%20Katzourakis&volume=9&publication_year=2021&pages=e01260-21&pmid=34612698&doi=10.1128/Spectrum.01260-21&)]
* 17.Alfaro E, Díaz-García E, García-Tovar S, Zamarrón E, Mangas A, Galera R, et al. Impaired kallikrein-kinin system in COVID-19 patients’ severity. Front Immunol. 2022;13:909342. doi: 10.3389/fimmu.2022.909342. [[DOI](https://doi.org/10.3389/fimmu.2022.909342)] [[PMC free article](/articles/PMC9258198/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35812405/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=Impaired%20kallikrein-kinin%20system%20in%20COVID-19%20patients%E2%80%99%20severity&author=E%20Alfaro&author=E%20D%C3%ADaz-Garc%C3%ADa&author=S%20Garc%C3%ADa-Tovar&author=E%20Zamarr%C3%B3n&author=A%20Mangas&volume=13&publication_year=2022&pages=909342&pmid=35812405&doi=10.3389/fimmu.2022.909342&)]
* 18.Sur S, Steele R, Isbell TS, Ray R, Ray RB. Circulatory exosomes from COVID-19 patients trigger NLRP3 inflammasome in endothelial cells. Mbio. 2022;13:e00951–22. doi: 10.1128/mbio.00951-22. [[DOI](https://doi.org/10.1128/mbio.00951-22)] [[PMC free article](/articles/PMC9239151/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35587188/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mbio&title=Circulatory%20exosomes%20from%20COVID-19%20patients%20trigger%20NLRP3%20inflammasome%20in%20endothelial%20cells&author=S%20Sur&author=R%20Steele&author=TS%20Isbell&author=R%20Ray&author=RB%20Ray&volume=13&publication_year=2022&pages=e00951-22&pmid=35587188&doi=10.1128/mbio.00951-22&)]
* 19.Pesce E, Manfrini N, Cordiglieri C, Santi S, Bandera A, Gobbini A, et al. Exosomes recovered from the plasma of COVID-19 patients expose SARS-CoV-2 spike-derived fragments and contribute to the adaptive immune response. Front Immunol. 2022;12:785941. doi: 10.3389/fimmu.2021.785941. [[DOI](https://doi.org/10.3389/fimmu.2021.785941)] [[PMC free article](/articles/PMC8801440/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35111156/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=Exosomes%20recovered%20from%20the%20plasma%20of%20COVID-19%20patients%20expose%20SARS-CoV-2%20spike-derived%20fragments%20and%20contribute%20to%20the%20adaptive%20immune%20response&author=E%20Pesce&author=N%20Manfrini&author=C%20Cordiglieri&author=S%20Santi&author=A%20Bandera&volume=12&publication_year=2022&pages=785941&pmid=35111156&doi=10.3389/fimmu.2021.785941&)]
* 20.Ching KL, de Vries M, Gago J, Dancel-Manning K, Sall J, Rice WJ, et al. ACE2-containing defensosomes serve as decoys to inhibit SARS-CoV-2 infection. PLoS Biol. 2022;20:e3001754. doi: 10.1371/journal.pbio.3001754. [[DOI](https://doi.org/10.1371/journal.pbio.3001754)] [[PMC free article](/articles/PMC9469972/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36099266/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=ACE2-containing%20defensosomes%20serve%20as%20decoys%20to%20inhibit%20SARS-CoV-2%20infection&author=KL%20Ching&author=M%20de%20Vries&author=J%20Gago&author=K%20Dancel-Manning&author=J%20Sall&volume=20&publication_year=2022&pages=e3001754&pmid=36099266&doi=10.1371/journal.pbio.3001754&)]
* 21.Peluso MJ, Deeks SG, Mustapic M, Kapogiannis D, Henrich TJ, Lu S, et al. SARS‐CoV‐2 and mitochondrial proteins in neural‐derived exosomes of COVID‐19. Ann Neurol. 2022;91:772–81. doi: 10.1002/ana.26350. [[DOI](https://doi.org/10.1002/ana.26350)] [[PMC free article](/articles/PMC9082480/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35285072/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Neurol&title=SARS%E2%80%90CoV%E2%80%902%20and%20mitochondrial%20proteins%20in%20neural%E2%80%90derived%20exosomes%20of%20COVID%E2%80%9019&author=MJ%20Peluso&author=SG%20Deeks&author=M%20Mustapic&author=D%20Kapogiannis&author=TJ%20Henrich&volume=91&publication_year=2022&pages=772-81&pmid=35285072&doi=10.1002/ana.26350&)]
* 22.Leitão AL, Enguita FJ. A structural view of miRNA biogenesis and function. Noncoding RNA. 2022;8:10. doi: 10.3390/ncrna8010010. [[DOI](https://doi.org/10.3390/ncrna8010010)] [[PMC free article](/articles/PMC8874510/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35202084/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Noncoding%20RNA&title=A%20structural%20view%20of%20miRNA%20biogenesis%20and%20function&author=AL%20Leit%C3%A3o&author=FJ%20Enguita&volume=8&publication_year=2022&pages=10&pmid=35202084&doi=10.3390/ncrna8010010&)]
* 23.Farr RJ, Rootes CL, Rowntree LC, Nguyen TH, Hensen L, Kedzierski L, et al. Altered microRNA expression in COVID-19 patients enables identification of SARS-CoV-2 infection. PLoS Pathog. 2021;17:e1009759. doi: 10.1371/journal.ppat.1009759. [[DOI](https://doi.org/10.1371/journal.ppat.1009759)] [[PMC free article](/articles/PMC8318295/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34320031/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&title=Altered%20microRNA%20expression%20in%20COVID-19%20patients%20enables%20identification%20of%20SARS-CoV-2%20infection&author=RJ%20Farr&author=CL%20Rootes&author=LC%20Rowntree&author=TH%20Nguyen&author=L%20Hensen&volume=17&publication_year=2021&pages=e1009759&pmid=34320031&doi=10.1371/journal.ppat.1009759&)]
* 24.McDonald JT, Enguita FJ, Taylor D, Griffin RJ, Priebe W, Emmett MR, et al. Role of miR-2392 in driving SARS-CoV-2 infection. Cell Rep. 2021;37:109839. [[DOI](https://doi.org/10.1016/j.celrep.2021.109839)] [[PMC free article](/articles/PMC8481092/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34624208/)]
* 25.Panda M, Kalita E, Singh S, Kumar K, Rao A, Prajapati VK. MiRNA-SARS-CoV-2 dialogue and prospective anti-COVID-19 therapies. Life Sci. 2022:120761. [[DOI](https://doi.org/10.1016/j.lfs.2022.120761)] [[PMC free article](/articles/PMC9249409/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35787998/)]
* 26.Shaath H, Vishnubalaji R, Elkord E, Alajez NM. Single-cell transcriptome analysis highlights a role for neutrophils and inflammatory macrophages in the pathogenesis of severe COVID-19. Cells. 2020;9:2374. doi: 10.3390/cells9112374. [[DOI](https://doi.org/10.3390/cells9112374)] [[PMC free article](/articles/PMC7693119/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33138195/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cells&title=Single-cell%20transcriptome%20analysis%20highlights%20a%20role%20for%20neutrophils%20and%20inflammatory%20macrophages%20in%20the%20pathogenesis%20of%20severe%20COVID-19&author=H%20Shaath&author=R%20Vishnubalaji&author=E%20Elkord&author=NM%20Alajez&volume=9&publication_year=2020&pages=2374&pmid=33138195&doi=10.3390/cells9112374&)]
* 27.Mukherjee S, Banerjee B, Karasik D, Frenkel-Morgenstern M. mRNA-lncRNA Co-expression network analysis reveals the role of lncRNAs in immune dysfunction during severe SARS-CoV-2 infection. Viruses. 2021;13:402. doi: 10.3390/v13030402. [[DOI](https://doi.org/10.3390/v13030402)] [[PMC free article](/articles/PMC7999169/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33802569/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Viruses&title=mRNA-lncRNA%20Co-expression%20network%20analysis%20reveals%20the%20role%20of%20lncRNAs%20in%20immune%20dysfunction%20during%20severe%20SARS-CoV-2%20infection&author=S%20Mukherjee&author=B%20Banerjee&author=D%20Karasik&author=M%20Frenkel-Morgenstern&volume=13&publication_year=2021&pages=402&pmid=33802569&doi=10.3390/v13030402&)]
* 28.Devadoss D, Acharya A, Manevski M, Houserova D, Cioffi MD, Pandey K, et al. Immunomodulatory LncRNA on antisense strand of ICAM-1 augments SARS-CoV-2 infection-associated airway mucoinflammatory phenotype. Iscience. 2022;25:104685. [[DOI](https://doi.org/10.1016/j.isci.2022.104685)] [[PMC free article](/articles/PMC9242679/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35789750/)]
* 29.Enguita FJ, Leitão AL, McDonald JT, Zaksas V, Das S, Galeano D, et al. The interplay between lncRNAs, RNA-binding proteins and viral genome during SARS-CoV-2 infection reveals strong connections with regulatory events involved in RNA metabolism and immune response. Theranostics. 2022;12:3946. doi: 10.7150/thno.73268. [[DOI](https://doi.org/10.7150/thno.73268)] [[PMC free article](/articles/PMC9131284/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35664076/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Theranostics&title=The%20interplay%20between%20lncRNAs,%20RNA-binding%20proteins%20and%20viral%20genome%20during%20SARS-CoV-2%20infection%20reveals%20strong%20connections%20with%20regulatory%20events%20involved%20in%20RNA%20metabolism%20and%20immune%20response&author=FJ%20Enguita&author=AL%20Leit%C3%A3o&author=JT%20McDonald&author=V%20Zaksas&author=S%20Das&volume=12&publication_year=2022&pages=3946&pmid=35664076&doi=10.7150/thno.73268&)]
* 30.Sullivan KD, Galbraith MD, Kinning KT, Bartsch KW, Levinsky NC, Araya P, et al. The COVIDome explorer researcher portal. Cell Rep. 2021;36:109527. [[DOI](https://doi.org/10.1016/j.celrep.2021.109527)] [[PMC free article](/articles/PMC8316015/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34348131/)]
* 31.Yang M, Qi M, Xu L, Huang P, Wang X, Sun J, et al. Differential host circRNA expression profiles in human lung epithelial cells infected with SARS-CoV-2. Infect, Genet Evol. 2021;93:104923. doi: 10.1016/j.meegid.2021.104923. [[DOI](https://doi.org/10.1016/j.meegid.2021.104923)] [[PMC free article](/articles/PMC8123525/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34004360/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect,%20Genet%20Evol&title=Differential%20host%20circRNA%20expression%20profiles%20in%20human%20lung%20epithelial%20cells%20infected%20with%20SARS-CoV-2&author=M%20Yang&author=M%20Qi&author=L%20Xu&author=P%20Huang&author=X%20Wang&volume=93&publication_year=2021&pages=104923&pmid=34004360&doi=10.1016/j.meegid.2021.104923&)]
* 32.Wu Y, Zhao T, Deng R, Xia X, Li B, Wang X. A study of differential circRNA and lncRNA expressions in COVID-19-infected peripheral blood. Sci Rep. 2021;11:7991. doi: 10.1038/s41598-021-86134-0. [[DOI](https://doi.org/10.1038/s41598-021-86134-0)] [[PMC free article](/articles/PMC8041881/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33846375/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=A%20study%20of%20differential%20circRNA%20and%20lncRNA%20expressions%20in%20COVID-19-infected%20peripheral%20blood&author=Y%20Wu&author=T%20Zhao&author=R%20Deng&author=X%20Xia&author=B%20Li&volume=11&publication_year=2021&pages=7991&pmid=33846375&doi=10.1038/s41598-021-86134-0&)]
* 33.Farshidfar F, Koleini N, Ardehali H. Cardiovascular complications of COVID-19. JCI insight. 2021;6:e148980. [[DOI](https://doi.org/10.1172/jci.insight.148980)] [[PMC free article](/articles/PMC8410051/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34061779/)]
* 34.Clerkin KJ, Fried JA, Raikhelkar J, Sayer G, Griffin JM, Masoumi A, et al. COVID-19 and cardiovascular disease. Circulation. 2020;141:1648–55. doi: 10.1161/CIRCULATIONAHA.120.046941. [[DOI](https://doi.org/10.1161/CIRCULATIONAHA.120.046941)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32200663/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=COVID-19%20and%20cardiovascular%20disease&author=KJ%20Clerkin&author=JA%20Fried&author=J%20Raikhelkar&author=G%20Sayer&author=JM%20Griffin&volume=141&publication_year=2020&pages=1648-55&pmid=32200663&doi=10.1161/CIRCULATIONAHA.120.046941&)]
* 35.Park J, Foox J, Hether T, Danko DC, Warren S, Kim Y, et al. System-wide transcriptome damage and tissue identity loss in COVID-19 patients. Cell Rep. Med. 2022;3:100522. doi: 10.1016/j.xcrm.2022.100522. [[DOI](https://doi.org/10.1016/j.xcrm.2022.100522)] [[PMC free article](/articles/PMC8784611/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35233546/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.%20Med&title=System-wide%20transcriptome%20damage%20and%20tissue%20identity%20loss%20in%20COVID-19%20patients&author=J%20Park&author=J%20Foox&author=T%20Hether&author=DC%20Danko&author=S%20Warren&volume=3&publication_year=2022&pages=100522&pmid=35233546&doi=10.1016/j.xcrm.2022.100522&)]
* 36.Lindner D, Fitzek A, Bräuninger H, Aleshcheva G, Edler C, Meissner K, et al. Association of cardiac infection with SARS-CoV-2 in confirmed COVID-19 autopsy cases. JAMA Cardiol. 2020;5:1281–5. doi: 10.1001/jamacardio.2020.3551. [[DOI](https://doi.org/10.1001/jamacardio.2020.3551)] [[PMC free article](/articles/PMC7385672/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32730555/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Cardiol&title=Association%20of%20cardiac%20infection%20with%20SARS-CoV-2%20in%20confirmed%20COVID-19%20autopsy%20cases&author=D%20Lindner&author=A%20Fitzek&author=H%20Br%C3%A4uninger&author=G%20Aleshcheva&author=C%20Edler&volume=5&publication_year=2020&pages=1281-5&pmid=32730555&doi=10.1001/jamacardio.2020.3551&)]
* 37.Varga Z, Flammer AJ, Steiger P, Haberecker M, Andermatt R, Zinkernagel AS, et al. Endothelial cell infection and endotheliitis in COVID-19. Lancet. 2020;395:1417–8. doi: 10.1016/S0140-6736(20)30937-5. [[DOI](https://doi.org/10.1016/S0140-6736%2820%2930937-5)] [[PMC free article](/articles/PMC7172722/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32325026/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Endothelial%20cell%20infection%20and%20endotheliitis%20in%20COVID-19&author=Z%20Varga&author=AJ%20Flammer&author=P%20Steiger&author=M%20Haberecker&author=R%20Andermatt&volume=395&publication_year=2020&pages=1417-8&pmid=32325026&doi=10.1016/S0140-6736(20)30937-5&)]
* 38.Monteil V, Kwon H, Prado P, Hagelkrüys A, Wimmer RA, Stahl M, et al. Inhibition of SARS-CoV-2 infections in engineered human tissues using clinical-grade soluble human ACE2. Cell. 2020;181:905–13. doi: 10.1016/j.cell.2020.04.004. [[DOI](https://doi.org/10.1016/j.cell.2020.04.004)] [[PMC free article](/articles/PMC7181998/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32333836/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Inhibition%20of%20SARS-CoV-2%20infections%20in%20engineered%20human%20tissues%20using%20clinical-grade%20soluble%20human%20ACE2&author=V%20Monteil&author=H%20Kwon&author=P%20Prado&author=A%20Hagelkr%C3%BCys&author=RA%20Wimmer&volume=181&publication_year=2020&pages=905-13&pmid=32333836&doi=10.1016/j.cell.2020.04.004&)]
* 39.Chang R, Mamun A, Dominic A, Le NT. SARS-CoV-2 mediated endothelial dysfunction: the potential role of chronic oxidative stress. Front Physiol. 2021;11:605908. doi: 10.3389/fphys.2020.605908. [[DOI](https://doi.org/10.3389/fphys.2020.605908)] [[PMC free article](/articles/PMC7844210/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33519510/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Physiol&title=SARS-CoV-2%20mediated%20endothelial%20dysfunction:%20the%20potential%20role%20of%20chronic%20oxidative%20stress&author=R%20Chang&author=A%20Mamun&author=A%20Dominic&author=NT%20Le&volume=11&publication_year=2021&pages=605908&pmid=33519510&doi=10.3389/fphys.2020.605908&)]
* 40.Pons S, Fodil S, Azoulay E, Zafrani L. The vascular endothelium: the cornerstone of organ dysfunction in severe SARS-CoV-2 infection. Crit Care. 2020;24:1–8. doi: 10.1186/s13054-020-03062-7. [[DOI](https://doi.org/10.1186/s13054-020-03062-7)] [[PMC free article](/articles/PMC7296907/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32546188/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Care&title=The%20vascular%20endothelium:%20the%20cornerstone%20of%20organ%20dysfunction%20in%20severe%20SARS-CoV-2%20infection&author=S%20Pons&author=S%20Fodil&author=E%20Azoulay&author=L%20Zafrani&volume=24&publication_year=2020&pages=1-8&pmid=32546188&doi=10.1186/s13054-020-03062-7&)]
* 41.Tersalvi G, Vicenzi M, Calabretta D, Biasco L, Pedrazzini G, Winterton D. Elevated troponin in patients with coronavirus disease 2019: possible mechanisms. J Card Fail. 2020;26:470–5. doi: 10.1016/j.cardfail.2020.04.009. [[DOI](https://doi.org/10.1016/j.cardfail.2020.04.009)] [[PMC free article](/articles/PMC7166030/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32315733/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Card%20Fail&title=Elevated%20troponin%20in%20patients%20with%20coronavirus%20disease%202019:%20possible%20mechanisms&author=G%20Tersalvi&author=M%20Vicenzi&author=D%20Calabretta&author=L%20Biasco&author=G%20Pedrazzini&volume=26&publication_year=2020&pages=470-5&pmid=32315733&doi=10.1016/j.cardfail.2020.04.009&)]
* 42.Arcanjo A, Logullo J, Menezes CC, de Souza Carvalho Giangiarulo TC, Dos Reis MC, de Castro GM, et al. The emerging role of neutrophil extracellular traps in severe acute respiratory syndrome coronavirus 2 (COVID-19) Sci Rep. 2020;10:19630. doi: 10.1038/s41598-020-76781-0. [[DOI](https://doi.org/10.1038/s41598-020-76781-0)] [[PMC free article](/articles/PMC7665044/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33184506/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=The%20emerging%20role%20of%20neutrophil%20extracellular%20traps%20in%20severe%20acute%20respiratory%20syndrome%20coronavirus%202%20(COVID-19)&author=A%20Arcanjo&author=J%20Logullo&author=CC%20Menezes&author=TC%20de%20Souza%20Carvalho%20Giangiarulo&author=MC%20Dos%20Reis&volume=10&publication_year=2020&pages=19630&pmid=33184506&doi=10.1038/s41598-020-76781-0&)]
* 43.Karki R, Lee S, Mall R, Pandian N, Wang Y, Sharma BR, et al. ZBP1-dependent inflammatory cell death, PANoptosis, and cytokine storm disrupt IFN therapeutic efficacy during coronavirus infection. Sci Immunol. 2022;7:eabo6294. doi: 10.1126/sciimmunol.abo6294. [[DOI](https://doi.org/10.1126/sciimmunol.abo6294)] [[PMC free article](/articles/PMC9161373/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35587515/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Immunol&title=ZBP1-dependent%20inflammatory%20cell%20death,%20PANoptosis,%20and%20cytokine%20storm%20disrupt%20IFN%20therapeutic%20efficacy%20during%20coronavirus%20infection&author=R%20Karki&author=S%20Lee&author=R%20Mall&author=N%20Pandian&author=Y%20Wang&volume=7&publication_year=2022&pages=eabo6294&pmid=35587515&doi=10.1126/sciimmunol.abo6294&)]
* 44.Rendeiro AF, Ravichandran H, Bram Y, Chandar V, Kim J, Meydan C, et al. The spatial landscape of lung pathology during COVID-19 progression. Nature. 2021;593:564–9. doi: 10.1038/s41586-021-03475-6. [[DOI](https://doi.org/10.1038/s41586-021-03475-6)] [[PMC free article](/articles/PMC8204801/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33780969/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20spatial%20landscape%20of%20lung%20pathology%20during%20COVID-19%20progression&author=AF%20Rendeiro&author=H%20Ravichandran&author=Y%20Bram&author=V%20Chandar&author=J%20Kim&volume=593&publication_year=2021&pages=564-9&pmid=33780969&doi=10.1038/s41586-021-03475-6&)]
* 45.Cheon IS, Li C, Son YM, Goplen NP, Wu Y, Cassmann T, et al. Immune signatures underlying post-acute COVID-19 lung sequelae. Sci Immunol. 2021;6:eabk1741. doi: 10.1126/sciimmunol.abk1741. [[DOI](https://doi.org/10.1126/sciimmunol.abk1741)] [[PMC free article](/articles/PMC8763087/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34591653/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Immunol&title=Immune%20signatures%20underlying%20post-acute%20COVID-19%20lung%20sequelae&author=IS%20Cheon&author=C%20Li&author=YM%20Son&author=NP%20Goplen&author=Y%20Wu&volume=6&publication_year=2021&pages=eabk1741&pmid=34591653&doi=10.1126/sciimmunol.abk1741&)]
* 46.Gonçalves JJ, da Mata CP, Lourenço AA, Ribeiro ÁL, Ferreira GM, Fraga-Silva TF, et al. Timeline kinetics of systemic and airway immune mediator storm for comprehensive analysis of disease outcome in critically ill COVID-19 patients. Front Immunol. 2022;13:903903. doi: 10.3389/fimmu.2022.903903. [[DOI](https://doi.org/10.3389/fimmu.2022.903903)] [[PMC free article](/articles/PMC9204232/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35720401/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=Timeline%20kinetics%20of%20systemic%20and%20airway%20immune%20mediator%20storm%20for%20comprehensive%20analysis%20of%20disease%20outcome%20in%20critically%20ill%20COVID-19%20patients&author=JJ%20Gon%C3%A7alves&author=CP%20da%20Mata&author=AA%20Louren%C3%A7o&author=%C3%81L%20Ribeiro&author=GM%20Ferreira&volume=13&publication_year=2022&pages=903903&pmid=35720401&doi=10.3389/fimmu.2022.903903&)]
* 47.Truffaut L, Demey L, Bruyneel AV, Roman A, Alard S, De Vos N, et al. Post-discharge critical COVID-19 lung function related to severity of radiologic lung involvement at admission. Respir Res. 2021;22:1–6. doi: 10.1186/s12931-021-01625-y. [[DOI](https://doi.org/10.1186/s12931-021-01625-y)] [[PMC free article](/articles/PMC7819622/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33478527/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Res&title=Post-discharge%20critical%20COVID-19%20lung%20function%20related%20to%20severity%20of%20radiologic%20lung%20involvement%20at%20admission&author=L%20Truffaut&author=L%20Demey&author=AV%20Bruyneel&author=A%20Roman&author=S%20Alard&volume=22&publication_year=2021&pages=1-6&pmid=33478527&doi=10.1186/s12931-021-01625-y&)]
* 48.Melms JC, Biermann J, Huang H, Wang Y, Nair A, Tagore S, et al. A molecular single-cell lung atlas of lethal COVID-19. Nature. 2021;595:114–9. doi: 10.1038/s41586-021-03569-1. [[DOI](https://doi.org/10.1038/s41586-021-03569-1)] [[PMC free article](/articles/PMC8814825/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33915568/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20molecular%20single-cell%20lung%20atlas%20of%20lethal%20COVID-19&author=JC%20Melms&author=J%20Biermann&author=H%20Huang&author=Y%20Wang&author=A%20Nair&volume=595&publication_year=2021&pages=114-9&pmid=33915568&doi=10.1038/s41586-021-03569-1&)]
* 49.Singh Y, Trautwein C, Fendel R, Krickeberg N, Berezhnoy G, Bissinger R, et al. SARS-CoV-2 infection paralyzes cytotoxic and metabolic functions of the immune cells. Heliyon. 2021;7:e07147. [[DOI](https://doi.org/10.1016/j.heliyon.2021.e07147)] [[PMC free article](/articles/PMC8159709/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34075347/)]
* 50.Sette A, Crotty S. Adaptive immunity to SARS-CoV-2 and COVID-19. Cell. 2021;184:861–80. doi: 10.1016/j.cell.2021.01.007. [[DOI](https://doi.org/10.1016/j.cell.2021.01.007)] [[PMC free article](/articles/PMC7803150/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33497610/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Adaptive%20immunity%20to%20SARS-CoV-2%20and%20COVID-19&author=A%20Sette&author=S%20Crotty&volume=184&publication_year=2021&pages=861-80&pmid=33497610&doi=10.1016/j.cell.2021.01.007&)]
* 51.Onofrio L, Caraglia M, Facchini G, Margherita V, Placido SD, Buonerba C. Toll-like receptors and COVID-19: a two-faced story with an exciting ending. Future Sci OA. 2020;6:FSO605. doi: 10.2144/fsoa-2020-0091. [[DOI](https://doi.org/10.2144/fsoa-2020-0091)] [[PMC free article](/articles/PMC7434222/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32974046/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Sci%20OA&title=Toll-like%20receptors%20and%20COVID-19:%20a%20two-faced%20story%20with%20an%20exciting%20ending&author=L%20Onofrio&author=M%20Caraglia&author=G%20Facchini&author=V%20Margherita&author=SD%20Placido&volume=6&publication_year=2020&pages=FSO605&pmid=32974046&doi=10.2144/fsoa-2020-0091&)]
* 52.Patra R, Das NC, Mukherjee S. Targeting human TLRs to combat COVID‐19: a solution? J Med Virol. 2021;93:615. doi: 10.1002/jmv.26387. [[DOI](https://doi.org/10.1002/jmv.26387)] [[PMC free article](/articles/PMC7436140/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32749702/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Virol&title=Targeting%20human%20TLRs%20to%20combat%20COVID%E2%80%9019:%20a%20solution?&author=R%20Patra&author=NC%20Das&author=S%20Mukherjee&volume=93&publication_year=2021&pages=615&pmid=32749702&doi=10.1002/jmv.26387&)]
* 53.Merad M, Martin JC. Pathological inflammation in patients with COVID-19: a key role for monocytes and macrophages. Nat Rev Immunol. 2020;20:355–62. doi: 10.1038/s41577-020-0331-4. [[DOI](https://doi.org/10.1038/s41577-020-0331-4)] [[PMC free article](/articles/PMC7201395/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32376901/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&title=Pathological%20inflammation%20in%20patients%20with%20COVID-19:%20a%20key%20role%20for%20monocytes%20and%20macrophages&author=M%20Merad&author=JC%20Martin&volume=20&publication_year=2020&pages=355-62&pmid=32376901&doi=10.1038/s41577-020-0331-4&)]
* 54.Mancini I, Baronciani L, Artoni A, Colpani P, Biganzoli M, Cozzi G, et al. The ADAMTS13‐von Willebrand factor axis in COVID‐19 patients. J Thromb Haemost. 2021;19:513–21. doi: 10.1111/jth.15191. [[DOI](https://doi.org/10.1111/jth.15191)] [[PMC free article](/articles/PMC7753796/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33230904/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=The%20ADAMTS13%E2%80%90von%20Willebrand%20factor%20axis%20in%20COVID%E2%80%9019%20patients&author=I%20Mancini&author=L%20Baronciani&author=A%20Artoni&author=P%20Colpani&author=M%20Biganzoli&volume=19&publication_year=2021&pages=513-21&pmid=33230904&doi=10.1111/jth.15191&)]
* 55.Hojyo S, Uchida M, Tanaka K, Hasebe R, Tanaka Y, Murakami M, et al. How COVID-19 induces cytokine storm with high mortality. Inflamm Regen. 2020;40:1–7. doi: 10.1186/s41232-020-00146-3. [[DOI](https://doi.org/10.1186/s41232-020-00146-3)] [[PMC free article](/articles/PMC7527296/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33014208/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Inflamm%20Regen&title=How%20COVID-19%20induces%20cytokine%20storm%20with%20high%20mortality&author=S%20Hojyo&author=M%20Uchida&author=K%20Tanaka&author=R%20Hasebe&author=Y%20Tanaka&volume=40&publication_year=2020&pages=1-7&pmid=33014208&doi=10.1186/s41232-020-00146-3&)]
* 56.Tan C, Li S, Liang Y, Chen M, Liu J. SARS-CoV-2 viremia may predict rapid deterioration of COVID-19 patients. Braz J Infect Dis. 2021;24:565–9. doi: 10.1016/j.bjid.2020.08.010. [[DOI](https://doi.org/10.1016/j.bjid.2020.08.010)] [[PMC free article](/articles/PMC7492058/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32950458/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Braz%20J%20Infect%20Dis&title=SARS-CoV-2%20viremia%20may%20predict%20rapid%20deterioration%20of%20COVID-19%20patients&author=C%20Tan&author=S%20Li&author=Y%20Liang&author=M%20Chen&author=J%20Liu&volume=24&publication_year=2021&pages=565-9&pmid=32950458&doi=10.1016/j.bjid.2020.08.010&)]
* 57.Bryce C, Grimes Z, Pujadas E, Ahuja S, Beasley MB, Albrecht R, et al. Pathophysiology of SARS-CoV-2: the Mount Sinai COVID-19 autopsy experience. Mod Pathol. 2021;34:1456–67. doi: 10.1038/s41379-021-00793-y. [[DOI](https://doi.org/10.1038/s41379-021-00793-y)] [[PMC free article](/articles/PMC8015313/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33795830/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mod%20Pathol&title=Pathophysiology%20of%20SARS-CoV-2:%20the%20Mount%20Sinai%20COVID-19%20autopsy%20experience&author=C%20Bryce&author=Z%20Grimes&author=E%20Pujadas&author=S%20Ahuja&author=MB%20Beasley&volume=34&publication_year=2021&pages=1456-67&pmid=33795830&doi=10.1038/s41379-021-00793-y&)]
* 58.Hadjadj J, Yatim N, Barnabei L, Corneau A, Boussier J, Smith N, et al. Impaired type I interferon activity and inflammatory responses in severe COVID-19 patients. Science. 2020;369:718–24. doi: 10.1126/science.abc6027. [[DOI](https://doi.org/10.1126/science.abc6027)] [[PMC free article](/articles/PMC7402632/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32661059/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Impaired%20type%20I%20interferon%20activity%20and%20inflammatory%20responses%20in%20severe%20COVID-19%20patients&author=J%20Hadjadj&author=N%20Yatim&author=L%20Barnabei&author=A%20Corneau&author=J%20Boussier&volume=369&publication_year=2020&pages=718-24&pmid=32661059&doi=10.1126/science.abc6027&)]
* 59.Han H, Ma Q, Li C, Liu R, Zhao L, Wang W, et al. Profiling serum cytokines in COVID-19 patients reveals IL-6 and IL-10 are disease severity predictors. Emerg Microbes Infect. 2020;9:1123–30. doi: 10.1080/22221751.2020.1770129. [[DOI](https://doi.org/10.1080/22221751.2020.1770129)] [[PMC free article](/articles/PMC7473317/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32475230/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Emerg%20Microbes%20Infect&title=Profiling%20serum%20cytokines%20in%20COVID-19%20patients%20reveals%20IL-6%20and%20IL-10%20are%20disease%20severity%20predictors&author=H%20Han&author=Q%20Ma&author=C%20Li&author=R%20Liu&author=L%20Zhao&volume=9&publication_year=2020&pages=1123-30&pmid=32475230&doi=10.1080/22221751.2020.1770129&)]
* 60.Huang C, Wang Y, Li X, Ren L, Zhao J, Hu Y, et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. lancet. 2020;395:497–506. doi: 10.1016/S0140-6736(20)30183-5. [[DOI](https://doi.org/10.1016/S0140-6736%2820%2930183-5)] [[PMC free article](/articles/PMC7159299/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31986264/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=lancet&title=Clinical%20features%20of%20patients%20infected%20with%202019%20novel%20coronavirus%20in%20Wuhan,%20China&author=C%20Huang&author=Y%20Wang&author=X%20Li&author=L%20Ren&author=J%20Zhao&volume=395&publication_year=2020&pages=497-506&pmid=31986264&doi=10.1016/S0140-6736(20)30183-5&)]
* 61.Liu K, Fang YY, Deng Y, Liu W, Wang MF, Ma JP, et al. Clinical characteristics of novel coronavirus cases in tertiary hospitals in Hubei Province. Chin Med J. 2020;133:1025–31. doi: 10.1097/CM9.0000000000000744. [[DOI](https://doi.org/10.1097/CM9.0000000000000744)] [[PMC free article](/articles/PMC7147277/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32044814/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chin%20Med%20J&title=Clinical%20characteristics%20of%20novel%20coronavirus%20cases%20in%20tertiary%20hospitals%20in%20Hubei%20Province&author=K%20Liu&author=YY%20Fang&author=Y%20Deng&author=W%20Liu&author=MF%20Wang&volume=133&publication_year=2020&pages=1025-31&pmid=32044814&doi=10.1097/CM9.0000000000000744&)]
* 62.Azarkish M, Laleh Far V, Eslami M, Mollazadeh R. Transient complete heart block in a patient with critical COVID-19. Eur Heart J. 2020;41:2131. doi: 10.1093/eurheartj/ehaa307. [[DOI](https://doi.org/10.1093/eurheartj/ehaa307)] [[PMC free article](/articles/PMC7184360/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32285920/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Heart%20J&title=Transient%20complete%20heart%20block%20in%20a%20patient%20with%20critical%20COVID-19&author=M%20Azarkish&author=V%20Laleh%20Far&author=M%20Eslami&author=R%20Mollazadeh&volume=41&publication_year=2020&pages=2131&pmid=32285920&doi=10.1093/eurheartj/ehaa307&)]
* 63.Chen G, Wu DI, Guo W, Cao Y, Huang D, Wang H, et al. Clinical and immunological features of severe and moderate coronavirus disease 2019. J Clin Investig. 2020;130:2620–9. doi: 10.1172/JCI137244. [[DOI](https://doi.org/10.1172/JCI137244)] [[PMC free article](/articles/PMC7190990/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32217835/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Investig&title=Clinical%20and%20immunological%20features%20of%20severe%20and%20moderate%20coronavirus%20disease%202019&author=G%20Chen&author=DI%20Wu&author=W%20Guo&author=Y%20Cao&author=D%20Huang&volume=130&publication_year=2020&pages=2620-9&pmid=32217835&doi=10.1172/JCI137244&)]
* 64.Pons S, Arnaud M, Loiselle M, Arrii E, Azoulay E, Zafrani L. Immune consequences of endothelial cells’ activation and dysfunction during sepsis. Crit Care Clin. 2020;36:401–13. doi: 10.1016/j.ccc.2019.12.001. [[DOI](https://doi.org/10.1016/j.ccc.2019.12.001)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32172821/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Care%20Clin&title=Immune%20consequences%20of%20endothelial%20cells%E2%80%99%20activation%20and%20dysfunction%20during%20sepsis&author=S%20Pons&author=M%20Arnaud&author=M%20Loiselle&author=E%20Arrii&author=E%20Azoulay&volume=36&publication_year=2020&pages=401-13&pmid=32172821&doi=10.1016/j.ccc.2019.12.001&)]
* 65.Yao XH, Luo T, Shi Y, He ZC, Tang R, Zhang PP, et al. A cohort autopsy study defines COVID-19 systemic pathogenesis. Cell Res. 2021;31:836–46. doi: 10.1038/s41422-021-00523-8. [[DOI](https://doi.org/10.1038/s41422-021-00523-8)] [[PMC free article](/articles/PMC8208380/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34135479/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=A%20cohort%20autopsy%20study%20defines%20COVID-19%20systemic%20pathogenesis&author=XH%20Yao&author=T%20Luo&author=Y%20Shi&author=ZC%20He&author=R%20Tang&volume=31&publication_year=2021&pages=836-46&pmid=34135479&doi=10.1038/s41422-021-00523-8&)]
* 66.Nobari NN, Montazer F, Seirafianpour F, Nikkhah F, Aryanian Z, Goodarzi A. Histopathologic changes and cellular events of organs systems in COVID-19. J Cell Mol Anesth. 2021;6:81–8. [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Mol%20Anesth&title=Histopathologic%20changes%20and%20cellular%20events%20of%20organs%20systems%20in%20COVID-19&author=NN%20Nobari&author=F%20Montazer&author=F%20Seirafianpour&author=F%20Nikkhah&author=Z%20Aryanian&volume=6&publication_year=2021&pages=81-8&)]
* 67.Shah MD, Sumeh AS, Sheraz M, Kavitha MS, Maran BA, Rodrigues KF. A mini-review on the impact of COVID 19 on vital organs. Biomed Pharmacother. 2021;143:112158. doi: 10.1016/j.biopha.2021.112158. [[DOI](https://doi.org/10.1016/j.biopha.2021.112158)] [[PMC free article](/articles/PMC8416601/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34507116/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Pharmacother&title=A%20mini-review%20on%20the%20impact%20of%20COVID%2019%20on%20vital%20organs&author=MD%20Shah&author=AS%20Sumeh&author=M%20Sheraz&author=MS%20Kavitha&author=BA%20Maran&volume=143&publication_year=2021&pages=112158&pmid=34507116&doi=10.1016/j.biopha.2021.112158&)]
* 68.Stein SR, Ramelli SC, Grazioli A, Chung JY, Singh M, Yinda CK, et al. SARS-CoV-2 infection and persistence in the human body and brain at autopsy. Nature. 2022;612:758–63. doi: 10.1038/s41586-022-05542-y. [[DOI](https://doi.org/10.1038/s41586-022-05542-y)] [[PMC free article](/articles/PMC9749650/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/36517603/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=SARS-CoV-2%20infection%20and%20persistence%20in%20the%20human%20body%20and%20brain%20at%20autopsy&author=SR%20Stein&author=SC%20Ramelli&author=A%20Grazioli&author=JY%20Chung&author=M%20Singh&volume=612&publication_year=2022&pages=758-63&pmid=36517603&doi=10.1038/s41586-022-05542-y&)]
* 69.Stafstrom CE. Neurological effects of COVID‐19 in infants and children. Dev Med Child Neurol. 2022;64:818–29. doi: 10.1111/dmcn.15185. [[DOI](https://doi.org/10.1111/dmcn.15185)] [[PMC free article](/articles/PMC9111795/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35243616/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dev%20Med%20Child%20Neurol&title=Neurological%20effects%20of%20COVID%E2%80%9019%20in%20infants%20and%20children&author=CE%20Stafstrom&volume=64&publication_year=2022&pages=818-29&pmid=35243616&doi=10.1111/dmcn.15185&)]
* 70.Crunfli F, Carregari VC, Veras FP, Silva LS, Nogueira MH, Antunes AS, et al. Morphological, cellular, and molecular basis of brain infection in COVID-19 patients. Proc Natl Acad Sci USA. 2022;119:e2200960119. doi: 10.1073/pnas.2200960119. [[DOI](https://doi.org/10.1073/pnas.2200960119)] [[PMC free article](/articles/PMC9436354/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35951647/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Morphological,%20cellular,%20and%20molecular%20basis%20of%20brain%20infection%20in%20COVID-19%20patients&author=F%20Crunfli&author=VC%20Carregari&author=FP%20Veras&author=LS%20Silva&author=MH%20Nogueira&volume=119&publication_year=2022&pages=e2200960119&pmid=35951647&doi=10.1073/pnas.2200960119&)]
* 71.Zhang L, Zhou L, Bao L, Liu J, Zhu H, Lv Q, et al. SARS-CoV-2 crosses the blood–brain barrier accompanied with basement membrane disruption without tight junctions alteration. Sig Transduct Target Ther. 2021;6:337. doi: 10.1038/s41392-021-00719-9. [[DOI](https://doi.org/10.1038/s41392-021-00719-9)] [[PMC free article](/articles/PMC8419672/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34489403/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sig%20Transduct%20Target%20Ther&title=SARS-CoV-2%20crosses%20the%20blood%E2%80%93brain%20barrier%20accompanied%20with%20basement%20membrane%20disruption%20without%20tight%20junctions%20alteration&author=L%20Zhang&author=L%20Zhou&author=L%20Bao&author=J%20Liu&author=H%20Zhu&volume=6&publication_year=2021&pages=337&pmid=34489403&doi=10.1038/s41392-021-00719-9&)]
* 72.Livanos AE, Jha D, Cossarini F, Gonzalez-Reiche AS, Tokuyama M, Aydillo T, et al. Intestinal host response to SARS-CoV-2 infection and COVID-19 outcomes in patients with gastrointestinal symptoms. Gastroenterology. 2021;160:2435–50. doi: 10.1053/j.gastro.2021.02.056. [[DOI](https://doi.org/10.1053/j.gastro.2021.02.056)] [[PMC free article](/articles/PMC7931673/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33676971/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Intestinal%20host%20response%20to%20SARS-CoV-2%20infection%20and%20COVID-19%20outcomes%20in%20patients%20with%20gastrointestinal%20symptoms&author=AE%20Livanos&author=D%20Jha&author=F%20Cossarini&author=AS%20Gonzalez-Reiche&author=M%20Tokuyama&volume=160&publication_year=2021&pages=2435-50&pmid=33676971&doi=10.1053/j.gastro.2021.02.056&)]
* 73.Nalbandian A, Sehgal K, Gupta A, Madhavan MV, McGroder C, Stevens JS, et al. Post-acute COVID-19 syndrome. Nat Med. 2021;27:601–15. doi: 10.1038/s41591-021-01283-z. [[DOI](https://doi.org/10.1038/s41591-021-01283-z)] [[PMC free article](/articles/PMC8893149/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33753937/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Post-acute%20COVID-19%20syndrome&author=A%20Nalbandian&author=K%20Sehgal&author=A%20Gupta&author=MV%20Madhavan&author=C%20McGroder&volume=27&publication_year=2021&pages=601-15&pmid=33753937&doi=10.1038/s41591-021-01283-z&)]
* 74.Crook H, Raza S, Nowell J, Young M, Edison P Long covid—mechanisms, risk factors, and management. BMJ. 2021;374:n1648. [[DOI](https://doi.org/10.1136/bmj.n1648)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34312178/)]
* 75.de Melo GD, Lazarini F, Levallois S, Hautefort C, Michel V, Larrous F, et al. COVID-19–related anosmia is associated with viral persistence and inflammation in human olfactory epithelium and brain infection in hamsters. Sci Transl Med. 2021;13:eabf8396. doi: 10.1126/scitranslmed.abf8396. [[DOI](https://doi.org/10.1126/scitranslmed.abf8396)] [[PMC free article](/articles/PMC8158965/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33941622/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&title=COVID-19%E2%80%93related%20anosmia%20is%20associated%20with%20viral%20persistence%20and%20inflammation%20in%20human%20olfactory%20epithelium%20and%20brain%20infection%20in%20hamsters&author=GD%20de%20Melo&author=F%20Lazarini&author=S%20Levallois&author=C%20Hautefort&author=V%20Michel&volume=13&publication_year=2021&pages=eabf8396&pmid=33941622&doi=10.1126/scitranslmed.abf8396&)]
* 76.Pretorius E, Venter C, Laubscher GJ, Kotze MJ, Oladejo SO, Watson LR, et al. Prevalence of symptoms, comorbidities, fibrin amyloid microclots and platelet pathology in individuals with Long COVID/Post-Acute Sequelae of COVID-19 (PASC) Cardiovasc Diabetol. 2022;21:148. doi: 10.1186/s12933-022-01579-5. [[DOI](https://doi.org/10.1186/s12933-022-01579-5)] [[PMC free article](/articles/PMC9356426/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35933347/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Diabetol&title=Prevalence%20of%20symptoms,%20comorbidities,%20fibrin%20amyloid%20microclots%20and%20platelet%20pathology%20in%20individuals%20with%20Long%20COVID/Post-Acute%20Sequelae%20of%20COVID-19%20(PASC)&author=E%20Pretorius&author=C%20Venter&author=GJ%20Laubscher&author=MJ%20Kotze&author=SO%20Oladejo&volume=21&publication_year=2022&pages=148&pmid=35933347&doi=10.1186/s12933-022-01579-5&)]
* 77.Castanares-Zapatero D, Chalon P, Kohn L, Dauvrin M, Detollenaere J, Maertens de Noordhout C, et al. Pathophysiology and mechanism of long COVID: a comprehensive review. Ann Med. 2022;54:1473–87. doi: 10.1080/07853890.2022.2076901. [[DOI](https://doi.org/10.1080/07853890.2022.2076901)] [[PMC free article](/articles/PMC9132392/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35594336/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Med&title=Pathophysiology%20and%20mechanism%20of%20long%20COVID:%20a%20comprehensive%20review&author=D%20Castanares-Zapatero&author=P%20Chalon&author=L%20Kohn&author=M%20Dauvrin&author=J%20Detollenaere&volume=54&publication_year=2022&pages=1473-87&pmid=35594336&doi=10.1080/07853890.2022.2076901&)]
* 78.Vaira LA, Hopkins C, Sandison A, Manca A, Machouchas N, Turilli D, et al. Olfactory epithelium histopathological findings in long-term coronavirus disease 2019 related anosmia. J Laryngol Otol. 2020;134:1123–7. doi: 10.1017/S0022215120002455. [[DOI](https://doi.org/10.1017/S0022215120002455)] [[PMC free article](/articles/PMC7729153/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33190655/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Laryngol%20Otol&title=Olfactory%20epithelium%20histopathological%20findings%20in%20long-term%20coronavirus%20disease%202019%20related%20anosmia&author=LA%20Vaira&author=C%20Hopkins&author=A%20Sandison&author=A%20Manca&author=N%20Machouchas&volume=134&publication_year=2020&pages=1123-7&pmid=33190655&doi=10.1017/S0022215120002455&)]
* 79.Guedj E, Campion JY, Dudouet P, Kaphan E, Bregeon F, Tissot-Dupont H, et al. 18F-FDG brain PET hypometabolism in patients with long COVID. Eur J Nucl Med Mol Imaging. 2021;48:2823–33. doi: 10.1007/s00259-021-05215-4. [[DOI](https://doi.org/10.1007/s00259-021-05215-4)] [[PMC free article](/articles/PMC7837643/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33501506/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Nucl%20Med%20Mol%20Imaging&title=18F-FDG%20brain%20PET%20hypometabolism%20in%20patients%20with%20long%20COVID&author=E%20Guedj&author=JY%20Campion&author=P%20Dudouet&author=E%20Kaphan&author=F%20Bregeon&volume=48&publication_year=2021&pages=2823-33&pmid=33501506&doi=10.1007/s00259-021-05215-4&)]
* 80.Wood E, Hall KH, Tate W. Role of mitochondria, oxidative stress and the response to antioxidants in myalgic encephalomyelitis/chronic fatigue syndrome: a possible approach to SARS-CoV-2 ‘long-haulers’? Chronic Dis Transl Med. 2021;7:14–26. doi: 10.1016/j.cdtm.2020.11.002. [[DOI](https://doi.org/10.1016/j.cdtm.2020.11.002)] [[PMC free article](/articles/PMC7680046/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33251031/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chronic%20Dis%20Transl%20Med&title=Role%20of%20mitochondria,%20oxidative%20stress%20and%20the%20response%20to%20antioxidants%20in%20myalgic%20encephalomyelitis/chronic%20fatigue%20syndrome:%20a%20possible%20approach%20to%20SARS-CoV-2%20%E2%80%98long-haulers%E2%80%99?&author=E%20Wood&author=KH%20Hall&author=W%20Tate&volume=7&publication_year=2021&pages=14-26&pmid=33251031&doi=10.1016/j.cdtm.2020.11.002&)]
* 81.Korompoki E, Gavriatopoulou M, Hicklen RS, Ntanasis-Stathopoulos I, Kastritis E, Fotiou D, et al. Epidemiology and organ specific sequelae of post-acute COVID19: a narrative review. J Infect. 2021;83:1–6. doi: 10.1016/j.jinf.2021.05.004. [[DOI](https://doi.org/10.1016/j.jinf.2021.05.004)] [[PMC free article](/articles/PMC8118709/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33992686/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect&title=Epidemiology%20and%20organ%20specific%20sequelae%20of%20post-acute%20COVID19:%20a%20narrative%20review&author=E%20Korompoki&author=M%20Gavriatopoulou&author=RS%20Hicklen&author=I%20Ntanasis-Stathopoulos&author=E%20Kastritis&volume=83&publication_year=2021&pages=1-6&pmid=33992686&doi=10.1016/j.jinf.2021.05.004&)]
* 82.Phetsouphanh C, Darley DR, Wilson DB, Howe A, Munier CM, Patel SK, et al. Immunological dysfunction persists for 8 months following initial mild-to-moderate SARS-CoV-2 infection. Nat Immunol. 2022;23:210–6. doi: 10.1038/s41590-021-01113-x. [[DOI](https://doi.org/10.1038/s41590-021-01113-x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35027728/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&title=Immunological%20dysfunction%20persists%20for%208%20months%20following%20initial%20mild-to-moderate%20SARS-CoV-2%20infection&author=C%20Phetsouphanh&author=DR%20Darley&author=DB%20Wilson&author=A%20Howe&author=CM%20Munier&volume=23&publication_year=2022&pages=210-6&pmid=35027728&doi=10.1038/s41590-021-01113-x&)]