<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6386654/"/><title>Symbiotic organs shaped by distinct modes of genome evolution in cephalopods - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Proc Natl Acad Sci U S A</button></div>. 2019 Jan 11;116(8):3030–3035. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1817322116" rel="noopener noreferrer" target="_blank">10.1073/pnas.1817322116</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Symbiotic organs shaped by distinct modes of genome evolution in cephalopods</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Belcaid%20M%22%5BAuthor%5D"><span class="name western">Mahdi Belcaid</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mahdi Belcaid</span></h3>
<div class="p">
<sup>a</sup>Pacific Biosciences Research Center, University of Hawaii, Honolulu, HI 96822;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Belcaid%20M%22%5BAuthor%5D"><span class="name western">Mahdi Belcaid</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Casaburi%20G%22%5BAuthor%5D"><span class="name western">Giorgio Casaburi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Giorgio Casaburi</span></h3>
<div class="p">
<sup>b</sup>Department of Microbiology and Cell Science, Space Life Science Lab, University of Florida, Merritt Island, FL 32953;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Casaburi%20G%22%5BAuthor%5D"><span class="name western">Giorgio Casaburi</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McAnulty%20SJ%22%5BAuthor%5D"><span class="name western">Sarah J McAnulty</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Sarah J McAnulty</span></h3>
<div class="p">
<sup>c</sup>Department of Molecular and Cell Biology, University of Connecticut, Storrs, CT 06269;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McAnulty%20SJ%22%5BAuthor%5D"><span class="name western">Sarah J McAnulty</span></a>
</div>
</div>
<sup>c</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schmidbaur%20H%22%5BAuthor%5D"><span class="name western">Hannah Schmidbaur</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Hannah Schmidbaur</span></h3>
<div class="p">
<sup>d</sup>Department of Molecular Evolution and Development, University of Vienna, 1090 Vienna, Austria;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schmidbaur%20H%22%5BAuthor%5D"><span class="name western">Hannah Schmidbaur</span></a>
</div>
</div>
<sup>d</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Suria%20AM%22%5BAuthor%5D"><span class="name western">Andrea M Suria</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Andrea M Suria</span></h3>
<div class="p">
<sup>c</sup>Department of Molecular and Cell Biology, University of Connecticut, Storrs, CT 06269;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Suria%20AM%22%5BAuthor%5D"><span class="name western">Andrea M Suria</span></a>
</div>
</div>
<sup>c</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moriano-Gutierrez%20S%22%5BAuthor%5D"><span class="name western">Silvia Moriano-Gutierrez</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Silvia Moriano-Gutierrez</span></h3>
<div class="p">
<sup>a</sup>Pacific Biosciences Research Center, University of Hawaii, Honolulu, HI 96822;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moriano-Gutierrez%20S%22%5BAuthor%5D"><span class="name western">Silvia Moriano-Gutierrez</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pankey%20MS%22%5BAuthor%5D"><span class="name western">M Sabrina Pankey</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">M Sabrina Pankey</span></h3>
<div class="p">
<sup>e</sup>Ecology, Evolution, and Marine Biology Department, University of California, Santa Barbara, CA 93106;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pankey%20MS%22%5BAuthor%5D"><span class="name western">M Sabrina Pankey</span></a>
</div>
</div>
<sup>e</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Oakley%20TH%22%5BAuthor%5D"><span class="name western">Todd H Oakley</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Todd H Oakley</span></h3>
<div class="p">
<sup>e</sup>Ecology, Evolution, and Marine Biology Department, University of California, Santa Barbara, CA 93106;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Oakley%20TH%22%5BAuthor%5D"><span class="name western">Todd H Oakley</span></a>
</div>
</div>
<sup>e</sup>, <a aria-describedby="id9" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kremer%20N%22%5BAuthor%5D"><span class="name western">Natacha Kremer</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Natacha Kremer</span></h3>
<div class="p">
<sup>f</sup>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon, Claude Bernard University Lyon 1, CNRS, UMR 5558, 69100 Villeurbanne, France;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kremer%20N%22%5BAuthor%5D"><span class="name western">Natacha Kremer</span></a>
</div>
</div>
<sup>f</sup>, <a aria-describedby="id10" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koch%20EJ%22%5BAuthor%5D"><span class="name western">Eric J Koch</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Eric J Koch</span></h3>
<div class="p">
<sup>a</sup>Pacific Biosciences Research Center, University of Hawaii, Honolulu, HI 96822;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koch%20EJ%22%5BAuthor%5D"><span class="name western">Eric J Koch</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id11" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Collins%20AJ%22%5BAuthor%5D"><span class="name western">Andrew J Collins</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Andrew J Collins</span></h3>
<div class="p">
<sup>c</sup>Department of Molecular and Cell Biology, University of Connecticut, Storrs, CT 06269;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Collins%20AJ%22%5BAuthor%5D"><span class="name western">Andrew J Collins</span></a>
</div>
</div>
<sup>c</sup>, <a aria-describedby="id12" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20H%22%5BAuthor%5D"><span class="name western">Hoan Nguyen</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Hoan Nguyen</span></h3>
<div class="p">
<sup>g</sup>Jackson Laboratory for Genomic Medicine, Farmington, CT 06032;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20H%22%5BAuthor%5D"><span class="name western">Hoan Nguyen</span></a>
</div>
</div>
<sup>g</sup>, <a aria-describedby="id13" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lek%20S%22%5BAuthor%5D"><span class="name western">Sai Lek</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Sai Lek</span></h3>
<div class="p">
<sup>g</sup>Jackson Laboratory for Genomic Medicine, Farmington, CT 06032;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lek%20S%22%5BAuthor%5D"><span class="name western">Sai Lek</span></a>
</div>
</div>
<sup>g</sup>, <a aria-describedby="id14" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goncharenko-Foster%20I%22%5BAuthor%5D"><span class="name western">Irina Goncharenko-Foster</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Irina Goncharenko-Foster</span></h3>
<div class="p">
<sup>b</sup>Department of Microbiology and Cell Science, Space Life Science Lab, University of Florida, Merritt Island, FL 32953;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goncharenko-Foster%20I%22%5BAuthor%5D"><span class="name western">Irina Goncharenko-Foster</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id15" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Minx%20P%22%5BAuthor%5D"><span class="name western">Patrick Minx</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Patrick Minx</span></h3>
<div class="p">
<sup>h</sup>McDonnell Genome Institute, Washington University, St. Louis, MO 63108;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Minx%20P%22%5BAuthor%5D"><span class="name western">Patrick Minx</span></a>
</div>
</div>
<sup>h</sup>, <a aria-describedby="id16" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sodergren%20E%22%5BAuthor%5D"><span class="name western">Erica Sodergren</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Erica Sodergren</span></h3>
<div class="p">
<sup>g</sup>Jackson Laboratory for Genomic Medicine, Farmington, CT 06032;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sodergren%20E%22%5BAuthor%5D"><span class="name western">Erica Sodergren</span></a>
</div>
</div>
<sup>g</sup>, <a aria-describedby="id17" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weinstock%20G%22%5BAuthor%5D"><span class="name western">George Weinstock</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">George Weinstock</span></h3>
<div class="p">
<sup>g</sup>Jackson Laboratory for Genomic Medicine, Farmington, CT 06032;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weinstock%20G%22%5BAuthor%5D"><span class="name western">George Weinstock</span></a>
</div>
</div>
<sup>g</sup>, <a aria-describedby="id18" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rokhsar%20DS%22%5BAuthor%5D"><span class="name western">Daniel S Rokhsar</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Daniel S Rokhsar</span></h3>
<div class="p">
<sup>i</sup>Molecular Genetics Unit, Okinawa Institute of Science and Technology, Okinawa 904-0495, Japan;</div>
<div class="p">
<sup>j</sup>Department of Molecular and Cell Biology, University of California, Berkeley, CA 94720;</div>
<div class="p">
<sup>k</sup>Department of Energy, Joint Genome Institute, Walnut Creek, CA 94598</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rokhsar%20DS%22%5BAuthor%5D"><span class="name western">Daniel S Rokhsar</span></a>
</div>
</div>
<sup>i,</sup><sup>j,</sup><sup>k</sup>, <a aria-describedby="id19" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McFall-Ngai%20M%22%5BAuthor%5D"><span class="name western">Margaret McFall-Ngai</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Margaret McFall-Ngai</span></h3>
<div class="p">
<sup>a</sup>Pacific Biosciences Research Center, University of Hawaii, Honolulu, HI 96822;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McFall-Ngai%20M%22%5BAuthor%5D"><span class="name western">Margaret McFall-Ngai</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id20" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Simakov%20O%22%5BAuthor%5D"><span class="name western">Oleg Simakov</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Oleg Simakov</span></h3>
<div class="p">
<sup>d</sup>Department of Molecular Evolution and Development, University of Vienna, 1090 Vienna, Austria;</div>
<div class="p">
<sup>i</sup>Molecular Genetics Unit, Okinawa Institute of Science and Technology, Okinawa 904-0495, Japan;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Simakov%20O%22%5BAuthor%5D"><span class="name western">Oleg Simakov</span></a>
</div>
</div>
<sup>d,</sup><sup>i,</sup><sup>1</sup>, <a aria-describedby="id21" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Foster%20JS%22%5BAuthor%5D"><span class="name western">Jamie S Foster</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Jamie S Foster</span></h3>
<div class="p">
<sup>b</sup>Department of Microbiology and Cell Science, Space Life Science Lab, University of Florida, Merritt Island, FL 32953;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Foster%20JS%22%5BAuthor%5D"><span class="name western">Jamie S Foster</span></a>
</div>
</div>
<sup>b,</sup><sup>1</sup>, <a aria-describedby="id22" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nyholm%20SV%22%5BAuthor%5D"><span class="name western">Spencer V Nyholm</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Spencer V Nyholm</span></h3>
<div class="p">
<sup>c</sup>Department of Molecular and Cell Biology, University of Connecticut, Storrs, CT 06269;</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nyholm%20SV%22%5BAuthor%5D"><span class="name western">Spencer V Nyholm</span></a>
</div>
</div>
<sup>c,</sup><sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>a</sup>Pacific Biosciences Research Center, University of Hawaii, Honolulu, HI 96822;</div>
<div id="aff2">
<sup>b</sup>Department of Microbiology and Cell Science, Space Life Science Lab, University of Florida, Merritt Island, FL 32953;</div>
<div id="aff3">
<sup>c</sup>Department of Molecular and Cell Biology, University of Connecticut, Storrs, CT 06269;</div>
<div id="aff4">
<sup>d</sup>Department of Molecular Evolution and Development, University of Vienna, 1090 Vienna, Austria;</div>
<div id="aff5">
<sup>e</sup>Ecology, Evolution, and Marine Biology Department, University of California, Santa Barbara, CA 93106;</div>
<div id="aff6">
<sup>f</sup>Laboratoire de Biométrie et Biologie Evolutive, Université de Lyon, Claude Bernard University Lyon 1, CNRS, UMR 5558, 69100 Villeurbanne, France;</div>
<div id="aff7">
<sup>g</sup>Jackson Laboratory for Genomic Medicine, Farmington, CT 06032;</div>
<div id="aff8">
<sup>h</sup>McDonnell Genome Institute, Washington University, St. Louis, MO 63108;</div>
<div id="aff9">
<sup>i</sup>Molecular Genetics Unit, Okinawa Institute of Science and Technology, Okinawa 904-0495, Japan;</div>
<div id="aff10">
<sup>j</sup>Department of Molecular and Cell Biology, University of California, Berkeley, CA 94720;</div>
<div id="aff11">
<sup>k</sup>Department of Energy, Joint Genome Institute, Walnut Creek, CA 94598</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>1</sup><p class="display-inline">To whom correspondence may be addressed. Email: <span>oleg.simakov@univie.ac.at</span>, <span>jfoster@ufl.edu</span>, or <span>spencer.nyholm@uconn.edu</span>.</p>
</div>
<div class="fn" id="fn1"><p>Edited by Nancy A. Moran, University of Texas at Austin, Austin, TX, and approved December 11, 2018 (received for review October 9, 2018)</p></div>
<div class="fn" id="fn2"><p>Author contributions: M.B., M.M.-N., O.S., J.S.F., and S.V.N. designed research; M.B., G.C., S.J.M., A.M.S., S.M.-G., M.S.P., T.H.O., N.K., E.J.K., A.J.C., H.N., S.L., I.G.-F., P.M., E.S., G.W., D.S.R., O.S., J.S.F., and S.V.N. performed research; S.M.-G., M.S.P., T.H.O., N.K., E.J.K., A.J.C., H.N., S.L., P.M., E.S., G.W., D.S.R., M.M.-N., O.S., J.S.F., and S.V.N. contributed new reagents/analytic tools; M.B., G.C., S.J.M., H.S., A.M.S., H.N., S.L., P.M., E.S., G.W., D.S.R., O.S., J.S.F., and S.V.N. analyzed data; and M.B., G.C., S.J.M., H.S., A.M.S., S.M.-G., M.S.P., T.H.O., N.K., E.J.K., A.J.C., H.N., S.L., I.G.-F., P.M., E.S., G.W., D.S.R., M.M.-N., O.S., J.S.F., and S.V.N. wrote the paper.</p></div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p">
<section class="series-info" id="article-meta1"><h3 class="pmc_sec_title">Series information</h3>
<p>From the Cover</p></section><section class="history" id="historyarticle-meta1"><p>Issue date 2019 Feb 19.</p></section>
</div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2019 the Author(s). Published by PNAS.</div>
<p>This open access article is distributed under <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by-nc-nd/4.0/" rel="noopener noreferrer" target="_blank">Creative Commons Attribution-NonCommercial-NoDerivatives License 4.0 (CC BY-NC-ND)</a>.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC6386654  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30635418/">30635418</a>
</div>
<div aria-label="Related or updated information about this article" class="ra xbox p" role="complementary"><div>See commentary "<a class="usa-link" href="/articles/PMC6386672/">Squid genomes in a bacterial world</a>" on page 2799.</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Significance</h2>
<p>Animal–microbe associations are critical drivers of evolutionary innovation, yet the origin of specialized symbiotic organs remains largely unexplored. We analyzed the genome of <em>Euprymna scolopes</em>, a model cephalopod, and observed large-scale genomic reorganizations compared with the ancestral bilaterian genome. We report distinct evolutionary signatures within the two symbiotic organs of <em>E. scolopes</em>, the light organ (LO) and the accessory nidamental gland (ANG). The LO evolved through subfunctionalization of genes expressed in the eye, indicating a deep evolutionary link between these organs. Alternatively, the ANG was enriched in novel, species-specific orphan genes suggesting these two tissues originated via different evolutionary strategies. These analyses represent the first genomic insights into the evolution of multiple symbiotic organs within a single animal host.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> cephalopods, symbiosis, evolution, genomics, transcriptomics</p></section></section><section class="abstract" id="abstract2"><h2>Abstract</h2>
<p>Microbes have been critical drivers of evolutionary innovation in animals. To understand the processes that influence the origin of specialized symbiotic organs, we report the sequencing and analysis of the genome of <em>Euprymna scolopes</em>, a model cephalopod with richly characterized host–microbe interactions. We identified large-scale genomic reorganization shared between <em>E. scolopes</em> and <em>Octopus bimaculoides</em> and posit that this reorganization has contributed to the evolution of cephalopod complexity. To reveal genomic signatures of host–symbiont interactions, we focused on two specialized organs of <em>E. scolopes</em>: the light organ, which harbors a monoculture of <em>Vibrio fischeri</em>, and the accessory nidamental gland (ANG), a reproductive organ containing a bacterial consortium. Our findings suggest that the two symbiotic organs within <em>E. scolopes</em> originated by different evolutionary mechanisms. Transcripts expressed in these microbe-associated tissues displayed their own unique signatures in both coding sequences and the surrounding regulatory regions. Compared with other tissues, the light organ showed an abundance of genes associated with immunity and mediating light, whereas the ANG was enriched in orphan genes known only from <em>E. scolopes</em>. Together, these analyses provide evidence for different patterns of genomic evolution of symbiotic organs within a single host.</p></section><hr class="headless"/>
<p>Numerous organisms have specialized organs to house their microbiota, yet the evolutionary processes underlying the origin of these tissues are not well understood (<a aria-describedby="r1" class="usa-link" href="#r1">1</a>–<a aria-describedby="r4" class="usa-link" href="#r4">4</a>). The Hawaiian bobtail squid, <em>Euprymna scolopes</em>, is a tractable animal model that has proven invaluable for interrogating symbiotic relationships and revealing common mechanisms by which successful colonization of animal epithelia by bacteria is established and maintained (<a class="usa-link" href="#fig01">Fig. 1<em>A</em></a>) (<a aria-describedby="r5" class="usa-link" href="#r5">5</a>). In the light organ (LO) of <em>E. scolopes</em>, a monospecific association with the bioluminescent bacterium <em>Vibrio fischeri</em> provides camouflage for the host and triggers a coordinated developmental remodeling of the surrounding epithelial tissues (<a aria-describedby="r6" class="usa-link" href="#r6">6</a>). Additionally, <em>E. scolopes</em> harbors a complex microbial consortium in the accessory nidamental gland (ANG), a component of the female reproductive system of many squids and cuttlefish that is hypothesized to play a role in egg defense (<a class="usa-link" href="#fig01">Fig. 1<em>B</em></a>) (<a aria-describedby="r7" class="usa-link" href="#r7">7</a>, <a aria-describedby="r8" class="usa-link" href="#r8">8</a>). <em>Octopus bimaculoides</em>, the other cephalopod for which a genome has been reported to date, has neither a light organ nor an ANG making it an ideal organism for comparison (<a class="usa-link" href="#fig01">Fig. 1<em>C</em></a>). Therefore, to define elements of genome structure and function that are critical for symbiosis and the evolution of cephalopods, we sequenced and characterized the <em>E. scolopes</em> genome, the first example from the superorder Decapodiformes.</p>
<figure class="fig xbox font-sm" id="fig01"><h2 class="obj_head">Fig. 1.</h2>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6386654_pnas.1817322116fig01.jpg" target="_blank"><img alt="Fig. 1." class="graphic zoom-in" height="916" id="gra1" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ff6c/6386654/b5217630a0b2/pnas.1817322116fig01.jpg" width="796"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig01/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>The Hawaiian bobtail squid, <em>Euprymna scolopes</em>, a model host for microbiome research and cephalopod innovations. (<em>A</em>) The animal, shown here in the water column, is a nocturnal predator that uses the luminescence of the LO symbiont <em>Vibrio fischeri</em> for camouflage. Scale bar, 3 cm. Image courtesy of Elizabeth Ellenwood (photographer). (<em>B</em>) Overview of key symbiotic and nonsymbiotic organs within <em>E. scolopes</em>. Whereas males only have an LO symbiosis, the females have the additional symbiosis of the ANG, a reproductive organ housing a consortium of bacteria from predominantly two phyla (<a aria-describedby="r8" class="usa-link" href="#r8">8</a>). In addition to these symbiotic tissues, gene expression was also analyzed from the gills, brain, eye, and skin. (<em>C</em>) Distribution of LOs and ANGs that occur only in coleoid cephalopods. Branch lengths are derived from Tanner et al. (<a aria-describedby="r35" class="usa-link" href="#r35">35</a>).</p></figcaption></figure><section id="s1"><h2 class="pmc_sec_title">Results and Discussion</h2>
<p>Our sequencing efforts have resulted in the most comprehensive cephalopod genome assembly to date, revealing the expansive and highly repetitive nature of the <em>E. scolopes</em> genome. Using a hybrid approach of shotgun and long-range linkage sequencing methods, an assembly estimated to be 5.1 Gigabases (Gb) was generated with a total half of the assembly in scaffolds of 3.7 Mb or longer (N50). The genome annotation was guided by 31 RNA-seq (Illumina) and ISO-seq (PacBio) transcriptomic libraries prepared from diverse tissues and developmental stages (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Table S1</a>). The annotation resulted in 29,259 expressed protein-coding genes that yielded a BUSCO (Benchmarking Universal Single-Copy Orthologs) (<a aria-describedby="r9" class="usa-link" href="#r9">9</a>) score of 97% against the eukaryotic core set (<a class="usa-link" href="#s3"><em>Materials and Methods</em></a>). A table summarizing the key statistics and a comparison with the <em>O. bimaculoides</em> genome is provided in <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Table S2</a>.</p>
<p>Genome analysis revealed differing histories of repeat element expansions between the bobtail squid and octopus, which may have contributed to the observed divergence in genome size (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Fig. S1</a>). Although the estimated proportion of repetitive elements was &gt;50% in both species, the abundance within individual repetitive element classes was strikingly different across the two species. Specifically, the abundance of long and short interspersed nuclear elements (LINEs and SINEs, respectively) differed; LINEs were the most abundant repeat class in <em>E. scolopes</em>, whereas SINEs dominated in <em>O. bimaculoides</em> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Fig. S1<em>B</em></a>). Gene families in <em>E. scolopes</em> are similar to those reported in <em>O. bimaculoides</em> and also include tandem expansions in protocadherins and zinc finger transcription factors (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Fig. S2</a>) (<a aria-describedby="r10" class="usa-link" href="#r10">10</a>). We infer that the last common coleoid cephalopod ancestor had 7,650 gene families, similar to the previously reported estimates for spiralian and bilaterian ancestors (<a aria-describedby="r10" class="usa-link" href="#r10">10</a>, <a aria-describedby="r11" class="usa-link" href="#r11">11</a>). Because we do not find signatures of whole-genome duplication or evidence of extensive horizontal gene transfer in <em>E. scolopes</em>, the difference in the genome sizes between octopus and bobtail squid can be attributed to the increased LINE content in the <em>E. scolopes</em> genome. Future sequencing of other Decapodiformes genomes should enable a more exact estimate of this repeat element expansion.</p>
<p>The <em>E. scolopes</em> genome revealed key genomic transitions toward cephalopod genomic architecture. We find that over half of the local gene linkages (i.e., microsynteny) conserved across many noncephalopod bilaterian genomes were disrupted in both <em>O. bimaculoides</em> and <em>E. scolopes</em>, indicating that a large genomic reorganization took place in the cephalopod ancestor (<a class="usa-link" href="#fig02">Fig. 2<em>A</em></a>). Additionally, we found numerous linkages shared between octopus and the bobtail squid that have not been previously identified in other animal genomes (<a class="usa-link" href="#fig02">Fig. 2 <em>B</em> and <em>C</em></a>). Those linkages contain genes expressed in highly developed organ systems (e.g., the central nervous system) as well as in symbiotic organs from <em>E. scolopes</em> and testes of <em>O. bimacuolides</em> (<a class="usa-link" href="#fig02">Fig. 2<em>C</em></a>). In total, this taxon-specific microsynteny constituted 67% (126 out of 189) of all conserved syntenic linkages in both cephalopod genomes (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Fig. S3 and Table S2</a>).</p>
<figure class="fig xbox font-sm" id="fig02"><h3 class="obj_head">Fig. 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6386654_pnas.1817322116fig02.jpg" target="_blank"><img alt="Fig. 2." class="graphic zoom-in" height="925" id="gra2" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ff6c/6386654/60c422b8999c/pnas.1817322116fig02.jpg" width="800"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig02/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Establishment of coleoid cephalopod genome architecture. (<em>A</em>) High rate of genome reorganization at the base of the coleoid cephalopods, as measured by the cumulative amount of microsyntenies lost and gained. Branch length estimation using MrBayes (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em></a>) on a fixed tree topology using binary presence or absence matrix of shared orthologous microsyntenic blocks. Squares (black/white) connected by lines above the nodes indicate two hypothetical microsyntenic blocks illustrating a common scenario where conserved bilaterian and lophotrochozoan synteny is disrupted (crossed lines) followed by the emergence of a new order through rearrangement in the cephalopod stem lineage. (<em>B</em>) Prevalence of the unique cephalopod microsynteny (green shaded area) in <em>O. bimaculoides</em> and <em>E. scolopes</em> genomes. Total length of arches for individual species corresponds to the number of genes in microsyntenies. (<em>C</em>) Heat map of cephalopod unique microsyntenic clusters showing both neuronal (<em>Upper</em>) and broader gene expression (<em>Lower</em>) in <em>E. scolopes</em> and <em>O. bimaculoides</em>. Color bar indicates relative normalized gene expression level. Individual orthologous genes between <em>E. scolopes</em> and <em>O. bimaculoides</em> are connected by solid lines between heat maps with numbers indicating gene order along the scaffold. ANC, axial nerve cord; OL, optic lobe; Psg, posterior salivary gland; Supra/Sub, supraesophageal and subesophageal brain. (<em>D</em>) A large partial <em>hox</em> cluster on two separate scaffolds was recovered, totaling to a length of at least 16 Mb. <em>Branchiostoma floridae hox</em> cluster is shown for comparison with colors indicating orthologous genes.</p></figcaption></figure><p>The improved contiguity of the genome assembly also uncovered the presence of a partial <em>hox</em> cluster in <em>E. scolopes</em> with very large (1.5–2 Mb) intergenic distances that is in stark contrast to the previously reported atomized <em>hox</em> cluster in the octopus genome (<a aria-describedby="r10" class="usa-link" href="#r10">10</a>). These are to our knowledge the longest inter <em>hox</em>-gene separations (<a class="usa-link" href="#fig02">Fig. 2<em>D</em></a>) known in animals and are consistent with the long, gene-free regions surrounding <em>hox</em> genes in <em>O. bimaculoides</em> (<a aria-describedby="r10" class="usa-link" href="#r10">10</a>) and the identified expansion of repetitive elements. Such large intergenic distances may be responsible for the evolution of unique regulatory mechanisms. This hypothesis is further supported by unconventional <em>hox</em> gene expression in <em>E. scolopes</em> (<a aria-describedby="r12" class="usa-link" href="#r12">12</a>).</p>
<p>To understand how this genome reorganization drove the evolution of symbiotic organs in <em>E. scolopes</em>, we studied genomic signatures of genes with tissue-specific expression in symbiotic versus nonsymbiotic organs. To identify possible candidate genes that evolved in symbiotic organs after a duplication event, we searched the <em>E. scolopes</em> genome for all paralogous pairs of genes in which one gene had tissue-specific expression and the second did not. Using these genes, we then calculated the number of synonymous substitutions (dS) between each gene pair (<a class="usa-link" href="#fig03">Fig. 3</a>). We found that for the LO and ANG, the dS distance between tissue-specific genes and the respective closest non-LO and non-ANG paralog was relatively small [dS &lt; 1, ∼130 Mya, using a calibration point from Albertin et al. (<a aria-describedby="r10" class="usa-link" href="#r10">10</a>)]. These distances were in sharp contrast compared with genes specific to the eye and brain, which were much older (dS &gt; 2, squid–octopus divergence, ∼270 Mya) (<a aria-describedby="r10" class="usa-link" href="#r10">10</a>, <a aria-describedby="r13" class="usa-link" href="#r13">13</a>). These paralog ages are consistent with a scenario in which the ANG and LO, which harbor dense populations of bacteria, underwent relatively recent innovations within the bobtail squid lineage through gene duplication, compared with the more ancient duplications of genes that are involved in development and function of cephalopod eyes and the nervous system.</p>
<figure class="fig xbox font-sm" id="fig03"><h3 class="obj_head">Fig. 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6386654_pnas.1817322116fig03.jpg" target="_blank"><img alt="Fig. 3." class="graphic zoom-in" height="835" id="gra3" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ff6c/6386654/74a443be0a72/pnas.1817322116fig03.jpg" width="800"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig03/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Characterization (i.e., functional categories) of key tissues revealing the high contribution of novel genes toward ANG evolution as well as strong similarity between LO and eye transcriptomes. (<em>A</em>) Total counts of unique isoforms across different functional categories in six adult tissues. (<em>B</em>) (<em>Upper</em>) Joy plot of the number of nucleotides within 20 kilobases (kb) windows located up and downstream of the tissue-specific genes from (<em>A</em>) that are attributed to repetitive elements. Regions around ANG genes show higher repeat content compared with other tissues (<em>P</em> &lt; 0.1, Wilcoxon rank sum test). (<em>Lower</em>) Joy plot of the synonymous substitutions distances (dS) between the genes specifically expressed in a given tissue to their closest paralog expressed elsewhere. Distributions from tissues representing cephalopod synapomorphies (brain and eyes) show an older mean (<em>P</em> &lt; 0.1, Wilcoxon rank sum test) compared with the distributions from ANG, LO, and skin tissues. (<em>C</em>) Venn diagram representing the number of shared transcripts among LO, ANG, and eye tissues identifying a significant overlap between LO and eye transcripts.</p></figcaption></figure><p>The main mechanism behind gene duplication in <em>O. bimaculoides</em> is through tandem or segmental duplication (<a aria-describedby="r10" class="usa-link" href="#r10">10</a>). Genes exclusively expressed in either LO, eyes, or ANG were typically found in tandem clusters of paralogs located on single scaffolds. In particular, we found two gene clusters, the first composed of reflectins and the second of peroxidases that were expressed in the LO (see below, <a class="usa-link" href="#fig04">Fig. 4</a>, and <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Figs. S4–S6</a>). In both cases, the tandem gene duplications forming those clusters were <em>E. scolopes</em>-specific and occurred after the octopus–squid split, consistent with the emergence of ANG and LO in the subsequent squid lineage. A large 2-Mb cluster of S-crystallins, which are predominantly expressed in the eyes (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Fig. S7</a>), was expanded in tandem in <em>E. scolopes</em> but was missing from the octopus genome. The omega-crystallins, which are typically found in the LO and the eyes of <em>E. scolopes</em>, on the other hand, did not show signs of any expansion (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Fig. S7</a>). The characteristic cluster in the ANG (∼1 Mb) contained both novel genes and transposable elements. Overall, there were significantly more tandemly duplicated genes expressed in the LO and eyes compared with the number of the tandemly clustered genes specifically expressed in other tissues (Fisher’s exact test <em>P</em> &lt; 0.001), which underscores the importance of gene duplication for those organ systems.</p>
<figure class="fig xbox font-sm" id="fig04"><h3 class="obj_head">Fig. 4.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6386654_pnas.1817322116fig04.jpg" target="_blank"><img alt="Fig. 4." class="graphic zoom-in" height="1337" id="gra4" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ff6c/6386654/2b9f1ddd85e5/pnas.1817322116fig04.jpg" width="800"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig04/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Independent tandem gene cluster formation in squid and octopus and the origin of light organ-specific gene expression. Phylogenetic trees highlight highly specific expansion patterns that correlate with general shared expression between the light organ and the eyes for reflectins (<em>A</em>) or the appearance of the light organ-specific expression pattern for heme peroxidases (<em>B</em>). Heat maps indicate relative normalized expression levels for each tissue (<em>Z</em> scores). Scale bar underneath phylogenetic trees indicates amino acid substitutions per site. Color of the nodes (<em>A</em> and <em>B</em>) identifies genomically colocalized genes (shown in <em>C</em>). Those genes are clustered in tandem on several scaffolds with positions and gene identifiers labeled. (<em>C</em>) Tandem clusters of reflectin and peroxidase genes in the <em>E. scolopes</em> genome. Scaffold ID and approximate location (kbp) is shown for each gene (represented by rectangle). Colors correspond to the sequences on the trees in <em>A</em> for reflectins and (<em>B</em>) for peroxidases.</p></figcaption></figure><p>Our data also reveal that despite the broad phylogenetic distribution among cephalopods (<a aria-describedby="r14" class="usa-link" href="#r14">14</a>), the ANG of <em>E. scolopes</em> is highly derived and shows an elevated evolutionary accumulation in both its coding (novel gene formation) and noncoding (turnover of the regulatory sequence) complements. The ANG is a secretory organ within the female reproductive system containing a bacterial consortium that is deposited into the egg capsule and is believed to play a role in defense from fouling and/or pathogens during embryogenesis (<a aria-describedby="r7" class="usa-link" href="#r7">7</a>, <a aria-describedby="r8" class="usa-link" href="#r8">8</a>, <a aria-describedby="r15" class="usa-link" href="#r15">15</a>, <a aria-describedby="r16" class="usa-link" href="#r16">16</a>). Taxonomically restricted (i.e., orphan) genes have contributed to the evolution of unique tissues and organs in a number of animals (<a aria-describedby="r17" class="usa-link" href="#r17">17</a>–<a aria-describedby="r19" class="usa-link" href="#r19">19</a>). Within the two symbiotic organs of <em>E. scolopes</em> (LO and ANG), we profiled protein-coding transcripts that mapped to genome scaffolds from tissue-specific transcriptomes against the National Center for Biotechnology Information’s nonredundant database [using BLASTP (Basic Local Alignment Search Tool - Protein) e-value threshold of 1E-5; <a class="usa-link" href="#fig03">Fig. 3<em>A</em></a>] and classified the transcripts as either <em>E. scolopes</em>-specific, cephalopod, molluscan, bilaterian, metazoan, or premetazoan (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Fig. S8</a>).</p>
<p>The ANG had the highest proportion of <em>E. scolopes</em>-specific transcripts (&gt;35%) among all tissues. Additionally, regions 20 kb up- and down-stream of genes specifically expressed in the ANG also showed a higher proportion of repetitive element content compared with genes expressed in any other tissue (<a class="usa-link" href="#fig03">Fig. 3<em>B</em></a>), suggesting a high evolutionary turnover in regulatory regions. These higher rates of genomic innovation, specific to the <em>E. scolopes</em> ANG genes, are also supported by our analysis of previously reported ANG transcriptomes from both <em>E. scolopes</em> and the (loliginid) swordtip squid, <em>Uroteuthis edulis</em>, the latter of which had fewer unique genes specific to this organ (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Fig. S8</a>) (<a aria-describedby="r20" class="usa-link" href="#r20">20</a>).</p>
<p>Within the LO of <em>E. scolopes</em>, there were several other genetic signatures contributing to the distinctive features of this symbiotic organ. First, the dominant transcripts within the light organ were reflectins (<a class="usa-link" href="#fig04">Fig. 4<em>A</em></a>), providing strong support for previous studies indicating that the anatomical features used to modulate light in the LO are similar to the eye, as are their physiology, biochemistry, molecular biology, and developmental induction (<a aria-describedby="r21" class="usa-link" href="#r21">21</a>–<a aria-describedby="r23" class="usa-link" href="#r23">23</a>). A comparison of adult LO and eye transcriptomes supports these previous studies (<a class="usa-link" href="#fig03">Figs. 3<em>C</em></a> and <a class="usa-link" href="#fig04">4<em>A</em></a>). Identified reflectins (<a class="usa-link" href="#fig04">Fig. 4<em>A</em></a>) were almost exclusively (&gt;99.99% of total expression) expressed in the LO, eyes, and skin. LO expression constituted around 19.5% of the total reflectin expression, whereas eyes and skin composed 72.8% and 7.6%, respectively. Reflectins usually shared two or more expression domains and were rarely expressed in a single tissue. The majority of genes showed a distinctive LO and eyes expression domain, and no reflectins were expressed exclusively in both LO and skin (<a class="usa-link" href="#fig04">Fig. 4 <em>A</em> and <em>C</em></a>). Evolutionary timing of the LO–eye reflectin clusters may thus hint at the origin of the LO. Molecular dating analysis using branch length and squid–octopus split as a calibration point (<a aria-describedby="r10" class="usa-link" href="#r10">10</a>, <a aria-describedby="r13" class="usa-link" href="#r13">13</a>) revealed the age of those clusters at around 30 Mya or younger. Additionally, among the genomic expansion of 14 peroxidases unique to <em>E. scolopes</em> (<a class="usa-link" href="#fig04">Fig. 4 <em>B</em> and <em>C</em></a> and <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Fig. S6</a>), six genes [including a previously reported halide peroxidase (<a aria-describedby="r24" class="usa-link" href="#r24">24</a>, <a aria-describedby="r25" class="usa-link" href="#r25">25</a>)] formed a single genomic cluster expressed exclusively in the light organ. Unlike reflectins, this expansion did not form a monophyletic group, suggesting high gain or loss of genes within this family, or gene conversion. The host is known to produce a halide peroxidase that is expressed in the LO and generates hypohalous acid, a potent antimicrobial compound thought to contribute to specificity in the squid–vibrio association (<a aria-describedby="r26" class="usa-link" href="#r26">26</a>, <a aria-describedby="r27" class="usa-link" href="#r27">27</a>). This peroxidase, along with several others, is expressed in the tissues directly in contact with <em>V. fischeri</em> (<a aria-describedby="r28" class="usa-link" href="#r28">28</a>, <a aria-describedby="r29" class="usa-link" href="#r29">29</a>). The expansion of the peroxidases and the unique expression of a subset of these genes exclusively in the LO may reflect the development of a unique microenvironment that helps to maintain specificity with <em>V. fischeri</em> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Fig. S6</a>).</p>
<p>Analysis of protein families also revealed a number of genes that had multiple immune-associated domains, many of which were expressed in the LO, ANG, and skin (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Fig. S9</a>). All three of these tissues interact directly with environmental and/or symbiotic microorganisms. A number of cellular and biochemical components of the innate immune system are known to play critical roles in mediating specificity in the LO symbiosis with <em>V. fischeri</em> (<a aria-describedby="r30" class="usa-link" href="#r30">30</a>–<a aria-describedby="r32" class="usa-link" href="#r32">32</a>). Previous observations of host hemocytes infiltrating the ANG (<a aria-describedby="r7" class="usa-link" href="#r7">7</a>), along with expression of immune-related genes reported here, suggest that the host’s immune system plays a critical role in this symbiotic organ as well.</p>
<p>Although bacterial LOs are restricted to just two families of cephalopods (sepiolids and loliginids), the ANG is a more broadly distributed organ, present in these and a number of other squid and cuttlefish species (<a class="usa-link" href="#fig01">Fig. 1<em>C</em></a>). Their distinct evolutionary histories and origins are supported by the different evolutionary patterns detected. The prevalence of functional novelty through gene duplication in the LO compared with taxon-specific genes in the ANG suggests two different patterns of genomic innovation underlying symbiotic organ evolution in <em>E. scolopes</em>. Because these two associations are not nutritionally coupled, the mechanisms for the evolution of these symbiotic organs may be different from what has been proposed for insect bacteriocytes where metabolic complementation appears to be the major driver of innovation (<a aria-describedby="r33" class="usa-link" href="#r33">33</a>). Rapid gene evolution and/or gene loss in ancestral lineages may also have contributed to these unique signatures, especially in the ANG. Together, the presence of these two distinctive tissues within <em>E. scolopes</em> renders it a unique model organism to investigate the genetic mechanisms associated with the evolution of symbiotic organs (<a aria-describedby="r34" class="usa-link" href="#r34">34</a>).</p></section><section id="s2"><h2 class="pmc_sec_title">Conclusions</h2>
<p>Our analyses of the Hawaiian bobtail squid genome revealed large-scale genome reorganization, which preceded the coleoid cephalopod radiation, and provide evidence for two different patterns of genomic evolution that contributed to functional novelty. First, the extensive structural reorganization through the loss of ancient bilaterian microsynteny and increase in genome size through repetitive element expansions resulted in a unique genomic architecture that may have contributed to the innovations in the general cephalopod body plan. Second, the coding and surrounding regulatory regions of microbe-associated tissues suggest distinctive evolutionary patterns, such as the expansion of genes associated with immunity and light production in the LO. In the ANG, the prevalence of taxon-specific, or orphan, gene expression suggests a highly derived organ that evolved by a different means than the LO, perhaps due to unique selection pressures. Overall, our results set the stage for further functional analysis of genomic innovations that led to the evolution of symbiotic organs in a morphologically and behaviorally complex clade.</p></section><section id="s3"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="s4"><h3 class="pmc_sec_title">Data Access.</h3>
<p>Genome and transcriptome sequencing reads have been deposited in the Sequence Read Archive as Bioproject PRJNA470951.</p></section><section id="s5"><h3 class="pmc_sec_title">Genome Sequencing and Assembly.</h3>
<p>Genomic DNA generated in this study was derived from a single adult male <em>E. scolopes</em> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em></a>). Illumina reads from several libraries (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Table S1</a>) were assembled with Meraculous resulting in a preliminary assembly; scaffolds were generated using the in vitro chromatin conformation capture (Chicago) at Dovetail Genomics.</p></section><section id="s6"><h3 class="pmc_sec_title">Gene Model Predictions.</h3>
<p>Gene prediction models were generated using Augustus training with transcriptomic data. The prediction was filtered to exclude sequences that overlap with masked repeats over at least 50% of the lengths. From the remaining models, those with support from transcript evidence and significant homology to metazoan protein sequences were included in the nonredundant database. This gene modeling set was combined with transcriptome mapping (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Table S2</a>) resulting in 29,089 protein sequences that passed the filtering stage.</p></section><section id="s7"><h3 class="pmc_sec_title">Transcriptome Sequencing.</h3>
<p>Thirty-one transcriptomes of <em>E. scolopes</em> were included in the reference transcriptome and sample metadata of the tissues and time points is provided in <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Table S1</a>. Different RNA extractions and sequencing platforms were used and are listed in <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Table S1</a>. For the tissue-specific transcriptomes, RNA was extracted from ANG, brain, eyes, gills, hemocytes, LO, and skin tissues, as well as juvenile head (white body, optic nerve, and brain), eyes, gills, and light organ. For the PacBio IsoSeq library, RNA was extracted, normalized, and pooled from adult brain, eyes, white body, optical lobe, gills, LO, and skin as well as whole juvenile hatchling, 24-h aposymbiotic, and 24-h symbiotic animals (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank"><em>SI Appendix</em>, Table S1</a>).</p></section><section id="s8"><h3 class="pmc_sec_title">Synteny Analysis.</h3>
<p>Using previously published methods of phylogeny-informed clustering, we have constructed the sets of orthologous gene families between the following species (<a aria-describedby="r11" class="usa-link" href="#r11">11</a>): <em>Capitella teleta</em>, <em>Helobdella robusta</em>, <em>Lottia gigantea</em>, <em>Octopus bimaculoides</em>, <em>Euprymna scolopes</em>, <em>Crassostrea gigas</em>, <em>Nematostella vectensis</em>, and <em>Branchiostoma floridae</em>. To account for differential gene loss that may impede our quantification of synteny loss/gain we focused only on 3,547 clusters that have an ortholog from each of the species. We have implemented microsynteny detection algorithm, as described in Simakov et al. (<a aria-describedby="r11" class="usa-link" href="#r11">11</a>), and found, in accordance with previous results (<a aria-describedby="r10" class="usa-link" href="#r10">10</a>, <a aria-describedby="r34" class="usa-link" href="#r34">34</a>), ∼600 microsyntenic blocks (in 163 orthologous groups) that can be traced back to the bilaterian ancestor because they are shared between either both ingroups (protostome and deuterostome) or an ingroup and an outgroup species (<em>Nematostella</em>) (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank">Dataset S1</a>).</p></section></section><section id="sec9"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="supplementary-material1"><div class="caption p"><span>Supplementary File</span></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/6386654/bin/pnas.1817322116.sapp.pdf">pnas.1817322116.sapp.pdf</a><sup> (3.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material2"><div class="caption p"><span>Supplementary File</span></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/6386654/bin/pnas.1817322116.sd01.xlsx">pnas.1817322116.sd01.xlsx</a><sup> (195.6KB, xlsx) </sup>
</div></div></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Bo Reese from the Center for Genome Innovation, University of Connecticut (UCONN), for sequencing assistance and Jill Wegrzyn, Stephen King, and Ion Moraru of the Computational Biology Core, Institute for Systems Genomics, UCONN, for computational support. Computation was conducted at the Vienna Life Sciences Cluster (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://cube.univie.ac.at/" rel="noopener noreferrer" target="_blank">cube.univie.ac.at/</a>) and the computing cluster at the Okinawa Institute of Science and Technology (OIST). This work was supported in part by NIH Grant R01-AI50661 (to M.M.-N. and E. G. Ruby, co-Principal Investigator); NIH Grant R01-OD11024 (to E. G. Ruby, co-Principal Investigator, and M.M.-N.); the University of Wisconsin (M.M.-N.); NASA Space Biology Grant NNX13AM44G (to J.S.F.); NSF Integrative Organismal Systems 1557914 and the Office of the Vice President for Research, UCONN (to S.V.N.); and the Molecular Genetics Unit of OIST Graduate University (to D.S.R.). O.S. and H.S. were supported by a grant from the Austrian Science Fund (Fonds zur Förderung der Wissenschaftlichen Forschung) P30686-B29.</p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn3"><p>The authors declare no conflict of interest.</p></div>
<div class="fn p" id="fn4"><p>This article is a PNAS Direct Submission.</p></div>
<div class="fn p" id="fn5"><p>Data deposition: Genome and transcriptome sequencing reads have been deposited in the National Center for Biotechnology Information Sequence Read Archive, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/sra" rel="noopener noreferrer" target="_blank">https://www.ncbi.nlm.nih.gov/sra</a> (BioProject <a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/sra?term=PRJNA470951" rel="noopener noreferrer" target="_blank">PRJNA470951</a>).</p></div>
<div class="fn p" id="fn6"><p>See Commentary on page <a class="usa-link" href="/articles/PMC6386672/">2799</a>.</p></div>
<div class="fn p" id="fn7"><p>This article contains supporting information online at <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental" rel="noopener noreferrer" target="_blank">www.pnas.org/lookup/suppl/doi:10.1073/pnas.1817322116/-/DCSupplemental</a>.</p></div>
</div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="r1">
<span class="label">1.</span><cite>Moran NA, McCutcheon JP, Nakabachi A. Genomics and evolution of heritable bacterial symbionts. Annu Rev Genet. 2008;42:165–190. doi: 10.1146/annurev.genet.41.110306.130119.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev.genet.41.110306.130119" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18983256/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Genet&amp;title=Genomics%20and%20evolution%20of%20heritable%20bacterial%20symbionts&amp;author=NA%20Moran&amp;author=JP%20McCutcheon&amp;author=A%20Nakabachi&amp;volume=42&amp;publication_year=2008&amp;pages=165-190&amp;pmid=18983256&amp;doi=10.1146/annurev.genet.41.110306.130119&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r2">
<span class="label">2.</span><cite>Markmann K, Parniske M. Evolution of root endosymbiosis with bacteria: How novel are nodules? Trends Plant Sci. 2009;14:77–86. doi: 10.1016/j.tplants.2008.11.009.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tplants.2008.11.009" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19167260/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci&amp;title=Evolution%20of%20root%20endosymbiosis%20with%20bacteria:%20How%20novel%20are%20nodules?&amp;author=K%20Markmann&amp;author=M%20Parniske&amp;volume=14&amp;publication_year=2009&amp;pages=77-86&amp;pmid=19167260&amp;doi=10.1016/j.tplants.2008.11.009&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r3">
<span class="label">3.</span><cite>Matsuura Y, et al.  Evolution of symbiotic organs and endosymbionts in lygaeid stinkbugs. ISME J. 2012;6:397–409. doi: 10.1038/ismej.2011.103.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ismej.2011.103" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3260496/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21814289/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ISME%20J&amp;title=Evolution%20of%20symbiotic%20organs%20and%20endosymbionts%20in%20lygaeid%20stinkbugs&amp;author=Y%20Matsuura&amp;volume=6&amp;publication_year=2012&amp;pages=397-409&amp;pmid=21814289&amp;doi=10.1038/ismej.2011.103&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r4">
<span class="label">4.</span><cite>Griesmann M, et al.  Phylogenomics reveals multiple losses of nitrogen-fixing root nodule symbiosis. Science. 2018;361:eaat1743. doi: 10.1126/science.aat1743.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.aat1743" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29794220/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Phylogenomics%20reveals%20multiple%20losses%20of%20nitrogen-fixing%20root%20nodule%20symbiosis&amp;author=M%20Griesmann&amp;volume=361&amp;publication_year=2018&amp;pages=eaat1743&amp;pmid=29794220&amp;doi=10.1126/science.aat1743&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r5">
<span class="label">5.</span><cite>McFall-Ngai MJ. The importance of microbes in animal development: Lessons from the squid-vibrio symbiosis. Annu Rev Microbiol. 2014;68:177–194. doi: 10.1146/annurev-micro-091313-103654.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev-micro-091313-103654" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6281398/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24995875/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Microbiol&amp;title=The%20importance%20of%20microbes%20in%20animal%20development:%20Lessons%20from%20the%20squid-vibrio%20symbiosis&amp;author=MJ%20McFall-Ngai&amp;volume=68&amp;publication_year=2014&amp;pages=177-194&amp;pmid=24995875&amp;doi=10.1146/annurev-micro-091313-103654&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r6">
<span class="label">6.</span><cite>McFall-Ngai MJ. Giving microbes their due—Animal life in a microbially dominant world. J Exp Biol. 2015;218:1968–1973. doi: 10.1242/jeb.115121.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/jeb.115121" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26085673/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Biol&amp;title=Giving%20microbes%20their%20due%E2%80%94Animal%20life%20in%20a%20microbially%20dominant%20world&amp;author=MJ%20McFall-Ngai&amp;volume=218&amp;publication_year=2015&amp;pages=1968-1973&amp;pmid=26085673&amp;doi=10.1242/jeb.115121&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r7">
<span class="label">7.</span><cite>Collins AJ, et al.  Diversity and partitioning of bacterial populations within the accessory nidamental gland of the squid Euprymna scolopes. Appl Environ Microbiol. 2012;78:4200–4208. doi: 10.1128/AEM.07437-11.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.07437-11" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3370523/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22504817/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Diversity%20and%20partitioning%20of%20bacterial%20populations%20within%20the%20accessory%20nidamental%20gland%20of%20the%20squid%20Euprymna%20scolopes&amp;author=AJ%20Collins&amp;volume=78&amp;publication_year=2012&amp;pages=4200-4208&amp;pmid=22504817&amp;doi=10.1128/AEM.07437-11&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r8">
<span class="label">8.</span><cite>Kerwin AH, Nyholm SV. Symbiotic bacteria associated with a bobtail squid reproductive system are detectable in the environment, and stable in the host and developing eggs. Environ Microbiol. 2017;19:1463–1475. doi: 10.1111/1462-2920.13665.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/1462-2920.13665" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28063183/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&amp;title=Symbiotic%20bacteria%20associated%20with%20a%20bobtail%20squid%20reproductive%20system%20are%20detectable%20in%20the%20environment,%20and%20stable%20in%20the%20host%20and%20developing%20eggs&amp;author=AH%20Kerwin&amp;author=SV%20Nyholm&amp;volume=19&amp;publication_year=2017&amp;pages=1463-1475&amp;pmid=28063183&amp;doi=10.1111/1462-2920.13665&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r9">
<span class="label">9.</span><cite>Simão FA, Waterhouse RM, Ioannidis P, Kriventseva EV, Zdobnov EM. BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs. Bioinformatics. 2015;31:3210–3212. doi: 10.1093/bioinformatics/btv351.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btv351" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26059717/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BUSCO:%20Assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;author=FA%20Sim%C3%A3o&amp;author=RM%20Waterhouse&amp;author=P%20Ioannidis&amp;author=EV%20Kriventseva&amp;author=EM%20Zdobnov&amp;volume=31&amp;publication_year=2015&amp;pages=3210-3212&amp;pmid=26059717&amp;doi=10.1093/bioinformatics/btv351&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r10">
<span class="label">10.</span><cite>Albertin CB, et al.  The octopus genome and the evolution of cephalopod neural and morphological novelties. Nature. 2015;524:220–224. doi: 10.1038/nature14668.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature14668" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4795812/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26268193/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20octopus%20genome%20and%20the%20evolution%20of%20cephalopod%20neural%20and%20morphological%20novelties&amp;author=CB%20Albertin&amp;volume=524&amp;publication_year=2015&amp;pages=220-224&amp;pmid=26268193&amp;doi=10.1038/nature14668&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r11">
<span class="label">11.</span><cite>Simakov O, et al.  Insights into bilaterian evolution from three spiralian genomes. Nature. 2013;493:526–531. doi: 10.1038/nature11696.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature11696" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4085046/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23254933/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Insights%20into%20bilaterian%20evolution%20from%20three%20spiralian%20genomes&amp;author=O%20Simakov&amp;volume=493&amp;publication_year=2013&amp;pages=526-531&amp;pmid=23254933&amp;doi=10.1038/nature11696&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r12">
<span class="label">12.</span><cite>Lee PN, Callaerts P, De Couet HG, Martindale MQ. Cephalopod Hox genes and the origin of morphological novelties. Nature. 2003;424:1061–1065. doi: 10.1038/nature01872.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature01872" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12944969/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Cephalopod%20Hox%20genes%20and%20the%20origin%20of%20morphological%20novelties&amp;author=PN%20Lee&amp;author=P%20Callaerts&amp;author=HG%20De%20Couet&amp;author=MQ%20Martindale&amp;volume=424&amp;publication_year=2003&amp;pages=1061-1065&amp;pmid=12944969&amp;doi=10.1038/nature01872&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r13">
<span class="label">13.</span><cite>Kröger B, Vinther J, Fuchs D. Cephalopod origin and evolution: A congruent picture emerging from fossils, development and molecules: Extant cephalopods are younger than previously realised and were under major selection to become agile, shell-less predators. BioEssays. 2011;33:602–613. doi: 10.1002/bies.201100001.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/bies.201100001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21681989/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BioEssays&amp;title=Cephalopod%20origin%20and%20evolution:%20A%20congruent%20picture%20emerging%20from%20fossils,%20development%20and%20molecules:%20Extant%20cephalopods%20are%20younger%20than%20previously%20realised%20and%20were%20under%20major%20selection%20to%20become%20agile,%20shell-less%20predators&amp;author=B%20Kr%C3%B6ger&amp;author=J%20Vinther&amp;author=D%20Fuchs&amp;volume=33&amp;publication_year=2011&amp;pages=602-613&amp;pmid=21681989&amp;doi=10.1002/bies.201100001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r14">
<span class="label">14.</span><cite>Lindgren AR, Pankey MS, Hochberg FG, Oakley TH. A multi-gene phylogeny of Cephalopoda supports convergent morphological evolution in association with multiple habitat shifts in the marine environment. BMC Evol Biol. 2012;12:129. doi: 10.1186/1471-2148-12-129.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2148-12-129" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3733422/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22839506/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evol%20Biol&amp;title=A%20multi-gene%20phylogeny%20of%20Cephalopoda%20supports%20convergent%20morphological%20evolution%20in%20association%20with%20multiple%20habitat%20shifts%20in%20the%20marine%20environment&amp;author=AR%20Lindgren&amp;author=MS%20Pankey&amp;author=FG%20Hochberg&amp;author=TH%20Oakley&amp;volume=12&amp;publication_year=2012&amp;pages=129&amp;pmid=22839506&amp;doi=10.1186/1471-2148-12-129&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r15">
<span class="label">15.</span><cite>Biggs J, Epel D. Egg capsule sheath of Loligo opalescens Berry: Structure and association with bacteria. J Exp Zool. 1991;259:263–267.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Zool&amp;title=Egg%20capsule%20sheath%20of%20Loligo%20opalescens%20Berry:%20Structure%20and%20association%20with%20bacteria&amp;author=J%20Biggs&amp;author=D%20Epel&amp;volume=259&amp;publication_year=1991&amp;pages=263-267&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r16">
<span class="label">16.</span><cite>Barbieri E, et al.  Phylogenetic characterization of epibiotic bacteria in the accessory nidamental gland and egg capsules of the squid Loligo pealei (Cephalopoda:Loliginidae) Environ Microbiol. 2001;3:151–167. doi: 10.1046/j.1462-2920.2001.00172.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1462-2920.2001.00172.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11321532/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&amp;title=Phylogenetic%20characterization%20of%20epibiotic%20bacteria%20in%20the%20accessory%20nidamental%20gland%20and%20egg%20capsules%20of%20the%20squid%20Loligo%20pealei%20(Cephalopoda:Loliginidae)&amp;author=E%20Barbieri&amp;volume=3&amp;publication_year=2001&amp;pages=151-167&amp;pmid=11321532&amp;doi=10.1046/j.1462-2920.2001.00172.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r17">
<span class="label">17.</span><cite>Gould RM, et al.  Myelin sheaths are formed with proteins that originated in vertebrate lineages. Neuron Glia Biol. 2008;4:137–152. doi: 10.1017/S1740925X09990238.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1017/S1740925X09990238" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3781599/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19737434/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Neuron%20Glia%20Biol&amp;title=Myelin%20sheaths%20are%20formed%20with%20proteins%20that%20originated%20in%20vertebrate%20lineages&amp;author=RM%20Gould&amp;volume=4&amp;publication_year=2008&amp;pages=137-152&amp;pmid=19737434&amp;doi=10.1017/S1740925X09990238&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r18">
<span class="label">18.</span><cite>Khalturin K, Hemmrich G, Fraune S, Augustin R, Bosch TC. More than just orphans: Are taxonomically-restricted genes important in evolution? Trends Genet. 2009;25:404–413. doi: 10.1016/j.tig.2009.07.006.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tig.2009.07.006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19716618/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Genet&amp;title=More%20than%20just%20orphans:%20Are%20taxonomically-restricted%20genes%20important%20in%20evolution?&amp;author=K%20Khalturin&amp;author=G%20Hemmrich&amp;author=S%20Fraune&amp;author=R%20Augustin&amp;author=TC%20Bosch&amp;volume=25&amp;publication_year=2009&amp;pages=404-413&amp;pmid=19716618&amp;doi=10.1016/j.tig.2009.07.006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r19">
<span class="label">19.</span><cite>Tautz D, Domazet-Lošo T. The evolutionary origin of orphan genes. Nat Rev Genet. 2011;12:692–702. doi: 10.1038/nrg3053.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nrg3053" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21878963/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&amp;title=The%20evolutionary%20origin%20of%20orphan%20genes&amp;author=D%20Tautz&amp;author=T%20Domazet-Lo%C5%A1o&amp;volume=12&amp;publication_year=2011&amp;pages=692-702&amp;pmid=21878963&amp;doi=10.1038/nrg3053&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r20">
<span class="label">20.</span><cite>Pankey MS, Minin VN, Imholte GC, Suchard MA, Oakley TH. Predictable transcriptome evolution in the convergent and complex bioluminescent organs of squid. Proc Natl Acad Sci USA. 2014;111:E4736–E4742. doi: 10.1073/pnas.1416574111.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1416574111" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4226117/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25336755/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Predictable%20transcriptome%20evolution%20in%20the%20convergent%20and%20complex%20bioluminescent%20organs%20of%20squid&amp;author=MS%20Pankey&amp;author=VN%20Minin&amp;author=GC%20Imholte&amp;author=MA%20Suchard&amp;author=TH%20Oakley&amp;volume=111&amp;publication_year=2014&amp;pages=E4736-E4742&amp;pmid=25336755&amp;doi=10.1073/pnas.1416574111&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r21">
<span class="label">21.</span><cite>Tong D, et al.  Evidence for light perception in a bioluminescent organ. Proc Natl Acad Sci USA. 2009;106:9836–9841. doi: 10.1073/pnas.0904571106.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0904571106" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2700988/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19509343/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Evidence%20for%20light%20perception%20in%20a%20bioluminescent%20organ&amp;author=D%20Tong&amp;volume=106&amp;publication_year=2009&amp;pages=9836-9841&amp;pmid=19509343&amp;doi=10.1073/pnas.0904571106&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r22">
<span class="label">22.</span><cite>McFall-Ngai M, Heath-Heckman EA, Gillette AA, Peyer SM, Harvie EA. The secret languages of coevolved symbioses: Insights from the Euprymna scolopes-Vibrio fischeri symbiosis. Semin Immunol. 2012;24:3–8. doi: 10.1016/j.smim.2011.11.006.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.smim.2011.11.006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3288948/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22154556/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Semin%20Immunol&amp;title=The%20secret%20languages%20of%20coevolved%20symbioses:%20Insights%20from%20the%20Euprymna%20scolopes-Vibrio%20fischeri%20symbiosis&amp;author=M%20McFall-Ngai&amp;author=EA%20Heath-Heckman&amp;author=AA%20Gillette&amp;author=SM%20Peyer&amp;author=EA%20Harvie&amp;volume=24&amp;publication_year=2012&amp;pages=3-8&amp;pmid=22154556&amp;doi=10.1016/j.smim.2011.11.006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r23">
<span class="label">23.</span><cite>Crookes WJ, et al.  Reflectins: The unusual proteins of squid reflective tissues. Science. 2004;303:235–238. doi: 10.1126/science.1091288.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1091288" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14716016/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Reflectins:%20The%20unusual%20proteins%20of%20squid%20reflective%20tissues&amp;author=WJ%20Crookes&amp;volume=303&amp;publication_year=2004&amp;pages=235-238&amp;pmid=14716016&amp;doi=10.1126/science.1091288&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r24">
<span class="label">24.</span><cite>Tomarev SI, et al.  Abundant mRNAs in the squid light organ encode proteins with a high similarity to mammalian peroxidases. Gene. 1993;132:219–226. doi: 10.1016/0378-1119(93)90199-d.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0378-1119(93)90199-d" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8224867/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Abundant%20mRNAs%20in%20the%20squid%20light%20organ%20encode%20proteins%20with%20a%20high%20similarity%20to%20mammalian%20peroxidases&amp;author=SI%20Tomarev&amp;volume=132&amp;publication_year=1993&amp;pages=219-226&amp;pmid=8224867&amp;doi=10.1016/0378-1119(93)90199-d&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r25">
<span class="label">25.</span><cite>Weis VM, Small AL, McFall-Ngai MJ. A peroxidase related to the mammalian antimicrobial protein myeloperoxidase in the Euprymna-Vibrio mutualism. Proc Natl Acad Sci USA. 1996;93:13683–13688. doi: 10.1073/pnas.93.24.13683.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.93.24.13683" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC19390/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8942994/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=A%20peroxidase%20related%20to%20the%20mammalian%20antimicrobial%20protein%20myeloperoxidase%20in%20the%20Euprymna-Vibrio%20mutualism&amp;author=VM%20Weis&amp;author=AL%20Small&amp;author=MJ%20McFall-Ngai&amp;volume=93&amp;publication_year=1996&amp;pages=13683-13688&amp;pmid=8942994&amp;doi=10.1073/pnas.93.24.13683&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r26">
<span class="label">26.</span><cite>Visick KL, Ruby EG. The periplasmic, group III catalase of Vibrio fischeri is required for normal symbiotic competence and is induced both by oxidative stress and by approach to stationary phase. J Bacteriol. 1998;180:2087–2092. doi: 10.1128/jb.180.8.2087-2092.1998.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/jb.180.8.2087-2092.1998" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC107134/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9555890/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=The%20periplasmic,%20group%20III%20catalase%20of%20Vibrio%20fischeri%20is%20required%20for%20normal%20symbiotic%20competence%20and%20is%20induced%20both%20by%20oxidative%20stress%20and%20by%20approach%20to%20stationary%20phase&amp;author=KL%20Visick&amp;author=EG%20Ruby&amp;volume=180&amp;publication_year=1998&amp;pages=2087-2092&amp;pmid=9555890&amp;doi=10.1128/jb.180.8.2087-2092.1998&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r27">
<span class="label">27.</span><cite>Small AL, McFall-Ngai MJ. Halide peroxidase in tissues that interact with bacteria in the host squid Euprymna scolopes. J Cell Biochem. 1999;72:445–457.</cite> [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10022605/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biochem&amp;title=Halide%20peroxidase%20in%20tissues%20that%20interact%20with%20bacteria%20in%20the%20host%20squid%20Euprymna%20scolopes&amp;author=AL%20Small&amp;author=MJ%20McFall-Ngai&amp;volume=72&amp;publication_year=1999&amp;pages=445-457&amp;pmid=10022605&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r28">
<span class="label">28.</span><cite>Wier AM, et al.  Transcriptional patterns in both host and bacterium underlie a daily rhythm of anatomical and metabolic change in a beneficial symbiosis. Proc Natl Acad Sci USA. 2010;107:2259–2264. doi: 10.1073/pnas.0909712107.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0909712107" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2836665/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20133870/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Transcriptional%20patterns%20in%20both%20host%20and%20bacterium%20underlie%20a%20daily%20rhythm%20of%20anatomical%20and%20metabolic%20change%20in%20a%20beneficial%20symbiosis&amp;author=AM%20Wier&amp;volume=107&amp;publication_year=2010&amp;pages=2259-2264&amp;pmid=20133870&amp;doi=10.1073/pnas.0909712107&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r29">
<span class="label">29.</span><cite>Schleicher TR, Nyholm SV. Characterizing the host and symbiont proteomes in the association between the bobtail squid, Euprymna scolopes, and the bacterium, Vibrio fischeri. PLoS One. 2011;6:e25649. doi: 10.1371/journal.pone.0025649.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0025649" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3187790/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21998678/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Characterizing%20the%20host%20and%20symbiont%20proteomes%20in%20the%20association%20between%20the%20bobtail%20squid,%20Euprymna%20scolopes,%20and%20the%20bacterium,%20Vibrio%20fischeri&amp;author=TR%20Schleicher&amp;author=SV%20Nyholm&amp;volume=6&amp;publication_year=2011&amp;pages=e25649&amp;pmid=21998678&amp;doi=10.1371/journal.pone.0025649&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r30">
<span class="label">30.</span><cite>McFall-Ngai M, Nyholm SV, Castillo MG. The role of the immune system in the initiation and persistence of the Euprymna scolopes–Vibrio fischeri symbiosis. Semin Immunol. 2010;22:48–53. doi: 10.1016/j.smim.2009.11.003.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.smim.2009.11.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3992935/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20036144/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Semin%20Immunol&amp;title=The%20role%20of%20the%20immune%20system%20in%20the%20initiation%20and%20persistence%20of%20the%20Euprymna%20scolopes%E2%80%93Vibrio%20fischeri%20symbiosis&amp;author=M%20McFall-Ngai&amp;author=SV%20Nyholm&amp;author=MG%20Castillo&amp;volume=22&amp;publication_year=2010&amp;pages=48-53&amp;pmid=20036144&amp;doi=10.1016/j.smim.2009.11.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r31">
<span class="label">31.</span><cite>McAnulty SJ, Nyholm SV. The role of hemocytes in the Hawaiian bobtail squid, Euprymna scolopes: A model organism for studying beneficial host-microbe interactions. Front Microbiol. 2017;7:2013. doi: 10.3389/fmicb.2016.02013.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2016.02013" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5216023/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28111565/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=The%20role%20of%20hemocytes%20in%20the%20Hawaiian%20bobtail%20squid,%20Euprymna%20scolopes:%20A%20model%20organism%20for%20studying%20beneficial%20host-microbe%20interactions&amp;author=SJ%20McAnulty&amp;author=SV%20Nyholm&amp;volume=7&amp;publication_year=2017&amp;pages=2013&amp;pmid=28111565&amp;doi=10.3389/fmicb.2016.02013&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r32">
<span class="label">32.</span><cite>Chen F, et al.  Bactericidal permeability-increasing proteins shape host-microbe interactions. MBio. 2017;8:e00040-17. doi: 10.1128/mBio.00040-17.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/mBio.00040-17" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5380838/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28377525/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=Bactericidal%20permeability-increasing%20proteins%20shape%20host-microbe%20interactions&amp;author=F%20Chen&amp;volume=8&amp;publication_year=2017&amp;pages=e00040-17&amp;pmid=28377525&amp;doi=10.1128/mBio.00040-17&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r33">
<span class="label">33.</span><cite>Wilson AC, Duncan RP. Signatures of host/symbiont genome coevolution in insect nutritional endosymbioses. Proc Natl Acad Sci USA. 2015;112:10255–10261. doi: 10.1073/pnas.1423305112.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1423305112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4547219/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26039986/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Signatures%20of%20host/symbiont%20genome%20coevolution%20in%20insect%20nutritional%20endosymbioses&amp;author=AC%20Wilson&amp;author=RP%20Duncan&amp;volume=112&amp;publication_year=2015&amp;pages=10255-10261&amp;pmid=26039986&amp;doi=10.1073/pnas.1423305112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r34">
<span class="label">34.</span><cite>Simakov O, Kawashima T. Independent evolution of genomic characters during major metazoan transitions. Dev Biol. 2017;427:179–192. doi: 10.1016/j.ydbio.2016.11.012.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ydbio.2016.11.012" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27890449/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Dev%20Biol&amp;title=Independent%20evolution%20of%20genomic%20characters%20during%20major%20metazoan%20transitions&amp;author=O%20Simakov&amp;author=T%20Kawashima&amp;volume=427&amp;publication_year=2017&amp;pages=179-192&amp;pmid=27890449&amp;doi=10.1016/j.ydbio.2016.11.012&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="r35">
<span class="label">35.</span><cite>Tanner AR, et al.  Molecular clocks indicate turnover and diversification of modern coleoid cephalopods during the Mesozoic marine revolution. Proc Biol Sci. 2017;284:20162818. doi: 10.1098/rspb.2016.2818.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1098/rspb.2016.2818" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5360930/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28250188/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Biol%20Sci&amp;title=Molecular%20clocks%20indicate%20turnover%20and%20diversification%20of%20modern%20coleoid%20cephalopods%20during%20the%20Mesozoic%20marine%20revolution&amp;author=AR%20Tanner&amp;volume=284&amp;publication_year=2017&amp;pages=20162818&amp;pmid=28250188&amp;doi=10.1098/rspb.2016.2818&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Supplementary File</span></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/6386654/bin/pnas.1817322116.sapp.pdf">pnas.1817322116.sapp.pdf</a><sup> (3.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>Supplementary File</span></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/6386654/bin/pnas.1817322116.sd01.xlsx">pnas.1817322116.sd01.xlsx</a><sup> (195.6KB, xlsx) </sup>
</div></div></section></section></section></section></section></article>
</main></body></html>