Microbiol Resour Announc

. 2020 Oct 15;9(42):e00923-20. doi: [10.1128/MRA.00923-20](https://doi.org/10.1128/MRA.00923-20)

# Draft Genome Sequences of *Klebsiella* Species Isolated from the International Space Station

[Samuel A Solomon](https://pubmed.ncbi.nlm.nih.gov/?term=%22Solomon%20SA%22%5BAuthor%5D)

### Samuel A Solomon

aJet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA

Find articles by [Samuel A Solomon](https://pubmed.ncbi.nlm.nih.gov/?term=%22Solomon%20SA%22%5BAuthor%5D)

a, [Achintya R Bharadwaj](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bharadwaj%20AR%22%5BAuthor%5D)

### Achintya R Bharadwaj

aJet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA

Find articles by [Achintya R Bharadwaj](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bharadwaj%20AR%22%5BAuthor%5D)

a, [Nitin K Singh](https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D)

### Nitin K Singh

aJet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA

Find articles by [Nitin K Singh](https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D)

a, [Jason M Wood](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wood%20JM%22%5BAuthor%5D)

### Jason M Wood

aJet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA

Find articles by [Jason M Wood](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wood%20JM%22%5BAuthor%5D)

a, [Marilyne Debieu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Debieu%20M%22%5BAuthor%5D)

### Marilyne Debieu

bBiotia, New York, New York, USA

Find articles by [Marilyne Debieu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Debieu%20M%22%5BAuthor%5D)

b, [Niamh B O’Hara](https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Hara%20NB%22%5BAuthor%5D)

### Niamh B O’Hara

bBiotia, New York, New York, USA

cDepartment of Cell Biology, College of Medicine, SUNY Downstate Health Sciences University, Brooklyn, New York, USA

Find articles by [Niamh B O’Hara](https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Hara%20NB%22%5BAuthor%5D)

b,c, [Christopher E Mason](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D)

### Christopher E Mason

dWorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, New York, USA

eDepartment of Physiology and Biophysics, Weill Cornell Medicine, New York, New York, USA

Find articles by [Christopher E Mason](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D)

d,e, [Kasthuri Venkateswaran](https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D)

### Kasthuri Venkateswaran

aJet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA

Find articles by [Kasthuri Venkateswaran](https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D)

a,✉

Editor: David A Baltrusf

* Author information
* Article notes
* Copyright and License information

aJet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA

bBiotia, New York, New York, USA

cDepartment of Cell Biology, College of Medicine, SUNY Downstate Health Sciences University, Brooklyn, New York, USA

dWorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, New York, USA

eDepartment of Physiology and Biophysics, Weill Cornell Medicine, New York, New York, USA

fUniversity of Arizona

✉

Address correspondence to Kasthuri Venkateswaran, kjvenkat@jpl.nasa.gov.

**Citation** Solomon SA, Bharadwaj AR, Singh NK, Wood JM, Debieu M, O’Hara NB, Mason CE, Venkateswaran K. 2020. Draft genome sequences of *Klebsiella* species isolated from the International Space Station. Microbiol Resour Announc 9:e00923-20. <https://doi.org/10.1128/MRA.00923-20>.

✉

Corresponding author.

#### Roles

**David A Baltrus**: Editor

Received 2020 Aug 11; Accepted 2020 Sep 24; Collection date 2020 Oct.

Copyright © 2020 Solomon et al.

This is an open-access article distributed under the terms of the [Creative Commons Attribution 4.0 International license](https://creativecommons.org/licenses/by/4.0/).

[PMC Copyright notice](/about/copyright/)

PMCID: PMC7561690  PMID: [33060271](https://pubmed.ncbi.nlm.nih.gov/33060271/)

---

Isolated across four locations aboard the International Space Station (ISS), 10 bacterial strains were compared using whole-genome sequencing analysis and were phylogenetically identified as *Klebsiella*. The whole-genome sequences will aid in comparative genomic studies of ISS *Klebsiella* strains with Earth counterparts, to gain insight into their adaptation to space conditions.

## ABSTRACT

Isolated across four locations aboard the International Space Station (ISS), 10 bacterial strains were compared using whole-genome sequencing analysis and were phylogenetically identified as *Klebsiella*. The whole-genome sequences will aid in comparative genomic studies of ISS *Klebsiella* strains with Earth counterparts to gain insight into their adaptation to space conditions.

## ANNOUNCEMENT

The genus *Klebsiella* was discovered by Carl Friedlander in 1882 from the lungs of patients who had died of pneumonia ([1](#B1)). In infected individuals, *Klebsiella* species can populate the gastrointestinal tract and nasopharynx, surviving on mucosal surfaces, and are known for being highly virulent and resistant to antibiotics ([2](#B2), [3](#B3)). When found on Earth, this genus of bacterial pathogens has various degrees of pathogenicity, which can lead to severe breathing problems necessitating a ventilator and rapid on-site treatment ([4](#B4)). When exposed to space conditions, however, *Klebsiella* species might become immunogenic and thus pose a risk to already immunocompromised astronauts aboard the International Space Station (ISS) ([5](#B5), [6](#B6)). Since microgravity and radiation in space are reported to induce multiple genetic adaptations in microbial species, such as structural modifications of the cell membrane, that can subsequently alter their virulence ([5](#B5)), the strains identified here might potentially pose a problem for the health of astronauts. Therefore, a genetic comparison is necessary to provide more details on the survival of *Klebsiella* species, which have gained more attention because classical *Klebsiella pneumoniae* and its hypervirulent pathotype are becoming increasingly resistant to various antibiotics, such as carbapenems ([3](#B3), [4](#B4), [7](#B7)). The pathogenicity and resistance of the members of the genus *Klebsiella* might potentially create a problem for space travel, specifically for the safety of astronauts.

Several strains of *Klebsiella* species, including *K. aerogenes* (*n* = 1), *K. pneumoniae* (*n* = 1), and *K. quasipneumoniae* (*n* = 8), were isolated from various locations on ISS environmental surfaces ([8](#B8)). The flight number, location, and other sampling characteristics of the ISS *Klebsiella* isolates are detailed in [Table 1](#tab1). Briefly, the environmental samples collected from the ISS and subsequently brought down to Earth at room temperature were aseptically handled according to established procedures ([8](#B8)), and 100-μl aliquots of concentrated samples were spread onto either Reasoner’s 2A (R2A) agar (25°C for 7 days) or blood agar (37°C for 2 days) for isolation of microorganisms. After morphological observation, pure colonies were archived at −80°C until further analyses. Cultures of the 10 *Klebsiella* strains were grown overnight on tryptic soy agar at 25°C until harvesting and DNA extraction using the ZymoBIOMICS DNA Magbead kit.

### TABLE 1.

Metadata and genome statistics of *Klebsiella* strains isolated from various ISS environmental surfaces during the Microbial Tracking-1 flight project

| Sample name | Nearest species[*a*](#ngtab1.1) | ANI (%)[*b*](#ngtab1.2) | GenBank accession no. | Raw read accession no. | Flight-location[*c*](#ngtab1.3) | Location description[*d*](#ngtab1.4) | No. of contigs | Genome size (bp) | *N*50 (bp) | Median coverage (×) | No. of raw reads (×106) | No. of quality-controlled reads (×106) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| IIIF7SW-P1 | *K. aerogenes* ATCC 13048T | 98.66 | [JACBPC000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPC000000000) | [SRR12071884](https://www.ncbi.nlm.nih.gov/sra/SRR12071884) | F3-7 | Lab 3 | 25 | 5,155,046 | 502,580 | 346.88 | 21.6 | 21.5 |
| F3-2P(2\*) | *K. pneumoniae* ATCC 13883T | 99.01 | [JACAUF000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACAUF000000000) | [SRR12068826](https://www.ncbi.nlm.nih.gov/sra/SRR12068826) | F3-2 | WHC | 75 | 5,489,009 | 332,420 | 293.62 | 19.0 | 18.9 |
| IF1SW-B2 | *K. quasipneumoniae* 01A030T | 96.54 | [JABWPD000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPD000000000) | [SRR11884995](https://www.ncbi.nlm.nih.gov/sra/SRR11884995) | F1-1 | Cupola | 24 | 5,192,853 | 601,624 | 533.04 | 31.4 | 31.2 |
| IF1SW-P3 | *K. quasipneumoniae* 01A030T | 96.60 | [JABWOZ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWOZ000000000) | [SRR11885008](https://www.ncbi.nlm.nih.gov/sra/SRR11885008) | F1-1 | Cupola | 28 | 5,192,422 | 600,611 | 467.41 | 28.2 | 28.0 |
| IF1SW-P4 | *K. quasipneumoniae* 01A030T | 96.55 | [JABWPA000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPA000000000) | [SRR11885009](https://www.ncbi.nlm.nih.gov/sra/SRR11885009) | F1-1 | Cupola | 29 | 5,192,468 | 343,111 | 297.32 | 18.6 | 18.6 |
| IF2SW-B3 | *K. quasipneumoniae* 01A030T | 96.54 | [JABWPC000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPC000000000) | [SRR11885011](https://www.ncbi.nlm.nih.gov/sra/SRR11885011) | F1-2 | WHC | 24 | 5,192,354 | 600,611 | 712.50 | 39.8 | 39.6 |
| IF2SW-P1 | *K. quasipneumoniae* 01A030T | 96.61 | [JABWPB000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPB000000000) | [SRR11885010](https://www.ncbi.nlm.nih.gov/sra/SRR11885010) | F1-2 | WHC | 27 | 5,192,385 | 601,139 | 480.80 | 30.2 | 30.1 |
| IIIF3SW-P1 | *K. quasipneumoniae* 01A030T | 96.63 | [JABXWM000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABXWM000000000) | [SRR12070037](https://www.ncbi.nlm.nih.gov/sra/SRR12070037) | F3-3 | ARED | 31 | 5,154,415 | 766,557 | 286.96 | 18.0 | 17.8 |
| F3-6P(1) | *K. quasipneumoniae* 01A030T | 96.57 | [JABXWL000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABXWL000000000) | [SRR12070038](https://www.ncbi.nlm.nih.gov/sra/SRR12070038) | F3-6 | PMM | 28 | 5,196,291 | 1,009,008 | 371.62 | 23.2 | 22.9 |
| F3-6P(2) | *K. quasipneumoniae* 01A030T | 96.60 | [JABXWK000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABXWK000000000) | [SRR12070039](https://www.ncbi.nlm.nih.gov/sra/SRR12070039) | F3-6 | PMM | 31 | 5,196,372 | 1,009,008 | 367.78 | 23.4 | 23.1 |

[Open in a new tab](table/tab1/)

a

The 16S rRNA gene sequences were retrieved from the whole-genome sequences of the queried genomes and subjected to BLAST analysis against type strains for all 16S rRNA sequences in the NCBI database. Bacterial species identity was determined when the queried sequence showed >97.5% similarity to the 16S rRNA gene sequence of the type strain. The whole-genome sequence of the nearest neighbor listed was selected for ANI evaluation.

b

ANI calculations were carried out using the EZBioCloud ANI calculator (<https://www.ezbiocloud.net/tools/ani>) by comparing with the listed type strain.

c

Hyphenated designations indicate the flight number followed by the location; for example, F3-7 indicates flight 3 and location 7.

d

WHC, waste and hygiene compartment; ARED, advanced resistive exercise device; PMM, permanent multipurpose module port 1.

Genomes were sequenced using the Illumina (San Diego, CA) Nextera Flex protocol for library preparation, and a NovaSeq 6000 S4 flow cell (paired end, 2 × 150 bp) was used for paired-end sequencing. The quality was assessed with FastQC (v0.11.7) ([9](#B9)). Adapter trimming and quality filtering were then carried out with fastp (v0.20.0) ([10](#B10)). After quality control, the sequences were assembled using SPAdes (v3.11.1) ([11](#B11)). To assess the quality of the final sequences, a QUAST (v5.0.2) analysis ([12](#B12)) was performed to check the *N*50 values, the number of contigs, and the total genome length ([Table 1](#tab1)). The GC contents are 54.96% for *K. aerogenes*, 57.25% for *K. pneumoniae*, and 58.11 to 58.13% for *K. quasipneumoniae*. The 16S rRNA gene sequences of the *Klebsiella* strains were compared to find the nearest neighbor, and phylogenic characterization was determined by calculating the average nucleotide identity (ANI) using the EZBioCloud calculator ([13](#B13)), in comparison with the respective type strains (*K. aerogenes* ATCC 13048T, *K. pneumoniae* ATCC 13883T, and *K. quasipneumoniae* 01A030T). Default parameters were used for all software.

### Data availability.

This whole-genome sequencing project has been deposited in GenBank, and the GenBank and raw read accession numbers are given in [Table 1](#tab1). The BioProject accession numbers are [PRJNA635942](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA635942), [PRJNA640688](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA640688), and [PRJNA640693](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA640693). Whole-genome sequencing data have also been deposited in NASA GeneLab (accession numbers [GLDS-302](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-302), [GLDS-309](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-309), and [GLDS-311](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-311)). The versions described in this paper are the first versions.

## ACKNOWLEDGMENTS

We thank astronauts Captain Terry Virts and Commander Jeffrey Williams for collecting samples aboard the ISS and the Implementation Team at NASA Ames Research Center (Fathi Karouia) for coordinating this effort. We thank Ryan Kemp (Zymo Corp.) for extracting DNA and Dan Butler (Cornell Medicine) for performing shotgun sequencing using the NovaSeq platform. We also acknowledge the Jet Propulsion Laboratory supercomputing facility staff, notably Narendra J. Patel (Jimmy) and Edward Villanueva, for their continuous support in providing the best possible infrastructure for BIG-DATA analysis.

The research described in this article was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with NASA. This research was funded by 2012 Space Biology project NNH12ZTT001N grant 19-12829-26 under task order NNN13D111T awarded to K.V., which also funded a postdoctoral fellowship for J.M.W. and a subcontract to Biotia, Inc. Government sponsorship is acknowledged.

## REFERENCES

* 1.Friedlaender C.
  1882.
  Ueber die Schizomyceten bei der acuten fibrösen Pneumonie. Arch Pathol Anat
  87:319–324. doi: 10.1007/BF01880516. [[DOI](https://doi.org/10.1007/BF01880516)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Pathol%20Anat&title=Ueber%20die%20Schizomyceten%20bei%20der%20acuten%20fibr%C3%B6sen%20Pneumonie&author=C%20Friedlaender&volume=87&publication_year=1882&pages=319-324&doi=10.1007/BF01880516&)]
* 2.Martin RM, Bachman MA.
  2018.
  Colonization, infection, and the accessory genome of *Klebsiella pneumoniae*. Front Cell Infect Microbiol
  8:4. doi: 10.3389/fcimb.2018.00004. [[DOI](https://doi.org/10.3389/fcimb.2018.00004)] [[PMC free article](/articles/PMC5786545/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29404282/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Infect%20Microbiol&title=Colonization,%20infection,%20and%20the%20accessory%20genome%20of%20Klebsiella%20pneumoniae&author=RM%20Martin&author=MA%20Bachman&volume=8&publication_year=2018&pages=4&pmid=29404282&doi=10.3389/fcimb.2018.00004&)]
* 3.Garbati MA, Al Godhair AI.
  2013.
  The growing resistance of *Klebsiella pneumoniae*; the need to expand our antibiogram: case report and review of the literature. Afr J Infect Dis
  7:8–10. doi: 10.4314/ajid.v7i1.2. [[DOI](https://doi.org/10.4314/ajid.v7i1.2)] [[PMC free article](/articles/PMC3647522/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24381721/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Afr%20J%20Infect%20Dis&title=The%20growing%20resistance%20of%20Klebsiella%20pneumoniae;%20the%20need%20to%20expand%20our%20antibiogram:%20case%20report%20and%20review%20of%20the%20literature&author=MA%20Garbati&author=AI%20Al%20Godhair&volume=7&publication_year=2013&pages=8-10&pmid=24381721&doi=10.4314/ajid.v7i1.2&)]
* 4.Domínguez-Andrés J, Eleveld M, Renieris G, Boltje TJ, Mesman RJ, van Niftrik L, Moons SJ, Rettberg P, van der Meer JWM, Giamarellos-Bourboulis EJ, Op den Camp HJM, de Jonge MI, Netea MG.
  2020.
  Growth on carbohydrates from carbonaceous meteorites alters the immunogenicity of environment-derived bacterial pathogens. Astrobiology. doi: 10.1089/ast.2019.2173. [[DOI](https://doi.org/10.1089/ast.2019.2173)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32391711/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Growth%20on%20carbohydrates%20from%20carbonaceous%20meteorites%20alters%20the%20immunogenicity%20of%20environment-derived%20bacterial%20pathogens&author=J%20Dom%C3%ADnguez-Andr%C3%A9s&author=M%20Eleveld&author=G%20Renieris&author=TJ%20Boltje&author=RJ%20Mesman&publication_year=2020&pmid=32391711&doi=10.1089/ast.2019.2173&)]
* 5.Taylor PW.
  2015.
  Impact of space flight on bacterial virulence and antibiotic susceptibility. Infect Drug Resist
  8:249–262. doi: 10.2147/IDR.S67275. [[DOI](https://doi.org/10.2147/IDR.S67275)] [[PMC free article](/articles/PMC4524529/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26251622/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect%20Drug%20Resist&title=Impact%20of%20space%20flight%20on%20bacterial%20virulence%20and%20antibiotic%20susceptibility&author=PW%20Taylor&volume=8&publication_year=2015&pages=249-262&pmid=26251622&doi=10.2147/IDR.S67275&)]
* 6.Russo TA, Marr CM.
  2019.
  Hypervirulent *Klebsiella pneumoniae*. Clin Microbiol Rev
  32:e00001-19. doi: 10.1128/CMR.00001-19. [[DOI](https://doi.org/10.1128/CMR.00001-19)] [[PMC free article](/articles/PMC6589860/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31092506/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Rev&title=Hypervirulent%20Klebsiella%20pneumoniae&author=TA%20Russo&author=CM%20Marr&volume=32&publication_year=2019&pages=e00001-19&pmid=31092506&doi=10.1128/CMR.00001-19&)]
* 7.Singh NK, Bezdan D, Checinska Sielaff A, Wheeler K, Mason CE, Venkateswaran K.
  2018.
  Multi-drug resistant *Enterobacter bugandensis* species isolated from the International Space Station and comparative genomic analyses with human pathogenic strains. BMC Microbiol
  18:175. doi: 10.1186/s12866-018-1325-2. [[DOI](https://doi.org/10.1186/s12866-018-1325-2)] [[PMC free article](/articles/PMC6251167/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30466389/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&title=Multi-drug%20resistant%20Enterobacter%20bugandensis%20species%20isolated%20from%20the%20International%20Space%20Station%20and%20comparative%20genomic%20analyses%20with%20human%20pathogenic%20strains&author=NK%20Singh&author=D%20Bezdan&author=A%20Checinska%20Sielaff&author=K%20Wheeler&author=CE%20Mason&volume=18&publication_year=2018&pages=175&pmid=30466389&doi=10.1186/s12866-018-1325-2&)]
* 8.Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, Minich J, McDonald D, Mayer T, Knight R, Karouia F, Fox GE, Venkateswaran K.
  2019.
  Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome
  7:50. doi: 10.1186/s40168-019-0666-x. [[DOI](https://doi.org/10.1186/s40168-019-0666-x)] [[PMC free article](/articles/PMC6452512/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&author=A%20Checinska%20Sielaff&author=C%20Urbaniak&author=GBM%20Mohan&author=VG%20Stepanov&author=Q%20Tran&volume=7&publication_year=2019&pages=50&pmid=30955503&doi=10.1186/s40168-019-0666-x&)]
* 9.Andrews S.
  2010.
  FastQC: a quality control tool for high throughput sequence data. <http://www.bioinformatics.babraham.ac.uk/projects/fastqc>.
* 10.Chen S, Zhou Y, Chen Y, Gu J.
  2018.
  fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics
  34:i884–i890. doi: 10.1093/bioinformatics/bty560. [[DOI](https://doi.org/10.1093/bioinformatics/bty560)] [[PMC free article](/articles/PMC6129281/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30423086/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&author=S%20Chen&author=Y%20Zhou&author=Y%20Chen&author=J%20Gu&volume=34&publication_year=2018&pages=i884-i890&pmid=30423086&doi=10.1093/bioinformatics/bty560&)]
* 11.Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA.
  2012.
  SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol
  19:455–477. doi: 10.1089/cmb.2012.0021. [[DOI](https://doi.org/10.1089/cmb.2012.0021)] [[PMC free article](/articles/PMC3342519/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&author=A%20Bankevich&author=S%20Nurk&author=D%20Antipov&author=AA%20Gurevich&author=M%20Dvorkin&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)]
* 12.Gurevich A, Saveliev V, Vyahhi N, Tesler G.
  2013.
  QUAST: quality assessment tool for genome assemblies. Bioinformatics
  29:1072–1075. doi: 10.1093/bioinformatics/btt086. [[DOI](https://doi.org/10.1093/bioinformatics/btt086)] [[PMC free article](/articles/PMC3624806/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23422339/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=QUAST:%20quality%20assessment%20tool%20for%20genome%20assemblies&author=A%20Gurevich&author=V%20Saveliev&author=N%20Vyahhi&author=G%20Tesler&volume=29&publication_year=2013&pages=1072-1075&pmid=23422339&doi=10.1093/bioinformatics/btt086&)]
* 13.Yoon S-H, Ha S, Lim J, Kwon S, Chun J.
  2017.
  A large-scale evaluation of algorithms to calculate average nucleotide identity. Antonie Van Leeuwenhoek
  110:1281–1286. doi: 10.1007/s10482-017-0844-4. [[DOI](https://doi.org/10.1007/s10482-017-0844-4)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28204908/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antonie%20Van%20Leeuwenhoek&title=A%20large-scale%20evaluation%20of%20algorithms%20to%20calculate%20average%20nucleotide%20identity&author=S-H%20Yoon&author=S%20Ha&author=J%20Lim&author=S%20Kwon&author=J%20Chun&volume=110&publication_year=2017&pages=1281-1286&pmid=28204908&doi=10.1007/s10482-017-0844-4&)]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

This whole-genome sequencing project has been deposited in GenBank, and the GenBank and raw read accession numbers are given in [Table 1](#tab1). The BioProject accession numbers are [PRJNA635942](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA635942), [PRJNA640688](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA640688), and [PRJNA640693](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA640693). Whole-genome sequencing data have also been deposited in NASA GeneLab (accession numbers [GLDS-302](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-302), [GLDS-309](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-309), and [GLDS-311](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-311)). The versions described in this paper are the first versions.