Microorganisms

. 2022 Aug 25;10(9):1714. doi: [10.3390/microorganisms10091714](https://doi.org/10.3390/microorganisms10091714)

# Microbial Burden Estimation of Food Items, Built Environments, and the International Space Station Using Film Media

[Anna C Simpson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Simpson%20AC%22%5BAuthor%5D)

### Anna C Simpson

1Jet Propulsion Laboratory, California Institute of Technology, 4800 Oak Grove Drive, Pasadena, CA 91109, USA

Find articles by [Anna C Simpson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Simpson%20AC%22%5BAuthor%5D)

1,†, [Takeo Suzuki](https://pubmed.ncbi.nlm.nih.gov/?term=%22Suzuki%20T%22%5BAuthor%5D)

### Takeo Suzuki

2Research & Development Division, Kikkoman Corporation, Noda 278-0037, Japan

Find articles by [Takeo Suzuki](https://pubmed.ncbi.nlm.nih.gov/?term=%22Suzuki%20T%22%5BAuthor%5D)

2,†, [Daniel R Miller](https://pubmed.ncbi.nlm.nih.gov/?term=%22Miller%20DR%22%5BAuthor%5D)

### Daniel R Miller

1Jet Propulsion Laboratory, California Institute of Technology, 4800 Oak Grove Drive, Pasadena, CA 91109, USA

Find articles by [Daniel R Miller](https://pubmed.ncbi.nlm.nih.gov/?term=%22Miller%20DR%22%5BAuthor%5D)

1, [Kasthuri Venkateswaran](https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D)

### Kasthuri Venkateswaran

1Jet Propulsion Laboratory, California Institute of Technology, 4800 Oak Grove Drive, Pasadena, CA 91109, USA

Find articles by [Kasthuri Venkateswaran](https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D)

1,\*,‡

Editor: Ricardo Amils

* Author information
* Article notes
* Copyright and License information

1Jet Propulsion Laboratory, California Institute of Technology, 4800 Oak Grove Drive, Pasadena, CA 91109, USA

2Research & Development Division, Kikkoman Corporation, Noda 278-0037, Japan

\*

Correspondence: kjvenkat@jpl.nasa.gov; Tel.: +1-(818)-393-1481; Fax: +1-(818)-393-4176

†

These authors contributed equally to this work.

‡

Biotechnology and Planetary Protection Group; M/S 89-2.

#### Roles

**Ricardo Amils**: Academic Editor

Received 2022 Aug 4; Accepted 2022 Aug 23; Collection date 2022 Sep.

© 2022 by the authors.

Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<https://creativecommons.org/licenses/by/4.0/>).

[PMC Copyright notice](/about/copyright/)

PMCID: PMC9503880  PMID: [36144316](https://pubmed.ncbi.nlm.nih.gov/36144316/)

## Abstract

The use of film media involves considerably less preparation, waste, and incubator space than conventional agar-media-based assays and has proven in past studies to provide counts of cultivable microbes similar to those of traditional agar media. Film media also have the advantage of allowing sample volumes similar to those used in pour plates and, therefore, are well-suited for cultivable microbial counts in extremely low-biomass environments such as clean rooms or space habitats, particularly where the subsequent isolation of colonies is necessary. As the preparation of film media plates relies on water cohesion/adhesion rather than manual spreading, they may have future applications in low- or microgravity settings. In this study, cultivable microbial count performance was compared between agar media and film media in three kinds of samples: food items, surfaces in built environments on Earth (homes), and on the environmental surfaces of the International Space Station (ISS). Easy Plates (Kikkoman Corporation) and Petrifilm (3M) were compared with traditional agar plating for food and home surfaces, while only Easy Plates were compared with agar for ISS samples. For both food items and built environments on Earth, both types of film media performed comparably to agar media for bacterial counts, with R2 values of 0.94–0.96. Fungal counts for built-environment samples had a lower correlation between film and agar counts, with R2 values of 0.72–0.73. Samples from the ISS, which ranged from below detection to 103 CFU per 100 cm2, had R2 values of 0.80 for bacterial counts and 0.73 for fungal counts, partially due to multiple samples recording below the detection limit for agar or too numerous to count, and the growth of fungal species on R2A medium. The species compositions of isolates picked from agar vs. film media plates were similar; however, further phylogenetic analysis is needed to confirm the differential microbial diversity composition. Overall, film media such as Easy Plates and Petrifilm are viable alternatives to agar plates for low-biomass built environments as well as for food samples, and the two brands tested in this study performed equally well.

**Keywords:** CFU, International Space Station, environmental microbiome, agar media, film media, Easy Plate, Petrifilm

## 1. Introduction

Counts of cultivable bacteria and fungi from food items, water samples, and surfaces are a traditional mainstay for public health standards and assessment, as well as the core method that NASA uses to ensure that any craft or robot launched into space meets planetary protection regulations [[1](#B1-microorganisms-10-01714)]. Preparing, plating, incubating, and counting samples using traditional agar plates, however, generates a great deal of plastic and organic waste and can be prohibitive in terms of the time and space required. Obtaining cultivable microbial counts from low-biomass arenas, including pure water samples, hospitals, clean rooms, and space habitats such as the International Space Station (ISS), presents additional challenges. The samples from these environments are often below detection limits when using standard plating methodology and dilutions [[2](#B2-microorganisms-10-01714),[3](#B3-microorganisms-10-01714),[4](#B4-microorganisms-10-01714)], requiring larger numbers and volumes of samples to characterize microbial bioburden [[5](#B5-microorganisms-10-01714)]. During a survey of multiple NASA clean rooms, for example, more than 30% of samples were below the detection limit for measuring the number of cultivable microbes [[6](#B6-microorganisms-10-01714)]. In addition, the whole-genome sequencing of isolated fungi and bacteria from cultivable approaches is often necessary for further phenotypic and genomic characterization, which can be used to understand the functional and physiological traits of cultivable microorganisms. For example, isolates from hospitals and human habitats are often assessed for changes or increases in virulence and antibiotic resistance, e.g., [[7](#B7-microorganisms-10-01714),[8](#B8-microorganisms-10-01714),[9](#B9-microorganisms-10-01714)]. Similarly, the NASA community is in need of understanding the yet-to-be cultured microorganisms for their possible survival or proliferation in extraterrestrial conditions [[10](#B10-microorganisms-10-01714),[11](#B11-microorganisms-10-01714),[12](#B12-microorganisms-10-01714),[13](#B13-microorganisms-10-01714)]. Therefore, any improvements or changes in plating methods must not only produce cultivable microbial counts comparable to traditional methods but must also not change the ratios or types of the isolated species.

For the last 50 years, miniaturized stackable media called “film” media have been available as alternatives to traditional agar [[14](#B14-microorganisms-10-01714)]. Petrifilm was developed by 3M in the 1980s as an alternative to agar spread and pour plates. It consists of a thin wafer of nutrient/gelling agent between two plastic films. Petrifilm was designed to plate 1 mL of a sample on a surface with the approximate size and volume of a playing card, using dyes such as tetrazolium to indicate colony growth [[15](#B15-microorganisms-10-01714)]. This greatly reduces both the volume of incubator space needed to grow a large number of samples and the time and materials needed to prepare agar plates. According to 3M, Petrifilm media “take up 85% less space, reduce waste 66%, greenhouse gas emissions by 75%, and water waste by 79%” compared with agar plates [[15](#B15-microorganisms-10-01714)]. Recently, Kikkoman/Dai Nippon Printing (DNP) has developed another brand of film media, Easy Plates, which are similar to Petrifilm in many respects but which have the advantage of a hydrophobic resin ring surrounding the medium which negates the use of a spreader tool, reducing preparation time even further. In addition, the Easy Plate design purportedly increases the ease of counting by its reduced liquefaction of the gel compared with Petrifilm [[16](#B16-microorganisms-10-01714)].

Previous reports on Petrifilm media found that they produced statistically similar results for cultivable microbial counts to those of agar plate methods for food and environmental safety testing, including aerobic bacteria in raw milk [[17](#B17-microorganisms-10-01714)], poultry [[18](#B18-microorganisms-10-01714)], fungal counts on dairy products [[19](#B19-microorganisms-10-01714)], food items spiked with a variety of common molds [[20](#B20-microorganisms-10-01714)], and aerobic or coliform bacteria in wastewater [[21](#B21-microorganisms-10-01714)]. More recent studies showed that film media obtained equivalent counts to agar for soil samples inoculated with varying concentrations of *E. coli* [[22](#B22-microorganisms-10-01714)], steel and porcelain surfaces inoculated with ubiquitous built-environment bacteria [[23](#B23-microorganisms-10-01714)], biofilm samples using drop plating [[24](#B24-microorganisms-10-01714)], and bacterial counts on a variety of food items [[25](#B25-microorganisms-10-01714)]. There has also been extensive testing of film media in the dairy industry which is leading to the adoption of Petrifilm as a standard method for the enumeration of aerobic and coliform bacteria [[26](#B26-microorganisms-10-01714),[27](#B27-microorganisms-10-01714),[28](#B28-microorganisms-10-01714),[29](#B29-microorganisms-10-01714)]. Petrifilm is now becoming a widely used method of bacterial and fungal cultivable microbial counting and isolation in place of agar media [[29](#B29-microorganisms-10-01714),[30](#B30-microorganisms-10-01714)].

Most studies testing film vs. agar media performance for cultivable microbial counts have focused on high-bioburden environments and samples such as dairy products and food, or samples/surfaces that are pre-inoculated with a known microbial consortium. Film media are only recently being assessed for cultivable microbial counts in extremely low-biomass environments such as clean rooms and hospitals [[31](#B31-microorganisms-10-01714),[32](#B32-microorganisms-10-01714),[33](#B33-microorganisms-10-01714)]. Recently, researchers from NASA compared Petrifilm Rapid Aerobic Count (RAC) plates with tryptic soy agar (TSA) plates for measuring the bioburden of an extremely low-biomass clean room environment and found that Petrifilm was equal to TSA plates in aerobic bacterial estimation [[31](#B31-microorganisms-10-01714)]. In addition, Petrifilm was found to be superior to TSA in containing colonies that could spread and grow to cover 25% or more area of agar plate [[31](#B31-microorganisms-10-01714)].

Kikkoman’s Easy Plates have not yet been compared in the scientific literature for their performance relative to both agar plates and Petrifilm. While film media have been previously used in place of agar plating to assess cultivable microbial counts for built environments and hospitals [[30](#B30-microorganisms-10-01714)], they have not been tested in extremely low-biomass environments, particularly those with high fiber counts. Additionally, because Easy Plates entirely rely on water adhesion/cohesion forces for inoculation rather than manual spreading, it is likely that they could be viable options for environmental monitoring in micro- or low-gravity settings in future space installations.

Therefore, our objectives for this study were to (1) test the bacterial enumeration performance among agar plates, Petrifilm, and Easy Plates for a variety of food items; (2) compare the cultivable microbial counts in an Earth-built-environment setting for agar plates, Petrifilm, and Easy Plates using bacterial- and fungal-specific media; and (3) test the efficacy of Easy Plates as an alternative method for bacterial and fungal bioburden estimation and the isolation of bacterial and fungal strains for surface samples from the ISS, for both cultivable microbial counts and species composition of isolates.

## 2. Materials and Methods

### 2.1. Descriptions of Petrifilm and Easy Plate Media Tested

According to the manufacturer’s instruction manuals, Petrifilm Aerobic Count (AC) Plate [[15](#B15-microorganisms-10-01714)] (3M, Saint Paul, MN, USA, Cat# 6400) and Easy Plate AC [[16](#B16-microorganisms-10-01714)] (Kikkoman, Noda, Chiba, Japan, Cat# 61973) were used for the enumeration of aerobic bacteria, and Petrifilm Rapid Yeast and Mold Count (RYM) Plate (3M, Cat# 6475) and Easy Plate YM-R (Kikkoman, Cat# 61977) were used for the enumeration of yeast and mold in the food and beverage industries. In general, in food and beverage microbiological testing, both the Petrifilm AC and Easy Plate AC count red colonies after 48 h of incubation at 35 °C. Petrifilm RYM counts blue-green colonies after 48 h of incubation at 25 °C or 28 °C, and Easy Plate YM-R counts purple colonies after 48 h at 25 °C. Both media can extend the incubation time up to 3–7 days if the colonies appear faint.

### 2.2. Food-Item Cultivable Microbial Counts and Media Testing

Food items (n = 63), including raw and cooked meats, vegetables, nuts, and processed foods (see [Supplementary Table S1](#app1-microorganisms-10-01714) for list), were homogenized with 90 mL (10 g samples) or 225 mL (25 g samples) of sterile phosphate-buffered saline (PBS), and 10-fold serial dilutions were performed. One milliliter of each dilution was inoculated into each medium (Plate Count Agar, Petrifilm AC, and Easy Plate AC), incubated for 48 h at 35 °C, which are standard FDA methods for food safety testing [[34](#B34-microorganisms-10-01714)], and the number of bacteria was counted. Fungal counts were not assayed for food samples.

### 2.3. Built-Environment Cultivable Microbial Counts and Media Testing

The surfaces of Japanese homes (n = 5) were sampled for bacterial and fungal cultivable microbial counts. At each home, three different surfaces were sampled: one expected low-bioburden (example: window glass), one expected medium-bioburden, and one expected high-bioburden (example: living room floor) sample. Pro-media ST-25 phosphate-buffered saline (PBS) swab kits from ELMEX LIMITED (Shinjuku-ku, Tokyo, Japan), consisting of a cotton swab in 10 mL of sterile PBS, were used to swab the surfaces and were kept refrigerated (~4 °C) until use.

In each home, for each surface, a 100 cm2 area was sampled in all cases except for complex shapes (such as doorknobs) in which case the shape was thoroughly wiped over its entire surface area, and the swab was returned to the sterile PBS and shaken. Aliquots of 1 mL of undiluted and 1 mL of 10−1 diluted samples were plated on 6 types of media for each sample: traditional plate count agar (PCA) pour plate, Easy Plate AC, and Petrifilm AC to target aerobic bacteria, and conventional potato dextrose agar (PDA, BD Diagnostics Cat #213400) with 0.05 g/L chloramphenicol pour plate, Easy Plate YM-R, and Petrifilm RYM to target environmental fungi. Plates were incubated for 7 days at 25 °C, with cultivable microbial counts taken on Day 2 and Day 7; however, if plates were close to being too numerous to count on Day 2, colonies were counted and not incubated further.

### 2.4. International Space Station Sampling and Media Testing

*Preparation of inoculum:* Methods for preparing sampling kits and processing samples were similar to those used in the Microbial Tracking 1 and 2 missions [[3](#B3-microorganisms-10-01714),[35](#B35-microorganisms-10-01714),[36](#B36-microorganisms-10-01714)]. Sterile TexTra™ 10 TX3224 Dry Cleanroom Wipers were moistened with 15 mL of sterile phosphate-buffered saline (PBS), folded into quarters, and placed in sterile Ziploc baggies, which were later used to assemble the sampling kits at NASA Ames. Sample kits were transported to the ISS in June 2021. Over the course of five days, astronauts sampled the same square meter area of the same eight surfaces each day (see [[3](#B3-microorganisms-10-01714)] for detailed surface descriptions). On each sampling day, for each surface, the astronaut tasked with sampling donned a new pair of sterile gloves, removed and unfolded a sterile wipe from its bag, and wiped a square meter area of the surface three times, folding the wipe in half for the second pass and into quarters for the third pass. Additionally, on each sampling day, a wipe was removed from its bag and briefly exposed to the air, as an environmental control sample. A total of 45 samples (8 surfaces + 1 control, ×5 days) were produced and stored at 4 °C aboard the ISS and during transport to Earth.

Upon delivery to the Jet Propulsion Laboratory (JPL), the wipes were separated into batches and stored at 4 °C until processing. All wipes were processed within 48 h of arrival at the JPL for the isolation of environmental microbes. All lab procedures were performed for all samples within a 12 h period. Each wipe was placed in a sterile 500 mL bottle with 200 mL sterile PBS and vigorously shaken for 2 min. In addition to the 45 ISS samples, an unopened wipe and a bottle of a sterile buffer without the addition of wipes were used as negative controls; no colonies resulted from these samples; thus, those data are not reported here. The PBS was then concentrated using an InnovaPrep CP-150 (Innovaprep LLC, Drexel, MI, USA) with a 0.2 µm Polysulfone hollow-fiber-concentrating pipette tip. Using additional sterile PBS, the resulting PBS concentrate was adjusted to 4 mL for all samples, and dilutions of 10−1, 10−2, and 10−3 were prepared.

*Inoculation and cultivable microbial counting:* Two replicate spread plates were prepared for all three dilution levels (10−1, 10−2, and 10−3), using two types of agar media: Reasoner’s 2 agar (R2A, BD Diagnostics Cat # 218263) and PDA plates supplemented with 0.05 g/L chloramphenicol, for a total of 12 plates per sample (2 agar types × 3 dilution levels × 2 replicates; ~540 plates in total). This occupied a total of almost 0.1 cubic meters, the equivalent of a large moving box. For each of the two types of film media (Kikkoman Easy Plate AC and Easy Plate YM-R), 1 mL of the 10−1 dilution was directly pipetted into the center of each plate and then sealed according to the manufacturer’s instructions. Agar and film plates were incubated at room temperature for 7 days; cultivable microbial counts were performed on Days 2 and 7. For ISS samples, PetriFilm media were not used.

*Isolation, purification, and identification of ISS strains:* After cultivable microbial counting, up to 5 isolates were chosen from each combination of media type and sample. On both the agar plates and Easy Plates, colonies were deliberately selected for morphological differences, size, and colony coloration on agar plates. Colonies from Easy Plates were first streaked out onto TSA/PDA plates, incubated at 25 °C for 7 days, and imaged, before being transferred to 1:10 agar stabs of respective media. Similarly, the colonies from agar plates were directly transferred to respective agar stabs. The isolates were later streaked out for purification onto TSA or PDA plates. The secondary streak plates of purified isolates were sent to Azenta Life Sciences (Southfield, NJ, USA) for colony PCR + Sanger sequencing for the 16S rRNA gene or the ribosomal Internal Transcriber Space (ITS) region, respectively, for bacteria and fungi. Primers 27F and 1429R (27F 5′-AGAGTTTGATCCTGGCTCAG-3′; 1492R 5′-GGTTACCTTGTTACGACTT-3′) [[37](#B37-microorganisms-10-01714)] were used to target the entire 16S region (V1-V9) for bacterial strains. ITS regions 1 and 2 were targeted for fungal taxonomy using primers ITS1-F and ITS4 (ITS1 5′-CTTGGTCATTTAGAGGAAGTAA-3′; ITS4 5′-TCCTCCGCTTATTGATATGC-3′) [[38](#B38-microorganisms-10-01714)]. Per Azenta Life Sciences standard operating procedure, enzymatic cleanup was performed after amplification using Exonuclease I—Shrimp Alkaline Phosphatase (ExoSAP), and dye-terminator Sanger sequencing was performed using an Applied Biosystems BigDye version 3.1 sequencing kit using an Applied Biosystem 3730xl DNA Analyzer.

*Statistical analysis:* All statistical analyses and figure creation were carried out in R. Consensus 16S rRNA gene and ITS sequences were generated using the sangeranalyzeR package [[39](#B39-microorganisms-10-01714)], and genus/species identity was determined using the nearest match to the NCBI nucleotide database. Type II linear models, Spearman and Pearson’s correlation coefficients, and paired and unpaired Wilcoxon rank-sum tests were used to assess the correlations and mean differences between the cultivable microbial counts of agar and film media. Although a 1 m2 area was used for ISS samples, for this research communication, we have converted the unit to cultivable microbes per 100 cm2 in order to illustrate the extremely low bioburden of the ISS samples compared with that of the food-item and built-environment studies also sampled.

## 3. Results

The microbial population enumerated with traditional agar plate, Petrifilm, and Easy Plate are depicted in [Figure 1](#microorganisms-10-01714-f001). All three types of media were used for the comparison of food products ([Figure 1](#microorganisms-10-01714-f001)A) and built environments ([Figure 1](#microorganisms-10-01714-f001)B for bacteria and [Figure 1](#microorganisms-10-01714-f001)C for fungi). Only the R2A medium was compared with the Easy Plate method for estimating the bacterial population ([Figure 1](#microorganisms-10-01714-f001)D) of the ISS samples. Similarly, the PDA medium and Easy Plate were employed to enumerate the fungal population ([Figure 1](#microorganisms-10-01714-f001)E) from the ISS environmental surfaces.

### Figure 1.

[![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0b2/9503880/49ba5f7a3c6c/microorganisms-10-01714-g001.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9503880_microorganisms-10-01714-g001.jpg)

[Open in a new tab](figure/microorganisms-10-01714-f001/)

Density distribution for overall food items (**A**), Earth-built-environment (**B**,**C**), and ISS surface (**D**,**E**) cultivable microbial counts for all media types. In food-item and Earth-built-environment testing (**A**–**C**), agar was compared with both Petrifilm and Easy Plates, while in ISS surface sampling (**D**,**E**), agar was only compared with Easy Plates.

### 3.1. Cultivable Population of Food Items

Aerobic bacterial counts for food items ranged from 102 to 109 cultivable microbes per gram ([Figure 1](#microorganisms-10-01714-f001)A). The counts correlated well between media types, with R2 values of 0.94 or greater and no significant difference between the mean bacterial cultivable microbial counts for each media type ([Table 1](#microorganisms-10-01714-t001), [Figure 2](#microorganisms-10-01714-f002)). The mean log10 differences between media types were <2.5 (i.e., less than 100.34) colonies in all cases. Paired Wilcoxon tests between film media and agar were not significant but were significant between Easy Plate and Petrifilm media, with Easy Plate having slightly higher cultivable microbial counts. When broken down by food category, only the microbial community of raw poultry did not show a high correlation between agar and film media counts. However, the correlation between Petrifilm and Easy Plate cultivable microbial counts was high for all food categories ([Table 2](#microorganisms-10-01714-t002)).

#### Table 1.

Results of paired and unpaired Wilcoxon tests and type II linear models comparing between media types. Bacterial-specific agar media were PCA for food items and built environment; R2A for ISS samples. Fungal-specific agar media were PDA w/chloramphenicol for all studies.

|  |  |  |  | Wilcoxon | | Type II Regression (MA) | | |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Media Target | Study | X | Y | Paired a | Unpaired | Intercept | Slope | R2 | Mean log10 Difference |
| Bacteria-specific | Food items | Agar | Easy Plate |  | -- | –0.36 | 1.05 | 0.94 | −0.143 |
|  | Food items | Agar | Petrifilm | -- | -- | 0.01 | 0.99 | 0.94 | −0.005 |
|  | Food items | Easy Plate | Petrifilm | \*\* | -- | 0.34 | 0.95 | 0.98 | 0.138 |
|  | Built Environment | Agar | Easy Plate | \*\* | -- | −0.002 | 0.93 | 0.96 | −0.186 |
|  | Built Environment | Agar | Petrifilm | -- | -- | 0.06 | 0.97 | 0.96 | −0.018 |
|  | Built Environment | Easy Plate | Petrifilm | \*\* | -- | 0.07 | 1.04 | 0.97 | 0.169 |
|  | ISS | Agar | Easy Plate | -- | -- | 0.06 | 0.74 | 0.80 | −0.044 |
| Fungal-specific | Built Environment | Agar | Easy Plate | -- | -- | 0.01 | 0.77 | 0.72 | −0.338 |
|  | Built Environment | Agar | Petrifilm | -- | -- | 0.05 | 0.75 | 0.73 | −0.29 |
|  | Built Environment | Easy Plate | Petrifilm | -- | -- | 0.16 | 0.9 | 0.85 | 0.048 |
|  | ISS | Agar | Easy Plate | \* | -- | 0.12 | 1.06 | 0.73 | −0.14 |

[Open in a new tab](table/microorganisms-10-01714-t001/)

a. \* corresponds to *p* < 0.05; \*\* corresponds to *p* < 0.01

#### Figure 2.

[![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0b2/9503880/3f3eeeabd67c/microorganisms-10-01714-g002.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9503880_microorganisms-10-01714-g002.jpg)

[Open in a new tab](figure/microorganisms-10-01714-f002/)

Correlation of agar and film media counts for PCA vs. Easy Plate AC, PCA vs. Petrifilm AC, and Easy Plate AC vs. Petrifilm AC for a variety of food items. Cultivable microbial counts are per milliliter, from swabbing food items and placing the swab in a 10 mL buffer solution. Solid blue line shows linear regression.

#### Table 2.

Paired Wilcoxon test results (P adjusted for multiple comparisons using FDR correction) and Pearson’s rho for bacterial counts for food samples, by food category.

|  |  | Pearson’s Rho | |
| --- | --- | --- | --- |
| Food Category | n | Easy Plate AC: PCA | Petrifilm AC: PCA |
| Multi-component foods or meal Components | 21 | 0.97 | 0.96 |
| Fresh produces and fruits | 20 | 0.96 | 0.97 |
| Raw meat and ready-to-cook meat products (except poultry) | 8 | 0.95 | 0.94 |
| Raw and ready-to-cook fish and seafood (unprocessed) | 5 | 0.99 | 0.99 |
| Raw poultry and ready-to-cook poultry products | 4 | 0.33 | 0.17 |
| Dried cereals, fruits, nuts, seeds, and vegetables | 4 | 1 | 1 |

[Open in a new tab](table/microorganisms-10-01714-t002/)

### 3.2. Cultivable Population of Built-Environment Surfaces

Cultivable microbial counts for both bacterial- and fungal-specific media ranged from below the detection limit to 106 per swab (per 100 cm2 surface area) for built-environment surface samples ([Figure 1](#microorganisms-10-01714-f001)B,C). There were no significant differences between mean cultivable microbial counts among the three media types, for either bacterial- or fungal-targeted media. In all cases, mean log10 differences between media types were less than 2.5 colonies (i.e., <100.34). Similar to the results for food items, the correlations between bacterial-specific film and agar media were high, with R2 values between 0.96 and 0.97, and no significant differences between the mean cultivable microbial counts were detected ([Table 1](#microorganisms-10-01714-t001), [Figure 3](#microorganisms-10-01714-f003)). The correlations between cultivable microbial counts for fungal-specific media were lower than those for bacteria, with R2 values of ~0.73 for both Petrifilm RYM and Easy Plate YM-R when correlated to PDA. Easy Plate YM-R and Petrifilm RYM had higher correlations between them (R2 = 0.85), but there was still considerable variation in their counts. Paired Wilcoxon tests showed that Easy Plate AC had slightly lower cultivable microbial counts than both PCA and Petrifilm AC, which, as stated previously, was not large enough to cause a difference in mean cultivable microbial counts. The mean log10 differences between the media types were <0.34 (i.e., a non-log difference of 2 or fewer colonies) in all cases.

#### Figure 3.

[![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0b2/9503880/68c709553d89/microorganisms-10-01714-g003.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9503880_microorganisms-10-01714-g003.jpg)

[Open in a new tab](figure/microorganisms-10-01714-f003/)

Cultivable microbial counts for built-environment surfaces in households in Japan for (**A**) bacterial-specific and (**B**) fungal-specific media, compared between film and agar media and media specificity type. PDA = potato dextrose agar; PCA = plate count agar. All cultivable microbial counts are per 100 cm2 of surface area. Blue solid line is linear regression; shading is 95% confidence interval.

### 3.3. Cultivable Population of the International Space Station

Cultivable microbial counts ranged from below the detection limit to 103 per wipe for both bacterial- ([Figure 1](#microorganisms-10-01714-f001)D) and fungal-specific ([Figure 1](#microorganisms-10-01714-f001)E) media (per 100 cm2 surface area). There was no difference between the mean cultivable microbial counts for Easy Plate AC vs. R2A, nor for Easy Plate YM-R vs. PDA ([Table 1](#microorganisms-10-01714-t001)). The cultivable microbial count correlations were similar in food and Earth built environments for fungi, with R2 values of 0.73 for fungal-specific media, but were lower than Earth-based studies for bacteria-specific media, with an R2 of 0.80 ([Figure 4](#microorganisms-10-01714-f004)A). This was due in part to the number of samples that had zero colonies for one or both media types ([Figure 4](#microorganisms-10-01714-f004)B). The samples plated on Easy Plates, both AC and YM, were more likely to have cultivable microbial counts above the detection limit compared with agar media ([Figure 4](#microorganisms-10-01714-f004)B).

#### Figure 4.

[![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0b2/9503880/3e41cfbb9651/microorganisms-10-01714-g004.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9503880_microorganisms-10-01714-g004.jpg)

[Open in a new tab](figure/microorganisms-10-01714-f004/)

(**A**) Correlations between EasyPlate vs. agar cultivable microbial counts for fungal-specific (PDA vs. EasyPlate YM-R) and bacterial-specific (R2A vs. Easy Plate AC) media for ISS samples. TNTC = too numerous to count. TNTCs are included as arbitrarily high numbers in this figure but are not included in model calculations; (**B**) number of samples with zero vs. non-zero values for agar media vs. Easy Plates for ISS samples.

### 3.4. Specificity of Media Types for Bacteria or Fungi

The differential isolation of bacteria and fungi by traditional agar and Easy Plate methods is shown in [Figure S1](#app1-microorganisms-10-01714). Among the 40 samples tested, 10 samples yielded at least one bacterial colony in Easy Plate AC but were not present in the R2A medium. In contrast, one R2A medium showed growth, whereas the Easy Plate AC method failed to yield any colony. The further isolation, purification, and identification of the bacteria grown in the R2A plate revealed the presence of *Sphingomonas lutea*. The pure strain of *S. lutea* isolate was regrown in the Easy Plate AC medium and showed growth. Similarly, among the 40 samples tested for fungal population estimation, 6 samples showed growth of at least one fungal colony in Easy Plate YM-R but did not show growth in the PDA medium. In contrast, two PDA plates showed growth, whereas the Easy Plate YM-R method failed to yield any colony. Further ITS-based identification using blast searches against the NCBI database revealed that these fungal isolates were closely related to *Aureobasidium leucospermi* and *Penicillium rubens*, and subsequent growth in Easy Plate YM-R yielded growth, revealing that there was no inhibition in growth in the Easy Plate YM-R medium.

For ISS samples, neither PDA + chloramphenicol agar media nor Easy Plate YM-R (which also contains chloramphenicol) isolated bacterial strains, but R2A and Easy Plate AC both promoted the growth of a significant number of fungi ([Figure 5](#microorganisms-10-01714-f005)B).

#### Figure 5.

[![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c0b2/9503880/55d3a5b2b026/microorganisms-10-01714-g005.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9503880_microorganisms-10-01714-g005.jpg)

[Open in a new tab](figure/microorganisms-10-01714-f005/)

(**A**) Distribution of bacterial species isolated from Easy Plate AC vs. R2A agar media; (**B**) distribution of fungal genera isolated from Easy Plate AC and YT, and R2A, and PDA agar media. Points are colored by genus.

### 3.5. Species and Genus Distributions for Bacterial and Fungal Isolates

The dominant bacterial species isolated from Easy Plate AC and R2A were similar, with multiple strains of *Staphylococcus* species and *Kalamiella piersonii* isolated by both types of media ([Figure 5](#microorganisms-10-01714-f005)A). However, after 7 days of growth, a single isolate of five slow-growing species was isolated from R2A (*Methylobacterium tolerans, Kocuria marina, Bacillus zhangzouensis, Psueodomonas parafulva*, and *Sphingomonas lutea*) but not from Easy Plate AC, while multiple strains of *Micrococcus aloverae* were isolated from Easy Plate AC but not from R2A. In addition, the16S rRNA gene-sequence-based species identity was at 97% or above for all the isolates and above 99% for most. Since the ITS is not suitable to define fungal species, the genus-level identification was recorded. The fungal identification for isolates ([Figure 5](#microorganisms-10-01714-f005)B) was similar across the board, with multiple strains of *Penicillium, Aspergillus*, and *Agicarus* isolated from R2A, PDA, and Easy Plate AC and YM-R. Easy Plate YM-R also isolated a single strain of *Cladosporium* and *Byssocorticium*, while PDA isolated a single strain of *Aureobasidium*. A more significant difference was the lack of the yeast genus *Rhodotorula* from Easy Plate AC isolates, whereas R2A plates promoted the growth of *Rhodotorula* species. We, therefore, plated a *Rhodotorula* isolate onto Easy Plate AC while also inoculating Easy Plate YM-R, R2A, and PDA with the same cell dilution for comparison. While *Rhodotorula* grew on Easy Plate AC medium, it was simply not visible to the eye, and the only way to detect it was to plate out random locations from the Easy Plate on different media. This was in contrast to non-yeast fungal species, which were highly visible on Easy Plate AC.

## 4. Discussion

In general, the use of film media over agar allows for lower costs, greatly increased inoculation speed, greatly reduced experimental preparation time, and much smaller amounts of space required in incubators or counters; these advantages have been discussed in numerous previous reports [[28](#B28-microorganisms-10-01714)]. When dealing with high volumes of samples that must be processed in a very short amount of time, film media such as Petrifilm and Easy Plates provide distinct advantages. In processing the ISS samples, plating the same number of samples on Easy Plates took one person 2 h, as opposed to three people taking 8 h to inoculate the agar plates. The use of film media or pour plates requires fewer dilutions than spread plates because it can produce accurate plate counts at higher bioburden levels, which is not necessarily a fair comparison of the time spent. However, in addition to a pipetting step, spread plates require either retrieving a pre-sterilized spreader from its packaging or sterilizing a reusable spreader and thoroughly spreading the sample over the surface without damaging the agar. For upwards of 300 agar plates per experiment, this adds up to multiple hours.

In this study, we found no particular advantage or disadvantage to using Petrifilm (3M) vs. Easy Plates (Kikkoman), except for the lack of necessity of using a spreader for Kikkoman plates. The cultivable microbial counts for built-environment samples and food samples correlated extremely well with counts from agar media and with each other. The one exception to this trend was the bacterial counts from raw poultry products, where Petrifilm and Easy Plate cultivable microbial counts correlated well with each other but did not correlate with agar; given the small number of poultry samples (n = 4), this trend would need to be investigated further to be considered significant.

The ranges of cultivable microbial counts for built-environment samples and ISS samples in this study were considerably lower, including many samples below the detection limit for one or both types of media, than almost all the previously published studies using film media in other environments. Most ISS samples had low cultivable microbial counts that were below the cultivable microbial count standard for hospital cleanliness (2.5 × 102 cultivable microbes per 100 cm2, OR 102.4 per 100 cm2) [[32](#B32-microorganisms-10-01714)]. The correlations between cultivable microbial counts for fungal-specific agar and film media were lower than the correlations for bacteria and ranged from adjusted R2 values 0.73 to 0.82 for built-environment samples and ISS samples. The correlation between bacterial-specific agar vs. film media was lower for ISS samples (adj R2 = 0.80) than Earth built environments (R2 = 0.94–0.98), probably due to both the low bioburden in ISS samples and the presence of considerable fungi growing on bacterial-specific media, which might have masked the presence of bacteria. It is noteworthy to mention here that the increase in inoculum size (1 mL) yielded cultivable counts more in film media (10 samples for bacteria and 6 samples for fungi; [Figure 5](#microorganisms-10-01714-f005)) than the traditional agar(s) used (100 µL).

The species distribution for film media vs. agar media was similar between both fungal-specific and bacteria-specific plates. Three main differences were that (1) yeast colonies grew but were not visible on Easy Plate AC (although non-yeast fungal colonies were visible and countable), whereas yeast colonies were of course visible on R2A; (2) Easy Plates isolated multiple strains of *Micrococcus*, whereas R2A did not; and (3) isolation from R2A allowed for the isolation of single strains of slow-growing low-nutrient-loving bacterial species not isolated on Easy Plates, from minuscule colonies. In some cases, the minuscule colony had a distinct color that allowed us to select it as a colony of interest from the agar, whereas on the Easy Plate, it was not distinguishable from other colonies; in some cases, there was only a single tiny colony on the agar plate vs. no colony on the Easy Plate, which could be due to random chance: In at least two cases, there would have been no way to pick the rare species out, and it was chance that led us to collect it. Inoculating well-grown colonies isolated from R2A onto Easy Plate AC medium allowed the growth of these minuscule colonies. Picking colonies out by their relative size on film media is difficult, as the dye used in Petrifilm and Easy Plates magnifies the size of tiny colonies. Additionally, without a clear view of colony morphology and color that agar provides, it is possible that isolating “rare” species might have been overlooked. This could be an advantage or disadvantage of film media depending on the goals and study design (it removes non-random sampling biases, but if one is sampling non-randomly, it may result in a lower diversity of isolates).

Other disadvantages of film media are that (1) because of the smaller surface area and the miniaturization of colonies, it was difficult to count colonies by eye for high bioburden samples, and in a few cases, counting from magnified photographs was necessary (3M has developed an automated CFU counter for use with Petrifilm plates, which negates this issue), and (2) because colony color was not observable, and morphology was somewhat obscured in film media, it would be extremely difficult to rule out the colonies of a known contaminant that appeared also in control plates (for example, in a situation where samples are plated out in the field or in otherwise non-laboratory conditions).

Surfaces in built environments inhabited by humans and animals are inherently difficult to consistently sample for cultivable microbial counts, due to having higher fungal:bacterial ratios and the presence of hairs and fibers, which adsorb bacterial and fungal spores, thus requiring multiple replicates to obtain an accurate count. Even with every effort made to vortex or otherwise homogenize samples before plating, in extremely low-biomass environments, the presence or absence of a single hair or fiber might make the difference between a zero and non-zero cultivable microbial count, or a lower or higher cultivable microbial count. Human habitation and the accompanying presence of dust, fibers, and hairs from bodies and cloth, combined with extremely low surface biomass due to constant cleaning and well-filtered air, is a combination peculiar to hospital environments [[40](#B40-microorganisms-10-01714)], clean rooms [[41](#B41-microorganisms-10-01714)], and space habitats [[42](#B42-microorganisms-10-01714),[43](#B43-microorganisms-10-01714)]; however, because astronauts do not wear masks, gloves, or other protective equipment aboard the ISS, the problem of hairs and fibers is exacerbated. One study tested the efficacy of Petrifilm vs. contact plates on regular hospital surfaces and found that Petrifilm detected larger numbers of cultivable microbes [[32](#B32-microorganisms-10-01714)]; however, cultivable microbial counts in that study were higher than what would be considered “low bioburden” (i.e., comparable to a clean room or the ISS). In a study of debris and lint collected via vacuuming the ISS surfaces, fibers and hairs—as opposed to tinier particles such as skin cells with a greater surface area—yielded a distinct microbial community with greater fungal diversity and only one order of magnitude fewer cultivable microbes than powdery debris despite a vastly smaller surface area [[42](#B42-microorganisms-10-01714)]. Therefore, the use of a larger volume is critical for accurate plate counts. As film media use a 10× higher volume than traditional spread plates, while not presenting the same challenges of possible heat death (molten agar at 55 °C) and excavation that isolating bacteria and fungi presents when using pour plates [[44](#B44-microorganisms-10-01714),[45](#B45-microorganisms-10-01714)], film media are well-suited for use with samples from the ISS and other future space habitats. In fact, because film media are extremely light, and thin, and rely on water cohesion and adhesion rather than gravity or manual spreading, they may be suitable for testing in microgravity in the future and may be a possible method for both food and environmental safety testing for future space habitats.

## Acknowledgments

Part of the research described was carried out at the Jet Propulsion Laboratory of the California Institute of Technology under a contract with NASA. We thank astronauts Akihiko Hoshide, Col. Shane Kimbrough, and Megan McArthur for collecting samples aboard the ISS, the Implementation Team (Gwo-Shing Sun, America Wang, and Elizabeth Pane) at the NASA Ames Research Center for coordinating this effort, and Jack Gilbert, UCSD, Principal Investigator of the team. ©2022 California Institute of Technology. Government sponsorship acknowledged.

## Supplementary Materials

The following supporting information can be downloaded at: <https://www.mdpi.com/article/10.3390/microorganisms10091714/s1>, Figure S1: Morphology of strains captured by film media but not agar media or vice versa; Table S1: CFU count data for food study; Table S2: CFU count data for built environment, Japan study.

[Click here for additional data file.](/articles/instance/9503880/bin/microorganisms-10-01714-s001.zip) (2.1MB, zip)

## Author Contributions

Conceptualization and funding acquisition, K.V. and T.S.; methodology, investigation, and validation, A.C.S., D.R.M., T.S. and K.V.; data curation, formal analysis, visualization, and writing—original draft preparation, A.C.S.; writing—review and editing, A.C.S. and K.V.; project administration, A.C.S. and K.V. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

Not applicable.

## Informed Consent Statement

Not applicable.

## Data Availability Statement

Raw CFU data are available under [Supplementary Table S1](#app1-microorganisms-10-01714) (food items), [Table S2](#app1-microorganisms-10-01714) (built environment). ISS data will be released upon publication of initial results for the Microbial Tracking 3 project.

## Conflicts of Interest

T.S. is employed by Kikkoman Corp. which produces and markets Easy Plate products. However, K.V. and A.C.S. declare that there are no conflicts of interest. This manuscript was prepared as an account of work sponsored by NASA, an agency of the US Government. The US Government, NASA, California Institute of Technology, Jet Propulsion Laboratory, and their employees make no warranty, expressed or implied, or assume any liability or responsibility for the accuracy, completeness, or usefulness of information, apparatus, product, or process disclosed in this manuscript, or represents that its use would not infringe upon privately held rights. The use of, and references to, any commercial product, process, or service does not necessarily constitute or imply endorsement, recommendation, or favoring by the US Government, NASA, California Institute of Technology, or Jet Propulsion Laboratory. The views and opinions presented herein by the authors of this manuscript do not necessarily reflect those of the US Government, NASA, California Institute of Technology, or Jet Propulsion Laboratory and shall not be used for advertisements or product endorsements.

## Funding Statement

The ISS portions of the research in this manuscript were funded by the NASA Space Biology grant 80NSSC19K1604 awarded to K.V. and the supported NASA postdoctoral program fellowship for A.S. Built-environment and food-item cultivable microbial count experiments were funded in-house by Kikkoman Corporation.

## Footnotes

**Publisher’s Note:** MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.

## References

* 1.NASA . Planetary Protection Provisions for Robotic Extraterrestrial Missions. NASA; Washington, DC, USA: 2011.  [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Planetary%20Protection%20Provisions%20for%20Robotic%20Extraterrestrial%20Missions&publication_year=2011&)]
* 2.Benardini J.N., 3rd, La Duc M.T., Beaudet R.A., Koukol R. Implementing planetary protection measures on the Mars Science Laboratory. Astrobiology. 2014;14:27–32. doi: 10.1089/ast.2013.0989. [[DOI](https://doi.org/10.1089/ast.2013.0989)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24432776/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Implementing%20planetary%20protection%20measures%20on%20the%20Mars%20Science%20Laboratory&author=J.N.%20Benardini&author=M.T.%20La%20Duc&author=R.A.%20Beaudet&author=R.%20Koukol&volume=14&publication_year=2014&pages=27-32&pmid=24432776&doi=10.1089/ast.2013.0989&)]
* 3.Checinska Sielaff A., Urbaniak C., Mohan G.B.M., Stepanov V.G., Tran Q., Wood J.M., Minich J., McDonald D., Mayer T., Knight R., et al. Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome. 2019;7:50. doi: 10.1186/s40168-019-0666-x. [[DOI](https://doi.org/10.1186/s40168-019-0666-x)] [[PMC free article](/articles/PMC6452512/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&author=A.%20Checinska%20Sielaff&author=C.%20Urbaniak&author=G.B.M.%20Mohan&author=V.G.%20Stepanov&author=Q.%20Tran&volume=7&publication_year=2019&pages=50&pmid=30955503&doi=10.1186/s40168-019-0666-x&)]
* 4.Ghosh S., Osman S., Vaishampayan P., Venkateswaran K. Recurrent isolation of extremotolerant bacteria from the clean room where Phoenix spacecraft components were assembled. Astrobiology. 2010;10:325–335. doi: 10.1089/ast.2009.0396. [[DOI](https://doi.org/10.1089/ast.2009.0396)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20446872/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Recurrent%20isolation%20of%20extremotolerant%20bacteria%20from%20the%20clean%20room%20where%20Phoenix%20spacecraft%20components%20were%20assembled&author=S.%20Ghosh&author=S.%20Osman&author=P.%20Vaishampayan&author=K.%20Venkateswaran&volume=10&publication_year=2010&pages=325-335&pmid=20446872&doi=10.1089/ast.2009.0396&)]
* 5.Kwan K., Cooper M., La Duc M.T., Vaishampayan P., Stam C., Benardini J.N., Scalzi G., Moissl-Eichinger C., Venkateswaran K. Evaluation of procedures for the collection, processing, and analysis of biomolecules from low-biomass surfaces. Appl. Environ. Microbiol. 2011;77:2943–2953. doi: 10.1128/AEM.02978-10. [[DOI](https://doi.org/10.1128/AEM.02978-10)] [[PMC free article](/articles/PMC3126404/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21398492/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=Evaluation%20of%20procedures%20for%20the%20collection,%20processing,%20and%20analysis%20of%20biomolecules%20from%20low-biomass%20surfaces&author=K.%20Kwan&author=M.%20Cooper&author=M.T.%20La%20Duc&author=P.%20Vaishampayan&author=C.%20Stam&volume=77&publication_year=2011&pages=2943-2953&pmid=21398492&doi=10.1128/AEM.02978-10&)]
* 6.La Duc M.T., Dekas A., Osman S., Moissl C., Newcombe D., Venkateswaran K. Isolation and characterization of bacteria capable of tolerating the extreme conditions of clean room environments. Appl. Environ. Microbiol. 2007;73:2600–2611. doi: 10.1128/AEM.03007-06. [[DOI](https://doi.org/10.1128/AEM.03007-06)] [[PMC free article](/articles/PMC1855582/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17308177/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=Isolation%20and%20characterization%20of%20bacteria%20capable%20of%20tolerating%20the%20extreme%20conditions%20of%20clean%20room%20environments&author=M.T.%20La%20Duc&author=A.%20Dekas&author=S.%20Osman&author=C.%20Moissl&author=D.%20Newcombe&volume=73&publication_year=2007&pages=2600-2611&pmid=17308177&doi=10.1128/AEM.03007-06&)]
* 7.Nirwati H., Sinanjung K., Fahrunissa F., Wijaya F., Napitupulu S., Hati V.P., Hakim M.S., Meliala A., Aman A.T., Nuryastuti T. Biofilm formation and antibiotic resistance of Klebsiella pneumoniae isolated from clinical samples in a tertiary care hospital, Klaten, Indonesia. BMC Proc. 2019;13:20. doi: 10.1186/s12919-019-0176-7. [[DOI](https://doi.org/10.1186/s12919-019-0176-7)] [[PMC free article](/articles/PMC6913045/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31890013/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Proc.&title=Biofilm%20formation%20and%20antibiotic%20resistance%20of%20Klebsiella%20pneumoniae%20isolated%20from%20clinical%20samples%20in%20a%20tertiary%20care%20hospital,%20Klaten,%20Indonesia&author=H.%20Nirwati&author=K.%20Sinanjung&author=F.%20Fahrunissa&author=F.%20Wijaya&author=S.%20Napitupulu&volume=13&publication_year=2019&pages=20&pmid=31890013&doi=10.1186/s12919-019-0176-7&)]
* 8.Sukhum K.V., Newcomer E.P., Cass C., Wallace M.A., Johnson C., Fine J., Sax S., Barlet M.H., Burnham C.-A.D., Dantas G., et al. Antibiotic-resistant organisms establish reservoirs in new hospital built environments and are related to patient blood infection isolates. Commun. Med. 2022;2:62. doi: 10.1038/s43856-022-00124-5. [[DOI](https://doi.org/10.1038/s43856-022-00124-5)] [[PMC free article](/articles/PMC9160058/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35664456/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Commun.%20Med.&title=Antibiotic-resistant%20organisms%20establish%20reservoirs%20in%20new%20hospital%20built%20environments%20and%20are%20related%20to%20patient%20blood%20infection%20isolates&author=K.V.%20Sukhum&author=E.P.%20Newcomer&author=C.%20Cass&author=M.A.%20Wallace&author=C.%20Johnson&volume=2&publication_year=2022&pages=62&pmid=35664456&doi=10.1038/s43856-022-00124-5&)]
* 9.Kot B., Piechota M., Jakubczak A., Gryzinska M., Witeska M., Gruzewska A., Baran K., Denkiewicz P. The prevalence of virulence determinants in methicillin-resistant Staphylococcus aureus isolated from different infections in hospitalized patients in Poland. Sci. Rep. 2022;12:5477. doi: 10.1038/s41598-022-09517-x. [[DOI](https://doi.org/10.1038/s41598-022-09517-x)] [[PMC free article](/articles/PMC8971418/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/35361858/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=The%20prevalence%20of%20virulence%20determinants%20in%20methicillin-resistant%20Staphylococcus%20aureus%20isolated%20from%20different%20infections%20in%20hospitalized%20patients%20in%20Poland&author=B.%20Kot&author=M.%20Piechota&author=A.%20Jakubczak&author=M.%20Gryzinska&author=M.%20Witeska&volume=12&publication_year=2022&pages=5477&pmid=35361858&doi=10.1038/s41598-022-09517-x&)]
* 10.Bruckner J., Osman S., Conley C., Venkateswaran K., Schaechter M. Space microbiology: Planetary protection, burden, diversity and significance of spacecraft associated microbes. Encycl. Microbiol. 2009;3:52–65. [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Encycl.%20Microbiol.&title=Space%20microbiology:%20Planetary%20protection,%20burden,%20diversity%20and%20significance%20of%20spacecraft%20associated%20microbes&author=J.%20Bruckner&author=S.%20Osman&author=C.%20Conley&author=K.%20Venkateswaran&author=M.%20Schaechter&volume=3&publication_year=2009&pages=52-65&)]
* 11.Rettberg P., Antunes A., Brucato J., Cabezas P., Collins G., Haddaji A., Kminek G., Leuko S., McKenna-Lawlor S., Moissl-Eichinger C., et al. Biological Contamination Prevention for Outer Solar System Moons of Astrobiological Interest: What Do We Need to Know? Astrobiology. 2019;19:951–974. doi: 10.1089/ast.2018.1996. [[DOI](https://doi.org/10.1089/ast.2018.1996)] [[PMC free article](/articles/PMC6767865/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30762429/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Biological%20Contamination%20Prevention%20for%20Outer%20Solar%20System%20Moons%20of%20Astrobiological%20Interest:%20What%20Do%20We%20Need%20to%20Know?&author=P.%20Rettberg&author=A.%20Antunes&author=J.%20Brucato&author=P.%20Cabezas&author=G.%20Collins&volume=19&publication_year=2019&pages=951-974&pmid=30762429&doi=10.1089/ast.2018.1996&)]
* 12.Spry J.A., Siegel B., Kminek G., Bakermans C., Benardini J.N., Beltran E., Bonaccorsi R., Canham J., Desjean M.-C., Doran P. Planetary protection knowledge gaps and enabling science for human Mars missions. Bull. Am. Astron. Soc. 2021;53:205. [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bull.%20Am.%20Astron.%20Soc.&title=Planetary%20protection%20knowledge%20gaps%20and%20enabling%20science%20for%20human%20Mars%20missions&author=J.A.%20Spry&author=B.%20Siegel&author=G.%20Kminek&author=C.%20Bakermans&author=J.N.%20Benardini&volume=53&publication_year=2021&pages=205&)]
* 13.Danko D.C., Sierra M.A., Benardini J.N., Guan L., Wood J.M., Singh N., Seuylemezian A., Butler D.J., Ryon K., Kuchin K. A comprehensive metagenomics framework to characterize organisms relevant for planetary protection. Microbiome. 2021;9:82. doi: 10.1186/s40168-021-01020-1. [[DOI](https://doi.org/10.1186/s40168-021-01020-1)] [[PMC free article](/articles/PMC8016160/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33795001/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=A%20comprehensive%20metagenomics%20framework%20to%20characterize%20organisms%20relevant%20for%20planetary%20protection&author=D.C.%20Danko&author=M.A.%20Sierra&author=J.N.%20Benardini&author=L.%20Guan&author=J.M.%20Wood&volume=9&publication_year=2021&pages=82&pmid=33795001&doi=10.1186/s40168-021-01020-1&)]
* 14.McCarron J.L., Keefe G.P., McKenna S.L.B., Dohoo I.R., Poole D.E. Laboratory evaluation of 3M Petrifilms and University of Minnesota Bi-plates as potential on-farm tests for clinical mastitis. J. Dairy Sci. 2009;92:2297–2305. doi: 10.3168/jds.2008-1661. [[DOI](https://doi.org/10.3168/jds.2008-1661)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/19389988/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Dairy%20Sci.&title=Laboratory%20evaluation%20of%203M%20Petrifilms%20and%20University%20of%20Minnesota%20Bi-plates%20as%20potential%20on-farm%20tests%20for%20clinical%20mastitis&author=J.L.%20McCarron&author=G.P.%20Keefe&author=S.L.B.%20McKenna&author=I.R.%20Dohoo&author=D.E.%20Poole&volume=92&publication_year=2009&pages=2297-2305&pmid=19389988&doi=10.3168/jds.2008-1661&)]
* 15.3M Petrifilm Plates Brochure. 2021. [(accessed on 3 June 2022)]. Available online: <https://multimedia.3m.com/mws/media/1505478O/3m-petrifilm-plates-brochure.pdf>.
* 16.Kikkoman Corporation Microbial Film Media (Easy Plate) [(accessed on 3 June 2022)]. Available online: <https://biochemifa.kikkoman.com/e/kit/product/easy-plate/>
* 17.Ginn R.E., Packard V.S., Fox T.L. Evaluation of the 3M Dry Medium Culture Plate (Petrifilm SM) Method for Determining Numbers of Bacteria in Raw Milk (1) J. Food Prot. 1984;47:753–755. doi: 10.4315/0362-028X-47.10.753. [[DOI](https://doi.org/10.4315/0362-028X-47.10.753)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30934511/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Food%20Prot.&title=Evaluation%20of%20the%203M%20Dry%20Medium%20Culture%20Plate%20(Petrifilm%20SM)%20Method%20for%20Determining%20Numbers%20of%20Bacteria%20in%20Raw%20Milk%20(1)&author=R.E.%20Ginn&author=V.S.%20Packard&author=T.L.%20Fox&volume=47&publication_year=1984&pages=753-755&pmid=30934511&doi=10.4315/0362-028X-47.10.753&)]
* 18.Bailey J.S., Cox N.A. Evaluation of the Petrifilm Sm and Vrb Dry Media Culture Plates for Determining Microbial Quality of Poultry. J. Food Prot. 1987;50:643–644. doi: 10.4315/0362-028X-50.8.643. [[DOI](https://doi.org/10.4315/0362-028X-50.8.643)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30965464/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Food%20Prot.&title=Evaluation%20of%20the%20Petrifilm%20Sm%20and%20Vrb%20Dry%20Media%20Culture%20Plates%20for%20Determining%20Microbial%20Quality%20of%20Poultry&author=J.S.%20Bailey&author=N.A.%20Cox&volume=50&publication_year=1987&pages=643-644&pmid=30965464&doi=10.4315/0362-028X-50.8.643&)]
* 19.Beuchat L.R., Nail B.V., Brackett R.E., Fox T.L. Evaluation of a Culture Film (Petrifilm() YM) Method for Enumerating Yeasts and Molds in Selected Dairy and High-Acid Foods. J. Food Prot. 1990;53:869–874. doi: 10.4315/0362-028X-53.10.869. [[DOI](https://doi.org/10.4315/0362-028X-53.10.869)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31018288/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Food%20Prot.&title=Evaluation%20of%20a%20Culture%20Film%20(Petrifilm()%20YM)%20Method%20for%20Enumerating%20Yeasts%20and%20Molds%20in%20Selected%20Dairy%20and%20High-Acid%20Foods&author=L.R.%20Beuchat&author=B.V.%20Nail&author=R.E.%20Brackett&author=T.L.%20Fox&volume=53&publication_year=1990&pages=869-874&pmid=31018288&doi=10.4315/0362-028X-53.10.869&)]
* 20.Beuchat L.R., Nail B.V., Brackett R.E., Fox T.L. Comparison of the Petrifilm™ Yeast and Mold Culture Film Method to Conventional Methods for Enumerating Yeasts and Molds in Foods. J. Food Prot. 1991;54:443–447. doi: 10.4315/0362-028X-54.6.443. [[DOI](https://doi.org/10.4315/0362-028X-54.6.443)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31051612/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Food%20Prot.&title=Comparison%20of%20the%20Petrifilm%E2%84%A2%20Yeast%20and%20Mold%20Culture%20Film%20Method%20to%20Conventional%20Methods%20for%20Enumerating%20Yeasts%20and%20Molds%20in%20Foods&author=L.R.%20Beuchat&author=B.V.%20Nail&author=R.E.%20Brackett&author=T.L.%20Fox&volume=54&publication_year=1991&pages=443-447&pmid=31051612&doi=10.4315/0362-028X-54.6.443&)]
* 21.Schraft H., Watterworth L.A. Enumeration of heterotrophs, fecal coliforms and Escherichia coli in water: Comparison of 3M Petrifilm plates with standard plating procedures. J. Microbiol. Methods. 2005;60:335–342. doi: 10.1016/j.mimet.2004.10.008. [[DOI](https://doi.org/10.1016/j.mimet.2004.10.008)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15649535/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Microbiol.%20Methods&title=Enumeration%20of%20heterotrophs,%20fecal%20coliforms%20and%20Escherichia%20coli%20in%20water:%20Comparison%20of%203M%20Petrifilm%20plates%20with%20standard%20plating%20procedures&author=H.%20Schraft&author=L.A.%20Watterworth&volume=60&publication_year=2005&pages=335-342&pmid=15649535&doi=10.1016/j.mimet.2004.10.008&)]
* 22.Samarajeewa A.D., Glasauer S.M., Dunfield K.E. Evaluation of Petrifilm EC method for enumeration of E. coli from soil. Lett. Appl. Microbiol. 2010;50:457–461. doi: 10.1111/j.1472-765X.2010.02819.x. [[DOI](https://doi.org/10.1111/j.1472-765X.2010.02819.x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/20184673/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lett.%20Appl.%20Microbiol.&title=Evaluation%20of%20Petrifilm%20EC%20method%20for%20enumeration%20of%20E.%20coli%20from%20soil&author=A.D.%20Samarajeewa&author=S.M.%20Glasauer&author=K.E.%20Dunfield&volume=50&publication_year=2010&pages=457-461&pmid=20184673&doi=10.1111/j.1472-765X.2010.02819.x&)]
* 23.Nelson M.T., LaBudde R.A., Tomasino S.F., Pines R.M. Comparison of 3M Petrifilm Aerobic Count Plates to standard plating methodology for use with AOAC antimicrobial efficacy methods 955.14, 955.15, 964.02, and 966.04 as an alternative enumeration procedure: Collaborative study. J. AOAC Int. 2013;96:717–722. doi: 10.5740/jaoacint.12-469. [[DOI](https://doi.org/10.5740/jaoacint.12-469)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24000742/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20AOAC%20Int.&title=Comparison%20of%203M%20Petrifilm%20Aerobic%20Count%20Plates%20to%20standard%20plating%20methodology%20for%20use%20with%20AOAC%20antimicrobial%20efficacy%20methods%20955.14,%20955.15,%20964.02,%20and%20966.04%20as%20an%20alternative%20enumeration%20procedure:%20Collaborative%20study&author=M.T.%20Nelson&author=R.A.%20LaBudde&author=S.F.%20Tomasino&author=R.M.%20Pines&volume=96&publication_year=2013&pages=717-722&pmid=24000742&doi=10.5740/jaoacint.12-469&)]
* 24.Fritz B.G., Walker D.K., Goveia D.E., Parker A.E., Goeres D.M. Evaluation of Petrifilm aerobic count plates as an equivalent alternative to drop plating on R2A agar plates in a biofilm disinfectant efficacy test. Curr. Microbiol. 2015;70:450–456. doi: 10.1007/s00284-014-0738-x. [[DOI](https://doi.org/10.1007/s00284-014-0738-x)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25471267/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Microbiol.&title=Evaluation%20of%20Petrifilm%20aerobic%20count%20plates%20as%20an%20equivalent%20alternative%20to%20drop%20plating%20on%20R2A%20agar%20plates%20in%20a%20biofilm%20disinfectant%20efficacy%20test&author=B.G.%20Fritz&author=D.K.%20Walker&author=D.E.%20Goveia&author=A.E.%20Parker&author=D.M.%20Goeres&volume=70&publication_year=2015&pages=450-456&pmid=25471267&doi=10.1007/s00284-014-0738-x&)]
* 25.Bird P., Flannery J., Crowley E., Agin J., Goins D., Jechorek R. Evaluation of the 3M Petrifilm Rapid Aerobic Count Plate for the Enumeration of Aerobic Bacteria: Collaborative Study, First Action 2015.13. J. AOAC Int. 2016;99:664–675. doi: 10.5740/jaoacint.15-0260. [[DOI](https://doi.org/10.5740/jaoacint.15-0260)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27297837/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20AOAC%20Int.&title=Evaluation%20of%20the%203M%20Petrifilm%20Rapid%20Aerobic%20Count%20Plate%20for%20the%20Enumeration%20of%20Aerobic%20Bacteria:%20Collaborative%20Study,%20First%20Action%202015.13&author=P.%20Bird&author=J.%20Flannery&author=E.%20Crowley&author=J.%20Agin&author=D.%20Goins&volume=99&publication_year=2016&pages=664-675&pmid=27297837&doi=10.5740/jaoacint.15-0260&)]
* 26.Morin M., Dubuc J., Freycon P., Buczinski S. Diagnostic accuracy of the Petrifilm culture system for identifying colostrum with excessive bacterial contamination in Quebec dairy herds. J. Dairy Sci. 2021;104:4923–4928. doi: 10.3168/jds.2020-19474. [[DOI](https://doi.org/10.3168/jds.2020-19474)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33358819/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Dairy%20Sci.&title=Diagnostic%20accuracy%20of%20the%20Petrifilm%20culture%20system%20for%20identifying%20colostrum%20with%20excessive%20bacterial%20contamination%20in%20Quebec%20dairy%20herds&author=M.%20Morin&author=J.%20Dubuc&author=P.%20Freycon&author=S.%20Buczinski&volume=104&publication_year=2021&pages=4923-4928&pmid=33358819&doi=10.3168/jds.2020-19474&)]
* 27.Kabera F., Dufour S., Keefe G., Cameron M., Roy J.-P. Evaluation of quarter-based selective dry cow therapy using Petrifilm on-farm milk culture: A randomized controlled trial. J. Dairy Sci. 2020;103:7276–7287. doi: 10.3168/jds.2019-17438. [[DOI](https://doi.org/10.3168/jds.2019-17438)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32505410/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Dairy%20Sci.&title=Evaluation%20of%20quarter-based%20selective%20dry%20cow%20therapy%20using%20Petrifilm%20on-farm%20milk%20culture:%20A%20randomized%20controlled%20trial&author=F.%20Kabera&author=S.%20Dufour&author=G.%20Keefe&author=M.%20Cameron&author=J.-P.%20Roy&volume=103&publication_year=2020&pages=7276-7287&pmid=32505410&doi=10.3168/jds.2019-17438&)]
* 28.Rojas A., Murphy S., Wiedmann M., Martin N. Coliform Petrifilm as an alternative method for detecting total gram-negative bacteria in fluid milk. J. Dairy Sci. 2020;103:5043–5046. doi: 10.3168/jds.2019-17792. [[DOI](https://doi.org/10.3168/jds.2019-17792)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32307175/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Dairy%20Sci.&title=Coliform%20Petrifilm%20as%20an%20alternative%20method%20for%20detecting%20total%20gram-negative%20bacteria%20in%20fluid%20milk&author=A.%20Rojas&author=S.%20Murphy&author=M.%20Wiedmann&author=N.%20Martin&volume=103&publication_year=2020&pages=5043-5046&pmid=32307175&doi=10.3168/jds.2019-17792&)]
* 29.Kock J., Wente N., Zhang Y., Paduch J.-H., Leimbach S., Krömker V. Accuracy of 12h-Petrifilm™-plates as a rapid on-farm test for evidence-based mastitis therapy on a dairy farm in Germany. Milk Sci. Int.-Milchwiss. 2018;71:10–13. [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Milk%20Sci.%20Int.-Milchwiss.&title=Accuracy%20of%2012h-Petrifilm%E2%84%A2-plates%20as%20a%20rapid%20on-farm%20test%20for%20evidence-based%20mastitis%20therapy%20on%20a%20dairy%20farm%20in%20Germany&author=J.%20Kock&author=N.%20Wente&author=Y.%20Zhang&author=J.-H.%20Paduch&author=S.%20Leimbach&volume=71&publication_year=2018&pages=10-13&)]
* 30.Nygaard A.B., Charnock C. Longitudinal development of the dust microbiome in a newly opened Norwegian kindergarten. Microbiome. 2018;6:159. doi: 10.1186/s40168-018-0553-x. [[DOI](https://doi.org/10.1186/s40168-018-0553-x)] [[PMC free article](/articles/PMC6138906/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30219104/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Longitudinal%20development%20of%20the%20dust%20microbiome%20in%20a%20newly%20opened%20Norwegian%20kindergarten&author=A.B.%20Nygaard&author=C.%20Charnock&volume=6&publication_year=2018&pages=159&pmid=30219104&doi=10.1186/s40168-018-0553-x&)]
* 31.Dean Z., Stott K., Schubert W., Seto E.P., Chandrapati S. Qualification of 3M Petrifilm Plates in Rapidly Estimating Bacterial Burden for Planetary Protection Flight Implementation. 2022.
  unpublished
  .
* 32.Claro T., O’Reilly M., Daniels S., Humphreys H. Surface microbial contamination in hospitals: A pilot study on methods of sampling and the use of proposed microbiologic standards. Am. J. Infect. Control. 2015;43:1000–1002. doi: 10.1016/j.ajic.2015.05.009. [[DOI](https://doi.org/10.1016/j.ajic.2015.05.009)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26082261/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Infect.%20Control&title=Surface%20microbial%20contamination%20in%20hospitals:%20A%20pilot%20study%20on%20methods%20of%20sampling%20and%20the%20use%20of%20proposed%20microbiologic%20standards&author=T.%20Claro&author=M.%20O%E2%80%99Reilly&author=S.%20Daniels&author=H.%20Humphreys&volume=43&publication_year=2015&pages=1000-1002&pmid=26082261&doi=10.1016/j.ajic.2015.05.009&)]
* 33.Hooker E.A., Allen S., Gray L., Kaufman C. A randomized trial to evaluate a launderable bed protection system for hospital beds. Antimicrob. Resist. Infect. Control. 2012;1:27. doi: 10.1186/2047-2994-1-27. [[DOI](https://doi.org/10.1186/2047-2994-1-27)] [[PMC free article](/articles/PMC3441859/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22958605/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Resist.%20Infect.%20Control&title=A%20randomized%20trial%20to%20evaluate%20a%20launderable%20bed%20protection%20system%20for%20hospital%20beds&author=E.A.%20Hooker&author=S.%20Allen&author=L.%20Gray&author=C.%20Kaufman&volume=1&publication_year=2012&pages=27&pmid=22958605&doi=10.1186/2047-2994-1-27&)]
* 34.Maturin L., Peeler J. BAM: Aerobic Plate Count. US Food and Drug Administration; Silver Spring, MD, USA: 2001.  [[Google Scholar](https://scholar.google.com/scholar_lookup?title=BAM:%20Aerobic%20Plate%20Count&author=L.%20Maturin&author=J.%20Peeler&publication_year=2001&)]
* 35.Singh N.K., Wood J.M., Karouia F., Venkateswaran K. Succession and persistence of microbial communities and antimicrobial resistance genes associated with International Space Station environmental surfaces. Microbiome. 2018;6:204. doi: 10.1186/s40168-018-0585-2. [[DOI](https://doi.org/10.1186/s40168-018-0585-2)] [[PMC free article](/articles/PMC6234677/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30424821/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Succession%20and%20persistence%20of%20microbial%20communities%20and%20antimicrobial%20resistance%20genes%20associated%20with%20International%20Space%20Station%20environmental%20surfaces&author=N.K.%20Singh&author=J.M.%20Wood&author=F.%20Karouia&author=K.%20Venkateswaran&volume=6&publication_year=2018&pages=204&pmid=30424821&doi=10.1186/s40168-018-0585-2&)]
* 36.Morrison M.D., Thissen J.B., Karouia F., Mehta S., Urbaniak C., Venkateswaran K., Smith D.J., Jaing C. Investigation of Spaceflight Induced Changes to Astronaut Microbiomes. Front. Microbiol. 2021;12:659179. doi: 10.3389/fmicb.2021.659179. [[DOI](https://doi.org/10.3389/fmicb.2021.659179)] [[PMC free article](/articles/PMC8207296/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/34149649/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&title=Investigation%20of%20Spaceflight%20Induced%20Changes%20to%20Astronaut%20Microbiomes&author=M.D.%20Morrison&author=J.B.%20Thissen&author=F.%20Karouia&author=S.%20Mehta&author=C.%20Urbaniak&volume=12&publication_year=2021&pages=659179&pmid=34149649&doi=10.3389/fmicb.2021.659179&)]
* 37.Lane D. Nucleic Acid Techniques in Bacterial Systematics. Wiley; Hoboken, NJ, USA: 1991. 16S/23S rRNA sequencing; pp. 115–175. [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Nucleic%20Acid%20Techniques%20in%20Bacterial%20Systematics&author=D.%20Lane&publication_year=1991&)]
* 38.Toju H., Tanabe A.S., Yamamoto S., Sato H. High-coverage ITS primers for the DNA-based identification of ascomycetes and basidiomycetes in environmental samples. PLoS ONE. 2012;7:e40863. doi: 10.1371/journal.pone.0040863. [[DOI](https://doi.org/10.1371/journal.pone.0040863)] [[PMC free article](/articles/PMC3395698/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22808280/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=High-coverage%20ITS%20primers%20for%20the%20DNA-based%20identification%20of%20ascomycetes%20and%20basidiomycetes%20in%20environmental%20samples&author=H.%20Toju&author=A.S.%20Tanabe&author=S.%20Yamamoto&author=H.%20Sato&volume=7&publication_year=2012&pages=e40863&pmid=22808280&doi=10.1371/journal.pone.0040863&)]
* 39.Chao K.-H., Barton K., Palmer S., Lanfear R. sangeranalyseR: Simple and interactive processing of Sanger sequencing data in R. Genome Biol. Evol. 2021;13:evab028. doi: 10.1093/gbe/evab028. [[DOI](https://doi.org/10.1093/gbe/evab028)] [[PMC free article](/articles/PMC7939931/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33591316/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&title=sangeranalyseR:%20Simple%20and%20interactive%20processing%20of%20Sanger%20sequencing%20data%20in%20R&author=K.-H.%20Chao&author=K.%20Barton&author=S.%20Palmer&author=R.%20Lanfear&volume=13&publication_year=2021&pages=evab028&pmid=33591316&doi=10.1093/gbe/evab028&)]
* 40.Thiele R.H., Huffmyer J.L., Nemergut E.C. The “six sigma approach” to the operating room environment and infection. Best Pract. Res. Clin. Anaesthesiol. 2008;22:537–552. doi: 10.1016/j.bpa.2008.06.002. [[DOI](https://doi.org/10.1016/j.bpa.2008.06.002)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18831302/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Best%20Pract.%20Res.%20Clin.%20Anaesthesiol.&title=The%20%E2%80%9Csix%20sigma%20approach%E2%80%9D%20to%20the%20operating%20room%20environment%20and%20infection&author=R.H.%20Thiele&author=J.L.%20Huffmyer&author=E.C.%20Nemergut&volume=22&publication_year=2008&pages=537-552&pmid=18831302&doi=10.1016/j.bpa.2008.06.002&)]
* 41.Checinska A., Probst A.J., Vaishampayan P., White J.R., Kumar D., Stepanov V.G., Fox G.E., Nilsson H.R., Pierson D.L., Perry J., et al. Microbiomes of the dust particles collected from the International Space Station and Spacecraft Assembly Facilities. Microbiome. 2015;3:50. doi: 10.1186/s40168-015-0116-3. [[DOI](https://doi.org/10.1186/s40168-015-0116-3)] [[PMC free article](/articles/PMC4624184/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26502721/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Microbiomes%20of%20the%20dust%20particles%20collected%20from%20the%20International%20Space%20Station%20and%20Spacecraft%20Assembly%20Facilities&author=A.%20Checinska&author=A.J.%20Probst&author=P.%20Vaishampayan&author=J.R.%20White&author=D.%20Kumar&volume=3&publication_year=2015&pages=50&pmid=26502721&doi=10.1186/s40168-015-0116-3&)]
* 42.Venkateswaran K., Vaishampayan P., Cisneros J., Pierson D.L., Rogers S.O., Perry J. International Space Station environmental microbiome—Microbial inventories of ISS filter debris. Appl. Microbiol. Biotechnol. 2014;98:6453–6466. doi: 10.1007/s00253-014-5650-6. [[DOI](https://doi.org/10.1007/s00253-014-5650-6)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24695826/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&title=International%20Space%20Station%20environmental%20microbiome%E2%80%94Microbial%20inventories%20of%20ISS%20filter%20debris&author=K.%20Venkateswaran&author=P.%20Vaishampayan&author=J.%20Cisneros&author=D.L.%20Pierson&author=S.O.%20Rogers&volume=98&publication_year=2014&pages=6453-6466&pmid=24695826&doi=10.1007/s00253-014-5650-6&)]
* 43.Mora M., Mahnert A., Koskinen K., Pausan M.R., Oberauner-Wappis L., Krause R., Perras A.K., Gorkiewicz G., Berg G., Moissl-Eichinger C. Microorganisms in Confined Habitats: Microbial Monitoring and Control of Intensive Care Units, Operating Rooms, Cleanrooms and the International Space Station. Front. Microbiol. 2016;7:1573. doi: 10.3389/fmicb.2016.01573. [[DOI](https://doi.org/10.3389/fmicb.2016.01573)] [[PMC free article](/articles/PMC5061736/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27790191/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&title=Microorganisms%20in%20Confined%20Habitats:%20Microbial%20Monitoring%20and%20Control%20of%20Intensive%20Care%20Units,%20Operating%20Rooms,%20Cleanrooms%20and%20the%20International%20Space%20Station&author=M.%20Mora&author=A.%20Mahnert&author=K.%20Koskinen&author=M.R.%20Pausan&author=L.%20Oberauner-Wappis&volume=7&publication_year=2016&pages=1573&pmid=27790191&doi=10.3389/fmicb.2016.01573&)]
* 44.Taylor R.H., Allen M.J., Geldreich E.E. Standard plate count: A comparison of pour plate and spread plate methods. J.-Am. Water Work. Assoc. 1983;75:35–37. doi: 10.1002/j.1551-8833.1983.tb05055.x. [[DOI](https://doi.org/10.1002/j.1551-8833.1983.tb05055.x)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.-Am.%20Water%20Work.%20Assoc.&title=Standard%20plate%20count:%20A%20comparison%20of%20pour%20plate%20and%20spread%20plate%20methods&author=R.H.%20Taylor&author=M.J.%20Allen&author=E.E.%20Geldreich&volume=75&publication_year=1983&pages=35-37&doi=10.1002/j.1551-8833.1983.tb05055.x&)]
* 45.Van Soestbergen A., Lee C.H. Pour plates or streak plates? Appl. Microbiol. 1969;18:1092–1093. doi: 10.1128/am.18.6.1092-1093.1969. [[DOI](https://doi.org/10.1128/am.18.6.1092-1093.1969)] [[PMC free article](/articles/PMC378201/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/4905698/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.&title=Pour%20plates%20or%20streak%20plates?&author=A.%20Van%20Soestbergen&author=C.H.%20Lee&volume=18&publication_year=1969&pages=1092-1093&pmid=4905698&doi=10.1128/am.18.6.1092-1093.1969&)]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

[Click here for additional data file.](/articles/instance/9503880/bin/microorganisms-10-01714-s001.zip) (2.1MB, zip)

### Data Availability Statement

Raw CFU data are available under [Supplementary Table S1](#app1-microorganisms-10-01714) (food items), [Table S2](#app1-microorganisms-10-01714) (built environment). ISS data will be released upon publication of initial results for the Microbial Tracking 3 project.