<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11930778/"/><title>Possible roles of CAHS proteins from Tardigrade in osmotic stress tolerance in mammalian cells - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Cell Struct Funct</button></div>. 2024 Nov 19;49(2):123–133. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1247/csf.24035" rel="noopener noreferrer" target="_blank">10.1247/csf.24035</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Possible roles of CAHS proteins from Tardigrade in osmotic stress tolerance in mammalian cells</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bino%20T%22%5BAuthor%5D"><span class="name western">Takahiro Bino</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Takahiro Bino</span></h3>
<div class="p">
<sup>1</sup>
Division of Quantitative Biology, National Institute for Basic Biology, National Institutes of Natural Sciences, 5-1 Higashiyama, Myodaiji-cho, Okazaki, Aichi 444-8787, Japan</div>
<div class="p">
<sup>2</sup>
Quantitative Biology Research Group, Exploratory Research Center on Life and Living Systems (ExCELLS), National Institutes of Natural Sciences, 5-1 Higashiyama, Myodaiji-cho, Okazaki, Aichi 444-8787, Japan</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bino%20T%22%5BAuthor%5D"><span class="name western">Takahiro Bino</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goto%20Y%22%5BAuthor%5D"><span class="name western">Yuhei Goto</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yuhei Goto</span></h3>
<div class="p">
<sup>3</sup>
Department of Basic Biology, School of Life Science, SOKENDAI (The Graduate University for Advanced Studies), 5-1 Higashiyama, Myodaiji-cho, Okazaki, Aichi 444-8787, Japan</div>
<div class="p">
<sup>4</sup>
Laboratory of Cell Cycle Regulation, Graduate School of Biostudies, Kyoto University, Yoshidakonoe-cho, Sakyo-ku, Kyoto, Kyoto 606-8501, Japan</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goto%20Y%22%5BAuthor%5D"><span class="name western">Yuhei Goto</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maryu%20G%22%5BAuthor%5D"><span class="name western">Gembu Maryu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Gembu Maryu</span></h3>
<div class="p">
<sup>1</sup>
Division of Quantitative Biology, National Institute for Basic Biology, National Institutes of Natural Sciences, 5-1 Higashiyama, Myodaiji-cho, Okazaki, Aichi 444-8787, Japan</div>
<div class="p">
<sup>2</sup>
Quantitative Biology Research Group, Exploratory Research Center on Life and Living Systems (ExCELLS), National Institutes of Natural Sciences, 5-1 Higashiyama, Myodaiji-cho, Okazaki, Aichi 444-8787, Japan</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maryu%20G%22%5BAuthor%5D"><span class="name western">Gembu Maryu</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arakawa%20K%22%5BAuthor%5D"><span class="name western">Kazuharu Arakawa</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kazuharu Arakawa</span></h3>
<div class="p">
<sup>2</sup>
Quantitative Biology Research Group, Exploratory Research Center on Life and Living Systems (ExCELLS), National Institutes of Natural Sciences, 5-1 Higashiyama, Myodaiji-cho, Okazaki, Aichi 444-8787, Japan</div>
<div class="p">
<sup>5</sup>
Institute for Advanced Biosciences, Keio University, Nihonkoku 403-1, Daihoji, Tsuruoka, Yamagata, 997-0017, Japan</div>
<div class="p">
<sup>6</sup>
Graduate School of Media and Governance, Keio University, Endo 5322, Fujisawa, Kanagawa 252-0882, Japan</div>
<div class="p">
<sup>7</sup>
Faculty of Environment and Information Studies, Keio University, Endo 5322, Fujisawa, Kanagawa 252-0882, Japan</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arakawa%20K%22%5BAuthor%5D"><span class="name western">Kazuharu Arakawa</span></a>
</div>
</div>
<sup>2,</sup><sup>5,</sup><sup>6,</sup><sup>7</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aoki%20K%22%5BAuthor%5D"><span class="name western">Kazuhiro Aoki</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Kazuhiro Aoki</span></h3>
<div class="p">
<sup>1</sup>
Division of Quantitative Biology, National Institute for Basic Biology, National Institutes of Natural Sciences, 5-1 Higashiyama, Myodaiji-cho, Okazaki, Aichi 444-8787, Japan</div>
<div class="p">
<sup>2</sup>
Quantitative Biology Research Group, Exploratory Research Center on Life and Living Systems (ExCELLS), National Institutes of Natural Sciences, 5-1 Higashiyama, Myodaiji-cho, Okazaki, Aichi 444-8787, Japan</div>
<div class="p">
<sup>3</sup>
Department of Basic Biology, School of Life Science, SOKENDAI (The Graduate University for Advanced Studies), 5-1 Higashiyama, Myodaiji-cho, Okazaki, Aichi 444-8787, Japan</div>
<div class="p">
<sup>4</sup>
Laboratory of Cell Cycle Regulation, Graduate School of Biostudies, Kyoto University, Yoshidakonoe-cho, Sakyo-ku, Kyoto, Kyoto 606-8501, Japan</div>
<div class="p">
<sup>8</sup>
Center for Living Systems Information Science, Graduate School of Biostudies, Kyoto University, Yoshidakonoe-cho, Sakyo-ku, Kyoto, Kyoto 606-8501, Japan</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aoki%20K%22%5BAuthor%5D"><span class="name western">Kazuhiro Aoki</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>8,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>
Division of Quantitative Biology, National Institute for Basic Biology, National Institutes of Natural Sciences, 5-1 Higashiyama, Myodaiji-cho, Okazaki, Aichi 444-8787, Japan</div>
<div id="aff2">
<sup>2</sup>
Quantitative Biology Research Group, Exploratory Research Center on Life and Living Systems (ExCELLS), National Institutes of Natural Sciences, 5-1 Higashiyama, Myodaiji-cho, Okazaki, Aichi 444-8787, Japan</div>
<div id="aff3">
<sup>3</sup>
Department of Basic Biology, School of Life Science, SOKENDAI (The Graduate University for Advanced Studies), 5-1 Higashiyama, Myodaiji-cho, Okazaki, Aichi 444-8787, Japan</div>
<div id="aff4">
<sup>4</sup>
Laboratory of Cell Cycle Regulation, Graduate School of Biostudies, Kyoto University, Yoshidakonoe-cho, Sakyo-ku, Kyoto, Kyoto 606-8501, Japan</div>
<div id="aff5">
<sup>5</sup>
Institute for Advanced Biosciences, Keio University, Nihonkoku 403-1, Daihoji, Tsuruoka, Yamagata, 997-0017, Japan</div>
<div id="aff6">
<sup>6</sup>
Graduate School of Media and Governance, Keio University, Endo 5322, Fujisawa, Kanagawa 252-0882, Japan</div>
<div id="aff7">
<sup>7</sup>
Faculty of Environment and Information Studies, Keio University, Endo 5322, Fujisawa, Kanagawa 252-0882, Japan</div>
<div id="aff8">
<sup>8</sup>
Center for Living Systems Information Science, Graduate School of Biostudies, Kyoto University, Yoshidakonoe-cho, Sakyo-ku, Kyoto, Kyoto 606-8501, Japan</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">Correspondence: Kazuhiro Aoki, Laboratory of Cell Cycle Regulation, Graduate School of Biostudies, Kyoto University, Yoshidakonoe-cho, Sakyo-ku, Kyoto, Kyoto 606-8501, Japan. Tel: +81-75-753-4195, E-mail: <span>aoki.kazuhiro.6v@kyoto-u.ac.jp</span></p>
</div></div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2024 Jun 14; Accepted 2024 Nov 3; Issue date 2024.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright 2024 The Author(s)</div>
<p>This is an open access article distributed under the terms of the Creative Commons BY (Attribution) License (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">https://creativecommons.org/licenses/by/4.0/legalcode</a>), which permits the unrestricted distribution, reproduction and use of the article provided the original source and authors are credited.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11930778  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/39566968/">39566968</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Anhydrobiosis, a phenomenon in which organisms survive extreme dehydration by entering a reversible ametabolic state, is a remarkable example of survival strategies. This study focuses on anhydrobiosis in tardigrades, which are known for their resilience to severe environmental conditions. Tardigrades utilize several protective mechanisms against desiccation, notably the constitutive expression of cytoplasmic abundant heat soluble (CAHS) proteins in <em>Ramazzottius varieornatus</em>. These proteins share similarities in their amphiphatic alpha helices with late embryogenesis abundant (LEA) proteins, but differ significantly in their amino acid sequences. In this study, we further explored the functionality of CAHS proteins by analyzing their role in aggregation and tolerance to hyperosmotic stress in mammalian cells. Using live cell imaging, we examined the subcellular localization of several CAHS and LEA proteins in response to hyperosmotic stress. The expression of CAHS1, CAHS3, and CAHS8 tended to enhance the resilience to the hyperosmotic conditions. These findings not only deepen our understanding of the molecular mechanisms of anhydrobiosis but also highlight the potential of CAHS proteins as cryoprotectants.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> anhydrobiosis, Tardigrades, live imaging, disordered proteins, desiccation tolerance</p></section></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="p001">Anhydrobiosis, derived from “life without water” in Greek, is a phenomenon where an organism loses almost all of its water and enters a state of reversible ametabolism (<a aria-describedby="B7" class="usa-link" href="#B7">Crowe <em>et al.</em>, 1998</a>; <a aria-describedby="B15" class="usa-link" href="#B15">Keilin, 1959</a>). An organism capable of anhydrobiosis tolerates extreme dehydration (desiccation), dries to equilibrium with moderately to extremely dry air, and then restores its vital functions after rehydration. Anhydrobiosis is observed in both microorganisms and animals, including bacteria, nematodes, seeds, spores, gemma, lichen soredium, egg, shoot fragments, and more (<a aria-describedby="B8" class="usa-link" href="#B8">García, 2011</a>; <a aria-describedby="B11" class="usa-link" href="#B11">Hibshman <em>et al.</em>, 2020</a>; <a aria-describedby="B21" class="usa-link" href="#B21">Leprince and Buitink, 2015</a>). Desiccation primarily damages proteins, nucleic acids, and membrane lipids, leading to cell death. Therefore, living organisms that exhibit anhydrobiosis have protective mechanisms to mitigate desiccation. Trehalose accumulation (<a aria-describedby="B7" class="usa-link" href="#B7">Crowe <em>et al.</em>, 1998</a>; <a aria-describedby="B20" class="usa-link" href="#B20">Lapinski and Tunnacliffe, 2003</a>), late embryogenesis abundant (LEA) proteins (<a aria-describedby="B9" class="usa-link" href="#B9">Goyal <em>et al.</em>, 2005</a>; <a aria-describedby="B16" class="usa-link" href="#B16">Kikawada <em>et al.</em>, 2006</a>; <a aria-describedby="B22" class="usa-link" href="#B22">MacRae, 2016</a>), heat shock proteins (<a aria-describedby="B6" class="usa-link" href="#B6">Cornette <em>et al.</em>, 2010</a>; <a aria-describedby="B17" class="usa-link" href="#B17">King and MacRae, 2015</a>), and ROS scavenging (<a aria-describedby="B10" class="usa-link" href="#B10">Gusev <em>et al.</em>, 2010</a>; <a aria-describedby="B27" class="usa-link" href="#B27">Rizzo <em>et al.</em>, 2010</a>) have been proposed as mechanisms of desiccation tolerance (<a aria-describedby="B12" class="usa-link" href="#B12">Janis <em>et al.</em>, 2018</a>).</p>
<p id="p002">Tardigrades, widely used as model organisms for anhydrobiosis research, exhibit extreme tolerance to high and low temperatures, high pressure, and radiation in their anhydrobiotic state (<a aria-describedby="B25" class="usa-link" href="#B25">Rebecchi <em>et al.</em>, 2007</a>; <a aria-describedby="B36" class="usa-link" href="#B36">Wright, 2001</a>). Although several reports have been reported on the defense mechanisms underlying anhydrobiosis in tardigrades, including induction of heat shock proteins and defensive peroxidases (<a aria-describedby="B26" class="usa-link" href="#B26">Reuner <em>et al.</em>, 2010</a>; <a aria-describedby="B27" class="usa-link" href="#B27">Rizzo <em>et al.</em>, 2010</a>; <a aria-describedby="B30" class="usa-link" href="#B30">Schokraie <em>et al.</em>, 2011</a>; <a aria-describedby="B39" class="usa-link" href="#B39">Yoshida <em>et al.</em>, 2022</a>), cytoplasmic abundant heat soluble (CAHS) proteins, identified as a family of highly hydrophilic and heat-soluble proteins, have attracted much attention. CAHS proteins are constitutively expressed in <em>Ramazzottius varieornatus</em>, which exhibits strong desiccation tolerance without <em>de novo</em> gene expression. On the other hand, in another tardigrade, <em>Hypsibius exemplaris</em>, the expression of CAHS proteins is strongly induced under desiccated condition, and these expressions are essential for desiccation tolerance (<a aria-describedby="B1" class="usa-link" href="#B1">Arakawa, 2022</a>; <a aria-describedby="B18" class="usa-link" href="#B18">Kondo <em>et al.</em>, 2015</a>, <a aria-describedby="B19" class="usa-link" href="#B19">2019</a>; <a aria-describedby="B38" class="usa-link" href="#B38">Yoshida <em>et al.</em>, 2017</a>). CAHS proteins share some similarities with LEA proteins in terms of bioinformatic predictions of a strong tendency for amphiphatic α-helix formation, despite their low amino acid sequence similarity to LEA proteins. Recently, several groups, including ours, have reported that the CAHS proteins demonstrate gelation, droplet, or fibrous structures <em>in vitro</em> and in cells (<a aria-describedby="B23" class="usa-link" href="#B23">Malki <em>et al.</em>, 2022</a>; <a aria-describedby="B31" class="usa-link" href="#B31">Tanaka <em>et al.</em>, 2022</a>; <a aria-describedby="B35" class="usa-link" href="#B35">Veling <em>et al.</em>, 2022</a>; <a aria-describedby="B37" class="usa-link" href="#B37">Yagi-Utsumi <em>et al.</em>, 2021</a>). It has also been shown that the heterologous expression of CAHS proteins suppresses hyperosmotic pressure-induced cell death of budding yeast, bacteria, and <em>Drosphila</em> S2 cells (<a aria-describedby="B2" class="usa-link" href="#B2">Boothby <em>et al.</em>, 2017</a>; <a aria-describedby="B31" class="usa-link" href="#B31">Tanaka <em>et al.</em>, 2022</a>). However, it remains unclear whether the heterologous expression of CAHS proteins confers hyperosmotic stress tolerance to cultured mammalian cells.</p>
<p id="p003">In this study, we show the changes in the subcellular localization of <em>R. varieornatus</em> CAHS1, CAHS3, CAHS8, CAHS12, late embryogenesis abundant protein mitochondrial (LEAM), and mitochondrial abundant heat soluble (MAHS) proteins upon hyperosmotic stress stimulation by live cell imaging in mammalian cultured cells. We also investigated the tolerance to hyperosmotic stress by the expression of CAHS proteins, and found that the expression of CAHS1, CAHS3, or CAHS8 rendered the cells tolerant to the hyperosmotic stress.</p></section><section id="sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="sec3"><h3 class="pmc_sec_title">Plasmid construction</h3>
<p id="p004">All plasmids used in this study are summarized in <a class="usa-link" href="#S6">Table S1</a>, along with Benchling links to the plasmid sequences and maps. Plasmids were constructed by standard molecular biology methods. The cDNAs of <em>R. varieornatus</em> CAHS1, CAHS3, CAHS8, CAHS12, LEAM, and MAHS were synthesized with codon optimization for the human genome using gBlocks Gene Fragments (Integrated DNA Technologies, Coralville, IA, USA) and inserted into the pCAGGS vector (<a aria-describedby="B24" class="usa-link" href="#B24">Niwa <em>et al.</em>, 1991</a>) for transient expression experiments (<a class="usa-link" href="#F1">Fig. 1</a>) and the pT2A vector (pT2Apuro-rtTA2-TRE) for drug-inducible expression experiments (<a class="usa-link" href="#F2">Fig. 2</a> and <a class="usa-link" href="#F3">3</a>). Of note, since CAHS proteins are known to be abundantly expressed in tardigrades, we optimized the codon of the CAHS genes in order to express sufficient amounts of CAHS proteins in HeLa cells. The Tol2-based transposon donor vector (pT2A) and the Tol2 transposase expression vector (pCAGGS-T2TP) were kindly provided by Koichi Kawakami (National Institute of Genetics, Mishima, Japan) (<a aria-describedby="B14" class="usa-link" href="#B14">Kawakami and Noda, 2004</a>). The cDNAs of rtTA (<a aria-describedby="B28" class="usa-link" href="#B28">Roney <em>et al.</em>, 2016</a>) synthesized by FASMAC, tet-response element, and multiple cloning sites were inserted into a Tol2 donor vector by ligation and Gibson assembly to generate pTol2puro-rtTA2-TRE-MCS. The cDNAs of monomeric enhanced green fluorescent protein (mEGFP), CAHS1-mEGFP, CAHS3-mEGFP, CAHS8-mEGFP, and CAHS12-mEGFP were subcloned into the pT2Apuro-rtTA2-TRE-MCS vector.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11930778_csf_49_24035-f001.jpg" target="_blank"><img alt="Fig. 1" class="graphic zoom-in" height="850" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d38/11930778/dc5eb9600e02/csf_49_24035-f001.jpg" width="667"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Subcellular localization of CAHS1, CAHS3, CAHS8, CAHS12, LEAM, and MAHS proteins in response to hyperosmotic stress in mammalian cells</p>
<p id="p033">(A) Schematic representation of the experiments. (B) Timeline of hyperosmotic stress time-lapse imaging. The red dots represent the time points when the representative cells are shown in panels C–I. (C–I) Representative images of HeLa cells overexpressing mEGFP (C), CAHS1-mEGFP (D), CAHS3-mEGFP (E), CAHS8-mEGFP (F), CAHS12-mEGFP (G), LEAM-mEGFP (H), and MAHS-mEGFP (I) proteins at the indicated time points. A solution of 0.5 M sorbitol (left panels) or 0.2 M sodium chloride (right panels) was added after 2 min. Insets are enlarged images of the enclosed area. After 5 min, the cells were washed with fresh medium using a microfluidic system. Scale bar, 10 μm.</p></figcaption></figure><figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11930778_csf_49_24035-f002.jpg" target="_blank"><img alt="Fig. 2" class="graphic zoom-in" height="492" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d38/11930778/659ccc0ad7ab/csf_49_24035-f002.jpg" width="690"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Effects of CAHS1-mEGFP expression on cell viability and cell death upon osmotic stress in HeLa cells</p>
<p id="p034">(A and B) Schematic representation of the experimental scheme (A) and time course (B) of the stress assays. (C) Doxycycline-induced expression of CAHS1-mEGFP in HeLa cells. HeLa cells were treated with the indicated concentration of doxycycline for 24 h. The cells were imaged by wide-field epi-fluorescence microscopy. The CAHS1-mEGFP expression in each cell is shown as a box plot, in which the box extends from the first to the third quartile with the whiskers denoting 1.5 times the interquartile range. The number of cells is as follows: n = 105 for 0 ng/mL; n = 142 for 1 ng/mL; n = 299 for 10 ng/mL; n = 252 for 100 ng/mL. The symbols indicate the results of Dunnett’s test analysis compared with the 0 ng/mL. **: <em>p</em>&lt;0.01, and N.S.: not significant. (D) Heatmap of cell viability assay (left, WST-1 assay) and cytotoxicity assay (right, LDH assay) in HeLa cells expressing CAHS1-mEGFP in a doxycycline-dependent manner. In these heatmaps, red color indicates a higher number of viable cells (WST-1 assay) or dead cells (LDH assay). The average values are shown based on three independent experiments. (E and F) The bar graphs show the mean IC50 values for cell viability (E) and EC50 values for cell death (F) in HeLa cells expressing CAHS1-mEGFP (left) or parental HeLa cells (right) treated with the indicated concentration of doxycycline with S.D. Each dot represents the raw data from three independent experiments. The color (blue, green, and orange) of these dots indicates the data obtained in the same experiments. The symbols indicate the results of <em>t</em>-test analysis compared with the 0 ng/mL. **<em>p</em>&lt;0.01, and N.S.: not significant.</p></figcaption></figure><figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 3" class="graphic" height="1183" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d38/11930778/44349c1f895f/csf_49_24035-f003.jpg" width="695"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Effects of CAHS3, CAHS8, or CAHS12 expression on cell viability upon osmotic stress in HeLa cells</p>
<p id="p035">(A) Doxycycline-induced expression of mEGFP, CAHS1-mEGFP, CAHS3-mEGFP, CAHS8-mEGFP, or CAHS12-mEGFP in HeLa cells. HeLa cells were treated with or without 100 ng/mL doxycycline for 24 h. The cells were imaged by wide-field epi-fluorescence microscopy. The mEGFP expression in each cell is shown as a box plot, in which the box extends from the first to the third quartile with the whiskers denoting 1.5 times the interquartile range. The numbers of cells are as follows: mEGFP, n = 238 and 373 (Dox– and Dox+); CAHS1-mEGFP, n = 68 and 298 (Dox– and Dox+); CAHS3-mEGFP, n = 107 and 172 (Dox– and Dox+); CAHS8-mEGFP, n = 245 and 364 (Dox– and Dox+); CAHS12-mEGFP, n = 61 and 286 (Dox– and Dox+). The symbols indicate the results of <em>t</em>-test analysis; **<em>p</em>&lt;0.01 compared with the Dox– data. (B) Heatmap of cell viability assay in HeLa cells expressing the indicated proteins without (left) or with 100 ng/mL doxycycline (right). In these heatmaps, red color indicates a higher number of viable cells. The average values are shown based on three independent experiments. (C) The mean of an IC50 value of cell viability against sorbitol before and after doxycycline treatment are shown as a bar graph with S.D. (n = 3). Although <em>t</em>-test was conducted for the data compared with the mEGFP data, the statistical significance was not observed.</p></figcaption></figure></section><section id="sec4"><h3 class="pmc_sec_title">Cell culture</h3>
<p id="p005">HeLa cells were kindly provided by Michiyuki Matsuda (Kyoto University). HeLa cells were maintained in Dulbecco’s Modified Eagle’s Medium (DMEM) high glucose (#08458-45, Nacalai Tesque, Kyoto, Japan) supplemented with 10% heat-inactivated fetal bovine serum (FBS) (#F7524, Sigma-Aldrich, St. Louis, MO, USA) at 37°C in a humidified atmosphere containing 5% CO<sub>2</sub>.</p></section><section id="sec5"><h3 class="pmc_sec_title">Establishment of stable cell lines</h3>
<p id="p006">For transposon-mediated gene transfer, HeLa cells were transfected with Tol2 donor vectors and Tol2 transposase-expressing vectors in a 1:1 ratio. Approximately 16–24 h after transfection, the cells were treated with 1.0 μg/mL puromycin (#ant-pr-1, InvivoGen, San Diego, CA, USA) for more than one week to select bulk cell lines. Cells were maintained in the presence of puromycin. In <a class="usa-link" href="#F2">Fig. 2</a>, the bright fluorescent cells were sorted into a 96-well plate using a cell sorter (MA900, SONY, Tokyo, Japan), and the single cell clone was used in <a class="usa-link" href="#F2">Fig. 2</a>. The bulk cell population was used in <a class="usa-link" href="#F3">Fig. 3</a>.</p></section><section id="sec6"><h3 class="pmc_sec_title">Time-lapse imaging with hyperosmotic stress</h3>
<p id="p007">HeLa cells were seeded on a 35-mm dish (#150460, Thermo Fisher Scientific, Rockford, IL, USA) or a 6 well plate (#140675, Thermo Fisher Scientific). The plasmid expressing the CAHS1-mEGFP protein, the CAHS3-mEGFP, the CAHS8-mEGFP, the CAHS12-mEGFP, the LEAM-mEGFP, the MAHS-mEGFP and the mEGFP were transfected into the HeLa cells with 293fectin<sup>TM</sup> Transfection Reagent (#12347-019, Invitrogen, Waltham, MA, USA). After 24 h of transfection, the cells were loaded onto a microfluidic plate (M04S-03-5PK, Millipore, Burlington, MA, USA) according to the manufacturer’s instructions. The microfluidic plates were controlled by a CellASIC ONIX2 Microfluidic System (CAX2-S0000, Millipore). After 48 h of transfection, the culture medium was replaced with an imaging medium containing FluoroBrite<sup>TM</sup> DMEM (#A1896701, Thermo Fisher Scientific) supplemented with 1% GlutaMAX<sup>TM</sup> Supplement (#35050061, Thermo Fisher Scientific) and 0.1% bovine serum albumin. The imaging medium continuously flowed at 5 psi (“psi” is a unit of pressure) while heating at 37°C by a temperature-controllable manifold (CAX2-MXT20, Millipore).</p>
<p id="p008">For hyperosmotic shock experiments, 0.5 M sorbitol solution and 0.2 M sodium chloride solution were prepared as follows: 4.5 M sorbitol in DDW and 5 M sodium chloride in DDW were dissolved by the imaging medium, resulting in 0.5 M sorbitol solution and 0.2 M sodium chloride solution, respectively. The transfected cells seeded on microfluidic plates were time-lapse imaged under the imaging medium condition every 10 sec for 2 min. The imaging medium continuously flowed at 1 psi while heating at 37°C. Then, 0.5 M sorbitol solution or 0.2 M sodium chloride solution was added, and fluorescence images were acquired every 10 sec for 5 min. The hyperosmotic medium continuously flowed at 5 psi while heating at 37°C. After incubation, the hyperosmotic medium was replaced with the imaging medium, and the cells were observed every 10 sec for 10 min. The imaging medium continuously flowed at 5 psi while heating at 37°C.</p>
<p id="p009">Cells were imaged with an inverted microscope (IX83, Olympus/Evident, Tokyo, Japan) equipped with an sCMOS camera (ORCA-Fusion BT, Hamamatsu photonics, Hamamatsu, Japan), a spinning disk confocal unit (CSU-W1, Yokogawa Electric Corporation, Musashino, Japan), and diode lasers at a wavelength of 488 nm. An oil immersion objective lens (UPLXAPO100XO, N.A. 1.45, Olympus/Evident) was used. The excitation laser and fluorescence filter settings were as follows: excitation laser, 488 nm; dichroic mirror (DM), 405/488/561 nm; and emission filters, 500–550 nm. During the observation, the cells were incubated with microfluidic plates at 37°C. The microscope was controlled by MetaMorph software (Molecular Devices, San Jose, CA, USA).</p>
<p id="p010">Fluorescence imaging data were analyzed and quantified in Fiji/ImageJ (<a aria-describedby="B29" class="usa-link" href="#B29">Schindelin <em>et al.</em>, 2012</a>). The background was subtracted by a constant value. If salt-and-pepper noise was not negligible, a median filter with 1-pixel was applied. To compensate for the misregistration caused during time-lapse imaging, the “StackReg function” was applied with the “Translation” option.</p></section><section id="sec7"><h3 class="pmc_sec_title">Fluorescence imaging for quantifying expression levels</h3>
<p id="p011">HeLa cells carrying the doxycycline-inducible expression cassette of CAHS1, 3, 8, 12-mEGFP or mEGFP were seeded on a 96-well plate (#167008, Thermo Fisher Scientific). One day after seeding, the cells were treated with the indicated concentration of doxycycline, and 24 h after doxycycline treatment, the cells were observed with a fluorescence microscope.</p>
<p id="p012">The cells were imaged with an inverted microscope (IX81, Olympus/Evident) equipped with a cooled CCD camera (Retiga 4000R, Photometrics, Tucson, AZ, USA), an excitation light source (Spectra-X light engine, Lumencor Inc., Beaverton, OR, USA), a laser-based autofocus system (IX2-ZDC, Olympus/Evident), a controller for filter wheels and XY stage (MAC5000, Ludl Electronic Products, Hawthorne, NY, USA), and an incubation chamber (STXG-IX3WX, Tokai Hit, Fujinomiya, Japan). An objective lens (UPlanApo 10x, N.A. 0.40, Olympus/Evident) was used. For EGFP imaging, a 475/28 excitation filter, a 20/80 beamsplitter dichroic mirror (Chroma Technology Corp., Bellows Falls, VT, Japan), and a 542/27 emission filter (Semrock, IDEX Health &amp; Science, LLC, West Henrietta, NY, USA) were used. During the observation, live cells were incubated in a stage incubator containing 5% CO<sub>2</sub> at 37°C. The microscope was controlled by MetaMorph software (Molecular Devices).</p>
<p id="p013">The fluorescence imaging data were analyzed and quantified in Fiji/ImageJ (<a aria-describedby="B29" class="usa-link" href="#B29">Schindelin <em>et al.</em>, 2012</a>). The background was subtracted by the rolling ball method. The region of interest (ROI) of each cell was extracted with the “StarDistD2” function (Probability/Score Threshold: 0.55), and the fluorescence intensity (Mean value) of each ROI was obtained with the “Measure” function.</p></section><section id="sec8"><h3 class="pmc_sec_title">Cell viability assay and cell death assay</h3>
<p id="p014">For the cell viability assay and the cell death assay, a low serum concentration medium (1% FBS), hereinafter referred to as the assay medium, was used to reduce the background absorbance in the measurement of lactate dehydrogenase (LDH) release assay because serum contains LDH. The assay medium contains 1x DME culture solution (#639-29631, Nitta Gelatin, Yao, Japan), 1% FBS, 26 mM NaHCO<sub>3</sub> (#09655-25, Nacalai Tesque), and 1 μg/mL puromycin (if needed). For hyperosmotic stress, a 3.2 M sorbitol solution in the assay medium was prepared by mixing 4.5 M sorbitol in DDW, 5x DME culture solution, 260 mM NaHCO<sub>3</sub>, 100% FBS, DDW, and 1 mg/mL puromycin. This 3.2 M sorbitol solution in the assay medium was serially diluted with the assay medium to produce the indicated concentration of sorbitol solution. For drug-inducible gene expression, 1 mg/mL doxycycline (#D4116, Tokyo Chemical Industry, Tokyo, Japan) was serially diluted with the assay medium to produce the indicated concentration of doxycycline solution in the assay medium.</p>
<p id="p015">HeLa cells were plated on 96-well plates (7.5 × 10<sup>2</sup> – 3.0 × 10<sup>3</sup> cells/well) (#167425, Thermo Fisher Scientific), and cultured in the assay medium for 24 h. To induce gene expression, the cells were treated with doxycycline for 24 h, followed by the treatment with sorbitol solution for 24 h. For the simultaneous cell viability assay and cell death assay (<a class="usa-link" href="#F2">Fig. 2</a>), half of the cell supernatant was used for the water-soluble tetrazolium (WST)-1 assay with a Premix WST-1 Cell Proliferation Assay System (#MK400, TaKaRa, Otsu, Japan) and the other half for the lactate dehydrogenase (LDH) release assay with an LDH Cytotoxicity Detection Kit (#MK401, TaKaRa) according to the manufacturer’s instructions. Both the WST-1 and LDH release assays were evaluated based on absorbance at 450 nm (and 620 nm as a control) using a microplate reader (#51119000, Thermo Fisher Scientific).</p>
<p id="p016">The absorbance values obtained from the microplate reader were exported and analyzed by Python (ver. 3.7.3). First, the absorbance values were subtracted from the value obtained from the well containing only the assay medium. Second, the subtracted values were normalized by the following equation:</p>
<table class="disp-formula p" id="disp-formula1"><tr><td class="formula"><math display="block" id="eq1" overflow="linebreak"><mi>Y</mi><mi> </mi><mo>=</mo><mi> </mi><mo>(</mo><mi>X</mi><mo>-</mo><mi>L</mi><mi>o</mi><mi>w</mi><mo>)</mo><mo>/</mo><mo>(</mo><mi>H</mi><mi>i</mi><mi>g</mi><mi>h</mi><mo>-</mo><mi>L</mi><mi>o</mi><mi>w</mi><mo>)</mo></math></td></tr></table>
<p id="p018">where <em>Y</em> is the normalized value, <em>X</em> is the experimental data, and <em>High</em> and <em>Low</em> are the highest and lowest values of the data obtained on the same day, respectively. For the cell viability assay in <a class="usa-link" href="#F2">Fig. 2</a>, the High and Low values were obtained from the maximum and minimum values on the same day, respectively. In <a class="usa-link" href="#F3">Fig. 3</a>, where the bulk population cell lines were used, the High and Low values were obtained from the value of the well in the absence of sorbitol and the minimum values on the same day, respectively. For the cell death assay, the High and Low values were taken from the value using Triton X-100 treated cells and the lowest values on the same day, respectively. Of note, the absolute number of cells seeded on the well plate varied between experiments, even though the number of cells was counted and the same number of cells were seeded between experiments. Therefore, the “High” and “Low” values were changeable. In addition, the LDH assay requires the supernatant to be taken from each well. These factors and procedures would cause the large variation between experiments. The normalized values were used to generate maps using the seaborn library in Python (<a class="usa-link" href="#F2">Figs. 2</a>D, <a class="usa-link" href="#F3">3</a>B, <a class="usa-link" href="#S6">S1</a>, <a class="usa-link" href="#S6">S2</a> and <a class="usa-link" href="#S6">S3</a>). The normalized data were fitted with the following functions using the scipy.optimize.corve_fit in Python:</p>
<table class="disp-formula p" id="disp-formula2"><tr><td class="formula"><math display="block" id="eq2" overflow="linebreak"><mi>W</mi><mi>S</mi><mi>T</mi><mo>(</mo><mi>x</mi><mo>)</mo><mo>=</mo><mo>(</mo><mi>m</mi><mi>a</mi><mi>x</mi><mo>-</mo><mi>m</mi><mi>i</mi><mi>n</mi><mo>)</mo><mo>∗</mo><msup><mrow><mi>I</mi><mi>C</mi><mn>50</mn></mrow><mrow><mi>n</mi></mrow></msup><mo>/</mo><mo>(</mo><msup><mrow><mi>x</mi></mrow><mrow><mi>n</mi></mrow></msup><mo>+</mo><mi>I</mi><mi>C</mi><mn>5</mn><msup><mrow><mn>0</mn></mrow><mrow><mi>n</mi></mrow></msup><mo>)</mo><mo>+</mo><mi>m</mi><mi>i</mi><mi>n</mi></math></td></tr></table>
<table class="disp-formula p" id="disp-formula3"><tr><td class="formula"><math display="block" id="eq3" overflow="linebreak"><mi>L</mi><mi>D</mi><mi>H</mi><mo>(</mo><mi>x</mi><mo>)</mo><mo>=</mo><mo>(</mo><mi>m</mi><mi>a</mi><mi>x</mi><mo>-</mo><mi>m</mi><mi>i</mi><mi>n</mi><mo>)</mo><mo>∗</mo><msup><mrow><mi>x</mi></mrow><mrow><mi>n</mi></mrow></msup><mo>/</mo><mo>(</mo><msup><mrow><mi>x</mi></mrow><mrow><mi>n</mi></mrow></msup><mo>+</mo><mi>E</mi><mi>C</mi><mn>5</mn><msup><mrow><mn>0</mn></mrow><mrow><mi>n</mi></mrow></msup><mo>)</mo><mo>+</mo><mi>m</mi><mi>i</mi><mi>n</mi></math></td></tr></table>
<p id="p021">where <em>WST(x)</em> and <em>LDH(x)</em> are the fitted WST and LDH values, <em>x</em> is the sorbitol concentration, max and min are the maximum and minimum values, <em>n</em> is the Hill coefficient, and <em>IC50</em> and <em>EC50</em> are the half inhibitory concentration and half effective concentration for cell viability and cell death, respectively.</p></section><section id="sec9"><h3 class="pmc_sec_title">Structure prediction by AlphaFold2</h3>
<p id="p022">The predicted structures of CAHS1, CAHS3, CAHS8, and CAHS12 were obtained from the AlphaFold Protein Structure Database (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://alphafold.ebi.ac.uk/" rel="noopener noreferrer" target="_blank">https://alphafold.ebi.ac.uk/</a>). Direct links are as follows: CAHS1 (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://alphafold.ebi.ac.uk/entry/J7M799" rel="noopener noreferrer" target="_blank">https://alphafold.ebi.ac.uk/entry/J7M799</a>), CAHS3 (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://alphafold.ebi.ac.uk/entry/J7M3T1" rel="noopener noreferrer" target="_blank">https://alphafold.ebi.ac.uk/entry/J7M3T1</a>), CAHS8 (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://alphafold.ebi.ac.uk/entry/A0A1D1UQN2" rel="noopener noreferrer" target="_blank">https://alphafold.ebi.ac.uk/entry/A0A1D1UQN2</a>), and CAHS12 (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://alphafold.ebi.ac.uk/entry/A0A1D1VNN8" rel="noopener noreferrer" target="_blank">https://alphafold.ebi.ac.uk/entry/A0A1D1VNN8</a>).</p></section></section><section id="sec10"><h2 class="pmc_sec_title">Results and Discussion</h2>
<section id="sec11"><h3 class="pmc_sec_title">Hyperosmotic stress-induced changes in the subcellular localization of CAHS, LEAM, and MAHS proteins in HeLa cells</h3>
<p id="p023">We examined changes in the subcellular localization of anhydrobiosis-related proteins in <em>R. varieornatus</em> induced by hyperosmotic stress, which is thought to be an early step in anhydrobiosis. Six types of proteins, including CAHS1, CAHS3, CAHS8, and CAHS12, mitochondrial LEA (LEAM), and mitochondrial abundant heat soluble (MAHS), were separately expressed as mEGFP-fused protein in HeLa cells (<a class="usa-link" href="#F1">Fig. 1</a>A). We selected CAHS1, CAHS3, CAHS8, and CAHS12 because they reversibly form different types of aggregates in response to osmotic stimulation (<a aria-describedby="B31" class="usa-link" href="#B31">Tanaka <em>et al.</em>, 2022</a>; <a aria-describedby="B37" class="usa-link" href="#B37">Yagi-Utsumi <em>et al.</em>, 2021</a>), but their roles in osmotic stress resistance have not been investigated except for CAHS3 (<a aria-describedby="B31" class="usa-link" href="#B31">Tanaka <em>et al.</em>, 2022</a>). The cells were loaded into a microfluidic chamber for rapid osmolyte solution perfusion. One day after loading into the microfluidic chamber, the cells were imaged with a spinning disk conformal microscope to visualize the subcellular distribution of these fusion proteins upon hyperosmotic stress such as 0.5 M sorbitol or 0.2 M NaCl (<a class="usa-link" href="#F1">Fig. 1</a>B). Of note, NaCl is the principal determinant of tonicity of the extracellular fluid (<a aria-describedby="B4" class="usa-link" href="#B4">Burg <em>et al.</em>, 2007</a>). However, cell membranes are permeable to Na<sup>+</sup>, and elevated chloride specifically increases the expression of α-Na<sup>+</sup>-K<sup>+</sup>-ATPase (<a aria-describedby="B5" class="usa-link" href="#B5">Capasso <em>et al.</em>, 2003</a>). For this reason, poorly permeating organic solutes like sorbitol, mannitol, or raffinose are often used to produce hypertonicity (<a aria-describedby="B4" class="usa-link" href="#B4">Burg <em>et al.</em>, 2007</a>).</p>
<p id="p024">HeLa cells expressing mEGFP treated with the osmolytes showed increased fluorescence signal in both the cytoplasm and nucleus, and puncta formation in the nucleus (<a class="usa-link" href="#F1">Fig. 1</a>C and <a class="usa-link" href="#S1">Movie 1</a>). These changes were reverted to the basal state by osmolyte washout (<a class="usa-link" href="#F1">Fig. 1</a>C). Under unstressed conditions, CAHS1-mEGFP, CAHS8-mEGFP, and CAHS12-mEGFP were broadly distributed in both the cytosol and nucleus, whereas CAHS3-mEGFP was mainly localized in the cytosol (<a class="usa-link" href="#F1">Fig. 1</a>D–G and <a class="usa-link" href="#S2">Movies 2</a>–<a class="usa-link" href="#S5">5</a>). When exposed to hyperosmotic medium supplemented with 0.5 M sorbitol or 0.2 M NaCl, these CAHS proteins formed different aggregation patterns; CAHS1-mEGFP mainly forms granules and filamentous structures and slightly accumulates at the plasma membrane (<a class="usa-link" href="#F1">Fig. 1</a>D and <a class="usa-link" href="#S2">Movie 2</a>). CAHS3-mEGFP shows fibrous structures in the cytoplasm (<a class="usa-link" href="#F1">Fig. 1</a>E and <a class="usa-link" href="#S3">Movie 3</a>). CAHS8-mEGFP seems to have the same pattern as mEGFP, but it slightly accumulates at the nuclear membrane (<a class="usa-link" href="#F1">Fig. 1</a>F and <a class="usa-link" href="#S4">Movie 4</a>). CAHS12-mEGFP appears to undergo decomposition in the nucleus (<a aria-describedby="B3" class="usa-link" href="#B3">Bracha <em>et al.</em>, 2018</a>), while it exhibits granules and filamentous structures (<a class="usa-link" href="#F1">Fig. 1</a>G and <a class="usa-link" href="#S5">Movie 5</a>). These results are, for the most part, consistent with previous reports (<a aria-describedby="B31" class="usa-link" href="#B31">Tanaka <em>et al.</em>, 2022</a>; <a aria-describedby="B37" class="usa-link" href="#B37">Yagi-Utsumi <em>et al.</em>, 2021</a>). Interestingly, CAHS proteins appear to behave differently between 0.5 M sorbitol and 0.2 M NaCl osmotic stress. For example, CAHS1 did not aggregate at the cell membrane under 0.2 M NaCl stress, but showed strong aggregation in the cytoplasm (<a class="usa-link" href="#F1">Fig. 1</a>D). CAHS12 also showed strong aggregation at the nuclear membrane and plasma membrane upon 0.5 M sorbitol and 0.2 M NaCl, respectively (<a class="usa-link" href="#F1">Fig. 1</a>G). Regardless of whether the aggregate forms granules or filaments during hyperosmotic stress, mEGFP and all CAHS protein condensates show rapid disassembly and restoration of coherent distribution upon the release of stress. It is reasonable to conclude that these rapid aggregation and disassembly processes are attributed to the biophysical properties of both mEGFP and each CAHS protein. Such efficient reversible phase transitioning mirrors the relatively fast induction and recovery of tardigrade anhydrobiosis. The varying modes of aggregation and localization patterns of different CAHS paralogs upon hyperosmotic stress suggest that these paralogs serve differentiated functions rather than for mere redundancy. Emerging tissue-specific expression patterns of tardigrade anhydrobiosis-related proteins (<a aria-describedby="B32" class="usa-link" href="#B32">Tanaka <em>et al.</em>, 2023</a>) could be part of the reasons for such differentiation.</p>
<p id="p025">It has been reported that two mitochondrial heat-soluble proteins, LEAM and MAHS, in <em>R. varieornatus</em> improve the hyperosmotic tolerance of human cells, possibly by acting as mitochondrial protectants (<a aria-describedby="B33" class="usa-link" href="#B33">Tanaka <em>et al.</em>, 2015</a>). Therefore, we investigated the subcellular localization of LEAM and MAHS in HeLa cells stimulated by hyperosmotic stress. Under unstressed conditions, LEAM-mEGFP and MAHS-mEGFP were localized to the mitochondria, and LEAM-mEGFP was weakly distributed in the cytosol and nucleus (<a class="usa-link" href="#F1">Fig. 1</a>H and I). Hyperosmotic stress arrested the random movement of mitochondria, but did not substantially change the subcellular localization of LEAM-mEGFP and MAHS-mEGFP (<a class="usa-link" href="#F1">Fig. 1</a>H and I). Removal of osmolytes by replacement with an isosmotic medium resumed the thermal fluctuation of the mitochondria. Based on the fact that both LEAM and MAHS proteins localize to mitochondria and are highly hydrophilic (<a aria-describedby="B33" class="usa-link" href="#B33">Tanaka <em>et al.</em>, 2015</a>), LEAM and MAHS are likely to be localized to the mitochondrial matrix. Taken together, these results suggest that the resistance to hyperosmotic stress by LEAM and MAHS expression is not cell-wide protection as observed for CAHS proteins, so we focused on the CAHS proteins for our subsequent analyses.</p></section><section id="sec12"><h3 class="pmc_sec_title">CAHS1 expression renders HeLa cells resistant to hyperosmotic stress</h3>
<p id="p026">Recent studies have elucidated that the CAHS3 protein forms fibrillar aggregates under hyperosmotic conditions in mammalian cells, thereby enhancing their tolerance to hyperosmotic stress, as observed in <em>Drosophila</em> S2 cells (<a aria-describedby="B31" class="usa-link" href="#B31">Tanaka <em>et al.</em>, 2022</a>). However, the question remained as to whether other CAHS proteins, which exhibit varied aggregation patterns under elevated osmotic pressure, also contribute to augmented resistance to such conditions. To this end, we developed a method for quantitatively assessing osmotic stress resistance using HeLa cells expressing CAHS1 (<a class="usa-link" href="#F2">Fig. 2</a>A). We first established a single-cell clone of HeLa cells, expressing CAHS1-mEGFP in a manner dependent on doxycycline treatment. The experimental procedure was as follows: One day after cell seeding, doxycycline was added to induce CAHS1-mEGFP expression for two days, followed by different concentrations of sorbitol as osmolite. Cell viability and increased cell death were evaluated using the WST-1 and LDH assays, respectively (see Methods for details)(<a class="usa-link" href="#F2">Fig. 2</a>B).</p>
<p id="p027">We first confirmed the increase in CAHS1-mEGFP expression levels when we treated the cells with doxycycline (<a class="usa-link" href="#F2">Fig. 2</a>C). Despite performing single-cell cloning, the variation in the expression levels of CAHS1-mEGFP between cells was unexpectedly large. Under this condition, the cells were further exposed to hyperosmotic stress. Cell viability was strongly inhibited even in 0.1 M sorbitol in the absence of doxycycline (<a class="usa-link" href="#F2">Fig. 2</a>D, left, first row, and <a class="usa-link" href="#S6">Fig. S1</a>). The increase in CAHS1-mEGFP expression attenuated the decrease in cell viability by hyperosmotic stress with sorbitol (<a class="usa-link" href="#F2">Fig. 2</a>D, left, and <a class="usa-link" href="#S6">Fig. S1</a>). The half maximal inhibitory concentration (IC50) increased depending on the CAHS1-mEGFP expression level (<a class="usa-link" href="#F2">Fig. 2</a>E, left). Parental HeLa cells as a negative control showed no change in cell viability with or without doxycycline treatment (<a class="usa-link" href="#F2">Fig. 2</a>E, right). In line with the cell viability assay, the number of hyperosmotic stress-induced cell death was also reduced by the expression of CAHS1-mEGFP (<a class="usa-link" href="#F2">Fig. 2</a>D, right, and <a class="usa-link" href="#S6">Fig. S1</a>). The half-maximal effective concentration (EC50) demonstrated a tendency to elevate with increasing expression of CAHS1-mEGFP (<a class="usa-link" href="#F2">Fig. 2</a>F, left). As a negative control, doxycycline treatment did not affect the EC50 values of sorbitol-induced cell death in parental HeLa cells (<a class="usa-link" href="#F2">Fig. 2</a>F, right). These results suggest that CAHS1 expression confers resistance to hyperosmotic stress in HeLa cells. CAHS1-mEGFP localizes to aggregates and fibrous structures and even to the plasma membrane at high osmotic pressure (<a class="usa-link" href="#F1">Fig. 1</a>D), suggesting that CAHS1 exerts some protective function against osmotic stress at these sites. It should be noted that the time-scales of CAHS1 aggregation (~5 min) is much faster than the inhibition of cell proliferation and cell death (~24 h) in response to hyperosmotic stress. This difference may explain the non-linear relationship between CAHS1 expression level (<a class="usa-link" href="#F2">Fig. 2</a>C) and the effects of sorbitol on cell viability (<a class="usa-link" href="#F2">Fig. 2</a>E) and cell death (<a class="usa-link" href="#F2">Fig. 2</a>F). In addition, in this study, the single-cell clone of HeLa cells stably expressing CAHS1-mEGFP indicated lower IC50 values for cell viability and EC50 values for cell death in response to sorbitol stimulation in the absence of doxycycline than parental HeLa cells (<a class="usa-link" href="#F2">Fig. 2</a>E and <a class="usa-link" href="#F2">2</a>F). This may be because the single-cell clones established in this study happen to retain such properties. Further investigation would be needed in future studies to address this concern.</p></section><section id="sec13"><h3 class="pmc_sec_title">Expression of CAHS1, CAHS3, or CAHS8 confers osmotic stress resistance to HeLa cells</h3>
<p id="p028">We next examined the other CAHS proteins for their ability to confer osmotic stress tolerance to HeLa cells. To simplify the experiment, HeLa cell lines were established that express mEGFP, CAHS1-mEGFP, CAHS3-mEGFP, CAHS8-mEGFP, and CAHS12-mEGFP in a doxycycline-dependent manner. Of note, these HeLa cell lines were bulk population cells, but not single-cell clones, because it takes longer time to establish the single-cell clones. First, we quantified the CAHS expression levels induced by doxycycline treatment in these HeLa cells, and found that doxycycline treatment elevated CAHS expression levels several-fold (<a class="usa-link" href="#F3">Fig. 3</a>A). The greater cellular heterogeneity compared to the single-cell cloned CAHS1-mEGFP cell line (<a class="usa-link" href="#F2">Fig. 2</a>C) may be because these cell lines are bulk population cells that have not been subjected to single-cell cloning. Second, we examined the changes in cell viability in these cell lines upon hyperosmotic stress with sorbitol using the WST-1 assay. The average cell viability of each cell line under conditions without and with doxycycline is shown in the heat map (<a class="usa-link" href="#F3">Fig. 3</a>B). Even before doxycycline treatment, cell viability was higher in cells expressing CAHS1-mEGFP and CAHS8-mEGFP, but not CAHS3-mEGFP and CAHS12-mEGFP, than in control mEGFP-expressing cells (<a class="usa-link" href="#F3">Fig. 3</a>B, left). This may be because leakage of CAHS1 or CAHS8 proteins prior to doxycycline treatment contributes to tolerance to high osmotic pressure, while the expression levels of CAHS3 and CAHS12 did not reach the critical thresholds. In addition, doxycycline treatment rendered CAHS1-mEGFP, CAHS3-mEGFP, or CAHS8-mEGFP expressing cells more resistant to higher concentrations of sorbitol than control mEGFP-expressing cells (<a class="usa-link" href="#F3">Fig. 3</a>B, right). In our hand, CAHS1-mEGFP, CAHS3-mEGFP, CAHS8-mEGFP, or CAHS12-mEGFP expressing cells have rather increased cell viability under low concentrations of sorbitol (<a class="usa-link" href="#F3">Fig. 3</a>B, right). At this time, we do not know whether this is a result of variability in our experiments or whether the CAHS proteins potentiate cell viability under low concentrations of sorbitol. To quantitatively evaluate the resistance to hyperosmotic stress, we calculated the ratio of the IC50 value of cell viability against sorbitol after doxycycline treatment to that before doxycycline treatment. The IC50 values of CAHS1-mEGFP, CAHS3-mEGFP, and CAHS8-mEGFP cells increased by 10–20% in a doxycycline-dependent manner (<a class="usa-link" href="#F3">Fig. 3</a>C). No increase in the ratio of IC50 values was observed in mEGFP or CAHS12-mEGFP cells. Although the <em>p</em>-value (<em>p</em> = 0.058 for CAHS1 compared with mEGFP) indicated that the observed difference was not statistically significant, the data suggest a potential trend toward increased resistance to osmotic stress through CAHS1 expression, which may warrant further investigation. The expression level of CAHS12 remained relatively low compared to CAHS3 and CAHS8, which had comparable expression levels before doxycycline treatment (<a class="usa-link" href="#F3">Fig. 3</a>A). We have previously shown that CAHS1 condensation displays concentration-dependent sigmoidal threshold-like behavior between 0.3–0.6 mM at neutral pH (<a aria-describedby="B31" class="usa-link" href="#B31">Tanaka <em>et al.</em>, 2022</a>; <a aria-describedby="B37" class="usa-link" href="#B37">Yagi-Utsumi <em>et al.</em>, 2021</a>); therefore, there still remains the possibility that the CAHS12 concentration did not exceed the critical threshold. Nevertheless, from these results, we conclude that overexpression of CAHS1, CAHS3, and CAHS8 moderately confers tolerance to hyperosmotic stress in HeLa cells, and it is regardless of the mode of aggregation, whether granular or fibrous.</p>
<p id="p029">Finally, we investigated the structure of the CAHS proteins in AlphaFold (<a aria-describedby="B13" class="usa-link" href="#B13">Jumper <em>et al.</em>, 2021</a>; <a aria-describedby="B34" class="usa-link" href="#B34">Varadi <em>et al.</em>, 2022</a>), and found that CAHS1, CAHS3, CAHS8, and CAHS12 all share a single long α-helix and intrinsically disordered regions (IDRs) at the N- and C-termini (<a class="usa-link" href="#F4">Fig. 4</a>); CAHS3 and CAHS12 have a predicted structure at the N-terminus, but their confidence is low. Despite the close structural similarity between CAHS proteins, the subcellular localization of these CAHS proteins under high osmotic pressure differed from each other (<a class="usa-link" href="#F1">Fig. 1</a>). As shown in the previous report, CAHS3 shows a coiled-coil structure and forms a fibrous structure, thereby stabilizing the cell structure and increasing the resistance to high osmotic pressure (<a aria-describedby="B31" class="usa-link" href="#B31">Tanaka <em>et al.</em>, 2022</a>). We have demonstrated that CAHS1 and CAHS8 localize to granules, plasma membrane, and nuclear membrane at high osmotic pressure (<a class="usa-link" href="#F1">Fig. 1</a>), suggesting that CAHS1 and CAHS8 may protect organelles such as plasma membrane and nuclear membrane from hyperosmotic stimuli. These diverse aggregation patterns of CAHS proteins may coordinate anhydrobiosis in tardigrades.</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 4" class="graphic" height="898" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d38/11930778/3866685b5602/csf_49_24035-f004.jpg" width="666"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>The predicted structure of CAHS1, CAHS3, CAHS8, and CAHS12</p>
<p id="p036">The predicted structure of CAHS1 (A), CAHS3 (B), CAHS8 (C), and CAHS12 (D) by AlphaFold2 are shown. A per-residue model confidence score (pLDDT) produced by AlphaFold are represented as colors; blue, pLDDT&gt;90; cyan, 90&gt;pLDDT&gt;70; yellow, 70&gt;pLDDT 50; orange, pLDDT&lt;50.</p></figcaption></figure></section></section><section id="sec14"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="p030">In this study, we expressed <em>R. varieornatus</em> CAHS proteins in HeLa cells to investigate their effects on resistance to hyperosmotic stress, an early step in anhydrobiosis. All of these proteins demonstrated reversible aggregation upon hyperosmotic stress in HeLa cells, exhibiting granular and filamentous structures (<a class="usa-link" href="#F1">Fig. 1</a>). Furthermore, we found that expression of CAHS1, CAHS3, and CAHS8 conferred resistance to sorbitol-induced hyperosmotic stress in HeLa cells (<a class="usa-link" href="#F2">Figs. 2</a> and <a class="usa-link" href="#F3">3</a>). These results not only provide new insights into the mechanism of desiccation tolerance in <em>R. varieornatus</em>, but also suggest a potential use of these CAHS proteins as cryoprotectants.</p></section><section id="sec15"><h2 class="pmc_sec_title">Fundings</h2>
<p id="p031">This work was financially supported in part by grants from the MEXT/JSPS KAKENHI (22H02625 and 24H01416) to K. Aoki and to K. Arakawa (21H05279), the Takeda Science Foundation to K. Aoki, the NAGASE Science Technology Foundation to K. Aoki, and the Joint Research of the Exploratory Research Center on Life and Living Systems (ExCELLS) (ExCELLS program No. 19-501, 24EXC601) to K. Arakawa and K. Aoki, and partly by research funds from the Yamagata Prefectural Government and Tsuruoka City, Japan to K. Arakawa.</p></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p id="p032">We thank all members of the Aoki Laboratory for their helpful discussions and assistance. We thank E. Ebine-Sato and K. Onoda for their technical support and the other members of the Aoki laboratory for their helpful discussions. We are also grateful to S. Tanaka for advice on osmotic stress experiments and K. Kawakami for the kind gift of Tol2 transposon vectors. The <em>Chlorella vulgaris</em> used to feed the tardigrades from which the cDNA sequences were based, was provided courtesy of the Chlorella Industry.</p></section><section class="glossary" id="glossary1"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>
CAHS
</dt>
<dd><p>cytoplasmic abundant heat soluble</p></dd>
<dt>
LEA
</dt>
<dd><p>late embryogenesis abundant</p></dd>
<dt>
LEAM
</dt>
<dd><p>late embryogenesis abundant protein mitochondrial</p></dd>
<dt>
MAHS
</dt>
<dd><p>mitochondrial abundant heat soluble</p></dd>
<dt>
mEGFP
</dt>
<dd><p>monomeric enhanced green fluorescent protein</p></dd>
<dt>
psi
</dt>
<dd><p>pound per square inch</p></dd>
<dt>
WST
</dt>
<dd><p>water-soluble tetrazolium</p></dd>
<dt>
LDH
</dt>
<dd><p>lactate dehydrogenase</p></dd>
<dt>
Dox
</dt>
<dd><p>doxycycline</p></dd>
<dt>
IC50
</dt>
<dd><p>half inhibitory concentration</p></dd>
<dt>
EC50
</dt>
<dd><p>half effective concentration</p></dd>
<dt>
IDR
</dt>
<dd><p>intrinsically disordered region</p></dd>
</dl></section><section id="sec16"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The supporting information for this article is available in J-STAGE Data.</p>
<section class="ref-list" id="ref-list1"><section id="ref-list1_sec2"><ol class="ref-list font-sm"><li id="data">
<cite>Raw datasets acquired to investigate the possible roles of CAHS proteins from tardigrade in osmotic stress tolerance in mammalian cells and their analysis program. </cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.51021/data.csf.27948918" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC11930778/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/39566968/">PubMed</a>]</li></ol></section></section></section><section id="sec17"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="S1"><div class="media p" id="d67e3212"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11930778/bin/csf_49_24035_1.mp4">Download video file</a><sup> (738.8KB, mp4) </sup><p>Movie 1. </p>
<p>HeLa cells expressing mEGFP were loaded onto a microfluidic plate and grown for 1 day. Time-lapse imaging was performed using a spinning disk confocal microscope. The cells were treated with 0.5 M Sorbitol (left) and 0.2 M NaCl (right) at 2 min and incubated for 5 min, followed by the replacement of imaging medium. Images were acquired every 10 sec. Total imaging time = 17 min.</p>
</div></div></section><section class="sm xbox font-sm" id="S2"><div class="media p" id="d67e3217"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11930778/bin/csf_49_24035_2.mp4">Download video file</a><sup> (721.6KB, mp4) </sup><p>Movie 2. </p>
<p>HeLa cells expressing CAHS1-mEGFP were loaded onto a microfluidic plate and grown for 1 day. Time-lapse imaging was performed using a spinning disk confocal microscope. The cells were treated with 0.5 M Sorbitol (left) and 0.2 M NaCl (right) at 2 min and incubated for 5 min, followed by the replacement of imaging medium. Images were acquired every 10 sec. Total imaging time = 17 min.</p>
</div></div></section><section class="sm xbox font-sm" id="S3"><div class="media p" id="d67e3222"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11930778/bin/csf_49_24035_3.mp4">Download video file</a><sup> (724.6KB, mp4) </sup><p>Movie 3. </p>
<p>HeLa cells expressing CAHS3-mEGFP were loaded onto a microfluidic plate and grown for 1 day. Time-lapse imaging was performed using a spinning disk confocal microscope. The cells were treated with 0.5 M Sorbitol (left) and 0.2 M NaCl (right) at 2 min and incubated for 5 min, followed by the replacement of imaging medium. Images were acquired every 10 sec. Total imaging time = 17 min.</p>
</div></div></section><section class="sm xbox font-sm" id="S4"><div class="media p" id="d67e3227"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11930778/bin/csf_49_24035_4.mp4">Download video file</a><sup> (706.8KB, mp4) </sup><p>Movie 4. </p>
<p>HeLa cells expressing CAHS8-mEGFP were loaded onto a microfluidic plate and grown for 1 day. Time-lapse imaging was performed using a spinning disk confocal microscope. The cells were treated with 0.5 M Sorbitol (left) and 0.2 M NaCl (right) at 2 min and incubated for 5 min, followed by the replacement of imaging medium. Images were acquired every 10 sec. Total imaging time = 17 min.</p>
</div></div></section><section class="sm xbox font-sm" id="S5"><div class="media p" id="d67e3232"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11930778/bin/csf_49_24035_5.mp4">Download video file</a><sup> (716.5KB, mp4) </sup><p>Movie 5. </p>
<p>HeLa cells expressing CAHS12-mEGFP were loaded onto a microfluidic plate and grown for 1 day. Time-lapse imaging was performed using a spinning disk confocal microscope. The cells were treated with 0.5 M Sorbitol (left) and 0.2 M NaCl (right) at 2 min and incubated for 5 min, followed by the replacement of imaging medium. Images were acquired every 10 sec. Total imaging time = 17 min.</p>
</div></div></section><section class="sm xbox font-sm" id="S6"><div class="media p" id="d67e3237"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11930778/bin/csf_49_24035_6.pdf">Supplementary Materials</a><sup> (265.4KB, pdf) </sup>
</div></div></section></section><section class="ref-list" id="ref-list2"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list2_sec2"><ol class="ref-list font-sm">
<li id="B1">
<cite>Arakawa, K.
2022. Examples of extreme survival: Tardigrade genomics and molecular anhydrobiology. Annu. Rev. Anim. Biosci., 10: 17–37.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev-animal-021419-083711" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35167318/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Anim.%20Biosci.&amp;title=Examples%20of%20extreme%20survival:%20Tardigrade%20genomics%20and%20molecular%20anhydrobiology&amp;volume=10&amp;publication_year=2022&amp;pages=17-37&amp;pmid=35167318&amp;doi=10.1146/annurev-animal-021419-083711&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<cite>Boothby, T.C., Tapia, H., Brozena, A.H., Piszkiewicz, S., Smith, A.E., Giovannini, I., Rebecchi, L., Pielak, G.J., Koshland, D., and Goldstein, B.. 
2017. Tardigrades use intrinsically disordered proteins to survive desiccation. Mol. Cell, 65: 975–984.e5.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.molcel.2017.02.018" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5987194/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28306513/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Tardigrades%20use%20intrinsically%20disordered%20proteins%20to%20survive%20desiccation&amp;volume=65&amp;publication_year=2017&amp;pages=975-984&amp;pmid=28306513&amp;doi=10.1016/j.molcel.2017.02.018&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<cite>Bracha, D., Walls, M.T., Wei, M.-T., Zhu, L., Kurian, M., Avalos, J.L., Toettcher, J.E., and Brangwynne, C.P.. 
2018. Mapping local and global liquid phase behavior in living cells using photo-oligomerizable seeds. Cell, 175: 1467–1480.e13.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cell.2018.10.048" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6724719/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30500534/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Mapping%20local%20and%20global%20liquid%20phase%20behavior%20in%20living%20cells%20using%20photo-oligomerizable%20seeds&amp;volume=175&amp;publication_year=2018&amp;pages=1467-1480&amp;pmid=30500534&amp;doi=10.1016/j.cell.2018.10.048&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<cite>Burg, M.B., Ferraris, J.D., and Dmitrieva, N.I.. 
2007. Cellular response to hyperosmotic stresses. Physiol. Rev., 87: 1441–1474.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1152/physrev.00056.2006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17928589/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Physiol.%20Rev.&amp;title=Cellular%20response%20to%20hyperosmotic%20stresses&amp;volume=87&amp;publication_year=2007&amp;pages=1441-1474&amp;pmid=17928589&amp;doi=10.1152/physrev.00056.2006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<cite>Capasso, J.M., Rivard, C.J., Enomoto, L.M., and Berl, T.. 
2003. Chloride, not sodium, stimulates expression of the gamma subunit of Na/K-ATPase and activates JNK in response to hypertonicity in mouse IMCD3 cells. Proc. Natl. Acad. Sci. USA, 100: 6428–6433.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1130871100" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC164463/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12746499/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Chloride,%20not%20sodium,%20stimulates%20expression%20of%20the%20gamma%20subunit%20of%20Na/K-ATPase%20and%20activates%20JNK%20in%20response%20to%20hypertonicity%20in%20mouse%20IMCD3%20cells&amp;volume=100&amp;publication_year=2003&amp;pages=6428-6433&amp;pmid=12746499&amp;doi=10.1073/pnas.1130871100&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<cite>Cornette, R., Kanamori, Y., Watanabe, M., Nakahara, Y., Gusev, O., Mitsumasu, K., Kadono-Okuda, K., Shimomura, M., Mita, K., Kikawada, T., and Okuda, T.. 
2010. Identification of anhydrobiosis-related genes from an expressed sequence tag database in the cryptobiotic midge <em>Polypedilum vanderplanki</em> (diptera; chironomidae). J. Biol. Chem., 285: 35889–35899.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M110.150623" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2975212/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20833722/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Identification%20of%20anhydrobiosis-related%20genes%20from%20an%20expressed%20sequence%20tag%20database%20in%20the%20cryptobiotic%20midge%20Polypedilum%20vanderplanki%20(diptera;%20chironomidae)&amp;volume=285&amp;publication_year=2010&amp;pages=35889-35899&amp;pmid=20833722&amp;doi=10.1074/jbc.M110.150623&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<cite>Crowe, J.H., Carpenter, J.F., and Crowe, L.M.. 
1998. The role of vitrification in anhydrobiosis. Annu. Rev. Physiol., 60: 73–103.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev.physiol.60.1.73" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9558455/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Physiol.&amp;title=The%20role%20of%20vitrification%20in%20anhydrobiosis&amp;volume=60&amp;publication_year=1998&amp;pages=73-103&amp;pmid=9558455&amp;doi=10.1146/annurev.physiol.60.1.73&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<cite>García, A.H.
2011. Anhydrobiosis in bacteria: From physiology to applications. J. Biosci., 36: 939–950.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s12038-011-9107-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22116292/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biosci.&amp;title=Anhydrobiosis%20in%20bacteria:%20From%20physiology%20to%20applications&amp;volume=36&amp;publication_year=2011&amp;pages=939-950&amp;pmid=22116292&amp;doi=10.1007/s12038-011-9107-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<cite>Goyal, K., Walton, L.J., and Tunnacliffe, A.. 
2005. LEA proteins prevent protein aggregation due to water stress. Biochem. J., 388: 151–157.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1042/BJ20041931" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1186703/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15631617/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=LEA%20proteins%20prevent%20protein%20aggregation%20due%20to%20water%20stress&amp;volume=388&amp;publication_year=2005&amp;pages=151-157&amp;pmid=15631617&amp;doi=10.1042/BJ20041931&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<cite>Gusev, O., Nakahara, Y., Vanyagina, V., Malutina, L., Cornette, R., Sakashita, T., Hamada, N., Kikawada, T., Kobayashi, Y., and Okuda, T.. 
2010. Anhydrobiosis-associated nuclear DNA damage and repair in the sleeping chironomid: Linkage with radioresistance. PLoS One, 5: e14008.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0014008" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2982815/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21103355/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Anhydrobiosis-associated%20nuclear%20DNA%20damage%20and%20repair%20in%20the%20sleeping%20chironomid:%20Linkage%20with%20radioresistance&amp;volume=5&amp;publication_year=2010&amp;pages=e14008&amp;pmid=21103355&amp;doi=10.1371/journal.pone.0014008&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<cite>Hibshman, J.D., Clegg, J.S., and Goldstein, B.. 
2020. Mechanisms of desiccation tolerance: Themes and variations in brine shrimp, roundworms, and tardigrades. Front. Physiol., 11: 592016.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fphys.2020.592016" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7649794/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33192606/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Physiol.&amp;title=Mechanisms%20of%20desiccation%20tolerance:%20Themes%20and%20variations%20in%20brine%20shrimp,%20roundworms,%20and%20tardigrades&amp;volume=11&amp;publication_year=2020&amp;pages=592016&amp;pmid=33192606&amp;doi=10.3389/fphys.2020.592016&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<cite>Janis, B., Belott, C., and Menze, M.A.. 
2018. Role of intrinsic disorder in animal desiccation tolerance. Proteomics, 18: e1800067.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/pmic.201800067" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30144288/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Role%20of%20intrinsic%20disorder%20in%20animal%20desiccation%20tolerance&amp;volume=18&amp;publication_year=2018&amp;pages=e1800067&amp;pmid=30144288&amp;doi=10.1002/pmic.201800067&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<cite>Jumper, J., Evans, R., Pritzel, A., Green, T., Figurnov, M., Ronneberger, O., Tunyasuvunakool, K., Bates, R., Žídek, A., Potapenko, A., Bridgland, A., Meyer, C., Kohl, S.A.A., Ballard, A.J., Cowie, A., Romera-Paredes, B., Nikolov, S., Jain, R., Adler, J., et al. 
2021. Highly accurate protein structure prediction with AlphaFold. Nature, 596: 583–589.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41586-021-03819-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8371605/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34265844/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;volume=596&amp;publication_year=2021&amp;pages=583-589&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14">
<cite>Kawakami, K. and Noda, T.. 
2004. Transposition of the Tol2 element, an Ac-like element from the Japanese medaka fish Oryzias latipes, in mouse embryonic stem cells. Genetics, 166: 895–899.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1534/genetics.166.2.895" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1470731/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15020474/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Transposition%20of%20the%20Tol2%20element,%20an%20Ac-like%20element%20from%20the%20Japanese%20medaka%20fish%20Oryzias%20latipes,%20in%20mouse%20embryonic%20stem%20cells&amp;volume=166&amp;publication_year=2004&amp;pages=895-899&amp;pmid=15020474&amp;doi=10.1534/genetics.166.2.895&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15">
<cite>Keilin, D.
1959. The problem of anabiosis or latent life: History and current concept. Proc. R. Soc. Lond. B Biol. Sci., 150: 149–191.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1098/rspb.1959.0013" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/13633975/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.&amp;title=The%20problem%20of%20anabiosis%20or%20latent%20life:%20History%20and%20current%20concept&amp;volume=150&amp;publication_year=1959&amp;pages=149-191&amp;pmid=13633975&amp;doi=10.1098/rspb.1959.0013&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16">
<cite>Kikawada, T., Nakahara, Y., Kanamori, Y., Iwata, K.-I., Watanabe, M., McGee, B., Tunnacliffe, A., and Okuda, T.. 
2006. Dehydration-induced expression of LEA proteins in an anhydrobiotic chironomid. Biochem. Biophys. Res. Commun., 348: 56–61.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.bbrc.2006.07.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16875677/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;title=Dehydration-induced%20expression%20of%20LEA%20proteins%20in%20an%20anhydrobiotic%20chironomid&amp;volume=348&amp;publication_year=2006&amp;pages=56-61&amp;pmid=16875677&amp;doi=10.1016/j.bbrc.2006.07.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17">
<cite>King, A.M. and MacRae, T.H.. 
2015. Insect heat shock proteins during stress and diapause. Annu. Rev. Entomol., 60: 59–75.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev-ento-011613-162107" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25341107/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Entomol.&amp;title=Insect%20heat%20shock%20proteins%20during%20stress%20and%20diapause&amp;volume=60&amp;publication_year=2015&amp;pages=59-75&amp;pmid=25341107&amp;doi=10.1146/annurev-ento-011613-162107&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18">
<cite>Kondo, K., Kubo, T., and Kunieda, T.. 
2015. Suggested involvement of PP1/PP2A activity and De Novo gene expression in anhydrobiotic survival in a tardigrade, Hypsibius dujardini, by chemical genetic approach. PLoS One, 10: e0144803.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0144803" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4686906/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26690982/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Suggested%20involvement%20of%20PP1/PP2A%20activity%20and%20De%20Novo%20gene%20expression%20in%20anhydrobiotic%20survival%20in%20a%20tardigrade,%20Hypsibius%20dujardini,%20by%20chemical%20genetic%20approach&amp;volume=10&amp;publication_year=2015&amp;pages=e0144803&amp;pmid=26690982&amp;doi=10.1371/journal.pone.0144803&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19">
<cite>Kondo, K., Mori, M., Tomita, M., and Arakawa, K.. 
2019. AMPK activity is required for the induction of anhydrobiosis in a tardigrade Hypsibius exemplaris, and its potential up-regulator is PP2A. Genes Cells, 24: 768–780.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/gtc.12726" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31608545/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genes%20Cells&amp;title=AMPK%20activity%20is%20required%20for%20the%20induction%20of%20anhydrobiosis%20in%20a%20tardigrade%20Hypsibius%20exemplaris,%20and%20its%20potential%20up-regulator%20is%20PP2A&amp;volume=24&amp;publication_year=2019&amp;pages=768-780&amp;pmid=31608545&amp;doi=10.1111/gtc.12726&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B20">
<cite>Lapinski, J. and Tunnacliffe, A.. 
2003. Anhydrobiosis without trehalose in bdelloid rotifers. FEBS Lett., 553: 387–390.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s0014-5793(03)01062-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14572656/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&amp;title=Anhydrobiosis%20without%20trehalose%20in%20bdelloid%20rotifers&amp;volume=553&amp;publication_year=2003&amp;pages=387-390&amp;pmid=14572656&amp;doi=10.1016/s0014-5793(03)01062-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21">
<cite>Leprince, O. and Buitink, J.. 
2015. Introduction to desiccation biology: From old borders to new frontiers. Planta, 242: 369–378.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00425-015-2357-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26142353/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Introduction%20to%20desiccation%20biology:%20From%20old%20borders%20to%20new%20frontiers&amp;volume=242&amp;publication_year=2015&amp;pages=369-378&amp;pmid=26142353&amp;doi=10.1007/s00425-015-2357-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22">
<cite>MacRae, T.H.
2016. Stress tolerance during diapause and quiescence of the brine shrimp, Artemia. Cell Stress Chaperones, 21: 9–18.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s12192-015-0635-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4679736/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26334984/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell%20Stress%20Chaperones&amp;title=Stress%20tolerance%20during%20diapause%20and%20quiescence%20of%20the%20brine%20shrimp,%20Artemia&amp;volume=21&amp;publication_year=2016&amp;pages=9-18&amp;pmid=26334984&amp;doi=10.1007/s12192-015-0635-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23">
<cite>Malki, A., Teulon, J.-M., Camacho-Zarco, A.R., Chen, S.-W.W., Adamski, W., Maurin, D., Salvi, N., Pellequer, J.-L., and Blackledge, M.. 
2022. Intrinsically disordered tardigrade proteins self-assemble into fibrous gels in response to environmental stress. Angew. Chem. Int. Ed. Engl., 61: e202109961.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/anie.202109961" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9299615/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34750927/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.%20Int.%20Ed.%20Engl.&amp;title=Intrinsically%20disordered%20tardigrade%20proteins%20self-assemble%20into%20fibrous%20gels%20in%20response%20to%20environmental%20stress&amp;volume=61&amp;publication_year=2022&amp;pages=e202109961&amp;pmid=34750927&amp;doi=10.1002/anie.202109961&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24">
<cite>Niwa, H., Yamamura, K., and Miyazaki, J.. 
1991. Efficient selection for high-expression transfectants with a novel eukaryotic vector. Gene, 108: 193–199.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0378-1119(91)90434-d" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/1660837/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Efficient%20selection%20for%20high-expression%20transfectants%20with%20a%20novel%20eukaryotic%20vector&amp;volume=108&amp;publication_year=1991&amp;pages=193-199&amp;pmid=1660837&amp;doi=10.1016/0378-1119(91)90434-d&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25">
<cite>Rebecchi, L., Altiero, T., and Guidetti, R.. 
2007. Anhydrobiosis: The extreme limit of desiccation tolerance. Invertebr. Surviv. J., 4: 65–81.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Invertebr.%20Surviv.%20J.&amp;title=Anhydrobiosis:%20The%20extreme%20limit%20of%20desiccation%20tolerance&amp;volume=4&amp;publication_year=2007&amp;pages=65-81&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26">
<cite>Reuner, A., Hengherr, S., Mali, B., Förster, F., Arndt, D., Reinhardt, R., Dandekar, T., Frohme, M., Brümmer, F., and Schill, R.O.. 
2010. Stress response in tardigrades: Differential gene expression of molecular chaperones. Cell Stress Chaperones, 15: 423–430.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s12192-009-0158-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3082643/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19943197/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell%20Stress%20Chaperones&amp;title=Stress%20response%20in%20tardigrades:%20Differential%20gene%20expression%20of%20molecular%20chaperones&amp;volume=15&amp;publication_year=2010&amp;pages=423-430&amp;pmid=19943197&amp;doi=10.1007/s12192-009-0158-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27">
<cite>Rizzo, A.M., Negroni, M., Altiero, T., Montorfano, G., Corsetto, P., Berselli, P., Berra, B., Guidetti, R., and Rebecchi, L.. 
2010. Antioxidant defences in hydrated and desiccated states of the tardigrade Paramacrobiotus richtersi. Comp. Biochem. Physiol. B Biochem. Mol. Biol., 156: 115–121.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cbpb.2010.02.009" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20206711/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Comp.%20Biochem.%20Physiol.%20B%20Biochem.%20Mol.%20Biol.&amp;title=Antioxidant%20defences%20in%20hydrated%20and%20desiccated%20states%20of%20the%20tardigrade%20Paramacrobiotus%20richtersi&amp;volume=156&amp;publication_year=2010&amp;pages=115-121&amp;pmid=20206711&amp;doi=10.1016/j.cbpb.2010.02.009&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28">
<cite>Roney, I.J., Rudner, A.D., Couture, J.-F., and Kærn, M.. 
2016. Improvement of the reverse tetracycline transactivator by single amino acid substitutions that reduce leaky target gene expression to undetectable levels. Sci. Rep., 6: 27697.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/srep27697" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4914848/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27323850/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Improvement%20of%20the%20reverse%20tetracycline%20transactivator%20by%20single%20amino%20acid%20substitutions%20that%20reduce%20leaky%20target%20gene%20expression%20to%20undetectable%20levels&amp;volume=6&amp;publication_year=2016&amp;pages=27697&amp;pmid=27323850&amp;doi=10.1038/srep27697&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29">
<cite>Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J.Y., White, D.J., Hartenstein, V., Eliceiri, K., Tomancak, P., and Cardona, A.. 
2012. Fiji: An open-source platform for biological-image analysis. Nat. Methods, 9: 676–682.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.2019" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3855844/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22743772/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Fiji:%20An%20open-source%20platform%20for%20biological-image%20analysis&amp;volume=9&amp;publication_year=2012&amp;pages=676-682&amp;pmid=22743772&amp;doi=10.1038/nmeth.2019&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30">
<cite>Schokraie, E., Hotz-Wagenblatt, A., Warnken, U., Frohme, M., Dandekar, T., Schill, R.O., and Schnölzer, M.. 
2011. Investigating heat shock proteins of tardigrades in active versus anhydrobiotic state using shotgun proteomics. J. Zoolog. Syst. Evol. Res., 49: 111–119.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Zoolog.%20Syst.%20Evol.%20Res.&amp;title=Investigating%20heat%20shock%20proteins%20of%20tardigrades%20in%20active%20versus%20anhydrobiotic%20state%20using%20shotgun%20proteomics&amp;volume=49&amp;publication_year=2011&amp;pages=111-119&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31">
<cite>Tanaka, A., Nakano, T., Watanabe, K., Masuda, K., Honda, G., Kamata, S., Yasui, R., Kozuka-Hata, H., Watanabe, C., Chinen, T., Kitagawa, D., Sawai, S., Oyama, M., Yanagisawa, M., and Kunieda, T.. 
2022. Stress-dependent cell stiffening by tardigrade tolerance proteins that reversibly form a filamentous network and gel. PLoS Biol., 20: e3001780.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pbio.3001780" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9592077/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36067153/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&amp;title=Stress-dependent%20cell%20stiffening%20by%20tardigrade%20tolerance%20proteins%20that%20reversibly%20form%20a%20filamentous%20network%20and%20gel&amp;volume=20&amp;publication_year=2022&amp;pages=e3001780&amp;pmid=36067153&amp;doi=10.1371/journal.pbio.3001780&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32">
<cite>Tanaka, S., Aoki, K., and Arakawa, K.. 
2023. In vivo expression vector derived from anhydrobiotic tardigrade genome enables live imaging in Eutardigrada. Proc. Natl. Acad. Sci. USA, 120: e2216739120.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.2216739120" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9945988/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36693101/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=In%20vivo%20expression%20vector%20derived%20from%20anhydrobiotic%20tardigrade%20genome%20enables%20live%20imaging%20in%20Eutardigrada&amp;volume=120&amp;publication_year=2023&amp;pmid=36693101&amp;doi=10.1073/pnas.2216739120&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B33">
<cite>Tanaka, S., Tanaka, J., Miwa, Y., Horikawa, D.D., Katayama, T., Arakawa, K., Toyoda, A., Kubo, T., and Kunieda, T.. 
2015. Novel mitochondria-targeted heat-soluble proteins identified in the anhydrobiotic Tardigrade improve osmotic tolerance of human cells. PLoS One, 10: e0118272.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0118272" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4326354/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25675104/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Novel%20mitochondria-targeted%20heat-soluble%20proteins%20identified%20in%20the%20anhydrobiotic%20Tardigrade%20improve%20osmotic%20tolerance%20of%20human%20cells&amp;volume=10&amp;publication_year=2015&amp;pages=e0118272&amp;pmid=25675104&amp;doi=10.1371/journal.pone.0118272&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34">
<cite>Varadi, M., Anyango, S., Deshpande, M., Nair, S., Natassia, C., Yordanova, G., Yuan, D., Stroe, O., Wood, G., Laydon, A., Žídek, A., Green, T., Tunyasuvunakool, K., Petersen, S., Jumper, J., Clancy, E., Green, R., Vora, A., Lutfi, M., et al. 
2022. AlphaFold protein structure database: Massively expanding the structural coverage of protein-sequence space with high-accuracy models. Nucleic Acids Res., 50: D439–D444.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkab1061" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8728224/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34791371/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=AlphaFold%20protein%20structure%20database:%20Massively%20expanding%20the%20structural%20coverage%20of%20protein-sequence%20space%20with%20high-accuracy%20models&amp;volume=50&amp;publication_year=2022&amp;pages=D439-D444&amp;pmid=34791371&amp;doi=10.1093/nar/gkab1061&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35">
<cite>Veling, M.T., Nguyen, D.T., Thadani, N.N., Oster, M.E., Rollins, N.J., Brock, K.P., Bethel, N.P., Lim, S., Baker, D., Way, J.C., Marks, D.S., Chang, R.L., and Silver, P.A.. 
2022. Natural and designed proteins inspired by extremotolerant organisms can form condensates and attenuate apoptosis in human cells. ACS Synth. Biol., 11: 1292–1302.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/acssynbio.1c00572" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9651988/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35176859/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ACS%20Synth.%20Biol.&amp;title=Natural%20and%20designed%20proteins%20inspired%20by%20extremotolerant%20organisms%20can%20form%20condensates%20and%20attenuate%20apoptosis%20in%20human%20cells&amp;volume=11&amp;publication_year=2022&amp;pages=1292-1302&amp;pmid=35176859&amp;doi=10.1021/acssynbio.1c00572&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36">
<cite>Wright, J.C.
2001. Cryptobiosis 300 years on from van Leuwenhoek: What have we learned about tardigrades?
Zool. Anz., 240: 563–582.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Zool.%20Anz.&amp;title=Cryptobiosis%20300%20years%20on%20from%20van%20Leuwenhoek:%20What%20have%20we%20learned%20about%20tardigrades?&amp;volume=240&amp;publication_year=2001&amp;pages=563-582&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B37">
<cite>Yagi-Utsumi, M., Aoki, K., Watanabe, H., Song, C., Nishimura, S., Satoh, T., Yanaka, S., Ganser, C., Tanaka, S., Schnapka, V., Goh, E.W., Furutani, Y., Murata, K., Uchihashi, T., Arakawa, K., and Kato, K.. 
2021. Desiccation-induced fibrous condensation of CAHS protein from an anhydrobiotic tardigrade. Sci. Rep., 11: 21328.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-021-00724-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8569203/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34737320/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Desiccation-induced%20fibrous%20condensation%20of%20CAHS%20protein%20from%20an%20anhydrobiotic%20tardigrade&amp;volume=11&amp;publication_year=2021&amp;pages=21328&amp;pmid=34737320&amp;doi=10.1038/s41598-021-00724-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B38">
<cite>Yoshida, Y., Koutsovoulos, G., Laetsch, D.R., Stevens, L., Kumar, S., Horikawa, D.D., Ishino, K., Komine, S., Kunieda, T., Tomita, M., Blaxter, M., and Arakawa, K.. 
2017. Comparative genomics of the tardigrades Hypsibius dujardini and Ramazzottius varieornatus. PLoS Biol., 15: e2002266.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pbio.2002266" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5531438/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28749982/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&amp;title=Comparative%20genomics%20of%20the%20tardigrades%20Hypsibius%20dujardini%20and%20Ramazzottius%20varieornatus&amp;volume=15&amp;publication_year=2017&amp;pages=e2002266&amp;pmid=28749982&amp;doi=10.1371/journal.pbio.2002266&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B39">
<cite>Yoshida, Y., Satoh, T., Ota, C., Tanaka, S., Horikawa, D.D., Tomita, M., Kato, K., and Arakawa, K.. 
2022. Time-series transcriptomic screening of factors contributing to the cross-tolerance to UV radiation and anhydrobiosis in tardigrades. BMC Genomics, 23: 405.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12864-022-08642-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9145152/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35643424/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Time-series%20transcriptomic%20screening%20of%20factors%20contributing%20to%20the%20cross-tolerance%20to%20UV%20radiation%20and%20anhydrobiosis%20in%20tardigrades&amp;volume=23&amp;publication_year=2022&amp;pages=405&amp;pmid=35643424&amp;doi=10.1186/s12864-022-08642-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ol></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="data-citations" id="_adc93_" lang="en"><h3 class="pmc_sec_title">Data Citations</h3>
<ol class="ref-list font-sm"><li id="_dbx_ref_mixed-citation1">
<cite>Raw datasets acquired to investigate the possible roles of CAHS proteins from tardigrade in osmotic stress tolerance in mammalian cells and their analysis program. </cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.51021/data.csf.27948918" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC11930778/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/39566968/">PubMed</a>]</li></ol></section><section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11930778/bin/csf_49_24035_1.mp4">Download video file</a><sup> (738.8KB, mp4) </sup><p>Movie 1. </p>
<p>HeLa cells expressing mEGFP were loaded onto a microfluidic plate and grown for 1 day. Time-lapse imaging was performed using a spinning disk confocal microscope. The cells were treated with 0.5 M Sorbitol (left) and 0.2 M NaCl (right) at 2 min and incubated for 5 min, followed by the replacement of imaging medium. Images were acquired every 10 sec. Total imaging time = 17 min.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11930778/bin/csf_49_24035_2.mp4">Download video file</a><sup> (721.6KB, mp4) </sup><p>Movie 2. </p>
<p>HeLa cells expressing CAHS1-mEGFP were loaded onto a microfluidic plate and grown for 1 day. Time-lapse imaging was performed using a spinning disk confocal microscope. The cells were treated with 0.5 M Sorbitol (left) and 0.2 M NaCl (right) at 2 min and incubated for 5 min, followed by the replacement of imaging medium. Images were acquired every 10 sec. Total imaging time = 17 min.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11930778/bin/csf_49_24035_3.mp4">Download video file</a><sup> (724.6KB, mp4) </sup><p>Movie 3. </p>
<p>HeLa cells expressing CAHS3-mEGFP were loaded onto a microfluidic plate and grown for 1 day. Time-lapse imaging was performed using a spinning disk confocal microscope. The cells were treated with 0.5 M Sorbitol (left) and 0.2 M NaCl (right) at 2 min and incubated for 5 min, followed by the replacement of imaging medium. Images were acquired every 10 sec. Total imaging time = 17 min.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11930778/bin/csf_49_24035_4.mp4">Download video file</a><sup> (706.8KB, mp4) </sup><p>Movie 4. </p>
<p>HeLa cells expressing CAHS8-mEGFP were loaded onto a microfluidic plate and grown for 1 day. Time-lapse imaging was performed using a spinning disk confocal microscope. The cells were treated with 0.5 M Sorbitol (left) and 0.2 M NaCl (right) at 2 min and incubated for 5 min, followed by the replacement of imaging medium. Images were acquired every 10 sec. Total imaging time = 17 min.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11930778/bin/csf_49_24035_5.mp4">Download video file</a><sup> (716.5KB, mp4) </sup><p>Movie 5. </p>
<p>HeLa cells expressing CAHS12-mEGFP were loaded onto a microfluidic plate and grown for 1 day. Time-lapse imaging was performed using a spinning disk confocal microscope. The cells were treated with 0.5 M Sorbitol (left) and 0.2 M NaCl (right) at 2 min and incubated for 5 min, followed by the replacement of imaging medium. Images were acquired every 10 sec. Total imaging time = 17 min.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11930778/bin/csf_49_24035_6.pdf">Supplementary Materials</a><sup> (265.4KB, pdf) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The supporting information for this article is available in J-STAGE Data.</p>
<section class="ref-list"><section><ol class="ref-list font-sm"><li id="db_da_ref1_reqid_">
<cite>Raw datasets acquired to investigate the possible roles of CAHS proteins from tardigrade in osmotic stress tolerance in mammalian cells and their analysis program. </cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.51021/data.csf.27948918" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC11930778/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/39566968/">PubMed</a>]</li></ol></section></section></section></section></section></section></article>
</main></body></html>