<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4411111/"/><title>REGULATOR OF BULB BIOGENESIS1 (RBB1) Is Involved in Vacuole Bulb Formation in Arabidopsis - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">PLoS One</button></div>. 2015 Apr 27;10(4):e0125621. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0125621" rel="noopener noreferrer" target="_blank">10.1371/journal.pone.0125621</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>REGULATOR OF BULB BIOGENESIS1 (RBB1) Is Involved in Vacuole Bulb Formation in Arabidopsis</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20SW%22%5BAuthor%5D"><span class="name western">Sang Won Han</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sang Won Han</span></h3>
<div class="p">
<sup>1</sup>Department of Plant and Microbial Biology, North Carolina State University, Raleigh, North Carolina, United States of America</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20SW%22%5BAuthor%5D"><span class="name western">Sang Won Han</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alonso%20JM%22%5BAuthor%5D"><span class="name western">Jose M Alonso</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jose M Alonso</span></h3>
<div class="p">
<sup>1</sup>Department of Plant and Microbial Biology, North Carolina State University, Raleigh, North Carolina, United States of America</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alonso%20JM%22%5BAuthor%5D"><span class="name western">Jose M Alonso</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rojas-Pierce%20M%22%5BAuthor%5D"><span class="name western">Marcela Rojas-Pierce</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Marcela Rojas-Pierce</span></h3>
<div class="p">
<sup>1</sup>Department of Plant and Microbial Biology, North Carolina State University, Raleigh, North Carolina, United States of America</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rojas-Pierce%20M%22%5BAuthor%5D"><span class="name western">Marcela Rojas-Pierce</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Diane Bassham</span><sup>2</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Department of Plant and Microbial Biology, North Carolina State University, Raleigh, North Carolina, United States of America</div>
<div id="edit1">
<sup>2</sup>Iowa State University, UNITED STATES</div>
<div class="author-notes p">
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist.</p></div>
<div class="fn" id="contrib001"><p>Conceived and designed the experiments: SH MRP. Performed the experiments: SH. Analyzed the data: SH MRP. Contributed reagents/materials/analysis tools: JMA. Wrote the paper: SH MRP.</p></div>
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>mrojasp@ncsu.edu</span></p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Diane Bassham</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2015 Jan 8; Accepted 2015 Mar 24; Collection date 2015.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© 2015 Han et al</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC4411111  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25915922/">25915922</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Vacuoles are dynamic compartments with constant fluctuations and transient structures such as trans-vacuolar strands and bulbs. Bulbs are highly dynamic spherical structures inside vacuoles that are formed by multiple layers of membranes and are continuous with the main tonoplast. We recently carried out a screen for mutants with abnormal trafficking to the vacuole or aberrant vacuole morphology. We characterized <em>regulator of bulb biogenesis1-1</em> (<em>rbb1-1</em>), a mutant in Arabidopsis that contains increased numbers of bulbs when compared to the parental control. <em>rbb1-1</em> mutants also contain fewer transvacuolar strands than the parental control, and we propose the hypothesis that the formation of transvacuolar strands and bulbs is functionally related. We propose that the bulbs may function transiently to accommodate membranes and proteins when transvacuolar strands fail to elongate. We show that RBB1 corresponds to a very large protein of unknown function that is specific to plants, is present in the cytosol, and may associate with cellular membranes. RBB1 is involved in the regulation of vacuole morphology and may be involved in the establishment or stability of trans-vacuolar strands and bulbs.</p></section><section id="sec001"><h2 class="pmc_sec_title">Introduction</h2>
<p>The lytic vacuole in vegetative cells may occupy more 90% of the cell volume. This essential organelle is highly dynamic and has critical functions in cellular homeostasis, maintenance of turgor, storage and recycling [<a aria-describedby="pone.0125621.ref001" class="usa-link" href="#pone.0125621.ref001">1</a>–<a aria-describedby="pone.0125621.ref003" class="usa-link" href="#pone.0125621.ref003">3</a>]. Vacuoles are dynamic compartments with constant membrane fluctuations and transient and highly mobile structures such as trans-vacuolar strands (TVS), sheets and bulbs [<a aria-describedby="pone.0125621.ref004" class="usa-link" href="#pone.0125621.ref004">4</a>, <a aria-describedby="pone.0125621.ref005" class="usa-link" href="#pone.0125621.ref005">5</a>]. TVS are dynamic tubules delimited by the tonoplast that are thought to distribute cytoplasmic contents, including organelles between opposite sides of the cortical cytoskeleton [<a aria-describedby="pone.0125621.ref006" class="usa-link" href="#pone.0125621.ref006">6</a>]. In root hairs of <em>Hydrocharis dubia</em>, TVS form predomindantly along the tip growing axis and are thought to deliver cytosolic components to the growing tip [<a aria-describedby="pone.0125621.ref007" class="usa-link" href="#pone.0125621.ref007">7</a>, <a aria-describedby="pone.0125621.ref008" class="usa-link" href="#pone.0125621.ref008">8</a>]. Many TVS sorrounding the phragmoplast were also observed in tobacco BY-2 cells during cell division [<a aria-describedby="pone.0125621.ref009" class="usa-link" href="#pone.0125621.ref009">9</a>]. The actin cytoskeleton has been localized to TVS [<a aria-describedby="pone.0125621.ref010" class="usa-link" href="#pone.0125621.ref010">10</a>], and disruption of actin filaments results in their dissasembly [<a aria-describedby="pone.0125621.ref011" class="usa-link" href="#pone.0125621.ref011">11</a>]. Similarly, microinjection of antiserum against two actin-binding proteins of the villin group disrupted the TVS in root hair cells of <em>H</em>. <em>dubia</em> [<a aria-describedby="pone.0125621.ref007" class="usa-link" href="#pone.0125621.ref007">7</a>, <a aria-describedby="pone.0125621.ref012" class="usa-link" href="#pone.0125621.ref012">12</a>]. The current model for TVS formation is that the actomyosin system regulates TVS formation and dynamics [<a aria-describedby="pone.0125621.ref013" class="usa-link" href="#pone.0125621.ref013">13</a>, <a aria-describedby="pone.0125621.ref014" class="usa-link" href="#pone.0125621.ref014">14</a>].</p>
<p>Bulbs are spherical structures of diameters between 1 and 10 μm that are highly dynamic in the lumen of the vacuole. These membrane structures are associated with the outer tonoplast membrane or transvacuolar strands and contain cytoplasmic structures, even organelles, between the folded membranes [<a aria-describedby="pone.0125621.ref005" class="usa-link" href="#pone.0125621.ref005">5</a>, <a aria-describedby="pone.0125621.ref015" class="usa-link" href="#pone.0125621.ref015">15</a>]. Imaging by electron microscopy has indicated that bulbs are formed by multiple layers of tonoplast membranes, which explains the increased intensity of bulbs in fluorescently labeled tonoplasts [<a aria-describedby="pone.0125621.ref015" class="usa-link" href="#pone.0125621.ref015">15</a>]. Bulbs are present in LVs of cotyledon and leaf epidermis and their numbers decrease during cotyledon expansion [<a aria-describedby="pone.0125621.ref015" class="usa-link" href="#pone.0125621.ref015">15</a>], and leaf maturation [<a aria-describedby="pone.0125621.ref016" class="usa-link" href="#pone.0125621.ref016">16</a>]. Bulbs have been visualized by fluorescencence microscopy of fluorescent protein (FP) fusions with several tonoplast proteins and in multiple cell types. For example, in vacuoles of tobacco leaves, bulbs were observed when GFP-VAM3/SYP22 [<a aria-describedby="pone.0125621.ref017" class="usa-link" href="#pone.0125621.ref017">17</a>] or a GFP fusion with the phosphate transporter NPT2 [<a aria-describedby="pone.0125621.ref018" class="usa-link" href="#pone.0125621.ref018">18</a>] were transiently expressed. In Arabidopsis, bulbs have been detected in epidermis of the root elongation zone and the leaf, sepal bundle sheet, and germinating pollen of stably transformed plants expressing either GFP-AtVAM3 [<a aria-describedby="pone.0125621.ref005" class="usa-link" href="#pone.0125621.ref005">5</a>], TPK1-GFP [<a aria-describedby="pone.0125621.ref019" class="usa-link" href="#pone.0125621.ref019">19</a>, <a aria-describedby="pone.0125621.ref020" class="usa-link" href="#pone.0125621.ref020">20</a>], TIP1;1-GFP [<a aria-describedby="pone.0125621.ref021" class="usa-link" href="#pone.0125621.ref021">21</a>], or FP fusions with other tonoplast intrinsic proteins [<a aria-describedby="pone.0125621.ref016" class="usa-link" href="#pone.0125621.ref016">16</a>, <a aria-describedby="pone.0125621.ref022" class="usa-link" href="#pone.0125621.ref022">22</a>]. In addition, the phosphatidylinositol 3-phosphate sensor 2xFYVE-YFP also labels the bulbs [<a aria-describedby="pone.0125621.ref023" class="usa-link" href="#pone.0125621.ref023">23</a>]. These structures are not simply the result of ectopic expression of FP fusions because they were also detected in Arabidopsis WT plants by electron microscopy [<a aria-describedby="pone.0125621.ref015" class="usa-link" href="#pone.0125621.ref015">15</a>]. It was recently proposed that bulbs may form by dimerization of GFP or YFP tags when fused to tonoplast markers, and that some of these structures may be artifacts from this interaction [<a aria-describedby="pone.0125621.ref024" class="usa-link" href="#pone.0125621.ref024">24</a>]. Two types of structures were distinguished: bulbs associated with high levels of expression of GFP fusions that showed 3 or more fold increase in fluorescence intensity compared to the tonoplast, and “intra-vacuolar spherical structures” (IVSPs) present in transgenic lines in which the intensity of fluorescence was only two fold that of the tonoplast and were visualized with a non-dimerizing GFP molecule. This distinction was most dramatic in the root elongation zone where only very bright bulbs were observed, but it was more complex in other tissues, such as the cotyledon epidermis, where both types of structures existed [<a aria-describedby="pone.0125621.ref024" class="usa-link" href="#pone.0125621.ref024">24</a>].</p>
<p>Two hypotheses for the function of bulbs have been proposed. Bulbs may act as reservoirs of membranes for cell expansion or they may represent the first step for proteolysis of tonoplast proteins inside the vacuole [<a aria-describedby="pone.0125621.ref015" class="usa-link" href="#pone.0125621.ref015">15</a>, <a aria-describedby="pone.0125621.ref019" class="usa-link" href="#pone.0125621.ref019">19</a>]. Only two mutants with bulb phenotypes have been identified thus far. These are <em>sgr2-1</em> and <em>vti11</em>, and in both cases these mutants show a reduced number of bulbs [<a aria-describedby="pone.0125621.ref023" class="usa-link" href="#pone.0125621.ref023">23</a>]. SGR2 encodes a protein similar to phospholipase A1, and <em>sgr2</em> mutants have an abnormal vacuole morphology with multiple small vacuole-like structures and abnormal distribution of the vacuole and the cytosol in shoot endodermal cells [<a aria-describedby="pone.0125621.ref025" class="usa-link" href="#pone.0125621.ref025">25</a>]. VTI11 encodes a SNARE (soluble N-ethylmaleimide-sensitive factor attachment protein receptor) protein that mediates membrane fusion between pre-vacuolar compartments and the vacuole [<a aria-describedby="pone.0125621.ref026" class="usa-link" href="#pone.0125621.ref026">26</a>, <a aria-describedby="pone.0125621.ref027" class="usa-link" href="#pone.0125621.ref027">27</a>]. VTI11 is also involved in vacuole homotypic fusion, and <em>vti11</em> mutant alleles have highly fragmented vacuoles [<a aria-describedby="pone.0125621.ref028" class="usa-link" href="#pone.0125621.ref028">28</a>]. Given the highly abnormal vacuole morphology of these two mutants, it is unclear if the lack of bulbs is a direct or indirect effect of the loss of SGR2 and VTI11.</p>
<p>Here we describe the phenotype of <em>regulator of bulb biogenesis1</em> (<em>rbb1-1</em>), a novel mutant in Arabidopsis that displays increased numbers of bulbs in otherwise normal vacuoles. We show that <em>rbb1-1</em> mutants make fewer TVS than the parental control. We propose the hypothesis that RBB1 may function in TVS formation and that increased bulb accumulation in <em>rbb1-1</em> may result from impaired formation of TVS. In addition, we show that the differential labeling of bulbs between <em>rbb1-1</em> and the parental line does not result from abnormal trafficking to the vacuole. We also identified <em>RBB1</em> locus, which encodes a large plant-specific protein of unknown function.</p></section><section id="sec002"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="sec003"><h3 class="pmc_sec_title">Plant Material and Growth Conditions</h3>
<p>The parental line carrying GFP-TIP2;1 and mCherry-HDEL [<a aria-describedby="pone.0125621.ref029" class="usa-link" href="#pone.0125621.ref029">29</a>], and the marker lines TIP1;1-YFP [<a aria-describedby="pone.0125621.ref016" class="usa-link" href="#pone.0125621.ref016">16</a>] and SYP22-RFP [<a aria-describedby="pone.0125621.ref030" class="usa-link" href="#pone.0125621.ref030">30</a>] were previously described. SALK_095160, SALK_067590 and SALK_074387 were obtained from the Arabidopsis Biological Resource Center (ABRC) and genotyped by PCR. pUBQ10::RFP-TIP1;1 was generated by Gateway cloning (Invitrogen). Briefly, the TIP1;1 CDS was amplified from Col-0 cDNA by PCR and the amplicon was introduced into pENTR-D-TOPO vector (Invitrogen). A RFP-TIP1;1 expression clone was generated by recombination with pUBN-RFP [<a aria-describedby="pone.0125621.ref031" class="usa-link" href="#pone.0125621.ref031">31</a>] using LR clonase II (Invitrogen). The promoter of the resulting construct was substituted with the full pUBQ10 promoter (1,596 bp) from pNIGEL07 (Geldner et al 2009) to generate <em>pUBQ10</em>::<em>RFP-TIP1;1</em>. The <em>RBB1</em>::<em>3xYpet-RBB1</em> and <em>RBB1</em>::<em>RBB1-3xYpet</em> constructs were generated by a Recombineering-based gene tagging system [<a aria-describedby="pone.0125621.ref032" class="usa-link" href="#pone.0125621.ref032">32</a>] using the JAtY76M24 clone and the primers listed in <a class="usa-link" href="#pone.0125621.s009">S1 Table</a>. After introducing 3xYpet, the JAtY clone was shortened by homologous recombination using "delleft" and "RB-amp" primers (<a class="usa-link" href="#pone.0125621.s009">S1 Table</a>). This resulted in a fragment of the chromosomal region containing 12.8 kb upstream of start codon of <em>RBB1</em> and up to 5 kb downstream from the stop codon. These constructs were transformed into Agrobacterium GV3101 and then transformed into Col-0 wild type plants via floral dip [<a aria-describedby="pone.0125621.ref033" class="usa-link" href="#pone.0125621.ref033">33</a>].</p>
<p>Seeds were plated on AGM (0.5X MS media and 1% sucrose) with 4 g /L GelRite. For light-grown seedlings, seeds were cold treated for 4 days in the dark to break dormancy and then incubated at 22°C under a 16 h light photoperiod. For dark-grown seedlings, seeds were cold treated in the dark and then incubated at 22°C while covered in aluminum foil.</p></section><section id="sec004"><h3 class="pmc_sec_title">Chemical Stocks and Treatments</h3>
<p>C834 (ID 6982834) was from Chembridge, Brefeldin A from Sigma and Lysotracker Red and FM4-64 from Invitrogen. All chemicals were dissolved in 100% DMSO and all treatments were carried out with 4-day old seedlings, unless specified. BFA treatment was done in liquid AGM containing 75 μM BFA and for 3 h. C834 treatment was done with 3-d-old seedlings in solid AGM-GelRite supplemented with 55μM C834 for 48 hours as described [<a aria-describedby="pone.0125621.ref029" class="usa-link" href="#pone.0125621.ref029">29</a>]. For staining, seedlings were incubated in liquid AGM containing 2 μM Lysotracker Red for 2 h or 5 μM FM4-64 for 5 min.</p></section><section id="sec005"><h3 class="pmc_sec_title">Microscopy</h3>
<p>A Zeiss LSM 710 confocal microscope was used for all imaging experiments. A 40x water objective (1.1 N.A.) was used to image embryos and roots. A 20x objective (0.8 N.A.) was used to image hypocotyls and cotyledons. The excitation/emission wavelengths during acquisition were 488 nm/492–570 nm for GFP and Ypet, 561 nm/588–696 nm for mCherry and RFP, 514 nm/588–700 nm for FM4-64, and 561 nm/566-690 nm for Lysotracker red. To count the number of TVS, seedlings were imaged using a large pinhole to generate 600 μm thick optical sections for hypocotyls and 102 μm sections for cotyledons. A 40X objective was used and image zoom was kept at 0.6. Epidermal cells within the bottom half of the hypocotyl, but excluding the root-shoot junction, were used for TVS counting in hypocotyls. Epidermal cells from the adaxial side were used for cotyledons. TVS were counted in each cell from the collected images.</p></section><section id="sec006"><h3 class="pmc_sec_title">RT-PCR</h3>
<p>Total mRNA was extracted using TRI reagent (Ambion) and treated with DNase I (NEB). cDNA was synthesized using iScript cDNA Synthesis Kit (BioRad) and was used a template for RT-PCR. Oligo sequences used for PCR are shown in <a class="usa-link" href="#pone.0125621.s010">S2 Table</a> and 30 cycles were used for all reactions.</p></section><section id="sec007"><h3 class="pmc_sec_title">Map-based cloning and genome sequencing</h3>
<p>A mapping population was generated from a cross between <em>rbb1-1</em> and the L<em>er</em> wild type background, and segregating homozygous <em>rbb1-1</em> mutants were identified based on the bulb phenotype. Two Single Sequence Length Polymorphism (SSLP) markers [<a aria-describedby="pone.0125621.ref034" class="usa-link" href="#pone.0125621.ref034">34</a>] co-segregated with the <em>rbb1-1</em> phenotype and placed the mutation in Chromosome 5. For SNP detection, 12 homozygous F<sub>3</sub> families from 1 backcross to the parental line were selected based on the bulb phenotype and 500 seedlings per family were used to generate DNA libraries. Samples were sequenced in an Illumina HiSeq sequencer as 100 bp reads at BGI (formerly Beijing Genomics Institute). The expected genome coverage was 35X. Data analysis was carried out at the Bioinformatics Consulting and Service Core at North Carolina State University as follows. Sequence reads were aligned to the TAIR10 genome using bowtie2 and variant calls were made using samtools mpileup. Variants were filtered for those located within TAIR10 genes and data was exported to a spreadsheet. Only SNPs located between markers RBCS-B and PINHEAD in Chromosome 5 were analyzed further. From these, mutations that were originally present in the parental line were identified from the sequencing of an unrelated mutant from the same mutagenesis screen. In addition, mutations in introns or UTRs, silent mutations and mutations unlikely to result from EMS mutagenesis were discarded.</p></section><section id="sec008"><h3 class="pmc_sec_title">Western Blot</h3>
<p>Total proteins were extracted from 7 day-old seedlings. Samples were homogenized in 50mM HEPES-KOH, pH 6.5, 5 mM EDTA, 8% sucrose, 1mM DTT and proteinase inhibitors (Complete Mini, EDTA-free, Roche Diagnostics) in a chilled mortar. The homogenates were spun at 5,000g for 10 min at 4°C and the supernatant was filtered through 3 layers of miracloth. For membrane protein fractions (P100), filtered total proteins were spun at 100,000g for 1 h at 4°C and the pellet was resuspended in buffer. Protein concentration was quantified by the Bradford assay. For immunoblot of 3xYpet-RBB1, 8 μg of proteins were loaded in a NuPAGE 3–8% Tris-Acetate gel (Life Technologies) following manufacturer instructions. The following antibodies were used: anti-GFP (Thermo MA5-15256), anti-cFBase (Agrisera AS04043), anti-H<sup>+</sup>ATPase (Agrisera AS07260) and anti-calreticulin [<a aria-describedby="pone.0125621.ref035" class="usa-link" href="#pone.0125621.ref035">35</a>]. Peroxidase-Conjugated Goat Anti-Mouse IgG (Thermo 32430) or peroxidase-conjugated Goat Anti-Rabbit (Thermo 32460) antibodies were used as secondary antibodies. Immunoblots were visualized by chemiluminescence with Clarity Western ECL Substrate (BioRad) using a G:BOX Chemi XRQ system (Syngene) for image capture. Signal was quantified using the GeneTools image analysis software (Syngene).</p></section></section><section id="sec009"><h2 class="pmc_sec_title">Results</h2>
<section id="sec010"><h3 class="pmc_sec_title">GFP-TIP2;1 labels excess bulbs in <em>regulator of bulb biogenesis1</em>
</h3>
<p>We recently carried out a screen for mutants in tonoplast protein trafficking and vacuole biogenesis and characterized the function of the VTI11 SNARE protein and phosphoinositides in vacuole fusion [<a aria-describedby="pone.0125621.ref028" class="usa-link" href="#pone.0125621.ref028">28</a>]. During the course of this work, we identified a mutant in which the tonoplast protein GFP-TIP2;1 labeled many vacuolar bulbs and it was named <em>regulator of bulb biogenesis1-1</em> (<em>rbb1-1)</em>. In the parental line, the GFP-TIP2;1 protein labels the tonoplast and very few bulbs in root epidermis and cortex and the epidermis of hypocotyls, cotyledons and rosette leaves (<a class="usa-link" href="#pone.0125621.g001">Fig 1A–1D</a>) [<a aria-describedby="pone.0125621.ref016" class="usa-link" href="#pone.0125621.ref016">16</a>, <a aria-describedby="pone.0125621.ref028" class="usa-link" href="#pone.0125621.ref028">28</a>, <a aria-describedby="pone.0125621.ref036" class="usa-link" href="#pone.0125621.ref036">36</a>]. In the <em>rbb1-1</em> mutant, GFP-TIP2;1 also labels the tonoplast, and the size and morphology of the vacuole appeared normal. However, many more bulbs were evident as bright sub-vacuolar structures in multiple tissues (<a class="usa-link" href="#pone.0125621.g001">Fig 1E–1H</a>). In hypocotyls, bulbs were observed consistently in many cells in the epidermis and cortex (<a class="usa-link" href="#pone.0125621.g001">Fig 1E</a>). Bulbs were also apparent in high numbers in the epidermis of cotyledons and rosette leaves of <em>rbb1-1</em> (<a class="usa-link" href="#pone.0125621.g001">Fig 1F and 1H</a>). Vacuolar bulbs were detected in roots (<a class="usa-link" href="#pone.0125621.g001">Fig 1G</a>), but this phenotype was variable amongst different seed stocks and was not characterized further. The variability in the root phenotype may be due to root-specific modifying factors that segregate in the population. No major defects in plant morphology were detected in <em>rbb1-1</em> homozygous plants (<a class="usa-link" href="#pone.0125621.s001">S1 Fig</a>). Sequencing demonstrated that the <em>GFP-TIP2;1</em> transgene is intact in <em>rbb1-1</em> and consequently, the bulb phenotype is not related to a mutation in the GFP-TIP2;1 protein sequence.</p>
<figure class="fig xbox font-sm" id="pone.0125621.g001"><h4 class="obj_head">Fig 1. 
<em>regulator of bulb biogenesis1</em> contains many bulbs in many cell types.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4411111_pone.0125621.g001.jpg" target="_blank"><img alt="Fig 1" class="graphic zoom-in" height="692" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1845/4411111/a4e9d3622d25/pone.0125621.g001.jpg" width="732"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/pone.0125621.g001/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(a-h) <em>rbb1-1</em> mutants have more bulbs than the parental line. GFP-TIP2;1 localization in the parental line (WT, a-d) or <em>rbb1-1</em> (e-h). Four-day-old seedlings were imaged by confocal microscopy to visualize morphology of the vacuole in the epidermis of hypocotyl (a, e) and cotyledon (b, f), and in root epidermis and cortex (c, g). The bulb phenotype was also observed in rosette leaves of 6-week-old plants (d, h). Scale bar = 20 μm. (i-n) GFP-TIP2;1 and RFP-TIP1;1 co-localize in <em>rbb1-1</em> bulbs. The hypocotyls of 4-day-old dark-grown seedlings expressing GFP-TIP2;1 (green, i, l) and RFP-TIP1;1 (magenta, j, m) in the parental line (i-k) and <em>rbb1-1</em> mutants (l-n) are shown. Merged images (k, n) are also shown. Scale bar: 20 μm. (o) <em>rbb1-1</em> mutants do not accumulate higher levels of GFP-TIP2;1 in seedlings. Immunoblot of GFP-TIP2;1 accumulation in the parental line and <em>rbb1-1</em> using antibodies against GFP (α-GFP) and Calreticulin (α-CRT) as loading control.</p></figcaption></figure><p>TIP1;1-YFP is often found in vacuolar bulbs [<a aria-describedby="pone.0125621.ref016" class="usa-link" href="#pone.0125621.ref016">16</a>, <a aria-describedby="pone.0125621.ref023" class="usa-link" href="#pone.0125621.ref023">23</a>, <a aria-describedby="pone.0125621.ref037" class="usa-link" href="#pone.0125621.ref037">37</a>]. In order to determine if the <em>rbb1-1</em> bulbs accumulate TIP1;1 in a similar manner, we introduced a RFP-TIP1;1 fusion in the <em>rbb1-1</em> mutant background. In the parental line, RFP-TIP1;1 localized to the tonoplast, and labeled very few bulbs in roots and hypocotyls (<a class="usa-link" href="#pone.0125621.g001">Fig 1I–1K</a>). RFP-TIP1;1 co-localized with GFP-TIP2;1 in the tonoplast and the bulbs when the two markers were co-expressed in <em>rbb1-1</em> (<a class="usa-link" href="#pone.0125621.g001">Fig 1L–1N</a>). It was previously proposed that vacuolar bulbs are involved in the degradation of membrane proteins [<a aria-describedby="pone.0125621.ref019" class="usa-link" href="#pone.0125621.ref019">19</a>] and that bulbs may form as a response to overexpression of dimer-type FP [<a aria-describedby="pone.0125621.ref024" class="usa-link" href="#pone.0125621.ref024">24</a>]. We then tested whether the increased number of bulbs in <em>rbb1-1</em> could be caused by enhanced accumulation of GFP-TIP2;1 in <em>rbb1-1</em> by Western blot. No significant differences in GFP-TIP2;1 protein accumulation were detected between <em>rbb1-1</em> and the parental control, indicating that the increased number of bulbs in <em>rbb1-1</em> is not related to differences in protein accumulation or stability (<a class="usa-link" href="#pone.0125621.g001">Fig 1O</a> and <a class="usa-link" href="#pone.0125621.s001">S1C Fig</a>). These results overall indicate that, independent on the nature of the bulbs in <em>rbb1-1</em>, the <em>RBB1</em> locus is required to prevent the formation or accumulation of bulbs in the vacuole.</p>
<p>In order to compare <em>rbb1-1</em> bulbs with the IVSPs and bulbs previously described [<a aria-describedby="pone.0125621.ref024" class="usa-link" href="#pone.0125621.ref024">24</a>], we measured the maximum intensity values for bulb membranes in <em>rbb1-1</em>. As shown in <a class="usa-link" href="#pone.0125621.s002">S2 Fig</a>, maximum intensity values for the bulbs in the parental line are 1.2- and 1.3-fold higher than the outer tonoplast in cotyledons and hypocotyls, respectively. In contrast, <em>rbb1-1</em> bulbs have 2 fold higher intensity than the tonoplast in hypocotyl, and 3.5-fold higher in cotyledons. Therefore, <em>rbb1-1</em> bulbs would fit the description of both IVSPs and bulbs [<a aria-describedby="pone.0125621.ref024" class="usa-link" href="#pone.0125621.ref024">24</a>]. Moreover, given the fact that the GFP-TIP2;1 protein levels in <em>rbb1-1</em> mutants is similar to the parental line, we conclude that the <em>rbb1-1</em> bulbs are not simply the result of increased GFP dimerization in the mutant. However, we can not discard the possibility that GFP dimerization acts as an enhancer of the <em>rbb1</em> phenotype. Given the complexity of these structures, we will refer to them here simply as bulbs.</p>
<p>The vacuole membrane in many plant species constantly remodels with the formation of transvacuolar strands (TVS). We then hypothesized that the mechanisms involved in bulb formation and the dynamics of TVS may be related and that <em>RBB1</em> may have a role in TVS formation. In order to test this possibility, we compared the number of TVS between the parental line and <em>rbb1-1</em> mutants. In contrast to the parental line, which shows 4.06 ± 0.39 TVS per cell in cotyledon epidermis, <em>rbb1-1</em> mutants have only 1.40 ± 0.16 (<a class="usa-link" href="#pone.0125621.g002">Fig 2A</a> and <a class="usa-link" href="#pone.0125621.s003">S3A and S3B Fig</a>). Therefore, <em>rbb1-1</em> has ~3-fold fewer TVS than the parental line, and this result provides support to a hypothesis that RBB1 is involved in TVS formation. This phenotype was also observed in the hypocotyls of light- and dark-grown seedlings. Thus, the parental line showed an average of 2.2 ± 0.2 TVS per cell in the light and 1.75 ± 0.21 in the dark, while the <em>rbb1-1</em> mutant had 1.25 ± 0.15 and 1.03 ± 0.19, respectively (<a class="usa-link" href="#pone.0125621.g002">Fig 2B</a>). This indicates that <em>rbb1-1</em> had a 40% reduction in the number of TVS per cell in hypocotyls in both light and dark-grown seedlings. These results point towards important roles of RBB1 in the remodeling of the vacuole that generates either bulbs or TVS.</p>
<figure class="fig xbox font-sm" id="pone.0125621.g002"><h4 class="obj_head">Fig 2. 
<em>rbb1-1</em> mutants have fewer transvacuolar strands.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4411111_pone.0125621.g002.jpg" target="_blank"><img alt="Fig 2" class="graphic zoom-in" height="315" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1845/4411111/4673a8d08a9e/pone.0125621.g002.jpg" width="750"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/pone.0125621.g002/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(a) The number of trans-vacuolar strands (TVS) per cell was counted in cotyledon cells from the parental line and <em>rbb1-1</em> seedlings that were grown in the light. n = 48 cells from 8 seedlings. (b) <em>rbb1-1</em> mutants have fewer TVS in hypocotyl from light- and dark-grown seedlings. Seedlings were germinated in the light or dark for 4 days and TVS were counted. n = 60 cells from 10 seedlings. Bars represent standard error. * Significantly different to the parental line in a t-test (P ≤ 0.05).</p></figcaption></figure></section><section id="sec011"><h3 class="pmc_sec_title">The <em>rbb1-1</em> phenotype can be detected 3 days post-germination</h3>
<p>In order to characterize the developmental progression of the bulb phenotype in <em>rbb1-1</em>, vacuoles were visualized at different times during seed germination (<a class="usa-link" href="#pone.0125621.g003">Fig 3A–3J</a>). No bulbs were observed in PSVs of imbibed seeds or 1 d after incubation at 22°C in either the parental line or <em>rbb1-1</em> (<a class="usa-link" href="#pone.0125621.g003">Fig 3A, 3B, 3F and 3G</a>). Bulbs were first detected at day 2 post-germination in both the parental line and <em>rbb1-1</em> when the large lytic vacuole is already established [<a aria-describedby="pone.0125621.ref016" class="usa-link" href="#pone.0125621.ref016">16</a>]. The bulbs were very transient in the parental line and very few were observed at 3 days or later. In contrast, the number of bulbs in <em>rbb1-1</em> continued to increase at days 3 and 4. These results indicate that <em>RBB1</em> function may be important after day 3, but not during the early stages of germination.</p>
<figure class="fig xbox font-sm" id="pone.0125621.g003"><h4 class="obj_head">Fig 3. Developmental progression of the <em>rbb1-1</em> phenotype and effects from light.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4411111_pone.0125621.g003.jpg" target="_blank"><img alt="Fig 3" class="graphic zoom-in" height="675" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1845/4411111/79878dfe87df/pone.0125621.g003.jpg" width="744"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/pone.0125621.g003/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(a-j) The bulb phenotype is apparent in hypocotyls after 3 d of germination. Germinating seedlings from the parental line (GFP-TIP2;1, a-e) and <em>rbb1-1</em> mutants (f-j) were imaged after 0–4 days of incubation. (k-n) The bulb phenotype of <em>rbb1-1</em> is present in light- and dark-grown seedlings. GFP-TIP2;1 localization in hypocotyls was visualized in 4-d-old seedlings from the parental line (WT, k, m) or <em>rbb1-1</em> (l, n) when grown under light (k, l) or dark (m, n) conditions. n = 36 cells from 6 seedlings. All scale bars = 20 μm. (o) Dark treatment does not alter the <em>rbb1-1</em> phenotype. The number of bulbs was counted in hypocotyls from parental line and <em>rbb1-1</em> seedlings that were grown in the light or in the dark as in k-n. n = 9–15 seedlings. Bars represent standard error. (p) <em>rbb1-1</em> hypocotyls have fewer TVS after 3 d of germination. Seedlings were germinated in the light and TVS were counted after 2 days of incubation. n = 50 cells from 5 seedlings. Bars represent standard error. * Significantly different to the parental line in a t-test (P ≤ 0.01).</p></figcaption></figure><p>One proposed function of the bulbs is to contribute membranes during cell expansion [<a aria-describedby="pone.0125621.ref015" class="usa-link" href="#pone.0125621.ref015">15</a>]. This prompted us to investigate whether the <em>rbb1-1</em> phenotype was affected by conditions that promoted cell expansion such as the elongation of hypocotyls in the dark. To this end, vacuoles from the hypocotyl epidermis of parental and <em>rbb1-1</em> seedlings that were grown either in the light or the dark were imaged in the microscope, and bulbs were counted. While the parental control seedlings grown in the light contain 0.10 ± 0.03 bulbs per 10<sup>3</sup> μm<sup>2</sup> (<a class="usa-link" href="#pone.0125621.g003">Fig 3K and 3O</a>), the <em>rbb1-1</em> mutant contains 1.00 ± 0.16 (<a class="usa-link" href="#pone.0125621.g003">Fig 3L and 3O</a>). Therefore, the <em>rbb1-1</em> mutant contains 10 fold more bulbs per area in hypocotyls when compared to the parental control grown under normal light conditions. Next we analyzed the phenotype of <em>rbb1-1</em> mutants in the dark. In contrast to the dark-grown parental control, which contains 0.07 ± 0.01 bulbs per 10<sup>3</sup> μm<sup>2</sup> (<a class="usa-link" href="#pone.0125621.g003">Fig 3M and 3O</a>), the dark-grown <em>rbb1-1</em> mutant showed 1.18 ± 0.09 bulbs per 10<sup>3</sup> μm<sup>2</sup> (<a class="usa-link" href="#pone.0125621.g003">Fig 3N and 3O</a>), or a 16-fold increase. Thus, the number of bulbs in <em>rbb1-1</em> was higher than the parental under both light treatments, but light availability did not significantly alter the <em>rbb1-1</em> phenotype. We then tested whether the increased number of bulbs would enhance the elongation rate of dark-grown <em>rbb1-1</em> seedlings, but no differences in hypocotyl length were detected between the mutant and the parental control at 3 or 5 d (<a class="usa-link" href="#pone.0125621.s003">S3C Fig</a>). Furthermore, cell elongation was not affected, as we could not detect significant differences in cell size between the wild type and <em>rbb1-1</em> hypocotyls when grown in the dark (<a class="usa-link" href="#pone.0125621.s003">S3D Fig</a>).</p>
<p>Finally, if the <em>RBB1</em> is responsible for the bulb and TVS phenotypes of <em>rbb1-1</em>, we expected a similar developmental progression, which can be tested by quantification of TVS during germination. TVS could not clearly be detected before day 2, and no significant difference was detected at day 2 between wild type and <em>rbb1-1</em> hypocotyls when grown in the light (<a class="usa-link" href="#pone.0125621.g003">Fig 3P</a>) However, the number of TVS was reduced by 28% in <em>rbb1-1</em> when compared to the wild type. This reduction was even greater (41%) at day 4. Therefore, the appearance of the bulb phenotype at days 3 and 4 correlates well with the reduction of TVS numbers in <em>rbb1-1</em>.</p></section><section id="sec012"><h3 class="pmc_sec_title">GFP-TIP2;1 trafficking to the vacuole is not altered in <em>rbb1-1</em>
</h3>
<p>The increased number of bulbs in <em>rbb1-1</em> could be the result of abnormal GFP-TIP2;1 protein trafficking to the vacuole. To test this hypothesis, two inhibitors for tonoplast protein trafficking were used. Brefeldin A (BFA) inhibits a Golgi-dependent pathway for tonoplast proteins in hypocotyls and C834 inhibits a Golgi-independent traffic of GFP-TIP2;1 to the tonoplast [<a aria-describedby="pone.0125621.ref029" class="usa-link" href="#pone.0125621.ref029">29</a>]. We treated the GFP-TIP2;1 (parental control), the TIP1;1-YFP line and the <em>rbb1-1</em> mutant with BFA and visualized each fusion protein in hypocotyls. The fusion proteins localized to the tonoplast (GFP-TIP2;1) or tonoplast and bulbs (<em>rbb1-1</em> and TIP1;1-YFP) in the DMSO control as expected (<a class="usa-link" href="#pone.0125621.g004">Fig 4A–4C</a>). In BFA-treated cells, the GFP-TIP2;1 localization was unchanged in the parental control (<a class="usa-link" href="#pone.0125621.g004">Fig 4D</a>) or <em>rbb1-1</em> (<a class="usa-link" href="#pone.0125621.g004">Fig 4E</a>), indicating that the trafficking of this protein was still BFA-insensitive. In contrast and consistent with previous reports [<a aria-describedby="pone.0125621.ref029" class="usa-link" href="#pone.0125621.ref029">29</a>], the TIP1;1-YFP marker line displayed large and bright aggregates in the presence of BFA (<a class="usa-link" href="#pone.0125621.g004">Fig 4F</a>). Next we tested sensitivity to C834. GFP-TIP2;1 localized to the tonoplast and did not co-localize with the ER marker mCherry-HDEL in neither the parental line or <em>rbb1-1</em> (<a class="usa-link" href="#pone.0125621.g004">Fig 4G–4I and 4M–4O</a>). In contrast, GFP-TIP2;1 co-localized with the ER marker in C834-treated roots of the parental line (<a class="usa-link" href="#pone.0125621.g004">Fig 4J–4L</a>) and <em>rbb1-1</em> (<a class="usa-link" href="#pone.0125621.g004">Fig 4P–4R</a>). Therefore, we conclude that GFP-TIP2;1 traffics in a similar pathway in <em>rbb1-1</em> and the parental line, at least early in the secretory pathway, and that the bulb-localization of GFP-TIP2;1 in <em>rbb1-1</em> does not appear to be the result of altered trafficking to the vacuole.</p>
<figure class="fig xbox font-sm" id="pone.0125621.g004"><h4 class="obj_head">Fig 4. GFP-TIP2;1 trafficking to the vacuole is similar between <em>rbb1-1</em> and the parental line.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig 4" class="graphic" height="933" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1845/4411111/bede2006a7ed/pone.0125621.g004.jpg" width="652"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/pone.0125621.g004/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(a-f) BFA inhibits trafficking of TIP1;1YFP but not GFP-TIP2;1 in the parental line or <em>rbb1-1</em>. Four-day-old seedlings from the parental (GFP-TIP2;1, a, d), <em>rbb1-1</em> (b, e) or TIP1;1-YFP (c, f) were exposed to 0.1% DMSO (control, a, b, c) or 75 μM BFA (d, e, f) for 3 hours and hypocotyl cells were imaged. BFA compartments are indicated with arrows. Scale: 20 μm. (g-r) C834 inhibits GFP-TIP2;1 trafficking in the parental line and <em>rbb1-1</em>. Three-day-old seedlings from the parental line and <em>rbb1-1</em> were exposed to 0.5% DMSO (g-i, m-o) or 55 μM C834 (j-l, p-r) for 48 h, and root cells were imaged in the microscope. Signal from GFP-TIP2;1 (green, g, j, m, p), the ER marker mCherry-HDEL (magenta, h, k, n, p) or the merged image is shown (i, l, o, r). Co-localization of GFP with mCherry-HDEL at the ER is indicated with arrowheads. Scale bar: 20 μm.</p></figcaption></figure></section><section id="sec013"><h3 class="pmc_sec_title">The <em>RBB1</em> locus corresponds to <em>At5g40450</em>
</h3>
<p>The bulb phenotype of <em>rbb1-1</em> segregated as 25 out of 96 (1/3.84) in a backcross to the GFP-TIP2;1 parental line, suggesting that the <em>rbb1-1</em> mutation is recessive. In order to identify the <em>RBB1</em> locus, we used a combination of map-based cloning and whole genome sequencing. The <em>RBB1</em> gene mapped to a region of 2.2Mb in Chromosome 5 between markers RBCS-B and PINHEAD. Using Illumina whole-genome sequencing, we identified nucleotide variants in the <em>rbb1-1</em> mutant genome between these two markers. From those, only two SNPs located within gene coding sequences and corresponding to EMS-type non-silent mutations were detected, in <em>At5g38540</em> and <em>At5g40450</em>. In order to determine which gene corresponded to the <em>RBB1</em> locus, we identified Salk T-DNA insertion alleles [<a aria-describedby="pone.0125621.ref038" class="usa-link" href="#pone.0125621.ref038">38</a>] for both candidate genes. Only one T-DNA line, SALK_095160, for <em>At5g38540</em> was available, but no homozygote plants could be recovered. Heterozygous plants were crossed to <em>rbb1-1</em> and the vacuole phenotype of F<sub>1</sub> seedlings grown in the dark was analyzed. Out of 27 F<sub>1</sub> seedlings, none showed the <em>rbb1-1</em> bulb phenotype indicating that <em>RBB1</em> was unlikely to correspond to <em>At5g38540</em>. We obtained two SALK T-DNA alleles for <em>At5g40450</em> (<a class="usa-link" href="#pone.0125621.g005">Fig 5A</a>). However, one of the T-DNA lines, SALK_067590, had a complex transgene insertion and could not be used. We used RT-PCR to determine if <em>At5g40450</em> was expressed in SALK_074387. The expression of the full length <em>At5g40450</em> was disrupted in SALK_074387 (<a class="usa-link" href="#pone.0125621.g005">Fig 5B</a>), and therefore, this was a good mutant allele to test for the bulb phenotype. However, it is unclear if this allele is a complete loss of function allele because partial transcripts can be detected by RT-PCR. We named this allele <em>rbb1-2</em>.</p>
<figure class="fig xbox font-sm" id="pone.0125621.g005"><h4 class="obj_head">Fig 5. The <em>RBB1</em> locus corresponds to <em>At5g40450</em>.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig 5" class="graphic" height="1277" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1845/4411111/29d412512cb6/pone.0125621.g005.jpg" width="652"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/pone.0125621.g005/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(a) Schematic structure of the <em>RBB1</em> locus indicating the positions of <em>rbb1</em> alleles. Exons are indicated as black boxes and introns are indicated as white boxes. 5' and 3' UTRs are indicated in gray. Triangles indicate T-DNA insertions of SALK lines. The position of three diagnostic RT-PCR amplicons at the 5’ end, middle and 3’ end are shown. (b) Accumulation of <em>RBB1</em> transcripts as determined by RT-PCR with gene specific primers in seedlings from Col-0 (WT), <em>rbb1-2</em>, the GFP-TIP2;1 parental line and <em>rbb1-1</em>. Three pairs of gene specific primers were used to detect the transcript at the 5' end, middle, and 3' end of the <em>RBB1</em> gene as indicated in (a). Numbers represent amplicon size. (c-e) Lack of genetic complementation between <em>rbb1-1</em> and <em>rbb1-2</em>. Hypocotyls of four-day-old dark-grown seedlings from the parental control (WT, c), <em>rbb1-1</em> (d) and the F<sub>1</sub> progeny from a cross between <em>rbb1-1</em> and <em>rbb1-2</em> (<em>rbb1-1/rbb1-2</em> double heterozygotes, e) were imaged by confocal microscopy. Scale bar: 20μm. (f-g) <em>rbb1-2</em> contains bulbs in a similar manner as <em>rbb1-1</em>. Four-day-old seedlings were stained with 2 μM Lysotracker Red for 2 hours to label the vacuole in Col-0 (WT) and <em>rbb1-2</em>. Bulbs are indicated with arrows. Scale: 20 μm. (h) <em>RBB1</em> is expressed in many developmental stages. The accumulation of <em>RBB1</em> transcripts was determined by RT-PCR with gene specific primers in various tissues of 4 and 7-day old Col-0 (WT) plants. Two pairs of gene specific primers detected the 5' end and 3' end of <em>RBB1</em> gene as indicated in (a).</p></figcaption></figure><p>To determine if <em>RBB1</em> is <em>At5g40450</em>, a complementation test between <em>rbb1-1</em> and <em>rbb1-2</em> was carried out. F<sub>1</sub> seedlings from a cross between <em>rbb1-1</em> and homozygous <em>rbb1-2</em> were incubated in the dark and analyzed by fluorescence microscopy for the bulb phenotype. As shown in <a class="usa-link" href="#pone.0125621.g005">Fig 5E</a>, the double heterozygote F<sub>1</sub> plants display the <em>rbb1</em> phenotype and indicate the lack of complementation between the two mutant alleles. We were unable to confirm this result in the F<sub>2</sub> population as the GFP-TIP2;1 transgene was consistently silenced in the segregating population whenever the SALK insertion was present. This result is consistent with highly silenced transgenes in SALK insertion lines [<a aria-describedby="pone.0125621.ref039" class="usa-link" href="#pone.0125621.ref039">39</a>]. To confirm the identity of <em>RBB1</em>, we visualized the vacuole morphology of <em>rbb1-2</em> mutants with Lysotracker Red, a fluorescent dye that stains acidic organelles including plant vacuoles [<a aria-describedby="pone.0125621.ref040" class="usa-link" href="#pone.0125621.ref040">40</a>] and the tonoplast membrane [<a aria-describedby="pone.0125621.ref041" class="usa-link" href="#pone.0125621.ref041">41</a>, <a aria-describedby="pone.0125621.ref042" class="usa-link" href="#pone.0125621.ref042">42</a>] (<a class="usa-link" href="#pone.0125621.s004">S4 Fig</a>). This experiment could only be carried out in cotyledons from light-grown seedlings as penetration of fluorescent dyes in hypocotyls is very poor. Similar to the reduced number of bulbs in the GFP-TIP2;1 line, bulbs were rarely detected in wild type Col-0 cotyledons (<a class="usa-link" href="#pone.0125621.g005">Fig 5F</a> and <a class="usa-link" href="#pone.0125621.s007">S1 Movie</a>). On the contrary, the <em>rbb1-2</em> mutant allele showed a similar bulb phenotype as the <em>rbb1-1</em> mutant with 4–5 vacuolar bulbs per cell in these cells (<a class="usa-link" href="#pone.0125621.g005">Fig 5G</a> and <a class="usa-link" href="#pone.0125621.s008">S2 Movie</a>). This phenotype also segregated as ~1/4 and therefore <em>rbb1-2</em> is likely to be recessive. These results confirmed that the <em>RBB1</em> locus corresponds to <em>At5g40450</em> and that the bulb phenotype may be detected in the absence of the GFP-TIP2;1 transgene.</p>
<p>
<em>At5g40450</em> encodes a large protein of 2,890 amino acids. The mutation in <em>rbb1-1</em> is a C/T nucleotide substitution that results in an A1070V substitution in the predicted protein sequence. BLASTp analysis in NCBI identified only two proteins with protein similarity covering at least 40% of the RBB1 protein sequence, CARUB_v10003962mg from <em>Capsella rubella</em> and EUTSA_v10027617mg from <em>Eutrema salsugineum</em>. All other hits (~40) show high similarity to a 75 amino acid region at the C terminus of RBB1. Remarkably, no significant similarity was found outside of plant taxa using BLASTp, which indicates that RBB1 is a plant-specific protein. In addition, no specific domain hits were detected using the Conserved Domain Database search function in NCBI [<a aria-describedby="pone.0125621.ref043" class="usa-link" href="#pone.0125621.ref043">43</a>], but regions of similarity to three multi-domain structures were detected. Only two proteins were identified with significant similarity to RBB1 in <em>C</em>. <em>rubella</em> and <em>E</em>. <em>salsugineum</em>. RBB1 belongs to the putative Plant Model Organism Orthologous Group APK_ORTHOMCL5144, which includes genes from Rice, Poplar, Sorghum, Maize, and Brachypodium (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://rice.plantbiology.msu.edu/cgi-bin/ortholog_group_apk.pl" rel="noopener noreferrer" target="_blank">http://rice.plantbiology.msu.edu/cgi-bin/ortholog_group_apk.pl</a>). This group is also supported by gene family clustering of orthologous genes in Phytozome (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.phytozome.net" rel="noopener noreferrer" target="_blank">www.phytozome.net</a>) [<a aria-describedby="pone.0125621.ref044" class="usa-link" href="#pone.0125621.ref044">44</a>]. The putative orthologs also encode very large proteins (up to 3,715 aa). One of the putative orthologs, <em>Ricinus communis</em> gene 29917.t000066, contains a 32 amino acid region that is 32% identical to the microtubule-associated protein futsch in <em>Drosophila melanogaster</em>. These results overall indicate that RBB1 is a novel plant-specific protein with unknown molecular function.</p>
<p>In order to determine the spatial and temporal expression of <em>RBB1</em>, the accumulation of <em>RBB1</em> transcripts was analyzed by RT-PCR in Col-0 wild type plants. <em>RBB1</em> accumulated at similar levels in 4- and 7-day-old light-grown seedlings, and in rosette leaves and inflorescences (<a class="usa-link" href="#pone.0125621.g005">Fig 5H</a>). These results are consistent with publicly available databases for <em>At5g40450</em>. According to Arabidopsis eFP browser [<a aria-describedby="pone.0125621.ref045" class="usa-link" href="#pone.0125621.ref045">45</a>], <em>RBB1</em> is expressed in leaves and roots of both seedlings and mature plants, flowers and all stages of embryo development. Seedlings treated for 12 h with ABA, heat, cold and osmotic stress have decreased levels of <em>RBB1</em> transcripts [<a aria-describedby="pone.0125621.ref045" class="usa-link" href="#pone.0125621.ref045">45</a>, <a aria-describedby="pone.0125621.ref046" class="usa-link" href="#pone.0125621.ref046">46</a>]. In contrast, 3 h treatment with photosystem II inhibitor N-octyl-3-nitro-2,4,6-trihydroxybenzamide (PNO8) or the brassinosteroid biosynthesis inhibitor brassinazole 220 resulted in a 2-fold induction of <em>RBB1</em> [<a aria-describedby="pone.0125621.ref047" class="usa-link" href="#pone.0125621.ref047">47</a>]. These results indicate that <em>RBB1</em> may have important functions in multiple developmental stages. The <em>RBB1</em> transcript accumulation was also analyzed in the parental line and <em>rbb1-1</em>, but no differences in the expression level were detected (<a class="usa-link" href="#pone.0125621.g005">Fig 5B</a>). Therefore, the <em>rbb1-1</em> mutation did not affect the expression of <em>RBB1</em> at the transcriptional level.</p></section><section id="sec014"><h3 class="pmc_sec_title">RBB1 is a cytosolic protein</h3>
<p>In order to characterize the function of <em>RBB1</em>, a fluorescent protein (FP) fusion was generated to visualize its protein localization. Given the large size of the <em>RBB1</em> coding sequence, we used Recombineering [<a aria-describedby="pone.0125621.ref032" class="usa-link" href="#pone.0125621.ref032">32</a>] to generate a FP fusion with RBB1 driven by its own promoter and other regulatory sequences. This system utilizes homologous recombination with TAC clones to fuse the gene encoding 3xYpet (YFP for energy transfer), a bright variation of YFP [<a aria-describedby="pone.0125621.ref048" class="usa-link" href="#pone.0125621.ref048">48</a>], to the gene of interest within the native genomic context. Both N- and C-terminal fusions of RBB1 with 3xYpet were generated and introduced into Col-0 wild type. We could not detect fluorescence in RBB1-3xYpet lines after selection of 15 independent transformants. However, 3xYpet-RBB1 was detected in cotyledon epidermal cells, root cortical cells and, although very faint, in the cortex of hypocotyls of light-grown seedlings (<a class="usa-link" href="#pone.0125621.s005">S5A and S5B Fig</a>). No Ypet signal was detected at the root tip corresponding to the cell division zone. In order to determine if the 3xYpet-RBB1 fusion is functional, we introgressed the transgene into the <em>rbb1-2</em> allele by crossing. This experiment was carried out in <em>rbb1-2</em> because the GFP-TIP2;1 marker in <em>rbb1-1</em> is tightly linked to the <em>RBB1</em> locus and the probability of identifying <em>rbb1-1</em> segregants without GFP-TIP2;1 was very low. In contrast to <em>rbb1-2</em> seedlings stained with Lysotracker red, which showed many bulbs in cotyledons (<a class="usa-link" href="#pone.0125621.g006">Fig 6A</a>), <em>rbb1-2</em> seedlings expressing the 3xYpet-RBB1 protein have vacuole morphology very similar to the wild type (<a class="usa-link" href="#pone.0125621.g006">Fig 6B and 6C</a>). Therefore, the 3xYpet-RBB1 construct complements the bulb phenotype of <em>rbb1-2</em> indicating that this fusion protein is functional.</p>
<figure class="fig xbox font-sm" id="pone.0125621.g006"><h4 class="obj_head">Fig 6. 3xYpet-RBB1 is a cytoplasmic protein that associates with the tonoplast.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig 6" class="graphic" height="1349" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1845/4411111/9825d8aedda5/pone.0125621.g006.jpg" width="652"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/pone.0125621.g006/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(a-c) A 3xYpet-RBB1 fusion complements the bulb phenotype of <em>rbb1-2</em>. Four-day-old seedlings from <em>rbb1-2</em> (a) or <em>rbb1-2</em> 3xYpet-RBB1 (b, c) were stained with Lysotracker Red (magenta, a, b) to label the vacuole. The signal from 3xYpet-RBB1 (c) for the complemented plant is shown. (d-f) 3xYpet-RBB1 does not co-localize with FM4-64. Images show the cotyledon of 7-d-old seedlings expressing 3xYpet-RBB1 (d) stained with 5 μM FM4-64 (e) and the merged image (f). (g-i) 3xYpet-RBB1 and RFP-SYP22 do not co-localize. Cotyledons of 7-d-old seedlings expressing 3xYpet-RBB1 (g) and RFP-SYP22 (h) were imaged. Merged image (i) is shown. (j-l) 3xYpet-RBB1 and Lysotracker Red do not co-localize in rosette leaves. Seedlings from the 3xYpet-RBB1 line were stained with Lysotracker Red. Signal from 3xYpet-RBB1 (j), Lysotracker Red (k) and the merged image (l) are shown. Arrowheads indicate the localization of 3xYpet-RBB1 at the tip of elongating trans-vacuolar strands. (m-o) GFP molecules can also label the tips of transvacuolar strands. Seedlings from a 35::GFP marker line were stained with Lysotracker Red. The GFP signal (m), Lysotracker Red (n) and the merged image (o) are shown. Arrowheads indicate the tip of TVS. All scale bars = 20 μm. (p) 3xYpet-RBB1 accumulates in the soluble (S100) and membrane pellet (P100) fractions from whole seedlings. Immunoblot of 3xYpet-RBB1 soluble (S100) and membrane (P100) fractions from Col-0 and 3xYpet-RBB1 transgenic plants using antibodies against GFP (α-GFP), cFBPase (α-cFBPase, control for soluble fraction), and Plasma Membrane H+ATPase (α-H+ATPase, control for membrane fraction).</p></figcaption></figure><p>The localization of 3xYpet-RBB1 was diffuse and suggestive of cytosolic localization in cotyledons of 7-day-old seedlings (<a class="usa-link" href="#pone.0125621.g006">Fig 6D</a>). In order to determine its sub-cellular distribution in detail, we tested for co-localization between 3xYpet-RBB1 and the plasma membrane dye FM4-64, a marker for the tonoplast, RFP-SYP22 [<a aria-describedby="pone.0125621.ref030" class="usa-link" href="#pone.0125621.ref030">30</a>], and the vacuole dye Lysotracker Red. The 3xYpet-RBB1 marker does not co-localize with FM4-64 (<a class="usa-link" href="#pone.0125621.g006">Fig 6D–6F</a>) or with RFP-SYP22 (<a class="usa-link" href="#pone.0125621.g006">Fig 6G–6I</a>). The marker was found in the cytosol and did not co-localize with Lysotracker Red (<a class="usa-link" href="#pone.0125621.g006">Fig 6J–6L</a>), even though occasionally there was good co-localization between these two signals (<a class="usa-link" href="#pone.0125621.s005">S5C–S5E Fig</a>). These results indicate 3xYpet-RBB1 protein is most likely a soluble protein that accumulates in the cytosol. We also detected 3xYpet-RBB1 in a punctate pattern inside the vacuole that seemed to correspond to the tip of TVS labeled with Lysotracker Red in confocal images (<a class="usa-link" href="#pone.0125621.g006">Fig 6J–6L</a>). Interestingly, 3xYpet-RBB1 remained associated with the tip during the elongation of those TVS (<a class="usa-link" href="#pone.0125621.s005">S5F Fig</a>) in a similar manner as actin bundles [<a aria-describedby="pone.0125621.ref049" class="usa-link" href="#pone.0125621.ref049">49</a>]. These TVS structures may represent cross-sections of membrane sheets that transverse the vacuole, as previously proposed [<a aria-describedby="pone.0125621.ref010" class="usa-link" href="#pone.0125621.ref010">10</a>, <a aria-describedby="pone.0125621.ref049" class="usa-link" href="#pone.0125621.ref049">49</a>]. In order to determine the significance of this localization, we compared it to the localization of a soluble GFP marker in the same manner. As shown in <a class="usa-link" href="#pone.0125621.g006">Fig 6M–6O</a>, a simple GFP marker can also label the tip of TVS, and therefore, the punctate localization of RBB1 may simply result from accumulation of cytoplasmic proteins at the tip of emerging TVS. Immunoblot analysis was used to confirm that the 3xYpet-RBB1 fusion protein accumulates as the expected size of ~400 kDa in seedlings. Membrane and soluble fractions were isolated from wild type Col-0 and the 3xYpet-RBB1 line and the fusion protein was detected with an anti-GFP antibody. The cytosolic fructose-1,6-bisphosphatase (cFBPase) and plasma membrane H<sup>+</sup>ATPase were used as soluble and membrane protein controls, respectively. 3xYpet-RBB1 was detected in both the soluble (S100) and the membrane (P100) fractions in 4 days-old seedlings (<a class="usa-link" href="#pone.0125621.g006">Fig 6L</a> and <a class="usa-link" href="#pone.0125621.s006">S6 Fig</a>). These results overall indicate that 3xYpet-RBB1 is a large soluble protein that is likely to associate with cellular membranes.</p></section></section><section id="sec015"><h2 class="pmc_sec_title">Discussion</h2>
<p>We have identified <em>rbb1-1</em>, a novel mutant with defects in vacuole morphology. Vacuoles in <em>rbb1-1</em> contain increased numbers of bulbs in the large central vacuole of cotyledons and hypocotyl cells when compared to the parental control. These defined vacuolar sub-structures can be visualized with fluorescent protein markers for the tonoplast [<a aria-describedby="pone.0125621.ref015" class="usa-link" href="#pone.0125621.ref015">15</a>, <a aria-describedby="pone.0125621.ref023" class="usa-link" href="#pone.0125621.ref023">23</a>]. It was recently proposed that bulbs, specially when they have more than double the intensity of fluorescence when compared to the tonoplast, are artifacts that result from GFP dimerization that stabilizes the adhesion of tonoplast membranes [<a aria-describedby="pone.0125621.ref024" class="usa-link" href="#pone.0125621.ref024">24</a>]. This model, however, does not explain the accumulation of bulbs in the 2xFYVE-YFP line in which the fusion protein associates with the membrane in a reversible manner by binding to a specific phosphoinositide [<a aria-describedby="pone.0125621.ref015" class="usa-link" href="#pone.0125621.ref015">15</a>, <a aria-describedby="pone.0125621.ref023" class="usa-link" href="#pone.0125621.ref023">23</a>, <a aria-describedby="pone.0125621.ref050" class="usa-link" href="#pone.0125621.ref050">50</a>]. The GFP-TIP2;1-labeled bulbs in <em>rbb1-1</em> showed between 2 and 3.5-fold higher fluorescence intensity when compared to the tonoplast, indicating that they are likely to have a similar structure as previously characterized IVPS and bulbs [<a aria-describedby="pone.0125621.ref015" class="usa-link" href="#pone.0125621.ref015">15</a>, <a aria-describedby="pone.0125621.ref023" class="usa-link" href="#pone.0125621.ref023">23</a>, <a aria-describedby="pone.0125621.ref024" class="usa-link" href="#pone.0125621.ref024">24</a>]. We can not discard the possibility that the GFP fusion enhances the stability of the bulbs in <em>rbb1-1</em>, but <em>rbb1</em> mutant alleles comprise a sensitized genetic background for studying the formation and regulation of these structures, which are quite elusive under normal conditions. The two other mutants that have been identified as having abnormal bulb accumulation, <em>sgr2</em> and <em>vti11</em>, have fewer bulbs, also have very abnormal vacuole morphology [<a aria-describedby="pone.0125621.ref023" class="usa-link" href="#pone.0125621.ref023">23</a>, <a aria-describedby="pone.0125621.ref025" class="usa-link" href="#pone.0125621.ref025">25</a>, <a aria-describedby="pone.0125621.ref028" class="usa-link" href="#pone.0125621.ref028">28</a>]. Hence, <em>rbb1-1</em> is a unique tool to characterize the structure and function of the bulbs in the absence of other vacuolar defects.</p>
<p>
<em>RBB1</em> encodes a very large protein with unknown molecular function. In order to determine the function of <em>RBB1</em>, we generated a 3xYpet-RBB1 fusion protein under the control of the <em>RBB1</em> promoter. This construct complemented the <em>rbb1-2</em> mutant phenotype indicating that the 3xYpet-RBB1 fusion is functional. In seedlings, 3xYpet-RBB1 was detected in the root in the elongation and differentiation zones, but not in the cell division zone, and in hypocotyl and cotyledon epidermal cells. Therefore, 3xYpet-RBB1 accumulates in cells that contain a large lytic vacuole [<a aria-describedby="pone.0125621.ref051" class="usa-link" href="#pone.0125621.ref051">51</a>], consistent with a role of RBB1 in regulating the morphology of this important organelle. Confocal scanning laser microscopy with the 3xYpet-RBB1 line indicated that this protein accumulates in the cytosol, and that in a few instances also co-localizes with Lysotracker red, a fluorescent dye that labels the tonoplast. RBB1 was found both in the soluble and membrane fractions by fractionation experiments, which raises the possibility that RBB1 may associate with membranes. 3xYpet-RBB1 was often present at the tips of elongating TVS in cotyledons and leaves, but visualization of a GFP control indicated that this localization may be detected some times with other cytosolic proteins.</p>
<p>We detected a significant decrease in the number of TVS in <em>rbb1-1</em> mutants both in light- and dark-grown hypocotyls and in cotyledons of light-grown seedlings. The TVS phenotype of <em>rbb1-1</em> supports the hypothesis that RBB1 functions in the formation or stability of TVS. This hypothesis is supported by the observation that both the bulb phenotype and the reduced TVS phenotype can be detected at day 3 during germination. TVS are very dynamic structures that require the actomyosin cytoskeleton [<a aria-describedby="pone.0125621.ref004" class="usa-link" href="#pone.0125621.ref004">4</a>, <a aria-describedby="pone.0125621.ref005" class="usa-link" href="#pone.0125621.ref005">5</a>, <a aria-describedby="pone.0125621.ref014" class="usa-link" href="#pone.0125621.ref014">14</a>, <a aria-describedby="pone.0125621.ref049" class="usa-link" href="#pone.0125621.ref049">49</a>], and are thought redistribute cytoplasmic contents or to enhance the structural stability of the cytoskeleton [<a aria-describedby="pone.0125621.ref014" class="usa-link" href="#pone.0125621.ref014">14</a>]. A potential role of RBB1 in TVS formation is consistent with the expression of 3xYpet-RBB1 in cells that contain a large vacuole. Given the localization of 3xYpet-RBB1 to the cytosol and its accumulation in both the soluble and membrane fractions, it is tempting to speculate that RBB1 interacts with the tonoplast during the formation and elongation of TVS and contributes to the remodeling of the vacuolar membrane. However, more experiments beyond the scope of this work are needed to test this possibility. Moreover, we speculate that the increased number of bulbs in <em>rbb1-1</em> results from its inability to form functional TVS. Therefore, we propose an alternative model for bulb biogenesis in which a bulb forms when a TVS initiates but fails to elongate and the bulb functions as a mean to accommodate tonoplast membranes and cytoskeletal proteins that may have accumulated at the initiation site, until the bulb membrane is reabsorbed back into the tonoplast. It follows that the bulbs may have no specific function in the vacuole other than the transient accumulation of membranes and proteins when TVS failed to elongate. Further characterization of the function of RBB1 and interacting partners may shed light in the mechanisms that remodel the vacuole membrane into TVS or bulbs. Unlike the effects of GFP dimerization in vacuole morphology after fusion with vacuolar H+-pyrophosphatase [<a aria-describedby="pone.0125621.ref024" class="usa-link" href="#pone.0125621.ref024">24</a>], it is unlikely that GFP dimerization plays a big role in the TVS phenotype of <em>rbb1</em>. While the bulb fluorescence intensity for cotyledons was 3.5 fold higher than that of tonoplast, this ratio is only 2.0 for hypocotyls in <em>rbb1-1</em>, the same tissue that was used to quantify TVS in <a class="usa-link" href="#pone.0125621.g002">Fig 2</a>. Therefore, a ratio higher than 2.0 is not necessary in <em>rbb1-1</em> to induce the accumulation of bulbs or a reduction in TVS.</p>
<p>Previously proposed functions of vacuolar bulbs include acting as a tonoplast reservoir for rapid growth or serving as the sites of degradation of tonoplast membrane and proteins (Saito et al. 2002 and Maîtrejean et al. 2011). We tested one of these hypotheses by measuring the hypocotyl growth of <em>rbb1-1</em> in the dark, a condition that promotes cell elongation. However, no obvious growth differences were observed between <em>rbb1-1</em> and the parental line. One possibility is that the additional tonoplast membranes contained in <em>rbb1-1</em> bulbs do not contribute to cell elongation. Alternatively, other factors such as the elasticity of the cell wall [<a aria-describedby="pone.0125621.ref049" class="usa-link" href="#pone.0125621.ref049">49</a>] may be limiting and would prevent us from detecting a difference in this assay.</p>
<p>The <em>RBB1</em> gene is expressed in seedlings and adult plants both in shoots and roots. That <em>RBB1</em> is expressed as a full-length transcript of 8,957 bp is supported by several ESTs as well as the alignment of RNAseq reads spanning the entire coding sequence by the Integrated Genome Browser [<a aria-describedby="pone.0125621.ref052" class="usa-link" href="#pone.0125621.ref052">52</a>, <a aria-describedby="pone.0125621.ref053" class="usa-link" href="#pone.0125621.ref053">53</a>]. The molecular function of RBB1 is unknown as the protein has no conserved domains, and it represents a new plant-specific protein. The identification of proteins that interact with RBB1 may shed light on the molecular mechanisms by which RBB1 could regulate the morphology of the vacuole.</p></section><section id="sec016"><h2 class="pmc_sec_title">Supporting Information</h2>
<section class="sm xbox font-sm" id="pone.0125621.s001"><div class="caption p">
<span>S1 Fig. 
<em>rbb1-1</em> mutants show normal growth.</span><p>(a-b) Normal growth phenotype of 4 day-old seedlings (a), and 6 week-old plants (b) from the parental line, <em>rbb1-1</em>, Col-0, and <em>rbb1-2</em>. (c) Relative GFP content in the parental line and <em>rbb1-1</em> by immunoblotting. The intensity values for GFP from <a class="usa-link" href="#pone.0125621.g001">Fig 1O</a> were normalized against the Calreticulin (CRT) loading control.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s001.tif">Click here for additional data file.</a><sup> (32.4MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0125621.s002"><div class="caption p">
<span>S2 Fig. Maximum fluorescence intensity for bulbs and tonoplast.</span><p>(a, b) Examples of fluorescence intensity profiles that were used to estimate intensity values in the parental line (a) and <em>rbb1-1</em> (b). Intensity value profiles from a line selection (white lines) across each bulb or the tonoplast were extracted in Image J from confocal images. Only the maximum value was collected for each line selection (arrowheads). (c, d) Maximum fluorescence intensity of bulbs and the tonoplast from cotyledon (c) and hypocotyl cells (d) in the parental line or <em>rbb1-1</em> grown in the light. Images were analyzed as in (a, b). Data shown is the average of maximum intensity values for 10–15 images from at least 3 seedlings each. The numbers inside the white bars correspond to the calculated ratio between the maximum fluorescence intensity in the bulbs and that of the tonoplast. Identical microscope settings were used when collecting data for cotyledons or when collecting data for hypocotyls.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s002.tif">Click here for additional data file.</a><sup> (3MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0125621.s003"><div class="caption p">
<span>S3 Fig. Quantification of TVS and growth phenotypes of <em>rbb1-1</em>.</span><p>(a-b) An example of images used for TVS quantification. Thick optical sections were captured from hypocotyls (a). A seedling from the parental line is shown. The boxed area is enlarged in (b) to show the TVS that were counted (arrowhead). Bar = 20 μm. (c-d) The parental line and <em>rbb1-1</em> have similar hypocotyl growth in the dark. Seedlings were grown in the dark for up to 5 days and seedlings were imaged in a scanner. Hypocotyl length was measured using Image J (NIH) from seedlings at days 3 and 5 (n = 10). Black bars represent parental line; gray bars represent <em>rbb1-1</em> mutant.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s003.tif">Click here for additional data file.</a><sup> (672.7KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0125621.s004"><div class="caption p">
<span>S4 Fig. Lysotracker Red labels the tonoplast in addition to the vacuole lumen in Arabidopsis.</span><p>GFP-TIP2;1 seedlings were stained with Lysotracker Red for 2 h and imaged by confocal microscopy. The tonoplast marker GFP-TIP2;1 (green, a) co-localizes with the membrane signal of Lysotracker Red (magenta, b). White signal in the merged image (c) represents co-localized pixels. Scale bar = 20 μm.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s004.tif">Click here for additional data file.</a><sup> (16.3MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0125621.s005"><div class="caption p">
<span>S5 Fig. Localization of 3xYpet-RBB1.</span><p>(a-b) 3xYpet-RBB1 can be detected in cotyledons, hypocotyls and roots. Four-day-old seedlings expressing 3xYpet-RBB1 under the control of its native promoter were imaged by confocal microscopy. An overlay of the fluorescence signal and the bright field image is shown in (a) and the fluorescence signal is shown in (b). The arrow in (b) indicates the position of the root tip. Scale bar = 200μm. (c-e) An example where the 3xYpet-RBB1 co-localizes with Lysotracker Red. 3xYpet-RBB1 is shown in green (c), Lysotracker Red is shown in magenta (d) and the merged image is shown (e). Scale bar = 10 μm. (f) The 3xYpet-RBB1 signal remains associated with the TVS tip. Cotyledons that were stained with Lysotracker were captured by time lapse microscopy every 7.75 sec. Note that the 3xYpet-RBB1 signal (green) remains associated with the tip of the elongating TVS labeled with Lysotracker Red (magenta). Scale bar = 10 μm.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s005.tif">Click here for additional data file.</a><sup> (16.1MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0125621.s006"><div class="caption p">
<span>S6 Fig. 3xYpet-RBB1 Immunoblot.</span><p>This blot corresponds to the image shown in <a class="usa-link" href="#pone.0125621.g006">Fig 6O</a> but includes the entire blot to show the size of the 3xYpet-RBB1 fusion. * Non-specific bands.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s006.tif">Click here for additional data file.</a><sup> (8.5MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0125621.s007"><div class="caption p">
<span>S1 Movie. Z-stack of Col-WT cotyledon stained with Lysotracker Red.</span><p>Four-day-old Col-WT seedlings stained with Lysotracker Red as in <a class="usa-link" href="#pone.0125621.g005">Fig 5F</a>. The stack represents 20 optical sections every 1 μm. Image dimensions are x:106.17 μm, y:106.17 μm.</p>
<p>(AVI)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s007.avi">Click here for additional data file.</a><sup> (6.5MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0125621.s008"><div class="caption p">
<span>S2 Movie. Z-stack of <em>rbb1-2</em> cotyledon stained with Lysotracker Red.</span><p>Four-day-old <em>rbb1-2</em> seedlings stained with Lysotracker Red as in <a class="usa-link" href="#pone.0125621.g005">Fig 5G</a>. The stack represents 45 optical sections every 1 μm. Image dimensions are x:106.17 μm, y:106.17 μm.</p>
<p>(AVI)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s008.avi">Click here for additional data file.</a><sup> (4.1MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0125621.s009"><div class="caption p">
<span>S1 Table. List of primers used for Recombineering.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s009.docx">Click here for additional data file.</a><sup> (16.9KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0125621.s010"><div class="caption p">
<span>S2 Table. List of primers used for RT-PCR.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s010.docx">Click here for additional data file.</a><sup> (14.2KB, docx) </sup>
</div></div></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank W. Boss, T. Long and G. Drakakaki for critical reading of the manuscript, and B. Boston and J. Gillikin for sharing of materials. We thank the Rojas-Pierce laboratory and the plant genetics group for useful discussions.</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>All relevant data are within the paper and its Supporting Information files.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This work was supported by National Science Foundation grant MCB-1244354 (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.nsf.gov" rel="noopener noreferrer" target="_blank">http://www.nsf.gov</a>) to M.R.P. and North Carolina State University. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0125621.ref001">
<span class="label">1.</span><cite>Bassham DC, Brandizzi F, Otegui MS, Sanderfoot AA. The secretory system of Arabidopsis. The Arabidopsis Book 2008. p. 1–29.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1199/tab.0116" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3243370/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22303241/">PubMed</a>]</li>
<li id="pone.0125621.ref002">
<span class="label">2.</span><cite>
Muntz K. Protein dynamics and proteolysis in plant vacuoles. J Exp Bot. 2007;58(10):2391–407. 10.1093/jxb/erm089
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/erm089" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17545219/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Bot&amp;title=Protein%20dynamics%20and%20proteolysis%20in%20plant%20vacuoles&amp;author=K%20Muntz&amp;volume=58&amp;issue=10&amp;publication_year=2007&amp;pages=2391-407&amp;pmid=17545219&amp;doi=10.1093/jxb/erm089&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref003">
<span class="label">3.</span><cite>
Xiang L, Etxeberria E, Van den Ende W. Vacuolar protein sorting mechanisms in plants. FEBS J. 2013;280(4):979–93. 10.1111/febs.12092
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/febs.12092" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23241209/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20J&amp;title=Vacuolar%20protein%20sorting%20mechanisms%20in%20plants&amp;author=L%20Xiang&amp;author=E%20Etxeberria&amp;author=W%20Van%20den%20Ende&amp;volume=280&amp;issue=4&amp;publication_year=2013&amp;pages=979-93&amp;pmid=23241209&amp;doi=10.1111/febs.12092&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref004">
<span class="label">4.</span><cite>
Ruthardt N, Gulde N, Spiegel H, Fischer R, Emans N. Four-dimensional imaging of transvacuolar strand dynamics in tobacco BY-2 cells. Protoplasma. 2005;225(3–4):205–15. 10.1007/s00709-005-0093-7
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00709-005-0093-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16228899/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Protoplasma&amp;title=Four-dimensional%20imaging%20of%20transvacuolar%20strand%20dynamics%20in%20tobacco%20BY-2%20cells&amp;author=N%20Ruthardt&amp;author=N%20Gulde&amp;author=H%20Spiegel&amp;author=R%20Fischer&amp;author=N%20Emans&amp;volume=225&amp;issue=3%E2%80%934&amp;publication_year=2005&amp;pages=205-15&amp;pmid=16228899&amp;doi=10.1007/s00709-005-0093-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref005">
<span class="label">5.</span><cite>
Uemura T, Yoshimura SH, Takeyasu K, Sato MH. Vacuolar membrane dynamics revealed by GFP-AtVam3 fusion protein. Genes to Cells. 2002;7(7):743–53.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1365-2443.2002.00550.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12081650/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genes%20to%20Cells&amp;title=Vacuolar%20membrane%20dynamics%20revealed%20by%20GFP-AtVam3%20fusion%20protein&amp;author=T%20Uemura&amp;author=SH%20Yoshimura&amp;author=K%20Takeyasu&amp;author=MH%20Sato&amp;volume=7&amp;issue=7&amp;publication_year=2002&amp;pages=743-53&amp;pmid=12081650&amp;doi=10.1046/j.1365-2443.2002.00550.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref006">
<span class="label">6.</span><cite>
Nebenfuhr A, Gallagher LA, Dunahay TG, Frohlick JA, Mazurkiewicz AM, Meehl JB, et al. 
Stop-and-go movements of plant Golgi stacks are mediated by the acto-myosin system. Plant Physiol. 1999;121(4):1127–42.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.121.4.1127" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC59480/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10594100/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Stop-and-go%20movements%20of%20plant%20Golgi%20stacks%20are%20mediated%20by%20the%20acto-myosin%20system&amp;author=A%20Nebenfuhr&amp;author=LA%20Gallagher&amp;author=TG%20Dunahay&amp;author=JA%20Frohlick&amp;author=AM%20Mazurkiewicz&amp;volume=121&amp;issue=4&amp;publication_year=1999&amp;pages=1127-42&amp;pmid=10594100&amp;doi=10.1104/pp.121.4.1127&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref007">
<span class="label">7.</span><cite>
Tominaga M, Yokota E, Vidali L, Sonobe S, Hepler PK, Shimmen T. The role of plant villin in the organization of the actin cytoskeleton, cytoplasmic streaming and the architecture of the transvacuolar strand in root hair cells of Hydrocharis. Planta. 2000;210(5):836–43. 10.1007/s004250050687
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s004250050687" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10805457/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=The%20role%20of%20plant%20villin%20in%20the%20organization%20of%20the%20actin%20cytoskeleton,%20cytoplasmic%20streaming%20and%20the%20architecture%20of%20the%20transvacuolar%20strand%20in%20root%20hair%20cells%20of%20Hydrocharis&amp;author=M%20Tominaga&amp;author=E%20Yokota&amp;author=L%20Vidali&amp;author=S%20Sonobe&amp;author=PK%20Hepler&amp;volume=210&amp;issue=5&amp;publication_year=2000&amp;pages=836-43&amp;pmid=10805457&amp;doi=10.1007/s004250050687&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref008">
<span class="label">8.</span><cite>
Tominaga M, Sonobe S, Shimmen T. Mechanism of Inhibition of Cytoplasmic Streaming by Auxin in Root Hair Cells of Hydrocharis. Plant Cell Physiol. 1998;39(12):1342–9.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol&amp;title=Mechanism%20of%20Inhibition%20of%20Cytoplasmic%20Streaming%20by%20Auxin%20in%20Root%20Hair%20Cells%20of%20Hydrocharis&amp;author=M%20Tominaga&amp;author=S%20Sonobe&amp;author=T%20Shimmen&amp;volume=39&amp;issue=12&amp;publication_year=1998&amp;pages=1342-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref009">
<span class="label">9.</span><cite>
Kutsuna N, Hasezawa S. Dynamic Organization of Vacuolar and Microtubule Structures during Cell Cycle Progression in Synchronized Tobacco BY-2 Cells. Plant Cell Physiol. 2002;43(9):965–73. 10.1093/pcp/pcf138
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pcf138" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12354913/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol&amp;title=Dynamic%20Organization%20of%20Vacuolar%20and%20Microtubule%20Structures%20during%20Cell%20Cycle%20Progression%20in%20Synchronized%20Tobacco%20BY-2%20Cells&amp;author=N%20Kutsuna&amp;author=S%20Hasezawa&amp;volume=43&amp;issue=9&amp;publication_year=2002&amp;pages=965-73&amp;pmid=12354913&amp;doi=10.1093/pcp/pcf138&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref010">
<span class="label">10.</span><cite>
Higaki T, Kutsuna N, Okubo E, Sano T, Hasezawa S. Actin Microfilaments Regulate Vacuolar Structures and Dynamics: Dual Observation of Actin Microfilaments and Vacuolar Membrane in Living Tobacco BY-2 Cells. Plant Cell Physiol. 2006;47(7):839–52. 10.1093/pcp/pcj056
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pcj056" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16672254/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol&amp;title=Actin%20Microfilaments%20Regulate%20Vacuolar%20Structures%20and%20Dynamics:%20Dual%20Observation%20of%20Actin%20Microfilaments%20and%20Vacuolar%20Membrane%20in%20Living%20Tobacco%20BY-2%20Cells&amp;author=T%20Higaki&amp;author=N%20Kutsuna&amp;author=E%20Okubo&amp;author=T%20Sano&amp;author=S%20Hasezawa&amp;volume=47&amp;issue=7&amp;publication_year=2006&amp;pages=839-52&amp;pmid=16672254&amp;doi=10.1093/pcp/pcj056&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref011">
<span class="label">11.</span><cite>
Staiger CJ, Yuan M, Valenta R, Shaw PJ, Warn RM, Lloyd CW. Microinjected profilin affects cytoplasmic streaming in plant cells by rapidly depolymerizing actin microfilaments. Curr Biol. 1994;4(3):215–9. doi: S0960-9822(00)00050-6
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s0960-9822(00)00050-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/7922326/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&amp;title=Microinjected%20profilin%20affects%20cytoplasmic%20streaming%20in%20plant%20cells%20by%20rapidly%20depolymerizing%20actin%20microfilaments&amp;author=CJ%20Staiger&amp;author=M%20Yuan&amp;author=R%20Valenta&amp;author=PJ%20Shaw&amp;author=RM%20Warn&amp;volume=4&amp;issue=3&amp;publication_year=1994&amp;pages=215-9&amp;pmid=7922326&amp;doi=10.1016/s0960-9822(00)00050-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref012">
<span class="label">12.</span><cite>
Yokota E, Vidali L, Tominaga M, Tahara H, Orii H, Morizane Y, et al. 
Plant 115-kDa Actin-Filament Bundling Protein, P-115-ABP, is a Homologue of Plant Villin and is Widely Distributed in Cells. Plant Cell Physiol. 2003;44(10):1088–99. 10.1093/pcp/pcg132
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pcg132" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14581634/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol&amp;title=Plant%20115-kDa%20Actin-Filament%20Bundling%20Protein,%20P-115-ABP,%20is%20a%20Homologue%20of%20Plant%20Villin%20and%20is%20Widely%20Distributed%20in%20Cells&amp;author=E%20Yokota&amp;author=L%20Vidali&amp;author=M%20Tominaga&amp;author=H%20Tahara&amp;author=H%20Orii&amp;volume=44&amp;issue=10&amp;publication_year=2003&amp;pages=1088-99&amp;pmid=14581634&amp;doi=10.1093/pcp/pcg132&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref013">
<span class="label">13.</span><cite>
van der Honing HS, de Ruijter NC, Emons AM, Ketelaar T. Actin and myosin regulate cytoplasm stiffness in plant cells: a study using optical tweezers. New Phytol. 2010;185(1):90–102. 10.1111/j.1469-8137.2009.03017.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1469-8137.2009.03017.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19761443/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol&amp;title=Actin%20and%20myosin%20regulate%20cytoplasm%20stiffness%20in%20plant%20cells:%20a%20study%20using%20optical%20tweezers&amp;author=HS%20van%20der%20Honing&amp;author=NC%20de%20Ruijter&amp;author=AM%20Emons&amp;author=T%20Ketelaar&amp;volume=185&amp;issue=1&amp;publication_year=2010&amp;pages=90-102&amp;pmid=19761443&amp;doi=10.1111/j.1469-8137.2009.03017.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref014">
<span class="label">14.</span><cite>
Hoffmann A, Nebenführ A. Dynamic rearrangements of transvacuolar strands in BY-2 cells imply a role of myosin in remodeling the plant actin cytoskeleton. Protoplasma. 2004;224(3):201–10. 10.1007/s00709-004-0068-0
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00709-004-0068-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15614481/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Protoplasma&amp;title=Dynamic%20rearrangements%20of%20transvacuolar%20strands%20in%20BY-2%20cells%20imply%20a%20role%20of%20myosin%20in%20remodeling%20the%20plant%20actin%20cytoskeleton&amp;author=A%20Hoffmann&amp;author=A%20Nebenf%C3%BChr&amp;volume=224&amp;issue=3&amp;publication_year=2004&amp;pages=201-10&amp;pmid=15614481&amp;doi=10.1007/s00709-004-0068-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref015">
<span class="label">15.</span><cite>
Saito C, Ueda T, Abe H, Wada Y, Kuroiwa T, Hisada A, et al. 
A complex and mobile structure forms a distinct subregion within the continuous vacuolar membrane in young cotyledons of Arabidopsis. Plant J. 2002;29(3):245–55.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.0960-7412.2001.01189.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11844103/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=A%20complex%20and%20mobile%20structure%20forms%20a%20distinct%20subregion%20within%20the%20continuous%20vacuolar%20membrane%20in%20young%20cotyledons%20of%20Arabidopsis&amp;author=C%20Saito&amp;author=T%20Ueda&amp;author=H%20Abe&amp;author=Y%20Wada&amp;author=T%20Kuroiwa&amp;volume=29&amp;issue=3&amp;publication_year=2002&amp;pages=245-55&amp;pmid=11844103&amp;doi=10.1046/j.0960-7412.2001.01189.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref016">
<span class="label">16.</span><cite>
Hunter PR, Craddock CP, Di Benedetto S, Roberts LM, Frigerio L. Fluorescent reporter proteins for the tonoplast and the vacuolar lumen identify a single vacuolar compartment in Arabidopsis cells. Plant Physiol. 2007;145(4):1371–82.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.107.103945" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2151705/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17905861/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Fluorescent%20reporter%20proteins%20for%20the%20tonoplast%20and%20the%20vacuolar%20lumen%20identify%20a%20single%20vacuolar%20compartment%20in%20Arabidopsis%20cells&amp;author=PR%20Hunter&amp;author=CP%20Craddock&amp;author=S%20Di%20Benedetto&amp;author=LM%20Roberts&amp;author=L%20Frigerio&amp;volume=145&amp;issue=4&amp;publication_year=2007&amp;pages=1371-82&amp;pmid=17905861&amp;doi=10.1104/pp.107.103945&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref017">
<span class="label">17.</span><cite>Bottanelli F, Foresti O, Hanton S, Denecke J. Vacuolar transport in tobacco leaf epidermis cells involves a single route for soluble cargo and multiple routes for membrane cargo. Plant Cell. 2011. 10.1105/tpc.111.085480
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.111.085480" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3180807/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21856792/">PubMed</a>]</li>
<li id="pone.0125621.ref018">
<span class="label">18.</span><cite>
Escobar NM, Haupt S, Thow G, Boevink P, Chapman S, Oparka K. High-throughput viral expression of cDNA-green fluorescent protein fusions reveals novel subcellular addresses and identifies unique proteins that interact with plasmodesmata. Plant Cell. 2003;15(7):1507–23.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.013284" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC165397/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12837943/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=High-throughput%20viral%20expression%20of%20cDNA-green%20fluorescent%20protein%20fusions%20reveals%20novel%20subcellular%20addresses%20and%20identifies%20unique%20proteins%20that%20interact%20with%20plasmodesmata&amp;author=NM%20Escobar&amp;author=S%20Haupt&amp;author=G%20Thow&amp;author=P%20Boevink&amp;author=S%20Chapman&amp;volume=15&amp;issue=7&amp;publication_year=2003&amp;pages=1507-23&amp;pmid=12837943&amp;doi=10.1105/tpc.013284&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref019">
<span class="label">19.</span><cite>
Maitrejean M, Vitale A. How are tonoplast proteins degraded?
Plant Signal Behav. 2011;6(11):1809–12. doi: 17867 [pii] 10.4161/psb.6.11.17867
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4161/psb.6.11.17867" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3329355/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22057339/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signal%20Behav&amp;title=How%20are%20tonoplast%20proteins%20degraded?&amp;author=M%20Maitrejean&amp;author=A%20Vitale&amp;volume=6&amp;issue=11&amp;publication_year=2011&amp;pages=1809-12&amp;pmid=22057339&amp;doi=10.4161/psb.6.11.17867&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref020">
<span class="label">20.</span><cite>
Maitrejean M, Wudick MM, Voelker C, Prinsi B, Mueller-Roeber B, Czempinski K, et al. 
Assembly and sorting of the tonoplast potassium channel AtTPK1 and its turnover by internalization into the vacuole. Plant Physiol. 2011;156(4):1783–96. 10.1104/pp.111.177816
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.111.177816" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3149923/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21697507/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Assembly%20and%20sorting%20of%20the%20tonoplast%20potassium%20channel%20AtTPK1%20and%20its%20turnover%20by%20internalization%20into%20the%20vacuole&amp;author=M%20Maitrejean&amp;author=MM%20Wudick&amp;author=C%20Voelker&amp;author=B%20Prinsi&amp;author=B%20Mueller-Roeber&amp;volume=156&amp;issue=4&amp;publication_year=2011&amp;pages=1783-96&amp;pmid=21697507&amp;doi=10.1104/pp.111.177816&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref021">
<span class="label">21.</span><cite>
Beebo A, Thomas D, Der C, Sanchez L, Leborgne-Castel N, Marty F, et al. 
Life with and without <em>AtTIP1;1</em>, an Arabidopsis aquaporin preferentially localized in the apposing tonoplasts of adjacent vacuoles. Plant Mol Biol. 2009;70(1–2):193–209. 10.1007/s11103-009-9465-2
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11103-009-9465-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19229639/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol%20Biol&amp;title=Life%20with%20and%20without%20AtTIP1;1,%20an%20Arabidopsis%20aquaporin%20preferentially%20localized%20in%20the%20apposing%20tonoplasts%20of%20adjacent%20vacuoles&amp;author=A%20Beebo&amp;author=D%20Thomas&amp;author=C%20Der&amp;author=L%20Sanchez&amp;author=N%20Leborgne-Castel&amp;volume=70&amp;issue=1%E2%80%932&amp;publication_year=2009&amp;pages=193-209&amp;pmid=19229639&amp;doi=10.1007/s11103-009-9465-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref022">
<span class="label">22.</span><cite>
Hicks GR, Rojo E, Hong S, Carter DG, Raikhel NV. Geminating pollen has tubular vacuoles, displays highly dynamic vacuole biogenesis, and requires VACUOLESS1 for proper function. Plant Physiol. 2004;134(3):1227–39.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.103.037382" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC389947/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14988481/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Geminating%20pollen%20has%20tubular%20vacuoles,%20displays%20highly%20dynamic%20vacuole%20biogenesis,%20and%20requires%20VACUOLESS1%20for%20proper%20function&amp;author=GR%20Hicks&amp;author=E%20Rojo&amp;author=S%20Hong&amp;author=DG%20Carter&amp;author=NV%20Raikhel&amp;volume=134&amp;issue=3&amp;publication_year=2004&amp;pages=1227-39&amp;pmid=14988481&amp;doi=10.1104/pp.103.037382&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref023">
<span class="label">23.</span><cite>
Saito C, Uemura T, Awai C, Tominaga M, Ebine K, Ito J, et al. 
The occurrence of 'bulbs', a complex configuration of the vacuolar membrane, is affected by mutations of vacuolar SNARE and phospholipase in Arabidopsis. Plant J. 2011;68(1):64–73. 10.1111/j.1365-313X.2011.04665.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2011.04665.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21645145/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=The%20occurrence%20of%20'bulbs',%20a%20complex%20configuration%20of%20the%20vacuolar%20membrane,%20is%20affected%20by%20mutations%20of%20vacuolar%20SNARE%20and%20phospholipase%20in%20Arabidopsis&amp;author=C%20Saito&amp;author=T%20Uemura&amp;author=C%20Awai&amp;author=M%20Tominaga&amp;author=K%20Ebine&amp;volume=68&amp;issue=1&amp;publication_year=2011&amp;pages=64-73&amp;pmid=21645145&amp;doi=10.1111/j.1365-313X.2011.04665.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref024">
<span class="label">24.</span><cite>
Segami S, Makino S, Miyake A, Asaoka M, Maeshima M. Dynamics of Vacuoles and H+-Pyrophosphatase Visualized by Monomeric Green Fluorescent Protein in Arabidopsis: Artifactual Bulbs and Native Intravacuolar Spherical Structures. Plant Cell. 2014;26(8):3416–34. 10.1105/tpc.114.127571
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.114.127571" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4371836/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25118245/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Dynamics%20of%20Vacuoles%20and%20H+-Pyrophosphatase%20Visualized%20by%20Monomeric%20Green%20Fluorescent%20Protein%20in%20Arabidopsis:%20Artifactual%20Bulbs%20and%20Native%20Intravacuolar%20Spherical%20Structures&amp;author=S%20Segami&amp;author=S%20Makino&amp;author=A%20Miyake&amp;author=M%20Asaoka&amp;author=M%20Maeshima&amp;volume=26&amp;issue=8&amp;publication_year=2014&amp;pages=3416-34&amp;pmid=25118245&amp;doi=10.1105/tpc.114.127571&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref025">
<span class="label">25.</span><cite>
Kato T, Morita MT, Fukaki H, Yamauchi Y, Uehara M, Niihama M, et al. 
SGR2, a phospholipase-like protein, and zig/sgr4, a SNARE, are involved in the shoot gravitropism of Arabidopsis. Plant Cell. 2002;14(1):33–46.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.010215" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC150549/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11826297/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=SGR2,%20a%20phospholipase-like%20protein,%20and%20zig/sgr4,%20a%20SNARE,%20are%20involved%20in%20the%20shoot%20gravitropism%20of%20Arabidopsis&amp;author=T%20Kato&amp;author=MT%20Morita&amp;author=H%20Fukaki&amp;author=Y%20Yamauchi&amp;author=M%20Uehara&amp;volume=14&amp;issue=1&amp;publication_year=2002&amp;pages=33-46&amp;pmid=11826297&amp;doi=10.1105/tpc.010215&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref026">
<span class="label">26.</span><cite>
Ebine K, Okatani Y, Uemura T, Goh T, Shoda K, Niihama M, et al. 
A SNARE complex unique to seed plants is required for protein storage vacuole biogenesis and seed development of Arabidopsis thaliana. Plant Cell. 2008;20(11):3006–21. 10.1105/tpc.107.057711
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.107.057711" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2613668/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18984676/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=A%20SNARE%20complex%20unique%20to%20seed%20plants%20is%20required%20for%20protein%20storage%20vacuole%20biogenesis%20and%20seed%20development%20of%20Arabidopsis%20thaliana&amp;author=K%20Ebine&amp;author=Y%20Okatani&amp;author=T%20Uemura&amp;author=T%20Goh&amp;author=K%20Shoda&amp;volume=20&amp;issue=11&amp;publication_year=2008&amp;pages=3006-21&amp;pmid=18984676&amp;doi=10.1105/tpc.107.057711&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref027">
<span class="label">27.</span><cite>
Sanderfoot AA, Kovaleva V, Bassham DC, Raikhel NV. Interactions between syntaxins identify at least five SNARE complexes within the Golgi/prevacuolar system of the Arabidopsis cell. Mol Biol Cell. 2001;12(12):3733–43.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.12.12.3733" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC60751/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11739776/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&amp;title=Interactions%20between%20syntaxins%20identify%20at%20least%20five%20SNARE%20complexes%20within%20the%20Golgi/prevacuolar%20system%20of%20the%20Arabidopsis%20cell&amp;author=AA%20Sanderfoot&amp;author=V%20Kovaleva&amp;author=DC%20Bassham&amp;author=NV%20Raikhel&amp;volume=12&amp;issue=12&amp;publication_year=2001&amp;pages=3733-43&amp;pmid=11739776&amp;doi=10.1091/mbc.12.12.3733&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref028">
<span class="label">28.</span><cite>
Zheng J, Han SW, Rodriguez-Welsh MF, Rojas-Pierce M. Homotypic Vacuole Fusion Requires VTI11 and Is Regulated by Phosphoinositides. Mol Plant. 2014;7(6):1026–40. 10.1093/mp/ssu019
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/mp/ssu019" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24569132/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol%20Plant&amp;title=Homotypic%20Vacuole%20Fusion%20Requires%20VTI11%20and%20Is%20Regulated%20by%20Phosphoinositides&amp;author=J%20Zheng&amp;author=SW%20Han&amp;author=MF%20Rodriguez-Welsh&amp;author=M%20Rojas-Pierce&amp;volume=7&amp;issue=6&amp;publication_year=2014&amp;pages=1026-40&amp;pmid=24569132&amp;doi=10.1093/mp/ssu019&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref029">
<span class="label">29.</span><cite>
Rivera-Serrano EE, Rodriguez-Welsh MF, Hicks GR, Rojas-Pierce M. A small molecule inhibitor partitions two distinct pathways for trafficking of tonoplast intrinsic proteins in Arabidopsis. PLoS ONE. 2012;7(9):e44735
10.1371/journal.pone.0044735PONE-D-12-19870 [pii].
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0044735PONE-D-12-19870" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3434187/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22957103/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=A%20small%20molecule%20inhibitor%20partitions%20two%20distinct%20pathways%20for%20trafficking%20of%20tonoplast%20intrinsic%20proteins%20in%20Arabidopsis&amp;author=EE%20Rivera-Serrano&amp;author=MF%20Rodriguez-Welsh&amp;author=GR%20Hicks&amp;author=M%20Rojas-Pierce&amp;volume=7&amp;issue=9&amp;publication_year=2012&amp;pages=e44735&amp;pmid=22957103&amp;doi=10.1371/journal.pone.0044735PONE-D-12-19870&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref030">
<span class="label">30.</span><cite>
Geldner N, Denervaud-Tendon V, Hyman DL, Mayer U, Stierhof YD, Chory J. Rapid, combinatorial analysis of membrane compartments in intact plants with a multicolor marker set. Plant J. 2009;59(1):169–78. doi: TPJ3851 [pii] 10.1111/j.1365-313X.2009.03851.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2009.03851.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4854200/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19309456/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Rapid,%20combinatorial%20analysis%20of%20membrane%20compartments%20in%20intact%20plants%20with%20a%20multicolor%20marker%20set&amp;author=N%20Geldner&amp;author=V%20Denervaud-Tendon&amp;author=DL%20Hyman&amp;author=U%20Mayer&amp;author=YD%20Stierhof&amp;volume=59&amp;issue=1&amp;publication_year=2009&amp;pages=169-78&amp;pmid=19309456&amp;doi=10.1111/j.1365-313X.2009.03851.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref031">
<span class="label">31.</span><cite>
Grefen C, Donald N, Schumacher K, Blatt MR. A Ubiquitin-10 promoter-based vector set for fluorescent protein tagging facilitates temporal stability and native protein distribution in transient and stable expression studies. Plant J. 2010;64(2):355–65. 10.1111/j.1365-313X.2010.04322.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2010.04322.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20735773/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=A%20Ubiquitin-10%20promoter-based%20vector%20set%20for%20fluorescent%20protein%20tagging%20facilitates%20temporal%20stability%20and%20native%20protein%20distribution%20in%20transient%20and%20stable%20expression%20studies&amp;author=C%20Grefen&amp;author=N%20Donald&amp;author=K%20Schumacher&amp;author=MR%20Blatt&amp;volume=64&amp;issue=2&amp;publication_year=2010&amp;pages=355-65&amp;pmid=20735773&amp;doi=10.1111/j.1365-313X.2010.04322.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref032">
<span class="label">32.</span><cite>
Zhou R, Benavente LM, Stepanova AN, Alonso JM. A recombineering-based gene tagging system for Arabidopsis. Plant J. 2011;66(4):712–23. 10.1111/j.1365-313X.2011.04524.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2011.04524.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21294796/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=A%20recombineering-based%20gene%20tagging%20system%20for%20Arabidopsis&amp;author=R%20Zhou&amp;author=LM%20Benavente&amp;author=AN%20Stepanova&amp;author=JM%20Alonso&amp;volume=66&amp;issue=4&amp;publication_year=2011&amp;pages=712-23&amp;pmid=21294796&amp;doi=10.1111/j.1365-313X.2011.04524.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref033">
<span class="label">33.</span><cite>
Clough SJ, Bent AF. Floral dip: a simplified method for <em>Agrobacterium</em>-mediated transformation of Arabidopsis thaliana. Plant J. 1998;16(6):735–46.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1365-313x.1998.00343.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10069079/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Floral%20dip:%20a%20simplified%20method%20for%20Agrobacterium-mediated%20transformation%20of%20Arabidopsis%20thaliana&amp;author=SJ%20Clough&amp;author=AF%20Bent&amp;volume=16&amp;issue=6&amp;publication_year=1998&amp;pages=735-46&amp;pmid=10069079&amp;doi=10.1046/j.1365-313x.1998.00343.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref034">
<span class="label">34.</span><cite>
Lukowitz W, Gillmor CS, Scheible W-R. Positional cloning in Arabidopsis Why it feels good to have a Genome Initiative working for you. Plant Physiol. 2000;123(3):795–805.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.123.3.795" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1539260/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10889228/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Positional%20cloning%20in%20Arabidopsis%20Why%20it%20feels%20good%20to%20have%20a%20Genome%20Initiative%20working%20for%20you&amp;author=W%20Lukowitz&amp;author=CS%20Gillmor&amp;author=W-R%20Scheible&amp;volume=123&amp;issue=3&amp;publication_year=2000&amp;pages=795-805&amp;pmid=10889228&amp;doi=10.1104/pp.123.3.795&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref035">
<span class="label">35.</span><cite>
Pagny S, Cabanes-Macheteau M, Gillikin JW, Leborgne-Castel N, Lerouge P, Boston RS, et al. 
Protein recycling from the Golgi apparatus to the endoplasmic reticulum in plants and its minor contribution to calreticulin retention. Plant Cell. 2000;12(5):739–56.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.12.5.739" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC139924/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10810147/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Protein%20recycling%20from%20the%20Golgi%20apparatus%20to%20the%20endoplasmic%20reticulum%20in%20plants%20and%20its%20minor%20contribution%20to%20calreticulin%20retention&amp;author=S%20Pagny&amp;author=M%20Cabanes-Macheteau&amp;author=JW%20Gillikin&amp;author=N%20Leborgne-Castel&amp;author=P%20Lerouge&amp;volume=12&amp;issue=5&amp;publication_year=2000&amp;pages=739-56&amp;pmid=10810147&amp;doi=10.1105/tpc.12.5.739&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref036">
<span class="label">36.</span><cite>
Cutler SR, Ehrhardt DW, Griffitts JS, Somerville CR. Random GFP: cDNA fusions enable visualization of subcellular structures in cells of Arabidopsis at a high frequency. Proc Natl Acad Sci U S A. 2000;97(7):3718–23.
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.97.7.3718" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC16306/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10737809/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Random%20GFP:%20cDNA%20fusions%20enable%20visualization%20of%20subcellular%20structures%20in%20cells%20of%20Arabidopsis%20at%20a%20high%20frequency&amp;author=SR%20Cutler&amp;author=DW%20Ehrhardt&amp;author=JS%20Griffitts&amp;author=CR%20Somerville&amp;volume=97&amp;issue=7&amp;publication_year=2000&amp;pages=3718-23&amp;pmid=10737809&amp;doi=10.1073/pnas.97.7.3718&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref037">
<span class="label">37.</span><cite>
Gattolin S, Sorieul M, Hunter PR, Khonsari RH, Frigerio L. In vivo imaging of the tonoplast intrinsic protein family in Arabidopsis roots. BMC Plant Biol. 2009;9:133. doi: 1471-2229-9-133 [pii] 10.1186/1471-2229-9-133
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2229-9-133" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2784467/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19922653/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biol&amp;title=In%20vivo%20imaging%20of%20the%20tonoplast%20intrinsic%20protein%20family%20in%20Arabidopsis%20roots&amp;author=S%20Gattolin&amp;author=M%20Sorieul&amp;author=PR%20Hunter&amp;author=RH%20Khonsari&amp;author=L%20Frigerio&amp;volume=9&amp;publication_year=2009&amp;pages=133&amp;pmid=19922653&amp;doi=10.1186/1471-2229-9-133&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref038">
<span class="label">38.</span><cite>
Alonso JM, Stepanova AN, Leisse TJ, Kim CJ, Chen H, Shinn P, et al. 
Genome-wide insertional mutagenesis of Arabidopsis thaliana. Science. 2003;301(5633):653–7. 10.1126/science.1086391301/5633/653 [pii].
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1086391301/5633/653" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12893945/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide%20insertional%20mutagenesis%20of%20Arabidopsis%20thaliana&amp;author=JM%20Alonso&amp;author=AN%20Stepanova&amp;author=TJ%20Leisse&amp;author=CJ%20Kim&amp;author=H%20Chen&amp;volume=301&amp;issue=5633&amp;publication_year=2003&amp;pages=653-7&amp;pmid=12893945&amp;doi=10.1126/science.1086391301/5633/653&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref039">
<span class="label">39.</span><cite>
Daxinger L, Hunter B, Sheikh M, Jauvion V, Gasciolli V, Vaucheret H, et al. 
Unexpected silencing effects from T-DNA tags in Arabidopsis. Trends Plant Sci. 2008;13(1):4–6. 10.1016/j.tplants.2007.10.007
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tplants.2007.10.007" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18178509/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci&amp;title=Unexpected%20silencing%20effects%20from%20T-DNA%20tags%20in%20Arabidopsis&amp;author=L%20Daxinger&amp;author=B%20Hunter&amp;author=M%20Sheikh&amp;author=V%20Jauvion&amp;author=V%20Gasciolli&amp;volume=13&amp;issue=1&amp;publication_year=2008&amp;pages=4-6&amp;pmid=18178509&amp;doi=10.1016/j.tplants.2007.10.007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref040">
<span class="label">40.</span><cite>
Laxmi A, Pan J, Morsy M, Chen R. Light Plays an Essential Role in Intracellular Distribution of Auxin Efflux Carrier PIN2 in <em>Arabidopsis thaliana</em>
. PLoS ONE. 2008;3(1):e1510
10.1371/journal.pone.0001510
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0001510" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2200863/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18231596/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Light%20Plays%20an%20Essential%20Role%20in%20Intracellular%20Distribution%20of%20Auxin%20Efflux%20Carrier%20PIN2%20in%20Arabidopsis%20thaliana&amp;author=A%20Laxmi&amp;author=J%20Pan&amp;author=M%20Morsy&amp;author=R%20Chen&amp;volume=3&amp;issue=1&amp;publication_year=2008&amp;pages=e1510&amp;pmid=18231596&amp;doi=10.1371/journal.pone.0001510&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref041">
<span class="label">41.</span><cite>
Jaillais Y, Fobis-Loisy I, Miege C, Gaude T. Evidence for a sorting endosome in Arabidopsis root cells. Plant J. 2008;53(2):237–47. doi: TPJ3338 [pii] 10.1111/j.1365-313X.2007.03338.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2007.03338.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17999644/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Evidence%20for%20a%20sorting%20endosome%20in%20Arabidopsis%20root%20cells&amp;author=Y%20Jaillais&amp;author=I%20Fobis-Loisy&amp;author=C%20Miege&amp;author=T%20Gaude&amp;volume=53&amp;issue=2&amp;publication_year=2008&amp;pages=237-47&amp;pmid=17999644&amp;doi=10.1111/j.1365-313X.2007.03338.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref042">
<span class="label">42.</span><cite>
Nováková P, Hirsch S, Feraru E, Tejos R, van Wijk R, Viaene T, et al. 
SAC phosphoinositide phosphatases at the tonoplast mediate vacuolar function in Arabidopsis. Proceedings of the National Academy of Sciences. 2014;111(7):2818–23. 10.1073/pnas.1324264111
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1324264111" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3932866/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24550313/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=SAC%20phosphoinositide%20phosphatases%20at%20the%20tonoplast%20mediate%20vacuolar%20function%20in%20Arabidopsis&amp;author=P%20Nov%C3%A1kov%C3%A1&amp;author=S%20Hirsch&amp;author=E%20Feraru&amp;author=R%20Tejos&amp;author=R%20van%20Wijk&amp;volume=111&amp;issue=7&amp;publication_year=2014&amp;pages=2818-23&amp;pmid=24550313&amp;doi=10.1073/pnas.1324264111&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref043">
<span class="label">43.</span><cite>
Marchler-Bauer A, Lu S, Anderson JB, Chitsaz F, Derbyshire MK, DeWeese-Scott C, et al. 
CDD: a Conserved Domain Database for the functional annotation of proteins. Nucleic Acids Res. 2011;39(Database issue):D225–9. 10.1093/nar/gkq1189
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkq1189" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3013737/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21109532/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=CDD:%20a%20Conserved%20Domain%20Database%20for%20the%20functional%20annotation%20of%20proteins&amp;author=A%20Marchler-Bauer&amp;author=S%20Lu&amp;author=JB%20Anderson&amp;author=F%20Chitsaz&amp;author=MK%20Derbyshire&amp;volume=39&amp;publication_year=2011&amp;pages=D225-9&amp;pmid=21109532&amp;doi=10.1093/nar/gkq1189&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref044">
<span class="label">44.</span><cite>
Goodstein DM, Shu S, Howson R, Neupane R, Hayes RD, Fazo J, et al. 
Phytozome: a comparative platform for green plant genomics. Nucleic Acids Res. 2012;40(Database issue):D1178–86. 10.1093/nar/gkr944
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkr944" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3245001/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22110026/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Phytozome:%20a%20comparative%20platform%20for%20green%20plant%20genomics&amp;author=DM%20Goodstein&amp;author=S%20Shu&amp;author=R%20Howson&amp;author=R%20Neupane&amp;author=RD%20Hayes&amp;volume=40&amp;publication_year=2012&amp;pages=D1178-86&amp;pmid=22110026&amp;doi=10.1093/nar/gkr944&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref045">
<span class="label">45.</span><cite>
Winter D, Vinegar B, Nahal H, Ammar R, Wilson GV, Provart NJ. An “Electronic Fluorescent Pictograph” Browser for Exploring and Analyzing Large-Scale Biological Data Sets. PLoS ONE. 2007;2(8):e718
10.1371/journal.pone.0000718
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0000718" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1934936/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17684564/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=An%20%E2%80%9CElectronic%20Fluorescent%20Pictograph%E2%80%9D%20Browser%20for%20Exploring%20and%20Analyzing%20Large-Scale%20Biological%20Data%20Sets&amp;author=D%20Winter&amp;author=B%20Vinegar&amp;author=H%20Nahal&amp;author=R%20Ammar&amp;author=GV%20Wilson&amp;volume=2&amp;issue=8&amp;publication_year=2007&amp;pages=e718&amp;pmid=17684564&amp;doi=10.1371/journal.pone.0000718&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref046">
<span class="label">46.</span><cite>
Zeller G, Henz SR, Widmer CK, Sachsenberg T, Rätsch G, Weigel D, et al. 
Stress-induced changes in the Arabidopsis thaliana transcriptome analyzed using whole-genome tiling arrays. Plant J. 2009;58(6):1068–82. 10.1111/j.1365-313X.2009.03835.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2009.03835.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19222804/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Stress-induced%20changes%20in%20the%20Arabidopsis%20thaliana%20transcriptome%20analyzed%20using%20whole-genome%20tiling%20arrays&amp;author=G%20Zeller&amp;author=SR%20Henz&amp;author=CK%20Widmer&amp;author=T%20Sachsenberg&amp;author=G%20R%C3%A4tsch&amp;volume=58&amp;issue=6&amp;publication_year=2009&amp;pages=1068-82&amp;pmid=19222804&amp;doi=10.1111/j.1365-313X.2009.03835.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref047">
<span class="label">47.</span><cite>
Goda H, Sasaki E, Akiyama K, Maruyama-Nakashita A, Nakabayashi K, Li W, et al. 
The AtGenExpress hormone and chemical treatment data set: experimental design, data evaluation, model data analysis and data access. Plant J. 2008;55(3):526–42. 10.1111/j.0960-7412.2008.03510.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.0960-7412.2008.03510.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18419781/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=The%20AtGenExpress%20hormone%20and%20chemical%20treatment%20data%20set:%20experimental%20design,%20data%20evaluation,%20model%20data%20analysis%20and%20data%20access&amp;author=H%20Goda&amp;author=E%20Sasaki&amp;author=K%20Akiyama&amp;author=A%20Maruyama-Nakashita&amp;author=K%20Nakabayashi&amp;volume=55&amp;issue=3&amp;publication_year=2008&amp;pages=526-42&amp;pmid=18419781&amp;doi=10.1111/j.0960-7412.2008.03510.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref048">
<span class="label">48.</span><cite>
Shaner NC, Steinbach PA, Tsien RY. A guide to choosing fluorescent proteins. Nat Methods. 2005;2(12):905–9. 10.1038/nmeth819
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth819" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16299475/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=A%20guide%20to%20choosing%20fluorescent%20proteins&amp;author=NC%20Shaner&amp;author=PA%20Steinbach&amp;author=RY%20Tsien&amp;volume=2&amp;issue=12&amp;publication_year=2005&amp;pages=905-9&amp;pmid=16299475&amp;doi=10.1038/nmeth819&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref049">
<span class="label">49.</span><cite>
Szymanski DB, Cosgrove DJ. Dynamic coordination of cytoskeletal and cell wall systems during plant cell morphogenesis. Curr Biol. 2009;19(17):R800–11. 10.1016/j.cub.2009.07.056
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cub.2009.07.056" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19906582/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&amp;title=Dynamic%20coordination%20of%20cytoskeletal%20and%20cell%20wall%20systems%20during%20plant%20cell%20morphogenesis&amp;author=DB%20Szymanski&amp;author=DJ%20Cosgrove&amp;volume=19&amp;issue=17&amp;publication_year=2009&amp;pages=R800-11&amp;pmid=19906582&amp;doi=10.1016/j.cub.2009.07.056&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref050">
<span class="label">50.</span><cite>
Vermeer JE, van Leeuwen W, Tobena-Santamaria R, Laxalt AM, Jones DR, Divecha N, et al. 
Visualization of PtdIns3P dynamics in living plant cells. Plant J. 2006;47(5):687–700. doi: TPJ2830 [pii] 10.1111/j.1365-313X.2006.02830.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2006.02830.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16856980/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Visualization%20of%20PtdIns3P%20dynamics%20in%20living%20plant%20cells&amp;author=JE%20Vermeer&amp;author=W%20van%20Leeuwen&amp;author=R%20Tobena-Santamaria&amp;author=AM%20Laxalt&amp;author=DR%20Jones&amp;volume=47&amp;issue=5&amp;publication_year=2006&amp;pages=687-700&amp;pmid=16856980&amp;doi=10.1111/j.1365-313X.2006.02830.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref051">
<span class="label">51.</span><cite>
Zouhar J, Rojo E. Plant vacuoles: where did they come from and where are they heading?
Curr Opin Plant Biol. 2009;12(6):677–84. 10.1016/j.pbi.2009.08.004
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.pbi.2009.08.004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19783468/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Plant%20Biol&amp;title=Plant%20vacuoles:%20where%20did%20they%20come%20from%20and%20where%20are%20they%20heading?&amp;author=J%20Zouhar&amp;author=E%20Rojo&amp;volume=12&amp;issue=6&amp;publication_year=2009&amp;pages=677-84&amp;pmid=19783468&amp;doi=10.1016/j.pbi.2009.08.004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref052">
<span class="label">52.</span><cite>
Gulledge AA, Roberts AD, Vora H, Patel K, Loraine AE. Mining Arabidopsis thaliana RNA-seq data with Integrated Genome Browser reveals stress-induced alternative splicing of the putative splicing regulator SR45a. Am J Bot. 2012;99(2):219–31. 10.3732/ajb.1100355
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3732/ajb.1100355" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22291167/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am%20J%20Bot&amp;title=Mining%20Arabidopsis%20thaliana%20RNA-seq%20data%20with%20Integrated%20Genome%20Browser%20reveals%20stress-induced%20alternative%20splicing%20of%20the%20putative%20splicing%20regulator%20SR45a&amp;author=AA%20Gulledge&amp;author=AD%20Roberts&amp;author=H%20Vora&amp;author=K%20Patel&amp;author=AE%20Loraine&amp;volume=99&amp;issue=2&amp;publication_year=2012&amp;pages=219-31&amp;pmid=22291167&amp;doi=10.3732/ajb.1100355&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="pone.0125621.ref053">
<span class="label">53.</span><cite>
Nicol JW, Helt GA, Blanchard SG Jr., Raja A, Loraine AE. The Integrated Genome Browser: free software for distribution and exploration of genome-scale datasets. Bioinformatics. 2009;25(20):2730–1. 10.1093/bioinformatics/btp472
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btp472" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2759552/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19654113/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The%20Integrated%20Genome%20Browser:%20free%20software%20for%20distribution%20and%20exploration%20of%20genome-scale%20datasets&amp;author=JW%20Nicol&amp;author=GA%20Helt&amp;author=SG%20Blanchard&amp;author=A%20Raja&amp;author=AE%20Loraine&amp;volume=25&amp;issue=20&amp;publication_year=2009&amp;pages=2730-1&amp;pmid=19654113&amp;doi=10.1093/bioinformatics/btp472&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Fig. 
<em>rbb1-1</em> mutants show normal growth.</span><p>(a-b) Normal growth phenotype of 4 day-old seedlings (a), and 6 week-old plants (b) from the parental line, <em>rbb1-1</em>, Col-0, and <em>rbb1-2</em>. (c) Relative GFP content in the parental line and <em>rbb1-1</em> by immunoblotting. The intensity values for GFP from <a class="usa-link" href="#pone.0125621.g001">Fig 1O</a> were normalized against the Calreticulin (CRT) loading control.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s001.tif">Click here for additional data file.</a><sup> (32.4MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S2 Fig. Maximum fluorescence intensity for bulbs and tonoplast.</span><p>(a, b) Examples of fluorescence intensity profiles that were used to estimate intensity values in the parental line (a) and <em>rbb1-1</em> (b). Intensity value profiles from a line selection (white lines) across each bulb or the tonoplast were extracted in Image J from confocal images. Only the maximum value was collected for each line selection (arrowheads). (c, d) Maximum fluorescence intensity of bulbs and the tonoplast from cotyledon (c) and hypocotyl cells (d) in the parental line or <em>rbb1-1</em> grown in the light. Images were analyzed as in (a, b). Data shown is the average of maximum intensity values for 10–15 images from at least 3 seedlings each. The numbers inside the white bars correspond to the calculated ratio between the maximum fluorescence intensity in the bulbs and that of the tonoplast. Identical microscope settings were used when collecting data for cotyledons or when collecting data for hypocotyls.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s002.tif">Click here for additional data file.</a><sup> (3MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S3 Fig. Quantification of TVS and growth phenotypes of <em>rbb1-1</em>.</span><p>(a-b) An example of images used for TVS quantification. Thick optical sections were captured from hypocotyls (a). A seedling from the parental line is shown. The boxed area is enlarged in (b) to show the TVS that were counted (arrowhead). Bar = 20 μm. (c-d) The parental line and <em>rbb1-1</em> have similar hypocotyl growth in the dark. Seedlings were grown in the dark for up to 5 days and seedlings were imaged in a scanner. Hypocotyl length was measured using Image J (NIH) from seedlings at days 3 and 5 (n = 10). Black bars represent parental line; gray bars represent <em>rbb1-1</em> mutant.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s003.tif">Click here for additional data file.</a><sup> (672.7KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S4 Fig. Lysotracker Red labels the tonoplast in addition to the vacuole lumen in Arabidopsis.</span><p>GFP-TIP2;1 seedlings were stained with Lysotracker Red for 2 h and imaged by confocal microscopy. The tonoplast marker GFP-TIP2;1 (green, a) co-localizes with the membrane signal of Lysotracker Red (magenta, b). White signal in the merged image (c) represents co-localized pixels. Scale bar = 20 μm.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s004.tif">Click here for additional data file.</a><sup> (16.3MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>S5 Fig. Localization of 3xYpet-RBB1.</span><p>(a-b) 3xYpet-RBB1 can be detected in cotyledons, hypocotyls and roots. Four-day-old seedlings expressing 3xYpet-RBB1 under the control of its native promoter were imaged by confocal microscopy. An overlay of the fluorescence signal and the bright field image is shown in (a) and the fluorescence signal is shown in (b). The arrow in (b) indicates the position of the root tip. Scale bar = 200μm. (c-e) An example where the 3xYpet-RBB1 co-localizes with Lysotracker Red. 3xYpet-RBB1 is shown in green (c), Lysotracker Red is shown in magenta (d) and the merged image is shown (e). Scale bar = 10 μm. (f) The 3xYpet-RBB1 signal remains associated with the TVS tip. Cotyledons that were stained with Lysotracker were captured by time lapse microscopy every 7.75 sec. Note that the 3xYpet-RBB1 signal (green) remains associated with the tip of the elongating TVS labeled with Lysotracker Red (magenta). Scale bar = 10 μm.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s005.tif">Click here for additional data file.</a><sup> (16.1MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>S6 Fig. 3xYpet-RBB1 Immunoblot.</span><p>This blot corresponds to the image shown in <a class="usa-link" href="#pone.0125621.g006">Fig 6O</a> but includes the entire blot to show the size of the 3xYpet-RBB1 fusion. * Non-specific bands.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s006.tif">Click here for additional data file.</a><sup> (8.5MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>S1 Movie. Z-stack of Col-WT cotyledon stained with Lysotracker Red.</span><p>Four-day-old Col-WT seedlings stained with Lysotracker Red as in <a class="usa-link" href="#pone.0125621.g005">Fig 5F</a>. The stack represents 20 optical sections every 1 μm. Image dimensions are x:106.17 μm, y:106.17 μm.</p>
<p>(AVI)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s007.avi">Click here for additional data file.</a><sup> (6.5MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p">
<span>S2 Movie. Z-stack of <em>rbb1-2</em> cotyledon stained with Lysotracker Red.</span><p>Four-day-old <em>rbb1-2</em> seedlings stained with Lysotracker Red as in <a class="usa-link" href="#pone.0125621.g005">Fig 5G</a>. The stack represents 45 optical sections every 1 μm. Image dimensions are x:106.17 μm, y:106.17 μm.</p>
<p>(AVI)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s008.avi">Click here for additional data file.</a><sup> (4.1MB, avi) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="caption p">
<span>S1 Table. List of primers used for Recombineering.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s009.docx">Click here for additional data file.</a><sup> (16.9KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="caption p">
<span>S2 Table. List of primers used for RT-PCR.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4411111/bin/pone.0125621.s010.docx">Click here for additional data file.</a><sup> (14.2KB, docx) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All relevant data are within the paper and its Supporting Information files.</p></section></section></section></section></article>
</main></body></html>