<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7516158/"/><title>Draft Genome Sequences of Lactobacillales Isolated from the International Space Station - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Microbiol Resour Announc</button></div>. 2020 Sep 24;9(39):e00942-20. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/MRA.00942-20" rel="noopener noreferrer" target="_blank">10.1128/MRA.00942-20</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Draft Genome Sequences of <em>Lactobacillales</em> Isolated from the International Space Station</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bharadwaj%20AR%22%5BAuthor%5D"><span class="name western">Achintya R Bharadwaj</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Achintya R Bharadwaj</span></h3>
<div class="p">
<sup>a</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bharadwaj%20AR%22%5BAuthor%5D"><span class="name western">Achintya R Bharadwaj</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D"><span class="name western">Nitin K Singh</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Nitin K Singh</span></h3>
<div class="p">
<sup>a</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D"><span class="name western">Nitin K Singh</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wood%20JM%22%5BAuthor%5D"><span class="name western">Jason M Wood</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jason M Wood</span></h3>
<div class="p">
<sup>a</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wood%20JM%22%5BAuthor%5D"><span class="name western">Jason M Wood</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Debieu%20M%22%5BAuthor%5D"><span class="name western">Marilyne Debieu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Marilyne Debieu</span></h3>
<div class="p">
<sup>b</sup>Biotia, New York, New York, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Debieu%20M%22%5BAuthor%5D"><span class="name western">Marilyne Debieu</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Hara%20NB%22%5BAuthor%5D"><span class="name western">Niamh B O’Hara</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Niamh B O’Hara</span></h3>
<div class="p">
<sup>b</sup>Biotia, New York, New York, USA</div>
<div class="p">
<sup>c</sup>Department Cell Biology/College of Medicine, SUNY Downstate Health Sciences University, Brooklyn, New York, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Hara%20NB%22%5BAuthor%5D"><span class="name western">Niamh B O’Hara</span></a>
</div>
</div>
<sup>b,</sup><sup>c</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Karouia%20F%22%5BAuthor%5D"><span class="name western">Fathi Karouia</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Fathi Karouia</span></h3>
<div class="p">
<sup>d</sup>Blue Marble Space Institute of Science, Exobiology Branch, NASA Ames Research Center, Moffett Field, California, USA</div>
<div class="p">
<sup>e</sup>Space Research Within Reach, San Francisco, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Karouia%20F%22%5BAuthor%5D"><span class="name western">Fathi Karouia</span></a>
</div>
</div>
<sup>d,</sup><sup>e</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D"><span class="name western">Christopher E Mason</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Christopher E Mason</span></h3>
<div class="p">
<sup>f</sup>The WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, New York, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D"><span class="name western">Christopher E Mason</span></a>
</div>
</div>
<sup>f</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Kasthuri Venkateswaran</span></h3>
<div class="p">
<sup>a</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a>
</div>
</div>
<sup>a,</sup><sup>✉</sup>
</div>
<div class="cg p">Editor: <span class="name western">Julie C Dunning Hotopp</span><sup>g</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>a</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div id="aff2">
<sup>b</sup>Biotia, New York, New York, USA</div>
<div id="aff3">
<sup>c</sup>Department Cell Biology/College of Medicine, SUNY Downstate Health Sciences University, Brooklyn, New York, USA</div>
<div id="aff4">
<sup>d</sup>Blue Marble Space Institute of Science, Exobiology Branch, NASA Ames Research Center, Moffett Field, California, USA</div>
<div id="aff5">
<sup>e</sup>Space Research Within Reach, San Francisco, California, USA</div>
<div id="aff6">
<sup>f</sup>The WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, New York, USA</div>
<div id="aff7">
<sup>g</sup>University of Maryland School of Medicine</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>✉</sup><p class="display-inline">Address correspondence to Kasthuri Venkateswaran, <span>kjvenkat@jpl.nasa.gov</span>.</p>
</div>
<div class="fn" id="fn1"><p><strong>Citation</strong> Bharadwaj AR, Singh NK, Wood JM, Debieu M, O’Hara NB, Karouia F, Mason CE, Venkateswaran K. 2020. Draft genome sequences of <em>Lactobacillales</em> isolated from the International Space Station. Microbiol Resour Announc 9:e00942-20. <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/MRA.00942-20" rel="noopener noreferrer" target="_blank">https://doi.org/10.1128/MRA.00942-20</a>.</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Julie C Dunning Hotopp</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2020 Aug 9; Accepted 2020 Aug 27; Collection date 2020 Sep.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2020 Bharadwaj et al.</div>
<p>This is an open-access article distributed under the terms of the <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">Creative Commons Attribution 4.0 International license</a>.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC7516158  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32972947/">32972947</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><hr class="headless"/>
<p>Nineteen strains from the order <em>Lactobacillales</em> were isolated from the International Space Station and commercial resupply vehicle, and whole-genome sequences (WGS) were generated. WGS would permit the characterization of these potentially pathogenic bacteria that have been adapting to the extreme conditions of the space environment.</p></section><section class="abstract" id="abstract2"><h2>ABSTRACT</h2>
<p>Nineteen strains from the order <em>Lactobacillales</em> were isolated from the International Space Station and commercial resupply vehicle, and whole-genome sequences (WGS) were generated. WGS would permit the characterization of these potentially pathogenic bacteria that have been adapting to the extreme conditions of the space environment.</p></section><section id="s1"><h2 class="pmc_sec_title">ANNOUNCEMENT</h2>
<p>The order <em>Lactobacillales</em> consists of Gram stain-positive, facultative anaerobes validly described by Ludwig et al. (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>). Members of the genus <em>Enterococcus</em> are found to possess human pathogenicity characteristics such as antibiotic resistance (<a aria-describedby="B2" class="usa-link" href="#B2">2</a>) and therefore pose health concerns for those on Earth and astronauts residing in the International Space Station (ISS). However, <em>Aerococcus urinaeequi</em>, a nonpathogenic strain, was first isolated from horse urine (<a aria-describedby="B3" class="usa-link" href="#B3">3</a>). Astronauts on long flights are immunocompromised due to microgravity-induced physiological and mental stress. Decreased immune response allows bacteria to take growth advantage due to their adaptability potential in the space environment (<a aria-describedby="B4" class="usa-link" href="#B4">4</a>). Understanding the genomic makeup of these potential pathogens will help the development of suitable countermeasure and mitigation strategies. Members of the order <em>Lactobacillales</em> isolated from the ISS and the commercial resupply vehicle (CRV) surfaces were <em>Enterococcus faecalis</em>, <em>Enterococcus faecium</em>, and <em>Aerococcus urinaeequi</em> (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>, <a aria-describedby="B6" class="usa-link" href="#B6">6</a>). <em>E. faecalis</em> and <em>E. faecium</em> have been reported as nosocomial isolates harboring vancomycin and ampicillin resistance (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>). <em>A. urinaeequi</em> was isolated from a chronic kidney disease patient and has also been reported to be resistant to vancomycin (<a aria-describedby="B6" class="usa-link" href="#B6">6</a>). Further characterization of the whole-genome sequences (WGS) of these ISS environmental strains, including virulence genes, and subsequent confirmation in animal models are required to decipher their potential pathogenicity.</p>
<p>The strains used for the WGS were collected from three different ISS locations across two flights and seven different surface locations, including one field control on CRV6, and are detailed in <a class="usa-link" href="#tab1">Table 1</a> (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>). The samples collected from the ISS were brought back to Earth and aseptically processed, and suitable aliquots of the sample concentrate (100 μl) were plated onto Reasoner’s 2A (R2A) or Trypticase soy agar (TSA) medium and incubated at 25°C for 7 days. A single well-isolated colony on a culture plate was archived at −80°C. Genomic DNA was extracted from the overnight-grown cultures on TSA medium using a ZymoBIOMICS DNA MagBead kit according to the manufacturer’s instructions.</p>
<section class="tw xbox font-sm" id="tab1"><h3 class="obj_head">TABLE 1.</h3>
<div class="caption p"><p>Metadata and genome statistics of <em>Aerococcus</em> and <em>Enterococcus</em> strains isolated from various ISS and CRV6 environmental surfaces during the Microbial Tracking-1 Flight Project<a class="usa-link" href="#ngtab1.1"><sup><em>a</em></sup></a></p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
</colgroup>
<thead><tr>
<th colspan="1" rowspan="1">Sample name</th>
<th colspan="1" rowspan="1">ANI (%)<a class="usa-link" href="#ngtab1.2"><sup><em>b</em></sup>
</a>
</th>
<th colspan="1" rowspan="1">GenBank accession no.</th>
<th colspan="1" rowspan="1">Raw sequence accession no.</th>
<th colspan="1" rowspan="1">Flight no./location<a class="usa-link" href="#ngtab1.3"><sup><em>c</em></sup>
</a>
</th>
<th colspan="1" rowspan="1">Location description</th>
<th colspan="1" rowspan="1">No. of contigs</th>
<th colspan="1" rowspan="1">Genome size (bp)</th>
<th colspan="1" rowspan="1">
<em>N</em><sub>50</sub> (bp)</th>
<th colspan="1" rowspan="1">Median sequencing depth (×)</th>
<th colspan="1" rowspan="1">No. of QC reads</th>
<th colspan="1" rowspan="1">No. of raw reads</th>
<th colspan="1" rowspan="1">G+C content (%)</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">151250015-1-258-55</td>
<td colspan="1" rowspan="1">96(A)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACGAN000000000" rel="noopener noreferrer" target="_blank">JACGAN000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341118" rel="noopener noreferrer" target="_blank">SRR12341118</a></td>
<td colspan="1" rowspan="1">F1-1</td>
<td colspan="1" rowspan="1">Cupola</td>
<td colspan="1" rowspan="1">35</td>
<td colspan="1" rowspan="1">1,981,406</td>
<td colspan="1" rowspan="1">130,552</td>
<td colspan="1" rowspan="1">282.59</td>
<td colspan="1" rowspan="1">6,701,742</td>
<td colspan="1" rowspan="1">3,361,020</td>
<td colspan="1" rowspan="1">39.5</td>
</tr>
<tr>
<td colspan="1" rowspan="1">151250015-2-258-56</td>
<td colspan="1" rowspan="1">96(A)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACGAM000000000" rel="noopener noreferrer" target="_blank">JACGAM000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341117" rel="noopener noreferrer" target="_blank">SRR12341117</a></td>
<td colspan="1" rowspan="1">F1-2</td>
<td colspan="1" rowspan="1">WHC</td>
<td colspan="1" rowspan="1">36</td>
<td colspan="1" rowspan="1">1,981,307</td>
<td colspan="1" rowspan="1">130,552</td>
<td colspan="1" rowspan="1">645.54</td>
<td colspan="1" rowspan="1">15,523,394</td>
<td colspan="1" rowspan="1">7,803,035</td>
<td colspan="1" rowspan="1">39.5</td>
</tr>
<tr>
<td colspan="1" rowspan="1">151250009-4-258-51</td>
<td colspan="1" rowspan="1">96(A)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACGAO000000000" rel="noopener noreferrer" target="_blank">JACGAO000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341119" rel="noopener noreferrer" target="_blank">SRR12341119</a></td>
<td colspan="1" rowspan="1">F1-4</td>
<td colspan="1" rowspan="1">Dining table</td>
<td colspan="1" rowspan="1">38</td>
<td colspan="1" rowspan="1">1,981,891</td>
<td colspan="1" rowspan="1">130,552</td>
<td colspan="1" rowspan="1">885.27</td>
<td colspan="1" rowspan="1">20,960,030</td>
<td colspan="1" rowspan="1">10,516,330</td>
<td colspan="1" rowspan="1">39.5</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF2*SW-B2</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPC000000000" rel="noopener noreferrer" target="_blank">JACDPC000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341307" rel="noopener noreferrer" target="_blank">SRR12341307</a></td>
<td colspan="1" rowspan="1">F2-2</td>
<td colspan="1" rowspan="1">WHC</td>
<td colspan="1" rowspan="1">26</td>
<td colspan="1" rowspan="1">2,928,643</td>
<td colspan="1" rowspan="1">679,975</td>
<td colspan="1" rowspan="1">736.61</td>
<td colspan="1" rowspan="1">22,911,864</td>
<td colspan="1" rowspan="1">11,494,983</td>
<td colspan="1" rowspan="1">37.4</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF2SG-B4</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPE000000000" rel="noopener noreferrer" target="_blank">JACDPE000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341300" rel="noopener noreferrer" target="_blank">SRR12341300</a></td>
<td colspan="1" rowspan="1">CRV6-2</td>
<td colspan="1" rowspan="1">Outside capsule</td>
<td colspan="1" rowspan="1">29</td>
<td colspan="1" rowspan="1">2,926,313</td>
<td colspan="1" rowspan="1">293,834</td>
<td colspan="1" rowspan="1">499.55</td>
<td colspan="1" rowspan="1">15,063,826</td>
<td colspan="1" rowspan="1">7,552,409</td>
<td colspan="1" rowspan="1">37.4</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF3SG-B2</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPF000000000" rel="noopener noreferrer" target="_blank">JACDPF000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341299" rel="noopener noreferrer" target="_blank">SRR12341299</a></td>
<td colspan="1" rowspan="1">CRV6-3</td>
<td colspan="1" rowspan="1">Outside capsule</td>
<td colspan="1" rowspan="1">20</td>
<td colspan="1" rowspan="1">2,948,392</td>
<td colspan="1" rowspan="1">1,487,444</td>
<td colspan="1" rowspan="1">600.00</td>
<td colspan="1" rowspan="1">19,513,428</td>
<td colspan="1" rowspan="1">9,781,100</td>
<td colspan="1" rowspan="1">37.3</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF4SG-B3</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPG000000000" rel="noopener noreferrer" target="_blank">JACDPG000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341298" rel="noopener noreferrer" target="_blank">SRR12341298</a></td>
<td colspan="1" rowspan="1">CRV6-4</td>
<td colspan="1" rowspan="1">Inside capsule</td>
<td colspan="1" rowspan="1">24</td>
<td colspan="1" rowspan="1">2,928,137</td>
<td colspan="1" rowspan="1">352,081</td>
<td colspan="1" rowspan="1">559.82</td>
<td colspan="1" rowspan="1">17,637,974</td>
<td colspan="1" rowspan="1">8,849,109</td>
<td colspan="1" rowspan="1">37.4</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF4SG-B5</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPH000000000" rel="noopener noreferrer" target="_blank">JACDPH000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341297" rel="noopener noreferrer" target="_blank">SRR12341297</a></td>
<td colspan="1" rowspan="1">CRV6-4</td>
<td colspan="1" rowspan="1">Inside capsule</td>
<td colspan="1" rowspan="1">22</td>
<td colspan="1" rowspan="1">2,929,029</td>
<td colspan="1" rowspan="1">680,116</td>
<td colspan="1" rowspan="1">675.00</td>
<td colspan="1" rowspan="1">21,186,826</td>
<td colspan="1" rowspan="1">10,633,808</td>
<td colspan="1" rowspan="1">37.4</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF5SG-B2</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPI000000000" rel="noopener noreferrer" target="_blank">JACDPI000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra?run=SRR12341296" rel="noopener noreferrer" target="_blank">SRR12341296</a></td>
<td colspan="1" rowspan="1">CRV6-5</td>
<td colspan="1" rowspan="1">Inside capsule</td>
<td colspan="1" rowspan="1">27</td>
<td colspan="1" rowspan="1">2,926,858</td>
<td colspan="1" rowspan="1">293,439</td>
<td colspan="1" rowspan="1">467.41</td>
<td colspan="1" rowspan="1">13,779,006</td>
<td colspan="1" rowspan="1">6,902,599</td>
<td colspan="1" rowspan="1">37.4</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF6SG-B1</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPJ000000000" rel="noopener noreferrer" target="_blank">JACDPJ000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341295" rel="noopener noreferrer" target="_blank">SRR12341295</a></td>
<td colspan="1" rowspan="1">CRV6-6</td>
<td colspan="1" rowspan="1">Inside capsule</td>
<td colspan="1" rowspan="1">21</td>
<td colspan="1" rowspan="1">2,928,522</td>
<td colspan="1" rowspan="1">680,116</td>
<td colspan="1" rowspan="1">673.66</td>
<td colspan="1" rowspan="1">20,619,150</td>
<td colspan="1" rowspan="1">10,336,467</td>
<td colspan="1" rowspan="1">37.4</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF6SG-B2</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPK000000000" rel="noopener noreferrer" target="_blank">JACDPK000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341294" rel="noopener noreferrer" target="_blank">SRR12341294</a></td>
<td colspan="1" rowspan="1">CRV6-6</td>
<td colspan="1" rowspan="1">Inside capsule</td>
<td colspan="1" rowspan="1">23</td>
<td colspan="1" rowspan="1">2,928,581</td>
<td colspan="1" rowspan="1">352,365</td>
<td colspan="1" rowspan="1">835.71</td>
<td colspan="1" rowspan="1">25,542,378</td>
<td colspan="1" rowspan="1">12,807,968</td>
<td colspan="1" rowspan="1">37.4</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF6SG-B4</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPL000000000" rel="noopener noreferrer" target="_blank">JACDPL000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341293" rel="noopener noreferrer" target="_blank">SRR12341293</a></td>
<td colspan="1" rowspan="1">CRV6-6</td>
<td colspan="1" rowspan="1">Inside capsule</td>
<td colspan="1" rowspan="1">21</td>
<td colspan="1" rowspan="1">2,928,384</td>
<td colspan="1" rowspan="1">679,976</td>
<td colspan="1" rowspan="1">811.61</td>
<td colspan="1" rowspan="1">24,973,062</td>
<td colspan="1" rowspan="1">12,536,216</td>
<td colspan="1" rowspan="1">37.4</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF7SG-B2</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPM000000000" rel="noopener noreferrer" target="_blank">JACDPM000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341305" rel="noopener noreferrer" target="_blank">SRR12341305</a></td>
<td colspan="1" rowspan="1">CRV6-7</td>
<td colspan="1" rowspan="1">Inside capsule</td>
<td colspan="1" rowspan="1">19</td>
<td colspan="1" rowspan="1">2,948,759</td>
<td colspan="1" rowspan="1">1,487,531</td>
<td colspan="1" rowspan="1">595.98</td>
<td colspan="1" rowspan="1">19,817,626</td>
<td colspan="1" rowspan="1">9,946,584</td>
<td colspan="1" rowspan="1">37.3</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF7SG-B3</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPN000000000" rel="noopener noreferrer" target="_blank">JACDPN000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341304" rel="noopener noreferrer" target="_blank">SRR12341304</a></td>
<td colspan="1" rowspan="1">CRV6-7</td>
<td colspan="1" rowspan="1">Inside capsule</td>
<td colspan="1" rowspan="1">21</td>
<td colspan="1" rowspan="1">2,928,555</td>
<td colspan="1" rowspan="1">680,118</td>
<td colspan="1" rowspan="1">523.66</td>
<td colspan="1" rowspan="1">16,480,536</td>
<td colspan="1" rowspan="1">8,263,734</td>
<td colspan="1" rowspan="1">37.4</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF8SG-B1</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPO000000000" rel="noopener noreferrer" target="_blank">JACDPO000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341303" rel="noopener noreferrer" target="_blank">SRR12341303</a></td>
<td colspan="1" rowspan="1">CRV6-8</td>
<td colspan="1" rowspan="1">Inside capsule</td>
<td colspan="1" rowspan="1">20</td>
<td colspan="1" rowspan="1">2,948,399</td>
<td colspan="1" rowspan="1">1,487,531</td>
<td colspan="1" rowspan="1">543.75</td>
<td colspan="1" rowspan="1">17,797,982</td>
<td colspan="1" rowspan="1">8,925,021</td>
<td colspan="1" rowspan="1">37.3</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF8SG-B2</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPP000000000" rel="noopener noreferrer" target="_blank">JACDPP000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341302" rel="noopener noreferrer" target="_blank">SRR12341302</a></td>
<td colspan="1" rowspan="1">CRV6-8</td>
<td colspan="1" rowspan="1">Inside capsule</td>
<td colspan="1" rowspan="1">30</td>
<td colspan="1" rowspan="1">2,926,820</td>
<td colspan="1" rowspan="1">293,834</td>
<td colspan="1" rowspan="1">570.54</td>
<td colspan="1" rowspan="1">16,894,654</td>
<td colspan="1" rowspan="1">8,447,267</td>
<td colspan="1" rowspan="1">37.4</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIF8SG-B3</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPQ000000000" rel="noopener noreferrer" target="_blank">JACDPQ000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341301" rel="noopener noreferrer" target="_blank">SRR12341301</a></td>
<td colspan="1" rowspan="1">CRV6-8</td>
<td colspan="1" rowspan="1">Inside capsule</td>
<td colspan="1" rowspan="1">20</td>
<td colspan="1" rowspan="1">2,948,924</td>
<td colspan="1" rowspan="1">1,487,531</td>
<td colspan="1" rowspan="1">495.54</td>
<td colspan="1" rowspan="1">16,544,212</td>
<td colspan="1" rowspan="1">8,303,048</td>
<td colspan="1" rowspan="1">37.3</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIFCSG-B3</td>
<td colspan="1" rowspan="1">99(B)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACDPD000000000" rel="noopener noreferrer" target="_blank">JACDPD000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341306" rel="noopener noreferrer" target="_blank">SRR12341306</a></td>
<td colspan="1" rowspan="1">CRV6-FC</td>
<td colspan="1" rowspan="1">Field control</td>
<td colspan="1" rowspan="1">30</td>
<td colspan="1" rowspan="1">2,926,028</td>
<td colspan="1" rowspan="1">293,439</td>
<td colspan="1" rowspan="1">570.54</td>
<td colspan="1" rowspan="1">17,408,136</td>
<td colspan="1" rowspan="1">8,727,751</td>
<td colspan="1" rowspan="1">37.4</td>
</tr>
<tr>
<td colspan="1" rowspan="1">IIFCSG-B5</td>
<td colspan="1" rowspan="1">95(C)</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/JACGAP000000000" rel="noopener noreferrer" target="_blank">JACGAP000000000</a></td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341224" rel="noopener noreferrer" target="_blank">SRR12341224</a></td>
<td colspan="1" rowspan="1">CRV6-FC</td>
<td colspan="1" rowspan="1">Field control</td>
<td colspan="1" rowspan="1">71</td>
<td colspan="1" rowspan="1">2,821,574</td>
<td colspan="1" rowspan="1">91,275</td>
<td colspan="1" rowspan="1">866.52</td>
<td colspan="1" rowspan="1">28,642,870</td>
<td colspan="1" rowspan="1">14,378,989</td>
<td colspan="1" rowspan="1">38.0</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/tab1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="ngtab1.1">
<sup>a</sup><p class="display-inline">Abbreviations: ANI, average nucleotide identity; F1, ISS flight 1; F2, ISS flight 2; WHC, waste and hygiene compartment; FC, field control (a sampling wipe was exposed to the air for 120 s at the center of CRV6); QC, quality control.</p>
</div>
<div class="fn" id="ngtab1.2">
<sup>b</sup><p class="display-inline">The 16S rRNA gene sequences were retrieved from the WGS, and BLAST analysis was conducted against type strains of all 16S rRNA sequences in the NCBI database. The bacterial species identity was determined when the queried sequence showed &gt;97.5% similarity with the 16S rRNA gene sequences of the type strain. The WGS of the nearest neighbor was further selected for ANI evaluation: A, <em>A. urinaeequi</em> DSM 20341<sup>T</sup>; B, <em>E. faecalis</em> DSM 20478<sup>T</sup>; C, <em>E. faecium</em> DSM 20477<sup>T</sup>.</p>
</div>
<div class="fn" id="ngtab1.3">
<sup>c</sup><p class="display-inline">Hyphenated designations indicate the flight number followed by the location; for example, F1-1 indicates flight 1 and location 1.</p>
</div>
</div></section><p>The WGS of 19 bacterial isolates were prepared using the Illumina Nextera Flex protocol for library preparation, as used in similar studies (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>). The NovaSeq 6000 S4 flow cell paired-end 2 × 150-bp platform was used to execute paired-end sequencing. FastQC v0.11.7 was used to validate the quality of the raw sequencing data (<a aria-describedby="B9" class="usa-link" href="#B9">9</a>). Adapter trimming and quality filtering were carried out using the software fastp v0.20.0 to perform quality control (<a aria-describedby="B10" class="usa-link" href="#B10">10</a>). The cleaned sequences were assembled using SPAdes v3.11.1 (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>). The <em>N</em><sub>50</sub> values, numbers of contigs, and total genome lengths were generated using QUAST v5.0.2 and used to assess the quality of the final assembly (<a aria-describedby="B12" class="usa-link" href="#B12">12</a>). The average nucleotide identity was calculated by comparing all strains with their respective type strains, and their taxonomic affiliations, as well as genome statistics, are given in <a class="usa-link" href="#tab1">Table 1</a> (<a aria-describedby="B13" class="usa-link" href="#B13">13</a>). The NCBI Prokaryotic Genome Annotation Pipeline v4.12 was used for genome annotation. Default parameters were used for all software.</p>
<section id="s1.1"><h3 class="pmc_sec_title">Data availability.</h3>
<p>This WGS project has been deposited at DDBJ/ENA/GenBank, and the accession numbers are given in <a class="usa-link" href="#tab1">Table 1</a> (BioProject accession no. <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA645454" rel="noopener noreferrer" target="_blank">PRJNA645454</a> with 16 strains and <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA649272" rel="noopener noreferrer" target="_blank">PRJNA649272</a> with 3 strains). The versions described in this paper are the first versions.</p></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>Part of the research described in this publication was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with NASA. We thank astronauts Captain Terry Virts for collecting samples aboard the ISS and the implementation team at NASA Ames Research Center for coordinating this effort. We thank Ryan Kemp (Zymo Corporation) for extracting DNA and Dan Butler (Cornell Medicine) for generating shotgun sequencing using NovaSeq.</p>
<p>Government sponsorship is acknowledged. This research was funded by a 2012 Space Biology NNH12ZTT001N grant (no. 19-12829-26) under task order NNN13D111T awarded to K.V., which also funded a postdoctoral fellowship for J.M.W., and a subcontract to Biotia, Inc.</p></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>Ludwig W, Schleifer KH, Whitman WB. 
2009. 
Order II. <em>Lactobacillales</em> ord. nov, p 464
<em>In</em>
De Vos P, Garrity GM, Jones D, Krieg NR, Ludwig W, Rainey FA, Schleifer KH, Whitman WB (ed), Bergey’s manual of systematic bacteriology, 2nd ed, vol 3
The Firmicutes
Springer, New York, NY. doi: 10.1002/9781118960608.obm00058.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/9781118960608.obm00058" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Bergey%E2%80%99s%20manual%20of%20systematic%20bacteriology&amp;author=W%20Ludwig&amp;author=KH%20Schleifer&amp;author=WB%20Whitman&amp;publication_year=2009&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>Christoff AP, Sereia AFR, Cruz GNF, Bastiani DCd, Silva VL, Hernandes C, Nascente APM, dos Reis AA, Viessi RG, Marques ASP, Braga BS, Raduan TPL, Martino MDV, de Menezes FG, de Oliveira LFV. 
2020. 
One year cross-sectional study in adult and neonatal intensive care units reveals the bacterial and antimicrobial resistance genes profiles in patients and hospital surfaces. PLoS One
15:e0234127. doi: 10.1371/journal.pone.0234127.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0234127" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7269242/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32492060/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=One%20year%20cross-sectional%20study%20in%20adult%20and%20neonatal%20intensive%20care%20units%20reveals%20the%20bacterial%20and%20antimicrobial%20resistance%20genes%20profiles%20in%20patients%20and%20hospital%20surfaces&amp;author=AP%20Christoff&amp;author=AFR%20Sereia&amp;author=GNF%20Cruz&amp;author=DCd%20Bastiani&amp;author=VL%20Silva&amp;volume=15&amp;publication_year=2020&amp;pages=e0234127&amp;pmid=32492060&amp;doi=10.1371/journal.pone.0234127&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>Felis GE, Torriani S, Dellaglio F. 
2005. 
Reclassification of <em>Pediococcus urinaeequi</em> (ex Mees 1934) Garvie 1988 as <em>Aerococcus urinaeequi</em> comb. nov. Int J Syst Evol Microbiol
55:1325–1327. doi: 10.1099/ijs.0.63324-0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1099/ijs.0.63324-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15879276/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&amp;title=Reclassification%20of%20Pediococcus%20urinaeequi%20(ex%20Mees%201934)%20Garvie%201988%20as%20Aerococcus%20urinaeequi%20comb.%20nov&amp;author=GE%20Felis&amp;author=S%20Torriani&amp;author=F%20Dellaglio&amp;volume=55&amp;publication_year=2005&amp;pages=1325-1327&amp;pmid=15879276&amp;doi=10.1099/ijs.0.63324-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>Taylor PW.
2015. 
Impact of space flight on bacterial virulence and antibiotic susceptibility. Infect Drug Resist
8:249–262. doi: 10.2147/IDR.S67275.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2147/IDR.S67275" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4524529/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26251622/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Infect%20Drug%20Resist&amp;title=Impact%20of%20space%20flight%20on%20bacterial%20virulence%20and%20antibiotic%20susceptibility&amp;author=PW%20Taylor&amp;volume=8&amp;publication_year=2015&amp;pages=249-262&amp;pmid=26251622&amp;doi=10.2147/IDR.S67275&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>Agudelo Higuita NI, Huycke MM. 
2014. 
Enterococcal disease, epidemiology, and implications for treatment
<em>In</em>
Gilmore MS, Clewell DB, Ike Y, et al.  (ed), Enterococci: from commensals to leading causes of drug resistant infection. Eye and Ear Infirmary, Boston, MA: 
<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/books/NBK190429/" rel="noopener noreferrer" target="_blank">https://www.ncbi.nlm.nih.gov/books/NBK190429/</a>.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Enterococci:%20from%20commensals%20to%20leading%20causes%20of%20drug%20resistant%20infection&amp;author=NI%20Agudelo%20Higuita&amp;author=MM%20Huycke&amp;publication_year=2014&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>Zhou W, Niu D, Zhang Z, Ning M, Shen H, Zhang K. 
2014. 
Vancomycin resistance due to vanA in an <em>Aerococcus viridans</em> isolate. Indian J Med Microbiol
32:462–465. doi: 10.4103/0255-0857.142238.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4103/0255-0857.142238" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25297044/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Indian%20J%20Med%20Microbiol&amp;title=Vancomycin%20resistance%20due%20to%20vanA%20in%20an%20Aerococcus%20viridans%20isolate&amp;author=W%20Zhou&amp;author=D%20Niu&amp;author=Z%20Zhang&amp;author=M%20Ning&amp;author=H%20Shen&amp;volume=32&amp;publication_year=2014&amp;pages=462-465&amp;pmid=25297044&amp;doi=10.4103/0255-0857.142238&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, Minich J, McDonald D, Mayer T, Knight R, Karouia F, Fox GE, Venkateswaran K. 
2019. 
Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome
7:50. doi: 10.1186/s40168-019-0666-x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40168-019-0666-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6452512/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30955503/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&amp;author=A%20Checinska%20Sielaff&amp;author=C%20Urbaniak&amp;author=GBM%20Mohan&amp;author=VG%20Stepanov&amp;author=Q%20Tran&amp;volume=7&amp;publication_year=2019&amp;pages=50&amp;pmid=30955503&amp;doi=10.1186/s40168-019-0666-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>Singh NK, Bezdan D, Checinska Sielaff A, Wheeler K, Mason CE, Venkateswaran K. 
2018. 
Multi-drug resistant <em>Enterobacter bugandensis</em> species isolated from the International Space Station and comparative genomic analyses with human pathogenic strains. BMC Microbiol
18:175. doi: 10.1186/s12866-018-1325-2.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12866-018-1325-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6251167/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30466389/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&amp;title=Multi-drug%20resistant%20Enterobacter%20bugandensis%20species%20isolated%20from%20the%20International%20Space%20Station%20and%20comparative%20genomic%20analyses%20with%20human%20pathogenic%20strains&amp;author=NK%20Singh&amp;author=D%20Bezdan&amp;author=A%20Checinska%20Sielaff&amp;author=K%20Wheeler&amp;author=CE%20Mason&amp;volume=18&amp;publication_year=2018&amp;pages=175&amp;pmid=30466389&amp;doi=10.1186/s12866-018-1325-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>Andrews S.
2010. 
FastQC: a quality control tool for high throughput sequence data. <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" rel="noopener noreferrer" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>.</cite>
</li>
<li id="B10">
<span class="label">10.</span><cite>Chen S, Zhou Y, Chen Y, Gu J. 
2018. 
fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics
34:i884–i890. doi: 10.1093/bioinformatics/bty560.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/bty560" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6129281/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30423086/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&amp;author=S%20Chen&amp;author=Y%20Zhou&amp;author=Y%20Chen&amp;author=J%20Gu&amp;volume=34&amp;publication_year=2018&amp;pages=i884-i890&amp;pmid=30423086&amp;doi=10.1093/bioinformatics/bty560&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 
2012. 
SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol
19:455–477. doi: 10.1089/cmb.2012.0021.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/cmb.2012.0021" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3342519/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22506599/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&amp;title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&amp;author=A%20Bankevich&amp;author=S%20Nurk&amp;author=D%20Antipov&amp;author=AA%20Gurevich&amp;author=M%20Dvorkin&amp;volume=19&amp;publication_year=2012&amp;pages=455-477&amp;pmid=22506599&amp;doi=10.1089/cmb.2012.0021&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>Gurevich A, Saveliev V, Vyahhi N, Tesler G. 
2013. 
QUAST: quality assessment tool for genome assemblies. Bioinformatics
29:1072–1075. doi: 10.1093/bioinformatics/btt086.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btt086" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3624806/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23422339/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=QUAST:%20quality%20assessment%20tool%20for%20genome%20assemblies&amp;author=A%20Gurevich&amp;author=V%20Saveliev&amp;author=N%20Vyahhi&amp;author=G%20Tesler&amp;volume=29&amp;publication_year=2013&amp;pages=1072-1075&amp;pmid=23422339&amp;doi=10.1093/bioinformatics/btt086&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>Yoon S-H, Ha S-m, Lim J, Kwon S, Chun J. 
2017. 
A large-scale evaluation of algorithms to calculate average nucleotide identity. Antonie Van Leeuwenhoek
110:1281–1286. doi: 10.1007/s10482-017-0844-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s10482-017-0844-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28204908/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Antonie%20Van%20Leeuwenhoek&amp;title=A%20large-scale%20evaluation%20of%20algorithms%20to%20calculate%20average%20nucleotide%20identity&amp;author=S-H%20Yoon&amp;author=S-m%20Ha&amp;author=J%20Lim&amp;author=S%20Kwon&amp;author=J%20Chun&amp;volume=110&amp;publication_year=2017&amp;pages=1281-1286&amp;pmid=28204908&amp;doi=10.1007/s10482-017-0844-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>This WGS project has been deposited at DDBJ/ENA/GenBank, and the accession numbers are given in <a class="usa-link" href="#tab1">Table 1</a> (BioProject accession no. <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA645454" rel="noopener noreferrer" target="_blank">PRJNA645454</a> with 16 strains and <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA649272" rel="noopener noreferrer" target="_blank">PRJNA649272</a> with 3 strains). The versions described in this paper are the first versions.</p></section></section></section></section></article>
</main></body></html>