<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3981873/"/><title>INTER-REGULATION OF THE UNFOLDED PROTEIN RESPONSE AND AUXIN SIGNALING - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Plant J</button></div>. Author manuscript; available in PMC: 2015 Jan 1.</div>
<div>
<em>Published in final edited form as: </em>Plant J. 2013 Dec 3;77(1):97–107. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.12373" rel="noopener noreferrer" target="_blank">10.1111/tpj.12373</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>INTER-REGULATION OF THE UNFOLDED PROTEIN RESPONSE AND AUXIN SIGNALING</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D"><span class="name western">Yani Chen</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yani Chen</span></h3>
<div class="p">
<sup>1</sup>Michigan State University/Department of Energy Plant Research Laboratory and Department of Plant Biology, Michigan State University, East Lansing, MI 48824, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D"><span class="name western">Yani Chen</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aung%20K%22%5BAuthor%5D"><span class="name western">Kyaw Aung</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Kyaw Aung</span></h3>
<div class="p">
<sup>1</sup>Michigan State University/Department of Energy Plant Research Laboratory and Department of Plant Biology, Michigan State University, East Lansing, MI 48824, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aung%20K%22%5BAuthor%5D"><span class="name western">Kyaw Aung</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rol%C4%8D%C3%ADk%20J%22%5BAuthor%5D"><span class="name western">Jakub Rolčík</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jakub Rolčík</span></h3>
<div class="p">
<sup>2</sup>Laboratory of Growth Regulators, Faculty of Science, Palacký University and Institute of Experimental Botany AS CR, Šlechtitelů 11, 78371 Olomouc, Czech Republic</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rol%C4%8D%C3%ADk%20J%22%5BAuthor%5D"><span class="name western">Jakub Rolčík</span></a>
</div>
</div>
<sup>2</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Walicki%20K%22%5BAuthor%5D"><span class="name western">Kathryn Walicki</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kathryn Walicki</span></h3>
<div class="p">
<sup>1</sup>Michigan State University/Department of Energy Plant Research Laboratory and Department of Plant Biology, Michigan State University, East Lansing, MI 48824, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Walicki%20K%22%5BAuthor%5D"><span class="name western">Kathryn Walicki</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Friml%20J%22%5BAuthor%5D"><span class="name western">Jiří Friml</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jiří Friml</span></h3>
<div class="p">
<sup>3</sup>Department of Functional Genomics and Proteomics, Central European Institute of Technology (CEITEC), Masaryk University, 62500 Brno, Czech Republic</div>
<div class="p">
<sup>4</sup>Institute of Science and Technology Austria (IST Austria), 3400 Klosterneuburg, Austria</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Friml%20J%22%5BAuthor%5D"><span class="name western">Jiří Friml</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brandizzi%20F%22%5BAuthor%5D"><span class="name western">Federica Brandizzi</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Federica Brandizzi</span></h3>
<div class="p">
<sup>1</sup>Michigan State University/Department of Energy Plant Research Laboratory and Department of Plant Biology, Michigan State University, East Lansing, MI 48824, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brandizzi%20F%22%5BAuthor%5D"><span class="name western">Federica Brandizzi</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="A1">
<sup>1</sup>Michigan State University/Department of Energy Plant Research Laboratory and Department of Plant Biology, Michigan State University, East Lansing, MI 48824, USA</div>
<div id="A2">
<sup>2</sup>Laboratory of Growth Regulators, Faculty of Science, Palacký University and Institute of Experimental Botany AS CR, Šlechtitelů 11, 78371 Olomouc, Czech Republic</div>
<div id="A3">
<sup>3</sup>Department of Functional Genomics and Proteomics, Central European Institute of Technology (CEITEC), Masaryk University, 62500 Brno, Czech Republic</div>
<div id="A4">
<sup>4</sup>Institute of Science and Technology Austria (IST Austria), 3400 Klosterneuburg, Austria</div>
<div class="author-notes p"><div class="fn" id="FN1">
<sup>✉</sup><p class="display-inline">Correspondence to: Federica Brandizzi, Tel: +1 517 353 7872; Fax: +1 517 353 9168; <span>fb@msu.edu</span></p>
</div></div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Issue date 2014 Jan.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none"><div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div></div>
</div>
<div>PMCID: PMC3981873  NIHMSID: NIHMS538200  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24180465/">24180465</a>
</div>
<div aria-label="Related or updated information about this article" class="ra xbox p" role="complementary"><div>The publisher's version of this article is available at <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.12373" rel="noopener noreferrer" target="_blank">Plant J</a>
</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>SUMMARY</h2>
<p id="P2">The unfolded protein response (UPR) is a signaling network triggered by overload of protein-folding demand in the endoplasmic reticulum (ER), a condition termed ER stress. The UPR is critical for growth and development; nonetheless, connections between the UPR and other cellular regulatory processes remain largely unknown. Here, we identify a link between the UPR and the phytohormone auxin, a master regulator of plant physiology. We show that ER stress triggers down-regulation of auxin sensors and transporters in <em>Arabidopsis thaliana</em>. We also demonstrate that an <em>Arabidopsis</em> mutant of a conserved ER stress sensor IRE1 exhibits defects in the auxin response and levels. These data not only support that the plant IRE1 is required for auxin homeostasis, they also reveal a species-specific feature of IRE1 in multicellular eukaryotes. Furthermore, by establishing that UPR activation is reduced in mutants of ER-localized auxin transporters, including PIN5, we define a long-neglected biological significance of ER-based auxin regulation. We further examine the functional relationship of IRE1 and PIN5 by showing that an <em>ire1 pin5</em> triple mutant enhances defects of UPR activation and auxin homeostasis in <em>ire1</em> or <em>pin5</em>. Our results imply that the plant UPR has evolved a hormone-dependent strategy for coordinating ER function with physiological processes.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> ER stress, unfolded protein response, auxin response, IRE1, PIN5, <em>Arabidopsis thaliana</em></p></section></section><section id="S1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p id="P3">The UPR adjusts the ER protein folding capacity to cope with the dynamic secretory protein demands in cells (<a aria-describedby="R30" class="usa-link" href="#R30">Kozutsumi <em>et al.</em> 1988</a>, <a aria-describedby="R44" class="usa-link" href="#R44">Schroder and Kaufman 2005</a>). When the ER protein folding machinery is competent, stress sensors are restrained in the ER by ER-resident chaperones (<a aria-describedby="R3" class="usa-link" href="#R3">Bertolotti <em>et al.</em> 2000</a>, <a aria-describedby="R28" class="usa-link" href="#R28">Kimata <em>et al.</em> 2003</a>). Accumulation of unfolded proteins in the ER activates ER stress sensors either by causing them to dissociate from protein chaperones or to associate with unfolded proteins (<a aria-describedby="R3" class="usa-link" href="#R3">Bertolotti, <em>et al</em>. 2000</a>, <a aria-describedby="R12" class="usa-link" href="#R12">Credle <em>et al.</em> 2005</a>, <a aria-describedby="R19" class="usa-link" href="#R19">Gardner and Walter 2011</a>, <a aria-describedby="R28" class="usa-link" href="#R28">Kimata <em>et al</em>. 2003</a>). Activated ER stress sensors transmit signals to the nucleus for transcriptional regulation of UPR target genes (<a aria-describedby="R30" class="usa-link" href="#R30">Kozutsumi <em>et al</em>. 1988</a>, <a aria-describedby="R44" class="usa-link" href="#R44">Schroder and Kaufman 2005</a>). If ER stress is not resolved, the UPR triggers the activation of cell death (<a aria-describedby="R32" class="usa-link" href="#R32">Lin <em>et al.</em> 2007</a>). IRE1, the only identified ER stress sensor in yeast, is conserved in multicellular eukaryotes (<a aria-describedby="R10" class="usa-link" href="#R10">Cox <em>et al.</em> 1993</a>, <a aria-describedby="R36" class="usa-link" href="#R36">Mori <em>et al.</em> 1993</a>). Two IRE1 homologues, AtIRE1A and AtIRE1B, have been proven to be functional ER stress sensors in <em>Arabidopsis</em> (<a aria-describedby="R8" class="usa-link" href="#R8">Chen and Brandizzi 2012</a>, <a aria-describedby="R38" class="usa-link" href="#R38">Nagashima <em>et al.</em> 2011</a>). The activation of IRE1 relies on auto-phosphorylation, conformational modification, and oligomerization. Activated IRE1 splices an intron from the mRNA of a UPR-specific bZIP transcription factor (<a aria-describedby="R11" class="usa-link" href="#R11">Cox and Walter 1996</a>). The spliced transcription factor enters the nucleus to control UPR target genes (<a aria-describedby="R11" class="usa-link" href="#R11">Cox and Walter 1996</a>).</p>
<p id="P4">The UPR is critical for numerous fundamental cellular processes (<a aria-describedby="R49" class="usa-link" href="#R49">Wu and Kaufman 2006</a>). IRE1 alpha knockout mice exhibit embryonic lethality (<a aria-describedby="R23" class="usa-link" href="#R23">Iwawaki <em>et al.</em> 2009</a>). Dysregulation of the UPR contributes to the pathology of several significant diseases, including diabetes, neurodegeneration, and cancer (<a aria-describedby="R34" class="usa-link" href="#R34">Marciniak and Ron 2006</a>). In <em>Arabidopsis</em>, mutations of IRE1 lead to a short primary root phenotype (<a aria-describedby="R8" class="usa-link" href="#R8">Chen and Brandizzi 2012</a>). Despite the high significance of the UPR in growth and development in multicellular eukaryotes, the regulatory connections between the UPR and other cellular responses remain unclear.</p>
<p id="P5">Because the hormone auxin has profound roles in most plant developmental processes, nucleus-based auxin signaling and plasma membrane (PM)-based intercellular auxin transport have been intensively studied. Three major classes of auxin signaling regulators exist in the nucleus: TIR1/AFB auxin co-receptors (<a aria-describedby="R14" class="usa-link" href="#R14">Dharmasiri <em>et al.</em> 2005</a>, <a aria-describedby="R20" class="usa-link" href="#R20">Gray <em>et al.</em> 1999</a>, <a aria-describedby="R26" class="usa-link" href="#R26">Kepinski and Leyser 2005</a>), AUX/IAA transcriptional repressors (<a aria-describedby="R21" class="usa-link" href="#R21">Gray <em>et al.</em> 2001</a>), and ARF transcription factors (<a aria-describedby="R46" class="usa-link" href="#R46">Ulmasov <em>et al.</em> 1997</a>). To initiate the auxin response in the nucleus, TIR1/AFBs and auxin coordinately promote degradation of AUX/IAA transcriptional repressors. Consequently, ARFs are released from repression and activate the transcription of auxin responsive genes (<a aria-describedby="R14" class="usa-link" href="#R14">Dharmasiri <em>et al</em>. 2005</a>, <a aria-describedby="R20" class="usa-link" href="#R20">Gray, del Pozo <em>et al</em>. 1999</a>, <a aria-describedby="R26" class="usa-link" href="#R26">Kepinski and Leyser 2005</a>).</p>
<p id="P6">Directional (polar) transport between cells is another crucial regulatory aspect of the auxin response. The auxin efflux carriers of the PIN family are the principal components of the polar auxin transport machinery (<a aria-describedby="R41" class="usa-link" href="#R41">Petrasek <em>et al.</em> 2006</a>, <a aria-describedby="R48" class="usa-link" href="#R48">Wisniewska <em>et al.</em> 2006</a>). Based on protein topology and subcellular localization, PINs can be classified into PM- or ER-localized types (<a aria-describedby="R13" class="usa-link" href="#R13">Dal Bosco <em>et al.</em> 2012</a>, <a aria-describedby="R15" class="usa-link" href="#R15">Ding <em>et al.</em> 2012</a>, <a aria-describedby="R37" class="usa-link" href="#R37">Mravec <em>et al.</em> 2009</a>). While PM-based intercellular auxin transport has been considered the most critical point of regulation in the auxin response, it has recently been revealed that ER-based auxin regulation is also important. A putative auxin receptor, ABP1, and several auxin transporters (PIN5, PIN6, PIN8, and PILSs) have been shown to localize to the ER. The requirement of the ER-localized regulators in the auxin response underscores the existence of ER-based auxin biology (<a aria-describedby="R2" class="usa-link" href="#R2">Barbez <em>et al.</em> 2012</a>, <a aria-describedby="R13" class="usa-link" href="#R13">Dal Bosco <em>et al</em>. 2012</a>, <a aria-describedby="R37" class="usa-link" href="#R37">Mravec <em>et al</em>. 2009</a>). Despite accumulating evidence that the ER is crucial for auxin regulation (<a aria-describedby="R17" class="usa-link" href="#R17">Friml and Jones 2010</a>), the physiological impact of ER-based auxin signaling is largely unknown.</p>
<p id="P7">As the UPR is critical for growth and development, we sought to identify the regulatory connection between the UPR and other cellular regulatory processes. Given the central roles of auxin in numerous aspects of plant physiology, we hypothesized that the UPR regulates auxin signaling for coordinating secretory activities and physiological responses. Through biochemical, molecular biology, and genetic analyses, we demonstrate a connection between the UPR and auxin biology. Specifically, we show that ER stress negatively influences auxin signaling and that the ER-based auxin homoeostasis is important for UPR activation, supporting that the plant UPR alters auxin signaling to cope with ER stress. On the contrary, by establishing that IRE1 is required for the auxin responses, our work reveals that IRE1 has a specific role in hormonal signaling. The regulatory connections between the UPR and auxin biology revealed here hint that plants have evolved an organism-specific strategy to maintain balance between stress adaption and growth regulation.</p></section><section id="S2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="S3"><h3 class="pmc_sec_title">ER stress alters the expression of auxin regulators</h3>
<p id="P8">To examine whether ER stress modulates the transcription of auxin regulators, we monitored the expression of four auxin co-receptors: <em>TIR1</em>, <em>AFB1</em>, <em>AFB2</em>, and <em>AFB3</em> (<em>TIR1</em>/<em>AFBs</em>), under ER stress. The UPR was activated by inhibiting protein <em>N</em>-glycosylation using a classical ER stress inducer termed tunicamycin (Tm). <em>Arabidopsis</em> seedlings were subjected to Tm for various periods of time, as adopted in established protocols (<a aria-describedby="R29" class="usa-link" href="#R29">Koizumi <em>et al.</em> 2001</a>, <a aria-describedby="R35" class="usa-link" href="#R35">Mishiba <em>et al.</em> 2013</a>, <a aria-describedby="R39" class="usa-link" href="#R39">Noh <em>et al.</em> 2002</a>). The transcriptional induction of UPR target genes is a molecular indication of UPR activation. To quantify the UPR activation levels, we monitored the transcription of classical UPR activation indicators, <em>BiP1/2</em> and <em>PDI6</em> over a 4-h time course of Tm treatment using quantitative reverse transcription–polymerase chain reaction (RT-qPCR) analyses (<a aria-describedby="R24" class="usa-link" href="#R24">Kamauchi <em>et al.</em> 2005</a>). <em>BiP1</em>/<em>2</em> is an ER chaperone essential for the UPR and a primary UPR target gene. <em>PDI6</em> encodes protein disulfide isomerase. Similar to BiP proteins, upregulation of PDI6 under ER stress contributes to increasing protein-folding capacity in the ER. RT-qPCR showed that both <em>BiP1/2</em> and <em>PDI6</em> were induced more than 2-fold at 0.5 h of Tm treatment and their levels increased over the time course of treatment (<a class="usa-link" href="#SD1">Figure S1</a>). Interestingly, we found that there was a 20 to 55% percent reduction in the level of <em>TIR1</em>/<em>AFB</em> transcripts 4 h after Tm treatment in wild-type Col-0 seedlings (<a class="usa-link" href="#F1">Figure 1a</a>). These results imply that ER stress negatively influences auxin signaling by repressing <em>TIR1</em>/<em>AFB</em> transcripts. Since <em>TIR1</em>/<em>AFBs</em> activate the auxin response by promoting degradation of the AUX/IAA transcriptional repressors, we sought to determine whether the ER stress-induced repression of <em>TIR1</em>/<em>AFB</em> transcripts resulted in the stabilization of AUX/IAA proteins. To do so, we conducted western blot analyses using transgenic <em>Arabidopsis</em> plants expressing DII-VENUS, a fluorescently tagged auxin-interaction domain (DII) of AUX/IAAs that contains the canonical degron responsible for auxin- and TIR1/AFB-mediated protein degradation (<a aria-describedby="R4" class="usa-link" href="#R4">Brunoud <em>et al.</em> 2012</a>). As the stabilization of AUX/IAAs is a downstream response of TIR1/AFB reduction, we examined DII-VENUS protein levels 6 h after Tm treatment. Indeed, immunoblot analyses showed that the protein levels of DII-VENUS increased in wild-type Col-0 roots under ER stress treatment (<a class="usa-link" href="#F1">Figure 1b</a>). Consistent with the western blot analysis results, a confocal microscopy approach revealed that DII-VENUS fluorescence levels, and therefore AUX/IAA protein levels, were consistently greater in roots challenged by the ER stress inducer than in mock-treated ones (<a class="usa-link" href="#F1">Figure 1c</a>). Together these observations support that ER stress leads to an increase in AUX/IAA levels, which is most likely a consequence of protein stabilization resulting from the down-regulation of <em>TIR1</em>/<em>AFBs</em> (<a class="usa-link" href="#F1">Figure 1a</a>).</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Figure 1. ER stress alters the expression of auxin regulators.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Figure 1" class="graphic" height="263" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0dde/3981873/ce7ee3d4abcd/nihms538200f1.jpg" width="800"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(a) RT-qPCR analyses of <em>TIR1</em>, <em>AFB1</em>, <em>AFB2</em>, and <em>AFB3</em> expression in ten-day-old Col-0 <em>Arabidopsis</em> seedlings after treatment with 5 μg/ml Tm for 0.5, 1, or 4 h. Error bars represent standard error of the mean (SEM) from three independent biological replicates. <em>P</em>-values were calculated by Student’s two-tailed t test against expression levels at 4 h relative to 0 h: <em>TIR1</em> (<em>P</em> = 0.00036), <em>AFB1</em> (<em>P</em> = 0.00041), <em>AFB2</em> (<em>P</em> = 0.00048), <em>AFB3</em> (<em>P</em> = 0.00032).</p>
<p>(b) The levels of DII-VENUS fusion proteins increase upon exposure to ER stress. Ten-day-old DII-VENUS transgenic plants were treated with 5 μg/ml Tm or DMSO for 6 h. Proteins were extracted from root tissues and the fusion proteins were detected by immunoblot analysis using anti-GFP serum (upper panel). Coomassie blue staining gel used as loading control (lower panel).</p>
<p>(c) Ten-day-old transgenic plants expressing DII-VENUS were treated with 5 μg/ml Tm or DMSO for 6 h. Primary root tips were subjected to confocal microscopy analyses. Scale bars = 50 μm.</p>
<p>(d) <em>PIN</em> mRNA levels decrease upon exposure to ER stress. RT-qPCR analyses of <em>PIN</em> family transcripts in ten-day-old wild-type Col-0 seedlings during treatment with 5 μg/ml Tm for 0.5, 1, or 4 h. Error bars represent SEM from three independent biological replicates. <em>P</em>-values were calculated against expression levels at 4 h relative to 0 h: <em>PIN1</em> (<em>P</em> = 0.00221), <em>PIN2</em> (<em>P</em> = 0.00316), <em>PIN3</em> (<em>P</em> = 5.4E-05), <em>PIN4</em> (<em>P</em> = 4.9E-05), <em>PIN5</em> (<em>P</em> = 6.4E-05), <em>PIN6</em> (<em>P</em> = 0.00012), <em>PIN7</em> (<em>P</em> = 0.00353). The transcriptional level of ETR1, an ER-associated ethylene receptor, was unchanged after treatment with Tm for 0.5, 1, or 4 h.</p></figcaption></figure><p id="P9">Next, we investigated whether ER stress could control the transcription of auxin transporters. Using RT-qPCR analyses, we detected a 30–80% decrease in the mRNA levels of <em>PIN1</em>, <em>PIN2</em>, <em>PIN3</em>, <em>PIN4</em>, <em>PIN5</em>, <em>PIN6</em>, and <em>PIN7</em> in wild-type Col-0 seedlings during ER stress treatment (<a class="usa-link" href="#F1">Figure 1d</a>). In contrast, the transcriptional levels of an ER-associated ethylene receptor (ETR1), two ER-localized cytokinin receptors (AHK2 and AHK3), three nuclear protein (RAN2, ABH1, and FIB1), and two secretory proteins (VSR1 and SCAMP3) (<a aria-describedby="R1" class="usa-link" href="#R1">Ahmed <em>et al.</em> 1997</a>, <a aria-describedby="R6" class="usa-link" href="#R6">Chang <em>et al.</em> 1993</a>, <a aria-describedby="R25" class="usa-link" href="#R25">Kanneganti <em>et al.</em> 2007</a>, <a aria-describedby="R27" class="usa-link" href="#R27">Kierzkowski <em>et al.</em> 2009</a>, <a aria-describedby="R31" class="usa-link" href="#R31">Law <em>et al.</em> 2012</a>, <a aria-describedby="R33" class="usa-link" href="#R33">Ma <em>et al.</em> 2007</a>, <a aria-describedby="R50" class="usa-link" href="#R50">Wulfetange <em>et al.</em> 2011</a>) remained unchanged in ER stress conditions (<a class="usa-link" href="#SD1">Figure S2</a>). Thus, we conclude that the Tm-induced decrease in the abundance of <em>TIR1</em>/<em>AFB</em> and <em>PIN</em> transcripts is a specific cellular response. When ER stress was triggered by reduction of disulfide bond formation using dithiotreitol (DTT) treatment, similar down-regulation of <em>TIR1/AFB</em> and <em>PIN</em> transcripts was observed (<a class="usa-link" href="#SD1">Figure S3</a>). Overall, these results show that ER stress specifically modulates the auxin response by repressing the transcription of auxin co-receptors and transporters.</p>
<p id="P10">Next, we examined whether either IRE1 or TIR1/AFBs is essential for ER stress-induced down-regulation of auxin regulators. To do so, we performed the same ER stress treatment coupled with RT-qPCR analyses in an <em>atire1a atire1b</em> double mutant and a <em>tir1 afb1 afb2 afb3</em> mutant (<a aria-describedby="R8" class="usa-link" href="#R8">Chen and Brandizzi 2012</a>, <a aria-describedby="R14" class="usa-link" href="#R14">Dharmasiri <em>et al.</em> 2005</a>). In <em>atire1a atire1b</em>, both <em>TIR1</em>/<em>AFB</em> and <em>PIN</em> transcripts were still reduced under ER stress conditions (<a class="usa-link" href="#F2">Figure 2a, b</a>), similar to the decreased <em>TIR1</em>/<em>AFB</em> and <em>PIN</em> transcripts pattern seen in wild-type Col-0 (<a class="usa-link" href="#F1">Figure 1a</a>). The <em>PIN</em> transcripts also decreased under ER stress conditions in <em>tir1 afb1 afb2 afb3</em> mutant backgrounds (<a class="usa-link" href="#F2">Figure 2b</a>). However, with the exception of <em>PIN7</em>, in <em>atire1a atire1b</em> the <em>PIN</em> and <em>TIR1</em>/<em>AFB</em> transcription levels were further slightly reduced compared to wild-type Col-0 (<a class="usa-link" href="#F2">Figure 2c</a> and <a class="usa-link" href="#SD1">S4</a>). In contrast, the reduction of <em>PIN1</em>, <em>PIN2</em>, and <em>PIN4</em> transcript levels was larger in the <em>tir1 afb1 afb2 afb3</em> mutant compared to wild-type Col-0 (<a class="usa-link" href="#F2">Figure 2c</a>). These data indicate that down-regulation of <em>PIN</em> transcripts on ER stress is partially and slightly affected by mutations of either IRE1 or TIR1/AFBs. These results further suggest that the <em>IRE1</em> and <em>TIR1</em>/<em>AFBs</em> play unessential but fine-tuning roles in ER stress-mediated repression of auxin regulators.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Figure 2. IRE1 and TIR1/AFBs play fine-tuning roles in ER stress-induced down-regulation of auxin regulators.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3981873_nihms538200f2.jpg" target="_blank"><img alt="Figure 2" class="graphic zoom-in" height="382" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0dde/3981873/8e622730ac6d/nihms538200f2.jpg" width="600"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(a) RT-qPCR analyses of <em>TIR1</em>, <em>AFB1</em>, <em>AFB2</em>, and <em>AFB3</em> expression in ten-day-old <em>atire1a atire1b</em> (<em>ire1</em>) <em>Arabidopsis</em> seedlings after treatment with 5 μg/ml Tm for 0.5, 1, or 4 h.</p>
<p>(b) <em>PIN</em> mRNA levels decrease upon exposure to ER stress in ten-day-old <em>ire1</em> or <em>tir1 afb1 afb2 afb3</em> (<em>tir1 afb</em>). RT-qPCR analyses of <em>PIN</em> family transcripts in ten-day-old <em>ire1</em> or <em>tir1 afb Arabidopsis</em> seedlings after treatment with 5 μg/ml Tm for 0.5, 1, or 4 h.</p>
<p>(c) Transcriptional repression of <em>PIN</em>s after treatment with 5 μg/ml Tm for 4 h in ten-day-old Col-0, <em>ire1</em>, or <em>tir1 afb Arabidopsis</em> seedlings.</p>
<p>Error bars represent SEM from three independent biological replicates.</p></figcaption></figure></section><section id="S4"><h3 class="pmc_sec_title">IRE1 is required for auxin response and homeostasis</h3>
<p id="P11">While the UPR is necessary for growth and development, the manner by which the UPR influences other cellular regulatory processes is largely unknown. Mammalian IRE1 controls multiple physiological responses under normal growth conditions. Our findings that auxin signaling is altered under ER stress hint that the UPR participates in the auxin response in plants. Although IRE1 is unessential for ER stress-triggered down-regulation of auxin regulators, we aimed to determine whether IRE1-dependent UPR is required for the auxin response without chemical induction of ER stress. Thus, we performed root inhibition assays to test the sensitivity of exogenous auxin application in <em>atire1a atire1b</em>. To do so, <em>atire1a atire1b</em> and wild-type Col-0 were germinated on medium containing a synthetic auxin analog, 1-naphthaleneacetic-acid (NAA), a naturally occurring auxin, indole-3-acetic acid (IAA), or an auxin transport inhibitor, 1-N-Naphthylphthalamic-acid (NPA). Interestingly, we found that <em>atire1a atire1b</em> was significantly less sensitive to exogenously applied NAA, IAA, or NPA than wild-type Col-0 (<a class="usa-link" href="#F3">Figure 3a-c</a>). The findings that <em>atire1a atire1b</em> and wild-type Col-0 displayed comparable root-inhibition responses to three other plant hormone, jasmonic acid (JA), abscisic acid (ABA) and ethylene, indicate that the plant <em>IRE1</em> has a role specifically in the auxin response, as opposed to general hormones responses or growth regulation (<a class="usa-link" href="#SD1">Figure S5</a>). To further confirm that AtIRE1A/AtIRE1B is involved in the auxin response, we investigated the transcriptional activation of auxin-responsive genes in <em>atire1a atire1b</em> upon external application of auxin. RT-qPCR analyses showed that the transcriptional induction of five auxin-responsive genes, <em>IAA3</em>, <em>IAA5</em>, <em>IAA19</em>, <em>IAA20</em>, and <em>GH3.6</em>, was compromised after 2 and 4 h of NAA treatment in <em>atire1a atire1b</em> relative to wild-type Col-0 (<a class="usa-link" href="#F3">Figure 3d</a> and <a class="usa-link" href="#SD1">Figure S6a</a>). The lower induction of auxin-responsive genes was repeatedly observed after both 1 and 2 h of IAA treatment (<a class="usa-link" href="#SD1">Figure S6b</a>). These data further indicate that <em>atire1a atire1b</em> exhibits an impaired response to exogenously applied auxin. The transcription of <em>BiP1/2</em> or <em>PDI6</em> was not significantly altered under IAA or NPA treatment in wild-type Col-0, suggesting that IAA or NPA treatment does not trigger ER stress like Tm treatment (<a class="usa-link" href="#SD1">Figure S7</a>). Furthermore, we found that there was a 30% reduction in the free auxin level in roots of ten-day-old <em>atire1a atire1b</em> plants compared to wild-type Col-0 (<a class="usa-link" href="#F3">Figure 3e</a>). While IRE1 is required to maintain the free auxin level without ER stress treatment, the free auxin level remained unaffected within 4 h after Tm treatment (<a class="usa-link" href="#SD1">Figure S8</a>). All together, these data support that plant <em>IRE1</em> is required for auxin response and homeostasis.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Figure 3. <em>atire1a atire1b</em> (<em>ire1</em>) exhibits compromised auxin responses.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3981873_nihms538200f3.jpg" target="_blank"><img alt="Figure 3" class="graphic zoom-in" height="460" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0dde/3981873/5cd123e3fbc0/nihms538200f3.jpg" width="795"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(a–c) In <em>ire1</em>, root growth is largely resistant to treatments with auxin (NAA and IAA) or an auxin transport inhibitor (NPA). Relative primary root length of ten-day-old Col-0 and <em>ire1 Arabidopsis</em> seedlings grown in the presence 50, 100, 200 nM NAA (a), or 1, 2, 3, 4, 5 μM IAA (b), or 50, 100 nM NPA (c) compared to those grown in the absence of the chemicals. Error bars represent standard error of the mean (SEM), n &gt; 30. Scale bars = 1 cm. <em>P</em>-values are relative to Col-0: 100, 200 nM NAA (<em>P</em> &lt; 0.00078), 1, 2, 3, 4, 5 μM IAA (<em>P</em> &lt; 0.00050), 50, 100 nM NPA (<em>P</em> &lt; 0.00344).</p>
<p>(d) RT-qPCR analyses of <em>IAA5</em> and <em>GH3.6</em> expression in ten-day-old Col-0 and <em>ire1 Arabidopsis</em> seedlings after a 2- or 4-h treatment with 10 μM NAA. Error bars represent SEM from three independent biological replicates. <em>P</em>-values are relative to Col-0: <em>IAA5</em> (<em>P</em> &lt; 0.00016), <em>GH3.6</em> (<em>P</em> &lt; 0.00391).</p>
<p>(e) Free IAA concentration in ten-day-old Col-0 and <em>ire1</em> roots. Error bars represent SEM from three independent biological replicates. <em>P</em>-value is relative to Col-0: <em>P</em> = 4.9E-05.</p></figcaption></figure></section><section id="S5"><h3 class="pmc_sec_title">ER-localized auxin regulators are involved in UPR activation</h3>
<p id="P12">Our observations that auxin signaling is regulated by ER stress led us to test whether auxin homeostasis influences UPR activation. To this end, we examined whether mutations in auxin signaling, polar transport, or biosynthesis affected the induction of UPR target genes. Intriguingly, we found that a mutation in <em>PIN5</em> or <em>PIN6,</em> two ER-localized auxin transporters, compromised UPR activation under ER stress. Compared to wild type, <em>pin5-5</em> (<a aria-describedby="R37" class="usa-link" href="#R37">Mravec <em>et al</em>. 2009</a>) and <em>pin6-4</em> (<a aria-describedby="R5" class="usa-link" href="#R5">Cazzonelli <em>et al.</em> 2013</a>) exhibited a 30–40% reduction in the level of <em>BiP1/2</em> and <em>PDI6</em> transcripts during ER stress treatment (<a class="usa-link" href="#F4">Figure 4a</a> and <a class="usa-link" href="#SD1">S9</a>). These data imply that ER-based auxin homeostasis contributes to UPR regulation in plants. This hypothesis was supported by the observation that mutants of other types of ER-localized putative auxin transporters, <em>pils2-2</em>, <em>pils5-2</em>, and <em>pils2-2 pils5-2</em> (<a aria-describedby="R2" class="usa-link" href="#R2">Barbez <em>et al.</em> 2012</a>) exhibited similar defects in UPR activation (<a class="usa-link" href="#F4">Figure 4a</a> and <a class="usa-link" href="#SD1">S9</a>). We found that the expression levels of <em>BiP1/2</em> and <em>PDI6</em> were comparable among wild-type Col-0 and the auxin mutants without ER stress treatment (<a class="usa-link" href="#SD1">Figure S10</a>), supporting that the ER-localized auxin transporters are involved in ER stress-triggered UPR activation. In contrast, we found no significant differences in the induction of UPR target genes in mutants defective in either PM-localized auxin exporters, <em>pin1-1</em> (<em>pin1</em>), <em>eir1-1</em> (<em>pin2</em>), <em>pin3-4</em> (<em>pin3</em>), <em>pin4-3</em> (<em>pin4</em>)<em>, pin7-2</em> (<em>pin7</em>)<em>, pin3-5 pin4-3 pin7-1</em> (<em>pin3 pin4 pin7</em>)<em>,</em> an overexpressor of <em>PIN1</em> (<em>OxPIN1</em>), or the auxin importer <em>aux1-22</em> (<em>aux1</em>) compared to wild-type Col-0 under the same ER stress treatment conditions (<a aria-describedby="R40" class="usa-link" href="#R40">Okada <em>et al.</em> 1991</a>, <a aria-describedby="R42" class="usa-link" href="#R42">Roman <em>et al.</em>, 1995</a>, <a aria-describedby="R52" class="usa-link" href="#R52">Zádníková <em>et al.</em>, 2010</a>, <a aria-describedby="R16" class="usa-link" href="#R16">Friml <em>et al.</em> 2002</a>, <a aria-describedby="R18" class="usa-link" href="#R18">Friml <em>et al.</em> 2003</a>, <a aria-describedby="R43" class="usa-link" href="#R43">Sauer <em>et al.</em> 2006</a>, <a aria-describedby="R45" class="usa-link" href="#R45">Swarup <em>et al.</em> 2004</a>) (<a class="usa-link" href="#F4">Figure 4b</a>). These results show that ER-based intracellular auxin transport, but not intercellular auxin transport, is required for the optimal UPR activation in plants.</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Figure 4. Mutants impaired in intracellular auxin transport display a defective UPR phenotype.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3981873_nihms538200f4.jpg" target="_blank"><img alt="Figure 4" class="graphic zoom-in" height="688" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0dde/3981873/9262d0681034/nihms538200f4.jpg" width="515"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(a) RT-qPCR analyses of <em>BiP1/2</em> and <em>PDI6</em> in ten-day-old <em>pin5-5</em> (<em>pin5</em>), <em>pin6-4</em> (<em>pin6</em>), <em>pils2-2</em> (<em>pils2</em>), <em>pils5-2</em> (<em>pils5</em>), <em>pils2-2 pils5-2</em> (<em>pils2 pils5</em>), <em>abp1-5</em> (<em>abp1)</em>, <em>YUC</em>, and <em>tir1 afb1 afb2 afb3</em> (<em>tir1 afb</em>) relative to wild-type Col-0 <em>Arabidopsis</em> seedlings after a 1-h treatment with 5 μg/ml Tm. Error bars represent standard error of the mean (SEM) from three independent biological replicates. <em>P</em>-values are relative to Col-0: <em>pin5</em> (<em>P</em> = 0.00029), <em>pin6</em> (<em>P</em> = 0.00095), <em>pils2</em> (<em>P</em> = 0.00093), <em>pils5</em> (<em>P</em> = 0.00067), <em>pils2 pils5</em> (<em>P</em> = 0.00089), <em>abp1</em> (<em>P</em> = 0.00215), <em>YUC</em> (<em>P</em> = 0.00014), <em>tir1 afb</em> (<em>P</em> = 0.00026).</p>
<p>(b) RT-qPCR analyses of <em>BiP1/2</em> and <em>PDI6</em> in ten-day-old <em>pin1-1</em> (<em>pin1</em>), <em>eir1-1</em> (<em>pin2</em>), <em>pin3-4</em> (<em>pin3</em>), <em>pin4-3</em> (<em>pin4</em>)<em>, pin7-2</em> (<em>pin7</em>)<em>, pin3-4 pin4-3 pin7-2</em> (<em>pin3 pin4 pin7</em>)<em>, OxPIN1</em>, and <em>aux1-22</em> (<em>aux1</em>) relative to wild-type Col-0 <em>Arabidopsis</em> seedlings after a 1-h treatment with 5 μg/ml Tm. Error bars represent SEM from three independent biological replicates.</p></figcaption></figure><p id="P13">We next investigated whether the ER-localized putative auxin receptor ABP1 and the auxin biosynthesis enzyme YUC were also involved in the UPR activation. Similar to the auxin mutants defective in ER-based transport, the <em>abp1</em>-<em>5</em> and <em>YUC</em> (<a aria-describedby="R51" class="usa-link" href="#R51">Xu <em>et al.</em> 2010</a>, <a aria-describedby="R53" class="usa-link" href="#R53">Zhao <em>et al.</em> 2001</a>) mutants also exhibited reduced levels in the activation of UPR target genes under ER stress (<a class="usa-link" href="#F4">Figure 4a</a> and <a class="usa-link" href="#SD1">S9</a>). Conversely, the transcription levels of UPR target genes were higher in the <em>tir1 afb1 afb2 afb3</em> auxin co-receptor mutant than in wild-type Col-0 (<a class="usa-link" href="#F3">Figure 3a</a> and <a class="usa-link" href="#SD1">S9</a>), suggesting that TIR1/AFBs plays a negative role in UPR target gene induction. Altogether, these data highlight that the ER-based regulation of auxin homeostasis may operate as a molecular link between the UPR and other cellular processes.</p></section><section id="S6"><h3 class="pmc_sec_title">
<em>pin5</em> enhances the <em>ire1</em> phenotype in auxin responses and UPR activation</h3>
<p id="P14">To investigate a functional relationship of IRE1 and PIN5 in the UPR and auxin response<em>,</em> we generated an <em>atire1a atire1b pin5</em> triple mutant and performed phenotypic analyses. Consistent with previous reports, <em>atire1a atire1b</em> and <em>pin5-5</em> displayed a short primary root phenotype (<a aria-describedby="R8" class="usa-link" href="#R8">Chen and Brandizzi 2012</a>, <a aria-describedby="R37" class="usa-link" href="#R37">Mravec <em>et al.</em> 2009</a>). We found that the roots of the <em>atire1a atire1b pin5</em> triple mutant were significantly shorter than those of <em>atire1a atire1b</em> or <em>pin5-5</em> (<a class="usa-link" href="#F5">Figure 5a</a>). However, <em>atire1a atire1b</em>, <em>pin5-5</em>, and <em>atire1a atire1b pin5</em> showed comparable lateral root density and hypocotyl length to wild-type Col-0 (<a class="usa-link" href="#SD1">Figure S11</a>), suggesting that the IRE1 and PIN5 have a role specifically in regulation of primary root elongation. In addition, <em>atire1a atire1b pin5</em> also displayed lower free auxin levels compared to <em>atire1a atire1b</em> or <em>pin5-5</em> (<a class="usa-link" href="#F5">Fig 5b</a>). Specifically, compared to wild-type Col-0, the roots of <em>pin5-5</em> and <em>atire1a atire1b</em> exhibited a 15 and 30% reduction in free auxin levels, respectively. Nonetheless, the roots of <em>atire1a atire1b pin5</em> displayed a 45% reduction in free auxin level (<a class="usa-link" href="#F5">Figure 5b</a>). Finally, in agreement with previous findings (<a aria-describedby="R37" class="usa-link" href="#R37">Mravec <em>et al.</em> 2009</a>), root inhibition assays showed that <em>pin5-5</em> was less sensitive than wild-type Col-0 to low concentrations of IAA but displayed a normal response to NAA, NPA, or high concentrations of IAA (<a aria-describedby="R37" class="usa-link" href="#R37">Mravec <em>et al.</em> 2009</a>). Intriguingly, the <em>atire1a atire1b pin5</em> mutant was significantly less sensitive than <em>atire1a atire1b</em> to all three treatments (<a class="usa-link" href="#F5">Figure 5c-e</a>). A comparable root-inhibition response to JA, ABA, and ethylene in <em>atire1a atire1b pin5</em> indicated that the genetic interaction between IRE1 and PIN5 is specific to the auxin response (<a class="usa-link" href="#SD1">Figure S5</a>).</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Figure 5. <em>pin5</em> enhances the auxin and ER stress response phenotype in <em>atire1a atire1b</em> (<em>ire1</em>).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Figure 5" class="graphic" height="387" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0dde/3981873/c4add7bf8874/nihms538200f5.jpg" width="629"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(a) <em>pin5-5</em> (<em>pin5</em>) enhances the short root phenotype of <em>ire1</em>. Relative primary root length of <em>pin5</em>, <em>ire1</em>, and <em>ire1 pin5</em> compared to Col-0 under unstressed conditions. Error bars represent standard error of the mean (SEM), n &gt; 30. <em>P</em>-value is <em>ire1 pin5</em> relative to <em>ire1</em>: *<em>P</em> = 0.00226.</p>
<p>(b) Free IAA measurement in the roots of ten-day-old Col-0<em>, pin5</em>, <em>ire1</em>, and <em>ire1 pin5</em> seedlings. Error bars represent SEM from three independent biological replicates. <em>P</em>-value is <em>ire1 pin5</em> relative to <em>ire1</em>: *<em>P</em> = 0.00182.</p>
<p>(c–e) Relative primary root length of ten-day-old Col-0<em>, pin5</em>, <em>ire1</em>, and <em>ire1 pin5 Arabidopsis</em> seedlings grown in the presence 50, 100, 200 nM NAA (c), or 1, 2, 3, 4, 5 μM IAA (d), or 50, 100 nM NPA (e) compared to those grown in the absence of the chemicals. Error bars represent SEM, n &gt; 30. <em>P</em>-values are <em>ire1 pin5</em> relative to <em>ire1</em>: 50, 100 or 200 nM NAA (<em>P</em> &lt; 0.00032), 1, 2, 3, 4, or 5 μM IAA (<em>P</em> &lt; 0.00075), 50 or 100 nM NPA (<em>P</em> &lt; 0.00149). Scale bars = 1 cm.</p>
<p>(f) <em>pin5</em> enhances the UPR defects in <em>ire1</em> under ER stress. RT-qPCR analyses of <em>BiP1/2</em> and <em>PDI6</em> in ten-day-old Col-0<em>, pin5</em>, <em>ire1</em>, and <em>ire1 pin5</em> relative to DMSO mock control after a 1-h treatment with 5 μg/ml Tm. Error bars represent SEM from three independent biological replicates. <em>P</em>-value is <em>ire1 pin5</em> relative to <em>ire1</em>: *<em>P</em> = 0.01856.</p></figcaption></figure><p id="P15">We next tested the functional relationship of IRE1 and PIN5 under ER stress. RT-qPCR revealed that the induction of <em>BiP1/2</em> and <em>PDI6</em> was also reduced in <em>atire1a atire1b pin5</em> compared to <em>atire1a atire1b</em> or <em>pin5-5</em>, supporting that PIN5 participates in the UPR activation in a manner not entirely dependent on <em>IRE1</em>. Altogether, our results imply that regulation of ER-based auxin homeostasis is part of ER stress adaptive mechanisms that plants have evolved to parallel the classical UPR signaling pathways.</p></section></section><section id="S7"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p id="P16">Our findings uncover an unpredicted but critical regulatory relationship between two fundamental signaling pathways in plants, the UPR and auxin response. Studies of the mammalian UPR indicate that distinct UPR signaling pathways mediate specific physiological processes (<a aria-describedby="R49" class="usa-link" href="#R49">Wu and Kaufman 2006</a>). While the <em>IRE1</em>-dependent mRNA splicing event is the most evolutionarily conserved UPR pathway in eukaryotes, <em>IRE1</em> has also evolved specific functions in multicellular organisms to associate the UPR with more complex physiological processes (<a aria-describedby="R22" class="usa-link" href="#R22">Hetz <em>et al.</em> 2006</a>, <a aria-describedby="R47" class="usa-link" href="#R47">Urano <em>et al.</em> 2000</a>). Nevertheless, our understanding of the connection between the UPR and other cellular processes is still in its infancy. Here, we have defined a plant-specific regulatory role for <em>IRE1</em> in the auxin response. Although we cannot exclude the possibility that IRE1 regulates auxin response independently from the classical UPR, it is plausible that IRE1-dependent UPR signaling is involved in auxin transport. The auxin transport is one of the most crucial regulatory mechanisms in the auxin biology. As most regulatory components of the auxin transport system are secretory proteins, we speculate that the <em>IRE1</em>-dependent UPR signaling maintains a robust and efficient membrane trafficking system for the supply of functional auxin regulators. The identification of auxin regulators directly controlled by IRE1 will elucidate how IRE1 modulates specific aspects of auxin biology to coordinate the secretory pathway with physiological responses. Together with the involvement of UPR-specific membrane tethered transcription factors in brassinosteroid signaling (<a aria-describedby="R7" class="usa-link" href="#R7">Che <em>et al.</em> 2010</a>), our results support the significance of the plant UPR in hormone signaling.</p>
<p id="P17">IRE1 regulates the UPR through various mechanisms including unconventional splicing, RNA decay, and protein-protein interaction. It has recently been reported that similar to its mammalian counterpart, plant IRE1 controls gene expression through RNA decay in addition of splicing bZIP60 transcription factor (<a aria-describedby="R35" class="usa-link" href="#R35">Mishiba <em>et al.</em> 2013</a>). Mammalian IRE1 operates RNA decay to trigger diverse UPR signaling pathways. It would be interesting to test whether plant IRE1 also relies on its RNA decay function for the auxin response. If IRE1-dependent RNA decay contributes at least partially to the regulation of auxin signaling on ER stress, it would represent a specific regulatory event of the UPR as opposed to random RNA decay under stress. Notably, we have established that IRE1 is required for the optimal auxin response under exogenously applied auxin (<a class="usa-link" href="#F3">Figure 3</a> and <a class="usa-link" href="#F6">6a</a>), but plays only partial role in ER stress-induced down-regulation of auxin regulators (<a class="usa-link" href="#F2">Figure 2</a>). These findings support that distinct mechanisms regulate auxin signaling under various conditions to achieve context-specific auxin responses.</p>
<figure class="fig xbox font-sm" id="F6"><h3 class="obj_head">Figure 6. Working model.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3981873_nihms538200f6.jpg" target="_blank"><img alt="Figure 6" class="graphic zoom-in" height="770" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0dde/3981873/c6cb261ded25/nihms538200f6.jpg" width="621"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>(a) IRE1 is required for the auxin responses upon external auxin application. IRE1, ER- and PM-localized PINs are involved in the maintenance of auxin homeostasis without chemical induction of ER stress.</p>
<p>(b) ER stress triggers down-regulation of auxin receptors TIR1/AFBs, ER- and PM-localized PINs. IRE1 and ER-localized PINs are required for the optimal induction of UPR target genes.</p></figcaption></figure><p id="P18">We have established that only ER-localized auxin transporters, but not PM-localized auxin exporters or importers, are required for the optimal UPR activation (<a class="usa-link" href="#F6">Figure 6b</a>). Studies of ER-localized auxin regulators suggest that a distinct auxin signaling pathway exists in the ER (<a aria-describedby="R17" class="usa-link" href="#R17">Friml and Jones 2010</a>). Accordingly, we propose that ER-based auxin signaling actively transports free auxin through the ER membranes to modulate the signaling response in the nucleus. More specifically, plant cells can transmit signals between sub-cellular compartments by adjusting the free auxin level in the ER, cytosol, and nucleus. We thus propose a previously uncharacterized cellular function for auxin as a signaling molecule that connects subcellular compartments and maintains cellular homeostasis in plants. It has long been believed that intercellular polar auxin transport is the key regulatory component of the auxin response; however, the biological significance of intracellular auxin transport has been overlooked. Our findings support a specific cellular function of ER-based intracellular auxin distribution in the UPR activation, and thus emphasize the importance of ER auxin biology in plant physiology.</p>
<p id="P19">We have shown that ER-localized auxin transporters have a role in the UPR activation. A plausible hypothesis to explain this is that the auxin levels in the ER lumen contribute to UPR activation on ER stress. Namely, ER-localized auxin transporters or their associated proteins might sense ER stress and rapidly adjust auxin levels in the ER lumen. The consequent fluctuation of auxin levels in the ER could in turn affect the magnitude of UPR activation. Nevertheless, because a mutation of PIN5 enhances the <em>atire1a atire1b</em> mutant phenotype in the UPR activation (<a class="usa-link" href="#F5">Figure 5f</a>), PIN5-dependent regulation of auxin levels under ER stress does not completely rely on IRE1. Whether ER-localized auxin transporters can directly monitor ER stress or indirectly sense ER stress-related cellular homeostasis is yet to be established; however, the findings presented here support that ER-localized transporters play a role in the UPR activation. Once a reliable system to monitor auxin levels in the ER lumen is developed, it will be interesting to experimentally confirm that ER-localized transporters mediate auxin transport between ER lumen and cytosol on ER stress.</p>
<p id="P20">PIN5 has been proposed to play a unique role in the auxin response since its transcriptional regulation and regulatory mechanisms appear to be different from PM-localized PINs. It was reported that the transcription of PIN5 is decreased under exogenous application of auxin although PIN5 is required for the auxin response (<a aria-describedby="R37" class="usa-link" href="#R37">Mravec <em>et al.</em> 2009</a>). Likewise, our study also showed that ER stress induces a decrease in the transcription of PIN5 while PIN5 is required for the optimal induction of UPR activation. As the PIN5 protein levels have not been monitored under auxin or ER stress treatment, one possibility is that the down-regulation of PIN5 transcript represents a feedback regulatory mechanism. Namely, the cellular availability or the activity of PIN5 may be increased in response to ER stress (e.g. by protein stabilization events or post-translational modifications). This in turn may cause reduction of PIN5 transcriptional levels to safeguard cellular auxin homeostasis. Another possibility is that ER stress represses general auxin responses including inter- and intra-cellular auxin transport to optimize cellular responses to cope with stress. Thus, both PM- and ER-localized transporter are down-regulated under ER stress; however, a basal level of ER-localized transporters may be still required for optimal induction of UPR target gene as they might be involved in stress signal transmission through transport the auxin between subcellular compartments. Thus, mutants of ER-localized auxin regulators would display a compromised UPR activation. Further experimental evidences are needed to verify the possibilities. Nonetheless, our data support that regulation of PIN5 transcripts is a mechanism to maintain PIN5-related cellular homeostasis. Also, <em>pin5-5</em> was shown to have a higher free auxin levels (<a aria-describedby="R37" class="usa-link" href="#R37">Mravec <em>et al.</em> 2009</a>) but we found that <em>pin5-5</em> displayed lower free auxin level. This is possibly because unlike Mravec <em>et al</em>, which used intact seedlings, we used only root tissues in the free auxin level assay. Future comprehensive quantification analyses of free auxin levels in various tissues will likely reveal whether PIN5 regulates auxin distribution among tissues.</p>
<p id="P21">The molecular mechanisms underlying transcriptional down-regulation of auxin receptors and transporters on ER stress are still unknown. Whether UPR regulators can directly control transcription of auxin receptors and transporters or ER stress-dependent cellular responses mediate auxin homeostasis in a manner independent of classical UPR regulation awaits further validation.</p>
<p id="P22">In contrast to animals, plants, as sessile organisms, have an extraordinary plasticity in post-embryonic development, responding to both internal and external cues. Nonetheless, our understanding of how plants integrate developmental and environmental signals to balance growth and adaptive regulation is limited. The inter-regulation between the UPR and auxin response demonstrated in this study opens a new area of investigation in plant physiology. Given the essential roles of the UPR in multiple stresses adaptation, the integrated action of the UPR and auxin response highlights a plant-specific strategy that evolved to maintain the crucial balance between stress response and growth regulation for ultimate fitness.</p></section><section id="S8"><h2 class="pmc_sec_title">EXPERIMENTAL PROCEDURES</h2>
<section id="S9"><h3 class="pmc_sec_title">Plant material and growth conditions</h3>
<p id="P23"><em>Arabidopsis thaliana</em> ecotype Columbia (Col-0) plants were used. Surface-sterilized seeds were plated directly onto petri dishes containing half-strength Linsmaier and Skoog (LS) medium, 1.5% w/v sucrose, and 0.4% Phytagel (Sigma). For normal growth conditions, plants were grown at 21°C under a 16-h light/8-h dark cycle.</p></section><section id="S10"><h3 class="pmc_sec_title">Tm treatment</h3>
<p id="P24">Seeds were germinated and grown on half-strength LS medium for 10 days, and then transferred to half-strength LS medium containing 5 μg/ml Tm (Sigma) for the indicated periods of time.</p></section><section id="S11"><h3 class="pmc_sec_title">RNA extraction and quantitative RT-PCR analysis</h3>
<p id="P25">Total RNA was extracted from whole seedlings using an RNeasy Plant Mini Kit (Qiagen) and treated with DNase I (Qiagen). All samples within an experiment were reverse-transcribed simultaneously using SuperScript® VILO<sup>™</sup> Master Mix, (Invitrogen). A no-RT reaction, in which RNA was subjected to the same conditions of cDNA synthesis but without reverse transcriptase, was included as a negative control in all real-time quantitative PCR (RT-qPCR) assays. RT-qPCR with SYBR Green detection using a relative standard curve method was performed in triplicate using the Applied Biosystem 7500 Fast Real-Time 7500 PCR system. Data were analyzed by the summary of efficiency (DDCT) method. The values presented are the mean of three independent biological replicates. Primers used are listed in <a class="usa-link" href="#SD3">Supplementary Table S1</a>.</p></section><section id="S12"><h3 class="pmc_sec_title">Phenotypic analysis</h3>
<p id="P26">Root length and hypocotyl elongation measurements were averaged from 30 plants for each genotype. Data were analyzed by Student’s two-tailed <em>t</em>-test, assuming equal variance; differences with a <em>P</em>-value &lt; 0.05 were considered significant.</p></section><section id="S13"><h3 class="pmc_sec_title">Immunoblotting and confocal microscopy analyses</h3>
<p id="P27">Fifty milligrams of fresh root tissues was ground in plastic tubes with plastic pestles using liquid nitrogen and 500 ml of SDS-containing extraction buffer (60 mM Tris-HCL (pH 8.8), 2% SDS, 2.5% glycerol, 0.13 mM EDTA (pH 8.0), and 1X Protease Inhibitor Cocktail Complete (Roche)). The tissue lysates were vortexed for 30 s, heated at 70°C for 10 minutes, and then centrifuged at 13,000 g twice for 5 minutes at room temperature. The supernatants were transferred to new tubes. For SDS-PAGE analysis, 5 ml of the extract in 1x NuPAGE LDS Sample Buffer (Invitrogen) was separated on 4–12% NuPage gel (Invitrogen) and transferred to PVDF (polyvinyl difluoride) membrane. The membrane was incubated with 3% BSA in 1x TBST (50 mM Tris-base, 150 mM NaCl, 0.05% Tween 20, pH 8.0) overnight at 4°C, and was probed with antibody (α-GFP, 1:20,000; Abcam) diluted in the blocking buffer (1:20,000) at room temperature for 1 h. The probed membrane was washed three times with 1x TBST for 5 min and then incubated with secondary antibody (goat anti-rabbit IgG for α-GFP, 1:20,000; Abcam) at room temperature for 1 h. The membrane was further washed four times with 1x TBST for 10 min before the signals were visualized with SuperSignal® West Dura Extended Duration Substrate (Pierce Biotechnology). To visualize YFP fluorescence, an inverted laser scanning confocal microscope Zeiss LSM510 was used to detect the DII expression.</p></section><section id="S14"><h3 class="pmc_sec_title">Free IAA analysis</h3>
<p id="P28">Approximately 20 roots were cut from ten-day-old seedlings and trasferred into an Eppendorf tube containing 1 ml of methanol. Internal standard of [2H5] IAA was added to the sample at amount of 100 fmol per root.</p></section></section><section id="S15"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="SD1"><div class="caption p">
<span>Supp Fig S1-S11</span><p id="P29"><strong>Figure S1</strong>. Tunicamycin induces activation of UPR target gene.</p>
<p id="P30"><strong>Figure S2.</strong> The transcripts of genes encoding ER-localized and nuclear proteins remain unchanged under Tm treatment.</p>
<p id="P31"><strong>Figure S3.</strong> DTT transcriptionally activates UPR target genes and down-regulates auxin regulators.</p>
<p id="P32"><strong>Figure S4.</strong> IRE1 plays fine-tuning roles in ER stress-induced down-regulation of TIR1/AFBs.</p>
<p id="P33"><strong>Figure S5.</strong>
<em>ire1</em> and <em>ire1 pin5</em> display comparable sensitivity to JA, ACC, and ABA.</p>
<p id="P34"><strong>Figure S6</strong>. <em>ire1</em> displays reduced activation of auxin-responsive genes upon NAA and IAA treatment.</p>
<p id="P35"><strong>Figure S7.</strong> The UPR target genes were not altered under IAA or NPA treatment.</p>
<p id="P36"><strong>Figure S8.</strong> The free auxin level is unchanged under ER stress.</p>
<p id="P37"><strong>Figure S9</strong>. Mutants impaired in auxin intracellular transport display a defective UPR phenotype.</p>
<p id="P38"><strong>Figure S10</strong>. Mutants impaired in intracellular auxin transport display comparable expression levels of UPR genes.</p>
<p id="P39"><strong>Figure S11</strong>. <em>ire1</em> and <em>ire1 pin5</em> display normal root density and hypocotyl elongation.</p>
</div>
<div class="media p" id="d36e1362"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/3981873/bin/NIHMS538200-supplement-Supp_Fig_S1-S11.pdf">NIHMS538200-supplement-Supp_Fig_S1-S11.pdf</a><sup> (249.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD2"><div class="caption p"><span>Supp Legends</span></div>
<div class="media p" id="d36e1366"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/3981873/bin/NIHMS538200-supplement-Supp_Legends.doc">NIHMS538200-supplement-Supp_Legends.doc</a><sup> (33KB, doc) </sup>
</div></div></section><section class="sm xbox font-sm" id="SD3"><div class="caption p">
<span>Supp Table S1. Table S1.</span><p id="P40">Primer list.</p>
</div>
<div class="media p" id="d36e1375"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/3981873/bin/NIHMS538200-supplement-Supp_Table_S1.doc">NIHMS538200-supplement-Supp_Table_S1.doc</a><sup> (78.5KB, doc) </sup>
</div></div></section></section><section class="ack" id="S16"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Teva Vernoux for sharing the DII-VENUS seeds and Jürgen Kleine-Vehn for sharing the <em>pils2-2</em>, <em>pils5-2</em>, and <em>pils2-2pils5-2</em> seeds and the Arabidopsis Biological Resource Center (ABRC) for seed stocks. This study was supported by grants from the Chemical Sciences, Geosciences and Biosciences Division, Office of Basic Energy Sciences, Office of Science, U.S. DOE (DE-FG02-91ER20021) for the infrastructure, National Institutes of Health (R01 GM101038-01), and NASA (NNX12AN71G).</p></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="R1">
<cite>Ahmed SU, Bar-Peled M, Raikhel NV. Cloning and subcellular location of an Arabidopsis receptor-like protein that shares common features with protein-sorting receptors of eukaryotic cells. Plant Physiol. 1997;114:325–336. doi: 10.1104/pp.114.1.325.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.114.1.325" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC158308/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9159954/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Cloning%20and%20subcellular%20location%20of%20an%20Arabidopsis%20receptor-like%20protein%20that%20shares%20common%20features%20with%20protein-sorting%20receptors%20of%20eukaryotic%20cells&amp;author=SU%20Ahmed&amp;author=M%20Bar-Peled&amp;author=NV%20Raikhel&amp;volume=114&amp;publication_year=1997&amp;pages=325-336&amp;pmid=9159954&amp;doi=10.1104/pp.114.1.325&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R2">
<cite>Barbez E, Kubes M, Rolcik J, Beziat C, Pencik A, Wang B, Rosquete MR, Zhu J, Dobrev PI, Lee Y, Zazimalova E, Petrasek J, Geisler M, Friml J, Kleine-Vehn J. A novel putative auxin carrier family regulates intracellular auxin homeostasis in plants. Nature. 2012;485:119–122. doi: 10.1038/nature11001.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature11001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22504182/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20novel%20putative%20auxin%20carrier%20family%20regulates%20intracellular%20auxin%20homeostasis%20in%20plants&amp;author=E%20Barbez&amp;author=M%20Kubes&amp;author=J%20Rolcik&amp;author=C%20Beziat&amp;author=A%20Pencik&amp;volume=485&amp;publication_year=2012&amp;pages=119-122&amp;pmid=22504182&amp;doi=10.1038/nature11001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R3">
<cite>Bertolotti A, Zhang Y, Hendershot LM, Harding HP, Ron D. Dynamic interaction of BiP and ER stress transducers in the unfolded-protein response. Nat Cell Biol. 2000;2:326–332. doi: 10.1038/35014014.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/35014014" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10854322/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat%20Cell%20Biol&amp;title=Dynamic%20interaction%20of%20BiP%20and%20ER%20stress%20transducers%20in%20the%20unfolded-protein%20response&amp;author=A%20Bertolotti&amp;author=Y%20Zhang&amp;author=LM%20Hendershot&amp;author=HP%20Harding&amp;author=D%20Ron&amp;volume=2&amp;publication_year=2000&amp;pages=326-332&amp;pmid=10854322&amp;doi=10.1038/35014014&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R4">
<cite>Brunoud G, Wells DM, Oliva M, Larrieu A, Mirabet V, Burrow AH, Beeckman T, Kepinski S, Traas J, Bennett MJ, Vernoux T. A novel sensor to map auxin response and distribution at high spatio-temporal resolution. Nature. 2012;482:103–106. doi: 10.1038/nature10791.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature10791" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22246322/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20novel%20sensor%20to%20map%20auxin%20response%20and%20distribution%20at%20high%20spatio-temporal%20resolution&amp;author=G%20Brunoud&amp;author=DM%20Wells&amp;author=M%20Oliva&amp;author=A%20Larrieu&amp;author=V%20Mirabet&amp;volume=482&amp;publication_year=2012&amp;pages=103-106&amp;pmid=22246322&amp;doi=10.1038/nature10791&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R5">
<cite>Cazzonelli CI, Vanstraelen M, Simon S, Yin K, Carron-Arthur A, Nisar N, Tarle G, Cuttriss AJ, Searle IR, Benkova E, Mathesius U, Masle J, Friml J, Pogson BJ. Role of the Arabidopsis PIN6 auxin transporter in auxin homeostasis and auxin-mediated development. PLoS One. 2013;8(7):e70069. doi: 10.1371/journal.pone.0070069.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0070069" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3726503/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23922907/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Role%20of%20the%20Arabidopsis%20PIN6%20auxin%20transporter%20in%20auxin%20homeostasis%20and%20auxin-mediated%20development&amp;author=CI%20Cazzonelli&amp;author=M%20Vanstraelen&amp;author=S%20Simon&amp;author=K%20Yin&amp;author=A%20Carron-Arthur&amp;volume=8&amp;issue=7&amp;publication_year=2013&amp;pages=e70069&amp;pmid=23922907&amp;doi=10.1371/journal.pone.0070069&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R6">
<cite>Chang C, Kwok SF, Bleecker AB, Meyerowitz EM. Arabidopsis ethylene-response gene ETR1: similarity of product to two-component regulators. Science. 1993;262:539–544. doi: 10.1126/science.8211181.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.8211181" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8211181/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Arabidopsis%20ethylene-response%20gene%20ETR1:%20similarity%20of%20product%20to%20two-component%20regulators&amp;author=C%20Chang&amp;author=SF%20Kwok&amp;author=AB%20Bleecker&amp;author=EM%20Meyerowitz&amp;volume=262&amp;publication_year=1993&amp;pages=539-544&amp;pmid=8211181&amp;doi=10.1126/science.8211181&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R7">
<cite>Che P, Bussell JD, Zhou W, Estavillo GM, Pogson BJ, Smith SM. Signaling from the endoplasmic reticulum activates brassinosteroid signaling and promotes acclimation to stress in Arabidopsis. Sci Signal. 2010;3:ra69. doi: 10.1126/scisignal.2001140.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/scisignal.2001140" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20876872/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci%20Signal&amp;title=Signaling%20from%20the%20endoplasmic%20reticulum%20activates%20brassinosteroid%20signaling%20and%20promotes%20acclimation%20to%20stress%20in%20Arabidopsis&amp;author=P%20Che&amp;author=JD%20Bussell&amp;author=W%20Zhou&amp;author=GM%20Estavillo&amp;author=BJ%20Pogson&amp;volume=3&amp;publication_year=2010&amp;pages=ra69&amp;pmid=20876872&amp;doi=10.1126/scisignal.2001140&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R8">
<cite>Chen Y, Brandizzi F. AtIRE1A/AtIRE1B and AGB1 independently control two essential unfolded protein response pathways in Arabidopsis. Plant J. 2012;69:266–277. doi: 10.1111/j.1365-313X.2011.04788.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2011.04788.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21914012/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=AtIRE1A/AtIRE1B%20and%20AGB1%20independently%20control%20two%20essential%20unfolded%20protein%20response%20pathways%20in%20Arabidopsis&amp;author=Y%20Chen&amp;author=F%20Brandizzi&amp;volume=69&amp;publication_year=2012&amp;pages=266-277&amp;pmid=21914012&amp;doi=10.1111/j.1365-313X.2011.04788.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R9">
<cite>Cheng Y, Dai X, Zhao Y. Auxin synthesized by the YUCCA flavin monooxygenases is essential for embryogenesis and leaf formation in Arabidopsis. Plant Cell. 2007;19:2430–2439. doi: 10.1105/tpc.107.053009.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.107.053009" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2002601/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17704214/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Auxin%20synthesized%20by%20the%20YUCCA%20flavin%20monooxygenases%20is%20essential%20for%20embryogenesis%20and%20leaf%20formation%20in%20Arabidopsis&amp;author=Y%20Cheng&amp;author=X%20Dai&amp;author=Y%20Zhao&amp;volume=19&amp;publication_year=2007&amp;pages=2430-2439&amp;pmid=17704214&amp;doi=10.1105/tpc.107.053009&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R10">
<cite>Cox JS, Shamu CE, Walter P. Transcriptional induction of genes encoding endoplasmic reticulum resident proteins requires a transmembrane protein kinase. Cell. 1993;73:1197–1206. doi: 10.1016/0092-8674(93)90648-a.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0092-8674(93)90648-a" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8513503/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Transcriptional%20induction%20of%20genes%20encoding%20endoplasmic%20reticulum%20resident%20proteins%20requires%20a%20transmembrane%20protein%20kinase&amp;author=JS%20Cox&amp;author=CE%20Shamu&amp;author=P%20Walter&amp;volume=73&amp;publication_year=1993&amp;pages=1197-1206&amp;pmid=8513503&amp;doi=10.1016/0092-8674(93)90648-a&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R11">
<cite>Cox JS, Walter P. A novel mechanism for regulating activity of a transcription factor that controls the unfolded protein response. Cell. 1996;87:391–404. doi: 10.1016/s0092-8674(00)81360-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s0092-8674(00)81360-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8898193/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20novel%20mechanism%20for%20regulating%20activity%20of%20a%20transcription%20factor%20that%20controls%20the%20unfolded%20protein%20response&amp;author=JS%20Cox&amp;author=P%20Walter&amp;volume=87&amp;publication_year=1996&amp;pages=391-404&amp;pmid=8898193&amp;doi=10.1016/s0092-8674(00)81360-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R12">
<cite>Credle JJ, Finer-Moore JS, Papa FR, Stroud RM, Walter P. On the mechanism of sensing unfolded protein in the endoplasmic reticulum. Proc Natl Acad Sci U S A. 2005;102:18773–18784. doi: 10.1073/pnas.0509487102.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0509487102" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1316886/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16365312/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=On%20the%20mechanism%20of%20sensing%20unfolded%20protein%20in%20the%20endoplasmic%20reticulum&amp;author=JJ%20Credle&amp;author=JS%20Finer-Moore&amp;author=FR%20Papa&amp;author=RM%20Stroud&amp;author=P%20Walter&amp;volume=102&amp;publication_year=2005&amp;pages=18773-18784&amp;pmid=16365312&amp;doi=10.1073/pnas.0509487102&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R13">
<cite>Dal Bosco C, Dovzhenko A, Liu X, Woerner N, Rensch T, Eismann M, Eimer S, Hegermann J, Paponov IA, Ruperti B, Heberle-Bors E, Touraev A, Cohen JD, Palme K. The endoplasmic reticulum localized PIN8 is a pollen-specific auxin carrier involved in intracellular auxin homeostasis. Plant J. 2012;71:860–870. doi: 10.1111/j.1365-313X.2012.05037.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2012.05037.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22540348/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=The%20endoplasmic%20reticulum%20localized%20PIN8%20is%20a%20pollen-specific%20auxin%20carrier%20involved%20in%20intracellular%20auxin%20homeostasis&amp;author=C%20Dal%20Bosco&amp;author=A%20Dovzhenko&amp;author=X%20Liu&amp;author=N%20Woerner&amp;author=T%20Rensch&amp;volume=71&amp;publication_year=2012&amp;pages=860-870&amp;pmid=22540348&amp;doi=10.1111/j.1365-313X.2012.05037.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R14">
<cite>Dharmasiri N, Dharmasiri S, Estelle M. The F-box protein TIR1 is an auxin receptor. Nature. 2005;435:441–445. doi: 10.1038/nature03543.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature03543" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15917797/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20F-box%20protein%20TIR1%20is%20an%20auxin%20receptor&amp;author=N%20Dharmasiri&amp;author=S%20Dharmasiri&amp;author=M%20Estelle&amp;volume=435&amp;publication_year=2005&amp;pages=441-445&amp;pmid=15917797&amp;doi=10.1038/nature03543&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R15">
<cite>Ding Z, Wang B, Moreno I, Duplakova N, Simon S, Carraro N, Reemmer J, Pencik A, Chen X, Tejos R, Skupa P, Pollmann S, Mravec J, Petrasek J, Zazimalova E, Honys D, Rolcik J, Murphy A, Orellana A, Geisler M, Friml J. ER-localized auxin transporter PIN8 regulates auxin homeostasis and male gametophyte development in Arabidopsis. Nat Commun. 2012;3:941. doi: 10.1038/ncomms1941.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ncomms1941" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22760640/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=ER-localized%20auxin%20transporter%20PIN8%20regulates%20auxin%20homeostasis%20and%20male%20gametophyte%20development%20in%20Arabidopsis&amp;author=Z%20Ding&amp;author=B%20Wang&amp;author=I%20Moreno&amp;author=N%20Duplakova&amp;author=S%20Simon&amp;volume=3&amp;publication_year=2012&amp;pages=941&amp;pmid=22760640&amp;doi=10.1038/ncomms1941&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R16">
<cite>Friml J, Benková E, Blilou I, Wisniewska J, Hamann T, Ljung K, Woody S, Sandberg G, Scheres B, Jürgens G, Palme K. AtPIN4 mediates sink-driven auxin gradients and root patterning in Arabidopsis. Cell. 2002;108:661–673. doi: 10.1016/s0092-8674(02)00656-6.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s0092-8674(02)00656-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11893337/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=AtPIN4%20mediates%20sink-driven%20auxin%20gradients%20and%20root%20patterning%20in%20Arabidopsis&amp;author=J%20Friml&amp;author=E%20Benkov%C3%A1&amp;author=I%20Blilou&amp;author=J%20Wisniewska&amp;author=T%20Hamann&amp;volume=108&amp;publication_year=2002&amp;pages=661-673&amp;pmid=11893337&amp;doi=10.1016/s0092-8674(02)00656-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R17">
<cite>Friml J, Jones AR. Endoplasmic reticulum: the rising compartment in auxin biology. Plant Physiol. 2010;154:458–462. doi: 10.1104/pp.110.161380.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.110.161380" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2948984/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20921163/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Endoplasmic%20reticulum:%20the%20rising%20compartment%20in%20auxin%20biology&amp;author=J%20Friml&amp;author=AR%20Jones&amp;volume=154&amp;publication_year=2010&amp;pages=458-462&amp;pmid=20921163&amp;doi=10.1104/pp.110.161380&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R18">
<cite>Friml J, Vieten A, Sauer M, Weijers D, Schwarz H, Hamann T, Offringa R, Jurgens G. Efflux-dependent auxin gradients establish the apical-basal axis of Arabidopsis. Nature. 2003;426:147–153. doi: 10.1038/nature02085.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature02085" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14614497/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Efflux-dependent%20auxin%20gradients%20establish%20the%20apical-basal%20axis%20of%20Arabidopsis&amp;author=J%20Friml&amp;author=A%20Vieten&amp;author=M%20Sauer&amp;author=D%20Weijers&amp;author=H%20Schwarz&amp;volume=426&amp;publication_year=2003&amp;pages=147-153&amp;pmid=14614497&amp;doi=10.1038/nature02085&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R19">
<cite>Gardner BM, Walter P. Unfolded Proteins Are Ire1-Activating Ligands That Directly Induce the Unfolded Protein Response. Science. 2011;333:1891–1894. doi: 10.1126/science.1209126.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1209126" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3202989/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21852455/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Unfolded%20Proteins%20Are%20Ire1-Activating%20Ligands%20That%20Directly%20Induce%20the%20Unfolded%20Protein%20Response&amp;author=BM%20Gardner&amp;author=P%20Walter&amp;volume=333&amp;publication_year=2011&amp;pages=1891-1894&amp;pmid=21852455&amp;doi=10.1126/science.1209126&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R20">
<cite>Gray WM, del Pozo JC, Walker L, Hobbie L, Risseeuw E, Banks T, Crosby WL, Yang M, Ma H, Estelle M. Identification of an SCF ubiquitin-ligase complex required for auxin response in Arabidopsis thaliana. Genes Dev. 1999;13:1678–1691. doi: 10.1101/gad.13.13.1678.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gad.13.13.1678" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC316846/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10398681/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&amp;title=Identification%20of%20an%20SCF%20ubiquitin-ligase%20complex%20required%20for%20auxin%20response%20in%20Arabidopsis%20thaliana&amp;author=WM%20Gray&amp;author=JC%20del%20Pozo&amp;author=L%20Walker&amp;author=L%20Hobbie&amp;author=E%20Risseeuw&amp;volume=13&amp;publication_year=1999&amp;pages=1678-1691&amp;pmid=10398681&amp;doi=10.1101/gad.13.13.1678&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R21">
<cite>Gray WM, Kepinski S, Rouse D, Leyser O, Estelle M. Auxin regulates SCF(TIR1)-dependent degradation of AUX/IAA proteins. Nature. 2001;414:271–276. doi: 10.1038/35104500.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/35104500" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11713520/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Auxin%20regulates%20SCF(TIR1)-dependent%20degradation%20of%20AUX/IAA%20proteins&amp;author=WM%20Gray&amp;author=S%20Kepinski&amp;author=D%20Rouse&amp;author=O%20Leyser&amp;author=M%20Estelle&amp;volume=414&amp;publication_year=2001&amp;pages=271-276&amp;pmid=11713520&amp;doi=10.1038/35104500&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R22">
<cite>Hetz C, Bernasconi P, Fisher J, Lee AH, Bassik MC, Antonsson B, Brandt GS, Iwakoshi NN, Schinzel A, Glimcher LH, Korsmeyer SJ. Proapoptotic BAX and BAK modulate the unfolded protein response by a direct interaction with IRE1alpha. Science. 2006;312:572–576. doi: 10.1126/science.1123480.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1123480" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16645094/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Proapoptotic%20BAX%20and%20BAK%20modulate%20the%20unfolded%20protein%20response%20by%20a%20direct%20interaction%20with%20IRE1alpha&amp;author=C%20Hetz&amp;author=P%20Bernasconi&amp;author=J%20Fisher&amp;author=AH%20Lee&amp;author=MC%20Bassik&amp;volume=312&amp;publication_year=2006&amp;pages=572-576&amp;pmid=16645094&amp;doi=10.1126/science.1123480&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R23">
<cite>Iwawaki T, Akai R, Yamanaka S, Kohno K. Function of IRE1 alpha in the placenta is essential for placental development and embryonic viability. Proc Natl Acad Sci U S A. 2009;106:16657–16662. doi: 10.1073/pnas.0903775106.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0903775106" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2757843/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19805353/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Function%20of%20IRE1%20alpha%20in%20the%20placenta%20is%20essential%20for%20placental%20development%20and%20embryonic%20viability&amp;author=T%20Iwawaki&amp;author=R%20Akai&amp;author=S%20Yamanaka&amp;author=K%20Kohno&amp;volume=106&amp;publication_year=2009&amp;pages=16657-16662&amp;pmid=19805353&amp;doi=10.1073/pnas.0903775106&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R24">
<cite>Kamauchi S, Nakatani H, Nakano C, Urade R. Gene expression in response to endoplasmic reticulum stress in Arabidopsis thaliana. FEBS J. 2005;272:3461–3476. doi: 10.1111/j.1742-4658.2005.04770.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1742-4658.2005.04770.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15978049/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20J&amp;title=Gene%20expression%20in%20response%20to%20endoplasmic%20reticulum%20stress%20in%20Arabidopsis%20thaliana&amp;author=S%20Kamauchi&amp;author=H%20Nakatani&amp;author=C%20Nakano&amp;author=R%20Urade&amp;volume=272&amp;publication_year=2005&amp;pages=3461-3476&amp;pmid=15978049&amp;doi=10.1111/j.1742-4658.2005.04770.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R25">
<cite>Kanneganti TD, Bai X, Tsai CW, Win J, Meulia T, Goodin M, Kamoun S, Hogenhout SA. A functional genetic assay for nuclear trafficking in plants. Plant J. 2007;50:149–158. doi: 10.1111/j.1365-313X.2007.03029.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2007.03029.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17346267/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=A%20functional%20genetic%20assay%20for%20nuclear%20trafficking%20in%20plants&amp;author=TD%20Kanneganti&amp;author=X%20Bai&amp;author=CW%20Tsai&amp;author=J%20Win&amp;author=T%20Meulia&amp;volume=50&amp;publication_year=2007&amp;pages=149-158&amp;pmid=17346267&amp;doi=10.1111/j.1365-313X.2007.03029.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R26">
<cite>Kepinski S, Leyser O. The Arabidopsis F-box protein TIR1 is an auxin receptor. Nature. 2005;435:446–451. doi: 10.1038/nature03542.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature03542" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15917798/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20Arabidopsis%20F-box%20protein%20TIR1%20is%20an%20auxin%20receptor&amp;author=S%20Kepinski&amp;author=O%20Leyser&amp;volume=435&amp;publication_year=2005&amp;pages=446-451&amp;pmid=15917798&amp;doi=10.1038/nature03542&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R27">
<cite>Kierzkowski D, Kmieciak M, Piontek P, Wojtaszek P, Szweykowska-Kulinska Z, Jarmolowski A. The Arabidopsis CBP20 targets the cap-binding complex to the nucleus, and is stabilized by CBP80. Plant J. 2009;59:814–825. doi: 10.1111/j.1365-313X.2009.03915.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2009.03915.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19453442/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=The%20Arabidopsis%20CBP20%20targets%20the%20cap-binding%20complex%20to%20the%20nucleus,%20and%20is%20stabilized%20by%20CBP80&amp;author=D%20Kierzkowski&amp;author=M%20Kmieciak&amp;author=P%20Piontek&amp;author=P%20Wojtaszek&amp;author=Z%20Szweykowska-Kulinska&amp;volume=59&amp;publication_year=2009&amp;pages=814-825&amp;pmid=19453442&amp;doi=10.1111/j.1365-313X.2009.03915.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R28">
<cite>Kimata Y, Kimata YI, Shimizu Y, Abe H, Farcasanu IC, Takeuchi M, Rose MD, Kohno K. Genetic evidence for a role of BiP/Kar2 that regulates Ire1 in response to accumulation of unfolded proteins. Mol Biol Cell. 2003;14:2559–2569. doi: 10.1091/mbc.E02-11-0708.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E02-11-0708" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC194903/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12808051/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&amp;title=Genetic%20evidence%20for%20a%20role%20of%20BiP/Kar2%20that%20regulates%20Ire1%20in%20response%20to%20accumulation%20of%20unfolded%20proteins&amp;author=Y%20Kimata&amp;author=YI%20Kimata&amp;author=Y%20Shimizu&amp;author=H%20Abe&amp;author=IC%20Farcasanu&amp;volume=14&amp;publication_year=2003&amp;pages=2559-2569&amp;pmid=12808051&amp;doi=10.1091/mbc.E02-11-0708&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R29">
<cite>Koizumi N, Martinez IM, Kimata Y, Kohno K, Sano H, Chrispeels MJ. Molecular characterization of two Arabidopsis Ire1 homologs, endoplasmic reticulum-located transmembrane protein kinases. Plant Physiol. 2001;127:949–962.</cite> [<a class="usa-link" href="/articles/PMC129266/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11706177/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Molecular%20characterization%20of%20two%20Arabidopsis%20Ire1%20homologs,%20endoplasmic%20reticulum-located%20transmembrane%20protein%20kinases&amp;author=N%20Koizumi&amp;author=IM%20Martinez&amp;author=Y%20Kimata&amp;author=K%20Kohno&amp;author=H%20Sano&amp;volume=127&amp;publication_year=2001&amp;pages=949-962&amp;pmid=11706177&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R30">
<cite>Kozutsumi Y, Segal M, Normington K, Gething MJ, Sambrook J. The presence of malfolded proteins in the endoplasmic reticulum signals the induction of glucose-regulated proteins. Nature. 1988;332:462–464. doi: 10.1038/332462a0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/332462a0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/3352747/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20presence%20of%20malfolded%20proteins%20in%20the%20endoplasmic%20reticulum%20signals%20the%20induction%20of%20glucose-regulated%20proteins&amp;author=Y%20Kozutsumi&amp;author=M%20Segal&amp;author=K%20Normington&amp;author=MJ%20Gething&amp;author=J%20Sambrook&amp;volume=332&amp;publication_year=1988&amp;pages=462-464&amp;pmid=3352747&amp;doi=10.1038/332462a0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R31">
<cite>Law AH, Chow CM, Jiang L. Secretory carrier membrane proteins. Protoplasma. 2012;249:269–283. doi: 10.1007/s00709-011-0295-0.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00709-011-0295-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21633931/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Protoplasma&amp;title=Secretory%20carrier%20membrane%20proteins&amp;author=AH%20Law&amp;author=CM%20Chow&amp;author=L%20Jiang&amp;volume=249&amp;publication_year=2012&amp;pages=269-283&amp;pmid=21633931&amp;doi=10.1007/s00709-011-0295-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R32">
<cite>Lin JH, Li H, Yasumura D, Cohen HR, Zhang C, Panning B, Shokat KM, Lavail MM, Walter P. IRE1 signaling affects cell fate during the unfolded protein response. Science. 2007;318:944–949. doi: 10.1126/science.1146361.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1146361" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3670588/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17991856/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=IRE1%20signaling%20affects%20cell%20fate%20during%20the%20unfolded%20protein%20response&amp;author=JH%20Lin&amp;author=H%20Li&amp;author=D%20Yasumura&amp;author=HR%20Cohen&amp;author=C%20Zhang&amp;volume=318&amp;publication_year=2007&amp;pages=944-949&amp;pmid=17991856&amp;doi=10.1126/science.1146361&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R33">
<cite>Ma L, Hong Z, Zhang Z. Perinuclear and nuclear envelope localizations of Arabidopsis Ran proteins. Plant Cell Rep. 2007;26:1373–1382. doi: 10.1007/s00299-007-0367-y.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00299-007-0367-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17530257/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Rep&amp;title=Perinuclear%20and%20nuclear%20envelope%20localizations%20of%20Arabidopsis%20Ran%20proteins&amp;author=L%20Ma&amp;author=Z%20Hong&amp;author=Z%20Zhang&amp;volume=26&amp;publication_year=2007&amp;pages=1373-1382&amp;pmid=17530257&amp;doi=10.1007/s00299-007-0367-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R34">
<cite>Marciniak SJ, Ron D. Endoplasmic reticulum stress signaling in disease. Physiol Rev. 2006;86:1133–1149. doi: 10.1152/physrev.00015.2006.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1152/physrev.00015.2006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17015486/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Physiol%20Rev&amp;title=Endoplasmic%20reticulum%20stress%20signaling%20in%20disease&amp;author=SJ%20Marciniak&amp;author=D%20Ron&amp;volume=86&amp;publication_year=2006&amp;pages=1133-1149&amp;pmid=17015486&amp;doi=10.1152/physrev.00015.2006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R35">
<cite>Mishiba K, Nagashima Y, Suzuki E, Hayashi N, Ogata Y, Shimada Y, Koizumi N. Defects in IRE1 enhance cell death and fail to degrade mRNAs encoding secretory pathway proteins in the Arabidopsis unfolded protein response. Proc Natl Acad Sci U S A. 2013;110:5713–5718. doi: 10.1073/pnas.1219047110.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1219047110" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3619347/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23509268/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Defects%20in%20IRE1%20enhance%20cell%20death%20and%20fail%20to%20degrade%20mRNAs%20encoding%20secretory%20pathway%20proteins%20in%20the%20Arabidopsis%20unfolded%20protein%20response&amp;author=K%20Mishiba&amp;author=Y%20Nagashima&amp;author=E%20Suzuki&amp;author=N%20Hayashi&amp;author=Y%20Ogata&amp;volume=110&amp;publication_year=2013&amp;pages=5713-5718&amp;pmid=23509268&amp;doi=10.1073/pnas.1219047110&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R36">
<cite>Mori K, Ma W, Gething MJ, Sambrook J. A transmembrane protein with a cdc2+/CDC28-related kinase activity is required for signaling from the ER to the nucleus. Cell. 1993;74:743–756. doi: 10.1016/0092-8674(93)90521-q.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0092-8674(93)90521-q" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8358794/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20transmembrane%20protein%20with%20a%20cdc2+/CDC28-related%20kinase%20activity%20is%20required%20for%20signaling%20from%20the%20ER%20to%20the%20nucleus&amp;author=K%20Mori&amp;author=W%20Ma&amp;author=MJ%20Gething&amp;author=J%20Sambrook&amp;volume=74&amp;publication_year=1993&amp;pages=743-756&amp;pmid=8358794&amp;doi=10.1016/0092-8674(93)90521-q&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R37">
<cite>Mravec J, Skupa P, Bailly A, Hoyerova K, Krecek P, Bielach A, Petrasek J, Zhang J, Gaykova V, Stierhof YD, Dobrev PI, Schwarzerova K, Rolcik J, Seifertova D, Luschnig C, Benkova E, Zazimalova E, Geisler M, Friml J. Subcellular homeostasis of phytohormone auxin is mediated by the ER-localized PIN5 transporter. Nature. 2009;459:1136–1140. doi: 10.1038/nature08066.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature08066" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19506555/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Subcellular%20homeostasis%20of%20phytohormone%20auxin%20is%20mediated%20by%20the%20ER-localized%20PIN5%20transporter&amp;author=J%20Mravec&amp;author=P%20Skupa&amp;author=A%20Bailly&amp;author=K%20Hoyerova&amp;author=P%20Krecek&amp;volume=459&amp;publication_year=2009&amp;pages=1136-1140&amp;pmid=19506555&amp;doi=10.1038/nature08066&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R38">
<cite>Nagashima Y, Mishiba K, Suzuki E, Shimada Y, Iwata Y, Koizumi N. Arabidopsis IRE1 catalyses unconventional splicing of bZIP60 mRNA to produce the active transcription factor. Sci Rep. 2011;1:29. doi: 10.1038/srep00029.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/srep00029" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3216516/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22355548/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Arabidopsis%20IRE1%20catalyses%20unconventional%20splicing%20of%20bZIP60%20mRNA%20to%20produce%20the%20active%20transcription%20factor&amp;author=Y%20Nagashima&amp;author=K%20Mishiba&amp;author=E%20Suzuki&amp;author=Y%20Shimada&amp;author=Y%20Iwata&amp;volume=1&amp;publication_year=2011&amp;pages=29&amp;pmid=22355548&amp;doi=10.1038/srep00029&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R39">
<cite>Noh SJ, Kwon CS, Chung WI. Characterization of two homologs of Ire1p, a kinase/endoribonuclease in yeast, in Arabidopsis thaliana. Biochim Biophys Acta. 2002;1575:130–134. doi: 10.1016/s0167-4781(02)00237-3.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s0167-4781(02)00237-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12020828/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&amp;title=Characterization%20of%20two%20homologs%20of%20Ire1p,%20a%20kinase/endoribonuclease%20in%20yeast,%20in%20Arabidopsis%20thaliana&amp;author=SJ%20Noh&amp;author=CS%20Kwon&amp;author=WI%20Chung&amp;volume=1575&amp;publication_year=2002&amp;pages=130-134&amp;pmid=12020828&amp;doi=10.1016/s0167-4781(02)00237-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R40">
<cite>Okada K, Ueda J, Komaki MK, Bell CJ, Shimura Y. Requirement of the Auxin Polar Transport System in Early Stages of Arabidopsis Floral Bud Formation. The plant cell. 1991;3:677–684. doi: 10.1105/tpc.3.7.677.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.3.7.677" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC160035/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12324609/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=The%20plant%20cell&amp;title=Requirement%20of%20the%20Auxin%20Polar%20Transport%20System%20in%20Early%20Stages%20of%20Arabidopsis%20Floral%20Bud%20Formation&amp;author=K%20Okada&amp;author=J%20Ueda&amp;author=MK%20Komaki&amp;author=CJ%20Bell&amp;author=Y%20Shimura&amp;volume=3&amp;publication_year=1991&amp;pages=677-684&amp;pmid=12324609&amp;doi=10.1105/tpc.3.7.677&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R41">
<cite>Petrasek J, Mravec J, Bouchard R, Blakeslee JJ, Abas M, Seifertova D, Wisniewska J, Tadele Z, Kubes M, Covanova M, Dhonukshe P, Skupa P, Benkova E, Perry L, Krecek P, Lee OR, Fink GR, Geisler M, Murphy AS, Luschnig C, Zazimalova E, Friml J. PIN proteins perform a rate-limiting function in cellular auxin efflux. Science. 2006;312:914–918. doi: 10.1126/science.1123542.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1123542" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16601150/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=PIN%20proteins%20perform%20a%20rate-limiting%20function%20in%20cellular%20auxin%20efflux&amp;author=J%20Petrasek&amp;author=J%20Mravec&amp;author=R%20Bouchard&amp;author=JJ%20Blakeslee&amp;author=M%20Abas&amp;volume=312&amp;publication_year=2006&amp;pages=914-918&amp;pmid=16601150&amp;doi=10.1126/science.1123542&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R42">
<cite>Roman G, Lubarsky B, Kieber JJ, Rothenberg M, Ecker JR. Genetic Analysis of Ethylene Signal Transduction in Arabidopsis Thaliana: Five Novel Mutant Loci Integrated into a Stress Response Pathway. Genetics. 1995;139:1393–1409. doi: 10.1093/genetics/139.3.1393.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/genetics/139.3.1393" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1206465/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/7768447/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genetic%20Analysis%20of%20Ethylene%20Signal%20Transduction%20in%20Arabidopsis%20Thaliana:%20Five%20Novel%20Mutant%20Loci%20Integrated%20into%20a%20Stress%20Response%20Pathway&amp;author=G%20Roman&amp;author=B%20Lubarsky&amp;author=JJ%20Kieber&amp;author=M%20Rothenberg&amp;author=JR%20Ecker&amp;volume=139&amp;publication_year=1995&amp;pages=1393-1409&amp;pmid=7768447&amp;doi=10.1093/genetics/139.3.1393&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R43">
<cite>Sauer M, Balla J, Luschnig C, Wisniewska J, Reinöhl V, Friml J, Benková E. Canalization of auxin flow by Aux/IAA-ARF-dependent feedback regulation of PIN polarity. Genes Dev. 2006;20:2902–2911. doi: 10.1101/gad.390806.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gad.390806" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1619939/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17043314/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&amp;title=Canalization%20of%20auxin%20flow%20by%20Aux/IAA-ARF-dependent%20feedback%20regulation%20of%20PIN%20polarity&amp;author=M%20Sauer&amp;author=J%20Balla&amp;author=C%20Luschnig&amp;author=J%20Wisniewska&amp;author=V%20Rein%C3%B6hl&amp;volume=20&amp;publication_year=2006&amp;pages=2902-2911&amp;pmid=17043314&amp;doi=10.1101/gad.390806&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R44">
<cite>Schroder M, Kaufman RJ. The mammalian unfolded protein response. Annu Rev Biochem. 2005;74:739–789. doi: 10.1146/annurev.biochem.73.011303.074134.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev.biochem.73.011303.074134" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15952902/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Biochem&amp;title=The%20mammalian%20unfolded%20protein%20response&amp;author=M%20Schroder&amp;author=RJ%20Kaufman&amp;volume=74&amp;publication_year=2005&amp;pages=739-789&amp;pmid=15952902&amp;doi=10.1146/annurev.biochem.73.011303.074134&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R45">
<cite>Swarup R, Kargul J, Marchant A, et al.  Structure–function analysis of the presumptive Arabidopsis auxin permease AUX1. Plant Cell. 2004;16:3069–3083. doi: 10.1105/tpc.104.024737.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.104.024737" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC527199/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15486104/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Structure%E2%80%93function%20analysis%20of%20the%20presumptive%20Arabidopsis%20auxin%20permease%20AUX1&amp;author=R%20Swarup&amp;author=J%20Kargul&amp;author=A%20Marchant&amp;volume=16&amp;publication_year=2004&amp;pages=3069-3083&amp;pmid=15486104&amp;doi=10.1105/tpc.104.024737&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R46">
<cite>Ulmasov T, Hagen G, Guilfoyle TJ. ARF1, a transcription factor that binds to auxin response elements. Science. 1997;276:1865–1868. doi: 10.1126/science.276.5320.1865.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.276.5320.1865" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9188533/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=ARF1,%20a%20transcription%20factor%20that%20binds%20to%20auxin%20response%20elements&amp;author=T%20Ulmasov&amp;author=G%20Hagen&amp;author=TJ%20Guilfoyle&amp;volume=276&amp;publication_year=1997&amp;pages=1865-1868&amp;pmid=9188533&amp;doi=10.1126/science.276.5320.1865&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R47">
<cite>Urano F, Wang X, Bertolotti A, Zhang Y, Chung P, Harding HP, Ron D. Coupling of stress in the ER to activation of JNK protein kinases by transmembrane protein kinase IRE1. Science. 2000;287:664–666. doi: 10.1126/science.287.5453.664.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.287.5453.664" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10650002/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Coupling%20of%20stress%20in%20the%20ER%20to%20activation%20of%20JNK%20protein%20kinases%20by%20transmembrane%20protein%20kinase%20IRE1&amp;author=F%20Urano&amp;author=X%20Wang&amp;author=A%20Bertolotti&amp;author=Y%20Zhang&amp;author=P%20Chung&amp;volume=287&amp;publication_year=2000&amp;pages=664-666&amp;pmid=10650002&amp;doi=10.1126/science.287.5453.664&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R48">
<cite>Wisniewska J, Xu J, Seifertova D, Brewer PB, Ruzicka K, Blilou I, Rouquie D, Benkova E, Scheres B, Friml J. Polar PIN localization directs auxin flow in plants. Science. 2006;312:883. doi: 10.1126/science.1121356.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1121356" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16601151/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Polar%20PIN%20localization%20directs%20auxin%20flow%20in%20plants&amp;author=J%20Wisniewska&amp;author=J%20Xu&amp;author=D%20Seifertova&amp;author=PB%20Brewer&amp;author=K%20Ruzicka&amp;volume=312&amp;publication_year=2006&amp;pages=883&amp;pmid=16601151&amp;doi=10.1126/science.1121356&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R49">
<cite>Wu J, Kaufman RJ. From acute ER stress to physiological roles of the Unfolded Protein Response. Cell Death Differ. 2006;13:374–384. doi: 10.1038/sj.cdd.4401840.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/sj.cdd.4401840" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16397578/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ&amp;title=From%20acute%20ER%20stress%20to%20physiological%20roles%20of%20the%20Unfolded%20Protein%20Response&amp;author=J%20Wu&amp;author=RJ%20Kaufman&amp;volume=13&amp;publication_year=2006&amp;pages=374-384&amp;pmid=16397578&amp;doi=10.1038/sj.cdd.4401840&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R50">
<cite>Wulfetange K, Lomin SN, Romanov GA, Stolz A, Heyl A, Schmulling T. The cytokinin receptors of Arabidopsis are located mainly to the endoplasmic reticulum. Plant Physiol. 2011;156:1808–1818. doi: 10.1104/pp.111.180539.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.111.180539" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3149959/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21709172/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=The%20cytokinin%20receptors%20of%20Arabidopsis%20are%20located%20mainly%20to%20the%20endoplasmic%20reticulum&amp;author=K%20Wulfetange&amp;author=SN%20Lomin&amp;author=GA%20Romanov&amp;author=A%20Stolz&amp;author=A%20Heyl&amp;volume=156&amp;publication_year=2011&amp;pages=1808-1818&amp;pmid=21709172&amp;doi=10.1104/pp.111.180539&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R51">
<cite>Xu T, Wen M, Nagawa S, Fu Y, Chen JG, Wu MJ, Perrot-Rechenmann C, Friml J, Jones AM, Yang Z. Cell surface- and rho GTPase-based auxin signaling controls cellular interdigitation in Arabidopsis. Cell. 2010;143:99–110. doi: 10.1016/j.cell.2010.09.003.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cell.2010.09.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2950838/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20887895/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Cell%20surface-%20and%20rho%20GTPase-based%20auxin%20signaling%20controls%20cellular%20interdigitation%20in%20Arabidopsis&amp;author=T%20Xu&amp;author=M%20Wen&amp;author=S%20Nagawa&amp;author=Y%20Fu&amp;author=JG%20Chen&amp;volume=143&amp;publication_year=2010&amp;pages=99-110&amp;pmid=20887895&amp;doi=10.1016/j.cell.2010.09.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R52">
<cite>Zádníková P, Petrásek J, Marhavy P, Raz V, Vandenbussche F, Ding Z, Schwarzerová K, Morita MT, Tasaka M, Hejátko J, Van Der Straeten D, Friml J, Benková E. Role of PIN-mediated auxin efflux in apical hook development of Arabidopsis thaliana. Development. 2010;137:607–617. doi: 10.1242/dev.041277.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/dev.041277" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20110326/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Role%20of%20PIN-mediated%20auxin%20efflux%20in%20apical%20hook%20development%20of%20Arabidopsis%20thaliana&amp;author=P%20Z%C3%A1dn%C3%ADkov%C3%A1&amp;author=J%20Petr%C3%A1sek&amp;author=P%20Marhavy&amp;author=V%20Raz&amp;author=F%20Vandenbussche&amp;volume=137&amp;publication_year=2010&amp;pages=607-617&amp;pmid=20110326&amp;doi=10.1242/dev.041277&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="R53">
<cite>Zhao Y, Christensen SK, Fankhauser C, Cashman JR, Cohen JD, Weigel D, Chory J. A role for flavin monooxygenase-like enzymes in auxin biosynthesis. Science. 2001;291:306–309. doi: 10.1126/science.291.5502.306.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.291.5502.306" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11209081/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A%20role%20for%20flavin%20monooxygenase-like%20enzymes%20in%20auxin%20biosynthesis&amp;author=Y%20Zhao&amp;author=SK%20Christensen&amp;author=C%20Fankhauser&amp;author=JR%20Cashman&amp;author=JD%20Cohen&amp;volume=291&amp;publication_year=2001&amp;pages=306-309&amp;pmid=11209081&amp;doi=10.1126/science.291.5502.306&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ol></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>Supp Fig S1-S11</span><p><strong>Figure S1</strong>. Tunicamycin induces activation of UPR target gene.</p>
<p><strong>Figure S2.</strong> The transcripts of genes encoding ER-localized and nuclear proteins remain unchanged under Tm treatment.</p>
<p><strong>Figure S3.</strong> DTT transcriptionally activates UPR target genes and down-regulates auxin regulators.</p>
<p><strong>Figure S4.</strong> IRE1 plays fine-tuning roles in ER stress-induced down-regulation of TIR1/AFBs.</p>
<p><strong>Figure S5.</strong>
<em>ire1</em> and <em>ire1 pin5</em> display comparable sensitivity to JA, ACC, and ABA.</p>
<p><strong>Figure S6</strong>. <em>ire1</em> displays reduced activation of auxin-responsive genes upon NAA and IAA treatment.</p>
<p><strong>Figure S7.</strong> The UPR target genes were not altered under IAA or NPA treatment.</p>
<p><strong>Figure S8.</strong> The free auxin level is unchanged under ER stress.</p>
<p><strong>Figure S9</strong>. Mutants impaired in auxin intracellular transport display a defective UPR phenotype.</p>
<p><strong>Figure S10</strong>. Mutants impaired in intracellular auxin transport display comparable expression levels of UPR genes.</p>
<p><strong>Figure S11</strong>. <em>ire1</em> and <em>ire1 pin5</em> display normal root density and hypocotyl elongation.</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/3981873/bin/NIHMS538200-supplement-Supp_Fig_S1-S11.pdf">NIHMS538200-supplement-Supp_Fig_S1-S11.pdf</a><sup> (249.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>Supp Legends</span></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/3981873/bin/NIHMS538200-supplement-Supp_Legends.doc">NIHMS538200-supplement-Supp_Legends.doc</a><sup> (33KB, doc) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>Supp Table S1. Table S1.</span><p>Primer list.</p>
</div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/3981873/bin/NIHMS538200-supplement-Supp_Table_S1.doc">NIHMS538200-supplement-Supp_Table_S1.doc</a><sup> (78.5KB, doc) </sup>
</div></div></section></section></section></section></section></article>
</main></body></html>