<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6529585/"/><title>Proteomic and Metabolomic Characteristics of Extremophilic Fungi Under Simulated Mars Conditions - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Front Microbiol</button></div>. 2019 May 15;10:1013. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2019.01013" rel="noopener noreferrer" target="_blank">10.3389/fmicb.2019.01013</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Proteomic and Metabolomic Characteristics of Extremophilic Fungi Under Simulated Mars Conditions</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blachowicz%20A%22%5BAuthor%5D"><span class="name western">Adriana Blachowicz</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Adriana Blachowicz</span></h3>
<div class="p">
<sup>1</sup>Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, Los Angeles, CA, United States</div>
<div class="p">
<sup>2</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blachowicz%20A%22%5BAuthor%5D"><span class="name western">Adriana Blachowicz</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chiang%20AJ%22%5BAuthor%5D"><span class="name western">Abby J Chiang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Abby J Chiang</span></h3>
<div class="p">
<sup>3</sup>Department of Molecular Imaging and Therapy, Beckman Research Institute of City of Hope, Duarte, CA, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chiang%20AJ%22%5BAuthor%5D"><span class="name western">Abby J Chiang</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elsaesser%20A%22%5BAuthor%5D"><span class="name western">Andreas Elsaesser</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Andreas Elsaesser</span></h3>
<div class="p">
<sup>4</sup>Department of Physics, Free University of Berlin, Berlin, Germany</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elsaesser%20A%22%5BAuthor%5D"><span class="name western">Andreas Elsaesser</span></a>
</div>
</div>
<sup>4</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kalkum%20M%22%5BAuthor%5D"><span class="name western">Markus Kalkum</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Markus Kalkum</span></h3>
<div class="p">
<sup>3</sup>Department of Molecular Imaging and Therapy, Beckman Research Institute of City of Hope, Duarte, CA, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kalkum%20M%22%5BAuthor%5D"><span class="name western">Markus Kalkum</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ehrenfreund%20P%22%5BAuthor%5D"><span class="name western">Pascale Ehrenfreund</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Pascale Ehrenfreund</span></h3>
<div class="p">
<sup>5</sup>Leiden Institute of Chemistry, Leiden University, Leiden, Netherlands</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ehrenfreund%20P%22%5BAuthor%5D"><span class="name western">Pascale Ehrenfreund</span></a>
</div>
</div>
<sup>5</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stajich%20JE%22%5BAuthor%5D"><span class="name western">Jason E Stajich</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jason E Stajich</span></h3>
<div class="p">
<sup>6</sup>Department of Microbiology and Plant Pathology, Institute of Integrative Genome Biology, University of California, Riverside, Riverside, CA, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stajich%20JE%22%5BAuthor%5D"><span class="name western">Jason E Stajich</span></a>
</div>
</div>
<sup>6</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Torok%20T%22%5BAuthor%5D"><span class="name western">Tamas Torok</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Tamas Torok</span></h3>
<div class="p">
<sup>7</sup>Department of Ecology, Lawrence Berkeley National Laboratory, Berkeley, CA, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Torok%20T%22%5BAuthor%5D"><span class="name western">Tamas Torok</span></a>
</div>
</div>
<sup>7</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CCC%22%5BAuthor%5D"><span class="name western">Clay C C Wang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Clay C C Wang</span></h3>
<div class="p">
<sup>1</sup>Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, Los Angeles, CA, United States</div>
<div class="p">
<sup>8</sup>Department of Chemistry, Dornsife College of Letters, Arts, and Sciences, University of Southern California, Los Angeles, CA, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CCC%22%5BAuthor%5D"><span class="name western">Clay C C Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>8</sup>, <a aria-describedby="id9" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Kasthuri Venkateswaran</span></h3>
<div class="p">
<sup>2</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a>
</div>
</div>
<sup>2,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, Los Angeles, CA, United States</div>
<div id="aff2">
<sup>2</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, United States</div>
<div id="aff3">
<sup>3</sup>Department of Molecular Imaging and Therapy, Beckman Research Institute of City of Hope, Duarte, CA, United States</div>
<div id="aff4">
<sup>4</sup>Department of Physics, Free University of Berlin, Berlin, Germany</div>
<div id="aff5">
<sup>5</sup>Leiden Institute of Chemistry, Leiden University, Leiden, Netherlands</div>
<div id="aff6">
<sup>6</sup>Department of Microbiology and Plant Pathology, Institute of Integrative Genome Biology, University of California, Riverside, Riverside, CA, United States</div>
<div id="aff7">
<sup>7</sup>Department of Ecology, Lawrence Berkeley National Laboratory, Berkeley, CA, United States</div>
<div id="aff8">
<sup>8</sup>Department of Chemistry, Dornsife College of Letters, Arts, and Sciences, University of Southern California, Los Angeles, CA, United States</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Edited by: Claudia P. Saavedra, Universidad Andrés Bello, Chile</p></div>
<div class="fn" id="fn2"><p>Reviewed by: Guilherme T. P. Brancini, Universidade de São Paulo Ribeirão Preto, Brazil; Yanchun Shao, Huazhong Agricultural University, China</p></div>
<div class="fn" id="c001">
<sup>✉</sup><p class="display-inline">*Correspondence: Kasthuri Venkateswaran, <span>kjvenkat@jpl.nasa.gov</span></p>
</div>
<div class="fn" id="fn002"><p>This article was submitted to Extreme Microbiology, a section of the journal Frontiers in Microbiology</p></div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2019 Jan 7; Accepted 2019 Apr 24; Collection date 2019.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2019 Blachowicz, Chiang, Elsaesser, Kalkum, Ehrenfreund, Stajich, Torok, Wang and Venkateswaran.</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC6529585  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31156574/">31156574</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Filamentous fungi have been associated with extreme habitats, including nuclear power plant accident sites and the International Space Station (ISS). Due to their immense adaptation and phenotypic plasticity capacities, fungi may thrive in what seems like uninhabitable niches. This study is the first report of fungal survival after exposure of monolayers of conidia to simulated Mars conditions (SMC). Conidia of several Chernobyl nuclear accident-associated and ISS-isolated strains were tested for UV-C and SMC sensitivity, which resulted in strain-dependent survival. Strains surviving exposure to SMC for 30 min, ISSFT-021-30 and IMV 00236-30, were further characterized for proteomic, and metabolomic changes. Differential expression of proteins involved in ribosome biogenesis, translation, and carbohydrate metabolic processes was observed. No significant metabolome alterations were revealed. Lastly, ISSFT-021-30 conidia re-exposed to UV-C exhibited enhanced UV-C resistance when compared to the conidia of unexposed ISSFT-021.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> fungi, simulated mars conditions, proteome, metabolome, Chernobyl, international space station, extremophiles</p></section></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Extremophiles are of interest to the National Aeronautics and Space Administration (NASA) due to their potential to survive hostile and extraterrestrial conditions (<a aria-describedby="B59" class="usa-link" href="#B59">Rummel et al., 2002</a>). Bacteria, fungi, and archaea have been shown to thrive in habitats characterized by low nutrient availability (<a aria-describedby="B54" class="usa-link" href="#B54">Onofri et al., 2004</a>), desiccation (<a aria-describedby="B5" class="usa-link" href="#B5">Barnard et al., 2013</a>), high and low temperatures (<a aria-describedby="B44" class="usa-link" href="#B44">McKay et al., 2003</a>; <a aria-describedby="B53" class="usa-link" href="#B53">Onofri et al., 2007</a>), acidic and alkaline pH (<a aria-describedby="B29" class="usa-link" href="#B29">Gonzalez-Toril et al., 2003</a>; <a aria-describedby="B28" class="usa-link" href="#B28">Gonsalves, 2012</a>), or radiation (<a aria-describedby="B76" class="usa-link" href="#B76">Zhdanova et al., 2004</a>; <a aria-describedby="B18" class="usa-link" href="#B18">Dadachova and Casadevall, 2008</a>; <a aria-describedby="B22" class="usa-link" href="#B22">Fendrihan et al., 2009</a>; <a aria-describedby="B66" class="usa-link" href="#B66">Stan-Lotter and Fendrihan, 2013</a>). There is a need for studies that focus on elucidating microbial survival mechanisms in extreme habitats, as a primary goal of the planetary protection policy is to prevent forward and backward contamination of any celestial bodies and the Earth (<a aria-describedby="B59" class="usa-link" href="#B59">Rummel et al., 2002</a>). So far, the majority of such studies have focused on investigating extremophilic bacteria, mainly spore-formers (<a aria-describedby="B48" class="usa-link" href="#B48">Nicholson et al., 2012</a>; <a aria-describedby="B71" class="usa-link" href="#B71">Tirumalai et al., 2013</a>), and rocks containing fungi (<a aria-describedby="B51" class="usa-link" href="#B51">Onofri et al., 2012</a>; <a aria-describedby="B75" class="usa-link" href="#B75">Zakharova et al., 2014</a>) however, monolayers of fungal conidia have not been studied.</p>
<p>One investigation revealed that the survival of spacecraft-associated bacteria under simulated Mars UV irradiation, is strain-specific and depends on the isolation site (<a aria-describedby="B46" class="usa-link" href="#B46">Newcombe et al., 2005</a>). Spacecraft-associated bacteria were more resistant than <em>Bacillus subtillis</em> 168, used as a dosimetric control (<a aria-describedby="B46" class="usa-link" href="#B46">Newcombe et al., 2005</a>). Additionally, exposure of <em>B. subtilis</em> 168 spores to dark space conditions (no UV) outside the International Space Station (ISS) and SMC for 559 days induced the expression of genes involved in DNA and protein damage responses, and oxidative and envelope stress (<a aria-describedby="B48" class="usa-link" href="#B48">Nicholson et al., 2012</a>). Further, the first-generation of spores of another bacterium, <em>Bacillus pumillus</em> SAFR-032, isolated from the Jet Propulsion Laboratory (JPL) spacecraft assembly facility (SAF) was more UV-C resistant than the ground control counterparts following exposure to space UV conditions for 18 months. Extensive studies into the whole genome of SAFR-032 revealed the presence of several DNA repair-associated genes that may have facilitated its survival and adaptation to harsh environmental conditions. Further, proteome analysis showed that stress response proteins, like superoxide dismutase, were increased in abundance when compared to the control (<a aria-describedby="B73" class="usa-link" href="#B73">Vaishampayan et al., 2012</a>).</p>
<p>Fungi produce conidia or spores as part of their life cycle. Some of these structures can be more resistant to environmental impact than the typical fungal coenocytic cell. Therefore, molecular characterization and further understanding of fungal resistance to UV-C and SMC are of high importance. Additionally, fungi possess a wide variety of mechanisms to protect against solar radiation, which include enzymes removing reactive oxygen species (ROS), DNA repair mechanisms, including nucleotide excision repair (NER) and photoreactivation (PR), and the production of pigments such as melanins and melanin-like compounds, and UV-absorbing metabolites that act as sunscreens (<a aria-describedby="B11" class="usa-link" href="#B11">Braga et al., 2015</a>). The hardy nature, adaptability and plasticity of fungi enable their persistence in extreme conditions, making them potential contaminants of cleanrooms and, therefore, a forward contamination source.</p>
<p>On April 25, 1986 one of the most significant nuclear accidents in history took place. Reactor 4 of the Chernobyl nuclear power plant (ChNPP) exploded and high levels of released radioactivity turned the surrounding area into a hostile environment. Within the following 18 years over 2,000 fungal isolates were collected from the nuclear power plant, its 30-km Exclusion Zone and beyond, representing some 200 species in over 90 genera (<a aria-describedby="B76" class="usa-link" href="#B76">Zhdanova et al., 2004</a>). The observed fungal communities were of low complexity and dominated by melanin-containing strains. Approximately 20% of the isolates displayed a previously unknown phenomenon of growing toward the radiation source referred to as positive radiotropism (<a aria-describedby="B76" class="usa-link" href="#B76">Zhdanova et al., 2004</a>; <a aria-describedby="B20" class="usa-link" href="#B20">Dighton et al., 2008</a>). Therefore, this unique characteristic of Chernobyl fungi makes them an ideal fungal model for studying adaptation to SMC. Further, the fungal species isolated from Chernobyl have also been detected in built environments, including JPL clean rooms (<a aria-describedby="B40" class="usa-link" href="#B40">La Duc et al., 2012</a>; <a aria-describedby="B74" class="usa-link" href="#B74">Weinmaier et al., 2015</a>), simulated closed habitats (<a aria-describedby="B9" class="usa-link" href="#B9">Blachowicz et al., 2017</a>), and on board of the ISS (<a aria-describedby="B15" class="usa-link" href="#B15">Checinska et al., 2015</a>), which further emphasizes the need to study their potential for forward contamination.</p>
<p>The ISS is another type of environment that may be considered hostile for microorganisms, featuring constant temperature and humidity, controlled airflow, enhanced irradiation, and microgravity (<a aria-describedby="B45" class="usa-link" href="#B45">Mora et al., 2016</a>). Several reports examining microbial burden aboard the ISS have shown that fungal populations thrive in this environment (<a aria-describedby="B15" class="usa-link" href="#B15">Checinska et al., 2015</a>; <a aria-describedby="B6" class="usa-link" href="#B6">Be et al., 2017</a>). Other studies have investigated molecular adaptations of selected species to space conditions, revealing changes in metabolome, and proteome of ISS-isolated strains (<a aria-describedby="B37" class="usa-link" href="#B37">Knox et al., 2016</a>; <a aria-describedby="B58" class="usa-link" href="#B58">Romsdahl et al., 2018</a>; <a aria-describedby="B8" class="usa-link" href="#B8">Blachowicz et al., 2019</a>). In other investigations, several multilayered or embedded cryptoendolithic fungal communities were exposed to space conditions (<a aria-describedby="B7" class="usa-link" href="#B7">Billi et al., 2011</a>; <a aria-describedby="B51" class="usa-link" href="#B51">Onofri et al., 2012</a>, <a aria-describedby="B52" class="usa-link" href="#B52">2015</a>; <a aria-describedby="B60" class="usa-link" href="#B60">Scalzi et al., 2012</a>; <a aria-describedby="B56" class="usa-link" href="#B56">Pacelli et al., 2018</a>). Tested isolates that adapted to environmental extremes of their habitats survived SMC for extended periods and revealed high stability of the DNA in the surviving cells (<a aria-describedby="B7" class="usa-link" href="#B7">Billi et al., 2011</a>; <a aria-describedby="B52" class="usa-link" href="#B52">Onofri et al., 2015</a>; <a aria-describedby="B56" class="usa-link" href="#B56">Pacelli et al., 2018</a>). Despite these studies, there are significant gaps in our understanding of the molecular mechanisms that facilitate survival of filamentous fungi under SMC and their potential to adapt and survive in outer space.</p>
<p>This is the first report that evaluated the survival of Chernobyl-associated and ISS-isolated fungal conidia exposed in monolayers to SMC. Dried monolayers of conidia of four filamentous fungi were exposed to SMC and two strains, <em>Aspergillus fumigatus</em> and <em>Cladosporium cladosporoides</em>, which survived exposure to SMC for 30 min, were further analyzed for phenotypic, proteomic and metabolomic changes.</p></section><section id="sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="sec3"><h3 class="pmc_sec_title">Identification of Fungal Strains</h3>
<p>Twelve fungal strains isolated from Chernobyl nuclear accident sites, belonging to nine genera, and one ISS-isolated strain were included in this study. Eight of the Chernobyl isolates were collected over time from the wall surface of the exploded Unit-4 of ChNPP. The other four fungi isolated from soil within the exclusion zone were included for comparison. The strains used in the study were selected based on their ecological and biological significance from the collection of over 2,000 isolates. Speciation was determined by classical cell and colony morphology-based identification techniques. A smaller set of fungi underwent whole genome sequencing. Names, GenBank accession numbers, and their significance are presented in <a class="usa-link" href="#T1">Table 1</a>.</p>
<section class="tw xbox font-sm" id="T1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Fungal isolates used in the study and their significance.</p></div>
<div class="tbl-box p" tabindex="0"><table cellpadding="5" cellspacing="5" class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="top">Strain number</th>
<th align="left" colspan="1" rowspan="1" valign="top">Species identification</th>
<th align="left" colspan="1" rowspan="1" valign="top">NCBI accession number</th>
<th align="left" colspan="1" rowspan="1" valign="top">Isolation site</th>
<th align="left" colspan="1" rowspan="1" valign="top">Significance</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 00034<sup>∗</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Cladosporium herbarum</em></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top">Wall surface, unit-4, ChNPP</td>
<td align="left" colspan="1" rowspan="1" valign="top">Exhibits radiotropism</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 00045<sup>∗</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Cladosporium sphaerospermum</em></td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MSJI00000000" rel="noopener noreferrer" target="_blank">MSJI00000000</a><sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">Wall surface, unit-4, ChNPP</td>
<td align="left" colspan="1" rowspan="1" valign="top">Exhibits radiotropism</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 00236<sup>∗</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Cladosporium cladosporioides</em></td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MSJH00000000" rel="noopener noreferrer" target="_blank">MSJH00000000</a><sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">Wall surface, unit-4, ChNPP</td>
<td align="left" colspan="1" rowspan="1" valign="top">Exhibits radiotropism</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 00253<sup>∗</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Acremonium murorum</em></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top">Soil, 10-km ChEZ</td>
<td align="left" colspan="1" rowspan="1" valign="top">Exhibits radiotropism</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 00265</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Beauveria bassiana</em></td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MSJG00000000" rel="noopener noreferrer" target="_blank">MSJG00000000</a><sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">Wall surface, unit-4, ChNPP</td>
<td align="left" colspan="1" rowspan="1" valign="top">Produces tenellin</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 00293</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Fusarium oxysporum</em></td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MSJJ00000000" rel="noopener noreferrer" target="_blank">MSJJ00000000</a><sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">Wall surface, unit-4, ChNPP</td>
<td align="left" colspan="1" rowspan="1" valign="top">Produces dihydronaphthoquinone</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 00454</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Trichoderma virens</em></td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MSJK00000000" rel="noopener noreferrer" target="_blank">MSJK00000000</a><sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">Soil, 10-km ChEZ</td>
<td align="left" colspan="1" rowspan="1" valign="top">Exhibits auxin-dependent lateral root growth/mycoparasitism</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 00738</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Penicillium citreonigrum</em></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top">Soil in the ternapol region</td>
<td align="left" colspan="1" rowspan="1" valign="top">European patent 2 333 088</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 00882</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Aureobasidium pullulans</em></td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MSJF00000000" rel="noopener noreferrer" target="_blank">MSJF00000000</a><sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">Wall surface, unit-4, ChNPP</td>
<td align="left" colspan="1" rowspan="1" valign="top">Produces diketopiperazine</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 01167</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Aspergillus terreus</em></td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MSJE00000000" rel="noopener noreferrer" target="_blank">MSJE00000000</a><sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">Soil, Kirovograd region</td>
<td align="left" colspan="1" rowspan="1" valign="top">Produces citreoviridin</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 01221</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Aspergillus sydowii</em></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top">Wall surface, unit-4, ChNPP</td>
<td align="left" colspan="1" rowspan="1" valign="top">Produces sesquiterpeonids</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 01851</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Apiospora montagnei</em></td>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<td align="left" colspan="1" rowspan="1" valign="top">Wall surface, unit-4, ChNPP</td>
<td align="left" colspan="1" rowspan="1" valign="top">Produces TMC-95A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">ISSFT-021</td>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Aspergillus fumigatus</em></td>
<td align="left" colspan="1" rowspan="1" valign="top">
<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/KT832787" rel="noopener noreferrer" target="_blank">KT832787</a><sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">ISS HEPA filter</td>
<td align="left" colspan="1" rowspan="1" valign="top">Opportunistic pathogen</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1" valign="top"></td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib1"><p><em>ChNPP, Chernobyl nuclear power plant; ChEZ, chernobyl exclusion zone. <sup><em>a</em></sup>whole genome sequencing. <sup>∗</sup>, radiotropism.</em></p></div></div></section></section><section id="sec4"><h3 class="pmc_sec_title">Survival of Desiccated Conidial Spores Under UV-C Irradiation</h3>
<p>The impact of UV-C irradiation on the survival of dried fungal conidia is presented in <a class="usa-link" href="#SM1">Supplementary Table 1</a>. Out of 13 irradiated strains, all but three (<em>Beauveria bassiana</em> IMV 00265, <em>Fusarium oxysporum</em> IMV 00293, and <em>Aureobasidium pullulans</em> IMV 00882) survived exposure to the dose of 2,000 J/m<sup>2</sup>. Two radiotropic strains, <em>Cladosporium sphaerospermum</em> IMV 00045, and <em>Cladosporium cladosporioides</em> IMV 00236, and the non-radiotropic <em>Penicillium citreonigrum</em> IMV 00738 showed survival at the level of 3.48, 3.60, and 2.18%, respectively, which was higher than the ∼0.1% survival rate observed for the other strains (<a class="usa-link" href="#F1">Figure 1</a> and <a class="usa-link" href="#SM1">Supplementary Table 1</a>). Rapid decrease in conidia survivability was observed at dose of 500 J/m<sup>2</sup> (∼2–3 log reduction). From that point until the doses of 1000 or 2000 J/m<sup>2</sup>, the decrease in survival was less pronounced.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6529585_fmicb-10-01013-g001.jpg" target="_blank"><img alt="FIGURE 1" class="graphic zoom-in" height="380" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/06c4/6529585/3d1d10a2695a/fmicb-10-01013-g001.jpg" width="716"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>UV-C resistance of Chernobyl and ISS-isolated fungal strains. Purified conidia of 13 strains were exposed to various UV-C doses. The UV-C survival rates were calculated using formula: N/N<sub>0</sub>, # of conidia survived at any given dose/# of conidia exposed at Time 0.</p></figcaption></figure></section><section id="sec5"><h3 class="pmc_sec_title">Survival of Desiccated Conidia Under Simulated Mars Conditions (SMC)</h3>
<p>Based on the initial analysis of survival under SMC (not shown) four fungal strains were exposed to SMC for 5 and 30 min at the Leiden Institute of Chemistry, Netherlands. All exposed strains, <em>C. cladosporioides</em> IMV 00236, <em>Apiospora montagnei</em> IMV 01851, <em>C. herbarum</em> IMV 00034 and <em>A. fumigatus</em> ISSFT-021, survived exposure to SMC for 5 min, whereas only two strains: <em>C. cladosporioides</em> IMV 00236 and <em>A. fumigatus</em> ISSFT-021 survived exposure for 30 min (<a class="usa-link" href="#T2">Table 2</a>). Probably the most striking observation was the highest survival rate of IMV 00034 after 5 min exposure to SMC followed by complete eradication after 30 min exposure to SMC. However, because the main focus of the manuscript was to discuss the potential of fungi to survive long-term exposure to SMC we did not further investigate why IMV 00034 may be highly tolerant to SMC-exposure for the short time, but it fails to survive during longer exposure. Nevertheless, it remains an interesting question and a topic for further investigation.</p>
<section class="tw xbox font-sm" id="T2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Quantitative analysis of the simulated Mars conditions (SMC) tolerance of selected extremotolerant Chernobyl- and ISS-isolated fungi.</p></div>
<div class="tbl-box p" tabindex="0"><table cellpadding="5" cellspacing="5" class="content" frame="hsides" rules="groups">
<thead><tr>
<td align="left" colspan="1" rowspan="1" valign="top"></td>
<th align="center" colspan="5" rowspan="1" valign="top">Growth after exposure of cultures to simulated Mars conditions (SMC)<sup>b</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"><strong>Strain<sup>a</sup></strong></td>
<td align="center" colspan="1" rowspan="1" valign="top"><strong>Unexposed</strong></td>
<td align="center" colspan="1" rowspan="1" valign="top"><strong>SMC – 5 min</strong></td>
<td align="center" colspan="1" rowspan="1" valign="top"><strong>SMC – 30 min</strong></td>
<td align="center" colspan="1" rowspan="1" valign="top"><strong>Percent of survival – 5 min exposure</strong></td>
<td align="center" colspan="1" rowspan="1" valign="top"><strong>Percent of survival – 30 min exposure</strong></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 00034<sup>∗</sup>
</td>
<td align="center" colspan="1" rowspan="1" valign="top">+</td>
<td align="center" colspan="1" rowspan="1" valign="top">+</td>
<td align="center" colspan="1" rowspan="1" valign="top">-</td>
<td align="center" colspan="1" rowspan="1" valign="top">21.11</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.00</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 00236<sup>∗</sup>
</td>
<td align="center" colspan="1" rowspan="1" valign="top">+</td>
<td align="center" colspan="1" rowspan="1" valign="top">+</td>
<td align="center" colspan="1" rowspan="1" valign="top">+</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.14</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.83</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">IMV 01851</td>
<td align="center" colspan="1" rowspan="1" valign="top">+</td>
<td align="center" colspan="1" rowspan="1" valign="top">+</td>
<td align="center" colspan="1" rowspan="1" valign="top">-</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.17</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.00</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">ISSFT-021</td>
<td align="center" colspan="1" rowspan="1" valign="top">+</td>
<td align="center" colspan="1" rowspan="1" valign="top">+</td>
<td align="center" colspan="1" rowspan="1" valign="top">+</td>
<td align="center" colspan="1" rowspan="1" valign="top">15.00</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.50</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1" valign="top"></td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib2"><p><em><sup><em>a</em></sup>IMV, Institute for Microbiology and Virology (academy of sciences), Kiyv, Ukraine. ISSFT, international space station filter; <sup>∗</sup>, radiotropism. <sup><em>b</em></sup>“+” growth after exposure. “-”, no growth after exposure.</em></p></div></div></section></section><section id="sec6"><h3 class="pmc_sec_title">Secondary Metabolite (SM) Profiling of <em>Aspergillus fumigatus</em> and <em>Cladosporium cladosporioides</em> Exposed to SMC</h3>
<p>Organic extracts of unexposed ISSFT-021 and IMV 00236, and 30 min SMC-exposed ISSFT-021-30 and IMV 00236-30 strains were examined to test if exposure to SMC alters SM production. No significant differences were observed in SM production when comparing samples before and after SMC exposure, including yield or type of compound produced, in either of the strains (<a class="usa-link" href="#F2">Figure 2</a>). However, there appeared to be a tendency of increased SM production yield in both strains following exposure to SMC when compared to unexposed counterparts (<a class="usa-link" href="#F2">Figure 2B,D</a>).</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6529585_fmicb-10-01013-g002.jpg" target="_blank"><img alt="FIGURE 2" class="graphic zoom-in" height="285" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/06c4/6529585/df7c248a0553/fmicb-10-01013-g002.jpg" width="708"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Secondary metabolite production of SMC-exposed ISSFT-021-30 and IMV 00236-30 when compared to unexposed ISSFT-021 and IMV 00236. <strong>(A)</strong> Secondary metabolite profiles of ISSFT-021-30 and ISSFT-021 when grown on GMM. <strong>(B)</strong> Secondary metabolite profiles of IMV 00236-30 and IMV 00236 when grown on MEA. <strong>(C)</strong> Metabolite quantification, showing the percent change for each metabolite in relation to unexposed ISSFT-021; significance was determined using Welch’s <em>t</em>-test. <strong>(D)</strong> Metabolite quantification, showing the percent change for each metabolite in relation to unexposed IMV 00236; significance was determined using Welch’s <em>t</em>-test. <sup>∗</sup>, radiotropism.</p></figcaption></figure></section><section id="sec7"><h3 class="pmc_sec_title">Proteome Profiling of <em>Aspergillus fumigatus</em> Exposed to SMC</h3>
<p>The proteomic characterization of SMC-exposed ISSFT-021-30 revealed 51-up and 24 down-regulated proteins when compared to unexposed ISSFT-021 (fold-change (FC) &gt; | 2|, <em>P</em> &lt; 0.05) (<a class="usa-link" href="#SM1">Supplementary Table 2</a>). Analysis of the distribution of differentially expressed proteins among biological processes revealed that 27 proteins were involved in translation and ribosome biogenesis, 11 in carbohydrate metabolism, and 10 in stress response (<a class="usa-link" href="#F3">Figure 3</a>). Further, significantly over-represented up-regulated biological processes included translation (50% of all up-regulated proteins), and carbohydrate metabolic processes (15%), whereas significantly over-represented down-regulated processes included carbohydrate derivative metabolic processes (12%) (<a class="usa-link" href="#SM1">Supplementary Table 3</a>).</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6529585_fmicb-10-01013-g003.jpg" target="_blank"><img alt="FIGURE 3" class="graphic zoom-in" height="533" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/06c4/6529585/e70e5a9bf9a6/fmicb-10-01013-g003.jpg" width="724"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>AspGD GO Slim terms of differentially expressed proteins in ISSFT-021-30. Differentially expressed proteins in (FC &gt; |2|, <em>P</em> &lt; 0.05) were mapped to terms representing various biological processes using AspGD gene ontology (GO) Slim Mapper.</p></figcaption></figure><p>Approximately 50% of all up-regulated proteins in ISSFT-021-30 were involved in translation and ribosome biogenesis, including proteins that comprise the small and large ribosomal subunit (<a class="usa-link" href="#T3">Table 3</a>). The majority of these proteins were reported to be differentially expressed during the early development of <em>A. fumigatus</em> (<a aria-describedby="B13" class="usa-link" href="#B13">Cagas et al., 2011</a>). Both Rpl3 (AFUA_2G11850) and ribosomal protein P0 (AFUA_1G05080), whose expression during conidiation are regulated by BrlAp, were more than twofold up-regulated (<a aria-describedby="B72" class="usa-link" href="#B72">Twumasi-Boateng et al., 2009</a>). A number of differentially expressed proteins were involved in carbohydrate metabolism (<a class="usa-link" href="#T4">Table 4</a>), including isocitrate lyase AcuD (AFUA_4G13510), one of the key enzymes in the glyoxylate cycle (<a aria-describedby="B50" class="usa-link" href="#B50">Olivas et al., 2008</a>). Cellobiohydrolases CbhB (AFUA_6G11610) and AFUA_8G01490, both involved in cellulose degradation, and hydrolase AFUA_5G07080 were more than twofold up-regulated in SMC-exposed ISSFT-021-30 (<a aria-describedby="B1" class="usa-link" href="#B1">Adav et al., 2013</a>). A twofold increase in protein abundance was observed for glycerol dehydrogenase GldB (AFUA_4G11730) (<a aria-describedby="B70" class="usa-link" href="#B70">Teutschbein et al., 2010</a>), phosphoglycerate kinase (AFUA_1G10350), and hexokinase HxkA (AFUA_2G05910) (<a aria-describedby="B2" class="usa-link" href="#B2">Albrecht et al., 2010</a>). Among the proteins involved in carbohydrate metabolism that exhibited decreased abundance were malate and alcohol dehydrogenases [AFUA_6G05210 (<a aria-describedby="B70" class="usa-link" href="#B70">Teutschbein et al., 2010</a>), AFUA_5G06240 (<a aria-describedby="B13" class="usa-link" href="#B13">Cagas et al., 2011</a>), respectively] and mannose-6-phosphate isomerase (AFUA_4G08410) (<a aria-describedby="B27" class="usa-link" href="#B27">Gautam et al., 2011</a>). Several proteins with increased abundance were involved in response to stress (<a class="usa-link" href="#T5">Table 5</a>). Dehydrin-like protein DprC (AFUA_7G04520), which is known to play a role in protecting cells against freezing (<a aria-describedby="B33" class="usa-link" href="#B33">Hoi et al., 2011</a>), and AFUA_1G14090, which is predicted to be involved in histidine biosynthesis (<a aria-describedby="B49" class="usa-link" href="#B49">Nierman et al., 2005</a>), were twofold up-regulated. Down-regulated stress response proteins included the proliferating cell nuclear antigen (PCNA) (AFUA_1G04900), and the formaldehyde dehydrogenase (AFUA_2G01040), and AFUA_8G04890 with a predicted role in response to salt stress (<a aria-describedby="B49" class="usa-link" href="#B49">Nierman et al., 2005</a>).</p>
<section class="tw xbox font-sm" id="T3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Differentially expressed proteins involved in translation and ribosome biogenesis in ISSFT-021-30 subjected to SMC.</p></div>
<div class="tbl-box p" tabindex="0"><table cellpadding="5" cellspacing="5" class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="top">ORF</th>
<th align="center" colspan="1" rowspan="1" valign="top">Protein</th>
<th align="center" colspan="1" rowspan="1" valign="top">Relative protein abundance<sup>∗</sup>
</th>
<th align="center" colspan="1" rowspan="1" valign="top">
<em>P</em>-value</th>
<th align="left" colspan="1" rowspan="1" valign="top">Putative function/activity</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_5G05630</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">2.15</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.19E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L23</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_6G05200</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">2.01</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.67E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L28</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_4G03880</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.66</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.86E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_4G07435</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.60</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.68E-04</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L36</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_5G06360</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.59</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.46E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_2G03380</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.58</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.37E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">large ribosomal subunit</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_4G07730</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.56</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.32E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G03390</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.54</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.06E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G09100</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.47</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.38E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_6G11260</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.46</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.21E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ribosomal protein L26</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_5G03020</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.43</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.84E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_2G11850</td>
<td align="center" colspan="1" rowspan="1" valign="top">Rpl3</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.39</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.36E-04</td>
<td align="left" colspan="1" rowspan="1" valign="top">Allergenic ribosomal L3 protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_2G16370</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.39</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.93E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L32</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G14410</td>
<td align="center" colspan="1" rowspan="1" valign="top">Rpl17</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.36</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.82E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L17</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_2G09210</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.34</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.17E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L10</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_2G03040</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.34</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.84E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ribosomal protein L34</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_3G06760</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.32</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.48E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ribosomal protein L37</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G05080</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.27</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.66E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein P0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_4G04460</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.25</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.15E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L13</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_3G13480</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.22</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.30E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">Translation initiation factor 2 alpha subunit</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_6G03830</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.22</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.22E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ribosomal protein L14</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_6G12660</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.18</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.46E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">40S ribosomal protein S10b</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G11130</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.18</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.20E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G12890</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.17</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.89E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_2G09200</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.10</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.83E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L30</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_2G16010</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.07</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.97E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">Prolyl-tRNA synthetase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_2G03590</td>
<td align="center" colspan="1" rowspan="1" valign="top">Rps21</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.05</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.30E-04</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ribosomal protein S21e</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1" valign="top"></td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib3"><p><em><sup>∗</sup>Log2 fold change of ISSFT-021-30 compared to ISSFT-021 (<em>P</em> &lt; 0.05).</em></p></div></div></section><section class="tw xbox font-sm" id="T4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Differentially expressed proteins involved in carbohydrate metabolism in ISSFT-021-30 subjected to SMC.</p></div>
<div class="tbl-box p" tabindex="0"><table cellpadding="5" cellspacing="5" class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="top">ORF</th>
<th align="center" colspan="1" rowspan="1" valign="top">Protein</th>
<th align="center" colspan="1" rowspan="1" valign="top">Relative protein abundance<sup>∗</sup>
</th>
<th align="center" colspan="1" rowspan="1" valign="top">
<em>P</em>-value</th>
<th align="left" colspan="1" rowspan="1" valign="top">Putative function/activity</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_4G13510</td>
<td align="center" colspan="1" rowspan="1" valign="top">AcuD/Icl1</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.78</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.60E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">Isocitrate lyase involved in the glyoxylate cycle</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_6G11610</td>
<td align="center" colspan="1" rowspan="1" valign="top">CbhB</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.42</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.85E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cellobiohydrolase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_5G07080</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.39</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.37E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">Hydrolase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_8G01490</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.32</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.64E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cellobiohydrolase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_4G11730</td>
<td align="center" colspan="1" rowspan="1" valign="top">GldB</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.08</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.70E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glycerol dehydrogenase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G10350</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.03</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.74E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">Phosphoglycerate kinase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_2G05910</td>
<td align="center" colspan="1" rowspan="1" valign="top">HxkA</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.01</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.07E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">Hexokinase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G06960</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.00</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.76E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">Pyruvate dehydrogenase complex subunit alpha</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_6G05210</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.18</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.36E-04</td>
<td align="left" colspan="1" rowspan="1" valign="top">Malate dehydrogenase involved in the citric acid cycle</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_5G06240</td>
<td align="center" colspan="1" rowspan="1" valign="top">AlcC</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.22</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.83E-04</td>
<td align="left" colspan="1" rowspan="1" valign="top">Alcohol dehydrogenase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_4G08410</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">-2.23</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.53E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">Mannose-6-phosphate isomerase</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1" valign="top"></td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib4"><p><em><sup>∗</sup>Log2 fold change of ISSFT-021-30 compared to ISSFT-021 (<em>P</em> &lt; 0.05).</em></p></div></div></section><section class="tw xbox font-sm" id="T5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Differentially expressed proteins involved in response to stress in ISSFT-021-30 subjected to SMC.</p></div>
<div class="tbl-box p" tabindex="0"><table cellpadding="5" cellspacing="5" class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="top">ORF</th>
<th align="center" colspan="1" rowspan="1" valign="top">Protein</th>
<th align="center" colspan="1" rowspan="1" valign="top">Relative protein abundance<sup>∗</sup>
</th>
<th align="center" colspan="2" rowspan="1" valign="top">
<em>P</em>-value</th>
<th align="left" colspan="1" rowspan="1" valign="top">Putative function/activity</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G14410</td>
<td align="center" colspan="1" rowspan="1" valign="top">Rpl17</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.36</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.82E-03</td>
<td align="left" colspan="2" rowspan="1" valign="top">60S ribosomal protein L17</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_3G13480</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.22</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.30E-02</td>
<td align="left" colspan="2" rowspan="1" valign="top">Translation initiation factor 2 alpha subunit</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_7G04520</td>
<td align="center" colspan="1" rowspan="1" valign="top">DprC</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.21</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.74E-03</td>
<td align="left" colspan="2" rowspan="1" valign="top">Dehydrin-like protein, acts downstream of SakA to confer cold tolerance</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G11130</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.18</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.20E-02</td>
<td align="left" colspan="2" rowspan="1" valign="top">60S ribosomal protein L6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G14090</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">1.15</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.56E-03</td>
<td align="left" colspan="2" rowspan="1" valign="top">Histidine biosynthesis</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_4G11730</td>
<td align="center" colspan="1" rowspan="1" valign="top">GldB</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.08</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.70E-02</td>
<td align="left" colspan="2" rowspan="1" valign="top">Glycerol dehydrogenase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_8G04890</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.21</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.06E-02</td>
<td align="left" colspan="2" rowspan="1" valign="top">Role in response to salt stress</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G15450</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.31</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.62E-03</td>
<td align="left" colspan="2" rowspan="1" valign="top">Adenylosuccinate synthase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_1G04900</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.57</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.26E-03</td>
<td align="left" colspan="2" rowspan="1" valign="top">Proliferating cell nuclear antigen (PCNA)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">AFUA_2G01040</td>
<td align="center" colspan="1" rowspan="1" valign="top"></td>
<td align="center" colspan="1" rowspan="1" valign="top">-2.67</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.06E-04</td>
<td align="left" colspan="2" rowspan="1" valign="top">Formaldehyde dehydrogenase</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1" valign="top"></td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib5"><p><em><sup>∗</sup>Log2 fold change of ISSFT-021-30 compared to ISSFT-021 (<em>P</em> &lt; 0.05).</em></p></div></div></section></section><section id="sec8"><h3 class="pmc_sec_title">Proteome Profiling of <em>Cladosporium cladosporioides</em> Exposed to SMC</h3>
<p>The proteomic characterization of <em>C. cladosporioides</em> upon exposure to SMC for 30 min revealed that 51 proteins were up-regulated, and 218 proteins were down-regulated when compared to unexposed IMV 00236 (fold-change (FC) &gt; ∣2∣, <em>P</em> &lt; 0.05) in response to SMC (<a class="usa-link" href="#SM1">Supplementary Table 4</a>). The distribution of differentially expressed proteins in SMC-exposed IMV 00236-30 among biological processes is shown in <a class="usa-link" href="#F4">Figure 4</a>. Among differentially expressed proteins 22 were involved in post-translation modification, protein turnover, and chaperones, 21 in carbohydrate transport and metabolism, 20 in energy production and conversion, and 17 in translation and ribosomal structure and biogenesis. Interestingly, the majority of the proteins involved in the translation and ribosomal structure and biogenesis in SMC-exposed IMV 00236-30 exhibited down-regulation (<a class="usa-link" href="#T6">Table 6</a>), which is the opposite expression pattern to ISSFT-021-30. Additionally, a number of proteins involved in post-translational modification and chaperones (<a class="usa-link" href="#T7">Table 7</a>) showed decreased abundance, including an aspartic endopeptidase (BS090_008183/ENOG410PH8I), which is an ortholog of <em>A. fumigatus</em> AFUA_5G13300 (<a aria-describedby="B49" class="usa-link" href="#B49">Nierman et al., 2005</a>). BS090_010805/ENOG410PMR5, an ortholog of mitochondrial matrix cochaperone Mge1p (YOR232W) in <em>Saccharomyces cerevisiae</em> (<a aria-describedby="B16" class="usa-link" href="#B16">Costanzo et al., 2010</a>) was fourfold down-regulated when compared to SMC-unexposed IMV 00236. Proteins involved in carbohydrate metabolism displayed differential abundance (<a class="usa-link" href="#T8">Table 8</a>), including the fourfold up-regulated exo-polygalacturonase involved in pectin degradation BS090_001871/ENOG410PG7M, which is an ortholog of An12g07500 in <em>Aspergillus niger</em> (<a aria-describedby="B42" class="usa-link" href="#B42">Martens-Uzunova et al., 2006</a>). Chitin deacetylases BS090_000013 and BS090_000044/ENOG410PMJX, and the chitin recognition protein BS090_010953/ENOG410PMF7 were at least threefold up-regulated. Down-regulated proteins involved in carbohydrate metabolism included phosphoglycerate mutase BS090_004087/ENOG410QEDC, which is an ortholog of <em>Aspergillus nidulans</em> AN8720 with a predicted role in gluconeogenesis and glycolysis (<a aria-describedby="B43" class="usa-link" href="#B43">Masuo et al., 2010</a>), glucanase BS090_003291/ENOG410PM6H, and alpha-amylase BS090_011829/ENOG410PMDW. Differentially expressed proteins involved in energy production and conversion (<a class="usa-link" href="#T9">Table 9</a>) included up-regulated BS090_001715/ENOG410PGTG, an ortholog of <em>A. niger</em> NADPH dehydrogenase (An11g08510), and isocytrate lyase and dehydrogenase BS090_001881/ENOG410PGND, and BS090_000939/ENOG410PFHR, respectively. Proteins with decreased abundance included nitrate reductase BS090_004112/ENOG410PUCE and ATP synthase BS090_005644/ENOG41KOG1758.</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6529585_fmicb-10-01013-g004.jpg" target="_blank"><img alt="FIGURE 4" class="graphic zoom-in" height="643" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/06c4/6529585/7c1caf9a469e/fmicb-10-01013-g004.jpg" width="660"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Biological process COG categories of differentially expressed proteins in IMV 00236-30. Differentially abundant proteins (FC &gt; |2|, <em>P</em> &lt; 0.05) were mapped to terms representing various biological processes using cluster of orthologous genes (COG) database in CloVR.</p></figcaption></figure><section class="tw xbox font-sm" id="T6"><h4 class="obj_head">Table 6.</h4>
<div class="caption p"><p>Differentially expressed proteins involved in translation, ribosomal structure, and biogenesis in IMV 00236-30 subjected to SMC.</p></div>
<div class="tbl-box p" tabindex="0"><table cellpadding="5" cellspacing="5" class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="top">Accession</th>
<th align="center" colspan="1" rowspan="1" valign="top">Relative protein abundance<sup>∗</sup>
</th>
<th align="center" colspan="1" rowspan="1" valign="top">
<em>P</em>-value</th>
<th align="left" colspan="1" rowspan="1" valign="top">EggNog</th>
<th align="left" colspan="1" rowspan="1" valign="top">EggNog annotation</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_000406</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.44</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.67E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PN3M</td>
<td align="left" colspan="1" rowspan="1" valign="top">40S ribosomal protein S12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_001853</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.13</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.89E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PG8Y</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ribosomal protein L15</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_003273</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.03</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.19E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PN86</td>
<td align="left" colspan="1" rowspan="1" valign="top">La domain</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_000015</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.12</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.60E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PMV6</td>
<td align="left" colspan="1" rowspan="1" valign="top">40s ribosomal protein S17</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_011340</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.16</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.27E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PNPS</td>
<td align="left" colspan="1" rowspan="1" valign="top">Translation initiation factor</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_009099</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.18</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.88E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PGC2</td>
<td align="left" colspan="1" rowspan="1" valign="top">Eukaryotic translation initiation factor 5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_002173</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.23</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.84E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PP8V</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L31</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_005915</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.24</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.15E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PHV2</td>
<td align="left" colspan="1" rowspan="1" valign="top">Prolyl-tRNA synthetase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_006863</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.35</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.19E-04</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PPAS</td>
<td align="left" colspan="1" rowspan="1" valign="top">60s ribosomal protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_002466</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.35</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.65E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PQS0</td>
<td align="left" colspan="1" rowspan="1" valign="top">Processing of the 20S rRNA-precursor to mature 18S rRNA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_003537</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.35</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.24E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PI34</td>
<td align="left" colspan="1" rowspan="1" valign="top">Component of the eukaryotic translation initiation factor 3 (eIF-3) complex</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_000480</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.48</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.36E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PFEB</td>
<td align="left" colspan="1" rowspan="1" valign="top">Seryl-tRNA synthetase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_001464</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.53</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.78E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PRUB</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S acidic ribosomal protein P2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_006330</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.65</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.30E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PQ50</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ribosome biogenesis protein Nhp2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_009967</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.81</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.19E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PRWG</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S acidic ribosomal protein P1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_006228</td>
<td align="center" colspan="1" rowspan="1" valign="top">-2.13</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.69E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PP4P</td>
<td align="left" colspan="1" rowspan="1" valign="top">L-PSP endoribonuclease family protein (Hmf1)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_007862</td>
<td align="center" colspan="1" rowspan="1" valign="top">-2.52</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.56E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PQSK</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L22</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1" valign="top"></td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib6"><p><em><sup>∗</sup>Log2 fold change of IMV 00236-30 compared to IMV 00236 (<em>P</em> &lt; 0.05).</em></p></div></div></section><section class="tw xbox font-sm" id="T7"><h4 class="obj_head">Table 7.</h4>
<div class="caption p"><p>Differentially expressed proteins involved in post-translational modification, protein turnover, and chaperones in IMV 00236-30 subjected to SMC.</p></div>
<div class="tbl-box p" tabindex="0"><table cellpadding="5" cellspacing="5" class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="top">Accession</th>
<th align="center" colspan="1" rowspan="1" valign="top">Relative protein abundance<sup>∗</sup>
</th>
<th align="center" colspan="1" rowspan="1" valign="top">
<em>P</em>-value</th>
<th align="left" colspan="1" rowspan="1" valign="top">EggNog</th>
<th align="left" colspan="1" rowspan="1" valign="top">EggNog annotation</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_010341</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.55</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.80E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PJAB</td>
<td align="left" colspan="1" rowspan="1" valign="top">Thioredoxin reductase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_002416</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.42</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.68E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG41KOG1339</td>
<td align="left" colspan="1" rowspan="1" valign="top">Aspartic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_010922</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.12</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.34E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PX4S</td>
<td align="left" colspan="1" rowspan="1" valign="top">OsmC-like protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_011023</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.01</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.52E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PNQ9</td>
<td align="left" colspan="1" rowspan="1" valign="top">Peptidyl prolyl cis-trans isomerase Cyclophilin</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_008028</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.12</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.88E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PPYQ</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ubiquitin conjugating enzyme</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_005834</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.20</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.64E-04</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PJ50</td>
<td align="left" colspan="1" rowspan="1" valign="top">26S proteasome non-ATPase regulatory subunit 11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_010452</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.24</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.83E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PQY3</td>
<td align="left" colspan="1" rowspan="1" valign="top">Peptidyl-prolyl cis-trans isomerase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_010972</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.28</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.67E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PP80</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cupin domain protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_011316</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.32</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.31E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PKHZ</td>
<td align="left" colspan="1" rowspan="1" valign="top">Protein-L-isoaspartate O-methyltransferase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_005149</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.33</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.63E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PP3T</td>
<td align="left" colspan="1" rowspan="1" valign="top">Subunit 3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_009030</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.34</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.91E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PP19</td>
<td align="left" colspan="1" rowspan="1" valign="top">Peptidyl-prolyl cis-trans isomerase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_004399</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.59</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.87E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PPJH</td>
<td align="left" colspan="1" rowspan="1" valign="top">Heat shock protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_007304</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.63</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.98E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PHFF</td>
<td align="left" colspan="1" rowspan="1" valign="top">Protease S8 tripeptidyl peptidase I</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_009384</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.63</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.01E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PI5I</td>
<td align="left" colspan="1" rowspan="1" valign="top">Tripeptidyl-peptidase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_008147</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.81</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.06E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG41KOG0541</td>
<td align="left" colspan="1" rowspan="1" valign="top">Peroxiredoxin</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_005718</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.90</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.72E-04</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PGPE</td>
<td align="left" colspan="1" rowspan="1" valign="top">Disulfide-isomerase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_010805</td>
<td align="center" colspan="1" rowspan="1" valign="top">-2.03</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.03E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PMR5</td>
<td align="left" colspan="1" rowspan="1" valign="top">Component of the PAM complex</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_010009</td>
<td align="center" colspan="1" rowspan="1" valign="top">-2.07</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.52E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PQV2</td>
<td align="left" colspan="1" rowspan="1" valign="top">Prefoldin subunit 6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_008183</td>
<td align="center" colspan="1" rowspan="1" valign="top">-2.19</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.69E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PH8I</td>
<td align="left" colspan="1" rowspan="1" valign="top">Aspartic endopeptidase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_009852</td>
<td align="center" colspan="1" rowspan="1" valign="top">-2.47</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.17E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PSDM</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glutaredoxin</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_008141</td>
<td align="center" colspan="1" rowspan="1" valign="top">-3.13</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.96E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PRTR</td>
<td align="left" colspan="1" rowspan="1" valign="top">Heat shock protein</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1" valign="top"></td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T7/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib7"><p><em><sup>∗</sup>Log2 fold change of IMV 00236-30 compared to IMV 00236 (<em>P</em> &lt; 0.05).</em></p></div></div></section><section class="tw xbox font-sm" id="T8"><h4 class="obj_head">Table 8.</h4>
<div class="caption p"><p>Differentially expressed proteins involved in carbohydrate transport and metabolism in IMV 00236-30 subjected to SMC.</p></div>
<div class="tbl-box p" tabindex="0"><table cellpadding="5" cellspacing="5" class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="top">Accession</th>
<th align="center" colspan="1" rowspan="1" valign="top">Relative protein abundance<sup>∗</sup>
</th>
<th align="center" colspan="1" rowspan="1" valign="top">
<em>P</em>-value</th>
<th align="left" colspan="1" rowspan="1" valign="top">EggNog</th>
<th align="left" colspan="1" rowspan="1" valign="top">EggNog annotation</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_010953</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.31</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.94E-04</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PMF7</td>
<td align="left" colspan="1" rowspan="1" valign="top">Chitin recognition protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_001871</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.14</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.38E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PG7M</td>
<td align="left" colspan="1" rowspan="1" valign="top">Exo-polygalacturonase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_000013</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.95</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.89E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PMJX</td>
<td align="left" colspan="1" rowspan="1" valign="top">Chitin deacetylase-like mannoprotein MP98</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_003824</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.93</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.34E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PMRY</td>
<td align="left" colspan="1" rowspan="1" valign="top">LysM domain</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_001404</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.67</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.92E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG41KOG1458</td>
<td align="left" colspan="1" rowspan="1" valign="top">Fructose-1,6-bisphosphatase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_000044</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.58</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.22E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PMJX</td>
<td align="left" colspan="1" rowspan="1" valign="top">Chitin deacetylase-like mannoprotein MP98</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_000502</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.28</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.07E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PJKF</td>
<td align="left" colspan="1" rowspan="1" valign="top">Catalyzes the epimerization of the S- and R-forms of NAD(P)HX</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_011896</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.21</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.04E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PK51</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glyco_18</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_010859</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.09</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.80E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PGWP</td>
<td align="left" colspan="1" rowspan="1" valign="top">Mannose-6-phosphate isomerase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_011008</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.10</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.27E-04</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PG84</td>
<td align="left" colspan="1" rowspan="1" valign="top">Beta-glucosidase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_007695</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.13</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.36E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PJ6P</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glucan 1,4-alpha-glucosidase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_008700</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.16</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.40E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PK8I</td>
<td align="left" colspan="1" rowspan="1" valign="top">WSC domain</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_006133</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.17</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.18E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PF9K</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glyceraldehyde-3-phosphate dehydrogenase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_002903</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.37</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.43E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PJIN</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glycolipid transfer protein HET-C2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_003039</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.47</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.13E-04</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PH6W</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cell wall</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_008425</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.49</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.12E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PIQS</td>
<td align="left" colspan="1" rowspan="1" valign="top">Snf1 kinase complex beta-subunit Gal83</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_003291</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.51</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.22E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PM6H</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glucanase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_011829</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.66</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.43E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PMDW</td>
<td align="left" colspan="1" rowspan="1" valign="top">Alpha-amylase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_011280</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.91</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.40E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PKN9</td>
<td align="left" colspan="1" rowspan="1" valign="top">Oxalate decarboxylase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_004931</td>
<td align="center" colspan="1" rowspan="1" valign="top">-2.02</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.61E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PM9J</td>
<td align="left" colspan="1" rowspan="1" valign="top">Major intrinsic protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_004087</td>
<td align="center" colspan="1" rowspan="1" valign="top">-2.34</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.93E-05</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410QEDC</td>
<td align="left" colspan="1" rowspan="1" valign="top">Phosphoglycerate mutase</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1" valign="top"></td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T8/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib8"><p><em><sup>∗</sup>Log2 fold change of IMV 00236-30 compared to IMV 00236 (<em>P</em> &lt; 0.05).</em></p></div></div></section><section class="tw xbox font-sm" id="T9"><h4 class="obj_head">Table 9.</h4>
<div class="caption p"><p>Differentially expressed proteins involved in energy production and conversion in IMV 00236-30 subjected to SMC.</p></div>
<div class="tbl-box p" tabindex="0"><table cellpadding="5" cellspacing="5" class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="top">Accession</th>
<th align="center" colspan="1" rowspan="1" valign="top">Relative protein abundance<sup>∗</sup>
</th>
<th align="center" colspan="1" rowspan="1" valign="top">
<em>P</em>-value</th>
<th align="left" colspan="1" rowspan="1" valign="top">EggNog</th>
<th align="left" colspan="1" rowspan="1" valign="top">EggNog annotation</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_001053</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.45</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.24E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PGVS</td>
<td align="left" colspan="1" rowspan="1" valign="top">Oxidoreductase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_001715</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.25</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.00E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PGTG</td>
<td align="left" colspan="1" rowspan="1" valign="top">NADH flavin oxidoreductase NADH oxidase family protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_001881</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.23</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.71E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PGND</td>
<td align="left" colspan="1" rowspan="1" valign="top">Isocitrate lyase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_000071</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.16</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.95E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PFIA</td>
<td align="left" colspan="1" rowspan="1" valign="top">Component of the ubiquinol-cytochrome c reductase complex</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_007434</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.04</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.25E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PI78</td>
<td align="left" colspan="1" rowspan="1" valign="top">Phosphoenolpyruvate carboxykinase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_000939</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.03</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.87E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PFHR</td>
<td align="left" colspan="1" rowspan="1" valign="top">Isocitrate dehydrogenase NADP</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_000792</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.02</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.27E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PN6K</td>
<td align="left" colspan="1" rowspan="1" valign="top">Mitochondrial membrane ATP synthase [F(1)F(0) ATP synthase or complex V]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_004086</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.11</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.06E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PFM5</td>
<td align="left" colspan="1" rowspan="1" valign="top">Inorganic pyrophosphatase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_011745</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.14</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.04E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PNH4</td>
<td align="left" colspan="1" rowspan="1" valign="top">Conserved hypothetical protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_003155</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.19</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.65E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PNBT</td>
<td align="left" colspan="1" rowspan="1" valign="top">Regulatory protein SUAPRGA1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_010935</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.23</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.25E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PFFW</td>
<td align="left" colspan="1" rowspan="1" valign="top">Electron transfer flavoprotein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_011347</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.26</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.29E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PNPT</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cytochrome c oxidase polypeptide VIa</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_003274</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.27</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.98E-05</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PFBB</td>
<td align="left" colspan="1" rowspan="1" valign="top">Stomatin family</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_005666</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.28</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.59E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PH2F</td>
<td align="left" colspan="1" rowspan="1" valign="top">Mitochondrial membrane ATP synthase [F(1)F(0) ATP synthase or Complex V]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_006999</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.33</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.00E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PFI6</td>
<td align="left" colspan="1" rowspan="1" valign="top">Electron transfer flavoprotein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_004112</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.51</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.16E-02</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PUCE</td>
<td align="left" colspan="1" rowspan="1" valign="top">Nitrate reductase</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_011473</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.53</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.63E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PJA9</td>
<td align="left" colspan="1" rowspan="1" valign="top">Vacuolar ATP synthase subunit e</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_008223</td>
<td align="center" colspan="1" rowspan="1" valign="top">-1.59</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.26E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PNQY</td>
<td align="left" colspan="1" rowspan="1" valign="top">Iron sulfur cluster assembly protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_008509</td>
<td align="center" colspan="1" rowspan="1" valign="top">-2.04</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.62E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG410PS16</td>
<td align="left" colspan="1" rowspan="1" valign="top">Mitochondrial ATP synthase epsilon chain domain-containing protein</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">BS090_005644</td>
<td align="center" colspan="1" rowspan="1" valign="top">-2.40</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.70E-03</td>
<td align="left" colspan="1" rowspan="1" valign="top">ENOG41KOG1758</td>
<td align="left" colspan="1" rowspan="1" valign="top">ATP synthase</td>
</tr>
<tr><td align="left" colspan="1" rowspan="1" valign="top"></td></tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T9/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib9"><p><em><sup>∗</sup>Log2 fold change of IMV 00236-30 compared to IMV 00236 (<em>P</em> &lt; 0.05).</em></p></div></div></section></section><section id="sec9"><h3 class="pmc_sec_title">Increased Resistance to UV-C of SMC Exposed <em>Aspergillus fumigatus</em> Conidia</h3>
<p>Survival rates of several ISS-isolated and clinical isolates of <em>A. fumigatus</em> following exposure to UV-C are presented in <a class="usa-link" href="#F5">Figure 5</a>. SMC exposed ISSFT-021-30 exhibited increased UV-C resistance (∼20% of conidia survived the UV-C dose of 4,000 J/m<sup>2</sup>) when compared to unexposed ISSFT-021 and another <em>A. fumigatus</em> ISS isolate IF1SW-F4. Additionally, IF1SW-F4 was more resistant to UV-C exposure, than ISSFT-021 and clinical CEA10 strain, which displayed similar resistance patterns. The clinical isolate Af293 was the most sensitive, showing 2-log reduction when exposed to the highest tested UV-C dose. Exposure experiments were repeated 3 times and showed the same trends.</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6529585_fmicb-10-01013-g005.jpg" target="_blank"><img alt="FIGURE 5" class="graphic zoom-in" height="449" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/06c4/6529585/6e4c60e064c3/fmicb-10-01013-g005.jpg" width="706"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>UV-C resistance of <em>A. fumigatus</em> ISS-isolated and clinical strains. Purified conidia of ISS-isolated (ISSFT-021 and IF1SW-F4) SMC-exposed (ISSFT-021-30), and clinical isolates (Af293 and CEA10) were exposed to varying doses of UV-C. The UV-C survival rates were calculated using formula: N/N<sub>0</sub>, # of conidia survived at any given dose/# of conidia exposed at Time 0. The average fungal conidia survival rates from three different experiments are plotted.</p></figcaption></figure></section></section><section id="sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p>Although it has been well documented that bacteria are associated with the spacecraft environment (<a aria-describedby="B39" class="usa-link" href="#B39">La Duc et al., 2003</a>, <a aria-describedby="B38" class="usa-link" href="#B38">2004</a>; <a aria-describedby="B46" class="usa-link" href="#B46">Newcombe et al., 2005</a>), few studies address the persistence of fungi in this environment (<a aria-describedby="B40" class="usa-link" href="#B40">La Duc et al., 2012</a>; <a aria-describedby="B74" class="usa-link" href="#B74">Weinmaier et al., 2015</a>). Prepared in monolayers, spacecraft-associated spore-forming bacteria have been shown to survive exposure to UV-C and SMC (<a aria-describedby="B46" class="usa-link" href="#B46">Newcombe et al., 2005</a>; <a aria-describedby="B55" class="usa-link" href="#B55">Osman et al., 2008</a>), but similar studies have not been performed for fungal conidia. Microorganisms exposed as multilayers are shielded from UV penetration (submicron level) and subsequent UV-damage is prevented, therefore generation of monolayers of tested cells/spores/conidia is crucial for characterizing irradiation-induced microbial lethality. The observed strain-dependent UV-C sensitivity of fungal conidia was not surprising, as similar conclusions were drawn for UV-exposed bacterial species (<a aria-describedby="B55" class="usa-link" href="#B55">Osman et al., 2008</a>). Interestingly, ten out of thirteen selected fungal species survived the UV-C dose of 2,000 J/m<sup>2</sup>, while bacterial species in a similar study survived exposure to 1,000 J/m<sup>2</sup> (<a aria-describedby="B55" class="usa-link" href="#B55">Osman et al., 2008</a>), suggesting that some extremophilic fungi are as hardy, if not more, than tested bacteria. It should be noted, that conducting experiments that investigate the protective nature of fungi or bacteria could help in understanding the microbial survival in extreme niches, as bacteria and fungi coexist in the environment, and form communities to survive harsh conditions.</p>
<p>The space environment varies significantly from Earth. It is characterized by enhanced irradiation and distinct atmospheric conditions. Therefore, it was imperative to assess fungal survival under SMC, as fungi are known to be present during manned space missions. Among the four strains that survived 5-min exposure to SMC two, ISSFT-021 and IMV 00236, survived exposure to SMC for 30 min. Most studies testing survival of microorganisms under SMC have been conducted using bacterial spores or fungal communities. In one such study several <em>Bacillus</em> spp. were tested for simulated Mars UV irradiation tolerance, which resulted in no survival beyond 30 min exposure for all strains except <em>B. pumilus</em> SAFR-032, which was inactivated after 180 min of exposure (<a aria-describedby="B62" class="usa-link" href="#B62">Schuerger et al., 2006</a>). However, when <em>B. subtilis</em> spores were exposed to SMC, including irradiation and atmospheric conditions, 99.9% of spores were eradicated within 30 s, and a 15 min exposure resulted in no recovery of viable spores from aluminum coupons (<a aria-describedby="B61" class="usa-link" href="#B61">Schuerger et al., 2003</a>). Interestingly, when the more extremotolerant <em>B. pumilus</em> SAFR-032 was tested under SMC, no spores were recovered from the aluminum coupon after 30 min exposure, but bacterial growth was observed once coupons were placed in tryptic soy broth (TSB) (<a aria-describedby="B55" class="usa-link" href="#B55">Osman et al., 2008</a>). In this study, recovery of fungal conidia exposed in monolayers from aluminum coupons was possible for both ISSFT-021 and IMV 00236 even after 30 min exposure, suggesting an enhanced ability of fungal conidia to withstand such environments. The results from this study, combined with those revealing that cryptoendolithic fungal communities embedded in rocks can withstand SMC for an extended period of time (<a aria-describedby="B51" class="usa-link" href="#B51">Onofri et al., 2012</a>, <a aria-describedby="B52" class="usa-link" href="#B52">2015</a>; <a aria-describedby="B56" class="usa-link" href="#B56">Pacelli et al., 2018</a>), imply that fungi should be considered as a possible forward contamination source. This is further supported by the fact that the omnipresence of filamentous fungi has been documented in spacecraft assembly facilities (<a aria-describedby="B40" class="usa-link" href="#B40">La Duc et al., 2012</a>; <a aria-describedby="B74" class="usa-link" href="#B74">Weinmaier et al., 2015</a>).</p>
<p>It has been reported that upon exposure to space conditions bacteria become more UV resistant (<a aria-describedby="B73" class="usa-link" href="#B73">Vaishampayan et al., 2012</a>). Similarly, this study showed that SMC-exposed ISSFT-021-30 had a higher tolerance to UV irradiation than its unexposed counterpart ISSFT-021. Additionally, it appeared to be more tolerant to UV exposure than any of the additionally tested <em>A. fumigaus</em> strain, including another ISS-isolated strain IF1SW-F4 and two clinical isolates Af293 and CEA10. These results suggest that exposure to an enhanced irradiation environment may lead to adaptive alterations that give fungi an increased environmental advantage when exposed to unique space conditions.</p>
<p>One significant way that filamentous fungi respond to external stimuli is through alterations in SM production. Although these bioactive molecules are not directly necessary for survival, they often confer an environmental advantage (<a aria-describedby="B36" class="usa-link" href="#B36">Keller et al., 2005</a>). Both fungal species subjected to SMC, ISSFT-021-30 and IMV 00236-30, displayed slightly increased yield of produced SMs. Such a tendency agrees with previously observed elevated yields of SM in ISS-isolated JSC-093350089 <em>A. niger</em> when compared to culture collection strain ATCC 1015 (Romsdahl et al., unpublished). Additionally, when the metabolome of ISSFT-021 was characterized and compared to the well-studied Af293, production yields of pyripyropene A and fumiquinazolines increased (<a aria-describedby="B37" class="usa-link" href="#B37">Knox et al., 2016</a>). Therefore, the observed tendency of increased SM production in strains following exposure to SMC supports the hypothesis that space conditions might have altered secondary metabolite production yields.</p>
<p>This study revealed that exposure to SMC altered the proteome of both ISSFT-021-30 and IMV 00236-30 when compared to unexposed counterparts. Interestingly, in both species, the highest number of differentially expressed proteins were translation-related ribosomal components. Interestingly, exposed to ionizing radiation <em>C. sphaerospermum, Wangiella dermatitidis</em>, and <em>Cryptococcus neoformans</em> showed increased growth when compared to unexposed controls due to electronic changes in melanin (<a aria-describedby="B17" class="usa-link" href="#B17">Dadachova et al., 2007</a>), however, in our study only ISSFT-021-30 seemed to follow that pattern, revealing up-regulation of translation-related proteins. Observed opposite expression patterns of translation-related proteins, which underlay species-related unique defense system, may have been shaped by the varying environmental origins of each isolate (<a aria-describedby="B26" class="usa-link" href="#B26">Gasch, 2007</a>; <a aria-describedby="B32" class="usa-link" href="#B32">Heitman et al., 2017</a>). This discrepancy suggests that different species of filamentous fungi alter their growth and development in response to adverse environmental conditions in a species/strain-specific manner. Furthermore, the difference in the expression levels of translation-related ribosomal protein may lead to the overall up- and down-regulation of other proteins in ISSFT-021-30 and IMV 00236-30, respectively. Interestingly, ribosomal protein Rpl17, has been indicated as crucial for survival in <em>A. fumigatus</em> (<a aria-describedby="B23" class="usa-link" href="#B23">Firon et al., 2003</a>), <em>C. neoformans</em> (<a aria-describedby="B35" class="usa-link" href="#B35">Ianiri et al., 2017</a>), and <em>S. cerevisiae</em> (<a aria-describedby="B24" class="usa-link" href="#B24">Gamalinda et al., 2013</a>) especially once grown on glucose. Induced abundance of Rpl17 upon exposure to SMC may suggest that it modulates <em>A. fumigatus</em> response to harsh conditions depending on a carbon source. Several differentially expressed proteins were involved in carbohydrate metabolism and energy conversion, including isocitrate lyase AcuD (AFUA_4G13510) in proteome of ISSFT-021-30. AcuD is one of the key enzymes in glyoxylate cycle, which facilitates fungal growth on alternative C<sub>2</sub> carbon sources (<a aria-describedby="B19" class="usa-link" href="#B19">De Lucas et al., 1999</a>). In addition, AcuE, another enzyme in the glyoxylate cycle, exhibited increased abundance in ISS-isolated ISSFT-021 and IF1SW-F4 when compared to clinical isolates Af293 and CEA10 (<a aria-describedby="B8" class="usa-link" href="#B8">Blachowicz et al., 2019</a>). Further, increased abundance of proteins involved in starvation response was observed in the ISS-isolated JSC-093350089 <em>A. niger</em> when compared to the well-studied culture collection strain ATCC 1015 (<a aria-describedby="B58" class="usa-link" href="#B58">Romsdahl et al., 2018</a>). These findings suggest that increased production of starvation-response enzymes play a role in the adaptation to space conditions. Several enzymes involved in chitin recognition and degradation were up-regulated in IMV 00236-30. These enzymes enable using chitinous debris as an alternative carbon source and allow morphogenetic changes during growth and differentiation (<a aria-describedby="B30" class="usa-link" href="#B30">Gooday et al., 1986</a>), which further suggests that alterations in carbohydrate metabolism are an adaptive response to SMC. Interestingly, when protein patterns of <em>Cryomyces antarcticus</em>, <em>Knufia perforans</em>, and <em>Exophiala jeanselmei</em> exposed in multilayers to SMC were analyzed by 2D gel electrophoresis no additional stress-induced proteins were observed (<a aria-describedby="B75" class="usa-link" href="#B75">Zakharova et al., 2014</a>).</p>
<p>This study affirms the enormous capability of filamentous fungi to adapt to extreme environmental conditions and thrive in a wide variety of ecological niches. To our knowledge, this is the first report of shotgun proteomic and metabolomic analyses of filamentous fungi in response to SMC. Such a complex state of the art analyses of fungal adaptive responses to space conditions are essential for ensuring safety in the era of future outer space explorations, as fungi will undoubtedly accompany people during space voyages. Thorough understanding of how filamentous fungi adapt to space conditions is important for both maintaining crew health and preventing biocorrosion of the spacecraft, as both opportunistic pathogenic fungi (<a aria-describedby="B37" class="usa-link" href="#B37">Knox et al., 2016</a>) and technophiles (<a aria-describedby="B3" class="usa-link" href="#B3">Alekhova et al., 2005</a>) have been reported on board of the ISS and Mir space stations.</p></section><section id="s1"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="sec12"><h3 class="pmc_sec_title">Sample Collection Sites</h3>
<p>Subcultures of the isolated strains were obtained from the Institute of Microbiology and Virology, Ukrainian Academy of Sciences, within the framework of a multiyear collaborative research program, to the Center for Environmental Biotechnology at Lawrence Berkeley National Laboratory (LBNL). For this study 12 Chernobyl nuclear accident-associated isolates were selected (<a class="usa-link" href="#T1">Table 1</a>).</p></section><section id="sec13"><h3 class="pmc_sec_title">Preparation of Aluminum Coupons With Monolayers of Dried Fungal Conidia</h3>
<p>High-grade aluminum coupons (Al 6061-T6) were precision cleaned for sterility as previously described (<a aria-describedby="B55" class="usa-link" href="#B55">Osman et al., 2008</a>). Each coupon was seeded with 100 μL of conidia suspension to contain ∼10<sup>5</sup> conidia per coupon. Conidia were counted using a hemocytometer (Double Neubauer Counting Chamber, Hausser Scientific, Horsham, PA, United States) after harvesting 5 days grown cultures at 26°C on potato dextrose agar (PDA). Conidial suspensions were diluted in molecular biology grade water (Fisher Scientific, Waltham, MA, United States) and ∼10<sup>5</sup> conidia were added to each coupon followed by drying overnight at the room temperature in a bio-hood. The monolayers of conidia were confirmed by scanning electron microscopy (data not shown).</p></section><section id="sec14"><h3 class="pmc_sec_title">UV-C Exposure and Recovery</h3>
<p>Aluminum coupons with dried fungal conidia were placed in a plastic Petri dish, without a lid, and exposed to UV-C using a low-pressure handheld mercury arc UV lamp (model UVG-11; UVP Inc., Upland, CA, United States). The lamp was placed above the sample, and the UV flux at the surface of exposed sample was measured using UVX digital radiometer (UVP Inc.). The exposure time required to produce doses: 0, 50, 100, 500, 1000, and 2,000 J/m<sup>2</sup> was calculated at 100 μW cm<sup>-2</sup>. After exposure to UV-C 100 μL of 10% polyvinyl alcohol (PVA) was applied on each coupon and dried at 37°C for 50 min. Dried PVA along with fungal conidia was peeled using sterile forceps and added to 1 mL of molecular biology grade water (Fisher Scientific). The PVA extraction step was repeated. When PVA was dissolved, serial dilutions were prepared and plated on PDA in duplicates. Colony forming units (CFUs) were counted after 7 days of incubation at 26°C.</p></section><section id="sec15"><h3 class="pmc_sec_title">Simulated Martian Conditions (SMC)</h3>
<p>Survival and response of fungal strains under SMC were tested in a Mars simulation chamber equipped with a UV transparent fused silica window according to a previously described set up (<a aria-describedby="B25" class="usa-link" href="#B25">Garry et al., 2006</a>; <a aria-describedby="B55" class="usa-link" href="#B55">Osman et al., 2008</a>; <a aria-describedby="B57" class="usa-link" href="#B57">Peeters et al., 2010</a>). Coupons prepared following the protocol described above were placed in the sterile Falcon tubes and sent to Netherlands. After arrival (∼1.5 weeks) samples deposited on aluminum coupons were placed inside the simulation chamber and subsequently the chamber was evacuated <em>via</em> an oil-free scroll pump (XDS5, Edwards Vacuum, Crawley, United Kingdom) to reach a base pressure of 20 Pa. While continuously being pumped, the chamber was purged five times with high purity CO<sub>2</sub> (99.995%, H<sub>2</sub>0 &lt; 5 ppm, O<sub>2</sub> &lt; 5 ppm, Praxair, Danbury, CT, United States), before establishing a continuous CO<sub>2</sub> gas flow to maintain a stable chamber pressure of approximately 600 Pa. Samples were exposed at room temperature to simulated solar light (SF150 with a xenon arc lamp, 150W, Sciencetech Inc., London, Canada) <em>via</em> the fused silica window of the simulation chamber. The 200–400 nm wavelength integrated irradiance at the sample distance was 58.7 W/m<sup>2</sup> (<a class="usa-link" href="#F6">Figure 6</a>). Samples were exposed for 5 and 30 min to cumulative doses of 2,670 and 16,110 J/m<sup>2</sup>, respectively, before being removed from the chamber and placed in sterile Falcon tubes for shipment back to JPL (Pasadena, CA, United States) for further analysis. Upon return to JPL, SMC-exposed samples were processed following the aforementioned PVA protocol to assess the survival rates.</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=6529585_fmicb-10-01013-g006.jpg" target="_blank"><img alt="FIGURE 6" class="graphic zoom-in" height="460" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/06c4/6529585/a064dff143d2/fmicb-10-01013-g006.jpg" width="708"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>UV spectra (200–400 nm) of the solar simulator employed in this study. Other lighting spectra of Mars models are presented along with the integrated irradiance over the wavelength range from 200 to 400 nm.</p></figcaption></figure></section><section id="sec16"><h3 class="pmc_sec_title">Secondary Metabolite Extraction and Analysis</h3>
<p><em>Aspergillus fumigatus</em> strains were cultivated at 30°C on GMM agar plates while <em>C. cladosporioides</em> strains were grown at 26°C on MEA agar plates starting with 10<sup>7</sup> conidia/per Petri plate (<em>D</em> = 10 cm). After 5 days, agar was chopped, extracted with 25 mL methanol, and 25 mL of 1:1 methanol/dichloromethane each followed by 1 h sonication and filtration. After the second filtration, combined crude extracts from each isolate were evaporated in <em>vacuo</em>, suspended in 20 mL of water and partitioned with ethylacetate (20 mL). The ethylacetate layer was evaporated in <em>vacuo</em>, re-dissolved in 1 mL of 20% dimethyl sulfoxide/methanol and 10 μL was examined by high performance liquid chromatography-photodiode array detection-mass spectrometry (HPLC-DAD-MS) analysis. HPLC-MS was carried out using ThermoFinnigan LCQ Advantage ion trap mass spectrometer with an RP C<sub>18</sub> column (Alltech Prevail C<sub>18</sub> 3 mm 2.1 × 100 mm) at a flow rate 125 μL/min. The solvent gradient for LC/MS was 95% acetonitrile/H<sub>2</sub>O (solvent B) in 5% acetonitrile/H<sub>2</sub>O (solvent A) both containing 0.05% formic acid, as follows: 0% solvent B from 0 to 5 min, 0 to 100% solvent B from 5 to 35 min, 100% solvent B from 35 to 40 min, 100 to 0% solvent B from 40 to 45 min, and re-equilibration with 0% solvent B from 45 to 50 min.</p></section><section id="sec17"><h3 class="pmc_sec_title">Proteome Samples Extraction and Processing</h3>
<p>Exposure of fungal conidia to SMC required using ∼10<sup>6</sup> conidia/coupon to avoid the shadowing effect (<a aria-describedby="B55" class="usa-link" href="#B55">Osman et al., 2008</a>), however such amount of biomass was not enough to perform detailed proteome analyses. Therefore, to observe permanent alterations in proteomes of SMC-exposed strains when compared to unexposed ones <em>A. fumigatus</em> strains were regrown at 30°C on GMM and <em>C. cladosporioides</em> strains at 26°C on MEA starting with 10<sup>7</sup> conidia/per Petri plate (<em>D</em> = 10 cm). After 5 days, mycelia and spores from agar plates were collected and stored at -80 °C prior to protein extraction at City of Hope (Duarte, CA, United States). The protein was extracted as previously described (<a aria-describedby="B58" class="usa-link" href="#B58">Romsdahl et al., 2018</a>). In brief, mycelia, and spores were lysed and homogenized using a bead beater. Protein concentrations were measured by Bradford assay with a bovine serum albumin standard curve (Bio-Rad Laboratories, Inc., Hercules, CA, United States).</p>
<p>The samples were processed for a tandem mass tag (TMT) labeling as described by <a aria-describedby="B58" class="usa-link" href="#B58">Romsdahl et al. (2018)</a> with modification. The proteomic profiling of <em>A. fumigatus</em> and <em>C. cladosporioides</em> strains was carried out in two separate TMT LC/MS experiments. <em>A. fumigatus</em> strains ISSFT-021 and ISSFT-021-30 were labeled with TMT<sup>6</sup>-128 and TMT<sup>6</sup>-129, respectively. Two biological samples of <em>C. cladosporioides</em> strains IMV00 236 and IMV00 236-30 were labeled with TMT<sup>6</sup>-128/130 and TMT<sup>6</sup>-129/131, respectively.</p>
<p>The samples were analyzed on an Orbitrap Fusion Tribrid mass spectrometer with an EASY-nLC 1000 Liquid Chromatograph, a 75 μm × 2 cm Acclaim PepMap100 C<sub>18</sub> trapping column, and a 75 μm × 25 cm PepMap RSLC C<sub>18</sub> analytical column, and an Easy-Spray ion source (Thermo Fisher Scientific) as previously described (<a aria-describedby="B58" class="usa-link" href="#B58">Romsdahl et al., 2018</a>).</p></section><section id="sec18"><h3 class="pmc_sec_title">Quantitative Proteomics Analysis</h3>
<p>All MS spectra were analyzed using Proteome Discoverer (version 2.2.0.388, Thermo Fisher Scientific) with Sequest-HT search engines. Protein databases were either <em>A. fumigatus</em> Af293 database from NCBI containing 9845 non-redundant sequences or an in-house annotated draft genome sequence of <em>C. cladosporioides</em> (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MSJH00000000" rel="noopener noreferrer" target="_blank">MSJH00000000</a>). The search parameters were described by <a aria-describedby="B58" class="usa-link" href="#B58">Romsdahl et al. (2018)</a>.</p>
<p>Technical triplicate measurements for each protein were averaged. Only proteins that were identified with at least one peptide detected in each technical replicate, and quantified in all technical and biological replicates, were considered for the analysis. The identified proteins were then averaged and Log2 transformed. Student <em>t</em>-test was performed to identify proteins that are differentially expressed between each SMC-exposed and unexposed group. Proteins with <em>p</em>-value ≤ 0.05 were further evaluated for up- and down-regulation using a cut-off value of ≥±2-fold change. AspGD gene ontology (GO) Slim terms (<a aria-describedby="B14" class="usa-link" href="#B14">Cerqueira et al., 2014</a>) were used to study the distribution of differentially expressed proteins among biological processes in SMC-exposed ISSFT-021-30 while the cluster of orthologous genes (COG) database (<a aria-describedby="B68" class="usa-link" href="#B68">Tatusov et al., 2000</a>) used in CloVR (<a aria-describedby="B4" class="usa-link" href="#B4">Angiuoli et al., 2011</a>) was used to study the distribution of differentially expressed proteins in SMC-exposed IMV 00236-30.</p></section><section id="sec19"><h3 class="pmc_sec_title">Genome Annotation</h3>
<p>Genome annotation of <em>C. cladosporioides</em> IMV 00236 was performed on the deposited assembly (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MSJH00000000" rel="noopener noreferrer" target="_blank">MSJH00000000</a>) with Funannotate (v1.5.1) (<a aria-describedby="B41" class="usa-link" href="#B41">Love et al., 2018</a>). Proteins from <em>Capnodiales</em> fungi (<em>Dothideomycetes</em>), Swissprot database (<a aria-describedby="B67" class="usa-link" href="#B67">Swiss Institute of Bioinformatics Members [SIB], 2016</a>), and transcripts from <em>Cladosporium sphaerospermum</em> UM 843 (<a aria-describedby="B47" class="usa-link" href="#B47">Ng et al., 2012</a>) were used as informant sequences. Conserved genes were identified from BUSCO core set “ascomycota_odb9” and were used to create training set for <em>ab initio</em> gene prediction by Augustus (<a aria-describedby="B65" class="usa-link" href="#B65">Stanke et al., 2006</a>; <a aria-describedby="B63" class="usa-link" href="#B63">Simão et al., 2015</a>). The <em>ab initio</em> predictor GenemarkHMM-ES was trained using its self-training procedures (<a aria-describedby="B69" class="usa-link" href="#B69">Ter-Hovhannisyan et al., 2008</a>). These predictions along with splice-aware aligned proteins using DIAMOND (<a aria-describedby="B12" class="usa-link" href="#B12">Buchfink et al., 2015</a>) followed by refinement with exonerate to improve spliced alignment accuracy (<a aria-describedby="B64" class="usa-link" href="#B64">Slater and Birney, 2005</a>). Consensus gene models were generated from the combined evidence with Evidence Modeler (<a aria-describedby="B31" class="usa-link" href="#B31">Haas et al., 2008</a>). Predicted gene function from conserved protein domains (<a aria-describedby="B21" class="usa-link" href="#B21">El-Gebali et al., 2018</a>), Swissprot (<a aria-describedby="B67" class="usa-link" href="#B67">Swiss Institute of Bioinformatics Members [SIB], 2016</a>) and inferred homology to conserved protein clusters in eggNOGdb (<a aria-describedby="B34" class="usa-link" href="#B34">Huerta-Cepas et al., 2018</a>), and secondary metabolite cluster prediction (<a aria-describedby="B10" class="usa-link" href="#B10">Blin et al., 2017</a>).</p></section></section><section id="sec20"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>AB drafted the manuscript, carried out the UV-C exposure experiments, and contributed to data analysis and interpretation and strain identification. AC and MK conducted protein sample processing, LC/MS analyses, and proteome data processing. AE and PE enabled the simulated Martian chamber at the Leiden Institute of Chemistry, conducted the SMC experiments, and drafted parts of the manuscript. JS annotated the genome of IMV 00236 for proteome analysis. TT provided Chernobyl isolated strains and conducted morphological characterization and identification. CW drafted the manuscript and interpreted the metabolomics data. KV designed the study, interpreted the data, and drafted and critically reviewed the manuscript. All authors read and approved the final manuscript.</p></section><section id="sec21"><h2 class="pmc_sec_title">Conflict of Interest Statement</h2>
<p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>Part of the research described in this publication was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with NASA. We are grateful to the Institute of Microbiology and Virology, Ukrainian Academy of Sciences for collecting the samples. We thank Jillian Romsdahl for proofreading the manuscript. Aleksandra Checinska-Sielaff is acknowledged for identifying ISS isolated strains. We thank the Leiden Institute of Chemistry for allowing the use of the simulated Mars chamber, and the members of the Planetary Protection group at JPL for their technical assistance. We also thank the Space Biology program for the funding and sponsoring the student fellowship to AB. ©2019 California Institute of Technology. Government sponsorship acknowledged.</p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn4"><p><strong>Funding.</strong> This research was funded by a JPL Internal Development Program awarded to KV and also funded by a 2012 Space Biology NNH12ZTT001N grant no. 19-12829-26 under Task Order NNN13D111T awarded to KV and a student fellowship to AB. AC and MK were supported through JPL subcontract 1574847. The use of the City of Hope Mass Spectrometry and Proteomics core facility was partially supported by NIH, National Cancer Institute Grant P30 CA33572.</p></div></div></section><section id="sec22"><h2 class="pmc_sec_title">Supplementary Material</h2>
<p>The Supplementary Material for this article can be found online at: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.frontiersin.org/articles/10.3389/fmicb.2019.01013/full#supplementary-material" rel="noopener noreferrer" target="_blank">https://www.frontiersin.org/articles/10.3389/fmicb.2019.01013/full#supplementary-material</a></p>
<section class="sm xbox font-sm" id="SM1"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/6529585/bin/Data_Sheet_1.PDF">Click here for additional data file.</a><sup> (667KB, PDF) </sup>
</div></div></section></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="B1">
<cite>Adav S. S., Ravindran A., Sze S. K. (2013). Proteomic analysis of temperature dependent extracellular proteins from <em>Aspergillus fumigatus</em> grown under solid-state culture condition.
<em>J. Proteome Res.</em>
12
2715–2731. 10.1021/pr4000762
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/pr4000762" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23647126/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Proteomic%20analysis%20of%20temperature%20dependent%20extracellular%20proteins%20from%20Aspergillus%20fumigatus%20grown%20under%20solid-state%20culture%20condition.&amp;author=S.%20S.%20Adav&amp;author=A.%20Ravindran&amp;author=S.%20K.%20Sze&amp;volume=12&amp;publication_year=2013&amp;pages=2715-2731&amp;pmid=23647126&amp;doi=10.1021/pr4000762&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<cite>Albrecht D., Guthke R., Brakhage A. A., Kniemeyer O. (2010). Integrative analysis of the heat shock response in <em>Aspergillus fumigatus</em>.
<em>BMC Genomics</em>
11:32. 10.1186/1471-2164-11-32
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2164-11-32" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2820008/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20074381/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Integrative%20analysis%20of%20the%20heat%20shock%20response%20in%20Aspergillus%20fumigatus.&amp;author=D.%20Albrecht&amp;author=R.%20Guthke&amp;author=A.%20A.%20Brakhage&amp;author=O.%20Kniemeyer&amp;volume=11&amp;issue=32&amp;publication_year=2010&amp;pmid=20074381&amp;doi=10.1186/1471-2164-11-32&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<cite>Alekhova T. A., Aleksandrova A. A., Novozhilova T. I., Lysak L. V., Zagustina N. A., Bezborodov A. M. (2005). Monitoring of microbial degraders in manned space stations.
<em>Prikl. Biokhim. Mikrobiol.</em>
41
435–443. </cite> [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16212041/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Prikl.%20Biokhim.%20Mikrobiol.&amp;title=Monitoring%20of%20microbial%20degraders%20in%20manned%20space%20stations.&amp;author=T.%20A.%20Alekhova&amp;author=A.%20A.%20Aleksandrova&amp;author=T.%20I.%20Novozhilova&amp;author=L.%20V.%20Lysak&amp;author=N.%20A.%20Zagustina&amp;volume=41&amp;publication_year=2005&amp;pages=435-443&amp;pmid=16212041&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<cite>Angiuoli S. V., Matalka M., Gussman A., Galens K., Vangala M., Riley D. R., et al.  (2011). CloVR: a virtual machine for automated and portable sequence analysis from the desktop using cloud computing.
<em>BMC Bioinformatics</em>
12:356. 10.1186/1471-2105-12-356
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2105-12-356" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3228541/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21878105/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=CloVR:%20a%20virtual%20machine%20for%20automated%20and%20portable%20sequence%20analysis%20from%20the%20desktop%20using%20cloud%20computing.&amp;author=S.%20V.%20Angiuoli&amp;author=M.%20Matalka&amp;author=A.%20Gussman&amp;author=K.%20Galens&amp;author=M.%20Vangala&amp;volume=12&amp;issue=356&amp;publication_year=2011&amp;pmid=21878105&amp;doi=10.1186/1471-2105-12-356&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<cite>Barnard R. L., Osborne C. A., Firestone M. K. (2013). Responses of soil bacterial and fungal communities to extreme desiccation and rewetting.
<em>ISME J.</em>
7
2229–2241. 10.1038/ismej.2013.104
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ismej.2013.104" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3806258/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23823489/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=Responses%20of%20soil%20bacterial%20and%20fungal%20communities%20to%20extreme%20desiccation%20and%20rewetting.&amp;author=R.%20L.%20Barnard&amp;author=C.%20A.%20Osborne&amp;author=M.%20K.%20Firestone&amp;volume=7&amp;publication_year=2013&amp;pages=2229-2241&amp;pmid=23823489&amp;doi=10.1038/ismej.2013.104&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<cite>Be N. A., Avila-Herrera A., Allen J. E., Singh N., Checinska Sielaff A., Jaing C., et al.  (2017). Whole metagenome profiles of particulates collected from the international space station.
<em>Microbiome</em>
5:81. 10.1186/s40168-017-0292-4
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40168-017-0292-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5514531/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28716113/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Whole%20metagenome%20profiles%20of%20particulates%20collected%20from%20the%20international%20space%20station.&amp;author=N.%20A.%20Be&amp;author=A.%20Avila-Herrera&amp;author=J.%20E.%20Allen&amp;author=N.%20Singh&amp;author=A.%20Checinska%20Sielaff&amp;volume=5&amp;issue=81&amp;publication_year=2017&amp;pmid=28716113&amp;doi=10.1186/s40168-017-0292-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<cite>Billi D., Viaggiu E., Cockell C. S., Rabbow E., Horneck G., Onofri S. (2011). Damage escape and repair in dried <em>Chroococcidiopsis</em> spp. from hot and cold deserts exposed to simulated space and martian conditions.
<em>Astrobiology</em>
11
65–73. 10.1089/ast.2009.0430
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2009.0430" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21294638/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Damage%20escape%20and%20repair%20in%20dried%20Chroococcidiopsis%20spp.%20from%20hot%20and%20cold%20deserts%20exposed%20to%20simulated%20space%20and%20martian%20conditions.&amp;author=D.%20Billi&amp;author=E.%20Viaggiu&amp;author=C.%20S.%20Cockell&amp;author=E.%20Rabbow&amp;author=G.%20Horneck&amp;volume=11&amp;publication_year=2011&amp;pages=65-73&amp;pmid=21294638&amp;doi=10.1089/ast.2009.0430&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<cite>Blachowicz A., Chiang A. J., Romsdahl J., Kalkum M., Wang C. C. C., Venkateswaran K. (2019). Proteomic characterization of <em>Aspergillus fumigatus</em> isolated from air and surfaces of the international space station.
<em>Fungal Genet. Biol.</em>
124
39–46. 10.1016/j.fgb.2019.01.001
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.fgb.2019.01.001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9116463/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30611835/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&amp;title=Proteomic%20characterization%20of%20Aspergillus%20fumigatus%20isolated%20from%20air%20and%20surfaces%20of%20the%20international%20space%20station.&amp;author=A.%20Blachowicz&amp;author=A.%20J.%20Chiang&amp;author=J.%20Romsdahl&amp;author=M.%20Kalkum&amp;author=C.%20C.%20C.%20Wang&amp;volume=124&amp;publication_year=2019&amp;pages=39-46&amp;pmid=30611835&amp;doi=10.1016/j.fgb.2019.01.001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<cite>Blachowicz A., Mayer T., Bashir M., Pieber T. R., De León P., Venkateswaran K. (2017). Human presence impacts fungal diversity of inflated lunar/Mars analog habitat.
<em>Microbiome</em>
5:62. 10.1186/s40168-017-0280-8
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40168-017-0280-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5504618/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28693587/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Human%20presence%20impacts%20fungal%20diversity%20of%20inflated%20lunar/Mars%20analog%20habitat.&amp;author=A.%20Blachowicz&amp;author=T.%20Mayer&amp;author=M.%20Bashir&amp;author=T.%20R.%20Pieber&amp;author=P.%20De%20Le%C3%B3n&amp;volume=5&amp;issue=62&amp;publication_year=2017&amp;pmid=28693587&amp;doi=10.1186/s40168-017-0280-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<cite>Blin K., Wolf T., Chevrette M. G., Lu X., Schwalen C. J., Kautsar S. A., et al.  (2017). antiSMASH 4.0-improvements in chemistry prediction and gene cluster boundary identification.
<em>Nucleic Acids Res.</em>
45
W36–W41. 10.1093/nar/gkx319
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkx319" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5570095/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28460038/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=antiSMASH%204.0-improvements%20in%20chemistry%20prediction%20and%20gene%20cluster%20boundary%20identification.&amp;author=K.%20Blin&amp;author=T.%20Wolf&amp;author=M.%20G.%20Chevrette&amp;author=X.%20Lu&amp;author=C.%20J.%20Schwalen&amp;volume=45&amp;publication_year=2017&amp;pages=W36-W41&amp;pmid=28460038&amp;doi=10.1093/nar/gkx319&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<cite>Braga G. U. L., Rangel D. E. N., Fernandes ÉK. K., Flint S. D., Roberts D. W. (2015). Molecular and physiological effects of environmental UV radiation on fungal conidia.
<em>Curr. Genet.</em>
61
405–425. 10.1007/s00294-015-0483-0
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00294-015-0483-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25824285/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Genet.&amp;title=Molecular%20and%20physiological%20effects%20of%20environmental%20UV%20radiation%20on%20fungal%20conidia.&amp;author=G.%20U.%20L.%20Braga&amp;author=D.%20E.%20N.%20Rangel&amp;author=%C3%89K.%20K.%20Fernandes&amp;author=S.%20D.%20Flint&amp;author=D.%20W.%20Roberts&amp;volume=61&amp;publication_year=2015&amp;pages=405-425&amp;pmid=25824285&amp;doi=10.1007/s00294-015-0483-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<cite>Buchfink B., Xie C., Huson D. H. (2015). Fast and sensitive protein alignment using DIAMOND.
<em>Nat. Methods</em>
12
59–60. 10.1038/nmeth.3176
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.3176" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25402007/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Fast%20and%20sensitive%20protein%20alignment%20using%20DIAMOND.&amp;author=B.%20Buchfink&amp;author=C.%20Xie&amp;author=D.%20H.%20Huson&amp;volume=12&amp;publication_year=2015&amp;pages=59-60&amp;pmid=25402007&amp;doi=10.1038/nmeth.3176&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<cite>Cagas S. E., Jain M. R., Li H., Perlin D. S. (2011). The proteomic signature of <em>Aspergillus fumigatus</em> during early development.
<em>Mol. Cell. Proteomics</em>
10:M111.010108. 10.1074/mcp.M111.010108
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/mcp.M111.010108" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3226407/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21825280/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=The%20proteomic%20signature%20of%20Aspergillus%20fumigatus%20during%20early%20development.&amp;author=S.%20E.%20Cagas&amp;author=M.%20R.%20Jain&amp;author=H.%20Li&amp;author=D.%20S.%20Perlin&amp;volume=10&amp;issue=M111&amp;publication_year=2011&amp;pmid=21825280&amp;doi=10.1074/mcp.M111.010108&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14">
<cite>Cerqueira G. C., Arnaud M. B., Inglis D. O., Skrzypek M. S., Binkley G., Simison M., et al.  (2014). The <em>Aspergillus</em> genome database: multispecies curation and incorporation of RNA-Seq data to improve structural gene annotations.
<em>Nucleic Acids Res.</em>
42
D705–D710. 10.1093/nar/gkt1029
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkt1029" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3965050/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24194595/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20Aspergillus%20genome%20database:%20multispecies%20curation%20and%20incorporation%20of%20RNA-Seq%20data%20to%20improve%20structural%20gene%20annotations.&amp;author=G.%20C.%20Cerqueira&amp;author=M.%20B.%20Arnaud&amp;author=D.%20O.%20Inglis&amp;author=M.%20S.%20Skrzypek&amp;author=G.%20Binkley&amp;volume=42&amp;publication_year=2014&amp;pages=D705-D710&amp;pmid=24194595&amp;doi=10.1093/nar/gkt1029&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15">
<cite>Checinska A., Probst A. J., Vaishampayan P., White J. R., Kumar D., Stepanov V. G., et al.  (2015). Microbiomes of the dust particles collected from the international space station and spacecraft assembly facilities.
<em>Microbiome</em>
3:50. 10.1186/s40168-015-0116-3
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40168-015-0116-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4624184/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26502721/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Microbiomes%20of%20the%20dust%20particles%20collected%20from%20the%20international%20space%20station%20and%20spacecraft%20assembly%20facilities.&amp;author=A.%20Checinska&amp;author=A.%20J.%20Probst&amp;author=P.%20Vaishampayan&amp;author=J.%20R.%20White&amp;author=D.%20Kumar&amp;volume=3&amp;issue=50&amp;publication_year=2015&amp;pmid=26502721&amp;doi=10.1186/s40168-015-0116-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16">
<cite>Costanzo M., Baryshnikova A., Bellay J., Kim Y., Spear E. D., Sevier C. S., et al.  (2010). The genetic landscape of a cell.
<em>Science</em>
327
425–431. 10.1126/science.1180823
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1180823" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5600254/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20093466/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20genetic%20landscape%20of%20a%20cell.&amp;author=M.%20Costanzo&amp;author=A.%20Baryshnikova&amp;author=J.%20Bellay&amp;author=Y.%20Kim&amp;author=E.%20D.%20Spear&amp;volume=327&amp;publication_year=2010&amp;pages=425-431&amp;pmid=20093466&amp;doi=10.1126/science.1180823&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17">
<cite>Dadachova E., Bryan R. A., Huang X., Moadel T., Schweitzer A. D., Aisen P., et al.  (2007). Ionizing radiation changes the electronic properties of melanin and enhances the growth of melanized fungi.
<em>PLoS One</em>
2:e457. 10.1371/journal.pone.0000457
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0000457" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1866175/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17520016/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Ionizing%20radiation%20changes%20the%20electronic%20properties%20of%20melanin%20and%20enhances%20the%20growth%20of%20melanized%20fungi.&amp;author=E.%20Dadachova&amp;author=R.%20A.%20Bryan&amp;author=X.%20Huang&amp;author=T.%20Moadel&amp;author=A.%20D.%20Schweitzer&amp;volume=2&amp;issue=e457&amp;publication_year=2007&amp;pmid=17520016&amp;doi=10.1371/journal.pone.0000457&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18">
<cite>Dadachova E., Casadevall A. (2008). Ionizing radiation: how fungi cope, adapt, and exploit with the help of melanin.
<em>Curr. Opin. Microbiol.</em>
11
525–531. 10.1016/j.mib.2008.09.013
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.mib.2008.09.013" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2677413/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18848901/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Microbiol.&amp;title=Ionizing%20radiation:%20how%20fungi%20cope,%20adapt,%20and%20exploit%20with%20the%20help%20of%20melanin.&amp;author=E.%20Dadachova&amp;author=A.%20Casadevall&amp;volume=11&amp;publication_year=2008&amp;pages=525-531&amp;pmid=18848901&amp;doi=10.1016/j.mib.2008.09.013&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19">
<cite>De Lucas J. R., Domínguez A. I., Valenciano S., Turner G., Laborda F. (1999). The acuH gene of <em>Aspergillus nidulans</em>, required for growth on acetate and long-chain fatty acids, encodes a putative homologue of the mammalian carnitine/acylcarnitine carrier.
<em>Arch. Microbiol.</em>
171
386–396. 10.1007/s002030050725
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s002030050725" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10369894/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Arch.%20Microbiol.&amp;title=The%20acuH%20gene%20of%20Aspergillus%20nidulans,%20required%20for%20growth%20on%20acetate%20and%20long-chain%20fatty%20acids,%20encodes%20a%20putative%20homologue%20of%20the%20mammalian%20carnitine/acylcarnitine%20carrier.&amp;author=J.%20R.%20De%20Lucas&amp;author=A.%20I.%20Dom%C3%ADnguez&amp;author=S.%20Valenciano&amp;author=G.%20Turner&amp;author=F.%20Laborda&amp;volume=171&amp;publication_year=1999&amp;pages=386-396&amp;pmid=10369894&amp;doi=10.1007/s002030050725&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B20">
<cite>Dighton J., Tugay T., Zhdanova N. (2008). Fungi and ionizing radiation from radionuclides.
<em>FEMS Microbiol. Lett.</em>
281
109–120. 10.1111/j.1574-6968.2008.01076.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1574-6968.2008.01076.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18279333/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Lett.&amp;title=Fungi%20and%20ionizing%20radiation%20from%20radionuclides.&amp;author=J.%20Dighton&amp;author=T.%20Tugay&amp;author=N.%20Zhdanova&amp;volume=281&amp;publication_year=2008&amp;pages=109-120&amp;pmid=18279333&amp;doi=10.1111/j.1574-6968.2008.01076.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21">
<cite>El-Gebali S., Mistry J., Bateman A., Eddy S. R., Luciani A., Potter S. C., et al.  (2018). The Pfam protein families database in 2019.
<em>Nucleic Acids Res.</em>
47
D427–D432. 10.1093/nar/gky995
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gky995" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6324024/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30357350/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20Pfam%20protein%20families%20database%20in%202019.&amp;author=S.%20El-Gebali&amp;author=J.%20Mistry&amp;author=A.%20Bateman&amp;author=S.%20R.%20Eddy&amp;author=A.%20Luciani&amp;volume=47&amp;publication_year=2018&amp;pages=D427-D432&amp;pmid=30357350&amp;doi=10.1093/nar/gky995&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22">
<cite>Fendrihan S., Bérces A., Lammer H., Musso M., Rontó G., Polacsek T. K., et al.  (2009). Investigating the effects of simulated martian ultraviolet radiation on <em>Halococcus dombrowskii</em> and other extremely halophilic archaebacteria.
<em>Astrobiology</em>
9
104–112. 10.1089/ast.2007.0234
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2007.0234" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3182532/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19215203/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Investigating%20the%20effects%20of%20simulated%20martian%20ultraviolet%20radiation%20on%20Halococcus%20dombrowskii%20and%20other%20extremely%20halophilic%20archaebacteria.&amp;author=S.%20Fendrihan&amp;author=A.%20B%C3%A9rces&amp;author=H.%20Lammer&amp;author=M.%20Musso&amp;author=G.%20Ront%C3%B3&amp;volume=9&amp;publication_year=2009&amp;pages=104-112&amp;pmid=19215203&amp;doi=10.1089/ast.2007.0234&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23">
<cite>Firon A., Villalba F., Beffa R., d’Enfert C. (2003). Identification of essential genes in the human fungal pathogen <em>Aspergillus fumigatus</em> by transposon mutagenesis.
<em>Eukaryot. Cell</em>
2
247–255. 10.1128/EC.2.2.247-255.2003
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/EC.2.2.247-255.2003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC154855/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12684374/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Eukaryot.%20Cell&amp;title=Identification%20of%20essential%20genes%20in%20the%20human%20fungal%20pathogen%20Aspergillus%20fumigatus%20by%20transposon%20mutagenesis.&amp;author=A.%20Firon&amp;author=F.%20Villalba&amp;author=R.%20Beffa&amp;author=C.%20d%E2%80%99Enfert&amp;volume=2&amp;publication_year=2003&amp;pages=247-255&amp;pmid=12684374&amp;doi=10.1128/EC.2.2.247-255.2003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24">
<cite>Gamalinda M., Jakovljevic J., Babiano R., Talkish J., de la Cruz J., Woolford J. L. (2013). Yeast polypeptide exit tunnel ribosomal proteins L17, L35 and L37 are necessary to recruit late-assembling factors required for 27SB pre-rRNA processing.
<em>Nucleic Acids Res.</em>
41
1965–1983. 10.1093/nar/gks1272
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gks1272" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3561946/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23268442/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Yeast%20polypeptide%20exit%20tunnel%20ribosomal%20proteins%20L17,%20L35%20and%20L37%20are%20necessary%20to%20recruit%20late-assembling%20factors%20required%20for%2027SB%20pre-rRNA%20processing.&amp;author=M.%20Gamalinda&amp;author=J.%20Jakovljevic&amp;author=R.%20Babiano&amp;author=J.%20Talkish&amp;author=J.%20de%20la%20Cruz&amp;volume=41&amp;publication_year=2013&amp;pages=1965-1983&amp;pmid=23268442&amp;doi=10.1093/nar/gks1272&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25">
<cite>Garry J. R. C., ten Kate I. L., Martins Z., Nørnberg P., Ehrenfreund P. (2006). Analysis and survival of amino acids in Martian regolith analogs.
<em>Meteorit. Planet. Sci.</em>
41
391–405. 10.1111/j.1945-5100.2006.tb00470.x</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1945-5100.2006.tb00470.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Meteorit.%20Planet.%20Sci.&amp;title=Analysis%20and%20survival%20of%20amino%20acids%20in%20Martian%20regolith%20analogs.&amp;author=J.%20R.%20C.%20Garry&amp;author=I.%20L.%20ten%20Kate&amp;author=Z.%20Martins&amp;author=P.%20N%C3%B8rnberg&amp;author=P.%20Ehrenfreund&amp;volume=41&amp;publication_year=2006&amp;pages=391-405&amp;doi=10.1111/j.1945-5100.2006.tb00470.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26">
<cite>Gasch A. P. (2007). Comparative genomics of the environmental stress response in ascomycete fungi.
<em>Yeast</em>
24
961–976. 10.1002/yea.1512
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/yea.1512" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17605132/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Yeast&amp;title=Comparative%20genomics%20of%20the%20environmental%20stress%20response%20in%20ascomycete%20fungi.&amp;author=A.%20P.%20Gasch&amp;volume=24&amp;publication_year=2007&amp;pages=961-976&amp;pmid=17605132&amp;doi=10.1002/yea.1512&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27">
<cite>Gautam P., Upadhyay S. K., Hassan W., Madan T., Sirdeshmukh R., Sundaram C. S., et al.  (2011). Transcriptomic and proteomic profile of <em>Aspergillus fumigatus</em> on exposure to artemisinin.
<em>Mycopathologia</em>
172
331–346. 10.1007/s11046-011-9445-3
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11046-011-9445-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21755315/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycopathologia&amp;title=Transcriptomic%20and%20proteomic%20profile%20of%20Aspergillus%20fumigatus%20on%20exposure%20to%20artemisinin.&amp;author=P.%20Gautam&amp;author=S.%20K.%20Upadhyay&amp;author=W.%20Hassan&amp;author=T.%20Madan&amp;author=R.%20Sirdeshmukh&amp;volume=172&amp;publication_year=2011&amp;pages=331-346&amp;pmid=21755315&amp;doi=10.1007/s11046-011-9445-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28">
<cite>Gonsalves V. (2012). Halophilic fungi in a polyhaline estuarine habitat.
<em>J. Yeast Fungal Res.</em>
3
30–36. 10.5897/JYFR12.007</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.5897/JYFR12.007" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Yeast%20Fungal%20Res.&amp;title=Halophilic%20fungi%20in%20a%20polyhaline%20estuarine%20habitat.&amp;author=V.%20Gonsalves&amp;volume=3&amp;publication_year=2012&amp;pages=30-36&amp;doi=10.5897/JYFR12.007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29">
<cite>Gonzalez-Toril E., Llobet-Brossa E., Casamayor E. O., Amann R., Amils R. (2003). Microbial ecology of an extreme acidic environment, the tinto river.
<em>Appl. Environ. Microbiol.</em>
69
4853–4865. 10.1128/AEM.69.8.4853-4865.2003
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.69.8.4853-4865.2003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC169134/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12902280/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&amp;title=Microbial%20ecology%20of%20an%20extreme%20acidic%20environment,%20the%20tinto%20river.&amp;author=E.%20Gonzalez-Toril&amp;author=E.%20Llobet-Brossa&amp;author=E.%20O.%20Casamayor&amp;author=R.%20Amann&amp;author=R.%20Amils&amp;volume=69&amp;publication_year=2003&amp;pages=4853-4865&amp;pmid=12902280&amp;doi=10.1128/AEM.69.8.4853-4865.2003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30">
<cite>Gooday G. W., Humphreys A. M., McIntosh W. H. (1986). “Roles of chitinases in fungal growth,” in <em>Chitin in Nature and Technology</em>, eds
Muzzarelli R., Jeuniaux C., Gooday G. W. (Boston, MA: Springer; ), 83–91. 10.1007/978-1-4613-2167-5_12</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-1-4613-2167-5_12" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Chitin%20in%20Nature%20and%20Technology&amp;author=G.%20W.%20Gooday&amp;author=A.%20M.%20Humphreys&amp;author=W.%20H.%20McIntosh&amp;publication_year=1986&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31">
<cite>Haas B. J., Salzberg S. L., Zhu W., Pertea M., Allen J. E., Orvis J., et al.  (2008). Automated eukaryotic gene structure annotation using EVidenceModeler and the program to assemble spliced alignments.
<em>Genome Biol.</em>
9:R7. 10.1186/gb-2008-9-1-r7
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/gb-2008-9-1-r7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2395244/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18190707/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20program%20to%20assemble%20spliced%20alignments.&amp;author=B.%20J.%20Haas&amp;author=S.%20L.%20Salzberg&amp;author=W.%20Zhu&amp;author=M.%20Pertea&amp;author=J.%20E.%20Allen&amp;volume=9&amp;issue=R7&amp;publication_year=2008&amp;pmid=18190707&amp;doi=10.1186/gb-2008-9-1-r7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32">
<cite>Heitman J., Howlett B., Crous P., Stukenbrock E., James T., et al.  (eds). (2017). “Stress adaptation,” in <em>The Fungal Kingdom</em> (Washington, DC: American Society of Microbiology; ), 463–485. 10.1128/microbiolspec.FUNK-0048-2016
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/microbiolspec.FUNK-0048-2016" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=The%20Fungal%20Kingdom&amp;author=J.%20Heitman&amp;author=B.%20Howlett&amp;author=P.%20Crous&amp;author=E.%20Stukenbrock&amp;author=T.%20James&amp;publication_year=2017&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B33">
<cite>Hoi J. W. S., Lamarre C., Beau R., Meneau I., Berepiki A., Barre A., et al.  (2011). A novel family of dehydrin-like proteins is involved in stress response in the human fungal pathogen <em>Aspergillus fumigatus</em>.
<em>Mol. Biol. Cell</em>
22
1896–1906. 10.1091/mbc.E10-11-0914
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E10-11-0914" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3103405/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21490150/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=A%20novel%20family%20of%20dehydrin-like%20proteins%20is%20involved%20in%20stress%20response%20in%20the%20human%20fungal%20pathogen%20Aspergillus%20fumigatus.&amp;author=J.%20W.%20S.%20Hoi&amp;author=C.%20Lamarre&amp;author=R.%20Beau&amp;author=I.%20Meneau&amp;author=A.%20Berepiki&amp;volume=22&amp;publication_year=2011&amp;pages=1896-1906&amp;pmid=21490150&amp;doi=10.1091/mbc.E10-11-0914&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34">
<cite>Huerta-Cepas J., Szklarczyk D., Heller D., Hernández-Plaza A., Forslund S. K., Cook H., et al.  (2018). eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses.
<em>Nucleic Acids Res.</em>
47
D309–D314. 10.1093/nar/gky1085
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gky1085" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6324079/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30418610/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=eggNOG%205.0:%20a%20hierarchical,%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses.&amp;author=J.%20Huerta-Cepas&amp;author=D.%20Szklarczyk&amp;author=D.%20Heller&amp;author=A.%20Hern%C3%A1ndez-Plaza&amp;author=S.%20K.%20Forslund&amp;volume=47&amp;publication_year=2018&amp;pages=D309-D314&amp;pmid=30418610&amp;doi=10.1093/nar/gky1085&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35">
<cite>Ianiri G., Boyce K. J., Idnurm A. (2017). Isolation of conditional mutations in genes essential for viability of <em>Cryptococcus neoformans</em>.
<em>Curr. Genet.</em>
63
519–530. 10.1007/s00294-016-0659-2
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00294-016-0659-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5405016/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27783209/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Genet.&amp;title=Isolation%20of%20conditional%20mutations%20in%20genes%20essential%20for%20viability%20of%20Cryptococcus%20neoformans.&amp;author=G.%20Ianiri&amp;author=K.%20J.%20Boyce&amp;author=A.%20Idnurm&amp;volume=63&amp;publication_year=2017&amp;pages=519-530&amp;pmid=27783209&amp;doi=10.1007/s00294-016-0659-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36">
<cite>Keller N. P., Turner G., Bennett J. W. (2005). Fungal secondary metabolism — from biochemistry to genomics.
<em>Nat. Rev. Microbiol.</em>
3
937–947. 10.1038/nrmicro1286
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nrmicro1286" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16322742/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&amp;title=Fungal%20secondary%20metabolism%20%E2%80%94%20from%20biochemistry%20to%20genomics.&amp;author=N.%20P.%20Keller&amp;author=G.%20Turner&amp;author=J.%20W.%20Bennett&amp;volume=3&amp;publication_year=2005&amp;pages=937-947&amp;pmid=16322742&amp;doi=10.1038/nrmicro1286&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B37">
<cite>Knox B. P., Blachowicz A., Palmer J. M., Romsdahl J., Huttenlocher A., Wang C. C. C., et al.  (2016). Characterization of <em>Aspergillus fumigatus</em> isolates from air and surfaces of the international space station.
<em>mSphere</em>
1:e00227–16. 10.1128/mSphere.00227-16
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/mSphere.00227-16" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5082629/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27830189/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=mSphere&amp;title=Characterization%20of%20Aspergillus%20fumigatus%20isolates%20from%20air%20and%20surfaces%20of%20the%20international%20space%20station.&amp;author=B.%20P.%20Knox&amp;author=A.%20Blachowicz&amp;author=J.%20M.%20Palmer&amp;author=J.%20Romsdahl&amp;author=A.%20Huttenlocher&amp;volume=1&amp;issue=e00227&amp;publication_year=2016&amp;pmid=27830189&amp;doi=10.1128/mSphere.00227-16&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B38">
<cite>La Duc M. T., Kern R., Venkateswaran K. (2004). Microbial monitoring of spacecraft and associated environments.
<em>Microb. Ecol.</em>
47
150–158. 10.1007/s00248-003-1012-0
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00248-003-1012-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14749906/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microb.%20Ecol.&amp;title=Microbial%20monitoring%20of%20spacecraft%20and%20associated%20environments.&amp;author=M.%20T.%20La%20Duc&amp;author=R.%20Kern&amp;author=K.%20Venkateswaran&amp;volume=47&amp;publication_year=2004&amp;pages=150-158&amp;pmid=14749906&amp;doi=10.1007/s00248-003-1012-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B39">
<cite>La Duc M. T., Nicholson W., Kern R., Venkateswaran K. (2003). Microbial characterization of the Mars Odyssey spacecraft and its encapsulation facility.
<em>Environ. Microbiol.</em>
5
977–985. 10.1046/j.1462-2920.2003.00496.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1462-2920.2003.00496.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14510851/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&amp;title=Microbial%20characterization%20of%20the%20Mars%20Odyssey%20spacecraft%20and%20its%20encapsulation%20facility.&amp;author=M.%20T.%20La%20Duc&amp;author=W.%20Nicholson&amp;author=R.%20Kern&amp;author=K.%20Venkateswaran&amp;volume=5&amp;publication_year=2003&amp;pages=977-985&amp;pmid=14510851&amp;doi=10.1046/j.1462-2920.2003.00496.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B40">
<cite>La Duc M. T., Vaishampayan P., Nilsson H. R., Torok T., Venkateswaran K. (2012). Pyrosequencing-derived bacterial, archaeal, and fungal diversity of spacecraft hardware destined for Mars.
<em>Appl. Environ. Microbiol.</em>
78
5912–5922. 10.1128/AEM.01435-12
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.01435-12" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3406123/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22729532/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&amp;title=Pyrosequencing-derived%20bacterial,%20archaeal,%20and%20fungal%20diversity%20of%20spacecraft%20hardware%20destined%20for%20Mars.&amp;author=M.%20T.%20La%20Duc&amp;author=P.%20Vaishampayan&amp;author=H.%20R.%20Nilsson&amp;author=T.%20Torok&amp;author=K.%20Venkateswaran&amp;volume=78&amp;publication_year=2012&amp;pages=5912-5922&amp;pmid=22729532&amp;doi=10.1128/AEM.01435-12&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B41">
<cite>Love J., Palmer J., Stajich J., Esser T., Kastman E., Winter D. (2018). <em>Nextgenusfs/Funannotate: Funannotate v1.5.1</em>. Zenodo; 
10.5281/zenodo.1471785</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.5281/zenodo.1471785" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Nextgenusfs/Funannotate:%20Funannotate%20v1.5.1&amp;author=J.%20Love&amp;author=J.%20Palmer&amp;author=J.%20Stajich&amp;author=T.%20Esser&amp;author=E.%20Kastman&amp;publication_year=2018&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B42">
<cite>Martens-Uzunova E. S., Zandleven J. S., Benen J. A. E., Awad H., Kools H. J., Beldman G., et al.  (2006). A new group of exo-acting family 28 glycoside hydrolases of Aspergillus niger that are involved in pectin degradation.
<em>Biochem. J.</em>
400
43–52. 10.1042/BJ20060703
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1042/BJ20060703" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1635439/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16822232/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=A%20new%20group%20of%20exo-acting%20family%2028%20glycoside%20hydrolases%20of%20Aspergillus%20niger%20that%20are%20involved%20in%20pectin%20degradation.&amp;author=E.%20S.%20Martens-Uzunova&amp;author=J.%20S.%20Zandleven&amp;author=J.%20A.%20E.%20Benen&amp;author=H.%20Awad&amp;author=H.%20J.%20Kools&amp;volume=400&amp;publication_year=2006&amp;pages=43-52&amp;pmid=16822232&amp;doi=10.1042/BJ20060703&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B43">
<cite>Masuo S., Terabayashi Y., Shimizu M., Fujii T., Kitazume T., Takaya N. (2010). Global gene expression analysis of <em>Aspergillus nidulans</em> reveals metabolic shift and transcription suppression under hypoxia.
<em>Mol. Genet. Genomics</em>
284
415–424. 10.1007/s00438-010-0576-x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00438-010-0576-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20878186/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Genet.%20Genomics&amp;title=Global%20gene%20expression%20analysis%20of%20Aspergillus%20nidulans%20reveals%20metabolic%20shift%20and%20transcription%20suppression%20under%20hypoxia.&amp;author=S.%20Masuo&amp;author=Y.%20Terabayashi&amp;author=M.%20Shimizu&amp;author=T.%20Fujii&amp;author=T.%20Kitazume&amp;volume=284&amp;publication_year=2010&amp;pages=415-424&amp;pmid=20878186&amp;doi=10.1007/s00438-010-0576-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B44">
<cite>McKay C. P., Friedmann E. I., Gómez-Silva B., Cáceres-Villanueva L., Andersen D. T., Landheim R. (2003). Temperature and moisture conditions for life in the extreme arid region of the Atacama desert: four years of observations including the El Niño of 1997-1998.
<em>Astrobiology</em>
3
393–406. 10.1089/153110703769016460
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/153110703769016460" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14577886/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Temperature%20and%20moisture%20conditions%20for%20life%20in%20the%20extreme%20arid%20region%20of%20the%20Atacama%20desert:%20four%20years%20of%20observations%20including%20the%20El%20Ni%C3%B1o%20of%201997-1998.&amp;author=C.%20P.%20McKay&amp;author=E.%20I.%20Friedmann&amp;author=B.%20G%C3%B3mez-Silva&amp;author=L.%20C%C3%A1ceres-Villanueva&amp;author=D.%20T.%20Andersen&amp;volume=3&amp;publication_year=2003&amp;pages=393-406&amp;pmid=14577886&amp;doi=10.1089/153110703769016460&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B45">
<cite>Mora M., Mahnert A., Koskinen K., Pausan M. R., Oberauner-Wappis L., Krause R., et al.  (2016). Microorganisms in confined habitats: microbial monitoring and control of intensive care units, operating rooms, cleanrooms and the international space station.
<em>Front. Microbiol.</em>
7:1573. 10.3389/fmicb.2016.01573
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2016.01573" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5061736/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27790191/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Microorganisms%20in%20confined%20habitats:%20microbial%20monitoring%20and%20control%20of%20intensive%20care%20units,%20operating%20rooms,%20cleanrooms%20and%20the%20international%20space%20station.&amp;author=M.%20Mora&amp;author=A.%20Mahnert&amp;author=K.%20Koskinen&amp;author=M.%20R.%20Pausan&amp;author=L.%20Oberauner-Wappis&amp;volume=7&amp;issue=1573&amp;publication_year=2016&amp;pmid=27790191&amp;doi=10.3389/fmicb.2016.01573&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B46">
<cite>Newcombe D. A., Schuerger A. C., Benardini J. N., Dickinson D., Tanner R., Venkateswaran K. (2005). Survival of spacecraft-associated microorganisms under simulated martian UV irradiation.
<em>Appl. Environ. Microbiol.</em>
71
8147–8156. 10.1128/AEM.71.12.8147-8156.2005
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.71.12.8147-8156.2005" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1317311/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16332797/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&amp;title=Survival%20of%20spacecraft-associated%20microorganisms%20under%20simulated%20martian%20UV%20irradiation.&amp;author=D.%20A.%20Newcombe&amp;author=A.%20C.%20Schuerger&amp;author=J.%20N.%20Benardini&amp;author=D.%20Dickinson&amp;author=R.%20Tanner&amp;volume=71&amp;publication_year=2005&amp;pages=8147-8156&amp;pmid=16332797&amp;doi=10.1128/AEM.71.12.8147-8156.2005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B47">
<cite>Ng K. P., Yew S. M., Chan C. L., Soo-Hoo T. S., Na S. L., Hassan H., et al.  (2012). Sequencing of <em>Cladosporium sphaerospermum</em>, a dematiaceous fungus isolated from blood culture.
<em>Eukaryot. Cell</em>
11
705–706. 10.1128/EC.00081-12
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/EC.00081-12" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3346424/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22544899/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Eukaryot.%20Cell&amp;title=Sequencing%20of%20Cladosporium%20sphaerospermum,%20a%20dematiaceous%20fungus%20isolated%20from%20blood%20culture.&amp;author=K.%20P.%20Ng&amp;author=S.%20M.%20Yew&amp;author=C.%20L.%20Chan&amp;author=T.%20S.%20Soo-Hoo&amp;author=S.%20L.%20Na&amp;volume=11&amp;publication_year=2012&amp;pages=705-706&amp;pmid=22544899&amp;doi=10.1128/EC.00081-12&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B48">
<cite>Nicholson W. L., Moeller R., Horneck G.
Protect Team (2012). Transcriptomic responses of germinating Bacillus subtilis spores exposed to 1.5 years of space and simulated martian conditions on the EXPOSE-E experiment PROTECT.
<em>Astrobiology</em>
12
469–486. 10.1089/ast.2011.0748
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2011.0748" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22680693/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Transcriptomic%20responses%20of%20germinating%20Bacillus%20subtilis%20spores%20exposed%20to%201.5%20years%20of%20space%20and%20simulated%20martian%20conditions%20on%20the%20EXPOSE-E%20experiment%20PROTECT.&amp;author=W.%20L.%20Nicholson&amp;author=R.%20Moeller&amp;author=G.%20Horneck&amp;volume=12&amp;publication_year=2012&amp;pages=469-486&amp;pmid=22680693&amp;doi=10.1089/ast.2011.0748&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B49">
<cite>Nierman W. C., Pain A., Anderson M. J., Wortman J. R., Kim H. S., Arroyo J., et al.  (2005). Genomic sequence of the pathogenic and allergenic filamentous fungus <em>Aspergillus fumigatus</em>.
<em>Nature</em>
438
1151–1156. 10.1038/nature04332
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature04332" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16372009/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genomic%20sequence%20of%20the%20pathogenic%20and%20allergenic%20filamentous%20fungus%20Aspergillus%20fumigatus.&amp;author=W.%20C.%20Nierman&amp;author=A.%20Pain&amp;author=M.%20J.%20Anderson&amp;author=J.%20R.%20Wortman&amp;author=H.%20S.%20Kim&amp;volume=438&amp;publication_year=2005&amp;pages=1151-1156&amp;pmid=16372009&amp;doi=10.1038/nature04332&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B50">
<cite>Olivas I., Royuela M., Romero B., Monteiro M. C., Mínguez J. M., Laborda F., et al.  (2008). Ability to grow on lipids accounts for the fully virulent phenotype in neutropenic mice of <em>Aspergillus fumigatus</em> null mutants in the key glyoxylate cycle enzymes.
<em>Fungal Genet. Biol.</em>
45
45–60. 10.1016/j.fgb.2007.05.002
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.fgb.2007.05.002" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17616408/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&amp;title=Ability%20to%20grow%20on%20lipids%20accounts%20for%20the%20fully%20virulent%20phenotype%20in%20neutropenic%20mice%20of%20Aspergillus%20fumigatus%20null%20mutants%20in%20the%20key%20glyoxylate%20cycle%20enzymes.&amp;author=I.%20Olivas&amp;author=M.%20Royuela&amp;author=B.%20Romero&amp;author=M.%20C.%20Monteiro&amp;author=J.%20M.%20M%C3%ADnguez&amp;volume=45&amp;publication_year=2008&amp;pages=45-60&amp;pmid=17616408&amp;doi=10.1016/j.fgb.2007.05.002&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B51">
<cite>Onofri S., de la Torre R., de Vera J. P., Ott S., Zucconi L., Selbmann L., et al.  (2012). Survival of rock-colonizing organisms after 1.5 years in outer space.
<em>Astrobiology</em>
12
508–516. 10.1089/ast.2011.0736
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2011.0736" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22680696/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Survival%20of%20rock-colonizing%20organisms%20after%201.5%20years%20in%20outer%20space.&amp;author=S.%20Onofri&amp;author=R.%20de%20la%20Torre&amp;author=J.%20P.%20de%20Vera&amp;author=S.%20Ott&amp;author=L.%20Zucconi&amp;volume=12&amp;publication_year=2012&amp;pages=508-516&amp;pmid=22680696&amp;doi=10.1089/ast.2011.0736&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B52">
<cite>Onofri S., de Vera J.-P., Zucconi L., Selbmann L., Scalzi G., Venkateswaran K. J., et al.  (2015). Survival of antarctic cryptoendolithic fungi in simulated martian conditions on board the international space station.
<em>Astrobiology</em>
15
1052–1059. 10.1089/ast.2015.1324
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2015.1324" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26684504/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Survival%20of%20antarctic%20cryptoendolithic%20fungi%20in%20simulated%20martian%20conditions%20on%20board%20the%20international%20space%20station.&amp;author=S.%20Onofri&amp;author=J.-P.%20de%20Vera&amp;author=L.%20Zucconi&amp;author=L.%20Selbmann&amp;author=G.%20Scalzi&amp;volume=15&amp;publication_year=2015&amp;pages=1052-1059&amp;pmid=26684504&amp;doi=10.1089/ast.2015.1324&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B53">
<cite>Onofri S., Selbmann L., de Hoog G. S., Grube M., Barreca D., Ruisi S., et al.  (2007). Evolution and adaptation of fungi at boundaries of life.
<em>Adv. Space Res.</em>
40
1657–1664. 10.1016/j.asr.2007.06.004</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.asr.2007.06.004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res.&amp;title=Evolution%20and%20adaptation%20of%20fungi%20at%20boundaries%20of%20life.&amp;author=S.%20Onofri&amp;author=L.%20Selbmann&amp;author=G.%20S.%20de%20Hoog&amp;author=M.%20Grube&amp;author=D.%20Barreca&amp;volume=40&amp;publication_year=2007&amp;pages=1657-1664&amp;doi=10.1016/j.asr.2007.06.004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B54">
<cite>Onofri S., Selbmann L., Zucconi L., Pagano S. (2004). Antarctic microfungi as models for exobiology.
<em>Planet. Space Sci.</em>
52
229–237. 10.1016/j.pss.2003.08.019</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.pss.2003.08.019" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planet.%20Space%20Sci.&amp;title=Antarctic%20microfungi%20as%20models%20for%20exobiology.&amp;author=S.%20Onofri&amp;author=L.%20Selbmann&amp;author=L.%20Zucconi&amp;author=S.%20Pagano&amp;volume=52&amp;publication_year=2004&amp;pages=229-237&amp;doi=10.1016/j.pss.2003.08.019&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B55">
<cite>Osman S., Peeters Z., La Duc M. T., Mancinelli R., Ehrenfreund P., Venkateswaran K. (2008). Effect of shadowing on survival of bacteria under conditions simulating the martian atmosphere and UV radiation.
<em>Appl. Environ. Microbiol.</em>
74
959–970. 10.1128/AEM.01973-07
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.01973-07" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2258572/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18083857/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&amp;title=Effect%20of%20shadowing%20on%20survival%20of%20bacteria%20under%20conditions%20simulating%20the%20martian%20atmosphere%20and%20UV%20radiation.&amp;author=S.%20Osman&amp;author=Z.%20Peeters&amp;author=M.%20T.%20La%20Duc&amp;author=R.%20Mancinelli&amp;author=P.%20Ehrenfreund&amp;volume=74&amp;publication_year=2008&amp;pages=959-970&amp;pmid=18083857&amp;doi=10.1128/AEM.01973-07&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B56">
<cite>Pacelli C., Selbmann L., Zucconi L., Coleine C., de Vera J.-P., Rabbow E., et al.  (2018). Responses of the black fungus <em>Cryomyces antarcticus</em> to simulated mars and space conditions on rock analogues.
<em>Astrobiology</em>
19
209–220. 10.1089/ast.2016.1631
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2016.1631" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30067087/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Responses%20of%20the%20black%20fungus%20Cryomyces%20antarcticus%20to%20simulated%20mars%20and%20space%20conditions%20on%20rock%20analogues.&amp;author=C.%20Pacelli&amp;author=L.%20Selbmann&amp;author=L.%20Zucconi&amp;author=C.%20Coleine&amp;author=J.-P.%20de%20Vera&amp;volume=19&amp;publication_year=2018&amp;pages=209-220&amp;pmid=30067087&amp;doi=10.1089/ast.2016.1631&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B57">
<cite>Peeters Z., Vos D., ten Kate I. L., Selch F., van Sluis C. A., Sorokin D. Y., et al.  (2010). Survival and death of the haloarchaeon <em>Natronorubrum</em> strain HG-1 in a simulated martian environment.
<em>Adv. Space Res.</em>
46
1149–1155. 10.1016/j.asr.2010.05.025</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.asr.2010.05.025" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res.&amp;title=Survival%20and%20death%20of%20the%20haloarchaeon%20Natronorubrum%20strain%20HG-1%20in%20a%20simulated%20martian%20environment.&amp;author=Z.%20Peeters&amp;author=D.%20Vos&amp;author=I.%20L.%20ten%20Kate&amp;author=F.%20Selch&amp;author=C.%20A.%20van%20Sluis&amp;volume=46&amp;publication_year=2010&amp;pages=1149-1155&amp;doi=10.1016/j.asr.2010.05.025&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B58">
<cite>Romsdahl J., Blachowicz A., Chiang A. J., Singh N., Stajich J. E., Kalkum M., et al.  (2018). Characterization of <em>Aspergillus niger</em> Isolated from the international space station.
<em>mSystems</em>
3:e00112–18. 10.1128/mSystems.00112-18
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/mSystems.00112-18" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6143729/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30246146/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Characterization%20of%20Aspergillus%20niger%20Isolated%20from%20the%20international%20space%20station.&amp;author=J.%20Romsdahl&amp;author=A.%20Blachowicz&amp;author=A.%20J.%20Chiang&amp;author=N.%20Singh&amp;author=J.%20E.%20Stajich&amp;volume=3&amp;issue=e00112&amp;publication_year=2018&amp;pmid=30246146&amp;doi=10.1128/mSystems.00112-18&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B59">
<cite>Rummel J. D., Stabekis P. D., Devincenzi D. L., Barengoltz J. B. (2002). COSPAR’s planetary protection policy: a consolidated draft.
<em>Adv. Space Res.</em>
30
1567–1571. 10.1016/S0273-1177(02)00479-9</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0273-1177(02)00479-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res.&amp;title=COSPAR%E2%80%99s%20planetary%20protection%20policy:%20a%20consolidated%20draft.&amp;author=J.%20D.%20Rummel&amp;author=P.%20D.%20Stabekis&amp;author=D.%20L.%20Devincenzi&amp;author=J.%20B.%20Barengoltz&amp;volume=30&amp;publication_year=2002&amp;pages=1567-1571&amp;doi=10.1016/S0273-1177(02)00479-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B60">
<cite>Scalzi G., Selbmann L., Zucconi L., Rabbow E., Horneck G., Albertano P., et al.  (2012). LIFE experiment: isolation of cryptoendolithic organisms from Antarctic colonized sandstone exposed to space and simulated Mars conditions on the international space station.
<em>Orig. Life Evol. Biosph.</em>
42
253–262. 10.1007/s11084-012-9282-5
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11084-012-9282-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22688852/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Orig.%20Life%20Evol.%20Biosph.&amp;title=LIFE%20experiment:%20isolation%20of%20cryptoendolithic%20organisms%20from%20Antarctic%20colonized%20sandstone%20exposed%20to%20space%20and%20simulated%20Mars%20conditions%20on%20the%20international%20space%20station.&amp;author=G.%20Scalzi&amp;author=L.%20Selbmann&amp;author=L.%20Zucconi&amp;author=E.%20Rabbow&amp;author=G.%20Horneck&amp;volume=42&amp;publication_year=2012&amp;pages=253-262&amp;pmid=22688852&amp;doi=10.1007/s11084-012-9282-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B61">
<cite>Schuerger A. C., Mancinelli R. L., Kern R. G., Rothschild L. J., McKay C. P. (2003). Survival of endospores of <em>Bacillus subtilis</em> on spacecraft surfaces under simulated martian environments: implications for the forward contamination of Mars.
<em>Icarus</em>
165
253–276. 10.1016/S0019-1035(03)00200-8
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0019-1035(03)00200-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14649627/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Icarus&amp;title=Survival%20of%20endospores%20of%20Bacillus%20subtilis%20on%20spacecraft%20surfaces%20under%20simulated%20martian%20environments:%20implications%20for%20the%20forward%20contamination%20of%20Mars.&amp;author=A.%20C.%20Schuerger&amp;author=R.%20L.%20Mancinelli&amp;author=R.%20G.%20Kern&amp;author=L.%20J.%20Rothschild&amp;author=C.%20P.%20McKay&amp;volume=165&amp;publication_year=2003&amp;pages=253-276&amp;pmid=14649627&amp;doi=10.1016/S0019-1035(03)00200-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B62">
<cite>Schuerger A. C., Richards J. T., Newcombe D. A., Venkateswaran K. (2006). Rapid inactivation of seven <em>Bacillus</em> spp. under simulated Mars UV irradiation.
<em>Icarus</em>
181
52–62. 10.1016/j.icarus.2005.10.008</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.icarus.2005.10.008" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Icarus&amp;title=Rapid%20inactivation%20of%20seven%20Bacillus%20spp.%20under%20simulated%20Mars%20UV%20irradiation.&amp;author=A.%20C.%20Schuerger&amp;author=J.%20T.%20Richards&amp;author=D.%20A.%20Newcombe&amp;author=K.%20Venkateswaran&amp;volume=181&amp;publication_year=2006&amp;pages=52-62&amp;doi=10.1016/j.icarus.2005.10.008&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B63">
<cite>Simão F. A., Waterhouse R. M., Ioannidis P., Kriventseva E. V., Zdobnov E. M. (2015). BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs.
<em>Bioinformatics</em>
31
3210–3212. 10.1093/bioinformatics/btv351
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btv351" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26059717/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BUSCO:%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs.&amp;author=F.%20A.%20Sim%C3%A3o&amp;author=R.%20M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.%20V.%20Kriventseva&amp;author=E.%20M.%20Zdobnov&amp;volume=31&amp;publication_year=2015&amp;pages=3210-3212&amp;pmid=26059717&amp;doi=10.1093/bioinformatics/btv351&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B64">
<cite>Slater G. S. C., Birney E. (2005). Automated generation of heuristics for biological sequence comparison.
<em>BMC Bioinformatics</em>
6:31. 10.1186/1471-2105-6-31
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2105-6-31" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC553969/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15713233/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Automated%20generation%20of%20heuristics%20for%20biological%20sequence%20comparison.&amp;author=G.%20S.%20C.%20Slater&amp;author=E.%20Birney&amp;volume=6&amp;issue=31&amp;publication_year=2005&amp;pmid=15713233&amp;doi=10.1186/1471-2105-6-31&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B65">
<cite>Stanke M., Schöffmann O., Morgenstern B., Waack S. (2006). Gene prediction in eukaryotes with a generalized hidden Markov model that uses hints from external sources.
<em>BMC Bioinformatics</em>
7:62. 10.1186/1471-2105-7-62
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2105-7-62" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1409804/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16469098/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Gene%20prediction%20in%20eukaryotes%20with%20a%20generalized%20hidden%20Markov%20model%20that%20uses%20hints%20from%20external%20sources.&amp;author=M.%20Stanke&amp;author=O.%20Sch%C3%B6ffmann&amp;author=B.%20Morgenstern&amp;author=S.%20Waack&amp;volume=7&amp;issue=62&amp;publication_year=2006&amp;pmid=16469098&amp;doi=10.1186/1471-2105-7-62&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B66">
<cite>Stan-Lotter H., Fendrihan S. (2013). “Survival strategies of halophilic oligotrophic and desiccation resistant prokaryotes,” in <em>Polyextremophiles</em>, eds
Seckbach J., Oren A., Stan-Lotter H. (Dordrecht: Springer; ), 233–248. 10.1007/978-94-007-6488-0_9</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-94-007-6488-0_9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Polyextremophiles&amp;author=H.%20Stan-Lotter&amp;author=S.%20Fendrihan&amp;publication_year=2013&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B67">
<cite>Swiss Institute of Bioinformatics Members [SIB] (2016). The SIB swiss institute of bioinformatics’ resources: focus on curated databases.
<em>Nucleic Acids Res.</em>
44
D27–D37. 10.1093/nar/gkv1310
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkv1310" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4702916/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26615188/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20SIB%20swiss%20institute%20of%20bioinformatics%E2%80%99%20resources:%20focus%20on%20curated%20databases.&amp;volume=44&amp;publication_year=2016&amp;pages=D27-D37&amp;pmid=26615188&amp;doi=10.1093/nar/gkv1310&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B68">
<cite>Tatusov R. L., Galperin M. Y., Natale D. A., Koonin E. V. (2000). The COG database: a tool for genome-scale analysis of protein functions and evolution.
<em>Nucleic Acids Res.</em>
28
33–36. 10.1093/nar/28.1.33
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/28.1.33" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC102395/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10592175/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20COG%20database:%20a%20tool%20for%20genome-scale%20analysis%20of%20protein%20functions%20and%20evolution.&amp;author=R.%20L.%20Tatusov&amp;author=M.%20Y.%20Galperin&amp;author=D.%20A.%20Natale&amp;author=E.%20V.%20Koonin&amp;volume=28&amp;publication_year=2000&amp;pages=33-36&amp;pmid=10592175&amp;doi=10.1093/nar/28.1.33&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B69">
<cite>Ter-Hovhannisyan V., Lomsadze A., Chernoff Y. O., Borodovsky M. (2008). Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training.
<em>Genome Res.</em>
18
1979–1990. 10.1101/gr.081612.108
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.081612.108" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2593577/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18757608/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Gene%20prediction%20in%20novel%20fungal%20genomes%20using%20an%20ab%20initio%20algorithm%20with%20unsupervised%20training.&amp;author=V.%20Ter-Hovhannisyan&amp;author=A.%20Lomsadze&amp;author=Y.%20O.%20Chernoff&amp;author=M.%20Borodovsky&amp;volume=18&amp;publication_year=2008&amp;pages=1979-1990&amp;pmid=18757608&amp;doi=10.1101/gr.081612.108&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B70">
<cite>Teutschbein J., Albrecht D., Pötsch M., Guthke R., Aimanianda V., Clavaud C., et al.  (2010). Proteome profiling and functional classification of intracellular proteins from conidia of the human-pathogenic mold <em>Aspergillus fumigatus</em>.
<em>J. Proteome Res.</em>
9
3427–3442. 10.1021/pr9010684
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/pr9010684" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20507060/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Proteome%20profiling%20and%20functional%20classification%20of%20intracellular%20proteins%20from%20conidia%20of%20the%20human-pathogenic%20mold%20Aspergillus%20fumigatus.&amp;author=J.%20Teutschbein&amp;author=D.%20Albrecht&amp;author=M.%20P%C3%B6tsch&amp;author=R.%20Guthke&amp;author=V.%20Aimanianda&amp;volume=9&amp;publication_year=2010&amp;pages=3427-3442&amp;pmid=20507060&amp;doi=10.1021/pr9010684&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B71">
<cite>Tirumalai M. R., Rastogi R., Zamani N., O’Bryant Williams E., Allen S., Diouf F., et al.  (2013). Candidate genes that may be responsible for the unusual resistances exhibited by <em>Bacillus pumilus</em> SAFR-032 Spores.
<em>PLoS One</em>
8:e66012. 10.1371/journal.pone.0066012
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0066012" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3682946/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23799069/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Candidate%20genes%20that%20may%20be%20responsible%20for%20the%20unusual%20resistances%20exhibited%20by%20Bacillus%20pumilus%20SAFR-032%20Spores.&amp;author=M.%20R.%20Tirumalai&amp;author=R.%20Rastogi&amp;author=N.%20Zamani&amp;author=E.%20O%E2%80%99Bryant%20Williams&amp;author=S.%20Allen&amp;volume=8&amp;issue=e66012&amp;publication_year=2013&amp;pmid=23799069&amp;doi=10.1371/journal.pone.0066012&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B72">
<cite>Twumasi-Boateng K., Yu Y., Chen D., Gravelat F. N., Nierman W. C., Sheppard D. C. (2009). Transcriptional profiling identifies a role for BrlA in the response to nitrogen depletion and for StuA in the regulation of secondary metabolite clusters in <em>Aspergillus fumigatus</em>.
<em>Eukaryot. Cell</em>
8
104–115. 10.1128/EC.00265-08
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/EC.00265-08" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2620752/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19028996/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Eukaryot.%20Cell&amp;title=Transcriptional%20profiling%20identifies%20a%20role%20for%20BrlA%20in%20the%20response%20to%20nitrogen%20depletion%20and%20for%20StuA%20in%20the%20regulation%20of%20secondary%20metabolite%20clusters%20in%20Aspergillus%20fumigatus.&amp;author=K.%20Twumasi-Boateng&amp;author=Y.%20Yu&amp;author=D.%20Chen&amp;author=F.%20N.%20Gravelat&amp;author=W.%20C.%20Nierman&amp;volume=8&amp;publication_year=2009&amp;pages=104-115&amp;pmid=19028996&amp;doi=10.1128/EC.00265-08&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B73">
<cite>Vaishampayan P. A., Rabbow E., Horneck G., Venkateswaran K. J. (2012). Survival of <em>Bacillus pumilus</em> spores for a prolonged period of time in real space conditions.
<em>Astrobiology</em>
12
487–497. 10.1089/ast.2011.0738
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2011.0738" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22680694/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Survival%20of%20Bacillus%20pumilus%20spores%20for%20a%20prolonged%20period%20of%20time%20in%20real%20space%20conditions.&amp;author=P.%20A.%20Vaishampayan&amp;author=E.%20Rabbow&amp;author=G.%20Horneck&amp;author=K.%20J.%20Venkateswaran&amp;volume=12&amp;publication_year=2012&amp;pages=487-497&amp;pmid=22680694&amp;doi=10.1089/ast.2011.0738&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B74">
<cite>Weinmaier T., Probst A. J., La Duc M. T., Ciobanu D., Cheng J.-F., Ivanova N., et al.  (2015). A viability-linked metagenomic analysis of cleanroom environments: eukarya, prokaryotes, and viruses.
<em>Microbiome</em>
3:62. 10.1186/s40168-015-0129-y
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40168-015-0129-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4672508/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26642878/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=A%20viability-linked%20metagenomic%20analysis%20of%20cleanroom%20environments:%20eukarya,%20prokaryotes,%20and%20viruses.&amp;author=T.%20Weinmaier&amp;author=A.%20J.%20Probst&amp;author=M.%20T.%20La%20Duc&amp;author=D.%20Ciobanu&amp;author=J.-F.%20Cheng&amp;volume=3&amp;issue=62&amp;publication_year=2015&amp;pmid=26642878&amp;doi=10.1186/s40168-015-0129-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B75">
<cite>Zakharova K., Marzban G., de Vera J.-P., Lorek A., Sterflinger K. (2014). Protein patterns of black fungi under simulated Mars-like conditions.
<em>Sci. Rep.</em>
4:5114. 10.1038/srep05114
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/srep05114" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4037706/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24870977/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Protein%20patterns%20of%20black%20fungi%20under%20simulated%20Mars-like%20conditions.&amp;author=K.%20Zakharova&amp;author=G.%20Marzban&amp;author=J.-P.%20de%20Vera&amp;author=A.%20Lorek&amp;author=K.%20Sterflinger&amp;volume=4&amp;issue=5114&amp;publication_year=2014&amp;pmid=24870977&amp;doi=10.1038/srep05114&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B76">
<cite>Zhdanova N. N., Tugay T., Dighton J., Zheltonozhsky V., Mcdermott P. (2004). Ionizing radiation attracts soil fungi.
<em>Mycol. Res.</em>
108
1089–1096. 10.1017/S0953756204000966
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1017/S0953756204000966" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15506020/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycol.%20Res.&amp;title=Ionizing%20radiation%20attracts%20soil%20fungi.&amp;author=N.%20N.%20Zhdanova&amp;author=T.%20Tugay&amp;author=J.%20Dighton&amp;author=V.%20Zheltonozhsky&amp;author=P.%20Mcdermott&amp;volume=108&amp;publication_year=2004&amp;pages=1089-1096&amp;pmid=15506020&amp;doi=10.1017/S0953756204000966&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ol></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/6529585/bin/Data_Sheet_1.PDF">Click here for additional data file.</a><sup> (667KB, PDF) </sup>
</div></div></section></section></section></section></section></article>
</main></body></html>