<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4826010/"/><title>HLB1 Is a Tetratricopeptide Repeat Domain-Containing Protein That Operates at the Intersection of the Exocytic and Endocytic Pathways at the TGN/EE in Arabidopsis - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Plant Cell</button></div>. 2016 Mar 3;28(3):746–769. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.15.00794" rel="noopener noreferrer" target="_blank">10.1105/tpc.15.00794</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>HLB1 Is a Tetratricopeptide Repeat Domain-Containing Protein That Operates at the Intersection of the Exocytic and Endocytic Pathways at the TGN/EE in Arabidopsis<sup><a class="usa-link" href="#fn1"><sup>[OPEN]</sup></a></sup>
</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sparks%20JA%22%5BAuthor%5D"><span class="name western">J Alan Sparks</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">J Alan Sparks</span></h3>
<div class="p">
<sup>a</sup>Plant Biology Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sparks%20JA%22%5BAuthor%5D"><span class="name western">J Alan Sparks</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwon%20T%22%5BAuthor%5D"><span class="name western">Taegun Kwon</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Taegun Kwon</span></h3>
<div class="p">
<sup>a</sup>Plant Biology Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwon%20T%22%5BAuthor%5D"><span class="name western">Taegun Kwon</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Renna%20L%22%5BAuthor%5D"><span class="name western">Luciana Renna</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Luciana Renna</span></h3>
<div class="p">
<sup>b</sup>Department of Energy, Plant Research Laboratory, Michigan State University, East Lansing, Michigan 48824</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Renna%20L%22%5BAuthor%5D"><span class="name western">Luciana Renna</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20F%22%5BAuthor%5D"><span class="name western">Fuqi Liao</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Fuqi Liao</span></h3>
<div class="p">
<sup>c</sup>Computing Services Department, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401 </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20F%22%5BAuthor%5D"><span class="name western">Fuqi Liao</span></a>
</div>
</div>
<sup>c</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brandizzi%20F%22%5BAuthor%5D"><span class="name western">Federica Brandizzi</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Federica Brandizzi</span></h3>
<div class="p">
<sup>b</sup>Department of Energy, Plant Research Laboratory, Michigan State University, East Lansing, Michigan 48824</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brandizzi%20F%22%5BAuthor%5D"><span class="name western">Federica Brandizzi</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blancaflor%20EB%22%5BAuthor%5D"><span class="name western">Elison B Blancaflor</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Elison B Blancaflor</span></h3>
<div class="p">
<sup>a</sup>Plant Biology Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blancaflor%20EB%22%5BAuthor%5D"><span class="name western">Elison B Blancaflor</span></a>
</div>
</div>
<sup>a,</sup><sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>a</sup>Plant Biology Division, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401</div>
<div id="aff2">
<sup>b</sup>Department of Energy, Plant Research Laboratory, Michigan State University, East Lansing, Michigan 48824</div>
<div id="aff3">
<sup>c</sup>Computing Services Department, The Samuel Roberts Noble Foundation, Ardmore, Oklahoma 73401 </div>
<div class="author-notes p">
<div class="fn" id="fn1_pmc"><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/doi/10.1105/tpc.15.00794" rel="noopener noreferrer" target="_blank">www.plantcell.org/cgi/doi/10.1105/tpc.15.00794</a></p></div>
<div class="fn" id="cor1">
<sup>1</sup><p class="display-inline">Address correspondence to <span>eblancaflor@noble.org</span>.</p>
</div>
<div class="fn" id="afn1"><p>The author responsible for distribution of materials integral to the findings presented in this article in accordance with the policy described in the Instructions for Authors (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org" rel="noopener noreferrer" target="_blank">www.plantcell.org</a>) is: Elison B. Blancaflor (<span>eblancaflor@noble.org</span>).</p></div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2015 Sep 15; Revised 2016 Feb 16; Accepted 2016 Feb 25; Issue date 2016 Mar.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© 2016 American Society of Plant Biologists. All rights reserved.</div>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC4826010  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26941089/">26941089</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><hr class="headless"/>
<p>A forward-genetic screen for Arabidopsis mutants hypersensitive to latrunculin B identifies HLB1 as a plant-specific protein that functions in post-Golgi traffic.</p></section><section class="abstract" id="abstract2"><h2>Abstract</h2>
<p>The endomembrane system plays essential roles in plant development, but the proteome responsible for its function and organization remains largely uncharacterized in plants. Here, we identified and characterized the HYPERSENSITIVE TO LATRUNCULIN B1 (HLB1) protein isolated through a forward-genetic screen in <em>Arabidopsis thaliana</em> for mutants with heightened sensitivity to actin-disrupting drugs. HLB1 is a plant-specific tetratricopeptide repeat domain-containing protein of unknown function encoded by a single Arabidopsis gene. HLB1 associated with the <em>trans</em>-Golgi network (<a class="usa-link" href="#def3">TGN</a>)/early endosome (<a class="usa-link" href="#def4">EE</a>) and tracked along filamentous actin, indicating that it could link post-Golgi traffic with the actin cytoskeleton in plants. HLB1 was found to interact with the ADP-ribosylation-factor guanine nucleotide exchange factor, MIN7/BEN1 (HOPM INTERACTOR7/BREFELDIN A-VISUALIZED ENDOCYTIC TRAFFICKING DEFECTIVE1) by coimmunoprecipitation. The <em>min7/ben1</em> mutant phenocopied the mild root developmental defects and latrunculin B hypersensitivity of <em>hlb1</em>, and analyses of a <em>hlb1/ min7/ben1</em> double mutant showed that <em>hlb1</em> and <em>min7/ben1</em> operate in common genetic pathways. Based on these data, we propose that HLB1 together with MIN7/BEN1 form a complex with actin to modulate the function of the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> at the intersection of the exocytic and endocytic pathways in plants.</p></section><section id="s1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p>Normal plant development relies on the endomembrane system as it plays a central role in directing the delivery of enzyme complexes and polysaccharide precursors to the cell surface and to the vacuole (<a aria-describedby="bib59" class="usa-link" href="#bib59">Kim and Brandizzi, 2014</a>; <a aria-describedby="bib60" class="usa-link" href="#bib60">Kim et al., 2015</a>). In addition, protein traffic through endomembranes is pivotal for specifying how plants respond and adapt to a range of biotic and abiotic stresses (<a aria-describedby="bib14" class="usa-link" href="#bib14">Chen et al., 2008</a>; <a aria-describedby="bib68" class="usa-link" href="#bib68">Lozano‐Durán et al., 2014</a>; <a aria-describedby="bib37" class="usa-link" href="#bib37">Garcia de la Garma et al., 2015</a>). The endomembrane system is highly dynamic and consists of a network of functionally interconnected organelles beginning with the endoplasmic reticulum (<a class="usa-link" href="#def1">ER</a>) where proteins are synthesized and then trafficked to the plasma membrane (<a class="usa-link" href="#def2">PM</a>) and vacuole after processing and sorting in the Golgi apparatus and post-Golgi organelles (<a aria-describedby="bib89" class="usa-link" href="#bib89">Richter et al., 2009</a>; <a aria-describedby="bib12" class="usa-link" href="#bib12">Brandizzi and Barlowe, 2013</a>). Recycling and degradation of <a class="usa-link" href="#def2">PM</a> proteins, which also are facilitated by the endomembrane trafficking machinery, are essential for establishing and maintaining the polar localization of lipids and critical proteins, including auxin carriers, transmembrane receptors, channels, and ion transporters that regulate diverse physiological processes in plants (<a aria-describedby="bib41" class="usa-link" href="#bib41">Geldner et al., 2007</a>; <a aria-describedby="bib99" class="usa-link" href="#bib99">Takano et al., 2010</a>; <a aria-describedby="bib5" class="usa-link" href="#bib5">Barberon and Geldner, 2014</a>; <a aria-describedby="bib47" class="usa-link" href="#bib47">Hachez et al., 2014</a>; <a aria-describedby="bib27" class="usa-link" href="#bib27">Doyle et al., 2015</a>).</p>
<p>Among the various organelles that comprise the endomembrane system, the <em>trans</em>-Golgi network (<a class="usa-link" href="#def3">TGN</a>) is arguably the most mysterious. The complexity of <a class="usa-link" href="#def3">TGN</a> function stems from the fact that it lies at the intersection of secretory and endocytic pathways, through which cargo destined for the <a class="usa-link" href="#def2">PM</a> and vacuole transit (<a aria-describedby="bib18" class="usa-link" href="#bib18">Contento and Bassham, 2012</a>; <a aria-describedby="bib42" class="usa-link" href="#bib42">Gendre et al., 2015</a>). In contrast to mammalian cells, there is mounting evidence that the <a class="usa-link" href="#def3">TGN</a> in plants functions as an early endosome (<a class="usa-link" href="#def4">EE</a>) that directly receives cargo taken up via endocytosis for recycling back to the <a class="usa-link" href="#def2">PM</a> or directed to the vacuole for degradation (<a aria-describedby="bib26" class="usa-link" href="#bib26">Dettmer et al., 2006</a>; <a aria-describedby="bib111" class="usa-link" href="#bib111">Viotti et al., 2010</a>). In recent years, much has been learned about the structure and dynamics of the plant <a class="usa-link" href="#def3">TGN</a>. Using transmission electron tomography on high-pressure frozen <em>Arabidopsis thaliana</em> roots, for example, it was shown that the formation of the <a class="usa-link" href="#def3">TGN</a> occurs through distinct structural changes including separation of <em>trans</em>-most Golgi cisterna via a peeling mechanism and secretory vesicle budding from the plane of the original <em>trans</em>-Golgi cisterna (<a aria-describedby="bib56" class="usa-link" href="#bib56">Kang et al., 2011</a>). In another study, live-cell spinning-disc confocal microscopy of Arabidopsis hypocotyl cells uncovered the formation of thin tube-like protrusions connecting two <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> units prior to their homotypic fusion (<a aria-describedby="bib111" class="usa-link" href="#bib111">Viotti et al., 2010</a>). Furthermore, live-cell super-resolution microscopy described two types of <a class="usa-link" href="#def3">TGN</a> in Arabidopsis roots: a Golgi-associated <a class="usa-link" href="#def3">TGN</a> on the <em>trans</em>-side of the Golgi apparatus and a Golgi-independent <a class="usa-link" href="#def3">TGN</a> that moved independently of Golgi bodies (<a aria-describedby="bib106" class="usa-link" href="#bib106">Uemura et al., 2014</a>).</p>
<p>In addition to new knowledge about its structure and dynamics, several protein components of the plant <a class="usa-link" href="#def3">TGN</a> have been discovered that mediate crucial aspects of post-Golgi membrane traffic and maintain <a class="usa-link" href="#def3">TGN</a> integrity. These <a class="usa-link" href="#def3">TGN</a> resident proteins include soluble <em>N</em>-ethylmaleimide-sensitive factor attachment protein receptors, the so-called SYNTAXINS of PLANTS (SYPs), vacuolar H<sup>+</sup>-ATPases, adaptor protein complexes, small monomeric GTPases of the ADP ribosylation factor (ARF) and RAS genes from rat brain (RAB) families, and the ECHIDNA/YPT/RAB GTPase complex (<a aria-describedby="bib26" class="usa-link" href="#bib26">Dettmer et al., 2006</a>; <a aria-describedby="bib15" class="usa-link" href="#bib15">Chow et al., 2008</a>; <a aria-describedby="bib120" class="usa-link" href="#bib120">Zouhar et al., 2009</a>; <a aria-describedby="bib44" class="usa-link" href="#bib44">Gendre et al., 2011, 2013</a>; <a aria-describedby="bib107" class="usa-link" href="#bib107">Uemura et al., 2012</a>; <a aria-describedby="bib4" class="usa-link" href="#bib4">Asaoka et al., 2013</a>; <a aria-describedby="bib76" class="usa-link" href="#bib76">Naramoto et al., 2014</a>). Whereas many of the plant <a class="usa-link" href="#def3">TGN</a>-localized proteins are evolutionarily conserved in eukaryotes, some appear to be unique to the plant post-Golgi trafficking machinery (<a aria-describedby="bib31" class="usa-link" href="#bib31">Ebine et al., 2008</a>, <a aria-describedby="bib32" class="usa-link" href="#bib32">2011</a>; <a aria-describedby="bib36" class="usa-link" href="#bib36">Fujimoto and Ueda, 2012</a>), and it is likely that more factors remain uncharacterized.</p>
<p>Mobility of organelles within the cell is typically driven by the cytoskeleton, a network of filamentous proteins that consists of microtubules, microfilaments, and intermediate filaments. Microfilaments, also known as filamentous actin (F-actin), assemble in an energy-dependent manner from monomeric globular actin (G-actin) units to form two-stranded, helical structures that are ∼5 to 7 nm in diameter (<a aria-describedby="bib67" class="usa-link" href="#bib67">Li et al., 2014</a>). The higher order organization of the actin cytoskeleton consists mostly of bundles and cables, and their assembly into these structures is specified by several actin binding proteins such as fimbrin, villin, profilin, LIM domain-containing proteins, thrumin, and crolin (<a aria-describedby="bib82" class="usa-link" href="#bib82">Papuga et al., 2010</a>; <a aria-describedby="bib114" class="usa-link" href="#bib114">Whippo et al., 2011</a>; <a aria-describedby="bib97" class="usa-link" href="#bib97">Su et al., 2012</a>; <a aria-describedby="bib53" class="usa-link" href="#bib53">Jia et al., 2013</a>; <a aria-describedby="bib87" class="usa-link" href="#bib87">Qu et al., 2013</a>; <a aria-describedby="bib98" class="usa-link" href="#bib98">Sun et al., 2013</a>). The movement and dynamics of the <a class="usa-link" href="#def1">ER</a> and the various organelles that make up the plant endomembrane system are largely dependent on the actin cytoskeleton, as demonstrated using actin-disrupting drugs such as latrunculin B (<a class="usa-link" href="#def5">LatB</a>) and cytochalasin D (<a aria-describedby="bib9" class="usa-link" href="#bib9">Boevink et al., 1998</a>; <a aria-describedby="bib77" class="usa-link" href="#bib77">Nebenführ et al., 1999</a>; <a aria-describedby="bib93" class="usa-link" href="#bib93">Sparkes et al., 2009</a>). However, actin appears to be dispensable for protein transport between the <a class="usa-link" href="#def1">ER</a> and Golgi apparatus (<a aria-describedby="bib13" class="usa-link" href="#bib13">Brandizzi et al., 2002</a>). While it was shown that cargo traffic from the Golgi to the vacuoles requires an intact actin cytoskeleton, the underlying mechanisms by which this process is accomplished are unknown (<a aria-describedby="bib58" class="usa-link" href="#bib58">Kim et al., 2005</a>).</p>
<p>There is a wealth of information on the relationship between post-Golgi traffic and its mediation by the actin cytoskeleton in yeast and animal cells. Most of the evidence to date in these systems has pointed toward a pivotal role for F-actin and its associated regulatory and motor proteins in post-Golgi vesicle biogenesis and membrane deformation (<a aria-describedby="bib3" class="usa-link" href="#bib3">Anitei and Hoflack, 2012</a>). In mammalian HeLa cells, for example, myosin 1b was shown to generate the forces that regulate the assembly of F-actin foci for tubule formation at the <a class="usa-link" href="#def3">TGN</a> (<a aria-describedby="bib1" class="usa-link" href="#bib1">Almeida et al., 2011</a>). In another class of mammalian cells called 3T3, the actin-related protein 2/3 (ARP2/3) complex and their activators in the WASP (Wiskott-Aldrich syndrome protein) family drive the nucleation of F-actin on endosomes, which is necessary to promote endosome fission (<a aria-describedby="bib25" class="usa-link" href="#bib25">Derivery et al., 2009</a>). Recently, a combination of elegant optical tweezer and live-cell confocal imaging in human epithelial cells demonstrated that the actin cytoskeleton confers rigidity to the Golgi complex, making it stiffer than the surrounding cytoplasm. It was proposed that mechanical stress imposed by actin affects membrane budding and fission during intra-Golgi transport (<a aria-describedby="bib46" class="usa-link" href="#bib46">Guet et al., 2014</a>).</p>
<p>In plants, several actin binding proteins appear to link actin dynamics with membrane trafficking. Many of these proteins, such as the ARP2/3 complex, profilin, formins, and capping proteins, which localize to the <a class="usa-link" href="#def2">PM</a> and various membranes of organelles within the endomembrane system, have orthologs in other eukaryotes (<a aria-describedby="bib117" class="usa-link" href="#bib117">Zhang et al., 2010</a>, <a aria-describedby="bib118" class="usa-link" href="#bib118">2013</a>; <a aria-describedby="bib98" class="usa-link" href="#bib98">Sun et al., 2013</a>; <a aria-describedby="bib54" class="usa-link" href="#bib54">Jimenez-Lopez et al., 2014</a>; <a aria-describedby="bib22" class="usa-link" href="#bib22">Cvrčková et al., 2015</a>). In fact, some of the mechanisms by which F-actin on membranes is nucleated and polymerized in plants mirror those of animals and yeast, where small GTPases of the Rho family directly activate membrane-localized actin nucleators (<a aria-describedby="bib100" class="usa-link" href="#bib100">Takenawa and Suetsugu, 2007</a>). For example, genetic and biochemical studies in Arabidopsis showed that a Rho of plants (ROP)-guanine nucleotide exchange factor (GEF) protein called SPIKE1 regulates F-actin polymerization through the ARP2/3 complex (<a aria-describedby="bib6" class="usa-link" href="#bib6">Basu et al., 2008</a>; <a aria-describedby="bib117" class="usa-link" href="#bib117">Zhang et al., 2010</a>).</p>
<p>Recently, a number of plant-specific proteins that function as actin membrane adaptors have been uncovered. One example is RISAP, a <a class="usa-link" href="#def3">TGN</a>-localized ROP-GTPase effector that interacts with F-actin and binds to the globular tail domain of myosin XI in tobacco (<em>Nicotiana tabacum</em>) pollen tubes. RISAP is ∼33% identical to the Arabidopsis family of myosin binding proteins at the amino acid level (<a aria-describedby="bib85" class="usa-link" href="#bib85">Peremyslov et al., 2013</a>). It was proposed that the RISAP-mediated interaction between the actin cytoskeleton and endomembrane system is an essential component of the tip growth machinery in pollen tubes (<a aria-describedby="bib96" class="usa-link" href="#bib96">Stephan et al., 2014</a>). Another family of plant-specific proteins that mediate actin-endomembrane crosstalk is the 13-member Networked (NET) proteins. These NET proteins interacted directly with F-actin through a conserved N-terminal domain. The NET proteins were also shown to localize to <a class="usa-link" href="#def2">PM</a>-<a class="usa-link" href="#def1">ER</a> contact sites and the tonoplast (<a aria-describedby="bib24" class="usa-link" href="#bib24">Deeks et al., 2012</a>; <a aria-describedby="bib112" class="usa-link" href="#bib112">Wang et al., 2014</a>).</p>
<p>Here, we report the discovery of a plant-specific protein called HLB1 (for HYPERSENSITIVE TO LATRUNCULIN B1) through a forward-genetic screen in Arabidopsis seedlings for mutants that exhibited differential responses to the inhibitory effects of <a class="usa-link" href="#def5">LatB</a> on primary root growth. We determined that HLB1 tracks along F-actin and colocalizes with the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> in vivo and that <em>hlb1</em> has marked defects in the recycling of proteins to the <a class="usa-link" href="#def2">PM</a> after endocytic uptake. Furthermore, we identified the ARF-GEF, MIN7/BEN1 (<a aria-describedby="bib79" class="usa-link" href="#bib79">Nomura et al., 2006</a>; <a aria-describedby="bib102" class="usa-link" href="#bib102">Tanaka et al., 2009</a>), as an HLB1 interactor and showed that <em>hlb1</em> functions in a common genetic pathway as <em>min7/ben1</em> in regard to <a class="usa-link" href="#def5">LatB</a> hypersensitivity despite their opposite effects on membrane trafficking. These findings are significant because they support the conclusion that HLB1 represents a class of plant-specific proteins that function at the crossroads of exocytic and endocytic trafficking pathways, possibly through mechanisms coordinated by the actin cytoskeleton.</p></section><section id="s2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="s3"><h3 class="pmc_sec_title">Isolation and Characterization of Arabidopsis Mutants Hypersensitive to <a class="usa-link" href="#def5">LatB</a>
</h3>
<p>Approximately 20,000 T-DNA activation-tagged seedlings in the Col-0 background were grown on media supplemented with 100 nM <a class="usa-link" href="#def5">LatB</a>. Seedlings with significantly reduced primary root length were isolated and their progeny were retested for increased <a class="usa-link" href="#def5">LatB</a> sensitivity. Three nonallelic mutants whose primary roots were hypersensitive to <a class="usa-link" href="#def5">LatB</a> were identified and <em>hlb1</em> was selected for further study (<a class="usa-link" href="#fig1">Figures 1A</a> to <a class="usa-link" href="#fig1">1D</a>). Hypocotyls of <em>hlb1</em> grown in the dark and root hairs of light-grown seedlings were also hypersensitive to <a class="usa-link" href="#def5">LatB</a> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 1</a>). Moreover, <em>hlb1</em> primary roots displayed increased sensitivity to cytochalasins, another class of actin-disrupting compounds (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 2</a>). Analysis of 224 F2 seedlings revealed a 172:52 segregation of wild type:mutant for a 3:1 ratio (calculated χ<sup>2</sup> = 0.381 &lt; χ<sup>2</sup><sub>0.95</sub> = 3.841), indicating that <em>hlb1</em> is recessive in a single Mendelian locus. Homozygous <em>hlb1</em> plants were backcrossed at least three times with Col-0 to reduce the frequency of extraneous mutations.</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4826010_PC_TPC201500794RAR2_f1.jpg" target="_blank"><img alt="Figure 1." class="graphic zoom-in" height="1536" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bed/4826010/95fd353b9a32/PC_TPC201500794RAR2_f1.jpg" width="717"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Primary Root Growth of <em>hlb1</em> Is Hypersensitive to <a class="usa-link" href="#def5">LatB</a>.</p>
<p><strong>(A)</strong> to <strong>(C)</strong> Seedlings of the wild type (Col-0 ecotype) and <em>hlb1</em> 2 d (<strong>[A]</strong> and <strong>[B]</strong>) and 6 d <strong>(C)</strong> after transplanting 3-d-old seedlings to growth media supplemented with the solvent control solution <strong>(A)</strong> or 50 nM <a class="usa-link" href="#def5">LatB</a> (<strong>[B]</strong> and <strong>[C]</strong>). Note <em>hlb1</em> grown in <a class="usa-link" href="#def5">LatB</a> has shorter primary roots compared with the wild type. Bars = 5 mm.</p>
<p><strong>(D)</strong> Dose–response analyses of primary root growth (expressed as percent of control) of the wild type and <em>hlb1</em> 7 d after transplanting 4-d-old seedlings to different concentrations of <a class="usa-link" href="#def5">LatB</a>. Statistical significance was determined by one-way ANOVA. Means (<em>n</em> = 45 to 60 roots) ± <span class="font-variant-small-caps">se</span>. Different letters are significantly different (P &lt; 0.05, Tukey’s test).</p></figcaption></figure><p>Osmotic stress, salt stress, and the auxins indole-3-acetic acid and 1-naphthaleneacetic acid inhibited primary root growth in <em>hlb1</em> to a similar extent as in the wild type, indicating that <em>hlb1</em> is hypersensitive only to <a class="usa-link" href="#def5">LatB</a> and cytochalasin, compounds that directly interfere with actin polymerization (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 3</a>). The response of other mutants to genes encoding actin regulatory proteins including single mutants <em>brk1</em>, <em>arp2</em>, <em>adf4</em>, <em>cpa1</em>, <em>cpb1</em>, and <em>cpb3</em> and quadruple mutant <em>scar1234</em> (<a aria-describedby="bib72" class="usa-link" href="#bib72">Mathur et al., 2003</a>; <a aria-describedby="bib30" class="usa-link" href="#bib30">Dyachok et al., 2011</a>; <a aria-describedby="bib65" class="usa-link" href="#bib65">Li et al., 2012</a>, <a aria-describedby="bib66" class="usa-link" href="#bib66">2015</a>; <a aria-describedby="bib48" class="usa-link" href="#bib48">Henty-Ridilla et al., 2014</a>) to <a class="usa-link" href="#def5">LatB</a> was also compared with <em>hlb1</em>. These mutants exhibited wild-type sensitivity to <a class="usa-link" href="#def5">LatB</a>. <em>act8</em>, a mutant of a root-expressed vegetative actin isoform, also showed wild-type sensitivity to <a class="usa-link" href="#def5">LatB</a>. Although <em>act2</em>, a mutant of another root-expressed vegetative actin isoform (<a aria-describedby="bib55" class="usa-link" href="#bib55">Kandasamy et al., 2009</a>), was hypersensitive to <a class="usa-link" href="#def5">LatB</a>, its heightened sensitivity to the chemical was not as severe as that of <em>hlb1</em> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 4</a>).</p></section><section id="s4"><h3 class="pmc_sec_title">Cell Division, Cell Expansion, and F-Actin Organization in <em>hlb1</em> Are Hypersensitive to <a class="usa-link" href="#def5">LatB</a>
</h3>
<p>We asked whether the increased sensitivity of <em>hlb1</em> roots to <a class="usa-link" href="#def5">LatB</a> could be explained by inhibition of cell division, cell expansion, or both. To address this question, cortical cell length in the root elongation zone and cortical cell number in the meristem was obtained from confocal microscopy analyses of propidium iodide-stained roots 2 d after transplanting seedlings on media with or without 50 nM <a class="usa-link" href="#def5">LatB</a> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 5A</a>). Both cortical cell number in the root meristem and cortical cell length in the elongation zone were reduced in <a class="usa-link" href="#def5">LatB</a>-treated <em>hlb1</em> seedlings compared with the wild type, indicating that inhibition of both processes contributed to the hypersensitivity of the mutant to the drug (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figures 5B to 5E</a>).</p>
<p>We next analyzed F-actin organization in cells of the elongation zone of wild-type and <em>hlb1</em> seedlings expressing the live-cell F-actin reporter <em>UBQ10pro:GFP-ABD2-GFP</em> (<a aria-describedby="bib28" class="usa-link" href="#bib28">Dyachok et al., 2014</a>) transplanted to growth media with or without 50 nM <a class="usa-link" href="#def5">LatB</a>. Cells in the root elongation zone of both genotypes on control medium showed dense and random F-actin arrays, a feature typical of cells within this region (<a class="usa-link" href="#fig2">Figure 2A</a>). When transplanted on <a class="usa-link" href="#def5">LatB</a>, wild-type cells maintained their dense network of F-actin that roughly mirrored the controls with the exception of a few fluorescent aggregates. By contrast, cells in the root elongation zone of <a class="usa-link" href="#def5">LatB</a>-treated <em>hlb1</em> had mostly diffuse fluorescence, fewer distinct F-actin arrays, and more numerous fluorescent aggregates compared with the wild type (<a class="usa-link" href="#fig2">Figure 2A</a>). To quantify the effect of <a class="usa-link" href="#def5">LatB</a> on F-actin integrity, filament density (occupancy) in cells from the elongation zone was measured using an algorithm developed earlier (<a aria-describedby="bib49" class="usa-link" href="#bib49">Higaki et al., 2010</a>). Whereas the wild type and <em>hlb1</em> on the control medium and wild type on <a class="usa-link" href="#def5">LatB</a>-supplemented medium had similar occupancy values, <a class="usa-link" href="#def5">LatB</a>-treated <em>hlb1</em> had significantly lower F-actin occupancy values, indicating that F-actin structures of <em>hlb1</em> were indeed more sensitive to the chemical (<a class="usa-link" href="#fig2">Figure 2B</a>).</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4826010_PC_TPC201500794RAR2_f2.jpg" target="_blank"><img alt="Figure 2." class="graphic zoom-in" height="1452" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bed/4826010/dea6f98db090/PC_TPC201500794RAR2_f2.jpg" width="789"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>F-Actin of <em>hlb1</em> Roots Is More Sensitive to <a class="usa-link" href="#def5">LatB</a>.</p>
<p><strong>(A)</strong> Representative images of F-actin organization in the elongation zone of the wild type and <em>hlb1</em> expressing <em>UBQ10pro:GFP-ABD2-GFP</em> 2 d after transplanting 3-d-old seedlings on media supplemented with the solvent control solution or 50 nM <a class="usa-link" href="#def5">LatB</a>. Bar = 20 μm.</p>
<p><strong>(B)</strong> Quantitative analysis of F-actin density (occupancy) in epidermal cells from the root elongation zone. Statistical significance was determined by one way ANOVA. Means (<em>n</em> &gt; 50 cells) ± <span class="font-variant-small-caps">se</span>. Different letters are significantly different (P &lt; 0.05, Tukey’s test).</p></figcaption></figure></section><section id="s5"><h3 class="pmc_sec_title">
<em>hlb1</em> Exhibits Reduced Root Hair and Primary Root Growth in the Absence of <a class="usa-link" href="#def5">LatB</a>
</h3>
<p>Close examination of <em>hlb1</em> seedlings revealed that they had mild growth defects even when <a class="usa-link" href="#def5">LatB</a> was excluded from the growth medium. Primary roots of <em>hlb1</em> were slightly shorter than the wild type at 3, 5, and 7 d after planting (<a class="usa-link" href="#fig3">Figures 3A</a> and <a class="usa-link" href="#fig3">3B</a>). <em>hlb1</em> root hairs were also shorter than those of the wild type (<a class="usa-link" href="#fig3">Figures 3C</a> to <a class="usa-link" href="#fig3">3E</a>). Furthermore, differential interference contrast (<a class="usa-link" href="#def6">DIC</a>) microscopy showed that <em>hlb1</em> root hairs were wider and had subtle shape distortions when compared with wild-type root hairs (<a class="usa-link" href="#fig3">Figure 3F</a>).</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4826010_PC_TPC201500794RAR2_f3.jpg" target="_blank"><img alt="Figure 3." class="graphic zoom-in" height="1411" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bed/4826010/50f6a019c61e/PC_TPC201500794RAR2_f3.jpg" width="789"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Primary Roots and Root Hairs of <em>hlb1</em> Are Shorter Than the Wild Type in the Absence of <a class="usa-link" href="#def5">LatB</a>.</p>
<p><strong>(A)</strong> Seven-day-old wild-type and <em>hlb1</em> seedlings grown vertically on <a class="usa-link" href="#def5">LatB</a>-free growth medium. Bar = 5 mm.</p>
<p><strong>(B)</strong> Quantification of primary root length at 3, 5, and 7 d after planting on <a class="usa-link" href="#def5">LatB</a>-free medium. Values are means (<em>n</em> = 50 to 65 roots) ± <span class="font-variant-small-caps">se</span> (<em>t</em> test, ***P &lt; 0.0001).</p>
<p><strong>(C)</strong> and <strong>(D)</strong> Low magnification images of wild-type <strong>(C)</strong> and <em>hlb1</em>
<strong>(D)</strong> seedlings grown on <a class="usa-link" href="#def5">LatB</a>-free medium showing multiple root hairs. Note the shorter root hairs of <em>hlb1</em> compared with the wild type. Bar = 100 μm.</p>
<p><strong>(E)</strong> Average root hair length of wild-type and <em>hlb1</em> seedlings. Values are means (<em>n</em> = 86 to 120 root hairs) ± <span class="font-variant-small-caps">se</span> (<em>t</em> test, ***P &lt; 0.0001).</p>
<p><strong>(F)</strong>
<a class="usa-link" href="#def6">DIC</a> images of single root hairs from 5-d-old seedlings of the wild type and <em>hlb1</em> grown on <a class="usa-link" href="#def5">LatB</a>-free medium. Bar = 20 μm.</p></figcaption></figure><p>As noted, distinct qualitative and quantitative differences in F-actin organization in cells of the root elongation zone were observed between wild-type and <em>hlb1</em> seedlings but only when treated with 50 nM <a class="usa-link" href="#def5">LatB</a> (<a class="usa-link" href="#fig2">Figure 2</a>). We therefore evaluated F-actin dynamics in root hairs, where growth differences between the wild type and <em>hlb1</em> appeared to be more pronounced (<a class="usa-link" href="#fig3">Figures 3C</a> to <a class="usa-link" href="#fig3">3F</a>). In wild-type seedlings, the majority of the actively elongating root hairs were characterized by a distinct enrichment of fine and dynamic meshwork of F-actin at the extreme apex (<a class="usa-link" href="#fig4">Figure 4A</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 1</a>). In <em>hlb1</em> root hairs, the tip-focused fine meshwork of F-actin was not always observed, and in cases where it was present, it would often dissipate rapidly (<a class="usa-link" href="#fig4">Figure 4A</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 1</a>). Similar results were obtained with wild-type and <em>hlb1</em> root hairs expressing the F-actin marker <em>UBQ10pro:Lifeact-mGFP</em>, which was previously reported to clearly label apical F-actin meshwork at the apex of pollen tubes and moss protonema (<a aria-describedby="bib109" class="usa-link" href="#bib109">Vidali et al., 2009</a>). Using a 100× oil immersion objective on the spinning-disc confocal microscope, wild-type root hairs expressing <em>UBQ10pro:Lifeact-mGFP</em> had distinct filamentous meshwork at the tip that would often dissipate in the tips of <em>hlb1</em> root hairs (<a class="usa-link" href="#fig4">Figure 4B</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 2</a>). F-actin tip dynamics in the wild type and <em>hlb1</em> were quantified from 3-min time-lapse movies of growing roots hairs by obtaining the ratio of the total time where the apical meshwork of F-actin was clearly visible to the total time elapsed (<a class="usa-link" href="#fig4">Figure 4B</a>). This ratio provided an indication of the percentage of time that the root hair was able to maintain its tip-focused fine meshwork of F-actin as it elongated. These analyses showed that <em>hlb1</em> root hairs had a significantly lower ratio value than the wild type, indicating that the mutant largely failed to maintain its tip-focused fine meshwork of F-actin (<a class="usa-link" href="#fig4">Figure 4C</a>).</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4826010_PC_TPC201500794RAR2_f4.jpg" target="_blank"><img alt="Figure 4." class="graphic zoom-in" height="1132" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bed/4826010/173972e260d3/PC_TPC201500794RAR2_f4.jpg" width="789"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Apical F-Actin Organization Is Disrupted in <em>hlb1</em> Root Hairs.</p>
<p><strong>(A)</strong> Fine, dynamic mesh works of F-actin at the extreme tip or subapical region of wild-type root hairs (arrows) expressing <em>UBQ10pro:GFP-ABD2-GFP</em>. The fine apical mesh works of F-actin is often absent or occasionally dissipates in <em>hlb1</em> root hairs. See <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 1</a> for a corresponding time-lapse sequence. Bar = 20 μm.</p>
<p><strong>(B)</strong> F-actin labeled using <em>UBQ10pro:Lifeact-mGFP</em> in root hair tips of the wild-type and <em>hlb1</em> backgrounds. Filamentous structures are distinct in the apex of wild-type root hairs (arrow). F-actin dynamics in growing root hairs was quantified by obtaining the total time where the apical F-actin gradient was visible (arrow) divided by 180 s (total elapsed time of the video) to obtain the average ratio values shown in <strong>(C)</strong>.</p>
<p><strong>(C)</strong> Quantification of tip F-actin dynamics in growing wild-type and <em>hlb1</em> root hairs. Means (<em>n</em> = 28 to 42 root hairs) ± <span class="font-variant-small-caps">se</span> (<em>t</em> test, **P &lt; 0.001).</p></figcaption></figure><p>We also asked whether <em>hlb1</em> mutants had any measureable differences in global F-actin dynamics when compared with the wild type. Global F-actin dynamics was quantified from time-lapse movie sequences obtained from a spinning-disc confocal microscope using the metrics of pixel difference and correlation values developed earlier (<a aria-describedby="bib108" class="usa-link" href="#bib108">Vidali et al., 2010</a>). For this type of analysis, we focused our imaging on cortical F-actin in root epidermal cells in the mature zone and the center of growing root hairs because cells in these areas of the root stayed within the same field of view during the entire 1-min time-lapse sequence (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 3</a>). Time-lapse data sets from epidermal cells of the root transition or elongation zone were significantly displaced as the primary root grew in the medium and therefore were not suitable for the global F-actin analysis described here. Based on changes in pixel difference and correlation coefficient decay, there were no dramatic differences in global F-actin dynamics between the wild type and <em>hlb1</em> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 6</a>).</p></section><section id="s6"><h3 class="pmc_sec_title">Cloning of the Gene Conferring the <em>hlb1</em> Phenotype</h3>
<p>Because attempts to identify the mutation responsible for the <em>hlb1</em> phenotype by standard thermal asymmetric interlaced-PCR were unsuccessful, we conducted map-based cloning to identify the <em>HLB1</em> gene. We found that the <em>AT5G41950</em> gene had a 2-bp deletion in the first exon corresponding to amino acid 63 in the <em>hlb1</em> mutant (Arabidopsis Chromosome 5 Positions 16786010 and 16786011). This frameshift mutation resulted in a premature stop codon at amino acid 67, which may possibly lead to a truncated protein (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 7A</a>). To confirm whether loss of <em>AT5G41950</em> corresponds to <em>hlb1</em>, we searched the publicly available SALK collection for T-DNA insertions in the <em>AT5G41950</em> gene (<a aria-describedby="bib2" class="usa-link" href="#bib2">Alonso et al., 2003</a>). We genotyped the segregating progeny by PCR and identified homozygous plants for three SALK lines (SALK_046760, SALK_144501, and SALK_065836), which had insertions in the 8th exon, the 7th intron, and the 10th intron of <em>AT5G41950</em>, respectively (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 7B</a>). Seedlings of these three T-DNA insertion mutants exhibited similar hypersensitivity to <a class="usa-link" href="#def5">LatB</a> and root hair growth defects as <em>hlb1</em>, indicating that these were mutant alleles of <em>hlb1</em> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figures 7C and 7D</a>). The original <em>hlb1</em> mutant was designated as <em>hlb1-1</em>, whereas SALK_046760, SALK_144501, and SALK_065836 were named <em>hlb1-2</em>, <em>hlb1-3</em>, and <em>hlb1-4</em>, respectively (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 7B</a>). The average primary root length was significantly less than that of the wild type only in <em>hlb1-2</em>, which had an insertion in an exon, suggesting that <em>hlb1-3</em> and <em>hlb1-4</em> are weaker alleles (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 7D</a>). The <em>hlb1-1</em> mutant allele was used for the analyses detailed below. Further evidence showing that <em>hlb1</em> is linked to <em>AT5G41950</em> was obtained from functional complementation of <em>hlb1</em> by expression of an HLB1-GFP fusion protein that restored the wild-type phenotype (see below).</p></section><section id="s7"><h3 class="pmc_sec_title">HLB1 Is a Tetratricopeptide Repeat-Containing Protein Encoded by a Single Arabidopsis Gene</h3>
<p><em>HLB1</em> encodes a 565-amino acid protein of unknown function (<a class="usa-link" href="#fig5">Figure 5A</a>). We analyzed 141 HLB1 homologs from 79 plant species to understand their phylogenetic relationship. Phylogenetic relationships were inferred by the maximum likelihood method (<a aria-describedby="bib64" class="usa-link" href="#bib64">Le and Gascuel, 2008</a>) using MEGA6 software (<a aria-describedby="bib101" class="usa-link" href="#bib101">Tamura et al., 2013</a>). Based on this analysis, HLB1 appears to be well conserved across the land plant lineage (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 8</a>). Sequence alignment of HLB1 homologs from six plant species representing moss, monocots, and eudicots showed that HLB1 has a highly variable N terminus, six tetratricopeptide repeat (<a class="usa-link" href="#def7">TPR</a>) motifs, and a well conserved C terminus predicted to form a helical structure. The conservation of amino acids at the C terminus of several plant HLB1 proteins suggests that HLB1 has functional domains that have yet to be characterized (<a class="usa-link" href="#fig5">Figure 5B</a>).</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">Figure 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4826010_PC_TPC201500794RAR2_f5.jpg" target="_blank"><img alt="Figure 5." class="graphic zoom-in" height="787" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bed/4826010/6a427009659a/PC_TPC201500794RAR2_f5.jpg" width="740"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>HLB1 Domain Structure and Amino Acid Sequence Alignment with Homologs from Selected Land Plants.</p>
<p><strong>(A)</strong> Full-length HLB1 domain structure showing the position of six TPR motifs and C-terminal conserved domain (CTD).</p>
<p><strong>(B)</strong> Alignment of Arabidopsis HLB1 with HLB1 homologs from the five selected model species: <em>Medicago truncatula</em>, <em>Populus trichocarpa</em>, <em>Oryza sativa</em>, <em>Brachypodium distachyon</em>, and <em>Physcomitrella patens</em>. Dark-green shading depicts identical amino acid residues. Light green indicates other conserved residues.</p></figcaption></figure><p>Including HLB1, the Arabidopsis genome has 147 proteins containing a total of 839 <a class="usa-link" href="#def7">TPR</a> motifs that are predicted to function in a range of biological processes and localize to a variety of subcellular compartments (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Data Set 1</a>). However, HLB1 has no paralogs among any of the 146 <a class="usa-link" href="#def7">TPR</a> motif-containing Arabidopsis proteins, indicating that it exists as a single gene. Furthermore, HLB1 is specific to land plants, as no homologous sequences were identified in fungi and animals. <a class="usa-link" href="#def7">TPR</a> domains mediate protein-protein interactions, and proteins containing these domains are part of protein complexes with diverse functions including transcriptional repression, protein import, heat shock, and cell cycle regulation (<a aria-describedby="bib8" class="usa-link" href="#bib8">Blatch and Lässle, 1999</a>; <a aria-describedby="bib23" class="usa-link" href="#bib23">D’Andrea and Regan, 2003</a>). <a class="usa-link" href="#def7">TPR</a> domain-containing proteins in plants have been implicated in responses to various hormones, assembly of the photosynthetic machinery, plastid partitioning, and root development (<a aria-describedby="bib52" class="usa-link" href="#bib52">Jacobsen et al., 1996</a>; <a aria-describedby="bib92" class="usa-link" href="#bib92">Rosado et al., 2006</a>; <a aria-describedby="bib115" class="usa-link" href="#bib115">Yang et al., 2011</a>; <a aria-describedby="bib50" class="usa-link" href="#bib50">Hu et al., 2014</a>; <a aria-describedby="bib7" class="usa-link" href="#bib7">Bhuiyan et al., 2015</a>).</p></section><section id="s8"><h3 class="pmc_sec_title">HLB1 Colocalizes with <a class="usa-link" href="#def3">TGN</a> and <a class="usa-link" href="#def4">EE</a> Markers through Its Conserved C-Terminal Domain</h3>
<p>HLB1 fusions to GFP under the control of the native <em>HLB1</em> promoter (<em>HLB1pro:HLB1-GFP</em>) and the <em>CaMV 35S</em> promoter (<em>35Spro:GFP-HLB1</em>) were generated to gain more insight into HLB1 function. Both <em>HLB1pro:HLB1-GFP</em> and <em>35Spro:GFP-HLB1</em> complemented the <a class="usa-link" href="#def5">LatB</a>-hypersensitive and root hair phenotypes of <em>hlb1-1</em> in young seedlings, indicating that the HLB1 fluorescent protein fusions are functional (<a class="usa-link" href="#fig6">Figures 6A</a> and <a class="usa-link" href="#fig6">6B</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 9</a>).</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">Figure 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4826010_PC_TPC201500794RAR2_f6.jpg" target="_blank"><img alt="Figure 6." class="graphic zoom-in" height="1273" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bed/4826010/1657e54124b8/PC_TPC201500794RAR2_f6.jpg" width="789"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>A Functional HLB1-GFP Fusion Localizes to Endomembranes.</p>
<p><strong>(A)</strong> and <strong>(B)</strong>
<em>HLBpro:HLB1-GFP</em> complements the hypersensitivity to <a class="usa-link" href="#def5">LatB</a> and short root hair phenotypes of <em>hlb1-1</em>. Bars = 5 mm in <strong>(A)</strong> and 200 μm in <strong>(B)</strong>.</p>
<p><strong>(C)</strong> and <strong>(D)</strong> HLB1-GFP decorates distinct fluorescent bodies (arrowheads) in the root elongation zone and growing root hairs. See <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 4</a> for a corresponding time-lapse sequence.</p>
<p><strong>(E)</strong> and <strong>(F)</strong> HLB1-GFP in the root elongation zone and root hairs form intracellular agglomerates (arrows) upon treatment with 50 μM <a class="usa-link" href="#def8">BFA</a>. Bars = 20 μm.</p></figcaption></figure><p>Confocal microscopy of cells from the elongation zone and root hairs of <em>hlb1-1</em> seedlings expressing <em>HLB1pro:HLB1-GFP</em> revealed that the fusion protein localized to the cytoplasm and decorated distinct and dynamic foci (<a class="usa-link" href="#fig6">Figures 6C</a> and <a class="usa-link" href="#fig6">6D</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 4</a>). The fluorescent bodies decorated by HLB1-GFP were reminiscent of those observed in plant cells expressing fluorescently tagged compartments of the endomembrane system (<a aria-describedby="bib39" class="usa-link" href="#bib39">Geldner et al., 2009</a>). Endomembrane compartments such as endosomes visualized with GFP have been shown to form large intracellular fluorescent agglomerates when treated with the fungal metabolite Brefeldin A (<a class="usa-link" href="#def8">BFA</a>) (<a aria-describedby="bib91" class="usa-link" href="#bib91">Robinson et al., 2008</a>). Consistent with the hypothesis that the HLB1-GFP bodies were associated with the endomembrane system, seedlings expressing <em>HLB1pro:HLB1-GFP</em> treated with 50 μM <a class="usa-link" href="#def8">BFA</a> for 1 h developed intracellular fluorescent agglomerates (<a class="usa-link" href="#fig6">Figures 6E</a> and <a class="usa-link" href="#fig6">6F</a>).</p>
<p>To determine the identity of the endomembrane compartment with which HLB1 is associated, <em>HLB1pro:HLB1-GFP</em> lines were crossed with a set of marker lines expressing mCherry and CFP endomembrane markers (<a aria-describedby="bib78" class="usa-link" href="#bib78">Nelson et al., 2007</a>; <a aria-describedby="bib39" class="usa-link" href="#bib39">Geldner et al., 2009</a>). HLB1-GFP overlapped substantially with the <a class="usa-link" href="#def3">TGN</a> markers mCherry-VTI12 and SYP61-CFP and an early/recycling endosomal marker, mCherry-RabA1g (<a class="usa-link" href="#fig7">Figures 7A</a> to <a class="usa-link" href="#fig7">7D</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movies 5 and 6</a>), but not with the Golgi marker Man49-mCherry. Although the bodies decorated by HLB1-GFP and Man49-mCherry were clearly distinct, many HLB1-GFP bodies were adjacent to the larger Golgi bodies and often moved alongside Golgi stacks, a feature typical of <a class="usa-link" href="#def3">TGN</a> dynamics (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 7</a>) (<a aria-describedby="bib106" class="usa-link" href="#bib106">Uemura et al., 2014</a>). A number of HLB1-GFP bodies were also observed to move independently of Man49-mCherry (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 7</a>). HLB1-GFP did not overlap substantially with mCherry-ARA6, a marker for the late endosomes/prevacuolar compartment (<a class="usa-link" href="#def9">PVC</a>) (<a class="usa-link" href="#fig7">Figure 7E</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 8</a>). The extent of colocalization of HLB1-GFP with selected endomembrane markers was quantified by extracting Pearson’s correlation coefficients from scatterplots of green-red image pairs (<a aria-describedby="bib70" class="usa-link" href="#bib70">Manders et al., 1993</a>; <a aria-describedby="bib19" class="usa-link" href="#bib19">Costes et al., 2004</a>; <a aria-describedby="bib11" class="usa-link" href="#bib11">Bottanelli et al., 2012</a>) (<a class="usa-link" href="#fig7">Figure 7F</a>). The Pearson’s correlation coefficient was higher in the HLB1-GFP-mCherry-VTI12 pair when compared with the HLB1-GFP-mCherry-RabA1g pair. The higher value for the HLB1-GFP-mCherry-VTI12 pair indicates that HLB1 foci are associated more closely with the VTI12-resident <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> population and to a lesser extent with recycling endosomes decorated by RabA1g (<a class="usa-link" href="#fig7">Figure 7G</a>). Lower Pearson’s correlation coefficients were obtained from the HLB1-GFP-mCherry-ARA6 pair, consistent with visual observations of minimal overlap between HLB1 and late endosomes/<a class="usa-link" href="#def9">PVC</a> (<a class="usa-link" href="#fig7">Figures 7E</a> and <a class="usa-link" href="#fig7">7G</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 8</a>). Together, these data strongly support the conclusion that HLB1 is a <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a>-localized protein.</p>
<figure class="fig xbox font-sm" id="fig7"><h4 class="obj_head">Figure 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4826010_PC_TPC201500794RAR2_f7.jpg" target="_blank"><img alt="Figure 7." class="graphic zoom-in" height="864" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bed/4826010/46351a479a88/PC_TPC201500794RAR2_f7.jpg" width="797"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig7/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>HLB1-GFP Colocalizes with <em>Trans</em>-Golgi Network and Early/Recycling Endosome Markers.</p>
<p><strong>(A)</strong> and <strong>(B)</strong> Peripheral root cap <strong>(A)</strong> and epidermal cells in the root elongation zone <strong>(B)</strong> coexpressing HLB1-GFP and mCherry-VTI12, a <a class="usa-link" href="#def3">TGN</a> marker. Arrows indicate where HLB1-GFP and mCherry-VTI12 overlap. See <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 5</a> for a corresponding time-lapse sequence.</p>
<p><strong>(C)</strong> Epidermal cells in the root elongation zone coexpressing HLB1-GFP and SYP61-CFP, another <a class="usa-link" href="#def3">TGN</a> marker. Arrows indicate where HLB1-GFP and SYP61-CFP overlap.</p>
<p><strong>(D)</strong> Peripheral root cap cells coexpressing HLB1-GFP and mCherry-RabA1g, an early/recycling endosome marker. Arrows indicate where HLB1-GFP and mCherry- RabA1g overlap. See <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 6</a> for a corresponding time-lapse sequence.</p>
<p><strong>(E)</strong> Peripheral root cap cells coexpressing HLB1-GFP and mCherry-ARA6, a late endosome/<a class="usa-link" href="#def9">PVC</a> marker. Note that HLB1-GFP (arrows) and mCherry-ARA6 (arrowheads) do not overlap. See <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 8</a> for a corresponding time-lapse sequence. Bars = 5 μm in <strong>(A)</strong> to <strong>(E)</strong>.</p>
<p><strong>(F)</strong> A representative scatterplot for quantification of the extent of colocalization between green- and red-emitting organelles. The scatterplot for colocalization was obtained from an individual cell as indicated by the white rectangle in <strong>(A)</strong>.</p>
<p><strong>(G)</strong> Pearson’s correlation coefficients were calculated from scatterplots using Volocity quantitation software. Statistical significance was determined by one-way ANOVA. Means (<em>n</em> &gt; 170 cells) ± <span class="font-variant-small-caps">se</span>. Different letters are significantly different (P &lt; 0.05, Tukey’s test).</p></figcaption></figure><p>As noted above, HLB1 contains six <a class="usa-link" href="#def7">TPR</a> motifs and a conserved C-terminal domain of unknown function. To determine whether these domains target HLB1 to the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a>, we generated GFP fusions to truncated HLB1 where either the six <a class="usa-link" href="#def7">TPR</a> motifs (GFP-HLB1<sup>∆TPR</sup>) or the conserved C terminus (GFP-HLB1<sup>∆CTD</sup>) was deleted (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 10A</a>). When expressed in seedlings, the GFP-HLB1<sup>∆CTD</sup> fusion that had the intact <a class="usa-link" href="#def7">TPR</a> motifs localized exclusively to the cytoplasm. By contrast, the GFP-HLB1<sup>∆TPR</sup> with the conserved C terminus localized to both the cytoplasm and <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> bodies, similar to the localization of full-length HLB1 (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 10B</a>). Taken together, the data indicate that the C terminus of HLB1 is responsible for its localization to the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a>.</p></section><section id="s9"><h3 class="pmc_sec_title">HLB1-GFP Bodies Track along F-Actin in Planta</h3>
<p>Given that the <em>hlb1</em> mutant is hypersensitive to <a class="usa-link" href="#def5">LatB</a>, we tested the association of HLB1-GFP TGNs with F-actin in planta. HLB1-GFP plants were crossed with plants expressing the red fluorescent F-actin construct <em>UBQ10pro:mCherry-ABD2-mCherry</em> (<a aria-describedby="bib28" class="usa-link" href="#bib28">Dyachok et al., 2014</a>). It was difficult to obtain clear images of mCherry-ABD2-mCherry and HLB1-GFP bodies together in root cells that were highly cytoplasmic because the HLB1-GFP cytoplasmic component would often obscure the red F-actin label. Thus, imaging of mCherry-ABD2-mCherry and HLB1-GFP was conducted on vacuolated epidermal cells of the root maturation zone and fully mature root hairs, where signal from the cytoplasm was minimal. In these cell types, HLB1-GFP bodies aligned with and tracked along F-actin bundles in a “beads-on-a-string”-like localization, typical of actin membrane adaptors such as the NET proteins, which are associated with the actin cytoskeleton in vivo (<a class="usa-link" href="#fig8">Figures 8A</a> and <a class="usa-link" href="#fig8">8B</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movies 9 and 10</a>) (<a aria-describedby="bib112" class="usa-link" href="#bib112">Wang et al., 2014</a>).</p>
<figure class="fig xbox font-sm" id="fig8"><h4 class="obj_head">Figure 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4826010_PC_TPC201500794RAR2_f8.jpg" target="_blank"><img alt="Figure 8." class="graphic zoom-in" height="1470" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bed/4826010/ba768a8083dc/PC_TPC201500794RAR2_f8.jpg" width="797"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig8/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>HLB1-GFP Tracks along F-Actin Cables in Living Cells.</p>
<p>Root epidermal cell from the maturation zone <strong>(A)</strong> and the base of a mature root hair <strong>(B)</strong> coexpressing HLB1-GFP and mCherry-ABD2-mCherry (mCh-ABD2-mCh), an F-actin marker. Arrows indicate individual or clusters of HLB1-GFP bodies aligned along the red-emitting F-actin cables. See <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movies 9 and 10</a> for corresponding time-lapse sequences. Bars = 10 μm.</p></figcaption></figure><p>The increased sensitivity of <em>hlb1</em> to <a class="usa-link" href="#def5">LatB</a> prompted us to ask whether the HLB1-GFP bodies themselves responded to <a class="usa-link" href="#def5">LatB</a> treatment. We found that roots of seedlings formed intracellular HLB1-GFP agglomerates within 10 min of <a class="usa-link" href="#def5">LatB</a> application. However, other <a class="usa-link" href="#def3">TGN</a> markers, including YFP-VTI12 and SYP61-CFP, formed similar aggregates as HLB1-GFP after <a class="usa-link" href="#def5">LatB</a> treatment, indicating that the compound has a global effect on <a class="usa-link" href="#def3">TGN</a> distribution in root cells (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 11</a>).</p></section><section id="s10"><h3 class="pmc_sec_title">The Rice Ortholog of Arabidopsis <em>HLB1</em> Partially Complements <em>hlb1</em>
</h3>
<p>Bioinformatic analyses indicated that HLB1 is a highly conserved protein that did not diverge much during the split between eudicots and monocots (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 8</a>). To test whether an HLB1 ortholog from a monocot could have similar functions as the Arabidopsis HLB1, the rice (<em>Oryza sativa</em>) <em>HLB1</em> gene (Os-<em>HLB1</em>) was cloned and a <em>35Spro:OsHLB1-GFP</em> construct was generated. The OsHLB1-GFP fusion partially complemented the <a class="usa-link" href="#def5">LatB</a>-hypersensitive phenotype of Arabidopsis <em>hlb1</em> but fully complemented its short primary root defects. Furthermore, OsHLB1-GFP localized to the cytoplasm and labeled similar dynamic punctate bodies as the Arabidopsis HLB1-GFP in the <em>hlb1-1</em> mutant background (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figures 12A and 12B</a>). The partial complementation of <em>hlb1</em> by OsHLB1-GFP and similar localization patterns as the Arabidopsis HLB1 supports the conclusion that the role of HLB1 in endomembrane traffic is likely conserved in land plants.</p></section><section id="s11"><h3 class="pmc_sec_title">Coimmunoprecipitation Identifies HOPM INTERACTOR7/BREFELDIN A-VISUALIZED ENDOCYTIC TRAFFICKING DEFECTIVE1 as a Potential HLB1 Interactor</h3>
<p>Because <a class="usa-link" href="#def7">TPR</a> domains are known to mediate protein-protein interactions (<a aria-describedby="bib23" class="usa-link" href="#bib23">D’Andrea and Regan, 2003</a>), coimmunoprecipitation (<a class="usa-link" href="#def10">co-IP</a>) experiments were performed to identify proteins interacting with HLB1 using roots of <em>hlb1-1</em> seedlings complemented with the <em>35Spro:GFP-HLB1</em> construct (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 9</a>). Seedlings expressing <em>35Spro:GFP</em> served as control plants. GFP-HLB1 and its putative interactors were coimmunoprecipitated using anti-GFP serum. Coimmunoprecipitated proteins from <em>35Spro:GFP-HLB1</em> and <em>35Spro:GFP</em> seedlings were analyzed with SDS-PAGE, and the components were identified by mass spectrometry-based proteomic analysis. Peptides corresponding to both HLB1 and GFP were readily detected in the coimmunoprecipitates from the roots of the <em>35Spro:GFP-HLB1</em> line, indicating successful isolation of chimeric GFP-HLB1 from the root extracts (<a class="usa-link" href="#fig9">Figure 9A</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Data Set 2</a>).</p>
<figure class="fig xbox font-sm" id="fig9"><h4 class="obj_head">Figure 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4826010_PC_TPC201500794RAR2_f9.jpg" target="_blank"><img alt="Figure 9." class="graphic zoom-in" height="635" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bed/4826010/671b3f7e6966/PC_TPC201500794RAR2_f9.jpg" width="747"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig9/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>MIN7/BEN1 Interacts with HLB1 and <em>ben1</em> Phenocopies the Hypersensitivity to <a class="usa-link" href="#def5">LatB</a> and Root Hair Phenotypes of <em>hlb1</em>.</p>
<p><strong>(A)</strong> Heat map of two independent <a class="usa-link" href="#def10">co-IP</a> experiments from roots of the GFP-only control and GFP-HLB1 seedlings indicating relative abundance of peptides. GFP peptides were detected in all the samples. HLB1 and MIN7/BEN1 peptides were detected in the respective GFP-HLB1 and MIN7/BEN1-GFP samples, but not in the GFP-only controls (a value of 0 indicates absence of peptide). Note that HLB1 and MIN7/BEN1 were pulled down, reciprocally.</p>
<p><strong>(B)</strong> Hypersensitivity of <em>ben1</em> and <em>hlb1</em> to <a class="usa-link" href="#def5">LatB</a>. Three-day-old seedlings were transplanted onto 50 nM <a class="usa-link" href="#def5">LatB</a> and the image was taken 6 d later. Bar <strong>=</strong> 5 mm.</p>
<p><strong>(C)</strong> Representative bright-field images of wild-type, <em>hlb1</em>, and <em>ben1</em> root hairs from seedlings without <a class="usa-link" href="#def5">LatB</a> treatment. Bar = 20 μm.</p>
<p><strong>(D)</strong> MIN7/BEN1-GFP and HLB1-mCherry colocalize (arrows) in common <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> compartments in root peripheral cap cells. Bar = 10 μm.</p>
<p><strong>(E)</strong> Pearson’s correlation coefficients were similar between MIN7/BEN1-GFP and HLB1-mCherry and HLB1-GFP and mCherry-VTI12 (<em>t</em> test; n.s., not significant).</p></figcaption></figure><p>Two independent <a class="usa-link" href="#def10">co-IP</a> experiments were conducted, and proteins present in <em>35Spro:GFP-HLB1</em> immunoprecipitates but not in <em>35Spro:GFP</em> controls were considered for further analyses. Surprisingly, the first <a class="usa-link" href="#def10">co-IP</a> experiment yielded a limited number of proteins with only 22 proteins unique to <em>35Spro:GFP-HLB1</em>. By contrast, the second <a class="usa-link" href="#def10">co-IP</a> experiment yielded ∼650 proteins unique to <em>35Spro:GFP-HLB1</em>. It is unclear why there was a large discrepancy in the protein list between two independent <a class="usa-link" href="#def10">co-IP</a> experiments. In both <a class="usa-link" href="#def10">co-IP</a> experiments, however, peptides of HOPM INTERACTOR7/BREFELDIN A-VISUALIZED ENDOCYTIC TRAFFICKING DEFECTIVE1 (MIN7/BEN1) were detected as unique to <em>35Spro:GFP-HLB1</em> root extracts (<a aria-describedby="bib79" class="usa-link" href="#bib79">Nomura et al., 2006</a>; <a aria-describedby="bib102" class="usa-link" href="#bib102">Tanaka et al., 2009</a>). To validate the results of the two <a class="usa-link" href="#def10">co-IP</a> experiments, a reciprocal <a class="usa-link" href="#def10">co-IP</a> was conducted with root extracts of <em>35Spro:MIN7/BEN1-GFP</em> (<a aria-describedby="bib80" class="usa-link" href="#bib80">Nomura et al., 2011</a>). Consistent with the possibility that MIN7/BEN1 interacts with HLB1, HLB1 peptides were identified in the immunoprecipitates of <em>35Spro:MIN7/BEN1-GFP</em> but not in the <em>35Spro:GFP</em> controls (<a class="usa-link" href="#fig9">Figure 9A</a>).</p>
<p>MIN7/BEN1 has been previously shown to localize to the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> and function in endocytosis of <a class="usa-link" href="#def2">PM</a> proteins (<a aria-describedby="bib102" class="usa-link" href="#bib102">Tanaka et al., 2009</a>; <a aria-describedby="bib80" class="usa-link" href="#bib80">Nomura et al., 2011</a>; <a aria-describedby="bib103" class="usa-link" href="#bib103">Tanaka et al., 2013</a>). Therefore, its connection to HLB1 was evaluated in more detail. Genetic and pharmacological evidence that MIN7/BEN1 is biologically relevant to HLB1 function was obtained from analyses of <em>min7/ben1</em> mutants (SALK_013761), which we refer to here as <em>ben1</em> for simplicity. Roots of the <em>ben1</em> mutant were hypersensitive to <a class="usa-link" href="#def5">LatB</a>, as much as <em>hlb1</em> (<a class="usa-link" href="#fig9">Figure 9B</a>). Even in the absence of <a class="usa-link" href="#def5">LatB</a>, <em>ben1</em> had short primary roots and root hairs, mirroring <em>hlb1</em> phenotypes (<a class="usa-link" href="#fig9">Figure 9C</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 13</a>). To determine if HLB1 and MIN7/BEN1 colocalized in planta, we generated an HLB1-mCherry fusion and expressed it in plants harboring MIN7/BEN1-GFP (<a aria-describedby="bib80" class="usa-link" href="#bib80">Nomura et al., 2011</a>). Spinning-disc confocal microscopy observations followed by quantitative colocalization analyses revealed that HLB1-mCherry bodies overlapped significantly with MIN7/BEN1-GFP foci, providing support to our <a class="usa-link" href="#def10">co-IP</a> results that the two proteins indeed interact in vivo (<a class="usa-link" href="#fig9">Figures 9D</a> and <a class="usa-link" href="#fig9">9E</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 11</a>).</p></section><section id="s12"><h3 class="pmc_sec_title">The <em>ben1</em> and <em>hlb1</em> Mutations Affect Distinct Membrane Trafficking Pathways</h3>
<p>A mutant allele of <em>ben1</em> was isolated from a confocal microscopy-based forward-genetic screen for defects in the internalization of the PIN-FORMED (PIN) <a class="usa-link" href="#def2">PM</a>-localized auxin efflux carriers. <em>ben1</em> was found to be defective in endocytic trafficking as it had diminished ability to form PIN-GFP agglomerates when treated with <a class="usa-link" href="#def8">BFA</a> (<a aria-describedby="bib102" class="usa-link" href="#bib102">Tanaka et al., 2009</a>, <a aria-describedby="bib103" class="usa-link" href="#bib103">2013</a>). Because <em>ben1</em> mirrored the <a class="usa-link" href="#def5">LatB</a> hypersensitivity and root hair phenotypes of <em>hlb1</em> (<a class="usa-link" href="#fig9">Figure 9</a>), we hypothesized that <em>hlb1</em> could also have defects in the formation of intracellular PIN agglomerates when exposed to <a class="usa-link" href="#def8">BFA</a>. PIN2-GFP was therefore expressed in <em>hlb1</em> and treated with <a class="usa-link" href="#def8">BFA</a>. Like wild-type roots, <em>hlb1</em> roots accumulated intracellular PIN2-GFP agglomerates after 1 h exposure to 50 μM <a class="usa-link" href="#def8">BFA</a> (<a class="usa-link" href="#fig10">Figure 10A</a>). Although the number of PIN2-GFP agglomerates per cell in <em>hlb1</em> and the wild type was not statistically different, fluorescence of <a class="usa-link" href="#def8">BFA</a>-induced agglomerates was more intense in <em>hlb1</em> compared with the wild type when imaged under similar confocal microscopy settings (<a class="usa-link" href="#fig10">Figures 10B</a> and <a class="usa-link" href="#fig10">10C</a>). The stronger fluorescence intensity of PIN2-GFP agglomerates in <em>hlb1</em> prompted us to ask whether the <em>hlb1</em> mutation might be affecting the recycling of PIN2-GFP to the <a class="usa-link" href="#def2">PM</a>. To address this question, <a class="usa-link" href="#def8">BFA</a> washout experiments were conducted on wild-type and <em>hlb1</em> seedlings expressing PIN2-GFP. After 1 to 2 h of <a class="usa-link" href="#def8">BFA</a> washout, the number of PIN2-GFP agglomerates in wild-type roots decreased significantly, whereas those in the <em>hlb1</em> mutant background persisted (<a class="usa-link" href="#fig10">Figures 10A</a> and <a class="usa-link" href="#fig10">10C</a>).</p>
<figure class="fig xbox font-sm" id="fig10"><h4 class="obj_head">Figure 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4826010_PC_TPC201500794RAR2_f10.jpg" target="_blank"><img alt="Figure 10." class="graphic zoom-in" height="1244" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bed/4826010/3a50cd370ddf/PC_TPC201500794RAR2_f10.jpg" width="723"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig10/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><em>hlb1</em> Is Defective in Recycling of PIN2-GFP to the Plasma Membrane.</p>
<p><strong>(A)</strong> One-hour treatment with <a class="usa-link" href="#def8">BFA</a> results in the formation of intracellular PIN2-GFP agglomerates in both the wild type and <em>hlb1</em>. Two-hour washout of <a class="usa-link" href="#def8">BFA</a> led to a significant reduction in the number of PIN2-GFP agglomerates in the wild type but not in <em>hlb1</em>. Bar = 20 μm.</p>
<p><strong>(B)</strong> Relative fluorescence intensity of PIN2-GFP agglomerates in root cells of the wild type and <em>hlb1</em> treated with <a class="usa-link" href="#def8">BFA</a> for 1 h. Values are means (<em>n</em> = 200 to 276 PIN2-GFP bodies) ± <span class="font-variant-small-caps">se</span> from &gt;120 cells from at least 15 seedlings (<em>t</em> test, **P &lt; 0.001).</p>
<p><strong>(C)</strong> Quantification of PIN2-GFP agglomerates after 1 h <a class="usa-link" href="#def8">BFA</a> treatment and after 1 h of <a class="usa-link" href="#def8">BFA</a> washout. Values are means (<em>n</em> = 70 to 171 cells) ± <span class="font-variant-small-caps">se</span> from at least 20 seedlings. Statistical significance was determined by one-way ANOVA. Different letters indicate significant differences among means (P &lt; 0.05; Tukey’s test).</p>
<p><strong>(D)</strong>
<em>hlb1</em> forms more PIN2-GFP agglomerates in response to low (10 μM) <a class="usa-link" href="#def8">BFA</a> than the wild type after 1 and 3 h. Bar = 20 μm.</p>
<p><strong>(E)</strong> Quantification of PIN2-GFP agglomerates at 1 and 3 h treatment with 10 μM <a class="usa-link" href="#def8">BFA</a>. Values are means (<em>n</em> = 125 to 170 cells) ± <span class="font-variant-small-caps">se</span> from at least 15 seedlings. Statistical significance was determined by one-way ANOVA. Different letters indicate significant differences among means (P &lt; 0.05; Tukey’s test).</p>
<p><strong>(F)</strong> One-hour treatment with 50 nM <a class="usa-link" href="#def5">LatB</a> results in more significant accumulation of PIN2-GFP agglomerates in <em>hlb1</em> compared with the wild type. Bar = 20 μm.</p>
<p><strong>(G)</strong> Relative fluorescence intensity of PIN2-GFP agglomerates in root cells of the wild type and <em>hlb1</em> treated with <a class="usa-link" href="#def5">LatB</a> for 1 h. Values are means (<em>n</em> = 104 to 898 PIN2-GFP bodies) ± <span class="font-variant-small-caps">se</span> from &gt;300 root cortical cells and at least 15 independent seedlings (<em>t</em> test, **P &lt; 0.001).</p>
<p><strong>(H)</strong> Quantification of the number of PIN2-GFP agglomerates in root cells of the wild type and <em>hlb1</em> after treatment with <a class="usa-link" href="#def5">LatB</a> for 1 h. Values are means (<em>n</em> = 341 root cortical cells) ± <span class="font-variant-small-caps">se</span> from at least 15 independent seedlings (<em>t</em> test, ***P &lt; 0.0001).</p></figcaption></figure><p>The PIN2 trafficking phenotypes observed in <em>hlb1</em> were surprisingly opposite to those reported for <em>ben1</em> but strikingly similar to those described for a class of dominant membrane trafficking mutants called <em>bex</em> (for <em>bfa-visualized exocytic trafficking defective</em>) (<a aria-describedby="bib35" class="usa-link" href="#bib35">Feraru et al., 2012</a>; <a aria-describedby="bib104" class="usa-link" href="#bib104">Tanaka et al., 2014</a>). In addition to the persistence of PIN agglomerates after <a class="usa-link" href="#def8">BFA</a> washout, <em>bex1</em> and <em>bex5</em> mutants formed intracellular PIN2-GFP agglomerates in response to low (10 μM) concentrations of <a class="usa-link" href="#def8">BFA</a>, whereas the wild type showed only minor agglomerate formation (<a aria-describedby="bib35" class="usa-link" href="#bib35">Feraru et al., 2012</a>; <a aria-describedby="bib104" class="usa-link" href="#bib104">Tanaka et al., 2014</a>). <em>hlb1</em> seedlings expressing PIN2-GFP were therefore treated with 10 μM <a class="usa-link" href="#def8">BFA</a> for 1 and 3 h to determine if <em>hlb1</em> trafficking defects mirrored those reported for <em>bex1</em> and <em>bex5</em>. Consistent with previous studies, 10 μM <a class="usa-link" href="#def8">BFA</a> induced only minor intracellular accumulation of PIN2-GFP in wild-type seedlings (<a class="usa-link" href="#fig10">Figure 10D</a>). By contrast, <em>hlb1</em> roots exposed to 10 μM <a class="usa-link" href="#def8">BFA</a> for the same time periods as the wild type showed significant formation of PIN2-GFP agglomerates that were reminiscent of the <em>bex</em> mutants (<a class="usa-link" href="#fig10">Figures 10D</a> and <a class="usa-link" href="#fig10">10E</a>) (<a aria-describedby="bib35" class="usa-link" href="#bib35">Feraru et al., 2012</a>; <a aria-describedby="bib104" class="usa-link" href="#bib104">Tanaka et al., 2014</a>). Taken together, the data indicate that HLB1 functions in protein recycling to the <a class="usa-link" href="#def2">PM</a>.</p></section><section id="s13"><h3 class="pmc_sec_title">Recycling of PIN2-GFP to the Plasma Membrane in <em>hlb1</em> Is Hypersensitive to <a class="usa-link" href="#def5">LatB</a>
</h3>
<p>Because <em>hlb1</em> is hypersensitive to <a class="usa-link" href="#def5">LatB</a> and the actin cytoskeleton has been implicated in the intracellular trafficking of PIN proteins (<a aria-describedby="bib40" class="usa-link" href="#bib40">Geldner et al., 2001</a>; <a aria-describedby="bib74" class="usa-link" href="#bib74">Nagawa et al., 2012</a>), we hypothesized that PIN2-GFP recycling in the <em>hlb1</em> background could also exhibit heightened sensitivity to concentrations of <a class="usa-link" href="#def5">LatB</a> that only minimally affected PIN2-GFP recycling in the wild type. To test this hypothesis, the wild type and <em>hlb1</em> expressing PIN2-GFP were treated with 50 nM <a class="usa-link" href="#def5">LatB</a>. At this <a class="usa-link" href="#def5">LatB</a> concentration, PIN2-GFP in roots of the wild type formed a few, weakly fluorescing intracellular PIN2-GFP agglomerates within 1 h of exposure to the drug. On the other hand, roots of <em>hlb1</em> seedlings formed bright and numerous PIN2-GFP intracellular agglomerates that were reminiscent of <a class="usa-link" href="#def8">BFA</a>-induced PIN2-GFP bodies (<a class="usa-link" href="#fig10">Figures 10F</a> to <a class="usa-link" href="#fig10">10H</a>). These results indicate that the recycling of PIN2-GFP to the <a class="usa-link" href="#def2">PM</a> in <em>hlb1</em> is more sensitive to <a class="usa-link" href="#def5">LatB</a> compared with the wild type and consistent with the heightened response of its roots and F-actin structures to the chemical.</p></section><section id="s14"><h3 class="pmc_sec_title">
<em>hlb1</em> Exhibits Enhanced Accumulation of PIN2-GFP at the <a class="usa-link" href="#def2">PM</a> and Endosomes but Has No Measurable Defects in <a class="usa-link" href="#def3">TGN</a> and MIN1/BEN1 Dynamics</h3>
<p>We next asked if the localization of PIN2-GFP was altered in <em>hlb1</em> root cells without <a class="usa-link" href="#def8">BFA</a> or <a class="usa-link" href="#def5">LatB</a> treatment. Live-cell microscopy and corresponding quantitative analyses showed that PIN2-GFP signal in the <em>hlb1</em> background was higher in the <a class="usa-link" href="#def2">PM</a> and in the cell interior compared with the wild type when imaged with similar confocal microscopy settings. The stronger fluorescence in the cell interior of <em>hlb1</em> root cells was due to brighter and more distinct PIN2-GFP foci (<a class="usa-link" href="#fig11">Figures 11A</a> and <a class="usa-link" href="#fig11">11B</a>).</p>
<figure class="fig xbox font-sm" id="fig11"><h4 class="obj_head">Figure 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4826010_PC_TPC201500794RAR2_f11.jpg" target="_blank"><img alt="Figure 11." class="graphic zoom-in" height="704" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bed/4826010/2ecfc477e536/PC_TPC201500794RAR2_f11.jpg" width="747"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig11/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>PIN2-GFP Distribution, Constitutive Secretion, and Traffic to the Vacuole Are Partially Disrupted in <em>hlb1</em>.</p>
<p><strong>(A)</strong> Representative images of non-<a class="usa-link" href="#def5">LatB</a>-treated root cells expressing PIN2-GFP. Bar = 10 μm.</p>
<p><strong>(B)</strong> Quantitative analysis of PIN2-GFP fluorescence in the <a class="usa-link" href="#def2">PM</a> (arrows) and cell interior (white rectangle). Mean fluorescence intensity was obtained from at least 20 roots ± <span class="font-variant-small-caps">se</span> (<em>t</em> test, ***P &lt; 0.0001).</p>
<p><strong>(C)</strong> Representative images of non-<a class="usa-link" href="#def5">LatB</a>-treated root hairs expressing SEC-RFP. Note the intense SEC-RFP signal at the extreme root hair tip of wild type (arrow). See <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 13</a> for a corresponding time-lapse sequence. Bar = 10 μm.</p>
<p><strong>(D)</strong> Root hair tip secretion was quantified by dividing average fluorescence intensity at the apex by average fluorescence intensity at the subapex as marked by the white elliptical area to obtain ratio values shown in <strong>(E)</strong>.</p>
<p><strong>(E)</strong> Apex to subapex SEC-RFP ratio taken every 10 s for 1 min. A higher ratio indicates a stronger tip-focused SEC-RFP gradient. Values are means (<em>n</em> = 28 to 41 root hairs) ± <span class="font-variant-small-caps">se</span> (<em>t</em> test, ***P &lt; 0.0001).</p>
<p><strong>(F)</strong> SEC-RFP in epidermal cells from cotyledons of 5-d-old seedlings. SEC-RFP is detected in the apoplast of the wild type and <em>hlb1</em> but appears to be retained strongly in the <a class="usa-link" href="#def1">ER</a> (arrows) in <em>hlb1</em>. Bars = 5 μm.</p>
<p><strong>(G)</strong> RFP-AFVY accumulates in the vacuole (<em>V</em>) lumen of epidermal cells of wild-type and <em>hlb1</em> hypocotyls. In <em>hlb1</em>, RFP-AFVY is mis-sorted to the apoplast (<em>Ap</em>). Bar = 20 μm.</p>
<p><strong>(H)</strong> Mis-sorting of RFP-AFVY was quantified by dividing the average fluorescence intensity of the apoplast by the average fluorescence intensity of the vacuole as indicated by the white rectangles in <strong>(G)</strong>. A higher ratio indicates strong RFP-AFVY accumulation in the apoplast. Values are means (<em>n</em> = 38 to 43 cells from 15 independent seedlings) ± <span class="font-variant-small-caps">se</span> (<em>t</em> test, ***P &lt; 0.0001).</p></figcaption></figure><p>The more intense PIN2-GFP signals in the <a class="usa-link" href="#def2">PM</a> and cell interior of <em>hlb1</em> roots was reminiscent of observations made in <em>bex5</em>, which showed similar PIN2-GFP recycling defects as <em>hlb1</em> (<a aria-describedby="bib35" class="usa-link" href="#bib35">Feraru et al., 2012</a>). Given the defects in PIN2-GFP recycling in <em>hlb1</em>, we asked whether the distribution of a <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> marker was altered in the mutant. To address this question, we expressed YFP-VTI12 in the <em>hlb1</em> background. The distribution of YFP-VTI12 foci in peripheral root cap cells was similar in the wild type and <em>hlb1</em> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 14A</a>). Furthermore, global analyses of YFP-VTI12 dynamics in root hairs revealed no appreciable differences between the wild type and <em>hlb1</em> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figures 14B and 14C</a> and <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 12</a>). Because we established that MIN7/BEN1 is an HLB1 interactor at the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a>, we also looked at the distribution of MIN7/BEN1-GFP in the <em>hlb1</em> background. Unfortunately, lines expressing MIN7/BEN1-GFP in <em>hlb1</em> and the wild type were extensively silenced and we could observe GFP signal only from cells in the peripheral root cap. Like YFP-VTI12, the distribution of MIN7/BEN1-GFP in these cells was similar between the wild type and <em>hlb1</em> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 14D</a>). Although we did not detect any substantial differences between the wild type and <em>hlb1</em> with regard to VTI12 or MIN7/BEN1 distribution at the light microscope level, we cannot discount the possibility that <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> ultrastructure might be affected in the <em>hlb1</em> mutant as has been reported in <em>bex5</em> (<a aria-describedby="bib35" class="usa-link" href="#bib35">Feraru et al., 2012</a>).</p>
<p>Because conspicuous differences between the wild type and <em>hlb1</em> were noted with regard to PIN2-GFP intensity in the <a class="usa-link" href="#def2">PM</a> and cell interior (<a class="usa-link" href="#fig11">Figures 11A</a> and <a class="usa-link" href="#fig11">11B</a>), we next asked if <em>hlb1</em> displayed any <em>pin2</em>-like phenotypes in regard to seedling root development. Consistent with previous reports, <em>pin2</em> seedlings had distinct root agravitropic defects as evident from their high deviation from vertical growth (<a aria-describedby="bib73" class="usa-link" href="#bib73">Müller et al., 1998</a>). By contrast, the deviation from vertical growth of <em>hlb1</em> roots was similar to the wild type (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figures 15A and 15B</a>). Moreover, although the deviation from vertical growth of <em>pin2</em> roots appeared to be enhanced by <a class="usa-link" href="#def5">LatB</a>, their primary roots exhibited wild-type growth sensitivity to <a class="usa-link" href="#def5">LatB</a> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figures 15C and 15D</a>). Root deviation from vertical growth of <em>ben1</em> and <em>hlb1 ben1</em> was similar to <em>hlb1</em> and the wild type (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figures 15A and 15B</a>). These data indicate that the HLB1-MIN7/BEN1-resident <a class="usa-link" href="#def3">TGN</a> complex is not required for full functionality of PIN2.</p>
<p>Because PIN2 recycling is defective in <em>hlb1</em> and post-Golgi traffic of some PIN proteins is a crucial component of the root gravitropic response (<a aria-describedby="bib62" class="usa-link" href="#bib62">Kleine-Vehn et al., 2008</a>; <a aria-describedby="bib61" class="usa-link" href="#bib61">Kleine-Vehn et al., 2010</a>), we next asked whether <em>hlb1</em> roots were compromised in gravitropism. Seedlings of the wild type and <em>hlb1</em> with roots growing vertically were rotated by 90 degrees, and the kinetics of root curvature was monitored at various time points for a 12-h period. No measureable differences in root gravitropic response between wild-type and <em>hlb1</em> seedlings under our growth conditions were noted (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figures 15E and 15F</a>).</p></section><section id="s15"><h3 class="pmc_sec_title">
<em>hlb1</em> Has Partial Defects in Bulk Flow Secretion and Trafficking to the Vacuole</h3>
<p>The impaired trafficking of PIN2-GFP to the <a class="usa-link" href="#def2">PM</a> in <em>hlb1</em> could be due to defects in protein secretion or recycling. We therefore expressed the secreted (SEC)-RFP fusion, a bulk-flow secretion reporter, in <em>hlb1</em> (<a aria-describedby="bib34" class="usa-link" href="#bib34">Faso et al., 2009</a>; <a aria-describedby="bib88" class="usa-link" href="#bib88">Renna et al., 2013</a>) to determine whether the mutant has defects in constitutive secretion mechanisms. We first examined SEC-RFP in actively elongating root hairs because the tips of these cell types are characterized by active secretion (<a aria-describedby="bib86" class="usa-link" href="#bib86">Preuss et al., 2004</a>), and the most obvious growth and F-actin organizational defects of <em>hlb1</em> seedlings were observed in root hairs (<a class="usa-link" href="#fig3">Figures 3</a> and <a class="usa-link" href="#fig4">4</a>). Using spinning-disc confocal microscopy, we found that growing root hairs of the wild type had intense SEC-RFP fluorescence at the extreme tip. By contrast, most <em>hlb1</em> root hairs had weak SEC-RFP fluorescence at the tip or failed to maintain a distinct tip-focused SEC-RFP gradient (<a class="usa-link" href="#fig11">Figure 11C</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Movie 13</a>). The tip-focused SEC-RFP gradient was quantified by obtaining the ratio of the average SEC-RFP fluorescence at the tip apex to the average SEC-RFP fluorescence at the subapex from 1-min time-lapse movie sequences (<a class="usa-link" href="#fig11">Figure 11D</a>). We found that ratio values at every 10-s time point were higher in the wild type compared with <em>hlb1</em>, verifying our visual observations that secretion in root hair tips of <em>hlb1</em> is altered (<a class="usa-link" href="#fig11">Figure 11E</a>). Furthermore, retention of SEC-RFP in the <a class="usa-link" href="#def1">ER</a> appeared to be more pronounced in <em>hlb1</em> cotyledons compared with the wild type (<a class="usa-link" href="#fig11">Figure 11F</a>). However, like in the wild type, strong SEC-RFP fluorescence was still observed in the apoplast of various cell types of <em>hlb1</em> seedlings (<a class="usa-link" href="#fig11">Figure 11F</a>; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figures 16A to 16D</a>). Taken together, our results indicate that bulk flow secretion in <em>hlb1</em> is only partially affected, a defect that is most strongly manifested in growing root hairs, which have measurable F-actin defects.</p>
<p>The localization of HLB1 to the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> suggests that other post-Golgi trafficking pathways such as trafficking to the vacuole might be affected in the <em>hlb1</em> mutant. To address this question, we expressed RFP-AFVY, a fusion to the C-terminal of phaseolin that accumulates in the vacuole lumen (<a aria-describedby="bib51" class="usa-link" href="#bib51">Hunter et al., 2007</a>). Because RFP-AFVY was strongly silenced in root cells, we focused our analyses on hypocotyls and cotyledons. We found that RFP-AFVY in hypocotyl and cotyledon epidermal cells of <em>hlb1</em> seedlings appeared to be mis-sorted to the apoplast (<a class="usa-link" href="#fig11">Figure 11G</a>). Mis-sorting of RFP-AFVY was quantified by obtaining the ratio of fluorescence from a fixed region of the apoplast to an equivalent area within the vacuole lumen in epidermal cells of the hypocotyl (<a class="usa-link" href="#fig11">Figure 11G</a>). The higher ratio values for <em>hlb1</em> compared with the wild type support that RFP-AFVY is mis-sorted in the mutant (<a class="usa-link" href="#fig11">Figure 11H</a>). However, RFP-AFVY still accumulated in the vacuole lumen of several cell types in <em>hlb1</em> similar to the wild type, indicating that vacuolar targeting defects in the mutant are only partial (<a class="usa-link" href="#fig11">Figure 11G</a>).</p>
<p>Other evidence showing that trafficking to the vacuole in <em>hlb1</em> is not completely compromised was obtained from incubating wild-type and <em>hlb1</em> seedlings expressing PIN2-GFP in the dark for 12 h. Under such conditions, PIN2-GFP in the wild type was shown to accumulate in lytic vacuoles (<a aria-describedby="bib35" class="usa-link" href="#bib35">Feraru et al., 2012</a>). Here, we observed that both the wild type and <em>hlb1</em> accumulated PIN2-GFP in lytic vacuoles in the dark (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 16E</a>). In addition, the styryl dye FM1-43 labeled the tonoplast in elongating root cells of light-grown <em>hlb1</em> seedlings, indicating that trafficking to the vacuole can still proceed in the mutant. FM1-43 labeling of root cells in the elongation zone and <a class="usa-link" href="#def6">DIC</a> microscopy of peripheral root cap cells also did not reveal dramatic differences in vacuolar morphology between the wild type and <em>hlb1</em> (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figures 16F and 16G</a>). However, the possibility that there are subtle differences in vacuolar dynamics and morphology between the wild type and <em>hlb1</em> cannot be ruled out. Furthermore, differences in vacuole morphology between the wild type and <em>hlb1</em> could be more apparent in other cell types or amplified under certain growth conditions (e.g., light versus dark).</p></section><section id="s16"><h3 class="pmc_sec_title">Genetic Interaction Studies Indicate that <em>hlb1</em> and <em>ben1</em> Function in Common Root Developmental Pathways despite Opposite Effects on Membrane Trafficking</h3>
<p>As noted above, it was previously shown that <em>ben1</em> had reduced intracellular accumulation of PIN-GFP following treatment with 50 μM <a class="usa-link" href="#def8">BFA</a>, a phenotype opposite to that of <em>hlb1</em> (<a aria-describedby="bib102" class="usa-link" href="#bib102">Tanaka et al., 2009</a>, <a aria-describedby="bib103" class="usa-link" href="#bib103">2013</a>). The opposite trafficking phenotypes of <em>ben1</em> and <em>hlb1</em> were surprising given their similar root developmental phenotypes and <a class="usa-link" href="#def5">LatB</a> hypersensitivity (<a class="usa-link" href="#fig9">Figure 9</a>). To better understand the genetic relationship between <em>hlb1</em> and <em>ben1</em>, <em>hlb1 ben1</em> double mutants were generated and their phenotypes were compared with <em>hlb1</em> and <em>ben1</em> single mutants. The extent of hypersensitivity of <em>hlb1 ben1</em> double mutants to <a class="usa-link" href="#def5">LatB</a> was similar to single <em>hlb1</em> and <em>ben1</em> mutants (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 13A</a>). Furthermore, root hair length of <em>hlb1 ben1</em> in the absence of <a class="usa-link" href="#def5">LatB</a> was slightly shorter than <em>hlb1</em> single mutants but similar to <em>ben1</em> single mutants. In addition, <em>hlb1 ben1</em> had slightly shorter primary roots in <a class="usa-link" href="#def5">LatB</a>-free media compared with <em>hlb1</em> and <em>ben1</em> single mutants (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figures 13B to 13D</a>).</p>
<p>We next asked how trafficking of <a class="usa-link" href="#def2">PM</a> markers is affected in <em>hlb1 ben1</em> double mutants compared with <em>hlb1</em> and <em>ben1</em> single mutants. For these assays, we used the endocytic styryl tracer dye FM1-43. Consistent with the PIN2-GFP assays, FM1-43 in the wild type and <em>hlb1</em> formed fluorescent agglomerates after a 1-h treatment with 50 μM <a class="usa-link" href="#def8">BFA</a> (<a class="usa-link" href="#fig12">Figures 12A</a> and <a class="usa-link" href="#fig12">12B</a>). Also in agreement with the PIN2-GFP assays was the observation that FM1-43 agglomerates persisted in <em>hlb1</em> compared with the wild type after <a class="usa-link" href="#def8">BFA</a> washout (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 17</a>). By contrast, <em>ben1</em> had diminished accumulation of FM1-43 agglomerates after <a class="usa-link" href="#def8">BFA</a> treatment, in agreement with previous reports (<a class="usa-link" href="#fig12">Figure 12C</a>) (<a aria-describedby="bib102" class="usa-link" href="#bib102">Tanaka et al., 2009</a>, <a aria-describedby="bib103" class="usa-link" href="#bib103">2013</a>). In <em>hlb1 ben1</em>, FM1-43 accumulation was diminished to a similar extent as <em>ben1</em> single mutants (<a class="usa-link" href="#fig12">Figures 12D</a> and <a class="usa-link" href="#fig12">12E</a>). The similar <a class="usa-link" href="#def5">LatB</a>-hypersensitive response and root growth defects of <em>hlb1 ben1</em> to single <em>hlb1</em> and <em>ben1</em> mutants indicate that <em>hlb1</em> and <em>ben1</em> function in common genetic pathways. Because <em>hlb1 ben1</em> trafficking and root hair defects mirrored those of the <em>ben1</em> single mutant, <em>ben1</em> likely functions upstream of <em>hlb1</em>.</p>
<figure class="fig xbox font-sm" id="fig12"><h4 class="obj_head">Figure 12.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4826010_PC_TPC201500794RAR2_f12.jpg" target="_blank"><img alt="Figure 12." class="graphic zoom-in" height="942" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4bed/4826010/1b82f32905e2/PC_TPC201500794RAR2_f12.jpg" width="789"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/fig12/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><em>HLB1</em> Acts Downstream of <em>BEN1</em>.</p>
<p><strong>(A)</strong> to <strong>(D)</strong> Intracellular accumulation of the styryl dye FM 1-43 is altered in <em>ben1</em> and <em>hlb1 ben1</em> but not in the wild type or <em>hlb1</em>. FM1-43 intracellular agglomerates after treatment with <a class="usa-link" href="#def8">BFA</a> in <em>ben1</em> and <em>hlb1 ben1</em> are fewer and have less intense fluorescence than the wild type and <em>hlb1</em>. Bar = 20 μm.</p>
<p><strong>(E)</strong> Quantification of FM1-43 agglomerates in root cells after treatment with <a class="usa-link" href="#def8">BFA</a> for 1 h. Statistical significance was determined by one-way ANOVA. Values are means (<em>n</em> &gt; 200 cells) ± <span class="font-variant-small-caps">se</span> from at least 20 independent seedlings. Different letters indicate significant differences among means (P &lt; 0.05; Tukey’s test).</p>
<p><strong>(F)</strong> Growth of <em>ben1</em> and <em>hlb1 ben1</em> seedlings is more sensitive to 5 μM <a class="usa-link" href="#def8">BFA</a> compared with the wild type and <em>hlb1</em>. Bar = 5 mm.</p>
<p><strong>(G)</strong> Quantification of primary root growth (expressed as percent of controls) of the wild type, <em>hlb1</em>, <em>ben1</em>, and <em>hlb1 ben1</em> in 5 μM <a class="usa-link" href="#def8">BFA</a>. Statistical significance was determined by one-way ANOVA. Values are means (<em>n</em> = 18 to 32 roots) ± <span class="font-variant-small-caps">se</span>. Different letters indicate significant differences among means (P &lt; 0.05; Tukey’s test).</p></figcaption></figure><p>Another phenotype reported for <em>ben1</em> seedlings is their hypersensitivity to growth on moderate concentrations of <a class="usa-link" href="#def8">BFA</a> (<a aria-describedby="bib102" class="usa-link" href="#bib102">Tanaka et al., 2009</a>). Given that both <em>ben1</em> and <em>hlb1</em> phenocopied each other in regard to root development and hypersensitivity to <a class="usa-link" href="#def5">LatB</a>, the response of <em>hlb1</em> to <a class="usa-link" href="#def8">BFA</a> was examined. Consistent with previous reports, <em>ben1</em> seedlings displayed heightened sensitivity to low doses of <a class="usa-link" href="#def8">BFA</a> compared with the wild type. By contrast, <em>hlb1</em> seedlings showed wild-type sensitivity to <a class="usa-link" href="#def8">BFA</a>. Furthermore, seedlings of the <em>hlb1 ben1</em> double mutant displayed similar hypersensitivity to <a class="usa-link" href="#def8">BFA</a> compared to seedlings of <em>ben1</em> single mutants, providing additional genetic evidence that <em>ben1</em> functions upstream of <em>hlb1</em> (<a class="usa-link" href="#fig12">Figures 12F</a> and <a class="usa-link" href="#fig12">12G</a>).</p></section></section><section id="s17"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p>The endomembrane system is essential for normal plant cell development, yet many protein components that modulate its function remain to be identified. The distinct effects of cytoskeleton-disrupting chemicals on plant growth provide the basis for convenient forward-genetic screens to isolate mutants with altered sensitivity to these compounds. However, such screens have rarely been applied for studies on the plant cytoskeleton despite their strong potential to uncover mechanisms underlying the cytoskeletal basis of plant development. Forward-genetic screens to isolate mutants that respond differentially to cytoskeleton-disrupting compounds have been limited to those that alter microtubules (<a aria-describedby="bib75" class="usa-link" href="#bib75">Naoi and Hashimoto, 2004</a>; <a aria-describedby="bib83" class="usa-link" href="#bib83">Paredez et al., 2008</a>). Here, we used a forward-genetic approach with the potent actin-disrupting compound <a class="usa-link" href="#def5">LatB</a> and identified the recessive mutant <em>hlb1</em>, which displayed enhanced <a class="usa-link" href="#def5">LatB</a> sensitivity compared with the wild type. We determined that <em>HLB1</em> encodes a six <a class="usa-link" href="#def7">TPR</a> motif-containing protein with a conserved C-terminal domain. We found that HLB1 is a plant-specific protein that could potentially link the actin cytoskeleton with post-Golgi traffic. Loss of function of <em>HLB1</em> leads to alterations in protein recycling to the <a class="usa-link" href="#def2">PM</a> and to an F-actin network that is more susceptible to actin-specific chemical antagonists. This, in turn, results in seedlings with root hair polarity defects and reduced primary root growth. We demonstrated that HLB1 associates with the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> through its conserved C-terminal domain and functions with the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a>-localized ARF-GEF protein MIN7/BEN1 (<a aria-describedby="bib79" class="usa-link" href="#bib79">Nomura et al., 2006</a>; <a aria-describedby="bib102" class="usa-link" href="#bib102">Tanaka et al., 2009</a>) in overlapping endomembrane trafficking pathways. These data provide important insight into a previously undescribed component of the endomembrane trafficking machinery that is highly conserved and unique to land plants.</p>
<section id="s18"><h3 class="pmc_sec_title">HLB1 Is a Component of the Plant Post-Golgi Trafficking Machinery</h3>
<p>Our work experimentally demonstrates that a <a class="usa-link" href="#def7">TPR</a> domain protein, namely, HLB1, is localized to post-Golgi compartments. Other <a class="usa-link" href="#def7">TPR</a> domain-containing proteins among the 147 in Arabidopsis, including SHORT AND SWOLLEN ROOT1 and MET1, localize to mitochondria and chloroplasts, respectively, consistent with their functions in oxidative stress-mediated root development and in the assembly of the photosynthetic apparatus (<a aria-describedby="bib7" class="usa-link" href="#bib7">Bhuiyan et al., 2015</a>; <a aria-describedby="bib119" class="usa-link" href="#bib119">Zhang et al., 2015</a>). The <a class="usa-link" href="#def7">TPR</a> protein CLUMPED CHLOROPLAST1 (CLMP1) localizes to distinct foci in the cytoplasm. Although the exact identity of the CLMP1 foci remains unknown, their adjacent location to chloroplasts supports their role in plastid partitioning and distribution (<a aria-describedby="bib115" class="usa-link" href="#bib115">Yang et al., 2011</a>). Here, we show that the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a>-localized HLB1 protein functions in the recycling of proteins and other lipophilic cargo from post-Golgi compartments to the <a class="usa-link" href="#def2">PM</a> after their uptake by endocytosis. This conclusion is supported by the observation that <a class="usa-link" href="#def8">BFA</a>-induced PIN2-GFP and FM1-43 agglomerates persisted in <em>hlb1</em> mutants after <a class="usa-link" href="#def8">BFA</a> washout.</p>
<p><em>hlb1</em> also had partial defects in constitutive secretion, which were most apparent in cells of young seedlings including root hairs and epidermal pavement cells of cotyledons. In this regard, <a class="usa-link" href="#def5">LatB</a> treatment was previously shown to alter the tip localization of the <a class="usa-link" href="#def3">TGN</a>-localized secretory marker YFP-RabA4B in growing root hairs (<a aria-describedby="bib86" class="usa-link" href="#bib86">Preuss et al., 2004</a>). The disrupted gradient of F-actin meshwork in <em>hlb1</em> root hairs could explain the tip secretion and root hair growth defects of the mutant. The mis-sorting of RFP-AFVY to the apoplast of <em>hlb1</em> is reminiscent of another post-Golgi trafficking mutant called <em>unhinged-1</em>, which is disrupted in a gene encoding VASCULAR SORTING PROTEIN51 (<a aria-describedby="bib81" class="usa-link" href="#bib81">Pahari et al., 2014</a>). Mis-sorting of RFP-AFVY was only partial as the marker still accumulated to a large extent in the vacuole lumen of <em>hlb1</em>. The RFP-AFVY assays were verified by observations that PIN2-GFP accumulated in lytic vacuoles when incubated in the dark and that FM1-43 decorated the tonoplast in <em>hlb1</em>. Future studies will determine whether other markers of vacuolar targeting and morphology are altered in <em>hlb1</em>.</p>
<p>In animals, some <a class="usa-link" href="#def7">TPR</a> domain proteins localize to the endomembrane system. For instance, the endosome-localized sorting nexins SNX20 and SNX21 contain <a class="usa-link" href="#def7">TPR</a> motifs in addition to their phox homology (PX) phosphoinositide binding domain. These sorting nexins have been proposed to function as membrane-associated scaffolding proteins that could potentially interact with transmembrane ligands for transport through the endomembrane system (<a aria-describedby="bib16" class="usa-link" href="#bib16">Clairfeuille et al., 2015</a>). In addition to the <a class="usa-link" href="#def7">TPR</a> motifs, HLB1 contains a C-terminal domain that is highly conserved among land plants. Protein domain prediction software indicates that this C-terminal domain can form a helical structure and manual annotation revealed that it had weak similarity (∼15%) to the pleckstrin homology (PH) domain of Arabidopsis PLECKSTRIN HOMOLOG1. Like PX domains, PH domains bind to phosphoinositides and could facilitate targeting of proteins containing these domains to membranes (<a aria-describedby="bib10" class="usa-link" href="#bib10">Boss and Im, 2012</a>). Consistent with the possibility that the C-terminal domain of HLB1 has similar functions to canonical PX or PH domains, we showed through GFP-HLB1 deletion constructs that HLB1 localized to <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> compartments primarily through its C-terminal domain. Although direct binding of the HLB1 to lipids has yet to be demonstrated, the root hair defects of <em>hlb1</em> suggest that the HLB1 C-terminal domain could provide a mechanism for integrating phosphoinositide signaling with actin-mediated tip growth regulation. In this regard, phosphoinositides have been implicated in actin-mediated root hair tip growth (<a aria-describedby="bib110" class="usa-link" href="#bib110">Vincent et al., 2005</a>; <a aria-describedby="bib116" class="usa-link" href="#bib116">Yoo et al., 2012</a>) and mutants disrupted in various steps of phosphoinositide metabolism have altered root hair development (<a aria-describedby="bib63" class="usa-link" href="#bib63">Kusano et al., 2008</a>; <a aria-describedby="bib95" class="usa-link" href="#bib95">Stenzel et al., 2008</a>; <a aria-describedby="bib105" class="usa-link" href="#bib105">Thole et al., 2008</a>) that mirrors the root hair phenotypes of <em>hlb1</em>. One explanation for the root hair defects of the <em>hlb1</em> mutant is that HLB1 functions in recycling of phosphoinositide metabolizing enzymes or phosphoinositides to the <a class="usa-link" href="#def2">PM</a> of the root hair apex. Future studies will test this hypothesis using fluorescently tagged phosphoinositide reporters and enzymes.</p>
<p>The trafficking defects of <em>hlb1</em> were similar to the previously characterized dominant <em>bex1</em> and <em>bex5</em> mutants (<a aria-describedby="bib35" class="usa-link" href="#bib35">Feraru et al., 2012</a>; <a aria-describedby="bib104" class="usa-link" href="#bib104">Tanaka et al., 2014</a>). In this regard, we speculate that BEX1, an ARF1A1C small GTPase, and BEX5, a RABA1b small GTPase, could functionally converge onto similar trafficking pathways mediated by HLB1. It is important to note, however, that despite the similarities in membrane trafficking phenotypes of <em>hlb1</em> to those of the <em>bex</em> mutants, they differed in their seedling sensitivity to <a class="usa-link" href="#def8">BFA</a>. Whereas <em>bex1</em> and <em>bex5</em> seedling growth was reported to be hypersensitive to moderate <a class="usa-link" href="#def8">BFA</a> concentrations (<a aria-describedby="bib35" class="usa-link" href="#bib35">Feraru et al., 2012</a>; <a aria-describedby="bib104" class="usa-link" href="#bib104">Tanaka et al., 2014</a>), <em>hlb1</em> seedlings showed wild-type <a class="usa-link" href="#def8">BFA</a> sensitivity. T-DNA knockout <em>bex5</em> mutants did not show the trafficking defects characteristic of the dominant-negative <em>bex5-1</em>, suggesting functional redundancy with other closely related RABA1b GTPases (<a aria-describedby="bib35" class="usa-link" href="#bib35">Feraru et al., 2012</a>). In the future, it will be interesting to test how <em>bex</em> loss-of-function mutants respond to <a class="usa-link" href="#def5">LatB</a> and to determine whether ARF1A1C and RABA1b operate in common exocytic trafficking pathways as HLB1. Although HLB1 is encoded by a single Arabidopsis gene and the <em>hlb1</em> mutant showed clear defects in exocytosis and heightened sensitivity to <a class="usa-link" href="#def5">LatB</a>, the mild root phenotypes of <em>hlb1</em> in the absence of <a class="usa-link" href="#def5">LatB</a> suggests that HLB1 could function redundantly with other unknown nonhomologous proteins in plant development.</p>
<p>A significant finding from our study is the identification of MIN7/BEN1, a <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a>-localized ARF-GEF protein belonging to the BIG subfamily (<a aria-describedby="bib79" class="usa-link" href="#bib79">Nomura et al., 2006</a>; <a aria-describedby="bib102" class="usa-link" href="#bib102">Tanaka et al., 2009</a>), as an HLB1 interactor. Although the nature of the HLB1-MIN7/BEN1 interaction at the biochemical level has yet to be established, results presented here point to HLB1 and MIN7/BEN1 operating in converging endomembrane trafficking pathways. In addition to the clear colocalization of HLB1 and MIN7/BEN1 in distinct <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> foci, double mutant studies demonstrate that <em>ben1</em> is epistatic to <em>hlb1</em> with regard to <a class="usa-link" href="#def5">LatB</a> hypersensitivity, FM1-43 recycling, root hair growth, and sensitivity to moderate concentrations of <a class="usa-link" href="#def8">BFA</a>. Because the <em>hlb1 ben1</em> double mutant most closely resembles the phenotypes of the <em>ben1</em> single mutant, <em>MIN7/BEN1</em> likely functions upstream of <em>HLB1</em>. This observation reinforces the conclusion that the <em>hlb1</em> mutant is deficient specifically in endocytic membrane recycling. The epistatic effect of <em>ben1</em> on <em>hlb1</em> with respect to the formation of FM1-43 <a class="usa-link" href="#def8">BFA</a> bodies suggests that in <em>hlb1</em>, the pool of <a class="usa-link" href="#def2">PM</a>-derived cargo for <a class="usa-link" href="#def3">TGN</a>-regulated recycling likely originates from endocytic uptake mediated by BEN1.</p></section><section id="s19"><h3 class="pmc_sec_title">HLB1 Potentially Links Post-Golgi Traffic to the Actin Cytoskeleton in Plants</h3>
<p>In addition to the root hair tip growth defects and strong hypersensitivity of <em>hlb1</em> seedlings to <a class="usa-link" href="#def5">LatB</a>, our results support the possibility that HLB1 may exert its functions in endomembrane traffic through actin. This is supported by several lines of evidence. First, confocal microscopy analyses showed that the HLB1-GFP bodies tracked along F-actin bundles in vivo. HLB1-GFP localization was reminiscent of the “beads-on-a-string” pattern exhibited by the NET family of actin membrane adaptors (<a aria-describedby="bib112" class="usa-link" href="#bib112">Wang et al., 2014</a>). Second, the localization of the HLB1-GFP foci was rapidly disrupted by low doses of <a class="usa-link" href="#def5">LatB</a>. Third, PIN2-GFP recycling to the <a class="usa-link" href="#def2">PM</a>, which was only minimally affected by low doses of <a class="usa-link" href="#def5">LatB</a> in the wild type, was severely inhibited in <em>hlb1</em>. The <a class="usa-link" href="#def5">LatB</a>-induced PIN2-GFP agglomerates were strikingly similar to those induced by high concentrations of <a class="usa-link" href="#def8">BFA</a>. Despite these results, we cannot discard the possibility that the movement of HLB1-GFP punctae along F-actin could simply reflect that HLB1 is on the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> surface, which would not support specific functional linkage between actin and HLB1. Furthermore, we have found that <a class="usa-link" href="#def5">LatB</a> also rapidly affected the distribution of other <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> markers in root cells, indicating that the observed effects of the drug on HLB1 localization are not exclusive for this protein. Nonetheless, we showed that actin dynamics in the tip of elongating root hairs are disrupted in <em>hlb1</em> in support of the notion that HLB1 could be linked to actin function. The lack of differences in F-actin organization and dynamics in cells of the root elongation zone between non-<a class="usa-link" href="#def5">LatB</a>-treated wild type and <em>hlb1</em> does not preclude the possibility that subtle differences in F-actin dynamics exist between the wild type and <em>hlb1</em>, and such differences may manifest only at the level of single actin filaments (<a aria-describedby="bib94" class="usa-link" href="#bib94">Staiger et al., 2009</a>). While a direct interaction between HLB1 with actin awaits for experimental validation, our analyses support that HLB1 exerts a role in post-Golgi membrane traffic in relation to the actin cytoskeleton, which may occur either directly or through interactions with actin modulators.</p>
<p>The involvement of the actin cytoskeleton in endocytosis and protein recycling has been documented in plants using both pharmacological and genetic approaches. For example, sterol endocytosis was found to be disrupted in <em>act2</em> mutants and by treating wild-type plants with cytochalasin D (<a aria-describedby="bib45" class="usa-link" href="#bib45">Grebe et al., 2003</a>). <a class="usa-link" href="#def5">LatB</a> and cytochalasin were also shown to influence the recycling and polar localization of PIN proteins in root and leaf epidermal cells (<a aria-describedby="bib40" class="usa-link" href="#bib40">Geldner et al., 2001</a>; <a aria-describedby="bib74" class="usa-link" href="#bib74">Nagawa et al., 2012</a>). Finally, <a class="usa-link" href="#def5">LatB</a> was reported to interfere with protein traffic to the vacuole in protoplasts and intact roots (<a aria-describedby="bib58" class="usa-link" href="#bib58">Kim et al., 2005</a>). It is important to note, however, that in the aforementioned studies, high micromolar concentrations of <a class="usa-link" href="#def5">LatB</a> were used to trigger the reported effects on protein trafficking. The fact that <em>hlb1</em> displayed strong protein trafficking and seedling growth defects in response to low nanomolar doses of <a class="usa-link" href="#def5">LatB</a> supports the hypothesis that HLB1 is a crucial component of post-Golgi traffic that is strongly dependent on the maintenance of an intact actin cytoskeleton. This hypothesis is further reinforced by the observation that heightened sensitivity to <a class="usa-link" href="#def5">LatB</a> was more pronounced in <em>hlb1</em> and <em>ben1</em> mutants compared with mutants with lesions in genes encoding actin itself (e.g., <em>act2</em> and <em>act8</em>) or mutants to a suite of other actin regulatory proteins, most of which were not <a class="usa-link" href="#def5">LatB</a> hypersensitive. In this regard, another possible explanation for the hypersensitivity of <em>hlb1</em> mutants to <a class="usa-link" href="#def5">LatB</a> is that loss of function of <em>HLB1</em> causes mild recycling defects of several proteins directly involved in actin function particularly those that localized to the <a class="usa-link" href="#def2">PM</a>. The collective recycling defects of several <a class="usa-link" href="#def2">PM</a>-localized actin regulatory proteins, although mild, could sensitize <em>hlb1</em> to even low doses of actin antagonists.</p>
<p>Despite the recycling defects of PIN2-GFP in the <em>hlb1</em> background, <em>hlb1</em> did not exhibit the agravitropic phenotype characteristic of <em>pin2</em> mutants and were not compromised in root gravitropism when reoriented by 90 degrees. The absence of any measurable <em>pin2</em>-like phenotypes in <em>hlb1</em>, <em>ben1</em>, and <em>hlb1 ben1</em> double mutants supports that the protein recycling defects resulting from loss of HLB1 and BEN1 function may not be sufficient to completely alter certain PIN2-mediated seedling root growth processes. This observation further reinforces the hypothesis that HLB1 and BEN1 function redundantly with other yet to be identified <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a>-localized proteins in cargo recycling. The possibility that HLB1 has redundant functions with other nonhomologous proteins can also be inferred from the lack of global defects in VTI12 and MIN7/BEN1 distribution in the <em>hlb1</em> mutant. Alternatively, MIN7/BEN1 could recruit HLB1 to the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> explaining the lack of any dramatic differences in MIN7/BEN1-GFP distribution between the wild type and <em>hlb1</em>. By contrast, root gravitropism has been shown to be driven by differential degradation of PIN2 in the vacuole of cells in the elongation zone and PIN3 relocalization to the bottom side of the columella cell <a class="usa-link" href="#def2">PM</a> (<a aria-describedby="bib62" class="usa-link" href="#bib62">Kleine-Vehn et al., 2008</a>, <a aria-describedby="bib61" class="usa-link" href="#bib61">2010</a>). The lack of root gravitropism defects in <em>hlb1</em> is consistent with the observation that loss of HLB1 function has only mild effects on vacuolar trafficking and suggests that HLB1-mediated PIN2 recycling to the <a class="usa-link" href="#def2">PM</a> has minimal contributions to root gravitropic responses. The normal root gravity response of <em>hlb1</em> also indicates that unlike PIN2, recycling of PIN3 and possibly other PIN proteins is independent of HLB1 function. However, additional experiments are needed to determine whether the trafficking of other PIN or <a class="usa-link" href="#def2">PM</a>-localized proteins is affected in the <em>hlb1</em> mutant.</p>
<p>In summary, a forward-genetic screen for altered sensitivity to the actin-disrupting drug <a class="usa-link" href="#def5">LatB</a> led to the discovery of a previously uncharacterized protein involved in endomembrane trafficking in plants. The <a class="usa-link" href="#def7">TPR</a> domain-containing HLB1 protein, which is encoded by a single gene in Arabidopsis, functions in protein recycling to the <a class="usa-link" href="#def2">PM</a> but could also contribute to constitutive secretion and trafficking to the vacuole in specific cell types. The shared phenotypes of <em>hlb1</em> with other exocytic and endocytic trafficking mutants such as <em>ben1</em>, <em>bex1</em>, and <em>bex5</em> indicate that the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a>-localized HLB1 protein lies at a crucial step where exocytic and endocytic pathways converge and therefore represent a class of plant-specific membrane scaffolding proteins likely involved in linking post-Golgi traffic with the actin cytoskeleton.</p></section></section><section id="s20"><h2 class="pmc_sec_title">METHODS</h2>
<section id="s21"><h3 class="pmc_sec_title">Forward-Genetic Screening and Mapping of <em>hlb1</em>
</h3>
<p>An <em>Arabidopsis thaliana</em> activation-tagged T-DNA seed stock (CS31100) from the ABRC was sterilized in 95% ethanol and 20% bleach according to <a aria-describedby="bib29" class="usa-link" href="#bib29">Dyachok et al. (2009)</a>. Sterilized seeds were suspended in an autoclaved solution of 0.5× Murashige and Skoog (<a class="usa-link" href="#def11">MS</a>) medium supplemented with 1% sucrose (pH 5.7), 100 nM <a class="usa-link" href="#def5">LatB</a> (Calbiochem-EMD Chemicals), and 0.5% agar (Sigma-Aldrich) after the solution had cooled to 55°C. The seed suspension was gently swirled in a 500-mL Erlenmeyer flask to disperse the seeds uniformly and poured into 10 × 10-cm square Petri plates before the agar solidified. Petri dishes were left at 4°C for 2 d and transferred to a growth chamber with 120 μmol m<sup>−2</sup> s<sup>−1</sup> light intensity supplied with fluorescent bulbs and set to a 14-h-light and 10-h-dark cycle. Temperature in the chamber was maintained at 23°C while keeping plates in a vertical position. After 5 to 6 d, seedlings with severely inhibited primary root growth were selected. Primary root growth of the progeny of selected seedlings was compared with the wild type on a range of <a class="usa-link" href="#def5">LatB</a> concentrations.</p>
<p>Homozygous <em>hlb1</em> (Col-0 ecotype) was crossed to the Landsberg <em>erecta</em> ecotype, and F2 seedlings were selected based on their enhanced sensitivity to 50 nM <a class="usa-link" href="#def5">LatB</a>. Simple sequence length polymorphism and cleavage of amplified polymorphic site markers were used to map <em>hlb1</em> to a 60-kb region on the lower arm of chromosome 5 (<a aria-describedby="bib69" class="usa-link" href="#bib69">Lukowitz et al., 2000</a>). DNA was extracted from root tissue of 2-week-old <em>hlb1</em> seedlings by grinding in liquid nitrogen with a mortar and pestle followed by mixing with an equal volume of Plant DNAzol Reagent (Invitrogen) and chloroform. After precipitation with ethanol, DNA was washed multiple times and then dissolved in 50 μL water. Full-length genomic DNA of various genes in the 60-kb region was sequenced and compared with the Arabidopsis reference genome (TAIR10) until the 2-bp deletion was found in <em>AT5G41950</em>.</p>
<p>Three SALK lines with predicted insertions in <em>AT5G41950</em> (SALK_046760, SALK_144501, and SALK_065836) were obtained from ABRC and genotyped to ensure homozygosity using the following primers: SALK_046760LP (5′-TTGCTGGAGGAAGCTTGTAAG-3′), SALK_046760RP (5′-TGGCTGTATAACTCACCAGGG-3′), SALK_065836LP (5′-TTCCCTCTCGCATAAATTGTG-3′), SALK_065836RP (5′-TCAGGTTGCAATTTGATTTCC-3′), and LBb1.3 (5′-ATTTTGCCGATTTCGGAAC-3′). The primer pair used for SALK_144501 was the same as SALK_046760.</p>
<p>One SALK line (SALK_013761) with a predicted exon insertion in <em>AT3G43300</em> (<em>MIN7/BEN1</em>) was obtained from ABRC. Homozygous <em>ben1</em> mutants were isolated using the following primers: SALK_013761LP (5′-TGGAAAGTGAAATTGGTGAGC-3′), SALK_013761RP (5′-CAAGGATTCTTCTCTGCATGG-3′), and LBb1.3.</p></section><section id="s22"><h3 class="pmc_sec_title">Chemical Treatments and Seedling Growth Assays</h3>
<p>Working solutions of <a class="usa-link" href="#def5">LatB</a> (Calbiochem), <a class="usa-link" href="#def8">BFA</a> (Sigma-Aldrich), cytochalasin B and D (MP Biomedicals), indole-3-acetic acid, and 1-naphthaleneacetic acid (Sigma-Aldrich) were diluted from stock solutions in DMSO. Working solutions of sodium chloride and mannitol (Sigma-Aldrich) were prepared directly from solids from the manufacturer’s containers. Seedling growth assays were conducted by transplanting 3- to 4-d-old seedlings grown on solid (1% agar) onto <a class="usa-link" href="#def11">MS</a> medium supplemented with the various chemicals. Seedlings were grown vertically for an additional 4 to 7 d after transplanting in a 14-h-light (120 μmol m<sup>−2</sup> s<sup>−1</sup>) and 10-h-dark cycle or in complete darkness in a growth chamber set to 23°C. In another set of experiments, 4-d-old seedlings with straight primary roots were transplanted on new Petri plates with <a class="usa-link" href="#def11">MS</a> medium. After 1 h of vertical growth, plates were reoriented by 90 degrees and primary roots were photographed every hour for 12 h. Images of seedlings were captured using a DXM1200C digital camera (Nikon) mounted on a copy stand or an SZX12 stereomicroscope (Olympus). Hypocotyl, primary root and root hair lengths, and root curvature were measured using the Image J v1.49 software (<a class="usa-link" data-ga-action="click_feat_suppl" href="http://rsb.info.nih.gov/ij/" rel="noopener noreferrer" target="_blank">http://rsb.info.nih.gov/ij/</a>).</p>
<p>The 3- to 4-d-old wild-type (Col-0 ecotype) and <em>hlb1</em> seedlings expressing <em>PIN2-GFP</em> or <em>HLB1-GFP</em> were incubated on <a class="usa-link" href="#def8">BFA</a> or <a class="usa-link" href="#def5">LatB</a> diluted with <a class="usa-link" href="#def11">MS</a> medium on welled plates for various time periods and mounted on glass slides prior to imaging with a Leica TCS AOBS confocal scanning laser microscope (Leica Microsystems). For endocytic assays, a working solution of 4 μM FM1-43 (Invitrogen) was prepared in <a class="usa-link" href="#def11">MS</a> medium from a 10 mM aqueous solution. After incubating for 30 min in liquid <a class="usa-link" href="#def11">MS</a> medium, 3- to 4-d-old seedlings were pulse labeled with FM1-43 for 5 min at room temperature. Seedlings were then treated with 50 μM <a class="usa-link" href="#def8">BFA</a> for 1 h prior to confocal imaging. In a separate set of seedlings expressing PIN2-GFP or pulse-labeled with FM1-43, <a class="usa-link" href="#def8">BFA</a> was washed out by transferring seedlings to liquid <a class="usa-link" href="#def11">MS</a> medium prior to imaging.</p></section><section id="s23"><h3 class="pmc_sec_title">Generation of HLB1-Green Fluorescent Protein Constructs</h3>
<p>The <em>HLB1pro:HLB1-GFP</em> construct was generated from full-length <em>HLB1</em> gDNA extracted from roots of 2-week-old wild-type Arabidopsis seedlings using the Plant DNAzol Reagent (Invitrogen). Full-length <em>HLB1</em> gDNA including ∼400 bp immediately preceding the start codon (3319 bp total) was amplified using the following primers: HLBpro-F-XmaI (5′-CATCCCGGGGACTACTTATATCTTTCACTCTTGTTC-3′) and HLBpro-R-XmaI (5′-TACCCCGGGCACCGGTGATAATACCGGCTA-3′). The product was cloned into a modified pCAMBIA1390 vector as described (<a aria-describedby="bib113" class="usa-link" href="#bib113">Wang et al., 2008</a>).</p>
<p><em>35Spro:GFP-HLB1</em> was generated from full-length <em>HLB1</em> cDNA (1698 bp) amplified using the following primers: HLB-F-NcoI (5′-CATCCATGGTAATGGCGGATACTGTTGAAGAG-3′) and HLB-R-SpeI (5′- CATACTAGTTTAACCGGTGATAATACCGGC-3′). The resulting product was digested with <em>Nco</em>I and <em>Spe</em>I (New England Biolabs) and cloned into a modified pCAMBIA1390 vector behind the CaMV 35S promoter driving expression of GFP (<a aria-describedby="bib113" class="usa-link" href="#bib113">Wang et al., 2008</a>). To generate the GFP-HLB1<sup>∆CTD</sup> fusion, a truncated <em>HLB1</em> cDNA fragment, from position 1 to 1482 bp, was amplified using the primers HLB1-F-NcoI and HLB1<sup>∆CTD</sup>-R-SpeI (5′-TACACTAGTTTATACCGTCTTCCTCTCCACGT-3′), and the resulting product was digested with <em>Nco</em>I and <em>Spe</em>I (New England Biolabs) and cloned into a modified pCAMBIA1390 vector. Similarly, the GFP-HLB1<sup>∆TPR</sup> fusion was created by first amplifying <em>HLB1</em> cDNA from position 1 to 609 bp with the following primers: HLB1<sup>∆TPR</sup>-F-NcoI (5′-CATCCATGGTAATGGCGGATACTGTTGAAGAG-3′) and HLB1<sup>∆TPR</sup>-R-SpeI (5′-TACACTAGTACGTTGGCGAGATCGACC-3′). This fragment was digested with <em>Nco</em>I and <em>Spe</em>I (New England Biolabs) and cloned into a modified pCAMBIA1390 vector to created HLB1<sup>1-609</sup>. Next, an <em>HLB1</em> cDNA fragment was amplified from position 1276 to 1698 bp with the following primers: HLB1<sup>∆TPR(2)</sup>-F-SpeI (5′-CATACTAGTCTACCTCTCCCGCATCTTAAAGT-3′) and HLB1<sup>∆TPR(2)</sup>-R-PmlI (5′-TACCACGTGTTAACCGGTGATAATACCGGC-3′). This fragment was digested with <em>Spe</em>I and <em>Pml</em>I (New England Biolabs) and cloned into the HLB1<sup>1-609</sup> construct. The <em>35Spro:OsHLB1-GFP</em> construct was generated by amplifying full-length <em>OsHLB1</em> cDNA (1449 bp) with the primers OsHLB1-F-SalI (5′-CATGTCGACATGGAGGATTCTGCGCGG-3′) and OsHLB1-R-EcoRI (5′-GTAGAATTCACCAGTAATGATACCTGCCAATACA-3′). The product was digested with <em>Sal</em>I and <em>Eco</em>RI (New England Biolabs) and again cloned into a modified pCAMBIA1390 vector. Constructs were introduced into <em>hlb1-1</em> via <em>Agrobacterium tumefaciens-</em>mediated transformation using the floral dip method (<a aria-describedby="bib17" class="usa-link" href="#bib17">Clough and Bent, 1998</a>).</p></section><section id="s24"><h3 class="pmc_sec_title">Confocal Microscopy and Colocalization Studies</h3>
<p><em>hlb1-1</em> seedlings expressing HLB1pro:HLB1-GFP were crossed with various red- or cyan-emitting marker lines. The PIN2-GFP, Man49-mCherry, SYP61-CFP, mCherry-VTI12, mCherry-RabA1g, mCherry-Ara6, GFP-ABD2-GFP, mCherry-ABD2-mCherry, MIN7/BEN1-GFP, RFP-AFVY, and SEC-RFP lines were described previously (<a aria-describedby="bib78" class="usa-link" href="#bib78">Nelson et al., 2007</a>; <a aria-describedby="bib90" class="usa-link" href="#bib90">Robert et al., 2008</a>; <a aria-describedby="bib39" class="usa-link" href="#bib39">Geldner et al., 2009</a>; <a aria-describedby="bib80" class="usa-link" href="#bib80">Nomura et al., 2011</a>; <a aria-describedby="bib35" class="usa-link" href="#bib35">Feraru et al., 2012</a>; <a aria-describedby="bib88" class="usa-link" href="#bib88">Renna et al., 2013</a>; <a aria-describedby="bib28" class="usa-link" href="#bib28">Dyachok et al., 2014</a>). To generate the UBQ10pro:Lifeact-mEGFP construct, mEGFP with the 51-bp Lifeact sequence directly upstream was amplified from plasmid DNA described by <a aria-describedby="bib109" class="usa-link" href="#bib109">Vidali et al. (2009)</a> using the following primers: Lifeact-F-EcoRI (5′-CATGAATTCATGGGTGTCGCAGATTTGATC-3′) and Lifeact-R-SpeI (5′-TACACTAGTTTACTTGTACAGCTCGTCCATGC-3′). The resulting product was digested with <em>Eco</em>RI and <em>Spe</em>I (New England Biolabs) and cloned behind the UBQ10 promoter in a modified pCAMBIA1390 vector. Five-day-old seedlings coexpressing HLB1-GFP and various red- or cyan-emitting markers were imaged with a Leica confocal microscope or Perkin-Elmer UltraView ERS spinning-disc confocal microscope.</p>
<p>For quantification of F-actin and <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> dynamics, images of epidermal cells in the root maturation zone or root hairs growing along the bottom of 48 × 64-mm cover slips coated with <a class="usa-link" href="#def11">MS</a> medium in 0.5% agar were captured every 0.5 to 1 s for 1 to 3 min with a Perkin-Elmer spinning-disc confocal microscope equipped with a Zeiss C-Apochromat 63× water or a 100× oil immersion objective.</p></section><section id="s25"><h3 class="pmc_sec_title">Image Analysis</h3>
<p>Percentage of occupancy of F-actin was determined using the algorithm described by <a aria-describedby="bib49" class="usa-link" href="#bib49">Higaki et al. (2010)</a>. Prior to applying the Higaki algorithm, single cells from the root elongation zone were converted into binary images and filaments skeletonized by software that was developed using MATLAB R2013a (The Mathworks Inc.) as described (<a aria-describedby="bib28" class="usa-link" href="#bib28">Dyachok et al., 2014</a>). Intensity of <a class="usa-link" href="#def8">BFA</a> agglomerates was obtained using custom software written in MATLAB 8.5. The original images from the confocal microscope were first converted to gray scale and separated into 48 layers by intensity values from zero to maximum (255*255), and a binary image was created. The circle Hough Transform method was used to trace areas occupied by <a class="usa-link" href="#def8">BFA</a>- and <a class="usa-link" href="#def5">LatB</a>-induced PIN2-GFP or FM1-43 agglomerates in the binary images. The average intensity in grayscale images in each of marked agglomerate was calculated. From the same set of images used to calculate agglomerate fluorescence intensity, the number of agglomerates per cell was manually counted.</p>
<p>Quantification and statistical analyses of global F-actin and YFP-VTI12 dynamics were conducted following the methods of <a aria-describedby="bib108" class="usa-link" href="#bib108">Vidali et al. (2010)</a> and <a aria-describedby="bib28" class="usa-link" href="#bib28">Dyachok et al. (2014)</a> using images of root epidermal cells and root hairs from at least 15 time-lapse movie sequences. To adopt the method used for global analysis of F-actin to dynamics of YFP-VTI12 foci in root hairs, we first created an algorithm that utilized the Hough Transform method to detect individual organelles from binary images obtained from time-lapse movies. This algorithm allowed us to set a threshold so that fluorescence originating from the cytoplasm was excluded from our analysis.</p>
<p>To quantify fluorescence in the <a class="usa-link" href="#def2">PM</a> and cell interior of PIN2-GFP-expressing root cells, we created another algorithm. After converting images to gray scale, binary images were generated after separating the image into 48 layers and classifying the layers based on their intensity values. This allowed us to separate fluorescence of the <a class="usa-link" href="#def2">PM</a> from the cell interior where the white area in the binary image corresponded to <a class="usa-link" href="#def2">PM</a> signals and the black area corresponded to the cell interior. Fluorescence intensity was calculated by accumulating the intensity value of each pixel from the <a class="usa-link" href="#def2">PM</a> and cell interior.</p>
<p>To quantify the extent of colocalization between HLB1-GFP and other mCherry-tagged <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> markers, spinning-disc confocal images were first thresholded using the Volocity quantitation software (Perkin-Elmer). The colocalization function of Volocity allows automated thresholding and statistical analyses of red and green channels according to the methods of <a aria-describedby="bib70" class="usa-link" href="#bib70">Manders et al. (1993)</a> and <a aria-describedby="bib19" class="usa-link" href="#bib19">Costes et al. (2004)</a>. From the colocalization menu of Volocity quantitation, a region of interest representing a single cell (<a class="usa-link" href="#fig7">Figure 7A</a>) was selected, and from generated scatterplots (<a class="usa-link" href="#fig7">Figure 7F</a>) (<a aria-describedby="bib11" class="usa-link" href="#bib11">Bottanelli et al., 2012</a>), Pearson’s correlation coefficient values were calculated.</p></section><section id="s26"><h3 class="pmc_sec_title">Phylogenetic Analysis and Bioinformatics</h3>
<p>To deduce phylogeny of HLB1, sequences for 141 homologous proteins from 79 species were obtained from GenBank. The amino acid sequences were aligned by MUSCLE algorithm (<a aria-describedby="bib33" class="usa-link" href="#bib33">Edgar, 2004</a>), and unreliable sequences with confidence score lower than 0.6 were removed (<a aria-describedby="bib84" class="usa-link" href="#bib84">Penn et al., 2010</a>). The alignment is available as <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Data Set 3</a>. Subsequent phylogenetic analyses were conducted in MEGA6 (<a aria-describedby="bib101" class="usa-link" href="#bib101">Tamura et al., 2013</a>) based on the maximum likelihood method (<a aria-describedby="bib64" class="usa-link" href="#bib64">Le and Gascuel, 2008</a>). A matrix of pairwise distances was estimated by the JTT model, and the tree with the highest log likelihood (12,977) was chosen with the Neighbor-join and BIONJ algorithm. A discrete gamma distribution was used to model evolutionary rate differences among sites (+G, parameter = 3.2081). All positions with &lt;95% site coverage were eliminated.</p>
<p>To find <a class="usa-link" href="#def7">TPR</a>-containing proteins, the Arabidopsis genome was scanned with TPRpred (<a aria-describedby="bib57" class="usa-link" href="#bib57">Karpenahalli et al., 2007</a>). The resulting 587 proteins were compared against the Conserved Domain Database (<a aria-describedby="bib71" class="usa-link" href="#bib71">Marchler-Bauer et al., 2015</a>). The 147 Arabidopsis proteins were identified to have more than one <a class="usa-link" href="#def7">TPR</a> motifs. Predicated subcellular localization for the <a class="usa-link" href="#def7">TPR</a> proteins was obtained from the SubCellular Proteome Database and presented in <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Data Set 1</a>.</p></section><section id="s27"><h3 class="pmc_sec_title">Co-IP</h3>
<p>Co-IP experiments were conducted using the Pierce Classic IP kit (Thermo Scientific). Briefly, proteins were extracted by grinding roots of 2-week-old Arabidopsis seedlings expressing <em>35Spro:GFP</em>, <em>35Spro:GFP-HLB</em>, and <em>35Spro:MIN7/BEN1-GFP</em> in liquid nitrogen. The fine powder was then mixed with extraction/wash solution supplemented with 1:100 protease inhibitor cocktail (Sigma-Aldrich) by vortexing. The protein extracts were centrifuged at 16,000<em>g</em> for 10 min at 4°C. The supernatant was mixed with control resin via end-over-end shaking in a 15-mL Falcon tube for 1 h at 4°C. The mixture was then added to a spin column and centrifuged at 1000<em>g</em> for 30 s at 4°C. Rabbit Anti-GFP Monoclonal Antibody (ABCAM catalog number1218, lot number GR181836-3) was added to the cleared protein lysate and mixed by end-over-end shaking overnight at 4°C. Protein A/G plus agarose resin was then added and allowed to mix for 1 h at 4°C. The mixture was again added to a spin column and centrifuged at 1000<em>g</em> for 30 s at 4°C. The columns were washed five times with extraction/wash solution and then one time with 1× conditioning buffer. The proteins were eluted by adding 2× nonreducing lane marker sample buffer with 20 mM DTT and heating for 10 min at 100°C. Once cooled, the columns were placed inside Eppendorf tubes and centrifuged at 1000<em>g</em> for 30 s at 4°C. Composition of the eluent was analyzed by tandem mass spectrometry (Bioproximity). Briefly, the samples were digested with trypsin (10 μg/mL) overnight at 37°C. The resulting peptides were dried and redissolved in 50% acetonitrile with 0.5% trifluoroacetic acid. The peptides were resolved with Thermo Easy-nLC1000 column (C18 reverse phase, 50 cm, ϕ 75 μm), ionized with Thermo Easy Spray, and then analyzed with a Thermo Q-exactive quadrupole-Orbitrap mass spectrometer. The spectra were assigned to peptides using the OMSSA (<a aria-describedby="bib38" class="usa-link" href="#bib38">Geer et al., 2004</a>), X!Tandem (<a aria-describedby="bib20" class="usa-link" href="#bib20">Craig and Beavis, 2004</a>), and X!Hunter (<a aria-describedby="bib21" class="usa-link" href="#bib21">Craig et al., 2006</a>) algorithms.</p></section><section id="s28"><h3 class="pmc_sec_title">Accession Numbers</h3>
<p>Sequence data from this article can be found in the Arabidopsis Genome Initiative or GenBank/ EMBL databases under the following accession numbers: HLB1 (AT5G41950) and MIN7/BEN1 (AT3G43300). Protein accession numbers for HLB1 orthologs are as follows: <em>Aegilops tauschii</em>, gi|475447842; <em>Amborella trichopoda</em>, gi|548852614; <em>Arabidopsis lyrata</em>, gi|297805444; <em>Arabidopsis thaliana</em>, gi|133778836; <em>Arabis alpina</em>, gi|674240218; <em>Beta vulgaris</em>, gi|731331895; <em>Brachypodium distachyon</em>, gi|357114392; <em>Brassica napus</em>, gi|674870500; <em>Brassica rapa</em>, gi|685298732; <em>Camelina sativa</em>, gi|727533856; <em>Capsella rubella</em>, gi|565442422; <em>Chlorella variabilis</em>, gi|552830027; <em>Cicer arietinum</em>, gi|502116595; <em>Citrus clementina</em>, gi|557541493; <em>Citrus sinensis</em>, gi|641857964; <em>Coccomyxa subellipsoidea</em>, gi|545359525; <em>Coffea canephora</em>, gi|661884304; <em>Cucumis melo</em>, gi|659095407, gi|307136085; <em>Cucumis sativus</em>, gi|700200538; <em>Elaeis guineensis</em>, gi|743777795; <em>Erythranthe guttata</em>, gi|604327766; <em>Eucalyptus grandis</em>, gi|702322982; <em>Eutrema salsugineum</em>, gi|567192832; <em>Fragaria vesca</em>, gi|470147958; <em>Genlisea aurea</em>, gi|527191692; <em>Glycine max</em>, gi|356576620; <em>Glycine soja</em>, gi|734324220; <em>Gossypium arboreum</em>, gi|728819858; <em>Hordeum vulgare</em>, gi|326492972; <em>Jatropha curcas</em>, gi|643711772; <em>Lolium perenne</em>, gi|374921937; <em>Malus domestica</em>, gi|657995845; <em>Medicago truncatula</em>, gi|657404132; <em>Micromonas pusilla</em>, gi|303278672; <em>Morus notabilis</em>, gi|703112722; <em>Musa acuminate</em>, gi|695027101; <em>Nelumbo nucifera</em>, gi|720036758; <em>Nicotiana sylvestris</em>, gi|698531617; <em>Nicotiana tomentosiformis</em>, gi|697157841; <em>Oryza brachyantha</em>, gi|573926901; <em>Oryza sativa Indica</em>, gi|125532651; <em>Oryza sativa Japonica</em>, gi|108705829; <em>Ostreococcus tauri</em>, gi|693498951; <em>Phaseolus vulgaris</em>, gi|561035046; <em>Phoenix dactylifera</em>, gi|672133847; <em>Physcomitrella patens</em>, gi|162670631; <em>Populus euphratica</em>, gi|743825082; <em>Populus trichocarpa</em>, gi|550343167; <em>Prunus mume</em>, gi|645267755; <em>Prunus persica</em>, gi|462406973; <em>Pyrus × bretschneideri</em>, gi|694355322; <em>Ricinus communis</em>, gi|255555737; <em>Selaginella moellendorffii</em>, gi|302804999; <em>Sesamum indicum</em>, gi|747048717; <em>Setaria italica</em>, gi|514815687; <em>Solanum lycopersicum</em>, gi|525313946; <em>Solanum tuberosum</em>, gi|565353752; <em>Sorghum bicolor</em>, gi|242039083; <em>Tarenaya hassleriana</em>, gi|729396960; <em>Theobroma cacao</em>, gi|590675266; <em>Triticum urartu</em>, gi|474237686; <em>Vitis vinifera</em>, gi|731378821; and <em>Zea mays</em>, gi|194707904.</p></section><section id="s29"><h3 class="pmc_sec_title">Supplemental Data</h3>
<ul class="list" id="L1" style="list-style-type:none">
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 1.</strong></a> Hypocotyl and Root Hair Growth of <em>hlb1</em> Are Hypersensitive to <a class="usa-link" href="#def5">LatB</a>.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 2.</strong></a>
<em>hlb1</em> Is Hypersensitive to Cytochalasin B and D.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 3.</strong></a>
<em>hlb1</em> Is Not Hypersensitive to Other Treatments That Inhibit Primary Root Growth.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 4.</strong></a>
<em>hlb1</em> Is More Sensitive to <a class="usa-link" href="#def5">LatB</a> Compared to Other Mutants That Directly Affect the Actin Cytoskeleton.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 5.</strong></a> Cell Division and Expansion of <em>hlb1</em> Roots Are Strongly Inhibited by <a class="usa-link" href="#def5">LatB</a>.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 6.</strong></a> Quantitative Analysis of Global F-Actin Dynamics in Root Epidermal Cells and Root Hairs.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 7.</strong></a> Nature of <em>HLB1</em> Mutation and Characterization of Other <em>hlb1</em> Alleles.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 8.</strong></a> Phylogenetic Analysis of HLB1 Sequences in Land Plants.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 9.</strong></a>
<em>35Spro:GFP-HLB1</em> Complements the Hypersensitivity to <a class="usa-link" href="#def5">LatB</a> and Short Primary Root Phenotypes of <em>hlb1-1</em>.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 1</strong></a><strong>0.</strong> The C-Terminal Domain Targets HLB1 to the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a>.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 11.</strong></a> HLB1-GFP and the <a class="usa-link" href="#def3">TGN</a>/<a class="usa-link" href="#def4">EE</a> Markers (YFP-VTI12 and SYP61-CFP) in the Root Elongation Zone Form Intracellular Agglomerates upon Exposure to <a class="usa-link" href="#def5">LatB</a>.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 12.</strong></a>
<em>Oryza sativa HLB1</em> Partially Complements the Arabidopsis <em>hlb1</em> Mutant.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 13.</strong></a> HLB1 Functions in Similar Genetic Pathways as BEN1.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 14.</strong></a> Global Organization and Dynamics of YFP-VTI12 and MIN7/BEN1-GFP in <em>hlb1</em>.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 15.</strong></a> Quantification of Root Growth Orientation and Gravitropism in <em>hlb1</em> Seedlings.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 16.</strong></a>
<em>hlb1</em> Has Mild Defects in Bulk Protein Secretion and Trafficking to the Vacuole.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Figure 17.</strong></a> FM1-43 Agglomerates Persist in <em>hlb1</em> after <a class="usa-link" href="#def8">BFA</a> Washout.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 1.</strong></a> Spinning-Disc Confocal Microscopy of F-Actin Dynamics in a Root Hair from a Wild-Type and <em>hlb1</em> Seedling Expressing <em>UBQ10pro:GFP-ABD2-GFP</em>.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 2.</strong></a> Spinning-Disc Confocal Microscopy of F-Actin Dynamics in a Root Hair from a Wild-Type and <em>hlb1</em> Seedling Expressing <em>UBQ10pro:Lifeact-mGFP</em>.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 3.</strong></a> Spinning-Disc Confocal Microscopy of F-Actin Dynamics in a Root Epidermal Cell from the Maturation Zone of a Wild-Type Seedling.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 4.</strong></a> Spinning-Disc Confocal Microscopy of HLB1-GFP Dynamics in a Root Hair.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 5.</strong></a> Spinning-Disc Confocal Microscopy of Root Cells Coexpressing HLB1-GFP and mCherry-VTI12.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 6.</strong></a> Spinning-Disc Confocal Microscopy of Root Cells Coexpressing HLB1-GFP and mCherry-RabA1g.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 7.</strong></a> Spinning-Disc Confocal Microscopy of Hypocotyl Epidermal Cells Coexpressing HLB1-GFP and Man49-mCherry.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 8.</strong></a> Spinning-Disc Confocal Microscopy of Root Cells Coexpressing HLB1-GFP and mCherry-ARA6.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 9.</strong></a> Spinning-Disc Confocal Microscopy of Root Epidermal Cells in the Maturation Zone Coexpressing HLB1-GFP and mCherry-ABD2-mCherry.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 10.</strong></a> Spinning-Disc Confocal Microscopy of the Base of a Mature Root Hair Cell Coexpressing HLB1-GFP and mCherry-ABD2-mCherry.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 11.</strong></a> Spinning-Disc Confocal Microscopy of Root Cells Coexpressing HLB1-mCherry and MIN7/BEN1-GFP.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 12.</strong></a> Spinning-Disc Confocal Microscopy of YFP-VTI12 Dynamics in Wild-Type and <em>hlb1</em> Root Hairs.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Movie 13.</strong></a> Spinning-Disc Confocal Microscopy of SEC-RFP Dynamics in Wild-type and <em>hlb1</em> Root Hairs.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Data Set 1.</strong></a> List of 147 Proteins in the Arabidopsis Genome with <a class="usa-link" href="#def7">TPR</a> Motifs.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Data Set 2.</strong></a> Peptides Detected by Tandem Mass Spectrometry in the Coimmunoprecipitates.</p></li>
<li><p><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank"><strong>Supplemental Data Set 3.</strong></a> Text File of the Alignment used for the Phylogenetic Analysis Shown in <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.plantcell.org/cgi/content/full/tpc.15.00794/DC1" rel="noopener noreferrer" target="_blank">Supplemental Figure 8</a>.</p></li>
</ul></section></section><section id="sec30"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="PMC_1"><div class="caption p"><span>Supplemental Data</span></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_28_3_746__index.html">supp_28_3_746__index.html</a><sup> (3.1KB, html) </sup>
</div></div></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>This work was supported by the National Aeronautics and Space Administration (NASA Grant NNX12AM94G) and the Samuel Roberts Noble Foundation to E.B.B. and the Department of Energy (Grants BER DE-FC02-07ER64494 and DE-FG02-91ER20021) and MSU AgBioResearch to F.B. We thank Niko Geldner, Georgia Drakakaki, Christopher Staiger, Magdalena Bezanilla, and Sheng Yang He for providing published material used in this study.</p></section><section class="ack" id="ack2"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>J.A.S. generated plant material and research reagents used in the study. J.A.S. and E.B.B. conducted forward-genetic screens and plant growth assays. J.A.S. and T.K. performed <a class="usa-link" href="#def10">co-IP</a> experiments. J.A.S., L.R., F.B., and E.B.B. performed confocal imaging experiments. T.K. conducted bioinformatics studies. F.L. conducted image analysis and quantification of confocal microscope images and contributed new computational image analysis tools. All authors contributed to writing specific sections of the manuscript.</p></section><section id="notes1"><section class="glossary" id="glossary1"><h3 class="pmc_sec_title">Glossary</h3>
<dl class="def-list">
<dt id="term1">ER</dt>
<dd id="def1"><p>endoplasmic reticulum</p></dd>
<dt id="term2">PM</dt>
<dd id="def2"><p>plasma membrane</p></dd>
<dt id="term3">TGN</dt>
<dd id="def3"><p><em>trans</em>-Golgi network</p></dd>
<dt id="term4">EE</dt>
<dd id="def4"><p>early endosome</p></dd>
<dt id="term5">LatB</dt>
<dd id="def5"><p>latrunculin B</p></dd>
<dt id="term6">DIC</dt>
<dd id="def6"><p>differential interference contrast</p></dd>
<dt id="term7">TPR</dt>
<dd id="def7"><p>tetratricopeptide repeat</p></dd>
<dt id="term8">BFA</dt>
<dd id="def8"><p>Brefeldin A</p></dd>
<dt id="term9">PVC</dt>
<dd id="def9"><p>prevacuolar compartment</p></dd>
<dt id="term10">co-IP</dt>
<dd id="def10"><p>coimmunoprecipitation</p></dd>
<dt id="term11">MS</dt>
<dd id="def11"><p>Murashige and Skoog</p></dd>
</dl></section></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<sup>[OPEN]</sup><p class="display-inline">Articles can be viewed without a subscription.</p>
</div></div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="bib1">
<cite>Almeida C.G., Yamada A., Tenza D., Louvard D., Raposo G., Coudrier E. (2011). Myosin 1b promotes the formation of post-Golgi carriers by regulating actin assembly and membrane remodelling at the trans-Golgi network. Nat. Cell Biol.
13: 779–789.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ncb2262" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21666684/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol.&amp;title=Myosin%C2%A01b%20promotes%20the%20formation%20of%20post-Golgi%20carriers%20by%20regulating%20actin%20assembly%20and%20membrane%20remodelling%20at%20the%20trans-Golgi%20network&amp;author=C.G.%20Almeida&amp;author=A.%20Yamada&amp;author=D.%20Tenza&amp;author=D.%20Louvard&amp;author=G.%20Raposo&amp;volume=13&amp;publication_year=2011&amp;pages=779-789&amp;pmid=21666684&amp;doi=10.1038/ncb2262&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib2">
<cite>Alonso J.M., et al.  (2003). Genome-wide insertional mutagenesis of <em>Arabidopsis thaliana.</em>
Science
301: 653–657.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1086391" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12893945/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide%20insertional%20mutagenesis%20of%20Arabidopsis%20thaliana.&amp;author=J.M.%20Alonso&amp;volume=301&amp;publication_year=2003&amp;pages=653-657&amp;pmid=12893945&amp;doi=10.1126/science.1086391&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib3">
<cite>Anitei M., Hoflack B. (2012). Bridging membrane and cytoskeleton dynamics in the secretory and endocytic pathways. Nat. Cell Biol.
14: 11–19.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ncb2409" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22193159/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol.&amp;title=Bridging%20membrane%20and%20cytoskeleton%20dynamics%20in%20the%20secretory%20and%20endocytic%20pathways&amp;author=M.%20Anitei&amp;author=B.%20Hoflack&amp;volume=14&amp;publication_year=2012&amp;pages=11-19&amp;pmid=22193159&amp;doi=10.1038/ncb2409&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib4">
<cite>Asaoka R., Uemura T., Ito J., Fujimoto M., Ito E., Ueda T., Nakano A. (2013). <em>Arabidopsis</em> RABA1 GTPases are involved in transport between the trans-Golgi network and the plasma membrane, and are required for salinity stress tolerance. Plant J.
73: 240–249.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.12023" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22974509/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=Arabidopsis%20RABA1%20GTPases%20are%20involved%20in%20transport%20between%20the%20trans-Golgi%20network%20and%20the%20plasma%20membrane,%20and%20are%20required%20for%20salinity%20stress%20tolerance&amp;author=R.%20Asaoka&amp;author=T.%20Uemura&amp;author=J.%20Ito&amp;author=M.%20Fujimoto&amp;author=E.%20Ito&amp;volume=73&amp;publication_year=2013&amp;pages=240-249&amp;pmid=22974509&amp;doi=10.1111/tpj.12023&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib5">
<cite>Barberon M., Geldner N. (2014). Radial transport of nutrients: the plant root as a polarized epithelium. Plant Physiol.
166: 528–537.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.114.246124" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4213085/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25136061/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Radial%20transport%20of%20nutrients:%20the%20plant%20root%20as%20a%20polarized%20epithelium&amp;author=M.%20Barberon&amp;author=N.%20Geldner&amp;volume=166&amp;publication_year=2014&amp;pages=528-537&amp;pmid=25136061&amp;doi=10.1104/pp.114.246124&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib6">
<cite>Basu D., Le J., Zakharova T., Mallery E.L., Szymanski D.B. (2008). A SPIKE1 signaling complex controls actin-dependent cell morphogenesis through the heteromeric WAVE and ARP2/3 complexes. Proc. Natl. Acad. Sci. USA
105: 4044–4049.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0710294105" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2268813/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18308939/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=A%20SPIKE1%20signaling%20complex%20controls%20actin-dependent%20cell%20morphogenesis%20through%20the%20heteromeric%20WAVE%20and%20ARP2/3%20complexes&amp;author=D.%20Basu&amp;author=J.%20Le&amp;author=T.%20Zakharova&amp;author=E.L.%20Mallery&amp;author=D.B.%20Szymanski&amp;volume=105&amp;publication_year=2008&amp;pages=4044-4049&amp;pmid=18308939&amp;doi=10.1073/pnas.0710294105&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib7">
<cite>Bhuiyan N.H., Friso G., Poliakov A., Ponnala L., van Wijk K.J. (2015). MET1 is a thylakoid-associated TPR protein involved in photosystem II supercomplex formation and repair in <em>Arabidopsis.</em>
Plant Cell
27: 262–285.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.114.132787" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4330576/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25587003/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=MET1%20is%20a%20thylakoid-associated%20TPR%20protein%20involved%20in%20photosystem%20II%20supercomplex%20formation%20and%20repair%20in%20Arabidopsis.&amp;author=N.H.%20Bhuiyan&amp;author=G.%20Friso&amp;author=A.%20Poliakov&amp;author=L.%20Ponnala&amp;author=K.J.%20van%20Wijk&amp;volume=27&amp;publication_year=2015&amp;pages=262-285&amp;pmid=25587003&amp;doi=10.1105/tpc.114.132787&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib8">
<cite>Blatch G.L., Lässle M. (1999). The tetratricopeptide repeat: a structural motif mediating protein-protein interactions. BioEssays
21: 932–939.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/(SICI)1521-1878(199911)21:11&lt;932::AID-BIES5&gt;3.0.CO;2-N" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10517866/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BioEssays&amp;title=The%20tetratricopeptide%20repeat:%20a%20structural%20motif%20mediating%20protein-protein%20interactions&amp;author=G.L.%20Blatch&amp;author=M.%20L%C3%A4ssle&amp;volume=21&amp;publication_year=1999&amp;pages=932-939&amp;pmid=10517866&amp;doi=10.1002/(SICI)1521-1878(199911)21:11&lt;932::AID-BIES5&gt;3.0.CO;2-N&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib9">
<cite>Boevink P., Oparka K., Santa Cruz S., Martin B., Betteridge A., Hawes C. (1998). Stacks on tracks: the plant Golgi apparatus traffics on an actin/ER network. Plant J.
15: 441–447.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1365-313x.1998.00208.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9750355/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=Stacks%20on%20tracks:%20the%20plant%20Golgi%20apparatus%20traffics%20on%20an%20actin/ER%20network&amp;author=P.%20Boevink&amp;author=K.%20Oparka&amp;author=S.%20Santa%20Cruz&amp;author=B.%20Martin&amp;author=A.%20Betteridge&amp;volume=15&amp;publication_year=1998&amp;pages=441-447&amp;pmid=9750355&amp;doi=10.1046/j.1365-313x.1998.00208.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib10">
<cite>Boss W.F., Im Y.J. (2012). Phosphoinositide signaling. Annu. Rev. Plant Biol.
63: 409–429.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev-arplant-042110-103840" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22404474/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Plant%20Biol.&amp;title=Phosphoinositide%20signaling&amp;author=W.F.%20Boss&amp;author=Y.J.%20Im&amp;volume=63&amp;publication_year=2012&amp;pages=409-429&amp;pmid=22404474&amp;doi=10.1146/annurev-arplant-042110-103840&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib11">
<cite>Bottanelli F., Gershlick D.C., Denecke J. (2012). Evidence for sequential action of Rab5 and Rab7 GTPases in prevacuolar organelle partitioning. Traffic
13: 338–354.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1600-0854.2011.01303.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22004564/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Evidence%20for%20sequential%20action%20of%20Rab5%20and%20Rab7%20GTPases%20in%20prevacuolar%20organelle%20partitioning&amp;author=F.%20Bottanelli&amp;author=D.C.%20Gershlick&amp;author=J.%20Denecke&amp;volume=13&amp;publication_year=2012&amp;pages=338-354&amp;pmid=22004564&amp;doi=10.1111/j.1600-0854.2011.01303.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib12">
<cite>Brandizzi F., Barlowe C. (2013). Organization of the ER-Golgi interface for membrane traffic control. Nat. Rev. Mol. Cell Biol.
14: 382–392.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nrm3588" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4064004/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23698585/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;title=Organization%20of%20the%20ER-Golgi%20interface%20for%20membrane%20traffic%20control&amp;author=F.%20Brandizzi&amp;author=C.%20Barlowe&amp;volume=14&amp;publication_year=2013&amp;pages=382-392&amp;pmid=23698585&amp;doi=10.1038/nrm3588&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib13">
<cite>Brandizzi F., Snapp E.L., Roberts A.G., Lippincott-Schwartz J., Hawes C. (2002). Membrane protein transport between the endoplasmic reticulum and the Golgi in tobacco leaves is energy dependent but cytoskeleton independent: evidence from selective photobleaching. Plant Cell
14: 1293–1309.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.001586" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC150781/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12084828/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Membrane%20protein%20transport%20between%20the%20endoplasmic%20reticulum%20and%20the%20Golgi%20in%20tobacco%20leaves%20is%20energy%20dependent%20but%20cytoskeleton%20independent:%20evidence%20from%20selective%20photobleaching&amp;author=F.%20Brandizzi&amp;author=E.L.%20Snapp&amp;author=A.G.%20Roberts&amp;author=J.%20Lippincott-Schwartz&amp;author=C.%20Hawes&amp;volume=14&amp;publication_year=2002&amp;pages=1293-1309&amp;pmid=12084828&amp;doi=10.1105/tpc.001586&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib14">
<cite>Chen Y.-N., Slabaugh E., Brandizzi F. (2008). Membrane-tethered transcription factors in <em>Arabidopsis thaliana</em>: novel regulators in stress response and development. Curr. Opin. Plant Biol.
11: 695–701.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.pbi.2008.10.005" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19019722/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Plant%20Biol.&amp;title=Membrane-tethered%20transcription%20factors%20in%20Arabidopsis%20thaliana:%20novel%20regulators%20in%20stress%20response%20and%20development&amp;author=Y.-N.%20Chen&amp;author=E.%20Slabaugh&amp;author=F.%20Brandizzi&amp;volume=11&amp;publication_year=2008&amp;pages=695-701&amp;pmid=19019722&amp;doi=10.1016/j.pbi.2008.10.005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib15">
<cite>Chow C.-M., Neto H., Foucart C., Moore I. (2008). Rab-A2 and Rab-A3 GTPases define a trans-golgi endosomal membrane domain in <em>Arabidopsis</em> that contributes substantially to the cell plate. Plant Cell
20: 101–123.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.107.052001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2254926/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18239134/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Rab-A2%20and%20Rab-A3%20GTPases%20define%20a%20trans-golgi%20endosomal%20membrane%20domain%20in%20Arabidopsis%20that%20contributes%20substantially%20to%20the%20cell%20plate&amp;author=C.-M.%20Chow&amp;author=H.%20Neto&amp;author=C.%20Foucart&amp;author=I.%20Moore&amp;volume=20&amp;publication_year=2008&amp;pages=101-123&amp;pmid=18239134&amp;doi=10.1105/tpc.107.052001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib16">
<cite>Clairfeuille T., Norwood S.J., Qi X., Teasdale R.D., Collins B.M. (2015). Structure and membrane binding properties of the endosomal tetratricopeptide repeat (TPR) domain-containing sorting nexins SNX20 and SNX21. J. Biol. Chem.
290: 14504–14517.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M115.650598" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4505518/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25882846/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Structure%20and%20membrane%20binding%20properties%20of%20the%20endosomal%20tetratricopeptide%20repeat%20(TPR)%20domain-containing%20sorting%20nexins%20SNX20%20and%20SNX21&amp;author=T.%20Clairfeuille&amp;author=S.J.%20Norwood&amp;author=X.%20Qi&amp;author=R.D.%20Teasdale&amp;author=B.M.%20Collins&amp;volume=290&amp;publication_year=2015&amp;pages=14504-14517&amp;pmid=25882846&amp;doi=10.1074/jbc.M115.650598&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib17">
<cite>Clough S.J., Bent A.F. (1998). Floral dip: a simplified method for <em>Agrobacterium</em>-mediated transformation of <em>Arabidopsis thaliana.</em>
Plant J.
16: 735–743.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1365-313x.1998.00343.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10069079/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=Floral%20dip:%20a%20simplified%20method%20for%20Agrobacterium-mediated%20transformation%20of%20Arabidopsis%20thaliana.&amp;author=S.J.%20Clough&amp;author=A.F.%20Bent&amp;volume=16&amp;publication_year=1998&amp;pages=735-743&amp;pmid=10069079&amp;doi=10.1046/j.1365-313x.1998.00343.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib18">
<cite>Contento A.L., Bassham D.C. (2012). Structure and function of endosomes in plant cells. J. Cell Sci.
125: 3511–3518.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/jcs.093559" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22935651/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci.&amp;title=Structure%20and%20function%20of%20endosomes%20in%20plant%20cells&amp;author=A.L.%20Contento&amp;author=D.C.%20Bassham&amp;volume=125&amp;publication_year=2012&amp;pages=3511-3518&amp;pmid=22935651&amp;doi=10.1242/jcs.093559&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib19">
<cite>Costes S.V., Daelemans D., Cho E.H., Dobbin Z., Pavlakis G., Lockett S. (2004). Automatic and quantitative measurement of protein-protein colocalization in live cells. Biophys. J.
86: 3993–4003.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1529/biophysj.103.038422" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1304300/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15189895/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20J.&amp;title=Automatic%20and%20quantitative%20measurement%20of%20protein-protein%20colocalization%20in%20live%20cells&amp;author=S.V.%20Costes&amp;author=D.%20Daelemans&amp;author=E.H.%20Cho&amp;author=Z.%20Dobbin&amp;author=G.%20Pavlakis&amp;volume=86&amp;publication_year=2004&amp;pages=3993-4003&amp;pmid=15189895&amp;doi=10.1529/biophysj.103.038422&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib20">
<cite>Craig R., Beavis R.C. (2004). TANDEM: matching proteins with tandem mass spectra. Bioinformatics
20: 1466–1467.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/bth092" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14976030/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=TANDEM:%20matching%20proteins%20with%20tandem%20mass%20spectra&amp;author=R.%20Craig&amp;author=R.C.%20Beavis&amp;volume=20&amp;publication_year=2004&amp;pages=1466-1467&amp;pmid=14976030&amp;doi=10.1093/bioinformatics/bth092&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib21">
<cite>Craig R., Cortens J.C., Fenyo D., Beavis R.C. (2006). Using annotated peptide mass spectrum libraries for protein identification. J. Proteome Res.
5: 1843–1849.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/pr0602085" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16889405/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Using%20annotated%20peptide%20mass%20spectrum%20libraries%20for%20protein%20identification&amp;author=R.%20Craig&amp;author=J.C.%20Cortens&amp;author=D.%20Fenyo&amp;author=R.C.%20Beavis&amp;volume=5&amp;publication_year=2006&amp;pages=1843-1849&amp;pmid=16889405&amp;doi=10.1021/pr0602085&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib22">
<cite>Cvrčková F., Oulehlová D., Žárský V. (2015). Formins: linking cytoskeleton and endomembranes in plant cells. Int. J. Mol. Sci.
16: 1–18.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/ijms16010001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4307232/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25546384/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Formins:%20linking%20cytoskeleton%20and%20endomembranes%20in%20plant%20cells&amp;author=F.%20Cvr%C4%8Dkov%C3%A1&amp;author=D.%20Oulehlov%C3%A1&amp;author=V.%20%C5%BD%C3%A1rsk%C3%BD&amp;volume=16&amp;publication_year=2015&amp;pages=1-18&amp;pmid=25546384&amp;doi=10.3390/ijms16010001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib23">
<cite>D’Andrea L.D., Regan L. (2003). TPR proteins: the versatile helix. Trends Biochem. Sci.
28: 655–662.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tibs.2003.10.007" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14659697/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem.%20Sci.&amp;title=TPR%20proteins:%20the%20versatile%20helix&amp;author=L.D.%20D%E2%80%99Andrea&amp;author=L.%20Regan&amp;volume=28&amp;publication_year=2003&amp;pages=655-662&amp;pmid=14659697&amp;doi=10.1016/j.tibs.2003.10.007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib24">
<cite>Deeks M.J., Calcutt J.R., Ingle E.K., Hawkins T.J., Chapman S., Richardson A.C., Mentlak D.A., Dixon M.R., Cartwright F., Smertenko A.P., Oparka K., Hussey P.J. (2012). A superfamily of actin-binding proteins at the actin-membrane nexus of higher plants. Curr. Biol.
22: 1595–1600.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cub.2012.06.041" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22840520/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=A%20superfamily%20of%20actin-binding%20proteins%20at%20the%20actin-membrane%20nexus%20of%20higher%20plants&amp;author=M.J.%20Deeks&amp;author=J.R.%20Calcutt&amp;author=E.K.%20Ingle&amp;author=T.J.%20Hawkins&amp;author=S.%20Chapman&amp;volume=22&amp;publication_year=2012&amp;pages=1595-1600&amp;pmid=22840520&amp;doi=10.1016/j.cub.2012.06.041&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib25">
<cite>Derivery E., Sousa C., Gautier J.J., Lombard B., Loew D., Gautreau A. (2009). The Arp2/3 activator WASH controls the fission of endosomes through a large multiprotein complex. Dev. Cell
17: 712–723.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.devcel.2009.09.010" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19922875/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Dev.%20Cell&amp;title=The%20Arp2/3%20activator%20WASH%20controls%20the%20fission%20of%20endosomes%20through%20a%20large%20multiprotein%20complex&amp;author=E.%20Derivery&amp;author=C.%20Sousa&amp;author=J.J.%20Gautier&amp;author=B.%20Lombard&amp;author=D.%20Loew&amp;volume=17&amp;publication_year=2009&amp;pages=712-723&amp;pmid=19922875&amp;doi=10.1016/j.devcel.2009.09.010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib26">
<cite>Dettmer J., Hong-Hermesdorf A., Stierhof Y.-D., Schumacher K. (2006). Vacuolar H<sup>+</sup>-ATPase activity is required for endocytic and secretory trafficking in <em>Arabidopsis.</em>
Plant Cell
18: 715–730.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.105.037978" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1383645/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16461582/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Vacuolar%20H+-ATPase%20activity%20is%20required%20for%20endocytic%20and%20secretory%20trafficking%20in%20Arabidopsis.&amp;author=J.%20Dettmer&amp;author=A.%20Hong-Hermesdorf&amp;author=Y.-D.%20Stierhof&amp;author=K.%20Schumacher&amp;volume=18&amp;publication_year=2006&amp;pages=715-730&amp;pmid=16461582&amp;doi=10.1105/tpc.105.037978&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib27">
<cite>Doyle S.M., Haeger A., Vain T., Rigal A., Viotti C., Łangowska M., Ma Q., Friml J., Raikhel N.V., Hicks G.R., Robert S. (2015). An early secretory pathway mediated by GNOM-LIKE 1 and GNOM is essential for basal polarity establishment in <em>Arabidopsis thaliana.</em>
Proc. Natl. Acad. Sci. USA
112: E806–E815.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1424856112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4343110/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25646449/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=An%20early%20secretory%20pathway%20mediated%20by%20GNOM-LIKE%201%20and%20GNOM%20is%20essential%20for%20basal%20polarity%20establishment%20in%20Arabidopsis%20thaliana.&amp;author=S.M.%20Doyle&amp;author=A.%20Haeger&amp;author=T.%20Vain&amp;author=A.%20Rigal&amp;author=C.%20Viotti&amp;volume=112&amp;publication_year=2015&amp;pages=E806-E815&amp;pmid=25646449&amp;doi=10.1073/pnas.1424856112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib28">
<cite>Dyachok J., Sparks J.A., Liao F., Wang Y.S., Blancaflor E.B. (2014). Fluorescent protein-based reporters of the actin cytoskeleton in living plant cells: fluorophore variant, actin binding domain, and promoter considerations. Cytoskeleton (Hoboken)
71: 311–327.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/cm.21174" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24659536/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cytoskeleton%20(Hoboken)&amp;title=Fluorescent%20protein-based%20reporters%20of%20the%20actin%20cytoskeleton%20in%20living%20plant%20cells:%20fluorophore%20variant,%20actin%20binding%20domain,%20and%20promoter%20considerations&amp;author=J.%20Dyachok&amp;author=J.A.%20Sparks&amp;author=F.%20Liao&amp;author=Y.S.%20Wang&amp;author=E.B.%20Blancaflor&amp;volume=71&amp;publication_year=2014&amp;pages=311-327&amp;pmid=24659536&amp;doi=10.1002/cm.21174&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib29">
<cite>Dyachok J., Yoo C.-M., Palanichelvam K., Blancaflor E.B. (2009). Sample preparation for fluorescence imaging of the cytoskeleton in fixed and living plant roots. In Cytoskeleton Methods and Protocols: Methods in Molecular Biology, 2nd ed, R.H. Gavin, ed (Totowa, NJ: Humana Press), pp. 157–169.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-1-60761-376-3_8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19768429/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Cytoskeleton%20Methods%20and%20Protocols&amp;author=J.%20Dyachok&amp;author=C.-M.%20Yoo&amp;author=K.%20Palanichelvam&amp;author=E.B.%20Blancaflor&amp;publication_year=2009&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib30">
<cite>Dyachok J., Zhu L., Liao F., He J., Huq E., Blancaflor E.B. (2011). SCAR mediates light-induced root elongation in <em>Arabidopsis</em> through photoreceptors and proteasomes. Plant Cell
23: 3610–3626.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.111.088823" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3229138/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21972261/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=SCAR%20mediates%20light-induced%20root%20elongation%20in%20Arabidopsis%20through%20photoreceptors%20and%20proteasomes&amp;author=J.%20Dyachok&amp;author=L.%20Zhu&amp;author=F.%20Liao&amp;author=J.%20He&amp;author=E.%20Huq&amp;volume=23&amp;publication_year=2011&amp;pages=3610-3626&amp;pmid=21972261&amp;doi=10.1105/tpc.111.088823&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib31">
<cite>Ebine K., Okatani Y., Uemura T., Goh T., Shoda K., Niihama M., Morita M.T., Spitzer C., Otegui M.S., Nakano A., Ueda T. (2008). A SNARE complex unique to seed plants is required for protein storage vacuole biogenesis and seed development of <em>Arabidopsis thaliana.</em>
Plant Cell
20: 3006–3021.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.107.057711" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2613668/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18984676/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=A%20SNARE%20complex%20unique%20to%20seed%20plants%20is%20required%20for%20protein%20storage%20vacuole%20biogenesis%20and%20seed%20development%20of%20Arabidopsis%20thaliana.&amp;author=K.%20Ebine&amp;author=Y.%20Okatani&amp;author=T.%20Uemura&amp;author=T.%20Goh&amp;author=K.%20Shoda&amp;volume=20&amp;publication_year=2008&amp;pages=3006-3021&amp;pmid=18984676&amp;doi=10.1105/tpc.107.057711&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib32">
<cite>Ebine K., et al.  (2011). A membrane trafficking pathway regulated by the plant-specific RAB GTPase ARA6. Nat. Cell Biol.
13: 853–859.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ncb2270" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21666683/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol.&amp;title=A%20membrane%20trafficking%20pathway%20regulated%20by%20the%20plant-specific%20RAB%20GTPase%20ARA6&amp;author=K.%20Ebine&amp;volume=13&amp;publication_year=2011&amp;pages=853-859&amp;pmid=21666683&amp;doi=10.1038/ncb2270&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib33">
<cite>Edgar R.C. (2004). MUSCLE: a multiple sequence alignment method with reduced time and space complexity. BMC Bioinformatics
5: 113.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2105-5-113" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC517706/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15318951/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=MUSCLE:%20a%20multiple%20sequence%20alignment%20method%20with%20reduced%20time%20and%20space%20complexity&amp;author=R.C.%20Edgar&amp;volume=5&amp;publication_year=2004&amp;pages=113&amp;pmid=15318951&amp;doi=10.1186/1471-2105-5-113&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib34">
<cite>Faso C., Chen Y.-N., Tamura K., Held M., Zemelis S., Marti L., Saravanan R., Hummel E., Kung L., Miller E., Hawes C., Brandizzi F. (2009). A missense mutation in the <em>Arabidopsis</em> COPII coat protein Sec24A induces the formation of clusters of the endoplasmic reticulum and Golgi apparatus. Plant Cell
21: 3655–3671.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.109.068262" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2798327/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19933202/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=A%20missense%20mutation%20in%20the%20Arabidopsis%20COPII%20coat%20protein%20Sec24A%20induces%20the%20formation%20of%20clusters%20of%20the%20endoplasmic%20reticulum%20and%20Golgi%20apparatus&amp;author=C.%20Faso&amp;author=Y.-N.%20Chen&amp;author=K.%20Tamura&amp;author=M.%20Held&amp;author=S.%20Zemelis&amp;volume=21&amp;publication_year=2009&amp;pages=3655-3671&amp;pmid=19933202&amp;doi=10.1105/tpc.109.068262&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib35">
<cite>Feraru E., Feraru M.I., Asaoka R., Paciorek T., De Rycke R., Tanaka H., Nakano A., Friml J. (2012). BEX5/RabA1b regulates trans-Golgi network-to-plasma membrane protein trafficking in Arabidopsis. Plant Cell
24: 3074–3086.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.112.098152" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3426133/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22773752/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=BEX5/RabA1b%20regulates%20trans-Golgi%20network-to-plasma%20membrane%20protein%20trafficking%20in%20Arabidopsis&amp;author=E.%20Feraru&amp;author=M.I.%20Feraru&amp;author=R.%20Asaoka&amp;author=T.%20Paciorek&amp;author=R.%20De%20Rycke&amp;volume=24&amp;publication_year=2012&amp;pages=3074-3086&amp;pmid=22773752&amp;doi=10.1105/tpc.112.098152&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib36">
<cite>Fujimoto M., Ueda T. (2012). Conserved and plant-unique mechanisms regulating plant post-Golgi traffic. Front. Plant Sci.
3: 197.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2012.00197" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3428585/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22973281/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=Conserved%20and%20plant-unique%20mechanisms%20regulating%20plant%20post-Golgi%20traffic&amp;author=M.%20Fujimoto&amp;author=T.%20Ueda&amp;volume=3&amp;publication_year=2012&amp;pages=197&amp;pmid=22973281&amp;doi=10.3389/fpls.2012.00197&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib37">
<cite>Garcia de la Garma J., Fernandez-Garcia N., Bardisi E., Pallol B., Asensio-Rubio J.S., Bru R., Olmos E. (2015). New insights into plant salt acclimation: the roles of vesicle trafficking and reactive oxygen species signalling in mitochondria and the endomembrane system. New Phytol.
205: 216–239.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/nph.12997" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25187269/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=New%20insights%20into%20plant%20salt%20acclimation:%20the%20roles%20of%20vesicle%20trafficking%20and%20reactive%20oxygen%20species%20signalling%20in%20mitochondria%20and%20the%20endomembrane%20system&amp;author=J.%20Garcia%20de%20la%20Garma&amp;author=N.%20Fernandez-Garcia&amp;author=E.%20Bardisi&amp;author=B.%20Pallol&amp;author=J.S.%20Asensio-Rubio&amp;volume=205&amp;publication_year=2015&amp;pages=216-239&amp;pmid=25187269&amp;doi=10.1111/nph.12997&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib38">
<cite>Geer L.Y., Markey S.P., Kowalak J.A., Wagner L., Xu M., Maynard D.M., Yang X., Shi W., Bryant S.H. (2004). Open mass spectrometry search algorithm. J. Proteome Res.
3: 958–964.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/pr0499491" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15473683/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Open%20mass%20spectrometry%20search%20algorithm&amp;author=L.Y.%20Geer&amp;author=S.P.%20Markey&amp;author=J.A.%20Kowalak&amp;author=L.%20Wagner&amp;author=M.%20Xu&amp;volume=3&amp;publication_year=2004&amp;pages=958-964&amp;pmid=15473683&amp;doi=10.1021/pr0499491&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib39">
<cite>Geldner N., Dénervaud-Tendon V., Hyman D.L., Mayer U., Stierhof Y.D., Chory J. (2009). Rapid, combinatorial analysis of membrane compartments in intact plants with a multicolor marker set. Plant J.
59: 169–178.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2009.03851.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4854200/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19309456/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=Rapid,%20combinatorial%20analysis%20of%20membrane%20compartments%20in%20intact%20plants%20with%20a%20multicolor%20marker%20set&amp;author=N.%20Geldner&amp;author=V.%20D%C3%A9nervaud-Tendon&amp;author=D.L.%20Hyman&amp;author=U.%20Mayer&amp;author=Y.D.%20Stierhof&amp;volume=59&amp;publication_year=2009&amp;pages=169-178&amp;pmid=19309456&amp;doi=10.1111/j.1365-313X.2009.03851.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib40">
<cite>Geldner N., Friml J., Stierhof Y.-D., Jürgens G., Palme K. (2001). Auxin transport inhibitors block PIN1 cycling and vesicle trafficking. Nature
413: 425–428.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/35096571" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11574889/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Auxin%20transport%20inhibitors%20block%20PIN1%20cycling%20and%20vesicle%20trafficking&amp;author=N.%20Geldner&amp;author=J.%20Friml&amp;author=Y.-D.%20Stierhof&amp;author=G.%20J%C3%BCrgens&amp;author=K.%20Palme&amp;volume=413&amp;publication_year=2001&amp;pages=425-428&amp;pmid=11574889&amp;doi=10.1038/35096571&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib41">
<cite>Geldner N., Hyman D.L., Wang X., Schumacher K., Chory J. (2007). Endosomal signaling of plant steroid receptor kinase BRI1. Genes Dev.
21: 1598–1602.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gad.1561307" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1899468/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17578906/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Endosomal%20signaling%20of%20plant%20steroid%20receptor%20kinase%20BRI1&amp;author=N.%20Geldner&amp;author=D.L.%20Hyman&amp;author=X.%20Wang&amp;author=K.%20Schumacher&amp;author=J.%20Chory&amp;volume=21&amp;publication_year=2007&amp;pages=1598-1602&amp;pmid=17578906&amp;doi=10.1101/gad.1561307&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib42">
<cite>Gendre D., Jonsson K., Boutté Y., Bhalerao R.P. (2015). Journey to the cell surface--the central role of the trans-Golgi network in plants. Protoplasma
252: 385–398.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00709-014-0693-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25187082/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Protoplasma&amp;title=Journey%20to%20the%20cell%20surface--the%20central%20role%20of%20the%20trans-Golgi%20network%20in%20plants&amp;author=D.%20Gendre&amp;author=K.%20Jonsson&amp;author=Y.%20Boutt%C3%A9&amp;author=R.P.%20Bhalerao&amp;volume=252&amp;publication_year=2015&amp;pages=385-398&amp;pmid=25187082&amp;doi=10.1007/s00709-014-0693-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib43">
<cite>Gendre D., McFarlane H.E., Johnson E., Mouille G., Sjödin A., Oh J., Levesque-Tremblay G., Watanabe Y., Samuels L., Bhalerao R.P. (2013). Trans-Golgi network localized ECHIDNA/Ypt interacting protein complex is required for the secretion of cell wall polysaccharides in <em>Arabidopsis.</em>
Plant Cell
25: 2633–2646.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.113.112482" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3753388/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23832588/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Trans-Golgi%20network%20localized%20ECHIDNA/Ypt%20interacting%20protein%20complex%20is%20required%20for%20the%20secretion%20of%20cell%20wall%20polysaccharides%20in%20Arabidopsis.&amp;author=D.%20Gendre&amp;author=H.E.%20McFarlane&amp;author=E.%20Johnson&amp;author=G.%20Mouille&amp;author=A.%20Sj%C3%B6din&amp;volume=25&amp;publication_year=2013&amp;pages=2633-2646&amp;pmid=23832588&amp;doi=10.1105/tpc.113.112482&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib44">
<cite>Gendre D., Oh J., Boutté Y., Best J.G., Samuels L., Nilsson R., Uemura T., Marchant A., Bennett M.J., Grebe M., Bhalerao R.P. (2011). Conserved <em>Arabidopsis</em> ECHIDNA protein mediates trans-Golgi-network trafficking and cell elongation. Proc. Natl. Acad. Sci. USA
108: 8048–8053.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1018371108" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3093476/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21512130/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Conserved%20Arabidopsis%20ECHIDNA%20protein%20mediates%20trans-Golgi-network%20trafficking%20and%20cell%20elongation&amp;author=D.%20Gendre&amp;author=J.%20Oh&amp;author=Y.%20Boutt%C3%A9&amp;author=J.G.%20Best&amp;author=L.%20Samuels&amp;volume=108&amp;publication_year=2011&amp;pages=8048-8053&amp;pmid=21512130&amp;doi=10.1073/pnas.1018371108&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib45">
<cite>Grebe M., Xu J., Möbius W., Ueda T., Nakano A., Geuze H.J., Rook M.B., Scheres B. (2003). <em>Arabidopsis</em> sterol endocytosis involves actin-mediated trafficking via ARA6-positive early endosomes. Curr. Biol.
13: 1378–1387.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/s0960-9822(03)00538-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12932321/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=Arabidopsis%20sterol%20endocytosis%20involves%20actin-mediated%20trafficking%20via%20ARA6-positive%20early%20endosomes&amp;author=M.%20Grebe&amp;author=J.%20Xu&amp;author=W.%20M%C3%B6bius&amp;author=T.%20Ueda&amp;author=A.%20Nakano&amp;volume=13&amp;publication_year=2003&amp;pages=1378-1387&amp;pmid=12932321&amp;doi=10.1016/s0960-9822(03)00538-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib46">
<cite>Guet D., Mandal K., Pinot M., Hoffmann J., Abidine Y., Sigaut W., Bardin S., Schauer K., Goud B., Manneville J.-B. (2014). Mechanical role of actin dynamics in the rheology of the Golgi complex and in Golgi-associated trafficking events. Curr. Biol.
24: 1700–1711.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cub.2014.06.048" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25042587/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=Mechanical%20role%20of%20actin%20dynamics%20in%20the%20rheology%20of%20the%20Golgi%20complex%20and%20in%20Golgi-associated%20trafficking%20events&amp;author=D.%20Guet&amp;author=K.%20Mandal&amp;author=M.%20Pinot&amp;author=J.%20Hoffmann&amp;author=Y.%20Abidine&amp;volume=24&amp;publication_year=2014&amp;pages=1700-1711&amp;pmid=25042587&amp;doi=10.1016/j.cub.2014.06.048&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib47">
<cite>Hachez C., Laloux T., Reinhardt H., Cavez D., Degand H., Grefen C., De Rycke R., Inzé D., Blatt M.R., Russinova E., Chaumont F. (2014). <em>Arabidopsis</em> SNAREs SYP61 and SYP121 coordinate the trafficking of plasma membrane aquaporin PIP2;7 to modulate the cell membrane water permeability. Plant Cell
26: 3132–3147.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.114.127159" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4145137/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25082856/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Arabidopsis%20SNAREs%20SYP61%20and%20SYP121%20coordinate%20the%20trafficking%20of%20plasma%20membrane%20aquaporin%20PIP2;7%20to%20modulate%20the%20cell%20membrane%20water%20permeability&amp;author=C.%20Hachez&amp;author=T.%20Laloux&amp;author=H.%20Reinhardt&amp;author=D.%20Cavez&amp;author=H.%20Degand&amp;volume=26&amp;publication_year=2014&amp;pages=3132-3147&amp;pmid=25082856&amp;doi=10.1105/tpc.114.127159&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib48">
<cite>Henty-Ridilla J.L., Li J., Day B., Staiger C.J. (2014). ACTIN DEPOLYMERIZING FACTOR4 regulates actin dynamics during innate immune signaling in <em>Arabidopsis.</em>
Plant Cell
26: 340–352.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.113.122499" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3963580/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24464292/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=ACTIN%20DEPOLYMERIZING%20FACTOR4%20regulates%20actin%20dynamics%20during%20innate%20immune%20signaling%20in%20Arabidopsis.&amp;author=J.L.%20Henty-Ridilla&amp;author=J.%20Li&amp;author=B.%20Day&amp;author=C.J.%20Staiger&amp;volume=26&amp;publication_year=2014&amp;pages=340-352&amp;pmid=24464292&amp;doi=10.1105/tpc.113.122499&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib49">
<cite>Higaki T., Kutsuna N., Sano T., Kondo N., Hasezawa S. (2010). Quantification and cluster analysis of actin cytoskeletal structures in plant cells: role of actin bundling in stomatal movement during diurnal cycles in <em>Arabidopsis</em> guard cells. Plant J.
61: 156–165.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313x.2009.04032.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20092030/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=Quantification%20and%20cluster%20analysis%20of%20actin%20cytoskeletal%20structures%20in%20plant%20cells:%20role%20of%20actin%20bundling%20in%20stomatal%20movement%20during%20diurnal%20cycles%20in%20Arabidopsis%20guard%20cells&amp;author=T.%20Higaki&amp;author=N.%20Kutsuna&amp;author=T.%20Sano&amp;author=N.%20Kondo&amp;author=S.%20Hasezawa&amp;volume=61&amp;publication_year=2010&amp;pages=156-165&amp;pmid=20092030&amp;doi=10.1111/j.1365-313x.2009.04032.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib50">
<cite>Hu Z., Xu F., Guan L., Qian P., Liu Y., Zhang H., Huang Y., Hou S. (2014). The tetratricopeptide repeat-containing protein slow green1 is required for chloroplast development in <em>Arabidopsis.</em>
J. Exp. Bot.
65: 1111–1123.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/ert463" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3935568/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24420572/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=The%20tetratricopeptide%20repeat-containing%20protein%20slow%20green1%20is%20required%20for%20chloroplast%20development%20in%20Arabidopsis.&amp;author=Z.%20Hu&amp;author=F.%20Xu&amp;author=L.%20Guan&amp;author=P.%20Qian&amp;author=Y.%20Liu&amp;volume=65&amp;publication_year=2014&amp;pages=1111-1123&amp;pmid=24420572&amp;doi=10.1093/jxb/ert463&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib51">
<cite>Hunter P.R., Craddock C.P., Di Benedetto S., Roberts L.M., Frigerio L. (2007). Fluorescent reporter proteins for the tonoplast and the vacuolar lumen identify a single vacuolar compartment in <em>Arabidopsis</em> cells. Plant Physiol.
145: 1371–1382.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.107.103945" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2151705/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17905861/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Fluorescent%20reporter%20proteins%20for%20the%20tonoplast%20and%20the%20vacuolar%20lumen%20identify%20a%20single%20vacuolar%20compartment%20in%20Arabidopsis%20cells&amp;author=P.R.%20Hunter&amp;author=C.P.%20Craddock&amp;author=S.%20Di%20Benedetto&amp;author=L.M.%20Roberts&amp;author=L.%20Frigerio&amp;volume=145&amp;publication_year=2007&amp;pages=1371-1382&amp;pmid=17905861&amp;doi=10.1104/pp.107.103945&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib52">
<cite>Jacobsen S.E., Binkowski K.A., Olszewski N.E. (1996). SPINDLY, a tetratricopeptide repeat protein involved in gibberellin signal transduction in <em>Arabidopsis.</em>
Proc. Natl. Acad. Sci. USA
93: 9292–9296.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.93.17.9292" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC38635/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8799194/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=SPINDLY,%20a%20tetratricopeptide%20repeat%20protein%20involved%20in%20gibberellin%20signal%20transduction%20in%20Arabidopsis.&amp;author=S.E.%20Jacobsen&amp;author=K.A.%20Binkowski&amp;author=N.E.%20Olszewski&amp;volume=93&amp;publication_year=1996&amp;pages=9292-9296&amp;pmid=8799194&amp;doi=10.1073/pnas.93.17.9292&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib53">
<cite>Jia H., Li J., Zhu J., Fan T., Qian D., Zhou Y., Wang J., Ren H., Xiang Y., An L. (2013). <em>Arabidopsis</em> CROLIN1, a novel plant actin-binding protein, functions in cross-linking and stabilizing actin filaments. J. Biol. Chem.
288: 32277–32288.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.M113.483594" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3820865/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24072702/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Arabidopsis%20CROLIN1,%20a%20novel%20plant%20actin-binding%20protein,%20functions%20in%20cross-linking%20and%20stabilizing%20actin%20filaments&amp;author=H.%20Jia&amp;author=J.%20Li&amp;author=J.%20Zhu&amp;author=T.%20Fan&amp;author=D.%20Qian&amp;volume=288&amp;publication_year=2013&amp;pages=32277-32288&amp;pmid=24072702&amp;doi=10.1074/jbc.M113.483594&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib54">
<cite>Jimenez-Lopez J.C., Wang X., Kotchoni S.O., Huang S., Szymanski D.B., Staiger C.J. (2014). Heterodimeric capping protein from <em>Arabidopsis</em> is a membrane-associated, actin-binding protein. Plant Physiol.
166: 1312–1328.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.114.242487" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4226361/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25201878/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Heterodimeric%20capping%20protein%20from%20Arabidopsis%20is%20a%20membrane-associated,%20actin-binding%20protein&amp;author=J.C.%20Jimenez-Lopez&amp;author=X.%20Wang&amp;author=S.O.%20Kotchoni&amp;author=S.%20Huang&amp;author=D.B.%20Szymanski&amp;volume=166&amp;publication_year=2014&amp;pages=1312-1328&amp;pmid=25201878&amp;doi=10.1104/pp.114.242487&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib55">
<cite>Kandasamy M.K., McKinney E.C., Meagher R.B. (2009). A single vegetative actin isovariant overexpressed under the control of multiple regulatory sequences is sufficient for normal <em>Arabidopsis</em> development. Plant Cell
21: 701–718.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.108.061960" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2671709/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19304937/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=A%20single%20vegetative%20actin%20isovariant%20overexpressed%20under%20the%20control%20of%20multiple%20regulatory%20sequences%20is%20sufficient%20for%20normal%20Arabidopsis%20development&amp;author=M.K.%20Kandasamy&amp;author=E.C.%20McKinney&amp;author=R.B.%20Meagher&amp;volume=21&amp;publication_year=2009&amp;pages=701-718&amp;pmid=19304937&amp;doi=10.1105/tpc.108.061960&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib56">
<cite>Kang B.H., Nielsen E., Preuss M.L., Mastronarde D., Staehelin L.A. (2011). Electron tomography of RabA4b- and PI-4Kβ1-labeled trans Golgi network compartments in <em>Arabidopsis.</em>
Traffic
12: 313–329.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1600-0854.2010.01146.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21134079/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Electron%20tomography%20of%20RabA4b-%20and%20PI-4K%CE%B21-labeled%20trans%20Golgi%20network%20compartments%20in%20Arabidopsis.&amp;author=B.H.%20Kang&amp;author=E.%20Nielsen&amp;author=M.L.%20Preuss&amp;author=D.%20Mastronarde&amp;author=L.A.%20Staehelin&amp;volume=12&amp;publication_year=2011&amp;pages=313-329&amp;pmid=21134079&amp;doi=10.1111/j.1600-0854.2010.01146.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib57">
<cite>Karpenahalli M.R., Lupas A.N., Söding J. (2007). TPRpred: a tool for prediction of TPR-, PPR- and SEL1-like repeats from protein sequences. BMC Bioinformatics
8: 2.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2105-8-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1774580/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17199898/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=TPRpred:%20a%20tool%20for%20prediction%20of%20TPR-,%20PPR-%20and%20SEL1-like%20repeats%20from%20protein%20sequences&amp;author=M.R.%20Karpenahalli&amp;author=A.N.%20Lupas&amp;author=J.%20S%C3%B6ding&amp;volume=8&amp;publication_year=2007&amp;pages=2&amp;pmid=17199898&amp;doi=10.1186/1471-2105-8-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib58">
<cite>Kim H., Park M., Kim S.J., Hwang I. (2005). Actin filaments play a critical role in vacuolar trafficking at the Golgi complex in plant cells. Plant Cell
17: 888–902.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.104.028829" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1069706/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15722471/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Actin%20filaments%20play%20a%20critical%20role%20in%20vacuolar%20trafficking%20at%20the%20Golgi%20complex%20in%20plant%20cells&amp;author=H.%20Kim&amp;author=M.%20Park&amp;author=S.J.%20Kim&amp;author=I.%20Hwang&amp;volume=17&amp;publication_year=2005&amp;pages=888-902&amp;pmid=15722471&amp;doi=10.1105/tpc.104.028829&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib59">
<cite>Kim S.J., Brandizzi F. (2014). The plant secretory pathway: an essential factory for building the plant cell wall. Plant Cell Physiol.
55: 687–693.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pct197" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24401957/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol.&amp;title=The%20plant%20secretory%20pathway:%20an%20essential%20factory%20for%20building%20the%20plant%20cell%20wall&amp;author=S.J.%20Kim&amp;author=F.%20Brandizzi&amp;volume=55&amp;publication_year=2014&amp;pages=687-693&amp;pmid=24401957&amp;doi=10.1093/pcp/pct197&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib60">
<cite>Kim S.J., Held M.A., Zemelis S., Wilkerson C., Brandizzi F. (2015). CGR2 and CGR3 have critical overlapping roles in pectin methylesterification and plant growth in <em>Arabidopsis thaliana.</em>
Plant J.
82: 208–220.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.12802" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25704846/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=CGR2%20and%20CGR3%20have%20critical%20overlapping%20roles%20in%20pectin%20methylesterification%20and%20plant%20growth%20in%20Arabidopsis%20thaliana.&amp;author=S.J.%20Kim&amp;author=M.A.%20Held&amp;author=S.%20Zemelis&amp;author=C.%20Wilkerson&amp;author=F.%20Brandizzi&amp;volume=82&amp;publication_year=2015&amp;pages=208-220&amp;pmid=25704846&amp;doi=10.1111/tpj.12802&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib61">
<cite>Kleine-Vehn J., Ding Z., Jones A.R., Tasaka M., Morita M.T., Friml J. (2010). Gravity-induced PIN transcytosis for polarization of auxin fluxes in gravity-sensing root cells. Proc. Natl. Acad. Sci. USA
107: 22344–22349.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1013145107" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3009804/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21135243/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Gravity-induced%20PIN%20transcytosis%20for%20polarization%20of%20auxin%20fluxes%20in%20gravity-sensing%20root%20cells&amp;author=J.%20Kleine-Vehn&amp;author=Z.%20Ding&amp;author=A.R.%20Jones&amp;author=M.%20Tasaka&amp;author=M.T.%20Morita&amp;volume=107&amp;publication_year=2010&amp;pages=22344-22349&amp;pmid=21135243&amp;doi=10.1073/pnas.1013145107&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib62">
<cite>Kleine-Vehn J., Leitner J., Zwiewka M., Sauer M., Abas L., Luschnig C., Friml J. (2008). Differential degradation of PIN2 auxin efflux carrier by retromer-dependent vacuolar targeting. Proc. Natl. Acad. Sci. USA
105: 17812–17817.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0808073105" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2584678/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19004783/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Differential%20degradation%20of%20PIN2%20auxin%20efflux%20carrier%20by%20retromer-dependent%20vacuolar%20targeting&amp;author=J.%20Kleine-Vehn&amp;author=J.%20Leitner&amp;author=M.%20Zwiewka&amp;author=M.%20Sauer&amp;author=L.%20Abas&amp;volume=105&amp;publication_year=2008&amp;pages=17812-17817&amp;pmid=19004783&amp;doi=10.1073/pnas.0808073105&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib63">
<cite>Kusano H., Testerink C., Vermeer J.E., Tsuge T., Shimada H., Oka A., Munnik T., Aoyama T. (2008). The <em>Arabidopsis</em> phosphatidylinositol phosphate 5-kinase PIP5K3 is a key regulator of root hair tip growth. Plant Cell
20: 367–380.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.107.056119" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2276443/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18281506/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20Arabidopsis%20phosphatidylinositol%20phosphate%205-kinase%20PIP5K3%20is%20a%20key%20regulator%20of%20root%20hair%20tip%20growth&amp;author=H.%20Kusano&amp;author=C.%20Testerink&amp;author=J.E.%20Vermeer&amp;author=T.%20Tsuge&amp;author=H.%20Shimada&amp;volume=20&amp;publication_year=2008&amp;pages=367-380&amp;pmid=18281506&amp;doi=10.1105/tpc.107.056119&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib64">
<cite>Le S.Q., Gascuel O. (2008). An improved general amino acid replacement matrix. Mol. Biol. Evol.
25: 1307–1320.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/msn067" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18367465/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=An%20improved%20general%20amino%20acid%20replacement%20matrix&amp;author=S.Q.%20Le&amp;author=O.%20Gascuel&amp;volume=25&amp;publication_year=2008&amp;pages=1307-1320&amp;pmid=18367465&amp;doi=10.1093/molbev/msn067&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib65">
<cite>Li J., Henty-Ridilla J.L., Huang S., Wang X., Blanchoin L., Staiger C.J. (2012). Capping protein modulates the dynamic behavior of actin filaments in response to phosphatidic acid in <em>Arabidopsis.</em>
Plant Cell
24: 3742–3754.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.112.103945" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3480299/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22960908/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Capping%20protein%20modulates%20the%20dynamic%20behavior%20of%20actin%20filaments%20in%20response%20to%20phosphatidic%20acid%20in%20Arabidopsis.&amp;author=J.%20Li&amp;author=J.L.%20Henty-Ridilla&amp;author=S.%20Huang&amp;author=X.%20Wang&amp;author=L.%20Blanchoin&amp;volume=24&amp;publication_year=2012&amp;pages=3742-3754&amp;pmid=22960908&amp;doi=10.1105/tpc.112.103945&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib66">
<cite>Li J., Henty-Ridilla J.L., Staiger B.H., Day B., Staiger C.J. (2015). Capping protein integrates multiple MAMP signalling pathways to modulate actin dynamics during plant innate immunity. Nat. Commun.
6: 7206.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ncomms8206" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4458898/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26018794/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Capping%20protein%20integrates%20multiple%20MAMP%20signalling%20pathways%20to%20modulate%20actin%20dynamics%20during%20plant%20innate%20immunity&amp;author=J.%20Li&amp;author=J.L.%20Henty-Ridilla&amp;author=B.H.%20Staiger&amp;author=B.%20Day&amp;author=C.J.%20Staiger&amp;volume=6&amp;publication_year=2015&amp;pages=7206&amp;pmid=26018794&amp;doi=10.1038/ncomms8206&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib67">
<cite>Li J., Staiger B.H., Henty-Ridilla J.L., Abu-Abied M., Sadot E., Blanchoin L., Staiger C.J. (2014). The availability of filament ends modulates actin stochastic dynamics in live plant cells. Mol. Biol. Cell
25: 1263–1275.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.E13-07-0378" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3982992/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24523291/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=The%20availability%20of%20filament%20ends%20modulates%20actin%20stochastic%20dynamics%20in%20live%20plant%20cells&amp;author=J.%20Li&amp;author=B.H.%20Staiger&amp;author=J.L.%20Henty-Ridilla&amp;author=M.%20Abu-Abied&amp;author=E.%20Sadot&amp;volume=25&amp;publication_year=2014&amp;pages=1263-1275&amp;pmid=24523291&amp;doi=10.1091/mbc.E13-07-0378&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib68">
<cite>Lozano-Durán R., Bourdais G., He S.Y., Robatzek S. (2014). The bacterial effector HopM1 suppresses PAMP-triggered oxidative burst and stomatal immunity. New Phytol.
202: 259–269.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/nph.12651" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24372399/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=The%20bacterial%20effector%20HopM1%20suppresses%20PAMP-triggered%20oxidative%20burst%20and%20stomatal%20immunity&amp;author=R.%20Lozano-Dur%C3%A1n&amp;author=G.%20Bourdais&amp;author=S.Y.%20He&amp;author=S.%20Robatzek&amp;volume=202&amp;publication_year=2014&amp;pages=259-269&amp;pmid=24372399&amp;doi=10.1111/nph.12651&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib69">
<cite>Lukowitz W., Gillmor C.S., Scheible W.-R. (2000). Positional cloning in <em>Arabidopsis</em>. Why it feels good to have a genome initiative working for you. Plant Physiol.
123: 795–805.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.123.3.795" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1539260/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10889228/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Positional%20cloning%20in%20Arabidopsis.%20Why%20it%20feels%20good%20to%20have%20a%20genome%20initiative%20working%20for%20you&amp;author=W.%20Lukowitz&amp;author=C.S.%20Gillmor&amp;author=W.-R.%20Scheible&amp;volume=123&amp;publication_year=2000&amp;pages=795-805&amp;pmid=10889228&amp;doi=10.1104/pp.123.3.795&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib70">
<cite>Manders E.M.M., Verbeek F.J., Aten J.A. (1993). Measurement of co-localization of objects in dual-colour confocal images. J. Microsc.
169: 375–382.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-2818.1993.tb03313.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33930978/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Microsc.&amp;title=Measurement%20of%20co-localization%20of%20objects%20in%20dual-colour%20confocal%20images&amp;author=E.M.M.%20Manders&amp;author=F.J.%20Verbeek&amp;author=J.A.%20Aten&amp;volume=169&amp;publication_year=1993&amp;pages=375-382&amp;pmid=33930978&amp;doi=10.1111/j.1365-2818.1993.tb03313.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib71">
<cite>Marchler-Bauer A., et al.  (2015). CDD: NCBI’s conserved domain database. Nucleic Acids Res.
43: D222–D226.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gku1221" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4383992/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25414356/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=CDD:%20NCBI%E2%80%99s%20conserved%20domain%20database&amp;author=A.%20Marchler-Bauer&amp;volume=43&amp;publication_year=2015&amp;pages=D222-D226&amp;pmid=25414356&amp;doi=10.1093/nar/gku1221&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib72">
<cite>Mathur J., Mathur N., Kernebeck B., Hülskamp M. (2003). Mutations in actin-related proteins 2 and 3 affect cell shape development in <em>Arabidopsis.</em>
Plant Cell
15: 1632–1645.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.011676" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC165406/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12837952/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Mutations%20in%20actin-related%20proteins%202%20and%203%20affect%20cell%20shape%20development%20in%20Arabidopsis.&amp;author=J.%20Mathur&amp;author=N.%20Mathur&amp;author=B.%20Kernebeck&amp;author=M.%20H%C3%BClskamp&amp;volume=15&amp;publication_year=2003&amp;pages=1632-1645&amp;pmid=12837952&amp;doi=10.1105/tpc.011676&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib73">
<cite>Müller A., Guan C., Gälweiler L., Tänzler P., Huijser P., Marchant A., Parry G., Bennett M., Wisman E., Palme K. (1998). AtPIN2 defines a locus of <em>Arabidopsis</em> for root gravitropism control. EMBO J.
17: 6903–6911.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/emboj/17.23.6903" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1171038/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9843496/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&amp;title=AtPIN2%20defines%20a%20locus%20of%20Arabidopsis%20for%20root%20gravitropism%20control&amp;author=A.%20M%C3%BCller&amp;author=C.%20Guan&amp;author=L.%20G%C3%A4lweiler&amp;author=P.%20T%C3%A4nzler&amp;author=P.%20Huijser&amp;volume=17&amp;publication_year=1998&amp;pages=6903-6911&amp;pmid=9843496&amp;doi=10.1093/emboj/17.23.6903&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib74">
<cite>Nagawa S., Xu T., Lin D., Dhonukshe P., Zhang X., Friml J., Scheres B., Fu Y., Yang Z. (2012). ROP GTPase-dependent actin microfilaments promote PIN1 polarization by localized inhibition of clathrin-dependent endocytosis. PLoS Biol.
10: e1001299.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pbio.1001299" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3317906/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22509133/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&amp;title=ROP%20GTPase-dependent%20actin%20microfilaments%20promote%20PIN1%20polarization%20by%20localized%20inhibition%20of%20clathrin-dependent%20endocytosis&amp;author=S.%20Nagawa&amp;author=T.%20Xu&amp;author=D.%20Lin&amp;author=P.%20Dhonukshe&amp;author=X.%20Zhang&amp;volume=10&amp;publication_year=2012&amp;pages=e1001299&amp;pmid=22509133&amp;doi=10.1371/journal.pbio.1001299&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib75">
<cite>Naoi K., Hashimoto T. (2004). A semidominant mutation in an <em>Arabidopsis</em> mitogen-activated protein kinase phosphatase-like gene compromises cortical microtubule organization. Plant Cell
16: 1841–1853.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.021865" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC514165/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15208393/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=A%20semidominant%20mutation%20in%20an%20Arabidopsis%20mitogen-activated%20protein%20kinase%20phosphatase-like%20gene%20compromises%20cortical%20microtubule%20organization&amp;author=K.%20Naoi&amp;author=T.%20Hashimoto&amp;volume=16&amp;publication_year=2004&amp;pages=1841-1853&amp;pmid=15208393&amp;doi=10.1105/tpc.021865&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib76">
<cite>Naramoto S., Otegui M.S., Kutsuna N., de Rycke R., Dainobu T., Karampelias M., Fujimoto M., Feraru E., Miki D., Fukuda H., Nakano A., Friml J. (2014). Insights into the localization and function of the membrane trafficking regulator GNOM ARF-GEF at the Golgi apparatus in <em>Arabidopsis.</em>
Plant Cell
26: 3062–3076.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.114.125880" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4145132/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25012191/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Insights%20into%20the%20localization%20and%20function%20of%20the%20membrane%20trafficking%20regulator%20GNOM%20ARF-GEF%20at%20the%20Golgi%20apparatus%20in%20Arabidopsis.&amp;author=S.%20Naramoto&amp;author=M.S.%20Otegui&amp;author=N.%20Kutsuna&amp;author=R.%20de%20Rycke&amp;author=T.%20Dainobu&amp;volume=26&amp;publication_year=2014&amp;pages=3062-3076&amp;pmid=25012191&amp;doi=10.1105/tpc.114.125880&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib77">
<cite>Nebenführ A., Gallagher L.A., Dunahay T.G., Frohlick J.A., Mazurkiewicz A.M., Meehl J.B., Staehelin L.A. (1999). Stop-and-go movements of plant Golgi stacks are mediated by the acto-myosin system. Plant Physiol.
121: 1127–1142.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.121.4.1127" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC59480/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10594100/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Stop-and-go%20movements%20of%20plant%20Golgi%20stacks%20are%20mediated%20by%20the%20acto-myosin%20system&amp;author=A.%20Nebenf%C3%BChr&amp;author=L.A.%20Gallagher&amp;author=T.G.%20Dunahay&amp;author=J.A.%20Frohlick&amp;author=A.M.%20Mazurkiewicz&amp;volume=121&amp;publication_year=1999&amp;pages=1127-1142&amp;pmid=10594100&amp;doi=10.1104/pp.121.4.1127&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib78">
<cite>Nelson B.K., Cai X., Nebenführ A. (2007). A multicolored set of in vivo organelle markers for co-localization studies in <em>Arabidopsis</em> and other plants. Plant J.
51: 1126–1136.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2007.03212.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17666025/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=A%20multicolored%20set%20of%20in%20vivo%20organelle%20markers%20for%20co-localization%20studies%20in%20Arabidopsis%20and%20other%20plants&amp;author=B.K.%20Nelson&amp;author=X.%20Cai&amp;author=A.%20Nebenf%C3%BChr&amp;volume=51&amp;publication_year=2007&amp;pages=1126-1136&amp;pmid=17666025&amp;doi=10.1111/j.1365-313X.2007.03212.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib79">
<cite>Nomura K., Debroy S., Lee Y.H., Pumplin N., Jones J., He S.Y. (2006). A bacterial virulence protein suppresses host innate immunity to cause plant disease. Science
313: 220–223.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1129523" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16840699/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A%20bacterial%20virulence%20protein%20suppresses%20host%20innate%20immunity%20to%20cause%20plant%20disease&amp;author=K.%20Nomura&amp;author=S.%20Debroy&amp;author=Y.H.%20Lee&amp;author=N.%20Pumplin&amp;author=J.%20Jones&amp;volume=313&amp;publication_year=2006&amp;pages=220-223&amp;pmid=16840699&amp;doi=10.1126/science.1129523&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib80">
<cite>Nomura K., Mecey C., Lee Y.-N., Imboden L.A., Chang J.H., He S.Y. (2011). Effector-triggered immunity blocks pathogen degradation of an immunity-associated vesicle traffic regulator in <em>Arabidopsis.</em>
Proc. Natl. Acad. Sci. USA
108: 10774–10779.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1103338108" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3127868/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21670267/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Effector-triggered%20immunity%20blocks%20pathogen%20degradation%20of%20an%20immunity-associated%20vesicle%20traffic%20regulator%20in%20Arabidopsis.&amp;author=K.%20Nomura&amp;author=C.%20Mecey&amp;author=Y.-N.%20Lee&amp;author=L.A.%20Imboden&amp;author=J.H.%20Chang&amp;volume=108&amp;publication_year=2011&amp;pages=10774-10779&amp;pmid=21670267&amp;doi=10.1073/pnas.1103338108&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib81">
<cite>Pahari S., Cormark R.D., Blackshaw M.T., Liu C., Erickson J.L., Schultz E.A. (2014). Arabidopsis UNHINGED encodes a VPS51 homolog and reveals a role for the GARP complex in leaf shape and vein patterning. Development
141: 1894–1905.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/dev.099333" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24757006/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Arabidopsis%20UNHINGED%20encodes%20a%20VPS51%20homolog%20and%20reveals%20a%20role%20for%20the%20GARP%20complex%20in%20leaf%20shape%20and%20vein%20patterning&amp;author=S.%20Pahari&amp;author=R.D.%20Cormark&amp;author=M.T.%20Blackshaw&amp;author=C.%20Liu&amp;author=J.L.%20Erickson&amp;volume=141&amp;publication_year=2014&amp;pages=1894-1905&amp;pmid=24757006&amp;doi=10.1242/dev.099333&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib82">
<cite>Papuga J., Hoffmann C., Dieterle M., Moes D., Moreau F., Tholl S., Steinmetz A., Thomas C. (2010). <em>Arabidopsis</em> LIM proteins: a family of actin bundlers with distinct expression patterns and modes of regulation. Plant Cell
22: 3034–3052.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.110.075960" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2965535/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20817848/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Arabidopsis%20LIM%20proteins:%20a%20family%20of%20actin%20bundlers%20with%20distinct%20expression%20patterns%20and%20modes%20of%20regulation&amp;author=J.%20Papuga&amp;author=C.%20Hoffmann&amp;author=M.%20Dieterle&amp;author=D.%20Moes&amp;author=F.%20Moreau&amp;volume=22&amp;publication_year=2010&amp;pages=3034-3052&amp;pmid=20817848&amp;doi=10.1105/tpc.110.075960&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib83">
<cite>Paredez A.R., Persson S., Ehrhardt D.W., Somerville C.R. (2008). Genetic evidence that cellulose synthase activity influences microtubule cortical array organization. Plant Physiol.
147: 1723–1734.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.108.120196" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2492609/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18583534/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Genetic%20evidence%20that%20cellulose%20synthase%20activity%20influences%20microtubule%20cortical%20array%20organization&amp;author=A.R.%20Paredez&amp;author=S.%20Persson&amp;author=D.W.%20Ehrhardt&amp;author=C.R.%20Somerville&amp;volume=147&amp;publication_year=2008&amp;pages=1723-1734&amp;pmid=18583534&amp;doi=10.1104/pp.108.120196&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib84">
<cite>Penn O., Privman E., Ashkenazy H., Landan G., Graur D., Pupko T. (2010). GUIDANCE: a web server for assessing alignment confidence scores. Nucleic Acids Res.
38: W23–W28.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkq443" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2896199/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20497997/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=GUIDANCE:%20a%20web%20server%20for%20assessing%20alignment%20confidence%20scores&amp;author=O.%20Penn&amp;author=E.%20Privman&amp;author=H.%20Ashkenazy&amp;author=G.%20Landan&amp;author=D.%20Graur&amp;volume=38&amp;publication_year=2010&amp;pages=W23-W28&amp;pmid=20497997&amp;doi=10.1093/nar/gkq443&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib85">
<cite>Peremyslov V.V., Morgun E.A., Kurth E.G., Makarova K.S., Koonin E.V., Dolja V.V. (2013). Identification of myosin XI receptors in <em>Arabidopsis</em> defines a distinct class of transport vesicles. Plant Cell
25: 3022–3038.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.113.113704" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3784596/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23995081/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Identification%20of%20myosin%20XI%20receptors%20in%20Arabidopsis%20defines%20a%20distinct%20class%20of%20transport%20vesicles&amp;author=V.V.%20Peremyslov&amp;author=E.A.%20Morgun&amp;author=E.G.%20Kurth&amp;author=K.S.%20Makarova&amp;author=E.V.%20Koonin&amp;volume=25&amp;publication_year=2013&amp;pages=3022-3038&amp;pmid=23995081&amp;doi=10.1105/tpc.113.113704&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib86">
<cite>Preuss M.L., Serna J., Falbel T.G., Bednarek S.Y., Nielsen E. (2004). The <em>Arabidopsis</em> Rab GTPase RabA4b localizes to the tips of growing root hair cells. Plant Cell
16: 1589–1603.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.021634" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC490048/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15155878/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20Arabidopsis%20Rab%20GTPase%20RabA4b%20localizes%20to%20the%20tips%20of%20growing%20root%20hair%20cells&amp;author=M.L.%20Preuss&amp;author=J.%20Serna&amp;author=T.G.%20Falbel&amp;author=S.Y.%20Bednarek&amp;author=E.%20Nielsen&amp;volume=16&amp;publication_year=2004&amp;pages=1589-1603&amp;pmid=15155878&amp;doi=10.1105/tpc.021634&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib87">
<cite>Qu X., Zhang H., Xie Y., Wang J., Chen N., Huang S. (2013). <em>Arabidopsis</em> villins promote actin turnover at pollen tube tips and facilitate the construction of actin collars. Plant Cell
25: 1803–1817.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.113.110940" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3694707/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23715472/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Arabidopsis%20villins%20promote%20actin%20turnover%20at%20pollen%20tube%20tips%20and%20facilitate%20the%20construction%20of%20actin%20collars&amp;author=X.%20Qu&amp;author=H.%20Zhang&amp;author=Y.%20Xie&amp;author=J.%20Wang&amp;author=N.%20Chen&amp;volume=25&amp;publication_year=2013&amp;pages=1803-1817&amp;pmid=23715472&amp;doi=10.1105/tpc.113.110940&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib88">
<cite>Renna L., Stefano G., Majeran W., Micalella C., Meinnel T., Giglione C., Brandizzi F. (2013). Golgi traffic and integrity depend on N-myristoyl transferase-1 in <em>Arabidopsis.</em>
Plant Cell
25: 1756–1773.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.113.111393" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3694704/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23673980/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Golgi%20traffic%20and%20integrity%20depend%20on%20N-myristoyl%20transferase-1%20in%20Arabidopsis.&amp;author=L.%20Renna&amp;author=G.%20Stefano&amp;author=W.%20Majeran&amp;author=C.%20Micalella&amp;author=T.%20Meinnel&amp;volume=25&amp;publication_year=2013&amp;pages=1756-1773&amp;pmid=23673980&amp;doi=10.1105/tpc.113.111393&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib89">
<cite>Richter S., Voss U., Jürgens G. (2009). Post-Golgi traffic in plants. Traffic
10: 819–828.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1600-0854.2009.00916.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19416470/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Post-Golgi%20traffic%20in%20plants&amp;author=S.%20Richter&amp;author=U.%20Voss&amp;author=G.%20J%C3%BCrgens&amp;volume=10&amp;publication_year=2009&amp;pages=819-828&amp;pmid=19416470&amp;doi=10.1111/j.1600-0854.2009.00916.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib90">
<cite>Robert S., Chary S.N., Drakakaki G., Li S., Yang Z., Raikhel N.V., Hicks G.R. (2008). Endosidin1 defines a compartment involved in endocytosis of the brassinosteroid receptor BRI1 and the auxin transporters PIN2 and AUX1. Proc. Natl. Acad. Sci. USA
105: 8464–8469.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0711650105" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2448859/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18550817/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Endosidin1%20defines%20a%20compartment%20involved%20in%20endocytosis%20of%20the%20brassinosteroid%20receptor%20BRI1%20and%20the%20auxin%20transporters%20PIN2%20and%20AUX1&amp;author=S.%20Robert&amp;author=S.N.%20Chary&amp;author=G.%20Drakakaki&amp;author=S.%20Li&amp;author=Z.%20Yang&amp;volume=105&amp;publication_year=2008&amp;pages=8464-8469&amp;pmid=18550817&amp;doi=10.1073/pnas.0711650105&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib91">
<cite>Robinson D.G., Jiang L., Schumacher K. (2008). The endosomal system of plants: charting new and familiar territories. Plant Physiol.
147: 1482–1492.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.108.120105" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2492610/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18678740/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=The%20endosomal%20system%20of%20plants:%20charting%20new%20and%20familiar%20territories&amp;author=D.G.%20Robinson&amp;author=L.%20Jiang&amp;author=K.%20Schumacher&amp;volume=147&amp;publication_year=2008&amp;pages=1482-1492&amp;pmid=18678740&amp;doi=10.1104/pp.108.120105&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib92">
<cite>Rosado A., Schapire A.L., Bressan R.A., Harfouche A.L., Hasegawa P.M., Valpuesta V., Botella M.A. (2006). The <em>Arabidopsis</em> tetratricopeptide repeat-containing protein TTL1 is required for osmotic stress responses and abscisic acid sensitivity. Plant Physiol.
142: 1113–1126.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.106.085191" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1630727/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16998088/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=The%20Arabidopsis%20tetratricopeptide%20repeat-containing%20protein%20TTL1%20is%20required%20for%20osmotic%20stress%20responses%20and%20abscisic%20acid%20sensitivity&amp;author=A.%20Rosado&amp;author=A.L.%20Schapire&amp;author=R.A.%20Bressan&amp;author=A.L.%20Harfouche&amp;author=P.M.%20Hasegawa&amp;volume=142&amp;publication_year=2006&amp;pages=1113-1126&amp;pmid=16998088&amp;doi=10.1104/pp.106.085191&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib93">
<cite>Sparkes I., Runions J., Hawes C., Griffing L. (2009). Movement and remodeling of the endoplasmic reticulum in nondividing cells of tobacco leaves. Plant Cell
21: 3937–3949.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.109.072249" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2814503/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20040535/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Movement%20and%20remodeling%20of%20the%20endoplasmic%20reticulum%20in%20nondividing%20cells%20of%20tobacco%20leaves&amp;author=I.%20Sparkes&amp;author=J.%20Runions&amp;author=C.%20Hawes&amp;author=L.%20Griffing&amp;volume=21&amp;publication_year=2009&amp;pages=3937-3949&amp;pmid=20040535&amp;doi=10.1105/tpc.109.072249&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib94">
<cite>Staiger C.J., Sheahan M.B., Khurana P., Wang X., McCurdy D.W., Blanchoin L. (2009). Actin filament dynamics are dominated by rapid growth and severing activity in the <em>Arabidopsis</em> cortical array. J. Cell Biol.
184: 269–280.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.200806185" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2654301/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19171759/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol.&amp;title=Actin%20filament%20dynamics%20are%20dominated%20by%20rapid%20growth%20and%20severing%20activity%20in%20the%20Arabidopsis%20cortical%20array&amp;author=C.J.%20Staiger&amp;author=M.B.%20Sheahan&amp;author=P.%20Khurana&amp;author=X.%20Wang&amp;author=D.W.%20McCurdy&amp;volume=184&amp;publication_year=2009&amp;pages=269-280&amp;pmid=19171759&amp;doi=10.1083/jcb.200806185&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib95">
<cite>Stenzel I., Ischebeck T., König S., Hołubowska A., Sporysz M., Hause B., Heilmann I. (2008). The type B phosphatidylinositol-4-phosphate 5-kinase 3 is essential for root hair formation in <em>Arabidopsis thaliana.</em>
Plant Cell
20: 124–141.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.107.052852" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2254927/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18178770/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20type%20B%20phosphatidylinositol-4-phosphate%205-kinase%203%20is%20essential%20for%20root%20hair%20formation%20in%20Arabidopsis%20thaliana.&amp;author=I.%20Stenzel&amp;author=T.%20Ischebeck&amp;author=S.%20K%C3%B6nig&amp;author=A.%20Ho%C5%82ubowska&amp;author=M.%20Sporysz&amp;volume=20&amp;publication_year=2008&amp;pages=124-141&amp;pmid=18178770&amp;doi=10.1105/tpc.107.052852&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib96">
<cite>Stephan O., Cottier S., Fahlén S., Montes-Rodriguez A., Sun J., Eklund D.M., Klahre U., Kost B. (2014). RISAP is a TGN-associated RAC5 effector regulating membrane traffic during polar cell growth in tobacco. Plant Cell
26: 4426–4447.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.114.131078" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4277221/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25387880/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=RISAP%20is%20a%20TGN-associated%20RAC5%20effector%20regulating%20membrane%20traffic%20during%20polar%20cell%20growth%20in%20tobacco&amp;author=O.%20Stephan&amp;author=S.%20Cottier&amp;author=S.%20Fahl%C3%A9n&amp;author=A.%20Montes-Rodriguez&amp;author=J.%20Sun&amp;volume=26&amp;publication_year=2014&amp;pages=4426-4447&amp;pmid=25387880&amp;doi=10.1105/tpc.114.131078&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib97">
<cite>Su H., Zhu J., Cai C., Pei W., Wang J., Dong H., Ren H. (2012). FIMBRIN1 is involved in lily pollen tube growth by stabilizing the actin fringe. Plant Cell
24: 4539–4554.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.112.099358" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3531851/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23150633/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=FIMBRIN1%20is%20involved%20in%20lily%20pollen%20tube%20growth%20by%20stabilizing%20the%20actin%20fringe&amp;author=H.%20Su&amp;author=J.%20Zhu&amp;author=C.%20Cai&amp;author=W.%20Pei&amp;author=J.%20Wang&amp;volume=24&amp;publication_year=2012&amp;pages=4539-4554&amp;pmid=23150633&amp;doi=10.1105/tpc.112.099358&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib98">
<cite>Sun T., Li S., Ren H. (2013). Profilin as a regulator of the membrane-actin cytoskeleton interface in plant cells. Front. Plant Sci.
4: 512.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2013.00512" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3867660/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24391654/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=Profilin%20as%20a%20regulator%20of%20the%20membrane-actin%20cytoskeleton%20interface%20in%20plant%20cells&amp;author=T.%20Sun&amp;author=S.%20Li&amp;author=H.%20Ren&amp;volume=4&amp;publication_year=2013&amp;pages=512&amp;pmid=24391654&amp;doi=10.3389/fpls.2013.00512&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib99">
<cite>Takano J., Tanaka M., Toyoda A., Miwa K., Kasai K., Fuji K., Onouchi H., Naito S., Fujiwara T. (2010). Polar localization and degradation of <em>Arabidopsis</em> boron transporters through distinct trafficking pathways. Proc. Natl. Acad. Sci. USA
107: 5220–5225.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0910744107" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2841934/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20194745/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Polar%20localization%20and%20degradation%20of%20Arabidopsis%20boron%20transporters%20through%20distinct%20trafficking%20pathways&amp;author=J.%20Takano&amp;author=M.%20Tanaka&amp;author=A.%20Toyoda&amp;author=K.%20Miwa&amp;author=K.%20Kasai&amp;volume=107&amp;publication_year=2010&amp;pages=5220-5225&amp;pmid=20194745&amp;doi=10.1073/pnas.0910744107&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib100">
<cite>Takenawa T., Suetsugu S. (2007). The WASP-WAVE protein network: connecting the membrane to the cytoskeleton. Nat. Rev. Mol. Cell Biol.
8: 37–48.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nrm2069" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17183359/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;title=The%20WASP-WAVE%20protein%20network:%20connecting%20the%20membrane%20to%20the%20cytoskeleton&amp;author=T.%20Takenawa&amp;author=S.%20Suetsugu&amp;volume=8&amp;publication_year=2007&amp;pages=37-48&amp;pmid=17183359&amp;doi=10.1038/nrm2069&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib101">
<cite>Tamura K., Stecher G., Peterson D., Filipski A., Kumar S. (2013). MEGA6: molecular evolutionary genetics analysis version 6.0. Mol. Biol. Evol.
30: 2725–2729.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/molbev/mst197" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3840312/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24132122/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=MEGA6:%20molecular%20evolutionary%20genetics%20analysis%20version%206.0&amp;author=K.%20Tamura&amp;author=G.%20Stecher&amp;author=D.%20Peterson&amp;author=A.%20Filipski&amp;author=S.%20Kumar&amp;volume=30&amp;publication_year=2013&amp;pages=2725-2729&amp;pmid=24132122&amp;doi=10.1093/molbev/mst197&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib102">
<cite>Tanaka H., Kitakura S., De Rycke R., De Groodt R., Friml J. (2009). Fluorescence imaging-based screen identifies ARF GEF component of early endosomal trafficking. Curr. Biol.
19: 391–397.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cub.2009.01.057" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19230664/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=Fluorescence%20imaging-based%20screen%20identifies%20ARF%20GEF%20component%20of%20early%20endosomal%20trafficking&amp;author=H.%20Tanaka&amp;author=S.%20Kitakura&amp;author=R.%20De%20Rycke&amp;author=R.%20De%20Groodt&amp;author=J.%20Friml&amp;volume=19&amp;publication_year=2009&amp;pages=391-397&amp;pmid=19230664&amp;doi=10.1016/j.cub.2009.01.057&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib103">
<cite>Tanaka H., Kitakura S., Rakusová H., Uemura T., Feraru M.I., De Rycke R., Robert S., Kakimoto T., Friml J. (2013). Cell polarity and patterning by PIN trafficking through early endosomal compartments in <em>Arabidopsis thaliana.</em>
PLoS Genet.
9: e1003540.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pgen.1003540" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3667747/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23737757/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Cell%20polarity%20and%20patterning%20by%20PIN%20trafficking%20through%20early%20endosomal%20compartments%20in%20Arabidopsis%20thaliana.&amp;author=H.%20Tanaka&amp;author=S.%20Kitakura&amp;author=H.%20Rakusov%C3%A1&amp;author=T.%20Uemura&amp;author=M.I.%20Feraru&amp;volume=9&amp;publication_year=2013&amp;pages=e1003540&amp;pmid=23737757&amp;doi=10.1371/journal.pgen.1003540&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib104">
<cite>Tanaka H., Nodzyłski T., Kitakura S., Feraru M.I., Sasabe M., Ishikawa T., Kleine-Vehn J., Kakimoto T., Friml J. (2014). BEX1/ARF1A1C is required for BFA-sensitive recycling of PIN auxin transporters and auxin-mediated development in <em>Arabidopsis.</em>
Plant Cell Physiol.
55: 737–749.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pct196" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3982122/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24369434/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol.&amp;title=BEX1/ARF1A1C%20is%20required%20for%20BFA-sensitive%20recycling%20of%20PIN%20auxin%20transporters%20and%20auxin-mediated%20development%20in%20Arabidopsis.&amp;author=H.%20Tanaka&amp;author=T.%20Nodzy%C5%82ski&amp;author=S.%20Kitakura&amp;author=M.I.%20Feraru&amp;author=M.%20Sasabe&amp;volume=55&amp;publication_year=2014&amp;pages=737-749&amp;pmid=24369434&amp;doi=10.1093/pcp/pct196&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib105">
<cite>Thole J.M., Vermeer J.E., Zhang Y., Gadella T.W. Jr., Nielsen E. (2008). Root hair defective4 encodes a phosphatidylinositol-4-phosphate phosphatase required for proper root hair development in <em>Arabidopsis thaliana.</em>
Plant Cell
20: 381–395.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.107.054304" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2276440/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18281508/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Root%20hair%20defective4%20encodes%20a%20phosphatidylinositol-4-phosphate%20phosphatase%20required%20for%20proper%20root%20hair%20development%20in%20Arabidopsis%20thaliana.&amp;author=J.M.%20Thole&amp;author=J.E.%20Vermeer&amp;author=Y.%20Zhang&amp;author=T.W.%20Gadella&amp;author=E.%20Nielsen&amp;volume=20&amp;publication_year=2008&amp;pages=381-395&amp;pmid=18281508&amp;doi=10.1105/tpc.107.054304&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib106">
<cite>Uemura T., Suda Y., Ueda T., Nakano A. (2014). Dynamic behavior of the trans-golgi network in root tissues of <em>Arabidopsis</em> revealed by super-resolution live imaging. Plant Cell Physiol.
55: 694–703.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pcu010" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24443496/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol.&amp;title=Dynamic%20behavior%20of%20the%20trans-golgi%20network%20in%20root%20tissues%20of%20Arabidopsis%20revealed%20by%20super-resolution%20live%20imaging&amp;author=T.%20Uemura&amp;author=Y.%20Suda&amp;author=T.%20Ueda&amp;author=A.%20Nakano&amp;volume=55&amp;publication_year=2014&amp;pages=694-703&amp;pmid=24443496&amp;doi=10.1093/pcp/pcu010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib107">
<cite>Uemura T., Ueda T., Nakano A. (2012). The physiological role of SYP4 in the salinity and osmotic stress tolerances. Plant Signal. Behav.
7: 1118–1120.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4161/psb.21307" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3489641/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22899062/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signal.%20Behav.&amp;title=The%20physiological%20role%20of%20SYP4%20in%20the%20salinity%20and%20osmotic%20stress%20tolerances&amp;author=T.%20Uemura&amp;author=T.%20Ueda&amp;author=A.%20Nakano&amp;volume=7&amp;publication_year=2012&amp;pages=1118-1120&amp;pmid=22899062&amp;doi=10.4161/psb.21307&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib108">
<cite>Vidali L., Burkart G.M., Augustine R.C., Kerdavid E., Tüzel E., Bezanilla M. (2010). Myosin XI is essential for tip growth in <em>Physcomitrella patens.</em>
Plant Cell
22: 1868–1882.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.109.073288" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2910981/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20525854/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Myosin%20XI%20is%20essential%20for%20tip%20growth%20in%20Physcomitrella%20patens.&amp;author=L.%20Vidali&amp;author=G.M.%20Burkart&amp;author=R.C.%20Augustine&amp;author=E.%20Kerdavid&amp;author=E.%20T%C3%BCzel&amp;volume=22&amp;publication_year=2010&amp;pages=1868-1882&amp;pmid=20525854&amp;doi=10.1105/tpc.109.073288&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib109">
<cite>Vidali L., Rounds C.M., Hepler P.K., Bezanilla M. (2009). Lifeact-mEGFP reveals a dynamic apical F-actin network in tip growing plant cells. PLoS One
4: e5744.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0005744" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2684639/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19478943/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Lifeact-mEGFP%20reveals%20a%20dynamic%20apical%20F-actin%20network%20in%20tip%20growing%20plant%20cells&amp;author=L.%20Vidali&amp;author=C.M.%20Rounds&amp;author=P.K.%20Hepler&amp;author=M.%20Bezanilla&amp;volume=4&amp;publication_year=2009&amp;pages=e5744&amp;pmid=19478943&amp;doi=10.1371/journal.pone.0005744&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib110">
<cite>Vincent P., Chua M., Nogue F., Fairbrother A., Mekeel H., Xu Y., Allen N., Bibikova T.N., Gilroy S., Bankaitis V.A. (2005). A Sec14p-nodulin domain phosphatidylinositol transfer protein polarizes membrane growth of <em>Arabidopsis thaliana</em> root hairs. J. Cell Biol.
168: 801–812.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.200412074" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2171805/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15728190/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol.&amp;title=A%20Sec14p-nodulin%20domain%20phosphatidylinositol%20transfer%20protein%20polarizes%20membrane%20growth%20of%20Arabidopsis%20thaliana%20root%20hairs&amp;author=P.%20Vincent&amp;author=M.%20Chua&amp;author=F.%20Nogue&amp;author=A.%20Fairbrother&amp;author=H.%20Mekeel&amp;volume=168&amp;publication_year=2005&amp;pages=801-812&amp;pmid=15728190&amp;doi=10.1083/jcb.200412074&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib111">
<cite>Viotti C., et al.  (2010). Endocytic and secretory traffic in <em>Arabidopsis</em> merge in the trans-Golgi network/early endosome, an independent and highly dynamic organelle. Plant Cell
22: 1344–1357.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.109.072637" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2879741/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20435907/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Endocytic%20and%20secretory%20traffic%20in%20Arabidopsis%20merge%20in%20the%20trans-Golgi%20network/early%20endosome,%20an%20independent%20and%20highly%20dynamic%20organelle&amp;author=C.%20Viotti&amp;volume=22&amp;publication_year=2010&amp;pages=1344-1357&amp;pmid=20435907&amp;doi=10.1105/tpc.109.072637&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib112">
<cite>Wang P., Hawkins T.J., Richardson C., Cummins I., Deeks M.J., Sparkes I., Hawes C., Hussey P.J. (2014). The plant cytoskeleton, NET3C, and VAP27 mediate the link between the plasma membrane and endoplasmic reticulum. Curr. Biol.
24: 1397–1405.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cub.2014.05.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24909329/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=The%20plant%20cytoskeleton,%20NET3C,%20and%20VAP27%20mediate%20the%20link%20between%20the%20plasma%20membrane%20and%20endoplasmic%20reticulum&amp;author=P.%20Wang&amp;author=T.J.%20Hawkins&amp;author=C.%20Richardson&amp;author=I.%20Cummins&amp;author=M.J.%20Deeks&amp;volume=24&amp;publication_year=2014&amp;pages=1397-1405&amp;pmid=24909329&amp;doi=10.1016/j.cub.2014.05.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib113">
<cite>Wang Y.S., Yoo C.M., Blancaflor E.B. (2008). Improved imaging of actin filaments in transgenic <em>Arabidopsis</em> plants expressing a green fluorescent protein fusion to the C- and N-termini of the fimbrin actin-binding domain 2. New Phytol.
177: 525–536.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1469-8137.2007.02261.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18028299/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Improved%20imaging%20of%20actin%20filaments%20in%20transgenic%20Arabidopsis%20plants%20expressing%20a%20green%20fluorescent%20protein%20fusion%20to%20the%20C-%20and%20N-termini%20of%20the%20fimbrin%20actin-binding%20domain%202&amp;author=Y.S.%20Wang&amp;author=C.M.%20Yoo&amp;author=E.B.%20Blancaflor&amp;volume=177&amp;publication_year=2008&amp;pages=525-536&amp;pmid=18028299&amp;doi=10.1111/j.1469-8137.2007.02261.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib114">
<cite>Whippo C.W., Khurana P., Davis P.A., DeBlasio S.L., DeSloover D., Staiger C.J., Hangarter R.P. (2011). THRUMIN1 is a light-regulated actin-bundling protein involved in chloroplast motility. Curr. Biol.
21: 59–64.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cub.2010.11.059" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21185188/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=THRUMIN1%20is%20a%20light-regulated%20actin-bundling%20protein%20involved%20in%20chloroplast%20motility&amp;author=C.W.%20Whippo&amp;author=P.%20Khurana&amp;author=P.A.%20Davis&amp;author=S.L.%20DeBlasio&amp;author=D.%20DeSloover&amp;volume=21&amp;publication_year=2011&amp;pages=59-64&amp;pmid=21185188&amp;doi=10.1016/j.cub.2010.11.059&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib115">
<cite>Yang Y., Sage T.L., Liu Y., Ahmad T.R., Marshall W.F., Shiu S.-H., Froehlich J.E., Imre K.M., Osteryoung K.W. (2011). CLUMPED CHLOROPLASTS 1 is required for plastid separation in <em>Arabidopsis.</em>
Proc. Natl. Acad. Sci. USA
108: 18530–18535.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1106706108" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3215037/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22025705/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=CLUMPED%20CHLOROPLASTS%201%20is%20required%20for%20plastid%20separation%20in%20Arabidopsis.&amp;author=Y.%20Yang&amp;author=T.L.%20Sage&amp;author=Y.%20Liu&amp;author=T.R.%20Ahmad&amp;author=W.F.%20Marshall&amp;volume=108&amp;publication_year=2011&amp;pages=18530-18535&amp;pmid=22025705&amp;doi=10.1073/pnas.1106706108&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib116">
<cite>Yoo C.M., Quan L., Cannon A.E., Wen J., Blancaflor E.B. (2012). AGD1, a class 1 ARF-GAP, acts in common signaling pathways with phosphoinositide metabolism and the actin cytoskeleton in controlling <em>Arabidopsis</em> root hair polarity. Plant J.
69: 1064–1076.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2011.04856.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22098134/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=AGD1,%20a%20class%201%20ARF-GAP,%20acts%20in%20common%20signaling%20pathways%20with%20phosphoinositide%20metabolism%20and%20the%20actin%20cytoskeleton%20in%20controlling%20Arabidopsis%20root%20hair%20polarity&amp;author=C.M.%20Yoo&amp;author=L.%20Quan&amp;author=A.E.%20Cannon&amp;author=J.%20Wen&amp;author=E.B.%20Blancaflor&amp;volume=69&amp;publication_year=2012&amp;pages=1064-1076&amp;pmid=22098134&amp;doi=10.1111/j.1365-313X.2011.04856.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib117">
<cite>Zhang C., Kotchoni S.O., Samuels A.L., Szymanski D.B. (2010). SPIKE1 signals originate from and assemble specialized domains of the endoplasmic reticulum. Curr. Biol.
20: 2144–2149.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cub.2010.11.016" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21109438/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=SPIKE1%20signals%20originate%20from%20and%20assemble%20specialized%20domains%20of%20the%20endoplasmic%20reticulum&amp;author=C.%20Zhang&amp;author=S.O.%20Kotchoni&amp;author=A.L.%20Samuels&amp;author=D.B.%20Szymanski&amp;volume=20&amp;publication_year=2010&amp;pages=2144-2149&amp;pmid=21109438&amp;doi=10.1016/j.cub.2010.11.016&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib118">
<cite>Zhang C., Mallery E., Reagan S., Boyko V.P., Kotchoni S.O., Szymanski D.B. (2013). The endoplasmic reticulum is a reservoir for WAVE/SCAR regulatory complex signaling in the <em>Arabidopsis</em> leaf. Plant Physiol.
162: 689–706.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.113.217422" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3668063/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23613272/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=The%20endoplasmic%20reticulum%20is%20a%20reservoir%20for%20WAVE/SCAR%20regulatory%20complex%20signaling%20in%20the%20Arabidopsis%20leaf&amp;author=C.%20Zhang&amp;author=E.%20Mallery&amp;author=S.%20Reagan&amp;author=V.P.%20Boyko&amp;author=S.O.%20Kotchoni&amp;volume=162&amp;publication_year=2013&amp;pages=689-706&amp;pmid=23613272&amp;doi=10.1104/pp.113.217422&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib119">
<cite>Zhang M., Wang C., Lin Q., Liu A., Wang T., Feng X., Liu J., Han H., Ma Y., Bonea D., Zhao R., Hua X. (2015). A tetratricopeptide repeat domain-containing protein SSR1 located in mitochondria is involved in root development and auxin polar transport in <em>Arabidopsis.</em>
Plant J.
83: 582–599.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.12911" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26072661/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=A%20tetratricopeptide%20repeat%20domain-containing%20protein%20SSR1%20located%20in%20mitochondria%20is%20involved%20in%20root%20development%20and%20auxin%20polar%20transport%20in%20Arabidopsis.&amp;author=M.%20Zhang&amp;author=C.%20Wang&amp;author=Q.%20Lin&amp;author=A.%20Liu&amp;author=T.%20Wang&amp;volume=83&amp;publication_year=2015&amp;pages=582-599&amp;pmid=26072661&amp;doi=10.1111/tpj.12911&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="bib120">
<cite>Zouhar J., Rojo E., Bassham D.C. (2009). AtVPS45 is a positive regulator of the SYP41/SYP61/VTI12 SNARE complex involved in trafficking of vacuolar cargo. Plant Physiol.
149: 1668–1678.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.108.134361" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2663731/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19251905/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=AtVPS45%20is%20a%20positive%20regulator%20of%20the%20SYP41/SYP61/VTI12%20SNARE%20complex%20involved%20in%20trafficking%20of%20vacuolar%20cargo&amp;author=J.%20Zouhar&amp;author=E.%20Rojo&amp;author=D.C.%20Bassham&amp;volume=149&amp;publication_year=2009&amp;pages=1668-1678&amp;pmid=19251905&amp;doi=10.1104/pp.108.134361&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ol></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Supplemental Data</span></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_28_3_746__index.html">supp_28_3_746__index.html</a><sup> (3.1KB, html) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Data.pdf">supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Data.pdf</a><sup> (5.3MB, pdf) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_Legends.pdf">supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_Legends.pdf</a><sup> (31.4KB, pdf) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Data_sets_1_2.xlsx">supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Data_sets_1_2.xlsx</a><sup> (37.1KB, xlsx) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Data_set_3.txt">supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Data_set_3.txt</a><sup> (65.2KB, txt) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_1.mp4">Download video file</a><sup> (9.3MB, mp4) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_2.mp4">Download video file</a><sup> (9MB, mp4) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_3.avi">Download video file</a><sup> (1,003.5KB, avi) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_4.mp4">Download video file</a><sup> (9.4MB, mp4) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_5.mp4">Download video file</a><sup> (9.2MB, mp4) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_6.mp4">Download video file</a><sup> (8.9MB, mp4) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_7.avi">Download video file</a><sup> (9.2MB, avi) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_8.avi">Download video file</a><sup> (8.3MB, avi) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_9.mp4">Download video file</a><sup> (9.3MB, mp4) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_10.avi">Download video file</a><sup> (4.9MB, avi) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_11.mp4">Download video file</a><sup> (9.5MB, mp4) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_12.mp4">Download video file</a><sup> (7.8MB, mp4) </sup>
</div></div>
<div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/4826010/bin/supp_tpc.15.00794_TPC2015-00794-RAR2_Supplemental_Movie_13.mp4">Download video file</a><sup> (8.1MB, mp4) </sup>
</div></div></section></section></section></section></section></article>
</main></body></html>