<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10712242/"/><title>Integrative transcriptomics and proteomics profiling of Arabidopsis thaliana elucidates novel mechanisms underlying spaceflight adaptation - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Front Plant Sci</button></div>. 2023 Nov 27;14:1260429. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2023.1260429" rel="noopener noreferrer" target="_blank">10.3389/fpls.2023.1260429</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Integrative transcriptomics and proteomics profiling of <em>Arabidopsis thaliana</em> elucidates novel mechanisms underlying spaceflight adaptation</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Olanrewaju%20GO%22%5BAuthor%5D"><span class="name western">Gbolaga O Olanrewaju</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Gbolaga O Olanrewaju</span></h3>
<div class="p">
<sup>1</sup>
Interdisciplinary Molecular and Cellular Biology Program, Ohio University, Athens, OH, United States
</div>
<div class="p">
<sup>2</sup>
Department of Environmental and Plant Biology, Ohio University Athens, OH, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Olanrewaju%20GO%22%5BAuthor%5D"><span class="name western">Gbolaga O Olanrewaju</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Haveman%20NJ%22%5BAuthor%5D"><span class="name western">Natasha J Haveman</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Natasha J Haveman</span></h3>
<div class="p">
<sup>3</sup>
NASA Utilization &amp; Life Sciences Office (UB-A), Kennedy Space Center, Merritt Island, FL, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Haveman%20NJ%22%5BAuthor%5D"><span class="name western">Natasha J Haveman</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Naldrett%20MJ%22%5BAuthor%5D"><span class="name western">Michael J Naldrett</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Michael J Naldrett</span></h3>
<div class="p">
<sup>4</sup>
Proteomics &amp; Metabolomics Facility, Nebraska Center for Biotechnology, University of Nebraska–Lincoln, Lincoln, NE, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Naldrett%20MJ%22%5BAuthor%5D"><span class="name western">Michael J Naldrett</span></a>
</div>
</div>
<sup>4</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paul%20AL%22%5BAuthor%5D"><span class="name western">Anna-Lisa Paul</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Anna-Lisa Paul</span></h3>
<div class="p">
<sup>5</sup>
Department of Horticultural Sciences, University of Florida, Gainesville, FL, United States
</div>
<div class="p">
<sup>6</sup>
Interdisciplinary Center for Biotechnology Research, University of Florida, Gainesville, FL, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paul%20AL%22%5BAuthor%5D"><span class="name western">Anna-Lisa Paul</span></a>
</div>
</div>
<sup>5,</sup><sup>6</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferl%20RJ%22%5BAuthor%5D"><span class="name western">Robert J Ferl</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Robert J Ferl</span></h3>
<div class="p">
<sup>5</sup>
Department of Horticultural Sciences, University of Florida, Gainesville, FL, United States
</div>
<div class="p">
<sup>7</sup>
Office of Research, University of Florida, Gainesville, FL, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferl%20RJ%22%5BAuthor%5D"><span class="name western">Robert J Ferl</span></a>
</div>
</div>
<sup>5,</sup><sup>7</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wyatt%20SE%22%5BAuthor%5D"><span class="name western">Sarah E Wyatt</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Sarah E Wyatt</span></h3>
<div class="p">
<sup>1</sup>
Interdisciplinary Molecular and Cellular Biology Program, Ohio University, Athens, OH, United States
</div>
<div class="p">
<sup>2</sup>
Department of Environmental and Plant Biology, Ohio University Athens, OH, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wyatt%20SE%22%5BAuthor%5D"><span class="name western">Sarah E Wyatt</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>
Interdisciplinary Molecular and Cellular Biology Program, Ohio University, Athens, OH, United States
</div>
<div id="aff2">
<sup>2</sup>
Department of Environmental and Plant Biology, Ohio University Athens, OH, United States
</div>
<div id="aff3">
<sup>3</sup>
NASA Utilization &amp; Life Sciences Office (UB-A), Kennedy Space Center, Merritt Island, FL, United States
</div>
<div id="aff4">
<sup>4</sup>
Proteomics &amp; Metabolomics Facility, Nebraska Center for Biotechnology, University of Nebraska–Lincoln, Lincoln, NE, United States
</div>
<div id="aff5">
<sup>5</sup>
Department of Horticultural Sciences, University of Florida, Gainesville, FL, United States
</div>
<div id="aff6">
<sup>6</sup>
Interdisciplinary Center for Biotechnology Research, University of Florida, Gainesville, FL, United States
</div>
<div id="aff7">
<sup>7</sup>
Office of Research, University of Florida, Gainesville, FL, United States
</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Edited by: John Z. Kiss, University of North Carolina at Greensboro, United States</p></div>
<div class="fn" id="fn2"><p>Reviewed by: Patrick H. Masson, University of Wisconsin-Madison, United States; Huiqiong Zheng, Shanghai Institutes for Biological Sciences (CAS), China</p></div>
<div class="fn" id="fn001">
<sup>✉</sup><p class="display-inline">*Correspondence: Sarah E. Wyatt, <span>wyatts@ohio.edu</span>
</p>
</div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2023 Jul 17; Accepted 2023 Nov 8; Collection date 2023.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2023 Olanrewaju, Haveman, Naldrett, Paul, Ferl and Wyatt</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC10712242  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/38089794/">38089794</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Spaceflight presents a unique environment with complex stressors, including microgravity and radiation, that can influence plant physiology at molecular levels. Combining transcriptomics and proteomics approaches, this research gives insights into the coordination of transcriptome and proteome in Arabidopsis’ molecular and physiological responses to Spaceflight environmental stress. Arabidopsis seedlings were germinated and grown in microgravity (µ<em>g</em>) aboard the International Space Station (ISS) in NASA Biological Research in Canisters – Light Emitting Diode (BRIC LED) hardware, with the ground control established on Earth. At 10 days old, seedlings were frozen in RNA-later and returned to Earth. RNA-seq transcriptomics and TMT-labeled LC-MS/MS proteomic analysis of cellular fractionates from the plant tissues suggest the alteration of the photosynthetic machinery (PSII and PSI) in spaceflight, with the plant shifting photosystem core-regulatory proteins in an organ-specific manner to adapt to the microgravity environment. An overview of the ribosome, spliceosome, and proteasome activities in spaceflight revealed a significant abundance of transcripts and proteins involved in protease binding, nuclease activities, and mRNA binding in spaceflight, while those involved in tRNA binding, exoribonuclease activity, and RNA helicase activity were less abundant in spaceflight. CELLULOSE SYNTHASES (CESA1, CESA3, CESA5, CESA7) and CELLULOSE-LIKE PROTEINS (CSLE1, CSLG3), involved in cellulose deposition and TUBULIN COFACTOR B (TFCB) had reduced abundance in spaceflight. This contrasts with the increased expression of UDP-ARABINOPYRANOSE MUTASEs, involved in the biosynthesis of cell wall non-cellulosic polysaccharides, in spaceflight. Both transcripts and proteome suggested an altered polar auxin redistribution, lipid, and ionic intracellular transportation in spaceflight. Analyses also suggest an increased metabolic energy requirement for plants in Space than on Earth, hence, the activation of several shunt metabolic pathways. This study provides novel insights, based on integrated RNA and protein data, on how plants adapt to the spaceflight environment and it is a step further at achieving sustainable crop production in Space.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> spaceflight, gravitropism, Arabidopsis, proteomics, RNAseq, TMT, BRIC LED, International Space Station</p></section></section><section id="s1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Numerous plant spaceflight studies have long focused on unraveling the complex physiological changes plants undergo in the spaceflight environment. In their diversity, these studies have accumulated an extensive array of phenotypic characterizations and uncovered some of the fundamental concepts related to physiological adaptation in space (<a aria-describedby="B27" class="usa-link" href="#B27">Kordyum and Hasenstein, 2021</a>). However, the traditional approaches have recently been complemented and in many ways superseded by the advent of genomic, transcriptomic, and proteomic methods, setting the stage for high throughput analyses that investigates the effects of spaceflight on plants from a molecular and functional perspective (<a aria-describedby="B19" class="usa-link" href="#B19">Hughes and Kiss, 2022</a>). This allows the physiological effects of the spaceflight environment to be characterized beyond phenotyping and extensively mapped out along broader biochemical pathways and cryptic signaling molecular cascades. While some remarkable advancements in genomic and transcriptomics studies of plants in spaceflight have been widely recorded (<a aria-describedby="B12" class="usa-link" href="#B12">Ferl et al., 2002</a>; <a aria-describedby="B43" class="usa-link" href="#B43">Paul et al., 2012</a>; <a aria-describedby="B3" class="usa-link" href="#B3">Barker et al., 2023</a>), such studies have largely been conducted in isolation, independent of proteomics. Unlike genomic and transcriptomic studies, proteomics studies face certain limitations in the spaceflight context, predominantly due to the limited availability of plant tissues for analysis. As a result, comprehensive spaceflight proteomics studies conducted on the whole Arabidopsis plant have been relatively few, with only three reported so far (<a aria-describedby="B35" class="usa-link" href="#B35">Mazars et al., 2014</a>; <a aria-describedby="B11" class="usa-link" href="#B11">Ferl et al., 2015</a>; <a aria-describedby="B4" class="usa-link" href="#B4">Barker et al., 2020</a>; <a aria-describedby="B30" class="usa-link" href="#B30">Kruse et al., 2020</a>).</p>
<p>
<a aria-describedby="B35" class="usa-link" href="#B35">Mazars et al. (2014)</a> conducted the first true proteomics-scale study of Arabidopsis microsome-associated membrane proteins on the ISS followed by <a aria-describedby="B15" class="usa-link" href="#B15">Hausmann et al. (2014)</a> study on the proteome of Arabidopsis cell cultures in parabolic flights and <a aria-describedby="B60" class="usa-link" href="#B60">Zhang et al. (2015)</a> proteomic study on Arabidopsis callus aboard the Chinese SZ-8 spacecraft in spaceflight. <a aria-describedby="B11" class="usa-link" href="#B11">Ferl et al. (2015)</a> comparison of the transcriptome and protein abundances between the leaf and root of Arabidopsis in spaceflight represented the first use of a combined transcriptomics-proteomics investigative approach for whole plant spaceflight experiment, followed by a similar but more in-depth approach by <a aria-describedby="B30" class="usa-link" href="#B30">Kruse et al. (2020)</a> where they revealed novel regulatory responses of Arabidopsis seedlings in spaceflight. Reports from these two multi-omics studies indicated that the true power of omics approaches becomes apparent when they are combined, providing a comprehensive view of how gene expression and protein synthesis respond to the unique conditions of spaceflight. Likewise, the reported low correlation between transcripts and proteome of Arabidopsis grown in spaceflight and spaceflight hardware (<a aria-describedby="B5" class="usa-link" href="#B5">Basu et al., 2017</a>; <a aria-describedby="B29" class="usa-link" href="#B29">Kruse et al., 2017</a>; <a aria-describedby="B30" class="usa-link" href="#B30">Kruse et al., 2020</a>) reinforced the benefits of a multi-omics approach.</p>
<p>Studying plants’ adaptation to the spaceflight environment usually involves plants being flown to the ISS in designated hardware, whose designs and subsequent improvements are often based on the objectives of the experiment at hand (<a aria-describedby="B22" class="usa-link" href="#B22">Kanapskyte et al., 2021</a>; <a aria-describedby="B39" class="usa-link" href="#B39">Nicholson et al., 2021</a>). One such hardware is the Biological Research In Canister – Light Emitting Diode (BRIC LED). BRIC LED hardware is designed to enhance the capabilities of existing BRIC Petri Dish Fixation Units (PDFU) by the incorporation of a customizable discreet lighting system. Hence, biological organisms growing in the BRIC LED hardware are illuminated by LEDs with options of four different wavelengths (blue, red, far-red, and white) (<a aria-describedby="B18" class="usa-link" href="#B18">Howard and Caron, 2016</a>). Spaceflight experiments in the BRIC LED hardware expose plants to light cues in microgravity, hence closely simulating a natural environment where plants have to simultaneously integrate light and gravitational signaling. The incorporation of weightlessness and light signaling is expected to elicit several physiological and molecular responses which are distinct from other hardware without a light source (<a aria-describedby="B49" class="usa-link" href="#B49">Stankovic, 2018</a>), with the responses often novel and a function of cross-talks among various signaling pathways rather than an individual signal transduction pathway. Tightly regulated by complex cross-signaling pathways, the responses are usually specific to each plant organ and most times, tissues (<a aria-describedby="B58" class="usa-link" href="#B58">Wolinska and Berens, 2019</a>). Transcripts and proteomes encoding these stress response pathways are usually localized to a particular intracellular or extracellular region, further reinforcing the idea that studies on plant stress response should be focused on finer resolutions of organs, tissues, cells, and organelles. Several studies have identified that the plant root and shoot respond in an organ-specific manner to gravity (<a aria-describedby="B44" class="usa-link" href="#B44">Paul et al., 2013</a>; <a aria-describedby="B54" class="usa-link" href="#B54">Vandenbrink et al., 2014</a>; <a aria-describedby="B33" class="usa-link" href="#B33">Lamers et al., 2020</a>) and other environmental stresses (<a aria-describedby="B41" class="usa-link" href="#B41">Paponov et al., 2021</a>). Hence, resolving plant proteomic responses to the spaceflight environment on an organ-specific level is a step further in understanding stress response signaling pathways and the molecular adaptations of each plant organ to the spaceflight environment and how such adaptation mechanisms can be manipulated for improved plant production in space and on earth.</p>
<p>Therefore, building upon these previous achievements in plant multi-omics research in spaceflight, we took a significant step forward by integrating both proteomics and transcriptomics approaches to understand the response of Arabidopsis plants to the spaceflight environment. Through increased coverage and finer spatial proteomic resolution, we assessed the effects of the earth’s low orbit spaceflight environment aboard the ISS on the shoot and root of the Arabidopsis seedlings. The seedlings were grown in the NASA BRIC LED hardware, hence we provided robust insight into the plant transcriptome landscape and organ-specific proteomic responses of Arabidopsis to the spaceflight environment, while at the same time conducting a hardware verification test for the novel BRIC LED hardware.</p></section><section id="s2"><h2 class="pmc_sec_title">2. Materials and methods</h2>
<section id="s2_1"><h3 class="pmc_sec_title">2.1. Plant material and spaceflight growth conditions</h3>
<p>
<em>Arabidopsis thaliana</em> L., ecotype Columbia-0 seeds were surface sterilized with 30% (v/v) bleach, and 0.1% (v/v) Tween-20 for 2 mins, washed repeatedly with sterile water (5×) and dried on round filter paper. Approximately 30 seeds were plated on each 60mm round petri plate containing 6.7ml of 0.5× + Benomyl Murashige and Skoog (MS) growth media with Gamborg’s B5 vitamins, 0.5% (w/v) sucrose and 0.75% (w/v) phytagel, and cold stratified for 16h. Seeded plates were loaded in a horizontal orientation, with the face of the growth media parallel to the ground in the BRIC LED hardware petri dish fixation units (PDFUs). Each hardware canister holds 6 PDFUs, with each PDFU holding a 60mm petri plate. This configuration was replicated 6 times, making a total of 36 seeded 60mm plates loaded aboard ISS. All BRICs were kept in cold storage (4°C) prior to the spaceflight launch. The experiment was flown on the SpaceX CRS-13 mission launched aboard the Falcon 9 Full Thrust rocket on December 15, 2017. The dormant Arabidopsis seeds were activated with light and placed at ambient temperature aboard the ISS, with a light wavelength composition of 85% red, 15% blue, and 60 µmol m<sup>−2</sup> s<sup>−1</sup> intensity. The plants were grown for 10 days with a 4-hour/2-hour light-dark cycle, prior to actuation of the PDFUs, releasing about 6–7ml RNAlater into the petri plates for seedling fixation. The seedlings were then incubated in Invitrogen RNAlater for 12 hours after which they were frozen aboard the ISS MELFI at −80°C until returned to Earth on May 5, 2018, aboard the SpaceX CRS-14 cargo dragon. A similar experimental setup was established on the ground, for same duration, at the Kennedy Space Center in Florida in the Space Station Environmental Simulator (ISSES) chamber, USA. The samples were transported on dry ice from the dragon capsule to the receiving laboratories. Frozen plates containing both the spaceflight and ground samples were packed in dry ice and transported to Ohio University where they were stored at −80°C prior to protein extraction and processing while another group was also sent to the University of Florida, Gainesville for RNA extraction, library preparation and sequencing.</p></section><section id="s2_2"><h3 class="pmc_sec_title">2.2. RNA extraction and sequencing</h3>
<p>For each condition (Spaceflight and Ground), four individual plants from each replicate were pooled for RNA extraction. Individual plants were first dissected into leaves, hypocotyls, and root tissues under an Olympus dissecting microscope (Olympus, Japan). The intervening hypocotyls were set aside and not processed for downstream analyses. Leaf tissues without their hypocotyl are referred to herein as shoots and are the only ones processed for RNA analysis. RNA extraction was performed using the Qiagen RNeasy Plant Mini Kit (Qiagen, Germany) according to the manufacturer’s guidelines. RNA concentration was assessed using the Qubit 2.0 Fluorometer (ThermoFisher Scientific, USA), and RNA quality was determined on the Agilent 2100 Bioanalyzer (Agilent Technologies Inc., USA). For all samples, the RNA integrity number (RIN) was more than 6.</p>
<p>RNA-Sequencing Library Preparation was performed at the University of Florida Interdisciplinary Center for Biotechnology Research (ICBR) Gene Expression Core (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://biotech.ufl.edu/gene-expression-genotyping/" rel="noopener noreferrer" target="_blank">https://biotech.ufl.edu/gene-expression-genotyping/</a> RRID: SCR_019145). Here, 100 ng of total RNA was used for mRNA isolation using the NEBNext Poly(A) mRNA Magnetic Isolation Module (New England Biolabs, catalog #E7490). Next, RNA library construction was done with the NEBNext Ultra II Directional RNA Library Prep Kit (New England Biolabs, catalog #E7760) according to the manufacturer’s user guide. Briefly, RNA was fragmented in NEBNext First Strand Synthesis Buffer via incubation at 94°C for the desired time. This step was followed by first-strand cDNA synthesis using reverse transcriptase and random hexamer primer. Synthesis of ds-cDNA was performed using the 2nd strand master mix provided in the kit, followed by end-repair and adaptor ligation. At this point, Illumina adaptors were ligated to the sample. Finally, each library (uniquely barcoded) was enriched by 12 cycles of amplification, and purified with Agencourt AMPure beads (Beckman Coulter, catalog #A63881). The 12 barcoded libraries were sized on the Bioanalyzer and quantified with the Qubit<sup>®</sup> 2.0 Fluorometer. Finally, these 20 individual libraries were pooled in equimolar concentration.</p>
<p>The Illumina NovaSeq 6000 was used to sequence the libraries for 2 × 150 cycles. Sequencing was performed at the ICBR NextGen Sequencing (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://biotech.ufl.edu/next-gen-dna/" rel="noopener noreferrer" target="_blank">https://biotech.ufl.edu/next-gen-dna/</a>, RRID: SCR_019152). Normalized libraries were submitted to the “Free Adapter Blocking Reagent” protocol (FAB, catalog #20024145) to minimize the presence of adaptor-dimers and index hopping rates. The library pool was diluted to 0.8 nM and sequenced on one S4 flow cell lane (2 × 150 cycles) of the Illumina NovaSeq6000. The instrument’s computer utilized the NovaSeq Control Software v1.6. Cluster and SBS consumables were v1.5. The final loading concentration of the library was 120 pM with 1% PhiX (v/v) spike-in control. One lane generated 2.5–3 billion paired-end reads (~950Gb) with an average Q30%≥92.5% and Cluster PF=85.4%. FastQ files were generated using the BCL2fastQ function in the Illumina BaseSpace portal. One NovaSeq S4, 2 × 150 cycles lane resulted in an average of 50 million demultiplexed, paired-end reads when sequencing a pool of 48 samples.</p></section><section id="s2_3"><h3 class="pmc_sec_title">2.3. Protein extraction, sample processing and LC-MS/MS</h3>
<p>Seedlings from spaceflight and ground controls were divided into root and shoot for soluble and membrane protein extractions according to a protocol adapted from <a aria-describedby="B6" class="usa-link" href="#B6">Basu et al. (2015)</a>. Approximately 0.1g of tissue per replicate was crushed in liquid nitrogen and vortexed thoroughly in the extraction buffer. The samples were subsequently centrifuged at 8000g and 100,000g to isolate membrane protein. Both the precipitated soluble protein stored in methanol and the membrane protein in microsome resuspension buffer were sent to the Proteomics and Metabolomics Facility, at the University of Nebraska, Lincoln for 6-plex TMT labeling and mass spectrometric analysis (<a class="usa-link" href="#SM3">
<strong>Supplementary protocol</strong>
</a>).</p></section><section id="s2_4"><h3 class="pmc_sec_title">2.4. Data analysis</h3>
<p>RNA read pairs for each biological replicate for the flight and ground control samples were checked for sequence quality using the FastQC, trimmed using the TrimGalore, and aligned to TAIR10 assembly using the Spliced Transcripts Alignment to a Reference (STAR) (<a aria-describedby="B30" class="usa-link" href="#B30">Kruse et al., 2020</a>). Determination of gene-level read counts was accomplished using the Feature-count tool while the differential expression was determined using the negative binomial generalized linear model likelihood tests with the DESeq2 package in R. LC-MS/MS peptide data were analyzed using Proteome Discoverer 2.4 software (ThermoFisher Scientific, USA) using the Sequest HT search engine. Peptides were validated by Percolator with a 0.01 posterior error probability (PEP) threshold (<a aria-describedby="B46" class="usa-link" href="#B46">Rioseras et al., 2018</a>). The data were searched using a decoy database to set the false discovery rate to 1% (high confidence). Protein quantification was processed using the co-isolation threshold set to 50% and the average S/N to 10, and peptides were quantified using the peak intensity of the reporter ions in the MS3 spectra. The peak abundance was normalized for differences in sample loading using total peptide amount where the peptide group abundances are summed for each sample and the maximum sum across all runs is determined. The protein ratios and log2 fold changes (L<sub>2</sub>FC) were calculated from the grouped protein abundances. RNA transcripts and Proteins with adjusted <em>p</em>-value &lt;0.05 were considered statistically significant while thresholds of L<sub>2</sub>FC≥1 or ≤−1 and L<sub>2</sub>FC≥0.2 or ≤−0.2 were established for the RNA and protein respectively to decide differential abundance between spaceflight and earth.</p></section><section id="s2_5"><h3 class="pmc_sec_title">2.5. Analysis of the gene ontology enrichment</h3>
<p>Analysis of the gene ontology (GO) enrichment was performed using the ShinyGO V0.77 (based on Ensembl Release 107 with revision, archived on April 4, 2022) (<a aria-describedby="B13" class="usa-link" href="#B13">Ge et al., 2020</a>) and confirmed using the DAVID database (<a aria-describedby="B48" class="usa-link" href="#B48">Sherman et al., 2022</a>). Only differentially abundant (<em>p</em>&lt;0.05) RNA and proteins with significant abundance in spaceflight (L<sub>2</sub>FC≥1 and L<sub>2</sub>FC≥0.2 respectively) or significant abundance on earth (L<sub>2</sub>FC≤−1 and L<sub>2</sub>FC≤−0.2 respectively) were analyzed for fold enrichment. The GO term filter condition was set at a false discovery rate (FDR) of 0.01 and <em>p</em>-values (ratio of total genes involved in a pathway found in the dataset to total genes in the pathway) were adjusted by the Benjamini–Hochberg (B–H) method to reduce false positives. Hence, poorly represented annotated terms and overly broad terms in the protein dataset were eliminated. Likewise, a comparison was drawn between the araPath enrichment terms (<a aria-describedby="B32" class="usa-link" href="#B32">Lai et al., 2012</a>) and STRINGdb (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.https://string-db.org/" rel="noopener noreferrer" target="_blank">www.https://string-db.org/</a>) terms in the dataset to ensure we get an accurate overview of the GO enrichment. In cases where we have similar enrichment terms, only the most informative terms, with fewer similarities and the highest enrichment value were retained. Visual representations of the resulting enrichment terms were created using the graph tools on ShinyGO and the “ggplot”, “GOsummaries” and “Heatmap” packages in RStudio (<a aria-describedby="B45" class="usa-link" href="#B45">Posit team, 2023</a>).</p></section><section id="s2_6"><h3 class="pmc_sec_title">2.6. Confirmation of subcellular localization</h3>
<p>The subcellular locations of differentially abundant (<em>p</em>&lt;0.05) RNA and proteins in spaceflight and on the ground were analyzed using the Subcellular Localization Database for Arabidopsis Proteins (SUBA4, <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://suba.live" rel="noopener noreferrer" target="_blank">http://suba.live</a>, accessed May 2023). Specific protein consensus subcellular localizations were decided using the SUBAcon (<a aria-describedby="B17" class="usa-link" href="#B17">Hooper et al., 2014</a>), a consensus algorithm for unifying the subcellular localization data of the Arabidopsis proteome. The resulting subcellular localizations were plotted on a sunburst chart.</p></section></section><section id="s3"><h2 class="pmc_sec_title">3. Results</h2>
<section id="s3_1"><h3 class="pmc_sec_title">3.1. Transcriptomics and proteomics recovery in the BRIC LED experiment</h3>
<p>The differential analysis of the transcriptome expressed between spaceflight and ground control Arabidopsis samples yielded 15,505 distinctly mapped RNA transcripts, with 2,474 being statistically significant at adjusted <em>p</em>&lt;0.05. Of this, 195 RNA are upregulated in spaceflight at L<sub>2</sub>FC<sub>RNA</sub>≥1, while 761 are downregulated in spaceflight at L<sub>2</sub>FC<sub>RNA</sub>≤−1 (<a class="usa-link" href="#f1">
<strong>Figure 1A</strong>
</a>). Protein extraction from the different organs (root and shoot) and across the cellular fractions (soluble and membrane) led to the quantification, in the root, of 20,923 peptides mapped to 3973 membrane proteins and 18,636 peptides mapped to 3412 soluble proteins. Similar to this, in the shoot, 20,658 peptides were mapped to 3965 membrane proteins and 20,091 peptides were mapped to 3537 soluble proteins. Only proteins with a FDR of 0.01 and a minimum of two uniquely mapped non-nested peptides (amino acid length &gt;6 were retained. Proteins with B–H adjusted <em>p</em>≤0.05 were identified as statistically significant in the dataset. A total of 2187 membrane proteins and 332 soluble proteins were statistically significant in the root, while 1856 membrane proteins and 476 soluble proteins were statistically significant in the shoot. At the decision thresholds of L<sub>2</sub>FC<sub>protein</sub>≥0.2 and L<sub>2</sub>FC<sub>protein</sub>≤−0.2, 821 membrane proteins and 135 soluble proteins in the root were more abundant in spaceflight, whereas 1199 membrane proteins and 162 soluble proteins were less abundant in spaceflight (<a class="usa-link" href="#f1">
<strong>Figure 1A</strong>
</a>). Likewise, in the shoot, 1031 and 270 membrane and soluble proteins respectively were less abundant in spaceflight compared to ground control, while 778 membrane proteins and 172 soluble proteins were more abundant in spaceflight compared to ground control (<a class="usa-link" href="#f1">
<strong>Figure 1A</strong>
</a>).</p>
<figure class="fig xbox font-sm" id="f1"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10712242_fpls-14-1260429-g001.jpg" target="_blank"><img alt="Figure 1" class="graphic zoom-in" height="466" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/897a/10712242/5cd91b9d99a1/fpls-14-1260429-g001.jpg" width="769"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/f1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Transcriptome and proteome recovery in BRIC LED experiment. <strong>(A)</strong>. Bar plots of the RNA and protein abundance distribution in spaceflight and in ground control. On the protein plots, solid-colored bars represent the membrane proteins while hashed bars represent the soluble protein. The quantity of differentially abundant RNA and proteins (<em>p</em>&lt;0.05) is indicated in parentheses. RNA and proteins with L<sub>2</sub>FC<sub>RNA</sub>≥1 and L<sub>2</sub>FC<sub>protein</sub>≥0.2 respectively were considered abundant in spaceflight while those with L<sub>2</sub>FC<sub>RNA</sub>≤−1 and L<sub>2</sub>FC<sub>protein</sub>≤−0.2 respectively were considered less abundant in space. <strong>(B)</strong>. Volcano plot of the total abundance and distribution of RNA and proteins between spaceflight and ground tissues. The negative log of the adjusted <em>p</em>-values was plotted on the y-axis against the normalized log2 fold change on the x-axis. Dots represent RNA and protein abundance with black dots representing non-significantly expressed proteins (<em>p&gt;0.05</em> and those with L<sub>2</sub>FC not within the threshold), red dots represent less abundant RNA and proteins in spaceflight while blue dots represent proteins that were more abundant in spaceflight than on earth. <strong>(C)</strong>. Venn diagram of differential protein abundance in spaceflight. Combination of differential proteins more abundant in spaceflight and less abundant in spaceflight, as distinct to each organ and as shared by both organs. Shoot UP indicates shoot proteins more abundant in spaceflight, shoot DOWN indicates shoot proteins less abundant in spaceflight, Root UP indicates root proteins more abundant in spaceflight while root DOWN indicates root proteins less abundant in spaceflight.</p></figcaption></figure><p>The volcano plot (<a class="usa-link" href="#f1">
<strong>Figure 1B</strong>
</a>) of the total mapped RNA and protein across the organs and cell fractions gives a broader view of the pattern of abundance of transcriptome and proteome in spaceflight. Applying the widely accepted threshold for RNA tier II analysis, L<sub>2</sub>FC<sub>RNA</sub>≥1 or L<sub>2</sub>FC<sub>RNA</sub>≤−1, reduces false positives in interpreting the biological significance of the differentially expressed transcripts (<a aria-describedby="B36" class="usa-link" href="#B36">McDermaid et al., 2019</a>). However, this also led to the exclusion of over 61% of our total statistically significant transcripts (<a class="usa-link" href="#f1">
<strong>Figure 1B</strong>
</a>), with the possibility of the exclusion of some transcripts with biological significance. Membrane proteins were more differentially abundant (<em>p</em>&lt;0.05) and highly perturbed in spaceflight compared to the soluble proteins. Membrane proteins in the root had the largest abundance and fold enrichment, followed by the membrane and soluble proteins in the shoots respectively, while soluble proteins in the root were the least quantified and least differentially abundant (<a class="usa-link" href="#f1">
<strong>Figure 1B</strong>
</a>).</p>
<p>Several studies (<a aria-describedby="B47" class="usa-link" href="#B47">Sarasketa et al., 2016</a>; <a aria-describedby="B41" class="usa-link" href="#B41">Paponov et al., 2021</a>) have identified specificity in the proteomic profile of root and shoot in response to various environmental perturbations such as magnetic field, hormonal homeostasis, and gravity. This study further provides insight into how the specific plant organs (root and shoot) respond to microgravity in spaceflight. The organ-targeted proteomic profiling in response to microgravity, in this study, revealed that roots and shoots adapt differently to the spaceflight environment, and at the same time, they share certain commonalities. Comparing root-specific protein abundance in spaceflight to Earth, 400 membrane proteins and 75 soluble proteins were more abundant in space, whereas 606 membrane proteins and 71 soluble proteins were less abundant in spaceflight (<a class="usa-link" href="#f1">
<strong>Figure 1C</strong>
</a>). For proteins specific to the shoot, 357 membrane, and 112 soluble proteins were more abundant in spaceflight, whereas 438 membrane and 179 soluble proteins were less abundant in spaceflight (<a class="usa-link" href="#f1">
<strong>Figure 1C</strong>
</a>). Common to both root and shoot, 421 membrane proteins and 60 soluble proteins were more abundant in spaceflight compared to on Earth. Likewise, the abundance of 593 membrane proteins and 91 soluble proteins was reduced in spaceflight across both organs. Interestingly, a comparison of the differentially abundant proteins shared by both organs revealed that 18 membrane proteins were more abundant in spaceflight in the root and less abundant in the shoot, while 34 membrane proteins were more abundant in spaceflight in the shoot and less abundant in the root (<a class="usa-link" href="#f1">
<strong>Figure 1C</strong>
</a>, <a class="usa-link" href="#T1">
<strong>Table 1</strong>
</a>). A GO enrichment and KEGG pathway analysis of the 34 membrane proteins with increased abundance in the shoot and decreased abundance in the root in spaceflight revealed that they were majorly involved in the photosynthetic pathway, carbon metabolism pathway, and amide biosynthetic process. Likewise, these proteins were majorly chloroplast localized and are enriched for glyceraldehyde 3-phosphate dehydrogenase (NAD+) phosphorylating activity, poly(U) RNA binding, and oxidoreductase activity (<a class="usa-link" href="#SM1">
<strong>Supplementary Figure 1</strong>
</a>). KEGG pathway analysis of the 18 membrane proteins that were more abundant in spaceflight in the root and less abundant in the shoot did not indicate their involvements in a major metabolic pathway. However, some of these proteins were the heat shock proteins (HSP70, HSP90.4), host defense proteins such as the Importin (IMPA-4) and Pectinesterase inhibitor 17 (PME17), proteins involved in shunt energy processes such as the Transketolase (TLK2) and Electron Transfer flavoprotein (ETFQ). (<a class="usa-link" href="#T1">
<strong>Table 1</strong>
</a>).</p>
<section class="tw xbox font-sm" id="T1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Proteins differentially expressed between the root and shoot.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="2" valign="bottom">TAIR ID</th>
<th align="left" colspan="1" rowspan="2" valign="bottom">Protein name</th>
<th align="left" colspan="2" rowspan="1" valign="bottom">Root</th>
<th align="left" colspan="2" rowspan="1" valign="bottom">Shoot</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1" valign="bottom">L2FC</th>
<th align="left" colspan="1" rowspan="1" valign="bottom">
<em>p</em>-value</th>
<th align="left" colspan="1" rowspan="1" valign="bottom">L2FC</th>
<th align="left" colspan="1" rowspan="1" valign="bottom">
<em>p</em>-value</th>
</tr>
</thead>
<tbody>
<tr><th align="left" colspan="6" rowspan="1" valign="bottom">Upregulated in shoot, downregulated in root</th></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT1G04800</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Glycine-rich protein</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.34</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.029</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.97</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT1G13440</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Glyceraldehyde-3-ph</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.51</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.031</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.37</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT1G18080</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Receptor for activated C kinase 1A</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.23</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.036</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.26</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT1G21380</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">TOM1-like protein 3</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.27</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.022</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.95</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT1G44446</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Chlorophyllide a oxygenase</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.64</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.41</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT1G79530</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Glyceraldehyde-3-ph</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.22</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.017</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.56</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT2G02050</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">NADH dehydrogenase [ubiquinone] 1 beta subcomplex subunit 7</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.32</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.018</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.68</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT2G34420</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Chlorophyll a–b binding protein</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.39</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">3.04</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT2G34430</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Chlorophyll a–b binding protein</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.64</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.048</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">1.69</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT2G37040</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Phenylalanine ammonia-lyase 1</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.38</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.33</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT2G38040</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Acetyl-coenzyme A carboxylase carboxyl transferase subunit alpha</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.2</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.003</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.24</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT2G44120</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">60S ribosomal protein</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.39</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.011</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.37</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT2G45180</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Disease non-specific lipid transfer protein</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.35</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.012</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">1.47</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT2G47510</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Fumarate hydratase 1</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.27</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.033</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.35</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT3G13460</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Evolutionarily conserved C-terminal region 2</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.21</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.039</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.48</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT3G19480</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">D-3-phosphoglycerate dehydrogenase</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.27</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.44</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT3G25920</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">50S large ribosomal subunit</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.34</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.011</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.82</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT3G50820</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Oxygen-evolving enhancer protein 1–2</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.41</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.014</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.72</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT3G60770</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">40S ribosomal subunit</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.21</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.66</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT3G62530</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">ARM repeat superfamily protein</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.34</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.004</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.69</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT4G04020</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Probable plastid-lipid-associated protein 1</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.33</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.012</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.4</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT4G11420</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Eukaryotic translation initiation factor 3 subunit A</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.24</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.007</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.54</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT4G29480</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Mitochondrial ATP synthase subunit G protein</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.33</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.015</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">1.19</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT4G30720</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">FAD/NAD(P)-binding oxidoreductase family protein</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.3</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.009</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.53</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT4G34110</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Polyadenylate-binding protein 2</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.24</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.022</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.41</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G22880</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Histone H2B.10</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.27</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.017</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.25</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G46580</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Pentatricopeptide repeat-containing protein At5g46580</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.68</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.004</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.56</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G48030</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Chaperone protein dnaJ GFA2</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.55</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.013</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.96</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G48810</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Cytochrome B5 isoform D</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.32</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.021</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">1.38</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G53620</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">RNA polymerase II degradation factor</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.37</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.037</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.48</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G64110</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Peroxidase 70</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.34</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.46</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G64860</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">4-alpha-glucanotransferase DPE1</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.29</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.036</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.78</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G65010</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Asparagine synthetase [glutamine-hydrolyzing] 2</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.33</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.012</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.3</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">ATCG00790</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">50S large ribosomal subunit</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">−0.42</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.82</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr><th align="left" colspan="6" rowspan="1" valign="bottom">Downregulated in shoot, upregulated in root</th></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT1G09270</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Importin subunit alpha-4</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.25</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.49</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT1G11660</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Heat shock protein 70 (Hsp 70) family protein</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.2</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.039</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.21</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT1G24100</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">UDP-Glucosyl transferase 74B1</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.66</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.013</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.24</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT1G49730</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Probable receptor-like protein kinase At1g49730</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.31</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.026</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-1.4</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT1G55020</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Linoleate 9S-lipoxygenase 1</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.32</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.017</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.26</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT1G65820</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Microsomal gluthatione s-transferase</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.31</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.017</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.27</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT2G43400</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Electron transfer flavoprotein-ubiquinone oxidoreductase</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.58</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.006</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.6</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT2G45220</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Probable pectinesterase/pectinesterase inhibitor 17</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.87</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.00</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.28</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT2G45290</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Transketolase-2</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.23</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.045</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.81</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT3G22960</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Plastidial pyruvate kinase 1</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.29</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.013</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.45</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT4G25080</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Magnesium protoporphyrin IX methyltransferase</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.24</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.035</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.78</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT4G38690</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">PLC-like phosphodiesterases</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.28</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.017</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.41</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G01750</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Protein LURP-one-related 15</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.44</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.044</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.29</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G03300</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Adenosine kinase 2</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.45</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.035</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.44</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G23575</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Transmembrane CLPTM1 family protein</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.33</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.38</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G56000</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Heat shock protein 90-4</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.4</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.02</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.41</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G56590</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">O-Glycosyl hydrolases family 17 protein</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.31</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.36</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="bottom">AT5G59420</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">Oxysterol-binding protein-related protein 3C</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.35</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.003</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">-0.36</td>
<td align="left" colspan="1" rowspan="1" valign="bottom">0.03</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></section></section><section id="s3_2"><h3 class="pmc_sec_title">3.2. Subcellular distribution and abundance of protein in spaceflight</h3>
<p>Several metabolic processes in cells, such as signal transduction, transport of molecules, cell division, cell death, etc., are tightly linked with the location of specific transcripts and proteins (<a aria-describedby="B10" class="usa-link" href="#B10">Christopher et al., 2022</a>). Transcripts are targeted toward different subcellular organelles in the plant cell based on the specific functions of encoded proteins. Assessing which of the targeted organelles have the highest amount of perturbed proteins gives insight into the metabolic process most affected by the spaceflight environment. Our expectation is that since transcripts encode proteins that are targeted toward different organelles, the pattern of subcellular localization of the RNA transcripts differentially expressed should be similar to that of proteins differentially expressed in spaceflight. To ascertain this, we compared transcripts and proteins with differential abundance in our spaceflight experiments against SUBA4, and results were subsequently fed into SUBAcon (<a aria-describedby="B16" class="usa-link" href="#B16">Hooper et al., 2017</a>) to recover more accurate proteome-wide subcellular localizations.</p>
<p>Transcripts encoding nucleus-targeted proteins constituted the largest percentage of the transcripts upregulated (25%) and downregulated (28.5%) in spaceflight (<a class="usa-link" href="#f2">
<strong>Figure 2A</strong>
</a>). KEGG pathway analysis of the nucleus-targeted transcripts upregulated in spaceflight indicated that they are majorly involved in plant hormone signal transduction pathways which are tryptophan metabolism, zeatin biosynthesis, diterpenoid biosynthesis, and phenylalanine metabolism (<a class="usa-link" href="#SM1">
<strong>Supplementary Figure 2A</strong>
</a>). Transcripts encoding nucleus-targeted proteins downregulated in spaceflight are involved in protein processing in the endoplasmic reticulum and ribosome pathway, with many being part of the ubiquitin ligase complex and the ER-associated degradation complex (ERAD) (<a class="usa-link" href="#SM1">
<strong>Supplementary Figure 2B</strong>
</a>). Another interesting group of transcripts are those encoding proteins targeted towards extracellular matrix, being 20% of the total upregulated RNA and 11.1% of the total downregulated RNA. Upregulated transcripts encoding extracellular-matrix targeted proteins are enriched in cell wall biogenesis, cell wall macromolecule metabolic process, and flavonoid metabolic process, while the downregulated ones are uniquely enriched in pathways involved in response to salicylic acid, polysaccharide biosynthetic process, response to brassinosteroid and cell wall organization (<a class="usa-link" href="#f2">
<strong>Figure 2B</strong>
</a>). The metabolic pathways enriched were significantly different in the extracellular protein dataset in comparison with the RNA transcriptome (<a class="usa-link" href="#f2">
<strong>Figure 2B</strong>
</a>).</p>
<figure class="fig xbox font-sm" id="f2"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10712242_fpls-14-1260429-g002.jpg" target="_blank"><img alt="Figure 2" class="graphic zoom-in" height="571" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/897a/10712242/c799d9d13238/fpls-14-1260429-g002.jpg" width="708"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/f2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Subcellular localization of differentially abundant RNA and proteins in spaceflight. <strong>(A)</strong>. Bar chart comparing the percentage of RNA transcripts (hashed bars) and root (dotted bars) and shoot (black bars) proteins localized in each subcellular compartment. The subcellular localizations were identified using SUBA4 and localization consensus was determined using SUBAcon. <strong>(B)</strong>. GO enrichment heat map and hierarchical cluster tree of transcripts that encode extracellular matrix-targeted proteins and the proteins that are localized in the extracellular matrix. The color gradient of the heatmap indicates the significance of the genes and proteins in the enriched metabolic pathways. Labels with “Up” means upregulated in spaceflight and downregulated on Earth while labels with “Down” indicates downregulated in spaceflight and upregulated on Earth.</p></figcaption></figure><p>In the root, plastid localized proteins constituted 35.5% of soluble and membrane proteins with reduced abundance in spaceflight, while they constituted 10.3% of proteins more abundant in spaceflight than on earth (<a class="usa-link" href="#f2">
<strong>Figure 2A</strong>
</a>). Enrichment analysis of root plastid localized proteins that were differentially abundant in spaceflight indicated that proteins involved in plastid movements, organization of plastid membranes, and light harvesting were less abundant in spaceflight than on earth, while those involved in valine biosynthetic pathways, reductive pentose-phosphate cycle, and the dark reaction of photosynthesis were more abundant in spaceflight. The valine biosynthetic pathway is intertwined with the isoleucine biosynthetic pathway, and they both inhibit the growth of shoots (<a aria-describedby="B8" class="usa-link" href="#B8">Cao et al., 2019</a>) and increase root formation. Notably, 154 membrane and 51 soluble extracellular localized proteins were abundant in spaceflight in the root and they are majorly involved in glycan degradation, phenylpropanoid biosynthesis, and pentose-glucuronate interconversion. This is indicative of increased restructuring of the root cell wall polysaccharides in spaceflight.</p>
<p>The distribution of protein localization in the shoot is similar to the root, with plastid localized proteins constituting 32.4% of proteins less abundant in spaceflight, while cytosolic protein constituted 33.4% of the proteins more abundant in spaceflight (<a class="usa-link" href="#f2">
<strong>Figure 2A</strong>
</a>). However, enrichment analysis of shoot plastid localized proteins indicated high enrichment of proteins involved in plastid transcription and organization, chlorophyll metabolism, and photosynthesis. Contrary to the root, 83 membrane and 63 soluble extracellular proteins were more abundant in spaceflight and they are highly enriched in pathways for cellular oxidation regulation and negative regulation of peptidase activities. Extracellular proteins less abundant in spaceflight are enriched in pathways responsible for plant defense. An interesting observation as regards the comparative subcellular localization of transcriptome and root and shoot proteome in spaceflight is that most downregulated RNA in spaceflight encode proteins targeted back to the nucleus whereas most downregulated proteins in spaceflight are plastids localized. Cytosolic proteins are most upregulated in spaceflight, whereas transcripts encoding nucleus-targeted proteins are most upregulated (<a class="usa-link" href="#f2">
<strong>Figure 2A</strong>
</a>).</p></section><section id="s3_3"><h3 class="pmc_sec_title">3.3. Enrichment analysis of the gene ontology terms</h3>
<p>Creating a functional profile of transcriptome and proteome abundance in spaceflight elucidates various underlying biological, cellular and molecular processes affected by the harsh spaceflight environmental conditions such as microgravity, ionizing radiation and so on. Insight into the biological roles of these RNAs and proteins was acquired through enrichment analysis of the protein ontologies. A common feature in spaceflight, across the transcriptome and proteome landscapes, is the upregulation and enrichment of stress response proteins, especially those involved in response to light stimulus, radiation, oxygen, and water deprivation (<a class="usa-link" href="#f3">
<strong>Figure 3A</strong>
</a>). Enriched among the downregulated RNA and proteins are those involved in translation, amide biosynthetic process, mitochondrial and peroxisome fission (<a class="usa-link" href="#f3">
<strong>Figure 3A</strong>
</a>). A correlation plot of the expression (log2 fold change) of transcripts and the shoot and root proteins indicated very low gene expression correlation between shoot proteins and transcripts (<em>r</em>=−0.004) and between root proteins and transcripts (<em>r</em>=0.021). However, protein expression between the shoot and root is positively correlated (<em>r</em>=0.878) (<a class="usa-link" href="#f3">
<strong>Figure 3B</strong>
</a>). An overview of the gene ontology of the gene expression pattern between the transcripts and shoot and root proteins is represented by a heat map (<a class="usa-link" href="#f3">
<strong>Figure 3C</strong>
</a>).</p>
<figure class="fig xbox font-sm" id="f3"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10712242_fpls-14-1260429-g003.jpg" target="_blank"><img alt="Figure 3" class="graphic zoom-in" height="968" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/897a/10712242/71e160a45bcb/fpls-14-1260429-g003.jpg" width="739"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/f3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Gene ontology enrichment of RNA and proteins differentially expressed in spaceflight <strong>(A)</strong>. Network interaction map of RNA and proteins with a similar pattern of expression in spaceflight. Upregulated transcripts with upregulated proteins are above, while downregulated transcripts with downregulated proteins are below. The node sizes indicate the number of genes enriched in that pathway while two pathways (nodes) are connected only if they share a minimum of 20% genes. GO terms enrichment analyses were conducted in ShinyGO 0.77 (GO terms were accessed in June 2023). <strong>(B)</strong>. Correlation graph of the log2 fold change of shoot and root proteins with total RNA transcripts. The correlations were plotted separately with black triangles representing the correlation of the log2 fold change of proteins in the root and the log2 fold change of the plant total RNA (root vs total RNA) with a correlation coefficient, <em>r<sup>2</sup>=</em>0.021, and circles representing the correlation of the log2 fold change of proteins in the shoot and the log2 fold change of the plant total RNA (shoot vs total RNA) with a correlation coefficient<em>, r<sup>2</sup>
</em>=−0.004. The plot of the correlation of the log2 fold change of shared proteins in the root and shoot (Root vs Shoot) is not shown on the graph. However, its correlation coefficient (<em>r<sup>2</sup>=</em>0.878) is displayed. <strong>(C)</strong>. Heatmap of the GO enrichment of gene expression patterns in RNA, shoot, and root proteins. The corresponding GO terms are listed to the right of the heat map. The color gradients represent the Z-score calculated from the L<sub>2</sub>FC, with blue as downregulated and red as upregulated. Both row and column hierarchical clusters indicate the similarities of the categories.</p></figcaption></figure><p>Membrane and soluble proteins involved in the activation of protein kinase activity, regulation of microtubule depolymerization, and cellular detoxification were abundant and notably enriched in root and shoot (<a class="usa-link" href="#f3">
<strong>Figure 3C</strong>
</a>). Another important observation among proteins less abundant in spaceflight is the enrichment of transport proteins, cargo sorting proteins and vesicle loading proteins (<a class="usa-link" href="#SM2">
<strong>Supplementary Table A</strong>
</a>). Proteins involved in the negative regulation of the cytoskeleton were abundant and highly enriched in spaceflight in both organs (<a class="usa-link" href="#SM1">
<strong>Supplementary Figures 3, 4</strong>
</a>). It is unclear whether this was a response to spaceflight or a regulator of the response.</p>
<p>Specific to the root, proteins involved in DNA topological change, amino acid activation, and regulation of fiber organization were enriched among proteins more abundant in spaceflight while those involved in photoinhibition, cytoskeleton organization, and reactive oxygen species metabolic process were enriched among proteins less abundant in spaceflight (<a class="usa-link" href="#f3">
<strong>Figure 3C</strong>
</a>). Soluble proteins involved in mitochondrial elongation and the glyoxylate cycle were highly enriched in spaceflight (<a class="usa-link" href="#SM1">
<strong>Supplementary Figure 5</strong>
</a>) suggesting the compensation of photosynthetic ATP production deficit by increasing metabolic fluxes through the glyoxylate cycle. Further analysis of the photosystems’ proteome supports this suggestion. In the shoot, a large proportion of the enriched proteins are involved in nucleotide metabolic processes, protein synthesis, and localization to several destinations (<a class="usa-link" href="#SM1">
<strong>Supplementary Figure 5</strong>
</a>).</p></section><section id="s3_4"><h3 class="pmc_sec_title">3.4. Alterations to the photosystems transcriptome and proteome in the spaceflight environment</h3>
<p>The presence of customizable discrete lighting in the BRIC LED hardware led to the enrichment of several RNA transcripts and proteins involved in the photosynthetic pathways (<a class="usa-link" href="#f4">
<strong>Figure 4</strong>
</a>, <a class="usa-link" href="#SM2">
<strong>Supplementary Table B</strong>
</a>). Interestingly, all the photosystem RNA transcripts differentially expressed in spaceflight were upregulated, whereas most of them became less abundant in both the shoot and root proteomes (<a class="usa-link" href="#f4">
<strong>Figure 4</strong>
</a>). This is indicative of post-transcriptional regulation of the photosystem transcripts suggestive of a suppressive adaptation mechanism to reduce light harvesting ability of the photosystems.</p>
<figure class="fig xbox font-sm" id="f4"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10712242_fpls-14-1260429-g004.jpg" target="_blank"><img alt="Figure 4" class="graphic zoom-in" height="523" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/897a/10712242/2eea31cf51eb/fpls-14-1260429-g004.jpg" width="790"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/f4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Abundance of photosystem RNA transcripts and proteins in spaceflight. The differential abundance of photosynthetic RNA transcript (top) proteins in the root (middle) and shoot (bottom) as determined by their Log2 fold change. The orange color indicates RNA transcripts and proteins that are more abundant in spaceflight, while the blue color indicates RNA transcripts and proteins that are less abundant in spaceflight. Uncolored boxes indicate RNA transcripts and proteins that were not significantly abundant in spaceflight and on Earth. This figure was developed by the KEGG path view. Supporting data can be found in the <a class="usa-link" href="#SM2">
<strong>Supplementary Table B</strong>
</a>.</p></figcaption></figure><p>Common in both organs and more abundant in spaceflight than on earth are PHOTOSYSTEM II PROTEIN D1 (PSBA L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.56, 1.02), a complex that absorbs PSII photoinhibition damages and facilitates the repair of photosystem II, and PPL1 (L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.61, 1.6) a member of the oxygen-evolving complex. Likewise, ATP SYNTHASE EPSILON CHAIN (ATPBE, L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.51, 0.51) was more abundant in spaceflight than on Earth in both organs (<a class="usa-link" href="#f4">
<strong>Figure 4</strong>
</a>). The Epsilon subunit of the F-ATPase unit selectively inhibits ATP hydrolysis. This leads to a reduced yield of ATP from photosynthesis. Of the proteins involved in the binding of ferredoxin within the reaction center, only PHOTOSYSTEM, I REACTION CENTER SUBUNIT II-1 (PSAD, L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.35, 1.69) was abundant in spaceflight in both organs, while the other member of the complex were less abundant in spaceflight.</p>
<p>Less abundant in spaceflight in both organs are the α-subunit of CYTOCHROME B559 (PSBE, L<sub>2</sub>FC<sub>Root, Shoot</sub>=−1.37, −1.81) which reduces susceptibility to high irradiance and PHOTOSYSTEM II CP43 REACTION CENTER PROTEINs (PSBC, L<sub>2</sub>FC<sub>Root, Shoot</sub>=−0.52, −0.64, PSBB, L<sub>2</sub>FC<sub>Root, Shoot</sub>=−0.72, −1.20) which serve as the proximal antennae for Photosystem II. Arabidopsis <em>psbe</em> mutants show high susceptibility to strong light damage and reduced plastoquinone pool (<a aria-describedby="B14" class="usa-link" href="#B14">Giardi et al., 2013</a>). The decreased abundance of CYTOCHROME F (PETA, L<sub>2</sub>FC<sub>Root, Shoot</sub>=−0.77, −1.18) in the cytochrome b6/f complex in both organs suggests the exposure of the PSI to high electron influx from the PSII. The FERREDOXIN-NADP(+) OXIDOREDUCTASE-LIKE PROTEIN (PETH, L<sub>2</sub>FC<sub>Root, Shoot</sub>=−0.2, −0.29), an FNR component of the electron transport in photosystem I and the ATP SYNTHASE subunits (ATPA, L<sub>2</sub>FC<sub>Root, Shoot</sub>=−0.27, −0.24, ATPB, L<sub>2</sub>FC<sub>Root, Shoot</sub>=−0.73, −0.8, ATPC1, L<sub>2</sub>FC<sub>Root, Shoot</sub>=−0.49, −0.77) were also less abundant in spaceflight in both organs (<a class="usa-link" href="#f4">
<strong>Figure 4</strong>
</a>, <a class="usa-link" href="#SM2">
<strong>Supplementary Table B</strong>
</a>). The F-type ATPase subunit drives ATP synthesis, using proton gradient, by allowing passive flux of protons across the membrane, and their reduced abundance in spaceflight coupled with increased ATP hydrolysis inhibition suggests decreased energy production via photosynthesis in spaceflight compared to earth. Notably specific to the root and more abundant in spaceflight are PSBD (L<sub>2</sub>FC<sub>Root</sub>=0.25) and PSBO1 (L<sub>2</sub>FC<sub>Root</sub>=0.5) which are mediators of oxygen evolution (<a aria-describedby="B1" class="usa-link" href="#B1">Antonacci et al., 2018</a>). The CYTOCHROME B6-F COMPLEX IRON-SULFUR SUBUNIT (PET C, L<sub>2</sub>FC<sub>Shoot</sub>=−0.64), a mediator of electron transfer from photosystem II to photosystem I was less abundant in spaceflight and specific to only the shoot (<a class="usa-link" href="#f4">
<strong>Figure 4</strong>
</a>). PSAD, PSBA and oxygen evolution complex proteins (PSBO, PPL1 and PSBP) had a significantly higher log2-fold increase in the shoot than in the root (<a class="usa-link" href="#f4">
<strong>Figure 4</strong>
</a>, <a class="usa-link" href="#SM2">
<strong>Supplementary Table B</strong>
</a>), suggesting higher photosynthetic stress in the shoot than root despite equal exposure to light in the BRIC LED hardware.</p></section><section id="s3_5"><h3 class="pmc_sec_title">3.5. Spaceflight alters ionic, molecular, and vesicular transports in both organs</h3>
<p>Several signaling molecules are delivered to target sites via translocation across different cellular membranes, with most ions moving passively through gated channels and molecules via active transports (<a aria-describedby="B59" class="usa-link" href="#B59">Yang and Hinner, 2015</a>). Hence, we decided to assess how the spaceflight environment affected the intracellular transport transcriptome and proteome landscape in Arabidopsis. The RNA transcript and different plant organ proteome had profoundly distinct distributions of upregulated and downregulated intracellular transporters in spaceflight (<a class="usa-link" href="#f5">
<strong>Figure 5</strong>
</a>). More ionic, molecular, and vesicular transport pathways were upregulated in spaceflight compared to the downregulated. Common to all the molecular fractions are RNA transcripts and proteins of auxin polar transporters. Likewise, RNA transcripts and proteins involved in the transport of reactive oxygen species (ROS), lipids, water, ions, sucrose, and vesicles were upregulated in spaceflight (<a class="usa-link" href="#f5">
<strong>Figure 5A</strong>
</a>).</p>
<figure class="fig xbox font-sm" id="f5"><h4 class="obj_head">Figure 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10712242_fpls-14-1260429-g005.jpg" target="_blank"><img alt="Figure 5" class="graphic zoom-in" height="589" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/897a/10712242/74d141db48a6/fpls-14-1260429-g005.jpg" width="708"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/f5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Spaceflight enrichment of RNA transcripts and proteins involved in cellular transport. <strong>(A)</strong>. Upregulated RNA and protein in spaceflight (top), downregulated RNA and protein in spaceflight (bottom). RNA transcripts are represented by hashed bars, shoot proteins by black bars and root proteins by dotted bars. <strong>(B)</strong>. Hierarchical clustering tree of enriched RNA transcripts and protein involved in auxin polar transport. The cluster tree for the upregulated auxin transporter term and downregulated auxin transporter term is bottom. Bigger dots indicate more significant <em>p</em>-values. The biological functional characterization was conducted in ShinyGO 0.77, with redundant and/or repeated GO-terms eliminated across the dataset. In the case of overlapping functional groups, the functionality with the highest enrichment is selected.</p></figcaption></figure><section id="s3_5_1"><h4 class="pmc_sec_title">3.5.1. Ionic transport proteins</h4>
<p>Proteins involved in ionic transport were more abundant in spaceflight across both organs (<a class="usa-link" href="#f5">
<strong>Figure 5A</strong>
</a>, <a class="usa-link" href="#SM2">
<strong>Supplementary Table A</strong>
</a>). The abundance of proton and calcium-pump ATPases such as the H<sup>+</sup>ATPase, (AHA11, L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.36, 0.99), V-TYPE PROTON PUMPS (VHAB3, VHA-A2, VHA-D, VHA-F, VHA-G2, VHA-B2 and VHA-C3) and CALCIUM-TRANSPORTING ATPASES (ACA2, ACA4, ACA8) signifies an increased effort towards maintaining cellular homeostasis in spaceflight. The calcium-transporting ATPases regulate intracellular Ca<sup>2+</sup> levels and are essential for adaptation to stressful environments. The SODIUM/CALCIUM EXCHANGER (NCL, L<sub>2</sub>FC<sub>Root</sub>=0.36), also involved in the maintenance of Ca<sup>2+</sup> homeostasis and the FERRITINS (FER1, FER3, FER4) were abundant in spaceflight. Ferritins provide the cell with the optimal iron level while maintaining its bioavailability for physiological conditions. The abundance of ferritin in spaceflight suggests a high requirement for iron, either as a form of defense or to protect the DNA against oxidative damage as a result of the abundance of ROS.</p>
<p>The organ specificity of some cation transporters is noteworthy (<a class="usa-link" href="#SM2">
<strong>Supplementary Table A</strong>
</a> Cation transporters such as CHLORIDE CHANNEL PROTEIN CLC-A (CLC-A, L<sub>2</sub>FC<sub>Root</sub>=0.2), and POTASSIUM TRANSPORTER 13 (POT13, L<sub>2</sub>FC<sub>Root</sub>=0.37) were root-specific. Cation transporters were more abundant in the root in spaceflight than in the shoot (<a class="usa-link" href="#SM2">
<strong>Supplementary Table A</strong>
</a>). This might be indicative of more homeostatic needs in the root than shoot in microgravity. The VACOULAR IRON TRANSPORTER PROTEIN (MEB1, L<sub>2</sub>FC<sub>Root, Shoot</sub>=1.4, −0.5) was more abundant in the root but less abundant in the shoot in spaceflight. This further confirms more iron requirements by the root in spaceflight. The CASPARIN STRIP PROTEIN (L<sub>2</sub>FC<sub>Root, Shoot</sub>=−0.62, −0.71) was also less abundant in spaceflight. CASPs are thought to direct transport across the plasma membrane by forming a protein scaffold and they also play a role in plant cell wall lignification (<a aria-describedby="B25" class="usa-link" href="#B25">Kolbeck et al., 2022</a>). Eleven of the Ras-related family proteins (RABs) were abundant in spaceflight in an organ-specific manner. These small GTP binding proteins function in targeting and tethering transport vesicles to specific acceptor compartments. The RAN GTPASE-ACTIVATING PROTEIN (RANGAF, L<sub>2</sub>FC<sub>Shoot</sub>=−0.42) was less abundant in spaceflight, while two proteins that bind to Ran GTPase homolog (RANBP1B, RANBP1C) were abundant in spaceflight than on earth (<a class="usa-link" href="#SM2">
<strong>Supplementary Table A</strong>
</a>).</p></section><section id="s3_5_2"><h4 class="pmc_sec_title">3.5.2. Vesicular transport</h4>
<p>Proteins involved in Golgi and Intra-golgi vesicular transport were more enriched on Earth than in spaceflight (<a class="usa-link" href="#SM2">
<strong>Supplementary Table A</strong>
</a>). The majority of the VACUOLAR-SORTING RECEPTORS (VSR1, VSR2, VSR3, VSR4) with the exception of VSR7 (L<sub>2</sub>FC<sub>Root, Shoot</sub>=1.4, −0.5) were less abundant in spaceflight. The abundance of Ca<sup>2+</sup> channel proteins in spaceflight is expected to lead to an abundance of VSRs as cargo binding to VSRs is enhanced by high Ca<sup>2+</sup> concentration (<a aria-describedby="B23" class="usa-link" href="#B23">Kang and Hwang, 2014</a>), but this was not so. The decreased abundance of CLATHRINE HEAVY CHAIN PROTEIN (CHC2, L<sub>2</sub>FC<sub>Root, Shoot</sub>=−0.43, −0.41) in spaceflight suggests a decreased endocytosis and vesicle formation in both organs. SYNTAXIN 61 (SYP61, L<sub>2</sub>FC<sub>Root</sub>=−0.33), SYNTAXIN 81 (SYP81, L<sub>2</sub>FC<sub>Shoot</sub>=−0.57), SYNTAXIN 43 (SYP43, L<sub>2</sub>FC<sub>Root</sub>=−0.57), SYNTAXIN 22 (SYP22, L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.46, 1.48) and SYNTAXIN 132 (SYP132, L<sub>2</sub>FC<sub>Root</sub>= 0.34), member of SNARE proteins, were organ-specific in their abundance in spaceflight. The NOVEL PLANT SNARE 11 (NPSN11, L<sub>2</sub>FC<sub>Root, Shoot</sub>=1.27, 1.14) was also abundant in both organs in space (<a class="usa-link" href="#SM2">
<strong>Supplementary Table A</strong>
</a>). Synaptotagmins (SYTA L<sub>2</sub>FC<sub>Root</sub>=0.29), a calcium sensor that regulates rapid and synchronous vesicle exocytosis for plant viral movement and a requirement for tethering the ER to the plasma membrane, was also abundant in space.</p></section><section id="s3_5_3"><h4 class="pmc_sec_title">3.5.3. Lipid and hormone transport</h4>
<p>RNA transcripts and proteins involved in the intracellular transportation of lipids were more enriched and abundant in spaceflight than on Earth (<a class="usa-link" href="#f5">
<strong>Figure 5</strong>
</a>). GLYCOLIPID TRANSFER PROTEIN (GTP, L<sub>2</sub>FC<sub>Root</sub>=0.32) and ACCELERATED CELL DEATH 11 (ACD11, L<sub>2</sub>FC<sub>Shoot</sub>=0.32), involved in sphingolipid transport were abundant in spaceflight in both root and shoot respectively. The abundance of NON-SPECIFIC LIPID-TRANSFER PROTEINS (LTP1, LTP5, LTPG31) and PROTEIN FATTY ACID EXPORT 3 (FAX3, L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.32, 0.66) in spaceflight suggests the use of long-distance systemic signaling in space and the formation of lipid polymer barriers. The PHOSPHOLIPID-TRANSPORTING ATPASE 10 (ALA10, L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.69, 0.55), a lipid flippase, was abundant in spaceflight. ALA10 is involved in the desaturation process of phosphatidylcholine, a lipid abundant during phosphate starvation and in the regulation of auxin polar transport. The abundance of PATELLINs (PATL2, PATL4, PATL6), regulators of auxin-mediated PIN1 relocation, and AUXIN EFFLUX CARRIERS 3 (PIN3, L<sub>2</sub>FC<sub>Root</sub>=0.49) in spaceflight and the decrease in abundance of the PIN4 (<sub>L2FC Root/Shoot</sub>=−0.46, −0.75), PIN7(<sub>L2FC Root/Shoot</sub>=−0.27, −0.84), AUXIN RESPONSE 4 (AXR4, L<sub>2</sub>FC<sub>Root, Shoot</sub>=−0.34, −0.47), AUXIN TRANSPORTER PROTEIN1 (LAX1, L<sub>2</sub>FC<sub>Shoot</sub>=−0.74), ABCB1 (L<sub>2</sub>FC<sub>Shoot</sub>=−0.21) and ABCB20 (L<sub>2</sub>FC<sub>Root, Shoot</sub>=−0.35, −0.65) suggests an alteration in polar auxin redistribution in plants grown in spaceflight (<a class="usa-link" href="#f5">
<strong>Figure 5B</strong>
</a>). The MYOSINs (XIB, ATM1, XI-H, XI-K, MYA2), which are motor proteins that drive movement along the actin filaments were not significantly abundant (<em>p</em>&lt;0.05) in the shoot in spaceflight but were significantly less abundant in the root. The expression of auxin transporters was organ-specific and this suggested that the distribution of auxin is altered differently in the root and shoot for adaptation to the space environment (<a class="usa-link" href="#SM2">
<strong>Supplementary Table A</strong>
</a>).</p></section></section><section id="s3_6"><h3 class="pmc_sec_title">3.6. Influence of spaceflight on proteins involved in cell wall synthesis, integrity and maintenance</h3>
<p>Altered gravity has been identified to have profound effects on the architecture of plant cell walls (<a aria-describedby="B26" class="usa-link" href="#B26">Kordyum and Chapman, 2017</a>). Common to both organs and more abundant in spaceflight than on Earth are Endoglucanase 20 (L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.86, 1.13), involved in the degradation of cellulose to its monomeric subunit, FASCICLIN-LIKE ARABINOGALACTAN PROTEINs (FLA 7, L<sub>2</sub>FC<sub>Root, Shoot</sub>=1.51, 1.43, and FLA 9, L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.73, 0.53) and PLASMA MEMBRANE-ASSOCIATED CATION-BINDING PROTEINs (PCAP1, L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.49, 0.52 and PCAP2, L<sub>2</sub>FC<sub>Root, Shoot</sub>=1.08, 1.47) which are have been implicated in microtubule destabilization and identified as possible regulators of phosphatidylinositol phosphates membrane reformations during intracellular signaling (<a class="usa-link" href="#T2">
<strong>Table 2</strong>
</a>). VILLINs (VLN2, VLN3, VLN4), proteins involved in the bundling, severing, nucleating, and capping of the actin cytoskeleton, were also more abundant in spaceflight than on Earth, in both organs (<a class="usa-link" href="#T2">
<strong>Table 2</strong>
</a>). Spaceflight alters the biogenesis of microtubules and this is achieved differently in the root and shoot.</p>
<section class="tw xbox font-sm" id="T2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Differentially abundant proteins related to cell wall biosynthesis and maintenance.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="2" valign="middle">TAIR.ID</th>
<th align="left" colspan="1" rowspan="2" valign="middle">Protein ID</th>
<th align="left" colspan="1" rowspan="2" valign="middle">Symbol</th>
<th align="left" colspan="1" rowspan="1" valign="middle">Root</th>
<th align="left" colspan="1" rowspan="1" valign="middle">Shoot</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1" valign="middle">L<sub>2</sub>FC</th>
<th align="left" colspan="1" rowspan="1" valign="middle">L<sub>2</sub>FC</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT4G23560</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Endoglucanase 20</td>
<td align="left" colspan="1" rowspan="1" valign="middle"></td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.86</td>
<td align="left" colspan="1" rowspan="1" valign="middle">1.13</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G49360</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Beta-D-xylosidase 1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">BXL1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.48</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT4G24510</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Protein ECERIFERUM 2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CER2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.68</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT4G32410</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Cellulose synthase A catalytic subunit 1 [UDP-forming]</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CESA1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.44</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G05170</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Cellulose synthase A catalytic subunit 3 [UDP-forming]</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CESA3</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.44</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G09870</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Cellulose synthase A catalytic subunit 5 [UDP-forming]</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CESA5</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.52</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G64740</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Cellulose synthase A catalytic subunit 6 [UDP-forming]</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CESA6</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.51</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.83</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G17420</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Cellulose synthase A catalytic subunit 7 [UDP-forming]</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CESA7</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−2.37</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G55850</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Cellulose synthase-like protein E1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CSLE1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.22</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.33</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT4G23990</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Cellulose synthase-like protein G3</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CSLG3</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.51</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT2G04780</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Fasciclin-like arabinogalactan protein 7</td>
<td align="left" colspan="1" rowspan="1" valign="middle">FLA7</td>
<td align="left" colspan="1" rowspan="1" valign="middle">1.51</td>
<td align="left" colspan="1" rowspan="1" valign="middle">1.43</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G03870</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Fasciclin-like arabinogalactan protein 9</td>
<td align="left" colspan="1" rowspan="1" valign="middle">FLA9</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.73</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.57</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT2G36250</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Tubulin/FtsZ family protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">FTSZ2-1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.36</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.42</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G06680</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Gamma-tubulin complex component 3</td>
<td align="left" colspan="1" rowspan="1" valign="middle">GCP3</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.25</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.45</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G01640</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Glucuronokinase 1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">GLCAK1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.28</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT4G20260</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Plasma membrane-associated cation-binding protein 1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">PCAP1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.49</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.52</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G44610</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Plasma membrane-associated cation-binding protein 2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">PCAP2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">1.08</td>
<td align="left" colspan="1" rowspan="1" valign="middle">1.47</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G02230</td>
<td align="left" colspan="1" rowspan="1" valign="middle">UDP-arabinopyranose mutase 1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">RGP1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.22</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G15650</td>
<td align="left" colspan="1" rowspan="1" valign="middle">UDP-arabinopyranose mutase 2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">RGP2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.56</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G05530</td>
<td align="left" colspan="1" rowspan="1" valign="middle">26S proteasome regulatory subunit 6A homolog A</td>
<td align="left" colspan="1" rowspan="1" valign="middle">RPT5A</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.35</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.77</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G16640</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Translationally controlled tumor protein 1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">TCTP1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.38</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G10220</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Tubulin-folding cofactor B</td>
<td align="left" colspan="1" rowspan="1" valign="middle">TFCB</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.4</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G64740</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Tubulin alpha-1 chain</td>
<td align="left" colspan="1" rowspan="1" valign="middle">TUBA1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.78</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G19770</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Tubulin alpha-5 chain</td>
<td align="left" colspan="1" rowspan="1" valign="middle">TUBA5</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G62690</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Tubulin beta-2 chain</td>
<td align="left" colspan="1" rowspan="1" valign="middle">TUBB2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.23</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G44340</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Tubulin beta-4 chain</td>
<td align="left" colspan="1" rowspan="1" valign="middle">TUBB4</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.36</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT2G29550</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Tubulin beta-7 chain</td>
<td align="left" colspan="1" rowspan="1" valign="middle">TUBB7</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.41</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G23860</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Tubulin beta-8 chain</td>
<td align="left" colspan="1" rowspan="1" valign="middle">TUBB8</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.43</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT4G20890</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Tubulin beta-9 chain</td>
<td align="left" colspan="1" rowspan="1" valign="middle">TUBB9</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.6</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G05620</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Tubulin gamma-2 chain</td>
<td align="left" colspan="1" rowspan="1" valign="middle">TUBG2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">−0.31</td>
<td align="left" colspan="1" rowspan="1" valign="middle">–</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G76030</td>
<td align="left" colspan="1" rowspan="1" valign="middle">V-type proton ATPase subunit B1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">VHA-B1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.16</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.23</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT2G41740</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Villin-2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">VLN2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.39</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G57410</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Villin-3</td>
<td align="left" colspan="1" rowspan="1" valign="middle">VLN3</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.34</td>
<td align="left" colspan="1" rowspan="1" valign="middle">1.12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT4G30160</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Villin-4</td>
<td align="left" colspan="1" rowspan="1" valign="middle">VLN4</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.39</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.49</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></section><p>The majority of the differentially abundant cell wall-related proteins between spaceflight and Earth were specific to the root. CELLULOSE SYNTHASES (CESA1, CESA3, CESA5, CESA7) and CELLULOSE-LIKE PROTEINS (CSLE1, CSLG3), involved in cellulose deposition and TUBULIN COFACTOR B (TFCB, L<sub>2</sub>FC<sub>Root</sub>=−0.4), whose expression reduces the number of microtubules, were root-specific and less in spaceflight (<a class="usa-link" href="#T2">
<strong>Table 2</strong>
</a>). β-tubulin dimers (TUBB2, TUBB4, TUBB8, TUBB9) were abundant in spaceflight in the root alone. β-tubulins are more exposed to phosphorylation than α-tubulin which are prone to detyrosination. Modification of tubulin is usually mediated by the tubulin cofactors. The UDP-ARABINOPYRANOSE MUTASEs (RGP1, L<sub>2</sub>FC<sub>Root</sub>=0.22 and RGP2, L<sub>2</sub>FC<sub>Root</sub>=0.56) involved in the biosynthesis of cell wall non-cellulosic polysaccharides were abundant in the root in spaceflight. Specific to the shoot are the polymers of α-tubulin, TUBA1 (L<sub>2</sub>FC<sub>Shoot</sub>=−0.78) and TUBA5 (L<sub>2</sub>FC<sub>Shoot</sub>=−0.3), which were less abundant in spaceflight compared to Earth (<a class="usa-link" href="#T2">
<strong>Table 2</strong>
</a>).</p></section><section id="s3_7"><h3 class="pmc_sec_title">3.7. Spaceflight and the plant REDOX homeostasis</h3>
<p>Regulating the accumulation of reactive oxygen species (ROS), antioxidants and redox enzymes in plant cells is a key adaptation mechanism for plant survival. ROS are involved in plant systemic signaling, and at the same time can be a result of stress metabolism. Over 132 RNA transcripts and 241 proteins involved in the maintenance of cellular redox homeostasis and ROS signaling were significantly altered (<em>p</em>&lt;0.05) in spaceflight (<a class="usa-link" href="#f6">
<strong>Figure 6</strong>
</a>, <a class="usa-link" href="#SM2">
<strong>Supplementary Table C</strong>
</a>). While the RNA transcripts included only a few of the traditionally known ROS and ROS-associated genes such as peroxidases, -redoxins, dehydrogenases, and so on, the proteome had many of them. The RNA transcripts involved in the response to ROS pathway differentially expressed in spaceflight include heat shock proteins (HSPs), transcription factors, and ubiquitin ligases (<a class="usa-link" href="#f6">
<strong>Figure 6</strong>
</a>). In both organs, 18 of the 20 PEROXIDASE proteins were abundant in spaceflight, indicating a high rate of ROS generation through the reduction of peroxide. Proteins regulating isocitrate metabolism, AC01 (L<sub>2</sub>FC<sub>Root</sub>=0.33), ACO3 (L<sub>2</sub>FC<sub>Root</sub>=0.57), CICDH (L<sub>2</sub>FC<sub>Root</sub>=0.54), and IDH5 (L<sub>2</sub>FC<sub>Shoot</sub>=0.43) were more abundant in spaceflight than on earth. The isocitrate pathway is the branchpoint of many other metabolic pathways in plants and the abundance of the Isocitrate dehydrogenase NAD+ catalytic enzymes suggests that plants in spaceflight environments use the isocitrate pathway to supplement NADPH generation from the pentose phosphate pathway.</p>
<figure class="fig xbox font-sm" id="f6"><h4 class="obj_head">Figure 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10712242_fpls-14-1260429-g006.jpg" target="_blank"><img alt="Figure 6" class="graphic zoom-in" height="393" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/897a/10712242/3103985ffb42/fpls-14-1260429-g006.jpg" width="748"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/f6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Redox homeostasis in spaceflight. Heatmap of GO enrichment terms (molecular and biological) of RNA transcripts, shoot proteins, and root proteins involved in the cellular response to oxidative stress and maintenance of redox homeostasis. The color gradient represents the significance of the enrichment of the genes and proteins in the metabolic pathway indicated by the right side of the plot.</p></figcaption></figure><p>SUPEROXIDE DISMUTASE, FSD1 (L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.9, 0.99), FSD3 (L<sub>2</sub>FC <sub>Shoot</sub>=0.31) and THIOREDOXIN REDUCTASE (NTRA, L<sub>2</sub>FC<sub>Root, Shoot</sub>=1.0, 193) were abundant in spaceflight and they facilitate the removal of superoxide in an NADPH-dependent manner. A vacuolar phosphatase (PAP26, L<sub>2</sub>FC<sub>Root, Shoot</sub>=0.43, 0.76) involved in the recycling of inorganic phosphate from intracellular phosphate metabolite (<a aria-describedby="B55" class="usa-link" href="#B55">Veljanovski et al., 2006</a>) was more abundant in spaceflight than on Earth. Enriched in the spaceflight environment are proteins (PGD1, TKL-2, NADP-ME2, and G6PD1), recognized as major participants of the pentose-phosphate shunt pathway (<a class="usa-link" href="#SM2">
<strong>Supplementary Table C</strong>
</a>). The pentose phosphate shunt pathway is a parallel alternative to the glycolysis pathway. Surprisingly, both RESPIRATORY BURST OXIDASE HOMOLOG PROTEIN D (RBOHD, L<sub>2</sub>FC<sub>Shoot</sub>=−0.67) and L-ASCORBATE PEROXIDASE 3, L<sub>2</sub>FC<sub>Shoot</sub>=−0.95) were less abundant in spaceflight in the shoot. The RBOHD produces ROS in response to environmental stress and plant signaling. Here, roots seem to be more impacted by oxidative stress and abundance of ROS than shoots in spaceflight (<a class="usa-link" href="#SM2">
<strong>Supplementary Table C</strong>
</a>).</p></section></section><section id="s4"><h2 class="pmc_sec_title">4. Discussion</h2>
<section id="s4_1"><h3 class="pmc_sec_title">4.1. Arabidopsis adapts photosystems to the spaceflight environment to minimize impacts on photosynthesis</h3>
<p>This study assesses the physiological and molecular impacts of the spaceflight environment on Arabidopsis seedlings grown in NASA BRIC LED hardware using a combined transcriptomics and proteomics approach. The BRIC LED hardware, with integrated LED illumination, provides the growing seedlings with a source of light. Hence, plants growing in the BRIC LED hardware on ISS will have light cues but lack gravitational cues. This enabled us to study the effects of microgravity on plant transcriptome and organ-specific proteome, while also providing insight into how the plant root and shoot individually integrate light signaling in the absence of gravity. Evidence from this study suggests that the operations of the photosynthetic machinery (PSII and PSI) were greatly impacted in spaceflight (<a class="usa-link" href="#f4">
<strong>Figure 4</strong>
</a>), with the plant shifting photosystem core-regulatory proteins in an organ-specific manner to adapt to the microgravity environment. While the net photosynthetic rate might not be affected in microgravity as it was not captured in this study, the molecular functions of the photosynthetic apparatus, as revealed by this study, were impacted. The abundance of the NusG-like proteins (PTAC12, PTAC14, PTAC3, PTAC4) in the shoot and NusG-like proteins (PTAC16, PTAC14) in the root (<a class="usa-link" href="#SM2">
<strong>Supplementary Table D</strong>
</a>) suggests attempts at boosting the translational efficiency of chloroplast ribosomes to reduce the plant sensitivity to environmental stress in spaceflight. The abundance of D1 complex protein (PSBA) and the PPL1 in spaceflight suggests attempts to optimize the water-oxidizing reaction and repair damages to the PSII. Reduced abundance of the PSAB and PSAE isoforms of the PSI in spaceflight, signifies an increased susceptibility of the PSII to photoinhibition (<a aria-describedby="B28" class="usa-link" href="#B28">Krieger-Liszkay et al., 2020</a>; <a aria-describedby="B20" class="usa-link" href="#B20">Ji et al., 2022</a>) and a deficiency of the PSI steady state redox-level and re-oxidation. The PPL1, which was abundant in spaceflight in both organs, works to facilitate the assembly of PSII and optimize plant fitness under increased sensitivity to the BRIC LED hardware light.</p>
<p>Suppression of the PSI electron transport subunit protein, PETH, in spaceflight, is associated with reduced efficiency of the electron transport chain and instability of the PSI. However, this reduced efficiency is compensated for by an increased abundance of chloroplast gene translations (PTACs), which helps to prevent over-reduction of the plastoquinone pool. The reduced abundance of PSII proximal antennae protein cp43 (PSBC) and the light-harvesting chlorophyll complexes (LHCB1, LHCB4, LHCB5, LHCB6) in spaceflight suggest an attempt to decrease light harvest by the photosystems, hence decrease photoinhibition, increase the oxygen-evolving capability and improve repair to PSII. Varying degrees of molecular perturbations to the plant photosystem complexes in spaceflight have earlier been reported, which further validate our findings (<a aria-describedby="B51" class="usa-link" href="#B51">Stutte et al., 2005</a>; <a aria-describedby="B9" class="usa-link" href="#B9">Chen and Wang, 2016</a>; <a aria-describedby="B53" class="usa-link" href="#B53">Vandenbrink et al., 2019</a>). However, unlike in this current study, where PSII and PSI complexes transcripts (RNA) were upregulated in spaceflight, <a aria-describedby="B53" class="usa-link" href="#B53">Vandenbrink et al. (2019)</a> reported significant downregulation of transcripts enriched in photosynthetic pathways in spaceflight compared to ground samples. They also indicated the downregulation of four RNA transcripts associated with photosynthetic electron transport (petE: plastocyanin, petF: ferredoxin, petH: ferredoxin–NADP+ reductase, petJ: cytochrome c6). These transcripts were however upregulated in spaceflight in our current study, but they were all downregulated at the proteome level in both the plant root and shoot (<a class="usa-link" href="#f4">
<strong>Figure 4</strong>
</a>). These discrepancies at the transcript level may be due to the difference in Arabidopsis wildtype ecotypes and spaceflight hardware utilized in both experiments. While <a aria-describedby="B53" class="usa-link" href="#B53">Vandenbrink et al. (2019)</a> utilized the <em>Arabidopsis thaliana</em> ecotype Landsberg and the European Modular Cultivation System (EMCS) hardware, this study utilized the <em>Arabidopsis thaliana</em> ecotype Columbia-0 and the BRIC LED hardware. <a aria-describedby="B5" class="usa-link" href="#B5">Basu et al. (2017)</a> and <a aria-describedby="B40" class="usa-link" href="#B40">Olanrewaju et al. (2023)</a> emphasize that the choice of hardware is a significant source of variations in the results of plant spaceflight experiments transcriptomics and proteomics.</p>
<p>
<a aria-describedby="B9" class="usa-link" href="#B9">Chen and Wang (2016)</a> delineated another tangent by revealing that out of 38 photosystem-associated proteins that exhibited differential expression in spaceflight-grown rice seedlings, 34 were downregulated in spaceflight compared to Earth controls. While our findings and that of <a aria-describedby="B53" class="usa-link" href="#B53">Vandenbrink et al. (2019)</a> support the coherent perturbation of both the PSII and PS I as revealed by the transcriptome and proteome, <a aria-describedby="B9" class="usa-link" href="#B9">Chen and Wang (2016)</a> suggested from their fluorescent emission spectra studies that despite the changes to the PSII proteins in spaceflight, their activity under microgravity is not affected, in contrast with the PSI which had significantly reduced energy transfer rates, hence impaired photosynthetic activity. Of course, <a aria-describedby="B9" class="usa-link" href="#B9">Chen and Wang (2016)</a> study was carried out on Rice seedlings, not Arabidopsis and directly extrapolating the differential PS I and PSII findings to Arabidopsis might not be so efficient. However, a consensus is that the spaceflight environment leads to a reduction in the PS I and PSII complexes, which may be accompanied by reduced photosynthetic efficiency and consequent smaller plant tissues.</p>
<p>The expression of the photosynthesis-associated proteins in the root might be due to the architecture of the BRIC LED hardware, which places the LEDs in close proximity to the roots, making them prone to direct hit by light. Interestingly, <a aria-describedby="B44" class="usa-link" href="#B44">Paul et al. (2013)</a> also reported the expression of several genes involved in light regulation in Arabidopsis roots in spaceflight transcriptomics experiments, even when the roots were not in close proximity to light. This phenomenon was also observed by <a aria-describedby="B30" class="usa-link" href="#B30">Kruse et al. (2020)</a> in their whole seedling transcriptomics study, despite that the seedlings were grown in the total absence of light cue in spaceflight. Hence, it is suggested that when gravity is absent, plant roots rely more heavily on negative phototropism for navigation, inducing light-sensing “tools” in roots in the spaceflight environment (<a aria-describedby="B42" class="usa-link" href="#B42">Paul et al., 2017</a>). The proteome landscape of the root and shoot, however, differs significantly in spaceflight, suggesting different adaptation mechanisms to the space environment, a further confirmation of <a aria-describedby="B44" class="usa-link" href="#B44">Paul et al. (2013)</a> transcriptomic study’s observation of organ-specificity in plant response to spaceflight. <a aria-describedby="B52" class="usa-link" href="#B52">Vandenbrink et al. (2016)</a> also proposed an interaction between plants’ gravity and light harvesting machinery at a genetic level, based on their observations of blue-light phototropic response in roots of Arabidopsis grown in microgravity.</p>
<p>The upregulation of most of the photosystem and contrasting decreased abundance of corresponding proteins in spaceflight (<a class="usa-link" href="#f4">
<strong>Figure 4</strong>
</a>) however, suggests that major adaptive response to spaceflight environmental stimulus might be regulated at the post-transcriptional level, even though the direct effect of the stimulus is still observed at the transcriptional level. A combined translatome-proteome study approach will be useful to elucidate this phenomenon.</p></section><section id="s4_2"><h3 class="pmc_sec_title">4.2. Intracellular trafficking and cell wall synthesis in spaceflight</h3>
<p>Plants, throughout their evolutionary history, have developed remarkable adaptations to counteract the unrelenting force of gravity (<a aria-describedby="B37" class="usa-link" href="#B37">Medina et al., 2021</a>). To withstand the gravitational force, plants have to ensure a balance between the internal turgor pressure, mediated by intracellular trafficking, and tensile resistance of both the primary and secondary cell walls. One pivotal molecular insight is the potential shift in cellulose metabolism in spaceflight, as indicated by the heightened abundance of Endoglucanase 20. Enhanced cellulose degradation might be a strategic adaptation, possibly optimizing cell wall flexibility in response to the unique stresses of a microgravity environment. This could be complemented by shifts in cellular adhesion dynamics, given the altered levels of FASCICLIN-LIKE ARABINOGALACTAN PROTEINs. The implications of these shifts could range from altered tissue integrity to modifications in intercellular communication. In a comprehensive transcriptomics meta-analysis of Arabidopsis plants cultivated in spaceflight within the BRIC Petri-dish fixation unit (PDFU) hardware, <a aria-describedby="B21" class="usa-link" href="#B21">Johnson et al. (2017)</a> highlighted potential alterations in the cell wall matrix of plants in spaceflight, as revealed by the glycomics evaluation of the BRIC-16-Cyt samples. They implied that microgravity influences the biosynthesis of xylan and pectic elements, leading to variations in the composition of the cell wall matrix.</p>
<p>Alongside this, <a aria-describedby="B21" class="usa-link" href="#B21">Johnson et al. (2017)</a> also reported the reduced abundance of xylan and pectic epitopes in spaceflight-grown plants, which resonate with our findings, highlighting the potential changes in the biosynthesis of cell wall non-cellulosic polysaccharides in spaceflight conditions, particularly in the root. <a aria-describedby="B38" class="usa-link" href="#B38">Nakashima et al. (2023)</a> in their novel glycome profiling and immunohistochemistry of Arabidopsis in spaceflight also observed that roots from spaceflight Arabidopsis seedlings displayed a generally increased labeling intensity when targeted with xyloglucan monoclonal antibodies (mAbs) compared to ground controls. They concluded that the heightened fluorescence intensity post-labeling with xyloglucan mAbs might result from changes in polysaccharide biosynthesis and secretion due to microgravity conditions. <a aria-describedby="B56" class="usa-link" href="#B56">Wakabayashi et al. (2020)</a> in their studies on Rice seedling cell wall under spaceflight conditions indicated that microgravity reduces the 1,3:1,4-β-glucan content in rice shoot cell walls, implicating an increase in the expression of an endo-1,3:1,4-β-glucanase gene. This reduction might serve to keep the cell wall more flexible under microgravity conditions. This observation is in harmony with our data, revealing potential changes in the biosynthesis of cell wall non-cellulosic polysaccharides, in spaceflight environments. Interestingly, while these studies reported that glucan metabolism is impacted, the spaceflight environment (microgravity) seems to exert minimal influence on the metabolism of arabinoxylans. This differentiation in response indicates a level of specificity in how plants adjust their cell wall components in response to altered gravity.</p>
<p>In the absence of gravity, seedlings have to default to secondary signals such as other environmental impulses. This affects internal homeostasis, inter- and intra-cellular transports, and cell wall composition. Our results identified intracellular Ca<sup>2+</sup> transport as a major player in adaptation to microgravity in both root and shoot. This is characterized by the increased abundance of the vacuolar proton ATPases and calcium-transporting ATPases. The vacuolar ATPases, contributing to the cytosolic pH homeostasis, form electrochemical proton gradients and drive active transport into cell organelles and they are important targets of phosphorylation and redox modification. The root-specific abundance of the cation transporters suggests that the homeostatic needs of the root differ from that of the shoot in spaceflight. The altered abundance of small GTPases such as Ran, Rab, and Ras, between spaceflight and Earth suggests a functional dysregulation of vesicle trafficking in spaceflight. The spaceflight environment alters molecular switches between the membrane-bound active GTP and cytosolic inactive GDP which tremendously affects signal transduction.</p>
<p>Molecular mechanisms driving polar auxin distribution were affected and organ-specific, with the altered abundance of auxin transporters in spaceflight and on Earth. However, auxin transport and response appear to be more pronounced on Earth than in microgravity. Results from this study also suggest a decreased stability of the cellulose cell wall in spaceflight, due to the abundance of cellulose-degrading proteins and less abundance of cellulose synthase (<a class="usa-link" href="#T2">
<strong>Table 2</strong>
</a>). The spaceflight environment led to the decreased formation and increased destabilization of the microtubules which suggests that seedlings seem to fall back on the actin skeleton for support. Interactions between PIP2 which is abundant in space, with VILLINs which were also abundant in space inhibit the capping and severing actions of actin but rather increase actin binding at the headpiece region. This suggests that actin cross-nucleation and cross-linking is an adaptation mechanism for plants in space. An in-depth study into vesicle and cargo sorting under microgravity will give more insight into how plants adapt their intracellular trafficking and cell wall depositions to the spaceflight environment.</p></section><section id="s4_3"><h3 class="pmc_sec_title">4.3. Spaceflight significantly affects plant translation machinery in an organ-specific pattern</h3>
<p>The varied abundance of over 150 ribosomal subunit proteins, spliceosomes and proteasomes in spaceflight and across both organs suggest major alterations to plant transcriptional and translation machinery as an adaptation mechanism to the spaceflight environment (<a class="usa-link" href="#SM2">
<strong>Supplementary Tables E, F</strong>
</a>). The decreased abundance of various mRNA slicing factors (ATO, CUV, SC35, SCL30A, U2AF65A, U2AF65B, DEAH3) in the shoot in spaceflight suggests less mRNA splicing in the shoot compared to the root, which might lead to the abundance of non-functional proteins in the shoot and increased need for proteasome activities. Combining transcript data with this proteome data will give more insight into this. The abundance of HEAT SHOCK PROTEIN families (HS60, HSP70, HSP90) in both organs suggests an increased control of protein homeostasis in space characterized by folding of nascent polypeptides released from the ribosome, sorting of proteins to cell organelles and providing a link to the ubiquitin-mediated proteasome degradation. The relative abundance of 26S proteasomes involved in the degradation of misfolded and orphaned proteins suggests that the spaceflight environment led to errors in protein translation, hence the need to increase the efficiency of the proteasome unit.</p></section><section id="s4_4"><h3 class="pmc_sec_title">4.4. Combating oxidative stress while maintaining ROS signaling in spaceflight</h3>
<p>Seedlings in spaceflight are faced with the need to maintain a balance between the abundance of ROS for systemic stress signaling and their damaging effects. Our results suggest dedicated efforts to the removal of superoxide radicals by the abundance of several peroxides, thioredoxin, ferritin, and superoxide dismutase in spaceflight. We infer that plants in spaceflight utilize a shunt pathway, which is a parallel alternative to the glycolytic pathway, to complement the NADPH production from the pentose-phosphate pathway. The pentose-phosphate shunt pathway is driven by NADPH demands in different tissues (<a aria-describedby="B2" class="usa-link" href="#B2">Aziz and Mohiuddin, 2022</a>), and in this extremity, plants in spaceflight recycle inorganic phosphate from intracellular phosphate metabolites in the vacuole, mediated by PAP26 to meet the NADPH demands. This suggests that the metabolic energy requirement for plants in space is higher than on Earth. The abundance of aldehyde dehydrogenases (ALDHs) and several glutathione dehydrogenases suggests the involvement of the S-nitrosation reaction in cellular homeostasis in spaceflight. A common feature of the ALDHs is the susceptibility of their catalytic cysteine residue to GSNO-mediated S-nitrosation (<a aria-describedby="B50" class="usa-link" href="#B50">Stiti et al., 2020</a>), which has been suggested as a central gravitropism signaling event (<a aria-describedby="B31" class="usa-link" href="#B31">Kruse and Wyatt, 2022</a>).</p>
<p>While this study substantiates that the BRIC LED hardware supports Arabidopsis plant growth in spaceflight, it is important to note that the BRIC hardware, unlike some newly developed advanced spaceflight hardware such as the Advance Plant Habitat (APH), is a closed system, hence no gas exchange between the plant and the external environment. This is a major limitation for the hardware, as this promotes the buildup of gases such as ethylene, oxygen and other volatile organic compounds (VOCs) (<a aria-describedby="B24" class="usa-link" href="#B24">Klymchuk et al., 2003</a>). Despite being an important hormone to plant, elevated ethylene level had been identified to cause anomalous Arabidopsis plant growth in spaceflight (<a aria-describedby="B49" class="usa-link" href="#B49">Stankovic, 2018</a>) and sterility in male wheat plants grown in spaceflight (<a aria-describedby="B7" class="usa-link" href="#B7">Campbell et al., 2001</a>). Beyond ethylene, the accumulation of oxygen and other VOCs can potentially reach toxic levels in the BRIC LED hardware. This is a substantial source of stress as indicated by <a aria-describedby="B5" class="usa-link" href="#B5">Basu et al. (2017)</a>. Drawing parallels with Earth, closed environments like terrariums might encounter similar challenges with gas accumulation. However, there’s a critical difference. On Earth, natural processes and microbial interactions often intervene to balance out the system over time. But in the unique microgravity setting of space, these natural balancing acts might not function as they do here, thereby amplifying the challenges of a closed system, as we consider plants for bio-regenerative life support in long-term spaceflight missions.</p></section><section id="s4_5"><h3 class="pmc_sec_title">4.5. Non-correlated transcriptome and proteome: indicative of post-transcriptional regulation in spaceflight?</h3>
<p>Spaceflight environmental conditions pose unique challenges to plants, leading to significant alterations in gene expression and protein synthesis. Our analysis of Arabidopsis plants grown in space reveals that the expression of RNA transcripts did not correspond to their translation to shoot and root proteins. The observed non-correlation between transcriptome and proteome data from Arabidopsis plants grown in spaceflight may indeed suggest a prevalent post-transcriptional regulation (<a aria-describedby="B57" class="usa-link" href="#B57">Wang et al., 2022</a>). Transcript-protein discordance can be attributed to multiple factors. One such factor is the differential extraction protocols and data processing for RNA and protein, which can introduce variation in the extracted quantities (<a aria-describedby="B34" class="usa-link" href="#B34">Liu et al., 2016</a>). Furthermore, due to the unique environmental stressors of spaceflight, the mechanisms to arrest translation could be compromised, resulting in skewed protein synthesis rates. Moreover, the phenomenon of alternative splicing, often upregulated under stress conditions, can result in multiple protein isoforms from a single transcript, further complicating the correlation. Another source of discordance to note is that the RNA transcriptome might be a reflection of the plant’s immediate response to stress, while the proteome represents a more stabilized, adaptive response. These unique constraints of spaceflight can thus evoke distinct reactions at plants’ transcriptome and proteome levels, leading to non-correlated outcomes. Hence, attributing the non-correlation of the transcriptome and proteome only to alternative splicing will be a bias.</p>
<p>This study, by combining transcriptomics and proteomics, provided insight into plants’ adaptive responses to the spaceflight environment at multiple molecular levels. While we did not directly measure photosynthetic net production, our findings indicate a series of modifications to photosystems II and I aimed at adapting to the space environment, potentially decreasing the plant’s sensitivity to light intensity and enhancing its oxygen-evolving capacity. Concurrently, we observed evidence for an increased metabolic energy requirement in spaceflight, with plants seemingly engaging alternate energy generation pathways to complement photosynthetic production. Furthermore, our findings indicate significant impacts on cellular trafficking and potential compromises to cell wall integrity, suggesting an adaptation in structural support provided by actin filament cross-linking and cross-nucleation. This research not only serves as a comprehensive study of plant response in spaceflight but also holds profound implications for terrestrial agriculture, providing valuable insights for developing crop varieties resilient to environmental stressors on Earth, such as drought, extreme temperatures, and salinity.</p>
<p>This study also contributes to the broader goal of space agriculture – a key component of long-duration space missions and off-Earth colonization. While centered around the model plant Arabidopsis, our findings could provide the foundation for developing crop varieties suitable for growth in extraterrestrial environments, supporting food production for future missions to Mars and the Moon. Thus, the findings of our research, while intrinsically fascinating, also bear practical implications for the future of both terrestrial and space agriculture. As we continue to explore the final frontier, understanding how life – in this case, plant life – adapts and thrives is crucial for the success of our extraterrestrial endeavors and the sustainable development of our agriculture on Earth.</p></section></section><section id="s5"><h2 class="pmc_sec_title">Data availability statement</h2>
<p>The datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found below: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://osdr.nasa.gov/bio/repo/data/studies/OSD-522/preview/w5Q5ElZE-Wy6DDS1ZFzeRjBAPuKss-x0" rel="noopener noreferrer" target="_blank">https://osdr.nasa.gov/bio/repo/data/studies/OSD-522/preview/w5Q5ElZE-Wy6DDS1ZFzeRjBAPuKss-x0</a>, OSD-522.</p></section><section id="s6"><h2 class="pmc_sec_title">Author contributions</h2>
<p>GO: Conceptualization, Data curation, Formal Analysis, Investigation, Methodology, Project administration, Resources, Software, Validation, Visualization, Writing – original draft, Writing – review &amp; editing. NH: Conceptualization, Formal Analysis, Investigation, Software, Writing – original draft, Writing – review &amp; editing. MN: Formal Analysis, Investigation, Software, Writing – review &amp; editing. AP: Conceptualization, Funding acquisition, Project administration, Resources, Supervision, Validation, Writing – review &amp; editing. RF: Conceptualization, Funding acquisition, Project administration, Resources, Supervision, Validation, Writing – review &amp; editing. SW: Conceptualization, Formal Analysis, Funding acquisition, Investigation, Project administration, Resources, Supervision, Visualization, Writing – review &amp; editing.</p></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The author would like to thank the hardware and flight support team at Kennedy Space Center for their work to validate, prepare and harvest the BRIC LED experiment. We also appreciate the guidance of Dr. Colin Kruse (Los Alamos National Lab) in analyzing the proteomics dataset.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>The author(s) declare financial support was received for the research, authorship, and/or publication of this article. This work was partially funded by the National Aeronautics and Space Administration (NASA) Science Mission Directorate grants NASA#80NSSC19K1481 to SW, and NASA#80NSSC19K0130 to A-LP and RF. NASA employees had a key role in payload management, delivery, and processing but otherwise had no direct involvement in sample processing or downstream analysis.</p></section><section id="s8"><h2 class="pmc_sec_title">Conflict of interest</h2>
<p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></section><section id="s9"><h2 class="pmc_sec_title">Publisher’s note</h2>
<p>All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.</p></section><section id="s10"><h2 class="pmc_sec_title">Supplementary material</h2>
<p>The Supplementary Material for this article can be found online at: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.frontiersin.org/articles/10.3389/fpls.2023.1260429/full#supplementary-material" rel="noopener noreferrer" target="_blank">https://www.frontiersin.org/articles/10.3389/fpls.2023.1260429/full#supplementary-material</a></p>
<section class="sm xbox font-sm" id="SM1"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10712242/bin/DataSheet_1.pdf">Click here for additional data file.</a><sup> (1.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM2"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10712242/bin/DataSheet_2.zip">Click here for additional data file.</a><sup> (162.5KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM3"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10712242/bin/DataSheet_3.pdf">Click here for additional data file.</a><sup> (1,019.7KB, pdf) </sup>
</div></div></section></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="B1">
<cite>
Antonacci A., Lambreva M. D., Margonelli A., Sobolev A. P., Pastorelli S., Bertalan I., et al. (2018). Photosystem-II D1 protein mutants of Chlamydomonas reinhardtii in relation to metabolic rewiring and remodeling of H-bond network at QB site. Sci. Rep.
8, 14745. doi:  10.1038/s41598-018-33146-y

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-018-33146-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6170454/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30283151/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Photosystem-II%20D1%20protein%20mutants%20of%20Chlamydomonas%20reinhardtii%20in%20relation%20to%20metabolic%20rewiring%20and%20remodeling%20of%20H-bond%20network%20at%20QB%20site&amp;author=A.%20Antonacci&amp;author=M.%20D.%20Lambreva&amp;author=A.%20Margonelli&amp;author=A.%20P.%20Sobolev&amp;author=S.%20Pastorelli&amp;volume=8&amp;publication_year=2018&amp;pages=14745&amp;pmid=30283151&amp;doi=10.1038/s41598-018-33146-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<cite>
Aziz H., Mohiuddin S. S. (2022). Biochemistry, Hexose Monophosphate Pathway (Treasure Island (FL: StatPearls Publishing; ).</cite> [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31869151/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Biochemistry,%20Hexose%20Monophosphate%20Pathway&amp;author=H.%20Aziz&amp;author=S.%20S.%20Mohiuddin&amp;publication_year=2022&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<cite>
Barker R., Kruse C. P. S., Johnson C., Saravia-Butler A., Fogle H., Chang H.-S., et al. (2023). Meta-analysis of the space flight and microgravity response of the Arabidopsis plant transcriptome. NPJ Microgravity
9, 21. doi:  10.1038/s41526-023-00247-6

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41526-023-00247-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10027818/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36941263/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Microgravity&amp;title=Meta-analysis%20of%20the%20space%20flight%20and%20microgravity%20response%20of%20the%20Arabidopsis%20plant%20transcriptome&amp;author=R.%20Barker&amp;author=C.%20P.%20S.%20Kruse&amp;author=C.%20Johnson&amp;author=A.%20Saravia-Butler&amp;author=H.%20Fogle&amp;volume=9&amp;publication_year=2023&amp;pages=21&amp;pmid=36941263&amp;doi=10.1038/s41526-023-00247-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<cite>
Barker R., Lombardino J., Rasmussen K., Gilroy S. (2020). Test of arabidopsis space transcriptome: A discovery environment to explore multiple plant biology spaceflight experiments. Front. Plant Sci.
11. doi:  10.3389/fpls.2020.00147
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2020.00147" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7076552/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32265943/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=Test%20of%20arabidopsis%20space%20transcriptome:%20A%20discovery%20environment%20to%20explore%20multiple%20plant%20biology%20spaceflight%20experiments&amp;author=R.%20Barker&amp;author=J.%20Lombardino&amp;author=K.%20Rasmussen&amp;author=S.%20Gilroy&amp;volume=11&amp;publication_year=2020&amp;pmid=32265943&amp;doi=10.3389/fpls.2020.00147&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<cite>
Basu P., Kruse C. P. S., Luesse D. R., Wyatt S. E. (2017). Growth in spaceflight hardware results in alterations to the transcriptome and proteome. Life Sci. Space Res. (Amst)
15, 88–96. doi:  10.1016/j.lssr.2017.09.001

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.lssr.2017.09.001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29198318/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Life%20Sci.%20Space%20Res.%20(Amst)&amp;title=Growth%20in%20spaceflight%20hardware%20results%20in%20alterations%20to%20the%20transcriptome%20and%20proteome&amp;author=P.%20Basu&amp;author=C.%20P.%20S.%20Kruse&amp;author=D.%20R.%20Luesse&amp;author=S.%20E.%20Wyatt&amp;volume=15&amp;publication_year=2017&amp;pages=88-96&amp;pmid=29198318&amp;doi=10.1016/j.lssr.2017.09.001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<cite>
Basu P., Luesse D. R., Wyatt S. E. (2015). “Proteomic approaches and their application to plant gravitropism.” in Plant gravitropism methods in molecular biology. Eds. Blancaflor E. (New York, NY: Humana Press; ), 1309. doi:  10.1007/978-1-4939-2697-8_10
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-1-4939-2697-8_10" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25981772/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Plant%20gravitropism%20methods%20in%20molecular%20biology&amp;author=P.%20Basu&amp;author=D.%20R.%20Luesse&amp;author=S.%20E.%20Wyatt&amp;publication_year=2015&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<cite>
Campbell W. F., Salisbury F. B., Bugbee B., Klassen S., Naegle E., Strickland D. T., et al. (2001). Comparative floral development of Mir-grown and ethylene-treated, earth-grown Super Dwarf wheat. J. Plant Physiol.
158, 1051–1060. doi:  10.1078/S0176-1617(04)70129-7

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1078/S0176-1617(04)70129-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12033229/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Plant%20Physiol.&amp;title=Comparative%20floral%20development%20of%20Mir-grown%20and%20ethylene-treated,%20earth-grown%20Super%20Dwarf%20wheat&amp;author=W.%20F.%20Campbell&amp;author=F.%20B.%20Salisbury&amp;author=B.%20Bugbee&amp;author=S.%20Klassen&amp;author=E.%20Naegle&amp;volume=158&amp;publication_year=2001&amp;pages=1051-1060&amp;pmid=12033229&amp;doi=10.1078/S0176-1617(04)70129-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<cite>
Cao P., Kim S.-J., Xing A., Schenck C. A., Liu L., Jiang N., et al. (2019). Homeostasis of branched-chain amino acids is critical for the activity of TOR signaling in Arabidopsis. Elife
8. doi:  10.7554/eLife.50747
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.7554/eLife.50747" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6937141/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31808741/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Homeostasis%20of%20branched-chain%20amino%20acids%20is%20critical%20for%20the%20activity%20of%20TOR%20signaling%20in%20Arabidopsis&amp;author=P.%20Cao&amp;author=S.-J.%20Kim&amp;author=A.%20Xing&amp;author=C.%20A.%20Schenck&amp;author=L.%20Liu&amp;volume=8&amp;publication_year=2019&amp;pmid=31808741&amp;doi=10.7554/eLife.50747&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<cite>
Chen B., Wang Y. (2016). Proteomic and Physiological Studies Provide Insight into Photosynthetic Response of Rice ( <em>Oryza sativa</em> L.) Seedlings to Microgravity. Photochem. Photobiol.
92, 561–570. doi:  10.1111/php.12593

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/php.12593" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27096703/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Photochem.%20Photobiol.&amp;title=Proteomic%20and%20Physiological%20Studies%20Provide%20Insight%20into%20Photosynthetic%20Response%20of%20Rice%20(%20Oryza%20sativa%20L.)%20Seedlings%20to%20Microgravity&amp;author=B.%20Chen&amp;author=Y.%20Wang&amp;volume=92&amp;publication_year=2016&amp;pages=561-570&amp;pmid=27096703&amp;doi=10.1111/php.12593&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<cite>
Christopher J. A., Geladaki A., Dawson C. S., Vennard O. L., Lilley K. S. (2022). Subcellular Transcriptomics and proteomics: A comparative methods review. Mol. Cell. Proteomics
21, 100186. doi:  10.1016/j.mcpro.2021.100186

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.mcpro.2021.100186" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8864473/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34922010/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Subcellular%20Transcriptomics%20and%20proteomics:%20A%20comparative%20methods%20review&amp;author=J.%20A.%20Christopher&amp;author=A.%20Geladaki&amp;author=C.%20S.%20Dawson&amp;author=O.%20L.%20Vennard&amp;author=K.%20S.%20Lilley&amp;volume=21&amp;publication_year=2022&amp;pages=100186&amp;pmid=34922010&amp;doi=10.1016/j.mcpro.2021.100186&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<cite>
Ferl R. J., Koh J., Denison F., Paul A.-L. (2015). Spaceflight induces specific alterations in the proteomes of arabidopsis. Astrobiology
15, 32–56. doi:  10.1089/ast.2014.1210

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2014.1210" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4290804/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25517942/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Spaceflight%20induces%20specific%20alterations%20in%20the%20proteomes%20of%20arabidopsis&amp;author=R.%20J.%20Ferl&amp;author=J.%20Koh&amp;author=F.%20Denison&amp;author=A.-L.%20Paul&amp;volume=15&amp;publication_year=2015&amp;pages=32-56&amp;pmid=25517942&amp;doi=10.1089/ast.2014.1210&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<cite>
Ferl R., Wheeler R., Levine H. G., Paul A.-L. (2002). Plants in space. Curr. Opin. Plant Biol.
5, 258–263. doi:  10.1016/S1369-5266(02)00254-6

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S1369-5266(02)00254-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11960745/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Plant%20Biol.&amp;title=Plants%20in%20space&amp;author=R.%20Ferl&amp;author=R.%20Wheeler&amp;author=H.%20G.%20Levine&amp;author=A.-L.%20Paul&amp;volume=5&amp;publication_year=2002&amp;pages=258-263&amp;pmid=11960745&amp;doi=10.1016/S1369-5266(02)00254-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<cite>
Ge S. X., Jung D., Yao R. (2020). ShinyGO: a graphical gene-set enrichment tool for animals and plants. Bioinformatics
36, 2628–2629. doi:  10.1093/bioinformatics/btz931

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btz931" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7178415/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31882993/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=ShinyGO:%20a%20graphical%20gene-set%20enrichment%20tool%20for%20animals%20and%20plants&amp;author=S.%20X.%20Ge&amp;author=D.%20Jung&amp;author=R.%20Yao&amp;volume=36&amp;publication_year=2020&amp;pages=2628-2629&amp;pmid=31882993&amp;doi=10.1093/bioinformatics/btz931&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14">
<cite>
Giardi M. T., Rea G., Lambreva M. D., Antonacci A., Pastorelli S., Bertalan I., et al. (2013). Mutations of Photosystem II D1 Protein That Empower Efficient Phenotypes of Chlamydomonas reinhardtii under Extreme Environment in Space. PloS One
8, e64352. doi:  10.1371/journal.pone.0064352

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0064352" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3653854/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23691201/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PloS%20One&amp;title=Mutations%20of%20Photosystem%20II%20D1%20Protein%20That%20Empower%20Efficient%20Phenotypes%20of%20Chlamydomonas%20reinhardtii%20under%20Extreme%20Environment%20in%20Space&amp;author=M.%20T.%20Giardi&amp;author=G.%20Rea&amp;author=M.%20D.%20Lambreva&amp;author=A.%20Antonacci&amp;author=S.%20Pastorelli&amp;volume=8&amp;publication_year=2013&amp;pages=e64352&amp;pmid=23691201&amp;doi=10.1371/journal.pone.0064352&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15">
<cite>
Hausmann N., Fengler S., Hennig A., Franz-Wachtel M., Hampp R., Neef M. (2014). Cytosolic calcium, hydrogen peroxide and related gene expression and protein modulation in A rabidopsis thaliana cell cultures respond immediately to altered gravitation: parabolic flight data. Plant Biol.
16, 120–128. doi:  10.1111/plb.12051

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/plb.12051" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23870071/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biol.&amp;title=Cytosolic%20calcium,%20hydrogen%20peroxide%20and%20related%20gene%20expression%20and%20protein%20modulation%20in%20A%20rabidopsis%20thaliana%20cell%20cultures%20respond%20immediately%20to%20altered%20gravitation:%20parabolic%20flight%20data&amp;author=N.%20Hausmann&amp;author=S.%20Fengler&amp;author=A.%20Hennig&amp;author=M.%20Franz-Wachtel&amp;author=R.%20Hampp&amp;volume=16&amp;publication_year=2014&amp;pages=120-128&amp;pmid=23870071&amp;doi=10.1111/plb.12051&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16">
<cite>
Hooper C. M., Castleden I. R., Tanz S. K., Aryamanesh N., Millar A. H. (2017). SUBA4: the interactive data analysis center for Arabidopsis subcellular protein locations. Nucleic Acids Res.
45, D1064–D1074. doi:  10.1093/nar/gkw1041

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkw1041" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5210537/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27899614/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=SUBA4:%20the%20interactive%20data%20analysis%20center%20for%20Arabidopsis%20subcellular%20protein%20locations&amp;author=C.%20M.%20Hooper&amp;author=I.%20R.%20Castleden&amp;author=S.%20K.%20Tanz&amp;author=N.%20Aryamanesh&amp;author=A.%20H.%20Millar&amp;volume=45&amp;publication_year=2017&amp;pages=D1064-D1074&amp;pmid=27899614&amp;doi=10.1093/nar/gkw1041&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17">
<cite>
Hooper C. M., Tanz S. K., Castleden I. R., Vacher M. A., Small I. D., Millar A. H. (2014). SUBAcon: a consensus algorithm for unifying the subcellular localization data of the Arabidopsis proteome. Bioinformatics
30, 3356–3364. doi:  10.1093/bioinformatics/btu550

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/btu550" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25150248/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SUBAcon:%20a%20consensus%20algorithm%20for%20unifying%20the%20subcellular%20localization%20data%20of%20the%20Arabidopsis%20proteome&amp;author=C.%20M.%20Hooper&amp;author=S.%20K.%20Tanz&amp;author=I.%20R.%20Castleden&amp;author=M.%20A.%20Vacher&amp;author=I.%20D.%20Small&amp;volume=30&amp;publication_year=2014&amp;pages=3356-3364&amp;pmid=25150248&amp;doi=10.1093/bioinformatics/btu550&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18"><cite>
Howard G.L., Caron A. (2016) Biological Research in Canisters (BRIC) - Light Emitting Diode (LED). Available at: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://ntrs.nasa.gov/api/citations/20160009021/downloads/20160009021.pdf" rel="noopener noreferrer" target="_blank">https://ntrs.nasa.gov/api/citations/20160009021/downloads/20160009021.pdf</a>.</cite></li>
<li id="B19">
<cite>
Hughes A. M., Kiss J. Z. (2022). -Omics studies of plant biology in spaceflight: A critical review of recent experiments. Front. Astronomy Space Sci.
9. doi:  10.3389/fspas.2022.964657
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fspas.2022.964657" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Astronomy%20Space%20Sci.&amp;title=-Omics%20studies%20of%20plant%20biology%20in%20spaceflight:%20A%20critical%20review%20of%20recent%20experiments&amp;author=A.%20M.%20Hughes&amp;author=J.%20Z.%20Kiss&amp;volume=9&amp;publication_year=2022&amp;doi=10.3389/fspas.2022.964657&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B20">
<cite>
Ji D., Li Q., Guo Y., An W., Manavski N., Meurer J., et al. (2022). NADP+ supply adjusts the synthesis of photosystem I in Arabidopsis chloroplasts. Plant Physiol.
189, 2128–2143. doi:  10.1093/plphys/kiac161

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/plphys/kiac161" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9343004/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35385122/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=NADP+%20supply%20adjusts%20the%20synthesis%20of%20photosystem%20I%20in%20Arabidopsis%20chloroplasts&amp;author=D.%20Ji&amp;author=Q.%20Li&amp;author=Y.%20Guo&amp;author=W.%20An&amp;author=N.%20Manavski&amp;volume=189&amp;publication_year=2022&amp;pages=2128-2143&amp;pmid=35385122&amp;doi=10.1093/plphys/kiac161&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21">
<cite>
Johnson C. M., Subramanian A., Pattathil S., Correll M. J., Kiss J. Z. (2017). Comparative transcriptomics indicate changes in cell wall organization and stress response in seedlings during spaceflight. Am. J. Bot.
104, 1219–1231. doi:  10.3732/ajb.1700079

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3732/ajb.1700079" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5821596/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28827451/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Bot.&amp;title=Comparative%20transcriptomics%20indicate%20changes%20in%20cell%20wall%20organization%20and%20stress%20response%20in%20seedlings%20during%20spaceflight&amp;author=C.%20M.%20Johnson&amp;author=A.%20Subramanian&amp;author=S.%20Pattathil&amp;author=M.%20J.%20Correll&amp;author=J.%20Z.%20Kiss&amp;volume=104&amp;publication_year=2017&amp;pages=1219-1231&amp;pmid=28827451&amp;doi=10.3732/ajb.1700079&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22">
<cite>
Kanapskyte A., Hawkins E. M., Liddell L. C., Bhardwaj S. R., Gentry D., Santa Maria S. R. (2021). Space biology research and biosensor technologies: past, present, and future. Biosensors (Basel)
11, 38. doi:  10.3390/bios11020038

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/bios11020038" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7912197/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33572823/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biosensors%20(Basel)&amp;title=Space%20biology%20research%20and%20biosensor%20technologies:%20past,%20present,%20and%20future&amp;author=A.%20Kanapskyte&amp;author=E.%20M.%20Hawkins&amp;author=L.%20C.%20Liddell&amp;author=S.%20R.%20Bhardwaj&amp;author=D.%20Gentry&amp;volume=11&amp;publication_year=2021&amp;pages=38&amp;pmid=33572823&amp;doi=10.3390/bios11020038&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23">
<cite>
Kang H., Hwang I. (2014). Vacuolar sorting receptor-mediated trafficking of soluble vacuolar proteins in plant cells. Plants
3, 392–408. doi:  10.3390/plants3030392

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/plants3030392" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4844349/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27135510/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Vacuolar%20sorting%20receptor-mediated%20trafficking%20of%20soluble%20vacuolar%20proteins%20in%20plant%20cells&amp;author=H.%20Kang&amp;author=I.%20Hwang&amp;volume=3&amp;publication_year=2014&amp;pages=392-408&amp;pmid=27135510&amp;doi=10.3390/plants3030392&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24">
<cite>
Klymchuk D. O., Kordyum E. L., Vorobyova T. V., Chapman D. K., Brown C. S. (2003). Changes in vacuolation in the root apex cells of soybean seedlings in microgravity. Adv. Space Res.
31, 2283–2288. doi:  10.1016/S0273-1177(03)00256-4

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0273-1177(03)00256-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14686444/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res.&amp;title=Changes%20in%20vacuolation%20in%20the%20root%20apex%20cells%20of%20soybean%20seedlings%20in%20microgravity&amp;author=D.%20O.%20Klymchuk&amp;author=E.%20L.%20Kordyum&amp;author=T.%20V.%20Vorobyova&amp;author=D.%20K.%20Chapman&amp;author=C.%20S.%20Brown&amp;volume=31&amp;publication_year=2003&amp;pages=2283-2288&amp;pmid=14686444&amp;doi=10.1016/S0273-1177(03)00256-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25">
<cite>
Kolbeck A., Marhavý P., De Bellis D., Li B., Kamiya T., Fujiwara T., et al. (2022). CASP microdomain formation requires cross cell wall stabilization of domains and non-cell autonomous action of LOTR1. Elife
11. doi:  10.7554/eLife.69602
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.7554/eLife.69602" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8794472/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35029147/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=CASP%20microdomain%20formation%20requires%20cross%20cell%20wall%20stabilization%20of%20domains%20and%20non-cell%20autonomous%20action%20of%20LOTR1&amp;author=A.%20Kolbeck&amp;author=P.%20Marhav%C3%BD&amp;author=D.%20De%20Bellis&amp;author=B.%20Li&amp;author=T.%20Kamiya&amp;volume=11&amp;publication_year=2022&amp;pmid=35029147&amp;doi=10.7554/eLife.69602&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26">
<cite>
Kordyum E. L., Chapman D. K. (2017). Plants and microgravity: Patterns of microgravity effects at the cellular and molecular levels. Cytol Genet.
51, 108–116. doi:  10.3103/S0095452717020049
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3103/S0095452717020049" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30484618/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cytol%20Genet.&amp;title=Plants%20and%20microgravity:%20Patterns%20of%20microgravity%20effects%20at%20the%20cellular%20and%20molecular%20levels&amp;author=E.%20L.%20Kordyum&amp;author=D.%20K.%20Chapman&amp;volume=51&amp;publication_year=2017&amp;pages=108-116&amp;pmid=30484618&amp;doi=10.3103/S0095452717020049&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27">
<cite>
Kordyum E., Hasenstein K. H. (2021). Plant biology for space exploration – Building on the past, preparing for the future. Life Sci. Space Res. (Amst)
29, 1–7. doi:  10.1016/j.lssr.2021.01.003

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.lssr.2021.01.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33888282/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Life%20Sci.%20Space%20Res.%20(Amst)&amp;title=Plant%20biology%20for%20space%20exploration%20%E2%80%93%20Building%20on%20the%20past,%20preparing%20for%20the%20future&amp;author=E.%20Kordyum&amp;author=K.%20H.%20Hasenstein&amp;volume=29&amp;publication_year=2021&amp;pages=1-7&amp;pmid=33888282&amp;doi=10.1016/j.lssr.2021.01.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28">
<cite>
Krieger-Liszkay A., Shimakawa G., Sétif P. (2020). Role of the two PsaE isoforms on O2 reduction at photosystem I in Arabidopsis thaliana. Biochim. Biophys. Acta (BBA) - Bioenergetics
1861, 148089. doi:  10.1016/j.bbabio.2019.148089

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.bbabio.2019.148089" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31669487/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta%20(BBA)%20-%20Bioenergetics&amp;title=Role%20of%20the%20two%20PsaE%20isoforms%20on%20O2%20reduction%20at%20photosystem%20I%20in%20Arabidopsis%20thaliana&amp;author=A.%20Krieger-Liszkay&amp;author=G.%20Shimakawa&amp;author=P.%20S%C3%A9tif&amp;volume=1861&amp;publication_year=2020&amp;pages=148089&amp;pmid=31669487&amp;doi=10.1016/j.bbabio.2019.148089&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29">
<cite>
Kruse C. P. S., Basu P., Luesse D. R., Wyatt S. E. (2017). Transcriptome and proteome responses in RNAlater preserved tissue of Arabidopsis thaliana. PLoS One
12, e0175943. doi:  10.1371/journal.pone.0175943

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0175943" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5397022/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28423006/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Transcriptome%20and%20proteome%20responses%20in%20RNAlater%20preserved%20tissue%20of%20Arabidopsis%20thaliana&amp;author=C.%20P.%20S.%20Kruse&amp;author=P.%20Basu&amp;author=D.%20R.%20Luesse&amp;author=S.%20E.%20Wyatt&amp;volume=12&amp;publication_year=2017&amp;pages=e0175943&amp;pmid=28423006&amp;doi=10.1371/journal.pone.0175943&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30">
<cite>
Kruse C. P. S., Meyers A. D., Basu P., Hutchinson S., Luesse D. R., Wyatt S. E. (2020). Spaceflight induces novel regulatory responses in Arabidopsis seedling as revealed by combined proteomic and transcriptomic analyses. BMC Plant Biol.
20, 237. doi:  10.1186/s12870-020-02392-6

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12870-020-02392-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7251690/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32460700/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biol.&amp;title=Spaceflight%20induces%20novel%20regulatory%20responses%20in%20Arabidopsis%20seedling%20as%20revealed%20by%20combined%20proteomic%20and%20transcriptomic%20analyses&amp;author=C.%20P.%20S.%20Kruse&amp;author=A.%20D.%20Meyers&amp;author=P.%20Basu&amp;author=S.%20Hutchinson&amp;author=D.%20R.%20Luesse&amp;volume=20&amp;publication_year=2020&amp;pages=237&amp;pmid=32460700&amp;doi=10.1186/s12870-020-02392-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31">
<cite>
Kruse C. P. S., Wyatt S. E. (2022). Nitric oxide, gravity response, and a unified schematic of plant signaling. Plant Sci.
314, 111105. doi:  10.1016/j.plantsci.2021.111105

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.plantsci.2021.111105" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34895542/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Sci.&amp;title=Nitric%20oxide,%20gravity%20response,%20and%20a%20unified%20schematic%20of%20plant%20signaling&amp;author=C.%20P.%20S.%20Kruse&amp;author=S.%20E.%20Wyatt&amp;volume=314&amp;publication_year=2022&amp;pages=111105&amp;pmid=34895542&amp;doi=10.1016/j.plantsci.2021.111105&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32">
<cite>
Lai L., Liberzon A., Hennessey J., Jiang G., Qi J., Mesirov J. P., et al. (2012). AraPath: a knowledgebase for pathway analysis in Arabidopsis. Bioinformatics
28, 2291–2292. doi:  10.1093/bioinformatics/bts421

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/bts421" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3426845/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22760305/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=AraPath:%20a%20knowledgebase%20for%20pathway%20analysis%20in%20Arabidopsis&amp;author=L.%20Lai&amp;author=A.%20Liberzon&amp;author=J.%20Hennessey&amp;author=G.%20Jiang&amp;author=J.%20Qi&amp;volume=28&amp;publication_year=2012&amp;pages=2291-2292&amp;pmid=22760305&amp;doi=10.1093/bioinformatics/bts421&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B33">
<cite>
Lamers J., van der Meer T., Testerink C. (2020). How plants sense and respond to stressful environments. Plant Physiol.
182, 1624–1635. doi:  10.1104/pp.19.01464

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.19.01464" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7140927/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32132112/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=How%20plants%20sense%20and%20respond%20to%20stressful%20environments&amp;author=J.%20Lamers&amp;author=T.%20van%20der%20Meer&amp;author=C.%20Testerink&amp;volume=182&amp;publication_year=2020&amp;pages=1624-1635&amp;pmid=32132112&amp;doi=10.1104/pp.19.01464&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34">
<cite>
Liu Y., Beyer A., Aebersold R. (2016). On the dependency of cellular protein levels on mRNA abundance. Cell
165, 535–550. doi:  10.1016/j.cell.2016.03.014

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cell.2016.03.014" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27104977/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=On%20the%20dependency%20of%20cellular%20protein%20levels%20on%20mRNA%20abundance&amp;author=Y.%20Liu&amp;author=A.%20Beyer&amp;author=R.%20Aebersold&amp;volume=165&amp;publication_year=2016&amp;pages=535-550&amp;pmid=27104977&amp;doi=10.1016/j.cell.2016.03.014&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35">
<cite>
Mazars C., Brière C., Grat S., Pichereaux C., Rossignol M., Pereda-Loth V., et al. (2014). Microsome-associated proteome modifications of <em>Arabidopsis</em> seedlings grown on board the International Space Station reveal the possible effect on plants of space stresses other than microgravity. Plant Signal Behav.
9, e29637. doi:  10.4161/psb.29637

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4161/psb.29637" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4205128/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25763699/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signal%20Behav.&amp;title=Microsome-associated%20proteome%20modifications%20of%20Arabidopsis%20seedlings%20grown%20on%20board%20the%20International%20Space%20Station%20reveal%20the%20possible%20effect%20on%20plants%20of%20space%20stresses%20other%20than%20microgravity&amp;author=C.%20Mazars&amp;author=C.%20Bri%C3%A8re&amp;author=S.%20Grat&amp;author=C.%20Pichereaux&amp;author=M.%20Rossignol&amp;volume=9&amp;publication_year=2014&amp;pages=e29637&amp;pmid=25763699&amp;doi=10.4161/psb.29637&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36">
<cite>
McDermaid A., Monier B., Zhao J., Liu B., Ma Q. (2019). Interpretation of differential gene expression results of RNA-seq data: review and integration. Brief Bioinform.
20, 2044–2054. doi:  10.1093/bib/bby067

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bib/bby067" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6954399/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30099484/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform.&amp;title=Interpretation%20of%20differential%20gene%20expression%20results%20of%20RNA-seq%20data:%20review%20and%20integration&amp;author=A.%20McDermaid&amp;author=B.%20Monier&amp;author=J.%20Zhao&amp;author=B.%20Liu&amp;author=Q.%20Ma&amp;volume=20&amp;publication_year=2019&amp;pages=2044-2054&amp;pmid=30099484&amp;doi=10.1093/bib/bby067&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B37">
<cite>
Medina F. J., Manzano A., Villacampa A., Ciska M., Herranz R. (2021). Understanding reduced gravity effects on early plant development before attempting life-support farming in the moon and mars. Front. Astronomy Space Sci.
8. doi:  10.3389/fspas.2021.729154
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fspas.2021.729154" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Astronomy%20Space%20Sci.&amp;title=Understanding%20reduced%20gravity%20effects%20on%20early%20plant%20development%20before%20attempting%20life-support%20farming%20in%20the%20moon%20and%20mars&amp;author=F.%20J.%20Medina&amp;author=A.%20Manzano&amp;author=A.%20Villacampa&amp;author=M.%20Ciska&amp;author=R.%20Herranz&amp;volume=8&amp;publication_year=2021&amp;doi=10.3389/fspas.2021.729154&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B38">
<cite>
Nakashima J., Pattathil S., Avci U., Chin S., Alan Sparks J., Hahn M. G., et al. (2023). Glycome profiling and immunohistochemistry uncover changes in cell walls of Arabidopsis thaliana roots during spaceflight. NPJ Microgravity
9, 68. doi:  10.1038/s41526-023-00312-0

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41526-023-00312-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10444889/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37608048/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Microgravity&amp;title=Glycome%20profiling%20and%20immunohistochemistry%20uncover%20changes%20in%20cell%20walls%20of%20Arabidopsis%20thaliana%20roots%20during%20spaceflight&amp;author=J.%20Nakashima&amp;author=S.%20Pattathil&amp;author=U.%20Avci&amp;author=S.%20Chin&amp;author=J.%20Alan%20Sparks&amp;volume=9&amp;publication_year=2023&amp;pages=68&amp;pmid=37608048&amp;doi=10.1038/s41526-023-00312-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B39">
<cite>
Nicholson W. L., Fajardo-Cavazos P., Turner C., Currie T. M., Gregory G., Jurca T., et al. (2021). Design and validation of a device for mitigating fluid microgravity effects in biological research in canister spaceflight hardware. Front. Space Technol.
2. doi:  10.3389/frspt.2021.797518
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/frspt.2021.797518" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Space%20Technol.&amp;title=Design%20and%20validation%20of%20a%20device%20for%20mitigating%20fluid%20microgravity%20effects%20in%20biological%20research%20in%20canister%20spaceflight%20hardware&amp;author=W.%20L.%20Nicholson&amp;author=P.%20Fajardo-Cavazos&amp;author=C.%20Turner&amp;author=T.%20M.%20Currie&amp;author=G.%20Gregory&amp;volume=2&amp;publication_year=2021&amp;doi=10.3389/frspt.2021.797518&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B40">
<cite>
Olanrewaju G. O., Kruse C. P. S., Wyatt S. E. (2023). Functional meta-analysis of the proteomic responses of arabidopsis seedlings to the spaceflight environment reveals multi-dimensional sources of variability across spaceflight experiments. Int. J. Mol. Sci.
24, 14425. doi:  10.3390/ijms241914425

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/ijms241914425" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10573023/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37833871/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Functional%20meta-analysis%20of%20the%20proteomic%20responses%20of%20arabidopsis%20seedlings%20to%20the%20spaceflight%20environment%20reveals%20multi-dimensional%20sources%20of%20variability%20across%20spaceflight%20experiments&amp;author=G.%20O.%20Olanrewaju&amp;author=C.%20P.%20S.%20Kruse&amp;author=S.%20E.%20Wyatt&amp;volume=24&amp;publication_year=2023&amp;pages=14425&amp;pmid=37833871&amp;doi=10.3390/ijms241914425&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B41">
<cite>
Paponov I. A., Fliegmann J., Narayana R., Maffei M. E. (2021). Differential root and shoot magnetoresponses in Arabidopsis thaliana. Sci. Rep.
11, 9195. doi:  10.1038/s41598-021-88695-6

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-021-88695-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8080623/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33911161/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Differential%20root%20and%20shoot%20magnetoresponses%20in%20Arabidopsis%20thaliana&amp;author=I.%20A.%20Paponov&amp;author=J.%20Fliegmann&amp;author=R.%20Narayana&amp;author=M.%20E.%20Maffei&amp;volume=11&amp;publication_year=2021&amp;pages=9195&amp;pmid=33911161&amp;doi=10.1038/s41598-021-88695-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B42">
<cite>
Paul A.-L., Sng N. J., Zupanska A. K., Krishnamurthy A., Schultz E. R., Ferl R. J. (2017). Genetic dissection of the Arabidopsis spaceflight transcriptome: Are some responses dispensable for the physiological adaptation of plants to spaceflight? PLoS One
12, e0180186. doi:  10.1371/journal.pone.0180186

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0180186" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5491145/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28662188/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Genetic%20dissection%20of%20the%20Arabidopsis%20spaceflight%20transcriptome:%20Are%20some%20responses%20dispensable%20for%20the%20physiological%20adaptation%20of%20plants%20to%20spaceflight&amp;author=A.-L.%20Paul&amp;author=N.%20J.%20Sng&amp;author=A.%20K.%20Zupanska&amp;author=A.%20Krishnamurthy&amp;author=E.%20R.%20Schultz&amp;volume=12&amp;publication_year=2017&amp;pages=e0180186&amp;pmid=28662188&amp;doi=10.1371/journal.pone.0180186&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B43">
<cite>
Paul A.-L., Zupanska A. K., Ostrow D. T., Zhang Y., Sun Y., Li J.-L., et al. (2012). Spaceflight transcriptomes: unique responses to a novel environment. Astrobiology
12, 40–56. doi:  10.1089/ast.2011.0696

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2011.0696" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3264962/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22221117/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Spaceflight%20transcriptomes:%20unique%20responses%20to%20a%20novel%20environment&amp;author=A.-L.%20Paul&amp;author=A.%20K.%20Zupanska&amp;author=D.%20T.%20Ostrow&amp;author=Y.%20Zhang&amp;author=Y.%20Sun&amp;volume=12&amp;publication_year=2012&amp;pages=40-56&amp;pmid=22221117&amp;doi=10.1089/ast.2011.0696&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B44">
<cite>
Paul A.-L., Zupanska A. K., Schultz E. R., Ferl R. J. (2013). Organ-specific remodeling of the Arabidopsis transcriptome in response to spaceflight. BMC Plant Biol.
13, 112. doi:  10.1186/1471-2229-13-112

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2229-13-112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3750915/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23919896/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biol.&amp;title=Organ-specific%20remodeling%20of%20the%20Arabidopsis%20transcriptome%20in%20response%20to%20spaceflight&amp;author=A.-L.%20Paul&amp;author=A.%20K.%20Zupanska&amp;author=E.%20R.%20Schultz&amp;author=R.%20J.%20Ferl&amp;volume=13&amp;publication_year=2013&amp;pages=112&amp;pmid=23919896&amp;doi=10.1186/1471-2229-13-112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B45">
<cite>
Posit team  (2023). RStudio: Integrated Development Environment for R (Boston, MA: Posit Software, PBC; ). Available at: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.posit.co/" rel="noopener noreferrer" target="_blank">http://www.posit.co/</a>.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=RStudio:%20Integrated%20Development%20Environment%20for%20R&amp;publication_year=2023&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B46">
<cite>
Rioseras B., Shliaha P. V., Gorshkov V., Yagüe P., López-García M. T., Gonzalez-Quiñonez N., et al. (2018). Quantitative proteome and phosphoproteome analyses of streptomyces coelicolor reveal proteins and phosphoproteins modulating differentiation and secondary metabolism. Mol. Cell. Proteomics
17, 1591–1611. doi:  10.1074/mcp.RA117.000515

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/mcp.RA117.000515" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6072539/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29784711/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Quantitative%20proteome%20and%20phosphoproteome%20analyses%20of%20streptomyces%20coelicolor%20reveal%20proteins%20and%20phosphoproteins%20modulating%20differentiation%20and%20secondary%20metabolism&amp;author=B.%20Rioseras&amp;author=P.%20V.%20Shliaha&amp;author=V.%20Gorshkov&amp;author=P.%20Yag%C3%BCe&amp;author=M.%20T.%20L%C3%B3pez-Garc%C3%ADa&amp;volume=17&amp;publication_year=2018&amp;pages=1591-1611&amp;pmid=29784711&amp;doi=10.1074/mcp.RA117.000515&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B47">
<cite>
Sarasketa A., González-Moro M. B., González-Murua C., Marino D. (2016). Nitrogen source and external medium pH interaction differentially affects root and shoot metabolism in arabidopsis. Front. Plant Sci.
7. doi:  10.3389/fpls.2016.00029
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2016.00029" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4734181/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26870054/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=Nitrogen%20source%20and%20external%20medium%20pH%20interaction%20differentially%20affects%20root%20and%20shoot%20metabolism%20in%20arabidopsis&amp;author=A.%20Sarasketa&amp;author=M.%20B.%20Gonz%C3%A1lez-Moro&amp;author=C.%20Gonz%C3%A1lez-Murua&amp;author=D.%20Marino&amp;volume=7&amp;publication_year=2016&amp;pmid=26870054&amp;doi=10.3389/fpls.2016.00029&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B48">
<cite>
Sherman B. T., Hao M., Qiu J., Jiao X., Baseler M. W., Lane H. C., et al. (2022). DAVID: a web server for functional enrichment analysis and functional annotation of gene lists, (2021 update). Nucleic Acids Res.
50, W216–W221. doi:  10.1093/nar/gkac194

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkac194" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9252805/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35325185/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=DAVID:%20a%20web%20server%20for%20functional%20enrichment%20analysis%20and%20functional%20annotation%20of%20gene%20lists,%20(2021%20update)&amp;author=B.%20T.%20Sherman&amp;author=M.%20Hao&amp;author=J.%20Qiu&amp;author=X.%20Jiao&amp;author=M.%20W.%20Baseler&amp;volume=50&amp;publication_year=2022&amp;pages=W216-W221&amp;pmid=35325185&amp;doi=10.1093/nar/gkac194&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B49">
<cite>
Stankovic B. (2018). “Plants in Space,” in Into Space – A Journey of How Humans Adapt and Live in Microgravity (InTech; ). doi:  10.5772/intechopen.74230
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.5772/intechopen.74230" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Into%20Space%20%E2%80%93%20A%20Journey%20of%20How%20Humans%20Adapt%20and%20Live%20in%20Microgravity&amp;author=B.%20Stankovic&amp;publication_year=2018&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B50">
<cite>
Stiti N., Podgórska K. A., Bartels D. (2020). S-Nitrosation impairs activity of stress-inducible aldehyde dehydrogenases from Arabidopsis thaliana. Plant Sci.
292, 110389. doi:  10.1016/j.plantsci.2019.110389

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.plantsci.2019.110389" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32005394/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Sci.&amp;title=S-Nitrosation%20impairs%20activity%20of%20stress-inducible%20aldehyde%20dehydrogenases%20from%20Arabidopsis%20thaliana&amp;author=N.%20Stiti&amp;author=K.%20A.%20Podg%C3%B3rska&amp;author=D.%20Bartels&amp;volume=292&amp;publication_year=2020&amp;pages=110389&amp;pmid=32005394&amp;doi=10.1016/j.plantsci.2019.110389&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B51">
<cite>
Stutte G. W., Monje O., Goins G. D., Tripathy B. C. (2005). Microgravity effects on thylakoid, single leaf, and whole canopy photosynthesis of dwarf wheat. Planta
223, 46–56. doi:  10.1007/s00425-005-0066-2

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00425-005-0066-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16160842/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Microgravity%20effects%20on%20thylakoid,%20single%20leaf,%20and%20whole%20canopy%20photosynthesis%20of%20dwarf%20wheat&amp;author=G.%20W.%20Stutte&amp;author=O.%20Monje&amp;author=G.%20D.%20Goins&amp;author=B.%20C.%20Tripathy&amp;volume=223&amp;publication_year=2005&amp;pages=46-56&amp;pmid=16160842&amp;doi=10.1007/s00425-005-0066-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B52">
<cite>
Vandenbrink J. P., Herranz R., Medina F. J., Edelmann R. E., Kiss J. Z. (2016). A novel blue-light phototropic response is revealed in roots of Arabidopsis thaliana in microgravity. Planta
244, 1201–1215. doi:  10.1007/s00425-016-2581-8

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00425-016-2581-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5748516/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27507239/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=A%20novel%20blue-light%20phototropic%20response%20is%20revealed%20in%20roots%20of%20Arabidopsis%20thaliana%20in%20microgravity&amp;author=J.%20P.%20Vandenbrink&amp;author=R.%20Herranz&amp;author=F.%20J.%20Medina&amp;author=R.%20E.%20Edelmann&amp;author=J.%20Z.%20Kiss&amp;volume=244&amp;publication_year=2016&amp;pages=1201-1215&amp;pmid=27507239&amp;doi=10.1007/s00425-016-2581-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B53">
<cite>
Vandenbrink J. P., Herranz R., Poehlman W. L., Alex Feltus F., Villacampa A., Ciska M., et al. (2019). RNA -seq analyses of <em>Arabidopsis thaliana</em> seedlings after exposure to blue-light phototropic stimuli in microgravity. Am. J. Bot.
106, 1466–1476. doi:  10.1002/ajb2.1384

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/ajb2.1384" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31709515/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Bot.&amp;title=RNA%20-seq%20analyses%20of%20Arabidopsis%20thaliana%20seedlings%20after%20exposure%20to%20blue-light%20phototropic%20stimuli%20in%20microgravity&amp;author=J.%20P.%20Vandenbrink&amp;author=R.%20Herranz&amp;author=W.%20L.%20Poehlman&amp;author=F.%20Alex%20Feltus&amp;author=A.%20Villacampa&amp;volume=106&amp;publication_year=2019&amp;pages=1466-1476&amp;pmid=31709515&amp;doi=10.1002/ajb2.1384&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B54">
<cite>
Vandenbrink J. P., Kiss J. Z., Herranz R., Medina F. J. (2014). Light and gravity signals synergize in modulating plant development. Front. Plant Sci.
5. doi:  10.3389/fpls.2014.00563
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2014.00563" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4211383/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25389428/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=Light%20and%20gravity%20signals%20synergize%20in%20modulating%20plant%20development&amp;author=J.%20P.%20Vandenbrink&amp;author=J.%20Z.%20Kiss&amp;author=R.%20Herranz&amp;author=F.%20J.%20Medina&amp;volume=5&amp;publication_year=2014&amp;pmid=25389428&amp;doi=10.3389/fpls.2014.00563&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B55">
<cite>
Veljanovski V., Vanderbeld B., Knowles V. L., Snedden W. A., Plaxton W. C. (2006). Biochemical and molecular characterization of atPAP26, a vacuolar purple acid phosphatase up-regulated in phosphate-deprived arabidopsis suspension cells and seedlings. Plant Physiol.
142, 1282–1293. doi:  10.1104/pp.106.087171

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.106.087171" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1630754/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16963519/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Biochemical%20and%20molecular%20characterization%20of%20atPAP26,%20a%20vacuolar%20purple%20acid%20phosphatase%20up-regulated%20in%20phosphate-deprived%20arabidopsis%20suspension%20cells%20and%20seedlings&amp;author=V.%20Veljanovski&amp;author=B.%20Vanderbeld&amp;author=V.%20L.%20Knowles&amp;author=W.%20A.%20Snedden&amp;author=W.%20C.%20Plaxton&amp;volume=142&amp;publication_year=2006&amp;pages=1282-1293&amp;pmid=16963519&amp;doi=10.1104/pp.106.087171&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B56">
<cite>
Wakabayashi K., Soga K., Hoson T., Kotake T., Yamazaki T., Ishioka N., et al. (2020). Microgravity affects the level of matrix polysaccharide 1,3:1,4-β-glucans in cell walls of rice shoots by increasing the expression level of a gene involved in their breakdown. Astrobiology
20, 820–829. doi:  10.1089/ast.2019.2140

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2019.2140" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32207981/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Microgravity%20affects%20the%20level%20of%20matrix%20polysaccharide%201,3:1,4-%CE%B2-glucans%20in%20cell%20walls%20of%20rice%20shoots%20by%20increasing%20the%20expression%20level%20of%20a%20gene%20involved%20in%20their%20breakdown&amp;author=K.%20Wakabayashi&amp;author=K.%20Soga&amp;author=T.%20Hoson&amp;author=T.%20Kotake&amp;author=T.%20Yamazaki&amp;volume=20&amp;publication_year=2020&amp;pages=820-829&amp;pmid=32207981&amp;doi=10.1089/ast.2019.2140&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B57">
<cite>
Wang W., Wang Y., Chen T., Qin G., Tian S. (2022). Current insights into posttranscriptional regulation of fleshy fruit ripening. Plant Physiol. 192 (3), 1785–1798. doi:  10.1093/plphys/kiac483
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/plphys/kiac483" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC10315313/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36250906/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Current%20insights%20into%20posttranscriptional%20regulation%20of%20fleshy%20fruit%20ripening&amp;author=W.%20Wang&amp;author=Y.%20Wang&amp;author=T.%20Chen&amp;author=G.%20Qin&amp;author=S.%20Tian&amp;volume=192&amp;issue=3&amp;publication_year=2022&amp;pages=1785-1798&amp;pmid=36250906&amp;doi=10.1093/plphys/kiac483&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B58">
<cite>
Wolinska K. W., Berens M. L. (2019). Optimal Defense Theory 2.0: tissue-specific stress defense prioritization as an extra layer of complexity. Commun. Integr. Biol.
12, 91–95. doi:  10.1080/19420889.2019.1625661

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/19420889.2019.1625661" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6615527/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31308873/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Commun.%20Integr.%20Biol.&amp;title=Optimal%20Defense%20Theory%202.0:%20tissue-specific%20stress%20defense%20prioritization%20as%20an%20extra%20layer%20of%20complexity&amp;author=K.%20W.%20Wolinska&amp;author=M.%20L.%20Berens&amp;volume=12&amp;publication_year=2019&amp;pages=91-95&amp;pmid=31308873&amp;doi=10.1080/19420889.2019.1625661&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B59">
<cite>
Yang N. J., Hinner M. J. (2015). “Getting across the cell membrane: an overview for small molecules, peptides, and proteins.” in Site-Specific Protein Labeling. Methods in Molecular Biology. Eds. Gautier A., Hinner M. (New York, NY: Humana Press; ), 1266. doi:  10.1007/978-1-4939-2272-7_3
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-1-4939-2272-7_3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4891184/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25560066/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Site-Specific%20Protein%20Labeling.%20Methods%20in%20Molecular%20Biology&amp;author=N.%20J.%20Yang&amp;author=M.%20J.%20Hinner&amp;publication_year=2015&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B60">
<cite>
Zhang Y., Wang L., Xie J., Zheng H. (2015). Differential protein expression profiling of Arabidopsis thaliana callus under microgravity on board the Chinese SZ-8 spacecraft. Planta
241, 475–488. doi:  10.1007/s00425-014-2196-x

</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00425-014-2196-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25374148/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Differential%20protein%20expression%20profiling%20of%20Arabidopsis%20thaliana%20callus%20under%20microgravity%20on%20board%20the%20Chinese%20SZ-8%20spacecraft&amp;author=Y.%20Zhang&amp;author=L.%20Wang&amp;author=J.%20Xie&amp;author=H.%20Zheng&amp;volume=241&amp;publication_year=2015&amp;pages=475-488&amp;pmid=25374148&amp;doi=10.1007/s00425-014-2196-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ol></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10712242/bin/DataSheet_1.pdf">Click here for additional data file.</a><sup> (1.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10712242/bin/DataSheet_2.zip">Click here for additional data file.</a><sup> (162.5KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10712242/bin/DataSheet_3.pdf">Click here for additional data file.</a><sup> (1,019.7KB, pdf) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found below: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://osdr.nasa.gov/bio/repo/data/studies/OSD-522/preview/w5Q5ElZE-Wy6DDS1ZFzeRjBAPuKss-x0" rel="noopener noreferrer" target="_blank">https://osdr.nasa.gov/bio/repo/data/studies/OSD-522/preview/w5Q5ElZE-Wy6DDS1ZFzeRjBAPuKss-x0</a>, OSD-522.</p></section></section></section></section></article>
</main></body></html>