<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10264680/"/><title>The EXO70 inhibitor Endosidin2 alters plasma membrane protein composition in Arabidopsis roots - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Front Plant Sci</button></div>. 2023 May 31;14:1171957. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2023.1171957" rel="noopener noreferrer" target="_blank">10.3389/fpls.2023.1171957</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The EXO70 inhibitor Endosidin2 alters plasma membrane protein composition in Arabidopsis roots</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D"><span class="name western">Xiaohui Li</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xiaohui Li</span></h3>
<div class="p">
<sup>1</sup>
Department of Botany and Plant Pathology, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">
<sup>2</sup>
Center for Plant Biology, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D"><span class="name western">Xiaohui Li</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20P%22%5BAuthor%5D"><span class="name western">Peipei Zhu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Peipei Zhu</span></h3>
<div class="p">
<sup>3</sup>
Department of Biochemistry, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">
<sup>4</sup>
Department of Chemistry, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20P%22%5BAuthor%5D"><span class="name western">Peipei Zhu</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20YJ%22%5BAuthor%5D"><span class="name western">Yen-Ju Chen</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yen-Ju Chen</span></h3>
<div class="p">
<sup>1</sup>
Department of Botany and Plant Pathology, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">
<sup>2</sup>
Center for Plant Biology, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20YJ%22%5BAuthor%5D"><span class="name western">Yen-Ju Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20L%22%5BAuthor%5D"><span class="name western">Lei Huang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Lei Huang</span></h3>
<div class="p">
<sup>1</sup>
Department of Botany and Plant Pathology, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">
<sup>2</sup>
Center for Plant Biology, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20L%22%5BAuthor%5D"><span class="name western">Lei Huang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20D%22%5BAuthor%5D"><span class="name western">Diwen Wang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Diwen Wang</span></h3>
<div class="p">
<sup>1</sup>
Department of Botany and Plant Pathology, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">
<sup>2</sup>
Center for Plant Biology, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20D%22%5BAuthor%5D"><span class="name western">Diwen Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Newton%20DT%22%5BAuthor%5D"><span class="name western">David T Newton</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">David T Newton</span></h3>
<div class="p">
<sup>5</sup>
Department of Statistics, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Newton%20DT%22%5BAuthor%5D"><span class="name western">David T Newton</span></a>
</div>
</div>
<sup>5</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hsu%20CC%22%5BAuthor%5D"><span class="name western">Chuan-Chih Hsu</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Chuan-Chih Hsu</span></h3>
<div class="p">
<sup>3</sup>
Department of Biochemistry, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hsu%20CC%22%5BAuthor%5D"><span class="name western">Chuan-Chih Hsu</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20G%22%5BAuthor%5D"><span class="name western">Guang Lin</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Guang Lin</span></h3>
<div class="p">
<sup>6</sup>
Department of Mathematics, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">
<sup>7</sup>
School of Mechanical Engineering, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20G%22%5BAuthor%5D"><span class="name western">Guang Lin</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a aria-describedby="id9" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tao%20WA%22%5BAuthor%5D"><span class="name western">W Andy Tao</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">W Andy Tao</span></h3>
<div class="p">
<sup>3</sup>
Department of Biochemistry, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">
<sup>4</sup>
Department of Chemistry, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tao%20WA%22%5BAuthor%5D"><span class="name western">W Andy Tao</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a aria-describedby="id10" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Staiger%20CJ%22%5BAuthor%5D"><span class="name western">Christopher J Staiger</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Christopher J Staiger</span></h3>
<div class="p">
<sup>1</sup>
Department of Botany and Plant Pathology, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">
<sup>2</sup>
Center for Plant Biology, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">
<sup>8</sup>
Department of Biological Sciences, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Staiger%20CJ%22%5BAuthor%5D"><span class="name western">Christopher J Staiger</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>8,</sup><sup>*</sup>, <a aria-describedby="id11" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20C%22%5BAuthor%5D"><span class="name western">Chunhua Zhang</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Chunhua Zhang</span></h3>
<div class="p">
<sup>1</sup>
Department of Botany and Plant Pathology, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">
<sup>2</sup>
Center for Plant Biology, Purdue University, West Lafayette, IN, United States
</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20C%22%5BAuthor%5D"><span class="name western">Chunhua Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>
Department of Botany and Plant Pathology, Purdue University, West Lafayette, IN, United States
</div>
<div id="aff2">
<sup>2</sup>
Center for Plant Biology, Purdue University, West Lafayette, IN, United States
</div>
<div id="aff3">
<sup>3</sup>
Department of Biochemistry, Purdue University, West Lafayette, IN, United States
</div>
<div id="aff4">
<sup>4</sup>
Department of Chemistry, Purdue University, West Lafayette, IN, United States
</div>
<div id="aff5">
<sup>5</sup>
Department of Statistics, Purdue University, West Lafayette, IN, United States
</div>
<div id="aff6">
<sup>6</sup>
Department of Mathematics, Purdue University, West Lafayette, IN, United States
</div>
<div id="aff7">
<sup>7</sup>
School of Mechanical Engineering, Purdue University, West Lafayette, IN, United States
</div>
<div id="aff8">
<sup>8</sup>
Department of Biological Sciences, Purdue University, West Lafayette, IN, United States
</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Edited by: Emily R. Larson, University of Bristol, United Kingdom</p></div>
<div class="fn" id="fn2"><p>Reviewed by: Fatima Cvrckova, Charles University, Czechia; Jitka Ortmannová, (ASCR), Czechia; Jinbo Shen, Zhejiang Agriculture and Forestry University, China</p></div>
<div class="fn" id="fn001">
<sup>✉</sup><p class="display-inline">*Correspondence: Christopher J. Staiger, <span>staiger@purdue.edu</span>
</p>
</div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2023 Feb 22; Accepted 2023 May 15; Collection date 2023.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2023 Li, Zhu, Chen, Huang, Wang, Newton, Hsu, Lin, Tao, Staiger and Zhang</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC10264680  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37324680/">37324680</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>To sustain normal growth and allow rapid responses to environmental cues, plants alter the plasma membrane protein composition under different conditions presumably by regulation of delivery, stability, and internalization. Exocytosis is a conserved cellular process that delivers proteins and lipids to the plasma membrane or extracellular space in eukaryotes. The octameric exocyst complex contributes to exocytosis by tethering secretory vesicles to the correct site for membrane fusion; however, whether the exocyst complex acts universally for all secretory vesicle cargo or just for specialized subsets used during polarized growth and trafficking is currently unknown. In addition to its role in exocytosis, the exocyst complex is also known to participate in membrane recycling and autophagy. Using a previously identified small molecule inhibitor of the plant exocyst complex subunit EXO70A1, Endosidin2 (ES2), combined with a plasma membrane enrichment method and quantitative proteomic analysis, we examined the composition of plasma membrane proteins in the root of Arabidopsis seedlings, after inhibition of the ES2-targetted exocyst complex, and verified our findings by live imaging of GFP-tagged plasma membrane proteins in root epidermal cells. The abundance of 145 plasma membrane proteins was significantly reduced following short-term ES2 treatments and these likely represent candidate cargo proteins of exocyst-mediated trafficking. Gene Ontology analysis showed that these proteins play diverse functions in cell growth, cell wall biosynthesis, hormone signaling, stress response, membrane transport, and nutrient uptake. Additionally, we quantified the effect of ES2 on the spatial distribution of EXO70A1 with live-cell imaging. Our results indicate that the plant exocyst complex mediates constitutive dynamic transport of subsets of plasma membrane proteins during normal root growth.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> plasma membrane protein enrichment, exocytosis, proteomics, small molecule inhibitors, exocyst</p></section></section><section id="s1"><h2 class="pmc_sec_title">Highlights</h2>
<p>• The proteomics profile of Arabidopsis plasma membrane proteins changes following 2-h treatment with the EXO70A1 inhibitor, ES2.</p></section><section id="s2"><h2 class="pmc_sec_title">Introduction</h2>
<p>The plant plasma membrane serves as a selective physical barrier between intracellular contents and the environment as well as a signaling platform for transmitting information from the exterior to induce cellular responses. Plasma membrane proteins can be classified as integral, lipid-anchored, and peripheral, based on the mode of association with the plasma membrane (<a aria-describedby="B68" class="usa-link" href="#B68">Stillwell, 2016</a>). Integral membrane proteins possess one to several transmembrane domains; lipid-anchored proteins associate with the plasma membrane through covalently attached fatty acids; peripheral membrane proteins associate with the plasma membrane through interactions with other membrane proteins, polar head groups of the lipid bilayer, or both. Several well-characterized plasma membrane-localized nutrient transporters, hormone receptors and transporters, and cell wall biosynthetic enzymes are known to undergo constitutive delivery and turnover (<a aria-describedby="B47" class="usa-link" href="#B47">Li et al., 2002</a>; <a aria-describedby="B28" class="usa-link" href="#B28">Friml, 2010</a>; <a aria-describedby="B72" class="usa-link" href="#B72">Takano et al., 2010</a>; <a aria-describedby="B39" class="usa-link" href="#B39">Kleine-Vehn et al., 2011</a>; <a aria-describedby="B37" class="usa-link" href="#B37">Irani et al., 2012</a>; <a aria-describedby="B3" class="usa-link" href="#B3">Bashline et al., 2013</a>; <a aria-describedby="B8" class="usa-link" href="#B8">Chen et al., 2015</a>; <a aria-describedby="B79" class="usa-link" href="#B79">Wang et al., 2015</a>; <a aria-describedby="B94" class="usa-link" href="#B94">Zhu et al., 2018</a>; <a aria-describedby="B76" class="usa-link" href="#B76">Vellosillo et al., 2021</a>). Proteomic analyses of plasma membrane proteins reveal that the composition is altered in response to biotic and abiotic stresses (<a aria-describedby="B54" class="usa-link" href="#B54">Nohzadeh Malakshah et al., 2007</a>; <a aria-describedby="B55" class="usa-link" href="#B55">Nouri and Komatsu, 2010</a>; <a aria-describedby="B24" class="usa-link" href="#B24">Elmore et al., 2012</a>; <a aria-describedby="B71" class="usa-link" href="#B71">Takahashi et al., 2013</a>; <a aria-describedby="B53" class="usa-link" href="#B53">Nie et al., 2015</a>; <a aria-describedby="B6" class="usa-link" href="#B6">Cao et al., 2016</a>; <a aria-describedby="B7" class="usa-link" href="#B7">Chen et al., 2016</a>; <a aria-describedby="B77" class="usa-link" href="#B77">Voothuluru et al., 2016</a>); however, the mechanisms for delivery of most of these plasma membrane proteins are not well characterized. Investigating how plants maintain the variety, abundance, and dynamics of plasma membrane proteins is essential for understanding cell growth, development, and response to stresses.</p>
<p>The conventional plant secretory pathway involves trafficking through the endoplasmic reticulum, Golgi and post-Golgi vesicles to deliver lipids and proteins to the plasma membrane or extracellular space (<a aria-describedby="B14" class="usa-link" href="#B14">Denecke et al., 2012</a>; <a aria-describedby="B18" class="usa-link" href="#B18">Drakakaki and Dandekar, 2013</a>; <a aria-describedby="B95" class="usa-link" href="#B95">Zhu et al., 2020</a>; <a aria-describedby="B1" class="usa-link" href="#B1">Aniento et al., 2022</a>). The exocyst complex is a conserved octameric protein complex that primarily functions to mediate the tethering of secretory vesicles to the site of membrane fusion during the last step of exocytic trafficking (<a aria-describedby="B23" class="usa-link" href="#B23">Elias et al., 2003</a>; <a aria-describedby="B91" class="usa-link" href="#B91">Zhang et al., 2010</a>; <a aria-describedby="B87" class="usa-link" href="#B87">Žárský et al., 2013</a>; <a aria-describedby="B84" class="usa-link" href="#B84">Wu and Guo, 2015</a>; <a aria-describedby="B69" class="usa-link" href="#B69">Synek et al., 2021</a>; <a aria-describedby="B86" class="usa-link" href="#B86">Žárský, 2022</a>). In addition, the exocyst complex also associates with the endocytic and autophagy machinery to modulate membrane recycling and autophagy (<a aria-describedby="B62" class="usa-link" href="#B62">Prigent et al., 2003</a>; <a aria-describedby="B41" class="usa-link" href="#B41">Kulich et al., 2013</a>; <a aria-describedby="B87" class="usa-link" href="#B87">Žárský et al., 2013</a>; <a aria-describedby="B60" class="usa-link" href="#B60">Ortega et al., 2022</a>; <a aria-describedby="B86" class="usa-link" href="#B86">Žárský, 2022</a>). One plant exocyst subunit, EXO70A1, was recently characterized as a landmark for exocyst complex recruitment to the plasma membrane, through its interactions with anionic phospholipids (<a aria-describedby="B69" class="usa-link" href="#B69">Synek et al., 2021</a>). Loss of EXO70A1 function disrupts the recruitment of other exocyst subunits to the plasma membrane in Arabidopsis root epidermal cells (<a aria-describedby="B69" class="usa-link" href="#B69">Synek et al., 2021</a>). Some known cargo proteins for plant exocyst complex-mediated trafficking include PIN auxin transporters (<a aria-describedby="B20" class="usa-link" href="#B20">Drdová et al., 2013</a>), brassinosteroid receptor (<a aria-describedby="B88" class="usa-link" href="#B88">Zhang et al., 2016</a>), and cellulose synthase complexes (<a aria-describedby="B94" class="usa-link" href="#B94">Zhu et al., 2018</a>; <a aria-describedby="B89" class="usa-link" href="#B89">Zhang et al., 2021</a>). However, it is not known what other plasma membrane proteins are delivered to the plasma membrane through the exocyst complex. Whereas the half-life for turnover of plasma membrane proteins in mammalian cells varies from a few hours to a few days (<a aria-describedby="B73" class="usa-link" href="#B73">Tweto and Doyle, 1976</a>; <a aria-describedby="B17" class="usa-link" href="#B17">Doyle and Baumann, 1979</a>; <a aria-describedby="B9" class="usa-link" href="#B9">Chu and Doyle, 1985</a>), we know much less about the mechanisms and dynamics of protein delivery to the plasma membrane in plants.</p>
<p>Previously, the small molecule Endosidin2 (ES2) was shown to target the EXO70A1 subunit of Arabidopsis exocyst complex and to inhibit exocytosis in plant and mammalian cells (<a aria-describedby="B88" class="usa-link" href="#B88">Zhang et al., 2016</a>; <a aria-describedby="B36" class="usa-link" href="#B36">Huang et al., 2019</a>). Short-term ES2 treatment inhibits the transport of PIN-FORMED (PIN) family auxin transporter, PIN2, and the brassinosteroid receptor BRI1, to the plasma membrane (<a aria-describedby="B88" class="usa-link" href="#B88">Zhang et al., 2016</a>), and also impairs the recycling of PIN2 (<a aria-describedby="B44" class="usa-link" href="#B44">Lešková et al., 2020</a>). In mammalian cells, ES2 treatment reduces the cell surface abundance of a membrane-anchored metalloproteinase MT2-MMP (<a aria-describedby="B29" class="usa-link" href="#B29">Gómez-Escudero et al., 2017</a>). In addition, given the role of the exocyst complex in autophagy, a recent study examined the consequences of ES2 treatment in autophagy events in mammalian cells, and found that a 24-h ES2 treatment resulted in decreased autophagy (<a aria-describedby="B60" class="usa-link" href="#B60">Ortega et al., 2022</a>). To characterize changes in plasma membrane protein composition after short-term inhibition of the exocyst complex, and to identify additional proteins that are transported through the exocyst complex, we used ES2 to inhibit the exocyst complex. We enriched plasma membrane from roots of Arabidopsis seedlings treated with ES2 and DMSO solvent control and identified proteins using mass spectrometry analysis. We then performed quantitative and comparative proteomic analyses to identify plasma membrane proteins with altered abundance in ES2-treated samples and considered proteins with reduced abundance as candidate cargo proteins of exocyst-mediated trafficking in plants. We identified a subset of proteins with significantly reduced abundance at the plasma membrane after two hours of 40 μM ES2 treatment. Gene Ontology analysis of proteins with reduced abundance demonstrated enrichment for functions in cell growth, cell wall biosynthesis, membrane transport, hormone signaling, and stress responses. The abundance of many protein kinases and several uncharacterized proteins was reduced by ES2 treatment as well. Our results show that the exocyst complex mediates dynamic transport of a diverse set of proteins essential for plant growth and environmental adaptation.</p></section><section id="s3"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="s3_1"><h3 class="pmc_sec_title">Plasma membrane enrichment</h3>
<p>
<em>Arabidopsis</em> PIP2A-GFP seeds (<a aria-describedby="B13" class="usa-link" href="#B13">Cutler et al., 2000</a>) were sterilized and sowed on half-strength Murashige and Skoog (½ MS) media with vitamins (Caisson Labs, Smithfield, UT, USA), 1% (w/v) sucrose, and 0.8% (w/v) agar (Fisher Scientific, Waltham, MA, USA). The seeds were well separated from each other in both horizontal and vertical orientations on 10 cm x 10 cm square petri plates. Plates were placed in vertical orientation so that the seedings grew on the surface of the agar media. The plants were grown in a growth chamber (Percival Scientific, Perry, IA, USA) under continuous light of 130 μmol m<sup>-2</sup>s<sup>-1</sup> at 23°C. Six-day-old seedlings were removed from the agar plates and transferred to liquid ½ MS media with 1% (w/v) sucrose and 40 μM ES2 or 0.5% DMSO (mock). After two hours of treatment under light, the roots of the seedlings were quickly excised and transferred to the plasma membrane enrichment buffer. The process of plasma membrane enrichment followed exactly the published protocol (<a aria-describedby="B12" class="usa-link" href="#B12">Collins et al., 2017</a>). To evaluate the enrichment of plasma membrane proteins by Western blot assay, 12 μg of protein from each fraction was loaded onto SDS-PAGE gels and the presence of PIP2A-GFP, Sec12, and SYP41 was detected after transfer to PVDF membrane. The anti-GFP antibody was used at 1:1000 dilution (Takara Bio USA, catalog # 632381, San Jose, CA, USA); anti-Sec12 antibody (<a aria-describedby="B2" class="usa-link" href="#B2">Bar-Peled and Raikhel, 1997</a>) was used at 1:5000 dilution; and anti-SYP41 antibody (<a aria-describedby="B4" class="usa-link" href="#B4">Bassham et al., 2000</a>) was used at 1:500 dilution. HRP-conjugated goat anti-mouse and goat anti-rabbit antibodies were used at 1:10,000 dilution. The plasma membrane enrichment experiments were repeated four times under identical conditions.</p></section><section id="s3_2"><h3 class="pmc_sec_title">Methanol-chloroform precipitation</h3>
<p>The mass spectrometry sample preparation procedure was based on previous literature (<a aria-describedby="B35" class="usa-link" href="#B35">Hsu et al., 2018</a>). Protein concentrations from enriched plasma membrane fractions were quantified using the bicinchoninic acid method. For each sample, 250 μg of enriched plasma membrane protein was processed for mass spectrometry analysis. Proteins from each sample were reduced and alkylated with 10 mM tris-(2-carboxyethyl)phosphine (TECP) and 40 mM chloroacetamide (CAA) at 95°C for 5 min. Methanol, chloroform, and ddH<sub>2</sub>O were added to each 100 µl sample in a volume ratio of 4:1:3, respectively (<a aria-describedby="B83" class="usa-link" href="#B83">Wessel and Flügge, 1984</a>). The solution was mixed and centrifuged at 16,000x g for 3 min to obtain aqueous and organic phases. Four volumes of methanol were added to the sample after removing the upper aqueous layer carefully without disturbing the intermediate protein disk. The solution was mixed and centrifuged again at the same settings. The supernatant was removed, the precipitated protein pellet was washed with 400 µL methanol, and the sample was air-dried.</p></section><section id="s3_3"><h3 class="pmc_sec_title">Protein digestion</h3>
<p>The precipitated protein pellet was dissolved with 100 µl of 12 mM sodium deoxycholate (SDC) and 12 mM sodium lauroyl sarcosinate (SLS) in 100 mM Tris-HCl (pH 8.5). Protein amount was quantified by BCA assay (Thermo Fisher Scientific). Dissolved protein extract was diluted 5-fold with 50 mM triethylammonium hydrogen carbonate buffer (TEAB) and digested with Lys-C (Wako Chemicals USA Inc., Richmond, VA, USA) in a ratio of 1:100 (w/w) enzyme-to-protein ratio for 3 h at 37°C, and trypsin (Sigma-Aldrich, St. Louis, MO, USA) in a ratio of 1:50 (w/w) enzyme-to-protein ratio for overnight at 37°C. SDC and SLS were removed by adding 100% acetyl acetate in a ratio of 1:1 (vol/vol) sample to acetyl acetate volume ratio. The digests were desalted by homemade stage tip with styrene divinyl benzene (SDB-XC) membrane (3M) after the digests were acidified with 10% trifluoroacetic acid (TFA) to a pH ~3.</p></section><section id="s3_4"><h3 class="pmc_sec_title">Dimethyl labeling</h3>
<p>Every 50–100 µg of tryptic peptides was dissolved in 100 µl of 100 mM TEAB and mixed with 4 µl of 4% <sup>13</sup>CD<sub>2</sub>O or <sup>12</sup>CH<sub>2</sub>O, and then 4 µl of freshly prepared 0.6 M sodium cyanoborohydride was immediately added (<a aria-describedby="B5" class="usa-link" href="#B5">Boersema et al., 2009</a>). The mixture was agitated for 1 h at room temperature. The reaction was quenched by adding 16 µl of ice-cold 1% ammonium hydroxide and agitated for 1 min. Heavy and light dimethyl-labeled peptides were mixed after the labeled peptides were acidified with 20 µl of 10% formic acid and then fractionated by basic pH reverse-phase stage tips.</p></section><section id="s3_5"><h3 class="pmc_sec_title">Basic pH reverse-phase fractionation</h3>
<p>Two mg of the Magic C18-AQ beads (5 µm particles) were suspended in 200 µL of methanol and loaded into a 200 µL StageTip with a 20 µm polypropylene frit (Agilent, Santa Clara, CA, USA) (<a aria-describedby="B15" class="usa-link" href="#B15">Dimayacyac-Esleta et al., 2015</a>). The C18 StageTip was activated with 50 µL of 100% acetonitrile (ACN) and washed with 50 µL of 200 mM ammonium formate, pH 10. After the StageTip was activated and washed, 25–50 µg of dimethyl-labeled peptides were fractionated from the StageTip with 50 µL of 6 different ACN concentrations: 5, 10, 15, 20, 25% and 80% of ACN in 200 mM ammonium formate, pH 10. The eluted peptides were dried and stored at -20°C.</p></section><section id="s3_6"><h3 class="pmc_sec_title">LC-MS/MS analysis</h3>
<p>Twenty-five µg of dimethyl-labeled peptides used in fractionations were dissolved in 5 µL of 0.3% formic acid (FA) with 3% ACN and 4 µL of each fraction injected into an EasynLC 1000 (Thermo Fisher Scientific). Peptides were separated on a 45 cm in-house packed column (360 µm OD × 75 µm ID) containing C18 resin (2.2 µm, 100Å, Michrom Bioresources, Auburn, CA, USA) with a 30 cm column heater (Analytical Sales and Services, Flanders, NJ, USA) set at 60°C. The mobile phase buffer for 25 µg peptide fractions consisted of 0.1% FA in ultrapure water (buffer A) with an eluting buffer of 0.1% FA in 80% ACN (buffer B) run with a 150 min gradient of 5–30% buffer B at a flow rate of 250 nL/min. The Easy-nLC 1000 was coupled online with a Velos Pro LTQ-Orbitrap mass spectrometer (Thermo Fisher Scientific). The mass spectrometer was operated in the data-dependent mode in which a full MS scan (from m/z 350–1500 with a resolution of 30,000 at m/z 400) was followed by MS/MS of the 10 most intense ions being subjected to collision-induced dissociation (CID) fragmentation (normalized collision energy–30%, automatic gain control–3E4, max injection time–100 ms).</p></section><section id="s3_7"><h3 class="pmc_sec_title">Proteomics data analyses</h3>
<p>The raw files were searched directly against the <em>Arabidopsis thaliana</em> protein database (TAIR10) with no redundant entries using MaxQuant software (version 1.6.1.0) (<a aria-describedby="B74" class="usa-link" href="#B74">Tyanova et al., 2016a</a>) with 1% false-discovery rate (FDR) cutoff for protein and peptide. The first peptide precursor mass tolerance was set at 20 ppm, and MS/MS tolerance was set at 0.6 Da. Search criteria included a static carbamidomethylation of cysteines and variable modifications of oxidation on methionine residues and acetylation at N-terminus of proteins. The search was performed with full tryptic digestion and allowed a maximum of two missed cleavages on the peptides analyzed from the sequence database. Dimethyl-labeling quantitation was performed by setting the multiplicity as 2 (DimethLys0 and DimethNter0; DimethLys6 and DimethNter6). Re-quantify and match between runs function were enabled.</p>
<p>After database searches, the Perseus software platform (Version 1.6.1.3) (<a aria-describedby="B75" class="usa-link" href="#B75">Tyanova et al., 2016b</a>) was used to further analyze the MaxQuant-derived protein groups file. First, the rows marked to be “only identified by site”, “reverse” and “potential contaminants” were filtered out. The Heavy/Light ratio was transformed to Light/Heavy (L/H) ratio for convenience, and then to log<sub>2</sub>L/H ratio. Normalization was performed by median subtraction. Proteins with less than 3 valid values among the 4 replicates were filtered out. One sample t-test was performed with both side and p-value threshold of 0.05. Proteins that passed t-test and with a mean log<sub>2</sub>L/H ratio ≤ -0.32193 were considered to be 20% reduction candidates. Histogram and multi-scatter plots were generated by Perseus software. The reduced abundance candidate list was uploaded to PANTHER platform for Gene Ontology (GO) analysis, including functional classification and statistical overrepresentation (<a aria-describedby="B51" class="usa-link" href="#B51">Mi et al., 2013</a>; <a aria-describedby="B50" class="usa-link" href="#B50">Mi et al., 2017</a>).</p>
<p>The Venn diagram was generated using the online tool provided by Van de Peer Lab at the Bioinformatics and Evolutionary Genomics Core of Ghent University, Belgium (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://bioinformatics.psb.ugent.be/webtools/Venn/" rel="noopener noreferrer" target="_blank">http://bioinformatics.psb.ugent.be/webtools/Venn/</a>). The presence of signal peptide and transmembrane domains in candidate proteins was predicted using the DeepTMHMM package (v1.0.24) provided by Department of Bio and Health Informatics, Technical University of Denmark (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://dtu.biolib.com/DeepTMHMM" rel="noopener noreferrer" target="_blank">https://dtu.biolib.com/DeepTMHMM</a>) (<a aria-describedby="B32" class="usa-link" href="#B32">Hallgren et al., 2022</a>).</p></section><section id="s3_8"><h3 class="pmc_sec_title">Confocal microscopy</h3>
<p>For imaging of fluorescence-tagged protein marker lines, 6-d-old light-grown seedlings expressing PIN2-GFP (<a aria-describedby="B85" class="usa-link" href="#B85">Xu and Scheres, 2005</a>), BRI1-GFP (<a aria-describedby="B80" class="usa-link" href="#B80">Wang et al., 2001</a>), FER-GFP (<a aria-describedby="B25" class="usa-link" href="#B25">Escobar-Restrepo et al., 2007</a>), PIP2A-GFP (<a aria-describedby="B13" class="usa-link" href="#B13">Cutler et al., 2000</a>), or GFP-EXO70A1 (<a aria-describedby="B27" class="usa-link" href="#B27">Fendrych et al., 2010</a>) were treated in liquid ½ MS medium containing 0.5% DMSO (mock) or 40 μM ES2 for 2 h. Live imaging was performed with a Zeiss LSM 710 Axio Imager laser-scanning confocal microscope operated with Zeiss ZEN software (Carl Zeiss Microscopy LLC, White Plains, NY, USA). The PIN2-GFP, BRI1-GFP, FER-GFP and PIP2A-GFP lines were imaged with a Zeiss Plan-Apochromat 100x/NA1.40 DIC Oil immersion objective at 2048×2048 pixel scanning resolution. The GFP-EXO70A1 line was imaged with a Zeiss Plan-Apochromat 63x/NA1.40 Oil immersion objective at 4096×4096 pixel scanning resolution. The GFP-EXO70A1 seedlings were stained with 10 μg/mL Propidium Iodide (PI) (Biotium Inc., Fremont, CA, USA; Cat # 40016) in ½ liquid MS medium containing DMSO or ES2 in the dark for 10 min, and quickly rinsed in PI-free medium before imaging. GFP was excited with a 488-nm laser line, and PI was excited with a 543-nm laser line. Images were collected from root epidermal cells at the transition zone.</p>
<p>Quantitative image analysis of GFP fluorescence intensity was performed using the FIJI distribution of ImageJ (Ver 1.54b). The regions of interest (ROI) were selected for plasma membrane, cytoplasmic background in the compartment-free region, and the entire cytoplasmic region. The mean fluorescence intensity of the GFP signal for each ROI was measured. The average PM fluorescence intensity was normalized by subtracting the mean fluorescence intensity of the cytoplasmic background. The plasma membrane / cytoplasm ratio was calculated by first subtracting the mean fluorescence intensity of the region outside of the root and then dividing the PM mean fluorescence intensity by the mean fluorescence intensity of the entire cytoplasmic region. Values for individual cells from each seedling were averaged and considered as one data point. Normalization and averaging were performed in Microsoft Excel. Statistical analyses using a Student’s <em>t</em>-test were conducted, and box plots were generated with GraphPad Prism software.</p></section></section><section id="s4"><h2 class="pmc_sec_title">Results</h2>
<section id="s4_1"><h3 class="pmc_sec_title">Plasma membrane enrichment</h3>
<p>We chose to use a transgenic line expressing PIP2A-GFP as the plant material for our experiments because PIP2A-GFP is localized predominantly at the plasma membrane by microscopy and its cellular localization is not affected by 2 h of 40 μM ES2 treatment (<a aria-describedby="B13" class="usa-link" href="#B13">Cutler et al., 2000</a>; <a aria-describedby="B88" class="usa-link" href="#B88">Zhang et al., 2016</a>). The GFP tag also allowed the use of anti-GFP antibody to examine the quality of enriched plasma membrane fractions. Six-day-old PIP2A-GFP seedlings were treated with either 40 μM ES2 or 0.5% DMSO for 2 h, a condition in which plant exocytosis is reduced but not completely inhibited (<a aria-describedby="B88" class="usa-link" href="#B88">Zhang et al., 2016</a>). After treatment, roots were excised and used for plasma membrane enrichment (<a class="usa-link" href="#f1">
<strong>Figure 1A</strong>
</a>). We adapted a protocol that allows quick enrichment of plasma membrane and does not require large amounts of plant tissue (<a aria-describedby="B92" class="usa-link" href="#B92">Zhang and Peck, 2011</a>; <a aria-describedby="B12" class="usa-link" href="#B12">Collins et al., 2017</a>).</p>
<figure class="fig xbox font-sm" id="f1"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10264680_fpls-14-1171957-g001.jpg" target="_blank"><img alt="Figure 1" class="graphic zoom-in" height="469" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/446d/10264680/a3453f1ddcc1/fpls-14-1171957-g001.jpg" width="757"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/f1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Characterization of enriched plasma membrane fractions shows the high quality of the sample preparation. <strong>(A)</strong> Flowchart of sample preparation for mass spectrometry analysis. <strong>(B)</strong> Western blot characterization of enriched plasma membrane fractions. Protein amount of 12.5 μg from total protein (total), soluble protein (sup.), crude membrane fraction (crude M.) and enriched plasma membrane fraction (e.PM) was separated in SDS-PAGE and transferred to PVDF membrane. The same membrane was blotted with anti-GFP, anti-Sec12 and anti-SYP41 antibodies. <strong>(C)</strong> The Venn diagram comparing the proteins that were identified from different proteomics experiments.</p></figcaption></figure><p>Enrichment was evaluated by detecting the presence of proteins known to localize to the plasma membrane (PIP2A-GFP) (<a aria-describedby="B13" class="usa-link" href="#B13">Cutler et al., 2000</a>), endoplasmic reticulum (ER) (Sec12) (<a aria-describedby="B2" class="usa-link" href="#B2">Bar-Peled and Raikhel, 1997</a>), and Golgi/trans-Golgi network (TGN) (SYP41) (<a aria-describedby="B4" class="usa-link" href="#B4">Bassham et al., 2000</a>) in different fractions using Western blots with anti-GFP, anti-Sec12, and anti-SYP41 antibodies, respectively (<a class="usa-link" href="#f1">
<strong>Figure 1B</strong>
</a>). PIP2A-GFP was enriched in the plasma membrane fraction when compared to the crude membrane fraction (<a class="usa-link" href="#f1">
<strong>Figure 1B</strong>
</a>, anti-GFP). The ER membrane protein, Sec12, was detected in both crude membrane and plasma membrane fractions; however, the abundance of Sec12 in the plasma membrane fraction was markedly lower than the crude membrane fraction (<a class="usa-link" href="#f1">
<strong>Figure 1B</strong>
</a>, anti-Sec12). The Golgi/TGN protein, SYP41, was detected in the crude membrane fraction but not in the enriched plasma membrane fraction (<a class="usa-link" href="#f1">
<strong>Figure 1B</strong>
</a>, anti-SYP41). Immunoblots demonstrated that although proteins from other organelles were present in the enriched plasma membrane fraction, their abundance was substantially reduced in comparison with plasma membrane proteins. We prepared four independent biological replicates of enriched plasma membrane fractions for the mass spectrometry proteomics analysis.</p></section><section id="s4_2"><h3 class="pmc_sec_title">Mass spectrometry analyses of enriched plasma membrane proteins</h3>
<p>From each enriched plasma membrane fraction, 25 μg of protein was digested with Lys-C and trypsin. Resulting peptides from ES2-treated plant samples were dimethyl labeled with <sup>12</sup>CH<sub>2</sub>O (light, L) and peptides from DMSO-treated samples were dimethyl labeled with <sup>13</sup>CD<sub>2</sub>O (heavy, H). Dimethyl-labeled peptides from ES2-treatment and control samples were combined and fractionated before LC-MS/MS analysis. The peptides that were detected by LC-MS/MS were assigned to proteins and the abundance of each protein was calculated through a MaxQuant database search. The ratio of each protein’s abundance in differentially labeled samples was calculated using the same process. The data were processed further with the Perseus platform by filtering out rows with invalid values (potential contaminants, the reverse peptides, and peptides only identified by site) from the MaxQuant database search. We further filtered the data by removing proteins with less than three detected values in the four replicates. To compare the data from different replicates, the ratio of protein abundance between differentially labeled proteins was transformed to log<sub>2</sub> phase and further normalized by median subtraction and then analyzed by histogram (<a class="usa-link" href="#SF1">
<strong>Supplemental Figure S1</strong>
</a>). The normal distribution of log<sub>2</sub> phase of ratios centered by zero between differentially labeled proteins indicates that the majority of proteins that we detected were not altered by ES2 treatment.</p>
<p>We detected 3675 proteins from four independent replicate experiments (<a class="usa-link" href="#SM1">
<strong>Supplementary Table S1</strong>
</a>). To further evaluate the plasma membrane preparation, we compared our protein list with those published from different Arabidopsis tissues using another plasma membrane preparation method (<a class="usa-link" href="#f1">
<strong>Figure 1C</strong>
</a>). First, we compared our plasma membrane protein list with one generated from two-week-old liquid cultured <em>Arabidopsis</em> seedlings using a two-phase partition of microsomes and Trypsin-catalyzed <sup>18</sup>O labeling (<a aria-describedby="B52" class="usa-link" href="#B52">Nelson et al., 2006</a>). We found 295 overlapping proteins, accounting for 95% of the proteins detected in Nelson et al.’s experiments (309 proteins in total) (<a class="usa-link" href="#f1">
<strong>Figure 1C</strong>
</a>). Elmore et al. used an aqueous two-phase partitioning approach for leaves of four-week-old Arabidopsis plants grown in soil condition and consistently identified more than 1300 proteins in replicate experiments (<a aria-describedby="B24" class="usa-link" href="#B24">Elmore et al., 2012</a>). We found 960 overlapping proteins, accounting for more than 70% of the proteins detected in Elmore et al.’s experiments (<a class="usa-link" href="#f1">
<strong>Figure 1C</strong>
</a>). Li et al. isolated plasma membrane fractions from Arabidopsis suspension-cultured cells using a two-phase partition method and identified over 600 proteins by proteomic analysis (<a aria-describedby="B45" class="usa-link" href="#B45">Li et al., 2012</a>). About 60% of the proteins that were detected from cultured cells were consistently identified in our experiments (<a class="usa-link" href="#f1">
<strong>Figure 1C</strong>
</a>). Substantial overlap between our plasma membrane protein list and others that use a different method or tissue indicates that our proteomic data can be used to compare the effects of ES2 on the abundance of plasma membrane proteins.</p>
<p>To examine whether the enriched plasma membrane fraction was free from contaminants from other large membrane-bound compartments, we checked our proteomics list for the presence of proteins residing in the nuclear envelope and vacuole. The nuclear envelope protein SAD1/UNC-84 DOMAIN PROTEIN 1 (SUN1, AT5G04990; <a aria-describedby="B57" class="usa-link" href="#B57">Oda and Fukuda, 2011</a>) was detected, but its abundance was unchanged following ES2 treatment, with a mean log2 fold-change of 0.00. Another nuclear envelope protein SAD1/UNC-84 DOMAIN PROTEIN 2 (SUN2, AT3G10730; <a aria-describedby="B57" class="usa-link" href="#B57">Oda and Fukuda, 2011</a>) was not detected. The vacuole membrane protein TONOPLAST INTRINSIC PROTEIN 1;1 (γ-TIP or TIP1;1, AT2G36830; <a aria-describedby="B52" class="usa-link" href="#B52">Nelson et al., 2006</a>) was not detected. These results indicate that the enriched plasma membrane fraction was relatively free from contamination of nuclear envelope or vacuolar membranes.</p>
<p>To investigate whether ES2 treatment altered the membrane identity of the plasma membrane fraction, we checked our proteomics list for the presence of several markers of membrane-bound organelles. The autophagosome markers ATG8e (AT2G45170) and ATG8f (AT4G16520) were not detected. The early endosome marker ARA6 (AT3G54840; <a aria-describedby="B22" class="usa-link" href="#B22">Ebine et al., 2011</a>), ER marker SEC12 (AT2G04170; <a aria-describedby="B2" class="usa-link" href="#B2">Bar-Peled and Raikhel, 1997</a>), and Golgi marker SYP32 (AT3G24350) and ALPHA-MANNOSIDASE 1 (ManI, AT1G51590; <a aria-describedby="B38" class="usa-link" href="#B38">Kajiura et al., 2010</a>) were all present but unchanged following ES2 treatment, with a mean log2 fold-change of 0.06 for ARA6, 0.05 for SEC12, 0.00 for SYP32, and -0.01 for ManI (<a class="usa-link" href="#SM1">
<strong>Supplementary Table S1</strong>
</a>). These results indicate that the membrane identify of the plasma membrane was not altered following ES2 treatment.</p>
<p>To identify proteins with altered abundance after ES2 treatment, we performed a moderated t-test (<a aria-describedby="B66" class="usa-link" href="#B66">Smyth, 2004</a>) for each protein using the limma package in R. The moderated t-test uses a hierarchical empirical Bayesian model to shrink the sample standard deviation estimates for each protein towards a pooled value. This stabilizes the t-statistics and increases the power of the test to detect true differences in protein abundance. Since multiple significance tests were performed, we used the Benjamini-Hochberg procedure to bound the expected false-discovery rate at 10%. After the correction, 330 proteins had adjusted p-values less than 0.10. Thus, we can estimate that roughly 300 of the 330 proteins have log<sub>2</sub> ratios significantly different from 0. Among these 330 proteins, 145 had reduced abundance (<a class="usa-link" href="#SM2">
<strong>Supplementary Tables S2</strong>
</a>, <a class="usa-link" href="#SM3">
<strong>S3</strong>
</a>) and 185 had increased abundance (<a class="usa-link" href="#SM4">
<strong>Supplementary Table S4</strong>
</a>) in the plasma membrane fraction of ES2-treated samples. To highlight the proteins with the most dramatic changes, the top twenty proteins with decreased and increased abundance were listed in <a class="usa-link" href="#T1">
<strong>Tables 1</strong>
</a>, <a class="usa-link" href="#T2">
<strong>2</strong>
</a>, respectively. Because ES2 binds to the EXO70A1 subunit of the exocyst complex and inhibits its function (<a aria-describedby="B88" class="usa-link" href="#B88">Zhang et al., 2016</a>), and because EXO70A1 serves as a landmark for exocyst complex recruitment to the plasma membrane (<a aria-describedby="B69" class="usa-link" href="#B69">Synek et al., 2021</a>), we expect that proteins with reduced abundance at the plasma membrane are candidate cargos of exocyst-mediated trafficking.</p>
<section class="tw xbox font-sm" id="T1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Top 20 proteins with reduced abundance in enriched plasma membrane fractions after a 2-h ES2 treatment and their Gene Ontology terms in biological processes category.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="middle">Protein ID</th>
<th align="left" colspan="1" rowspan="1" valign="middle">Protein Name</th>
<th align="left" colspan="1" rowspan="1" valign="middle">Protein Symbol</th>
<th align="left" colspan="1" rowspan="1" valign="middle">Mean L/H</th>
<th align="left" colspan="1" rowspan="1" valign="top">Representative GO Terms (Biological <br/>Processes)</th>
<th align="left" colspan="1" rowspan="1" valign="middle">No. of TM Helices</th>
<th align="left" colspan="1" rowspan="1" valign="middle">Presence of signal peptide</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G66230</td>
<td align="left" colspan="1" rowspan="1" valign="middle">MYB DOMAIN PROTEIN 20</td>
<td align="left" colspan="1" rowspan="1" valign="middle">ATMYB20</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.02</td>
<td align="left" colspan="1" rowspan="1" valign="middle">regulation of secondary cell wall biogenesis (GO:2000652)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G16630</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Kinesin-like protein KIN-13A</td>
<td align="left" colspan="1" rowspan="1" valign="middle">KIN13A</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.09</td>
<td align="left" colspan="1" rowspan="1" valign="middle">microtubule depolymerization<br/>(GO:0007019)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G61480</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Quinoprotein amine dehydrogenase, beta chain-like RIC1-like guanyl-nucleotide exchange factor</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.10</td>
<td align="left" colspan="1" rowspan="1" valign="middle">intracellular protein transport<br/>(GO:0006886)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G22240</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CYSTEINE-RICH TRANSMEMBRANE MODULE 9</td>
<td align="left" colspan="1" rowspan="1" valign="middle">ATHCYSTM9</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.26</td>
<td align="left" colspan="1" rowspan="1" valign="middle">indole-containing compound biosynthetic process<br/>(GO:0042435)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G08610</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NADH dehydrogenase ubiquinone 1 alpha subcomplex subunit</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.30</td>
<td align="left" colspan="1" rowspan="1" valign="middle">catabolic process<br/>(GO:0009056)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT4G36670</td>
<td align="left" colspan="1" rowspan="1" valign="middle">POLYOL/MONOSACCHARIDE TRANSPORTER 6</td>
<td align="left" colspan="1" rowspan="1" valign="middle">ATPMT6</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.31</td>
<td align="left" colspan="1" rowspan="1" valign="middle">response to temperature stimulus<br/>(GO:0009266)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">12</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G45650</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Protein NRT1/PTR FAMILY 2.7</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NPF2.7</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.35</td>
<td align="left" colspan="1" rowspan="1" valign="middle">GO:0015698</td>
<td align="left" colspan="1" rowspan="1" valign="middle">12</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G25820</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Exostosin family protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.35</td>
<td align="left" colspan="1" rowspan="1" valign="middle">root morphogenesis<br/>(GO:0010015)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G65730</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Probable metal-nicotianamine transporter YSL7</td>
<td align="left" colspan="1" rowspan="1" valign="middle">YSL7</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.37</td>
<td align="left" colspan="1" rowspan="1" valign="middle">catabolic process<br/>(GO:0009056)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">17</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G12950</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Protein DETOXIFICATION 31</td>
<td align="left" colspan="1" rowspan="1" valign="middle">DTX31</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.38</td>
<td align="left" colspan="1" rowspan="1" valign="middle">root hair elongation<br/>(GO:0048767)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">12</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT2G25980</td>
<td align="left" colspan="1" rowspan="1" valign="middle">JACALIN-RELATED LECTIN 20</td>
<td align="left" colspan="1" rowspan="1" valign="middle">JAL20</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.38</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G35940</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Mannose-binding lectin superfamily protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.38</td>
<td align="left" colspan="1" rowspan="1" valign="middle">organic acid metabolic process<br/>(GO:0006082)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G40780</td>
<td align="left" colspan="1" rowspan="1" valign="middle">LYSINE HISTIDINE TRANSPORTER 1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">LHT1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.39</td>
<td align="left" colspan="1" rowspan="1" valign="middle">amino acid transmembrane transport<br/>(GO:0003333)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">11</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G17260</td>
<td align="left" colspan="1" rowspan="1" valign="middle">AUTOINHIBITED H(+)-ATPASE 10</td>
<td align="left" colspan="1" rowspan="1" valign="middle">AHA10</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.40</td>
<td align="left" colspan="1" rowspan="1" valign="middle">proton transmembrane transport<br/>(GO:1902600)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">10</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT2G01520</td>
<td align="left" colspan="1" rowspan="1" valign="middle">MLP-LIKE PROTEIN 328</td>
<td align="left" colspan="1" rowspan="1" valign="middle">MLP328</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.40</td>
<td align="left" colspan="1" rowspan="1" valign="middle">defense response<br/>(GO:0006952)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT2G39510</td>
<td align="left" colspan="1" rowspan="1" valign="middle">USUALLY MULTIPLE ACIDS MOVE IN AND OUT TRANSPORTERS 14</td>
<td align="left" colspan="1" rowspan="1" valign="middle">UMAMIT14</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.40</td>
<td align="left" colspan="1" rowspan="1" valign="middle">L-glutamate import across plasma membrane<br/>(GO:0098712)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">10</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G26260</td>
<td align="left" colspan="1" rowspan="1" valign="middle">TRAF-like family protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.40</td>
<td align="left" colspan="1" rowspan="1" valign="middle">plant epidermis development<br/>(GO:0090558)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">YES</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G20380</td>
<td align="left" colspan="1" rowspan="1" valign="middle">TRAF-like family protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.41</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">YES</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G14160</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CASP-LIKE PROTEIN 1A1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CASPL1A1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.42</td>
<td align="left" colspan="1" rowspan="1" valign="middle">cell wall modification<br/>(GO:0042545)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">4</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G50630</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Extracellular ligand-gated ion channel protein (DUF3537)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0.42</td>
<td align="left" colspan="1" rowspan="1" valign="middle">root development<br/>(GO:0048364)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">7</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn3"><p>The L/H value is the ratio of the abundance in ES2-treated samples divided by the abundance in mock-treated samples. The number of TM helices and presence of signal peptide were predicted by the DeepTMHMM package.</p></div></div></section><section class="tw xbox font-sm" id="T2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Top 20 proteins with increased abundance in enriched plasma membrane fractions after a 2-h ES2 treatment and their Gene Ontology terms in biological processes category.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1" valign="middle">Protein ID</th>
<th align="left" colspan="1" rowspan="1" valign="middle">Protein Name</th>
<th align="center" colspan="1" rowspan="1" valign="middle">Protein Symbol</th>
<th align="left" colspan="1" rowspan="1" valign="middle">Mean L/H</th>
<th align="left" colspan="1" rowspan="1" valign="middle">Representative GO Term (Biological Processes)</th>
<th align="left" colspan="1" rowspan="1" valign="middle">No. of TM Helices</th>
<th align="left" colspan="1" rowspan="1" valign="middle">Presence of signal peptide</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT4G14320</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Zinc-binding ribosomal protein family protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">RPL36AB</td>
<td align="left" colspan="1" rowspan="1" valign="middle">13.84</td>
<td align="left" colspan="1" rowspan="1" valign="middle">translation<br/>(GO:0006412)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G15520</td>
<td align="left" colspan="1" rowspan="1" valign="middle">ATP-BINDING CASSETTE G40</td>
<td align="left" colspan="1" rowspan="1" valign="middle">ABCG40/<br/>PDR12</td>
<td align="left" colspan="1" rowspan="1" valign="middle">10.93</td>
<td align="left" colspan="1" rowspan="1" valign="middle">abscisic acid transport<br/>(GO:0080168)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">12</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G80660</td>
<td align="left" colspan="1" rowspan="1" valign="middle">H(+)-ATPASE 9</td>
<td align="left" colspan="1" rowspan="1" valign="middle">AHA9</td>
<td align="left" colspan="1" rowspan="1" valign="middle">10.25</td>
<td align="left" colspan="1" rowspan="1" valign="middle">proton transmembrane transport<br/>(GO:1902600)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">10</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G56860</td>
<td align="left" colspan="1" rowspan="1" valign="middle">UBP1-ASSOCIATED PROTEIN 2A</td>
<td align="left" colspan="1" rowspan="1" valign="middle">UBA2A</td>
<td align="left" colspan="1" rowspan="1" valign="middle">8.33</td>
<td align="left" colspan="1" rowspan="1" valign="middle">abscisic acid-activated signaling pathway<br/>(GO:0009738)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G33680</td>
<td align="left" colspan="1" rowspan="1" valign="middle">KH domain-containing protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">ATKH3</td>
<td align="left" colspan="1" rowspan="1" valign="middle">7.77</td>
<td align="left" colspan="1" rowspan="1" valign="middle">regulation of gene expression<br/>(GO:0010468)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G42380</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CALMODULIN LIKE 37</td>
<td align="left" colspan="1" rowspan="1" valign="middle">CML37</td>
<td align="left" colspan="1" rowspan="1" valign="middle">7.35</td>
<td align="left" colspan="1" rowspan="1" valign="middle">response to water deprivation<br/>(GO:0009414)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G20880</td>
<td align="left" colspan="1" rowspan="1" valign="middle">RNA-binding (RRM/RBD/RNP motifs) family protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">7.10</td>
<td align="left" colspan="1" rowspan="1" valign="middle">biological_process_unknown<br/>(GO:0008150)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G78260</td>
<td align="left" colspan="1" rowspan="1" valign="middle">RNA-binding (RRM/RBD/RNP motifs) family protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">7.07</td>
<td align="left" colspan="1" rowspan="1" valign="middle">plant epidermis development<br/>(GO:0090558)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G18150</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Methyltransferase-related protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">7.00</td>
<td align="left" colspan="1" rowspan="1" valign="middle">regulation of defense response<br/>(GO:0031347)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G13060</td>
<td align="left" colspan="1" rowspan="1" valign="middle">EVOLUTIONARILY CONSERVED C-TERMINAL REGION 5</td>
<td align="left" colspan="1" rowspan="1" valign="middle">ECT5</td>
<td align="left" colspan="1" rowspan="1" valign="middle">6.96</td>
<td align="left" colspan="1" rowspan="1" valign="middle">mRNA destabilization<br/>(GO:0061157)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT4G03110</td>
<td align="left" colspan="1" rowspan="1" valign="middle">RNA-BINDING PROTEIN-DEFENSE RELATED 1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">RBP-DR1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">6.78</td>
<td align="left" colspan="1" rowspan="1" valign="middle">plant-type hypersensitive response<br/>(GO:0009626)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G07660</td>
<td align="left" colspan="1" rowspan="1" valign="middle">flocculation protein (DUF1296)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">6.37</td>
<td align="left" colspan="1" rowspan="1" valign="middle">biological_process_unknown<br/>(GO:0008150)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT5G56670</td>
<td align="left" colspan="1" rowspan="1" valign="middle">Ribosomal protein S30 family protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">RPS30C</td>
<td align="left" colspan="1" rowspan="1" valign="middle">5.57</td>
<td align="left" colspan="1" rowspan="1" valign="middle">translation<br/>(GO:0006412)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT4G36960</td>
<td align="left" colspan="1" rowspan="1" valign="middle">RNA-binding (RRM/RBD/RNP motifs) family protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">5.46</td>
<td align="left" colspan="1" rowspan="1" valign="middle">organic cyclic compound catabolic process<br/>(GO:1901361)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G13222</td>
<td align="left" colspan="1" rowspan="1" valign="middle">GBF-INTERACTING PROTEIN 1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">GIP1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">5.31</td>
<td align="left" colspan="1" rowspan="1" valign="middle">positive regulation of DNA binding<br/>(GO:0043388)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT3G13460</td>
<td align="left" colspan="1" rowspan="1" valign="middle">EVOLUTIONARILY CONSERVED C-TERMINAL REGION 2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">ECT2</td>
<td align="left" colspan="1" rowspan="1" valign="middle">5.27</td>
<td align="left" colspan="1" rowspan="1" valign="middle">mRNA destabilization<br/>(GO:0061157)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT1G29350</td>
<td align="left" colspan="1" rowspan="1" valign="middle">RNA polymerase II degradation factor-like protein (DUF1296)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">5.11</td>
<td align="left" colspan="1" rowspan="1" valign="middle">regulation of nitrogen compound metabolic process<br/>(GO:0051171)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT2G38540</td>
<td align="left" colspan="1" rowspan="1" valign="middle">LIPID TRANSFER PROTEIN 1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">ATLTP1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">5.09</td>
<td align="left" colspan="1" rowspan="1" valign="middle">lipid transport<br/>(GO:0006869)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">YES</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT4G22670</td>
<td align="left" colspan="1" rowspan="1" valign="middle">HSP70-INTERACTING PROTEIN 1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">HIP1</td>
<td align="left" colspan="1" rowspan="1" valign="middle">4.98</td>
<td align="left" colspan="1" rowspan="1" valign="middle">chaperone cofactor-dependent protein refolding<br/>(GO:0051085)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="middle">AT2G46780</td>
<td align="left" colspan="1" rowspan="1" valign="middle">RNA-binding (RRM/RBD/RNP motifs) family protein</td>
<td align="left" colspan="1" rowspan="1" valign="middle">
</td>
<td align="left" colspan="1" rowspan="1" valign="middle">4.93</td>
<td align="left" colspan="1" rowspan="1" valign="middle">plant epidermis development<br/>(GO:0090558)</td>
<td align="left" colspan="1" rowspan="1" valign="middle">0</td>
<td align="left" colspan="1" rowspan="1" valign="middle">NO</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="fn4"><p>The L/H value is the ratio of the abundance in ES2-treated samples divided by the abundance in mock-treated samples. The number of TM helices and presence of signal peptide were predicted by the DeepTMHMM package.</p></div></div></section></section><section id="s4_3"><h3 class="pmc_sec_title">Confocal imaging of GFP-tagged plasma membrane proteins following ES2 treatment</h3>
<p>To confirm the identification of proteins with reduced abundance after ES2 treatment, we performed live imaging of GFP-tagged plasma membrane proteins in Arabidopsis root epidermal cells using laser scanning confocal microscopy (<a class="usa-link" href="#f2">
<strong>Figure 2</strong>
</a>). We selected three protein candidates: the auxin efflux carrier protein PIN-FORMED 2 (PIN2), the leucine-rich-repeat receptor kinase of brassinosteroid signaling BRASSINOSTEROID INSENSITIVE 1 (BRI1), and the receptor kinase FERONIA (FER), each of which had reduced plasma membrane abundance after ES2 treatment in our proteomics analysis. PIN2 had a mean log2 fold-change of -0.27, BRI1 had a mean log2 fold-change of -0.31, and FER had a mean log2 fold-change of -0.38. (<a class="usa-link" href="#SM1">
<strong>Supplementary Table S1</strong>
</a>). We quantified GFP-tagged protein fluorescence intensity on the plasma membrane and normalized by subtracting the mean background fluorescence intensity from a cytoplasmic compartment-free region (<a class="usa-link" href="#f2">
<strong>Figure 2C</strong>
</a>). The abundance of all three proteins at the plasma membrane region were significantly decreased after a 2-h ES2 treatment (<a class="usa-link" href="#f2">
<strong>Figure 2D</strong>
</a>). To further evaluate the change of abundance between the plasma membrane and entire cytoplasmic region, we also quantified the plasma membrane / cytoplasm intensity ratio as another indicator of plasma membrane protein abundance (<a class="usa-link" href="#f2">
<strong>Figure 2E</strong>
</a>). The plasma membrane / cytoplasm intensity ratios for all three proteins were significantly reduced after a 2-h ES2 treatment compared to mock treatment (<a class="usa-link" href="#f2">
<strong>Figure 2E</strong>
</a>). Our analysis showed for the first time that the plasma membrane abundance of FER was decreased following ES2 treatment. As a negative control, we measured the abundance of PLASMA MEMBRANE INTRINSIC PROTEIN 2A (PIP2A), which was shown to have unaltered plasma membrane abundance after ES2 treatment in the proteomics analyses, with a mean log2 fold-change of 0.05 (<a class="usa-link" href="#SM1">
<strong>Supplementary Table S1</strong>
</a>). The plasma membrane abundance and plasma membrane / cytoplasm intensity ratio of PIP2A-GFP were not altered by ES2 treatment (<a class="usa-link" href="#f2">
<strong>Figures 2D, E</strong>
</a>).</p>
<figure class="fig xbox font-sm" id="f2"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10264680_fpls-14-1171957-g002.jpg" target="_blank"><img alt="Figure 2" class="graphic zoom-in" height="872" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/446d/10264680/b135281a6dd0/fpls-14-1171957-g002.jpg" width="665"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/f2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>The abundance of several proteins at the plasma membrane is reduced following ES2 treatment. <strong>(A, B)</strong> Representative confocal microscope images of root epidermal cells from 6-d-old transgenic seedlings expressing PIN2-GFP, BRI1-GFP, FER-GFP or PIP2A-GFP following a 2-h mock treatment <strong>(A)</strong> or a 40 μM ES2 treatment <strong>(B)</strong>. <strong>(C)</strong> Schematic illustration of the region of interest (ROI) selection. The plasma membrane (PM) ROI is indicated by a magenta line; cytoplasmic background in the compartment-free subregion is indicated by a blue box (used as the background for normalization); the entire cytoplasmic region is indicated by a green box. Scale bar = 5 μm for all images. <strong>(D)</strong> Quantification of the normalized average PM fluorescence intensity from images similar to those shown in <strong>(A</strong>, <strong>B)</strong> The average PM fluorescence intensity was normalized by subtracting the mean fluorescence intensity of the cytoplasmic background. <strong>(E)</strong> Quantification of the average PM/cytoplasm fluorescence intensity ratio. The PM/cytoplasm mean fluorescence intensity ratio was calculated by first subtracting the mean fluorescence intensity of the region outside of the root as the imaging background, and then dividing the PM mean fluorescence intensity by the entire cytoplasm mean fluorescence intensity. (N = 21–75 cells from 5–7 seedlings). Statistical analysis was performed with an unpaired two-tailed Student’s <em>t</em>-test, * denotes P value &lt; 0.05, ** denotes P value &lt; 0.01, ns denotes not significant. In the box plot, whiskers indicate the maximum and minimum values of the dataset, whereas individual data points represent the average value from each seedling.</p></figcaption></figure></section><section id="s4_4"><h3 class="pmc_sec_title">Gene Ontology enrichment analysis of candidate cargos for ES2-targeted trafficking</h3>
<p>To better understand the biological functions of the candidate cargo proteins for ES2-targeted trafficking, we performed Gene Ontology (GO) enrichment analysis of the biological processes category on the 145 proteins with reduced abundance at the plasma membrane using the PANTHER classification system (<a aria-describedby="B51" class="usa-link" href="#B51">Mi et al., 2013</a>; <a aria-describedby="B50" class="usa-link" href="#B50">Mi et al., 2017</a>). We found that 49 proteins could be classified into protein families that were enriched in abundance on our list when compared to their abundance in the reference list of <em>Arabidopsis thaliana</em> genome (<a class="usa-link" href="#f3">
<strong>Figure 3</strong>
</a>). The fold enrichment for different biological processes ranged from 3.7 to 200. The 49 candidate exocytosis cargo proteins belong to gene families that function in plant cell growth, cell wall biosynthesis, hormone signaling, response to stresses and membrane transport (<a class="usa-link" href="#f3">
<strong>Figure 3</strong>
</a>; <a class="usa-link" href="#SM2">
<strong>Supplementary Table S2</strong>
</a>). The proteins for the regulation of DNA-templated transcription (GO:0006355), nucleic acid metabolic process (GO:0090304) and gene expression (GO:0010467) were much lower than the numbers that were expected from the Arabidopsis genome (<a class="usa-link" href="#f3">
<strong>Figure 3</strong>
</a>), indicating the trafficking of these proteins was not mediated by the exocyst complex.</p>
<figure class="fig xbox font-sm" id="f3"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10264680_fpls-14-1171957-g003.jpg" target="_blank"><img alt="Figure 3" class="graphic zoom-in" height="394" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/446d/10264680/a978570548f0/fpls-14-1171957-g003.jpg" width="725"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/f3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Gene Ontology terms of biological processes category that are enriched in our protein list with significantly reduced abundance after ES2 treatment. Numbers on each row represent: fold enrichment (total number of proteins in the category/number of proteins identified in the reduced protein list/number of proteins expected in reduced protein list). Fold enrichment represents the ratio of the number of proteins identified divided by the number of proteins expected in the list.</p></figcaption></figure><p>We further examined the annotated functions of proteins with significantly reduced abundance in ES2-treated samples. We found the abundance of three primary cell wall cellulose synthases (CESA1, CESA3, and CESA6), CELLULOSE SYNTHASE-INTERACTIVE PROTEIN 1 (CSI1), and KORRIGAN (KOR) was significantly reduced in ES2-treated samples (<a class="usa-link" href="#SM2">
<strong>Supplementary Table S2</strong>
</a>). Proteins involved in lignin metabolism were also reduced, as were multiple proteins involved in brassinosteroid and auxin signaling. The abundance of some proteins that respond to cold and phosphate deficiency was significantly reduced in ES2-treated samples. Another group of proteins that was affected by ES2 treatment are transporters responsible for the movement of phosphate, sulphate, potassium, and oligopeptides across the plasma membrane. The abundance of two annexins involved in phloem sucrose unloading (<a aria-describedby="B81" class="usa-link" href="#B81">Wang et al., 2018</a>) was also significantly reduced (<a class="usa-link" href="#SM2">
<strong>Supplementary Table S2</strong>
</a>).</p>
<p>In addition to the proteins mentioned above, there are other proteins with reduced abundance at the plasma membrane, but they were not enriched in functional groups in the GO enrichment analysis. These include multiple receptor-like protein kinases, transporters, enzymes for carbohydrate metabolism and protein glycosylation, as well as proteins with uncharacterized functions. Although these proteins were not enriched in our list, they may have diverse functions and cellular localization patterns among different members of the family. Collectively, our GO term enrichment analysis indicates that the exocyst complex mediates constitutive transport of numerous proteins that function in cell growth, cell wall biosynthesis, hormone signaling, stress response, and membrane transport.</p></section><section id="s4_5"><h3 class="pmc_sec_title">Bioinformatic analysis of potential cargo proteins for plant exocyst complex</h3>
<p>Plants use both conventional and unconventional trafficking pathways to deliver proteins to the plasma membrane and extracellular space (<a aria-describedby="B18" class="usa-link" href="#B18">Drakakaki and Dandekar, 2013</a>; <a aria-describedby="B16" class="usa-link" href="#B16">Ding et al., 2014</a>; <a aria-describedby="B63" class="usa-link" href="#B63">Robinson et al., 2016</a>; <a aria-describedby="B1" class="usa-link" href="#B1">Aniento et al., 2022</a>). In the conserved conventional trafficking pathway, proteins with a signal peptide are inserted into the endoplasmic reticulum during translation and are then transported to Golgi through vesicles and further secreted through post-Golgi vesicles (<a aria-describedby="B56" class="usa-link" href="#B56">Novick et al., 1981</a>; <a aria-describedby="B64" class="usa-link" href="#B64">Rothman, 2002</a>; <a aria-describedby="B65" class="usa-link" href="#B65">Schekman, 2010</a>). However, increasing evidence from proteomic analysis shows that many proteins that are secreted out of the cell or targeted to the plasma membrane do not have signal peptides, indicating the existence of unconventional trafficking pathways for these proteins (<a aria-describedby="B59" class="usa-link" href="#B59">Oh et al., 2005</a>; <a aria-describedby="B67" class="usa-link" href="#B67">Soares et al., 2007</a>; <a aria-describedby="B82" class="usa-link" href="#B82">Wen et al., 2007</a>; <a aria-describedby="B33" class="usa-link" href="#B33">Han et al., 2010</a>; <a aria-describedby="B7" class="usa-link" href="#B7">Chen et al., 2016</a>). Plant exocyst complex has been shown to regulate cargo proteins through both conventional and unconventional pathways (<a aria-describedby="B78" class="usa-link" href="#B78">Wang et al., 2010</a>; <a aria-describedby="B20" class="usa-link" href="#B20">Drdová et al., 2013</a>; <a aria-describedby="B48" class="usa-link" href="#B48">Lin et al., 2015</a>; <a aria-describedby="B49" class="usa-link" href="#B49">Mayers et al., 2017</a>).</p>
<p>To test whether the candidate cargo proteins contain signal peptides that allow them to traffic through the conventional pathway, we analyzed the presence of signal peptides in proteins that had reduced abundance in ES2-treated samples using the DeepTMHMM package, a deep-learning method for signal peptide and transmembrane domain prediction (<a aria-describedby="B32" class="usa-link" href="#B32">Hallgren et al., 2022</a>). DeepTMHMM predicted that 36 of the 145 proteins with reduced abundance contained signal peptides (<a class="usa-link" href="#SM5">
<strong>Supplementary Table S5</strong>
</a>). The existence of a signal peptide in a small percentage of proteins indicates that consistent with previous findings, plant exocyst complex mediates the transport of proteins in both conventional and unconventional trafficking pathways. To understand how these candidate cargo proteins are associated to the plasma membrane, we analyzed the presence of potential transmembrane domains in these 145 proteins using the DeepTMHMM package. We found 64 of the 145 proteins were predicted to have at least one transmembrane domain (<a class="usa-link" href="#SM5">
<strong>Supplementary Table S5</strong>
</a>).</p></section><section id="s4_6"><h3 class="pmc_sec_title">The behavior of the exocyst complex and its regulators is different than candidate cargo proteins</h3>
<p>All eight subunits of the exocyst complex were detected in the enriched plasma membrane fractions (<a class="usa-link" href="#SM1">
<strong>Supplementary Table S1</strong>
</a>). However, we did not find a statistically significant reduction in their abundance in ES2-treated samples (<a class="usa-link" href="#SM1">
<strong>Supplementary Table S1</strong>
</a>). Among the eight exocyst subunits, only SEC3 showed a 13% reduction in the mean value after ES2 treatment but was not statistically different (<a class="usa-link" href="#SM1">
<strong>Supplementary Table S1</strong>
</a>). STOMATAL CYTOKINESIS-DEFECTIVE 1 (SCD1), a component of the candidate activator of plant RabE GTPase that regulates the exocyst complex (<a aria-describedby="B49" class="usa-link" href="#B49">Mayers et al., 2017</a>), was detected in plasma membrane fractions but its abundance was not altered by ES2 treatment (<a class="usa-link" href="#SM1">
<strong>Supplementary Table S1</strong>
</a>). We identified 12 Syntaxins of Plants (SYP), SYP111, SYP121, SYP122, SYP132, SYP21, SYP22, SYP32, SYP43, SYP51, SYP52, SYP61, and SYP71, but none of these were statistically significantly reduced (<a class="usa-link" href="#SM1">
<strong>Supplementary Table S1</strong>
</a>). In summary, our proteomics results indicate that the abundance of candidate tSNARE proteins, together with the exocyst complex and the SCD complex, were not significantly affected by a 2-h, 40 μM ES2 treatment.</p></section><section id="s4_7"><h3 class="pmc_sec_title">Confocal imaging of GFP-EXO70A1 following ES2 treatment</h3>
<p>To investigate whether ES2 affects the spatial distribution or abundance at the plasma membrane of its target, EXO70A1, we performed live imaging of root epidermal cells from GFP-EXO70A1 seedlings using laser scanning confocal microscopy (<a class="usa-link" href="#f4">
<strong>Figure 4</strong>
</a>). Consistent with previous findings (<a aria-describedby="B88" class="usa-link" href="#B88">Zhang et al., 2016</a>; <a aria-describedby="B43" class="usa-link" href="#B43">Larson et al., 2020</a>; <a aria-describedby="B69" class="usa-link" href="#B69">Synek et al., 2021</a>), EXO70A1 in mock-treated epidermal cells showed a polarized distribution with highest abundance at the plasma membrane on the outer periclinal face, but was present at all six faces of the cell (<a class="usa-link" href="#f4">
<strong>Figure 4A</strong>
</a>). A 2-h, 40 µM ES2 treatment significantly reduced the abundance of EXO70A1 at the plasma membrane on the outer periclinal face (<a class="usa-link" href="#f4">
<strong>Figures 4B, D</strong>
</a>), and disrupted the polarized distribution of EXO70A1 at the outer periclinal face when fluorescence intensity was expressed as a ratio with the cytosolic region (<a class="usa-link" href="#f4">
<strong>Figure 4E</strong>
</a>) or as a ratio with the other faces of the same cell (<a class="usa-link" href="#f4">
<strong>Figure 4F</strong>
</a>). However, when EXO70A1 at the other three faces of the plasma membrane visible in medial optical sections, including the inner periclinal face and the two anticlinal faces (green line in <a class="usa-link" href="#f4">
<strong>Figure 4C</strong>
</a>), was examined, the apparent abundance expressed as a ratio with cytosol was not affected by ES2 treatment (<a class="usa-link" href="#f4">
<strong>Figure 4G</strong>
</a>). When all four faces of the plasma membrane were combined and averaged, there was a modest decrease in the abundance of EXO70A1, but this was not statistically significant (<a class="usa-link" href="#f4">
<strong>Figure 4H</strong>
</a>). In total, a 2-h ES2 treatment had greatest effect on the polarized distribution of EXO70A1 and its association with the plasma membrane at the outer periclinal face of root epidermal cells, but a rather modest reduction in overall abundance at the plasma membrane on all faces of the cell.</p>
<figure class="fig xbox font-sm" id="f4"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10264680_fpls-14-1171957-g004.jpg" target="_blank"><img alt="Figure 4" class="graphic zoom-in" height="833" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/446d/10264680/665dfef378f1/fpls-14-1171957-g004.jpg" width="659"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/f4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>The abundance of EXO70A1 at the plasma membrane on the outer periclinal face is decreased following ES2 treatment. <strong>(A, B)</strong> Representative confocal microscopy images of root epidermal cells from 6-d-old transgenic seedlings expressing GFP-EXO70A1 following a 2-h mock treatment <strong>(A)</strong> or a 40 μM ES2 treatment <strong>(B)</strong>. Scale bar = 5 μm for all images. <strong>(C)</strong> Schematic illustration of the region of interest (ROI) selection. The plasma membrane on the outer periclinal face was marked with a magenta line, whereas the plasma membrane of the other three faces was marked with a green line. The cytoplasmic background in the vacuole or nucleus region (used for background subtraction) was indicated with a blue box. <strong>(D)</strong> Quantification of the average GFP-EXO70A1 fluorescence intensity at the plasma membrane on the outer periclinal face (as marked with a magenta line in <strong>C</strong>) from images similar to those shown in <strong>(A</strong>, <strong>B, E)</strong> Quantification of the ratio of fluorescence intensity of GFP-EXO70A1 at the plasma membrane on the outer periclinal face compared to the entire cytoplasmic region after normalization. <strong>(F)</strong> Quantification of the ratio of GFP-EXO70A1 fluorescence intensity at the plasma membrane on the outer periclinal face compared to the other three faces (marked with a green line in <strong>C</strong>). <strong>(G)</strong> Quantification of the ratio of fluorescence intensity of GFP-EXO70A1 at the plasma membrane on the other three faces compared to the entire cytoplasmic region after normalization. <strong>(H)</strong> Quantification of the average fluorescence intensity of GFP-EXO70A1 at the plasma membrane on all visible faces of the plasma membrane in medial views of the epidermal cell. Fluorescence intensities at the plasma membrane and entire cytoplasmic region were normalized by subtracting the cytoplasmic background as indicated with a blue box in <strong>(C)</strong> (N = 17–23 cells from 5 seedlings). Statistical analysis was performed with an unpaired two-tailed Student’s <em>t</em>-test, * denotes P value &lt; 0.05, ns denotes not significant. In the box plot, whiskers indicate the maximum and minimum values of the dataset, whereas individual data points represent the average value from each seedling.</p></figcaption></figure></section><section id="s4_8"><h3 class="pmc_sec_title">Gene Ontology enrichment analysis of proteins with increased abundance after ES2 treatment</h3>
<p>Our statistical analysis identified 185 proteins with increased abundance in enriched plasma membrane fractions of ES2-treated samples (<a class="usa-link" href="#SM4">
<strong>Supplementary Table S4</strong>
</a>). To better understand the function of these proteins, we performed Gene Ontology enrichment analysis of the biological function category. We found that this list of proteins is enriched in biological functions of COPII-coated vesicle cargo loading, mRNA catabolic process, negative regulation of translation, proton membrane transport, aerobic respiration, leaf senescence, ribonucleoprotein complex biogenesis, and amide biosynthesis process (<a class="usa-link" href="#SF1">
<strong>Supplementary Figure S3</strong>
</a>). The functional relationship between exocyst complex and these proteins has not been well characterized. We speculate that some of these proteins might be involved in cellular responses to the inhibition of exocytosis.</p></section></section><section id="s5"><h2 class="pmc_sec_title">Discussion</h2>
<section id="s5_1"><h3 class="pmc_sec_title">Exocyst complex mediates constitutive transport of proteins with diverse functions during plant growth and development</h3>
<p>Using quantitative and comparative proteomics analysis, we identified 145 proteins with significantly reduced abundance at the plasma membrane of Arabidopsis roots following ES2 treatment. Some of these proteins possess transmembrane domains and are likely integral membrane proteins delivered to the plasma membrane by secretion. The candidate proteins without a predicted transmembrane domain could be associated with the plasma membrane through lipid anchors, through interaction with other membrane proteins, or by association with polar head groups of the lipid bilayer. These candidates of exocyst-mediated trafficking perform functions in cell growth, cell wall biosynthesis, hormone signaling, stress responses, membrane transport, nutrient transport, and protein phosphorylation. The dynamics of many hormone-signaling proteins are well characterized; however, the role of exocyst complex in regulating their trafficking awaits further study. The functional diversity of proteins that are transported through the exocyst complex is consistent with the pleiotropic phenotypes observed in plants deficient for the exocyst complex (<a aria-describedby="B11" class="usa-link" href="#B11">Cole et al., 2005</a>; <a aria-describedby="B70" class="usa-link" href="#B70">Synek et al., 2006</a>; <a aria-describedby="B31" class="usa-link" href="#B31">Hála et al., 2008</a>; <a aria-describedby="B27" class="usa-link" href="#B27">Fendrych et al., 2010</a>; <a aria-describedby="B40" class="usa-link" href="#B40">Kulich et al., 2010</a>; <a aria-describedby="B78" class="usa-link" href="#B78">Wang et al., 2010</a>; <a aria-describedby="B61" class="usa-link" href="#B61">Pečenková et al., 2011</a>; <a aria-describedby="B20" class="usa-link" href="#B20">Drdová et al., 2013</a>; <a aria-describedby="B90" class="usa-link" href="#B90">Zhang et al., 2013</a>; <a aria-describedby="B42" class="usa-link" href="#B42">Kulich et al., 2015</a>; <a aria-describedby="B19" class="usa-link" href="#B19">Drdová et al., 2019</a>; <a aria-describedby="B58" class="usa-link" href="#B58">Ogura et al., 2019</a>; <a aria-describedby="B69" class="usa-link" href="#B69">Synek et al., 2021</a>; <a aria-describedby="B34" class="usa-link" href="#B34">Hématy et al., 2022</a>). Our results indicate that the plant exocyst complex plays essential roles for growth and adaptation by mediating constitutive transport of proteins with diverse functions to the plasma membrane.</p>
<p>Some other important plasma membrane proteins, such as Rho family small GTPases (ROPs), multidrug resistance-like ABC transporter family proteins, and aquaporin plasma membrane intrinsic proteins were not affected by ES2 treatment in this proteomics study (<a class="usa-link" href="#SM1">
<strong>Supplementary Table S1</strong>
</a>), which is consistent with the previous live-cell imaging results (<a aria-describedby="B88" class="usa-link" href="#B88">Zhang et al., 2016</a>). Our current results indicate that these important plasma membrane proteins could have slow turnover rates or are not delivered to the plasma membrane through the ES2-targeted exocyst complex.</p></section><section id="s5_2"><h3 class="pmc_sec_title">Cargo protein exocytosis and exocyst complex membrane association</h3>
<p>Time-lapse imaging revealed that the plant exocyst complex colocalizes with its cargo protein (Cellulose Synthase Complex or CSC) during the tethering process and then separates from the cargo after successful fusion (<a aria-describedby="B94" class="usa-link" href="#B94">Zhu et al., 2018</a>; <a aria-describedby="B93" class="usa-link" href="#B93">Zhang and Staiger, 2021</a>; <a aria-describedby="B89" class="usa-link" href="#B89">Zhang et al., 2021</a>). In our proteomics study, we detected a significant reduction in the abundance of cargo proteins, such as cellulose synthases, but did not detect a statistically significant reduction in the abundance of the exocyst complex subunits. The abundance of SCD complex components and candidate tSNAREs on the plasma membrane was also not affected by ES2 treatment. Previous live-cell imaging data reveal that exocyst subunits are polarized to the outer periclinal face of root epidermal cells (<a aria-describedby="B88" class="usa-link" href="#B88">Zhang et al., 2016</a>; <a aria-describedby="B43" class="usa-link" href="#B43">Larson et al., 2020</a>; <a aria-describedby="B69" class="usa-link" href="#B69">Synek et al., 2021</a>) and are disrupted by ES2 treatment (<a aria-describedby="B88" class="usa-link" href="#B88">Zhang et al., 2016</a>) or genetic disruption of EXO70A1 (<a aria-describedby="B69" class="usa-link" href="#B69">Synek et al., 2021</a>), but not by genetic disruption of some SNARE proteins that physically interact with the exocyst complex (<a aria-describedby="B43" class="usa-link" href="#B43">Larson et al., 2020</a>). The plasma membrane enrichment reflects the overall abundance of each plasma membrane protein, but not their polarized distribution on a specific face of the plasma membrane. Results from live-cell imaging and our proteomics analyses indicated that ES2 treatment affected the polarized distribution of the exocyst complex in root epidermal cells but did not markedly affect its association with the plasma membrane. The lifetime of exocyst foci on the plasma membrane of root epidermal cells is around 10 s; however, inhibition of exocytosis by a 2-h Brefeldin A (BFA) treatment did not affect membrane localization of the exocyst complex (<a aria-describedby="B26" class="usa-link" href="#B26">Fendrych et al., 2013</a>). Thus, exocyst complex association with the plasma membrane may not depend on the other components of the exocytosis machinery.</p>
<p>Previous studies demonstrate that exocyst subunits physically interact and associate with the SCD1/2 complex (<a aria-describedby="B49" class="usa-link" href="#B49">Mayers et al., 2017</a>), and EXO70 physically interacts with several SNAREs (<a aria-describedby="B43" class="usa-link" href="#B43">Larson et al., 2020</a>). Although ES2 disrupts the polarized localization of EXO70A1, our current data do not directly reflect whether ES2 disrupts the physical interaction of exocyst subunits with SCD1/2 complex or SNARE proteins, and/or affects the localization pattern of SCD1/2 and SNAREs. One future direction would be to quantify the localization of fluorescent-tagged SCD1/2 and SNAREs following ES2 treatment. Further, it would be interesting to quantify the association of exocyst subunits with other components of the exocytosis machinery through high spatial-temporal resolution live-cell imaging of double marker lines.</p></section><section id="s5_3"><h3 class="pmc_sec_title">Future directions in understanding plant plasma membrane protein dynamics</h3>
<p>Due to the importance of plant plasma membrane proteins, multiple comparative proteomic studies have been conducted to investigate the profiles of plasma membrane proteins in response to environmental stresses and cell growth status. The composition of plasma membrane proteins changes upon salt stress, cold treatment, osmotic stress, immune signaling, nutrient deficiency and different growth stages of cultured cells (<a aria-describedby="B54" class="usa-link" href="#B54">Nohzadeh Malakshah et al., 2007</a>; <a aria-describedby="B55" class="usa-link" href="#B55">Nouri and Komatsu, 2010</a>; <a aria-describedby="B24" class="usa-link" href="#B24">Elmore et al., 2012</a>; <a aria-describedby="B45" class="usa-link" href="#B45">Li et al., 2012</a>; <a aria-describedby="B6" class="usa-link" href="#B6">Cao et al., 2016</a>; <a aria-describedby="B7" class="usa-link" href="#B7">Chen et al., 2016</a>; <a aria-describedby="B46" class="usa-link" href="#B46">Li et al., 2018</a>). These proteomic studies show that the plasma membrane serves as a hub for cells to respond to environmental and growth cues. These studies further indicate that the vesicle trafficking machinery that facilitates the transport of these proteins is essential for plants to respond to biotic and abiotic stresses. Previous findings on the essential functions of plant exocyst complex in plant growth and immune responses (<a aria-describedby="B70" class="usa-link" href="#B70">Synek et al., 2006</a>; <a aria-describedby="B27" class="usa-link" href="#B27">Fendrych et al., 2010</a>; <a aria-describedby="B40" class="usa-link" href="#B40">Kulich et al., 2010</a>; <a aria-describedby="B42" class="usa-link" href="#B42">Kulich et al., 2015</a>; <a aria-describedby="B10" class="usa-link" href="#B10">Cole et al., 2018</a>; <a aria-describedby="B21" class="usa-link" href="#B21">Du et al., 2018</a>; <a aria-describedby="B30" class="usa-link" href="#B30">Guo et al., 2018</a>; <a aria-describedby="B86" class="usa-link" href="#B86">Žárský, 2022</a>) support the hypothesis that the exocyst complex choreographs the dynamic change of specific proteins at the plasma membrane in response to different environmental stimuli and growth cues.</p>
<p>In comparison with yeast and mammalian cells, we understand less about the mechanisms of plasma membrane protein delivery and turnover in plants. In this study, using very stringent filtering and cutoff, we found 145 proteins that have significantly (at 10% FDR) reduced abundance at the plasma membrane after a 2-h, 40 μM ES2 treatment, a condition that inhibits but does not completely arrest exocytosis and plant growth. We found that the abundance of some plant plasma membrane proteins was reduced by more than 50%, indicating that the half-time of turnover of these proteins is less than 2 h. We speculate that these proteins undergo fast delivery to the plasma membrane mediated by the exocyst complex and have a rapid turnover at the plasma membrane. There are other proteins that have lower abundance in ES2-treated samples but did not pass the statistical test. We believe these proteins have slower delivery and/or turnover rates at the plasma membrane. It is not well understood how plants control precise delivery of cargo proteins with different dynamics.</p></section></section><section id="s6"><h2 class="pmc_sec_title">Conclusion</h2>
<p>Using a 2-h treatment with the exocyst subunit EXO70A1 inhibitor ES2 in combination with plasma membrane enrichment, quantitative proteomic analyses, and live-cell imaging, we identified subsets of plasma membrane proteins that require the ES2-targeted exocyst complex for constitutive trafficking to the plasma membrane in Arabidopsis root cells. Our dataset provides new insights for the identification of novel cargo proteins of the exocyst complex.</p></section><section id="s7"><h2 class="pmc_sec_title">Data availability statement</h2>
<p>The datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found below: ProteomeXchange Consortium via the PRIDE database, with the identifier of PXD037455.</p></section><section id="s8"><h2 class="pmc_sec_title">Author contributions</h2>
<p>CZ and XL: conceptualization. XL, PZ, Y-JC, LH, and DW: investigation. XL and DN: formal analysis. XL, PZ, C-CH, GL, WAT, and CZ: methodology. CZ and WAT: resources. XL and CZ: data curation, visualization, and validation. CZ, XL, and CJS: writing - original draft. PZ, Y-JC, DN, LH, DW, C-CH, GL, and WAT: writing - review &amp; editing. CZ, GL, WAT, and CJS: supervision. CZ and XL: funding acquisition. CJS: agrees to serve as the author responsible for contact and ensures communication. All authors contributed to the article and approved the submitted version.</p></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We dedicate this work to the memory of CZ, who initiated and conceived the project, and unfortunately passed away on May 15, 2021. CZ was a brilliant scientist with a sharp mind, a dear friend to all, a supportive mentor, and a valued colleague. We thank Dr. Glenn R. Hicks (University of California Riverside and Swedish University of Agricultural Sciences) for critical comments and editing of the manuscript, and Dr. Sharon Kessler (Purdue University) for providing the FER-GFP marker line.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>GL gratefully acknowledges the support from National Science Foundation (DMS-1555072, DMS-1736364 and DMS-1821233). XL, LH, DW, and CZ were supported by Start-up funds from the Office of the Provost at Purdue University. XL was also partially supported by a research assistantship from the Purdue Center for Plant Biology.</p></section><section id="s10"><h2 class="pmc_sec_title">Conflict of interest</h2>
<p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></section><section id="s11"><h2 class="pmc_sec_title">Publisher’s note</h2>
<p>All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.</p></section><section id="s12"><h2 class="pmc_sec_title">Supplementary material</h2>
<p>The Supplementary Material for this article can be found online at: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.frontiersin.org/articles/10.3389/fpls.2023.1171957/full#supplementary-material" rel="noopener noreferrer" target="_blank">https://www.frontiersin.org/articles/10.3389/fpls.2023.1171957/full#supplementary-material</a>
</p>
<section class="sm xbox font-sm" id="SF1"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/DataSheet_1.pdf">Click here for additional data file.</a><sup> (337.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM1"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/Table_1.xlsx">Click here for additional data file.</a><sup> (1.7MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM2"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/Table_2.xlsx">Click here for additional data file.</a><sup> (19KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM3"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/Table_3.xlsx">Click here for additional data file.</a><sup> (78.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM4"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/Table_4.xlsx">Click here for additional data file.</a><sup> (96.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM5"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/Table_5.xlsx">Click here for additional data file.</a><sup> (15.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="SM6"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/Table_6.xlsx">Click here for additional data file.</a><sup> (16.3KB, xlsx) </sup>
</div></div></section></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="B1">
<cite>
Aniento F., Sánchez de Medina Hernández V., Dagdas Y., Rojas-Pierce M., Russinova E. (2022). Molecular mechanisms of endomembrane trafficking in plants. Plant Cell
34, 146–173. doi: 10.1093/plcell/koab235
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/plcell/koab235" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8773984/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34550393/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Molecular%20mechanisms%20of%20endomembrane%20trafficking%20in%20plants&amp;author=F.%20Aniento&amp;author=V.%20S%C3%A1nchez%20de%20Medina%20Hern%C3%A1ndez&amp;author=Y.%20Dagdas&amp;author=M.%20Rojas-Pierce&amp;author=E.%20Russinova&amp;volume=34&amp;publication_year=2022&amp;pages=146-173&amp;pmid=34550393&amp;doi=10.1093/plcell/koab235&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<cite>
Bar-Peled M., Raikhel N. V. (1997). Characterization of AtSEC12 and AtSAR1 (proteins likely involved in endoplasmic reticulum and golgi transport). Plant Physiol.
114, 315–324. doi: 10.1104/pp.114.1.315
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.114.1.315" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC158307/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9159953/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Characterization%20of%20AtSEC12%20and%20AtSAR1%20(proteins%20likely%20involved%20in%20endoplasmic%20reticulum%20and%20golgi%20transport)&amp;author=M.%20Bar-Peled&amp;author=N.%20V.%20Raikhel&amp;volume=114&amp;publication_year=1997&amp;pages=315-324&amp;pmid=9159953&amp;doi=10.1104/pp.114.1.315&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<cite>
Bashline L., Li S., Anderson C. T., Lei L., Gu Y. (2013). The endocytosis of cellulose synthase in arabidopsis is dependent on μ2, a clathrin-mediated endocytosis adaptin. Plant Physiol.
163, 150–160. doi: 10.1104/pp.113.221234
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.113.221234" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3762637/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23843604/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=The%20endocytosis%20of%20cellulose%20synthase%20in%20arabidopsis%20is%20dependent%20on%20%CE%BC2,%20a%20clathrin-mediated%20endocytosis%20adaptin&amp;author=L.%20Bashline&amp;author=S.%20Li&amp;author=C.%20T.%20Anderson&amp;author=L.%20Lei&amp;author=Y.%20Gu&amp;volume=163&amp;publication_year=2013&amp;pages=150-160&amp;pmid=23843604&amp;doi=10.1104/pp.113.221234&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<cite>
Bassham D. C., Sanderfoot A. A., Kovaleva V., Zheng H., Raikhel N. V. (2000). AtVPS45 complex formation at the trans-golgi network. Mol. Biol. Cell
11, 2251–2265. doi: 10.1091/mbc.11.7.2251
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.11.7.2251" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC14917/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10888666/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=AtVPS45%20complex%20formation%20at%20the%20trans-golgi%20network&amp;author=D.%20C.%20Bassham&amp;author=A.%20A.%20Sanderfoot&amp;author=V.%20Kovaleva&amp;author=H.%20Zheng&amp;author=N.%20V.%20Raikhel&amp;volume=11&amp;publication_year=2000&amp;pages=2251-2265&amp;pmid=10888666&amp;doi=10.1091/mbc.11.7.2251&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<cite>
Boersema P. J., Raijmakers R., Lemeer S., Mohammed S., Heck A. J. R. (2009). Multiplex peptide stable isotope dimethyl labeling for quantitative proteomics. Nat. Protoc.
4, 484–494. doi: 10.1038/nprot.2009.21
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nprot.2009.21" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19300442/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Multiplex%20peptide%20stable%20isotope%20dimethyl%20labeling%20for%20quantitative%20proteomics&amp;author=P.%20J.%20Boersema&amp;author=R.%20Raijmakers&amp;author=S.%20Lemeer&amp;author=S.%20Mohammed&amp;author=A.%20J.%20R.%20Heck&amp;volume=4&amp;publication_year=2009&amp;pages=484-494&amp;pmid=19300442&amp;doi=10.1038/nprot.2009.21&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<cite>
Cao J., Yang C., Li L., Jiang L., Wu Y., Wu C., et al. (2016). Rice plasma membrane proteomics reveals <em>Magnaporthe oryzae</em> promotes susceptibility by sequential activation of host hormone signaling pathways. Mol. Plant-Microbe Interact.
29, 902–913. doi: 10.1094/MPMI-08-16-0165-R
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1094/MPMI-08-16-0165-R" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27800704/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Plant-Microbe%20Interact.&amp;title=Rice%20plasma%20membrane%20proteomics%20reveals%20Magnaporthe%20oryzae%20promotes%20susceptibility%20by%20sequential%20activation%20of%20host%20hormone%20signaling%20pathways&amp;author=J.%20Cao&amp;author=C.%20Yang&amp;author=L.%20Li&amp;author=L.%20Jiang&amp;author=Y.%20Wu&amp;volume=29&amp;publication_year=2016&amp;pages=902-913&amp;pmid=27800704&amp;doi=10.1094/MPMI-08-16-0165-R&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<cite>
Chen S., Luo Y., Ding G., Xu F. (2016). Comparative analysis of <em>Brassica napus</em> plasma membrane proteins under phosphorus deficiency using label-free and MaxQuant-based proteomics approaches. J. Proteomics
133, 144–152. doi: 10.1016/j.jprot.2015.12.020
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jprot.2015.12.020" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26746009/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteomics&amp;title=Comparative%20analysis%20of%20Brassica%20napus%20plasma%20membrane%20proteins%20under%20phosphorus%20deficiency%20using%20label-free%20and%20MaxQuant-based%20proteomics%20approaches&amp;author=S.%20Chen&amp;author=Y.%20Luo&amp;author=G.%20Ding&amp;author=F.%20Xu&amp;volume=133&amp;publication_year=2016&amp;pages=144-152&amp;pmid=26746009&amp;doi=10.1016/j.jprot.2015.12.020&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<cite>
Chen J., Wang Y., Wang F., Yang J., Gao M., Li C., et al. (2015). The rice CK2 kinase regulates trafficking of phosphate transporters in response to phosphate levels. Plant Cell
27, 711–723. doi: 10.1105/tpc.114.135335
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.114.135335" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4558666/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25724641/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20rice%20CK2%20kinase%20regulates%20trafficking%20of%20phosphate%20transporters%20in%20response%20to%20phosphate%20levels&amp;author=J.%20Chen&amp;author=Y.%20Wang&amp;author=F.%20Wang&amp;author=J.%20Yang&amp;author=M.%20Gao&amp;volume=27&amp;publication_year=2015&amp;pages=711-723&amp;pmid=25724641&amp;doi=10.1105/tpc.114.135335&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<cite>
Chu F. F., Doyle D. (1985). Turnover of plasma membrane proteins in rat hepatoma cells and primary cultures of rat hepatocytes. J. Biol. Chem.
260, 3097–3107. doi: 10.1016/S0021-9258(18)89478-9
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0021-9258(18)89478-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/3972818/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Turnover%20of%20plasma%20membrane%20proteins%20in%20rat%20hepatoma%20cells%20and%20primary%20cultures%20of%20rat%20hepatocytes&amp;author=F.%20F.%20Chu&amp;author=D.%20Doyle&amp;volume=260&amp;publication_year=1985&amp;pages=3097-3107&amp;pmid=3972818&amp;doi=10.1016/S0021-9258(18)89478-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<cite>
Cole R. A., Peremyslov V. V., Van Why S., Moussaoui I., Ketter A., Cool R., et al. (2018). A broadly conserved NERD genetically interacts with the exocyst to affect root growth and cell expansion. J. Exp. Bot.
69, 3625–3637. doi: 10.1093/jxb/ery162
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/ery162" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6022600/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29722827/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=A%20broadly%20conserved%20NERD%20genetically%20interacts%20with%20the%20exocyst%20to%20affect%20root%20growth%20and%20cell%20expansion&amp;author=R.%20A.%20Cole&amp;author=V.%20V.%20Peremyslov&amp;author=S.%20Van%20Why&amp;author=I.%20Moussaoui&amp;author=A.%20Ketter&amp;volume=69&amp;publication_year=2018&amp;pages=3625-3637&amp;pmid=29722827&amp;doi=10.1093/jxb/ery162&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<cite>
Cole R. A., Synek L., Zarsky V., Fowler J. E. (2005). SEC8, a subunit of the putative arabidopsis exocyst complex, facilitates pollen germination and competitive pollen tube growth. Plant Physiol.
138, 2005–2018. doi: 10.1104/pp.105.062273
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.105.062273" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1183391/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16040664/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=SEC8,%20a%20subunit%20of%20the%20putative%20arabidopsis%20exocyst%20complex,%20facilitates%20pollen%20germination%20and%20competitive%20pollen%20tube%20growth&amp;author=R.%20A.%20Cole&amp;author=L.%20Synek&amp;author=V.%20Zarsky&amp;author=J.%20E.%20Fowler&amp;volume=138&amp;publication_year=2005&amp;pages=2005-2018&amp;pmid=16040664&amp;doi=10.1104/pp.105.062273&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<cite>
Collins C. A., Leslie M. E., Peck S. C., Heese A. (2017). “Simplified enrichment of plasma membrane proteins from arabidopsis thaliana seedlings using differential centrifugation and brij-58 treatment,” in Methods in molecular biology. brassinosteroids: methods and protocols. Eds. Russinova E., Caño-Delgado A. I. (New York, NY: Springer; ), 155–168.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-1-4939-6813-8_13" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28124253/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20molecular%20biology.%20brassinosteroids:%20methods%20and%20protocols&amp;author=C.%20A.%20Collins&amp;author=M.%20E.%20Leslie&amp;author=S.%20C.%20Peck&amp;author=A.%20Heese&amp;publication_year=2017&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<cite>
Cutler S. R., Ehrhardt D. W., Griffitts J. S., Somerville C. R. (2000). Random GFP∷cDNA fusions enable visualization of subcellular structures in cells of <em>Arabidopsis</em> at a high frequency. Proc. Natl. Acad. Sci.
97, 3718–3723. doi: 10.1073/pnas.97.7.3718
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.97.7.3718" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC16306/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10737809/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.&amp;title=Random%20GFP%E2%88%B7cDNA%20fusions%20enable%20visualization%20of%20subcellular%20structures%20in%20cells%20of%20Arabidopsis%20at%20a%20high%20frequency&amp;author=S.%20R.%20Cutler&amp;author=D.%20W.%20Ehrhardt&amp;author=J.%20S.%20Griffitts&amp;author=C.%20R.%20Somerville&amp;volume=97&amp;publication_year=2000&amp;pages=3718-3723&amp;pmid=10737809&amp;doi=10.1073/pnas.97.7.3718&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14">
<cite>
Denecke J., Aniento F., Frigerio L., Hawes C., Hwang I., Mathur J., et al. (2012). Secretory pathway research: the more experimental systems the better. Plant Cell
24, 1316–1326. doi: 10.1105/tpc.112.096362
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.112.096362" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3398477/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22523202/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Secretory%20pathway%20research:%20the%20more%20experimental%20systems%20the%20better&amp;author=J.%20Denecke&amp;author=F.%20Aniento&amp;author=L.%20Frigerio&amp;author=C.%20Hawes&amp;author=I.%20Hwang&amp;volume=24&amp;publication_year=2012&amp;pages=1316-1326&amp;pmid=22523202&amp;doi=10.1105/tpc.112.096362&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15">
<cite>
Dimayacyac-Esleta B. R. T., Tsai C.-F., Kitata R. B., Lin P.-Y., Choong W.-K., Lin T.-D., et al. (2015). Rapid high-pH reverse phase StageTip for sensitive small-scale membrane proteomic profiling. Anal. Chem.
87, 12016–12023. doi: 10.1021/acs.analchem.5b03639
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/acs.analchem.5b03639" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26554430/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Rapid%20high-pH%20reverse%20phase%20StageTip%20for%20sensitive%20small-scale%20membrane%20proteomic%20profiling&amp;author=B.%20R.%20T.%20Dimayacyac-Esleta&amp;author=C.-F.%20Tsai&amp;author=R.%20B.%20Kitata&amp;author=P.-Y.%20Lin&amp;author=W.-K.%20Choong&amp;volume=87&amp;publication_year=2015&amp;pages=12016-12023&amp;pmid=26554430&amp;doi=10.1021/acs.analchem.5b03639&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16">
<cite>
Ding Y., Robinson D. G., Jiang L. (2014). Unconventional protein secretion (UPS) pathways in plants. Curr. Opin. Cell Biol.
29, 107–115. doi: 10.1016/j.ceb.2014.05.008
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ceb.2014.05.008" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24949560/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Cell%20Biol.&amp;title=Unconventional%20protein%20secretion%20(UPS)%20pathways%20in%20plants&amp;author=Y.%20Ding&amp;author=D.%20G.%20Robinson&amp;author=L.%20Jiang&amp;volume=29&amp;publication_year=2014&amp;pages=107-115&amp;pmid=24949560&amp;doi=10.1016/j.ceb.2014.05.008&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17">
<cite>
Doyle D., Baumann H. (1979). Turnover of the plasma membrane of mammalian cells. Life Sci.
24, 951–966. doi: 10.1016/0024-3205(79)90313-8
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0024-3205(79)90313-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/376998/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Life%20Sci.&amp;title=Turnover%20of%20the%20plasma%20membrane%20of%20mammalian%20cells&amp;author=D.%20Doyle&amp;author=H.%20Baumann&amp;volume=24&amp;publication_year=1979&amp;pages=951-966&amp;pmid=376998&amp;doi=10.1016/0024-3205(79)90313-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18">
<cite>
Drakakaki G., Dandekar A. (2013). Protein secretion: how many secretory routes does a plant cell have? Plant Sci.
203–204, 74–78. doi: 10.1016/j.plantsci.2012.12.017
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.plantsci.2012.12.017" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23415330/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Sci.&amp;title=Protein%20secretion:%20how%20many%20secretory%20routes%20does%20a%20plant%20cell%20have&amp;author=G.%20Drakakaki&amp;author=A.%20Dandekar&amp;volume=203%E2%80%93204&amp;publication_year=2013&amp;pages=74-78&amp;pmid=23415330&amp;doi=10.1016/j.plantsci.2012.12.017&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19">
<cite>
Drdová E. J., Klejchová M., Janko K., Hála M., Soukupová H., Cvrčková F., et al. (2019). Developmental plasticity of arabidopsis hypocotyl is dependent on exocyst complex function. J. Exp. Bot.
70, 1255–1265. doi: 10.1093/jxb/erz005
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/erz005" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6382343/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30649396/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=Developmental%20plasticity%20of%20arabidopsis%20hypocotyl%20is%20dependent%20on%20exocyst%20complex%20function&amp;author=E.%20J.%20Drdov%C3%A1&amp;author=M.%20Klejchov%C3%A1&amp;author=K.%20Janko&amp;author=M.%20H%C3%A1la&amp;author=H.%20Soukupov%C3%A1&amp;volume=70&amp;publication_year=2019&amp;pages=1255-1265&amp;pmid=30649396&amp;doi=10.1093/jxb/erz005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B20">
<cite>
Drdová E. J., Synek L., Pečenková T., Hála M., Kulich I., Fowler J. E., et al. (2013). The exocyst complex contributes to PIN auxin efflux carrier recycling and polar auxin transport in arabidopsis. Plant J.
73, 709–719. doi: 10.1111/tpj.12074
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.12074" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23163883/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=The%20exocyst%20complex%20contributes%20to%20PIN%20auxin%20efflux%20carrier%20recycling%20and%20polar%20auxin%20transport%20in%20arabidopsis&amp;author=E.%20J.%20Drdov%C3%A1&amp;author=L.%20Synek&amp;author=T.%20Pe%C4%8Denkov%C3%A1&amp;author=M.%20H%C3%A1la&amp;author=I.%20Kulich&amp;volume=73&amp;publication_year=2013&amp;pages=709-719&amp;pmid=23163883&amp;doi=10.1111/tpj.12074&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21">
<cite>
Du Y., Overdijk E. J. R., Berg J. A., Govers F., Bouwmeester K. (2018). Solanaceous exocyst subunits are involved in immunity to diverse plant pathogens. J. Exp. Bot.
69, 655–666. doi: 10.1093/jxb/erx442
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/erx442" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5853398/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29329405/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=Solanaceous%20exocyst%20subunits%20are%20involved%20in%20immunity%20to%20diverse%20plant%20pathogens&amp;author=Y.%20Du&amp;author=E.%20J.%20R.%20Overdijk&amp;author=J.%20A.%20Berg&amp;author=F.%20Govers&amp;author=K.%20Bouwmeester&amp;volume=69&amp;publication_year=2018&amp;pages=655-666&amp;pmid=29329405&amp;doi=10.1093/jxb/erx442&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22">
<cite>
Ebine K., Fujimoto M., Okatani Y., Nishiyama T., Goh T., Ito E., et al. (2011). A membrane trafficking pathway regulated by the plant-specific RAB GTPase ARA6. Nat. Cell Biol.
13, 853–859. doi:  10.1038/ncb2270
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ncb2270" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21666683/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol.&amp;title=A%20membrane%20trafficking%20pathway%20regulated%20by%20the%20plant-specific%20RAB%20GTPase%20ARA6&amp;author=K.%20Ebine&amp;author=M.%20Fujimoto&amp;author=Y.%20Okatani&amp;author=T.%20Nishiyama&amp;author=T.%20Goh&amp;volume=13&amp;publication_year=2011&amp;pages=853-859&amp;pmid=21666683&amp;doi=10.1038/ncb2270&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23">
<cite>
Elias M., Drdova E., Ziak D., Bavlnka B., Hala M., Cvrckova F., et al. (2003). The exocyst complex in plants. Cell Biol. Int.
27, 199–201. doi: 10.1016/S1065-6995(02)00349-9
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S1065-6995(02)00349-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12681307/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell%20Biol.%20Int.&amp;title=The%20exocyst%20complex%20in%20plants&amp;author=M.%20Elias&amp;author=E.%20Drdova&amp;author=D.%20Ziak&amp;author=B.%20Bavlnka&amp;author=M.%20Hala&amp;volume=27&amp;publication_year=2003&amp;pages=199-201&amp;pmid=12681307&amp;doi=10.1016/S1065-6995(02)00349-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24">
<cite>
Elmore J. M., Liu J., Smith B., Phinney B., Coaker G. (2012). Quantitative proteomics reveals dynamic changes in the plasma membrane during <em>Arabidopsis</em> immune signaling. Mol. Cell. Proteomics
11, M111.014555. doi: 10.1074/mcp.M111.014555
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/mcp.M111.014555" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3322570/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22215637/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Quantitative%20proteomics%20reveals%20dynamic%20changes%20in%20the%20plasma%20membrane%20during%20Arabidopsis%20immune%20signaling&amp;author=J.%20M.%20Elmore&amp;author=J.%20Liu&amp;author=B.%20Smith&amp;author=B.%20Phinney&amp;author=G.%20Coaker&amp;volume=11&amp;publication_year=2012&amp;pages=M111.014555&amp;pmid=22215637&amp;doi=10.1074/mcp.M111.014555&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25">
<cite>
Escobar-Restrepo J.-M., Huck N., Kessler S., Gagliardini V., Gheyselinck J., Yang W.-C., et al. (2007). The FERONIA receptor-like kinase mediates male-female interactions during pollen tube reception. Science
317, 656–660. doi: 10.1126/science.1143562
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1143562" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17673660/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20FERONIA%20receptor-like%20kinase%20mediates%20male-female%20interactions%20during%20pollen%20tube%20reception&amp;author=J.-M.%20Escobar-Restrepo&amp;author=N.%20Huck&amp;author=S.%20Kessler&amp;author=V.%20Gagliardini&amp;author=J.%20Gheyselinck&amp;volume=317&amp;publication_year=2007&amp;pages=656-660&amp;pmid=17673660&amp;doi=10.1126/science.1143562&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26">
<cite>
Fendrych M., Synek L., Pečenková T., Drdová E. J., Sekereš J., de Rycke R., et al. (2013). Visualization of the exocyst complex dynamics at the plasma membrane of arabidopsis thaliana. Mol. Biol. Cell
24, 510–520. doi: 10.1091/mbc.e12-06-0492
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.e12-06-0492" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3571873/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23283982/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=Visualization%20of%20the%20exocyst%20complex%20dynamics%20at%20the%20plasma%20membrane%20of%20arabidopsis%20thaliana&amp;author=M.%20Fendrych&amp;author=L.%20Synek&amp;author=T.%20Pe%C4%8Denkov%C3%A1&amp;author=E.%20J.%20Drdov%C3%A1&amp;author=J.%20Sekere%C5%A1&amp;volume=24&amp;publication_year=2013&amp;pages=510-520&amp;pmid=23283982&amp;doi=10.1091/mbc.e12-06-0492&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27">
<cite>
Fendrych M., Synek L., Pečenková T., Toupalová H., Cole R., Drdová E., et al. (2010). The arabidopsis exocyst complex is involved in cytokinesis and cell plate maturation. Plant Cell
22, 3053–3065. doi: 10.1105/tpc.110.074351
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.110.074351" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2965533/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20870962/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=The%20arabidopsis%20exocyst%20complex%20is%20involved%20in%20cytokinesis%20and%20cell%20plate%20maturation&amp;author=M.%20Fendrych&amp;author=L.%20Synek&amp;author=T.%20Pe%C4%8Denkov%C3%A1&amp;author=H.%20Toupalov%C3%A1&amp;author=R.%20Cole&amp;volume=22&amp;publication_year=2010&amp;pages=3053-3065&amp;pmid=20870962&amp;doi=10.1105/tpc.110.074351&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28">
<cite>
Friml J. (2010). Subcellular trafficking of PIN auxin efflux carriers in auxin transport. Eur. J. Cell Biol.
89, 231–235. doi: 10.1016/j.ejcb.2009.11.003
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ejcb.2009.11.003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19944476/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Cell%20Biol.&amp;title=Subcellular%20trafficking%20of%20PIN%20auxin%20efflux%20carriers%20in%20auxin%20transport&amp;author=J.%20Friml&amp;volume=89&amp;publication_year=2010&amp;pages=231-235&amp;pmid=19944476&amp;doi=10.1016/j.ejcb.2009.11.003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29">
<cite>
Gómez-Escudero J., Moreno V., Martín-Alonso M., et al. (2017). E-cadherin cleavage by MT2-MMP regulates apical junctional signaling and epithelial homeostasis in the intestine. J. Cell Sci.
130, 4013–4027. doi:  10.1242/jcs.203687
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/jcs.203687" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5769589/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29061881/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci.&amp;title=E-cadherin%20cleavage%20by%20MT2-MMP%20regulates%20apical%20junctional%20signaling%20and%20epithelial%20homeostasis%20in%20the%20intestine&amp;author=J.%20G%C3%B3mez-Escudero&amp;author=V.%20Moreno&amp;author=M.%20Mart%C3%ADn-Alonso&amp;volume=130&amp;publication_year=2017&amp;pages=4013-4027&amp;pmid=29061881&amp;doi=10.1242/jcs.203687&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30">
<cite>
Guo J., Xu C., Wu D., Zhao Y., Qiu Y., Wang X., et al. (2018). Bph6 encodes an exocyst-localized protein and confers broad resistance to planthoppers in rice. Nat. Genet.
50, 297–306. doi: 10.1038/s41588-018-0039-6
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41588-018-0039-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29358653/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&amp;title=Bph6%20encodes%20an%20exocyst-localized%20protein%20and%20confers%20broad%20resistance%20to%20planthoppers%20in%20rice&amp;author=J.%20Guo&amp;author=C.%20Xu&amp;author=D.%20Wu&amp;author=Y.%20Zhao&amp;author=Y.%20Qiu&amp;volume=50&amp;publication_year=2018&amp;pages=297-306&amp;pmid=29358653&amp;doi=10.1038/s41588-018-0039-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31">
<cite>
Hála M., Cole R., Synek L., Drdová E., Pečenková T., Nordheim A., et al. (2008). An exocyst complex functions in plant cell growth in <em>Arabidopsis</em> and tobacco. Plant Cell
20, 1330–1345. doi: 10.1105/tpc.108.059105
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.108.059105" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2438459/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18492870/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=An%20exocyst%20complex%20functions%20in%20plant%20cell%20growth%20in%20Arabidopsis%20and%20tobacco&amp;author=M.%20H%C3%A1la&amp;author=R.%20Cole&amp;author=L.%20Synek&amp;author=E.%20Drdov%C3%A1&amp;author=T.%20Pe%C4%8Denkov%C3%A1&amp;volume=20&amp;publication_year=2008&amp;pages=1330-1345&amp;pmid=18492870&amp;doi=10.1105/tpc.108.059105&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32">
<cite>
Hallgren J., Tsirigos K. D., Pedersen M. D., Almagro Armenteros J. J., Marcatili P., Nielsen H., et al. (2022). DeepTMHMM predicts alpha and beta transmembrane proteins using deep neural networks. Bioinformatics. 2022.04.08.487609. doi:  10.1101/2022.04.08.487609
35134862</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/2022.04.08.487609" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=DeepTMHMM%20predicts%20alpha%20and%20beta%20transmembrane%20proteins%20using%20deep%20neural%20networks&amp;author=J.%20Hallgren&amp;author=K.%20D.%20Tsirigos&amp;author=M.%20D.%20Pedersen&amp;author=J.%20J.%20Almagro%20Armenteros&amp;author=P.%20Marcatili&amp;publication_year=2022&amp;pages=2022.04.08.487609&amp;doi=10.1101/2022.04.08.487609&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B33">
<cite>
Han B., Chen S., Dai S., Yang N., Wang T. (2010). Isobaric tags for relative and absolute quantification-based comparative proteomics reveals the features of plasma membrane-associated proteomes of pollen grains and pollen tubes from <em>Lilium davidii</em>
. J. Integr. Plant Biol.
52, 1043–1058. doi: 10.1111/j.1744-7909.2010.00996.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1744-7909.2010.00996.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21106004/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Integr.%20Plant%20Biol.&amp;title=Isobaric%20tags%20for%20relative%20and%20absolute%20quantification-based%20comparative%20proteomics%20reveals%20the%20features%20of%20plasma%20membrane-associated%20proteomes%20of%20pollen%20grains%20and%20pollen%20tubes%20from%20Lilium%20davidii&amp;author=B.%20Han&amp;author=S.%20Chen&amp;author=S.%20Dai&amp;author=N.%20Yang&amp;author=T.%20Wang&amp;volume=52&amp;publication_year=2010&amp;pages=1043-1058&amp;pmid=21106004&amp;doi=10.1111/j.1744-7909.2010.00996.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34">
<cite>
Hématy K., De Bellis D., Wang X., Mähönen A. P., Geldner N. (2022). Analysis of exocyst function in endodermis reveals its widespread contribution and specificity of action. Plant Physiol.
189, 557–566. doi: 10.1093/plphys/kiac019
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/plphys/kiac019" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9157074/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35099565/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Analysis%20of%20exocyst%20function%20in%20endodermis%20reveals%20its%20widespread%20contribution%20and%20specificity%20of%20action&amp;author=K.%20H%C3%A9maty&amp;author=D.%20De%20Bellis&amp;author=X.%20Wang&amp;author=A.%20P.%20M%C3%A4h%C3%B6nen&amp;author=N.%20Geldner&amp;volume=189&amp;publication_year=2022&amp;pages=557-566&amp;pmid=35099565&amp;doi=10.1093/plphys/kiac019&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35">
<cite>
Hsu C.-C., Zhu Y., Arrington J. V., Paez J. S., Wang P., Zhu P., et al. (2018). Universal plant phosphoproteomics workflow and its application to tomato signaling in response to cold stress. Mol. Cell. Proteomics
17, 2068–2080. doi: 10.1074/mcp.TIR118.000702
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/mcp.TIR118.000702" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6166681/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30006488/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Universal%20plant%20phosphoproteomics%20workflow%20and%20its%20application%20to%20tomato%20signaling%20in%20response%20to%20cold%20stress&amp;author=C.-C.%20Hsu&amp;author=Y.%20Zhu&amp;author=J.%20V.%20Arrington&amp;author=J.%20S.%20Paez&amp;author=P.%20Wang&amp;volume=17&amp;publication_year=2018&amp;pages=2068-2080&amp;pmid=30006488&amp;doi=10.1074/mcp.TIR118.000702&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36">
<cite>
Huang L., Li X., Li Y., Yin X., Li Y., Wu B., et al. (2019). Endosidin2-14 targets the exocyst complex in plants and fungal pathogens to inhibit exocytosis. Plant Physiol.
180, 1756–1770. doi: 10.1104/pp.18.01457
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.18.01457" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6752926/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31072814/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Endosidin2-14%20targets%20the%20exocyst%20complex%20in%20plants%20and%20fungal%20pathogens%20to%20inhibit%20exocytosis&amp;author=L.%20Huang&amp;author=X.%20Li&amp;author=Y.%20Li&amp;author=X.%20Yin&amp;author=Y.%20Li&amp;volume=180&amp;publication_year=2019&amp;pages=1756-1770&amp;pmid=31072814&amp;doi=10.1104/pp.18.01457&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B37">
<cite>
Irani N. G., Di Rubbo S., Mylle E., Van den Begin J., Schneider-Pizoń J., Hniliková J., et al. (2012). Fluorescent castasterone reveals BRI1 signaling from the plasma membrane. Nat. Chem. Biol.
8, 583–589. doi: 10.1038/nchembio.958
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nchembio.958" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22561410/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.%20Biol.&amp;title=Fluorescent%20castasterone%20reveals%20BRI1%20signaling%20from%20the%20plasma%20membrane&amp;author=N.%20G.%20Irani&amp;author=S.%20Di%20Rubbo&amp;author=E.%20Mylle&amp;author=J.%20Van%20den%20Begin&amp;author=J.%20Schneider-Pizo%C5%84&amp;volume=8&amp;publication_year=2012&amp;pages=583-589&amp;pmid=22561410&amp;doi=10.1038/nchembio.958&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B38">
<cite>
Kajiura H., Koiwa H., Nakazawa Y., Okazawa A., Kobayashi A., Seki T., et al. (2010). Two Arabidopsis thaliana Golgi α-mannosidase I enzymes are responsible for plant N-glycan maturation. Glycobiology
20, 235–247. doi:  10.1093/glycob/cwp170
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/glycob/cwp170" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19914916/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Glycobiology&amp;title=Two%20Arabidopsis%20thaliana%20Golgi%20%CE%B1-mannosidase%20I%20enzymes%20are%20responsible%20for%20plant%20N-glycan%20maturation&amp;author=H.%20Kajiura&amp;author=H.%20Koiwa&amp;author=Y.%20Nakazawa&amp;author=A.%20Okazawa&amp;author=A.%20Kobayashi&amp;volume=20&amp;publication_year=2010&amp;pages=235-247&amp;pmid=19914916&amp;doi=10.1093/glycob/cwp170&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B39">
<cite>
Kleine-Vehn J., Wabnik K., Martinière A., Łangowski Ł., Willig K., Naramoto S., et al. (2011). Recycling, clustering, and endocytosis jointly maintain PIN auxin carrier polarity at the plasma membrane. Mol. Syst. Biol.
7, 540. doi: 10.1038/msb.2011.72
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/msb.2011.72" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3261718/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22027551/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Syst.%20Biol.&amp;title=Recycling,%20clustering,%20and%20endocytosis%20jointly%20maintain%20PIN%20auxin%20carrier%20polarity%20at%20the%20plasma%20membrane&amp;author=J.%20Kleine-Vehn&amp;author=K.%20Wabnik&amp;author=A.%20Martini%C3%A8re&amp;author=%C5%81.%20%C5%81angowski&amp;author=K.%20Willig&amp;volume=7&amp;publication_year=2011&amp;pages=540&amp;pmid=22027551&amp;doi=10.1038/msb.2011.72&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B40">
<cite>
Kulich I., Cole R., Drdová E., Cvrčková F., Soukup A., Fowler J., et al. (2010). Arabidopsis exocyst subunits SEC8 and EXO70A1 and exocyst interactor ROH1 are involved in the localized deposition of seed coat pectin. New Phytol.
188, 615–625. doi: 10.1111/j.1469-8137.2010.03372.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1469-8137.2010.03372.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20618910/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Arabidopsis%20exocyst%20subunits%20SEC8%20and%20EXO70A1%20and%20exocyst%20interactor%20ROH1%20are%20involved%20in%20the%20localized%20deposition%20of%20seed%20coat%20pectin&amp;author=I.%20Kulich&amp;author=R.%20Cole&amp;author=E.%20Drdov%C3%A1&amp;author=F.%20Cvr%C4%8Dkov%C3%A1&amp;author=A.%20Soukup&amp;volume=188&amp;publication_year=2010&amp;pages=615-625&amp;pmid=20618910&amp;doi=10.1111/j.1469-8137.2010.03372.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B41">
<cite>
Kulich I., Pečenková T., Sekereš J., Smetana O., Fendrych M., Foissner I., et al. (2013). Arabidopsis exocyst subcomplex containing subunit EXO70B1 is involved in autophagy-related transport to the vacuole. Traffic
14, 1155–1165. doi: 10.1111/tra.12101
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tra.12101" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23944713/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Arabidopsis%20exocyst%20subcomplex%20containing%20subunit%20EXO70B1%20is%20involved%20in%20autophagy-related%20transport%20to%20the%20vacuole&amp;author=I.%20Kulich&amp;author=T.%20Pe%C4%8Denkov%C3%A1&amp;author=J.%20Sekere%C5%A1&amp;author=O.%20Smetana&amp;author=M.%20Fendrych&amp;volume=14&amp;publication_year=2013&amp;pages=1155-1165&amp;pmid=23944713&amp;doi=10.1111/tra.12101&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B42">
<cite>
Kulich I., Vojtíková Z., Glanc M., Ortmannová J., Rasmann S., Žárský V. (2015). Cell wall maturation of arabidopsis trichomes is dependent on exocyst subunit EXO70H4 and involves callose deposition. Plant Physiol.
168, 120–131. doi: 10.1104/pp.15.00112
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.15.00112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4424025/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25767057/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Cell%20wall%20maturation%20of%20arabidopsis%20trichomes%20is%20dependent%20on%20exocyst%20subunit%20EXO70H4%20and%20involves%20callose%20deposition&amp;author=I.%20Kulich&amp;author=Z.%20Vojt%C3%ADkov%C3%A1&amp;author=M.%20Glanc&amp;author=J.%20Ortmannov%C3%A1&amp;author=S.%20Rasmann&amp;volume=168&amp;publication_year=2015&amp;pages=120-131&amp;pmid=25767057&amp;doi=10.1104/pp.15.00112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B43">
<cite>
Larson E. R., Ortmannová J., Donald N. A., Alvim J., Blatt M. R., Žárský V. (2020). Synergy among exocyst and SNARE interactions identifies a functional hierarchy in secretion during vegetative growth. Plant Cell
32, 2951–2963. doi: 10.1105/tpc.20.00280
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.20.00280" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7474273/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32699172/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Synergy%20among%20exocyst%20and%20SNARE%20interactions%20identifies%20a%20functional%20hierarchy%20in%20secretion%20during%20vegetative%20growth&amp;author=E.%20R.%20Larson&amp;author=J.%20Ortmannov%C3%A1&amp;author=N.%20A.%20Donald&amp;author=J.%20Alvim&amp;author=M.%20R.%20Blatt&amp;volume=32&amp;publication_year=2020&amp;pages=2951-2963&amp;pmid=32699172&amp;doi=10.1105/tpc.20.00280&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B44">
<cite>
Lešková A., Labajová M., Krausko M., Zahradníková A., Baluška F., Mičieta K., et al. (2020). Endosidin 2 accelerates PIN2 endocytosis and disturbs intracellular trafficking of PIN2, PIN3, and PIN4 but not of SYT1 (DC bassham, ed.). PloS One
15, e0237448. doi:  10.1371/journal.pone.0237448
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0237448" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7425933/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32790800/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PloS%20One&amp;title=Endosidin%202%20accelerates%20PIN2%20endocytosis%20and%20disturbs%20intracellular%20trafficking%20of%20PIN2,%20PIN3,%20and%20PIN4%20but%20not%20of%20SYT1%20(DC%20bassham,%20ed.)&amp;author=A.%20Le%C5%A1kov%C3%A1&amp;author=M.%20Labajov%C3%A1&amp;author=M.%20Krausko&amp;author=A.%20Zahradn%C3%ADkov%C3%A1&amp;author=F.%20Balu%C5%A1ka&amp;volume=15&amp;publication_year=2020&amp;pages=e0237448&amp;pmid=32790800&amp;doi=10.1371/journal.pone.0237448&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B45">
<cite>
Li B., Takahashi D., Kawamura Y., Uemura M. (2012). Comparison of plasma membrane proteomic changes of arabidopsis suspension-cultured cells (T87 line) after cold and ABA treatment in association with freezing tolerance development. Plant Cell Physiol.
53, 543–554. doi: 10.1093/pcp/pcs010
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/pcp/pcs010" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22318864/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol.&amp;title=Comparison%20of%20plasma%20membrane%20proteomic%20changes%20of%20arabidopsis%20suspension-cultured%20cells%20(T87%20line)%20after%20cold%20and%20ABA%20treatment%20in%20association%20with%20freezing%20tolerance%20development&amp;author=B.%20Li&amp;author=D.%20Takahashi&amp;author=Y.%20Kawamura&amp;author=M.%20Uemura&amp;volume=53&amp;publication_year=2012&amp;pages=543-554&amp;pmid=22318864&amp;doi=10.1093/pcp/pcs010&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B46">
<cite>
Li B., Takahashi D., Kawamura Y., Uemura M. (2018). “Plasma membrane proteomics of arabidopsis suspension-cultured cells associated with growth phase using nano-LC-MS/MS,” in Methods in molecular biology. plant membrane proteomics: methods and protocols. Eds. Mock H.-P., Matros A., Witzel K. (New York, NY: Springer; ), 185–194.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-1-4939-7411-5_12" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29086404/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20molecular%20biology.%20plant%20membrane%20proteomics:%20methods%20and%20protocols&amp;author=B.%20Li&amp;author=D.%20Takahashi&amp;author=Y.%20Kawamura&amp;author=M.%20Uemura&amp;publication_year=2018&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B47">
<cite>
Li J., Wen J., Lease K. A., Doke J. T., Tax F. E., Walker J. C. (2002). BAK1, an <em>Arabidopsis</em> LRR receptor-like protein kinase, interacts with BRI1 and modulates brassinosteroid signaling. Cell
110, 213–222. doi: 10.1016/S0092-8674(02)00812-7
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0092-8674(02)00812-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12150929/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=BAK1,%20an%20Arabidopsis%20LRR%20receptor-like%20protein%20kinase,%20interacts%20with%20BRI1%20and%20modulates%20brassinosteroid%20signaling&amp;author=J.%20Li&amp;author=J.%20Wen&amp;author=K.%20A.%20Lease&amp;author=J.%20T.%20Doke&amp;author=F.%20E.%20Tax&amp;volume=110&amp;publication_year=2002&amp;pages=213-222&amp;pmid=12150929&amp;doi=10.1016/S0092-8674(02)00812-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B48">
<cite>
Lin Y., Ding Y., Wang J., Shen J., Kung C. H., Zhuang X., et al. (2015). Exocyst-positive organelles and autophagosomes are distinct organelles in plants. Plant Physiol.
169, 1917–1932. doi:  10.1104/pp.15.00953
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.15.00953" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4634068/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26358417/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Exocyst-positive%20organelles%20and%20autophagosomes%20are%20distinct%20organelles%20in%20plants&amp;author=Y.%20Lin&amp;author=Y.%20Ding&amp;author=J.%20Wang&amp;author=J.%20Shen&amp;author=C.%20H.%20Kung&amp;volume=169&amp;publication_year=2015&amp;pages=1917-1932&amp;pmid=26358417&amp;doi=10.1104/pp.15.00953&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B49">
<cite>
Mayers J. R., Hu T., Wang C., Cárdenas J. J., Tan Y., Pan J., et al. (2017). SCD1 and SCD2 form a complex that functions with the exocyst and RabE1 in exocytosis and cytokinesis. Plant Cell
29, 2610–2625. doi: 10.1105/tpc.17.00409
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.17.00409" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5774579/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28970336/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=SCD1%20and%20SCD2%20form%20a%20complex%20that%20functions%20with%20the%20exocyst%20and%20RabE1%20in%20exocytosis%20and%20cytokinesis&amp;author=J.%20R.%20Mayers&amp;author=T.%20Hu&amp;author=C.%20Wang&amp;author=J.%20J.%20C%C3%A1rdenas&amp;author=Y.%20Tan&amp;volume=29&amp;publication_year=2017&amp;pages=2610-2625&amp;pmid=28970336&amp;doi=10.1105/tpc.17.00409&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B50">
<cite>
Mi H., Huang X., Muruganujan A., Tang H., Mills C., Kang D., et al. (2017). PANTHER version 11: expanded annotation data from gene ontology and reactome pathways, and data analysis tool enhancements. Nucleic Acids Res.
45, D183–D189. doi: 10.1093/nar/gkw1138
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkw1138" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5210595/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27899595/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=PANTHER%20version%2011:%20expanded%20annotation%20data%20from%20gene%20ontology%20and%20reactome%20pathways,%20and%20data%20analysis%20tool%20enhancements&amp;author=H.%20Mi&amp;author=X.%20Huang&amp;author=A.%20Muruganujan&amp;author=H.%20Tang&amp;author=C.%20Mills&amp;volume=45&amp;publication_year=2017&amp;pages=D183-D189&amp;pmid=27899595&amp;doi=10.1093/nar/gkw1138&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B51">
<cite>
Mi H., Muruganujan A., Casagrande J. T., Thomas P. D. (2013). Large-Scale gene function analysis with the PANTHER classification system. Nat. Protoc.
8, 1551–1566. doi: 10.1038/nprot.2013.092
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nprot.2013.092" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6519453/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23868073/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Large-Scale%20gene%20function%20analysis%20with%20the%20PANTHER%20classification%20system&amp;author=H.%20Mi&amp;author=A.%20Muruganujan&amp;author=J.%20T.%20Casagrande&amp;author=P.%20D.%20Thomas&amp;volume=8&amp;publication_year=2013&amp;pages=1551-1566&amp;pmid=23868073&amp;doi=10.1038/nprot.2013.092&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B52">
<cite>
Nelson C. J., Hegeman A. D., Harms A. C., Sussman M. R. (2006). A quantitative analysis of <em>Arabidopsis</em> plasma membrane using trypsin-catalyzed <sup>18</sup>O labeling. Mol. Cell. Proteomics
5, 1382–1395. doi: 10.1074/mcp.M500414-MCP200
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/mcp.M500414-MCP200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16635983/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=A%20quantitative%20analysis%20of%20Arabidopsis%20plasma%20membrane%20using%20trypsin-catalyzed%2018O%20labeling&amp;author=C.%20J.%20Nelson&amp;author=A.%20D.%20Hegeman&amp;author=A.%20C.%20Harms&amp;author=M.%20R.%20Sussman&amp;volume=5&amp;publication_year=2006&amp;pages=1382-1395&amp;pmid=16635983&amp;doi=10.1074/mcp.M500414-MCP200&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B53">
<cite>
Nie L., Feng J., Fan P., Chen X., Guo J., Lv S., et al. (2015). Comparative proteomics of root plasma membrane proteins reveals the involvement of calcium signalling in NaCl-facilitated nitrate uptake in <em>Salicornia europaea</em>
. J. Exp. Bot.
66, 4497–4510. doi: 10.1093/jxb/erv216
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/erv216" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4507759/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25956883/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=Comparative%20proteomics%20of%20root%20plasma%20membrane%20proteins%20reveals%20the%20involvement%20of%20calcium%20signalling%20in%20NaCl-facilitated%20nitrate%20uptake%20in%20Salicornia%20europaea&amp;author=L.%20Nie&amp;author=J.%20Feng&amp;author=P.%20Fan&amp;author=X.%20Chen&amp;author=J.%20Guo&amp;volume=66&amp;publication_year=2015&amp;pages=4497-4510&amp;pmid=25956883&amp;doi=10.1093/jxb/erv216&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B54">
<cite>
Nohzadeh Malakshah S., Habibi Rezaei M., Heidari M., Hosseini Salekdeh G. (2007). Proteomics reveals new salt responsive proteins associated with rice plasma membrane. Biosci. Biotechnol. Biochem.
71, 2144–2154. doi: 10.1271/bbb.70027
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1271/bbb.70027" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17827676/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biosci.%20Biotechnol.%20Biochem.&amp;title=Proteomics%20reveals%20new%20salt%20responsive%20proteins%20associated%20with%20rice%20plasma%20membrane&amp;author=S.%20Nohzadeh%20Malakshah&amp;author=M.%20Habibi%20Rezaei&amp;author=M.%20Heidari&amp;author=G.%20Hosseini%20Salekdeh&amp;volume=71&amp;publication_year=2007&amp;pages=2144-2154&amp;pmid=17827676&amp;doi=10.1271/bbb.70027&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B55">
<cite>
Nouri M.-Z., Komatsu S. (2010). Comparative analysis of soybean plasma membrane proteins under osmotic stress using gel-based and LC MS/MS-based proteomics approaches. Proteomics
10, 1930–1945. doi: 10.1002/pmic.200900632
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/pmic.200900632" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20209511/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Comparative%20analysis%20of%20soybean%20plasma%20membrane%20proteins%20under%20osmotic%20stress%20using%20gel-based%20and%20LC%20MS/MS-based%20proteomics%20approaches&amp;author=M.-Z.%20Nouri&amp;author=S.%20Komatsu&amp;volume=10&amp;publication_year=2010&amp;pages=1930-1945&amp;pmid=20209511&amp;doi=10.1002/pmic.200900632&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B56">
<cite>
Novick P., Ferro S., Schekman R. (1981). Order of events in the yeast secretory pathway. Cell
25, 461–469. doi: 10.1016/0092-8674(81)90064-7
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0092-8674(81)90064-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/7026045/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Order%20of%20events%20in%20the%20yeast%20secretory%20pathway&amp;author=P.%20Novick&amp;author=S.%20Ferro&amp;author=R.%20Schekman&amp;volume=25&amp;publication_year=1981&amp;pages=461-469&amp;pmid=7026045&amp;doi=10.1016/0092-8674(81)90064-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B57">
<cite>
Oda Y., Fukuda H. (2011). Dynamics of arabidopsis SUN proteins during mitosis and their involvement in nuclear shaping: dynamics and functions of arabidopsis SUN proteins. Plant J.
66, 629–641. doi: 10.1111/j.1365-313X.2011.04523.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2011.04523.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21294795/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=Dynamics%20of%20arabidopsis%20SUN%20proteins%20during%20mitosis%20and%20their%20involvement%20in%20nuclear%20shaping:%20dynamics%20and%20functions%20of%20arabidopsis%20SUN%20proteins&amp;author=Y.%20Oda&amp;author=H.%20Fukuda&amp;volume=66&amp;publication_year=2011&amp;pages=629-641&amp;pmid=21294795&amp;doi=10.1111/j.1365-313X.2011.04523.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B58">
<cite>
Ogura T., Goeschl C., Filiault D., Mirea M., Slovak R., Wolhrab B., et al. (2019). Root system depth in arabidopsis is shaped by <em>EXOCYST70A3 </em>via the dynamic modulation of auxin transport. Cell
178, 400–412.e16. doi: 10.1016/j.cell.2019.06.021
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cell.2019.06.021" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31299202/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Root%20system%20depth%20in%20arabidopsis%20is%20shaped%20by%20EXOCYST70A3%20via%20the%20dynamic%20modulation%20of%20auxin%20transport&amp;author=T.%20Ogura&amp;author=C.%20Goeschl&amp;author=D.%20Filiault&amp;author=M.%20Mirea&amp;author=R.%20Slovak&amp;volume=178&amp;publication_year=2019&amp;pages=400-412.e16&amp;pmid=31299202&amp;doi=10.1016/j.cell.2019.06.021&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B59">
<cite>
Oh I. S., Park A. R., Bae M. S., Kwon S. J., Kim Y. S., Lee J. E., et al. (2005). Secretome analysis reveals an <em>Arabidopsis</em> lipase involved in defense against <em>Alternaria brassicicola</em>
. Plant Cell
17, 2832–2847. doi: 10.1105/tpc.105.034819
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.105.034819" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1242276/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16126835/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Secretome%20analysis%20reveals%20an%20Arabidopsis%20lipase%20involved%20in%20defense%20against%20Alternaria%20brassicicola&amp;author=I.%20S.%20Oh&amp;author=A.%20R.%20Park&amp;author=M.%20S.%20Bae&amp;author=S.%20J.%20Kwon&amp;author=Y.%20S.%20Kim&amp;volume=17&amp;publication_year=2005&amp;pages=2832-2847&amp;pmid=16126835&amp;doi=10.1105/tpc.105.034819&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B60">
<cite>
Ortega M. A., Villiger R. K., Harrison-Chau M., Lieu S., Tamashiro K.-K., Lee A. J., et al. (2022). Exocyst inactivation in urothelial cells disrupts autophagy and activates non-canonical NF-κB signaling. Dis. Models Mech.
15, dmm049785. doi: 10.1242/dmm.049785
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/dmm.049785" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9586569/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36004645/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Dis.%20Models%20Mech.&amp;title=Exocyst%20inactivation%20in%20urothelial%20cells%20disrupts%20autophagy%20and%20activates%20non-canonical%20NF-%CE%BAB%20signaling&amp;author=M.%20A.%20Ortega&amp;author=R.%20K.%20Villiger&amp;author=M.%20Harrison-Chau&amp;author=S.%20Lieu&amp;author=K.-K.%20Tamashiro&amp;volume=15&amp;publication_year=2022&amp;pages=dmm049785&amp;pmid=36004645&amp;doi=10.1242/dmm.049785&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B61">
<cite>
Pečenková T., Hála M., Kulich I., Kocourková D., Drdová E., Fendrych M., et al. (2011). The role for the exocyst complex subunits Exo70B2 and Exo70H1 in the plant–pathogen interaction. J. Exp. Bot.
62, 2107–2116. doi: 10.1093/jxb/erq402
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/jxb/erq402" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3060688/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21199889/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Bot.&amp;title=The%20role%20for%20the%20exocyst%20complex%20subunits%20Exo70B2%20and%20Exo70H1%20in%20the%20plant%E2%80%93pathogen%20interaction&amp;author=T.%20Pe%C4%8Denkov%C3%A1&amp;author=M.%20H%C3%A1la&amp;author=I.%20Kulich&amp;author=D.%20Kocourkov%C3%A1&amp;author=E.%20Drdov%C3%A1&amp;volume=62&amp;publication_year=2011&amp;pages=2107-2116&amp;pmid=21199889&amp;doi=10.1093/jxb/erq402&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B62">
<cite>
Prigent M., Dubois T., Raposo G., Derrien V., Tenza D., Rossé C., et al. (2003). ARF6 controls post-endocytic recycling through its downstream exocyst complex effector. J. Cell Biol.
163, 1111–1121. doi: 10.1083/jcb.200305029
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1083/jcb.200305029" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2173613/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/14662749/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol.&amp;title=ARF6%20controls%20post-endocytic%20recycling%20through%20its%20downstream%20exocyst%20complex%20effector&amp;author=M.%20Prigent&amp;author=T.%20Dubois&amp;author=G.%20Raposo&amp;author=V.%20Derrien&amp;author=D.%20Tenza&amp;volume=163&amp;publication_year=2003&amp;pages=1111-1121&amp;pmid=14662749&amp;doi=10.1083/jcb.200305029&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B63">
<cite>
Robinson D. G., Ding Y., Jiang L. (2016). Unconventional protein secretion in plants: a critical assessment. Protoplasma
253, 31–43. doi: 10.1007/s00709-015-0887-1
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00709-015-0887-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26410830/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Protoplasma&amp;title=Unconventional%20protein%20secretion%20in%20plants:%20a%20critical%20assessment&amp;author=D.%20G.%20Robinson&amp;author=Y.%20Ding&amp;author=L.%20Jiang&amp;volume=253&amp;publication_year=2016&amp;pages=31-43&amp;pmid=26410830&amp;doi=10.1007/s00709-015-0887-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B64">
<cite>
Rothman J. E. (2002). The machinery and principles of vesicle transport in the cell. Nat. Med.
8, 1059–1062. doi: 10.1038/nm770
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nm770" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12357232/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&amp;title=The%20machinery%20and%20principles%20of%20vesicle%20transport%20in%20the%20cell&amp;author=J.%20E.%20Rothman&amp;volume=8&amp;publication_year=2002&amp;pages=1059-1062&amp;pmid=12357232&amp;doi=10.1038/nm770&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B65">
<cite>
Schekman R. (2010). Charting the secretory pathway in a simple eukaryote. Mol. Biol. Cell
21, 3781–3784. doi: 10.1091/mbc.e10-05-0416
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1091/mbc.e10-05-0416" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2982102/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21079008/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=Charting%20the%20secretory%20pathway%20in%20a%20simple%20eukaryote&amp;author=R.%20Schekman&amp;volume=21&amp;publication_year=2010&amp;pages=3781-3784&amp;pmid=21079008&amp;doi=10.1091/mbc.e10-05-0416&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B66">
<cite>
Smyth G. K. (2004). Linear models and empirical bayes methods for assessing differential expression in microarray experiments. Stat. Appl. Genet. Mol. Biol.
3, 1–25. doi: 10.2202/1544-6115.1027
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2202/1544-6115.1027" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16646809/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Stat.%20Appl.%20Genet.%20Mol.%20Biol.&amp;title=Linear%20models%20and%20empirical%20bayes%20methods%20for%20assessing%20differential%20expression%20in%20microarray%20experiments&amp;author=G.%20K.%20Smyth&amp;volume=3&amp;publication_year=2004&amp;pages=1-25&amp;pmid=16646809&amp;doi=10.2202/1544-6115.1027&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B67">
<cite>
Soares N. C., Francisco R., Ricardo C. P., Jackson P. A. (2007). Proteomics of ionically bound and soluble extracellular proteins in <em>Medicago truncatula</em> leaves. Proteomics
7, 2070–2082. doi: 10.1002/pmic.200600953
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/pmic.200600953" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17514681/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Proteomics%20of%20ionically%20bound%20and%20soluble%20extracellular%20proteins%20in%20Medicago%20truncatula%20leaves&amp;author=N.%20C.%20Soares&amp;author=R.%20Francisco&amp;author=C.%20P.%20Ricardo&amp;author=P.%20A.%20Jackson&amp;volume=7&amp;publication_year=2007&amp;pages=2070-2082&amp;pmid=17514681&amp;doi=10.1002/pmic.200600953&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B68">
<cite>
Stillwell W. (2016). “Membrane proteins,” in An introduction to biological membranes (Elsevier; ), 89–110. doi:  10.1016/B978-0-444-63772-7.00006-3
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/B978-0-444-63772-7.00006-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=An%20introduction%20to%20biological%20membranes&amp;author=W.%20Stillwell&amp;publication_year=2016&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B69">
<cite>
Synek L., Pleskot R., Sekereš J., Serrano N., Vukašinović N., Ortmannová J., et al. (2021). Plasma membrane phospholipid signature recruits the plant exocyst complex via the EXO70A1 subunit. Proc. Natl. Acad. Sci.
118, e2105287118. doi: 10.1073/pnas.2105287118
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.2105287118" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8433549/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34470819/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.&amp;title=Plasma%20membrane%20phospholipid%20signature%20recruits%20the%20plant%20exocyst%20complex%20via%20the%20EXO70A1%20subunit&amp;author=L.%20Synek&amp;author=R.%20Pleskot&amp;author=J.%20Sekere%C5%A1&amp;author=N.%20Serrano&amp;author=N.%20Vuka%C5%A1inovi%C4%87&amp;volume=118&amp;publication_year=2021&amp;pages=e2105287118&amp;pmid=34470819&amp;doi=10.1073/pnas.2105287118&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B70">
<cite>
Synek L., Schlager N., Eliáš M., Quentin M., Hauser M.-T., Žárský V. (2006). AtEXO70A1, a member of a family of putative exocyst subunits specifically expanded in land plants, is important for polar growth and plant development. Plant J.
48, 54–72. doi: 10.1111/j.1365-313X.2006.02854.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2006.02854.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2865999/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16942608/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=AtEXO70A1,%20a%20member%20of%20a%20family%20of%20putative%20exocyst%20subunits%20specifically%20expanded%20in%20land%20plants,%20is%20important%20for%20polar%20growth%20and%20plant%20development&amp;author=L.%20Synek&amp;author=N.%20Schlager&amp;author=M.%20Eli%C3%A1%C5%A1&amp;author=M.%20Quentin&amp;author=M.-T.%20Hauser&amp;volume=48&amp;publication_year=2006&amp;pages=54-72&amp;pmid=16942608&amp;doi=10.1111/j.1365-313X.2006.02854.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B71">
<cite>
Takahashi D., Li B., Nakayama T., Kawamura Y., Uemura M. (2013). Plant plasma membrane proteomics for improving cold tolerance. Front. Plant Sci.
4, 90. doi: 10.3389/fpls.2013.00090
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2013.00090" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3628364/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23616787/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=Plant%20plasma%20membrane%20proteomics%20for%20improving%20cold%20tolerance&amp;author=D.%20Takahashi&amp;author=B.%20Li&amp;author=T.%20Nakayama&amp;author=Y.%20Kawamura&amp;author=M.%20Uemura&amp;volume=4&amp;publication_year=2013&amp;pages=90&amp;pmid=23616787&amp;doi=10.3389/fpls.2013.00090&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B72">
<cite>
Takano J., Tanaka M., Toyoda A., Miwa K., Kasai K., Fuji K., et al. (2010). Polar localization and degradation of <em>Arabidopsis</em> boron transporters through distinct trafficking pathways. Proc. Natl. Acad. Sci.
107, 5220–5225. doi: 10.1073/pnas.0910744107
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0910744107" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2841934/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20194745/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.&amp;title=Polar%20localization%20and%20degradation%20of%20Arabidopsis%20boron%20transporters%20through%20distinct%20trafficking%20pathways&amp;author=J.%20Takano&amp;author=M.%20Tanaka&amp;author=A.%20Toyoda&amp;author=K.%20Miwa&amp;author=K.%20Kasai&amp;volume=107&amp;publication_year=2010&amp;pages=5220-5225&amp;pmid=20194745&amp;doi=10.1073/pnas.0910744107&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B73">
<cite>
Tweto J., Doyle D. (1976). Turnover of the plasma membrane proteins of hepatoma tissue culture cells. J. Biol. Chem.
251, 872–882. doi: 10.1016/S0021-9258(17)33865-6
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0021-9258(17)33865-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/175063/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Turnover%20of%20the%20plasma%20membrane%20proteins%20of%20hepatoma%20tissue%20culture%20cells&amp;author=J.%20Tweto&amp;author=D.%20Doyle&amp;volume=251&amp;publication_year=1976&amp;pages=872-882&amp;pmid=175063&amp;doi=10.1016/S0021-9258(17)33865-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B74">
<cite>
Tyanova S., Temu T., Cox J. (2016. a). The MaxQuant computational platform for mass spectrometry-based shotgun proteomics. Nat. Protoc.
11, 2301–2319. doi: 10.1038/nprot.2016.136
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nprot.2016.136" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27809316/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=The%20MaxQuant%20computational%20platform%20for%20mass%20spectrometry-based%20shotgun%20proteomics&amp;author=S.%20Tyanova&amp;author=T.%20Temu&amp;author=J.%20Cox&amp;volume=11&amp;publication_year=2016&amp;pages=2301-2319&amp;pmid=27809316&amp;doi=10.1038/nprot.2016.136&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B75">
<cite>
Tyanova S., Temu T., Sinitcyn P., Carlson A., Hein M. Y., Geiger T., et al. (2016. b). The Perseus computational platform for comprehensive analysis of (prote)omics data. Nat. Methods
13, 731–740. doi: 10.1038/nmeth.3901
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.3901" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27348712/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=The%20Perseus%20computational%20platform%20for%20comprehensive%20analysis%20of%20(prote)omics%20data&amp;author=S.%20Tyanova&amp;author=T.%20Temu&amp;author=P.%20Sinitcyn&amp;author=A.%20Carlson&amp;author=M.%20Y.%20Hein&amp;volume=13&amp;publication_year=2016&amp;pages=731-740&amp;pmid=27348712&amp;doi=10.1038/nmeth.3901&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B76">
<cite>
Vellosillo T., Dinneny J. R., Somerville C. R., Ehrhardt D. W. (2021). TRANVIA (TVA) facilitates cellulose synthase trafficking and delivery to the plasma membrane. Proc. Natl. Acad. Sci.
118, e2021790118. doi: 10.1073/pnas.2021790118
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.2021790118" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8325160/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34290139/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.&amp;title=TRANVIA%20(TVA)%20facilitates%20cellulose%20synthase%20trafficking%20and%20delivery%20to%20the%20plasma%20membrane&amp;author=T.%20Vellosillo&amp;author=J.%20R.%20Dinneny&amp;author=C.%20R.%20Somerville&amp;author=D.%20W.%20Ehrhardt&amp;volume=118&amp;publication_year=2021&amp;pages=e2021790118&amp;pmid=34290139&amp;doi=10.1073/pnas.2021790118&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B77">
<cite>
Voothuluru P., Anderson J. C., Sharp R. E., Peck S. C. (2016). Plasma membrane proteomics in the maize primary root growth zone: novel insights into root growth adaptation to water stress. Plant Cell Environ.
39, 2043–2054. doi: 10.1111/pce.12778
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/pce.12778" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27341663/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Environ.&amp;title=Plasma%20membrane%20proteomics%20in%20the%20maize%20primary%20root%20growth%20zone:%20novel%20insights%20into%20root%20growth%20adaptation%20to%20water%20stress&amp;author=P.%20Voothuluru&amp;author=J.%20C.%20Anderson&amp;author=R.%20E.%20Sharp&amp;author=S.%20C.%20Peck&amp;volume=39&amp;publication_year=2016&amp;pages=2043-2054&amp;pmid=27341663&amp;doi=10.1111/pce.12778&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B78">
<cite>
Wang J., Ding Y., Wang J., Hillmer S., Miao Y., Lo S. W., et al. (2010). EXPO, an exocyst-positive organelle distinct from multivesicular endosomes and autophagosomes, mediates cytosol to cell wall exocytosis in <em>Arabidopsis</em> and tobacco cells. Plant Cell
22, 4009–4030. doi: 10.1105/tpc.110.080697
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.110.080697" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3027174/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21193573/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=EXPO,%20an%20exocyst-positive%20organelle%20distinct%20from%20multivesicular%20endosomes%20and%20autophagosomes,%20mediates%20cytosol%20to%20cell%20wall%20exocytosis%20in%20Arabidopsis%20and%20tobacco%20cells&amp;author=J.%20Wang&amp;author=Y.%20Ding&amp;author=J.%20Wang&amp;author=S.%20Hillmer&amp;author=Y.%20Miao&amp;volume=22&amp;publication_year=2010&amp;pages=4009-4030&amp;pmid=21193573&amp;doi=10.1105/tpc.110.080697&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B79">
<cite>
Wang L., Li H., Lv X., Chen T., Li R., Xue Y., et al. (2015). Spatiotemporal dynamics of the BRI1 receptor and its regulation by membrane microdomains in living <em>Arabidopsis</em> cells. Mol. Plant
8, 1334–1349. doi: 10.1016/j.molp.2015.04.005
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.molp.2015.04.005" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25896454/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Plant&amp;title=Spatiotemporal%20dynamics%20of%20the%20BRI1%20receptor%20and%20its%20regulation%20by%20membrane%20microdomains%20in%20living%20Arabidopsis%20cells&amp;author=L.%20Wang&amp;author=H.%20Li&amp;author=X.%20Lv&amp;author=T.%20Chen&amp;author=R.%20Li&amp;volume=8&amp;publication_year=2015&amp;pages=1334-1349&amp;pmid=25896454&amp;doi=10.1016/j.molp.2015.04.005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B80">
<cite>
Wang Z.-Y., Seto H., Fujioka S., Yoshida S., Chory J. (2001). BRI1 is a critical component of a plasma-membrane receptor for plant steroids. Nature
410, 380–383. doi: 10.1038/35066597
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/35066597" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11268216/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=BRI1%20is%20a%20critical%20component%20of%20a%20plasma-membrane%20receptor%20for%20plant%20steroids&amp;author=Z.-Y.%20Wang&amp;author=H.%20Seto&amp;author=S.%20Fujioka&amp;author=S.%20Yoshida&amp;author=J.%20Chory&amp;volume=410&amp;publication_year=2001&amp;pages=380-383&amp;pmid=11268216&amp;doi=10.1038/35066597&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B81">
<cite>
Wang J., Song J., Clark G., Roux S. J. (2018). ANN1 and ANN2 function in post-phloem sugar transport in root tips to affect primary root growth. Plant Physiol.
178, 390–401. doi: 10.1104/pp.18.00713
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.18.00713" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6130020/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30018170/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=ANN1%20and%20ANN2%20function%20in%20post-phloem%20sugar%20transport%20in%20root%20tips%20to%20affect%20primary%20root%20growth&amp;author=J.%20Wang&amp;author=J.%20Song&amp;author=G.%20Clark&amp;author=S.%20J.%20Roux&amp;volume=178&amp;publication_year=2018&amp;pages=390-401&amp;pmid=30018170&amp;doi=10.1104/pp.18.00713&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B82">
<cite>
Wen F., VanEtten H. D., Tsaprailis G., Hawes M. C. (2007). Extracellular proteins in pea root tip and border cell exudates. Plant Physiol.
143, 773–783. doi: 10.1104/pp.106.091637
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.106.091637" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1803736/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17142479/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&amp;title=Extracellular%20proteins%20in%20pea%20root%20tip%20and%20border%20cell%20exudates&amp;author=F.%20Wen&amp;author=H.%20D.%20VanEtten&amp;author=G.%20Tsaprailis&amp;author=M.%20C.%20Hawes&amp;volume=143&amp;publication_year=2007&amp;pages=773-783&amp;pmid=17142479&amp;doi=10.1104/pp.106.091637&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B83">
<cite>
Wessel D., Flügge U. I. (1984). A method for the quantitative recovery of protein in dilute solution in the presence of detergents and lipids. Anal. Biochem.
138, 141–143. doi: 10.1016/0003-2697(84)90782-6
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/0003-2697(84)90782-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/6731838/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Biochem.&amp;title=A%20method%20for%20the%20quantitative%20recovery%20of%20protein%20in%20dilute%20solution%20in%20the%20presence%20of%20detergents%20and%20lipids&amp;author=D.%20Wessel&amp;author=U.%20I.%20Fl%C3%BCgge&amp;volume=138&amp;publication_year=1984&amp;pages=141-143&amp;pmid=6731838&amp;doi=10.1016/0003-2697(84)90782-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B84">
<cite>
Wu B., Guo W. (2015). The exocyst at a glance. J. Cell Sci.
128, 2957–2964. doi: 10.1242/jcs.156398
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1242/jcs.156398" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4541039/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26240175/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci.&amp;title=The%20exocyst%20at%20a%20glance&amp;author=B.%20Wu&amp;author=W.%20Guo&amp;volume=128&amp;publication_year=2015&amp;pages=2957-2964&amp;pmid=26240175&amp;doi=10.1242/jcs.156398&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B85">
<cite>
Xu J., Scheres B. (2005). Dissection of arabidopsis ADP-RIBOSYLATION FACTOR 1 function in epidermal cell polarity. Plant Cell
17, 525–536. doi: 10.1105/tpc.104.028449
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.104.028449" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC548823/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15659621/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Dissection%20of%20arabidopsis%20ADP-RIBOSYLATION%20FACTOR%201%20function%20in%20epidermal%20cell%20polarity&amp;author=J.%20Xu&amp;author=B.%20Scheres&amp;volume=17&amp;publication_year=2005&amp;pages=525-536&amp;pmid=15659621&amp;doi=10.1105/tpc.104.028449&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B86">
<cite>
Žárský V. (2022). Exocyst functions in plants – secretion and autophagy. FEBS Lett.
1873–3468. doi:  10.1002/1873-3468.14430
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/1873-3468.14430" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35729750/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&amp;title=Exocyst%20functions%20in%20plants%20%E2%80%93%20secretion%20and%20autophagy&amp;author=V.%20%C5%BD%C3%A1rsk%C3%BD&amp;publication_year=2022&amp;pages=1873-3468&amp;pmid=35729750&amp;doi=10.1002/1873-3468.14430&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B87">
<cite>
Žárský V., Kulich I., Fendrych M., Pečenková T. (2013). Exocyst complexes multiple functions in plant cells secretory pathways. Curr. Opin. Plant Biol.
16, 726–733. doi: 10.1016/j.pbi.2013.10.013
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.pbi.2013.10.013" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24246229/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Plant%20Biol.&amp;title=Exocyst%20complexes%20multiple%20functions%20in%20plant%20cells%20secretory%20pathways&amp;author=V.%20%C5%BD%C3%A1rsk%C3%BD&amp;author=I.%20Kulich&amp;author=M.%20Fendrych&amp;author=T.%20Pe%C4%8Denkov%C3%A1&amp;volume=16&amp;publication_year=2013&amp;pages=726-733&amp;pmid=24246229&amp;doi=10.1016/j.pbi.2013.10.013&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B88">
<cite>
Zhang C., Brown M. Q., van de Ven W., Zhang Z.-M., Wu B., Young M. C., et al. (2016). Endosidin2 targets conserved exocyst complex subunit EXO70 to inhibit exocytosis. Proc. Natl. Acad. Sci.
113, E41–E50. doi: 10.1073/pnas.1521248112
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1521248112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4711834/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26607451/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.&amp;title=Endosidin2%20targets%20conserved%20exocyst%20complex%20subunit%20EXO70%20to%20inhibit%20exocytosis&amp;author=C.%20Zhang&amp;author=M.%20Q.%20Brown&amp;author=W.%20van%20de%20Ven&amp;author=Z.-M.%20Zhang&amp;author=B.%20Wu&amp;volume=113&amp;publication_year=2016&amp;pages=E41-E50&amp;pmid=26607451&amp;doi=10.1073/pnas.1521248112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B89">
<cite>
Zhang W., Huang L., Zhang C., Staiger C. J. (2021). Arabidopsis myosin XIK interacts with the exocyst complex to facilitate vesicle tethering during exocytosis. Plant Cell
33, 2454–2478. doi: 10.1093/plcell/koab116
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/plcell/koab116" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8364239/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33871640/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Arabidopsis%20myosin%20XIK%20interacts%20with%20the%20exocyst%20complex%20to%20facilitate%20vesicle%20tethering%20during%20exocytosis&amp;author=W.%20Zhang&amp;author=L.%20Huang&amp;author=C.%20Zhang&amp;author=C.%20J.%20Staiger&amp;volume=33&amp;publication_year=2021&amp;pages=2454-2478&amp;pmid=33871640&amp;doi=10.1093/plcell/koab116&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B90">
<cite>
Zhang Y., Immink R., Liu C.-M., Emons A. M., Ketelaar T. (2013). The arabidopsis exocyst subunit SEC3A is essential for embryo development and accumulates in transient puncta at the plasma membrane. New Phytol.
199, 74–88. doi: 10.1111/nph.12236
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/nph.12236" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23495664/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=The%20arabidopsis%20exocyst%20subunit%20SEC3A%20is%20essential%20for%20embryo%20development%20and%20accumulates%20in%20transient%20puncta%20at%20the%20plasma%20membrane&amp;author=Y.%20Zhang&amp;author=R.%20Immink&amp;author=C.-M.%20Liu&amp;author=A.%20M.%20Emons&amp;author=T.%20Ketelaar&amp;volume=199&amp;publication_year=2013&amp;pages=74-88&amp;pmid=23495664&amp;doi=10.1111/nph.12236&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B91">
<cite>
Zhang Y., Liu C.-M., Emons A.-M. C., Ketelaar T. (2010). The plant exocyst. J. Integr. Plant Biol.
52, 138–146. doi: 10.1111/j.1744-7909.2010.00929.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1744-7909.2010.00929.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20377676/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Integr.%20Plant%20Biol.&amp;title=The%20plant%20exocyst&amp;author=Y.%20Zhang&amp;author=C.-M.%20Liu&amp;author=A.-M.%20C.%20Emons&amp;author=T.%20Ketelaar&amp;volume=52&amp;publication_year=2010&amp;pages=138-146&amp;pmid=20377676&amp;doi=10.1111/j.1744-7909.2010.00929.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B92">
<cite>
Zhang Z. J., Peck S. C. (2011). Simplified enrichment of plasma membrane proteins for proteomic analyses in <em>Arabidopsis thaliana</em>
. Proteomics
11, 1780–1788. doi: 10.1002/pmic.201000648
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/pmic.201000648" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21433285/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Simplified%20enrichment%20of%20plasma%20membrane%20proteins%20for%20proteomic%20analyses%20in%20Arabidopsis%20thaliana&amp;author=Z.%20J.%20Zhang&amp;author=S.%20C.%20Peck&amp;volume=11&amp;publication_year=2011&amp;pages=1780-1788&amp;pmid=21433285&amp;doi=10.1002/pmic.201000648&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B93">
<cite>
Zhang W., Staiger C. J. (2021). Revising the role of cortical cytoskeleton during secretion: actin and myosin XI function in vesicle tethering. Int. J. Mol. Sci.
23, 317. doi: 10.3390/ijms23010317
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/ijms23010317" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC8745698/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35008741/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Revising%20the%20role%20of%20cortical%20cytoskeleton%20during%20secretion:%20actin%20and%20myosin%20XI%20function%20in%20vesicle%20tethering&amp;author=W.%20Zhang&amp;author=C.%20J.%20Staiger&amp;volume=23&amp;publication_year=2021&amp;pages=317&amp;pmid=35008741&amp;doi=10.3390/ijms23010317&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B94">
<cite>
Zhu X., Li S., Pan S., Xin X., Gu Y. (2018). CSI1, PATROL1, and exocyst complex cooperate in delivery of cellulose synthase complexes to the plasma membrane. Proc. Natl. Acad. Sci.
115, E3578–E3587. doi: 10.1073/pnas.1800182115
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1800182115" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5899483/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29581258/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.&amp;title=CSI1,%20PATROL1,%20and%20exocyst%20complex%20cooperate%20in%20delivery%20of%20cellulose%20synthase%20complexes%20to%20the%20plasma%20membrane&amp;author=X.%20Zhu&amp;author=S.%20Li&amp;author=S.%20Pan&amp;author=X.%20Xin&amp;author=Y.%20Gu&amp;volume=115&amp;publication_year=2018&amp;pages=E3578-E3587&amp;pmid=29581258&amp;doi=10.1073/pnas.1800182115&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B95">
<cite>
Zhu D., Shang M., Gao C., Shen J. (2020). Protein trafficking in plant cells: Tools and markers. China Life Sci.
63, 343–363. doi: 10.1007/s11427-019-9598-3
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11427-019-9598-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31707539/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=China%20Life%20Sci.&amp;title=Protein%20trafficking%20in%20plant%20cells:%20Tools%20and%20markers&amp;author=D.%20Zhu&amp;author=M.%20Shang&amp;author=C.%20Gao&amp;author=J.%20Shen&amp;volume=63&amp;publication_year=2020&amp;pages=343-363&amp;pmid=31707539&amp;doi=10.1007/s11427-019-9598-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ol></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/DataSheet_1.pdf">Click here for additional data file.</a><sup> (337.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/Table_1.xlsx">Click here for additional data file.</a><sup> (1.7MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/Table_2.xlsx">Click here for additional data file.</a><sup> (19KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/Table_3.xlsx">Click here for additional data file.</a><sup> (78.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/Table_4.xlsx">Click here for additional data file.</a><sup> (96.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/Table_5.xlsx">Click here for additional data file.</a><sup> (15.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/10264680/bin/Table_6.xlsx">Click here for additional data file.</a><sup> (16.3KB, xlsx) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found below: ProteomeXchange Consortium via the PRIDE database, with the identifier of PXD037455.</p></section></section></section></section></article>
</main></body></html>