<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4349987/"/><title>Proteomic Study of Microsomal Proteins Reveals a Key Role for Arabidopsis Annexin 1 in Mediating Heat Stress-Induced Increase in Intracellular Calcium Levels - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Mol Cell Proteomics</button></div>. 2015 Jan 13;14(3):686–694. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/mcp.M114.042697" rel="noopener noreferrer" target="_blank">10.1074/mcp.M114.042697</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Proteomic Study of Microsomal Proteins Reveals a Key Role for <em>Arabidopsis</em> Annexin 1 in Mediating Heat Stress-Induced Increase in Intracellular Calcium Levels<sup><a class="usa-link" href="#FN1">*</a></sup>
</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D"><span class="name western">Xu Wang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xu Wang</span></h3>
<div class="p">
<sup>‡</sup>From the ‡Key Laboratory of Molecular and Cellular Biology of Ministry of Education, Hebei Collaboration Innovation Center for Cell Signaling, Hebei Key Laboratory of Molecular and Cellular Biology, College of Life Science, Hebei Normal University, Shijiazhuang, Hebei 050024, China,</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D"><span class="name western">Xu Wang</span></a>
</div>
</div>
<sup>‡,</sup><sup>‖</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20X%22%5BAuthor%5D"><span class="name western">Xiaolong Ma</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Xiaolong Ma</span></h3>
<div class="p">
<sup>‡</sup>From the ‡Key Laboratory of Molecular and Cellular Biology of Ministry of Education, Hebei Collaboration Innovation Center for Cell Signaling, Hebei Key Laboratory of Molecular and Cellular Biology, College of Life Science, Hebei Normal University, Shijiazhuang, Hebei 050024, China,</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20X%22%5BAuthor%5D"><span class="name western">Xiaolong Ma</span></a>
</div>
</div>
<sup>‡,</sup><sup>‖</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D"><span class="name western">Hui Wang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Hui Wang</span></h3>
<div class="p">
<sup>‡</sup>From the ‡Key Laboratory of Molecular and Cellular Biology of Ministry of Education, Hebei Collaboration Innovation Center for Cell Signaling, Hebei Key Laboratory of Molecular and Cellular Biology, College of Life Science, Hebei Normal University, Shijiazhuang, Hebei 050024, China,</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D"><span class="name western">Hui Wang</span></a>
</div>
</div>
<sup>‡</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20B%22%5BAuthor%5D"><span class="name western">Bingjie Li</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Bingjie Li</span></h3>
<div class="p">
<sup>‡</sup>From the ‡Key Laboratory of Molecular and Cellular Biology of Ministry of Education, Hebei Collaboration Innovation Center for Cell Signaling, Hebei Key Laboratory of Molecular and Cellular Biology, College of Life Science, Hebei Normal University, Shijiazhuang, Hebei 050024, China,</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20B%22%5BAuthor%5D"><span class="name western">Bingjie Li</span></a>
</div>
</div>
<sup>‡</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Clark%20G%22%5BAuthor%5D"><span class="name western">Greg Clark</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Greg Clark</span></h3>
<div class="p">§Department of Molecular Biosciences, University of Texas, Austin, Texas 78712</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Clark%20G%22%5BAuthor%5D"><span class="name western">Greg Clark</span></a>
</div>
</div>
<sup>§</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20Y%22%5BAuthor%5D"><span class="name western">Yi Guo</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yi Guo</span></h3>
<div class="p">
<sup>‡</sup>From the ‡Key Laboratory of Molecular and Cellular Biology of Ministry of Education, Hebei Collaboration Innovation Center for Cell Signaling, Hebei Key Laboratory of Molecular and Cellular Biology, College of Life Science, Hebei Normal University, Shijiazhuang, Hebei 050024, China,</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20Y%22%5BAuthor%5D"><span class="name western">Yi Guo</span></a>
</div>
</div>
<sup>‡</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roux%20S%22%5BAuthor%5D"><span class="name western">Stan Roux</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Stan Roux</span></h3>
<div class="p">§Department of Molecular Biosciences, University of Texas, Austin, Texas 78712</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roux%20S%22%5BAuthor%5D"><span class="name western">Stan Roux</span></a>
</div>
</div>
<sup>§</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20D%22%5BAuthor%5D"><span class="name western">Daye Sun</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Daye Sun</span></h3>
<div class="p">
<sup>‡</sup>From the ‡Key Laboratory of Molecular and Cellular Biology of Ministry of Education, Hebei Collaboration Innovation Center for Cell Signaling, Hebei Key Laboratory of Molecular and Cellular Biology, College of Life Science, Hebei Normal University, Shijiazhuang, Hebei 050024, China,</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20D%22%5BAuthor%5D"><span class="name western">Daye Sun</span></a>
</div>
</div>
<sup>‡</sup>, <a aria-describedby="id9" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20W%22%5BAuthor%5D"><span class="name western">Wenqiang Tang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Wenqiang Tang</span></h3>
<div class="p">
<sup>‡</sup>From the ‡Key Laboratory of Molecular and Cellular Biology of Ministry of Education, Hebei Collaboration Innovation Center for Cell Signaling, Hebei Key Laboratory of Molecular and Cellular Biology, College of Life Science, Hebei Normal University, Shijiazhuang, Hebei 050024, China,</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20W%22%5BAuthor%5D"><span class="name western">Wenqiang Tang</span></a>
</div>
</div>
<sup>‡,</sup><sup>¶</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>‡</sup>From the ‡Key Laboratory of Molecular and Cellular Biology of Ministry of Education, Hebei Collaboration Innovation Center for Cell Signaling, Hebei Key Laboratory of Molecular and Cellular Biology, College of Life Science, Hebei Normal University, Shijiazhuang, Hebei 050024, China,</div>
<div id="aff2">§Department of Molecular Biosciences, University of Texas, Austin, Texas 78712</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>✉</sup><p class="display-inline">¶ To whom correspondence should be addressed:
College of Life Science, Hebei Normal University, East 20 Nan Er Huang Road, Shijiazhuang 050024 China. Tel.:
86 311 80787594; E-mail: <span>tangwq@mail.hebtu.edu.cn</span>.</p>
</div>
<div class="fn" id="FN2">
<sup>#</sup><p class="display-inline">‖ These authors contributed equally to this work.</p>
</div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2014 Jul 6; Revised 2015 Jan 9; Issue date 2015 Mar.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© 2015 by The American Society for Biochemistry and Molecular Biology, Inc.</div>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC4349987  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25587034/">25587034</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>To understand the early signaling steps in the response of plant cells to increased environmental temperature, 2-D difference gel electrophoresis was used to study the proteins in microsomes of <em>Arabidopsis</em> seedlings that are regulated early during heat stress. Using mass spectrometry, 19 microsomal proteins that showed an altered expression level within 5 min after heat treatment were identified. Among these proteins, annexin 1 (AtANN1) was one of those up-regulated rapidly after heat-shock treatment. Functional studies show loss-of-function mutants for <em>AtANN1</em> and its close homolog <em>AtANN2</em> were more sensitive to heat-shock treatment, whereas plants overexpressing <em>AtANN1</em> showed more resistance to this treatment. Correspondingly, the heat-induced expression of heat-shock proteins and heat-shock factors is inhibited in <em>ann1/ann2</em> double mutant, and the heat-activated increase in cytoplasmic calcium concentration ([Ca<sup>2+</sup>]<sub>cyt</sub>) is greatly impaired in the <em>ann1</em> mutant and almost undetectable in <em>ann1/ann2</em> double mutant. Taken together these results suggest that AtANN1 is important in regulating the heat-induced increase in [Ca<sup>2+</sup>]<sub>cyt</sub> and in the response of <em>Arabidopsis</em> seedlings to heat stress.</p></section><hr class="headless"/>
<p>Temperatures above the optimum are sensed as heat stress (HS)<a class="usa-link" href="#G1"><sup>1</sup></a> by all living organisms. When exposed to high temperature, plants have sophisticated mechanisms to maintain cellular homeostasis and minimize cell damage. Heat shock transcription factors (HSFs) mediate gene expression changes leading to the production of heat shock proteins (HSPs) that confer thermotolerance to plants (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>, <a aria-describedby="B2" class="usa-link" href="#B2">2</a>, <a aria-describedby="B3" class="usa-link" href="#B3">3</a>). In recent years, there has been considerable progress in identifying the signaling steps that enable plants to respond to heat stress at the molecular level. These include: activation of bZIP transcription factors by heat-induced proteolytic cleavage (<a aria-describedby="B4" class="usa-link" href="#B4">4</a>); inhibition of transgene-induced post-transcriptional gene silencing (PTGS) by SUPPRESSOR OF GENE SILENCING 3 (SGS3) (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>); accumulation of reactive oxygen species (ROS) by RBOHD activation (<a aria-describedby="B6" class="usa-link" href="#B6">6</a>); and reduced H2A.Z occupancy in nucleosomes of heat-regulated genes, which would be expected to stimulate transcription of these genes (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>).</p>
<p>Many reports also provide evidence that membrane-localized calcium channels can act as sensors for increased temperature in plant cells (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>, <a aria-describedby="B9" class="usa-link" href="#B9">9</a>). The increase in cytosolic calcium concentration ([Ca<sup>2+</sup>]<sub>cyt</sub>) caused by calcium influx from the extracellular matrix is an early critical step in the HS signal transduction pathway (<a aria-describedby="B10" class="usa-link" href="#B10">10</a>). In responding to this calcium influx, calcium-binding proteins such as calmodulin and calcium dependent protein kinases (CDPKs) are activated. Calmodulin binds to a calmodulin-binding protein kinase, which in turn, activates heat-regulated changes in gene expression by phosphorylating HSF (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>, <a aria-describedby="B12" class="usa-link" href="#B12">12</a>, <a aria-describedby="B13" class="usa-link" href="#B13">13</a>). CDPKs, on the other hand, might regulate plant heat-shock responses by activating multiple mitogen-activated protein kinases (<a aria-describedby="B14" class="usa-link" href="#B14">14</a>).</p>
<p>Despite considerable progress in studying the mechanisms of heat-shock signal transduction, our understanding of the heat signal perception and early signaling transduction mechanism is still very limited. Therefore, identifying novel proteins that regulate early heat-shock responses should help decoding high-temperature sensing and responding mechanism <em>in planta</em>. Along with advances in genome sequencing, new proteomic technologies have now emerged as an important tool in studying signal transduction mechanisms (<a aria-describedby="B15" class="usa-link" href="#B15">15</a>). Compared with traditional genetic studies, which rely on phenotypes caused by altering gene activity, proteomics not only provides real-time monitoring of abundance changes of certain proteins during cell and tissue responses to stimuli, but also reveals their mechanisms of function and regulation, such as post-translational modifications and protein–protein interaction. In this study, we used the two-dimensional difference gel electrophoresis (2-D DIGE) technology to find microsome-associated proteins whose level changes rapidly after heat-shock, and identified a set of membrane proteins that change abundance in response to a five-minute heat treatment. Among these proteins was annexin 1 (AtANN1), and here we demonstrate that this protein plays a key role in plant heat-stress responses by mediating a heat-induced increase in [Ca<sup>2+</sup>]<sub>cyt</sub>.</p>
<section id="sec1"><h2 class="pmc_sec_title">EXPERIMENTAL PROCEDURES</h2>
<section id="sec2"><h3 class="pmc_sec_title"></h3>
<section id="sec3"><h4 class="pmc_sec_title"></h4>
<section id="sec4"><h5 class="pmc_sec_title">Plant Materials and Heat Shock Treatment</h5>
<p><em>Arabidopsis thaliana</em> seeds were surface sterilized using 75% (v/v) ethanol and sowed in glass plates containing 25 ml growth medium containing half-strength Murashige and Skoog (1/2 MS) salt, 1.5% (w/v) sucrose, and 0.8% (w/v) agar. Plates were stratified at 4 °C in darkness for 3 days. Seedlings were allowed to grow at 22 °C under long-day conditions (16 h light/8 h dark) for 1 week. For heat shock treatment, the glass plates were wrapped with general kitchen plastic wrap and submerged in a water bath at the indicated temperature for different periods. The seedlings were returned to 22 °C under long-day to recover for 1 week before taking pictures for analysis. All the data showed in this study has been performed at least three times with similar results. Representative data from one repetition are shown in the figures.</p></section><section id="sec5"><h5 class="pmc_sec_title">2D-DIGE Protein Sample Preparation</h5>
<p>Because we were interested in finding heat shock-regulated proteins whose level changed rapidly, we used a water bath for our heat shock proteomic studies to make sure all the seedlings were instantly subjected to similar temperature at same time. To prepare the seedlings for 2D-DIGE protein sample extraction, <em>Arabidopsis</em> seedlings were grown in liquid medium containing 1/2 MS salt and 1.5% (w/v) sucrose under long day at 22 °C for 1 week. To avoid a nutrient shock effect, half of the old medium was transferred into a new container and incubated at 37 °C in the water bath, whereas the other half of the medium was kept at 22 °C with the seedlings. When the temperature of the water bath medium reached 37 °C, half of the seedlings were transferred into the medium to initiate the heat shock treatment. After heat shock treatment for 5 min, seedlings were harvested by filtering through a nylon mesh, quick tap-dried with tissue paper and frozen in liquid nitrogen. Microsomal proteins were extracted as described previously (<a aria-describedby="B16" class="usa-link" href="#B16">16</a>), and dissolved in DIGE buffer (7 <span class="font-variant-small-caps">m</span> urea, 2 <span class="font-variant-small-caps">m</span> thiourea, 4% CHAPS) at a concentration around 5–10 μg/μl.</p></section><section id="sec6"><h5 class="pmc_sec_title">2D-DIGE Procedure and Image Analysis</h5>
<p>Protein for 2D-DIGE analysis was labeled with CyDye<sup>TM</sup> DIGE fluor minimal labeling kit (GE Healthcare). In brief, 10 μl microsomal protein from control or heat treated sample was mixed with 0.25 μl of Tris-HCl (1.5 <span class="font-variant-small-caps">m</span>, pH 8.8) and 0.5 μl of 100 n<span class="font-variant-small-caps">m</span> Cy3 or Cy5, respectively, and incubated in darkness at 4 °C for 2 h. The reaction was terminated by adding 1 μl 10 m<span class="font-variant-small-caps">m</span> lysine and incubating with the protein on ice for 10 min. The Cy3- and Cy5- labeled pair of control and heat treated samples were combined, and to the mix was added 9 μl of 1 <span class="font-variant-small-caps">m</span> DTT, 4.5 μl of IPG buffer, pH 4–7. One hundred-micrograms of unlabeled protein each from control and heat-treated samples was then added to make the total protein amount in the mixture around 300 μg. The DIGE buffer was used to adjust the final volume to 450 μl for separation by isoelectric focusing (IEF) using 24 cm Immobiline Dry Strips, pH 4–7. Two-dimensional electrophoresis was performed according to (<a aria-describedby="B16" class="usa-link" href="#B16">16</a>). DIGE images were acquired using a Typhoon trio scanner (GE healthcare). At least six biological repeats were performed, and labeling of two pairs of biological repeat samples was swapped to avoid identifying proteins that are preferentially labeled by a specific CyDye. Spots that showed consistent heat shock regulated changes in at least five biological repeat samples were picked using a robotic spot picker (GE healthcare).</p></section><section id="sec7"><h5 class="pmc_sec_title">Protein In-gel Digestion and Mass Spectrometry Identification</h5>
<p>Protein in-gel digestion was performed as described previously (<a aria-describedby="B17" class="usa-link" href="#B17">17</a>). Extracted peptides in 0.1% formic acid were separated using a 100 μm ×150 mm C18 reverse phase column (Thermo Fisher Scientific) at a flow rate of 350 nl/min, and eluted using a linear acetonitrile (ACN) gradient (0–45%) with 0.1% formic acid over 60 min using a nano-LC system (Thermo Fisher Scientific). Eluted peptides were electrosprayed directly into a LTQ-XL linear ion trap mass spectrometer (Thermo Fisher Scientific) using an uncoated 15 μm inner diameter spraying needle with 2.1 kV electrospray voltage at 220 °C. Peptides were analyzed in positive ion mode with m/z range between 400 and 2000. Charged peaks over 1000 counts were selected for collision induced dissociation (CID) with 35% normalized collision energy and activation Q value was set to 0.25. The dynamic exclusion activation time was set to 30 s to prevent same <em>m/z</em> ions from being selected after its acquisition. CID product ions were analyzed on the linear ion trap in centroid mode.</p>
<p>Complete LC-MS/MS peak lists were searched against database generated from <em>Arabidopsis</em> subset of the NCBInr database (date 12/12/2012, 257,035 entries searched), using the SEQUEST search algorithm, which is part of the BioWorks 3.3.1 data analysis software (Thermo Fisher Scientific). The peptide mass range was set to 400–5000 amu. The precursor ions' tolerance and fragment ions' tolerance were set to 2.00 amu and 1.00 amu, respectively. Enzymatic digestion was specified as trypsin, with a maximum of two missed cleavages allowed. Cysteine carbamidomethylation was allowed as fixed modifications, and methionine oxidation were allowed as variable modification. Search result option filter was set as the following: Delta CN ≥ 0.1; Xcorr (±1, 2, 3) = 1.75, 2.5, 3.0; peptide probability≤0.01.</p></section><section id="sec8"><h5 class="pmc_sec_title">Isolation of Annexin T-DNA Mutants</h5>
<p>T-DNA insertional mutant for <em>AtANN1</em> (SALK_132169, SAIK_414_C01, WiscDsLox477–480P11), <em>AtANN2</em> (SALK_054223) and <em>AtANN6</em> (SALK_112492) were obtained from the Arabidopsis Biological Resource Center (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.arabidopsis.org" rel="noopener noreferrer" target="_blank">http://www.arabidopsis.org</a>), and are all in Columbia (Col-0) background. The identified <em>ann1–2</em> mutant was used to cross with <em>ann2</em> mutant. F3 segregated <em>ann1–2</em>/<em>ann2</em> double homozygous mutant were used for heat stress tolerance analysis. The gene specific primers used for genotyping the mutants were: 5′-gcctgcttcagcttttgtatg-3′ (left) and 5′-aacgctaccgacacaacattc-3′ (right) for SALK_132169; 5′-tgctccttctgatgatgctg-3′ (left) and 5′-ccaataaagcatcacgctca-3′ (right) for SAIL_414_C01; 5′-tggactcttgatccaccaga-3′ (left) and 5′-gagcaacaagcatgtcctca-3′ (right) for SALK_054223; 5′-ggcgtctctcaaaattccag-3′ (left) and 5′-caccagaaagctctccgtct-3′ (right) for SALK_112492. The transcript abundance of <em>AtANN1</em> and <em>AtANN2</em> in wild type and mutant seedlings was determined by semi-quantitative Reverse transcription PCR using 5′-atggcgactcttaaggtttc-3′ (left) and 5′-agcatcatcttcaccgagaag-3′ (right) for <em>AtANN1</em> and gene specific genotyping primer set for SALK_054223 and SALK_112492.</p></section><section id="sec9"><h5 class="pmc_sec_title">Plasmid Construction and Transformation</h5>
<p>Full-length CDS of <em>AtANN1</em> were amplified by PCR, cloned into pENTR/SD/D-TOPO vector (Invitrogen), and subcloned into Gateway compatible binary vector pEarleyGate 100 by LR clonase (Invitrogen). The construct was introduced into <em>Agrobacterium tumefaciens</em> strain GV3101 by electroporation and transformed into <em>Arabidopsis</em> (Col-0 ecotype) by the floral dipping method. Transgenic plants were selected by spraying plants with 5 μg/ml Basta (Finale). All transgenic lines used in this study are single insertional homozygous T3 lines.</p></section><section id="sec10"><h5 class="pmc_sec_title">Expression Analysis of HSFs and HSPs by RT-PCR and Western Blot</h5>
<p>Wild type and <em>ann1–2/ann2</em> double mutant seedlings were grown in the same glass plates containing 15 ml growth medium (1/2 MS salt, 1.5% sucrose, and 0.8% agar) for 1 week at 22 °C under long-day condition. For heat shock treatment, the plates were placed into a growth chamber with temperature set at 37 °C. Seedlings were harvested at different time intervals and frozen in liquid nitrogen. For RT-PCR analysis of the expression level of various HSFs transcript, total RNA was isolated using TRIzol reagent (Invitrogen) and reverse transcribed into cDNA using ExScript<sup>TM</sup> RT reagent kit (Takara) according to manufacturer's instruction. Primer pairs used for RT-PCR analysis were: 5′-atgatgaacccgtttctcccggaag-3′ and 5′-ggaggtggaagccaaactctcatcac-3′ for HSF A7A; 5′-atggaagaactgaaagtggaaat-3′ and 5′-aggttccgaaccaagaaaacccatt-3′ for HSF A2; 5′-atgacggctgtgacggcggcgcaaag-3′ and 5′-gttgcagactttgctgcttttccac-3′ for HSF B1; 5′-acagatttcgctaaagatttgctt-3′ and 5′-cgcttcttcttcttcttcaatctc-3′ for HSF B2A. For protein abundance analysis of HSPs, frozen tissues were ground into fine powder in liquid nitrogen, boiled with 2× SDS sampling buffer, loaded onto SDS-PAGE and transferred onto nitrocellulose membrane. Membrane was probed with different antibodies specifically against various HSPs (Agrisera, Vännäs, Sweden) and detected with SuperSignal West Dura Chemiluminescence System (Pierce).</p></section><section id="sec11"><h5 class="pmc_sec_title">Ca<sup>2+</sup> Measurement Using Aequorin</h5>
<p>Aequorin luminescence measurement was performed according to (<a aria-describedby="B18" class="usa-link" href="#B18">18</a>), with slight modifications. Transgenic <em>Arabidopsis</em> seedlings expressing cytosolic <em><sup>35</sup>S::Aequorin</em> were grown in 1/2 MS salt containing medium for 10 days under long day at 22 °C. Seedlings (n ≥ 15 per treatment) were transferred to a 96-well plate and incubated in dark with a calcium-measurement buffer (0.1 m<span class="font-variant-small-caps">m</span> KCl, 1 m<span class="font-variant-small-caps">m</span> CaCl<sub>2</sub>, 10 m<span class="font-variant-small-caps">m</span> MES, 5 μ<span class="font-variant-small-caps">m</span> coelenterazine-h) at 22 °C for 6 h. The seedlings were quick rinsed two times with the calcium-measurement buffer, 100 μl 22 °C (control) or 37 °C (heat shock) calcium measurement buffer was quickly added to each well, and the plate was loaded into a microplate luminometer (Centro LB 960, Berthold, Oak Ridge, TN) that is at room temperature (control) or has been prewarmed to 37 °C (heat shock). Aequorin luminescence signal in each well was recorded for 0.5 s every 30 s. After 45 counts, 100 μl reconstitution solution (2 <span class="font-variant-small-caps">m</span> CaCl<sub>2</sub>, 20% ethanol) was injected and luminescence signal in each well was recorded for 10 more cycles, 0.5 s each. Percentage reconstitution value was calculated by dividing the luminescence signal recorded at each time point by the highest reconstitution luminescence value recorded in each well, respectively, for wild type, <em>ann1</em> and <em>ann1/2</em> mutants. Relative aequorin signal is calculated by dividing percentage reconstitution value collected at 37 °C by percentage reconstitution value collected at 22 °C.</p></section><section id="sec12"><h5 class="pmc_sec_title">Determination of Total Ca<sup>2+</sup> in Plant Tissues</h5>
<p>Fifty seven-day-old <em>Arabidopsis</em> seedlings grown in 1/2 MS medium supplied with 1% sucrose were grouped as one sample and oven dried at 65 °C overnight. After recording the dry weights, the dried plants were digested with 5 ml HNO3 at 100 °C for 4 h. The digested mixture was transferred to a volumetric flask, and diluted precisely to 100 ml with double distilled water. Total plant Ca<sup>2+</sup> concentration was determined with a flame atomic absorption spectrophotomer (Model AA-240Z, Agilent Technologies, Santa Clara, CA).</p></section></section></section></section><section id="sec13"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="sec14"><h3 class="pmc_sec_title"></h3>
<section id="sec15"><h4 class="pmc_sec_title"></h4>
<section id="sec16"><h5 class="pmc_sec_title">The Effect of HS Temperature and Duration on the Survival of Arabidopsis <em>Seedlings</em>
</h5>
<p>Increased environmental temperature triggers heat-shock responses (HSR) in plants; however, if the temperature rises above a certain limit, or the duration of the high temperature is too long, plants will not be able to survive. To study the HS response of plant cells, the treatment used should be strong enough to stimulate the HSR, but at the same time mild enough not to kill the plant. In order to find a proper heat-shock treatment condition for our proteomic studies, plates containing one-week-old <em>Arabidopsis</em> seedlings were treated at a range of temperatures and treatment durations, and post-treatment survival rates were calculated after the seedlings recovered at 22 °C for 1 week. Treatment of <em>Arabidopsis</em> seedlings in a 42 °C water bath up to 10 min did not induce irreversible damage to seedlings, whereas treatment with a 42 °C water bath for 30 min killed almost all of the seedlings. In comparison, seedlings treated with a 37 °C water bath for 30 min survived (<a class="usa-link" href="#F1">Fig. 1</a>). As 37 °C has been previously shown to induce HSR (<a aria-describedby="B9" class="usa-link" href="#B9">9</a>, <a aria-describedby="B11" class="usa-link" href="#B11">11</a>, <a aria-describedby="B12" class="usa-link" href="#B12">12</a>), a 37 °C water bath was used as our treatment temperature during the following proteomic studies.</p>
<figure class="fig xbox font-sm" id="F1"><h6 class="obj_head">Fig. 1.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 1." class="graphic" height="201" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4700/4349987/f3fad4428d58/zjw0031549870001.jpg" width="500"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>The effect of heat shock temperature and duration on the survival of <em>Arabidopsis</em> seedlings.</strong> One-week-old <em>Arabidopsis</em> seedlings were heat shock treated at different temperatures for the indicated time period and allowed to recover at 22 °C for 7 days before taking pictures for comparison.</p></figcaption></figure></section><section id="sec17"><h5 class="pmc_sec_title">Identification of Microsomal Proteins Whose Level Changes Rapidly After Heat Shock</h5>
<p>Heat shock stimulates a rapid increase of the [Ca<sup>2+</sup>]<sub>cyt</sub>. This increase can be seen as early as 1 min after heat treatment, and reaches the maximum around 5 mins (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>). In order to study the early cellular responses stimulated by HS, we exposed one-week-old <em>Arabidopsis</em> seedlings to 37 °C for 5 mins and harvested them for 2D-DIGE analysis.</p>
<p>Protein samples from control and treated seedlings were analyzed in at least six biological-repeat experiments. In average, over 1500 protein spots were detected in each gel (<a class="usa-link" href="#F2">Fig. 2</a><em>A</em>). Examination of the DIGE images identified over 49 spots that showed consistent abundance changes induced by heat shock. Surprisingly, the abundance of most spots was increased by heat treatment, and only a few showed decreased levels. These spots of interest were picked, digested in gel by trypsin and analyzed by LC-MS/MS. A total of 27 heat-shock-regulated protein spots, which represented 19 unique proteins, were successfully identified (<a class="usa-link" href="#TI">Table I</a>). Functional classification showed these proteins are involved in processes such as hormone biosynthesis, protein translation, calcium signaling, cellular metabolism, and protein folding (chaperone).</p>
<figure class="fig xbox font-sm" id="F2"><h6 class="obj_head">Fig. 2.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4349987_zjw0031549870002.jpg" target="_blank"><img alt="Fig. 2." class="graphic zoom-in" height="793" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4700/4349987/d394d9a9eee2/zjw0031549870002.jpg" width="650"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>2D DIGE analysis of heat-regulated microsomal proteins identified AtANN1 as an early heat-responsive protein.</strong>
<em>A</em>, Microsomal protein from Col-0 seedlings treated with 37 °C for 5 min was labeled with Cy5, and microsomal protein from untreated seedlings was labeled with Cy3. The proteins were then separated using 24 cm, pH 4–7, IPG strips and a 10% SDS-PAGE gel. <em>B</em>, Three-dimensional view of spot 4191 in image <em>A</em>, left image represents Cy3 channel and right image represents Cy5 channel. <em>C</em>, Western analysis of AtANN1 protein accumulation after heat shock. One-week-old seedlings were heat-shock (37 °C) treated for the indicated time, microsomal proteins were extracted and separated by SDS-PAGE. Upper panel shows AtANN1 protein was detected by polyclonal antibodies raised specifically against AtANN1, lower panel shows Ponceau S stained rubisco large subunit as the equal-loading control.</p></figcaption></figure><section class="tw xbox font-sm" id="TI"><h6 class="obj_head">Table I. Early heat-regulated microsomal proteins identified by mass spectrometry. The heat regulated abundance changes were observed from at least five biological repeats. For MS/MS identification, number of unique peptides, sequence coverage of the identified proteins and the best matched scores are listed.</h6>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead valign="bottom"><tr>
<th align="center" colspan="1" rowspan="1">Spots</th>
<th align="center" colspan="1" rowspan="1">Accession number</th>
<th align="center" colspan="1" rowspan="1">Protein name</th>
<th align="center" colspan="1" rowspan="1">Unique peptides</th>
<th align="center" colspan="1" rowspan="1">Coverage (%)</th>
<th align="center" colspan="1" rowspan="1">Abundance change by HS</th>
<th align="center" colspan="1" rowspan="1">Score</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" colspan="1" rowspan="1">Hormone biosynthesis</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    2586</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At2g46370" rel="noopener noreferrer" target="_blank">At2g46370</a></td>
<td align="left" colspan="1" rowspan="1">Jasmonate-amido synthetase</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">6.26</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">30.19</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    3984</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At1g62380" rel="noopener noreferrer" target="_blank">At1g62380</a></td>
<td align="left" colspan="1" rowspan="1">ACC oxidase 2</td>
<td align="center" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">32.81</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">70.28</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    4095</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At3g44310" rel="noopener noreferrer" target="_blank">At3g44310</a></td>
<td align="left" colspan="1" rowspan="1">Nitrilase 1</td>
<td align="center" colspan="1" rowspan="1">9</td>
<td align="center" colspan="1" rowspan="1">37.28</td>
<td align="left" colspan="1" rowspan="1">down</td>
<td align="center" colspan="1" rowspan="1">100.28</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    4125</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At3g44310" rel="noopener noreferrer" target="_blank">At3g44310</a></td>
<td align="left" colspan="1" rowspan="1">Nitrilase 1</td>
<td align="center" colspan="1" rowspan="1">12</td>
<td align="center" colspan="1" rowspan="1">50</td>
<td align="left" colspan="1" rowspan="1">down</td>
<td align="center" colspan="1" rowspan="1">130.29</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    4285</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At3g44310" rel="noopener noreferrer" target="_blank">At3g44310</a></td>
<td align="left" colspan="1" rowspan="1">Nitrilase 1</td>
<td align="center" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">25.72</td>
<td align="left" colspan="1" rowspan="1">down</td>
<td align="center" colspan="1" rowspan="1">70.27</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Protein translation</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    3384</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At1g57720" rel="noopener noreferrer" target="_blank">At1g57720</a></td>
<td align="left" colspan="1" rowspan="1">Translation elongation factor EF1B</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">12.35</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">50.23</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    3394</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At1g57720" rel="noopener noreferrer" target="_blank">At1g57720</a></td>
<td align="left" colspan="1" rowspan="1">Translation elongation factor EF1B</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">15.5</td>
<td align="left" colspan="1" rowspan="1">down</td>
<td align="center" colspan="1" rowspan="1">50.24</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    3421</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At1g57720" rel="noopener noreferrer" target="_blank">At1g57720</a></td>
<td align="left" colspan="1" rowspan="1">Translation elongation factor EF1B</td>
<td align="center" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">26.63</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">170.29</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    3415</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At3g13920" rel="noopener noreferrer" target="_blank">At3g13920</a></td>
<td align="left" colspan="1" rowspan="1">Eukaryotic translation initiation factor 4A-1</td>
<td align="center" colspan="1" rowspan="1">7</td>
<td align="center" colspan="1" rowspan="1">28.16</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">100.31</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    3424</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At3g13920" rel="noopener noreferrer" target="_blank">At3g13920</a></td>
<td align="left" colspan="1" rowspan="1">Eukaryotic translation initiation factor 4A-1</td>
<td align="center" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">34.22</td>
<td align="left" colspan="1" rowspan="1">down</td>
<td align="center" colspan="1" rowspan="1">120.32</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    3804</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At4g20360" rel="noopener noreferrer" target="_blank">At4g20360</a></td>
<td align="left" colspan="1" rowspan="1">GTP binding Elongation factor Tu family protein</td>
<td align="center" colspan="1" rowspan="1">9</td>
<td align="center" colspan="1" rowspan="1">27.73</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">80.31</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Calcium signaling</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    4181</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At1g35720" rel="noopener noreferrer" target="_blank">At1g35720</a></td>
<td align="left" colspan="1" rowspan="1">Annexin 1</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">6.308</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">40.24</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    4191</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At1g35720" rel="noopener noreferrer" target="_blank">At1g35720</a></td>
<td align="left" colspan="1" rowspan="1">Annexin 1</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">15.77</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">70.28</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Protein folding</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    2371</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At5g42020" rel="noopener noreferrer" target="_blank">At5g42020</a></td>
<td align="left" colspan="1" rowspan="1">Luminal binding protein (BIP2)</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">4.8</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">30.19</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    2407</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At5g42020" rel="noopener noreferrer" target="_blank">At5g42020</a></td>
<td align="left" colspan="1" rowspan="1">Luminal binding protein (BIP2)</td>
<td align="center" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">11.98</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">50.24</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    6119</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At3g62030" rel="noopener noreferrer" target="_blank">At3g62030</a></td>
<td align="left" colspan="1" rowspan="1">Cyclophilin 20–3 (CYP20–3)</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">24.52</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">50.25</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Metabolism</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    1969</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At1g08520" rel="noopener noreferrer" target="_blank">At1g08520</a></td>
<td align="left" colspan="1" rowspan="1">CHLD subunit of the Mg-chelatase enzyme</td>
<td align="center" colspan="1" rowspan="1">16</td>
<td align="center" colspan="1" rowspan="1">25.58</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">160.23</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    3874</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At1g73110" rel="noopener noreferrer" target="_blank">At1g73110</a></td>
<td align="left" colspan="1" rowspan="1">Nucleoside triphosphate hydrolase</td>
<td align="center" colspan="1" rowspan="1">14</td>
<td align="center" colspan="1" rowspan="1">40.05</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">170.27</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    4097</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At2g05830" rel="noopener noreferrer" target="_blank">At2g05830</a></td>
<td align="left" colspan="1" rowspan="1">5-methylthioribose-1-phosphate isomerase</td>
<td align="center" colspan="1" rowspan="1">7</td>
<td align="center" colspan="1" rowspan="1">25.67</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">80.24</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    4113</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At5g15650" rel="noopener noreferrer" target="_blank">At5g15650</a></td>
<td align="left" colspan="1" rowspan="1">AtRGP2</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">11.67</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">40.23</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    4809</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At2g34470" rel="noopener noreferrer" target="_blank">At2g34470</a></td>
<td align="left" colspan="1" rowspan="1">Urease accessory protein G (UREG)</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">26.55</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">50.31</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    5092</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At4g30530" rel="noopener noreferrer" target="_blank">At4g30530</a></td>
<td align="left" colspan="1" rowspan="1">Gamma-glutamyl peptidase</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">17.60</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">30.26</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    5128</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At3g02780" rel="noopener noreferrer" target="_blank">At3g02780</a></td>
<td align="left" colspan="1" rowspan="1">Isopentenyl diphosphate isomerase 2</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">19.01</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">50.28</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    5116</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At3g11930" rel="noopener noreferrer" target="_blank">At3g11930</a></td>
<td align="left" colspan="1" rowspan="1">Adenine nucleotide alpha hydrolases like</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">11.5</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">20.19</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Others</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    2060</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At1g33790" rel="noopener noreferrer" target="_blank">At1g33790</a></td>
<td align="left" colspan="1" rowspan="1">Jacalin lectin family protein</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">11.69</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">40.17</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    2062</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At1g33790" rel="noopener noreferrer" target="_blank">At1g33790</a></td>
<td align="left" colspan="1" rowspan="1">Jacalin lectin family protein</td>
<td align="center" colspan="1" rowspan="1">14</td>
<td align="center" colspan="1" rowspan="1">36.85</td>
<td align="left" colspan="1" rowspan="1">up</td>
<td align="center" colspan="1" rowspan="1">120.31</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">    4734</td>
<td align="center" colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/At4g39090" rel="noopener noreferrer" target="_blank">At4g39090</a></td>
<td align="left" colspan="1" rowspan="1">Cysteine proteinases</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">9.24</td>
<td align="left" colspan="1" rowspan="1">down</td>
<td align="center" colspan="1" rowspan="1">30.22</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/TI/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></section></section><section id="sec18"><h5 class="pmc_sec_title">Annexin 1 is Essential for Plant Resistance to Heat-Shock Treatment</h5>
<p>Of all the microsomal proteins identified in <a class="usa-link" href="#TI">Table I</a>, we chose to further study AtANN1 (<a class="usa-link" href="#F2">Fig. 2</a><em>B</em>), because of prior evidence showing that it could play a major role in mediating plant stress responses. Using polyclonal antibody raised specific against a 31 amino acid sequence from AtANN1 (amino acids 200–231) (<a aria-describedby="B19" class="usa-link" href="#B19">19</a>), we confirmed the 2D-DIGE result in which AtANN1 protein levels increased in the microsomal fraction after 5 mins of HS (<a class="usa-link" href="#F2">Fig. 2</a><em>C</em>). We then isolated Transfer DNA (T-DNA) insertion mutants of <em>AtANN1</em> gene to investigate if <em>AtANN1</em> plays a significant role in HS responses in <em>Arabidopsis</em>. We obtained three insertion mutants for <em>AtANN1</em>, and these alleles were named as <em>ann1–1</em> (SalK_132169), <em>ann1–2</em> (Sail_414_C01), and <em>ann1–3</em> (WiscDsLox477–480P11; <a class="usa-link" href="#F3">Fig. 3</a><em>A</em>). The T-DNA insertion sites in these alleles were located at the 5′ UTR of the <em>AtANN1</em> gene (<em>ann1–1</em>) or within different positions of its second and third exon (<em>ann1–2</em> and <em>ann1–3</em>), respectively (<a class="usa-link" href="#F3">Fig. 3</a><em>A</em>). Reverse transcription PCR (RT-PCR) analysis of these T-DNA mutants showed that the expression of <em>AtANN1</em> in <em>ann1–2</em> and <em>ann1–3</em> is not detectable, whereas the expression of <em>AtANN1</em> in <em>ann1–1</em> was not affected (<a class="usa-link" href="#F3">Fig. 3</a><em>B</em>). We then compared the basal thermotolerance of these T-DNA insertional mutants with wild-type seedlings. One-week-old seedlings were subjected to 45 °C water bath for 13 min, and then allowed to recover at 22 °C for another week before they were photographed and calculate their survival rate. Consistent with the expression of endogenous <em>AtANN1</em> level, <em>ann1–2</em> and <em>ann1–3</em> seedlings were hypersensitive to heat-shock treatment, whereas heat sensitivity of <em>ann1–1</em> seedlings was similar to wild-type plants (<a class="usa-link" href="#F3">Fig. 3</a><em>C</em>, <a class="usa-link" href="#F3">3</a><em>D</em>).</p>
<figure class="fig xbox font-sm" id="F3"><h6 class="obj_head">Fig. 3.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4349987_zjw0031549870003.jpg" target="_blank"><img alt="Fig. 3." class="graphic zoom-in" height="465" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4700/4349987/89aa42c33909/zjw0031549870003.jpg" width="650"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong><em>AtANN1</em> is essential for plant resistance to heat shock treatment.</strong>
<em>A</em>, Diagram shows the T-DNA insertion positions for different <em>AtANN1</em> T-DNA mutants. <em>B</em>, RT-PCR analysis of <em>AtANN1</em> expression level in different <em>ann1</em> T-DNA mutants. <em>C</em>, The response of different <em>ann1</em> T-DNA mutants to heat-shock treatment. Seven-day-old seedlings grown on 1/2 MS plates at 22 °C were shifted to a 45 °C water bath for 13 min and then returned to 22 °C and allowed to recover for 1 week before taking photos. Scale bar represents 1 cm. <em>D</em>, Quantification of survival rate of seedlings shown in <em>C</em>. Data represent the mean value from five independent experiments, with 25 seedlings per experiment. One way ANOVA test was performed. Statistically significant differences are indicated by different lowercase letters (<em>p</em> &lt; 0.05). Error bars represent ±S.D.</p></figcaption></figure><p>There are eight members of the <em>Arabidopsis</em> annexin gene family. Phylogenetic analysis indicates that AtANN1, AtANN2, and AtANN6 are all closely related, with AtANN2 and AtANN6 having 75% identity at the amino acid level (<a aria-describedby="B20" class="usa-link" href="#B20">20</a>). There is expression and immunolocalization evidence suggesting that AtANN1 and AtANN2 have both distinct and overlapping functions in <em>Arabidopsis</em> (<a aria-describedby="B21" class="usa-link" href="#B21">21</a>, <a aria-describedby="B22" class="usa-link" href="#B22">22</a>). To investigate if other annexins also participate in plant HS responses, we isolated T-DNA insertion mutants for <em>AtANN2</em> and <em>AtANN6</em> and tested basal thermotolerance of the mutant seedlings (<a class="usa-link" href="#F4">Fig. 4</a><em>A</em>, <a class="usa-link" href="#F4">4</a><em>B</em>). The sensitivity of <em>ann2</em> mutants to heat-shock treatment is higher than wild type but lower than <em>ann1–2</em> mutants (<a class="usa-link" href="#F4">Fig. 4</a><em>C</em>). In contrast, knocking out the expression of <em>AtANN6</em> did not alter the sensitivity of the mutant to heat-shock treatment (<a class="usa-link" href="#F4">Fig. 4</a><em>C</em>). We then generated an <em>ann1–2/ann2</em> double mutant by crossing <em>ann2</em> with <em>ann1–2</em>, and found the <em>ann1–2/ann2</em> double mutant seedlings were more sensitive to heat stress treatment than the <em>ann1–2</em> single mutants. These results indicated <em>AtANN1</em> and <em>AtANN2</em>, but not <em>AtANN6</em>, play redundant roles in regulating plant heat-shock responses.</p>
<figure class="fig xbox font-sm" id="F4"><h6 class="obj_head">Fig. 4.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 4." class="graphic" height="492" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4700/4349987/20c53c7f3e29/zjw0031549870004.jpg" width="566"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong><em>AtANN1</em> and <em>AtANN2</em> play redundant roles in regulating plant heat shock responses.</strong>
<em>A</em>, Diagram shows the T-DNA insertion position for <em>ann2</em> (SALK_054223) and <em>ann6</em> (SALK_112492) T-DNA mutants. <em>B</em>, RT-PCR analysis of <em>AtANN2</em> and <em>AtANN6</em> expression level in <em>ann2</em> and <em>ann6</em> T-DNA mutants. <em>C</em>, The response of <em>ann1–2</em>, <em>ann2</em>, <em>ann6</em>, and <em>ann1–2/ann2</em> double T-DNA mutant (<em>ann1/2</em>) to heat shock treatment. Seven-day-old seedlings grown on 1/2 MS plates at 22 °C were shifted to a 45 °C water bath for 13 min and then returned to 22 °C and allowed to recover for 1 week before taking photos. Scale bar represents 1 cm.</p></figcaption></figure></section><section id="sec19"><h5 class="pmc_sec_title">Heat Stress-Induced Expression of HSF and HSP is Impaired in ann1/2 Mutant</h5>
<p>Heat stress-induced expression of HSFs and HSPs plays an important role in the acquisition of thermotolerance by plants. To investigate whether the susceptibility of annexin mutants to heat stress is related to the level of endogenous HSFs and HSPs, the heat-induced expression of different HSFs and HSPs were analyzed in seedlings of wild type and <em>ann1–2/ann2</em> double mutants. Under normal conditions, expression of HSFs is either not detected or at a similar level in both wild type and <em>ann1–2/ann2</em> double mutants. Thirty minutes of heat shock induces a dramatic increase of HSFs expression in wild-type seedlings. In comparison, the increase in expression of HSFs in <em>ann1–2/ann2</em> double mutants is much smaller (<a class="usa-link" href="#F5">Fig. 5</a><em>A</em>). Similar results were observed for the expression of HSPs. Before heat-shock treatment, the protein levels of the HSP70, HSP90, and HSP17.7 in <em>ann1–2/ann2</em> double mutants were similar to wild type. After heat shock at 37 °C for 3 h, the protein abundance of these three proteins in wild-type seedlings was significantly higher than in those of the <em>ann1–2/ann2</em> mutants (<a class="usa-link" href="#F5">Fig. 5</a><em>B</em>). The reduced expression of HSFs and HSPs in <em>ann1–2/ann2</em> mutants after heat treatment suggests annexin is required for the early changes in gene expression induced by heat shock.</p>
<figure class="fig xbox font-sm" id="F5"><h6 class="obj_head">Fig. 5.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 5." class="graphic" height="701" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4700/4349987/0f3b46df19ec/zjw0031549870005.jpg" width="566"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>The mRNA and protein expression level of several HSFs and HSPs is reduced in <em>ann1–2/ann2</em> double mutant.</strong>
<em>A</em>, Semi-quantitative RT-PCR analysis of the mRNA levels of <em>HSF-A7A</em>, <em>HSF-A2</em>, <em>HSF-B1</em>, and <em>HSF-B2A</em> in one-week-old wild type (Col-0) and <em>ann1/ann2</em> double mutant (<em>ann1/2</em>) seedlings that were HS treated for the indicated time. The RT-PCR products of UBC are shown as the equal-loading control. <em>B</em>, Immunoblot analysis of the protein levels of HSP 17.7, HSP 25.3, HSP 70, and HSP 90 in wild- type and <em>ann1–2/ann2</em> double mutant. One-week-old seedlings were treated with or without HS for the indicated time, total proteins were separated by SDS-PAGE, blotted onto nitrocellulose membrane and blotted with different HSP specific antibodies. Rubisco large subunit stained by Ponceau S is shown as the equal-loading control.</p></figcaption></figure></section><section id="sec20"><h5 class="pmc_sec_title">Plants Overexpressing AtANN1 are more Resistant to Heat-Shock Treatment</h5>
<p>To further confirm the function of annexin in plants' heat-shock response, we overexpressed <em>AtANN1</em> in <em>ann1–2</em> mutant background. RT-PCR analysis confirmed that the expression level of <em>AtANN1</em> in two transgenic lines, OX-1 and OX-2, were both higher than in wild type. Seedlings overexpressing <em>AtANN1</em> were more resistant to heat-shock treatment than wild-type seedlings. The average survival rate of annexin-overexpressing seedlings was 20% higher than wild-type seedlings, and the difference is statistically significant (<a class="usa-link" href="#F6">Fig. 6</a>). This result demonstrates that the expression level of annexin is directly related to the basal thermotolerance of young <em>Arabidopsis</em> seedlings.</p>
<figure class="fig xbox font-sm" id="F6"><h6 class="obj_head">Fig. 6.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 6." class="graphic" height="338" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4700/4349987/a334bada4c56/zjw0031549870006.jpg" width="566"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>Seedlings overexpressing <em>AtANN1</em> are more resistant to heat shock treatment.</strong>
<em>A</em>, Upper panel: RT-PCR analysis of the expression level of <em>AtANN1</em> in overexpressing plants. Lower panel: basal thermotolerance analysis of <em>AtANN1</em> overexpressing plants. Seven-day-old seedlings grown on 1/2 MS plates at 22 °C were shifted to a 45 °C water bath for 12 min and then returned to 22 °C and allowed to recover for 1 week before taking photos. <em>B</em>, Quantification of survival rate of seedlings shown in <em>A</em>. Data represents the mean value from five independent experiments, with 20 seedlings per experiment. One way ANOVA test was performed. Statistically significant differences are indicated by different lowercase letters (<em>p</em> &lt; 0.05). Error bars represent ±S.D.</p></figcaption></figure></section><section id="sec21"><h5 class="pmc_sec_title">Suppressing the Expression of AtANN1 and AtANN2 Inhibits Heat-Shock-Induced Increase in Intracellular Calcium Levels</h5>
<p>Prior reports indicated that AtANN1 could function to increase the Ca<sup>2+</sup> permeability of plasma membranes during plant stress responses (<a aria-describedby="B23" class="usa-link" href="#B23">23</a>, <a aria-describedby="B24" class="usa-link" href="#B24">24</a>). In order to test whether heat-induced calcium transport is altered in AtANN1 loss-of-function mutants, we crossed the <em>ann1–2</em> and <em>ann1–2/ann2</em> mutants with transgenic plants expressing <em><sup>35</sup>S::Aequorin</em> (<a aria-describedby="B25" class="usa-link" href="#B25">25</a>), F3 segregant lines that were homozygous for <em><sup>35</sup>S::Aequorin</em>/<em>ann1–2</em> and <em><sup>35</sup>S::Aequorin</em>/<em>ann1–2/ann2</em> mutants were chosen and used for monitoring heat shock stimulated changes in [Ca<sup>2+</sup>]<sub>cyt</sub>. RT-PCR showed the expression of <em>AtANN1</em> and <em>AtANN2</em> is not detectable in our <em><sup>35</sup>S::Aequorin</em> expressing <em>ann1–2/ann2</em> double knockout mutant (<a class="usa-link" href="#F7">Fig. 7</a><em>A</em>). Measurement of cytosolic calcium concentration showed the highest luminescence signal emitted by reconstituted aequorin is lower in <em><sup>35</sup>S::Aequorin</em> expressing <em>ann1–2</em> and <em>ann1–2/ann2</em> mutants (<a class="usa-link" href="#F7">Fig. 7</a><em>B</em>), which might be caused by the lower total calcium abundance in the seedlings (<a class="usa-link" href="#F7">Fig. 7</a><em>C</em>). Because there is a difference in the reconstituted luminescence signals observed between wild type and the mutants, we used percentage reconstitution values to determine the relative luminescence in response to heat-shock treatment for wild type and mutants seedlings. As shown in <a class="usa-link" href="#F7">Fig. 7</a><em>D</em>, in consistent with previous reports (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>), heat-shock treatment induced a rapid increase in [Ca<sup>2+</sup>]<sub>cyt</sub> within 5 min in wild type, <em>ann1–2</em>, and <em>ann1–2/ann2</em> mutants. The heat induced [Ca<sup>2+</sup>]<sub>cyt</sub> continues to increase in wild type and reaches the highest peak around 12 min. In comparison, this prolonged increase in the heat induced [Ca<sup>2+</sup>]<sub>cyt</sub> was diminished in <em>ann1–2</em> mutant and absent in <em>ann1–2/ann2</em> double mutant (<a class="usa-link" href="#F7">Fig. 7</a><em>D</em>). These results suggest that AtANN1 and AtANN2 function is important for the normal heat-induced increase in [Ca<sup>2+</sup>]<sub>cyt</sub> in <em>Arabidopsis</em>.</p>
<figure class="fig xbox font-sm" id="F7"><h6 class="obj_head">Fig. 7.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 7." class="graphic" height="704" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4700/4349987/1f64699c2b72/zjw0031549870007.jpg" width="680"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F7/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>Heat-induced increase in [Ca<sup>2+</sup>]<sub>cyt</sub> in seedlings is impaired in Annexin mutants.</strong>
<em>A</em>, RT-PCR analysis of <em>AtANN1</em> and <em>AtANN2</em> expression level in wild type (Col) and <em>ann1–2/ann2</em> (<em>ann1/2</em>) double T-DNA mutants. <em>B</em>, <em>In vivo</em> reconstitution of aequorin signal in wild type, <em>ann1</em>, and <em>ann1/2</em> mutants. Error bar represent ±S.E. <em>C</em>, Total calcium abundance in 50 one-week-old seedlings of <em><sup>35</sup>S::Aequorin</em> expressing wild type, <em>ann1</em> and <em>ann1/2</em> mutants. One way ANOVA test was performed in <em>B</em> and <em>C</em>. Statistically significant differences are indicated by different lowercase letters (<em>p</em> &lt; 0.05). Error bars represent ±S.D. <em>D</em>, Time-course measurement of aequorin signal in wild type, <em>ann1</em> and <em>ann1/2</em> mutants. Seedlings were treated with or without heat shock. Relative aequorin signal is calculated by dividing percentage reconstitution value collected at 37 °C by percentage reconstitution value collected at 22 °C. These experiments have been repeated at least three times, and the trends are similar. Data shown here is from one representative experiment.</p></figcaption></figure></section></section></section></section><section id="sec22"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p>The heat-induced rapid increase in [Ca<sup>2+</sup>]<sub>cyt</sub> is considered to be an essential step in plant responses to heat stress. Prior reports showed this increase is regulated by plasma membrane-localized cyclic nucleotide-gated channels (CNGCs) (<a aria-describedby="B8" class="usa-link" href="#B8">8</a>, <a aria-describedby="B9" class="usa-link" href="#B9">9</a>) and phospholipase C9 (PLC9) (<a aria-describedby="B26" class="usa-link" href="#B26">26</a>), but there may be other calcium channels or membrane-localized proteins involved in regulating this rapid and dynamic change in [Ca<sup>2+</sup>]<sub>cyt</sub> in response to HS treatment. Because of prior studies showing that AtANN1 promoted the Ca<sup>2+</sup> permeability of plasma membranes, we selected it for further study from the microsomal proteins in <em>Arabidopsis</em> seedlings whose level changed rapidly after heat shock. We found that AtANN1 plays an important role in regulating heat-shock tolerance in <em>Arabidopsis</em> seedlings and that seedlings knocked out in their expression of <em>AtANN1</em> show impaired calcium influx in response to heat-shock treatment. These results suggest AtANN1 is a new component in regulating heat-shock-induced elevation of intracellular calcium concentration in plant cells.</p>
<p>Annexins bind membranes in a calcium-dependent manner. Although they appear to have diverse functions, it has become clear that certain annexins play important roles in abiotic stress responses (<a aria-describedby="B20" class="usa-link" href="#B20">20</a>, <a aria-describedby="B27" class="usa-link" href="#B27">27</a>). For example, ectopic expression of a <em>Brassica</em> annexin in <em>Arabidopsis</em> provides tolerance to drought, salinity and osmotic stress (<a aria-describedby="B28" class="usa-link" href="#B28">28</a>), and overexpression of <em>AtANN1</em> in <em>Arabidopsis</em> increases both drought tolerance in plants (<a aria-describedby="B29" class="usa-link" href="#B29">29</a>), and salinity tolerance in seeds (<a aria-describedby="B30" class="usa-link" href="#B30">30</a>). There are also previous studies linking annexins to heat-stress responses in plants. For example, a recent proteomic study analyzing heat stress in radish leaves found that the level of an annexin protein increased at 12 h, and then decreased at 24 h in response to 40 °C heat treatment (<a aria-describedby="B31" class="usa-link" href="#B31">31</a>). Chu <em>et al.</em> also identified an annexin protein from the embryonic axes of sacred lotus (<em>Nelumbo nucifera</em> Gaertn.) that was up-regulated by heat treatment. Ectopic expression of this lotus annexin in <em>Arabidopsis</em> conferred resistance to heat treatment in seed germination (<a aria-describedby="B32" class="usa-link" href="#B32">32</a>). In this study, we discovered that <em>Arabidopsis</em> AtANN1 is an early participant in a plant's response to heat treatment. The increase in abundance of AtANN1 in the microsomal membrane fraction can be observed after only 5 min of heat treatment. Among the eight annexins in the <em>Arabidopsis</em> genome, AtANN1 is the most abundant and appears to be a multifunctional protein (<a aria-describedby="B33" class="usa-link" href="#B33">33</a>). We found that knocking out the expression of either endogenous <em>AtANN1</em> or <em>AtANN2</em> makes mutants hypersensitive to heat-shock treatment, whereas knocking out the expression <em>AtANN6</em> does not alter the sensitivity of the mutant, indicating <em>AtANN1</em> and <em>AtANN2</em>, but not <em>AtANN6</em>, are functionally redundant in regulating plant basal thermotolerance. Correspondingly, <em>Arabidopsis</em> seedlings overexpressing <em>AtANN1</em> were more resistant to heat stress. Furthermore, heat induced expression of HSFs and HSPs is attenuated in <em>ann1–2/ann2</em> double mutant. Taken together, these results suggest <em>AtANN1</em> and <em>AtANN2</em> are critical components of plants' adaptive responses to environmental heat stress.</p>
<p>There are numerous reports documenting AtANN1 functions as a calcium-permeable channel-like protein in response to abiotic stress, but most of the data was obtained from patch clamp experiments of spheroplasts and protoplasts of root epidermal cells (<a aria-describedby="B23" class="usa-link" href="#B23">23</a>, <a aria-describedby="B24" class="usa-link" href="#B24">24</a>, <a aria-describedby="B34" class="usa-link" href="#B34">34</a>). Here, using an aequorin reporting system, we found AtANN1 and AtANN2 play a role in regulating the elevation of [Ca<sup>2+</sup>]<sub>cyt</sub> induced by heat shock in whole seedlings. We found that heat-induced change in [Ca<sup>2+</sup>]<sub>cyt</sub> was greatly reduced in <em>ann1</em> mutant and was even lower in <em>ann1–2</em>/<em>ann2</em> double mutant, which suggests that AtANN1 and AtANN2 are important components of the channel transport system that regulates heat-induced increase in [Ca<sup>2+</sup>]<sub>cyt</sub>. However, heat shock still induces a small increase in [Ca<sup>2+</sup>]<sub>cyt</sub> in the <em>ann1–2/ann2</em> double mutant, especially in the first 5 min of heat shock treatment, which might be caused by the function of other proteins that facilitate Ca<sup>2+</sup> uptake, such as CNGCs in <em>Arabidopsis</em>. Interestingly, the total calcium concentration in <em>ann1</em> and <em>ann1/2</em> mutant seedlings is lower than wild-type control seedlings. However, future studies are needed to determine whether the reduced calcium accumulation is caused by the reduced calcium influx in <em>ann1</em> and <em>ann1/2</em> mutants.</p>
<p>In conclusion, we found that the abundance of AtANN1 protein in membranes increased rapidly in response to 5 min heat-shock treatment, and that AtANN1 is critical for plant heat-shock responses. Although it is still unresolved whether AtANN1 acts directly as a calcium-permeable channel <em>in vivo</em> or facilitates this calcium transport by some other mechanism (<a aria-describedby="B35" class="usa-link" href="#B35">35</a>, <a aria-describedby="B36" class="usa-link" href="#B36">36</a>, <a aria-describedby="B37" class="usa-link" href="#B37">37</a>), our results strongly suggest that in addition to previously characterized calcium-permeable channels, annexins are a significant component of the molecular machinery that promotes the heat-induced increase in [Ca<sup>2+</sup>]<sub>cyt</sub>. However, our data provide no evidence that AtANN1 functions as a heat sensor during plant heat-shock responses. Consequently, we propose that an elevated environmental temperature is first perceived by membrane sensors like CNGCs, which allows for an initial calcium influx into the cytosol. This increased [Ca<sup>2+</sup>]<sub>cyt</sub> would activate cytosolic AtANN1, recruiting it directly to membranes. The association of AtANN1 with membranes could then either form a calcium channel by itself or activate another calcium channel to allow more calcium influx into the cell needed to fully activate plant heat-shock responses.</p></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Dr. Xiaodong Xu (Hebei Normal University, China) for providing <em><sup>35</sup>S::Aequorin</em> expressing transgenic plants.</p>
<p>This work is dedicated in loving memory of Xu Wang who sadly passed away unexpectedly during this research study.</p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn2"><p>Author contributions: D.S. and W.T. designed research; X.W., X.M., H.W., and B.L. performed research; G.C., Y.G., and S.R. contributed new reagents or analytic tools; W.T. analyzed data; G.C., S.R., and W.T. wrote the paper.</p></div>
<div class="fn p" id="FN1"><p>* This work was supported by the Ministry of Agriculture of China (2013ZX08009–003-002); “one hundred talents project” of Hebei province (E2012100004 to W.T.); the National Natural Science Foundation of China (31270357 to Y.G.); and by grants from the National Science Foundation (IOS 0718890 to S.J.R. and G.C.) and the National Aeronautics and Space Administration (NNX13AM54G to S.J.R. and G.C.).</p></div>
<div class="fn p" id="G1">
<p><sup>1</sup> The abbreviations used are:</p>
<div class="p"><dl class="def-list">
<dt>CHAPS</dt>
<dd><div class="p">3-Cholamidopropyl dimethylammonio]-1-propanesulfonate</div></dd>
<dt>HS</dt>
<dd><div class="p">heat stress</div></dd>
<dt>HSFs</dt>
<dd><div class="p">heat shock transcription factors</div></dd>
<dt>HSPs</dt>
<dd><div class="p">heat shock proteins</div></dd>
<dt>AtANN1</dt>
<dd><div class="p">
<em>Arabidopsis</em> annexin 1</div></dd>
<dt>CDPKs</dt>
<dd><div class="p">calcium dependent protein kinases</div></dd>
<dt>HSR</dt>
<dd><div class="p">heat-shock responses</div></dd>
<dt>Col-0</dt>
<dd><div class="p">Columbia-0</div></dd>
<dt>CNGC</dt>
<dd><div class="p">cyclic nucleotide-gated channels</div></dd>
<dt>MS medium</dt>
<dd><div class="p">Murashige and Skoog medium</div></dd>
<dt>[Ca<sup>2+</sup>]<sub>cyt</sub>
</dt>
<dd><div class="p">cytoplasmic calcium concentration</div></dd>
<dt>PLC</dt>
<dd><div class="p">phospholipase C</div></dd>
<dt>ROS</dt>
<dd><div class="p">Reactive oxygen species.</div></dd>
</dl></div>
</div>
</div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>
Baniwal S. K., Bharti K., Chan K. Y., Fauth M., Ganguli A., Kotak S., Mishra S. K., Nover L., Port M., Scharf K. D., Tripp J., Weber C., Zielinski D., Con Koskull-Doring P. (2004) Heat stress response in plants, a complex game with chaperones and more than twenty heat stress transcription factors. J. Biosci. 29, 471–487</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/BF02712120" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15625403/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biosci&amp;title=Heat%20stress%20response%20in%20plants,%20a%20complex%20game%20with%20chaperones%20and%20more%20than%20twenty%20heat%20stress%20transcription%20factors&amp;author=S.%20K.%20Baniwal&amp;author=K.%20Bharti&amp;author=K.%20Y.%20Chan&amp;author=M.%20Fauth&amp;author=A.%20Ganguli&amp;volume=29&amp;publication_year=2004&amp;pages=471-487&amp;pmid=15625403&amp;doi=10.1007/BF02712120&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>
Nover L., Bharti K., Doring P., Mishra S. K., Ganguli A., Scharf K. D. (2001) Arabidopsis and the heat stress transcription factor world, how many heat stress transcription factors do we need?
Cell Stress Chaperones
6, 177–189</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1379/1466-1268(2001)006&lt;0177:aathst&gt;2.0.co;2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC434399/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11599559/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell%20Stress%20Chaperones&amp;title=Arabidopsis%20and%20the%20heat%20stress%20transcription%20factor%20world,%20how%20many%20heat%20stress%20transcription%20factors%20do%20we%20need?&amp;author=L.%20Nover&amp;author=K.%20Bharti&amp;author=P.%20Doring&amp;author=S.%20K.%20Mishra&amp;author=A.%20Ganguli&amp;volume=6&amp;publication_year=2001&amp;pages=177-189&amp;pmid=11599559&amp;doi=10.1379/1466-1268(2001)006&lt;0177:aathst&gt;2.0.co;2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>
Pirkkala L., Nykanen P., Sistonen L. (2001) Roles of the heat shock transcription factors in regulation of the heat shock response and beyond. FASEB J. 15, 1118–1131</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1096/fj00-0294rev" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11344080/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FASEB%20J&amp;title=Roles%20of%20the%20heat%20shock%20transcription%20factors%20in%20regulation%20of%20the%20heat%20shock%20response%20and%20beyond&amp;author=L.%20Pirkkala&amp;author=P.%20Nykanen&amp;author=L.%20Sistonen&amp;volume=15&amp;publication_year=2001&amp;pages=1118-1131&amp;pmid=11344080&amp;doi=10.1096/fj00-0294rev&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>
Deng Y., Humbert S., Liu J. X., Srivastava R., Rothstein S., Howell S. H. (2011) Heat induces the splicing by IRE1 of a mRNA encoding a transcription factor involved in the unfolded protein response in Arabidopsis. Proc. Natl. Acad. Sci. U.S.A. 108, 7247–7252</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1102117108" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3084119/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21482766/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;title=Heat%20induces%20the%20splicing%20by%20IRE1%20of%20a%20mRNA%20encoding%20a%20transcription%20factor%20involved%20in%20the%20unfolded%20protein%20response%20in%20Arabidopsis&amp;author=Y.%20Deng&amp;author=S.%20Humbert&amp;author=J.%20X.%20Liu&amp;author=R.%20Srivastava&amp;author=S.%20Rothstein&amp;volume=108&amp;publication_year=2011&amp;pages=7247-7252&amp;pmid=21482766&amp;doi=10.1073/pnas.1102117108&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>
Zhong S. H., Liu J. Z., Jin H., Lin L., Li Q., Chen Y., Yuan Y. Z., Wang Z.-Y., Huang H., Qi Y. J., Chen X. Y., Vaucheret H., Chory J., Li J. M., He Z. H. (2013) Warm temperatures induce transgenerational epigenetic release of RNA silencing by inhibiting siRNA biogenesis in Arabidopsis. Proc. Natl. Acad. Sci. U.S.A. 110, 9171–9176</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1219655110" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3670308/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23686579/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;title=Warm%20temperatures%20induce%20transgenerational%20epigenetic%20release%20of%20RNA%20silencing%20by%20inhibiting%20siRNA%20biogenesis%20in%20Arabidopsis&amp;author=S.%20H.%20Zhong&amp;author=J.%20Z.%20Liu&amp;author=H.%20Jin&amp;author=L.%20Lin&amp;author=Q.%20Li&amp;volume=110&amp;publication_year=2013&amp;pages=9171-9176&amp;pmid=23686579&amp;doi=10.1073/pnas.1219655110&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>
Mittler R., Finka A., Goloubinoff P. (2012) How do plants feel the heat?
Trends Biochem. Sci. 37, 118–125</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.tibs.2011.11.007" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22236506/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem.%20Sci&amp;title=How%20do%20plants%20feel%20the%20heat?&amp;author=R.%20Mittler&amp;author=A.%20Finka&amp;author=P.%20Goloubinoff&amp;volume=37&amp;publication_year=2012&amp;pages=118-125&amp;pmid=22236506&amp;doi=10.1016/j.tibs.2011.11.007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>
Kumar S. V., Wgge P. A. (2010) H2A.Z containing nucleosomes mediate the thermosensory response in Arabidopsis. Cell
140, 136–147</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cell.2009.11.006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20079334/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=H2A.Z%20containing%20nucleosomes%20mediate%20the%20thermosensory%20response%20in%20Arabidopsis&amp;author=S.%20V.%20Kumar&amp;author=P.%20A.%20Wgge&amp;volume=140&amp;publication_year=2010&amp;pages=136-147&amp;pmid=20079334&amp;doi=10.1016/j.cell.2009.11.006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>
Finka A., Cuendet A. F. H., Maathuis F. J. M., Saidi Y., Goloubinoff P. (2012) Plasma membrane cyclic nucleotide gated calcium channels control land plant thermal sensing and acquired thermotolerance. Plant Cell
24, 3333–3348</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.112.095844" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3462635/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22904147/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Plasma%20membrane%20cyclic%20nucleotide%20gated%20calcium%20channels%20control%20land%20plant%20thermal%20sensing%20and%20acquired%20thermotolerance&amp;author=A.%20Finka&amp;author=A.%20F.%20H.%20Cuendet&amp;author=F.%20J.%20M.%20Maathuis&amp;author=Y.%20Saidi&amp;author=P.%20Goloubinoff&amp;volume=24&amp;publication_year=2012&amp;pages=3333-3348&amp;pmid=22904147&amp;doi=10.1105/tpc.112.095844&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>
Gao F., Han X. W., Wu J. H., Zheng S. Z., Shang Z. L., Sun D. Y., Zhou R. G., Li B. (2012) A heat activated calcium permeable channel, Arabidopsis cyclic nucleotide-gated ion channel 6, is involved in heat shock responses. Plant J. 70, 1056–1069</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2012.04969.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22372427/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=A%20heat%20activated%20calcium%20permeable%20channel,%20Arabidopsis%20cyclic%20nucleotide-gated%20ion%20channel%206,%20is%20involved%20in%20heat%20shock%20responses&amp;author=F.%20Gao&amp;author=X.%20W.%20Han&amp;author=J.%20H.%20Wu&amp;author=S.%20Z.%20Zheng&amp;author=Z.%20L.%20Shang&amp;volume=70&amp;publication_year=2012&amp;pages=1056-1069&amp;pmid=22372427&amp;doi=10.1111/j.1365-313X.2012.04969.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>
Saidi Y., Finka A., Goloubinoff P. (2010) Heat perception and signaling in plants: a tortuous path to thermotolerance. New Phytol. 190, 556–565</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1469-8137.2010.03571.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21138439/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol&amp;title=Heat%20perception%20and%20signaling%20in%20plants:%20a%20tortuous%20path%20to%20thermotolerance&amp;author=Y.%20Saidi&amp;author=A.%20Finka&amp;author=P.%20Goloubinoff&amp;volume=190&amp;publication_year=2010&amp;pages=556-565&amp;pmid=21138439&amp;doi=10.1111/j.1469-8137.2010.03571.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>
Liu H. T., Li B., Shang Z. L., Li X. Z., Mu R. L., Sun D. Y., Zhou R. G. (2003) Calmodulin is involved in heat shock signal transduction in wheat. Plant Physiol. 132, 1186–1195</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.102.018564" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC167059/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12857801/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Calmodulin%20is%20involved%20in%20heat%20shock%20signal%20transduction%20in%20wheat&amp;author=H.%20T.%20Liu&amp;author=B.%20Li&amp;author=Z.%20L.%20Shang&amp;author=X.%20Z.%20Li&amp;author=R.%20L.%20Mu&amp;volume=132&amp;publication_year=2003&amp;pages=1186-1195&amp;pmid=12857801&amp;doi=10.1104/pp.102.018564&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>
Liu H. T., Gao F., Li G. L., Han J. L., Liu D. L., Sun D. Y., Zhou R. G. (2008) The calmodulin-binding protein kinase 3 is part of heat shock signal transduction in Arabidopsis thaliana. Plant J. 55, 760–773</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2008.03544.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18466301/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=The%20calmodulin-binding%20protein%20kinase%203%20is%20part%20of%20heat%20shock%20signal%20transduction%20in%20Arabidopsis%20thaliana&amp;author=H.%20T.%20Liu&amp;author=F.%20Gao&amp;author=G.%20L.%20Li&amp;author=J.%20L.%20Han&amp;author=D.%20L.%20Liu&amp;volume=55&amp;publication_year=2008&amp;pages=760-773&amp;pmid=18466301&amp;doi=10.1111/j.1365-313X.2008.03544.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>
Zhang W., Zhou R. G., Gao Y. J., Zheng S. Z., Xu P., Zhang S. Q., Sun D. Y. (2009) Molecular and Genetic evidence for the key role of AtCaM3 in heat-shock signal transduction in Arabidopsis. Plant Physiol. 149, 1773–1784</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.108.133744" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2663753/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19211698/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Molecular%20and%20Genetic%20evidence%20for%20the%20key%20role%20of%20AtCaM3%20in%20heat-shock%20signal%20transduction%20in%20Arabidopsis&amp;author=W.%20Zhang&amp;author=R.%20G.%20Zhou&amp;author=Y.%20J.%20Gao&amp;author=S.%20Z.%20Zheng&amp;author=P.%20Xu&amp;volume=149&amp;publication_year=2009&amp;pages=1773-1784&amp;pmid=19211698&amp;doi=10.1104/pp.108.133744&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14">
<span class="label">14.</span><cite>
Sangwan V., Orvar B. L., Beyerly J., Hirt H., Dhindsa R. S. (2002) Opposite changes in membrane fluidity mimic cold and heat stress activation of distinct plant MAP kinase pathways. Plant J. 31, 629–638</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1365-313x.2002.01384.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12207652/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Opposite%20changes%20in%20membrane%20fluidity%20mimic%20cold%20and%20heat%20stress%20activation%20of%20distinct%20plant%20MAP%20kinase%20pathways&amp;author=V.%20Sangwan&amp;author=B.%20L.%20Orvar&amp;author=J.%20Beyerly&amp;author=H.%20Hirt&amp;author=R.%20S.%20Dhindsa&amp;volume=31&amp;publication_year=2002&amp;pages=629-638&amp;pmid=12207652&amp;doi=10.1046/j.1365-313x.2002.01384.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15">
<span class="label">15.</span><cite>
Tang W. Q., Deng Z. P., Wang Z. Y. (2010) Protomics shed light on the brassinosteroid signaling mechanisms. Curr. Opin. Plant Biol. 13, 27–33</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.pbi.2009.10.007" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2818672/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20004136/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Plant%20Biol&amp;title=Protomics%20shed%20light%20on%20the%20brassinosteroid%20signaling%20mechanisms&amp;author=W.%20Q.%20Tang&amp;author=Z.%20P.%20Deng&amp;author=Z.%20Y.%20Wang&amp;volume=13&amp;publication_year=2010&amp;pages=27-33&amp;pmid=20004136&amp;doi=10.1016/j.pbi.2009.10.007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16">
<span class="label">16.</span><cite>
Tang W. Q. (2012) Quantitative analysis of plasma membrane proteome using two-dimensional difference gel electrophoresis. Methods Mol. Biol. 876, 67–82</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-1-61779-809-2_5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22576086/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol&amp;title=Quantitative%20analysis%20of%20plasma%20membrane%20proteome%20using%20two-dimensional%20difference%20gel%20electrophoresis&amp;author=W.%20Q.%20Tang&amp;volume=876&amp;publication_year=2012&amp;pages=67-82&amp;pmid=22576086&amp;doi=10.1007/978-1-61779-809-2_5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17">
<span class="label">17.</span><cite>
Tang W. Q., Deng Z. P., Oses-Prieto J., Suzuki N., Zhu S. W., Zhang X., Burlingame A. L., Wang Z. Y. (2008) Proteomics studies of brassinosteroid signal transduction using prefractionation and two dimensional DIGE. Mol. Cell. Proteomics
7, 728–738</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/mcp.M700358-MCP200" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2401332/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18182375/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Proteomics%20studies%20of%20brassinosteroid%20signal%20transduction%20using%20prefractionation%20and%20two%20dimensional%20DIGE&amp;author=W.%20Q.%20Tang&amp;author=Z.%20P.%20Deng&amp;author=J.%20Oses-Prieto&amp;author=N.%20Suzuki&amp;author=S.%20W.%20Zhu&amp;volume=7&amp;publication_year=2008&amp;pages=728-738&amp;pmid=18182375&amp;doi=10.1074/mcp.M700358-MCP200&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18">
<span class="label">18.</span><cite>
Knight H., Trewavas A. J., Knight M. R. (1996) Cold calcium signaling in Arabidopsis involves two cellular pools and a change in calcium signature after acclimation. Plant Cell
8, 489–503</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.8.3.489" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC161115/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/8721751/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Cold%20calcium%20signaling%20in%20Arabidopsis%20involves%20two%20cellular%20pools%20and%20a%20change%20in%20calcium%20signature%20after%20acclimation&amp;author=H.%20Knight&amp;author=A.%20J.%20Trewavas&amp;author=M.%20R.%20Knight&amp;volume=8&amp;publication_year=1996&amp;pages=489-503&amp;pmid=8721751&amp;doi=10.1105/tpc.8.3.489&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19">
<span class="label">19.</span><cite>
Clark G. B., Lee D., Dauwalder M., Roux S. J. (2005) Immunolocalization and histochemical evidence for the association of two different Arabidopsis annexins with secretion during early seedling growth and development. Planta
220, 621–631</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00425-004-1374-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15368128/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Immunolocalization%20and%20histochemical%20evidence%20for%20the%20association%20of%20two%20different%20Arabidopsis%20annexins%20with%20secretion%20during%20early%20seedling%20growth%20and%20development&amp;author=G.%20B.%20Clark&amp;author=D.%20Lee&amp;author=M.%20Dauwalder&amp;author=S.%20J.%20Roux&amp;volume=220&amp;publication_year=2005&amp;pages=621-631&amp;pmid=15368128&amp;doi=10.1007/s00425-004-1374-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B20">
<span class="label">20.</span><cite>
Clark G. B., Morgan R. O., Fernandez M.-P., Roux S. J. (2012) Evolutionary adaptation of plant annexins has diversified their molecular structures, interactions, and functional roles. New Phytol. 196, 695–712</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1469-8137.2012.04308.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22994944/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol&amp;title=Evolutionary%20adaptation%20of%20plant%20annexins%20has%20diversified%20their%20molecular%20structures,%20interactions,%20and%20functional%20roles&amp;author=G.%20B.%20Clark&amp;author=R.%20O.%20Morgan&amp;author=M.-P.%20Fernandez&amp;author=S.%20J.%20Roux&amp;volume=196&amp;publication_year=2012&amp;pages=695-712&amp;pmid=22994944&amp;doi=10.1111/j.1469-8137.2012.04308.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21">
<span class="label">21.</span><cite>
Clark G. B., Sessions A., Eastburn D. J., Roux S. J. (2001) Differential expression of members of the annexin multigene family in Arabidopsis. Plant Physiol. 126, 1072–1084</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.126.3.1072" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC116464/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11457958/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Differential%20expression%20of%20members%20of%20the%20annexin%20multigene%20family%20in%20Arabidopsis&amp;author=G.%20B.%20Clark&amp;author=A.%20Sessions&amp;author=D.%20J.%20Eastburn&amp;author=S.%20J.%20Roux&amp;volume=126&amp;publication_year=2001&amp;pages=1072-1084&amp;pmid=11457958&amp;doi=10.1104/pp.126.3.1072&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22">
<span class="label">22.</span><cite>
Clark G. B., Lee D., Dauwalder M., Roux S. J. (2005) Immunolocalization and histochemical evidence for the association of two different <em>Arabidopsis</em> annexins with secretion during early seedling growth and development. Planta
220, 621–631</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00425-004-1374-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15368128/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Immunolocalization%20and%20histochemical%20evidence%20for%20the%20association%20of%20two%20different%20Arabidopsis%20annexins%20with%20secretion%20during%20early%20seedling%20growth%20and%20development&amp;author=G.%20B.%20Clark&amp;author=D.%20Lee&amp;author=M.%20Dauwalder&amp;author=S.%20J.%20Roux&amp;volume=220&amp;publication_year=2005&amp;pages=621-631&amp;pmid=15368128&amp;doi=10.1007/s00425-004-1374-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23">
<span class="label">23.</span><cite>
Laohavisit A., Richards S. L., Shabala L., Chen C., Colaco R. D. D. R., Swarbreck S. M., Shaw E., Dark A., Shabala S., Shang Z., Davies J. M. (2013) Salinity-induced calcium signaling and root adaptation in Arabidopsis require the calcium regulatory protein Annexin1. Plant Physiol. 163, 253–262</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.113.217810" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3762646/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23886625/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Salinity-induced%20calcium%20signaling%20and%20root%20adaptation%20in%20Arabidopsis%20require%20the%20calcium%20regulatory%20protein%20Annexin1&amp;author=A.%20Laohavisit&amp;author=S.%20L.%20Richards&amp;author=L.%20Shabala&amp;author=C.%20Chen&amp;author=R.%20D.%20D.%20R.%20Colaco&amp;volume=163&amp;publication_year=2013&amp;pages=253-262&amp;pmid=23886625&amp;doi=10.1104/pp.113.217810&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24">
<span class="label">24.</span><cite>
Richards S. L., Laohavisit A., Mortimer J. C., Shabala L., Swarbreck S. M., Shabala S., Davies J. M. (2014) Annexin 1 regulates the H<sub>2</sub>O<sub>2</sub>-induced calcium signature in Arabidopsis thaliana roots. Plant J. 77, 136–145</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/tpj.12372" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24180429/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Annexin%201%20regulates%20the%20H2O2-induced%20calcium%20signature%20in%20Arabidopsis%20thaliana%20roots&amp;author=S.%20L.%20Richards&amp;author=A.%20Laohavisit&amp;author=J.%20C.%20Mortimer&amp;author=L.%20Shabala&amp;author=S.%20M.%20Swarbreck&amp;volume=77&amp;publication_year=2014&amp;pages=136-145&amp;pmid=24180429&amp;doi=10.1111/tpj.12372&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25">
<span class="label">25.</span><cite>
Xu X. D., Hotta C. T., Dodd A. N., Love J., Sharrock R., Lee Y. W., Xie Q., Johnson C. H., Wevv A. A. (2007) Distinct light and clock modulation of cytosolic free Ca<sup>2+</sup> oscillations and rhythmic CHLOROPHYLL A/B BINDING PROTEIN2 promoter activity in Arabidopsis. Plant Cell
19, 3474–3490</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.106.046011" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2174886/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17982000/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Distinct%20light%20and%20clock%20modulation%20of%20cytosolic%20free%20Ca2+%20oscillations%20and%20rhythmic%20CHLOROPHYLL%20A/B%20BINDING%20PROTEIN2%20promoter%20activity%20in%20Arabidopsis&amp;author=X.%20D.%20Xu&amp;author=C.%20T.%20Hotta&amp;author=A.%20N.%20Dodd&amp;author=J.%20Love&amp;author=R.%20Sharrock&amp;volume=19&amp;publication_year=2007&amp;pages=3474-3490&amp;pmid=17982000&amp;doi=10.1105/tpc.106.046011&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26">
<span class="label">26.</span><cite>
Zheng S. Z., Liu Y. L., Li B., Shang Z. L., Zhou R. G., Sun D. Y. (2012) Phosphoinositide-specific phospholipase C9 is involved in the thermotolerance of Arabidopsis. Plant J. 69, 689–700</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1365-313X.2011.04823.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22007900/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20J&amp;title=Phosphoinositide-specific%20phospholipase%20C9%20is%20involved%20in%20the%20thermotolerance%20of%20Arabidopsis&amp;author=S.%20Z.%20Zheng&amp;author=Y.%20L.%20Liu&amp;author=B.%20Li&amp;author=Z.%20L.%20Shang&amp;author=R.%20G.%20Zhou&amp;volume=69&amp;publication_year=2012&amp;pages=689-700&amp;pmid=22007900&amp;doi=10.1111/j.1365-313X.2011.04823.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27">
<span class="label">27.</span><cite>
Laohavisit A., Davies J. M. (2011) Annexins. New Phytol. 189, 40–53</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1469-8137.2010.03533.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21083562/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol&amp;title=Annexins&amp;author=A.%20Laohavisit&amp;author=J.%20M.%20Davies&amp;volume=189&amp;publication_year=2011&amp;pages=40-53&amp;pmid=21083562&amp;doi=10.1111/j.1469-8137.2010.03533.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28">
<span class="label">28.</span><cite>
Jami S. K., Clark G. B., Turlapati S. A., Handley C. A., Roux S. J., Kirti P. B. (2008) Ectopic expression of an annexin from Brassica juncea confers tolerance to abiotic and biotic stress treatments in transgenic tobacco. Plant Physiol. Biochem. 46, 1019–1030</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.plaphy.2008.07.006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18768323/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.%20Biochem&amp;title=Ectopic%20expression%20of%20an%20annexin%20from%20Brassica%20juncea%20confers%20tolerance%20to%20abiotic%20and%20biotic%20stress%20treatments%20in%20transgenic%20tobacco&amp;author=S.%20K.%20Jami&amp;author=G.%20B.%20Clark&amp;author=S.%20A.%20Turlapati&amp;author=C.%20A.%20Handley&amp;author=S.%20J.%20Roux&amp;volume=46&amp;publication_year=2008&amp;pages=1019-1030&amp;pmid=18768323&amp;doi=10.1016/j.plaphy.2008.07.006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29">
<span class="label">29.</span><cite>
Konopka-Postupolska D., Clark G., Goch G., Dębski J., Floras K., Cantero A., Fijolek B., Roux S., Hennig J. (2009) The role of annexin 1 in drought stress in Arabidopsis. Plant Physiol. 150, 1394–1410</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.109.135228" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2705051/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19482919/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=The%20role%20of%20annexin%201%20in%20drought%20stress%20in%20Arabidopsis&amp;author=D.%20Konopka-Postupolska&amp;author=G.%20Clark&amp;author=G.%20Goch&amp;author=J.%20D%C4%99bski&amp;author=K.%20Floras&amp;volume=150&amp;publication_year=2009&amp;pages=1394-1410&amp;pmid=19482919&amp;doi=10.1104/pp.109.135228&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30">
<span class="label">30.</span><cite>
Lee S., Lee E. J., Yang E. J., Lee J. E., Park A. R., Song W. H., Park O. K. (2004) Proteomic identification of annexins, calcium-dependent membrane binding proteins that mediate osmotic stress and abscisic acid signal transduction in Arabidopsis. Plant Cell
16, 1378–1391</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.021683" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC490033/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15161963/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Proteomic%20identification%20of%20annexins,%20calcium-dependent%20membrane%20binding%20proteins%20that%20mediate%20osmotic%20stress%20and%20abscisic%20acid%20signal%20transduction%20in%20Arabidopsis&amp;author=S.%20Lee&amp;author=E.%20J.%20Lee&amp;author=E.%20J.%20Yang&amp;author=J.%20E.%20Lee&amp;author=A.%20R.%20Park&amp;volume=16&amp;publication_year=2004&amp;pages=1378-1391&amp;pmid=15161963&amp;doi=10.1105/tpc.021683&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31">
<span class="label">31.</span><cite>
Zhang Y., Xu L., Zhu X., Gong Y., Xiang F., Sun X., Liu L. (2013) Proteomic analysis of heat stress response in leaves of radish (Raphanus sativus l.). Plant Mol. Biol. Rep. 31, 195–203</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Mol.%20Biol.%20Rep&amp;title=Proteomic%20analysis%20of%20heat%20stress%20response%20in%20leaves%20of%20radish%20(Raphanus%20sativus%20l.)&amp;author=Y.%20Zhang&amp;author=L.%20Xu&amp;author=X.%20Zhu&amp;author=Y.%20Gong&amp;author=F.%20Xiang&amp;volume=31&amp;publication_year=2013&amp;pages=195-203&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32">
<span class="label">32.</span><cite>
Chu P., Chen H., Zhou Y., Li Y., Ding Y., Jiang L., Tsang E. W. T., Wu K., Huang S. (2012) Proteomic and functional analyses of Nelumbo nucifera annexins involved in seed thermotolerance and germination vigor. Planta
235, 1271–1288</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00425-011-1573-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22167260/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Proteomic%20and%20functional%20analyses%20of%20Nelumbo%20nucifera%20annexins%20involved%20in%20seed%20thermotolerance%20and%20germination%20vigor&amp;author=P.%20Chu&amp;author=H.%20Chen&amp;author=Y.%20Zhou&amp;author=Y.%20Li&amp;author=Y.%20Ding&amp;volume=235&amp;publication_year=2012&amp;pages=1271-1288&amp;pmid=22167260&amp;doi=10.1007/s00425-011-1573-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B33">
<span class="label">33.</span><cite>
Clark G., Konopka-Postupolska D., Hennig J., Roux S. (2010) Is annexin 1 a multifunctional protein during stress responses?
Plant Signal. Behav. 5, 303–307</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4161/psb.5.3.10835" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2881285/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20215861/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signal.%20Behav&amp;title=Is%20annexin%201%20a%20multifunctional%20protein%20during%20stress%20responses?&amp;author=G.%20Clark&amp;author=D.%20Konopka-Postupolska&amp;author=J.%20Hennig&amp;author=S.%20Roux&amp;volume=5&amp;publication_year=2010&amp;pages=303-307&amp;pmid=20215861&amp;doi=10.4161/psb.5.3.10835&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34">
<span class="label">34.</span><cite>
Laohavisit A., Shang Z., Rubio L., Cuin T. A., Véry A. A., Wang A., Mortimer J. C., Macpherson N., Coxon K. M., Battey N. H. (2012) Arabidopsis annexin1 mediates the radical-activated plasma membrane Ca<sup>2+</sup>- and K<sup>+</sup>-permeable conductance in root cells. Plant Cell
24, 1522–1533</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.112.097881" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3398561/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22523205/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Arabidopsis%20annexin1%20mediates%20the%20radical-activated%20plasma%20membrane%20Ca2+-%20and%20K+-permeable%20conductance%20in%20root%20cells&amp;author=A.%20Laohavisit&amp;author=Z.%20Shang&amp;author=L.%20Rubio&amp;author=T.%20A.%20Cuin&amp;author=A.%20A.%20V%C3%A9ry&amp;volume=24&amp;publication_year=2012&amp;pages=1522-1533&amp;pmid=22523205&amp;doi=10.1105/tpc.112.097881&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35">
<span class="label">35.</span><cite>
Konopka-Postupolska D., Clark G., Hofmann A. (2011) Structure, function, and membrane interactions of plant annexins: an update. Plant Sci. 181, 230–241</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.plantsci.2011.05.013" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21763533/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Sci&amp;title=Structure,%20function,%20and%20membrane%20interactions%20of%20plant%20annexins:%20an%20update&amp;author=D.%20Konopka-Postupolska&amp;author=G.%20Clark&amp;author=A.%20Hofmann&amp;volume=181&amp;publication_year=2011&amp;pages=230-241&amp;pmid=21763533&amp;doi=10.1016/j.plantsci.2011.05.013&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36">
<span class="label">36.</span><cite>
Swarbreck S. M., Colaco R., Davies J. M. (2013) Plant calcium-permeable channels. Plant Physiol. 163, 514–522</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1104/pp.113.220855" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3793033/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23860348/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&amp;title=Plant%20calcium-permeable%20channels&amp;author=S.%20M.%20Swarbreck&amp;author=R.%20Colaco&amp;author=J.%20M.%20Davies&amp;volume=163&amp;publication_year=2013&amp;pages=514-522&amp;pmid=23860348&amp;doi=10.1104/pp.113.220855&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B37">
<span class="label">37.</span><cite>
Davies J. M. (2014) Annexin-mediated calcium signaling in plants. Plants
3, 128–140</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/plants3010128" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4844307/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27135495/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Annexin-mediated%20calcium%20signaling%20in%20plants&amp;author=J.%20M.%20Davies&amp;volume=3&amp;publication_year=2014&amp;pages=128-140&amp;pmid=27135495&amp;doi=10.3390/plants3010128&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section></section></section></article>
</main></body></html>