<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5552977/"/><title>Draft Genome Sequences from a Novel Clade of Bacillus cereus Sensu Lato Strains, Isolated from the International Space Station - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Genome Announc</button></div>. 2017 Aug 10;5(32):e00680-17. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/genomeA.00680-17" rel="noopener noreferrer" target="_blank">10.1128/genomeA.00680-17</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Draft Genome Sequences from a Novel Clade of <em>Bacillus cereus Sensu Lato</em> Strains, Isolated from the International Space Station</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Kasthuri Venkateswaran</span></h3>
<div class="p">
<sup>a</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a>
</div>
</div>
<sup>a,</sup><sup>✉</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Checinska%20Sielaff%20A%22%5BAuthor%5D"><span class="name western">Aleksandra Checinska Sielaff</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Aleksandra Checinska Sielaff</span></h3>
<div class="p">
<sup>a</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Checinska%20Sielaff%20A%22%5BAuthor%5D"><span class="name western">Aleksandra Checinska Sielaff</span></a>
</div>
</div>
<sup>a</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ratnayake%20S%22%5BAuthor%5D"><span class="name western">Shashikala Ratnayake</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Shashikala Ratnayake</span></h3>
<div class="p">
<sup>b</sup>National Biodefense Analysis and Countermeasures Center, Ft. Detrick, Maryland, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ratnayake%20S%22%5BAuthor%5D"><span class="name western">Shashikala Ratnayake</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pope%20RK%22%5BAuthor%5D"><span class="name western">Robert K Pope</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Robert K Pope</span></h3>
<div class="p">
<sup>b</sup>National Biodefense Analysis and Countermeasures Center, Ft. Detrick, Maryland, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pope%20RK%22%5BAuthor%5D"><span class="name western">Robert K Pope</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blank%20TE%22%5BAuthor%5D"><span class="name western">Thomas E Blank</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Thomas E Blank</span></h3>
<div class="p">
<sup>b</sup>National Biodefense Analysis and Countermeasures Center, Ft. Detrick, Maryland, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blank%20TE%22%5BAuthor%5D"><span class="name western">Thomas E Blank</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stepanov%20VG%22%5BAuthor%5D"><span class="name western">Victor G Stepanov</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Victor G Stepanov</span></h3>
<div class="p">
<sup>c</sup>Department of Biology and Biochemistry, University of Houston, Houston, Texas, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stepanov%20VG%22%5BAuthor%5D"><span class="name western">Victor G Stepanov</span></a>
</div>
</div>
<sup>c</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fox%20GE%22%5BAuthor%5D"><span class="name western">George E Fox</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">George E Fox</span></h3>
<div class="p">
<sup>c</sup>Department of Biology and Biochemistry, University of Houston, Houston, Texas, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fox%20GE%22%5BAuthor%5D"><span class="name western">George E Fox</span></a>
</div>
</div>
<sup>c</sup>, <a aria-describedby="id8" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20Tongeren%20SP%22%5BAuthor%5D"><span class="name western">Sandra P van Tongeren</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Sandra P van Tongeren</span></h3>
<div class="p">
<sup>d</sup>Department of Medical Microbiology, University of Groningen, University Medical Center Groningen, Groningen, The Netherlands</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22van%20Tongeren%20SP%22%5BAuthor%5D"><span class="name western">Sandra P van Tongeren</span></a>
</div>
</div>
<sup>d</sup>, <a aria-describedby="id9" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Torres%20C%22%5BAuthor%5D"><span class="name western">Clinton Torres</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Clinton Torres</span></h3>
<div class="p">
<sup>e</sup>Lawrence Livermore National Laboratory, Berkeley, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Torres%20C%22%5BAuthor%5D"><span class="name western">Clinton Torres</span></a>
</div>
</div>
<sup>e</sup>, <a aria-describedby="id10" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Allen%20J%22%5BAuthor%5D"><span class="name western">Jonathan Allen</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Jonathan Allen</span></h3>
<div class="p">
<sup>e</sup>Lawrence Livermore National Laboratory, Berkeley, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Allen%20J%22%5BAuthor%5D"><span class="name western">Jonathan Allen</span></a>
</div>
</div>
<sup>e</sup>, <a aria-describedby="id11" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaing%20C%22%5BAuthor%5D"><span class="name western">Crystal Jaing</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Crystal Jaing</span></h3>
<div class="p">
<sup>e</sup>Lawrence Livermore National Laboratory, Berkeley, California, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaing%20C%22%5BAuthor%5D"><span class="name western">Crystal Jaing</span></a>
</div>
</div>
<sup>e</sup>, <a aria-describedby="id12" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pierson%20D%22%5BAuthor%5D"><span class="name western">Duane Pierson</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Duane Pierson</span></h3>
<div class="p">
<sup>f</sup>Johnson Space Center, Houston, Texas, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pierson%20D%22%5BAuthor%5D"><span class="name western">Duane Pierson</span></a>
</div>
</div>
<sup>f</sup>, <a aria-describedby="id13" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Perry%20J%22%5BAuthor%5D"><span class="name western">Jay Perry</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Jay Perry</span></h3>
<div class="p">
<sup>g</sup>Marshall Space Flight Center, Huntsville, Alabama, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Perry%20J%22%5BAuthor%5D"><span class="name western">Jay Perry</span></a>
</div>
</div>
<sup>g</sup>, <a aria-describedby="id14" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koren%20S%22%5BAuthor%5D"><span class="name western">Sergey Koren</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Sergey Koren</span></h3>
<div class="p">
<sup>b</sup>National Biodefense Analysis and Countermeasures Center, Ft. Detrick, Maryland, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koren%20S%22%5BAuthor%5D"><span class="name western">Sergey Koren</span></a>
</div>
</div>
<sup>b,</sup><sup>*</sup>, <a aria-describedby="id15" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Phillippy%20A%22%5BAuthor%5D"><span class="name western">Adam Phillippy</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Adam Phillippy</span></h3>
<div class="p">
<sup>b</sup>National Biodefense Analysis and Countermeasures Center, Ft. Detrick, Maryland, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Phillippy%20A%22%5BAuthor%5D"><span class="name western">Adam Phillippy</span></a>
</div>
</div>
<sup>b,</sup><sup>*</sup>, <a aria-describedby="id16" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Klubnik%20J%22%5BAuthor%5D"><span class="name western">Joy Klubnik</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Joy Klubnik</span></h3>
<div class="p">
<sup>b</sup>National Biodefense Analysis and Countermeasures Center, Ft. Detrick, Maryland, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Klubnik%20J%22%5BAuthor%5D"><span class="name western">Joy Klubnik</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id17" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Treangen%20TJ%22%5BAuthor%5D"><span class="name western">Todd J Treangen</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Todd J Treangen</span></h3>
<div class="p">
<sup>b</sup>National Biodefense Analysis and Countermeasures Center, Ft. Detrick, Maryland, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Treangen%20TJ%22%5BAuthor%5D"><span class="name western">Todd J Treangen</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id18" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rosovitz%20MJ%22%5BAuthor%5D"><span class="name western">M J Rosovitz</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">M J Rosovitz</span></h3>
<div class="p">
<sup>b</sup>National Biodefense Analysis and Countermeasures Center, Ft. Detrick, Maryland, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rosovitz%20MJ%22%5BAuthor%5D"><span class="name western">M J Rosovitz</span></a>
</div>
</div>
<sup>b</sup>, <a aria-describedby="id19" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bergman%20NH%22%5BAuthor%5D"><span class="name western">Nicholas H Bergman</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Nicholas H Bergman</span></h3>
<div class="p">
<sup>b</sup>National Biodefense Analysis and Countermeasures Center, Ft. Detrick, Maryland, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bergman%20NH%22%5BAuthor%5D"><span class="name western">Nicholas H Bergman</span></a>
</div>
</div>
<sup>b,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>a</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, California, USA</div>
<div id="aff2">
<sup>b</sup>National Biodefense Analysis and Countermeasures Center, Ft. Detrick, Maryland, USA</div>
<div id="aff3">
<sup>c</sup>Department of Biology and Biochemistry, University of Houston, Houston, Texas, USA</div>
<div id="aff4">
<sup>d</sup>Department of Medical Microbiology, University of Groningen, University Medical Center Groningen, Groningen, The Netherlands</div>
<div id="aff5">
<sup>e</sup>Lawrence Livermore National Laboratory, Berkeley, California, USA</div>
<div id="aff6">
<sup>f</sup>Johnson Space Center, Houston, Texas, USA</div>
<div id="aff7">
<sup>g</sup>Marshall Space Flight Center, Huntsville, Alabama, USA</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>✉</sup><p class="display-inline">Address correspondence to Kasthuri Venkateswaran, <span>kjvenkat@jpl.nasa.gov</span>, or Nicholas H. Bergman, <span>nicholas.bergman@nbacc.dhs.gov</span>.</p>
</div>
<div class="fn" id="fn1">
<sup>#</sup><p class="display-inline">T.J.T., M.J.R., and N.H.B. contributed equally to this article.</p>
</div>
<div class="fn" id="afn1">
<sup>*</sup><p class="display-inline">Present address: Sergey Koren and Adam Phillippy, Genome Informatics Section, Computational and Statistical Genomics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, USA.</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2017 May 31; Accepted 2017 Jun 5; Collection date 2017 Aug.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2017 Venkateswaran et al.</div>
<p>This is an open-access article distributed under the terms of the <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">Creative Commons Attribution 4.0 International license</a>.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC5552977  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28798168/">28798168</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>The draft genome sequences of six <em>Bacillus</em> strains, isolated from the International Space Station and belonging to the <em>Bacillus anthracis</em>-<em>B. cereus</em>-<em>B. thuringiensis</em> group, are presented here. These strains were isolated from the Japanese Experiment Module (one strain), U.S. Harmony Node 2 (three strains), and Russian Segment Zvezda Module (two strains).</p></section><section id="s0"><h2 class="pmc_sec_title">GENOME ANNOUNCEMENT</h2>
<p>Among the six <em>Bacillus cereus sensu lato</em> strains reported here, three U.S. Harmony Node 2 isolates (ISSFR-3F, ISSFR-9F, and ISSFT-23F) and one Japanese Experiment Module isolate (JEM-2) were sequenced and assembled with both Illumina MiSeq and PacBio RSII sequence data. The remaining assemblies, including two Russian isolates, were sequenced and assembled with only MiSeq data. The MiSeq runs yielded on average 24 to 54 million 300-bp reads (from 1,402× to 3,093× average coverage), while PacBio yielded 4,000 to 116,000 reads (from 7× to 202× average coverage) (<a class="usa-link" href="#tab1">Table 1</a>). Due to the extremely high coverage (&gt;1,000×), Illumina MiSeq reads were randomly down-sampled to 100× using an estimated genome size of 5.3 Mb, resulting in an average of 1.2 to 1.5 million paired-end reads per isolate. Next, the down-sampled reads were assembled using iMetAMOS (<a aria-describedby="B1" class="usa-link" href="#B1">1</a>) with IDBA_UD and SPAdes (<a aria-describedby="B2" class="usa-link" href="#B2">2</a>). IDBA_UD was selected as the best assembly for all six isolates. Low confidence bases within the selected IDBA_UD (<a aria-describedby="B3" class="usa-link" href="#B3">3</a>) assemblies were masked out by mapping all reads to the assembled contigs and detecting conflicting variants with FreeBayes (<a aria-describedby="B4" class="usa-link" href="#B4">4</a>). The PacBio reads were assembled following the methods described by Berlin et al. (<a aria-describedby="B5" class="usa-link" href="#B5">5</a>) with Celera Assembler version 8.3rc1 and polished with Quiver (<a aria-describedby="B6" class="usa-link" href="#B6">6</a>). A second round of polishing was performed post-Quiver using the available MiSeq data as input to Pilon (<a aria-describedby="B7" class="usa-link" href="#B7">7</a>).</p>
<section class="tw xbox font-sm" id="tab1"><h3 class="obj_head">TABLE 1 .</h3>
<div class="caption p"><p><em>De novo</em> assembly statistics of the <em>Bacillus</em> spp. isolated from the ISS</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
<col span="1"/>
</colgroup>
<thead><tr>
<th colspan="1" rowspan="1">Isolate no.</th>
<th colspan="1" rowspan="1">ISS module</th>
<th colspan="1" rowspan="1">GenBank accession no.</th>
<th colspan="1" rowspan="1">Sequencing platform</th>
<th colspan="1" rowspan="1">No. of reads</th>
<th colspan="1" rowspan="1">Avg. read coverage (×)</th>
<th colspan="1" rowspan="1">
<em>N</em><sub>50</sub> (Mb)</th>
<th colspan="1" rowspan="1">No. of contigs</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">ISSFR-3F</td>
<td colspan="1" rowspan="1">U.S.<a class="usa-link" href="#ngtab1.1"><sup>a</sup></a>  </td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/CP018931" rel="noopener noreferrer" target="_blank">CP018931</a></td>
<td colspan="1" rowspan="1">PacBio</td>
<td colspan="1" rowspan="1">116,418</td>
<td colspan="1" rowspan="1">202</td>
<td colspan="1" rowspan="1">5.2</td>
<td colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ISSFR-9F</td>
<td colspan="1" rowspan="1">U.S.</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/CP018933" rel="noopener noreferrer" target="_blank">CP018933</a></td>
<td colspan="1" rowspan="1">PacBio</td>
<td colspan="1" rowspan="1">15,939</td>
<td colspan="1" rowspan="1">42</td>
<td colspan="1" rowspan="1">5.2 </td>
<td colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td colspan="1" rowspan="1">ISSFR-23F</td>
<td colspan="1" rowspan="1">U.S.</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MSMO00000000" rel="noopener noreferrer" target="_blank">MSMO00000000</a></td>
<td colspan="1" rowspan="1">PacBio</td>
<td colspan="1" rowspan="1">8,145</td>
<td colspan="1" rowspan="1">17</td>
<td colspan="1" rowspan="1">0.6 </td>
<td colspan="1" rowspan="1">16</td>
</tr>
<tr>
<td colspan="1" rowspan="1">JEM-2</td>
<td colspan="1" rowspan="1">Japan<a class="usa-link" href="#ngtab1.2"><sup>b</sup></a>  </td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/CP018935" rel="noopener noreferrer" target="_blank">CP018935</a></td>
<td colspan="1" rowspan="1">PacBio</td>
<td colspan="1" rowspan="1">40,305</td>
<td colspan="1" rowspan="1">80</td>
<td colspan="1" rowspan="1">5.2 </td>
<td colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td colspan="1" rowspan="1">S1-R4H1-FB</td>
<td colspan="1" rowspan="1">Russian Federation<a class="usa-link" href="#ngtab1.3"><sup>c</sup></a>
</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/NBNT00000000" rel="noopener noreferrer" target="_blank">NBNT00000000</a></td>
<td colspan="1" rowspan="1">Illumina</td>
<td colspan="1" rowspan="1">4,275,902</td>
<td colspan="1" rowspan="1">242</td>
<td colspan="1" rowspan="1">0.1</td>
<td colspan="1" rowspan="1">329</td>
</tr>
<tr>
<td colspan="1" rowspan="1">S2-R3J1-FB-BA1</td>
<td colspan="1" rowspan="1">Russian Federation</td>
<td colspan="1" rowspan="1"><a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/NBNR00000000" rel="noopener noreferrer" target="_blank">NBNR00000000</a></td>
<td colspan="1" rowspan="1">Illumina</td>
<td colspan="1" rowspan="1">11,266,760</td>
<td colspan="1" rowspan="1">638</td>
<td colspan="1" rowspan="1">0.4</td>
<td colspan="1" rowspan="1">388</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/tab1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="ngtab1.1">
<sup>a</sup><p class="display-inline">U.S., U.S. Segment Harmony Node 2.</p>
</div>
<div class="fn" id="ngtab1.2">
<sup>b</sup><p class="display-inline">Japan, Kibo Japanese Experiment Module (JEM).</p>
</div>
<div class="fn" id="ngtab1.3">
<sup>c</sup><p class="display-inline">Russian Federation, Russian Segment Zvezda Module.</p>
</div>
</div></section><p>The six International Space Station (ISS) isolates were aligned (NUCmer [<a aria-describedby="B8" class="usa-link" href="#B8">8</a>], Parsnp [<a aria-describedby="B9" class="usa-link" href="#B9">9</a>]) against members of the <em>B. cereus sensu lato</em> group genomes (<a aria-describedby="B10" class="usa-link" href="#B10">10</a>). The PacBio assemblies were used for all isolates with sufficient read coverage, and Illumina assemblies were used for the remaining isolates. Based on genome size estimates (5.2 to 5.3 Mb), NUCmer pairwise alignments (&gt;99.9% average pairwise nucleotide identity) and maximum-likelihood phylogenetic placement, all six isolates were found to exhibit a very high degree of similarity.</p>
<p>Finally, due to their high genomic similarity to the <em>B. anthracis</em> type strain (&gt;98% average nucleotide identity), all six genomes were examined for evidence of pathogenicity. However, none of the commonly known <em>B. anthracis</em> signature elements were identified. Specifically, all six ISS isolates (i) contain the <em>plcR</em> (<a aria-describedby="B11" class="usa-link" href="#B11">11</a>) ancestral “C” allele, which has been used in large-scale phylogenetic analyses to distinguish <em>B. anthracis</em> strains from the rest of the <em>B. cereus</em> group; (ii) lack significant hits to pXO1 and pXO2 plasmids; and (iii) are phylogenetically placed outside of the <em>B. anthracis</em> clade. Results were consistent with a comparative genomic analysis performed using the Lawrence Livermore National Laboratory Microbial Threat Characterization Pipeline. Altogether, the collective genomic evidence supports the conclusion that the six ISS isolates represent a novel <em>Bacillus</em> sp. located within the <em>B. cereus sensu lato</em> group.</p>
<section id="s1"><h3 class="pmc_sec_title">Accession number(s).</h3>
<p>The complete genome sequences were deposited in NCBI under the accession numbers listed in <a class="usa-link" href="#tab1">Table 1</a> and can be accessed from the NASA GeneLab system (GLDS-64; <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-64" rel="noopener noreferrer" target="_blank">https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-64</a>).</p></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>Part of the research described in this publication was carried out at the Jet Propulsion Laboratory (JPL), California Institute of Technology, under a contract with NASA. The contributions of S.R., R.K.P., T.E.B., S.K., A.P., J.K., T.J.T., M.J.R., and N.H.B. were funded under contract no. HSHQDC-07-C-00020 awarded by the Department of Homeland Security (DHS) Science and Technology Directorate (S&amp;T) for the management and operation of the National Biodefense Analysis and Countermeasures Center (NBACC), a Federally Funded Research and Development Center. This research was also funded by 2012 Space Biology (NNH12ZTT001N) grant no. 19-12829-26 under Task Order NNN13D111T awarded to K.V., which also funded the postdoctoral fellowship for A.C.S., and by JPL subcontract 1506453 to G.E.F. The research carried out at Lawrence Livermore National Laboratory was funded by 2014 Space Biology (NNH14ZTT002N) grant no. NNX15AJ29G awarded to C.J., which also funded J.A. and C.T. The contribution of S.P.V.T. was supported by the European Space Agency (MAP Project no. AO-LS-99-MAP-LSS-018 “biofilms”) and SRON (MG-064/MG-068).</p>
<p> We thank Kazuyuki Tasaki, Director of the JEM Utilization Center, and Julie Robinson, Chief Scientist of the ISS (U.S.) for arranging Japanese Aerospace Exploration Agency (JAXA) approval to use information on strains collected during the routine operation and maintenance of the JEM-Kibo Module. We also thank Hermie J. M. Harmsen and Hendrik I. J. Roest for providing the Russian isolates and logistical support, as well as Jonathan Hnath, Kathy Fronda, Gregory Horn, and Henry Lupari from NBACC for technical assistance.</p>
<p> The views and conclusions contained in this document are those of the authors and should not be interpreted as necessarily representing the official policies, either expressed or implied, of the DHS or S&amp;T. In no event, shall DHS, NBACC, S&amp;T, or Battelle National Biodefense Institute have any responsibility or liability for any use, misuse, inability to use, or reliance upon the information contained herein. DHS does not endorse any products or commercial services mentioned in this publication. </p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn6"><p><strong>Citation</strong> Venkateswaran K, Checinska Sielaff A, Ratnayake S, Pope RK, Blank TE, Stepanov VG, Fox GE, van Tongeren SP, Torres C, Allen J, Jaing C, Pierson D, Perry J, Koren S, Phillippy A, Klubnik J, Treangen TJ, Rosovitz MJ, Bergman NH. 2017. Draft genome sequences from a novel clade of <em>Bacillus cereus sensu lato</em> strains, isolated from the International Space Station. Genome Announc 5:e00680-17. <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/genomeA.00680-17" rel="noopener noreferrer" target="_blank">https://doi.org/10.1128/genomeA.00680-17</a>.</p></div></div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>Koren S, Treangen TJ, Hill CM, Pop M, Phillippy AM. 
2014. 
Automated ensemble assembly and validation of microbial genomes. BMC Bioinformatics
15:126. doi: 10.1186/1471-2105-15-126.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2105-15-126" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4030574/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24884846/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Automated%20ensemble%20assembly%20and%20validation%20of%20microbial%20genomes&amp;author=S%20Koren&amp;author=TJ%20Treangen&amp;author=CM%20Hill&amp;author=M%20Pop&amp;author=AM%20Phillippy&amp;volume=15&amp;publication_year=2014&amp;pages=126&amp;pmid=24884846&amp;doi=10.1186/1471-2105-15-126&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 
2012. 
SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol
19:455–477. doi: 10.1089/cmb.2012.0021.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/cmb.2012.0021" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3342519/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22506599/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&amp;title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&amp;author=A%20Bankevich&amp;author=S%20Nurk&amp;author=D%20Antipov&amp;author=AA%20Gurevich&amp;author=M%20Dvorkin&amp;volume=19&amp;publication_year=2012&amp;pages=455-477&amp;pmid=22506599&amp;doi=10.1089/cmb.2012.0021&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>Peng Y, Leung HC, Yiu SM, Chin FY. 
2012. 
IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth. Bioinformatics
28:1420–1428. doi: 10.1093/bioinformatics/bts174.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/bioinformatics/bts174" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22495754/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=IDBA-UD:%20a%20de%20novo%20assembler%20for%20single-cell%20and%20metagenomic%20sequencing%20data%20with%20highly%20uneven%20depth&amp;author=Y%20Peng&amp;author=HC%20Leung&amp;author=SM%20Yiu&amp;author=FY%20Chin&amp;volume=28&amp;publication_year=2012&amp;pages=1420-1428&amp;pmid=22495754&amp;doi=10.1093/bioinformatics/bts174&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>Garrison E, Marth G. 
2012. 
Haplotype-based variant detection from short-read sequencing. arXiv:1207.3907
<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://arxiv.org/abs/1207.3907" rel="noopener noreferrer" target="_blank">https://arxiv.org/abs/1207.3907</a>.</cite>
</li>
<li id="B5">
<span class="label">5.</span><cite>Berlin K, Koren S, Chin CS, Drake JP, Landolin JM, Phillippy AM. 
2015. 
Assembling large genomes with single-molecule sequencing and locality-sensitive hashing. Nat Biotechnol
33:623–630. doi: 10.1038/nbt.3238.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nbt.3238" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26006009/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Assembling%20large%20genomes%20with%20single-molecule%20sequencing%20and%20locality-sensitive%20hashing&amp;author=K%20Berlin&amp;author=S%20Koren&amp;author=CS%20Chin&amp;author=JP%20Drake&amp;author=JM%20Landolin&amp;volume=33&amp;publication_year=2015&amp;pages=623-630&amp;pmid=26006009&amp;doi=10.1038/nbt.3238&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>Chin CS, Alexander DH, Marks P, Klammer AA, Drake J, Heiner C, Clum A, Copeland A, Huddleston J, Eichler EE, Turner SW, Korlach J. 
2013. 
Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data. Nat Methods
10:563–569. doi: 10.1038/nmeth.2474.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.2474" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23644548/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Nonhybrid,%20finished%20microbial%20genome%20assemblies%20from%20long-read%20SMRT%20sequencing%20data&amp;author=CS%20Chin&amp;author=DH%20Alexander&amp;author=P%20Marks&amp;author=AA%20Klammer&amp;author=J%20Drake&amp;volume=10&amp;publication_year=2013&amp;pages=563-569&amp;pmid=23644548&amp;doi=10.1038/nmeth.2474&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>Walker BJ, Abeel T, Shea T, Priest M, Abouelliel A, Sakthikumar S, Cuomo CA, Zeng Q, Wortman J, Young SK, Earl AM. 
2014. 
Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. PLoS One
9:e112963. doi: 10.1371/journal.pone.0112963.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0112963" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4237348/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25409509/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Pilon:%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement&amp;author=BJ%20Walker&amp;author=T%20Abeel&amp;author=T%20Shea&amp;author=M%20Priest&amp;author=A%20Abouelliel&amp;volume=9&amp;publication_year=2014&amp;pages=e112963&amp;pmid=25409509&amp;doi=10.1371/journal.pone.0112963&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>Delcher AL, Salzberg SL, Phillippy AM. 
2003. 
Using MUMmer to identify similar regions in large sequence sets. Curr Protoc Bioinformatics Chapter 10:Unit 10.3. doi: 10.1002/0471250953.bi1003s00.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/0471250953.bi1003s00" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18428693/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr%20Protoc%20Bioinformatics&amp;title=Using%20MUMmer%20to%20identify%20similar%20regions%20in%20large%20sequence%20sets&amp;author=AL%20Delcher&amp;author=SL%20Salzberg&amp;author=AM%20Phillippy&amp;publication_year=2003&amp;pmid=18428693&amp;doi=10.1002/0471250953.bi1003s00&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>Treangen TJ, Ondov BD, Koren S, Phillippy AM. 
2014. 
The Harvest suite for rapid core-genome alignment and visualization of thousands of intraspecific microbial genomes. Genome Biol
15:524. doi: 10.1186/PREACCEPT-2573980311437212.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/PREACCEPT-2573980311437212" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4262987/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25410596/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=The%20Harvest%20suite%20for%20rapid%20core-genome%20alignment%20and%20visualization%20of%20thousands%20of%20intraspecific%20microbial%20genomes&amp;author=TJ%20Treangen&amp;author=BD%20Ondov&amp;author=S%20Koren&amp;author=AM%20Phillippy&amp;volume=15&amp;publication_year=2014&amp;pages=524&amp;pmid=25410596&amp;doi=10.1186/PREACCEPT-2573980311437212&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>Zwick ME, Joseph SJ, Didelot X, Chen PE, Bishop-Lilly KA, Stewart AC, Willner K, Nolan N, Lentz S, Thomason MK, Sozhamannan S, Mateczun AJ, Du L, Read TD. 
2012. 
Genomic characterization of the <em>Bacillus cereus</em> sensu lato species: backdrop to the evolution of Bacillus anthracis. Genome Res
22:1512–1524. doi: 10.1101/gr.134437.111.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.134437.111" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3409264/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22645259/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=Genomic%20characterization%20of%20the%20Bacillus%20cereus%20sensu%20lato%20species:%20backdrop%20to%20the%20evolution%20of%20Bacillus%20anthracis&amp;author=ME%20Zwick&amp;author=SJ%20Joseph&amp;author=X%20Didelot&amp;author=PE%20Chen&amp;author=KA%20Bishop-Lilly&amp;volume=22&amp;publication_year=2012&amp;pages=1512-1524&amp;pmid=22645259&amp;doi=10.1101/gr.134437.111&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>Easterday WR, Van Ert MN, Simonson TS, Wagner DM, Kenefic LJ, Allender CJ, Keim P. 
2005. 
Use of single nucleotide polymorphisms in the <em>plcR</em> gene for specific identification of <em>Bacillus anthracis</em>. J Clin Microbiol
43:1995–1997. doi: 10.1128/JCM.43.4.1995-1997.2005.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JCM.43.4.1995-1997.2005" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1081367/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15815042/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Microbiol&amp;title=Use%20of%20single%20nucleotide%20polymorphisms%20in%20the%20plcR%20gene%20for%20specific%20identification%20of%20Bacillus%20anthracis&amp;author=WR%20Easterday&amp;author=MN%20Van%20Ert&amp;author=TS%20Simonson&amp;author=DM%20Wagner&amp;author=LJ%20Kenefic&amp;volume=43&amp;publication_year=2005&amp;pages=1995-1997&amp;pmid=15815042&amp;doi=10.1128/JCM.43.4.1995-1997.2005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section></section></section></article>
</main></body></html>