<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8012687/"/><title>Effects of Simulated Microgravity on the Proteome and Secretome of the Polyextremotolerant Black Fungus Knufia chersonesos - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Front Genet</button></div>. 2021 Mar 18;12:638708. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fgene.2021.638708" rel="noopener noreferrer" target="_blank">10.3389/fgene.2021.638708</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Effects of Simulated Microgravity on the Proteome and Secretome of the Polyextremotolerant Black Fungus <em>Knufia chersonesos</em>
</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tesei%20D%22%5BAuthor%5D"><span class="name western">Donatella Tesei</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Donatella Tesei</span></h3>
<div class="p">
<sup>1</sup>Department of Biotechnology, University of Natural Resources and Life Sciences, Vienna, Austria</div>
<div class="p">
<sup>2</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tesei%20D%22%5BAuthor%5D"><span class="name western">Donatella Tesei</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>*</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chiang%20AJ%22%5BAuthor%5D"><span class="name western">Abby J Chiang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Abby J Chiang</span></h3>
<div class="p">
<sup>3</sup>Department of Molecular Imaging and Therapy, Beckman Research Institute of City of Hope, Duarte, CA, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chiang%20AJ%22%5BAuthor%5D"><span class="name western">Abby J Chiang</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kalkum%20M%22%5BAuthor%5D"><span class="name western">Markus Kalkum</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Markus Kalkum</span></h3>
<div class="p">
<sup>3</sup>Department of Molecular Imaging and Therapy, Beckman Research Institute of City of Hope, Duarte, CA, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kalkum%20M%22%5BAuthor%5D"><span class="name western">Markus Kalkum</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stajich%20JE%22%5BAuthor%5D"><span class="name western">Jason E Stajich</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jason E Stajich</span></h3>
<div class="p">
<sup>4</sup>Department of Microbiology and Plant Pathology, Institute of Integrative Genome Biology, University of California, Riverside, Riverside, CA, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stajich%20JE%22%5BAuthor%5D"><span class="name western">Jason E Stajich</span></a>
</div>
</div>
<sup>4</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mohan%20GBM%22%5BAuthor%5D"><span class="name western">Ganesh Babu Malli Mohan</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ganesh Babu Malli Mohan</span></h3>
<div class="p">
<sup>5</sup>Department of Biotechnology, Centre for Research and Infectious Diseases, SASTRA Deemed University, Thanjavur, India</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mohan%20GBM%22%5BAuthor%5D"><span class="name western">Ganesh Babu Malli Mohan</span></a>
</div>
</div>
<sup>5</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sterflinger%20K%22%5BAuthor%5D"><span class="name western">Katja Sterflinger</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Katja Sterflinger</span></h3>
<div class="p">
<sup>6</sup>Institute for Natural Sciences and Technology in the Arts, Academy of Fine Arts Vienna, Vienna, Austria</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sterflinger%20K%22%5BAuthor%5D"><span class="name western">Katja Sterflinger</span></a>
</div>
</div>
<sup>6</sup>, <a aria-describedby="id7" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Kasthuri Venkateswaran</span></h3>
<div class="p">
<sup>2</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, United States</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a>
</div>
</div>
<sup>2</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>Department of Biotechnology, University of Natural Resources and Life Sciences, Vienna, Austria</div>
<div id="aff2">
<sup>2</sup>Biotechnology and Planetary Protection Group, Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, United States</div>
<div id="aff3">
<sup>3</sup>Department of Molecular Imaging and Therapy, Beckman Research Institute of City of Hope, Duarte, CA, United States</div>
<div id="aff4">
<sup>4</sup>Department of Microbiology and Plant Pathology, Institute of Integrative Genome Biology, University of California, Riverside, Riverside, CA, United States</div>
<div id="aff5">
<sup>5</sup>Department of Biotechnology, Centre for Research and Infectious Diseases, SASTRA Deemed University, Thanjavur, India</div>
<div id="aff6">
<sup>6</sup>Institute for Natural Sciences and Technology in the Arts, Academy of Fine Arts Vienna, Vienna, Austria</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Edited by: Joanna Jankowicz-Cieslak, International Atomic Energy Agency, Austria</p></div>
<div class="fn" id="fn2"><p>Reviewed by: Mateusz Molon, University of Rzeszow, Poland; Khaled Y. Kamal, Texas A&amp;M University, United States; Ruth Bryan, Albert Einstein College of Medicine, United States</p></div>
<div class="fn" id="c001">
<sup>✉</sup><p class="display-inline">*Correspondence: Donatella Tesei, <span>donatella.tesei@boku.ac.at</span></p>
</div>
<div class="fn" id="fn004"><p>This article was submitted to Systems Biology, a section of the journal Frontiers in Genetics</p></div>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2020 Dec 7; Accepted 2021 Feb 19; Collection date 2021.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>Copyright © 2021 Tesei, Chiang, Kalkum, Stajich, Mohan, Sterflinger and Venkateswaran.</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC8012687  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33815472/">33815472</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Black fungi are a group of melanotic microfungi characterized by remarkable polyextremotolerance. Due to a broad ecological plasticity and adaptations at the cellular level, it is predicted that they may survive in a variety of extreme environments, including harsh niches on Earth and Mars, and in outer space. However, the molecular mechanisms aiding survival, especially in space, are yet to be fully elucidated. Based on these premises, the rock-inhabiting black fungus <em>Knufia chersonesos</em> (Wt) and its non-melanized mutant (Mut) were exposed to simulated microgravity—one of the prevalent features characterizing space conditions—by growing the cultures in high-aspect-ratio vessels (HARVs). Qualitative and quantitative proteomic analyses were performed on the mycelia and supernatant of culture medium (secretome) to assess alterations in cell physiology in response to low-shear simulated microgravity (LSSMG) and to ultimately evaluate the role of cell-wall melanization in stress survival. Differential expression was observed for proteins involved in carbohydrate and lipid metabolic processes, transport, and ribosome biogenesis and translation via ribosomal translational machinery. However, no evidence of significant activation of stress components or starvation response was detected, except for the scytalone dehydratase, enzyme involved in the synthesis of dihydroxynaphthalene (DNH) melanin, which was found to be upregulated in the secretome of the wild type and downregulated in the mutant. Differences in protein modulation were observed between <em>K. chersonesos</em> Wt and Mut, with several proteins being downregulated under LSSMG in the Mut when compared to the Wt. Lastly, no major morphological alterations were observed following exposure to LSSMG. Similarly, the strains’ survivability was not negatively affected. This study is the first to characterize the response to simulated microgravity in black fungi, which might have implications on future astrobiological missions.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> microgravity, black fungi, extremophiles, secretomics, proteomics, astrobiology, <em>Knufia chersonesos</em> (syn. <em>K. petricola</em>)</p></section></section><section id="S1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Based on their nature as settlers in extreme environments, microbial extremophiles are of great interest to studies aiming to elucidate stress adaptation and survival mechanisms. Successful examples of extremophiles can be found in the fungal domain, alongside bacteria, and archaea, which for a long time were considered to be the sole colonizers of habitats previously considered uninhabitable. Some of these fungal extremophiles show even higher resistance than that of prokaryotes (<a aria-describedby="B91" class="usa-link" href="#B91">Shtarkman et al., 2013</a>; <a aria-describedby="B1" class="usa-link" href="#B1">Aguilera and González-Toril, 2019</a>). Black fungi in particular represent a group of highly melanized microfungi, whose ability to survive in a variety of extreme environments has in recent decades attracted increasing attention (<a aria-describedby="B85" class="usa-link" href="#B85">Selbmann et al., 2005</a>; <a aria-describedby="B96" class="usa-link" href="#B96">Sterflinger, 2006</a>; <a aria-describedby="B29" class="usa-link" href="#B29">Gorbushina, 2007</a>; <a aria-describedby="B65" class="usa-link" href="#B65">Onofri et al., 2007</a>). Desiccation, low nutrient availability, excessive radiation, extreme temperatures, salinity, and pH are some of the multiple sources of stress characterizing the habitats where these organisms have been shown to thrive (<a aria-describedby="B34" class="usa-link" href="#B34">Gunde-Cimerman et al., 2000</a>, <a aria-describedby="B33" class="usa-link" href="#B33">2003</a>; <a aria-describedby="B86" class="usa-link" href="#B86">Selbmann et al., 2008</a>). The highest diversity of black fungi has especially been observed in rocky environments, ranging from mountain summits to the Atacama Desert and the Antarctic cold regions (<a aria-describedby="B28" class="usa-link" href="#B28">Gonçalves et al., 2016</a>; <a aria-describedby="B3" class="usa-link" href="#B3">Ametrano et al., 2019</a>). Rock represents a harsh habitat and a quite ancient niche for life, believed to reflect early Earth conditions, and thus considered to be a model for extraterrestrial life (<a aria-describedby="B87" class="usa-link" href="#B87">Selbmann et al., 2014</a>).</p>
<p>The discovery of melanized fungi in extreme environments has prompted researchers to investigate microbial physiology at the absolute edges of adaptability, aiming at a deeper understanding of the limits for life (<a aria-describedby="B20" class="usa-link" href="#B20">Dadachova and Casadevall, 2008</a>; <a aria-describedby="B103" class="usa-link" href="#B103">Tesei et al., 2012</a>; <a aria-describedby="B116" class="usa-link" href="#B116">Zakharova et al., 2013</a>). Other studies have focused on testing fungal survival in space conditions through simulations in ground-based facilities or in space missions that enabled the assessment of the habitability of extraterrestrial environments and, hence, the possibility of life beyond Earth (<a aria-describedby="B82" class="usa-link" href="#B82">Scalzi et al., 2012</a>; <a aria-describedby="B66" class="usa-link" href="#B66">Onofri et al., 2019</a>). Space and outer space conditions are by definition hostile, as they include enhanced irradiation, microgravity, and temperature extremes (<a aria-describedby="B72" class="usa-link" href="#B72">Rabbow et al., 2012</a>, <a aria-describedby="B73" class="usa-link" href="#B73">2017</a>; <a aria-describedby="B88" class="usa-link" href="#B88">Senatore et al., 2018</a>). Nevertheless, the isolation of melanized fungi from spacecraft and space stations—e.g., the International Space Station (ISS)—has been reported frequently (<a aria-describedby="B15" class="usa-link" href="#B15">Checinska et al., 2015</a>; <a aria-describedby="B16" class="usa-link" href="#B16">Checinska-Sielaff et al., 2019</a>). In this respect, a number of studies have examined the molecular adaptations of ISS-isolated strains to space conditions, showing alterations in metabolome and proteome (<a aria-describedby="B47" class="usa-link" href="#B47">Knox et al., 2016</a>; <a aria-describedby="B7" class="usa-link" href="#B7">Blachowicz et al., 2019b</a>; <a aria-describedby="B76" class="usa-link" href="#B76">Romsdahl et al., 2019</a>). Investigations of microbial survival in space are therefore relevant also in the context of space missions, to prevent contaminations and to develop strategies to reduce hazard, especially in the case of opportunistic species (<a aria-describedby="B107" class="usa-link" href="#B107">Urbaniak et al., 2019</a>).</p>
<p>In black fungi, astrobiological studies have evaluated the potential effects of Mars or ISS conditions on fungal viability and metabolism. One investigation revealed that a rock isolate from Antarctica, <em>Cryomyces minteri</em>, could survive simulated Martian atmosphere and pressure, temperature fluctuations between −20 and 20°C, ultraviolet radiation, and vacuum (<a aria-describedby="B63" class="usa-link" href="#B63">Onofri et al., 2008</a>). Comparative 2D-PAGE proteomics was carried out for other rock-inhabiting fungi (RIF) (i.e., <em>Cryomyces antarcticus</em>, <em>Knufia perforans</em>, and <em>Exophiala jeanselmei</em>) exposed to these conditions and showed a decrease in protein complexity, followed by recovery of the metabolic activity after 1 week of exposure (<a aria-describedby="B115" class="usa-link" href="#B115">Zakharova et al., 2014</a>). Further, survivability of <em>C</em>. <em>antarcticus</em> in outer space was shown via colony counts following a 1.5-year-long exposure on board the EXPOSE-E facility of the ISS (<a aria-describedby="B64" class="usa-link" href="#B64">Onofri et al., 2012</a>). A more recent experiment of <em>C</em>. <em>antarcticus</em> exposure on rock analogs under space and simulated Martian conditions revealed only slight ultra-structural and molecular damage and pointed out the high stability of DNA within melanized cells (<a aria-describedby="B68" class="usa-link" href="#B68">Pacelli et al., 2016</a>). In other investigations, the resistance to acute ionizing radiations was demonstrated in RIF (<a aria-describedby="B69" class="usa-link" href="#B69">Pacelli et al., 2017</a>, <a aria-describedby="B67" class="usa-link" href="#B67">2018</a>). Together, these studies have revealed the ability of rock-colonizing black fungi to endure space conditions; however, to date, reports that specifically evaluate the response to microgravity have not been produced.</p>
<p>Microgravity is an important factor influencing microbial life in space environments, a condition in which the gravity level is almost zero but not neutralized. Due to the technological and logistical hurdles linked to studies of microgravity in space, different methods have been developed to simulate microgravity and analyze microbial responses (<a aria-describedby="B37" class="usa-link" href="#B37">Herranz et al., 2013</a>). Accordingly, the term low-shear simulated microgravity (LSSMG) is used to describe the environmental condition created by these devices, resembling the low-shear effects of the fluid on the cells (<a aria-describedby="B112" class="usa-link" href="#B112">Yamaguchi et al., 2014</a>). Spaceflight and ground-based microgravity analog experiments have suggested that microgravity can affect microbial gene expression, cell morphology, physiology, and metabolism, also triggering increased virulence in pathogenic bacteria and fungi (<a aria-describedby="B2" class="usa-link" href="#B2">Altenburg et al., 2008</a>; <a aria-describedby="B100" class="usa-link" href="#B100">Taylor, 2015</a>; <a aria-describedby="B79" class="usa-link" href="#B79">Sathishkumar et al., 2016</a>; <a aria-describedby="B38" class="usa-link" href="#B38">Huang et al., 2018</a>). Although the effects of microgravity on microbes have been studied for several years, only the responses of a few typical prokaryotic and eukaryotic model organisms—e.g., <em>Escherichia coli</em>, <em>Candida albicans</em>, <em>Saccharomyces cerevisiae, Penicillium sp., Aspergillus sp.</em>—have hitherto been investigated (<a aria-describedby="B38" class="usa-link" href="#B38">Huang et al., 2018</a>). Hence, studying the reaction of black fungi to microgravity holds potential for the elucidation of the molecular basis of tolerance in extremotolerant and extremophilic fungi, and can also contribute to unearthing the biological uniqueness of these species and their adaptability to space conditions.</p>
<p>In the present study, the qualitative and quantitative proteomic characterization of a black fungus response to LSSMG was carried out for the first time. The rock-associated <em>Knufia chersonesos</em> (syn. <em>Knufia petricola</em>) was selected as the model organism due to its reported poikilo-tolerance, e.g., the ability to endure xeric conditions, desiccation, high UV-radiation and temperatures (<a aria-describedby="B98" class="usa-link" href="#B98">Sterflinger et al., 2012</a>) and to feed on alternative carbon sources like monoaromatic compounds (<a aria-describedby="B60" class="usa-link" href="#B60">Nai et al., 2013</a>) and synthetic polyesters (<a aria-describedby="B104" class="usa-link" href="#B104">Tesei et al., 2020</a>). <em>K. chersonesos</em> aptitude to withstand levels of gaseous ozone up to 11 ppm was also shown (Tesei et al., unpublished). Furthermore, being the only black fungus known to have a melanin-deficient spontaneous mutant (<a aria-describedby="B105" class="usa-link" href="#B105">Tesei et al., 2017</a>), <em>K. chersonesos</em> allows comparative studies attempting to evaluate the stress-protective role of melanin. Cultures of <em>K. chersonesos</em> wild type and mutant were grown in HARVs and analyzed for changes at the proteome—whole-cell proteome (mycelia) and secretome (culture supernatant)—and at the morphological level, with an eye toward the impact of cell-wall melanization on the physiological response to the stress.</p></section><section id="S2"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="S2.SS1"><h3 class="pmc_sec_title">Fungal Strains</h3>
<p>The fungal strains used in this study included the non-pathogenic rock-inhabiting fungus <em>K. chersonesos</em> MA5789 wild type (Wt) and MA5790 mutant (Mut), both obtained from the ACBR fungal culture collection of the University of Natural Resources and Life Sciences, Vienna, Austria. The Wt, characterized by a highly melanized mycelium, was isolated from red sandstone in Ny London, Svalbard, Norway. The pink mutant, whose pigmentation is due to unmasking of carotenoids resulting from the lack of melanin, originated spontaneously under laboratory conditions (<a aria-describedby="B105" class="usa-link" href="#B105">Tesei et al., 2017</a>). <em>K. chersonesos</em> (syn. <em>K. petricola</em>) is an emerging model organism for analyses aiming at the elucidation of the rock-lifestyle and RIF physiology (<a aria-describedby="B60" class="usa-link" href="#B60">Nai et al., 2013</a>). Along with its ascertained thermo-, pH-, UV- and desiccation tolerance (<a aria-describedby="B30" class="usa-link" href="#B30">Gorbushina et al., 2008</a>; <a aria-describedby="B98" class="usa-link" href="#B98">Sterflinger et al., 2012</a>), the fungus was recently reported to have an aptitude for using synthetic polymers as an alternative carbon source (<a aria-describedby="B104" class="usa-link" href="#B104">Tesei et al., 2020</a>). These features and the ability to endure high levels of gaseous ozone (Tesei et al., unpublished), altogether make <em>K. chersonesos</em> particularly suited for astrobiology studies. Further, the availability of a mutant strain allows comparative studies attempting to evaluate the role of melanin in stress protection. Fungal cultures were maintained in flasks containing 50 mL of 2% malt extract broth (MEB, pH 5, 2% malt extract, 2% glucose, 1% peptone) at 21°C with shaking at 63 rpm (Innova, Eppendorf). Cultures at exponential phase were used for inoculation of media for all experiments.</p></section><section id="S2.SS2"><h3 class="pmc_sec_title">Exposure to Low-Shear Simulated Microgravity (LSSMG)</h3>
<p>Fungal pellets were obtained from 5-day-old cultures by centrifugation, washed in 1X phosphate-buffered saline (PBS; Thermo Fisher Scientific) and subsequently mildly ribolyzed (3 × 20”, speed 4; MP Biomedicals) to separate cells. As clump-like growth is an inherent characteristic of black fungi, mechanical disruption of cell clusters is a prerequisite for both inoculation of cultures and cell counting (<a aria-describedby="B109" class="usa-link" href="#B109">Voigt et al., 2020</a>). Following the cell count (Neubauer), the cell number was adjusted to the initial concentration of 2 × 10<sup>5</sup> ml<sup>–1</sup> with 10 mL fresh 2% MEB and used as seed culture. High-aspect-ratio vessels (HARVs; Synthecon Inc) were filled with the cell suspensions (10 mL) and rotated at 30 rpm (initial rotation rate) in the vertical axis to provide LSSMG conditions inside a chamber with 60% humidity and 22°C, for 7 days. Control runs were set up by rotating the bioreactors in the horizontal axis to provide normal gravity condition (1G) (<a aria-describedby="B78" class="usa-link" href="#B78">Rosenzweig et al., 2010</a>; <a aria-describedby="B46" class="usa-link" href="#B46">Kim and Rhee, 2016</a>). During cultivation, the rotation speed was adjusted to 43 rpm in order to keep the cells pellets orbiting within the vessel in continual fall and to prevent their contact with the vessel walls (<a class="usa-link" href="#F1">Figures 1A,B</a>). A total of two biological replicates were maintained throughout the experiments. An additional set of experiments identical to the conditions mentioned above was established for measurement of cell concentration and microscope observations. Cell concentration was assessed via hemocytometer count at different time points until completion of the experiment, using two biological replicates (i.e., biomass from two different vessels) for each experimental condition. Following removal of the vessel from the rotator base, samples of cells were collected under sterile conditions in a laminar flow hood using a luer-lock syringe and the syringe port in the culture vessel. The sampling was carried out one vessel at a time while the remaining vessels were kept rotating on the rotator base. Cell survivability at the end of the LSSMG exposure was assessed by enumeration of colony forming units (CFU) using ImageJ software (<a aria-describedby="B83" class="usa-link" href="#B83">Schneider et al., 2012</a>), according to <a aria-describedby="B17" class="usa-link" href="#B17">Choudhry (2016)</a>.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8012687_fgene-12-638708-g001.jpg" target="_blank"><img alt="FIGURE 1" class="graphic zoom-in" height="610" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3759/8012687/9efb8ecffbff/fgene-12-638708-g001.jpg" width="770"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Rotary Cell Culture System (RCCS<sup>TM</sup>, Synthecon) and High Aspect Ratio Vessels (HARVs) used in the LSSMG experiments. <strong>(A)</strong> Schematic diagram of mechanical principle of the different RCCSs (design: Emily Klonicki). In normal gravity, the vessels (HARVs) rotate around a plane parallel to the gravity vector. Microgravity is simulated by rotating the samples around a plane perpendicular to the gravity vector. <strong>(B)</strong> The RCCS used to generate 1G condition (control) on ground (left); the RCCS used to generate LSSMG condition on ground (right); <strong>(C)</strong> Colony and cultural features of <em>Knufia chersonesos</em> MA 5789 (wild type, Wt) and MA5790 (mutant, Mut) under LSSMG and 1G on the first and last day of cultivation. The HARVs provide oxygenation via a flat, silicon rubber gas transfer membrane located at the base of the vessel and directly underneath the cultures.</p></figcaption></figure></section><section id="S2.SS3"><h3 class="pmc_sec_title">Microscopy Studies</h3>
<section id="S2.SS3.SSS1"><h4 class="pmc_sec_title">Scanning Electron Microscopy (SEM)</h4>
<p>For SEM studies, samples from the two strains were collected from all the established cultures—LSSMG and normal gravity condition (1G)—at various timepoints, i.e., 1, 3, 5, and 7 days from the beginning of the cultivation. A 1:10 dilution of each sample was prepared using 1X phosphate-buffered saline (PBS; Thermo Fisher Scientific) and thereafter transferred onto electron microscopy coupons. For each sample, two biological replicates (and two technical replicates each i.e., aliquots of 20 μl) were prepared and let dry at 35°C before examination. A Sirion (FEI, Hillsboro, OR, United States) field-emission scanning electron microscope (FE-SEM) was used for examination of the fungal cells. No specimen preparation procedures, such as sample coating with a surface-conducting (carbon or metal) layer, were performed as they were not necessary when using the FE-SEM. The samples were analyzed using an acceleration voltage of 10–20 kV, beam current of 40–50 mA and positioning the detector 6–10 mm away from the coupon. Secondary electron images were acquired in the high-vacuum mode. For each sample coupon, a minimum of 10 fields was observed, and images were acquired with various magnification (100×–5,000×).</p></section><section id="S2.SS3.SSS2"><h4 class="pmc_sec_title">Fluorescence Microscopy</h4>
<p>For the cell integrity and viability assay, wheat germ agglutinin (WGA) and propidium iodide (PI) in combination with SYTO 9 were used. The carbohydrate-binding lectin WGA has a known affinity for β-1,4-N-acetylglucosamine (GlcNAc) oligomers present in the fungal polysaccharide chitin. PI, a membrane impermeant dye that is generally excluded from viable cells, was applied together with the nuclear and chromosome counterstain SYTO 9 for a dead/live stain. 200 μl aliquots were collected from all the established cultures after 1, 3, 5, and 7 days from the beginning of the cultivation. Fungal cells were washed twice in 1X PBS and suspended in 500 μl. Following a 1:10 dilution, 100 μl cell suspension was incubated with 50 μl Alexa Fluor 350 conjugate of WGA (Thermo Fisher Scientific) and 15 μl of FungaLight containing equal amounts of PI and SYTO 9 solution (Thermo Fisher Scientific), for 15 min at room temperature in the dark. The chosen amounts of the fluorescent probes are justified by the thickening and the increase in melanization of the cell walls, which hindered the staining process. Cell suspensions were mounted over glass slides and analyzed under an Axioplane microscope equipped with an AxioCam camera (Carl Zeiss). For each sample, two technical replicates were prepared for observation.</p></section></section><section id="S2.SS4"><h3 class="pmc_sec_title">Tandem Mass Tag (TMT)-Based Quantitative Shotgun Proteomics</h3>
<section id="S2.SS4.SSS1"><h4 class="pmc_sec_title">Sample Preparation</h4>
<p>The content of each vessel was spun at 7500 × g (5810, Eppendorf) and at 4°C for 15 min to separate fungal cell pellet (mycelium) from culture supernatant (secretome). Protease inhibitors (10 μl:1 mL v/v, Halt, Thermo Fisher Scientific) were thereafter added—and mixed with 1 mL 1 × PBS in the case of biomass samples—and samples were stored at −80°C prior to protein extraction. Proteins were extracted as previously described by <a aria-describedby="B77" class="usa-link" href="#B77">Romsdahl et al. (2018)</a> with some modifications. In brief, 1 mL lysis buffer consisting of 100 mM triethylammonium bicarbonate (TEAB) with 1:100 Halt Protease Inhibitor Cocktail (Thermo Fisher Scientific) was added to fungal cell pellets (i.e., ∼0.5 g) and culture supernatants (i.e., ∼0.5 mL), the latter previously concentrated in a SpeedVac system. Sample homogenization was achieved using a bead beater (Bertin) at 4°C (3 × 5500 rpm for 1 min., with 15 s. breaks in between) and following centrifugation at 17,000 × <em>g</em> and 4°C for 15 min, protein concentrations were determined by BCA assay (Thermo Fisher Scientific). The protein extracts were processed for a tandem mass tag (TMT) labeling as previously described (<a aria-describedby="B77" class="usa-link" href="#B77">Romsdahl et al., 2018</a>). For each sample, 250 μg proteins were precipitated in 20% TCA at 4°C, reduced by tris(2-carboxyethyl)phosphine (TCEP), alkylated with iodoacetamide (IAA), and digested with Trypsin/LysC (Promega, Madison, WI, United States) overnight at 37°C. Peptide quantitation was performed using the Pierce Quantitative Colorimetric Peptide Assay (Thermo Fisher Scientific). The proteomic and secretomic profiling of <em>K. chersonesos</em> Wt and Mut were carried out in two separate TMT LC/MS experiments. A total of 40 μg peptides from each cell pellet sample or 13 μg peptides from each culture supernatant were labeled with the Thermo Scientific TMT10 plex (TMT<sup>10</sup>) Isobaric Mass Tagging Kit according to the manufacturer protocol. Eight TMT tags were used to label samples from the same experimental set. The TMT<sup>10</sup>-131 label was used as a reference that contained a pool of 5 μg of peptides from all samples. All nine labeled-peptide samples were combined into a single tube, mixed and fractionated using the Pierce High pH Reversed-Phase Peptide Fractionation Kit (Thermo Fisher Scientific). The fractionated samples were dried using a SpeedVac concentrator and dissolved in 1% formic acid prior to LC-MS/MS analysis.</p></section><section id="S2.SS4.SSS2"><h4 class="pmc_sec_title">LC-MS/MS Analysis</h4>
<p>The samples were analyzed on an Orbitrap Fusion Lumos mass spectrometer with a Dionex UltiMate 3000 RSLCnano system, a 300 μm × 5 mm PepMap100 C18 precolumn, a 75 μm × 50 cm PepMap RSLC C18 analytical column, and an Easy-Spray ion source (Thermo Scientific). The column temperature was maintained at 45°C, and the peptides were eluted at a flow rate of 300 nL/min over a 110 min gradient, from 3 to 30% solvent B (100 min), 30 to 50% solvent B (5 min), 50 to 90% solvent B (1 min), 90% solvent B (1 min), and 90% to 3% solvent B (3 min). Solvent A was 0.1% formic acid in water and solvent B was 0.1% formic acid in acetonitrile. The full MS survey scan (400–1,600 m/z) was acquired in the orbitrap at a resolution of 240,000 and with an automatic gain control (AGC) target of 4 × 10<sup>5</sup>. The maximum injection time for MS scans was 50 ms. Monoisotopic precursor ions were selected with charge states 2–7 within a ± 10 ppm mass window using a 60 s dynamic exclusion. The MS<sup>2</sup> scan (400–1,200 m/z) was performed using the linear ion trap with the CID collision energy set to 35%. The ion trap scan rate was set to “rapid,” with an AGC target of 1 × 10<sup>4</sup>, and a maximum injection time of 30 ms. Ten fragment ions from each MS<sup>2</sup> experiment were then simultaneously selected for an MS<sup>3</sup> experiment. The MS<sup>3</sup> scan (100–500 m/z) was performed to generate the TMT reporter ions in the orbitrap at a resolution of 30,000 and using HCD at a collision energy setting of 65%, an AGC target of 5 × 10<sup>4</sup>, and a maximum injection time of 54 ms.</p></section><section id="S2.SS4.SSS3"><h4 class="pmc_sec_title">Quantitative Proteomics Analysis</h4>
<p>All MS/MS spectra were analyzed using Proteome Discoverer (version 2.2.0.388, Thermo Fisher Scientific) with the Sequest-HT searching engines against an in-house annotated draft genome sequence of <em>K. chersonesos</em> MA5789 Wt (GCA_002319055.1, assembly ASM231905v1, NCBI) (<a aria-describedby="B105" class="usa-link" href="#B105">Tesei et al., 2017</a>) consisting of 9.818 predicted protein coding gene models. The genome was annotated using funannotate (v1.3.4) (doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://dx.doi.org/10.5281/zenodo.1284502" rel="noopener noreferrer" target="_blank">10.5281/zenodo.1284502</a>), which combined predictions from <em>ab initio</em> gene predictors with Augustus trained by BUSCO gene models (fungi_odb9) (<a aria-describedby="B117" class="usa-link" href="#B117">Zdobnov et al., 2017</a>) and GeneMark.hmm (<a aria-describedby="B101" class="usa-link" href="#B101">Ter-Hovhannisyan et al., 2008</a>) informed by protein evidence from Swiss-Prot (<a aria-describedby="B12" class="usa-link" href="#B12">Boutet et al., 2007</a>) together into composite gene models with EvidenceModeler (<a aria-describedby="B35" class="usa-link" href="#B35">Haas et al., 2008</a>). Functional predictions for genes was assigned by protein homology to Pfam (<a aria-describedby="B24" class="usa-link" href="#B24">El-gebali et al., 2019</a>), Swiss-Prot/UniProt (v 2018_05), and EggNog (v1.10) databases (<a aria-describedby="B39" class="usa-link" href="#B39">Huerta-Cepas et al., 2019</a>). The following parameters were selected for the search: A maximum of two missed cleavage sites, a minimum peptide length of six residues, 5 ppm tolerance for precursor ion masses, and 0.6 Da tolerance for fragment ion masses. The static modification settings included carbamidomethyl of cysteine residues, and dynamic modifications included oxidation of methionine, TMT modification of lysine ε-amino groups and peptide N-termini, and acetyl modification of protein N-terminus. A false discovery rate (FDR) of 1% for peptides and proteins was obtained using a target-decoy database search. The reporter ions integration tolerance was 0.5 Da while the co-isolation threshold was 75%. The average signal-to-noise threshold of all reporter peaks was greater than 10. The quantitative abundance of each protein was determined from the total intensity of the detected reporter ions. For statistical analysis, the sum of reporter ion intensities for each protein was log2 transformed, and the technical triplicate measurements for each protein were averaged. Only the proteins that were identified with at least one peptide and quantified in all technical (<em>n</em> = 3) and biological replicates (<em>n</em> = 2), were considered for the statistical analysis. Student’s <em>t</em>-test was performed to identify differentially expressed proteins between each LSSMG-exposed and 1G-exposed group as well as between all groups. To compare all four experimental conditions, each condition was normalized to the reference channel, which contained equal amounts of peptides from each group. To evaluate changes between treatment (LSSMG) and control (1G), the protein abundance levels of each LSSMG-exposed sample were normalized to the 1G-exposed counterpart. Proteins with <em>p</em>-values of ≤0.05 were further evaluated for increased or decreased abundance using a cut-off value of ≥ ± 1.5-fold change (log2 fold change of ≥ ± 0.584).</p></section></section><section id="S2.SS5"><h3 class="pmc_sec_title">Protein Identification and Bioinformatics Analysis</h3>
<p>Protein identifications and functional insights were obtained from searching for sequence homologs using OmicsBox v. 1.4.11 (BioBam Bioinformatics S.L). To characterize proteins with respect to the biological process they are involved in, Gene Ontology (GO) terms were assigned to domains. The sequences were blasted (cloud BLASTP 2.10.0+, <em>E</em>-Value 1.0E-3, Filter: Fungi), and the blast hits were mapped and annotated with GOs using the GO database (Goa version 2019_11<sup><a class="usa-link" href="#footnote1">1</a></sup>; <em>E</em>-Value 1.0E-6, Filter GO by Taxonomy: taxa: 4751, Fungi) (<a aria-describedby="B32" class="usa-link" href="#B32">Götz et al., 2008</a>). GOs were additionally assigned using InterProScan (IPS) to retrieve domains/motif information in a sequence-wise manner, and EggNOG using precomputed EggNOG-based orthology assignments. Corresponding GOs were then transferred to the sequences and merged with already existing GO terms. The annotations were validated based on the True-Path-Rule by removing all redundant terms to a given sequence. A GO-Slim analysis was run to summarize the GO annotation using the <em>Aspergillus</em> slim. Existing GO terms were additionally mapped to enzymes codes, when possible. A pathway analysis was performed to retrieve metabolic pathways based on the GO terms and the enzyme codes using the Load KEGG Pathway tool (<a aria-describedby="B45" class="usa-link" href="#B45">Kanehisa and Goto, 2000</a>). To elucidate the identity of the uncharacterized proteins, a search for homology was further performed in the UniProtKB database<sup><a class="usa-link" href="#footnote2">2</a></sup> (BLASTP parameters: E-Threshold: 10; matrix BLOSUM62). In cases of blast results where the most significant match was represented by an uncharacterized protein, the first match in the list of homologous proteins where a protein ID was available was considered. Information about the predicted protein localization was obtained using BUSCA<sup><a class="usa-link" href="#footnote3">3</a></sup>, based on the identification of signal and transit peptides, GPI-anchors and alpha-helical and beta-stranded transmembrane domains (<a aria-describedby="B81" class="usa-link" href="#B81">Savojardo et al., 2018</a>). Protein-protein interaction analyses were performed using STRING v11.0 with high confidence (0.70) (<a aria-describedby="B42" class="usa-link" href="#B42">Jensen et al., 2009</a>), selecting the proteome of the black yeast <em>Exophiala dermatitidis</em> as reference database based on its phylogenetic proximity to <em>K. chersonesos</em> (<a aria-describedby="B104" class="usa-link" href="#B104">Tesei et al., 2020</a>).</p></section></section><section id="S3"><h2 class="pmc_sec_title">Results</h2>
<section id="S3.SS1"><h3 class="pmc_sec_title">Growth Behavior</h3>
<p>Cell concentration in the LSSMG-exposed and unexposed samples was measured via hemocytometer at four different time points: (1) start (seed, day 0), (2) acceleration phase (in between lag and exponential phases, day 3), (3) exponential phase (day 5) and (4) at the end of the experiment (stationary phase, day 7). In <em>K. chersonesos</em> Wt<sub>LSSMG</sub>, ∼23.5-fold increase in cell concentration was observed during acceleration phase when compared to the original inoculum (2 × 10<sup>5</sup> cells/mL). Such an increase during acceleration phase was also noticed in <em>K. chersonesos</em> Mut<sub>LSSMG</sub> (10-fold), <em>K. chersonesos</em> Wt<sub>1</sub><sub>G</sub> (22-fold) and <em>K. chersonesos</em> Mut<sub>1</sub><sub>G</sub> (19-fold). During stationary phase the increase in cell concentration was 28-fold (<em>K. chersonesos</em> Wt<sub>LSSMG</sub>), 30-fold (<em>K. chersonesos</em> Mut<sub>LSSMG</sub>), 42-fold (<em>K. chersonesos</em> Wt<sub>1</sub><sub>G</sub>), and 36-fold (<em>K. chersonesos</em> Mut<sub>1</sub><sub>G</sub>) when compared to original inoculum. Among LSSMG and Earth gravity grown cultures, higher values were recorded at normal gravity condition. Overall, only ∼one log growth in 7 days might be due to the clumping nature of <em>Knufia</em> cells. Cell survivability in simulated microgravity was measured by CFU using ImageJ software (<a aria-describedby="B83" class="usa-link" href="#B83">Schneider et al., 2012</a>) according to established protocol (<a aria-describedby="B17" class="usa-link" href="#B17">Choudhry, 2016</a>) and were: <em>K. chersonesos</em> Wt<sub>LSSMG</sub> = 2.8 × 10<sup>6</sup> cell/mL, <em>K. chersonesos</em> Mut<sub>LSSMG</sub> 2.5 × 10<sup>6</sup> cell/mL, <em>K. chersonesos</em> Wt<sub>1</sub><sub>G</sub> 2.6 × 10<sup>6</sup> cell/mL, <em>K. chersonesos</em> Mut<sub>1</sub><sub>G</sub> 2.3 × 10<sup>6</sup> cell/mL (<a class="usa-link" href="#TS1">Supplementary Table 1</a>). Potential mechanical damages or cell lysis caused by cell separation via mild bead beating (i.e., ribolyzer, 3 × 20 s, speed 4) prior to cell count are to be excluded since optical microscope observations and tests involving repeated bead beater treatments indicated that it does not decrease cell viability. However, underestimation of cell survivability by CFU due to cell clumping, an inherent characteristic of black fungi, should be taken into account.</p>
<p>Under LSMMG conditions, fungal cells showed an increased clumping, resulting in the aggregation of cells to form a mycelial growth in the center of the vessel. However, growth morphology under Earth gravity resembled a biofilm of cells adhering to the vessel’s oxygenation membrane (<a class="usa-link" href="#F1">Figure 1C</a>).</p></section><section id="S3.SS2"><h3 class="pmc_sec_title">Microscopy</h3>
<p>To evaluate the presence of LSSMG-dependent alterations of fungal cell morphology in the melanotic (Wt) and non-melanized spontaneous mutant (Mut) strain, we examined cellular differentiation by growing cells at low density. In order to analyze single cellular morphology of the Wt and Mut strain before LSSMG-treatment, exponentially growing fungal cells were spotted on top of aluminum coupons and visualized using FE-SEM. The observation of the control fungal cell morphology in non-coated FE-SEM specimens revealed that the Wt produced biconcave cells, pseudohyphae and hyphae, while the mutant showed smooth spherical cells, pseudohyphae and cell aggregates (<a class="usa-link" href="#F2">Figures 2</a>, <a class="usa-link" href="#F3">3</a>).</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8012687_fgene-12-638708-g002.jpg" target="_blank"><img alt="FIGURE 2" class="graphic zoom-in" height="845" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3759/8012687/32e222529df3/fgene-12-638708-g002.jpg" width="758"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Field emission scanning electron microscopy (FE-SEM) micrographs of <em>Knufia chersonesos</em> Wt and Mut on aluminum coupons during a 7-days exposure to LSSMG or 1G (control). Arrows indicate hyphal growth in the wild type and dense cell aggregation in the mutant strain.</p></figcaption></figure><figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8012687_fgene-12-638708-g003.jpg" target="_blank"><img alt="FIGURE 3" class="graphic zoom-in" height="849" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3759/8012687/98ef51a34b8b/fgene-12-638708-g003.jpg" width="758"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>FE-SEM micrographs of <em>Knufia chersonesos</em> Wt and Mut on aluminum coupons during a 7-days exposure to LSSMG or 1G (control). Details of cells morphology are shown; arrows indicate biconcave cell morphology.</p></figcaption></figure><p>Low-shear simulated microgravity-exposed and 1G-exposed fungal cells were collected at various time points, as described in the section “Materials and Methods.” A detailed examination of fungal cellular morphology in non-coated FE-SEM specimens revealed 1G and LSSMG Wt and mutant strains showing similar morphology at day 1 and 3. Remarkably, the 1G Wt hyphae formation was delayed until day 7, whereas LSSMG induced hyphae formation at day 5 (<a class="usa-link" href="#F2">Figure 2</a>). Notably, the mutant showed no evidence of hyphae formation in LSSMG condition, but the aggregation of cells was evident at day 3 and increased until day 7, similarly to what was observed in the 1G cells (<a class="usa-link" href="#F2">Figure 2</a>). Furthermore, a closer observation of Wt and mutant fungal cells in LSSMG and 1G condition at 2500× magnification clearly shows that the Wt cells form biconcave morphology compared to the mutant cells (<a class="usa-link" href="#F3">Figure 3</a>). LSSMG treatment led to no significant change in cell size relative to 1G conditions, in either strain (<a class="usa-link" href="#TS1">Supplementary Table 2</a>). Similarly, images of PI-SYTO 9 and WGA staining generated by confocal microscopy did not reveal changes in cell viability and integrity in response to LSSMG exposure. Greatest fluorescence was seen with WGA at the beginning of the cultivation, although increasing concentrations of the dye were applied; this is most probably due to the thickening of the cell wall and increase of melanization, which naturally occurs in the fungus over the course of cultivation (<a class="usa-link" href="#TS1">Supplementary Figure 1</a>). Collectively, these findings uncover that the LSSMG and 1G conditions did not influence the fungal cellular growth and morphology.</p></section><section id="S3.SS3"><h3 class="pmc_sec_title">Overview of Whole-Cell Proteome and Secretome Analysis</h3>
<p>The 16 samples, comprised of four experimental conditions with two biological replicates each for each of the analyzed strains, yielded 3777 proteins as detected by isobaric TMT labeling-based LC-MS/MS. Of these, 3177 were in the whole-cell proteome and 600 in the secreted fraction. Furthermore, 2602 proteins were unique to the proteome and 25 to the secretome, while 575 were found to overlap (<a class="usa-link" href="#F4">Figure 4A</a>). A total of 3163 were identified by homology search. An overview of the biological and molecular functions of the detected proteins was obtained through an annotation statistics analysis. Distribution of the GO terms for all 3 categories (i.e., biological process BP, molecular function MF, and cellular component CC), with the highest number of associated protein sequences in the whole-cell proteome and the secretome, is displayed in <a class="usa-link" href="#TS1">Supplementary Figures 2A,B</a>. Several GOs were common to proteome and secretome, whereas others were instead unique to each set of proteins. GOs for different types of cell metabolic processes were the most represented in both secretory and intracellular proteins, whereas BPs such as response to chemical, cell cycle, and negative regulation of cellular processes were found exclusively within the latter. Conversely, proteins involved in cell communication and positive regulation of cellular process could solely be detected in the secreted fraction of the proteome. Proteins with hydrolase activity (MF GO:0016787) represented the most abundant group in the secretome and second most abundant in the whole-cell proteome, immediately followed by proteins with organic cyclic and heterocyclic compound binding, oxidoreductase and transferase activity. In both sets of samples, proteins with transcription and translation regulation activity were instead detected as the smallest groups. A distribution analysis for cellular component GO terms indicated a prevalence of intracellular, membrane, and cell wall proteins (i.e., CC GO: envelope) also in the culture supernatant—i.e., the ratio of total proteins with predicted extracellular to intracellular localization was approximately 25%:75% in the wild type and 6%:94% in the mutant—where the detection of cytoplasmic proteins is possibly due to mechanical stress and cell death (<a aria-describedby="B55" class="usa-link" href="#B55">Miura and Ueda, 2018</a>). Nonetheless, the presence of these proteins in the secreted fraction could also depend on active secretion of proteins with no predicted signal peptide, i.e., non-classical protein secretion, which has been recognized in various organisms including fungi, bacteria and plants based on the large number of leaderless proteins detected in extracellular compartments, including extracellular vesicles (EVs) (<a aria-describedby="B75" class="usa-link" href="#B75">Regente et al., 2012</a>; <a aria-describedby="B108" class="usa-link" href="#B108">Vallejo et al., 2012</a>; <a aria-describedby="B99" class="usa-link" href="#B99">Sun and Su, 2019</a>). The 25 proteins uniquely detected in the secretome encompassed a number of carboxylic ester hydrolases with predicted extracellular localization alongside cytoplasmic, cell wall– and plasma membrane-associated proteins (<a class="usa-link" href="#TS1">Supplementary Table 3</a>).</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8012687_fgene-12-638708-g004.jpg" target="_blank"><img alt="FIGURE 4" class="graphic zoom-in" height="748" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3759/8012687/2e0abf766e52/fgene-12-638708-g004.jpg" width="770"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Proteins identified and detected as regulated in the whole-cell proteome and secretome of <em>Knufia chersonesos</em> Wt and Mut following exposure to LSSMG. <strong>(A)</strong> Overlap of proteins between whole-cell proteome and secretome, identified in two separate TMT LC/MS experiments. <strong>(B)</strong> Number of differentially abundant proteins for each strain under LSSMG compared to 1G (control) (fold change of ≥ ± 1.5, <em>p</em> ≤ 0.05). <strong>(C)</strong> Heat map hierarchical clustering of co-varying proteins quantified in each strain. The color key indicates the values that represented the protein abundance ratio (log2) and the number of proteins counted in each value. A total of 6 runs – 3 technical replicates and 2 biological replicates – were performed for each sample. Wt, wild type; Mut, mutant; w-cp, whole-cell proteome; s, secretome. Heatmaps were generated using R.</p></figcaption></figure><p>In order to detect significant rearrangements in the protein pool upon exposure to LSSMG, a quantitative analysis was carried out by comparing experimental groups. The clustering of the samples, biological and technical replicates included, was confirmed by Hierarchical Cluster (HC) analysis (<a class="usa-link" href="#F4">Figure 4C</a>). However, one of the LSSMG-exposed secretomes from <em>K. chersonesos</em> Wt appeared to deviate from the biological duplicate of the same condition. The number of differentially abundant proteins for each strain is summarized in <a class="usa-link" href="#F4">Figure 4</a> (fold change of ≥ ± 1.5, <em>p</em> ≤ 0.05), where (B) shows the count of regulated proteins following direct comparison of treatment and control whole-cell proteome and secretome samples. The mutant proteome contained the largest number of modulated proteins (115) compared to the mutant secretome (71), the wild type proteome (97), and the wild type secretome (11).</p></section><section id="S3.SS4"><h3 class="pmc_sec_title">Effects of LSSMG on <em>Knufia chersonesos</em> Whole-Cell Proteome</h3>
<p>The proteomic quantitative analysis of LSSMG-exposed <em>K. chersonesos</em> Wt and Mut revealed 54-up and 43-down and 45-up and 70-down regulated proteins, respectively, when compared to the 1G-exposed counterparts (fold change of ≥ ± 1.5, <em>p</em> ≤ 0.05) (<a class="usa-link" href="#TS1">Supplementary Tables 4</a>, <a class="usa-link" href="#TS1">5</a>). Distribution of over-represented BP GO terms among differentially expressed proteins is displayed in <a class="usa-link" href="#F5">Figure 5A</a>. Most significantly upregulated biological processes included “lipid and carbohydrate metabolism” in both strains (24 to 28% of all upregulated proteins). However, 15% of downregulated proteins were also in the carbohydrate metabolism category. By contrast, proteins involved in lipid metabolic processes were not found to be decreased. Further, “transcription” (12%) and “transport” (27%) were additional highly represented categories of upregulated proteins in the wild type, whereas “transport” (28%) and “amino acid metabolic process” (16%) were well represented categories of upregulated proteins in the mutant.</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8012687_fgene-12-638708-g005.jpg" target="_blank"><img alt="FIGURE 5" class="graphic zoom-in" height="922" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3759/8012687/d71252832a62/fgene-12-638708-g005.jpg" width="785"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/F5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Biological processes GOs categories of differentially expressed proteins in <em>Knufia chersonesos</em> under LSSMG. <strong>(A)</strong> Whole-cell proteome <strong>(B)</strong> Secretome. Proteins with changed abundance (FC ≥ ± 1.5, <em>p</em> ≤ 0.05) were annotated with terms representing various biological processes using OmicsBox v. 1.4.11 (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.biobam.com" rel="noopener noreferrer" target="_blank">https://www.biobam.com</a>). GO terms were thereafter summarized using sequence distribution/GO multilevel pie charts (Filtered by sequence count, Cutoff = 2).</p></figcaption></figure><p>Proteins involved in lipid metabolism included CF317_002955-T1/C7ZGD6_NECH7—predicted to be phospholipase A2—an acetyl-CoA desaturase (CF317_004579-T1/A0A072PJ62_9EURO) and the uncharacterized protein CF317_000532-T1/A0A0D2C8U2_9EURO upregulated in both strains, along with an inositol-3-phosphate synthase (CF317_006213-T1/A0A0D2GI50_9EURO), which was instead solely observed in the mutant. These proteins participate in cellular pathways for the biosynthesis of unsaturated fatty acids (KEGG pathway 01040) and the metabolism of phospho- and glycerophospholipids (KEGG 00564), inositol phosphate (KEGG 00562), arachidonic and linoleic acid, among others (<a class="usa-link" href="#T1">Table 1</a>; <a aria-describedby="B71" class="usa-link" href="#B71">Passoth, 2017</a>). Pyruvate metabolism (KEGG 00620), glycolysis/gluconeogenesis (KEGG 00010) and starch and sucrose metabolism, were possibly also involved in the response to LSSMG, as revealed by the KEGG pathway analysis based on the increased abundance of proteins like alcohol dehydrogenase 1 (CF317_005767-T1/A0A0N0NIW7_9EURO) and alpha/beta-glucosidase agdC (CF317_006579-T1/A0A178BWA1_9EURO) in the wild type. Glutamine synthetase and protein FYV10, the first involved in glyoxylate and dicarboxylate metabolism (KEGG 00630) and in nitrogen metabolism (<a aria-describedby="B119" class="usa-link" href="#B119">Zhang et al., 2017</a>) and the latter mediating the degradation of enzymes of the gluconeogenesis pathway (<a aria-describedby="B13" class="usa-link" href="#B13">Braun et al., 2011</a>), were instead detected as upregulated in the mutant. Decreased levels of a number of hydrolytic enzymes destined for secretion were observed in the LSSMG-exposed proteomes for the carboxylic ester hydrolases CF317_0002308-T1/A0A1J9RJA8_9PEZI, CF317_007618-T1/A0A6A6HNE7_9PEZI, CF317_007621-T1/A0A0D2AG04_9PEZI (in Wt) and CF317_002086-T1/A0A0D2BHT9_9EURO (in Mut). The same was observed for the cell wall–degradation enzymes endo- and extracellular glucanases CF317_0009683-T1/W9ZBZ7_9EURO and CF317_009779-T1/H6BQE2_EXODN (in Wt) and the 3,2-<em>trans-</em>enoyl-CoA isomerase CF317_000932-T1/A0A1C1D1N7_9EURO (in Mut), the latter of which is involved in the metabolism of unsaturated fatty acids in beta oxidation (<a aria-describedby="B41" class="usa-link" href="#B41">Janssen et al., 1994</a>).</p>
<section class="tw xbox font-sm" id="T1"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Most abundant biological process GO categories regulated under LSSMG in <em>Knufia chersonesos</em> Wt and Mut whole-cell proteome. Differentially expressed proteins included in each category are also shown.</p></div>
<div class="tbl-box p" tabindex="0"><table cellpadding="5" cellspacing="5" class="content" frame="hsides" rules="groups">
<thead><tr>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>Protein</em> accession No.<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">Putative Protein function</td>
<td align="center" colspan="1" rowspan="1" valign="top">Protein relative abundance*</td>
<td align="center" colspan="1" rowspan="1" valign="top">
<em>p</em>-value</td>
</tr></thead>
<tbody>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><strong>Carbohydrate and lipid metabolic process</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top"><em>Wild type</em></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003648-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">GP-PDE domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.1133</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.79E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002955-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Phospholipase A2</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.1032</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.26E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_005767-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Alcohol dehydrogenase 1</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.8542</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.37E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006579-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Alpha/beta-glucosidase agdC</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.8177</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.11E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009031-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Carboxylic ester hydrolase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.8024</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.22E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000532-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Uncharacterized protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.775</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.78E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007434-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Serine/threonine-protein kinase ppk6</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6797</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.44E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004579-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Acyl-CoA desaturase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6334</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.95E-05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_0002308-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Carboxylic ester hydrolase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6053</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.47E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_0009683-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Endo-1,3(4)-beta-glucanase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6918</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.10E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007618-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Carboxylic ester hydrolase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7365</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.22E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009779-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Extracellular cell wall glucanase Crf1/allergen Asp F9</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.8811</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.70E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009003-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Serine/threonine-protein kinase TOR</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.9558</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.75E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007621-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Cutinase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.0366</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.69E-04</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Mutant</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009031-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Carboxylic ester hydrolase</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.1163</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.22E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007501-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glutamine synthetase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.9516</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.84E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006213-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Inositol-3-phosphate synthase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.8851</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.34E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004579-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Acyl-CoA desaturase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.8258</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.95E-05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000532-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Uncharacterized protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.7363</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.78E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007133-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Protein FYV10</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6298</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.26E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002086-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">AB hydrolase-1 domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6016</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.98E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009017-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">NodB homology domain-containing prote<strong>i</strong>n</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6135</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.38E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_005813-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">4HBT domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6213</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.10E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001523-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glucose-6-phosphate 1-epimerase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6226</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.42E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000932-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">3,2-<em>trans-</em>enoyl-CoA isomerase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6255</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.20E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009683-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Endo-1,3(4)-beta-glucanase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.0502</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.10E-02</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><strong>Transport</strong></td></tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Wild type</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004246-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Phosphate transporter</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.3345</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.85E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007988-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">VPS37 C-terminal domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.8635</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.08E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_0005675-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Mitochondrial thiamine pyrophosphate carrier 1</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.7750</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.05E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001330-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Phosphate transporter</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.7548</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.35E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001709-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">WD_REPEATS_REGION domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.7277</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.37E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003773-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Zinc-regulated transporter 1</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6633</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.42E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009191-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Choline transport protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6590</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.34E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007500-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ammonium transporter</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6548</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.80E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008807-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">HemS domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.5860</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.75E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_0006744-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ribosomal protein L37e</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6649</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.96E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_0001542-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">POT family proton-dependent oligopeptide transporter</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6830</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.37E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_0009683-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Endo-1,3(4)-beta-glucanase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6918</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.10E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001911-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">MFS transporter, SP family, major inositol transporter</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.8427</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.84E-02</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Mutant</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007500-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ammonium transporter</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.6152</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.80E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009191-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Choline transport protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.2684</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.34E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_005369-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">AA_permease domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.9504</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.30E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003222-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">CNT family concentrative nucleoside transporter</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.7381</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.62E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008807-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">HemS domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.7308</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.75E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002497-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">SEC7 domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.7191</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.38E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009070-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Stress response protein NST1</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6604</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.86E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002132-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">AA_permease domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6505</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.55E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001911-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">MFS transporter, SP family, major inositol transporter</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7301</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.84E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001542-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">POT family proton-dependent oligopeptide transporter</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.8932</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.37E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009683-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Endo-1,3(4)-beta-glucanase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.0502</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.10E-02</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><strong>Transcription, RNA and cellular amino acid metabolic processes</strong></td></tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Wild type</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004755-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glutamate dehydrogenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.7627</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.54E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004246-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Phosphate transporter</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.3345</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.85E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008227-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Alkaline phosphatase</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.2107</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.76E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003892-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Alkaline phosphatase</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.0475</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.25E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_005207-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ribonuclease T1</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.9813</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.38E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_005767-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Alcohol dehydrogenase 1</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.8542</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.37E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004551-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Carboxypeptidase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6315</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.66E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009661-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Fungal_<em>trans</em> domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.5936</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.26E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009392-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Fungal_<em>trans</em> domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6382</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.93E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_0006744-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ribosomal protein L37e</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6649</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.96E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000112-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L34-B</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7009</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.84E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000184-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Large subunit ribosomal protein L24e</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7218</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.77E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003175-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Poly(A) polymerase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7450</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.79E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006402-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">4-hydroxyphenylpyruvate dioxygenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7730</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.57E-04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009779-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Extracellular cell wall glucanase Crf1/allergen Asp F9</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.8811</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.70E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009003-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Serine/threonine-protein kinase TOR</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.9558</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.75E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002178-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Indoleamine 2,3-dioxygenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.1838</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.88E-04</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Mutant</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004755-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glutamate dehydrogenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.9810</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.54E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007501-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glutamine synthetase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.9516</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.84E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001165-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Malic enzyme</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6220</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.41E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004816-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">D-3-phosphoglycerate dehydrogenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6044</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.51E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002693-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ornithine transcarbamylase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.5899</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.51E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000519-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Arginase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6439</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.12E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006712-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Histidinol dehydrogenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6459</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.39E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008030-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">D-amino-acid oxidase domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6799</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.92E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008924-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Homogentisate 1,2-dioxygenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6970</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.01E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007970-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Multifunctional fusion protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7153</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.07E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000649-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Delta(3,5)-Delta(2,4)-dienoyl-CoA isomerase, mitochondrial</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7416</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.03E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007412-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Histidinol-phosphate aminotransferase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7514</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.09E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003892-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Alkaline phosphatase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.8825</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.25E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007070-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Multifunctional fusion protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.9090</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.03E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003175-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Poly(A) polymerase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.9882</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.79E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006402-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">4-hydroxyphenylpyruvate dioxygenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.1274</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.57E-04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002178-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Indoleamine 2,3-dioxygenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.4328</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.88E-04</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><strong>Response to stress and chemical</strong></td></tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Wild type</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004755-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glutamate dehydrogenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.7627</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.54E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006388-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Nitric oxide dioxygenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.9459</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.16E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_005767-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Alcohol dehydrogenase 1</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.8542</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.37E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003242-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Catalase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6330</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.85E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009003-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Serine/threonine-protein kinase TOR</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.9558</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.75E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007018-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">3-phytase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.9947</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.67E-04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000687-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Catalase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.1410</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.34E-02</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Mutant</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004755-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glutamate dehydrogenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.9810</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.54E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006388-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Nitric oxide dioxygenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.7290</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.16E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004909-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Adenylosuccinate synthetase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6917</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.11E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007970-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Multifunctional fusion protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7153</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.07E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008612-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Aldedh domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7312</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.05E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007412-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Histidinol-phosphate aminotransferase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7514</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.09E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008426-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Glutathione reductase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.7680</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.19E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007018-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">3-phytase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.8087</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.67E-04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000687-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Catalase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.8987</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.34E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008773-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Catalase-peroxidase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.9459</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.42E-02</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><strong>Proteolysis</strong></td></tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Wild type</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008840-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Carboxypeptidase</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.0950</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.46E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004157-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Peptidase_M14 domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.8180</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.52E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008998-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Putative fumarylacetoacetate hydrolase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.7082</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.54E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000103-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Zinc carboxypeptidase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6512</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.88E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004551-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Carboxypeptidase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.6315</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.66E-01</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><strong>Cytoskeleton and organelle organization</strong></td></tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Wild type</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009683-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Endo-1,3(4)-beta-glucanase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.6918</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.10E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_009003-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Serine/threonine-protein kinase TOR</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.9558</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.75E-02</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Mutant</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008503-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Thiamine thiazole synthase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.8255</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.30E-01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007856-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">WD_REPEATS_REGION domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.7141</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.88E-02</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib1"><p><em>*Log2 fold change of SMG-exposed compared to unexposed proteome (<em>p</em> ≤ 0.05). <sup>a</sup>Protein accession number in the <em>K. chersonesos</em> database of <em>ab initio</em> translated proteins.</em></p></div></div></section><p>A differential abundance of proteins involved in transport was also observed (<a class="usa-link" href="#T1">Table 1</a>). Transmembrane ammonium (CF317_007500-T1/A0A072PWW7_9EURO) and iron transporters (i.e., HemS domain-containing protein CF317_008807-T1/L7HNA7_MAGOY) and the choline transport protein (CF317_009191-T1/A0A2P8A4S9_9PEZI) were present in the LSSMG-exposed whole-cell proteome of both strains, but they were more increased in the mutant (over 3-, 2. 4-, and 1.5-folds, respectively) than in the wild type (slightly over 1.5-folds). Additionally, two AA permease 2 domain-containing proteins (CF317_005369-T1/A0A0D2FEL3_9EURO and CF317_002132-T1/A0A438N399_EXOME) responsible for transmembrane amino acid transport were also upregulated in the mutant, whereas proteins with a predicted role in vesicle-mediated transport—i.e., VPS37 C-terminal domain-containing protein (CF317_007988-T1/A0A0D2CPQ3_9EURO) and WD_REPEATS_REGION domain-containing protein (CF317_001709-T1/A0A438MTR2_EXOME)—were exclusively detected in the wild type counterpart. Downregulated proteins included POT family proton-dependent oligopeptide transporter (CF317_001542-T1/A0A072P870_9EURO) and major facilitator superfamily transporters (MFS) (CF317_001911-T1/W9Z1V7_9EURO), involved in transportation of substrate molecules, including sugars, drugs, metabolites, amino acids, vitamins, and both organic and inorganic ions, or small peptides (<a aria-describedby="B70" class="usa-link" href="#B70">Pao and Paulsen, 1998</a>). Together with ABC transporters, MFS transporters are often detected among the cell wall components undergoing changes in response to the shift from normal gravity to microgravity (<a aria-describedby="B79" class="usa-link" href="#B79">Sathishkumar et al., 2016</a>).</p>
<p>Low-shear simulated microgravity additionally triggered upregulation of some proteins involved in cellular amino acid metabolic processes and downregulation of others (<a class="usa-link" href="#T1">Table 1</a>). Glutamate dehydrogenase was threefold and fourfold upregulated in <em>K. chersonesos</em> Wt and Mut, respectively. Together with the glutamine synthase, whose levels were also found to be increased in the mutant, the enzyme is reportedly involved in the primary nitrogen metabolism and in the biosynthesis and metabolism of several amino acids (i.e., arginine biosynthesis, glutamine, alanine and aspartate metabolism) (<a aria-describedby="B54" class="usa-link" href="#B54">Meti et al., 2011</a>). Protein D-3-phosphoglycerate dehydrogenase (CF317_004816-T1/A0A0D2CHE2_9EURO), participating in cysteine, methionine, glycine, serine, and threonine metabolism, was also more enriched in LSSMG-exposed mutant samples. However, a higher number of enzymes involved in cellular amino acid metabolic processes was found to be decreased under LSSMG. Nine were downregulated exclusively in the mutant, while protein CF317_006402-T1 was observed also in the wild type. According to the KEGG analysis, some of these proteins are involved with more than one pathway: 4 proteins are implicated in arginine and proline metabolism (CF317_000519-T1, CF317_007970-T1, CF317_007070-T1, CF317_008030-T1), 3 in tyrosine metabolism (CF317_007412-T1, CF317_006402-T1, CF317_008924-T1), and 2 in histidine metabolism (CF317_007412-T1, CF317_006712-T1), arginine biosynthesis (CF317_002693-T1, CF317_000519-T1) and phenylalanine biosynthesis and metabolism (CF317_007412-T1, CF317_006402-T1). The putative indoleamine 2,3-dioxygenase, involved instead in tryptophan catabolic process to the NAD precursor kynurenine, represented the most decreased protein in both strains, with an over 2-fold downregulation.</p>
<p>Some proteins falling into the GO category “DNA-templated transcription” were enriched exclusively in the LSSMG-exposed wild type proteome. They were mostly phosphatases—known to also regulate pathways important for stress (<a aria-describedby="B89" class="usa-link" href="#B89">Serra-Cardona et al., 2015</a>)—having abundances nearly 2.5-fold higher than in the control samples (i.e., Alkaline phosphatase CF317_008227-T1/A0A0D2BGD8_9EURO and CF317_003892-T1/A0A0D2BGD8_9EURO) (<a class="usa-link" href="#T1">Table 1</a>). Within the same category, downregulation was observed for a putative serine/threonine-protein kinase TOR and two <em>trans</em> domain-containing proteins involved in DNA binding and transcription. Decreased levels of structural protein constituents of the large ribosomal subunit—i.e., ribosomal protein L37e, L34-B and L24e—were instead detected upon LSSMG exposure exclusively in the wild type, alongside the upregulation of the ribonuclease T1 (CF317_005207-T1/W9XBV2_9EURO) involved in RNA degradation. Also significantly decreased was poly(A)polymerase (CF317_003175-T1/A0A1C1CN69_9EURO), an important component in mRNA synthesis, responsible for the addition of the 3′ polyadenine tail to a newly synthesized pre-messenger RNA.</p>
<p>In the group of proteins involved in response to stress and chemical, most of the modulated proteins were decreased by LSSMG exposure (<a class="usa-link" href="#T1">Table 1</a>). Downregulated stress response proteins included catalases (i.e., CF317_000687-T1/R8BNJ4_TOGMI, levels 2.7- and 1.9-fold lower than in the unexposed wild type and mutant proteome, respectively; CF317_003242-T1/R4XE87_TAPDE and CF317_008773-T1/U1G9G0_ENDPU), notably involved in the response to oxidative stress, and the 3-phytase CF317_007018-T1/A0A0D2IHR9_9EURO—a phosphatase enzyme with a predicted role in counteracting phosphate deficiency and osmotic stress (<a aria-describedby="B4" class="usa-link" href="#B4">Belgaroui et al., 2018</a>)—in both strains. Further, the glutathione-recycling enzyme glutathione reductase (CF317_008426-T1/A0A0D1ZJU8_9EURO) (<a aria-describedby="B18" class="usa-link" href="#B18">Couto et al., 2016</a>), the histidinol-phosphate aminotransferase (CF317_007412-T1/H6BQ73_EXODN) and the adenylosuccinate synthetase (ASS; F317_004909-T1/W9W243_9EURO), showed lower levels in the mutant under LSSMG conditions. The latter, best known as the first enzyme in the <em>de novo</em> synthesis of AMP from inosine-5′-monophosphate (IMP), is responsive to salt stress, as reported in plants (<a aria-describedby="B121" class="usa-link" href="#B121">Zhao et al., 2013</a>). Among the proteins with increased abundance in both fungal strains, was glutamate dehydrogenase (CF317_004755-T1), involved in amino acid metabolism and for which a stress-dependent regulation has been demonstrated (e.g., abiotic stress generated ROS) (<a aria-describedby="B94" class="usa-link" href="#B94">Skopelitis et al., 2006</a>). Further, the putative cytosolic nitric oxide dioxygenase (CF317_006388-T1/S7Z8A1_PENO1)—reported to play a role in nitric oxide detoxification mechanisms by the conversion of this signaling molecule to nitrate (<a aria-describedby="B14" class="usa-link" href="#B14">Cánovas et al., 2016</a>)—was present in the wild type and mutant LSSMG-exposed proteomes at levels around 2-fold higher than that of the unexposed ones. Along with the above-mentioned proteins, a number of enzymes, specifically peptidases involved in proteolysis, were found to be upregulated exclusively in <em>K. chersonesos</em> wild type (<a class="usa-link" href="#T1">Table 1</a>). The putative carboxypeptidase CF317_008840-T1/W9XS38_9EURO and CF317_004157/W2S3B3_9EURO; and CF317_000103/A0A6A6DDQ8_9PEZI and CF317_004551/A0A0D2GVY9_9EURO, known to catalyze reactions that are important to various physiological processes, such as the cell cycle, cell growth and differentiation, apoptosis, and stress response (<a aria-describedby="B62" class="usa-link" href="#B62">Neto et al., 2018</a>), were 2-fold and 1.5-fold upregulated, respectively. Lastly, proteins involved in cytoskeleton and organelle organization showed opposite regulation in the two <em>K. chersonesos</em> strains: The first was decreased in the wild type, whereas the latter was upregulated in the mutant (<a class="usa-link" href="#T1">Table 1</a>).</p></section><section id="S3.SS5"><h3 class="pmc_sec_title">Effects of LSSMG on <em>Knufia chersonesos</em> Secretome</h3>
<p>The proteomic characterization of <em>K. chersonesos</em> secretome upon exposure to LSSMG for 7 days revealed solely 9-up and 2-down regulated proteins in the wild type and 23-up and 48-down regulated proteins in the mutant, when compared to the 1G-exposed counterparts (fold-change of ≥ ± 1.5, <em>p</em> ≤ 0.05) (<a class="usa-link" href="#F4">Figure 4B</a>). Among these proteins, the ratio of proteins with predicted extracellular to intracellular localization was approximately 20%:80% in both wild type and mutant (<a class="usa-link" href="#TS1">Supplementary Tables 6</a>, <a class="usa-link" href="#TS1">7</a>). Out of 14 proteins with predicted signal peptides in the mutant secretome, 12 were regulated following LSSMG-exposure. In the wild type secretome, 2 out of 61 proteins with predicted extracellular localization were regulated in response to microgravity.</p>
<p>The distribution of differentially expressed proteins among BP GO terms is presented in <a class="usa-link" href="#F5">Figure 5B</a>. In the wild type, the regulated proteins were prevalently involved in transport and metabolic processes, whereas in the mutant strain, regulated proteins were mostly involved with biological processes such as carbohydrate and lipid metabolism (15 proteins), response to chemical and stress (13 proteins), translation, and ribosome biogenesis and transcription (15 proteins). Interestingly, the majority of proteins involved in the above-mentioned GO BP categories exhibited downregulation in LSSMG-exposed secretomes of the mutant and exhibited upregulation in the wild type (<a class="usa-link" href="#T2">Table 2</a>). One protein exhibiting opposite regulation in the two analyzed strains, was the putative scytalone dehydratase CF317_002654-T1, homolog of <em>Phialophora attinorum</em> A0A0N1P280_9EURO. This enzyme, which is reportedly involved in the biosynthesis of dihydroxynaphthalene (DNH) melanin from endogenous substrate (<a aria-describedby="B23" class="usa-link" href="#B23">Eisenman and Casadevall, 2012</a>), was 1.5-fold increased in the wild type and 1.6-fold decreased in the melanin-deficient mutant. In the wild type, upregulated proteins also included the adenylosuccinate lyase CF317_004119-T1 and the M20_dimer domain-containing protein (Carboxypeptidase S) CF317_006825-T1, both implicated in amino acid (alanine, aspartate and glutamate) metabolism (KEGG pathway pae00250), and the plasma membrane ATPase CF317_002664-T1, partaking in oxidative phosphorylation (<a class="usa-link" href="#T2">Table 2</a>). Also in the wild type, the casein kinase I 1 CF317_004827-T1 and the histone H2A CF317_001633-T1, both involved in DNA repair (<a aria-describedby="B93" class="usa-link" href="#B93">Skoneczna et al., 2018</a>), were found in the LSSMG-exposed secretome at levels 1.7- and 1.6-fold higher than in the unexposed one. The two downregulated proteins were protein CF317_008131-T1, homolog of the arabinan endo-1,5-alpha-L-arabinosidase from <em>Aspergillus wentii DTO 134E9</em>, and protein CF317_002949-T1, homolog of the feruloyl esterase A0A177BY00_9PLEO from <em>Paraphaeosphaeria sporulosa</em>, both with a predicted extracellular localization and reportedly involved in polysaccharide degradation (<a aria-describedby="B104" class="usa-link" href="#B104">Tesei et al., 2020</a>).</p>
<section class="tw xbox font-sm" id="T2"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Most abundant biological process GO categories regulated under LSSMG in <em>Knufia chersonesos</em> Wt and Mut whole-cell secretome. Differentially expressed proteins included in each category are shown.</p></div>
<div class="tbl-box p" tabindex="0"><table cellpadding="5" cellspacing="5" class="content" frame="hsides" rules="groups">
<thead><tr>
<td align="left" colspan="1" rowspan="1" valign="top">
<em>Protein</em> accession No.<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1" valign="top">Putative Protein function</td>
<td align="center" colspan="1" rowspan="1" valign="top">Protein relative abundance*</td>
<td align="center" colspan="1" rowspan="1" valign="top">
<em>p</em>-value</td>
</tr></thead>
<tbody>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><strong>Carbohydrate and lipid metabolic processes</strong></td></tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Wild type</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004827-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Casein kinase I 1</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.656</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.63E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002949-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Feruloyl esterase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.859</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.16E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008131-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Arabinan endo-1,5-alpha-L-arabinosidase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.998</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.32E-02</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Mutant</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003266-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Malate dehydrogenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.244</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.48E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000614-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">1,3-beta-D-glucan-UDP glucosyltransferase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.893</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.26E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004588-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Alpha, alpha-trehalase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.813</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.42E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003037-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Acyl-CoA-dependent ceramide synthase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.797</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.16E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000059-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">PSDC domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.634</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.14E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004317-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">3′(2′),5′-bisphosphate nucleotidase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.650</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.52E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008979-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Epimerase domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.727</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.36E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003714-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Mannitol-1-phosphate 5-dehydrogenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.739</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.36E-05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008448-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Phosphotransferase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.790</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.34E-04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003513-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">2-phosphoglycerate dehydratase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.870</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.53E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006165-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Esterase/lipase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.917</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.78E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003202-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Concanavalin A-like lectin/glucanase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.918</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.79E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_005481-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">6-phosphogluconate dehydrogenase, decarboxylating</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.960</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.57E-04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004751-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Phosphoglycerate kinase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.143</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.27E-04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007096-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">6-phosphofructo-2-kinase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.494</td>
<td align="center" colspan="1" rowspan="1" valign="top">5.59E-03</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><strong>Transport</strong></td></tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Wild type</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001911-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">MFS transporter, SP family, major inositol transporter</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.747</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.92E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002664-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Plasma membrane ATPase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.702</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.25E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004827-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Casein kinase I 1</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.656</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.63E-02</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Mutant</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000614-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">1,3-beta-D-glucan-UDP glucosyltransferase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.893</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.26E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001401-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">t-SNARE coiled-coil homology domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.843</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.08E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004472-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Endoplasmic reticulum transmembrane protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.742</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.09E-04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006347-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Inorganic phosphate transport protein PHO88</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.588</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.10E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002531-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Putative inorganic phosphate transporter C8E4.01c</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.591</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.01E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007808-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">NTF2 domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.756</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.95E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008448-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Phosphotransferase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.790</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.34E-04</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><strong>Transcription, RNA and cellular amino acid metabolic processes</strong></td></tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Wild type</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001633-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Histone H2A</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.752</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.78E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002935-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Aromatic amino acid aminotransferase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.633</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.18E-02</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Mutant</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008030-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">D-amino-acid oxidase domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.783</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.10E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003734-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Ribosomal_L23eN domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.771</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.09E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007665-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Guanine nucleotide-binding protein subunit beta</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.775</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.98E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_005931-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Aspartate–tRNA ligase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.742</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.84E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004317-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">3′(2′),5′-bisphosphate nucleotidase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.650</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.52E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003885-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Putative RNA-binding protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.668</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.83E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001861-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">40S ribosomal protein S20</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.709</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.62E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008924-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Homogentisate 1,2-dioxygenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.730</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.76E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_005516-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S ribosomal protein L7</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.734</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.03E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008448-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Phosphotransferase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.790</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.34E-04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004501-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">40S ribosomal protein S24</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.796</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.53E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001633-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Histone H2A</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.084</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.78E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007960-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S acidic ribosomal protein P1</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.277</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.18E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008933-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Elongation factor EF-1 beta subunit</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.706</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.09E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000379-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Elongation factor EF-1 gamma subunit</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.838</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.56E-03</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><strong>Response to stress and chemical</strong></td></tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Wild type</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001633-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Histone H2A</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.752</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.78E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004827-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Casein kinase I 1</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.656</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.63E-02</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Mutant</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000051-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Plasma membrane phosphatase required for sodium stress response</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.906</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.73E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004588-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Alpha, alpha-trehalase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.813</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.42E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007665-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Guanine nucleotide-binding protein subunit beta</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.775</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.98E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006969-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">HRXXH domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.696</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.88E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004317-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">3′(2′),5′-bisphosphate nucleotidase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.650</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.52E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003570-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">S-formylglutathione hydrolase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.720</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.69E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008979-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Epimerase domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.727</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.36E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003714-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Mannitol-1-phosphate 5-dehydrogenase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.739</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.36E-05</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003513-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">2-phosphoglycerate dehydratase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.870</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.53E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_005481-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">6-phosphogluconate dehydrogenase, decarboxylating</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.960</td>
<td align="center" colspan="1" rowspan="1" valign="top">7.57E-04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000980-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Thioredoxin domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.983</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.08E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001633-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Histone H2A</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.084</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.78E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007147-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Nucleoside diphosphate kinase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.745</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.70E-04</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><strong>Biological processes and secondary metabolic process</strong></td></tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Wild type</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_001633-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Histone H2A</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.752</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.78E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_004119-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Adenylosuccinate lyase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.703</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.58E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006825-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">M20_dimer domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.588</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.22E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002654-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Scytalone dehydratase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.587</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.70E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002949-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Feruloyl esterase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.859</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.16E-03</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1" valign="top"><em>Mutant</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000614-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">1,3-beta-D-glucan-UDP glucosyltransferase</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.893</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.26E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006969-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">HRXXH domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">0.696</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.88E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003885-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Putative RNA-binding protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.668</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.83E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_002654-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Scytalone dehydratase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.679</td>
<td align="center" colspan="1" rowspan="1" valign="top">3.70E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006992-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Serine/threonine-protein kinase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.682</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.90E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_008448-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Phosphotransferase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.790</td>
<td align="center" colspan="1" rowspan="1" valign="top">4.34E-04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003513-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">2-phosphoglycerate dehydratase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.870</td>
<td align="center" colspan="1" rowspan="1" valign="top">1.53E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_000980-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Thioredoxin domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−0.983</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.08E-02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006154-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Pyruvate decarboxylase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.138</td>
<td align="center" colspan="1" rowspan="1" valign="top">6.87E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_003229-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">Putative versicolorin reductase</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.176</td>
<td align="center" colspan="1" rowspan="1" valign="top">2.61E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_007960-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">60S acidic ribosomal protein P1</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.277</td>
<td align="center" colspan="1" rowspan="1" valign="top">9.18E-03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1" valign="top">CF317_006677-T1</td>
<td align="left" colspan="1" rowspan="1" valign="top">HIT domain-containing protein</td>
<td align="center" colspan="1" rowspan="1" valign="top">−1.770</td>
<td align="center" colspan="1" rowspan="1" valign="top">8.90E-03</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/T2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="attrib2"><p><em>* Log2 fold change of SMG-exposed compared to unexposed proteome (<em>p</em> ≤ 0.05). <sup>a</sup>Protein accession number in the <em>K. chersonesos</em> database of <em>ab initio</em> translated proteins.</em></p></div></div></section><p>In the <em>K. chersonesos</em> mutant, LSSMG triggered the upregulation of just a low number of proteins. The malate dehydrogenase CF317_003266-T1/A0A1C1CNC5_9EURO, 1,3-beta-D-glucan-UDP glucosyltransferase CF317_000614-T1/A0A438MTW5_EXOME and alpha, alpha-trehalase CF317_004588-T1/H6C927_EXODN, involved in the metabolism of starch, sucrose, and sphingolipids, were 2. 4-, 1.9-, and 1.8-fold upregulated, respectively. Conversely, proteins with decreased abundance encompassed, among others, enzymes catalyzing different steps throughout the process of glycolysis/gluconeogenesis (KEGG pathway hsa00010)—such as the putative phosphotransferase CF317_008448/W2RU25_9EURO, the phosphoglycerate kinase CF317_004751/W2RJE3_9EURO and the 2-phosphoglycerate dehydratase CF317_003513/H6BNI5_EXODN—fructose and mannose (hsa00051), pyruvate (hsa00620) and glycerolipid metabolism (hsa00561), and the pentose phosphate pathway (hsa00030) (i.e., phosphotransferase, epimerase domain-containing protein CF317_008979 and 6-phosphogluconate dehydrogenase, decarboxylating CF317_005481). Further, the CF317_007096/A0A0U1M0E3_TALIS 6-phosphofructo-2-kinase, acting as an activator of the glycolysis/gluconeogenesis pathway (<a aria-describedby="B74" class="usa-link" href="#B74">Raben and Wolfgang, 2019</a>), was the most decreased protein in the GO category of carbohydrate and lipid metabolism, with a 3-fold downregulation.</p>
<p>Increased levels of proteins with a role in the response to stress and chemicals (<a class="usa-link" href="#T2">Table 2</a>) were observed in the exposed secretomes for the plasma membrane protein CF317_000051-T1/A0A178DA36_9EURO with reported phosphatase activity and the guanine nucleotide-binding protein subunit beta CF317_007665-T1/A0A0D2CV56_9EURO with a role in signal transduction, among others. However, as already observed for the whole-cell proteome, the majority of proteins involved in stress response showed decreased abundance upon exposure to LSSMG. The DNA-repair protein histone H2A (CF317_001633-T1), upregulated in the wild type secretome, was at least twofold downregulated in the mutant, together with the nucleoside diphosphate kinase CF317_007147-T1/A0A178C9G0_9EURO. The latter, ubiquitous enzyme involved in nucleotides biosynthetic process (<a aria-describedby="B40" class="usa-link" href="#B40">Janin and Deville-Bonne, 2002</a>), has been shown to participate in the metabolism of selective drugs like Isoniazid and Fluorouracil (KEGG map00983) in <em>Homo sapiens</em>. The same was observed for the S-formylglutathione hydrolase, whose activity is mainly linked to methane metabolism and the recycling of glutathione in cell detoxification pathways (<a aria-describedby="B36" class="usa-link" href="#B36">Haslam et al., 2002</a>; <a aria-describedby="B114" class="usa-link" href="#B114">Yurimoto et al., 2003</a>). Similarly, the thioredoxin domain-containing protein CF317_000980-T1 and the 6-phosphogluconate dehydrogenase, decarboxylating CF317_005481-T1/A0A0N0NLG1_9EURO, both involved in cell redox homeostasis and glutathione metabolism (<a aria-describedby="B49" class="usa-link" href="#B49">Li et al., 2019</a>), showed an around twofold downregulation.</p>
<p>Also in the categories of transcription, translation, and ribosome biogenesis, downregulated proteins outnumbered the upregulated ones (<a class="usa-link" href="#T2">Table 2</a>). The most increased protein (almost twofold) was the putative D-amino-acid oxidase (DAO) domain-containing protein CF317_008030-T1/A0A0D1YYX4_9EURO, also involved in penicillin and cephalosporin biosynthesis (KEGG 00311). Elongation factor EF-1 beta and gamma subunit (CF317_008933-T1 and CF317_000379-T1), homologs of <em>Exophiala dermatitidis</em> CBS 525.76 H6BVG8_EXODN and H6BY84_EXODN proteins and involved in purine metabolism (KEGG 00230), were detected in the mutant LSSMG-exposed secretome at levels at least 3.5-fold lower than that of the unexposed ones. Further, four ribosomal proteins—i.e., 40S ribosomal protein S20 and S24 (CF317_001861-T1 and CF317_004501-T1); 60S ribosomal protein P1 and L7 (CF317_007960-T1 and CF317_005516-T1)—showed decreased abundance.</p>
<p>Additionally downregulated in response to LSSMG were the uncharacterized HIT domain-containing protein CF317_006677-T1 (over 3-fold regulated) and the putative versicolorin reductase CF317_003229-T1 (over 2-fold regulated), the latter known to mediate aflatoxins biosynthetic processes (<a aria-describedby="B61" class="usa-link" href="#B61">Nakamura et al., 2011</a>). Modulation was also observed in proteins involved in transport such as the above-mentioned 1,3-beta-D-glucan-UDP glucosyltransferase CF317_000614-T1 with a role on starch and sucrose metabolism, the endoplasmic reticulum transmembrane protein CF317_004472-T1, and the inorganic phosphate transport protein PHO88 CF317_006347-T1, which all appeared to be upregulated. Conversely, the transmembrane transporter “putative inorganic phosphate transporter C8E4.01c” CF317_002531-T1, the uncharacterized NTF2 domain-containing protein CF317_007808-T1, and the glycolytic enzyme phosphotransferase CF317_008448-T1 were detected as downregulated.</p></section><section id="S3.SS6"><h3 class="pmc_sec_title">
<em>Knufia chersonesos</em> Wt and Mut Exhibit Opposite Regulation of Several Differentially Expressed Proteins</h3>
<p>The comparative analysis of all examined samples to a reference sample—a pool of all established experimental conditions—revealed qualitative and quantitative differences between <em>K. chersonesos</em> Wt and Mut at the proteome and secretome level. This was especially evident when examining the top 10 differentially regulated proteins i.e., the 10 most up- and downregulated proteins at each experimental condition. A number of these proteins were found to undergo opposite modulation in the two strains, mainly in the whole-cell proteome. Distribution of the top differentially regulated whole-cell proteins is summarized in <a class="usa-link" href="#TS1">Supplementary Table 8</a>: out of 41 different proteins, 6 were found to be top regulated in wild type and mutant under both normal gravity and microgravity condition, 10 only in the Wt and 10 exclusively in the Mut. Around 25% represented ribosomal proteins; 10% were proteins whose identity or function could not be elucidated after homology search; and the remaining proteins were involved in RNA binding, transcription, translation, transport, and carbohydrate metabolism. Out of the 49 top differentially regulated proteins from the secreted fraction, 2 were found in both strains at all experimental conditions, 17 only in the Wt and 17 only in the Mut (<a class="usa-link" href="#TS1">Supplementary Table 9</a>). Similar to what was observed for the proteome, 26% of the detected proteins were ribosomal, followed by enzymes involved in carbohydrate metabolism (18%)—secreted esterases and lipases included—and in transport (10%).</p>
<p>Altogether, the eight top differentially regulated proteins common to wild type and mutant in both normal gravity and microgravity—i.e., the large subunit (LSU) ribosomal proteins L28 (CF317_000935-T1), L32 (CF317_002712-T1), L35 (CF317_001879-T1) and L36 (CF317_007709-T1), the small subunit (SSU) ribosomal protein S30 (CF317_005321-T1), the U1 small nuclear ribonucleoprotein C (CF317_001622-T1), the murein transglycosylase (CF317_006383-T1) and the uncharacterized protein CF317_007965-T1—displayed an LSSMG-dependent regulation that also appeared to be specific to the Wt strain (<a class="usa-link" href="#TS1">Supplementary Figure 3</a>). As shown in <a class="usa-link" href="#TS1">Supplementary Figure 3</a>, all proteins that are upregulated in the Wt strain appear to be downregulated in the Mut strain and vice versa.</p>
<p>A protein-protein interaction analysis was performed to verify the occurrence of these 8 proteins in common pathways, using STRING. A 6-node network was obtained for the proteins which matched homologs (sequence homology) in the STRING database (<a class="usa-link" href="#TS1">Supplementary Figure 4</a>) and an interaction was confirmed for ribosomal proteins S30 (homolog of <em>E. dermatitidis</em> <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/protein/XP_009156842.1" rel="noopener noreferrer" target="_blank">XP_009156842.1</a>) and L35 (homolog of <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/protein/XP_009155966.1" rel="noopener noreferrer" target="_blank">XP_009155966.1</a>). As a component of the SSU, S30 has a role in mRNAs binding and selection of cognate aminoacyl-transferase RNA (tRNA), whereas L35 is required for polymerization of the amino acids delivered by tRNAs into a polypeptide chain (<a aria-describedby="B5" class="usa-link" href="#B5">Ben-Shem et al., 2011</a>). The involvement of protein L35 in cell growth regulation and apoptosis has additionally been reported (<a aria-describedby="B10" class="usa-link" href="#B10">Bommer and Stahl, 2005</a>). The proteins’ functional link (combined score: 0.999) was supported by evidence such as the co-expression of orthologs (score 0.963) and their interaction in other organisms (experimental/biochemical data, score: 0.904). Further interactions could not be detected possibly because functional characterization of these proteins in <em>E. dermatitidis</em> is yet to be thoroughly achieved.</p></section><section id="S3.SS7"><h3 class="pmc_sec_title">
<em>Knufia chersonesos</em> Wt and Mut Comparative Analysis Under Normal Gravity (1G)</h3>
<p>A comparative analysis of wild type and mutant was performed to evaluate different responses at the proteome and secretome level under control condition, i.e., normal gravity. The analysis of the whole-cell proteome yielded a total of 100 proteins with increased and 40 proteins with decreased abundance in <em>K. chersonesos</em> Mut, compared with the wild type (fold change of ≥ ± 1.5, <em>p</em> ≤ 0.05).</p>
<p>As shown in <a class="usa-link" href="#TS1">Supplementary Figure 5A</a>, “protein translation,” “organelle organization,” and “regulation of biological processes” were detected as the prevalent processes in the mutant, encompassing more than 50% of all upregulated proteins. This includes several ribosomal proteins and ribonucleoproteins. Ribosomal proteins L32 (CF317_002712), L36 (CF317_007709), S30 (CF317_005321) and L37 (CF317_004524-T1) and the small nuclear ribonucleoprotein C (CF317_001622-T1) were among the most upregulated proteins, being present in the mutant at level 8.7 to 3.6 (3.1 to 1.85 log2 fold change) higher than in the wild type (<a class="usa-link" href="#TS1">Supplementary Table 10</a>). Increased levels were also observed for proteins involved in RNA metabolic processes and in transcription such as the U3 small nucleolar RNA-associated protein 11 (A0A0D2DFT3_9EURO), the U6 snRNA-associated Sm-like protein LSm4 (CF317_002219-T1) and the pre-mRNA-splicing factor isy1 (CF317_006097-T1),—partaking in nucleolar processing of pre-18S ribosomal RNA and in pre-mRNA splicing (<a aria-describedby="B122" class="usa-link" href="#B122">Zhou et al., 2014</a>)—the transcription factors C2H2 and RfeG (CF317_009591-T1 and CF317_000265-T1) and the 2′-phosphotransferase (CF317_002731-T1). Protein modulation also affected enzymes involved in the oxidative phosphorylation, whose levels resulted to be increased in <em>K. chersonesos</em> Mut, i.e., the cytochrome c oxidase assembly factor 6 (CF317_000645-T1 and CF317_000273-T1), cytochrome b-c1 complex subunit 7 (CF317_008279-T1), NADH dehydrogenase ubiquinone 1 alpha subcomplex subunit CF317_000197-T1. Regulation of proteins involved in chemical and stress response was also observed. The histone H2A (CF317_001633-T1)—reported to have a role on DNA repair (<a aria-describedby="B57" class="usa-link" href="#B57">Moore et al., 2007</a>)—and the Nuclear transcription factor Y alpha (CF317_006017-T1)—with multiple roles in development and stress response (<a aria-describedby="B120" class="usa-link" href="#B120">Zhao et al., 2017</a>)—were found at levels at least 1.6-fold higher than in the melanized counterpart. Conversely, downregulated processes at normal gravity condition prevalently encompassed carbohydrate, lipid and general cellular metabolism (<a class="usa-link" href="#TS1">Supplementary Figure 5A</a>). Decreased levels were observed for glycoside hydrolases and esterolytic enzymes including the alpha-galactosidase (CF317_004369-T1) and cutinases (CF317_007621-T1)—levels 9.7- and 7-fold lower than in the wild type—for the glucan 1,3-beta glucosidase (CF317_002004-T1; 1.69-folds decreased) and for the murein transglycosylase (CF317_004960-T1, CF317_003460-T1, CF317_006383-T1; 1.83-, 2- and 4-fold decreased), enzymes responsible for polymers and microbial degradation by cleavage at the peptidoglycan or the cell wall polysaccharides level (<a aria-describedby="B50" class="usa-link" href="#B50">Lincoln et al., 1997</a>). Additionally, proteins involved in lipid metabolic process—i.e., acetyl-CoA C-acetyltransferase (CF317_005988-T1), isocitrate lyase (CF317_004212-T1) and lipase (CF317_005885-T1)—and in protein catabolism—i.e., proteasome subunit alpha type (CF317_008861-T1), proteasome endopeptidase complex (CF317_004788-T1) and peptidase S53 domain-containing protein (CF317_005049-T1)—showed decreased levels between 1.8- and 1.5-fold.</p>
<p>At the secretome level, the quantitative analysis of <em>K. chersonesos</em> Mut v/s Wt under normal gravity conditions, resulted in the detection of 104 proteins with increased and 87 proteins with decreased abundance in the mutant secretome. Of these proteins, 24% have predicted extracellular localization based on the presence of a signal peptide (<a class="usa-link" href="#TS1">Supplementary Table 11</a>). Distribution of over-represented biological process GO terms among differentially expressed proteins is displayed in <a class="usa-link" href="#TS1">Supplementary Figure 5B</a>. Most of proteins upregulated in the mutant were involved with carbohydrate metabolism and regulation of biological process (24% of all upregulated proteins), transport (9%), response to stress (8%) and transcription (8%). A number of hydrolases required for the breakdown of β-glucan chains and other cell-wall components— i.e., carboxylic ester hydrolyses (CF317_004653-T1, CF317_008040-T1, CF317_002308-T1, CF317_005799-T1), feruloyl esterase (CF317_002949-T1), alongside glucan 1,3-beta-glucosidase (CF317_002004-T1), murein transglycosylase (CF317_006383-T1) and cutinase (CF317_007621-T1) which were downregulated in the mutant proteome—were present in the secretome at levels comprised between 1.5- and 4-folds higher than in the wild type (<a class="usa-link" href="#TS1">Supplementary Table 11</a>). The same for mannan endo-1,6-alpha-mannosidase (CF317_001276-T1) and chitin deacetylase (CF317_003258-T1), which play multiple roles in the function of the fungal cell wall (<a aria-describedby="B95" class="usa-link" href="#B95">Spreghini et al., 2003</a>; <a aria-describedby="B59" class="usa-link" href="#B59">Mouyna et al., 2020</a>). In the category transport, the multicopper oxidase (CF317_009669-T1; 1.9-fold), the chloride channel protein (CF317_004136-T; 1.7-folds) and the nitrate/nitrite transporter (CF317_004846-T1; 1.6-fold) represented the most upregulated proteins. Increased levels of stress-response proteins were observed for the Histone H2A protein (CF317_001633-T1; 1.7-fold)—found to be overexpressed also in the mutant proteome—alongside the redox protein thioredoxin (CF317_006109-T1; 1.7-fold), catalase A (CF317_009216-T1; 1.7-fold) and the housekeeping enzyme nucleoside diphosphate kinase (CF317_007147-T1; 2.3-fold), whose involvement in the signaling pathway of oxidative stresses has been reported (<a aria-describedby="B22" class="usa-link" href="#B22">Desorption et al., 2000</a>). Normal gravity condition was also characterized by increased levels of peptidyl-prolyl <em>cis-trans</em> isomerase (CF317_005659-T1), EF-1 gamma subunit (CF317_000379-T1), and alkaline phosphatase (CF317_008227-T1; CF317_003892-T1), all involved in transcription. Conversely, decreased levels were observed in the secretome almost exclusively for proteins involved in translation and ribosome biogenesis (<a class="usa-link" href="#TS1">Supplementary Figure 5B</a>). Out of 87 downregulated proteins, more than 50 encompassed ribosomal proteins. Of these, the most downregulated ones— the 60S ribosomal proteins L32 (CF317_006070-T1), L35 (CF317_001879-T1) and L33-A (CF317_005595-T1)—were present at levels below 5-fold than their counterparts in the wild type secretome. Other proteins were prevalently involved in stress/chemical response, e.g., plasma membrane phosphatase (CF317_000051-T1), CipC-like antibiotic response protein (CF317_003991-T1), small heat shock protein (SHSP) domain-containing protein (CF317_001628-T1). These findings pose interesting questions regarding the presence of intracellular proteins in <em>K. chersonesos</em> secretome: while it may be indicative of cell lysis during cultivation, it may also suggest that ribosomal and other non-classical secretory proteins are found in the culture supernatant due to EVs-mediated secretion (<a aria-describedby="B99" class="usa-link" href="#B99">Sun and Su, 2019</a>). The co-isolation of these and classical secretory proteins during sample preparation can occur if no prior separation of the vesicle containing fraction from the secretome, is performed (<a aria-describedby="B108" class="usa-link" href="#B108">Vallejo et al., 2012</a>).</p></section></section><section id="S4"><h2 class="pmc_sec_title">Discussion</h2>
<p>To date, investigations into the responses of microorganisms to space and Mars-like conditions have been performed with few black fungi species; some of them involved stress simulation in ground-based facilities (<a aria-describedby="B69" class="usa-link" href="#B69">Pacelli et al., 2017</a>), while others carried out the exposure of fungal strains inside or outside the International Space Station (<a aria-describedby="B64" class="usa-link" href="#B64">Onofri et al., 2012</a>, <a aria-describedby="B66" class="usa-link" href="#B66">2019</a>; <a aria-describedby="B68" class="usa-link" href="#B68">Pacelli et al., 2016</a>).</p>
<p>With only one exception (<a aria-describedby="B115" class="usa-link" href="#B115">Zakharova et al., 2014</a>)—the comparative study of 2D protein patterns under Mars-like conditions—the majority of astrobiological work carried out with black fungi has mainly focused on the analysis of ultrastructural alterations and DNA integrity. Hence, this study represents the first qualitative and quantitative proteomic characterization of a black fungus response to simulated space conditions, i.e., microgravity, which has significance to exobiology and implications to planetary protection policy. In-depth understanding of proteome-related alterations in cell physiology is crucial to gaining new insight into the evolution of extremophiles and the actual limits for life. Further, the comparative analysis of a melanin-deficient mutant and the wild type is important to evaluate the role of melanization in stress survival.</p>
<p>Under microgravity conditions, a development toward a more clump-like growth was observed in both <em>K. chersonesos</em> Wt and Mut and may be due to the low shear and low turbulence suspension culture environment created by LSSMG. However, morphological differences were not detected via FE-SEM in cells grown under LSSMG compared to those grown in 1G. This is not surprising, as black fungi reportedly resort to strategies to minimize efforts at both the morphological and physiological level when exposed to stress conditions (<a aria-describedby="B96" class="usa-link" href="#B96">Sterflinger, 2006</a>). Microcolonial growth and the switch among different growth forms—i.e., budding and hyphae formation—ensure that these organisms have a lifestyle versatility to cope with a variety of stress in their natural habitats (<a aria-describedby="B97" class="usa-link" href="#B97">Sterflinger et al., 1999</a>; <a aria-describedby="B31" class="usa-link" href="#B31">Gostinčar et al., 2011</a>). Unaltered morphology upon exposure to LSSMG was previously described in filamentous fungi in both suspension (e.g., <em>Aspergillus niger</em> and <em>Penicillium chrysogenum</em>) (<a aria-describedby="B80" class="usa-link" href="#B80">Sathishkumar et al., 2014</a>) and agar cultures (<em>A. niger</em>, <em>Candida albicans</em>) (<a aria-describedby="B113" class="usa-link" href="#B113">Yamazaki et al., 2012</a>), albeit LSSMG-induced phenotypic changes have been to date substantiated by a higher number of studies on fungal species e.g., <em>Pleurotus</em> sp., <em>Candida</em> sp., <em>Cladosporium</em> sp., <em>Ulocladium</em> sp., <em>Basipetospora</em> sp., etc (<a aria-describedby="B56" class="usa-link" href="#B56">Miyazaki et al., 2010</a>; <a aria-describedby="B84" class="usa-link" href="#B84">Searles et al., 2011</a>; <a aria-describedby="B27" class="usa-link" href="#B27">Gomoiu et al., 2013</a>). Interestingly, the sole morphological response to microgravity detected in the present study was the early switch to hyphae formation observed in <em>K. chersonesos</em> Wt (at day 5 instead of day 7, as in normal gravity; <a class="usa-link" href="#F2">Figure 2</a>). This is consistent with previous reports of increased filamentous growth under LSSMG in the opportunistic fungal pathogen <em>C. albicans</em> (<a aria-describedby="B2" class="usa-link" href="#B2">Altenburg et al., 2008</a>). The observation of early hyphal development in non-pathogenic species in response to LSSMG is rather suggestive of biofilm formation as a strategy for enhanced resistance to stress and for the forage for nutrients (<a aria-describedby="B84" class="usa-link" href="#B84">Searles et al., 2011</a>). Both LSSMG-cultured <em>K. chersonesos</em> Mut and their 1G controls showed extensive cell self-aggregation, i.e., cell clumping and occasional filamentation that are characteristics of black fungi (<a aria-describedby="B60" class="usa-link" href="#B60">Nai et al., 2013</a>). Significant variations were also not found in the total cell number, and cell size was not affected by microgravity in both strains (<a class="usa-link" href="#TS1">Supplementary Tables 1</a>, <a class="usa-link" href="#TS1">2</a>), unlike what has been documented by studies on a variety of bacteria (<a aria-describedby="B38" class="usa-link" href="#B38">Huang et al., 2018</a>) and yeasts (<a aria-describedby="B19" class="usa-link" href="#B19">Crabbé et al., 2013</a>).</p>
<p>The results of proteomics analysis revealed that exposure to LSSMG altered the proteome and secretome of both <em>K. chersonesos</em> Wt and Mut when compared to the 1G counterparts, having an impact on different pathways. Interestingly, the mutant response mainly involved protein downregulation, which might suggest a general slowing of the metabolic rate (<a class="usa-link" href="#F4">Figure 4B</a>). In contrast, more subtle rearrangements in the protein repertoire were observed in the wild type, especially in the secreted fraction, which possibly reflect a fine-tuning of the regulation of protein expression. Regardless, both strains showed increased abundance of proteins involved in carbohydrate metabolism, especially at the whole-cell proteome level (<a class="usa-link" href="#F5">Figure 5A</a>). Glycolysis/gluconeogenesis and pyruvate metabolism were found to be promoted in the wild type, whereas the glyoxylate shunt, ancillary cycle to TCA cycle and essential for growth on two-carbon compounds (<a aria-describedby="B51" class="usa-link" href="#B51">Lorenz and Fink, 2001</a>), was upregulated in the mutant. Similar alterations in carbohydrate metabolism were previously observed in melanotic filamentous fungi exposed to simulated Mars conditions (SMC) or to ISS-conditions (<a aria-describedby="B77" class="usa-link" href="#B77">Romsdahl et al., 2018</a>, <a aria-describedby="B76" class="usa-link" href="#B76">2019</a>; <a aria-describedby="B6" class="usa-link" href="#B6">Blachowicz et al., 2019a</a>). Here, increased abundance was also observed for several starvation-induced glycoside hydrolases with roles in nutrient acquisition from biopolymers and in the recycling of cell wall components to support cell maintenance. Starvation response was thereby suggested as crucial adaptation to space conditions, especially oligotrophy. Conversely, one characteristic of both <em>K. chersonesos</em> Wt and Mut was decreased levels of glycoside hydrolases and cell wall–degradation enzymes such as the endo-1,3(4)-beta-glucanase (CF317_0009683-T1) and the extracellular cell wall glucanase Crf1 (CF317_009779-T1). The same was observed for carbohydrate-active enzymes like cutinases and for lipases at both the proteome and the secretome level, which suggests that, under space conditions, black fungi opt for strategies different than those adopted by filamentous fungi.</p>
<p>GO analysis and pathways prediction further revealed upregulation of proteins involved in the biosynthesis of unsaturated fatty acids (USFA) and in the metabolism of phospho- and glycerophospholipids. This may be explained by modifications in membrane lipid composition aimed at maintaining membranes stability against stress (<a aria-describedby="B111" class="usa-link" href="#B111">Xia et al., 2019</a>). More specifically, an LSSMG-dependent increase in membrane fluidity attributable to a higher USFA/SFA ratio was previously demonstrated by studies in plain lipid membranes, various microorganisms e.g., <em>Escherichia coli</em> and plants upon exposure to microgravity (<a aria-describedby="B92" class="usa-link" href="#B92">Sieber et al., 2014</a>; <a aria-describedby="B46" class="usa-link" href="#B46">Kim and Rhee, 2016</a>; <a aria-describedby="B48" class="usa-link" href="#B48">Kordyum and Chapman, 2017</a>). Potentially, this could affect the function of membrane-integrated proteins, thereby leading to altered transporter activity. Whether the activity of uptake transporters is altered in microgravity conditions is currently unknown (<a aria-describedby="B25" class="usa-link" href="#B25">Eyal and Derendorf, 2019</a>). However, similarly to what was observed in other fungi e.g., <em>C. albicans</em> (<a aria-describedby="B19" class="usa-link" href="#B19">Crabbé et al., 2013</a>) and bacteria e.g., <em>Bacillus subtilis</em> (<a aria-describedby="B58" class="usa-link" href="#B58">Morrison et al., 2019</a>) in space, <em>K. chersonesos</em> Wt and Mut ion-channels and integral membrane proteins were found to be upregulated. The increased abundance of specific transporters generally maximizes the uptake of nutrients (e.g., phosphate and nitrogen). As such, it possibly represents an adaptive response to temporary starvation caused either by zones of nutrient depletion developing around the colony or by the partial loss of contact of the cells with the culture medium, which may occur under microgravity conditions (<a aria-describedby="B53" class="usa-link" href="#B53">Mazars et al., 2014</a>; <a aria-describedby="B88" class="usa-link" href="#B88">Senatore et al., 2018</a>). Other types of transporters—i.e., ammonium transporters and permeases for transmembrane amino acid transport—could provide a link between nitrogen assimilation and proteins synthesis (<a aria-describedby="B52" class="usa-link" href="#B52">Martzivanou et al., 2006</a>). Indeed, upregulation of proteins involved in amino acid biosynthesis and metabolic processes was recorded under LSSMG conditions, but a higher number of these proteins was found to be decreased, especially in the mutant proteome.</p>
<p>Decreases in the levels of structural ribosomal proteins involved in cytoplasmic translation (i.e., L23a, L37a, and L34) and of the poly(A)polymerase, as well as the increase in RNA degrading ribonuclease T1, were additionally detected in microgravity. Together, the regulation of these proteins may suggest that protein translation via ribosomal translational machinery is reduced upon exposure to LSSMG, a phenomenon that has been indicated as a widespread response to microgravity, space, and Mars-like conditions not only in fungi (<a aria-describedby="B90" class="usa-link" href="#B90">Sheehan et al., 2007</a>; <a aria-describedby="B110" class="usa-link" href="#B110">Willaert, 2013</a>; <a aria-describedby="B26" class="usa-link" href="#B26">Feger et al., 2016</a>; <a aria-describedby="B43" class="usa-link" href="#B43">Kamal et al., 2018</a>, <a aria-describedby="B44" class="usa-link" href="#B44">2019</a>; <a aria-describedby="B6" class="usa-link" href="#B6">Blachowicz et al., 2019a</a>). Remarkably, in our study, a decrease in translation and ribosome biogenesis was only observed in the wild type whole-cell proteome and, to a minor extent, in the mutant secretome (<a class="usa-link" href="#F5">Figure 5B</a>). However, ribosomes are very dynamic organelles and additional tests will be needed to confirm that the 2-fold regulation of the above-mentioned proteins is actually indicative of reduced protein synthesis under microgravity.</p>
<p>In a similar fashion, proteins taking part in functional organization of cell organelles and cytoskeleton—for which a wide spectrum of microgravity-dependent changes has been reported (<a aria-describedby="B118" class="usa-link" href="#B118">Zhang et al., 2015</a>)—showed opposite regulation in the wild type and the mutant. The same was observed in the secretome for the DNA-repair Histone H2A and the scytalone dehydratase, the latter involved in the biosynthesis of DNH melanin from endogenous substrate (<a aria-describedby="B23" class="usa-link" href="#B23">Eisenman and Casadevall, 2012</a>), which were upregulated in the wild type and downregulated in the mutant. This is suggestive of increased measures for cell protection and is in line with the fact that melanin pigmentation and enhanced melanin synthesis is most often a feature of fungi living on space stations (<a aria-describedby="B20" class="usa-link" href="#B20">Dadachova and Casadevall, 2008</a>; <a aria-describedby="B21" class="usa-link" href="#B21">De Middeleer et al., 2019</a>). Further, versicolorin reductase (CF317_003229-T1), a protein mediating aflatoxins biosynthetic processes, was also decreased only in the mutant. Although production of aflatoxins in <em>K. chersonesos</em> has hitherto not been reported, this finding can be indicative of reduced production of allergenic or toxic metabolites (e.g., polyketides). Also in the mutant, the upregulation of hydrolytic enzymes such as the alpha, alpha-trehalase suggests a recourse to store carbohydrates as a carbon source. By contrast, production of compounds that increase the osmotolerancy (e.g., trehalose) represents a quite common protective measure implemented by microorganisms under microgravity and other types of stress (<a aria-describedby="B110" class="usa-link" href="#B110">Willaert, 2013</a>).</p>
<p>Opposite protein regulation was especially evident when examining the top 10 proteins differentially regulated in both strains at all experimental conditions, which included several ribosomal proteins (<a class="usa-link" href="#TS1">Supplementary Figure 3</a>). These discrepancies suggest that strategies to cope with suboptimal conditions of growth may be strain-specific and that diverse rearrangements of proteome repertoire are possibly related to the presence or absence of melanin in the cell wall. Indeed, differences between wild type and mutant proteomic profiles were even observed at normal gravity condition. Proteins involved in translation, transcription and RNA metabolic process were significantly more upregulated in the mutant proteome than in the wild type (up to 9-fold, as in the case of ribosomal proteins L32, L36, S30 and L37), whereas the mutant secretome showed increased abundance of a number of hydrolases required for the breakdown of biopolymers and cell wall components (<a class="usa-link" href="#TS1">Supplementary Figure 5</a>).</p>
<p>A number of proteins differentially expressed under simulated microgravity were involved in stress response, including glutamate dehydrogenase (GDH; CF317_004755-T1) and nitric oxide dioxygenase (NOD; CF317_006388-T1), over-represented in the proteome of both fungi, and chemical stress component proteins with proteolytic and phosphatase activity. While its role as a stress-responsive enzyme was speculated for GDH (i.e., ROS) (<a aria-describedby="B94" class="usa-link" href="#B94">Skopelitis et al., 2006</a>), NOD participates in mechanisms of detoxification of nitric oxide, a gas with multiple roles in cellular metabolism ranging from defense to signaling. The observation of peptidases, which were enriched in <em>K. chersonesos</em> Wt proteome under exposure to LSSMG, may instead suggest the removal of damaged proteins to enhance cellular fitness and maximize survival (<a aria-describedby="B11" class="usa-link" href="#B11">Bonham-Carter et al., 2013</a>; <a aria-describedby="B118" class="usa-link" href="#B118">Zhang et al., 2015</a>). In this respect, it should be noted that a role of altered gravity in the breakdown of protein structures has been previously reported (<a aria-describedby="B106" class="usa-link" href="#B106">Trotter et al., 2015</a>). However, a major part of detected stress response proteins involved in cell redox homeostasis, glutathione metabolism and recycling and osmotic stress defense—i.e., catalases, glutathione reductases, 3-phytases and s-formylglutathione hydrolases—appeared to be decreased under exposure to LSSMG. As previously suggested in plants (<a aria-describedby="B118" class="usa-link" href="#B118">Zhang et al., 2015</a>), downregulation of general stress response proteins under microgravity might indicate an impaired activation of the defense response components. However, decreased levels of common stress proteins and lack of a heat shock response (HSP) represent a key component of black fungi response to a variety of suboptimal conditions of growth and have been attributed to an energy-saving strategy that relies on a fine-tuning regulation of protein abundance (<a aria-describedby="B103" class="usa-link" href="#B103">Tesei et al., 2012</a>, <a aria-describedby="B102" class="usa-link" href="#B102">2015</a>; <a aria-describedby="B116" class="usa-link" href="#B116">Zakharova et al., 2013</a>, <a aria-describedby="B115" class="usa-link" href="#B115">2014</a>; <a aria-describedby="B8" class="usa-link" href="#B8">Blasi et al., 2017</a>).</p>
<p>One further interesting aspect was the high number of proteins traditionally recognized as cytoplasmic in <em>K. chersonesos</em> secretome, especially in the mutant (i.e., 6% of total secreted proteins with predicted signal peptide v/s 25% in the wild type; 20% of regulated secreted proteins had signal peptides in both strains). Such a discrepancy between wild type and mutant was also reported in a study of <em>K. chersonesos</em> secretome aimed at the screening for novel polyesterases (<a aria-describedby="B104" class="usa-link" href="#B104">Tesei et al., 2020</a>), hence it seems to suggest that a low number of extracellular proteins may be a peculiarity of the mutant secretome. The presence of cytoplasmic proteins in the culture supernatant may indicate that the mutant is more prone to cell lysis than the wild type (<a aria-describedby="B55" class="usa-link" href="#B55">Miura and Ueda, 2018</a>). However, given that proteins with no signal peptide (leaderless) can also be found in the secreted fraction as a result of non-classical protein secretion, e.g., via EVs-mediated pathways (<a aria-describedby="B108" class="usa-link" href="#B108">Vallejo et al., 2012</a>; <a aria-describedby="B99" class="usa-link" href="#B99">Sun and Su, 2019</a>), it may as well indicate co-isolation of vesicle proteins and classical secretory proteins during sample preparation procedures. These proteins (e.g., ribosomal proteins, proteins involved in carbohydrate metabolism, response to stress, signaling, cell division and transport, etc.), for which vesicular transport might be the only route of extracellular delivery, are generally predicted as non-secretory (<a aria-describedby="B9" class="usa-link" href="#B9">Bleackley et al., 2019</a>).</p>
<p>Together, the aspects of protein modulation observed in <em>K. chersonesos</em> Wt and Mut suggest that the basic energy metabolism was upregulated in the Wt strain. Here, rearrangements of the protein repertoire resembled the classic response to microgravity, but with no evidence of significant activation of stress components or starvation response. The mutant mostly engaged in protein downregulation, without affecting their cell growth and survivability. This study therefore indicates the ability of black fungi to cope with microgravity conditions and suggests that cell wall melanization may ultimately influence the metabolic response to microgravity. Point mutation will be needed to confirm whether mutagenesis played a role in protein downregulation in <em>K. chersonesos</em> Mut. To further our understanding about the impact of microgravity on ribosome biogenesis and protein translation, future work shall involve proteomics workflows such as radio-labeled protein expression assays.</p></section><section id="S5"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>WGS data for <em>K. chersonesos</em> MA5789 are available in NCBI GenBank (GCA_002319055.1), under BioSample accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/SAMN07326825" rel="noopener noreferrer" target="_blank">SAMN07326825</a> and BioProject accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/PRJNA393270" rel="noopener noreferrer" target="_blank">PRJNA393270</a>. The proteomics datasets generated during the current study are accessible through the ProteomeXchange Consortium (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://proteomecentral.proteomexchange.org" rel="noopener noreferrer" target="_blank">http://proteomecentral.proteomexchange.org</a>) via PRIDE with the data set identifier PXD022898.</p></section><section id="S6"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>DT designed the study, drafted the manuscript, carried out the LSSMG exposure experiments, and performed data analysis and interpretation. AC and MK conducted protein sample processing, LC/MS analyses, and proteome data processing. JS annotated the genome of <em>K. chersonesos</em> MA5789 for proteome analysis. GBMM conducted FE-SEM analysis. KV designed the study and critically reviewed the manuscript. DT, KS, and KV contributed to funding acquisition. All authors contributed to the review, editing, and approval of the final version of the manuscript.</p></section><section id="conf1"><h2 class="pmc_sec_title">Conflict of Interest</h2>
<p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors gratefully acknowledge the Austrian Science Fund (FWF, Der Wissenschaftsfonds) for sponsoring DT’s postdoctoral position and for the funding. Part of the research described in this publication was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with NASA. The authors thank Emily Klonicki, Jet Propulsion Laboratory (JPL) summer student, for contribution to the optimization of fungal cultivation under simulated microgravity and the members of the Planetary Protection group at Jet Propulsion Laboratory (JPL) for their technical assistance.</p></section><section class="fn-group" id="fn-group1"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn4"><p><strong>Funding.</strong> This research was funded by the Austrian Science Fund (FWF, Der Wissenschaftsfonds) project T872 Firnberg-Programm awarded to DT. A 2012 NASA Space Biology grant no. 19-12829-26 under Task Order NNN13D111T was awarded to KV which also supported GBMM. The Mass Spectrometry and Proteomics Core Facility at the City of Hope was supported in part by the National Cancer Institute of the National Institutes of Health under award no. P30CA033572. AC and MK were supported by Jet Propulsion Laboratory (JPL) subcontract no. 1611422.</p></div></div></section><section class="fn-group" id="fn-group2"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="footnote1">
<sup>1</sup><p class="display-inline"><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://geneontology.org" rel="noopener noreferrer" target="_blank">http://geneontology.org</a></p>
</div>
<div class="fn p" id="footnote2">
<sup>2</sup><p class="display-inline"><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://www.uniprot.org/blast" rel="noopener noreferrer" target="_blank">http://www.uniprot.org/blast</a></p>
</div>
<div class="fn p" id="footnote3">
<sup>3</sup><p class="display-inline"><a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://busca.biocomp.unibo.it" rel="noopener noreferrer" target="_blank">http://busca.biocomp.unibo.it</a></p>
</div>
</div></section><section id="S9"><h2 class="pmc_sec_title">Supplementary Material</h2>
<p>The Supplementary Material for this article can be found online at: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.frontiersin.org/articles/10.3389/fgene.2021.638708/full#supplementary-material" rel="noopener noreferrer" target="_blank">https://www.frontiersin.org/articles/10.3389/fgene.2021.638708/full#supplementary-material</a></p>
<section class="sm xbox font-sm" id="TS1"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/8012687/bin/Data_Sheet_1.docx">Click here for additional data file.</a><sup> (3.2MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="TS2"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/8012687/bin/Data_Sheet_2.docx">Click here for additional data file.</a><sup> (3MB, docx) </sup>
</div></div></section></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="B1">
<cite>Aguilera A., González-Toril E. (2019). <em>Fungi in Extreme Environments: Ecological Role and Biotechnological Significance.</em>
Cham: Springer, 21–38. 10.1007/978-3-030-19030-9</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-3-030-19030-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Fungi%20in%20Extreme%20Environments:%20Ecological%20Role%20and%20Biotechnological%20Significance.&amp;author=A.%20Aguilera&amp;author=E.%20Gonz%C3%A1lez-Toril&amp;publication_year=2019&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2">
<cite>Altenburg S. D., Nielsen-Preiss S. M., Hyman L. E. (2008). Increased filamentous growth of <em>Candida albicans</em> in simulated microgravity.
<em>Genomics Proteomics Bioinforma</em>
6
42–50. 10.1016/S1672-0229(08)60019-4</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S1672-0229(08)60019-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5054087/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18558384/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genomics%20Proteomics%20Bioinforma&amp;title=Increased%20filamentous%20growth%20of%20Candida%20albicans%20in%20simulated%20microgravity.&amp;author=S.%20D.%20Altenburg&amp;author=S.%20M.%20Nielsen-Preiss&amp;author=L.%20E.%20Hyman&amp;volume=6&amp;publication_year=2008&amp;pages=42-50&amp;pmid=18558384&amp;doi=10.1016/S1672-0229(08)60019-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3">
<cite>Ametrano C. G., Grewe F., Crous P. W., Goodwin S. B., Liang C., Selbmann L., et al.  (2019). Genome-scale data resolve ancestral rock- inhabiting lifestyle in Dothideomycetes (<em>Ascomycota</em>).
<em>IMA Fungus</em>
10:19.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s43008-019-0018-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7325674/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32647623/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=IMA%20Fungus&amp;title=Genome-scale%20data%20resolve%20ancestral%20rock-%20inhabiting%20lifestyle%20in%20Dothideomycetes%20(Ascomycota).&amp;author=C.%20G.%20Ametrano&amp;author=F.%20Grewe&amp;author=P.%20W.%20Crous&amp;author=S.%20B.%20Goodwin&amp;author=C.%20Liang&amp;volume=10&amp;issue=19&amp;publication_year=2019&amp;pmid=32647623&amp;doi=10.1186/s43008-019-0018-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4">
<cite>Belgaroui N., Lacombe B., Rouached H., Hanin M. (2018). Phytase overexpression in <em>Arabidopsis</em> improves plant growth under osmotic stress and in combination with phosphate deficiency article.
<em>Sci. Rep.</em>
8:1137. 10.1038/s41598-018-19493-w
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-018-19493-w" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5773496/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29348608/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Phytase%20overexpression%20in%20Arabidopsis%20improves%20plant%20growth%20under%20osmotic%20stress%20and%20in%20combination%20with%20phosphate%20deficiency%20article.&amp;author=N.%20Belgaroui&amp;author=B.%20Lacombe&amp;author=H.%20Rouached&amp;author=M.%20Hanin&amp;volume=8&amp;issue=1137&amp;publication_year=2018&amp;pmid=29348608&amp;doi=10.1038/s41598-018-19493-w&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5">
<cite>Ben-Shem A., de Loubresse N. G., Melnikov S., Jenner L., Yusupova G., Yusupov M. (2011). The structure of the eukaryotic ribosome at 3.0 Å resolution.
<em>Science</em>
334
1524–1529.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.1212642" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22096102/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20structure%20of%20the%20eukaryotic%20ribosome%20at%203.0%20%C3%85%20resolution.&amp;author=A.%20Ben-Shem&amp;author=N.%20G.%20de%20Loubresse&amp;author=S.%20Melnikov&amp;author=L.%20Jenner&amp;author=G.%20Yusupova&amp;volume=334&amp;publication_year=2011&amp;pages=1524-1529&amp;pmid=22096102&amp;doi=10.1126/science.1212642&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6">
<cite>Blachowicz A., Chiang A. J., Elsaesser A., Kalkum M., Ehrenfreund P., Stajich J. E., et al.  (2019a). Proteomic and metabolomic characteristics of extremophilic fungi under simulated Mars conditions.
<em>Front. Microbiol.</em>
10:1013. 10.3389/fmicb.2019.01013
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2019.01013" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6529585/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31156574/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Proteomic%20and%20metabolomic%20characteristics%20of%20extremophilic%20fungi%20under%20simulated%20Mars%20conditions.&amp;author=A.%20Blachowicz&amp;author=A.%20J.%20Chiang&amp;author=A.%20Elsaesser&amp;author=M.%20Kalkum&amp;author=P.%20Ehrenfreund&amp;volume=10&amp;issue=1013&amp;publication_year=2019a&amp;pmid=31156574&amp;doi=10.3389/fmicb.2019.01013&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7">
<cite>Blachowicz A., Chiang A. J., Romsdahl J., Kalkum M., Wang C. C. C., Venkateswaran K. (2019b). Proteomic characterization of <em>Aspergillus fumigatus</em> isolated from air and surfaces of the International Space Station.
<em>Fungal Genet. Biol.</em>
124
39–46. 10.1016/j.fgb.2019.01.001
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.fgb.2019.01.001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9116463/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30611835/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&amp;title=Proteomic%20characterization%20of%20Aspergillus%20fumigatus%20isolated%20from%20air%20and%20surfaces%20of%20the%20International%20Space%20Station.&amp;author=A.%20Blachowicz&amp;author=A.%20J.%20Chiang&amp;author=J.%20Romsdahl&amp;author=M.%20Kalkum&amp;author=C.%20C.%20C.%20Wang&amp;volume=124&amp;publication_year=2019b&amp;pages=39-46&amp;pmid=30611835&amp;doi=10.1016/j.fgb.2019.01.001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8">
<cite>Blasi B., Tafer H., Kustor C., Poyntner C., Lopandic K., Sterflinger K. (2017). Genomic and transcriptomic analysis of the toluene degrading black yeast Cladophialophora immunda.
<em>Sci. Rep.</em>
7:11436. 10.1038/s41598-017-11807-8
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-017-11807-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5595782/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28900256/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Genomic%20and%20transcriptomic%20analysis%20of%20the%20toluene%20degrading%20black%20yeast%20Cladophialophora%20immunda.&amp;author=B.%20Blasi&amp;author=H.%20Tafer&amp;author=C.%20Kustor&amp;author=C.%20Poyntner&amp;author=K.%20Lopandic&amp;volume=7&amp;issue=11436&amp;publication_year=2017&amp;pmid=28900256&amp;doi=10.1038/s41598-017-11807-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9">
<cite>Bleackley M. R., Dawson C. S., Anderson M. A. (2019). Fungal extracellular vesicles with a focus on proteomic analysis.
<em>Proteomics</em>
19
1–14. 10.1002/pmic.201800232
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/pmic.201800232" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30883019/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Fungal%20extracellular%20vesicles%20with%20a%20focus%20on%20proteomic%20analysis.&amp;author=M.%20R.%20Bleackley&amp;author=C.%20S.%20Dawson&amp;author=M.%20A.%20Anderson&amp;volume=19&amp;publication_year=2019&amp;pages=1-14&amp;pmid=30883019&amp;doi=10.1002/pmic.201800232&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10">
<cite>Bommer U. A., Stahl J. (2005). <em>Ribosomal Proteins in Eukaryotes.</em>
New York, NY: John Wiley &amp; Sons, Ltd. 10.1038/npg.els.0003867</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/npg.els.0003867" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Ribosomal%20Proteins%20in%20Eukaryotes.&amp;author=U.%20A.%20Bommer&amp;author=J.%20Stahl&amp;publication_year=2005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11">
<cite>Bonham-Carter O., Pedersen J., Najjar L., Bastola D. (2013). “Modeling the effects of microgravity on oxidation in mitochondria: a protein damage assessment across a diverse set of life forms,” in <em>Proceedings of the IEEE 13th International Conference on Data Mining Workshops ICDMW 2013</em>, (Dallas, TX: ), 250–257. 10.1109/ICDMW.2013.149</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1109/ICDMW.2013.149" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Proceedings%20of%20the%20IEEE%2013th%20International%20Conference%20on%20Data%20Mining%20Workshops%20ICDMW%202013&amp;author=O.%20Bonham-Carter&amp;author=J.%20Pedersen&amp;author=L.%20Najjar&amp;author=D.%20Bastola&amp;publication_year=2013&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12">
<cite>Boutet E., Lieberherr D., Tognolli M., Schneider M., Bairoch A. (2007). “UniProtKB/Swiss-Prot BT,” in <em>Plant Bioinformatics: Methods and Protocols</em>, ed.
Edwards D. (Totowa, NJ: Humana Press; ), 89–112. 10.1007/978-1-59745-535-0_4</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-1-59745-535-0_4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Plant%20Bioinformatics:%20Methods%20and%20Protocols&amp;author=E.%20Boutet&amp;author=D.%20Lieberherr&amp;author=M.%20Tognolli&amp;author=M.%20Schneider&amp;author=A.%20Bairoch&amp;publication_year=2007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13">
<cite>Braun B., Pfirrmann T., Menssen R., Hofmann K., Scheel H., Wolf D. H. (2011). Gid9, a second RING finger protein contributes to the ubiquitin ligase activity of the Gid complex required for catabolite degradation.
<em>FEBS Lett.</em>
585
3856–3861. 10.1016/j.febslet.2011.10.038
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.febslet.2011.10.038" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22044534/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&amp;title=Gid9,%20a%20second%20RING%20finger%20protein%20contributes%20to%20the%20ubiquitin%20ligase%20activity%20of%20the%20Gid%20complex%20required%20for%20catabolite%20degradation.&amp;author=B.%20Braun&amp;author=T.%20Pfirrmann&amp;author=R.%20Menssen&amp;author=K.%20Hofmann&amp;author=H.%20Scheel&amp;volume=585&amp;publication_year=2011&amp;pages=3856-3861&amp;pmid=22044534&amp;doi=10.1016/j.febslet.2011.10.038&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14">
<cite>Cánovas D., Marcos J. F., Marcos A. T., Strauss J. (2016). Nitric oxide in fungi: is there NO light at the end of the tunnel?
<em>Curr. Genet.</em>
62
513–518. 10.1007/s00294-016-0574-6
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00294-016-0574-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4929157/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26886232/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Genet.&amp;title=Nitric%20oxide%20in%20fungi:%20is%20there%20NO%20light%20at%20the%20end%20of%20the%20tunnel?&amp;author=D.%20C%C3%A1novas&amp;author=J.%20F.%20Marcos&amp;author=A.%20T.%20Marcos&amp;author=J.%20Strauss&amp;volume=62&amp;publication_year=2016&amp;pages=513-518&amp;pmid=26886232&amp;doi=10.1007/s00294-016-0574-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15">
<cite>Checinska A., Probst A. J., Vaishampayan P., White J. R., Kumar D., Stepanov V. G., et al.  (2015). Microbiomes of the dust particles collected from the International Space Station and Spacecraft Assembly Facilities.
<em>Microbiome</em>
3:50. 10.1186/s40168-015-0116-3
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40168-015-0116-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4624184/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26502721/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Microbiomes%20of%20the%20dust%20particles%20collected%20from%20the%20International%20Space%20Station%20and%20Spacecraft%20Assembly%20Facilities.&amp;author=A.%20Checinska&amp;author=A.%20J.%20Probst&amp;author=P.%20Vaishampayan&amp;author=J.%20R.%20White&amp;author=D.%20Kumar&amp;volume=3&amp;issue=50&amp;publication_year=2015&amp;pmid=26502721&amp;doi=10.1186/s40168-015-0116-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16">
<cite>Checinska-Sielaff A., Urbaniak C., Babu G., Mohan M., Stepanov V. G., Tran Q., et al.  (2019). Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces.
<em>Microbiome</em>
7:50.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40168-019-0666-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6452512/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30955503/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces.&amp;author=A.%20Checinska-Sielaff&amp;author=C.%20Urbaniak&amp;author=G.%20Babu&amp;author=M.%20Mohan&amp;author=V.%20G.%20Stepanov&amp;volume=7&amp;issue=50&amp;publication_year=2019&amp;pmid=30955503&amp;doi=10.1186/s40168-019-0666-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17">
<cite>Choudhry P. (2016). High-Throughput method for automated colony and cell counting by digital image analysis based on edge detection.
<em>PLoS One</em>
11:e0148469. 10.1371/journal.pone.0148469
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0148469" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4746068/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26848849/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=High-Throughput%20method%20for%20automated%20colony%20and%20cell%20counting%20by%20digital%20image%20analysis%20based%20on%20edge%20detection.&amp;author=P.%20Choudhry&amp;volume=11&amp;issue=e0148469&amp;publication_year=2016&amp;pmid=26848849&amp;doi=10.1371/journal.pone.0148469&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18">
<cite>Couto N., Wood J., Barber J. (2016). Free radical biology and medicine the role of glutathione reductase and related enzymes on cellular redox homoeostasis network.
<em>Free Radic. Biol. Med.</em>
95
27–42. 10.1016/j.freeradbiomed.2016.02.028
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.freeradbiomed.2016.02.028" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26923386/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Free%20Radic.%20Biol.%20Med.&amp;title=Free%20radical%20biology%20and%20medicine%20the%20role%20of%20glutathione%20reductase%20and%20related%20enzymes%20on%20cellular%20redox%20homoeostasis%20network.&amp;author=N.%20Couto&amp;author=J.%20Wood&amp;author=J.%20Barber&amp;volume=95&amp;publication_year=2016&amp;pages=27-42&amp;pmid=26923386&amp;doi=10.1016/j.freeradbiomed.2016.02.028&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19">
<cite>Crabbé A., Nielsen-Preiss S. M., Woolley C. M., Barrila J., Buchanan K., McCracken J., et al.  (2013). Spaceflight enhances cell aggregation and random budding in <em>Candida albicans</em>.
<em>PLoS One</em>
8:e80677. 10.1371/journal.pone.0080677
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0080677" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3851762/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24324620/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Spaceflight%20enhances%20cell%20aggregation%20and%20random%20budding%20in%20Candida%20albicans.&amp;author=A.%20Crabb%C3%A9&amp;author=S.%20M.%20Nielsen-Preiss&amp;author=C.%20M.%20Woolley&amp;author=J.%20Barrila&amp;author=K.%20Buchanan&amp;volume=8&amp;issue=e80677&amp;publication_year=2013&amp;pmid=24324620&amp;doi=10.1371/journal.pone.0080677&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B20">
<cite>Dadachova E., Casadevall A. (2008). Ionizing radiation: how fungi cope, adapt, and exploit with the help of melanin.
<em>Curr. Opin. Microbiol.</em>
11
525–531. 10.1016/j.mib.2008.09.013
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.mib.2008.09.013" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2677413/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18848901/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Microbiol.&amp;title=Ionizing%20radiation:%20how%20fungi%20cope,%20adapt,%20and%20exploit%20with%20the%20help%20of%20melanin.&amp;author=E.%20Dadachova&amp;author=A.%20Casadevall&amp;volume=11&amp;publication_year=2008&amp;pages=525-531&amp;pmid=18848901&amp;doi=10.1016/j.mib.2008.09.013&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21">
<cite>De Middeleer G., Leys N., Sas B., De Saeger S. (2019). Fungi and mycotoxins in space—a review.
<em>Astrobiology</em>
19
915–926. 10.1089/ast.2018.1854
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2018.1854" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30973270/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Fungi%20and%20mycotoxins%20in%20space%E2%80%94a%20review.&amp;author=G.%20De%20Middeleer&amp;author=N.%20Leys&amp;author=B.%20Sas&amp;author=S.%20De%20Saeger&amp;volume=19&amp;publication_year=2019&amp;pages=915-926&amp;pmid=30973270&amp;doi=10.1089/ast.2018.1854&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22">
<cite>Desorption M. L., Mass I. T., Song E. J., Kim Y. S., Chung J. Y., Kim E., et al.  (2000). Oxidative modification of nucleoside diphosphate kinase and its identification by matrix-assisted laser desorption/ionization time-of-flight mass spectrometry.
<em>Biochemistry</em>
39
10090–10097.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/bi000267a" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10955997/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Oxidative%20modification%20of%20nucleoside%20diphosphate%20kinase%20and%20its%20identification%20by%20matrix-assisted%20laser%20desorption/ionization%20time-of-flight%20mass%20spectrometry.&amp;author=M.%20L.%20Desorption&amp;author=I.%20T.%20Mass&amp;author=E.%20J.%20Song&amp;author=Y.%20S.%20Kim&amp;author=J.%20Y.%20Chung&amp;volume=39&amp;publication_year=2000&amp;pages=10090-10097&amp;pmid=10955997&amp;doi=10.1021/bi000267a&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23">
<cite>Eisenman H. C., Casadevall A. (2012). Synthesis and assembly of fungal melanin.
<em>Appl. Microbiol. Biotechnol.</em>
93
931–940. 10.1007/s00253-011-3777-2
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00253-011-3777-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4318813/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22173481/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&amp;title=Synthesis%20and%20assembly%20of%20fungal%20melanin.&amp;author=H.%20C.%20Eisenman&amp;author=A.%20Casadevall&amp;volume=93&amp;publication_year=2012&amp;pages=931-940&amp;pmid=22173481&amp;doi=10.1007/s00253-011-3777-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24">
<cite>El-gebali S., Mistry J., Bateman A., Eddy S. R., Potter S. C., Qureshi M., et al.  (2019). The Pfam protein families database in 2019.
<em>Nucleic Acids Res</em>
47
427–432. 10.1093/nar/gky995
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gky995" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6324024/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30357350/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20Pfam%20protein%20families%20database%20in%202019.&amp;author=S.%20El-gebali&amp;author=J.%20Mistry&amp;author=A.%20Bateman&amp;author=S.%20R.%20Eddy&amp;author=S.%20C.%20Potter&amp;volume=47&amp;publication_year=2019&amp;pages=427-432&amp;pmid=30357350&amp;doi=10.1093/nar/gky995&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25">
<cite>Eyal S., Derendorf H. (2019). Medications in space: in search of a pharmacologist’s guide to the galaxy.
<em>Pharm. Res.</em>
36:148. 10.1007/s11095-019-2679-3
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11095-019-2679-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31414302/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Pharm.%20Res.&amp;title=Medications%20in%20space:%20in%20search%20of%20a%20pharmacologist%E2%80%99s%20guide%20to%20the%20galaxy.&amp;author=S.%20Eyal&amp;author=H.%20Derendorf&amp;volume=36&amp;issue=148&amp;publication_year=2019&amp;pmid=31414302&amp;doi=10.1007/s11095-019-2679-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26">
<cite>Feger B. J., Thompson J. W., Dubois L. G., Kommaddi R. P., Foster M. W., Mishra R., et al.  (2016). Microgravity induces proteomics changes involved in endoplasmic reticulum stress and mitochondrial protection.
<em>Sci. Rep.</em>
6:34091. 10.1038/srep34091
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/srep34091" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5037457/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27670941/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Microgravity%20induces%20proteomics%20changes%20involved%20in%20endoplasmic%20reticulum%20stress%20and%20mitochondrial%20protection.&amp;author=B.%20J.%20Feger&amp;author=J.%20W.%20Thompson&amp;author=L.%20G.%20Dubois&amp;author=R.%20P.%20Kommaddi&amp;author=M.%20W.%20Foster&amp;volume=6&amp;issue=34091&amp;publication_year=2016&amp;pmid=27670941&amp;doi=10.1038/srep34091&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27">
<cite>Gomoiu I., Chatzitheodoridis E., Vadrucci S., Walther I. (2013). The effect of spaceflight on growth of ulocladium chartarum colonies on the International Space Station.
<em>PLoS One</em>
8:e62130. 10.1371/journal.pone.0062130
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0062130" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3634740/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23637980/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=The%20effect%20of%20spaceflight%20on%20growth%20of%20ulocladium%20chartarum%20colonies%20on%20the%20International%20Space%20Station.&amp;author=I.%20Gomoiu&amp;author=E.%20Chatzitheodoridis&amp;author=S.%20Vadrucci&amp;author=I.%20Walther&amp;volume=8&amp;issue=e62130&amp;publication_year=2013&amp;pmid=23637980&amp;doi=10.1371/journal.pone.0062130&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28">
<cite>Gonçalves V. N., Cantrell C. L., Wedge D. E., Ferreira M. C., Soares M. A., Jacob M. R., et al.  (2016). Fungi associated with rocks of the Atacama Desert: taxonomy, distribution, diversity, ecology and bioprospection for bioactive compounds.
<em>Environ. Microbiol.</em>
18
232–245. 10.1111/1462-2920.13005
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/1462-2920.13005" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26235221/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&amp;title=Fungi%20associated%20with%20rocks%20of%20the%20Atacama%20Desert:%20taxonomy,%20distribution,%20diversity,%20ecology%20and%20bioprospection%20for%20bioactive%20compounds.&amp;author=V.%20N.%20Gon%C3%A7alves&amp;author=C.%20L.%20Cantrell&amp;author=D.%20E.%20Wedge&amp;author=M.%20C.%20Ferreira&amp;author=M.%20A.%20Soares&amp;volume=18&amp;publication_year=2016&amp;pages=232-245&amp;pmid=26235221&amp;doi=10.1111/1462-2920.13005&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29">
<cite>Gorbushina A. A. (2007). Life on the rocks.
<em>Environ. Microbiol.</em>
9
1613–1631. 10.1111/j.1462-2920.2007.01301.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1462-2920.2007.01301.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17564597/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&amp;title=Life%20on%20the%20rocks.&amp;author=A.%20A.%20Gorbushina&amp;volume=9&amp;publication_year=2007&amp;pages=1613-1631&amp;pmid=17564597&amp;doi=10.1111/j.1462-2920.2007.01301.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30">
<cite>Gorbushina A. A., Kotlova E. R., Sherstneva O. A. (2008). Cellular responses of microcolonial rock fungi to long-term desiccation and subsequent rehydration.
<em>Stud. Mycol.</em>
61
91–97. 10.3114/sim.2008.61.09
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3114/sim.2008.61.09" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2610304/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19287531/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Stud.%20Mycol.&amp;title=Cellular%20responses%20of%20microcolonial%20rock%20fungi%20to%20long-term%20desiccation%20and%20subsequent%20rehydration.&amp;author=A.%20A.%20Gorbushina&amp;author=E.%20R.%20Kotlova&amp;author=O.%20A.%20Sherstneva&amp;volume=61&amp;publication_year=2008&amp;pages=91-97&amp;pmid=19287531&amp;doi=10.3114/sim.2008.61.09&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31">
<cite>Gostinčar C., Grube M., Gunde-Cimerman N. (2011). Evolution of fungal pathogens in domestic environments?
<em>Fungal Biol.</em>
115
1008–1018. 10.1016/j.funbio.2011.03.004
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.funbio.2011.03.004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21944213/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Biol.&amp;title=Evolution%20of%20fungal%20pathogens%20in%20domestic%20environments?&amp;author=C.%20Gostin%C4%8Dar&amp;author=M.%20Grube&amp;author=N.%20Gunde-Cimerman&amp;volume=115&amp;publication_year=2011&amp;pages=1008-1018&amp;pmid=21944213&amp;doi=10.1016/j.funbio.2011.03.004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32">
<cite>Götz S., García-Gómez J. M., Terol J., Williams T. D., Nagaraj S. H., Nueda M. J., et al.  (2008). High-throughput functional annotation and data mining with the Blast2GO suite.
<em>Nucleic Acids Res.</em>
36
3420–3435. 10.1093/nar/gkn176
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkn176" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2425479/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18445632/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=High-throughput%20functional%20annotation%20and%20data%20mining%20with%20the%20Blast2GO%20suite.&amp;author=S.%20G%C3%B6tz&amp;author=J.%20M.%20Garc%C3%ADa-G%C3%B3mez&amp;author=J.%20Terol&amp;author=T.%20D.%20Williams&amp;author=S.%20H.%20Nagaraj&amp;volume=36&amp;publication_year=2008&amp;pages=3420-3435&amp;pmid=18445632&amp;doi=10.1093/nar/gkn176&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B33">
<cite>Gunde-Cimerman N., Sonjak S., Zalar P., Frisvad J. C., Diderichsen B., Plemenitaš A. (2003). Extremophilic fungi in arctic ice: a relationship between adaptation to low temperature and water activity.
<em>Phys. Chem. Earth Parts A/B/C</em>
28
1273–1278. 10.1016/J.PCE.2003.08.056</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/J.PCE.2003.08.056" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Phys.%20Chem.%20Earth%20Parts%20A/B/C&amp;title=Extremophilic%20fungi%20in%20arctic%20ice:%20a%20relationship%20between%20adaptation%20to%20low%20temperature%20and%20water%20activity.&amp;author=N.%20Gunde-Cimerman&amp;author=S.%20Sonjak&amp;author=P.%20Zalar&amp;author=J.%20C.%20Frisvad&amp;author=B.%20Diderichsen&amp;volume=28&amp;publication_year=2003&amp;pages=1273-1278&amp;doi=10.1016/J.PCE.2003.08.056&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34">
<cite>Gunde-Cimerman N., Zalar P., de Hoog G. S., Plemenitaš A. (2000). Hypersaline waters in saltern – natural ecological niches for halophilic black yeasts.
<em>FEMS Microbiol. Ecol.</em>
32
235–240.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1574-6941.2000.tb00716.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10858582/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Ecol.&amp;title=Hypersaline%20waters%20in%20saltern%20%E2%80%93%20natural%20ecological%20niches%20for%20halophilic%20black%20yeasts.&amp;author=N.%20Gunde-Cimerman&amp;author=P.%20Zalar&amp;author=G.%20S.%20de%20Hoog&amp;author=A.%20Plemenita%C5%A1&amp;volume=32&amp;publication_year=2000&amp;pages=235-240&amp;pmid=10858582&amp;doi=10.1111/j.1574-6941.2000.tb00716.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35">
<cite>Haas B. J., Salzberg S. L., Zhu W., Pertea M., Allen J. E., Orvis J., et al.  (2008). Automated eukaryotic gene structure annotation using EVidenceModeler and the program to assemble spliced alignments.
<em>Genome Biol.</em>
9:R7. 10.1186/gb-2008-9-1-r7
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/gb-2008-9-1-r7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2395244/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18190707/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20program%20to%20assemble%20spliced%20alignments.&amp;author=B.%20J.%20Haas&amp;author=S.%20L.%20Salzberg&amp;author=W.%20Zhu&amp;author=M.%20Pertea&amp;author=J.%20E.%20Allen&amp;volume=9&amp;issue=R7&amp;publication_year=2008&amp;pmid=18190707&amp;doi=10.1186/gb-2008-9-1-r7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36">
<cite>Haslam R., Rust S., Pallett K., Cole D., Coleman J. (2002). Cloning and characterisation of S-formylglutathione hydrolase from <em>Arabidopsis thaliana</em>: a pathway for formaldehyde detoxification.
<em>Plant Physiol. Biochem.</em>
40
281–288. 10.1016/S0981-9428(02)01378-5</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0981-9428(02)01378-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.%20Biochem.&amp;title=Cloning%20and%20characterisation%20of%20S-formylglutathione%20hydrolase%20from%20Arabidopsis%20thaliana:%20a%20pathway%20for%20formaldehyde%20detoxification.&amp;author=R.%20Haslam&amp;author=S.%20Rust&amp;author=K.%20Pallett&amp;author=D.%20Cole&amp;author=J.%20Coleman&amp;volume=40&amp;publication_year=2002&amp;pages=281-288&amp;doi=10.1016/S0981-9428(02)01378-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B37">
<cite>Herranz R., Anken R., Boonstra J., Braun M., Christianen P. C. M., de Geest M., et al.  (2013). Ground-based facilities for simulation of microgravity: organism-specific recommendations for their use, and recommended terminology.
<em>Astrobiology</em>
13
1–17. 10.1089/ast.2012.0876
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2012.0876" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3549630/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23252378/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Ground-based%20facilities%20for%20simulation%20of%20microgravity:%20organism-specific%20recommendations%20for%20their%20use,%20and%20recommended%20terminology.&amp;author=R.%20Herranz&amp;author=R.%20Anken&amp;author=J.%20Boonstra&amp;author=M.%20Braun&amp;author=P.%20C.%20M.%20Christianen&amp;volume=13&amp;publication_year=2013&amp;pages=1-17&amp;pmid=23252378&amp;doi=10.1089/ast.2012.0876&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B38">
<cite>Huang B., Li D. G., Huang Y., Liu C. T. (2018). Effects of spaceflight and simulated microgravity on microbial growth and secondary metabolism.
<em>Mil. Med. Res.</em>
5
1–14. 10.1186/s40779-018-0162-9
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40779-018-0162-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5971428/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29807538/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mil.%20Med.%20Res.&amp;title=Effects%20of%20spaceflight%20and%20simulated%20microgravity%20on%20microbial%20growth%20and%20secondary%20metabolism.&amp;author=B.%20Huang&amp;author=D.%20G.%20Li&amp;author=Y.%20Huang&amp;author=C.%20T.%20Liu&amp;volume=5&amp;publication_year=2018&amp;pages=1-14&amp;pmid=29807538&amp;doi=10.1186/s40779-018-0162-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B39">
<cite>Huerta-Cepas J., Szklarczyk D., Heller D., Hernández-Plaza A., Forslund S. K., Cook H., et al.  (2019). EggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses.
<em>Nucleic Acids Res.</em>
47
D309–D314. 10.1093/nar/gky1085
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gky1085" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6324079/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30418610/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=EggNOG%205.0:%20a%20hierarchical,%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses.&amp;author=J.%20Huerta-Cepas&amp;author=D.%20Szklarczyk&amp;author=D.%20Heller&amp;author=A.%20Hern%C3%A1ndez-Plaza&amp;author=S.%20K.%20Forslund&amp;volume=47&amp;publication_year=2019&amp;pages=D309-D314&amp;pmid=30418610&amp;doi=10.1093/nar/gky1085&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B40">
<cite>Janin J., Deville-Bonne D. (2002). “7 – Nucleoside-diphosphate kinase: structural and kinetic analysis of reaction pathway and phosphohistidine intermediate,” in <em>Enzyme Kinetics and Mechanism</em>, ed.
Purich D. L. (Cambridge, MA: Academic Press; ), 118–134. 10.1016/S0076-6879(02)54009-X</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0076-6879(02)54009-X" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12418220/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Enzyme%20Kinetics%20and%20Mechanism&amp;author=J.%20Janin&amp;author=D.%20Deville-Bonne&amp;publication_year=2002&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B41">
<cite>Janssen U., Fink T., Lichter P., Stoffel W. (1994). Human mitochondrial 3,2-trans-enoyl-CoA isomerase (DCI): gene structure and localization to chromosome 16p13.3.
<em>Genomics</em>
23
223–228. 10.1006/geno.1994.1480
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1006/geno.1994.1480" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/7829074/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Human%20mitochondrial%203,2-trans-enoyl-CoA%20isomerase%20(DCI):%20gene%20structure%20and%20localization%20to%20chromosome%2016p13.3.&amp;author=U.%20Janssen&amp;author=T.%20Fink&amp;author=P.%20Lichter&amp;author=W.%20Stoffel&amp;volume=23&amp;publication_year=1994&amp;pages=223-228&amp;pmid=7829074&amp;doi=10.1006/geno.1994.1480&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B42">
<cite>Jensen L. J., Kuhn M., Stark M., Chaffron S., Creevey C., Muller J., et al.  (2009). STRING 8 — a global view on proteins and their functional interactions in 630 organisms.
<em>Nucleic Acids Res.</em>
37
412–416. 10.1093/nar/gkn760
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkn760" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2686466/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18940858/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=STRING%208%20%E2%80%94%20a%20global%20view%20on%20proteins%20and%20their%20functional%20interactions%20in%20630%20organisms.&amp;author=L.%20J.%20Jensen&amp;author=M.%20Kuhn&amp;author=M.%20Stark&amp;author=S.%20Chaffron&amp;author=C.%20Creevey&amp;volume=37&amp;publication_year=2009&amp;pages=412-416&amp;pmid=18940858&amp;doi=10.1093/nar/gkn760&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B43">
<cite>Kamal K. Y., Herranz R., Van Loon J. J. W. A., Medina F. J. (2018). Simulated microgravity, Mars gravity, and 2g hypergravity affect cell cycle regulation, ribosome biogenesis, and epigenetics in <em>Arabidopsis</em> cell cultures.
<em>Sci. Rep.</em>
8:6424. 10.1038/s41598-018-24942-7
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-018-24942-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5913308/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29686401/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Simulated%20microgravity,%20Mars%20gravity,%20and%202g%20hypergravity%20affect%20cell%20cycle%20regulation,%20ribosome%20biogenesis,%20and%20epigenetics%20in%20Arabidopsis%20cell%20cultures.&amp;author=K.%20Y.%20Kamal&amp;author=R.%20Herranz&amp;author=J.%20J.%20W.%20A.%20Van%20Loon&amp;author=F.%20J.%20Medina&amp;volume=8&amp;issue=6424&amp;publication_year=2018&amp;pmid=29686401&amp;doi=10.1038/s41598-018-24942-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B44">
<cite>Kamal K. Y., van Loon J. J. W. A., Medina F. J., Herranz R. (2019). Differential transcriptional profile through cell cycle progression in <em>Arabidopsis</em> cultures under simulated microgravity.
<em>Genomics</em>
111
1956–1965. 10.1016/j.ygeno.2019.01.007
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.ygeno.2019.01.007" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30641127/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Differential%20transcriptional%20profile%20through%20cell%20cycle%20progression%20in%20Arabidopsis%20cultures%20under%20simulated%20microgravity.&amp;author=K.%20Y.%20Kamal&amp;author=J.%20J.%20W.%20A.%20van%20Loon&amp;author=F.%20J.%20Medina&amp;author=R.%20Herranz&amp;volume=111&amp;publication_year=2019&amp;pages=1956-1965&amp;pmid=30641127&amp;doi=10.1016/j.ygeno.2019.01.007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B45">
<cite>Kanehisa M., Goto S. (2000). KEGG: kyoto encyclopedia of genes and genomes.
<em>Nucleic Acids Res.</em>
28
27–30.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/28.1.27" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC102409/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10592173/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=KEGG:%20kyoto%20encyclopedia%20of%20genes%20and%20genomes.&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;volume=28&amp;publication_year=2000&amp;pages=27-30&amp;pmid=10592173&amp;doi=10.1093/nar/28.1.27&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B46">
<cite>Kim H. W., Rhee M. S. (2016). Influence of low-shear modeled microgravity on heat resistance, membrane fatty acid composition, and heat stress-related gene.
<em>Appl. Environ. Microbiol.</em>
82
2893–2901. 10.1128/AEM.00050-16.Editor</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.00050-16.Editor" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4959082/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26944847/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&amp;title=Influence%20of%20low-shear%20modeled%20microgravity%20on%20heat%20resistance,%20membrane%20fatty%20acid%20composition,%20and%20heat%20stress-related%20gene.&amp;author=H.%20W.%20Kim&amp;author=M.%20S.%20Rhee&amp;volume=82&amp;publication_year=2016&amp;pages=2893-2901&amp;pmid=26944847&amp;doi=10.1128/AEM.00050-16.Editor&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B47">
<cite>Knox B. P., Blachowicz A., Palmer J. M., Romsdahl J., Huttenlocher A., Wang C. C. C., et al.  (2016). Characterization of <em>Aspergillus fumigatus</em> isolates from air and surfaces of the International Space Station.
<em>Clin. Sci. Epidemiol.</em>
1
1–15.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/mSphere.00227-16" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5082629/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27830189/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Sci.%20Epidemiol.&amp;title=Characterization%20of%20Aspergillus%20fumigatus%20isolates%20from%20air%20and%20surfaces%20of%20the%20International%20Space%20Station.&amp;author=B.%20P.%20Knox&amp;author=A.%20Blachowicz&amp;author=J.%20M.%20Palmer&amp;author=J.%20Romsdahl&amp;author=A.%20Huttenlocher&amp;volume=1&amp;publication_year=2016&amp;pages=1-15&amp;pmid=27830189&amp;doi=10.1128/mSphere.00227-16&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B48">
<cite>Kordyum E. L., Chapman D. K. (2017). Plants and microgravity: patterns of microgravity effects at the cellular and molecular levels.
<em>Cytol. Genet.</em>
51
108–116. 10.3103/S0095452717020049</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3103/S0095452717020049" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30484618/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cytol.%20Genet.&amp;title=Plants%20and%20microgravity:%20patterns%20of%20microgravity%20effects%20at%20the%20cellular%20and%20molecular%20levels.&amp;author=E.%20L.%20Kordyum&amp;author=D.%20K.%20Chapman&amp;volume=51&amp;publication_year=2017&amp;pages=108-116&amp;pmid=30484618&amp;doi=10.3103/S0095452717020049&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B49">
<cite>Li H., Ericsson M., Rabasha B., Budnik B., Chan S. H., Freinkman E., et al.  (2019). 6-phosphogluconate dehydrogenase links cytosolic carbohydrate metabolism to protein secretion via modulation of glutathione levels.
<em>Cell Chem. Biol.</em>
26
1306–1314.e5. 10.1016/j.chembiol.2019.05.006
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.chembiol.2019.05.006" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6754299/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31204288/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell%20Chem.%20Biol.&amp;title=6-phosphogluconate%20dehydrogenase%20links%20cytosolic%20carbohydrate%20metabolism%20to%20protein%20secretion%20via%20modulation%20of%20glutathione%20levels.&amp;author=H.%20Li&amp;author=M.%20Ericsson&amp;author=B.%20Rabasha&amp;author=B.%20Budnik&amp;author=S.%20H.%20Chan&amp;volume=26&amp;publication_year=2019&amp;pages=1306-1314.e5&amp;pmid=31204288&amp;doi=10.1016/j.chembiol.2019.05.006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B50">
<cite>Lincoln S. P., Fermor T. R., Wood D. A. (1997). Production and detection of muramidase and acetylglucosaminidase from <em>Agaricus bisporus</em>.
<em>Lett. Appl. Microbiol.</em>
25
24–29. 10.1046/j.1472-765X.1997.00163.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1046/j.1472-765X.1997.00163.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9248076/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Lett.%20Appl.%20Microbiol.&amp;title=Production%20and%20detection%20of%20muramidase%20and%20acetylglucosaminidase%20from%20Agaricus%20bisporus.&amp;author=S.%20P.%20Lincoln&amp;author=T.%20R.%20Fermor&amp;author=D.%20A.%20Wood&amp;volume=25&amp;publication_year=1997&amp;pages=24-29&amp;pmid=9248076&amp;doi=10.1046/j.1472-765X.1997.00163.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B51">
<cite>Lorenz M. C., Fink G. R. (2001). The glyoxylate cycle is required for fungal virulence.
<em>Nature</em>
412
83–86. 10.1038/35083594
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/35083594" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11452311/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20glyoxylate%20cycle%20is%20required%20for%20fungal%20virulence.&amp;author=M.%20C.%20Lorenz&amp;author=G.%20R.%20Fink&amp;volume=412&amp;publication_year=2001&amp;pages=83-86&amp;pmid=11452311&amp;doi=10.1038/35083594&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B52">
<cite>Martzivanou M., Babbick M., Cogoli-Greuter M., Hampp R. (2006). Microgravity-related changes in gene expression after short-term exposure of <em>Arabidopsis thaliana</em> cell cultures.
<em>Protoplasma</em>
229
155–162. 10.1007/s00709-006-0203-1
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00709-006-0203-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17180497/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Protoplasma&amp;title=Microgravity-related%20changes%20in%20gene%20expression%20after%20short-term%20exposure%20of%20Arabidopsis%20thaliana%20cell%20cultures.&amp;author=M.%20Martzivanou&amp;author=M.%20Babbick&amp;author=M.%20Cogoli-Greuter&amp;author=R.%20Hampp&amp;volume=229&amp;publication_year=2006&amp;pages=155-162&amp;pmid=17180497&amp;doi=10.1007/s00709-006-0203-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B53">
<cite>Mazars C., Briére C., Grat S., Pichereaux C., Rossignol M., Pereda-Loth V., et al.  (2014). Microgravity induces changes in microsome-associated proteins of <em>Arabidopsis</em> seedlings grown on board the international space station.
<em>PLoS One</em>
9:e91814. 10.1371/journal.pone.0091814
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0091814" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3950288/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24618597/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Microgravity%20induces%20changes%20in%20microsome-associated%20proteins%20of%20Arabidopsis%20seedlings%20grown%20on%20board%20the%20international%20space%20station.&amp;author=C.%20Mazars&amp;author=C.%20Bri%C3%A9re&amp;author=S.%20Grat&amp;author=C.%20Pichereaux&amp;author=M.%20Rossignol&amp;volume=9&amp;issue=e91814&amp;publication_year=2014&amp;pmid=24618597&amp;doi=10.1371/journal.pone.0091814&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B54">
<cite>Meti R. S., Ambarish S., Khajure P. V. (2011). Enzymes of ammonia assimilation in fungi: an overview.
<em>Sci. Technol.</em>
2
28–38.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Technol.&amp;title=Enzymes%20of%20ammonia%20assimilation%20in%20fungi:%20an%20overview.&amp;author=R.%20S.%20Meti&amp;author=S.%20Ambarish&amp;author=P.%20V.%20Khajure&amp;volume=2&amp;publication_year=2011&amp;pages=28-38&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B55">
<cite>Miura N., Ueda M. (2018). Evaluation of unconventional protein secretion by <em>Saccharomyces cerevisiae</em> and other Fungi.
<em>Cells</em>
7:128. 10.3390/cells7090128
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/cells7090128" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6162777/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30200367/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Evaluation%20of%20unconventional%20protein%20secretion%20by%20Saccharomyces%20cerevisiae%20and%20other%20Fungi.&amp;author=N.%20Miura&amp;author=M.%20Ueda&amp;volume=7&amp;issue=128&amp;publication_year=2018&amp;pmid=30200367&amp;doi=10.3390/cells7090128&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B56">
<cite>Miyazaki Y., Sunagawa M., Higashibata A., Ishioka N., Babasaki K., Yamazaki T. (2010). Differentially expressed genes under simulated microgravity in fruiting bodies of the fungus <em>Pleurotus ostreatus</em>.
<em>FEMS Microbiol. Lett.</em>
307
72–79. 10.1111/j.1574-6968.2010.01966.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1574-6968.2010.01966.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/20402784/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Lett.&amp;title=Differentially%20expressed%20genes%20under%20simulated%20microgravity%20in%20fruiting%20bodies%20of%20the%20fungus%20Pleurotus%20ostreatus.&amp;author=Y.%20Miyazaki&amp;author=M.%20Sunagawa&amp;author=A.%20Higashibata&amp;author=N.%20Ishioka&amp;author=K.%20Babasaki&amp;volume=307&amp;publication_year=2010&amp;pages=72-79&amp;pmid=20402784&amp;doi=10.1111/j.1574-6968.2010.01966.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B57">
<cite>Moore J. D., Yazgan O., Ataian Y., Krebs J. E. (2007). Diverse roles for histone H2A modifications in DNA damage response pathways in yeast.
<em>Genetics</em>
25
15–25. 10.1534/genetics.106.063792
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1534/genetics.106.063792" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1893062/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17028320/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Diverse%20roles%20for%20histone%20H2A%20modifications%20in%20DNA%20damage%20response%20pathways%20in%20yeast.&amp;author=J.%20D.%20Moore&amp;author=O.%20Yazgan&amp;author=Y.%20Ataian&amp;author=J.%20E.%20Krebs&amp;volume=25&amp;publication_year=2007&amp;pages=15-25&amp;pmid=17028320&amp;doi=10.1534/genetics.106.063792&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B58">
<cite>Morrison M. D., Fajardo-Cavazos P., Nicholson W. L. (2019). Comparison of <em>Bacillus subtilis</em> transcriptome profiles from two separate missions to the International Space Station.
<em>NPJ Microgravity</em>
5:1. 10.1038/s41526-018-0061-0
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41526-018-0061-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6323116/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30623021/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Microgravity&amp;title=Comparison%20of%20Bacillus%20subtilis%20transcriptome%20profiles%20from%20two%20separate%20missions%20to%20the%20International%20Space%20Station.&amp;author=M.%20D.%20Morrison&amp;author=P.%20Fajardo-Cavazos&amp;author=W.%20L.%20Nicholson&amp;volume=5&amp;issue=1&amp;publication_year=2019&amp;pmid=30623021&amp;doi=10.1038/s41526-018-0061-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B59">
<cite>Mouyna I., Dellière S., Beauvais A., Gravelat F., Carrion S. D. J., Pearlman E., et al.  (2020). What are the functions of chitin deacetylases in <em>Aspergillus fumigatus</em>?
<em>Front. Cell. Infect. Microbiol.</em>
10:28. 10.3389/fcimb.2020.00028
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fcimb.2020.00028" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7016196/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32117802/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell.%20Infect.%20Microbiol.&amp;title=What%20are%20the%20functions%20of%20chitin%20deacetylases%20in%20Aspergillus%20fumigatus?&amp;author=I.%20Mouyna&amp;author=S.%20Delli%C3%A8re&amp;author=A.%20Beauvais&amp;author=F.%20Gravelat&amp;author=S.%20D.%20J.%20Carrion&amp;volume=10&amp;issue=28&amp;publication_year=2020&amp;pmid=32117802&amp;doi=10.3389/fcimb.2020.00028&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B60">
<cite>Nai C., Wong H. Y., Pannenbecker A., Broughton W. J., Benoit I., de Vries R. P., et al.  (2013). Nutritional physiology of a rock-inhabiting, model microcolonial fungus from an ancestral lineage of the Chaetothyriales (<em>Ascomycetes</em>).
<em>Fungal Genet. Biol.</em>
56
54–66. 10.1016/j.fgb.2013.04.001
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.fgb.2013.04.001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23587800/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&amp;title=Nutritional%20physiology%20of%20a%20rock-inhabiting,%20model%20microcolonial%20fungus%20from%20an%20ancestral%20lineage%20of%20the%20Chaetothyriales%20(Ascomycetes).&amp;author=C.%20Nai&amp;author=H.%20Y.%20Wong&amp;author=A.%20Pannenbecker&amp;author=W.%20J.%20Broughton&amp;author=I.%20Benoit&amp;volume=56&amp;publication_year=2013&amp;pages=54-66&amp;pmid=23587800&amp;doi=10.1016/j.fgb.2013.04.001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B61">
<cite>Nakamura H., Narihiro T., Tsuruoka N., Mochimaru H., Matsumoto R., Tanabe Y., et al.  (2011). Evaluation of the aflatoxin biosynthetic genes for identification of the <em>Aspergillus</em> section Flavi.
<em>Microbes Environ.</em>
26
367–369. 10.1264/jsme2.ME11201
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1264/jsme2.ME11201" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21791886/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbes%20Environ.&amp;title=Evaluation%20of%20the%20aflatoxin%20biosynthetic%20genes%20for%20identification%20of%20the%20Aspergillus%20section%20Flavi.&amp;author=H.%20Nakamura&amp;author=T.%20Narihiro&amp;author=N.%20Tsuruoka&amp;author=H.%20Mochimaru&amp;author=R.%20Matsumoto&amp;volume=26&amp;publication_year=2011&amp;pages=367-369&amp;pmid=21791886&amp;doi=10.1264/jsme2.ME11201&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B62">
<cite>Neto Y. A. A. H., Garzon N. G. D. R., Pedezzi R., Cabral H. (2018). Specificity of peptidases secreted by filamentous fungi.
<em>Bioengineered</em>
9
30–37. 10.1080/21655979.2017.1373531
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1080/21655979.2017.1373531" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5972931/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28857638/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioengineered&amp;title=Specificity%20of%20peptidases%20secreted%20by%20filamentous%20fungi.&amp;author=Y.%20A.%20A.%20H.%20Neto&amp;author=N.%20G.%20D.%20R.%20Garzon&amp;author=R.%20Pedezzi&amp;author=H.%20Cabral&amp;volume=9&amp;publication_year=2018&amp;pages=30-37&amp;pmid=28857638&amp;doi=10.1080/21655979.2017.1373531&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B63">
<cite>Onofri S., Barreca D., Selbmann L., Isola D., Rabbow E., Horneck G., et al.  (2008). Resistance of Antarctic black fungi and cryptoendolithic communities to simulated space and Martian conditions.
<em>Stud. Mycol.</em>
61
99–109. 10.3114/sim.2008.61.10
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3114/sim.2008.61.10" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2610303/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19287532/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Stud.%20Mycol.&amp;title=Resistance%20of%20Antarctic%20black%20fungi%20and%20cryptoendolithic%20communities%20to%20simulated%20space%20and%20Martian%20conditions.&amp;author=S.%20Onofri&amp;author=D.%20Barreca&amp;author=L.%20Selbmann&amp;author=D.%20Isola&amp;author=E.%20Rabbow&amp;volume=61&amp;publication_year=2008&amp;pages=99-109&amp;pmid=19287532&amp;doi=10.3114/sim.2008.61.10&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B64">
<cite>Onofri S., de la Torre R., de Vera J.-P., Ott S., Zucconi L., Selbmann L., et al.  (2012). Survival of rock-colonizing organisms after 1.5 years in outer space.
<em>Astrobiology</em>
12
508–516. 10.1089/ast.2011.0736
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2011.0736" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22680696/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Survival%20of%20rock-colonizing%20organisms%20after%201.5%20years%20in%20outer%20space.&amp;author=S.%20Onofri&amp;author=R.%20de%20la%20Torre&amp;author=J.-P.%20de%20Vera&amp;author=S.%20Ott&amp;author=L.%20Zucconi&amp;volume=12&amp;publication_year=2012&amp;pages=508-516&amp;pmid=22680696&amp;doi=10.1089/ast.2011.0736&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B65">
<cite>Onofri S., Selbmann L., de Hoog G. S., Grube M., Barreca D., Ruisi S., et al.  (2007). Evolution and adaptation of fungi at boundaries of life.
<em>Adv. Space Res.</em>
40
1657–1664. 10.1016/j.asr.2007.06.004</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.asr.2007.06.004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res.&amp;title=Evolution%20and%20adaptation%20of%20fungi%20at%20boundaries%20of%20life.&amp;author=S.%20Onofri&amp;author=L.%20Selbmann&amp;author=G.%20S.%20de%20Hoog&amp;author=M.%20Grube&amp;author=D.%20Barreca&amp;volume=40&amp;publication_year=2007&amp;pages=1657-1664&amp;doi=10.1016/j.asr.2007.06.004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B66">
<cite>Onofri S., Selbmann L., Pacelli C., Zucconi L., Rabbow E., De Vera J. P. (2019). Survival, DNA, and ultrastructural integrity of a cryptoendolithic Antarctic fungus in Mars and Lunar Rock analogs exposed outside the International Space Station.
<em>Astrobiology</em>
19
170–182. 10.1089/ast.2017.1728
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2017.1728" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30376361/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Survival,%20DNA,%20and%20ultrastructural%20integrity%20of%20a%20cryptoendolithic%20Antarctic%20fungus%20in%20Mars%20and%20Lunar%20Rock%20analogs%20exposed%20outside%20the%20International%20Space%20Station.&amp;author=S.%20Onofri&amp;author=L.%20Selbmann&amp;author=C.%20Pacelli&amp;author=L.%20Zucconi&amp;author=E.%20Rabbow&amp;volume=19&amp;publication_year=2019&amp;pages=170-182&amp;pmid=30376361&amp;doi=10.1089/ast.2017.1728&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B67">
<cite>Pacelli C., Bryan R. A., Onofri S., Selbmann L., Zucconi L., Shuryak I., et al.  (2018). Survival and redox activity of Friedmanniomyces endolithicus, an Antarctic endemic black meristematic fungus, after gamma rays exposure.
<em>Fungal Biol.</em>
122
1222–1227. 10.1016/j.funbio.2018.10.002
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.funbio.2018.10.002" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30449360/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Biol.&amp;title=Survival%20and%20redox%20activity%20of%20Friedmanniomyces%20endolithicus,%20an%20Antarctic%20endemic%20black%20meristematic%20fungus,%20after%20gamma%20rays%20exposure.&amp;author=C.%20Pacelli&amp;author=R.%20A.%20Bryan&amp;author=S.%20Onofri&amp;author=L.%20Selbmann&amp;author=L.%20Zucconi&amp;volume=122&amp;publication_year=2018&amp;pages=1222-1227&amp;pmid=30449360&amp;doi=10.1016/j.funbio.2018.10.002&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B68">
<cite>Pacelli C., Selbmann L., Zucconi L., De Vera J.-P., Rabbow E., Horneck G., et al.  (2016). BIOMEX experiment: ultrastructural alterations, molecular damage and survival of the fungus cryomyces antarcticus after the experiment verification tests.
<em>Orig. Life Evol. Biosph.</em>
47
187–202. 10.1007/s11084-016-9485-2
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11084-016-9485-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27033201/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Orig.%20Life%20Evol.%20Biosph.&amp;title=BIOMEX%20experiment:%20ultrastructural%20alterations,%20molecular%20damage%20and%20survival%20of%20the%20fungus%20cryomyces%20antarcticus%20after%20the%20experiment%20verification%20tests.&amp;author=C.%20Pacelli&amp;author=L.%20Selbmann&amp;author=L.%20Zucconi&amp;author=J.-P.%20De%20Vera&amp;author=E.%20Rabbow&amp;volume=47&amp;publication_year=2016&amp;pages=187-202&amp;pmid=27033201&amp;doi=10.1007/s11084-016-9485-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B69">
<cite>Pacelli C., Selbmann L., Zucconi L., Raguse M., Moeller R., Shuryak I., et al.  (2017). Survival, DNA integrity, and ultrastructural damage in antarctic cryptoendolithic eukaryotic microorganisms exposed to ionizing radiation.
<em>Astrobiology</em>
17
126–135. 10.1089/ast.2015.1456
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2015.1456" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5314979/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28151696/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Survival,%20DNA%20integrity,%20and%20ultrastructural%20damage%20in%20antarctic%20cryptoendolithic%20eukaryotic%20microorganisms%20exposed%20to%20ionizing%20radiation.&amp;author=C.%20Pacelli&amp;author=L.%20Selbmann&amp;author=L.%20Zucconi&amp;author=M.%20Raguse&amp;author=R.%20Moeller&amp;volume=17&amp;publication_year=2017&amp;pages=126-135&amp;pmid=28151696&amp;doi=10.1089/ast.2015.1456&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B70">
<cite>Pao S. S., Paulsen I. T. S. M. (1998). Major facilitator superfamily.
<em>Microbiol. Mol. Biol. Rev</em>.
62
1–34.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/mmbr.62.1.1-34.1998" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC98904/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/9529885/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;title=Major%20facilitator%20superfamily.&amp;author=S.%20S.%20Pao&amp;author=I.%20T.%20S.%20M.%20Paulsen&amp;volume=62&amp;publication_year=1998&amp;pages=1-34&amp;pmid=9529885&amp;doi=10.1128/mmbr.62.1.1-34.1998&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B71">
<cite>Passoth V. (2017). “Lipids of yeasts and filamentous fungi and their importance for biotechnology BT,” in <em>Biotechnology of Yeasts and Filamentous Fungi</em>, ed.
Sibirny A. A. (Cham: Springer International Publishing; ), 149–204. 10.1007/978-3-319-58829-2_6</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-3-319-58829-2_6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Biotechnology%20of%20Yeasts%20and%20Filamentous%20Fungi&amp;author=V.%20Passoth&amp;publication_year=2017&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B72">
<cite>Rabbow E., Rettberg P., Barczyk S., Bohmeier M., Parpart A., Panitz C., et al.  (2012). EXPOSE-E: an ESA astrobiology mission 1.5 years in space.
<em>Astrobiology</em>
12
374–386. 10.1089/ast.2011.0760
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2011.0760" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22680684/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=EXPOSE-E:%20an%20ESA%20astrobiology%20mission%201.5%20years%20in%20space.&amp;author=E.%20Rabbow&amp;author=P.%20Rettberg&amp;author=S.%20Barczyk&amp;author=M.%20Bohmeier&amp;author=A.%20Parpart&amp;volume=12&amp;publication_year=2012&amp;pages=374-386&amp;pmid=22680684&amp;doi=10.1089/ast.2011.0760&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B73">
<cite>Rabbow E., Rettberg P., Parpart A., Panitz C., Schulte W., Molter F., et al.  (2017). EXPOSE-R2: the astrobiological ESA mission on board of the international space station.
<em>Front. Microbiol.</em>
8:1533. 10.3389/fmicb.2017.01533
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2017.01533" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5560112/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28861052/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=EXPOSE-R2:%20the%20astrobiological%20ESA%20mission%20on%20board%20of%20the%20international%20space%20station.&amp;author=E.%20Rabbow&amp;author=P.%20Rettberg&amp;author=A.%20Parpart&amp;author=C.%20Panitz&amp;author=W.%20Schulte&amp;volume=8&amp;issue=1533&amp;publication_year=2017&amp;pmid=28861052&amp;doi=10.3389/fmicb.2017.01533&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B74">
<cite>Raben D. M., Wolfgang M. J. (2019). <em>Phosphofructokinase-2/Fructose Bisphosphatase-2✩.</em>
Amsterdam: Elsevier. 10.1016/B978-0-12-801238-3.11340-6</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/B978-0-12-801238-3.11340-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Phosphofructokinase-2/Fructose%20Bisphosphatase-2%E2%9C%A9.&amp;author=D.%20M.%20Raben&amp;author=M.%20J.%20Wolfgang&amp;publication_year=2019&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B75">
<cite>Regente M., Pinedo M., Elizalde M. (2012). Apoplastic exosome-like vesicles: a new way of protein secretion in plants?
<em>Plant Signal Behav.</em>
7
544–546.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4161/psb.19675" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3419015/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22516827/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signal%20Behav.&amp;title=Apoplastic%20exosome-like%20vesicles:%20a%20new%20way%20of%20protein%20secretion%20in%20plants?&amp;author=M.%20Regente&amp;author=M.%20Pinedo&amp;author=M.%20Elizalde&amp;volume=7&amp;publication_year=2012&amp;pages=544-546&amp;pmid=22516827&amp;doi=10.4161/psb.19675&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B76">
<cite>Romsdahl J., Blachowicz A., Chiang A. J., Chiang Y. M., Masonjones S., Yaegashi J., et al.  (2019). International Space Station conditions alter genomics, proteomics, and metabolomics in <em>Aspergillus nidulans</em>.
<em>Appl. Microbiol. Biotechnol.</em>
103
1363–1377. 10.1007/s00253-018-9525-0
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00253-018-9525-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9098231/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30539259/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&amp;title=International%20Space%20Station%20conditions%20alter%20genomics,%20proteomics,%20and%20metabolomics%20in%20Aspergillus%20nidulans.&amp;author=J.%20Romsdahl&amp;author=A.%20Blachowicz&amp;author=A.%20J.%20Chiang&amp;author=Y.%20M.%20Chiang&amp;author=S.%20Masonjones&amp;volume=103&amp;publication_year=2019&amp;pages=1363-1377&amp;pmid=30539259&amp;doi=10.1007/s00253-018-9525-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B77">
<cite>Romsdahl J., Blachowicz A., Chiang A. J., Singh N., Stajich J. E., Kalkum M., et al.  (2018). Characterization of <em>Aspergillus niger</em> isolated from the International Space Station.
<em>mSystems</em>
3
1–13. 10.1128/msystems.00112-18
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/msystems.00112-18" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6143729/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30246146/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Characterization%20of%20Aspergillus%20niger%20isolated%20from%20the%20International%20Space%20Station.&amp;author=J.%20Romsdahl&amp;author=A.%20Blachowicz&amp;author=A.%20J.%20Chiang&amp;author=N.%20Singh&amp;author=J.%20E.%20Stajich&amp;volume=3&amp;publication_year=2018&amp;pages=1-13&amp;pmid=30246146&amp;doi=10.1128/msystems.00112-18&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B78">
<cite>Rosenzweig J. A., Abogunde O., Thomas K., Lawal A., Nguyen Y. U., Sodipe A., et al.  (2010). Spaceflight and modeled microgravity effects on microbial growth and virulence.
<em>Appl. Microbiol. Biotechnol.</em>
85
885–891. 10.1007/s00253-009-2237-8
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00253-009-2237-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2804794/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19847423/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&amp;title=Spaceflight%20and%20modeled%20microgravity%20effects%20on%20microbial%20growth%20and%20virulence.&amp;author=J.%20A.%20Rosenzweig&amp;author=O.%20Abogunde&amp;author=K.%20Thomas&amp;author=A.%20Lawal&amp;author=Y.%20U.%20Nguyen&amp;volume=85&amp;publication_year=2010&amp;pages=885-891&amp;pmid=19847423&amp;doi=10.1007/s00253-009-2237-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B79">
<cite>Sathishkumar Y., Krishnaraj C., Rajagopal K., Sen D., Lee Y. S. (2016). High throughput de novo RNA sequencing elucidates novel responses in <em>Penicillium chrysogenum</em> under microgravity.
<em>Bioprocess Biosyst. Eng.</em>
39
223–231. 10.1007/s00449-015-1506-4
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00449-015-1506-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26603994/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Bioprocess%20Biosyst.%20Eng.&amp;title=High%20throughput%20de%20novo%20RNA%20sequencing%20elucidates%20novel%20responses%20in%20Penicillium%20chrysogenum%20under%20microgravity.&amp;author=Y.%20Sathishkumar&amp;author=C.%20Krishnaraj&amp;author=K.%20Rajagopal&amp;author=D.%20Sen&amp;author=Y.%20S.%20Lee&amp;volume=39&amp;publication_year=2016&amp;pages=223-231&amp;pmid=26603994&amp;doi=10.1007/s00449-015-1506-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B80">
<cite>Sathishkumar Y., Velmurugan N., Lee H. M., Rajagopal K., Im C. K., Lee Y. S. (2014). Effect of low shear modeled microgravity on phenotypic and central chitin metabolism in the filamentous fungi <em>Aspergillus niger</em> and <em>Penicillium chrysogenum</em>.
<em>Antonie van Leeuwenhoek</em>
106
197–209. 10.1007/s10482-014-0181-9
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s10482-014-0181-9" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24803238/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Antonie%20van%20Leeuwenhoek&amp;title=Effect%20of%20low%20shear%20modeled%20microgravity%20on%20phenotypic%20and%20central%20chitin%20metabolism%20in%20the%20filamentous%20fungi%20Aspergillus%20niger%20and%20Penicillium%20chrysogenum.&amp;author=Y.%20Sathishkumar&amp;author=N.%20Velmurugan&amp;author=H.%20M.%20Lee&amp;author=K.%20Rajagopal&amp;author=C.%20K.%20Im&amp;volume=106&amp;publication_year=2014&amp;pages=197-209&amp;pmid=24803238&amp;doi=10.1007/s10482-014-0181-9&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B81">
<cite>Savojardo C., Martelli P. L., Fariselli P., Profiti G., Casadio R. (2018). BUSCA: an integrative web server to predict subcellular localization of proteins.
<em>Nucleic Acids Res.</em>
46
W459–W466. 10.1093/nar/gky320
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gky320" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6031068/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29718411/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=BUSCA:%20an%20integrative%20web%20server%20to%20predict%20subcellular%20localization%20of%20proteins.&amp;author=C.%20Savojardo&amp;author=P.%20L.%20Martelli&amp;author=P.%20Fariselli&amp;author=G.%20Profiti&amp;author=R.%20Casadio&amp;volume=46&amp;publication_year=2018&amp;pages=W459-W466&amp;pmid=29718411&amp;doi=10.1093/nar/gky320&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B82">
<cite>Scalzi G., Selbmann L., Zucconi L., Rabbow E., Horneck G., Albertano P., et al.  (2012). LIFE experiment: isolation of cryptoendolithic organisms from Antarctic colonized sandstone exposed to space and simulated Mars conditions on the International Space Station.
<em>Orig. Life Evol. Biosph.</em>
42
253–262. 10.1007/s11084-012-9282-5
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11084-012-9282-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22688852/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Orig.%20Life%20Evol.%20Biosph.&amp;title=LIFE%20experiment:%20isolation%20of%20cryptoendolithic%20organisms%20from%20Antarctic%20colonized%20sandstone%20exposed%20to%20space%20and%20simulated%20Mars%20conditions%20on%20the%20International%20Space%20Station.&amp;author=G.%20Scalzi&amp;author=L.%20Selbmann&amp;author=L.%20Zucconi&amp;author=E.%20Rabbow&amp;author=G.%20Horneck&amp;volume=42&amp;publication_year=2012&amp;pages=253-262&amp;pmid=22688852&amp;doi=10.1007/s11084-012-9282-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B83">
<cite>Schneider C. A., Rasband W. S., Eliceiri K. W. (2012). NIH image to imageJ: 25 years of image analysis.
<em>Nat. Methods</em>
9
671–675. 10.1038/nmeth.2089
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nmeth.2089" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5554542/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22930834/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=NIH%20image%20to%20imageJ:%2025%20years%20of%20image%20analysis.&amp;author=C.%20A.%20Schneider&amp;author=W.%20S.%20Rasband&amp;author=K.%20W.%20Eliceiri&amp;volume=9&amp;publication_year=2012&amp;pages=671-675&amp;pmid=22930834&amp;doi=10.1038/nmeth.2089&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B84">
<cite>Searles S. C., Woolley C. M., Petersen R. A., Hyman L. E., Nielsen-Preiss S. M. (2011). Modeled microgravity increases filamentation, biofilm formation, phenotypic switching, and antimicrobial resistance in <em>Candida albicans</em>.
<em>Astrobiology</em>
11
825–836. 10.1089/ast.2011.0664
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1089/ast.2011.0664" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21936634/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Modeled%20microgravity%20increases%20filamentation,%20biofilm%20formation,%20phenotypic%20switching,%20and%20antimicrobial%20resistance%20in%20Candida%20albicans.&amp;author=S.%20C.%20Searles&amp;author=C.%20M.%20Woolley&amp;author=R.%20A.%20Petersen&amp;author=L.%20E.%20Hyman&amp;author=S.%20M.%20Nielsen-Preiss&amp;volume=11&amp;publication_year=2011&amp;pages=825-836&amp;pmid=21936634&amp;doi=10.1089/ast.2011.0664&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B85">
<cite>Selbmann L., De Hoog G. S., Mazzaglia A., Friedmann E. I., Onofri S. (2005). Fungi at the edge of life: cryptoendolithic black fungi from Antarctic desert.
<em>Stud. Mycol.</em>
51
1–32.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Stud.%20Mycol.&amp;title=Fungi%20at%20the%20edge%20of%20life:%20cryptoendolithic%20black%20fungi%20from%20Antarctic%20desert.&amp;author=L.%20Selbmann&amp;author=G.%20S.%20De%20Hoog&amp;author=A.%20Mazzaglia&amp;author=E.%20I.%20Friedmann&amp;author=S.%20Onofri&amp;volume=51&amp;publication_year=2005&amp;pages=1-32&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B86">
<cite>Selbmann L., de Hoog G. S., Zucconi L., Isola D., Ruisi S., Gerrits van den Ende A. H. G., et al.  (2008). Drought meets acid: three new genera in a dothidealean clade of extremotolerant fungi.
<em>Stud. Mycol.</em>
61
1–20. 10.3114/sim.2008.61.01
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3114/sim.2008.61.01" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2610311/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19287523/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Stud.%20Mycol.&amp;title=Drought%20meets%20acid:%20three%20new%20genera%20in%20a%20dothidealean%20clade%20of%20extremotolerant%20fungi.&amp;author=L.%20Selbmann&amp;author=G.%20S.%20de%20Hoog&amp;author=L.%20Zucconi&amp;author=D.%20Isola&amp;author=S.%20Ruisi&amp;volume=61&amp;publication_year=2008&amp;pages=1-20&amp;pmid=19287523&amp;doi=10.3114/sim.2008.61.01&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B87">
<cite>Selbmann L., Zucconi L., Isola D., Onofri S. (2014). Rock black fungi: excellence in the extremes, from the Antarctic to space.
<em>Curr. Genet.</em>
61
335–345. 10.1007/s00294-014-0457-7
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00294-014-0457-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25381156/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Genet.&amp;title=Rock%20black%20fungi:%20excellence%20in%20the%20extremes,%20from%20the%20Antarctic%20to%20space.&amp;author=L.%20Selbmann&amp;author=L.%20Zucconi&amp;author=D.%20Isola&amp;author=S.%20Onofri&amp;volume=61&amp;publication_year=2014&amp;pages=335-345&amp;pmid=25381156&amp;doi=10.1007/s00294-014-0457-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B88">
<cite>Senatore G., Mastroleo F., Leys N., Mauriello G. (2018). Effect of microgravity &amp; space radiation on microbes.
<em>Future Microbiol.</em>
13
831–847. 10.2217/fmb-2017-0251
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2217/fmb-2017-0251" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29745771/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Future%20Microbiol.&amp;title=Effect%20of%20microgravity%20&amp;%20space%20radiation%20on%20microbes.&amp;author=G.%20Senatore&amp;author=F.%20Mastroleo&amp;author=N.%20Leys&amp;author=G.%20Mauriello&amp;volume=13&amp;publication_year=2018&amp;pages=831-847&amp;pmid=29745771&amp;doi=10.2217/fmb-2017-0251&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B89">
<cite>Serra-Cardona A., Canadell D., Ariño J. (2015). Coordinate responses to alkaline pH stress in budding yeast.
<em>Microb. Cell</em>
2
182–196. 10.15698/mic2015.06.205
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.15698/mic2015.06.205" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5349140/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28357292/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microb.%20Cell&amp;title=Coordinate%20responses%20to%20alkaline%20pH%20stress%20in%20budding%20yeast.&amp;author=A.%20Serra-Cardona&amp;author=D.%20Canadell&amp;author=J.%20Ari%C3%B1o&amp;volume=2&amp;publication_year=2015&amp;pages=182-196&amp;pmid=28357292&amp;doi=10.15698/mic2015.06.205&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B90">
<cite>Sheehan K. B., McInnerney K., Purevdorj-Gage B., Altenburg S. D., Hyman L. E. (2007). Yeast genomic expression patterns in response to low-shear modeled microgravity.
<em>BMC Genomics</em>
8:3. 10.1186/1471-2164-8-3
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1471-2164-8-3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1774566/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17201921/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Yeast%20genomic%20expression%20patterns%20in%20response%20to%20low-shear%20modeled%20microgravity.&amp;author=K.%20B.%20Sheehan&amp;author=K.%20McInnerney&amp;author=B.%20Purevdorj-Gage&amp;author=S.%20D.%20Altenburg&amp;author=L.%20E.%20Hyman&amp;volume=8&amp;issue=3&amp;publication_year=2007&amp;pmid=17201921&amp;doi=10.1186/1471-2164-8-3&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B91">
<cite>Shtarkman Y. M., Koçer Z. A., Edgar R., Veerapaneni R. S., D’Elia T., Morris P. F., et al.  (2013). Subglacial Lake Vostok (Antarctica) accretion ice contains a diverse set of sequences from aquatic, marine and sediment-inhabiting bacteria and eukarya.
<em>PLoS One</em>
8:e67221. 10.1371/journal.pone.0067221
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0067221" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3700977/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23843994/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Subglacial%20Lake%20Vostok%20(Antarctica)%20accretion%20ice%20contains%20a%20diverse%20set%20of%20sequences%20from%20aquatic,%20marine%20and%20sediment-inhabiting%20bacteria%20and%20eukarya.&amp;author=Y.%20M.%20Shtarkman&amp;author=Z.%20A.%20Ko%C3%A7er&amp;author=R.%20Edgar&amp;author=R.%20S.%20Veerapaneni&amp;author=T.%20D%E2%80%99Elia&amp;volume=8&amp;issue=e67221&amp;publication_year=2013&amp;pmid=23843994&amp;doi=10.1371/journal.pone.0067221&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B92">
<cite>Sieber M., Hanke W., Kohn F. P. M. (2014). Modification of membrane fluidity by gravity.
<em>Open J. Biophys.</em>
04
105–111. 10.4236/ojbiphy.2014.44012</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.4236/ojbiphy.2014.44012" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Open%20J.%20Biophys.&amp;title=Modification%20of%20membrane%20fluidity%20by%20gravity.&amp;author=M.%20Sieber&amp;author=W.%20Hanke&amp;author=F.%20P.%20M.%20Kohn&amp;volume=04&amp;publication_year=2014&amp;pages=105-111&amp;doi=10.4236/ojbiphy.2014.44012&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B93">
<cite>Skoneczna A., Krol K., Skoneczny M. (2018). “How do yeast and other fungi recognize and respond to genome perturbations?,” in <em>Stress Response Mechanisms in Fungi</em>, ed.
Skoneczny M. (Cham: Springer; ), 87–130. 10.1007/978-3-030-00683-9_3</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-3-030-00683-9_3" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Stress%20Response%20Mechanisms%20in%20Fungi&amp;author=A.%20Skoneczna&amp;author=K.%20Krol&amp;author=M.%20Skoneczny&amp;publication_year=2018&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B94">
<cite>Skopelitis D. S., Paranychianakis N. V., Paschalidis K. A., Pliakonis E. D., Delis I. D., Yakoumakis D. I., et al.  (2006). Abiotic stress generates ROS that signal expression of anionic glutamate dehydrogenases to form glutamate for proline synthesis in tobacco and grapevine.
<em>Plant Cell</em>
18
2767–2781. 10.1105/tpc.105.038323
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1105/tpc.105.038323" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1626620/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/17041150/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Abiotic%20stress%20generates%20ROS%20that%20signal%20expression%20of%20anionic%20glutamate%20dehydrogenases%20to%20form%20glutamate%20for%20proline%20synthesis%20in%20tobacco%20and%20grapevine.&amp;author=D.%20S.%20Skopelitis&amp;author=N.%20V.%20Paranychianakis&amp;author=K.%20A.%20Paschalidis&amp;author=E.%20D.%20Pliakonis&amp;author=I.%20D.%20Delis&amp;volume=18&amp;publication_year=2006&amp;pages=2767-2781&amp;pmid=17041150&amp;doi=10.1105/tpc.105.038323&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B95">
<cite>Spreghini E., Davis D. A., Subaran R., Kim M., Mitchell A. P. (2003). Roles of <em>Candida albicans</em> Dfg5p and Dcw1p cell surface proteins in growth and hypha formation.
<em>Eukaryot. Cell</em>
2
746–755. 10.1128/EC.2.4.746</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/EC.2.4.746" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC178370/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12912894/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Eukaryot.%20Cell&amp;title=Roles%20of%20Candida%20albicans%20Dfg5p%20and%20Dcw1p%20cell%20surface%20proteins%20in%20growth%20and%20hypha%20formation.&amp;author=E.%20Spreghini&amp;author=D.%20A.%20Davis&amp;author=R.%20Subaran&amp;author=M.%20Kim&amp;author=A.%20P.%20Mitchell&amp;volume=2&amp;publication_year=2003&amp;pages=746-755&amp;pmid=12912894&amp;doi=10.1128/EC.2.4.746&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B96">
<cite>Sterflinger K. (2006). “Black yeasts and meristematic fungi: ecology, diversity and identification,” in <em>Biodiversity and Ecophysiology of Yeasts. The Yeast Handbook</em>, eds
Péter G., Rosa C. (Heidelberg: Springer; ). 10.1007/3-540-30985-3_20</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/3-540-30985-3_20" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?title=Biodiversity%20and%20Ecophysiology%20of%20Yeasts.%20The%20Yeast%20Handbook&amp;author=K.%20Sterflinger&amp;publication_year=2006&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B97">
<cite>Sterflinger K., de Hoog G. S., Haase G. (1999). Phylogeny and ecology of meristematic ascomycetes.
<em>Stud. Mycol.</em>
43
5–22.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Stud.%20Mycol.&amp;title=Phylogeny%20and%20ecology%20of%20meristematic%20ascomycetes.&amp;author=K.%20Sterflinger&amp;author=G.%20S.%20de%20Hoog&amp;author=G.%20Haase&amp;volume=43&amp;publication_year=1999&amp;pages=5-22&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B98">
<cite>Sterflinger K., Tesei D., Zakharova K. (2012). Fungi in hot and cold deserts with particular reference to microcolonial fungi.
<em>Fungal Ecol.</em>
5
453–462. 10.1016/j.funeco.2011.12.007</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.funeco.2011.12.007" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Ecol.&amp;title=Fungi%20in%20hot%20and%20cold%20deserts%20with%20particular%20reference%20to%20microcolonial%20fungi.&amp;author=K.%20Sterflinger&amp;author=D.%20Tesei&amp;author=K.%20Zakharova&amp;volume=5&amp;publication_year=2012&amp;pages=453-462&amp;doi=10.1016/j.funeco.2011.12.007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B99">
<cite>Sun X., Su X. (2019). Harnessing the knowledge of protein secretion for enhanced protein production in filamentous fungi.
<em>World J. Microbiol. Biotechnol.</em>
35
1–10. 10.1007/s11274-019-2630-0
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11274-019-2630-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30900052/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=World%20J.%20Microbiol.%20Biotechnol.&amp;title=Harnessing%20the%20knowledge%20of%20protein%20secretion%20for%20enhanced%20protein%20production%20in%20filamentous%20fungi.&amp;author=X.%20Sun&amp;author=X.%20Su&amp;volume=35&amp;publication_year=2019&amp;pages=1-10&amp;pmid=30900052&amp;doi=10.1007/s11274-019-2630-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B100">
<cite>Taylor P. W. (2015). Impact of space flight on bacterial virulence and antibiotic susceptibility.
<em>Infect. Drug Resist.</em>
8
249–262. 10.2147/IDR.S67275
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2147/IDR.S67275" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4524529/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26251622/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Drug%20Resist.&amp;title=Impact%20of%20space%20flight%20on%20bacterial%20virulence%20and%20antibiotic%20susceptibility.&amp;author=P.%20W.%20Taylor&amp;volume=8&amp;publication_year=2015&amp;pages=249-262&amp;pmid=26251622&amp;doi=10.2147/IDR.S67275&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B101">
<cite>Ter-Hovhannisyan V., Lomsadze A., Chernoff Y. O., Borodovsky M. (2008). Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training.
<em>Genome Res.</em>
18
1979–1990. 10.1101/gr.081612.108
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1101/gr.081612.108" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2593577/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18757608/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Gene%20prediction%20in%20novel%20fungal%20genomes%20using%20an%20ab%20initio%20algorithm%20with%20unsupervised%20training.&amp;author=V.%20Ter-Hovhannisyan&amp;author=A.%20Lomsadze&amp;author=Y.%20O.%20Chernoff&amp;author=M.%20Borodovsky&amp;volume=18&amp;publication_year=2008&amp;pages=1979-1990&amp;pmid=18757608&amp;doi=10.1101/gr.081612.108&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B102">
<cite>Tesei D., Marzban G., Marchetti-Deschmann M., Tafer H., Arcalis E., Sterflinger K. (2015). Proteome of tolerance fine-tuning in the human pathogen black yeast <em>Exophiala dermatitidis</em>.
<em>J. Proteomics</em>
128
39–57. 10.1016/j.jprot.2015.07.007
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jprot.2015.07.007" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26189359/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteomics&amp;title=Proteome%20of%20tolerance%20fine-tuning%20in%20the%20human%20pathogen%20black%20yeast%20Exophiala%20dermatitidis.&amp;author=D.%20Tesei&amp;author=G.%20Marzban&amp;author=M.%20Marchetti-Deschmann&amp;author=H.%20Tafer&amp;author=E.%20Arcalis&amp;volume=128&amp;publication_year=2015&amp;pages=39-57&amp;pmid=26189359&amp;doi=10.1016/j.jprot.2015.07.007&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B103">
<cite>Tesei D., Marzban G., Zakharova K., Isola D., Selbmann L., Sterflinger K. (2012). Alteration of protein patterns in black rock inhabiting fungi as a response to different temperatures.
<em>Fungal Biol.</em>
116
932–940. 10.1016/j.funbio.2012.06.004
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.funbio.2012.06.004" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3429300/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22862921/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Biol.&amp;title=Alteration%20of%20protein%20patterns%20in%20black%20rock%20inhabiting%20fungi%20as%20a%20response%20to%20different%20temperatures.&amp;author=D.%20Tesei&amp;author=G.%20Marzban&amp;author=K.%20Zakharova&amp;author=D.%20Isola&amp;author=L.%20Selbmann&amp;volume=116&amp;publication_year=2012&amp;pages=932-940&amp;pmid=22862921&amp;doi=10.1016/j.funbio.2012.06.004&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B104">
<cite>Tesei D., Quartinello F., Guebitz G. M., Ribitsch D., Nöbauer K., Razzazi-Fazeli E., et al.  (2020). Shotgun proteomics reveals putative polyesterases in the secretome of the rock-inhabiting fungus Knufia chersonesos.
<em>Sci. Rep.</em>
10:9770. 10.1038/s41598-020-66256-7
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-020-66256-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7299934/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32555357/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Shotgun%20proteomics%20reveals%20putative%20polyesterases%20in%20the%20secretome%20of%20the%20rock-inhabiting%20fungus%20Knufia%20chersonesos.&amp;author=D.%20Tesei&amp;author=F.%20Quartinello&amp;author=G.%20M.%20Guebitz&amp;author=D.%20Ribitsch&amp;author=K.%20N%C3%B6bauer&amp;volume=10&amp;issue=9770&amp;publication_year=2020&amp;pmid=32555357&amp;doi=10.1038/s41598-020-66256-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B105">
<cite>Tesei D., Tafer H., Poyntner C., Piñar G., Lopandic K., Sterflinger K. (2017). Draft genome sequences of the black rock fungus Knufia petricola and its spontaneous nonmelanized mutant.
<em>Genome Announc.</em>
5
1–2. 10.1128/genomeA.01242-17
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/genomeA.01242-17" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5668551/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29097475/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Announc.&amp;title=Draft%20genome%20sequences%20of%20the%20black%20rock%20fungus%20Knufia%20petricola%20and%20its%20spontaneous%20nonmelanized%20mutant.&amp;author=D.%20Tesei&amp;author=H.%20Tafer&amp;author=C.%20Poyntner&amp;author=G.%20Pi%C3%B1ar&amp;author=K.%20Lopandic&amp;volume=5&amp;publication_year=2017&amp;pages=1-2&amp;pmid=29097475&amp;doi=10.1128/genomeA.01242-17&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B106">
<cite>Trotter B., Otte K. A., Schoppmann K., Hemmersbach R., Fröhlich T., Arnold G. J., et al.  (2015). The influence of simulated microgravity on the proteome of <em>Daphnia magna</em>.
<em>NPJ Microgravity</em>
1:15016. 10.1038/npjmgrav.2015.16
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/npjmgrav.2015.16" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5515502/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28725717/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Microgravity&amp;title=The%20influence%20of%20simulated%20microgravity%20on%20the%20proteome%20of%20Daphnia%20magna.&amp;author=B.%20Trotter&amp;author=K.%20A.%20Otte&amp;author=K.%20Schoppmann&amp;author=R.%20Hemmersbach&amp;author=T.%20Fr%C3%B6hlich&amp;volume=1&amp;issue=15016&amp;publication_year=2015&amp;pmid=28725717&amp;doi=10.1038/npjmgrav.2015.16&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B107">
<cite>Urbaniak C., van Dam P., Zaborin A., Zaborina O., Gilbert J. A., Torok T., et al.  (2019). Genomic characterization and virulence potential of two <em>Fusarium oxysporum</em> isolates cultured from the International Space Station.
<em>mSystems</em>
4:e00345-18. 10.1128/msystems.00345-18e00345-18 </cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/msystems.00345-18" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6426649/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30944876/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Genomic%20characterization%20and%20virulence%20potential%20of%20two%20Fusarium%20oxysporum%20isolates%20cultured%20from%20the%20International%20Space%20Station.&amp;author=C.%20Urbaniak&amp;author=P.%20van%20Dam&amp;author=A.%20Zaborin&amp;author=O.%20Zaborina&amp;author=J.%20A.%20Gilbert&amp;volume=4&amp;issue=e00345-18&amp;publication_year=2019&amp;pmid=30944876&amp;doi=10.1128/msystems.00345-18&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B108">
<cite>Vallejo M. C., Nakayasu E. S., Matsuo A. L., Sobreira T. J. P., Longo L. V. G., Ganiko L., et al.  (2012). Vesicle and vesicle-free extracellular proteome of <em>Paracoccidioides brasiliensis</em>: comparative analysis with other pathogenic fungi.
<em>J. Proteome Res.</em>
11
1676–1685.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/pr200872s" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3319080/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22288420/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Vesicle%20and%20vesicle-free%20extracellular%20proteome%20of%20Paracoccidioides%20brasiliensis:%20comparative%20analysis%20with%20other%20pathogenic%20fungi.&amp;author=M.%20C.%20Vallejo&amp;author=E.%20S.%20Nakayasu&amp;author=A.%20L.%20Matsuo&amp;author=T.%20J.%20P.%20Sobreira&amp;author=L.%20V.%20G.%20Longo&amp;volume=11&amp;publication_year=2012&amp;pages=1676-1685&amp;pmid=22288420&amp;doi=10.1021/pr200872s&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B109">
<cite>Voigt O., Knabe N., Nitsche S., Erdmann E. A., Schumacher J., Gorbushina A. A. (2020). An advanced genetic toolkit for exploring the biology of the rock – inhabiting black fungus Knufia petricola.
<em>Sci. Rep.</em>
10:22021. 10.1038/s41598-020-79120-5
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-020-79120-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7745021/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33328531/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=An%20advanced%20genetic%20toolkit%20for%20exploring%20the%20biology%20of%20the%20rock%20%E2%80%93%20inhabiting%20black%20fungus%20Knufia%20petricola.&amp;author=O.%20Voigt&amp;author=N.%20Knabe&amp;author=S.%20Nitsche&amp;author=E.%20A.%20Erdmann&amp;author=J.%20Schumacher&amp;volume=10&amp;issue=22021&amp;publication_year=2020&amp;pmid=33328531&amp;doi=10.1038/s41598-020-79120-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B110">
<cite>Willaert R. G. (2013). The growth behavior of the model eukaryotic yeast <em>Saccharomyces cerevisiae</em> in microgravity.
<em>Curr. Biotechnol.</em>
2
226–234.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biotechnol.&amp;title=The%20growth%20behavior%20of%20the%20model%20eukaryotic%20yeast%20Saccharomyces%20cerevisiae%20in%20microgravity.&amp;author=R.%20G.%20Willaert&amp;volume=2&amp;publication_year=2013&amp;pages=226-234&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B111">
<cite>Xia Z., Zhou X., Li J., Li L., Ma Y., Wu Y., et al.  (2019). Multiple-omics techniques reveal the role of glycerophospholipid metabolic pathway in the response of <em>Saccharomyces cerevisiae</em> against hypoxic stress.
<em>Front. Microbiol.</em>
10:1398. 10.3389/fmicb.2019.01398
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2019.01398" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6610297/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31316482/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Multiple-omics%20techniques%20reveal%20the%20role%20of%20glycerophospholipid%20metabolic%20pathway%20in%20the%20response%20of%20Saccharomyces%20cerevisiae%20against%20hypoxic%20stress.&amp;author=Z.%20Xia&amp;author=X.%20Zhou&amp;author=J.%20Li&amp;author=L.%20Li&amp;author=Y.%20Ma&amp;volume=10&amp;issue=1398&amp;publication_year=2019&amp;pmid=31316482&amp;doi=10.3389/fmicb.2019.01398&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B112">
<cite>Yamaguchi N., Roberts M., Castro S., Oubre C., Makimura K., Leys N., et al.  (2014). Microbial monitoring of crewed habitats in space-current status and future perspectives.
<em>Microbes Environ.</em>
29
250–260. 10.1264/jsme2.ME14031
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1264/jsme2.ME14031" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4159036/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25130885/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbes%20Environ.&amp;title=Microbial%20monitoring%20of%20crewed%20habitats%20in%20space-current%20status%20and%20future%20perspectives.&amp;author=N.%20Yamaguchi&amp;author=M.%20Roberts&amp;author=S.%20Castro&amp;author=C.%20Oubre&amp;author=K.%20Makimura&amp;volume=29&amp;publication_year=2014&amp;pages=250-260&amp;pmid=25130885&amp;doi=10.1264/jsme2.ME14031&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B113">
<cite>Yamazaki T., Yoshimoto M., Nishiyama Y., Okubo Y., Makimura K. (2012). Phenotypic characterization of <em>Aspergillus niger</em> and <em>Candida albicans</em> grown under simulated microgravity using a three-dimensional clinostat.
<em>Microbiol. Immunol.</em>
56
441–446. 10.1111/j.1348-0421.2012.00471.x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1348-0421.2012.00471.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22537211/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Immunol.&amp;title=Phenotypic%20characterization%20of%20Aspergillus%20niger%20and%20Candida%20albicans%20grown%20under%20simulated%20microgravity%20using%20a%20three-dimensional%20clinostat.&amp;author=T.%20Yamazaki&amp;author=M.%20Yoshimoto&amp;author=Y.%20Nishiyama&amp;author=Y.%20Okubo&amp;author=K.%20Makimura&amp;volume=56&amp;publication_year=2012&amp;pages=441-446&amp;pmid=22537211&amp;doi=10.1111/j.1348-0421.2012.00471.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B114">
<cite>Yurimoto H., Lee B., Yano T., Sakai Y., Kato N. (2003). Physiological role of S-formylglutathione hydrolase in C1 metabolism of the methylotrophic yeast <em>Candida boidinii</em>.
<em>Microbiology</em>
149
1971–1979. 10.1099/mic.0.26320-0
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1099/mic.0.26320-0" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12904537/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Physiological%20role%20of%20S-formylglutathione%20hydrolase%20in%20C1%20metabolism%20of%20the%20methylotrophic%20yeast%20Candida%20boidinii.&amp;author=H.%20Yurimoto&amp;author=B.%20Lee&amp;author=T.%20Yano&amp;author=Y.%20Sakai&amp;author=N.%20Kato&amp;volume=149&amp;publication_year=2003&amp;pages=1971-1979&amp;pmid=12904537&amp;doi=10.1099/mic.0.26320-0&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B115">
<cite>Zakharova K., Marzban G., de Vera J.-P., Lorek A., Sterflinger K. (2014). Protein patterns of black fungi under simulated Mars-like conditions.
<em>Sci. Rep.</em>
4:5114. 10.1038/srep05114
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/srep05114" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4037706/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24870977/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Protein%20patterns%20of%20black%20fungi%20under%20simulated%20Mars-like%20conditions.&amp;author=K.%20Zakharova&amp;author=G.%20Marzban&amp;author=J.-P.%20de%20Vera&amp;author=A.%20Lorek&amp;author=K.%20Sterflinger&amp;volume=4&amp;issue=5114&amp;publication_year=2014&amp;pmid=24870977&amp;doi=10.1038/srep05114&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B116">
<cite>Zakharova K., Tesei D., Marzban G., Dijksterhuis J., Wyatt T., Sterflinger K. (2013). Microcolonial fungi on rocks: a life in constant drought?
<em>Mycopathologia</em>
175
537–547.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11046-012-9592-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3669513/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23073825/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mycopathologia&amp;title=Microcolonial%20fungi%20on%20rocks:%20a%20life%20in%20constant%20drought?&amp;author=K.%20Zakharova&amp;author=D.%20Tesei&amp;author=G.%20Marzban&amp;author=J.%20Dijksterhuis&amp;author=T.%20Wyatt&amp;volume=175&amp;publication_year=2013&amp;pages=537-547&amp;pmid=23073825&amp;doi=10.1007/s11046-012-9592-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B117">
<cite>Zdobnov E. M., Tegenfeldt F., Kuznetsov D., Waterhouse R. M., Simao F. A., Ioannidis P., et al.  (2017). OrthoDB v9.1: cataloging evolutionary and functional annotations for animal, fungal, plant, archaeal, bacterial and viral orthologs.
<em>Nucleic Acids Res.</em>
45
D744–D749. 10.1093/nar/gkw1119
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1093/nar/gkw1119" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5210582/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27899580/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=OrthoDB%20v9.1:%20cataloging%20evolutionary%20and%20functional%20annotations%20for%20animal,%20fungal,%20plant,%20archaeal,%20bacterial%20and%20viral%20orthologs.&amp;author=E.%20M.%20Zdobnov&amp;author=F.%20Tegenfeldt&amp;author=D.%20Kuznetsov&amp;author=R.%20M.%20Waterhouse&amp;author=F.%20A.%20Simao&amp;volume=45&amp;publication_year=2017&amp;pages=D744-D749&amp;pmid=27899580&amp;doi=10.1093/nar/gkw1119&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B118">
<cite>Zhang Y., Wang L., Xie J., Zheng H. (2015). Differential protein expression profiling of <em>Arabidopsis thaliana</em> callus under microgravity on board the Chinese SZ-8 spacecraft.
<em>Planta</em>
241
475–488. 10.1007/s00425-014-2196-x
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00425-014-2196-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25374148/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Differential%20protein%20expression%20profiling%20of%20Arabidopsis%20thaliana%20callus%20under%20microgravity%20on%20board%20the%20Chinese%20SZ-8%20spacecraft.&amp;author=Y.%20Zhang&amp;author=L.%20Wang&amp;author=J.%20Xie&amp;author=H.%20Zheng&amp;volume=241&amp;publication_year=2015&amp;pages=475-488&amp;pmid=25374148&amp;doi=10.1007/s00425-014-2196-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B119">
<cite>Zhang Z., Mao C., Shi Z., Kou X. (2017). The amino acid metabolic and carbohydrate metabolic pathway play important roles during salt-stress response in tomato.
<em>Front. Plant Sci.</em>
8:1231. 10.3389/fpls.2017.01231
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2017.01231" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5511834/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28769946/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&amp;title=The%20amino%20acid%20metabolic%20and%20carbohydrate%20metabolic%20pathway%20play%20important%20roles%20during%20salt-stress%20response%20in%20tomato.&amp;author=Z.%20Zhang&amp;author=C.%20Mao&amp;author=Z.%20Shi&amp;author=X.%20Kou&amp;volume=8&amp;issue=1231&amp;publication_year=2017&amp;pmid=28769946&amp;doi=10.3389/fpls.2017.01231&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B120">
<cite>Zhao H., Wu D., Kong F., Lin K., Zhang H., Li G. (2017). The <em>Arabidopsis thaliana</em> nuclear factor Y transcription factors.
<em>Front Plant Sci.</em>
7:2045. 10.3389/fpls.2016.02045
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fpls.2016.02045" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5222873/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28119722/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front%20Plant%20Sci.&amp;title=The%20Arabidopsis%20thaliana%20nuclear%20factor%20Y%20transcription%20factors.&amp;author=H.%20Zhao&amp;author=D.%20Wu&amp;author=F.%20Kong&amp;author=K.%20Lin&amp;author=H.%20Zhang&amp;volume=7&amp;issue=2045&amp;publication_year=2017&amp;pmid=28119722&amp;doi=10.3389/fpls.2016.02045&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B121">
<cite>Zhao Q., Zhang H., Wang T., Chen S., Dai S. (2013). Proteomics-based investigation of salt-responsive mechanisms in plant roots.
<em>J. Proteomics</em>
82
230–253.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.jprot.2013.01.024" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23385356/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteomics&amp;title=Proteomics-based%20investigation%20of%20salt-responsive%20mechanisms%20in%20plant%20roots.&amp;author=Q.%20Zhao&amp;author=H.%20Zhang&amp;author=T.%20Wang&amp;author=S.%20Chen&amp;author=S.%20Dai&amp;volume=82&amp;publication_year=2013&amp;pages=230-253&amp;pmid=23385356&amp;doi=10.1016/j.jprot.2013.01.024&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B122">
<cite>Zhou L., Hang J., Zhou Y., Wan R., Lu G., Yin P., et al.  (2014). Crystal structures of the Lsm complex bound to the 3′ end sequence of U6 small nuclear RNA.
<em>Nature</em>
506
116–120. 10.1038/nature12803
</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature12803" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24240276/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Crystal%20structures%20of%20the%20Lsm%20complex%20bound%20to%20the%203%E2%80%B2%20end%20sequence%20of%20U6%20small%20nuclear%20RNA.&amp;author=L.%20Zhou&amp;author=J.%20Hang&amp;author=Y.%20Zhou&amp;author=R.%20Wan&amp;author=G.%20Lu&amp;volume=506&amp;publication_year=2014&amp;pages=116-120&amp;pmid=24240276&amp;doi=10.1038/nature12803&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ol></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/8012687/bin/Data_Sheet_1.docx">Click here for additional data file.</a><sup> (3.2MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/8012687/bin/Data_Sheet_2.docx">Click here for additional data file.</a><sup> (3MB, docx) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>WGS data for <em>K. chersonesos</em> MA5789 are available in NCBI GenBank (GCA_002319055.1), under BioSample accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/SAMN07326825" rel="noopener noreferrer" target="_blank">SAMN07326825</a> and BioProject accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://ncbi.nlm.nih.gov/nucleotide/PRJNA393270" rel="noopener noreferrer" target="_blank">PRJNA393270</a>. The proteomics datasets generated during the current study are accessible through the ProteomeXchange Consortium (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="http://proteomecentral.proteomexchange.org" rel="noopener noreferrer" target="_blank">http://proteomecentral.proteomexchange.org</a>) via PRIDE with the data set identifier PXD022898.</p></section></section></section></section></article>
</main></body></html>