<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11127935/"/><title>Tardigrade secretory proteins protect biological structures from desiccation - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Commun Biol</button></div>. 2024 May 25;7:633. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s42003-024-06336-w" rel="noopener noreferrer" target="_blank">10.1038/s42003-024-06336-w</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Tardigrade secretory proteins protect biological structures from desiccation</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lim%20S%22%5BAuthor%5D"><span class="name western">Samuel Lim</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Samuel Lim</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA 02115 USA </div>
<div class="p">
<sup>2</sup>Wyss Institute for Biologically Inspired Engineering at Harvard University, Boston, MA 02115 USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lim%20S%22%5BAuthor%5D"><span class="name western">Samuel Lim</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reilly%20CB%22%5BAuthor%5D"><span class="name western">Charles B Reilly</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Charles B Reilly</span></h3>
<div class="p">
<sup>2</sup>Wyss Institute for Biologically Inspired Engineering at Harvard University, Boston, MA 02115 USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reilly%20CB%22%5BAuthor%5D"><span class="name western">Charles B Reilly</span></a>
</div>
</div>
<sup>2</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Barghouti%20Z%22%5BAuthor%5D"><span class="name western">Zeina Barghouti</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Zeina Barghouti</span></h3>
<div class="p">
<sup>3</sup>Department of Mechanical Engineering, Massachusetts Institute of Technology, Cambridge, MA 02139 USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Barghouti%20Z%22%5BAuthor%5D"><span class="name western">Zeina Barghouti</span></a>
</div>
</div>
<sup>3</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marelli%20B%22%5BAuthor%5D"><span class="name western">Benedetto Marelli</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Benedetto Marelli</span></h3>
<div class="p">
<sup>4</sup>Department of Civil and Environmental Engineering, Massachusetts Institute of Technology, Cambridge, MA 02139 USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marelli%20B%22%5BAuthor%5D"><span class="name western">Benedetto Marelli</span></a>
</div>
</div>
<sup>4</sup>, <a aria-describedby="id5" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Way%20JC%22%5BAuthor%5D"><span class="name western">Jeffrey C Way</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jeffrey C Way</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA 02115 USA </div>
<div class="p">
<sup>2</sup>Wyss Institute for Biologically Inspired Engineering at Harvard University, Boston, MA 02115 USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Way%20JC%22%5BAuthor%5D"><span class="name western">Jeffrey C Way</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id6" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Silver%20PA%22%5BAuthor%5D"><span class="name western">Pamela A Silver</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Pamela A Silver</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA 02115 USA </div>
<div class="p">
<sup>2</sup>Wyss Institute for Biologically Inspired Engineering at Harvard University, Boston, MA 02115 USA </div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Silver%20PA%22%5BAuthor%5D"><span class="name western">Pamela A Silver</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA 02115 USA </div>
<div id="Aff2">
<sup>2</sup>Wyss Institute for Biologically Inspired Engineering at Harvard University, Boston, MA 02115 USA </div>
<div id="Aff3">
<sup>3</sup>Department of Mechanical Engineering, Massachusetts Institute of Technology, Cambridge, MA 02139 USA </div>
<div id="Aff4">
<sup>4</sup>Department of Civil and Environmental Engineering, Massachusetts Institute of Technology, Cambridge, MA 02139 USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2023 Dec 5; Accepted 2024 May 15; Collection date 2024.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© The Author(s) 2024</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11127935  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/38796644/">38796644</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Tardigrades, microscopic animals that survive a broad range of environmental stresses, express a unique set of proteins termed tardigrade-specific intrinsically disordered proteins (TDPs). TDPs are often expressed at high levels in tardigrades upon desiccation, and appear to mediate stress adaptation. Here, we focus on the proteins belonging to the secreted family of tardigrade proteins termed secretory-abundant heat soluble (“SAHS”) proteins, and investigate their ability to protect diverse biological structures. Recombinantly expressed SAHS proteins prevent desiccated liposomes from fusion, and enhance desiccation tolerance of <em>E. coli</em> and <em>Rhizobium tropici</em> upon extracellular application. Molecular dynamics simulation and comparative structural analysis suggest a model by which SAHS proteins may undergo a structural transition upon desiccation, in which removal of water and solutes from a large internal cavity in SAHS proteins destabilizes the beta-sheet structure. These results highlight the potential application of SAHS proteins as stabilizing molecules for preservation of cells.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Subject terms:</strong> Proteins, Synthetic biology</p></section></section><section class="abstract" id="Abs2"><hr class="headless"/>
<p id="Par2">Tardigrade secretory-abundant heat soluble (SAHS) proteins protect biological structures such as liposomes and microbial cells from dehydration-induced damages, potentially through undergoing structural transition upon desiccation.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Water is the universal solvent essential for survival. However, several organisms demonstrate remarkable abilities to withstand an almost complete loss of water for a prolonged time and recover upon rehydration<sup><a aria-describedby="CR1" class="usa-link" href="#CR1">1</a>,<a aria-describedby="CR2" class="usa-link" href="#CR2">2</a></sup>. This phenomenon of “anhydrobiosis”, or “life without water” has attracted a great deal of attention because it may reveal Nature’s strategies to endure adverse environments and open up ways to achieve efficient preservation of living matter. However, the precise mechanisms behind anhydrobiosis remain elusive, as they may involve a complex combination of protective proteins, solutes, and cellular machinery working in a concert<sup><a aria-describedby="CR3" class="usa-link" href="#CR3">3</a></sup>.</p>
<p id="Par4">Several molecules have been identified as mediators of desiccation tolerance in various organisms. One is the disaccharide trehalose, which accumulates in a large quantity in animals such as brine shrimps or nematodes<sup><a aria-describedby="CR4" class="usa-link" href="#CR4">4</a></sup>. Trehalose uptake enhances the survival of yeast and human cells under drying conditions and may protect dehydrated cells by vitrification (turning the cell interior into a glass-like state)<sup><a aria-describedby="CR4" class="usa-link" href="#CR4">4</a>–<a aria-describedby="CR6" class="usa-link" href="#CR6">6</a></sup>. Several “intrinsically disordered proteins (IDPs)”, which lack fixed tertiary structure, have also been associated with desiccation tolerance<sup><a aria-describedby="CR2" class="usa-link" href="#CR2">2</a></sup>. For instance, late embryogenesis-abundant (LEA) proteins are tolerance molecules found in plant seeds and anhydrobiotic species<sup><a aria-describedby="CR7" class="usa-link" href="#CR7">7</a></sup>. LEA proteins are mostly unstructured in an aqueous environment and assume helical structure upon desiccation to alleviate damage due to drying<sup><a aria-describedby="CR7" class="usa-link" href="#CR7">7</a>,<a aria-describedby="CR8" class="usa-link" href="#CR8">8</a></sup>.</p>
<p id="Par5">Recent studies on the survival mode of tardigrades revealed novel classes of stress proteins. Tardigrades (water bears) are microscopic animals that can survive under a broad range of environmental stressors, including desiccation, high and low temperatures, radiation, and even exposure to outer space<sup><a aria-describedby="CR9" class="usa-link" href="#CR9">9</a>–<a aria-describedby="CR11" class="usa-link" href="#CR11">11</a></sup>. Tardigrades express LEA proteins, and also three different families termed “tardigrade disordered proteins” (TDPs) unique to the phylum Tardigrada<sup><a aria-describedby="CR12" class="usa-link" href="#CR12">12</a></sup>. Some of these TDPs are highly disordered and less likely to undergo aggregation at high temperatures. (The classification of TDPs as disordered proteins is based on a sequence-based prediction<sup><a aria-describedby="CR13" class="usa-link" href="#CR13">13</a></sup>.) Three different groups of TDPs were termed cytoplasmic-, mitochondrial- and secretory-abundant heat soluble (CAHS, MAHS, SAHS) proteins according to their putative subcellular localization (Fig. <a class="usa-link" href="#Fig1">1a</a>). In each family, the proteins show sequence similarity<sup><a aria-describedby="CR14" class="usa-link" href="#CR14">14</a>,<a aria-describedby="CR15" class="usa-link" href="#CR15">15</a></sup>. TDPs are either constitutively expressed at high levels or enriched upon desiccation and knocking out even one CAHS or SAHS gene can cause diminished desiccation tolerance<sup><a aria-describedby="CR13" class="usa-link" href="#CR13">13</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Classification and sequence analysis of tardigrade SAHS proteins.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11127935_42003_2024_6336_Fig1_HTML.jpg" target="_blank"><img alt="Fig. 1" class="graphic zoom-in" height="999" id="d33e336" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9982/11127935/ae6328978b56/42003_2024_6336_Fig1_HTML.jpg" width="776"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig1/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Classification of tardigrade-specific desiccation-protective proteins. Tardigrades express unique sets of proteins termed tardigrade-specific intrinsically disordered proteins (TDPs). TDPs are further classified into three different subgroups, Cytosolic-, Mitochondrial- and Secretory-abundant heat soluble (C/M/SAHS) proteins, based on their subcellular localization. Created with BioRender.com. <strong>b</strong> An alignment of SAHS1-13 of <em>R. varieornatus</em>, SAHS4 of <em>H. exemplaris</em>, and two FABP (fatty acid binding protein) sequences, with amino acids whose side chains point inward toward the open cavity highlighted in orange, and those with side chains in the hydrophobic core highlighted in cyan. Underlined amino acid sequences at the N-termini of SAHS9, 10 and 12 are newly identified based on analysis of the <em>R. varieornatus</em> genome sequence. Underlined and bold sequences are putative N-linked glycosylation sites based on the consensus sequence Asn-X-Ser/Thr (X not Pro).</p></figcaption></figure><p id="Par6">SAHS proteins are classified as secreted as they are generally encoded with signal sequences and are exported to the extracellular space upon expression in eukaryotes<sup><a aria-describedby="CR14" class="usa-link" href="#CR14">14</a></sup>. Sequence-based structural prediction suggested that, unlike CAHS and MAHS proteins that are estimated to be highly disordered, the 42 putative SAHS proteins from tardigrade species <em>Hypsibius exemplaris, Ramazzottius varieornatus, Paramacrobiotus richtersi</em> and <em>Milnesium tardigradum</em> are more ordered<sup><a aria-describedby="CR13" class="usa-link" href="#CR13">13</a></sup>. Fukuda et al. solved the structure of <em>R. varieornatus</em> SAHS1 and SAHS4<sup><a aria-describedby="CR16" class="usa-link" href="#CR16">16</a>,<a aria-describedby="CR17" class="usa-link" href="#CR17">17</a></sup>, experimentally verifying that, at least under hydrated conditions, the SAHS proteins have defined structures, and their description as “intrinsically disordered proteins” is not accurate. RvSAHS1 and 4 proteins share sequence and structural similarity to fatty acid binding proteins (FABPs): their structures include β-barrels with large internal cavities that correspond to the fatty acid binding pockets in FABPs<sup><a aria-describedby="CR16" class="usa-link" href="#CR16">16</a>,<a aria-describedby="CR17" class="usa-link" href="#CR17">17</a></sup>. The function of these large cavities in the SAHS proteins is unclear, as is their biochemical mechanism in protecting against desiccation. Additionally, AlphaFold structures are available on the protein structure database (PDB) for all SAHS proteins identified from <em>R. varieornatus</em> and <em>H. exemplaris</em>.</p>
<p id="Par7">In this study, we investigated SAHS proteins from tardigrades <em>R. varieornatus</em> and <em>Hypsibius exemplaris</em> for their potential to protect various biological structures from desiccation-induced damage. We expressed four different SAHS proteins and found that they can specifically protect liposomes and bacterial cells but not enzymes upon desiccation. SAHS proteins undergo structural changes under dehydration with trifluoroethanol. Finally, we performed molecular dynamic and bioinformatic analyses to identify structural features of the SAHS proteins that may contribute to desiccation protection. These studies lead to a model for SAHS protein action, and also illustrate that SAHS proteins may be practical preservatives for cells in commercial applications.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Protein sequence analysis and expression</h3>
<p id="Par8">We selected 12 SAHS proteins from <em>R. varieornatus</em> and one from <em>H. exemplaris</em> to test for expression in the <em>E. coli</em> cytoplasm (Table <a class="usa-link" href="#MOESM2">S1</a>). <em>R. varieornatus</em> is the most desiccation-resistant tardigrade and is well-characterized; a draft genome sequence is available, and crystal structures of two SAHS proteins from this organism have been solved<sup><a aria-describedby="CR16" class="usa-link" href="#CR16">16</a>,<a aria-describedby="CR17" class="usa-link" href="#CR17">17</a></sup>. We used the TargetP program<sup><a aria-describedby="CR18" class="usa-link" href="#CR18">18</a></sup> that computationally predicts protein subcellular localization based on the N-terminal signal peptide, which predicted that only 8 out of 12 SAHS proteins would be secreted (Table <a class="usa-link" href="#MOESM2">S2</a>). However, the surprising implication that some of the SAHS proteins might not be secreted also caused us to investigate genomic regions encoding these genes.</p>
<p id="Par9">We found that the “short” SAHS protein sequences in Uniprot that lack amino acids corresponding to the first ~40 amino acids of mature SAHS1 and SAHS4 may be incorrectly annotated; an alignment shown in Fig. 5 of Fukuda et al.<sup><a aria-describedby="CR16" class="usa-link" href="#CR16">16</a></sup> shows three such proteins of the 13 SAHS proteins encoded by <em>R. varieornatus</em>. For example, the DNA encoding RvSAHS9 contains a splice acceptor immediately upstream of the putative start codon, further preceded by a splice donor and an in-frame coding sequence that encodes an amino acid sequence that aligns well with the N-terminus of other SAHS proteins; RvSAHS10 and RvSAHS12 have a similar organization (Fig. <a class="usa-link" href="#Fig1">1b</a>; <a class="usa-link" href="#MOESM2">Supplementary Discussion</a>). In each case, the new putative intron has a consensus splice donor, but the polypyrimidine tract of the splice acceptor has rather few pyrimidines, which may have misled the automated systems that analyzed the genomic sequences. When spliced out, the new N-terminal amino acid sequences align well with the other SAHS mature protein N-termini. We have further identified other candidate splice sites in these genes (<a class="usa-link" href="#MOESM2">Supplementary Discussion</a>).</p>
<p id="Par10">We recombinantly expressed the Uniprot-predicted mature proteins RvSAHS1–4 and RvSAHS6-12 as well as HeSAHS4 in the <em>E. Coli</em> cytoplasm. The target proteins were expressed with an N-terminal SUMO tag, whose cleavage with a highly specific protease leaves the protein “scarless” without any extraneous amino acids remaining (Fig. <a class="usa-link" href="#MOESM2">S1a</a>)<sup><a aria-describedby="CR19" class="usa-link" href="#CR19">19</a></sup>. Since the N-terminal signal sequences are expected to be cleaved during secretion, we expressed proteins without these sequences based on the cleavage sites predicted by TargetP (Table <a class="usa-link" href="#MOESM2">S3</a>). We successfully expressed and purified 4 proteins (RvSAHS1, 4, 6, and HeSAHS4) with high soluble expression and purity and conducted subsequent studies using them (Figs. <a class="usa-link" href="#MOESM2">S1</a>, <a class="usa-link" href="#MOESM2">S2</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">SAHS proteins stabilize liposomes from dehydration-induced damage</h3>
<p id="Par11">As a first step of assessing the protective potential of SAHS proteins, we tested if they can stabilize lipid membranes upon dehydration-induced damages, using 1-palmitoyl-2-oleoyl-glycero-3-phosphocholine (POPC) liposomes as a model membrane. Prior to drying, the diameter of the liposomes as measured by dynamic light scattering (DLS) was in the range of 50-100 nm (Fig. <a class="usa-link" href="#Fig2">2a</a>). Upon drying and rehydration at 1.4 mg/mL lipid concentration, less than 10% of the particles remained in this range, while larger particles appeared around 360 and 4000 nm (Fig. <a class="usa-link" href="#Fig2">2b</a>). This marked increase in particle diameter indicated that dehydration damaged liposome membranes and promoted their fusion and/or aggregation<sup><a aria-describedby="CR20" class="usa-link" href="#CR20">20</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Tardigrade SAHS proteins stabilize liposomes from desiccation-induced damage.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11127935_42003_2024_6336_Fig2_HTML.jpg" target="_blank"><img alt="Fig. 2" class="graphic zoom-in" height="915" id="d33e514" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9982/11127935/12c9dfb4eab4/42003_2024_6336_Fig2_HTML.jpg" width="800"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig2/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>POPC liposomes at 1.4 mg/mL were dried with and without the addition of SAHS proteins and BSA at varying concentrations of 0.1–10 mg/mL, and their size distributions were measured by DLS. <strong>a</strong> Size distribution of non-dried POPC liposomes. <strong>b</strong> Size distribution of POPC liposomes dried and rehydrated without additives. <strong>c</strong>–<strong>h</strong> Size distributions of the liposomes dried with <strong>c</strong> RvSAHS1, <strong>d</strong> RvSAHS4, <strong>e</strong> HeSAHS4, <strong>f</strong> RvSAHS6, <strong>g</strong> BSA, and <strong>h</strong> trehalose.</p></figcaption></figure><p id="Par12">Liposomes dried in the presence of SAHS proteins or other excipients at 10, 1, and 0.1 mg/mL concentrations were partially protected, with higher levels of excipient providing more protection. For all four SAHS proteins tested, particle sizes after a dehydration-rehydration cycle were notably smaller compared to those dried without an excipient (Figs. <a class="usa-link" href="#Fig2">2c–f</a> and <a class="usa-link" href="#MOESM2">S3</a>). While the diameters of the major peaks were slightly increased from ~60 nm to ~100 nm, none of the bigger particles having diameters larger than 200 nm were observed, which indicated that the addition of SAHS proteins prevented the fusion and aggregation of dried liposomes. Bovine serum albumin (BSA), a commonly used excipient for protein preservation, was less effective in protecting liposomes than SAHS proteins, as indicated by the signal in ~500 nm range (Fig. <a class="usa-link" href="#Fig2">2g</a>). Liposomes dried in 10 mg/mL trehalose showed no particles over 150 nm, suggesting that trehalose also prevented membrane fusion of dried liposomes (Fig. <a class="usa-link" href="#Fig2">2h</a>). These observations indicated that SAHS proteins have comparable effects to trehalose in terms of their ability to stabilize membranes. SAHS proteins at lower concentrations of 1 mg/mL were also able to protect liposomes, whereas at 0.1 mg/mL, the extent of protection was diminished, and larger aggregates ~1000 nm were formed (Fig. <a class="usa-link" href="#Fig2">2c–f</a>, dotted lines); also, the repeat experiments showed consistent results (Fig. <a class="usa-link" href="#MOESM2">S3</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">SAHS proteins enhance survival of desiccated bacterial cells</h3>
<p id="Par13">We then tested whether extracellularly added SAHS proteins can enhance the desiccation tolerance of bacterial cells (Figs. <a class="usa-link" href="#Fig3">3</a> and <a class="usa-link" href="#MOESM2">S4</a>). <em>E. Coli</em> cells were dried for 48 h at 10<sup>8</sup> cells/mL with 0.5 mg/mL of SAHS proteins. As measured by colony-forming units, <em>E. coli</em> showed enhanced survival compared to those dried without added excipients or with BSA, lactate dehydrogenase, or citrate synthase proteins (Fig. <a class="usa-link" href="#MOESM2">S4a</a>), indicating that this was not a general effect caused by adding high concentration of protein. RvSAHS1 provided the most reproducible protection of <em>E. coli</em>, with all replicates in each experiment showing &gt;10-fold enhancement of survival. At 0.1 mg/mL RvSAHS1, the protective effect decreased but was still statistically significant (Fig. <a class="usa-link" href="#MOESM2">S4b</a>). Trehalose, a well-known cryoprotectant that is expressed intracellularly in many organisms during desiccation, also protected <em>E. coli</em> against drying when added extracellularly, but not as strongly as RvSAHS1. Intracellular expression of SAHS proteins showed no clear-cut enhancement of <em>E. coli</em> survival upon desiccation (Fig. <a class="usa-link" href="#MOESM2">S4c</a>, <a class="usa-link" href="#MOESM2">d</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Tardigrade SAHS proteins enhance bacterial desiccation survival.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11127935_42003_2024_6336_Fig3_HTML.jpg" target="_blank"><img alt="Fig. 3" class="graphic zoom-in" height="508" id="d33e612" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9982/11127935/5b7475630e8d/42003_2024_6336_Fig3_HTML.jpg" width="725"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig3/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>
<em>E. Coli</em> cells were dried for 48 h with and without the addition of 0.5 mg/mL concentration of SAHS proteins, BSA and trehalose. Survival percentage was calculated by comparing the number of viable colonies after plating the rehydrated cells to the initial number of cells added. <strong>b</strong>
<em>R. tropici</em> cells were dried for 48 h with and without the addition of 0.5 mg/mL concentration of SAHS proteins, BSA, and trehalose. Survival percentage was calculated in the same way as the <em>E. coli</em> experiment. Individual data points represent independent replicates, and lines represent the mean survival. Student’s <em>t</em>-test was used to determine the statistical significance between the negative control (no excipient) and each group, which is indicated as asterisks. * <em>p</em> &lt; 0.05; ** <em>p</em> &lt; 0.01; *** <em>p</em> &lt; 0.001.</p></figcaption></figure><p id="Par14">We also tested if the SAHS proteins could enhance the desiccation survival of the plant-symbiotic bacterium <em>Rhizobium tropici</em>, which is a model organism for nitrogen-fixing biofertilizers. When <em>R. tropici</em> cells were dried at a concentration of 7.2 × 10<sup>9</sup> cells/mL with and without 0.5 mg/mL excipients, we observed ~40-fold enhancement of cell survival upon addition of SAHS proteins (Fig. <a class="usa-link" href="#Fig3">3b</a>). Cells dried with trehalose at 0.5 mg/mL concentration showed 16-fold increased survival, while 0.5 mg/mL BSA caused a 4.2-fold increased survival.</p></section><section id="Sec6"><h3 class="pmc_sec_title">SAHS proteins and BSA protect enzymes from dehydration-induced inactivation</h3>
<p id="Par15">Tardigrade proteins such as CAHS have been shown to protect desiccated enzyme activities<sup><a aria-describedby="CR21" class="usa-link" href="#CR21">21</a></sup>. We tested if the SAHS proteins can preserve an enzyme, specifically lactate dehydrogenase (LDH), from desiccation-induced inactivation in vitro. When dried, LDH (0.01 mg/mL) alone lost &gt;90% of its activity compared to the non-dried control. The four SAHS proteins helped preserve LDH activity as a function of increasing concentration from 0.001 to 5 mg/mL, but BSA’s protective effect was comparable at each concentration (Figure <a class="usa-link" href="#MOESM2">S5</a>). This observation contrasts with the specific effect of SAHS proteins vs. BSA on the protection of membranous structures such as liposomes and cells.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Structural changes of SAHS proteins under desolvated conditions</h3>
<p id="Par16">To investigate the behaviors of SAHS proteins upon water loss, we examined their secondary structures using circular dichroism (CD). CD spectra of all four SAHS proteins showed minima at ~215 nm, indicating that they adopt β-structures under aqueous conditions (Fig. <a class="usa-link" href="#Fig4">4a</a>). This result was consistent with previous literature reporting β-sheet structures in RvSAHS1 and SAHS4<sup><a aria-describedby="CR14" class="usa-link" href="#CR14">14</a>,<a aria-describedby="CR16" class="usa-link" href="#CR16">16</a>,<a aria-describedby="CR17" class="usa-link" href="#CR17">17</a></sup>. The CD spectra for SAHS1 and SAHS4 at 0% in aqueous solution correlate well with the solved structures of these proteins, which have a single short alpha helix and are dominated by beta strands.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Tardigrade SAHS proteins undergo structural changes upon dehydration-mimicking conditions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11127935_42003_2024_6336_Fig4_HTML.jpg" target="_blank"><img alt="Fig. 4" class="graphic zoom-in" height="508" id="d33e667" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9982/11127935/f472ccf54751/42003_2024_6336_Fig4_HTML.jpg" width="700"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig4/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> CD spectra of SAHS proteins upon the addition of increasing amounts of TFE from 0% to 75%. <strong>b</strong> Secondary structure compositions of SAHS proteins under different TFE levels, calculated from the CD spectra.</p></figcaption></figure><p id="Par17">To investigate if these proteins can change their structures upon environmental stress, we used increasing concentrations of trifluoroethanol (TFE) and glycerol to mimic water-deficit conditions. All four proteins remained mostly β-stranded up to 25% TFE added but displayed a notable increase in helical structures at 50% TFE. At 50–75% TFE, all proteins showed spectra with two minima around 208 and 222 nm, which are characteristic of α-helices. This result is consistent with the earlier report by Yamaguchi et al., in which RvSAHS1 adopted a more helical structure with the addition of TFE<sup><a aria-describedby="CR14" class="usa-link" href="#CR14">14</a></sup>. Calculating relative secondary structure content confirmed that the helical content in these proteins increased from 10% up to 40% with the addition of TFE (Fig. <a class="usa-link" href="#Fig4">4b</a>). In contrast, increasing concentrations of glycerol had no such effect, even though glycerol is a ‘crowding’ agent (Fig. <a class="usa-link" href="#MOESM2">S6</a>).</p>
<p id="Par18">A filamentous, mesh-like structure was observed from dried RvSAHS1 proteins upon transmission electron microscopy (TEM) imaging (Fig. <a class="usa-link" href="#MOESM2">S7</a>). RvSAHS1 proteins dried at a concentration of 1 mg/mL revealed an entangled network of fibrous structures, while those dried at a higher concentration of 10 mg/mL showed a denser network that covered most of the grid. Such observations suggested that RvSAHS1 may undergo structural changes upon loss of water to allow for higher-order structure formation.</p>
<p id="Par19">To further investigate the pathway for structural change of SAHS proteins, we conducted a molecular dynamics (MD) simulation of the RvSAHS1 protein. To account for the possible inter-protein interactions, we first generated a water bath with three copies of a RvSAHS1 monomer, and the simulation was conducted using explicit solvent, openMM, and the Amber force field. The simulation was performed for a microsecond at 550 K to induce the breaking of hydrogen bonds in a time frame amenable to simulation. We found that the root mean squared deviation (RMSD) of alpha carbons in the ensemble relative to the starting structures increased within the first 100 ns and then plateaued for the remainder of the 1 µs (Figs. <a class="usa-link" href="#Fig5">5a</a>, <a class="usa-link" href="#MOESM2">S8</a>). As the simulation progressed, RvSAHS1 proteins showed the structural unfolding of beta-sheet structures and formation of alpha-helices: the beta-sheet component decreased from ~50% to ~3%, and the alpha-helical component increased from ~7% to ~20% as the simulation progressed from 0 to 100 ns (Fig. <a class="usa-link" href="#Fig5">5b</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. MD simulation of SAHS protein structural change upon desolvation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11127935_42003_2024_6336_Fig5_HTML.jpg" target="_blank"><img alt="Fig. 5" class="graphic zoom-in" height="470" id="d33e715" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9982/11127935/b7ccb346a5b1/42003_2024_6336_Fig5_HTML.jpg" width="800"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig5/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> RMSD over time during the first 100 ns MD simulation of RvSAHS1 homotrimer. Results during the first 100 ns are magnified. Different color indicates each monomer chain in a trimer. Representative images of the RvSAHS1 protein structures reconstituted from the MD simulation at 0, 50, and 100 ns time points are provided at the top. <strong>b</strong> Changes in secondary structure components of RvSAHS1 during the MD simulation. Red: coil; blue: helix; orange: sheet. <strong>c</strong> Representative image showing how conserved beta-sheet motifs of RvSAHS1 become closely aligned upon structural shift into helices. Only the conserved C3 segments of RvSAHS1 of each monomer are highlighted to show the changes in their alignments as the simulation progresses. <strong>d</strong> Detailed image of C3 motif sheet-to-helix structural change. Highlighted in blue and red are two highly conserved regions found within the beta-sheet region that directly interact through ionic bonds. Structures of these regions at 0 and 180 ns are indicated.</p></figcaption></figure><p id="Par20">Figure <a class="usa-link" href="#Fig5">5c</a> shows the C-terminal segments of each monomer that are conserved among SAHS proteins, which contribute significantly to the initial beta-barrel structure (identified as the C3 motif in ref. <sup><a aria-describedby="CR14" class="usa-link" href="#CR14">14</a></sup>). As the simulation progressed, this region highlighted how, as helices form, the motifs between subunits may become more closely aligned during the 0–100 ns simulation period. Figure <a class="usa-link" href="#Fig5">5d</a> indicates how the beta-sheet region within the C3 motif, which is maintained through interactions between positively and negatively charged amino acid residues, gets separated to form a helix as the simulation proceeds. These sequences, highlighted in blue and red, were found to be highly conserved among the SAHS family (Fig. <a class="usa-link" href="#MOESM2">S9</a>) based on our evolutionary conservation analysis using ConSurf <sup><a aria-describedby="CR22" class="usa-link" href="#CR22">22</a></sup>, indicating that such structural shifts may be a shared trait among SAHS proteins.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Structural bioinformatic analysis of SAHS proteins and related binding proteins</h3>
<p id="Par21">To understand how SAHS proteins might undergo a dramatic transition from a beta-sheet conformation to a predominantly alpha-helical state and protect vesicles and cells during drying, we examined the solved structures of RvSAHS1 and RvSAHS4, compared them with solved structures of fatty acid binding proteins from higher organisms, and analyzed patterns of sequence conservation.</p>
<p id="Par22">The structures of RvSAHS1 and RvSAHS4 (PDB IDs 5xna and 5z4g), fatty acid binding protein from human muscle (1hmr)<sup><a aria-describedby="CR23" class="usa-link" href="#CR23">23</a></sup>, the intracellular lipid binding protein FABP1 (6do6)<sup><a aria-describedby="CR24" class="usa-link" href="#CR24">24</a></sup> were examined, along with alignments of related sequences.</p>
<p id="Par23">RvSAHS1 and RvSAHS4 are highly structured and similar to FABPs and related proteins<sup><a aria-describedby="CR16" class="usa-link" href="#CR16">16</a>,<a aria-describedby="CR17" class="usa-link" href="#CR17">17</a></sup>. Thus, the description of these proteins as “intrinsically disordered proteins” is not accurate, at least in the hydrated state. All of these proteins are unusual in that they have a large internal cavity that may be filled with ligands, solvent molecules, or both. The sides of the cavity are defined by beta sheets that, for the most part, do not pack against a typical hydrophobic core. In the RvSAHS1 and RvSAHS4 structures solved by Fukuda and colleagues, this volume is occupied by solvent molecules.</p>
<p id="Par24">In RvSAHS1, there is a small hydrophobic core, while the cavity is bordered by beta-hairpin loops from Phe50-His72, His83-Val96, Thr106-Ile113, Ala126-Asp139, and Lys150-Tyr163. (The Phe50-His72 loop also contains two short alpha helices.) In the absence of a ligand in the cavity, both faces of these loops are simply beta-sheets that face water on both sides. Thus, these structures would not be as stabilized as in a typical protein. We hypothesize that during a desiccation process, water would equilibrate out of the cavity, and as the environment becomes more dehydrated, other more stable proteins would pack against an SAHS protein, essentially crushing it and disrupting the beta-sheet structure that surrounds the cavity. In this way, the SAHS proteins could act as deformable packing material to prevent other biological structures from deforming or rupturing during desiccation.</p>
<p id="Par25">Among the 13 SAHS proteins of <em>R. varieornatus</em>, the amino acids pointing into the small hydrophobic center are generally conserved, while amino acids pointing into the cavity are not. The “hydrophobic core” is located on one side of the much larger interior and includes the amino acids Trp 32, Trp37, Met38, Trp41, Tyr64, Ile74, Trp75, Tyr81, Phe98, Leu100, Leu124, Tyr141, Val143, Leu148, and Ala159. Of these, Trp 32, Trp37, Met38, Trp41, Phe98, Leu100, Leu124, Tyr/Phe141, Val143, and Leu148 are rather well conserved, while the others are not. We note that the highly conserved tryptophans at positions 32, 37, and 41 are present in our newly proposed sequences for RvSAHS 9, 10, and 12, supporting the idea that these sequences were previously misannotated.</p>
<p id="Par26">The amino acids pointing inward towards the cavity include Phe50, Phe53, Ile54, Leu57, Leu59, His83, Ile85, Val87, Lys90, Tyr92, Asp94, Thr106, His108, Leu113, Tyr115, Tyr117, Ala126, Val128, Val130, Ile137, Asp139, Lys150, Tyr152, Arg151, and Tyr165. Most of these are within 4 Angstroms of ethylene glycol or the putative fatty acid in the RvSAHS1 structure<sup><a aria-describedby="CR16" class="usa-link" href="#CR16">16</a></sup>. Of these, Phe53, Val/Ile87, Tyr117, Arg/Lys161 and Tyr163 are largely conserved, and the hydrophobic residues Phe50, Ile54, Ile85, Ile118, Val123, Val125, and Ile137 are generally interchangeable with other hydrophobic residues. Arg/Lys161 is notable because it is conserved in fatty acid binding proteins and interacts with the carboxyl group in the fatty acid in the same way that this amino acid interacts with a carboxyl group on a putative fatty acid in the SAHS1 structure.</p>
<p id="Par27">Another feature of the SAHS proteins and fatty acid-binding proteins is the presence of an entry/exit point. In RvSAHS1, four turns define this area—three beta-hairpins with tips at Lys90, Asn109, and Ser132, and a one-turn alpha helix with Gln63 pointing towards the hole. The amino acid sequences around this region are extremely variable among the <em>R. varieornatus</em> SAHS proteins, with Gln63 and Ser132 near the edge of a gap in the alignment. Moreover, since the SAHS proteins are secreted eukaryotic proteins, they can undergo N-linked glycosylation at Asn-X-Ser/Thr sites (which are not captured in proteins made in <em>E. coli</em>). The distribution of putative N-linked glycosylation sites in RvSAHS1-13 is strongly biased toward the amino acids surrounding the entry/exit site: Asn109 is likely N-glycosylated in RvSAHS1, 2, 8, 10, and 11 (and Asn107 in RvSAHS5); in RvSAHS3, 8 and 9 sites near Gln63 are likely glycosylated; and in RvSAHS5, 9 and 11 sites near Ser132 are glycosylated. RvSAHS4, 6, 12, and 13 do not have N-linked glycosylation sites. Of the 14 putative N-linked glycosylation sites in RvSAHS1-13, twelve are near the opening into the cavity. The presence/absence and positioning of these sites may have a significant effect on the movement of water and other small molecules in and out of the cavity.</p>
<p id="Par28">Fukuda et al. discussed the possibility that RvSAHS1 may bind a ligand with a carboxyl group (e.g. acetate) but noted that not all of the SAHS proteins have these carboxyl-binding residues<sup><a aria-describedby="CR17" class="usa-link" href="#CR17">17</a></sup>. Our analysis of alignments of the <em>R. varieornatus</em> SAHS proteins indicates a great deal of variation in the cavity-forming residues, implying that these proteins do not bind a common ligand (such as a fatty acid) if they bind one at all.</p>
<p id="Par29">Based on this analysis and data presented here, we propose a model by which the thirteen SAHS proteins encoded by <em>R. varieornatus</em> undergo a transition upon dehydration in which water molecules equilibrate out of the cavity and crowding and pressure of other proteins, then disrupt the structure of the SAHS proteins, which then might reconfigure into a completely different conformation. The thirteen different SAHS proteins would have different characteristics for stability and water retention, such that they would collapse under different conditions of dehydration and pressure, allowing for a gradual process rather than a sudden phase transition. This idea is explored further in the Discussion.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par30">In this study, we tested the proteins belonging to the SAHS family of tardigrade proteins for their potential to protect diverse biological structures. In addition, we observed that the SAHS proteins undergo structural shift upon exposure to a desiccation-mimicking compound and used a diverse set of experimental and computational approaches to suggest possible models of how the structural transition of SAHS proteins might occur to protect biological structures. Specifically, we found that the SAHS proteins protect membrane-containing structures, specifically liposomes and living bacteria, against desiccation-induced disruption but are no better than BSA at protecting the enzyme lactate dehydrogenase against desiccation. Based on CD analysis, the SAHS proteins appear to undergo a shift from a primarily beta-sheet structure to an alpha-helical structure in increasing TFE concentrations (thought to be desiccation-mimicking conditions), and a 1 μs molecular dynamic simulation showed that short alpha-helices can transiently form upon denaturation of proteins. Lastly, we performed a comparative structural analysis based on the available putative sequences of SAHS proteins. In doing so, we found that sequences in the Uniprot database corresponding to short (N-terminally truncated) SAHS proteins may have been incorrectly annotated and that all SAHS proteins are likely secreted and share a common structure. As noted previously, SAHS proteins are distantly related to fatty acid-binding proteins and consist of a large cavity and a small hydrophobic core off to one side. Our structural analysis indicated that amino acid side chains pointing into the cavity are variable, and putative N-linked glycosylation sites are also variable in position but tend to cluster around the opening from which solvent molecules might enter and exit.</p>
<p id="Par31">SAHS proteins from the tardigrades <em>R. varieornatus</em> and <em>H. exemplaris</em> are thought to contribute to the desiccation resistance of these organisms. <em>R. varieornatus</em> and <em>H. exemplaris</em> encode multiple different SAHS proteins, respectively, and at least some of them are constitutively expressed at high levels<sup><a aria-describedby="CR13" class="usa-link" href="#CR13">13</a></sup>. It is not clear why so many members of the SAHS family are encoded in these tardigrades or whether their diversity is important in desiccation protection. We tested several different SAHS proteins for their ability to be expressed in the <em>E. coli</em> cytoplasm without signal sequences. From <em>R. varieornatus</em>, SAHS1, 2, 4, 6, 7, and 8 were expressed, while the Uniprot-derived sequences of SAHS3, 9-12 were not. SAHS1, SAHS4, and SAHS6 from <em>R. varieornatus</em> and SAHS4 from <em>H. exemplaris</em> were expressed at the highest levels (Fig. <a class="usa-link" href="#MOESM2">S2</a>) and were characterized further.</p>
<p id="Par32">The four tested SAHS proteins reduced the extent of liposome fusion and bacterial loss of viability during desiccation. When a commercial POPC-based liposome preparation was dried in the absence of excipients and then rehydrated, the average diameter of the liposomes increased from about 70 nm to about 350 nm, indicating that many liposomes fuse during this process to make much larger aggregates. In the presence of 10 mg/mL of any of the SAHS proteins, the extent of liposome fusion was much reduced, and even at lower concentrations, there was a protective effect. The excipient disaccharide trehalose showed a comparable protective effect, while BSA was less protective (Figs. <a class="usa-link" href="#Fig2">2</a>, <a class="usa-link" href="#MOESM2">S3</a>). When either <em>E. coli</em> or <em>R. tropici</em> was desiccated and rehydrated, the SAHS proteins had a strong survival-promoting effect, generally improving viability by about 50-fold and performing at least 10-fold better than BSA and several other proteins and about 3-fold better than trehalose (Figs. <a class="usa-link" href="#Fig3">3</a>, <a class="usa-link" href="#MOESM2">S4</a>). In addition, heterologous intracellular expression of SAHS proteins did not significantly improve <em>E. coli</em> desiccation survival (Fig. <a class="usa-link" href="#MOESM2">S4</a>) <em>R. tropici</em> is a nitrogen-fixing plant symbiont that represents a working model for the preservation of nitrogen-fixing bacteria such as rhizobia and could itself be deployed as biofertilizer.</p>
<p id="Par33">In contrast, the SAHS proteins did not specifically protect the enzyme lactate dehydrogenase against loss of activity upon desiccation and rehydration—the SAHS proteins had a slight effect, comparable to the protective effect of BSA (Fig. <a class="usa-link" href="#MOESM2">S5</a>). The ability to stabilize proteins/enzymes and prevent their aggregation has been associated with intracellular stress protection<sup><a aria-describedby="CR25" class="usa-link" href="#CR25">25</a>,<a aria-describedby="CR26" class="usa-link" href="#CR26">26</a></sup>. For instance, heterologous expressions of tardigrade small heat shock proteins (sHSPs) improved bacterial desiccation survival, and their protective mechanism has been attributed to their ability to reduce protein aggregation during desiccation<sup><a aria-describedby="CR27" class="usa-link" href="#CR27">27</a></sup>. Similarly, CAHS proteins, whose heterologous expressions in bacteria and yeast enhanced their desiccation survival, were superior to BSA in protecting enzymes from desiccation-induced inactivation in vitro<sup><a aria-describedby="CR13" class="usa-link" href="#CR13">13</a>,<a aria-describedby="CR21" class="usa-link" href="#CR21">21</a></sup>. The SAHS proteins tested in this study are presumed to function extracellularly, and it might be that their main role is not to protect proteins but to prevent damage to membranes.</p>
<p id="Par34">The solved structures of two SAHS proteins and the pattern of amino acid conservation in the SAHS proteins (Fig. <a class="usa-link" href="#Fig6">6</a>) indicate that the SAHS proteins all have a large cavity that may simply be full of water. (An alternative hypothesis, suggested by our AlphaFold-predicted structure of multiple SAHS proteins, would be that the N-terminus of one SAHS protein fits into the cavity of another (Fig. <a class="usa-link" href="#MOESM2">S10</a>); no independent evidence supports this hypothesis.) The SAHS proteins are similar in sequence to fatty acid binding proteins (FABPs), and the SAHS and FABP structures are superimposable. However, the amino acid side chains that point inward in the SAHS protein cavity are not strongly conserved, suggesting that these proteins are not binding to a particular ligand but may simply be filled with water or with variable solutes. Moreover, in FABPs the cavity is significantly larger than can be occupied by a fatty acid, and since fatty acids are moving in and out of the cavity during intracellular transport, they are presumably exchanged with water in this process.</p>
<figure class="fig xbox font-sm" id="Fig6"><h3 class="obj_head">Fig. 6. Glycosylation sites in the SAHS proteins of <em>R. varieornatus</em> cluster near the cavity opening.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img alt="Fig. 6" class="graphic" height="255" id="d33e913" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9982/11127935/a823b2d653bc/42003_2024_6336_Fig6_HTML.jpg" width="633"/></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/Fig6/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Structure of SAHS1 from Fukuda et al.<sup><a aria-describedby="CR16" class="usa-link" href="#CR16">16</a></sup>, showing an alpha-carbon trace of all amino acids, with the beta-hairpins surrounding the cavity colored in yellow, yellow-orange, orange and red. Amino acids in cyan with side chains shown define the hydrophobic core. Large spheres show the alpha carbons of asparagines in an Asn-X-Ser/Thr sequence in at least one SAHS protein. The putative opening of the cavity is on the left. <strong>b</strong> Structure of SAHS1, rotated 90 °C relative to panel A with the left side coming forward. All the side chains are shown. This view illustrates the opening into the cavity of the protein, such that the cyan-colored hydrophobic core side chains at the back of the protein can be seen through this hole.</p></figcaption></figure><p id="Par35">SAHS proteins, by analogy to CAHS proteins<sup><a aria-describedby="CR14" class="usa-link" href="#CR14">14</a></sup>, may undergo a shift to an alternative conformation in desiccation conditions. A high-temperature MD simulation of three SAHS proteins in a water box revealed that, as the beta-barrel structure of the protein became disordered, short alpha-helical regions would form and disappear (Fig. <a class="usa-link" href="#Fig5">5</a>, <a class="usa-link" href="#MOESM2">S8</a>). These structures might nucleate the formation of an alternative, protective conformation. However, the use of molecular dynamics to simulate desiccation is challenging because, during actual desiccation, the concentration of other proteins would increase and may play a dominating role in a conformational change of SAHS proteins, for example, through Brownian collisions. Moreover, desiccation occurs on long timescales that cannot be simulated with current molecular dynamics technology.</p>
<p id="Par36">One model for the protective mechanism of SAHS proteins is that the SAHS protein cavity is filled with water and possibly other solutes in the hydrated state, but during dehydration, the water is extruded. Thus, the SAHS protein would collapse as a result of the Brownian motion of macromolecules and pressure on the outside of the protein without a corresponding pressure from the inside. This would lead to the denaturation of the protein and transition to an alternative, protective conformation. The different SAHS proteins may have different kinetics and equilibria for this transition, such that when a tardigrade is dehydrated, there is a gradual conversion of the different SAHS proteins to a protective state. The presence or absence and positioning of N-linked oligosaccharides in the different SAHS proteins near the entry/exit point may give them distinct dehydration-transition characteristics. After the conversion from a globular conformation, the SAHS proteins may form higher-order, gel-like structures that stabilize membrane-bound elements of cells, which may constitute rampart-like “special extracellular structure (SES)” observed outside of the secretory cell membranes of dried tardigrade<sup><a aria-describedby="CR28" class="usa-link" href="#CR28">28</a></sup>.</p>
<p id="Par37">SAHS proteins may have a practical application in protecting non-tardigrade cells, such as agricultural microbes, against desiccation. We found that two bacteria, <em>E. coli</em> and <em>R. tropici</em>, could be stabilized against desiccation by the presence of a single SAHS protein. <em>R. tropici</em> is a “plant-growth promoting rhizobacterium,” and PGPR use is often limited by their low desiccation tolerance<sup><a aria-describedby="CR29" class="usa-link" href="#CR29">29</a>,<a aria-describedby="CR30" class="usa-link" href="#CR30">30</a></sup>, highlighting the potential applicability of SAHS proteins to improve the durability of microbial biofertilizers. Also, desiccation would enable easier and lower-cost transportation and safer handling of these microbes in the field. In summary, this work provides the evaluation of tardigrade SAHS proteins in terms of their protective abilities, which supports their potential applications as stabilizing molecules.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Sequence analysis</h3>
<p id="Par38">SAHS protein sequences used for expression in <em>E. coli</em> (Figure <a class="usa-link" href="#MOESM2">S2</a>) were obtained from Uniprot database (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.uniprot.org" rel="noopener noreferrer" target="_blank">https://www.uniprot.org</a>). Protein subcellular localization and signal peptides were predicted using TargetP 2.0 server available at <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://services.healthtech.dtu.dk" rel="noopener noreferrer" target="_blank">https://services.healthtech.dtu.dk</a><sup><a aria-describedby="CR18" class="usa-link" href="#CR18">18</a></sup>. Genomic sequences encoding <em>R. varieornatus</em> SAHS proteins were from the <em>Ramazzottius varieornatus</em> contig Scaffold001, strain YOKOZUNA-1, whole genome shotgun sequence, GenBank accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/BDGG01000001" rel="noopener noreferrer" target="_blank">BDGG01000001</a>, which encodes SAHS1-12, and <em>Ramazzottius varieornatus</em> contig Scaffold002, GenBank accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/BDGG01000002" rel="noopener noreferrer" target="_blank">BDGG01000002</a>, which encodes SAHS 13.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Protein expression and purification</h3>
<p id="Par39">Expression vectors encoding His<sub>6</sub>/SUMO-tagged SAHS proteins were obtained from Twist Bioscience and transformed into LEMO21(DE3) competent cells (New England Biolabs) according to the provider’s protocol. Transformed cells were grown in TBM-5052 autoinduction media containing 100 µg/mL Kanamycin, 25 µg/mL Chloramphenicol, and 2 mM <span class="font-variant-small-caps">l</span>-rhamnose at 37 °C overnight. The cells were harvested by centrifugation at 4000×<em>g</em> for 10 min, resuspended in lysis buffer (20 mM NaH<sub>2</sub>PO<sub>4</sub>, 500 mM NaCl, 20 mM Imidazole, pH 7.5) supplemented with 0.25 mg/mL lysozyme (Millipore Sigma), turbonuclease (Accelagen) and 1% n-Nonyl-Beta-<span class="font-variant-small-caps">d</span>-Glucopyranoside (Cube Biotech) and shaken vigorously for 1 h. The total lysate was then centrifuged at 4000×<em>g</em> for 30 min, and the supernatant was obtained. This clarified protein lysate was then shaken with Ni-charged IMAC Magbeads (Genscript) for 1 h to bind tagged proteins. Beads were then washed 3 times with wash buffer (20 mM NaH<sub>2</sub>PO<sub>4</sub>, 500 mM NaCl, 20 mM Imidazole, pH 7.5), washed once more with PBS buffer, and incubated with PBS buffer supplemented with Cth SUMO protease<sup><a aria-describedby="CR19" class="usa-link" href="#CR19">19</a></sup> at 0.02 mg/mL shaking overnight to achieve the cleavage of SAHS proteins from N-terminal SUMO tag. Cleaved proteins were recovered from the supernatant, filtered through 0.2 µm filters, concentrated using Amicon ultra 0.5 centrifugal columns (Millipore, 10 kDa MWCO), and stored at −20 °C. Purified proteins were inspected using SDS–PAGE and SimplyBlue staining (Invitrogen).</p></section><section id="Sec13"><h3 class="pmc_sec_title">Circular dichroism</h3>
<p id="Par40">Proteins were buffer exchanged by overnight dialysis with 20 mM NaH<sub>2</sub>PO<sub>4</sub>, pH 7.5, and diluted to a final concentration of 0.25 mg/mL. Protein secondary structure was examined using far-UV circular dichroism (CD) using a Jasco J-815 CD spectropolarimeter equipped with a Peltier temperature controller and single cuvette holder. The CD spectra were obtained by averaging three wavelength scans from 200 to 260 nm in 0.5 nm steps in a cuvette with a path length of 1 mm. To examine protein secondary structures in water loss-mimicking conditions and molecular crowding conditions, trifluoroethanol (TFE) or glycerol was added to the sample at final concentrations ranging from 25% to 75% and incubated for two hours before determining the CD spectra.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Lactate dehydrogenase inactivation assay</h3>
<p id="Par41"><span class="font-variant-small-caps">l</span>-lactate dehydrogenase (LDH) enzyme (Roche) was diluted to 0.01 mg/mL concentration in 100 µL of 25 mM Tris/HCl buffer containing various concentrations of SAHS proteins or BSA (Sigma-Aldrich). BSA were dialyzed against the same buffer prior to use. 50 µL of each sample was stored at 4 °C while the other 50 µL was dried using a Savant SPD131DDA SpeedVac (Thermo Scientific) for 2 h without heating. Dried samples were rehydrated, and activity was measured at 50× dilution in a buffer containing 100 mM sodium phosphate (pH 6), 100 µM NADH, and 2 mM pyruvate. Absorbance at 340 nm was measured using a Synergy H1 plate reader (BioTek). Relative activity after dehydration was calculated by comparing the initial linear reaction rate of the dried sample to that of the non-dried control sample.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Liposome drying assay</h3>
<p id="Par42">POPC liposomes (100 nm, Tribioscience) dialyzed against PBS buffer were diluted to 1.4 mg/mL in 50 µL PBS buffer containing various concentrations of proteins and excipients. Proteins were dialyzed against the PBS buffer prior to use. Samples were then dried in a 250 µL microcentrifuge tube with an opened lid placed inside a 750-mL sealed chamber filled with about 42 grams of Drierite desiccant for 48 h at room temperature. Samples were rehydrated and diluted 10-fold, and the liposome sizes were measured using a Wyatt DynaPro Plate Reader III dynamic light scattering (DLS) device.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Bacterial cell drying assay</h3>
<p id="Par43"><em>E. coli</em> vectors expressing SAHS proteins without SUMO tags and with an N-terminal methionine were custom-ordered and obtained from Twist Bioscience and transformed into LEMO21(DE3) competent cells (NEB). To test for the effects of intracellular SAHS expression, cells transformed with each SAHS protein as well as mEGFP expression vector were grown in LB medium containing 100 µg/mL Kanamycin and 25 µg/mL Chloramphenicol at 37 °C until OD600 of 0.6, added with 0.1 mM IPTG and 2 mM <span class="font-variant-small-caps">l</span>-rhamnose and further grown at room temperature overnight. Cells were harvested and resuspended in PBS buffer and diluted to contain 5 × 10<sup>6</sup> cells in 50 µL PBS buffer. Cells were then dried in a microcentrifuge tube with an opened lid placed inside a sealed chamber filled with Drierite desiccant for 48 h at room temperature. Subsequently, cells were rehydrated and immediately plated onto LB-agar Kan/Cm selection plates with a serial dilution to count the colony-forming units the next day. To test for protection by extracellular SAHS proteins, LEMO21(DE3) cells transformed with mEGFP expression vector were grown in LB medium containing 100 µg/mL Kanamycin, 25 µg/mL Chloramphenicol at 37 °C overnight, without any rhamnose and inducer added. Subsequently, cells were harvested and resuspended in PBS buffer and diluted to contain 5 × 10<sup>6</sup> cells in 50 µL PBS buffer supplemented with varying concentrations of SAHS proteins or other control excipients, dried, and rehydrated following the above protocol. BSA and excipients were dialyzed against the PBS buffer prior to use.</p>
<p id="Par44">For <em>R. tropici</em> drying assay, <em>R. tropici</em> CIAT 899 cells were grown in PY media overnight in round-bottom culture tubes. The culture tubes were placed in an incubator set to 28 °C and shaken at 200 rpm, then transferred to larger volume flasks the following day. Following incubation, the bacterial culture was centrifuged at 4000×<em>g</em> for 15 min. The resulting pellets were then resuspended in phosphate-buffered saline (PBS) and diluted to reach an OD600 of 0.4. The cultures were centrifuged again, and the supernatant was replaced with PBS buffer containing excipients. The solutions were pipetted into 250 µL tubes in 100 µL aliquots to contain a total of 7.2 × 10<sup>8</sup> cells and placed inside a sealed container with Drierite desiccant (10–20 mesh, Thermo Scientific) to allow the samples to dry for 48 h. Once the samples were dried, three samples of each condition were taken out at relevant time points and resuspended in PBS. Subsequently, the samples were diluted and spread onto PY agar plates for colony counting.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Transmission electron microscopy (TEM)</h3>
<p id="Par45">SAHS proteins in PBS buffer were imaged by transmission electron microscopy using a Tecnai G2 Spirit BioTWIN TEM equipped with an AMT 2k CCD camera. TEM samples were prepared by depositing proteins onto carbon/formvar coated copper grids, staining with 2% uranyl acetate solution.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Computational modeling and conservation analysis of RvSAHS1</h3>
<p id="Par46">An initial structure for RvSAHS1 was generated using alphaFold using the sequence acquired from UniProt with the accession code <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/protein/J7MFT5" rel="noopener noreferrer" target="_blank">J7MFT5</a>. Ambertools generated amber input files with the ff19SB forcefield and explicit TIP3 solvent conditions. OpenMM was used to simulate 1000 ns at a temperature of 550 K after an equilibration at 310 K for 1 ns. Trajectories were generated with conformations captured every 10 ps. RMSD analysis of the simulation trajectory was performed using the Python package mdtraj. Visualizations of protein structures were produced using Houdini (SideFX software) and the Python packages mdtraj, biopython, and prody. Additional details of computational methods are in Supplementary Information, and structural details are included in Supplementary Data <a class="usa-link" href="#MOESM5">2</a>.</p>
<p id="Par47">Evolutionary conservation analysis was performed on the RvSAHS1 sequence (<a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/protein/J7MFT5" rel="noopener noreferrer" target="_blank">J7MFT5</a>) using ConSurf and the default parameters, where 60 homologs were collected from the UNIREF90 database using HMMER. Of these, 24 homologs passed the default thresholds (min/max similarity, coverage, etc.), and 25 were CD-HIT unique. The conservation calculations were conducted on the 25 unique hits, including the initial query.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p id="Par48">For the desiccation survival experiments, the Student’s <em>t</em>-test was used to determine the statistical significance between the negative control (no excipient) and cells dried with SAHS proteins or excipients. The number of independent replicates is noted in the figure legend and separately attached source data. For the enzyme activity assay, the number of replicates is noted in the figure legend and source data.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par49">Further information on research design is available in the <a class="usa-link" href="#MOESM6">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec21"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11127935/bin/42003_2024_6336_MOESM1_ESM.pdf">Peer Review File</a><sup> (961KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11127935/bin/42003_2024_6336_MOESM2_ESM.pdf">Supplementary Information</a><sup> (2.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11127935/bin/42003_2024_6336_MOESM3_ESM.docx">Description of Supplementary Materials</a><sup> (14.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11127935/bin/42003_2024_6336_MOESM4_ESM.xlsx">Supplementary Data 1</a><sup> (1.2MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11127935/bin/42003_2024_6336_MOESM5_ESM.zip">Supplementary Data 2</a><sup> (765.1KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11127935/bin/42003_2024_6336_MOESM6_ESM.pdf">Reporting Summary</a><sup> (473KB, pdf) </sup>
</div></div></section></section><section class="ack" id="ack1"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by the Wyss Institute for Biologically Inspired Engineering at Harvard University as the validation project. This work was further supported by the DARPA under Cooperative Agreement Number W911NF-19-2-0017. The authors acknowledge the Harvard Medical School Electron Microscopy Facility for the use of TEM and the Center for Macromolecular Interactions at Harvard Medical School for the use of CD and DLS. B.M. acknowledges NSF CMMI-1752172 and ONR N000141912317. Z.B. acknowledges support from the NSF Graduate Research Fellowship Program.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>S.L. designed the research, performed the experiments, analyzed the data, and wrote the paper. C.B.R. conducted the molecular dynamics simulation and corresponding analysis. Z.B. and B.M. performed and analyzed the <em>R. tropici</em> experiments. J.C.W. performed structural analysis, analyzed the data and wrote the paper. P.A.S. analyzed the data, contributed to experiment design and edited the paper.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par50"><em>Communications Biology</em> thanks Yohta Fukuda and Luis Pacios for their contribution to the peer review of this work. Primary Handling Editors: Lei Zheng and Johannes Stortz. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data are either contained within the manuscript/supporting information or available from the authors upon request. Source data for figures are included in Supplementary Data <a class="usa-link" href="#MOESM4">1</a>. Supplementary Data <a class="usa-link" href="#MOESM5">2</a> includes the initial structural pdb file of the MD simulation and those at time points of 50, 100, 150, 200, 250, and 300 ns. Bacterial expression plasmids for the production of recombinant SAHS proteins can be acquired from Addgene (Addgene plasmid ID 215645–215648).</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par51">The authors declare no competing interests.</p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec23"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s42003-024-06336-w.</p></section><section class="ref-list" id="Bib1"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Moberg N, et al.  Survival in extreme environments—on the current knowledge of adaptations in tardigrades. Acta Physiol. 2011;202:409–420. doi: 10.1111/j.1748-1716.2011.02252.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1748-1716.2011.02252.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21251237/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Acta%20Physiol.&amp;title=Survival%20in%20extreme%20environments%E2%80%94on%20the%20current%20knowledge%20of%20adaptations%20in%20tardigrades&amp;author=N%20Moberg&amp;volume=202&amp;publication_year=2011&amp;pages=409-420&amp;pmid=21251237&amp;doi=10.1111/j.1748-1716.2011.02252.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Boothby TC, Pielak GJ. Intrinsically disordered proteins and desiccation tolerance: elucidating functional and mechanistic underpinnings of anhydrobiosis. BioEssays. 2017;39:1700119. doi: 10.1002/bies.201700119.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/bies.201700119" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28901557/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=BioEssays&amp;title=Intrinsically%20disordered%20proteins%20and%20desiccation%20tolerance:%20elucidating%20functional%20and%20mechanistic%20underpinnings%20of%20anhydrobiosis&amp;author=TC%20Boothby&amp;author=GJ%20Pielak&amp;volume=39&amp;publication_year=2017&amp;pages=1700119&amp;pmid=28901557&amp;doi=10.1002/bies.201700119&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Belott C, Janis B, Menze MA. Liquid–liquid phase separation promotes animal desiccation tolerance. Proc. Natl Acad. Sci. USA. 2020;117:27676–27684. doi: 10.1073/pnas.2014463117.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.2014463117" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7959570/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33077592/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=Liquid%E2%80%93liquid%20phase%20separation%20promotes%20animal%20desiccation%20tolerance&amp;author=C%20Belott&amp;author=B%20Janis&amp;author=MA%20Menze&amp;volume=117&amp;publication_year=2020&amp;pages=27676-27684&amp;pmid=33077592&amp;doi=10.1073/pnas.2014463117&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Tanaka A, et al.  Stress-dependent cell stiffening by tardigrade tolerance proteins that reversibly form a filamentous network and gel. PLoS Biol. 2022;20:e3001780. doi: 10.1371/journal.pbio.3001780.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pbio.3001780" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9592077/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36067153/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&amp;title=Stress-dependent%20cell%20stiffening%20by%20tardigrade%20tolerance%20proteins%20that%20reversibly%20form%20a%20filamentous%20network%20and%20gel&amp;author=A%20Tanaka&amp;volume=20&amp;publication_year=2022&amp;pages=e3001780&amp;pmid=36067153&amp;doi=10.1371/journal.pbio.3001780&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Tapia H, Young L, Fox D, Bertozzi CR, Koshland D. Increasing intracellular trehalose is sufficient to confer desiccation tolerance to Saccharomyces cerevisiae. Proc. Natl Acad. Sci. USA. 2015;112:6122–6127. doi: 10.1073/pnas.1506415112.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1506415112" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4434740/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25918381/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=Increasing%20intracellular%20trehalose%20is%20sufficient%20to%20confer%20desiccation%20tolerance%20to%20Saccharomyces%20cerevisiae&amp;author=H%20Tapia&amp;author=L%20Young&amp;author=D%20Fox&amp;author=CR%20Bertozzi&amp;author=D%20Koshland&amp;volume=112&amp;publication_year=2015&amp;pages=6122-6127&amp;pmid=25918381&amp;doi=10.1073/pnas.1506415112&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Li S, et al.  Late embryogenesis abundant proteins protect human hepatoma cells during acute desiccation. Proc. Natl Acad. Sci. USA. 2012;109:20859–20864. doi: 10.1073/pnas.1214893109.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1214893109" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3529014/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23185012/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=Late%20embryogenesis%20abundant%20proteins%20protect%20human%20hepatoma%20cells%20during%20acute%20desiccation&amp;author=S%20Li&amp;volume=109&amp;publication_year=2012&amp;pages=20859-20864&amp;pmid=23185012&amp;doi=10.1073/pnas.1214893109&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Hand SC, Menze MA, Toner M, Boswell L, Moore D. LEA proteins during water stress: not just for plants anymore. Annu. Rev. Physiol. 2011;73:115–134. doi: 10.1146/annurev-physiol-012110-142203.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev-physiol-012110-142203" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21034219/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Physiol.&amp;title=LEA%20proteins%20during%20water%20stress:%20not%20just%20for%20plants%20anymore&amp;author=SC%20Hand&amp;author=MA%20Menze&amp;author=M%20Toner&amp;author=L%20Boswell&amp;author=D%20Moore&amp;volume=73&amp;publication_year=2011&amp;pages=115-134&amp;pmid=21034219&amp;doi=10.1146/annurev-physiol-012110-142203&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Veling MT, et al.  Natural and designed proteins inspired by extremotolerant organisms can form condensates and attenuate apoptosis in human cells. ACS Synth. Biol. 2022;11:1292–1302. doi: 10.1021/acssynbio.1c00572.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1021/acssynbio.1c00572" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9651988/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/35176859/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ACS%20Synth.%20Biol.&amp;title=Natural%20and%20designed%20proteins%20inspired%20by%20extremotolerant%20organisms%20can%20form%20condensates%20and%20attenuate%20apoptosis%20in%20human%20cells&amp;author=MT%20Veling&amp;volume=11&amp;publication_year=2022&amp;pages=1292-1302&amp;pmid=35176859&amp;doi=10.1021/acssynbio.1c00572&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Persson D, et al.  Extreme stress tolerance in tardigrades: surviving space conditions in low earth orbit. J. Zool. Syst. Evol. Res. 2011;49:90–97. doi: 10.1111/j.1439-0469.2010.00605.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1439-0469.2010.00605.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Zool.%20Syst.%20Evol.%20Res.&amp;title=Extreme%20stress%20tolerance%20in%20tardigrades:%20surviving%20space%20conditions%20in%20low%20earth%20orbit&amp;author=D%20Persson&amp;volume=49&amp;publication_year=2011&amp;pages=90-97&amp;doi=10.1111/j.1439-0469.2010.00605.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Hengherr S, Heyer AG, Köhler HR, Schill RO. Trehalose and anhydrobiosis in tardigrades—evidence for divergence in responses to dehydration. FEBS J. 2008;275:281–288. doi: 10.1111/j.1742-4658.2007.06198.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1742-4658.2007.06198.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/18070104/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20J.&amp;title=Trehalose%20and%20anhydrobiosis%20in%20tardigrades%E2%80%94evidence%20for%20divergence%20in%20responses%20to%20dehydration&amp;author=S%20Hengherr&amp;author=AG%20Heyer&amp;author=HR%20K%C3%B6hler&amp;author=RO%20Schill&amp;volume=275&amp;publication_year=2008&amp;pages=281-288&amp;pmid=18070104&amp;doi=10.1111/j.1742-4658.2007.06198.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Ingemar Jonsson K, Persson O. Trehalose in three species of desiccation tolerant tardigrades. Open Zool. J. 2010;3:1–5. doi: 10.2174/1874336601003010001.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2174/1874336601003010001" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Open%20Zool.%20J.&amp;title=Trehalose%20in%20three%20species%20of%20desiccation%20tolerant%20tardigrades&amp;author=K%20Ingemar%20Jonsson&amp;author=O%20Persson&amp;volume=3&amp;publication_year=2010&amp;pages=1-5&amp;doi=10.2174/1874336601003010001&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Hesgrove C, Boothby TC. The biology of tardigrade disordered proteins in extreme stress tolerance. Cell Commun. Signal. 2020;18:178. doi: 10.1186/s12964-020-00670-2.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s12964-020-00670-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7640644/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/33148259/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell%20Commun.%20Signal.&amp;title=The%20biology%20of%20tardigrade%20disordered%20proteins%20in%20extreme%20stress%20tolerance&amp;author=C%20Hesgrove&amp;author=TC%20Boothby&amp;volume=18&amp;publication_year=2020&amp;pages=178&amp;pmid=33148259&amp;doi=10.1186/s12964-020-00670-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Boothby TC, et al.  Tardigrades use intrinsically disordered proteins to survive desiccation. Mol. Cell. 2017;65:975–984. doi: 10.1016/j.molcel.2017.02.018.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.molcel.2017.02.018" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5987194/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28306513/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Tardigrades%20use%20intrinsically%20disordered%20proteins%20to%20survive%20desiccation&amp;author=TC%20Boothby&amp;volume=65&amp;publication_year=2017&amp;pages=975-984&amp;pmid=28306513&amp;doi=10.1016/j.molcel.2017.02.018&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Yamaguchi A, et al.  Two novel heat-soluble protein families abundantly expressed in an anhydrobiotic tardigrade. PLoS ONE. 2012;7:e44209. doi: 10.1371/journal.pone.0044209.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0044209" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3429414/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22937162/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Two%20novel%20heat-soluble%20protein%20families%20abundantly%20expressed%20in%20an%20anhydrobiotic%20tardigrade&amp;author=A%20Yamaguchi&amp;volume=7&amp;publication_year=2012&amp;pages=e44209&amp;pmid=22937162&amp;doi=10.1371/journal.pone.0044209&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Tanaka S, et al.  Novel mitochondria-targeted heat-soluble proteins identified in the anhydrobiotic tardigrade improve osmotic tolerance of human cells. PLoS ONE. 2015;10:e0118272. doi: 10.1371/journal.pone.0118272.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.pone.0118272" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4326354/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25675104/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Novel%20mitochondria-targeted%20heat-soluble%20proteins%20identified%20in%20the%20anhydrobiotic%20tardigrade%20improve%20osmotic%20tolerance%20of%20human%20cells&amp;author=S%20Tanaka&amp;volume=10&amp;publication_year=2015&amp;pages=e0118272&amp;pmid=25675104&amp;doi=10.1371/journal.pone.0118272&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Fukuda Y, Miura Y, Mizohata E, Inoue T. Structural insights into a secretory abundant heat‐soluble protein from an anhydrobiotic tardigrade, Ramazzottius varieornatus. FEBS Lett. 2017;591:2458–2469. doi: 10.1002/1873-3468.12752.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/1873-3468.12752" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28703282/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&amp;title=Structural%20insights%20into%20a%20secretory%20abundant%20heat%E2%80%90soluble%20protein%20from%20an%20anhydrobiotic%20tardigrade,%20Ramazzottius%20varieornatus&amp;author=Y%20Fukuda&amp;author=Y%20Miura&amp;author=E%20Mizohata&amp;author=T%20Inoue&amp;volume=591&amp;publication_year=2017&amp;pages=2458-2469&amp;pmid=28703282&amp;doi=10.1002/1873-3468.12752&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Fukuda Y, Inoue T. Crystal structure of secretory abundant heat soluble protein 4 from one of the toughest “water bears” micro‐animals Ramazzottius varieornatus. Protein Sci. 2018;27:993–999. doi: 10.1002/pro.3393.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/pro.3393" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5916119/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29493034/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=Crystal%20structure%20of%20secretory%20abundant%20heat%20soluble%20protein%204%20from%20one%20of%20the%20toughest%20%E2%80%9Cwater%20bears%E2%80%9D%20micro%E2%80%90animals%20Ramazzottius%20varieornatus&amp;author=Y%20Fukuda&amp;author=T%20Inoue&amp;volume=27&amp;publication_year=2018&amp;pages=993-999&amp;pmid=29493034&amp;doi=10.1002/pro.3393&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Emanuelsson O, Nielsen H, Brunak S, Von Heijne G. Predicting subcellular localization of proteins based on their N-terminal amino acid sequence. J. Mol. Biol. 2000;300:1005–1016. doi: 10.1006/jmbi.2000.3903.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1006/jmbi.2000.3903" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/10891285/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Predicting%20subcellular%20localization%20of%20proteins%20based%20on%20their%20N-terminal%20amino%20acid%20sequence&amp;author=O%20Emanuelsson&amp;author=H%20Nielsen&amp;author=S%20Brunak&amp;author=G%20Von%20Heijne&amp;volume=300&amp;publication_year=2000&amp;pages=1005-1016&amp;pmid=10891285&amp;doi=10.1006/jmbi.2000.3903&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Lau YT, et al.  Discovery and engineering of enhanced SUMO protease enzymes. J. Biol. Chem. 2018;293:13224–13233. doi: 10.1074/jbc.RA118.004146.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.RA118.004146" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6109922/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29976752/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Discovery%20and%20engineering%20of%20enhanced%20SUMO%20protease%20enzymes&amp;author=YT%20Lau&amp;volume=293&amp;publication_year=2018&amp;pages=13224-13233&amp;pmid=29976752&amp;doi=10.1074/jbc.RA118.004146&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Furuki T, Sakurai M. Group 3 LEA protein model peptides protect liposomes during desiccation. Biochim. Biophys. Acta Biomembr. 2014;1838:2757–2766. doi: 10.1016/j.bbamem.2014.07.009.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.bbamem.2014.07.009" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25037007/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta%20Biomembr.&amp;title=Group%203%20LEA%20protein%20model%20peptides%20protect%20liposomes%20during%20desiccation&amp;author=T%20Furuki&amp;author=M%20Sakurai&amp;volume=1838&amp;publication_year=2014&amp;pages=2757-2766&amp;pmid=25037007&amp;doi=10.1016/j.bbamem.2014.07.009&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Piszkiewicz S, et al.  Protecting activity of desiccated enzymes. Protein Sci. 2019;28:941–951. doi: 10.1002/pro.3604.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/pro.3604" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6459994/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30868674/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=Protecting%20activity%20of%20desiccated%20enzymes&amp;author=S%20Piszkiewicz&amp;volume=28&amp;publication_year=2019&amp;pages=941-951&amp;pmid=30868674&amp;doi=10.1002/pro.3604&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Ben Chorin A, et al.  ConSurf-DB: an accessible repository for the evolutionary conservation patterns of the majority of PDB proteins. Protein Sci. 2020;29:258–267. doi: 10.1002/pro.3779.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/pro.3779" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6933843/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/31702846/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=ConSurf-DB:%20an%20accessible%20repository%20for%20the%20evolutionary%20conservation%20patterns%20of%20the%20majority%20of%20PDB%20proteins&amp;author=A%20Ben%20Chorin&amp;volume=29&amp;publication_year=2020&amp;pages=258-267&amp;pmid=31702846&amp;doi=10.1002/pro.3779&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Patil R, et al.  A ligand-induced structural change in fatty acid-binding protein 1 is associated with potentiation of peroxisome proliferator-activated receptor α agonists. J. Biol. Chem. 2019;294:3720–3734. doi: 10.1074/jbc.RA118.006848.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1074/jbc.RA118.006848" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6416440/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30598509/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=A%20ligand-induced%20structural%20change%20in%20fatty%20acid-binding%20protein%201%20is%20associated%20with%20potentiation%20of%20peroxisome%20proliferator-activated%20receptor%20%CE%B1%20agonists&amp;author=R%20Patil&amp;volume=294&amp;publication_year=2019&amp;pages=3720-3734&amp;pmid=30598509&amp;doi=10.1074/jbc.RA118.006848&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Young AC, et al.  Structural studies on human muscle fatty acid binding protein at 1.4 Å resolution: binding interactions with three C18 fatty acids. Structure. 1994;2:523–534. doi: 10.1016/S0969-2126(00)00052-6.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0969-2126(00)00052-6" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/7922029/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Structural%20studies%20on%20human%20muscle%20fatty%20acid%20binding%20protein%20at%201.4%E2%80%89%C3%85%20resolution:%20binding%20interactions%20with%20three%20C18%20fatty%20acids&amp;author=AC%20Young&amp;volume=2&amp;publication_year=1994&amp;pages=523-534&amp;pmid=7922029&amp;doi=10.1016/S0969-2126(00)00052-6&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Balchin D, Hayer-Hartl M, Hartl FU. In vivo aspects of protein folding and quality control. Science. 2016;353:aac4354. doi: 10.1126/science.aac4354.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1126/science.aac4354" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/27365453/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=In%20vivo%20aspects%20of%20protein%20folding%20and%20quality%20control&amp;author=D%20Balchin&amp;author=M%20Hayer-Hartl&amp;author=FU%20Hartl&amp;volume=353&amp;publication_year=2016&amp;pages=aac4354&amp;pmid=27365453&amp;doi=10.1126/science.aac4354&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Lim, S., Glover, D. J. &amp; Clark, D. S. Prefoldins in archaea. In <em>Prefoldins: the New Chaperones, Advances in Experimental Medicine and Biology</em> Vol. 1106 (ed. Djouder, N.) (Springer, 2018).</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/978-3-030-00737-9_2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30484150/">PubMed</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Hibshman JD, Carra S, Goldstein B. Tardigrade small heat shock proteins can limit desiccation-induced protein aggregation. Commun. Biol. 2023;6:121. doi: 10.1038/s42003-023-04512-y.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s42003-023-04512-y" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC9887055/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/36717706/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Commun.%20Biol.&amp;title=Tardigrade%20small%20heat%20shock%20proteins%20can%20limit%20desiccation-induced%20protein%20aggregation&amp;author=JD%20Hibshman&amp;author=S%20Carra&amp;author=B%20Goldstein&amp;volume=6&amp;publication_year=2023&amp;pages=121&amp;pmid=36717706&amp;doi=10.1038/s42003-023-04512-y&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Richaud M, et al.  Ultrastructural analysis of the dehydrated tardigrade Hypsibius exemplaris unveils an anhydrobiotic-specific architecture. Sci. Rep. 2020;10:4324. doi: 10.1038/s41598-020-61165-1.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-020-61165-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC7062702/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/32152342/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Ultrastructural%20analysis%20of%20the%20dehydrated%20tardigrade%20Hypsibius%20exemplaris%20unveils%20an%20anhydrobiotic-specific%20architecture&amp;author=M%20Richaud&amp;volume=10&amp;publication_year=2020&amp;pages=4324&amp;pmid=32152342&amp;doi=10.1038/s41598-020-61165-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Zvinavashe AT, et al.  Programmable design of seed coating function induces water-stress tolerance in semi-arid regions. Nat. Food. 2021;2:485–493. doi: 10.1038/s43016-021-00315-8.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s43016-021-00315-8" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/37117674/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Food&amp;title=Programmable%20design%20of%20seed%20coating%20function%20induces%20water-stress%20tolerance%20in%20semi-arid%20regions&amp;author=AT%20Zvinavashe&amp;volume=2&amp;publication_year=2021&amp;pages=485-493&amp;pmid=37117674&amp;doi=10.1038/s43016-021-00315-8&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Heveran CM, et al.  Biomineralization and successive regeneration of engineered living building materials. Matter. 2020;2:481–494. doi: 10.1016/j.matt.2019.11.016.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.matt.2019.11.016" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Matter&amp;title=Biomineralization%20and%20successive%20regeneration%20of%20engineered%20living%20building%20materials&amp;author=CM%20Heveran&amp;volume=2&amp;publication_year=2020&amp;pages=481-494&amp;doi=10.1016/j.matt.2019.11.016&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11127935/bin/42003_2024_6336_MOESM1_ESM.pdf">Peer Review File</a><sup> (961KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11127935/bin/42003_2024_6336_MOESM2_ESM.pdf">Supplementary Information</a><sup> (2.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11127935/bin/42003_2024_6336_MOESM3_ESM.docx">Description of Supplementary Materials</a><sup> (14.8KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11127935/bin/42003_2024_6336_MOESM4_ESM.xlsx">Supplementary Data 1</a><sup> (1.2MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11127935/bin/42003_2024_6336_MOESM5_ESM.zip">Supplementary Data 2</a><sup> (765.1KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/11127935/bin/42003_2024_6336_MOESM6_ESM.pdf">Reporting Summary</a><sup> (473KB, pdf) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data are either contained within the manuscript/supporting information or available from the authors upon request. Source data for figures are included in Supplementary Data <a class="usa-link" href="#MOESM4">1</a>. Supplementary Data <a class="usa-link" href="#MOESM5">2</a> includes the initial structural pdb file of the MD simulation and those at time points of 50, 100, 150, 200, 250, and 300 ns. Bacterial expression plasmids for the production of recombinant SAHS proteins can be acquired from Addgene (Addgene plasmid ID 215645–215648).</p></section></section></section></section></article>
</main></body></html>