Front Microbiol

. 2021 Mar 15;12:639396. doi: [10.3389/fmicb.2021.639396](https://doi.org/10.3389/fmicb.2021.639396)

# *Methylobacterium ajmalii* sp. nov., Isolated From the International Space Station

[Swati Bijlani](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bijlani%20S%22%5BAuthor%5D)

### Swati Bijlani

1Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, Los Angeles, CA, United States

Find articles by [Swati Bijlani](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bijlani%20S%22%5BAuthor%5D)

1, [Nitin K Singh](https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D)

### Nitin K Singh

2Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, United States

Find articles by [Nitin K Singh](https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20NK%22%5BAuthor%5D)

2, [V V Ramprasad Eedara](https://pubmed.ncbi.nlm.nih.gov/?term=%22Eedara%20VVR%22%5BAuthor%5D)

### V V Ramprasad Eedara

3Department of Plant Science, School of Life Sciences, University of Hyderabad, Hyderabad, India

Find articles by [V V Ramprasad Eedara](https://pubmed.ncbi.nlm.nih.gov/?term=%22Eedara%20VVR%22%5BAuthor%5D)

3, [Appa Rao Podile](https://pubmed.ncbi.nlm.nih.gov/?term=%22Podile%20AR%22%5BAuthor%5D)

### Appa Rao Podile

3Department of Plant Science, School of Life Sciences, University of Hyderabad, Hyderabad, India

Find articles by [Appa Rao Podile](https://pubmed.ncbi.nlm.nih.gov/?term=%22Podile%20AR%22%5BAuthor%5D)

3, [Christopher E Mason](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D)

### Christopher E Mason

4WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, NY, United States

Find articles by [Christopher E Mason](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D)

4, [Clay C C Wang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CCC%22%5BAuthor%5D)

### Clay C C Wang

1Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, Los Angeles, CA, United States

Find articles by [Clay C C Wang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CCC%22%5BAuthor%5D)

1,\*, [Kasthuri Venkateswaran](https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D)

### Kasthuri Venkateswaran

2Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, United States

Find articles by [Kasthuri Venkateswaran](https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D)

2,\*

* Author information
* Article notes
* Copyright and License information

1Department of Pharmacology and Pharmaceutical Sciences, School of Pharmacy, University of Southern California, Los Angeles, CA, United States

2Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, United States

3Department of Plant Science, School of Life Sciences, University of Hyderabad, Hyderabad, India

4WorldQuant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, NY, United States

Edited by: Brian P. Hedlund, University of Nevada, Las Vegas, United States

Reviewed by: Jian-Yu Jiao, Sun Yat-sen University, China; En-Min Zhou, Sun Yat-sen University, China

✉

\*Correspondence: Clay C. C. Wang, clayw@usc.edu

\*

Kasthuri Venkateswaran, kjvenkat@jpl.nasa.gov

This article was submitted to Extreme Microbiology, a section of the journal Frontiers in Microbiology

Received 2020 Dec 8; Accepted 2021 Feb 22; Collection date 2021.

Copyright © 2021 Bijlani, Singh, Eedara, Podile, Mason, Wang and Venkateswaran.

This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.

[PMC Copyright notice](/about/copyright/)

PMCID: PMC8005752  PMID: [33790880](https://pubmed.ncbi.nlm.nih.gov/33790880/)

## Abstract

Four strains belonging to the family of *Methylobacteriaceae* were isolated from different locations on the International Space Station (ISS) across two consecutive flights. Of these, three were identified as Gram-negative, rod-shaped, catalase-positive, oxidase-positive, motile bacteria, designated as IF7SW-B2T, IIF1SW-B5, and IIF4SW-B5, whereas the fourth was identified as *Methylorubrum rhodesianum*. The sequence similarity of these three ISS strains, designated as IF7SW-B2T, IIF1SW-B5, and IIF4SW-B5, was <99.4% for 16S rRNA genes and <97.3% for *gyrB* gene, with the closest being *Methylobacterium indicum* SE2.11T. Furthermore, the multi-locus sequence analysis placed these three ISS strains in the same clade of *M. indicum.* The average nucleotide identity (ANI) values of these three ISS strains were <93% and digital DNA-DNA hybridization (dDDH) values were <46.4% with any described *Methylobacterium* species. Based on the ANI and dDDH analyses, these three ISS strains were considered as novel species belonging to the genus *Methylobacterium.* The three ISS strains showed 100% ANI similarity and dDDH values with each other, indicating that these three ISS strains, isolated during various flights and from different locations, belong to the same species. These three ISS strains were found to grow optimally at temperatures from 25 to 30°C, pH 6.0 to 8.0, and NaCl 0 to 1%. Phenotypically, these three ISS strains resemble *M. aquaticum* and *M. terrae* since they assimilate similar sugars as sole carbon substrate when compared to other *Methylobacterium* species. Fatty acid analysis showed that the major fatty acid produced by the ISS strains are C18:1−ω7*c* and C18:1−ω6*c*. The predominant quinone was ubiquinone 10, and the major polar lipids were diphosphatidylglycerol, phosphatidylcholine, phosphatidylethanolamine, phosphatidylglycerol, and an unidentified lipid. Therefore, based on genomic, phylogenetic, biochemical, and fatty acid analyses, strains IF7SW-B2T, IIF1SW-B5, and IIF4SW-B5, are assigned to a novel species within the genus *Methylobacterium*, and the name *Methylobacterium ajmalii* sp. nov. is proposed. The type strain is IF7SW-B2T (NRRL B-65601T and LMG 32165T).

**Keywords:** *Methylobacterium*, polyphasic taxomony, ANI, international space station (ISS), whole genome sequencing

## Introduction

The genus *Methylobacterium* contains more species than any other genera within the family *Methylobacteriaceae*, order *Rhizobiales*, and class *Alphaproteobacteria* ([Kelly et al., 2014](#B20)). *Methylobacterium* species are Gram-negative, rod-shaped bacteria. The genus was first proposed by [Patt et al. (1976)](#B38) with *Methylobacterium organophilum* as the type species. The genus *Methylobacterium* was first emended to include facultative methylotrophs that have the ability to grow on methane or methanol as the source of carbon and energy, in addition to sugars and organic acids ([Patt et al., 1976](#B38)). Another taxonomic study classified all other previously known pink-pigmented facultative methylotrophic bacteria under the genus *Methylobacterium* ([Green and Bousfield, 1982](#B13)). Thereafter, 11 species from the genus *Methylobacterium* were redefined into a new genus proposed as *Methylorubrum*, based on 16S rRNA gene sequence, multi-locus sequence analysis (MLSA), genomic, and phenotypic data ([Green and Ardley, 2018](#B12)).

The genus *Methylobacterium* consists of 45 recognized species, which are ubiquitously present in a wide variety of habitats including air, soil, freshwater, and sediments, and can exist either in free-form or associated with plant tissues ([Gallego et al.,2005a,b](#B10); [Kang et al., 2007](#B19); [Veyisoglu et al., 2013](#B46); [Kelly et al., 2014](#B20); [Kwak et al., 2014](#B27); [Chaudhry et al., 2016](#B7); [Green and Ardley, 2018](#B12); [Park et al., 2018](#B36)). *Methylobacterium* species are involved in nitrogen fixation, phosphate solubilization, abiotic stress tolerance, plant growth promotion, and biocontrol activity against plant pathogens ([Madhaiyan et al., 2006](#B31); [Kumar M. et al., 2016](#B24); [Parasuraman et al., 2019](#B35); [Grossi et al., 2020](#B14); [Krug et al., 2020](#B23)). For instance, a novel *Methylobacterium* sp. 2A was observed to result in higher density of lateral roots in inoculated potato crops, even under salt stress conditions, compared with control plants that were not inoculated with the bacteria; it was also found to exhibit biocontrol activity against several plant pathogens ([Grossi et al., 2020](#B14)). Furthermore, genomic analysis of *Methylobacterium* sp. 2A revealed the presence of metabolic pathways involved in plant growth promotion, including the genes for producing an auxin, 3-indole acetic acid ([Grossi et al., 2020](#B14)).

In an ongoing Microbial Tracking experiment on the International Space Station (ISS), four strains belonging to the family *Methylobacteriaceae* were isolated ([Checinska Sielaff et al., 2019](#B9)). Some of the *Methylobacterium* species that are phylogenetically related to these ISS strains have been isolated from plant sources ([Kang et al., 2007](#B19); [Chaudhry et al., 2016](#B7)), indicating that the ISS strains might also display properties related to plant growth promotion. The objectives of this study were to generate whole genome sequences (WGS) and define the phylogenetic novelty of the ISS *Methylobacterium* strains using polyphasic taxonomic analyses. The WGS generated and annotated in this study was used to predict biotechnologically useful genetic determinants.

## Materials and Methods

### Sample Collection and Isolation of Bacteria

Several surface samples (1 m2) were collected from the ISS during Microbial Tracking–1 flight experiments from 2015 to 2016. Sample collection, processing, and isolation of cultivable microorganisms were published elsewhere ([Checinska Sielaff et al., 2019](#B9)). Briefly, the polyester wipes used to collect samples and particulates associated with the sampling devices were transported to Earth before being disassociated into sterile phosphate-buffered saline (pH 7.4) solution and plated onto R2A agar medium ([Checinska et al., 2015](#B8); [Checinska Sielaff et al., 2019](#B9)). The microbial cultures that were grown at 25°C for 7 days were picked from the R2A plates, purified, and stored for further analyses. Distinct colonies (*n* = 4) isolated from three different locations and from a high-efficiency particulate arrestance (HEPA) filter were characterized during this study. These colonies exhibited unique coloration and differential genomic phylogeny. The type strain IF7SW-B2T was isolated during Flight 1 (March 2015) at Location #7, the Overhead-3 panel surface of the Materials Science Research Rack 1, which is used for basic materials research in the microgravity environment of the ISS. The second strain, IIF1SW-B5, was isolated during Flight 2 (May 2015) at Location #1, the Port panel of the Cupola. The Cupola is a small module devoted to the observation of operations outside the ISS, such as robotic activities, spacecraft approaches, and extravehicular activities. The third strain, IIF4SW-B5, was isolated during Flight 2 (May 2015) at Location #4, the surface of the dining table. Even though the main function of the table was for dining, crewmembers also used the table for experimental work. The fourth strain was I1-R3, isolated from the ISS HEPA filter that was returned aboard STS-134/ULF6 in May 2011 and archived as reported earlier ([Checinska et al., 2015](#B8)).

### DNA Extraction and Whole Genome Sequencing Analysis

A biomass of approximately 1 μg wet weight was collected for DNA extraction from each strain after growing on R2A medium at 25°C for 3 days. Total nucleic acid extraction was carried out using ZymoBIOMICS 96 MagBead DNA kit (lysis tubes) (Zymo Research, United States) after bead beating with a Bertin Precellys homogenizer. This was followed by library preparation using the Illumina Nextera Flex Protocol as per Illumina document number 1000000025416 v07. The initial amount of DNA for library preparation was quantified, and 5 to 12 cycles of polymerase chain reaction (PCR) amplification were carried out to normalize the output depending on the input DNA concentration. The amplified genomic DNA fragments were indexed and pooled in 384-plex configuration. Whole-genome shotgun sequencing was performed on a NovaSeq 6000 S4 flowcell PE 2 × 150 platform with a paired-end module. The data were filtered with NGS QC Toolkit v2.3 ([Patel and Jain, 2012](#B37)) for high-quality (HQ) vector- and adaptor-free reads for genome assembly (cutoff read length for HQ, 80%; cutoff quality score, 20). The number of filtered reads obtained were used for assembly with SPAdes 3.14.0 ([Bankevich et al., 2012](#B5)) genome assembler (*k*-mer size- 32 to 72 bases) using default parameters. The genome was annotated using the National Center for Biotechnology Information (NCBI) Prokaryotic Genome Annotation Pipeline 4.11 ([Tatusova et al., 2016](#B44); [Haft et al., 2018](#B15)). In addition, functional annotation of genome and seed categories were assigned to the genome by implementing the Rapid Annotations using Subsystems Technology (RAST) tool ([Aziz et al., 2008](#B4)).

Genomes of all other strains used in this study were downloaded from NCBI, and the genomic relatedness of ISS strains was identified based on average nucleotide identity (ANI; FastANI) calculations ([Jain et al., 2018](#B16)) and digital DNA-DNA hybridization (dDDH) analysis ([Meier-Kolthoff et al., 2013](#B32)). FastANI was run on all the genomes using the default parameters: Mashmap identity cutoff *I*0 = 80%, non-overlapping fragments of size *l* = *3Kb*, and minimum count of reciprocal mappings *τ* = 50.

### Phylogenetic Analysis

Phylogenetic analysis was carried out based on 16S rRNA gene sequencing, and MLSA using six housekeeping genes: ATP synthase F1 beta subunit (*atpD*), DNA strand exchange and recombination gene (*recA*), chaperone gene (*dnaK*), DNA-directed RNA polymerase subunit beta (*rpoB*), glutamine synthetase type I (*glnI*), and DNA gyrase subunit B (*gyrB*), for differentiating *Methylobacterium* species ([Green and Ardley, 2018](#B12)). The 16S rRNA gene sequences of type strains of all 45 *Methylobacterium* species were included in the phylogenetic analysis. In addition, representative species of genus *Methylorubrum*, *Enterovirga*, *Microvirga*, and *Neomegalonema* from family *Methylobacteriaceae*, *Rhizobium* from order *Rhizobiales*, *Caulobacter* from order *Caulobacterales*, in class *Alphaproteobacteria* were included. *Pseudomonas aeruginosa* was selected as the outgroup.

The 16S rRNA gene sequences of all strains were retrieved from NCBI except for the four ISS strains, which were recovered from their respective WGS. Phylogenetic analysis based on housekeeping genes and MLSA was carried out with type strains of 24 *Methylobacterium* species and representative species of other genera. All the gene sequences were retrieved from the genome sequences using RAST v2.0[1](#footnote1) ([Aziz et al., 2008](#B4); [Overbeek et al., 2014](#B34); [Brettin et al., 2015](#B6)). The individual gene sequences for all strains were aligned separately using ClustalW, and then the maximum likelihood tree was generated using MEGA 7.0.26 ([Kumar S. et al., 2016](#B25)). For MLSA, six housekeeping gene sequences for each strain were concatenated manually and aligned using ClustalW, and then the maximum likelihood tree was generated using MEGA 7.0.26 ([Kumar S. et al., 2016](#B25)).

The genome-based tree for the *Methylobacterium* species, including ISS strains and representative species of other genus with available WGS, was constructed using GToTree ([Lee, 2019](#B28)). This tool takes the complete/draft genomes as input and creates a phylogenomic tree based on the prespecified single-copy gene set using a hidden Markov model (HMM); the tool currently has 2,044 unique HMM set as identifiers to cover all three domains of microbial life.

### Phenotypic Characterization of ISS Strains

Phenotypic characterization was performed according to standard protocols ([Jones, 1981](#B17)). Growth of the ISS strains at different temperatures (7, 25, 30, 37, and 45°C) was assessed after incubation on nutrient agar (Sigma, United States) for 7 days. Growth at different pH (4.0–10.0 at intervals of 1.0) was assessed after incubation in nutrient broth (Sigma, United States) at 30°C for 7 days. The pH of the nutrient medium was adjusted using citrate/NaH2PO4 buffer (pH 4.0–5.0), phosphate buffer (pH 6.0–8.0), and tris buffer (pH 9.0–10.0) ([Kim et al., 2019](#B21)). Salt tolerance was tested by streaking the strains on R2A supplemented with NaCl (0–10% at intervals of 1%) and incubating the plates at 30°C for 7 days. Motility was assessed via the “hanging drop” method by observing the culture under a light microscope ([Tindall et al., 2007](#B45)). Catalase activity was tested by adding 3% hydrogen peroxide to culture grown on R2A at 30°C for 7 days, and effervescence was monitored ([Tindall et al., 2007](#B45)). An oxidase test was carried out in a filter paper soaked with the substrate tetramethyl-p-phenylenediamine dihydrochloride, and coloration was documented ([Jurtshuk Jr. and McQuitty, 1976](#B18)). All other physiological and biochemical tests were carried out using API 20 NE, API 50 CH, and API ZYM kits as per manufacturer’s procedures (bioMérieux, France).

### Chemotaxonomic Analysis

All strains grown in the R2A broth were harvested when growth of the cultures reached around 70% of the maximal optical density (exponential growth phase), and then the cultures were used for analyses of cellular fatty acids, polar lipids, and quinones, which were carried out as described previously ([Ramaprasad et al., 2015](#B40)). Briefly, for cellular fatty acids analysis, 40 mg of bacterial cell pellet from each strain was subjected to a series of four different reagents followed by saponification and methylation of fatty acids, thus enabling their cleavage from lipids. The fatty acid methyl esters (FAME) thus obtained were analyzed by a gas chromatograph equipped with Sherlock MIS software (Microbial ID; MIDI 6.0 version; Agilent: 6850)[2](#footnote2). The peaks obtained were then labeled, and the equivalent chain length (ECL) values were computed by the Sherlock software.

The polar lipids profile was analyzed by extracting cells with methanol-chloroform-saline (2:1:0.8, v/v/v) from 1 g of freeze-dried bacterial cells. Separation of lipids was performed by two-dimensional chromatography on a silica gel thin-layer chromatography plate (Kieselgel 60 F254; Merck) using chloroform-methanol-water (75:32:4, v/v/v) in the first dimension and chloroform–methanol–acetic acid–water (86:16:15:4, v/v/v/v) in the second dimension. The total polar lipids profile was detected by spraying with 6% ethanolic molybdophosphoric acid. The respiratory isoprenoid quinone was extracted with a chloroform-methanol mixture (2:1, v/v), evaporated under vacuum, re-extracted with acetone, and analyzed using high-performance lipid chromatography as per established methods ([Ramaprasad et al., 2018](#B39)).

## Results and Discussion

This study reports the isolation and identification of four strains belonging to the family *Methylobacteriaceae*, collected from different locations on the ISS. Three of the strains, referred to as IF7SW-B2T, IIF1SW-B5, and IIF4SW-B5, were identified based on the traditional and genomic taxonomic approaches. The fourth strain, which was isolated from a HEPA filter and referred to as I1-R3, was identified based on genomic analyses only.

### Phylogenetic Analysis of Novel ISS Strains

To confirm that three of the ISS strains (IF7SW-B2T, IIF1SW-B5, and IIF4SW-B5) belong to a novel species, their phylogenetic affiliations were analyzed with other species belonging to the genus *Methylobacterium*. The sequence similarity of these three ISS strains with validly described *Methylobacterium* species was <99.4% for 16S rRNA gene ([Supplementary Table 1](#TS1)) and <97.3% for *gyrB* gene with the closest being *M. indicum* SE2.11T. Phylogenetic analysis of these three ISS strains was carried out by constructing a maximum likelihood tree based on 16S rRNA ([Figure 1](#F1)), *gyrB* ([Figure 2](#F2)), *atpD* ([Supplementary Figure 1](#FS1)), *recA* ([Supplementary Figure 2](#FS1)), *dnaK* ([Supplementary Figure 3](#FS1)), *rpoB* ([Supplementary Figure 4](#FS1)), and *glnI* ([Supplementary Figure 5](#FS1)) gene sequences. In addition, MLSA was carried out by concatenating the six housekeeping genes manually ([Figure 3](#F3)). In addition, a phylogenetic tree based on WGS was generated ([Figure 4](#F4)). The phylogenetic trees constructed based on all these genes, MLSA, and WGS showed that these three ISS strains (IF7SW-B2T, IIF1SW-B5, and IIF4SW-B5) are clustered together and in the same clade with *M. indicum* SE2.11T. The 16S rRNA gene-sequencing, housekeeping gene-based analyses, MLSA, and genome-based tree further supported the concept that these three ISS strains belong to the same species but are closely related to *M. indicum*. In addition, the identity of the ISS strain I1-R3 was further confirmed to be *M. rhodesianum* based on its 16S rRNA gene ([Figure 1](#F1)) and *gyrB* ([Figure 2](#F2)) phylogenetic affiliation to the type strain *M. rhodesianum* DSM 5687T.

#### FIGURE 1.

[![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5d23/8005752/c21b5f11d30b/fmicb-12-639396-g001.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8005752_fmicb-12-639396-g001.jpg)

[Open in a new tab](figure/F1/)

Maximum likelihood phylogenetic tree based on 16S rRNA gene sequences shows the relationship of *Methylobacterium ajmalii* sp. nov. with members of the family *Methylobacteriaceae*. Bootstrap values from 1,000 replications are shown at branch points. Bar, 0.02 substitution per site.

#### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5d23/8005752/0166e95f64be/fmicb-12-639396-g002.jpg)

[Open in a new tab](figure/F2/)

Maximum likelihood phylogenetic tree, based on DNA gyrase gene (*gyrB*) sequences, showing the phylogenetic relationship of *Methylobacterium ajmalii* sp. nov. with members of the family *Methylobacteriaceae*. Bootstrap values from 1,000 replications are shown at branch points. Bar, 0.05 substitution per site.

#### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5d23/8005752/c2cc77af78ab/fmicb-12-639396-g003.jpg)

[Open in a new tab](figure/F3/)

Maximum likelihood phylogenetic tree, based on six gene sequences (*atpD*, *recA*, *dnaK*, *rpoB*, *glnI*, and *gyrB*) concatenated manually, showing the phylogenetic relationship of *Methylobacterium ajmalii* sp. nov. with members of the family *Methylobacteriaceae*. Bootstrap values from 1,000 replications are shown at branch points. Bar, 0.05 substitution per site.

#### FIGURE 4.

[![FIGURE 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5d23/8005752/ac1a7e7ac8a6/fmicb-12-639396-g004.jpg)](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8005752_fmicb-12-639396-g004.jpg)

[Open in a new tab](figure/F4/)

Genome-based phylogenetic tree showing the phylogenetic relationship of *Methylobacterium ajmalii* sp. nov. with members of the family *Methylobacteriaceae*.

### Whole Genome Sequence–Based Phylogenetic Analysis

The genomes of the four isolated ISS strains were sequenced, with their draft genome assembled and annotated. The results are summarized in [Table 1](#T1). The genome varied in size from 6.1 to 6.8 Mbp with GC content between 68 and 71%, similar to other members of the family *Methylobacteriaceae*.

#### TABLE 1.

Summary of the draft whole-genome sequences of four strains belonging to the family *Methylobacteriaceae*, isolated from the ISS.

| Species/Strain | NCBI Accession no. | Isolation location | No. of scaffolds | Genome size (bp) | N5o (bp) | Average Coverage | G + C content (%) | Filtered reads used for assembly (million) | Coding sequences |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| *Methylobacterium ajmalii IF7SW-B2T* | [JACWCTOOOOOOOOO](https://ncbi.nlm.nih.gov/nucleotide/JACWCTOOOOOOOOO) | Lab 3 overhead | 192 | 6,802,552 | 59,313 | 698 | 71.07 | 33.55 | 6,255 |
| *Methylobacterium ajmalii* IIF1SW-B5 | [JACWCUOOOOOOOOO](https://ncbi.nlm.nih.gov/nucleotide/JACWCUOOOOOOOOO) | Cupola | 193 | 6,593,618 | 50,984 | 754 | 71.03 | 36.29 | 6,076 |
| *Methylobacterium ajmalii* IIF4SW-B5 | [JACWCVOOOOOOOOO](https://ncbi.nlm.nih.gov/nucleotide/JACWCVOOOOOOOOO) | Dining table | 966 | 6,534,937 | 10,467 | 538 | 70.77 | 25.01 | 6,538 |
| *Methylorubrum rhodesianum* I1-R3 | [JACWCWOOOOOOOOO](https://ncbi.nlm.nih.gov/nucleotide/JACWCWOOOOOOOOO) | HEPA filter | 160 | 6,159,250 | 98,200 | 302 | 68.96 | 13.07 | 5,555 |

[Open in a new tab](table/T1/)

Due to higher sequence similarities of three ISS strains with *M. indicum* SE2.11T (99.4% for 16S rRNA gene and 97.3% for *gyr*B gene), the draft genomes of three ISS strains were subjected to ANI and dDDH analysis with other species belonging to family *Methylobacteriaceae* ([Table 2](#T2)). The ANI indices of three ISS strains (IF7SW-B2T, IIF1SW-B5, and IIF4SW-B5) with *M. indicum* SE2.11T were 92.7 to 93%, and dDDH values were 45.8 to 46.4%. The ANI and dDDH values obtained for three ISS strains with other *Methylobacterium* species were below the threshold of 95% ANI ([Yoon et al., 2017](#B49)) and 70% dDDH values ([Auch et al., 2010](#B2)), which were established for prokaryotic species delineation. This suggested that these three ISS strains are novel species of the genus *Methylobacterium*. These three ISS strains exhibited ANI and dDDH values around 99–100% with each other, indicating that they belong to the same species. The entire genomes of these three ISS strains, *M. indicum* SE2.11T, and *M. platani* PMB02T were aligned to detect their divergence and similarity using the MUMmer 3.0 system ([Kurtz et al., 2004](#B26)). As shown in [Supplementary Figure 6](#FS1), genomes of these three ISS strains aligned perfectly, while the closest genomes of *M. indicum* and *M. platani* exhibited divergence with the ISS type strain IF7SW-B2T. Since these three ISS strains were isolated at different time periods and from various locations, their persistence in the ISS environment and ecological significance in the closed systems warrant further study.

#### TABLE 2.

Genomic analyses of *Methylobacterium ajmalii* in comparison to other species of the family *Methylobacteriaceae*.

|  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Species (NCBI accession no.) | ANI value (%)  --- | | | dDDH (%) (Formula 2)  --- | | |
|  | IF7SW-B2T | IIF1SW-B5 | IIF4SW-B5 | IF7SW-B2T | IIF1SW-B5 | IIF4SW-B5 |
| *Methylobacterium ajmalii* IF7SW-B2T ([JACWCT000000000](https://ncbi.nlm.nih.gov/nucleotide/JACWCT000000000)) | 100 | 100 | 100 | 100 | 100 | 99.9 |
| *Methylobacterium ajmalii* IIF1SW-B5 ([JACWCUOOOOOOOOO](https://ncbi.nlm.nih.gov/nucleotide/JACWCUOOOOOOOOO)) | 100 | 100 | 100 | 100 | 100 | 99.9 |
| *Methylobacterium ajmalii* IIF4SW-B5 ([JACWCVOOOOOOOOO](https://ncbi.nlm.nih.gov/nucleotide/JACWCVOOOOOOOOO)) | 99.7 | 99.6 | 100 | 99.9 | 99.9 | 100 |
| *Methylobacterium indicum* SE2.11T ([GCA\_001043895.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_001043895.1)) | 93.0 | 92.9 | 92.7 | 45.8 | 45.8 | 46.4 |
| *Methylobacterium currus* PR1016A1 ([GCA\_003058325.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_003058325.1)) | 90.5 | 90.5 | 90.8 | 36.4 | 36.4 | 37.5 |
| *Methylobacterium terrae* KCTC 52904T ([GCA\_003173755.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_003173755.1)) | 90.4 | 90.4 | 90.4 | 34.8 | 34.8 | 35.8 |
| *Methylobacterium platani* JCM 14648T ([GCA\_001043885.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_001043885.1)) | 90.2 | 90.3 | 90.4 | 35.4 | 35.5 | 36.4 |
| *Methylobacterium tarhaniae* DSM 25844T ([GCA\_001043955.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_001043955.1)) | 89.7 | 89.5 | 89.7 | 34.5 | 34.6 | 35.6 |
| Methylobacterium aquaticum DSM 16371T ([GCA\_001043915.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_001043915.1)) | 89.0 | 89.0 | 89.1 | 33.6 | 33.6 | 34.5 |
| *Methylobacterium variabile* DSM 16961T ([GCA\_001043975.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_001043975.1)) | 88.9 | 88.8 | 88.9 | 33.5 | 33.6 | 34.4 |
| *Methylobacterium crusticola* MIMD6T ([GCA\_003574465.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_003574465.1)) | 84.7 | 84.9 | 85.1 | 26.4 | 26.4 | 27.1 |
| *Methylobacterium nodulans* ORS 2060T ([GCA\_000022085.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_000022085.1)) | 82.7 | 82.7 | 82.8 | 24.2 | 24.3 | 24.9 |
| *Methylobacterium dankookense* DSM 22415T ([GCA\_902141855.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_902141855.1)) | 81.6 | 81.5 | 81.7 | 22.9 | 23.0 | 23.5 |
| *Methylobacterium segetis* 17J42-11 ([GCA\_004348265.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_004348265.1)) | 81.2 | 81.2 | 81.4 | 22.7 | 22.8 | 23.4 |
| *Methylobacterium oxalidis* DSM 24028T ([GCA\_007992195.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_007992195.1)) | 81.2 | 81.1 | 81.2 | 22.3 | 22.4 | 23.0 |
| *Methylobacterium durans* KCTC 52908T ([GCA\_003173715.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_003173715.1)) | 81.0 | 80.9 | 81.0 | 22.4 | 22.5 | 23.1 |
| *Methylobacterium organophilum* DSM 760T ([GCA\_003096615.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_003096615.1)) | 80.9 | 80.8 | 81.2 | 22.4 | 22.4 | 22.9 |
| *Methylobacterium radiotolerans* JCM 2831T ([GCA\_000019725.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_000019725.1)) | 80.9 | 80.8 | 81.2 | 22.2 | 22.3 | 22.8 |
| *Methylobacterium brachiatum* TX0642T ([GCA\_003697185.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_003697185.1)) | 80.9 | 80.9 | 81.1 | 22.5 | 22.5 | 22.9 |
| *Methylobacterium soli* YIM 48816T ([GCA\_008806385.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_008806385.1)) | 80.8 | 80.7 | 80.9 | 22.2 | 22.2 | 22.8 |
| *Methylobacterium pseudosasicola* BL36T ([GCA\_900114535.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_900114535.1)) | 80.7 | 80.6 | 80.8 | 21.7 | 21.8 | 22.3 |
| *Methylorubrum extorqunens* TK 0001T ([GCA\_900234795.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_900234795.1)) | 80.6 | 80.6 | 80.9 | 22.0 | 22.0 | 22.5 |
| *Methylobacterium oryzae* DSM 18207T ([GCA\_000757795.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_000757795.1)) | 80.6 | 80.4 | 80.6 | 21.9 | 22.0 | 22.5 |
| *Methylobacterium phyllosphaerae* DSM 19779T ([GCA\_001936175.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_001936175.1)) | 80.6 | 80.5 | 80.6 | 21.8 | 21.9 | 22.4 |
| Methylobacterium phyllostachyos BL47T ([GCA\_900103445.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_900103445.1)) | 80.4 | 80.4 | 80.4 | 21.8 | 21.8 | 22.3 |
| Methylobacterium gossipiicola CCM 7572T ([GCA\_900113485.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_900113485.1)) | 80.3 | 80.4 | 80.4 | 21.8 | 21.9 | 22.5 |
| *Methylobacterium haplocladii* DSM 24195T ([GCA\_007992175.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_007992175.1)) | 80.3 | 80.2 | 80.4 | 21.7 | 21.8 | 22.3 |
| *Methylobacterium gnaphalii* DSM 24027T ([GCA\_007992215.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_007992215.1)) | 79.9 | 79.8 | 80.0 | 21.4 | 21.5 | 21.9 |
| *Microvirga subterranea* DSM 14364T ([GCA\_003350535.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_003350535.1)) | 79.0 | 78.9 | 78.8 | 20.8 | 20.8 | 21.2 |
| *Enterovirga rhinocerotis* DSM 25903T ([GCA\_004363955.1](https://ncbi.nlm.nih.gov/nucleotide/GCA_004363955.1)) | 78.1 | 78.1 | 78.3 | 20.8 | 20.8 | 20.9 |

[Open in a new tab](table/T2/)

The fourth strain I1-R3 was identified as *M. rhodesianum* based on highly similar 16S rRNA (99.9%), *gyrB* (100%), ANI (98.9%), and dDDH (91.6%) genomic parameters with *M. rhodesianum* DSM 5687T. The pigmentation of the strain I1-R3 (light pink) was also different from the novel ISS *Methylobacterium* strains (reddish pink). The ANI and dDDH values between I1-R3 and the three novel ISS *Methylobacterium* strains were ∼82% and 24%, respectively. Hence, genomic and morphological analyses confirmed the phylogenetic affiliation of strain I1-R3 as *M. rhodesianum*. In this communication, phylogenetic affiliations of only IF7SW-B2T, IIF1SW-B5, and IIF4SW-B5 strains were presented.

### Phenotypic Characterization of Novel ISS Strains

The minimal information about the ISS strain genome characteristics are given in [Supplementary Table 2](#TS1). The differential phenotypic characteristics of IF7SW-B2T, IIF1SW-B5, and IIF4SW-B5 are listed in [Table 3](#T3), in comparison with other related *Methylobacterium* species. Three strains belonging to *Methylobacterium* sp. nov. are reddish pink–pigmented, Gram-stain-negative, catalase-positive, oxidase-positive, motile, and rod-shaped. These strains grew well on nutrient agar and R2A. These three strains grew optimally at temperatures between 25 and 30°C, were viable only at pH 6.0 to 8.0, and exhibited poor tolerance to salt (0 to 1%). Absence of growth was observed when grown at 7, 37, and 45°C. These strains were positive for assimilation of L-arabinose, D-glucose, maltose, D-mannitol, D-mannose, malic acid, potassium gluconate, and trisodium citrate. These strains also exhibited esterase lipase and trypsin enzymatic activities. The complete results of phenotypic characteristics determined using API 20 NE, API ZYM, and API 50 CH are detailed in [Supplementary Tables 3-5](#TS1), respectively. The majority of the phenotypic characteristics of the ISS strains were similar to other *Methylobacterium* species. Phenotypically, these three ISS strains were different from the closest genomic relative *M. indicum* in assimilating glucose, malic acid, maltose, mannitol, potassium gluconate, and trisodium citrate. Furthermore, unlike *M. indicum*, these ISS strains did not exhibit growth at pH 5.0. In comparison to other *Methylobacterium* species, *M. aquaticum* and *M. terrae* exhibit similar carbon substrate utilization and enzyme production profiles. However, malic acid was assimilated by these ISS strains but not by *M. aquaticum*. Maltose was also utilized by these ISS strains but not by *M. terrae* cells.

#### TABLE 3.

Differential phenotypic characteristics of *Methylobacterium ajmalii* and related species of genus *Methylobacterium*.

| Characteristic | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Growth temperature (°C) | 25-30\* | 18-42 | 20-30 | 20-30 | 20-30 | 10-37 | 18-37 | 20-37 | 10-40 |
| Growth pH | 6.0-8.0 | 5.0-9.0 | 5.0-7.0 | 5.0-8.0 | 6.0-8.0 | 4.0-9.0 | 6.0-8.0 | 5.0-8.0 | 4.0-7.0 |
| Catalase | + | W | + | W | + | + | + | + | + |
| Oxidase | + | + | − | − | + | + | + | − | + |
| Reduction of nitrate to nitrite | − | w | w | w | w | − | − | + | W |
| Starch hydrolysis | − | − | + | − | − | + | + | − | − |
| Assimilation (API-20NE) of: |  |  |  |  |  |  |  |  |  |
| D-glucose | + | w | W | + | − | + | + | − | − |
| L-arabinose | + | + | + | − | + | + | + | + | + |
| D-mannose | + | − | W | − | − | + | + | − | − |
| D-mannitol | + | − | + | − | − | - | + | − | − |
| Maltose | + | − | + | − | − | − | − | − | − |
| Potassium gluconate | + | + | + | − | + | − | − | − | −+ |
| Malic acid | + | + | − | + | − | + | + | − | w |
| Trisodium citrate | + | + | − | − | − | + | + | − | w |
| Phenyl acetic acid | − | .- | W | − | − | − | + | − | − |
| Enzymatic activity (API-ZYM) of: |  |  |  |  |  |  |  |  |  |
| Esterase lipase | + | + | W | w | + | − | + | N.D. | + |
| Cystine arylamidase | − | + | W | w | w | + | + | N.D. | − |
| Trypsin | + | + | W | w | w | − | + | N.D. | − |

[Open in a new tab](table/T3/)

*Strains: 1: *Methylobacterium* sp. nov. (*n* = 3, where n: IF7SW-B2*T*, IIF1SW-B5 and IIF4SW-B5) (this study); 2: *M. currus* PR1016A*T* ([Park et al., 2018](#B36)); 3: *M. aquaticum* DSM 16371*T* ([Gallego et al., 2005a](#B10); [Park et al., 2018](#B36)); 4: M. variable DSM 16961*T* ([Gallego et al., 2005b](#B11); [Park et al., 2018](#B36)); 5: M. platani JCM 14648*T* ([Kang et al., 2007](#B19); [Park et al., 2018](#B36)); 6: M. tarhaniae DSM 25844*T* ([Veyisoglu et al., 2013](#B46); [Kim et al., 2019](#B21)); 7: M. terrae KCTC 52904*T* ([Kim et al., 2019](#B21)); 8: M. indicum SE2.11*T* ([Chaudhry et al., 2016](#B7)); 9: M. frigidaeris IER25-16*T* ([Lee and Jeon, 2018](#B29)). w, weakly positive; +, Positive; -, Negative; N.D., not determined. All strains exhibited 0-1% NaCl tolerance and were motile. \*Growth was tested at 7, 25, 30, 37, and 45°C and cells were grown optimally at 25 and 30°C but no growth was observed at 7, 37, and 45°C. Growth lower than 25°C might be possible but not tested.*

The main phenotypic characteristics of the ISS strains IF7SW-B2T, IIF1SW-B5, and IIF4SW-B5 were in accordance with the description of the genus *Methylobacterium*, with the most important being reddish pink pigmentation ([Green and Bousfield, 1982](#B13)). The optimum growth conditions (temperature, pH, salt tolerance) of the ISS *Methylobacterium* strains were similar to other members belonging to the genus *Methylobacterium*. Also, these three ISS strains shared the properties of exhibiting catalase activity and motility with other *Methylobacterium* species. However, the three novel ISS *Methylobacterium* strains differed from other members of the genus *Methylobacterium* in some of the phenotypic characteristics, as shown in [Table 3](#T3). For instance, they exhibited properties like assimilation of certain sugars, which was absent in some of the *Methylobacterium* species. They also did not show cystine arylamidase activity as opposed to several related *Methylobacterium* species.

### Chemotaxonomic Characterization of Novel ISS Strains

The FAME profiling of three ISS strains and other related *Methylobacterium* species are given in [Table 4](#T4). The major fatty acids in these ISS strains were C18:1 ω7*c* and/or C18:1 ω6*c* (Sum in Feature 8; 82 to 85%) with small amounts of C18:0 3-OH, C16:0, C17:0, Sum in Feature 3, Sum in Feature 2, C18:0 and C12:0, and traces of C11:0, C13:0, and C14:0. The fatty acids, C18:1 ω7*c* and/or C18:1 ω6*c*, were observed to be dominant in these ISS strains, similar to other species. However, complete FAME profiles were not consistent among *Methylobacterium* species and some significant differences in the proportions of certain fatty acids were observed ([Table 4](#T4)). The notable difference in the FAME profile was the lower abundance of C18:1 ω7*c* in *M. indicum* (46%) when compared with these ISS strains (82 to 85%).

#### TABLE 4.

Percentage of total cellular fatty acids from *Methylobacterium ajmalii* and related species of genus *Methylobacterium*.

| Fatty acids | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| C9:0 | - | - | - | 0.2 | - | 0.5 | - | - | - | - | - |
| C11:0 | 0.31 | 0.66 | 0.32 | 0.7 | - | 0.8 | - | - | - | - | - |
| C12:0 | 1.06 | 0.75 | 1.81 | - | - | - | tr | - | - | 1.29 | 7.0 |
| C13:0 | 0.76 | 0.65 | 0.45 | - | - | - | - | - | - | - | - |
| C14:0 | 0.71 | 0.42 | 0.49 | 0.28 | tr | 0.9 | tr | - | tr | 1.18 | - |
| C16:0 | 3.02 | 2.66 | 2.22 | 8.7 | 4.1 | 8.0 | 7.6 | 5.7 | 5.8 | 6.16 | 4.2 |
| C17:0 | 3.09 | 2.32 | 2.02 | 0.4 | - | - | - | - | - | 1.07 | - |
| C18:0 | 1.09 | 0.71 | 0.57 | 5.9 | 1.0 | 6.6 | 1.71 | 2.8 | 1.6 | 2.43 | 2.8 |
| C19:0 | - | - | - | - | tr | - | - | - | tr | - | - |
| C8:o3-OH | - | - | - | 1.1 | - | 1.5 | - | - | - | - | - |
| iso- C10:0 | - | - | - | - | 2.1 | - | - | - | 2.4 | - | - |
| C10:02-OH | - | - | - | 0.2 | - | 0.4 | - | - | - | - | - |
| C11:03-OH | - | - | - | - | tr | - | - | - | tr | - | - |
| C12:1 at 11-12 | - | - | - | - | tr | - | - | - | tr | - | - |
| iso- C13:0 | - | - | - | - | tr | - | - | - | tr | 2.46 | - |
| C13:0 2-OH | - | - | - | - | 1.3 | - | - | - | - | - | - |
| anteiso- C14:0 | - | - | - | - | - | - | - | - | - | 1.94 | - |
| C14:1-ω5c | - | - | - | - | 2.4 | - | tr | - | 2.4 | 2.21 | - |
| anteiso- Ci5:o | - | - | - | - | tr | - | - | - | 1.2 | 3.10 | - |
| iso- C15:0 | - | - | - | - | - | - | - | - | - | 2.22 | - |
| iso- C15:0 3-OH | - | - | - | - | - | - | - | - | - | - | 1.6 |
| iso- C15:1 F | - | - | - | - | tr | - | - | - | tr | - | - |
| iso- C15:1 G | - | - | - | - | - | - | - | - | - | 1.86 | - |
| C16:0 N-alcohol | - | - | - | - | - | - | tr | - | - | - | - |
| C16:1- ω5c | - | - | - | - | 4.9 | - | - | - | 4.3 | - | - |
| anteiso- C17:0 | - | - | - | - | - | - | - | - | - | 1.80 | - |
| anteiso- C17:1-ω9c | - | - | - | - | tr | - | - | - | Tr | 2.07 | - |
| iso- C17:1-ω5c | - | - | - | - | - | - | - | - | - | 2.86 | - |
| iso- C17:0 3-OH | - | - | - | - | tr | - | - | - | tr | 1.76 | - |
| C17:1- ω7c | - | - | - | - | tr | - | - | - | tr | - | - |
| iso-C18:0 | 0.58 | 0.46 | 0.40 | - | - | - | - | - | - | - | - |
| iso-C18:1 H | - | - | - | - | tr | - | - | - | tr | - | - |
| C18:0- ω5c | 0.26 | 0.28 | 0.26 | - | - | 0.7 | - | - | - | - | - |
| C18:1- ω5c | - | - | - | 0.6 | - | - | - | - | - | - | - |
| C18:1- ω9c | - | - | - | - | - | - | tr | - | - | - | - |
| C18:0-3OH | 3.14 | 2.41 | 3.84 | 2.4 | 1.4 | 1.7 | 3.46 | - | 2.0 | - | 4.9 |
| C18:3- ω6c | - | - | - | - | - | - | tr | - | - | 4.99 | - |
| iso-C19:0 | - | - | - | - | tr | - | - | - | - | - | - |
| C19:0 10-methyl | - | - | - | - | - | - | tr | - | - | - | - |
| C19:0 cyclo- ω8c | - | - | - | - | - | - | - | - | - | - | 1.5 |
| C20:1- ω7c | - | - | - | 0.4 | - | - | - | - | - | - | - |
| C20:2- ω6,9c | - | - | - | - | - | - | - | - | - | - | 1.5 |
| Sum In Feature 8\* | 82.09 | 85.08 | 83.97 | 66.7 | 57.6 | 60.3 | 81.0 | 86.4 | 57.5 | 46.03 | 73.8 |
| Sum In Feature 5\* | - | - | - | 0.6 | - | 0.7 | - | - | - | - | - |
| Sum In Feature 3\* | 2.43 | 2.05 | 2.37 | 1.3 | 10.3 | 1.0 | 2.24 | 1.9 | 11.2 | - | 0.9 |
| Sum In Feature 2\* | 2.17 | 1.99 | 1.78 | 1.8 | 2.2 | 2.1 | 1.98 | 2.8 | 1.5 | 3.22 | 1.9 |

[Open in a new tab](table/T4/)

*Strains: 1: IF7SW-B2*T* (this study), 2: IIF1SW-B5 (this study); 3: IIF4SW-B5 (this study); 4: *M. currus* PR1016A*T* ([Park et al., 2018](#B36)); 5: *M. aquaticum* DSM 16371*T* ([Kim et al., 2019](#B21)); 6*: M. variable* DSM 16961*T* ([Park et al., 2018](#B36)); 7: *M. platani* JCM 14648*T* ([Kang et al., 2007](#B19)); 8: *M. tarhaniae* DSM 25844*T* ([Kim et al., 2019](#B21)); 9: *M. terrae* KCTC 52904*T* ([Kim et al., 2019](#B21)); 10: *M. indicum* SE2.11*T* ([Chaudhry et al., 2016](#B7)); 11: *M. frigidaeris* IER25-16*T* ([Lee and Jeon, 2018](#B29)). -: Not detected, tr: trace amount (<1%). \*Summed features represent groups of two or three fatty acids that cannot be separated using the MIDI system. Summed feature 2 (iso-C16:1 I and/or C14:0 3-OH), summed feature 3 (C16:1 ω7*c* and/or C16:1 ω6*c*), Sum In Feature 5 comprises of C18:0-ω6,9c and ante-C18:0 and summed feature 8 (C18:1 ω7*c* and/or C18:1ω6*c*).*

These ISS strains contained Q-10 as the major respiratory isoprenoid quinone, which is common in members of the genus *Methylobacterium*. The polar lipids present in these three strains were diphosphatidylglycerol (DPG), phosphatidylglycerol (PG), phosphatidyl choline (PC), phosphatidyl-ethanolamine (PE), and an unidentified lipid ([Supplementary Figure 7](#FS1)). The total polar lipid profile of these ISS strains was consistent with their close relatives, predominated with phospholipids, DPG, PG, and PE. Furthermore, the chemotaxonomic data together with the results of the genomic and phylogenetic analysis support the affiliation of strains IF7SW-B2T, IIF1SW-B5, and IIF4SW-B5 to the genus *Methylobacterium*.

### Functional Characteristics of the Novel ISS Strain

The genome of the ISS strain IF7SW-B2T, type strain, was annotated and analyzed to determine biotechnologically important genetic determinants. The whole genome and annotation analysis predicted a total of 6,531 genes in the assembled draft genome. Among these, 1,430 fell into various RAST categories, contributing to 2,067 predicted features described in [Table 5](#T5). All the 1,430 feature and subsystems have been documented in [Supplementary Data 1](#DS1). A major fraction of the annotated genes was composed of amino acids and derivatives (408), carbohydrate metabolism (246), protein metabolism (198), genes associated with cofactors, vitamins, prosthetic groups, pigments metabolism (190), and respiration (151) ([Table 5](#T5)). Genes responsible for motility and chemotaxis (95), metabolism of aromatic compounds (47), and stress response (72) were also observed.

#### TABLE 5.

Genes belonging to different functional categories based on annotation generated using RAST for *Methylobacterium ajmalii* IF7SW-B2T.

| Functional description | Predicted genes\* |
| --- | --- |
| Cofactors, Vitamins, Prosthetic Groups, Pigments | 190 |
| Cell Wall and Capsule | 26 |
| Virulence, Disease, and Defense | 61 |
| Potassium metabolism | 10 |
| Photosynthesis | 11 |
| Miscellaneous | 17 |
| Phages, Prophages, Transposable elements, Plasmids | 25 |
| Membrane Transport | 69 |
| RNA Metabolism | 40 |
| Nucleosides and Nucleotides | 92 |
| Protein Metabolism | 198 |
| Cell Division and Cell Cycle | 2 |
| Motility and Chemotaxis | 95 |
| Regulation and Cell signaling | 49 |
| Secondary Metabolism | 5 |
| DNA Metabolism | 99 |
| Fatty Acids, Lipids, and Isoprenoids | 94 |
| Nitrogen Metabolism | 14 |
| Dormancy and Sporulation | 1 |
| Respiration | 151 |
| Stress Response | 72 |
| Metabolism of Aromatic Compounds | 47 |
| Amino Acids and Derivatives | 408 |
| Sulfur Metabolism | 17 |
| Phosphorus Metabolism | 28 |
| Carbohydrates | 246 |

[Open in a new tab](table/T5/)

*\*Total protein coding genes as per annotated genome.*

Based on the genome annotation, genes for nitrogen metabolism were predicted in the genome of the ISS strain IF7SW-B2T. Most of the subsystem features aligned with the ammonia assimilation pathway (11 genes), which is a preferred nitrogen source for the bacteria ([Leigh and Dodsworth, 2007](#B30)). In addition, metabolic factors similar to high-affinity phosphate transporter and control of Pho regulon were also identified in the ISS strain IF7SW-B2T ([Wanner, 1993](#B47), [1996](#B48)). Interestingly, a higher number of stress tolerance genes, especially the oxidative stress response factors, were observed in the ISS strain IF7SW-B2T when compared with other novel species isolated from the ISS; *Methylobacterium* sp. IF7SW-B2T exhibited 58 features, whereas 36 features were identified in *Solibacillus kalamii* ([Seuylemezian et al., 2017](#B41)) and 18 features were identified in *Kalamiella piersonii* ([Singh et al., 2019](#B42)). The results obtained agree with the previous reports that showed altered regulation of the stress response factors in microorganisms, in the presence of microgravity conditions ([Orsini et al., 2017](#B33); [Aunins et al., 2018](#B3)). Further studies on the role of oxidative stress in species selection are warranted. The WGS assembly of these three ISS strains reported here will enable the comparative genomic characterization of ISS isolates with Earth counterparts in future studies. This will further aid in the identification of genetic determinants that might potentially be responsible for promoting plant growth under microgravity conditions and contribute to the development of self-sustainable plant crops for long-term space missions in future.

### Genes Essential for Interaction With Plants in the ISS Strain

A thorough genomic analysis of the ISS strain IF7SW-B2T revealed the presence of genes that have been involved in promoting plant growth. The isopentenyl tRNA transferase (*miaA*) essential for cytokinin production reported in *M. aquaticum* strain 22A ([Tani et al., 2015](#B43)) was also found in genome of the ISS strain IF7SW-B2T with high similarity. The product of the *miaA* gene was reported to be responsible for isopentenylation of a specific adenine in some tRNAs and confirmed the secretion of zeatin originated from tRNA in *M. extorquens* ([Koenig et al., 2002](#B22)). Furthermore, multiple components of the cobalamin synthesis pathway, such as cobalamin biosynthesis protein BluB, L-threonine 3-O-phosphate decarboxylase (EC 4.1.1.81), adenosylcobinamide-phosphate guanylyltransferase (EC 2.7.7.62), cobyric acid synthase (EC 6.3.5.10), nicotinate-nucleotide—dimethylbenzimidazole phosphoribosyltransferase (EC 2.4.2.21), adenosylcobinamide-phosphate synthase (EC 6.3.1.10), cob(I)alamin adenosyltransferase. (EC 2.5.1.17), cobalamin synthase (EC 2.7.8.26), and adenosylcobinamide kinase (EC 2.7.1.156), were identified in genome of the ISS strain IF7SW-B2T. The metabolic pathway for cobalamin synthesis predicted in the ISS strain is presented ([Supplementary Figure 8](#FS1)). Supporting this prediction, previous study also reported that *Methylobacterium* strains harbor genes involved in the production of a variety of vitamins, such as cobalamin, biotin, thiamin, and riboflavin, indicating the potential of methylobacteria promoting algal growth ([Krug et al., 2020](#B23)). In addition, genes associated with siderophore production, i.e., ferric siderophore transport system, biopolymer transport protein ExbB, and multiple flagellar proteins, were identified in genome of the ISS strain IF7SW-B2T and are listed in the [Supplementary Figure 8](#FS1). Genes involved in iron acquisition and metabolism in which microalgae benefit from bacterial siderophores have been reported previously in *Methylobacterium* spp. ([Krug et al., 2020](#B23)). In the “carbon for iron mutualism” concept, algae assimilated iron complexed in bacterial siderophores and in return provided essential dissolved organic matter for the bacteria ([Amin et al., 2015](#B1)). Similar studies are warranted to confirm the plant-growth promoting activities in the IF7SW-B2T ISS strain.

In summary, the phylogenetic and genetic distinctiveness and differential phenotypic properties were sufficient to categorize these three ISS strains as members of a species distinct from other recognized *Methylobacterium* species. Therefore, on the basis of the data presented, strains IF7SW-B2T, IIF1SW-B5, and IIF4SW-B5 represent a novel species of the genus *Methylobacterium*, for which the name *Methylobacterium ajmalii* sp. nov. is proposed. The type strain is IF7SW-B2T (NRRL B-65601T and LMG 32165T).

### Description of *Methylobacterium ajmalii* sp. nov.

*Methylobacterium ajmalii* (aj.ma’li.i. N.L. gen. n. *ajmalii* named after Ajmal Khan, a renowned Indian scientist on biodiversity). Cells are Gram-stain-negative, aerobic, and motile rods showing oxidase- and catalase-positive reactions. Cells are 1.6–1.8 μm wide and 2.2–3.2 μm long. Colonies on R2A agar are reddish pink–pigmented, circular, convex, and smooth, with a diameter of approximately 0.6–1.0 mm after 3 days of incubation on R2A agar. Growth occurs at 25–30°C (optimum, 30°C), at pH 6.0–8.0 (optimum, pH 7.0) and in the presence of 0–1.0% (w/v) NaCl (optimum, 0%). In API ZYM tests, the strain is positive for Alkaline phosphatase, Esterase (C4), Esterase lipase (C8), Leucine arylamidase, Trypsin, Acid phosphatase, and Naphthol-AS-BI-phosphohydrolase, but negative for other enzyme activities. Cells utilize Adipic acid, D-glucose, D-maltose, D-mannitol, D-mannose, L-arabinose, Malic acid, N-acetyl-glucosamine, Potassium gluconate, and Trisodium citrate for growth, but not other substrates in API 20NE. Cells are capable of weakly fermenting inulin and D-melezitose as observed in API 50 CH. Ubiquinone Q-10 is the predominant respiratory isoprenoid quinone. The major fatty acid is summed feature 8 (comprising C18:1 ω7*c* and/or C18:1 ω6*c*). The major polar lipids are diphosphatidylglycerol, phosphatidylethanolamine, phosphatidylcholine, and phosphatidylglycerol. The genomic DNA G + C content of the type strain is 71.07 mol%.

The type strain IF7SW-B2T is isolated from the International Space Station.

## Data Availability Statement

The 16S rRNA gene sequences of *Methylobacterium* sp. IIF1SW-B5, and *Methylobacterium* sp. IIF4SW-B5 are submitted under accession numbers [KY218843](https://ncbi.nlm.nih.gov/nucleotide/KY218843) and [KY218865](https://ncbi.nlm.nih.gov/nucleotide/KY218865), respectively. The WGS and the raw data deposited under BioProject accession number [PRJNA634337](https://ncbi.nlm.nih.gov/nucleotide/PRJNA634337). The WGS accession numbers are mentioned in [Table 1](#T1). The WGS was also deposited in GeneLab under GeneLab dataset (GLDS-300; <https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS339> 300). The version described in this paper is the first version.

## Author Contributions

KV and NKS conceived and designed the experiments. SB, VE, and NKS performed the experiments. NKS analyzed the genomic data inclusive of *de novo* assemblies and verification, scaffold quality assessment, and annotation and generation of the whole genome and protein level alignment for positional description of organism in the tree of life. SB independently verified the genome assembly, generated alignments for all gene trees in the manuscript, and manually curated the tree images. KV and NKS isolated the type strain, and NKS carried out the phenotypic assays and biochemical characterization. KV compiled the contribution of write-ups from all authors associated with phenotype, NKS generated genotype and tables, and SB generated phylogenetic trees and figures. VE conducted the SB generated chemotaxonomic analysis. All authors read and approved the final manuscript. CEM generated the genomic library and sequenced the genomes of all strains. CCCW and ARP reviewed the manuscript.

## Conflict of Interest

The author(s) declare that there are no conflicts of interest. This manuscript was prepared as an account of work sponsored by NASA, an agency of the US Government. The US Government, NASA, California Institute of Technology, Jet Propulsion Laboratory, and their employees make no warranty, expressed or implied, or assume any liability or responsibility for the accuracy, completeness, or usefulness of information, apparatus, product, or process disclosed in this manuscript, or represents that its use would not infringe upon privately held rights. The use of, and references to any commercial product, process, or service does not necessarily constitute or imply endorsement, recommendation, or favoring by the U.S. Government, NASA, California Institute of Technology, or Jet Propulsion Laboratory. Views and opinions presented herein by the authors of this manuscript do not necessarily reflect those of the U.S. Government, NASA, California Institute of Technology, or Jet Propulsion Laboratory, and shall not be used for advertisements or product endorsements.

## Acknowledgments

The research described in this manuscript was performed at the Jet Propulsion Laboratory, California Institute of Technology under a contract with NASA and University of Southern California. We would like to thank Aleksandra Checinska-Sielaff for isolating the strain. We thank astronauts Captain Terry Virts for collecting samples aboard the ISS and the Implementation Team at NASA Ames Research Center (Fathi Karouia) for coordinating this effort. We also thank Ryan Kemp (Zymo Corp.) for extracting DNA and Dan Butler (Cornell Medicine) for performing shotgun sequencing using the NovaSeq platform. We also acknowledged the Jet Propulsion Laboratory supercomputing facility staff, notably Narendra J. Patel (Jimmy) and Edward Villanueva, for their continuous support in providing the best possible infrastructure for BIG-DATA analysis. ©2021 California Institute of Technology. Government sponsorship acknowledged.

**Funding.** The research described in this manuscript was funded by a 2012 Space Biology NNH12ZTT001N Grant No. 19-12829-26 under Task Order NNN13D111T awarded to KV, and NASA’s 2018 Space Biology (ROSBio) NNH18ZTT001N-FG App B: Flight and Ground Space Biology Research Grant No. 80NSSC19K1501 awarded to CCCW.

1

<https://rast.nmpdr.org/>

2

<http://midi-inc.com>

## Supplementary Material

The Supplementary Material for this article can be found online at: <https://www.frontiersin.org/articles/10.3389/fmicb.2021.639396/full#supplementary-material>

[Click here for additional data file.](/articles/instance/8005752/bin/Data_Sheet_1.XLSX) (59.6KB, XLSX)

[Click here for additional data file.](/articles/instance/8005752/bin/Data_Sheet_2.pdf) (2.2MB, pdf)

[Click here for additional data file.](/articles/instance/8005752/bin/Data_Sheet_3.pdf) (314.3KB, pdf)

## References

1. Amin S. A., Hmelo L. R., van Tol H. M., Durham B. P., Carlson L. T., Heal K. R., et al. (2015). Interaction and signalling between a cosmopolitan phytoplankton and associated bacteria.
   *Nature*
   522
   98–101. 10.1038/nature14488
    [[DOI](https://doi.org/10.1038/nature14488)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26017307/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Interaction%20and%20signalling%20between%20a%20cosmopolitan%20phytoplankton%20and%20associated%20bacteria.&author=S.%20A.%20Amin&author=L.%20R.%20Hmelo&author=H.%20M.%20van%20Tol&author=B.%20P.%20Durham&author=L.%20T.%20Carlson&volume=522&publication_year=2015&pages=98-101&pmid=26017307&doi=10.1038/nature14488&)]
2. Auch A. F., von Jan M., Klenk H.-P., Göker M. (2010). Digital DNA-DNA hybridization for microbial species delineation by means of genome-to-genome sequence comparison.
   *Standards Genom. Sci.*
   2
   117–134. 10.4056/sigs.531120
    [[DOI](https://doi.org/10.4056/sigs.531120)] [[PMC free article](/articles/PMC3035253/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/21304684/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Standards%20Genom.%20Sci.&title=Digital%20DNA-DNA%20hybridization%20for%20microbial%20species%20delineation%20by%20means%20of%20genome-to-genome%20sequence%20comparison.&author=A.%20F.%20Auch&author=M.%20von%20Jan&author=H.-P.%20Klenk&author=M.%20G%C3%B6ker&volume=2&publication_year=2010&pages=117-134&pmid=21304684&doi=10.4056/sigs.531120&)]
3. Aunins T. R., Erickson K. E., Prasad N., Levy S. E., Jones A., Shrestha S., et al. (2018). Spaceflight modifies *Escherichia coli* gene expression in response to antibiotic exposure and reveals role of oxidative stress response.
   *Front. Microbiol.*
   9:310. 10.3389/fmicb.2018.00310
    [[DOI](https://doi.org/10.3389/fmicb.2018.00310)] [[PMC free article](/articles/PMC5865062/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29615983/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&title=Spaceflight%20modifies%20Escherichia%20coli%20gene%20expression%20in%20response%20to%20antibiotic%20exposure%20and%20reveals%20role%20of%20oxidative%20stress%20response.&author=T.%20R.%20Aunins&author=K.%20E.%20Erickson&author=N.%20Prasad&author=S.%20E.%20Levy&author=A.%20Jones&volume=9&issue=310&publication_year=2018&pmid=29615983&doi=10.3389/fmicb.2018.00310&)]
4. Aziz R. K., Bartels D., Best A. A., DeJongh M., Disz T., Edwards R. A., et al. (2008). The RAST server: rapid annotations using subsystems technology.
   *BMC Genom.*
   9:75. 10.1186/1471-2164-9-75
    [[DOI](https://doi.org/10.1186/1471-2164-9-75)] [[PMC free article](/articles/PMC2265698/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18261238/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&title=The%20RAST%20server:%20rapid%20annotations%20using%20subsystems%20technology.&author=R.%20K.%20Aziz&author=D.%20Bartels&author=A.%20A.%20Best&author=M.%20DeJongh&author=T.%20Disz&volume=9&issue=75&publication_year=2008&pmid=18261238&doi=10.1186/1471-2164-9-75&)]
5. Bankevich A., Nurk S., Antipov D., Gurevich A. A., Dvorkin M., Kulikov A. S., et al. (2012). SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing.
   *J. Comput. Biol.*
   19
   455–477. 10.1089/cmb.2012.0021
    [[DOI](https://doi.org/10.1089/cmb.2012.0021)] [[PMC free article](/articles/PMC3342519/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Biol.&title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing.&author=A.%20Bankevich&author=S.%20Nurk&author=D.%20Antipov&author=A.%20A.%20Gurevich&author=M.%20Dvorkin&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)]
6. Brettin T., Davis J. J., Disz T., Edwards R. A., Gerdes S., Olsen G. J., et al. (2015). RASTtk: a modular and extensible implementation of the RAST algorithm for building custom annotation pipelines and annotating batches of genomes.
   *Sci. Rep.*
   5:8365. [[DOI](https://doi.org/10.1038/srep08365)] [[PMC free article](/articles/PMC4322359/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25666585/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=RASTtk:%20a%20modular%20and%20extensible%20implementation%20of%20the%20RAST%20algorithm%20for%20building%20custom%20annotation%20pipelines%20and%20annotating%20batches%20of%20genomes.&author=T.%20Brettin&author=J.%20J.%20Davis&author=T.%20Disz&author=R.%20A.%20Edwards&author=S.%20Gerdes&volume=5&issue=8365&publication_year=2015&pmid=25666585&doi=10.1038/srep08365&)]
7. Chaudhry V., Baindara P., Pal V. K., Chawla N., Patil P. B., Korpole S. (2016). Methylobacterium indicum sp. nov., a facultative methylotrophic bacterium isolated from rice seed.
   *Syst. Appl. Microbiol.*
   39
   25–32. 10.1016/j.syapm.2015.12.006
    [[DOI](https://doi.org/10.1016/j.syapm.2015.12.006)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26776106/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Syst.%20Appl.%20Microbiol.&title=Methylobacterium%20indicum%20sp.%20nov.,%20a%20facultative%20methylotrophic%20bacterium%20isolated%20from%20rice%20seed.&author=V.%20Chaudhry&author=P.%20Baindara&author=V.%20K.%20Pal&author=N.%20Chawla&author=P.%20B.%20Patil&volume=39&publication_year=2016&pages=25-32&pmid=26776106&doi=10.1016/j.syapm.2015.12.006&)]
8. Checinska A., Probst A. J., Vaishampayan P., White J. R., Kumar D., Stepanov V. G., et al. (2015). Microbiomes of the dust particles collected from the international space station and spacecraft assembly facilities.
   *Microbiome*
   3:50. [[DOI](https://doi.org/10.1186/s40168-015-0116-3)] [[PMC free article](/articles/PMC4624184/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26502721/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Microbiomes%20of%20the%20dust%20particles%20collected%20from%20the%20international%20space%20station%20and%20spacecraft%20assembly%20facilities.&author=A.%20Checinska&author=A.%20J.%20Probst&author=P.%20Vaishampayan&author=J.%20R.%20White&author=D.%20Kumar&volume=3&issue=50&publication_year=2015&pmid=26502721&doi=10.1186/s40168-015-0116-3&)]
9. Checinska Sielaff A., Urbaniak C., Mohan G. B. M., Stepanov V. G., Tran Q., Wood J. M., et al. (2019). Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces.
   *Microbiome*
   7:50. [[DOI](https://doi.org/10.1186/s40168-019-0666-x)] [[PMC free article](/articles/PMC6452512/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces.&author=A.%20Checinska%20Sielaff&author=C.%20Urbaniak&author=G.%20B.%20M.%20Mohan&author=V.%20G.%20Stepanov&author=Q.%20Tran&volume=7&issue=50&publication_year=2019&pmid=30955503&doi=10.1186/s40168-019-0666-x&)]
10. Gallego V., Garcia M. T., Ventosa A. (2005a). Methylobacterium hispanicum sp. nov. and *Methylobacterium aquaticum* sp. nov., isolated from drinking water.
    *Int. J. Syst. Evol. Microbiol.*
    55
    281–287. 10.1099/ijs.0.63319-0
     [[DOI](https://doi.org/10.1099/ijs.0.63319-0)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/15653888/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Methylobacterium%20hispanicum%20sp.%20nov.%20and%20Methylobacterium%20aquaticum%20sp.%20nov.,%20isolated%20from%20drinking%20water.&author=V.%20Gallego&author=M.%20T.%20Garcia&author=A.%20Ventosa&volume=55&publication_year=2005a&pages=281-287&pmid=15653888&doi=10.1099/ijs.0.63319-0&)]
11. Gallego V., Garcia M. T., Ventosa A. (2005b). Methylobacterium variabile sp. nov., a methylotrophic bacterium isolated from an aquatic environment.
    *Int. J. Syst. Evol. Microbiol.*
    55
    1429–1433. 10.1099/ijs.0.63597-0
     [[DOI](https://doi.org/10.1099/ijs.0.63597-0)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16014462/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Methylobacterium%20variabile%20sp.%20nov.,%20a%20methylotrophic%20bacterium%20isolated%20from%20an%20aquatic%20environment.&author=V.%20Gallego&author=M.%20T.%20Garcia&author=A.%20Ventosa&volume=55&publication_year=2005b&pages=1429-1433&pmid=16014462&doi=10.1099/ijs.0.63597-0&)]
12. Green P. N., Ardley J. K. (2018). Review of the genus Methylobacterium and closely related organisms: a proposal that some Methylobacterium species be reclassified into a new genus. methylorubrum gen. nov.
    *Int. J. Syst. Evol. Microbiol.*
    68
    2727–2748. 10.1099/ijsem.0.002856
     [[DOI](https://doi.org/10.1099/ijsem.0.002856)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30024371/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Review%20of%20the%20genus%20Methylobacterium%20and%20closely%20related%20organisms:%20a%20proposal%20that%20some%20Methylobacterium%20species%20be%20reclassified%20into%20a%20new%20genus.%20methylorubrum%20gen.%20nov.&author=P.%20N.%20Green&author=J.%20K.%20Ardley&volume=68&publication_year=2018&pages=2727-2748&pmid=30024371&doi=10.1099/ijsem.0.002856&)]
13. Green P. N., Bousfield I. J. (1982). A taxonomic study of some Gram-negative facultatively methylotrophic bacteria.
    *Microbiology*
    128
    623–638. 10.1099/00221287-128-3-623 [[DOI](https://doi.org/10.1099/00221287-128-3-623)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiology&title=A%20taxonomic%20study%20of%20some%20Gram-negative%20facultatively%20methylotrophic%20bacteria.&author=P.%20N.%20Green&author=I.%20J.%20Bousfield&volume=128&publication_year=1982&pages=623-638&doi=10.1099/00221287-128-3-623&)]
14. Grossi C. E. M., Fantino E., Serral F., Zawoznik M. S., Fernandez, Do Porto D. A., et al. (2020). Methylobacterium sp. 2A is a plant growth-promoting rhizobacteria that has the potential to improve potato crop yield under adverse conditions.
    *Front. Plant Sci.*
    11:71. 10.3389/fpls.2020.00071
     [[DOI](https://doi.org/10.3389/fpls.2020.00071)] [[PMC free article](/articles/PMC7038796/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32127795/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&title=Methylobacterium%20sp.%202A%20is%20a%20plant%20growth-promoting%20rhizobacteria%20that%20has%20the%20potential%20to%20improve%20potato%20crop%20yield%20under%20adverse%20conditions.&author=C.%20E.%20M.%20Grossi&author=E.%20Fantino&author=F.%20Serral&author=M.%20S.%20Zawoznik&author=D.%20A.%20Fernandez,%20Do%20Porto&volume=11&issue=71&publication_year=2020&pmid=32127795&doi=10.3389/fpls.2020.00071&)]
15. Haft D. H., DiCuccio M., Badretdin A., Brover V., Chetvernin V., O’Neill K., et al. (2018). RefSeq: an update on prokaryotic genome annotation and curation.
    *Nucleic Acids Res.*
    46
    D851–D860. [[DOI](https://doi.org/10.1093/nar/gkx1068)] [[PMC free article](/articles/PMC5753331/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29112715/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=RefSeq:%20an%20update%20on%20prokaryotic%20genome%20annotation%20and%20curation.&author=D.%20H.%20Haft&author=M.%20DiCuccio&author=A.%20Badretdin&author=V.%20Brover&author=V.%20Chetvernin&volume=46&publication_year=2018&pages=D851-D860&pmid=29112715&doi=10.1093/nar/gkx1068&)]
16. Jain C., Rodriguez R. L., Phillippy A. M., Konstantinidis K. T., Aluru S. (2018). High throughput ANI analysis of 90K prokaryotic genomes reveals clear species boundaries.
    *Nat. Commun.*
    9:5114. [[DOI](https://doi.org/10.1038/s41467-018-07641-9)] [[PMC free article](/articles/PMC6269478/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30504855/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=High%20throughput%20ANI%20analysis%20of%2090K%20prokaryotic%20genomes%20reveals%20clear%20species%20boundaries.&author=C.%20Jain&author=R.%20L.%20Rodriguez&author=A.%20M.%20Phillippy&author=K.%20T.%20Konstantinidis&author=S.%20Aluru&volume=9&issue=5114&publication_year=2018&pmid=30504855&doi=10.1038/s41467-018-07641-9&)]
17. Jones D. M. (1981). Manual of methods for general bacteriology.
    *J. Clin. Pathol.*
    34:1069. 10.1136/jcp.34.9.1069-c [[DOI](https://doi.org/10.1136/jcp.34.9.1069-c)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pathol.&title=Manual%20of%20methods%20for%20general%20bacteriology.&author=D.%20M.%20Jones&volume=34&issue=1069&publication_year=1981&doi=10.1136/jcp.34.9.1069-c&)]
18. Jurtshuk P., Jr., McQuitty D. N. (1976). Use of a quantitative oxidase test for characterizing oxidative metabolism in bacteria.
    *Appl. Environ. Microbiol.*
    31
    668–679. 10.1128/aem.31.5.668-679.1976
     [[DOI](https://doi.org/10.1128/aem.31.5.668-679.1976)] [[PMC free article](/articles/PMC291174/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/1275489/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=Use%20of%20a%20quantitative%20oxidase%20test%20for%20characterizing%20oxidative%20metabolism%20in%20bacteria.&author=P.%20Jurtshuk&author=D.%20N.%20McQuitty&volume=31&publication_year=1976&pages=668-679&pmid=1275489&doi=10.1128/aem.31.5.668-679.1976&)]
19. Kang Y. S., Kim J., Shin H. D., Nam Y. D., Bae J. W., Jeon C. O., et al. (2007). Methylobacterium platani sp. nov., isolated from a leaf of the tree Platanus orientalis.
    *Int. J. Syst. Evol. Microbiol.*
    57
    2849–2853. 10.1099/ijs.0.65262-0
     [[DOI](https://doi.org/10.1099/ijs.0.65262-0)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/18048737/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Methylobacterium%20platani%20sp.%20nov.,%20isolated%20from%20a%20leaf%20of%20the%20tree%20Platanus%20orientalis.&author=Y.%20S.%20Kang&author=J.%20Kim&author=H.%20D.%20Shin&author=Y.%20D.%20Nam&author=J.%20W.%20Bae&volume=57&publication_year=2007&pages=2849-2853&pmid=18048737&doi=10.1099/ijs.0.65262-0&)]
20. Kelly D. P., McDonald I. R., Wood A. P. (2014). “The family methylobacteriaceae,” in *The Prokaryotes- Alphaproteobacteria and Betaproteobacteria*, eds
    Rosenberg E., DeLong E. F., Lory S., Stackebrandt E., Thompson F., (Berlin: Springer; ), 313–340. 10.1007/978-3-642-30197-1\_256 [[DOI](https://doi.org/10.1007/978-3-642-30197-1_256)] [[Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Prokaryotes-%20Alphaproteobacteria%20and%20Betaproteobacteria&author=D.%20P.%20Kelly&author=I.%20R.%20McDonald&author=A.%20P.%20Wood&publication_year=2014&)]
21. Kim J., Chhetri G., Kim I., Kim H., Kim M. K., Seo T. (2019). Methylobacterium terrae sp. nov., a radiation-resistant bacterium isolated from gamma ray-irradiated soil.
    *J. Microbiol.*
    57
    959–966. 10.1007/s12275-019-9007-9
     [[DOI](https://doi.org/10.1007/s12275-019-9007-9)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31463788/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Microbiol.&title=Methylobacterium%20terrae%20sp.%20nov.,%20a%20radiation-resistant%20bacterium%20isolated%20from%20gamma%20ray-irradiated%20soil.&author=J.%20Kim&author=G.%20Chhetri&author=I.%20Kim&author=H.%20Kim&author=M.%20K.%20Kim&volume=57&publication_year=2019&pages=959-966&pmid=31463788&doi=10.1007/s12275-019-9007-9&)]
22. Koenig R. L., Morris R. O., Polacco J. C. (2002). tRNA is the source of low-level trans-zeatin production in *Methylobacterium spp*.
    *J. Bacteriol.*
    184
    1832–1842. 10.1128/jb.184.7.1832-1842.2002
     [[DOI](https://doi.org/10.1128/jb.184.7.1832-1842.2002)] [[PMC free article](/articles/PMC134930/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/11889088/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=tRNA%20is%20the%20source%20of%20low-level%20trans-zeatin%20production%20in%20Methylobacterium%20spp.&author=R.%20L.%20Koenig&author=R.%20O.%20Morris&author=J.%20C.%20Polacco&volume=184&publication_year=2002&pages=1832-1842&pmid=11889088&doi=10.1128/jb.184.7.1832-1842.2002&)]
23. Krug L., Morauf C., Donat C., Muller H., Cernava T., Berg G. (2020). Plant growth-promoting methylobacteria selectively increase the biomass of biotechnologically relevant microalgae.
    *Front. Microbiol.*
    11:427. 10.3389/fmicb.2020.00427
     [[DOI](https://doi.org/10.3389/fmicb.2020.00427)] [[PMC free article](/articles/PMC7093331/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/32256478/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&title=Plant%20growth-promoting%20methylobacteria%20selectively%20increase%20the%20biomass%20of%20biotechnologically%20relevant%20microalgae.&author=L.%20Krug&author=C.%20Morauf&author=C.%20Donat&author=H.%20Muller&author=T.%20Cernava&volume=11&issue=427&publication_year=2020&pmid=32256478&doi=10.3389/fmicb.2020.00427&)]
24. Kumar M., Tomar R. S., Lade H., Paul D. (2016). Methylotrophic bacteria in sustainable agriculture.
    *World J. Microbiol. Biotechnol.*
    32:120. [[DOI](https://doi.org/10.1007/s11274-016-2074-8)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27263015/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J.%20Microbiol.%20Biotechnol.&title=Methylotrophic%20bacteria%20in%20sustainable%20agriculture.&author=M.%20Kumar&author=R.%20S.%20Tomar&author=H.%20Lade&author=D.%20Paul&volume=32&issue=120&publication_year=2016&pmid=27263015&doi=10.1007/s11274-016-2074-8&)]
25. Kumar S., Stecher G., Tamura K. (2016). MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets.
    *Mol. Biol. Evol.*
    33
    1870–1874. 10.1093/molbev/msw054
     [[DOI](https://doi.org/10.1093/molbev/msw054)] [[PMC free article](/articles/PMC8210823/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27004904/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&title=MEGA7:%20molecular%20evolutionary%20genetics%20analysis%20version%207.0%20for%20bigger%20datasets.&author=S.%20Kumar&author=G.%20Stecher&author=K.%20Tamura&volume=33&publication_year=2016&pages=1870-1874&pmid=27004904&doi=10.1093/molbev/msw054&)]
26. Kurtz S., Phillippy A., Delcher A. L., Smoot M., Shumway M., Antonescu C., et al. (2004). Versatile and open software for comparing large genomes.
    *Genome Biol.*
    5:R12. [[DOI](https://doi.org/10.1186/gb-2004-5-2-r12)] [[PMC free article](/articles/PMC395750/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/14759262/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=Versatile%20and%20open%20software%20for%20comparing%20large%20genomes.&author=S.%20Kurtz&author=A.%20Phillippy&author=A.%20L.%20Delcher&author=M.%20Smoot&author=M.%20Shumway&volume=5&issue=R12&publication_year=2004&pmid=14759262&doi=10.1186/gb-2004-5-2-r12&)]
27. Kwak M. J., Jeong H., Madhaiyan M., Lee Y., Sa T. M., Oh T. K., et al. (2014). Genome information of *Methylobacterium oryzae*, a plant-probiotic methylotroph in the phyllosphere.
    *PLoS One*
    9:e106704. 10.1371/journal.pone.0106704
     [[DOI](https://doi.org/10.1371/journal.pone.0106704)] [[PMC free article](/articles/PMC4161386/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25211235/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Genome%20information%20of%20Methylobacterium%20oryzae,%20a%20plant-probiotic%20methylotroph%20in%20the%20phyllosphere.&author=M.%20J.%20Kwak&author=H.%20Jeong&author=M.%20Madhaiyan&author=Y.%20Lee&author=T.%20M.%20Sa&volume=9&issue=e106704&publication_year=2014&pmid=25211235&doi=10.1371/journal.pone.0106704&)]
28. Lee M. D. (2019). GToTree: a user-friendly workflow for phylogenomics.
    *Bioinformatics*
    35
    4162–4164. 10.1093/bioinformatics/btz188
     [[DOI](https://doi.org/10.1093/bioinformatics/btz188)] [[PMC free article](/articles/PMC6792077/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30865266/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=GToTree:%20a%20user-friendly%20workflow%20for%20phylogenomics.&author=M.%20D.%20Lee&volume=35&publication_year=2019&pages=4162-4164&pmid=30865266&doi=10.1093/bioinformatics/btz188&)]
29. >Lee Y., Jeon C. O. (2018). *Methylobacterium frigidaeris* sp. nov., isolated from an air conditioning system.
    *Int. J. Syst. Evol. Microbiol.*
    68, 299–304. 10.1099/ijsem.0.002500
     [[DOI](https://doi.org/10.1099/ijsem.0.002500)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29160200/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Methylobacterium%20frigidaeris%20sp.%20nov.,%20isolated%20from%20an%20air%20conditioning%20system.&author=Y.%20>Lee&author=C.%20O.%20Jeon&volume=68&publication_year=2018&pages=299-304&pmid=29160200&doi=10.1099/ijsem.0.002500&)]
30. Leigh J. A., Dodsworth J. A. (2007). Nitrogen regulation in bacteria and archaea.
    *Annu. Rev. Microbiol.*
    61
    349–377. 10.1146/annurev.micro.61.080706.093409
     [[DOI](https://doi.org/10.1146/annurev.micro.61.080706.093409)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/17506680/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Microbiol.&title=Nitrogen%20regulation%20in%20bacteria%20and%20archaea.&author=J.%20A.%20Leigh&author=J.%20A.%20Dodsworth&volume=61&publication_year=2007&pages=349-377&pmid=17506680&doi=10.1146/annurev.micro.61.080706.093409&)]
31. Madhaiyan M., Suresh Reddy B. V., Anandham R., Senthilkumar M., Poonguzhali S., Sundaram S. P., et al. (2006). Plant growth-promoting methylobacterium induces defense responses in groundnut (*Arachis hypogaea L.*) compared with rot pathogens.
    *Curr. Microbiol.*
    53
    270–276. 10.1007/s00284-005-0452-9
     [[DOI](https://doi.org/10.1007/s00284-005-0452-9)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/16941245/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Microbiol.&title=Plant%20growth-promoting%20methylobacterium%20induces%20defense%20responses%20in%20groundnut%20(Arachis%20hypogaea%20L.)%20compared%20with%20rot%20pathogens.&author=M.%20Madhaiyan&author=B.%20V.%20Suresh%20Reddy&author=R.%20Anandham&author=M.%20Senthilkumar&author=S.%20Poonguzhali&volume=53&publication_year=2006&pages=270-276&pmid=16941245&doi=10.1007/s00284-005-0452-9&)]
32. Meier-Kolthoff J. P., Auch A. F., Klenk H. P., Goker M. (2013). Genome sequence-based species delimitation with confidence intervals and improved distance functions.
    *BMC Bioinform.*
    14:60. 10.1186/1471-2105-14-60
     [[DOI](https://doi.org/10.1186/1471-2105-14-60)] [[PMC free article](/articles/PMC3665452/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23432962/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinform.&title=Genome%20sequence-based%20species%20delimitation%20with%20confidence%20intervals%20and%20improved%20distance%20functions.&author=J.%20P.%20Meier-Kolthoff&author=A.%20F.%20Auch&author=H.%20P.%20Klenk&author=M.%20Goker&volume=14&issue=60&publication_year=2013&pmid=23432962&doi=10.1186/1471-2105-14-60&)]
33. Orsini S. S., Lewis A. M., Rice K. C. (2017). Investigation of simulated microgravity effects on Streptococcus mutans physiology and global gene expression.
    *NPJ Microgravity*
    3:4. [[DOI](https://doi.org/10.1038/s41526-016-0006-4)] [[PMC free article](/articles/PMC5460135/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28649626/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=NPJ%20Microgravity&title=Investigation%20of%20simulated%20microgravity%20effects%20on%20Streptococcus%20mutans%20physiology%20and%20global%20gene%20expression.&author=S.%20S.%20Orsini&author=A.%20M.%20Lewis&author=K.%20C.%20Rice&volume=3&issue=4&publication_year=2017&pmid=28649626&doi=10.1038/s41526-016-0006-4&)]
34. Overbeek R., Olson R., Pusch G. D., Olsen G. J., Davis J. J., Disz T., et al. (2014). The seed and the rapid annotation of microbial genomes using subsystems technology (RAST).
    *Nucleic Acids Res.*
    42
    D206–D214. [[DOI](https://doi.org/10.1093/nar/gkt1226)] [[PMC free article](/articles/PMC3965101/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/24293654/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=The%20seed%20and%20the%20rapid%20annotation%20of%20microbial%20genomes%20using%20subsystems%20technology%20(RAST).&author=R.%20Overbeek&author=R.%20Olson&author=G.%20D.%20Pusch&author=G.%20J.%20Olsen&author=J.%20J.%20Davis&volume=42&publication_year=2014&pages=D206-D214&pmid=24293654&doi=10.1093/nar/gkt1226&)]
35. Parasuraman P., Pattnaik S., Busi S. (2019). *Phyllosphere Microbiome: Functional Importance in Sustainable Agriculture, New and Future Developments in Microbial Biotechnology and Bioengineering.*
    Amsterdam: Elsevier, 135–148. [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Phyllosphere%20Microbiome:%20Functional%20Importance%20in%20Sustainable%20Agriculture,%20New%20and%20Future%20Developments%20in%20Microbial%20Biotechnology%20and%20Bioengineering.&author=P.%20Parasuraman&author=S.%20Pattnaik&author=S.%20Busi&publication_year=2019&)]
36. Park C., Lee Y. S., Park S. Y., Park W. (2018). Methylobacterium currus sp. nov., isolated from a car air conditioning system.
    *Int. J. Syst. Evol. Microbiol.*
    68
    3621–3626. 10.1099/ijsem.0.003045
     [[DOI](https://doi.org/10.1099/ijsem.0.003045)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/30252644/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Methylobacterium%20currus%20sp.%20nov.,%20isolated%20from%20a%20car%20air%20conditioning%20system.&author=C.%20Park&author=Y.%20S.%20Lee&author=S.%20Y.%20Park&author=W.%20Park&volume=68&publication_year=2018&pages=3621-3626&pmid=30252644&doi=10.1099/ijsem.0.003045&)]
37. Patel R. K., Jain M. (2012). NGS QC Toolkit: a toolkit for quality control of next generation sequencing data.
    *PLoS One*
    7:e30619. 10.1371/journal.pone.0030619
     [[DOI](https://doi.org/10.1371/journal.pone.0030619)] [[PMC free article](/articles/PMC3270013/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/22312429/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=NGS%20QC%20Toolkit:%20a%20toolkit%20for%20quality%20control%20of%20next%20generation%20sequencing%20data.&author=R.%20K.%20Patel&author=M.%20Jain&volume=7&issue=e30619&publication_year=2012&pmid=22312429&doi=10.1371/journal.pone.0030619&)]
38. Patt T. E., Cole G. C., Hanson R. S. (1976). Methylobacterium, a new genus of facultatively methylotrophic bacteria.
    *Int. J. Syst. Bacteriol.*
    26
    226–229. 10.1099/00207713-26-2-226 [[DOI](https://doi.org/10.1099/00207713-26-2-226)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Bacteriol.&title=Methylobacterium,%20a%20new%20genus%20of%20facultatively%20methylotrophic%20bacteria.&author=T.%20E.%20Patt&author=G.%20C.%20Cole&author=R.%20S.%20Hanson&volume=26&publication_year=1976&pages=226-229&doi=10.1099/00207713-26-2-226&)]
39. Ramaprasad E. V. V., Mahidhara G., Sasikala C., Ramana C. V. (2018). Rhodococcus electrodiphilus sp. nov., a marine electro active actinobacterium isolated from coral reef.
    *Int. J. Syst. Evol. Microbiol.*
    68
    2644–2649. 10.1099/ijsem.0.002895
     [[DOI](https://doi.org/10.1099/ijsem.0.002895)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/29957174/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Rhodococcus%20electrodiphilus%20sp.%20nov.,%20a%20marine%20electro%20active%20actinobacterium%20isolated%20from%20coral%20reef.&author=E.%20V.%20V.%20Ramaprasad&author=G.%20Mahidhara&author=C.%20Sasikala&author=C.%20V.%20Ramana&volume=68&publication_year=2018&pages=2644-2649&pmid=29957174&doi=10.1099/ijsem.0.002895&)]
40. Ramaprasad E. V. V., Sasikala C., Ramana C. V. (2015). Flectobacillus rhizosphaerae sp. nov., isolated from the rhizosphere soil of *Oryza sativa* (L.), and emended description of the genus *Flectobaicillus*.
    *Int. J. Syst. Evol. Microbiol.*
    65
    3451–3456. 10.1099/ijsem.0.000432
     [[DOI](https://doi.org/10.1099/ijsem.0.000432)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/26297007/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Flectobacillus%20rhizosphaerae%20sp.%20nov.,%20isolated%20from%20the%20rhizosphere%20soil%20of%20Oryza%20sativa%20(L.),%20and%20emended%20description%20of%20the%20genus%20Flectobaicillus.&author=E.%20V.%20V.%20Ramaprasad&author=C.%20Sasikala&author=C.%20V.%20Ramana&volume=65&publication_year=2015&pages=3451-3456&pmid=26297007&doi=10.1099/ijsem.0.000432&)]
41. Seuylemezian A., Singh N. K., Vaishampayan P., Venkateswaran K. (2017). Draft genome sequence of solibacillus kalamii, isolated from an air filter aboard the international space station.
    *Genome Announc.*
    5:e00696-17. [[DOI](https://doi.org/10.1128/genomeA.00696-17)] [[PMC free article](/articles/PMC5578834/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28860236/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Announc.&title=Draft%20genome%20sequence%20of%20solibacillus%20kalamii,%20isolated%20from%20an%20air%20filter%20aboard%20the%20international%20space%20station.&author=A.%20Seuylemezian&author=N.%20K.%20Singh&author=P.%20Vaishampayan&author=K.%20Venkateswaran&volume=5&publication_year=2017&pmid=28860236&doi=10.1128/genomeA.00696-17&)]
42. Singh N. K., Wood J. M., Mhatre S. S., Venkateswaran K. (2019). Metagenome to phenome approach enables isolation and genomics characterization of *Kalamiella piersonii* gen. nov., sp. nov. from the international space station.
    *Appl. Microbiol. Biotechnol.*
    103
    4483–4497. [[DOI](https://doi.org/10.1007/s00253-019-09813-z)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/31011775/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&title=Metagenome%20to%20phenome%20approach%20enables%20isolation%20and%20genomics%20characterization%20of%20Kalamiella%20piersonii%20gen.%20nov.,%20sp.%20nov.%20from%20the%20international%20space%20station.&author=N.%20K.%20Singh&author=J.%20M.%20Wood&author=S.%20S.%20Mhatre&author=K.%20Venkateswaran&volume=103&publication_year=2019&pages=4483-4497&pmid=31011775&doi=10.1007/s00253-019-09813-z&)]
43. Tani A., Ogura Y., Hayashi T., Kimbara K. (2015). Complete genome sequence of methylobacterium aquaticum strain 22a, isolated from *Racomitrium japonicum* moss.
    *Genome Announc.*
    3:e00266-15. [[DOI](https://doi.org/10.1128/genomeA.00266-15)] [[PMC free article](/articles/PMC4392154/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/25858842/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Announc.&title=Complete%20genome%20sequence%20of%20methylobacterium%20aquaticum%20strain%2022a,%20isolated%20from%20Racomitrium%20japonicum%20moss.&author=A.%20Tani&author=Y.%20Ogura&author=T.%20Hayashi&author=K.%20Kimbara&volume=3&publication_year=2015&pmid=25858842&doi=10.1128/genomeA.00266-15&)]
44. Tatusova T., DiCuccio M., Badretdin A., Chetvernin V., Nawrocki E. P., Zaslavsky L., et al. (2016). NCBI prokaryotic genome annotation pipeline.
    *Nucleic Acids Res.*
    44
    6614–6624. [[DOI](https://doi.org/10.1093/nar/gkw569)] [[PMC free article](/articles/PMC5001611/)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/27342282/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=NCBI%20prokaryotic%20genome%20annotation%20pipeline.&author=T.%20Tatusova&author=M.%20DiCuccio&author=A.%20Badretdin&author=V.%20Chetvernin&author=E.%20P.%20Nawrocki&volume=44&publication_year=2016&pages=6614-6624&pmid=27342282&doi=10.1093/nar/gkw569&)]
45. Tindall B., Sikorski J., Smibert R., Krieg N. (2007). “Phenotypic characterization and the principles of comparative systematics,” in *Methods for General and Molecular Microbiology*, eds
    Reddy C., Beveridge T., Breznak J., Marzluf G., Schmidt T., Snyder L., (Washington, DC: ASM Press; ), 330–393. [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Methods%20for%20General%20and%20Molecular%20Microbiology&author=B.%20Tindall&author=J.%20Sikorski&author=R.%20Smibert&author=N.%20Krieg&publication_year=2007&)]
46. Veyisoglu A., Camas M., Tatar D., Guven K., Sazak A., Sahin N. (2013). Methylobacterium tarhaniae sp. nov., isolated from arid soil.
    *Int. J. Syst. Evol. Microbiol.*
    63
    2823–2828. [[DOI](https://doi.org/10.1099/ijs.0.049551-0)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/23315404/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Methylobacterium%20tarhaniae%20sp.%20nov.,%20isolated%20from%20arid%20soil.&author=A.%20Veyisoglu&author=M.%20Camas&author=D.%20Tatar&author=K.%20Guven&author=A.%20Sazak&volume=63&publication_year=2013&pages=2823-2828&pmid=23315404&doi=10.1099/ijs.0.049551-0&)]
47. Wanner B. L. (1993). Gene regulation by phosphate in enteric bacteria.
    *J. Cell. Biochem.*
    51
    47–54. [[DOI](https://doi.org/10.1002/jcb.240510110)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/8432742/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell.%20Biochem.&title=Gene%20regulation%20by%20phosphate%20in%20enteric%20bacteria.&author=B.%20L.%20Wanner&volume=51&publication_year=1993&pages=47-54&pmid=8432742&doi=10.1002/jcb.240510110&)]
48. Wanner B. L. (1996). “Phosphorus assimilation and control of the phosphate regulon,” in *Escherichia coli and Salmonella: cellular and molecular biology*, ed.
    Neidhardt F. C., (Washington, DC: ASM press; ), 1357–1381. [[Google Scholar](https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20and%20Salmonella:%20cellular%20and%20molecular%20biology&author=B.%20L.%20Wanner&publication_year=1996&)]
49. Yoon S. H., Ha S. M., Lim J., Kwon S., Chun J. (2017). A large-scale evaluation of algorithms to calculate average nucleotide identity.
    *Antonie Van Leeuwenhoek*
    110
    1281–1286. [[DOI](https://doi.org/10.1007/s10482-017-0844-4)] [[PubMed](https://pubmed.ncbi.nlm.nih.gov/28204908/)] [[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antonie%20Van%20Leeuwenhoek&title=A%20large-scale%20evaluation%20of%20algorithms%20to%20calculate%20average%20nucleotide%20identity.&author=S.%20H.%20Yoon&author=S.%20M.%20Ha&author=J.%20Lim&author=S.%20Kwon&author=J.%20Chun&volume=110&publication_year=2017&pages=1281-1286&pmid=28204908&doi=10.1007/s10482-017-0844-4&)]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

[Click here for additional data file.](/articles/instance/8005752/bin/Data_Sheet_1.XLSX) (59.6KB, XLSX)

[Click here for additional data file.](/articles/instance/8005752/bin/Data_Sheet_2.pdf) (2.2MB, pdf)

[Click here for additional data file.](/articles/instance/8005752/bin/Data_Sheet_3.pdf) (314.3KB, pdf)

### Data Availability Statement

The 16S rRNA gene sequences of *Methylobacterium* sp. IIF1SW-B5, and *Methylobacterium* sp. IIF4SW-B5 are submitted under accession numbers [KY218843](https://ncbi.nlm.nih.gov/nucleotide/KY218843) and [KY218865](https://ncbi.nlm.nih.gov/nucleotide/KY218865), respectively. The WGS and the raw data deposited under BioProject accession number [PRJNA634337](https://ncbi.nlm.nih.gov/nucleotide/PRJNA634337). The WGS accession numbers are mentioned in [Table 1](#T1). The WGS was also deposited in GeneLab under GeneLab dataset (GLDS-300; <https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS339> 300). The version described in this paper is the first version.