<!DOCTYPE html>
<html><head><meta charset="utf-8"/><base href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8468678/"/><title>Simulated Microgravity Promotes Horizontal Gene Transfer of Antimicrobial Resistance Genes between Bacterial Genera in the Absence of Antibiotic Selective Pressure - PMC</title></head><body><main class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0" id="main-content">
<article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button aria-controls="journal_context_menu" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" type="button">Life (Basel)</button></div>. 2021 Sep 13;11(9):960. doi: <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3390/life11090960" rel="noopener noreferrer" target="_blank">10.3390/life11090960</a>
</div>
</section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Simulated Microgravity Promotes Horizontal Gene Transfer of Antimicrobial Resistance Genes between Bacterial Genera in the Absence of Antibiotic Selective Pressure</h1></hgroup><div class="cg p">
<a aria-describedby="id1" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Urbaniak%20C%22%5BAuthor%5D"><span class="name western">Camilla Urbaniak</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Camilla Urbaniak</span></h3>
<div class="p">
<sup>1</sup>NASA Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA 91109, USA; camilla.urbaniak@jpl.nasa.gov (C.U.); tgrams@ufl.edu (T.G.)</div>
<div class="p">
<sup>2</sup>ZIN Technologies Inc., Middleburg Heights, OH 44130, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Urbaniak%20C%22%5BAuthor%5D"><span class="name western">Camilla Urbaniak</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a aria-describedby="id2" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grams%20T%22%5BAuthor%5D"><span class="name western">Tristan Grams</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Tristan Grams</span></h3>
<div class="p">
<sup>1</sup>NASA Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA 91109, USA; camilla.urbaniak@jpl.nasa.gov (C.U.); tgrams@ufl.edu (T.G.)</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grams%20T%22%5BAuthor%5D"><span class="name western">Tristan Grams</span></a>
</div>
</div>
<sup>1</sup>, <a aria-describedby="id3" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D"><span class="name western">Christopher E Mason</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Christopher E Mason</span></h3>
<div class="p">
<sup>3</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY 10065, USA; chm2042@med.cornell.edu</div>
<div class="p">
<sup>4</sup>The World Quant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, NY 10065, USA</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mason%20CE%22%5BAuthor%5D"><span class="name western">Christopher E Mason</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a aria-describedby="id4" class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kasthuri Venkateswaran</span></h3>
<div class="p">
<sup>1</sup>NASA Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA 91109, USA; camilla.urbaniak@jpl.nasa.gov (C.U.); tgrams@ufl.edu (T.G.)</div>
<div class="p">Find articles by <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Venkateswaran%20K%22%5BAuthor%5D"><span class="name western">Kasthuri Venkateswaran</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<div class="cg p">Editors: <span class="name western">Fathi Karouia</span>, <span class="name western">Christopher H House</span>
</div>
<ul class="d-buttons inline-list">
<li><button aria-controls="aip_a" aria-expanded="false" class="d-button">Author information</button></li>
<li><button aria-controls="anp_a" aria-expanded="false" class="d-button">Article notes</button></li>
<li><button aria-controls="clp_a" aria-expanded="false" class="d-button">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div class="d-panel p" id="aip_a" style="display: none">
<div class="p" id="af1-life-11-00960">
<sup>1</sup>NASA Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA 91109, USA; camilla.urbaniak@jpl.nasa.gov (C.U.); tgrams@ufl.edu (T.G.)</div>
<div id="af2-life-11-00960">
<sup>2</sup>ZIN Technologies Inc., Middleburg Heights, OH 44130, USA</div>
<div id="af3-life-11-00960">
<sup>3</sup>Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY 10065, USA; chm2042@med.cornell.edu</div>
<div id="af4-life-11-00960">
<sup>4</sup>The World Quant Initiative for Quantitative Prediction, Weill Cornell Medicine, New York, NY 10065, USA</div>
<div class="author-notes p"><div class="fn" id="c1-life-11-00960">
<sup>*</sup><p class="display-inline">Correspondence: <span>kjvenkat@jpl.nasa.gov</span></p>
</div></div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Fathi Karouia</span></strong>: <span class="role">Academic Editor</span>
</div>
<div>
<strong class="contrib"><span class="name western">Christopher H House</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div class="d-panel p" id="anp_a" style="display: none"><div class="notes p"><section class="history" id="historyarticle-meta1"><p>Received 2021 Aug 24; Accepted 2021 Sep 9; Collection date 2021 Sep.</p></section></div></div>
<div class="d-panel p" id="clp_a" style="display: none">
<div>© 2021 by the authors.</div>
<p>Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://creativecommons.org/licenses/by/4.0/" rel="noopener noreferrer" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>).</p>
<div class="p"><a class="usa-link" href="/about/copyright/">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC8468678  PMID: <a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/34575109/">34575109</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Bacteria are able to adapt and survive in harsh and changing environments through many mechanisms, with one of them being horizontal gene transfer (HGT). This process is one of the leading culprits in the spread of antimicrobial resistance (AMR) within bacterial communities and could pose a significant health threat to astronauts if they fell ill, especially on long-duration space missions. In order to better understand the degree of HGT activity that could occur in space, biosafety level-2, donor and recipient bacteria were co-cultured under simulated microgravity (SMG) on Earth with concomitant 1G controls. Two AMR genes, <em>bla</em><sub>OXA-500</sub> and <em>ISAba1,</em> from the donor <em>Acinetobacter pittii,</em> were tracked in four recipient strains of <em>Staphylococcus aureus</em> (which did not harbor those genes) using polymerase chain reaction. All four <em>S. aureus</em> strains that were co-cultured with <em>A. pittii</em> under SMG had a significantly higher number of isolates that were now <em>bla</em><sub>OXA-500</sub>- and <em>ISAba1</em>-positive compared to growth at 1G. The acquisition of these genes by the recipient induced a phenotypic change, as these isolates were now resistant to oxacillin, which they were previously susceptible to. This is a novel study, presenting, for the first time, increased HGT activity under SMG and the potential impact of the space environment in promoting increased gene dissemination within bacterial communities.</p>
<section class="kwd-group" id="kwd-group1"><p><strong>Keywords:</strong> horizontal gene transfer, simulated microgravity, high aspect ratio vessel (HARV), antimicrobial resistance, space environment, <em>Staphylococcus aureus</em>, <em>Acinetobacter pittii</em></p></section></section><section id="sec1-life-11-00960"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Bacteria are able to adapt to a wide variety of conditions, as evidenced by their survival and growth in extreme environments, such as the arid Atacama desert [<a aria-describedby="B1-life-11-00960" class="usa-link" href="#B1-life-11-00960">1</a>], hypersaline salt mines and lakes [<a aria-describedby="B2-life-11-00960" class="usa-link" href="#B2-life-11-00960">2</a>,<a aria-describedby="B3-life-11-00960" class="usa-link" href="#B3-life-11-00960">3</a>], acidic rocks in the Yellowstone geothermal environment [<a aria-describedby="B4-life-11-00960" class="usa-link" href="#B4-life-11-00960">4</a>], and the multi-stressor environment of Antarctica [<a aria-describedby="B5-life-11-00960" class="usa-link" href="#B5-life-11-00960">5</a>]. Bacterial communities have also been found beyond the Earth’s surface on the International Space Station (ISS), a low-gravity, high-radiation, hermetically sealed built environment [<a aria-describedby="B6-life-11-00960" class="usa-link" href="#B6-life-11-00960">6</a>]. The microbial communities characterized here consist of environmental organisms, human-associated commensal bacteria and biosafety level 2 (BSL-2) microbes, resistant to various antibiotics, and carrying numerous anti-microbial resistance genes [<a aria-describedby="B7-life-11-00960" class="usa-link" href="#B7-life-11-00960">7</a>,<a aria-describedby="B8-life-11-00960" class="usa-link" href="#B8-life-11-00960">8</a>,<a aria-describedby="B9-life-11-00960" class="usa-link" href="#B9-life-11-00960">9</a>]. This resistance could potentially pose a serious threat to astronauts if they fell ill, especially on long-duration space missions, as medical attention would not be readily available.</p>
<p>Horizontal gene transfer (HGT) is the movement of genetic material (i.e., DNA) between microorganisms through either transformation, conjugation or transduction, and is considered to be central to microbial evolution [<a aria-describedby="B10-life-11-00960" class="usa-link" href="#B10-life-11-00960">10</a>,<a aria-describedby="B11-life-11-00960" class="usa-link" href="#B11-life-11-00960">11</a>]. In dynamic mixed communities, either human or environmental, HGT between species may confer an overall fitness advantage to the community, as the acquisition of new traits in individual cells within that community allows them to better adapt to changing conditions [<a aria-describedby="B12-life-11-00960" class="usa-link" href="#B12-life-11-00960">12</a>,<a aria-describedby="B13-life-11-00960" class="usa-link" href="#B13-life-11-00960">13</a>,<a aria-describedby="B14-life-11-00960" class="usa-link" href="#B14-life-11-00960">14</a>,<a aria-describedby="B15-life-11-00960" class="usa-link" href="#B15-life-11-00960">15</a>]. In other cases, HGT may have an unfavorable effect if it leads to an excessive accumulation of genes, or a neutral one [<a aria-describedby="B16-life-11-00960" class="usa-link" href="#B16-life-11-00960">16</a>], if there is no impact on the fitness of the organism [<a aria-describedby="B16-life-11-00960" class="usa-link" href="#B16-life-11-00960">16</a>].</p>
<p>In a recent study comparing bacteria from two extreme built environments, one on Earth (Concordia Research Station in Antarctica) and one in space (International Space Station), a comparison between isolates of the same species showed that those from the International Space Station (ISS) had more mobile genetic elements within their genomes than those from Concordia [<a aria-describedby="B17-life-11-00960" class="usa-link" href="#B17-life-11-00960">17</a>]. In a follow-up analysis conducted by our group using the ISS resistome and microbiome data presented by Urbaniak et al. 2018 [<a aria-describedby="B7-life-11-00960" class="usa-link" href="#B7-life-11-00960">7</a>], we performed a series of correlation analyses to determine which antimicrobial resistance (AMR) genes were associated with which species. Interestingly, no statistically significant correlations were found. These data, coupled by the Concordia/ISS study, led us to hypothesize that the stressors of the ISS, specifically microgravity, promote enhanced HGT between organisms, compared to what would be observed on Earth.</p>
<p>To address this hypothesis, co-cultures between donor and recipient strains were grown under simulated microgravity (SMG) (which we used as a proxy for µgravity conditions on the ISS) and normal gravity (1G), using the high-aspect-ratio vessel (HARV), with gene transfer tracked using polymerase chain reaction (PCR). The donor and recipient strains were <em>Acinetobacter pittii</em> (strain IIF1SW-P1) and <em>Staphylococcus aureus</em> (strains IF4SW, IIF6SW, IF7SW, IIF8SW), respectively. All strains were previously isolated from various locations on the ISS as part of the Microbial-1 Tracking study [<a aria-describedby="B8-life-11-00960" class="usa-link" href="#B8-life-11-00960">8</a>]. <em>A. pittii</em> IIF1SW-P1 was chosen as the donor, since it was previously shown by our group to be resistant to oxacillin and contained the chromosomally located <em>bla</em><sub>OXA-500</sub> gene within its genome [<a aria-describedby="B7-life-11-00960" class="usa-link" href="#B7-life-11-00960">7</a>], a beta-lactamase which alone can confer resistance to oxacillin [<a aria-describedby="B18-life-11-00960" class="usa-link" href="#B18-life-11-00960">18</a>]. The four <em>S. aureus</em> strains, IF4SW, IIF6SW, IF7SW, IIF8SW, were chosen as recipients because they were susceptible to oxacillin and harbored neither the <em>bla</em><sub>OXA-500</sub> gene nor any other beta-lactamase genes [<a aria-describedby="B7-life-11-00960" class="usa-link" href="#B7-life-11-00960">7</a>]. The donor, <em>A. pittii,</em> harbored <em>ISAba1</em>, a transposable element that can jump between segments of the genome and, in some cases, can act as a promoter region for <em>bla</em><sub>OXA</sub> genes, allowing for increased resistance to oxacillin and other beta-lactam antibiotics [<a aria-describedby="B19-life-11-00960" class="usa-link" href="#B19-life-11-00960">19</a>,<a aria-describedby="B20-life-11-00960" class="usa-link" href="#B20-life-11-00960">20</a>]. <em>ISAba1</em> was not found in either of the recipient <em>S. aureus</em> strains. HGT ability was, therefore, tracked between <em>A. pittii</em> and <em>S. aureus</em> using the AMR genes, <em>bla</em><sub>OXA-500</sub> and <em>ISAba1</em>.</p></section><section id="sec2-life-11-00960"><h2 class="pmc_sec_title">2. Methods</h2>
<section id="sec2dot1-life-11-00960"><h3 class="pmc_sec_title">2.1. Co-Culture Experiments</h3>
<p>All strains examined in this study were cultivated from the ISS environment collected during Flight 2 (May 2015) as per the published protocol [<a aria-describedby="B8-life-11-00960" class="usa-link" href="#B8-life-11-00960">8</a>]. <em>A. pittii</em> strain, IIF1SW-P1, was isolated at Location #1, the Port panel of the Cupola. The Cupola is a small module devoted to the observation of operations outside the ISS, such as robotic activities, spacecraft approaches, and extravehicular activities. <em>S. aureus</em> strain IF4SW was isolated from Location #4, the surface of the dining table. Even though the main function of the table was for dining, crew members also used the table for experimental work. <em>S. aureus</em> strain IIF6SW was cultured from the sample collected at Location #6, a stowage rack where experimental materials were stored for long duration. <em>S. aureus</em> strain IF7SW was isolated from Location #7, an overhead three-panel surface of the Materials Science Research Rack 1, which is used for basic material research in the microgravity environment of the ISS. <em>S. aureus</em> strain IIF8SW was retrieved from the samples at Location #8, an exterior aft wall of crew quarters. The crew quarters are a permanent personal space for crewmembers to sleep and perform personal recreation and communication, as well as provide on-orbit stowage of personal belongings.</p>
<p>Donor (<em>A. pittii</em> strain IIF1SW-P1) and recipient (<em>S. aureus</em> strains IF4SW-P1, IIF6SW-P1, IF7SW-P3, IIF8SW-P1) bacteria were streaked from −80 °C freezer stocks onto tryptic soy agar plates (Hardy Diagnostics, Santa Maria, CA, USA) and incubated overnight (~16 h) at 37 °C. A single colony from each plate was added to 10 mL of tryptic soy broth (TSB) in a 50 mL Falcon tube for <em>S. aureus</em>, and 8 mL of TSB in a snap cap tube for <em>A. pittii</em> and grown overnight (~20 h) in a shaking incubator at 85 rpm. A low speed was chosen, so as to not dislodge pili or flagella that could be used for HGT, but high enough that the bacteria did not settle out of culture over time. The overnight culture was centrifuged for 10 min at 3000× <em>g</em>, the supernatant decanted, and the pellet washed in 1× sterile phosphate-buffered saline (PBS). Pellets were then resuspended in 10 mL of 1× PBS. For <em>S. aureus</em>, 1 mL was then transferred to 9 mL of PBS and, for <em>A. pittii</em>, the tube was topped with 3 mL of PBS. The biomass of the cultures was measured with DENSICHEK<sup>®</sup> (bioMérieux, Marcy-l’Étoile, France) and adjusted with PBS to reach a concentration of 10<sup>8</sup> cfu/mL, which was then serially diluted to 10<sup>6</sup> cfu/mL. 100 µL of one <em>S. aureus</em> recipient strain was added to 10 mL of TSB and 100 µL of the donor <em>A. pittii</em> was added to the same 10 mL tube to set up the co-cultures. After gentle mixing, the complete 10 mL volume containing the donor and recipient was carefully added to HARVs to avoid any bubbles. HARVs were grown under SMG (vertical position) and 1G (horizontal position) at 33 rpm for 20 h at 37 °C.</p></section><section id="sec2dot2-life-11-00960"><h3 class="pmc_sec_title">2.2. Isolation of Colonies after HARV Growth</h3>
<p>After incubation, the liquid from each HARV vessel was collected and 100 µL, in duplicate, was plated on MSA (selective for <em>S. aureus</em>) (VWR, Radnor, PA, USA), MSA+ 4 µg/mL oxacillin sodium salt monohydrate (VWR, USA) (selective for <em>S. aureus</em> that had become resistant to oxacillin) and Leeds agar (selective for <em>A. pittii</em>) (VWR, USA). MSA and Leeds plates were incubated overnight at 37 °C and MSA+ oxacillin plates were incubated for 72 h at 37 °C. Colonies on MSA, MSA+ oxacillin, and Leeds plates were counted, and cfu/mL was calculated. Colonies that grew on MSA+ oxacillin plates were picked and placed in 250 µL 1× PBS and stored at 4 °C until DNA extraction. DNA was extracted using the ZymoBIOMICS DNA Microprep Kit (ZymoResearch, Irvine, CA, USA) following the manufacturer’s instructions.</p></section><section id="sec2dot3-life-11-00960"><h3 class="pmc_sec_title">2.3. Designing Primers</h3>
<p>Primers were designed using the <em>A. pittii</em> IIF1SW-P1 draft genome that we had previously published, in which <em>bla</em><sub>OXA-500</sub> gene resides on scaffold 5 and <em>ISAba1</em> gene on both scaffolds 44 and 70 [<a aria-describedby="B21-life-11-00960" class="usa-link" href="#B21-life-11-00960">21</a>]. Primers were designed using IDT DNA PrimerQuest for genes of interest. IDT DNA OligoAnalyzer was used to ensure no hairpins and correct PCR melting temperature. NCBI BLAST was used to ensure no off-target binding sites for strains used in this study. Amplification product of the primers for <em>bla</em><sub>OXA-500</sub> and the location within the genome is shown in <a class="usa-link" href="#app1-life-11-00960">Figure S3A</a> and for <em>ISAba1</em>, in <a class="usa-link" href="#app1-life-11-00960">Figure S3B in Supplementary Materials</a>. The product size of <em>bla</em><sub>OXA-500</sub> is 591 bp and for <em>ISAba1</em>, is 390 bp.</p></section><section id="sec2dot4-life-11-00960"><h3 class="pmc_sec_title">2.4. PCR</h3>
<p>Each 25-μL reaction consisted of 12.5 μL of GoTaq<sup>®</sup> Green Master Mix (Promega, Madison, WI, USA), 1 μL each of forward and reverse oligonucleotide primers (10 μM each), 5.5 µL of nuclease free water (Promega, USA) and 5 μL of template DNA. The following primer pairs were used:</p>
<ul class="list" style="list-style-type:disc">
<li><p><em>bla</em><sub>OXA-500</sub>_F (5′-CCGAGTTGTTCCAATCCCTTAT-3′) and</p></li>
<li><p><em>bla</em><sub>OXA-500</sub>_R (5′-ATATGTTCCCGCCTCTACCT-3′) to amplify <em>bla</em><sub>OXA-500</sub></p></li>
<li><p>and <em>ISAba1</em>_F (5′-ATGCAGCGCTTCTTTGCAGG-3′) and</p></li>
<li><p><em>ISAba1</em>_R (5′-AATGATTGGTGACAATGAAG-3′) to amplify <em>ISAba1</em>.</p></li>
</ul>
<p>The reaction conditions were as follows: 95 °C for 5 min, 35 cycles of 95 °C for 30 s, 55 °C for 30 s, 72 °C for 1 min and then a final extension of 72 °C for 5 min. Amplified PCR products were run on the E-Gel™ Power Snap Electrophoresis System (Thermo Fischer, Waltham, MA, USA, cat # G8300) using their pre-cast 1.2% SYBR Safe, E-gels (cat # G521801). Gels were run with a 1 Kb plus DNA ladder (cat # 10488090). Gel images were captured with the built-in camera. The positive control consisted of DNA isolated from the parental <em>A. pittii</em> strain and the negative controls were DNA isolated from the parental <em>S. aureus</em> strains. A no-template control (NTC) consisting of molecular-grade water instead of DNA was used.</p>
<p>For PCR reactions that were negative, a follow-up PCR was performed using the 16S rRNA gene to confirm that the DNA extraction procedure produced sufficient DNA. The following primers were used for 16S rRNA gene amplification: the forward primer, 27F (5′-AGA GTT TGA TCC TGG CTC AG-3′) and the reverse primer, 1492R (5′-GGT TAC CTT GTT ACG ACT T-3′). The PCR conditions were as follows: denaturation at 95 °C for 5 min, followed by 35 cycles consisting of denaturation at 95 °C for 50 s, annealing at 55 °C for 50 s, and extension at 72 °C for 1 min 30 s and finalized by extension at 72 °C for 10 min. PCR with the 16S rRNA gene showed that all our samples did indeed have the DNA product and that extraction was successful.</p></section><section id="sec2dot5-life-11-00960"><h3 class="pmc_sec_title">2.5. Calculating the Efficiency of Gene Transfer</h3>
<p>The number of colonies that had grown on MSA+ oxacillin and were positive for at least one gene (<em>bla</em><sub>OXA-500</sub> or <em>ISAba1</em>) was divided by 0.1 mL (the volume of the HARV culture used for plating). This number represents the theoretical concentration (cfu/mL) of cells in the culture that have acquired one of the genes. This number was then divided by the total concentration of <em>S. aureus</em> cells (cfu/mL) in the culture, which was calculated from the number of colonies that grew on MSA.</p></section><section id="sec2dot6-life-11-00960"><h3 class="pmc_sec_title">2.6. Chromosome vs. Plasmid Location of bla<sub>OXA-500</sub>
</h3>
<p>Others have documented in the literature that <em>bla</em><sub>OXA-500</sub> is located on the chromosome and is not a plasmid. To confirm that the same was true for our donor strain, <em>A. pittii</em> IIF1SW-P1, the following analyses were performed. plasmidSPAdes was used to assemble plasmid sequences from the whole genome-sequencing data of our strain. A simple blastn comparison between the <em>bla</em><sub>OXA-500</sub> sequence and the plasmid sequences showed the absence of the gene from the plasmid-assembled contigs. Furthermore, oriTfinder was used to determine whether scaffold_5 (length = 172 kb), on which <em>bla</em><sub>OXA-500</sub> was found, as well as the assembled sequences not characterized as plasmid sequences (from the plasmidSPAdes output above), contained evidence of plasmid markers, such as an origin of transfer site (<em>oriT</em>) and relaxases. None of these plasmid markers were detected on scaffold_5 and the “chromosomal” sequences.</p></section></section><section id="sec3-life-11-00960"><h2 class="pmc_sec_title">3. Results</h2>
<p>In order to determine whether HGT occurred between donor and recipient, suitable PCR primers were designed to amplify regions of the <em>bla</em><sub>OXA-500</sub> and <em>ISAba1</em> genes from <em>A. pittii</em> IIF1SW-P1. The sequences of the two genes, used to design the primers, were extracted from the draft whole-genome sequence of <em>A. pittii</em> IIF1SW-P1 [<a aria-describedby="B21-life-11-00960" class="usa-link" href="#B21-life-11-00960">21</a>]. PCR analysis performed on DNA isolated from <em>A. pittii</em> IIF1SW-P1 using the designed <em>bla</em><sub>OXA-500</sub> and <em>ISAba1</em> primers showed strong bands on an agarose gel, while PCR analysis performed on DNA isolated from the four <em>S. aureus</em> strains showed no amplicons.</p>
<p>To compare HGT potential between SMG and 1G, co-cultures of <em>A. pittii</em> (donor strain) and <em>S. aureus</em> (recipient strain) were grown in the HARV under SMG (vertical rotation) or 1G (horizontal rotation) for 20 h, after which the culture was plated on mannitol salt agar (MSA) to select for <em>S. aureus</em>. The MSA was also supplemented with oxacillin to select only those <em>S. aureus</em> colonies that had acquired resistance to the antibiotic. To verify that this newly acquired resistance to oxacillin was not due to spontaneous mutations but rather through the HGT of <em>bla</em><sub>OXA-500</sub> and <em>ISAba1</em> from <em>A. pittii</em>, the <em>S. aureus</em> colonies that grew on MSA supplemented with oxacillin were picked for PCR analysis. A schematic of the experimental set-up is shown in <a class="usa-link" href="#life-11-00960-f001">Figure 1</a>.</p>
<figure class="fig xbox font-sm" id="life-11-00960-f001"><h3 class="obj_head">Figure 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8468678_life-11-00960-g001.jpg" target="_blank"><img alt="Figure 1" class="graphic zoom-in" height="1067" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dce3/8468678/d0115e637f57/life-11-00960-g001.jpg" width="758"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/life-11-00960-f001/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Schematic of experimental protocol.</p></figcaption></figure><p><a class="usa-link" href="#life-11-00960-f002">Figure 2</a> shows the proportion of colonies that grew on MSA supplemented with oxacillin that were positive for <em>bla</em><sub>OXA-500</sub> (<a class="usa-link" href="#life-11-00960-f002">Figure 2</a>A), <em>ISAba1</em> (<a class="usa-link" href="#life-11-00960-f002">Figure 2</a>B) and both (<a class="usa-link" href="#life-11-00960-f002">Figure 2</a>C). Representative gel images are shown in <a class="usa-link" href="#app1-life-11-00960">Figure S1</a>. On average, 77% of the <em>S. aureus</em> colonies that were grown with <em>A. pittii</em> under SMG and had acquired resistance to oxacillin now harbored both <em>bla</em><sub>OXA-500</sub> and <em>ISAba1</em> (89% of the colonies had acquired at least one). This is in contrast with 1G, where only 1% of <em>S. aureus</em> colonies that had acquired oxacillin resistance were positive for both <em>bla</em><sub>OXA-500</sub> and <em>ISAba1</em> (8% of the colonies had acquired at least one).</p>
<figure class="fig xbox font-sm" id="life-11-00960-f002"><h3 class="obj_head">Figure 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=8468678_life-11-00960-g002.jpg" target="_blank"><img alt="Figure 2" class="graphic zoom-in" height="306" loading="lazy" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dce3/8468678/cce418e424b6/life-11-00960-g002.jpg" width="761"/></a></p>
<div class="p text-right font-secondary"><a class="usa-link" href="figure/life-11-00960-f002/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div>
<figcaption><p>Comparison of HGT activity between SMG and 1G. Four strains of <em>S. aureus</em> (IF4SW, IIF6SW, IF7SW, IIF8SW) were used as the recipient and were co-cultured (separately) with the donor strain <em>A. pittii</em> for 20 h at 37 °C under SMG or 1G conditions. Cultures were plated on MSA containing 4 µg/mL of oxacillin. Colonies that grew were picked, subjected to DNA extraction and then analyzed by PCR with primers specific to <em>bla</em><sub>OXA-500</sub> and <em>ISAba1</em> from the donor strain. The graphs show the percent of <em>S. aureus</em> colonies, from each strain, that grew on MSA+ oxacillin that had acquired (<strong>A</strong>) <em>bla</em><sub>OXA-500</sub>, (<strong>B</strong>) <em>ISAba1</em> or (<strong>C</strong>) both genes from the donor <em>A. pittii</em>.</p></figcaption></figure><p>The differences in proportions observed between SMG and 1G were not due to differences in growth at these different conditions, as each strain showed the same biomass after 20 h, whether grown under SMG or 1G (<a class="usa-link" href="#app1-life-11-00960">Figure S2</a>).</p>
<p>The efficiency of transfer under SMG for all four strains was, on average, 3.06 × 10<sup>−7</sup> while, for 1G, it was 3.21 × 10<sup>−9</sup> (<a class="usa-link" href="#life-11-00960-t001">Table 1</a>). While all strains had similar efficiencies, IIF8SW had the highest and IIF6SW the lowest.</p>
<section class="tw xbox font-sm" id="life-11-00960-t001"><h3 class="obj_head">Table 1.</h3>
<div class="caption p"><p>Efficiency of HGT. Average values of the three biological replicates, with the range of the values shown in parentheses. Efficiency was based on colonies that at acquired at least one gene (either <em>bla</em><sub>OXA-500</sub> or <em>ISAba1</em>).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="center" colspan="1" rowspan="1" style="border-top:solid thin;border-bottom:solid thin" valign="middle">
<em>S. aureus</em> Recipient Strain</th>
<th align="center" colspan="1" rowspan="1" style="border-top:solid thin;border-bottom:solid thin" valign="middle">SMG (Range)</th>
<th align="center" colspan="1" rowspan="1" style="border-top:solid thin;border-bottom:solid thin" valign="middle">1G (Range)</th>
</tr></thead>
<tbody>
<tr>
<td align="center" colspan="1" rowspan="1" valign="middle">IF4SW</td>
<td align="center" colspan="1" rowspan="1" valign="middle">4.21 × 10<sup>−7</sup> (1.1 × 10<sup>−6</sup> to 5.2 × 10<sup>−8</sup>)</td>
<td align="center" colspan="1" rowspan="1" valign="middle">2.05 × 10<sup>−9</sup> (0.0 to 4.1 × 10<sup>−9</sup>)</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1" valign="middle">IIF6SW</td>
<td align="center" colspan="1" rowspan="1" valign="middle">4.54 × 10<sup>−8</sup> (7.5 × 10<sup>−8</sup> to 6.3 × 10<sup>−9</sup>)</td>
<td align="center" colspan="1" rowspan="1" valign="middle">0.0 (no transfer)</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1" valign="middle">IF7SW</td>
<td align="center" colspan="1" rowspan="1" valign="middle">3.20 × 10<sup>−7</sup> (7.2 × 10<sup>−7</sup> to 9.9 × 10<sup>−8</sup>)</td>
<td align="center" colspan="1" rowspan="1" valign="middle">4.33 × 10<sup>−9</sup> (7.5 × 10<sup>−9</sup> to 2.2 × 10<sup>−9</sup>)</td>
</tr>
<tr>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">IIF8SW</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">4.38 × 10<sup>−7</sup> (6.8 × 10<sup>−7</sup> to 3.1 × 10<sup>−9</sup>)</td>
<td align="center" colspan="1" rowspan="1" style="border-bottom:solid thin" valign="middle">6.07 × 10<sup>−9</sup> (0.0 to 1.2 × 10<sup>−9</sup>)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a class="usa-link" href="table/life-11-00960-t001/" rel="noopener noreferrer" target="_blank">Open in a new tab</a></div></section><p>To determine how stable these donor genes were in the recipient, two resistant <em>S. aureus</em> 1F4SW colonies that had acquired both genes were sub-cultured on MSA + oxacillin plates, a total of four times. Even after the 4th sub-culture, the colonies were still resistant to oxacillin, and PCR confirmed that <em>bla</em><sub>OXA-500</sub> and <em>ISAba1</em> were still present within the genome.</p></section><section id="sec10"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>We have shown that HGT is increased under SMG compared to 1G. The transfer of AMR genes, which were tracked by PCR, occurred in the absence of selective pressure by antibiotics. This transfer led to functional changes, as the acquisition of the two AMR genes, <em>bla</em><sub>OXA-500</sub> and <em>ISAba1</em>, that were tracked in the four strains of <em>S. aureus</em>, made them resistant to oxacillin, to which they were previously susceptible. These results have significant implications for long-duration space missions because pathogens that are susceptible to antibiotics could become resistant and thus harder to treat. In addition, even if astronauts are screened for problematic microbes prior to flight, issues could still arise, as commensals are known to act as reservoirs for AMR genes that can be disseminated to (opportunistic) pathogens [<a aria-describedby="B22-life-11-00960" class="usa-link" href="#B22-life-11-00960">22</a>,<a aria-describedby="B23-life-11-00960" class="usa-link" href="#B23-life-11-00960">23</a>,<a aria-describedby="B24-life-11-00960" class="usa-link" href="#B24-life-11-00960">24</a>]. For example, a study conducted in a hospital in the Netherlands showed that <em>mecA</em> gene transfer occurred in a patient between <em>mecA</em><sup>+</sup> <em>S. epidermidis</em> and a <em>mecA</em><sup>−</sup> strain of <em>S. aureus</em> [<a aria-describedby="B25-life-11-00960" class="usa-link" href="#B25-life-11-00960">25</a>].</p>
<p>It was beyond the scope of the study to determine the cause of the increased HGT that was observed, but one mechanism could be increased competence, allowing for transformation to occur, one of three modes of gene transfer. Competence allows bacterial cells to uptake DNA from its environment and increase the response to different stressors [<a aria-describedby="B26-life-11-00960" class="usa-link" href="#B26-life-11-00960">26</a>,<a aria-describedby="B27-life-11-00960" class="usa-link" href="#B27-life-11-00960">27</a>,<a aria-describedby="B28-life-11-00960" class="usa-link" href="#B28-life-11-00960">28</a>,<a aria-describedby="B29-life-11-00960" class="usa-link" href="#B29-life-11-00960">29</a>]. In this case, naked DNA released from dead <em>A. pittii</em> cells could have been readily taken up by <em>S. aureus</em> when exposed to SMG due to an increase in its competence. While <em>S. aureus</em> is not classically considered to be naturally competent, as <em>Streptococcus pneumoniae</em> and <em>Bacillus subtilis</em> are, it does harbor genes that are involved in competence and thought to promote natural competence under the right conditions [<a aria-describedby="B30-life-11-00960" class="usa-link" href="#B30-life-11-00960">30</a>,<a aria-describedby="B31-life-11-00960" class="usa-link" href="#B31-life-11-00960">31</a>]. The conditions that <em>S. aureus</em> needs are not yet known, but, similar to other bacteria, it could be an adverse environment. Indeed, when <em>S. aureus</em> cells were subjected to heat-shock, cold-shock and the stringent response, there was increased expression of competence gene orthologs compared to the non-stressed conditions [<a aria-describedby="B32-life-11-00960" class="usa-link" href="#B32-life-11-00960">32</a>]. In <em>S. pneumoniae</em> and <em>B. subtilis</em>, competence confers a survival advantage when cells are subjected to stress [<a aria-describedby="B33-life-11-00960" class="usa-link" href="#B33-life-11-00960">33</a>,<a aria-describedby="B34-life-11-00960" class="usa-link" href="#B34-life-11-00960">34</a>], so there is a possibility that growth under SMG led to an adverse environment for <em>S. aureus</em>, causing cells to become more competent to increase survival. Conjugation is a less likely scenario since, consistent with what others have documented in the literature, <em>bla</em><sub>OXA-500</sub> is located on the chromosome and not a plasmid. A second mechanism could be increased transduction caused by growth under SMG. Transduction is the transfer of genetic material via bacterial phages. It is well known that prophage activity is increased during stress, leading to increased phage production, helping the host survive these adverse conditions [<a aria-describedby="B35-life-11-00960" class="usa-link" href="#B35-life-11-00960">35</a>,<a aria-describedby="B36-life-11-00960" class="usa-link" href="#B36-life-11-00960">36</a>].</p></section><section id="sec5-life-11-00960"><h2 class="pmc_sec_title">5. Conclusions</h2>
<p>This paper describes, for the first time, the effects of SMG on HGT and has shown that it is increased in SMG compared to 1G. If the same trend is observed on the ISS, this will have a significant impact on how we view bacterial interactions in space and its effects on astronaut health. The next steps would be to elucidate the mechanisms behind the HGT, to develop appropriate countermeasures for long-duration space missions.</p></section><section class="app" id="app1-life-11-00960"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<p>The following are available online at <a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://www.mdpi.com/article/10.3390/life11090960/s1" rel="noopener noreferrer" target="_blank">https://www.mdpi.com/article/10.3390/life11090960/s1</a>, Figure S1: Representative gel images of PCR analysis. Figure S2: Colony forming units of strains grown in SMG and 1G. Figure S3: Schematic of designed primers and their location within the donor genome.</p>
<section class="sm xbox font-sm" id="life-11-00960-s001"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/8468678/bin/life-11-00960-s001.zip">Click here for additional data file.</a><sup> (1.4MB, zip) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>C.U. conceptualized the study, designed the primers, performed all experiments, analyzed the data, and wrote the manuscript. T.G. helped design the primers and optimize the initial protocol. C.E.M. provides ongoing whole genome sequencing capabilities for the lab. K.V. provided guidance during concept maturation, critically reviewed the data and the manuscript. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was funded by the National Aeronautics and Space Administration 2020 Space Biology (NNH18ZTT001N-FG2) grant (80NSSC21K0320) awarded to CU and a 2012 Space Biology NNH12ZTT001N Grant No. 19-12829-26 under Task Order NNN13D111T awarded to KV.</p></section><section id="notes3"><h2 class="pmc_sec_title">Institutional Review Board Statement</h2>
<p>Not applicable.</p></section><section id="notes4"><h2 class="pmc_sec_title">Informed Consent Statement</h2>
<p>Not applicable.</p></section><section id="notes5"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The draft genomes of the parental strains used for the co-cultures can be found in NCBI under accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZX00000000" rel="noopener noreferrer" target="_blank">MIZX00000000</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZH00000000" rel="noopener noreferrer" target="_blank">MIZH00000000</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZN00000000" rel="noopener noreferrer" target="_blank">MIZN00000000</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZM00000000" rel="noopener noreferrer" target="_blank">MIZM00000000</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZP00000000" rel="noopener noreferrer" target="_blank">MIZP00000000</a> and are presented in the manuscript by Checinska Sielaff A et al. [<a aria-describedby="B21-life-11-00960" class="usa-link" href="#B21-life-11-00960">21</a>].</p></section><section id="notes6"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflict of interest.</p></section><section class="fn-group" id="fn-group1"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s Note:</strong> MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section class="ref-list" id="ref-list1"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1-life-11-00960">
<span class="label">1.</span><cite>Schulze-Makuch D., Wagner D., Kounaves S.P., Mangelsdorf K., Devine K.G., de Vera J.-P., Schmitt-Kopplin P., Grossart H.-P., Parro V., Kaupenjohann M., et al.  Transitory microbial habitat in the hyperarid Atacama Desert. Proc. Natl. Acad. Sci. USA. 2018;115:2670–2675. doi: 10.1073/pnas.1714341115.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.1714341115" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5856521/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29483268/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Transitory%20microbial%20habitat%20in%20the%20hyperarid%20Atacama%20Desert&amp;author=D.%20Schulze-Makuch&amp;author=D.%20Wagner&amp;author=S.P.%20Kounaves&amp;author=K.%20Mangelsdorf&amp;author=K.G.%20Devine&amp;volume=115&amp;publication_year=2018&amp;pages=2670-2675&amp;pmid=29483268&amp;doi=10.1073/pnas.1714341115&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B2-life-11-00960">
<span class="label">2.</span><cite>Carpa R., Keul A., Muntean V., Dobrotă C. Characterization of halophilic bacterial communities in Turda Salt Mine (Romania) Orig. Life Evol. Biosph. 2014;44:223–230. doi: 10.1007/s11084-014-9375-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s11084-014-9375-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4669543/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25476992/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Orig.%20Life%20Evol.%20Biosph.&amp;title=Characterization%20of%20halophilic%20bacterial%20communities%20in%20Turda%20Salt%20Mine%20(Romania)&amp;author=R.%20Carpa&amp;author=A.%20Keul&amp;author=V.%20Muntean&amp;author=C.%20Dobrot%C4%83&amp;volume=44&amp;publication_year=2014&amp;pages=223-230&amp;pmid=25476992&amp;doi=10.1007/s11084-014-9375-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B3-life-11-00960">
<span class="label">3.</span><cite>Humayoun S.B., Bano N., Hollibaugh J.T. Depth distribution of microbial diversity in Mono Lake, a meromictic soda lake in California. Appl. Environ. Microbiol. 2003;69:1030–1042. doi: 10.1128/AEM.69.2.1030-1042.2003.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/AEM.69.2.1030-1042.2003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC143613/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/12571026/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&amp;title=Depth%20distribution%20of%20microbial%20diversity%20in%20Mono%20Lake,%20a%20meromictic%20soda%20lake%20in%20California&amp;author=S.B.%20Humayoun&amp;author=N.%20Bano&amp;author=J.T.%20Hollibaugh&amp;volume=69&amp;publication_year=2003&amp;pages=1030-1042&amp;pmid=12571026&amp;doi=10.1128/AEM.69.2.1030-1042.2003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B4-life-11-00960">
<span class="label">4.</span><cite>Walker J.J., Spear J.R., Pace N.R. Geobiology of a microbial endolithic community in the Yellowstone geothermal environment. Nature. 2005;434:1011–1014. doi: 10.1038/nature03447.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature03447" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/15846344/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Geobiology%20of%20a%20microbial%20endolithic%20community%20in%20the%20Yellowstone%20geothermal%20environment&amp;author=J.J.%20Walker&amp;author=J.R.%20Spear&amp;author=N.R.%20Pace&amp;volume=434&amp;publication_year=2005&amp;pages=1011-1014&amp;pmid=15846344&amp;doi=10.1038/nature03447&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B5-life-11-00960">
<span class="label">5.</span><cite>Ji M., Greening C., Vanwonterghem I., Carere C.R., Bay S.K., Steen J.A., Montgomery K., Lines T., Beardall J., van Dorst J., et al.  Atmospheric trace gases support primary production in Antarctic desert surface soil. Nature. 2017;552:400. doi: 10.1038/nature25014.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nature25014" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29211716/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Atmospheric%20trace%20gases%20support%20primary%20production%20in%20Antarctic%20desert%20surface%20soil&amp;author=M.%20Ji&amp;author=C.%20Greening&amp;author=I.%20Vanwonterghem&amp;author=C.R.%20Carere&amp;author=S.K.%20Bay&amp;volume=552&amp;publication_year=2017&amp;pages=400&amp;pmid=29211716&amp;doi=10.1038/nature25014&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B6-life-11-00960">
<span class="label">6.</span><cite>Yamaguchi N., Roberts M., Castro S., Oubre C., Makimura K., Leys N., Grohmann E., Sugita T., Ichijo T., Nasu M. Microbial monitoring of crewed habitats in space-current status and future perspectives. Microb. Environ. 2014;29:250–260. doi: 10.1264/jsme2.ME14031.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1264/jsme2.ME14031" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4159036/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25130885/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microb.%20Environ.&amp;title=Microbial%20monitoring%20of%20crewed%20habitats%20in%20space-current%20status%20and%20future%20perspectives&amp;author=N.%20Yamaguchi&amp;author=M.%20Roberts&amp;author=S.%20Castro&amp;author=C.%20Oubre&amp;author=K.%20Makimura&amp;volume=29&amp;publication_year=2014&amp;pages=250-260&amp;pmid=25130885&amp;doi=10.1264/jsme2.ME14031&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B7-life-11-00960">
<span class="label">7.</span><cite>Urbaniak C., Sielaff A.C., Frey K.G., Allen J.E., Singh N., Jaing C., Wheeler K., Venkateswaran K. Detection of antimicrobial resistance genes associated with the International Space Station environmental surfaces. Sci. Rep. 2018;8:814. doi: 10.1038/s41598-017-18506-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-017-18506-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5770469/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/29339831/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Detection%20of%20antimicrobial%20resistance%20genes%20associated%20with%20the%20International%20Space%20Station%20environmental%20surfaces&amp;author=C.%20Urbaniak&amp;author=A.C.%20Sielaff&amp;author=K.G.%20Frey&amp;author=J.E.%20Allen&amp;author=N.%20Singh&amp;volume=8&amp;publication_year=2018&amp;pages=814&amp;pmid=29339831&amp;doi=10.1038/s41598-017-18506-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B8-life-11-00960">
<span class="label">8.</span><cite>Checinska Sielaff A., Urbaniak C., Mohan G.B.M., Stepanov V.G., Tran Q., Wood J.M., Minich J., McDonald D., Mayer T., Knight R., et al.  Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome. 2019;7:50. doi: 10.1186/s40168-019-0666-x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40168-019-0666-x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6452512/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30955503/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&amp;author=A.%20Checinska%20Sielaff&amp;author=C.%20Urbaniak&amp;author=G.B.M.%20Mohan&amp;author=V.G.%20Stepanov&amp;author=Q.%20Tran&amp;volume=7&amp;publication_year=2019&amp;pages=50&amp;pmid=30955503&amp;doi=10.1186/s40168-019-0666-x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B9-life-11-00960">
<span class="label">9.</span><cite>Singh N.K., Wood J.M., Karouia F., Venkateswaran K. Succession and persistence of microbial communities and antimicrobial resistance genes associated with International Space Station environmental surfaces. Microbiome. 2018;6:204. doi: 10.1186/s40168-018-0585-2.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/s40168-018-0585-2" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6234677/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30424821/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microbiome&amp;title=Succession%20and%20persistence%20of%20microbial%20communities%20and%20antimicrobial%20resistance%20genes%20associated%20with%20International%20Space%20Station%20environmental%20surfaces&amp;author=N.K.%20Singh&amp;author=J.M.%20Wood&amp;author=F.%20Karouia&amp;author=K.%20Venkateswaran&amp;volume=6&amp;publication_year=2018&amp;pages=204&amp;pmid=30424821&amp;doi=10.1186/s40168-018-0585-2&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B10-life-11-00960">
<span class="label">10.</span><cite>Soucy S.M., Huang J., Gogarten J.P. Horizontal gene transfer: Building the web of life. Nat. Rev. Genet. 2015;16:472–482. doi: 10.1038/nrg3962.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nrg3962" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/26184597/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=Horizontal%20gene%20transfer:%20Building%20the%20web%20of%20life&amp;author=S.M.%20Soucy&amp;author=J.%20Huang&amp;author=J.P.%20Gogarten&amp;volume=16&amp;publication_year=2015&amp;pages=472-482&amp;pmid=26184597&amp;doi=10.1038/nrg3962&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B11-life-11-00960">
<span class="label">11.</span><cite>Frost L.S., Leplae R., Summers A.O., Toussaint A. Mobile genetic elements: The agents of open source evolution. Nat. Rev. Microbiol. 2005;3:722–732. doi: 10.1038/nrmicro1235.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/nrmicro1235" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16138100/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&amp;title=Mobile%20genetic%20elements:%20The%20agents%20of%20open%20source%20evolution&amp;author=L.S.%20Frost&amp;author=R.%20Leplae&amp;author=A.O.%20Summers&amp;author=A.%20Toussaint&amp;volume=3&amp;publication_year=2005&amp;pages=722-732&amp;pmid=16138100&amp;doi=10.1038/nrmicro1235&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B12-life-11-00960">
<span class="label">12.</span><cite>Roberts A.P., Kreth J. The impact of horizontal gene transfer on the adaptive ability of the human oral microbiome. Front. Cell. Infect. Microbiol. 2014;4:124. doi: 10.3389/fcimb.2014.00124.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fcimb.2014.00124" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4157583/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25250243/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell.%20Infect.%20Microbiol.&amp;title=The%20impact%20of%20horizontal%20gene%20transfer%20on%20the%20adaptive%20ability%20of%20the%20human%20oral%20microbiome&amp;author=A.P.%20Roberts&amp;author=J.%20Kreth&amp;volume=4&amp;publication_year=2014&amp;pages=124&amp;pmid=25250243&amp;doi=10.3389/fcimb.2014.00124&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B13-life-11-00960">
<span class="label">13.</span><cite>Webster N.S., Reusch T.B.H. Microbial contributions to the persistence of coral reefs. ISME J. 2017;11:2167. doi: 10.1038/ismej.2017.66.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ismej.2017.66" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5607359/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28509908/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=Microbial%20contributions%20to%20the%20persistence%20of%20coral%20reefs&amp;author=N.S.%20Webster&amp;author=T.B.H.%20Reusch&amp;volume=11&amp;publication_year=2017&amp;pages=2167&amp;pmid=28509908&amp;doi=10.1038/ismej.2017.66&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B14-life-11-00960">
<span class="label">14.</span><cite>Koonin E.V., Makarova K.S., Aravind L. Horizontal gene transfer in prokaryotes: Quantification and classification. Annu. Rev. Microbiol. 2001;55:709–742. doi: 10.1146/annurev.micro.55.1.709.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev.micro.55.1.709" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4781227/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11544372/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Microbiol.&amp;title=Horizontal%20gene%20transfer%20in%20prokaryotes:%20Quantification%20and%20classification&amp;author=E.V.%20Koonin&amp;author=K.S.%20Makarova&amp;author=L.%20Aravind&amp;volume=55&amp;publication_year=2001&amp;pages=709-742&amp;pmid=11544372&amp;doi=10.1146/annurev.micro.55.1.709&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B15-life-11-00960">
<span class="label">15.</span><cite>Brooks A.N., Turkarslan S., Beer K.D., Lo F.Y., Baliga N.S. Adaptation of cells to new environments. Wiley Interdiscip. Rev. Syst. Biol. Med. 2011;3:544–561. doi: 10.1002/wsbm.136.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1002/wsbm.136" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3081528/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21197660/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Wiley%20Interdiscip.%20Rev.%20Syst.%20Biol.%20Med.&amp;title=Adaptation%20of%20cells%20to%20new%20environments&amp;author=A.N.%20Brooks&amp;author=S.%20Turkarslan&amp;author=K.D.%20Beer&amp;author=F.Y.%20Lo&amp;author=N.S.%20Baliga&amp;volume=3&amp;publication_year=2011&amp;pages=544-561&amp;pmid=21197660&amp;doi=10.1002/wsbm.136&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B16-life-11-00960">
<span class="label">16.</span><cite>Vogan A.A., Higgs P.G. The advantages and disadvantages of horizontal gene transfer and the emergence of the first species. Biol. Direct. 2011;6:1. doi: 10.1186/1745-6150-6-1.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1186/1745-6150-6-1" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3043529/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21199581/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Biol.%20Direct&amp;title=The%20advantages%20and%20disadvantages%20of%20horizontal%20gene%20transfer%20and%20the%20emergence%20of%20the%20first%20species&amp;author=A.A.%20Vogan&amp;author=P.G.%20Higgs&amp;volume=6&amp;publication_year=2011&amp;pages=1&amp;pmid=21199581&amp;doi=10.1186/1745-6150-6-1&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B17-life-11-00960">
<span class="label">17.</span><cite>Schiwon K., Arends K., Rogowski K.M., Fürch S., Prescha K., Sakinc T., Van Houdt R., Werner G., Grohmann E. Comparison of antibiotic resistance, biofilm formation and conjugative transfer of Staphylococcus and Enterococcus isolates from International Space Station and Antarctic Research Station Concordia. Microb. Ecol. 2013;65:638–651. doi: 10.1007/s00248-013-0193-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00248-013-0193-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23411852/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microb.%20Ecol.&amp;title=Comparison%20of%20antibiotic%20resistance,%20biofilm%20formation%20and%20conjugative%20transfer%20of%20Staphylococcus%20and%20Enterococcus%20isolates%20from%20International%20Space%20Station%20and%20Antarctic%20Research%20Station%20Concordia&amp;author=K.%20Schiwon&amp;author=K.%20Arends&amp;author=K.M.%20Rogowski&amp;author=S.%20F%C3%BCrch&amp;author=K.%20Prescha&amp;volume=65&amp;publication_year=2013&amp;pages=638-651&amp;pmid=23411852&amp;doi=10.1007/s00248-013-0193-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B18-life-11-00960">
<span class="label">18.</span><cite>Evans B.A., Amyes S.G.B. OXA β-lactamases. Clin. Microbiol. Rev. 2014;27:241–263. doi: 10.1128/CMR.00117-13.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/CMR.00117-13" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3993105/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24696435/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Microbiol.%20Rev.&amp;title=OXA%20%CE%B2-lactamases&amp;author=B.A.%20Evans&amp;author=S.G.B.%20Amyes&amp;volume=27&amp;publication_year=2014&amp;pages=241-263&amp;pmid=24696435&amp;doi=10.1128/CMR.00117-13&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B19-life-11-00960">
<span class="label">19.</span><cite>Khurshid M., Rasool M.H., Ashfaq U.A., Aslam B., Waseem M. Emergence of ISAba1 harboring carbapenem-resistant Acinetobacter baumannii isolates in Pakistan. Future Microbiol. 2017;12:1261–1269. doi: 10.2217/fmb-2017-0080.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2217/fmb-2017-0080" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28980827/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Future%20Microbiol.&amp;title=Emergence%20of%20ISAba1%20harboring%20carbapenem-resistant%20Acinetobacter%20baumannii%20isolates%20in%20Pakistan&amp;author=M.%20Khurshid&amp;author=M.H.%20Rasool&amp;author=U.A.%20Ashfaq&amp;author=B.%20Aslam&amp;author=M.%20Waseem&amp;volume=12&amp;publication_year=2017&amp;pages=1261-1269&amp;pmid=28980827&amp;doi=10.2217/fmb-2017-0080&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B20-life-11-00960">
<span class="label">20.</span><cite>Mugnier P.D., Poirel L., Nordmann P. Functional analysis of insertion sequence ISAba1, responsible for genomic plasticity of Acinetobacter baumannii. J. Bacteriol. 2009;191:2414–2418. doi: 10.1128/JB.01258-08.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JB.01258-08" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2655512/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19136598/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&amp;title=Functional%20analysis%20of%20insertion%20sequence%20ISAba1,%20responsible%20for%20genomic%20plasticity%20of%20Acinetobacter%20baumannii&amp;author=P.D.%20Mugnier&amp;author=L.%20Poirel&amp;author=P.%20Nordmann&amp;volume=191&amp;publication_year=2009&amp;pages=2414-2418&amp;pmid=19136598&amp;doi=10.1128/JB.01258-08&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B21-life-11-00960">
<span class="label">21.</span><cite>Checinska Sielaff A., Singh N.K., Allen J.E., Thissen J., Jaing C., Venkateswaran K. Draft Genome Sequences of Biosafety Level 2 Opportunistic Pathogens Isolated from the Environmental Surfaces of the International Space Station. Genome Announc. 2016;4:e01263–e01316. doi: 10.1128/genomeA.01263-16.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/genomeA.01263-16" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5201052/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28034853/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Genome%20Announc.&amp;title=Draft%20Genome%20Sequences%20of%20Biosafety%20Level%202%20Opportunistic%20Pathogens%20Isolated%20from%20the%20Environmental%20Surfaces%20of%20the%20International%20Space%20Station&amp;author=A.%20Checinska%20Sielaff&amp;author=N.K.%20Singh&amp;author=J.E.%20Allen&amp;author=J.%20Thissen&amp;author=C.%20Jaing&amp;volume=4&amp;publication_year=2016&amp;pages=e01263-e01316&amp;pmid=28034853&amp;doi=10.1128/genomeA.01263-16&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B22-life-11-00960">
<span class="label">22.</span><cite>Brinkac L., Voorhies A., Gomez A., Nelson K.E. The Threat of Antimicrobial Resistance on the Human Microbiome. Microb. Ecol. 2017;74:1001–1008. doi: 10.1007/s00248-017-0985-z.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1007/s00248-017-0985-z" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC5654679/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/28492988/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Microb.%20Ecol.&amp;title=The%20Threat%20of%20Antimicrobial%20Resistance%20on%20the%20Human%20Microbiome&amp;author=L.%20Brinkac&amp;author=A.%20Voorhies&amp;author=A.%20Gomez&amp;author=K.E.%20Nelson&amp;volume=74&amp;publication_year=2017&amp;pages=1001-1008&amp;pmid=28492988&amp;doi=10.1007/s00248-017-0985-z&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B23-life-11-00960">
<span class="label">23.</span><cite>Penders J., Stobberingh E.E., Savelkoul P.H.M., Wolffs P.F.G. The human microbiome as a reservoir of antimicrobial resistance. Front. Microbiol. 2013;4:87. doi: 10.3389/fmicb.2013.00087.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.3389/fmicb.2013.00087" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3627978/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23616784/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=The%20human%20microbiome%20as%20a%20reservoir%20of%20antimicrobial%20resistance&amp;author=J.%20Penders&amp;author=E.E.%20Stobberingh&amp;author=P.H.M.%20Savelkoul&amp;author=P.F.G.%20Wolffs&amp;volume=4&amp;publication_year=2013&amp;pages=87&amp;pmid=23616784&amp;doi=10.3389/fmicb.2013.00087&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B24-life-11-00960">
<span class="label">24.</span><cite>Huddleston J.R. Horizontal gene transfer in the human gastrointestinal tract: Potential spread of antibiotic resistance genes. Infect. Drug Resist. 2014;7:167–176. doi: 10.2147/IDR.S48820.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.2147/IDR.S48820" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC4073975/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25018641/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Drug%20Resist.&amp;title=Horizontal%20gene%20transfer%20in%20the%20human%20gastrointestinal%20tract:%20Potential%20spread%20of%20antibiotic%20resistance%20genes&amp;author=J.R.%20Huddleston&amp;volume=7&amp;publication_year=2014&amp;pages=167-176&amp;pmid=25018641&amp;doi=10.2147/IDR.S48820&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B25-life-11-00960">
<span class="label">25.</span><cite>Wielders C.L.C., Vriens M.R., Brisse S., de Graaf-Miltenburg L.A.M., Troelstra A., Fleer A., Schmitz F.J., Verhoef J., Fluit A.C. Evidence for in-vivo transfer of mecA DNA between strains of Staphylococcus aureus. Lancet. 2001;357:1674–1675. doi: 10.1016/S0140-6736(00)04832-7.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/S0140-6736(00)04832-7" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/11425376/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Evidence%20for%20in-vivo%20transfer%20of%20mecA%20DNA%20between%20strains%20of%20Staphylococcus%20aureus&amp;author=C.L.C.%20Wielders&amp;author=M.R.%20Vriens&amp;author=S.%20Brisse&amp;author=L.A.M.%20de%20Graaf-Miltenburg&amp;author=A.%20Troelstra&amp;volume=357&amp;publication_year=2001&amp;pages=1674-1675&amp;pmid=11425376&amp;doi=10.1016/S0140-6736(00)04832-7&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B26-life-11-00960">
<span class="label">26.</span><cite>Charpentier X., Kay E., Schneider D., Shuman H.A. Antibiotics and UV Radiation Induce Competence for Natural Transformation in Legionella pneumophila. J. Bacteriol. 2011;193:1114–1121. doi: 10.1128/JB.01146-10.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JB.01146-10" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3067580/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21169481/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&amp;title=Antibiotics%20and%20UV%20Radiation%20Induce%20Competence%20for%20Natural%20Transformation%20in%20Legionella%20pneumophila&amp;author=X.%20Charpentier&amp;author=E.%20Kay&amp;author=D.%20Schneider&amp;author=H.A.%20Shuman&amp;volume=193&amp;publication_year=2011&amp;pages=1114-1121&amp;pmid=21169481&amp;doi=10.1128/JB.01146-10&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B27-life-11-00960">
<span class="label">27.</span><cite>Claverys J.-P., Prudhomme M., Martin B. Induction of Competence Regulons as a General Response to Stress in Gram-Positive Bacteria. Annu. Rev. Microbiol. 2006;60:451–475. doi: 10.1146/annurev.micro.60.080805.142139.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1146/annurev.micro.60.080805.142139" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16771651/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Microbiol.&amp;title=Induction%20of%20Competence%20Regulons%20as%20a%20General%20Response%20to%20Stress%20in%20Gram-Positive%20Bacteria&amp;author=J.-P.%20Claverys&amp;author=M.%20Prudhomme&amp;author=B.%20Martin&amp;volume=60&amp;publication_year=2006&amp;pages=451-475&amp;pmid=16771651&amp;doi=10.1146/annurev.micro.60.080805.142139&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B28-life-11-00960">
<span class="label">28.</span><cite>Manna S., Waring A., Papanicolaou A., Hall N.E., Bozinovski S., Dunne E.M., Satzke C. The transcriptomic response of Streptococcus pneumoniae following exposure to cigarette smoke extract. Sci. Rep. 2018;8:15716. doi: 10.1038/s41598-018-34103-5.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41598-018-34103-5" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC6200755/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30356075/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=The%20transcriptomic%20response%20of%20Streptococcus%20pneumoniae%20following%20exposure%20to%20cigarette%20smoke%20extract&amp;author=S.%20Manna&amp;author=A.%20Waring&amp;author=A.%20Papanicolaou&amp;author=N.E.%20Hall&amp;author=S.%20Bozinovski&amp;volume=8&amp;publication_year=2018&amp;pages=15716&amp;pmid=30356075&amp;doi=10.1038/s41598-018-34103-5&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B29-life-11-00960">
<span class="label">29.</span><cite>Slager J., Kjos M., Attaiech L., Veening J.W. Antibiotic-induced replication stress triggers bacterial competence by increasing gene dosage near the origin. Cell. 2014;157:395–406. doi: 10.1016/j.cell.2014.01.068.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1016/j.cell.2014.01.068" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/24725406/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Antibiotic-induced%20replication%20stress%20triggers%20bacterial%20competence%20by%20increasing%20gene%20dosage%20near%20the%20origin&amp;author=J.%20Slager&amp;author=M.%20Kjos&amp;author=L.%20Attaiech&amp;author=J.W.%20Veening&amp;volume=157&amp;publication_year=2014&amp;pages=395-406&amp;pmid=24725406&amp;doi=10.1016/j.cell.2014.01.068&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B30-life-11-00960">
<span class="label">30.</span><cite>Fagerlund A., Granum P.E., Håvarstein L.S. Staphylococcus aureus competence genes: Mapping of the SigH, ComK1 and ComK2 regulons by transcriptome sequencing. Mol. Microbiol. 2014;94:557–579. doi: 10.1111/mmi.12767.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/mmi.12767" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/25155269/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&amp;title=Staphylococcus%20aureus%20competence%20genes:%20Mapping%20of%20the%20SigH,%20ComK1%20and%20ComK2%20regulons%20by%20transcriptome%20sequencing&amp;author=A.%20Fagerlund&amp;author=P.E.%20Granum&amp;author=L.S.%20H%C3%A5varstein&amp;volume=94&amp;publication_year=2014&amp;pages=557-579&amp;pmid=25155269&amp;doi=10.1111/mmi.12767&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B31-life-11-00960">
<span class="label">31.</span><cite>Morikawa K., Takemura A.J., Inose Y., Tsai M., Nguyen Thi le T., Ohta T., Msadek T. Expression of a cryptic secondary sigma factor gene unveils natural competence for DNA transformation in Staphylococcus aureus. PLoS Pathog. 2012;8:e1003003.  doi: 10.1371/journal.ppat.1003003.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1371/journal.ppat.1003003" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3486894/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/23133387/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=Expression%20of%20a%20cryptic%20secondary%20sigma%20factor%20gene%20unveils%20natural%20competence%20for%20DNA%20transformation%20in%20Staphylococcus%20aureus&amp;author=K.%20Morikawa&amp;author=A.J.%20Takemura&amp;author=Y.%20Inose&amp;author=M.%20Tsai&amp;author=T.%20Nguyen%20Thi%20le&amp;volume=8&amp;publication_year=2012&amp;pages=e1003003&amp;pmid=23133387&amp;doi=10.1371/journal.ppat.1003003&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B32-life-11-00960">
<span class="label">32.</span><cite>Anderson K.L., Roberts C., Disz T., Vonstein V., Hwang K., Overbeek R., Olson P.D., Projan S.J., Dunman P.M. Characterization of the Staphylococcus aureus heat shock, cold shock, stringent, and SOS responses and their effects on log-phase mRNA turnover. J. Bacteriol. 2006;188:6739–6756. doi: 10.1128/JB.00609-06.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1128/JB.00609-06" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC1595530/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/16980476/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&amp;title=Characterization%20of%20the%20Staphylococcus%20aureus%20heat%20shock,%20cold%20shock,%20stringent,%20and%20SOS%20responses%20and%20their%20effects%20on%20log-phase%20mRNA%20turnover&amp;author=K.L.%20Anderson&amp;author=C.%20Roberts&amp;author=T.%20Disz&amp;author=V.%20Vonstein&amp;author=K.%20Hwang&amp;volume=188&amp;publication_year=2006&amp;pages=6739-6756&amp;pmid=16980476&amp;doi=10.1128/JB.00609-06&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B33-life-11-00960">
<span class="label">33.</span><cite>Engelmoer D.J., Rozen D.E. Competence increases survival during stress in Streptococcus pneumoniae. Evol. Int. J. Org. Evol. 2011;65:3475–3485. doi: 10.1111/j.1558-5646.2011.01402.x.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1111/j.1558-5646.2011.01402.x" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/22133219/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Evol.%20Int.%20J.%20Org.%20Evol.&amp;title=Competence%20increases%20survival%20during%20stress%20in%20Streptococcus%20pneumoniae&amp;author=D.J.%20Engelmoer&amp;author=D.E.%20Rozen&amp;volume=65&amp;publication_year=2011&amp;pages=3475-3485&amp;pmid=22133219&amp;doi=10.1111/j.1558-5646.2011.01402.x&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B34-life-11-00960">
<span class="label">34.</span><cite>Schultz D., Wolynes P.G., Jacob E.B., Onuchic J.N. Deciding fate in adverse times: Sporulation and competence in Bacillus subtilis. Proc. Natl. Acad. Sci. USA. 2009;106:21027–21034. doi: 10.1073/pnas.0912185106.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1073/pnas.0912185106" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC2795487/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/19995980/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;title=Deciding%20fate%20in%20adverse%20times:%20Sporulation%20and%20competence%20in%20Bacillus%20subtilis&amp;author=D.%20Schultz&amp;author=P.G.%20Wolynes&amp;author=E.B.%20Jacob&amp;author=J.N.%20Onuchic&amp;volume=106&amp;publication_year=2009&amp;pages=21027-21034&amp;pmid=19995980&amp;doi=10.1073/pnas.0912185106&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B35-life-11-00960">
<span class="label">35.</span><cite>Wang X., Kim Y., Ma Q., Hong S.H., Pokusaeva K., Sturino J.M., Wood T.K. Cryptic prophages help bacteria cope with adverse environments. Nat. Commun. 2010;1:147. doi: 10.1038/ncomms1146.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/ncomms1146" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="/articles/PMC3105296/">PMC free article</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/21266997/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Cryptic%20prophages%20help%20bacteria%20cope%20with%20adverse%20environments&amp;author=X.%20Wang&amp;author=Y.%20Kim&amp;author=Q.%20Ma&amp;author=S.H.%20Hong&amp;author=K.%20Pokusaeva&amp;volume=1&amp;publication_year=2010&amp;pages=147&amp;pmid=21266997&amp;doi=10.1038/ncomms1146&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
<li id="B36-life-11-00960">
<span class="label">36.</span><cite>Du Toit A. Phage induction in different contexts. Nat. Rev. Microbiol. 2019;17:126–127. doi: 10.1038/s41579-019-0150-4.</cite> [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://doi.org/10.1038/s41579-019-0150-4" rel="noopener noreferrer" target="_blank">DOI</a>] [<a class="usa-link" href="https://pubmed.ncbi.nlm.nih.gov/30683886/">PubMed</a>] [<a class="usa-link usa-link--external" data-ga-action="click_feat_suppl" href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&amp;title=Phage%20induction%20in%20different%20contexts&amp;author=A.%20Du%20Toit&amp;volume=17&amp;publication_year=2019&amp;pages=126-127&amp;pmid=30683886&amp;doi=10.1038/s41579-019-0150-4&amp;" rel="noopener noreferrer" target="_blank">Google Scholar</a>]</li>
</ul></section></section><section class="associated-data" id="_ad93_" lang="en"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section class="supplementary-materials" id="_adsm93_" lang="en"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a class="usa-link" data-ga-action="click_feat_suppl" href="/articles/instance/8468678/bin/life-11-00960-s001.zip">Click here for additional data file.</a><sup> (1.4MB, zip) </sup>
</div></div></section></section><section class="data-availability-statement" id="_adda93_" lang="en"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The draft genomes of the parental strains used for the co-cultures can be found in NCBI under accession number <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZX00000000" rel="noopener noreferrer" target="_blank">MIZX00000000</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZH00000000" rel="noopener noreferrer" target="_blank">MIZH00000000</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZN00000000" rel="noopener noreferrer" target="_blank">MIZN00000000</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZM00000000" rel="noopener noreferrer" target="_blank">MIZM00000000</a>, <a class="usa-link" data-ga-action="click_feat_suppl" href="https://www.ncbi.nlm.nih.gov/nuccore/MIZP00000000" rel="noopener noreferrer" target="_blank">MIZP00000000</a> and are presented in the manuscript by Checinska Sielaff A et al. [<a aria-describedby="B21-life-11-00960" class="usa-link" href="#B21-life-11-00960">21</a>].</p></section></section></section></section></article>
</main></body></html>